
    <html>
        <body>
            <search-app>
                <article class="result" itemscope="" itemtype="http://schema.org/ScholarlyArticle">
    <h1 itemprop="pageTitle">US6287769B1 - Method of amplifying DNA fragment, apparatus for amplifying DNA fragment, method of assaying microorganisms, method of analyzing microorganisms and method of assaying contaminant 
        - Google Patents</h1><section itemprop="abstract" itemscope="">
<h2>Abstract</h2>
<div html="" itemprop="content"><abstract lang="EN" load-source="patent-office" mxw-id="PA72611741">
<div class="abstract">An apparatus for amplifying DNA fragments is formed by a support plate. A plurality of openings are formed on the upper surface of the support plate. A plurality of primers having different amplification probabilities are arranged in the plurality of openings in order of the amplification probabilities. A plurality of microorganisms contained in a microorganism flora are simultaneously amplified with all primers by a random PCR method, for obtaining an electrophoretic pattern amplified at the optimum amplification probability for each microorganism. The plurality of microorganisms contained in the microorganism flora can be discriminated by analyzing the electrophoretic pattern.</div>
</abstract>
</div>
</section><section itemprop="description" itemscope="">
<h2>Description</h2>
<div html="" itemprop="content"><div class="description" lang="EN" load-source="patent-office" mxw-id="PDES54723113">
<heading>BACKGROUND OF THE INVENTION</heading>
<div class="description-paragraph">1. Field of the Invention</div>
<div class="description-paragraph">The present invention relates to a method of amplifying DNA fragments, an apparatus for amplifying DNA fragments, a method of assaying microorganisms, a method of analyzing microorganisms and a method of assaying a contaminant.</div>
<div class="description-paragraph">2. Description of the Prior Art</div>
<div class="description-paragraph">In recent years, a garbage disposal for composting organic waste (the so-called kitchen garbage) discharged from a household kitchen or the like is now actively researched and developed. In the garbage disposal, microorganisms such as bacteria and protozoa degrade organic matter to form compost.</div>
<div class="description-paragraph">During the composting process (organic degradation process) in such a garbage disposal, the degree of composting is evaluated by monitoring the temperature or the like. The state of the garbage disposal is adjusted to prepare high-quality compost on the basis of the evaluation.</div>
<div class="description-paragraph">In order to prepare high-quality compost, it is necessary to obtain information of the microorganisms (at least the types of the microorganisms) functioning in the garbage disposal. The information of the microorganisms is also necessary for excellently controlling degradation of the kitchen garbage with the microorganisms. In order to improve soil by adding the prepared compost, it is also important to obtain information of microorganisms contained in the soil.</div>
<div class="description-paragraph">In general, information of microorganisms such as bacteria, for example, is obtained by a method of isolating each bacterium included in the bacteria and biochemically examining the same. However, this method requires much time, and it is difficult to analyze a bacterium which is hard to isolate by this method.</div>
<div class="description-paragraph">On the other hand, a PCR (polymerase chain reaction) method is employed for amplifying DNA (U.S. Pat. Nos. 4,683,195, 4,683,202, 4,965,188, 5,038,852 and 5,333,675). In the PCR method, a primer having a base sequence complementary to that at both ends of DNA (template DNA) to be amplified and heat-resistant DNA polymerase are employed for repeating a cycle formed by three stages of a thermal denaturation step, an annealing (heat treatment) step and an extension reaction step thereby enabling amplification of DNA fragments substantially identical to the template DNA. Employing this PCR method, a prescribed fragment in DNA of one of a small amount of bacteria can be amplified to hundred thousand to million times, for example.</div>
<div class="description-paragraph">In order to employ the PCR method, however, the base sequence of at least at both ends of a part of the template DNA must be known. If the types and base sequences of the microorganisms functioning in the garbage disposal or existing in the soil are unknown, therefore, DNA fragments of the microorganisms cannot be amplified in the conventional PCR method.</div>
<div class="description-paragraph">In this regard, there has been proposed a RAPD (random amplified polymorphic DNA) method or AP-PCR (arbitrarily primed-polymerase chain reaction) method of simultaneously amplifying many types of DNA fragments from a single type of DNA with a single primer, with no information of the base sequence. According to this method, the annealing temperature for the primer is reduced while the magnesium ion concentration in a reaction solution is increased during PCR, thereby reducing sequence specificity of the primer in bonding. Thus, the primer is bonded to chromosome DNA of a microorganism with mismatching, to duplicate DNA fragments.</div>
<div class="description-paragraph">According to the RAPD method or AP-PCR method, some DNA fragments are amplified in a large amount with a single primer, with no information on the base sequence of the DNA to be amplified. A DNA fingerprint is obtained by separating the amplified DNA fragments by gel electrophoresis. The state of the microorganism can be elucidated by analyzing the DNA fingerprint.</div>
<div class="description-paragraph">When applying the conventional RAPD method or AP-PCR method to a group of microorganisms formed by a plurality of microorganisms, however, the number of types of amplified DNA fragments is so large that it is difficult to associate a microorganism which is a template with amplified DNA fragments, and hence it is difficult to discriminate an ecosystem formed by the group of microorganisms.</div>
<div class="description-paragraph">In order to examine presence/absence of contaminants in soil, food or the like and the degree thereof, the soil or food must be analyzed by suitable methods varying with the types of the contaminants. Particularly when examining the contaminated state of organic matter, it is necessary to predict the types of contaminants for analyzing the same since the analytic methods vary with the elements contained in the organic matter. Thus, awaited is a method of effectively predicting the types of the contaminants.</div>
<heading>SUMMARY OF THE INVENTION</heading>
<div class="description-paragraph">An object of the present invention is to provide a method of amplifying DNA fragments capable of correctly discriminating a plurality of different DNA, an apparatus for amplifying DNA fragments, and a highly reliable method of assaying microorganisms and a method of analyzing microorganisms employing the same.</div>
<div class="description-paragraph">Another object of the present invention is to provide a method of assaying a contaminant capable of readily assaying presence/absence of a contaminant in soil, food or the like and the degree thereof.</div>
<div class="description-paragraph">A method of amplifying DNA fragments according to an aspect of the present invention comprises steps of preparing a plurality of primers having different amplification probabilities and simultaneously applying a polymerase chain reaction method of repeating a thermal denaturation step, a primer annealing step and an extension reaction step with polymerase in this order to a plurality of different DNA with each of the plurality of primers, thereby amplifying DNA fragments of the plurality of different DNA. The DNA includes not only that of organisms but also DNA fragments.</div>
<div class="description-paragraph">In the method of amplifying DNA fragments according to this aspect of the present invention, DNA fragments of a plurality of DNA having different base lengths can be amplified by simultaneously applying the polymerase chain reaction method to the plurality of different DNA with each of the plurality of primers having different amplification probabilities. Therefore, a plurality of microorganisms included in a group of microorganisms can be correctly discriminated.</div>
<div class="description-paragraph">Further, a plurality of types of DNA fragments are obtained from single DNA by employing the plurality of primers having different amplification probabilities. Thus, a plurality of information can be obtained from each microorganism included in the group of microorganisms, for improving the precision of assay.</div>
<div class="description-paragraph">Preferably, the method further comprises a step of employing a reference primer having a known base sequence and applying the polymerase chain reaction method to reference DNA having a base sequence complementary to that of the reference primer thereby amplifying a DNA fragment of the reference DNA, simultaneously with amplifying the DNA fragments of the plurality of different DNA.</div>
<div class="description-paragraph">The reference DNA has the base sequence complementary to the reference primer. Therefore, the DNA fragment is reliably amplified from the reference DNA by the polymerase chain reaction method. The amplification efficiency for the DNA fragment of the reference DNA in polymerase chain reaction can be obtained by quantitatively analyzing the amplified DNA fragment of the reference DNA. In this case, the polymerase chain reaction method is simultaneously applied under the same conditions to the reference DNA and the plurality of different DNA, and hence the amplification efficiency obtained from the DNA fragment of the reference DNA is also applicable to the DNA fragments of the plurality of different DNA. Therefore, the quantity of the DNA fragments of the plurality of different DNA can be corrected on the basis of the obtained amplification efficiency.</div>
<div class="description-paragraph">The method may further comprise a step of classifying the DNA fragments amplified from the plurality of different DNA by a discrimination method. Thus, a plurality of microorganisms included in a group of microorganisms can be correctly discriminated.</div>
<div class="description-paragraph">Further, the discrimination method may be electrophoresis.</div>
<div class="description-paragraph">An apparatus for amplifying DNA fragments according to another aspect of the present invention comprises a plurality of reaction solution storage parts and a plurality of primers having different amplification probabilities arranged in the plurality of reaction solution storage parts respectively.</div>
<div class="description-paragraph">In the apparatus for amplifying DNA fragments according to this aspect of the present invention, the plurality of primers having different amplification probabilities are arranged in the plurality of reaction solution storage parts respectively, whereby DNA fragments of a plurality of DNA can be simultaneously amplified by simultaneously applying the polymerase chain reaction method to the plurality of different DNA with each of the plurality of primers having different amplification probabilities. Thus, a plurality of microorganisms included in a group of microorganisms can be correctly discriminated.</div>
<div class="description-paragraph">Further, a plurality of types of DNA fragments are obtained from single DNA by simultaneously employing the plurality of primers having different amplification probabilities. Thus, a plurality of information can be obtained from each microorganism included in a group of microorganisms, for improving the precision of assay.</div>
<div class="description-paragraph">A method of assaying a group of microorganisms according to still another aspect of the present invention comprises steps of preparing a plurality of primers having different amplification probabilities, simultaneously applying a polymerase chain reaction method of repeating a thermal denaturation step, a primer annealing step and an extension reaction step with polymerase in this order to DNA of a plurality of different microorganisms with each of the plurality of primers thereby amplifying DNA fragments of the DNA of the plurality of different microorganisms, and classifying the amplified DNA fragments by a discrimination method for discriminating the plurality of microorganisms included in the group of microorganisms.</div>
<div class="description-paragraph">In the method of assaying a group of microorganisms according to this aspect of the present invention, DNA fragments can be amplified from the plurality of different microorganisms by simultaneously applying the polymerase chain reaction method to the DNA of the plurality of different microorganisms with each of the plurality of primers having different amplification probabilities. Thus, the plurality of microorganisms included in the group of microorganisms can be correctly discriminated.</div>
<div class="description-paragraph">Further, a plurality of types of DNA fragments are obtained from a microorganism by employing the plurality of primers having different amplification probabilities. Thus, a plurality of information can be obtained from each microorganism included in the group of microorganisms, for improving the precision of assay. Consequently, various microorganismal ecosystems can be correctly assayed in a short time.</div>
<div class="description-paragraph">While the number of types of microorganisms forming a group of microorganisms to be assayed and the size of chromosome DNA of the microorganisms are generally unknown, the result of amplification with primers amplifying a proper number of types of DNA fragments can be selected from an electrophoretic pattern by simultaneously employing primers having different amplification probabilities or different orders of amplification probabilities.</div>
<div class="description-paragraph">By employing a plurality of primers having a proper amplification probability selected in the aforementioned manner, DNA fragments can be amplified from microorganisms even if the number of types of the microorganisms forming the group of target microorganisms is unknown, for examining the number of types of the microorganisms forming the group of microorganisms from the number of types of the amplified DNA fragments.</div>
<div class="description-paragraph">By employing a plurality of primers having a proper amplification probability selected in the aforementioned manner, further, DNA fragments can be amplified from a principal microorganism or principal group of microorganisms also when the type of the principal group of microorganism forming the group of microorganisms is unknown, for predicting the size of chromosome DNA of the principal microorganism or the principal group of microorganisms from the number of types of the amplified DNA fragments. In addition, it is possible to examine from the predicted size of the chromosome DNA whether microorganisms forming the principal group of microorganisms belong to bacteria, actinomycetes or protozoa.</div>
<div class="description-paragraph">Preferably, the method further comprises steps of employing a reference primer having a known base sequence and applying the polymerase chain reaction method to reference DNA having a base sequence complementary to that of the reference primer thereby amplifying a DNA fragment of the reference DNA, simultaneously with amplifying DNA fragments of DNA of the plurality of different microorganisms, classifying the DNA fragment amplified from the reference DNA along with the DNA fragments amplified from the DNA of the plurality of different microorganisms by the discrimination method, obtaining the amplification efficiency for the reference DNA on the basis of the result of classification of the DNA fragment amplified from the reference DNA, and correcting the results of classification of the DNA fragments amplified from the plurality of different microorganisms on the basis of the obtained amplification efficiency.</div>
<div class="description-paragraph">In this case, the DNA fragment is reliably amplified from the reference DNA by the polymerase chain reaction method with the reference primer. The amplification efficiency for the DNA fragment of the reference DNA in the polymerase chain reaction can be obtained from the result of classification of the DNA fragment of the reference DNA thus amplified. The amplification efficiency obtained in this manner is also applicable to the DNA fragments of the DNA of the plurality of different microorganisms. Therefore, the quantity of the DNA fragments can be analyzed by correcting the results of classification of the DNA fragments of the DNA of the plurality of different microorganisms on the basis of the obtained amplification efficiency.</div>
<div class="description-paragraph">The discrimination method may be electrophoresis. In this case, the amplified DNA fragments are classified by the electrophoresis. Thus, the amplified DNA fragments appear in an electrophoretic pattern as bands classified in response to the size.</div>
<div class="description-paragraph">Preferably, the method further comprises steps of employing a DNA size marker along with the DNA fragments amplified from the DNA of the plurality of different microorganisms for the electrophoresis, staining an electrophoretic pattern obtained by the electrophoresis, and correcting the gradient of the electrophoretic pattern on the basis of the luminous intensity of a band of the DNA fragment amplified from the reference DNA or the DNA size marker.</div>
<div class="description-paragraph">In this case, the electrophoretic pattern obtained by the electrophoresis is stained for acquiring a stained electrophoretic image. Influence exerted by the degree of staining or the degree of exposure in image acquisition can be eliminated by correcting the gradient of the electrophoretic pattern on the basis of the luminous intensity of the band of the DNA fragment amplified from the reference DNA or the DNA size marker. Consequently, luminous intensities of bands in the electrophoretic pattern can be correctly compared.</div>
<div class="description-paragraph">Preferably, the method further comprises steps of setting a threshold based on the luminous intensity of the band of the DNA fragment amplified from the reference DNA or the DNA size marker in the electrophoretic pattern, and analyzing the group of microorganisms on the basis of a band having a luminous intensity exceeding the threshold in the electrophoretic pattern.</div>
<div class="description-paragraph">In this case, a band having a luminous intensity less than the threshold is that of a DNA fragment having low amplification efficiency and low reproducibility. The band having the luminous intensity exceeding the threshold is that of a DNA fragment having high amplification efficiency and high reproducibility. Thus, only the DNA fragment having high amplification efficiency and high reproducibility can be analyzed by employing the band having the luminous intensity exceeding the threshold. Thus, reliability of information obtained by analysis is improved.</div>
<div class="description-paragraph">The method may further comprise steps of isolating a bacterium, applying the polymerase chain reaction method to the isolated bacterium with each of the plurality of primers thereby amplifying a DNA fragment of DNA of the bacterium, classifying the DNA fragment amplified from the DNA of the bacterium by the discrimination method, and analyzing the results of discrimination of the DNA fragments amplified from the DNA of the plurality of different microorganisms on the basis of the result of classification of the DNA fragment amplified from the DNA of the bacterium.</div>
<div class="description-paragraph">Thus, bacteria of the same type as the bacterium isolated from the plurality of different microorganisms can be specified.</div>
<div class="description-paragraph">A method of analyzing a group of microorganisms according to a further aspect of the present invention comprises steps of preparing a plurality of primers having different amplification probabilities, simultaneously applying a polymerase chain reaction method of repeating a thermal denaturation step, a primer annealing step and an extension reaction step with polymerase in this order to DNA of a plurality of different microorganisms included in a first group of microorganisms with each of the plurality of primers thereby amplifying DNA fragments of the DNA of the plurality of different microorganisms included in the first group of microorganisms, classifying the amplified DNA fragments of the DNA of the plurality of microorganisms included in the first group of microorganisms by a discrimination method, simultaneously applying the polymerase chain reaction method to DNA of a plurality of different microorganisms included in a second group of microorganisms with each of the plurality of primers thereby amplifying DNA fragments of the DNA of the plurality of different microorganisms included in the second group of microorganisms, classifying the amplified DNA fragments of the DNA of the plurality of microorganisms included in the second group of microorganisms by the discrimination method, and comparing the results of classification of the first group of microorganisms with those of the second group of microorganisms.</div>
<div class="description-paragraph">Thus, the plurality of microorganisms included in the first group and second group of microorganisms can be correctly discriminated, while microorganisms included in both of the first and second groups of microorganisms and those included in the first or second group of microorganisms can be specified.</div>
<div class="description-paragraph">The discrimination method may be electrophoresis.</div>
<div class="description-paragraph">A method of analyzing groups of microorganisms according to a further aspect of the present invention comprises steps of sampling a group of microorganisms at a plurality of points of time, simultaneously applying a polymerase chain reaction method of repeating a thermal denaturation step, a primer annealing step and an extension reaction step with polymerase in this order to DNA of a plurality of different microorganisms included in the group of microorganisms with each of a plurality of primers having different amplification probabilities thereby amplifying DNA fragments of the DNA of the plurality of different microorganisms included in the group of microorganisms, classifying the amplified DNA fragments by a discrimination method, and analyzing time change of the states of the group of microorganisms on the basis of the results of classification at the plurality of points of time.</div>
<div class="description-paragraph">Thus, time change of the number of types of a plurality of microorganisms can be analyzed by analyzing the DNA fragments amplified from the plurality of different microorganisms included in the group microorganisms.</div>
<div class="description-paragraph">The discrimination method may be electrophoresis.</div>
<div class="description-paragraph">A method of assaying a contaminant according to a further aspect of the present invention comprises steps of applying a polymerase chain reaction method of repeating a thermal denaturation step, a primer annealing step and an extension reaction step with polymerase in this order to DNA of a microorganism related to a contaminant with each of a plurality of primers having different amplification probabilities thereby amplifying a DNA fragment of the DNA of the microorganism, classifying the DNA fragment amplified from the DNA of the microorganism by a discrimination method, preserving the relation between the type of the microorganism and the result of classification in a database, simultaneously applying the polymerase chain reaction method to DNA of a plurality of different microorganisms contained in an object of assay with each of the plurality of primers thereby amplifying DNA fragments of the DNA of the plurality of different microorganisms, classifying the DNA fragments amplified from the DNA of the plurality of different microorganisms by a discrimination method, and retrieving the types of the plurality of different microorganisms from the database on the basis of the results of classification of the DNA fragments amplified from the plurality of different microorganisms.</div>
<div class="description-paragraph">In the method of assaying a contaminant according to this aspect of the present invention, the DNA fragment of the DNA of the microorganism related to the contaminant is amplified by the polymerase chain reaction method employing each of the plurality of primers having different amplification probabilities, for classifying this DNA fragment by the discrimination method. The result of classification of the DNA fragment obtained in this manner and the type of the microorganism related to the contaminant are preserved in the database. On the other hand, DNA fragments of the DNA of the plurality of different microorganisms contained in the target are amplified with a plurality of primers similar to the above by the polymerase chain reaction method for classifying the DNA fragments by a classification method similar to the above. The plurality of microorganisms contained in the object of assay can be correctly discriminated by analyzing the results of classification of the DNA fragments of the DNA of the plurality of different microorganisms obtained in this manner. Further, a microorganism related to a contaminant contained in the target can be specified by retrieving the type of the microorganism from the database on the basis of the results of classification. Thus, the contaminant contained in the object of assay can be predicted.</div>
<div class="description-paragraph">The method may further comprise a step of determining presence/absence of the contaminant in the target on the basis of the result of retrieval of the database.</div>
<div class="description-paragraph">Thus, the contaminant contained in the object of assay and the contaminated state can be effectively predicted, whereby the contaminant of the object of assay can be assayed by an analytical method suitable for the predicted contaminant.</div>
<div class="description-paragraph">The database preferably preserves data of a plurality of types of microorganisms and results of classification corresponding thereto.</div>
<div class="description-paragraph">Thus, a plurality of types of microorganisms related to the contaminant can be specified in the object of assay. Thus, a plurality of contaminants contained in the object of assay can be simultaneously assayed.</div>
<div class="description-paragraph">The discrimination method may be electrophoresis, and the results of classification may be band patterns of an electrophoretic pattern, and the database may preserve the relation between the types of the microorganisms and the band patterns of the electrophoretic pattern.</div>
<div class="description-paragraph">Thus, the types of the microorganisms can be retrieved from the band patterns of the electrophoretic pattern.</div>
<div class="description-paragraph">The foregoing and other objects, features, aspects and advantages of the present invention will become more apparent from the following detailed description of the present invention when taken in conjunction with the accompanying drawings.</div>
<heading>BRIEF DESCRIPTION OF THE DRAWINGS</heading>
<div class="description-paragraph">FIG. 1 is a model diagram showing the process in a first cycle in a random PCR method;</div>
<div class="description-paragraph">FIG. 2 is a model diagram showing the process in a first time of a second cycle in the random PCR method;</div>
<div class="description-paragraph">FIG. 3 is a model diagram showing the process in a second time of the second cycle in the random PCR method;</div>
<div class="description-paragraph">FIG. 4 is a model diagram showing an exemplary apparatus for amplifying DNA fragments employed for a method of amplifying DNA fragments and a method of assaying a group of microorganisms according to the present invention;</div>
<div class="description-paragraph">FIG. 5 is a model diagram showing another exemplary apparatus for amplifying DNA fragments employed for the method of amplifying DNA fragments and the method of assaying a group of microorganisms according to the present invention;</div>
<div class="description-paragraph">FIG. 6 is a flow chart showing an exemplary image processing method employed for the method of assaying a group of microorganisms according to the present invention;</div>
<div class="description-paragraph">FIG. 7 is a flow chart showing an exemplary image processing method employed for the method of assaying a group of microorganisms according to the present invention;</div>
<div class="description-paragraph">FIG. 8 illustrates time change of DNA fragments amplified from a group of microorganisms inhabiting in the tank of a garbage disposal;</div>
<div class="description-paragraph">FIG. 9 illustrates the distribution of primer numbers every six bands;</div>
<div class="description-paragraph">FIG. 10 illustrates the numbers of primers for the respective band numbers as to primers each having not more than 15 bands;</div>
<div class="description-paragraph">FIG. 11 illustrates band numbers for GC contents of 216 primers;</div>
<div class="description-paragraph">FIG. 12 shows an electrophoretic pattern of a DNA fragment amplified from chromosome DNA of an isolated bacterium with primers of sequence Nos. 64 to 71;</div>
<div class="description-paragraph">FIG. 13 shows an electrophoretic pattern of DNA fragments amplified from a mixture of chromosome DNA of five types of isolated bacteria with primers of sequence Nos. 64 to 71;</div>
<div class="description-paragraph">FIG. 14 shows states of a filtrate observed with a microscope in filtration steps; and</div>
<div class="description-paragraph">FIG. 15 shows an electrophoretic pattern of DNA fragments amplified from chromosome DNA of microorganisms contained in the tank of a garbage disposal with primers of sequence Nos. 64 to 71.</div>
<heading>DESCRIPTION OF THE PREFERRED EMBODIMENTS</heading>
<div class="description-paragraph">A method of amplifying DNA fragments, an apparatus for amplifying DNA fragments and a method of assaying a group of microorganisms according to an embodiment of the present invention are now described.</div>
<div class="description-paragraph">FIG. 4 is a model diagram showing an exemplary apparatus for amplifying DNA fragments employed for the method of amplifying DNA fragments and the method of assaying a group of microorganisms according to the present invention.</div>
<div class="description-paragraph">Referring to FIG. 4, the apparatus for amplifying DNA fragments is formed by a support plate <b>50</b> called a titer plate. A plurality of openings (reaction solution storage parts) <b>51</b> are formed on the upper surface of the support plate <b>50</b>. In the example shown in FIG. 4, 96 openings <b>51</b> are formed on the upper surface of the support plate <b>50</b>.</div>
<div class="description-paragraph">Primers having an amplification probability for amplifying a DNA fragment for 10<sup>9 </sup>bp (base pairs) are arranged in 32 openings <b>51</b> closer to an end of the support plate <b>50</b>. Primers having an amplification probability for amplifying a DNA fragment for 10<sup>8 </sup>bp are arranged in 32 openings <b>51</b> at the center of the support plate <b>50</b>. Primers having an amplification probability for amplifying a DNA fragment for 10<sup>7 </sup>bp are arranged in 32 openings <b>51</b> closer to another end of the support plate <b>50</b>.</div>
<div class="description-paragraph">FIG. 5 is a model diagram showing another exemplary apparatus for amplifying DNA fragments employed for the method of amplifying DNA fragments and the method of assaying microorganisms according to the present invention.</div>
<div class="description-paragraph">Also in the apparatus for amplifying DNA fragments shown in FIG. 5, a plurality of openings <b>51</b> are formed on the upper surface of a support plate <b>50</b>. A plurality of primers having different amplification probabilities are continuously arranged in the plurality of openings <b>51</b> in order of the amplification probabilities.</div>
<div class="description-paragraph">Consider that the base length of chromosome DNA of a microorganism is about 10<sup>7 </sup>bp. When setting the amplification probability to amplify a DNA fragment for 10<sup>8 </sup>bp in order to assay a microorganism flora containing 10 types of microorganisms, for example, a DNA fragment is amplified, i.e., a single type of microorganism is detected for 10 types of microorganisms. Therefore, 10 types of microorganisms can be detected by preparing 10 types of primers. In practice, however, 20 to 30 types of primers are preferably prepared in order to cope with unbalanced amplification.</div>
<div class="description-paragraph">If the microorganism flora contains a number of microorganisms or chromosome DNA of the microorganisms has a large base length, it is effective to set the amplification probability at a low value. If the microorganism flora contains a small number of microorganisms or the chromosome DNA of the microorganisms has a small base length, on the contrary, it is effective to set the amplification probability at a high value.</div>
<div class="description-paragraph">In the apparatus for amplifying DNA fragments shown in FIG. 4, primers having different amplification probabilities or different orders of amplification probabilities are arranged in order, while 32 types of primers are prepared for each amplification probability. In the apparatus for amplifying DNA fragments shown in FIG. 5, on the other hand, types of primers having different amplification probabilities are prepared in order of the amplification probabilities.</div>
<div class="description-paragraph">In the apparatus for amplifying DNA fragments shown in FIG. 5, 94 types of primers having different amplification probabilities are arranged in the openings <b>51</b> in order of the amplification probabilities while a negative control is arranged in an opening <b>51</b> <i>a </i>and a positive control is arranged in an opening <b>51</b> <i>b, </i>for example.</div>
<div class="description-paragraph">The positive control is a sample employed for a control experiment for eliminating errors caused in a series of steps of amplifying DNA fragments. In general, the amplification efficiency in amplification of DNA fragments is conceivably influenced by errors caused in a series of steps such as errors in concentrations of DNA polymerase, magnesium and the like in preparation of a reaction solution for random PCR, errors in the degree of activation of an employed reagent and the temperature in DNA fragment amplification and the like, for example. In order to eliminate such errors, a reaction solution containing a primer and template DNA for amplifying a known type of DNA fragment quantitatively is prepared as the positive control. The DNA fragment amplified by the positive control can be quantitatively analyzed by electrophoresis. Therefore, the amplification efficiency for the DNA fragment can be obtained from the quantity of the DNA fragment amplified in the positive control. The influence exerted by the errors caused in the series of steps of amplification of DNA fragments can be eliminated by correcting the quantity of DNA fragments in the 94 types of reaction solutions for random PCR on the basis of the amplification efficiency obtained in the aforementioned manner. Consequently, the amplified DNA fragments can be compared in quantity.</div>
<div class="description-paragraph">On the other hand, the negative control is a sample employed in a control experiment for confirming that amplified DNA fragments belong to an object group of microorganisms or a microorganism to be analyzed. In general, microorganisms such as bacteria exist in various places in the air or the like. In preparation of the reaction solutions for random PCR, therefore, the reaction solutions may be contaminated by microorganisms. In this case, it is impossible to determine whether the amplified DNA fragments are derived from the object microorganisms or the contaminating microorganisms. Therefore, a reaction solution for random PCR containing a primer with no template DNA is prepared as the negative control. Random PCR and electrophoresis are performed with the negative control, to confirm that no band appears on the electrophoretic pattern of the negative control. Thus, it is possible to confirm that the amplified DNA fragments are derived from the object group of microorganisms or the object microorganism to be analyzed.</div>
<div class="description-paragraph">Each of the primers employed for the positive control and the negative control may have the same base sequence as one or two of the 94 types of primers.</div>
<div class="description-paragraph">A plurality of microorganisms contained in the microorganism flora are simultaneously amplified with all primers by a random PCR (polymerase chain reaction) method described later. The random PCR method is a reaction method of chain-reactionally amplifying DNA fragments from a plurality of different microorganisms having unknown base sequences with primers having a specific base sequence.</div>
<div class="description-paragraph">As shown in FIG. 4, a number of bands appear in the electrophoretic pattern of the DNA fragments amplified with the primers having a high amplification probability. On the contrary, a small number of bands appear in the electrophoretic pattern of the DNA fragments amplified with the primers having a low amplification probability. Thus, microorganisms can be analyzed on the basis of an electrophoretic pattern amplified at the optimum amplification probability in response to the number of types of microorganisms contained in the object microorganism flora, the size of chromosomes of the microorganisms, the number of types of DNA fragments or the size of the DNA fragments.</div>
<div class="description-paragraph">The random PCR method employed for the method of amplifying DNA fragments and the method of assaying a group of microorganisms according to the present invention is now described.</div>
<div class="description-paragraph">In the random PCR method employed in the method of amplifying DNA fragments according to the present invention, the following three stages of steps are repeated similarly to the conventional PCR method. In this random PCR method, primers having a specific base sequence are employed for a plurality of different microorganisms having unknown base sequences thereby amplifying an analyzable quantity of DNA fragments, as described later.</div>
<div class="description-paragraph">(1) Thermal Denaturation Step DNA (initial state) or a DNA fragment is heated and denatured into single strands (DNA strands).</div>
<div class="description-paragraph">(2) Primer Annealing Step (Primer Bonding Step) Heat treatment is performed to bond a primer to an end of an amplification region of each DNA strand.</div>
<div class="description-paragraph">(3) Extension Reaction Step with Polymerase (Duplication Step with Polymerase)</div>
<div class="description-paragraph">Starting from the primer, a complementary strand is synthesized by polymerase to form a double strand.</div>
<div class="description-paragraph">A cycle formed by the above steps (1) to (3) is repeated.</div>
<div class="description-paragraph">The random PCR method is now described more specifically. First prepared is a reaction solution containing a plurality of different microorganisms, a buffer solution for polymerase chain reaction, primers, heat-resistant thermophile DNA polymerase and four types of 5′-deoxyribonucleotide triphosphosphates (dATP, dGTP, dCTP and dTTP) serving as substrates.</div>
<div class="description-paragraph">The DNA polymerase is an enzyme having substrates of four types of 5′-deoxyribonucleotide triphosphosphates for catalyzing polymerization reaction of DNA strands having a base sequence complementary to template DNA. The directionality of polymerization of DNA strands with the DNA polymerase is at the 5′ to 3′ ends. The primers are DNA fragments (short oligonucleotides) having 3′-OH groups essential for the action of the DNA polymerase on ends thereof. In the present invention, primers having a specific base sequence and a specific base length are employed.</div>
<div class="description-paragraph">The following random PCR method is applied to the above reaction solution:</div>
<div class="description-paragraph">As a first cycle, a thermal denaturation step of keeping the above reaction solution at 94° C. for two minutes, for example, a primer annealing step of keeping the above reaction solution at 45° C. for two minutes, for example, and an extension reaction step with polymerase for keeping the above reaction solution at 72° C. for three minutes, for example, are carried out in this order. The time for the thermal denaturation step in this cycle is set longer than those in second and third cycles, in order to completely separate long complete DNA into single strands.</div>
<div class="description-paragraph">Then, as the second cycle, a thermal denaturation step of keeping the above reaction solution at 94° C. for one minute, for example, a primer annealing step of keeping the above reaction solution at 45° C. for two minutes, for example, and an extension reaction step with polymerase for keeping the above reaction solution at 72° C. for three minutes, for example, are repeated in this order 33 times, for example.</div>
<div class="description-paragraph">Finally, as the third cycle, a thermal denaturation step of keeping the above reaction solution at 94° C. for one minute, for example, a primer annealing step of keeping the above reaction solution at 45° C. for two minutes, for example, and an extension reaction step with polymerase for keeping the above reaction solution at 72° C. for 10 minutes, for example, are carried out in this order. In this cycle, the time for the extension reaction step with polymerase is set longer than those in the first and second cycles, in order to finally complete duplication.</div>
<div class="description-paragraph">The first cycle, a first time of the second cycle and a second time of the second cycle are now described with reference to FIGS. 1 to <b>3</b>. FIGS. 1 to <b>3</b> are model diagrams showing single strands of DNA etc. with the base sequence of parts bonded to the primers.</div>
<div class="description-paragraph">Referring to FIGS. 1 to <b>3</b>, the primers have a base sequence (sequence No. 69) of GGCTTCGAATCG. T stands for thymine, A for adenine, G for guanine, and C for cytosine.</div>
<div class="description-paragraph">As shown at (a) in FIG. 1, long DNA (complete DNA) 1 contained in a plurality of different microorganisms initially exists in the above reaction solution. The following description is made with reference to single complete DNA 1.</div>
<div class="description-paragraph">First, the thermal denaturation step longer than those in the second and third cycles is carried out in the first cycle, so that the long DNA 1 is heated, denatured and separated into two single strands (DNA strands) <b>2</b> <i>a </i>and <b>2</b> <i>b </i>from a double-stranded state, as shown at (b) in FIG. <b>1</b>.</div>
<div class="description-paragraph">Then, in the primer annealing step, each primer <b>11</b> <i>a </i>is bonded to be arranged (complementarily arranged) on a compatible position of each of the single strands <b>2</b> <i>a </i>and <b>2</b> <i>b </i>compatible with its base sequence, as shown at (c) in FIG. <b>1</b>. The term compatible position stands for the position of the base sequence to be bonded as viewed from that of the primer or the position of a base sequence similar to that to be bonded as viewed from the base sequence of the primer. In the aforementioned random PCR method, the annealing temperature for the primer annealing step is set at a low value so that the primer is bonded also to a part of the DNA strand having a base sequence similar to its base sequence. In other words, the primer can be bonded not only to the position of the single strand having the base sequence completely complementary to its base sequence but also to the single strand with slight mismatching. Referring to FIGS. 1 to <b>3</b>, each primer is bonded to the position of the base sequence to be bonded as viewed from its base sequence, for simplifying the illustration.</div>
<div class="description-paragraph">Then, in the extension reaction step with polymerase, extension reaction is caused by the polymerase so that single strands <b>2</b> <i>c </i>and <b>2</b> <i>d </i>extend along the single strands <b>2</b> <i>a </i>and <b>2</b> <i>b </i>respectively to form double strands <b>3</b> <i>a </i>and <b>3</b> <i>b, </i>as shown at (d) in FIG. <b>1</b>.</div>
<div class="description-paragraph">In the first time of the second cycle, the strands <b>3</b> <i>a </i>and <b>3</b> <i>b </i>doubled in the first cycle are separated into the single strands (DNA strands) <b>2</b> <i>a </i>and <b>2</b> <i>c </i>and the single strands (DNA strands) <b>2</b> <i>b </i>and <b>2</b> <i>d </i>respectively through the thermal denaturation step, while the following description is made with reference to the single strand <b>2</b> <i>c </i>separated from the strand <b>3</b> <i>a. </i> </div>
<div class="description-paragraph">As shown in FIG. 2, a primer <b>11</b> <i>b </i>is bonded to the single strand <b>2</b> <i>c </i>separated through the thermal denaturation step to be arranged on a compatible position in the subsequent primer annealing step. Thereafter in the extension reaction step with polymerase, extension reaction is caused by the polymerase so that a single strand <b>2</b> <i>e </i>extends along the single strand <b>2</b> <i>c </i>to form a double strand <b>3</b> <i>d. </i> </div>
<div class="description-paragraph">Thereafter the double strand <b>3</b> <i>d </i>is separated into the single strands <b>2</b> <i>c </i>and <b>2</b> <i>e </i>through the thermal denaturation step in the second time of the second cycle as shown in FIG. 3, while the following description is made with reference to the single strand <b>2</b> <i>e </i>separated from the strand <b>3</b> <i>d. </i> </div>
<div class="description-paragraph">As shown in FIG. 3, a primer <b>11</b> <i>c </i>is bonded to the single strand <b>2</b> <i>e </i>separated through the thermal denaturation step to be arranged on a compatible position in the subsequent primer annealing step. Thereafter in the extension reaction step with polymerase, extension reaction is caused by the polymerase so that a single strand <b>2</b> <i>f </i>extends along the single strand <b>2</b> <i>e </i>to form a double strand (DNA fragment) <b>3</b> <i>e. </i> </div>
<div class="description-paragraph">The DNA fragment is thus formed so that another DNA fragment is formed from this DNA fragment while DNA fragments are formed from another DNA of the same type followed by chain-reactional continuation of similar reaction, whereby DNA fragments are amplified by this method.</div>
<div class="description-paragraph">Gel electrophoresis is applied to the DNA fragments amplified by the aforementioned random PCR method, in order to classify the same in response to the different microorganisms. The existential states of the microorganisms can be estimated by analyzing bands on the electrophoretic pattern. Alternatively, the aforementioned random PCR method is applied to microorganisms sampled at time intervals for amplifying DNA fragments and gel electrophoresis is similarly applied to the amplified DNA fragments. Time change of the existential states of the microorganisms can be estimated by analyzing state change of bands on the electrophoretic pattern.</div>
<div class="description-paragraph">When a primer is strongly bonded to and arranged on a compatible position of template DNA having a prescribed base sequence in amplification of DNA fragments by the random PCR method, amplification efficiency for the DNA fragments is high. The DNA fragments amplified with such a primer appear as clear bands having high reproducibility on an electrophoretic pattern. When bonding between the primer and the compatible position of the template DNA is weak, on the other hand, the primer is bonded to another position having stronger bonding than the compatible position of the template DNA. Thus, the amplification reaction of DNA fragments so competitively progresses that the amplification efficiency for the DNA fragments is reduced if the bonding between the primer and the template DNA is weak. DNA fragments amplified with such a primer having weak bonding appear as unclear bands having low reproducibility on the electrophoretic pattern. When containing such DNA fragments having low reproducibility, data obtained by the random PCR method are inferior in total reliability.</div>
<div class="description-paragraph">In order to improve the reliability of data obtained by the random PCR method, the following image processing is performed in the method of assaying a group of microorganisms according to the present invention:</div>
<div class="description-paragraph">FIG. <b>6</b> and FIG. 7 are flow charts showing an example of the method of assaying a group of microorganisms according to the present invention.</div>
<div class="description-paragraph">As shown in FIG. <b>6</b> and FIG. 7, prescribed quantities of a buffer solution, MgCl<sub>2</sub>, bases (A, T, G and C), primers, DNA and DNA polymerase are mixed with each other to prepare a reaction solution for random PCR (step S<b>1</b>). In the random PCR employing a plurality of primers having different amplification probabilities, the reaction solution for random PCR is prepared for each primer. When employing 94 types of primers, for example, 94 types of reaction solutions for random PCR containing different primers are prepared. All of the 94 types of reaction solutions for random PCR are prepared at the same time.</div>
<div class="description-paragraph">Simultaneously with preparation of the reaction solutions for random PCR, a positive control and a negative control are prepared (step S<b>2</b>). The details of the positive control and the negative control are described above with reference to the apparatus for amplifying DNA fragments.</div>
<div class="description-paragraph">Then, DNA fragments are amplified by the random PCR method with the apparatus for amplifying DNA fragments (step S<b>3</b>). Thereafter the DNA fragments amplified in each reaction solution are fractioned per size by electrophoresis with the reaction solutions for random PCR, the positive control and the negative control and at the same time the DNA size marker having a known concentration and quantity is also fractioned (step S<b>4</b>). Further, electrophoretic patterns obtained through the electrophoresis are stained with a fluorochrome (step S<b>5</b>), for acquiring fluorescent images irradiated with ultraviolet light (step S<b>6</b>). Further, the electrophoretic images are incorporated in a computer with a scanner, and subjected to data processing for measuring the size and the luminous intensities of each band (step S<b>7</b>). If no band appears in the electrophoretic pattern of the negative control, it is possible to confirm that the amplified DNA fragments are derived from an object of a group of microorganisms.</div>
<div class="description-paragraph">Then, the amplified DNA fragment of the positive control is quantitatively analyzed by comparing the luminous intensity of the band of the positive control with that of the DNA size marker having a known quantity, for obtaining the amplification efficiency in the DNA fragment amplification reaction (step S<b>8</b>). Alternatively, the positive control may be quantitatively analyzed by another method of measuring absorption of ultraviolet light at 260 nm after purifying the DNA fragment, for example.</div>
<div class="description-paragraph">Further, the luminous intensities of the bands of the reaction solutions for random PCR are corrected with reference to the luminous intensity of the band of the quantitatively analyzable DNA size marker. Thus, the gradients of the electrophoretic images are corrected (step S<b>9</b>).</div>
<div class="description-paragraph">In general, errors are caused in the gradients of the electrophoretic images depending on the degrees of staining with ethidium bromide and the degrees of exposure in the image. Such errors can be eliminated by correcting the gradients of the electrophoretic images on the basis of the luminous intensity of the band of the DNA size marker having a known concentration.</div>
<div class="description-paragraph">Then, a threshold is set on the basis of the luminous intensity of the band of the quantitatively analyzable DNA size marker obtained by measurement, for removing bands having luminous intensities less than the threshold (step S<b>10</b>). Thus, bands having low amplification efficiency and low reproducibility can be removed. Bands having high amplification efficiency and high reproducibility thus obtained are analyzed for obtaining reliable data of the microorganisms.</div>
<div class="description-paragraph">In the random PCR method, a plurality of types of primers are employed for amplifying DNA fragments from chromosome DNA of one of bacteria at prescribed amplification probabilities, and hence the number of types of bacteria is correlated with that of the amplified DNA fragments such that the total number of the amplified DNA fragments reflects the number of types of bacteria. Thus, the number of types of a plurality of microorganisms included in the group of microorganisms can be examined by analyzing the DNA fragments. As to DNA fragments amplified by the same primer, bands appear on the same positions of electrophoretic patterns. Thus, the same type of microorganisms can be discriminated between a microorganism flora and isolated microorganisms or between microorganism florae by analyzing the bands appearing on the same positions.</div>
<div class="description-paragraph">When applying the aforementioned method of assaying a group of microorganisms to microorganisms sampled at time intervals, change of existential states of the microorganisms can be estimated from change of the total number of amplified DNA fragments.</div>
<div class="description-paragraph">When applying the aforementioned method of assaying a group of microorganisms to microorganisms sampled from the tank of a garbage disposal at time intervals, for example, the total number of bands appearing in electrophoretic patterns changes as shown in FIG. <b>8</b>.</div>
<div class="description-paragraph">In the tank containing wood chips as a treating carrier, only a small amount of microorganisms exist and hence the total band number is small in the initial state, as shown in FIG. <b>8</b>. However, the total band number increases as the days go on. The total band number increases since the tank is contaminated by a plurality of types of bacteria existing in kitchen garbage daily introduced into the tank, while the number of bacteria increases since those slow in multiplication multiply with time. Assuming that the mean band number of a single type of bacterium is 20, it is estimable that 20 types of bacteria exist on the fifth day since the band number of the fifth day is 400.</div>
<div class="description-paragraph">When applying the aforementioned method of amplifying DNA fragments to a plurality of microorganisms sampled from the tank of a garbage disposal or soil, existential states of the microorganisms contained in the tank of the garbage disposal or the soil can be estimated. When examining the existential states of microorganisms similarly sampled from the tank of the garbage disposal or the soil at time intervals, time change of the existential states of microorganisms in the tank of the garbage disposal or the soil can be estimated. In a degradation process of organic matter contained in kitchen garbage, further, the degraded state of the organic matter can also be estimated.</div>
<div class="description-paragraph">When changing the conditions of the tank of the garbage disposal in accordance with the results of the estimation, kitchen garbage can be excellently treated while preparing excellent compost.</div>
<div class="description-paragraph">Alternatively, a microorganism isolated from a group of microorganisms is identified by biochemical examination while analyzing the band pattern of DNA of the identified microorganism by the aforementioned method of assaying a group of microorganisms. Thus, the types of a plurality of microorganisms and band patterns of DNA thereof may be analyzed for establishing a database of the band patterns of the DNA of the microorganisms on the basis of the obtained data. In this case, DNA of microorganisms sampled from the tank of the garbage disposal or soil is analyzed by the aforementioned method of assaying a group of microorganisms and the types of the microorganisms are retrieved from the database on the basis of obtained band patterns of the DNA of the microorganisms. Thus, the types of the microorganisms forming the group of microorganisms can be examined. In particular, the method of retrieving the types of microorganisms from the aforementioned database is effective to find out soil, food or the like contaminated by toxic contaminants such as mercury, arsenic, dioxin, environmental hormones and the like and recognize the contaminated state. The database is searched on the basis of the band patterns of the DNA of the microorganisms for identifying the types of the microorganisms forming the object group of microorganisms. When the identified microorganisms include those related to the contaminants, such a possibility is suggested that the contaminants are contained in the soil from which the microorganisms have been sampled. Further, the existential degree of the microorganisms related to the contaminants suggests the contaminated state of the soil. Tables 1 and 2 show exemplary contaminants and microorganisms related thereto.</div>
<div class="description-paragraph">
<tables>
<patent-tables colsep="0" frame="none" rowsep="0">
<table align="left" class="description-table" cols="3" colsep="0" rowsep="0" width="100%">
<thead valign="bottom">
<tr class="description-tr">
<td class="description-td" colspan="3" morerows="0" nameend="3" namest="1" rowsep="1" valign="top">TABLE 1</td>
</tr>
<tr class="description-tr">
<td align="center" class="description-td" colspan="3" morerows="0" nameend="3" namest="1" rowsep="1" valign="top"> </td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top"> </td>
<td class="description-td" morerows="0" valign="top">Substance</td>
<td class="description-td" morerows="0" valign="top"> </td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top">Classification</td>
<td class="description-td" morerows="0" valign="top">Name</td>
<td class="description-td" morerows="0" valign="top">Related Microorganism</td>
</tr>
<tr class="description-tr">
<td align="center" class="description-td" colspan="3" morerows="0" nameend="3" namest="1" rowsep="1" valign="top"> </td>
</tr>
</thead>
<tbody><tr class="description-tr">
<td class="description-td" morerows="0" valign="top">Agricultural</td>
<td class="description-td" morerows="0" valign="top">organic</td>
<td class="description-td" morerows="0" valign="top">
<i>Pseudomonas diminuta</i>
</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top">chemicals</td>
<td class="description-td" morerows="0" valign="top">phosphorus</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top"> </td>
<td class="description-td" morerows="0" valign="top">such as</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top"> </td>
<td class="description-td" morerows="0" valign="top">parathion</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top"> </td>
<td class="description-td" morerows="0" valign="top">carbamate</td>
<td class="description-td" morerows="0" valign="top">Achromobacter sp.</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top"> </td>
<td class="description-td" morerows="0" valign="top">triazine</td>
<td class="description-td" morerows="0" valign="top">Rhodococcus sp.</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top"> </td>
<td class="description-td" morerows="0" valign="top"> </td>
<td class="description-td" morerows="0" valign="top">
<i>Rhodococcus corallinus.</i>
</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top"> </td>
<td class="description-td" morerows="0" valign="top"> </td>
<td class="description-td" morerows="0" valign="top">
<i>Phanerochaete chrysosporium</i>
</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top"> </td>
<td class="description-td" morerows="0" valign="top">organic</td>
<td class="description-td" morerows="0" valign="top">
<i>Alcaligenes eutrophus</i>
</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top"> </td>
<td class="description-td" morerows="0" valign="top">chlorine</td>
<td class="description-td" morerows="0" valign="top">Flavobacterium sp.</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top"> </td>
<td class="description-td" morerows="0" valign="top"> </td>
<td class="description-td" morerows="0" valign="top">
<i>Pseudomonas cepacia</i>
</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top">Insecticide</td>
<td class="description-td" morerows="0" valign="top">γ-BHC</td>
<td class="description-td" morerows="0" valign="top">
<i>Pseudomonas paucimobilis</i>
</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top"> </td>
<td class="description-td" morerows="0" valign="top"> </td>
<td class="description-td" morerows="0" valign="top">
<i>Sphingomonas paucimobilis</i>
</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top"> </td>
<td class="description-td" morerows="0" valign="top">PCP</td>
<td class="description-td" morerows="0" valign="top">
<i>Rhodococcus chlorophenolicus</i>
</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top"> </td>
<td class="description-td" morerows="0" valign="top"> </td>
<td class="description-td" morerows="0" valign="top">Pseudomonas sp.</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top"> </td>
<td class="description-td" morerows="0" valign="top"> </td>
<td class="description-td" morerows="0" valign="top">
<i>Phanerochaete chrysosporium</i>
</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top"> </td>
<td class="description-td" morerows="0" valign="top"> </td>
<td class="description-td" morerows="0" valign="top">
<i>Phanerochaete sordida</i>
</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top">Plastic</td>
<td class="description-td" morerows="0" valign="top">polyvinyl</td>
<td class="description-td" morerows="0" valign="top">
<i>Pseudomonas putida</i>
</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top"> </td>
<td class="description-td" morerows="0" valign="top">alcohol</td>
<td class="description-td" morerows="0" valign="top">
<i>Pseudomonas vesicularis</i>
</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top"> </td>
<td class="description-td" morerows="0" valign="top">polyether</td>
<td class="description-td" morerows="0" valign="top">
<i>Pseudomonas aeruginosa</i>
</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top"> </td>
<td class="description-td" morerows="0" valign="top">(polyethylene</td>
<td class="description-td" morerows="0" valign="top">Bacteroides sp.</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top"> </td>
<td class="description-td" morerows="0" valign="top">glycol)</td>
<td class="description-td" morerows="0" valign="top">
<i>Pelolobacter venetianus</i>
</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top"> </td>
<td class="description-td" morerows="0" valign="top"> </td>
<td class="description-td" morerows="0" valign="top">
<i> Rhizobium loti</i>
</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top"> </td>
<td class="description-td" morerows="0" valign="top"> </td>
<td class="description-td" morerows="0" valign="top">Corynebacterium sp.</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top"> </td>
<td class="description-td" morerows="0" valign="top"> </td>
<td class="description-td" morerows="0" valign="top">
<i>Sphingomonas pegritica</i>
</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top"> </td>
<td class="description-td" morerows="0" valign="top">polyester</td>
<td class="description-td" morerows="0" valign="top">Penicillium sp.</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top"> </td>
<td class="description-td" morerows="0" valign="top">polyurethane</td>
<td class="description-td" morerows="0" valign="top">
<i>Rhizobium delemar</i>
</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top"> </td>
<td class="description-td" morerows="0" valign="top">polyamide</td>
<td class="description-td" morerows="0" valign="top">
<i>Corynebacterium aurantiacum</i>
</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top"> </td>
<td class="description-td" morerows="0" valign="top"> </td>
<td class="description-td" morerows="0" valign="top">Flavobacterium sp.</td>
</tr>
<tr class="description-tr">
<td align="center" class="description-td" colspan="3" morerows="0" nameend="3" namest="1" rowsep="1" valign="top"> </td>
</tr>
</tbody></table>
</patent-tables>
</tables>
</div>
<div class="description-paragraph">
<tables>
<patent-tables colsep="0" frame="none" rowsep="0">
<table align="left" class="description-table" cols="3" colsep="0" rowsep="0" width="100%">
<thead valign="bottom">
<tr class="description-tr">
<td class="description-td" colspan="3" morerows="0" nameend="3" namest="1" rowsep="1" valign="top">TABLE 2</td>
</tr>
<tr class="description-tr">
<td align="center" class="description-td" colspan="3" morerows="0" nameend="3" namest="1" rowsep="1" valign="top"> </td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top"> </td>
<td class="description-td" morerows="0" valign="top">Substance</td>
<td class="description-td" morerows="0" valign="top"> </td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top">Classification</td>
<td class="description-td" morerows="0" valign="top">Name</td>
<td class="description-td" morerows="0" valign="top">Related Microorganism</td>
</tr>
<tr class="description-tr">
<td align="center" class="description-td" colspan="3" morerows="0" nameend="3" namest="1" rowsep="1" valign="top"> </td>
</tr>
</thead>
<tbody><tr class="description-tr">
<td class="description-td" morerows="0" valign="top">Metal</td>
<td class="description-td" morerows="0" valign="top">mercury</td>
<td class="description-td" morerows="0" valign="top"> Pseudomonas sp.</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top"> </td>
<td class="description-td" morerows="0" valign="top"> </td>
<td class="description-td" morerows="0" valign="top">
<i>Methanobacterium omelianskii</i>
</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top"> </td>
<td class="description-td" morerows="0" valign="top"> </td>
<td class="description-td" morerows="0" valign="top">
<i> Clostridium cochearium</i>
</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top"> </td>
<td class="description-td" morerows="0" valign="top">chromium</td>
<td class="description-td" morerows="0" valign="top">
<i> Streptococcus lactis</i>
</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top"> </td>
<td class="description-td" morerows="0" valign="top"> </td>
<td class="description-td" morerows="0" valign="top">
<i> Alcaligenes eutrophus</i>
</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top"> </td>
<td class="description-td" morerows="0" valign="top"> </td>
<td class="description-td" morerows="0" valign="top">
<i> Pseudomonas aeruginosa</i>
</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top"> </td>
<td class="description-td" morerows="0" valign="top"> </td>
<td class="description-td" morerows="0" valign="top">
<i> Enterobacter cloacae</i>
</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top"> </td>
<td class="description-td" morerows="0" valign="top">cadmium</td>
<td class="description-td" morerows="0" valign="top">
<i> Staphlococcus aureus</i>
</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top"> </td>
<td class="description-td" morerows="0" valign="top"> </td>
<td class="description-td" morerows="0" valign="top">
<i> Alcaligenes eutrophus</i>
</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top"> </td>
<td class="description-td" morerows="0" valign="top">aluminum</td>
<td class="description-td" morerows="0" valign="top">
<i> Chaetosphaeria inaequalis</i>
</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top"> </td>
<td class="description-td" morerows="0" valign="top"> </td>
<td class="description-td" morerows="0" valign="top">
<i> Paecilomyces lilacinus</i>
</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top"> </td>
<td class="description-td" morerows="0" valign="top"> </td>
<td class="description-td" morerows="0" valign="top">
<i> Metarhizium anisopliae</i>
</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top"> </td>
<td class="description-td" morerows="0" valign="top"> </td>
<td class="description-td" morerows="0" valign="top">
<i> Penicillium glabrum</i>
</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top"> </td>
<td class="description-td" morerows="0" valign="top"> </td>
<td class="description-td" morerows="0" valign="top">
<i> Aspergillus fumigatus</i>
</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top"> </td>
<td class="description-td" morerows="0" valign="top"> </td>
<td class="description-td" morerows="0" valign="top">
<i> Sporothrix inflata</i>
</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top"> </td>
<td class="description-td" morerows="0" valign="top"> </td>
<td class="description-td" morerows="0" valign="top">
<i> Emericellopsis minima</i>
</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top"> </td>
<td class="description-td" morerows="0" valign="top">iron</td>
<td class="description-td" morerows="0" valign="top">
<i> Thiobacillus ferrooxidans</i>
</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top"> </td>
<td class="description-td" morerows="0" valign="top"> </td>
<td class="description-td" morerows="0" valign="top">
<i> Thiobacillus thiooxidans</i>
</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top"> </td>
<td class="description-td" morerows="0" valign="top"> </td>
<td class="description-td" morerows="0" valign="top">
<i> Leptospirillum ferooxidans</i>
</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top"> </td>
<td class="description-td" morerows="0" valign="top">arsenic</td>
<td class="description-td" morerows="0" valign="top">
<i> Alcaligenes faecalis</i>
</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top"> </td>
<td class="description-td" morerows="0" valign="top"> </td>
<td class="description-td" morerows="0" valign="top">Pseudomonas sp.</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top"> </td>
<td class="description-td" morerows="0" valign="top"> </td>
<td class="description-td" morerows="0" valign="top">
<i> Micrococcus lactilyticus</i>
</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top"> </td>
<td class="description-td" morerows="0" valign="top"> </td>
<td class="description-td" morerows="0" valign="top">
<i> Staphylococcus aureus</i>
</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top">Chlorine</td>
<td class="description-td" morerows="0" valign="top">chlorobenzoic</td>
<td class="description-td" morerows="0" valign="top">
<i> Pseudomonas putida</i>
</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top">Organic</td>
<td class="description-td" morerows="0" valign="top">acid</td>
<td class="description-td" morerows="0" valign="top">
<i>Alcaligenes eutrophus</i>
</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top">Compound</td>
<td class="description-td" morerows="0" valign="top">chlorobenzene</td>
<td class="description-td" morerows="0" valign="top">
<i>Alcaligenes eutrophus</i>
</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top"> </td>
<td class="description-td" morerows="0" valign="top">etc.</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top"> </td>
<td class="description-td" morerows="0" valign="top">dioxin</td>
<td class="description-td" morerows="0" valign="top">
<i>Phanerochaete chrysosporium</i>
</td>
</tr>
<tr class="description-tr">
<td align="center" class="description-td" colspan="3" morerows="0" nameend="3" namest="1" rowsep="1" valign="top"> </td>
</tr>
</tbody></table>
</patent-tables>
</tables>
</div>
<div class="description-paragraph">If the soil contains Methanobacterium, Clostridium or Pseudomonas bacteria as shown in Tables 1 and 2, for example, the possibility of soil contamination with mercury is suggested. Further, it is also possible to estimate presence/absence of dioxin and the existential degree thereof from Phanerochaete which can degrade dioxin.</div>
<div class="description-paragraph">According to the aforementioned method of retrieving microorganisms from the database of band patterns of DNA, the types of contaminants and the degrees thereof can be quickly examined by collecting a small quantity of sample of soil or food allowing no prediction of the contaminants.</div>
<heading>EXAMPLES</heading>
<heading>Example 1</heading>
<div class="description-paragraph">High-quality primers employed for the method of amplifying DNA fragments according to the present invention were selected in the following method:</div>
<div class="description-paragraph">{circle around (1)} Primers and Standard Samples</div>
<div class="description-paragraph">216 types of DNA oligomers by Nippon Gene were employed as primers to be studied. Further, chromosome DNA samples of seven bacteria shown in Table 3 were employed as standard samples for selecting the primers.</div>
<div class="description-paragraph">
<tables>
<patent-tables colsep="0" frame="none" rowsep="0">
<table align="left" class="description-table" cols="1" colsep="0" rowsep="0" width="100%">
<thead valign="bottom">
<tr class="description-tr">
<td class="description-td" colspan="1" morerows="0" nameend="1" namest="1" rowsep="1" valign="top">TABLE 3</td>
</tr>
<tr class="description-tr">
<td align="center" class="description-td" colspan="1" morerows="0" nameend="1" namest="1" rowsep="1" valign="top"> </td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top">Seven Types of Bacteria employed for Primer Selection</td>
</tr>
<tr class="description-tr">
<td align="center" class="description-td" colspan="1" morerows="0" nameend="1" namest="1" rowsep="1" valign="top"> </td>
</tr>
</thead>
<tbody><tr class="description-tr">
<td class="description-td" morerows="0" valign="top"> </td>
</tr>
</tbody></table>
<table align="left" class="description-table" cols="2" colsep="0" rowsep="0" width="100%">
<tbody><tr class="description-tr">
<td class="description-td" morerows="0" valign="top"> </td>
<td class="description-td" morerows="0" valign="top">1. <i>Escherichia coli </i>strain K12</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top"> </td>
<td class="description-td" morerows="0" valign="top">2. <i>Bacillus subtilis </i>natto strain I2</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top"> </td>
<td class="description-td" morerows="0" valign="top">3. No. 10 (bacterium isolated from garbage disposal)</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top"> </td>
<td class="description-td" morerows="0" valign="top">4. No. 30 (bacterium isolated from garbage disposal)</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top"> </td>
<td class="description-td" morerows="0" valign="top">5. No. 38 (bacterium isolated from garbage disposal)</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top"> </td>
<td class="description-td" morerows="0" valign="top">6. No. 46 (bacterium isolated from garbage disposal)</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top"> </td>
<td class="description-td" morerows="0" valign="top">7. No. 103 <i>Proteus mirabilis </i>(bacterium isolated from</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top"> </td>
<td class="description-td" morerows="0" valign="top">garbage disposal)</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top"> </td>
<td align="center" class="description-td" colspan="2" morerows="0" nameend="1" namest="OFFSET" rowsep="1" valign="top"> </td>
</tr>
</tbody></table>
</patent-tables>
</tables>
</div>
<div class="description-paragraph">As shown in Table 3, <i>Escherichia coli </i>strain K12 and <i>Bacillus subtilis </i>natto strain I2 were employed as representatives of general bacteria. Further, bacteria degrading kitchen garbage in the tank of a garbage disposal were employed as the remaining five bacteria. Nos. 10, 30, 38, 46 and 103 are allotted to the five bacteria respectively.</div>
<div class="description-paragraph">{circle around (2)} Method of Driving Garbage Disposal</div>
<div class="description-paragraph">Household garbage disposal SNS-T1 (outer dimensions: 580 by 450 by 795 mm) by Sanyo Electric Co., Ltd. was employed and improved by connecting an air pump and an air adjuster to an outlet of this garbage disposal. The garbage disposal was set in a prefabricated laboratory of 30° C. in temperature and 60% in relative humidity.</div>
<div class="description-paragraph">Wood chips (Japan cedar material of 1.5 mm in mean particle diameter) of 25 kg (water content: 70%) were introduced into the tank of the garbage disposal as a treating carrier. 1 kg of kitchen garbage consisting of 450 g of vegetables, 300 g of fruit, 40 g of fish, 30 g of meat and 180 g of cooked rice was introduced into the garbage disposal five times a week (once a day), and thereafter the contents of the tank were stirred with stirring blades of the garbage disposal.</div>
<div class="description-paragraph">The water content of the wood chips was adjusted to 35 to 45% in the tank, for keeping an excellent treating state. The water content was finely controlled by adjusting the volume of air introduced from the air pump with the air adjuster.</div>
<div class="description-paragraph">{circle around (3)} Isolation of Bacteria for Treating Kitchen Garbage</div>
<div class="description-paragraph">An agar medium for culturing bacteria was prepared in the composition shown in Table 4.</div>
<div class="description-paragraph">
<tables>
<patent-tables colsep="0" frame="none" rowsep="0">
<table align="left" class="description-table" cols="3" colsep="0" rowsep="0" width="100%">
<thead valign="bottom">
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top"> </td>
<td class="description-td" colspan="3" morerows="0" nameend="2" namest="OFFSET" rowsep="1" valign="top">     TABLE 4</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top"> </td>
<td align="center" class="description-td" colspan="3" morerows="0" nameend="2" namest="OFFSET" rowsep="1" valign="top"> </td>
</tr>
</thead>
<tbody><tr class="description-tr">
<td class="description-td" morerows="0" valign="top"> </td>
<td class="description-td" morerows="0" valign="top">    nutrient broth medium</td>
<td class="description-td" morerows="0" valign="top">18 g/L</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top"> </td>
<td class="description-td" morerows="0" valign="top">(Eiken E-MC35)</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top"> </td>
<td class="description-td" morerows="0" valign="top">sodium chloride</td>
<td class="description-td" morerows="0" valign="top">0.5M</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top"> </td>
<td class="description-td" morerows="0" valign="top">pH</td>
<td class="description-td" morerows="0" valign="top">adjusted to 9 with</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top"> </td>
<td class="description-td" morerows="0" valign="top"> </td>
<td class="description-td" morerows="0" valign="top">sodium hydroxide</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top"> </td>
<td class="description-td" morerows="0" valign="top">agar</td>
<td class="description-td" morerows="0" valign="top">15 g/L</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top"> </td>
<td align="center" class="description-td" colspan="3" morerows="0" nameend="2" namest="OFFSET" rowsep="1" valign="top"> </td>
</tr>
</tbody></table>
</patent-tables>
</tables>
</div>
<div class="description-paragraph">After a lapse of about nine months from starting driving the garbage disposal, 10 g of the wood chips were sampled from the tank and suspended until bacteria were sufficiently isolated from the wood chips with addition of 90 mL of a sterilized 0.85% salt solution. The suspension was diluted to 10<sup>−6</sup>, and 100 μL of the diluted suspension was homogeneously inoculated on the agar medium. After cultivation at 37° C. for three days, all colonies were transferred to a new agar medium for isolating the bacteria.</div>
<div class="description-paragraph">Among the isolated colonies, five types of bacteria having different colony shapes were employed as standard samples for PCR. The above Nos. 10, 30, 38, 46 and 103 were allotted to the isolated bacteria. As the result of biochemical examination, it has been recognized that the bacterium No. 103 is <i>Proteus mirabilis. </i> </div>
<div class="description-paragraph">{circle around (4)} Method of Preparing Chromosome DNA of Bacteria employed as Standard Samples</div>
<div class="description-paragraph"> <i>Escherichia coli </i>strain K12 and <i>Bacillus subtilis </i>natto strain I2 were cultured on a nutrient broth medium (Eiken E-MC35) of 18 g/L for 16 hours. The five bacteria isolated from the garbage disposal were cultured on a medium shown in Table 5 for 16 hours.</div>
<div class="description-paragraph">
<tables>
<patent-tables colsep="0" frame="none" rowsep="0">
<table align="left" class="description-table" cols="3" colsep="0" rowsep="0" width="100%">
<thead valign="bottom">
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top"> </td>
<td class="description-td" colspan="3" morerows="0" nameend="2" namest="OFFSET" rowsep="1" valign="top">                                TABLE 5</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top"> </td>
<td align="center" class="description-td" colspan="3" morerows="0" nameend="2" namest="OFFSET" rowsep="1" valign="top"> </td>
</tr>
</thead>
<tbody><tr class="description-tr">
<td class="description-td" morerows="0" valign="top"> </td>
<td class="description-td" morerows="0" valign="top">       nutrient broth medium</td>
<td class="description-td" morerows="0" valign="top">18 g/L</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top"> </td>
<td class="description-td" morerows="0" valign="top">(Eiken E-MC35)</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top"> </td>
<td class="description-td" morerows="0" valign="top">sodium chloride</td>
<td class="description-td" morerows="0" valign="top">0.5M</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top"> </td>
<td class="description-td" morerows="0" valign="top">pH</td>
<td class="description-td" morerows="0" valign="top">adjusted to 9 with</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top"> </td>
<td class="description-td" morerows="0" valign="top"> </td>
<td class="description-td" morerows="0" valign="top">sodium hydroxide</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top"> </td>
<td align="center" class="description-td" colspan="3" morerows="0" nameend="2" namest="OFFSET" rowsep="1" valign="top"> </td>
</tr>
</tbody></table>
</patent-tables>
</tables>
</div>
<div class="description-paragraph">The chromosome DNA samples of the bacteria were prepared in accordance with the method of “Preparation of Genomic DNA from Bacteria” described in Current Protocols in Molecular Biology (published by Greene Publishing Associates and Wiley-Interscience), pp. 2.4.1 to 2.4.2.</div>
<div class="description-paragraph">{circle around (5)} Setting of PCR Conditions</div>
<div class="description-paragraph">In the random PCR method employed in the present invention, it is difficult to associate each of a plurality of microorganisms with DNA fragments if a number of DNA fragments are amplified from a single type of microorganism. Therefore, the number of amplified DNA fragments must be reduced by increasing selectivity of amplification. Important factors for improving the selectivity are the length of the primers, the magnesium concentration in the reaction solution composition, the annealing temperature in the reaction cycle and the number of the reaction cycle.</div>
<div class="description-paragraph">The PCR conditions were set through PCR System 9700 by PE Applied Biosystem and DNA amplifier MIR-D40 by Sanyo Electric Co., Ltd.</div>
<div class="description-paragraph">The PCR conditions were studied with DNA Oligomer H81 by Nippon Gene employed as a primer having a base length of 12 bp. In this study, the reaction solution was composed of Tris-HCl of 10 mMin concentration, KCl of 50 mMin concentration and each of dATP, dCTP, dGTP and dTTP of 200 μM in concentration. Consequently, it has been recognized that the optimum values of the magnesium concentration and the Taq polymerase concentration are 1.5 mM and 0.025 unit/μL respectively and the optimum values of the annealing temperature for the PCR cycle and the cycle number are 55° C. and 35 cycles respectively. The efficiency of PCR reaction slightly changes due to influence by the type of the primer and set conditions. In consideration of this, the annealing temperature was set at 45° C. in the following primer selection experiment, in order to slightly reduce the selectivity.</div>
<div class="description-paragraph">Table 6 shows the composition of the reaction solution employed for random PCR in the following primer selection experiment.</div>
<div class="description-paragraph">
<tables>
<patent-tables colsep="0" frame="none" rowsep="0">
<table align="left" class="description-table" cols="2" colsep="0" rowsep="0" width="100%">
<thead valign="bottom">
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top"> </td>
<td class="description-td" colspan="2" morerows="0" nameend="1" namest="OFFSET" rowsep="1" valign="top">                          TABLE 6</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top"> </td>
<td align="center" class="description-td" colspan="2" morerows="0" nameend="1" namest="OFFSET" rowsep="1" valign="top"> </td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top"> </td>
<td class="description-td" morerows="0" valign="top">Final Concentration</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top"> </td>
<td align="center" class="description-td" colspan="2" morerows="0" nameend="1" namest="OFFSET" rowsep="1" valign="top"> </td>
</tr>
</thead>
<tbody><tr class="description-tr">
<td class="description-td" morerows="0" valign="top"> </td>
</tr>
</tbody></table>
<table align="left" class="description-table" cols="3" colsep="0" rowsep="0" width="100%">
<tbody><tr class="description-tr">
<td class="description-td" morerows="0" valign="top"> </td>
<td class="description-td" morerows="0" valign="top">buffer</td>
<td class="description-td" morerows="0" valign="top"> </td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top"> </td>
<td class="description-td" morerows="0" valign="top">Tris-HCl (pH 8.3)</td>
<td class="description-td" morerows="0" valign="top">10 mM</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top"> </td>
<td class="description-td" morerows="0" valign="top">KCl</td>
<td class="description-td" morerows="0" valign="top">50 mM</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top"> </td>
<td class="description-td" morerows="0" valign="top">MgCl<sub>2</sub> </td>
<td class="description-td" morerows="0" valign="top">1.5 mM</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top"> </td>
<td class="description-td" morerows="0" valign="top">dNTPmix</td>
<td class="description-td" morerows="0" valign="top">200 μM</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top"> </td>
<td class="description-td" morerows="0" valign="top">primer</td>
<td class="description-td" morerows="0" valign="top">2 μM</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top"> </td>
<td class="description-td" morerows="0" valign="top">chromosome DNA</td>
<td class="description-td" morerows="0" valign="top">10 pg/μL</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top"> </td>
<td class="description-td" morerows="0" valign="top">Taq DNA polymerase</td>
<td class="description-td" morerows="0" valign="top">0.025 u/μL</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top"> </td>
<td align="center" class="description-td" colspan="3" morerows="0" nameend="2" namest="OFFSET" rowsep="1" valign="top"> </td>
</tr>
</tbody></table>
</patent-tables>
</tables>
</div>
<div class="description-paragraph">Referring to Table 6, Tris in Tris-HCl is the abbreviation of Tris(hydroxymethyl)aminomethane. dNTPmix stands for an isosbestic mixed solution of dATP (2′-deoxyadenosine-5′-triphosphate), dCTP (2′-deoxycytidine-5′-triphosphate), dGTP (2′-deoxyguanosine-5′-triphosphate) and dTTP (2′-deoxythymidine-5′-triphosphate). The quantity of the reaction solution was 20 μm.</div>
<div class="description-paragraph">Table 7 shows cycles of random PCR in the primer selection experiment.</div>
<div class="description-paragraph">
<tables>
<patent-tables colsep="0" frame="none" rowsep="0">
<table align="left" class="description-table" cols="2" colsep="0" rowsep="0" width="100%">
<thead valign="bottom">
<tr class="description-tr">
<td class="description-td" colspan="2" morerows="0" nameend="2" namest="1" rowsep="1" valign="top">                                  TABLE 7</td>
</tr>
<tr class="description-tr">
<td align="center" class="description-td" colspan="2" morerows="0" nameend="2" namest="1" rowsep="1" valign="top"> </td>
</tr>
</thead>
<tbody><tr class="description-tr">
<td class="description-td" morerows="0" valign="top">        94° C. for 1 min.</td>
<td class="description-td" morerows="0" valign="top"> 1 cycle</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top">94° C. for 1 min. + 45° C. for 2 min. + 72° C. for 3 min.</td>
<td class="description-td" morerows="0" valign="top">35 cycles</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top">72° C. for 7 min.</td>
<td class="description-td" morerows="0" valign="top"> 1 cycle</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top">4° C. (end of reaction, preserved)</td>
</tr>
<tr class="description-tr">
<td align="center" class="description-td" colspan="2" morerows="0" nameend="2" namest="1" rowsep="1" valign="top"> </td>
</tr>
</tbody></table>
</patent-tables>
</tables>
</div>
<div class="description-paragraph">{circle around (6)} Primer Selection Experiment</div>
<div class="description-paragraph">Seven types of random PCR experiments were made for each primer employing the aforementioned seven bacteria as standard samples, and 5 μL of the reaction solution was analyzed by 1.5% agarose gel electrophoresis after random PCR. The electrophoresis was made under a constant voltage of 3.6 V/cm. After the electrophoresis, the gel was stained with ethidium bromide and irradiated with ultraviolet light of 254 nm in wavelength for acquiring ethidium bromide fluorescent images with an instant camera.</div>
<div class="description-paragraph">{circle around (7)} Selection of Primers</div>
<div class="description-paragraph">The numbers of bands observed from the electrophoretic images were counted. Tables 8 to 15 show the band numbers (total band numbers) observed from 216 types of primers (sequence Nos. 1 to 216) and the electrophoretic images.</div>
<div class="description-paragraph">
<tables>
<patent-tables colsep="0" frame="none" rowsep="0">
<table align="left" class="description-table" cols="5" colsep="0" rowsep="0" width="100%">
<thead valign="bottom">
<tr class="description-tr">
<td class="description-td" colspan="5" morerows="0" nameend="5" namest="1" rowsep="1" valign="top">                          TABLE 8</td>
</tr>
<tr class="description-tr">
<td align="center" class="description-td" colspan="5" morerows="0" nameend="5" namest="1" rowsep="1" valign="top"> </td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top"> </td>
<td class="description-td" morerows="0" valign="top"> </td>
<td class="description-td" morerows="0" valign="top"> </td>
<td class="description-td" morerows="0" valign="top"> </td>
<td class="description-td" morerows="0" valign="top">Total Band</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top">No.</td>
<td class="description-td" morerows="0" valign="top"> </td>
<td class="description-td" morerows="0" valign="top">Base Sequence 5′ → 3′</td>
<td class="description-td" morerows="0" valign="top">GC %</td>
<td class="description-td" morerows="0" valign="top">Number</td>
</tr>
<tr class="description-tr">
<td align="center" class="description-td" colspan="5" morerows="0" nameend="5" namest="1" rowsep="1" valign="top"> </td>
</tr>
</thead>
<tbody><tr class="description-tr">
<td class="description-td" morerows="0" valign="top"> </td>
</tr>
</tbody></table>
<table align="left" class="description-table" cols="5" colsep="0" rowsep="0" width="100%">
<tbody><tr class="description-tr">
<td class="description-td" morerows="0" valign="top">1</td>
<td class="description-td" morerows="0" valign="top">A26</td>
<td class="description-td" morerows="0" valign="top">ACTGAGAAAATA</td>
<td class="description-td" morerows="0" valign="top">25.0</td>
<td class="description-td" morerows="0" valign="top">0</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top">2</td>
<td class="description-td" morerows="0" valign="top">A49</td>
<td class="description-td" morerows="0" valign="top">ATCTTCAAAGAT</td>
<td class="description-td" morerows="0" valign="top">25.0</td>
<td class="description-td" morerows="0" valign="top">0</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top">3</td>
<td class="description-td" morerows="0" valign="top">A64</td>
<td class="description-td" morerows="0" valign="top">ACAAAGAGATAT</td>
<td class="description-td" morerows="0" valign="top">25.0</td>
<td class="description-td" morerows="0" valign="top">0</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top">4</td>
<td class="description-td" morerows="0" valign="top">A47</td>
<td class="description-td" morerows="0" valign="top">GAGGTGATATTA</td>
<td class="description-td" morerows="0" valign="top">33.3</td>
<td class="description-td" morerows="0" valign="top">0</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top">5</td>
<td class="description-td" morerows="0" valign="top">A66</td>
<td class="description-td" morerows="0" valign="top">ATCTTCTCATCT</td>
<td class="description-td" morerows="0" valign="top">33.3</td>
<td class="description-td" morerows="0" valign="top">0</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top">6</td>
<td class="description-td" morerows="0" valign="top">A71</td>
<td class="description-td" morerows="0" valign="top">ACTCTTCTACAA</td>
<td class="description-td" morerows="0" valign="top">33.3</td>
<td class="description-td" morerows="0" valign="top">0</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top">7</td>
<td class="description-td" morerows="0" valign="top">A88</td>
<td class="description-td" morerows="0" valign="top">AGAGACATAGTT</td>
<td class="description-td" morerows="0" valign="top">33.3</td>
<td class="description-td" morerows="0" valign="top">0</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top">8</td>
<td class="description-td" morerows="0" valign="top">B08</td>
<td class="description-td" morerows="0" valign="top">GCCAGATATATA</td>
<td class="description-td" morerows="0" valign="top">33.3</td>
<td class="description-td" morerows="0" valign="top">0</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top">9</td>
<td class="description-td" morerows="0" valign="top">B24</td>
<td class="description-td" morerows="0" valign="top">CACACTACTTAT</td>
<td class="description-td" morerows="0" valign="top">33.3</td>
<td class="description-td" morerows="0" valign="top">0</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top">10</td>
<td class="description-td" morerows="0" valign="top">B26</td>
<td class="description-td" morerows="0" valign="top">ATGAGAAAGGAA</td>
<td class="description-td" morerows="0" valign="top">33.3</td>
<td class="description-td" morerows="0" valign="top">0</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top">11</td>
<td class="description-td" morerows="0" valign="top">B45</td>
<td class="description-td" morerows="0" valign="top">ATCAACACTTTC</td>
<td class="description-td" morerows="0" valign="top">33.3</td>
<td class="description-td" morerows="0" valign="top">0</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top">12</td>
<td class="description-td" morerows="0" valign="top">B64</td>
<td class="description-td" morerows="0" valign="top">GAGACTACAATA</td>
<td class="description-td" morerows="0" valign="top">33.3</td>
<td class="description-td" morerows="0" valign="top">0</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top">13</td>
<td class="description-td" morerows="0" valign="top">B91</td>
<td class="description-td" morerows="0" valign="top">CCATACATATTG</td>
<td class="description-td" morerows="0" valign="top">33.3</td>
<td class="description-td" morerows="0" valign="top">0</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top">14</td>
<td class="description-td" morerows="0" valign="top">C12</td>
<td class="description-td" morerows="0" valign="top">GATACTGATGAT</td>
<td class="description-td" morerows="0" valign="top">33.3</td>
<td class="description-td" morerows="0" valign="top">0</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top">15</td>
<td class="description-td" morerows="0" valign="top">C25</td>
<td class="description-td" morerows="0" valign="top">AGATTCTTACTG</td>
<td class="description-td" morerows="0" valign="top">33.3</td>
<td class="description-td" morerows="0" valign="top">0</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top">16</td>
<td class="description-td" morerows="0" valign="top">D01</td>
<td class="description-td" morerows="0" valign="top">AGCCCTTATTTA</td>
<td class="description-td" morerows="0" valign="top">33.3</td>
<td class="description-td" morerows="0" valign="top">0</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top">17</td>
<td class="description-td" morerows="0" valign="top">D05</td>
<td class="description-td" morerows="0" valign="top">GAGACTATGAAA</td>
<td class="description-td" morerows="0" valign="top">33.3</td>
<td class="description-td" morerows="0" valign="top">0</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top">18</td>
<td class="description-td" morerows="0" valign="top">D29</td>
<td class="description-td" morerows="0" valign="top">ATCAAGTATCCA</td>
<td class="description-td" morerows="0" valign="top">33.3</td>
<td class="description-td" morerows="0" valign="top">0</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top">19</td>
<td class="description-td" morerows="0" valign="top">A23</td>
<td class="description-td" morerows="0" valign="top">ACTGACCTAGTT</td>
<td class="description-td" morerows="0" valign="top">41.7</td>
<td class="description-td" morerows="0" valign="top">0</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top">20</td>
<td class="description-td" morerows="0" valign="top">A28</td>
<td class="description-td" morerows="0" valign="top">ATTTGGATAGGG</td>
<td class="description-td" morerows="0" valign="top">41.7</td>
<td class="description-td" morerows="0" valign="top">0</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top">21</td>
<td class="description-td" morerows="0" valign="top">A61</td>
<td class="description-td" morerows="0" valign="top">GACTGCTATACA</td>
<td class="description-td" morerows="0" valign="top">41.7</td>
<td class="description-td" morerows="0" valign="top">0</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top">22</td>
<td class="description-td" morerows="0" valign="top">A69</td>
<td class="description-td" morerows="0" valign="top">TGGTACGGTATA</td>
<td class="description-td" morerows="0" valign="top">41.7</td>
<td class="description-td" morerows="0" valign="top">0</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top">23</td>
<td class="description-td" morerows="0" valign="top">A85</td>
<td class="description-td" morerows="0" valign="top">TACTACTGTGGA</td>
<td class="description-td" morerows="0" valign="top">41.7</td>
<td class="description-td" morerows="0" valign="top">0</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top">24</td>
<td class="description-td" morerows="0" valign="top">B84</td>
<td class="description-td" morerows="0" valign="top">TGGCTGTAGAAA</td>
<td class="description-td" morerows="0" valign="top">41.7</td>
<td class="description-td" morerows="0" valign="top">0</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top">25</td>
<td class="description-td" morerows="0" valign="top">C32</td>
<td class="description-td" morerows="0" valign="top">TCTACACGAAGT</td>
<td class="description-td" morerows="0" valign="top">41.7</td>
<td class="description-td" morerows="0" valign="top">0</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top">26</td>
<td class="description-td" morerows="0" valign="top">C42</td>
<td class="description-td" morerows="0" valign="top">CCAGATTTTCTG</td>
<td class="description-td" morerows="0" valign="top">41.7</td>
<td class="description-td" morerows="0" valign="top">0</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top">27</td>
<td class="description-td" morerows="0" valign="top">C51</td>
<td class="description-td" morerows="0" valign="top">ATCAACGTACGT</td>
<td class="description-td" morerows="0" valign="top">41.7</td>
<td class="description-td" morerows="0" valign="top">0</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top">28</td>
<td class="description-td" morerows="0" valign="top">C71</td>
<td class="description-td" morerows="0" valign="top">TTCCGTAATCAC</td>
<td class="description-td" morerows="0" valign="top">41.7</td>
<td class="description-td" morerows="0" valign="top">0</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top">29</td>
<td class="description-td" morerows="0" valign="top">C89</td>
<td class="description-td" morerows="0" valign="top">GCTTACATAGAC</td>
<td class="description-td" morerows="0" valign="top">41.7</td>
<td class="description-td" morerows="0" valign="top">0</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top">30</td>
<td class="description-td" morerows="0" valign="top">D03</td>
<td class="description-td" morerows="0" valign="top">ACTCCAAATGTG</td>
<td class="description-td" morerows="0" valign="top">41.7</td>
<td class="description-td" morerows="0" valign="top">0</td>
</tr>
<tr class="description-tr">
<td align="center" class="description-td" colspan="5" morerows="0" nameend="5" namest="1" rowsep="1" valign="top"> </td>
</tr>
</tbody></table>
</patent-tables>
</tables>
</div>
<div class="description-paragraph">
<tables>
<patent-tables colsep="0" frame="none" rowsep="0">
<table align="left" class="description-table" cols="5" colsep="0" rowsep="0" width="100%">
<thead valign="bottom">
<tr class="description-tr">
<td class="description-td" colspan="5" morerows="0" nameend="5" namest="1" rowsep="1" valign="top">                       TABLE 9</td>
</tr>
<tr class="description-tr">
<td align="center" class="description-td" colspan="5" morerows="0" nameend="5" namest="1" rowsep="1" valign="top"> </td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top"> </td>
<td class="description-td" morerows="0" valign="top"> </td>
<td class="description-td" morerows="0" valign="top"> </td>
<td class="description-td" morerows="0" valign="top"> </td>
<td class="description-td" morerows="0" valign="top">Total Band</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top">No.</td>
<td class="description-td" morerows="0" valign="top"> </td>
<td class="description-td" morerows="0" valign="top">Base Sequence 5′ → 3′</td>
<td class="description-td" morerows="0" valign="top">GC %</td>
<td class="description-td" morerows="0" valign="top">Number</td>
</tr>
<tr class="description-tr">
<td align="center" class="description-td" colspan="5" morerows="0" nameend="5" namest="1" rowsep="1" valign="top"> </td>
</tr>
</thead>
<tbody><tr class="description-tr">
<td class="description-td" morerows="0" valign="top"> </td>
</tr>
</tbody></table>
<table align="left" class="description-table" cols="5" colsep="0" rowsep="0" width="100%">
<tbody><tr class="description-tr">
<td class="description-td" morerows="0" valign="top">31</td>
<td class="description-td" morerows="0" valign="top">A87</td>
<td class="description-td" morerows="0" valign="top">AAGTCGTTTGGG</td>
<td class="description-td" morerows="0" valign="top">50.0</td>
<td class="description-td" morerows="0" valign="top">0</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top">32</td>
<td class="description-td" morerows="0" valign="top">B82</td>
<td class="description-td" morerows="0" valign="top">CTAGTATGGGAC</td>
<td class="description-td" morerows="0" valign="top">50.0</td>
<td class="description-td" morerows="0" valign="top">0</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top">33</td>
<td class="description-td" morerows="0" valign="top">C26</td>
<td class="description-td" morerows="0" valign="top">GAGTTCGAACGA</td>
<td class="description-td" morerows="0" valign="top">50.0</td>
<td class="description-td" morerows="0" valign="top">0</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top">34</td>
<td class="description-td" morerows="0" valign="top">C61</td>
<td class="description-td" morerows="0" valign="top">ACTTTCCTACGG</td>
<td class="description-td" morerows="0" valign="top">50.0</td>
<td class="description-td" morerows="0" valign="top">0</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top">35</td>
<td class="description-td" morerows="0" valign="top">H91</td>
<td class="description-td" morerows="0" valign="top">TTCCCGTCTATC</td>
<td class="description-td" morerows="0" valign="top">50.0</td>
<td class="description-td" morerows="0" valign="top">0</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top">36</td>
<td class="description-td" morerows="0" valign="top">C03</td>
<td class="description-td" morerows="0" valign="top">AGCCTTACGGCA</td>
<td class="description-td" morerows="0" valign="top">58.3</td>
<td class="description-td" morerows="0" valign="top">0</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top">37</td>
<td class="description-td" morerows="0" valign="top">C67</td>
<td class="description-td" morerows="0" valign="top">GCTATGGCAACG</td>
<td class="description-td" morerows="0" valign="top">58.3</td>
<td class="description-td" morerows="0" valign="top">0</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top">38</td>
<td class="description-td" morerows="0" valign="top">C69</td>
<td class="description-td" morerows="0" valign="top">CCTTGGAACTCG</td>
<td class="description-td" morerows="0" valign="top">58.3</td>
<td class="description-td" morerows="0" valign="top">0</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top">39</td>
<td class="description-td" morerows="0" valign="top">C31</td>
<td class="description-td" morerows="0" valign="top">TCTGCTGACCGG</td>
<td class="description-td" morerows="0" valign="top">66.7</td>
<td class="description-td" morerows="0" valign="top">0</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top">40</td>
<td class="description-td" morerows="0" valign="top">C50</td>
<td class="description-td" morerows="0" valign="top">GGCAACTGGCCA</td>
<td class="description-td" morerows="0" valign="top">66.7</td>
<td class="description-td" morerows="0" valign="top">0</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top">41</td>
<td class="description-td" morerows="0" valign="top">B28</td>
<td class="description-td" morerows="0" valign="top">GTCATTAAAGCT</td>
<td class="description-td" morerows="0" valign="top">33.3</td>
<td class="description-td" morerows="0" valign="top">1</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top">42</td>
<td class="description-td" morerows="0" valign="top">D10</td>
<td class="description-td" morerows="0" valign="top">TACACTTTTGAC</td>
<td class="description-td" morerows="0" valign="top">33.3</td>
<td class="description-td" morerows="0" valign="top">1</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top">43</td>
<td class="description-td" morerows="0" valign="top">B88</td>
<td class="description-td" morerows="0" valign="top">TGGATCTTTGAC</td>
<td class="description-td" morerows="0" valign="top">41.7</td>
<td class="description-td" morerows="0" valign="top">1</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top">44</td>
<td class="description-td" morerows="0" valign="top">C06</td>
<td class="description-td" morerows="0" valign="top">GCTCTTTTGGAA</td>
<td class="description-td" morerows="0" valign="top">41.7</td>
<td class="description-td" morerows="0" valign="top">1</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top">45</td>
<td class="description-td" morerows="0" valign="top">C49</td>
<td class="description-td" morerows="0" valign="top">ATCATCGTACGT</td>
<td class="description-td" morerows="0" valign="top">41.7</td>
<td class="description-td" morerows="0" valign="top">1</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top">46</td>
<td class="description-td" morerows="0" valign="top">C68</td>
<td class="description-td" morerows="0" valign="top">TACGATATGGCT</td>
<td class="description-td" morerows="0" valign="top">41.7</td>
<td class="description-td" morerows="0" valign="top">1</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top">47</td>
<td class="description-td" morerows="0" valign="top">C87</td>
<td class="description-td" morerows="0" valign="top">GATCCAGTCTTT</td>
<td class="description-td" morerows="0" valign="top">41.7</td>
<td class="description-td" morerows="0" valign="top">1</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top">48</td>
<td class="description-td" morerows="0" valign="top">D27</td>
<td class="description-td" morerows="0" valign="top">AGAATGTCCGTA</td>
<td class="description-td" morerows="0" valign="top">41.7</td>
<td class="description-td" morerows="0" valign="top">1</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top">49</td>
<td class="description-td" morerows="0" valign="top">A09</td>
<td class="description-td" morerows="0" valign="top">CCGCAGTTAGAT</td>
<td class="description-td" morerows="0" valign="top">50.0</td>
<td class="description-td" morerows="0" valign="top">1</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top">50</td>
<td class="description-td" morerows="0" valign="top">B03</td>
<td class="description-td" morerows="0" valign="top">CAGTGGGAGTTT</td>
<td class="description-td" morerows="0" valign="top">50.0</td>
<td class="description-td" morerows="0" valign="top">1</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top">51</td>
<td class="description-td" morerows="0" valign="top">B62</td>
<td class="description-td" morerows="0" valign="top">TCTATGGACCCT</td>
<td class="description-td" morerows="0" valign="top">50.0</td>
<td class="description-td" morerows="0" valign="top">1</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top">52</td>
<td class="description-td" morerows="0" valign="top">C11</td>
<td class="description-td" morerows="0" valign="top">TTCATTCTGGGG</td>
<td class="description-td" morerows="0" valign="top">50.0</td>
<td class="description-td" morerows="0" valign="top">1</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top">53</td>
<td class="description-td" morerows="0" valign="top">C23</td>
<td class="description-td" morerows="0" valign="top">CCGTCTTTTCTG</td>
<td class="description-td" morerows="0" valign="top">50.0</td>
<td class="description-td" morerows="0" valign="top">1</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top">54</td>
<td class="description-td" morerows="0" valign="top">A82</td>
<td class="description-td" morerows="0" valign="top">TGGCCTATTGGC</td>
<td class="description-td" morerows="0" valign="top">58.3</td>
<td class="description-td" morerows="0" valign="top">1</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top">55</td>
<td class="description-td" morerows="0" valign="top">B02</td>
<td class="description-td" morerows="0" valign="top">GTCATGCCTGGA</td>
<td class="description-td" morerows="0" valign="top">58.3</td>
<td class="description-td" morerows="0" valign="top">1</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top">56</td>
<td class="description-td" morerows="0" valign="top">B92</td>
<td class="description-td" morerows="0" valign="top">CCTTGGCGAAGC</td>
<td class="description-td" morerows="0" valign="top">66.7</td>
<td class="description-td" morerows="0" valign="top">1</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top">57</td>
<td class="description-td" morerows="0" valign="top">C41</td>
<td class="description-td" morerows="0" valign="top">AGCCTGTGGGCT</td>
<td class="description-td" morerows="0" valign="top">66.7</td>
<td class="description-td" morerows="0" valign="top">1</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top">58</td>
<td class="description-td" morerows="0" valign="top">A21</td>
<td class="description-td" morerows="0" valign="top">AGAATTGGACGA</td>
<td class="description-td" morerows="0" valign="top">41.7</td>
<td class="description-td" morerows="0" valign="top">2</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top">59</td>
<td class="description-td" morerows="0" valign="top">A52</td>
<td class="description-td" morerows="0" valign="top">CTTGTCATGTGT</td>
<td class="description-td" morerows="0" valign="top">41.7</td>
<td class="description-td" morerows="0" valign="top">2</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top">60</td>
<td class="description-td" morerows="0" valign="top">B63</td>
<td class="description-td" morerows="0" valign="top">TACGTGGTAACA</td>
<td class="description-td" morerows="0" valign="top">41.7</td>
<td class="description-td" morerows="0" valign="top">2</td>
</tr>
<tr class="description-tr">
<td align="center" class="description-td" colspan="5" morerows="0" nameend="5" namest="1" rowsep="1" valign="top"> </td>
</tr>
</tbody></table>
</patent-tables>
</tables>
</div>
<div class="description-paragraph">
<tables>
<patent-tables colsep="0" frame="none" rowsep="0">
<table align="left" class="description-table" cols="5" colsep="0" rowsep="0" width="100%">
<thead valign="bottom">
<tr class="description-tr">
<td class="description-td" colspan="5" morerows="0" nameend="5" namest="1" rowsep="1" valign="top">                     TABLE 10</td>
</tr>
<tr class="description-tr">
<td align="center" class="description-td" colspan="5" morerows="0" nameend="5" namest="1" rowsep="1" valign="top"> </td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top"> </td>
<td class="description-td" morerows="0" valign="top"> </td>
<td class="description-td" morerows="0" valign="top"> </td>
<td class="description-td" morerows="0" valign="top"> </td>
<td class="description-td" morerows="0" valign="top">Total Band</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top">No.</td>
<td class="description-td" morerows="0" valign="top"> </td>
<td class="description-td" morerows="0" valign="top">Base Sequence 5′ → 3′</td>
<td class="description-td" morerows="0" valign="top">GC %</td>
<td class="description-td" morerows="0" valign="top">Number</td>
</tr>
<tr class="description-tr">
<td align="center" class="description-td" colspan="5" morerows="0" nameend="5" namest="1" rowsep="1" valign="top"> </td>
</tr>
</thead>
<tbody><tr class="description-tr">
<td class="description-td" morerows="0" valign="top"> </td>
</tr>
</tbody></table>
<table align="left" class="description-table" cols="5" colsep="0" rowsep="0" width="100%">
<tbody><tr class="description-tr">
<td class="description-td" morerows="0" valign="top">61</td>
<td class="description-td" morerows="0" valign="top">C45</td>
<td class="description-td" morerows="0" valign="top">GGACAAGTAATG</td>
<td class="description-td" morerows="0" valign="top">41.7</td>
<td class="description-td" morerows="0" valign="top">2</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top">62</td>
<td class="description-td" morerows="0" valign="top">A48</td>
<td class="description-td" morerows="0" valign="top">TACCCTCAAGCT</td>
<td class="description-td" morerows="0" valign="top">50.0</td>
<td class="description-td" morerows="0" valign="top">2</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top">63</td>
<td class="description-td" morerows="0" valign="top">B52</td>
<td class="description-td" morerows="0" valign="top">TTCGAGGATCGA</td>
<td class="description-td" morerows="0" valign="top">50.0</td>
<td class="description-td" morerows="0" valign="top">2</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top">64</td>
<td class="description-td" morerows="0" valign="top">A30</td>
<td class="description-td" morerows="0" valign="top">GACCTGCGATCT</td>
<td class="description-td" morerows="0" valign="top">58.3</td>
<td class="description-td" morerows="0" valign="top">2</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top">65</td>
<td class="description-td" morerows="0" valign="top">A81</td>
<td class="description-td" morerows="0" valign="top">TGGCCTCTTGGA</td>
<td class="description-td" morerows="0" valign="top">58.3</td>
<td class="description-td" morerows="0" valign="top">2</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top">66</td>
<td class="description-td" morerows="0" valign="top">A83</td>
<td class="description-td" morerows="0" valign="top">GGTTTCCCAGGA</td>
<td class="description-td" morerows="0" valign="top">58.3</td>
<td class="description-td" morerows="0" valign="top">2</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top">67</td>
<td class="description-td" morerows="0" valign="top">B06</td>
<td class="description-td" morerows="0" valign="top">TCGTCCGGAGAT</td>
<td class="description-td" morerows="0" valign="top">58.3</td>
<td class="description-td" morerows="0" valign="top">2</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top">68</td>
<td class="description-td" morerows="0" valign="top">C05</td>
<td class="description-td" morerows="0" valign="top">CGCTTCGTAGCA</td>
<td class="description-td" morerows="0" valign="top">58.3</td>
<td class="description-td" morerows="0" valign="top">2</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top">69</td>
<td class="description-td" morerows="0" valign="top">H81</td>
<td class="description-td" morerows="0" valign="top">GGCTTCGAATCG</td>
<td class="description-td" morerows="0" valign="top">58.3</td>
<td class="description-td" morerows="0" valign="top">2</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top">70</td>
<td class="description-td" morerows="0" valign="top">D26</td>
<td class="description-td" morerows="0" valign="top">GATGAGCTAAAA</td>
<td class="description-td" morerows="0" valign="top">33.3</td>
<td class="description-td" morerows="0" valign="top">3</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top">71</td>
<td class="description-td" morerows="0" valign="top">A70</td>
<td class="description-td" morerows="0" valign="top">GAGCAGGAATAT</td>
<td class="description-td" morerows="0" valign="top">41.7</td>
<td class="description-td" morerows="0" valign="top">3</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top">72</td>
<td class="description-td" morerows="0" valign="top">B30</td>
<td class="description-td" morerows="0" valign="top">CTTAGGTTACGT</td>
<td class="description-td" morerows="0" valign="top">41.7</td>
<td class="description-td" morerows="0" valign="top">3</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top">73</td>
<td class="description-td" morerows="0" valign="top">D04</td>
<td class="description-td" morerows="0" valign="top">GTGGATCTGAAT</td>
<td class="description-td" morerows="0" valign="top">41.7</td>
<td class="description-td" morerows="0" valign="top">3</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top">74</td>
<td class="description-td" morerows="0" valign="top">A63</td>
<td class="description-td" morerows="0" valign="top">CCTATCCCAACA</td>
<td class="description-td" morerows="0" valign="top">50.0</td>
<td class="description-td" morerows="0" valign="top">3</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top">75</td>
<td class="description-td" morerows="0" valign="top">D30</td>
<td class="description-td" morerows="0" valign="top">GAGACTACCGAA</td>
<td class="description-td" morerows="0" valign="top">50.0</td>
<td class="description-td" morerows="0" valign="top">3</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top">76</td>
<td class="description-td" morerows="0" valign="top">C66</td>
<td class="description-td" morerows="0" valign="top">GACAGCGTCCTA</td>
<td class="description-td" morerows="0" valign="top">58.3</td>
<td class="description-td" morerows="0" valign="top">3</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top">77</td>
<td class="description-td" morerows="0" valign="top">B09</td>
<td class="description-td" morerows="0" valign="top">CTTGAGCGTATT</td>
<td class="description-td" morerows="0" valign="top">41.7</td>
<td class="description-td" morerows="0" valign="top">4</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top">78</td>
<td class="description-td" morerows="0" valign="top">B10</td>
<td class="description-td" morerows="0" valign="top">ACTGAGATAGCA</td>
<td class="description-td" morerows="0" valign="top">41.7</td>
<td class="description-td" morerows="0" valign="top">4</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top">79</td>
<td class="description-td" morerows="0" valign="top">B42</td>
<td class="description-td" morerows="0" valign="top">GAGAGACGATTA</td>
<td class="description-td" morerows="0" valign="top">41.7</td>
<td class="description-td" morerows="0" valign="top">4</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top">80</td>
<td class="description-td" morerows="0" valign="top">A89</td>
<td class="description-td" morerows="0" valign="top">GACGCCCATTAT</td>
<td class="description-td" morerows="0" valign="top">50.0</td>
<td class="description-td" morerows="0" valign="top">4</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top">81</td>
<td class="description-td" morerows="0" valign="top">B66</td>
<td class="description-td" morerows="0" valign="top">GACGGTTCTACA</td>
<td class="description-td" morerows="0" valign="top">50.0</td>
<td class="description-td" morerows="0" valign="top">4</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top">82</td>
<td class="description-td" morerows="0" valign="top">C46</td>
<td class="description-td" morerows="0" valign="top">GATGGTCCGTTT</td>
<td class="description-td" morerows="0" valign="top">50.0</td>
<td class="description-td" morerows="0" valign="top">4</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top">83</td>
<td class="description-td" morerows="0" valign="top">H83</td>
<td class="description-td" morerows="0" valign="top">TTCACCAACGAG</td>
<td class="description-td" morerows="0" valign="top">50.0</td>
<td class="description-td" morerows="0" valign="top">4</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top">84</td>
<td class="description-td" morerows="0" valign="top">B07</td>
<td class="description-td" morerows="0" valign="top">CAGGTGTGGGTT</td>
<td class="description-td" morerows="0" valign="top">58.3</td>
<td class="description-td" morerows="0" valign="top">4</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top">85</td>
<td class="description-td" morerows="0" valign="top">A86</td>
<td class="description-td" morerows="0" valign="top">ATTGGTGCAGAA</td>
<td class="description-td" morerows="0" valign="top">41.7</td>
<td class="description-td" morerows="0" valign="top">5</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top">86</td>
<td class="description-td" morerows="0" valign="top">A90</td>
<td class="description-td" morerows="0" valign="top">AAGGCGTGTTTA</td>
<td class="description-td" morerows="0" valign="top">41.7</td>
<td class="description-td" morerows="0" valign="top">5</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top">87</td>
<td class="description-td" morerows="0" valign="top">C30</td>
<td class="description-td" morerows="0" valign="top">TATTGGGATTGG</td>
<td class="description-td" morerows="0" valign="top">41.7</td>
<td class="description-td" morerows="0" valign="top">5</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top">88</td>
<td class="description-td" morerows="0" valign="top">A92</td>
<td class="description-td" morerows="0" valign="top">AACATCTCCGGG</td>
<td class="description-td" morerows="0" valign="top">58.3</td>
<td class="description-td" morerows="0" valign="top">5</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top">89</td>
<td class="description-td" morerows="0" valign="top">B01</td>
<td class="description-td" morerows="0" valign="top">ATCATTGGCGAA</td>
<td class="description-td" morerows="0" valign="top">41.7</td>
<td class="description-td" morerows="0" valign="top">6</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top">90</td>
<td class="description-td" morerows="0" valign="top">B69</td>
<td class="description-td" morerows="0" valign="top">TTGAGTAGTTGC</td>
<td class="description-td" morerows="0" valign="top">41.7</td>
<td class="description-td" morerows="0" valign="top">6</td>
</tr>
<tr class="description-tr">
<td align="center" class="description-td" colspan="5" morerows="0" nameend="5" namest="1" rowsep="1" valign="top"> </td>
</tr>
</tbody></table>
</patent-tables>
</tables>
</div>
<div class="description-paragraph">
<tables>
<patent-tables colsep="0" frame="none" rowsep="0">
<table align="left" class="description-table" cols="5" colsep="0" rowsep="0" width="100%">
<thead valign="bottom">
<tr class="description-tr">
<td class="description-td" colspan="5" morerows="0" nameend="5" namest="1" rowsep="1" valign="top">                       TABLE 11</td>
</tr>
<tr class="description-tr">
<td align="center" class="description-td" colspan="5" morerows="0" nameend="5" namest="1" rowsep="1" valign="top"> </td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top"> </td>
<td class="description-td" morerows="0" valign="top"> </td>
<td class="description-td" morerows="0" valign="top"> </td>
<td class="description-td" morerows="0" valign="top"> </td>
<td class="description-td" morerows="0" valign="top">Total Band</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top">No.</td>
<td class="description-td" morerows="0" valign="top"> </td>
<td class="description-td" morerows="0" valign="top">Base Sequence 5′ → 3′</td>
<td class="description-td" morerows="0" valign="top">GC %</td>
<td class="description-td" morerows="0" valign="top">Number</td>
</tr>
<tr class="description-tr">
<td align="center" class="description-td" colspan="5" morerows="0" nameend="5" namest="1" rowsep="1" valign="top"> </td>
</tr>
</thead>
<tbody><tr class="description-tr">
<td class="description-td" morerows="0" valign="top"> </td>
</tr>
</tbody></table>
<table align="left" class="description-table" cols="5" colsep="0" rowsep="0" width="100%">
<tbody><tr class="description-tr">
<td class="description-td" morerows="0" valign="top">91</td>
<td class="description-td" morerows="0" valign="top">A91</td>
<td class="description-td" morerows="0" valign="top">TACGCCGGAATA</td>
<td class="description-td" morerows="0" valign="top">50.0</td>
<td class="description-td" morerows="0" valign="top">6</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top">92</td>
<td class="description-td" morerows="0" valign="top">A67</td>
<td class="description-td" morerows="0" valign="top">CCTGAGGTAGCT</td>
<td class="description-td" morerows="0" valign="top">58.3</td>
<td class="description-td" morerows="0" valign="top">6</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top">93</td>
<td class="description-td" morerows="0" valign="top">C47</td>
<td class="description-td" morerows="0" valign="top">GCCGCTTCAGCT</td>
<td class="description-td" morerows="0" valign="top">66.7</td>
<td class="description-td" morerows="0" valign="top">6</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top">94</td>
<td class="description-td" morerows="0" valign="top">B90</td>
<td class="description-td" morerows="0" valign="top">ATCTAAACCACG</td>
<td class="description-td" morerows="0" valign="top">41.7</td>
<td class="description-td" morerows="0" valign="top">7</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top">95</td>
<td class="description-td" morerows="0" valign="top">C65</td>
<td class="description-td" morerows="0" valign="top">AGAGCTGAAGTA</td>
<td class="description-td" morerows="0" valign="top">41.7</td>
<td class="description-td" morerows="0" valign="top">7</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top">96</td>
<td class="description-td" morerows="0" valign="top">A46</td>
<td class="description-td" morerows="0" valign="top">GGTGAGGATTCA</td>
<td class="description-td" morerows="0" valign="top">50.0</td>
<td class="description-td" morerows="0" valign="top">7</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top">97</td>
<td class="description-td" morerows="0" valign="top">C07</td>
<td class="description-td" morerows="0" valign="top">CTCAAGCGTACA</td>
<td class="description-td" morerows="0" valign="top">50.0</td>
<td class="description-td" morerows="0" valign="top">7</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top">98</td>
<td class="description-td" morerows="0" valign="top">A50</td>
<td class="description-td" morerows="0" valign="top">CCTTTCCGACGT</td>
<td class="description-td" morerows="0" valign="top">58.3</td>
<td class="description-td" morerows="0" valign="top">7</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top">99</td>
<td class="description-td" morerows="0" valign="top">H82</td>
<td class="description-td" morerows="0" valign="top">TCCTTCGAGCAG</td>
<td class="description-td" morerows="0" valign="top">58.3</td>
<td class="description-td" morerows="0" valign="top">7</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top">100</td>
<td class="description-td" morerows="0" valign="top">B83</td>
<td class="description-td" morerows="0" valign="top">CAGGCCGAAGTC</td>
<td class="description-td" morerows="0" valign="top">66.7</td>
<td class="description-td" morerows="0" valign="top">7</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top">101</td>
<td class="description-td" morerows="0" valign="top">B21</td>
<td class="description-td" morerows="0" valign="top">AAGCCTATACCA</td>
<td class="description-td" morerows="0" valign="top">41.7</td>
<td class="description-td" morerows="0" valign="top">8</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top">102</td>
<td class="description-td" morerows="0" valign="top">B86</td>
<td class="description-td" morerows="0" valign="top">CGACGATATGAT</td>
<td class="description-td" morerows="0" valign="top">41.7</td>
<td class="description-td" morerows="0" valign="top">8</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top">103</td>
<td class="description-td" morerows="0" valign="top">D08</td>
<td class="description-td" morerows="0" valign="top">GCCCTTTTGGAC</td>
<td class="description-td" morerows="0" valign="top">58.3</td>
<td class="description-td" morerows="0" valign="top">8</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top">104</td>
<td class="description-td" morerows="0" valign="top">B12</td>
<td class="description-td" morerows="0" valign="top">ACTTTCGATCCA</td>
<td class="description-td" morerows="0" valign="top">41.7</td>
<td class="description-td" morerows="0" valign="top">9</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top">105</td>
<td class="description-td" morerows="0" valign="top">B25</td>
<td class="description-td" morerows="0" valign="top">AGCACTGAATCT</td>
<td class="description-td" morerows="0" valign="top">41.7</td>
<td class="description-td" morerows="0" valign="top">9</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top">106</td>
<td class="description-td" morerows="0" valign="top">B29</td>
<td class="description-td" morerows="0" valign="top">GCCATCGAAAAA</td>
<td class="description-td" morerows="0" valign="top">41.7</td>
<td class="description-td" morerows="0" valign="top">9</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top">107</td>
<td class="description-td" morerows="0" valign="top">H87</td>
<td class="description-td" morerows="0" valign="top">GAGTACACGAAG</td>
<td class="description-td" morerows="0" valign="top">50.0</td>
<td class="description-td" morerows="0" valign="top">9</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top">108</td>
<td class="description-td" morerows="0" valign="top">C72</td>
<td class="description-td" morerows="0" valign="top">CTTGAGGGATGG</td>
<td class="description-td" morerows="0" valign="top">58.3</td>
<td class="description-td" morerows="0" valign="top">9</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top">109</td>
<td class="description-td" morerows="0" valign="top">A22</td>
<td class="description-td" morerows="0" valign="top">GCCTGCCTCACG</td>
<td class="description-td" morerows="0" valign="top">75.0</td>
<td class="description-td" morerows="0" valign="top">9</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top">110</td>
<td class="description-td" morerows="0" valign="top">C81</td>
<td class="description-td" morerows="0" valign="top">AGAGGTGTAAAT</td>
<td class="description-td" morerows="0" valign="top">33.3</td>
<td class="description-td" morerows="0" valign="top">10</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top">111</td>
<td class="description-td" morerows="0" valign="top">H84</td>
<td class="description-td" morerows="0" valign="top">AAGCTGCAGCAA</td>
<td class="description-td" morerows="0" valign="top">50.0</td>
<td class="description-td" morerows="0" valign="top">10</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top">112</td>
<td class="description-td" morerows="0" valign="top">C09</td>
<td class="description-td" morerows="0" valign="top">GCCTTCGTTACG</td>
<td class="description-td" morerows="0" valign="top">58.3</td>
<td class="description-td" morerows="0" valign="top">10</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top">113</td>
<td class="description-td" morerows="0" valign="top">C62</td>
<td class="description-td" morerows="0" valign="top">AGGGCTCTAGGC</td>
<td class="description-td" morerows="0" valign="top">66.7</td>
<td class="description-td" morerows="0" valign="top">10</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top">114</td>
<td class="description-td" morerows="0" valign="top">C82</td>
<td class="description-td" morerows="0" valign="top">TTGCATAATCGT</td>
<td class="description-td" morerows="0" valign="top">33.3</td>
<td class="description-td" morerows="0" valign="top">11</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top">115</td>
<td class="description-td" morerows="0" valign="top">C08</td>
<td class="description-td" morerows="0" valign="top">GGCAGATATCAT</td>
<td class="description-td" morerows="0" valign="top">41.7</td>
<td class="description-td" morerows="0" valign="top">11</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top">116</td>
<td class="description-td" morerows="0" valign="top">A42</td>
<td class="description-td" morerows="0" valign="top">TCCAAGCTACCA</td>
<td class="description-td" morerows="0" valign="top">50.0</td>
<td class="description-td" morerows="0" valign="top">11</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top">117</td>
<td class="description-td" morerows="0" valign="top">B72</td>
<td class="description-td" morerows="0" valign="top">TAACAACCGAGC</td>
<td class="description-td" morerows="0" valign="top">50.0</td>
<td class="description-td" morerows="0" valign="top">11</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top">118</td>
<td class="description-td" morerows="0" valign="top">B31</td>
<td class="description-td" morerows="0" valign="top">CACAAGGAACAT</td>
<td class="description-td" morerows="0" valign="top">41.7</td>
<td class="description-td" morerows="0" valign="top">12</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top">119</td>
<td class="description-td" morerows="0" valign="top">B05</td>
<td class="description-td" morerows="0" valign="top">TCGGTGGGAATA</td>
<td class="description-td" morerows="0" valign="top">50.0</td>
<td class="description-td" morerows="0" valign="top">12</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top">120</td>
<td class="description-td" morerows="0" valign="top">H86</td>
<td class="description-td" morerows="0" valign="top">ATGGAGCAGGAA</td>
<td class="description-td" morerows="0" valign="top">50.0</td>
<td class="description-td" morerows="0" valign="top">12</td>
</tr>
<tr class="description-tr">
<td align="center" class="description-td" colspan="5" morerows="0" nameend="5" namest="1" rowsep="1" valign="top"> </td>
</tr>
</tbody></table>
</patent-tables>
</tables>
</div>
<div class="description-paragraph">
<tables>
<patent-tables colsep="0" frame="none" rowsep="0">
<table align="left" class="description-table" cols="5" colsep="0" rowsep="0" width="100%">
<thead valign="bottom">
<tr class="description-tr">
<td class="description-td" colspan="5" morerows="0" nameend="5" namest="1" rowsep="1" valign="top">                         TABLE 12</td>
</tr>
<tr class="description-tr">
<td align="center" class="description-td" colspan="5" morerows="0" nameend="5" namest="1" rowsep="1" valign="top"> </td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top"> </td>
<td class="description-td" morerows="0" valign="top"> </td>
<td class="description-td" morerows="0" valign="top"> </td>
<td class="description-td" morerows="0" valign="top"> </td>
<td class="description-td" morerows="0" valign="top">Total Band</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top">No.</td>
<td class="description-td" morerows="0" valign="top"> </td>
<td class="description-td" morerows="0" valign="top">Base Sequence 5′ → 3′</td>
<td class="description-td" morerows="0" valign="top">GC %</td>
<td class="description-td" morerows="0" valign="top">Number</td>
</tr>
<tr class="description-tr">
<td align="center" class="description-td" colspan="5" morerows="0" nameend="5" namest="1" rowsep="1" valign="top"> </td>
</tr>
</thead>
<tbody><tr class="description-tr">
<td class="description-td" morerows="0" valign="top"> </td>
</tr>
</tbody></table>
<table align="left" class="description-table" cols="5" colsep="0" rowsep="0" width="100%">
<tbody><tr class="description-tr">
<td class="description-td" morerows="0" valign="top">121</td>
<td class="description-td" morerows="0" valign="top">A29</td>
<td class="description-td" morerows="0" valign="top">GGTTCGGGAATG</td>
<td class="description-td" morerows="0" valign="top">58.3</td>
<td class="description-td" morerows="0" valign="top">12</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top">122</td>
<td class="description-td" morerows="0" valign="top">C64</td>
<td class="description-td" morerows="0" valign="top">GAGCTCCCGACA</td>
<td class="description-td" morerows="0" valign="top">66.7</td>
<td class="description-td" morerows="0" valign="top">12</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top">123</td>
<td class="description-td" morerows="0" valign="top">B89</td>
<td class="description-td" morerows="0" valign="top">ACTAACCTGGAC</td>
<td class="description-td" morerows="0" valign="top">50.0</td>
<td class="description-td" morerows="0" valign="top">13</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top">124</td>
<td class="description-td" morerows="0" valign="top">B11</td>
<td class="description-td" morerows="0" valign="top">GGCGTGGTTGTA</td>
<td class="description-td" morerows="0" valign="top">58.3</td>
<td class="description-td" morerows="0" valign="top">13</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top">125</td>
<td class="description-td" morerows="0" valign="top">B41</td>
<td class="description-td" morerows="0" valign="top">GGCGAGGGAGGA</td>
<td class="description-td" morerows="0" valign="top">75.0</td>
<td class="description-td" morerows="0" valign="top">13</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top">126</td>
<td class="description-td" morerows="0" valign="top">B47</td>
<td class="description-td" morerows="0" valign="top">GCCGCCAGAGGA</td>
<td class="description-td" morerows="0" valign="top">75.0</td>
<td class="description-td" morerows="0" valign="top">13</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top">127</td>
<td class="description-td" morerows="0" valign="top">B71</td>
<td class="description-td" morerows="0" valign="top">TGACACACTGTC</td>
<td class="description-td" morerows="0" valign="top">50.0</td>
<td class="description-td" morerows="0" valign="top">14</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top">128</td>
<td class="description-td" morerows="0" valign="top">A01</td>
<td class="description-td" morerows="0" valign="top">TGCACTACAACA</td>
<td class="description-td" morerows="0" valign="top">41.7</td>
<td class="description-td" morerows="0" valign="top">15</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top">129</td>
<td class="description-td" morerows="0" valign="top">H85</td>
<td class="description-td" morerows="0" valign="top">CACTTCAACCAG</td>
<td class="description-td" morerows="0" valign="top">50.0</td>
<td class="description-td" morerows="0" valign="top">15</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top">130</td>
<td class="description-td" morerows="0" valign="top">B87</td>
<td class="description-td" morerows="0" valign="top">TATCCACCGCTC</td>
<td class="description-td" morerows="0" valign="top">58.3</td>
<td class="description-td" morerows="0" valign="top">15</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top">131</td>
<td class="description-td" morerows="0" valign="top">D22</td>
<td class="description-td" morerows="0" valign="top">TGCCCACTACGG</td>
<td class="description-td" morerows="0" valign="top">66.7</td>
<td class="description-td" morerows="0" valign="top">15</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top">132</td>
<td class="description-td" morerows="0" valign="top">B44</td>
<td class="description-td" morerows="0" valign="top">GAGACTGCTGAT</td>
<td class="description-td" morerows="0" valign="top">50.0</td>
<td class="description-td" morerows="0" valign="top">16</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top">133</td>
<td class="description-td" morerows="0" valign="top">B04</td>
<td class="description-td" morerows="0" valign="top">CAGGTGGGACCA</td>
<td class="description-td" morerows="0" valign="top">66.7</td>
<td class="description-td" morerows="0" valign="top">16</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top">134</td>
<td class="description-td" morerows="0" valign="top">B46</td>
<td class="description-td" morerows="0" valign="top">TCCTGGGGCGTT</td>
<td class="description-td" morerows="0" valign="top">66.7</td>
<td class="description-td" morerows="0" valign="top">16</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top">135</td>
<td class="description-td" morerows="0" valign="top">B51</td>
<td class="description-td" morerows="0" valign="top">GGCAAGGGATAT</td>
<td class="description-td" morerows="0" valign="top">50.0</td>
<td class="description-td" morerows="0" valign="top">17</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top">136</td>
<td class="description-td" morerows="0" valign="top">C27</td>
<td class="description-td" morerows="0" valign="top">GCATTGCAATCG</td>
<td class="description-td" morerows="0" valign="top">50.0</td>
<td class="description-td" morerows="0" valign="top">17</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top">137</td>
<td class="description-td" morerows="0" valign="top">D25</td>
<td class="description-td" morerows="0" valign="top">GTTTTGTCACCG</td>
<td class="description-td" morerows="0" valign="top">50.0</td>
<td class="description-td" morerows="0" valign="top">17</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top">138</td>
<td class="description-td" morerows="0" valign="top">C70</td>
<td class="description-td" morerows="0" valign="top">GGATCCGACGGC</td>
<td class="description-td" morerows="0" valign="top">75.0</td>
<td class="description-td" morerows="0" valign="top">17</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top">139</td>
<td class="description-td" morerows="0" valign="top">C01</td>
<td class="description-td" morerows="0" valign="top">ATGACTGTGCGA</td>
<td class="description-td" morerows="0" valign="top">50.0</td>
<td class="description-td" morerows="0" valign="top">18</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top">140</td>
<td class="description-td" morerows="0" valign="top">B48</td>
<td class="description-td" morerows="0" valign="top">GCGTCGGTTCGA</td>
<td class="description-td" morerows="0" valign="top">66.7</td>
<td class="description-td" morerows="0" valign="top">19</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top">141</td>
<td class="description-td" morerows="0" valign="top">A24</td>
<td class="description-td" morerows="0" valign="top">CTCCTGCTGTTG</td>
<td class="description-td" morerows="0" valign="top">58.3</td>
<td class="description-td" morerows="0" valign="top">20</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top">142</td>
<td class="description-td" morerows="0" valign="top">D28</td>
<td class="description-td" morerows="0" valign="top">ACTGAGGGGGGA</td>
<td class="description-td" morerows="0" valign="top">66.7</td>
<td class="description-td" morerows="0" valign="top">20</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top">143</td>
<td class="description-td" morerows="0" valign="top">A72</td>
<td class="description-td" morerows="0" valign="top">AAGGACACAACA</td>
<td class="description-td" morerows="0" valign="top">41.7</td>
<td class="description-td" morerows="0" valign="top">21</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top">144</td>
<td class="description-td" morerows="0" valign="top">B85</td>
<td class="description-td" morerows="0" valign="top">ACGGGTCGTAAC</td>
<td class="description-td" morerows="0" valign="top">58.3</td>
<td class="description-td" morerows="0" valign="top">21</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top">145</td>
<td class="description-td" morerows="0" valign="top">H92</td>
<td class="description-td" morerows="0" valign="top">GTCGGACGTCCA</td>
<td class="description-td" morerows="0" valign="top">66.7</td>
<td class="description-td" morerows="0" valign="top">21</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top">146</td>
<td class="description-td" morerows="0" valign="top">B50</td>
<td class="description-td" morerows="0" valign="top">ACTGAGCAACAA</td>
<td class="description-td" morerows="0" valign="top">41.7</td>
<td class="description-td" morerows="0" valign="top">23</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top">147</td>
<td class="description-td" morerows="0" valign="top">D09</td>
<td class="description-td" morerows="0" valign="top">CACACTCGTCAT</td>
<td class="description-td" morerows="0" valign="top">50.0</td>
<td class="description-td" morerows="0" valign="top">24</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top">148</td>
<td class="description-td" morerows="0" valign="top">C29</td>
<td class="description-td" morerows="0" valign="top">GTCGCCTTACCA</td>
<td class="description-td" morerows="0" valign="top">58.3</td>
<td class="description-td" morerows="0" valign="top">25</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top">149</td>
<td class="description-td" morerows="0" valign="top">A27</td>
<td class="description-td" morerows="0" valign="top">ATCGCGGAATAT</td>
<td class="description-td" morerows="0" valign="top">41.7</td>
<td class="description-td" morerows="0" valign="top">26</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top">150</td>
<td class="description-td" morerows="0" valign="top">B61</td>
<td class="description-td" morerows="0" valign="top">AGACCTGCTTCT</td>
<td class="description-td" morerows="0" valign="top">50.0</td>
<td class="description-td" morerows="0" valign="top">26</td>
</tr>
<tr class="description-tr">
<td align="center" class="description-td" colspan="5" morerows="0" nameend="5" namest="1" rowsep="1" valign="top"> </td>
</tr>
</tbody></table>
</patent-tables>
</tables>
</div>
<div class="description-paragraph">
<tables>
<patent-tables colsep="0" frame="none" rowsep="0">
<table align="left" class="description-table" cols="5" colsep="0" rowsep="0" width="100%">
<thead valign="bottom">
<tr class="description-tr">
<td class="description-td" colspan="5" morerows="0" nameend="5" namest="1" rowsep="1" valign="top">                       TABLE 13</td>
</tr>
<tr class="description-tr">
<td align="center" class="description-td" colspan="5" morerows="0" nameend="5" namest="1" rowsep="1" valign="top"> </td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top"> </td>
<td class="description-td" morerows="0" valign="top"> </td>
<td class="description-td" morerows="0" valign="top"> </td>
<td class="description-td" morerows="0" valign="top"> </td>
<td class="description-td" morerows="0" valign="top">Total Band</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top">No.</td>
<td class="description-td" morerows="0" valign="top"> </td>
<td class="description-td" morerows="0" valign="top">Base Sequence 5′ → 3′</td>
<td class="description-td" morerows="0" valign="top">GC %</td>
<td class="description-td" morerows="0" valign="top">Number</td>
</tr>
<tr class="description-tr">
<td align="center" class="description-td" colspan="5" morerows="0" nameend="5" namest="1" rowsep="1" valign="top"> </td>
</tr>
</thead>
<tbody><tr class="description-tr">
<td class="description-td" morerows="0" valign="top"> </td>
</tr>
</tbody></table>
<table align="left" class="description-table" cols="5" colsep="0" rowsep="0" width="100%">
<tbody><tr class="description-tr">
<td class="description-td" morerows="0" valign="top">151</td>
<td class="description-td" morerows="0" valign="top">A32</td>
<td class="description-td" morerows="0" valign="top">TTGCCGGGACCA</td>
<td class="description-td" morerows="0" valign="top">66.7</td>
<td class="description-td" morerows="0" valign="top">26</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top">152</td>
<td class="description-td" morerows="0" valign="top">B27</td>
<td class="description-td" morerows="0" valign="top">GGCGGTTATGAA</td>
<td class="description-td" morerows="0" valign="top">50.0</td>
<td class="description-td" morerows="0" valign="top">27</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top">153</td>
<td class="description-td" morerows="0" valign="top">A41</td>
<td class="description-td" morerows="0" valign="top">GTGACCGATCCA</td>
<td class="description-td" morerows="0" valign="top">58.3</td>
<td class="description-td" morerows="0" valign="top">27</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top">154</td>
<td class="description-td" morerows="0" valign="top">C63</td>
<td class="description-td" morerows="0" valign="top">GCTGGCGTATCT</td>
<td class="description-td" morerows="0" valign="top">58.3</td>
<td class="description-td" morerows="0" valign="top">27</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top">155</td>
<td class="description-td" morerows="0" valign="top">D12</td>
<td class="description-td" morerows="0" valign="top">GGACCTCCATCG</td>
<td class="description-td" morerows="0" valign="top">66.7</td>
<td class="description-td" morerows="0" valign="top">27</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top">156</td>
<td class="description-td" morerows="0" valign="top">H90</td>
<td class="description-td" morerows="0" valign="top">CCGAGGGCTGTA</td>
<td class="description-td" morerows="0" valign="top">66.7</td>
<td class="description-td" morerows="0" valign="top">27</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top">157</td>
<td class="description-td" morerows="0" valign="top">A62</td>
<td class="description-td" morerows="0" valign="top">CCTGCGGGAGGA</td>
<td class="description-td" morerows="0" valign="top">75.0</td>
<td class="description-td" morerows="0" valign="top">27</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top">158</td>
<td class="description-td" morerows="0" valign="top">A43</td>
<td class="description-td" morerows="0" valign="top">AAGTGGTGGTAT</td>
<td class="description-td" morerows="0" valign="top">41.7</td>
<td class="description-td" morerows="0" valign="top">28</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top">159</td>
<td class="description-td" morerows="0" valign="top">B70</td>
<td class="description-td" morerows="0" valign="top">TATCCTACCGGC</td>
<td class="description-td" morerows="0" valign="top">58.3</td>
<td class="description-td" morerows="0" valign="top">28</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top">160</td>
<td class="description-td" morerows="0" valign="top">C92</td>
<td class="description-td" morerows="0" valign="top">AGGCACCCTTCG</td>
<td class="description-td" morerows="0" valign="top">66.7</td>
<td class="description-td" morerows="0" valign="top">28</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top">161</td>
<td class="description-td" morerows="0" valign="top">C52</td>
<td class="description-td" morerows="0" valign="top">GTCGACGGACGT</td>
<td class="description-td" morerows="0" valign="top">66.7</td>
<td class="description-td" morerows="0" valign="top">29</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top">162</td>
<td class="description-td" morerows="0" valign="top">C04</td>
<td class="description-td" morerows="0" valign="top">GAGGAGAAACGG</td>
<td class="description-td" morerows="0" valign="top">58.3</td>
<td class="description-td" morerows="0" valign="top">30</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top">163</td>
<td class="description-td" morerows="0" valign="top">H88</td>
<td class="description-td" morerows="0" valign="top">GCTGGATTCGCA</td>
<td class="description-td" morerows="0" valign="top">58.3</td>
<td class="description-td" morerows="0" valign="top">30</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top">164</td>
<td class="description-td" morerows="0" valign="top">C22</td>
<td class="description-td" morerows="0" valign="top">GGTCACCGATCC</td>
<td class="description-td" morerows="0" valign="top">66.7</td>
<td class="description-td" morerows="0" valign="top">30</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top">165</td>
<td class="description-td" morerows="0" valign="top">B32</td>
<td class="description-td" morerows="0" valign="top">ATCGCGGCTTAT</td>
<td class="description-td" morerows="0" valign="top">50.0</td>
<td class="description-td" morerows="0" valign="top">31</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top">166</td>
<td class="description-td" morerows="0" valign="top">D23</td>
<td class="description-td" morerows="0" valign="top">ACCATCAAACGG</td>
<td class="description-td" morerows="0" valign="top">50.0</td>
<td class="description-td" morerows="0" valign="top">33</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top">167</td>
<td class="description-td" morerows="0" valign="top">B81</td>
<td class="description-td" morerows="0" valign="top">GGCCGACTTGGC</td>
<td class="description-td" morerows="0" valign="top">75.0</td>
<td class="description-td" morerows="0" valign="top">33</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top">168</td>
<td class="description-td" morerows="0" valign="top">C91</td>
<td class="description-td" morerows="0" valign="top">GAGTGGCAACGT</td>
<td class="description-td" morerows="0" valign="top">58.3</td>
<td class="description-td" morerows="0" valign="top">34</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top">169</td>
<td class="description-td" morerows="0" valign="top">A84</td>
<td class="description-td" morerows="0" valign="top">CCGCAGGGACCA</td>
<td class="description-td" morerows="0" valign="top">75.0</td>
<td class="description-td" morerows="0" valign="top">34</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top">170</td>
<td class="description-td" morerows="0" valign="top">B68</td>
<td class="description-td" morerows="0" valign="top">GGTCAGGAACAA</td>
<td class="description-td" morerows="0" valign="top">50.0</td>
<td class="description-td" morerows="0" valign="top">35</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top">171</td>
<td class="description-td" morerows="0" valign="top">B49</td>
<td class="description-td" morerows="0" valign="top">GTCGGTCGTGAA</td>
<td class="description-td" morerows="0" valign="top">58.3</td>
<td class="description-td" morerows="0" valign="top">35</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top">172</td>
<td class="description-td" morerows="0" valign="top">D24</td>
<td class="description-td" morerows="0" valign="top">GTGCAATTTGGC</td>
<td class="description-td" morerows="0" valign="top">50.0</td>
<td class="description-td" morerows="0" valign="top">36</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top">173</td>
<td class="description-td" morerows="0" valign="top">C10</td>
<td class="description-td" morerows="0" valign="top">ACTCACCACGCA</td>
<td class="description-td" morerows="0" valign="top">58.3</td>
<td class="description-td" morerows="0" valign="top">36</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top">174</td>
<td class="description-td" morerows="0" valign="top">C88</td>
<td class="description-td" morerows="0" valign="top">TGGCTTCATCAC</td>
<td class="description-td" morerows="0" valign="top">50.0</td>
<td class="description-td" morerows="0" valign="top">37</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top">175</td>
<td class="description-td" morerows="0" valign="top">D06</td>
<td class="description-td" morerows="0" valign="top">CCGTGGAATGAC</td>
<td class="description-td" morerows="0" valign="top">58.3</td>
<td class="description-td" morerows="0" valign="top">39</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top">176</td>
<td class="description-td" morerows="0" valign="top">C48</td>
<td class="description-td" morerows="0" valign="top">GGAGGATGGCCC</td>
<td class="description-td" morerows="0" valign="top">75.0</td>
<td class="description-td" morerows="0" valign="top">39</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top">177</td>
<td class="description-td" morerows="0" valign="top">C24</td>
<td class="description-td" morerows="0" valign="top">CCTTGGCATCGG</td>
<td class="description-td" morerows="0" valign="top">66.7</td>
<td class="description-td" morerows="0" valign="top">40</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top">178</td>
<td class="description-td" morerows="0" valign="top">C86</td>
<td class="description-td" morerows="0" valign="top">GTTAGCCCCAAT</td>
<td class="description-td" morerows="0" valign="top">50.0</td>
<td class="description-td" morerows="0" valign="top">41</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top">179</td>
<td class="description-td" morerows="0" valign="top">A02</td>
<td class="description-td" morerows="0" valign="top">GGCATGGCCTTT</td>
<td class="description-td" morerows="0" valign="top">58.3</td>
<td class="description-td" morerows="0" valign="top">43</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top">180</td>
<td class="description-td" morerows="0" valign="top">H89</td>
<td class="description-td" morerows="0" valign="top">GGTGACGATGCA</td>
<td class="description-td" morerows="0" valign="top">58.3</td>
<td class="description-td" morerows="0" valign="top">43</td>
</tr>
<tr class="description-tr">
<td align="center" class="description-td" colspan="5" morerows="0" nameend="5" namest="1" rowsep="1" valign="top"> </td>
</tr>
</tbody></table>
</patent-tables>
</tables>
</div>
<div class="description-paragraph">
<tables>
<patent-tables colsep="0" frame="none" rowsep="0">
<table align="left" class="description-table" cols="5" colsep="0" rowsep="0" width="100%">
<thead valign="bottom">
<tr class="description-tr">
<td class="description-td" colspan="5" morerows="0" nameend="5" namest="1" rowsep="1" valign="top">                        TABLE 14</td>
</tr>
<tr class="description-tr">
<td align="center" class="description-td" colspan="5" morerows="0" nameend="5" namest="1" rowsep="1" valign="top"> </td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top"> </td>
<td class="description-td" morerows="0" valign="top"> </td>
<td class="description-td" morerows="0" valign="top"> </td>
<td class="description-td" morerows="0" valign="top"> </td>
<td class="description-td" morerows="0" valign="top">Total Band</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top">No.</td>
<td class="description-td" morerows="0" valign="top"> </td>
<td class="description-td" morerows="0" valign="top">Base Sequence 5′ → 3′</td>
<td class="description-td" morerows="0" valign="top">GC %</td>
<td class="description-td" morerows="0" valign="top">Number</td>
</tr>
<tr class="description-tr">
<td align="center" class="description-td" colspan="5" morerows="0" nameend="5" namest="1" rowsep="1" valign="top"> </td>
</tr>
</thead>
<tbody><tr class="description-tr">
<td class="description-td" morerows="0" valign="top"> </td>
</tr>
</tbody></table>
<table align="left" class="description-table" cols="5" colsep="0" rowsep="0" width="100%">
<tbody><tr class="description-tr">
<td class="description-td" morerows="0" valign="top">181</td>
<td class="description-td" morerows="0" valign="top">B43</td>
<td class="description-td" morerows="0" valign="top">ACTGGCCGGCAT</td>
<td class="description-td" morerows="0" valign="top">66.7</td>
<td class="description-td" morerows="0" valign="top">43</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top">182</td>
<td class="description-td" morerows="0" valign="top">D32</td>
<td class="description-td" morerows="0" valign="top">AAGCTGGGGGGA</td>
<td class="description-td" morerows="0" valign="top">66.7</td>
<td class="description-td" morerows="0" valign="top">43</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top">183</td>
<td class="description-td" morerows="0" valign="top">C85</td>
<td class="description-td" morerows="0" valign="top">ATGGCTACTGGC</td>
<td class="description-td" morerows="0" valign="top">58.3</td>
<td class="description-td" morerows="0" valign="top">44</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top">184</td>
<td class="description-td" morerows="0" valign="top">B23</td>
<td class="description-td" morerows="0" valign="top">GGTGCCGGAGCA</td>
<td class="description-td" morerows="0" valign="top">75.0</td>
<td class="description-td" morerows="0" valign="top">44</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top">185</td>
<td class="description-td" morerows="0" valign="top">A25</td>
<td class="description-td" morerows="0" valign="top">CTCAGCGATACG</td>
<td class="description-td" morerows="0" valign="top">58.3</td>
<td class="description-td" morerows="0" valign="top">45</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top">186</td>
<td class="description-td" morerows="0" valign="top">A51</td>
<td class="description-td" morerows="0" valign="top">GGTGGTGGTATC</td>
<td class="description-td" morerows="0" valign="top">58.3</td>
<td class="description-td" morerows="0" valign="top">45</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top">187</td>
<td class="description-td" morerows="0" valign="top">C28</td>
<td class="description-td" morerows="0" valign="top">GTCGACGCATCA</td>
<td class="description-td" morerows="0" valign="top">58.3</td>
<td class="description-td" morerows="0" valign="top">46</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top">188</td>
<td class="description-td" morerows="0" valign="top">A44</td>
<td class="description-td" morerows="0" valign="top">GACGGTTCAAGC</td>
<td class="description-td" morerows="0" valign="top">58.3</td>
<td class="description-td" morerows="0" valign="top">47</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top">189</td>
<td class="description-td" morerows="0" valign="top">C90</td>
<td class="description-td" morerows="0" valign="top">AAGCTGTGGGCT</td>
<td class="description-td" morerows="0" valign="top">58.3</td>
<td class="description-td" morerows="0" valign="top">48</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top">190</td>
<td class="description-td" morerows="0" valign="top">A68</td>
<td class="description-td" morerows="0" valign="top">GCGGAGGAACCA</td>
<td class="description-td" morerows="0" valign="top">66.7</td>
<td class="description-td" morerows="0" valign="top">50</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top">191</td>
<td class="description-td" morerows="0" valign="top">D02</td>
<td class="description-td" morerows="0" valign="top">CCAGGAGGTGGT</td>
<td class="description-td" morerows="0" valign="top">66.7</td>
<td class="description-td" morerows="0" valign="top">51</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top">192</td>
<td class="description-td" morerows="0" valign="top">A65</td>
<td class="description-td" morerows="0" valign="top">AGCGCGGCAAAA</td>
<td class="description-td" morerows="0" valign="top">58.3</td>
<td class="description-td" morerows="0" valign="top">52</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top">193</td>
<td class="description-td" morerows="0" valign="top">D07</td>
<td class="description-td" morerows="0" valign="top">ACCACTCCCGCA</td>
<td class="description-td" morerows="0" valign="top">66.7</td>
<td class="description-td" morerows="0" valign="top">52</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top">194</td>
<td class="description-td" morerows="0" valign="top">C43</td>
<td class="description-td" morerows="0" valign="top">GGCGGCACAGGA</td>
<td class="description-td" morerows="0" valign="top">75.0</td>
<td class="description-td" morerows="0" valign="top">52</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top">195</td>
<td class="description-td" morerows="0" valign="top">A08</td>
<td class="description-td" morerows="0" valign="top">GCCCCGTTAGCA</td>
<td class="description-td" morerows="0" valign="top">66.7</td>
<td class="description-td" morerows="0" valign="top">53</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top">196</td>
<td class="description-td" morerows="0" valign="top">A31</td>
<td class="description-td" morerows="0" valign="top">AAGGCGCGAACG</td>
<td class="description-td" morerows="0" valign="top">66.7</td>
<td class="description-td" morerows="0" valign="top">53</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top">197</td>
<td class="description-td" morerows="0" valign="top">B22</td>
<td class="description-td" morerows="0" valign="top">GGTGACTGGTGG</td>
<td class="description-td" morerows="0" valign="top">66.7</td>
<td class="description-td" morerows="0" valign="top">53</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top">198</td>
<td class="description-td" morerows="0" valign="top">C44</td>
<td class="description-td" morerows="0" valign="top">CGCAGCCGAGAT</td>
<td class="description-td" morerows="0" valign="top">66.7</td>
<td class="description-td" morerows="0" valign="top">53</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top">199</td>
<td class="description-td" morerows="0" valign="top">C02</td>
<td class="description-td" morerows="0" valign="top">AAGAAGCAGGCG</td>
<td class="description-td" morerows="0" valign="top">58.3</td>
<td class="description-td" morerows="0" valign="top">54</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top">200</td>
<td class="description-td" morerows="0" valign="top">B65</td>
<td class="description-td" morerows="0" valign="top">GTGTGGAAGCCA</td>
<td class="description-td" morerows="0" valign="top">58.3</td>
<td class="description-td" morerows="0" valign="top">58</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top">201</td>
<td class="description-td" morerows="0" valign="top">A11</td>
<td class="description-td" morerows="0" valign="top">GATGGATTTGGG</td>
<td class="description-td" morerows="0" valign="top">50.0</td>
<td class="description-td" morerows="0" valign="top">60</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top">202</td>
<td class="description-td" morerows="0" valign="top">A45</td>
<td class="description-td" morerows="0" valign="top">GGTCAGGCACCA</td>
<td class="description-td" morerows="0" valign="top">66.7</td>
<td class="description-td" morerows="0" valign="top">63</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top">203</td>
<td class="description-td" morerows="0" valign="top">A07</td>
<td class="description-td" morerows="0" valign="top">TGCCTCGCACCA</td>
<td class="description-td" morerows="0" valign="top">66.7</td>
<td class="description-td" morerows="0" valign="top">65</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top">204</td>
<td class="description-td" morerows="0" valign="top">A05</td>
<td class="description-td" morerows="0" valign="top">AGCAGCGCCTCA</td>
<td class="description-td" morerows="0" valign="top">66.7</td>
<td class="description-td" morerows="0" valign="top">69</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top">205</td>
<td class="description-td" morerows="0" valign="top">A06</td>
<td class="description-td" morerows="0" valign="top">GCCAGCTGTACG</td>
<td class="description-td" morerows="0" valign="top">66.7</td>
<td class="description-td" morerows="0" valign="top">72</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top">206</td>
<td class="description-td" morerows="0" valign="top">D31</td>
<td class="description-td" morerows="0" valign="top">GGAGGTCGACCA</td>
<td class="description-td" morerows="0" valign="top">66.7</td>
<td class="description-td" morerows="0" valign="top">72</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top">207</td>
<td class="description-td" morerows="0" valign="top">A12</td>
<td class="description-td" morerows="0" valign="top">TTCGGACGAATA</td>
<td class="description-td" morerows="0" valign="top">41.7</td>
<td class="description-td" morerows="0" valign="top">74</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top">208</td>
<td class="description-td" morerows="0" valign="top">C21</td>
<td class="description-td" morerows="0" valign="top">GGAGAGCGGACG</td>
<td class="description-td" morerows="0" valign="top">75.0</td>
<td class="description-td" morerows="0" valign="top">77</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top">209</td>
<td class="description-td" morerows="0" valign="top">D21</td>
<td class="description-td" morerows="0" valign="top">GGCGATTCTGCA</td>
<td class="description-td" morerows="0" valign="top">58.3</td>
<td class="description-td" morerows="0" valign="top">83</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top">210</td>
<td class="description-td" morerows="0" valign="top">C84</td>
<td class="description-td" morerows="0" valign="top">GTGGGTGGACAA</td>
<td class="description-td" morerows="0" valign="top">58.3</td>
<td class="description-td" morerows="0" valign="top">87</td>
</tr>
<tr class="description-tr">
<td align="center" class="description-td" colspan="5" morerows="0" nameend="5" namest="1" rowsep="1" valign="top"> </td>
</tr>
</tbody></table>
</patent-tables>
</tables>
</div>
<div class="description-paragraph">
<tables>
<patent-tables colsep="0" frame="none" rowsep="0">
<table align="left" class="description-table" cols="5" colsep="0" rowsep="0" width="100%">
<thead valign="bottom">
<tr class="description-tr">
<td class="description-td" colspan="5" morerows="0" nameend="5" namest="1" rowsep="1" valign="top">                     TABLE 15</td>
</tr>
<tr class="description-tr">
<td align="center" class="description-td" colspan="5" morerows="0" nameend="5" namest="1" rowsep="1" valign="top"> </td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top"> </td>
<td class="description-td" morerows="0" valign="top"> </td>
<td class="description-td" morerows="0" valign="top"> </td>
<td class="description-td" morerows="0" valign="top"> </td>
<td class="description-td" morerows="0" valign="top">Total Band</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top">No.</td>
<td class="description-td" morerows="0" valign="top"> </td>
<td class="description-td" morerows="0" valign="top">Base Sequence 5′ → 3′</td>
<td class="description-td" morerows="0" valign="top">GC %</td>
<td class="description-td" morerows="0" valign="top">Number</td>
</tr>
<tr class="description-tr">
<td align="center" class="description-td" colspan="5" morerows="0" nameend="5" namest="1" rowsep="1" valign="top"> </td>
</tr>
</thead>
<tbody><tr class="description-tr">
<td class="description-td" morerows="0" valign="top"> </td>
</tr>
</tbody></table>
<table align="left" class="description-table" cols="5" colsep="0" rowsep="0" width="100%">
<tbody><tr class="description-tr">
<td class="description-td" morerows="0" valign="top">211</td>
<td class="description-td" morerows="0" valign="top">C83</td>
<td class="description-td" morerows="0" valign="top">GTGCACGTATGG</td>
<td class="description-td" morerows="0" valign="top">58.3</td>
<td class="description-td" morerows="0" valign="top">89</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top">212</td>
<td class="description-td" morerows="0" valign="top">A03</td>
<td class="description-td" morerows="0" valign="top">CGACGACGACGA</td>
<td class="description-td" morerows="0" valign="top">66.7</td>
<td class="description-td" morerows="0" valign="top">89</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top">213</td>
<td class="description-td" morerows="0" valign="top">A10</td>
<td class="description-td" morerows="0" valign="top">ACTGGCCGAGGG</td>
<td class="description-td" morerows="0" valign="top">75.0</td>
<td class="description-td" morerows="0" valign="top">89</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top">214</td>
<td class="description-td" morerows="0" valign="top">B67</td>
<td class="description-td" morerows="0" valign="top">GCGGTCAGCACA</td>
<td class="description-td" morerows="0" valign="top">66.7</td>
<td class="description-td" morerows="0" valign="top">99</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top">215</td>
<td class="description-td" morerows="0" valign="top">A04</td>
<td class="description-td" morerows="0" valign="top">ATCAGCGCACCA</td>
<td class="description-td" morerows="0" valign="top">58.3</td>
<td class="description-td" morerows="0" valign="top">112</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top">216</td>
<td class="description-td" morerows="0" valign="top">D11</td>
<td class="description-td" morerows="0" valign="top">ATGGCCGGTGGG</td>
<td class="description-td" morerows="0" valign="top">75.0</td>
<td class="description-td" morerows="0" valign="top">117</td>
</tr>
<tr class="description-tr">
<td align="center" class="description-td" colspan="5" morerows="0" nameend="5" namest="1" rowsep="1" valign="top"> </td>
</tr>
</tbody></table>
</patent-tables>
</tables>
</div>
<div class="description-paragraph">The band numbers, varying with the primers, were distributed in the range of 0 to 117. FIG. 9 shows the primer numbers every six bands. Most of the primers were shifted to smaller band numbers.</div>
<div class="description-paragraph">The inventive method of amplifying DNA fragments requires primers having the minimum appearance frequencies of bands in the electrophoretic patterns. FIG. 10 illustrates the numbers of primers for the respective band numbers as to primers each having not more than 15 bands.</div>
<div class="description-paragraph">Although the primers have the same length of 12 bp (base pairs), the appearance frequencies of the bands remarkably vary with the types of the primers, due to difference in affinity between the primers and template DNA. One of the indications deciding the affinity is the GC content. The GC content stands for the ratio occupied by the total number of C and G for the total number of the four bases A, C, G and T.</div>
<div class="description-paragraph">FIG. 11 illustrates band numbers (total band numbers) for the GC contents of the 216 primers. Tables 8 to 15 also show the GC contents of the primers. As shown in FIG. 11, positive correlation is observed between the GC contents and the band numbers, and such a tendency has been confirmed that the band number increases in proportion to the GC content.</div>
<div class="description-paragraph">Referring to FIGS. 9 and 10, the band number corresponding to each primer indicates the amplification probability of the primer. The genome size of bacteria is about 5×10<sup>6 </sup>bp, for example, and hence it is estimated that a DNA fragment is amplified for 10<sup>6 </sup>bp if the band number (total band number) confirmed from seven bacteria is 3.5. When arranging the primers in the openings 51 of the apparatus for amplifying DNA fragments shown in FIGS. 4 or <b>5</b> on the basis of the band numbers shown in FIGS. 9 and 10, therefore, a plurality of microorganisms contained in a microorganism flora can be effectively discriminated.</div>
<div class="description-paragraph">In this Example, 94 primers in total were selected from those having band numbers of 1 to 11, those having band numbers of 0 and GC contents in the range of 40 to 60% and those having band numbers of 20, and arranged in the apparatus for amplifying DNA fragments shown in FIGS. 4 or <b>5</b> substantially in order of the amplification probabilities. The 94 primers as selected had sequence Nos. 21 to 38, 42 to 115, 117 and 141. Referring to FIG. 5, a combination of chromosome DNA of the bacterium No. 10 in Table 3 and the primer having the sequence No. 69 was employed as the positive control, while the primer of the sequence No. 69 was employed as the negative control.</div>
<div class="description-paragraph">The selected primers are not restricted to the above but those having large band numbers can be selected from Tables 8 to 15 for assaying a microorganism flora formed by a small number of types of microorganisms while primers having small band numbers can be selected from Tables 8 to 15 when assaying a microorganism flora formed by a large number of types of microorganisms, for example.</div>
<heading>Example 2</heading>
<div class="description-paragraph">Then, a group of microorganisms, particularly bacteria contained in the tank of a garbage disposal were analyzed by the inventive method of assaying a group of microorganisms with the 94 types of primers (sequence Nos. 21 to 38, 42 to 115, 117 and 141) selected in Example 1.</div>
<div class="description-paragraph">{circle around (1)} DNA Analysis of Bacterium No.10 Isolated from Garbage Disposal</div>
<div class="description-paragraph">In this Example, the bacterium No. 10 in Table 3 isolated from the tank of the garbage disposal in Example 1 was employed as a sample and chromosome DNA of this bacterium was prepared. The bacterium No. 10 was isolated and the chromosome DNA was prepared by methods similar to those described with reference to Example 1.</div>
<div class="description-paragraph">Then, reaction solutions for random PCR were prepared with the chromosome DNA of the bacterium No. 10. The composition of the reaction solutions for random PCR is shown in Table 6 for Example 1. In this case, 94 types of reaction solutions for random PCR containing the 94 primers of the sequence Nos. 21 to 38, 42 to 115, 117 and 141 respectively were simultaneously prepared. Further, a reaction solution containing the primer of the sequence No. 69 and template DNA having a base sequence corresponding thereto was prepared as the positive control while a reaction solution containing the primer of the sequence No. 69 with no DNA was prepared as the negative control simultaneously with the 94 types of reaction solutions for random PCR.</div>
<div class="description-paragraph">Then, the 94 types of reaction solutions for random PCR were stored in the 94 openings 51 of the apparatus for amplifying DNA fragments shown in FIG. 5, while the negative control and the positive control were stored in the openings <b>51</b> <i>a </i>and <b>51</b> <i>b </i>respectively. Random PCR was performed with the apparatus for amplifying DNA fragments, and thereafter the reaction solutions were analyzed by electrophoresis. Electrophoretic patterns thus obtained were stained and photographed. The random PCR, the electrophoresis and the staining of the electrophoretic patterns and photographing after the electrophoresis are identical to those described with reference to Example 1.</div>
<div class="description-paragraph">In the electrophoretic patterns obtained by the electrophoresis, 14 clear bands (DNA fragments) appeared as to nine types of primers. No band was confirmed in relation to the negative control. Thus, it has been possible to confirm that the appearing bands belonged to the bacterium No. 10. Further, the luminous intensity of a band of the positive control was measured for correcting those of the remaining bands on the basis thereof. If the measured luminous intensity of the band of the positive control is 70%, for example, the luminous intensity of this band is corrected to be 100% while those of the remaining bands are also corrected in similar ratios. Thus, influence exerted on the amplification efficiency for the DNA fragments by errors of reaction conditions etc. in DNA fragment amplification was eliminated.</div>
<div class="description-paragraph">FIG. 12 shows an electrophoretic pattern as to eight types of primers of the sequence Nos. 64 to 71 among the 94 types of primers. Numerals 1 to 8 on the horizontal axis of the electrophoretic pattern in FIG. 12 denote the primers of the sequence Nos. 64 to 71 respectively.</div>
<div class="description-paragraph">As shown in FIG. 12, a single band appeared in each of the primers of the sequence Nos. 67 and 69. Thus, it was indicated that DNA fragments of the bacterium No. 10 were amplified by the primers of the sequence Nos. 67 and 69. From the positions of the appearing bands, further, it was estimated that the amplified DNA fragments were about 990 bp and about 1800 bp in size.</div>
<div class="description-paragraph">{circle around (2)} DNA Analysis of Five Types of Bacteria Isolated from Garbage Disposal</div>
<div class="description-paragraph">In this Example, five types of bacteria isolated from the garbage disposal on the 490<sup>th </sup>day from starting driving were employed as samples, and chromosome DNA was prepared from each bacterium. The five types of bacteria were isolated and the chromosome DNA was prepared by methods similar to those described with reference to Example 1.</div>
<div class="description-paragraph">Then, a chromosome DNA mixture of the five types of bacteria was prepared by mixing the chromosome DNA of the prepared bacteria. 94 types of reaction solutions for random PCR were prepared with the chromosome DNA mixture and 94 types of primers, while preparing a positive control and a negative control. The composition of the reaction solutions for random PCR is identical to that shown in Table 6 with reference to Example 1, except that each reaction solution contains 10 pg/μL of chromosome DNA of each bacterium in concentration. Further, random PCR was performed with the reaction solutions for random PCR, the positive control and the negative control, and thereafter the reaction solutions were analyzed by electrophoresis. Thereafter obtained electrophoretic patterns were stained and photographed. The random PCR and the electrophoresis are identical to those described with reference to Example 1. Further, bands were corrected on the basis of amplification efficiency with the positive control by a method similar to that described with reference to {circle around (1)} of example 2.</div>
<div class="description-paragraph">In the electrophoretic patterns of the reaction solutions for random PCR obtained by the electrophoresis, 85 clear bands appeared in 40 types of primers.</div>
<div class="description-paragraph">FIG. 13 shows an electrophoretic pattern as to eight types of primers of the sequence Nos. 64 to 71 among the 94 types of primers. Numerals 1 to 8 on the horizontal axis of the electrophoretic pattern shown in FIG. 13 denote the primers of the sequence Nos. 64 to 71 respectively.</div>
<div class="description-paragraph">As shown in FIG. 13, the number of the bands appearing when employing the chromosome DNA mixture of five types of bacteria was larger than that of the bands appearing when employing the chromosome DNA of the bacterium No. 10 shown in FIG. 12, and nine bands in total appeared in four primers of the sequence Nos. 65, 67, 69 and 71. Thus, the number of the bands appearing when employing five types of bacteria is about five times that of the bands appearing when employing the bacterium No. 10 shown in FIG. <b>9</b>. The primers of the sequence Nos. 64 to 71 amplify about 0.3 DNA fragments on the average from chromosome DNA of a bacterium. When amplifying chromosome DNA of a bacterium with eight types of primers having such an amplification probability, about two to three (2.4 in calculation) DNA fragments are obtained. Thus, when amplifying chromosome DNA of five types of bacteria with the above eight types of primers, DNA fragments of five times those amplified from a bacterium, i.e., 12 DNA fragments are obtained in calculation.</div>
<div class="description-paragraph">Thus, when a plurality of types of bacteria are mixed with each other, DNA fragments are amplified in a number responsive to the number of types of the bacteria. Further, the differences between the types of the bacteria can be detected as the combinations of the primer and the sizes of the DNA fragments amplified with that primer. Therefore, the state of each bacterium forming the bacteria can be readily estimated from the obtained electrophoretic pattern.</div>
<div class="description-paragraph">In the bands appearing in the primers of the sequence Nos. 69 and 71 shown in FIG. 13, those with arrows matched with the bands appearing in the electrophoretic pattern of the bacterium No. 10 shown in FIG. <b>9</b>. Thus, it is estimable that the five types of bacteria isolated from the garbage disposal include the bacterium No. 10.</div>
<div class="description-paragraph">The difference between the number of the actually amplified DNA fragments and the value in calculation results from that the lengths of chromosome DNA vary with the types of the bacteria and from the following:</div>
<div class="description-paragraph">The four bases are not contained in chromosome DNA of bacteria in the same ratio but the numbers of these bases are unbalanced and the order of sequence of each base is different. Such unbalanced states of the bases vary with the types of the bacteria. When chromosome DNA of a bacterium has unbalanced base sequences, the numbers of DNA fragments amplified by a plurality of primers differ from each other even if the primers have the same amplification probability. The number of amplified DNA fragments is large when employing a primer having a base sequence complementary to that contained in the chromosome DNA in a large number, while the number of amplified DNA fragments is small when employing a primer having a base sequence complementary to that contained in a small number. If the number of types of primers employed for the random PCR method is small, therefore, sufficient information cannot be obtained due to unbalanced types of the amplified DNA fragments.</div>
<div class="description-paragraph">For example, the primer of the sequence No. 67 amplifies a DNA fragment for the single bacterium No. 10, while amplifying two DNA fragments for the five types of bacteria. On the other hand, the primer of the sequence No. 69 amplifies a DNA fragment for the single bacterium No. 10, while amplifying five DNA fragments for the five types of bacteria. Thus, the number of types of bacteria and the number of DNA fragments are correlated in the primer of the sequence No. 69, while no correlation is observed between the number of types of bacteria and the number of DNA fragments in the primer of the sequence No. 67. This is because the five types of bacteria contain the base sequence complementary to the primer of the sequence No. 69 in a large number, while only the bacterium No. 10 contains the base sequence complementary to the primer of sequence No. 67 in a large number and the remaining four types of bacteria contain the base sequence complementary thereto in a small number. When employing a plurality of primers, on the contrary, DNA fragments are obtained from the plurality of primers and hence the total number of obtained bands approaches the value in calculation obtained from the types of the bacteria, the types of the primers and the amplification probabilities of the primers as a whole. Thus, the number of types of bacteria can be estimated from the total number of amplified DNA fragments. Also when employing about eight types of primers, the total number of amplified DNA fragments reflected the number of types of the bacteria, as described above.</div>
<div class="description-paragraph">{circle around (3)} Analysis of Bacteria in Garbage Disposal</div>
<div class="description-paragraph">In this Example, 10 g of wood chips were sampled from the tank of a garbage disposal on the 150<sup>th </sup>day from starting driving. 90 mL of a sterilized 0.85% salt solution was added to the wood chips and thereafter bacteria were isolated from the wood chips by a suspension method. In this case, a masticator by Gunze Sangyo, Ltd. was employed as a suspension apparatus for performing treatment for one minute. Thereafter the treated solution was filtered through a coarse prefilter (stoma filter by Gunze Sangyo, Ltd.). Further, the solution was successively filtered through membrane filters of 25 μm and 5 μm.</div>
<div class="description-paragraph">FIG. 14 shows states of the filtrate observed with a microscope in the above filtration steps. Referring to FIG. 14, (a) shows the state of the filtrate filtered through the prefilter, and (b) shows the state after filtration through the membrane filter of 25 μm. Further, (c) shows the state after filtration through the membrane filter of 5 μm.</div>
<div class="description-paragraph">As shown at (a) in FIG. 14, the filtrate contained foreign matter exceeding 25 μm such as the kitchen garbage in the process of degradation and bits of wood and microorganisms such as protozoa larger then the bacteria after filtration through the coarse prefilter. When filtering this filtrate through the membrane filter of 25 μm, the large foreign matter was removed as shown at (b) in FIG. <b>14</b>. This filtrate still contained large cells, conceivably protozoa, exceeding 10 μm. When further filtering this filtrate through the membrane filter of 5 μm, the large cells were removed while small dots remained. These small dots, exhibiting bacterial motion in observation with a microscope, were confirmed to be bacteria.</div>
<div class="description-paragraph">Thus, it was possible to efficiently gather bacteria by filtration through membrane filters.</div>
<div class="description-paragraph">Then, the filtrate filtered through the membrane filter of 5 μm was centrifuged by 1800 g for 15 minutes, to recover a pellet formed by precipitation of the bacteria. Thus, bacteria were extracted from the tank of the garbage disposal.</div>
<div class="description-paragraph">Then, chromosome DNA of the extracted bacteria was extracted by a method similar to that in Example 1, and dissolved in 50 μL of a mixed solution (hereinafter referred to as TE solution) of 10 mM of Tris-HCl (pH 8.0) and 1 mM of ethylenediamine-N, N, N′, N′-tetraacetic acid (EDTA). Thus prepared was a TE solution of a chromosome DNA mixture containing chromosome DNA of a plurality of different types of bacteria.</div>
<div class="description-paragraph">Then, reaction solutions for random PCR were prepared with the chromosome DNA mixture while preparing a positive control and a negative control. The compositions of the reaction solutions for random PCR, the positive control and the negative control are identical to those described with reference to {circle around (1)} of Example 2. In this case, 0.02 μL of the TE solution of the chromosome DNA mixture was added to 20 μL of each reaction solution for random PCR.</div>
<div class="description-paragraph">Random PCR was performed with the apparatus for amplifying DNA fragments shown in FIG. 5, and thereafter the reaction solutions were analyzed by electrophoresis. The random PCR and the electrophoresis are identical to those described with reference to Example 1. In this case, a quantitatively analyzable DNA size marker (λ/Hind III and BioLad Amplisize Standard) having a known concentration was electrophoresed simultaneously with the reaction solutions.</div>
<div class="description-paragraph">After the electrophoresis, obtained electrophoretic patterns were stained with ethidium bromide for acquiring ethidium bromide fluorescent images irradiated with ultraviolet light of 254 nm in wavelength with a Polaroid camera or a CCD camera. In this case, the electrophoretic images were acquired as to the first to 12<sup>th </sup>lanes of the apparatus for amplifying DNA fragments, to obtain 12 electrophoretic photographs Nos. 1 to 12. Each electrophoretic photograph shows electrophoretic images of eight types of different primers and the DNA size marker. FIG. 15 shows the electrophoretic photograph No. 6 among those obtained in the aforementioned manner.</div>
<div class="description-paragraph">FIG. 15 shows the electrophoretic photograph No. 6 on the sixth lane of the apparatus for amplifying DNA fragments with electrophoretic images of the eight types of primers of the sequence Nos. 64 to 71. Referring to FIG. 15, symbol M denotes the electrophoretic image of the DNA size marker, and numerals 1 to 8 on the horizontal axis denote the primers of the sequence Nos. 64 to 71 respectively.</div>
<div class="description-paragraph">As shown in FIG. 15, a plurality of bands appeared in each of the primers of the sequence Nos. 64, 65, 67, 68, 69 and 71.</div>
<div class="description-paragraph">Each of the electrophoretic photographs Nos. 1 to 12 was incorporated in a computer with a scanner, and thereafter the luminous intensities of the bands in the DNA size marker M at 560 bp and each primer were measured with software (Genomic Solutions Advance Quantifier I-D Match). Further, amplification efficiency for the DNA fragments was obtained from the positive control by a method similar to that described with reference to {circle around (1)} of Example 2, for correcting the bands of the reaction solutions for random PCR on the basis thereof.</div>
<div class="description-paragraph">In each electrophoretic photograph, the bands of the DNA size marker M were so corrected that the luminous intensities obtained by measurement were 100%, while the luminous intensities of the remaining bands were also corrected in a similar ratio. Errors in gradient can be eliminated in each electrophoretic photograph by thus correcting the gradient of the electrophoretic photograph.</div>
<div class="description-paragraph">A luminous intensity half that of the band of the DNA size marker M at 560 bp obtained by measurement was set as the threshold, for removing bands having luminous intensities less than the threshold in each primer. When thus removing bands having small luminous intensities, bands having low reproducibility in random PCR are removed for improving the reproducibility in random PCR. Thus, the reliability of the obtained data is improved.</div>
<div class="description-paragraph">Table 16 shows the numbers of bands having luminous intensities exceeding the threshold in the respective electrophoretic photographs.</div>
<div class="description-paragraph">
<tables>
<patent-tables colsep="0" frame="none" rowsep="0">
<table align="left" class="description-table" cols="4" colsep="0" rowsep="0" width="100%">
<thead valign="bottom">
<tr class="description-tr">
<td class="description-td" colspan="4" morerows="0" nameend="4" namest="1" rowsep="1" valign="top">                                   TABLE 16</td>
</tr>
<tr class="description-tr">
<td align="center" class="description-td" colspan="4" morerows="0" nameend="4" namest="1" rowsep="1" valign="top"> </td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top"> </td>
<td class="description-td" morerows="0" valign="top">Corresponding</td>
<td class="description-td" morerows="0" valign="top">Band Intensity</td>
<td class="description-td" morerows="0" valign="top">Total Number of Bands</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top"> </td>
<td class="description-td" morerows="0" valign="top">Lane of DNA</td>
<td class="description-td" morerows="0" valign="top">of DNA Size</td>
<td class="description-td" morerows="0" valign="top">having intensity</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top">Photo No.</td>
<td class="description-td" morerows="0" valign="top">Amplifier</td>
<td class="description-td" morerows="0" valign="top">Marker of 560 bp</td>
<td class="description-td" morerows="0" valign="top">exceeding Threshold</td>
</tr>
<tr class="description-tr">
<td align="center" class="description-td" colspan="4" morerows="0" nameend="4" namest="1" rowsep="1" valign="top"> </td>
</tr>
</thead>
<tbody><tr class="description-tr">
<td class="description-td" morerows="0" valign="top"> </td>
</tr>
</tbody></table>
<table align="left" class="description-table" cols="4" colsep="0" rowsep="0" width="100%">
<tbody><tr class="description-tr">
<td class="description-td" morerows="0" valign="top">1</td>
<td class="description-td" morerows="0" valign="top">1<sup>st </sup>lane</td>
<td class="description-td" morerows="0" valign="top">0.155</td>
<td class="description-td" morerows="0" valign="top">3</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top">2</td>
<td class="description-td" morerows="0" valign="top">2<sup>nd </sup>lane</td>
<td class="description-td" morerows="0" valign="top">0.265</td>
<td class="description-td" morerows="0" valign="top">7</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top">3</td>
<td class="description-td" morerows="0" valign="top">3<sup>rd </sup>lane</td>
<td class="description-td" morerows="0" valign="top">0.305</td>
<td class="description-td" morerows="0" valign="top">22</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top">4</td>
<td class="description-td" morerows="0" valign="top">4<sup>th </sup>lane</td>
<td class="description-td" morerows="0" valign="top">0.260</td>
<td class="description-td" morerows="0" valign="top">9</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top">5</td>
<td class="description-td" morerows="0" valign="top">5<sup>th </sup>lane</td>
<td class="description-td" morerows="0" valign="top">0.335</td>
<td class="description-td" morerows="0" valign="top">18</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top">6</td>
<td class="description-td" morerows="0" valign="top">6<sup>th </sup>lane</td>
<td class="description-td" morerows="0" valign="top">0.315</td>
<td class="description-td" morerows="0" valign="top">20</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top">7</td>
<td class="description-td" morerows="0" valign="top">7<sup>th </sup>lane</td>
<td class="description-td" morerows="0" valign="top">0.320</td>
<td class="description-td" morerows="0" valign="top">26</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top">8</td>
<td class="description-td" morerows="0" valign="top">8<sup>th </sup>lane</td>
<td class="description-td" morerows="0" valign="top">0.325</td>
<td class="description-td" morerows="0" valign="top">37</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top">9</td>
<td class="description-td" morerows="0" valign="top">9<sup>th </sup>lane</td>
<td class="description-td" morerows="0" valign="top">0.345</td>
<td class="description-td" morerows="0" valign="top">38</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top">10</td>
<td class="description-td" morerows="0" valign="top">10<sup>th </sup>lane </td>
<td class="description-td" morerows="0" valign="top">0.240</td>
<td class="description-td" morerows="0" valign="top">52</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top">11</td>
<td class="description-td" morerows="0" valign="top">11<sup>th </sup>lane </td>
<td class="description-td" morerows="0" valign="top">0.335</td>
<td class="description-td" morerows="0" valign="top">35</td>
</tr>
<tr class="description-tr">
<td class="description-td" morerows="0" valign="top">12</td>
<td class="description-td" morerows="0" valign="top">12<sup>th </sup>lane </td>
<td class="description-td" morerows="0" valign="top">0.255</td>
<td class="description-td" morerows="0" valign="top">42</td>
</tr>
<tr class="description-tr">
<td align="center" class="description-td" colspan="4" morerows="0" nameend="4" namest="1" rowsep="1" valign="top"> </td>
</tr>
</tbody></table>
</patent-tables>
</tables>
</div>
<div class="description-paragraph">As shown in Table 16, such a tendency was recognized that the number of amplified DNA fragments increases in proportion to the number allotted to the photograph. Thus, it has been suggested that employment of a plurality of primers having a low amplification probability arranged on upper lanes of the apparatus for amplifying DNA fragments is effective when a large number of types of bacteria form the bacteria flora or when analyzing bacteria having large-sized chromosome DNA while employment of a plurality of primers having a high amplification probability arranged on lower lanes of the apparatus for amplifying DNA fragments is effective when a small number of types of bacteria form the bacteria flora or when analyzing bacteria having small-sized chromosome DNA.</div>
<div class="description-paragraph">As described in the above Examples, it is possible to simultaneously amplify DNA fragments from a plurality of microorganisms forming a microorganism flora with each of a plurality of primers having different amplification probabilities or different orders of amplification probabilities. Thus, an electrophoretic image amplified at the optimum amplification probability can be obtained for every microorganism with no information as to the number of the microorganisms forming the microorganism flora or the size of the chromosomes thereof. Further, the plurality of microorganisms contained in the microorganism flora can be discriminated by analyzing the electrophoretic images. Consequently, various microorganic ecosystems can be correctly assayed in a short time.</div>
<div class="description-paragraph">Further, time change of the number of types of the microorganisms forming the microorganism flora can be assayed by assaying the microorganisms in time.</div>
<div class="description-paragraph">When discriminating the type of the microorganism and analyzing the band pattern of its chromosome DNA as to each of a plurality of microorganisms and establishing a database with the obtained data of the band pattern of the chromosome DNA of each microorganism, the types of the microorganisms forming the microorganism flora can be retrieved from the database by searching the database on the basis of the DNA band patterns of the microorganisms analyzed in the aforementioned manner.</div>
<div class="description-paragraph">Although the present invention has been described and illustrated in detail, it is clearly understood that the same is by way of illustration and example only and is not to be taken by way of limitation, the spirit and scope of the present invention being limited only by the terms of the appended claims.</div>
<div class="description-paragraph">
<sequence-cwu id="SEQLST-1">
<sequence-list-new-rules>
<s100>
<s160>216</s160>
</s100>
<s200>
<s210>1</s210>
<s211>12</s211>
<s212>DNA</s212>
<s213>Artificial Sequence</s213>
<s220>
<s223>Primer</s223>
</s220>
</s200>
<s400> 1
actgagaaaa ta                                                         12</s400>
<s200>
<s210>2</s210>
<s211>12</s211>
<s212>DNA</s212>
<s213>Artificial Sequence</s213>
<s220>
<s223>Primer</s223>
</s220>
</s200>
<s400> 2
atcttcaaag at                                                         12</s400>
<s200>
<s210>3</s210>
<s211>12</s211>
<s212>DNA</s212>
<s213>Artificial Sequence</s213>
<s220>
<s223>Primer</s223>
</s220>
</s200>
<s400> 3
acaaagagat at                                                         12</s400>
<s200>
<s210>4</s210>
<s211>12</s211>
<s212>DNA</s212>
<s213>Artificial Sequence</s213>
<s220>
<s223>Primer</s223>
</s220>
</s200>
<s400> 4
gaggtgatat ta                                                         12</s400>
<s200>
<s210>5</s210>
<s211>12</s211>
<s212>DNA</s212>
<s213>Artificial Sequence</s213>
<s220>
<s223>Primer</s223>
</s220>
</s200>
<s400> 5
atcttctcat ct                                                         12</s400>
<s200>
<s210>6</s210>
<s211>12</s211>
<s212>DNA</s212>
<s213>Artificial Sequence</s213>
<s220>
<s223>Primer</s223>
</s220>
</s200>
<s400> 6
actcttctac aa                                                         12</s400>
<s200>
<s210>7</s210>
<s211>12</s211>
<s212>DNA</s212>
<s213>Artificial Sequence</s213>
<s220>
<s223>Primer</s223>
</s220>
</s200>
<s400> 7
agagacatag tt                                                         12</s400>
<s200>
<s210>8</s210>
<s211>12</s211>
<s212>DNA</s212>
<s213>Artificial Sequence</s213>
<s220>
<s223>Primer</s223>
</s220>
</s200>
<s400> 8
gccagatata ta                                                         12</s400>
<s200>
<s210>9</s210>
<s211>12</s211>
<s212>DNA</s212>
<s213>Artificial Sequence</s213>
<s220>
<s223>Primer</s223>
</s220>
</s200>
<s400> 9
cacactactt at                                                         12</s400>
<s200>
<s210>10</s210>
<s211>12</s211>
<s212>DNA</s212>
<s213>Artificial Sequence</s213>
<s220>
<s223>Primer</s223>
</s220>
</s200>
<s400> 10
atgagaaagg aa                                                         12</s400>
<s200>
<s210>11</s210>
<s211>12</s211>
<s212>DNA</s212>
<s213>Artificial Sequence</s213>
<s220>
<s223>Primer</s223>
</s220>
</s200>
<s400> 11
atcaacactt tc                                                         12</s400>
<s200>
<s210>12</s210>
<s211>12</s211>
<s212>DNA</s212>
<s213>Artificial Sequence</s213>
<s220>
<s223>Primer</s223>
</s220>
</s200>
<s400> 12
gagactacaa ta                                                         12</s400>
<s200>
<s210>13</s210>
<s211>12</s211>
<s212>DNA</s212>
<s213>Artificial Sequence</s213>
<s220>
<s223>Primer</s223>
</s220>
</s200>
<s400> 13
ccatacatat tg                                                         12</s400>
<s200>
<s210>14</s210>
<s211>12</s211>
<s212>DNA</s212>
<s213>Artificial Sequence</s213>
<s220>
<s223>Primer</s223>
</s220>
</s200>
<s400> 14
gatactgatg at                                                         12</s400>
<s200>
<s210>15</s210>
<s211>12</s211>
<s212>DNA</s212>
<s213>Artificial Sequence</s213>
<s220>
<s223>Primer</s223>
</s220>
</s200>
<s400> 15
agattcttac tg                                                         12</s400>
<s200>
<s210>16</s210>
<s211>12</s211>
<s212>DNA</s212>
<s213>Artificial Sequence</s213>
<s220>
<s223>Primer</s223>
</s220>
</s200>
<s400> 16
agcccttatt ta                                                         12</s400>
<s200>
<s210>17</s210>
<s211>12</s211>
<s212>DNA</s212>
<s213>Artificial Sequence</s213>
<s220>
<s223>Primer</s223>
</s220>
</s200>
<s400> 17
gagactatga aa                                                         12</s400>
<s200>
<s210>18</s210>
<s211>12</s211>
<s212>DNA</s212>
<s213>Artificial Sequence</s213>
<s220>
<s223>Primer</s223>
</s220>
</s200>
<s400> 18
atcaagtatc ca                                                         12</s400>
<s200>
<s210>19</s210>
<s211>12</s211>
<s212>DNA</s212>
<s213>Artificial Sequence</s213>
<s220>
<s223>Primer</s223>
</s220>
</s200>
<s400> 19
actgacctag tt                                                         12</s400>
<s200>
<s210>20</s210>
<s211>12</s211>
<s212>DNA</s212>
<s213>Artificial Sequence</s213>
<s220>
<s223>Primer</s223>
</s220>
</s200>
<s400> 20
atttggatag gg                                                         12</s400>
<s200>
<s210>21</s210>
<s211>12</s211>
<s212>DNA</s212>
<s213>Artificial Sequence</s213>
<s220>
<s223>Primer</s223>
</s220>
</s200>
<s400> 21
gactgctata ca                                                         12</s400>
<s200>
<s210>22</s210>
<s211>12</s211>
<s212>DNA</s212>
<s213>Artificial Sequence</s213>
<s220>
<s223>Primer</s223>
</s220>
</s200>
<s400> 22
tggtacggta ta                                                         12</s400>
<s200>
<s210>23</s210>
<s211>12</s211>
<s212>DNA</s212>
<s213>Artificial Sequence</s213>
<s220>
<s223>Primer</s223>
</s220>
</s200>
<s400> 23
tactactgtg ga                                                         12</s400>
<s200>
<s210>24</s210>
<s211>12</s211>
<s212>DNA</s212>
<s213>Artificial Sequence</s213>
<s220>
<s223>Primer</s223>
</s220>
</s200>
<s400> 24
tggctgtaga aa                                                         12</s400>
<s200>
<s210>25</s210>
<s211>12</s211>
<s212>DNA</s212>
<s213>Artificial Sequence</s213>
<s220>
<s223>Primer</s223>
</s220>
</s200>
<s400> 25
tctacacgaa gt                                                         12</s400>
<s200>
<s210>26</s210>
<s211>12</s211>
<s212>DNA</s212>
<s213>Artificial Sequence</s213>
<s220>
<s223>Primer</s223>
</s220>
</s200>
<s400> 26
ccagattttc tg                                                         12</s400>
<s200>
<s210>27</s210>
<s211>12</s211>
<s212>DNA</s212>
<s213>Artificial Sequence</s213>
<s220>
<s223>Primer</s223>
</s220>
</s200>
<s400> 27
atcaacgtac gt                                                         12</s400>
<s200>
<s210>28</s210>
<s211>12</s211>
<s212>DNA</s212>
<s213>Artificial Sequence</s213>
<s220>
<s223>Primer</s223>
</s220>
</s200>
<s400> 28
ttccgtaatc ac                                                         12</s400>
<s200>
<s210>29</s210>
<s211>12</s211>
<s212>DNA</s212>
<s213>Artificial Sequence</s213>
<s220>
<s223>Primer</s223>
</s220>
</s200>
<s400> 29
gcttacatag ac                                                         12</s400>
<s200>
<s210>30</s210>
<s211>12</s211>
<s212>DNA</s212>
<s213>Artificial Sequence</s213>
<s220>
<s223>Primer</s223>
</s220>
</s200>
<s400> 30
actccaaatg tg                                                         12</s400>
<s200>
<s210>31</s210>
<s211>12</s211>
<s212>DNA</s212>
<s213>Artificial Sequence</s213>
<s220>
<s223>Primer</s223>
</s220>
</s200>
<s400> 31
aagtcgtttg gg                                                         12</s400>
<s200>
<s210>32</s210>
<s211>12</s211>
<s212>DNA</s212>
<s213>Artificial Sequence</s213>
<s220>
<s223>Primer</s223>
</s220>
</s200>
<s400> 32
ctagtatggg ac                                                         12</s400>
<s200>
<s210>33</s210>
<s211>12</s211>
<s212>DNA</s212>
<s213>Artificial Sequence</s213>
<s220>
<s223>Primer</s223>
</s220>
</s200>
<s400> 33
gagttcgaac ga                                                         12</s400>
<s200>
<s210>34</s210>
<s211>12</s211>
<s212>DNA</s212>
<s213>Artificial Sequence</s213>
<s220>
<s223>Primer</s223>
</s220>
</s200>
<s400> 34
actttcctac gg                                                         12</s400>
<s200>
<s210>35</s210>
<s211>12</s211>
<s212>DNA</s212>
<s213>Artificial Sequence</s213>
<s220>
<s223>Primer</s223>
</s220>
</s200>
<s400> 35
ttcccgtcta tc                                                         12</s400>
<s200>
<s210>36</s210>
<s211>12</s211>
<s212>DNA</s212>
<s213>Artificial Sequence</s213>
<s220>
<s223>Primer</s223>
</s220>
</s200>
<s400> 36
agccttacgg ca                                                         12</s400>
<s200>
<s210>37</s210>
<s211>12</s211>
<s212>DNA</s212>
<s213>Artificial Sequence</s213>
<s220>
<s223>Primer</s223>
</s220>
</s200>
<s400> 37
gctatggcaa cg                                                         12</s400>
<s200>
<s210>38</s210>
<s211>12</s211>
<s212>DNA</s212>
<s213>Artificial Sequence</s213>
<s220>
<s223>Primer</s223>
</s220>
</s200>
<s400> 38
ccttggaact cg                                                         12</s400>
<s200>
<s210>39</s210>
<s211>12</s211>
<s212>DNA</s212>
<s213>Artificial Sequence</s213>
<s220>
<s223>Primer</s223>
</s220>
</s200>
<s400> 39
tctgctgacc gg                                                         12</s400>
<s200>
<s210>40</s210>
<s211>12</s211>
<s212>DNA</s212>
<s213>Artificial Sequence</s213>
<s220>
<s223>Primer</s223>
</s220>
</s200>
<s400> 40
ggcaactggc ca                                                         12</s400>
<s200>
<s210>41</s210>
<s211>12</s211>
<s212>DNA</s212>
<s213>Artificial Sequence</s213>
<s220>
<s223>Primer</s223>
</s220>
</s200>
<s400> 41
gtcattaaag ct                                                         12</s400>
<s200>
<s210>42</s210>
<s211>12</s211>
<s212>DNA</s212>
<s213>Artificial Sequence</s213>
<s220>
<s223>Primer</s223>
</s220>
</s200>
<s400> 42
tacacttttg ac                                                         12</s400>
<s200>
<s210>43</s210>
<s211>12</s211>
<s212>DNA</s212>
<s213>Artificial Sequence</s213>
<s220>
<s223>Primer</s223>
</s220>
</s200>
<s400> 43
tggatctttg ac                                                         12</s400>
<s200>
<s210>44</s210>
<s211>12</s211>
<s212>DNA</s212>
<s213>Artificial Sequence</s213>
<s220>
<s223>Primer</s223>
</s220>
</s200>
<s400> 44
gctcttttgg aa                                                         12</s400>
<s200>
<s210>45</s210>
<s211>12</s211>
<s212>DNA</s212>
<s213>Artificial Sequence</s213>
<s220>
<s223>Primer</s223>
</s220>
</s200>
<s400> 45
atcatcgtac gt                                                         12</s400>
<s200>
<s210>46</s210>
<s211>12</s211>
<s212>DNA</s212>
<s213>Artificial Sequence</s213>
<s220>
<s223>Primer</s223>
</s220>
</s200>
<s400> 46
tacgatatgg ct                                                         12</s400>
<s200>
<s210>47</s210>
<s211>12</s211>
<s212>DNA</s212>
<s213>Artificial Sequence</s213>
<s220>
<s223>Primer</s223>
</s220>
</s200>
<s400> 47
gatccagtct tt                                                         12</s400>
<s200>
<s210>48</s210>
<s211>12</s211>
<s212>DNA</s212>
<s213>Artificial Sequence</s213>
<s220>
<s223>Primer</s223>
</s220>
</s200>
<s400> 48
agaatgtccg ta                                                         12</s400>
<s200>
<s210>49</s210>
<s211>12</s211>
<s212>DNA</s212>
<s213>Artificial Sequence</s213>
<s220>
<s223>Primer</s223>
</s220>
</s200>
<s400> 49
ccgcagttag at                                                         12</s400>
<s200>
<s210>50</s210>
<s211>12</s211>
<s212>DNA</s212>
<s213>Artificial Sequence</s213>
<s220>
<s223>Primer</s223>
</s220>
</s200>
<s400> 50
cagtgggagt tt                                                         12</s400>
<s200>
<s210>51</s210>
<s211>12</s211>
<s212>DNA</s212>
<s213>Artificial Sequence</s213>
<s220>
<s223>Primer</s223>
</s220>
</s200>
<s400> 51
tctatggacc ct                                                         12</s400>
<s200>
<s210>52</s210>
<s211>12</s211>
<s212>DNA</s212>
<s213>Artificial Sequence</s213>
<s220>
<s223>Primer</s223>
</s220>
</s200>
<s400> 52
ttcattctgg gg                                                         12</s400>
<s200>
<s210>53</s210>
<s211>12</s211>
<s212>DNA</s212>
<s213>Artificial Sequence</s213>
<s220>
<s223>Primer</s223>
</s220>
</s200>
<s400> 53
ccgtcttttc tg                                                         12</s400>
<s200>
<s210>54</s210>
<s211>12</s211>
<s212>DNA</s212>
<s213>Artificial Sequence</s213>
<s220>
<s223>Primer</s223>
</s220>
</s200>
<s400> 54
tggcctattg gc                                                         12</s400>
<s200>
<s210>55</s210>
<s211>12</s211>
<s212>DNA</s212>
<s213>Artificial Sequence</s213>
<s220>
<s223>Primer</s223>
</s220>
</s200>
<s400> 55
gtcatgcctg ga                                                         12</s400>
<s200>
<s210>56</s210>
<s211>12</s211>
<s212>DNA</s212>
<s213>Artificial Sequence</s213>
<s220>
<s223>Primer</s223>
</s220>
</s200>
<s400> 56
ccttggcgaa gc                                                         12</s400>
<s200>
<s210>57</s210>
<s211>12</s211>
<s212>DNA</s212>
<s213>Artificial Sequence</s213>
<s220>
<s223>Primer</s223>
</s220>
</s200>
<s400> 57
agcctgtggg ct                                                         12</s400>
<s200>
<s210>58</s210>
<s211>12</s211>
<s212>DNA</s212>
<s213>Artificial Sequence</s213>
<s220>
<s223>Primer</s223>
</s220>
</s200>
<s400> 58
agaattggac ga                                                         12</s400>
<s200>
<s210>59</s210>
<s211>12</s211>
<s212>DNA</s212>
<s213>Artificial Sequence</s213>
<s220>
<s223>Primer</s223>
</s220>
</s200>
<s400> 59
cttgtcatgt gt                                                         12</s400>
<s200>
<s210>60</s210>
<s211>12</s211>
<s212>DNA</s212>
<s213>Artificial Sequence</s213>
<s220>
<s223>Primer</s223>
</s220>
</s200>
<s400> 60
tacgtggtaa ca                                                         12</s400>
<s200>
<s210>61</s210>
<s211>12</s211>
<s212>DNA</s212>
<s213>Artificial Sequence</s213>
<s220>
<s223>Primer</s223>
</s220>
</s200>
<s400> 61
ggacaagtaa tg                                                         12</s400>
<s200>
<s210>62</s210>
<s211>12</s211>
<s212>DNA</s212>
<s213>Artificial Sequence</s213>
<s220>
<s223>Primer</s223>
</s220>
</s200>
<s400> 62
taccctcaag ct                                                         12</s400>
<s200>
<s210>63</s210>
<s211>12</s211>
<s212>DNA</s212>
<s213>Artificial Sequence</s213>
<s220>
<s223>Primer</s223>
</s220>
</s200>
<s400> 63
ttcgaggatc ga                                                         12</s400>
<s200>
<s210>64</s210>
<s211>12</s211>
<s212>DNA</s212>
<s213>Artificial Sequence</s213>
<s220>
<s223>Primer</s223>
</s220>
</s200>
<s400> 64
gacctgcgat ct                                                         12</s400>
<s200>
<s210>65</s210>
<s211>12</s211>
<s212>DNA</s212>
<s213>Artificial Sequence</s213>
<s220>
<s223>Primer</s223>
</s220>
</s200>
<s400> 65
tggcctcttg ga                                                         12</s400>
<s200>
<s210>66</s210>
<s211>12</s211>
<s212>DNA</s212>
<s213>Artificial Sequence</s213>
<s220>
<s223>Primer</s223>
</s220>
</s200>
<s400> 66
ggtttcccag ga                                                         12</s400>
<s200>
<s210>67</s210>
<s211>12</s211>
<s212>DNA</s212>
<s213>Artificial Sequence</s213>
<s220>
<s223>Primer</s223>
</s220>
</s200>
<s400> 67
tcgtccggag at                                                         12</s400>
<s200>
<s210>68</s210>
<s211>12</s211>
<s212>DNA</s212>
<s213>Artificial Sequence</s213>
<s220>
<s223>Primer</s223>
</s220>
</s200>
<s400> 68
cgcttcgtag ca                                                         12</s400>
<s200>
<s210>69</s210>
<s211>12</s211>
<s212>DNA</s212>
<s213>Artificial Sequence</s213>
<s220>
<s223>Primer</s223>
</s220>
</s200>
<s400> 69
ggcttcgaat cg                                                         12</s400>
<s200>
<s210>70</s210>
<s211>12</s211>
<s212>DNA</s212>
<s213>Artificial Sequence</s213>
<s220>
<s223>Primer</s223>
</s220>
</s200>
<s400> 70
gatgagctaa aa                                                         12</s400>
<s200>
<s210>71</s210>
<s211>12</s211>
<s212>DNA</s212>
<s213>Artificial Sequence</s213>
<s220>
<s223>Primer</s223>
</s220>
</s200>
<s400> 71
gagcaggaat at                                                         12</s400>
<s200>
<s210>72</s210>
<s211>12</s211>
<s212>DNA</s212>
<s213>Artificial Sequence</s213>
<s220>
<s223>Primer</s223>
</s220>
</s200>
<s400> 72
cttaggttac gt                                                         12</s400>
<s200>
<s210>73</s210>
<s211>12</s211>
<s212>DNA</s212>
<s213>Artificial Sequence</s213>
<s220>
<s223>Primer</s223>
</s220>
</s200>
<s400> 73
gtggatctga at                                                         12</s400>
<s200>
<s210>74</s210>
<s211>12</s211>
<s212>DNA</s212>
<s213>Artificial Sequence</s213>
<s220>
<s223>Primer</s223>
</s220>
</s200>
<s400> 74
cctatcccaa ca                                                         12</s400>
<s200>
<s210>75</s210>
<s211>12</s211>
<s212>DNA</s212>
<s213>Artificial Sequence</s213>
<s220>
<s223>Primer</s223>
</s220>
</s200>
<s400> 75
gagactaccg aa                                                         12</s400>
<s200>
<s210>76</s210>
<s211>12</s211>
<s212>DNA</s212>
<s213>Artificial Sequence</s213>
<s220>
<s223>Primer</s223>
</s220>
</s200>
<s400> 76
gacagcgtcc ta                                                         12</s400>
<s200>
<s210>77</s210>
<s211>12</s211>
<s212>DNA</s212>
<s213>Artificial Sequence</s213>
<s220>
<s223>Primer</s223>
</s220>
</s200>
<s400> 77
cttgagcgta tt                                                         12</s400>
<s200>
<s210>78</s210>
<s211>12</s211>
<s212>DNA</s212>
<s213>Artificial Sequence</s213>
<s220>
<s223>Primer</s223>
</s220>
</s200>
<s400> 78
actgagatag ca                                                         12</s400>
<s200>
<s210>79</s210>
<s211>12</s211>
<s212>DNA</s212>
<s213>Artificial Sequence</s213>
<s220>
<s223>Primer</s223>
</s220>
</s200>
<s400> 79
gagagacgat ta                                                         12</s400>
<s200>
<s210>80</s210>
<s211>12</s211>
<s212>DNA</s212>
<s213>Artificial Sequence</s213>
<s220>
<s223>Primer</s223>
</s220>
</s200>
<s400> 80
gacgcccatt at                                                         12</s400>
<s200>
<s210>81</s210>
<s211>12</s211>
<s212>DNA</s212>
<s213>Artificial Sequence</s213>
<s220>
<s223>Primer</s223>
</s220>
</s200>
<s400> 81
gacggttcta ca                                                         12</s400>
<s200>
<s210>82</s210>
<s211>12</s211>
<s212>DNA</s212>
<s213>Artificial Sequence</s213>
<s220>
<s223>Primer</s223>
</s220>
</s200>
<s400> 82
gatggtccgt tt                                                         12</s400>
<s200>
<s210>83</s210>
<s211>12</s211>
<s212>DNA</s212>
<s213>Artificial Sequence</s213>
<s220>
<s223>Primer</s223>
</s220>
</s200>
<s400> 83
ttcaccaacg ag                                                         12</s400>
<s200>
<s210>84</s210>
<s211>12</s211>
<s212>DNA</s212>
<s213>Artificial Sequence</s213>
<s220>
<s223>Primer</s223>
</s220>
</s200>
<s400> 84
caggtgtggg tt                                                         12</s400>
<s200>
<s210>85</s210>
<s211>12</s211>
<s212>DNA</s212>
<s213>Artificial Sequence</s213>
<s220>
<s223>Primer</s223>
</s220>
</s200>
<s400> 85
attggtgcag aa                                                         12</s400>
<s200>
<s210>86</s210>
<s211>12</s211>
<s212>DNA</s212>
<s213>Artificial Sequence</s213>
<s220>
<s223>Primer</s223>
</s220>
</s200>
<s400> 86
aaggcgtgtt ta                                                         12</s400>
<s200>
<s210>87</s210>
<s211>12</s211>
<s212>DNA</s212>
<s213>Artificial Sequence</s213>
<s220>
<s223>Primer</s223>
</s220>
</s200>
<s400> 87
tattgggatt gg                                                         12</s400>
<s200>
<s210>88</s210>
<s211>12</s211>
<s212>DNA</s212>
<s213>Artificial Sequence</s213>
<s220>
<s223>Primer</s223>
</s220>
</s200>
<s400> 88
aacatctccg gg                                                         12</s400>
<s200>
<s210>89</s210>
<s211>12</s211>
<s212>DNA</s212>
<s213>Artificial Sequence</s213>
<s220>
<s223>Primer</s223>
</s220>
</s200>
<s400> 89
atcattggcg aa                                                         12</s400>
<s200>
<s210>90</s210>
<s211>12</s211>
<s212>DNA</s212>
<s213>Artificial Sequence</s213>
<s220>
<s223>Primer</s223>
</s220>
</s200>
<s400> 90
ttgagtagtt gc                                                         12</s400>
<s200>
<s210>91</s210>
<s211>12</s211>
<s212>DNA</s212>
<s213>Artificial Sequence</s213>
<s220>
<s223>Primer</s223>
</s220>
</s200>
<s400> 91
tacgccggaa ta                                                         12</s400>
<s200>
<s210>92</s210>
<s211>12</s211>
<s212>DNA</s212>
<s213>Artificial Sequence</s213>
<s220>
<s223>Primer</s223>
</s220>
</s200>
<s400> 92
cctgaggtag ct                                                         12</s400>
<s200>
<s210>93</s210>
<s211>12</s211>
<s212>DNA</s212>
<s213>Artificial Sequence</s213>
<s220>
<s223>Primer</s223>
</s220>
</s200>
<s400> 93
gccgcttcag ct                                                         12</s400>
<s200>
<s210>94</s210>
<s211>12</s211>
<s212>DNA</s212>
<s213>Artificial Sequence</s213>
<s220>
<s223>Primer</s223>
</s220>
</s200>
<s400> 94
atctaaacca cg                                                         12</s400>
<s200>
<s210>95</s210>
<s211>12</s211>
<s212>DNA</s212>
<s213>Artificial Sequence</s213>
<s220>
<s223>Primer</s223>
</s220>
</s200>
<s400> 95
agagctgaag ta                                                         12</s400>
<s200>
<s210>96</s210>
<s211>12</s211>
<s212>DNA</s212>
<s213>Artificial Sequence</s213>
<s220>
<s223>Primer</s223>
</s220>
</s200>
<s400> 96
ggtgaggatt ca                                                         12</s400>
<s200>
<s210>97</s210>
<s211>12</s211>
<s212>DNA</s212>
<s213>Artificial Sequence</s213>
<s220>
<s223>Primer</s223>
</s220>
</s200>
<s400> 97
ctcaagcgta ca                                                         12</s400>
<s200>
<s210>98</s210>
<s211>12</s211>
<s212>DNA</s212>
<s213>Artificial Sequence</s213>
<s220>
<s223>Primer</s223>
</s220>
</s200>
<s400> 98
cctttccgac gt                                                         12</s400>
<s200>
<s210>99</s210>
<s211>12</s211>
<s212>DNA</s212>
<s213>Artificial Sequence</s213>
<s220>
<s223>Primer</s223>
</s220>
</s200>
<s400> 99
tccttcgagc ag                                                         12</s400>
<s200>
<s210>100</s210>
<s211>12</s211>
<s212>DNA</s212>
<s213>Artificial Sequence</s213>
<s220>
<s223>Primer</s223>
</s220>
</s200>
<s400> 100
caggccgaag tc                                                         12</s400>
<s200>
<s210>101</s210>
<s211>12</s211>
<s212>DNA</s212>
<s213>Artificial Sequence</s213>
<s220>
<s223>Primer</s223>
</s220>
</s200>
<s400> 101
aagcctatac ca                                                         12</s400>
<s200>
<s210>102</s210>
<s211>12</s211>
<s212>DNA</s212>
<s213>Artificial Sequence</s213>
<s220>
<s223>Primer</s223>
</s220>
</s200>
<s400> 102
cgacgatatg at                                                         12</s400>
<s200>
<s210>103</s210>
<s211>12</s211>
<s212>DNA</s212>
<s213>Artificial Sequence</s213>
<s220>
<s223>Primer</s223>
</s220>
</s200>
<s400> 103
gcccttttgg ac                                                         12</s400>
<s200>
<s210>104</s210>
<s211>12</s211>
<s212>DNA</s212>
<s213>Artificial Sequence</s213>
<s220>
<s223>Primer</s223>
</s220>
</s200>
<s400> 104
actttcgatc ca                                                         12</s400>
<s200>
<s210>105</s210>
<s211>12</s211>
<s212>DNA</s212>
<s213>Artificial Sequence</s213>
<s220>
<s223>Primer</s223>
</s220>
</s200>
<s400> 105
agcactgaat ct                                                         12</s400>
<s200>
<s210>106</s210>
<s211>12</s211>
<s212>DNA</s212>
<s213>Artificial Sequence</s213>
<s220>
<s223>Primer</s223>
</s220>
</s200>
<s400> 106
gccatcgaaa aa                                                         12</s400>
<s200>
<s210>107</s210>
<s211>12</s211>
<s212>DNA</s212>
<s213>Artificial Sequence</s213>
<s220>
<s223>Primer</s223>
</s220>
</s200>
<s400> 107
gagtacacga ag                                                         12</s400>
<s200>
<s210>108</s210>
<s211>12</s211>
<s212>DNA</s212>
<s213>Artificial Sequence</s213>
<s220>
<s223>Primer</s223>
</s220>
</s200>
<s400> 108
cttgagggat gg                                                         12</s400>
<s200>
<s210>109</s210>
<s211>12</s211>
<s212>DNA</s212>
<s213>Artificial Sequence</s213>
<s220>
<s223>Primer</s223>
</s220>
</s200>
<s400> 109
gcctgcctca cg                                                         12</s400>
<s200>
<s210>110</s210>
<s211>12</s211>
<s212>DNA</s212>
<s213>Artificial Sequence</s213>
<s220>
<s223>Primer</s223>
</s220>
</s200>
<s400> 110
agaggtgtaa at                                                         12</s400>
<s200>
<s210>111</s210>
<s211>12</s211>
<s212>DNA</s212>
<s213>Artificial Sequence</s213>
<s220>
<s223>Primer</s223>
</s220>
</s200>
<s400> 111
aagctgcagc aa                                                         12</s400>
<s200>
<s210>112</s210>
<s211>12</s211>
<s212>DNA</s212>
<s213>Artificial Sequence</s213>
<s220>
<s223>Primer</s223>
</s220>
</s200>
<s400> 112
gccttcgtta cg                                                         12</s400>
<s200>
<s210>113</s210>
<s211>12</s211>
<s212>DNA</s212>
<s213>Artificial Sequence</s213>
<s220>
<s223>Primer</s223>
</s220>
</s200>
<s400> 113
agggctctag gc                                                         12</s400>
<s200>
<s210>114</s210>
<s211>12</s211>
<s212>DNA</s212>
<s213>Artificial Sequence</s213>
<s220>
<s223>Primer</s223>
</s220>
</s200>
<s400> 114
ttgcataatc gt                                                         12</s400>
<s200>
<s210>115</s210>
<s211>12</s211>
<s212>DNA</s212>
<s213>Artificial Sequence</s213>
<s220>
<s223>Primer</s223>
</s220>
</s200>
<s400> 115
ggcagatatc at                                                         12</s400>
<s200>
<s210>116</s210>
<s211>12</s211>
<s212>DNA</s212>
<s213>Artificial Sequence</s213>
<s220>
<s223>Primer</s223>
</s220>
</s200>
<s400> 116
tccaagctac ca                                                         12</s400>
<s200>
<s210>117</s210>
<s211>12</s211>
<s212>DNA</s212>
<s213>Artificial Sequence</s213>
<s220>
<s223>Primer</s223>
</s220>
</s200>
<s400> 117
taacaaccga gc                                                         12</s400>
<s200>
<s210>118</s210>
<s211>12</s211>
<s212>DNA</s212>
<s213>Artificial Sequence</s213>
<s220>
<s223>Primer</s223>
</s220>
</s200>
<s400> 118
cacaaggaac at                                                         12</s400>
<s200>
<s210>119</s210>
<s211>12</s211>
<s212>DNA</s212>
<s213>Artificial Sequence</s213>
<s220>
<s223>Primer</s223>
</s220>
</s200>
<s400> 119
tcggtgggaa ta                                                         12</s400>
<s200>
<s210>120</s210>
<s211>12</s211>
<s212>DNA</s212>
<s213>Artificial Sequence</s213>
<s220>
<s223>Primer</s223>
</s220>
</s200>
<s400> 120
atggagcagg aa                                                         12</s400>
<s200>
<s210>121</s210>
<s211>12</s211>
<s212>DNA</s212>
<s213>Artificial Sequence</s213>
<s220>
<s223>Primer</s223>
</s220>
</s200>
<s400> 121
ggttcgggaa tg                                                         12</s400>
<s200>
<s210>122</s210>
<s211>12</s211>
<s212>DNA</s212>
<s213>Artificial Sequence</s213>
<s220>
<s223>Primer</s223>
</s220>
</s200>
<s400> 122
gagctcccga ca                                                         12</s400>
<s200>
<s210>123</s210>
<s211>12</s211>
<s212>DNA</s212>
<s213>Artificial Sequence</s213>
<s220>
<s223>Primer</s223>
</s220>
</s200>
<s400> 123
actaacctgg ac                                                         12</s400>
<s200>
<s210>124</s210>
<s211>12</s211>
<s212>DNA</s212>
<s213>Artificial Sequence</s213>
<s220>
<s223>Primer</s223>
</s220>
</s200>
<s400> 124
ggcgtggttg ta                                                         12</s400>
<s200>
<s210>125</s210>
<s211>12</s211>
<s212>DNA</s212>
<s213>Artificial Sequence</s213>
<s220>
<s223>Primer</s223>
</s220>
</s200>
<s400> 125
ggcgagggag ga                                                         12</s400>
<s200>
<s210>126</s210>
<s211>12</s211>
<s212>DNA</s212>
<s213>Artificial Sequence</s213>
<s220>
<s223>Primer</s223>
</s220>
</s200>
<s400> 126
gccgccagag ga                                                         12</s400>
<s200>
<s210>127</s210>
<s211>12</s211>
<s212>DNA</s212>
<s213>Artificial Sequence</s213>
<s220>
<s223>Primer</s223>
</s220>
</s200>
<s400> 127
tgacacactg tc                                                         12</s400>
<s200>
<s210>128</s210>
<s211>12</s211>
<s212>DNA</s212>
<s213>Artificial Sequence</s213>
<s220>
<s223>Primer</s223>
</s220>
</s200>
<s400> 128
tgcactacaa ca                                                         12</s400>
<s200>
<s210>129</s210>
<s211>12</s211>
<s212>DNA</s212>
<s213>Artificial Sequence</s213>
<s220>
<s223>Primer</s223>
</s220>
</s200>
<s400> 129
cacttcaacc ag                                                         12</s400>
<s200>
<s210>130</s210>
<s211>12</s211>
<s212>DNA</s212>
<s213>Artificial Sequence</s213>
<s220>
<s223>Primer</s223>
</s220>
</s200>
<s400> 130
tatccaccgc tc                                                         12</s400>
<s200>
<s210>131</s210>
<s211>12</s211>
<s212>DNA</s212>
<s213>Artificial Sequence</s213>
<s220>
<s223>Primer</s223>
</s220>
</s200>
<s400> 131
tgcccactac gg                                                         12</s400>
<s200>
<s210>132</s210>
<s211>12</s211>
<s212>DNA</s212>
<s213>Artificial Sequence</s213>
<s220>
<s223>Primer</s223>
</s220>
</s200>
<s400> 132
gagactgctg at                                                         12</s400>
<s200>
<s210>133</s210>
<s211>12</s211>
<s212>DNA</s212>
<s213>Artificial Sequence</s213>
<s220>
<s223>Primer</s223>
</s220>
</s200>
<s400> 133
caggtgggac ca                                                         12</s400>
<s200>
<s210>134</s210>
<s211>12</s211>
<s212>DNA</s212>
<s213>Artificial Sequence</s213>
<s220>
<s223>Primer</s223>
</s220>
</s200>
<s400> 134
tcctggggcg tt                                                         12</s400>
<s200>
<s210>135</s210>
<s211>12</s211>
<s212>DNA</s212>
<s213>Artificial Sequence</s213>
<s220>
<s223>Primer</s223>
</s220>
</s200>
<s400> 135
ggcaagggat at                                                         12</s400>
<s200>
<s210>136</s210>
<s211>12</s211>
<s212>DNA</s212>
<s213>Artificial Sequence</s213>
<s220>
<s223>Primer</s223>
</s220>
</s200>
<s400> 136
gcattgcaat cg                                                         12</s400>
<s200>
<s210>137</s210>
<s211>12</s211>
<s212>DNA</s212>
<s213>Artificial Sequence</s213>
<s220>
<s223>Primer</s223>
</s220>
</s200>
<s400> 137
gttttgtcac cg                                                         12</s400>
<s200>
<s210>138</s210>
<s211>12</s211>
<s212>DNA</s212>
<s213>Artificial Sequence</s213>
<s220>
<s223>Primer</s223>
</s220>
</s200>
<s400> 138
ggatccgacg gc                                                         12</s400>
<s200>
<s210>139</s210>
<s211>12</s211>
<s212>DNA</s212>
<s213>Artificial Sequence</s213>
<s220>
<s223>Primer</s223>
</s220>
</s200>
<s400> 139
atgactgtgc ga                                                         12</s400>
<s200>
<s210>140</s210>
<s211>12</s211>
<s212>DNA</s212>
<s213>Artificial Sequence</s213>
<s220>
<s223>Primer</s223>
</s220>
</s200>
<s400> 140
gcgtcggttc ga                                                         12</s400>
<s200>
<s210>141</s210>
<s211>12</s211>
<s212>DNA</s212>
<s213>Artificial Sequence</s213>
<s220>
<s223>Primer</s223>
</s220>
</s200>
<s400> 141
ctcctgctgt tg                                                         12</s400>
<s200>
<s210>142</s210>
<s211>12</s211>
<s212>DNA</s212>
<s213>Artificial Sequence</s213>
<s220>
<s223>Primer</s223>
</s220>
</s200>
<s400> 142
actgaggggg ga                                                         12</s400>
<s200>
<s210>143</s210>
<s211>12</s211>
<s212>DNA</s212>
<s213>Artificial Sequence</s213>
<s220>
<s223>Primer</s223>
</s220>
</s200>
<s400> 143
aaggacacaa ca                                                         12</s400>
<s200>
<s210>144</s210>
<s211>12</s211>
<s212>DNA</s212>
<s213>Artificial Sequence</s213>
<s220>
<s223>Primer</s223>
</s220>
</s200>
<s400> 144
acgggtcgta ac                                                         12</s400>
<s200>
<s210>145</s210>
<s211>12</s211>
<s212>DNA</s212>
<s213>Artificial Sequence</s213>
<s220>
<s223>Primer</s223>
</s220>
</s200>
<s400> 145
gtcggacgtc ca                                                         12</s400>
<s200>
<s210>146</s210>
<s211>12</s211>
<s212>DNA</s212>
<s213>Artificial Sequence</s213>
<s220>
<s223>Primer</s223>
</s220>
</s200>
<s400> 146
actgagcaac aa                                                         12</s400>
<s200>
<s210>147</s210>
<s211>12</s211>
<s212>DNA</s212>
<s213>Artificial Sequence</s213>
<s220>
<s223>Primer</s223>
</s220>
</s200>
<s400> 147
cacactcgtc at                                                         12</s400>
<s200>
<s210>148</s210>
<s211>12</s211>
<s212>DNA</s212>
<s213>Artificial Sequence</s213>
<s220>
<s223>Primer</s223>
</s220>
</s200>
<s400> 148
gtcgccttac ca                                                         12</s400>
<s200>
<s210>149</s210>
<s211>12</s211>
<s212>DNA</s212>
<s213>Artificial Sequence</s213>
<s220>
<s223>Primer</s223>
</s220>
</s200>
<s400> 149
atcgcggaat at                                                         12</s400>
<s200>
<s210>150</s210>
<s211>12</s211>
<s212>DNA</s212>
<s213>Artificial Sequence</s213>
<s220>
<s223>Primer</s223>
</s220>
</s200>
<s400> 150
agacctgctt ct                                                         12</s400>
<s200>
<s210>151</s210>
<s211>12</s211>
<s212>DNA</s212>
<s213>Artificial Sequence</s213>
<s220>
<s223>Primer</s223>
</s220>
</s200>
<s400> 151
ttgccgggac ca                                                         12</s400>
<s200>
<s210>152</s210>
<s211>12</s211>
<s212>DNA</s212>
<s213>Artificial Sequence</s213>
<s220>
<s223>Primer</s223>
</s220>
</s200>
<s400> 152
ggcggttatg aa                                                         12</s400>
<s200>
<s210>153</s210>
<s211>12</s211>
<s212>DNA</s212>
<s213>Artificial Sequence</s213>
<s220>
<s223>Primer</s223>
</s220>
</s200>
<s400> 153
gtgaccgatc ca                                                         12</s400>
<s200>
<s210>154</s210>
<s211>12</s211>
<s212>DNA</s212>
<s213>Artificial Sequence</s213>
<s220>
<s223>Primer</s223>
</s220>
</s200>
<s400> 154
gctggcgtat ct                                                         12</s400>
<s200>
<s210>155</s210>
<s211>12</s211>
<s212>DNA</s212>
<s213>Artificial Sequence</s213>
<s220>
<s223>Primer</s223>
</s220>
</s200>
<s400> 155
ggacctccat cg                                                         12</s400>
<s200>
<s210>156</s210>
<s211>12</s211>
<s212>DNA</s212>
<s213>Artificial Sequence</s213>
<s220>
<s223>Primer</s223>
</s220>
</s200>
<s400> 156
ccgagggctg ta                                                         12</s400>
<s200>
<s210>157</s210>
<s211>12</s211>
<s212>DNA</s212>
<s213>Artificial Sequence</s213>
<s220>
<s223>Primer</s223>
</s220>
</s200>
<s400> 157
cctgcgggag ga                                                         12</s400>
<s200>
<s210>158</s210>
<s211>12</s211>
<s212>DNA</s212>
<s213>Artificial Sequence</s213>
<s220>
<s223>Primer</s223>
</s220>
</s200>
<s400> 158
aagtggtggt at                                                         12</s400>
<s200>
<s210>159</s210>
<s211>12</s211>
<s212>DNA</s212>
<s213>Artificial Sequence</s213>
<s220>
<s223>Primer</s223>
</s220>
</s200>
<s400> 159
tatcctaccg gc                                                         12</s400>
<s200>
<s210>160</s210>
<s211>12</s211>
<s212>DNA</s212>
<s213>Artificial Sequence</s213>
<s220>
<s223>Primer</s223>
</s220>
</s200>
<s400> 160
aggcaccctt cg                                                         12</s400>
<s200>
<s210>161</s210>
<s211>12</s211>
<s212>DNA</s212>
<s213>Artificial Sequence</s213>
<s220>
<s223>Primer</s223>
</s220>
</s200>
<s400> 161
gtcgacggac gt                                                         12</s400>
<s200>
<s210>162</s210>
<s211>12</s211>
<s212>DNA</s212>
<s213>Artificial Sequence</s213>
<s220>
<s223>Primer</s223>
</s220>
</s200>
<s400> 162
gaggagaaac gg                                                         12</s400>
<s200>
<s210>163</s210>
<s211>12</s211>
<s212>DNA</s212>
<s213>Artificial Sequence</s213>
<s220>
<s223>Primer</s223>
</s220>
</s200>
<s400> 163
gctggattcg ca                                                         12</s400>
<s200>
<s210>164</s210>
<s211>12</s211>
<s212>DNA</s212>
<s213>Artificial Sequence</s213>
<s220>
<s223>Primer</s223>
</s220>
</s200>
<s400> 164
ggtcaccgat cc                                                         12</s400>
<s200>
<s210>165</s210>
<s211>12</s211>
<s212>DNA</s212>
<s213>Artificial Sequence</s213>
<s220>
<s223>Primer</s223>
</s220>
</s200>
<s400> 165
atcgcggctt at                                                         12</s400>
<s200>
<s210>166</s210>
<s211>12</s211>
<s212>DNA</s212>
<s213>Artificial Sequence</s213>
<s220>
<s223>Primer</s223>
</s220>
</s200>
<s400> 166
accatcaaac gg                                                         12</s400>
<s200>
<s210>167</s210>
<s211>12</s211>
<s212>DNA</s212>
<s213>Artificial Sequence</s213>
<s220>
<s223>Primer</s223>
</s220>
</s200>
<s400> 167
ggccgacttg gc                                                         12</s400>
<s200>
<s210>168</s210>
<s211>12</s211>
<s212>DNA</s212>
<s213>Artificial Sequence</s213>
<s220>
<s223>Primer</s223>
</s220>
</s200>
<s400> 168
gagtggcaac gt                                                         12</s400>
<s200>
<s210>169</s210>
<s211>12</s211>
<s212>DNA</s212>
<s213>Artificial Sequence</s213>
<s220>
<s223>Primer</s223>
</s220>
</s200>
<s400> 169
ccgcagggac ca                                                         12</s400>
<s200>
<s210>170</s210>
<s211>12</s211>
<s212>DNA</s212>
<s213>Artificial Sequence</s213>
<s220>
<s223>Primer</s223>
</s220>
</s200>
<s400> 170
ggtcaggaac aa                                                         12</s400>
<s200>
<s210>171</s210>
<s211>12</s211>
<s212>DNA</s212>
<s213>Artificial Sequence</s213>
<s220>
<s223>Primer</s223>
</s220>
</s200>
<s400> 171
gtcggtcgtg aa                                                         12</s400>
<s200>
<s210>172</s210>
<s211>12</s211>
<s212>DNA</s212>
<s213>Artificial Sequence</s213>
<s220>
<s223>Primer</s223>
</s220>
</s200>
<s400> 172
gtgcaatttg gc                                                         12</s400>
<s200>
<s210>173</s210>
<s211>12</s211>
<s212>DNA</s212>
<s213>Artificial Sequence</s213>
<s220>
<s223>Primer</s223>
</s220>
</s200>
<s400> 173
actcaccacg ca                                                         12</s400>
<s200>
<s210>174</s210>
<s211>12</s211>
<s212>DNA</s212>
<s213>Artificial Sequence</s213>
<s220>
<s223>Primer</s223>
</s220>
</s200>
<s400> 174
tggcttcatc ac                                                         12</s400>
<s200>
<s210>175</s210>
<s211>12</s211>
<s212>DNA</s212>
<s213>Artificial Sequence</s213>
<s220>
<s223>Primer</s223>
</s220>
</s200>
<s400> 175
ccgtggaatg ac                                                         12</s400>
<s200>
<s210>176</s210>
<s211>12</s211>
<s212>DNA</s212>
<s213>Artificial Sequence</s213>
<s220>
<s223>Primer</s223>
</s220>
</s200>
<s400> 176
ggaggatggc cc                                                         12</s400>
<s200>
<s210>177</s210>
<s211>12</s211>
<s212>DNA</s212>
<s213>Artificial Sequence</s213>
<s220>
<s223>Primer</s223>
</s220>
</s200>
<s400> 177
ccttggcatc gg                                                         12</s400>
<s200>
<s210>178</s210>
<s211>12</s211>
<s212>DNA</s212>
<s213>Artificial Sequence</s213>
<s220>
<s223>Primer</s223>
</s220>
</s200>
<s400> 178
gttagcccca at                                                         12</s400>
<s200>
<s210>179</s210>
<s211>12</s211>
<s212>DNA</s212>
<s213>Artificial Sequence</s213>
<s220>
<s223>Primer</s223>
</s220>
</s200>
<s400> 179
ggcatggcct tt                                                         12</s400>
<s200>
<s210>180</s210>
<s211>12</s211>
<s212>DNA</s212>
<s213>Artificial Sequence</s213>
<s220>
<s223>Primer</s223>
</s220>
</s200>
<s400> 180
ggtgacgatg ca                                                         12</s400>
<s200>
<s210>181</s210>
<s211>12</s211>
<s212>DNA</s212>
<s213>Artificial Sequence</s213>
<s220>
<s223>Primer</s223>
</s220>
</s200>
<s400> 181
actggccggc at                                                         12</s400>
<s200>
<s210>182</s210>
<s211>12</s211>
<s212>DNA</s212>
<s213>Artificial Sequence</s213>
<s220>
<s223>Primer</s223>
</s220>
</s200>
<s400> 182
aagctggggg ga                                                         12</s400>
<s200>
<s210>183</s210>
<s211>12</s211>
<s212>DNA</s212>
<s213>Artificial Sequence</s213>
<s220>
<s223>Primer</s223>
</s220>
</s200>
<s400> 183
atggctactg gc                                                         12</s400>
<s200>
<s210>184</s210>
<s211>12</s211>
<s212>DNA</s212>
<s213>Artificial Sequence</s213>
<s220>
<s223>Primer</s223>
</s220>
</s200>
<s400> 184
ggtgccggag ca                                                         12</s400>
<s200>
<s210>185</s210>
<s211>12</s211>
<s212>DNA</s212>
<s213>Artificial Sequence</s213>
<s220>
<s223>Primer</s223>
</s220>
</s200>
<s400> 185
ctcagcgata cg                                                         12</s400>
<s200>
<s210>186</s210>
<s211>12</s211>
<s212>DNA</s212>
<s213>Artificial Sequence</s213>
<s220>
<s223>Primer</s223>
</s220>
</s200>
<s400> 186
ggtggtggta tc                                                         12</s400>
<s200>
<s210>187</s210>
<s211>12</s211>
<s212>DNA</s212>
<s213>Artificial Sequence</s213>
<s220>
<s223>Primer</s223>
</s220>
</s200>
<s400> 187
gtcgacgcat ca                                                         12</s400>
<s200>
<s210>188</s210>
<s211>12</s211>
<s212>DNA</s212>
<s213>Artificial Sequence</s213>
<s220>
<s223>Primer</s223>
</s220>
</s200>
<s400> 188
gacggttcaa gc                                                         12</s400>
<s200>
<s210>189</s210>
<s211>12</s211>
<s212>DNA</s212>
<s213>Artificial Sequence</s213>
<s220>
<s223>Primer</s223>
</s220>
</s200>
<s400> 189
aagctgtggg ct                                                         12</s400>
<s200>
<s210>190</s210>
<s211>12</s211>
<s212>DNA</s212>
<s213>Artificial Sequence</s213>
<s220>
<s223>Primer</s223>
</s220>
</s200>
<s400> 190
gcggaggaac ca                                                         12</s400>
<s200>
<s210>191</s210>
<s211>12</s211>
<s212>DNA</s212>
<s213>Artificial Sequence</s213>
<s220>
<s223>Primer</s223>
</s220>
</s200>
<s400> 191
ccaggaggtg gt                                                         12</s400>
<s200>
<s210>192</s210>
<s211>12</s211>
<s212>DNA</s212>
<s213>Artificial Sequence</s213>
<s220>
<s223>Primer</s223>
</s220>
</s200>
<s400> 192
agcgcggcaa aa                                                         12</s400>
<s200>
<s210>193</s210>
<s211>12</s211>
<s212>DNA</s212>
<s213>Artificial Sequence</s213>
<s220>
<s223>Primer</s223>
</s220>
</s200>
<s400> 193
accactcccg ca                                                         12</s400>
<s200>
<s210>194</s210>
<s211>12</s211>
<s212>DNA</s212>
<s213>Artificial Sequence</s213>
<s220>
<s223>Primer</s223>
</s220>
</s200>
<s400> 194
ggcggcacag ga                                                         12</s400>
<s200>
<s210>195</s210>
<s211>12</s211>
<s212>DNA</s212>
<s213>Artificial Sequence</s213>
<s220>
<s223>Primer</s223>
</s220>
</s200>
<s400> 195
gccccgttag ca                                                         12</s400>
<s200>
<s210>196</s210>
<s211>12</s211>
<s212>DNA</s212>
<s213>Artificial Sequence</s213>
<s220>
<s223>Primer</s223>
</s220>
</s200>
<s400> 196
aaggcgcgaa cg                                                         12</s400>
<s200>
<s210>197</s210>
<s211>12</s211>
<s212>DNA</s212>
<s213>Artificial Sequence</s213>
<s220>
<s223>Primer</s223>
</s220>
</s200>
<s400> 197
ggtgactggt gg                                                         12</s400>
<s200>
<s210>198</s210>
<s211>12</s211>
<s212>DNA</s212>
<s213>Artificial Sequence</s213>
<s220>
<s223>Primer</s223>
</s220>
</s200>
<s400> 198
cgcagccgag at                                                         12</s400>
<s200>
<s210>199</s210>
<s211>12</s211>
<s212>DNA</s212>
<s213>Artificial Sequence</s213>
<s220>
<s223>Primer</s223>
</s220>
</s200>
<s400> 199
aagaagcagg cg                                                         12</s400>
<s200>
<s210>200</s210>
<s211>12</s211>
<s212>DNA</s212>
<s213>Artificial Sequence</s213>
<s220>
<s223>Primer</s223>
</s220>
</s200>
<s400> 200
gtgtggaagc ca                                                         12</s400>
<s200>
<s210>201</s210>
<s211>12</s211>
<s212>DNA</s212>
<s213>Artificial Sequence</s213>
<s220>
<s223>Primer</s223>
</s220>
</s200>
<s400> 201
gatggatttg gg                                                         12</s400>
<s200>
<s210>202</s210>
<s211>12</s211>
<s212>DNA</s212>
<s213>Artificial Sequence</s213>
<s220>
<s223>Primer</s223>
</s220>
</s200>
<s400> 202
ggtcaggcac ca                                                         12</s400>
<s200>
<s210>203</s210>
<s211>12</s211>
<s212>DNA</s212>
<s213>Artificial Sequence</s213>
<s220>
<s223>Primer</s223>
</s220>
</s200>
<s400> 203
tgcctcgcac ca                                                         12</s400>
<s200>
<s210>204</s210>
<s211>12</s211>
<s212>DNA</s212>
<s213>Artificial Sequence</s213>
<s220>
<s223>Primer</s223>
</s220>
</s200>
<s400> 204
agcagcgcct ca                                                         12</s400>
<s200>
<s210>205</s210>
<s211>12</s211>
<s212>DNA</s212>
<s213>Artificial Sequence</s213>
<s220>
<s223>Primer</s223>
</s220>
</s200>
<s400> 205
gccagctgta cg                                                         12</s400>
<s200>
<s210>206</s210>
<s211>12</s211>
<s212>DNA</s212>
<s213>Artificial Sequence</s213>
<s220>
<s223>Primer</s223>
</s220>
</s200>
<s400> 206
ggaggtcgac ca                                                         12</s400>
<s200>
<s210>207</s210>
<s211>12</s211>
<s212>DNA</s212>
<s213>Artificial Sequence</s213>
<s220>
<s223>Primer</s223>
</s220>
</s200>
<s400> 207
ttcggacgaa ta                                                         12</s400>
<s200>
<s210>208</s210>
<s211>12</s211>
<s212>DNA</s212>
<s213>Artificial Sequence</s213>
<s220>
<s223>Primer</s223>
</s220>
</s200>
<s400> 208
ggagagcgga cg                                                         12</s400>
<s200>
<s210>209</s210>
<s211>12</s211>
<s212>DNA</s212>
<s213>Artificial Sequence</s213>
<s220>
<s223>Primer</s223>
</s220>
</s200>
<s400> 209
ggcgattctg ca                                                         12</s400>
<s200>
<s210>210</s210>
<s211>12</s211>
<s212>DNA</s212>
<s213>Artificial Sequence</s213>
<s220>
<s223>Primer</s223>
</s220>
</s200>
<s400> 210
gtgggtggac aa                                                         12</s400>
<s200>
<s210>211</s210>
<s211>12</s211>
<s212>DNA</s212>
<s213>Artificial Sequence</s213>
<s220>
<s223>Primer</s223>
</s220>
</s200>
<s400> 211
gtgcacgtat gg                                                         12</s400>
<s200>
<s210>212</s210>
<s211>12</s211>
<s212>DNA</s212>
<s213>Artificial Sequence</s213>
<s220>
<s223>Primer</s223>
</s220>
</s200>
<s400> 212
cgacgacgac ga                                                         12</s400>
<s200>
<s210>213</s210>
<s211>12</s211>
<s212>DNA</s212>
<s213>Artificial Sequence</s213>
<s220>
<s223>Primer</s223>
</s220>
</s200>
<s400> 213
actggccgag gg                                                         12</s400>
<s200>
<s210>214</s210>
<s211>12</s211>
<s212>DNA</s212>
<s213>Artificial Sequence</s213>
<s220>
<s223>Primer</s223>
</s220>
</s200>
<s400> 214
gcggtcagca ca                                                         12</s400>
<s200>
<s210>215</s210>
<s211>12</s211>
<s212>DNA</s212>
<s213>Artificial Sequence</s213>
<s220>
<s223>Primer</s223>
</s220>
</s200>
<s400> 215
atcagcgcac ca                                                         12</s400>
<s200>
<s210>216</s210>
<s211>12</s211>
<s212>DNA</s212>
<s213>Artificial Sequence</s213>
<s220>
<s223>Primer</s223>
</s220>
</s200>
<s400> 216
atggccggtg gg                                                         12</s400>
</sequence-list-new-rules>
</sequence-cwu>
</div>
</div>
</div>
</section><section itemprop="claims" itemscope="">
<h2>Claims (<span itemprop="count">22</span>)</h2>
<div html="" itemprop="content"><div class="claims" lang="EN" load-source="patent-office" mxw-id="PCLM28699464">
<claim-statement>What is claimed is: </claim-statement>
<div class="claim"> <div class="claim" id="US-6287769-B1-CLM-00001" num="1">
<div class="claim-text">1. A method of amplifying DNA fragments in a population of DNA molecules obtained from a source, the method comprising:</div>
<div class="claim-text">preparing a plurality of primers selected to have proper amplification probabilities, as assessed using an electrophoretic pattern produced by amplification of DNA obtained from a similar source; and </div>
<div class="claim-text">simultaneously applying a polymerase chain reaction (PCR) method to a plurality of different DNAs with each of said plurality of primers, thereby amplifying fragments of said plurality of different DNAs. </div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="US-6287769-B1-CLM-00002" num="2">
<div class="claim-text">2. The method of claim <b>1</b>, further comprising employing a reference primer having a known sequence and applying said PCR method to a reference DNA having a sequence complementary to the sequence of said reference primer, thereby amplifying a fragment of said reference DNA simultaneously with amplifying said fragments of said plurality of different DNAs.</div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="US-6287769-B1-CLM-00003" num="3">
<div class="claim-text">3. The method of claim <b>1</b>, further comprising classifying said fragments amplified from said plurality of different DNAs by a discrimination method.</div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="US-6287769-B1-CLM-00004" num="4">
<div class="claim-text">4. The method of claim <b>3</b>, wherein said discrimination method is electrophoresis.</div>
</div>
</div> <div class="claim"> <div class="claim" id="US-6287769-B1-CLM-00005" num="5">
<div class="claim-text">5. An apparatus for amplifying DNA fragments, comprising:</div>
<div class="claim-text">a body having a plurality of wells; and </div>
<div class="claim-text">a plurality of primers having proper amplification probabilities individually arranged in said plurality of wells. </div>
</div>
</div> <div class="claim"> <div class="claim" id="US-6287769-B1-CLM-00006" num="6">
<div class="claim-text">6. A method of assaying a group of microorganisms obtained from a source, the method comprising:</div>
<div class="claim-text">preparing a plurality of primers selected to have proper amplification probabilities, as assessed using an electrophoretic pattern produced by amplification of DNA obtained from a similar source; </div>
<div class="claim-text">simultaneously applying a PCR method to DNA of a plurality of different microorganisms with each of said plurality of primers, thereby amplifying fragments of said DNA of said microorganisms; and </div>
<div class="claim-text">classifying said amplified fragments by a discrimination method for discriminating a plurality of different microorganisms included in said group of microorganisms. </div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="US-6287769-B1-CLM-00007" num="7">
<div class="claim-text">7. The method of claim <b>6</b>, further comprising:</div>
<div class="claim-text">employing a reference primer having a known sequence and applying said PCR method to reference DNA having a sequence complementary to the sequence of said reference primer, thereby amplifying a reference fragment of said reference DNA simultanously with amplifying said fragments of said DNA of said plurality of different microorganisms; </div>
<div class="claim-text">classifying said reference fragment along with said fragments amplified from said DNA of said plurality of different microorganisms by said discrimination method; </div>
<div class="claim-text">obtaining amplification efficiency for said reference DNA on the basis of the result of classification of said reference fragment; and </div>
<div class="claim-text">correcting the results of classification of said fragments amplified from said DNA of said plurality of different microorganisms on the basis of said amplification efficiency. </div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="US-6287769-B1-CLM-00008" num="8">
<div class="claim-text">8. The method of claim <b>7</b>, wherein said discrimination method is electrophoresis.</div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="US-6287769-B1-CLM-00009" num="9">
<div class="claim-text">9. The method of claim <b>8</b>, further comprising:</div>
<div class="claim-text">employing a DNA size marker with said fragments amplified from said DNA of said plurality of different microorganisms for said electrophoresis; </div>
<div class="claim-text">staining an electrophoretic pattern obtained by said electrophoresis; and </div>
<div class="claim-text">correcting the gradient of said electrophoretic pattern based on the luminous intensity of said DNA size marker in said stained electrophoretic pattern. </div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="US-6287769-B1-CLM-00010" num="10">
<div class="claim-text">10. The method of claim <b>9</b>, further comprising:</div>
<div class="claim-text">setting a threshold based on the luminous intensity of said DNA size marker in said electrophoretic pattern; and </div>
<div class="claim-text">analyzing said group of microorganisms on the basis of a band having a luminous intensity exceeding said threshold in said electrophoretic pattern. </div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="US-6287769-B1-CLM-00011" num="11">
<div class="claim-text">11. The method of claim <b>6</b>, further comprising:</div>
<div class="claim-text">isolating a bacterium; </div>
<div class="claim-text">applying said PCR method to said isolated bacterium with each of said plurality of primers, thereby amplifying a bacterial fragment of DNA from said bacterium; </div>
<div class="claim-text">classifying said bacterial fragment by said discrimination method; and </div>
<div class="claim-text">analyzing the results of discrimination of said amplified fragments from said DNA of said plurality of different microorganisms based on the classification of said bacterial fragment. </div>
</div>
</div> <div class="claim"> <div class="claim" id="US-6287769-B1-CLM-00012" num="12">
<div class="claim-text">12. A method of analyzing groups of microorganisms obtained from a source, the method comprising:</div>
<div class="claim-text">preparing a plurality of primers selected to have proper amplification probabilities as assessed by an electrophoretic pattern produced by amplification of DNA obtained from a similar source; </div>
<div class="claim-text">simultaneously applying a PCR method to DNA of a plurality of different microorganisms included in a first group of microorganisms with each of said plurality of primers, thereby amplifying fragments of said first group; </div>
<div class="claim-text">classifying said amplified DNA fragments of said first group by a discrimination method; </div>
<div class="claim-text">simultaneously applying said PCR method to DNA of a plurality of different microorganisms included in a second group of microorganisms with each of said plurality of primers, thereby amplifying fragments of said DNA of said second group; </div>
<div class="claim-text">classifying said amplified DNA fragments of said second group by said discrimination method; and </div>
<div class="claim-text">comparing the results of classification of said first group of microorganisms with those of said second group of microorganisms. </div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="US-6287769-B1-CLM-00013" num="13">
<div class="claim-text">13. The method of claim <b>12</b>, wherein said discrimination method is electrophoresis.</div>
</div>
</div> <div class="claim"> <div class="claim" id="US-6287769-B1-CLM-00014" num="14">
<div class="claim-text">14. A method of analyzing groups of microorganisms obtained from a source, the method comprising:</div>
<div class="claim-text">sampling a group of microorganisms at a plurality of points of time; </div>
<div class="claim-text">simultaneously applying a PCR method to DNA of a plurality of different microorganisms included in said group with each of a plurality of primers, wherein the primers are selected to have proper amplification probabilities, as assessed using an electrophoretic pattern produced by amplification of DNA obtained from a similar source, thereby amplifying fragments of said DNA of said group; </div>
<div class="claim-text">classifying said amplified DNA fragments by a discrimination method; and </div>
<div class="claim-text">analyzing time change of the state of said group of microorganisms on the basis of the results of classification at said plurality of points of time. </div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="US-6287769-B1-CLM-00015" num="15">
<div class="claim-text">15. The method of claim <b>14</b>, wherein said discrimination method is electrophoresis.</div>
</div>
</div> <div class="claim"> <div class="claim" id="US-6287769-B1-CLM-00016" num="16">
<div class="claim-text">16. A method of assaying a contaminant material obtained from a first source, the method comprising:</div>
<div class="claim-text">applying a PCR method to DNA of a microorganism related to said contaminant material with each of a plurality of primers, wherein the primers are selected to have proper amplification probabilities, as assessed using an electrophoretic pattern produced by amplification of DNA obtained from a similar source, thereby amplifying a DNA fragment of said contaminant material; </div>
<div class="claim-text">classifying said DNA fragment by a discrimination method; </div>
<div class="claim-text">preserving the relation between the type of said microorganism and the result of classification in a database; </div>
<div class="claim-text">simultaneously applying said PCR method to DNA of a plurality of different microorganisms obtained from the first source with each of said plurality of primers thereby amplifying first fragments of said DNA of said plurality of different microorganisms; </div>
<div class="claim-text">classifying said first fragments amplified by said discrimination method; and </div>
<div class="claim-text">retrieving the types of said plurality of different microorganisms from said database on the basis of the results of classification of said first fragments. </div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="US-6287769-B1-CLM-00017" num="17">
<div class="claim-text">17. The method of claim <b>16</b>, further comprising determining presence/absence of said contaminant material obtained from said similar source on the basis of the results of retrieval of said database.</div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="US-6287769-B1-CLM-00018" num="18">
<div class="claim-text">18. The method of claim <b>16</b>, further comprising determining the quantity of said contaminant material present in said similar source on the basis of the results of retrieval of said database.</div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="US-6287769-B1-CLM-00019" num="19">
<div class="claim-text">19. The method of claim <b>16</b>, wherein said database preserves a plurality of types of microorganisms and results of classification corresponding thereto.</div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="US-6287769-B1-CLM-00020" num="20">
<div class="claim-text">20. The method of claim <b>16</b>, wherein</div>
<div class="claim-text">said discrimination method is electrophoresis, </div>
<div class="claim-text">said results of classification are band patterns produced from an electrophoretic pattern, and </div>
<div class="claim-text">said database preserves the relation between the type of microorganism and band patterns produced from said electrophoretic pattern. </div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="US-6287769-B1-CLM-00021" num="21">
<div class="claim-text">21. The method of claim <b>6</b>, wherein said discrimination method is electrophoresis, the method further comprising:</div>
<div class="claim-text">staining an electrophoretic pattern obtained by said electrophoresis; and </div>
<div class="claim-text">correcting the gradient of said electrophoretic pattern based on the luminous intensity of said reference fragment in said stained electrophoretic pattern. </div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="US-6287769-B1-CLM-00022" num="22">
<div class="claim-text">22. The method of claim <b>21</b>, further comprising:</div>
<div class="claim-text">setting a threshold based on the luminous intensity of said reference fragment in said electrophoretic pattern; and </div>
<div class="claim-text">analyzing said group of microorganisms on the basis of a band having a luminous intensity exceeding said threshold in said electrophoretic pattern.</div>
</div>
</div> </div>
</div>
</section>
                </article>
            </search-app>
        </body>
    </html>
    