
    <html>
        <body>
            <search-app>
                <article class="result" itemscope="" itemtype="http://schema.org/ScholarlyArticle">
    <h1 itemprop="pageTitle">US9771565B2 - Thermostable reverse transcriptases and uses thereof 
        - Google Patents</h1><section itemprop="abstract" itemscope="">
<h2>Abstract</h2>
<div html="" itemprop="content"><abstract lang="EN" load-source="patent-office" mxw-id="PA205597348">
<div class="abstract" id="p-0001" num="0000">The present invention is in the fields of molecular and cellular biology. The invention is generally related to reverse transcriptase enzymes and methods for the reverse transcription of nucleic acid molecules, especially messenger RNA molecules. Specifically, the invention relates to reverse transcriptase enzymes which have been mutated or modified to increase thermostability, decrease terminal deoxynucleotidyl transferase activity, and/or increase fidelity, and to methods of producing, amplifying or sequencing nucleic acid molecules (particularly cDNA molecules) using these reverse transcriptase enzymes or compositions. The invention also relates to nucleic acid molecules produced by these methods and to the use of such nucleic acid molecules to produce desired polypeptides. The invention also concerns kits comprising such enzymes or compositions.</div>
</abstract>
</div>
</section><section itemprop="description" itemscope="">
<h2>Description</h2>
<div html="" itemprop="content"><div class="description" lang="EN" load-source="patent-office" mxw-id="PDES120451687">
<heading id="h-0001">CROSS-REFERENCE TO RELATED APPLICATIONS</heading>
<div class="description-paragraph" id="p-0002" num="0001">The present application is a continuation-in-part of U.S. patent application Ser. No. 11/492,194 filed Jul. 25, 2006, which is a continuation of U.S. patent application Ser. No. 10/661,819, filed Sep. 15, 2003 (abandoned) which claims priority from Provisional Application No. 60/410,283, filed Sep. 13, 2002, and is also a continuation-in-part of U.S. patent application Ser. No. 11/437,681, filed May 22, 2006 (abandoned), which is a continuation of U.S. application Ser. No. 09/845,157, filed May 1, 2001, now U.S. Pat. No. 7,078,208, which claims priority from Provisional Application No. 60/207,196, filed May 26, 2000, the entire disclosures of which are incorporated herein by reference.</div>
<heading id="h-0002">BACKGROUND OF THE INVENTION</heading>
<div class="description-paragraph" id="p-0003" num="0002">Field of the Invention</div>
<div class="description-paragraph" id="p-0004" num="0003">The present invention is in the fields of molecular and cellular biology. The invention is generally related to reverse transcriptase enzymes and methods for the reverse transcription of nucleic acid molecules, especially messenger RNA molecules. Specifically, the invention relates to reverse transcriptase enzymes which have been mutated or modified to increase thermostability, decrease terminal deoxynucleotidyl transferase activity, and/or increase fidelity, and to methods of producing, amplifying or sequencing nucleic acid molecules (particularly cDNA molecules) using these reverse transcriptase enzymes or compositions. The invention also relates to nucleic acid molecules produced by these methods and to the use of such nucleic acid molecules to produce desired polypeptides. The invention also relates to nucleic acid molecules encoding the reverse transcriptases of the invention, to vectors containing such nucleic acid molecules, and to host cells containing such nucleic acid molecules. The invention also concerns kits or compositions comprising such enzymes.</div>
<div class="description-paragraph" id="p-0005" num="0004">Related Art</div>
<div class="description-paragraph" id="h-0003" num="0000">cDNA and cDNA Libraries</div>
<div class="description-paragraph" id="p-0006" num="0005">In examining the structure and physiology of an organism, tissue or cell, it is often desirable to determine its genetic content. The genetic framework of an organism is encoded in the double-stranded sequence of nucleotide bases in the deoxyribonucleic acid (DNA) which is contained in the somatic and germ cells of the organism. The genetic content of a particular segment of DNA, or gene, is typically manifested upon production of the protein which the gene encodes. In order to produce a protein, a complementary copy of one strand of the DNA double helix is produced by RNA polymerase enzymes, resulting in a specific sequence of ribonucleic acid (RNA). This particular type of RNA, since it contains the genetic message from the DNA for production of a protein, is called messenger RNA (mRNA).</div>
<div class="description-paragraph" id="p-0007" num="0006">Within a given cell, tissue or organism, there exist myriad mRNA species, each encoding a separate and specific protein. This fact provides a powerful tool to investigators interested in studying genetic expression in a tissue or cell. mRNA molecules may be isolated and further manipulated by various molecular biological techniques, thereby allowing the elucidation of the full functional genetic content of a cell, tissue or organism.</div>
<div class="description-paragraph" id="p-0008" num="0007">One common approach to the study of gene expression is the production of complementary DNA (cDNA) clones. In this technique, the mRNA molecules from an organism are isolated from an extract of the cells or tissues of the organism. This isolation often employs solid chromatography matrices, such as cellulose or agarose, to which oligomers of thymidine (T) have been complexed. Since the 3′ termini on most eukaryotic mRNA molecules contain a string of adenosine (A) bases, and since A base pairs with T, the mRNA molecules can be rapidly purified from other molecules and substances in the tissue or cell extract. From these purified mRNA molecules, cDNA copies may be made using the enzyme reverse transcriptase (RT), which results in the production of single-stranded cDNA molecules. This reaction is typically referred to as the first strand reaction. The single-stranded cDNAs may then be converted into a complete double-stranded DNA copy (i.e., a double-stranded cDNA) of the original mRNA (and thus of the original double-stranded DNA sequence, encoding this mRNA, contained in the genome of the organism) by the action of a DNA polymerase. The protein-specific double-stranded cDNAs can then be inserted into a plasmid or viral vector, which is then introduced into a host bacterial, yeast, animal or plant cell. The host cells are then grown in culture media, resulting in a population of host cells containing (or in many cases, expressing) the gene of interest.</div>
<div class="description-paragraph" id="p-0009" num="0008">This entire process, from isolation of mRNA from a source organism or tissue to insertion of the cDNA into a plasmid or vector to growth of host cell populations containing the isolated gene, is termed “cDNA cloning.” The set of cDNAs prepared from a given source of mRNAs is called a “cDNA library.” The cDNA clones in a cDNA library correspond to the genes transcribed in the source tissue. Analysis of a cDNA library can yield much information on the pattern of gene expression in the organism or tissue from which it was derived.</div>
<div class="description-paragraph" id="h-0004" num="0000">Retroviral Reverse Transcriptase Enzymes</div>
<div class="description-paragraph" id="p-0010" num="0009">Three prototypical forms of retroviral reverse transcriptase have been studied thoroughly. Moloney Murine Leukemia Virus (M-MLV) reverse transcriptase contains a single subunit of 78 kDa with RNA-dependent DNA polymerase and RNase H activity. This enzyme has been cloned and expressed in a fully active form in <i>E. coli </i>(reviewed in Prasad, V. R., <i>Reverse Transcriptase</i>, Cold Spring Harbor, N.Y.: Cold Spring Harbor Laboratory Press, p. 135 (1993)). Human Immunodeficiency Virus (HIV) reverse transcriptase is a heterodimer of p66 and p51 subunits in which the smaller subunit is derived from the larger by proteolytic cleavage. The p66 subunit has both a RNA-dependent DNA polymerase and an RNase H domain, while the p51 subunit has only a DNA polymerase domain. Active HIV p66/p51 reverse transcriptase has been cloned and expressed successfully in a number of expression hosts, including <i>E. coli </i>(reviewed in Le Grice, S. F. J., <i>Reverse Transcriptase</i>, Cold Spring Harbor, N.Y.: Cold Spring Harbor Laboratory press, p. 163 (1993)). Within the HIV p66/p51 heterodimer, the 51-kD subunit is catalytically inactive, and the 66-kD subunit has both DNA polymerase and RNase H activity (Le Grice, S. F. J., et al., <i>EMBO Journal </i>10:3905 (1991); Hostomsky, Z., et al., <i>J. Virol. </i>66:3179 (1992)). Avian Sarcoma-Leukosis Virus (ASLV) reverse transcriptase, which includes but is not limited to Rous Sarcoma Virus (RSV) reverse transcriptase, Avian Myeloblastosis Virus (AMV) reverse transcriptase, Avian Erythroblastosis Virus (AEV) Helper Virus MCAV reverse transcriptase, Avian Myelocytomatosis Virus MC29 Helper Virus MCAV reverse transcriptase, Avian Reticuloendotheliosis Virus (REV-T) Helper Virus REV-A reverse transcriptase, Avian Sarcoma Virus UR2 Helper Virus UR2AV reverse transcriptase, Avian Sarcoma Virus Y73 Helper Virus YAV reverse transcriptase, Rous Associated Virus (RAV) reverse transcriptase, and Myeloblastosis Associated Virus (MAV) reverse transcriptase, is also a heterodimer of two subunits, α (approximately 62 kDa) and 13 (approximately 94 kDa), in which α is derived from β by proteolytic cleavage (reviewed in Prasad, V. R., <i>Reverse Transcriptase</i>, Cold Spring Harbor, N.Y.: Cold Spring Harbor Laboratory Press (1993), p. 135). ASLV reverse transcriptase can exist in two additional catalytically active structural forms, ββ and α (Hizi, A. and Joklik, W. K., <i>J. Biol. Chem. </i>252: 2281 (1977)). Sedimentation analysis suggests αβ and ββ are dimers and that the α form exists in an equilibrium between monomeric and dimeric forms (Grandgenett, D. P., et al., <i>Proc. Nat. Acad. Sci. USA </i>70:230 (1973); Hizi, A. and Joklik, W. K., <i>J. Biol. Chem. </i>252:2281 (1977); and Soltis, D. A. and Skalka, A. M., <i>Proc. Nat. Acad. Sci. USA </i>85:3372 (1988)). The ASLV αβ and ββ reverse transcriptases are the only known examples of retroviral reverse transcriptase that include three different activities in the same protein complex: DNA polymerase, RNase H, and DNA endonuclease (integrase) activities (reviewed in Skalka, A. M., <i>Reverse Transcriptase</i>, Cold Spring Harbor, N.Y.: Cold Spring Harbor Laboratory Press (1993), p. 193). The α form lacks the integrase domain and activity.</div>
<div class="description-paragraph" id="p-0011" num="0010">Various forms of the individual subunits of ASLV reverse transcriptase have been cloned and expressed. These include a 98-kDa precursor polypeptide that is normally processed proteolytically to β and a 4 kDa polypeptide removed from the β carboxy end (Alexander, F., et al., <i>J. Virol. </i>61:534 (1987) and Anderson, D. et al., <i>Focus </i>17:53 (1995)), and the mature β subunit (Weis, J. H. and Salstrom, J. S., U.S. Pat. No. 4,663,290 (1987); and Soltis, D. A. and Skalka, A. M., <i>Proc. Nat. Acad. Sci. USA </i>85:3372 (1988)). (See also Werner S, and Wohrl B. M., <i>Eur. J. Biochem. </i>267:4740-4744 (2000); Werner S, and Wohrl B. M., <i>J. Virol. </i>74:3245-3252 (2000); Werner S, and Wohrl B. M., <i>J. Biol. Chem. </i>274:26329-26336 (1999).) Heterodimeric RSV αβ reverse transcriptase has also been purified from <i>E. coli </i>cells expressing a cloned RSV β gene (Chernov, A. P., et al., <i>Biomed. Sci. </i>2:49 (1991)).</div>
<div class="description-paragraph" id="h-0005" num="0000">Reverse Transcription Efficiency</div>
<div class="description-paragraph" id="p-0012" num="0011">As noted above, the conversion of mRNA into cDNA by reverse transcriptase-mediated reverse transcription is an essential step in the study of proteins expressed from cloned genes. However, the use of unmodified reverse transcriptase to catalyze reverse transcription is inefficient for a number of reasons. First, reverse transcriptase sometimes degrades an RNA template before the first strand reaction is initiated or completed, primarily due to the intrinsic RNase H activity present in reverse transcriptase. In addition, mis-priming of the mRNA template molecule can lead to the introduction of errors in the cDNA first strand while secondary structure of the mRNA molecule itself may make some mRNAs refractory to first strand synthesis.</div>
<div class="description-paragraph" id="p-0013" num="0012">Removal of the RNase H activity of reverse transcriptase can eliminate the first problem and improve the efficiency of reverse transcription (Gerard, G. F., et al., <i>FOCUS </i>11(4):60 (1989); Gerard, G. F., et al., <i>FOCUS </i>14(3):91 (1992)). However such reverse transcriptases (“RNase H-” forms) do not address the additional problems of mis-priming and mRNA secondary structure.</div>
<div class="description-paragraph" id="p-0014" num="0013">Another factor which influences the efficiency of reverse transcription is the ability of RNA to form secondary structures. Such secondary structures can form, for example, when regions of RNA molecules have sufficient complementarity to hybridize and form double stranded RNA. Generally, the formation of RNA secondary structures can be reduced by raising the temperature of solutions which contain the RNA molecules. Thus, in many instances, it is desirable to reverse transcribe RNA at temperatures above 37° C. However, art known reverse transcriptases generally lose activity when incubated at temperatures much above 37° C. (e.g., 50° C.).</div>
<heading id="h-0006">SUMMARY OF THE INVENTION</heading>
<div class="description-paragraph" id="p-0015" num="0014">The present invention provides, in part, reverse transcriptase enzymes, compositions comprising such enzymes and methods useful in overcoming limitations of reverse transcription discussed above. In general, the invention provides compositions for use in reverse transcription of a nucleic acid molecule, these compositions comprising one or more (e.g., one, two, three, four, five, ten, fifteen, etc.) polypeptides having at least one reverse transcriptase activity. Such compositions may further comprise one or more (e.g., one, two, three, four, five, etc.) nucleotides (e.g. one or more fluorescentl-labeled nucleotides, one or more radiolabeled nucleotides, etc.), a suitable buffer, and/or one or more (e.g., one, two, three, four, five, ten, fifteen, etc.) DNA polymerases. Compositions of the invention may also comprise one or more (e.g., one, two, three, four, five, ten, fifteen, etc.) oligonucleotide primers, and/or one or more templates, and/or one or more nucleic acid molecules (which may be complementary to all or a portion of such templates).</div>
<div class="description-paragraph" id="p-0016" num="0015">Reverse transcriptases of the invention are preferably modified or mutated such that the thermostability of the enzyme is increased or enhanced and/or the fidelity of the enzyme is increased or enhanced. In specific embodiments, reverse transcriptases of the invention may be single chained (single subunit) or multi-chained (multi-subunit) and may be reduced or substantially reduced in RNase H activity or may have no detectable RNase H activity or may be lacking in RNase H activity. Preferably enzymes of the invention are enzymes selected from the group consisting of Moloney Murine Leukemia Virus (M-MLV) RNase H− reverse transcriptase, Rous Sarcoma Virus (RSV) RNase H− reverse transcriptase, Avian Myeloblastosis Virus (AMV) RNase H− reverse transcriptase, Rous Associated Virus (RAV) RNase H− reverse transcriptase, Myeloblastosis Associated Virus (MAV) RNase H− reverse transcriptase or other ASLV RNase H− reverse transcriptases and Human Immunodeficiency Virus (HIV) RNase H− reverse transcriptase and mutants thereof. In preferred compositions, the reverse transcriptases are present at working concentrations.</div>
<div class="description-paragraph" id="p-0017" num="0016">In certain aspects, the invention includes reverse transcriptases which have been modified or mutated to increase or enhance thermostability. Examples of such reverse transcriptases include enzymes comprising one or more modifications or mutations at positions corresponding to amino acids selected from the group consisting of:</div>
<div class="description-paragraph" id="p-0018" num="0017">(a) leucine 52 of M-MLV reverse transcriptase;</div>
<div class="description-paragraph" id="p-0019" num="0018">(b) tyrosine 64 of M-MLV reverse transcriptase;</div>
<div class="description-paragraph" id="p-0020" num="0019">(c) lysine 152 of M-MLV reverse transcriptase;</div>
<div class="description-paragraph" id="p-0021" num="0020">(d) histidine 204 of M-MLV reverse transcriptase;</div>
<div class="description-paragraph" id="p-0022" num="0021">(e) methionine 289 of M-MLV reverse transcriptase;</div>
<div class="description-paragraph" id="p-0023" num="0022">(f) threonine 306 of M-MLV reverse transcriptase; and</div>
<div class="description-paragraph" id="p-0024" num="0023">(g) phenylalanine 309 of M-MLV reverse transcriptase.</div>
<div class="description-paragraph" id="h-0007" num="0000">In some embodiments, a modification or mutation may be the addition of an N- and/or C-terminal tag sequence.</div>
<div class="description-paragraph" id="p-0025" num="0024">In specific embodiments, the invention is directed to M-MLV reverse transcriptases wherein leucine 52 is replaced with proline, tyrosine 64 is replaced with arginine, lysine 152 is replaced with methionine, histidine 204 is replaced with arginine, methionine 289 is replaced with leucine, threonine 306 is replaced with either lysine or arginine, and/or phenylalanine 309 is replaced with asparagine or serine. Further included within the scope of the invention are reverse transcriptases, other than M-MLV reverse transcriptase, which contain alterations corresponding to those set out above.</div>
<div class="description-paragraph" id="p-0026" num="0025">In additional aspects, the invention also include thermostable reverse transcriptases which retain at least about 50%, at least about 60%, at least about 70%, at least about 85%, at least about 95%, at least about 97%, at least about 98%, at least about 99%, at least about 100%, at least about 150%, at least about 200%, at least about 250%, or at least about 300% of reverse transcriptase activity after heating to 50° C. for 5 minutes.</div>
<div class="description-paragraph" id="p-0027" num="0026">As noted above, enzymes of the invention include reverse transcriptases which exhibit reverse transcriptase activity either upon the formation of multimers (e.g., dimers) or as individual protein molecules (i.e., in monomeric form). Examples of reverse transcriptases which exhibit reverse transcriptase activity upon the formation of multimers include AMV, RSV and HIV reverse transcriptases. One example of a reverse transcriptase which exhibits reverse transcriptase activity as separate, individual proteins (i.e., in monomeric form) is M-MLV reverse transcriptase.</div>
<div class="description-paragraph" id="p-0028" num="0027">Multimeric reverse transcriptases of the invention may form homo-multimers or hetero-multimers. In other words, the subunits of the multimeric protein complex may be identical or different. One example of a hetero-dimeric reverse transcriptase is AMV reverse transcriptase, which is composed of two subunits that differ in primary amino acid sequence. More specifically, as already discussed, AMV reverse transcriptase may be composed of two subunits wherein one of these subunits is generated by proteolytic processing of the other. Thus, dimeric AMV reverse transcriptase may be composed of subunits of differing size which share regions of amino acid sequence identity.</div>
<div class="description-paragraph" id="p-0029" num="0028">The present invention relates in particular to mutant or modified reverse transcriptases wherein one or more (e.g., one, two, three, four, five, ten, twelve, fifteen, twenty, etc.) amino acid changes have been made which renders the enzyme more thermostable in nucleic acid synthesis, as compared to the unmutated or unmodified reverse transcriptases. Sites for mutation or modification to produce the thermostable reverse transcriptase enzymes of the present invention and/or reverse transcriptases which exhibit other characteristics (e.g., increased fidelity, decreased TdT activity, etc.) are listed for some reverse transcriptases in Table 1. As will be appreciated by those skilled in the art, one or more of the amino acids identified may be deleted and/or replaced with one or a number of amino acid residues. In a preferred aspect, any one or more of the amino acids identified in Table 1 may be substituted with any one or more amino acid residues such as Ala, Arg, Asn, Asp, Cys, Gln, Glu, Gly, His, Ile, Leu, Lys, Met, Phe, Pro, Ser, Thr, Trp, Tyr, and/or Val. The modifications described in Table 1 preferably produce thermostable reverse transcriptases of the invention. Similar or equivalent sites or corresponding sites in other reverse transcriptases can be mutated or modified to produce additional thermostable reverse transcriptases, as well as reverse transcriptases which exhibit other characteristics (e.g., increased fidelity, decreased TdT activity, etc.). Thus, a reverse transcriptase of the present invention may have one or more of the following properties: (a) increased thermostability or increased half-life at elevated temperatures; (b) reduced, substantially reduced, or no detectable RNase H activity, (c) reduced or substantially reduced terminal deoxynucleotidyl transferase activity, and/or (d) increased fidelity. In some embodiments, a reverse transcriptase of the invention may have a plurality of the properties listed above (e.g., a reverse transcriptase may have enhanced thermostability, reduced RNase H activity, and enhanced fidelity).</div>
<div class="description-paragraph" id="p-0030" num="0029">
<tables id="TABLE-US-00001" num="00001">
<patent-tables colsep="0" frame="none" rowsep="0">
<table align="left" class="description-table" cols="4" colsep="0" rowsep="0" width="100%">
<thead>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td" colspan="4" nameend="3" namest="offset" rowsep="1">TABLE 1</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td align="center" class="description-td" colspan="4" nameend="3" namest="offset" rowsep="1"> </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">RT</td>
<td class="description-td">Amino Acids</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td align="center" class="description-td" colspan="4" nameend="3" namest="offset" rowsep="1"> </td>
</tr>
</thead>
<tbody><tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">M-MLV</td>
<td class="description-td">L52, Y64, L135, H143, K152, Q165, </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">G181, H204, I218, N249, M289, T306, </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">F309, A517, D524, T544, V546, W548, </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">E562, H577, D583, L604, S606, G608, </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">F625, L626, H629, H631, H638, G641</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">AMV</td>
<td class="description-td">V2, L4, W12, P14, H16, T17, W20, I21, </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">Q23, W24, L26, P27, G29, V32, Q36, </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">L42, Q43, L44, G45, H46, 147, P49, </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">S50, L51, S52, C53, W54, F59, I61, </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">A64, S65, G66, S67, Y68, L70, L71, </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">A76, A79, P83, A86, V87, Q88, Q89, </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">G90, A91, W101, P102, L108, Q120, </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">S131, V132, N133, N134, Q135, P137, </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">A138, Q142, Q148, T151, Y180, M181, </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">S190, H191, G193, A196, I201, S202, </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">P214, V217, Q218, P221, G222, Q224, </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">L226, G227, Y228, G231, T233, Y234, </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">A236, P237, G239, L240, P244, I246, </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">T248, W250, Q252, G257, Q260, W261, </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">P264, L266, G267, L272, Y277, Q279, </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">L280, G282, S283, P285, N286, A288, </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">N292, L293, M297, I302, V303, L305, </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">S306, T308, L311, L320, I332, G333, </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">V334, G336, Q337, G338, P345, W348, </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">L349, F350, S351, P354, A357, F358,</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">A360, W361, L362, V364, L365, T366, </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">T370, A374, V377, G381, C392, P400, </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">G402, L405, G412, I414, F423, I425, </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">A426, P428, L433, H440, P441, V443, </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">G444, P445, A451, S453, S454, T455, </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">H456, G458, V459, V460, W462, W468, </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">I470, I473, A474, L476, G477, A478, </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">S479, V480, Q481, Q482, L483, A491, </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">W495, P496, T497, T498, P499, T500, </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">A507, F508, M512, L513, G520, V521, </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">P522, S523, T524, A525, A527, F528, </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">L534, S535, Q536, S538, V543, S548, </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">H549, S550, V552, P553, F556, T557, </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">N560, A562</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">RSV</td>
<td class="description-td">V2, L4, W12, P14, H16, T17, W20, </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">I21, Q23, W24, L26, P27, G29, V32, </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">Q36, L42, Q43, L44, G45, H46, I47, </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">P49, S50, L51, S52, C53, W54, F59, </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">I61, A64, S65, G66, S67, Y68, L70, </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">L71, A76, A79, P83, A86, V87, Q88, </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">Q89, G90, A91, W101, P102, L108, </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">Q120, S131, V132, N133, N134, Q135, </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">P137, A138, Q142, Q148, T151, Y180,</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">M181, S190, H191, G193, A196, I201, </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">S202, P214, V217, Q218, P221, G222, </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">Q224, L226, G227, Y228, G231, T233, </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">Y234, A236, P237, G239, L240, P244, </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">I246, T248, W250, Q252, G257, Q260, </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">W261, P264, L266, G267, L272, Y277, </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">Q279, L280, G282, S283, P285, N286,</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">A288, N292, L293, M297, I302, V303, </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">L305, S306, T308, L311, L320, I332, </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">G333, V334, G336, Q337, G338, P345, </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">W348, L349, F350, S351, P354, A357, </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">F358, A360, W361, L362, V364, L365, </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">T366, T370, A374, V377, G381, C392, </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">P400, G402, L405, G412, I414, F423, </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">I425, A426, P428, L433, H440, P441, </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">V443, G444, P445, A451, S453, S454, </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">T455, H456, G458, V459, V460, W462, </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">W468, I470, I473, A474, L476, G477, </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">A478, S479, V480, Q481, Q482, L483, </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">A491, W495, P496, T497, T498, P499, </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">T500, A507, F508, M512, L513, G520, </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">V521, P522, S523, T524, A525, A527, </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">F528, L534, S535, Q536, S538, V543, </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">S548, H549, S550, V552, P553, F556, </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">T557, N560, A562</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">HIV</td>
<td class="description-td">I1, P3, L11, P13, G14, M15, Q22, </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">W23, L25, T26, T38, G44, I46, S47,</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">G50, P51, N53, P54, Y55, F60, I62, S67, </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">T68, W70, L73, V89, Q90L91, G92, I93, </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">S104, V110, G111, S133, I134, N135, </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">N136, P139, G140, I141, Q144, N146, </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">Q150,, Y182, M183, I194, G195, Q196,</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">T,199, Q206, L209, P216, Q221, P224, </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">P225, L227, M229, G230, Y231, H234, </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">Q241, P242, V244, L245, S250, T252, </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">N254, Q257, G261, N264, W265, Q268, </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">P271, G272, Q277, C279, L281, L282, </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">G284, T285, A287, L288, T289, V291, </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">P293, L294, T295, L300, A303, I308,</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">L309, P312, H314, Y317, L324, I328, </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">Q329, G332, Q333, G334, Y341, P344, </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">F345, Y353, M356, G358, A359, H360, </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">T361, Q372, T376, V380, Q392, W405, </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">Q406, A407, F415, V416, N417, T418, </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">P419, P420, L424, W425, P432, V434, </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">G435, A436, A444, A445, N446, T449, </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">L451, N459, G461, Q463, V465, V466, </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">P467, L468, T469, N470, T471, T472, </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">N473, Q474, Y482, Q486, S488, G489, </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">L490, Q499, Y500, G503, I504, S512, </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">S514, L516, N518, Q519, Q523, I525, </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">W534, P536, A537, H538, G540, I541, </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">G542, Q546, L550, S552, A553, V554, </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">I555</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td align="center" class="description-td" colspan="4" nameend="3" namest="offset" rowsep="1"> </td>
</tr>
</tbody></table>
</patent-tables>
</tables>
</div>
<div class="description-paragraph" id="p-0031" num="0030">Those skilled in the art will appreciate that a different isolate of virus may encode a reverse transcriptase enzyme having a different amino acid at the positions identified above. Such isolates may be modified to produce the reverse transcriptases (e.g., thermostable reverse transcriptases) of the present invention.</div>
<div class="description-paragraph" id="p-0032" num="0031">Reverse transcriptases of the invention may have one or more of the following properties: (a) increased thermostability or increased half-life at elevated temperatures; (b) reduced, substantially reduced, or no detectable RNase H activity, (c) reduced or substantially reduced terminal deoxynucleotidyl transferase activity, and/or (d) increased fidelity.</div>
<div class="description-paragraph" id="p-0033" num="0032">In some embodiments, reverse transcriptases of the invention may be an M-MLV reverse transcriptase having one or more substitions in the amino acid sequence corresponding to positions 34-716 of SEQ ID NO:2, wherein amino acid number 1 of the M-MLV reverse transcriptase corresponds to the threonine at position 34 of SEQ ID NO:2 and the substitutions are selected from the group consisting of:
</div> <ul> <li id="ul0001-0001" num="0033">(a) threonine 330;</li> <li id="ul0001-0002" num="0034">(b) leucine 333;</li> <li id="ul0001-0003" num="0035">(c) proline 127;</li> <li id="ul0001-0004" num="0036">(d) tyrosine 344;</li> <li id="ul0001-0005" num="0037">(e) glycine 138;</li> <li id="ul0001-0006" num="0038">(f) leucine 139;</li> <li id="ul0001-0007" num="0039">(g) glutamic acid 275; and,</li> <li id="ul0001-0008" num="0040">(h) alanine 32,
<br/>
wherein the thermal inactivation profile of the mutant M-MLV reverse transcriptase with one or more substitions shows greater activity than the thermal inactivation profile of the RT corresponding to positions 34-716 of SEQ ID NO:2. In specific embodiments, threonine 330 may be replaced with proline, leucine 333 may be replaced by glutamine, proline 127 may be replaced by histidine, tyrosine 344 may be replaced by phenylalanine, glycine 138 may be replaced arginine, leucine 139 may be replaced by proline, glutamic acid 275 may be replaced by arginine or alanine 32 may be replaced by proline.
</li> </ul>
<div class="description-paragraph" id="p-0034" num="0041">In further embodiments, reverse transcriptase of the invention may be an M-MLV reverse transcriptase having one or more substitions in the amino acid sequence corresponding to positions 34-716 of SEQ ID NO:2, wherein amino acid number 1 of the M-MLV reverse transcriptase corresponds to the threonine at position 34 of SEQ ID NO:2 and the substitutions are selected from the group consisting of:
</div> <ul> <li id="ul0002-0001" num="0042">(a) proline 196;</li> <li id="ul0002-0002" num="0043">(b) leucine 435;</li> <li id="ul0002-0003" num="0044">(c) aspartic acid 653;</li> <li id="ul0002-0004" num="0045">(d) serine 67; and,</li> <li id="ul0002-0005" num="0046">(e) tyrosine 598,
<br/>
wherein the reverse transcriptase activity at 60° C. is at least 95%, 100%, 150%, 200%, 250%, 300% or 500% of the reverse transcriptase activity at 37° C. In specific embodiments, proline 196 may be replaced by serine, leucine 435 may be replaced by glycine, aspartic acid 653 may be replaced by histidine or asparagines, serine 67 may be replaced by lysine, or tyrosine 598 may be replaced tryptophan.
</li> </ul>
<div class="description-paragraph" id="p-0035" num="0047">In other embodiments, reverse transcriptases of the invention may be an M-MLV reverse transcriptase having one or more substitions in the amino acid sequence corresponding to positions 34-716 of SEQ ID NO:2, wherein amino acid number 1 of the M-MLV reverse transcriptase corresponds to the threonine at position 34 of SEQ ID NO:2 and the substitutions are selected from the group consisting of:
</div> <ul> <li id="ul0003-0001" num="0048">(a) tyrosine 133;</li> <li id="ul0003-0002" num="0049">(b) glutamine 190;</li> <li id="ul0003-0003" num="0050">(c) proline 196;</li> <li id="ul0003-0004" num="0051">(d) aspartic acid 200; and,</li> <li id="ul0003-0005" num="0052">(e) valine 223,
<br/>
wherein the reverse transcriptase exhibits increased template binding. In specific embodiments, tyrosine 133 may be replaced by threonine, glutamine 190 may be replaced by methionine, proline 196 may be replaced by serine, aspartic acid 200 may be replaced by glutamic acid, or valine 223 may be replaced by phenylalanine or tyrosine.
</li> </ul>
<div class="description-paragraph" id="p-0036" num="0053">Enzymes of the invention which have reduced or substantially reduced terminal deoxynucleotidyl transferase activity may comprise one or more modifications or mutations at positions corresponding to amino acids selected from the group consisting of:</div>
<div class="description-paragraph" id="p-0037" num="0054">(a) tyrosine 133 of M-MLV reverse transcriptase;</div>
<div class="description-paragraph" id="p-0038" num="0055">(b) threonine 197 of M-MLV reverse transcriptase; and</div>
<div class="description-paragraph" id="p-0039" num="0056">(c) phenylalanine 309 of M-MLV reverse transcriptase.</div>
<div class="description-paragraph" id="p-0040" num="0057">In specific embodiments, the invention is directed to M-MLV reverse transcriptases wherein tyrosine 133 is replaced with alanine, threonine 197 is replaced with glutamic acid, and/or phenylalanine 309 is replaced with asparagine. As will be appreciated, one or more of the amino acids identified may be deleted and/or replaced with one or a number of amino acid residues. Further included within the scope of the invention are reverse transcriptases, other than M-MLV reverse transcriptase, which contain alterations corresponding to those set out above.</div>
<div class="description-paragraph" id="p-0041" num="0058">Additionally, enzymes which exhibit increased fidelity may comprise one or more modifications or mutations at positions corresponding to amino acids selected from the group consisting of:</div>
<div class="description-paragraph" id="p-0042" num="0059">(a) tyrosine 64 of M-MLV reverse transcriptase;</div>
<div class="description-paragraph" id="p-0043" num="0060">(b) arginine 116 of M-MLV reverse transcriptase;</div>
<div class="description-paragraph" id="p-0044" num="0061">(c) glutamine 190 of M-MLV reverse transcriptase; and</div>
<div class="description-paragraph" id="p-0045" num="0062">(d) valine 223 of M-MLV reverse transcriptase.</div>
<div class="description-paragraph" id="p-0046" num="0063">As will be appreciated, one or more of the amino acids identified may be deleted and/or replaced with any one or a number of amino acid residues. Further, included in the invention are reverse transcriptases, other than M-MLV reverse transcriptase, that contain alterations corresponding to those set out above.</div>
<div class="description-paragraph" id="p-0047" num="0064">In some embodiments, the present invention provides a modified or mutated reverse transcriptase (e.g., preferably a modified or mutated retroviral reverse transcriptase) having a reverse transcriptase activity that has a half-life of greater than that of the corresponding un-modified or un-mutated reverse transcriptase at an elevated temperature, i.e., greater than 37° C. In some embodiments, the half-life of a reverse transcriptase of the present invention may be 5 minutes or greater and preferably 10 minutes or greater at 50° C. In some embodiments, the reverse transcriptases of the invention may have a half-life (e.g., at 50° C.) equal to or greater than about 25 minutes, preferably equal to or greater than about 50 minutes, more preferably equal to or greater than about 100 minutes, and most preferably, equal to or greater than about 200 minutes.</div>
<div class="description-paragraph" id="p-0048" num="0065">In some embodiments, the reverse transcriptases of the invention may have a half-life at 50° C. that is from about 10 minutes to about 200 minutes, from about 10 minutes to about 150 minutes, from about 10 minutes to about 100 minutes, from about 10 minutes to about 75 minutes, from about 10 minutes to about 50 minutes, from about 10 minutes to about 40 minutes, from about 10 minutes to about 30 minutes, or from about 10 minutes to about 20 minutes.</div>
<div class="description-paragraph" id="p-0049" num="0066">A modified or mutated reverse transcriptase of the invention (e.g., one having a half-life at 50° C. as described above) may be a modified or mutated retroviral reverse transcriptase. A reverse transcriptase according to the invention may be selected from a group consisting of M-MLV reverse transcriptase, ASV reverse transcriptase, HIV reverse transcriptase, Avian Sarcoma-Leukosis Virus (ASLV) reverse transcriptase, Rous Sarcoma Virus (RSV) reverse transcriptase, Avian Myeloblastosis Virus (AMV) reverse transcriptase, Avian Erythroblastosis Virus (AEV) Helper Virus MCAV reverse transcriptase, Avian Myelocytomatosis Virus MC29 Helper Virus MCAV reverse transcriptase, Avian Reticuloendotheliosis Virus (REV-T) Helper Virus REV-A reverse transcriptase, Avian Sarcoma Virus UR2 Helper Virus UR2AV reverse transcriptase, Avian Sarcoma Virus Y73 Helper Virus YAV reverse transcriptase, Rous Associated Virus (RAV) reverse transcriptase, and Myeloblastosis Associated Virus (MAV) reverse transcriptase, and fragments of any of the above having reverse transcriptase activity.</div>
<div class="description-paragraph" id="p-0050" num="0067">Mutated or modified reverse transcriptases of the present invention may have a reverse transcriptase activity (e.g., RNA-dependent DNA polymerase activity) that has a longer half-life at 55° C. than the reverse transcriptase activity of a corresponding un-mutated or un-modified reverse transcriptases. For example, introduction of the H204R, M289K, T306K, and F309N mutation into His<sub>6</sub>-H<sup>−</sup>RT increases the half-life at 55° C. from 1.6 minutes to 8.1 minutes (see Table 9). At 55° C., the half-life of reverse transcriptase activity of a mutated or modified reverse transcriptase of the invention may be greater than about 2 minutes, greater than about 3 minutes, greater than about 4 minutes, greater than about 5 minutes, greater than about 6 minutes, greater than about 7 minutes, greater than about 8 minutes, greater than about 10 minutes, greater than about 15 minutes, greater than about 20 minutes, or greater than about 30 minutes. At 55° C., the half-life of reverse transcriptase activity of a reverse transcriptase of the invention may be from about 2 minutes to about 60 minutes, from about 2 minutes to about 45 minutes, from about 2 minutes to about 30 minutes, from about 2 minutes to about 20 minutes, from about 2 minutes to about 15 minutes, from about 2 minutes to about 10 minutes, from about 2 minutes to about 8 minutes, from about 2 minutes to about 7 minutes, from about 2 minutes to about 6 minutes, from about 2 minutes to about 5 minutes, from about 2 minutes to about 4 minutes, or from about 2 minutes to about 3 minutes. Such a reverse transcriptase may be a modified or mutant retroviral reverse transcriptase.</div>
<div class="description-paragraph" id="p-0051" num="0068">A modified or mutated reverse transcriptase of the invention (e.g., one having a half-life at 55° C. as described above) may be a modified or mutated retroviral reverse transcriptase. A mutated reverse transcriptase according to the present invention may be selected from a group consisting of M-MLV reverse transcriptase, ASV reverse transcriptase, HIV reverse transcriptase, Avian Sarcoma-Leukosis Virus (ASLV) reverse transcriptase, Rous Sarcoma Virus (RSV) reverse transcriptase, Avian Myeloblastosis Virus (AMV) reverse transcriptase, Avian Erythroblastosis Virus (AEV) Helper Virus MCAV reverse transcriptase, Avian Myelocytomatosis Virus MC29 Helper Virus MCAV reverse transcriptase, Avian Reticuloendotheliosis Virus (REV-T) Helper Virus REV-A reverse transcriptase, Avian Sarcoma Virus UR2 Helper Virus UR2AV reverse transcriptase, Avian Sarcoma Virus Y73 Helper Virus YAV reverse transcriptase, Rous Associated Virus (RAV) reverse transcriptase, and Myeloblastosis Associated Virus (MAV) reverse transcriptase and fragments of any of the above having reverse transcriptase activity.</div>
<div class="description-paragraph" id="p-0052" num="0069">Reverse transcriptases of the present invention may produce more product (e.g., full length product) at elevated temperatures than other reverse transcriptases. In one aspect, comparisons of full length product synthesis is made at different temperatures (e.g., one temperature being lower, such as between 37° C. and 50° C., and one temperature being higher, such as between 50° C. and 78° C.) while keeping all other reaction conditions similar or the same. The amount of full length product produced may be determined using techniques well known in the art, for example, by conducting a reverse transcription reaction at a first temperature (e.g., 37° C., 38° C., 39° C., 40° C., etc.) and determining the amount of full length transcript produced, conducting a second reverse transcription reaction at a temperature higher than the first temperature (e.g., 45° C., 50° C., 52.5° C., 55° C., etc.) and determining the amount of full length product produced, and comparing the amounts produced at the two temperatures. A convenient form of comparison is to determine the percentage of the amount of full length product at the first temperature that is produced at the second (i.e., elevated) temperature. The reaction conditions used for the two reactions (e.g., salt concentration, buffer concentration, pH, divalent metal ion concentration, nucleoside triphosphate concentration, template concentration, reverse transcriptase concentration, primer concentration, length of time the reaction is conducted, etc.) are preferably the same for both reactions. Suitable reaction conditions include, but are not limited to, a template concentration of from about 1 nM to about 1 μM, from about 100 nM to 1 μM, from about 300 nM to about 750 nM, or from about 400 nM to about 600 nM, and a reverse transcriptase concentration of from about 1 nM to about 1 μM, from about 10 nM to 500 nM, from about 50 nM to about 250 nM, or from about 75 nM to about 125 nM. The ratio of the template concentration to the reverse transcriptase concentration may be from about 100:1 to about 1:1, from about 50:1 to about 1:1, from about 25:1 to about 1:1, from about 10:1 to about 1:1, from about 5:1 to about 1:1, or from about 2.5:1 to 1:1. A reaction may be conducted from about 5 minutes to about 5 hours, from about 10 minutes to about 2.5 hours, from about 30 minutes to about 2 hours, from about 45 minutes to about 1.5 hours, or from about 45 minutes to about 1 hour. A suitable reaction time is about one hour. Other suitable reaction conditions may be determined by those skilled in the art using routine techniques and examples of such conditions are provided below.</div>
<div class="description-paragraph" id="p-0053" num="0070">When the amount of full length product produced by a reverse transcriptase of the invention at an elevated temperature is compared to the amount of full length product produced by the same reverse transcriptase at a lower temperature, at an elevated temperature, the reverse transcriptases of the invention may produce not less than about 25%, 35%, 45%, 55%, 65%, 75%, 85%, 95%, 100% of the amount of full length product produced at the lower temperature. In some cases, the reverse transcriptases of the invention may produce an amount of full length product at a higher temperature that is greater than the amount of full length product produced by the reverse transcriptase at a lower temperature (e.g., 1% to about 100% greater). In one aspect, reverse transcriptases of the invention produce approximately the same amount (e.g., no more than a 25% difference) of full length product at the lower temperature compared to the amount of full length product made at the higher temperature.</div>
<div class="description-paragraph" id="p-0054" num="0071">A reverse transcriptase of the present invention may be one that synthesizes an amount of full length product, wherein the amount of full length product synthesized at 50° C. is no less than 10% (e.g., from about 10% to about 95%, from about 10% to about 80%, from about 10% to about 70%, from about 10% to about 60%, from about 10% to about 50%, from about 10% to about 40%, from about 10% to about 30%, or from about 10% to about 20%) of the amount of full length product it synthesizes at 40° C. In some embodiments, a reverse transcriptase of the invention is one wherein the amount of full length product synthesized at 50° C. is no less than 50% (e.g., from about 50% to about 95%, from about 50% to about 80%, from about 50% to about 70%, or from about 50% to about 60%) of the amount of full length product it synthesizes at 40° C. In some embodiments, a reverse transcriptase of the invention is one wherein the amount of full length product synthesized at 50° C. is no less than 75% (e.g., from about 75% to about 95%, from about 75%, to about 90%, from about 75% to about 85%, or from about 75% to about 80%) of the amount of full length product it synthesizes at 40° C. In other embodiments, a reverse transcriptase of the invention is one wherein the amount of full length product synthesized at 50° C. is no less than 85% (e.g., from about 85% to about 95%, or from about 85% to about 90%) of the amount of full length product it synthesizes at 40° C.</div>
<div class="description-paragraph" id="p-0055" num="0072">A reverse transcriptase of the invention may be one that synthesizes an amount of full length product, wherein the amount of full length product synthesized at 52.5° C. is no less than 10% (e.g., from about 10% to about 30%, from about 10% to about to about 25%, from about 10% to about 20%, from about 10% to about 15%, from about 20% to about 60%, from about 20% to about 40%, from about 20% to about 30%, from about 30% to about 80%, from about 30% to about 60%, from about 30% to about 45%, from about 40% to about 90%, from about 40% to about 80%, from about 40% to about 60%, from about 40% to about 50% from about 50% to about 90%, or from about 50% to about 70%), of the amount of full length product it synthesizes at 40° C. In some embodiments, the amount of full length product synthesized at 52.5° C. is no less than 30% (e.g., from about 30% to about 70%, from about 30% to about 60%, from about 30% to about 50%, or from about 30% to about 40%) of the amount of full length product it synthesizes at 40° C. In some embodiments, the amount of full length product synthesized at 52.5° C. is no less than 50% (e.g., from about 50% to about 70%, from about 50% to about 65%, from about 50% to about 60%, or from about 50% to about 55%), of the amount of full length product it synthesizes at 40° C.</div>
<div class="description-paragraph" id="p-0056" num="0073">A reverse transcriptase of the invention may be one that synthesizes an amount of full length product, wherein the amount of full length product synthesized at 55° C. is no less than 1% (e.g., from about 1% to about 30%, from about 1% to about 25%, from about 1% to about 20%, from about 1% to about 15%, from about 1% to about 10%, or from about 1% to about 5%) of the amount of full length product it synthesizes at 40° C. In some embodiments, the amount of full length product synthesized at 55° C. is no less than 5% (e.g., from about 5% to about 30%, from about 5% to about to about 25%, from about 5% to about 20%, from about 5% to about 15%, or from about 5% to about 10%) of the amount of full length product it synthesizes at 40° C. In some embodiments, the amount of full length product synthesized at 55° C. is no less than 10% (e.g., from about 10% to about 30%, from about 10% to about to about 25%, from about 10% to about 20%, from about 10% to about 15%, from about 20% to about 60%, from about 20% to about 40%, from about 20% to about 30%, from about 30% to about 80%, from about 30% to about 60%, from about 30% to about 45%, from about 40% to about 90%, from about 40% to about 80%, from about 40% to about 60%, from about 40% to about 50% from about 50% to about 90%, or from about 50% to about 70%) of the amount of full length product it synthesizes at 40° C.</div>
<div class="description-paragraph" id="p-0057" num="0074">In another aspect, the reverse transcriptases of the invention are capable of producing more nucleic acid product (e.g., cDNA) and, preferably, more full length product, at one or a number of elevated temperatures (typically between 40° C. an 78° C.) compared to the corresponding un-mutated or un-modified reverse transcriptase (e.g., the control reverse transcriptase). Such comparisons are typically made under similar or the same reaction conditions and the amount of product synthesized by the control reverse transcriptase is compared to the amount of product synthesized by the reverse transcriptase of the invention. Preferably, the reverse transcriptases of the invention produce at least about 5%, at least 10%, at least 15%, at least 25%, at least 50%, at least 75%, at least 100%, or at least 200% more product or full length product compared to the corresponding control reverse transcriptase under the same reaction conditions and temperature. The reverse transcriptases of the invention preferably produce from about 10% to about 200%, from about 25% to about 200%, from about 50% to about 200%, from about 75% to about 200%, or from about 100% to about 200% more product or full length product compared to a control reverse transcriptase under the same reaction conditions and incubation temperature. The reverse transcriptases of the invention preferably produce at least 2 times, at least 3 times, at least 4 times, at least 5 times, at least 6 times, at least 7 times, at least 8 times, at least 9 times, at least 10 times, at least 25 times, at least 50 times, at least 75 times, at least 100 times, at least 150 times, at least 200 times, at least 300 times, at least 400 times, at least 500 times, at least 1000 times, at least 5,000 times, or at least 10,000 times more product or full length product compared to a control reverse transcriptase (e.g., the corresponding un-mutated or un-modified reverse transcriptase) under the same reaction conditions and temperature. The reverse transcriptases of the invention preferably produce from 2 to 10,000, 5 to 10,000, 10 to 5,000, 50 to 5,000, 50 to 500, 2 to 500, 5 to 500, 5 to 200, 5 to 100, or 5 to 75 times more product or full length product than a control reverse transcriptase under the same reaction conditions and temperature.</div>
<div class="description-paragraph" id="p-0058" num="0075">In one aspect, the reverse transcriptases of the invention produce, at 50° C., at least 25% more, preferably at least 50% more and more preferably at least 100% more nucleic acid product or full length product than a control reverse transcriptase (which is preferably the corresponding wild-type reverse transcriptase). In another aspect, at 52.5° C., the reverse transcriptases of the invention produce at least 1.5 times, at least 2 times, at least 2.5 times, at least 3 times, at least 4 times, at least 5 times, at least 6 times, at least 7 times, at least 8 times, at least 9 times, at least 10 times the amount of nucleic acid product or full length product compared to a control reverse transcriptase. In another aspect, at 55° C., the reverse transcriptases of the invention produce at least 2 times, at least 5 times, at least 10 times, at least 15 times, at least 20 times, at least 25 times, at least 50 times, at least 75 times, at least 100 times the amount of nucleic acid product or full length product compared to a control reverse transcriptase. Such comparisons are preferably made under the same reaction conditions and temperature.</div>
<div class="description-paragraph" id="p-0059" num="0076">Modified or mutated reverse transcriptases of the present invention may have an increased thermostability at elevated temperatures as compared to corresponding un-modified or un-mutated reverse transcriptases. They may show increased thermostability in the presence or absence an RNA template. In some instances, reverse transcriptases of the invention may show an increased thermostability in both the presence and absence of an RNA template. Those skilled in the art will appreciate that reverse transcriptase enzymes are typically more thermostable in the presence of an RNA template. The increase in thermostability may be measured by comparing suitable parameters of the modified or mutated reverse transcriptase of the invention to those of a corresponding un-modified or un-mutated reverse transcriptase. Suitable parameters to compare include, but are not limited to, the amount of product and/or full length product synthesized by the modified or mutated reverse transcriptase at an elevated temperature compared to the amount or product and/or full length product synthesized by the corresponding un-modified or un-mutated reverse transcriptase at the same temperature, and/or the half-life of reverse transcriptase activity at an elevated temperature of a modified or mutated reverse transcriptase at an elevated temperature compared to that of a corresponding un-modified or un-mutated reverse transcriptase.</div>
<div class="description-paragraph" id="p-0060" num="0077">A modified or mutated reverse transcriptase of the invention may have an increase in thermostability at 50° C. of at least about 1.5 fold (e.g., from about 1.5 fold to about 100 fold, from about 1.5 fold to about 50 fold, from about 1.5 fold to about 25 fold, from about 1.5 fold to about 10 fold) compared, for example, to the corresponding un-mutated or un-modified reverse transcriptase. A reverse transcriptase of the invention may have an increase in thermostability at 50° C. of at least about 10 fold (e.g., from about 10 fold to about 100 fold, from about 10 fold to about 50 fold, from about 10 fold to about 25 fold, or from about 10 fold to about 15 fold) compared, for example, to the corresponding un-mutated or un-modified reverse transcriptase. A reverse transcriptase of the invention may have an increase in thermostability at 50° C. of at least about 25 fold (e.g., from about 25 fold to about 100 fold, from about 25 fold to about 75 fold, from about 25 fold to about 50 fold, or from about 25 fold to about 35 fold) compared to a corresponding un-mutated or un-modified reverse transcriptase.</div>
<div class="description-paragraph" id="p-0061" num="0078">The present invention also contemplates a modified or mutated thermostable reverse transcriptase, wherein the reverse transcriptase has an increase in thermostability of greater than about 1.5 fold at 52.5° C. (e.g., from about 1.5 fold to about 100 fold, from about 1.5 fold to about 50 fold, from about 1.5 fold to about 25 fold, or from about 1.5 fold to about 10 fold) compared, for example, to the corresponding un-mutated or un-modified reverse transcriptase. A reverse transcriptase of the invention may have an increase in thermostability at 52.5° C. of at least about 10 fold (e.g., from about 10 fold to about 100 fold, from about 10 fold to about 50 fold, from about 10 fold to about 25 fold, or from about 10 fold to about 15 fold) compared, for example, to the corresponding un-mutated or un-modified reverse transcriptase. A reverse transcriptase of the invention may have an increase in thermostability at 52.5° C. of at least about 25 fold (e.g., from about 25 fold to about 100 fold, from about 25 fold to about 75 fold, from about 25 fold to about 50 fold, or from about 25 fold to about 35 fold) compared, for example, to the corresponding un-mutated or un-modified reverse transcriptase.</div>
<div class="description-paragraph" id="p-0062" num="0079">In other embodiments, the present invention provides a reverse transcriptase, wherein the reverse transcriptase has an increase in thermostability of greater than about 1.5 fold at 55° C. (e.g., from about 1.5 fold to about 100 fold, from about 1.5 fold to about 50 fold, from about 1.5 fold to about 25 fold, or from about 1.5 fold to about 10 fold) compared to a corresponding un-mutated or un-modified reverse transcriptase. In some embodiments, a reverse transcriptase of the invention may have an increase in thermostability at 55° C. of at least about 10 fold (e.g., from about 10 fold to about 100 fold, from about 10 fold to about 50 fold, from about 10 fold to about 25 fold, or from about 10 fold to about 15 fold) compared to a corresponding un-mutated or un-modified reverse transcriptase. In some embodiments, a reverse transcriptase of the invention may have an increase in thermostability at 55° C. of at least about 25 fold (e.g., from about 25 fold to about 100 fold, from about 25 fold to about 75 fold, from about 25 fold to about 50 fold, or from about 25 fold to about 35 fold) compared to a corresponding un-mutated or un-modified reverse transcriptase.</div>
<div class="description-paragraph" id="p-0063" num="0080">The present invention provides reverse transcriptase enzymes, compositions and kits comprising such enzymes, and methods useful in preparing labeled nucleic acid molecules by reverse transcription. In general, the invention relates to the use of polypeptides of the invention (e.g., reverse transcriptase enzymes having one or more of the mutations identified above) to synthesized labeled nucleic acid molecules. In some embodiments, polypeptides of the invention may be heterodimers and more specifically two subunit enzymes (e.g., dimers) such as HIV RT and ASLV RTs. In some embodiments, polypeptides of the invention may be single sub-unit enzymes (e.g., M-MLV reverse transcriptase). Preferably, such labeling involves the use of modified nucleotides (e.g., labeled nucleotides, particularly fluorescently labeled nucleotides, nucleotide analogs and the like) and one or more nucleic acid templates (preferably RNA and most preferably mRNA). In accordance with the invention, one or more labeled nucleic acid molecules are synthesized which are complementary to all or a portion of the one or more templates. The labeled nucleic acid molecules preferably have one or more labeled nucleotides incorporated into the synthesized molecule and in a preferred aspect, the labels are one or more fluorescent labels (which may be the same or different). In another aspect, nucleotides are used during nucleic acid synthesis using the reverse transcriptases of the invention to produce one or more nucleic acid molecules complementary to all or a portion of one or more templates. In such aspects, such nucleotides, which are incorporated in the synthesized nucleic acid molecules, may be modified (before or after incorporation) to contain one or more labels, which may then be detected.</div>
<div class="description-paragraph" id="p-0064" num="0081">The invention also relates to compositions for use in the invention and such compositions may comprise one or more polypeptides of the invention (e.g., single sub-unit such as M-MLV RT and/or multi-subunit RTs such as HIV and ASLV RTs). Such compositions may further comprise one or more nucleotides, a suitable buffer, and/or one or more DNA polymerases. The compositions of the invention may also comprise one or more primers. The reverse transcriptases in these compositions preferably have RNase H activity or are reduced or substantially reduced in RNase H activity, and most preferably are enzymes selected from the group consisting of Moloney Murine Leukemia Virus (M-MLV) reverse transcriptase, Rous Sarcoma Virus (RSV) reverse transcriptase, Avian Myeloblastosis Virus (AMV) reverse transcriptase, Rous Associated Virus (RAV) reverse transcriptase, Myeloblastosis Associated Virus (MAV) reverse transcriptase and Human Immunodeficiency Virus (HIV) reverse transcriptase or other ASLV reverse transcriptases. The reverse transcriptases of the invention may be composed of one or more subunits (which may be the same or different). When two subunit RTs are use in the practice of the invention, such enzymes may contain various forms and combinations of such subunits such as αβ, αα, ββ, etc. and mutants, variants or derivatives thereof. In preferred compositions, the reverse transcriptases are present at working concentrations.</div>
<div class="description-paragraph" id="p-0065" num="0082">The invention is also directed to methods for making one or more nucleic acid molecules and/or labeled nucleic acid molecules, comprising mixing one or more nucleic acid templates (preferably one or more RNA templates and most preferably one or more messenger RNA templates) with one or more polypeptides of the invention having reverse transcriptase activity and incubating the mixture under conditions sufficient to synthesize one or more first nucleic acid molecules complementary to all or a portion of the one or more nucleic acid templates, wherein said at least one of said synthesized molecules are optionally labeled and/or comprise one or more labeled nucleotides and/or wherein said synthesized molecules may optionally be modified to contain one or more labels. In a preferred embodiment, the one or more first nucleic acid molecules are single-stranded cDNA molecules. Nucleic acid templates suitable for reverse transcription according to this aspect of the invention include any nucleic acid molecule or population of nucleic acid molecules (preferably RNA and most preferably mRNA), particularly those derived from a cell or tissue. In a preferred aspect, a population of mRNA molecules (a number of different mRNA molecules, typically obtained from cells or tissue) are used to make a labeled cDNA library, in accordance with the invention. Preferred sources of nucleic acid templates include viruses, virally infected cells, bacterial cells, fungal cells, plant cells and animal cells.</div>
<div class="description-paragraph" id="p-0066" num="0083">The invention also concerns methods for making one or more double-stranded nucleic acid molecules (which may optionally be labeled). Such methods comprise (a) mixing one or more nucleic acid templates (preferably RNA or mRNA, and more preferably a population of mRNA templates) with one or more polypeptides of the invention having reverse transcriptase activity; (b) incubating the mixture under conditions sufficient to make one or more first nucleic acid molecules complementary to all or a portion of the one or more templates; and (c) incubating the one or more first nucleic acid molecules under conditions sufficient to make one or more second nucleic acid molecules complementary to all or a portion of the one or more first nucleic acid molecules, thereby forming one or more double-stranded nucleic acid molecules comprising the first and second nucleic acid molecules. In accordance with the invention, the first and/or second nucleic acid molecules may be labeled (e.g., may comprise one or more of the same or different labeled nucleotides and/or may be modified to contain one or more of the same or different labels). Thus, labeled nucleotides may be used at one or both synthesis steps. Such methods may include the use of one or more DNA polymerases as part of the process of making the one or more double-stranded nucleic acid molecules. The invention also concerns compositions useful for making such double-stranded nucleic acid molecules. Such compositions comprise one or more reverse transcriptases of the invention and optionally one or more DNA polymerases, a suitable buffer and/or one or more nucleotides (preferably including labeled nucleotides).</div>
<div class="description-paragraph" id="p-0067" num="0084">The invention is also directed to nucleic acid molecules and/or labeled nucleic acid molecules (particularly single- or double-stranded cDNA molecules) produced according to the above-described methods and to kits comprising these nucleic acid molecules. Such molecules or kits may be used to detect nucleic acid molecules (for example by hybridization) or for diagnostic purposes.</div>
<div class="description-paragraph" id="p-0068" num="0085">The invention is also directed to kits for use in the methods of the invention. Such kits can be used for making nucleic acid molecules and/or labeled nucleic acid molecules (single- or double-stranded). Kits of the invention may comprise a carrier, such as a box or carton, having in close confinement therein one or more containers, such as vials, tubes, bottles and the like. In kits of the invention, a first container may contain one or more of the reverse transcriptase enzymes of the invention or one or more of the compositions of the invention. Kits of the invention may also comprise, in the same or different containers, at least one component selected from one or more DNA polymerases (preferably thermostable DNA polymerases), a suitable buffer for nucleic acid synthesis and one or more nucleotides. Alternatively, the components of the kit may be divided into separate containers. In one aspect, kits of the invention comprise reverse transcriptases which have RNase H activity or are reduced or substantially reduced in RNase H activity (or lacking or having undetectable RNase H activity). Such RTs preferably are selected from the group consisting of M-MLV reverse transcriptase, RSV reverse transcriptase, AMV reverse transcriptase, RAV reverse transcriptase, MAV reverse transcriptase and HIV reverse transcriptase. In additional preferred kits of the invention, the enzymes (e.g. reverse transcriptases and/or DNA polymerases) in the containers are present at working concentrations.</div>
<div class="description-paragraph" id="p-0069" num="0086">In specific embodiments, reverse transcriptases of the invention may not include M-MLV reverse transcriptases, HIV reverse transcriptases, AMV reverse transcriptases, and/or RSV reverse transcriptases. Thus, for example, in certain embodiments, the invention is directed to reverse transcriptases with increased thermostability that are not a HIV reverse transcriptase. In other embodiments, the invention is directed to reverse transcriptases with increased thermostability that are not a M-MLV reverse transcriptase. In yet other embodiments, the invention is directed to reverse transcriptases with increased thermostability that are not an AMV reverse transcriptase. In still other embodiments, the invention is directed to reverse transcriptases with increased thermostability that are not a RSV reverse transcriptase.</div>
<div class="description-paragraph" id="p-0070" num="0087">The present invention is also directed to nucleic acid molecules (e.g., vectors) containing a gene or nucleic acid molecules encoding the mutant or modified reverse transcriptases of the present invention (or fragments thereof including fragments having polymerase activity) and to host cells containing such DNA or other nucleic acid molecules. Any number of hosts may be used to express the gene or nucleic acid molecule of interest, including prokaryotic and eukaryotic cells. In specific embodiments, prokaryotic cells are used to express the reverse transcriptases of the invention. One example of a prokaryotic host suitable for use with the present invention is <i>Escherichia coli</i>. Examples of eukaryotic hosts suitable for use with the present invention include fungal cells (e.g., <i>Saccharomyces cerevisiae cells, Pichia pastoris </i>cells, etc.), plant cells, and animal cells (e.g., <i>Drosophila melanogaster </i>cells, <i>Spodoptera frugiperda </i>Sf9 and Sf21 cells, <i>Trichoplusa </i>High-Five cells, <i>C. elegans </i>cells, <i>Xenopus laevis </i>cells, CHO cells, COS cells, VERO cells, BHK cells, etc.). Preferably, polypeptides of the invention may be purified and/or isolated from a cell or organism expressing them, which may be a wild type cell or organism or a recombinant cell or organism. In some embodiments, such polypeptides may be substantially isolated from the cell or organism in which they are expressed.</div>
<div class="description-paragraph" id="p-0071" num="0088">The invention also relates to a method of producing reverse transcriptases of the invention, said method comprising:</div>
<div class="description-paragraph" id="p-0072" num="0089">(a) culturing a host cell comprising a gene or other nucleic acid molecule encoding a reverse transcriptase of the invention (preferably such reverse transcriptase gene or other nucleic acid molecule is contained by a vector within the host cell);</div>
<div class="description-paragraph" id="p-0073" num="0090">(b) expressing the gene or nucleic acid molecule; and</div>
<div class="description-paragraph" id="p-0074" num="0091">(c) isolating or purifying said reverse transcriptase.</div>
<div class="description-paragraph" id="p-0075" num="0092">The invention is also directed to methods for making one or more (e.g., one, two, three, four, five, ten, twelve, fifteen, etc.) nucleic acid molecules, comprising mixing one or more (e.g., one, two, three, four, five, ten, twelve, fifteen, etc.) nucleic acid templates (preferably one or more RNA templates and most preferably one or more messenger RNA templates or a population of messenger RNA templates) with one or more (e.g., one, two, three, four, five, ten, fifteen, etc.) reverse transcriptases of the invention and incubating the mixture under conditions sufficient to make a first nucleic acid molecule or molecules complementary to all or a portion of the one or more nucleic acid templates. In some embodiments, the mixture is incubated at an elevated temperature, i.e., greater than 37° C. In specific embodiments, the elevated temperature may be from about 40° C. or greater, from about 45° C. or greater, from about 50° C. or greater, from about 51° C. or greater, from about 52° C. or greater, from about 53° C. or greater, from about 54° C. or greater, from about 55° C. or greater, from about 56° C. or greater, from about 57° C. or greater, from about 58° C. or greater, from about 59° C. or greater, from about 60° C. or greater, from about 61° C. or greater, from about 62° C. or greater, from about 63° C. or greater, from about 64° C. or greater, from about 65° C. or greater, from about 66° C. or greater, from about 67° C. or greater, from about 68° C. or greater, from about 69° C. or greater, from about 70° C. or greater, from about 71° C. or greater, from about 72° C. or greater, from about 73° C. or greater, from about 74° C. or greater, from about 75° C. or greater, from about 76° C. or greater, from about 77° C. or greater, or from about 78° C. or greater. An elevated temperature may be within a temperature range of from about 40° C. to about 45° C., from about 40° C. to about 48° C., from about 40° C. to about 50° C., from about 40° C. to about 52° C., from about 40° C. to about 55° C., from about 40° C. to about 58° C., from about 40° C. to about 60° C., from about 40° C. to about 65° C., from about 42° C. to about 45° C., from about 42° C. to about 48° C., from about 42° C. to about 50° C., from about 42° C. to about 52° C., from about 42° C. to about 55° C., from about 42° C. to about 58° C., from about 42° C. to about 60° C., from about 42° C. to about 65° C., from about 45° C. to about 48° C., from about 45° C. to about 50° C., from about 45° C. to about 52° C., from about 45° C. to about 55° C., from about 45° C. to about 58° C., from about 45° C. to about 60° C., from about 45° C. to about 65° C., from about 48° C. to about 50° C., from about 48° C. to about 52° C., from about 48° C. to about 55° C., from about 48° C. to about 58° C., from about 48° C. to about 60° C., from about 48° C. to about 65° C., from about 50° C. to about 52° C., from about 50° C. to about 55° C., from about 50° C. to about 58° C., from about 50° C. to about 60° C., from about 50° C. to about 65° C., from about 52° C. to about 55° C., from about 52° C. to about 58° C., from about 52° C. to about 60° C., from about 52° C. to about 65° C., from about 55° C. to about 58° C., from about 55° C. to about 60° C., from about 55° C. to about 65° C., from about 55° C. to about 70° C., from about 58° C. to about 60° C., from about 58° C. to about 65° C., from about 58° C. to about 70° C. An elevated temperature may be within a temperature range from about 37° C. to about 75° C., from about 40° C. to about 75° C., from about 45° C. to about 75° C., from about 50° C. to about 75° C., from about 51° C. to about 75° C., from about 52° C. to about 75° C., from about 53° C. to about 75° C., from about 54° C. to about 75° C., from about 55° C. to about 75° C. In other embodiments, the elevated temperature may be within the range of about 50° C. to about 70° C., from about 51° C. to about 70° C., from about 52° C. to about 70° C., from about 53° C. to about 70° C., from about 54° C. to about 70° C., from about 55° C. to about 70° C., from about 56° C. to about 65° C., from about 56° C. to about 64° C. or about 56° C. to about 62° C. In other embodiments, the elevated temperature may be within the range of about 46° C. to about 60° C., from about 47° C. to about 60° C., from about 49° C. to about 60° C., from about 51° C. to about 60° C., from about 53° C. to about 60° C., or from about 54° C. to about 60° C. In additional specific embodiments, the first nucleic acid molecule is a single-stranded cDNA. The invention further includes nucleic acid molecules prepared by the above methods and reaction mixtures used in and formed by such methods. Such conditions for incubation may include the use of one or more buffers or buffering salts, one or more primers (such as oligo dT primers) and/or one or more nucleotides (e.g.; one or more nucleoside triphosphates). The invention also concerns compositions for making one or more nucleic acid molecules comprising one or more components selected from the group consisting of one or more reverse transcriptases of the invention, one or more primers, one or more nucleotides and one or more suitable buffers.</div>
<div class="description-paragraph" id="p-0076" num="0093">Nucleic acid templates suitable for reverse transcription according to this aspect of the invention include any nucleic acid molecule or population of nucleic acid molecules (preferably RNA and most preferably mRNA), particularly those derived from a cell or tissue. In a specific aspect, a population of mRNA molecules (a number of different mRNA molecules, typically obtained from a particular cell or tissue type) is used to make a cDNA library, in accordance with the invention. Examples of cellular sources of nucleic acid templates include bacterial cells, fungal cells, plant cells and animal cells.</div>
<div class="description-paragraph" id="p-0077" num="0094">The invention also concerns methods for making one or more (e.g., one, two, three, four, five, ten, twelve, fifteen, etc.) double-stranded nucleic acid molecules. Such methods comprise (a) mixing one or more nucleic acid templates (preferably RNA or mRNA, and more preferably a population of mRNA templates) with one or more (e.g., one, two, three, four, five, ten, fifteen, etc.) reverse transcriptases of the invention; (b) incubating the mixture under conditions sufficient to make a first nucleic acid molecule or molecules complementary to all or a portion of the one or more templates; and (c) incubating the first nucleic acid molecule or molecules under conditions sufficient to make a second nucleic acid molecule or molecules complementary to all or a portion of the first nucleic acid molecule or molecules, thereby forming one or more double-stranded nucleic acid molecules comprising the first and second nucleic acid molecules. In some embodiments, the incubation of step (b) is performed at an elevated temperature. In some embodiments, conditions may comprise the use of one or more labeled nucleotides and the double stranded nucleic acid molecules may be labeled. In specific embodiments, the elevated temperature may be from about 40° C. or greater, from about 45° C. or greater, from about 50° C. or greater, from about 51° C. or greater, from about 52° C. or greater, from about 53° C. or greater, from about 54° C. or greater, from about 55° C. or greater, from about 56° C. or greater, from about 57° C. or greater, from about 58° C. or greater, from about 59° C. or greater, from about 60° C. or greater, from about 61° C. or greater, from about 62° C. or greater, from about 63° C. or greater, from about 64° C. or greater, from about 65° C. or greater, from about 66° C. or greater, from about 67° C. or greater, from about 68° C. or greater, from about 69° C. or greater, from about 70° C. or greater, from about 71° C. or greater, from about 72° C. or greater, from about 73° C. or greater, from about 74° C. or greater, from about 75° C. or greater, from about 76° C. or greater, from about 77° C. or greater, or from about 78° C. or greater. An elevated temperature may be within a temperature range of from about 40° C. to about 45° C., from about 40° C. to about 48° C., from about 40° C. to about 50° C., from about 40° C. to about 52° C., from about 40° C. to about 55° C., from about 40° C. to about 58° C., from about 40° C. to about 60° C., from about 40° C. to about 65° C., from about 42° C. to about 45° C., from about 42° C. to about 48° C., from about 42° C. to about 50° C., from about 42° C. to about 52° C., from about 42° C. to about 55° C., from about 42° C. to about 58° C., from about 42° C. to about 60° C., from about 42° C. to about 65° C., from about 45° C. to about 48° C., from about 45° C. to about 50° C., from about 45° C. to about 52° C., from about 45° C. to about 55° C., from about 45° C. to about 58° C., from about 45° C. to about 60° C., from about 45° C. to about 65° C., from about 48° C. to about 50° C., from about 48° C. to about 52° C., from about 48° C. to about 55° C., from about 48° C. to about 58° C., from about 48° C. to about 60° C., from about 48° C. to about 65° C., from about 50° C. to about 52° C., from about 50° C. to about 55° C., from about 50° C. to about 58° C., from about 50° C. to about 60° C., from about 50° C. to about 65° C., from about 52° C. to about 55° C., from about 52° C. to about 58° C., from about 52° C. to about 60° C., from about 52° C. to about 65° C., from about 55° C. to about 58° C., from about 55° C. to about 60° C., from about 55° C. to about 65° C., from about 55° C. to about 70° C., from about 58° C. to about 60° C., from about 58° C. to about 65° C., from about 58° C. to about 70° C. An elevated temperature may be within a temperature range from about 37° C. to about 75° C., from about 40° C. to about 75° C., from about 45° C. to about 75° C., from about 50° C. to about 75° C., from about 51° C. to about 75° C., from about 52° C. to about 75° C., from about 53° C. to about 75° C., from about 54° C. to about 75° C., from about 55° C. to about 75° C. In other embodiments, the elevated temperature may be within the range of about 50° C. to about 70° C., from about 51° C. to about 70° C., from about 52° C. to about 70° C., from about 53° C. to about 70° C., from about 54° C. to about 70° C., from about 55° C. to about 70° C., from about 56° C. to about 65° C., from about 56° C. to about 64° C. or about 56° C. to about 62° C. In other embodiments, the elevated temperature may be within the range of about 46° C. to about 60° C., from about 47° C. to about 60° C., from about 49° C. to about 60° C., from about 51° C. to about 60° C., from about 53° C. to about 60° C., or from about 54° C. to about 60° C. Such conditions may involve the use of one or more suitable buffers or buffer salts, on or more primers (such as oligo dT primers), and one or more nucleotides. Such methods may include the use of one or more (e.g., one, two, three, four, five, ten, twelve, fifteen, etc.) DNA polymerases as part of the process of making the one or more double-stranded nucleic acid molecules. Such DNA polymerases are preferably thermostable DNA polymerases and most preferably the nucleic acid synthesis accomplished with such DNA polymerases is conducted at elevated temperatures, i.e., greater than 37° C. The invention also concerns compositions useful for making such double-stranded nucleic acid molecules. Such compositions comprise one or more (e.g., one, two, three, four, five, ten, twelve, fifteen, twenty, etc.) reverse transcriptases of the invention and optionally one or more DNA polymerases, a suitable buffer, one or more (e.g., one, two, three, four, five, ten, twelve, fifteen, etc.) primers, and/or one or more (e.g., one, two, three, four, five, etc.) nucleotides. The invention further includes nucleic acid molecules prepared by the above methods and reaction mixtures used in and formed by such methods.</div>
<div class="description-paragraph" id="p-0078" num="0095">The invention also relates to methods for amplifying a nucleic acid molecule. Such amplification methods comprise mixing the double-stranded nucleic acid molecule or molecules produced as described above with one or more (e.g., one, two, three, four, five, ten, twelve, fifteen, etc.) DNA polymerases (preferably thermostable DNA polymerases) and incubating the mixture under conditions sufficient to amplify the double-stranded nucleic acid molecule. In a first embodiment, the invention concerns a method for amplifying a nucleic acid molecule, the method comprising (a) mixing one or more (e.g., one, two, three, four, five, ten, twelve, fifteen, twenty, etc.) nucleic acid templates (preferably one or more RNA or mRNA templates and more preferably a population of mRNA templates) with one or more reverse transcriptases of the invention and with one or more DNA polymerases and (b) incubating the mixture under conditions sufficient to amplify nucleic acid molecules complementary to all or a portion of the one or more templates. In some embodiments, the incubation of step (b) is performed at an elevated temperature. In specific embodiments, the elevated temperature may be from about 40° C. or greater, from about 45° C. or greater, from about 50° C. or greater, from about 51° C. or greater, from about 52° C. or greater, from about 53° C. or greater, from about 54° C. or greater, from about 55° C. or greater, from about 56° C. or greater, from about 57° C. or greater, from about 58° C. or greater, from about 59° C. or greater, from about 60° C. or greater, from about 61° C. or greater, from about 62° C. or greater, from about 63° C. or greater, from about 64° C. or greater, from about 65° C. or greater, from about 66° C. or greater, from about 67° C. or greater, from about 68° C. or greater, from about 69° C. or greater, from about 70° C. or greater, from about 71° C. or greater, from about 72° C. or greater, from about 73° C. or greater, from about 74° C. or greater, from about 75° C. or greater, from about 76° C. or greater, from about 77° C. or greater, or from about 78° C. or greater. An elevated temperature may be within a temperature range of from about 40° C. to about 45° C., from about 40° C. to about 48° C., from about 40° C. to about 50° C., from about 40° C. to about 52° C., from about 40° C. to about 55° C., from about 40° C. to about 58° C., from about 40° C. to about 60° C., from about 40° C. to about 65° C., from about 42° C. to about 45° C., from about 42° C. to about 48° C., from about 42° C. to about 50° C., from about 42° C. to about 52° C., from about 42° C. to about 55° C., from about 42° C. to about 58° C., from about 42° C. to about 60° C., from about 42° C. to about 65° C., from about 45° C. to about 48° C., from about 45° C. to about 50° C., from about 45° C. to about 52° C., from about 45° C. to about 55° C., from about 45° C. to about 58° C., from about 45° C. to about 60° C., from about 45° C. to about 65° C., from about 48° C. to about 50° C., from about 48° C. to about 52° C., from about 48° C. to about 55° C., from about 48° C. to about 58° C., from about 48° C. to about 60° C., from about 48° C. to about 65° C., from about 50° C. to about 52° C., from about 50° C. to about 55° C., from about 50° C. to about 58° C., from about 50° C. to about 60° C., from about 50° C. to about 65° C., from about 52° C. to about 55° C., from about 52° C. to about 58° C., from about 52° C. to about 60° C., from about 52° C. to about 65° C., from about 55° C. to about 58° C., from about 55° C. to about 60° C., from about 55° C. to about 65° C., from about 55° C. to about 70° C., from about 58° C. to about 60° C., from about 58° C. to about 65° C., from about 58° C. to about 70° C. An elevated temperature may be within a temperature range from about 37° C. to about 75° C., from about 40° C. to about 75° C., from about 45° C. to about 75° C., from about 50° C. to about 75° C., from about 51° C. to about 75° C., from about 52° C. to about 75° C., from about 53° C. to about 75° C., from about 54° C. to about 75° C., from about 55° C. to about 75° C. In other embodiments, the elevated temperature may be within the range of about 50° C. to about 70° C., from about 51° C. to about 70° C., from about 52° C. to about 70° C., from about 53° C. to about 70° C., from about 54° C. to about 70° C., from about 55° C. to about 70° C., from about 56° C. to about 65° C., from about 56° C. to about 64° C. or about 56° C. to about 62° C. In other embodiments, the elevated temperature may be within the range of about 46° C. to about 60° C., from about 47° C. to about 60° C., from about 49° C. to about 60° C., from about 51° C. to about 60° C., from about 53° C. to about 60° C., or from about 54° C. to about 60° C.</div>
<div class="description-paragraph" id="p-0079" num="0096">Preferably, reverse transcriptases of the invention, used in methods of the invention, and/or present in compositions of the invention (1) are reduced or substantially reduced in RNase H activity, (2) are reduced or substantially reduced in TdT activity, and/or (3) exhibit increased fidelity. Preferably, DNA polymerases used with the invention may comprise a first DNA polymerase having 3′ exonuclease activity and a second DNA polymerase having substantially reduced 3′ exonuclease activity. The invention further includes nucleic acid molecules prepared by the above methods and reaction mixtures used in and formed by such methods.</div>
<div class="description-paragraph" id="p-0080" num="0097">The invention also concerns compositions comprising one or more reverse transcriptases of the invention and one or more DNA polymerases for use in amplification reactions. Such compositions may further comprise one or more nucleotides and/or a buffer suitable for amplification. Compositions of the invention may also comprise one or more oligonucleotide primers. Compositions of the invention may further include nucleic acid molecules prepared by the above methods and reaction mixtures used in and formed by such methods.</div>
<div class="description-paragraph" id="p-0081" num="0098">The invention is also directed to nucleic acid molecules (particularly single- or double-stranded cDNA molecules) or amplified nucleic acid molecules produced according to the above-described methods and to vectors (particularly expression vectors) comprising these nucleic acid molecules or amplified nucleic acid molecules.</div>
<div class="description-paragraph" id="p-0082" num="0099">The invention is further directed to recombinant host cells comprising the above-described nucleic acid molecules, amplified nucleic acid molecules or vectors. Examples of such host cells include bacterial cells, yeast cells, plant cells and animal cells (including insect cells and mammalian cells).</div>
<div class="description-paragraph" id="p-0083" num="0100">The invention is additionally directed to methods of producing polypeptides encoded by the nucleic acid molecules produced by the methods of the invention. Such methods include those comprising culturing the above-described recombinant host cells and isolating the encoded polypeptides. The invention further includes polypeptides produced by such methods.</div>
<div class="description-paragraph" id="p-0084" num="0101">The invention also concerns methods for sequencing one or more (e.g., one, two, three, four, five, ten, twelve, fifteen, etc.) nucleic acid molecules using compositions or enzymes of the invention. Such methods comprise (a) mixing one or more nucleic acid molecules (e.g., one or more RNA or DNA molecules) to be sequenced with one or more reverse transcriptases of the invention, and, optionally, one or more nucleotides, one or more terminating agents, such as one or more dideoxynucleoside triphosphates, and one or more primers; (b) incubating the mixture under conditions sufficient to synthesize a population of nucleic acid molecules complementary to all or a portion of the one or more (e.g., one, two, three, four, five, ten, twelve, fifteen, twenty, thirty, fifty, one hundred, two hundred, etc.) nucleic acid molecules to be sequenced; and (c) separating the population of nucleic acid molecules to determine the nucleotide sequence of all or a portion of the one or more nucleic acid molecules to be sequenced. Such methods may also comprise (a) mixing a nucleic acid molecule (e.g., one or more RNA or DNA molecules) to be sequenced with one or more primers, one or more reverse transcriptases of the invention, one or more nucleotides and one or more terminating agents, such as one or more dideoxynucleoside triphosphates; (b) incubating the mixture under conditions sufficient to synthesize a population of nucleic acid molecules complementary to all or a portion of the nucleic acid molecule to be sequenced; and (c) separating members of the population of nucleic acid molecules to determine the nucleotide sequence of all or a portion of the nucleic acid molecule to be sequenced. In some embodiments, such incubation may be performed at elevated temperatures as described herein. The invention further includes sequence data generated by the above methods, as well as methods for generating such sequence data, and reaction mixtures used in and formed by such methods.</div>
<div class="description-paragraph" id="p-0085" num="0102">The invention is also directed to kits for use in methods of the invention. Such kits can be used for making, sequencing or amplifying nucleic acid molecules (single- or double-stranded), preferably at the elevated temperatures described herein. Kits of the invention may comprise a carrier, such as a box or carton, having in close confinement therein one or more (e.g., one, two, three, four, five, ten, twelve, fifteen, etc.) containers, such as vials, tubes, bottles and the like. In kits of the invention, a first container contains one or more of the reverse transcriptase enzymes of the present invention. Kits of the invention may also comprise, in the same or different containers, one or more DNA polymerases (preferably thermostable DNA polymerases), one or more (e.g., one, two, three, four, five, ten, twelve, fifteen, etc.) suitable buffers for nucleic acid synthesis, one or more nucleotides and one or more (e.g., one, two, three, four, five, ten, twelve, fifteen, etc.) oligonucleotide primers. Alternatively, the components of the kit may be divided into separate containers (e.g., one container for each enzyme and/or component). Kits of the invention also may comprise instructions or protocols for carrying out the methods of the invention. In preferred kits of the invention, the reverse transcriptases are reduced or substantially reduced in RNase H activity (or lacking or having undetectable RNase H activity), and are most preferably selected from the group consisting of M-MLV RNase H− reverse transcriptase, RSV RNase H− reverse transcriptase, AMV RNase H− reverse transcriptase, RAV RNase H− reverse transcriptase, MAV RNase H− reverse transcriptase and HIV RNase H− reverse transcriptase. In other preferred kits of the invention, the reverse transcriptases are reduced or substantially reduced in TdT activity, and/or exhibit increased fidelity, as described elsewhere herein.</div>
<div class="description-paragraph" id="p-0086" num="0103">In additional preferred kits of the invention, the enzymes (reverse transcriptases and/or DNA polymerases) in the containers are present at working concentrations.</div>
<div class="description-paragraph" id="p-0087" num="0104">Thus, the invention is further directed to kits for use in reverse transcription, amplification or sequencing of a nucleic acid molecule, the kit comprising one or more reverse transcriptases of the invention.</div>
<div class="description-paragraph" id="p-0088" num="0105">As indicated above, kits of the invention may contain any number of various components for practicing methods of the invention. One example of such a component is instructions for performing methods of the invention. Example of such instructions include those which direct individuals using the kits to perform methods for amplifying nucleic acid molecules using one or more reverse transcriptases of the invention.</div>
<div class="description-paragraph" id="p-0089" num="0106">As one skilled in the art would recognize, the full text of these instructions need not be included with the kit. One example of a situation in which kits of the invention would not contain such full length instructions is where directions are provided which inform individuals using the kits where to obtain instructions for using the kit. Thus, instructions for performing methods of the invention may be obtain from internet web pages, separately sold or distributed manuals or other product literature, etc. The invention thus includes kits which direct kit users to locations where they can find instructions which are not directly packaged and/or distributed with the kits. These instructions may be in any form including, but not limited to, electronic or printed forms.</div>
<div class="description-paragraph" id="p-0090" num="0107">The invention thus also provides, in part, kits for performing methods using the reverse transcriptases of the invention. In specific embodiments, kits of the invention contain instructions for performing methods for amplifying and/or sequencing nucleic acid molecules. These methods will often involve reacting RNA molecules with one or more reverse transcriptases of the invention.</div>
<div class="description-paragraph" id="p-0091" num="0108">In specific embodiments, reverse transcriptases of kits of the invention may comprise one or more modifications or mutations at positions corresponding to amino acids selected from the group consisting of:</div>
<div class="description-paragraph" id="p-0092" num="0109">(a) leucine 52 of M-MLV reverse transcriptase;</div>
<div class="description-paragraph" id="p-0093" num="0110">(b) tyrosine 64 of M-MLV reverse transcriptase;</div>
<div class="description-paragraph" id="p-0094" num="0111">(c) lysine 152 of M-MLV reverse transcriptase;</div>
<div class="description-paragraph" id="p-0095" num="0112">(d) arginine 204 of M-MLV reverse transcriptase;</div>
<div class="description-paragraph" id="p-0096" num="0113">(e) methionine 289 of M-MLV reverse transcriptase;</div>
<div class="description-paragraph" id="p-0097" num="0114">(f) threonine 306 of M-MLV reverse transcriptase; and</div>
<div class="description-paragraph" id="p-0098" num="0115">(g) phenylalanine 309 of M-MLV reverse transcriptase.</div>
<div class="description-paragraph" id="p-0099" num="0116">Reverse transcriptases of the invention include any reverse transcriptase having one or a combination of the properties described herein. Such properties include, but are not limited to, enhanced thermostability, reduced or eliminated RNase H activity, reduced terminal deoxynucleotidyl transferase activity, and/or increased fidelity. Such reverse transcriptases include retroviral reverse transcriptases, bacterial reverse transcriptases, retrotransposon reverse transcriptases (e.g., reverse transcriptases of the Ty1 and/or Ty3 retrotransposons), and DNA polymerases having reverse transcriptase activity. Preferred reverse transcriptases of the invention include a single and multi-subunit reverse transcriptase and preferably retroviral reverse transcriptases. In particular, the invention relates to M-MLV-reverse transcriptases and ASLV-reverse transcriptases (such as AMV-RT and RSV-RT). Such reverse transcriptases of the invention preferably have reduced, substantially reduced, or no detectable RNase H activity.</div>
<div class="description-paragraph" id="p-0100" num="0117">Other embodiments of the present invention will be apparent to one of ordinary skill in light of the following drawings and description of the invention, and of the claims.</div>
<description-of-drawings>
<heading id="h-0008">BRIEF DESCRIPTION OF THE DRAWINGS/FIGURES</heading>
<div class="description-paragraph" id="p-0101" num="0118"> <figref idrefs="DRAWINGS">FIG. 1</figref> is a map of plasmid pBAD-6-His-M-MLV H− (F1).</div>
<div class="description-paragraph" id="p-0102" num="0119"> <figref idrefs="DRAWINGS">FIG. 2A</figref> is a linear representation of the coding sequence of the M-MLV reverse transcriptase showing the locations of the restriction enzyme cleavage sites used to generate the segments of the gene used to generate mutations. <figref idrefs="DRAWINGS">FIG. 2B</figref> is a schematic representation showing the insertion of a mutagenized PCR fragment into the coding sequence of the remaining portion of the reverse transcriptase gene.</div>
<div class="description-paragraph" id="p-0103" num="0120"> <figref idrefs="DRAWINGS">FIG. 3</figref> represents a scanned phosphoimage of an extension assay using (1) S<smallcaps>UPER</smallcaps>S<smallcaps>CRIPT</smallcaps>™ II reverse transcriptase, and (2) F309N. The [<sup>32</sup>P]-labeled 18-mer primer annealed to a 47-mer DNA template (5 nM) was extended by equal units of reverse transcriptase at 37° C. for 30 minutes as seen in the extension reactions with all 4 nucleotides. The extension reactions were analyzed by denaturing 6% gel electrophoresis. P, non-extended primer.</div>
<div class="description-paragraph" id="p-0104" num="0121"> <figref idrefs="DRAWINGS">FIG. 4</figref> represents a scanned phosphoimage showing a TdT extension assay of S<smallcaps>UPER</smallcaps>S<smallcaps>CRIPT</smallcaps>™ II reverse transcriptase and the mutants F309N, T197E, and Y133A. The [<sup>32</sup>P]-labeled 18-mer primer annealed to a 47-mer DNA template (5 nM) was extended with decreasing units of reverse transcriptase (lane (1) 646 units, lane (2) 200 units, lane (3) 50 units, and lane (4) 20 units) at 37° C. for 30 minutes with all four nucleotides (see the Methods section below in Example 3). The extension reactions were analyzed by denaturing 6% gel electrophoresis. In this assay, extension past the 47 nucleotide templates is considered non-template directed addition or TdT activity. P, non-extended primer.</div>
<div class="description-paragraph" id="p-0105" num="0122"> <figref idrefs="DRAWINGS">FIG. 5</figref> represents a scanned phosphoimage showing misinsertion assays of S<smallcaps>UPER</smallcaps>S<smallcaps>CRIPT</smallcaps>™ II reverse transcriptase (1) and mutant protein F309N reverse transcriptase (2) with DNA template. The [<sup>32</sup>P]-labeled 18-mer primer annealed to a 47-mer DNA template (5 nM) was extended by equal units of reverse transcriptase protein at 37° C. for 30 min. as seen in the extension reactions with all four nucleotides. The extension reactions were also performed in the presence of only 3 complementary dNTPs; minus dCTP, minus dATP, minus TTP, and minus dGTP. The extension reactions were analyzed by denaturing 6% gel electrophoresis. In this assay, the higher efficiency of elongation of terminated primer with only three nucleotides will reflect the lower fidelity of the S<smallcaps>UPER</smallcaps>S<smallcaps>CRIPT</smallcaps>™ II reverse transcriptase assayed. P, non-extended primer.</div>
<div class="description-paragraph" id="p-0106" num="0123"> <figref idrefs="DRAWINGS">FIG. 6</figref> represents a scanned phosphoimage showing a misinsertion assay of S<smallcaps>UPER</smallcaps>S<smallcaps>CRIPT</smallcaps>™ II reverse transcriptase (1) and mutant protein T197A/F309N reverse transcriptase (2) and V223H/F309N (3) with DNA template. The [<sup>32</sup>P]-labeled 18-mer primer annealed to a 47-mer DNA template (5 nM) was extended by equal units of reverse transcriptase protein at 37° C. for 30 min. as seen in the extension reactions with all four nucleotides. The extension reactions were also performed in the presence of only 3 complementary dNTPs; minus dATP, and minus dCTP. The extension reactions were analyzed by denaturing 6% gel electrophoresis. In this assay, the higher efficiency of elongation of terminated primer with only three nucleotides will reflect the lower fidelity of the S<smallcaps>UPER</smallcaps>S<smallcaps>CRIPT</smallcaps>™ II reverse transcriptase assayed. P, non-extended primer.</div>
<div class="description-paragraph" id="p-0107" num="0124"> <figref idrefs="DRAWINGS">FIGS. 7A-7C</figref> show representative results obtained from the screen for thermal stable RT mutants. Lysates of mutants were assayed for RT activity in a 96-well plate format. <sup>32</sup>P-Labeled DNA product was trapped on a membrane and the amount of radioactivity present was quantified with a phosphorimager. <figref idrefs="DRAWINGS">FIG. 7A</figref> shows the results of an initial screen of RT mutants in 4, 96-well plates. Heat pretreatment of lysates was at 58° C. for 10 min. RT mutants that retained the most activity after heat treatment at 58° C. were selected and lysates were screened again and the results are shown in <b>7</b>B. A duplicate screen was performed with no heat pretreatment (<figref idrefs="DRAWINGS">FIG. 7B</figref> upper panel) and heat pretreatment at 58° C. (<figref idrefs="DRAWINGS">FIG. 7B</figref> lower panel). RT mutants with the highest resistance to heat inactivation in crude extracts were purified by nickel-affinity chromatography and screened again for RT activity and the results are shown in <figref idrefs="DRAWINGS">FIG. 7C</figref>. The results after heat treatment at 37° C. are shown in <figref idrefs="DRAWINGS">FIG. 7C</figref> in the upper row, after heat treatment at 53° C. in <figref idrefs="DRAWINGS">FIG. 7C</figref> middle row, and after heat treatment at 58° C. in <b>7</b>C bottom row.</div>
<div class="description-paragraph" id="p-0108" num="0125"> <figref idrefs="DRAWINGS">FIG. 8</figref> shows a comparison of the thermal inactivation profiles of His<sub>6</sub>H− RT and His<sub>6 </sub>H− H204R T306K RT in crude extracts. Crude extracts were subjected to a heat treatment in a 96-well plate for 5 min. The temperature of the heat treatment increased from left to right, except that the wells on the far right were not heat treated.</div>
<div class="description-paragraph" id="p-0109" num="0126"> <figref idrefs="DRAWINGS">FIG. 9</figref> is a ribbon diagram of the crystal structure of amino acids 193 to 232 of M-MLV RT showing the sites of some of the amino acids identified by the methods of the present invention. Potential interactions of arginine substituted for histidine at M-MLV RT position 204 in α-helix H with E201 or T128. The catalytic site amino acids D224 and D225 in the turn between β10 and β11 are also shown. The three-dimensional structure is taken from Georgiadis, et al., (1995) <i>Structure </i>3, 879-892. Thus, the invention also includes reverse transcriptases having one or more mutations or modifications in various regions including the α-helix H region.</div>
<div class="description-paragraph" id="p-0110" num="0127"> <figref idrefs="DRAWINGS">FIGS. 10A and 10B</figref> are graphs of reverse transcriptase activity as a function of Mg<sup>2+</sup> concentration (<figref idrefs="DRAWINGS">FIG. 10A</figref>) and KCl concentration (<figref idrefs="DRAWINGS">FIG. 10B</figref>). The DNA polymerase assay for S<smallcaps>UPER</smallcaps>S<smallcaps>CRIPT</smallcaps>™ III (SuIII) RT was conducted at 37° C. or 50° C. for 10 minutes under various concentrations of A) Mg<sup>2+</sup> or B) KCl. S<smallcaps>UPER</smallcaps>S<smallcaps>CRIPT</smallcaps>™ II (SuII) at 37° C. included for comparison).</div>
<div class="description-paragraph" id="p-0111" num="0128"> <figref idrefs="DRAWINGS">FIGS. 11A and 11B</figref> show autoradiograms of TdT activity measure by extension for 60 minutes at various temperatures of a labeled DNA primer on DNA (<figref idrefs="DRAWINGS">FIG. 11A</figref>) or RNA (<figref idrefs="DRAWINGS">FIG. 11B</figref>) template forming a blunt end. T is template only, Lanes marked (−) is T-P plus enzyme without dNTPs. Since S<smallcaps>UPER</smallcaps>S<smallcaps>CRIPT</smallcaps>™ III is more thermostable, its TdT activity appears greater at 50 degrees than S<smallcaps>UPER</smallcaps>S<smallcaps>CRIPT</smallcaps>™ II.</div>
<div class="description-paragraph" id="p-0112" num="0129"> <figref idrefs="DRAWINGS">FIGS. 12A, 12B and 12C</figref> are graphs of RT activity as a function of incubation time. <figref idrefs="DRAWINGS">FIG. 12A</figref> shows the data obtained at 50° C., <figref idrefs="DRAWINGS">FIG. 12B</figref> shows the data obtained at 55° C., and <figref idrefs="DRAWINGS">FIG. 12C</figref> shows the data obtained at 60° C.</div>
<div class="description-paragraph" id="p-0113" num="0130"> <figref idrefs="DRAWINGS">FIG. 13</figref> is an autoradiogram comparing reverse transcriptase activity of a variety of commercially available reverse transcriptase enzymes at 45° C., 50° C., and 55° C. S<smallcaps>UPER</smallcaps>S<smallcaps>CRIPT</smallcaps>™ III is designated SS III and S<smallcaps>UPER</smallcaps>S<smallcaps>CRIPT</smallcaps>™ II is designated SS II.</div>
<div class="description-paragraph" id="p-0114" num="0131"> <figref idrefs="DRAWINGS">FIG. 14</figref> is a photograph of ethidium bromide stained gels showing the results of the evaluation of the pH of the first strand buffer. RT reactions with first-strand buffers at pHs from 8.0 to 8.8 were performed with 500 ng of total Hela RNA and 200 units of S<smallcaps>UPER</smallcaps>S<smallcaps>CRIPT</smallcaps>™ II (SS II) or 400 units of S<smallcaps>UPER</smallcaps>S<smallcaps>CRIPT</smallcaps>™ III (LEFN RT, which contains an N-terminal tag sequence=MASGTGGQQMGRDLYDDDDKH (SEQ ID NO:3) and the following point mutations H204R, T306K, M289L, and F309N). 2 μl of the resulting cDNA were then added to 50 μl PCR reactions containing the BF 2.4 kb or Pol ε 6.8 kb primer set. Resulting PCR products were then run on a 0.8% agarose gel containing 0.4 mg/ml ethidium bromide.</div>
<div class="description-paragraph" id="p-0115" num="0132"> <figref idrefs="DRAWINGS">FIG. 15</figref> is a photograph of ethidium bromide stained gels showing the results of the evaluation of the effect of temperature on the reverse transcription reaction with various reverse transcriptases. S<smallcaps>UPER</smallcaps>S<smallcaps>CRIPT</smallcaps>™ II (SS II) was compared to the His-tagged EFN reverse transcriptase (His tag sequence=MGGSHHHHHHGMASMTGGQQMGRDLYDDDDKH, amino acids 1-32 of SEQ ID NO:2 and Table 3, EFN mutations are H204R, T306K, and F309N), His-tagged LEFN reverse transcriptase (same His tag sequence, LEFN mutations are H204R, T306K, M289L, and F309N), and to S<smallcaps>UPER</smallcaps>S<smallcaps>CRIPT</smallcaps>™ III, which is the tagged, no His LEFN reverse transcriptase (tag sequence=MASGTGGQQMGRDLYDDDDKH (SEQ ID NO:3), LEFN mutations are H204R, T306K, M289L, and F309N).</div>
<div class="description-paragraph" id="p-0116" num="0133"> <figref idrefs="DRAWINGS">FIG. 16</figref> is a photograph of ethidium bromide stained gels showing the results of the evaluation of the effect of reverse transcriptase concentration on the reverse transcription reaction with S<smallcaps>UPER</smallcaps>S<smallcaps>CRIPT</smallcaps>™ III designated LEFN which contains the tag sequence=MASGTGGQQMGRDLYDDDDKH (SEQ ID NO:1), and LEFN mutations, which are H204R, T306K, M289L, and F309N.</div>
<div class="description-paragraph" id="p-0117" num="0134"> <figref idrefs="DRAWINGS">FIG. 17</figref> is a photograph of ethidium bromide stained gels showing the results of the comparison of hot start RT-PCR amplification by S<smallcaps>UPER</smallcaps>S<smallcaps>CRIPT</smallcaps>™ II (Panel A) or S<smallcaps>UPER</smallcaps>S<smallcaps>CRIPT</smallcaps>™ III (Panel B). Lanes (in duplicate) 1, 4, 7, and 10 are products reverse transcribed at 42° C. Lanes 2, 5, 8, and 11 are products reverse transcribed at 50° C. Lanes 3, 6, 9, and 12 are products transcribed at 55° C. Lanes 1-3 are the result of RNAs reverse transcribed by gene-specific priming from FGF, lanes 4-6 CBS 2.4, lanes 7-9 from TOP 3.2, lanes 10-12 VIN 4.6. Arrows indicate expected product sizes of 240 bp, 2390 bp, 3162 bp, and 4641 bp. S<smallcaps>UPER</smallcaps>S<smallcaps>CRIPT</smallcaps>™ III contains the tag sequence=MASGTGGQQMGRDLYDDDDKH (SEQ ID NO:3), and the LEFN mutations, which are H204R, T306K, M289L, and F309N.</div>
<div class="description-paragraph" id="p-0118" num="0135"> <figref idrefs="DRAWINGS">FIG. 18</figref> shows the results of RT-PCR performed with varying amounts S<smallcaps>UPER</smallcaps>S<smallcaps>CRIPT</smallcaps>™ III from 25 units to 250 units per reaction with a variety of primer sets.</div>
<div class="description-paragraph" id="p-0119" num="0136"> <figref idrefs="DRAWINGS">FIG. 19</figref> shows a comparison of S<smallcaps>UPER</smallcaps>S<smallcaps>CRIPT</smallcaps>™ II (SS II) and His tagged LEFN RT in RT-PCR using 200 or 400 units in the first strand reaction. His tagged LEFN has the His tag sequence=MGGSHHHHHHGMASMTGGQQMGRDLYDDDDKH, amino acids 1-32 of SEQ ID NO:2 and Table 3, and the LEFN mutations, which are H204R, T306K, M289L, and F309N).</div>
<div class="description-paragraph" id="p-0120" num="0137"> <figref idrefs="DRAWINGS">FIG. 20</figref> shows the use of S<smallcaps>UPER</smallcaps>S<smallcaps>CRIPT</smallcaps>™ III (LEFN RT) in RT-PCR with varying amounts of RT in the first strand reaction. S<smallcaps>UPER</smallcaps>S<smallcaps>CRIPT</smallcaps>™ III contains the tag sequence=MASGTGGQQMGRDLYDDDDKH (SEQ ID NO:3), and the LEFN mutations, which are H204R, T306K, M289L, and F309N.</div>
<div class="description-paragraph" id="p-0121" num="0138"> <figref idrefs="DRAWINGS">FIG. 21</figref> shows the results of a comparison of various primers in RT-PCR reactions using the polypeptides of the invention. EFN contains the tag sequence=MASGTGGQQMGRDLYDDDDKH (SEQ ID NO:3), and the EFN mutations, which are H204R, T306K, and F309N.</div>
<div class="description-paragraph" id="p-0122" num="0139"> <figref idrefs="DRAWINGS">FIG. 22</figref> shows a comparison of the thermal inactivation profiles of S<smallcaps>UPER</smallcaps>S<smallcaps>CRIPT</smallcaps>™ III and S<smallcaps>UPER</smallcaps>S<smallcaps>CRIPT</smallcaps>™ II to mutants of M-MLV recovered from random mutagenesis of the region from the start methionine to F369 of the MMLV genome. Increased thermostability is indicated by increased intensity of dots and more dots in a row compared to SSII controls.</div>
<div class="description-paragraph" id="p-0123" num="0140"> <figref idrefs="DRAWINGS">FIG. 23</figref> shows the results of a polymerisation reaction of various RT enzymes using a template primer with a stretch of 9 adenosines. Lane Nos. with specific RT or mutations are: 1, SSII; 2, Y133T; 18, Q190M; 22, P196S; 23, D200E; 27, V223C; 28, V223F; 29, V223W; 32, V223S; 34, Y222F/V223Y; 35, Y222F/V223F. The primer is indicated by “P” and the fully extended product is indicated by “F.”</div>
</description-of-drawings>
<heading id="h-0009">DETAILED DESCRIPTION OF THE INVENTION</heading>
<div class="description-paragraph" id="p-0124" num="0141">In the description that follows, a number of terms used in recombinant DNA, virology and immunology are utilized. In order to provide a clearer and consistent understanding of the specification and claims, including the scope to be given such terms, the following definitions are provided.</div>
<div class="description-paragraph" id="p-0125" num="0142">Cloning vector. As used herein “cloning vector” means a nucleic acid molecule such as plasmid, cosmid, phage, phagemid or other nucleic acid molecule which is able to replicate autonomously in a host cell, and which is characterized by one or a small number of recognition sequences, (e.g., restriction endonuclease recognition sites, recombination sites, topoisomerase recognition sites, etc.) at which such nucleic acid sequences may be manipulated in a determinable fashion, and into which a nucleic acid segment of interest may be inserted in order to bring about its replication and cloning. The cloning vector may further contain a marker suitable for use in the identification of cells transformed with the cloning vector. Markers, for example, are genes that confer a recognizable phenotype on host cells in which such markers are expressed. Commonly used markers include, but are not limited to, antibiotic resistance genes such as tetracycline resistance or ampicillin resistance.</div>
<div class="description-paragraph" id="p-0126" num="0143">Expression vector. As used herein “expression vector” means a nucleic acid molecule similar to a cloning vector but which may additionally comprise nucleic acid sequences capable of enhancing and/or controlling the expression of a gene or other nucleic acid molecule which has been cloned into it, after transformation into a host. The additional nucleic acid sequences may comprise promoter sequences, repressor binding sequences and the like. The cloned gene or nucleic acid molecule is usually operably linked to one or more (e.g., one, two, three, four, etc.) of such control sequences such as promoter sequences.</div>
<div class="description-paragraph" id="p-0127" num="0144">Recombinant host. As used herein “recombinant” means any prokaryotic or eukaryotic or microorganism which contains the desired cloned genes or nucleic acid molecules, for example, in an expression vector, cloning vector or any nucleic acid molecule. The term “recombinant host” is also meant to include those host cells which have been genetically engineered to contain the desired gene or other nucleic acid molecule on the host chromosome or genome.</div>
<div class="description-paragraph" id="p-0128" num="0145">Host. As used herein “host” means any prokaryotic or eukaryotic cell or organism that is the recipient of a replicable expression vector, cloning vector or any nucleic acid molecule. The nucleic acid molecule may contain, but is not limited to, a structural gene, a promoter and/or an origin of replication.</div>
<div class="description-paragraph" id="p-0129" num="0146">Promoter. As used herein “promoter” means a nucleic acid sequence generally described as the 5′ region of a gene, located proximal to the start codon which is capable of directing the transcription of a gene or other nucleic acid molecule. At the promoter region, transcription of an adjacent gene(s) or nucleic acid(s) is initiated.</div>
<div class="description-paragraph" id="p-0130" num="0147">Gene. As used herein “gene” means a nucleic acid sequence that contains information necessary for expression of a polypeptide or protein. It includes the promoter and the structural gene as well as other sequences involved in expression of the protein.</div>
<div class="description-paragraph" id="p-0131" num="0148">Structural gene. As used herein “structural gene” means a DNA or other nucleic acid sequence that is transcribed into messenger RNA that is then translated into a sequence of amino acids characteristic of a specific polypeptide.</div>
<div class="description-paragraph" id="p-0132" num="0149">Operably linked. As used herein “operably linked” means that a nucleic acid element is positioned so as to influence the initiation of expression of the polypeptide encoded by the structural gene or other nucleic acid molecule.</div>
<div class="description-paragraph" id="p-0133" num="0150">Expression. As used herein “expression” refers to the process by which a gene or other nucleic acid molecule produces a polypeptide. It includes transcription of the gene or nucleic acid molecule into messenger RNA (mRNA) and the translation of such mRNA into polypeptide(s).</div>
<div class="description-paragraph" id="p-0134" num="0151">Substantially Pure. As used herein “substantially pure” means that the desired material is essentially free from contaminating cellular components which are associated with the desired material in nature. In a preferred aspect, a reverse transcriptase of the invention has 25% or less, preferably 15% or less, more preferably 10% or less, more preferably 5% or less, and still more preferably 1% or less contaminating cellular components. In another aspect, the reverse transcriptases of the invention have no detectable protein contaminants when 200 units of reverse transcriptase are run on a protein gel (e.g., SDS-PAGE) and stained with Comassie blue. Contaminating cellular components may include, but are not limited to, enzymatic activities such as phosphatases, exonucleases, endonucleases or undesirable DNA polymerase enzymes. Preferably, reverse transcriptases of the invention are substantially pure.</div>
<div class="description-paragraph" id="p-0135" num="0152">Substantially isolated. As used herein “substantially isolated” means that the polypeptide of the invention is essentially free from contaminating proteins, which may be associated with the polypeptide of the invention in nature and/or in a recombinant host. In one aspect, a substantially isolated reverse transcriptase of the invention has 25% or less, preferably 15% or less, more preferably 10% or less, more preferably 5% or less, and still more preferably 1% or less contaminating proteins. In another aspect, in a sample of a substantially isolated polypeptide of the invention, 75% or greater (preferably 80%, 85%, 90%, 95%, 98%, or 99% or greater) of the protein in the sample is the desired reverse transcriptase of the invention. The percentage of contaminating protein and/or protein of interest in a sample may be determined using techniques known in the art, for example, by using a protein gel (e.g., SDS-PAGE) and staining the gel with a protein dye (e.g., Coomassie blue, silver stain, amido black, etc.). In another aspect, the reverse transcriptases of the invention have no detectable protein contaminants when 200 units of reverse transcriptase are run on a protein gel (e.g., SDS-PAGE) and stained with Comassie blue.</div>
<div class="description-paragraph" id="p-0136" num="0153">Primer. As used herein “primer” refers to a single-stranded oligonucleotide that is extended by covalent bonding of nucleotide monomers during amplification or polymerization of a DNA molecule.</div>
<div class="description-paragraph" id="p-0137" num="0154">Template. The term “template” as used herein refers to a double-stranded or single-stranded nucleic acid molecule which is to be amplified, copied or sequenced. In the case of a double-stranded DNA molecule, denaturation of its strands to form single-stranded first and second strands may be performed before these molecules are amplified, copied or sequenced. A primer complementary to a portion of a nucleic acid template is hybridized under appropriate conditions and a nucleic acid polymerase, such as the reverse transcriptase enzymes of the invention, may then add nucleotide monomers to the primer thereby synthesizing a nucleic acid molecule complementary to said template or a portion thereof. The newly synthesized nucleic acid molecule, according to the invention, may be equal or shorter in length than the original template. Mismatch incorporation during the synthesis or extension of the newly synthesized nucleic acid molecule may result in one or a number of mismatched base pairs. Thus, the synthesized nucleic acid molecule need not be exactly complementary to the template.</div>
<div class="description-paragraph" id="p-0138" num="0155">Incorporating. The term “incorporating” as used herein means becoming a part of a nucleic acid molecule or primer.</div>
<div class="description-paragraph" id="p-0139" num="0156">Oligonucleotide. “Oligonucleotide” refers to a synthetic or natural molecule comprising a covalently linked sequence of nucleotides which are joined by a phosphodiester bond between the 3′ position of the pentose of one nucleotide and the 5′ position of the pentose of the adjacent nucleotide.</div>
<div class="description-paragraph" id="p-0140" num="0157">Nucleotide. As used herein “nucleotide” refers to a base-sugar-phosphate combination. Nucleotides are monomeric units of a nucleic acid sequence (DNA and RNA). The term nucleotide includes ribonucleoside triphosphates ATP, UTP, CTG, GTP and deoxyribonucleoside triphosphates such as dATP, dCTP, dITP, dUTP, dGTP, dTTP, or derivatives thereof. Such derivatives include, for example, [αS]dATP, 7-deaza-dGTP and 7-deaza-dATP, and nucleotide derivatives that confer nuclease resistance on the nucleic acid molecule containing them. The term nucleotide as used herein also refers to dideoxyribonucleoside triphosphates (ddNTPs) and their derivatives. Illustrated examples of dideoxyribonucleoside triphosphates include, but are not limited to, ddATP, ddCTP, ddGTP, ddITP, and ddTTP. According to the present invention, a “nucleotide” may be unlabeled or detectably labeled by well known techniques. Detectable labels include, for example, radioactive isotopes, fluorescent labels, chemiluminescent labels, bioluminescent labels and enzyme labels. Fluorescent labels of nucleotides may include but are not limited fluorescein, 5-carboxyfluorescein (FAM), 2′7′-dimethoxy-4′5-dichloro-6-carboxyfluorescein (JOE), rhodamine, 6-carboxyrhodamine (R6G), N,N,N′,N′-tetramethyl-6-carboxyrhodamine (TAMRA), 6-carboxy-X-rhodamine (ROX), 4-(4′ dimethylaminophenylazo) benzoic acid (DABCYL), Cascade Blue, Oregon Green, Texas Red, Cyanine and 5-(2′-aminoethyl)aminonaphthalene-1-sulfonic acid (EDANS). Specific examples of fluorescently labeled nucleotides include [R6G]dUTP, [TAMRA]dUTP, [R110]dCTP, [R6G]dCTP, [TAMRA]dCTP, [JOE]ddATP, [R6G]ddATP, [FAM]ddCTP, [R110]ddCTP, [TAMRA]ddGTP, [ROX]ddTTP, [dR6G]ddATP, [dR110]ddCTP, [dTAMRA]ddGTP, and [dROX]ddTTP available from Perkin Elmer, Foster City, Calif. FluoroLink DeoxyNucleotides, FluoroLink Cy3-dCTP, FluoroLink Cy5-dCTP, FluoroLink Fluor X-dCTP, FluoroLink Cy3-dUTP, and FluoroLink Cy5-dUTP available from Amersham Arlington Heights, Ill.; Fluorescein-15-dATP, Fluorescein-12-dUTP, Tetramethyl-rodamine-6-dUTP, IR<sub>770</sub>-9-dATP, Fluorescein-12-ddUTP, Fluorescein-12-UTP, and Fluorescein-15-2′-dATP available from Boehringer Mannheim Indianapolis, Ind.; and ChromaTide Labeled Nucleotides, BODIPY-FL-14-UTP, BODIPY-FL-4-UTP, BODIPY-TMR-14-UTP, BODIPY-TMR-14-dUTP, BODIPY-TR-14-UTP, BODIPY-TR-14-dUTP, Cascade Blue-7-UTP, Cascade Blue-7-dUTP, fluorescein-12-UTP, fluorescein-12-dUTP, Oregon Green 488-5-dUTP, Rhodamine Green-5-UTP, Rhodamine Green-5-dUTP, tetramethylrhodamine-6-UTP, tetramethylrhodamine-6-dUTP, Texas Red-5-UTP, Texas Red-5-dUTP, and Texas Red-12-dUTP available from Molecular Probes, Eugene, Oreg.</div>
<div class="description-paragraph" id="p-0141" num="0158">Probes. The term probes refer to single or double stranded nucleic acid molecules or oligonucleotides which are detectably labeled by one or more detectable markers or labels. Such labels or markers may be the same or different and may include radioactive labels, fluorescent labels, chemiluminescent labels, bioluminescent labels and enzyme labels, although one or more fluorescent labels (which are the same or different) are preferred in accordance with the invention. Probes have specific utility in the detection of nucleic acid molecules by hybridization and thus may be used in diagnostic assays.</div>
<div class="description-paragraph" id="p-0142" num="0159">Hybridization. As used herein, hybridization (hybridizing) refers to the pairing of two complementary single-stranded nucleic acid molecules (RNA and/or DNA) to give a double-stranded molecule. As one skilled in the art will recognize, two nucleic acid molecules may be hybridized, although the base pairing is not completely complementary. Accordingly, mismatched bases do not prevent hybridization of two nucleic acid molecules provided that appropriate conditions, well known in the art, are used.</div>
<div class="description-paragraph" id="p-0143" num="0160">Thermostable Reverse Transcriptase. For the purposes of this disclosure, a thermostable reverse transcriptase includes a reverse transcriptase which retains a greater percentage or amount of its activity after a heat treatment than is retained by a reverse transcriptase that has wild-type thermostability after an identical treatment. Thus, a reverse transcriptase having increased/enhanced thermostability may be defined as a reverse transcriptase having any increase in thermostability, preferably from about 1.2 to about 10,000 fold, from about 1.5 to about 10,000 fold, from about 2 to about 5,000 fold, or from about 2 to about 2000 fold (preferably greater than about 5 fold, more preferably greater than about 10 fold, still more preferably greater than about 50 fold, still more preferably greater than about 100 fold, still more preferably greater than about 500 fold, and most preferably greater than about 1000 fold) retention of activity after a heat treatment sufficient to cause a reduction in the activity of a reverse transcriptase that is wild-type for thermostability. Preferably, the mutant or modified reverse transcriptase of the invention is compared to the corresponding unmodified or wild-type reverse transcriptase to determine the relative enhancement or increase in thermostability. For example, after a heat treatment at 52° C. for 5 minutes, a thermostable reverse transcriptase may retain approximately 90% of the activity present before the heat treatment, whereas a reverse transcriptase that is wild-type for thermostability may retain 10% of its original activity. Likewise, after a heat treatment at 53° C. for five minutes, a thermostable reverse transcriptase may retain approximately 80% of its original activity, whereas a reverse transcriptase that is wild-type for thermostability may have no measurable activity. Similarly, after a heat treatment at 50° C. for five minutes, a thermostable reverse transcriptase may retain approximately 50%, approximately 55%, approximately 60%, approximately 65%, approximately 70%, approximately 75%, approximately 80%, approximately 85%, approximately 90%, or approximately 95% of its original activity, whereas a reverse transcriptase that is wild-type for thermostability may have no measurable activity or may retain 10%, 15% or 20% of its original activity. In the first instance (i.e., after heat treatment at 52° C. for 5 minutes), the thermostable reverse transcriptase would be said to be 9-fold more thermostable than the wild-type reverse transcriptase. Examples of conditions which may be used to measure thermostability of reverse transcriptases are set out below, for example, in the Examples.</div>
<div class="description-paragraph" id="p-0144" num="0161">The thermostability of a reverse transcriptase can be determined by comparing the residual activity of a sample of the reverse transcriptase that has been subjected to a heat treatment, i.e., incubated at 52° C. for a given period of time, for example, five minutes, to a control sample of the same reverse transcriptase that has been incubated at room temperature for the same length of time as the heat treatment. Typically the residual activity may be measured by following the incorporation of a radiolabled deoxyribonucleotide into an oligodeoxyribonucleotide primer using a complementary oligoribonucleotide template. For example, the ability of the reverse transcriptase to incorporate [α-<sup>32</sup>P]-dGTP into an oligo-dG primer using a poly(riboC) template may be assayed to determine the residual activity of the reverse transcriptase.</div>
<div class="description-paragraph" id="p-0145" num="0162">In another aspect, thermostable reverse transcriptases of the invention may include any reverse transcriptase which is inactivated at a higher temperature compared to the corresponding wild-type, unmutated, or unmodified reverse transcriptase. Preferably, the inactivation temperature for the thermostable reverse transcriptases of the invention is from about 2° C. to about 50° C. (e.g., about 2° C., about 4° C., about 5° C., about 8° C., about 10° C., about 12° C., about 14° C., about 16° C., about 18° C., about 20° C., about 24° C., about 26° C., about 28° C., about 30° C., about 33° C., about 35° C., about 38° C., about 40° C., about 42° C., about 44° C., about 46° C., about 48° C., or about 50° C.) higher than the inactivation temperature for the corresponding wild-type, unmutated, or unmodified reverse transcriptase. More preferably, the inactivation temperature for the reverse transcriptases of the invention is from about 5° C. to about 50° C., from about 5° C. to about 40° C., from about 5° C. to about 30° C., or from about 5° C. to about 25° C. greater than the inactivation temperature for the corresponding wild-type, unmutated or unmodified reverse transcriptase, when compared under the same conditions.</div>
<div class="description-paragraph" id="p-0146" num="0163">The difference in inactivation temperature for the reverse transcriptase of the invention compared to its corresponding wild-type, unmutated or unmodified reverse transcriptase can be determined by treating samples of such reverse transcriptases at different temperatures for a defined time period and then measuring residual reverse transcriptase activity, if any, after the samples have been heat treated. Determination of the difference or delta in the inactivation temperature between the test reverse transcriptase compared to the wild-type, unmutated or unmodified control is determined by comparing the difference in temperature at which each reverse transcriptase is inactivated (i.e., no residual reverse transcriptase activity is measurable in the particular assay used). As will be recognized, any number of reverse transcriptase assays may be used to determine the different or delta of inactivation temperatures for any reverse transcriptases tested.</div>
<div class="description-paragraph" id="p-0147" num="0164">In another aspect, thermostability of a reverse transcriptase of the invention may be determined by measuring the half-life of the reverse transcriptase activity of a reverse transcriptase of interest. Such half-life may be compared to a control or wild-type reverse transcriptase to determine the difference (or delta) in half-life. Half-lifes of the reverse transcriptases of the invention are preferably determined at elevated temperatures (e.g., greater than 37° C.) and preferably at temperatures ranging from 40° C. to 80° C., more preferably at temperatures ranging from 45° C. to 75° C., 50° C. to 70° C., 50° C. to 65° C., and 50° C. to 60° C. Preferred half-lifes of the reverse transcriptases of the invention may range from 4 minutes to 10 hours, 4 minutes to 7.5 hours, 4 minutes to 5 hours, 4 minutes to 2.5 hours, or 4 minutes to 2 hours, depending upon the temperature used. For example, the reverse transcriptase activity of the reverse transcriptases of the invention may have a half-life of at least 4 minutes, at least 5 minutes, at least 6 minutes, at least 7 minutes, at least 8 minutes, at least 9 minutes, at least 10 minutes, at least 11 minutes, at least 12 minutes, at least 13 minutes, at least 14 minutes, at least 15 minutes, at least 20 minute, at least 25 minutes, at least 30 minutes, at least 40 minutes, at least 50 minutes, at least 60 minutes, at least 70 minutes, at least 80 minutes, at least 90 minutes, at least 100 minutes, at least 115 minutes, at least 125 minutes, at least 150 minutes, at least 175 minutes, at least 200 minutes, at least 225 minutes, at least 250 minutes, at least 275 minutes, at least 300 minutes, at least 400 minutes, at least 500 minutes at temperatures of 48° C., 50° C., 52° C., 52.5° C., 55° C., 57° C., 60° C., 62° C., 65° C., 68° C., and/or 70° C.</div>
<div class="description-paragraph" id="p-0148" num="0165">Terminal extension activity. As used herein, terminal extension activity refers to the ability of a reverse transcriptase (RT) to add additional bases on to the 3′ end of a newly synthesized cDNA strand beyond the 5′ end of the DNA or mRNA template. Terminal extension activity may add bases specifically (with a nucleotide bias) or randomly.</div>
<div class="description-paragraph" id="p-0149" num="0166">Terminal extension activity is also known as terminal deoxynucleotidyl transferase (TdT) activity. A reverse transcriptase having reduced TdT activity is defined as any reverse transcriptase having lower TdT specific activity than the TdT specific activity of the corresponding wild-type, unmutated, or unmodified enzyme, for example, less than about 90% of the TdT specific activity of the corresponding wild-type, unmutated, or unmodified enzyme, less than about 85% of the TdT specific activity of the corresponding wild-type, unmutated, or unmodified enzyme, less than about 80% of the TdT specific activity of the corresponding wild-type, unmutated, or unmodified enzyme, less than about 75% of the TdT specific activity of the corresponding wild-type, unmutated, or unmodified enzyme, less than about 50% of the TdT specific activity of the corresponding wild-type, unmutated, or unmodified enzyme, less than about 25% of the TdT specific activity of the corresponding wild-type, unmutated, or unmodified enzyme, less than about 15% of the TdT specific activity of the corresponding wild-type, unmutated, or unmodified enzyme, less than 10% of the TdT specific activity of the corresponding wild-type, unmutated, or unmodified enzyme, less than about 5% of the TdT specific activity of the corresponding wild-type, unmutated, or unmodified enzyme, or less than about 1% of the TdT specific activity of the corresponding wild-type, unmutated, or unmodified enzyme. A reverse transcriptase of the invention having substantially reduced TdT activity refers to a reverse transcriptase having a TdT specific activity level of 30% or less than the TdT specific activity of the corresponding wild-type or TdT<sup>+</sup> reverse transcriptase. Eliminated TdT activity is defined as a level of activity that is undetectable by the assay methods set out herein in Example 3.</div>
<div class="description-paragraph" id="p-0150" num="0167">As noted below in Example 3, reverse transcriptases are known in the art which extend nucleic acid molecules 2-3 nucleotides past the end of templates (e.g., RNA or DNA templates). Further, in any one reaction mixture in which reverse transcription occurs, mixtures of molecules may be present which contain different numbers of nucleotides that extend beyond the end of the template. TdT activity may be determined herein in reference to the number or percentage of molecules which contain one or more nucleotides which extend beyond the end of the template. For example, if a wild-type reverse transcriptase adds 1 or more nucleotides past the end of a template to 90% of the molecules generated during reverse transcription and a modified reverse transcriptase adds 1 or more nucleotides past the end of a template to 45% of the molecules under the same or similar conditions, then the modified reverse transcriptase would be said to exhibit a 50% decrease in TdT activity as compared to the wild-type enzyme. Further, an F309N, T306K, H204R mutant of M-MLV S<smallcaps>UPER</smallcaps>S<smallcaps>CRIPT</smallcaps>™ II has been generated which exhibits about 0% of the TdT activity exhibited by S<smallcaps>UPER</smallcaps>S<smallcaps>CRIPT</smallcaps>™ II when DNA is used as a template and about 10-20% of the TdT activity exhibited by S<smallcaps>UPER</smallcaps>S<smallcaps>CRIPT</smallcaps>™ II when RNA is used as a template.</div>
<div class="description-paragraph" id="p-0151" num="0168">Fidelity. Fidelity refers to the accuracy of polymerization, or the ability of the reverse transcriptase to discriminate correct from incorrect substrates, (e.g., nucleotides) when synthesizing nucleic acid molecules which are complementary to a template. The higher the fidelity of a reverse transcriptase, the less the reverse transcriptase misincorporates nucleotides in the growing strand during nucleic acid synthesis; that is, an increase or enhancement in fidelity results in a more faithful reverse transcriptase having decreased error rate or decreased misincorporation rate.</div>
<div class="description-paragraph" id="p-0152" num="0169">A reverse transcriptase having increased/enhanced/higher fidelity is defined as a polymerase having any increase in fidelity, preferably about 1.2 to about 10,000 fold, about 1.5 to about 10,000 fold, about 2 to about 5,000 fold, or about 2 to about 2000 fold (preferably greater than about 5 fold, more preferably greater than about 10 fold, still more preferably greater than about 50 fold, still more preferably greater than about 100 fold, still more preferably greater than about 500 fold and most preferably greater than about 100 fold) reduction in the number of misincorporated nucleotides during synthesis of any given nucleic acid molecule of a given length compared to the control reverse transcriptase. Preferably, the mutant or modified reverse transcriptase of the invention is compared to the corresponding unmodified or wild-type reverse transcriptase to determine the relative enhancement or increase in fidelity. For example, a mutated reverse transcriptase may misincorporate one nucleotide in the synthesis of a nucleic acid molecule segment of 1000 bases compared to an unmutated reverse transcriptase misincorporating 10 nucleotides in the same size segment. Such a mutant reverse transcriptase would be said to have an increase of fidelity of 10 fold.</div>
<div class="description-paragraph" id="p-0153" num="0170">Fidelity can also be measured by the decrease in the incidence of frame shifting, as described below in Example 5. A reverse transcriptase having increased fidelity may be defined as a polymerase or reverse transcriptase having any increase in fidelity with respect to frame shifting, as compared to a control reverse transcriptase (e.g., a corresponding wild-type and/or a corresponding un-mutated or un-modified reverse transcriptase), for example, a reverse transcriptase having greater than about 1.2 fold increased fidelity with respect to frame shifting, having greater than about 1.5 fold increased fidelity with respect to frame shifting, having greater than about 5 fold increased fidelity with respect to frame shifting, having greater than about 10 fold increased fidelity with respect to frame shifting, having greater than about 20 fold increased fidelity with respect to frame shifting, having greater than about 30 fold increased fidelity with respect to frame shifting, or having greater than about 40 fold increased fidelity with respect to frame shifting.</div>
<div class="description-paragraph" id="p-0154" num="0171">A reverse transcriptase having increased/enhanced/higher fidelity, with respect to frame shifting, can also be defined as a reverse transcriptase or polymerase having any increase in fidelity, such as from about 1.5 to about 10,000 fold, from about 2 to about 5,000 fold, from about 2 to about 2000 fold, from about 1.5 to about 40 fold, from about 5 to about 40 fold, from about 10 to about 40 fold, from about 20 to about 40 fold, from about 30 to about 40 fold, from about 5 to about 30 fold, from about 10 to about 30 fold, from about 15 to about 30 fold, from about 20 to about 30 fold, from about 5 to about 20 fold, from about 10 to about 20 fold, from about 15 to about 20 fold, from about 10 to about 100 fold, from about 15 to about 100 fold, from about 20 to about 100 fold, from about 30 to about 100 fold, or from about 50 to about 100 fold increased fidelity with respect to frame shifting.</div>
<div class="description-paragraph" id="p-0155" num="0172">A reverse transcriptase having reduced misincorporation is defined herein as either a mutated or modified reverse transcriptase that has about or less than 90%, has about or less than 85%, has about or less than 75%, has about or less than 70%, has about or less than 60%, or preferably has about or less than 50%, preferably has about or less than 25%, more preferably has about or less than 10%, and most preferably has about or less than 1% of relative misincorporation compared to the corresponding wild-type, unmutated, or unmodified enzyme.</div>
<div class="description-paragraph" id="p-0156" num="0173">The fidelity or misincorporation rate of a reverse transcriptase can be determined by sequencing or by other methods known in the art (Eckert &amp; Kunkel, 1990, <i>Nucl. Acids Res. </i>18:3739-3744). In one example, the sequence of a DNA molecule synthesized by the unmutated and mutated reverse transcriptases can be compared to the expected (known) sequence. In this way, the number of errors (misincorporation or frame shifts) can be determined for each enzyme and compared. In another example, the unmutated and mutated reverse transcriptases may be used to sequence a DNA molecule having a known sequence. The number of sequencing errors (misincorporation or frame shifts) can be compared to determine the fidelity or misincorporation rate of the enzymes. Other means of determining the fidelity or misincorporation rate include a forward complementation assay using an RNA template as described below and previously in Boyer J. C. et al. <i>Methods Enzymol. </i>275:523 (1996), and are set out in the examples. Other methods of determining the fidelity or misincorporation rate will be recognized by one of skill in the art.</div>
<div class="description-paragraph" id="p-0157" num="0174">Strand jumping. Strand jumping, as used herein, refers to a type of random mutation caused by an reverse transcriptase “skipping” more than one (e.g., two, five, ten, fifty, one-hundred, etc.) nucleotides on the mRNA template, resulting in a deletion of the corresponding nucleotides in the resulting cDNA. Sequencing the synthesized nucleic acid molecule and comparing to the expected sequence may allow determination of the level or amount of strand jumping for the reverse transcriptases of the invention. This level or amount may then be compared to the level or amount of strand jumping caused by the corresponding wild type and/or un-modified or un-mutated reverse transcriptase.</div>
<div class="description-paragraph" id="p-0158" num="0175">Hand domain. The hand domain, as used herein, refers to those amino acids which are in the area or areas that control the template, primer, or nucleotide interaction of the reverse transcriptase. This domain is further characterized by a group of three regions of secondary structure in a reverse transcriptase enzyme, the thumb, fingers and palm regions. The thumb region is defined as residing between amino acids 240-315 of HIV reverse transcriptase, or between amino acids 280-355 of M-MLV reverse transcriptase. The fingers region is defined as residing between amino acids 1-85 and 120-154 of HIV reverse transcriptase, or between 1-124 and 161-193 of M-MLV reverse transcriptase. The palm region is defined as residing between amino acids 86-199 and 155-239 of HIV reverse transcriptase, or between amino acids 126-160 and 193-279 of M-MLV reverse transcriptase. These areas are generally defined, and the amino acids defining the N-termini and C-termini are approximate. Corresponding regions may also be defined for other reverse transcriptases. Preferred reverse transcriptases of the invention have one or more modifications or mutations within the hand domain. More particularly, reverse transcriptases of the invention comprise one or more mutations or modifications within one or more regions, including the thumb, finger, and palm regions.</div>
<div class="description-paragraph" id="p-0159" num="0176">Full length. As used herein, full length when used to describe a product molecule, e.g., a cDNA molecule, indicates that the product molecule is the same length or substantially the same length as the template molecule, e.g., an mRNA molecule, from which it is produced by the activity of polypeptides of the invention. A cDNA molecule may be substantially the same length as the template from which it is copied when it is about 90%, 91%, 92%, 93%, 94%, 95%, 96%, 97%, 98%, 99% or greater of the length of the portion of the template located 3′ to the 3′ most nucleotide of the primer used to reverse transcribe the template. Thus, if a primer anneals in the center of a template, a full length product would be one that contains a cDNA copy of half of the template. Template molecules may be from about 100 bases to about 50 kb in length, from about 200 bases to about 50 kb in length, from about 300 bases to about 50 kb in length, from about 400 bases to about 50 kb in length, from about 500 bases to about 50 kb in length, from about 600 bases to about 50 kb in length, from about 700 bases to about 50 kb in length, from about 800 bases to about 50 kb in length, from about 900 bases to about 50 kb in length, and from about 1 kb to about 50 kb in length. In some embodiments, template molecules may be from about 500 bases to about 10 kb in length, from about 600 bases to about 10 kb in length, from about 700 bases to about 10 kb in length, from about 800 bases to about 10 kb in length, from about 900 bases to about 10 kb in length, from about 1000 bases to about 10 kb in length, from about 1100 bases to about 10 kb in length, and/or from about 1200 bases to about 10 kb in length. In some embodiments, template molecules may be from about 250 bases to about 5 kb in length, from about 300 bases to about 5 kb in length, and from about 350 bases to about 5 kb in length, from about 400 bases to about 5 kb in length, from about 450 bases to about 5 kb in length, from about 500 bases to about 5 kb in length, from about 550 bases to about 5 kb in length, from about 600 bases to about 5 kb in length, from about 650 bases to about 5 kb in length, from about 700 bases to about 5 kb in length, from about 750 bases to about 5 kb in length, from about 800 bases to about 5 kb in length, and from about 850 bases to about 5 kb in length.</div>
<div class="description-paragraph" id="p-0160" num="0177">In some embodiments, the ability of a reverse transcriptase to synthesize a full length product may be determined using a defined template and primer, for example, a polyadenylated template corresponding to the chloramphenicol acetyl transferase gene and an oligo(dT) primer, under defined reaction conditions, e.g., pH, salt concentration, divalent metal concentration, template concentration, temperature, etc. In some embodiments, a template molecule is greater than about 500 base pairs in length, and the amount of full length product synthesized may determined by separating full length product from truncated product, for example, by gel electrophoresis, and quantifying the full length product, for example, by incorporating a radiolabel in to the product and using a scintillation counter.</div>
<div class="description-paragraph" id="p-0161" num="0178">About. The term “about” as used herein, means the recited number plus or minus 10%. Thus, “about 100” includes 90-110.</div>
<div class="description-paragraph" id="h-0010" num="0000">Overview</div>
<div class="description-paragraph" id="p-0162" num="0179">In general, the invention provides, in part, compositions for use in reverse transcription of a nucleic acid molecule comprising a reverse transcriptase with one or more (e.g., one, two, three, four, five, ten, twelve, fifteen, twenty, thirty, etc.) mutations or modification which render the reverse transcriptase more thermostable. The invention also provides compositions for use in reverse transcription of a nucleic acid molecule, the compositions comprising a reverse transcriptase with one or more mutations or modification which render the reverse transcriptase more efficient, that is having higher fidelity, and/or has lower TdT activity than a corresponding un-mutated or un-modified reverse transcriptase. The invention further provides compositions comprising a reverse transcriptase with one or more mutations or modification which render the reverse transcriptase more thermostable and/or more efficient than a corresponding un-mutated or un-modified reverse transcriptase.</div>
<div class="description-paragraph" id="p-0163" num="0180">The enzymes in these compositions are preferably present in working concentrations and are also preferably reduced, substantially reduced, or eliminated in RNase H activity. Alternatively, reverse transcriptases used in the compositions of the invention may have RNase H activity. Preferred reverse transcriptases include retroviral reverse transcriptases such as M-MLV reverse transcriptase, HIV reverse transcriptase, RSV reverse transcriptase, AMV reverse transcriptase, RAV reverse transcriptase, and MAV reverse transcriptase or other ASLV reverse transcriptases or their corresponding RNase H− derivatives. Additional reverse transcriptases which may be used to prepare compositions of the invention include bacterial reverse transcriptases (e.g., <i>Escherichia coli </i>reverse transcriptase) (see, e.g., Mao et al., <i>Biochem. Biophys. Res. Commun. </i>227:489-93 (1996)) and reverse transcriptases of <i>Saccharomyces cerevisiae </i>(e.g., reverse transcriptases of the Ty1 or Ty3 retrotransposons) (see, e.g., Cristofari et al., <i>Jour. Biol. Chem. </i>274:36643-36648 (1999); Mules et al., <i>Jour. Virol. </i>72:6490-6503 (1998)).</div>
<div class="description-paragraph" id="p-0164" num="0181">In accordance with the invention, any number of mutations can be made to the reverse transcriptases and, in a preferred aspect, multiple mutations can be made to result in an increased thermostability and/or to confer other desired properties on reverse transcriptases of the invention. Such mutations include point mutations, frame shift mutations, deletions and insertions, with one or more (e.g., one, two, three, four, five, ten, twelve, fifteen, twenty, thirty, etc.) point mutations preferred. Mutations may be introduced into reverse transcriptases of the present invention using any methodology known to those of skill in the art. Mutations may be introduced randomly by, for example, conducting a PCR reaction in the presence of manganese as a divalent metal ion cofactor. Alternatively, oligonucleotide directed mutagenesis may be used to create the mutant polymerases which allows for all possible classes of base pair changes at any determined site along the encoding DNA molecule. In general, this technique involves annealing an oligonucleotide complementary (except for one or more mismatches) to a single stranded nucleotide sequence coding for the reverse transcriptase of interest. The mismatched oligonucleotide is then extended by DNA polymerase, generating a double-stranded DNA molecule which contains the desired change in sequence in one strand. The changes in sequence can, for example, result in the deletion, substitution, or insertion of an amino acid. The double-stranded polynucleotide can then be inserted into an appropriate expression vector, and a mutant or modified polypeptide can thus be produced. The above-described oligonucleotide directed mutagenesis can, for example, be carried out via PCR.</div>
<div class="description-paragraph" id="p-0165" num="0182">The invention is also directed to methods for reverse transcription of one or more (e.g., one, two, three, four, five, ten, twelve, fifteen, twenty, etc.) nucleic acid molecules comprising mixing one or more (e.g., one, two, three, four, five, ten, twelve, fifteen, twenty, etc.) nucleic acid templates, which are preferably RNA or messenger RNA (mRNA) and more preferably a population of mRNA molecules, with one or more reverse transcriptase of the present invention and incubating the mixture under conditions sufficient to make a nucleic acid molecule or molecules complementary to all or a portion of the one or more (e.g., one, two, three, four, five, ten, twelve, fifteen, twenty, thirty, etc.) templates. To make the nucleic acid molecule or molecules complementary to the one or more templates, a primer (e.g., an oligo(dT) primer) and one or more nucleotides are preferably used for nucleic acid synthesis in the 5′ to 3′ direction. Nucleic acid molecules suitable for reverse transcription according to this aspect of the invention include any nucleic acid molecule, particularly those derived from a prokaryotic or eukaryotic cell. Such cells may include normal cells, diseased cells, transformed cells, established cells, progenitor cells, precursor cells, fetal cells, embryonic cells, bacterial cells, yeast cells, animal cells (including human cells), avian cells, plant cells and the like, or tissue isolated from a plant or an animal (e.g., human, cow, pig, mouse, sheep, horse, monkey, canine, feline, rat, rabbit, bird, fish, insect, etc.). Nucleic acid molecules suitable for reverse transcription may also be isolated and/or obtained from viruses and/or virally infected cells.</div>
<div class="description-paragraph" id="p-0166" num="0183">The invention further provides methods for amplifying or sequencing a nucleic acid molecule comprising contacting the nucleic acid molecule with a reverse transcriptase of the present invention. Preferred such methods comprise one or more polymerase chain reactions (PCRs).</div>
<div class="description-paragraph" id="h-0011" num="0000">Sources of Reverse Transcriptases</div>
<div class="description-paragraph" id="p-0167" num="0184">Enzymes for use in compositions, methods and kits of the invention include any enzyme having reverse transcriptase activity. Such enzymes include, but are not limited to, retroviral reverse transcriptase, retrotransposon reverse transcriptase, hepatitis B reverse transcriptase, cauliflower mosaic virus reverse transcriptase, bacterial reverse transcriptase, Tth DNA polymerase, Taq DNA polymerase (Saiki, R. K., et al., <i>Science </i>239:487-491 (1988); U.S. Pat. Nos. 4,889,818 and 4,965,188), Tne DNA polymerase (PCT Publication No. WO 96/10640), Tma DNA polymerase (U.S. Pat. No. 5,374,553) and mutants, fragments, variants or derivatives thereof (see, e.g., commonly owned U.S. Pat. Nos. 5,948,614 and 6,015,668, which are incorporated by reference herein in their entireties). Preferably, reverse transcriptases for use in the invention include retroviral reverse transcriptases such as M-MLV reverse transcriptase, AMV reverse transcriptase, RSV reverse transcriptase, RAV reverse transcriptase, MAV reverse transcriptase, and generally ASLV reverse transcriptases. As will be understood by one of ordinary skill in the art, modified reverse transcriptases may be obtained by recombinant or genetic engineering techniques that are routine and well-known in the art. Mutant reverse transcriptases can, for example, be obtained by mutating the gene or genes encoding the reverse transcriptase of interest by site-directed or random mutagenesis. Such mutations may include point mutations, deletion mutations and insertional mutations. For example, one or more point mutations (e.g., substitution of one or more amino acids with one or more different amino acids) may be used to construct mutant reverse transcriptases of the invention.</div>
<div class="description-paragraph" id="p-0168" num="0185">The invention further includes reverse transcriptases which are 70%, 75%, 80%, 85%, 90%, 95%, 96%, 97%, 98%, or 99% identical at the amino acid level to a wild-type reverse transcriptase (e.g., M-MLV reverse transcriptase, AMV reverse transcriptase, RSV reverse transcriptase, HIV reverse transcriptase, etc.) and exhibit increased thermostability and/or other desired properties of the invention. Also included within the invention are reverse transcriptases which are 70%, 75%, 80%, 85%, 90%, 95%, 96%, 97%, 98%, or 99% identical at the amino acid level to a reverse transcriptase comprising the amino acid sequence set out below in Table 3 (SEQ ID NO:2) and exhibit increased thermostability and/or more efficient (e.g., having higher fidelity and/or having lower TdT activity). The invention also includes nucleic acid molecules which encode the above described reverse transcriptases.</div>
<div class="description-paragraph" id="p-0169" num="0186">The invention also includes fragments of reverse transcriptases which comprise at least 200, 250, 300, 350, 400, 450, 500, 550, 600, 650, or 700 amino acid residues and retain one or more activities associated with reverse transcriptases. Such fragments may be obtained by deletion mutation, by recombinant techniques that are routine and well-known in the art, or by enzymatic digestion of the reverse transcriptase(s) of interest using any of a number of well-known proteolytic enzymes. The invention further includes nucleic acid molecules which encode the above described mutant reverse transcriptases and reverse transcriptase fragments.</div>
<div class="description-paragraph" id="p-0170" num="0187">Reverse transcriptase fragments of the invention also comprise amino acids 1-10, 11-20, 21-30, 31-40, 41-50, 51-60, 61-70, 71-80, 81-90, 91-100, 101-110, 111-120, 121-130, 131-140, 141-150, 151-160, 161-170, 171-180, 181-190, 191-200, 201-210, 211-220, 221-230, 231-240, 241-250, 251-260, 261-270, 271-280, 281-290, 291-300, 301-310, 311-320, 321-330, 331-340, 341-350, 351-360, 361-370, 371-380, 381-390, 391-400, 401-410, 411-420, 421-430, 431-440, 441-450, 451-460, 461-470, 471-480, 481-490, 491-500, 501-510, 511-520, 521-530, 531-540, and/or 541-550 and/or amino acids 1-355, 1-498, 1-500, and/or 1-550 of M-MLV reverse transcriptase (and more preferably the sequence shown in Table 3, which may further contain one or more of the modifications or mutations discussed herein), as well as corresponding fragments of other reverse transcriptases. Reverse transcriptase fragments of the invention further comprise polypeptides which are 70%, 75%, 80%, 85%, 90%, 95%, 96%, 97%, 98%, or 99% identical to one or more of the fragments set out above. The invention also concerns various combinations of any number of these fragments.</div>
<div class="description-paragraph" id="p-0171" num="0188">By a protein or protein fragment having an amino acid sequence at least, for example, 70% “identical” to a reference amino acid sequence it is intended that the amino acid sequence of the protein is identical to the reference sequence except that the protein sequence may include up to 30 amino acid alterations per each 100 amino acids of the amino acid sequence of the reference protein. In other words, to obtain a protein having an amino acid sequence at least 70% identical to a reference amino acid sequence, up to 30% of the amino acid residues in the reference sequence may be deleted or substituted with another amino acid, or a number of amino acids up to 30% of the total amino acid residues in the reference sequence may be inserted into the reference sequence. These alterations of the reference sequence may occur at the amino (N—) and/or carboxy (C—) terminal positions of the reference amino acid sequence and/or anywhere between those terminal positions, interspersed either individually among residues in the reference sequence and/or in one or more contiguous groups within the reference sequence. As a practical matter, whether a given amino acid sequence is, for example, at least 70% identical to the amino acid sequence of a reference protein can be determined conventionally using known computer programs such as those described above for nucleic acid sequence identity determinations, or using the CLUSTAL W program (Thompson, J. D., et al., <i>Nucleic Acids Res. </i>22:4673-4680 (1994)).</div>
<div class="description-paragraph" id="p-0172" num="0189">Sequence identity may be determined by comparing a reference sequence or a subsequence of the reference sequence to a test sequence. The reference sequence and the test sequence are optimally aligned over an arbitrary number of residues termed a comparison window. In order to obtain optimal alignment, additions or deletions, such as gaps, may be introduced into the test sequence. The percent sequence identity is determined by determining the number of positions at which the same residue is present in both sequences and dividing the number of matching positions by the total length of the sequences in the comparison window and multiplying by 100 to give the percentage. In addition to the number of matching positions, the number and size of gaps is also considered in calculating the percentage sequence identity.</div>
<div class="description-paragraph" id="p-0173" num="0190">Sequence identity is typically determined using computer programs. A representative program is the BLAST (Basic Local Alignment Search Tool) program publicly accessible at the National Center for Biotechnology Information (NCBI, http://www.ncbi.nlm.nih.gov/). This program compares segments in a test sequence to sequences in a database to determine the statistical significance of the matches, then identifies and reports only those matches that that are more significant than a threshold level. A suitable version of the BLAST program is one that allows gaps, for example, version 2.X (Altschul, et al., <i>Nucleic Acids Res. </i>25(17):3389-402, 1997). Standard BLAST programs for searching nucleotide sequences (blastn) or protein (blastp) may be used. Translated query searches in which the query sequence is translated, i.e., from nucleotide sequence to protein (blastx) or from protein to nucleic acid sequence (tbblastn) may also be used as well as queries in which a nucleotide query sequence is translated into protein sequences in all 6 reading frames and then compared to an NCBI nucleotide database which has been translated in all six reading frames (tbblastx).</div>
<div class="description-paragraph" id="p-0174" num="0191">Additional suitable programs for identifying proteins with sequence identity to the proteins of the invention include, but are not limited to, PHI-BLAST (Pattern Hit Initiated BLAST, Zhang, et al., <i>Nucleic Acids Res. </i>26(17):3986-90, 1998) and PSI-BLAST (Position-Specific Iterated BLAST, Altschul, et al., <i>Nucleic Acids Res. </i>25(17):3389-402, 1997).</div>
<div class="description-paragraph" id="p-0175" num="0192">Programs may be used with default searching parameters. Alternatively, one or more search parameter may be adjusted. Selecting suitable search parameter values is within the abilities of one of ordinary skill in the art.</div>
<div class="description-paragraph" id="p-0176" num="0193">Preferred enzymes for use in the invention include those that are reduced, substantially reduced, or lacking in RNase H activity. Such enzymes that are reduced or substantially reduced in RNase H activity may be obtained by mutating, for example, the RNase H domain within the reverse transcriptase of interest, for example, by introducing one or more (e.g., one, two, three, four, five, ten, twelve, fifteen, twenty, thirty, etc.) point mutations, one or more (e.g., one, two, three, four, five, ten, twelve, fifteen, twenty, thirty, etc.) deletion mutations, and/or one or more (e.g., one, two, three, four, five, ten, twelve, fifteen, twenty, thirty, etc.) insertion mutations as described above. In some embodiments, the reverse transcriptase of the invention does not contain a modification or mutation in the RNase H domain and preferably does not contain a modification which reduces RNase H activity. In one aspect, the reverse transcriptase of the invention has 90%, 95%, or 100% of the RNase H activity compared to the corresponding wildtype reverse transcriptase.</div>
<div class="description-paragraph" id="p-0177" num="0194">By an enzyme “substantially reduced in RNase H activity” is meant that the enzyme has less than about 30%, less than about 25%, less than about 20%, more preferably less than about 15%, less than about 10%, less than about 7.5%, or less than about 5%, and most preferably less than about 5% or less than about 2%, of the RNase H activity of the corresponding wild-type or RNase H<sup>+</sup> enzyme, such as wild-type Moloney Murine Leukemia Virus (M-MLV), Avian Myeloblastosis Virus (AMV) or Rous Sarcoma Virus (RSV) reverse transcriptases. A reduction in RNase H activity means any reduction in the activity compared, for example, to the corresponding wild type or un-mutatated or un-modified reverse transcriptase.</div>
<div class="description-paragraph" id="p-0178" num="0195">Reverse transcriptases having reduced, substantially reduced, undetectable or lacking RNase H activity have been previously described (see U.S. Pat. Nos. 5,668,005, 6,063,608, and PCT Publication No. WO 98/47912). The RNase H activity of any enzyme may be determined by a variety of assays, such as those described, for example, in U.S. Pat. No. 5,244,797, in Kotewicz, M. L., et al., <i>Nucl. Acids Res. </i>16:265 (1988), in Gerard, G. F., et al., <i>FOCUS </i>14(5):91 (1992), in PCT publication number WO 98/47912, and in U.S. Pat. No. 5,668,005, the disclosures of all of which are fully incorporated herein by reference. Reverse transcriptases having no detectable RNase H activity or lacking RNase H activity by one or more of the described assays are particularly preferred.</div>
<div class="description-paragraph" id="p-0179" num="0196">Particularly preferred enzymes for use in the invention include, but are not limited to, M-MLV RNase H− reverse transcriptase, RSV RNase H− reverse transcriptase, AMV RNase H− reverse transcriptase, RAV RNase H− reverse transcriptase, MAV RNase H− reverse transcriptase and HIV RNase H− reverse transcriptase. It will be understood by one of ordinary skill, however, that any enzyme capable of producing a DNA molecule from a ribonucleic acid molecule (i.e., an enzyme having reverse transcriptase activity) that is reduced, substantially reduced, or lacking in RNase H activity may be equivalently used in the compositions, methods and kits of the invention.</div>
<div class="description-paragraph" id="p-0180" num="0197">Enzymes for use in the invention also include those in which terminal deoxynucleotidyl transferase (TdT) activity has been reduced, substantially reduced, or eliminated. Such enzymes that are reduced or substantially reduced in terminal deoxynucleotidyl transferase activity, or in which TdT activity has been eliminated, may be obtained by mutating, for example, amino acid residues within the reverse transcriptase of interest which are in close proximity or in contact with the template-primer, for example, by introducing one or more (e.g., one, two, three, four, five, ten, twelve, fifteen, twenty, thirty, etc.) point mutations, one or more deletion mutations, and/or one or more insertion mutations. Reverse transcriptases which exhibit decreased TdT activity are described in U.S. application Ser. No. 09/808,124, filed Mar. 15, 2001 (the entire disclosure of which is incorporated herein by reference), and include reverse transcriptases with one or more alterations at amino acid positions equivalent or corresponding to Y64, M289, F309, T197 and/or Y133 of M-MLV reverse transcriptase.</div>
<div class="description-paragraph" id="p-0181" num="0198">In one aspect, amino acid substitutions are made at one or more of the above identified positions (i.e., amino acid positions equivalent or corresponding to Y64, M289, F309, T197 or Y133 of M-MLV reverse transcriptase). Thus, the amino acids at these positions may be substituted with any other amino acid including Ala, Arg, Asn, Asp, Cys, Gln, Glu, Gly, His, Ile, Leu, Lys, Met, Phe, Pro, Ser, Thr, Trp, Tyr, and Val. Specific example of reverse transcriptases which exhibit reduced, substantially reduced, or eliminated TdT activity include M-MLV reverse transcriptases (e.g., S<smallcaps>UPER</smallcaps>S<smallcaps>CRIPT</smallcaps>™ II) in which (1) the phenylalanine residue at position 309 has been replaced with asparagine, (2) the threonine residue at position 197 has been replaced with either alanine or glutamic acid, and/or (3) the tyrosine residue at position 133 has been replaced with alanine.</div>
<div class="description-paragraph" id="p-0182" num="0199">Enzymes for use in the invention also include those that exhibit increased fidelity. Reverse transcriptases which exhibit increased fidelity are described in U.S. Appl. No. 60/189,454, filed Mar. 15, 2000, and U.S. application Ser. No. 09/808,124, filed Mar. 15, 2001 (the entire disclosures of each of which are incorporated herein by reference), and include reverse transcriptases with one or more alterations at positions equivalent or corresponding to those set out below in Table 2.</div>
<div class="description-paragraph" id="p-0183" num="0200">
<tables id="TABLE-US-00002" num="00002">
<patent-tables colsep="0" frame="none" rowsep="0">
<table align="left" class="description-table" cols="3" colsep="0" rowsep="0" width="100%">
<thead>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td" colspan="3" nameend="2" namest="offset" rowsep="1">TABLE 2</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td align="center" class="description-td" colspan="3" nameend="2" namest="offset" rowsep="1"> </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">RT</td>
<td class="description-td">Amino Acid</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td align="center" class="description-td" colspan="3" nameend="2" namest="offset" rowsep="1"> </td>
</tr>
</thead>
<tbody><tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">M-MLV </td>
<td class="description-td">Y64 (e.g., Y64W, Y64R), R116 </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">(e.g., R116M), K152 (e.g., K152R),</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">Q190 (e.g., Q190F), T197 (e.g., T197A, T197E), </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">V223 (e.g., V223H, V223I, V223F), </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">D124, H126, Y133 (e.g., Y133A, Y133H),</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">F309 (e.g., F309N, F309R)</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">AMV</td>
<td class="description-td">W25, R76, K110, Q149, T156, M182</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">RSV</td>
<td class="description-td">W25, R76, K110, Q149, T156, M182</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">HIV</td>
<td class="description-td">W24, R78, G112, Q151, A158, M184</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td align="center" class="description-td" colspan="3" nameend="2" namest="offset" rowsep="1"> </td>
</tr>
</tbody></table>
</patent-tables>
</tables>
</div>
<div class="description-paragraph" id="p-0184" num="0201">In some embodiments of the invention, amino acid substitutions are made at one or more of the above identified positions. Thus, the amino acids at these positions may be substituted with any other amino acid including Ala, Arg, Asn, Arg, Asp, Cys, Gln, Glu, Gly, His, Ile, Leu, Lys, Met, Phe, Pro, Ser, Thr, Trp, Tyr, and Val. Specific example of reverse transcriptases which exhibit increased fidelity include M-MLV reverse transcriptase in which (1) the valine residue at position 223 has been replaced with histidine, phenylalanine or isoleucine, (2) the arginine residue at position 116 has been replaced with methionine, (3) the lysine residue at position 152 has been replaced with arginine, (4) the glutamic acid residue at position 190 has been replaced with phenylalanine, (5) the threonine residue at position 197 has been replaced with alanine or glutamic acid, (6) the phenylalanine residue at position 309 has been replaced with asparagine or arginine, (7) the tyrosine residue at position 133 has been replaced with histidine or alanine, and/or (8) the tyrosine residue at position 64 has been replaced with tryptophan or arginine.</div>
<div class="description-paragraph" id="p-0185" num="0202">Thus, in specific embodiments, the invention includes reverse transcriptases which exhibit increased thermostability and, optionally, also exhibit one or more of the following characteristics: (1) reduced or substantially reduced RNase H activity, (2) reduced or substantially reduced TdT activity, and/or (3) increased fidelity.</div>
<div class="description-paragraph" id="p-0186" num="0203">The invention also relates to reverse transcriptase mutants, where the mutations or substitutions have been made in a recognized region of the reverse transcriptase enzyme. Such regions include, but are not limited to, the fingers, palm, thumb, α-helix H, β-sheet 10, and/or β-sheet 11 regions. Thus, the invention includes reverse transcriptases which exhibit increased thermostability (as well as other properties), as described elsewhere herein, and have one or more (e.g., one, two, three, four, five, ten, fifteen, etc.) mutations or modification in the hand domain and, more specifically, in one or more regions including the fingers, palm and/or thumb regions.</div>
<div class="description-paragraph" id="p-0187" num="0204">Polypeptides having reverse transcriptase activity for use in the invention may be isolated from their natural viral or bacterial sources according to standard procedures for isolating and purifying natural proteins that are well-known to one of ordinary skill in the art (see, e.g., Houts, G. E., et al., <i>J. Virol. </i>29:517 (1979); U.S. Pat. No. 5,668,005; and PCT publication number WO 98/47912). In addition, polypeptides having reverse transcriptase activity may be prepared by recombinant DNA techniques that are familiar to one of ordinary skill in the art (see, e.g., Kotewicz, M. L., et al., <i>Nucl. Acids Res. </i>16:265 (1988); Soltis, D. A., and Skalka, A. M., <i>Proc. Natl. Acad. Sci. USA </i>85:3372-3376 (1988)); U.S. Pat. No. 5,668,005; and PCT publication no. WO 98/47912.</div>
<div class="description-paragraph" id="p-0188" num="0205">In one aspect of the invention, mutant or modified reverse transcriptases are made by recombinant techniques. A number of cloned reverse transcriptase genes are available or may be obtained using standard recombinant techniques (see U.S. Pat. No. 5,668,005 and PCT Publication No. WO 98/47912).</div>
<div class="description-paragraph" id="p-0189" num="0206">To clone a gene or other nucleic acid molecule encoding a reverse transcriptase which will be modified in accordance with the invention, isolated DNA which contains the reverse transcriptase gene or open reading frame may be used to construct a recombinant DNA library. Any vector, well known in the art, can be used to clone the reverse transcriptase of interest. However, the vector used must be compatible with the host in which the recombinant vector will be transformed.</div>
<div class="description-paragraph" id="p-0190" num="0207">Prokaryotic vectors for constructing the plasmid library include plasmids such as those capable of replication in <i>E. coli </i>such as, for example, pBR322, ColE1, pSC101, pUC-vectors (pUC18, pUC19, etc.: In: <i>Molecular Cloning, A Laboratory Manual</i>, Cold Spring Harbor Laboratory Press, Cold Spring Harbor, N.Y. (1982); and Sambrook et al., In: Molecular Cloning A Laboratory Manual (2d ed.) Cold Spring Harbor Laboratory Press, Cold Spring Harbor, N.Y. (1989)). <i>Bacillus </i>plasmids include pC194, pUB110, pE194, pC221, pC217, etc. Such plasmids are disclosed by Glyczan, T. In: <i>The Molecular Biology Bacilli</i>, Academic Press, York (1982), 307-329. Suitable <i>Streptomyces </i>plasmids include pIJ101 (Kendall et al., <i>J. Bacteriol. </i>169:4177-4183 (1987)). <i>Pseudomonas </i>plasmids are reviewed by John et al., (<i>Rad. Insec. Dis. </i>8:693-704 (1986)), and Igaki, (<i>Jpn. J. Bacteriol. </i>33:729-742 (1978)). Broad-host range plasmids or cosmids, such as pCP13 (Darzins and Chakrabarty, <i>J. Bacteriol. </i>159:9-18 (1984)) can also be used for the present invention. Preferred vectors for cloning the genes and nucleic acid molecules of the present invention are prokaryotic vectors. Preferably, pBAD, pCP13 and pUC vectors are used to clone the genes of the present invention. Other suitable vectors are known to those skilled in the art and are commercially available, for example, from Invitrogen Corporation, Carlsbad, Calif.</div>
<div class="description-paragraph" id="p-0191" num="0208">Suitable host for cloning the reverse transcriptase genes and nucleic acid molecules of interest are prokaryotic hosts. One example of a prokaryotic host is <i>E. coli</i>. However, the desired reverse transcriptase genes and nucleic acid molecules of the present invention may be cloned in other prokaryotic hosts including, but not limited to, hosts in the genera <i>Escherichia, Bacillus, Streptomyces, Pseudomonas, Salmonella, Serratia</i>, and <i>Proteus</i>. Bacterial hosts of particular interest include <i>E. coli </i>DH10B, which may be obtained from Invitrogen Corporation (Carlsbad, Calif.).</div>
<div class="description-paragraph" id="p-0192" num="0209">Eukaryotic hosts for cloning and expression of the reverse transcriptase of interest include yeast, fungal, and mammalian cells. Expression of the desired reverse transcriptase in such eukaryotic cells may require the use of eukaryotic regulatory regions which include eukaryotic promoters. Cloning and expressing the reverse transcriptase gene or nucleic acid molecule in eukaryotic cells may be accomplished by well known techniques using well known eukaryotic vector systems.</div>
<div class="description-paragraph" id="p-0193" num="0210">Once a DNA library has been constructed in a particular vector, an appropriate host is transformed by well known techniques. Transformed cells are plated at a density to produce approximately 200-300 transformed colonies per petri dish. For selection of reverse transcriptase, colonies are then screened for the expression of a reverse transcriptase or a thermostable reverse transcriptase as described in the Examples below. Briefly, overnight cultures of individual transformant colonies are assayed directly for reverse transcriptase or thermostable reverse transcriptase activity and/or other desirable activities using a labeled deoxynucleotide and analyzed for the presence of labeled product. If thermostable reverse transcriptase activity and/or other desirable activity is detected, the mutant is sequenced to determine which amino acids maintained reverse transcriptase activity. The gene or nucleic acid molecule encoding a reverse transcriptase of the present invention can be cloned using techniques known to those in the art.</div>
<div class="description-paragraph" id="h-0012" num="0000">Modifications or Mutations of Polymerases</div>
<div class="description-paragraph" id="p-0194" num="0211">In accordance with the invention, one or more mutations may be made in any reverse transcriptase in order to increase the thermostability of the enzyme, or confer other properties described herein upon the enzyme, in accordance with the invention. Such mutations include point mutations, frame shift mutations, deletions and insertions. Preferably, one or more point mutations, resulting in one or more amino acid substitutions, are used to produce reverse transcriptases having enhanced or increased thermostability. In a preferred aspect of the invention, one or more mutations at positions equivalent or corresponding to position H204 (e.g., H204R) and/or T306 (e.g., T306K or T306R) of M-MLV reverse transcriptase may be made to produced the desired result in other reverse transcriptases of interest.</div>
<div class="description-paragraph" id="p-0195" num="0212">In specific embodiments, one or more mutations at positions equivalent or corresponding to position L52, Y64, R116, Y133, K152 Q190, T197, H204, V223, M289, T306 and/or F309 of M-MLV reverse transcriptase may be made to produced a desired result (e.g., increased thermostability, increased fidelity, decreased TdT activity, etc.). Thus, in specific embodiments, using amino acid positions of M-MLV reverse transcriptase as a frame of reference, proteins of the invention include reverse transcriptases (e.g., M-MLV reverse transcriptase, AMV reverse transcriptase, HIV reverse transcriptase, RSV reverse transcriptase, etc.) having one or more of the following alterations: L52P, Y64S, Y64W, Y64R, R116M, Y133A, Y133H, K152R, K152M, Q190F, T197R, T197E, T197A, T197K, H204R, V223H, V223F, V223I, M289L, T306K, T306R, F309R, and/or F309N, as well as compositions containing these proteins, nucleic acid molecules which encode these proteins, and host cells which contain these nucleic acid molecules.</div>
<div class="description-paragraph" id="p-0196" num="0213">Mutations in reverse transcriptases which alter thermostability properties of these proteins may be present in conjunction with alterations which either have little or no effect on activities normally associated with reverse transcriptases (e.g., RNase H activity, reverse transcriptase or polymerase activity, terminal deoxynucleotidyl transferase (TdTase) activity, etc.) or substantially alter one or more activities normally associated with reverse transcriptases. One example of a reverse transcriptase which has such a combination of mutations is a M-MLV reverse transcriptase which has the following alterations: K152M, V223H.</div>
<div class="description-paragraph" id="p-0197" num="0214">One mutation which has been shown to enhanced the fidelity of S<smallcaps>UPER</smallcaps>S<smallcaps>CRIPT</smallcaps>™ II (Invitrogen Corporation (Carlsbad, Calif.) Catalog No. 18064-022) is V223H (see U.S. Appl. No. 60/189,454, filed Mar. 15, 2000, U.S. application Ser. No. 09/808,124, filed Mar. 15, 2001, and PCT publication number WO 01/68895, the entire disclosures of each of which are incorporated herein by reference). However, the V223H alteration decreases the thermostability of this enzyme. One mutant was identified, K152M, which suppress the destabilizing effect of enzymes having the V223H mutation. Thus, the invention includes M-MLV reverse transcriptase which contain alterations at positions K152 and V223 and exhibit both increased fidelity and increased thermostability. Specific examples of such reverse transcriptases are those in which K152 is replaced with methionine and V223 is replaced with histidine. Other reverse transcriptases (e.g., AMV reverse transcriptase, HIV reverse transcriptase, RSV reverse transcriptase, etc.) with corresponding alterations are also included within the scope of the invention.</div>
<div class="description-paragraph" id="p-0198" num="0215">S<smallcaps>UPER</smallcaps>S<smallcaps>CRIPT</smallcaps>™ II is an RNase H− reverse transcriptase from M-MLV which has the following substitutions: D524G, E562Q, and D583N (see U.S. Pat. Nos. 5,017,492, 5,244,797, 5,405,776, 5,668,005, and 6,063,608, the entire disclosures of which are incorporated herein by reference). The invention includes reverse transcriptases that contain alterations, such as those reference positions (i.e., 524, 562, and/or 583) or equivalent positions.</div>
<div class="description-paragraph" id="p-0199" num="0216">One or more amino acid substitutions are made at one or more selected positions for any reverse transcriptase of interest. Thus, the amino acids at the selected positions may be substituted with any other amino acid including Ala, Arg, Asn, Asp, Cys, Gln, Glu, Gly, His, Ile, Leu, Lys, Met, Phe, Pro, Ser, Thr, Trp, Tyr, and Val. In some preferred embodiments, the selected amino acid will be a non-charged surface residue and will be replaced by a charged residue. In some preferred embodiments, the non-charged surface residue may be replaced by a positively charged amino acid (e.g., lysine or arginine). In one aspect, a charged residue will be replaced with an un-charged residue. In one aspect, a non-charged residue will be replaced with a negatively charged residue. In another aspect, a negatively charged residue will be replaced with a positively charged residue and/or a positively charged residue will be replaced with a negatively charged residue.</div>
<div class="description-paragraph" id="p-0200" num="0217">The corresponding positions of M-MLV reverse transcriptase identified above may be readily identified for other reverse transcriptases by one with skill in the art. Thus, given the defined region and the assays described in the present application, one with skill in the art can make one or a number of modifications which would result in increased thermostability and/or other desired features of any reverse transcriptase of interest. Residues to be modified in accordance with the present invention may include those listed in Table 1 above.</div>
<div class="description-paragraph" id="p-0201" num="0218">The nucleotide sequences for M-MLV reverse transcriptase (Shinnick et al., 1981, <i>Nature </i>293:543-548; Georgiadis et al., 1995, <i>Structure </i>3:879-892), AMV reverse transcriptase (Joliot et al., 1993, <i>Virology </i>195:812-819), RSV reverse transcriptase (Schwartz et al., 1983, <i>Cell </i>32:853-859), and HIV reverse transcriptase (Wong-Staal et al., 1985, <i>Nature </i>313:277-284) are known and are incorporated herein by reference in their entirety.</div>
<div class="description-paragraph" id="p-0202" num="0219">Preferably, oligonucleotide directed mutagenesis is used to create the mutant reverse transcriptases which allows for all possible classes of base pair changes at any determined site along the encoding DNA molecule.</div>
<div class="description-paragraph" id="h-0013" num="0000">Enhancing Expression of Reverse Transcriptases</div>
<div class="description-paragraph" id="p-0203" num="0220">To optimize expression of reverse transcriptases of the present invention, inducible or constitutive promoters are well known and may be used to express high levels of a reverse transcriptase structural gene in a recombinant host. Similarly, high copy number vectors, well known in the art, may be used to achieve high levels of expression. Vectors having an inducible high copy number may also be useful to enhance expression of the reverse transcriptases of the invention in a recombinant host.</div>
<div class="description-paragraph" id="p-0204" num="0221">To express the desired structural gene in a prokaryotic cell (such as <i>E. coli, B. subtilis, Pseudomonas</i>, etc.), it is preferable to operably link the desired structural gene to a functional prokaryotic promoter. However, the natural promoter of the reverse transcriptase gene may function in prokaryotic hosts allowing expression of the reverse transcriptase gene. Thus, the natural promoter or other promoters may be used to express the reverse transcriptase gene. Such other promoters that may be used to enhance expression include constitutive or regulatable (i.e., inducible or derepressible) promoters. Examples of constitutive promoters include the int promoter of bacteriophage λ, and the bla promoter of the β-lactamase gene of pBR322. Examples of inducible prokaryotic promoters include the major right and left promoters of bacteriophage λ (P<sub>R </sub>and P<sub>L</sub>), trp, recA, lacZ, lad, tet, gal, trc, ara BAD (Guzman, et al., 1995, <i>J. Bacteriol. </i>177(14):4121-4130) and tac promoters of <i>E. coli</i>. The <i>B. subtilis </i>promoters include α-amylase (Ulmanen et al., <i>J. Bacteriol </i>162:176-182 (1985)) and <i>Bacillus </i>bacteriophage promoters (Gryczan, T., In: <i>The Molecular Biology Of Bacilli</i>, Academic Press, New York (1982)). <i>Streptomyces </i>promoters are described by Ward et al., <i>Mol. Gen. Genet. </i>203:468-478 (1986)). Prokaryotic promoters are also reviewed by Glick, <i>J. Ind. Microbiol. </i>1:277-282 (1987); Cenatiempto, Y., <i>Biochimie </i>68:505-516 (1986); and Gottesman, <i>Ann. Rev. Genet. </i>18:415-442 (1984). Expression in a prokaryotic cell also requires the presence of a ribosomal binding site upstream of the gene-encoding sequence. Such ribosomal binding sites are disclosed, for example, by Gold et al., <i>Ann. Rev. Microbiol. </i>35:365404 (1981).</div>
<div class="description-paragraph" id="p-0205" num="0222">To enhance the expression of polymerases of the invention in a eukaryotic cell, well known eukaryotic promoters and hosts may be used. Enhanced expression of the polymerases may be accomplished in a prokaryotic host. One example of a prokaryotic host suitable for use with the present invention is <i>Escherichia coli. </i> </div>
<div class="description-paragraph" id="h-0014" num="0000">Isolation and Purification of Reverse Transcriptases</div>
<div class="description-paragraph" id="p-0206" num="0223">The enzyme(s) of the present invention is preferably produced by growth in culture of the recombinant host containing and expressing the desired reverse transcriptase gene. However, the reverse transcriptase of the present invention may be isolated from any strain, organism, or tissue which produces the reverse transcriptase of the present invention. Fragments of the reverse transcriptase are also included in the present invention. Such fragments include proteolytic fragments and fragments having reverse transcriptase activity. Such fragments may also be produced by cloning and expressing portions of the reverse transcriptase gene of interest, creating frame shift mutations and/or by adding one or more stop codons in the gene of interest for expression of a truncated protein or polypeptide. Preferably, polypeptides of the invention may be purified and/or isolated from a cell or organism expressing them, which may be a wild type cell or organism or a recombinant cell or organism. In some embodiments, such polypeptides may be substantially isolated from the cell or organism in which they are expressed.</div>
<div class="description-paragraph" id="p-0207" num="0224">Any nutrient that can be assimilated by a host containing the cloned reverse transcriptase gene may be added to the culture medium. Optimal culture conditions should be selected case by case according to the strain used and the composition of the culture medium. Antibiotics may also be added to the growth media to insure maintenance of vector DNA containing the desired gene to be expressed. Media formulations have been described in DSM or ATCC Catalogs and Sambrook et al., In: <i>Molecular Cloning, a Laboratory Manual </i>(2nd ed.), Cold Spring Harbor Laboratory Press, Cold Spring Harbor, N.Y. (1989).</div>
<div class="description-paragraph" id="p-0208" num="0225">Recombinant host cells producing the reverse transcriptases of this invention can be separated from liquid culture, for example, by centrifugation. In general, the collected microbial cells are dispersed in a suitable buffer, and then broken open by ultrasonic treatment or by other well known procedures to allow extraction of the enzymes by the buffer solution. After removal of cell debris by ultracentrifugation or centrifugation, the reverse transcriptases can be purified by standard protein purification techniques such as extraction, precipitation, chromatography, affinity chromatography, electrophoresis or the like. Assays to detect the presence of the reverse transcriptase during purification are well known in the art and can be used during conventional biochemical purification methods to determine the presence of these enzymes.</div>
<div class="description-paragraph" id="p-0209" num="0226">In some embodiments, reverse transcriptases of the present invention may be modified to contain an affinity tag in order to facilitate the purification of the reverse transcriptase. Suitable affinity tags are well known to those skilled in the art and include, but are not limited to, repeated sequences of amino acids such as six histidines, epitopes such as the hemagglutinin epitope and the myc epitope, and other amino acid sequences that permit the simplified purification of the reverse transcriptase.</div>
<div class="description-paragraph" id="p-0210" num="0227">The invention further relates to fusion proteins comprising (1) a protein, or fragment thereof, having one or more activity associated with a reverse transcriptase and (2) a tag (e.g., an affinity tag). In particular embodiments, the invention includes a reverse transcriptase (e.g., a thermostable reverse transcriptase) described herein having one or more (e.g., one, two, three, four, five, six, seven, eight, etc.) tags. These tags may be located, for example, (1) at the N-terminus, (2) at the C-terminus, or (3) at both the N-terminus and C-terminus of the protein, or a fragment thereof having one or more activities associated with a reverse transcriptase. A tag may also be located internally (e.g., between regions of amino acid sequence derived from a reverse transcriptase and/or attached to an amino acid side chain). For example, Ferguson et al., <i>Protein Sci. </i>7:1636-1638 (1998), describe a siderophore receptor, FhuA, from <i>Escherichia coli </i>into which an affinity tag (i.e., a hexahistidine sequence) was inserted. This tag was shown to function in purification protocols employing metal chelate affinity chromatography. Additional fusion proteins with internal tags are described in U.S. Pat. No. 6,143,524, the entire disclosure of which is incorporated herein by reference.</div>
<div class="description-paragraph" id="p-0211" num="0228">Tags used to prepare compositions of the invention may vary in length but will typically be from about 5 to about 500, from about 5 to about 100, from about 10 to about 100, from about 15 to about 100, from about 20 to about 100, from about 25 to about 100, from about 30 to about 100 from about 35 to about 100, from about 40 to about 100, from about 45 to about 100, from about 50 to about 100, from about 55 to about 100, from about 60 to about 100, from about 65 to about 100, from about 70 to about 100, from about 75 to about 100, from about 80 to about 100, from about 85 to about 100, from about 90 to about 100, from about 95 to about 100, from about 5 to about 80, from about 10 to about 80, from about 20 to about 80, from about 30 to about 80, from about 40 to about 80, from about 50 to about 80, from about 60 to about 80, from about 70 to about 80, from about 5 to about 60, from about 10 to about 60, from about 20 to about 60, from about 30 to about 60, from about 40 to about 60, from about 50 to about 60, from about 5 to about 40, from about 10 to about 40, from about 20 to about 40, from about 30 to about 40, from about 5 to about 30, from about 10 to about 30, from about 20 to about 30, from about 5 to about 25, from about 10 to about 25, or from about 15 to about 25 amino acid residues in length.</div>
<div class="description-paragraph" id="p-0212" num="0229">Tags used to prepare compositions of the invention include those which contribute to the thermostability of the fusion protein. Thus, these tags may be at least partly responsible, for example, for a particular protein (e.g., a protein having one or more activities of a reverse transcriptase activity) having increased thermostability. Examples of tags that enhance the thermostability of a reverse transcriptase (i.e., M-MLV reverse transcriptase) include, but are not limited to, tags having the following amino acid sequences: MGGSHHHHHHGMASMTGGQQMGRDLYDDDDKH, which corresponds to amino acids 1-32 of the sequence set forth in SEQ ID NO:2 and Table 3, and MASGTGGQQMGRDLYDDDDKH, (SEQ ID NO:3). Fragments of these tags may also be used in accordance with the invention (preferably those having 3 or more, 5 or more, 10 or more, or 15 or more amino acids) Thus, the invention includes, in part, reverse transcriptases, or fragments thereof that comprise tags and demonstrate enhanced thermostability. Using well known methods, one of skill in the art can attach one or more of above-mentioned tags to one or more RT enzymes, or fragments thereof having reverse transcriptase activity, to produce a thermostable RT enzyme or fragment thereof. Suitable RT enzymes include, but are not limited to, retroviral reverse transcriptase, retrotransposon reverse transcriptase, hepatitis B reverse transcriptase, cauliflower mosaic virus reverse transcriptase, bacterial reverse transcriptase, Tth DNA polymerase, Taq DNA polymerase (Saiki, R. K., et al., <i>Science </i>239:487-491 (1988); U.S. Pat. Nos. 4,889,818 and 4,965,188), Tne DNA polymerase (PCT Publication No. WO 96/10640), Tma DNA polymerase (U.S. Pat. No. 5,374,553) and mutants, fragments, variants or derivatives thereof (see, e.g., commonly owned U.S. Pat. Nos. 5,948,614 and 6,015,668, which are incorporated by reference herein in their entireties). Reverse transcriptases for use in the invention also include retroviral reverse transcriptases such as M-MLV reverse transcriptase, AMV reverse transcriptase, RSV reverse transcriptase, RAV reverse transcriptase, MAV reverse transcriptase, and generally ASLV reverse transcriptases.</div>
<div class="description-paragraph" id="p-0213" num="0230">Tags used in the practice of the invention may serve any number of purposes and a number of tags may be added to impart one or more different functions to the reverse transcriptase of the invention. For example, tags may (1) contribute to protein-protein interactions both internally within a protein and with other protein molecules, (2) make the protein amenable to particular purification methods, (3) enable one to identify whether the protein is present in a composition; or (4) give the protein other functional characteristics.</div>
<div class="description-paragraph" id="p-0214" num="0231">Examples of tags which may be used in the practice of the invention include metal binding domains (e.g., a poly-histidine segments such as a three, four, five, six, or seven histidine region), immunoglobulin binding domains (e.g., (1) Protein A; (2) Protein G; (3) T cell, B cell, and/or Fc receptors; and/or (4) complement protein antibody-binding domain); sugar binding domains (e.g., a maltose binding domain, chitin-binding domain); and detectable domains (e.g., at least a portion of beta-galactosidase). Fusion proteins may contain one or more tags such as those described above. Typically, fusion proteins that contain more than one tag will contain these tags at one terminus or both termini (i.e., the N-terminus and the C-terminus) of the reverse transcriptase, although one or more tags may be located internally instead of or in addition to those present at termini. Further, more than one tag may be present at one terminus, internally and/or at both termini of the reverse transcriptase. For example, three consecutive tags could be linked end-to-end at the N-terminus of the reverse transcriptase. The invention further include compositions and reaction mixture which contain the above fusion proteins, as well as methods for preparing these fusion proteins, nucleic acid molecules (e.g., vectors) which encode these fusion proteins and recombinant host cells which contain these nucleic acid molecules. The invention also includes methods for using these fusion proteins as described elsewhere herein (e.g., methods for reverse transcribing nucleic acid molecules).</div>
<div class="description-paragraph" id="p-0215" num="0232">Tags which enable one to identify whether the fusion protein is present in a composition include, for example, tags which can be used to identify the protein in an electrophoretic gel. A number of such tags are known in the art and include epitopes and antibody binding domains which can be used for Western blots.</div>
<div class="description-paragraph" id="p-0216" num="0233">The amino acid composition of the tags for use in the present invention may vary. In some embodiments, a tag may contain from about 1% to about 5% amino acids that have a positive charge at physiological pH, e.g., lysine, arginine, and histidine, or from about 5% to about 10% amino acids that have a positive charge at physiological pH, or from about 10% to about 20% amino acids that have a positive charge at physiological pH, or from about 10% to about 30% amino acids that have a positive charge at physiological pH, or from about 10% to about 50% amino acids that have a positive charge at physiological pH, or from about 10% to about 75% amino acids that have a positive charge at physiological pH. In some embodiments, a tag may contain from about 1% to about 5% amino acids that have a negative charge at physiological pH, e.g., aspartic acid and glutamic acid, or from about 5% to about 10% amino acids that have a negative charge at physiological pH, or from about 10% to about 20% amino acids that have a negative charge at physiological pH, or from about 10% to about 30% amino acids that have a negative charge at physiological pH, or from about 10% to about 50% amino acids that have a negative charge at physiological pH, or from about 10% to about 75% amino acids that have a negative charge at physiological pH. In some embodiments, a tag may comprise a sequence of amino acids that contains two or more contiguous charged amino acids that may be the same or different and may be of the same or different charge. For example, a tag may contain a series (e.g., two, three, four, five, six, ten etc.) of positively charged amino acids that may be the same or different. A tag may contain a series (e.g., two, three, four, five, six, ten etc.) of negatively charged amino acids that may be the same or different. In some embodiments, a tag may contain a series (e.g., two, three, four, five, six, ten etc.) of alternating positively charged and negatively charged amino acids that may be the same or different (e.g., positive, negative, positive, negative, etc.). Any of the above-described series of amino acids (e.g., positively charged, negatively charged or alternating charge) may comprise one or more neutral polar or non-polar amino acids (e.g., two, three, four, five, six, ten etc.) spaced between the charged amino acids. Such neutral amino acids may be evenly distributed through out the series of charged amino acids (e.g., charged, neutral, charged, neutral) or may be unevenly distributed throughout the series (e.g., charged, a plurality of neutral, charged, neutral, a plurality of charged, etc.). In some embodiments, tags to be attached to the polypeptides of the invention may have an overall charge at physiological pH (e.g., positive charge or negative charge). The size of the overall charge may vary, for example, the tag may contain a net plus one, two, three, four, five, etc. or may possess a net negative one, two, three, four, five, etc. The present invention also relates to reverse transcriptases (e.g., thermostable reverse transcriptases) comprising one or more of the above-described tag sequences, vectors encoding such reverse transcriptases, host cells reaction mixture, compositions and reaction mixtures comprising such reverse transcriptases, as well as kits comprising containers containing such reverse transcriptases.</div>
<div class="description-paragraph" id="p-0217" num="0234">In some embodiments, it may be desirable to remove all or a portion of a tag sequence from a fusion protein comprising a tag sequence and a sequence having reverse transcriptase (RT) activity. In embodiments of this type, one or more amino acids forming a cleavage site, e.g., for a protease enzyme, may be incorporated into the primary sequence of the fusion protein. The cleavage site may be located such that cleavage at the site may remove all or a portion of the tag sequence from the fusion protein. In some embodiments, the cleavage site may be located between the tag sequence and the sequence having RT activity such that all of the tag sequence is removed by cleavage with a protease enzyme that recognizes the cleavage site. Examples of suitable cleavage sites include, but are not limited to, the Factor Xa cleavage site having the sequence Ile-Glu-Gly-Arg (SEQ ID NO:4), which is recognized and cleaved by blood coagulation factor Xa, and the thrombin cleavage site having the sequence Leu-Val-Pro-Arg (SEQ ID NO:5), which is recognized and cleaved by thrombin. Other suitable cleavage sites are known to those skilled in the art and may be used in conjunction with the present invention.</div>
<div class="description-paragraph" id="p-0218" num="0235">The reverse transcriptases of the invention preferably have specific activities (e.g., RNA-directed DNA polymerase activity and/or RNase H activity) greater than about 5 units/mg, more preferably greater than about 50 units/mg, still more preferably greater than about 100 units/mg, 250 units/mg, 500 units/mg, 1000 units/mg, 5000 units/mg or 10,000 units/mg, and most preferably greater than about 15,000 units/mg, greater than about 16,000 units/mg, greater than about 17,000 units/mg, greater than about 18,000 units/mg, greater than about 19,000 units/mg and greater than about 20,000 units/mg. In some embodiments, the reverse transcriptases of the present invention may have specific activities greater than about 50,000 units/mg, greater than about 100,000 units/mg, greater than about 150,000 units/mg, greater than about 200,000 units/mg, greater than about 250,000 units/mg and greater than about 300,000 units/mg. Preferred ranges of specific activities for the reverse transcriptases of the invention include a specific activity from about 5 units/mg to about 750,000 units/mg, a specific activity from about 5 units/mg to about 500,000 units/mg, from about 5 units/mg to about 300,000 units/mg, a specific activity of from about 50 units/mg to about 750,000 units/mg, a specific activity from about 100 units/mg to about 750,000 units/mg, a specific activity from about 250 units/mg to about 750,000 units/mg, a specific activity from about 500 units/mg to about 750,000 units/mg, a specific activity from about 1000 units/mg to about 750,000 units/mg, a specific activity from about 5000 units/mg to about 750,000 units/mg, a specific activity from about 10,000 units/mg to about 750,000 units/mg, a specific activity from about 25,000 units/mg to about 750,000 units/mg, a specific activity from about 100 units/mg to about 500 units/mg, a specific activity from about 100 units/mg to about 400 units/mg, and a specific activity from about 200 units/mg to about 500 units/mg. Other preferred ranges of specific activities include a specific activity of from about 200,000 units/mg to about 350,000 units/mg, a specific activity from about 225,000 units/mg to about 300,000 units/mg, a specific activity from about 250,000 units/mg to about 300,000 units/mg, a specific activity of from about 200,000 units/mg to about 750,000 units/mg, a specific activity of from about 200,000 units/mg to about 500,000 units/mg, a specific activity of from about 200,000 units/mg to about 400,000 units/mg, a specific activity of from about 250,000 units/mg to about 750,000 units/mg, a specific activity of from about 250,000 units/mg to about 500,000 units/mg, and a specific activity of from about 250,000 units/mg to about 400,000 units/mg. Preferably, the lower end of the specific activity range may vary from 50, 100, 200, 300, 400, 500, 700, 900, 1,000, 5,000, 10,000, 20,000, 30,000, 35,000, 40,000, 45,000, 50,000, 55,000, 60,000, 65,000, 70,000, 75,000, and 80,000 units/mg, while the upper end of the range may vary from 750,000, 650,000, 600,000, 550,000, 500,000, 450,000, 400,000, 350,000, 300,000, 250,000, 200,000, 150,000, 140,000, 130,000, 120,000, 110,000, 100,000, and 90,000 units/mg. Specific activity may be determined using enzyme assays and protein assays well known in the art. DNA polymerase assays and RNase H assays are described, for example, in U.S. Pat. No. 5,244,797 and WO 98/47912. In some embodiments of the present invention, the specific activity of the thermostable reverse transcriptase prepared in accordance with the present invention may be higher than the specific activity of a non-thermostable reverse transcriptase. In some embodiments, the specific activity of the thermostable reverse transcriptase may be 5%, 10,%, 15%, 20%, 25%, 30%, 40%, 50%, 60%, 70%, 80%, 90%, 100% or more higher than the specific activity of a corresponding non-thermostable reverse transcriptase. In some preferred embodiments, the specific activity of the thermostable reverse transcriptase according to the present invention may be 30% or more higher than the specific activity of a corresponding non-thermostable reverse transcriptase. In accordance with the invention, specific activity is a measurement of the enzymatic activity (in units) of the protein or enzyme relative to the total amount of protein or enzyme used in a reaction. The measurement of a specific activity may be determined by standard techniques well-known to one of ordinary skill in the art.</div>
<div class="description-paragraph" id="p-0219" num="0236">The reverse transcriptases of the invention may be used to make nucleic acid molecules from one or more templates. Such methods comprise mixing one or more nucleic acid templates (e.g., mRNA, and more preferably a population of mRNA molecules) with one or more of the reverse transcriptases of the invention and incubating the mixture under conditions sufficient to make one or more nucleic acid molecules complementary to all or a portion of the one or more nucleic acid templates.</div>
<div class="description-paragraph" id="p-0220" num="0237">The invention also relates to methods for the amplification of one or more nucleic acid molecules comprising mixing one or more nucleic acid templates with one of the reverse transcriptases of the invention, and incubating the mixture under conditions sufficient to amplify one or more nucleic acid molecules complementary to all or a portion of the one or more nucleic acid templates. Such amplification methods may further comprise the use of one or more DNA polymerases and may be employed as in standard RT-PCR reactions.</div>
<div class="description-paragraph" id="p-0221" num="0238">The invention also concerns methods for the sequencing of one or more nucleic acid molecules comprising (a) mixing one or more nucleic acid molecules to be sequenced with one or more primer nucleic acid molecules, one or more reverse transcriptases of the invention, one or more nucleotides and one or more terminating agents; (b) incubating the mixture under conditions sufficient to synthesize a population of nucleic acid molecules complementary to all or a portion of the one or more nucleic acid molecules to be sequenced; and (c) separating the population of nucleic acid molecules to determine the nucleotide sequence of all or a portion of the one or more nucleic acid molecules to be sequenced.</div>
<div class="description-paragraph" id="p-0222" num="0239">The invention also concerns nucleic acid molecules produced by such methods (which may be full-length cDNA molecules), vectors (particularly expression vectors) comprising these nucleic acid molecules and host cells comprising these vectors and nucleic acid molecules.</div>
<div class="description-paragraph" id="h-0015" num="0000">Sources of DNA Polymerase</div>
<div class="description-paragraph" id="p-0223" num="0240">A variety of DNA polymerases are useful in accordance with the present invention. Such polymerases include, but are not limited to, <i>Thermus thermophilus </i>(Tth) DNA polymerase, <i>Thermus aquaticus </i>(Taq) DNA polymerase, <i>Thermotoga neapolitana </i>(Tne) DNA polymerase, <i>Thermotoga maritima </i>(Tma) DNA polymerase, <i>Thermococcus litoralis </i>(Tli or VENT™) DNA polymerase, <i>Thermococcus kodakaraensis </i>KOD1 DNA Polymerase, <i>Pyrococcus furiosis </i>(Pfu) DNA polymerase, <i>Pyrococcus </i>species GB-D (DEEPVENT™) DNA polymerase, <i>Pyrococcus woosii </i>(Pwo) DNA polymerase, <i>Bacillus sterothermophilus </i>(Bst) DNA polymerase, <i>Bacillus caldophilus </i>(Bea) DNA polymerase, <i>Sulfolobus acidocaldarius </i>(Sac) DNA polymerase, <i>Thermoplasma acidophilum </i>(Tac) DNA polymerase, <i>Thermus flavus </i>(Tfl/Tub) DNA polymerase, <i>Thermus ruber </i>(Tru) DNA polymerase, <i>Thermus brockianus </i>(DYNAZYME™) DNA polymerase, <i>Methanobacterium thermoautotrophicum </i>(Mth) DNA polymerase, <i>Mycobacterium </i>spp. DNA polymerase (Mtb, Mlep), and mutants, variants and derivatives thereof.</div>
<div class="description-paragraph" id="p-0224" num="0241">DNA polymerases used in accordance with the invention may be any enzyme that can synthesize a DNA molecule from a nucleic acid template, typically in the 5′ to 3′ direction. Such polymerases may be mesophilic or thermophilic, but are preferably thermophilic. Mesophilic polymerases include T5 DNA polymerase, T7 DNA polymerase, Klenow fragment DNA polymerase, DNA polymerase III, and the like. Preferred DNA polymerases are thermostable DNA polymerases such as Taq, Tne, Tma, Pfu, VENT™, DEEPVENT™, Tth and mutants, variants and derivatives thereof (U.S. Pat. Nos. 5,436,149; 5,512,462; PCT Publication No. WO 92/06188; PCT Publication No. WO 92/06200; PCT Publication No. WO 96/10640; Barnes, W. M., <i>Gene </i>112:29-35 (1992); Lawyer, F. C., et al., <i>PCR Meth. Appl. </i>2:275-287 (1993); Flaman, J.-M., et al., <i>Nucl. Acids Res. </i>22(15):3259-3260 (1994)). For amplification of long nucleic acid molecules (e.g., nucleic acid molecules longer than about 3-5 Kb in length), at least two DNA polymerases (one substantially lacking 3′ exonuclease activity and the other having 3′ exonuclease activity) are typically used. See U.S. Pat. Nos. 5,436,149; 5,512,462; Barnes, W. M., <i>Gene </i>112:29-35 (1992); PCT Publication No. WO 98/06736; and commonly owned, co-pending U.S. patent application Ser. No. 08/801,720, filed Feb. 14, 1997, the disclosures of all of which are incorporated herein in their entireties. Examples of DNA polymerases substantially lacking in 3′ exonuclease activity include, but are not limited to, Taq, Tne(exo<sup>−</sup>), Tma, Pfu(exo<sup>−</sup>), Pwo and Tth DNA polymerases, and mutants, variants and derivatives thereof. Nonlimiting examples of DNA polymerases having 3′ exonuclease activity include Pfu, DEEPVENT™ and Tli/VENT™ and mutants, variants and derivatives thereof.</div>
<div class="description-paragraph" id="h-0016" num="0000">Formulation of Compositions and Reaction Mixtures</div>
<div class="description-paragraph" id="p-0225" num="0242">To form the compositions of the present invention, one or more reverse transcriptases are preferably admixed in a buffered salt solution. One or more DNA polymerases and/or one or more nucleotides, and/or one or more primers may optionally be added to make the compositions of the invention. More preferably, the enzymes are provided at working concentrations in stable buffered salt solutions. The terms “stable” and “stability” as used herein generally mean the retention by a composition, such as an enzyme composition, of at least 70%, preferably at least 80%, and most preferably at least 90%, of the original enzymatic activity (in units) after the enzyme or composition containing the enzyme has been stored for about one week at a temperature of about 4° C., about two to six months at a temperature of about −20° C., and about six months or longer at a temperature of about −80° C. As used herein, the term “working concentration” means the concentration of an enzyme that is at or near the optimal concentration used in a solution to perform a particular function (such as reverse transcription of nucleic acids).</div>
<div class="description-paragraph" id="p-0226" num="0243">The water used in forming the compositions of the present invention is preferably distilled, deionized and sterile filtered (through a 0.1-0.2 micrometer filter), and is free of contamination by DNase and RNase enzymes. Such water is available commercially, for example from Sigma Chemical Company (Saint Louis, Mo.), or may be made as needed according to methods well known to those skilled in the art.</div>
<div class="description-paragraph" id="p-0227" num="0244">In addition to the enzyme components, the present compositions preferably comprise one or more buffers and cofactors necessary for synthesis of a nucleic acid molecule such as a cDNA molecule. Particularly preferred buffers for use in forming the present compositions are the acetate, sulfate, hydrochloride, phosphate or free acid forms of Tris-(hydroxymethyl)aminomethane (TRIS®), although alternative buffers of the same approximate ionic strength and pKa as TRIS® may be used with equivalent results. In addition to the buffer salts, cofactor salts such as those of potassium (preferably potassium chloride or potassium acetate) and magnesium (preferably magnesium chloride or magnesium acetate) are included in the compositions. Addition of one or more carbohydrates and/or sugars to the compositions and/or synthesis reaction mixtures may also be advantageous, to support enhanced stability of the compositions and/or reaction mixtures upon storage. Preferred such carbohydrates or sugars for inclusion in the compositions and/or synthesis reaction mixtures of the invention include, but are not limited to, sucrose, trehalose, glycerol, and the like. Furthermore, such carbohydrates and/or sugars may be added to the storage buffers for the enzymes used in the production of the enzyme compositions and kits of the invention. Such carbohydrates and/or sugars are commercially available from a number of sources, including Sigma (St. Louis, Mo.).</div>
<div class="description-paragraph" id="p-0228" num="0245">It is often preferable to first dissolve the buffer salts, cofactor salts and carbohydrates or sugars at working concentrations in water and to adjust the pH of the solution prior to addition of the enzymes. In this way, the pH-sensitive enzymes will be less subject to acid- or alkaline-mediated inactivation during formulation of the present compositions.</div>
<div class="description-paragraph" id="p-0229" num="0246">To formulate the buffered salts solution, a buffer salt which is preferably a salt of Tris(hydroxymethyl)aminomethane (TRIS®), and most preferably the hydrochloride salt thereof, is combined with a sufficient quantity of water to yield a solution having a TRIS® concentration of 5-150 millimolar, preferably 10-60 millimolar, and most preferably about 20-60 millimolar. To this solution, a salt of magnesium (preferably either the chloride or acetate salt thereof) or other divalent cation, may be added to provide a working concentration thereof of 1-10 millimolar, preferably 1.5-8.0 millimolar, and most preferably about 3-7.5 millimolar. A salt of potassium (preferably a chloride or acetate salt of potassium), or other monovalent cation (e.g., Na), may also be added to the solution, at a working concentration of 10-100 millimolar and most preferably about 75 millimolar. A reducing agent, such as dithiothreitol, may be added to the solution, preferably at a final concentration of about 1-100 mM, more preferably a concentration of about 5-50 mM or about 7.5-20 mM, and most preferably at a concentration of about 10 mM. Preferred concentrations of carbohydrates and/or sugars for inclusion in the compositions of the invention range from about 5% (w/v) to about 30% (w/v), from about 7.5% (w/v) to about 25% (w/v), from about 10% (w/v) to about 25% (w/v), from about 10% (w/v) to about 20% (w/v), and preferably from about 10% (w/v) to about 15% (w/v). A small amount of a salt of ethylenediaminetetraacetate (EDTA), such as disodium EDTA, may also be added (preferably about 0.1 millimolar), although inclusion of EDTA does not appear to be essential to the function or stability of the compositions of the present invention. After addition of all buffers and salts, this buffered salt solution is mixed well until all salts are dissolved, and the pH is adjusted using methods known in the art to a pH value of from about 7.4 to about 9.2, preferably from about 8.0 to about 9.0, and most preferably to about 8.4.</div>
<div class="description-paragraph" id="p-0230" num="0247">To these buffered salt solutions, the enzymes (reverse transcriptases and/or DNA polymerases) are added to produce the compositions of the present invention. M-MLV reverse transcriptases are preferably added at a working concentration in the solution of from about 1,000 to about 50,000 units per milliliter, from about 2,000 to about 30,000 units per milliliter, from about 2,500 to about 25,000 units per milliliter, from about 3,000 to about 22,500 units per milliliter, from about 4,000 to about 20,000 units per milliliter, and most preferably at a working concentration of from about 5,000 to about 20,000 units per milliliter. In some embodiments, a reverse transcriptases of the invention (e.g., an M-MLV reverse transcriptase) may be added at a working concentration described above in a first strand reaction mixture (e.g., a reaction to reverse transcribe an mRNA molecule) and/or in a couple RT/PCR. A suitable concentration of a reverse transcriptase of the invention for these reactions may be from about 5,000 units/ml to about 50,000 units/ml, from about 5,000 units/ml to about 40,000 units/ml, from about 5,000 units/ml to about 30,000 units/ml, or from about 5,000 units/ml to about 20,000 units/ml of reverse transcriptase. A reaction may be conducted in a volume of 20 μl to 50 μl and such a reaction may contain 50 units, 100, units, 200 units, 300 units, 400 units or more of a reverse transcriptase of the invention. Those skilled in the art will appreciate that adding additional reverse transcriptase may allow increased synthesis of the first strand (e.g., the DNA strand complementary to the mRNA strand) at the expense of increased enzyme usage. The skilled artisan can determine without undue experimentation the amount of a reverse transcriptase of the invention to add to a reaction in order to produce a desired amount of product at an acceptable expense.</div>
<div class="description-paragraph" id="p-0231" num="0248">AMV reverse transcriptases, RSV reverse transcriptases and HIV reverse transcriptases, including those of the invention described above, are preferably added at a working concentration in the solution of from about 100 to about 5000 units per milliliter, from about 125 to about 4000 units per milliliter, from about 150 to about 3000 units per milliliter, from about 200 to about 2500 units per milliliter, from about 225 to about 2000 units per milliliter, and most preferably at a working concentration of from about 250 to about 1000 units per milliliter. The enzymes in the thermophilic DNA polymerase group (Taq, Tne, Tma, Pfu, VENT, DEEPVENT, Tth and mutants, variants and derivatives thereof) are preferably added at a working concentration in the solution of from about 100 to about 1000 units per milliliter, from about 125 to about 750 units per milliliter, from about 150 to about 700 units per milliliter, from about 200 to about 650 units per milliliter, from about 225 to about 550 units per milliliter, and most preferably at a working concentration of from about 250 to about 500 units per milliliter. The enzymes may be added to the solution in any order, or may be added simultaneously.</div>
<div class="description-paragraph" id="p-0232" num="0249">The compositions of the invention may further comprise one or more nucleotides, which are preferably deoxynucleoside triphosphates (dNTPs) or dideoxynucleoside triphosphates (ddNTPs). The dNTP components of the present compositions serve as the “building blocks” for newly synthesized nucleic acids, being incorporated therein by the action of the polymerases, and the ddNTPs may be used in sequencing methods according to the invention. Examples of nucleotides suitable for use in the present compositions include, but are not limited to, dUTP, dATP, dTTP, dCTP, dGTP, dITP, 7-deaza-dGTP, α-thio-dATP, α-thio-dTTP, α-thio-dGTP, thio-dCTP, ddUTP, ddATP, ddTTP, ddCTP, ddGTP, ddITP, 7-deaza-ddGTP, α-thio-ddATP, α-thio-ddTTP, α-thio-ddGTP, α-thio-ddCTP or derivatives thereof, all of which are available commercially from sources including Invitrogen Corporation (Carlsbad, Calif.), New England BioLabs (Beverly, Mass.) and Sigma Chemical Company (Saint Louis, Mo.). The nucleotides may be unlabeled, or they may be detectably labeled by coupling them by methods known in the art with radioisotopes (e.g., <sup>3</sup>H, <sup>14</sup>C, <sup>32</sup>P or <sup>35</sup>S) vitamins (e.g., biotin), fluorescent moieties (e.g., fluorescein, rhodamine, Texas Red, or phycoerythrin), chemiluminescent labels (e.g., using the PHOTO-GENE™ or ACES™ chemiluminescence systems, available commercially from Invitrogen Corporation (Carlsbad, Calif.)), dioxigenin and the like. Labeled nucleotides may also be obtained commercially, for example from Invitrogen Corporation (Carlsbad, Calif.) or Sigma Chemical Company (Saint Louis, Mo.). In the present compositions, the nucleotides are added to give a working concentration of each nucleotide of about 10-4000 micromolar, about 50-2000 micromolar, about 100-1500 micromolar, or about 200-1200 micromolar, and most preferably a concentration of about 1000 micromolar.</div>
<div class="description-paragraph" id="p-0233" num="0250">To reduce component deterioration, storage of the reagent compositions is preferably at about 4° C. for up to one day, or most preferably at −20° C. for up to one year.</div>
<div class="description-paragraph" id="p-0234" num="0251">In another aspect, the compositions and reverse transcriptases of the invention may be prepared and stored in dry form in the presence of one or more carbohydrates, sugars, or synthetic polymers. Preferred carbohydrates, sugars or polymers for the preparation of dried compositions or reverse transcriptases include, but are not limited to, sucrose, trehalose, and polyvinylpyrrolidone (PVP) or combinations thereof. See, e.g., U.S. Pat. Nos. 5,098,893, 4,891,319, and 5,556,771, the disclosures of which are entirely incorporated herein by reference. Such dried compositions and enzymes may be stored at various temperatures for extended times without significant deterioration of enzymes or components of the compositions of the invention. Preferably, the dried reverse transcriptases or compositions are stored at 4° C. or at −20° C.</div>
<div class="description-paragraph" id="p-0235" num="0252">The invention further includes reaction solutions for reverse transcribing nucleic acid molecules, as well as reverse transcription methods employing such reaction solutions and product nucleic acid molecules produced using such methods. In many instances, reaction solutions of the invention will contain one or more of the following components: (1) one or more buffering agent (e.g., sodium phosphate, sodium acetate, 2-(N-moropholino)-ethanesulfonic acid (MES), tris-(hydroxymethyl)aminomethane (Tris), 3-(cyclohexylamino)-2-hydroxy-1-propanesulfonic acid (CAPS), citrate, N-2-hydroxyethylpiperazine-N′-2-ethanesulfonic acid (HEPES), acetate, 3-(N-morpholino)prpoanesulfonic acid (MOPS), N-tris(hydroxymethyl)methyl-3-aminopropanesulfonio acid (TAPS), etc.), (2) one or more monovalent cationic salt (e.g., NaCl, KCl, etc.), (3) one or more divalent cationic salt (e.g., MnCl<sub>2</sub>, MgCl<sub>2</sub>, MgSO<sub>4</sub>, CaCl<sub>2</sub>, etc.), (4) one or more reducing agent (e.g., dithiothreitol, (3-mercaptoethanol, etc.), (5) one or more ioninc or non-ionic detergent (e.g., TRITON X-100™, N<smallcaps>ONIDET </smallcaps>P40™, sodium dodecyl sulphate, etc.), (6) one or more DNA polymerase inhibitor (e.g., Actinomycin D, etc.), (7) nucleotides (e.g., dNTPs, such as dGTP, dATP, dCTP, dTTP, etc.), (8) RNA to be reverse transcribed and/or amplified, (9) one or more RNase inhibitor (e.g., RN<smallcaps>ASE</smallcaps>O<smallcaps>UT</smallcaps>™, Invitrogen Corporation, Carlsbad, Calif., catalog number 10777-019 etc.), (10) a reverse transcriptase (e.g., a reverse transcriptase of the invention, and/or (11) one or more diluent (e.g., water). Other components and/or constituents (e.g., primers, DNA polymerases, etc.) may also be present in reaction solutions.</div>
<div class="description-paragraph" id="p-0236" num="0253">The concentration of the buffering agent in the reaction solutions of the invention will vary with the particular buffering agent used. Typically, the working concentration (i.e., the concentration in the reaction mixture) of the buffering agent will be from about 5 mM to about 500 mM (e.g., about 10 mM, about 15 mM, about 20 mM, about 25 mM, about 30 mM, about 35 mM, about 40 mM, about 45 mM, about 50 mM, about 55 mM, about 60 mM, about 65 mM, about 70 mM, about 75 mM, about 80 mM, about 85 mM, about 90 mM, about 95 mM, about 100 mM, from about 5 mM to about 500 mM, from about 10 mM to about 500 mM, from about 20 mM to about 500 mM, from about 25 mM to about 500 mM, from about 30 mM to about 500 mM, from about 40 mM to about 500 mM, from about 50 mM to about 500 mM, from about 75 mM to about 500 mM, from about 100 mM to about 500 mM, from about 25 mM to about 50 mM, from about 25 mM to about 75 mM, from about 25 mM to about 100 mM, from about 25 mM to about 200 mM, from about 25 mM to about 300 mM, etc.). When Tris (e.g., Tris-HCl) is used, the Tris working concentration will typically be from about 5 mM to about 100 mM, from about 5 mM to about 75 mM, from about 10 mM to about 75 mM, from about 10 mM to about 60 mM, from about 10 mM to about 50 mM, from about 25 mM to about 50 mM, etc.</div>
<div class="description-paragraph" id="p-0237" num="0254">The final pH of solutions of the invention will generally be set and maintained by buffering agents present in reaction solutions of the invention. The pH of reaction solutions of the invention, and hence reaction mixtures of the invention, will vary with the particular use and the buffering agent present but will often be from about pH 5.5 to about pH 9.0 (e.g., about pH 6.0, about pH 6.5, about pH 7.0, about pH 7.1, about pH 7.2, about pH 7.3, about pH 7.4, about pH 7.5, about pH 7.6, about pH 7.7, about pH 7.8, about pH 7.9, about pH 8.0, about pH 8.1, about pH 8.2, about pH 8.3, about pH 8.4, about pH 8.5, about pH 8.6, about pH 8.7, about pH 8.8, about pH 8.9, about pH 9.0, from about pH 6.0 to about pH 8.5, from about pH 6.5 to about pH 8.5, from about pH 7.0 to about pH 8.5, from about pH 7.5 to about pH 8.5, from about pH 6.0 to about pH 8.0, from about pH 6.0 to about pH 7.7, from about pH 6.0 to about pH 7.5, from about pH 6.0 to about pH 7.0, from about pH 7.2 to about pH 7.7, from about pH 7.3 to about pH 7.7, from about pH 7.4 to about pH 7.6, from about pH 7.0 to about pH 7.4, from about pH 7.6 to about pH 8.0, from about pH 7.6 to about pH 8.5, from about pH 7.7 to about pH 8.5, from about pH 7.9 to about pH 8.5, from about pH 8.0 to about pH 8.5, from about pH 8.2 to about pH 8.5, from about pH 8.3 to about pH 8.5, from about pH 8.4 to about pH 8.5, from about pH 8.4 to about pH 9.0, from about pH 8.5 to about pH 9.0, etc.)</div>
<div class="description-paragraph" id="p-0238" num="0255">As indicated, one or more monovalent cationic salts (e.g., NaCl, KCl, etc.) may be included in reaction solutions of the invention. In many instances, salts used in reaction solutions of the invention will dissociate in solution to generate at least one species which is monovalent (e.g., Na+, K+, etc.) When included in reaction solutions of the invention, salts will often be present either individually or in a combined concentration of from about 0.5 mM to about 500 mM (e.g., about 1 mM, about 2 mM, about 3 mM, about 5 mM, about 10 mM, about 12 mM, about 15 mM, about 17 mM, about 20 mM, about 22 mM, about 23 mM, about 24 mM, about 25 mM, about 27 mM, about 30 mM, about 35 mM, about 40 mM, about 45 mM, about 50 mM, about 55 mM, about 60 mM, about 64 mM, about 65 mM, about 70 mM, about 75 mM, about 80 mM, about 85 mM, about 90 mM, about 95 mM, about 100 mM, about 120 mM, about 140 mM, about 150 mM, about 175 mM, about 200 mM, about 225 mM, about 250 mM, about 275 mM, about 300 mM, about 325 mM, about 350 mM, about 375 mM, about 400 mM, from about 1 mM to about 500 mM, from about 5 mM to about 500 mM, from about 10 mM to about 500 mM, from about 20 mM to about 500 mM, from about 30 mM to about 500 mM, from about 40 mM to about 500 mM, from about 50 mM to about 500 mM, from about 60 mM to about 500 mM, from about 65 mM to about 500 mM, from about 75 mM to about 500 mM, from about 85 mM to about 500 mM, from about 90 mM to about 500 mM, from about 100 mM to about 500 mM, from about 125 mM to about 500 mM, from about 150 mM to about 500 mM, from about 200 mM to about 500 mM, from about 10 mM to about 100 mM, from about 10 mM to about 75 mM, from about 10 mM to about 50 mM, from about 20 mM to about 200 mM, from about 20 mM to about 150 mM, from about 20 mM to about 125 mM, from about 20 mM to about 100 mM, from about 20 mM to about 80 mM, from about 20 mM to about 75 mM, from about 20 mM to about 60 mM, from about 20 mM to about 50 mM, from about 30 mM to about 500 mM, from about 30 mM to about 100 mM, from about 30 mM to about 70 mM, from about 30 mM to about 50 mM, etc.).</div>
<div class="description-paragraph" id="p-0239" num="0256">As indicated, one or more divalent cationic salts (e.g., MnCl<sub>2</sub>, MgCl<sub>2</sub>, MgSO<sub>4</sub>, CaCl<sub>2</sub>, etc.) may be included in reaction solutions of the invention. In many instances, salts used in reaction solutions of the invention will dissociate in solution to generate at least one species which is monovalent (e.g., Mg<sup>++</sup>, Mn<sup>++</sup>, Ca<sup>++</sup>, etc.) When included in reaction solutions of the invention, salts will often be present either individually or in a combined concentration of from about 0.5 mM to about 500 mM (e.g., about 1 mM, about 2 mM, about 3 mM, about 4 mM, about 5 mM, about 6 mM, about 7 mM, about 8 mM, about 9 mM, about 10 mM, about 12 mM, about 15 mM, about 17 mM, about 20 mM, about 22 mM, about 23 mM, about 24 mM, about 25 mM, about 27 mM, about 30 mM, about 35 mM, about 40 mM, about 45 mM, about 50 mM, about 55 mM, about 60 mM, about 64 mM, about 65 mM, about 70 mM, about 75 mM, about 80 mM, about 85 mM, about 90 mM, about 95 mM, about 100 mM, about 120 mM, about 140 mM, about 150 mM, about 175 mM, about 200 mM, about 225 mM, about 250 mM, about 275 mM, about 300 mM, about 325 mM, about 350 mM, about 375 mM, about 400 mM, from about 1 mM to about 500 mM, from about 5 mM to about 500 mM, from about 10 mM to about 500 mM, from about 20 mM to about 500 mM, from about 30 mM to about 500 mM, from about 40 mM to about 500 mM, from about 50 mM to about 500 mM, from about 60 mM to about 500 mM, from about 65 mM to about 500 mM, from about 75 mM to about 500 mM, from about 85 mM to about 500 mM, from about 90 mM to about 500 mM, from about 100 mM to about 500 mM, from about 125 mM to about 500 mM, from about 150 mM to about 500 mM, from about 200 mM to about 500 mM, from about 10 mM to about 100 mM, from about 10 mM to about 75 mM, from about 10 mM to about 50 mM, from about 20 mM to about 200 mM, from about 20 mM to about 150 mM, from about 20 mM to about 125 mM, from about 20 mM to about 100 mM, from about 20 mM to about 80 mM, from about 20 mM to about 75 mM, from about 20 mM to about 60 mM, from about 20 mM to about 50 mM, from about 30 mM to about 500 mM, from about 30 mM to about 100 mM, from about 30 mM to about 70 mM, from about 30 mM to about 50 mM, etc.).</div>
<div class="description-paragraph" id="p-0240" num="0257">When included in reaction solutions of the invention, reducing agents (e.g., dithiothreitol, β-mercaptoethanol, etc.) will often be present either individually or in a combined concentration of from about 0.1 mM to about 50 mM (e.g., about 0.2 mM, about 0.3 mM, about 0.5 mM, about 0.7 mM, about 0.9 mM, about 1 mM, about 2 mM, about 3 mM, about 4 mM, about 5 mM, about 6 mM, about 10 mM, about 12 mM, about 15 mM, about 17 mM, about 20 mM, about 22 mM, about 23 mM, about 24 mM, about 25 mM, about 27 mM, about 30 mM, about 35 mM, about 40 mM, about 45 mM, about 50 mM, from about 0.1 mM to about 50 mM, from about 0.5 mM to about 50 mM, from about 1 mM to about 50 mM, from about 2 mM to about 50 mM, from about 3 mM to about 50 mM, from about 0.5 mM to about 20 mM, from about 0.5 mM to about 10 mM, from about 0.5 mM to about 5 mM, from about 0.5 mM to about 2.5 mM, from about 1 mM to about 20 mM, from about 1 mM to about 10 mM, from about 1 mM to about 5 mM, from about 1 mM to about 3.4 mM, from about 0.5 mM to about 3.0 mM, from about 1 mM to about 3.0 mM, from about 1.5 mM to about 3.0 mM, from about 2 mM to about 3.0 mM, from about 0.5 mM to about 2.5 mM, from about 1 mM to about 2.5 mM, from about 1.5 mM to about 2.5 mM, from about 2 mM to about 3.0 mM, from about 2.5 mM to about 3.0 mM, from about 0.5 mM to about 2 mM, from about 0.5 mM to about 1.5 mM, from about 0.5 mM to about 1.1 mM, from about 5.0 mM to about 10 mM, from about 5.0 mM to about 15 mM, from about 5.0 mM to about 20 mM, from about 10 mM to about 15 mM, from about 10 mM to about 20 mM, etc.).</div>
<div class="description-paragraph" id="p-0241" num="0258">Reaction solutions of the invention may also contain one or more ioninc or non-ionic detergent (e.g., T<smallcaps>RITON </smallcaps>X-100™, N<smallcaps>ONIDET </smallcaps>P40™, sodium dodecyl sulphate, etc.). When included in reaction solutions of the invention, detergents will often be present either individually or in a combined concentration of from about 0.01% to about 5.0% (e.g., about 0.01%, about 0.02%, about 0.03%, about 0.04%, about 0.05%, about 0.06%, about 0.07%, about 0.08%, about 0.09%, about 0.1%, about 0.15%, about 0.2%, about 0.3%, about 0.5%, about 0.7%, about 0.9%, about 1%, about 2%, about 3%, about 4%, about 5%, from about 0.01% to about 5.0%, from about 0.01% to about 4.0%, from about 0.01% to about 3.0%, from about 0.01% to about 2.0%, from about 0.01% to about 1.0%, from about 0.05% to about 5.0%, from about 0.05% to about 3.0%, from about 0.05% to about 2.0%, from about 0.05% to about 1.0%, from about 0.1% to about 5.0%, from about 0.1% to about 4.0%, from about 0.1% to about 3.0%, from about 0.1% to about 2.0%, from about 0.1% to about 1.0%, from about 0.1% to about 0.5%, etc.). For example, reaction solutions of the invention may contain T<smallcaps>RITON </smallcaps>X-100™ at a concentration of from about 0.01% to about 2.0%, from about 0.03% to about 1.0%, from about 0.04% to about 1.0%, from about 0.05% to about 0.5%, from about 0.04% to about 0.6%, from about 0.04% to about 0.3%, etc.</div>
<div class="description-paragraph" id="p-0242" num="0259">Reaction solutions of the invention may also contain one or more DNA polymerase inhibitor (e.g., Actinomycin D, etc.). When included in reaction solutions of the invention, such inhibitors will often be present either individually or in a combined concentration of from about 0.1 μg/ml to about 100 μg/ml (e.g., about 0.1 μg/ml, about 0.2 μg/ml, about 0.3 μg/ml, about 0.4 μg/ml, about 0.5 μg/ml, about 0.6 μg/ml, about 0.7 μg/ml, about 0.8 μg/ml, about 0.9 μg/ml, about 1.0 μg/ml, about 1.1 μg/ml, about 1.3 μg/ml, about 1.5 μg/ml, about 1.7 μg/ml, about 2.0 μg/ml, about 2.5 μg/ml, about 3.5 μg/ml, about 5.0 μg/ml, about 7.5 μg/ml, about 10 μg/ml, about 15 μg/ml, about 20 μg/ml, about 25 μg/ml, about 30 μg/ml, about 35 μg/ml, about 40 μg/ml, about 50 μg/ml, about 60 μg/ml, about 70 μg/ml, about 80 μg/ml, about 90 μg/ml, about 100 μg/ml, from about 0.5 μg/ml to about 30 μg/ml, from about 0.75 μg/ml to about 30 μg/ml, from about 1.0 μg/ml to about 30 μg/ml, from about 2.0 μg/ml to about 30 μg/ml, from about 3.0 μg/ml to about 30 μg/ml, from about 4.0 μg/ml to about 30 μg/ml, from about 5.0 μg/ml to about 30 μg/ml, from about 7.5 μg/ml to about 30 μg/ml, from about 10 μg/ml to about 30 μg/ml, from about 15 μg/ml to about 30 μg/ml, from about 0.5 μg/ml to about 20 μg/ml, from about 0.5 μg/ml to about 10 μg/ml, from about 0.5 μg/ml to about 5 μg/ml, from about 0.5 μg/ml to about 2 μg/ml, from about 0.5 μg/ml to about 1 μg/ml, from about 1 μg/ml to about 10 μg/ml, from about 1 μg/ml to about 5 μg/ml, from about 1 μg/ml to about 2 μg/ml, from about 1 μg/ml to about 100 μg/ml, from about 10 μg/ml to about 100 μg/ml, from about 20 μg/ml to about 100 μg/ml, from about 40 μg/ml to about 100 μg/ml, from about 30 μg/ml to about 80 μg/ml, from about 30 μg/ml to about 70 μg/ml, from about 40 μg/ml to about 60 μg/ml, from about 40 μg/ml to about 70 μg/ml, from about 40 μg/ml to about 80 μg/ml, etc.).</div>
<div class="description-paragraph" id="p-0243" num="0260">In many instances, nucleotides (e.g., dNTPs, such as dGTP, dATP, dCTP, dTTP, etc.) will be present in reaction mixtures of the invention. Typically, individually nucleotides will be present in concentrations of from about 0.05 mM to about 50 mM (e.g., about 0.07 mM, about 0.1 mM, about 0.15 mM, about 0.18 mM, about 0.2 mM, about 0.3 mM, about 0.5 mM, about 0.7 mM, about 0.9 mM, about 1 mM, about 2 mM, about 3 mM, about 4 mM, about 5 mM, about 6 mM, about 10 mM, about 12 mM, about 15 mM, about 17 mM, about 20 mM, about 22 mM, about 23 mM, about 24 mM, about 25 mM, about 27 mM, about 30 mM, about 35 mM, about 40 mM, about 45 mM, about 50 mM, from about 0.1 mM to about 50 mM, from about 0.5 mM to about 50 mM, from about 1 mM to about 50 mM, from about 2 mM to about 50 mM, from about 3 mM to about 50 mM, from about 0.5 mM to about 20 mM, from about 0.5 mM to about 10 mM, from about 0.5 mM to about 5 mM, from about 0.5 mM to about 2.5 mM, from about 1 mM to about 20 mM, from about 1 mM to about 10 mM, from about 1 mM to about 5 mM, from about 1 mM to about 3.4 mM, from about 0.5 mM to about 3.0 mM, from about 1 mM to about 3.0 mM, from about 1.5 mM to about 3.0 mM, from about 2 mM to about 3.0 mM, from about 0.5 mM to about 2.5 mM, from about 1 mM to about 2.5 mM, from about 1.5 mM to about 2.5 mM, from about 2 mM to about 3.0 mM, from about 2.5 mM to about 3.0 mM, from about 0.5 mM to about 2 mM, from about 0.5 mM to about 1.5 mM, from about 0.5 mM to about 1.1 mM, from about 5.0 mM to about 10 mM, from about 5.0 mM to about 15 mM, from about 5.0 mM to about 20 mM, from about 10 mM to about 15 mM, from about 10 mM to about 20 mM, etc.). The combined nucleotide concentration, when more than one nucleotides is present, can be determined by adding the concentrations of the individual nucleotides together. When more than one nucleotide is present in reaction solutions of the invention, the individual nucleotides may not be present in equimolar amounts. Thus, a reaction solution may contain, for example, 1 mM dGTP, 1 mM dATP, 0.5 mM dCTP, and 1 mM dTTP.</div>
<div class="description-paragraph" id="p-0244" num="0261">RNA will typically be present in reaction solutions of the invention. In most instances, RNA will be added to the reaction solution shortly prior to reverse transcription. Thus, reaction solutions may be provided without RNA. This will typically be the case when reaction solutions are provided in kits. RNA, when present in reaction solutions will often be present in a concentration of 1 picogram to 100 μg/20 μl reaction mixture (e.g., about 1 picogram/20 μl, about 10 picograms/20 μl, about 50 picograms/20 μl, about 100 picograms/20 μl, about 200 picograms/20 μl, about 10 picograms/20 μl, about 500 picograms/20 μl, about 800 picograms/20 μl, about 1.0 nanogram/20 μl, about 5.0 nanograms/20 μl, about 10 nanograms/20 μl, about 25 nanograms/20 μl, about 50 nanograms/20 μl, about 75 nanograms/20 μl, about 100 nanograms/20 μl, about 150 nanograms/20 μl, about 250 nanograms/20 μl, about 400 nanograms/20 μl, about 500 nanograms/20 μl, about 750 nanograms/20 μl, about 1.0 μg/20 μl, about 5.0 μg/20 μl, about 10 μg/20 μl, about 20 μg/20 μl, about 30 μg/20 μl, about 40 μg/20 μl, about 50 μg/20 μl, about 70 μg/20 μl, about 85 μg/20 μl, about 100 μg/20 μl, from about 10 picograms/20 μl to about 100 μg/20 μl, from about 10 picograms/20 μl to about 100 μg/20 μl, from about 100 picograms/20 μl to about 100 μg/20 μl, from about 1.0 nanograms/20 μl to about 100 μg/20 μl, from about 100 nanograms/20 μl to about 100 μg/20 μl, from about 10 picograms/20 μl to about 10 μg/20 μl, from about 10 picograms/20 μl to about 5 μg/20 μl, from about 100 nanograms/20 μl to about 5 μg/20 μl, from about 1 μg/20 μl to about 10 μg/20 μl, from about 1 μg/20 μl to about 5 μg/20 μl, from about 100 nanograms/20 μl to about 1 μg/20 μl, from about 500 nanograms/20 μl to about 5 μg/20 μl, etc.). As one skilled in the art would recognize, different reverse transcription reactions may be performed in volumes other than 20 μl. In such instances, the total amount of RNA present will vary with the volume used. Thus, the above amounts are provided as examples of the amount of RNA/20 μl of reaction solution.</div>
<div class="description-paragraph" id="p-0245" num="0262">Reverse transcriptases (e.g., reverse transcriptases of the invention) may also be present in reaction solutions. When present, reverse transcriptases, will often be present in a concentration which results in about 0.01 to about 1,000 units of reverse transcriptase activity/μl (e.g., about 0.01 unit/μl, about 0.05 unit/μl, about 0.1 unit/μl, about 0.2 unit/μl, about 0.3 unit/μl, about 0.4 unit/μl, about 0.5 unit/μl, about 0.7 unit/μl, about 1.0 unit/μl, about 1.5 unit/μl, about 2.0 unit/μl, about 2.5 unit/μl, about 5.0 unit/μl, about 7.5 unit/μl, about 10 unit/μl, about 20 unit/μl, about 25 unit/μl, about 50 unit/μl, about 100 unit/μl, about 150 unit/μl, about 200 unit/μl, about 250 unit/μl, about 350 unit/μl, about 500 unit/μl, about 750 unit/μl, about 1,000 unit/μl, from about 0.1 unit/μl to about 1,000 unit/μl, from about 0.2 unit/μl to about 1,000 unit/μl, from about 1.0 unit/μl to about 1,000 unit/μl, from about 5.0 unit/μl to about 1,000 unit/μl, from about 10 unit/μl to about 1,000 unit/μl, from about 20 unit/μl to about 1,000 unit/μl, from about 50 unit/μl to about 1,000 unit/μl, from about 100 unit/μl to about 1,000 unit/μl, from about 200 unit/μl to about 1,000 unit/μl, from about 400 unit/μl to about 1,000 unit/μl, from about 500 unit/μl to about 1,000 unit/μl, from about 0.1 unit/μl to about 300 unit/μl, from about 0.1 unit/μl to about 200 unit/μl, from about 0.1 unit/μl to about 100 unit/μl, from about 0.1 unit/μl to about 50 unit/μl, from about 0.1 unit/μl to about 10 unit/μl, from about 0.1 unit/μl to about 5.0 unit/μl, from about 0.1 unit/μl to about 1.0 unit/μl, from about 0.2 unit/μl to about 0.5 unit/μl, etc.</div>
<div class="description-paragraph" id="p-0246" num="0263">Reaction solutions of the invention may be prepared as concentrated solutions (e.g., 5× solutions) which are diluted to a working concentration for final use. With respect to a 5× reaction solution, a 5:1 dilution is required to bring such a 5× solution to a working concentration. Reaction solutions of the invention may be prepared, for examples, as a 2×, a 3×, a 4×, a 5×, a 6×, a 7×, a 8×, a 9×, a 10×, etc. solutions. One major limitation on the fold concentration of such solutions is that, when compounds reach particular concentrations in solution, precipitation occurs. Thus, concentrated reaction solutions will generally be prepared such that the concentrations of the various components are low enough so that precipitation of buffer components will not occur. As one skilled in the art would recognize, the upper limit of concentration which is feasible for each solution will vary with the particular solution and the components present.</div>
<div class="description-paragraph" id="p-0247" num="0264">In many instances, reaction solutions of the invention will be provided in sterile form. Sterilization may be performed on the individual components of reaction solutions prior to mixing or on reaction solutions after they are prepared. Sterilization of such solutions may be performed by any suitable means including autoclaving or ultrafiltration.</div>
<div class="description-paragraph" id="h-0017" num="0000">Labeling Nucleic Acids</div>
<div class="description-paragraph" id="p-0248" num="0265">In general, the invention provides, in part, compositions for use in reverse transcription of a nucleic acid molecule to produce labeled nucleic acid molecules. Such compositions may comprise one or more reverse transcriptases (e.g., single subunit and/or multi-subunit RTs). The enzymes in these compositions are preferably present in working concentrations and have RNase H activity or are reduced or substantially reduced or lacking in RNase H activity, although mixtures of enzymes, some having RNase H activity and some reduced or substantially reduced or lacking in RNase H activity, may be used in the compositions of the invention. Preferred reverse transcriptases include M-MLV reverse transcriptase, RSV reverse transcriptase, AMV reverse transcriptase, RAV reverse transcriptase, MAV reverse transcriptase and HIV reverse transcriptase or other ASLV reverse transcriptases.</div>
<div class="description-paragraph" id="p-0249" num="0266">The invention is also directed to methods for reverse transcription of one or more nucleic acid molecules comprising mixing one or more nucleic acid templates, which is preferably RNA or messenger RNA (mRNA) and more preferably a population of mRNA molecules, with one or more polypeptides having reverse transcriptase activity and incubating the mixture under conditions sufficient to make one or more labeled nucleic acid molecules complementary to all or a portion of the one or more templates. To make the nucleic acid molecule or molecules complementary to the one or more templates, at least one primer (e.g., an oligo(dT) primer) and one or more nucleotides (a portion of which are preferably labeled, most preferably fluorescently labeled) are used for nucleic acid synthesis. Nucleic acid templates suitable for reverse transcription according to this aspect of the invention include any nucleic acid molecule, particularly those derived from a prokaryotic or eukaryotic cell. Such cells may include normal cells, diseased cells, transformed cells, established cells, progenitor cells, precursor cells, fetal cells, embryonic cells, bacterial cells, yeast cells, animal cells (including human cells), avian cells, plant cells and the like, or tissue isolated from a plant or an animal (e.g., human, cow, pig, mouse, sheep, horse, monkey, canine, feline, rat, rabbit, bird, fish, insect, etc.). Such nucleic acid molecules may also be isolated from viruses. In some embodiments, methods of the invention result in the direct labeling of a nucleic acid molecule by incorporation of a labeled nucleotide (e.g., a nucleotide containing a fluorescent label). In other methods, nucleic acid molecules are indirectly labeled by first, incorporating a nucleotide analog containing a reactive functionality to produce a nucleic acid containing one or more reactive functionalities. The nucleic acid containing reactive functionalities may subsequently be reacted with a molecule containing a label that reacts with the functionality to attach the label to the nucleic acid molecule. The reaction may be result in covalent attachment of all or a portion of the label-containing molecule to the nucleic acid molecule (e.g., chemical coupling). In some embodiments, amine-modified NTPs (e.g., amino allyl-dUTP/UTP) are incorporated during reverse transcription. Amino allyl-NTPs are incorporated with similar efficiency as unmodified NTPs during enzymatic reactions such as reverse transcription. The amine functionality is then coupled with a dye using standard techniques. Kits for indirect labeling of cDNA are commercially available from, for example, Ambion, Inc., Austin, Tex. In some embodiments, the label-containing molecule may be non-covalently bound to the reactive functionality. For example, the reactive functionality may be a biotin moiety and the label-containing molecule may be a labeled (e.g., fluorescently labeled) avidin or streptavidin molecule.</div>
<div class="description-paragraph" id="p-0250" num="0267">The invention also provides labeled nucleic acid molecules produced according to the above-described methods. Such labeled nucleic acid molecules may be single or double stranded and are useful as detection probes. Depending on the labeled nucleotide(s) used during synthesis, the labeled molecules may contain one or a number of labels. Where multiple labels are used, the molecules may comprise a number of the same or different labels. Thus, one type or multiple different labeled nucleotides may be used during synthesis of nucleic acid molecules to provide for the labeled nucleic acid molecules of the invention. Such labeled nucleic acid molecules will thus comprise one or more (e.g., two, three, four, five, six, seven, eight, nine, ten, etc.) labeled nucleotides (which may be the same or different).</div>
<div class="description-paragraph" id="p-0251" num="0268">Labeled nucleic acid molecules produced by methods of the invention may either (1) comprise a particular numbers of labeled nucleotides or (2) a particular percentage of the nucleotides present in the nucleic acid molecule may be labeled. In either instance, the concentration of labeled nucleotides present in the reaction mixture may be adjusted, with respect to un-labeled nucleotides, such that product nucleic acid molecules are produced which contain (1) a particular number of labeled nucleotides or (2) a particular percentage of label nucleotides as compared to un-labeled nucleotides. In particular instances, from about 0.1% to about 20%, from about 0.1% to about 15%, from about 0.1% to about 10%, from about 0.1% to about 5.0%, from about 0.1% to about 2.5%, from about 0.1% to about 1.5%, from about 0.1% to about 1.0%, from about 0.1% to about 0.5%, from about 2.0% to about 20%, from about 4.0% to about 20%, from about 0.5% to about 10%, from about 0.5% to about 5%, from about 0.5% to about 2.0%, or from about 0.5% to about 1.0% of the total nucleotides present in product nucleic acid molecules are labeled. As one skilled in the art would recognize, the actual number of labeled nucleotides, and thus the percentage of nucleotides which are labeled, present in individual molecules of a population will typically differ. In other words, different members of populations of nucleic acid molecules will typically contain different numbers of labeled nucleotides with the overall average of labeled nucleotides present in each product molecule varying with a number of factors (e.g., the ratio of labeled to un-labeled nucleotides present in the reaction mixture). In most instances, at least 85%, at least 90%, at least 95%, or at least 99% of the individual product nucleic acid molecules in the population will contain labeled nucleotides which fall within a range set out above.</div>
<div class="description-paragraph" id="p-0252" num="0269">In accordance with the invention, the amount of labeled product is preferably measured based on percent incorporation of the label of interest into synthesized product as may be determined by one skilled in the art, although other means of measuring the amount or efficiency of labeling of product will be recognized by one of ordinary skill in the art. The invention provides for enhanced or increased percent incorporation of labeled nucleotide during synthesis of a nucleic acid molecule from a template, preferably during synthesis of one or more cDNA molecules from RNA. According to the invention, such enhancement or increase in percent incorporation is preferably about equal to or greater than a 2-fold, a 5-fold, a 10-fold, a 15-fold, a 20-fold, a 25-fold, a 30-fold, a 40-fold or a 50-fold increase or enhancement in percent incorporation compared to a standard reverse transcriptase.</div>
<div class="description-paragraph" id="p-0253" num="0270">The invention also provides kits for use in accordance with the invention. Such kits comprise a carrier means, such as a box or carton, having in close confinement therein one or more container means, such as vials, tubes, bottles and the like, wherein the kit comprises, in the same or different containers, one or more reverse transcriptases. The kits of the invention may also comprise, in the same or different containers, one or more DNA polymerases, one or more primers, one or more suitable buffers and/or one or more nucleotides (such as deoxynucleoside triphosphates (dNTPs) and preferably labeled dNTPs (e.g., fluorescently labeled dNTPs)).</div>
<div class="description-paragraph" id="p-0254" num="0271">In some embodiments, the RTs used in the invention comprise two or more subunits (or derivatives, variants, fragments or mutants thereof) and preferably comprise two subunits (e.g., a dimer or heterodimer). Two subunit reverse transcriptases typically have an α and a β subunit forming a dimer, although any form or combination of subunits (and derivatives, variants or mutants of such subunits) may be used. Such combinations may include αβ, ββ, αα and the like. Preferred two subunit RTs for use in the invention include RSV RT, AMV RT, AEV RT, RAV RT, HIV RT and MAV RT, or other ASLV RTs, or mutants, variants or derivatives thereof. In a particular embodiment, AMV RT and/or RSV RT is used in accordance with the invention. Preferred single subunit RTs include M-MLV reverse transcriptase.</div>
<div class="description-paragraph" id="h-0018" num="0000">Production/Sources of cDNA Molecules</div>
<div class="description-paragraph" id="p-0255" num="0272">In accordance with the invention, cDNA molecules (single-stranded or double-stranded) may be prepared from a variety of nucleic acid template molecules. Preferred nucleic acid molecules for use in the present invention include single-stranded or double-stranded DNA and RNA molecules, as well as double-stranded DNA:RNA hybrids. More preferred nucleic acid molecules include messenger RNA (mRNA), transfer RNA (tRNA) and ribosomal RNA (rRNA) molecules, although mRNA molecules are the preferred template according to the invention.</div>
<div class="description-paragraph" id="p-0256" num="0273">The nucleic acid molecules that are used to prepare cDNA molecules according to the methods of the present invention may be prepared synthetically according to standard organic chemical synthesis methods that will be familiar to one of ordinary skill. More preferably, the nucleic acid molecules may be obtained from natural sources, such as a variety of cells, tissues, organs or organisms. Cells that may be used as sources of nucleic acid molecules may be prokaryotic (bacterial cells, including but not limited to those of species of the genera <i>Escherichia, Bacillus, Serratia, Salmonella, Staphylococcus, Streptococcus, Clostridium, Chlamydia, Neisseria, Treponema, Mycoplasma, Borrelia, Legionella, Pseudomonas, Mycobacterium, Helicobacter, Erwinia, Agrobacterium, Rhizobium, Xanthomonas </i>and <i>Streptomyces</i>) or eukaryotic (including fungi (especially yeasts), plants, protozoans and other parasites, and animals including insects (particularly <i>Drosophila </i>spp. cells), nematodes (particularly <i>Caenorhabditis elegans </i>cells), and mammals (particularly human cells)).</div>
<div class="description-paragraph" id="p-0257" num="0274">Mammalian somatic cells that may be used as sources of nucleic acids include blood cells (reticulocytes and leukocytes), endothelial cells, epithelial cells, neuronal cells (from the central or peripheral nervous systems), muscle cells (including myocytes and myoblasts from skeletal, smooth or cardiac muscle), connective tissue cells (including fibroblasts, adipocytes, chondrocytes, chondroblasts, osteocytes and osteoblasts) and other stromal cells (e.g., macrophages, dendritic cells, Schwann cells). Mammalian germ cells (spermatocytes and oocytes) may also be used as sources of nucleic acids for use in the invention, as may the progenitors, precursors and stem cells that give rise to the above somatic and germ cells. Also suitable for use as nucleic acid sources are mammalian tissues or organs such as those derived from brain, kidney, liver, pancreas, blood, bone marrow, muscle, nervous, skin, genitourinary, circulatory, lymphoid, gastrointestinal and connective tissue sources, as well as those derived from a mammalian (including human) embryo or fetus.</div>
<div class="description-paragraph" id="p-0258" num="0275">Any of the above prokaryotic or eukaryotic cells, tissues and organs may be normal, diseased, transformed, established, progenitors, precursors, fetal or embryonic. Diseased cells may, for example, include those involved in infectious diseases (caused by bacteria, fungi or yeast, viruses (including AIDS, HIV, HTLV, herpes, hepatitis and the like) or parasites), in genetic or biochemical pathologies (e.g., cystic fibrosis, hemophilia, Alzheimer's disease, muscular dystrophy or multiple sclerosis) or in cancerous processes. Transformed or established animal cell lines may include, for example, COS cells, CHO cells, VERO cells, BHK cells, HeLa cells, HepG2 cells, K562 cells, 293 cells, L929 cells, F9 cells, and the like. Other cells, cell lines, tissues, organs and organisms suitable as sources of nucleic acids for use in the present invention will be apparent to one of ordinary skill in the art.</div>
<div class="description-paragraph" id="p-0259" num="0276">Once the starting cells, tissues, organs or other samples are obtained, nucleic acid molecules (such as mRNA) may be isolated therefrom by methods that are well-known in the art (See, e.g., Maniatis, T., et al., <i>Cell </i>15:687-701 (1978); Okayama, H., and Berg, P., <i>Mol. Cell. Biol. </i>2:161-170 (1982); Gubler, U., and Hoffman, B. J., <i>Gene </i>25:263-269 (1983)). The nucleic acid molecules thus isolated may then be used to prepare cDNA molecules and cDNA libraries in accordance with the present invention.</div>
<div class="description-paragraph" id="p-0260" num="0277">In the practice of the invention, cDNA molecules or cDNA libraries are produced by mixing one or more nucleic acid molecules obtained as described above, which is preferably one or more mRNA molecules such as a population of mRNA molecules, with a polypeptide having reverse transcriptase activity of the present invention, or with one or more of the compositions of the invention, under conditions favoring the reverse transcription of the nucleic acid molecule by the action of the enzymes or the compositions to form one or more cDNA molecules (single-stranded or double-stranded). Thus, the method of the invention comprises (a) mixing one or more nucleic acid templates (preferably one or more RNA or mRNA templates, such as a population of mRNA molecules) with one or more reverse transcriptases of the invention and (b) incubating the mixture under conditions sufficient to make one or more nucleic acid molecules complementary to all or a portion of the one or more templates. Such methods may include the use of one or more DNA polymerases, one or more nucleotides, one or more primers, one or more buffers, and the like. The invention may be used in conjunction with methods of cDNA synthesis such as those described in the Examples below, or others that are well-known in the art (see, e.g., Gubler, U., and Hoffman, B. J., <i>Gene </i>25:263-269 (1983); Krug, M. S., and Berger, S. L., <i>Meth. Enzymol. </i>152:316-325 (1987); Sambrook, J., et al., <i>Molecular Cloning: A Laboratory Manual, </i>2nd ed., Cold Spring Harbor, N.Y.: Cold Spring Harbor Laboratory Press, pp. 8.60-8.63 (1989); PCT Publication No. WO 99/15702; PCT Publication No. WO 98/47912; and PCT Publication No. WO 98/51699), to produce cDNA molecules or libraries.</div>
<div class="description-paragraph" id="p-0261" num="0278">Other methods of cDNA synthesis which may advantageously use the present invention will be readily apparent to one of ordinary skill in the art.</div>
<div class="description-paragraph" id="p-0262" num="0279">Having obtained cDNA molecules or libraries according to the present methods, these cDNAs may be isolated for further analysis or manipulation. Detailed methodologies for purification of cDNAs are taught in the GENETRAPPER™ manual (Invitrogen Corporation (Carlsbad, Calif.)), which is incorporated herein by reference in its entirety, although alternative standard techniques of cDNA isolation that are known in the art (see, e.g., Sambrook, J., et al., <i>Molecular Cloning: A Laboratory Manual, </i>2nd ed., Cold Spring Harbor, N.Y.: Cold Spring Harbor Laboratory Press, pp. 8.60-8.63 (1989)) may also be used.</div>
<div class="description-paragraph" id="p-0263" num="0280">In other aspects of the invention, the invention may be used in methods for amplifying and sequencing nucleic acid molecules. Nucleic acid amplification methods according to this aspect of the invention may be one-step (e.g., one-step RT-PCR) or two-step (e.g., two-step RT-PCR) reactions. According to the invention, one-step RT-PCR type reactions may be accomplished in one tube thereby lowering the possibility of contamination. Such one-step reactions comprise (a) mixing a nucleic acid template (e.g., mRNA) with one or more reverse transcriptases of the present invention and with one or more DNA polymerases and (b) incubating the mixture under conditions sufficient to amplify a nucleic acid molecule complementary to all or a portion of the template. Such amplification may be accomplished by the reverse transcriptase activity alone or in combination with the DNA polymerase activity. Two-step RT-PCR reactions may be accomplished in two separate steps. Such a method comprises (a) mixing a nucleic acid template (e.g., mRNA) with a reverse transcriptase of the present invention, (b) incubating the mixture under conditions sufficient to make a nucleic acid molecule (e.g., a DNA molecule) complementary to all or a portion of the template, (c) mixing the nucleic acid molecule with one or more DNA polymerases, and (d) incubating the mixture of step (c) under conditions sufficient to amplify the nucleic acid molecule. For amplification of long nucleic acid molecules (i.e., greater than about 3-5 Kb in length), a combination of DNA polymerases may be used, such as one DNA polymerase having 3′ exonuclease activity and another DNA polymerase being substantially reduced in 3′ exonuclease activity.</div>
<div class="description-paragraph" id="p-0264" num="0281">Nucleic acid sequencing methods according to this aspect of the invention may comprise both cycle sequencing (sequencing in combination with amplification) and standard sequencing reactions. The sequencing method of the invention thus comprises (a) mixing a nucleic acid molecule to be sequenced with one or more primers, one or more reverse transcriptases of the invention, one or more nucleotides and one or more terminating agents, (b) incubating the mixture under conditions sufficient to synthesize a population of nucleic acid molecules complementary to all or a portion of the molecule to be sequenced, and (c) separating the population to determine the nucleotide sequence of all or a portion of the molecule to be sequenced. According to the invention, one or more DNA polymerases (preferably thermostable DNA polymerases) may be used in combination with or separate from the reverse transcriptases of the invention.</div>
<div class="description-paragraph" id="p-0265" num="0282">Amplification methods which may be used in accordance with the present invention include PCR (U.S. Pat. Nos. 4,683,195 and 4,683,202), Strand Displacement Amplification (SDA; U.S. Pat. No. 5,455,166; EP 0 684 315), and Nucleic Acid Sequence-Based Amplification (NASBA; U.S. Pat. No. 5,409,818; EP 0 329 822), as well as more complex PCR-based nucleic acid fingerprinting techniques such as Random Amplified Polymorphic DNA (RAPD) analysis (Williams, J. G. K., et al., <i>Nucl. Acids Res. </i>18(22):6531-6535, 1990), Arbitrarily Primed PCR (AP-PCR; Welsh, J., and McClelland, M., <i>Nucl. Acids Res. </i>18(24):7213-7218, 1990), DNA Amplification Fingerprinting (DAF; Caetano-Anollés et al., <i>Bio/Technology </i>9:553-557, 1991), microsatellite PCR or Directed Amplification of Minisatellite-region DNA (DAMD; Heath, D. D., et al., <i>Nucl. Acids Res. </i>21(24): 5782-5785, 1993), and Amplification Fragment Length Polymorphism (AFLP) analysis (EP 0 534 858; Vos, P., et al., <i>Nucl. Acids Res. </i>23(21):4407-4414, 1995; Lin, J. J., and Kuo, J., <i>FOCUS </i>17(2):66-70, 1995). Nucleic acid sequencing techniques which may employ the present compositions include dideoxy sequencing methods such as those disclosed in U.S. Pat. Nos. 4,962,022 and 5,498,523. In a particularly preferred aspects, the invention may be used in methods of amplifying or sequencing a nucleic acid molecule comprising one or more polymerase chain reactions (PCRs), such as any of the PCR-based methods described above.</div>
<div class="description-paragraph" id="h-0019" num="0000">Kits</div>
<div class="description-paragraph" id="p-0266" num="0283">In another embodiment, the present invention may be assembled into kits, which may be used in reverse transcription or amplification of a nucleic acid molecule, or into kits for use in sequencing of a nucleic acid molecule. Kits according to this aspect of the invention comprise a carrier means, such as a box, carton, tube or the like, having in close confinement therein one or more container means, such as vials, tubes, ampoules, bottles and the like, wherein a first container means contains one or more polypeptides of the present invention having reverse transcriptase activity. When more than one polypeptide having reverse transcriptase activity is used, they may be in a single container as mixtures of two or more polypeptides, or in separate containers. The kits of the invention may also comprise (in the same or separate containers) one or more DNA polymerases, a suitable buffer, one or more nucleotides and/or one or more primers. The kits of the invention may also comprise one or more hosts or cells including those that are competent to take up nucleic acids (e.g., DNA molecules including vectors). Preferred hosts may include chemically competent or electrocompetent bacteria such as <i>E. coli </i>(including DH5, DH5α, DH10B, HB101, Top 10, and other K-12 strains as well as <i>E. coli </i>B and <i>E. coli </i>W strains).</div>
<div class="description-paragraph" id="p-0267" num="0284">In a specific aspect of the invention, the kits of the invention (e.g., reverse transcription and amplification kits) may comprise one or more components (in mixtures or separately) including one or more polypeptides having reverse transcriptase activity of the invention, one or more nucleotides (one or more of which may be labeled, e.g., fluorescently labeled) used for synthesis of a nucleic acid molecule, and/or one or more primers (e.g., oligo(dT) for reverse transcription). Such kits (including the reverse transcription and amplification kits) may further comprise one or more DNA polymerases. Sequencing kits of the invention may comprise one or more polypeptides having reverse transcriptase activity of the invention, and optionally one or more DNA polymerases, one or more terminating agents (e.g., dideoxynucleoside triphosphate molecules) used for sequencing of a nucleic acid molecule, one or more nucleotides and/or one or more primers. Preferred polypeptides having reverse transcriptase activity, DNA polymerases, nucleotides, primers and other components suitable for use in the reverse transcription, amplification and sequencing kits of the invention include those described above. The kits encompassed by this aspect of the present invention may further comprise additional reagents and compounds necessary for carrying out standard nucleic acid reverse transcription, amplification or sequencing protocols. Such polypeptides having reverse transcriptase activity of the invention, DNA polymerases, nucleotides, primers, and additional reagents, components or compounds may be contained in one or more containers, and may be contained in such containers in a mixture of two or more of the above-noted components or may be contained in the kits of the invention in separate containers. Such kits may also comprise instructions (e.g., for performing the methods of the invention such as for labeling nucleic acid molecules in accordance with the invention).</div>
<div class="description-paragraph" id="h-0020" num="0000">Use of Nucleic Acid Molecules</div>
<div class="description-paragraph" id="p-0268" num="0285">The nucleic acid molecules or cDNA libraries prepared by the methods of the present invention may be further characterized, for example by cloning and sequencing (i.e., determining the nucleotide sequence of the nucleic acid molecule), by the sequencing methods of the invention or by others that are standard in the art (see, e.g., U.S. Pat. Nos. 4,962,022 and 5,498,523, which are directed to methods of DNA sequencing). Alternatively, these nucleic acid molecules may be used for the manufacture of various materials in industrial processes, such as hybridization probes by methods that are well-known in the art. Production of hybridization probes from cDNAs will, for example, provide the ability for those in the medical field to examine a patient's cells or tissues for the presence of a particular genetic marker such as a marker of cancer, of an infectious or genetic disease, or a marker of embryonic development. Furthermore, such hybridization probes can be used to isolate DNA fragments from genomic DNA or cDNA libraries prepared from a different cell, tissue or organism for further characterization.</div>
<div class="description-paragraph" id="p-0269" num="0286">The nucleic acid molecules of the present invention may also be used to prepare compositions for use in recombinant DNA methodologies. Accordingly, the present invention relates to recombinant vectors which comprise the cDNA or amplified nucleic acid molecules of the present invention, to host cells which are genetically engineered with the recombinant vectors, to methods for the production of a recombinant polypeptide using these vectors and host cells, and to recombinant polypeptides produced using these methods.</div>
<div class="description-paragraph" id="p-0270" num="0287">Recombinant vectors may be produced according to this aspect of the invention by inserting, using methods that are well-known in the art, one or more of the cDNA molecules or amplified nucleic acid molecules prepared according to the present methods into a vector. The vector used in this aspect of the invention may be, for example, a phage or a plasmid, and is preferably a plasmid. Preferred are vectors comprising cis-acting control regions to the nucleic acid encoding the polypeptide of interest. Appropriate trans-acting factors may be supplied by the host, supplied by a complementing vector or supplied by the vector itself upon introduction into the host.</div>
<div class="description-paragraph" id="p-0271" num="0288">In certain preferred embodiments in this regard, the vectors provide for specific expression (and are therefore termed “expression vectors”), which may be inducible and/or cell type-specific. Particularly preferred among such vectors are those inducible by environmental factors that are easy to manipulate, such as temperature and nutrient additives.</div>
<div class="description-paragraph" id="p-0272" num="0289">Expression vectors useful in the present invention include chromosomal-, episomal- and virus-derived vectors, e.g., vectors derived from bacterial plasmids or bacteriophages, and vectors derived from combinations thereof, such as cosmids and phagemids, and will preferably include at least one selectable marker such as a tetracycline or ampicillin resistance gene for culturing in a bacterial host cell. Prior to insertion into such an expression vector, the cDNA or amplified nucleic acid molecules of the invention should be operatively linked to an appropriate promoter, such as the phage lambda P<sub>L </sub>promoter, the <i>E. coli </i>lac, trp and tac promoters. Other suitable promoters will be known to the skilled artisan.</div>
<div class="description-paragraph" id="p-0273" num="0290">Among vectors preferred for use in the present invention include pQE70, pQE60 and pQE-9, available from Qiagen; pBS vectors, Phagescript vectors, Bluescript vectors, pNH8A, pNH16a, pNH18A, pNH46A, available from Stratagene; pcDNA3 available from Invitrogen; pGEX, pTrxfus, pTrc99a, pET-5, pET-9, pKK223-3, pKK233-3, pDR540, pRIT5 available from Pharmacia; and pSPORT1, pSPORT2 and pSV.SPORT1, available from Invitrogen Corporation (Carlsbad, Calif.). Other suitable vectors will be readily apparent to the skilled artisan.</div>
<div class="description-paragraph" id="p-0274" num="0291">The invention also provides methods of producing a recombinant host cell comprising the cDNA molecules, amplified nucleic acid molecules or recombinant vectors of the invention, as well as host cells produced by such methods. Representative host cells (prokaryotic or eukaryotic) that may be produced according to the invention include, but are not limited to, bacterial cells, yeast cells, plant cells and animal cells. Preferred bacterial host cells include <i>Escherichia coli </i>cells (most particularly <i>E. coli </i>strains DH10B and Stb12, which are available commercially (Invitrogen Corporation (Carlsbad, Calif.)), <i>Bacillus subtilis </i>cells, <i>Bacillus megaterium </i>cells, <i>Streptomyces </i>spp. cells, <i>Erwinia </i>spp. cells, <i>Klebsiella </i>spp. cells and <i>Salmonella typhimurium </i>cells. Preferred animal host cells include insect cells (most particularly <i>Spodoptera frugiperda </i>Sf9 and Sf21 cells and <i>Trichoplusa </i>HigH-Five cells) and mammalian cells (most particularly CHO, COS, VERO, BHK and human cells). Such host cells may be prepared by well-known transformation, electroporation or transfection techniques that will be familiar to one of ordinary skill in the art.</div>
<div class="description-paragraph" id="p-0275" num="0292">In addition, the invention provides methods for producing a recombinant polypeptide, and polypeptides produced by these methods. According to this aspect of the invention, a recombinant polypeptide may be produced by culturing any of the above recombinant host cells under conditions favoring production of a polypeptide therefrom, and isolation of the polypeptide. Methods for culturing recombinant host cells, and for production and isolation of polypeptides therefrom, are well-known to one of ordinary skill in the art.</div>
<div class="description-paragraph" id="p-0276" num="0293">It will be readily apparent to one of ordinary skill in the relevant arts that other suitable modifications and adaptations to the methods and applications described herein are obvious and may be made without departing from the scope of the invention or any embodiment thereof. Having now described the present invention in detail, the same will be more clearly understood by reference to the following examples, which are included herewith for purposes of illustration only and are not intended to be limiting of the invention.</div>
<heading id="h-0021">EXAMPLES</heading>
<heading id="h-0022">Example 1</heading>
<heading id="h-0023">Preparation of Mutant Reverse Transcriptases</heading>
<div class="description-paragraph" id="p-0277" num="0294">Plasmid pBAD was obtained from Invitrogen Corporation, Carlsbad, Calif. and the coding sequence of M-MLV reverse transcriptase was inserted to produce plasmid pBAD-6-His-M-MLV H− (F1). Plasmid pBAD-6-His-M-MLV H− (F1) was used as both a cloning vector and as a target for PCR mutagenesis (<figref idrefs="DRAWINGS">FIG. 1</figref>). pBAD-6-His-M-MLV H− (F1) replicates in <i>E. coli </i>and confers ampicillin resistance to transformed cells. The M-MLV reverse transcriptase gene is expressed from the ara BAD promoter which is induced by the presence of arabinose. The promoter is repressed by the product of the araC gene, which is present on the plasmid. The host used, <i>E. coli </i>strain DH10B, is an araD mutant and cannot metabolize arabinose, making arabinose a gratuitous inducer in DH10B cells transformed with pBAD-6-His-M-MLV H− (F1). The plasmid contains a 6 histidine containing leader sequence in frame with the coding sequence of the M-MLV reverse transcriptase gene. The gene starting at nucleotide 2598 and ending at nucleotide 4628 (Shinnick, et al., (1981) <i>Nature </i>293, 543-548) was cloned under control of an araD promoter into plasmid pBAD/HisA (Invitrogen). The M-MLV gene was further modified by site-directed mutagenesis without changing amino acid coding to include several unique restriction endonuclease sites that divided the gene into five segments (<figref idrefs="DRAWINGS">FIG. 2</figref>). The amino end of the protein contained a His<sub>6 </sub>tag to simplify purification that included the following amino acids: MGGSHHHHHHGMASMTGGQQMGRDLYDDDDKH corresponding to amino acids 1-32 of SEQ ID NO:2. The carboxy end of the protein contained the additional amino acids NSRLIN, corresponding to amino acids 711-716 of SEQ ID NO:2, present as the result of subcloning from pRT601. In addition, the M-MLV RT gene was mutated (D524G, E562Q, D583N) to eliminate RNase H activity. The final construct was designated pBAD-HSS2 (<figref idrefs="DRAWINGS">FIG. 1</figref>), and the gene and gene product were designated His<sub>6 </sub>H− RT. In addition to this construct, other constructs having different N-terminal sequences are contemplated in the present invention. For example, a construct beginning at methionine 12 of SEQ ID NO:6 and Table 3 and containing a mutation changing methionine 15 to glycine (M15G) to produce a protein with an N-terminal sequence MASGTGGQQMGRDLYDDDDKH (SEQ ID NO:1) followed by the remaining sequence of M-MLV RT from Table 3 has been produced as well as a construct beginning with methionine 33 of SEQ ID NO:6 and Table 3.</div>
<div class="description-paragraph" id="p-0278" num="0295">With reference to the sequence of this plasmid provided in Table 3 (SEQ ID NOs:1 and 2), nucleotides 1-96 encode the leader sequence and nucleotides 97-99 encode a methionine. Those skilled in the art will appreciate that the wild-type M-MLV reverse transcriptase is derived by proteolysis from a precursor polyprotein and thus the wild-type M-MLV reverse transcriptase does not start with a methionine. Therefore, amino acid number 1 of the M-MLV reverse transcriptase is the threonine (amino acid 34 in SEQ ID NO:2 and Table 3) following the methionine encoded by nucleotides 97-99 (amino acid 33 in SEQ ID NO:2 and Table 3).</div>
<div class="description-paragraph" id="p-0279" num="0296">The sequence of the M-MLV reverse transcriptase gene in pBAD-6-His-M-MLV H− (F1) which was used in these experiments was derived from the sequence of plasmid pRT601. pRT601 is described in Kotewicz, et al., (1988) <i>Nuc. Acids Res. </i>16, 265-277, Gerard, et al., (1986) <i>DNA </i>5, 271-279, U.S. Pat. Nos. 5,668,005 and 5,017,492, which are incorporated herein by reference in their entireties.</div>
<div class="description-paragraph" id="p-0280" num="0297">
<tables id="TABLE-US-00003" num="00003">
<patent-tables colsep="0" frame="none" pgwide="1" rowsep="0" tabstyle="monospace">
<table align="left" class="description-table" cols="1" colsep="0" rowsep="0" width="100%">
<thead>
<tr class="description-tr">
<td class="description-td" colspan="1" nameend="1" namest="1" rowsep="1">TABLE 3</td>
</tr>
<tr class="description-tr">
<td align="center" class="description-td" colspan="1" nameend="1" namest="1" rowsep="1"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">(SEQ ID NOs: 1 and 2). </td>
</tr>
<tr class="description-tr">
<td align="center" class="description-td" colspan="1" nameend="1" namest="1" rowsep="1"> </td>
</tr>
</thead>
<tbody><tr class="description-tr">
<td class="description-td"> </td>
</tr>
</tbody></table>
<table align="left" class="description-table" cols="2" colsep="0" rowsep="0" width="100%">
<tbody><tr class="description-tr">
<td class="description-td">1</td>
<td class="description-td">atggggggtt ctcatcatca tcatcatcat ggtatggcta gcatgactgg tggacagcaa</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">  m  g  g   s  h  h   h  h  h  h   g  m  a   s  m  t   g  g  q  q</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">61</td>
<td class="description-td">atgggtcggg atctgtacga cgatgacgat aagcatatga ccctaaatat agaagatgag</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">  m  g  r   d  l  y   d  d  d  d   k  h  m   t  l  n   i  e  d  e</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">121</td>
<td class="description-td">tatcggctac atgagacctc aaaagagcca gatgtttctc tagggtccac atggctgtct</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">  y  r  l   h  e  t   s  k  e  p   d  v  s   l  g  s   t  w  l  s</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">181</td>
<td class="description-td">gattttcctc aggcctgggc ggaaaccggg ggcatgggac tggcagttcg ccaagctcct</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">  d  f  p   q  a  w   a  e  t  g   g  m  g   l  a  v   r  q  a  p</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">241</td>
<td class="description-td">ctgatcatac ttctgaaagc aacctctacc cccgtgtcca taaaacaata ccccatgtca</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">  l  i  i   l  l  k   a  t  s  t   p  v  s   i  k  q   y  p  m  s</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">301</td>
<td class="description-td">caagaagcca gactggggat caagccccac atacagagac tgttggacca gggaatactg</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">  q  e  a   r  l  g   i  k  p  h   i  q  r   l  l  d   q  g  i  l</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">361</td>
<td class="description-td">gtaccctgcc agtccccctg gaacacgccc ctgctacccg tcaagaaacc cgggactaat</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">  v  p  c   q  s  p   w  n  t  p   l  l  p   v  k  k   p  g  t  n</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">421</td>
<td class="description-td">gattacaggc ctgtccaaga tctgagagag gtcaacaaac gcgtagaaga catccacccc</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">  d  y  r   p  v  q   d  l  r  e   v  n  k   r  v  e   d  i  h  p</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">481</td>
<td class="description-td">accgtaccca acccctacaa cctcttgagt gggctcccac cgtcccacca gtggtacact</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">  t  v  p   n  p  y   n  l  l  s   g  l  p   p  s  h   q  w  y  t</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">541</td>
<td class="description-td">gttctagact taaaagatgc ctttttctgc ctgagactcc acccgacgtc tcagcctctc</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">  v  l  d   l  k  d   a  f  f  c   l  r  l   h  p  t   s  q  p  l</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">601</td>
<td class="description-td">ttcgcctttg aatggagaga cccagagatg ggaatctctg gccaactaac ctggaccaga</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">  f  a  f   e  w  r   d  p  e  m   g  i  s   g  q  l   t  w  t  r</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">661</td>
<td class="description-td">ctcccacagg gattcaaaaa cagtcccacc ctgtttgatg aggcactgcg cagagaccta</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">  l  p  q   g  f  k   n  s  p  t   l  f  d   e  a  l   r  r  d  l</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">721</td>
<td class="description-td">gcagacttcc ggatccagca cccagacttg atcctgctac agtacgtaga tgacttactg</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">  a  d  f   r  i  q   h  p  d  l   i  l  l   q  y  v   d  d  l  l</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">781</td>
<td class="description-td">ctggccgcca cttctgagct cgactgccaa caaggtactc gggccctgtt acaaacccta</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">  l  a  a   t  s  e   l  d  c  q   q  g  t   r  a  l   l  q  t  l</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">841</td>
<td class="description-td">ggagacctcg ggtatcgggc ctcggccaag aaagcccaaa tttgccagaa acaggtcaag</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">  g  d  l   g  y  r   a  s  a  k   k  a  q   i  c  q   k  q  v  k</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">901</td>
<td class="description-td">tatctggggt atcttctaaa agagggtcag agatggctga ctgaggccag aaaagagact</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">  y  l  g   y  l  l   k  e  g  q   r  w  l   t  e  a   r  k  e  t</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">961</td>
<td class="description-td">gtgatggggc agcctactcc gaagaccccg cggcaactaa gggagttcct agggacggca</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">  v  m  g   q  p  t   p  k  t  p   r  q  l   r  e  f   l  g  t  a</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">1021</td>
<td class="description-td">ggcttctgtc gcctctggat ccctgggttt gcagaaatgg cagccccctt gtaccctctc</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">  g  f  c   r  l  w   i  p  g  f   a  e  m   a  a  p   l  y  p  l</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">1081</td>
<td class="description-td">accaaaacgg ggactctgtt taattggggc ccagaccaac aaaaggccta tcaagaaatc</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">  t  k  t   g  t  l   f  n  w  g   p  d  q   q  k  a   y  q  e  i</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">1141</td>
<td class="description-td">aagcaagctc ttctaactgc cccagccctg gggttgccag atttgactaa gccctttgaa</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">  k  q  a   l  l  t   a  p  a  l   g  l  p   d  l  t   k  p  f  e</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">1201</td>
<td class="description-td">ctctttgtcg acgagaagca gggctacgcc aaaggtgtcc taacgcaaaa actgggacct</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">  l  f  v   d  e  k   q  g  y  a   k  g  v   l  t  q   k  l  g  p</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">1261</td>
<td class="description-td">tggcgtcggc cggtggccta cctgtccaaa aagctagacc cagtagcagc tgggtggccc</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">  w  r  r   p  v  a   y  l  s  k   k  l  d   p  v  a   a  g  w  p </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">1321</td>
<td class="description-td">ccttgcctac ggatggtagc agccattgcc gtactgacaa aggatgcagg caagctaacc</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">  p  c  l   r  m  v   a  a  i  a   v  l  t   k  d  a   g  k  l  t</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">1381</td>
<td class="description-td">atgggacagc cactagtcat tctggccccc catgcagtag aggcactagt caaacaaccc</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">  m  g  q   p  l  v   i  l  a  p   h  a  v   e  a  l   v  k  q  p</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">1441</td>
<td class="description-td">cccgatcgat ggctttccaa cgcccggatg actcactatc aggccttgct tttggacacg</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">  p  d  r   w  l  s   n  a  r  m   t  h  y   q  a  l   l  l  d  t</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">1501</td>
<td class="description-td">gaccgggtcc agttcggacc ggtggtagcc ctgaacccgg ctacactgct cccactgcct</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">  d  r  v   q  f  g   p  v  v  a   l  n  p   a  t  l   l  p  l  p</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">1561</td>
<td class="description-td">gaggaagggc tgcagcacaa ctgccttgat atcctggccg aagcccacgg aacccgaccc</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">  e  e  g   l  q  h   n  c  l  d   i  l  a   e  a  h   g  t  r  p</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">1621</td>
<td class="description-td">gacctaacgg accagccgct cccagacgcc gaccacacct ggtacacggg tggatccagt</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">  d  l  t   d  q  p   l  p  d  a   d  h  t   w  y  t   g  g  s  s</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">1681</td>
<td class="description-td">ctcttgcaag agggacagcg taaggcggga gctgcggtga ccaccgagac cgaggtaatc</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">  l  l  q   e  g  q   r  k  a  g   a  a  v   t  t  e   t  e  v  i</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">1741</td>
<td class="description-td">tgggctaaag ccctgccagc cgggacatcc gctcagcggg ctcagctgat agcactcacc</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">  w  a  k   a  l  p   a  g  t  s   a  q  r   a  q  l   i  a  l  t</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">1801</td>
<td class="description-td">caggccctaa ggatggcaga aggtaagaag ctaaatgttt atacgaattc ccgttatgct</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">  q  a  l   r  m  a   e  g  k  k   l  n  v   y  t  n   s  r  y  a</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">1861</td>
<td class="description-td">tttgctactg cccatatcca tggagaaata tacagaaggc gtgggttgct cacatcagaa</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">  f  a  t   a  h  i   h  g  e  i   y  r  r   r  g  l   l  t  s  e</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">1921</td>
<td class="description-td">ggcaaagaga tcaaaaataa ggacgagata ttggccctac taaaagccct ctttctgccc</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">  g  k  e   i  k  n   k  d  e  i   l  a  l   l  k  a   l  f  l  p</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">1981</td>
<td class="description-td">aaaagactta gcataatcca ttgtccagga catcaaaagg gacacagcgc cgaggctaga</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">  k  r  l   s  i  i   h  c  p  g   h  q  k   g  h  s   a  e  a  r</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">2041</td>
<td class="description-td">ggcaaccgga tggctgacca agcggcccga aaggcagcca tcacagagaa tccagacacc</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">  g  n  r   m  a  d   q  a  a  r   k  a  a   i  t  e   n  p  d  t</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">2101</td>
<td class="description-td">tctaccctcc tcatagaaaa ttcatcaccc aattcccgct taattaatta a</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">  s  t  l   l  i  e   n  s  s  p   n  s  r   l  i  n   -</td>
</tr>
<tr class="description-tr">
<td align="center" class="description-td" colspan="2" nameend="2" namest="1" rowsep="1"> </td>
</tr>
</tbody></table>
</patent-tables>
</tables>
</div>
<div class="description-paragraph" id="p-0281" num="0298">Table 4 provides a list of the point mutations introduced in the M-MLV reverse transcriptase coding sequence of pRT601 to produce the plasmid used. The numbering of the point mutations corresponds to the nucleotide sequence presented in Table 3.</div>
<div class="description-paragraph" id="p-0282" num="0299">
<tables id="TABLE-US-00004" num="00004">
<patent-tables colsep="0" frame="none" rowsep="0">
<table align="left" class="description-table" cols="4" colsep="0" rowsep="0" width="100%">
<thead>
<tr class="description-tr">
<td class="description-td" colspan="4" nameend="4" namest="1" rowsep="1">TABLE 4</td>
</tr>
<tr class="description-tr">
<td align="center" class="description-td" colspan="4" nameend="4" namest="1" rowsep="1"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">Nucleotide #</td>
<td class="description-td"> </td>
<td class="description-td">Nucleotide #</td>
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">in Table 3</td>
<td class="description-td">change</td>
<td class="description-td">in Table 3</td>
<td class="description-td">change</td>
</tr>
<tr class="description-tr">
<td align="center" class="description-td" colspan="4" nameend="4" namest="1" rowsep="1"> </td>
</tr>
</thead>
<tbody><tr class="description-tr">
<td class="description-td"> </td>
</tr>
</tbody></table>
<table align="left" class="description-table" cols="4" colsep="0" rowsep="0" width="100%">
<tbody><tr class="description-tr">
<td class="description-td">411</td>
<td class="description-td">a<sub>→</sub>c</td>
<td class="description-td">993</td>
<td class="description-td">a<sub>→</sub>g</td>
</tr>
<tr class="description-tr">
<td class="description-td">459</td>
<td class="description-td">g<sub>→</sub>a</td>
<td class="description-td">1446</td>
<td class="description-td">c<sub>→</sub>t</td>
</tr>
<tr class="description-tr">
<td class="description-td">462</td>
<td class="description-td">g<sub>→</sub>c</td>
<td class="description-td">1449</td>
<td class="description-td">c<sub>→</sub>a</td>
</tr>
<tr class="description-tr">
<td class="description-td">543</td>
<td class="description-td">g<sub>→</sub>t </td>
<td class="description-td">1670</td>
<td class="description-td">a<sub>→</sub>g</td>
</tr>
<tr class="description-tr">
<td class="description-td">546</td>
<td class="description-td">t<sub>→</sub>a</td>
<td class="description-td">1675</td>
<td class="description-td">a<sub>→</sub>t</td>
</tr>
<tr class="description-tr">
<td class="description-td">585</td>
<td class="description-td">c<sub>→</sub>g</td>
<td class="description-td">1676</td>
<td class="description-td">g<sub>→</sub>c</td>
</tr>
<tr class="description-tr">
<td class="description-td">588</td>
<td class="description-td">c<sub>→</sub>g</td>
<td class="description-td">1783</td>
<td class="description-td">g<sub>→</sub>c</td>
</tr>
<tr class="description-tr">
<td class="description-td">589</td>
<td class="description-td">a<sub>→</sub>t</td>
<td class="description-td">1785</td>
<td class="description-td">a<sub>→</sub>g</td>
</tr>
<tr class="description-tr">
<td class="description-td">590</td>
<td class="description-td">g<sub>→</sub>c</td>
<td class="description-td">1845</td>
<td class="description-td">t<sub>→</sub>g</td>
</tr>
<tr class="description-tr">
<td class="description-td">639</td>
<td class="description-td">a<sub>→</sub>t</td>
<td class="description-td">1846</td>
<td class="description-td">g<sub>→</sub>a</td>
</tr>
<tr class="description-tr">
<td class="description-td">642</td>
<td class="description-td">a<sub>→</sub>c</td>
<td class="description-td">1849</td>
<td class="description-td">a<sub>→</sub>t</td>
</tr>
<tr class="description-tr">
<td class="description-td">710</td>
<td class="description-td">a<sub>→</sub>g</td>
<td class="description-td">1850</td>
<td class="description-td">g<sub>→</sub>c</td>
</tr>
<tr class="description-tr">
<td class="description-td">801</td>
<td class="description-td">a<sub>→</sub>c </td>
<td class="description-td">1950</td>
<td class="description-td">c<sub>→</sub>a</td>
</tr>
<tr class="description-tr">
<td class="description-td">990</td>
<td class="description-td">t<sub>→</sub>g</td>
<td class="description-td"> </td>
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td align="center" class="description-td" colspan="4" nameend="4" namest="1" rowsep="1"> </td>
</tr>
</tbody></table>
</patent-tables>
</tables>
</div>
<div class="description-paragraph" id="p-0283" num="0300">The mutations which were introduced to make RNase H− mutants of M-MLV reverse transcriptase are D524G, D583N, and E562Q. The remaining mutations were introduced to insert or remove restriction enzyme sites to facilitate the production of appropriately sized segments for the random PCR mutagenesis. This RNase H− mutant is referred to herein as S<smallcaps>UPER</smallcaps>S<smallcaps>CRIPT</smallcaps>™ II or S<smallcaps>UPER</smallcaps>S<smallcaps>CRIPT</smallcaps>™ II gene.</div>
<div class="description-paragraph" id="p-0284" num="0301">The sequence of the M-MLV reverse transcriptase was engineered to introduce restriction enzyme cleavage sites as shown schematically in <figref idrefs="DRAWINGS">FIG. 2</figref> without changing the amino acids encoded by the sequence. The sequence was divided into 5 segments and oligonucleotides were designed so that each segment could be amplified. The segments roughly corresponded to the coding sequences of the five separate structural subdomains of RT (Kohlstaedt, et al., (1992) <i>Science </i>256, 1783-1789, Jacobo-Molina, et al., (1993) <i>Proc. Natl. Acad. Sci. USA </i>90, 6320-6324). Segments one through four corresponded to the polymerase subdomain of fingers, palm, thumb, and connection, respectively, and segment five corresponded to the RNase H domain (<figref idrefs="DRAWINGS">FIG. 2</figref>). An upper limit cut-off of 1 to 2 mutations per segment was set as the target for mutation frequency to suppress accumulation of deleterious mutations, and to minimize the amount of screening required to find active mutants. Mutation frequencies of &gt;5 mutations/segment in segment one, two, or three produced only about 5% active mutants with all the mutants having less than wild-type activity. At the mutation frequency used of 1 to 2 mutations per segment, approximately one-third of the mutants had little or no activity, one-third had less than 50% of His<sub>6 </sub>H− RT activity, and one-third had up to 100% of His<sub>6 </sub>H− RT activity.</div>
<div class="description-paragraph" id="p-0285" num="0302">Segments were prepared from pBAD-6-His-M-MLV H− (F1) by restriction enzyme digests and the segments were gel purified away from the vector backbone. Each segment was randomly mutagenized by PCR in the presence of manganese. The PCR conditions were standard except that 0.25 mM MnCl<sub>2 </sub>was present, and the nucleotide triphosphate concentration was limited to 20 μM of each dNTP (50 mM Tris HCl pH 8.3, 50 mM KCl, 3 mM MgCl<sub>2</sub>, 20 μM dGTP, 20 μM dCTP, 20 μM dATP, 20 μM dTTP, 1 unit Taq DNA polymerase per 100 μl reaction). The PCR product was extracted with phenol-chloroform, precipitated with ethanol and the mutated segments were cloned into a vector from which the given segment had been removed.</div>
<div class="description-paragraph" id="p-0286" num="0303">In some random mutagenesis experiments, mutagenic PCR was performed in a reaction mixture (100 μl) containing 20 mM Tris-HCl (pH 8.4), 50 mM KCl, 1.8 mM MgCl<sub>2</sub>, 0.3 mM MnSO<sub>4</sub>, 200 μM each of dCTP, dGTP, dTTP, and dATP, and 0.5 units of Taq DNA polymerase (Invitrogen Corporation, Carlsbad, Calif.). After a 1 min denaturation step at 94° C., the cycling protocol was 15 sec at 94° C., 15 sec at 55° C., and 30 sec at 72° C. for 20 cycles. Amplification was 100 fold from 50 ng of target to 5 μg of amplified product. PCR primers included appropriate restriction endonuclease cut sites. An amplified DNA segment was cleaved with appropriate restriction endonucleases, gel purified, and cloned into gel purified vector DNA cut with the corresponding restriction enzymes. The vectors containing the mutated segments were transformed into appropriate host cells to produce a library.</div>
<div class="description-paragraph" id="p-0287" num="0304">Libraries were sampled by DNA sequencing of the mutagenized H− RT gene segment of a small number of clones to determine the mutation frequency. The goal was to achieve rates of PCR random mutagenesis that produced 1 to 2 mutations per segment. It was found that random mutagenesis that produced greater than 2 mutations per segment tended to produce a large proportion of inactive RT mutants. If the library met this criterion, further screening by heat treatment was carried out to identify mutants that showed greater RT activity in lysates than H− RT after a heat treatment step. Those mutants with the highest apparent thermal stability were screened again in duplicate by pretreatment at 24° C. (to normalize for activity) and at 52 to 58° C. to confirm the presence of enhanced thermal stability. In all, about 15,000 clones were screened by heat treatment in the 96-well format for each segment or a total of about 100,000 mutants. Libraries of transformants for each mutated segment were screened for thermostable variants.</div>
<heading id="h-0024">Example 2</heading>
<heading id="h-0025">Screening for Thermostable Reverse Transcriptases</heading>
<div class="description-paragraph" id="p-0288" num="0305">In this example the following solutions were used:
</div> <ul> <li id="ul0004-0001" num="0306">EG-per liter: 20 g bacto-tryptone, 10 g bacto yeast extract, 2 ml glycerol, 0.54 g NaCl, 0.194 g KCl</li> <li id="ul0004-0002" num="0307">EG-arabinose—150 ml EG plus 1.5 ml of 10 mg/ml ampicillin and 1.5 ml of 20% (w/v) arabinose (if plates are to have arabinose)</li> <li id="ul0004-0003" num="0308">20×PEB-I Buffer—18% (w/v) glucose, 500 mM Tris-HCl (pH 8.0), 200 mM EDTA</li> <li id="ul0004-0004" num="0309">Kinase Storage Buffer—50% (v/v) glycerol, 20 mM Tris-HCl (pH 8.0), 100 mM KCl, 5 mM βME</li> <li id="ul0004-0005" num="0310">100 mg/ml lysozyme—made in Kinase Storage Buffer and stored at −20° C.</li> <li id="ul0004-0006" num="0311">2×PLD—5 ml of 20×PEB-I, 1 ml of 1 M DTT, 5 ml of 10% (v/v) Triton X-100, 1 ml of 100 mg/ml lysozyme and 38 ml of water</li> <li id="ul0004-0007" num="0312">2×PZD—0.5 ml of 20×PEB-I, 100 μl of 1 M DTT, 0.5 ml of 10% (v/v) Triton X-100, 10 μl of zymolase and 3.9 ml of water</li> <li id="ul0004-0008" num="0313">10× Poly(C) Reaction Buffer—500 mM Tris-HCl (pH 8.4), 500 mM KCl, 100 mM MgCl<sub>2 </sub> </li> <li id="ul0004-0009" num="0314">1.25× Reaction Mix—1 ml of 10× Poly(C) Reaction Buffer, 100 μl of 1 M DTT, 1 ml of poly(C)/oligo(dG) (30 mM/12 mM in nucleotide), 10 μl of 100 mM dGTP, 5.87 ml of water and 20 μl of [α-<sup>32</sup>P] dGTP at 10 μCi/μl</li> </ul>
<div class="description-paragraph" id="p-0289" num="0315"> <i>E. coli </i>DH10B (Invitrogen Corporation, Carlsbad, Calif.) was used in all experiments. Bacterial liquid cultures were grown at 37° C. in EG: 2% tryptone, 1% yeast extract, 0.5% glycerol, 10 mM NaCl, and 1 mM KCl. Solid medium was LB (1% bacto-tryptone, 0.5% yeast extract, and 86 mM NaCl)+1.5% agar. Selective media included 100 μg/ml ampicillin. For induction of cultures, cells were inoculated into selective EG+0.2% arabinose and grown for 18 hr.</div>
<div class="description-paragraph" id="p-0290" num="0316">Mutant populations were plated on selective agar. Individual transformant colonies were inoculated into single wells of a 96 well culture plate. Each well contained 120 μl of EG-Ap medium (EG medium with 100 μg/ml ampicillin). Although colonies from the selective agar may be grown in media containing 0.2% arabinose, it is preferable to first inoculate a 96 well plate with selective medium without the inducer, to grow that master plate overnight, and then to make a replica of the master plate into a 96-well plate with the inducer and grow that plate overnight.</div>
<div class="description-paragraph" id="p-0291" num="0317">An aliquot of the un-induced culture from each well was transferred into a new 96-well plate containing 120 μl per well of selective media+0.2% arabinose. The cultures containing the inducer were grown overnight (e.g., 15-20 hours) at 37° C. without shaking to induce RT production. An aliquot (5 μl) of culture from each well was then transferred into another 96-well plate containing 5 μl per well of 2×PLD (50 mM Tris-HCl, pH 8.0, 20 mM EDTA, 1.8% (w/v) sucrose, 1% (v/v) Triton X-100, 10 mM DTT, and 2 mg/ml lysozyme) at room temperature. These extracts were sometimes assayed directly for reverse transcriptase before the heating step. The amount of RT activity in a 5-μl aliquot of extract was within the linear range of the assay. Lysates were stable at room temperature for at least 1 hr.</div>
<div class="description-paragraph" id="p-0292" num="0318">The extracts were heated, for example, using a water bath or thermocycler, for 5 or 10 minutes at temperatures that ranged from 50° C. to 60° C. Preferably, the cultures were heated for 5 minutes at 52° C. After heating and cooling to room temperature, RT activity in a 5-μl aliquot from the lysate in each well was assayed with (rC)<sub>n</sub>●(dG)<sub>15 </sub>in another 96-well plate. An aliquot (5 to 10 μl) of the extract was mixed with 1.25×RT reaction mix. This reaction was placed in a 37° C. water bath for 10 minutes. A small aliquot of the reaction mixture (5 μl) was spotted onto a charged nylon membrane (Genescreen+, NEN). The membrane was washed twice with 10% TCA+1% sodium pyrophosphate, rinsed with ethanol, dried, and placed next to a phosphor screen. As an alternative, the membrane may be washed twice with 4% sodium pyrophosphate (pH 8.0), rinsed with ethanol, dried, and then placed next to a phosphor screen. Radioactive product that had been trapped on the filter was detected by analyzing the screen in a Posphorimager, using ImageQuant software (Molecular Devices).</div>
<div class="description-paragraph" id="p-0293" num="0319">Candidates were selected if they showed more reverse transcriptase activity (radioactivity) after the heat inactivation step. These candidates were screened a second time to confirm the phenotype. Candidates which appeared to be thermostable after the second screen were grown in small cultures and tested a third time for thermostable reverse transcriptase activity. Candidates that were reproducibly heat resistant were sequenced and the mutation in each clone was determined.</div>
<div class="description-paragraph" id="p-0294" num="0320">Plasmid DNA was prepared from an over night <i>E. coli </i>culture bearing an RT mutant using a Concert High Purity Miniprep Kit (Invitrogen Corporation, Carlsbad, Calif.) following the manufacturer's instructions. Each DNA was sequenced using a forward and reverse primer bordering the segment that had been mutagenized to generate the mutant. The sequencing reactions were carried out as specified for plasmid DNA using the ABI Big Dye Terminator Sequencing Ready Reaction Kit. The reactions were analyzed using an ABI PRISM 377 DNA Sequencer.</div>
<div class="description-paragraph" id="p-0295" num="0321">An oligonucleotide corresponding to the mutagenized site was designed in which the codon for the mutagenized amino acid was randomized (NNK or NNN). Oligonucleotide site-directed mutagenesis was carried out by established procedures. Saturation of an amino acid coding site in the H− RT gene with all possible amino acids was performed by introducing the sequence NNK (N=A, C, G or T and K=G or T) at the codon site into the mutagenic oligonucleotide. These oligonucleotides were used in site-directed mutagenesis to generate a library in which all possible substitutions at the mutagenized site were made. This library was screened for thermostable reverse transcriptase activity, and the most promising clones were sequenced.</div>
<div class="description-paragraph" id="p-0296" num="0322">Screening of mutants in Segment 2 (see <figref idrefs="DRAWINGS">FIG. 2</figref>) resulted in the identification of one mutant, H204R. Screening of a library mutagenized at site H204 resulted in several mutants, but the only one that was more thermostable than M-MLV reverse transcriptase was another H204R mutant. H204R mutants of M-MLV reverse transcriptase have enhanced thermostability. Screening of mutants in segment 3 (see <figref idrefs="DRAWINGS">FIG. 2</figref>) resulted in one mutant, T306K. Randomization of the T306 position produced thermostable mutants which, when sequenced, were T306R. Both T306K and T306R mutants of M-MLV reverse transcriptase have about 1.5 fold enhanced thermostability.</div>
<heading id="h-0026">Example 3</heading>
<heading id="h-0027">TdT Reverse Transcriptase Mutants</heading>
<div class="description-paragraph" id="p-0297" num="0323">In checking fidelity mutants of reverse transcriptase (RT) for misextension in a 3 dNTP assay, it was observed that S<smallcaps>UPER</smallcaps>S<smallcaps>CRIPT</smallcaps>™ II reverse transcriptase extended 2-3 bases past the end of the template in the presence of 3 and 4 dNTPs. This non-template directed extension or TdT activity is reduced in many mutants, but in a few such as F309N and T197E it appears that this activity is severely reduced or eliminated. These mutants are probably in close proximity or in contact with the template-primer as determined by homology to HIV reverse transcriptase and its crystal structure with bound template-primer.</div>
<div class="description-paragraph" id="h-0028" num="0000">Methods</div>
<div class="description-paragraph" id="p-0298" num="0324">Mutagenesis</div>
<div class="description-paragraph" id="p-0299" num="0325">For F309N:</div>
<div class="description-paragraph" id="p-0300" num="0326">Primers were designed corresponding to the mutant position F309 with the silent insertion of a NgoMIV restriction site at amino acid positions 310-311. The primers encoded a random NNK sequence for this position generating a random library of F309 mutants, where N is any of the four bases and K is T or G. The primers along with internal S<smallcaps>UPER</smallcaps>S<smallcaps>CRIPT</smallcaps>™ II reverse transcriptase primers at an upstream SstI restriction site and a downstream SalI restriction site were used in a standard PCR reaction (10 ng S<smallcaps>UPER</smallcaps>S<smallcaps>CRIPT</smallcaps>™ II reverse transcriptase template, 2 μM of each primer, 48 μl SuperMix (Invitrogen Corporation (Carlsbad, Calif.)) for 20 cycles of 94° C. 15 sec, 55° C. 15 sec, 72° C. 30 sec) to generate two PCR fragments. These were a 240 base pair SstI-NgoMIV fragment and a 200 base pair NgoMIV-SalI fragment. The fragments were isolated and digested and ligated together and then inserted into the original S<smallcaps>UPER</smallcaps>S<smallcaps>CRIPT</smallcaps>™ II reverse transcriptase clone cut with SstI and SalI. The resulting ligation product was transformed in Max Efficiency DH10B (Invitrogen Corporation (Carlsbad, Calif.)) competent cells to create the library of mutants at site F309. This library was then plated overnight for selection.</div>
<div class="description-paragraph" id="p-0301" num="0327">For T197E and Y133A:</div>
<div class="description-paragraph" id="p-0302" num="0328">The mutants T197E and Y133A were made by oligo-directed mutagenesis as described in Kunkel, T. A. et al. <i>Methods Enzymol. </i>204:125 (1991). Briefly, the S<smallcaps>UPER</smallcaps>S<smallcaps>CRIPT</smallcaps>™ II reverse transcriptase gene was inserted into pBADhisA (Invitrogen Corporation) vector and named pBAD-SSII. This plasmid was transformed into DH11S cells and the cells were infected with M13K07 helper phage from which single strand DNA was isolated. Oligos were designed corresponding to each mutation: T197E and Y133A. Each oligo (100 μM) was kinased with T4 polynucleotide kinase (Invitrogen Corporation (Carlsbad, Calif.)) using the Forward Reaction Buffer (Invitrogen Corporation (Carlsbad, Calif.)). The oligo was annealed to single stranded pBAD-SSII DNA. Native T7 DNA polymerase (USB) and T4 DNA ligase (Invitrogen Corporation (Carlsbad, Calif.)) were added with synthesis buffer (0.4 mM dNTPs, 17.5 mM Tris-HCl, pH 7.5, 5 mM MgCl<sub>2</sub>, 2.5 mM DTT, and 1 mM ATP) to the annealed reaction on ice. The reactions were incubated at 37° C. for 30 minutes and terminated by adding 1 μl of 0.5 M EDTA. The reactions were transformed and plated with DH10B cells. Colonies were picked and mutants were determined by restriction enzyme analysis and sequenced for confirmation using an ABI 377 instrument and ABI Big Dye Terminator Cycle Sequencing Ready Reaction kit.</div>
<div class="description-paragraph" id="p-0303" num="0329">Selecting Colonies Containing Active Reverse Transcriptase.</div>
<div class="description-paragraph" id="p-0304" num="0330">Individual transformant colonies were inoculated into single wells of a 96 well culture plate. Each well contained 120 μl of media (EG-Ap) containing 0.2% arabinose. It is preferable to first inoculate a 96 well plate with selective medium without the inducer, to grow that master plate overnight, and then to make a replica of the master plate into a 96-well plate with the inducer and grow that plate overnight. The cultures were grown overnight at 37° C. without shaking Overnight cultures were mixed with an equal volume of 2×PLD (1.8% glucose, 50 mM Tris-HCl, pH 8.0, 20 mM EDTA, 20 mM DTT, 1% Triton X-100, 2 mg/mL lysozyme) at room temperature. These extracts were assayed directly for reverse transcriptase activity by mixing 10 μl of the extract with 40 μl of 1.25×RT reaction mix (62.5 mM Tris-HCl, pH 8.4, 62.5 mM KCl, 12.5 mM MgCl<sub>2</sub>, 12.5 mM DTT, 1.25 mM dGTP, polyC/oligo dG (3.75 mM/1.5 mM in nucleotide), [<sup>32</sup>P] dGTP). This reaction was placed in a 37° C. water bath for 10 minutes. A small aliquot of the reaction mixture (5 μl) was spotted onto a charged nylon membrane (Genescreen+, NEN). The membrane was washed twice with 10% TCA+1% sodium pyrophosphate, rinsed with ethanol, dried, and placed next to a phosphor screen. Radioactive product that had been trapped on the filter was detected by analyzing the screen in a Phosphorimager, using ImageQuant software (Molecular Devices). Candidates were selected if they showed reverse transcriptase activity (radioactivity). These candidates were screened a second time to confirm the phenotype. The confirmed candidates were then sequenced to determine which amino acids maintained detectable reverse transcriptase activity.</div>
<div class="description-paragraph" id="p-0305" num="0331">Purification of Reverse Transcriptase Mutants.</div>
<div class="description-paragraph" id="p-0306" num="0332">The cell pellet containing induced reverse transcriptase was suspended in a ratio of 2 mL Lysis buffer (40 mM Tris-HCl, pH 8.0, 0.1 M KCl, 1 mM PMSF)/1 gram of cell pellet. The suspension was sonicated on ice and then centrifuged at 27,000 g for 30 minutes. The cell-free extract was filtered through a 0.45μ syringe filter. The cell-free extract was applied to a 5 mL Ni<sup>2+</sup> HI-TRAP column (Pharmacia) pre-equilibrated with 5 volumes 5 mM imidazole in buffer A (40 mM Tris HCl, pH 8.0, 10% glycerol, 0.01% Triton X-100, 0.1 M KCl) at 1 mL/min. The column was washed with 10 volumes 5 mM imidazole in buffer A. The reverse transcriptase was eluted by washing with 20 volumes of a gradient of 5 mM to 1M imidazole in buffer A. The eluate containing reverse transcriptase protein was applied to a 1 mL Mono-S column (Pharmacia) pre-equilibrated with 10 column volumes 50 mM KCl in buffer B (40 mM Tris-HCl, pH 8.0, 10% glycerol, 0.01% Triton X-100, 0.1 mM EDTA, 1 mM DTT) at a flow rate of 1.0 mL/min. The column was washed with 10 volumes of 50 mM KCl in buffer B. Reverse transcriptase was eluted with 20 volumes of a gradient from 50 mM to 1 M KCl in buffer B. The individual fractions were analyzed for RT activity. The fraction containing peak RT activity was dialyzed against storage buffer (40 mM Tris-HCl, pH 8.0, 50% glycerol, 0.01% Triton X-100, 0.1 mM EDTA, 1 mM DTT, 0.1 M KCl). The purified reverse transcriptases were more than 95% pure, as judged by SDS-PAGE. The protein concentrations were determined by using the Biorad colorimetric kit.</div>
<div class="description-paragraph" id="p-0307" num="0333">3 dNTP Assay Method.</div>
<div class="description-paragraph" id="p-0308" num="0334">Procedures were modified from those of Preston, B. D., et al. <i>Science </i>242:1168 (1988). The DNA template-primer was prepared by annealing a 47-mer template (5′-GAGTTACAGTGTTTTTGTTCCAGTCTGTAGCAGTGTGTGAATGGAAG-3′) (SEQ ID NO:6) to an 18-mer primer (5′-CTTCCATTCACACACTGC-3′) (SEQ ID NO:7) [<sup>32</sup>P]-labeled at the 5′-end with T4 polynucleotide kinase (template:primer, 3:1). Assay mixture (10 μl) contained 5 nM template-primer, 50-200 nM reverse transcriptase as specified in figure legends, 3 or 4 dNTPs (250 μM each), 50 mM Tris-HCl (pH 8.3), 75 mM KCl, 3 mM MgCl<sub>2</sub>, 10 mM DTT. Reactions were incubated at 37° C. for 30 minutes and terminated by the addition of 5 μl of 40 mM EDTA, 99% formamide. Reaction products were denatured by incubating at 95° C. for 5 minutes and analyzed by electrophoresis on urea 6% polyacrylamide gels.</div>
<div class="description-paragraph" id="p-0309" num="0335">To determine if any TdT activity was occurring in the control reaction of the 3 dNTP assay, which uses all 4 dNTPs, the control reaction was repeated with varying amounts of enzyme, &gt;600 units to 20 units, at 37° C. for 30 minutes. For S<smallcaps>UPER</smallcaps>S<smallcaps>CRIPT</smallcaps>™ II, T197E, and Y133A, 200, 100, 50, and 20 units were used. For F309N, 646, 200, 50, and 20 units were used.</div>
<div class="description-paragraph" id="h-0029" num="0000">Results</div>
<div class="description-paragraph" id="p-0310" num="0336">We carried out a misinsertion assay of F309N(H204R, T306K) S<smallcaps>UPER</smallcaps>S<smallcaps>CRIPT</smallcaps>™ II reverse transcriptase, hereafter referred to as F309N, with DNA template. This assay was employed to compare the misincorporation capability of the mutant to S<smallcaps>UPER</smallcaps>S<smallcaps>CRIPT</smallcaps>™ II. The assay is a primer extension assay using synthetic DNA template-primer and biased dNTP pools containing only three of four dNTPs. The reactions are displayed on a gel in <figref idrefs="DRAWINGS">FIG. 3</figref>. While conducting this procedure to screen for mutants with lower misinsertion/misextension rates it was observed that S<smallcaps>UPER</smallcaps>S<smallcaps>CRIPT</smallcaps>™ II reverse transcriptase extended 2-3 nucleotides past the template end and that some mutations reduced or appeared to eliminate this non-template directed extension or TdT activity. As shown in <figref idrefs="DRAWINGS">FIG. 4</figref>, in the presence of all 4 dNTPs, S<smallcaps>UPER</smallcaps>S<smallcaps>CRIPT</smallcaps>™ II reverse transcriptase and the mutant F309N were able to extend the primer approximately equally, with S<smallcaps>UPER</smallcaps>S<smallcaps>CRIPT</smallcaps>™ II reverse transcriptase adding 2 nucleotides past the template, and F309N adding none beyond the end of the template. To further evaluate this non-templated directed extension the control reaction for the 3 dNTP misextension assay containing all 4 dNTPs was carried out with S<smallcaps>UPER</smallcaps>S<smallcaps>CRIPT</smallcaps>™ II, F309N, T197E, and Y133A reverse transcriptase for 30 minutes with varying amounts of enzyme. The three mutants had shown very reduced levels of TdT activity in prior screens. Since it had been observed that 5 minutes with 20 units of enzyme was more than enough time for the primer extension to be completed, a 30 minute incubation and 200 to 646 units of reverse transcriptase were both in large excess over what was necessary for the reaction to be completed. As seen in <figref idrefs="DRAWINGS">FIG. 4</figref>, all the reverse transcriptase reactions at the lowest amount tested had similar extension products to the reactions at the highest unit concentrations demonstrating that the reaction had gone to completion. S<smallcaps>UPER</smallcaps>S<smallcaps>CRIPT</smallcaps>™ II reverse transcriptase added 2 nucleotides past the end of the template, F309N and T197E did not extend past the end of the template, and Y133A appears to have a small amount of product that is 1 nucleotide past the end of the template.</div>
<heading id="h-0030">Example 4</heading>
<heading id="h-0031">Dual Thermostable and TdT Mutants</heading>
<div class="description-paragraph" id="p-0311" num="0337">The F309 amino acid position in M-MLV reverse transcriptase (RT) aligns with the W266 position in HIV reverse transcriptase. This position is at the base of the thumb domain and is considered part of the minor groove binding tract which interacts with the minor groove of the template-primer. The mutations H204R and T306K have been shown to increase the thermostability of the enzyme. The F309N mutation in an H204R/T306K clone displays 2.3× lower mutation frequency in a lacZ forward assay (Table 5) on RNA template and shorter extension products in a 3 dNTP extension assay than S<smallcaps>UPER</smallcaps>S<smallcaps>CRIPT</smallcaps>™ II reverse transcriptase or H204R/T306K in S<smallcaps>UPER</smallcaps>S<smallcaps>CRIPT</smallcaps>™ II reverse transcriptase. Both findings support the claim of an enzyme with higher fidelity (Table 6).</div>
<div class="description-paragraph" id="p-0312" num="0338">
<tables id="TABLE-US-00005" num="00005">
<patent-tables colsep="0" frame="none" rowsep="0">
<table align="left" class="description-table" cols="1" colsep="0" rowsep="0" width="100%">
<thead>
<tr class="description-tr">
<td class="description-td" colspan="1" nameend="1" namest="1" rowsep="1">TABLE 5</td>
</tr>
</thead>
<tbody><tr class="description-tr">
<td align="center" class="description-td" colspan="1" nameend="1" namest="1" rowsep="1"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">Mutation Frequency of M-MLV Reverse </td>
</tr>
<tr class="description-tr">
<td class="description-td">Transcriptase High Fidelity Mutants</td>
</tr>
</tbody></table>
<table align="left" class="description-table" cols="4" colsep="0" rowsep="0" width="100%">
<tbody><tr class="description-tr">
<td class="description-td">Construct</td>
<td class="description-td">total plaques </td>
<td class="description-td">mutant plaques</td>
<td class="description-td">MF(× 10<sup>−4</sup>)</td>
</tr>
<tr class="description-tr">
<td align="center" class="description-td" colspan="4" nameend="4" namest="1" rowsep="1"> </td>
</tr>
</tbody></table>
<table align="left" class="description-table" cols="4" colsep="0" rowsep="0" width="100%">
<tbody><tr class="description-tr">
<td class="description-td">SUPERSCRIPT ™ II</td>
<td class="description-td">15689</td>
<td class="description-td">87</td>
<td class="description-td">39</td>
</tr>
<tr class="description-tr">
<td class="description-td">SUPERSCRIPT ™ II</td>
<td class="description-td">14410</td>
<td class="description-td">83</td>
<td class="description-td">41</td>
</tr>
<tr class="description-tr">
<td class="description-td">(H204R, T306K)</td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">SUPERSCRIPT ™ II</td>
<td class="description-td">11623</td>
<td class="description-td">39</td>
<td class="description-td">17</td>
</tr>
<tr class="description-tr">
<td class="description-td">(H204R, T306K, F309N)</td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">SUPERSCRIPT ™ II</td>
<td class="description-td">11415</td>
<td class="description-td">39</td>
<td class="description-td">14</td>
</tr>
<tr class="description-tr">
<td class="description-td">(H204R, T306K,</td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">F309N, V223H)</td>
</tr>
<tr class="description-tr">
<td align="center" class="description-td" colspan="4" nameend="4" namest="1" rowsep="1"> </td>
</tr>
<tr class="description-tr">
<td align="left" class="description-td" colspan="4" id="FOO-00001" nameend="4" namest="1">The mutation frequency of SUPERSCRIPT ™ II reverse transcriptase and point mutants. Mutation frequency (MF) was determined by dividing the number of mutant plaques (light blue or white) by the total number of plaques. The background mutant frequency of the starting DNA was 17 × 10<sup>−4 </sup>for the first 3 constructs and 20 × 10<sup>−4 </sup>for the last construct.</td>
</tr>
</tbody></table>
</patent-tables>
</tables>
</div>
<div class="description-paragraph" id="p-0313" num="0339"> <tables id="TABLE-US-00006" num="00006"> <patent-tables colsep="0" frame="none" rowsep="0"> <table align="left" class="description-table" cols="1" colsep="0" rowsep="0" width="100%"> <thead> <tr class="description-tr"> <td class="description-td" colspan="1" nameend="1" namest="1" rowsep="1">TABLE 6</td> </tr> </thead> <tbody><tr class="description-tr"> <td align="center" class="description-td" colspan="1" nameend="1" namest="1" rowsep="1"> </td> </tr> <tr class="description-tr"> <td class="description-td">Error Rates of M-MLV Reverse Transcriptase High Fidelity Mutants</td> </tr> </tbody></table> <table align="left" class="description-table" cols="5" colsep="0" rowsep="0" width="100%"> <tbody><tr class="description-tr"> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td">SUPERSCRIPT ™</td> <td class="description-td"> </td> <td class="description-td">V223H/</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">M-MLV</td> <td class="description-td">II</td> <td class="description-td">F309N</td> <td class="description-td">F309N</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td align="center" class="description-td" colspan="5" nameend="4" namest="offset" rowsep="1"> </td> </tr> </tbody></table> <table align="left" class="description-table" cols="5" colsep="0" rowsep="0" width="100%"> <tbody><tr class="description-tr"> <td class="description-td">Overall ER</td> <td class="description-td">1/17,000</td> <td class="description-td">1/15,000</td> <td class="description-td">1/34,000 </td> <td class="description-td">1/41,000 </td> </tr> <tr class="description-tr"> <td class="description-td">(oER)</td> </tr> <tr class="description-tr"> <td class="description-td">Mismatch</td> </tr> <tr class="description-tr"> <td class="description-td">% of total</td> <td class="description-td">46</td> <td class="description-td">35</td> <td class="description-td">68 72</td> </tr> <tr class="description-tr"> <td class="description-td">ER (mER)</td> <td class="description-td">1/37,000</td> <td class="description-td">1/42,000</td> <td class="description-td">1/50,000 </td> <td class="description-td">1/58,000 </td> </tr> <tr class="description-tr"> <td class="description-td">Frameshift</td> </tr> <tr class="description-tr"> <td class="description-td">% of total</td> <td class="description-td">46</td> <td class="description-td">60</td> <td class="description-td">21 22</td> </tr> <tr class="description-tr"> <td class="description-td">ER (rER)</td> <td class="description-td">1/37,000</td> <td class="description-td">1/25,000</td> <td class="description-td">1/162,000</td> <td class="description-td">1/188,000</td> </tr> <tr class="description-tr"> <td class="description-td">Strand Jump</td> </tr> <tr class="description-tr"> <td class="description-td">% of total</td> <td class="description-td"> 8</td> <td class="description-td"> 5</td> <td class="description-td">11 6 </td> </tr> <tr class="description-tr"> <td class="description-td">ER (jER)</td> <td class="description-td"> 1/213,000</td> <td class="description-td"> 1/297,000</td> <td class="description-td">1/324,000</td> <td class="description-td">1/690,000</td> </tr> <tr class="description-tr"> <td align="center" class="description-td" colspan="5" nameend="5" namest="1" rowsep="1"> </td> </tr> </tbody></table> </patent-tables> </tables> <br/>
Methods
</div>
<div class="description-paragraph" id="p-0314" num="0340">Mutagenesis. Using a standard site directed mutagenesis protocol, as described in Example 3, a primer containing the V223H mutation was annealed to single strand DNA of S<smallcaps>UPER</smallcaps>S<smallcaps>CRIPT</smallcaps>™ II with the following mutations: H204R, T306K, F309N. The colonies were sequenced to confirm the new combination of V223H, H204R, T306K, and F309N.</div>
<div class="description-paragraph" id="p-0315" num="0341">Selecting Colonies Containing Active Reverse Transcriptase. Colony selection was performed as in Example 3.</div>
<div class="description-paragraph" id="p-0316" num="0342">Purification of RT mutants. Purification was performed as in Example 3.</div>
<div class="description-paragraph" id="p-0317" num="0343">Sequencing of plaques. The plaques from the lacZ forward assay were transferred from the soft agar plate to Whatmann 3 MM paper and allowed to dry for at least 1 hour. The plaque was then punched out and the plaque/paper disk was added directly to a sequencing reaction mix containing 4-8 μl ABI PRISM Dye Terminator Cycle Sequencing Ready Reaction (Perkin Elmer), 1 μl primer (GAAGATCGCACTCCAGCCAGC) (SEQ ID NO:8), and distilled water to 20 μl total volume. The ABI cycle sequencing protocol was used for 96° C. 10 seconds, 50° C. 5 seconds, 60° C. 4 minutes for 25 cycles. The paper disks were removed and the reactions were precipitated, then resuspended in loading dye and run on an ABI 377 sequencing machine.</div>
<div class="description-paragraph" id="p-0318" num="0344">The sequences were compared to wild type lacZ alpha sequence and then classified as frameshift (either 1 nucleotide insertion or deletion), mismatch, or strand jump (an insertion or deletion between repeated sequences). The overall error rate for each class was determined by dividing the mutation frequency by the number of detectable sites (i.e., sites the alteration of which results in a phenotypic change) (116) multiplied by 0.5 (to exclude the original single strand contribution) and then multiplied by the percentage of mutants observed to be in each class. ER=MF/(detectable sites*0.5)*(% in each class).</div>
<div class="description-paragraph" id="p-0319" num="0345">3dNTP assay method. 3dNTP assays were performed as in Example 3.</div>
<div class="description-paragraph" id="h-0032" num="0000">Results</div>
<div class="description-paragraph" id="p-0320" num="0346">We carried out a misinsertion assay of F309N(H204R T306K) S<smallcaps>UPER</smallcaps>S<smallcaps>CRIPT</smallcaps>™ II reverse transcriptase, hereafter referred to as F309N, and V223H F309N (H204R T306K), hereafter referred to as V223H/F309N with DNA template. This assay was employed to compare the misincorporation capability of the mutant to S<smallcaps>UPER</smallcaps>S<smallcaps>CRIPT</smallcaps>™ II. The assay is a primer extension assay using synthetic DNA template-primer and biased dNTP pools containing only three of the four dNTPs. The reactions are displayed on a gel in <figref idrefs="DRAWINGS">FIG. 5</figref> and <figref idrefs="DRAWINGS">FIG. 6</figref>. In this assay, higher efficiency of primer extension denotes lower fidelity. As shown in <figref idrefs="DRAWINGS">FIGS. 5 and 6</figref>, in the presence of all 4 dNTPs, S<smallcaps>UPER</smallcaps>S<smallcaps>CRIPT</smallcaps>™ II reverse transcriptase and the mutants F309N and V223H/F309N were able to extend the primer approximately equally, with some variance in the addition of non-template directed nucleotides at the end of the primer. However when incubated with biased pools of nucleotides, S<smallcaps>UPER</smallcaps>S<smallcaps>CRIPT</smallcaps>™ II reverse transcriptase was able to catalyze substantial extension past template nucleotides for which a complementary dNTP was missing, indicating use of incorrect nucleotides and lower fidelity. In <figref idrefs="DRAWINGS">FIG. 5</figref>, the F309N (2) mutant showed shorter extension products than S<smallcaps>UPER</smallcaps>S<smallcaps>CRIPT</smallcaps>™ II reverse transcriptase in each of the biased pools of three dNTPs, indicating less ability to incorporate incorrect nucleotides and thus higher fidelity. In <figref idrefs="DRAWINGS">FIG. 6</figref>, the V223H/F309N mutant was extended with just the dATP and dCTP pools. In each case V223H/F309N also had lower extension products than S<smallcaps>UPER</smallcaps>S<smallcaps>CRIPT</smallcaps>™ II. This corresponds with the results of the lacZα assay where the F309N and V223H/F309N mutants had a lower mutation frequency than S<smallcaps>UPER</smallcaps>S<smallcaps>CRIPT</smallcaps>™ II reverse transcriptase (17×10<sup>−4 </sup>and 14×10<sup>−4 </sup>to 39×10<sup>−4</sup>). The reverse transcriptase with just the H204R T306K mutations without F309N has a mutation frequency similar to S<smallcaps>UPER</smallcaps>S<smallcaps>CRIPT</smallcaps>™ II reverse transcriptase (41×10<sup>−4 </sup>to 39×10<sup>−4</sup>), suggesting that these mutations do not influence fidelity. This data shows a correlation between the misinsertion assay on DNA and the lacZα assay on RNA wherein higher fidelity mutants had both shorter extension products with biased pools of dNTPs and lower mutation frequencies in the lacZα assay.</div>
<heading id="h-0033">Example 5</heading>
<heading id="h-0034">Error Rate Determination</heading>
<div class="description-paragraph" id="p-0321" num="0347">To determine Error Rates, mutant plaques from the lacZ forward assay were sequenced using known methods. The mutations were then classified into one of the following categories: mismatches for misinsertion events, frameshifts for single insertion or deletion events, or jumps for large insertions or deletions caused by jumping between similar sequences. An overall Error Rate was then determined for nucleic acid encoding the lacZ alpha peptide using the following equation:
<br/>
ER(error rate)=MF(mutation frequency)/(number of detectable sites×0.5), where the number of detectable sites is 116.
</div>
<div class="description-paragraph" id="p-0322" num="0348">Not all bases mutated in lacZ forward assays result in a detectable phenotypic change. To determine specific error rates for mismatch, frameshift and jumps, the mutation frequency was modified by multiplying by the percent of the total of each mutant category, and then used to determine the specific error rate. The following is a sequence map of the lacZα peptide in M13mp19 from S<smallcaps>UPER</smallcaps>S<smallcaps>CRIPT</smallcaps>™ II reverse transcriptase and the high fidelity S<smallcaps>UPER</smallcaps>S<smallcaps>CRIPT</smallcaps>™ II H203R T306K F309N reverse transcriptase assays. Underlining indicates deletions; “^” indicates insertions of the base A, T, C, or G shown above; A, T, C, or G shown above the complete sequence indicates mismatches.</div>
<div class="description-paragraph" id="h-0035" num="0000">Map of Mutations Introduced by S<smallcaps>UPER</smallcaps>S<smallcaps>CRIPT</smallcaps>™ II</div>
<div class="description-paragraph" id="p-0323" num="0349">
<tables id="TABLE-US-00007" num="00007">
<patent-tables colsep="0" frame="none" pgwide="1" rowsep="0" tabstyle="monospace">
<table align="left" class="description-table" cols="2" colsep="0" rowsep="0" width="100%">
<tbody><tr class="description-tr">
<td class="description-td">                                                     T C</td>
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">       T                      T                     TC C</td>
</tr>
<tr class="description-tr">
<td class="description-td">AGCGCAACGC AATTAATGTG AGTTAGCTCA CTCATTAGGC ACCCCAGGCT TTACACTTTA </td>
</tr>
<tr class="description-tr">
<td class="description-td">                  1                  1               4</td>
</tr>
<tr class="description-tr">
<td class="description-td">              CG                                  C      CC</td>
</tr>
<tr class="description-tr">
<td class="description-td">TGCTTCCGGC TCGTATGTTG TGTGGAATTG TGAGCGGATA ACAATTTCAC ACAGGAAACA</td>
</tr>
<tr class="description-tr">
<td class="description-td">     1</td>
</tr>
<tr class="description-tr">
<td class="description-td">    C</td>
</tr>
<tr class="description-tr">
<td class="description-td">    CC                CG       C</td>
</tr>
<tr class="description-tr">
<td class="description-td">GCTATG ACC ATG ATT ACG{circumflex over ( )}CCA AGC TTG CAT GCC TGC AGG TCG ACT CTA GAG GAT CCC CGG</td>
</tr>
<tr class="description-tr">
<td class="description-td">                                                   1</td>
</tr>
<tr class="description-tr">
<td class="description-td">                                       T                          AAAA</td>
</tr>
<tr class="description-tr">
<td class="description-td">                                       T A                        AAA</td>
</tr>
<tr class="description-tr">
<td class="description-td">                     T                 T A                         A</td>
</tr>
<tr class="description-tr">
<td class="description-td">                     T      T          T A               T         A    C </td>
</tr>
<tr class="description-tr">
<td class="description-td">GTA CCG AGC TCG AAT TCA CTG GCC GTC GTT{circumflex over ( )}TTA CAA CGT CGT GAC TGG GAA AAC CCT GGC</td>
</tr>
<tr class="description-tr">
<td class="description-td">                                     7                       1   1      1</td>
</tr>
<tr class="description-tr">
<td class="description-td">                                               TTTTT</td>
</tr>
<tr class="description-tr">
<td class="description-td">                                               TTTTT</td>
</tr>
<tr class="description-tr">
<td class="description-td">                                           C   TTTTT</td>
</tr>
<tr class="description-tr">
<td class="description-td">                                           C   TTT</td>
</tr>
<tr class="description-tr">
<td class="description-td">                                           A T T</td>
</tr>
<tr class="description-tr">
<td class="description-td">        <div class="patent-image small-patent-image"><a href="https://patentimages.storage.googleapis.com/fa/bc/7e/ace7c9c9a29220/US09771565-20170926-P00001.png"><img alt="Figure US09771565-20170926-P00001" class="patent-full-image" file="US09771565-20170926-P00001.TIF" he="2.46mm" height="10" id="CUSTOM-CHARACTER-00001" img-content="character" img-format="tif" inline="no" orientation="portrait" src="https://patentimages.storage.googleapis.com/fa/bc/7e/ace7c9c9a29220/US09771565-20170926-P00001.png" wi="3.22mm" width="13"/></a></div>     <div class="patent-image small-patent-image"><a href="https://patentimages.storage.googleapis.com/33/65/cc/250739f23119e5/US09771565-20170926-P00002.png"><img alt="Figure US09771565-20170926-P00002" class="patent-full-image" file="US09771565-20170926-P00002.TIF" he="2.46mm" height="10" id="CUSTOM-CHARACTER-00002" img-content="character" img-format="tif" inline="no" orientation="portrait" src="https://patentimages.storage.googleapis.com/33/65/cc/250739f23119e5/US09771565-20170926-P00002.png" wi="1.78mm" width="7"/></a></div>            <div class="patent-image small-patent-image"><a href="https://patentimages.storage.googleapis.com/33/65/cc/250739f23119e5/US09771565-20170926-P00002.png"><img alt="Figure US09771565-20170926-P00002" class="patent-full-image" file="US09771565-20170926-P00002.TIF" he="2.46mm" height="10" id="CUSTOM-CHARACTER-00003" img-content="character" img-format="tif" inline="no" orientation="portrait" src="https://patentimages.storage.googleapis.com/33/65/cc/250739f23119e5/US09771565-20170926-P00002.png" wi="1.78mm" width="7"/></a></div>        <div class="patent-image small-patent-image"><a href="https://patentimages.storage.googleapis.com/c5/df/fe/c73b605c6cf35a/US09771565-20170926-P00003.png"><img alt="Figure US09771565-20170926-P00003" class="patent-full-image" file="US09771565-20170926-P00003.TIF" he="2.46mm" height="10" id="CUSTOM-CHARACTER-00004" img-content="character" img-format="tif" inline="no" orientation="portrait" src="https://patentimages.storage.googleapis.com/c5/df/fe/c73b605c6cf35a/US09771565-20170926-P00003.png" wi="1.78mm" width="7"/></a></div>   <div class="patent-image small-patent-image"><a href="https://patentimages.storage.googleapis.com/fa/bc/7e/ace7c9c9a29220/US09771565-20170926-P00001.png"><img alt="Figure US09771565-20170926-P00001" class="patent-full-image" file="US09771565-20170926-P00001.TIF" he="2.46mm" height="10" id="CUSTOM-CHARACTER-00005" img-content="character" img-format="tif" inline="no" orientation="portrait" src="https://patentimages.storage.googleapis.com/fa/bc/7e/ace7c9c9a29220/US09771565-20170926-P00001.png" wi="3.22mm" width="13"/></a></div>    <div class="patent-image small-patent-image"><a href="https://patentimages.storage.googleapis.com/c5/df/fe/c73b605c6cf35a/US09771565-20170926-P00003.png"><img alt="Figure US09771565-20170926-P00003" class="patent-full-image" file="US09771565-20170926-P00003.TIF" he="2.46mm" height="10" id="CUSTOM-CHARACTER-00006" img-content="character" img-format="tif" inline="no" orientation="portrait" src="https://patentimages.storage.googleapis.com/c5/df/fe/c73b605c6cf35a/US09771565-20170926-P00003.png" wi="1.78mm" width="7"/></a></div>  <div class="patent-image small-patent-image"><a href="https://patentimages.storage.googleapis.com/33/65/cc/250739f23119e5/US09771565-20170926-P00002.png"><img alt="Figure US09771565-20170926-P00002" class="patent-full-image" file="US09771565-20170926-P00002.TIF" he="2.46mm" height="10" id="CUSTOM-CHARACTER-00007" img-content="character" img-format="tif" inline="no" orientation="portrait" src="https://patentimages.storage.googleapis.com/33/65/cc/250739f23119e5/US09771565-20170926-P00002.png" wi="1.78mm" width="7"/></a></div>  <div class="patent-image small-patent-image"><a href="https://patentimages.storage.googleapis.com/c5/df/fe/c73b605c6cf35a/US09771565-20170926-P00003.png"><img alt="Figure US09771565-20170926-P00003" class="patent-full-image" file="US09771565-20170926-P00003.TIF" he="2.46mm" height="10" id="CUSTOM-CHARACTER-00008" img-content="character" img-format="tif" inline="no" orientation="portrait" src="https://patentimages.storage.googleapis.com/c5/df/fe/c73b605c6cf35a/US09771565-20170926-P00003.png" wi="1.78mm" width="7"/></a></div>  <div class="patent-image small-patent-image"><a href="https://patentimages.storage.googleapis.com/c5/df/fe/c73b605c6cf35a/US09771565-20170926-P00003.png"><img alt="Figure US09771565-20170926-P00003" class="patent-full-image" file="US09771565-20170926-P00003.TIF" he="2.46mm" height="10" id="CUSTOM-CHARACTER-00009" img-content="character" img-format="tif" inline="no" orientation="portrait" src="https://patentimages.storage.googleapis.com/c5/df/fe/c73b605c6cf35a/US09771565-20170926-P00003.png" wi="1.78mm" width="7"/></a></div>    <div class="patent-image small-patent-image"><a href="https://patentimages.storage.googleapis.com/33/65/cc/250739f23119e5/US09771565-20170926-P00002.png"><img alt="Figure US09771565-20170926-P00002" class="patent-full-image" file="US09771565-20170926-P00002.TIF" he="2.46mm" height="10" id="CUSTOM-CHARACTER-00010" img-content="character" img-format="tif" inline="no" orientation="portrait" src="https://patentimages.storage.googleapis.com/33/65/cc/250739f23119e5/US09771565-20170926-P00002.png" wi="1.78mm" width="7"/></a></div>     <div class="patent-image small-patent-image"><a href="https://patentimages.storage.googleapis.com/00/b4/5e/1297d09aee526e/US09771565-20170926-P00004.png"><img alt="Figure US09771565-20170926-P00004" class="patent-full-image" file="US09771565-20170926-P00004.TIF" he="2.46mm" height="10" id="CUSTOM-CHARACTER-00011" img-content="character" img-format="tif" inline="no" orientation="portrait" src="https://patentimages.storage.googleapis.com/00/b4/5e/1297d09aee526e/US09771565-20170926-P00004.png" wi="1.78mm" width="7"/></a></div>     <div class="patent-image small-patent-image"><a href="https://patentimages.storage.googleapis.com/c5/df/fe/c73b605c6cf35a/US09771565-20170926-P00003.png"><img alt="Figure US09771565-20170926-P00003" class="patent-full-image" file="US09771565-20170926-P00003.TIF" he="2.46mm" height="10" id="CUSTOM-CHARACTER-00012" img-content="character" img-format="tif" inline="no" orientation="portrait" src="https://patentimages.storage.googleapis.com/c5/df/fe/c73b605c6cf35a/US09771565-20170926-P00003.png" wi="1.78mm" width="7"/></a></div> </td>
</tr>
<tr class="description-tr">
<td class="description-td">GTT ACC CAA CTT AAT CGC CTT GCA GCA CAT CCC{circumflex over ( )}CCT{circumflex over ( )}TTC{circumflex over ( )}GCC AGC TGG CGT</td>
</tr>
<tr class="description-tr">
<td class="description-td">                                        1    4</td>
</tr>
</tbody></table>
<table align="left" class="description-table" cols="3" colsep="0" rowsep="0" width="100%">
<tbody><tr class="description-tr">
<td class="description-td">
<div class="patent-image"><a href="https://patentimages.storage.googleapis.com/7e/44/27/40869115ea2dde/US09771565-20170926-P00005.png"><img alt="Figure US09771565-20170926-P00005" class="patent-full-image" file="US09771565-20170926-P00005.TIF" he="2.46mm" height="10" id="CUSTOM-CHARACTER-00013" img-content="character" img-format="tif" inline="no" orientation="portrait" src="https://patentimages.storage.googleapis.com/7e/44/27/40869115ea2dde/US09771565-20170926-P00005.png" wi="14.14mm" width="57"/></a></div>
</td>
<td class="description-td">(SEQ ID NO: 9)</td>
<td class="description-td"> </td>
</tr>
</tbody></table>
</patent-tables>
</tables>
</div>
<div class="description-paragraph" id="p-0324" num="0350">
<tables id="TABLE-US-00008" num="00008">
<patent-tables colsep="0" frame="none" rowsep="0">
<table align="left" class="description-table" cols="5" colsep="0" rowsep="0" width="100%">
<thead>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td" colspan="5" nameend="4" namest="offset" rowsep="1">TABLE 7</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td align="center" class="description-td" colspan="5" nameend="4" namest="offset" rowsep="1"> </td>
</tr>
</thead>
<tbody><tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">Insertions</td>
<td class="description-td">40</td>
<td class="description-td">38%</td>
<td class="description-td">60% frameshift</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">(insertion or</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">deletion)</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">Deletions</td>
<td class="description-td">23</td>
<td class="description-td">22%</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">Mismatches</td>
<td class="description-td">36</td>
<td class="description-td">35%</td>
<td class="description-td">35% mismatch</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">Jumps</td>
<td class="description-td">5</td>
<td class="description-td">5%</td>
<td class="description-td"> 5% Jumps</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td align="center" class="description-td" colspan="5" nameend="4" namest="offset" rowsep="1"> </td>
</tr>
</tbody></table>
</patent-tables>
</tables>
</div>
<div class="description-paragraph" id="p-0325" num="0351">
<tables id="TABLE-US-00009" num="00009">
<patent-tables colsep="0" frame="none" rowsep="0">
<table align="left" class="description-table" cols="3" colsep="0" rowsep="0" width="100%">
<thead>
<tr class="description-tr">
<td class="description-td" colspan="3" nameend="3" namest="1" rowsep="1">TABLE 8</td>
</tr>
<tr class="description-tr">
<td align="center" class="description-td" colspan="3" nameend="3" namest="1" rowsep="1"> </td>
</tr>
</thead>
<tbody><tr class="description-tr">
<td class="description-td">Overall Error Rate (oER)</td>
<td class="description-td">1/15,000</td>
<td class="description-td">     (39 × 10<sup>−4</sup>)/(116 × 0.5)</td>
</tr>
<tr class="description-tr">
<td class="description-td">Mismatch Error</td>
<td class="description-td">1/42,500</td>
<td class="description-td">(0.35 × 39 × 10<sup>−4</sup>)/(116 × 0.5)</td>
</tr>
<tr class="description-tr">
<td class="description-td">Rate (mER)</td>
</tr>
<tr class="description-tr">
<td class="description-td">Frameshift Error</td>
<td class="description-td">1/25,000</td>
<td class="description-td">(0.60 × 39 × 10<sup>−4</sup>)/(116 × 0.5)</td>
</tr>
<tr class="description-tr">
<td class="description-td">Rate (fER)</td>
</tr>
<tr class="description-tr">
<td class="description-td">Jumps Error Rate (jER)</td>
<td class="description-td"> 1/297,000</td>
<td class="description-td">(0.05 × 39 × 10<sup>−4</sup>)/(116 × 0.5)</td>
</tr>
<tr class="description-tr">
<td align="center" class="description-td" colspan="3" nameend="3" namest="1" rowsep="1"> </td>
</tr>
</tbody></table>
</patent-tables>
</tables>
</div>
<heading id="h-0036">Example 6</heading>
<heading id="h-0037">Analysis of Enzymatic Activity in Mutant RTs</heading>
<div class="description-paragraph" id="p-0326" num="0352">Reverse transcriptase (RT) (e.g., retroviral RT) is one of the most intensely studied DNA polymerases, both because of its use as an essential tool for the synthesis and cloning of cDNA and because of its importance as a target for inhibition of HIV (Skalka, A. M. and Goff, S. P., <i>Reverse Transcriptase</i>, Cold Spring Harbor Laboratory Press, Cold Spring, N.Y. (1993)). A dichotomy exists however in that HIV RT, the focus of much of recent study (Le Grice, S. F. J. in <i>Reverse Transcriptase </i>(Skalka, A. M. and Goff, S. P., eds) Cold Spring Harbor Laboratory Press, Cold Spring, N.Y., pp. 163-191, (1993)), has not been used widely as a tool for cDNA synthesis. This is because HIV RT has a relatively high error rate (Bebenek, K. and Kunkel, T. A. in <i>Reverse Transcriptase </i>(Skalka, A. M. and Goff, S. P., eds) Cold Spring Harbor Laboratory Press, Cold Spring, N.Y., pp. 85-102 (1993), and because it does not synthesize efficiently full-length copies of long mRNAs in vitro. Other forms of retroviral RT used widely to synthesize cDNA, M-MLV RT and AMV RT, also suffer from these limitations, but to a lesser extent (Bebenek, K. and Kunkel, T. A. in <i>Reverse Transcriptase </i>(Skalka, A. M. and Goff, S. P., eds) Cold Spring Harbor Laboratory Press, Cold Spring, N.Y., pp. 85-102 (1993); Krug, M. S, and Berger, S. L. <i>Methods in Enzymol. </i>152:316-325 (1987); Gerard, G. F. and D'Alessio, J. M. (1993) in <i>Methods in Molecular Biology</i>, Vol 16: <i>Enzymes of Molecular Biology </i>(Burrell, M. M., ed) pp. 73-93, Humana Press, Totowa, N.J. (1993); Gerard, G. F., et al., <i>Molecular Biotechnology </i>8:61-77 (1997)). In addition to polymerase activity, retroviral RT possesses RNase H activity that degrades the RNA in an RNA-DNA hybrid (Moelling, K., et al., <i>Nature New Biology </i>234:240-244 (1971)). The presence of this degradative activity is responsible in part for the limitation on efficient synthesis of long cDNA (Krug, M. S. and Berger, S. L. <i>Methods in Enzymol. </i>152:316-325 (1987), Berger, S. L., et al., <i>Biochem. </i>22:2365-2372 (1983)). The RNase H domain of RT can be mutated to reduce or eliminate RNase H activity while maintaining mRNA-directed DNA polymerase activity (Kotewicz, M. L., et al., <i>Nuc. Acids Res. </i>16:265-277 (1988), DeStefano, J. J., et al., <i>Biochim. Biophys. Acta </i>1219:380-388 (1994)), improving the efficiency of cDNA synthesis (Kotewicz, M. L., et al., <i>Nuc. Acids Res. </i>16:265-277 (1988)).</div>
<div class="description-paragraph" id="p-0327" num="0353">A second significant drawback to copying mRNA is the tendency of RT to pause during cDNA synthesis resulting in the generation of truncated products (Harrison, G. P., et al., <i>Nuc. Acids Res. </i>26:3433-3442 (1998), DeStefano, J. J., et al., <i>J. Biol. Chem. </i>266:7423-7431 (1991)). This pausing is due in part to the secondary structure of RNA (Harrison, G. P., et al., <i>Nuc. Acids Res. </i>26:3433-3442 (1998), Wu, W., et al., <i>J. Virol. </i>70:7132-7142 (1996)). Performing cDNA synthesis at reaction temperatures that melt the secondary structure of mRNA helps to alleviate this problem (Myers, T. W. and Gelfand, D. H., <i>Biochem. </i>30:7661-7666 (1991)). In addition, the oligo(dT)<sub>n </sub>primer often used to initiate cDNA synthesis tends to prime at internal stretches of A residues in mRNA at lower temperatures, resulting in the synthesis of 3′-end truncated cDNA products. M-MLV RT does not efficiently synthesize cDNA from mRNA above 43° C. (Tosh, C., et al., <i>Acta Virol. </i>41:153-155 (1997)). RNase H-minus (H−) M-MLV RT can be used up to 48° C. because in the absence of RNase H activity the mRNA template-DNA product complex is maintained during cDNA synthesis in a structural form that protects RT from thermal inactivation.</div>
<div class="description-paragraph" id="p-0328" num="0354">In an effort to raise the temperature at which M-MLV RT can be used to synthesize cDNA, we have randomly mutagenized the H− M-MLV RT gene and screened for thermal stable mutants. Several thermal stable mutants of H− M-MLV RT were identified and purified enzymes were characterized. We show that when the mutations are present together they increase RT thermal activity by increasing its intrinsic thermal stability without altering catalytic activity.</div>
<div class="description-paragraph" id="h-0038" num="0000">Experimental Procedures</div>
<div class="description-paragraph" id="p-0329" num="0355">Bacterial Strains and Plasmids—<i>E. coli </i>DH10B (Invitrogen) was used in all experiments. Bacterial liquid cultures were grown at 37° C. in EG: 2% tryptone, 1% yeast extract, 0.5% glycerol, 10 mM NaCl, and 1 mM KCl. Solid medium was LB (1% bactotryptone, 0.5% yeast extract, and 86 mM NaCl)+1.5% agar. Selective media included 100 μg/ml ampicillin. For induction of cultures, cells were inoculated into selective EG+0.2% arabinose and grown for 18 hr.</div>
<div class="description-paragraph" id="p-0330" num="0356">RNA and DNA—Chloramphenicol acetyl transferase (CAT) cRNA (˜900 nt) with a 40-nucleotide poly(A) tail at the 3′-end was synthesized by T7 RNA polymerase run-off transcription from linearized plasmid DNA (D'Alessio, J. M. and Gerard, G. F., <i>Nuc. Acids Res. </i>16:1999-2014 (1988)). The cRNA was selected on oligo(dT)-cellulose to ensure the presence of a poly(A) tail. For labeling, the 5′ end of CAT cRNA was dephosphorylated with alkaline phosphatase. (rC)<sub>n</sub>, p(dT)<sub>12-18</sub>, and p(dT)<sub>25-30 </sub>were purchased from Amersham Pharmacia. (rA)<sub>630 </sub>was purchased from Miles. (dG)<sub>15 </sub>and (dT)<sub>20 </sub>were from Invitrogen. A DNA 24-mer complementary to CAT cRNA that annealed between nucleotides 679 and 692 with its 5′ end 146 nucleotides distant from the first base at the 5′ end of the CAT cRNA poly(A) tail was from Invitrogen. Oligonucleotides to prime PCR and perform site-directed mutagenesis were from Invitrogen.</div>
<div class="description-paragraph" id="p-0331" num="0357">M-MLV RT Gene—The M-MLV RT gene used in these studies was derived from pRT601 (Kotewicz, M. L., et al., <i>Nuc. Acids Res. </i>16:265-277 (1988), Gerard, G. F., et al., <i>DNA </i>5:271-279 (1986)). The gene starting at nucleotide 2598 and ending at nucleotide 4628 (Shinnick, T. M., et al., <i>Nature </i>293:543-548 (1981)) was cloned under control of an araD promoter into plasmid pBAD/HisA (Invitrogen). The M-MLV gene was further modified by site-directed mutagenesis without changing amino acid coding to include several unique restriction endonuclease sites that divided the gene into five segments (<figref idrefs="DRAWINGS">FIG. 2A</figref>). The amino end of the protein contained a His<sub>6 </sub>tag to simplify purification that included the following amino acids: MGGSHHHHHHGMASMTGGQQMGRDLYDDDDKH (amino acids 1-32 of SEQ ID NO:2). The carboxy end of the protein contained the additional amino acids NSRLIN present as the result of subcloning from pRT601 (17). In addition, the M-MLV RT gene was mutated (D524G, E562Q, D583N) to eliminate RNase H activity. The final construct was designated pBAD-HSS2 (<figref idrefs="DRAWINGS">FIG. 2B</figref>), and the gene and gene product were designated His<sub>6 </sub>H− RT.</div>
<div class="description-paragraph" id="p-0332" num="0358">DNA Sequencing—Plasmid DNA was prepared from an over night <i>E. coli </i>culture bearing an RT mutant using a Concert High Purity Miniprep Kit (Invitrogen) following the manufacturer's instructions. Each DNA was sequenced using a forward and reverse primer bordering the segment that had been mutagenized to generate the mutant. The sequencing reactions were carried out as specified for plasmid DNA using the ABI Big Dye Terminator Sequencing Ready Reaction Kit. The reactions were analyzed using an ABI PRISM 377 DNA Sequencer.</div>
<div class="description-paragraph" id="p-0333" num="0359">Random Mutagenesis—Mutagenic PCR was performed in a reaction mixture (100 μl) containing 20 mM Tris-HCl (pH 8.4), 50 mM KCl, 1.8 mM MgCl<sub>2</sub>, 0.3 mM MnSO<sub>4</sub>, 200 μM each of dCTP, dGTP, dTTP, and dATP, and 0.5 units of Taq DNA polymerase (Invitrogen Corporation, Carlsbad, Calif.). After a 1 min denaturation step at 94° C., the cycling protocol was 15 sec at 94° C., 15 sec at 55° C., and 30 sec at 72° C. for 20 cycles. Amplification was 100 fold from 50 ng of target to 5 μg of amplified product. PCR primers included appropriate restriction endonuclease cut sites. An amplified DNA segment was cleaved with appropriate restriction endonucleases, gel purified, and cloned into gel purified vector DNA cut with the corresponding restriction enzymes.</div>
<div class="description-paragraph" id="p-0334" num="0360">Site-directed Mutagenesis—Oligonucleotide site-directed mutagenesis was carried out by established procedures (Kunkel, T. A., et al., <i>Methods Enzymol. </i>154:812-819 (1987), Kunkel, T. A., et al., <i>Methods Enzymol. </i>204:125-139 (1991)). Saturation of an amino acid coding site in the H− RT gene with all possible amino acids was performed by introducing the sequence NNK (N=A, C, G or T and K=G or T) at the codon site into the mutagenic oligonucleotide.</div>
<div class="description-paragraph" id="p-0335" num="0361">DNA Polymerase Assays—During screening of RT mutants, RT RNA-directed DNA polymerase activity was assayed with (rC)<sub>n</sub>●(dG)<sub>15</sub>, which is specific for RT under the reaction conditions used (Gerard, G. F., et al., <i>Biochem. </i>13:1632-1641 (1974)). Reaction mixtures (50 μl) containing 50 mM Tris-HCl (pH 8.4), 50 mM KCl, 10 mM MgCl<sub>2</sub>, 300 μM (rC)<sub>n</sub>, 120 μM (dG)<sub>15</sub>, 0.01% (v/v) Triton X-100, and 100 μM [α-<sup>32</sup>P]dGTP (1,000 cpm/pmole), were incubated at 37° C. for 10 min in the wells of a 96-well plate. An aliquot (5 μl) from each well was spotted onto a Genescreen+ (NEN) filter and the filter was washed twice for 10 min with 4% (w/v) sodium pyrophosphate (pH 8.0). Radioactivity bound to the dried filter was quantified in a phosphorimager (Molecular Dynamics).</div>
<div class="description-paragraph" id="p-0336" num="0362">RT DNA polymerase unit activity was assayed with (rA)<sub>630</sub>● p(dT)<sub>12-18 </sub>(Houts, G. E., et al., <i>J. Virol. </i>29:517-522 (1979)). One unit of DNA polymerase activity is the amount of RT that incorporates one nmole of deoxynucleoside triphosphate into acid insoluble product at 37° C. in 10 min.</div>
<div class="description-paragraph" id="p-0337" num="0363">cDNA synthesis from CAT cRNA was carried out in reaction mixtures (20 μl) containing 50 mM Tris-HCl (pH 8.4), 75 mM KCl, 3 mM MgCl<sub>2</sub>, 10 mM dithiotreitol (DTT), 500 μM each of dATP, dTTP, dGTP, and [α-<sup>32</sup>P]dCTP (300 cpm/pmole), 1,750 units/ml RNase Inhibitor, 130 μg/ml (465 nM) CAT cRNA, 20 μg/ml (2,300 nM) p(dT)<sub>25-30</sub>, and 3,250 units/ml (100 nM) RT. Incubation was at various temperatures for 60 min in individual tubes. An aliquot of the reaction mixture was precipitated with TCA to determine yield of cDNA synthesized, and the remaining cDNA product was size fractionated on an alkaline 1.2% agarose gel (McDonell, M. W., et al., <i>J. Mol. Biol. </i>110:199-146 (1977)).</div>
<div class="description-paragraph" id="p-0338" num="0364">To establish mono- and divalent metal reaction optima, initial reaction rates were determined under conditions of limiting RT concentration during a 10-min incubation at 37 or 50° C. Reaction mixtures (20 μl) contained 50 mM Tris-HCl (pH 8.4), 10 mM DTT, 500 μM each of dTTP, dATP, dCTP, and [<sup>3</sup>H]dGTP (100 cpm/pmole), 10 pmoles (2.8 μg) CAT cRNA, 50 pmoles DNA 24-mer, 0.5 pmoles RT, and KCl and MgCl<sub>2</sub>, varied in concentration one at a time.</div>
<div class="description-paragraph" id="p-0339" num="0365">Steady-state Kinetic Measurements—The steady-state kinetic parameters K<sub>m(dTTP) </sub>and k<sub>cat </sub>were determined as described (Polesky, A. H., et al., <i>J. Biol. Chem. </i>265:14579-14591 (1990)), using (A)<sub>n</sub>●(dT)<sub>30</sub>. A range of five [<sup>32</sup>P]dTTP concentrations, which bracketed the K<sub>m(dTTP) </sub>value, was used for each determination of the kinetic parameters. The concentration of the template-primer and enzyme in the reaction were 2 μM (in primer termini) and 4 μM, respectively. Reaction mixtures (50 μl) also contained 50 mM Tris-HCl, pH 8.4, 75 mM KCl, 3 mM MgCl<sub>2</sub>, and 10 mM DTT and were incubated at 37° C.</div>
<div class="description-paragraph" id="p-0340" num="0366">Mutant Screening—Mutant populations were plated on selective agar. Individual colonies were inoculated into 120 μl of selective EG in 96 well plates and grown overnight at 37° C. The cell density of the cultures was ˜10<sup>9 </sup>cfu/ml. An aliquot of the culture from each well was transferred into a new 96-well plate containing 120 piper well of selective media +0.2% arabinose. This plate was incubated at 37° C. for 20 hr to induce expression of RT. An aliquot (5 μl) of culture from each well was then transferred into another 96-well plate containing 5 μl per well of 2×PLD (50 mM Tris-HCl, pH 8.0, 20 mM EDTA, 1.8% (w/v) sucrose, 1% (v/v) Triton X-100, 10 mM DTT, and 2 mg/ml lysozyme). After heating for 5 to 10 min at various temperatures (52 to 58° C.) in a thermocycler and cooling to room temperature, RT activity in a 5-μl aliquot from the lysate in each well was assayed with (rC)<sub>n</sub>●(dG)<sub>15 </sub>in another 96-well plate as described above. The amount of RT activity in a 5-μl aliquot of extract was within the linear range of the assay. Lysates were stable at room temperature for at least 1 hr.</div>
<div class="description-paragraph" id="p-0341" num="0367">Purification of RTs—All operations were at 4° C. Induced <i>E. coli </i>cells (5 g) bearing pBAD-HSS2 or a derivative were suspended in 10 mL of buffer (20 mM Tris-HCl, pH 8.0, 100 mM KCl, and 1 mM PMSF) and were sonicated for disruption. After clarification by centrifugation at 20,000×g for 30 min, RT was purified by sequential chromatography on a 5-ml Chelating Sepharose column charged with Ni<sup>2+</sup> and a Mono S HR 5/5 column (Amersham Pharmacia). In some cases the RT was fractionated on a third column (AF-Heparin-650 from TosoHaas) to eliminate traces of RNase contamination. The purified RT was dialyzed against storage buffer (40 mM Tris-HCl, pH 8.0, 100 mM KCl, 0.01% (v/v) Triton X-100, 0.1 mM EDTA, 1 mM DTT, and 50% v/v glycerol) and stored at −20° C. RT purified by this procedure was &gt;90% homogeneous as judged by SDS PAGE and was free of detectable contaminating DNA endonuclease, DNA exonuclease, and RNA endonuclease.</div>
<div class="description-paragraph" id="p-0342" num="0368">Thermal Inactivation Profile of RT in Extracts—Cell lysates prepared in PLD as described already were heated in a 96-well plate in a thermocycler in which a temperature gradient exists through the rows, but the temperature in each column is the same. After incubation for 5 min at temperatures ranging from 25° C. to 56° C., RT activity was assayed with (rC)<sub>n</sub>●(dG)<sub>15 </sub>as described already.</div>
<div class="description-paragraph" id="p-0343" num="0369">Half Life Determination—Mixtures (20 μl) were incubated for various times in 0.5-ml tubes in a thermocycler at 50 or 55° C. and contained 50 mM Tris-HCl (pH 8.4), 75 mM KCl, 3 mM MgCl<sub>2</sub>, 10 mM DTT, 0.1% (v/v) Triton X-100, and 3-7 μg/ml purified RT. Incubation was stopped by placing the tube in ice. An aliquot (5 μl) was assayed for residual activity with (rA)<sub>630</sub>● p(dT)<sub>12-18</sub>.</div>
<div class="description-paragraph" id="p-0344" num="0370">Measurement of K<sub>D </sub>by Filter Binding—A nitrocellulose filter-binding assay (Bailey, J. M., <i>Anal. Biochem. </i>93:204-206 (1979), Strauss, H. S., et al., <i>Gene </i>13:75-87 (1981)) was used to determine the nucleic acid binding constants (K<sub>D</sub>) of RTs for CAT cRNA●(dT)<sub>20</sub>. Dephosphorylated CAT cRNA was labeled at the 5′ end with [γ-<sup>32</sup>P]ATP and T4 polynucleotide kinase (Boehringer). Oligo(dT)<sub>20 </sub>was annealed to the poly(A)-tailed CAT cRNA in a buffer containing 10 mM Tris-HCl, pH 7.5, and 80 mM KCl at 65° C. for 5 min followed by chilling on ice. Reaction mixtures (100 μl) containing binding buffer (50 mM Tris-HCl, pH 8.4, 75 mM KCl, 3 mM MgCl<sub>2</sub>, and 10 mM DTT), 0.05 nM <sup>32</sup>P-labeled CAT cRNA, 1 nM (dT)<sub>20</sub>, and 1 to 50 nM RT were incubated at 23° C. for 5 min. After incubation, the mixture was filtered through a nitrocelullose filter (Millipore, HA 0.45 mm) soaked in binding buffer, which was then washed with binding buffer. The K<sub>D </sub>is equal to that enzyme concentration at which 50% of the labeled CAT cRNA is bound. For this method of analysis to be valid, the CAT cRNA concentration in the reaction must be substantially below K<sub>D</sub>, so that the total enzyme concentration approximates the concentration of free unbound enzyme.</div>
<div class="description-paragraph" id="h-0039" num="0000">Results</div>
<div class="description-paragraph" id="p-0345" num="0371">Mutants Generated by Random Mutagenesis—PCR primers were designed to amplify each of five segments of the M-MLV His<sub>6 </sub>H− RT gene (<figref idrefs="DRAWINGS">FIG. 2A</figref>) and to contain appropriate restriction endonuclease cut sites. Each segment was randomly mutagenized by PCR mutagenesis (Experimental Procedures), digested with the appropriate restriction endonucleases, and cloned into a similarly digested pBAD-HSS2 plasmid in such a way as to replace the corresponding segment in the H− RT gene with the mutagenized segment (<figref idrefs="DRAWINGS">FIG. 2B</figref>). Libraries were sampled by DNA sequencing of the mutagenized H− RT gene segment of a small number of clones to determine the mutation frequency. The goal was to achieve rates of PCR random mutagenesis that produced 1 to 2 mutations per segment. We found that random mutagenesis that produced greater than 2 mutations per segment tended to produce a large proportion of inactive RT mutants. If the library met this criterion, further screening by heat treatment was carried out to identify mutants that showed greater RT activity in lysates than H− RT after a heat treatment step (Experimental Procedures; <figref idrefs="DRAWINGS">FIG. 7A</figref>). Those mutants with the highest apparent thermal stability were screened again in duplicate by pretreatment at 24° C. (to normalize for activity) and at 52 to 58° C. to confirm the presence of enhanced thermal stability (<figref idrefs="DRAWINGS">FIG. 7B</figref>). In all, about 15,000 clones were screened by heat treatment in the 96-well format for each segment or a total of about 100,000 mutants.</div>
<div class="description-paragraph" id="p-0346" num="0372">Two mutants were found that produced RT with greater thermal stability than H-RT. DNA sequencing of their genes revealed them to be H204R in segment 2 and T306R in segment 3. Having identified two amino acid locations that influence RT thermal stability, we wished to establish if the particular mutant amino acid selected gave the greatest possible increase in thermal stability. The two sites in the H− RT gene were independently subjected to site-directed mutagenesis in which the sequence NNK was substituted for the existing codon in the mutagenic oligonucleotide (Experimental Procedures). Eighty-four members from each of these libraries were screened for thermal stable mutants. The results showed that T306K and T306R were the most thermal stable variants at T306, with T306K being somewhat more thermal stable, and H204R was the most thermal stable variant at H204.</div>
<div class="description-paragraph" id="p-0347" num="0373">The mutations were combined in one clone by sequential site-directed mutagenesis. Assay of heat-treated extracts showed that the double mutant His<sub>6 </sub>H− H204R T306K RT was more thermal stable than either mutant alone and much more thermal stable than His<sub>6 </sub>H− RT (<figref idrefs="DRAWINGS">FIG. 8</figref>).</div>
<div class="description-paragraph" id="p-0348" num="0374">Utilizing plasmid pBAD-HSS2 that contained the H204R and T306K mutations as a starting point, RT gene segments 1 through 5 were again randomly mutagenized and libraries were screened for additional mutations that rendered His<sub>6 </sub>H− H204R T306K RT more thermal stable. One additional mutation, M289L, in segment 3 was identified that increased the thermal stability of the RT double mutant in crude extracts.</div>
<div class="description-paragraph" id="p-0349" num="0375">Mutants from Single-Site Mutagenesis—His<sub>6 </sub>H− T306K RT was mutated at position F309 to F309N by site-directed mutagenesis as part of a study of the fidelity of H− RT. The F309N mutation increased the thermal stability of His<sub>6 </sub>H− T306K RT in crude extracts.</div>
<div class="description-paragraph" id="p-0350" num="0376">Thermal Stability of Purified Mutant RTs—A number of RT mutants were purified to near homogeneity (Experimental Procedures) in order to establish their intrinsic thermal stability in the absence of contaminants and to characterize their enzymatic properties. The intrinsic thermal stability of purified forms of single mutants of His<sub>6 </sub>H− RT at each of the 4 positions were established and compared with that of the starting enzyme. Mutations T306K, M289L, or F309N increased the half-life of His<sub>6 </sub>H− RT a small amount at 50° C. from 8 min to between 10 and 13 min (Table 9).</div>
<div class="description-paragraph" id="p-0351" num="0377">
<tables id="TABLE-US-00010" num="00010">
<patent-tables colsep="0" frame="none" rowsep="0">
<table align="left" class="description-table" cols="1" colsep="0" rowsep="0" width="100%">
<thead>
<tr class="description-tr">
<td class="description-td" colspan="1" nameend="1" namest="1" rowsep="1">TABLE 9</td>
</tr>
</thead>
<tbody><tr class="description-tr">
<td align="center" class="description-td" colspan="1" nameend="1" namest="1" rowsep="1"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">Half-lives of purified RT mutants at 50° C.</td>
</tr>
</tbody></table>
<table align="left" class="description-table" cols="2" colsep="0" rowsep="0" width="100%">
<tbody><tr class="description-tr">
<td class="description-td">Enzyme</td>
<td class="description-td">Half-life at 50° C. (min)<sup>a</sup> </td>
</tr>
<tr class="description-tr">
<td align="center" class="description-td" colspan="2" nameend="2" namest="1" rowsep="1"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">H-RT (SUPERSCRIPT ™ II)</td>
<td class="description-td"> 3.2 ± 0.2</td>
</tr>
<tr class="description-tr">
<td class="description-td">His<sub>6 </sub>H-RT</td>
<td class="description-td">  8 ± 1.2 (1.6 ± 0.3)<sup>b</sup> </td>
</tr>
<tr class="description-tr">
<td class="description-td">His<sub>6 </sub>H-H204R RT</td>
<td class="description-td"> 43 ± 9</td>
</tr>
<tr class="description-tr">
<td class="description-td">His<sub>6 </sub>H-T306K RT</td>
<td class="description-td"> 10 ± 0.4</td>
</tr>
<tr class="description-tr">
<td class="description-td">His<sub>6 </sub>H-F309N RT</td>
<td class="description-td"> 13 ± 3</td>
</tr>
<tr class="description-tr">
<td class="description-td">His<sub>6 </sub>H-M289L RT</td>
<td class="description-td"> 13 ± 1</td>
</tr>
<tr class="description-tr">
<td class="description-td">His<sub>6 </sub>H-H204R T306K RT</td>
<td class="description-td"> 78 ± 8</td>
</tr>
<tr class="description-tr">
<td class="description-td">H-H204R T306K F309N RT</td>
<td class="description-td">30</td>
</tr>
<tr class="description-tr">
<td class="description-td">His<sub>6 </sub>H-H204R T306K F309N RT</td>
<td class="description-td">105 ± 11</td>
</tr>
<tr class="description-tr">
<td class="description-td">His<sub>6 </sub>H-H204R M289L T306K F309N RT</td>
<td class="description-td">240 ± 10 (8.1 ± 0.3)</td>
</tr>
<tr class="description-tr">
<td align="center" class="description-td" colspan="2" nameend="2" namest="1" rowsep="1"> </td>
</tr>
<tr class="description-tr">
<td align="left" class="description-td" colspan="2" id="FOO-00002" nameend="2" namest="1"> <sup>a</sup>Mean ± standard deviation of two or three determinations</td>
</tr>
<tr class="description-tr">
<td align="left" class="description-td" colspan="2" id="FOO-00003" nameend="2" namest="1"> <sup>b</sup>Half-lives at 55° C. are in parentheses</td>
</tr>
</tbody></table>
</patent-tables>
</tables>
</div>
<div class="description-paragraph" id="p-0352" num="0378">The single mutation with the greatest impact was H204R, which increased the half-life of RT at 50° C. 5-fold (Table 9). However, combining the mutations had at a minimum a positive additive effect upon thermal stability. Combining H204R and T306K increased the half-life at 50° C. 10-fold, and combining M289L and F309N with these two mutations increased the half-life at 50° C. another 3-fold to 240 min (a 30-fold increase relative to the half-life of the starting enzyme; Table 9).</div>
<div class="description-paragraph" id="p-0353" num="0379">The half-lives of His<sub>6 </sub>H− RT and all its mutants dropped off at 55° C. relative to 50° C. (Table 9). The magnitude of the relative increase in intrinsic half-life observed at 50° C. for the quadruple mutant was maintained partially at 55° C. (Table 9). The half-life of His<sub>6 </sub>H− RT of 1.6 min at 55° C. was increased 5-fold by H204R M289L T306K F309N to 8.1 min (Table 9).</div>
<div class="description-paragraph" id="p-0354" num="0380">The presence of an N-terminal tag was seen to increase thermal stability of the RTs examined. A comparison of an RT with no N-terminal tag (S<smallcaps>UPER</smallcaps>S<smallcaps>CRIPT</smallcaps>™ II) with an RT having the same point mutations and having an N-terminal tag with the sequence MGGSHHHHHHGMASMTGGQQMGRDLYDDDDKH corresponding to amino acids 1-32 of SEQ ID NO:2 (His<sub>6 </sub>H− RT) shows that the presence of the tag increases the thermal stability of the RT. The presence of the tag increased the half-life of the RT at 50° C. by a factor of 2.5-fold (from 3.2 minutes to 8 minutes). A comparison of the triple mutant H204R T306K F309N with and without tag showed an increase in the half-life of the enzyme at 50° C. by a factor of 3.5-fold (30 minutes to 105 minutes). Thus, the present invention contemplates RTs with an increased thermal stability that comprise one or more amino acids added to the N-terminal of the RT. In addition, the invention contemplates RTs with enhanced thermal stability that comprise one or more amino acids added to the C-terminal of the RT.</div>
<div class="description-paragraph" id="p-0355" num="0381">Enzymatic Characterization of Purified Mutant RTs—A number of catalytic properties of the purified RT mutants were compared to ascertain if introduction of all four point mutations in RT altered its DNA synthetic activity. With one exception, all of the thermal stable mutants characterized had RNA-directed DNA polymerase specific activities greater than that of starting His<sub>6 </sub>H− RT, but within a factor of 1.5-fold (Table 10).</div>
<div class="description-paragraph" id="p-0356" num="0382">
<tables id="TABLE-US-00011" num="00011">
<patent-tables colsep="0" frame="none" rowsep="0">
<table align="left" class="description-table" cols="1" colsep="0" rowsep="0" width="100%">
<thead>
<tr class="description-tr">
<td class="description-td" colspan="1" nameend="1" namest="1" rowsep="1">TABLE 10</td>
</tr>
</thead>
<tbody><tr class="description-tr">
<td align="center" class="description-td" colspan="1" nameend="1" namest="1" rowsep="1"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">RNA-directed DNA polymerase specific activity of purified RT mutants</td>
</tr>
</tbody></table>
<table align="left" class="description-table" cols="2" colsep="0" rowsep="0" width="100%">
<tbody><tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">DNA Polymerase Specific</td>
</tr>
<tr class="description-tr">
<td class="description-td">Enzyme</td>
<td class="description-td">Activity (units/μg)</td>
</tr>
<tr class="description-tr">
<td align="center" class="description-td" colspan="2" nameend="2" namest="1" rowsep="1"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">His<sub>6 </sub>H-RT</td>
<td class="description-td">281 (1.0)<sup>a</sup> </td>
</tr>
<tr class="description-tr">
<td class="description-td">His<sub>6 </sub>H-H204R RT</td>
<td class="description-td">325 (1.16)</td>
</tr>
<tr class="description-tr">
<td class="description-td">His<sub>6 </sub>H-T306K RT</td>
<td class="description-td">402 (1.43)</td>
</tr>
<tr class="description-tr">
<td class="description-td">His<sub>6 </sub>H-F309N RT</td>
<td class="description-td">257 (0.91)</td>
</tr>
<tr class="description-tr">
<td class="description-td">His<sub>6 </sub>H-M289L RT</td>
<td class="description-td">339 (1.21)</td>
</tr>
<tr class="description-tr">
<td class="description-td">His<sub>6 </sub>H-H204R T306K RT</td>
<td class="description-td">385 (1.37)</td>
</tr>
<tr class="description-tr">
<td class="description-td">His<sub>6 </sub>H-H204R T306K F309N RT</td>
<td class="description-td">391 (1.39)</td>
</tr>
<tr class="description-tr">
<td class="description-td">His<sub>6 </sub>H-H204R M289L T306K F309N RT</td>
<td class="description-td">410 (1.46)</td>
</tr>
<tr class="description-tr">
<td align="center" class="description-td" colspan="2" nameend="2" namest="1" rowsep="1"> </td>
</tr>
<tr class="description-tr">
<td align="left" class="description-td" colspan="2" id="FOO-00004" nameend="2" namest="1"> <sup>a</sup>Ratio of specific activity setting that of His<sub>6 </sub>H-RT at 1.0.</td>
</tr>
</tbody></table>
</patent-tables>
</tables>
</div>
<div class="description-paragraph" id="p-0357" num="0383">His<sub>6 </sub>H− F309N RT had a slightly reduced DNA polymerase specific activity (Table 10). The catalytic efficiency (k<sub>cat</sub>/K<sub>m</sub>) of His<sub>6 </sub>H− RT and His<sub>6 </sub>H− H204R M289L T306K F309N were within a factor of two of each other and their Kms for nucleotide substrate were similar (Table 11).</div>
<div class="description-paragraph" id="p-0358" num="0384">
<tables id="TABLE-US-00012" num="00012">
<patent-tables colsep="0" frame="none" rowsep="0">
<table align="left" class="description-table" cols="1" colsep="0" rowsep="0" width="100%">
<thead>
<tr class="description-tr">
<td class="description-td" colspan="1" nameend="1" namest="1" rowsep="1">TABLE 11</td>
</tr>
</thead>
<tbody><tr class="description-tr">
<td align="center" class="description-td" colspan="1" nameend="1" namest="1" rowsep="1"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">Catalytic constants of purified RT mutants<sup>a</sup> </td>
</tr>
</tbody></table>
<table align="left" class="description-table" cols="5" colsep="0" rowsep="0" width="100%">
<tbody><tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">Enzyme</td>
<td class="description-td">K<sub>m </sub>(μM)</td>
<td class="description-td">k<sub>cat </sub>(sec<sup>−1</sup>)</td>
<td class="description-td">k<sub>cat</sub>/K<sub>m</sub> </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td align="center" class="description-td" colspan="5" nameend="4" namest="offset" rowsep="1"> </td>
</tr>
</tbody></table>
<table align="left" class="description-table" cols="5" colsep="0" rowsep="0" width="100%">
<tbody><tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">His<sub>6 </sub>H-RT</td>
<td class="description-td">390 ± 98</td>
<td class="description-td">45 ± 18</td>
<td class="description-td">0.115</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">His<sub>6 </sub>H-H204R M289L</td>
<td class="description-td">274 ± 92</td>
<td class="description-td">46 ± 11</td>
<td class="description-td">0.17</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">T306K F309N RT</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td align="center" class="description-td" colspan="5" nameend="4" namest="offset" rowsep="1"> </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td align="left" class="description-td" colspan="5" id="FOO-00005" nameend="4" namest="offset"> <sup>a</sup>Mean ± standard deviation of three determinations</td>
</tr>
</tbody></table>
</patent-tables>
</tables>
</div>
<div class="description-paragraph" id="p-0359" num="0385">The monovalent and divalent metal ion optima of His<sub>6 </sub>H− RT and His<sub>6 </sub>H− H204R M289L T306K F309N RT were determined. CAT cRNA● DNA 24-mer in excess over enzyme was used as template-primer (Experimental Procedures). The optima were the same: 75 mM KCl and 3 mM MgCl<sub>2 </sub>at 37° C. for His<sub>6 </sub>H− RT and His<sub>6 </sub>H− H204R M289L T306K F309N and at 50° C. for the quadruple point mutant. Taken together these results indicate that the four mutations identified that substantially increased the thermal stability of H− RT did not appreciably affect its DNA polymerase catalytic capability.</div>
<div class="description-paragraph" id="p-0360" num="0386">In the presence of a template-primer, the half-life of M-MLV H− RT at 50° C. is increased by a factor of 3 to 4 relative to the half-life in the absence of nucleic acids. This is the result of the enzyme being more resistant to heat inactivation when bound to template-primer than when unbound in solution. Mutations that impact the affinity of RT for template-primer will affect the apparent thermal stability of RT when it is engaged in cDNA synthesis. Therefore the affinities (K<sub>D</sub>) of His<sub>6 </sub>H− RT and several of its mutants for CAT cRNA●(dT)<sub>20 </sub>were determined (Table 12).</div>
<div class="description-paragraph" id="p-0361" num="0387">
<tables id="TABLE-US-00013" num="00013">
<patent-tables colsep="0" frame="none" rowsep="0">
<table align="left" class="description-table" cols="1" colsep="0" rowsep="0" width="100%">
<thead>
<tr class="description-tr">
<td class="description-td" colspan="1" nameend="1" namest="1" rowsep="1">TABLE 12</td>
</tr>
</thead>
<tbody><tr class="description-tr">
<td align="center" class="description-td" colspan="1" nameend="1" namest="1" rowsep="1"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">Nucleic acid dissociation constants of purified RT mutants</td>
</tr>
</tbody></table>
<table align="left" class="description-table" cols="3" colsep="0" rowsep="0" width="100%">
<tbody><tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">Enzyme</td>
<td class="description-td">K<sub>D </sub>(nM)<sup>a,b</sup> </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td align="center" class="description-td" colspan="3" nameend="2" namest="offset" rowsep="1"> </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">His<sub>6 </sub>H-RT</td>
<td class="description-td">  8 ± 0.5</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">His<sub>6 </sub>H-H204R RT</td>
<td class="description-td"> 35 ± 0.5</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">His<sub>6 </sub>H-T306K RT</td>
<td class="description-td">6.6 ± 0.4</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">His<sub>6 </sub>H-F309N RT</td>
<td class="description-td"> 28 ± 2.5</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">His<sub>6 </sub>H-M289L RT</td>
<td class="description-td">4.7 ± 0.7</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">His<sub>6 </sub>H-H204R T306K RT</td>
<td class="description-td">6.1 ± 1.5</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">His<sub>6 </sub>H-H204R T306K F309N RT</td>
<td class="description-td">7.3 ± 2.4</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">His<sub>6 </sub>H-H204R M289L T306K F309N RT</td>
<td class="description-td"> 11 ± 1.0</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td align="center" class="description-td" colspan="3" nameend="2" namest="offset" rowsep="1"> </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td align="left" class="description-td" colspan="3" id="FOO-00006" nameend="2" namest="offset"> <sup>a</sup>Mean ± standard deviation of two or three determinations</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td align="left" class="description-td" colspan="3" id="FOO-00007" nameend="2" namest="offset"> <sup>b</sup>The nucleic acid was CAT cRNA•(dT)<sub>20</sub> </td>
</tr>
</tbody></table>
</patent-tables>
</tables>
</div>
<div class="description-paragraph" id="p-0362" num="0388">The K<sub>D </sub>of the His<sub>6 </sub>H− RT quadruple mutant was increased somewhat from 8 nM to 11 nM, indicating that nucleic acid binding affinity was reduced slightly by the four point mutations together. Interestingly two of the point mutations when present alone, H204R and F309N, reduced the binding affinity of His<sub>6 </sub>H− RT substantially more, having a K<sub>D </sub>of 35 and 28, respectively. The effects of these mutations on binding were apparently counter balanced by mutation T306K (K<sub>D</sub>=6.6 nM), since the triple mutant H204R T306K F309N had a K<sub>D </sub>of 7.3 nM. We conclude that when the triple (H204R T306K F309N) or quadruple mutant of His<sub>6 </sub>H− RT is engaged in cDNA synthesis, any increase in apparent thermal stability observed relative to His<sub>6 </sub>H− RT is not due to increased protection by virtue of binding more tightly to template-primer.</div>
<div class="description-paragraph" id="p-0363" num="0389">Practical Implication of Higher RT Thermal Stability—To assess the practical impact of the increases in thermal stability imparted by the H− RT mutations selected in this study, we measured the ability of some of the mutants to synthesize full-length CAT cDNA between 40 and 55° C. (Table 13).</div>
<div class="description-paragraph" id="p-0364" num="0390">
<tables id="TABLE-US-00014" num="00014">
<patent-tables colsep="0" frame="none" rowsep="0">
<table align="left" class="description-table" cols="1" colsep="0" rowsep="0" width="100%">
<thead>
<tr class="description-tr">
<td class="description-td" colspan="1" nameend="1" namest="1" rowsep="1">TABLE 13</td>
</tr>
</thead>
<tbody><tr class="description-tr">
<td align="center" class="description-td" colspan="1" nameend="1" namest="1" rowsep="1"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">Synthesis of full-length cDNA product from CAT cRNA</td>
</tr>
<tr class="description-tr">
<td class="description-td">by purified RT mutants at elevated temperatures<sup>a</sup> </td>
</tr>
</tbody></table>
<table align="left" class="description-table" cols="2" colsep="0" rowsep="0" width="100%">
<tbody><tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">Amount of Full Length </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">Product (ng) Synthesized at</td>
</tr>
</tbody></table>
<table align="left" class="description-table" cols="6" colsep="0" rowsep="0" width="100%">
<tbody><tr class="description-tr">
<td class="description-td">Enzyme</td>
<td class="description-td">40° C.</td>
<td class="description-td">45° C.</td>
<td class="description-td">50° C.</td>
<td class="description-td">52.5° C.</td>
<td class="description-td">55° C.</td>
</tr>
<tr class="description-tr">
<td align="center" class="description-td" colspan="6" nameend="6" namest="1" rowsep="1"> </td>
</tr>
</tbody></table>
<table align="left" class="description-table" cols="6" colsep="0" rowsep="0" width="100%">
<tbody><tr class="description-tr">
<td class="description-td">His<sub>6 </sub>H-RT</td>
<td class="description-td">517</td>
<td class="description-td">523</td>
<td class="description-td">229</td>
<td class="description-td">40</td>
<td class="description-td">2</td>
</tr>
<tr class="description-tr">
<td class="description-td">His<sub>6 </sub>H-H204R RT</td>
<td class="description-td">521</td>
<td class="description-td">446</td>
<td class="description-td">146</td>
<td class="description-td">36</td>
<td class="description-td">17</td>
</tr>
<tr class="description-tr">
<td class="description-td">His<sub>6 </sub>H-T306K RT</td>
<td class="description-td">456</td>
<td class="description-td">512</td>
<td class="description-td">338</td>
<td class="description-td">122</td>
<td class="description-td">28</td>
</tr>
<tr class="description-tr">
<td class="description-td">His<sub>6 </sub>H-H204R T306K RT</td>
<td class="description-td">664</td>
<td class="description-td">528</td>
<td class="description-td">330</td>
<td class="description-td">202</td>
<td class="description-td">43</td>
</tr>
<tr class="description-tr">
<td class="description-td">His<sub>6 </sub>H-H204R T306K </td>
<td class="description-td">488</td>
<td class="description-td">556</td>
<td class="description-td">396</td>
<td class="description-td">209</td>
<td class="description-td">53</td>
</tr>
<tr class="description-tr">
<td class="description-td">F309N RT</td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">His<sub>6 </sub>H-H204R M289L </td>
<td class="description-td">540</td>
<td class="description-td">547</td>
<td class="description-td">539</td>
<td class="description-td">376</td>
<td class="description-td">120</td>
</tr>
<tr class="description-tr">
<td class="description-td">T306K F309N RT</td>
</tr>
<tr class="description-tr">
<td align="center" class="description-td" colspan="6" nameend="6" namest="1" rowsep="1"> </td>
</tr>
<tr class="description-tr">
<td align="left" class="description-td" colspan="6" id="FOO-00008" nameend="6" namest="1"> <sup>a</sup>cDNA synthesis reaction mixtures (as described above) contained 9.3 pmoles (2.6 μg) of CAT cRNA. The amounts of full-length products were established by cutting the region corresponding to the size of each full-length band from a dried alkaline 1.2% agarose gel and counting it in a scintillation counter. Reaction mixtures contained 2 pmoles of RT and were incubated at the temperatures indicated for 60 min. The results of a single experiment are shown. Similar results were obtained in one other experiment.</td>
</tr>
</tbody></table>
</patent-tables>
</tables>
</div>
<div class="description-paragraph" id="p-0365" num="0391">The amount of RT relative to template-primer was limiting in these reactions. In the presence of limiting RT, failure to achieve full-length cDNA synthesis as the temperature is increased is an indicator of thermal inactivation of the enzyme under cDNA synthesis conditions. With one exception at temperatures above 50° C., the amount of full-length CAT cDNA synthesized by a particular RT mutant (Table 13) correlated well with the thermal stability (half-life) of the enzyme at 50° C. (Table 9). The one exception is mutant H204R. It produced less full-length cDNA than His<sub>6 </sub>H− RT at 50 and 52.5° C., in spite of having a 5-fold greater half-life at 50° C. This is probably due to the weaker binding of H204R than H− RT to CAT cRNA● DNA, diminishing thermal protection by template-primer. All the RTs tested maintained activity at 45° C. relative to 40° C., but with the exception of the quadruple mutant, all RTs synthesized less full-length CAT cDNA at 50° C. and above. The His<sub>6 </sub>H− RT quadruple mutant was fully active at 50° C. and at 55° C. synthesized 60 times more full-length CAT cDNA than His<sub>6 </sub>H− RT. This was 22% of the amount synthesized by both enzymes at 40° C.</div>
<div class="description-paragraph" id="p-0366" num="0392">The methods of the present invention can be used to engineer a polypeptide to have a desired characteristic, for example, a desired level of thermostability. The nucleic acid sequence of the polypeptide may be sub-divided into segments and the segments individually mutagenized, for example, using PCR random mutagenesis. The individual segments may then be re-assembled and used to express a mutated polypeptide that is screened for the desired activity. The Examples above provide working embodiments of the invention in which the rather large RT gene (2154 bases) was randomly mutated in segments by PCR random mutagenesis (Leung, et al., (1989) <i>Technique </i>1, 11-15. Cadwell, et al., (1992) <i>PCR Methods and Applications </i>2, 28-33). It may be desirable to sub-divide the sequence of the polypeptide of interest into fragments that correspond to functional or structural domains of the polypeptide of interest if such domains are known. In the working embodiments above, the segments roughly corresponded to the coding sequences of the five separate structural subdomains of RT (Kohlstaedt, et al., (1992) <i>Science </i>256, 1783-1789. Jacobo-Molina, et al., (1993) <i>Proc. Natl. Acad. Sci. USA </i>90, 6320-6324). Segments one through four corresponded to the polymerase subdomain of fingers, palm, thumb, and connection, respectively, and segment five corresponded to the RNase H domain (<figref idrefs="DRAWINGS">FIG. 2A</figref>).</div>
<div class="description-paragraph" id="p-0367" num="0393">Using the materials and methods of the present invention, one skilled in the art can construct a modified RT enzyme having a desired level of thermostability. In the examples presented above, the operating temperature of H− M-MLV RT was increased to at least 10° C. above 45° C. This target temperature was selected as a compromise: high enough to help reduce RNA secondary structure and low enough to avoid RNA breakdown. Although, the rate of chemical RNA breakdown catalyzed by Mg<sup>++</sup> increases dramatically as the temperature is increased above 55 to 60° C. under cDNA synthesis reaction conditions, it may be desirable in certain instances (e.g., RNA with pronounced secondary structure) to conduct a reverse transcription reaction at a temperature higher than 55° C., for example, at about 60° C., or at about 65° C., or at about 70° C. Using the methods described above a suitable RT can be constructed using routine experimentation.</div>
<div class="description-paragraph" id="p-0368" num="0394">Reaction conditions for the mutagenesis reaction may be adjusted to introduce the desired number of mutations in each segment. In the present study, an upper limit cut-off of 1 to 2 mutations per segment was set as the target for mutation frequency to suppress accumulation of deleterious mutations (Lehmann, et al., (2001) <i>Current Opinion in Biotechnology </i>12, 371-375), and to minimize the amount of screening required to find active mutants. Those skilled in the art will appreciate that conditions may be adjusted in order to induce a greater mutation frequency, for example, by adjusting the concentration of Mn<sup>2+</sup>, pH, salt concentration etc. Mutation frequencies of &gt;5 mutations/segment in segment one, two, or three produced only about 5% active mutants with all the mutants having less than wild-type activity. At the mutation frequency used of 1 to 2 mutations per segment, approximately one-third of the mutants had little or no activity, one-third had less than 50% of His<sub>6 </sub>H− RT activity, and one-third had up to 100% of His<sub>6 </sub>H− RT activity.</div>
<div class="description-paragraph" id="p-0369" num="0395">Polypeptides produced from the mutated sequences are screened for a desired activity. The polypeptides may be screened from crude extracts, partially purified extracts or from purified polypeptide preparations. In the Examples above, crude extracts of mutants were screened in a 96-well plate format with template-primer specific for RT activity (Gerard, et al., (1974) <i>Biochem. </i>13, 1632-1641). The procedure used to produce <i>E. coli </i>cell crude extracts permeabilized rather the lysed the cells, releasing proteins and small RNAs to the outside of the cell matrix, while keeping DNA inside. This made representative sampling and assay with an exogenous template-primer of the cell suspension feasible. The signal-to-noise ratio of the assay was such that to differentiate starting from mutant cells at least a two-fold difference in activity was required. Those skilled in the art will appreciate that other methods of screening, for example, by first purifying the mutated polypeptides of interest (e.g., RTs) will permit detection of smaller changes in activity level.</div>
<div class="description-paragraph" id="p-0370" num="0396">Utilizing this approach, two mutations were found in segment three (thumb) and one in segment two (fingers). A rational sequence homology approach combined with site-directed mutagenesis was used to identify a fourth mutation located in the fingers subdomain. The small number of mutations found and the failure to identify mutations in segments one and four were probably due to limitations of both the random mutagenesis approach and the screen. The mutation frequency cut off of 1 to 2 mutations per segment imposed on the random mutagenesis limited the mutant space available in the mutant population. The inability of the assay to detect less than a two-fold difference in activity probably made many active and slightly more thermal stable mutants undetectable. Since the RNase H domain of wild-type M-MLV RT is already substantially more thermal stable than the polymerase domain (Verma, I. (1975) <i>J. Virol. </i>15, 843-854), it was unlikely that stabilizing mutations introduced in segment five would be picked up in a screen for more thermal stable polymerase activity.</div>
<div class="description-paragraph" id="p-0371" num="0397">In spite of the limited number of mutations found with the approach used, those that were identified had a significant positive impact on RT thermal stability. The thermal stability of His<sub>6 </sub>H− M-MLV H204R M289L T306K F309N RT was increased 30 fold at 50° C. The RT operating temperature was increased 7° C. to 55° C. Three of the four mutations alone increased the RT half-life 1.2 to 1.6-fold and the fourth mutation gave a 5-fold increase. A well established operating principle in protein stabilization is that the total stabilization obtained by multiple mutations is the sum of the effects of individual mutations, assuming each mutation exerts its effect independent of the other mutations (Wells, J. A. (1990) <i>Biochem. </i>29, 8509-8517). The thermal stabilizing effects of the four mutations appear to be additive, but an additional contribution to increased thermal stability due to cooperative interaction between mutations cannot be excluded.</div>
<div class="description-paragraph" id="p-0372" num="0398">Discussion of the mechanisms whereby each of these four mutations affect the thermal stability and template-primer binding affinity of M-MLV RT requires a 3-dimensional structural framework. In the case of the mutation at H204 in the M-MLV RT palm subdomain, this is easily accomplished because the structure of a fingers-palm fragment of M-MLV RT is known (Georgiadis, et al., (1995) <i>Structure </i>3, 879-892). For T306K and F309N located in the thumb, we must rely on 3-dimensional structural data taken from HIV RT (Kohlstaedt, et al., Jacobo-Molina, et al., Ding, et al., (1998) <i>J. Mol. Biol. </i>284, 1095-1111, and Huang, et al., (1998) <i>Science </i>282, 1669-1675). Fortunately in this region of the thumb subdomain of HIV RT (T253 to P272) and the corresponding region of M-MLV RT (T296 to P315), the amino acid sequence is reasonably well conserved. M-MLV RT can be predicted to have an α-helix in this region that resembles the structure of HIV RT α-helix H (Kohlstaedt, et al., Jacobo-Molina, et al., Kneller, et al., (1990) <i>J. Mol. Biol. </i>214, 171-182, and Rost, B. (1996) <i>Methods Enzymol. </i>266, 525-539.)</div>
<div class="description-paragraph" id="p-0373" num="0399">Together the four RT amino acid changes exerted their stabilizing effect without significantly altering the DNA synthesizing catalytic activity of the enzyme. However two mutations, H204R and F309N, did significantly reduce RT binding to template-primer (Table 12). M-MLV RT H204R is located in the middle of the long α-helix H (αH) on the backside of the palm subdomain, directly below β-sheets 10-11 (β10-β11) (<figref idrefs="DRAWINGS">FIG. 9</figref>, Georgiadis, et al., supra). Beta-sheets 10-11 contain the M-MLV RT polymerase catalytic site (YVDD) at residues 222 to 225 on the inside of the palm that contact the template-primer (Georgiadis, et al., supra). Changes in H204R could directly affect packing of amino acid residues near it in αH and β10-β11, thus influencing how β10-β11 interact with template-primer. Or its effect could be translated along the polypeptide backbone since the α-helix containing H204 and the β-sheet containing Y222 are separated by a single short turn (<figref idrefs="DRAWINGS">FIG. 9</figref>). As discussed above based upon an amino acid homology comparison of M-MLV and HIV RT and from crystal structures of HIV RT (Kohlstaedt, et al., supra, Jacobo-Molina, et al., supra, Ding, et al., supra, Huang, et al., supra, Kneller, et al., supra, Rost, Bebenek, et al., (1997) <i>Nature Structural Biology </i>4, 194-197, and Beard, et al., (1998) <i>J. Biol. Chem. </i>273, 30435-30442), F309 in M-MLV RT can be predicted to be part of a minor groove binding track in an α-helix of the RT thumb subdomain. In HIV RT this structure contributes significantly to RT binding to template-primer (Bebenek, et al., supra). The aromatic phenylalanine at position 309 in M-MLV RT corresponds to tryptophan 266 in HIV RT (Bebenek, et al., supra). Extensive amino acid substitution studies of HIV RT W266 show that any substitution at this position reduces the binding affinity of RT for template-primer (Beard, et al., supra). We postulate that alteration of M-MLV RT F309 to asparagine impacts the structure of a putative M-MLV RT minor groove binding track resulting in a reduction in binding affinity for template-primer. Interestingly T306K was able to compensate for the reduced binding produced by H204R or H204R and F309N together, restoring template-primer binding affinity to the original level (Table 12). Again based upon amino acid homology in this region, T306 in M-MLV RT corresponds to a lysine in position 263 in the α-helix of HIV RT that contains the minor groove binding track. The crystal structure of HIV RT complexed with double-stranded DNA shows that K263 contacts the n-2 phosphate of the DNA primer (Ding, et al., supra, Huang, et al., supra). Introduction into M-MLV RT of a positively charged lysine in place of phenylalanine at position 306 compensates for the reduced binding of H204R and F309N, perhaps by enhancing RT binding at the n-2 primer phosphate position.</div>
<div class="description-paragraph" id="p-0374" num="0400">H204R had the greatest individual thermal stabilizing effect on H− M-MLV RT. At position 204 histidine has been replaced with a more highly charged arginine in α-helix H at the underside of the palm subdomain (Georgiadis, et al., supra). Examination of the 3-dimensional structural model of the fingers-palm fragment of M-MLV RT (Georgiadis, et al., supra) reveals this substitution places arginine at a proper distance to form a salt bridge with E201 in α-helix H or a hydrogen bond with T128 in the loop between β-sheet 6 and α-helix E (<figref idrefs="DRAWINGS">FIG. 9</figref>). Either event would help stabilize the RT molecule (Kumar, et al., (2001) <i>Cell. Mol. Life. Sci. </i>58, 1216-1233).</div>
<div class="description-paragraph" id="p-0375" num="0401">The mutation at position 306 from threonine to lysine probably contributes to the thermal stability of M-MLV RT by making an electrostatic contribution to stabilizing the polypeptide backbone. Saturation of this site with all possible amino acids showed that only substitution of arginine or lysine increased M-MLV RT thermal stability. Both of these amino acids could allow for hydrogen bonding to main chain carbonyl groups and amino acid side chains in a loop adjacent to the putative α-helix containing residue 306.</div>
<div class="description-paragraph" id="p-0376" num="0402">Predictions about the mechanism whereby F309N increases M-MLV RT thermal stability are complicated by its proposed role in template-primer binding (see discussion above). M-MLV RT position 309 was also examined by saturation with all possible amino acid changes. Only serine and asparagine increased thermal stability. Substitution of charged amino acids for the exposed hydrophobic phenylalanine, which in other protein families correlates well with amino acid changes observed in transitioning from mesophilic to thermophilic family representatives (Kumar, et al., supra, Cambillau, et al., (2000) <i>J. Biol. Chem. </i>275, 32383-32386), reduced or eliminated polymerase activity. Correlation of increased thermal stability with substitution of polar for exposed hydrophobic amino acids is observed much less frequently in protein families (Kumar, et al., supra, Cambillau, et al., supra), so that the structural basis for the stabilizing effect of F→N at position 309 is not clear.</div>
<div class="description-paragraph" id="p-0377" num="0403">In the absence of both 3-dimensional structural information and reasonable sequence homology between HIV RT and M-MLV RT in the region of M-MLV RT M289, it is difficult to predict the structural basis for the contribution of M289L to M-MLV RT thermal stability. As a general tendency, thermally stable proteins have more and larger hydrophobic amino acids than their mesophilic counterparts that are better able to exclude water from the protein core (Cambillau, et al., supra). It is possible that M→L at position 289 increases M-MLV RT thermal stability through allowing better hydrophobic packing by replacing an unbranched methionine with a β-branched leucine (Cambillau, et al., supra).</div>
<div class="description-paragraph" id="p-0378" num="0404">Consistent with the results of other studies focused on identifying mutations that increase protein thermal stability (Arnold, et al., (2001) <i>Trends in Biochemical Sciences </i>26, 100-106), the mutations identified in this study probably reside on the RT surface and influence thermal stability without decreasing catalytic activity. In the case of two of the stabilizing amino acids, T306 and F309, they probably reside on a protein surface that contacts template-primer and they bind in such a way that their effects on binding are off setting.</div>
<div class="description-paragraph" id="p-0379" num="0405">The present invention provides materials and methods for engineering a polypeptide to contain a desired characteristic. As an example, the M-MLV RT gene was mutated to increase the thermostability of the enzyme. The goal of this research was to increase the temperature at which M-MLV RT efficiently catalyzes cDNA synthesis from 48° C. to about 55° C. to about 60° C. by increasing the intrinsic thermal stability of RT. The ability to use retroviral RT at 55 to 60° C. increases both the efficiency and specificity of priming of cDNA synthesis. With the mutagenesis and screening approaches used, mutations were identified that increased the M-MLV RT operating temperature limit to at least about 55° C. These mutations created a dramatic increase in intrinsic thermal stability measured as a half-life increase at 50° C. (30-fold), that was maintained partially at 55° C. (5-fold increase in half-life). There are alternative approaches that could potentially build on the four mutants already identified. For example, the target mutation frequency could be set much higher (5-10 mutations/segment) during random mutagenesis of several larger segments of the H− M-MLV RT quadruple mutant polymerase domain, giving a much smaller proportion of active mutants but a much broader mutant spectrum. Several libraries generated in this fashion could then be shuffled by recombination (Stemmer, W. P. C. (1994) <i>Nature </i>370, 389-391), which would tend to eliminate deleterious mutations (Lehmann, et al., supra). When combined with a 96-well DNA polymerase activity screen made more sensitive and a sampling procedure used to assay multiple mutants per well to increase the number of mutations screened, such an approach should yield additional mutations that further increase the thermal stability of M-MLV RT.</div>
<heading id="h-0040">Example 7</heading>
<heading id="h-0041">Engineering of S<smallcaps>UPER</smallcaps>S<smallcaps>CRIPT</smallcaps>™ III RT, Thermal Stability, cDNA Synthesis</heading>
<div class="description-paragraph" id="p-0380" num="0406">The gene for Moloney murine leukemia virus (M-MLV) RNase H-minus (H−) reverse trenscriptase (RT), also known as S<smallcaps>UPER</smallcaps>S<smallcaps>CRIPT</smallcaps>™ II was randomly mutagenized and mutants were screened for increased thermal stability. Four mutations were identified that together increased the half-life of M-MLV H− RT at 50° C. 35-fold and raised the RT cDNA synthesis operating temperature from 42° C. to 55° C. This increase in thermal stability was achieved by increasing the intrinsic thermal stability of M-MLV H− RT and without diminishing the DNA polymerase catalytic activity of the enzyme.</div>
<div class="description-paragraph" id="p-0381" num="0407">Reverse transcriptase (RT, e.g., retroviral RT) is an essential tool for the synthesis and cloning of cDNA. Forms of retroviral RT used widely to synthesize cDNA are M-MLV RT and AMV RT. In addition to polymerase activity, retroviral RT possesses RNase H activity that degrades the RNA in an RNA-DNA hybrid (Moelling, et al., (1971) <i>Nature New Biology </i>234, 240-244). The presence of this degradative activity is responsible in part for the limitation on efficient synthesis of long cDNA (Krug and Berger (1987) <i>Methods in Enzymol. </i>152, 316-325; Berger, et al., (1983) <i>Biochem. </i>22, 2365-2372). The RNase H domain of RT can be mutated to reduce or eliminate RNase H activity while maintaining mRNA-directed DNA polymerase activity (Kotewicz, et al., (1988) <i>Nuc. Acids Res. </i>16, 265-277; DeStefano, et al., (1994) <i>Biochim. Biophys. Acta </i>1219, 380-388), improving the efficiency of cDNA synthesis (Kotewicz, et al., supra). This has been done in S<smallcaps>UPER</smallcaps>S<smallcaps>CRIPT</smallcaps>™ II and ThermoScript.</div>
<div class="description-paragraph" id="p-0382" num="0408">A second significant drawback to copying mRNA is the tendency of RT to pause during cDNA synthesis resulting in the generation of truncated products (Harrison, et al., (1998) <i>Nuc. Acids Res. </i>26, 3433-3442; DeStefano, et al., (1991) <i>J. Biol. Chem. </i>266, 7423-7431). This pausing is due in part to the secondary structure of RNA (Harrison, et al., supra, Wu, et al., (1996) <i>J. Virol. </i>70, 7132-7142). Performing cDNA synthesis at reaction temperatures that melt the secondary structure of mRNA helps to alleviate this problem (Myers and Gelfand, (1991) <i>Biochem. </i>30, 7661-7666). In addition, the oligo(dT)n primer often used to initiate cDNA synthesis tends to prime at internal stretches of A residues in mRNA at lower temperatures, resulting in the synthesis of 3′-end truncated cDNA products. M-MLV RT does not efficiently synthesize cDNA from mRNA above 43° C. (Tosh, et al., (1997) <i>Acta Virol. </i>41, 153-155).</div>
<div class="description-paragraph" id="p-0383" num="0409">In an effort to raise the temperature at which S<smallcaps>UPER</smallcaps>S<smallcaps>CRIPT</smallcaps>™ II RT can be used to synthesize cDNA, we have randomly mutagenized the S<smallcaps>UPER</smallcaps>S<smallcaps>CRIPT</smallcaps>™ II RT gene and screened for thermal stable mutants. Several thermal stable mutants of S<smallcaps>UPER</smallcaps>S<smallcaps>CRIPT</smallcaps>™ II RT were identified and purified enzymes were characterized. We show that when the mutations are present together they increase RT thermal activity by increasing its intrinsic thermal stability without altering catalytic activity.</div>
<div class="description-paragraph" id="p-0384" num="0410">S<smallcaps>UPER</smallcaps>S<smallcaps>CRIPT</smallcaps>™ III RT Purification: The S<smallcaps>UPER</smallcaps>S<smallcaps>CRIPT</smallcaps>™ III RT gene was derived from S<smallcaps>UPER</smallcaps>S<smallcaps>CRIPT</smallcaps>™ II RT. Four mutations, in addition to the 3 RNase H mutations present in S<smallcaps>UPER</smallcaps>S<smallcaps>CRIPT</smallcaps>™ II, were included along with modifications to the N-terminus to increase the thermostability. The gene was cloned into plasmid pBAD (Invitrogen Corporation, Carlsbad, Calif.) under control of an araD promoter. The purified protein has an apparent molecular weight of 78 kDa. The plasmid was transformed into <i>E. coli </i>DH10B (Invitrogen Corporation, Carlsbad, Calif.) cells and a seed stock was grown up to an O.D. of 1.0 in EG (2% Tryptone, 1% yeast extract, 0.5% glycerol, 10 mM NaCl, and 1 mM KCl) and frozen in 60% S.O.B./40% glycerol at −80° C.</div>
<div class="description-paragraph" id="p-0385" num="0411">For purification, a stab from the frozen seed was used to inoculate a starter culture of 500 ml. The media used for the R&amp;D prep was either EG or CircleGrow (Bio101), supplemented with 100 μg/ml Ampicillin. The starter culture was incubated overnight at 37° C. and used to inoculate 10 liters EG+Amp media. The culture was incubated at 37° C. to an OD<sub>600 </sub>of 1.0 (3-4 hours), induced with 0.2% arabinose, and incubation continued another 3 hours. Cells were pelleted by centrifugation at 5,000 g for 20 minutes at 4° C. The supernatant was removed and cells were either frozen at −80° C. or processed.</div>
<div class="description-paragraph" id="p-0386" num="0412">Purification of S<smallcaps>UPER</smallcaps>S<smallcaps>CRIPT</smallcaps>™ III was done the same as S<smallcaps>UPER</smallcaps>S<smallcaps>CRIPT</smallcaps>™ II and is outlined as follows, with the exception of using an AF-Heparin 650M resin (TosoHaas) in place of Heparin Agarose as the third column. All buffers must be autoclaved and columns thoroughly cleaned to prevent RNase contamination. The procedure is done at 4° C. unless otherwise indicated.</div>
<div class="description-paragraph" id="p-0387" num="0413">Purification Method A. Thaw 50 grams of cells in 100 ml Buffer A (2 ml per g of cell pellet: 20 mM Tris-HCL pH 7.5, 25 mM NaCl, 1 mM EDTA, 1 mM DTT, 1.3 mM PMSF), stirring at 4° C. until homogenous. Add another 0.78 ml 0.1 M PMSF and mix well (5 minutes). Strain cells through 6 layers of cheesecloth. Filter the sample through gaulin pre-equilibrated with 1 volume Buffer A (avoid bubbles and particulates in gaulin) at 4-6° C. with pressure at 8000 psi. Repeat and check sample for &gt;80% lysis. Determine volume of cells and add 0.053 V of 5M NaCl to lysate. Stir at 4° C. till mixed. Add 0.111 V of 5% polymin P (pH 7.9) over 30 minutes while mixing and then mix for another 30 minutes. Centrifuge at 18,100 g for 60 min at 4° C., keep the supernatant. Add 226 g/L (NH<sub>4</sub>)<sub>2</sub>SO<sub>4 </sub>to supernatant over 30 minutes while stirring and mix for another 30 minutes. Centrifuge at 18,100 g for 60 min at 4° C., keep the supernatant. Resuspend the pellet in Buffer B (20 mM Tris-HCL pH 7.5, 100 mM NaCl, 1 mM EDTA, 1 mM DTT, 0.01% NOG, 5% glycerol) at 0.52 ml/g original pellet weight (26 ml) while stirring for 1 hour.</div>
<div class="description-paragraph" id="p-0388" num="0414">A G-25 F column (AP-Biotech, 270 ml resin volume) was equilibrated with Buffer B and then the sample was loaded. The column was washed and the sample eluted with 2 column volumes of Buffer B at a flow rate of 3.3 ml/min and fraction volume of 6 ml/tube. Fractions were pooled based on UV absorbance and a clear amber color. If precipitation formed, the pool was centrifuged at 18,100 g for 30 min at 4° C.</div>
<div class="description-paragraph" id="p-0389" num="0415">The fraction pool was loaded onto a P-11 column (Whatman: 2× cycled, 25 ml resin volume) pre-equilibrated with Buffer B at a flow rate of 1.0 ml/min. The column was washed with 10 column volumes of Buffer C (20 mM Tris-HCL pH 7.5, 100 mM NaCl, 1 mM EDTA, 1 mM DTT, 0.01% NP-40, 5% glycerol) and eluted with 12 column volumes of a 100 to 500 mM NaCl gradient (Buffer C and 0 to 100% of Buffer D: 20 mM Tris-HCL pH 7.5, 500 mM NaCl, 1 mM EDTA, 1 mM DTT, 0.01% NP-40, 5% glycerol) at a flow rate of 1.6 ml/min and fraction volume of 4 ml. Fractions were pooled by either OD <sub>280</sub>&gt;50% peak height or &gt;50% peak RT activity. The fraction pool was then diluted with one volume of Buffer E (20 mM Tris-HCL pH 7.5, 1 mM EDTA, 1 mM DTT, 0.01% NP-40, 5% glycerol).</div>
<div class="description-paragraph" id="p-0390" num="0416">The diluted pool was then loaded onto an AF-Heparin 650M column (TosoHaas, 12.5 ml resin volume) pre-equilibrated with Buffer F (20 mM Tris-HCL pH 7.5, 160 mM NaCl, 1 mM EDTA, 1 mM DTT, 0.01% NP-40, 5% glycerol) at a flow rate of 0.7 ml/min.</div>
<div class="description-paragraph" id="p-0391" num="0417">The column was washed with 10 column volumes of Buffer F (20 mM Tris-HCL pH 7.5, 160 mM NaCl, 1 mM EDTA, 1 mM DTT, 0.01% NP-40, 5% glycerol) and eluted with 10 column volumes of a 160 to 700 mM NaCl gradient (Buffer F and 0 to 100% of Buffer G: 20 mM Tris-HCL pH 7.5, 700 mM NaCl, 1 mM EDTA, 1 mM DTT, 0.01% NP-40, 5% glycerol) at a flow rate of 0.5 ml/min and fraction volume of 2 ml. Fractions were pooled by either OD<sub>280</sub>&gt;50% peak height or &gt;50% peak RT activity. The fraction pool was then diluted with three volumes of Buffer H (20 mM Tris-HCL pH 7.5, 100 mM NaCl 1 mM EDTA, 1 mM DTT, 0.03% NP-40, 5% glycerol).</div>
<div class="description-paragraph" id="p-0392" num="0418">The diluted pool was then loaded onto an SP Sepharose HP column (Ap Biotech, 12.5 ml resin volume) pre-equilibrated with Buffer I (20 mM Tris-HCl pH 7.5, 100 mM NaCl, 1 mM EDTA, 1 mM DTT, 0.01% NP-40, 5% glycerol) at a flow rate of 1.2 ml/min. The column was washed with 10 column volumes of Buffer I (20 mM Tris-HCL pH 7.5, 100 mM NaCl, 1 mM EDTA, 1 mM DTT, 0.01% NP-40, 5% glycerol) and eluted with 10 column volumes of a 100 to 280 mM NaCl gradient (Buffer 1 and 0 to 40% of Buffer G) at a flow rate of 0.6 ml/min and fraction volume of 2 ml. Fractions were pooled by either OD <sub>280</sub>&gt;50% peak height or &gt;50% peak RT activity.</div>
<div class="description-paragraph" id="p-0393" num="0419">The RT containing fractions from the Sepharose column were dialyzed against 20 volumes Buffer J (20 mM Tris-HCL pH 7.5, 100 mM NaCl, 0.1 mM EDTA, 1 mM DTT, 0.01% NP-40, 50% glycerol) overnight at 4° C.</div>
<div class="description-paragraph" id="p-0394" num="0420">Purification Method B. One hundred grams of cells is suspended in 80 ml of permeabilization buffer (500 mM Bis-Tris, pH 7.0 at 4° C./50 mM EDTA/5 mM DTT). (Note: The DTT can be added as a separate component and the volume of water added adjusted accordingly) 79.6 ml of H<sub>2</sub>O is added and the cell suspension is stirred with an overhead stirrer at ˜700 rpm with a Heidolph mixer or at 6+ with a StirPak mixer using the three blade mixer shaft. The suspension is checked to ensure that it is homogeneous by stopping the mixer and pouring the suspension into a glass beaker slowly while looking for clumps. Once thoroughly mixed, 80 ml 25% Triton X-100/10% deoxycholic acid (at ambient temperature) is added while mixing vigorously (as above). 400 ul of 100 mM PMSF is added and the suspension is allowed to stand with mixing for 30 min at 4° C. 60 ml of 2 M (NH<sub>4</sub>)<sub>2</sub>SO<sub>4 </sub>is added and mixing is continued for an additional 15 min at 4° C. Samples can now be taken or proceed to the filtration stage.</div>
<div class="description-paragraph" id="h-0042" num="0000">Volumes:</div>
<div class="description-paragraph" id="p-0395" num="0421">
<tables id="TABLE-US-00015" num="00015">
<patent-tables colsep="0" frame="none" rowsep="0">
<table align="left" class="description-table" cols="3" colsep="0" rowsep="0" width="100%">
<thead>
<tr class="description-tr">
<td class="description-td"> </td>
<td align="center" class="description-td" colspan="3" nameend="2" namest="offset" rowsep="1"> </td>
</tr>
</thead>
<tbody><tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">Cells</td>
<td class="description-td">100 ml </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">Buffer</td>
<td class="description-td">80 ml</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">H<sub>2</sub>O</td>
<td class="description-td">79.6 ml  </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">Detergent</td>
<td class="description-td">80 ml</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">PMSF</td>
<td class="description-td">0.400 ml  </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">(NH<sub>4</sub>)<sub>2</sub>SO<sub>4</sub> </td>
<td class="description-td">60 ml</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">400 ml </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td align="center" class="description-td" colspan="3" nameend="2" namest="offset" rowsep="1"> </td>
</tr>
</tbody></table>
</patent-tables>
</tables>
</div>
<div class="description-paragraph" id="p-0396" num="0422">Filtration. The cell slurry is poured into a circulation vessel and a flow rate of 1.5 L/min/ft2 through a 0.2 um Spectrum mixed ester cellulose hollow fiber filter with a 1.0 mm lumen is established. Extraction is performed with seven volumes of 100 mM TRIS, pH 8.0/300 mM ammonium sulfate/10 mM EDTA/1 mM DTT/10% glycerol (v/v) while maintaining 4-6° C. and a TMP of 5-10 psi.</div>
<div class="description-paragraph" id="p-0397" num="0423">Once enough filtrate has been collected, the second stage filtration using an AG/Tech 30,000 MWCO hollow fiber with a maximum inlet pressure of 50 psi. (all operations chilled to 4-6° C.) is initiated. Once all seven volumes of extraction have been collected, the solution is concentrate to 600 ml and diafiltered against five volumes of 100 mM Tris, pH 8.0/100 mM NaCl/10 mM EDTA/1 mM DTT.</div>
<div class="description-paragraph" id="p-0398" num="0424">The diafiltered retentate is collected and 50 ml previously equilibrated DEAE cellulose is added. The suspension is mixed gently for 15 min at 4° C. The resulting solution is clarified by filtration through a CUNO 30 SP depth filter.</div>
<div class="description-paragraph" id="p-0399" num="0425">Chromatography. The above solution is applied to a previously equilibrated 120 ml column of Macroprep High S (20 mM Tris, pH 8.0/150 mM NaCl/10% glycerol/1 mM EDTA/1 mM DTT/0.01% Triton X-100). The column is washed with 10 column volumes of equilibration buffer and eluted with a linear gradient of equilibration buffer to the same buffer at 800 mM NaCl. 50 fractions are collected and the major uv peak is pooled.</div>
<div class="description-paragraph" id="p-0400" num="0426">The pool is applied to a previously equilibrated 120 ml column of Ceramic hydroxy apatite (20 mM Kpi, pH 7.0/100 mM KCl/10% glycerol/1 mM DTT/0.01% Triton X-100) at 20 cm/h. Elution is with this same buffer and 10 ml fractions are collected. The uv peak is pooled and an equal volume of solubilization buffer (100 mM Tris, pH7.5/300 mM NaCl/0.2 mM EDTA/30% glycerol/1 mM DTT) is added.</div>
<div class="description-paragraph" id="p-0401" num="0427">The pool is applied to a previously equilibrated 16 ml column of EM COO— (20 mM Tris, pH 7.5/100 mM NaCl/0.1 mM EDTA/20% glycerol/1 mM DTT/0.01% Triton X-100) at 20 cm/h. The column is washed with two column volecules of equilibration buffer. Elution is with a linear gradient of equilibration buffer to the same buffer with 200 mM NaCl. The uv peak is pooled and dialyzed against 20 volumes of 20 mM Tris, pH 7.5/100 mM NaCl/0.1 mM/1 mM DTT/0.01% Triton X-100/50% glycerol.</div>
<div class="description-paragraph" id="p-0402" num="0428">DNA Polymerase Assays: RT DNA polymerase unit activity was assayed with (rA)<sub>630</sub>● p(dT)<sub>12-18</sub>. One unit of DNA polymerase activity is the amount of RT that incorporates one nmole of deoxynucleoside triphosphate into acid insoluble product at 37° C. in 10 min. Specific Activity is determined by dividing the Units/μl by the protein concentration to get units/mg.</div>
<div class="description-paragraph" id="p-0403" num="0429">cDNA synthesis from CAT cRNA was carried out in reaction mixtures (20 μl) containing 50 mM Tris-HCl (pH 8.4), 75 mM KCl, 3 mM MgCl<sub>2</sub>, 10 mM dithiotreitol (DTT), 500 μM each of dATP, dTTP, dGTP, and [α-<sup>32</sup>P]dCTP (300 cpm/pmole), 1,750 units/ml RNase Inhibitor, 130 μg/ml (465 nM) CAT cRNA, 20 μg/ml (2,300 nM) p(dT)<sub>25-30</sub>, and 3,250 units/ml (100 nM) RT. Incubation was at various temperatures for 60 min individual tubes. An aliquot of the reaction mixture was precipitated with TCA to determine yield of cDNA synthesized, and the remaining cDNA product was size fractionated on an alkaline 1.2% agarose gel. To establish mono- and divalent metal reaction optima, initial reaction rates were determined under conditions of limiting RT concentration during 10 min incubation at 37° C. or 50° C. Reaction mixtures (20 μl) contained 50 mM Tris-HCl (pH 8.4), 10 mM DTT, 500 μM each of dTTP, dATP, dCTP, and [<sup>3</sup>H]dGTP (100 cpm/pmole), 10 pmoles (2.8 μg) CAT cRNA, 50 pmoles DNA 24-mer, 0.5 pmoles RT, and KCl and MgCl<sub>2</sub>, varied in concentration one at a time.</div>
<div class="description-paragraph" id="p-0404" num="0430">Half Life Determination: Mixtures (20 μl) were incubated for various times in 0.5-ml tubes in a thermocycler at 50° C. or 55° C. and contained 50 mM Tris-HCl (pH 8.4), 75 mM KCl, 3 mM MgCl<sub>2</sub>, 10 mM DTT, 0.1% (v/v) Triton X-100, and 3-7 μg/ml purified RT. Incubation was stopped by placing the tube in ice. An aliquot (5 μl) was assayed for residual activity with (rA)<sub>630</sub>● p(dT)<sub>12-18</sub>.</div>
<div class="description-paragraph" id="p-0405" num="0431">Measurement of K<sub>D </sub>by Filter Binding: A nitrocellulose filter-binding assay was used to determine the nucleic acid binding constants (K<sub>D</sub>) of RTs for CAT cRNA●(dT)<sub>20</sub>. Dephosphorylated CAT cRNA was labeled at the 5′ end with [γ-<sup>32</sup>P]ATP and T4 polynucleotide kinase (Boehringer). Oligo(dT)<sub>20 </sub>was annealed to the poly(A)-tailed CAT cRNA in a buffer containing 10 mM Tris-HCl, pH 7.5, and 80 mM KCl at 65° C. for 5 min followed by chilling on ice. Reaction mixtures (100 μl) containing binding buffer (50 mM Tris-HCl, pH 8.4, 75 mM KCl, 3 mM MgCl<sub>2</sub>, and 10 mM DTT), 0.05 nM <sup>32</sup>P-labeled CAT cRNA, 1 nM (dT)<sub>20</sub>, and 1 to 50 nM RT were incubated at 23° C. for 5 min. After incubation, the mixture was filtered through a nitrocellulose filter (Millipore, HA 0.45 mm) soaked in binding buffer, which was then washed with binding buffer. The K<sub>D </sub>is equal to that enzyme concentration at which 50% of the labeled CAT cRNA is bound. For this method of analysis to be valid, the CAT cRNA concentration in the reaction must be substantially below K<sub>D</sub>, so that the total enzyme concentration approximates the concentration of free unbound enzyme.</div>
<div class="description-paragraph" id="p-0406" num="0432">LacZα Fidelity measurements: Fidelity measurement were conducted using a modified lacZα gapped procedure from Boyer, J. C. et al., (<i>Methods in Enzymology</i>, Vol 275, p 523-537). Briefly, a modified pUC19 plasmid containing the promoter for T7 RNA polymerase between nucleotides −112 and −113 (where position +1 is the first transcribed nucleotide of the lacZα gene) was used. The plasmid was linearized with FspI, and transcription by T7 RNA polymerase produces a 344 nucleotide transcript. Using this RNA as template, cDNA synthesis under desired conditions (200 units S<smallcaps>UPER</smallcaps>S<smallcaps>CRIPT</smallcaps>™ III RT or 15 units AMV H+ RT in their respective buffers) was initiated from a 15-mer DNA primer. After heat denaturation, the RNA was digested and an excess of cDNA was used for hybridization to a circular DNA substrate containing a complementary single-stranded gap. This substrate was made by cutting M13mp19 RF with PvuI and PvuII in the appropriate buffer and isolating the large linear fragment. This fragment is then denatured and reannealed to a 2-fold excess of single-strand M13mp19. The resulting DNA product is transfected into competent DH12S cells, plated onto LB indicator plates containing X-gal and IPTG and plaques are scored for color. Mutation frequency was determined by dividing the number of mutant plaques (light blue or white) by the total. The enzyme's mutation frequency is corrected by subtracting the mutation frequency of the starting DNA. The error rate is determined by dividing the mutation frequency by the number of known detectable sites (116) and then dividing by the probability of expressing the minus strand (50%).</div>
<div class="description-paragraph" id="p-0407" num="0433">rpsL Fidelity Assay: Fidelity assay was performed based on the streptomycin resistance that rpsL mutatants exhibit (Fujii, et al., (1999) <i>J Mol Biol. </i>289(4):835-50). Briefly, pMOL 21 plasmid DNA (4 kb), containing the ampicillin (Ap<sup>r</sup>) and (rpsL) genes, was modified with the insertion of a T7 RNA promoter sequence downstream of the rpsL gene. The plasmid was linearized with ScaI and this template was used for in vitro transcription of a 1.4 kb RNA containing the rpsL gene. A first strand cDNA reaction using the desired RT and reaction conditions was carried out, followed by second strand synthesis using pol I, RNase H and ligase. This double-strand DNA was then cut with HindIII and EcoRI and cloned back into the original pMOL 21 plasmid and transformed into MF101 competent cells. Cells were plated on ampicillin plates to determine the total number of transformed cells. Cells were plated on ampicillin and streptomycin plates to determine the total number of rpsL mutants. Mutation frequency was determined by dividing the total number of mutations by the total number of transformed cells. The error rate was determined by dividing the mutation frequency by 130 (the number of amino acids that cause phenotypic changes for rpsL) and the template doubling. In addition rpsL was further modified by altering the sequence prone to mutations to remove template bias.</div>
<div class="description-paragraph" id="p-0408" num="0434">TDT activity: The template-primer was prepared by annealing a 47-mer template (either RNA or DNA):</div>
<div class="description-paragraph" id="p-0409" num="0435">5′-GAGTTACAGTGTTTTTGTTCCAGTCTGTAGCAGTGTGTGAATGG</div>
<div class="description-paragraph" id="p-0410" num="0436">AAG-3′ (SEQ ID NO:6) to a 18-mer DNA primer (5′-GAACAAAAACACTGTAACTC-3′ (SEQ ID NO:10)) [<sup>32</sup>P]-labeled at the 5′-end with T4 polynucleotide kinase (template:primer, 3:1). Assay mixture (10 μl) contained 5 nM template-primer, 200 units each of either S<smallcaps>UPER</smallcaps>S<smallcaps>CRIPT</smallcaps>™ II or S<smallcaps>UPER</smallcaps>S<smallcaps>CRIPT</smallcaps>™ III, all 4 dNTPs, none or dCTP, dATP, dTTP, dGTP (250 uM each) as shown in the figure legends, 50 mM Tris-HCl (pH 8.3), 75 mM KCl, 3 mM MgCl<sub>2</sub>, 10 mM DTT. Reactions were incubated at various temperatures for 10 or 60 min and terminated by the addition of 5 μl of 40 mM EDTA, 99% formamide. Reaction products were denatured by incubating at 95° C. for 5 min and analyzed by electrophoresis on urea 6% polyacrylamide gels.</div>
<div class="description-paragraph" id="p-0411" num="0437">Half Life Determination: Mixtures (20 μl) were incubated for various times in 0.5-ml tubes in a thermocycler at 50° C., 55° C., or 60° C. and contained 1× First Strand Buffer (50 mM Tris-HCl (pH 8.3), 75 mM KCl, 3 mM MgCl<sub>2</sub>), 10 mM DTT, 3 mM MgCl<sub>2 </sub>(total to 6 mM), 0.5 mM dTTP, 0.05% (v/v) NP-40, [methyl-3H]dTTP (20 μCi/ml), and 5 μl of RT. Incubation was stopped by placing the tube in ice. An aliquot (5 μl) was assayed for residual Unit activity with 5 mM poly(A)/3 mM oligo(dT)<sub>25</sub>. One unit of RT DNA polymerase activity is the amount of RT that incorporates 1 nmol of deoxynucleoside triphosphate into acid-insoluble product at 37° C. in 10 min.</div>
<div class="description-paragraph" id="p-0412" num="0438">Full-length cDNA profiling: cDNA synthesis from cRNA by S<smallcaps>UPER</smallcaps>S<smallcaps>CRIPT</smallcaps>™ III was carried out in reaction mixtures (20 μl) containing 1× First Strand Buffer (50 mM Tris-HCl (pH 8.3), 75 mM KCl, 3 mM MgCl<sub>2</sub>), 10 mM dithiotreitol (DTT), 500 μM each of dATP, dTTP, dGTP, and dCTP, [α-<sup>32</sup>P]-dCTP (300 cpm/pmole), 2 units/μl RNaseOUT, different amounts of RNA, and 0.5 μg oligo(dT)<sub>25</sub>. The mix is prewarmed at various temperatures for 2 minutes. After warming, 200 or 400 units of S<smallcaps>UPER</smallcaps>S<smallcaps>CRIPT</smallcaps>™ III RT are added. The mixtures were incubated at the same temperature for another 50 min in individual tubes. The reactions were stopped by adding 10 μl of 0.5 M EDTA. An aliquot (5 μl) of the reaction mixture was precipitated with TCA to determine yield of cDNA synthesized, and the remaining cDNA product was size fractionated on an alkaline 1.2% agarose gel (McDonell, et al., <i>J. Mol. Biol. </i>110, 199-146, 1977). For the RTs from other vendors, each was assayed following the manufacturer's suggestion.</div>
<div class="description-paragraph" id="h-0043" num="0000">Results and Discussion</div>
<div class="description-paragraph" id="p-0413" num="0439">Designation of polypeptides. Unless otherwise indicated, the following designations will be used to describe various polypeptides of the invention. S<smallcaps>UPER</smallcaps>S<smallcaps>CRIPT</smallcaps>™ II is M-MLV reverse transcriptase with a methionine added as a start codon (as discussed earlier, wild type M-MLV RT is a cleavage product that does not contain a methionine) and having three point mutations that reduce RNase H activity. EFN indicates a polypeptide having the S<smallcaps>UPER</smallcaps>S<smallcaps>CRIPT</smallcaps>™ II sequence with an N-terminal six histidine tag sequence MGGSHHHHHHGMASMTGGQQMGRDLYDDDDKH (amino acids 1-32 of SEQ ID NO:2 and Table 3) and containing the EFN mutations, which are H204R, T306K, and F309N of the S<smallcaps>UPER</smallcaps>S<smallcaps>CRIPT</smallcaps>™ II sequence. Tagged, no His EFN has the tag sequence MASGTGGQQMGRDLYDDDDKH (SEQ ID NO:3) and the EFN mutations, no tag EFN contains the EFN mutations. S<smallcaps>UPER</smallcaps>S<smallcaps>CRIPT</smallcaps>™ III, which may be designated LEFN, has the S<smallcaps>UPER</smallcaps>S<smallcaps>CRIPT</smallcaps>™ II sequence and an N-terminal tag sequence MASGTGGQQMGRDLYDDDDK (SEQ ID NO:11) and the LEFN mutations, which are H204R, M289L, T306K, and F309N of the S<smallcaps>UPER</smallcaps>S<smallcaps>CRIPT</smallcaps>™ II sequence. His tagged LEFN indicates a polypeptide having an N-terminal six histidine tag sequence MGGSHHHHHHGMASMTGGQQMGRDLYDDDDKH (amino acids 1-32 of SEQ ID NO:2 and Table 3) and containing the LEFN mutations.</div>
<div class="description-paragraph" id="p-0414" num="0440">Purification: The protein expresses well and appears to not be toxic. Varying levels of arabinose induction were used with 0.2% being optimal but not very different from 2.0%. Induction times longer than 3 hours can be done, however there is a 70 kDa product that is generated over longer incubation periods. It appears to be a proteolysis product removing the C-terminus as purified N-terminal Histidine-tagged clones also had this protein co-purify.</div>
<div class="description-paragraph" id="p-0415" num="0441">The purification protocol is essentially the same as was used for S<smallcaps>UPER</smallcaps>S<smallcaps>CRIPT</smallcaps>™ II and the S<smallcaps>UPER</smallcaps>S<smallcaps>CRIPT</smallcaps>™ III enzyme behaves almost the same throughout the purification. S<smallcaps>UPER</smallcaps>S<smallcaps>CRIPT</smallcaps>™ III may be prepared at concentrations of 2,000-4,000 units/μl after dialysis with no precipitation problems.</div>
<div class="description-paragraph" id="p-0416" num="0442">Fidelity: Initial results have shown no difference between S<smallcaps>UPER</smallcaps>S<smallcaps>CRIPT</smallcaps>™ II and S<smallcaps>UPER</smallcaps>S<smallcaps>CRIPT</smallcaps>™ III using either the rpsL or lacZα fidelity assay. RTs have sequence dependant mutation hotspots in homopolymeric runs of nucleotides, primarily runs of A or T. These most common mutations at these runs are one nucleotide insertion or deletion which is caused by the template or primer breathing and re-annealing with an overlap. The resulting frameshift mutation usually results in premature termination of the gene product. The modified EFN polypeptide, which contains the H204R, T306K, and F309N mutations, had improved frameshift error rates compared to S<smallcaps>UPER</smallcaps>S<smallcaps>CRIPT</smallcaps>™ II and MMLV H+. Experiments are ongoing to determine if S<smallcaps>UPER</smallcaps>S<smallcaps>CRIPT</smallcaps>™ III, which contains the M289L mutation in addition to the H204R, T306K, and F309N mutations, also has this improvement.</div>
<div class="description-paragraph" id="p-0417" num="0443">Specific Activity: The specific activity for S<smallcaps>UPER</smallcaps>S<smallcaps>CRIPT</smallcaps>™ III is about 25% higher than S<smallcaps>UPER</smallcaps>S<smallcaps>CRIPT</smallcaps>™ II. RT DNA polymerase unit activity was assayed with (rA)<sub>630</sub>● p(dT)<sub>12-18 </sub>at 37° C. and protein concentration was determined by Bradford assay. Table 14 shows the RNA-directed DNA polymerase specific activity of S<smallcaps>UPER</smallcaps>S<smallcaps>CRIPT</smallcaps>™ II and S<smallcaps>UPER</smallcaps>S<smallcaps>CRIPT</smallcaps>™ III using (rA)<sub>630</sub>● p(dT)<sub>12-18 </sub>at 37° C. for 10 min.</div>
<div class="description-paragraph" id="p-0418" num="0444">
<tables id="TABLE-US-00016" num="00016">
<patent-tables colsep="0" frame="none" rowsep="0">
<table align="left" class="description-table" cols="3" colsep="0" rowsep="0" width="100%">
<thead>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td" colspan="3" nameend="2" namest="offset" rowsep="1">TABLE 14</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td align="center" class="description-td" colspan="3" nameend="2" namest="offset" rowsep="1"> </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">DNA Polymerase Specific Activity</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">Enzyme</td>
<td class="description-td">(units/mg)</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td align="center" class="description-td" colspan="3" nameend="2" namest="offset" rowsep="1"> </td>
</tr>
</thead>
<tbody><tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">SUPERSCRIPT ™ II</td>
<td class="description-td">330,000</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">SUPERSCRIPT ™ III</td>
<td class="description-td">410,000</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td align="center" class="description-td" colspan="3" nameend="2" namest="offset" rowsep="1"> </td>
</tr>
</tbody></table>
</patent-tables>
</tables>
</div>
<div class="description-paragraph" id="p-0419" num="0445">Mono- and Divalent Metal Reaction Optima: Using primed CAT RNA as substrate, the Mg<sup>2+</sup> and KCl concentration optima were determined and are shown in <figref idrefs="DRAWINGS">FIGS. 10A and 10B</figref>. The DNA polymerase assay for S<smallcaps>UPER</smallcaps>S<smallcaps>CRIPT</smallcaps>™ III RT was conducted at 37° C. or 50° C. for 10 minutes under various concentrations of Mg<sup>2+</sup> (<figref idrefs="DRAWINGS">FIG. 10A</figref>) or KCl (<figref idrefs="DRAWINGS">FIG. 10B</figref>). S<smallcaps>UPER</smallcaps>S<smallcaps>CRIPT</smallcaps>™ II at 37° C. is included for comparison. The Mg<sup>2+</sup> concentration optima is at 3 mM at both 37° C. and 50° C. Though at 50° C. there appears to be a broader working range of 2-5 mM without much difference. For KCl, the optima is at 75 mM at both temperatures, but the working range is from 25-125 mM without much difference. Again at the higher temperature a broader range is allowed (25-200 mM). Since there is no change in optima between S<smallcaps>UPER</smallcaps>S<smallcaps>CRIPT</smallcaps>™ II and III, the 5× First Strand Buffer from S<smallcaps>UPER</smallcaps>S<smallcaps>CRIPT</smallcaps>™ II can be used as S<smallcaps>UPER</smallcaps>S<smallcaps>CRIPT</smallcaps>™ III's reaction buffer.</div>
<div class="description-paragraph" id="p-0420" num="0446">TdT activity: TdT or non-template directed nucleotide addition to the 3′ primer end is a well known property of many polymerases including RTs (Chen, D. et al., <i>Biotechniques </i>2001; 30(3):574-582). Using a DNA template (<figref idrefs="DRAWINGS">FIG. 11A</figref>), S<smallcaps>UPER</smallcaps>S<smallcaps>CRIPT</smallcaps>™ II adds 1-3 additional nucleotides to the end of the transcript. S<smallcaps>UPER</smallcaps>S<smallcaps>CRIPT</smallcaps>™ III however has a greatly reduced TdT and on a DNA template will not add any detectable nucleotides. On an RNA template (<figref idrefs="DRAWINGS">FIG. 11B</figref>), S<smallcaps>UPER</smallcaps>S<smallcaps>CRIPT</smallcaps>™ II adds 1-2 bases with 69% of the primer extended after 10 minutes and 90% extended after an hour. S<smallcaps>UPER</smallcaps>S<smallcaps>CRIPT</smallcaps>™ III does have some TdT activity on RNA template, but it is reduced with about 14% extended after 10 minutes and 50% extended after an hour at 45° C. as shown in <figref idrefs="DRAWINGS">FIG. 11B</figref>. This TdT activity is biased with the addition of dATP being strongly favored followed by dGTP, dCTP and dTTP for both S<smallcaps>UPER</smallcaps>S<smallcaps>CRIPT</smallcaps>™ II and III. TdT activity is temperature and time dependant. The optimal temperature for S<smallcaps>UPER</smallcaps>S<smallcaps>CRIPT</smallcaps>™ III is 45° C. to 50° C. while S<smallcaps>UPER</smallcaps>S<smallcaps>CRIPT</smallcaps>™ II is at 45° C. (<figref idrefs="DRAWINGS">FIG. 11</figref>). To further minimize TdT activity in any RT one must shorten the extension time or lower the temperature.</div>
<div class="description-paragraph" id="p-0421" num="0447">Fidelity assays. The results of the fidelity assays are shown in Tables 15 and 16.</div>
<div class="description-paragraph" id="p-0422" num="0448">
<tables id="TABLE-US-00017" num="00017">
<patent-tables colsep="0" frame="none" rowsep="0">
<table align="left" class="description-table" cols="1" colsep="0" rowsep="0" width="100%">
<thead>
<tr class="description-tr">
<td class="description-td" colspan="1" nameend="1" namest="1" rowsep="1">TABLE 15</td>
</tr>
</thead>
<tbody><tr class="description-tr">
<td align="center" class="description-td" colspan="1" nameend="1" namest="1" rowsep="1"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">rpsL Fidelity Assay</td>
</tr>
</tbody></table>
<table align="left" class="description-table" cols="5" colsep="0" rowsep="0" width="100%">
<tbody><tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">Mutated</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">RT</td>
<td class="description-td">Temp</td>
<td class="description-td">Wt rpsL</td>
<td class="description-td">rpsL</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td align="center" class="description-td" colspan="5" nameend="4" namest="offset" rowsep="1"> </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">SUPERSCRIPT ™ III</td>
<td class="description-td">45° C.</td>
<td class="description-td">3.9 × 10<sup>−4</sup> </td>
<td class="description-td">3.4 × 10<sup>−5</sup> </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">SUPERSCRIPT ™ II</td>
<td class="description-td">45° C.</td>
<td class="description-td">3.5 × 10<sup>−4</sup> </td>
<td class="description-td">3.1 × 10<sup>−5</sup> </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">MMLV H+</td>
<td class="description-td">37° C.</td>
<td class="description-td">3.9 × 10<sup>−4</sup> </td>
<td class="description-td">NA</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td align="center" class="description-td" colspan="5" nameend="4" namest="offset" rowsep="1"> </td>
</tr>
</tbody></table>
</patent-tables>
</tables>
</div>
<div class="description-paragraph" id="p-0423" num="0449">
<tables id="TABLE-US-00018" num="00018">
<patent-tables colsep="0" frame="none" rowsep="0">
<table align="left" class="description-table" cols="1" colsep="0" rowsep="0" width="100%">
<thead>
<tr class="description-tr">
<td class="description-td" colspan="1" nameend="1" namest="1" rowsep="1">TABLE 16</td>
</tr>
</thead>
<tbody><tr class="description-tr">
<td align="center" class="description-td" colspan="1" nameend="1" namest="1" rowsep="1"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">lacZα Fidelity Assay</td>
</tr>
</tbody></table>
<table align="left" class="description-table" cols="4" colsep="0" rowsep="0" width="100%">
<tbody><tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">RT</td>
<td class="description-td">Temp</td>
<td class="description-td">Error Rate</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td align="center" class="description-td" colspan="4" nameend="3" namest="offset" rowsep="1"> </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">SUPERSCRIPT ™ III</td>
<td class="description-td">45° C.</td>
<td class="description-td">2.9 × 10<sup>−5</sup> </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">50° C.</td>
<td class="description-td">2.7 × 10<sup>−5</sup> </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">55° C.</td>
<td class="description-td">1.8 × 10<sup>−5</sup> </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">SUPERSCRIPT ™ II</td>
<td class="description-td">45° C.</td>
<td class="description-td">2.9 × 10<sup>−5</sup> </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">MMLV H+</td>
<td class="description-td">45° C.</td>
<td class="description-td">4.8 × 10<sup>−5</sup> </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td align="center" class="description-td" colspan="4" nameend="3" namest="offset" rowsep="1"> </td>
</tr>
</tbody></table>
</patent-tables>
</tables>
</div>
<div class="description-paragraph" id="p-0424" num="0450">Fidelity assays for S<smallcaps>UPER</smallcaps>S<smallcaps>CRIPT</smallcaps>™ III, S<smallcaps>UPER</smallcaps>S<smallcaps>CRIPT</smallcaps>™ II and MMLV Rnase H+ RTs. rpsL assay was conducted with either WT rpsL gene or rpsL gene with mutation hotspots removed. A wt rpsL DNA fragment was used as control to see the spontaneously mutation. The error rate is about 10<sup>−6 </sup>to 10<sup>−7</sup>. The lacZα assay was conducted with gapped DNA starting substrate used as a control with a background error rate of 2.2×10<sup>−5</sup>.</div>
<div class="description-paragraph" id="p-0425" num="0451">Thermal Stability of Purified Mutant RTs: The rate of chemical RNA breakdown catalyzed by Mg<sup>2+</sup> increases dramatically as the temperature is increased above 55° C. to 60° C. under cDNA synthesis reaction conditions. In order to characterize their intrinsic thermal stability, the half lives of M-MLV, S<smallcaps>UPER</smallcaps>S<smallcaps>CRIPT</smallcaps>™ II and S<smallcaps>UPER</smallcaps>S<smallcaps>CRIPT</smallcaps>™ III were measured at elevated temperatures. The RT enzymes were first diluted to 0.2-0.5 Unit/μl in the 1× first strand buffer in the presence of 0.05% of NP-40. If diluted in absence of detergent, the RT activity is reduced to more than 90% without incubation (data not shown). <figref idrefs="DRAWINGS">FIGS. 12A, 12B</figref>, and <b>12</b>C and Table 17 show the half-lives of M-MLV, S<smallcaps>UPER</smallcaps>S<smallcaps>CRIPT</smallcaps>™ II, and S<smallcaps>UPER</smallcaps>S<smallcaps>CRIPT</smallcaps>™ III at various temperatures. The RT half-lives were detected in 1× First strand reaction buffer in presence of 0.05% NP-40. Mixtures (20 μl) were incubated for various times in 0.5-ml tubes in a thermocycler at 50° C., 55° C., or 60° C. and contained 1× First Strand Buffer (50 mM Tris-HCl (pH 8.3), 75 mM KCl, 3 mM MgCl<sub>2</sub>), 10 mM DTT, 3 mM MgCl<sub>2 </sub>(total to 6 mM), 0.5 mM dTTP, 0.05% (v/v) NP-40, [methyl-3H]dTTP (20 μCi/ml), and 5 μl of RT. Incubation was stopped by placing the tube in ice. An aliquot (5 μl) was assayed for remaining units with 5 mM poly(A)/3 mM oligo(dT)<sub>25</sub>. The half live of S<smallcaps>UPER</smallcaps>S<smallcaps>CRIPT</smallcaps>™ III at 50° C. was 220 minutes, about 35 fold longer than that of S<smallcaps>UPER</smallcaps>S<smallcaps>CRIPT</smallcaps>™ II. The half live of S<smallcaps>UPER</smallcaps>S<smallcaps>CRIPT</smallcaps>™ III at 55° C. was 24 min, ˜10 fold longer than that of S<smallcaps>UPER</smallcaps>S<smallcaps>CRIPT</smallcaps>™ II.</div>
<div class="description-paragraph" id="p-0426" num="0452">
<tables id="TABLE-US-00019" num="00019">
<patent-tables colsep="0" frame="none" rowsep="0">
<table align="left" class="description-table" cols="1" colsep="0" rowsep="0" width="100%">
<thead>
<tr class="description-tr">
<td class="description-td" colspan="1" nameend="1" namest="1" rowsep="1">TABLE 17</td>
</tr>
</thead>
<tbody><tr class="description-tr">
<td align="center" class="description-td" colspan="1" nameend="1" namest="1" rowsep="1"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">Summary of RT Half lives at 50° C., 55° C., and 60° C.</td>
</tr>
</tbody></table>
<table align="left" class="description-table" cols="4" colsep="0" rowsep="0" width="100%">
<tbody><tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">M-MLV</td>
<td class="description-td">SUPERSCRIPT ™ II</td>
<td class="description-td">SUPERSCRIPT ™ III</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">(min)</td>
<td class="description-td">(min)</td>
<td class="description-td">(min)</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td align="center" class="description-td" colspan="4" nameend="3" namest="offset" rowsep="1"> </td>
</tr>
</tbody></table>
<table align="left" class="description-table" cols="4" colsep="0" rowsep="0" width="100%">
<tbody><tr class="description-tr">
<td class="description-td">50° C.</td>
<td class="description-td">ND</td>
<td class="description-td">6.1</td>
<td class="description-td">220</td>
</tr>
<tr class="description-tr">
<td class="description-td">55° C.</td>
<td class="description-td">1.5</td>
<td class="description-td">2.2</td>
<td class="description-td">24</td>
</tr>
<tr class="description-tr">
<td class="description-td">60° C.</td>
<td class="description-td">0.6</td>
<td class="description-td">ND</td>
<td class="description-td">2.5</td>
</tr>
<tr class="description-tr">
<td align="center" class="description-td" colspan="4" nameend="4" namest="1" rowsep="1"> </td>
</tr>
</tbody></table>
</patent-tables>
</tables>
</div>
<div class="description-paragraph" id="p-0427" num="0453">Full-length cDNA profiling of RT: A practical method for judging the useful higher temperature of the DNA polymerase activity of RT is an assessment of the effect of increasing reaction temperature on the amounts of full-length cDNA products synthesized by RT from mixture of cRNA of various lengths. The labeled full-length cDNA products can be separated and quantified on an alkaline agarose gel. As the reaction temperate is increased, full-length cDNA products disappear starting with those derived from longer cRNA until a temperature is reached where no discernible full-length product of any length is synthesized. For comparison, we included Clontech's PowerScript RT, Stratagene's StrataScript RT, Promega's ImProm II RT, and S<smallcaps>UPER</smallcaps>S<smallcaps>CRIPT</smallcaps>™ II (Invitrogen Corporation, Carlsbad, Calif., catalog #18064022). Only S<smallcaps>UPER</smallcaps>S<smallcaps>CRIPT</smallcaps>™ III could produce a clear 9.5 kb full-length product at 50° C., and S<smallcaps>UPER</smallcaps>S<smallcaps>CRIPT</smallcaps>™ II can get a very faint 9.5 kb band.</div>
<div class="description-paragraph" id="p-0428" num="0454">To compare RT from different companies, the same number of units of RT were used for comparison. PowerScript RT is ˜350 unit/μl, StrataScript RT is 50 unit/μl, ImProm II is 50 unit/μl. S<smallcaps>UPER</smallcaps>S<smallcaps>CRIPT</smallcaps>™ II, S<smallcaps>UPER</smallcaps>S<smallcaps>CRIPT</smallcaps>™ III and PowerScript RT were diluted to 50 unit/μl in RT dilution buffer. 100 units (2 μl) of RT enzyme were used. The amounts of full-length products generated were determined and are shown in Tables 17 and 18 for various length templates. S<smallcaps>UPER</smallcaps>S<smallcaps>CRIPT</smallcaps>™ III RT shows improved performance at 50° C. and 55° C. compared to the other RTs.</div>
<div class="description-paragraph" id="p-0429" num="0455">
<tables id="TABLE-US-00020" num="00020">
<patent-tables colsep="0" frame="none" rowsep="0">
<table align="left" class="description-table" cols="1" colsep="0" rowsep="0" width="100%">
<thead>
<tr class="description-tr">
<td class="description-td" colspan="1" nameend="1" namest="1" rowsep="1">TABLE 17</td>
</tr>
</thead>
<tbody><tr class="description-tr">
<td align="center" class="description-td" colspan="1" nameend="1" namest="1" rowsep="1"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">Comparison of full length cDNA produced by various RTs.</td>
</tr>
</tbody></table>
<table align="left" class="description-table" cols="3" colsep="0" rowsep="0" width="100%">
<tbody><tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">Total</td>
<td class="description-td">Amount of</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">cDNA</td>
<td class="description-td">Full-Length cDNA</td>
</tr>
</tbody></table>
<table align="left" class="description-table" cols="5" colsep="0" rowsep="0" width="100%">
<tbody><tr class="description-tr">
<td class="description-td">Temp.</td>
<td class="description-td">RT</td>
<td class="description-td">Pmol</td>
<td class="description-td">7.5 kb (ng)</td>
<td class="description-td">2.4 kb (ng)</td>
</tr>
<tr class="description-tr">
<td align="center" class="description-td" colspan="5" nameend="5" namest="1" rowsep="1"> </td>
</tr>
</tbody></table>
<table align="left" class="description-table" cols="5" colsep="0" rowsep="0" width="100%">
<tbody><tr class="description-tr">
<td class="description-td">45° C.</td>
<td class="description-td">PowerScript</td>
<td class="description-td">293</td>
<td class="description-td">75</td>
<td class="description-td">64</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">StrataScript</td>
<td class="description-td">199</td>
<td class="description-td">47</td>
<td class="description-td">35</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">ImProm II</td>
<td class="description-td">256</td>
<td class="description-td">41</td>
<td class="description-td">44</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">SUPERSCRIPT ™ III</td>
<td class="description-td">345</td>
<td class="description-td">99</td>
<td class="description-td">72</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">SUPERSCRIPT ™ II</td>
<td class="description-td">300</td>
<td class="description-td">100</td>
<td class="description-td">59</td>
</tr>
<tr class="description-tr">
<td class="description-td">50° C.</td>
<td class="description-td">PowerScript</td>
<td class="description-td">180</td>
<td class="description-td">54</td>
<td class="description-td">31</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">StrataScript</td>
<td class="description-td">9</td>
<td class="description-td">&lt;2</td>
<td class="description-td">&lt;2</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">ImProm II</td>
<td class="description-td">42</td>
<td class="description-td">&lt;2</td>
<td class="description-td">&lt;2</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">SUPERSCRIPT ™ III</td>
<td class="description-td">323</td>
<td class="description-td">97</td>
<td class="description-td">68</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">SUPERSCRIPT ™ II</td>
<td class="description-td">170</td>
<td class="description-td">46</td>
<td class="description-td">22</td>
</tr>
<tr class="description-tr">
<td class="description-td">55° C.</td>
<td class="description-td">PowerScript</td>
<td class="description-td">5</td>
<td class="description-td">&lt;2</td>
<td class="description-td">&lt;2</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">StrataScript</td>
<td class="description-td">5</td>
<td class="description-td">&lt;2</td>
<td class="description-td">&lt;2</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">ImProm II</td>
<td class="description-td">3</td>
<td class="description-td">&lt;2</td>
<td class="description-td">&lt;2</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">SUPERSCRIPT ™ III</td>
<td class="description-td">37</td>
<td class="description-td">3</td>
<td class="description-td">22</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">SUPERSCRIPT ™ II</td>
<td class="description-td">3</td>
<td class="description-td">&lt;2</td>
<td class="description-td">&lt;2</td>
</tr>
<tr class="description-tr">
<td align="center" class="description-td" colspan="5" nameend="5" namest="1" rowsep="1"> </td>
</tr>
</tbody></table>
</patent-tables>
</tables>
</div>
<div class="description-paragraph" id="p-0430" num="0456">The amounts of full-length product were established by cutting the region in a dried 1.2% alkaline agarose gel corresponding to the size of each full-length band and counting it in a scintillation counter. Only amounts of full-length products &gt;2 ng could be seen as discernible bands on the gel autoradiograph. The cDNA synthesis reactions were performed in 20 μl with 0.25 μg of 2.4 kb, 0.5 mg of 7.5 kb cRNA, and 0.1 μg of oligo (dT)<sub>25</sub>. If provided, buffers and other components were used from their own kits plus 0.1 μl of α-<sup>32</sup>P-dCTP (10 μCi/μl). All the reactions were “hot start.” In “hot start” reactions, tubes were incubated at reaction temperature for 2 minutes before adding enzyme. 2 μl of RT (50 u/μl) was added and incubated at 45° C. or 50° C. or 55° C. for 50 min. The reactions were stopped by adding 10 μl of 0.5 M EDTA. After ethanol precipitation, the products were loaded on 1.2% agarose gel. The dried gel was exposed on the film for 1 hour.</div>
<div class="description-paragraph" id="p-0431" num="0457">
<tables id="TABLE-US-00021" num="00021">
<patent-tables colsep="0" frame="none" rowsep="0">
<table align="left" class="description-table" cols="1" colsep="0" rowsep="0" width="100%">
<thead>
<tr class="description-tr">
<td class="description-td" colspan="1" nameend="1" namest="1" rowsep="1">TABLE 18</td>
</tr>
</thead>
<tbody><tr class="description-tr">
<td align="center" class="description-td" colspan="1" nameend="1" namest="1" rowsep="1"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">Comparison of full length cDNA produced by various RTs.</td>
</tr>
</tbody></table>
<table align="left" class="description-table" cols="3" colsep="0" rowsep="0" width="100%">
<tbody><tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">Amount of full</td>
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">length cDNA (ng)</td>
</tr>
</tbody></table>
<table align="left" class="description-table" cols="5" colsep="0" rowsep="0" width="100%">
<tbody><tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">RT</td>
<td class="description-td">Temp (° C.)</td>
<td class="description-td">9.5 kb</td>
<td class="description-td">7.5 kb</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td align="center" class="description-td" colspan="5" nameend="4" namest="offset" rowsep="1"> </td>
</tr>
</tbody></table>
<table align="left" class="description-table" cols="5" colsep="0" rowsep="0" width="100%">
<tbody><tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">SUPERSCRIPT ™</td>
<td class="description-td">45</td>
<td class="description-td">19</td>
<td class="description-td">54</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">III</td>
<td class="description-td">50</td>
<td class="description-td">26</td>
<td class="description-td">78</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">55</td>
<td class="description-td">13</td>
<td class="description-td">53</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">StratScript</td>
<td class="description-td">45</td>
<td class="description-td">4</td>
<td class="description-td">10</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">50</td>
<td class="description-td">0</td>
<td class="description-td">0</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">55</td>
<td class="description-td">0</td>
<td class="description-td">0</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">ImProm II</td>
<td class="description-td">45</td>
<td class="description-td">3</td>
<td class="description-td">9</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">50</td>
<td class="description-td">0</td>
<td class="description-td">0</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">55</td>
<td class="description-td">0</td>
<td class="description-td">0</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">OmniScript</td>
<td class="description-td">45</td>
<td class="description-td">0</td>
<td class="description-td">0</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">50</td>
<td class="description-td">0</td>
<td class="description-td">0</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">55</td>
<td class="description-td">0</td>
<td class="description-td">0</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">PowerScript</td>
<td class="description-td">45</td>
<td class="description-td">19</td>
<td class="description-td">38</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">50</td>
<td class="description-td">3</td>
<td class="description-td">25</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">55</td>
<td class="description-td">0</td>
<td class="description-td">0</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">M-MLV</td>
<td class="description-td">45</td>
<td class="description-td">10</td>
<td class="description-td">39</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">50</td>
<td class="description-td">0</td>
<td class="description-td">3</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">55</td>
<td class="description-td">0</td>
<td class="description-td">0</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">ThermoScript</td>
<td class="description-td">45</td>
<td class="description-td">16</td>
<td class="description-td">38</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">50</td>
<td class="description-td">29</td>
<td class="description-td">56</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">55</td>
<td class="description-td">3</td>
<td class="description-td">13</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td align="center" class="description-td" colspan="5" nameend="4" namest="offset" rowsep="1"> </td>
</tr>
</tbody></table>
</patent-tables>
</tables>
</div>
<div class="description-paragraph" id="p-0432" num="0458">cDNA synthesis reaction mixtures contained 9.5 kb (1.1 μg) and 7.5 kb (0.8 μg) cRNA. The cDNA synthesis reactions were performed in 20 μl with 20 μg of Hela total RNA, and 0.5 μg of oligo (dT)<sub>25</sub>. If provided, buffers and other components used were from their own kits plus 0.1 μl of α-<sup>32</sup>P-dCTP (10 μCi/μl). All the reactions were “hot start,” 1 μl of RT was added and incubated at 45° C. or 55° C. for 50 min. 400 or 200 units of S<smallcaps>UPER</smallcaps>S<smallcaps>CRIPT</smallcaps>™ III were used for comparison. The reactions were stopped by adding 10 μl of 0.5 M EDTA. After ethanol precipitation, the products were loaded on 1.2% agarose gel. The dried gel was exposed on the film for 1 hour.</div>
<div class="description-paragraph" id="p-0433" num="0459">In an alternate procedure for comparing RTs from various sources, a comparison of cDNA synthesis from Hela total RNA was performed. The cDNA size synthesized by S<smallcaps>UPER</smallcaps>S<smallcaps>CRIPT</smallcaps>™ III RT at 55° C. is from 0.5-10 kb. cDNA synthesis reactions were performed in 20 μl with 20 μg of Hela total RNA, and 0.5 μg of oligo(dT)<sub>25</sub>. If provided, buffers and other components used were from their own kits plus 0.1 μl of α-<sup>32</sup>P-dCTP (10 μCi/μl). All the reactions were “hot start,” 1 μl of RT was added and incubated at 45° C. or 55° C. for 50 min. 400 or 200 units of S<smallcaps>UPER</smallcaps>S<smallcaps>CRIPT</smallcaps>™ III were used for comparison. The reactions were stopped by adding 10 μl of 0.5 M EDTA. Total cDNA synthesis was obtained by TCA-precipitation of 5 μl of mixtures. After ethanol precipitation of the rest of the mixture, the products were loaded on 1.2% agarose gel. The dried gel was exposed on the film for 1 hour.</div>
<div class="description-paragraph" id="p-0434" num="0460">Each reaction was performed according to each vendor's recommendation. In these experiments, we used the buffers provided with the RTs. DTT, RNase inhibitor, dNTP were used at the recommended concentration and from the kit if provided. Otherwise, Invitrogen's reagents were used. 1 μl of RT was used in these competitive audit. Table 18 shows the result when using 9.5 kb and 7.5 kb cRNA. <figref idrefs="DRAWINGS">FIG. 13</figref> shows the results of a competitive audit for cDNA profiling using poly (A)-tailed RNA ladder. The cDNA synthesis reactions were performed in 20 μl with 2 μg of RNA Ladder and 0.5 μg of oligo(dT)<sub>25</sub>. If provided, buffers and other components were used from their own kits plus 0.1 μl of α-<sup>32</sup>P-dCTP (10 μCi/μl). All the reactions were “hot start,” 1 μl of RT was added and incubated at 45° C. or 50° C. or 55° C. for 50 min. 400 or 200 units of S<smallcaps>UPER</smallcaps>S<smallcaps>CRIPT</smallcaps>™ III were used for comparison. The reactions were stopped by adding 10 μl of 0.5 M EDTA. After ethanol precipitation, the products were loaded in different order on two 1.2% agarose gels. The dried gels were exposed on the film for 1 hour.</div>
<heading id="h-0044">Example 8</heading>
<heading id="h-0045">RT-PCR Applications</heading>
<div class="description-paragraph" id="p-0435" num="0461">RT-PCR has become an effective tool in such applications as detecting RNA, gene expression, profiling, gene quantitation, and cloning full-length genes. An RNase H minus mutant of MMLV RT, S<smallcaps>UPER</smallcaps>S<smallcaps>CRIPT</smallcaps>™ II RT, is widely accepted as the RT choice since it has already proven that it produces higher cDNA yield and longer target capacity than MMLV RT.</div>
<div class="description-paragraph" id="p-0436" num="0462">Recently we have engineered a new generation RT—S<smallcaps>UPER</smallcaps>S<smallcaps>CRIPT</smallcaps>™ III™ RT. In addition to all the premier features of S<smallcaps>UPER</smallcaps>S<smallcaps>CRIPT</smallcaps>™ II RT, it also provides a high temperature RT capacity up to 55 degrees, which is approximately 5 degrees or more higher than S<smallcaps>UPER</smallcaps>S<smallcaps>CRIPT</smallcaps>™ II RT. The enzyme half-life of S<smallcaps>UPER</smallcaps>S<smallcaps>CRIPT</smallcaps>™ II RT and S<smallcaps>UPER</smallcaps>S<smallcaps>CRIPT</smallcaps>™ III RT at 50° C. is 6 minutes and 220 minutes respectively. Efficient full-length cDNA synthesis activity occurs with S<smallcaps>UPER</smallcaps>S<smallcaps>CRIPT</smallcaps>™ III even at 55 degrees RT temperature. High yield and more specific RT-PCR products were also produced by this enzyme at elevated RT temperatures. S<smallcaps>UPER</smallcaps>S<smallcaps>CRIPT</smallcaps>™ III provides a tool for efficient cDNA synthesis for difficult RNA targets such as high GC and secondary structured templates.</div>
<div class="description-paragraph" id="p-0437" num="0463">The enzyme has also been optimized for RT-PCR for both sensitivity and long targets. We have demonstrated that S<smallcaps>UPER</smallcaps>S<smallcaps>CRIPT</smallcaps>™ III RT is able to amplify as little as 1 pg of starting HeLa RNA and amplify targets up to 12.3 kb in length. This enzyme, when used with the accompanying buffers and conditions, provides performance better than that of other MMLV derivative reverse transcriptases.</div>
<div class="description-paragraph" id="h-0046" num="0000">Materials and Methods</div>
<div class="description-paragraph" id="p-0438" num="0464">Kit Components: S<smallcaps>UPER</smallcaps>S<smallcaps>CRIPT</smallcaps>™ III RT (200 U/μl), 5× First-Strand Buffer, 0.1M DTT.</div>
<div class="description-paragraph" id="p-0439" num="0465">S<smallcaps>UPER</smallcaps>S<smallcaps>CRIPT</smallcaps>™ III RT=H204R, M289L, T306K, and F309N mutations and may be referred to as the LEFN RT.</div>
<div class="description-paragraph" id="p-0440" num="0466">5× First-Strand Buffer: 250 mM Tris-HCl (pH 8.3), 375 mM potassium chloride, and 15 mM magnesium chloride.</div>
<div class="description-paragraph" id="p-0441" num="0467">S<smallcaps>UPER</smallcaps>S<smallcaps>CRIPT</smallcaps>™ III RT Storage Buffer: 20 mM Tris-HCl, pH 7.5, 100 mM NaCl, 1 mM EDTA, 1 mM DTT, 0.01% NP-40, 50% (v/v) glycerol.</div>
<div class="description-paragraph" id="p-0442" num="0468">RNA: Total HeLa RNA and total rat brain RNA were isolated using TRIZOL® Reagent.</div>
<div class="description-paragraph" id="p-0443" num="0469">Gel Electrophoresis: RT-PCR products (10 μl) were analyzed by electrophoresis on 0.8% to 1.5% (w/v) agarose gels in 0.5×TBE with 0.4 μg/ml ethidium bromide.</div>
<div class="description-paragraph" id="p-0444" num="0470">RT-PCR Procedure: First-Strand cDNA Synthesis for RT-PCR: A 20-μl reaction volume can be used for 10 pg-5 μg of total RNA or 10 pg-500 ng of mRNA. Add the following components to a nuclease-free microfuge tube: 1 μl Oligo(dT)<sub>20 </sub>(500 μg/ml), 10 pg to 5 μg total RNA, 1 μl 10 mM dNTP mix (10 mM each dATP, dTTP, dGTP, and dCTP at neutral pH), and sterile, distilled water to 12 μl. Alternatively, 50-250 ng random primers or 2 pmole of a gene specific primer may be used. Use of random primers requires incubation at 25° C. for 10 min before the 50° C. incubation.</div>
<div class="description-paragraph" id="p-0445" num="0471">Heat the mixture at 65° C. for 5 min and then place on ice. Collect the contents of the tube by brief centrifugation and add: 4 μl 5× First-Strand Buffer, 2 μl 0.1 M DTT, 1 μl RNaseOUT Recombinant Ribonuclease Inhibitor (40 units/μl, when using less than 50 ng of starting RNA, the addition of an RNase inhibitor (e.g., RNaseOUT, Invitrogen Corporation, Carlsbad, Calif., catalog #10777019) is essential).</div>
<div class="description-paragraph" id="p-0446" num="0472">Mix the contents of the tube gently and incubate at 50° C. for 2 minutes. Add 1 μl (200-400 units) of S<smallcaps>UPER</smallcaps>S<smallcaps>CRIPT</smallcaps>™ III RT, mix by pipetting gently up and down. Incubate 50 min at 50° C. (cDNA synthesis can be performed at 42° C.-60° C. for oligo(dT)<sub>20 </sub>or gene specific primers). Inactivate the reaction by heating at 70° C. for 15 min. The cDNA can now be used as a template for amplification in PCR or can be stored at −20° C. until use. However, amplification of some PCR targets (those &gt;1 kb) may require removal of RNA complementary to the cDNA. To remove RNA complementary to the cDNA, add 1 μl (2 units) of <i>E. coli </i>RNase H and incubate at 37° C. for 20 min.</div>
<div class="description-paragraph" id="h-0047" num="0000">RT-PCR Optimization</div>
<div class="description-paragraph" id="p-0447" num="0473">For the following experiments, either EFN (His-tag), EFN (no tag), LEFN (His-tag), or LEFN (tag, no His) were used in finding the optimal conditions for LEFN (tag, no His). These 4 reverse transcriptases differ in their thermal-stability profiles. The modified version of S<smallcaps>UPER</smallcaps>S<smallcaps>CRIPT</smallcaps>™ III, EFN, contains the H204R, T306K, and F309N mutations. LEFN, otherwise known as S<smallcaps>UPER</smallcaps>S<smallcaps>CRIPT</smallcaps>™ III RT, contains the M289L mutation in addition to the H204R, T306K, and F309N mutations.</div>
<div class="description-paragraph" id="p-0448" num="0474">Unless otherwise noted, 20 μl RT reactions were typically done using 2.5 μM Oligo(dT)<sub>20 </sub>(500 ng Oligo(dT)<sub>12-18 </sub>for S<smallcaps>UPER</smallcaps>S<smallcaps>CRIPT</smallcaps>™ II RT), 0.5 mM dNTPs, 1× First-Strand Buffer, 10 mM DTT, and 40 units of RNaseOUT. In “hot start” reactions, tubes were incubated at reaction temperature for 2 minutes before adding enzyme. Reactions were treated with 2 units of RNase H at 37° C. for 20 min after cDNA synthesis.</div>
<div class="description-paragraph" id="p-0449" num="0475">For the evaluation of these RTs in RT-PCR, standard 50 μl PCR reactions were performed. For primer sets deigned to amplify &lt;4 kb, PCR reactions consisted of 0.2 μM primers, 200 μM each dNTP, 1×PCR Buffer, 1.5 mM MgCl<sub>2</sub>, 2 μl of the cDNA reaction, and 2 units of PLATINUM® Taq DNA Polymerase. For primer sets designed to amplify &gt;4 kb, PCR reactions consisted of 0.2 μM primers, 200 μM each dNTP, 1× High Fidelity PCR Buffer, 2 mM MgSO<sub>4</sub>, 2 μl of the cDNA reaction, and 1 unit of PLATINUM® Taq DNA Polymerase High Fidelity. After an incubation of 94° C. for 2 min, amplification was 35 to 40 cycles of 94° C. for 15 s, 55° C.-60° C. for 30 s, and 68° C. for 1 min/kb. 10 μl of the PCR reactions were analyzed on agarose gels containing 0.4 μg/ml EtBr. Primers used in PCR are found in Table 19.</div>
<div class="description-paragraph" id="p-0450" num="0476">cDNA synthesis buffer: Reactions with 50 or 200 units of S<smallcaps>UPER</smallcaps>S<smallcaps>CRIPT</smallcaps>™ II RT or EFN (His-tag) were assembled using the reagent systems of S<smallcaps>UPER</smallcaps>S<smallcaps>CRIPT</smallcaps>™ II RT (stand-alone), S<smallcaps>UPER</smallcaps>S<smallcaps>CRIPT</smallcaps>™ II First-Strand Synthesis System for RT-PCR, and ThermoScript RT-PCR System. The conditions for each are as follows:
</div> <ul> <li id="ul0005-0001" num="0477">1) S<smallcaps>UPER</smallcaps>S<smallcaps>CRIPT</smallcaps>™ II RT stand-alone: 50 mM Tris-HCl (pH 8.3), 75 mM KCl, 3 mM MgCl<sub>2</sub>, 0.5 mM dNTPs</li> <li id="ul0005-0002" num="0478">2) S<smallcaps>UPER</smallcaps>S<smallcaps>CRIPT</smallcaps>™ II First-Strand Synthesis System for RT-PCR: 20 mM Tris-HCl (pH 8.4), 50 mM KCl, 5 mM MgCl<sub>2</sub>, 0.5 mM dNTPs</li> <li id="ul0005-0003" num="0479">3) ThermoScript RT-PCR System: 50 mM Tris-acetate (pH 8.4), 75 mM potassium acetate, 8 mM magnesium acetate, 1 mM dNTPs
<br/>
20 μl RT reactions containing 1 pg to 100 ng of starting total HeLa RNA were performed at 45° C. for 50 min followed by 85° C. for 5 min. 2 μl of the resulting cDNA were added to 50 μl PCR reactions containing the β-actin 353 bp, BF 2.4 kb, Pol ε 6.8 kb, or APC 8.5 kb primer set (see Table 19).
</li> </ul>
<div class="description-paragraph" id="p-0451" num="0480">Magnesium chloride was titrated into cDNA synthesis buffers in final reaction concentrations of 1 to 10 mM, with dNTP concentrations of 0.5 mM and 1 mM tested concurrently. These buffers were used in 20 μl RT reactions performed at 45° C. for 50 min followed by 85° C. for 5 min with 1 pg, 100 ng, or 5 μg of total HeLa RNA and 50 units or 200 units of EFN (His-tag). 2 μl of the resulting cDNA were added to 50 μl PCR reactions containing the β-actin 353 bp or Pol ε 6.8 kb target (the 5 μg cDNA samples were diluted 100 fold before addition).</div>
<div class="description-paragraph" id="p-0452" num="0481">5× First-Strand Buffer containing Tris-HCl at pH 8.0, 8.4, 8.8 and 8.3 were prepared to determine the ideal pH for the enzyme to function at higher reaction temperatures. 20 μl RT reactions containing 100 ng to 500 ng of total HeLa RNA and 200 units of S<smallcaps>UPER</smallcaps>S<smallcaps>CRIPT</smallcaps>™ II or 400 units of LEFN (His-tag) were performed at 50° C.-60° C. for 50 min (hot start) followed by 70° C. for 15 min using these different buffers. 2 μl of the resulting cDNA were added to 50 μl PCR reactions containing the BF 2.4 kb or Pol ε 6.8 kb primer set.</div>
<div class="description-paragraph" id="p-0453" num="0482">RT reaction temperature: S<smallcaps>UPER</smallcaps>S<smallcaps>CRIPT</smallcaps>™ II RT (50 units), EFN (His-tag) (50 and 200 units), and ThermoScript RT (15 units) were compared in RT reactions from 42° C. to 60° C. 20 μl RT reactions containing 1 pg to 100 ng of starting total HeLa RNA were performed at the given temperatures for 50 min (hot start) followed by 85° C. for 5 min. 2 μl of the resulting cDNA were added to 50 μl PCR reactions containing the β-actin 353 bp, TSC 5.3 kb, or Pol ε 6.8 kb primer set.</div>
<div class="description-paragraph" id="p-0454" num="0483">S<smallcaps>UPER</smallcaps>S<smallcaps>CRIPT</smallcaps>™ II RT and EFN (no tag) were compared at 200 units in RT reactions at 45° C. and 50° C. 20 μl RT reactions containing 1 ng to 1 μg of total HeLa RNA were performed at the given temperatures for 50 min (hot start) followed by 70° C. for 15 min. 2 μl of the resulting cDNA were added to 50 μl PCR reactions containing primer sets from 2.4 kb to 9.3 kb.</div>
<div class="description-paragraph" id="p-0455" num="0484">S<smallcaps>UPER</smallcaps>S<smallcaps>CRIPT</smallcaps>™ II RT, EFN (His-tag), and EFN (no tag) were compared at 200 units in RT reactions from 45° C. to 55° C. 20 μl RT reactions containing 1 pg to 100 ng of total HeLa RNA were performed at the given temperatures for 50 min (hot start) followed by 70° C. for 15 min. 2 μl of the resulting cDNA were added to 50 μl PCR reactions containing either the β-actin 353 bp or Pol ε 6.8 kb primer set.</div>
<div class="description-paragraph" id="p-0456" num="0485">S<smallcaps>UPER</smallcaps>S<smallcaps>CRIPT</smallcaps>™ II RT, EFN (His-tag), EFN (no tag), and LEFN (His-tag) were compared at 200 units in RT reactions from 45° C. to 55° C. 20 μl RT reactions containing 1 pg to 1 μg of total HeLa RNA were performed at the given temperatures for 50 min (hot start) followed by 70° C. for 15 min. 2 μl of the resulting cDNA were added to 50 μl PCR reactions containing either the β-actin 353 bp or Pol ε 6.8 kb primer set.</div>
<div class="description-paragraph" id="p-0457" num="0486">S<smallcaps>UPER</smallcaps>S<smallcaps>CRIPT</smallcaps>™ II RT, EFN (His-tag), EFN (no tag), and LEFN (His-tag) were compared at 200 units in RT reactions from 45° C. to 55° C. with a gene-specific primer (CBP 1.6 kb) instead of Oligo(dT). 20 μl RT reactions containing 1 ng or 10 ng of total HeLa RNA were performed at the given temperatures for 50 min (hot start) followed by 70° C. for 15 min. 2 μl of the resulting cDNA were added to 50 μl PCR reactions containing the CBP 1.6 kb primer set.</div>
<div class="description-paragraph" id="p-0458" num="0487">S<smallcaps>UPER</smallcaps>S<smallcaps>CRIPT</smallcaps>™ II RT, EFN (no tag), EFN (His-tag), and LEFN (His-tag) were compared at 200 and 400 units in RT reactions from 45° C. to 60° C. 20 μl RT reactions containing 500 ng of total HeLa RNA were performed at the given temperatures for 50 min (hot start) followed by 70° C. for 15 min. 2 μl of the resulting cDNA were added to 50 μl PCR reactions containing either the BF 2.4 kb or Pol ε 6.8 kb primer set.</div>
<div class="description-paragraph" id="p-0459" num="0488">S<smallcaps>UPER</smallcaps>S<smallcaps>CRIPT</smallcaps>™ II RT, EFN (His-tag), LEFN (His-tag), and LEFN (tag, no His) were compared at 200 and 400 units in RT reactions from 45° C. to 60° C. 20 μl RT reactions containing 500 ng of total HeLa RNA were performed at the given temperatures for 50 min (hot start) followed by 70° C. for 15 min. 2 μl of the resulting cDNA were added to 50 μl PCR reactions containing either the BF 2.4 kb or Pol ε 6.8 kb primer set.</div>
<div class="description-paragraph" id="p-0460" num="0489">RT enzyme concentration: 20 μl RT reactions containing 0.1 pg to 1 μg of starting total HeLa RNA and 25 units to 250 units of EFN (His-tag) were performed at 45° C. for 50 min followed by 85° C. for 5 min. 2 μl of the resulting cDNA were added to 50 μl PCR reactions containing either the β-actin 353 bp, CBP 1.6 kb, TSC 5.3 kb, Pol ε 6.8 kb, or APC 8.5 kb primer set.</div>
<div class="description-paragraph" id="p-0461" num="0490">S<smallcaps>UPER</smallcaps>S<smallcaps>CRIPT</smallcaps>™ II RT and LEFN (His-tag) were compared at 200 units and 400 units in RT reactions from 50° C. to 60° C. 20 μl RT reactions containing 10 ng to 1 μg of total HeLa RNA were performed at the given temperatures for 50 min (hot start) followed by 70° C. for 15 min. 2 μl of the resulting cDNA were added to 50 μl PCR reactions containing either BF 2.4 kb or Pol ε 6.8 kb primer set.</div>
<div class="description-paragraph" id="p-0462" num="0491">S<smallcaps>UPER</smallcaps>S<smallcaps>CRIPT</smallcaps>™ II RT and LEFN (His-tag) were compared at 200 units and 400 units (LEFN was also used at 800 units) in RT reactions (with or without 0.05% Triton X-100) from 50° C. to 60° C. 20 μl RT reactions containing 500 ng of total HeLa RNA were performed at the given temperatures for 50 min (hot start) followed by 70° C. for 15 min. 2 μl of the resulting cDNA were added to 50 μl PCR reactions containing either BF 2.4 kb or Pol ε 6.8 kb primer set. Triton did not improve the yield of the reaction and actually reduced the yield slightly.</div>
<div class="description-paragraph" id="p-0463" num="0492">LEFN (tag, no His) was compared at 50, 200, and 400 units to determine the effect on sensitivity. 20 μl RT reactions containing 0.1 pg to 100 pg of total HeLa RNA were performed at 50° C. under both hot start and cold start conditions followed by 70° C. for 15 min. 2 μl of the resulting cDNA were added to 50 μA PCR reactions containing either the β-actin 353 bp or GAPDH 532 bp primer set.</div>
<div class="description-paragraph" id="p-0464" num="0493">Comparison of 85° C., 5 min and 70° C., 15 min Inactivation Steps. S<smallcaps>UPER</smallcaps>S<smallcaps>CRIPT</smallcaps>™ II RT and EFN (no tag) were compared in RT reactions with either a 70° C., 15 min or an 85°, 5 min inactivation step. 20 μl RT reactions containing 1 pg to 100 ng of starting total HeLa RNA and 200 units of RT were performed at 45° C. for 50 min followed by one of the inactivation steps. 2 μl of the resulting cDNA were added to 50 μl PCR reactions containing either the β-actin 353 bp, CBP 1.6 kb, Pol ε 3.5 kb, TSC 5.3 kb, or Pol ε 6.8 kb primer set.</div>
<div class="description-paragraph" id="p-0465" num="0494">Priming with Oligo(dT)<sub>20</sub>, Random Hexamers, and Gene-Specific Primers (GSP). EFN (no tag) and S<smallcaps>UPER</smallcaps>S<smallcaps>CRIPT</smallcaps>™ II RT were used in RT reactions containing either 2.5 μM oligo(dT)<sub>20</sub>, 50 ng random hexamers, or 0.1 μM GSP. 20 μl RT reactions containing 1 ng to 1 μg of starting total HeLa RNA and 200 units of RT were performed at 45° C. for 50 min followed by 70° C. for 15 min. 2 μl of the resulting cDNA were added to 50 μl PCR reactions containing either the CBP 1.6 kb, TSC 5.3 kb, or APC 8.5 kb primer set (<figref idrefs="DRAWINGS">FIG. 21</figref>).</div>
<div class="description-paragraph" id="p-0466" num="0495">Comparison of reverse transcriptases from various sources. LEFN (tag, no His) was compared to Clontech PowerScript™ RT, Stratagene StrataScript™ RT, Qiagen SensiScript™ RT, Qiagen OmniScript™ RT, and Promega ImProm-II™ RT System. RT reactions containing 0.1 pg to 1 μg of starting total HeLa RNA or 1 μg of rat brain RNA were performed in duplicate with each of the reverse transcriptases using the procedure and conditions recommended by each supplier. 10% of the cDNA reactions were added to 50 μl PCR reactions containing either the β-actin 353 bp, GAPDH 532 bp, CBP 1.6 kb, BF 2.4 kb, VIN 4.6 kb, FIB 9.4 kb, or Dynein 12.3 kb primer set.</div>
<div class="description-paragraph" id="p-0467" num="0496"> <tables id="TABLE-US-00022" num="00022"> <patent-tables colsep="0" frame="none" pgwide="1" rowsep="0" tabstyle="monospace"> <table align="left" class="description-table" cols="1" colsep="0" rowsep="0" width="100%"> <thead> <tr class="description-tr"> <td class="description-td" colspan="1" nameend="1" namest="1" rowsep="1">TABLE 19</td> </tr> <tr class="description-tr"> <td align="center" class="description-td" colspan="1" nameend="1" namest="1" rowsep="1"> </td> </tr> <tr class="description-tr"> <td class="description-td"> Primer list</td> </tr> <tr class="description-tr"> <td align="center" class="description-td" colspan="1" nameend="1" namest="1" rowsep="1"> </td> </tr> </thead> <tbody><tr class="description-tr"> <td class="description-td"> </td> </tr> </tbody></table> <table align="left" class="description-table" cols="1" colsep="0" rowsep="0" width="100%"> <tbody><tr class="description-tr"> <td class="description-td">Human β-actin-353 bp</td> </tr> <tr class="description-tr"> <td class="description-td">sense GCTCGTCGTCGACAACGGCTC (SEQ ID NO: 12)</td> </tr> <tr class="description-tr"> <td class="description-td">antisense CAAACATGATCTGGGTCATCTTCTC (SEQ ID NO: 13)</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">Human GAPDH-532 bp</td> </tr> <tr class="description-tr"> <td class="description-td">sense GTGAAGGTCGGAGTCAACGGATTT (SEQ ID NO: 14)</td> </tr> <tr class="description-tr"> <td class="description-td">antisense CACAGTCTTCTGGGTGGCAGTGAT (SEQ ID NO: 15)</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">Human Cap Binding Protein  (CBP)-1.6 kb</td> </tr> <tr class="description-tr"> <td class="description-td">sense ATGGCGATCGTCGAACCGGA (SEQ ID NO: 16)</td> </tr> <tr class="description-tr"> <td class="description-td">antisense CACTGTCTTAATATGAATGGGACCTACTGAG (SEQ ID NO: 17)</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">Human B-factor properdin (BF)-2.4 kb</td> </tr> <tr class="description-tr"> <td class="description-td">sense GAGCCAAGCAGACAAGCAAAGCAAGC (SEQ ID NO: 18)</td> </tr> <tr class="description-tr"> <td class="description-td">antisense TGTTTTAATTCAATCCCACGCCCCTGT (SEQ ID NO: 19)</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">Human DNA Polymerase ε (Pol ε)-3.5 kb</td> </tr> <tr class="description-tr"> <td class="description-td">sense AAGGCTGGCGGATTACTGCC (SEQ ID NO: 20)</td> </tr> <tr class="description-tr"> <td class="description-td">antisense GATGCTGCTGGTGATGTACTC (SEQ ID NO: 21)</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">Human Vinculin (VIN)-4.6 kb</td> </tr> <tr class="description-tr"> <td class="description-td">sense GAGGAGGGCGAGGTGGACGGC (SEQ ID NO: 22)</td> </tr> <tr class="description-tr"> <td class="description-td">antisense GAACTAACACACAGCGATGGGTGGGAA (SEQ ID NO: 23)</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">Human Tuberous Sclerosis 2 (TSC-2)-5.3 kb</td> </tr> <tr class="description-tr"> <td class="description-td">sense GGAGTTTATCATCACCGCGGAAATACTGAGAG (SEQ ID NO: 24)</td> </tr> <tr class="description-tr"> <td class="description-td">antisense TATTTCACTGACAGGCAATACCGTCCAAGG (SEQ ID NO: 25)</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">Human DNA Polymerase ε (Pol ε)-6.8 kb</td> </tr> <tr class="description-tr"> <td class="description-td">sense CGCCAAATTTCTCCCCTGAA (SEQ ID NO: 26)</td> </tr> <tr class="description-tr"> <td class="description-td">antisense CCGTAGTGCTGGGCAATGTTC (SEQ ID NO: 27)</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">Human Adenomatous Polyposis coli (APC)-8.5 kb</td> </tr> <tr class="description-tr"> <td class="description-td">sense GCTGCAGCTTCATATGATCAGTTGTTA (SEQ ID NO: 28)</td> </tr> <tr class="description-tr"> <td class="description-td">antisense AATGGCGCTTAGGACTTTGG (SEQ ID NO: 29)</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">Human Fibrillin (FIB)-9.4 kb</td> </tr> <tr class="description-tr"> <td class="description-td">sense TGGAGGCTGGGAACGTGAAGGAAA (SEQ ID NO: 30)</td> </tr> <tr class="description-tr"> <td class="description-td">antisense ACAGGAATGACCGAGGGTAATCTTGGC (SEQ ID NO: 31)</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">Rat Dynein-12.3 kb</td> </tr> <tr class="description-tr"> <td class="description-td">sense GCGGCGCTGGAGGAGAA (SEQ ID NO: 32)</td> </tr> <tr class="description-tr"> <td class="description-td">antisense AGGTGGCGGCTCAAACACAAAG (SEQ ID NO: 33)</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">H-fibroblast growth factor 11 (FGF)-240 bp; </td> </tr> <tr class="description-tr"> <td class="description-td">97° C. denaturing, 60° C. annealing temp.</td> </tr> <tr class="description-tr"> <td class="description-td">sense (fl-60)-CGGGTGGTAACTGGCTGCTGTGGA (SEQ ID NO: 34)</td> </tr> <tr class="description-tr"> <td class="description-td">antisense TGGAGGCTGGGAACGTGAAGGAAA (SEQ ID NO: 35)</td> </tr> <tr class="description-tr"> <td class="description-td">antisense (r2-299)-GCGGACCTCCCGCTTCTGCCGGA (SEQ ID NO: 36)</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">H-cystathionine-beta-synthase (CBS 2.4)-2390 bp; </td> </tr> <tr class="description-tr"> <td class="description-td">64° C. annealing temperature</td> </tr> <tr class="description-tr"> <td class="description-td">sense (f2-71)-CCAAGTAAAACAGCATCGGAACACCAGG (SEQ ID NO: 37)</td> </tr> <tr class="description-tr"> <td class="description-td">antisense (r2-2460)-AAAGTCGATCAGCAGTTGCCAGGGG (SEQ ID NO: 38)</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">H-topoisomerase I (TOP3.2)-3162 bp; </td> </tr> <tr class="description-tr"> <td class="description-td">60° C. annealing temperature</td> </tr> <tr class="description-tr"> <td class="description-td">sense (f2-80)-CCCACAGTCACCGCCGCTTACCT (SEQ ID NO: 39)</td> </tr> <tr class="description-tr"> <td class="description-td">antisense (r1-3241)-CTTCATCCCTCCCCAACCCCAATCT (SEQ ID NO: 40)</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">H-vinculin (VIN4.6)-4641 bp; </td> </tr> <tr class="description-tr"> <td class="description-td">66° C. annealing temperature</td> </tr> <tr class="description-tr"> <td class="description-td">sense (f1-132)-GAGGAGGGCGAGGTGGACGGC (SEQ ID NO: 41)</td> </tr> <tr class="description-tr"> <td class="description-td">antisense (r2-4772)-GAACTAACACACAGCGATGGGTGGGAA (SEQ ID NO: 42)</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">H-PolE (PolE 6.8)-6800 bp; </td> </tr> <tr class="description-tr"> <td class="description-td">60° C. annealing temperature</td> </tr> <tr class="description-tr"> <td class="description-td">sense-CGCCAAATTTCTCCCCTGAA (SEQ ID NO: 43)</td> </tr> <tr class="description-tr"> <td class="description-td">antisense-CCGTAGTGCTGGGCAATGTTC (SEQ ID NO: 44)</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">H-β-actin (βact353)-353 bp; </td> </tr> <tr class="description-tr"> <td class="description-td">55° C. annealing temperature</td> </tr> <tr class="description-tr"> <td class="description-td">sense-5′ GCTCGTCGTCGACAACGGCTC (SEQ ID NO: 45)</td> </tr> <tr class="description-tr"> <td class="description-td">antisense-ACCACATGATCTGGGTCATCTTCTC (SEQ ID NO: 46)</td> </tr> <tr class="description-tr"> <td align="center" class="description-td" colspan="1" nameend="1" namest="1" rowsep="1"> </td> </tr> </tbody></table> </patent-tables> </tables> <br/>
Comparison of S<smallcaps>UPER</smallcaps>S<smallcaps>CRIPT</smallcaps>™ III to S<smallcaps>UPER</smallcaps>S<smallcaps>CRIPT</smallcaps>™ II
</div>
<div class="description-paragraph" id="p-0468" num="0497">A comparison of S<smallcaps>UPER</smallcaps>S<smallcaps>CRIPT</smallcaps>™ III to S<smallcaps>UPER</smallcaps>S<smallcaps>CRIPT</smallcaps>™ II was made using the following assay. Total HeLa RNA (1 μg) was combined with 10 pmol gene specific primer (antisense primer, Table 19) and hybridized at 65 C for 5 min. Transcription with 200 U of each enzyme was carried out at 42° C., 50° C., or 55° C. for 50 min. Reverse transcription for comparison of enzyme preparations were performed using 1 pg, 10 pg, 100 ng or 1 μg of total RNA from HeLa cells with 50 μmol OligodT<sub>(20) </sub>at 55° C. 50 min. Reaction components included 0.5 mM dNTPs, 0.01 M DTT, 40 U RNase OUT™, 50 mM Tris-KCl (pH 8.3), 75 mM KCl, 3 mM MgCl<sub>2</sub>. After reverse transcription, the enzyme was heat inactivated by incubation at 70° C. for 15 min. After a 20 min. incubation with 2 U RnaseH at 37° C., 2 μL of the 20 μL reaction was amplified by PCR using Platinum Taq hi-fidelity. The amplification reaction contained 60 mM Tris-SO<sub>4 </sub>(pH 8.9), 18 mM NH<sub>4</sub>SO<sub>4</sub>, 2 mM MgSO<sub>4</sub>, 0.2 mM each dNTP, 0.2 μM each primer (Table 19), and 1 U Platinum Taq high fidelity. The amplification was carried out by an initial 94° C. denaturation step for two min., followed by 35 cycles of the following steps: 94° C. 15 sec., X° C. 30 sec., 68° C. 1 min/kb, where X indicates the annealing temperature of the primer set listed in Table 19 (55° C.-66° C.). High G/C content products were amplified with the following cycling beginning with a 94° C. denaturation step for two min. followed by 35 cycles of the following: 97° C., 15 sec.; 60° C. 30 sec.; 68° C. one min. The reactions were combined with 5 μL 10× BlueJuice® and 20% (11 μL) of each reaction was electrophorsed through 0.8% or 1.5% agarose gels stained with 0.5 μg/ml ethidium bromide.</div>
<div class="description-paragraph" id="h-0048" num="0000">Results</div>
<div class="description-paragraph" id="p-0469" num="0498">RT-PCR Optimization: cDNA synthesis buffer. S<smallcaps>UPER</smallcaps>S<smallcaps>CRIPT</smallcaps>™ III RT was used with the S<smallcaps>UPER</smallcaps>S<smallcaps>CRIPT</smallcaps>™ II RT stand-alone conditions. These conditions were found to be optimal for S<smallcaps>UPER</smallcaps>S<smallcaps>CRIPT</smallcaps>™ III RT in first-strand synthesis. No significant difference in performance in RT-PCR was seen with S<smallcaps>UPER</smallcaps>S<smallcaps>CRIPT</smallcaps>™ III RT using the reaction conditions of S<smallcaps>UPER</smallcaps>S<smallcaps>CRIPT</smallcaps>™ II RT stand-alone, S<smallcaps>UPER</smallcaps>S<smallcaps>CRIPT</smallcaps>™ II First-Strand Synthesis System for RT-PCR, or ThermoScript RT-PCR System with a β-actin 353 bp target or with larger targets.</div>
<div class="description-paragraph" id="p-0470" num="0499">A magnesium concentration of 3 mM was chosen for S<smallcaps>UPER</smallcaps>S<smallcaps>CRIPT</smallcaps>™ III RT as this the minimum concentration needed in the RT reaction and it was the same concentration found with the S<smallcaps>UPER</smallcaps>S<smallcaps>CRIPT</smallcaps>™ II RT stand-alone. With 0.5 mM dNTPs, optimal performance was seen with magnesium concentrations from 3 mM to 7 mM with both low RNA concentrations of 1 pg, 100 ng, and 5 μg.</div>
<div class="description-paragraph" id="p-0471" num="0500">The pH of the 5× First-Strand Buffer will remain at 8.3 as this is the pH found in the S<smallcaps>UPER</smallcaps>S<smallcaps>CRIPT</smallcaps>™ II RT buffer and little difference was seen with different pHs at higher temperatures (<figref idrefs="DRAWINGS">FIG. 14</figref>).</div>
<div class="description-paragraph" id="p-0472" num="0501">RT reaction temperature. 200 units of LEFN(S<smallcaps>UPER</smallcaps>S<smallcaps>CRIPT</smallcaps>™ III RT) showed little reduction in product up to 50° C., which is 5° C. higher than the capabilities of S<smallcaps>UPER</smallcaps>S<smallcaps>CRIPT</smallcaps>™ II RT (50 U), and 5° C. lower than that of ThermoScript RT (15 U). Results were similar with both low levels of RNA and the β-actin 353 bp primer set, and higher levels of RNA and the TSC 5.3 kb primer set.</div>
<div class="description-paragraph" id="p-0473" num="0502">When a His-tag is part of the RT, slightly higher thermal-stability is noted. RT reactions containing 10 to 1000 ng of total HeLa RNA and 200 units of S<smallcaps>UPER</smallcaps>S<smallcaps>CRIPT</smallcaps>™ II or LEFN RT were run at 45° C. to 55° C. for 50 minutes in a hot start reaction. 2 μl of the resulting cDNA products were then added to 50 μl PCR reactions containing the Pol ε 6.8 kb primer set. Resulting PCR products were then run on a 0.8% agarose gel containing 0.4 mg/ml ethidium bromide. Not much difference was seen between 200 units of EFN (His-tag), EFN (no tag), and LEFN (His-tag) with the β-actin 353 bp primer set with all three showing product up to 55° C. However, with the Pol ε 6.8 kb target, product was also seen up to 55° C. with all three mutants, except EFN (no tag) showed less product at this temperature than either of the mutants with His-tags.</div>
<div class="description-paragraph" id="p-0474" num="0503">The same temperature profile as found with oligo(dT)<sub>20 </sub>and the advantage of the His-tag was also seen with a gene-specific primer. RT reactions containing 1 to 10 ng of total HeLa RNA and 200 units of S<smallcaps>UPER</smallcaps>S<smallcaps>CRIPT</smallcaps>™ II, EFN, or LEFN RT were run at 45° C. to 65° C. for 50 minutes in a hot start reaction. 2 μl of the resulting cDNA products were then added to 50 μl PCR reactions containing the CBP 1.6 kb primer set. Resulting PCR products were then run on a 0.8% agarose gel containing 0.4 mg/ml ethidium bromide. Using a gene-specific primer in the RT reaction, 200 units of EFN (His-tag), EFN (no tag), and LEFN (His-tag) all still showed CBP 1.6 kb product up to 55° C., but EFN (no tag) showed slightly less than the others at 55° C.</div>
<div class="description-paragraph" id="p-0475" num="0504">In order to exploit the stability provided by the His-tag, LEFN with a tag minus the 6×His was designed. RT reactions containing 500 ng of total HeLa RNA and 200 or 400 units of S<smallcaps>UPER</smallcaps>S<smallcaps>CRIPT</smallcaps>™ II, EFN, or LEFN RT were run at 45° C. to 65° C. for 50 minutes in a hot start reaction. 2 μl of the resulting cDNA products were then added to 50 μl PCR reactions containing the BF 2.4 kb primer set. Resulting PCR products were then run on a 0.8% agarose gel containing 0.4 mg/ml ethidium bromide. Little difference was seen with the mutant when compared to both EFN (His-tag) and LEFN (His-tag), product was seen up to 60° C. with all three with the BF 2.4 kb primer set (<figref idrefs="DRAWINGS">FIG. 15</figref>).</div>
<div class="description-paragraph" id="p-0476" num="0505">RT enzyme concentration. An experiment comparing units of EFN (His-tag) from 25 units to 250 units showed little difference in performance over the entire range, when looking at both sensitivity with the β-actin 353 primer set or with longer targets. RT reactions containing 0.1 to 1000 ng of total HeLa RNA and 25 to 250 units of EFN were run at 45° C. for 50 minutes in a hot start reaction. 2 μl of the resulting cDNA products were then added to 50 μl PCR reactions containing the β-actin 353 bp, CBP 1.6 kb, TSC 5.3 kb, Pol ε 6.8 kb, or APC 8.5 kb primer set. Resulting PCR products were then run on a 0.8% agarose gel containing 0.4 mg/ml ethidium bromide.</div>
<div class="description-paragraph" id="p-0477" num="0506">200 and 400 units of LEFN (tag, no His) showed slightly better performance than 50 units with low concentrations of RNA. RT reactions containing 0.1 to 100 pg of total HeLa RNA and 50 to 400 units of LEFN RT were run at 50° C. for 50 minutes in a hot start reaction. 2 μl of the resulting cDNA products were then added to 50 μl PCR reactions containing the β-actin 353 bp primer (not shown) or the GAPDH 532 primer set. Resulting PCR products were then run on a 1.5% agarose gel containing 0.4 mg/ml ethidium bromide. The higher RT units yielded slightly more PCR product under hot start conditions with both β-actin 353 bp (data not shown) and GAPDH 532 bp (<figref idrefs="DRAWINGS">FIG. 16</figref>).</div>
<div class="description-paragraph" id="p-0478" num="0507">Priming with Oligo(dT)<sub>20</sub>, Random Hexamers, and Gene-Specific Primers (GSP). S<smallcaps>UPER</smallcaps>S<smallcaps>CRIPT</smallcaps>™ III RT performed similarly to S<smallcaps>UPER</smallcaps>S<smallcaps>CRIPT</smallcaps>™ II RT when RT reactions were run with different priming methods. Oligo(dT) yielded the cleanest product and highest specific yield with GSP having high levels on non-specific products (at 55 degrees RT temperature) and random hexamers having high specificity but low yield.</div>
<div class="description-paragraph" id="p-0479" num="0508">Comparison of reverse transcriptases from various sources. Using β-actin 353 bp primer set, S<smallcaps>UPER</smallcaps>S<smallcaps>CRIPT</smallcaps>™ III RT was again able to detect down to 1 pg of starting HeLa RNA. RT reactions containing 1 to 100 pg of total HeLa RNA were run with each RT using the reagents and conditions specified in each protocol. 2 μl of the resulting cDNA products were then added to 50 μl PCR reactions containing the β-actin 353 bp primer set. Resulting PCR products were then run on a 1.5% agarose gel containing 0.4 mg/ml ethidium bromide. ImProm II RT and SensiScript were also able to detect down to 1 pg, but StrataScript and OmniScript could only detect down to 10 pg, and PowerScript was unable to even detect 100 pg of starting HeLa RNA.</div>
<div class="description-paragraph" id="p-0480" num="0509">With larger targets and higher concentrations of total HeLa RNA, S<smallcaps>UPER</smallcaps>S<smallcaps>CRIPT</smallcaps>™ III RT (LEFN tag, no His) performed significantly better than most of the competitors in relation to the RT-PCR product yield and length. RT reactions containing 10 to 1000 ng of total HeLa RNA were run with each RT using the reagents and conditions specified in each protocol. 2 μl of the resulting cDNA products were then added to 50 μl PCR reactions containing the CBP 1.6 kb, BF 2.4 kb, VIN 4.6 kb or 9.4 kb primer set. Resulting PCR products were then run on a 0.8% agarose gel containing 0.4 mg/ml ethidium bromide. CLONTECH PowerScript only showed product when the total HeLa RNA was 1000 ng, and the yield of specific products was not as high as with S<smallcaps>UPER</smallcaps>S<smallcaps>CRIPT</smallcaps>™ III RT. Qiagen SensiScript and OmniScript RT showed sufficient product yield with the smaller targets (CBP 1.6 kb and BF 2.4 kb), but were not able to detect the longer targets. Stratagene StrataScript was able to detect all the targets but with lower yields than S<smallcaps>UPER</smallcaps>S<smallcaps>CRIPT</smallcaps>™ III RT. Promega ImProm-II RT System showed performance similar to S<smallcaps>UPER</smallcaps>S<smallcaps>CRIPT</smallcaps>™ III RT, but did not have as high a yield with the longest target (FIB 9.4 kb).</div>
<div class="description-paragraph" id="p-0481" num="0510">Additional reactions with lower starting HeLa RNA concentrations showed a similar pattern. RT reactions containing 10 to 100 ng of total HeLa RNA were run with each RT using the reagents and conditions specified in each protocol. 2 μl of the resulting cDNA products were then added to 50 μl PCR reactions containing the CBP 1.6 kb, BF 2.4 kb, VIN 4.6 kb or 9.4 kb primer set. Resulting PCR products were then run on a 0.8% agarose gel containing 0.4 mg/ml ethidium bromide. CLONTECH PowerScript was unable to detect any of the targets with 100 ng or less of starting total HeLa RNA. Qiagen SensiScript was able to detect the VIN 4.6 kb target with this lower concentration of RNA when it had not been able to detect this target previously with 1 μg of starting total HeLa RNA. Promega ImProm-II did not perform as well as S<smallcaps>UPER</smallcaps>S<smallcaps>CRIPT</smallcaps>™ III RT with lower concentrations of RNA. S<smallcaps>UPER</smallcaps>S<smallcaps>CRIPT</smallcaps>™ III RT was the only RT that was able to detect the FIB 9.4 kb target with 100 ng starting total HeLa RNA.</div>
<div class="description-paragraph" id="p-0482" num="0511">RT-PCR analysis. S<smallcaps>UPER</smallcaps>S<smallcaps>CRIPT</smallcaps>™ II and S<smallcaps>UPER</smallcaps>S<smallcaps>CRIPT</smallcaps>™ III performance in hot start RT-PCR amplification at 42° C., 50° C., or 55° C. were compared and the results are shown in <figref idrefs="DRAWINGS">FIG. 17</figref> S<smallcaps>UPER</smallcaps>S<smallcaps>CRIPT</smallcaps>™ II (Panel A) or S<smallcaps>UPER</smallcaps>S<smallcaps>CRIPT</smallcaps>™ III (Panel B). Lanes (in duplicate) 1, 4, 7, and 10 are products reverse transcribed at 42° C. Lanes 2, 5, 8, and 11 are products reverse transcribed at 50° C. Lanes 3, 6, 9, and 12 are products transcribed at 55° C. Lanes 1-3 are the result of RNAs reverse transcribed by gene-specific priming from FGF, lanes 4-6 CBS 2.4, lanes 7-9 from TOP 3.2, lanes 10-12 VIN 4.6. Arrows indicate expected product sizes of 240 bp, 2390 bp, 3162 bp, and 4641 bp. S<smallcaps>UPER</smallcaps>S<smallcaps>CRIPT</smallcaps>™ II transcribed more robustly at 50° C. whereas S<smallcaps>UPER</smallcaps>S<smallcaps>CRIPT</smallcaps>™ III best transcribes at 55° C. Amplification at 55° C. for some applications that require higher annealing temperatures for gene specific priming and/or to remove secondary structure from an RNA template may be performed with the polypeptides of the invention. This will allow the reduction of non-specific priming during reverse transcription with gene specific primers and/or the increased reverse transcription of refractory templates.</div>
<div class="description-paragraph" id="p-0483" num="0512">S<smallcaps>UPER</smallcaps>S<smallcaps>CRIPT</smallcaps>™ III RT: Two different purification techniques. S<smallcaps>UPER</smallcaps>S<smallcaps>CRIPT</smallcaps>™ III has been purified by two different methods (see above). In brief, Method A is similar to the purification technique used for S<smallcaps>UPER</smallcaps>S<smallcaps>CRIPT</smallcaps>™ II, Method B differs from method A by the use of a membrane permeabilization protocol and filtration protocol to reduce cellular debris which results in a high-purity preparation. RT-PCR was performed using 200 U of S<smallcaps>UPER</smallcaps>S<smallcaps>CRIPT</smallcaps>™ III purified by Method A or Method B. RT-PCR was performed using the Pol ε 6.8 kb primers and ▪-act 353 bp primers. Product yield and quality were compared for both purification methods. The methods yield similar results and either are viable methods for purification of S<smallcaps>UPER</smallcaps>S<smallcaps>CRIPT</smallcaps>™ III.</div>
<div class="description-paragraph" id="p-0484" num="0513">Optimum S<smallcaps>UPER</smallcaps>S<smallcaps>CRIPT</smallcaps>™ III RT enzyme concentration. RT reactions containing 0.1 pg to 1000 ng of total HeLa RNA and 25 to 250 units of S<smallcaps>UPER</smallcaps>S<smallcaps>CRIPT</smallcaps>™ II, EFN, or LEFN RT were run at 45° C. for 50 min (hot start). 2 μl of the resulting cDNA were then added to 50 μl PCR reactions containing the β-actin 353 bp, CBP 1.6 kb, TSC 5.3 kb, Pol ε 6.8 kb, or APC 8.5 kb primer set. Resulting PCR products were then run on a 0.8% or 1.5% agarose gel containing 0.4 μg/ml ethidium bromide. No significant differences was obtained from 25 units to 250 units of S<smallcaps>UPER</smallcaps>S<smallcaps>CRIPT</smallcaps>™ III RT. (<figref idrefs="DRAWINGS">FIG. 18</figref>). However, higher amount of S<smallcaps>UPER</smallcaps>S<smallcaps>CRIPT</smallcaps>™ III RT (400 units) shows higher RT-PCR product yield at an elevated RT temperature (55 degrees, <figref idrefs="DRAWINGS">FIG. 19</figref>). In <figref idrefs="DRAWINGS">FIG. 19</figref>, RT reactions containing 10 to 1000 ng of total HeLa RNA and 200 or 400 units of S<smallcaps>UPER</smallcaps>S<smallcaps>CRIPT</smallcaps>™ II or LEFN RT were run at 50° C. to 60° C. for 50 min (hot start). 2 μl of the resulting cDNA were then added to 50 μl PCR reactions containing the Pol ε 6.8 kb primer set. Resulting PCR products were then run on a 0.8% agarose gel containing 0.4 μg/ml ethidium bromide. Unlike S<smallcaps>UPER</smallcaps>S<smallcaps>CRIPT</smallcaps>™ II RT, increased amount of S<smallcaps>UPER</smallcaps>S<smallcaps>CRIPT</smallcaps>™ III RT (up to 400 units) did not inhibit subsequent PCR amplification (<figref idrefs="DRAWINGS">FIG. 20</figref>). In <figref idrefs="DRAWINGS">FIG. 20</figref>, RT reactions containing 0.1 to 100 pg of total HeLa RNA and 50 to 800 units of LEFN RT were run at 50° C. for 50 min in hot start conditions (Oligo(dT)<sub>20</sub>). 2 μl of the resulting cDNA were then added to 50 μl PCR reactions containing the GADPH 532 bp primer set. Resulting PCR products were then run on a 0.8% agarose gel containing 0.4 μg/ml ethidium bromide.</div>
<div class="description-paragraph" id="p-0485" num="0514">S<smallcaps>UPER</smallcaps>S<smallcaps>CRIPT</smallcaps>™ III RT has been engineered to increase the temperature at which the enzyme can perform RT activity. The enzyme has been optimized for RT-PCR for sensitivity, yield, and target length. With this optimized system, it was demonstrated that S<smallcaps>UPER</smallcaps>S<smallcaps>CRIPT</smallcaps>™ III RT was able to detect mRNA targets with as little as 1 pg of starting total HeLa RNA and to produce high yield cDNAs from targets up to 12.3 kb in size. It has the ability to function at elevated temperatures up to 55° C. and to detect a wide variety of targets. The performance of this enzyme is superior to any other RTs commercially available, and the optimized buffers and RT protocol provide a sensitive and robust RNA detection system.</div>
<heading id="h-0049">Example 9</heading>
<heading id="h-0050">Use of Polypeptides of the Invention to Prepare Labeled Nucleic Acid Molecules</heading>
<div class="description-paragraph" id="p-0486" num="0515">Polypeptides of the invention can be used to prepare labeled nucleic acids from a variety of templates (e.g., total RNA, mRNA, etc.). For direct labeling using a polyA tailed RNA template, suitable reaction conditions may entail the use of from about 1 μg to about 1000 μg, from about 1 μg to about 500 μg, from about 1 μg to about 250 μg, from about 1 μg to about 100 μg, from about 10 μg to about 1000 μg, from about 10 μg to about 500 μg, from about 10 μg to about 250 μg, or from about 10 μg to about 100 μg of RNA.</div>
<div class="description-paragraph" id="p-0487" num="0516">RNA can be mixed with a suitable primer (e.g., an oligo dT primer or a gene specific primer). After mixing, the template primer mixture can be incubated at a suitable temperature (e.g., 70° C. for an oligo(dT)<sub>25 </sub>primer) and incubated for a suitable period of time (e.g., 5 minutes). Those skilled in the art can readily determine incubation times and temperatures for any template primer pair using routine experimentation. The mixture may then be chilled on ice and the remaining reaction components added.</div>
<div class="description-paragraph" id="p-0488" num="0517">Suitable reaction components, which may be provided in a reaction mixture and/or solution either individually or in combination, include a buffering agent, reducing agent, one or more nucleotides, at least one of which may contain a label (e.g., a fluorescent label), one or more polypeptide of the invention, and suitable diluent (e.g., H<sub>2</sub>O). A suitable reaction mixture can be prepared by combining 4 μl 5× first strand buffer, 2 μl 0.1M DTT, 1 μl 10 mM dNTP, 2 μl 1 mM Fluorescent dNTP, 2 μl S<smallcaps>UPER</smallcaps>S<smallcaps>CRIPT</smallcaps>™ III (200 u/μl), an aliquot containing RNA, and dH<sub>2</sub>O to 20 μl. Additional reaction mixtures and/or solutions, as well as components thereof, which may be used with this aspect of the invention are described elsewhere herein. The mixture may be incubated at a suitable temperature, for example, from about 42° C. to about 60° C., from about 45° C. to about 60° C., from about 48° C. to about 60° C., from about 50° C. to about 60° C., from about 52° C. to about 60° C., from about 55° C. to about 60° C., from about 42° C. to about 55° C., from about 45° C. to about 55° C., from about 45° C. to about 50° C., from about 45° C. to about 48° C., from about 48° C. to about 60° C., from about 48° C. to about 55° C., from about 48° C. to about 52° C., from about 50° C. to about 60° C., from about 50° C. to about 55° C., or from about 50° C. to about 52° C. The mixture may be incubated until a sufficient incorporation of label is seen, for example, from about 5 minutes to about 24 hours, from about 10 minutes to about 24 hours, from about 30 minutes to about 24 hours, from about 1 hour to about 24 hours, from about 2 hours to about 24 hours, from about 4 hours to about 24 hours, from about 8 hours to about 24 hours, from about 30 minutes to about 16 hours, from about 30 minutes to about 8 hours, from about 30 minutes to about 4 hours, from about 30 minutes to about 2 hours, from about 30 minutes to about 1 hour, from about 1 hour to about 4 hours, or from about 1 hour to about 2 hours.</div>
<div class="description-paragraph" id="p-0489" num="0518">The reaction may be stopped, for example, by addition of a suitable stopping reagent (e.g., 5 μl of 0.5M EDTA in a 20 μl reaction). The resultant labeled nucleic acids may be purified using any standard technique (e.g., column purification using a SNAP column, Invitrogen Corporation, Carlsbad, Calif.).</div>
<div class="description-paragraph" id="p-0490" num="0519">Labeled nucleic acid produced by the methods of the invention may be used for a variety of purposes, for example, to detect a target sequence, which may be present on an array. Typically, labeled nucleic acid of the invention may be hybridized to one or more target sequences (e.g., a microarray).</div>
<div class="description-paragraph" id="p-0491" num="0520">In some embodiments, polypeptides of the invention may be used to prepare labeled nucleic acids by indirect labeling. For example, a modified nucleotide (e.g., an amino allyl nucleotide) may be incorporated into a nucleic acid molecule using the polypeptides of the invention. The nucleic acid molecules containing the modified nucleotide may then be reacted with a reactive molecule that comprises a detectable moiety (e.g., a fluorescent moiety, radiolabel and the like). All or a portion of the reactive molecule and the detectable moiety may then become attached to the nucleic acid molecule.</div>
<div class="description-paragraph" id="p-0492" num="0521">Suitable reaction conditions for indirect labeling include those listed above. For example, RNA (e.g., 5-50 μg) can be mixed with a suitable primer (e.g., an oligo(dT)<sub>25 </sub>for polyA tailed RNA), mixed and incubated, for example, at 70° C. for 5 min, and then chilled on ice. To the primer: template mixture, addition reaction components may be added. For example, for a 30 μl reaction volume, suitable component may include: 6 μl 5× first strand buffer, 1.5 μl 0.1M DTT, 1 μl RNaseOUT (40u/μl), 1.5 μl 10 mM dNTP mixture containing 10 mM dATP, 10 mM dCTP, 10 mM dGTP, 4 mM dTTP, and 6 mM Aminoallyl-dUTP), 2 μl S<smallcaps>UPER</smallcaps>S<smallcaps>CRIPT</smallcaps>™ III (200 u/μl), X μl dH<sub>2</sub>O to a total volume of 30 μl. The mixture may be incubated at a suitable temperature for a suitable time such as those described above, for example, at 50° C. for 2 hours. The mixture may be treated to degrade the RNA template (e.g., by the addition of 15 μl 1N NaOH, and heating to 70° C. for 10 min). The pH of the solution may then be adjusted (e.g., by the addition of 15 μl 1N HCl) and the nucleic acid containing the modified nucleotide may be purified by standard techniques (e.g., using a SNAP column, Invitrogen Corporation, Carlsbad, Calif.). The purified nucleic acid may be concentrated (e.g., by ethanol precipitation).</div>
<div class="description-paragraph" id="p-0493" num="0522">After ethanol precipitation, the nucleic acid containing the modified nucleotide may be resuspended in a buffer suitable for coupling the reactive molecule containing the detectable moiety to the nucleic acid. For example, the nucleic acid may be resuspended in 5 μl coupling buffer (e.g., 0.1 M sodium borate, pH 8.5). The reactive molecule (e.g., a molecule comprising a dye and a reactive functionality) may be added (e.g., 5 μl monofunctional Cy3 or Cy5 dye from APB Cat#PA23001 and # PA25001). When the reactive molecule is a dye, the dye may be suspended in any suitable solvent (e.g., DMSO). In one embodiment, 1 pack of CyDye was resuspended in 45 μl DMSO and used as above. The reaction may be incubated for a suitable length of time and at a suitable temperature, for example, at room temperature for 1 hour. When the dye is light sensitive, the incubation is preferably performed in the dark. The coupling reaction is stopped by the addition of a reagent such as 5 μl of 4M hydroxylamine. The labeled nucleic acid is then purified (e.g., using a SNAP column, Invitrogen Corporation, Carlsbad, Calif.).</div>
<div class="description-paragraph" id="p-0494" num="0523">Labeled nucleic acids (whether prepared by direct or indirect labeling) may be hybridized to one or more target sequences, which may be in any suitable form, for example, in a microarray. The labeled nucleic acids are denatured (e.g., by heating at 95° C. for 2 min). The denatured, labeled nucleic acids are brought into contact with the target sequences in a suitable buffer. For example, a microarray slide can be placed in a hybridization buffer (e.g., 25% Formamide, 5×SSC, 50 mM MES pH 6.5, 0.1% SDS) that contains labeled nucleic acid (e.g., ˜1 μg of Dye-label probe). The microarray can be incubated for a suitable time at a suitable temperature, for example, at 42° C. for 16 hours. The microarray may then be washed one or more times, for example, the hybridization solution may be removed and the microarray washed in Wash I (5×SSC, 0.2% SDS), followed by a wash in Wash II (1×SSC, 0.2% SDS) at 42° C. for 5 min, followed by a wash in Wash III (0.1×SSC) for 2 min. The microarray may be dried, for example, by spin drying the microarray at 600 rpm for 5 min.</div>
<div class="description-paragraph" id="p-0495" num="0524">The hybridized microarray may be analyzed using any techniques known in the art.</div>
<heading id="h-0051">Example 10</heading>
<heading id="h-0052">Thermostability Mutants</heading>
<div class="description-paragraph" id="p-0496" num="0525">In an effort to raise the temperature at which an M-MLV RT can be used to synthesize cDNA, random mutagenesis of two regions of the M-MLV sequence was performed. These regions are defined by NdeI-AatII and AatII-SalI restriction enzyme sites which cover the M-MLV sequence from the starting methionine (position 33 of SEQ ID NO:2) to position F369. Several mutations were sequenced and identified from these studies.</div>
<div class="description-paragraph" id="p-0497" num="0526">In addition, site directed mutagenesis was used to modify amino acid positions within the M-MLV sequence which were identified by structural analysis and potential interactions with the template. For example P196 is on the same helix and facing the same direction as other mutations found to be useful for increasing thermostability (H204R (SUPERSCRIPT™ III U.S. Pat. No. 7,078,308), D200N (WO 2009/125006 A2)). Of interest, the P127 position is on a strand adjacent to this cluster of amino acids. L435 has been identified as an area that could increase solubility and stability of the M-MLV RT (Das et al. Protein Sci. 10:1936-1941, 2001, and U.S. Patent Application Publication No. 2008/0227661). It is known from SUPERSCRIPT™ II that the Rnase H− mutations increased the thermostability of the enzyme. Other Rnase H− mutations may also increase thermostability compared to wild type M-MLV RT. D653 has been identified as a location that affects RNase H activity (U.S. Pat. No. 7,595,179 and WO 2009/125006 A2). By sequence alignment with thermostable and <i>E. coli </i>Rnase H, a strongly conserved cluster of tryptophans corresponding to MMLV positions H594, Y598, L603 and L618 is observed.</div>
<div class="description-paragraph" id="h-0053" num="0000">Methods</div>
<div class="description-paragraph" id="p-0498" num="0527">Random Mutagenesis: Mutagenic PCR was performed in a reaction mixture (100 μl) containing 20 mM Tris-HCl (pH 8.4), 50 mM KCl, 1.8 mM MgCl<sub>2</sub>, 0.3 mM MnSO<sub>4</sub>, 200 μM each of dCTP, dGTP, dTTP, and dATP, and 0.5 units of Taq DNA polymerase. The cycling protocol was 15 sec at 94° C., 15 sec at 55° C., and 30 sec at 72° C. for 20 cycles. PCR primers included appropriate restriction endonuclease cut sites. An amplified DNA segment was cleaved with appropriate restriction endonucleases, gel purified, and cloned into gel purified vector DNA cut with the corresponding restriction enzymes.</div>
<div class="description-paragraph" id="p-0499" num="0528">Site-directed Mutagenesis: Oligonucleotide site-directed mutagenesis was carried out by established procedures (Kunkel et al., 1987; Kunkel et al., 1991). Saturation of an amino acid coding site in the H− RT gene with all possible amino acids was performed by introducing the sequence NNK (N=A, C, G or T and K=G or T) at the codon site into the mutagenic oligonucleotide.</div>
<div class="description-paragraph" id="p-0500" num="0529">DNA Polymerase Assays: During screening of RT mutants, RT RNA-directed DNA polymerase activity was assayed with (rC)n●(dG)<sub>15</sub>, which is specific for RT (Gerard et al., 1974). Reaction mixtures (50 μl) containing 50 mM Tris-HCl (pH 8.4), 50 mM KCl, 10 mM MgCl<sub>2</sub>, 300 μM (rC)n, 120 μM (dG)<sub>15</sub>, 0.01% (v/v) Triton X-100, and 100 μM [α-<sup>32</sup>P]dGTP (1,000 cpm/pmole), were incubated at 37° C. for 10 min in the wells of a 96-well plate. An aliquot (5 μl) from each well was spotted onto a Genescreen+ (NEN) filter and the filter was washed twice for 10 min with 4% (w/v) sodium pyrophosphate (pH 8.0). Radioactivity bound to the dried filter was quantified in a phosphorimager (Molecular Dynamics).</div>
<div class="description-paragraph" id="p-0501" num="0530">RT DNA polymerase unit activity was assayed with (rA)<sub>630</sub>● p(dT)<sub>12-18 </sub>(Houts et al., 1974). One unit of DNA polymerase activity is the amount of RT that incorporates one nmole of deoxynucleoside triphosphate into acid insoluble product at 37° C. in 10 min. RT DNA polymerase activity was also measured in the presence of (rC)n●(dG)<sub>15 </sub>at various temperatures from 37° C., 55° C. and 60° C. for 10 min. The RT DNA polymerase activity was also measured in the presence or absence of (rC)n●(dG)<sub>15 </sub>at 37° C. for 10 min after a preincubation of the RT at various temperatures from 55° C. to 60° C. for 5′ min. This was done to distinguish intrinsic thermostability from thermostability in the presence of the stabilizing template-primer.</div>
<div class="description-paragraph" id="p-0502" num="0531">Mutant Screening: Mutant populations were plated on selective agar, and individual colonies were inoculated into 120 μl of selective EG in 96 well plates and grown overnight at 37° C. The cell density of the cultures was ˜109 cfu/ml. The cultures were then replicated into plates containing 120 μl per well of selective media +0.2% arabinose and grown for 20 hr at 37° C. An aliquot (5 μl) of the cultures was mixed with 5 μl of 2×PLD (50 mM Tris-HCl, pH 8.0, 20 mM EDTA, 1.8% (w/v) sucrose, 1% (v/v) Triton X-100, 10 mM DTT, and 2 mg/ml lysozyme). After heating for 5 min at various temperatures and cooling to room temperature, RT activity in the 10-μl lysate was assayed with (rC)n●(dG)<sub>15</sub>. The amount of RT in a 5-μl aliquot of extract was within the linear range of the assay. Lysates were stable at room temperature for at least 1 hr.</div>
<div class="description-paragraph" id="p-0503" num="0532">Thermal Inactivation Profile of RT in Extracts: Cell lysates prepared in PLD as described above were heated in a 96-well plate in a thermocycler in which a temperature gradient exists through the rows, but the temperature in each column is the same. After incubation for 5 min at temperatures ranging from 50° C. to 61° C., RT activity was assayed with (rC)<sub>n</sub>●(dG)<sub>15 </sub>as described above. The results of this analysis are shown in <figref idrefs="DRAWINGS">FIG. 22</figref>. The intensity and number of spots in a row as shown in <figref idrefs="DRAWINGS">FIG. 22</figref> correspond to the RT activity and provide a thermal inactivation profile of the RT. Mutations H204R, P127H/T, T330P, G138R, L333P and F309S show a thermal inactivation profile indicating increased thermostability compared to SUPERSCRIPT™ II controls.</div>
<div class="description-paragraph" id="p-0504" num="0533">Additional mutations have been screened by comparing polymerase activity at 37° C. and 60° C. Mutations were prepared by site directed mutagenesis and RT DNA polymerase activity measured in the presence of (rC)n.(dG)<sub>15 </sub>at various temperatures from 37° C., to 60° C. for 10 minutes or after preincubation at temperatures from 37° C., to 60° C. for 5 minutes. The results of this analysis are shown in Table 20.</div>
<div class="description-paragraph" id="p-0505" num="0534"> <tables id="TABLE-US-00023" num="00023"> <patent-tables colsep="0" frame="none" rowsep="0"> <table align="left" class="description-table" cols="4" colsep="0" rowsep="0" width="100%"> <thead> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td" colspan="4" nameend="3" namest="offset" rowsep="1">TABLE 20</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td align="center" class="description-td" colspan="4" nameend="3" namest="offset" rowsep="1"> </td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td">Ratio 60° C. vs. 37° C.</td> <td class="description-td">Ratio 60° C. Heatkill to</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">Mutants</td> <td class="description-td">Activity</td> <td class="description-td">37° C. Control</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td align="center" class="description-td" colspan="4" nameend="3" namest="offset" rowsep="1"> </td> </tr> </thead> <tbody><tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">SSIII</td> <td class="description-td">0.98</td> <td class="description-td">0.40</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">P196S</td> <td class="description-td">2.13</td> <td class="description-td">0.59</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">V223Y</td> <td class="description-td">1.32</td> <td class="description-td">0.47</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">L435G</td> <td class="description-td">1.56</td> <td class="description-td">0.56</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">D653H</td> <td class="description-td">4.89</td> <td class="description-td">0.48</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">D653N</td> <td class="description-td">2.65</td> <td class="description-td">0.41</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">S67K</td> <td class="description-td">1.32</td> <td class="description-td">0.53</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">Y598W</td> <td class="description-td">1.77</td> <td class="description-td">0.34</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td align="center" class="description-td" colspan="4" nameend="3" namest="offset" rowsep="1"> </td> </tr> </tbody></table> </patent-tables> </tables> <br/>
All of the mutants listed in Table 20 retained RT DNA polymerase activity after exposure to a temperature of 60° C. for 5 minutes or when the polymerase activity was measured at a temperature of 60° C. indicating the thermostability of these mutations.
</div>
<heading id="h-0054">Example 11</heading>
<heading id="h-0055">AT Template Screen</heading>
<div class="description-paragraph" id="p-0506" num="0535">In order to identify M-MLV RT mutants which may have improved processivity through long stretches of adenosine residues or may have improved template binding, mutants of SUPERSCRIPT™ II were screened for their ability to read a template with a stretch of 9 adenosines.</div>
<div class="description-paragraph" id="p-0507" num="0536">Oligos with the following sequences were used as primers for a reverse transcription reaction: stop-T-A (5′AGGGGGGAAAAGAAAAAAAAAGA 3′) (SEQ ID NO:47) and stop-primer (5′TCCCCCCTTTTCTTTT 3′) (SEQ ID NO:48). The stop-primer was labeled. The oligos were hybridized at a 10:1 excess of stop-T-A primer to labeled stop-primer and used as the template-primer for a standard RT reaction where 250 nM of labeled oligo and 20 to 200 units of purified RT was added and incubated for 10 to 30 minutes at 37° C. The reaction was stopped and was then visualized on a TBE-Urea sequencing gel and imaged with a Typhoon phosphoimager. The results of this analysis are shown in <figref idrefs="DRAWINGS">FIG. 23</figref>. Mutations Y133T, Q190M, P196S, D200E, V223C/F/W/S and dual mutations Y222F/V223Y and Y222F/V223F were able to successfully read through the extended AT sequence in the template. As evidenced by their ability to read through a template with a stretch of 9 adenosines, these mutations may enable tighter binding to the poly(dA) template and may have tighter binding in general.</div>
<div class="description-paragraph" id="p-0508" num="0537">All publications, patents and patent applications mentioned in this specification are indicative of the level of skill of those skilled in the art to which this invention pertains, and are herein incorporated by reference to the same extent as if each individual publication, patent or patent application was specifically and individually indicated to be incorporate by reference.</div>
</div>
</div>
</section><section itemprop="claims" itemscope="">
<h2>Claims (<span itemprop="count">4</span>)</h2>
<div html="" itemprop="content"><div class="claims" lang="EN" load-source="patent-office" mxw-id="PCLM114400487">
<claim-statement>What is claimed is:</claim-statement>
<div class="claim"> <div class="claim" id="CLM-00001" num="00001">
<div class="claim-text">1. A mutant M-MLV reverse transcriptase having at least 90% sequence identity to SEQ ID NO:2 and having a first amino acid substitution at position T330 and at least one or more additional amino acid substitutions in the amino acid sequence corresponding to positions 34-716of SEQ ID NO:2, wherein amino acid number 1 of said mutant M-MLV reverse transcriptase corresponds to the threonine at position 34 of SEQ ID NO:2 and the at least one or more additional amino acid substitutions are selected from the group consisting of P127, and G138;
<div class="claim-text">wherein said mutant M-MLV reverse transcriptase shows increased thermostability compared to a M-MLV reverse transcriptase comprising amino acids 34-716 of SEQ ID NO:2 having no amino acid substitutions.</div>
</div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="CLM-00002" num="00002">
<div class="claim-text">2. The mutant M-MLV reverse transcriptase of <claim-ref idref="CLM-00001">claim 1</claim-ref>, wherein said position corresponding to amino acid T330 of SEQ ID NO:2 is substituted with P.</div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="CLM-00003" num="00003">
<div class="claim-text">3. The mutant M-MLV reverse transcriptase of <claim-ref idref="CLM-00001">claim 1</claim-ref>, wherein said position corresponding to amino acid P127 of SEQ ID NO:2 is substituted with T or H.</div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="CLM-00004" num="00004">
<div class="claim-text">4. The mutant M-MLV reverse transcriptase of <claim-ref idref="CLM-00001">claim 1</claim-ref>, wherein said position corresponding to amino acid G138 of SEQ ID NO:2 is substituted with R.</div>
</div>
</div> </div>
</div>
</section>
                </article>
            </search-app>
        </body>
    </html>
    