
    <html>
        <body>
            <search-app>
                <article class="result" itemscope="" itemtype="http://schema.org/ScholarlyArticle">
    <h1 itemprop="pageTitle">US20020037850A1 - Novel polypeptides and nucleic acids encoding same 
        - Google Patents</h1><section itemprop="abstract" itemscope="">
<h2>Abstract</h2>
<div html="" itemprop="content"><abstract lang="EN" load-source="patent-office" mxw-id="PA48314689">
<div class="abstract" id="A-0001">The present invention provides novel isolated NOVX polynucleotides and polypeptides encoded by the NOVX polynucleotides. Also provided are the antibodies that immunospecifically bind to a NOVX polypeptide or any derivative, variant, mutant or fragment of the NOVX polypeptide, polynucleotide, or antibody. The invention additionally provides methods in which the NOVX polypeptide, polynucleotide, and antibody are utilized in the detection and treatment of a broad range of pathological states, as well as to other uses. </div>
</abstract>
</div>
</section><section itemprop="description" itemscope="">
<h2>Description</h2>
<div html="" itemprop="content"><ul class="description" lang="EN" load-source="patent-office" mxw-id="PDES68339899">
<heading>RELATED APPLICATIONS </heading>
<li> <para-num num="[0001]"> </para-num> <div class="description-line" id="P-0001" num="0001">This application claims priority to U.S. Ser. No. 60/182,637, filed Feb. 15, 2000; U.S. Ser No. 60/237,862, filed Oct. 4, 2000; and U.S. Ser. No. 60/240,316, filed Oct. 13, 2000, which are incorporated herein by reference in their entireties.<span style="display: none">[0001] </span> </div>
</li> <summary-of-invention>
<heading>BACKGROUND OF THE INVENTION </heading>
<li> <para-num num="[0002]"> </para-num> <div class="description-line" id="P-0002" num="0002">The invention generally relates to nucleic acids and polypeptides encoded therefrom. More specifically, the invention relates to nucleic acids encoding cytoplasmic, nuclear, membrane bound, and secreted polypeptides, as well as vectors, host cells, antibodies, and recombinant methods for producing these nucleic acids and polypeptides. <span style="display: none">[0002] </span> </div>
</li> <heading>SUMMARY OF THE INVENTION </heading>
<li> <para-num num="[0003]"> </para-num> <div class="description-line" id="P-0003" num="0003">The invention is based, in part, upon the discovery of polynucleotide sequences encoding novel polypeptides. <span style="display: none">[0003] </span> </div>
</li> <li> <para-num num="[0004]"> </para-num> <div class="description-line" id="P-0004" num="0004">Accordingly, in one aspect, the invention provides an isolated nucleic acid molecule that includes the sequence of SEQ ID NO: 1, 3, 5, 7, 9, 11 or 13 or a fragment, homolog, analog or derivative thereof. The nucleic acid can include, e.g., a nucleic acid sequence encoding a polypeptide at least 85% identical to a polypeptide that includes the amino acid sequences of SEQ ID NO: 2, 4, 6, 8, 10, 12 or 14. The nucleic acid can be, e.g, a genomic DNA fragment, or a cDNA molecule. <span style="display: none">[0004] </span> </div>
</li> <li> <para-num num="[0005]"> </para-num> <div class="description-line" id="P-0005" num="0005">Also included in the invention is a vector containing one or more of the nucleic acids described herein, and a cell containing the vectors or nucleic acids described herein. <span style="display: none">[0005] </span> </div>
</li> <li> <para-num num="[0006]"> </para-num> <div class="description-line" id="P-0006" num="0006">The invention is also directed to host cells transformed with a vector comprising any of the nucleic acid molecules described above. <span style="display: none">[0006] </span> </div>
</li> <li> <para-num num="[0007]"> </para-num> <div class="description-line" id="P-0007" num="0007">In another aspect, the invention includes a pharmaceutical composition that includes a NOVX nucleic acid and a pharmaceutically acceptable carrier or diluent. <span style="display: none">[0007] </span> </div>
</li> <li> <para-num num="[0008]"> </para-num> <div class="description-line" id="P-0008" num="0008">In a further aspect, the invention includes a substantially purified NOVX polypeptide, e.g., any of the NOVX polypeptides encoded by an NOVX nucleic acid, and fragments, homologs, analogs, and derivatives thereof. The invention also includes a pharmaceutical composition that includes an NOVX polypeptide and a pharmaceutically acceptable carrier or diluent. <span style="display: none">[0008] </span> </div>
</li> <li> <para-num num="[0009]"> </para-num> <div class="description-line" id="P-0009" num="0009">In still a further aspect, the invention provides an antibody that binds specifically to an NOVX polypeptide. The antibody can be, e.g., a monoclonal or polyclonal antibody, and fragments, homologs, analogs, and derivatives thereof. The invention also includes a pharmaceutical composition including NOVX antibody and a pharmaceutically acceptable carrier or diluent. The invention is also directed to isolated antibodies that bind to an epitope on a polypeptide encoded by any of the nucleic acid molecules described above. <span style="display: none">[0009] </span> </div>
</li> <li> <para-num num="[0010]"> </para-num> <div class="description-line" id="P-0010" num="0010">The invention also includes kits comprising any of the pharmaceutical compositions described above. <span style="display: none">[0010] </span> </div>
</li> <li> <para-num num="[0011]"> </para-num> <div class="description-line" id="P-0011" num="0011">The invention further provides a method for producing an NOVX polypeptide by providing a cell containing an NOVX nucleic acid, e.g., a vector that includes an NOVX nucleic acid, and culturing the cell under conditions sufficient to express the NOVX polypeptide encoded by the nucleic acid. The expressed NOVX polypeptide is then recovered from the cell. Preferably, the cell produces little or no endogenous NOVX polypeptide. The cell can be, e.g., a prokaryotic cell or a eukaryotic cell. <span style="display: none">[0011] </span> </div>
</li> <li> <para-num num="[0012]"> </para-num> <div class="description-line" id="P-0012" num="0012">The invention is also directed to methods of identifying an NOVX polypeptide or nucleic acid in a sample by contacting the sample with a compound that specifically binds to the polypeptide or nucleic acid, and detecting complex formation, if present. <span style="display: none">[0012] </span> </div>
</li> <li> <para-num num="[0013]"> </para-num> <div class="description-line" id="P-0013" num="0013">The invention further provides methods of identifying a compound that modulates the activity of an NOVX polypeptide by contacting an NOVX polypeptide with a compound and determining whether the NOVX polypeptide activity is modified. <span style="display: none">[0013] </span> </div>
</li> <li> <para-num num="[0014]"> </para-num> <div class="description-line" id="P-0014" num="0014">The invention is also directed to compounds that modulate NOVX polypeptide activity identified by contacting an NOVX polypeptide with the compound and determining whether the compound modifies activity of the NOVX polypeptide, binds to the NOVX polypeptide, or binds to a nucleic acid molecule encoding an NOVX polypeptide. <span style="display: none">[0014] </span> </div>
</li> <li> <para-num num="[0015]"> </para-num> <div class="description-line" id="P-0015" num="0015">In another aspect, the invention provides a method of determining the presence of or predisposition of an NOVX-associated disorder in a subject. The method includes providing a sample from the subject and measuring the amount of NOVX polypeptide in the subject sample. The amount of NOVX polypeptide in the subject sample is then compared to the amount of NOVX polypeptide in a control sample. An alteration in the amount of NOVX polypeptide in the subject protein sample relative to the amount of NOVX polypeptide in the control protein sample indicates the subject has a tissue proliferation-associated condition. A control sample is preferably taken from a matched individual, i.e., an individual of similar age, sex, or other general condition but who is not suspected of having a tissue proliferation-associated condition. Alternatively, the control sample may be taken from the subject at a time when the subject is not suspected of having a tissue proliferation-associated disorder. In some embodiments, the NOVX is detected using an NOVX antibody. <span style="display: none">[0015] </span> </div>
</li> <li> <para-num num="[0016]"> </para-num> <div class="description-line" id="P-0016" num="0016">In a further aspect, the invention provides a method of determining the presence of or predisposition of an NOVX-associated disorder in a subject. The method includes providing a nucleic acid sample, e.g., RNA or DNA, or both, from the subject and measuring the amount of the NOVX nucleic acid in the subject nucleic acid sample. The amount of NOVX nucleic acid sample in the subject nucleic acid is then compared to the amount of an NOVX nucleic acid in a control sample. An alteration in the amount of NOVX nucleic acid in the sample relative to the amount of NOVX in the control sample indicates the subject has a NOVX-associated disorder. <span style="display: none">[0016] </span> </div>
</li> <li> <para-num num="[0017]"> </para-num> <div class="description-line" id="P-0017" num="0017">In a still further aspect, the invention provides a method of treating or preventing or delaying an NOVX-associated disorder. The method includes administering to a subject in which such treatment or prevention or delay is desired an NOVX nucleic acid, an NOVX polypeptide, or an NOVX antibody in an amount sufficient to treat, prevent, or delay a NOVX-associated disorder in the subject. <span style="display: none">[0017] </span> </div>
</li> <li> <para-num num="[0018]"> </para-num> <div class="description-line" id="P-0018" num="0018">Unless otherwise defined, all technical and scientific terms used herein have the same meaning as commonly understood by one of ordinary skill in the art to which this invention belongs. Although methods and materials similar or equivalent to those described herein can be used in the practice or testing of the present invention, suitable methods and materials are described below. All publications, patent applications, patents, and other references mentioned herein are incorporated by reference in their entirety. In the case of conflict, the present specification, including definitions, will control. In addition, the materials, methods, and examples are illustrative only and not intended to be limiting. <span style="display: none">[0018] </span> </div>
</li> <li> <para-num num="[0019]"> </para-num> <div class="description-line" id="P-0019" num="0019">Other features and advantages of the invention will be apparent from the following detailed description and claims. <span style="display: none">[0019] </span> </div>
</li> </summary-of-invention>
<heading>BRIEF DESCRIPTION OF THE DRAWINGS </heading>
<li> <para-num num="[0020]"> </para-num> <div class="description-line" id="P-0020" num="0020">FIG. 1 depicts quantitative gene expression analysis results measuring a NOV7 nucleic acid in normal and diseased tissue and cell lines. <span style="display: none">[0020]</span> </div>
</li> <li> <para-num num="[0021]"> </para-num> <div class="description-line" id="P-0021" num="0021">FIG. 2 depicts gene expression profile analysis of a NOV7 nucleic acid in a number of tumo cell line-derived libraries.<span style="display: none">[0021]</span> </div>
</li> <heading>DETAILED DESCRIPTION OF THE INVENTION </heading>
<li> <para-num num="[0022]"> </para-num> <div class="description-line" id="P-0022" num="0022">The present invention provides novel nucleotides and polypeptides encoded thereby. The sequences are collectively referred to as “NOVX nucleic acids” or “NOVX polynucleotides” and the corresponding encoded polypeptides are referred to as “NOVX polypeptides” or “NOVX proteins.” Unless indicated otherwise, “NOVX” is meant to refer to any of the novel sequences disclosed herein. Table 1 provides a summary of various disclosed NOVX nucleic acids and their encoded polypeptides. Example 1 provides a description of how the novel nucleic acids were identified.  
<span style="display: none">[0022] </span> <tables id="TABLE-US-00001" num="1"> <patent-tables colsep="0" frame="none" rowsep="0"> <table align="left" class="description-table" cols="1" colsep="0" rowsep="0" width="100%"> <thead> <tr class="description-tr"> <td align="center" class="description-td" colspan="1" nameend="1" namest="1">TABLE 1</td> </tr> </thead> <tbody><tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td align="center" class="description-td" colspan="1" nameend="1" namest="1" rowsep="1"> </td> </tr> <tr class="description-tr"> <td class="description-td">Sequences and Corresponding SEQ ID Numbers</td> </tr> </tbody></table> <table align="left" class="description-table" cols="5" colsep="0" rowsep="0" width="100%"> <tbody><tr class="description-tr"> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td">SEQ ID</td> <td class="description-td">SEQ ID</td> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">Internal</td> <td class="description-td">NO</td> <td class="description-td">NO</td> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">NOVX</td> <td class="description-td">Identification</td> <td class="description-td">(nucleic</td> <td class="description-td">(poly-</td> <td class="description-td">Homology/</td> </tr> <tr class="description-tr"> <td class="description-td">Assignment</td> <td class="description-td">Number</td> <td class="description-td">acid)</td> <td class="description-td">peptide)</td> <td class="description-td">expression</td> </tr> <tr class="description-tr"> <td align="center" class="description-td" colspan="5" nameend="5" namest="1" rowsep="1"> </td> </tr> <tr class="description-tr"> <td class="description-td">1</td> <td class="description-td">28804279.0.7</td> <td class="description-td"> 1</td> <td class="description-td"> 2</td> <td class="description-td">Expressed in</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td">fetal kidney</td> </tr> <tr class="description-tr"> <td class="description-td">2</td> <td class="description-td">28326488.0.55</td> <td class="description-td"> 3</td> <td class="description-td"> 4</td> <td class="description-td">Expressed in</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td">fetal kidney</td> </tr> <tr class="description-tr"> <td class="description-td">3</td> <td class="description-td">10312947.0.40</td> <td class="description-td"> 5</td> <td class="description-td"> 6</td> <td class="description-td">Expressed in</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td">pituitary gland</td> </tr> <tr class="description-tr"> <td class="description-td">4</td> <td class="description-td">25330368.0.1</td> <td class="description-td"> 7</td> <td class="description-td"> 8</td> <td class="description-td">Expressed in</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td">mammary gland</td> </tr> <tr class="description-tr"> <td class="description-td">5</td> <td class="description-td">4004056.0.143</td> <td class="description-td"> 9</td> <td class="description-td">10</td> <td class="description-td">Expressed in</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td">adrenal, mammary,</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td">prostate and fetal</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td">kidney</td> </tr> <tr class="description-tr"> <td class="description-td">6</td> <td class="description-td">3084780.0.73</td> <td class="description-td">11</td> <td class="description-td">12</td> <td class="description-td">Expressed in</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td">pancreas, fetal lung,</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td">stomach</td> </tr> <tr class="description-tr"> <td class="description-td">7</td> <td class="description-td">SC20692369</td> <td class="description-td">13</td> <td class="description-td">14</td> <td class="description-td">Homologous to N-</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td">acetylglucosaminyl</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td">transferases</td> </tr> <tr class="description-tr"> <td align="center" class="description-td" colspan="5" nameend="5" namest="1" rowsep="1"> </td> </tr> </tbody></table> </patent-tables> </tables>
</div>
</li> <li> <para-num num="[0023]"> </para-num> <div class="description-line" id="P-0023" num="0023">NOVX nucleic acids and their encoded polypeptides are useful in a variety of applications and contexts. The various NOVX nucleic acids and polypeptides according to the invention are useful as novel members of the protein families according to the presence of domains and sequence relatedness to previously described proteins. Additionally, NOVX nucleic acids and polypeptides can also be used to identify proteins that are members of the family to which the NOVX polypeptides belong. <span style="display: none">[0023] </span> </div>
</li> <li> <para-num num="[0024]"> </para-num> <div class="description-line" id="P-0024" num="0024">For example, NOVI and NOV2 nucleic acids were isolated from fetal kidney. Accordingly, NOVI and NOV2 nucleic acids, polypeptides, antibodies and other compositions of the present invention can be used to identify novel genes, proteins and signaling pathways in nephrogenesis and in kidney cancer. Similarly, a NOV3 nucleic acid originated from the pituitary gland, and thus can be used to identify novel genes, proteins, and signaling pathways in growth hormone regulation, vascular disease, and Cushing's Syndrome. <span style="display: none">[0024] </span> </div>
</li> <li> <para-num num="[0025]"> </para-num> <div class="description-line" id="P-0025" num="0025">Also, a NOV4 nucleic acid was isolated from the mammary gland and has been localized to a region of chromosome 21a. Thus, NOV4 nucleic acids, polypeptides, antibodies and other compositions of the present invention can be used as markers for chromosome 21 and to identify novel genes, proteins and signaling pathways in lactation mastisis and in breast cancer. <span style="display: none">[0025] </span> </div>
</li> <li> <para-num num="[0026]"> </para-num> <div class="description-line" id="P-0026" num="0026">Also, a NOV6 nucleic acid was isolated from pancreas. As such, NOV6 nucleic acids, polypeptides, antibodies and other compositions of the present invention can be used to identify novel genes, proteins and signaling pathways in diabetes and in pancreatic cancer. <span style="display: none">[0026] </span> </div>
</li> <li> <para-num num="[0027]"> </para-num> <div class="description-line" id="P-0027" num="0027">Further, NOV7 is a novel member of the N-acetylglucosaminyltransferase family, and NOV7 nucleic acids, polypeptides, antibodies and other compositions of the present invention can be used as such to identify other actetylglucosaminyltransferase family members. <span style="display: none">[0027] </span> </div>
</li> <li> <para-num num="[0028]"> </para-num> <div class="description-line" id="P-0028" num="0028">The NOVX nucleic acids and polypeptides can also be used to screen for molecules, which inhibit or enhance NOVX activity or function. Specifically, the nucleic acids and polypeptides according to the invention may be used as targets for the identification of small molecules that modulate or inhibit, e.g., nephrogenesis, cell differentiation, cell motility, cell proliferation and angiogenesis. <span style="display: none">[0028] </span> </div>
</li> <li> <para-num num="[0029]"> </para-num> <div class="description-line" id="P-0029" num="0029">Additional utilities for the NOVX nucleic acids and polypeptides according to the invention are disclosed herein. <span style="display: none">[0029] </span> </div>
</li> <li> <para-num num="[0030]"> </para-num> <div class="description-line" id="P-0030" num="0030">NOV1 <span style="display: none">[0030] </span> </div>
</li> <li> <para-num num="[0031]"> </para-num> <div class="description-line" id="P-0031" num="0031">A NOV1 sequence according to the invention includes a nucleic acid sequence encoding a polypeptide that is found in fetal kidney, including human fetal kidney. The disclosed nucleic acid (SEQ ID NO:1) as shown in Table 2 is 579 nucleotides in length and contains an open reading frame (ORF) that begins with an ATG initiation codon at nucleotides 30-32 and ends with a TAA stop codon at nucleotides 312-314. No Kozak consensus sequence is associated with this sequence. The representative ORF encodes a 94 amino acid polypeptide (SEQ ID NO:2). The predicted molecular weight of the NOV2 polypeptide is 10354.6 Daltons (Da). The encoded protein is a putative secreted protein, because PSORT analysis predicts that it is found outside of the cell with a certainty of 0.5135. The protein appears to have a cleavable N-terminal signal sequence, with the most likely cleavage site between positions 36 and 37. Putative untranslated regions upstream and downstream of the coding sequence are underlined in SEQ ID NO: 1.  
<span style="display: none">[0031] </span> <tables id="TABLE-US-00002" num="2"> <patent-tables colsep="0" frame="none" pgwide="1" rowsep="0" tabstyle="monospace"> <table align="left" class="description-table" cols="4" colsep="0" rowsep="0" width="100%"> <thead> <tr class="description-tr"> <td align="center" class="description-td" colspan="4" nameend="4" namest="1">TABLE 2</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td align="center" class="description-td" colspan="4" nameend="4" namest="1" rowsep="1"> </td> </tr> </thead> <tbody><tr class="description-tr"> <td class="description-td">1</td> <td class="description-td"> <u>TATGGAATAAAGAACCATGACGGAGTCCC</u> <b>ATG</b>CGCAGCCAGAGAAGAGAC</td> <td class="description-td">(SEQ ID NO.: 1)</td> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">51</td> <td class="description-td">CACCACCCGAGAGAGGTTTCATCCTACCATGTAACTCTGCTTACAGCCTA</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">101</td> <td class="description-td">CTTGCTTCTCACCGGCGTGCTGGGGACAGCAAAGTCTGAGGACTCTGGTT</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">151</td> <td class="description-td">GGTGTGGGCCTGTGTGCAAGGAGAGCAGTGGCCATGGGATAAGGCCTCTG</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">201</td> <td class="description-td">CACAGCTCTAGAAGCTTCAATCCCATTTCCACCCATACATCTCTTTGTGC</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">251</td> <td class="description-td">TCTCACACCCCCACAGCCCTTCTGGAATAAGACCATCACAGCACAGGGTT</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">301</td> <td class="description-td">TGCAAGATGTC<u> <b>TAA</b> </u> <u>TGCCAGTCATTCACAGGGCAGCTCAGACCCTGGCCT</u> </td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">351</td> <td class="description-td"> <u>GCGGTGCATACTAGGTGACTCCACATGAGGTGTCATGCTAGATCCTGCAG</u> </td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">401</td> <td class="description-td"> <u>GGAGAATAAGCACACACAGGCCCGTGACCCATGCTGTGGACTTCATGTTC</u> </td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">451</td> <td class="description-td"> <u>TAGGAGGTAGAGGGAGACAGACAAGAATCAAATGACTGTACTAGGCCGGG</u> </td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">501</td> <td class="description-td"> <u>CGCACTGGCTCACGCCTGTAATCCCAGCACTTTGGGGAGGCCGAGGCAGG</u> </td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">551</td> <td class="description-td"> <u>TGGATCACGAGGCCAGGCGTTCGAGACCA</u> </td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">1</td> <td class="description-td">MetArgSerGlnArgArgAspHisHisProArgGluValSerSerTyrHis</td> <td class="description-td">(SEQ ID NO.: 2)</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">18</td> <td class="description-td">ValThrLeuLeuThrAlaTyrLeuLeuLeuThrGlyValLeuGlyThrAla</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">35</td> <td class="description-td">LysSerGluAspSerGlyTrpCysGlyProValCysLysGluSerSerGly</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">52</td> <td class="description-td">HisGlyIleArgProLeuHisSerSerArgSerPheAsnProIleSerThr</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">69</td> <td class="description-td">HisThrSerLeuCysAlaLeuThrProProGlnProPheTrpAsnLysThr</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">86</td> <td class="description-td">IleThrAlaGlnGlyLeuGlnAspVal</td> </tr> <tr class="description-tr"> <td align="center" class="description-td" colspan="4" nameend="4" namest="1" rowsep="1"> </td> </tr> </tbody></table> </patent-tables> </tables>
</div>
</li> <li> <para-num num="[0032]"> </para-num> <div class="description-line" id="P-0032" num="0032">The disclosed NOV1 nucleic acid sequence has a high degree of homology (100% identity) with portions of a region of human chromosome 1, which is contained in clone RP4-709L21 (EMBL Accession No.: 078646.29) on chromosome 1 q42.13-43, as is shown in Table 3. A search of the Online Mendelian Inheritance in Man (OMIM) database indicates that this region contains the human renin gene, which is associated with essential hypertension and kidney failure (OMIM database entry number 179820); angiotensin I, which is also associated with onset and progression of hypertesion (OMIM database entry number 106150); a locus predisposing to prostate cancer (OMIM database entry number 602759); and a prostate cancer locus (OMIM database entry number 176807).  
<span style="display: none">[0032] </span> <tables id="TABLE-US-00003" num="3"> <patent-tables colsep="0" frame="none" pgwide="1" rowsep="0" tabstyle="monospace"> <table align="left" class="description-table" cols="6" colsep="0" rowsep="0" width="100%"> <thead> <tr class="description-tr"> <td align="center" class="description-td" colspan="6" nameend="6" namest="1">TABLE 3</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td align="center" class="description-td" colspan="6" nameend="6" namest="1" rowsep="1"> </td> </tr> </thead> <tbody><tr class="description-tr"> <td class="description-td">NOV1:</td> <td class="description-td">1</td> <td class="description-td">tatggaataaagaaccatgacggagtcccatgcgcagccagagaagagaccaccacccga</td> <td class="description-td">60</td> <td class="description-td"> </td> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td">||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||</td> </tr> <tr class="description-tr"> <td class="description-td">CHR 1:</td> <td class="description-td">96874</td> <td class="description-td">tatggaataaagaaccatgacggagtcccatgcgcagccagagaagagaccaccacccga</td> <td class="description-td">96815</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">NOV1: </td> <td class="description-td">61</td> <td class="description-td">gagaggtttcatcctaccatgtaactctgcttacagcctacttgcttctcaccggcgtgc</td> <td class="description-td">120</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td">||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||</td> </tr> <tr class="description-tr"> <td class="description-td">CHR 1:</td> <td class="description-td">96814</td> <td class="description-td">gagaggtttcatcctaccatgtaactctgcttacagcctacttgcttctcaccggcgtgc</td> <td class="description-td">96755</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">NOV1:</td> <td class="description-td">121</td> <td class="description-td">tggggacagcaaagtctgaggactctggttggtgtgggcctgtgtgcaaggagagcagtg</td> <td class="description-td">180</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td">||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||</td> </tr> <tr class="description-tr"> <td class="description-td">CHR 1:</td> <td class="description-td">96754</td> <td class="description-td">tggggacagcaaagtctgaggactctggttggtgtgggcctgtgtgcaaggagagcagtg</td> <td class="description-td">96695</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">NOV1:</td> <td class="description-td">181</td> <td class="description-td">gccatggyataaggcctctgcacagctctagaagcttcaatcccatttccacccatacat</td> <td class="description-td">240</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td">||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||</td> </tr> <tr class="description-tr"> <td class="description-td">CHR 1:</td> <td class="description-td">96694</td> <td class="description-td">gccatgggataaggcctctgcacagctctagaagcttcaatcccatttccacccatacat</td> <td class="description-td">96635</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">NOV1:</td> <td class="description-td">241</td> <td class="description-td">ctctttgtgctctcacacccccacagcccttctggaataagaccatcacagcacagggtt</td> <td class="description-td">300</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td">||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||</td> </tr> <tr class="description-tr"> <td class="description-td">CHR 1:</td> <td class="description-td">96634</td> <td class="description-td">ctctttgtgctctcacacccccacagcccttctggaataagaccatcacagcacagggtt</td> <td class="description-td">96575</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">NOV1:</td> <td class="description-td">301</td> <td class="description-td">tgcaagatgtctaatgccagtcattcacagggcagctcagaccctggcctgcggtgcata</td> <td class="description-td">360</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td">||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||</td> </tr> <tr class="description-tr"> <td class="description-td">CHR 1:</td> <td class="description-td">96574</td> <td class="description-td">tgcaagatgtctaatgccagtcattcacagggcagctcagaccctggcctgcggtgcata</td> <td class="description-td">96515</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">NOV1:</td> <td class="description-td">361</td> <td class="description-td">ctaggtgactccacatgaggtgtcatgctagatcctgcagggagaataagcacacacagg</td> <td class="description-td">420</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td">||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||</td> </tr> <tr class="description-tr"> <td class="description-td">CHR 1:</td> <td class="description-td">96514</td> <td class="description-td">ctaggtgactccacatgaggtgtcatgctagatcctgcagggagaataagcacacacagg</td> <td class="description-td">96455</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">NOV1:</td> <td class="description-td">421</td> <td class="description-td">cccgtgacccatget9tggacttcatgttctaggaggtagagggagacagacaagaatca</td> <td class="description-td">480</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td">||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||</td> </tr> <tr class="description-tr"> <td class="description-td">CHR 1:</td> <td class="description-td">96454</td> <td class="description-td">cccgtgacccatgctgtggacttcatgttctaggaggtagagggagacagacaagaatca</td> <td class="description-td">96395</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">NOV1:</td> <td class="description-td">481</td> <td class="description-td">aatgactgtactaggccgggcgcactggctcacgcctgtaatcccagcactttggggagg</td> <td class="description-td">540</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td">||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||</td> </tr> <tr class="description-tr"> <td class="description-td">CHR 1:</td> <td class="description-td">96394</td> <td class="description-td">aatgactgtactaggccgggcgcactggctcacgcctgtaatcccagcactttggggagg</td> <td class="description-td">96335</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">NOV1:</td> <td class="description-td">541</td> <td class="description-td">ccgaggcaggtggatcacgaggccaggcgttcgagacca</td> <td class="description-td">579</td> <td class="description-td">(SEQ ID NO. 1)</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td">|||||||||||||||||||||||||||||||||||||||</td> </tr> <tr class="description-tr"> <td class="description-td">CHR 1:</td> <td class="description-td">96334</td> <td class="description-td">ccgaggcaggtggatcacgaggccaggcgttcgagacca</td> <td class="description-td">96296</td> <td class="description-td">(SEQ ID NO. 15)</td> </tr> <tr class="description-tr"> <td align="center" class="description-td" colspan="6" nameend="6" namest="1" rowsep="1"> </td> </tr> </tbody></table> </patent-tables> </tables>
</div>
</li> <li> <para-num num="[0033]"> </para-num> <div class="description-line" id="P-0033" num="0033">The q42.13-43 region of chromosome 1 contains multiple hypertension-associated loci. Reagents that detect NOV1 nucleic acids and polypeptides can be used to detect kidney tissue, and is useful in determining changes in expression of genes contained within the 1q42.13-43 locus. NOV1 provides new diagnostic or therapeutic compositions useful in the treatment of disorders associated with alterations in the expression of members of renal and hypertension-associated proteins. Also, NOV1 is useful in the diagnosis of prostate cancer. NOV1 nucleic acids, polypeptides, antibodies, and other compositions of the present invention are useful in the treatment and/or diagnosis of a variety of diseases and pathologies, including by way of nonlimiting example, those involving renal failure and cihrrosis, stroke, heart failure and other vascular diseases. <span style="display: none">[0033] </span> </div>
</li> <li> <para-num num="[0034]"> </para-num> <div class="description-line" id="P-0034" num="0034">NOV2 <span style="display: none">[0034] </span> </div>
</li> <li> <para-num num="[0035]"> </para-num> <div class="description-line" id="P-0035" num="0035">A NOV2 nucleic acid according to the invention includes the nucleic acid disclosed in Table 4. The tissue of origin of the disclosed NOV2 nucleic acid is fetal kidney. The disclosed sequence localizes to human chromosome 1. The disclosed nucleic acid (SEQ ID NO:3) is 692 nucleotides in length and contains an open reading frame (ORF) that begins with an ATG initiation codon at nucleotides 224-226 and ends with a TAA stop codon at nucleotides 569-571. The nucleic acid sequence contains a Kozak consensus sequence. The representative ORF encodes a 115 amino acid polypeptide (SEQ ID NO: 4). The predicted MW of the NOV2 polypeptide is 13671.8 Da. NOV2 is predicted by PSORT analysis to segregate to the microbody (peroxisome) with a certainty of 0.6400. The protein appears to have no N-terminal signal sequence. Putative untranslated regions upstream and downstream of the coding sequence are underlined in SEQ ID NO: 3.  
<span style="display: none">[0035] </span> <tables id="TABLE-US-00004" num="4"> <patent-tables colsep="0" frame="none" pgwide="1" rowsep="0" tabstyle="monospace"> <table align="left" class="description-table" cols="3" colsep="0" rowsep="0" width="100%"> <thead> <tr class="description-tr"> <td align="center" class="description-td" colspan="3" nameend="3" namest="1">TABLE 4</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td align="center" class="description-td" colspan="3" nameend="3" namest="1" rowsep="1"> </td> </tr> </thead> <tbody><tr class="description-tr"> <td class="description-td">1</td> <td class="description-td"> <u>ATGACCATGCATCCATTTACAGTAAAGGGATTGCCTACATCTCAGACAAC</u> </td> <td class="description-td">(SEQ ID NO.: 3)</td> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">51</td> <td class="description-td"> <u>ACTTCATGTACAGTACACAAATCAAGGAAACAGCTTCATCACTGATGTTA</u> </td> </tr> <tr class="description-tr"> <td class="description-td">101</td> <td class="description-td"> <u>CCTTTAATCTAACAAGATCTCTATAAAACAAGAAAACCTCTACGTACAGA</u> </td> </tr> <tr class="description-tr"> <td class="description-td">151</td> <td class="description-td"> <u>TCTTTTAAAATTAAAGCAGGCATCTTTGCTGATCCACCTCTATAAGTTGC</u> </td> </tr> <tr class="description-tr"> <td class="description-td">201</td> <td class="description-td"> <u>AGGTTGAGTATCTCTTATCTGAA</u>ATGCTAGAGACCAGAAGTGTTTCAGGT</td> </tr> <tr class="description-tr"> <td class="description-td">251</td> <td class="description-td">TTCAGATATTTAGATTTTGGAATATTTGCATATACACGAGATATCCAGGG</td> </tr> <tr class="description-tr"> <td class="description-td">301</td> <td class="description-td">GAAGAGACCCAAGTCTAAACATGAAATTCATTTATGTTTCATATACACCT</td> </tr> <tr class="description-tr"> <td class="description-td">351</td> <td class="description-td">CATATATATATAGCCTGAAGGTAATTTTATACAGTATTTATAATTTGTCC</td> </tr> <tr class="description-tr"> <td class="description-td">401</td> <td class="description-td">AAGGAACAAAGTTTTGACTGTGTTTTOACTATGACTCGTCATGTGAAGTC</td> </tr> <tr class="description-tr"> <td class="description-td">451</td> <td class="description-td">ATATGTGGAATTTTCCACTTGTGGCATCACACAGGCACTCAAAAAGCTTC</td> </tr> <tr class="description-tr"> <td class="description-td">501</td> <td class="description-td">AGATTTGGGAGCATATTGGATTTCGCATATTCAGATTAGGGATGCTCAAC</td> </tr> <tr class="description-tr"> <td class="description-td">551</td> <td class="description-td">CCATACTCAGTTTACCAG<u>TAAAAAAACATAATGTTTGCAATTACTCCTCC</u> </td> </tr> <tr class="description-tr"> <td class="description-td">601</td> <td class="description-td"> <u>TTTTAAATATATAATTATTTTTGGTATGGGGGAAAAGAGTGAGAACTTTA</u> </td> </tr> <tr class="description-tr"> <td class="description-td">651</td> <td class="description-td"> <u>TTTCACCTGCCCGGGCAGCCGCTCGAGCCCTATAGTGAGTAA</u> </td> </tr> <tr class="description-tr"> <td class="description-td">1</td> <td class="description-td">MetLeuGluThrArgSerValSerGlyPheArgTyrLeuAspPheGlyIle</td> <td class="description-td">(SEQ ID NO.: 4)</td> </tr> <tr class="description-tr"> <td class="description-td">18</td> <td class="description-td">PheAlaTyrThrArgAspIleGlnGlyLysArgProLysSerLysHisGlu</td> </tr> <tr class="description-tr"> <td class="description-td">35</td> <td class="description-td">IleHisLeuCysPheIleTyrThrSerTyrIleTyrSerLeuLysValIle</td> </tr> <tr class="description-tr"> <td class="description-td">52</td> <td class="description-td">LeuTyrSerIleTyrAsnLeuSerLysGluGlnSerPheAspCysValLeu</td> </tr> <tr class="description-tr"> <td class="description-td">69</td> <td class="description-td">ThrMetThrArgHisValLysSerTyrValGluPheSerThrCysGlyIle</td> </tr> <tr class="description-tr"> <td class="description-td">86</td> <td class="description-td">ThrGlnAlaLeuLysLysLeuGlnIleTrpGluHisIleGlyPheArgIle</td> </tr> <tr class="description-tr"> <td class="description-td">103</td> <td class="description-td">PheArgLeuGlyMetLeuAsnProTyrSer<u>V</u>alTyrGln</td> </tr> <tr class="description-tr"> <td align="center" class="description-td" colspan="3" nameend="3" namest="1" rowsep="1"> </td> </tr> </tbody></table> </patent-tables> </tables>
</div>
</li> <li> <para-num num="[0036]"> </para-num> <div class="description-line" id="P-0036" num="0036">The NOV2 nucleic acid sequence has a high degree of homology (98% identity) with a region of a human genomic clone corresponding to chromosome 11 (CHRI 1) (EMBL Accession No.: AL133295), as shown in Table 5. The p12-14.2 region of chromosome 11 contains multiple diabetes-associated loci. This clone is contained within the region on 11p12-14.2. A search of the Online Mendelian Inheritance in Man (OMIM) database indicates that this region contains the LIM homeobox gene 1 (OMIM database entry number 601999) and the mitogen-activated protein kinase 8-interacting protein (MAPK8IP1; OMIM database entry number 604641), as well a locus for diabetes millitus type II (OMIM database entry number 125853) and the insulin gene (OMIM database entry number 176730).  
<span style="display: none">[0036] </span> <tables id="TABLE-US-00005" num="5"> <patent-tables colsep="0" frame="none" pgwide="1" rowsep="0" tabstyle="monospace"> <table align="left" class="description-table" cols="6" colsep="0" rowsep="0" width="100%"> <thead> <tr class="description-tr"> <td align="center" class="description-td" colspan="6" nameend="6" namest="1">TABLE 5</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td align="center" class="description-td" colspan="6" nameend="6" namest="1" rowsep="1"> </td> </tr> </thead> <tbody><tr class="description-tr"> <td class="description-td">NOV1:</td> <td class="description-td">1</td> <td class="description-td">atgaccatgcatccatttacagtaaagggattgcctacatctcagacaacacttcatgta</td> <td class="description-td">60</td> <td class="description-td">(SEQ ID NO. 16)</td> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td">||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||</td> </tr> <tr class="description-tr"> <td class="description-td">CHR11:</td> <td class="description-td">90905</td> <td class="description-td">atgaccatgeatccattiacagtaaagggattgcctacatctcagacaacacttcatgta</td> <td class="description-td">90846</td> <td class="description-td">(SEQ ID NO. 17)</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">NOV1:</td> <td class="description-td">61</td> <td class="description-td">aagtacacaaatcaaggaaacagcttcatcactgatgttacctttaatctaacaagatct</td> <td class="description-td">120</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td">||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||</td> </tr> <tr class="description-tr"> <td class="description-td">CHR11:</td> <td class="description-td">90845</td> <td class="description-td">aagtacacaaatcaaggaaacagcttcatcactgatgttacctttaatctaacaagatct</td> <td class="description-td">90786</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">NOV1:</td> <td class="description-td">121</td> <td class="description-td">ctataaaacaagaaaacctctacgtacagatcttttaaaattaaagcaggcatctttgct</td> <td class="description-td">180</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td">||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||</td> </tr> <tr class="description-tr"> <td class="description-td">CHR11:</td> <td class="description-td">90785</td> <td class="description-td">ctataaaacaagaaaacctctacgtacagatcttttaaaattaaagcaggcatctttgct</td> <td class="description-td">90726</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">NOV1:</td> <td class="description-td">181</td> <td class="description-td">gatccacctctataagttgcaggttgagtatctcttatctgaaatgctagagaccagaag</td> <td class="description-td">240</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td">||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||</td> </tr> <tr class="description-tr"> <td class="description-td">CHR11:</td> <td class="description-td">90725</td> <td class="description-td">gatccacctctataagttgcaggttgagtatctcttatctgaaatgctagagaccagaag</td> <td class="description-td">90666</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">NOV1:</td> <td class="description-td">241</td> <td class="description-td">tgtttcaggtttcagatatttagattttggaatatttgcatatacacgagatatccaggg</td> <td class="description-td">300</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td">||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||</td> </tr> <tr class="description-tr"> <td class="description-td">CHR11: 90665</td> <td class="description-td">tgtttcaggtttcagatatttagattttggaatatttgcatatacacgagatatccaggg</td> <td class="description-td">90606</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">NOV1:</td> <td class="description-td">301</td> <td class="description-td">gaagagaeccaagtctaaacatgaaattcatttatgtttcatatacacctcatatatata</td> <td class="description-td">360</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td">||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||</td> </tr> <tr class="description-tr"> <td class="description-td">CHR11:</td> <td class="description-td">90605</td> <td class="description-td">gaagagacccaagtctaaacatgaaattcatttatgtttcatatacacctcatatatata</td> <td class="description-td">90546</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">NOV1:</td> <td class="description-td">361</td> <td class="description-td">tagcctgaaggtaattttatacagtatttataatttgtccaaggaacaaagttttgactg</td> <td class="description-td">420</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td">||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||</td> </tr> <tr class="description-tr"> <td class="description-td">CHR11:</td> <td class="description-td">90545</td> <td class="description-td">tagcctgaaggtaattttatacagtatttataatttgtccaaggaacaaagttttgactg</td> <td class="description-td">90486</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">NOV1:</td> <td class="description-td">421</td> <td class="description-td">tgttttgactatgactcgtcatgtgaagtcatatgtggaattttccacttgtggcatcac</td> <td class="description-td">480</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td">||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||</td> </tr> <tr class="description-tr"> <td class="description-td">CHR11:</td> <td class="description-td">90485</td> <td class="description-td">tgttttgactatgactcgtcatgtgaagtcatatgtggaattttccacttgtggcatcac</td> <td class="description-td">90426</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">NOV1:</td> <td class="description-td">481</td> <td class="description-td">acaggcactcaaaaagcttcagatttgggagcatattggatttcgcatattcagattagg</td> <td class="description-td">540</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td">||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||</td> </tr> <tr class="description-tr"> <td class="description-td">CHR11:</td> <td class="description-td">90425</td> <td class="description-td">acaggcactcaaaaagcttcagatttgggagcatattggatttcgcatattcagattagg</td> <td class="description-td">90366</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">NOV1:</td> <td class="description-td">541</td> <td class="description-td">gatgctcaacccatactcagtttaccagtnnnnnnncataatgtttgcaattactcctcc</td> <td class="description-td">600</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td">|||||||||||||||||||||||||||||       ||||||||||||||||||||||||</td> </tr> <tr class="description-tr"> <td class="description-td">CHR11:</td> <td class="description-td">90365</td> <td class="description-td">gatgctcaacccatactcagtttaccagtaaaaaaacataatgtttgcaattactcctcc</td> <td class="description-td">90306</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">NOV:1:</td> <td class="description-td">601</td> <td class="description-td">ttttaaatatataattatttttggtatgggggaaaagagtgagaactttatttcac</td> <td class="description-td">656</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td">||||||||||||||||||||||||||||||||||||||||||||||||||||||||</td> </tr> <tr class="description-tr"> <td class="description-td">CHR11:</td> <td class="description-td">90305</td> <td class="description-td">ttttaaatatataattatttttggtatgggggaaaagagtgagaactttatttcac</td> <td class="description-td">90250</td> </tr> <tr class="description-tr"> <td align="center" class="description-td" colspan="6" nameend="6" namest="1" rowsep="1"> </td> </tr> </tbody></table> </patent-tables> </tables>
</div>
</li> <li> <para-num num="[0037]"> </para-num> <div class="description-line" id="P-0037" num="0037">NOV2 can be used to detect kidney tissue and is useful in determining changes in expression of genes contained within the 11p12-14.2 locus. NOV2 provides new diagnostic or therapeutic compositions useful in the treatment of disorders associated with alterations in the expression of members of renal and diabetes-associated proteins. Also, NOV2 is useful in the diagnosis of cancer. NOV2 nucleic acids, polypeptides, antibodies, and other compositions of the present invention are useful in the treatment and/or diagnosis of a variety of diseases and pathologies, including by way of nonlimiting example, those involving renal failure and cihrrosis, diabetes, macular degeneration, retinopathy and other vascular diseases. <span style="display: none">[0037] </span> </div>
</li> <li> <para-num num="[0038]"> </para-num> <div class="description-line" id="P-0038" num="0038">NOV3 <span style="display: none">[0038] </span> </div>
</li> <li> <para-num num="[0039]"> </para-num> <div class="description-line" id="P-0039" num="0039">A NOV3 sequence according to the invention includes the nucleic acid shown in Table 6. The disclosed sequence encodes a polypeptide related to acetylglucosaminyltransferase. The tissue of origin of the disclosed NOV3 nucleic acid sequence is the pituitary gland. Sequences homologous to the disclosed NOV3 sequence are found in a mixture of eight human adult and two fetal tissues, including pancreas, placenta and kidney. <span style="display: none">[0039] </span> </div>
</li> <li> <para-num num="[0040]"> </para-num> <div class="description-line" id="P-0040" num="0040">The disclosed nucleic acid (SEQ ID NO:5) is 2,351 nucleotides in length and contains an open reading frame (ORF) that begins with an ATG initiation codon at nucleotides 260-262 and ends with a TGA stop codon at nucleotides 2,111-2,113, as shown in Table 6. The ORF encodes a 617 amino acid polypeptide (SEQ ID NO: 6). The predicted MW of the NOV3 polypeptide is 70921.7 Da. The NOV3 polypeptide is a putative secreted protein but additionally carries a nuclear localization signal, according to PSORT, with a certainty of 0.6000. SignalP predicts that there may be a cleavage site between residues 17 and 18. Putative untranslated regions upstream and downstream of the coding sequence are underlined in SEQ ID NO: 5.  
<span style="display: none">[0040] </span> <tables id="TABLE-US-00006" num="6"> <patent-tables colsep="0" frame="none" pgwide="1" rowsep="0" tabstyle="monospace"> <table align="left" class="description-table" cols="4" colsep="0" rowsep="0" width="100%"> <thead> <tr class="description-tr"> <td align="center" class="description-td" colspan="4" nameend="4" namest="1">TABLE 6</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td align="center" class="description-td" colspan="4" nameend="4" namest="1" rowsep="1"> </td> </tr> </thead> <tbody><tr class="description-tr"> <td class="description-td">1</td> <td class="description-td">TAAAAATACAAAAAATTAGCCGGGCGTAGTGGCGGGCGCCTGTAGTCCCA</td> <td class="description-td"> </td> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">51</td> <td class="description-td">GCTACTTGGGAGGCTGAGGCAGGAGAATGGCGTGAACCCGGGAGGCAGAG</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">101</td> <td class="description-td">CTTGCAGTGAGCCGAGATCCCGCCACTGCACTCCAGCCTGGGCGACAGAG</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">151</td> <td class="description-td">CGAGACTCCGTCTCAAAAAAAAAAAAAAAGAACATCCTGAGCCGGGCGTG</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">201</td> <td class="description-td">GAAAAGCTCTTTGCAGATGGCGCTTCCATCTCTGCGCCCCTCGGGGTGGG</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">251</td> <td class="description-td">GGCTGTCCCATGTTGCTCCTGCTGGGGCCTCTCAGGCTTCCTCTTTGCCC</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">301</td> <td class="description-td">ACCCAAAAGGAAAAATCCACTGCACCTCCACTTGGTGACTGACGCCGTGG</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">351</td> <td class="description-td">CCAGAAACATCCTGGAGACGCTCTTCCACACATGGATGGTGCCTGCTATC</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">401</td> <td class="description-td">GATCCCCNTGTCAGCTTTTATCATGCCGACCAGCTCAAGCCCCAGGTCTC</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">451</td> <td class="description-td">CTGGATCCCCAACAAGCACTACTCCGGCCTCTATGGGCTAATGAAGCTGG</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">501</td> <td class="description-td">TGCTGCCCAATGCCTTGCCTGCTGAGCTGGCCCGCGTCATTGTCCTGGAC</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">551</td> <td class="description-td">ACGGATGTCACCTTCGCCTCTGACATCTCGGAGCTCTGGGCCCTCTTTGC</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">601</td> <td class="description-td">TCACTTTTCTGACACGCAGGCGATCGGTCTTGTGGAGAACCAGAGTGACT</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">651</td> <td class="description-td">GGTACCTGGGCAACCTCTGGAAGAACCACAGGCCCTGGCCTGCCTTGGGC</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">701</td> <td class="description-td">CGGGGATTTAACACAGGTGTGATCCTGCTGCGGCTGGACCGGCTCCGGCA</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">751</td> <td class="description-td">GGCTGGCTGGGAGCAGATGTGGAGGCTGACAGCCAGGCGGGAGCTCCTTA</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">801</td> <td class="description-td">GCCTGCCTGCCACCTCACTGGCTGACCAGGACATCTTCAACGCTGTGATC</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">851</td> <td class="description-td">AAGGAGCACCCGGGGCTAGTGCAGCGTCTGCCTTGTGTCTGGAATGTGCA</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">901</td> <td class="description-td">GCTGTCAGATCACACACTGGCCGAGCGCTGCTACTCTGAGGCGTCTGACC</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">951</td> <td class="description-td">TCAAGGTGATCCACTGGAACTCACCAAAGAAGCTTCGGGTGAAGAACAAG</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">1001</td> <td class="description-td">CATGTGGAATTCTTCCGCAATTTCTACCTGACCTTCCTGGAGTACGATGG</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">1051</td> <td class="description-td">GAACCTGCTGCGGAGAGAGCTCTTTGTGTGCCCCAGCCAGCCCCCACCTG</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">1101</td> <td class="description-td">GTGCTGAGCAGTTGCAGCAGGCCCTGGCACAACTGGACGAGGAAGACCCC</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">1151</td> <td class="description-td">TGCTTTGAGTTCCGGCAGCAGCAGCTCACTGTGCACCGTGTGCATGTCAC</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">1201</td> <td class="description-td">TTTCCTGCCCCATGAACCGCCACCCCCCCGGCCTCACGATGTCACCCTTG</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">1251</td> <td class="description-td">TGGCCCAGCTGTCCATGGACCGGCTGCAGATGTTGGAAGCCCTGTGCAGG</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">1301</td> <td class="description-td">CACTGGCCTGGCCCCATGAGCCTGGCCTTGTACCTGACAGACGCAGAAGC</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">1351</td> <td class="description-td">TCAGCAGTTCCTGCATTTCGTCGAGGCCTCACCAGTGCTTGCTGCCCGGC</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">1401</td> <td class="description-td">AGGACGTGGCCTACCATGTGGTGTACCGTGAGGGGCCCCTATACCCCGTC</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">1451</td> <td class="description-td">AACCAGCTTCGCAACGTGGCCTTGGCCCAGGCCCTCACGCCTTACGTCTT</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">1501</td> <td class="description-td">CCTCAGTGACATTGACTTCCTGCCTGCCTATTCTCTCTACGACTACCTCA</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">1551</td> <td class="description-td">GGGCCTCCATTGAGCAGCTGGGGCTGGGCAGCCGGCGCAAGGCAGCACTG</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">1601</td> <td class="description-td">GTGGTGCCGGCATTTGAGACCCTGCGCTACCGCTTCAGCTTCCCCCATTC</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">1651</td> <td class="description-td">CAAGGTGGAGCTGTTGGCCTTGCTGGATGCGGGCACTCTCTACACCTTCA</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">1701</td> <td class="description-td">GGTACCACGAGTCGCCCCGAGGCCACGCACCCACAGACTATGCCCGCTGG</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">1751</td> <td class="description-td">CGGGAGGCTCAGGCCCCGTACCGTGTGCAATGGGCGGCCAACTATGAACC</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">1801</td> <td class="description-td">CTACGTGGTGGTGCCACGAGACTGTCCCCGCTATGATCCTCGCTTTGTGG</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">1851</td> <td class="description-td">GCTTCGGCTGGAACAAAGTGGCCcACATTGTGGAGCTGGATGCCCAGGAA</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">1901</td> <td class="description-td">TATGAGCTCCTGGTGCTGCCCGAGGCCTTCACCATCCATCTGCCCCACGC</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">1951</td> <td class="description-td">TCCAAGCCTGGACATCTCCCGCTTCCGCTCCAGCCCCACCTATCGTGACT</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">2001</td> <td class="description-td">GCCTCCAGGCCCTCAAGGACGAATTCCACCAGGACTTGTCCCGCCACCAT</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">2051</td> <td class="description-td">GGGGCTGCTGCCCTCAAATACCTCCCAGCCCTGCAGCAGCCCCAGAGCCC</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">2101</td> <td class="description-td">TGCCCGAGGCTGAGGCTGGGCCGGCGCTGCCCCTCATCTTAGCATTGGGC</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">2151</td> <td class="description-td">AGACACCAGGGCAACCTGCCCTCCGCCATCCCTGCTATTTAAATTATTTA</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">2201</td> <td class="description-td">AGGTCTCTGGGAAGGGCTGGGGCAGAGCATCTGTGGGGTGGGGTCTTCCC</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">2251</td> <td class="description-td">CTTGCTGCTATTGTATGGCTGGGGACTGGTCTCTCTCTGCCCCAGCCAGT</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">2301</td> <td class="description-td">TTGGGGCTGGTTCCCCCATCTTGAATTGTTTATCCCTTTTTCATAATTAA</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">2351</td> <td class="description-td">A</td> <td class="description-td">(SEQ ID NO.: 5)</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">1</td> <td class="description-td">MetLeuLeuLeuLeuGlyProLeuArgLeuProLeuCysProProLysArg</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">18</td> <td class="description-td">LysAsnProLeuHisLeuHisLeuValThrAspAlaValAlaArgAsnIle</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">35</td> <td class="description-td">LeuGluThrLeuPheHisThrTrpMetValProAlalleASpPro---Val</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">52</td> <td class="description-td">SerpheTyrHisAlaAspGlnLeuLysProGlnValSerTrpIleProAsn</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">69</td> <td class="description-td">LysHisTyrSerGlyLeuTyrGlyLeuMetLysLeuValLeuProAsnAla</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">86</td> <td class="description-td">LeuProAlaGluLeuAlaArgValIleValLeuAspThrAspValThrPhe</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">103</td> <td class="description-td">AlaSerAspIleSerGluLeuTrpAlaLeuPheAlaHisPheSerAspThr</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">120</td> <td class="description-td">GlnAlaIleGlyLeuValGluAsnGlnSerAspTrpTyrLeuGlyAsnLeu</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">137</td> <td class="description-td">TrpLysAsnHisArgProTrpProAlaLeuGlyArgGlyPheAsnThrGly</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">154</td> <td class="description-td">VaLIleLeuLeuArgLeuAspArgLeuArgGlnAlaGlyTrpGluGlnMet</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">171</td> <td class="description-td">TrpArgLeuThrAlaArgArgGluLeuLeuSerLeuProAlaThrSerLeu</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">188</td> <td class="description-td">AlaAspGlnAspIlePheAsnAlaValIleLysGluHisProGlyLeuVal</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">205</td> <td class="description-td">GlnArgLeuProCysValTrpAsnValGlnLeuSerAspHisThrLeuAla</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">222</td> <td class="description-td">GluArgCysTyrSerGluAlaSerAspLeuLysValIleHisTrpAsnSer</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">239</td> <td class="description-td">ProLysLysLeuArgValLysAsnLysHisValGluPhePheArgAsnPhe</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">256</td> <td class="description-td">TyrLeuThrPheLeuGLuTyrAspGlyAsnLeuLeuArgArgGluLeuPhe</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">273</td> <td class="description-td">ValCysProSerGlnProProProGlyAlaGluGlnLeuGlnGlnAlaLeu</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">290</td> <td class="description-td">AlaGlnLeuAspGluGluAspProCysPheGluPheArgGlnGlnGlnLeu</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">307</td> <td class="description-td">ThrValHisArgValHisValThrPheLeuProHisGluProProProPro</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">324</td> <td class="description-td">ArgProHisAspValThrLeuValAlaGlnLeuSerMetAspArgLeuGln</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">341</td> <td class="description-td">MetLeuGluAlaLeuCysArgHisTrpProGlyProMetSerLeuAlaLeu</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">358</td> <td class="description-td">TyrLeuThrAspAlaGluAlaGlnGlnPheLeuHisPheValGluAlaSer</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">375</td> <td class="description-td">ProValLeuAlaAlaArgGlnAspValAlaTyrHisValValTyrArgGlu</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">392</td> <td class="description-td">GlyProLeUTyrProValAsnGlnLeuArgAsnValAlaLeuAlaGlnAla</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">409</td> <td class="description-td">LeuThrproTyrValPheLeuSerAsplleAspPheLEuProAlaTyrSer</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">426</td> <td class="description-td">LeuTyrAspTyrLeuArgAlaSerIleGluGlnLeuGlyLeuGlySerArg</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">443</td> <td class="description-td">ArgLysAlaAlaLeuValValProAlaPheGluThrLeuArgTyrArgPhe</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">460</td> <td class="description-td">SerPheproHisSerLysValGluLeuLeuAlaLeuLeuAspAlaGlyThr</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">477</td> <td class="description-td">LeuTyrThrPheArgTyrHisGluTrpProArgGlyEisAlaProThrAsp</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">494</td> <td class="description-td">TyrAlaArgTrpArgGluAlaGLnAlaProTyrArgValGlnTrpAlaAla</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">511</td> <td class="description-td">AsnTyrGluProTyrValValValProArgAspCysProArgTyrAspPro</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">528</td> <td class="description-td">ArgPheValGlyPheGlyTrpAsnLysValAlaHislleValGluLeuAsp</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">545</td> <td class="description-td">AlaGlnGluTyrGluLeuLeuValLeuProGluAlaPheThrIleHisLeu</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">562</td> <td class="description-td">ProHisAlaProSerLeuAsplleSerArgPheArgSerSerProThrTyr</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">579</td> <td class="description-td">ArgAspCysLeuGlnAlaLeuLysAspGluPheHisGlnAspLeuSerArg</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">596</td> <td class="description-td">HisHisGlyAlaAlaAlaLeuLysTyrLeuProAlaLeuGlnGlnProGln</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">613</td> <td class="description-td">SerProAlaArgGly</td> <td class="description-td">(SEQ ID NO.: 6)</td> </tr> <tr class="description-tr"> <td align="center" class="description-td" colspan="4" nameend="4" namest="1" rowsep="1"> </td> </tr> </tbody></table> </patent-tables> </tables>
</div>
</li> <li> <para-num num="[0041]"> </para-num> <div class="description-line" id="P-0041" num="0041">The disclosed NOV3 nucleic acid sequence has a high degree of homology (100% identity) with a region of a human BAC genomic clone containing sequence from chromosome Y (CHR Y) (GenBank Accession No.:AC006366.3), as shown in Table 7. Also, a NOV3 nucleic acid sequence has a high degree of homology (100% identity) with another uncharacterized human BAC genomic clone corresponding to the chromosome Y (CHR Y) (GenBank Accession No.:AC010153.3), as is shown in Table 8. A NOV3 polypeptide has homology (68% identity, 78% similarity) with a region of an acetylglucosaminyltransferase-like-protein (GLY T; Accession No.: PD042423) as shown in Table 9. The NOV3 polypeptide also has homology (68% identity, 78% similarity) with an uncharacterized human protein KIAA0609 (KIAA; Accession No.: AB0111181), as shown in Table 10. A Pfam search of a NOV3 polypeptide indicates that NOV3 is a member of the acetylglucosaminyltransferase family (glycosyl tranferase family 8), as shown in Table 11, where a NOV3 polypeptide is aligned with a known acetylglucosaminyltransferase (GLY T; Accession No.: PF01501). The glycosyl transferase family 8 includes enzymes that transfer sugar residues to donor molecules. Members of this family are involved in lipopolysaccharide biosynthesis and glycogen synthesis. This family includes lipopolysaccharide galactosyltransferase, lipopolysaccharide glucosyltransferase 1, and human glycogenin glucosyltransferase.  
<span style="display: none">[0041] </span> <tables id="TABLE-US-00007" num="7"> <patent-tables colsep="0" frame="none" pgwide="1" rowsep="0" tabstyle="monospace"> <table align="left" class="description-table" cols="6" colsep="0" rowsep="0" width="100%"> <thead> <tr class="description-tr"> <td align="center" class="description-td" colspan="6" nameend="6" namest="1">TABLE 7</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td align="center" class="description-td" colspan="6" nameend="6" namest="1" rowsep="1"> </td> </tr> </thead> <tbody><tr class="description-tr"> <td class="description-td">NOV3:</td> <td class="description-td">1</td> <td class="description-td">taaaaatacaaaaaattagccgggcgtagtggcgggegcctgtagtcccagctacttggg</td> <td class="description-td">60</td> <td class="description-td"> </td> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td">||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||</td> </tr> <tr class="description-tr"> <td class="description-td">CHR Y:</td> <td class="description-td">78289 taaaaatacaaaaaattagccgggcgtagtggcgggcgcctgtagtcccagctacttggg</td> <td class="description-td">78348</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">NOV3:</td> <td class="description-td">61</td> <td class="description-td">aggctgaggcaggagaatggcgtgaacccgggaggcagagcttgcagtgagccgagatcc</td> <td class="description-td">120</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td">||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||</td> </tr> <tr class="description-tr"> <td class="description-td">CHR Y:</td> <td class="description-td">78349</td> <td class="description-td">aggctgaggcaggagaatggegtgaacccgggaggcagagcttgca9tgagccgagatcc</td> <td class="description-td">78408</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">NOV3:</td> <td class="description-td">121</td> <td class="description-td">cgccactgcactccagcctgggcgacagagcgagactccgtctc</td> <td class="description-td">164</td> <td class="description-td">(SEQ ID NO.: 18)</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td">|||||||||||||||||||||||||||||||||||||||||||||</td> </tr> <tr class="description-tr"> <td class="description-td">CHR Y:</td> <td class="description-td">78409</td> <td class="description-td">cgccactgcactccagcctgggcgacagagcgagactccgtctc</td> <td class="description-td">78452</td> <td class="description-td">(SEQ ID NO.: 19)</td> </tr> <tr class="description-tr"> <td align="center" class="description-td" colspan="6" nameend="6" namest="1" rowsep="1"> </td> </tr> </tbody></table> </patent-tables> </tables>
</div>
</li> <li> <para-num num="[0042]"> </para-num> <div class="description-line" id="P-0042" num="0042"> <span style="display: none">[0042]</span> <tables id="TABLE-US-00008" num="8"> <patent-tables colsep="0" frame="none" pgwide="1" rowsep="0" tabstyle="monospace"> <table align="left" class="description-table" cols="6" colsep="0" rowsep="0" width="100%"> <thead> <tr class="description-tr"> <td align="center" class="description-td" colspan="6" nameend="6" namest="1">TABLE 8</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td align="center" class="description-td" colspan="6" nameend="6" namest="1" rowsep="1"> </td> </tr> </thead> <tbody><tr class="description-tr"> <td class="description-td">NOV3:</td> <td class="description-td">1</td> <td class="description-td">taaaaatacaaaaaattagccgggcgtagtggcgggcgcctgtagtcccagctacttggg</td> <td class="description-td">60</td> <td class="description-td"> </td> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td">||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||</td> </tr> <tr class="description-tr"> <td class="description-td">CHR Y:</td> <td class="description-td">26694</td> <td class="description-td">taaaaatacaaaaaattagccgggcgtagtggcgggcgcctgtagtcccagctacttggg</td> <td class="description-td">26753</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">NOV3:</td> <td class="description-td">61</td> <td class="description-td">aggctgaggcaggagaatggcgtgaacccgggaggcagagcttgcagtgagccgagatcc</td> <td class="description-td">120</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td">||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||</td> </tr> <tr class="description-tr"> <td class="description-td">CHR Y:</td> <td class="description-td">26754</td> <td class="description-td">aggctgaggcaggagaatggcgtgaacccgggaggcagagcttgcagtgagccgagatcc</td> <td class="description-td">26813</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">NOV3:</td> <td class="description-td">121</td> <td class="description-td">cgccactgcactccagcctgggcgacagagogagactccgtctc</td> <td class="description-td">164</td> <td class="description-td">(SEQ ID NO.: 20)</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td">||||||||||||||||||||||||||||||||||||||||||||</td> </tr> <tr class="description-tr"> <td class="description-td">CHR Y:</td> <td class="description-td">26814</td> <td class="description-td">cgccactgcactccagcctgggcgacagagcgagactccgtctc</td> <td class="description-td">26857</td> <td class="description-td">(SEQ ID NO.: 21)</td> </tr> <tr class="description-tr"> <td align="center" class="description-td" colspan="6" nameend="6" namest="1" rowsep="1"> </td> </tr> </tbody></table> </patent-tables> </tables> </div>
</li> <li> <para-num num="[0043]"> </para-num> <div class="description-line" id="P-0043" num="0043"> <span style="display: none">[0043]</span> <tables id="TABLE-US-00009" num="9"> <patent-tables colsep="0" frame="none" pgwide="1" rowsep="0" tabstyle="monospace"> <table align="left" class="description-table" cols="6" colsep="0" rowsep="0" width="100%"> <thead> <tr class="description-tr"> <td align="center" class="description-td" colspan="6" nameend="6" namest="1">TABLE 9</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td align="center" class="description-td" colspan="6" nameend="6" namest="1" rowsep="1"> </td> </tr> </thead> <tbody><tr class="description-tr"> <td class="description-td">NOV3:</td> <td class="description-td">22</td> <td class="description-td">HLHLVTDAVARNILETLFHTWMVPAIDPVSPYHADQLKPQVSWIPNKHYSGLYGLMKLVL</td> <td class="description-td">81</td> <td class="description-td"> </td> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td">* **+ *++*  ** *** ******+  *  *+**+** +***********+********</td> </tr> <tr class="description-tr"> <td class="description-td">GLY T:</td> <td class="description-td">169</td> <td class="description-td">HFHLIADSIAEQILATLFQTWMVPAVR-VDFYNADELKSEVSWIPNKHYSGIYGLMKLVL</td> <td class="description-td">227</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">NOV3:</td> <td class="description-td">82</td> <td class="description-td">PNALPAELARVIVLDTDVTFASDISELWALFAEFSDTQAIGLVENQSDWYLGNLWLNHRP</td> <td class="description-td">141</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td">   *** * ********+***+**+****+*  *   *+*************** ****</td> </tr> <tr class="description-tr"> <td class="description-td">GLY T:</td> <td class="description-td">228</td> <td class="description-td">TKTLPANLERVIVLDTDITFATDIAELWAVFHKFKGQQVLGLVENQSDWYLGNLWKNHRP</td> <td class="description-td">287</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">NOV3:</td> <td class="description-td">142</td> <td class="description-td">WPALGRGFNTGVILLRLDRLRQAGWEQMWRLTARRELLSLPATSLADQDIFNAVIKEEPG</td> <td class="description-td">201</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td">********+******* **+**+ ********* ***+ + +**************++*</td> </tr> <tr class="description-tr"> <td class="description-td">GLY T:</td> <td class="description-td">288</td> <td class="description-td">WPALGRGYNTGVILLLLDKLRKMKWEQMWRLTAERELMGMLSTSLADQDIFNAVIKQNPF</td> <td class="description-td">347</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">NOV3:</td> <td class="description-td">202</td> <td class="description-td">LVQRLPCVWNVQLSDHTLAERCYSEASDLKVIHWNSPKKLRVKNKEVEFFRNFYLTFLEY</td> <td class="description-td">261</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td">** +*** ********* +*+** + ************************** *******</td> </tr> <tr class="description-tr"> <td class="description-td">GLY T:</td> <td class="description-td">348</td> <td class="description-td">LVYQLPCFWNVQLSDHTRSEQCYRDVSDLKVIHWNSPKKLRVKNKHVEFFRNLYLTFLEY</td> <td class="description-td">407</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">NOV3:</td> <td class="description-td">262</td> <td class="description-td">DGNLLRRELFVCPSQPPPGXXXXXXXXXXXXXXXPCFEFRQQQLTVERVHVTFLXXXXX-</td> <td class="description-td">320</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td">********** ***+                    *+***+++ **** *+**</td> </tr> <tr class="description-tr"> <td class="description-td">GLY T:</td> <td class="description-td">408</td> <td class="description-td">DGNLLRRELFGCPSETDVNNENLQKQLSELDEDDLCYEFRRERFTVERTHLYFLHYEFEP</td> <td class="description-td">467</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">NOV3:</td> <td class="description-td">321</td> <td class="description-td">XXXXXDVTLVAQLSMDRLQMLEALCRHTPGPMSLALYLTDAEAQQFLHFVEASPVLAARQ</td> <td class="description-td">380</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td">      ******************+*+ **+******+******** + + * ** +**</td> </tr> <tr class="description-tr"> <td class="description-td">GLY T:</td> <td class="description-td">468</td> <td class="description-td">SADNTDVTLVAQLSMDRLQMLEAICKEWEGPISLALYLSDAEAQQFLRYAQGSEVLMSRQ</td> <td class="description-td">527</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">NOV3:</td> <td class="description-td">381</td> <td class="description-td">DVAYHVVYREGPLYPVNQLRNVALAQALTFYVFLSDIDFLPAYSLYDYLRASIEQLGLGS</td> <td class="description-td">440</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td">+* **+**+**  **** *****    + ***+********* * **+*** *+ ** * +</td> </tr> <tr class="description-tr"> <td class="description-td">GLY T:</td> <td class="description-td">528</td> <td class="description-td">NVGYHIVYKEGQFYPVNLLRNVAMKEISTPYMFLSDIDFLPMYGLYEYLRKSVIQLDLAN</td> <td class="description-td">587</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">NOV3:</td> <td class="description-td">441</td> <td class="description-td">RRKAALWPAFETLRYRFSFPESKVELLALLDAGTL</td> <td class="description-td">476</td> <td class="description-td">(SEQ ID NO.: 22)</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td"> +* *++********** *** ** ***++** ***</td> </tr> <tr class="description-tr"> <td class="description-td">GLY T:</td> <td class="description-td">588</td> <td class="description-td">TKK-ANIVPAFETLRYRLSFPKSKAELLSMLDMGTL</td> <td class="description-td">622</td> <td class="description-td">(SEQ ID NO.: 23)</td> </tr> <tr class="description-tr"> <td align="center" class="description-td" colspan="6" nameend="6" namest="1" rowsep="1"> </td> </tr> <tr class="description-tr"> <td align="left" class="description-td" colspan="6" nameend="6" namest="1"> </td> </tr> </tbody></table> </patent-tables> </tables> </div>
</li> <li> <para-num num="[0044]"> </para-num> <div class="description-line" id="P-0044" num="0044"> <span style="display: none">[0044]</span> <tables id="TABLE-US-00010" num="10"> <patent-tables colsep="0" frame="none" pgwide="1" rowsep="0" tabstyle="monospace"> <table align="left" class="description-table" cols="6" colsep="0" rowsep="0" width="100%"> <thead> <tr class="description-tr"> <td align="center" class="description-td" colspan="6" nameend="6" namest="1">TABLE 10</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td align="center" class="description-td" colspan="6" nameend="6" namest="1" rowsep="1"> </td> </tr> </thead> <tbody><tr class="description-tr"> <td class="description-td">NOV3:</td> <td class="description-td">22</td> <td class="description-td">HLHLVTDAVARNILETLFHTWMVPAIDPVSPYHADQLKPQVSWIPNKHYSGLYGLMKLVL</td> <td class="description-td">81</td> <td class="description-td"> </td> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td">* **+ *++* ** *** ******+ * *+**+** +***********+********</td> </tr> <tr class="description-tr"> <td class="description-td">GLY T:</td> <td class="description-td">234</td> <td class="description-td">IIFHLIADSIAEQILATLFQTWMVPAVR-VDFYNADELKSEVSWIPNKHYSGIYGLMKLVL</td> <td class="description-td">292</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">NOV3:</td> <td class="description-td">82</td> <td class="description-td">PNALPAELARVIVLDTDVTFASDISELWALFAHFSDTQAIGLVENQSDWYLGNLWLNHRP</td> <td class="description-td">141</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td">   *** * ********+***+**+****+* * * +*************** ****</td> </tr> <tr class="description-tr"> <td class="description-td">GLY T:</td> <td class="description-td">293</td> <td class="description-td">TKTLPANLERVIVLDTDITFATDIAELWAVFHKFKGQQVLGLVENQSDWYLGNLWKNHRP</td> <td class="description-td">352</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">NOV3:</td> <td class="description-td">142</td> <td class="description-td">WPALGRGFNTGVILLRLDRLRQAGWEQMWRLTARRELLSLPATSLADQDIFNAVIKEHPG</td> <td class="description-td">201</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td">********+******* **+**+ ******** ***+++**************++*</td> </tr> <tr class="description-tr"> <td class="description-td">GLY T:</td> <td class="description-td">353</td> <td class="description-td">WPALGRGYNTGVILLLLDKLRKMKWEQMWRLTAERELMGMLSTSLADQDIFNAVIKQNPF</td> <td class="description-td">412</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">NOV3:</td> <td class="description-td">202</td> <td class="description-td">LVQRLPCVWNVQLSDHTLAERCYSEASDLKVIHWNSPKKLRVKNKHVEFFRNFYLTFLEY</td> <td class="description-td">261</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td">** +*** ********* +*+** + ************************** *******</td> </tr> <tr class="description-tr"> <td class="description-td">GLY T:</td> <td class="description-td">413</td> <td class="description-td">LVYQLPCFWNVQLSDHTRSEQCYRDVSDLKVIHWNSPKXLRVKNKHVEFFRNLYLTFLEY</td> <td class="description-td">472</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">NOV3:</td> <td class="description-td">262</td> <td class="description-td">DGNLLRRELFVCPSQPPPGXXXXXXXXXXXXXXXPCFEFRQQQLTVHRVHVTFL-XXXXX</td> <td class="description-td">320</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td">********** ***+                    *+***+++ **** *+ **</td> </tr> <tr class="description-tr"> <td class="description-td">GLY T:</td> <td class="description-td">473</td> <td class="description-td">DGNLLRRELFGCPSEADVNSENLQKQLSELDEDDLCYEFRRERFTVHRTHLYFLHYEYEP</td> <td class="description-td">532</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">NOV3:</td> <td class="description-td">321</td> <td class="description-td">XXXXXDVTLVAQLSMDRLQMLEALCRHTPGPMSLALYLTDAEAQQFLHFVEASPVLAARQ</td> <td class="description-td">380</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td">     ******************+*+* **+******+******** + + * **</td> </tr> <tr class="description-tr"> <td class="description-td">GLY T:</td> <td class="description-td">533</td> <td class="description-td">AADSTDVTLVAQLSMDRLQMLEAICKHWEGPISLALYLSDAEAQQFLRYAQGSEVLMSRH</td> <td class="description-td">592</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">NOV3:</td> <td class="description-td">381</td> <td class="description-td">DVAYHVVYREGPLYPVNQLRNVALAQALTPYVFLSDIDFLPAYSLYDYLRASIEQLGLGS</td> <td class="description-td">440</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td">+* **+**+** **** *****+ ***+********* * **+*** *+** * +</td> </tr> <tr class="description-tr"> <td class="description-td">GLY T:</td> <td class="description-td">593</td> <td class="description-td">NVGYHIVYKEGQFYPVNLLRNVAMKHISTPYMFLSDIDFLPMYGLYEYLRKSVIQLDLAN</td> <td class="description-td">652</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">NOV3:</td> <td class="description-td">441</td> <td class="description-td">RRKAALVVPAFETLRYRFSFPHSKVELLALLDAGTLYTFRYGEWPRGHAPTDYARWREAQ</td> <td class="description-td">500</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td"> +* *++********** *** ** ***++** ***+**** * +*****++*+** *</td> </tr> <tr class="description-tr"> <td class="description-td">GLY T:</td> <td class="description-td">653</td> <td class="description-td">TKK-AMIVPAFETLRYRLSFPKSKAELLSMLDMGTLFTFRYHVWTKGHAPTNFAKWRTAT</td> <td class="description-td">711</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">NOV3:</td> <td class="description-td">501</td> <td class="description-td">APYRVQWAANYEPYVVVPRDCPRYDPRFVGFGWNKVAEIVELDAQEYELLVLPEAFTIHL</td> <td class="description-td">560</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td"> ****+* *++****** **** ** *************+*** **** +*** *+ **+</td> </tr> <tr class="description-tr"> <td class="description-td">GLY T:</td> <td class="description-td">712</td> <td class="description-td">TPYRVEWEADFEPYVVVRRDCPEYDRRFVGFGWNKVAHIMELDVQEYEFIVLPNAYMIHM</td> <td class="description-td">771</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">NOV3:</td> <td class="description-td">561</td> <td class="description-td">PHAPSLDISRFRSSPTYRDCLQALKDEFHQDLSRHHGAAALKYLPA</td> <td class="description-td">606</td> <td class="description-td">(SEQ ID NO.: 24)</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td">***** **++***+ ** **+ **+** **+** +* ****** *</td> </tr> <tr class="description-tr"> <td class="description-td">GLY T:</td> <td class="description-td">772</td> <td class="description-td">PHAPSFDITKFRSNKQYRICLKTLKEEFQQDMSRRYGFAALKYLTA</td> <td class="description-td">817</td> <td class="description-td">(SEQ ID NO.: 25)</td> </tr> <tr class="description-tr"> <td align="center" class="description-td" colspan="6" nameend="6" namest="1" rowsep="1"> </td> </tr> <tr class="description-tr"> <td align="left" class="description-td" colspan="6" nameend="6" namest="1"> </td> </tr> </tbody></table> </patent-tables> </tables> </div>
</li> <li> <para-num num="[0045]"> </para-num> <div class="description-line" id="P-0045" num="0045"> <span style="display: none">[0045]</span> <tables id="TABLE-US-00011" num="11"> <patent-tables colsep="0" frame="none" pgwide="1" rowsep="0" tabstyle="monospace"> <table align="left" class="description-table" cols="6" colsep="0" rowsep="0" width="100%"> <thead> <tr class="description-tr"> <td align="center" class="description-td" colspan="6" nameend="6" namest="1">TABLE 11</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td align="center" class="description-td" colspan="6" nameend="6" namest="1" rowsep="1"> </td> </tr> </thead> <tbody><tr class="description-tr"> <td class="description-td">GLY T</td> <td class="description-td">434</td> <td class="description-td">LSELDEDDLCYEFRRERFTVHRTHLYFLHYEYEPAADSTDVTLVAQLSMD</td> <td class="description-td">483</td> <td class="description-td"> </td> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td">*++***+* *+***+++ **** *+** +*  *      ***********</td> </tr> <tr class="description-tr"> <td class="description-td">NOV3</td> <td class="description-td">288</td> <td class="description-td">LAQLDEEDPCFEFRQQQLTVHRVEVTFLPHEPPPPRPH-DVTLVAQLSMD</td> <td class="description-td">336</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">GLY T</td> <td class="description-td">484</td> <td class="description-td">RLQMLEAICKHWEGPISLALYLSDAEAQQFLRYAQGSEVLMSRHNVGYHI</td> <td class="description-td">533</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td">********+*+* **+******+******** + + * ** +* +* **+</td> </tr> <tr class="description-tr"> <td class="description-td">NOV3</td> <td class="description-td">337</td> <td class="description-td">RLQMLEALCRHTPGPMSLALYLTDAEAQQFLHFVEASPVLAARQDVAYHV</td> <td class="description-td">386</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">GLY T</td> <td class="description-td">534</td> <td class="description-td">VYKEGQFYPVNLLRNVAMKHISTPYMFLSDIDFLPMYGLYEYLRKSVIQL</td> <td class="description-td">583</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td">**+**  **** *****+    *** ********* * **+*** *+ **</td> </tr> <tr class="description-tr"> <td class="description-td">NOV3</td> <td class="description-td">387</td> <td class="description-td">VYREGPLYPVNQLRNVALAQALTPYVFLSDIDFLPAYSLYDYLRASIEQL</td> <td class="description-td">436</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">GLY T</td> <td class="description-td">584</td> <td class="description-td">DLANTKKA-MIVPAFETLRYRLSFPKSKAELLSMLDMGTL</td> <td class="description-td">622</td> <td class="description-td">(SEQ ID NO.: 26)</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td"> * + +** ++********** *** ** ***++** ***</td> </tr> <tr class="description-tr"> <td class="description-td">NOV3</td> <td class="description-td">437</td> <td class="description-td">GLGSRRKAALVVPAFETLRYRFSFPHSKVELLALLDAGTL</td> <td class="description-td">476</td> <td class="description-td">(SEQ ID NO.: 27)</td> </tr> <tr class="description-tr"> <td align="center" class="description-td" colspan="6" nameend="6" namest="1" rowsep="1"> </td> </tr> </tbody></table> </patent-tables> </tables> </div>
</li> <li> <para-num num="[0046]"> </para-num> <div class="description-line" id="P-0046" num="0046">NOV3 is a new member of the acetylglucosaminyltransferase family. NOV3 can be used to detect pituitary gland tissue and is useful in determining changes in expression of genes associated with physiological and pathological pituitary gland activity. Loss of acetylglucosaminyltransferase activity is associated with decrease in tumor growth and metastasis (See e.g. Gransovsky et al., 2000 <span style="display: none">[0046] </span> <i>Nature Medicine </i>6(3): 306). NOV3 is useful in diagnosing and measuring carcinogenic and metastatic onset and progression. NOV3 provides new diagnostic or therapeutic compositions useful in the treatment of disorders associated with alterations in the expression of members of pituitary gland and acetylglucosaminyltransferase protein families. NOV3 nucleic acids, polypeptides, antibodies, and other compositions of the present invention are useful in the treatment and/or diagnosis of a variety of diseases and pathologies, including by way of nonlimiting example, those involving growth retardation and cancer. </div>
</li> <li> <para-num num="[0047]"> </para-num> <div class="description-line" id="P-0047" num="0047">NOV4 <span style="display: none">[0047] </span> </div>
</li> <li> <para-num num="[0048]"> </para-num> <div class="description-line" id="P-0048" num="0048">A NOV4 sequence according to the invention includes the nucleic acid shown in Table 12. The disclosed NOV4 nucleic acid was isolated from mammary gland tissue. The disclosed nucleic acid (SEQ ID NO: 7) is 812 nucleotides in length and contains an open reading frame (ORF) that begins with an ATG initiation codon at nucleotides 250-252 and ends with a TGA stop codon at nucleotides 646-648, as is shown in Table 12. The nucleic acid sequence includes a Kozak consensus sequence. The representative ORF encodes a 132 amino acid polypeptide (SEQ ID NO: 8). The predicted MW of the NOV4 polypeptide is 13348 Da. The PSORT program predicts localization in the mitochondrial matrix space with a moderate certainty of 0.4635. Putative untranslated regions upstream and downstream of the coding sequence are underlined in SEQ ID NO: 7.  
<span style="display: none">[0048] </span> <tables id="TABLE-US-00012" num="12"> <patent-tables colsep="0" frame="none" pgwide="1" rowsep="0" tabstyle="monospace"> <table align="left" class="description-table" cols="3" colsep="0" rowsep="0" width="100%"> <thead> <tr class="description-tr"> <td align="center" class="description-td" colspan="3" nameend="3" namest="1">TABLE 12</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td align="center" class="description-td" colspan="3" nameend="3" namest="1" rowsep="1"> </td> </tr> </thead> <tbody><tr class="description-tr"> <td class="description-td">1</td> <td class="description-td"> <u>TTTTTTTTGGCAAAACCCCATTTCTACTAAAAATACAAAAATTAGCTGGG</u> </td> <td class="description-td">(SEQ ID NO.:7)</td> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">51</td> <td class="description-td"> <u>TACCGTGGTGCACACAGATAATCCCAGCTACTCAGGAGGATGAGGCAGGA</u> </td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">101</td> <td class="description-td"> <u>GAATCGCTTTAAAATGGGAGGCGGAGGCTGTAGTGAGCCAAGATTGTGCC</u> </td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">151</td> <td class="description-td"> <u>ACTGCACTCCAGCCTGGGCAACAAAGTGAGACTCTTATCTTACAAGAAAA</u> </td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">201</td> <td class="description-td"> <u>AAAAGAATGCTTAGGAATCAACTCCCCTCCTAATGCCCAGACAGGTGAAA</u> </td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">251</td> <td class="description-td">TGACTTGCTCAGGCCACAGTTCTGCCCAAGCCAGCACCAGCCAGTGGCAC</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">301</td> <td class="description-td">AGCAGAATGCAAGGAGGGAGTGGTACCTACTTTCACAGCATAGGTGCTGC</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">351</td> <td class="description-td">CGGGGTCCTCAGAGCAGGTGGCACAGTAATAAATGGCATCCCCCGAGTCA</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">401</td> <td class="description-td">CAGCAGGGCTTGTTACAAGTCAGCTTGAAGAGCGACCAGTTATTCTCATT</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">451</td> <td class="description-td">GAAGTGGAGCTCCTTTTTCTGGCCGCCCATGAAGAGGTCCTCACATTTGG</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">501</td> <td class="description-td">CTACAAGGCGGGCCAGGGACTGGGTGTAGAGTCCCCCCAGCTTGGCATAG</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">551</td> <td class="description-td">GTGCCCTCCTTGCTGCTGATGTTGCTCAGGAGACCGTGCAGCTGAAGCTG</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">601</td> <td class="description-td">GGTGCTCCCGGTGGAGGCTTGACTGGACACGCTCAGCTGGGAGGATGAGG</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">651</td> <td class="description-td"> <u>CGGAGGGGGCCCCTTTGCACTGGAGGCCAGGGCTCCCGCAGCCGCCTCTG</u> </td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">701</td> <td class="description-td"> <u>TTGCCCGCCAGCCCTGCTGCCGGAAGCCAGTGGCCCTGCCTTTGAAAGCT</u> </td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">751</td> <td class="description-td"> <u>TTCTCCAGAGGATCGGAAGAGGAGAAGACAGGATGGTGGCGGTTCCCGGG</u> </td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">801</td> <td class="description-td"> <u>GCTCACCGAAATA</u> </td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">1</td> <td class="description-td">MetThrCysSerGlyHisSerSerAlaGlnAlaSerThrSerGlnTrpHis</td> <td class="description-td">(SEQ ID NO.:8)</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">18</td> <td class="description-td">SerArgMetGlnGlyGlySerGlyThrTyrPheHisSerIleGlyAlaAla</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">35</td> <td class="description-td">GlyValLeuArgAlaGlyGlyThrValIleAsnGlyIleProArgValThr</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">52</td> <td class="description-td">AlaGlyLeuValThrSerGlnLeuGluGluArgProValIleLeuIleGlu</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">69</td> <td class="description-td">ValGluLeuLeupheLeuAlaAlaHisGluGluValLeuThrPheGlyTyr</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">86</td> <td class="description-td">LysAlaGlyGlnGlyLeuGlyValGluSerProGlnLeuGlyIleGlyAla</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">103</td> <td class="description-td">LeuLeuAlaAlaAspValAlaGlnGluThrValGlnLeuLysLeuGlyAla</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">120</td> <td class="description-td">ProGlyGlyGlyLeuThrGlyHisAlaGlnLeuGlyGly</td> </tr> <tr class="description-tr"> <td align="center" class="description-td" colspan="3" nameend="3" namest="1" rowsep="1"> </td> </tr> </tbody></table> </patent-tables> </tables>
</div>
</li> <li> <para-num num="[0049]"> </para-num> <div class="description-line" id="P-0049" num="0049">A NOV4 nucleic acid has homology with a section of human chromosome 21 q DNA (CHR 21; Accession No.: AP001695.1), as is shown in Table 13.  
<span style="display: none">[0049] </span> <tables id="TABLE-US-00013" num="13"> <patent-tables colsep="0" frame="none" pgwide="1" rowsep="0" tabstyle="monospace"> <table align="left" class="description-table" cols="4" colsep="0" rowsep="0" width="100%"> <thead> <tr class="description-tr"> <td align="center" class="description-td" colspan="4" nameend="4" namest="1">TABLE 13</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td align="center" class="description-td" colspan="4" nameend="4" namest="1" rowsep="1"> </td> </tr> </thead> <tbody><tr class="description-tr"> <td class="description-td">NOV4: 13</td> <td class="description-td">aaaccccatttctactaaaaatacaaaaattagctgggtaccgtggtgcacacagataat</td> <td class="description-td">72</td> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">||||||||||||||||||||||||||||||||||||||||||||||||||||||</td> </tr> <tr class="description-tr"> <td class="description-td">CHR 21: 3708</td> <td class="description-td">aaaccccgtctctactaaaaatacaaaaattagctgggtgtggtggtgcgcgcctataat</td> <td class="description-td">3767</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">NOV4: 73</td> <td class="description-td">cccagctactcaggaggatgaggcaggagaatcgctttaaaatgggaggcggaggctgta</td> <td class="description-td">132</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">||||||||||||||||||||||||||||||||  |||||  |||||||||||||||||</td> </tr> <tr class="description-tr"> <td class="description-td">CHR 21: 3768</td> <td class="description-td">cccagctactcaggaggctgaggcaggagaatcatttgaaaccgggaggcggaggctgta</td> <td class="description-td">3827</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">NOV4: 133</td> <td class="description-td">gtgagccaagattgtgccactgcactccagcctgggcaacaaagtgagactct</td> <td class="description-td">185 (SEQ ID NO.: 28)</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">|||||||||||||||||||||||||||||||||||||||||||||||||</td> </tr> <tr class="description-tr"> <td class="description-td">CHR 21: 3828</td> <td class="description-td">gtgagccgagatcgtgccactgcattccagcctgggcaacagagtgagactct</td> <td class="description-td">3880 (SEQ ID NO.: 29)</td> </tr> <tr class="description-tr"> <td align="center" class="description-td" colspan="4" nameend="4" namest="1" rowsep="1"> </td> </tr> </tbody></table> </patent-tables> </tables>
</div>
</li> <li> <para-num num="[0050]"> </para-num> <div class="description-line" id="P-0050" num="0050">NOV4 is useful as a marker for the region of human chromosome 21 q. Also, NOV4 can be used to detect mammary gland tissue, and is useful in determining changes in expression of genes associated with physiological and pathological mammary gland activity, e.g breast cancer and lactation mastitis. NOV4 provides new diagnostic and therapeutic compositions useful in the treatment of disorders associated with alterations in the expression of members of mammary gland-associated protein families. NOV4 nucleic acids, polypeptides, antibodies, and other compositions of the present invention are useful in the treatment and/or diagnosis of a variety of diseases and pathologies, including by way of nonlimiting example, those involving bacterial infection and cancer. <span style="display: none">[0050] </span> </div>
</li> <li> <para-num num="[0051]"> </para-num> <div class="description-line" id="P-0051" num="0051">NOV5 <span style="display: none">[0051] </span> </div>
</li> <li> <para-num num="[0052]"> </para-num> <div class="description-line" id="P-0052" num="0052">A NOV5 sequence according to the invention includes the nucleic acid shown in Table <span style="display: none">[0052] </span> <b>14</b>. The disclosed nucleic acid encodes a polypeptide related to a neural membrane protein. The disclosed NOV5 nucleic acid are present in adrenal, mammary, prostate, testis, uterus, bone marrow, melanoma, pituitary, thyroid, spleen, placenta, bone marrow, mammary gland, fetal thymus—CRL7046, osteogenic sarcoma cell lines—HTB96, fetal lung, thalamus, fetal kidney, and Burkitt's lymphoma (Raji). Expressed sequence tag (EST) data suggest NOV5 sequences are expressed in aorta, blood, bone, brain, breast, central nervous system, colon, foreskin, germ cell, heart, kidney, larynx, lung, lymph, muscle, ovary, pancreas, parathyroid, placenta, pooled, prostate, stomach, testis, tonsil, uterus, whole embryo, blood, breast, cervix, colon, head and neck, lung, ovary, and stomach. The disclosed nucleic acid (SEQ ID NO:9) is 2,059 nucleotides in length and contains an open reading frame (ORF) that begins with an ATG initiation codon at nucleotides 63-65 and ends with a TGA stop codon at nucleotides 1,022-1,024, as is shown in Table 14. The nucleic acid sequence includes a Kozak consensus sequence. The representative ORF encodes a 320 amino acid polypeptide (SEQ ID NO: 9). The predicted MW of the NOV5 polypeptide is 35,204.3 Da. Putative untranslated regions upstream and downstream of the coding sequence are underlined in SEQ ID NO: 9. </div>
</li> <li> <para-num num="[0053]"> </para-num> <div class="description-line" id="P-0053" num="0053">NOV5 is predicted to be a neural membrane protein or a proline-rich membrane protein. The PSORT algorithm predicts NOV5 polypeptide localization to the plasma membrane with a certainty of 0.60.  
<span style="display: none">[0053] </span> <tables id="TABLE-US-00014" num="14"> <patent-tables colsep="0" frame="none" pgwide="1" rowsep="0" tabstyle="monospace"> <table align="left" class="description-table" cols="4" colsep="0" rowsep="0" width="100%"> <thead> <tr class="description-tr"> <td align="center" class="description-td" colspan="4" nameend="4" namest="1">TABLE 14</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td align="center" class="description-td" colspan="4" nameend="4" namest="1" rowsep="1"> </td> </tr> </thead> <tbody><tr class="description-tr"> <td class="description-td">1</td> <td class="description-td"> <u>CGCTCCGTCTGGAACGGCGCAGGTCCCAGCAGCTGGGGTTCCCCCTCAGC</u> </td> <td class="description-td">(SEQ ID NO.: 9)</td> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">51</td> <td class="description-td"> <u>CCGTGAGCAGCC</u>ATGTCCAACCCCAGCGCCCCACCACCATATGAAGACCG</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">101</td> <td class="description-td">CAACCCCCTGTACCCAGGCCCTCTGCCCCCTGGGGGCTATGGGCAGCCAT</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">151</td> <td class="description-td">CTGTCCTGCCAGGAGGGTATCCTGCCTACCCTGGCTACCCGCAGCCTGGC</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">201</td> <td class="description-td">TACGGTCACCCTGCTGGCTACCCACAGCCCATGCCCCCCACCCACCCGAT</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">251</td> <td class="description-td">GCCcATGAACTACGGCCCAGGCCATGGCTATGATGGGGAGGAGAGAGCGG</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">301</td> <td class="description-td">TGAGTGATAGCTTCGGGCCTGGAGAGTGGGATGACCGGAAAGTGCGACAC</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">351</td> <td class="description-td">ACTTTTATCCGAAAGGTTTACTCCATCATCTCCGTGCAGCTGCTCATCAC</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">401</td> <td class="description-td">TGTGGCCATCATTGCTATCTTCACCTTTGTGGAACCTGTCAGCGCCTTTG</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">451</td> <td class="description-td">TGAGGAGAAATGTGGCTGTCTACTACGTGTCCTATGCTGTCTTCGTTGTC</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">501</td> <td class="description-td">ACCTACCTGATCCTTGCCTGCTGCCAGGGACCCAGACGCCGTTTCCCATG</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">551</td> <td class="description-td">GAACATCATTCTGCTGACCCTTTTTACTTTTGCCATGGGCTTCATGACGG</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">601</td> <td class="description-td">GCACCATTTCCAGTATGTACCAAACCAAAGCCGTCATCATTGCAATGATC</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">651</td> <td class="description-td">ATCACTGCGGTGGTATCCATTTCAGTCACCATCTTCTGCTTTCAGACCAA</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">701</td> <td class="description-td">GGTGAGGGCATGGAGGGCCCTTCCCTGGCCCCCCGACTCCCCTTTCTTAT</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">751</td> <td class="description-td">CAGGCCCGGACCCCGGTACACTAGGGATGTTCCCTACAOACCTGATCCCC</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">801</td> <td class="description-td">TTCTCCTCATCCGCACCTACAAAACTGTGTCCTGTTTCTGTCCTTAGAAT</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">851</td> <td class="description-td">GTTGTGGACATTCCCATACCCCCTACGAGGCAGCACTGGGACTCCCTGGC</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">901</td> <td class="description-td">AGGGCCAGTCTGACTGGGCTGGTTGTCACAGCCATCTGACAGGTGCCTCT</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">951</td> <td class="description-td">TTCTTGCTTCCTGGCAGGTGGACTTCACCTCGTGCACAGGCCTCTTCTGT</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">1001</td> <td class="description-td">GTCCTGGGAATTGTGCTCCTGG<u>TGACTGGGATTGTCACTAGCATTGTGCT</u> </td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">1051</td> <td class="description-td"> <u>CTTAGCATTGTGCTCTACTTCCAATACGTTTACTGGCTCCACATGCTCTA</u> </td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">1101</td> <td class="description-td"> <u>TGCTGCTCTGGGGGCCATTTGTTTCACCCTGTTCCTGGCTTACGACACAC</u> </td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">1151</td> <td class="description-td"> <u>AGCTGGTCCTGGGGAACCGGAAGCACACCATCAGCCCCGAGGACTACATC</u> </td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">1201</td> <td class="description-td"> <u>ACTGGCGCCCTGCAGATTTACACAGACATCATCTACATCTTCACCTTTGT</u> </td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">1251</td> <td class="description-td"> <u>GCTGCAGCTGATGGCGGATCGCAATTAAGGAGCAAGCCCCCATTTTCACC</u> </td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">1301</td> <td class="description-td"> <u>CGATCCTGGGCTCTCCCTTCCAAGCTAGAGGGCTOGGCCCTATGACTGTG</u> </td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">1351</td> <td class="description-td"> <u>GTCTGGGCTTTAGGCCCCTTTCCTTCCCCTTGAGTAACATGCCCAGTTTC</u> </td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">1401</td> <td class="description-td"> <u>CTTTCTGTCCTGGAGACAGGTGGCCTCTCTGGCTATGGATGTGTGGGTAC</u> </td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">1451</td> <td class="description-td"> <u>TTGGTGGGGACGGAGGAGCTAGGGACTAACTGTTGCTCTTGGTGGGCTTG</u> </td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">1501</td> <td class="description-td"> <u>GCAGGGACTAGGCTGAAGATGTGTCTTCTCCCCGCCACCTACTGTATGAC</u> </td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">1551</td> <td class="description-td"> <u>ACCACATTCTTCCTAACAGCTGGGGTTGTGAGGAATATGAAAAGAGCCTA</u> </td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">1601</td> <td class="description-td"> <u>TTCGATAGCTAGAAGGGAATATGAAAGGTAGAAGTGACTTCAAGGTCACG</u> </td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">1651</td> <td class="description-td"> <u>AGGTTCCCCTCCCACCTCTGTCACAGGCTTCTTGACTACGTAGTTGGAGC</u> </td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">1701</td> <td class="description-td"> <u>TATTTCTTCCCCCAGCAAAGCCAGAGAGCTTTGTCCCCGGCCTCCTGGAC</u> </td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">1751</td> <td class="description-td"> <u>ACATAGGCCATTATCCTGTATTCCTTTGGCTTGGCATCTTTTAGCTCAGG</u> </td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">1801</td> <td class="description-td"> <u>AAGGTAGAAGAGATCTGTGCCCATGGGTCTCCTTGCTTCAATCCCTTCTT</u> </td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">1851</td> <td class="description-td"> <u>GTTTCAGTGACATATGTATTGTTTATCTGGGTTAGGGATGGGGGACAGAT</u> </td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">1901</td> <td class="description-td"> <u>AATAGAACGAGCAAAGTAACCTATACAGGCCAGCATGGAACAGCATCTCC</u> </td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">1951</td> <td class="description-td"> <u>CCTGGGCTTGCTCCTGGCTTCTGACGCTATAAGACAGAGCAGGCCACATG</u> </td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">2001</td> <td class="description-td"> <u>TGGCCATCTGCTCCCCATTCTTGAAAGCTGCTGGGGCCTCCTTGCAGGCT</u> </td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">2051</td> <td class="description-td"> <u>TCTGGATCC</u> </td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">1</td> <td class="description-td">MetSerAsnProSerAlaProProProTyrGluAspArgAsnProLeuTyr</td> <td class="description-td">(SEQ ID NO. 10)</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">18</td> <td class="description-td">ProGlyProLeuProProGlyGlyTyrGlyGlnProSerValLeuProGly</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">35</td> <td class="description-td">GlyTyrProAlaTyrProGlyTyrProGlnProGlyTyrGlyHisProAla</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">52</td> <td class="description-td">GlyTyrProGlnProMetProProTHrHisProMetProMetAsnTyrGly</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">69</td> <td class="description-td">ProGlyHisGlyTyrAspGlyGluGluArgAlaValSerAspSerPheGly</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">86</td> <td class="description-td">ProGlyGluTrpAspAspArgLysValArgHisThrPheIleArgLysVal</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">103</td> <td class="description-td">TyrSerIleIleSerValGlnLeuLeuIleThrValAlaIleIleAlaIle</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">120</td> <td class="description-td">PheThrPheValGluProValSerAlaPheValArgArgAsnValAlaVal</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">137</td> <td class="description-td">TyrTyrValSerTyrAlaValPheValValThrTyrLeuIleLeuAlaCys</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">154</td> <td class="description-td">CysGlnGlyProArgArgArgPheProTrpAsnIleIleLeuLeuThrLeu</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">171</td> <td class="description-td">PheThrPheAlaMetGlyPheMetThrGlyThrIleSerSerMetTyrGln</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">188</td> <td class="description-td">ThrLysAlaValIleIlEAlaMetIleIleThrAlaValValSerIleSer</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">205</td> <td class="description-td">ValThrIlePheCysPheGlnThrLysValArgAlaTrpArgAlaLEuPro</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">222</td> <td class="description-td">TrpProProAspSerProPheLeuSerGlyProAspProGlyThrLeuGly</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">239</td> <td class="description-td">MetPheProArgAspLeuIleProPheSerSerSerAlaProThrLysLeu</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">256</td> <td class="description-td">CysProValSerValLeuArgMetLeuTrpThrPheProTyrProLeuGly</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">273</td> <td class="description-td">GlySerThrGlyThrProTrpGlnGlyGlnSerAspTrpAlaGlyCysHis</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">290</td> <td class="description-td">SerHisLeuThrGlyAlaSerPheLeuLeuProGlyArgTrpThrSerPro</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">307</td> <td class="description-td">ArgAlaGlnAlaSerSerValSerTrpGluLeuCysSerTrp</td> </tr> <tr class="description-tr"> <td align="center" class="description-td" colspan="4" nameend="4" namest="1" rowsep="1"> </td> </tr> </tbody></table> </patent-tables> </tables>
</div>
</li> <li> <para-num num="[0054]"> </para-num> <div class="description-line" id="P-0054" num="0054">The disclosed NOV5 nucleic acid has a high degree of homology (100% identity) with a region of the gene encoding the uncharacterized human PP1201 protein (PP1201; Genbank Accession No.: NM022152.1), as is shown in Table 15. Also, the NOV5 nucleic acid has a high degree of homology (99% identity) with regions of the human BAC genomic clone RP11-378A13 from chromosome 2 (Genbank Accession No.: AC021016.4; CHR 2), as is shown in Table 16. Furthermore, the NOV5 nucleic acid has a high degree of homology (100% identity) with portions of a polynucleotide sequence from U.S. Pat. No. 5,843,716 (Seq2; Accession No.: <span style="display: none">[0054] </span> </div>
</li> <li> <para-num num="[0055]"> </para-num> <div class="description-line" id="P-0055" num="0055">AR062278), as is shown in Table 17. Still further, the NOV5 nucleic acid has homology (70% similarity, 83% identity) with the rat neural membrane protein 35 (NMP 35; Accession No.: AAC 324631.1), as shown in Table 18.  
<span style="display: none">[0055] </span> <tables id="TABLE-US-00015" num="15"> <patent-tables colsep="0" frame="none" pgwide="1" rowsep="0" tabstyle="monospace"> <table align="left" class="description-table" cols="4" colsep="0" rowsep="0" width="100%"> <thead> <tr class="description-tr"> <td align="center" class="description-td" colspan="4" nameend="4" namest="1">TABLE 15</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td align="center" class="description-td" colspan="4" nameend="4" namest="1" rowsep="1"> </td> </tr> </thead> <tbody><tr class="description-tr"> <td class="description-td">NOV5: 1</td> <td class="description-td">cgctccgtctggaacggcgcaggtcccagcagctggggttccccctcagcccgtgagcag</td> <td class="description-td">60</td> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||</td> </tr> <tr class="description-tr"> <td class="description-td">PP1201: 5</td> <td class="description-td">cgctccgtctggaacggcgcaggtcccagcagctggggttccccctcagcccgtgagcag</td> <td class="description-td">64</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">NOV5: 61</td> <td class="description-td">ccatgtccaaccccagcgccccaccaccatatgaagaccgcaaccccctgtacccaggcc</td> <td class="description-td">120</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||</td> </tr> <tr class="description-tr"> <td class="description-td">PP1201: 65</td> <td class="description-td">ccatgtccaaccccagcgccccaccaccatatgaagaccgcaaccccctgtacccaggcc</td> <td class="description-td">124</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">NOV5: 121</td> <td class="description-td">ctctgccccctgggggctatgggcagccatctgtcctgccaggagggtatcctgcctacc</td> <td class="description-td">180</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||</td> </tr> <tr class="description-tr"> <td class="description-td">PP1201: 125</td> <td class="description-td">ctctgccccctgggggctatgggcagccatctgtcctgccaggagggtatcctgcctacc</td> <td class="description-td">184</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">NOV5: 181</td> <td class="description-td">ctggctacccgcagcctggctacggtcaccctgctggctacccacagcccatgcccccca</td> <td class="description-td">240</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||</td> </tr> <tr class="description-tr"> <td class="description-td">PP1201: 185</td> <td class="description-td">ctggctacccgcagcctggctacggtcaccctgctggctacccacagcccatgcccccca</td> <td class="description-td">244</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">NOV5: 241</td> <td class="description-td">cccacacgatgcccatgaactacggcccaggccatggctatgatggggaggagagagcgg</td> <td class="description-td">300</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||</td> </tr> <tr class="description-tr"> <td class="description-td">PP1201: 245</td> <td class="description-td">cccacccgatgcccatgaactacggcccaggccatggctatgatggggaggagagagcgg</td> <td class="description-td">304</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">NOV5: 301</td> <td class="description-td">tgagtgatagcttcgggcctggagagtgggatgaccggaaagtgcgacacacttttatcc</td> <td class="description-td">360</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||</td> </tr> <tr class="description-tr"> <td class="description-td">PP1201: 305</td> <td class="description-td">tgagtgatagcttcgggcctggagagtgggatgaccggaaagtgcgacacacttttatcc</td> <td class="description-td">364</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">NOV5: 361</td> <td class="description-td">gaaaggtttactccatcatctccgtgcagctgctcatcactgtggccatcattgctatct</td> <td class="description-td">420</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||</td> </tr> <tr class="description-tr"> <td class="description-td">PP1201: 365</td> <td class="description-td">gaaaggtttactccatcatctccgtgcagctgctcatcactgtggccatcattgctatct</td> <td class="description-td">424</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">NOV5: 421</td> <td class="description-td">tcacctttgtggaacctgtcagcgcctttgtgaggagaaatgtggctgtctactacgtgt</td> <td class="description-td">480</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||</td> </tr> <tr class="description-tr"> <td class="description-td">PP1201: 425</td> <td class="description-td">tcacctttgtggaacctgtcagcgcctttgtgaggagaaatgtggctgtctactacgtgt</td> <td class="description-td">484</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">NOV5: 481</td> <td class="description-td">cctatgctgtcttcgttgtcacctacctgatccttgcctgctgccagggacccagacgcc</td> <td class="description-td">540</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||</td> </tr> <tr class="description-tr"> <td class="description-td">PP1201: 485</td> <td class="description-td">cctatgctgtcttcgttgtcacctacctgatccttgcctgctgccagggacccagacgcc</td> <td class="description-td">544</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">NOV5: 541</td> <td class="description-td">gtttcccatggaacatcattctgctgaccctttttacttttgccatgggcttcatgacgg</td> <td class="description-td">600</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||</td> </tr> <tr class="description-tr"> <td class="description-td">PP1201: 545</td> <td class="description-td">gtttcccatggaacatcattctgctgaccctttttacttttgccatgggcttcatgacgg</td> <td class="description-td">604</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">NOV5: 601</td> <td class="description-td">gcaccatttccagtatgtaccaaaccaaagccgtcatcattgcaatgatcatcactgcgg</td> <td class="description-td">660</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||</td> </tr> <tr class="description-tr"> <td class="description-td">PP1201: 605</td> <td class="description-td">gcaccatttccagtatgtaccaaaccaaagccgtcatcattgcaatgatcatcactgcgg</td> <td class="description-td">664</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">NOV5: 661</td> <td class="description-td">tggtatccatttcagtcaccatcttctgctttcagaccaaggtg</td> <td class="description-td">704 (SEQ ID NO.: 30)</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">||||||||||||||||||||||||||||||||||||||||||||</td> </tr> <tr class="description-tr"> <td class="description-td">PP1201: 665</td> <td class="description-td">tggtatccatttcagtcaccatcttctgctttcagaccaaggtg</td> <td class="description-td">708 (SEQ ID NO.: 31)</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">NOV5: 1053</td> <td class="description-td">tagcattgtgctctacttccaatacgtttactggctccacatgctctatgctgctctggg</td> <td class="description-td">1112</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||</td> </tr> <tr class="description-tr"> <td class="description-td">PP1201: 777</td> <td class="description-td">tagcattgtgctctacttccaatacgtttactggctccacatgctctatgctgctctggg</td> <td class="description-td">836</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">NOV5: 1113</td> <td class="description-td">ggccatttgtttcaccctgttcctggcttacgacacacagctggtcctggggaaccggaa</td> <td class="description-td">1172</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||</td> </tr> <tr class="description-tr"> <td class="description-td">PP1201: 837</td> <td class="description-td">ggccatttgtttcaccctgttcctggcttacgacacacagctggtcctggggaaccggaa</td> <td class="description-td">896</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">NOV5: 1173</td> <td class="description-td">gcacaccatcagccccgaggactacatcactggcgccctgcagatttacacagacatcat</td> <td class="description-td">1232</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||</td> </tr> <tr class="description-tr"> <td class="description-td">PP1201: 897</td> <td class="description-td">gcacaccatcagccccgaggactacatcactggcgccctgcagatttacacagacatcat</td> <td class="description-td">956</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">NOV5: 1233</td> <td class="description-td">ctacatcttcacctttgtgctgcagctgatgggggatcgcaattaaggagcaagccccca</td> <td class="description-td">1292</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||</td> </tr> <tr class="description-tr"> <td class="description-td">PP1201: 957</td> <td class="description-td">ctacatcttcacctttgtgctgcagctgatgggggatcgcaattaaggagcaagccccca</td> <td class="description-td">1016</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">NOV5: 1293</td> <td class="description-td">ttttcacccgatcctgggctctcccttccaagctagagggctgggccctatgactgtggt</td> <td class="description-td">1352</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||</td> </tr> <tr class="description-tr"> <td class="description-td">PP1201: 1017</td> <td class="description-td">ttttcacccgatcctgggctctcccttccaagctagagggctgggccctatgactgtggt</td> <td class="description-td">1076</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">NOV5: 1353</td> <td class="description-td">ctgggctttaggcccctttccttccccttgagtaacatgcccagtttcctttctgtcctg</td> <td class="description-td">1412</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||</td> </tr> <tr class="description-tr"> <td class="description-td">PP1201: 1077</td> <td class="description-td">ctgggctttaggcccctttccttccccttgagtaacatgcccagtttcctttctgtcctg</td> <td class="description-td">1136</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">NOV5: 1413</td> <td class="description-td">gagacaggtggcctctctggctatggatgtgtgggtacttggtggggacggaggagctag</td> <td class="description-td">1472</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||</td> </tr> <tr class="description-tr"> <td class="description-td">PP1201: 1137</td> <td class="description-td">gagacaggtggcctctctggctatggatgtgtgggtacttggtggggacggaggagctag</td> <td class="description-td">1196</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">NOV5: 1473</td> <td class="description-td">ggactaactgttgctcttggtgggcttggcagggactaggctgaagatgtgtcttctccc</td> <td class="description-td">1532</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||</td> </tr> <tr class="description-tr"> <td class="description-td">PP1201: 1197</td> <td class="description-td">ggactaactgttgctcttggtgggcttggcagggactaggctgaagatgtgtcttctccc</td> <td class="description-td">1256</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">NOV5: 1533</td> <td class="description-td">cgccacctactgtatgacaccacattcttcctaacagctggggttgtgaggaatatgaaa</td> <td class="description-td">1592</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||</td> </tr> <tr class="description-tr"> <td class="description-td">PP1201: 1257</td> <td class="description-td">cgccacctactgtatgacaccacattcttcctaacagctggggttgtgaggaatatgaaa</td> <td class="description-td">1316</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">NOV5: 1593</td> <td class="description-td">agagcctattcgatagctagaagggaatatgaaaggtagaagtgacttcaaggtcacgag</td> <td class="description-td">1652</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||</td> </tr> <tr class="description-tr"> <td class="description-td">PP1201: 1317</td> <td class="description-td">agagcctattcgatagctagaagggaatatgaaaggtagaagtgacttcaaggtcacgag</td> <td class="description-td">1376</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">NOV5: 1653</td> <td class="description-td">gttcccctcccacctctgtcacaggcttcttgactacgtagttggagctatttcttcccc</td> <td class="description-td">1712</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||</td> </tr> <tr class="description-tr"> <td class="description-td">PP1201: 1377</td> <td class="description-td">gttcccctcccacctctgtcacaggcttcttgactacgtagttggagctatttcttcccc</td> <td class="description-td">1436</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">NOV5: 1713</td> <td class="description-td">cagcaaagccagagagctttgtccccggcctcctggacacataggccattatcctgtatt</td> <td class="description-td">1772</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||</td> </tr> <tr class="description-tr"> <td class="description-td">PP1201: 1437</td> <td class="description-td">cagcaaagccagagagctttgtccccggcctcctggacacataggccattatcctgtatt</td> <td class="description-td">1496</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">NOV5: 1773</td> <td class="description-td">cctttggcttggcatcttttagctcaggaaggtagaagagatctgtgcccatgggtctcc</td> <td class="description-td">1832</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||</td> </tr> <tr class="description-tr"> <td class="description-td">PP1201: 1497</td> <td class="description-td">cctttggcttggcatcttttagctcaggaaggtagaagagatctgtgcccatgggtctcc</td> <td class="description-td">1556</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">NOV5: 1833</td> <td class="description-td">ttgcttcaatcccttcttgtttcagtgacatatgtattgtttatctgggttagggatggg</td> <td class="description-td">1892</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||</td> </tr> <tr class="description-tr"> <td class="description-td">PP1201: 1557</td> <td class="description-td">ttgcttcaatcccttcttgtttcagtgacatatgtattgtttatctgggttagggatggg</td> <td class="description-td">1616</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">NOV5: 1893</td> <td class="description-td">ggacagataatagaacgagcaaagtaacctatacaggccagcatggaacagcatctcccc</td> <td class="description-td">1952</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||</td> </tr> <tr class="description-tr"> <td class="description-td">PP1201: 1617</td> <td class="description-td">ggacagataatagaacgagcaaagtaacctatacaggccagcatggaacagcatctcccc</td> <td class="description-td">1676</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">NOV5: 1953</td> <td class="description-td">tgggcttgctcctggcttgtgacgctataagacagagcaggccacatgtggccatctgct</td> <td class="description-td">2012</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||</td> </tr> <tr class="description-tr"> <td class="description-td">PP1201: 1677</td> <td class="description-td">tgggcttgctcctggcttgtgacgctataagacagagcaggccacatgtggccatctgct</td> <td class="description-td">1736</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">NOV5: 2013</td> <td class="description-td">ccccattcttgaaagctgctggggcctccttgcaggcttctggatc</td> <td class="description-td">2058 (SEQ ID NO. 32)</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">||||||||||||||||||||||||||||||||||||||||||||||</td> </tr> <tr class="description-tr"> <td class="description-td">PP1201: 1737</td> <td class="description-td">ccccattcttgaaagctgctggggcctccttgcaggcttctggatc</td> <td class="description-td">1782 (SEQ ID NO. 33)</td> </tr> <tr class="description-tr"> <td align="center" class="description-td" colspan="4" nameend="4" namest="1" rowsep="1"> </td> </tr> </tbody></table> </patent-tables> </tables>
</div>
</li> <li> <para-num num="[0056]"> </para-num> <div class="description-line" id="P-0056" num="0056"> <span style="display: none">[0056]</span> <tables id="TABLE-US-00016" num="16"> <patent-tables colsep="0" frame="none" pgwide="1" rowsep="0" tabstyle="monospace"> <table align="left" class="description-table" cols="4" colsep="0" rowsep="0" width="100%"> <thead> <tr class="description-tr"> <td align="center" class="description-td" colspan="4" nameend="4" namest="1">TABLE 16</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td align="center" class="description-td" colspan="4" nameend="4" namest="1" rowsep="1"> </td> </tr> </thead> <tbody><tr class="description-tr"> <td class="description-td">NOV5: 1131</td> <td class="description-td">gttcctggcttacgacacacagctggtcctggggaaccggaagcacaccatcagccccga</td> <td class="description-td">1190</td> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||</td> </tr> <tr class="description-tr"> <td class="description-td">CHR 2: 21496</td> <td class="description-td">gttcctggcttacgacacacagctggtcctggggaaccggaagcacaccatcagccccga</td> <td class="description-td">21437</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">NOV5: 1191</td> <td class="description-td">ggactacatcactggcgccctgcagatttacacagacatcatctacatcttcacctttgt</td> <td class="description-td">1250</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||</td> </tr> <tr class="description-tr"> <td class="description-td">CHR 2: 21436</td> <td class="description-td">ggactacatcactggcgccctgcagatttacacagacatcatctacatcttcacctttgt</td> <td class="description-td">21377</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">NOV5: 1251</td> <td class="description-td">gctgcagctgatgggggatcgcaattaaggagcaagcccccattttcacccgatcctggg</td> <td class="description-td">1310</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||</td> </tr> <tr class="description-tr"> <td class="description-td">CHR 2: 21376</td> <td class="description-td">gctgcagctgatgggggatcgcaattaaggagcaagcccccattttcacccgatcctggg</td> <td class="description-td">21317</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">NOV5: 1311</td> <td class="description-td">ctctcccttccaagctagagggctgggccctatgactgtggtctgggctttaggcccctt</td> <td class="description-td">1370</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||</td> </tr> <tr class="description-tr"> <td class="description-td">CHR 2: 21316</td> <td class="description-td">ctctcccttccaagctagagggctgggccctatgactgtggtctgggctttaggcccctt</td> <td class="description-td">21257</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">NOV5: 1371</td> <td class="description-td">tccttccccttgagtaacatgcccagtttcctttctgtcctggagacaggtggcctctct</td> <td class="description-td">1430</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||</td> </tr> <tr class="description-tr"> <td class="description-td">CHR 2: 21256</td> <td class="description-td">tccttccccttgagtaacatgcccagtttcctttctgtcctggagacagytggcctctct</td> <td class="description-td">21197</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">NOV5: 1431</td> <td class="description-td">ggctatggatgtgtgggtacttggtggggacggaggagctagggactaactgttgctctt</td> <td class="description-td">1490</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||</td> </tr> <tr class="description-tr"> <td class="description-td">CHR 2: 21196</td> <td class="description-td">ggctatggatgtgtgggtacttggtggggacggaggagctagggactaactgttgctctt</td> <td class="description-td">21137</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">NOV5: 1491</td> <td class="description-td">ggtgggcttggcagggactaggctgaagatgtgtcttctccccgccacctactgtatgac</td> <td class="description-td">1550</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||</td> </tr> <tr class="description-tr"> <td class="description-td">CHR 2: 21136</td> <td class="description-td">ggtgggcttggcagggactaggctgaagatgtgtcttctccccgccacctactgtatgac</td> <td class="description-td">21077</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">NOV5: 1551</td> <td class="description-td">accacattcttcctaacagctggggttgtgaggaatatgaaaagagcctattcgatagct</td> <td class="description-td">1610</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||</td> </tr> <tr class="description-tr"> <td class="description-td">CHR 2: 21076</td> <td class="description-td">accacattcttcctaacagctggggttgtgaggaatatgaaaagagcctattcgatagct</td> <td class="description-td">21017</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">NOV5: 1611</td> <td class="description-td">agaagggaatatgaaaggtagaagtgacttcaaggtcacgaggttcccctcccacctctg</td> <td class="description-td">1670</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||</td> </tr> <tr class="description-tr"> <td class="description-td">CHR 2: 21016</td> <td class="description-td">agaagggaatatgaaaggtagaagtgacttcaaggtcacgaggttcccctcccacctctg</td> <td class="description-td">20957</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">NOV5: 1671</td> <td class="description-td">tcacaggcttcttgactacgtagttggagctatttcttcccccagcaaagccagagagct</td> <td class="description-td">1730</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||</td> </tr> <tr class="description-tr"> <td class="description-td">CHR 2: 20956</td> <td class="description-td">tcacaggcttcttgactacgtagttggagctatttcttcccccagcaaagccagagagct</td> <td class="description-td">20897</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">NOV5: 1731</td> <td class="description-td">ttgtccccggcctcctggacacataggccattatcctgtattcctttggcttggcatctt</td> <td class="description-td">1790</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||</td> </tr> <tr class="description-tr"> <td class="description-td">CHR 2: 20896</td> <td class="description-td">ttgtccccggcctcctggacacataggccattatcctgtattcctttggcttggcatctt</td> <td class="description-td">20837</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">NOV5: 1791</td> <td class="description-td">ttagctcaggaaggtagaagagatctgtgcccatgggtctccttgcttcaatcccttctt</td> <td class="description-td">1850</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||</td> </tr> <tr class="description-tr"> <td class="description-td">CHR 2: 20836</td> <td class="description-td">ttagctcaggaaggtagaagagatctgtgcccatgggtctccttgcttcaatcccttctt</td> <td class="description-td">20777</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">NOV5: 1851</td> <td class="description-td">gtttcagtgacatatgtattgtttatctgggttagggatgggggacagataatagaacga</td> <td class="description-td">1910</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||</td> </tr> <tr class="description-tr"> <td class="description-td">CHR 2: 20776</td> <td class="description-td">gtttcagtgacatatgtattgtttatctgggttagggatgggggacagataacagaacga</td> <td class="description-td">20717</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">NOV5: 1911</td> <td class="description-td">gcaaagtaacctatacaggccagcatggaacagcatctcccctgggcttgctcctggctt</td> <td class="description-td">1970</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||</td> </tr> <tr class="description-tr"> <td class="description-td">CHR 2: 20716</td> <td class="description-td">gcaaagtaacctatacaggccagcatggaacagcatctcccctgggcttgctcctggctt</td> <td class="description-td">20657</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">NOV5: 1971</td> <td class="description-td">gtgacgctataagacagagcaggccacatgtggccatctgctccccattattgaaagctg</td> <td class="description-td">2030</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||</td> </tr> <tr class="description-tr"> <td class="description-td">CHR 2: 20656</td> <td class="description-td">gtgacgctataagacagagcaggccacatgtggccatctgctccccattcttgaaagctg</td> <td class="description-td">20597</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">NOV5: 2031</td> <td class="description-td">ctggggcctccttgcaggcttctggatc</td> <td class="description-td">2058 (SEQ ID NO. 34)</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">////////////////////////////</td> </tr> <tr class="description-tr"> <td class="description-td">CHR 2: 20596</td> <td class="description-td">ctggggcctccttgcaggcttctggatc</td> <td class="description-td">20569 (SEQ ID NO. 35)</td> </tr> <tr class="description-tr"> <td align="center" class="description-td" colspan="4" nameend="4" namest="1" rowsep="1"> </td> </tr> </tbody></table> </patent-tables> </tables> </div>
</li> <li> <para-num num="[0057]"> </para-num> <div class="description-line" id="P-0057" num="0057"> <span style="display: none">[0057]</span> <tables id="TABLE-US-00017" num="17"> <patent-tables colsep="0" frame="none" pgwide="1" rowsep="0" tabstyle="monospace"> <table align="left" class="description-table" cols="4" colsep="0" rowsep="0" width="100%"> <thead> <tr class="description-tr"> <td align="center" class="description-td" colspan="4" nameend="4" namest="1">TABLE 17</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td align="center" class="description-td" colspan="4" nameend="4" namest="1" rowsep="1"> </td> </tr> </thead> <tbody><tr class="description-tr"> <td class="description-td">NOV5: 1053</td> <td class="description-td">tagcattgtgctctacttccaatacgtttactggctccacatgctctatgctgctctggg</td> <td class="description-td">1112</td> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||</td> </tr> <tr class="description-tr"> <td class="description-td">Seq2: 924</td> <td class="description-td">tagcattgtgctctacttccaatacgtttactggctccacatgctctatgctgctctggg</td> <td class="description-td">983</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">NOV5: 1113</td> <td class="description-td">ggccatttgtttcaccctgttcctggcttacgacacacagctggtcctggggaaccggaa</td> <td class="description-td">1172</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||</td> </tr> <tr class="description-tr"> <td class="description-td">Seq2: 984</td> <td class="description-td">ggccatttgtttcaccctgttcctggcttacgacacacagctggtcctggggaaccggaa</td> <td class="description-td">1043</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">NOV5: 1173</td> <td class="description-td">gcacaccatcagccccgaggactacatcactggcgccctgcagatttacacagacatcat</td> <td class="description-td">1232</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||</td> </tr> <tr class="description-tr"> <td class="description-td">Seq2: 1044</td> <td class="description-td">gcacaccatcagccccgaggactacatcactggcgccctgcagatttacacagacatcat</td> <td class="description-td">1103</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">NOV5: 1233</td> <td class="description-td">ctacatcttcacctttgtgctgcagctgatgggggatcgcaattaaggagcaagccccca</td> <td class="description-td">1292</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||</td> </tr> <tr class="description-tr"> <td class="description-td">Seq2: 1104</td> <td class="description-td">ctacatcttcacctttgtgctgcagctgatgggggatcgcaattaaggagcaagccccca</td> <td class="description-td">1163</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">NOV5: 1293</td> <td class="description-td">ttttcacccgatcctgggctctcccttccaagctagagggctgggccctatgactgtggt</td> <td class="description-td">1352</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||</td> </tr> <tr class="description-tr"> <td class="description-td">Seq2: 1164</td> <td class="description-td">ttttcacccgatcctgggctctcccttccaagctagagggctgggccctatgactgtggt</td> <td class="description-td">1223</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">NOV5: 1353</td> <td class="description-td">ctgggctttaggcccctttccttccccttgagtaacatgcccagtttcctttctgtcctg</td> <td class="description-td">1412</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||</td> </tr> <tr class="description-tr"> <td class="description-td">Seq2: 1224</td> <td class="description-td">ctgggctttaggcccctttccttccccttgagtaacatgcccagtttcctttctgtcctg</td> <td class="description-td">1283</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">NOV5: 1413</td> <td class="description-td">gagacaggtggcctctctggctatggatgtgtgggtacttggtggggacggaggagctag</td> <td class="description-td">1472</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||</td> </tr> <tr class="description-tr"> <td class="description-td">Seq2: 1284</td> <td class="description-td">gagacaggtggcctctctggctatggatgtgtgggtacttggtggggacggaggagctag</td> <td class="description-td">1343</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">NOV5: 1473</td> <td class="description-td">ggactaactgttgctcttggtgggcttggcagggactaggctgaagatgtgtcttctccc</td> <td class="description-td">1532</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||</td> </tr> <tr class="description-tr"> <td class="description-td">Seq2: 1344</td> <td class="description-td">ggactaactgttgctcttggtgggcttggcagggactaggctgaagatgtgtcttctccc</td> <td class="description-td">1403</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">NOV5: 1533</td> <td class="description-td">cgccacctactgtatgacaccacattcttcctaacagctggggttgtgaggaatatgaaa</td> <td class="description-td">1592</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||</td> </tr> <tr class="description-tr"> <td class="description-td">Seq2: 1404</td> <td class="description-td">cgccacctactgtatgacaccacattcttcctaacagctggggttgtgaggaatatgaaa</td> <td class="description-td">1463</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">NOV5: 1593</td> <td class="description-td">agagcctattcgatagctagaagggaatatgaaaggtagaagtgacttcaaggtcacgag</td> <td class="description-td">1652</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||</td> </tr> <tr class="description-tr"> <td class="description-td">Seq2: 1464</td> <td class="description-td">agagcctattcgatagctagaagggaatatgaaaggtagaagtgacttcaaggtcacgag</td> <td class="description-td">1523</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">NOV5: 1653</td> <td class="description-td">gttcccctcccacctctgtcacaggcttcttgactacgtagttggagctatttcttcccc</td> <td class="description-td">1712</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||</td> </tr> <tr class="description-tr"> <td class="description-td">Seq2: 1524</td> <td class="description-td">gttcccctcccacctctgtcacaggcttcttgactacgtagttggagctatttcttcccc</td> <td class="description-td">1583</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">NOV5: 1713</td> <td class="description-td">cagcaaagccagagagctttgtccccggcctcctggacacataggccattatcctgtatt</td> <td class="description-td">1772</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||</td> </tr> <tr class="description-tr"> <td class="description-td">Seq2: 1584</td> <td class="description-td">cagcaaagccagagagctttgtccccggcctcctggacacataggccattatcctgtatt</td> <td class="description-td">1643</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">NOV5: 1773</td> <td class="description-td">cctttggcttggcatcttttagctcaggaaggtagaagagatctgtgcccatgggtctcc</td> <td class="description-td">1832</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||</td> </tr> <tr class="description-tr"> <td class="description-td">Seq2: 1644</td> <td class="description-td">cctttggcttggcatcttttagctcaggaaggtagaagagatctgtgcccatgggtctcc</td> <td class="description-td">1703</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">NOV5: 1833</td> <td class="description-td">ttgcttcaatcccttcttgtttcagtgacatatgtattgtttatctgggttagggatggg</td> <td class="description-td">1892</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||</td> </tr> <tr class="description-tr"> <td class="description-td">Seq2: 1704</td> <td class="description-td">ttgcttcaatcccttcttgtttcagtgacatatgtattgtttatctgggttagggatggg</td> <td class="description-td">1763</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">NOV5: 1893</td> <td class="description-td">ggacagataatagaacgagcaaagtaacctatacaggccagcatggaacagcatctcccc</td> <td class="description-td">1952</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||</td> </tr> <tr class="description-tr"> <td class="description-td">Seq2: 1764</td> <td class="description-td">ggacagataatagaacgagcaaagtaacctatacaggccagcatggaacagcatctcccc</td> <td class="description-td">1823</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">NOV5: 1953</td> <td class="description-td">tgggcttgctcctggcttgtgacgctataagacagagcaggccacatgtggccatctgct</td> <td class="description-td">2012</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||</td> </tr> <tr class="description-tr"> <td class="description-td">Seq2: 1824</td> <td class="description-td">tgggcttgctcctggcttgtgacgctataagacagagcaggccacatgtggccatctgct</td> <td class="description-td">1883</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">NOV5: 2013</td> <td class="description-td">ccccattcttgaaagctgctggggcctccttgcaggcttctggatc</td> <td class="description-td">2058 (SEQ ID NO. 36)</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||</td> </tr> <tr class="description-tr"> <td class="description-td">Seq2: 1884</td> <td class="description-td">ccccattcttgaaagctgctggggcctccttgcaggcttctggatc</td> <td class="description-td">1929 (SEQ ID NO. 37)</td> </tr> <tr class="description-tr"> <td align="center" class="description-td" colspan="4" nameend="4" namest="1" rowsep="1"> </td> </tr> </tbody></table> </patent-tables> </tables> </div>
</li> <li> <para-num num="[0058]"> </para-num> <div class="description-line" id="P-0058" num="0058"> <span style="display: none">[0058]</span> <tables id="TABLE-US-00018" num="18"> <patent-tables colsep="0" frame="none" pgwide="1" rowsep="0" tabstyle="monospace"> <table align="left" class="description-table" cols="4" colsep="0" rowsep="0" width="100%"> <thead> <tr class="description-tr"> <td align="center" class="description-td" colspan="4" nameend="4" namest="1">TABLE 18</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td align="center" class="description-td" colspan="4" nameend="4" namest="1" rowsep="1"> </td> </tr> </thead> <tbody><tr class="description-tr"> <td class="description-td">NOV 5: 1051</td> <td class="description-td">LSIVLYFQYVYWLHMLYAALGAICFTLFLAYDTQLVLGNRKHTISPEDYITGALQIYTDI</td> <td class="description-td">1230</td> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">*+*+* **** *** +** ***  ******+****++***+*++***+** *** ** **</td> </tr> <tr class="description-tr"> <td class="description-td">NMP 35: 241</td> <td class="description-td">LAILLPFQYVPWLHAVYAVLGAGVFTLFLAFDTQLLMGNRRHSLSPEEYIFGALNIYLDI</td> <td class="description-td">300</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">NOV5: 1231</td> <td class="description-td">IYIFTFVLQLMG</td> <td class="description-td">1266 (SEQ ID NO. 38)</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">****** *** *</td> </tr> <tr class="description-tr"> <td class="description-td">NMP 35: 301</td> <td class="description-td">IYIFTFFLQLFG</td> <td class="description-td">312 (SEQ ID NO. 39)+TZ,1 58 </td> </tr> <tr class="description-tr"> <td align="left" class="description-td" colspan="4" nameend="4" namest="1"> </td> </tr> </tbody></table> </patent-tables> </tables> </div>
</li> <li> <para-num num="[0059]"> </para-num> <div class="description-line" id="P-0059" num="0059">The disclosed NOV5 nucleic acid maps to human chromosome 2 between markers D2S164 and D2S163. The disclosed NOV5 nucleic acid sequence localizes to 2q33-36. The Online Mendelian Inheritance in Man (OMIM) entry for this region indicates the following diseases are associated with this locus. Therefore, this gene may be of therapeutic or diagnostic use in these diseases (numbers indicate OMIM entries for the disorders): Waardenburg syndrome, type 1(3); Waardenburg syndrome, type III, 148820 (3); Rhabdomyosarcoma, alveolar, 268220; Craniofacial-deafness-hand syndrome, 122880 (3); Insulin-dependent diabetes mellitus-12; Amyotrophic lateral sclerosis-2, juvenile recessive; Ehlers-Danlos syndrome, type X; insulin-dependent diabetes mellitus-13; leukemia/lymphoma, T-cell; Bjornstad syndrome (pili torti and deafness); myopathy, desmin-related, cardioskeletal, 601419 (3);cardiomyopathy, dilated, 11; cerebrotendinous xanthomatosis; Finnish lethal neonatal metabolic syndrome; choreoathetosis, familial paroxysmal (Paroxysmal nonkinesiogenic dyskinesia). <span style="display: none">[0059] </span> </div>
</li> <li> <para-num num="[0060]"> </para-num> <div class="description-line" id="P-0060" num="0060">The expression pattern, map location and protein homology information reveals that NOV5 may function as an ion channel and a glutamate receptor, including in diseases associated with this map location (described above). Therefore this gene can serve as a drug target and antibody target as well as a diagnostic marker in these diseases and other tissues that are involved in these diseases. In vitro uses includes tissue regeneration for all these tissues and cell types composing these tissues and cell types derived from these tissues. <span style="display: none">[0060] </span> </div>
</li> <li> <para-num num="[0061]"> </para-num> <div class="description-line" id="P-0061" num="0061">Glutamate receptors are the predominant excitatory neurotransmitter receptors in the mammalian brain and are activated in a variety of normal neurophysiologic processes. The classification of glutamate receptors is based on their activation by different pharmacologic agonists. <span style="display: none">[0061] </span> </div>
</li> <li> <para-num num="[0062]"> </para-num> <div class="description-line" id="P-0062" num="0062">NOV5 is useful as a marker for the region of human chromosome 2 between markers D2S164 and D2S163. Also, NOV5 can be used to detect neural tissue and tissue from adrenal glands, mammary, prostate, testis, uterus, bone marrow, melanoma, pituitary, thyroid, spleen, placenta, bone marrow, mammary gland, fetal thymus, fetal lung, thalamus, fetal kidney, and Burkitt's lymphoma. In addition, reagents that detect NOV5 nucleic acids or polypeptides provide new diagnostic or therapeutic compositions useful in the treatment of disorders associated with alterations in the expression of members of adrenal gland-associated protein families. NOV5 nucleic acids, polypeptides, antibodies, and other compositions of the present invention are useful in the treatment and/or diagnosis of a variety of diseases and pathologies, including by way of nonlimiting example, those involving neurological disorders, aberrant cell proliferation and differentiation, and cancer. <span style="display: none">[0062] </span> </div>
</li> <li> <para-num num="[0063]"> </para-num> <div class="description-line" id="P-0063" num="0063">NOV6 <span style="display: none">[0063] </span> </div>
</li> <li> <para-num num="[0064]"> </para-num> <div class="description-line" id="P-0064" num="0064">A NOV6 sequence according to the invention includes the nucleic acid sequence shown in Table 19. Also shown in Table 19 is a polypeptide encoded by the disclosed nucleic acid sequence. The encoded polypeptide is related to LIM-interacting proteins. The disclosed NOV6 nucleic acid was isolated from pancreas RNA. Other tissues expressing sequences homologous to the NOV6 nucleic acid include spinal cord, stomach, fetal lung, and cell lines U-937 and CADMEC LA. The disclosed nucleic acid (SEQ ID NO:11) is 807 nucleotides in length and contains an open reading frame (ORF) that begins with an ATG initiation codon at nucleotides 207-209 and ends with a TAG stop codon at nucleotides 612-614, as shown in Table 19. A Kozak consensus sequence was observed. The representative ORF encodes a 135 amino acid polypeptide (SEQ ID NO: 12). The predicted MW of the NOV6 polypeptide is 14,712.5 Da. Putative untranslated regions upstream and downstream of the coding sequence are underlined in SEQ ID NO: 11.  
<span style="display: none">[0064] </span> <tables id="TABLE-US-00019" num="19"> <patent-tables colsep="0" frame="none" pgwide="1" rowsep="0" tabstyle="monospace"> <table align="left" class="description-table" cols="3" colsep="0" rowsep="0" width="100%"> <thead> <tr class="description-tr"> <td align="center" class="description-td" colspan="3" nameend="3" namest="1">TABLE 19</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td align="center" class="description-td" colspan="3" nameend="3" namest="1" rowsep="1"> </td> </tr> </thead> <tbody><tr class="description-tr"> <td class="description-td">1</td> <td class="description-td">CCAAGATACAAGCTGGAGTATTTAGGAGTGATATCTATAACTAGCACTCA</td> <td class="description-td">(SEQ ID NO.: 11)</td> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">51</td> <td class="description-td">AGTTATTCAACAAAATTCGACCTGGACGAGACCCTGGTGCACAGCTCCTT</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">101</td> <td class="description-td">CAAGCCAGTGAACAACGCGGACTTCATCATCCCTGTGGAGATTGATGGGG</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">151</td> <td class="description-td">TGGTCCACCAGGTCTACGTGTTGAAGCGTCCTCACGTGGATGAGTTCCTG</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">201</td> <td class="description-td">CAGCGAATGGGCGAGCTCTTTGAATGTGTGCTGTTCACTGCTAGCCTCGC</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">251</td> <td class="description-td">CAAGTACGCAGACCCAGTAGCTGACCTGCTGGACAAATGGGGGGCCTTCC</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">301</td> <td class="description-td">GGOCCCGGCTGTTTCGAGAGTCCTGCGTCTTCCACCGGGGGAACTACGTG</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">351</td> <td class="description-td">AAGGACCTGAGCCGGTTGGGTCOAGACCTGCGGCGGGTGCTCATCCTGGA</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">401</td> <td class="description-td">CAATTCACCTGCCTCCTATGTCTTCCATCCAGACAATGCTGTGAGTGCGG</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">451</td> <td class="description-td">GCTGGACTGGGACTGGGACAGGAGCTGAGACCCAGGAAGGGGTCAGTCCA</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">501</td> <td class="description-td">TTCAGGCCACCTTGGCCTCTTGGATCCCCAGTTGGGGGGTGGGTGCCCTC</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">551</td> <td class="description-td">CCAGTCCTTCCTGCATTCATTGCCTGTGCCTGCCGCCCACTCCCCTCATC</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">601</td> <td class="description-td">CACCTGCCCTGTAGCCATATGGTCTTTTCCCCTCGCACAAAGCAGAGCAT</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">651</td> <td class="description-td">CTGCCATGCACAGGGGCCCCCACAGGGCAACGGAGTTTGGAAAGTTTCAA</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">701</td> <td class="description-td">TTTTTCGAATTGCCAGTTGTGACCTACTGATGGCCCACAGAATTAATTTA</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">751</td> <td class="description-td">GTGGGTTCTGATTGGGAATTTTAACAAAATGAAATAGAATAGAAAATATC</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">801</td> <td class="description-td">CAAAAAA</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">1</td> <td class="description-td">MetGlyGluLeuPheGluCysValLeuPheThrAlaSerLeuAlaLysTyr</td> <td class="description-td">(SEQ ID NO.: 12)</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">18</td> <td class="description-td">AlaAspProValAlaAspLeuLeuAspLysTrpGlyAlaPheArgAlaArg</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">35</td> <td class="description-td">LeuPheArgGluSerCysValPheHisArgGlyAsnTyrValLysAspLeu</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">52</td> <td class="description-td">SerArgLeuGlyArgAspLeuArgArgValLeuIleLeuAspAspSerPro</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">69</td> <td class="description-td">AlaSerTyrValPheHisProAspAsnAlaValSerAlaGlyTrpThrGly</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">86</td> <td class="description-td">ThrGlyThrGlyAlaGluThrGlnGluGlyValSerProPheArgProPro</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">103</td> <td class="description-td">TrpProLeuGlySerProValGlyGlyTrpValProSerGlnSerPheLeu</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">120</td> <td class="description-td">HisSerLeuProValProAlaAlaHisSerProHisProProAlaLeu</td> </tr> <tr class="description-tr"> <td align="center" class="description-td" colspan="3" nameend="3" namest="1" rowsep="1"> </td> </tr> </tbody></table> </patent-tables> </tables>
</div>
</li> <li> <para-num num="[0065]"> </para-num> <div class="description-line" id="P-0065" num="0065">Regions of the disclosed NOV6 nucleic acid has a high degree of homology (100% identity) with regions of the human BAC genomic clone RP11-378A13 from chromosome 2 (CHR 2; Genbank Accession No.: AC021016.4), as shown in Table 20. Also, the NOV6 nucleic acid has a high degree of homology (100% identity) with regions of the human SLC11A1 gene, encoding the natural resistance-associated macrophage protein-1 (NRAM 1; Genbank Accession No.: AF229163.1), as is shown in Table 21. Furthermore, the NOV6 nucleic acid has a high degree of homology (100% identity) with regions of the human nuclear LIM-interactor-interacting factor (LIM; Accession No.: XM 002584. 1), as shown in Table 22.  
<span style="display: none">[0065] </span> <tables id="TABLE-US-00020" num="20"> <patent-tables colsep="0" frame="none" pgwide="1" rowsep="0" tabstyle="monospace"> <table align="left" class="description-table" cols="4" colsep="0" rowsep="0" width="100%"> <tbody><tr class="description-tr"> <td class="description-td">TABLE 20+HZ,1 61 </td> </tr> <tr class="description-tr"> <td class="description-td">NOV6: 253</td> <td class="description-td">agtacgcagacccagtagctgacctgctggacaaatggggggccttccgggcccggctgt</td> <td class="description-td">312</td> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||</td> </tr> <tr class="description-tr"> <td class="description-td">CHR 2: 149104</td> <td class="description-td">agtacgcagacccagtagctgacctgctggacaaatggggggccttccgggcccggctgt</td> <td class="description-td">149163</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">NOV6: 313</td> <td class="description-td">ttcgagagtcctgcgtcttccaccgggggaactacgtgaaggacctgagccggttgggtc</td> <td class="description-td">372</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||</td> </tr> <tr class="description-tr"> <td class="description-td">CHR 2: 149164</td> <td class="description-td">ttcgagagtcctgcgtcttccaccgggggaactacgtgaaggacctgagccggttgggtc</td> <td class="description-td">149223</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">NOV6: 373</td> <td class="description-td">gagacctgcggcgggtgctcatcctggacaattcacctgcctcctatgtcttCCatccag</td> <td class="description-td">432</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||</td> </tr> <tr class="description-tr"> <td class="description-td">CHR 2: 149224</td> <td class="description-td">gagacctgcggcgggtgctcatcctggacaattcacctgcctcctatgtcttccatccag</td> <td class="description-td">149283</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">NOV6: 433</td> <td class="description-td">acaatgctgtgagtgcgggctggactgggactgggacaggagctgagacccaggaagggg</td> <td class="description-td">492</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||</td> </tr> <tr class="description-tr"> <td class="description-td">CHR 2: 149284</td> <td class="description-td">acaatgctgtgagtgcgggctggactgggactgggacaggagctgagacccaggaagggg</td> <td class="description-td">149343</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">NOV6: 493</td> <td class="description-td">tcagtccattcaggccaccttggcctcttggatccccagttggggggtgggtgccctccc</td> <td class="description-td">552</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||</td> </tr> <tr class="description-tr"> <td class="description-td">CHR 2: 149344</td> <td class="description-td">tcagtccattcaggccaccttggcctcttggatccccagttggggggtgggtgccctccc</td> <td class="description-td">149403</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">NOV6: 553</td> <td class="description-td">agtccttactgcattcattgcctgtgcctgccgcccactcccctcatccacctgccctgt</td> <td class="description-td">612</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||</td> </tr> <tr class="description-tr"> <td class="description-td">CHR 2: 149404</td> <td class="description-td">agtccttcctgcattcattgcctgtgcctgccgcccactcccctcatccacctgccctgt</td> <td class="description-td">149463</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">NOV6: 613</td> <td class="description-td">agccatatggtcttttcccctcgcacaaagcagagcatctgccatgcacaggggccccca</td> <td class="description-td">672</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||</td> </tr> <tr class="description-tr"> <td class="description-td">CHR 2: 149464</td> <td class="description-td">agccatatggtcttttcccctcgcacaaagcagagcatctgccatgcacaggggccccca</td> <td class="description-td">149523</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">NOV6: 673</td> <td class="description-td">cagggcaacggagtttggaaagtttcaatttttcgaattgccagttgtgacctactgatg</td> <td class="description-td">732</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||</td> </tr> <tr class="description-tr"> <td class="description-td">CHR 2: 149524</td> <td class="description-td">cagggcaacggagtttggaaagtttcaatttttcgaattgccagttgtgacctactgatg</td> <td class="description-td">149583</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">NOV6: 733</td> <td class="description-td">gcccacagaattaatttagtgggttctgattgggaattttaacaaaatgaaatagaatag</td> <td class="description-td">792</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||</td> </tr> <tr class="description-tr"> <td class="description-td">CHR 2: 149584</td> <td class="description-td">gcccacagaattaatttagtgggttctgattgggaattttaacaaaatgaaatagaatag</td> <td class="description-td">149643</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">NOV6: 793</td> <td class="description-td">aaaatatcc</td> <td class="description-td">801 (SEQ ID NO.: 40)</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">|||||||||</td> </tr> <tr class="description-tr"> <td class="description-td">CHR 2: 149644</td> <td class="description-td">aaaatatcc</td> <td class="description-td">149652 (SEQ ID NO.: 41)</td> </tr> <tr class="description-tr"> <td align="center" class="description-td" colspan="4" nameend="4" namest="1" rowsep="1"> </td> </tr> </tbody></table> </patent-tables> </tables>
</div>
</li> <li> <para-num num="[0066]"> </para-num> <div class="description-line" id="P-0066" num="0066"> <span style="display: none">[0066]</span> <tables id="TABLE-US-00021" num="21"> <patent-tables colsep="0" frame="none" pgwide="1" rowsep="0" tabstyle="monospace"> <table align="left" class="description-table" cols="3" colsep="0" rowsep="0" width="100%"> <thead> <tr class="description-tr"> <td align="center" class="description-td" colspan="3" nameend="3" namest="1">TABLE 21</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td align="center" class="description-td" colspan="3" nameend="3" namest="1" rowsep="1"> </td> </tr> </thead> <tbody><tr class="description-tr"> <td class="description-td">NOV6: 253</td> <td class="description-td">agtacgcagacccagtagctgacctgctggacaaatggggggccttccgggcccggctgt</td> <td class="description-td">312</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||</td> </tr> <tr class="description-tr"> <td class="description-td">NRAM1: 27097</td> <td class="description-td">agtacgcagacccagtagctgacctgctggacaaatggggggccttccgggcccggctgt</td> <td class="description-td">27156</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">NOV6: 313</td> <td class="description-td">ttcgagagtcctgcgtcttccaccgggggaactacgtgaaggaactgagccggttgggtc</td> <td class="description-td">372</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||</td> </tr> <tr class="description-tr"> <td class="description-td">NRAM1: 27157</td> <td class="description-td">ttcgagagtcctgcgtcttccaccgggggaactacgtgaaggacctgagccggttgggtc</td> <td class="description-td">27216</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">NOV6:</td> <td class="description-td">373</td> <td class="description-td">gagacctgcggcgggtgctcatcctggacaattcacctgcctcctatgtcttccatccag</td> <td class="description-td">432</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||</td> </tr> <tr class="description-tr"> <td class="description-td">NRAM1: 27217</td> <td class="description-td">gagacctgcggcgggtgctcatcctggacaattcacctgcctcctatgtcttccatccag</td> <td class="description-td">27276</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">NOV6: 433</td> <td class="description-td">acaatgctgtgagtgcgggctggactgggactgggacaggagctgagacccaggaagggg</td> <td class="description-td">492</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||</td> </tr> <tr class="description-tr"> <td class="description-td">NRAM1: 27277</td> <td class="description-td">acaatgctgtgagtgcgggctggactgggactgggacaggagctgagacccaggaagggg</td> <td class="description-td">27336</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">NOV6: 493</td> <td class="description-td">tcagtccattcaggccaccttggcctcttggatccccagttggggggtgggtgccctccc</td> <td class="description-td">552</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||</td> </tr> <tr class="description-tr"> <td class="description-td">NRAM1: 27337</td> <td class="description-td">tcagtccattcaggccaccttggcctcttggatccccagttggggggtgggtgccctccc</td> <td class="description-td">27396</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">NOV6: 553</td> <td class="description-td">agtccttcctgcattcattgcctgtgcctgccgcccactcccctcatccacctgccctgt</td> <td class="description-td">612</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||</td> </tr> <tr class="description-tr"> <td class="description-td">NRAM1: 27397</td> <td class="description-td">agtccttcctgcattcattgcctgtgcctgccgcccactcccctcatccacctgccctgt</td> <td class="description-td">27456</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">NOV6: 613</td> <td class="description-td">agccatatggtcttttcccctcgcacaaagcagagcatctgccatgcacaggggccccca</td> <td class="description-td">672</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||</td> </tr> <tr class="description-tr"> <td class="description-td">NRAM1: 27457</td> <td class="description-td">agccatatggtcttttcccctcgcacaaagcagagcatctgccatgcacaggggccccca</td> <td class="description-td">27516</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">NOV6: 673</td> <td class="description-td">cagggcaacggagtttggaaagtttcaatttttcgaattgccagttgtgacctactgatg</td> <td class="description-td">732</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||</td> </tr> <tr class="description-tr"> <td class="description-td">NRAM1: 27517</td> <td class="description-td">cagggcaacggagtttggaaagtttcaatttttcgaattgccagttgtgacctactgatg</td> <td class="description-td">27576</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">NOV6: 733</td> <td class="description-td">gcccacagaattaatttagtgggttctgattgggaattttaacaaaatgaaatagaatag</td> <td class="description-td">792</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||</td> </tr> <tr class="description-tr"> <td class="description-td">NRAM1: 27577</td> <td class="description-td">gcccacagaattaatttagtgggttctgattgggaattttaacaaaatgaaatagaatag</td> <td class="description-td">27636</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">NOV6: 793</td> <td class="description-td">aaaatatcc</td> <td class="description-td">801 (SEQ ID NO.: 40)</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">|||||||||</td> </tr> <tr class="description-tr"> <td class="description-td">NRAM1: 27637</td> <td class="description-td">aaaatatcc</td> <td class="description-td">27645 (SEQ ID NO.: 42)</td> </tr> <tr class="description-tr"> <td align="center" class="description-td" colspan="3" nameend="3" namest="1" rowsep="1"> </td> </tr> </tbody></table> </patent-tables> </tables> </div>
</li> <li> <para-num num="[0067]"> </para-num> <div class="description-line" id="P-0067" num="0067"> <span style="display: none">[0067]</span> <tables id="TABLE-US-00022" num="22"> <patent-tables colsep="0" frame="none" pgwide="1" rowsep="0" tabstyle="monospace"> <table align="left" class="description-table" cols="4" colsep="0" rowsep="0" width="100%"> <thead> <tr class="description-tr"> <td align="center" class="description-td" colspan="4" nameend="4" namest="1">TABLE 22</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td align="center" class="description-td" colspan="4" nameend="4" namest="1" rowsep="1"> </td> </tr> </thead> <tbody><tr class="description-tr"> <td class="description-td">NOV6: 67</td> <td class="description-td">tcgacctggacgagaccctggtgcacagctccttcaagccagtgaacaacgcggacttca</td> <td class="description-td">126</td> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||</td> </tr> <tr class="description-tr"> <td class="description-td">LIM 1: 615</td> <td class="description-td">tcgacctggacgagaccctggtgcacagctccttcaagccagtgaacaacgcggacttca</td> <td class="description-td">674</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">NOV6: 127</td> <td class="description-td">tcatccctgtggagattgatggggtggtccaccaggtctacgtgttgaagcgtcctcacg</td> <td class="description-td">186</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||</td> </tr> <tr class="description-tr"> <td class="description-td">LIM 1: 675</td> <td class="description-td">tcatccctgtggagattgatggggtggtccaccaggtctacgtgttgaagcgtcctcacg</td> <td class="description-td">734</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">NOV6: 187</td> <td class="description-td">tggatgagttcctgcagcgaatgggcgagctctttgaatgtgtgctgttcactgctagcc</td> <td class="description-td">246</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||</td> </tr> <tr class="description-tr"> <td class="description-td">LIM 1: 735</td> <td class="description-td">tggatgacttcctgcagcgaatgggcgagctctttgaatgtgtgctgttcactgctagcc</td> <td class="description-td">794</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">NOV6: 247</td> <td class="description-td">tcgccaagtacgcagacccagtagctgacctgctggacaaatggggggccttccgggccc</td> <td class="description-td">306</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||</td> </tr> <tr class="description-tr"> <td class="description-td">LIM1: 795</td> <td class="description-td">tcgccaagtacgcagacccagtagctgacctgctggacaaatggggggccttccgggccc</td> <td class="description-td">854</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">NOV6: 307 ggctgtttcgagagtcctgcgtcttccaccgggggaactacgtgaaggaCCtgagCcggt</td> <td class="description-td">366</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||</td> </tr> <tr class="description-tr"> <td class="description-td">LIM 1: 855</td> <td class="description-td">ggctgtttcgagagtcctgcgtcttccaccgggggaactacgtgaaggacctgagccggt</td> <td class="description-td">914</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">NOV6: 367</td> <td class="description-td">tgggtcgagacctgcggcgggtgctcatcctggacaattcacctgcctcctatgtcttcc</td> <td class="description-td">426</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||</td> </tr> <tr class="description-tr"> <td class="description-td">LIM 1: 915</td> <td class="description-td">tgggtcgagacctgcggcgggtgctcatcctggacaattcacctgcctcctatgtcttcc</td> <td class="description-td">974</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">NOV6: 427</td> <td class="description-td">atccagacaatgctgt</td> <td class="description-td">442 (SEQ ID NO.: 43)</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">||||||||||||||||</td> </tr> <tr class="description-tr"> <td class="description-td">LIM 1: 975</td> <td class="description-td">atccagacaatgctgt</td> <td class="description-td">990 (SEQ ID NO.: 44)</td> </tr> <tr class="description-tr"> <td align="center" class="description-td" colspan="4" nameend="4" namest="1" rowsep="1"> </td> </tr> </tbody></table> </patent-tables> </tables> </div>
</li> <li> <para-num num="[0068]"> </para-num> <div class="description-line" id="P-0068" num="0068">NOV6 is useful as a marker for human chromosome 2. Also, NOV6 can be used to detect tissue from pancreas, spinal cord, stomach, and fetal lung. LIM-interacting factors are capable of binding to a variety of transcription factors and are likely to function as enhancers to bring together diverse transcription factors and form higher order activation complexes or to block formation of such complexes. LIM-interacting factors are involved in developmental regulation of several tissues, especially in the skeletal system. Therefore this protein may be involved in developmental regulation in the skeletal system. NOV6 satisfies a need in the art by providing new diagnostic or therapeutic compositions useful in the treatment of disorders associated with alterations in the expression of members of pancreas-associated and LIM-interacting protein families. NOV6 nucleic acids, polypeptides, antibodies, and other compositions of the present invention are useful in the treatment and/or diagnosis of a variety of diseases and pathologies, including by way of nonlimiting example, those involving pancreatic disorders, e.g. diabetes, aberrant cell proliferation and differentiation, and cancer. <span style="display: none">[0068] </span> </div>
</li> <li> <para-num num="[0069]"> </para-num> <div class="description-line" id="P-0069" num="0069">Genes encoding LIM domain-binding factors were initially isolated in a screen for proteins that physically interact with the LIM domains of nuclear proteins. These proteins are capable of binding to a variety of transcription factors and are likely to function at enhancers to bring together diverse transcription factors and form higher order activation complexes or to block formation of such complexes (See Jurata and Gill, 1997, <span style="display: none">[0069] </span> <i>Molec. Cell. Biol. </i>17:5688). The family of genes encoding LIM domain-binding factors includes 2 members isolated from the mouse, Clim1 (See Bach et al, 1997, <i>Genes Dev. </i>11: 1370) and Clim2/Lbd1/Nli (See Agulnick et al, 1996, <i>Nature </i>384: 270; Jurata and Gill, 1997, <i>Molec. Cell. Biol. </i>17:5688; and Bach et al, 1997, <i>Genes Dev. </i>11: 1370) and their homologs cloned from the frog, chicken, and fly. The fact that LIM domain-binding factors are likely to be involved in the coordination of the transcriptional activity of many diverse factors, and, thus, are implicated in diseases characterized by multiple affected locations. </div>
</li> <li> <para-num num="[0070]"> </para-num> <div class="description-line" id="P-0070" num="0070">Hydrophobicity plot analysis shows a hydrophobic peak at the amino terminal of a NOV6 polypeptide, as would be expected for a secreted or plasma membrane-localized protein. The PSORT program predicts localization in the microbody (peroxisome) with a strong certainty of 0.7480. There appears to be no N-terminal signal sequence. The Protein Subcellular Locator Prediction Analysis of NOV6 predicts that the protein is secreted extracellularly or translocated to the nucleus. <span style="display: none">[0070] </span> </div>
</li> <li> <para-num num="[0071]"> </para-num> <div class="description-line" id="P-0071" num="0071">NOV7 <span style="display: none">[0071] </span> </div>
</li> <li> <para-num num="[0072]"> </para-num> <div class="description-line" id="P-0072" num="0072">A NOV7 sequence according to the invention includes the nucleic acid and encoded polypeptide shown in Table 23. The encoded polypeptide is related to N-acetylglucosaminyltransferase III (GlcNAc-TIII). The tissue of origin of the NOV6 nucleic acid is pancreas. The disclosed nucleic acid (SEQ ID NO:13) is 2,357 nucleotides in length and contains an open reading frame (ORF) that begins with an ATG initiation codon at nucleotides 18-20 and ends with a TGA stop codon at nucleotides 2103-2105, as shown in Table 23. The representative ORF encodes a 695 amino acid polypeptide (SEQ ID NO: 14). Putative untranslated regions upstream and downstream of the coding sequence are underlined in SEQ ID NO: 13.  
<span style="display: none">[0072] </span> <tables id="TABLE-US-00023" num="23"> <patent-tables colsep="0" frame="none" pgwide="1" rowsep="0" tabstyle="monospace"> <table align="left" class="description-table" cols="3" colsep="0" rowsep="0" width="100%"> <thead> <tr class="description-tr"> <td align="center" class="description-td" colspan="3" nameend="3" namest="1">TABLE 23</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td align="center" class="description-td" colspan="3" nameend="3" namest="1" rowsep="1"> </td> </tr> </thead> <tbody><tr class="description-tr"> <td class="description-td">       CGCCCCCGTCGGCGGCCATGCTGCCCCGAGGGCGCCCCCGGGCGCTGGGGGCCGCCGCGCTGTTGCTGCTGCTGC</td> <td class="description-td">(SEQ ID NO.: 13)</td> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">TGCTGCTCGGATTCCTCCTGTTCGGTGGGGACCTGGGACGGGAGGCTGCGGAGAGCCGCCGCCCTCGACGGAACCCCGGGGG</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">GCCGGCCCCGGGGACCACAACCGCTCCGACTGCGGCCCGCAGCCGCCGCCGCCCGCCCAAGTGCGAGCTCTTGCATGTGGCC</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">ATCGTGTGTGCGGGGCATAACTCCAGCCGAGACGTCATCATCCTGGTGAAGTCCATGCTCTTCTACAGGAAAAATCCACTGC</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">ACCTCCACTTGGTGACTGACGCCGTGGCCAGAAACATCCTGGAGACGCTCTTCCACACATGGATGGTGCCTGCTGTCCGTGT</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">CAGCTTTTATCATGCCGACCAGCTCAAGCCCCAGGTCTCCTGGATCCCCAACAAGCACTACTCCGGCCTCTATGG</td> <td class="description-td">GCTAAT</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">GAAGCTGGTGCTGCCCAGTGCCTTGCCTGCTGAGCTGGCCCGCGTCATTGTCCTGGACACGGATGTCACCTTCGCCTCTGAC</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">ATCTCGGAGCTCTGGGCCCTCTTTGCTCACTTTTCTGACACGCAGGCGATCGGTCTTGTGGAGAACCAGAGTGACTGGTACC</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">TGGGCAACCTCTGGAAGAACCACAGGCCCTGGCCTGCCTTGGGCCGGGGATTTAACACAGGTGTGATCCTGCTGCGGCTGGA</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">CCGGCTCCGGCAGGCTGGCTGGGAGCAGATGTGGAGGCTGACAGCCAGGCGGGAGCTCCTTAGCCTGCCTGCCACCTCACTG</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">GCTGACCAGGACATCTTCAACGCTGTGATCAAGGAGCACCCGGGGCTAGTGCAGCGTCTGCCTTGTGTCTGGAATGTGCAGC</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">TGTCAGATCACACACTGGCCGAGCGCTGCTACTCTGAGGCGTCTGACCTCAAGGTGATCCACTGGAACTCACCAAAGAAGCT</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">TCGGGTGAAGAACAAGCATGTGGAATTCTTCCGCAATTTCTACCTGACCTTCCTGGAGTACGATGGGAACCTGCTGCGGAGA</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">GAGCTCTTTGTGTGCCCCAGCCAGCCCCCACCTGGTGCTGAGCAGTTGCAGCAGGCCCTGGCACAACTGGACGAGGAAGACC</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">CCTGCTTTGAGTTCCGGCAGCAGCAGCTCACTGTGCACCGTGTGCATGTCACTTTCCTGCCCCATGAACCGCCACCCCCCCG</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">GCCTCACGATGTCACCCTTGTGGCCCAGCTGTCCATGGACCGGCTGCAGATGTTGGAAGCCCTGTGCAGGCACTGGCCTGGC</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">CCCATGAGCCTGGCCTTGTACCTGACAGACGCAGAAGCTCAGCAGTTCCTGCATTTCGTCGAGGCCTCACCAGTGCTTGCTG</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">CCCGGCAGGACGTGGCCTACCATGTGGTGTACCGTGAGGGGCCCCTATACCCCGTCAACCAGCTTCGCAACGTGGCCTTGGC</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">CCAGGCCCTCACGCCTTACGTCTTCCTCAGTGACATTGACTTCCTGCCTGCCTATTCTCTCTACGACTACCTCAGGGCCTCC</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">ATTGAGCAGCTGGGGCTGGGCAGCCGGCGCAAGGCAGCACTGGTGGTGCCGGCATTTGAGACCCTGCGCTACCGCTTCAGCT</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">TCCCCCATTCCAAGGTGGAGCTGTTGGCCTTGCTGGATGCGGGCACTCTCTACACCTTCAGGTACCACGAGTGGCCCCGAGG</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">CCACGCACCCACAGACTATGCCCGCTGGCGGGAGGCTCAGGCCCCGTACCGTGTGCAATGGGCGGCCAACTATGAACCCTAC</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">GTGGTGGTGCCACGAGACTGTCCCCGCTATGATCCTCGCTTTGTGGGCTTCGGCTGGAACAAAGTGGCCCACATTGTGGAGC</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">TGGATGCCCAGGAATATGAGCTCCTGGTGCTGCCCGAGGCCTTCACCATCCATCTGCCCCACGCTCCAAGCCTGGACATCTC</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">CCGCTTCCGCTCCAGCCCCACCTATCGTGACTGCCTCCAGGCCCTCAAGGACGAATTCCACCAGGACTTGTCCCGCCACCAT</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">GGGGCTGCTGCCCTCAAATACCTCCCAGCCCTGCAGCAGCCCCAGAGCCCTGCCCGAGGCTGAGGCTGGGCCGGCGCTGCCC</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">CTCATCTTAGCATTGGQCAGACACCAGGGCAACCTGCCCTCCGCCATCCCTGCTATTTAAATTATTTAAGGTCTCTGGGAAG</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">GGCTGGGGCAGAGCATCTGTGGGGTGGGGTCTTCCCCTTGCTGCTATTGTATGGCTGGGGACTGGTCTCTCTCTGCCCCAGC</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">CAGTTTGGGGCTGGTTCCCCCATCTTGAATTGTTTATCCCTTTTTCATAATTAAAGTTTTAAAACATCA</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">     MLPRGRPRALGAAALLLLLLLLGFLLFGGDLGREAAESRRPRRNPGGPAPGTTTAPTAARSRRRPP</td> <td class="description-td">(SEQ ID NO.: 14)</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">KCELLHVAIVCAGHNSSRDVIILVKSMLFYRKNPLHLHLVTDAVARNILETLFHTWMVPAVRVSFYHADQL</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">KPQVSWIPNKHYSGLYGLMKLVLPSALPAELARVIVLDTDVTFASDISELWALFAHFSDTQALGLVENQSD</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">WYLGNLWKNHRPWPALGRGFNTGVILLRLDRLRQAGWEQMWRLTARRELLSLPATSLADQDIFNAVIKE</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">HPGLVQRLPCVWNVQLSDHTLAERCYSEASDLKVIHWNSPKKLRVKNKHVEFFRNFYLTFLEYDGNLLRR</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">ELFVCPSQPPPGAEQLQQALAQLDEEDPCFEFRQQQLTVHRVHVTFLPHEPPPPRPHDVTLVAQLSMDRLQ</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">MLEALCRHWPGPMSLALYLTDAEAQQFLHFVEASPVLAARQDVAYHVVYREGPLYPVNQLRNVALAQAL</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">TPYVFLSDIDFLPAYSLYDYLRASIEQLGLGSRRKAALVVPAFETLRYRFSFPHSKVELLALLDAGTLYTFRY</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">HEWPRGHAPTDYARWREAQAPYRVQWAANYEPYVVVPRDCPRYDPRFVGFGWNKVAHIVELDAQEYEL</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">LVLPEAFTIHLPHAPSLDISRFRSSPTYRDCLQALKDEFHQDLSRHHGAAALKYLPALQQPQSPARG</td> </tr> <tr class="description-tr"> <td align="center" class="description-td" colspan="3" nameend="3" namest="1" rowsep="1"> </td> </tr> </tbody></table> </patent-tables> </tables>
</div>
</li> <li> <para-num num="[0073]"> </para-num> <div class="description-line" id="P-0073" num="0073">The disclosed NOV7 nucleic acid has homology (84% identity) with a human glycosyltransferase-like protein (GLY T; Genbank Accession No.: NM004737.1), as shown in Table 24. Also, a NOV7 polypeptide has homology (70% identity, 83% similarity) with a human acetylglucosaminyltransferase-like protein (GLY T; EMBL Accession No.: 095461), as shown in Table 25. Furthermore, the disclosed NOV7 polypeptide contains a conserved region common among many glycosotransferases, as is shown in Table 26, which is a PFAM alignment comparing the glycosotransferase domain of NOV7 with a known glycosotransferase representative of the family, the LPS biosynthesis protein from <span style="display: none">[0073] </span> <i>helicobacter pylori </i>(Accession No.: 7465085). </div>
</li> <li> <para-num num="[0074]"> </para-num> <div class="description-line" id="P-0074" num="0074">The disclosed NOV7 nucleic acid maps to Chromosome 11. NOV7 sequences are expressed in at least the following tissues: brain, tonsils, colon, prostate, liver, kidney, stomach, lung, pituitary gland, pancreas, mammary gland/breast, uterus, and bone marrow. In addition, the sequence is predicted to be expressed in fetal brain because of the expression pattern of a closely related human acetylglucosaminyltransferase homolog (Genbank Accession No.: HSA7583). <span style="display: none">[0074] </span> </div>
</li> <li> <para-num num="[0075]"> </para-num> <div class="description-line" id="P-0075" num="0075">The SignalP, Psort and/or Hydropathy profile for the acetylglucosaminyltransferase-like protein predict that this sequence has a signal peptide and is likely to be localized to the lysosome (p=0.865) or (more characteristic of the family) extracellularly (p=0.8200). <span style="display: none">[0075] </span> </div>
</li> <li> <para-num num="[0076]"> </para-num> <div class="description-line" id="P-0076" num="0076">The protein similarity information, expression pattern, and map location for the acetylglucosaminyltransferase-like protein and nucleic acid disclosed herein suggest that this acetylglucosaminyltransferase may have structural and/or physiological functions characteristic of the acetylglucosaminyltransferase family. Therefore, the nucleic acids and amino acids (encoding protein) of the invention are useful in potential diagnostic and therapeutic applications. These include serving as a specific or selective nucleic acid or protein diagnostic and/or prognostic marker, wherein the presence or amount of the nucleic acid or the protein are to be assessed, as well as potential therapeutic applications such as the following: (i) a protein therapeutic, (ii) a small molecule drug target, (iii) an antibody target (therapeutic, diagnostic, drug targeting/cytotoxic antibody), (iv) a nucleic acid useful in gene therapy (gene delivery/gene ablation), and (v) a composition promoting tissue regeneration in vitro and in vivo. <span style="display: none">[0076] </span> </div>
</li> <li> <para-num num="[0077]"> </para-num> <div class="description-line" id="P-0077" num="0077">The nucleic acids and amino acids encoding protein(s) of the invention are useful in potential diagnostic and therapeutic applications implicated in various diseases and disorders described below and/or other pathologies. For example, the compositions of the present invention have efficacy for treatment of patients suffering from congenital deficiency of mgat2 (associated with severe impairment of normal embryogenesis), particularly in the nervous system in patients with carbohydrate-deficient glycoprotein syndrome II (cdgs type II or cdg2). It has also been implicated in the etiology of a variant form of congenital dyserythropoietic anemia type ii. This acetylglucosaminyltransferase-like gene may also be used for diagnostic and therapeutic applications for various cancers and tumor suppression. In addition, this acetylglucosaminyltransferase-like gene may also have applications for Endometriosis, Fertility, Inflammatory bowel disease (IBD), Diverticular disease, Diabetes, Pancreatitis, Von Hippel-Lindau (VHL) syndrome, Cirrhosis, Transplantation, Alzheimer's disease, Stroke, Tuberous sclerosis, hypercalceimia, Parkinson's disease, Huntington's disease, Cerebral palsy, Epilepsy, Lesch-Nyhan syndrome, Multiple sclerosis, Ataxia-telangiectasia, Leukodystrophies, Behavioral disorders, Addiction, Anxiety, Pain, Neuroprotection, Endocrine dysfunctions, Growth and reproductive disorders, Systemic lupus erythematosus, Autoimmune disease, Asthma, Emphysema, Scleroderma, allergy, ARDS, Autoimmune disease, Renal artery stenosis, Interstitial nephritis, Glomerulonephritis, Polycystic kidney disease, Systemic lupus erythematosus, Renal tubular acidosis, IgA nephropathy, Hypercalceimia, Lesch-Nyhan syndrome, cystitis, and incontinence. TABLE 24.  
<span style="display: none">[0077] </span> <tables id="TABLE-US-00024" num="24"> <patent-tables colsep="0" frame="none" pgwide="1" rowsep="0" tabstyle="monospace"> <table align="left" class="description-table" cols="4" colsep="0" rowsep="0" width="100%"> <thead> <tr class="description-tr"> <td align="center" class="description-td" colspan="4" nameend="4" namest="1">TABLE 24</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td align="center" class="description-td" colspan="4" nameend="4" namest="1" rowsep="1"> </td> </tr> </thead> <tbody><tr class="description-tr"> <td class="description-td">NOV 7: 880</td> <td class="description-td">tctggaatgtgcagctgtcagatcacacactggccgagcgctgctactctgaggcgtctg</td> <td class="description-td">939</td> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">||||||||||||||||||||||||||||   ||||||  ||||||   ||||||||</td> </tr> <tr class="description-tr"> <td class="description-td">GLY T: 1553</td> <td class="description-td">tctggaatgtgcagctgtcagaccacacccgctccgagcagtgctacagagacgtgtctg</td> <td class="description-td">1612</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">NOV 7: 940</td> <td class="description-td">acctcaaggtgatccactggaactcaccaaagaagcttcgggtgaagaacaagcatgtgg</td> <td class="description-td">999</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">|||||||||||||||||||||||||||||||||||||||||||||||||||||</td> </tr> <tr class="description-tr"> <td class="description-td">GLY T: 1613</td> <td class="description-td">atctaaaggtcattcactggaactcccccaagaagctccgggtgaagaacaagcatgtgg</td> <td class="description-td">1672</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">NOV 7: 1000</td> <td class="description-td">aattcttccgcaatttctacctgaccttcctggagta</td> <td class="description-td">1036 (SEQ ID NO.: 45)</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">||||||||||  ||||||||||||||||||||||</td> </tr> <tr class="description-tr"> <td class="description-td">GLY T: 1673</td> <td class="description-td">agttttttcgcaacctctacctgaccttcctggagta</td> <td class="description-td">1709 (SEQ ID NO.: 46)</td> </tr> <tr class="description-tr"> <td align="center" class="description-td" colspan="4" nameend="4" namest="1" rowsep="1"> </td> </tr> </tbody></table> </patent-tables> </tables>
</div>
</li> <li> <para-num num="[0078]"> </para-num> <div class="description-line" id="P-0078" num="0078"> <span style="display: none">[0078]</span> <tables id="TABLE-US-00025" num="25"> <patent-tables colsep="0" frame="none" pgwide="1" rowsep="0" tabstyle="monospace"> <table align="left" class="description-table" cols="3" colsep="0" rowsep="0" width="100%"> <thead> <tr class="description-tr"> <td align="center" class="description-td" colspan="3" nameend="3" namest="1">TABLE 25</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td align="center" class="description-td" colspan="3" nameend="3" namest="1" rowsep="1"> </td> </tr> </thead> <tbody><tr class="description-tr"> <td class="description-td">NOV 7: 199</td> <td class="description-td">KCELLHVAIVCAGHNSSRDVIILVKSMLFYRKNPLHLHLVTDAVAPNILETLFHTWMVPA</td> <td class="description-td">378</td> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">*** +********+*+****+****+**+*+**** **+*++*  ** *** ******</td> </tr> <tr class="description-tr"> <td class="description-td">GLY T:  134</td> <td class="description-td">KCETIHVAIVCAGYNASRDVVTLVKSVLFHRRNPLHFHLIADSIAEQILATLFQTWMVPA</td> <td class="description-td">193</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">NOV 7:  379</td> <td class="description-td">VRVSFYHADQLKPQVSWIPNKHYSGLYGLMKLVLPSALPAELARVIVLDTDVTFASDISE</td> <td class="description-td">558</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">*** **+**+** +***********+********   *** * ********+***+**+*</td> </tr> <tr class="description-tr"> <td class="description-td">GLY T:  194</td> <td class="description-td">VRVDFYNADELKSEVSWIPNKHYSGIYGLMKLVLTKTLPANLERVIVLDTDITFATDIAE</td> <td class="description-td">253</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">NOV 7:  559</td> <td class="description-td">LWALFAHFSDTQAIGLVENQSDWYLGNLWKNHRPWPALGRGFNTGVILLRLDRLRQAGWE</td> <td class="description-td">738</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">***+*  *   * +**************************+******* **+**+  **</td> </tr> <tr class="description-tr"> <td class="description-td">GLY T:  254</td> <td class="description-td">LWAVFHKFKGQQVLGLVENQSDWYLGNLWKNHRPWPALGRGYNTGVILLLLDKLRKMKWE</td> <td class="description-td">313</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">NOV 7:  739</td> <td class="description-td">QMWRLTARRELLSLPATSLADQDIFNAVIKEHPGLVQRLPCVWNVQLSDHTLAERCYSEA</td> <td class="description-td">918</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">******* ***+++**************++* ** +*** ********* +*+** +</td> </tr> <tr class="description-tr"> <td class="description-td">GLY T:  314</td> <td class="description-td">QMWRLTAERELMGMLSTSLADQDIFNAVIKQNPFLVYQLPCFWNVQLSDHTRSEQCYRDV</td> <td class="description-td">373</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">NOV 7:  919</td> <td class="description-td">SDLKVIHWNSPKKLRVKNKHVEFFRNFYLTFLEYDGNLLRRELFVCPSQPPPGAEQLQQA</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">*************************** ***************** ***+    +* **+</td> </tr> <tr class="description-tr"> <td class="description-td">GLY T:  374</td> <td class="description-td">SDLKVIHWNSPKKLRVKNKHVEFFRNLYLTFLEYDGNLLRRELFGCPSEADVNSENLQKQ</td> <td class="description-td">433</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">NOV7: 1099</td> <td class="description-td">LAQLDEEDPCFEFRQQQLTVHRVHVTFLPHEPPPPRPH-DVTLVAQLSMDRLQMLEALCR</td> <td class="description-td">1275</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">*++***+* *+***+++**** *+** +*  *     ******************+*+</td> </tr> <tr class="description-tr"> <td class="description-td">GLY T:  434</td> <td class="description-td">LSELDEDDLCYEFRRERFTVHRTHLYFLHYEYEPAADSTDVTLVAQLSMDRLQMLEAICK</td> <td class="description-td">493</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">NOV 7: 1276</td> <td class="description-td">HWPGPMSLALYLTDAEAQQFLHFVEASPVLAARQDVAYHVVYREGFLYPVNQLRNVALAQ</td> <td class="description-td">1455</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">** **+******+******** ++* ** +* +* **+**+**  **** *****+</td> </tr> <tr class="description-tr"> <td class="description-td">GLY T:  494</td> <td class="description-td">HWEGPISLALYLSDAIAQQFLRYAQGSEVLMSRHNVGYHIVYKEGQFYPVNLLRNVAMKH</td> <td class="description-td">553</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">NOV 7: 1456</td> <td class="description-td">ALTPYVFLSDIDFLPAYSLYDYLRASIEQLGLGSRRKAALVVPAFETLRYRFSFPESKVE</td> <td class="description-td">1635</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">  ***+********* * **+*** *+** * ++** ++********** *** ** *</td> </tr> <tr class="description-tr"> <td class="description-td">GLY T:  554</td> <td class="description-td">ISTPYMFLSDIDFLPMYGLYEYLRKSVIQLDLANTKKA-MIVPAFETLRYRLSFPKSKAE</td> <td class="description-td">612</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">NOV 7: 1636</td> <td class="description-td">LLALLDAGTLYTFRYEEWPRGHAPTDYARWREAQAPYRVQWAANYEPYVVVPRDCPRYDP</td> <td class="description-td">1815</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">**++** ***+***** * +*****++*+** *  ****+* *++****** **** **</td> </tr> <tr class="description-tr"> <td class="description-td">GLY T:  613</td> <td class="description-td">LLSMLDMGTLFTFRYHVWTKGHAPTNFAKWRTATTPYRVEWEADFEPYVVVRRDCPEYDR</td> <td class="description-td">672</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">NOV 7: 1816</td> <td class="description-td">RFVGFGWNKVAHIVELDAQEYELLVLPEAFTIHLPHAPSLDISRFRSSPTYRDCLQALKD</td> <td class="description-td">1995</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">*************+*** **** +*** *+**+***** **++***+  ** **+**+</td> </tr> <tr class="description-tr"> <td class="description-td">GLY T:  673</td> <td class="description-td">RFVGFGWNKVAHIMELDVQEYEFIVLPNAYMIHMPNAPSFDITKFRSNKQYRICLKTLKE</td> <td class="description-td">732</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">NOV 7: 1996</td> <td class="description-td">EFEQDLSRHHGAAALKYLPA</td> <td class="description-td">2055 (SEQ ID NO.: 47)</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">** **+** +* ****** *</td> </tr> <tr class="description-tr"> <td class="description-td">GLY T:  733</td> <td class="description-td">EFQQDMSRRYGFAALKYLTA</td> <td class="description-td">752 (SEQ ID NO.: 48)</td> </tr> <tr class="description-tr"> <td align="center" class="description-td" colspan="3" nameend="3" namest="1" rowsep="1"> </td> </tr> <tr class="description-tr"> <td align="left" class="description-td" colspan="3" nameend="3" namest="1"> </td> </tr> </tbody></table> </patent-tables> </tables> </div>
</li> <li> <para-num num="[0079]"> </para-num> <div class="description-line" id="P-0079" num="0079"> <span style="display: none">[0079]</span> <tables id="TABLE-US-00026" num="26"> <patent-tables colsep="0" frame="none" pgwide="1" rowsep="0" tabstyle="monospace"> <table align="left" class="description-table" cols="4" colsep="0" rowsep="0" width="100%"> <thead> <tr class="description-tr"> <td align="center" class="description-td" colspan="4" nameend="4" namest="1">TABLE 26</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td align="center" class="description-td" colspan="4" nameend="4" namest="1" rowsep="1"> </td> </tr> </thead> <tbody><tr class="description-tr"> <td class="description-td">NOV 7: 158</td> <td class="description-td">LVLPSALPAEL-ARVIVLDTDVTYASDISELWALFAHFSDTQAIGLV-ENQSDWYLGNLW</td> <td class="description-td">215</td> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">+             *+++        +               ++    ++   * *</td> </tr> <tr class="description-tr"> <td class="description-td">GLY 8: 96</td> <td class="description-td">FLNLEENDENYFYGVLEVEKHHMNEGFLFCNLDY--QRKKNFTLRMHELLRGNEAKGELD</td> <td class="description-td">153</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">NOV 7: 216</td> <td class="description-td">---------------KNHRPWPALGR-----GFNTGVILLRLDRLRQAGWEQMWR-LTAR</td> <td class="description-td">254</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">                   *+    +       *       *+               *</td> </tr> <tr class="description-tr"> <td class="description-td">GLY 8: 154</td> <td class="description-td">FTKWCWPNMKALGIEYCVFPYYYTIKDFSNAYLNENYKKTILEARENPTIIHYDAWWGAV</td> <td class="description-td">213</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">NOV 7: 255</td> <td class="description-td">RELLSLFATSLADQDIFNAVIKE--------HPGLVQRLPCVWNVQLSD-----HTLAER</td> <td class="description-td">301</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">+         *      **+*             *+     *+   *      +     +</td> </tr> <tr class="description-tr"> <td class="description-td">GLY8:214</td> <td class="description-td">KP--WDYPFGLKADLWLNALAKTFFMSDWIDSIARVEIGSEKWHRYHSIVAYHYYFPLWK</td> <td class="description-td">271</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">NOV 7: 302</td> <td class="description-td">CYSE----------ASDLKVIHWNSP-------KKLRVKNKHVEFFRNFYLTFL</td> <td class="description-td">338 (SEQ ID NO.: 49)</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">   +                **                 + * *   *    **</td> </tr> <tr class="description-tr"> <td class="description-td">GLY 8: 272</td> <td class="description-td">TEEQIAHDALKTFLDHYFSCIHAAIKQENLGMFLNHYFSHAHAEIKENSLEMFL</td> <td class="description-td">325 (SEQ ID NO.: 50)</td> </tr> <tr class="description-tr"> <td align="center" class="description-td" colspan="4" nameend="4" namest="1" rowsep="1"> </td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td align="left" class="description-td" colspan="4" nameend="4" namest="1"> </td> </tr> </tbody></table> </patent-tables> </tables> </div>
</li> <li> <para-num num="[0080]"> </para-num> <div class="description-line" id="P-0080" num="0080"> <span style="display: none">[0080]</span> </div>
</li> <li> <para-num num="[0081]"> </para-num> <div class="description-line" id="P-0081" num="0081">A NOV7 polypeptide sequence was subjected to CLUSTALW alignment analysis with three known or predicted acetylglucosaminyltransferase (AGT)-like proteins, human AGT-like protein (hAGT; EMBL Accession No.: 095461), human protein KIAA0609 (KIAA; EMBL Accession No.: 060348) and mouse AGT-like protein (mAGT; EMBL Accession No.: Q9Z1M7). Results are shown in Table 27.  
<span style="display: none">[0081] </span> <tables id="TABLE-US-00027" num="27"> <patent-tables colsep="0" frame="none" pgwide="1" rowsep="0" tabstyle="monospace"> <table align="left" class="description-table" cols="4" colsep="0" rowsep="0" width="100%"> <thead> <tr class="description-tr"> <td align="center" class="description-td" colspan="4" nameend="4" namest="1">TABLE 27</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td align="center" class="description-td" colspan="4" nameend="4" namest="1" rowsep="1"> </td> </tr> </thead> <tbody><tr class="description-tr"> <td class="description-td">hAGT</td> <td class="description-td">-----MLGICRGRRKFLAASLSLLCIPAITWIYLFSGSFEDGKPVSLSPLESQAHSPRYT</td> <td class="description-td"> </td> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">KIAA</td> <td class="description-td">ATSERMLGICRGRRKFLAASLSLLCIPAITWIYLFSGSFEDGKPVSLSPLESQARSPRYT</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">mAGT</td> <td class="description-td">-----MLGICRGRRKFLAASLTLLCIPAITWIYLFAGSFEDGKPVSLSPLESQAHSPRYT</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">NOV7</td> <td class="description-td">-------MLPRGRPRALGAAALLLLLLLLGFL-LFGGDLG--------------------</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">        : *** : *.*:  ** :  : :: **.*.:</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">hAGT</td> <td class="description-td">ASSQRERESLEVRMREVEEENRALRRQLSLAQGRAPSHRRGNHSKTYSMEEGTGDSENLR</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">KIAA</td> <td class="description-td">ASSQRERESLEVRNREVEEENRALRRQLSLAQGRAPSHRRGMiSKTYSMEEGTGDSENLR</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">mAGT</td> <td class="description-td">ASSQRERESLEVRVREVEEENRALRRQLSLAQGQSPAHHRGNHSKTYSMEEGTGDSENLR</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">NOV7</td> <td class="description-td">--------------REAAESRRPRRNPGGPAPGTTTAP--------------TAARSRRR</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">              **. *..*. *.  . * * :.:               *.  .. *</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">bAGT</td> <td class="description-td">AGIVAGNSSECGQQPVVEKCETIHVAIVCAGYNASRDVVTLVKSVLFHRRNPLHFHLIAD</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">KIAA</td> <td class="description-td">AGIVAGNSSECGQQPVVEKCETIHVAIVCAGYNASRDVVTLVKSVLFHRPNPLHFHLIAD</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">mAGT</td> <td class="description-td">AGIVAGNSSECGQQPAVEKCETIHVAIVCAGYNASRDVVTLVKSVLFHRRNPLHFHLIAD</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">NOV7</td> <td class="description-td">P----------------PKCELLHVAIVCAGHNSSRDVIILVKSMLFYRKNPLHLHLVTD</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">.                 *** :********:*:****: ****:**:*:****:**::*</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">hAGT</td> <td class="description-td">SIAEQILATLFQTWMVPAVRVDFYNADELKSEVSWIPNKHYSGIYGLMKLVLTKTLPANL</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">KIAA</td> <td class="description-td">SIAEQILATLFQTWMVPAVRVDFYNADELKSEVSWIPNKHYSGIYGLMKLVLTKTLPANL</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">mAGT</td> <td class="description-td">SIAEQILATLFQTWMVPAVRVDFYNADELKSEVSWIPNKHYSGIYGLMKLVLTKTLPANL</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">NOV7</td> <td class="description-td">AVARNILETLFHTWMVPAVRVSFYHADQLKPQVSWIPNKHYSGLYGLMKLVLPSALPAEL</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">::*.:** ***:*********.**:**:**.:***********:********..:***.*</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">hAGT</td> <td class="description-td">ERVIVLDTDITFATDIAELWAVFHKFKGQQVLGLVENQSDWYLGNLWKNHRPWPALGHGY</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">KIAA</td> <td class="description-td">ERVIVLDTDITFATDIAELWAVFHKFKGQQVLGLVENQSDWYLGNLWKNHRPWPALGRGY</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">mAGT</td> <td class="description-td">ERVIVLDTDITFATDIAELWAVFHKFKGQQVLGLVENQSDWYLGNLWKNHRPWPALGRGY</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">NOV7</td> <td class="description-td">ARVIVLDTDVTFASDISELWALFAHFSDTQAILGVENQSDWYLGNLWKNERPWPALGRGF</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">*********:***:**:****:* :*.. *.:  **************************</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">hAGT</td> <td class="description-td">NTGVILLLLDKLRKMKWEQMWRLTAERELMGMLSTSLADQDIFNAVIKQNPFLVYQLPCF</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">KIAA</td> <td class="description-td">NTGVILLLLDKLRKMKWEQMWRLTAERELMGMLSTSLADQDIFNAVIKQNPFLVYQLPCF</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">mAGT</td> <td class="description-td">NTGVILLLLDKLRKMKWEQMWRLTAERELMGMLSTSLADQDIFNAVIKQNPFLVYQLPCF</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">NOV7</td> <td class="description-td">NTGVILLRLDRLRQAGWEQMWRLTARRELLSLPATSLADQDIFNAVIKEHPGLVQRLPCV</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">******* **:**:  *********.***:.: :**************::* ** :***.</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">hAGT</td> <td class="description-td">WNVQLSDHTRSEQCYRDVSDLKVIHNNSPKKLRVKNKHVEFFRNLYLTFLEYDGNLIRRE</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">KIAA</td> <td class="description-td">WNVQLSDHTRSEQCYRDVSDLKVIHNNSPKKLRVKNKHVEFFRNLYLTFLEYDGNLIRRE</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">mAGT</td> <td class="description-td">WNVQLSDHTRSEQCYRDVSDLKVIHWNSPKKLRVKNKEVEFFRNLYLTFLEYDGNLIRRE</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">NOV7</td> <td class="description-td">WNVQLSDHTLAERCYSEASDLKVIHWNSPKKLRVKNKHVEFFRNFYLTFLEYDGNLLRRE</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">********* :*:** :.**************************:***********:***</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">hAGT</td> <td class="description-td">LFGCPSEADVNSENLQKQLSELDEDDLCYEFRRERFTVHRTHLYFLHYEYEPAADSTDVT</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">KIAA</td> <td class="description-td">LFGCPSEADVNSENLQKQLSELDEDDLCYEFRRERFTVHRTHLYFLHYEYEPAADSTDVT</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">mAGT</td> <td class="description-td">LFGCPSETDVNNENLQKQLSELDEDDLCYEFRRERFTVHRTHLYBLHYEFEPSADNTDVT</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">NOV7</td> <td class="description-td">LFVCPSQPPPGAEQLQQALAQLDEEDPCFEFRQQQLTVHRVHVTFLPHE-PPPPRPHDVT</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">** ***:.  . *:**: *::***:* *:***::::****.*: ** :* *..    ***</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">hAGT</td> <td class="description-td">LVAQLSMDRLQMLEAICKHWEGPISLALYLSDAEAQQFLRYAQGSEVLMSRHNVGYHIVY</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">KIAA</td> <td class="description-td">LVAQLSMDRLQMLEAICKHWEGPISLALYLSDAEAQQFLRYAQGSEVLMSRHINGYHIVY</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">mAGT</td> <td class="description-td">LVAQLSMDRLQMLEAICKHWEGPISLALYLSDAEAQQFLRYAQGSEVLMSRQNVGYHIVY</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">NOV7</td> <td class="description-td">LVAQLSMDRLQMLEALCRHWPGPMSLALYLTDAEAQQFLHFVEASPVLAARQDVAYHVVY</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">***************:*:** **:******:********::.:.* ** :*::*.**:**</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">hAGT</td> <td class="description-td">KEGQFYPVNLLRNVANKHISTPYMFLSDIDFLPMYGLYEYLRKSVIQLDLAN-TKKAMIV</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">KIAA</td> <td class="description-td">KEGQFYPVNLLRNVAMKHISTPYMFLSDIDFLPMYGLVEYLRKSVIQLDLAN-TKKANIV</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">mAGT</td> <td class="description-td">KEGQFYPVNLLRNVAMKHISTPYMFLSDIDFLPMYGLVEYLRKSVTQLDLAN-TKKAMTV</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">NOV7</td> <td class="description-td">REGPLYPVNQLRNVALAQALTPYVFLSDIDFLPAYSLYDYLRASIEQLGLGSRRKAALVV</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">:** :**** *****: : ***:********* *.* :*** *: **.*..   * *::*</td> </tr> <tr class="description-tr"> <td class="description-td">hAGT</td> <td class="description-td">PAFETLRYRLSFPKSKAELLSMLDMGTLFTFRYHVWTKGHAPTNFAKWRTATTPYRVEWE</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">KIAA</td> <td class="description-td">PAFETLRYRLSFPKSKAELLSMLDMGTLFTFRYHVWTKGHAPTNFAKWRTATTPYRVEWE</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">mAGT</td> <td class="description-td">PAFETLRYRLSFPKSKAELLSMLDMGTLFTFRYHVWTKGHAPTNFAKWRTATTPYQVEWE</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">NOV7</td> <td class="description-td">PAFETLRYRFSFPHSKVELLALLDAGTLYTFRYEEWPRGHAPTDYARWREAQAPYRVQWA</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">*********:***:**.***::** ***:***** *.:*****::*:** * :**:*:*</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">hAGT</td> <td class="description-td">ADFEPYVVVRRDCPEYDRRFVGFGWNKVAHIMELDVQEYEFIVLPNAYMIHMPHAPSFDI</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">KIAA</td> <td class="description-td">ADFEPYVVVRRDCPEYDRRFVGFGWNKVAHIMELDVQEYEFIVLPNAYMIHMPHAPSFDI</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">mAGT</td> <td class="description-td">ADFEPYVVVRRDCPEYDRRFVGFGWNKVAHIMELDAQEYEFTVLPNAYMIHMPHAPSFDI</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">NOV7</td> <td class="description-td">ANYEPYVVVPRDCPRYDPRFVGFGWNKVAHIVELDAQEYELLVLPEAFTIHLPHAPSLDI</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">*::****** ****.** *************:***.****: ***:*: **:*****:**</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">hAGT</td> <td class="description-td">TKFRSNKQYRICLKTLKEEFQQDMSRRYGFAALKYLTAENNS------</td> <td class="description-td">(SEQ ID NO.: 51)</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">KIAA</td> <td class="description-td">TKFRSNKQYPICLKTLKEEFQQDMSRRYGFAALKYLTAENNS------</td> <td class="description-td">(SEQ ID NO.: 52)</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">mAGT</td> <td class="description-td">TKFRSNKQYRICLKTLKEEFQQDMSRRYGFAALKYLTAENNS------</td> <td class="description-td">(SEQ ID NO.: 53)</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">NOV7</td> <td class="description-td">SRFRSSPTYRDCLQALKDEFHQDLSRHEGAAALKYLPALQQPQSPARG</td> <td class="description-td">(SEQ ID NO.: 14)</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td"> ::***. ** **::**:**:**:**::* ******.* ::.</td> </tr> <tr class="description-tr"> <td align="center" class="description-td" colspan="4" nameend="4" namest="1" rowsep="1"> </td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td align="left" class="description-td" colspan="4" nameend="4" namest="1"> </td> </tr> </tbody></table> </patent-tables> </tables>
</div>
</li> <li> <para-num num="[0082]"> </para-num> <div class="description-line" id="P-0082" num="0082">Quantitative expression analysis (Example 2) and Serial Analysis of Gene Expression (SAGE; Example 3) were performed with a NOV7 nucleic acid. The NOV7 gene is over-expressed in several types of cancer relative to corresponding unaffected tissue, especially breast cancer, lung cancer, colon cancer, colon metastases to the lung, ovarian cancer and gastric cancer (See FIG. 1). NOV7 is also over-expressed in an ovarian cancer cell line, OVCA432-2 (See FIG. 2). <span style="display: none">[0082] </span> </div>
</li> <li> <para-num num="[0083]"> </para-num> <div class="description-line" id="P-0083" num="0083">NOV7 is a new member in the acetylglucosaminyltransferase family. NOV7 can be used to detect pancreas tissue, and is useful in determining changes in expression of genes associated with physiological and pathological pancreas activity, e.g. diabetes. Loss of acetylglucosaminyltransferase activity is associated with decrease in tumor growth and metastasis (See e.g. Gransovsky et al, 2000 <span style="display: none">[0083] </span> <i>Nature Medicine </i>6(3): 306). NOV7 is useful in diagnosing and measuring carcinogenic and metastatic onset and progression. The NOV7 nucleic acids and polypeptides provide new diagnostic or therapeutic compositions useful in the treatment of disorders associated with alterations in the expression of members of pancreas and acetylglucosaminyltransferase protein families. NOV7 nucleic acids, polypeptides, antibodies, and other compositions of the present invention are useful in the treatment and/or diagnosis of a variety of diseases and pathologies, including by way of nonlimiting example, those involving diabetes and cancer. </div>
</li> <li> <para-num num="[0084]"> </para-num> <div class="description-line" id="P-0084" num="0084">The acetylglucosaminyltransferase family includes enzymes that transfer sugar residues to donor molecules. Members of this family are involved in lipopolysaccharide biosynthesis and glycogen synthesis. This family includes Lipopolysaccharide galactosyltransferase (SWISSPROT Accession No.: P27128), lipopolysaccharide glucosyltransferase 1 (SWISSPROT Accession No.: P27129), and glycogenin glucosyltransferase (SWISSPROT Accession No.: P46976). <span style="display: none">[0084] </span> </div>
</li> <li> <para-num num="[0085]"> </para-num> <div class="description-line" id="P-0085" num="0085">One member of the acetylglucosaminyltransferase family is N-acetylglucosaminyltransferase III (GlcNAc-TIII), which is encoded by the Mgat3 gene and catalyzes the addition of the bisecting GlcNAc to the core of N-glycans. Mice lacking GlcNAc-TIII due to the insertion mutation Mgat3tmlPst (termed Mgat3neo), exhibit retarded progression of liver tumors induced by diethylnitrosamine <span style="display: none">[0085] </span> </div>
</li> <li> <para-num num="[0086]"> </para-num> <div class="description-line" id="P-0086" num="0086">Metastatic potential of various cancer cells has been shown to correlate with increase of GnT-V activity and concomitant beta 1-6 branching of N-acetylglucosamine. Over expression of H-ras or v-sis/PDGF-B up-regulated the activities of GnT-V to various degrees in the transfected cells.Stable transfection of GnT-V into human glioma U-373 MG cells resulted in changes in cell morphology and focal adhesions and a marked increase in glioma invasivity in vitro. Increased susceptibility to apoptosis of human hepatocarcinoma cells transfected with antisense N-acetylglucosaminyltransferase V cDNA. <span style="display: none">[0086] </span> </div>
</li> <li> <para-num num="[0087]"> </para-num> <div class="description-line" id="P-0087" num="0087">The nucleic acids and proteins of the invention are useful in potential therapeutic applications implicated in disorders, e.g. of the pancreas, mammary gland, and kidney, such as e.g. diabetes, hypertension and cancer. By way of nonlimiting example, the compositions of the present invention will have efficacy for treatment of patients suffering from disorders of the renal and pancreaticsystems. The novel nucleic acids encoding novel proteins, and the novel proteins of the invention, or fragments thereof, may further be useful in the treatment of adenocarcinoma; lymphoma; prostate cancer; uterus cancer, immune response, AIDS, asthma, Crohn's disease, multiple sclerosis, treatment of Albright hereditary ostoeodystrophy, the development of powerful assay systems for functional analysis of various human disorders which will help in understanding of pathology of the disease, and development of new drug targets for various disorders. They may also be used in diagnostic applications, wherein the presence or amount of the nucleic acid or the protein are to be assessed. These materials are further useful in the generation of antibodies that bind immunospecifically to the novel substances of the invention for use in therapeutic or diagnostic methods. <span style="display: none">[0087] </span> </div>
</li> <li> <para-num num="[0088]"> </para-num> <div class="description-line" id="P-0088" num="0088">NOVX Nucleic Acids <span style="display: none">[0088] </span> </div>
</li> <li> <para-num num="[0089]"> </para-num> <div class="description-line" id="P-0089" num="0089">The nucleic acids of the invention include those that encode a NOVX polypeptide or protein. As used herein, the terms polypeptide and protein are interchangeable. <span style="display: none">[0089] </span> </div>
</li> <li> <para-num num="[0090]"> </para-num> <div class="description-line" id="P-0090" num="0090">In some embodiments, a NOVX nucleic acid encodes a mature NOVX polypeptide. As used herein, a “mature” form of a polypeptide or protein described herein relates to the product of a naturally occurring polypeptide or precursor form or proprotein. The naturally occurring polypeptide, precursor or proprotein includes, by way of nonlimiting example, the full-length gene product, encoded by the corresponding gene. Alternatively, it may be defined as the polypeptide, precursor or proprotein encoded by an open reading frame described herein. The product “mature” form arises, again by way of nonlimiting example, as a result of one or more naturally occurring processing steps that may take place within the cell in which the gene product arises. Examples of such processing steps leading to a “mature” form of a polypeptide or protein include the cleavage of the N-terminal methionine residue encoded by the initiation codon of an open reading frame, or the proteolytic cleavage of a signal peptide or leader sequence. Thus a mature form arising from a precursor polypeptide or protein that has residues 1 to N, where residue 1 is the N-terminal methionine, would have residues 2 through N remaining after removal of the N-terminal methionine. Alternatively, a mature form arising from a precursor polypeptide or protein having residues 1 to N, in which an N-terminal signal sequence from residue 1 to residue M is cleaved, would have the residues from residue M+1 to residue N remaining. Further as used herein, a “mature” form of a polypeptide or protein may arise from a step of post-translational modification other than a proteolytic cleavage event. Such additional processes include, by way of non-limiting example, glycosylation, myristoylation or phosphorylation. In general, a mature polypeptide or protein may result from the operation of only one of these processes, or a combination of any of them. <span style="display: none">[0090] </span> </div>
</li> <li> <para-num num="[0091]"> </para-num> <div class="description-line" id="P-0091" num="0091">Among the NOVX nucleic acids is the nucleic acid whose sequence is provided in SEQ ID NO: 1, 3, 5, 7, 9, 11, or 13, or a fragment thereof. Additionally, the invention includes mutant or variant nucleic acids of SEQ ID NO: 1, 3, 5, 7, 9, 11, or 13, or a fragment thereof, any of whose bases may be changed from the corresponding bases shown inSEQ ID NO: 1, 3, 5, 7, 9, 11, or 13, while still encoding a protein that maintains at least one of its NOVX-like activities and physiological functions (i.e., modulating angiogenesis, neuronal development). The invention further includes the complement of the nucleic acid sequence of SEQ ID NO: 1, 3, 5, 7, 9, 11, or 13, including fragments, derivatives, analogs and homologs thereof. The invention additionally includes nucleic acids or nucleic acid fragments, or complements thereto, whose structures include chemical modifications. <span style="display: none">[0091] </span> </div>
</li> <li> <para-num num="[0092]"> </para-num> <div class="description-line" id="P-0092" num="0092">One aspect of the invention pertains to isolated nucleic acid molecules that encode NOVX proteins or biologically active portions thereof. Also included are nucleic acid fragments sufficient for use as hybridization probes to identify NOVX-encoding nucleic acids (e.g., NOVX mRNA) and fragments for use as polymerase chain reaction (PCR) primers for the amplification or mutation of NOVX nucleic acid molecules. As used herein, the term “nucleic acid molecule” is intended to include DNA molecules (e.g, cDNA or genomic DNA), RNA molecules (e.g., mRNA), analogs of the DNA or RNA generated using nucleotide analogs, and derivatives, fragments and homologs thereof. The nucleic acid molecule can be single-stranded or double-stranded, but preferably is double-stranded DNA. <span style="display: none">[0092] </span> </div>
</li> <li> <para-num num="[0093]"> </para-num> <div class="description-line" id="P-0093" num="0093">“Probes” refer to nucleic acid sequences of variable length, preferably between at least about 10 nucleotides (nt), 100 nt, or as many as about, e.g., 6,000 nt, depending on use. Probes are used in the detection of identical, similar, or complementary nucleic acid sequences. Longer length probes are usually obtained from a natural or recombinant source, are highly specific and much slower to hybridize than oligomers. Probes may be single- or double-stranded and designed to have specificity in PCR, membrane-based hybridization technologies, or ELISA-like technologies. <span style="display: none">[0093] </span> </div>
</li> <li> <para-num num="[0094]"> </para-num> <div class="description-line" id="P-0094" num="0094">An “isolated” nucleic acid molecule is one that is separated from other nucleic acid molecules that are present in the natural source of the nucleic acid. Examples of isolated nucleic acid molecules include, but are not limited to, recombinant DNA molecules contained in a vector, recombinant DNA molecules maintained in a heterologous host cell, partially or substantially purified nucleic acid molecules, and synthetic DNA or RNA molecules. Preferably, an “isolated” nucleic acid is free of sequences which naturally flank the nucleic acid (i.e., sequences located at the 5′ and 3′ ends of the nucleic acid) in the genomic DNA of the organism from which the nucleic acid is derived. For example, in various embodiments, the isolated NOVX nucleic acid molecule can contain less than about 50 kb, 25 kb, 5 kb, 4 kb, 3 kb, 2 kb, 1 kb, 0.5 kb or 0.1 kb of nucleotide sequences which naturally flank the nucleic acid molecule in genomic DNA of the cell from which the nucleic acid is derived. Moreover, an “isolated” nucleic acid molecule, such as a cDNA molecule, can be substantially free of other cellular material or culture medium when produced by recombinant techniques, or of chemical precursors or other chemicals when chemically synthesized. <span style="display: none">[0094] </span> </div>
</li> <li> <para-num num="[0095]"> </para-num> <div class="description-line" id="P-0095" num="0095">A nucleic acid molecule of the present invention, e.g., a nucleic acid molecule having the nucleotide sequence of SEQ ID NO: 1, 3, 5, 7, 9, 11, or 13, or a complement of any of this nucleotide sequence, can be isolated using standard molecular biology techniques and the sequence information provided herein. Using all or a portion of the nucleic acid sequence of SEQ ID NO: 1, 3, 5, 7, 9, 11, or 13, as a hybridization probe, NOVX nucleic acid sequences can be isolated using standard hybridization and cloning techniques (e.g, as described in Sambrook et al., eds., <span style="display: none">[0095] </span> <i>MOLECULAR CLONING: A LABORATORY MANUAL </i>2<sup>nd </sup>Ed., Cold Spring Harbor Laboratory Press, Cold Spring Harbor, N.Y., 1989; and Ausubel, et al., eds., <i>CURRENT PROTOCOLS IN MOLECULAR BIOLOGY</i>, John Wiley &amp; Sons, New York, N.Y., 1993.) A nucleic acid of the invention can be amplified using cDNA, mRNA or alternatively, genomic DNA, as a template and appropriate oligonucleotide primers according to standard PCR amplification techniques. The nucleic acid so amplified can be cloned into an appropriate vector and characterized by DNA sequence analysis. Furthermore, oligonucleotides corresponding to NOVX nucleotide sequences can be prepared by standard synthetic techniques, e.g., using an automated DNA synthesizer. </div>
</li> <li> <para-num num="[0096]"> </para-num> <div class="description-line" id="P-0096" num="0096">As used herein, the term “oligonucleotide” refers to a series of linked nucleotide residues, which oligonucleotide has a sufficient number of nucleotide bases to be used in a PCR reaction. A short oligonucleotide sequence may be based on, or designed from, a genomic or cDNA sequence and is used to amplify, confirm, or reveal the presence of an identical, similar or complementary DNA or RNA in a particular cell or tissue. Oligonucleotides comprise portions of a nucleic acid sequence having about 10 nt, 50 nt, or 100 nt in length, preferably about 15 nt to 30 nt in length. In one embodiment, an oligonucleotide comprising a nucleic acid molecule less than 100 nt in length would further comprise at lease 6 contiguous nucleotides of SEQ ID NO: 1, 3, 5, 7, 9, 11, or 13, or a complement thereof. Oligonucleotides may be chemically synthesized and may be used as probes. <span style="display: none">[0096] </span> </div>
</li> <li> <para-num num="[0097]"> </para-num> <div class="description-line" id="P-0097" num="0097">In another embodiment, an isolated nucleic acid molecule of the invention comprises a nucleic acid molecule that is a complement of the nucleotide sequence shown in SEQ ID NO: 1, 3, 5, 7, 9, 11, or 13, or a portion of this nucleotide sequence. A nucleic acid molecule that is complementary to the nucleotide sequence shown in SEQ ID NO: 1, 3, 5, 7, 9, 11, or 13 is one that is sufficiently complementary to the nucleotide sequence shown in SEQ ID NO: 1, 3, 5, 7, 9, 11, or 13 that it can hydrogen bond with little or no mismatches to the nucleotide sequence shown in SEQ ID NO: 1, 3, 5, 7, 9, 11, or 13, thereby forming a stable duplex. <span style="display: none">[0097] </span> </div>
</li> <li> <para-num num="[0098]"> </para-num> <div class="description-line" id="P-0098" num="0098">As used herein, the term “complementary” refers to Watson-Crick or Hoogsteen base pairing between nucleotide units of a nucleic acid molecule, and the term “binding” means the physical or chemical interaction between two polypeptides or compounds or associated polypeptides or compounds or combinations thereof. Binding includes ionic, non-ionic, Von der Waals, hydrophobic interactions, etc. A physical interaction can be either direct or indirect. Indirect interactions may be through or due to the effects of another polypeptide or compound. Direct binding refers to interactions that do not take place through, or due to, the effect of another polypeptide or compound, but instead are without other substantial chemical intermediates. <span style="display: none">[0098] </span> </div>
</li> <li> <para-num num="[0099]"> </para-num> <div class="description-line" id="P-0099" num="0099">Moreover, the nucleic acid molecule of the invention can comprise only a portion of the T5 nucleic acid sequence of SEQ ID NO: 1, 3, 5, 7, 9, 11, or 13, e.g., a fragment that can be used as a probe or primer, or a fragment encoding a biologically active portion of NOVX. Fragments provided herein are defined as sequences of at least 6 (contiguous) nucleic acids or at least 4 (contiguous) amino acids, a length sufficient to allow for specific hybridization in the case of nucleic acids or for specific recognition of an epitope in the case of amino acids, respectively, and are at most some portion less than a full length sequence. Fragments may be derived from any contiguous portion of a nucleic acid or amino acid sequence of choice. Derivatives are nucleic acid sequences or amino acid sequences formed from the native compounds either directly or by modification or partial substitution. Analogs are nucleic acid sequences or amino acid sequences that have a structure similar to, but not identical to, the native compound but differs from it in respect to certain components or side chains. Analogs may be synthetic or from a different evolutionary origin and may have a similar or opposite metabolic activity compared to wild type. <span style="display: none">[0099] </span> </div>
</li> <li> <para-num num="[0100]"> </para-num> <div class="description-line" id="P-0100" num="0100">Derivatives and analogs may be full length or other than full length, if the derivative or analog contains a modified nucleic acid or amino acid, as described below. Derivatives or analogs of the nucleic acids or proteins of the invention include, but are not limited to, molecules comprising regions that are substantially homologous to the nucleic acids or proteins of the invention, in various embodiments, by at least about 70%, 80%, 85%, 90%, 95%, 98%, or even 99% identity (with a preferred identity of 80-99%) over a nucleic acid or amino acid sequence of identical size or when compared to an aligned sequence in which the alignment is done by a computer homology program known in the art, or whose encoding nucleic acid is capable of hybridizing to the complement of a sequence encoding the aforementioned proteins under stringent, moderately stringent, or low stringent conditions. See e.g Ausubel, et al., <span style="display: none">[0100] </span> <i>CURRENT PROTOCOLS IN MOLECULAR BIOLOGY</i>, John Wiley &amp; Sons, New York, N.Y., 1993, and below. An exemplary program is the Gap program (Wisconsin Sequence Analysis Package, Version 8 for UNIX, Genetics Computer Group, University Research Park, Madison, Wis.) using the default settings, which uses the algorithm of Smith and Waterman (Adv. Appl. Math., 1981, 2: 482-489, which is incorporated herein by reference in its entirety). </div>
</li> <li> <para-num num="[0101]"> </para-num> <div class="description-line" id="P-0101" num="0101">A “homologous nucleic acid sequence” or “homologous amino acid sequence,” or variations thereof, refer to sequences characterized by a homology at the nucleotide level or amino acid level as discussed above. Homologous nucleotide sequences encode those sequences coding for isoforms of a NOVX polypeptide. Isoforms can be expressed in different tissues of the same organism as a result of, for example, alternative splicing of RNA. Alternatively, isoforms can be encoded by different genes. In the present invention, homologous nucleotide sequences include nucleotide sequences encoding for a NOVX polypeptide of species other than humans, including, but not limited to, mammals, and thus can include, e.g., mouse, rat, rabbit, dog, cat cow, horse, and other organisms. Homologous nucleotide sequences also include, but are not limited to, naturally occurring allelic variations and mutations of the nucleotide sequences set forth herein. A homologous nucleotide sequence does not, however, include the nucleotide sequence encoding human NOVX protein. Homologous nucleic acid sequences include those nucleic acid sequences that encode conservative amino acid substitutions (see below) in SEQ ID NO: 2, 4, 6, 8, 10, 12 or 14, as well as a polypeptide having NOVX activity. Biological activities of the NOVX proteins are described below. A homologous amino acid sequence does not encode the amino acid sequence of a human NOVX polypeptide. <span style="display: none">[0101] </span> </div>
</li> <li> <para-num num="[0102]"> </para-num> <div class="description-line" id="P-0102" num="0102">The nucleotide sequence determined from the cloning of the human NOVX gene allows for the generation of probes and primers designed for use in identifying and/or cloning NOVX homologues in other cell types, e.g., from other tissues, as well as NOVX homologues from other mammals. The probe/primer typically comprises a substantially purified oligonucleotide. The oligonucleotide typically comprises a region of nucleotide sequence that hybridizes under stringent conditions to at least about 12, 25, 50, 100, 150, 200, 250, 300, 350 or 400 or more consecutive sense strand nucleotide sequence of SEQ ID NO: 1, 3, 5, 7, 9, 11, or 13; or an anti-sense strand nucleotide sequence of SEQ ID NO: 1, 3, 5, 7, 9, 11, or 13; or of a naturally occurring mutant of SEQ ID NO: 1, 3, 5, 7, 9, 11, or 13. <span style="display: none">[0102] </span> </div>
</li> <li> <para-num num="[0103]"> </para-num> <div class="description-line" id="P-0103" num="0103">Probes based on the human NOVX nucleotide sequence can be used to detect transcripts or genomic sequences encoding the same or homologous proteins. In various embodiments, the probe further comprises a label group attached thereto, e.g., the label group can be a radioisotope, a fluorescent compound, an enzyme, or an enzyme co-factor. Such probes can be used as a part of a diagnostic test kit for identifying cells or tissue which misexpress a NOVX protein, such as by measuring a level of a NOVX-encoding nucleic acid in a sample of cells from a subject e.g., detecting NOVX mRNA levels or determining whether a genomic NOVX gene has been mutated or deleted. <span style="display: none">[0103] </span> </div>
</li> <li> <para-num num="[0104]"> </para-num> <div class="description-line" id="P-0104" num="0104">A “polypeptide having a biologically active portion of NOVX” refers to polypeptides exhibiting activity similar, but not necessarily identical to, an activity of a polypeptide of the present invention, including mature forms, as measured in a particular biological assay, with or without dose dependency. A nucleic acid fragment encoding a “biologically active portion of NOVX” can be prepared by isolating a portion of SEQ ID NO: 1, 3, 5, 7, 9, 11, or 13 that encodes a polypeptide having a NOVX biological activity (biological activities of the NOVX proteins are described below), expressing the encoded portion of NOVX protein (e.g., by recombinant expression in vitro) and assessing the activity of the encoded portion of NOVX. For example, a nucleic acid fragment encoding a biologically active portion of NOVX can optionally include an ATP-binding domain. In another embodiment, a nucleic acid fragment encoding a biologically active portion of NOVX includes one or more regions. <span style="display: none">[0104] </span> </div>
</li> <li> <para-num num="[0105]"> </para-num> <div class="description-line" id="P-0105" num="0105">NOVX Variants <span style="display: none">[0105] </span> </div>
</li> <li> <para-num num="[0106]"> </para-num> <div class="description-line" id="P-0106" num="0106">The invention further encompasses nucleic acid molecules that differ from the nucleotide sequences shown in SEQ ID NO: 1, 3, 5, 7, 9, 11, or 13 due to the degeneracy of the genetic code. These nucleic acids thus encode the same NOVX protein as that encoded by the nucleotide sequence shown in SEQ ID NO: 1, 3, 5, 7, 9, 11, or 13 e.g., the polypeptide of SEQ ID NO: 2, 4, 6, 8, 10, 12 or 14. In another embodiment, an isolated nucleic acid molecule of the invention has a nucleotide sequence encoding a protein having an amino acid sequence shown in SEQ ID NO: 2, 4, 6, 8, 10, 12 or 14. <span style="display: none">[0106] </span> </div>
</li> <li> <para-num num="[0107]"> </para-num> <div class="description-line" id="P-0107" num="0107">In addition to the human NOVX nucleotide sequence shown in SEQ ID NO: 1, 3, 5, 7, 9, 11, or 13, it will be appreciated by those skilled in the art that DNA sequence polymorphisms that lead to changes in the amino acid sequences of NOVX may exist within a population (e.g., the human population). Such genetic polymorphism in the NOVX gene may exist among individuals within a population due to natural allelic variation. As used herein, the terms “gene” and “recombinant gene” refer to nucleic acid molecules comprising an open reading frame encoding a NOVX protein, preferably a mammalian NOVX protein. Such natural allelic variations can typically result in 1-5% variance in the nucleotide sequence of the NOVX gene. Any and all such nucleotide variations and resulting amino acid polymorphisms in NOVX that are the result of natural allelic variation and that do not alter the functional activity of NOVX are intended to be within the scope of the invention. <span style="display: none">[0107] </span> </div>
</li> <li> <para-num num="[0108]"> </para-num> <div class="description-line" id="P-0108" num="0108">Moreover, nucleic acid molecules encoding NOVX proteins from other species, and thus that have a nucleotide sequence that differs from the human sequence of SEQ ID NO: 1, 3, 5, 7, 9, 11, or 13 are intended to be within the scope of the invention. Nucleic acid molecules corresponding to natural allelic variants and homologues of the NOVX cDNAs of the invention can be isolated based on their homology to the human NOVX nucleic acids disclosed herein using the human cDNAs, or a portion thereof, as a hybridization probe according to standard hybridization techniques under stringent hybridization conditions. For example, a soluble human NOVX cDNA can be isolated based on its homology to human membrane-bound NOVX. Likewise, a membrane-bound human NOVX cDNA can be isolated based on its homology to soluble human NOVX. <span style="display: none">[0108] </span> </div>
</li> <li> <para-num num="[0109]"> </para-num> <div class="description-line" id="P-0109" num="0109">Accordingly, in another embodiment, an isolated nucleic acid molecule of the invention is at least 6 nucleotides in length and hybridizes under stringent conditions to the nucleic acid molecule comprising the nucleotide sequence of SEQ ID NO: 1, 3, 5, 7, 9, 11, or 13. In another embodiment, the nucleic acid is at least 10, 25, 50, 100, 250, 500 or 750 nucleotides in length. In another embodiment, an isolated nucleic acid molecule of the invention hybridizes to the coding region. As used herein, the term “hybridizes under stringent conditions” is intended to describe conditions for hybridization and washing under which nucleotide sequences at least 60% homologous to each other typically remain hybridized to each other. <span style="display: none">[0109] </span> </div>
</li> <li> <para-num num="[0110]"> </para-num> <div class="description-line" id="P-0110" num="0110">Homologs (i.e., nucleic acids encoding NOVX proteins derived from species other than human) or other related sequences (e.g., paralogs) can be obtained by low, moderate or high stringency hybridization with all or a portion of the particular human sequence as a probe using methods well known in the art for nucleic acid hybridization and cloning. <span style="display: none">[0110] </span> </div>
</li> <li> <para-num num="[0111]"> </para-num> <div class="description-line" id="P-0111" num="0111">As used herein, the phrase “stringent hybridization conditions” refers to conditions under which a probe, primer or oligonucleotide will hybridize to its target sequence, but to no other sequences. Stringent conditions are sequence-dependent and will be different in different circumstances. Longer sequences hybridize specifically at higher temperatures than shorter sequences. Generally, stringent conditions are selected to be about 5° C. lower than the thermal melting point (T<span style="display: none">[0111] </span> <sub>m</sub>) for the specific sequence at a defined ionic strength and pH. The Tm is the temperature (under defined ionic strength, pH and nucleic acid concentration) at which 50% of the probes complementary to the target sequence hybridize to the target sequence at equilibrium. Since the target sequences are generally present at excess, at Tm, 50% of the probes are occupied at equilibrium. Typically, stringent conditions will be those in which the salt concentration is less than about 1.0 M sodium ion, typically about 0.01 to 1.0 M sodium ion (or other salts) at pH 7.0 to 8.3 and the temperature is at least about 30° C. for short probes, primers or oligonucleotides (e.g., 10 nt to 50 nt) and at least about 60° C. for longer probes, primers and oligonucleotides. Stringent conditions may also be achieved with the addition of destabilizing agents, such as formamide. </div>
</li> <li> <para-num num="[0112]"> </para-num> <div class="description-line" id="P-0112" num="0112">Stringent conditions are known to those skilled in the art and can be found in <span style="display: none">[0112] </span> <i>CURRENT PROTOCOLS IN MOLECULAR BIOLOGY</i>, John Wiley &amp; Sons, N.Y. (1989), 6.3.1-6.3.6. Preferably, the conditions are such that sequences at least about 65%, 70%, 75%, 85%, 90%, 95%, 98%, or 99% homologous to each other typically remain hybridized to each other. A non-limiting example of stringent hybridization conditions is hybridization in a high salt buffer comprising 6X SSC, 50 mM Tris-HCl (pH 7.5), 1 mM EDTA, 0.02% PVP, 0.02% Ficoll, 0.02% BSA, and 500 mg/ml denatured salmon sperm DNA at 65° C. This hybridization is followed by one or more washes in 0.2× SSC, 0.01% BSA at 50° C. An isolated nucleic acid molecule of the invention that hybridizes under stringent conditions to the sequence of SEQ ID NO: 1, 3, 5, 7, 9, 11, or 13 corresponds to a naturally occurring nucleic acid molecule. As used herein, a “naturally-occurring” nucleic acid molecule refers to an RNA or DNA molecule having a nucleotide sequence that occurs in nature (e.g., encodes a natural protein). </div>
</li> <li> <para-num num="[0113]"> </para-num> <div class="description-line" id="P-0113" num="0113">In a second embodiment, a nucleic acid sequence that is hybridizable to the nucleic acid molecule comprising the nucleotide sequence of SEQ ID NO: 1, 3, 5, 7, 9, 11, or 13, or fragments, analogs or derivatives thereof, under conditions of moderate stringency is provided. A non-limiting example of moderate stringency hybridization conditions are hybridization in 6× SSC, 5× Denhardt's solution, 0.5% SDS and 100 mg/ml denatured salmon sperm DNA at 55° C., followed by one or more washes in 1× SSC, 0.1% SDS at 37° C. Other conditions of moderate stringency that may be used are well known in the art. See, e.g., Ausubel et al. (eds.), 1993, <span style="display: none">[0113] </span> <i>CURRENT PROTOCOLS IN MOLECULAR BIOLOGY</i>, John Wiley &amp; Sons, NY, and Kriegler, 1990, <i>GENE TRANSFER AND EXPRESSION, A LABORATORY MANUAL</i>, Stockton Press, NY. </div>
</li> <li> <para-num num="[0114]"> </para-num> <div class="description-line" id="P-0114" num="0114">In a third embodiment, a nucleic acid that is hybridizable to the nucleic acid molecule comprising the nucleotide sequence of SEQ ID NO: 1, 3, 5, 7, 9, 11, or 13, or fragments, analogs or derivatives thereof, under conditions of low stringency, is provided. A non-limiting example of low stringency hybridization conditions are hybridization in 35% formamide, 5X SSC, 50 mM Tris-HCl (pH 7.5), 5 mM EDTA, 0.02% PVP, 0.02% Ficoll, 0.2% BSA, 100 mg/ml denatured salmon sperm DNA, 10% (wt/vol) dextran sulfate at 40° C., followed by one or more washes in 2× SSC, 25 mM Tris-HCl (pH 7.4), 5 mM EDTA, and 0.1% SDS at 50° C. Other conditions of low stringency that may be used are well known in the art (e.g., as employed for cross-species hybridizations). See, e.g., Ausubel et al. (eds.), 1993, <span style="display: none">[0114] </span> <i>CURRENT PROTOCOLS IN MOLECULAR BIOLOGY</i>, John Wiley &amp; Sons, NY, and Kriegler, 1990, <i>GENE TRANSFER AND EXPRESSION, A LABORATORY MANUAL, Stockton Press, NY; Shilo and Weinberg, </i>1981, <i>Proc Natl Acad Sci USA </i>78: 6789-6792. </div>
</li> <li> <para-num num="[0115]"> </para-num> <div class="description-line" id="P-0115" num="0115">Conservative Mutations <span style="display: none">[0115] </span> </div>
</li> <li> <para-num num="[0116]"> </para-num> <div class="description-line" id="P-0116" num="0116">In addition to naturally-occurring allelic variants of the NOVX sequence that may exist in the population, the skilled artisan will further appreciate that changes can be introduced by mutation into the nucleotide sequence of SEQ ID NO: 1, 3, 5, 7, 9, 11, or 13, thereby leading to changes in the amino acid sequence of the encoded NOVX protein, without altering the functional ability of the NOVX protein. For example, nucleotide substitutions leading to amino acid substitutions at “non-essential” amino acid residues can be made in the sequence of SEQ ID NO: 1, 3, 5, 7, 9, 11, or 13. A “non-essential” amino acid residue is a residue that can be altered from the wild-type sequence of NOVX without altering the biological activity, whereas an “essential” amino acid residue is required for biological activity. For example, amino acid residues that are conserved among the NOVX proteins of the present invention, are predicted to be particularly unamenable to alteration. <span style="display: none">[0116] </span> </div>
</li> <li> <para-num num="[0117]"> </para-num> <div class="description-line" id="P-0117" num="0117">Another aspect of the invention pertains to nucleic acid molecules encoding NOVX proteins that contain changes in amino acid residues that are not essential for activity. Such NOVX proteins differ in amino acid sequence from SEQ ID NO: 2, 4, 6, 8, 10, 12 or 14, yet retain biological activity. In one embodiment, the isolated nucleic acid molecule comprises a nucleotide sequence encoding a protein, wherein the protein comprises an amino acid sequence at least about 75% homologous to the amino acid sequence of SEQ ID NO: 2, 4, 6, 8, 10, 12 or 14. Preferably, the protein encoded by the nucleic acid is at least about 80% homologous to SEQ ID NO: 2, 4, 6, 8, 10, 12 or 14, more preferably at least about 90%, 95%, 98%, and most preferably at least about 99% homologous to SEQ ID NO: 2, 4, 6, 8, 10, 12 or 14. <span style="display: none">[0117] </span> </div>
</li> <li> <para-num num="[0118]"> </para-num> <div class="description-line" id="P-0118" num="0118">An isolated nucleic acid molecule encoding a NOVX protein homologous to the protein of can be created by introducing one or more nucleotide substitutions, additions or deletions into the nucleotide sequence of SEQ ID NO: 1, 3, 5, 7, 9, 11, or 13, such that one or more amino acid substitutions, additions or deletions are introduced into the encoded protein. <span style="display: none">[0118] </span> </div>
</li> <li> <para-num num="[0119]"> </para-num> <div class="description-line" id="P-0119" num="0119">Mutations can be introduced into the nucleotide sequence of SEQ ID NO: 1, 3, 5, 7, 9, 11, or 13 by standard techniques, such as site-directed mutagenesis and PCR-mediated mutagenesis. Preferably, conservative amino acid substitutions are made at one or more predicted non-essential amino acid residues. A “conservative amino acid substitution” is one in which the amino acid residue is replaced with an amino acid residue having a similar side chain. Families of amino acid residues having similar side chains have been defined in the art. These families include amino acids with basic side chains (e.g., lysine, arginine, histidine), acidic side chains (e.g., aspartic acid, glutamic acid), uncharged polar side chains (e.g., glycine, asparagine, glutamine, serine, threonine, tyrosine, cysteine), nonpolar side chains (e.g., alanine, valine, leucine, isoleucine, proline, phenylalanine, methionine, tryptophan), beta-branched side chains (e.g., threonine, valine, isoleucine) and aromatic side chains (e.g., tyrosine, phenylalanine, tryptophan, histidine). Thus, a predicted nonessential amino acid residue in NOVX is replaced with another amino acid residue from the same side chain family. Alternatively, in another embodiment, mutations can be introduced randomly along all or part of a NOVX coding sequence, such as by saturation mutagenesis, and the resultant mutants can be screened for NOVX biological activity to identify mutants that retain activity. Following mutagenesis of SEQ ID NO: 1, 3, 5, 7, 9, 11, or 13 the encoded protein can be expressed by any recombinant technology known in the art and the activity of the protein can be determined. <span style="display: none">[0119] </span> </div>
</li> <li> <para-num num="[0120]"> </para-num> <div class="description-line" id="P-0120" num="0120">In one embodiment, a mutant NOVX protein can be assayed for (1) the ability to form protein:protein interactions with other NOVX proteins, other cell-surface proteins, or biologically active portions thereof, (2) complex formation between a mutant NOVX protein and a NOVX receptor; (3) the ability of a mutant NOVX protein to bind to an intracellular target protein or biologically active portion thereof; (e.g., avidin proteins); (4) the ability to bind NOVX protein; or (5) the ability to specifically bind an anti-NOVX protein antibody. <span style="display: none">[0120] </span> </div>
</li> <li> <para-num num="[0121]"> </para-num> <div class="description-line" id="P-0121" num="0121">Antisense NOVX Nucleic Acids <span style="display: none">[0121] </span> </div>
</li> <li> <para-num num="[0122]"> </para-num> <div class="description-line" id="P-0122" num="0122">Another aspect of the invention pertains to isolated antisense nucleic acid molecules that are hybridizable to or complementary to the nucleic acid molecule comprising the nucleotide sequence of SEQ ID NO: 1, 3, 5, 7, 9, 11, or 13, or fragments, analogs or derivatives thereof. An “antisense” nucleic acid comprises a nucleotide sequence that is complementary to a “sense” nucleic acid encoding a protein, e.g., complementary to the coding strand of a double-stranded cDNA molecule or complementary to an mRNA sequence. In specific aspects, antisense nucleic acid molecules are provided that comprise a sequence complementary to at least about 10, 25, 50, 100, 250 or 500 nucleotides or an entire NOVX coding strand, or to only a portion thereof. Nucleic acid molecules encoding fragments, homologs, derivatives and analogs of a NOVX protein of SEQ ID NO: 2, 4, 6, 8, 10, 12 or 14 or antisense nucleic acids complementary to a NOVX nucleic acid sequence of SEQ ID NO: 1, 3, 5, 7, 9, 11, or 13 are additionally provided. <span style="display: none">[0122] </span> </div>
</li> <li> <para-num num="[0123]"> </para-num> <div class="description-line" id="P-0123" num="0123">In one embodiment, an antisense nucleic acid molecule is antisense to a “coding region” of the coding strand of a nucleotide sequence encoding NOVX. The term “coding region” refers to the region of the nucleotide sequence comprising codons which are translated into amino acid residues (e.g, the protein coding region of human NOVX corresponds to SEQ ID NO: 2, 4, 6, 8, 10, 12 or 14). In another embodiment, the antisense nucleic acid molecule is antisense to a “noncoding region” of the coding strand of a nucleotide sequence encoding NOVX. The term “noncoding region” refers to 5′ and 3′ sequences which flank the coding region that are not translated into amino acids (i.e., also referred to as 5′ and 3′ untranslated regions). <span style="display: none">[0123] </span> </div>
</li> <li> <para-num num="[0124]"> </para-num> <div class="description-line" id="P-0124" num="0124">Given the coding strand sequences encoding NOVX disclosed herein (e.g., SEQ ID NO: 1, 3, 5, 7, 9, 11, or 13), antisense nucleic acids of the invention can be designed according to the rules of Watson and Crick or Hoogsteen base pairing. The antisense nucleic acid molecule can be complementary to the entire coding region of NOVX mRNA, but more preferably is an oligonucleotide that is antisense to only a portion of the coding or noncoding region of NOVX mRNA. For example, the antisense oligonucleotide can be complementary to the region surrounding the translation start site of NOVX mRNA. An antisense oligonucleotide can be, for example, about 5, 10, 15, 20, 25, 30, 35, 40, 45 or 50 nucleotides in length. An antisense nucleic acid of the invention can be constructed using chemical synthesis or enzymatic ligation reactions using procedures known in the art. For example, an antisense nucleic acid (e.g., an antisense oligonucleotide) can be chemically synthesized using naturally occurring nucleotides or variously modified nucleotides designed to increase the biological stability of the molecules or to increase the physical stability of the duplex formed between the antisense and sense nucleic acids, e.g., phosphorothioate derivatives and acridine substituted nucleotides can be used. <span style="display: none">[0124] </span> </div>
</li> <li> <para-num num="[0125]"> </para-num> <div class="description-line" id="P-0125" num="0125">Examples of modified nucleotides that can be used to generate the antisense nucleic acid include: 5-fluorouracil, 5-bromouracil, 5-chlorouracil, 5-iodouracil, hypoxanthine, xanthine, 4-acetylcytosine, 5-(carboxyhydroxylmethyl) uracil, 5-carboxymethylaminomethyl-2-thiouridine, 5-carboxymethylaminomethyluracil, dihydrouracil, beta-D-galactosylqueosine, inosine, N6-isopentenyladenine, 1-methylguanine, 1-methylinosine, 2,2-dimethylguanine, 2-methyladenine, 2-methylguanine, 3-methylcytosine, 5-methylcytosine, N6-adenine, 7-methylguanine, 5-methylaminomethyluracil, 5-methoxyaminomethyl-2-thiouracil, beta-D-mannosylqueosine, 5′-methoxycarboxymethyluracil, 5-methoxyuracil, 2-methylthio-N6-isopentenyladenine, uracil-5-oxyacetic acid (v), wybutoxosine, pseudouracil, queosine, 2-thiocytosine, 5-methyl-2-thiouracil, 2-thiouracil, 4-thiouracil, 5-methyluracil, uracil-5-oxyacetic acid methylester, uracil-5-oxyacetic acid (v), 5-methyl-2-thiouracil, 3-(3-amino-3-N-2-carboxypropyl) uracil, (acp3)w, and 2,6-diaminopurine. Alternatively, the antisense nucleic acid can be produced biologically using an expression vector into which a nucleic acid has been subcloned in an antisense orientation (i.e., RNA transcribed from the inserted nucleic acid will be of an antisense orientation to a target nucleic acid of interest, described further in the following subsection). <span style="display: none">[0125] </span> </div>
</li> <li> <para-num num="[0126]"> </para-num> <div class="description-line" id="P-0126" num="0126">The antisense nucleic acid molecules of the invention are typically administered to a subject or generated in situ such that they hybridize with or bind to cellular mRNA and/or genomic DNA encoding a NOVX protein to thereby inhibit expression of the protein, e.g., by inhibiting transcription and/or translation. The hybridization can be by conventional nucleotide complementarity to form a stable duplex, or, for example, in the case of an antisense nucleic acid molecule that binds to DNA duplexes, through specific interactions in the major groove of the double helix. An example of a route of administration of antisense nucleic acid molecules of the invention includes direct injection at a tissue site. Alternatively, antisense nucleic acid molecules can be modified to target selected cells and then administered systemically. For example, for systemic administration, antisense molecules can be modified such that they specifically bind to receptors or antigens expressed on a selected cell surface, e.g., by linking the antisense nucleic acid molecules to peptides or antibodies that bind to cell surface receptors or antigens. The antisense nucleic acid molecules can also be delivered to cells using the vectors described herein. To achieve sufficient intracellular concentrations of antisense molecules, vector constructs in which the antisense nucleic acid molecule is placed under the control of a strong pol II or pol III promoter are preferred. <span style="display: none">[0126] </span> </div>
</li> <li> <para-num num="[0127]"> </para-num> <div class="description-line" id="P-0127" num="0127">In yet another embodiment, the antisense nucleic acid molecule of the invention is an α-anomeric nucleic acid molecule. An α-anomeric nucleic acid molecule forms specific double-stranded hybrids with complementary RNA in which, contrary to the usual β-units, the strands run parallel to each other (Gaultier et al. (1987) <span style="display: none">[0127] </span> <i>Nucleic Acids Res </i>15: 6625-6641). The antisense nucleic acid molecule can also comprise a 2′-o-methylribonucleotide (Inoue et al. (1987) <i>Nucleic Acids Res </i>15: 6131-6148) or a chimeric RNA -DNA analogue (Inoue et al. (1987) <i>FEBS Lett </i>215: 327-330). </div>
</li> <li> <para-num num="[0128]"> </para-num> <div class="description-line" id="P-0128" num="0128">Such modifications include, by way of nonlimiting example, modified bases, and nucleic acids whose sugar phosphate backbones are modified or derivatized. These modifications are carried out at least in part to enhance the chemical stability of the modified nucleic acid, such that they may be used, for example, as antisense binding nucleic acids in therapeutic applications in a subject. <span style="display: none">[0128] </span> </div>
</li> <li> <para-num num="[0129]"> </para-num> <div class="description-line" id="P-0129" num="0129">NOVX Ribozymes and PNA Moieties <span style="display: none">[0129] </span> </div>
</li> <li> <para-num num="[0130]"> </para-num> <div class="description-line" id="P-0130" num="0130">In still another embodiment, an antisense nucleic acid of the invention is a ribozyme. Ribozymes are catalytic RNA molecules with ribonuclease activity that are capable of cleaving a single-stranded nucleic acid, such as a mRNA, to which they have a complementary region. Thus, ribozymes (e.g., hammerhead ribozymes (described in Haselhoff and Gerlach (1988) <span style="display: none">[0130] </span> <i>Nature </i>334:585-591)) can be used to catalytically cleave NOVX mRNA transcripts to thereby inhibit translation of NOVX mRNA. A ribozyme having specificity for a NOVX-encoding nucleic acid can be designed based upon the nucleotide sequence of a NOVX DNA disclosed herein (i.e., SEQ ID NO: 1, 3, 5, 7, 9, 11, or 13). For example, a derivative of a Tetrahymena L-19 IVS RNA can be constructed in which the nucleotide sequence of the active site is complementary to the nucleotide sequence to be cleaved in a NOVX-encoding mRNA. See, e.g., Cech et al. U.S. Pat. No. 4,987,071; and Cech et al. U.S. Pat. No. 5,116,742. Alternatively, NOVX mRNA can be used to select a catalytic RNA having a specific ribonuclease activity from a pool of RNA molecules. See, e.g., Bartel et al., (1993) <i>Science </i>261:1411-1418. </div>
</li> <li> <para-num num="[0131]"> </para-num> <div class="description-line" id="P-0131" num="0131">Alternatively, NOVX gene expression can be inhibited by targeting nucleotide sequences complementary to the regulatory region of the NOVX (e.g., the NOVX promoter and/or enhancers) to form triple helical structures that prevent transcription of the NOVX gene in target cells. See generally, Helene. (1991) <span style="display: none">[0131] </span> <i>Anticancer Drug Des. </i>6: 569-84; Helene. et al. (1992) <i>Ann. N.Y. Acad. Sci. </i>660:27-36; and Maher (1992) <i>Bioassays </i>14: 807-15. </div>
</li> <li> <para-num num="[0132]"> </para-num> <div class="description-line" id="P-0132" num="0132">In various embodiments, the nucleic acids of NOVX can be modified at the base moiety, sugar moiety or phosphate backbone to improve, e.g., the stability, hybridization, or solubility of the molecule. For example, the deoxyribose phosphate backbone of the nucleic acids can be modified to generate peptide nucleic acids (see Hyrup et al. (1996) <span style="display: none">[0132] </span> <i>Bioorg Med Chem </i>4: 5-23). As used herein, the terms “peptide nucleic acids” or “PNAS” refer to nucleic acid mimics, e.g., DNA mimics, in which the deoxyribose phosphate backbone is replaced by a pseudopeptide backbone and only the four natural nucleobases are retained. The neutral backbone of PNAs has been shown to allow for specific hybridization to DNA and RNA under conditions of low ionic strength. The synthesis of PNA oligomers can be performed using standard solid phase peptide synthesis protocols as described in Hyrup et al. (1996) above; Perry-O'Keefe et al. (1996) <i>PNAS </i>93: 14670-675. </div>
</li> <li> <para-num num="[0133]"> </para-num> <div class="description-line" id="P-0133" num="0133">PNAs of NOVX can be used in therapeutic and diagnostic applications. For example, PNAs can be used as antisense or antigene agents for sequence-specific modulation of gene expression by, e.g., inducing transcription or translation arrest or inhibiting replication. PNAs of NOVX can also be used, e.g., in the analysis of single base pair mutations in a gene by, e.g., PNA directed PCR clamping; as artificial restriction enzymes when used in combination with other enzymes, e.g., S1 nucleases (Hyrup B. (1996) above); or as probes or primers for DNA sequence and hybridization (Hyrup et al. (1996), above; Perry-O'Keefe (1996), above). <span style="display: none">[0133] </span> </div>
</li> <li> <para-num num="[0134]"> </para-num> <div class="description-line" id="P-0134" num="0134">In another embodiment, PNAs of NOVX can be modified, e.g., to enhance their stability or cellular uptake, by attaching lipophilic or other helper groups to PNA, by the formation of PNA-DNA chimeras, or by the use of liposomes or other techniques of drug delivery known in the art. For example, PNA-DNA chimeras of NOVX can be generated that may combine the advantageous properties of PNA and DNA. Such chimeras allow DNA recognition enzymes, e.g., RNase H and DNA polymerases, to interact with the DNA portion while the PNA portion would provide high binding affinity and specificity. PNA-DNA chimeras can be linked using linkers of appropriate lengths selected in terms of base stacking, number of bonds between the nucleobases, and orientation (Hyrup (1996) above). The synthesis of PNA-DNA chimeras can be performed as described in Hyrup (1996) above and Finn et al. (1996) <span style="display: none">[0134] </span> <i>Nucl Acids Res </i>24: 3357-63. For example, a DNA chain can be synthesized on a solid support using standard phosphoramidite coupling chemistry, and modified nucleoside analogs, e.g., 5′-(4-methoxytrityl) amino-5′-deoxy-thymidine phosphoramidite, can be used between the PNA and the 5′ end of DNA (Mag et al. (1989) <i>Nucl Acid Res </i>17: 5973-88). PNA monomers are then coupled in a stepwise manner to produce a chimeric molecule with a 5′ PNA segment and a 3′ DNA segment (Finn et al. (1996) above). Alternatively, chimeric molecules can be synthesized with a 5′ DNA segment and a 3′ PNA segment. See, Petersen et al. (1975) <i>Bioorg Med Chem Lett </i>5: 1119-11124. </div>
</li> <li> <para-num num="[0135]"> </para-num> <div class="description-line" id="P-0135" num="0135">In other embodiments, the oligonucleotide may include other appended groups such as peptides (e.g., for targeting host cell receptors in vivo), or agents facilitating transport across the cell membrane (see, e.g., Letsinger et al., 1989, <span style="display: none">[0135] </span> <i>Proc. Natl. Acad. Sci. U.S.A. </i>86:6553-6556; Lemaitre et al., 1987, <i>Proc. Natl. Acad. Sci. </i>84:648-652; PCT Publication No. W088/09810) or the blood-brain barrier (see, e.g., PCT Publication No. W089/10134). In addition, oligonucleotides can be modified with hybridization triggered cleavage agents (See, e.g., Krol et al., 1988, <i>BioTechniques </i>6:958-976) or intercalating agents. (See, e.g., Zon, 1988, <i>Pharm. Res. </i>5: 539-549). To this end, the oligonucleotide may be conjugated to another molecule, e.g., a peptide, a hybridization triggered cross-linking agent, a transport agent, a hybridization-triggered cleavage agent, etc. </div>
</li> <li> <para-num num="[0136]"> </para-num> <div class="description-line" id="P-0136" num="0136">NOVX Polypeptides <span style="display: none">[0136] </span> </div>
</li> <li> <para-num num="[0137]"> </para-num> <div class="description-line" id="P-0137" num="0137">A NOVX polypeptide of the invention includes the NOVX-like protein whose sequence is provided in SEQ ID NO: 2, 4, 6, 8, 10, 12 or 14. The invention also includes a mutant or variant protein any of whose residues may be changed from the corresponding residue shown in SEQ ID NO: 2, 4, 6, 8, 10, 12 or 14 while still encoding a protein that maintains its NOVX-like activities and physiological functions, or a functional fragment thereof. In some embodiments, up to 20% or more of the residues may be so changed in the mutant or variant protein. In some embodiments, the NOVX polypeptide according to the invention is a mature polypeptide. <span style="display: none">[0137] </span> </div>
</li> <li> <para-num num="[0138]"> </para-num> <div class="description-line" id="P-0138" num="0138">In general, a NOVX -like variant that preserves NOVX-like function includes any variant in which residues at a particular position in the sequence have been substituted by other amino acids, and further include the possibility of inserting an additional residue or residues between two residues of the parent protein as well as the possibility of deleting one or more residues from the parent sequence. Any amino acid substitution, insertion, or deletion is encompassed by the invention. In favorable circumstances, the substitution is a conservative substitution as defined above. <span style="display: none">[0138] </span> </div>
</li> <li> <para-num num="[0139]"> </para-num> <div class="description-line" id="P-0139" num="0139">One aspect of the invention pertains to isolated NOVX proteins, and biologically active portions thereof, or derivatives, fragments, analogs or homologs thereof. Also provided are polypeptide fragments suitable for use as immunogens to raise anti-NOVX antibodies. In one embodiment, native NOVX proteins can be isolated from cells or tissue sources by an appropriate purification scheme using standard protein purification techniques. In another embodiment, NOVX proteins are produced by recombinant DNA techniques. Alternative to recombinant expression, a NOVX protein or polypeptide can be synthesized chemically using standard peptide synthesis techniques. <span style="display: none">[0139] </span> </div>
</li> <li> <para-num num="[0140]"> </para-num> <div class="description-line" id="P-0140" num="0140">An “isolated” or “purified” protein or biologically active portion thereof is substantially free of cellular material or other contaminating proteins from the cell or tissue source from which the NOVX protein is derived, or substantially free from chemical precursors or other chemicals when chemically synthesized. The language “substantially free of cellular material” includes preparations of NOVX protein in which the protein is separated from cellular components of the cells from which it is isolated or recombinantly produced. In one embodiment, the language “substantially free of cellular material” includes preparations of NOVX protein having less than about 30% (by dry weight) of non-NOVX protein (also referred to herein as a “contaminating protein”), more preferably less than about 20% of non-NOVX protein, still more preferably less than about 10% of non-NOVX protein, and most preferably less than about 5% non-NOVX protein. When the NOVX protein or biologically active portion thereof is recombinantly produced, it is also preferably substantially free of culture medium, i.e., culture medium represents less than about 20%, more preferably less than about 10%, and most preferably less than about 5% of the volume of the protein preparation. <span style="display: none">[0140] </span> </div>
</li> <li> <para-num num="[0141]"> </para-num> <div class="description-line" id="P-0141" num="0141">The language “substantially free of chemical precursors or other chemicals” includes preparations of NOVX protein in which the protein is separated from chemical precursors or other chemicals that are involved in the synthesis of the protein. In one embodiment, the language “substantially free of chemical precursors or other chemicals” includes preparations of NOVX protein having less than about 30% (by dry weight) of chemical precursors or non-NOVX chemicals, more preferably less than about 20% chemical precursors or non-NOVX chemicals, still more preferably less than about 10% chemical precursors or non-NOVX chemicals, and most preferably less than about 5% chemical precursors or non-NOVX chemicals. <span style="display: none">[0141] </span> </div>
</li> <li> <para-num num="[0142]"> </para-num> <div class="description-line" id="P-0142" num="0142">Biologically active portions of a NOVX protein include peptides comprising amino acid sequences sufficiently homologous to or derived from the amino acid sequence of the NOVX protein, e.g., the amino acid sequence shown in SEQ ID NO: 2, 4, 6, 8, 10, 12 or 14 that include fewer amino acids than the full length NOVX proteins, and exhibit at least one activity of a NOVX protein. Typically, biologically active portions comprise a domain or motif with at least one activity of the NOVX protein. A biologically active portion of a NOVX protein can be a polypeptide which is, for example, 10, 25, 50, 100 or more amino acids in length. <span style="display: none">[0142] </span> </div>
</li> <li> <para-num num="[0143]"> </para-num> <div class="description-line" id="P-0143" num="0143">A biologically active portion of a NOVX protein of the present invention may contain at least one of the above-identified domains conserved between the NOVX proteins, e.g. TSR modules. Moreover, other biologically active portions, in which other regions of the protein are deleted, can be prepared by recombinant techniques and evaluated for one or more of the functional activities of a native NOVX protein. <span style="display: none">[0143] </span> </div>
</li> <li> <para-num num="[0144]"> </para-num> <div class="description-line" id="P-0144" num="0144">In an embodiment, the NOVX protein has an amino acid sequence shown in SEQ ID NO: 2, 4, 6, 8, 10, 12 or 14. In other embodiments, the NOVX protein is substantially homologous to SEQ ID NO: 2, 4, 6, 8, 10, 12 or 14 and retains the functional activity of the protein of SEQ ID NO: 2, 4, 6, 8, 10, 12 or 14 yet differs in amino acid sequence due to natural allelic variation or mutagenesis, as described in detail below. Accordingly, in another embodiment, the NOVX protein is a protein that comprises an amino acid sequence at least about 45% homologous to the amino acid sequence of SEQ ID NO: 2, 4, 6, 8, 10, 12 or 14 and retains the functional activity of the NOVX proteins of SEQ ID NO: 2, 4, 6, 8, 10, 12 or 14. <span style="display: none">[0144] </span> </div>
</li> <li> <para-num num="[0145]"> </para-num> <div class="description-line" id="P-0145" num="0145">Determining Homology Between Two or More Sequence <span style="display: none">[0145] </span> </div>
</li> <li> <para-num num="[0146]"> </para-num> <div class="description-line" id="P-0146" num="0146">To determine the percent homology of two amino acid sequences or of two nucleic acids, the sequences are aligned for optimal comparison purposes (e.g., gaps can be introduced in either of the sequences being compared for optimal alignment between the sequences). The amino acid residues or nucleotides at corresponding amino acid positions or nucleotide positions are then compared. When a position in the first sequence is occupied by the same amino acid residue or nucleotide as the corresponding position in the second sequence, then the molecules are homologous at that position (i.e., as used herein amino acid or nucleic acid “homology” is equivalent to amino acid or nucleic acid “identity”). <span style="display: none">[0146] </span> </div>
</li> <li> <para-num num="[0147]"> </para-num> <div class="description-line" id="P-0147" num="0147">The nucleic acid sequence homology may be determined as the degree of identity between two sequences. The homology may be determined using computer programs known in the art, such as GAP software provided in the GCG program package. See, Needleman and Wunsch 1970 <span style="display: none">[0147] </span> <i>J Mol Biol </i>48: 443-453. Using GCG GAP software with the following settings for nucleic acid sequence comparison: GAP creation penalty of 5.0 and GAP extension penalty of 0.3, the coding region of the analogous nucleic acid sequences referred to above exhibits a degree of identity preferably of at least 70%, 75%, 80%, 85%, 90%, 95%, 98%, or 99%, with the CDS (encoding) part of the DNA sequence shown in SEQ ID NO: 1, 3, 5, 7, 9, 11, or 13. </div>
</li> <li> <para-num num="[0148]"> </para-num> <div class="description-line" id="P-0148" num="0148">The term “sequence identity” refers to the degree to which two polynucleotide or polypeptide sequences are identical on a residue-by-residue basis over a particular region of comparison. The term “percentage of sequence identity” is calculated by comparing two optimally aligned sequences over that region of comparison, determining the number of positions at which the identical nucleic acid base (e.g., A, T, C, G, U, or I, in the case of nucleic acids) occurs in both sequences to yield the number of matched positions, dividing the number of matched positions by the total number of positions in the region of comparison (i.e., the window size), and multiplying the result by 100 to yield the percentage of sequence identity. The term “substantial identity” as used herein denotes a characteristic of a polynucleotide sequence, wherein the polynucleotide comprises a sequence that has at least 80 percent sequence identity, preferably at least 85 percent identity and often 90 to 95 percent sequence identity, more usually at least 99 percent sequence identity as compared to a reference sequence over a comparison region. The term “percentage of positive residues” is calculated by comparing two optimally aligned sequences over that region of comparison, determining the number of positions at which the identical and conservative amino acid substitutions, as defined above, occur in both sequences to yield the number of matched positions, dividing the number of matched positions by the total number of positions in the region of comparison (i.e., the window size), and multiplying the result by 100 to yield the percentage of positive residues. <span style="display: none">[0148] </span> </div>
</li> <li> <para-num num="[0149]"> </para-num> <div class="description-line" id="P-0149" num="0149">Chimeric and Fusion Proteins <span style="display: none">[0149] </span> </div>
</li> <li> <para-num num="[0150]"> </para-num> <div class="description-line" id="P-0150" num="0150">The invention also provides NOVX chimeric or fusion proteins. As used herein, a NOVX “chimeric protein” or “fusion protein” comprises a NOVX polypeptide operatively linked to a non-NOVX polypeptide. An “NOVX polypeptide” refers to a polypeptide having an amino acid sequence corresponding to NOVX, whereas a “non-NOVX polypeptide” refers to a polypeptide having an amino acid sequence corresponding to a protein that is not substantially homologous to the NOVX protein, e g., a protein that is different from the NOVX protein and that is derived from the same or a different organism. Within a NOVX fusion protein the NOVX polypeptide can correspond to all or a portion of a NOVX protein. In one embodiment, a NOVX fusion protein comprises at least one biologically active portion of a NOVX protein. In another embodiment, a NOVX fusion protein comprises at least two biologically active portions of a NOVX protein. Within the fusion protein, the term “operatively linked” is intended to indicate that the NOVX polypeptide and the non-NOVX polypeptide are fused in-frame to each other. The non-NOVX polypeptide can be fused to the N-terminus or C-terminus of the NOVX polypeptide. <span style="display: none">[0150] </span> </div>
</li> <li> <para-num num="[0151]"> </para-num> <div class="description-line" id="P-0151" num="0151">For example, in one embodiment a NOVX fusion protein comprises a NOVX polypeptide operably linked to the extracellular domain of a second protein. Such fusion proteins can be further utilized in screening assays for compounds that modulate NOVX activity (such assays are described in detail below). <span style="display: none">[0151] </span> </div>
</li> <li> <para-num num="[0152]"> </para-num> <div class="description-line" id="P-0152" num="0152">In another embodiment, the fusion protein is a GST-NOVX fusion protein in which the NOVX sequences are fused to the C-terminus of the GST (i.e., glutathione S-transferase) sequences. Such fusion proteins can facilitate the purification of recombinant NOVX. <span style="display: none">[0152] </span> </div>
</li> <li> <para-num num="[0153]"> </para-num> <div class="description-line" id="P-0153" num="0153">In another embodiment, the fusion protein is a NOVX-immunoglobulin fusion protein in which the NOVX sequences comprising one or more domains are fused to sequences derived from a member of the immunoglobulin protein family. The NOVX-immunoglobulin fusion proteins of the invention can be incorporated into pharmaceutical compositions and administered to a subject to inhibit an interaction between a NOVX ligand and a NOVX protein on the surface of a cell, to thereby suppress NOVX-mediated signal transduction in vivo. In one nonlimiting example, a contemplated NOVX ligand of the invention is the NOVX receptor. The NOVX-immunoglobulin fusion proteins can be used to affect the bioavailability of a NOVX cognate ligand. Inhibition of the NOVX ligand/NOVX interaction may be useful therapeutically for both the treatment of proliferative and differentiative disorders, e,g, cancer as well as modulating (e.g., promoting or inhibiting) cell survival, as well as acute and chronic inflammatory disorders and hyperplastic wound healing, e.g. hypertrophic scars and keloids. Moreover, the NOVX-immunoglobulin fusion proteins of the invention can be used as immunogens to produce anti-NOVX antibodies in a subject, to purify NOVX ligands, and in screening assays to identify molecules that inhibit the interaction of NOVX with a NOVX ligand. <span style="display: none">[0153] </span> </div>
</li> <li> <para-num num="[0154]"> </para-num> <div class="description-line" id="P-0154" num="0154">A NOVX chimeric or fusion protein of the invention can be produced by standard recombinant DNA techniques. For example, DNA fragments coding for the different polypeptide sequences are ligated together in-frame in accordance with conventional techniques, e.g., by employing blunt-ended or stagger-ended termini for ligation, restriction enzyme digestion to provide for appropriate termini, filling-in of cohesive ends as appropriate, alkaline phosphatase treatment to avoid undesirable joining, and enzymatic ligation. In another embodiment, the fusion gene can be synthesized by conventional techniques including automated DNA synthesizers. Alternatively, PCR amplification of gene fragments can be carried out using anchor primers that give rise to complementary overhangs between two consecutive gene fragments that can subsequently be annealed and reamplified to generate a chimeric gene sequence (see, for example, Ausubel et al. (eds.) <span style="display: none">[0154] </span> <i>CURRENT PROTOCOLS IN MOLECULAR BIOLOGY</i>, John Wiley &amp; Sons, 1992). Moreover, many expression vectors are commercially available that already encode a fusion moiety (e.g., a GST polypeptide). A NOVX-encoding nucleic acid can be cloned into such an expression vector such that the fusion moiety is linked in-frame to the NOVX protein. </div>
</li> <li> <para-num num="[0155]"> </para-num> <div class="description-line" id="P-0155" num="0155">NOVX Agonists and Antagonists <span style="display: none">[0155] </span> </div>
</li> <li> <para-num num="[0156]"> </para-num> <div class="description-line" id="P-0156" num="0156">The present invention also pertains to variants of the NOVX proteins that function as either NOVX agonists (mimetics) or as NOVX antagonists. Variants of the NOVX protein can be generated by mutagenesis, e.g., discrete point mutation or truncation of the NOVX protein. An agonist of the NOVX protein can retain substantially the same, or a subset of, the biological activities of the naturally occurring form of the NOVX protein. An antagonist of the NOVX protein can inhibit one or more of the activities of the naturally occurring form of the NOVX protein by, for example, competitively binding to a downstream or upstream member of a cellular signaling cascade which includes the NOVX protein. Thus, specific biological effects can be elicited by treatment with a variant of limited function. In one embodiment, treatment of a subject with a variant having a subset of the biological activities of the naturally occurring form of the protein has fewer side effects in a subject relative to treatment with the naturally occurring form of the NOVX proteins. <span style="display: none">[0156] </span> </div>
</li> <li> <para-num num="[0157]"> </para-num> <div class="description-line" id="P-0157" num="0157">Variants of the NOVX protein that function as either NOVX agonists (mimetics) or as NOVX antagonists can be identified by screening combinatorial libraries of mutants, e.g., truncation mutants, of the NOVX protein for NOVX protein agonist or antagonist activity. In one embodiment, a variegated library of NOVX variants is generated by combinatorial mutagenesis at the nucleic acid level and is encoded by a variegated gene library. A variegated library of NOVX variants can be produced by, for example, enzymatically ligating a mixture of synthetic oligonucleotides into gene sequences such that a degenerate set of potential NOVX sequences is expressible as individual polypeptides, or alternatively, as a set of larger fusion proteins (e.g., for phage display) containing the set of NOVX sequences therein. There are a variety of methods which can be used to produce libraries of potential NOVX variants from a degenerate oligonucleotide sequence. Chemical synthesis of a degenerate gene sequence can be performed in an automatic DNA synthesizer, and the synthetic gene then ligated into an appropriate expression vector. Use of a degenerate set of genes allows for the provision, in one mixture, of all of the sequences encoding the desired set of potential NOVX sequences. Methods for synthesizing degenerate oligonucleotides are known in the art (see, e.g., Narang (1983) <span style="display: none">[0157] </span> <i>Tetrahedron </i>39:3; Itakura et al. (1984) <i>Annu Rev Biochem </i>53:323; Itakura et al. (1984) <i>Science </i>198:1056; Ike et al. (1983) <i>Nucl Acid Res </i>11:477. </div>
</li> <li> <para-num num="[0158]"> </para-num> <div class="description-line" id="P-0158" num="0158">Polypeptide Libraries <span style="display: none">[0158] </span> </div>
</li> <li> <para-num num="[0159]"> </para-num> <div class="description-line" id="P-0159" num="0159">In addition, libraries of fragments of the NOVX protein coding sequence can be used to generate a variegated population of NOVX fragments for screening and subsequent selection of variants of a NOVX protein. In one embodiment, a library of coding sequence fragments can be generated by treating a double stranded PCR fragment of a NOVX coding sequence with a nuclease under conditions wherein nicking occurs only about once per molecule, denaturing the double stranded DNA, renaturing the DNA to form double stranded DNA that can include sense/antisense pairs from different nicked products, removing single stranded portions from reformed duplexes by treatment with S1 nuclease, and ligating the resulting fragment library into an expression vector. By this method, an expression library can be derived which encodes N-terminal and internal fragments of various sizes of the NOVX protein. <span style="display: none">[0159] </span> </div>
</li> <li> <para-num num="[0160]"> </para-num> <div class="description-line" id="P-0160" num="0160">Several techniques are known in the art for screening gene products of combinatorial libraries made by point mutations or truncation, and for screening cDNA libraries for gene products having a selected property. Such techniques are adaptable for rapid screening of the gene libraries generated by the combinatorial mutagenesis of NOVX proteins. The most widely used techniques, which are amenable to high throughput analysis, for screening large gene libraries typically include cloning the gene library into replicable expression vectors, transforming appropriate cells with the resulting library of vectors, and expressing the combinatorial genes under conditions in which detection of a desired activity facilitates isolation of the vector encoding the gene whose product was detected. Recrusive ensemble mutagenesis (REM), a new technique that enhances the frequency of functional mutants in the libraries, can be used in combination with the screening assays to identify NOVX variants (Arkin and Yourvan (1992) PNAS 89:7811-7815; Delgrave et al. (1993) Protein Engineering 6:327-331). <span style="display: none">[0160] </span> </div>
</li> <li> <para-num num="[0161]"> </para-num> <div class="description-line" id="P-0161" num="0161">NOVX Antibodies <span style="display: none">[0161] </span> </div>
</li> <li> <para-num num="[0162]"> </para-num> <div class="description-line" id="P-0162" num="0162">Also included in the invention are antibodies to NOVX proteins, or fragments of NOVX proteins. The term “antibody” as used herein refers to immunoglobulin molecules and immunologically active portions of immunoglobulin (Ig) molecules, i.e., molecules that contain an antigen binding site that specifically binds (immunoreacts with) an antigen. Such antibodies include, but are not limited to, polyclonal, monoclonal, chimeric, single chain, F<span style="display: none">[0162] </span> <sub>ab</sub>, F<sub>ab′</sub> and F<sub>(ab′)2 </sub>fragments, and an F<sub>ab </sub>expression library. In general, an antibody molecule obtained from humans relates to any of the classes IgG, IgM, IgA, IgE and IgD, which differ from one another by the nature of the heavy chain present in the molecule. Certain classes have subclasses as well, such as IgG<sub>1</sub>, IgG<sub>2</sub>, and others. Furthermore, in humans, the light chain may be a kappa chain or a lambda chain. Reference herein to antibodies includes a reference to all such classes, subclasses and types of human antibody species. </div>
</li> <li> <para-num num="[0163]"> </para-num> <div class="description-line" id="P-0163" num="0163">An isolated NOVX-related protein of the invention may be intended to serve as an antigen, or a portion or fragment thereof, and additionally can be used as an immunogen to generate antibodies that immunospecifically bind the antigen, using standard techniques for polyclonal and monoclonal antibody preparation. The full-length protein can be used or, alternatively, the invention provides antigenic peptide fragments of the antigen for use as immunogens. An antigenic peptide fragment comprises at least 6 amino acid residues of the amino acid sequence of the full length protein, such as an amino acid sequence shown in SEQ ID NO: 2, 4, 6 ,8 ,10, 12, 14, 16, 18, or 20, and encompasses an epitope thereof such that an antibody raised against the peptide forms a specific immune complex with the full length protein or with any fragment that contains the epitope. Preferably, the antigenic peptide comprises at least 10 amino acid residues, or at least 15 amino acid residues, or at least 20 amino acid residues, or at least 30 amino acid residues. Preferred epitopes encompassed by the antigenic peptide are regions of the protein that are located on its surface; commonly these are hydrophilic regions. <span style="display: none">[0163] </span> </div>
</li> <li> <para-num num="[0164]"> </para-num> <div class="description-line" id="P-0164" num="0164">In certain embodiments of the invention, at least one epitope encompassed by the antigenic peptide is a region of NOVX-related protein that is located on the surface of the protein, e.g., a hydrophilic region. A hydrophobicity analysis of the human NOVX-related protein sequence will indicate which regions of a NOVX-related protein are particularly hydrophilic and, therefore, are likely to encode surface residues useful for targeting antibody production. As a means for targeting antibody production, hydropathy plots showing regions of hydrophilicity and hydrophobicity may be generated by any method well known in the art, including, for example, the Kyte Doolittle or the Hopp Woods methods, either with or without Fourier transformation. See, e.g., Hopp and Woods, 1981, <span style="display: none">[0164] </span> <i>Proc. Nat. Acad. Sci. USA </i>78: 3824-3828; Kyte and Doolittle 1982, <i>J. Mol. Biol. </i>157: 105-142, each of which is incorporated herein by reference in its entirety. Antibodies that are specific for one or more domains within an antigenic protein, or derivatives, fragments, analogs or homologs thereof, are also provided herein. </div>
</li> <li> <para-num num="[0165]"> </para-num> <div class="description-line" id="P-0165" num="0165">A protein of the invention, or a derivative, fragment, analog, homolog or ortholog thereof, may be utilized as an immunogen in the generation of antibodies that immunospecifically bind these protein components. <span style="display: none">[0165] </span> </div>
</li> <li> <para-num num="[0166]"> </para-num> <div class="description-line" id="P-0166" num="0166">Various procedures known within the art may be used for the production of polyclonal or monoclonal antibodies directed against a protein of the invention, or against derivatives, fragments, analogs homologs or orthologs thereof (see, for example, Antibodies: A Laboratory Manual, Harlow E, and Lane D, 1988, Cold Spring Harbor Laboratory Press, Cold Spring Harbor, NY, incorporated herein by reference). Some of these antibodies are discussed below. <span style="display: none">[0166] </span> </div>
</li> <li> <para-num num="[0167]"> </para-num> <div class="description-line" id="P-0167" num="0167">Polyclonal Antibodies <span style="display: none">[0167] </span> </div>
</li> <li> <para-num num="[0168]"> </para-num> <div class="description-line" id="P-0168" num="0168">For the production of polyclonal antibodies, various suitable host animals (e.g., rabbit, goat, mouse or other mammal) may be immunized by one or more injections with the native protein, a synthetic variant thereof, or a derivative of the foregoing. An appropriate immunogenic preparation can contain, for example, the naturally occurring immunogenic protein, a chemically synthesized polypeptide representing the immunogenic protein, or a recombinantly expressed immunogenic protein. Furthermore, the protein may be conjugated to a second protein known to be immunogenic in the mammal being immunized. Examples of such immunogenic proteins include but are not limited to keyhole limpet hemocyanin, serum albumin, bovine thyroglobulin, and soybean trypsin inhibitor. The preparation can further include an adjuvant. Various adjuvants used to increase the immunological response include, but are not limited to, Freund's (complete and incomplete), mineral gels (e.g., aluminum hydroxide), surface active substances (e.g., lysolecithin, pluronic polyols, polyanions, peptides, oil emulsions, dinitrophenol, etc.), adjuvants usable in humans such as Bacille Calmette-Guerin and Corynebacterium parvum, or similar immunostimulatory agents. Additional examples of adjuvants which can be employed include MPL-TDM adjuvant (monophosphoryl Lipid A, synthetic trehalose dicorynomycolate). <span style="display: none">[0168] </span> </div>
</li> <li> <para-num num="[0169]"> </para-num> <div class="description-line" id="P-0169" num="0169">The polyclonal antibody molecules directed against the immunogenic protein can be isolated from the mammal (e.g., from the blood) and further purified by well known techniques, such as affinity chromatography using protein A or protein G, which provide primarily the IgG fraction of immune serum. Subsequently, or alternatively, the specific antigen which is the target of the immunoglobulin sought, or an epitope thereof, may be immobilized on a column to purify the immune specific antibody by immunoaffinity chromatography. Purification of immunoglobulins is discussed, for example, by D. Wilkinson (The Scientist, published by The Scientist, Inc., Philadelphia Pa., Vol. 14, No. 8 (April 17, 2000), pp. 25-28). <span style="display: none">[0169] </span> </div>
</li> <li> <para-num num="[0170]"> </para-num> <div class="description-line" id="P-0170" num="0170">Monoclonal Antibodies <span style="display: none">[0170] </span> </div>
</li> <li> <para-num num="[0171]"> </para-num> <div class="description-line" id="P-0171" num="0171">The term “monoclonal antibody” (MAb) or “monoclonal antibody composition”, as used herein, refers to a population of antibody molecules that contain only one molecular species of antibody molecule consisting of a unique light chain gene product and a unique heavy chain gene product. In particular, the complementarity determining regions (CDRs) of the monoclonal antibody are identical in all the molecules of the population. MAbs thus contain an antigen binding site capable of immunoreacting with a particular epitope of the antigen characterized by a unique binding affinity for it. <span style="display: none">[0171] </span> </div>
</li> <li> <para-num num="[0172]"> </para-num> <div class="description-line" id="P-0172" num="0172">Monoclonal antibodies can be prepared using hybridoma methods, such as those described by Kohler and Milstein, <span style="display: none">[0172] </span> <i>Nature, </i>256:495 (1975). In a hybridoma method, a mouse, hamster, or other appropriate host animal, is typically immunized with an immunizing agent to elicit lymphocytes that produce or are capable of producing antibodies that will specifically bind to the immunizing agent. Alternatively, the lymphocytes can be immunized in vitro. </div>
</li> <li> <para-num num="[0173]"> </para-num> <div class="description-line" id="P-0173" num="0173">The immunizing agent will typically include the protein antigen, a fragment thereof or a fusion protein thereof. Generally, either peripheral blood lymphocytes are used if cells of human origin are desired, or spleen cells or lymph node cells are used if non-human mammalian sources are desired. The lymphocytes are then fused with an immortalized cell line using a suitable fusing agent, such as polyethylene glycol, to form a hybridoma cell (Goding, <span style="display: none">[0173] </span> <i>Monoclonal Antibodies: Principles and Practice</i>, Academic Press, (1986) pp. 59-103). Immortalized cell lines are usually transformed mammalian cells, particularly myeloma cells of rodent, bovine and human origin. Usually, rat or mouse myeloma cell lines are employed. The hybridoma cells can be cultured in a suitable culture medium that preferably contains one or more substances that inhibit the growth or survival of the unfused, immortalized cells. For example, if the parental cells lack the enzyme hypoxanthine guanine phosphoribosyl transferase (HGPRT or HPRT), the culture medium for the hybridomas typically will include hypoxanthine, aminopterin, and thymidine (“HAT medium”), which substances prevent the growth of HGPRT-deficient cells. </div>
</li> <li> <para-num num="[0174]"> </para-num> <div class="description-line" id="P-0174" num="0174">Preferred immortalized cell lines are those that fuse efficiently, support stable high level expression of antibody by the selected antibody-producing cells, and are sensitive to a medium such as HAT medium. More preferred immortalized cell lines are murine myeloma lines, which can be obtained, for instance, from the Salk Institute Cell Distribution Center, San Diego, Calif. and the American Type Culture Collection, Manassas, Va. Human myeloma and mouse-human heteromyeloma cell lines also have been described for the production of human monoclonal antibodies (Kozbor, <span style="display: none">[0174] </span> <i>J. Immunol., </i>133:3001 (1984); Brodeur et al., <i>Monoclonal Antibody Production Techniques and Applications</i>, Marcel Dekker, Inc., New York, (1987) pp. 51-63). </div>
</li> <li> <para-num num="[0175]"> </para-num> <div class="description-line" id="P-0175" num="0175">The culture medium in which the hybridoma cells are cultured can then be assayed for the presence of monoclonal antibodies directed against the antigen. Preferably, the binding specificity of monoclonal antibodies produced by the hybridoma cells is determined by immunoprecipitation or by an in vitro binding assay, such as radioimmunoassay (RIA) or enzyme-linked immunoabsorbent assay (ELISA). Such techniques and assays are known in the art. The binding affinity of the monoclonal antibody can, for example, be determined by the Scatchard analysis of Munson and Pollard, <span style="display: none">[0175] </span> <i>Anal. Biochem., </i>107:220 (1980). Preferably, antibodies having a high degree of specificity and a high binding affinity for the target antigen are isolated. </div>
</li> <li> <para-num num="[0176]"> </para-num> <div class="description-line" id="P-0176" num="0176">After the desired hybridoma cells are identified, the clones can be subcloned by limiting dilution procedures and grown by standard methods. Suitable culture media for this purpose include, for example, Dulbecco's Modified Eagle's Medium and RPMI-1640 medium. Alternatively, the hybridoma cells can be grown iv vivo as ascites in a mammal. <span style="display: none">[0176] </span> </div>
</li> <li> <para-num num="[0177]"> </para-num> <div class="description-line" id="P-0177" num="0177">The monoclonal antibodies secreted by the subclones can be isolated or purified from the culture medium or ascites fluid by conventional immunoglobulin purification procedures such as, for example, protein A-Sepharose, hydroxylapatite chromatography, gel electrophoresis, dialysis, or affinity chromatography. <span style="display: none">[0177] </span> </div>
</li> <li> <para-num num="[0178]"> </para-num> <div class="description-line" id="P-0178" num="0178">The monoclonal antibodies can also be made by recombinant DNA methods, such as those described in U.S. Pat. No. 4,816,567. DNA encoding the monoclonal antibodies of the invention can be readily isolated and sequenced using conventional procedures (e.g., by using oligonucleotide probes that are capable of binding specifically to genes encoding the heavy and light chains of murine antibodies). The hybridoma cells of the invention serve as a preferred source of such DNA. Once isolated, the DNA can be placed into expression vectors, which are then transfected into host cells such as simian COS cells, Chinese hamster ovary (CHO) cells, or myeloma cells that do not otherwise produce immunoglobulin protein, to obtain the synthesis of monoclonal antibodies in the recombinant host cells. The DNA also can be modified, for example, by substituting the coding sequence for human heavy and light chain constant domains in place of the homologous murine sequences (U.S. Patent No. 4,816,567; Morrison, Nature 368, 812-13 (1994)) or by covalently joining to the immunoglobulin coding sequence all or part of the coding sequence for a non-immunoglobulin polypeptide. Such a non-immunoglobulin polypeptide can be substituted for the constant domains of an antibody of the invention, or can be substituted for the variable domains of one antigen-combining site of an antibody of the invention to create a chimeric bivalent antibody. Humanized Antibodies The antibodies directed against the protein antigens of the invention can further comprise humanized antibodies or human antibodies. These antibodies are suitable for administration to humans without engendering an immune response by the human against the administered immunoglobulin. Humanized forms of antibodies are chimeric immunoglobulins, immunoglobulin chains or fragments thereof (such as Fv, Fab, Fab′, F(ab′)<span style="display: none">[0178] </span> <sub>2 </sub>or other antigen-binding subsequences of antibodies) that are principally comprised of the sequence of a human immunoglobulin, and contain minimal sequence derived from a non-human immunoglobulin. Humanization can be performed following the method of Winter and co-workers (Jones et al., <i>Nature, </i>321:522-525 (1986); Riechmann et al., <i>Nature, </i>332:323-327 (1988); Verhoeyen et al., <i>Science, </i>239:1534-1536 (1988)), by substituting rodent CDRs or CDR sequences for the corresponding sequences of a human antibody. (See also U.S. Pat. No. 5,225,539.) In some instances, Fv framework residues of the human immunoglobulin are replaced by corresponding non-human residues. Humanized antibodies can also comprise residues which are found neither in the recipient antibody nor in the imported CDR or framework sequences. In general, the humanized antibody will comprise substantially all of at least one, and typically two, variable domains, in which all or substantially all of the CDR regions correspond to those of a non-human immunoglobulin and all or substantially all of the framework regions are those of a human immunoglobulin consensus sequence. The humanized antibody optimally also will comprise at least a portion of an immunoglobulin constant region (Fc), typically that of a human immunoglobulin (Jones et al., 1986; Riechmann et al., 1988; and Presta, <i>Curr. Op. Struct. Biol., </i>2:593-596 (1992)). </div>
</li> <li> <para-num num="[0179]"> </para-num> <div class="description-line" id="P-0179" num="0179">Human Antibodies <span style="display: none">[0179] </span> </div>
</li> <li> <para-num num="[0180]"> </para-num> <div class="description-line" id="P-0180" num="0180">Fully human antibodies relate to antibody molecules in which essentially the entire sequences of both the light chain and the heavy chain, including the CDRs, arise from human genes. Such antibodies are termed “human antibodies”, or “fully human antibodies” herein. Human monoclonal antibodies can be prepared by the trioma technique; the human B-cell hybridoma technique (see Kozbor, et al., 1983 Immunol Today 4: 72) and the EBV hybridoma technique to produce human monoclonal antibodies (see Cole, et al., 1985 In: <span style="display: none">[0180] </span> <i>MONOCLONAL ANTIBODIES AND CANCER THERAPY</i>, Alan R. Liss, Inc., pp. 77-96). Human monoclonal antibodies may be utilized in the practice of the present invention and may be produced by using human hybridomas (see Cote, et al., 1983. <i>Proc Natl Acad Sci USA </i>80: 2026-2030) or by transforming human B-cells with Epstein Barr Virus in vitro (see Cole, et al., 1985 In: <i>MONOCLONAL ANTIBODIES AND CANCER THERAPY</i>, Alan R. Liss, Inc., pp. 77-96). </div>
</li> <li> <para-num num="[0181]"> </para-num> <div class="description-line" id="P-0181" num="0181">In addition, human antibodies can also be produced using additional techniques, including phage display libraries (Hoogenboom and Winter, <span style="display: none">[0181] </span> <i>J. Mol. Biol., </i>227:381 (1991); Marks et al., <i>J. Mol. Biol., </i>222:581 (1991)). Similarly, human antibodies can be made by introducing human immunoglobulin loci into transgenic animals, e.g., mice in which the endogenous immunoglobulin genes have been partially or completely inactivated. Upon challenge, human antibody production is observed, which closely resembles that seen in humans in all respects, including gene rearrangement, assembly, and antibody repertoire. This approach is described, for example, in U.S. Pat. Nos. 5,545,807; 5,545,806; 5,569,825; 5,625,126; 5,633,425; 5,661,016, and in Marks et al. (<i>Bio/Technology </i>10, 779-783 (1992)); Lonberg et al. (<i>Nature </i>368 856-859 (1994)); Morrison (<i>Nature </i>368, 812-13 (1994)); Fishwild et al, (<i>Nature Biotechnology </i>14, 845-51 (1996)); Neuberger (<i>Nature Biotechnology </i>14, 826 (1996)); and Lonberg and Huszar (<i>Intern. Rev. Immunol. </i>13 65-93 (1995)). </div>
</li> <li> <para-num num="[0182]"> </para-num> <div class="description-line" id="P-0182" num="0182">Human antibodies may additionally be produced using transgenic nonhuman animals which are modified so as to produce fully human antibodies rather than the animal's endogenous antibodies in response to challenge by an antigen. (See PCT publication WO94/02602). The endogenous genes encoding the heavy and light immunoglobulin chains in the nonhuman host have been incapacitated, and active loci encoding human heavy and light chain immunoglobulins are inserted into the host's genome. The human genes are incorporated, for example, using yeast artificial chromosomes containing the requisite human DNA segments. An animal which provides all the desired modifications is then obtained as progeny by crossbreeding intermediate transgenic animals containing fewer than the full complement of the modifications. The preferred embodiment of such a nonhuman animal is a mouse, and is termed the Xenomouse™ as disclosed in PCT publications WO 96/33735 and WO 96/34096. This animal produces B cells which secrete fully human immunoglobulins. The antibodies can be obtained directly from the animal after immunization with an immunogen of interest, as, for example, a preparation of a polyclonal antibody, or alternatively from immortalized B cells derived from the animal, such as hybridomas producing monoclonal antibodies. Additionally, the genes encoding the immunoglobulins with human variable regions can be recovered and expressed to obtain the antibodies directly, or can be further modified to obtain analogs of antibodies such as, for example, single chain Fv molecules. <span style="display: none">[0182] </span> </div>
</li> <li> <para-num num="[0183]"> </para-num> <div class="description-line" id="P-0183" num="0183">An example of a method of producing a nonhuman host, exemplified as a mouse, lacking expression of an endogenous immunoglobulin heavy chain is disclosed in U.S. Pat. No. 5,939,598. It can be obtained by a method including deleting the J segment genes from at least one endogenous heavy chain locus in an embryonic stem cell to prevent rearrangement of the locus and to prevent formation of a transcript of a rearranged immunoglobulin heavy chain locus, the deletion being effected by a targeting vector containing a gene encoding a selectable marker; and producing from the embryonic stem cell a transgenic mouse whose somatic and germ cells contain the gene encoding the selectable marker. <span style="display: none">[0183] </span> </div>
</li> <li> <para-num num="[0184]"> </para-num> <div class="description-line" id="P-0184" num="0184">A method for producing an antibody of interest, such as a human antibody, is disclosed in U.S. Pat. No. 5,916,771. It includes introducing an expression vector that contains a nucleotide sequence encoding a heavy chain into one mammalian host cell in culture, introducing an expression vector containing a nucleotide sequence encoding a light chain into another mammalian host cell, and fusing the two cells to form a hybrid cell. The hybrid cell expresses an antibody containing the heavy chain and the light chain. <span style="display: none">[0184] </span> </div>
</li> <li> <para-num num="[0185]"> </para-num> <div class="description-line" id="P-0185" num="0185">In a further improvement on this procedure, a method for identifying a clinically relevant epitope on an immunogen, and a correlative method for selecting an antibody that binds immunospecifically to the relevant epitope with high affinity, are disclosed in PCT publication WO 99/53049. <span style="display: none">[0185] </span> </div>
</li> <li> <para-num num="[0186]"> </para-num> <div class="description-line" id="P-0186" num="0186">Fab Fragments and Single Chain Antibodies <span style="display: none">[0186] </span> </div>
</li> <li> <para-num num="[0187]"> </para-num> <div class="description-line" id="P-0187" num="0187">According to the invention, techniques can be adapted for the production of single-chain antibodies specific to an antigenic protein of the invention (see e.g., U.S. Pat. No. 4,946,778). In addition, methods can be adapted for the construction of Fab expression libraries (see e.g., Huse, et al., 1989 Science 246: 1275-1281) to allow rapid and effective identification of monoclonal F<span style="display: none">[0187] </span> <sub>ab </sub>fragments with the desired specificity for a protein or derivatives, fragments, analogs or homologs thereof. Antibody fragments that contain the idiotypes to a protein antigen may be produced by techniques known in the art including, but not limited to: (i) an F<sub>(ab′)2 </sub>fragment produced by pepsin digestion of an antibody molecule; (ii) an F<sub>ab </sub>fragment generated by reducing the disulfide bridges of an F<sub>(ab)2 </sub>fragment; (iii) an F<sub>ab </sub>fragment generated by the treatment of the antibody molecule with papain and a reducing agent and (iv) F<sub>v </sub>fragments. </div>
</li> <li> <para-num num="[0188]"> </para-num> <div class="description-line" id="P-0188" num="0188">Bispecific Antibodies <span style="display: none">[0188] </span> </div>
</li> <li> <para-num num="[0189]"> </para-num> <div class="description-line" id="P-0189" num="0189">Bispecific antibodies are monoclonal, preferably human or humanized, antibodies that have binding specificities for at least two different antigens. In the present case, one of the binding specificities is for an antigenic protein of the invention. The second binding target is any other antigen, and advantageously is a cell-surface protein or receptor or receptor subunit. <span style="display: none">[0189] </span> </div>
</li> <li> <para-num num="[0190]"> </para-num> <div class="description-line" id="P-0190" num="0190">Methods for making bispecific antibodies are known in the art. Traditionally, the recombinant production of bispecific antibodies is based on the co-expression of two immunoglobulin heavy-chain/light-chain pairs, where the two heavy chains have different specificities (Milstein and Cuello, <span style="display: none">[0190] </span> <i>Nature, </i>305:537-539 (1983)). Because of the random assortment of immunoglobulin heavy and light chains, these hybridomas (quadromas) produce a potential mixture of ten different antibody molecules, of which only one has the correct bispecific structure. The purification of the correct molecule is usually accomplished by affinity chromatography steps. Similar procedures are disclosed in WO 93/08829, published May 13, 1993, and in Traunecker et al., 1991 <i>EMBO J., </i>10:3655-3659. </div>
</li> <li> <para-num num="[0191]"> </para-num> <div class="description-line" id="P-0191" num="0191">Antibody variable domains with the desired binding specificities (antibody-antigen combining sites) can be fused to immunoglobulin constant domain sequences. The fusion preferably is with an immunoglobulin heavy-chain constant domain, comprising at least part of the hinge, CH2, and CH3 regions. It is preferred to have the first heavy-chain constant region (CH1) containing the site necessary for light-chain binding present in at least one of the fusions. DNAs encoding the immunoglobulin heavy-chain fusions and, if desired, the immunoglobulin light chain, are inserted into separate expression vectors, and are co-transfected into a suitable host organism. For further details of generating bispecific antibodies see, for example, Suresh et al., <span style="display: none">[0191] </span> <i>Methods in Enzymology, </i>121:210 (1986). </div>
</li> <li> <para-num num="[0192]"> </para-num> <div class="description-line" id="P-0192" num="0192">According to another approach described in WO 96/27011, the interface between a pair of antibody molecules can be engineered to maximize the percentage of heterodimers which are recovered from recombinant cell culture. The preferred interface comprises at least a part of the CH3 region of an antibody constant domain. In this method, one or more small amino acid side chains from the interface of the first antibody molecule are replaced with larger side chains (e.g. tyrosine or tryptophan). Compensatory “cavities” of identical or similar size to the large side chain(s) are created on the interface of the second antibody molecule by replacing large amino acid side chains with smaller ones (e.g. alanine or threonine). This provides a mechanism for increasing the yield of the heterodimer over other unwanted end-products such as homodimers. <span style="display: none">[0192] </span> </div>
</li> <li> <para-num num="[0193]"> </para-num> <div class="description-line" id="P-0193" num="0193">Bispecific antibodies can be prepared as full length antibodies or antibody fragments (e.g. F(ab′)<span style="display: none">[0193] </span> <sub>2 </sub>bispecific antibodies). Techniques for generating bispecific antibodies from antibody fragments have been described in the literature. For example, bispecific antibodies can be prepared using chemical linkage. Brennan et al., <i>Science </i>229:81 (1985) describe a procedure wherein intact antibodies are proteolytically cleaved to generate F(ab′)<sub>2 </sub>fragments. These fragments are reduced in the presence of the dithiol complexing agent sodium arsenite to stabilize vicinal dithiols and prevent intermolecular disulfide formation. The Fab′ fragments generated are then converted to thionitrobenzoate (TNB) derivatives. One of the Fab′-TNB derivatives is then reconverted to the Fab′-thiol by reduction with mercaptoethylamine and is mixed with an equimolar amount of the other Fab′-TNB derivative to form the bispecific antibody. The bispecific antibodies produced can be used as agents for the selective immobilization of enzymes. </div>
</li> <li> <para-num num="[0194]"> </para-num> <div class="description-line" id="P-0194" num="0194">Additionally, Fab′ fragments can be directly recovered from <span style="display: none">[0194] </span> <i>E. coli </i>and chemically coupled to form bispecific antibodies. Shalaby et al., <i>J. Exp. Med. </i>175:217-225 (1992) describe the production of a fully humanized bispecific antibody F(ab′)<sub>2 </sub>molecule. Each Fab′ fragment was separately secreted from <i>E. coli </i>and subjected to directed chemical coupling in vitro to form the bispecific antibody. The bispecific antibody thus formed was able to bind to cells overexpressing the ErbB2 receptor and normal human T cells, as well as trigger the lytic activity of human cytotoxic lymphocytes against human breast tumor targets. </div>
</li> <li> <para-num num="[0195]"> </para-num> <div class="description-line" id="P-0195" num="0195">Various techniques for making and isolating bispecific antibody fragments directly from recombinant cell culture have also been described. For example, bispecific antibodies have been produced using leucine zippers. Kostelny et al., <span style="display: none">[0195] </span> <i>J. Immunol. </i>148(5):1547-1553 (1992). The leucine zipper peptides from the Fos and Jun proteins were linked to the Fab′ portions of two different antibodies by gene fusion. The antibody homodimers were reduced at the hinge region to form monomers and then re-oxidized to form the antibody heterodimers. This method can also be utilized for the production of antibody homodimers. The “diabody” technology described by Hollinger et al., <i>Proc. Natl. Acad. Sci. USA </i>90:6444-6448 (1993) has provided an alternative mechanism for making bispecific antibody fragments. The fragments comprise a heavy-chain variable domain (V<sub>H</sub>) connected to a light-chain variable domain (V<sub>L</sub>) by a linker which is too short to allow pairing between the two domains on the same chain. Accordingly, the V<sub>H </sub>and V<sub>L </sub>domains of one fragment are forced to pair with the complementary V<sub>L </sub>and V<sub>H </sub>domains of another fragment, thereby forming two antigen-binding sites. Another strategy for making bispecific antibody fragments by the use of single-chain Fv (sFv) dimers has also been reported. See, Gruber et al., <i>J. Immunol. </i>152:5368 (1994). </div>
</li> <li> <para-num num="[0196]"> </para-num> <div class="description-line" id="P-0196" num="0196">Antibodies with more than two valencies are contemplated. For example, trispecific antibodies can be prepared. Tutt et al., <span style="display: none">[0196] </span> <i>J. Immunol. </i>147:60 (1991). </div>
</li> <li> <para-num num="[0197]"> </para-num> <div class="description-line" id="P-0197" num="0197">Exemplary bispecific antibodies can bind to two different epitopes, at least one of which originates in the protein antigen of the invention. Alternatively, an anti-antigenic arm of an immunoglobulin molecule can be combined with an arm which binds to a triggering molecule on a leukocyte such as a T-cell receptor molecule (e.g. CD2, CD3, CD28, or B7), or Fc receptors for IgG (Fc R), such as Fc RI (CD64), Fc RII (CD32) and Fc RIII (CD16) so as to focus cellular defense mechanisms to the cell expressing the particular antigen. Bispecific antibodies can also be used to direct cytotoxic agents to cells which express a particular antigen. These antibodies possess an antigen-binding arm and an arm which binds a cytotoxic agent or a radionuclide chelator, such as EOTUBE, DPTA, DOTA, or TETA. Another bispecific antibody of interest binds the protein antigen described herein and further binds tissue factor (TF). <span style="display: none">[0197] </span> </div>
</li> <li> <para-num num="[0198]"> </para-num> <div class="description-line" id="P-0198" num="0198">Heteroconjugate Antibodies <span style="display: none">[0198] </span> </div>
</li> <li> <para-num num="[0199]"> </para-num> <div class="description-line" id="P-0199" num="0199">Heteroconjugate antibodies are also within the scope of the present invention. Heteroconjugate antibodies are composed of two covalently joined antibodies. Such antibodies have, for example, been proposed to target immune system cells to unwanted cells (U.S. Pat. No. 4,676,980), and for treatment of HIV infection (WO 91/00360; WO 92/200373; EP 03089). It is contemplated that the antibodies can be prepared in vitro using known methods in synthetic protein chemistry, including those involving crosslinking agents. For example, immunotoxins can be constructed using a disulfide exchange reaction or by forming a thioether bond. Examples of suitable reagents for this purpose include iminothiolate and methyl-4-mercaptobutyrimidate and those disclosed, for example, in U.S. Pat. No. 4,676,980. <span style="display: none">[0199] </span> </div>
</li> <li> <para-num num="[0200]"> </para-num> <div class="description-line" id="P-0200" num="0200">Effector Function Engineering <span style="display: none">[0200] </span> </div>
</li> <li> <para-num num="[0201]"> </para-num> <div class="description-line" id="P-0201" num="0201">It can be desirable to modify the antibody of the invention with respect to effector function, so as to enhance, e.g., the effectiveness of the antibody in treating cancer. For example, cysteine residue(s) can be introduced into the Fc region, thereby allowing interchain disulfide bond formation in this region. The homodimeric antibody thus generated can have improved internalization capability and/or increased complement-mediated cell killing and antibody-dependent cellular cytotoxicity (ADCC). See Caron et al., J. Exp Med., 176: 1191-1195 (1992) and Shopes, <span style="display: none">[0201] </span> <i>J. Immunol., </i>148: 2918-2922 (1992). Homodimeric antibodies with enhanced anti-tumor activity can also be prepared using heterobifunctional cross-linkers as described in Wolff et al. Cancer Research, 53: 2560-2565 (1993). Alternatively, an antibody can be engineered that has dual Fc regions and can thereby have enhanced complement lysis and ADCC capabilities. See Stevenson et al., Anti-Cancer Drug Design, 3: 219-230 (1989). </div>
</li> <li> <para-num num="[0202]"> </para-num> <div class="description-line" id="P-0202" num="0202">Immunoconjugates <span style="display: none">[0202] </span> </div>
</li> <li> <para-num num="[0203]"> </para-num> <div class="description-line" id="P-0203" num="0203">The invention also pertains to immunoconjugates comprising an antibody conjugated to a cytotoxic agent such as a chemotherapeutic agent, toxin (e.g., an enzymatically active toxin of bacterial, fungal, plant, or animal origin, or fragments thereof), or a radioactive isotope (i.e., a radioconjugate). <span style="display: none">[0203] </span> </div>
</li> <li> <para-num num="[0204]"> </para-num> <div class="description-line" id="P-0204" num="0204">Chemotherapeutic agents useful in the generation of such immunoconjugates have been described above. Enzymatically active toxins and fragments thereof that can be used include diphtheria A chain, nonbinding active fragments of diphtheria toxin, exotoxin A chain (from Pseudomonas aeruginosa), ricin A chain, abrin A chain, modeccin A chain, alpha-sarcin, Aleurites fordii proteins, dianthin proteins, Phytolaca americana proteins (PAPI, PAPII, and PAP-S), momordica charantia inhibitor, curcin, crotin, sapaonaria officinalis inhibitor, gelonin, mitogellin, restrictocin, phenomycin, enomycin, and the tricothecenes. A variety of radionuclides are available for the production of radioconjugated antibodies. Examples include <span style="display: none">[0204] </span> <sup>212</sup>Bi, <sup>131</sup>I, <sup>131</sup>In, <sup>90</sup>Y, and <sup>186</sup>Re. </div>
</li> <li> <para-num num="[0205]"> </para-num> <div class="description-line" id="P-0205" num="0205">Conjugates of the antibody and cytotoxic agent are made using a variety of bifunctional protein-coupling agents such as N-succinimidyl-3-(2-pyridyldithiol) propionate (SPDP), iminothiolane (IT), bifunctional derivatives of imidoesters (such as dimethyl adipimidate HCL), active esters (such as disuccinimidyl suberate), aldehydes (such as glutareldehyde), bis-azido compounds (such as bis (p-azidobenzoyl) hexanediamine), bis-diazonium derivatives (such as bis-(p-diazoniumbenzoyl)-ethylenediamine), diisocyanates (such as tolyene 2,6-diisocyanate), and bis-active fluorine compounds (such as 1,5-difluoro-2,4-dinitrobenzene). For example, a ricin immunotoxin can be prepared as described in Vitetta et al., Science, 238: 1098 (1987). Carbon- 14-labeled 1-isothiocyanatobenzyl-3-methyldiethylene triaminepentaacetic acid (MX-DTPA) is an exemplary chelating agent for conjugation of radionucleotide to the antibody. See WO094/11026. <span style="display: none">[0205] </span> </div>
</li> <li> <para-num num="[0206]"> </para-num> <div class="description-line" id="P-0206" num="0206">In another embodiment, the antibody can be conjugated to a “receptor” (such streptavidin) for utilization in tumor pretargeting wherein the antibody-receptor conjugate is administered to the patient, followed by removal of unbound conjugate from the circulation using a clearing agent and then administration of a “ligand” (e.g., avidin) that is in turn conjugated to a cytotoxic agent. <span style="display: none">[0206] </span> </div>
</li> <li> <para-num num="[0207]"> </para-num> <div class="description-line" id="P-0207" num="0207">NOVX Recombinant Expression Vectors and Host Cells <span style="display: none">[0207] </span> </div>
</li> <li> <para-num num="[0208]"> </para-num> <div class="description-line" id="P-0208" num="0208">Another aspect of the invention pertains to vectors, preferably expression vectors, containing a nucleic acid encoding a NOVX protein, or derivatives, fragments, analogs or homologs thereof. As used herein, the term “vector” refers to a nucleic acid molecule capable of transporting another nucleic acid to which it has been linked. One type of vector is a “plasmid”, which refers to a circular double stranded DNA loop into which additional DNA segments can be ligated. Another type of vector is a viral vector, wherein additional DNA segments can be ligated into the viral genome. Certain vectors are capable of autonomous replication in a host cell into which they are introduced (e.g., bacterial vectors having a bacterial origin of replication and episomal mammalian vectors). Other vectors (e.g., non-episomal mammalian vectors) are integrated into the genome of a host cell upon introduction into the host cell, and thereby are replicated along with the host genome. Moreover, certain vectors are capable of directing the expression of genes to which they are operatively-linked. Such vectors are referred to herein as “expression vectors”. In general, expression vectors of utility in recombinant DNA techniques are often in the form of plasmids. In the present specification, “plasmid” and “vector” can be used interchangeably as the plasmid is the most commonly used form of vector. However, the invention is intended to include such other forms of expression vectors, such as viral vectors (e.g., replication defective retroviruses, adenoviruses and adeno-associated viruses), which serve equivalent functions. <span style="display: none">[0208] </span> </div>
</li> <li> <para-num num="[0209]"> </para-num> <div class="description-line" id="P-0209" num="0209">The recombinant expression vectors of the invention comprise a nucleic acid of the invention in a form suitable for expression of the nucleic acid in a host cell, which means that the recombinant expression vectors include one or more regulatory sequences, selected on the basis of the host cells to be used for expression, that is operatively-linked to the nucleic acid sequence to be expressed. Within a recombinant expression vector, “operably-linked” is intended to mean that the nucleotide sequence of interest is linked to the regulatory sequence(s) in a manner that allows for expression of the nucleotide sequence (e.g., in an in vitro transcription/translation system or in a host cell when the vector is introduced into the host cell). <span style="display: none">[0209] </span> </div>
</li> <li> <para-num num="[0210]"> </para-num> <div class="description-line" id="P-0210" num="0210">The term “regulatory sequence” is intended to includes promoters, enhancers and other expression control elements (e.g., polyadenylation signals). Such regulatory sequences are described, for example, in Goeddel, <span style="display: none">[0210] </span> <i>GENE EXPRESSION TECHNOLOGY: METHODS IN ENZYMOLOGY </i>185, Academic Press, San Diego, Calif. (1990). Regulatory sequences include those that direct constitutive expression of a nucleotide sequence in many types of host cell and those that direct expression of the nucleotide sequence only in certain host cells (e.g., tissue-specific regulatory sequences). It will be appreciated by those skilled in the art that the design of the expression vector can depend on such factors as the choice of the host cell to be transformed, the level of expression of protein desired, etc. The expression vectors of the invention can be introduced into host cells to thereby produce proteins or peptides, including fusion proteins or peptides, encoded by nucleic acids as described herein (e.g., NOVX proteins, mutant forms of NOVX proteins, fusion proteins, etc.). </div>
</li> <li> <para-num num="[0211]"> </para-num> <div class="description-line" id="P-0211" num="0211">The recombinant expression vectors of the invention can be designed for expression of NOVX proteins in prokaryotic or eukaryotic cells. For example, NOVX proteins can be expressed in bacterial cells such as <span style="display: none">[0211] </span> <i>Escherichia coli</i>, insect cells (using baculovirus expression vectors) yeast cells or mammalian cells. Suitable host cells are discussed further in Goeddel, <i>GENE EXPRESSION TECHNOLOGY: METHODS IN ENZYMOLOGY </i>185, Academic Press, San Diego, Calif. (1990). Alternatively, the recombinant expression vector can be transcribed and translated in vitro, for example using T7 promoter regulatory sequences and T7 polymerase. </div>
</li> <li> <para-num num="[0212]"> </para-num> <div class="description-line" id="P-0212" num="0212">Expression of proteins in prokaryotes is most often carried out in <span style="display: none">[0212] </span> <i>Escherichia coli </i>with vectors containing constitutive or inducible promoters directing the expression of either fusion or non-fusion proteins. Fusion vectors add a number of amino acids to a protein encoded therein, usually to the amino terminus of the recombinant protein. Such fusion vectors typically serve three purposes: (i) to increase expression of recombinant protein; (ii) to increase the solubility of the recombinant protein; and (iii) to aid in the purification of the recombinant protein by acting as a ligand in affinity purification. Often, in fusion expression vectors, a proteolytic cleavage site is introduced at the junction of the fusion moiety and the recombinant protein to enable separation of the recombinant protein from the fusion moiety subsequent to purification of the fusion protein. Such enzymes, and their cognate recognition sequences, include Factor Xa, thrombin and enterokinase. Typical fusion expression vectors include pGEX (Pharmacia Biotech Inc; Smith and Johnson, 1988. <i>Gene </i>67: 31-40), PMAL (New England Biolabs, Beverly, Mass.) and pRIT5 (Pharmacia, Piscataway, N.J.) that fuse glutathione S-transferase (GST), maltose E binding protein, or protein A, respectively, to the target recombinant protein. </div>
</li> <li> <para-num num="[0213]"> </para-num> <div class="description-line" id="P-0213" num="0213">Examples of suitable inducible non-fusion <span style="display: none">[0213] </span> <i>E. coli </i>expression vectors include pTrc (Amrann et al., (1988) <i>Gene </i>69:301-315) and pET I1 d (Studier et al., <i>GENE EXPRESSION TECHNOLOGY: METHODS IN ENZYMOLOGY </i>185, Academic Press, San Diego, Calif. (1990) 60-89). </div>
</li> <li> <para-num num="[0214]"> </para-num> <div class="description-line" id="P-0214" num="0214">One strategy to maximize recombinant protein expression in <span style="display: none">[0214] </span> <i>E. coli </i>is to express the protein in a host bacteria with an impaired capacity to proteolytically cleave the recombinant protein. See, e.g., Gottesman, <i>GENE EXPRESSION TECHNOLOGY: METHODS IN ENZYMOLOGY </i>185, Academic Press, San Diego, Calif. (1990) 119-128. Another strategy is to alter the nucleic acid sequence of the nucleic acid to be inserted into an expression vector so that the individual codons for each amino acid are those preferentially utilized in <i>E. coli </i>(see, e.g., Wada, et al., 1992. <i>Nucl. Acids Res. </i>20: 2111-2118). Such alteration of nucleic acid sequences of the invention can be carried out by standard DNA synthesis techniques. </div>
</li> <li> <para-num num="[0215]"> </para-num> <div class="description-line" id="P-0215" num="0215">In another embodiment, the NOVX expression vector is a yeast expression vector. Examples of vectors for expression in yeast <span style="display: none">[0215] </span> <i>Saccharomyces cerivisae </i>include pYepSec1 (Baldari, et al., 1987. <i>EMBO J </i>6: 229-234), pMFa (Kurjan and Herskowitz, 1982. <i>Cell </i>30: 933-943), pJRY88 (Schultz et al., 1987. <i>Gene </i>54: 113-123), pYES2 (Invitrogen Corporation, San Diego, Calif.), and picZ (InVitrogen Corp, San Diego, Calif.). </div>
</li> <li> <para-num num="[0216]"> </para-num> <div class="description-line" id="P-0216" num="0216">Alternatively, NOVX can be expressed in insect cells using baculovirus expression vectors. Baculovirus vectors available for expression of proteins in cultured insect cells (e.g., SF9 cells) include the pAc series (Smith, et al., 1983. <span style="display: none">[0216] </span> <i>Mol Cell. Biol. </i>3: 2156-2165) and the pVL series (Lucklow and Summers, 1989. <i>Virology </i>170: 31-39). </div>
</li> <li> <para-num num="[0217]"> </para-num> <div class="description-line" id="P-0217" num="0217">In yet another embodiment, a nucleic acid of the invention is expressed in mammalian cells using a mammalian expression vector. Examples of mammalian expression vectors include pCDM8 (Seed, 1987. Nature 329: 840) and pMT2PC (Kaufman, et al., 1987. <span style="display: none">[0217] </span> <i>EMBO J </i>6: 187-195). When used in mammalian cells, the expression vector's control functions are often provided by viral regulatory elements. For example, commonly used promoters are derived from polyoma, adenovirus 2, cytomegalovirus, and simian virus 40. For other suitable expression systems for both prokaryotic and eukaryotic cells see, e.g., Chapters 16 and 17 of Sambrook, et al., <i>MOLECULAR CLONING: A LABORATORY MANUAL. </i>2nd ed., Cold Spring Harbor Laboratory, Cold Spring Harbor Laboratory Press, Cold Spring Harbor, N.Y., 1989. </div>
</li> <li> <para-num num="[0218]"> </para-num> <div class="description-line" id="P-0218" num="0218">In another embodiment, the recombinant mammalian expression vector is capable of directing expression of the nucleic acid preferentially in a particular cell type (e.g., tissue-specific regulatory elements are used to express the nucleic acid). Tissue-specific regulatory elements are known in the art. Non-limiting examples of suitable tissue-specific promoters include the albumin promoter (liver-specific; Pinkert, et al., 1987. <span style="display: none">[0218] </span> <i>Genes Dev. </i>1: 268-277), lymphoid-specific promoters (Calame and Eaton, 1988. <i>Adv. Immunol. </i>43: 235-275), in particular promoters of T cell receptors (Winoto and Baltimore, 1989. <i>EMBO J </i>8: 729-733) and immunoglobulins (Banerji, et al., 1983. <i>Cell </i>33: 729-740; Queen and Baltimore, 1983. <i>Cell </i>33: 741-748), neuron-specific promoters (e.g., the neurofilament promoter; Byrne and Ruddle, 1989. <i>Proc. Natl. Acad. Sci. USA </i>86: 5473-5477), pancreas-specific promoters (Edlund, et al., 1985. <i>Science </i>230: 912-916), and mammary gland-specific promoters (e.g., milk whey promoter; U.S. Pat. No. 4,873,316 and European Application Publication No. 264,166). Developmentally-regulated promoters are also encompassed, e.g., the murine hox promoters (Kessel and Gruss, 1990. <i>Science </i>249: 374-379) and the α-fetoprotein promoter (Campes and Tilghman, 1989. <i>Genes Dev. </i>3: 537-546). </div>
</li> <li> <para-num num="[0219]"> </para-num> <div class="description-line" id="P-0219" num="0219">The invention further provides a recombinant expression vector comprising a DNA molecule of the invention cloned into the expression vector in an antisense orientation. That is, the DNA molecule is operatively-linked to a regulatory sequence in a manner that allows for expression (by transcription of the DNA molecule) of an RNA molecule that is antisense to NOVX mRNA. Regulatory sequences operatively linked to a nucleic acid cloned in the antisense orientation can be chosen that direct the continuous expression of the antisense RNA molecule in a variety of cell types, for instance viral promoters and/or enhancers, or regulatory sequences can be chosen that direct constitutive, tissue specific or cell type specific expression of antisense RNA. The antisense expression vector can be in the form of a recombinant plasmid, phagemid or attenuated virus in which antisense nucleic acids are produced under the control of a high efficiency regulatory region, the activity of which can be determined by the cell type into which the vector is introduced. For a discussion of the regulation of gene expression using antisense genes see, e.g., Weintraub, et al., “Antisense RNA as a molecular tool for genetic analysis,” <span style="display: none">[0219] </span> <i>Reviews</i>-<i>Trends in Genetics</i>, Vol. 1(1) 1986. </div>
</li> <li> <para-num num="[0220]"> </para-num> <div class="description-line" id="P-0220" num="0220">Another aspect of the invention pertains to host cells into which a recombinant expression vector of the invention has been introduced. The terms “host cell” and “recombinant host cell” are used interchangeably herein. It is understood that such terms refer not only to the particular subject cell but also to the progeny or potential progeny of such a cell. Because certain modifications may occur in succeeding generations due to either mutation or environmental influences, such progeny may not, in fact, be identical to the parent cell, but are still included within the scope of the term as used herein. <span style="display: none">[0220] </span> </div>
</li> <li> <para-num num="[0221]"> </para-num> <div class="description-line" id="P-0221" num="0221">A host cell can be any prokaryotic or eukaryotic cell. For example, NOVX protein can be expressed in bacterial cells such as <span style="display: none">[0221] </span> <i>E. coli</i>, insect cells, yeast or mammalian cells (such as human, Chinese hamster ovary cells (CHO) or COS cells). Other suitable host cells are known to those skilled in the art. </div>
</li> <li> <para-num num="[0222]"> </para-num> <div class="description-line" id="P-0222" num="0222">Vector DNA can be introduced into prokaryotic or eukaryotic cells via conventional transformation or transfection techniques. As used herein, the terms “transformation” and “transfection” are intended to refer to a variety of art-recognized techniques for introducing foreign nucleic acid (e.g., DNA) into a host cell, including calcium phosphate or calcium chloride co-precipitation, DEAE-dextran-mediated transfection, lipofection, or electroporation. Suitable methods for transforming or transfecting host cells can be found in Sambrook, et al. (<span style="display: none">[0222] </span> <i>MOLECULAR CLONING: A LABORATORY MANUAL. </i>2nd ed., Cold Spring Harbor Laboratory, Cold Spring Harbor Laboratory Press, Cold Spring Harbor, N.Y., 1989), and other laboratory manuals. </div>
</li> <li> <para-num num="[0223]"> </para-num> <div class="description-line" id="P-0223" num="0223">For stable transfection of mammalian cells, it is known that, depending upon the expression vector and transfection technique used, only a small fraction of cells may integrate the foreign DNA into their genome. In order to identify and select these integrants, a gene that encodes a selectable marker (e.g., resistance to antibiotics) is generally introduced into the host cells along with the gene of interest. Various selectable markers include those that confer resistance to drugs, such as G418, hygromycin and methotrexate. Nucleic acid encoding a selectable marker can be introduced into a host cell on the same vector as that encoding NOVX or can be introduced on a separate vector. Cells stably transfected with the introduced nucleic acid can be identified by drug selection (e.g., cells that have incorporated the selectable marker gene will survive, while the other cells die). <span style="display: none">[0223] </span> </div>
</li> <li> <para-num num="[0224]"> </para-num> <div class="description-line" id="P-0224" num="0224">A host cell of the invention, such as a prokaryotic or eukaryotic host cell in culture, can be used to produce (i.e., express) NOVX protein. Accordingly, the invention further provides methods for producing NOVX protein using the host cells of the invention. In one embodiment, the method comprises culturing the host cell of invention (into which a recombinant expression vector encoding NOVX protein has been introduced) in a suitable medium such that NOVX protein is produced. In another embodiment, the method further comprises isolating NOVX protein from the medium or the host cell. <span style="display: none">[0224] </span> </div>
</li> <li> <para-num num="[0225]"> </para-num> <div class="description-line" id="P-0225" num="0225">Transgenic NOVX Animals <span style="display: none">[0225] </span> </div>
</li> <li> <para-num num="[0226]"> </para-num> <div class="description-line" id="P-0226" num="0226">The host cells of the invention can also be used to produce non-human transgenic animals. For example, in one embodiment, a host cell of the invention is a fertilized oocyte or an embryonic stem cell into which NOVX protein-coding sequences have been introduced. Such host cells can then be used to create non-human transgenic animals in which exogenous NOVX sequences have been introduced into their genome or homologous recombinant animals in which endogenous NOVX sequences have been altered. Such animals are useful for studying the function and/or activity of NOVX protein and for identifying and/or evaluating modulators of NOVX protein activity. As used herein, a “transgenic animal” is a non-human animal, preferably a mammal, more preferably a rodent such as a rat or mouse, in which one or more of the cells of the animal includes a transgene. Other examples of transgenic animals include non-human primates, sheep, dogs, cows, goats, chickens, amphibians, etc. A transgene is exogenous DNA that is integrated into the genome of a cell from which a transgenic animal develops and that remains in the genome of the mature animal, thereby directing the expression of an encoded gene product in one or more cell types or tissues of the transgenic animal. As used herein, a “homologous recombinant animal” is a non-human animal, preferably a mammal, more preferably a mouse, in which an endogenous NOVX gene has been altered by homologous recombination between the endogenous gene and an exogenous DNA molecule introduced into a cell of the animal, e.g., an embryonic cell of the animal, prior to development of the animal. <span style="display: none">[0226] </span> </div>
</li> <li> <para-num num="[0227]"> </para-num> <div class="description-line" id="P-0227" num="0227">A transgenic animal of the invention can be created by introducing NOVX-encoding nucleic acid into the male pronuclei of a fertilized oocyte (e.g., by microinjection, retroviral infection) and allowing the oocyte to develop in a pseudopregnant female foster animal. Sequences including SEQ ID NO: 1, 3, 5, 7, 9, 11, or 13 can be introduced as a transgene into the genome of a non-human animal. Alternatively, a non-human homologue of the human NOVX gene, such as a mouse NOVX gene, can be isolated based on hybridization to the human NOVX cDNA (described further supra) and used as a transgene. Intronic sequences and polyadenylation signals can also be included in the transgene to increase the efficiency of expression of the transgene. A tissue-specific regulatory sequence(s) can be operably-linked to the NOVX transgene to direct expression of NOVX protein to particular cells. Methods for generating transgenic animals via embryo manipulation and microinjection, particularly animals such as mice, have become conventional in the art and are described, for example, in U.S. Pat. Nos. 4,736,866; 4,870,009; and 4,873,191; and Hogan, 1986. In: <span style="display: none">[0227] </span> <i>MANIPULATING THE MOUSE EMBRYO</i>, Cold Spring Harbor Laboratory Press, Cold Spring Harbor, N.Y. Similar methods are used for production of other transgenic animals. A transgenic founder animal can be identified based upon the presence of the NOVX transgene in its genome and/or expression of NOVX mRNA in tissues or cells of the animals. A transgenic founder animal can then be used to breed additional animals carrying the transgene. Moreover, transgenic animals carrying a transgene-encoding NOVX protein can further be bred to other transgenic animals carrying other transgenes. </div>
</li> <li> <para-num num="[0228]"> </para-num> <div class="description-line" id="P-0228" num="0228">To create a homologous recombinant animal, a vector is prepared which contains at least a portion of a NOVX gene into which a deletion, addition or substitution has been introduced to thereby alter, e.g., functionally disrupt, the NOVX gene. The NOVX gene can be a human gene (e.g., the DNA of SEQ ID NO: 1,3,5,7,9, 11, or 13), but more preferably, is a non-human homologue of a human NOVX gene. For example, a mouse homologue of human NOVX gene of SEQ ID NO: 1, 3, 5, 7, 9, 11, or 13 can be used to construct a homologous recombination vector suitable for altering an endogenous NOVX gene in the mouse genome. In one embodiment, the vector is designed such that, upon homologous recombination, the endogenous NOVX gene is functionally disrupted (i e., no longer encodes a functional protein; also referred to as a “knock out” vector). <span style="display: none">[0228] </span> </div>
</li> <li> <para-num num="[0229]"> </para-num> <div class="description-line" id="P-0229" num="0229">Alternatively, the vector can be designed such that, upon homologous recombination, the endogenous NOVX gene is mutated or otherwise altered but still encodes functional protein (e.g., the upstream regulatory region can be altered to thereby alter the expression of the endogenous NOVX protein). In the homologous recombination vector, the altered portion of the NOVX gene is flanked at its 5′- and 3′-termini by additional nucleic acid of the NOVX gene to allow for homologous recombination to occur between the exogenous NOVX gene carried by the vector and an endogenous NOVX gene in an embryonic stem cell. The additional flanking NOVX nucleic acid is of sufficient length for successful homologous recombination with the endogenous gene. Typically, several kilobases of flanking DNA (both at the 5′- and 3′-termini) are included in the vector. See, e.g., Thomas, et al., 1987. <span style="display: none">[0229] </span> <i>Cell </i>51: 503 for a description of homologous recombination vectors. The vector is ten introduced into an embryonic stem cell line (e.g., by electroporation) and cells in which the introduced NOVX gene has homologously-recombined with the endogenous NOVX gene are selected. See, e.g., Li, et al., 1992. <i>Cell </i>69: 915. </div>
</li> <li> <para-num num="[0230]"> </para-num> <div class="description-line" id="P-0230" num="0230">The selected cells are then injected into a blastocyst of an animal (e.g., a mouse) to form aggregation chimeras. See, e.g., Bradley, 1987. In: <span style="display: none">[0230] </span> <i>TERATOCARCINOMAS AND EMBRYONIC STEM CELLS: A PRACTICAL APPROACH, Robertson, ed. IRL, Oxford, pp. </i>113-152. A chimeric embryo can then be implanted into a suitable pseudopregnant female foster animal and the embryo brought to term. Progeny harboring the homologously-recombined DNA in their germ cells can be used to breed animals in which all cells of the animal contain the homologously-recombined DNA by germline transmission of the transgene. Methods for constructing homologous recombination vectors and homologous recombinant animals are described further in Bradley, 1991. <i>Curr. Opin. Biotechnol. </i>2: 823-829; PCT International Publication Nos.: WO 90/11354; WO 91/01140; WO 92/0968; and WO 93/04169. </div>
</li> <li> <para-num num="[0231]"> </para-num> <div class="description-line" id="P-0231" num="0231">In another embodiment, transgenic non-humans animals can be produced that contain selected systems that allow for regulated expression of the transgene. One example of such a system is the cre/loxP recombinase system of bacteriophage P1. For a description of the cre/loxP recombinase system, See, e.g., Lakso, et al., 1992. <span style="display: none">[0231] </span> <i>Proc. Natl. Acad. Sci. USA </i>89: 6232-6236. Another example of a recombinase system is the FLP recombinase system of <i>Saccharomyces cerevisiae</i>. See, O'Gorman, et al., 1991. <i>Science </i>251:1351-1355. If a cre/loxP recombinase system is used to regulate expression of the transgene, animals containing transgenes encoding both the Cre recombinase and a selected protein are required. Such animals can be provided through the construction of “double” transgenic animals, e.g., by mating two transgenic animals, one containing a transgene encoding a selected protein and the other containing a transgene encoding a recombinase. </div>
</li> <li> <para-num num="[0232]"> </para-num> <div class="description-line" id="P-0232" num="0232">Clones of the non-human transgenic animals described herein can also be produced according to the methods described in Wilmut, et al., 1997. <span style="display: none">[0232] </span> <i>Nature </i>385: 810-813. In brief, a cell (e.g., a somatic cell) from the transgenic animal can be isolated and induced to exit the growth cycle and enter G<sub>0 </sub>phase. The quiescent cell can then be fused, e.g., through the use of electrical pulses, to an enucleated oocyte from an animal of the same species from which the quiescent cell is isolated. The reconstructed oocyte is then cultured such that it develops to morula or blastocyte and then transferred to pseudopregnant female foster animal. The offspring borne of this female foster animal will be a clone of the animal from which the cell (e.g., the somatic cell) is isolated. </div>
</li> <li> <para-num num="[0233]"> </para-num> <div class="description-line" id="P-0233" num="0233">Pharmaceutical Compositions <span style="display: none">[0233] </span> </div>
</li> <li> <para-num num="[0234]"> </para-num> <div class="description-line" id="P-0234" num="0234">The NOVX nucleic acid molecules, NOVX proteins, and anti-NOVX antibodies (also referred to herein as “active compounds”) of the invention, and derivatives, fragments, analogs and homologs thereof, can be incorporated into pharmaceutical compositions suitable for administration. Such compositions typically comprise the nucleic acid molecule, protein, or antibody and a pharmaceutically acceptable carrier. As used herein, “pharmaceutically acceptable carrier” is intended to include any and all solvents, dispersion media, coatings, antibacterial and antifungal agents, isotonic and absorption delaying agents, and the like, compatible with pharmaceutical administration. Suitable carriers are described in the most recent edition of Remington's Pharmaceutical Sciences, a standard reference text in the field, which is incorporated herein by reference. Preferred examples of such carriers or diluents include, but are not limited to, water, saline, finger's solutions, dextrose solution, and 5% human serum albumin. Liposomes and non-aqueous vehicles such as fixed oils may also be used. The use of such media and agents for pharmaceutically active substances is well known in the art. Except insofar as any conventional media or agent is incompatible with the active compound, use thereof in the compositions is contemplated. Supplementary active compounds can also be incorporated into the compositions. <span style="display: none">[0234] </span> </div>
</li> <li> <para-num num="[0235]"> </para-num> <div class="description-line" id="P-0235" num="0235">The antibodies disclosed herein can also be formulated as immunoliposomes. Liposomes containing the antibody are prepared by methods known in the art, such as described in Epstein et al., Proc. Natl. Acad. Sci. USA, 82: 3688 (1985); Hwang et al., Proc. Natl Acad. Sci. USA, 77: 4030 (1980); and U.S. Pat. Nos. 4,485,045 and 4,544,545. Liposomes with enhanced circulation time are disclosed in U.S. Pat. No. 5,013,556. <span style="display: none">[0235] </span> </div>
</li> <li> <para-num num="[0236]"> </para-num> <div class="description-line" id="P-0236" num="0236">Particularly useful liposomes can be generated by the reverse-phase evaporation method with a lipid composition comprising phosphatidylcholine, cholesterol, and PEG-derivatized phosphatidylethanolamine (PEG-PE). Liposomes are extruded through filters of defined pore size to yield liposomes with the desired diameter. Fab′ fragments of the antibody of the present invention can be conjugated to the liposomes as described in Martin et al., J. Biol. Chem., 257: 286-288 (1982) via a disulfide-interchange reaction. A chemotherapeutic agent (such as Doxorubicin) is optionally contained within the liposome. See Gabizon et al., J. National Cancer Inst., 81(19): 1484 (1989). <span style="display: none">[0236] </span> </div>
</li> <li> <para-num num="[0237]"> </para-num> <div class="description-line" id="P-0237" num="0237">A pharmaceutical composition of the invention is formulated to be compatible with its intended route of administration. Examples of routes of administration include parenteral, e.g., intravenous, intradermal, subcutaneous, oral (e.g., inhalation), transdermal (i.e., topical), transmucosal, and rectal administration. Solutions or suspensions used for parenteral, intradermal, or subcutaneous application can include the following components: a sterile diluent such as water for injection, saline solution, fixed oils, polyethylene glycols, glycerine, propylene glycol or other synthetic solvents; antibacterial agents such as benzyl alcohol or methyl parabens; antioxidants such as ascorbic acid or sodium bisulfite; chelating agents such as ethylenediaminetetraacetic acid (EDTA); buffers such as acetates, citrates or phosphates, and agents for the adjustment of tonicity such as sodium chloride or dextrose. The pH can be adjusted with acids or bases, such as hydrochloric acid or sodium hydroxide. The parenteral preparation can be enclosed in ampoules, disposable syringes or multiple dose vials made of glass or plastic. <span style="display: none">[0237] </span> </div>
</li> <li> <para-num num="[0238]"> </para-num> <div class="description-line" id="P-0238" num="0238">Pharmaceutical compositions suitable for injectable use include sterile aqueous solutions (where water soluble) or dispersions and sterile powders for the extemporaneous preparation of sterile injectable solutions or dispersion. For intravenous administration, suitable carriers include physiological saline, bacteriostatic water, Cremophor EL™ (BASF, Parsippany, N.J.) or phosphate buffered saline (PBS). In all cases, the composition must be sterile and should be fluid to the extent that easy syringeability exists. It must be stable under the conditions of manufacture and storage and must be preserved against the contaminating action of microorganisms such as bacteria and fungi. The carrier can be a solvent or dispersion medium containing, for example, water, ethanol, polyol (for example, glycerol, propylene glycol, and liquid polyethylene glycol, and the like), and suitable mixtures thereof. The proper fluidity can be maintained, for example, by the use of a coating such as lecithin, by the maintenance of the required particle size in the case of dispersion and by the use of surfactants. Prevention of the action of microorganisms can be achieved by various antibacterial and antifungal agents, for example, parabens, chlorobutanol, phenol, ascorbic acid, thimerosal, and the like. In many cases, it will be preferable to include isotonic agents, for example, sugars, polyalcohols such as manitol, sorbitol, sodium chloride in the composition. Prolonged absorption of the injectable compositions can be brought about by including in the composition an agent which delays absorption, for example, aluminum monostearate and gelatin. <span style="display: none">[0238] </span> </div>
</li> <li> <para-num num="[0239]"> </para-num> <div class="description-line" id="P-0239" num="0239">Sterile injectable solutions can be prepared by incorporating the active compound (e.g., a NOVX protein or anti-NOVX antibody) in the required amount in an appropriate solvent with one or a combination of ingredients enumerated above, as required, followed by filtered sterilization. Generally, dispersions are prepared by incorporating the active compound into a sterile vehicle that contains a basic dispersion medium and the required other ingredients from those enumerated above. In the case of sterile powders for the preparation of sterile injectable solutions, methods of preparation are vacuum drying and freeze-drying that yields a powder of the active ingredient plus any additional desired ingredient from a previously sterile-filtered solution thereof. <span style="display: none">[0239] </span> </div>
</li> <li> <para-num num="[0240]"> </para-num> <div class="description-line" id="P-0240" num="0240">Oral compositions generally include an inert diluent or an edible carrier. They can be enclosed in gelatin capsules or compressed into tablets. For the purpose of oral therapeutic administration, the active compound can be incorporated with excipients and used in the form of tablets, troches, or capsules. Oral compositions can also be prepared using a fluid carrier for use as a mouthwash, wherein the compound in the fluid carrier is applied orally and swished and expectorated or swallowed. Pharmaceutically compatible binding agents, and/or adjuvant materials can be included as part of the composition. The tablets, pills, capsules, troches and the like can contain any of the following ingredients, or compounds of a similar nature: a binder such as microcrystalline cellulose, gum tragacanth or gelatin; an excipient such as starch or lactose, a disintegrating agent such as alginic acid, Primogel, or corn starch; a lubricant such as magnesium stearate or Sterotes; a glidant such as colloidal silicon dioxide; a sweetening agent such as sucrose or saccharin; or a flavoring agent such as peppermint, methyl salicylate, or orange flavoring. <span style="display: none">[0240] </span> </div>
</li> <li> <para-num num="[0241]"> </para-num> <div class="description-line" id="P-0241" num="0241">For administration by inhalation, the compounds are delivered in the form of an aerosol spray from pressured container or dispenser which contains a suitable propellant, e.g., a gas such as carbon dioxide, or a nebulizer. <span style="display: none">[0241] </span> </div>
</li> <li> <para-num num="[0242]"> </para-num> <div class="description-line" id="P-0242" num="0242">Systemic administration can also be by transmucosal or transdermal means. For transmucosal or transdermal administration, penetrants appropriate to the barrier to be permeated are used in the formulation. Such penetrants are generally known in the art, and include, for example, for transmucosal administration, detergents, bile salts, and fusidic acid derivatives. Transmucosal administration can be accomplished through the use of nasal sprays or suppositories. For transdermal administration, the active compounds are formulated into ointments, salves, gels, or creams as generally known in the art. <span style="display: none">[0242] </span> </div>
</li> <li> <para-num num="[0243]"> </para-num> <div class="description-line" id="P-0243" num="0243">The compounds can also be prepared in the form of suppositories (e.g., with conventional suppository bases such as cocoa butter and other glycerides) or retention enemas for rectal delivery. <span style="display: none">[0243] </span> </div>
</li> <li> <para-num num="[0244]"> </para-num> <div class="description-line" id="P-0244" num="0244">In one embodiment, the active compounds are prepared with carriers that will protect the compound against rapid elimination from the body, such as a controlled release formulation, including implants and microencapsulated delivery systems. Biodegradable, biocompatible polymers can be used, such as ethylene vinyl acetate, polyanhydrides, polyglycolic acid, collagen, polyorthoesters, and polylactic acid. Methods for preparation of such formulations will be apparent to those skilled in the art. The materials can also be obtained commercially from Alza Corporation and Nova Pharmaceuticals, Inc. Liposomal suspensions (including liposomes targeted to infected cells with monoclonal antibodies to viral antigens) can also be used as pharmaceutically acceptable carriers. These can be prepared according to methods known to those skilled in the art, for example, as described in U.S. Pat. No. 4,522,811. <span style="display: none">[0244] </span> </div>
</li> <li> <para-num num="[0245]"> </para-num> <div class="description-line" id="P-0245" num="0245">It is especially advantageous to formulate oral or parenteral compositions in dosage unit form for ease of administration and uniformity of dosage. Dosage unit form as used herein refers to physically discrete units suited as unitary dosages for the subject to be treated; each unit containing a predetermined quantity of active compound calculated to produce the desired therapeutic effect in association with the required pharmaceutical carrier. The specification for the dosage unit forms of the invention are dictated by and directly dependent on the unique characteristics of the active compound and the particular therapeutic effect to be achieved, and the limitations inherent in the art of compounding such an active compound for the treatment of individuals. <span style="display: none">[0245] </span> </div>
</li> <li> <para-num num="[0246]"> </para-num> <div class="description-line" id="P-0246" num="0246">The nucleic acid molecules of the invention can be inserted into vectors and used as gene therapy vectors. Gene therapy vectors can be delivered to a subject by, for example, intravenous injection, local administration (see, e.g., U.S. Pat. No. 5,328,470) or by stereotactic injection (see, e.g., Chen, et al., 1994. <span style="display: none">[0246] </span> <i>Proc. Natl. Acad. Sci. USA </i>91: 3054-3057). The pharmaceutical preparation of the gene therapy vector can include the gene therapy vector in an acceptable diluent, or can comprise a slow release matrix in which the gene delivery vehicle is imbedded. Alternatively, where the complete gene delivery vector can be produced intact from recombinant cells, e.g., retroviral vectors, the pharmaceutical preparation can include one or more cells that produce the gene delivery system. </div>
</li> <li> <para-num num="[0247]"> </para-num> <div class="description-line" id="P-0247" num="0247">Antibodies specifically binding a protein of the invention, as well as other molecules identified by the screening assays disclosed herein, can be administered for the treatment of various disorders in the form of pharmaceutical compositions. Principles and considerations involved in preparing such compositions, as well as guidance in the choice of components are provided, for example, in Remington: The Science And Practice Of Pharmacy 19th ed. (Alfonso R. Gennaro, et al., editors) Mack Pub. Co., Easton, Pa.: 1995; Drug Absorption Enhancement: Concepts, Possibilities, Limitations, And Trends, Harwood Academic Publishers, Langhorne, Pa., 1994; and Peptide And Protein Drug Delivery (Advances In Parenteral Sciences, Vol. 4), 1991, M. Dekker, New York. If the antigenic protein is intracellular and whole antibodies are used as inhibitors, internalizing antibodies are preferred. However, liposomes can also be used to deliver the antibody, or an antibody fragment, into cells. Where antibody fragments are used, the smallest inhibitory fragment that specifically binds to the binding domain of the target protein is preferred. For example, based upon the variable-region sequences of an antibody, peptide molecules can be designed that retain the ability to bind the target protein sequence. Such peptides can be synthesized chemically and/or produced by recombinant DNA technology. See, e.g., Marasco et al., 1993 <span style="display: none">[0247] </span> <i>Proc. Natl. Acad. Sci. USA, </i>90: 7889-7893. The formulation herein can also contain more than one active compound as necessary for the particular indication being treated, preferably those with complementary activities that do not adversely affect each other. Alternatively, or in addition, the composition can comprise an agent that enhances its function, such as, for example, a cytotoxic agent, cytokine, chemotherapeutic agent, or growth-inhibitory agent. Such molecules are suitably present in combination in amounts that are effective for the purpose intended. The active ingredients can also be entrapped in microcapsules prepared, for example, by coacervation techniques or by interfacial polymerization, for example, hydroxymethylcellulose or gelatin-microcapsules and poly-(methylmethacrylate) microcapsules, respectively, in colloidal drug delivery systems (for example, liposomes, albumin microspheres, microemulsions, nano-particles, and nanocapsules) or in macroemulsions. </div>
</li> <li> <para-num num="[0248]"> </para-num> <div class="description-line" id="P-0248" num="0248">The formulations to be used for iv vivo administration must be sterile. This is readily accomplished by filtration through sterile filtration membranes. <span style="display: none">[0248] </span> </div>
</li> <li> <para-num num="[0249]"> </para-num> <div class="description-line" id="P-0249" num="0249">Sustained-release preparations can be prepared. Suitable examples of sustained-release preparations include semipermeable matrices of solid hydrophobic polymers containing the antibody, which matrices are in the form of shaped articles, e.g., films, or microcapsules. Examples of sustained-release matrices include polyesters, hydrogels (for example, poly(2-hydroxyethyl-methacrylate), or poly(vinylalcohol)), polylactides (U.S. Pat. No. 3,773,919), copolymers of L-glutamic acid and ethyl-L-glutamate, non-degradable ethylene-vinyl acetate, degradable lactic acid-glycolic acid copolymers such as the LUPRON DEPOT™ (injectable microspheres composed of lactic acid-glycolic acid copolymer and leuprolide acetate), and poly-D-(-)-3-hydroxybutyric acid. While polymers such as ethylene-vinyl acetate and lactic acid-glycolic acid enable release of molecules for over 100 days, certain hydrogels release proteins for shorter time periods. <span style="display: none">[0249] </span> </div>
</li> <li> <para-num num="[0250]"> </para-num> <div class="description-line" id="P-0250" num="0250">The pharmaceutical compositions can be included in a container, pack, or dispenser together with instructions for administration. <span style="display: none">[0250] </span> </div>
</li> <li> <para-num num="[0251]"> </para-num> <div class="description-line" id="P-0251" num="0251">Screening and Detection Methods <span style="display: none">[0251] </span> </div>
</li> <li> <para-num num="[0252]"> </para-num> <div class="description-line" id="P-0252" num="0252">The isolated nucleic acid molecules of the invention can be used to express NOVX protein (e.g., via a recombinant expression vector in a host cell in gene therapy applications), to detect NOVX mRNA (e.g., in a biological sample) or a genetic lesion in a NOVX gene, and to modulate NOVX activity, as described further, below. In addition, the NOVX proteins can be used to screen drugs or compounds that modulate the NOVX protein activity or expression as well as to treat disorders characterized by insufficient or excessive production of NOVX protein or production of NOVX protein forms that have decreased or aberrant activity compared to NOVX wild-type protein. In addition, the anti-NOVX antibodies of the invention can be used to detect and isolate NOVX proteins and modulate NOVX activity. For example, NOVX activity includes growth and differentiation, antibody production, and tumor growth. <span style="display: none">[0252] </span> </div>
</li> <li> <para-num num="[0253]"> </para-num> <div class="description-line" id="P-0253" num="0253">The invention further pertains to novel agents identified by the screening assays described herein and uses thereof for treatments as described, supra. <span style="display: none">[0253] </span> </div>
</li> <li> <para-num num="[0254]"> </para-num> <div class="description-line" id="P-0254" num="0254">Screening Assays <span style="display: none">[0254] </span> </div>
</li> <li> <para-num num="[0255]"> </para-num> <div class="description-line" id="P-0255" num="0255">The invention provides a method (also referred to herein as a “screening assay”) for identifying modulators, i.e., candidate or test compounds or agents (e.g., peptides, peptidomimetics, small molecules or other drugs) that bind to NOVX proteins or have a stimulatory or inhibitory effect on, e.g., NOVX protein expression or NOVX protein activity. The invention also includes compounds identified in the screening assays described herein. <span style="display: none">[0255] </span> </div>
</li> <li> <para-num num="[0256]"> </para-num> <div class="description-line" id="P-0256" num="0256">In one embodiment, the invention provides assays for screening candidate or test compounds which bind to or modulate the activity of the membrane-bound form of a NOVX protein or polypeptide or biologically-active portion thereof. The test compounds of the invention can be obtained using any of the numerous approaches in combinatorial library methods known in the art, including: biological libraries; spatially addressable parallel solid phase or solution phase libraries; synthetic library methods requiring deconvolution; the “one-bead one-compound” library method; and synthetic library methods using affinity chromatography selection. The biological library approach is limited to peptide libraries, while the other four approaches are applicable to peptide, non-peptide oligomer or small molecule libraries of compounds. See, e.g., Lam, 1997. <span style="display: none">[0256] </span> <i>Anticancer Drug Design </i>12: 145. </div>
</li> <li> <para-num num="[0257]"> </para-num> <div class="description-line" id="P-0257" num="0257">A “small molecule” as used herein, is meant to refer to a composition that has a molecular weight of less than about 5 kD and most preferably less than about 4 kD. Small molecules can be, e.g., nucleic acids, peptides, polypeptides, peptidomimetics, carbohydrates, lipids or other organic or inorganic molecules. Libraries of chemical and/or biological mixtures, such as fungal, bacterial, or algal extracts, are known in the art and can be screened with any of the assays of the invention. <span style="display: none">[0257] </span> </div>
</li> <li> <para-num num="[0258]"> </para-num> <div class="description-line" id="P-0258" num="0258">Examples of methods for the synthesis of molecular libraries can be found in the art, for example in: DeWitt, et al., 1993. <span style="display: none">[0258] </span> <i>Proc. Natl. Acad. Sci. U.S.A. </i>90: 6909; Erb, et al., 1994. <i>Proc. Natl. Acad. Sci. U.S.A. </i>91: 11422; Zuckermann, et al., 1994. <i>J. Med. Chem. </i>37: 2678; Cho, et al., 1993. <i>Science </i>261: 1303; Carrell, et al., 1994. <i>Angew. Chem. Int. Ed. Engl. </i>33: 2059; Carell, et al., 1994. <i>Angew. Chem. Int. Ed. Engl. </i>33: 2061; and Gallop, et al., 1994. <i>J. Med. Chem. </i>37: 1233. </div>
</li> <li> <para-num num="[0259]"> </para-num> <div class="description-line" id="P-0259" num="0259">Libraries of compounds may be presented in solution (e.g., Houghten, 1992. <span style="display: none">[0259] </span> <i>Biotechniques </i>13: 412-421), or on beads (Lam, 1991. <i>Nature </i>354: 82-84), on chips (Fodor, 1993. <i>Nature </i>364: 555-556), bacteria (Ladner, U.S. Pat. No. 5,223,409), spores (Ladner, U.S. Pat. No. 5,233,409), plasmids (Cull, et al., 1992. <i>Proc. Natl. Acad. Sci. USA </i>89: 1865-1869) or on phage (Scott and Smith, 1990. <i>Science </i>249: 386-390; Devlin, 1990. <i>Science </i>249: 404-406; Cwirla, et al., 1990. <i>Proc. Natl. Acad. Sci. U.S.A. </i>87: 6378-6382; Felici, 1991. <i>J. Mol. Biol. </i>222: 301-310; Ladner, U.S. Pat. No. 5,233,409.). </div>
</li> <li> <para-num num="[0260]"> </para-num> <div class="description-line" id="P-0260" num="0260">In one embodiment, an assay is a cell-based assay in which a cell which expresses a membrane-bound form of NOVX protein, or a biologically-active portion thereof, on the cell surface is contacted with a test compound and the ability of the test compound to bind to a NOVX protein determined. The cell, for example, can be of mammalian origin or a yeast cell. Determining the ability of the test compound to bind to the NOVX protein can be accomplished, for example, by coupling the test compound with a radioisotope or enzymatic label such that binding of the test compound to the NOVX protein or biologically-active portion thereof can be determined by detecting the labeled compound in a complex. For example, test compounds can be labeled with <span style="display: none">[0260] </span> <sup>125</sup>I, <sup>35</sup>S, <sup>14</sup>C, or <sup>3</sup>H, either directly or indirectly, and the radioisotope detected by direct counting of radioemission or by scintillation counting. Alternatively, test compounds can be enzymatically-labeled with, for example, horseradish peroxidase, alkaline phosphatase, or luciferase, and the enzymatic label detected by determination of conversion of an appropriate substrate to product. In one embodiment, the assay comprises contacting a cell which expresses a membrane-bound form of NOVX protein, or a biologically-active portion thereof, on the cell surface with a known compound which binds NOVX to form an assay mixture, contacting the assay mixture with a test compound, and determining the ability of the test compound to interact with a NOVX protein, wherein determining the ability of the test compound to interact with a NOVX protein comprises determining the ability of the test compound to preferentially bind to NOVX protein or a biologically-active portion thereof as compared to the known compound. </div>
</li> <li> <para-num num="[0261]"> </para-num> <div class="description-line" id="P-0261" num="0261">In another embodiment, an assay is a cell-based assay comprising contacting a cell expressing a membrane-bound form of NOVX protein, or a biologically-active portion thereof, on the cell surface with a test compound and determining the ability of the test compound to modulate (e.g., stimulate or inhibit) the activity of the NOVX protein or biologically-active portion thereof. Determining the ability of the test compound to modulate the activity of NOVX or a biologically-active portion thereof can be accomplished, for example, by determining the ability of the NOVX protein to bind to or interact with a NOVX target molecule. As used herein, a “target molecule” is a molecule with which a NOVX protein binds or interacts in nature, for example, a molecule on the surface of a cell which expresses a NOVX interacting protein, a molecule on the surface of a second cell, a molecule in the extracellular milieu, a molecule associated with the internal surface of a cell membrane or a cytoplasmic molecule. A NOVX target molecule can be a non-NOVX molecule or a NOVX protein or polypeptide of the invention In one embodiment, a NOVX target molecule is a component of a signal transduction pathway that facilitates transduction of an extracellular signal (e.g. a signal generated by binding of a compound to a membrane-bound NOVX molecule) through the cell membrane and into the cell. The target, for example, can be a second intercellular protein that has catalytic activity or a protein that facilitates the association of downstream signaling molecules with NOVX. <span style="display: none">[0261] </span> </div>
</li> <li> <para-num num="[0262]"> </para-num> <div class="description-line" id="P-0262" num="0262">Determining the ability of the NOVX protein to bind to or interact with a NOVX target molecule can be accomplished by one of the methods described above for determining direct binding. In one embodiment, determining the ability of the NOVX protein to bind to or interact with a NOVX target molecule can be accomplished by determining the activity of the target molecule. For example, the activity of the target molecule can be determined by detecting induction of a cellular second messenger of the target (i.e. intracellular Ca<span style="display: none">[0262] </span> <sup>2+</sup>, diacylglycerol, IP<sub>3</sub>, etc.), detecting catalytic/enzymatic activity of the target an appropriate substrate, detecting the induction of a reporter gene (comprising a NOVX-responsive regulatory element operatively linked to a nucleic acid encoding a detectable marker, e.g., luciferase), or detecting a cellular response, for example, cell survival, cellular differentiation, or cell proliferation. </div>
</li> <li> <para-num num="[0263]"> </para-num> <div class="description-line" id="P-0263" num="0263">In yet another embodiment, an assay of the invention is a cell-free assay comprising contacting a NOVX protein or biologically-active portion thereof with a test compound and determining the ability of the test compound to bind to the NOVX protein or biologically-active portion thereof. Binding of the test compound to the NOVX protein can be determined either directly or indirectly as described above. In one such embodiment, the assay comprises contacting the NOVX protein or biologically-active portion thereof with a known compound which binds NOVX to form an assay mixture, contacting the assay mixture with a test compound, and determining the ability of the test compound to interact with a NOVX protein, wherein determining the ability of the test compound to interact with a NOVX protein comprises determining the ability of the test compound to preferentially bind to NOVX or biologically-active portion thereof as compared to the known compound. <span style="display: none">[0263] </span> </div>
</li> <li> <para-num num="[0264]"> </para-num> <div class="description-line" id="P-0264" num="0264">In still another embodiment, an assay is a cell-free assay comprising contacting NOVX protein or biologically-active portion thereof with a test compound and determining the ability of the test compound to modulate (e.g. stimulate or inhibit) the activity of the NOVX protein or biologically-active portion thereof. Determining the ability of the test compound to modulate the activity of NOVX can be accomplished, for example, by determining the ability of the NOVX protein to bind to a NOVX target molecule by one of the methods described above for determining direct binding. In an alternative embodiment, determining the ability of the test compound to modulate the activity of NOVX protein can be accomplished by determining the ability of the NOVX protein further modulate a NOVX target molecule. For example, the catalytic/enzymatic activity of the target molecule on an appropriate substrate can be determined as described above. <span style="display: none">[0264] </span> </div>
</li> <li> <para-num num="[0265]"> </para-num> <div class="description-line" id="P-0265" num="0265">In yet another embodiment, the cell-free assay comprises contacting the NOVX protein or biologically-active portion thereof with a known compound which binds NOVX protein to form an assay mixture, contacting the assay mixture with a test compound, and determining the ability of the test compound to interact with a NOVX protein, wherein determining the ability of the test compound to interact with a NOVX protein comprises determining the ability of the NOVX protein to preferentially bind to or modulate the activity of a NOVX target molecule. <span style="display: none">[0265] </span> </div>
</li> <li> <para-num num="[0266]"> </para-num> <div class="description-line" id="P-0266" num="0266">The cell-free assays of the invention are amenable to use of both the soluble form or the membrane-bound form of NOVX protein. In the case of cell-free assays comprising the membrane-bound form of NOVX protein, it may be desirable to utilize a solubilizing agent such that the membrane-bound form of NOVX protein is maintained in solution. Examples of such solubilizing agents include non-ionic detergents such as n-octylglucoside, n-dodecylglucoside, n-dodecylmaltoside, octanoyl-N-methylglucamide, decanoyl-N-methylglucamide, Triton® X-100, Triton® X-114, Thesit®, Isotridecypoly(ethylene glycol ether)<span style="display: none">[0266] </span> <sub>n</sub>, N-dodecyl-N,N-dimethyl-3-ammonio-1-propane sulfonate, 3-(3-cholamidopropyl) dimethylamminiol-1-propane sulfonate (CHAPS), or 3-(3-cholamidopropyl)dimethylamminiol-2-hydroxy-1-propane sulfonate (CHAPSO). </div>
</li> <li> <para-num num="[0267]"> </para-num> <div class="description-line" id="P-0267" num="0267">In more than one embodiment of the above assay methods of the invention, it may be desirable to immobilize either NOVX protein or its target molecule to facilitate separation of complexed from uncomplexed forms of one or both of the proteins, as well as to accommodate automation of the assay. Binding of a test compound to NOVX protein, or interaction of NOVX protein with a target molecule in the presence and absence of a candidate compound, can be accomplished in any vessel suitable for containing the reactants. Examples of such vessels include microtiter plates, test tubes, and micro-centrifuge tubes. In one embodiment, a fusion protein can be provided that adds a domain that allows one or both of the proteins to be bound to a matrix. For example, GST-NOVX fusion proteins or GST-target fusion proteins can be adsorbed onto glutathione sepharose beads (Sigma Chemical, St. Louis, Mo.) or glutathione derivatized microtiter plates, that are then combined with the test compound or the test compound and either the non-adsorbed target protein or NOVX protein, and the mixture is incubated under conditions conducive to complex formation (e.g., at physiological conditions for salt and pH). Following incubation, the beads or microtiter plate wells are washed to remove any unbound components, the matrix immobilized in the case of beads, complex determined either directly or indirectly, for example, as described, supra. Alternatively, the complexes can be dissociated from the matrix, and the level of NOVX protein binding or activity determined using standard techniques. <span style="display: none">[0267] </span> </div>
</li> <li> <para-num num="[0268]"> </para-num> <div class="description-line" id="P-0268" num="0268">Other techniques for immobilizing proteins on matrices can also be used in the screening assays of the invention. For example, either the NOVX protein or its target molecule can be immobilized utilizing conjugation of biotin and streptavidin. Biotinylated NOVX protein or target molecules can be prepared from biotin-NHS (N-hydroxy-succinimide) using techniques well-known within the art (e.g., biotinylation kit, Pierce Chemicals, Rockford, Ill.), and immobilized in the wells of streptavidin-coated 96 well plates (Pierce Chemical). Alternatively, antibodies reactive with NOVX protein or target molecules, but which do not interfere with binding of the NOVX protein to its target molecule, can be derivatized to the wells of the plate, and unbound target or NOVX protein trapped in the wells by antibody conjugation. Methods for detecting such complexes, in addition to those described above for the GST-immobilized complexes, include immunodetection of complexes using antibodies reactive with the NOVX protein or target molecule, as well as enzyme-linked assays that rely on detecting an enzymatic activity associated with the NOVX protein or target molecule. <span style="display: none">[0268] </span> </div>
</li> <li> <para-num num="[0269]"> </para-num> <div class="description-line" id="P-0269" num="0269">In another embodiment, modulators of NOVX protein expression are identified in a method wherein a cell is contacted with a candidate compound and the expression of NOVX mRNA or protein in the cell is determined. The level of expression of NOVX mRNA or protein in the presence of the candidate compound is compared to the level of expression of NOVX mRNA or protein in the absence of the candidate compound. The candidate compound can then be identified as a modulator of NOVX mRNA or protein expression based upon this comparison. For example, when expression of NOVX mRNA or protein is greater (i.e., statistically significantly greater) in the presence of the candidate compound than in its absence, the candidate compound is identified as a stimulator of NOVX mRNA or protein expression. Alternatively, when expression of NOVX mRNA or protein is less (statistically significantly less) in the presence of the candidate compound than in its absence, the candidate compound is identified as an inhibitor of NOVX mRNA or protein expression. The level of NOVX mRNA or protein expression in the cells can be determined by methods described herein for detecting NOVX mRNA or protein. <span style="display: none">[0269] </span> </div>
</li> <li> <para-num num="[0270]"> </para-num> <div class="description-line" id="P-0270" num="0270">In yet another aspect of the invention, the NOVX proteins can be used as “bait proteins” in a two-hybrid assay or three hybrid assay (see, e.g., U.S. Pat. No. 5,283,317; Zervos, et al., 1993. <span style="display: none">[0270] </span> <i>Cell </i>72: 223-232; Madura, et al., 1993. <i>J. Biol. Chem. </i>268: 12046-12054; Bartel, et al., 1993. <i>Biotechniques </i>14: 920-924; Iwabuchi, et al., 1993. <i>Oncogene </i>8: 1693-1696; and Brent WO 94/10300), to identify other proteins that bind to or interact with NOVX (“NOVX-binding proteins” or “NOVX-bp”) and modulate NOVX activity. Such NOVX-binding proteins are also likely to be involved in the propagation of signals by the NOVX proteins as, for example, upstream or downstream elements of the NOVX pathway. </div>
</li> <li> <para-num num="[0271]"> </para-num> <div class="description-line" id="P-0271" num="0271">The two-hybrid system is based on the modular nature of most transcription factors, which consist of separable DNA-binding and activation domains. Briefly, the assay utilizes two different DNA constructs. In one construct, the gene that codes for NOVX is fused to a gene encoding the DNA binding domain of a known transcription factor (e.g., GAL-4). In the other construct, a DNA sequence, from a library of DNA sequences, that encodes an unidentified protein (“prey” or “sample”) is fused to a gene that codes for the activation domain of the known transcription factor. If the “bait” and the “prey” proteins are able to interact, in vivo, forming a NOVX-dependent complex, the DNA-binding and activation domains of the transcription factor are brought into close proximity. This proximity allows transcription of a reporter gene (e.g., LacZ) that is operably linked to a transcriptional regulatory site responsive to the transcription factor. Expression of the reporter gene can be detected and cell colonies containing the functional transcription factor can be isolated and used to obtain the cloned gene that encodes the protein which interacts with NOVX. <span style="display: none">[0271] </span> </div>
</li> <li> <para-num num="[0272]"> </para-num> <div class="description-line" id="P-0272" num="0272">The invention further pertains to novel agents identified by the aforementioned screening assays and uses thereof for treatments as described herein. <span style="display: none">[0272] </span> </div>
</li> <li> <para-num num="[0273]"> </para-num> <div class="description-line" id="P-0273" num="0273">Detection Assays <span style="display: none">[0273] </span> </div>
</li> <li> <para-num num="[0274]"> </para-num> <div class="description-line" id="P-0274" num="0274">Portions or fragments of the cDNA sequences identified herein (and the corresponding complete gene sequences) can be used in numerous ways as polynucleotide reagents. By way of example, and not of limitation, these sequences can be used to: (i) identify an individual from a minute biological sample (tissue typing); and (ii) aid in forensic identification of a biological sample. Some of these applications are described in the subsections, below. <span style="display: none">[0274] </span> </div>
</li> <li> <para-num num="[0275]"> </para-num> <div class="description-line" id="P-0275" num="0275">Tissue Typing <span style="display: none">[0275] </span> </div>
</li> <li> <para-num num="[0276]"> </para-num> <div class="description-line" id="P-0276" num="0276">The NOVX sequences of the invention can be used to identify individuals from minute biological samples. In this technique, an individual's genomic DNA is digested with one or more restriction enzymes, and probed on a Southern blot to yield unique bands for identification. The sequences of the invention are useful as additional DNA markers for RFLP (“restriction fragment length polymorphisms,” described in U.S. Pat. No. 5,272,057). <span style="display: none">[0276] </span> </div>
</li> <li> <para-num num="[0277]"> </para-num> <div class="description-line" id="P-0277" num="0277">Furthermore, the sequences of the invention can be used to provide an alternative technique that determines the actual base-by-base DNA sequence of selected portions of an individual's genome. Thus, the NOVX sequences described herein can be used to prepare two PCR primers from the 5′- and 3′-termini of the sequences. These primers can then be used to amplify an individual's DNA and subsequently sequence it. <span style="display: none">[0277] </span> </div>
</li> <li> <para-num num="[0278]"> </para-num> <div class="description-line" id="P-0278" num="0278">Panels of corresponding DNA sequences from individuals, prepared in this manner, can provide unique individual identifications, as each individual will have a unique set of such DNA sequences due to allelic differences. The sequences of the invention can be used to obtain such identification sequences from individuals and from tissue. The NOVX sequences of the invention uniquely represent portions of the human genome. Allelic variation occurs to some degree in the coding regions of these sequences, and to a greater degree in the noncoding regions. It is estimated that allelic variation between individual humans occurs with a frequency of about once per each 500 bases. Much of the allelic variation is due to single nucleotide polymorphisms (SNPs), which include restriction fragment length polymorphisms (RFLPs). <span style="display: none">[0278] </span> </div>
</li> <li> <para-num num="[0279]"> </para-num> <div class="description-line" id="P-0279" num="0279">Each of the sequences described herein can, to some degree, be used as a standard against which DNA from an individual can be compared for identification purposes. Because greater numbers of polymorphisms occur in the noncoding regions, fewer sequences are necessary to differentiate individuals. The noncoding sequences can comfortably provide positive individual identification with a panel of perhaps 10 to 1,000 primers that each yield a noncoding amplified sequence of 100 bases. If predicted coding sequences, such as those in SEQ ID NO: 1, 3, 5, 7, 9, 11, or 13 are used, a more appropriate number of primers for positive individual identification would be 500-2,000. <span style="display: none">[0279] </span> </div>
</li> <li> <para-num num="[0280]"> </para-num> <div class="description-line" id="P-0280" num="0280">Predictive Medicine <span style="display: none">[0280] </span> </div>
</li> <li> <para-num num="[0281]"> </para-num> <div class="description-line" id="P-0281" num="0281">The invention also pertains to the field of predictive medicine in which diagnostic assays, prognostic assays, pharmacogenomics, and monitoring clinical trials are used for prognostic (predictive) purposes to thereby treat an individual prophylactically. Accordingly, one aspect of the invention relates to diagnostic assays for determining NOVX protein and/or nucleic acid expression as well as NOVX activity, in the context of a biological sample (e.g., blood, serum, cells, tissue) to thereby determine whether an individual is afflicted with a disease or disorder, or is at risk of developing a disorder, associated with aberrant NOVX expression or activity. Disorders associated with aberrant NOVX expression of activity include, for example, disorders of renal and pancreatic dysfunction, e.g. diabetes, hypertension, cirrhosis, and cancer. <span style="display: none">[0281] </span> </div>
</li> <li> <para-num num="[0282]"> </para-num> <div class="description-line" id="P-0282" num="0282">The invention also provides for prognostic (or predictive) assays for determining whether an individual is at risk of developing a disorder associated with NOVX protein, nucleic acid expression or activity. For example, mutations in a NOVX gene can be assayed in a biological sample. Such assays can be used for prognostic or predictive purpose to thereby prophylactically treat an individual prior to the onset of a disorder characterized by or associated with NOVX protein, nucleic acid expression, or biological activity. <span style="display: none">[0282] </span> </div>
</li> <li> <para-num num="[0283]"> </para-num> <div class="description-line" id="P-0283" num="0283">Another aspect of the invention provides methods for determining NOVX protein, nucleic acid expression or activity in an individual to thereby select appropriate therapeutic or prophylactic agents for that individual (referred to herein as “pharmacogenomics”). Pharmacogenomics allows for the selection of agents (e.g., drugs) for therapeutic or prophylactic treatment of an individual based on the genotype of the individual (e.g., the genotype of the individual examined to determine the ability of the individual to respond to a particular agent.) <span style="display: none">[0283] </span> </div>
</li> <li> <para-num num="[0284]"> </para-num> <div class="description-line" id="P-0284" num="0284">Yet another aspect of the invention pertains to monitoring the influence of agents (e.g., drugs, compounds) on the expression or activity of NOVX in clinical trials. <span style="display: none">[0284] </span> </div>
</li> <li> <para-num num="[0285]"> </para-num> <div class="description-line" id="P-0285" num="0285">These and other agents are described in further detail in the following sections. <span style="display: none">[0285] </span> </div>
</li> <li> <para-num num="[0286]"> </para-num> <div class="description-line" id="P-0286" num="0286">Diagnostic Assays <span style="display: none">[0286] </span> </div>
</li> <li> <para-num num="[0287]"> </para-num> <div class="description-line" id="P-0287" num="0287">An exemplary method for detecting the presence or absence of NOVX in a biological sample involves obtaining a biological sample from a test subject and contacting the biological sample with a compound or an agent capable of detecting NOVX protein or nucleic acid (e.g., mRNA, genomic DNA) that encodes NOVX protein such that the presence of NOVX is detected in the biological sample. An agent for detecting NOVX mRNA or genomic DNA is a labeled nucleic acid probe capable of hybridizing to NOVX mRNA or genomic DNA. The nucleic acid probe can be, for example, a full-length NOVX nucleic acid, such as the nucleic acid of SEQ ID NO: 1, 3, 5, 7, 9, 11, or 13, or a portion thereof, such as an oligonucleotide of at least 15, 30, 50, 100, 250 or 500 nucleotides in length and sufficient to specifically hybridize under stringent conditions to NOVX mRNA or genomic DNA. Other suitable probes for use in the diagnostic assays of the invention are described herein. <span style="display: none">[0287] </span> </div>
</li> <li> <para-num num="[0288]"> </para-num> <div class="description-line" id="P-0288" num="0288">One agent for detecting NOVX protein is an antibody capable of binding to NOVX protein, preferably an antibody with a detectable label. Antibodies directed against a protein of the invention may be used in methods known within the art relating to the localization and/or quantitation of the protein (e.g., for use in measuring levels of the protein within appropriate physiological samples, for use in diagnostic methods, for use in imaging the protein, and the like). In a given embodiment, antibodies against the proteins, or derivatives, fragments, analogs or homologs thereof, that contain the antigen binding domain, are utilized as pharmacologically-active compounds. <span style="display: none">[0288] </span> </div>
</li> <li> <para-num num="[0289]"> </para-num> <div class="description-line" id="P-0289" num="0289">An antibody specific for a protein of the invention can be used to isolate the protein by standard techniques, such as immunoaffinity chromatography or immunoprecipitation. Such an antibody can facilitate the purification of the natural protein antigen from cells and of recombinantly produced antigen expressed in host cells. Moreover, such an antibody can be used to detect the antigenic protein (e.g., in a cellular lysate or cell supernatant) in order to evaluate the abundance and pattern of expression of the antigenic protein. Antibodies directed against the protein can be used diagnostically to monitor protein levels in tissue as part of a clinical testing procedure, e.g., to, for example, determine the efficacy of a given treatment regimen. Detection can be facilitated by coupling (i.e., physically linking) the antibody to a detectable substance. Examples of detectable substances include various enzymes, prosthetic groups, fluorescent materials, luminescent materials, bioluminescent materials, and radioactive materials. Examples of suitable enzymes include horseradish peroxidase, alkaline phosphatase, β-galactosidase, or acetylcholinesterase; examples of suitable prosthetic group complexes include streptavidin/biotin and avidin/biotin; examples of suitable fluorescent materials include umbelliferone, fluorescein, fluorescein isothiocyanate, rhodamine, dichlorotriazinylamine fluorescein, dansyl chloride or phycoerythrin; an example of a luminescent material includes luminol; examples of bioluminescent materials include luciferase, luciferin, and aequorin, and examples of suitable radioactive material include <span style="display: none">[0289] </span> <sup>125</sup>I, <sup>131</sup>I, <sup>35</sup>S or <sup>3</sup>H. </div>
</li> <li> <para-num num="[0290]"> </para-num> <div class="description-line" id="P-0290" num="0290">Antibodies can be polyclonal, or more preferably, monoclonal. An intact antibody, or a fragment thereof (e.g., Fab or F(ab′)<span style="display: none">[0290] </span> <sub>2</sub>) can be used. The term “labeled”, with regard to the probe or antibody, is intended to encompass direct labeling of the probe or antibody by coupling (i.e., physically linking) a detectable substance to the probe or antibody, as well as indirect labeling of the probe or antibody by reactivity with another reagent that is directly labeled. Examples of indirect labeling include detection of a primary antibody using a fluorescently-labeled secondary antibody and end-labeling of a DNA probe with biotin such that it can be detected with fluorescently-labeled streptavidin. The term “biological sample” is intended to include tissues, cells and biological fluids isolated from a subject, as well as tissues, cells and fluids present within a subject. That is, the detection method of the invention can be used to detect NOVX mRNA, protein, or genomic DNA in a biological sample in vitro as well as in vivo. For example, in vitro techniques for detection of NOVX mRNA include Northern hybridizations and in situ hybridizations. In vitro techniques for detection of NOVX protein include enzyme linked immunosorbent assays (ELISAs), Western blots, immunoprecipitations, and immunofluorescence. In vitro techniques for detection of NOVX genomic DNA include Southern hybridizations. Furthermore, in vivo techniques for detection of NOVX protein include introducing into a subject a labeled anti-NOVX antibody. For example, the antibody can be labeled with a radioactive marker whose presence and location in a subject can be detected by standard imaging techniques. </div>
</li> <li> <para-num num="[0291]"> </para-num> <div class="description-line" id="P-0291" num="0291">In one embodiment, the biological sample contains protein molecules from the test subject. Alternatively, the biological sample can contain mRNA molecules from the test subject or genomic DNA molecules from the test subject. A preferred biological sample is a peripheral blood leukocyte sample isolated by conventional means from a subject. <span style="display: none">[0291] </span> </div>
</li> <li> <para-num num="[0292]"> </para-num> <div class="description-line" id="P-0292" num="0292">In one embodiment, the methods further involve obtaining a control biological sample from a control subject, contacting the control sample with a compound or agent capable of detecting NOVX protein, mRNA, or genomic DNA, such that the presence of NOVX protein, mRNA or genomic DNA is detected in the biological sample, and comparing the presence of NOVX protein, mRNA or genomic DNA in the control sample with the presence of NOVX protein, mRNA or genomic DNA in the test sample. <span style="display: none">[0292] </span> </div>
</li> <li> <para-num num="[0293]"> </para-num> <div class="description-line" id="P-0293" num="0293">The invention also encompasses kits for detecting the presence of NOVX in a biological sample. For example, the kit can comprise: a labeled compound or agent capable of detecting NOVX protein or mRNA in a biological sample; means for determining the amount of NOVX in the sample; and means for comparing the amount of NOVX in the sample with a standard. The compound or agent can be packaged in a suitable container. The kit can further comprise instructions for using the kit to detect NOVX protein or nucleic acid. <span style="display: none">[0293] </span> </div>
</li> <li> <para-num num="[0294]"> </para-num> <div class="description-line" id="P-0294" num="0294">Prognostic Assays <span style="display: none">[0294] </span> </div>
</li> <li> <para-num num="[0295]"> </para-num> <div class="description-line" id="P-0295" num="0295">The diagnostic methods described herein can furthermore be utilized to identify subjects having or at risk of developing a disease or disorder associated with aberrant NOVX expression or activity. For example, the assays described herein, such as the preceding diagnostic assays or the following assays, can be utilized to identify a subject having or at risk of developing a disorder associated with NOVX protein, nucleic acid expression or activity. Such disorders include for example, disorders of renal and pancreas dysfunction, e.g. diabetes, hypertension, cirrhosis, and cancer. <span style="display: none">[0295] </span> </div>
</li> <li> <para-num num="[0296]"> </para-num> <div class="description-line" id="P-0296" num="0296">Alternatively, the prognostic assays can be utilized to identify a subject having or at risk for developing a disease or disorder. Thus, the invention provides a method for identifying a disease or disorder associated with aberrant NOVX expression or activity in which a test sample is obtained from a subject and NOVX protein or nucleic acid (e.g., mRNA, genomic DNA) is detected, wherein the presence of NOVX protein or nucleic acid is diagnostic for a subject having or at risk of developing a disease or disorder associated with aberrant NOVX expression or activity. As used herein, a “test sample” refers to a biological sample obtained from a subject of interest. For example, a test sample can be a biological fluid (e.g., serum), cell sample, or tissue. <span style="display: none">[0296] </span> </div>
</li> <li> <para-num num="[0297]"> </para-num> <div class="description-line" id="P-0297" num="0297">Furthermore, the prognostic assays described herein can be used to determine whether a subject can be administered an agent,(e.g., an agonist, antagonist, peptidomimetic, protein, peptide, nucleic acid, small molecule, or other drug candidate) to treat a disease or disorder associated with aberrant NOVX expression or activity. For example, such methods can be used to determine whether a subject can be effectively treated with an agent for a disorder. Thus, the invention provides methods for determining whether a subject can be effectively treated with an agent for a disorder associated with aberrant NOVX expression or activity in which a test sample is obtained and NOVX protein or nucleic acid is detected (e.g., wherein the presence of NOVX protein or nucleic acid is diagnostic for a subject that can be administered the agent to treat a disorder associated with aberrant NOVX expression or activity). <span style="display: none">[0297] </span> </div>
</li> <li> <para-num num="[0298]"> </para-num> <div class="description-line" id="P-0298" num="0298">The methods of the invention can also be used to detect genetic lesions in a NOVX gene, thereby determining if a subject with the lesioned gene is at risk for a disorder characterized by aberrant cell proliferation and/or differentiation. In various embodiments, the methods include detecting, in a sample of cells from the subject, the presence or absence of a genetic lesion characterized by at least one of an alteration affecting the integrity of a gene encoding a NOVX-protein, or the misexpression of the NOVX gene. For example, such genetic lesions can be detected by ascertaining the existence of at least one of: (i) a deletion of one or more nucleotides from a NOVX gene; (ii) an addition of one or more nucleotides to a NOVX gene; (iii) a substitution of one or more nucleotides of a NOVX gene, (iv) a chromosomal rearrangement of a NOVX gene; (v) an alteration in the level of a messenger RNA transcript of a NOVX gene, (vi) aberrant modification of a NOVX gene, such as of the methylation pattern of the genomic DNA, (vii) the presence of a non-wild-type splicing pattern of a messenger RNA transcript of a NOVX gene, (viii) a non-wild-type level of a NOVX protein, (ix) allelic loss of a NOVX gene, and (x) inappropriate post-translational modification of a NOVX protein. As described herein, there are a large number of assay techniques known in the art which can be used for detecting lesions in a NOVX gene. A preferred biological sample is a peripheral blood leukocyte sample isolated by conventional means from a subject. However, any biological sample containing nucleated cells may be used, including, for example, buccal mucosal cells. <span style="display: none">[0298] </span> </div>
</li> <li> <para-num num="[0299]"> </para-num> <div class="description-line" id="P-0299" num="0299">In certain embodiments, detection of the lesion involves the use of a probe/primer in a polymerase chain reaction (PCR) (see, e.g., U.S. Pat. Nos. 4,683,195 and 4,683,202), such as anchor PCR or RACE PCR, or, alternatively, in a ligation chain reaction (LCR) (see, e.g., Landegran, et al., 1988. <span style="display: none">[0299] </span> <i>Science </i>241: 1077-1080; and Nakazawa, et al., 1994. <i>Proc. Natl. Acad. Sci. USA </i>91: 360-364), the latter of which can be particularly useful for detecting point mutations in the NOVX-gene (see, Abravaya, et al., 1995. <i>Nucl. Acids Res. </i>23: 675-682). This method can include the steps of collecting a sample of cells from a patient, isolating nucleic acid (e.g., genomic, mRNA or both) from the cells of the sample, contacting the nucleic acid sample with one or more primers that specifically hybridize to a NOVX gene under conditions such that hybridization and amplification of the NOVX gene (if present) occurs, and detecting the presence or absence of an amplification product, or detecting the size of the amplification product and comparing the length to a control sample. It is anticipated that PCR and/or LCR may be desirable to use as a preliminary amplification step in conjunction with any of the techniques used for detecting mutations described herein. </div>
</li> <li> <para-num num="[0300]"> </para-num> <div class="description-line" id="P-0300" num="0300">Alternative amplification methods include: self sustained sequence replication (see, Guatelli, et al., 1990. <span style="display: none">[0300] </span> <i>Proc. Natl. Acad. Sci. USA </i>87: 1874-1878), transcriptional amplification system (see, Kwoh, et al., 1989. <i>Proc. Natl. Acad. Sci. USA </i>86: 1173-1177); Qβ Replicase (see, Lizardi, et al, 1988. <i>BioTechnology </i>6: 1197), or any other nucleic acid amplification method, followed by the detection of the amplified molecules using techniques well known to those of skill in the art. These detection schemes are especially useful for the detection of nucleic acid molecules if such molecules are present in very low numbers. </div>
</li> <li> <para-num num="[0301]"> </para-num> <div class="description-line" id="P-0301" num="0301">In an alternative embodiment, mutations in a NOVX gene from a sample cell can be identified by alterations in restriction enzyme cleavage patterns. For example, sample and control DNA is isolated, amplified (optionally), digested with one or more restriction endonucleases, and fragment length sizes are determined by gel electrophoresis and compared. Differences in fragment length sizes between sample and control DNA indicates mutations in the sample DNA. Moreover, the use of sequence specific ribozymes (see, e.g., U.S. Pat. No. 5,493,531) can be used to score for the presence of specific mutations by development or loss of a ribozyme cleavage site. <span style="display: none">[0301] </span> </div>
</li> <li> <para-num num="[0302]"> </para-num> <div class="description-line" id="P-0302" num="0302">In other embodiments, genetic mutations in NOVX can be identified by hybridizing a sample and control nucleic acids, e.g., DNA or RNA, to high-density arrays containing hundreds or thousands of oligonucleotides probes. See, e.g., Cronin, et al., 1996. <span style="display: none">[0302] </span> <i>Human Mutation </i>7: 244-255; Kozal, et al., 1996. <i>Nat. Med. </i>2: 753-759. For example, genetic mutations in NOVX can be identified in two dimensional arrays containing light-generated DNA probes as described in Cronin, et al., supra. Briefly, a first hybridization array of probes can be used to scan through long stretches of DNA in a sample and control to identify base changes between the sequences by making linear arrays of sequential overlapping probes. This step allows the identification of point mutations. This is followed by a second hybridization array that allows the characterization of specific mutations by using smaller, specialized probe arrays complementary to all variants or mutations detected. Each mutation array is composed of parallel probe sets, one complementary to the wild-type gene and the other complementary to the mutant gene. </div>
</li> <li> <para-num num="[0303]"> </para-num> <div class="description-line" id="P-0303" num="0303">In yet another embodiment, any of a variety of sequencing reactions known in the art can be used to directly sequence the NOVX gene and detect mutations by comparing the sequence of the sample NOVX with the corresponding wild-type (control) sequence. Examples of sequencing reactions include those based on techniques developed by Maxim and Gilbert, 1977. <span style="display: none">[0303] </span> <i>Proc. Natl. Acad. Sci. USA </i>74: 560 or Sanger, 1977. <i>Proc. Natl. Acad. Sci. USA </i>74: 5463. It is also contemplated that any of a variety of automated sequencing procedures can be utilized when performing the diagnostic assays (see, e.g., Naeve, et al., 1995. <i>Biotechniques </i>19: 448), including sequencing by mass spectrometry (see, e.g., PCT International Publication No. WO 94/16101; Cohen, et al., 1996. <i>Adv. Chromatography </i>36:127-162; and Griffin, et al., 1993. <i>Appl. Biochem. Biotechnol. </i>38: 147-159). </div>
</li> <li> <para-num num="[0304]"> </para-num> <div class="description-line" id="P-0304" num="0304">Other methods for detecting mutations in the NOVX gene include methods in which protection from cleavage agents is used to detect mismatched bases in RNA/RNA or RNA/DNA heteroduplexes. See, e.g., Myers, et al., 1985. <span style="display: none">[0304] </span> <i>Science </i>230: 1242. In general, the art technique of “mismatch cleavage” starts by providing heteroduplexes of formed by hybridizing (labeled) RNA or DNA containing the wild-type NOVX sequence with potentially mutant RNA or DNA obtained from a tissue sample. The double-stranded duplexes are treated with an agent that cleaves single-stranded regions of the duplex such as which will exist due to basepair mismatches between the control and sample strands. For instance, RNA/DNA duplexes can be treated with RNase and DNA/DNA hybrids treated with S<sub>1 </sub>nuclease to enzymatically digesting the mismatched regions. In other embodiments, either DNA/DNA or RNA/DNA duplexes can be treated with hydroxylamine or osmium tetroxide and with piperidine in order to digest mismatched regions. After digestion of the mismatched regions, the resulting material is then separated by size on denaturing polyacrylamide gels to determine the site of mutation. See, e.g., Cotton, et al., 1988. <i>Proc. Natl. Acad. Sci. USA </i>85: 4397; Saleeba, et al., 1992. <i>Methods Enzymol. </i>217: 286-295. In an embodiment, the control DNA or RNA can be labeled for detection. </div>
</li> <li> <para-num num="[0305]"> </para-num> <div class="description-line" id="P-0305" num="0305">In still another embodiment, the mismatch cleavage reaction employs one or more proteins that recognize mismatched base pairs in double-stranded DNA (so called “DNA mismatch repair” enzymes) in defined systems for detecting and mapping point mutations in NOVX cDNAs obtained from samples of cells. For example, the mutY enzyme of <span style="display: none">[0305] </span> <i>E. coli </i>cleaves A at G/A mismatches and the thymidine DNA glycosylase from HeLa cells cleaves T at G/T mismatches. See, e.g., Hsu, et al., 1994. <i>Carcinogenesis </i>15: 1657-1662. According to an exemplary embodiment, a probe based on a NOVX sequence, e.g., a wild-type NOVX sequence, is hybridized to a cDNA or other DNA product from a test cell(s). The duplex is treated with a DNA mismatch repair enzyme, and the cleavage products, if any, can be detected from electrophoresis protocols or the like. See, e.g., U.S. Pat. No. 5,459,039. </div>
</li> <li> <para-num num="[0306]"> </para-num> <div class="description-line" id="P-0306" num="0306">In other embodiments, alterations in electrophoretic mobility will be used to identify mutations in NOVX genes. For example, single strand conformation polymorphism (SSCP) may be used to detect differences in electrophoretic mobility between mutant and wild type nucleic acids. See, e.g., Orita, et al., 1989. <span style="display: none">[0306] </span> <i>Proc. Natl. Acad. Sci. USA: </i>86: 2766; Cotton, 1993. <i>Mutat. Res. </i>285: 125-144; Hayashi, 1992. <i>Genet. Anal. Tech. Appl. </i>9: 73-79. Single-stranded DNA fragments of sample and control NOVX nucleic acids will be denatured and allowed to renature. The secondary structure of single-stranded nucleic acids varies according to sequence, the resulting alteration in electrophoretic mobility enables the detection of even a single base change. The DNA fragments may be labeled or detected with labeled probes. The sensitivity of the assay may be enhanced by using RNA (rather than DNA), in which the secondary structure is more sensitive to a change in sequence. In one embodiment, the subject method utilizes heteroduplex analysis to separate double stranded heteroduplex molecules on the basis of changes in electrophoretic mobility. See, e.g., Keen, et al., 1991. <i>Trends Genet. </i>7:5. </div>
</li> <li> <para-num num="[0307]"> </para-num> <div class="description-line" id="P-0307" num="0307">In yet another embodiment, the movement of mutant or wild-type fragments in polyacrylamide gels containing a gradient of denaturant is assayed using denaturing gradient gel electrophoresis (DGGE). See, e.g., Myers, et al., 1985. <span style="display: none">[0307] </span> <i>Nature </i>313: 495. When DGGE is used as the method of analysis, DNA will be modified to insure that it does not completely denature, for example by adding a GC clamp of approximately 40 bp of high-melting GC-rich DNA by PCR. In a further embodiment, a temperature gradient is used in place of a denaturing gradient to identify differences in the mobility of control and sample DNA. See, e.g., Rosenbaum and Reissner, 1987. <i>Biophys. Chem. </i>265: 12753. </div>
</li> <li> <para-num num="[0308]"> </para-num> <div class="description-line" id="P-0308" num="0308">Examples of other techniques for detecting point mutations include, but are not limited to, selective oligonucleotide hybridization, selective amplification, or selective primer extension. For example, oligonucleotide primers may be prepared in which the known mutation is placed centrally and then hybridized to target DNA under conditions that permit hybridization only if a perfect match is found. See, e.g., Saiki, et al., 1986. <span style="display: none">[0308] </span> <i>Nature </i>324: 163; Saiki, et al., 1989. <i>Proc. Natl. Acad. Sci. USA </i>86: 6230. Such allele specific oligonucleotides are hybridized to PCR amplified target DNA or a number of different mutations when the oligonucleotides are attached to the hybridizing membrane and hybridized with labeled target DNA. </div>
</li> <li> <para-num num="[0309]"> </para-num> <div class="description-line" id="P-0309" num="0309">Alternatively, allele specific amplification technology that depends on selective PCR amplification may be used in conjunction with the instant invention. Oligonucleotides used as primers for specific amplification may carry the mutation of interest in the center of the molecule (so that amplification depends on differential hybridization; see, e.g., Gibbs, et al., 1989. <span style="display: none">[0309] </span> <i>Nucl. Acids Res. </i>17: 2437-2448) or at the extreme 3′-terminus of one primer where, under appropriate conditions, mismatch can prevent, or reduce polymerase extension (see, e.g., Prossner, 1993. <i>Tibtech. </i>11: 238). In addition it may be desirable to introduce a novel restriction site in the region of the mutation to create cleavage-based detection. See, e.g., Gasparini, et al., 1992. <i>Mol. Cell Probes </i>6: 1. It is anticipated that in certain embodiments amplification may also be performed using Taq ligase for amplification. See, e.g., Barany, 1991. <i>Proc. Natl. Acad. Sci. USA </i>88: 189. In such cases, ligation will occur only if there is a perfect match at the 3′-terminus of the 5′ sequence, making it possible to detect the presence of a known mutation at a specific site by looking for the presence or absence of amplification. </div>
</li> <li> <para-num num="[0310]"> </para-num> <div class="description-line" id="P-0310" num="0310">The methods described herein may be performed, for example, by utilizing pre-packaged diagnostic kits comprising at least one probe nucleic acid or antibody reagent described herein, which may be conveniently used, e.g., in clinical settings to diagnose patients exhibiting symptoms or family history of a disease or illness involving a NOVX gene. <span style="display: none">[0310] </span> </div>
</li> <li> <para-num num="[0311]"> </para-num> <div class="description-line" id="P-0311" num="0311">Furthermore, any cell type or tissue, preferably peripheral blood leukocytes, in which NOVX is expressed may be utilized in the prognostic assays described herein. However, any biological sample containing nucleated cells may be used, including, for example, buccal mucosal cells. <span style="display: none">[0311] </span> </div>
</li> <li> <para-num num="[0312]"> </para-num> <div class="description-line" id="P-0312" num="0312">Pharmacogenomics <span style="display: none">[0312] </span> </div>
</li> <li> <para-num num="[0313]"> </para-num> <div class="description-line" id="P-0313" num="0313">Agents, or modulators that have a stimulatory or inhibitory effect on NOVX activity (e.g., NOVX gene expression), as identified by a screening assay described herein can be administered to individuals to treat (prophylactically or therapeutically) disorders (e.g. disorders of of renal and pancreas dysfunction, e.g. diabetes, hypertension, cirrhosis, and cancer). In conjunction with such treatment, the pharmacogenomics (i.e., the study of the relationship between an individual's genotype and that individual's response to a foreign compound or drug) of the individual may be considered. Differences in metabolism of therapeutics can lead to severe toxicity or therapeutic failure by altering the relation between dose and blood concentration of the pharmacologically active drug. Thus, the pharmacogenomics of the individual permits the selection of effective agents (e.g., drugs) for prophylactic or therapeutic treatments based on a consideration of the individual's genotype. Such pharmacogenomics can further be used to determine appropriate dosages and therapeutic regimens. Accordingly, the activity of NOVX protein, expression of NOVX nucleic acid, or mutation content of NOVX genes in an individual can be determined to thereby select appropriate agent(s) for therapeutic or prophylactic treatment of the individual. <span style="display: none">[0313] </span> </div>
</li> <li> <para-num num="[0314]"> </para-num> <div class="description-line" id="P-0314" num="0314">Pharmacogenomics deals with clinically significant hereditary variations in the response to drugs due to altered drug disposition and abnormal action in affected persons. See e.g., Eichelbaum, 1996. <span style="display: none">[0314] </span> <i>Clin. Exp. Pharmacol. Physiol., </i>23: 983-985; Linder, 1997. Clin. Chem., 43: 254-266. In general, two types of pharmacogenetic conditions can be differentiated. Genetic conditions transmitted as a single factor altering the way drugs act on the body (altered drug action) or genetic conditions transmitted as single factors altering the way the body acts on drugs (altered drug metabolism). These pharmacogenetic conditions can occur either as rare defects or as polymorphisms. For example, glucose-6-phosphate dehydrogenase (G6PD) deficiency is a common inherited enzymopathy in which the main clinical complication is hemolysis after ingestion of oxidant drugs (anti-malarials, sulfonamides, analgesics, nitrofurans) and consumption of fava beans. </div>
</li> <li> <para-num num="[0315]"> </para-num> <div class="description-line" id="P-0315" num="0315">As an illustrative embodiment, the activity of drug metabolizing enzymes is a major determinant of both the intensity and duration of drug action. The discovery of genetic polymorphisms of drug metabolizing enzymes (e.g., N-acetyltransferase 2 (NAT 2) and cytochrome P450 enzymes CYP2D6 and CYP2C19) has provided an explanation as to why some patients do not obtain the expected drug effects or show exaggerated drug response and serious toxicity after taking the standard and safe dose of a drug. These polymorphisms are expressed in two phenotypes in the population, the extensive metabolizer (EM) and poor metabolizer (PM). The prevalence of PM is different among different populations. For example, the gene coding for CYP2D6 is highly polymorphic and several mutations have been identified in PM, which all lead to the absence of functional CYP2D6. Poor metabolizers of CYP2D6 and CYP2C19 quite frequently experience exaggerated drug response and side effects when they receive standard doses. If a metabolite is the active therapeutic moiety, PM show no therapeutic response, as demonstrated for the analgesic effect of codeine mediated by its CYP2D6-formed metabolite morphine. At the other extreme are the so called ultra-rapid metabolizers who do not respond to standard doses. Recently, the molecular basis of ultra-rapid metabolism has been identified to be due to CYP2D6 gene amplification. <span style="display: none">[0315] </span> </div>
</li> <li> <para-num num="[0316]"> </para-num> <div class="description-line" id="P-0316" num="0316">Thus, the activity of NOVX protein, expression of NOVX nucleic acid, or mutation content of NOVX genes in an individual can be determined to thereby select appropriate agent(s) for therapeutic or prophylactic treatment of the individual. In addition, pharmacogenetic studies can be used to apply genotyping of polymorphic alleles encoding drug-metabolizing enzymes to the identification of an individual's drug responsiveness phenotype. This knowledge, when applied to dosing or drug selection, can avoid adverse reactions or therapeutic failure and thus enhance therapeutic or prophylactic efficiency when treating a subject with a NOVX modulator, such as a modulator identified by one of the exemplary screening assays described herein. <span style="display: none">[0316] </span> </div>
</li> <li> <para-num num="[0317]"> </para-num> <div class="description-line" id="P-0317" num="0317">Monitoring of Effects During Clinical Trials <span style="display: none">[0317] </span> </div>
</li> <li> <para-num num="[0318]"> </para-num> <div class="description-line" id="P-0318" num="0318">Monitoring the influence of agents (e.g., drugs, compounds) on the expression or activity of NOVX (e.g., the ability to modulate aberrant cell proliferation) can be applied not only in basic drug screening, but also in clinical trials. For example, the effectiveness of an agent determined by a screening assay as described herein to increase NOVX gene expression, protein levels, or upregulate NOVX activity, can be monitored in clinical trails of subjects exhibiting decreased NOVX gene expression, protein levels, or downregulated NOVX activity. Alternatively, the effectiveness of an agent determined by a screening assay to decrease NOVX gene expression, protein levels, or downregulate NOVX activity, can be monitored in clinical trails of subjects exhibiting increased NOVX gene expression, protein levels, or upregulated NOVX activity. In such clinical trials, the expression or activity of NOVX and, preferably, other genes that have been implicated in, for example, a cellular proliferation or immune disorder can be used as a “read out” or markers of the immune responsiveness of a particular cell. <span style="display: none">[0318] </span> </div>
</li> <li> <para-num num="[0319]"> </para-num> <div class="description-line" id="P-0319" num="0319">By way of example, and not of limitation, genes, including NOVX, that are modulated in cells by treatment with an agent (e.g., compound, drug or small molecule) that modulates NOVX activity (e.g., identified in a screening assay as described herein) can be identified. Thus, to study the effect of agents on cellular proliferation disorders, for example, in a clinical trial, cells can be isolated and RNA prepared and analyzed for the levels of expression of NOVX and other genes implicated in the disorder. The levels of gene expression (i.e., a gene expression pattern) can be quantified by Northern blot analysis or RT-PCR, as described herein, or alternatively by measuring the amount of protein produced, by one of the methods as described herein, or by measuring the levels of activity of NOVX or other genes. In this manner, the gene expression pattern can serve as a marker, indicative of the physiological response of the cells to the agent. Accordingly, this response state may be determined before, and at various points during, treatment of the individual with the agent. <span style="display: none">[0319] </span> </div>
</li> <li> <para-num num="[0320]"> </para-num> <div class="description-line" id="P-0320" num="0320">In one embodiment, the invention provides a method for monitoring the effectiveness of treatment of a subject with an agent (e.g., an agonist, antagonist, protein, peptide, peptidomimetic, nucleic acid, small molecule, or other drug candidate identified by the screening assays described herein) comprising the steps of (i) obtaining a pre-administration sample from a subject prior to administration of the agent; (ii) detecting the level of expression of a NOVX protein, mRNA, or genomic DNA in the preadministration sample; (iii) obtaining one or more post-administration samples from the subject; (iv) detecting the level of expression or activity of the NOVX protein, mRNA, or genomic DNA in the post-administration samples; (v) comparing the level of expression or activity of the NOVX protein, mRNA, or genomic DNA in the pre-administration sample with the NOVX protein, mRNA, or genomic DNA in the post administration sample or samples; and (vi) altering the administration of the agent to the subject accordingly. For example, increased administration of the agent may be desirable to increase the expression or activity of NOVX to higher levels than detected, i.e., to increase the effectiveness of the agent. Alternatively, decreased administration of the agent may be desirable to decrease expression or activity of NOVX to lower levels than detected, i.e., to decrease the effectiveness of the agent. <span style="display: none">[0320] </span> </div>
</li> <li> <para-num num="[0321]"> </para-num> <div class="description-line" id="P-0321" num="0321">Methods of Treatment <span style="display: none">[0321] </span> </div>
</li> <li> <para-num num="[0322]"> </para-num> <div class="description-line" id="P-0322" num="0322">The invention provides for both prophylactic and therapeutic methods of treating a subject at risk of (or susceptible to) a disorder or having a disorder associated with aberrant NOVX expression or activity. Disorders associated with aberrant NOVX expression include, for example, disorders of renal and pancreas dysfunction, e.g. diabetes, hypertension, cirrhosis, and cancer. <span style="display: none">[0322] </span> </div>
</li> <li> <para-num num="[0323]"> </para-num> <div class="description-line" id="P-0323" num="0323">These methods of treatment will be discussed more fully, below. <span style="display: none">[0323] </span> </div>
</li> <li> <para-num num="[0324]"> </para-num> <div class="description-line" id="P-0324" num="0324">Disease and Disorders <span style="display: none">[0324] </span> </div>
</li> <li> <para-num num="[0325]"> </para-num> <div class="description-line" id="P-0325" num="0325">Diseases and disorders that are characterized by increased (relative to a subject not suffering from the disease or disorder) levels or biological activity may be treated with Therapeutics that antagonize (i e., reduce or inhibit) activity. Therapeutics that antagonize activity may be administered in a therapeutic or prophylactic manner. Therapeutics that may be utilized include, but are not limited to: (i) an aforementioned peptide, or analogs, derivatives, fragments or homologs thereof; (ii) antibodies to an aforementioned peptide; (iii) nucleic acids encoding an aforementioned peptide; (iv) administration of antisense nucleic acid and nucleic acids that are “dysfunctional” (i.e., due to a heterologous insertion within the coding sequences of coding sequences to an aforementioned peptide) that are utilized to “knockout” endogenous function of an aforementioned peptide by homologous recombination (see, e.g., Capecchi, 1989. <span style="display: none">[0325] </span> <i>Science </i>244:1288-1292); or (v) modulators ( i.e., inhibitors, agonists and antagonists, including additional peptide mimetic of the invention or antibodies specific to a peptide of the invention) that alter the interaction between an aforementioned peptide and its binding partner. </div>
</li> <li> <para-num num="[0326]"> </para-num> <div class="description-line" id="P-0326" num="0326">Diseases and disorders that are characterized by decreased (relative to a subject not suffering from the disease or disorder) levels or biological activity may be treated with Therapeutics that increase (i.e., are agonists to) activity. Therapeutics that upregulate activity may be administered in a therapeutic or prophylactic manner. Therapeutics that may be utilized include, but are not limited to, an aforementioned peptide, or analogs, derivatives, fragments or homologs thereof; or an agonist that increases bioavailability. <span style="display: none">[0326] </span> </div>
</li> <li> <para-num num="[0327]"> </para-num> <div class="description-line" id="P-0327" num="0327">Increased or decreased levels can be readily detected by quantifying peptide and/or RNA, by obtaining a patient tissue sample (e.g., from biopsy tissue) and assaying it in vitro for RNA or peptide levels, structure and/or activity of the expressed peptides (or mRNAs of an aforementioned peptide). Methods that are well-known within the art include, but are not limited to, immunoassays (e.g., by Western blot analysis, immunoprecipitation followed by sodium dodecyl sulfate (SDS) polyacrylamide gel electrophoresis, immunocytochemistry, etc.) and/or hybridization assays to detect expression of mRNAs (e.g., Northern assays, dot blots, in situ hybridization, and the like). <span style="display: none">[0327] </span> </div>
</li> <li> <para-num num="[0328]"> </para-num> <div class="description-line" id="P-0328" num="0328">Prophylactic Methods <span style="display: none">[0328] </span> </div>
</li> <li> <para-num num="[0329]"> </para-num> <div class="description-line" id="P-0329" num="0329">In one aspect, the invention provides a method for preventing, in a subject, a disease or condition associated with an aberrant NOVX expression or activity, by administering to the subject an agent that modulates NOVX expression or at least one NOVX activity. Subjects at risk for a disease that is caused or contributed to by aberrant NOVX expression or activity can be identified by, for example, any or a combination of diagnostic or prognostic assays as described herein. Administration of a prophylactic agent can occur prior to the manifestation of symptoms characteristic of the NOVX aberrancy, such that a disease or disorder is prevented or, alternatively, delayed in its progression. Depending upon the type of NOVX aberrancy, for example, a NOVX agonist or NOVX antagonist agent can be used for treating the subject. The appropriate agent can be determined based on screening assays described herein. The prophylactic methods of the invention are further discussed in the following subsections. <span style="display: none">[0329] </span> </div>
</li> <li> <para-num num="[0330]"> </para-num> <div class="description-line" id="P-0330" num="0330">Therapeutic Methods <span style="display: none">[0330] </span> </div>
</li> <li> <para-num num="[0331]"> </para-num> <div class="description-line" id="P-0331" num="0331">Another aspect of the invention pertains to methods of modulating NOVX expression or activity for therapeutic purposes. The modulatory method of the invention involves contacting a cell with an agent that modulates one or more of the activities of NOVX protein activity associated with the cell. An agent that modulates NOVX protein activity can be an agent as described herein, such as a nucleic acid or a protein, a naturally-occurring cognate ligand of a NOVX protein, a peptide, a NOVX peptidomimetic, or other small molecule. In one embodiment, the agent stimulates one or more NOVX protein activity. Examples of such stimulatory agents include active NOVX protein and a nucleic acid molecule encoding NOVX that has been introduced into the cell. In another embodiment, the agent inhibits one or more NOVX protein activity. Examples of such inhibitory agents include antisense NOVX nucleic acid molecules and anti-NOVX antibodies. These modulatory methods can be performed in vitro (e.g., by culturing the cell with the agent) or, alternatively, in vivo (e.g., by administering the agent to a subject). As such, the invention provides methods of treating an individual afflicted with a disease or disorder characterized by aberrant expression or activity of a NOVX protein or nucleic acid molecule. In one embodiment, the method involves administering an agent (e.g., an agent identified by a screening assay described herein), or combination of agents that modulates (e.g., up-regulates or down-regulates) NOVX expression or activity. In another embodiment, the method involves administering a NOVX protein or nucleic acid molecule as therapy to compensate for reduced or aberrant NOVX expression or activity. <span style="display: none">[0331] </span> </div>
</li> <li> <para-num num="[0332]"> </para-num> <div class="description-line" id="P-0332" num="0332">Stimulation of NOVX activity is desirable in situations in which NOVX is abnormally downregulated and/or in which increased NOVX activity is likely to have a beneficial effect. One example of such a situation is where a subject has a disorder characterized by aberrant cell proliferation and/or differentiation (e.g., cancer or immune associated). Another example of such a situation is where the subject has an immunodeficiency disease (e.g., AIDS). <span style="display: none">[0332] </span> </div>
</li> <li> <para-num num="[0333]"> </para-num> <div class="description-line" id="P-0333" num="0333">Antibodies of the invention, including polyclonal, monoclonal, humanized and fully human antibodies, may used as therapeutic agents. Such agents will generally be employed to treat or prevent a disease or pathology in a subject. An antibody preparation, preferably one having high specificity and high affinity for its target antigen, is administered to the subject and will generally have an effect due to its binding with the target. Such an effect may be one of two kinds, depending on the specific nature of the interaction between the given antibody molecule and the target antigen in question. In the first instance, administration of the antibody may abrogate or inhibit the binding of the target with an endogenous ligand to which it naturally binds. In this case, the antibody binds to the target and masks a binding site of the naturally occurring ligand, wherein the ligand serves as an effector molecule. Thus the receptor mediates a signal transduction pathway for which ligand is responsible. <span style="display: none">[0333] </span> </div>
</li> <li> <para-num num="[0334]"> </para-num> <div class="description-line" id="P-0334" num="0334">Alternatively, the effect may be one in which the antibody elicits a physiological result by virtue of binding to an effector binding site on the target molecule. In this case the target, a receptor having an endogenous ligand which may be absent or defective in the disease or pathology, binds the antibody as a surrogate effector ligand, initiating a receptor-based signal transduction event by the receptor. <span style="display: none">[0334] </span> </div>
</li> <li> <para-num num="[0335]"> </para-num> <div class="description-line" id="P-0335" num="0335">A therapeutically effective amount of an antibody of the invention relates generally to the amount needed to achieve a therapeutic objective. As noted above, this may be a binding interaction between the antibody and its target antigen that, in certain cases, interferes with the functioning of the target, and in other cases, promotes a physiological response. The amount required to be administered will furthermore depend on the binding affinity of the antibody for its specific antigen, and will also depend on the rate at which an administered antibody is depleted from the free volume other subject to which it is administered. Common ranges for therapeutically effective dosing of an antibody or antibody fragment of the invention may be, by way of nonlimiting example, from about 0.1 mg/kg body weight to about 50 mg/kg body weight. Common dosing frequencies may range, for example, from twice daily to once a week. <span style="display: none">[0335] </span> </div>
</li> <li> <para-num num="[0336]"> </para-num> <div class="description-line" id="P-0336" num="0336">Determination of the Biological Effect of the Therapeutic <span style="display: none">[0336] </span> </div>
</li> <li> <para-num num="[0337]"> </para-num> <div class="description-line" id="P-0337" num="0337">In various embodiments of the invention, suitable in vitro or in vivo assays are performed to determine the effect of a specific Therapeutic and whether its administration is indicated for treatment of the affected tissue. <span style="display: none">[0337] </span> </div>
</li> <li> <para-num num="[0338]"> </para-num> <div class="description-line" id="P-0338" num="0338">In various specific embodiments, in vitro assays may be performed with representative cells of the type(s) involved in the patient's disorder, to determine if a given Therapeutic exerts the desired effect upon the cell type(s). Compounds for use in therapy may be tested in suitable animal model systems including, but not limited to rats, mice, chicken, cows, monkeys, rabbits, and the like, prior to testing in human subjects. Similarly, for in vivo testing, any of the animal model system known in the art may be used prior to administration to human subjects. <span style="display: none">[0338] </span> </div>
</li> <li> <para-num num="[0339]"> </para-num> <div class="description-line" id="P-0339" num="0339">The invention will be further described in the following examples, which do not limit the scope of the invention described in the claims. <span style="display: none">[0339] </span> </div>
</li> <heading>EXAMPLES </heading>
<heading>Example 1 </heading>
<heading>Method of Identifying NOV1-7 Nucleic Acids </heading>
<li> <para-num num="[0340]"> </para-num> <div class="description-line" id="P-0340" num="0340">NOV1-7 nucleic acid sequences of the invention were derived by laboratory cloning of cDNA fragments covering the full length and/or part of the DNA sequences of the invention, and/or by in silico prediction of the full length and/or part of the DNA sequences of the invention from public human sequence databases. The laboratory cloning was performed by the following methods- SeqCalling™, RACE, Exon Linking, CuraSelect. These methods are briefly summarized below: <span style="display: none">[0340] </span> </div>
</li> <li> <para-num num="[0341]"> </para-num> <div class="description-line" id="P-0341" num="0341">SeqCalling™: cDNA was derived from various human samples representing multiple tissue types, normal and diseased states, physiological states, and developmental states from different donors. Samples were obtained as whole tissue, cell lines, primary cells or tissue cultured primary cells and cell lines. Cells and cell lines may have been treated with biological or chemical agents that regulate gene expression for example, growth factors, chemokines, steroids. The cDNA thus derived was then sequenced using CuraGen's proprietary SeqCalling technology. cDNA sequences from all samples were assembled with themselves and with public ESTs using bioinformatics programs to generate CuraGen's human SeqCalling database of SeqCalling assemblies. Each assembly contains one or more overlapping cDNA sequences derived from the same/different human sample(s). Fragments and ESTs were included as components for an assembly when the extent of their overlap with another component of the assembly was at a minimum of 95% over 50 bp. Each assembly can represent a gene and/or its variants such as splice forms and/or single nucleotide polymorphisms (SNPs) and their combinations. In addition, sequence traces were evaluated manually and edited for corrections if appropriate. <span style="display: none">[0341] </span> </div>
</li> <li> <para-num num="[0342]"> </para-num> <div class="description-line" id="P-0342" num="0342">RACE: Techniques based on polymerase chain reaction like Rapid amplification of cDNA ends (RACE) were used to isolate or complete the predicted sequence of the cDNA of the invention. Usually multiple clones were sequenced from one or more samples,as in the case of SeqCalling to derive the sequence which was then assembled similar to the SeqCalling process. In addition, sequence traces were evaluated manually and edited for corrections if appropriate. <span style="display: none">[0342] </span> </div>
</li> <li> <para-num num="[0343]"> </para-num> <div class="description-line" id="P-0343" num="0343">Exon Linking: The cDNA coding for the sequence was cloned by polymerase chain reaction (PCR) on the following pool of human cDNAs: adrenal gland, bone marrow, brain—amygdala, brain—cerebellum, brain—hippocampus, brain—substantia nigra, brain—thalamus, brain -whole, fetal brain, fetal kidney, fetal liver, fetal lung, heart, kidney, lymphoma—Raji, mammary gland, pancreas, pituitary gland, placenta, prostate, salivary gland, skeletal muscle, small intestine, spinal cord, spleen, stomach, testis, thyroid, trachea, uterus. Primers were designed based on in silico predictions for the full length or part (one or more exons) of the DNA/Protein sequence of the invention or by translated homology of the predicted exons to closely related human sequences or to sequences from other species. Usually multiple clones were sequenced to derive the sequence which was then assembled similar to the SeqCalling process. In addition, sequence traces were evaluated manually and edited for corrections if appropriate. <span style="display: none">[0343] </span> </div>
</li> <li> <para-num num="[0344]"> </para-num> <div class="description-line" id="P-0344" num="0344">Variant sequences have also been included in this application. A variant sequence can include a single nucleotide polymorphism (SNP). A SNP can, in some instances, be referred to as a “cSNP” to denote that the nucleotide sequence containing the SNP originates as a cDNA. A SNP can arise in several ways. For example, a SNP may be due to a substitution of one nucleotide for another at the polymorphic site. Such a substitution can be either a transition or a transversion. A SNP can also arise from a deletion of a nucleotide or an insertion of a nucleotide, relative to a reference allele. Thus, the polymorphic site is a site at which one allele bears a gap with respect to a particular nucleotide in another allele. SNPs occurring within genes may result in an alteration of the amino acid encoded by the gene at the position of the SNP. Intragenic SNPs may also be silent, however, in the case that a codon including a SNP encodes the same amino acid as a result of the redundancy of the genetic code. SNPs occurring outside the region of a gene, or in an intron within a gene, do not result in changes in any amino acid sequence of a protein but may result in altered regulation of the expression pattern for example, alteration in temporal expression, physiological response regulation, cell type expression regulation, intensity of expression, stability of transcribed message. <span style="display: none">[0344] </span> </div>
</li> <li> <para-num num="[0345]"> </para-num> <div class="description-line" id="P-0345" num="0345">For example, the genomic clone AC068385 was identified on chromosome 11 by TBLASTN using CuraGen Corporation's sequence file for members of Acetylglucosaminyltransferase and/or Acetylglucosaminyltransferase family, run against the genomic daily files made available by GenBank or obtained from Human Genome Project Sequencing Centers. These genomic clones were analyzed by the exon prediction programs Genscan and Grail as well as by other analysis programs (i.e., BLASTN, TBLASTN, TFASTN, TFASTA, BLASTX) to identify regions that corresponded to putative exons (i.e. putative coding sequences) which, when translated to an amino acid string, maintained similarity to the original protein or protein family of interest. Some additional genomic regions may have also been identified as inclusions in the invention because SeqCalling assemblies (SeqCalling is CuraGen Corporation's proprietary technology that identifies differentially expressed gene fragments, including the differential expression of genes whose overall expression is low) and/or ESTs mapped to those regions and the assemblies and/or ESTs were also overlapping with regions defined by homology or exon prediction, or because the assembly and/or EST location was in the vicinity of genomic regions identified by homology or exon prediction that had been included in the invention. <span style="display: none">[0345] </span> </div>
</li> <li> <para-num num="[0346]"> </para-num> <div class="description-line" id="P-0346" num="0346">The sequence so identified was manually concatenated and then also may have been extended using one or more additional sequences taken from CuraGen's SeqCalling assembly sequences and/or from publicly available EST sequences. SeqCalling assembly sequences were initially identified by BLASTN searches with the predicted sequence against CuraGen's SeqCalling database. SeqCalling assembly sequences which had regions with 100% identity to the predicted sequence were selected for further analysis because this identity indicates that these sequences were derived from the same genomic locus as the selected genomic clone. SeqCalling assembly s3aq:139758431; s3aq:134071499 and s3aq:105331086 were identified as having suitably significant similarity. SeqCalling assembly s3aq:139758431 has 237 components; s3aq:134071499 has 18 components; s3aq:105331086 has 3 components. <span style="display: none">[0346] </span> </div>
</li> <li> <para-num num="[0347]"> </para-num> <div class="description-line" id="P-0347" num="0347">Sequences from the following sources were thus included arriving at the novel sequence identified herein: AC068385, s3aq:139758431, s3aq:134071499, and s3aq:105331086. <span style="display: none">[0347] </span> </div>
</li> <li> <para-num num="[0348]"> </para-num> <div class="description-line" id="P-0348" num="0348">The DNA sequence and protein sequence for a novel Acetylglucosaminyltransferase-like gene or one of its splice forms thus derived is designated here as NOV7 (CuraGen Acc. No. SC20692369). <span style="display: none">[0348] </span> </div>
</li> <heading>Example 2 </heading>
<heading>Quantitative Expression Analysis of NOV7 in Various Cells and Tissues </heading>
<li> <para-num num="[0349]"> </para-num> <div class="description-line" id="P-0349" num="0349">The quantitative expression of various clones was assessed using microtiter plates containing RNA samples from a variety of normal and pathology-derived cells, cell lines and 25 tissues using real time quantitative PCR (RTQ PCR; TAQMAN®). RTQ PCR was performed on a Perkin-Elmer Biosystems ABI PRISM® 7700 Sequence Detection System. Various collections of samples are assembled on the plates, and referred to as Panel 1 (containing cells and cell lines from normal and cancer sources) and Panel 3 (containing samples derived from a wide variety of cancer sources). <span style="display: none">[0349] </span> </div>
</li> <li> <para-num num="[0350]"> </para-num> <div class="description-line" id="P-0350" num="0350">First, the RNA samples were normalized to constitutively expressed genes such as -actin and GAPDH. RNA (˜50 ng total or ˜1 ng polyA+) was converted to CDNA using the TAQMAN® Reverse Transcription Reagents Kit (PE Biosystems, Foster City, Calif.; Catalog No. N808-0234) and random hexamers according to the manufacturer's protocol. Reactions were performed in 20 ul and incubated for 30 min. at 48° C. cDNA (5 ul) was then transferred to a separate plate for the TAQMAN® reaction using β-actin and GAPDH TAQMAN® Assay Reagents (PE Biosystems; Catalog Nos. 4310881E and 4310884E, respectively) and TAQMAN(® universal PCR Master Mix (PE Biosystems; Catalog No. 4304447) according to the manufacturer's protocol. Reactions were performed in 25 ul using the following parameters: 2 min. at 50° C.; 10 min. at 95° C.; 15 sec. at 95° C./1 min. at 60° C. (40 cycles). Results were recorded as CT values (cycle at which a given sample crosses a threshold level of fluorescence) using a log scale, with the difference in RNA concentration between a given sample and the sample with the lowest CT value being represented as 2 to the power of delta CT. The percent relative expression is then obtained by taking the reciprocal of this RNA difference and multiplying by 100. The average CT values obtained for β-actin and GAPDH were used to normalize RNA samples. The RNA sample generating the highest CT value required no further diluting, while all other samples were diluted relative to this sample according to their β-actin/GAPDH average CT values. <span style="display: none">[0350] </span> </div>
</li> <li> <para-num num="[0351]"> </para-num> <div class="description-line" id="P-0351" num="0351">Normalized RNA (5 ul) was converted to cDNA and analyzed via TAQMAN® using One Step RT-PCR Master Mix Reagents (PE Biosystems; Catalog No. 4309169) and gene-specific primers according to the manufacturer's instructions. Probes and primers were designed for each assay according to Perkin Elmer Biosystem's Primer Express Software package (version I for Apple Computer's Macintosh Power PC) or a similar algorithm using the target sequence as input. Default settings were used for reaction conditions and the following parameters were set before selecting primers: primer concentration=250 nM, primer melting temperature (T<span style="display: none">[0351] </span> <sub>m</sub>) range=58°-60° C., primer optimal T<sub>m</sub>=59° C., maximum primer difference=2° C., probe does not have 5′ G, probe Tm must be 10° C. greater than primer Tm, amplicon size 75 bp to 100 bp. The probes and primers selected (see below) were synthesized by Synthegen (Houston, Tex., USA). Probes were double purified by HPLC to remove uncoupled dye and evaluated by mass spectroscopy to verify coupling of reporter and quencher dyes to the 5′ and 3′ ends of the probe, respectively. Their final concentrations were: forward and reverse primers, 900 nM each, and probe, 200 nM. </div>
</li> <li> <para-num num="[0352]"> </para-num> <div class="description-line" id="P-0352" num="0352">PCR conditions: Normalized RNA from each tissue and each cell line was spotted in each well of a 96 well PCR plate (Perkin Elmer Biosystems). PCR cocktails including two probes (a probe specific for the target clone and another gene-specific probe multiplexed with the target probe) were set up using 1× TaqMan™ PCR Master Mix for the PE Biosystems 7700, with 5 mM MgCl2, dNTPs (dA, G, C, U at 1:1:1:2 ratios), 0.25 U/ml AmpliTaq Gold™ (PE Biosystems), and 0.4 U/μl RNase inhibitor, and 0.25 U/μl reverse transcriptase. Reverse transcription was performed at 48° C. for 30 minutes followed by amplification/PCR cycles as follows: 95° C. 10 min, then 40 cycles of 95° C. for 15 seconds, 60° C. for 1 minute. <span style="display: none">[0352] </span> </div>
</li> <li> <para-num num="[0353]"> </para-num> <div class="description-line" id="P-0353" num="0353">In the results for Panel 1, the following abbreviations are used: <span style="display: none">[0353] </span> </div>
</li> <li> <para-num num="[0354]"> </para-num> <div class="description-line" id="P-0354" num="0354">ca.=carcinoma, <span style="display: none">[0354] </span> </div>
</li> <li> <para-num num="[0355]"> </para-num> <div class="description-line" id="P-0355" num="0355">*=established from metastasis, <span style="display: none">[0355] </span> </div>
</li> <li> <para-num num="[0356]"> </para-num> <div class="description-line" id="P-0356" num="0356">met=metastasis, <span style="display: none">[0356] </span> </div>
</li> <li> <para-num num="[0357]"> </para-num> <div class="description-line" id="P-0357" num="0357">s cell var=small cell variant, <span style="display: none">[0357] </span> </div>
</li> <li> <para-num num="[0358]"> </para-num> <div class="description-line" id="P-0358" num="0358">non-s=non-sm=non-small, <span style="display: none">[0358] </span> </div>
</li> <li> <para-num num="[0359]"> </para-num> <div class="description-line" id="P-0359" num="0359">squam=squamous, <span style="display: none">[0359] </span> </div>
</li> <li> <para-num num="[0360]"> </para-num> <div class="description-line" id="P-0360" num="0360">pl. eff=pl effusion=pleural effusion, <span style="display: none">[0360] </span> </div>
</li> <li> <para-num num="[0361]"> </para-num> <div class="description-line" id="P-0361" num="0361">glio=glioma, <span style="display: none">[0361] </span> </div>
</li> <li> <para-num num="[0362]"> </para-num> <div class="description-line" id="P-0362" num="0362">astro=astrocytoma, and <span style="display: none">[0362] </span> </div>
</li> <li> <para-num num="[0363]"> </para-num> <div class="description-line" id="P-0363" num="0363">neuro=neuroblastoma. <span style="display: none">[0363] </span> </div>
</li> <li> <para-num num="[0364]"> </para-num> <div class="description-line" id="P-0364" num="0364">The primer-probe set used to study clone 20692369 is Ag795, shown in Table 28 and the results of the TaqMan analysis for Panel 1 are shown in Table 29.  
<span style="display: none">[0364] </span> <tables id="TABLE-US-00028" num="28"> <patent-tables colsep="0" frame="none" pgwide="1" rowsep="0" tabstyle="monospace"> <table align="left" class="description-table" cols="1" colsep="0" rowsep="0" width="100%"> <thead> <tr class="description-tr"> <td align="center" class="description-td" colspan="1" nameend="1" namest="1">TABLE 28</td> </tr> </thead> <tbody><tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td align="center" class="description-td" colspan="1" nameend="1" namest="1" rowsep="1"> </td> </tr> <tr class="description-tr"> <td class="description-td">Primer-probe set Ag795.</td> </tr> </tbody></table> <table align="left" class="description-table" cols="5" colsep="0" rowsep="0" width="100%"> <tbody><tr class="description-tr"> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td">Start</td> </tr> <tr class="description-tr"> <td class="description-td">Primers</td> <td class="description-td">Sequences</td> <td class="description-td">TM, ° C.</td> <td class="description-td">Length</td> <td class="description-td">Position</td> </tr> <tr class="description-tr"> <td align="center" class="description-td" colspan="5" nameend="5" namest="1" rowsep="1"> </td> </tr> </tbody></table> <table align="left" class="description-table" cols="5" colsep="0" rowsep="0" width="100%"> <tbody><tr class="description-tr"> <td class="description-td">Forward</td> <td class="description-td">5′-TGGAGAACCAGAGTGACTGGTA-3′</td> <td class="description-td">59.8</td> <td class="description-td">22</td> <td class="description-td">597</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">(SEQ ID NO:54)</td> </tr> <tr class="description-tr"> <td class="description-td">Probe</td> <td class="description-td">FAM-5′-</td> <td class="description-td">70</td> <td class="description-td">25</td> <td class="description-td">626</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">AACCTCTGGAAGAACCACAGGCCCT-</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">3′-TAMRA (SEQ ID NO:55)</td> </tr> <tr class="description-tr"> <td class="description-td">Reverse</td> <td class="description-td">5′-AGCAGGATCACACCTGTGTTAA-3′</td> <td class="description-td">59.7</td> <td class="description-td">22</td> <td class="description-td">672</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">(SEQ ID NO:56)</td> </tr> <tr class="description-tr"> <td align="center" class="description-td" colspan="5" nameend="5" namest="1" rowsep="1"> </td> </tr> </tbody></table> </patent-tables> </tables>
</div>
</li> <li> <para-num num="[0365]"> </para-num> <div class="description-line" id="P-0365" num="0365"> <span style="display: none">[0365]</span> <tables id="TABLE-US-00029" num="29"> <patent-tables colsep="0" frame="none" pgwide="1" rowsep="0"> <table align="left" class="description-table" cols="1" colsep="0" rowsep="0" width="100%"> <thead> <tr class="description-tr"> <td align="center" class="description-td" colspan="1" nameend="1" namest="1">TABLE 29</td> </tr> </thead> <tbody><tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td align="center" class="description-td" colspan="1" nameend="1" namest="1" rowsep="1"> </td> </tr> <tr class="description-tr"> <td class="description-td">Real time expression analysis of clone 20692369 in cells of Panel 1.</td> </tr> </tbody></table> <table align="left" class="description-table" cols="6" colsep="0" rowsep="0" width="100%"> <tbody><tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">Rel.</td> <td class="description-td">Rel.</td> <td class="description-td"> </td> <td class="description-td">Rel.</td> <td class="description-td">Rel.</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">Expr.,</td> <td class="description-td">Expr.,</td> <td class="description-td"> </td> <td class="description-td">Expr.,</td> <td class="description-td">Expr.,</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">1.2tm</td> <td class="description-td">1.2tm</td> <td class="description-td"> </td> <td class="description-td">1.2tm</td> <td class="description-td">1.2tm1</td> </tr> <tr class="description-tr"> <td class="description-td">Tissue_Name</td> <td class="description-td">954f</td> <td class="description-td">1127f</td> <td class="description-td">Tissue_Name</td> <td class="description-td">954f</td> <td class="description-td">127f</td> </tr> <tr class="description-tr"> <td align="center" class="description-td" colspan="6" nameend="6" namest="1" rowsep="1"> </td> </tr> </tbody></table> <table align="left" class="description-table" cols="6" colsep="0" rowsep="0" width="100%"> <tbody><tr class="description-tr"> <td class="description-td">Endothelial cells</td> <td class="description-td">0.0</td> <td class="description-td">0.0</td> <td class="description-td">Renal ca.786-0</td> <td class="description-td">0.0</td> <td class="description-td">0.0</td> </tr> <tr class="description-tr"> <td class="description-td">Endothelial cells</td> <td class="description-td">0.3</td> <td class="description-td">1.2</td> <td class="description-td">Renal ca.A498</td> <td class="description-td">0.0</td> <td class="description-td">0.0</td> </tr> <tr class="description-tr"> <td class="description-td">(treated)</td> </tr> <tr class="description-tr"> <td class="description-td">Pancreas</td> <td class="description-td">43.8</td> <td class="description-td">18.4</td> <td class="description-td">Renal ca. RXF 393</td> <td class="description-td">0.0</td> <td class="description-td">0.0</td> </tr> <tr class="description-tr"> <td class="description-td">Pancreatic ca. CAPAN 2</td> <td class="description-td">0.2</td> <td class="description-td">0.3</td> <td class="description-td">Renal ca. ACHN</td> <td class="description-td">0.0</td> <td class="description-td">4.9</td> </tr> <tr class="description-tr"> <td class="description-td">Adrenal Gland (new</td> <td class="description-td">1.1</td> <td class="description-td">5.0</td> <td class="description-td">Renal ca. UO-31</td> <td class="description-td">0.0</td> <td class="description-td">0.0</td> </tr> <tr class="description-tr"> <td class="description-td">lot*)</td> </tr> <tr class="description-tr"> <td class="description-td">Thyroid</td> <td class="description-td">19.6</td> <td class="description-td">20.3</td> <td class="description-td">Renal ca. TK-10</td> <td class="description-td">0.0</td> <td class="description-td">0.2</td> </tr> <tr class="description-tr"> <td class="description-td">Salivary gland</td> <td class="description-td">9.2</td> <td class="description-td">27.7</td> <td class="description-td">Liver</td> <td class="description-td">0.6</td> <td class="description-td">1.4</td> </tr> <tr class="description-tr"> <td class="description-td">Pituitary gland</td> <td class="description-td">12.5</td> <td class="description-td">8.4</td> <td class="description-td">Liver (fetal)</td> <td class="description-td">0.6</td> <td class="description-td">7.3</td> </tr> <tr class="description-tr"> <td class="description-td">Brain (fetal)</td> <td class="description-td">0.0</td> <td class="description-td">0.0</td> <td class="description-td">Liver ca. (hepatoblast) HepG2</td> <td class="description-td">9.7</td> <td class="description-td">53.6</td> </tr> <tr class="description-tr"> <td class="description-td">Brain (whole)</td> <td class="description-td">0.0</td> <td class="description-td">0.0</td> <td class="description-td">Lung</td> <td class="description-td">1.9</td> <td class="description-td">5.2</td> </tr> <tr class="description-tr"> <td class="description-td">Brain (amygdala)</td> <td class="description-td">0.0</td> <td class="description-td">0.0</td> <td class="description-td">Lung (fetal)</td> <td class="description-td">2.8</td> <td class="description-td">4.8</td> </tr> <tr class="description-tr"> <td class="description-td">Brain (cerebellum)</td> <td class="description-td">0.0</td> <td class="description-td">0.0</td> <td class="description-td">Lung ca. (small cell) LX-1</td> <td class="description-td">67.8</td> <td class="description-td">59.9</td> </tr> <tr class="description-tr"> <td class="description-td">Brain (hippocampus)</td> <td class="description-td">0.2</td> <td class="description-td">0.6</td> <td class="description-td">Lung ca. (small cell) NCI-H69</td> <td class="description-td">2.5</td> <td class="description-td">3.2</td> </tr> <tr class="description-tr"> <td class="description-td">Brain (thalamus)</td> <td class="description-td">0.1</td> <td class="description-td">0.2</td> <td class="description-td">Lung ca. (s.cell var.) SHP-77</td> <td class="description-td">5.7</td> <td class="description-td">7.8</td> </tr> <tr class="description-tr"> <td class="description-td">Cerebral Cortex</td> <td class="description-td">0.0</td> <td class="description-td">0.0</td> <td class="description-td">Lung ca. (large cell) NCI-</td> <td class="description-td">0.1</td> <td class="description-td">0.2</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td">H460</td> </tr> <tr class="description-tr"> <td class="description-td">Spinal cord</td> <td class="description-td">0.1</td> <td class="description-td">0.3</td> <td class="description-td">Lung ca. (non-sm. cell) A549</td> <td class="description-td">0.2</td> <td class="description-td">2.7</td> </tr> <tr class="description-tr"> <td class="description-td">CNS ca. (glio/astro) U87-</td> <td class="description-td">0.0</td> <td class="description-td">0.0</td> <td class="description-td">Lung ca. (non-s.cell) NCI-</td> <td class="description-td">1.2</td> <td class="description-td">4.6</td> </tr> <tr class="description-tr"> <td class="description-td">MG</td> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td">H23</td> </tr> <tr class="description-tr"> <td class="description-td">CNS ca. (glio/astro) U-</td> <td class="description-td">0.0</td> <td class="description-td">0.0</td> <td class="description-td">Lung ca (non-s.cell) HOP-62</td> <td class="description-td">1.8</td> <td class="description-td">1.5</td> </tr> <tr class="description-tr"> <td class="description-td">118-MG</td> </tr> <tr class="description-tr"> <td class="description-td">CNS ca. (astro) SW1783</td> <td class="description-td">0.0</td> <td class="description-td">0.0</td> <td class="description-td">Lung ca. (non-s.cl) NCI-H522</td> <td class="description-td">2.3</td> <td class="description-td">3.2</td> </tr> <tr class="description-tr"> <td class="description-td">CNS ca.* (neuro; met)</td> <td class="description-td">7.4</td> <td class="description-td">7.9</td> <td class="description-td">Lung ca. (squam.) SW 900</td> <td class="description-td">1.7</td> <td class="description-td">3.7</td> </tr> <tr class="description-tr"> <td class="description-td">SK-N-AS</td> </tr> <tr class="description-tr"> <td class="description-td">CNS ca. (astro) SF-539</td> <td class="description-td">0.2</td> <td class="description-td">0.1</td> <td class="description-td">Lung ca. (squam.) NCI-H596</td> <td class="description-td">1.8</td> <td class="description-td">2.4</td> </tr> <tr class="description-tr"> <td class="description-td">CNS Ca. (astro) SNB-75</td> <td class="description-td">0.0</td> <td class="description-td">0.0</td> <td class="description-td">Mammary gland</td> <td class="description-td">7.1</td> <td class="description-td">5.7</td> </tr> <tr class="description-tr"> <td class="description-td">CNS ca. (glio) SNB-19</td> <td class="description-td">0.3</td> <td class="description-td">0.5</td> <td class="description-td">Breast ca.* (pl. effusion)</td> <td class="description-td">1.5</td> <td class="description-td">8.3</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td">MCF-7</td> </tr> <tr class="description-tr"> <td class="description-td">CNS Ca. (glio) U251</td> <td class="description-td">0.2</td> <td class="description-td">0.1</td> <td class="description-td">Breast ca.* (pl.ef) MDA-MB-</td> <td class="description-td">0.1</td> <td class="description-td">0.0</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td">231</td> </tr> <tr class="description-tr"> <td class="description-td">CNS Ca. (glio) SF-295</td> <td class="description-td">0.0</td> <td class="description-td">0.0</td> <td class="description-td">Breast ca.* (pl. effusion) T47D</td> <td class="description-td">16.6</td> <td class="description-td">19.9</td> </tr> <tr class="description-tr"> <td class="description-td">Heart</td> <td class="description-td">0.0</td> <td class="description-td">0.0</td> <td class="description-td">Breast ca. BT-549</td> <td class="description-td">0.0</td> <td class="description-td">0.0</td> </tr> <tr class="description-tr"> <td class="description-td">Skeletal Muscle (new</td> <td class="description-td">0.1</td> <td class="description-td">0.0</td> <td class="description-td">Breast ca. MDA-N</td> <td class="description-td">0.0</td> <td class="description-td">0.0</td> </tr> <tr class="description-tr"> <td class="description-td">lot*)</td> </tr> <tr class="description-tr"> <td class="description-td">Bone marrow</td> <td class="description-td">0.7</td> <td class="description-td">1.9</td> <td class="description-td">Ovary</td> <td class="description-td">0.7</td> <td class="description-td">3.5</td> </tr> <tr class="description-tr"> <td class="description-td">Thymus</td> <td class="description-td">1.9</td> <td class="description-td">4.8</td> <td class="description-td">Ovarian ca. OVCAR-3</td> <td class="description-td">15.8</td> <td class="description-td">17.3</td> </tr> <tr class="description-tr"> <td class="description-td">Spleen</td> <td class="description-td">2.0</td> <td class="description-td">3.2</td> <td class="description-td">Ovarian ca. OVCAR-4</td> <td class="description-td">4.4</td> <td class="description-td">13.7</td> </tr> <tr class="description-tr"> <td class="description-td">Lymph node</td> <td class="description-td">4.5</td> <td class="description-td">12.7</td> <td class="description-td">Ovarian ca. OVCAR-5</td> <td class="description-td">0.9</td> <td class="description-td">2.4</td> </tr> <tr class="description-tr"> <td class="description-td">Colorectal</td> <td class="description-td">0.0</td> <td class="description-td">0.3</td> <td class="description-td">Ovarian ca. OVCAR-8</td> <td class="description-td">1.7</td> <td class="description-td">8.6</td> </tr> <tr class="description-tr"> <td class="description-td">Stomach</td> <td class="description-td">5.7</td> <td class="description-td">24.0</td> <td class="description-td">Ovarian ca. IGROV-1</td> <td class="description-td">3.5</td> <td class="description-td">13.7</td> </tr> <tr class="description-tr"> <td class="description-td">Small intestine</td> <td class="description-td">1.1</td> <td class="description-td">1.0</td> <td class="description-td">Ovarian ca.* (ascites) SK-</td> <td class="description-td">0.0</td> <td class="description-td">0.2</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td">OV-3</td> </tr> <tr class="description-tr"> <td class="description-td">Colon ca. SW480</td> <td class="description-td">14.0</td> <td class="description-td">21.9</td> <td class="description-td">Uterus</td> <td class="description-td">0.4</td> <td class="description-td">2.0</td> </tr> <tr class="description-tr"> <td class="description-td">Colon ca.* (SW480</td> <td class="description-td">100.0</td> <td class="description-td">100.0</td> <td class="description-td">Placenta</td> <td class="description-td">33.2</td> <td class="description-td">76.3</td> </tr> <tr class="description-tr"> <td class="description-td">met)SW620</td> </tr> <tr class="description-tr"> <td class="description-td">Colon ca. HT29</td> <td class="description-td">3.0</td> <td class="description-td">5.3</td> <td class="description-td">Prostate</td> <td class="description-td">8.0</td> <td class="description-td">25.4</td> </tr> <tr class="description-tr"> <td class="description-td">Colon ca. HCT-116</td> <td class="description-td">31.6</td> <td class="description-td">68.8</td> <td class="description-td">Prostate ca.* (bone met) PC-3</td> <td class="description-td">2.4</td> <td class="description-td">11.0</td> </tr> <tr class="description-tr"> <td class="description-td">Colon ca. CaCo-2</td> <td class="description-td">15.3</td> <td class="description-td">36.4</td> <td class="description-td">Testis</td> <td class="description-td">1.5</td> <td class="description-td">2.1</td> </tr> <tr class="description-tr"> <td class="description-td">83219 CC Well to Mod</td> <td class="description-td">0.4</td> <td class="description-td">3.3</td> <td class="description-td">Melanoma Hs688(A).T</td> <td class="description-td">0.0</td> <td class="description-td">0.0</td> </tr> <tr class="description-tr"> <td class="description-td">Diff (ODO3866)</td> </tr> <tr class="description-tr"> <td class="description-td">Colon ca. HCC-2998</td> <td class="description-td">41.5</td> <td class="description-td">47.6</td> <td class="description-td">Melanoma* (met) Hs688(B).T</td> <td class="description-td">0.0</td> <td class="description-td">0.0</td> </tr> <tr class="description-tr"> <td class="description-td">Gastric ca.* (liver met)</td> <td class="description-td">29.3</td> <td class="description-td">29.5</td> <td class="description-td">Melanoma UACC-62</td> <td class="description-td">0.1</td> <td class="description-td">0.0</td> </tr> <tr class="description-tr"> <td class="description-td">NCI-N87</td> </tr> <tr class="description-tr"> <td class="description-td">Bladder</td> <td class="description-td">3.7</td> <td class="description-td">6.3</td> <td class="description-td">Melanoma M14</td> <td class="description-td">0.2</td> <td class="description-td">1.2</td> </tr> <tr class="description-tr"> <td class="description-td">Trachea</td> <td class="description-td">5.2</td> <td class="description-td">15.4</td> <td class="description-td">Melanoma LOX IMVI</td> <td class="description-td">0.0</td> <td class="description-td">0.0</td> </tr> <tr class="description-tr"> <td class="description-td">Kidney</td> <td class="description-td">11.5</td> <td class="description-td">29.1</td> <td class="description-td">Melanoma* (met)SK-MEL-5</td> <td class="description-td">0.3</td> <td class="description-td">0.3</td> </tr> <tr class="description-tr"> <td class="description-td">Kidney (fetal)</td> <td class="description-td">0.0</td> <td class="description-td">18.4</td> <td class="description-td">Adipose</td> <td class="description-td">0.0</td> <td class="description-td">0.2</td> </tr> <tr class="description-tr"> <td align="center" class="description-td" colspan="6" nameend="6" namest="1" rowsep="1"> </td> </tr> </tbody></table> </patent-tables> </tables> </div>
</li> <li> <para-num num="[0366]"> </para-num> <div class="description-line" id="P-0366" num="0366">The results in Table 29 show that clone 20692369 is expressed to high degree in several colon cancer samples but not at all in normal Colorectal tissue. It is also highly expressed differentially in a lung small cell cancer sample, in liver cancer, and moderately in certain other cancers. It also expressed in certain normal tissues not related to these cancer tissues. <span style="display: none">[0366] </span> </div>
</li> <li> <para-num num="[0367]"> </para-num> <div class="description-line" id="P-0367" num="0367">The results for Panel 3 are shown in Table 30.  
<span style="display: none">[0367] </span> <tables id="TABLE-US-00030" num="30"> <patent-tables colsep="0" frame="none" pgwide="1" rowsep="0"> <table align="left" class="description-table" cols="1" colsep="0" rowsep="0" width="100%"> <thead> <tr class="description-tr"> <td align="center" class="description-td" colspan="1" nameend="1" namest="1">TABLE 30</td> </tr> </thead> <tbody><tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td align="center" class="description-td" colspan="1" nameend="1" namest="1" rowsep="1"> </td> </tr> <tr class="description-tr"> <td class="description-td">Real time expression analysis of clone 20692369 in cells of Panel 3.</td> </tr> </tbody></table> <table align="left" class="description-table" cols="4" colsep="0" rowsep="0" width="100%"> <tbody><tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">Rel.</td> <td class="description-td"> </td> <td class="description-td">Rel.</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">Expr.</td> <td class="description-td"> </td> <td class="description-td">Expr.</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">%,</td> <td class="description-td"> </td> <td class="description-td">%,</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">3Dtm</td> <td class="description-td"> </td> <td class="description-td">3Dtm3</td> </tr> <tr class="description-tr"> <td class="description-td">Tissue_Name</td> <td class="description-td">3745f</td> <td class="description-td">Tissue_Name</td> <td class="description-td">745f</td> </tr> <tr class="description-tr"> <td align="center" class="description-td" colspan="4" nameend="4" namest="1" rowsep="1"> </td> </tr> </tbody></table> <table align="left" class="description-table" cols="4" colsep="0" rowsep="0" width="100%"> <tbody><tr class="description-tr"> <td class="description-td">94905_Daoy_Medulloblastoma/</td> <td class="description-td">0.3</td> <td class="description-td">94954_Ca Ski_Cervical epidermoid</td> <td class="description-td">3.4</td> </tr> <tr class="description-tr"> <td class="description-td">Cerebellum_sscDNA</td> <td class="description-td"> </td> <td class="description-td">carcinoma (metastasis)_sscDNA</td> </tr> <tr class="description-tr"> <td class="description-td">94906_TE671_Medulloblastom/</td> <td class="description-td">0.2</td> <td class="description-td">94955_ES-2_Ovarian clear cell</td> <td class="description-td">0.0</td> </tr> <tr class="description-tr"> <td class="description-td">Cerebellum_sscDNA</td> <td class="description-td"> </td> <td class="description-td">carcinoma_sscDNA</td> </tr> <tr class="description-tr"> <td class="description-td">94907_D283</td> <td class="description-td">5.6</td> <td class="description-td">94957_Ramos/6h stim_″; Stimulated</td> <td class="description-td">0.2</td> </tr> <tr class="description-tr"> <td class="description-td">Med_Medulloblastoma/Cerebellu</td> <td class="description-td"> </td> <td class="description-td">with PMA/ionomycin 6h_sscDNA</td> </tr> <tr class="description-tr"> <td class="description-td">m_sscDNA</td> </tr> <tr class="description-tr"> <td class="description-td">94908_PFSK-1_Primitive</td> <td class="description-td">0.5</td> <td class="description-td">94958_Ramos/14h stim_″;</td> <td class="description-td">0.2</td> </tr> <tr class="description-tr"> <td class="description-td">Neuroectodermal/Cerebellum_ssc</td> <td class="description-td"> </td> <td class="description-td">Stimulated with PMA/ionomycin</td> </tr> <tr class="description-tr"> <td class="description-td">DNA</td> <td class="description-td"> </td> <td class="description-td">14h_sscDNA</td> </tr> <tr class="description-tr"> <td class="description-td">94909_XF-498_CNS_sscDNA</td> <td class="description-td">34.2</td> <td class="description-td">94962_MEG-0l_Chronic</td> <td class="description-td">0.0</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td">myelogenous leukemia</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td">(megokaryoblast)_sscDNA</td> </tr> <tr class="description-tr"> <td class="description-td">94910_SNB </td> <td class="description-td">0.1</td> <td class="description-td">94963_Raji_Burkitt's</td> <td class="description-td">0.0</td> </tr> <tr class="description-tr"> <td class="description-td">78_CNS/glioma_sscDNA</td> <td class="description-td"> </td> <td class="description-td">lymphoma_sscDNA</td> </tr> <tr class="description-tr"> <td class="description-td">94911_SF-</td> <td class="description-td">0.3</td> <td class="description-td">94964_Daudi_Burkitt's</td> <td class="description-td">0.0</td> </tr> <tr class="description-tr"> <td class="description-td">268_CNS/glioblastoma_sscDNA</td> <td class="description-td"> </td> <td class="description-td">lymphoma_sscDNA</td> </tr> <tr class="description-tr"> <td class="description-td">94912_T98G_Glioblastoma_ssc</td> <td class="description-td">0.0</td> <td class="description-td">94965_U266_B-cell</td> <td class="description-td">0.1</td> </tr> <tr class="description-tr"> <td class="description-td">DNA</td> <td class="description-td"> </td> <td class="description-td">plasmacytoma/myeloma_sscDNA</td> </tr> <tr class="description-tr"> <td class="description-td">96776_SK-N-SH_Neuroblastoma</td> <td class="description-td">1.4</td> <td class="description-td">94968_CA46_Burkitt's</td> <td class="description-td">0.1</td> </tr> <tr class="description-tr"> <td class="description-td">(metastasis)_sscDNA</td> <td class="description-td"> </td> <td class="description-td">lymphoma_sscDNA</td> </tr> <tr class="description-tr"> <td class="description-td">94913_SF-</td> <td class="description-td">0.0</td> <td class="description-td">94970_RL_non-Hodgkin's B-cell</td> <td class="description-td">0.0</td> </tr> <tr class="description-tr"> <td class="description-td">295_CNS/glioblastoma_sscDNA</td> <td class="description-td"> </td> <td class="description-td">lymphoma_sscDNA</td> </tr> <tr class="description-tr"> <td class="description-td">94914_Cerebellum_sscDNA</td> <td class="description-td">0.3</td> <td class="description-td">94972_JM1_pre-B-cell</td> <td class="description-td">0.1</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td">lymphoma/leukemia_sscDNA</td> </tr> <tr class="description-tr"> <td class="description-td">96777_Cerebellum_sscDNA</td> <td class="description-td">0.1</td> <td class="description-td">94973_Jurkat_T cell</td> <td class="description-td">0.3</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td">leukemia_sscDNA</td> </tr> <tr class="description-tr"> <td class="description-td">94916_NCI-</td> <td class="description-td">8.9</td> <td class="description-td">94974_TF-</td> <td class="description-td">0.0</td> </tr> <tr class="description-tr"> <td class="description-td">H292_Mucoepidermoid lung</td> <td class="description-td"> </td> <td class="description-td">1_Erythroleukemia_sscDNA</td> </tr> <tr class="description-tr"> <td class="description-td">carcinoma_sscDNA</td> </tr> <tr class="description-tr"> <td class="description-td">94917_DMS-114_Small cell lung</td> <td class="description-td">0.6</td> <td class="description-td">94975_HUT 78_T-cell</td> <td class="description-td">0.1</td> </tr> <tr class="description-tr"> <td class="description-td">cancer_sscDNA</td> <td class="description-td"> </td> <td class="description-td">lymphoma_sscDNA</td> </tr> <tr class="description-tr"> <td class="description-td">94918_DMS-79_Small cell lung</td> <td class="description-td">57.4</td> <td class="description-td">94977_U937_Histiocytic</td> <td class="description-td">0.0</td> </tr> <tr class="description-tr"> <td class="description-td">cancer/neuroendocrine_sscDNA</td> <td class="description-td"> </td> <td class="description-td">lymphoma_sscDNA</td> </tr> <tr class="description-tr"> <td class="description-td">94919_NCI-H146_Small cell</td> <td class="description-td">25.5</td> <td class="description-td">94980_KU-812_Myelogenous</td> <td class="description-td">0.0</td> </tr> <tr class="description-tr"> <td class="description-td">lung cancer/neuroendocrine</td> <td class="description-td"> </td> <td class="description-td">leukemia_sscDNA</td> </tr> <tr class="description-tr"> <td class="description-td">sscDNA</td> </tr> <tr class="description-tr"> <td class="description-td">94920_NCI-H526_Small cell</td> <td class="description-td">24.5</td> <td class="description-td">94981_769-P_Clear cell renal</td> <td class="description-td">0.1</td> </tr> <tr class="description-tr"> <td class="description-td">lung cancer/neuroendocrine</td> <td class="description-td"> </td> <td class="description-td">carcinoma_sscDNA</td> </tr> <tr class="description-tr"> <td class="description-td">sscDNA</td> </tr> <tr class="description-tr"> <td class="description-td">94921_NCI-N417_Small cell</td> <td class="description-td">0.0</td> <td class="description-td">94983_Caki-2_Clear cell renal</td> <td class="description-td">0.4</td> </tr> <tr class="description-tr"> <td class="description-td">lung cancer/neuroendocrine_</td> <td class="description-td"> </td> <td class="description-td">carcinoma_sscDNA</td> </tr> <tr class="description-tr"> <td class="description-td">sscDNA</td> </tr> <tr class="description-tr"> <td class="description-td">94923_NCI-H82_Small cell lung</td> <td class="description-td">4.7</td> <td class="description-td">94984_SW 839_Clear cell renal</td> <td class="description-td">0.0</td> </tr> <tr class="description-tr"> <td class="description-td">cancer/neuroendocrine_sscDNA</td> <td class="description-td"> </td> <td class="description-td">carcinoma_sscDNA</td> </tr> <tr class="description-tr"> <td class="description-td">94924_NCI-H157_Squamous</td> <td class="description-td">0.4</td> <td class="description-td">94986_G401_Wilms'</td> <td class="description-td">0.0</td> </tr> <tr class="description-tr"> <td class="description-td">cell lung cancer (metastasis)_</td> <td class="description-td"> </td> <td class="description-td">tumor_sscDNA</td> </tr> <tr class="description-tr"> <td class="description-td">sscDNA</td> </tr> <tr class="description-tr"> <td class="description-td">94925_NCI-H1155_Large cell</td> <td class="description-td">1.7</td> <td class="description-td">94987_Hs766T_Pancreatic</td> <td class="description-td">0.1</td> </tr> <tr class="description-tr"> <td class="description-td">lung cancer/neuroendocrine_</td> <td class="description-td"> </td> <td class="description-td">carcinoma (LN metastasis)_sscDNA</td> </tr> <tr class="description-tr"> <td class="description-td">sscDNA</td> </tr> <tr class="description-tr"> <td class="description-td">94926_NCI-H1299_Large cell</td> <td class="description-td">2.2</td> <td class="description-td">94988_CAPAN-1_Pancreatic</td> <td class="description-td">8.4</td> </tr> <tr class="description-tr"> <td class="description-td">lung cancer/neuroendocrine_</td> <td class="description-td"> </td> <td class="description-td">adenocarcinoma (liver</td> </tr> <tr class="description-tr"> <td class="description-td">sscDNA</td> <td class="description-td"> </td> <td class="description-td">metastasis)_sscDNA</td> </tr> <tr class="description-tr"> <td class="description-td">94927_NCI-H727_Lung</td> <td class="description-td">14.9</td> <td class="description-td">94989_SU86.86_Pancreatic</td> <td class="description-td">10.8</td> </tr> <tr class="description-tr"> <td class="description-td">carcinoid_sscDNA</td> <td class="description-td"> </td> <td class="description-td">carcinoma (liver</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td">metastasis)_sscDNA</td> </tr> <tr class="description-tr"> <td class="description-td">94928_NCI-UMC-11_Lung</td> <td class="description-td">100.0</td> <td class="description-td">94990_BxPC-3_Pancreatic</td> <td class="description-td">1.1</td> </tr> <tr class="description-tr"> <td class="description-td">carcinoid_sscDNA</td> <td class="description-td"> </td> <td class="description-td">adenocarcinoma_sscDNA</td> </tr> <tr class="description-tr"> <td class="description-td">94929_LX-1_Small cell lung</td> <td class="description-td">28.3</td> <td class="description-td">94991_HPAC_Pancreatic</td> <td class="description-td">1.0</td> </tr> <tr class="description-tr"> <td class="description-td">cancer_sscDNA</td> <td class="description-td"> </td> <td class="description-td">adenocarcinoma sscDNA</td> </tr> <tr class="description-tr"> <td class="description-td">94930_Colo-205_Colon</td> <td class="description-td">0.4</td> <td class="description-td">94992_MIA PaCa-2_Pancreatic</td> <td class="description-td">0.0</td> </tr> <tr class="description-tr"> <td class="description-td">cancer_sscDNA</td> <td class="description-td"> </td> <td class="description-td">carcinoma_sscDNA</td> </tr> <tr class="description-tr"> <td class="description-td">94931_KM12_Colon</td> <td class="description-td">18.1</td> <td class="description-td">94993_CFPAC-1_Pancreatic ductal</td> <td class="description-td">5.2</td> </tr> <tr class="description-tr"> <td class="description-td">cancer_sscDNA</td> <td class="description-td"> </td> <td class="description-td">adenocarcinoma_sscDNA</td> </tr> <tr class="description-tr"> <td class="description-td">94932_KM2OL2_Colon</td> <td class="description-td">1.4</td> <td class="description-td">94994_PANC-1_Pancreatic</td> <td class="description-td">0.3</td> </tr> <tr class="description-tr"> <td class="description-td">cancer_sscDNA</td> <td class="description-td"> </td> <td class="description-td">epithelioid ductal</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td">carcinoma_sscDNA</td> </tr> <tr class="description-tr"> <td class="description-td">94933_NCI-H716_Colon</td> <td class="description-td">0.4</td> <td class="description-td">94996_T24_Bladder carcinma</td> <td class="description-td">0.1</td> </tr> <tr class="description-tr"> <td class="description-td">cancer sscDNA</td> <td class="description-td"> </td> <td class="description-td">(transitional cell)_sscDNA</td> </tr> <tr class="description-tr"> <td class="description-td">94935_SW-48_Colon</td> <td class="description-td">37.1</td> <td class="description-td">94997_5637_Bladder</td> <td class="description-td">0.1</td> </tr> <tr class="description-tr"> <td class="description-td">adenocarcinoma_sscDNA</td> <td class="description-td"> </td> <td class="description-td">carcinoma_sscDNA</td> </tr> <tr class="description-tr"> <td class="description-td">94936_SW1116_Colon</td> <td class="description-td">11.0</td> <td class="description-td">94998_HT-1197_Bladder</td> <td class="description-td">3.7</td> </tr> <tr class="description-tr"> <td class="description-td">adenocarcinoma sscDNA</td> <td class="description-td"> </td> <td class="description-td">carcinoma_sscDNA</td> </tr> <tr class="description-tr"> <td class="description-td">94937_LS 174T_Colon</td> <td class="description-td">12.8</td> <td class="description-td">94999_UM-UC-3_Bladder carcinma</td> <td class="description-td">0.0</td> </tr> <tr class="description-tr"> <td class="description-td">adenocarcinoma_sscDNA</td> <td class="description-td"> </td> <td class="description-td">(transitional cell)_sscDNA</td> </tr> <tr class="description-tr"> <td class="description-td">94938_SW-948_Colon</td> <td class="description-td">3.8</td> <td class="description-td">95000_A204_Rhabdomyosarcoma_s</td> <td class="description-td">1.8</td> </tr> <tr class="description-tr"> <td class="description-td">adenocarcinoma_sscDNA</td> <td class="description-td"> </td> <td class="description-td">scDNA</td> </tr> <tr class="description-tr"> <td class="description-td">94939_SW-480_Colon</td> <td class="description-td">17.4</td> <td class="description-td">95001_HT-</td> <td class="description-td">0.0</td> </tr> <tr class="description-tr"> <td class="description-td">adenocarcinoma_sscDNA</td> <td class="description-td"> </td> <td class="description-td">1080_Fibrosarcoma_sscDNA</td> </tr> <tr class="description-tr"> <td class="description-td">94940_NCI-SNU-5_Gastric</td> <td class="description-td">15.6</td> <td class="description-td">95002_MG-63_Osteosarcoma</td> <td class="description-td">0.0</td> </tr> <tr class="description-tr"> <td class="description-td">carcinoma_sscDNA</td> <td class="description-td"> </td> <td class="description-td">(bone)_sscDNA</td> </tr> <tr class="description-tr"> <td class="description-td">94941_KATO III_Gastric 0.5</td> <td class="description-td"> </td> <td class="description-td">95003_SK-LMS-</td> <td class="description-td">0.3</td> </tr> <tr class="description-tr"> <td class="description-td">carcinoma_sscDNA</td> <td class="description-td"> </td> <td class="description-td">1_Leiomyosarcoma</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td">(vulva)_sscDNA</td> </tr> <tr class="description-tr"> <td class="description-td">94943_NCI-SNU-16_Gastric</td> <td class="description-td">0.0</td> <td class="description-td">95004_SJRH30_Rhabdomyosarcom</td> <td class="description-td">0.0</td> </tr> <tr class="description-tr"> <td class="description-td">carcinoma_sscDNA</td> <td class="description-td"> </td> <td class="description-td">a (met to bone marrow)_sscDNA</td> </tr> <tr class="description-tr"> <td class="description-td">94944_NCI-SNU-1_Gastric</td> <td class="description-td">0.2</td> <td class="description-td">95005_A431_Epidermoid</td> <td class="description-td">1.5</td> </tr> <tr class="description-tr"> <td class="description-td">carcinoma_sscDNA</td> <td class="description-td"> </td> <td class="description-td">carcinoma_sscDNA</td> </tr> <tr class="description-tr"> <td class="description-td">94946_RF-1_Gastric</td> <td class="description-td">0.2</td> <td class="description-td">95007_WM266-</td> <td class="description-td">0.0</td> </tr> <tr class="description-tr"> <td class="description-td">adenocarcinoma_sscDNA</td> <td class="description-td"> </td> <td class="description-td">4_Melanoma_sscDNA</td> </tr> <tr class="description-tr"> <td class="description-td">94947_RF-48_Gastric</td> <td class="description-td">0.3</td> <td class="description-td">95010_DU 145_Prostate carcinoma</td> <td class="description-td">0.0</td> </tr> <tr class="description-tr"> <td class="description-td">adenocarcinoma_sscDNA</td> <td class="description-td"> </td> <td class="description-td">(brain metastasis)_sscDNA</td> </tr> <tr class="description-tr"> <td class="description-td">96778_MKN-45_Gastric</td> <td class="description-td">38.4</td> <td class="description-td">95012_MDA-MB-468_Breast</td> <td class="description-td">1.8</td> </tr> <tr class="description-tr"> <td class="description-td">carcinoma_sscDNA</td> <td class="description-td"> </td> <td class="description-td">adenocarcinoma_sscDNA</td> </tr> <tr class="description-tr"> <td class="description-td">94949_NCI-N87_Gastric</td> <td class="description-td">15.0</td> <td class="description-td">95013_SCC-4_Squamous cell</td> <td class="description-td">0.0</td> </tr> <tr class="description-tr"> <td class="description-td">carcinoma_sscDNA</td> <td class="description-td"> </td> <td class="description-td">carcinoma of tongue_sscDNA</td> </tr> <tr class="description-tr"> <td class="description-td">94951_OVCAR-5_Ovarian</td> <td class="description-td">1.9</td> <td class="description-td">95014_SCC-9_Squamous cell</td> <td class="description-td">0.0</td> </tr> <tr class="description-tr"> <td class="description-td">carcinoma_sscDNA</td> <td class="description-td"> </td> <td class="description-td">carcinoma of tongue_sscDNA</td> </tr> <tr class="description-tr"> <td class="description-td">94952_RL95-2_Uterine</td> <td class="description-td">6.5</td> <td class="description-td">95015_SCC-15_Squamous cell</td> <td class="description-td">0.0</td> </tr> <tr class="description-tr"> <td class="description-td">carcinoma_sscDNA</td> <td class="description-td"> </td> <td class="description-td">carcinoma of tongue_sscDNA</td> </tr> <tr class="description-tr"> <td class="description-td">94953_HelaS3_Cervical</td> <td class="description-td">0.0</td> <td class="description-td">95017_CAL 27_Squamous cell</td> <td class="description-td">2.8</td> </tr> <tr class="description-tr"> <td class="description-td">adenocarcinoma_sscDNA</td> <td class="description-td"> </td> <td class="description-td">carcinoma of tongue_sscDNA</td> </tr> <tr class="description-tr"> <td align="center" class="description-td" colspan="4" nameend="4" namest="1" rowsep="1"> </td> </tr> </tbody></table> </patent-tables> </tables>
</div>
</li> <li> <para-num num="[0368]"> </para-num> <div class="description-line" id="P-0368" num="0368">The results shown in Table 30 confirm and extend the results from Table 29, in that several lung, colon and gastric cancer tissues show high expression of clone 20692369. <span style="display: none">[0368] </span> </div>
</li> <li> <para-num num="[0369]"> </para-num> <div class="description-line" id="P-0369" num="0369">The results in Tables 29 and 30 establish uses for clone 20692369 as a probe for various cancers, and for its gene product as a potential therapeutic target for a specific antibody, for use in treatment of such cancers. <span style="display: none">[0369] </span> </div>
</li> <heading>Example 3 </heading>
<heading>Serial Analysis of Gene Expression of NOV7 in Various Cells and Tissues </heading>
<li> <para-num num="[0370]"> </para-num> <div class="description-line" id="P-0370" num="0370">Serial Analysis of Gene Expression (SAGE) uses a SAGE tag, a short polynucleotide sequence, generally under about 20 nucleotides, that occurs in a certain position in messenger RNA. The SAGE tag can be used to identify the corresponding transcript and gene from which it was transcribed. SAGE analysis begins with providing complementary deoxyribonucleic acid (cDNA) from tumor cell-line derived libraries. cDNAs can be linked to primer sites. Sequence tags are then created, for example, using the appropriate primers to amplify the DNA. By measuring the differences in these tags between the cDNA libraries, sequences which are aberrantly expressed in the tumor cell lines can be identified. SAGE was performed on a NOV7 nucleic acid, as shown in FIG. 2. <span style="display: none">[0370] </span> </div>
</li> <heading>Example 4 </heading>
<heading>Molecular Cloning of a Mature Form of Clone 28804279.0.7 </heading>
<li> <para-num num="[0371]"> </para-num> <div class="description-line" id="P-0371" num="0371">Oligonucleotide primers were designed to amplify a DNA segment coding for a mature form of the 28804279.0.7 protein from residues 37 to 94 of the protein sequence disclosed in Table 6 (SEQ ID NO:6). The forward primer includes an in frame BamHI restriction site and the reverse primer contains an in frame XhoI restriction site. The sequences of the primers are the following:  
<span style="display: none">[0371] </span> <tables id="TABLE-US-00031" num="31"> <patent-tables colsep="0" frame="none" pgwide="1" rowsep="0" tabstyle="monospace"> <table align="left" class="description-table" cols="4" colsep="0" rowsep="0" width="100%"> <tbody><tr class="description-tr"> <td class="description-td"> </td> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">    28804279.0.7 Forward:</td> <td class="description-td"> </td> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">GGATCCGAGGACTCTGGTTGGTGTGGGCCTGTGTGC</td> <td class="description-td">(SEQ ID NO: 57)</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">    28804279.0.7 Reverse:</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">CTCGAGGACATCTTGCAAACCCTGTGCTGTGATGG</td> <td class="description-td">(SEQ ID NO: 58)</td> </tr> </tbody></table> </patent-tables> </tables>
</div>
</li> <li> <para-num num="[0372]"> </para-num> <div class="description-line" id="P-0372" num="0372">PCR reactions were set up using 5 ng human fetal kidney cDNA template, 1 microM of each of the 28804279.0.7 Forward and 28804279.0.7 Reverse primers, 5 micromoles dNTP (Clontech Laboratories, Palo Alto Calif.) and 1 microliter of 50xAdvantage-HF 2 polymerase (Clontech Laboratories) in a 50 microliter sample volume. The following reaction conditions were used:  
<span style="display: none">[0372] </span> <tables id="TABLE-US-00032" num="32"> <patent-tables colsep="0" frame="none" rowsep="0"> <table align="left" class="description-table" cols="2" colsep="0" rowsep="0" width="100%"> <thead> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td align="center" class="description-td" colspan="2" nameend="2" namest="1" rowsep="1"> </td> </tr> </thead> <tbody><tr class="description-tr"> <td class="description-td">a)</td> <td class="description-td">96° C. 3 minutes</td> </tr> <tr class="description-tr"> <td class="description-td">b)</td> <td class="description-td">96° C. 30 seconds denaturation</td> </tr> <tr class="description-tr"> <td class="description-td">c)</td> <td class="description-td">70° C. 30 seconds, primer annealing. This temperature was gradually</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">decreased by 1° C./cycle</td> </tr> <tr class="description-tr"> <td class="description-td">d)</td> <td class="description-td">72° C. 1 minute extension.</td> </tr> </tbody></table> <table align="left" class="description-table" cols="1" colsep="0" rowsep="0" width="100%"> <tbody><tr class="description-tr"> <td class="description-td">Repeat steps b-d 10 times</td> </tr> </tbody></table> <table align="left" class="description-table" cols="2" colsep="0" rowsep="0" width="100%"> <tbody><tr class="description-tr"> <td class="description-td">e)</td> <td class="description-td">96° C. 30 seconds denaturation</td> </tr> <tr class="description-tr"> <td class="description-td">f)</td> <td class="description-td">60° C. 30 seconds annealing</td> </tr> <tr class="description-tr"> <td class="description-td">g)</td> <td class="description-td">72° C. 1 minute extension</td> </tr> </tbody></table> <table align="left" class="description-table" cols="1" colsep="0" rowsep="0" width="100%"> <tbody><tr class="description-tr"> <td class="description-td">Repeat steps e-g 25 times</td> </tr> </tbody></table> <table align="left" class="description-table" cols="2" colsep="0" rowsep="0" width="100%"> <tbody><tr class="description-tr"> <td class="description-td">h)</td> <td class="description-td">72° C. 5 minutes final extension</td> </tr> <tr class="description-tr"> <td align="center" class="description-td" colspan="2" nameend="2" namest="1" rowsep="1"> </td> </tr> </tbody></table> </patent-tables> </tables>
</div>
</li> <li> <para-num num="[0373]"> </para-num> <div class="description-line" id="P-0373" num="0373">A single amplified product having the expected size of approximately 160 bp was detected by agarose gel electrophoresis. The product was isolated by QuiaX (QIAGEN Inc, Valencia Calif.) in a final volume of 20 microliters. <span style="display: none">[0373] </span> </div>
</li> <li> <para-num num="[0374]"> </para-num> <div class="description-line" id="P-0374" num="0374">The isolated product was ligated into the pCR2.1 vector (Invitrogen, Carlsbad, Calif.) and sequenced. The nucleotide sequence was determined to be 100% identical to the corresponding portion of clone 28804279.0.7 (bases 138-257 of the sequence shown in Table 6 (SEQ ID NO:6). The construct is called pCR2.1-28804279.0.7-S371-2B. <span style="display: none">[0374] </span> </div>
</li> <heading>Example 5 </heading>
<heading>Molecular Cloning of a Mature Form of 20692369 EXT </heading>
<li> <para-num num="[0375]"> </para-num> <div class="description-line" id="P-0375" num="0375">Oligonucleotide primers were designed to amplify a DNA segment coding for an extracellular portion of 20692369_EXT (Table 23; SEQ ID NO:14). The forward primer includes an in frame BglI restriction site and the reverse primer contains an in frame XhoI restriction site. The sequences of the primers are the following:  
<span style="display: none">[0375] </span> <tables id="TABLE-US-00033" num="33"> <patent-tables colsep="0" frame="none" pgwide="1" rowsep="0" tabstyle="monospace"> <table align="left" class="description-table" cols="3" colsep="0" rowsep="0" width="100%"> <tbody><tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">20692369 MAT-FORW:</td> <td class="description-td">+TL,25</td> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">AGATCTCGGGAGGCTGCGGAGAGCCGCCGCCCTCGACG</td> <td class="description-td">(SEQ ID NO: 59)</td> </tr> <tr class="description-tr"> <td class="description-td">20692369 REV:</td> </tr> <tr class="description-tr"> <td class="description-td">CTCGAGGCCTCGGGCAGGGCTCTGGGGCTGCTGCAGG</td> <td class="description-td">(SEQ ID NO: 60)</td> </tr> </tbody></table> </patent-tables> </tables>
</div>
</li> <li> <para-num num="[0376]"> </para-num> <div class="description-line" id="P-0376" num="0376"> <span style="display: none">[0376]</span> </div>
</li> <li> <para-num num="[0377]"> </para-num> <div class="description-line" id="P-0377" num="0377">PCR reactions were set up using 5 ng of a mixture of human cDNA templates representing 30 adult tissues, 1 microM of each of the 28804279.0.7 MAT-FORW and 28804279.0.7 REV primers, 5 micromoles dNTP (Clontech Laboratories, Palo Alto Calif.) and 1 microliter of 5oxAdvantage-HF 2 polymerase (Clontech Laboratories) in 50 microliter volume. The following reaction conditions were used:  
<span style="display: none">[0377] </span> <tables id="TABLE-US-00034" num="34"> <patent-tables colsep="0" frame="none" rowsep="0"> <table align="left" class="description-table" cols="2" colsep="0" rowsep="0" width="100%"> <thead> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td align="center" class="description-td" colspan="2" nameend="2" namest="1" rowsep="1"> </td> </tr> </thead> <tbody><tr class="description-tr"> <td class="description-td">a)</td> <td class="description-td">96° C. 3 minutes</td> </tr> <tr class="description-tr"> <td class="description-td">b)</td> <td class="description-td">96° C. 30 seconds denaturation</td> </tr> <tr class="description-tr"> <td class="description-td">c)</td> <td class="description-td">70° C. 30 seconds, primer annealing. This temperature was gradually</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">decreased by 1° C./cycle</td> </tr> <tr class="description-tr"> <td class="description-td">d)</td> <td class="description-td">72° C. 3 minute extension.</td> </tr> </tbody></table> <table align="left" class="description-table" cols="1" colsep="0" rowsep="0" width="100%"> <tbody><tr class="description-tr"> <td class="description-td">Repeat steps b-d 10 times</td> </tr> </tbody></table> <table align="left" class="description-table" cols="2" colsep="0" rowsep="0" width="100%"> <tbody><tr class="description-tr"> <td class="description-td">e)</td> <td class="description-td">96° C. 30 seconds denaturation</td> </tr> <tr class="description-tr"> <td class="description-td">f)</td> <td class="description-td">60° C. 30 seconds annealing</td> </tr> <tr class="description-tr"> <td class="description-td">g)</td> <td class="description-td">72° C. 3 minute extension</td> </tr> </tbody></table> <table align="left" class="description-table" cols="1" colsep="0" rowsep="0" width="100%"> <tbody><tr class="description-tr"> <td class="description-td">Repeat steps e-g 25 times</td> </tr> </tbody></table> <table align="left" class="description-table" cols="2" colsep="0" rowsep="0" width="100%"> <tbody><tr class="description-tr"> <td class="description-td">h)</td> <td class="description-td">72° C. 10 minutes final extension</td> </tr> <tr class="description-tr"> <td align="center" class="description-td" colspan="2" nameend="2" namest="1" rowsep="1"> </td> </tr> </tbody></table> </patent-tables> </tables>
</div>
</li> <li> <para-num num="[0378]"> </para-num> <div class="description-line" id="P-0378" num="0378">A single amplified product having the expected size of approximately 2 kbp was detected by agarose gel electrophoresis. The product was isolated by QuiaX (QIAGEN Inc, Valencia Calif.) in a final volume of 20 microliters. <span style="display: none">[0378] </span> </div>
</li> <li> <para-num num="[0379]"> </para-num> <div class="description-line" id="P-0379" num="0379">The isolated product was ligated into the pCR2.1 vector (Invitrogen, Carlsbad, Calif.) to provide a construct called pCR2.1-20692369-S929-2C. It was sequenced using the following sequencing primers:  
<span style="display: none">[0379] </span> <tables id="TABLE-US-00035" num="35"> <patent-tables colsep="0" frame="none" rowsep="0" tabstyle="monospace"> <table align="left" class="description-table" cols="3" colsep="0" rowsep="0" width="100%"> <tbody><tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">20692369 S1:</td> <td class="description-td">CACTTGGTGACTGACGCCGT</td> <td class="description-td">(SEQ ID NO: 61),</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">20692369 S2:</td> <td class="description-td">ACGGCGTCAGTCACCAAGTG</td> <td class="description-td">(SEQ ID NO: 62),</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">20692369 S3:</td> <td class="description-td">CGGCAGGCTGGCTGGGAGC</td> <td class="description-td">(SEQ ID NO: 63),</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">20692369 S4:</td> <td class="description-td">GCTCCCAGCCAGCCTGCCG</td> <td class="description-td">(SEQ ID NO: 64),</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">20692369 S5:</td> <td class="description-td">GCTGCGGAGAGAGCTCTT</td> <td class="description-td">(SEQ ID NO: 65),</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">20692369 S6:</td> <td class="description-td">AAGAGCTCTCTCCGCAGC</td> <td class="description-td">(SEQ ID NO: 66),</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">20692369 S7:</td> <td class="description-td">CGAGGCCTCACCAGTGCTTGC</td> <td class="description-td">(SEQ ID NO: 67),</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">20692369 S8;</td> <td class="description-td">GCAAGCACTGGTGAGGCCTCG</td> <td class="description-td">(SEQ ID NO: 68),</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">20692369 S9:</td> <td class="description-td">GCACTCTCTACACCTTCAG</td> <td class="description-td">(SEQ ID NO: 69),</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">20692369 S10:</td> <td class="description-td">CTGAAGGTGTAGAGAGTGC</td> <td class="description-td">(SEQ ID NO: 70).</td> </tr> </tbody></table> </patent-tables> </tables>
</div>
</li> <li> <para-num num="[0380]"> </para-num> <div class="description-line" id="P-0380" num="0380">The resulting sequence showed that the insert is an ORF coding for a variant of 20692369_EXT. The 5′ end of the DNA insert was different from the predicted DNA sequence (see below). For this reason the forward primer (20692369 MAT-FORW; SEQ ID NO: 59) was not in frame. To be able to express the cloned variant of 20692369_EXT, an in-frame 5′ primer was designed. The sequence of this primer is shown below. <span style="display: none">[0380] </span> </div>
</li> <li> <para-num num="[0381]"> </para-num> <div class="description-line" id="P-0381" num="0381">20692369-S929<span style="display: none">[0381] </span> <sub>—</sub>2C-FORW: AGATCTAACCGCTCCGACTGCGGCCCGCAGC (SEQ ID NO: 71). </div>
</li> <li> <para-num num="[0382]"> </para-num> <div class="description-line" id="P-0382" num="0382">PCR reactions were set up using pCR2.1-20692369-S929-2C DNA as template, 1 microM of each of the 20692369-S929<span style="display: none">[0382] </span> <sub>—</sub>2C-FORW (SEQ ID NO: 71) and 20692369 REV (SEQ ID NO: 60) primers, 5 micromoles dNTP (Clontech Laboratories, Palo Alto Calif.) and 1 microliter of 50xAdvantage-HF 2 polymerase (Clontech Laboratories) in a volume of 50 microliters. The following reaction conditions were used:  
<tables id="TABLE-US-00036" num="36"> <patent-tables colsep="0" frame="none" rowsep="0"> <table align="left" class="description-table" cols="4" colsep="0" rowsep="0" width="100%"> <thead> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td align="center" class="description-td" colspan="4" nameend="3" namest="OFFSET" rowsep="1"> </td> </tr> </thead> <tbody><tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">a)</td> <td class="description-td">96° C.</td> <td class="description-td"> 3 minutes</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">b)</td> <td class="description-td">96° C.</td> <td class="description-td">30 seconds denaturation</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">c)</td> <td class="description-td">60° C.</td> <td class="description-td">30 seconds annealing</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">d)</td> <td class="description-td">72° C.</td> <td class="description-td"> 3 minute extension</td> </tr> </tbody></table> <table align="left" class="description-table" cols="1" colsep="0" rowsep="0" width="100%"> <tbody><tr class="description-tr"> <td class="description-td">(repeat steps b, c and d 15 times)</td> </tr> </tbody></table> <table align="left" class="description-table" cols="4" colsep="0" rowsep="0" width="100%"> <tbody><tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">e)</td> <td class="description-td">72° C.</td> <td class="description-td">10 minute extension</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td align="center" class="description-td" colspan="4" nameend="3" namest="OFFSET" rowsep="1"> </td> </tr> </tbody></table> </patent-tables> </tables>
</div>
</li> <li> <para-num num="[0383]"> </para-num> <div class="description-line" id="P-0383" num="0383">The isolated PCR product was ligated into the pCR2.1 vector to provide a construct called pCR2.1-20692369_EXT-A98<span style="display: none">[0383] </span> <sub>—</sub>10A and sequenced. The resulting sequence is shown in Table 31 (SEQ ID NO:72), and verifies that the insert codes for a variant of 20692369_EXT.  
<tables id="TABLE-US-00037" num="37"> <patent-tables colsep="0" frame="none" pgwide="1" rowsep="0" tabstyle="monospace"> <table align="left" class="description-table" cols="1" colsep="0" rowsep="0" width="100%"> <thead> <tr class="description-tr"> <td align="center" class="description-td" colspan="1" nameend="1" namest="1">TABLE 31</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td align="center" class="description-td" colspan="1" nameend="1" namest="1" rowsep="1"> </td> </tr> <tr class="description-tr"> <td class="description-td">DNA sequence of the cloned insert in pCR2.1-20692369_EXT-A98_A.</td> </tr> <tr class="description-tr"> <td align="center" class="description-td" colspan="1" nameend="1" namest="1" rowsep="1"> </td> </tr> </thead> <tbody><tr class="description-tr"> <td class="description-td"> </td> </tr> </tbody></table> <table align="left" class="description-table" cols="3" colsep="0" rowsep="0" width="100%"> <tbody><tr class="description-tr"> <td class="description-td">AACCGCTCCGACTGCGGCCCGCAGCCGCCGCCGCCGCCCAAGTGCGAGCTCTTGCAT</td> <td class="description-td">(SEQ ID NO:72)</td> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">GTGGCCATCGTGTGTGCGGGGCATAACTCCAGCCGAGACGTCATCACCCTGGTGAA</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">GTCCATGCTCTTCTACAGGAAAAATCCACTGCACCTCCACTTGGTGACTGACGCCGT</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">GGCCAGAAACATCCTGGAGACGCTCTTCCACACATGGATGGTGCCTGCTGTCCGTGT</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">CAGCTTTTATCATACCGACCAGCTCAAGCCCCAGGTCTCCTGGATCCCCAACAAGCA</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">CTACTCCGGCCTCTATGGGCTAATGAAGCTGGTGCTGCCCAGTGCCTTGCCTGCTGA</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">GCTGGCCCGCGTCATTGTCCTGGACACGGATGTCACCTTCGCCTCTGACATCTCGGA</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">GCTCTGGGCCCTCTGTGCTCACTTTTCTGACACGCAGGCGATCGGTCTTGTGGAGAA</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">CCAGAGTGACTGGTACCTGGGCAACCTCTGGAAGAACCACAGGCCCTGGCCTGCCTT</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">GGGCCGGGGATTTAACACAGGTGTGATCCTGCTGCGGCTGGACCGGCTCCGGCAGG</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">CTGGCTGGGAGCAGATGTGGAGGCTGACAGCCAGGCGGGAGCTCCTTAGCCTGCCT</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">GCCACCTCACTGGCTGACCAGGACATCTTCAACGCTGTGATCAAGGAGCACCCGGG</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">GCTAGTGCAGCGTCTGCCTTGTGTCTGGAATGTGCAGCTGTCAGATCACACACTGGC</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">CGAGCGCTGCTACTCTGAGGCGTCTGACCTCAAGGTGATCCACTGGAACTCACCAAA</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">GAAGCTTCGGGTGAAGAACAAGCATGTGGAATTCTTCCGCAATTTCTACCTGACCTT</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">CCTGGAGTACGATGGGAACCTGCTGCGGAGAGAGCTCTTTGTGTGCCCCAGCCAGCC</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">CCCACCTGGTGCTGAGCAGTTGCAGCAGGCCCTGGCACAACTGGACGAGGAAGACC</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">CCTGCTTTGAGTTCCGGCAGCAGCAGCTCACTGTGCACCGTGTGCATGTCACTTTCCT</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">GCCCCATGAACCGCCACCCCCCCGGCCTCACGATGTCACCCTTGTGGCCCAGCTGTC</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">CATGGACCGGCTGCAGATGTTGGAAGCCCTGTGCAGGCACTGGCCTGGCCCCATGA</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">GCCTGGCCTTGTACCTGACAGACGCAGAAGCTCAGCAGTTCCTGCATTTCGTCGAGG</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">CCTCACCAGTGCTTGCTGCCCGGCAGGACGTGGCCTACCATGTGGTGTACCGTGAGG</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">GGCCCCTATACCCCGTCAACCAGCTTCGCAACGTGGCCTTGGCCCAGGCCCTCACGC</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">CTTACGTCTTCCTCAGTGACATTGACTTCCTGCCTGCCTATTCTCTCTACGACTACCT</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">CAGGGCCTCCATTGAGCAGCTGGGGCTGGGCAGCCGGCGCAAGGCAGCACTGGTGG</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">TGCCGGCATTTGAGACCCTGCGCTACCGCTTCAGCTTCCCCCATTCCAAGGTGGAGC</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">TGTTGGCCTTGCTGGATGCGGGCACTCTCTACACCTTCAGGTACCACGAGTGGCCCC</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">GAGGCCACGCACCCACAGACTATGCCCGCTGGCGGGAGGCTCAGGCCCCGTACCGT</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">GTGCAATGGGCGGCCAACTATGAACCCTACGTGGTGGTGCCACGAGACTGTCCCCG</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">CTATGATCCTCGCTTTGTGGGCTTCGGCTGGAACAAAGTGGCCCACATTGTGGAGCT</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">GGATGCCCAGGAATATGAGCTCCTGGTGCTGCCCGAGGCCTTCACCATCCATCTGCC</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">CCACGCTCCAAGCCTGGACATCTCCCGCTTCCGCTCCAGCCCCACCTATCGTGACTG</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">CCTCCAGGCCCTCAAGGACGAATTCCACCAGGACTTGTCCCGCCACCATGGGGCTGC</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">TGCCCTCAAATACCTCCCAGCCCTGCAGCAGCCCCAGAGCCCTGCCCGAGGC</td> </tr> <tr class="description-tr"> <td align="center" class="description-td" colspan="3" nameend="3" namest="1" rowsep="1"> </td> </tr> </tbody></table> </patent-tables> </tables>
</div>
</li> <li> <para-num num="[0384]"> </para-num> <div class="description-line" id="P-0384" num="0384">The amino acid sequence of the polypeptide coded by the insert pCR2.1-20692369_EXT-A98<span style="display: none">[0384] </span> <sub>—</sub>10A is shown in Table 32 (SEQ ID NO:73)  
<tables id="TABLE-US-00038" num="38"> <patent-tables colsep="0" frame="none" pgwide="1" rowsep="0" tabstyle="monospace"> <table align="left" class="description-table" cols="3" colsep="0" rowsep="0" width="100%"> <thead> <tr class="description-tr"> <td align="center" class="description-td" colspan="3" nameend="3" namest="1">TABLE 32</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td align="center" class="description-td" colspan="3" nameend="3" namest="1" rowsep="1"> </td> </tr> </thead> <tbody><tr class="description-tr"> <td class="description-td">    NRSDCGPQPPPPPKCELLHVAIVCAGHNSSRDVITLVKSMLFYRKNPLHLHLVTD</td> <td class="description-td">(SEQ ID NO:73)</td> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">AVARNILETLFHTWMVPAVRVSFYHTDQLKPQVSWIPNKHYSGLYGLMKLVLPSALPA</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">ELARVIVLDTDVTFASDISELWALCAHFSDTQAJGLVENQSDWYLGNLWKNHRPWPAL</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">GRGFNTGVILLRLDRLRQAGWEQMWRLTARRELLSLPATSLADQDIFNAVIKEHPGLVQ</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">RLPCVWNVQLSDHTLAERCYSEASDLKVIHWNSPKKLRVKNKHVEFFRNFYLTFLEYD</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">GNLLRRELFVCPSQPPPGAEQLQQALAQLDEEDPCFEFRQQQLTVHRVHVTFLPHEPPPP</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">RPHDVTLVAQLSMDRLQMLEALCRHWPGPMSLALYLTDAEAQQFLHFVEASPVLAAR</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">QDVAYHVVYREGPLYPVNQLRNVALAQALTPYVFLSDIDFLPAYSLYDYLRASIEQLGL</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">GSRRKAALVVPAFETLRYRFSFPHSKVELLALLDAGTLYTFRYHEWPRGHAPTDYARW</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">REAQAPYRVQWAANYEPYVVVPRDCPRYDPRFVGFGWNKVAHIVELDAQEYELLVLP</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">EAFTIHLPHAPSLDJSRFRSSPTYRDCLQALKDEFHQDLSRHHGAAALKYLPALQQPQSP</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">ARG</td> </tr> <tr class="description-tr"> <td align="center" class="description-td" colspan="3" nameend="3" namest="1" rowsep="1"> </td> </tr> </tbody></table> </patent-tables> </tables>
</div>
</li> <li> <para-num num="[0385]"> </para-num> <div class="description-line" id="P-0385" num="0385">The relationship between the acetylglucosaminyltransferase-like proteins encoded by clones 10312947.0.40, SC20692369ext and pCR2.1-20692369_EXT-A98<span style="display: none">[0385] </span> <sub>—</sub>10A is represented in an alignment prepared using the ClustalW algorithm shown below.  
<tables id="TABLE-US-00039" num="39"> <patent-tables colsep="0" frame="none" pgwide="1" rowsep="0" tabstyle="monospace"> <table align="left" class="description-table" cols="4" colsep="0" rowsep="0" width="100%"> <tbody><tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">SC2069236 Sext</td> <td class="description-td">MLPRGRPRALGAAALLLLLLLLGFLLFGGDLGREAAESRRPRRNPGGPAP</td> <td class="description-td"> </td> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">pCR2.1-20692369_EXT-A98</td> <td class="description-td">-----------------------------------------NRSDCGPQP</td> </tr> <tr class="description-tr"> <td class="description-td">10312947.0.40</td> <td class="description-td">--------------------------------------------------</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">SC20692369ext</td> <td class="description-td">GTTTAPTAARSRRRPPKCELLHVAIVCAGHNSSRDVIILVKSMLFYRKNP</td> </tr> <tr class="description-tr"> <td class="description-td">pCR2.1-20692369_EXT-A98</td> <td class="description-td">------------PFPPKCELLHVAIVCAGHNSSRDVITLVKSMLFYRKNP</td> </tr> <tr class="description-tr"> <td class="description-td">10312947.0.40</td> <td class="description-td">------------------MLLLLGPLRLPLCPPK------------RKNP</td> </tr> <tr class="description-tr"> <td class="description-td"> ** :. : ..: ****</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">SC20692369ext</td> <td class="description-td">LHLHLVTDAVARNILETLFHTWMVPAVR--VSFYHADQLKPQVSWIPNKH</td> </tr> <tr class="description-tr"> <td class="description-td">pCR2.1-20692369_EXT-A98</td> <td class="description-td">LHLHLVTDAVARNILETLFHTWMVPAVR--VSFYHTDQLKPQVSWIPNKH</td> </tr> <tr class="description-tr"> <td class="description-td">10312947.0.40</td> <td class="description-td">LHLHLVTDAVARNILETLFHTWMVPAIDPXVSFYHADQLKPQVSWIPNKH</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">**************************:   *****:*************</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">SC20692369ext</td> <td class="description-td">YSGLYGLMKLVLPSALPAELARVIVLDTDVTFASDISELWALFAHFSDTQ</td> </tr> <tr class="description-tr"> <td class="description-td">pCR2.1-20692369_EXT-A98</td> <td class="description-td">YSGLYGLMKLVLPSALPAELARVIVLDTDVTFASDISELWALCAHFSDTQ</td> </tr> <tr class="description-tr"> <td class="description-td">10312947.0.40</td> <td class="description-td">YSGLYGLMKLVLPNALPAELARVIVLDTDVTFASDISELWALCAHFSDTQ</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">*************.**************************** *******</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">SC20692369_ext-98</td> <td class="description-td">AIGLVENQSDWYLGNLWKNHRPWPAIJGRGFNTGVILLRLDRLRQAGWEQM</td> </tr> <tr class="description-tr"> <td class="description-td">pCR2.1-20692369_EXT-A98</td> <td class="description-td">AIGLVENQSDWYLGNLWKNHRPWPALGRGFNTGVILLRLDRLRQAGWEQM</td> </tr> <tr class="description-tr"> <td class="description-td">         10312947.0.40</td> <td class="description-td">AIGLVENQSDWYLGNLWKNIIRPWPALGRGFNTGVILLRLDRLRQAGWEQM</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">**************************************************</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">SC20692369ext</td> <td class="description-td">WRLTARRELLSLPATSLADQDIFNAVIKEHPGLVQRLPCVWNVQLSDHTL</td> </tr> <tr class="description-tr"> <td class="description-td">pCR2.1-20692369_EXT-A98</td> <td class="description-td">WRLTARRELLSLPATSLADQDIFNAVIKEHPGLVQRLPCVNNVQLSDHTL</td> </tr> <tr class="description-tr"> <td class="description-td">10312947.0.40</td> <td class="description-td">WRLTARRELLSLPATSLADQDIFNAVIKEHPGLVQRLPCVWNVQLSDMTL</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">**************************************************</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">SC20692369ext</td> <td class="description-td">AERCYSEASDLKVIHWNSPKKLRVKNKHVEFFRNFYLTFLEYDGNLLRRE</td> </tr> <tr class="description-tr"> <td class="description-td">pCR2.1-20692369_EXT-A98</td> <td class="description-td">AERCYSEASDLKVIHWNSPKKLRVKNKHVEFFRNFYLTFLEYDGNLLRRE</td> </tr> <tr class="description-tr"> <td class="description-td">10312947.0.40</td> <td class="description-td">AERCYSEASDLKVIHWNSPKKLRVKNKHVEFFRNFYLTFLEYDGNLLRRE</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">**************************************************</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">SC20692369ext</td> <td class="description-td">LFVCPSQPPPGAEQLQQALAQLDEEDPCFEFRQQQLTVHRVHVTFLPHEP</td> </tr> <tr class="description-tr"> <td class="description-td">pCR2.1-20692369_EXT-A98</td> <td class="description-td">LFVCFSQPPPGAEQLQQALAQLDEEDPCFEFRQQQLTVHRVHVTFLPHEP</td> </tr> <tr class="description-tr"> <td class="description-td">10312947.0.40</td> <td class="description-td">LFVCPSQPPPGAEQLQQALAQLDEEDPCFEFRQQQLTVHRVHVTFLPHEP</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">**************************************************</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">SC20692369ext</td> <td class="description-td">PPPRPHDVTLVAQLSMDRLQMLEALCRHWPGPMSLALYLTDAEAQQFLHF</td> </tr> <tr class="description-tr"> <td class="description-td">pCR2.1-20692369_EXT-A98</td> <td class="description-td">PPPRPHDVTLVAQLSMDRLQMLEALCRHWPGPMSLALYLTDAEAQQFLHF</td> </tr> <tr class="description-tr"> <td class="description-td">10312947.0.40</td> <td class="description-td">PPPRPHDVTLVAQLSMDRLQMLEALCRHWPGPMSLALYLTDAEAQQFLHF</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">**************************************************</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">SC20692369ext</td> <td class="description-td">VEASPVLAARQDVAYHVVYREGPLYPVNQLRNVALAQALTPYVFLSDIDF</td> </tr> <tr class="description-tr"> <td class="description-td">pCR2.1-20692369_EXT-A98</td> <td class="description-td">VEASPVLAARQDVAYHVVYREGPLYPVNQLRNVALAQALTPYVFLSDIDF</td> </tr> <tr class="description-tr"> <td class="description-td">10312947.0.40</td> <td class="description-td">VEASPVLAARQDVAYHVVYREGPLYFVNQLRNVALAQALTPYVFLSDIDF</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">**************************************************</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">SC20692369ext</td> <td class="description-td">LPAYSLYDYLRASIEQLGLGSRRKAALVVPAFETLRYRFSFPHSKVELLA</td> </tr> <tr class="description-tr"> <td class="description-td">pCR2.1-20692369_EXT-A98</td> <td class="description-td">LPAYSLYDYLRASIEQLGLGSRRKAALVVPAFETLRYRFSFPNSKVELLA</td> </tr> <tr class="description-tr"> <td class="description-td">10312947.0.40</td> <td class="description-td">LPAYSLYDYLRASIEQLGLGSRRKAALVVPAFETLRYRFSFPHSKVELLA</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">**************************************************</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">SC20692369ext</td> <td class="description-td">LLDAGTLYTFRYHEWPRGHAPTDYARWREAQAPYRVQWAANYEPYVVVPR</td> </tr> <tr class="description-tr"> <td class="description-td">pCR2.1-20692369_EXT-A98</td> <td class="description-td">LLDAGTLYTFRYHEWPRGHAPTDYARWREAQAPYRVQWAANYEPYVVVPR</td> </tr> <tr class="description-tr"> <td class="description-td">10312947.0.40</td> <td class="description-td">LLDAGTLYTFRYHEWPRGHAPTDYARWREAQAPYRVQWAANYEPYVVVPR</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">**************************************************</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">SC20692369ext</td> <td class="description-td">DCPRYDPRFVGFGWNKVAHIVELDAQEYELLVLPEAFTIHLPHAPSLDIS</td> </tr> <tr class="description-tr"> <td class="description-td">pCR2.1-20692369_EXT-A98</td> <td class="description-td">DCPRYDPRFVGFGWNKVAHIVELDAQEYELLVLPEAFTIHLPHAPSLDIS</td> </tr> <tr class="description-tr"> <td class="description-td">10312947.0.40</td> <td class="description-td">DCPRYDPRFVGFGWNKVAHIVELDAQEYELLVLPEAFTIHLPHAPSLDIS</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">**************************************************</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">SC20692369ext</td> <td class="description-td">RFRSSPTYRDCLQALKDEFHQDLSRHHGAAALKYLPALQQPQSPARG</td> <td class="description-td">(SEQ ID NO.: 74)</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">pCR2.1-20692369_EXT-A98</td> <td class="description-td">RFRSSPTYRDCLQALKDEFHQDLSRHHGAAALKYLPALQQPQSPARG</td> <td class="description-td">(SEQ ID NO.: 73)</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">10312947.0.40</td> <td class="description-td">RFRSSPTYRDCLQALKDEFHQDLSRHHGAAALKYLPALQQPQSPARG</td> <td class="description-td">(SEQ ID NO.: 75)</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> </tbody></table> <table align="left" class="description-table" cols="1" colsep="0" rowsep="0" width="100%"> <tbody><tr class="description-tr"> <td class="description-td">Where * indicates identity, indicates strong similarity and . indicates weak similarity.</td> </tr> </tbody></table> </patent-tables> </tables>
</div>
</li> <heading>OTHER EMBODIMENTS </heading>
<li> <para-num num="[0386]"> </para-num> <div class="description-line" id="P-0386" num="0386">While the invention has been described in conjunction with the detailed description thereof, the foregoing description is intended to illustrate and not limit the scope of the invention, which is defined by the scope of the appended claims. Other aspects, advantages, and modifications are within the scope of the following claims. <span style="display: none">[0386] </span> </div>
</li> </ul>
</div>
</section><section itemprop="claims" itemscope="">
<h2>Claims (<span itemprop="count">43</span>)</h2>
<div html="" itemprop="content"><div class="claims" lang="EN" load-source="patent-office" mxw-id="PCLM60335998">
<claim-statement>What is claimed is: </claim-statement>
<div class="claim"> <div class="claim" id="US-20020037850-A1-CLM-00001" num="1">
<div class="claim-text"> <b>1</b>. An isolated polypeptide comprising an amino acid sequence selected from the group consisting of: 
<div class="claim-text">a) a mature form of the amino acid sequence selected from the group consisting of SEQ ID NO: 2, 4, 6, 8, 10, 12 or 14; </div> <div class="claim-text">b) a variant of a mature form of the amino acid sequence selected from the group consisting of SEQ ID NO: 2, 4, 6, 8, 10, 12 or 14, wherein any amino acid in the mature form is changed to a different amino acid, provided that no more than 15% of the amino acid residues in the sequence of the mature form are so changed; </div> <div class="claim-text">c) the amino acid sequence selected from the group consisting of SEQ ID NO: 2, 4, 6, 8, 10, 12 or 14; </div> <div class="claim-text">d) a variant of the amino acid sequence selected from the group consisting of SEQ ID NO: 2, 4, 6, 8, 10, 12 or 14 wherein any amino acid specified in the chosen sequence is changed to a different amino acid, provided that no more than 15% of the amino acid residues in the sequence are so changed; and </div> <div class="claim-text">e) a fragment of any of a) through d). </div> </div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="US-20020037850-A1-CLM-00002" num="2">
<div class="claim-text"> <b>2</b>. The polypeptide of <claim-ref idref="US-20020037850-A1-CLM-00001">claim 1</claim-ref> that is a naturally occurring allelic variant of the sequence selected from the group consisting of SEQ ID NO: 2, 4, 6, 8, 10, 12 or 14. </div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="US-20020037850-A1-CLM-00003" num="3">
<div class="claim-text"> <b>3</b>. The polypeptide of <claim-ref idref="US-20020037850-A1-CLM-00002">claim 2</claim-ref>, wherein the variant is the translation of a single nucleotide polymorphism. </div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="US-20020037850-A1-CLM-00004" num="4">
<div class="claim-text"> <b>4</b>. The polypeptide of <claim-ref idref="US-20020037850-A1-CLM-00001">claim 1</claim-ref> that is a variant polypeptide described therein, wherein any amino acid specified in the chosen sequence is changed to provide a conservative substitution. </div>
</div>
</div> <div class="claim"> <div class="claim" id="US-20020037850-A1-CLM-00005" num="5">
<div class="claim-text"> <b>5</b>. An isolated nucleic acid molecule comprising a nucleic acid sequence encoding a polypeptide comprising an amino acid sequence selected from the group consisting of: 
<div class="claim-text">a) a mature form of the amino acid sequence given SEQ ID NO: 2, 4, 6, 8, 10, 12 or 14; </div> <div class="claim-text">b) a variant of a mature form of the amino acid sequence selected from the group consisting of SEQ ID NO: 2, 4, 6, 8, 10, 12 or 14 wherein any amino acid in the mature form of the chosen sequence is changed to a different amino acid, provided that no more than 15% of the amino acid residues in the sequence of the mature form are so changed; </div> <div class="claim-text">c) the amino acid sequence selected from the group consisting of SEQ ID NO: 2, 4, 6, 8, 10, 12 or 14; </div> <div class="claim-text">d) a variant of the amino acid sequence selected from the group consisting of SEQ ID NO: 2, 4, 6, 8, 10, 12 or 14, in which any amino acid specified in the chosen sequence is changed to a different amino acid, provided that no more than 15% of the amino acid residues in the sequence are so changed; </div> <div class="claim-text">e) a nucleic acid fragment encoding at least a portion of a polypeptide comprising the amino acid sequence selected from the group consisting of SEQ ID NO: 2, 4, 6, 8, 10, 12 or 14 or any variant of said polypeptide wherein any amino acid of the chosen sequence is changed to a different amino acid, provided that no more than 10% of the amino acid residues in the sequence are so changed; and </div> <div class="claim-text">f) the complement of any of said nucleic acid molecules. </div> </div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="US-20020037850-A1-CLM-00006" num="6">
<div class="claim-text"> <b>6</b>. The nucleic acid molecule of <claim-ref idref="US-20020037850-A1-CLM-00005">claim 5</claim-ref>, wherein the nucleic acid molecule comprises the nucleotide sequence of a naturally occurring allelic nucleic acid variant. </div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="US-20020037850-A1-CLM-00007" num="7">
<div class="claim-text"> <b>7</b>. The nucleic acid molecule of <claim-ref idref="US-20020037850-A1-CLM-00005">claim 5</claim-ref> that encodes a variant polypeptide, wherein the variant polypeptide has the polypeptide sequence of a naturally occurring polypeptide variant. </div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="US-20020037850-A1-CLM-00008" num="8">
<div class="claim-text"> <b>8</b>. The nucleic acid molecule of <claim-ref idref="US-20020037850-A1-CLM-00005">claim 5</claim-ref>, wherein the nucleic acid molecule comprises a single nucleotide polymorphism encoding said variant polypeptide. </div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="US-20020037850-A1-CLM-00009" num="9">
<div class="claim-text"> <b>9</b>. The nucleic acid molecule of <claim-ref idref="US-20020037850-A1-CLM-00005">claim 5</claim-ref>, wherein said nucleic acid molecule comprises a nucleotide sequence selected from the group consisting of 
<div class="claim-text">a) the nucleotide sequence selected from the group consisting of SEQ ID NO: 1, 3, 5,7,9, 11, or 13; </div> <div class="claim-text">b) a nucleotide sequence wherein one or more nucleotides in the nucleotide sequence selected from the group consisting of SEQ ID NO: 1, 3, 5, 7, 9, 11, or 13 is changed from that selected from the group consisting of the chosen sequence to a different nucleotide provided that no more than 15% of the nucleotides are so changed; </div> <div class="claim-text">c) a nucleic acid fragment of the sequence selected from the group consisting of SEQ ID NO: 1, 3, 5, 7, 9, 11, or 13; and </div> <div class="claim-text">d) a nucleic acid fragment wherein one or more nucleotides in the nucleotide sequence selected from the group consisting of SEQ ID NO: 1, 3, 5, 7, 9, 11, or 13 is changed from that selected from the group consisting of the chosen sequence to a different nucleotide provided that no more than 15% of the nucleotides are so changed. </div> </div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="US-20020037850-A1-CLM-00010" num="10">
<div class="claim-text"> <b>10</b>. The nucleic acid molecule of <claim-ref idref="US-20020037850-A1-CLM-00005">claim 5</claim-ref>, wherein said nucleic acid molecule hybridizes under stringent conditions to the nucleotide sequence selected from the group consisting of SEQ ID NO: 1, 3, 5, 7, 9, 11, or 13, or a complement of said nucleotide sequence. </div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="US-20020037850-A1-CLM-00011" num="11">
<div class="claim-text"> <b>11</b>. The nucleic acid molecule of <claim-ref idref="US-20020037850-A1-CLM-00005">claim 5</claim-ref>, wherein the nucleic acid molecule comprises a nucleotide sequence in which any nucleotide specified in the coding sequence of the chosen nucleotide sequence is changed from that selected from the group consisting of the chosen sequence to a different nucleotide provided that no more than 15% of the nucleotides in the chosen coding sequence are so changed, an isolated second polynucleotide that is a complement of the first polynucleotide, or a fragment of any of them. </div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="US-20020037850-A1-CLM-00012" num="12">
<div class="claim-text"> <b>12</b>. A vector comprising the nucleic acid molecule of <claim-ref idref="US-20020037850-A1-CLM-00011">claim 11</claim-ref>. </div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="US-20020037850-A1-CLM-00013" num="13">
<div class="claim-text"> <b>13</b>. The vector of <claim-ref idref="US-20020037850-A1-CLM-00011">claim 12</claim-ref>, further comprising a promoter operably linked to said nucleic acid molecule. </div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="US-20020037850-A1-CLM-00014" num="14">
<div class="claim-text"> <b>14</b>. A cell comprising the vector of <claim-ref idref="US-20020037850-A1-CLM-00011">claim 12</claim-ref>. </div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="US-20020037850-A1-CLM-00015" num="15">
<div class="claim-text"> <b>15</b>. An antibody that binds immunospecifically to the polypeptide of <claim-ref idref="US-20020037850-A1-CLM-00001">claim 1</claim-ref>. </div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="US-20020037850-A1-CLM-00016" num="16">
<div class="claim-text"> <b>16</b>. The antibody of <claim-ref idref="US-20020037850-A1-CLM-00011">claim 15</claim-ref>, wherein said antibody is a monoclonal antibody. </div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="US-20020037850-A1-CLM-00017" num="17">
<div class="claim-text"> <b>17</b>. The antibody of <claim-ref idref="US-20020037850-A1-CLM-00011">claim 15</claim-ref>, wherein the antibody is a humanized antibody. </div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="US-20020037850-A1-CLM-00018" num="18">
<div class="claim-text"> <b>18</b>. A method for determining the presence or amount of the polypeptide of <claim-ref idref="US-20020037850-A1-CLM-00001">claim 1</claim-ref> in a sample, the method comprising: 
<div class="claim-text">(a) providing said sample; </div>
<div class="claim-text">(b) introducing said sample to an antibody that binds immunospecifically to the polypeptide; and </div>
<div class="claim-text">(c) determining the presence or amount of antibody bound to said polypeptide, thereby determining the presence or amount of polypeptide in said sample. </div>
</div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="US-20020037850-A1-CLM-00019" num="19">
<div class="claim-text"> <b>19</b>. A method for determining the presence or amount of the nucleic acid molecule of <claim-ref idref="US-20020037850-A1-CLM-00005">claim 5</claim-ref> in a sample, the method comprising: 
<div class="claim-text">(a) providing said sample; </div>
<div class="claim-text">(b) introducing said sample to a probe that binds to said nucleic acid molecule; and </div>
<div class="claim-text">(c) determining the presence or amount of said probe bound to said nucleic acid molecule, </div>
<div class="claim-text">thereby determining the presence or amount of the nucleic acid molecule in said sample. </div>
</div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="US-20020037850-A1-CLM-00020" num="20">
<div class="claim-text"> <b>20</b>. A method of identifying an agent that binds to the polypeptide of <claim-ref idref="US-20020037850-A1-CLM-00001">claim 1</claim-ref>, the method comprising: 
<div class="claim-text">(a) introducing said polypeptide to said agent; and </div> <div class="claim-text">(b) determining whether said agent binds to said polypeptide. </div> </div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="US-20020037850-A1-CLM-00021" num="21">
<div class="claim-text"> <b>21</b>. A method for identifying a potential therapeutic agent for use in treatment of a pathology, wherein the pathology is related to aberrant expression or aberrant physiological interactions of the polypeptide of <claim-ref idref="US-20020037850-A1-CLM-00001">claim 1</claim-ref>, the method comprising: 
<div class="claim-text">(a) providing a cell expressing the polypeptide of <claim-ref idref="US-20020037850-A1-CLM-00001">claim 1</claim-ref> and having a property or function ascribable to the polypeptide; </div> <div class="claim-text">(b) contacting the cell with a composition comprising a candidate substance; and </div> <div class="claim-text">(c) determining whether the substance alters the property or function ascribable to the polypeptide; </div> <div class="claim-text">whereby, if an alteration observed in the presence of the substance is not observed when the cell is contacted with a composition devoid of the substance, the substance is identified as a potential therapeutic agent. </div> </div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="US-20020037850-A1-CLM-00022" num="22">
<div class="claim-text"> <b>22</b>. A method for modulating the activity of the polypeptide of <claim-ref idref="US-20020037850-A1-CLM-00001">claim 1</claim-ref>, the method comprising introducing a cell sample expressing the polypeptide of said claim with a compound that binds to said polypeptide in an amount sufficient to modulate the activity of the polypeptide. </div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="US-20020037850-A1-CLM-00023" num="23">
<div class="claim-text"> <b>23</b>. A method of treating or preventing a pathology associated with the polypeptide of <claim-ref idref="US-20020037850-A1-CLM-00001">claim 1</claim-ref>, said method comprising administering the polypeptide of <claim-ref idref="US-20020037850-A1-CLM-00001">claim 1</claim-ref> to a subject in which such treatment or prevention is desired in an amount sufficient to treat or prevent said pathology in said subject. </div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="US-20020037850-A1-CLM-00024" num="24">
<div class="claim-text"> <b>24</b>. The method of <claim-ref idref="US-20020037850-A1-CLM-00022">claim 23</claim-ref>, wherein said subject is a human. </div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="US-20020037850-A1-CLM-00025" num="25">
<div class="claim-text"> <b>25</b>. A method of treating or preventing a pathology associated with the polypeptide of <claim-ref idref="US-20020037850-A1-CLM-00001">claim 1</claim-ref>, said method comprising administering to a subject in which such treatment or prevention is desired a NOVX nucleic acid in an amount sufficient to treat or prevent said pathology in said subject. </div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="US-20020037850-A1-CLM-00026" num="26">
<div class="claim-text"> <b>26</b>. The method of <claim-ref idref="US-20020037850-A1-CLM-00022">claim 25</claim-ref>, wherein said subject is a human. </div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="US-20020037850-A1-CLM-00027" num="27">
<div class="claim-text"> <b>27</b>. A method of treating or preventing a pathology associated with the polypeptide of <claim-ref idref="US-20020037850-A1-CLM-00001">claim 1</claim-ref>, said method comprising administering to a subject in which such treatment or prevention is desired a NOVX antibody in an amount sufficient to treat or prevent said pathology in said subject. </div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="US-20020037850-A1-CLM-00028" num="28">
<div class="claim-text"> <b>28</b>. The method of <claim-ref idref="US-20020037850-A1-CLM-00022">claim 27</claim-ref>, wherein the subject is a human. </div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="US-20020037850-A1-CLM-00029" num="29">
<div class="claim-text"> <b>29</b>. A pharmaceutical composition comprising the polypeptide of <claim-ref idref="US-20020037850-A1-CLM-00001">claim 1</claim-ref> and a pharmaceutically acceptable carrier. </div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="US-20020037850-A1-CLM-00030" num="30">
<div class="claim-text"> <b>30</b>. A pharmaceutical composition comprising the nucleic acid molecule of <claim-ref idref="US-20020037850-A1-CLM-00005">claim 5</claim-ref> and a pharmaceutically acceptable carrier. </div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="US-20020037850-A1-CLM-00031" num="31">
<div class="claim-text"> <b>31</b>. A pharmaceutical composition comprising the antibody of <claim-ref idref="US-20020037850-A1-CLM-00011">claim 15</claim-ref> and a pharmaceutically acceptable carrier. </div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="US-20020037850-A1-CLM-00032" num="32">
<div class="claim-text"> <b>32</b>. A kit comprising in one or more containers, the pharmaceutical composition of <claim-ref idref="US-20020037850-A1-CLM-00022">claim 29</claim-ref>. </div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="US-20020037850-A1-CLM-00033" num="33">
<div class="claim-text"> <b>33</b>. A kit comprising in one or more containers, the pharmaceutical composition of <claim-ref idref="US-20020037850-A1-CLM-00033">claim 30</claim-ref>. </div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="US-20020037850-A1-CLM-00034" num="34">
<div class="claim-text"> <b>34</b>. A kit comprising in one or more containers, the pharmaceutical composition of <claim-ref idref="US-20020037850-A1-CLM-00033">claim 31</claim-ref>. </div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="US-20020037850-A1-CLM-00035" num="35">
<div class="claim-text"> <b>35</b>. The use of a therapeutic in the manufacture of a medicament for treating a syndrome associated with a human disease, the disease selected from a pathology associated with the polypeptide of <claim-ref idref="US-20020037850-A1-CLM-00001">claim 1</claim-ref>, wherein said therapeutic is the polypeptide of <claim-ref idref="US-20020037850-A1-CLM-00001">claim 1</claim-ref>. </div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="US-20020037850-A1-CLM-00036" num="36">
<div class="claim-text"> <b>36</b>. The use of a therapeutic in the manufacture of a medicament for treating a syndrome associated with a human disease, the disease selected from a pathology associated with the polypeptide of <claim-ref idref="US-20020037850-A1-CLM-00001">claim 1</claim-ref>, wherein said therapeutic is a NOVX nucleic acid. </div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="US-20020037850-A1-CLM-00037" num="37">
<div class="claim-text"> <b>37</b>. The use of a therapeutic in the manufacture of a medicament for treating a syndrome associated with a human disease, the disease selected from a pathology associated with the polypeptide of <claim-ref idref="US-20020037850-A1-CLM-00001">claim 1</claim-ref>, wherein said therapeutic is a NOVX antibody. </div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="US-20020037850-A1-CLM-00038" num="38">
<div class="claim-text"> <b>38</b>. A method for screening for a modulator of activity or of latency or predisposition to a pathology associated with the polypeptide of <claim-ref idref="US-20020037850-A1-CLM-00001">claim 1</claim-ref>, said method comprising: 
<div class="claim-text">a) administering a test compound to a test animal at increased risk for a pathology associated with the polypeptide of <claim-ref idref="US-20020037850-A1-CLM-00001">claim 1</claim-ref>, wherein said test animal recombinantly expresses the polypeptide of <claim-ref idref="US-20020037850-A1-CLM-00011">claim 1;</claim-ref> </div> <div class="claim-text">b) measuring the activity of said polypeptide in said test animal after administering the compound of step (a); and </div> <div class="claim-text">c) comparing the activity of said protein in said test animal with the activity of said polypeptide in a control animal not administered said polypeptide, wherein a change in the activity of said polypeptide in said test animal relative to said control animal indicates the test compound is a modulator of latency of, or predisposition to, a pathology associated with the polypeptide of <claim-ref idref="US-20020037850-A1-CLM-00001">claim 1</claim-ref>. </div> </div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="US-20020037850-A1-CLM-00039" num="39">
<div class="claim-text"> <b>39</b>. The method of <claim-ref idref="US-20020037850-A1-CLM-00033">claim 38</claim-ref>, wherein said test animal is a recombinant test animal that expresses a test protein transgene or expresses said transgene under the control of a promoter at an increased level relative to a wild-type test animal, and wherein said promoter is not the native gene promoter of said transgene. </div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="US-20020037850-A1-CLM-00040" num="40">
<div class="claim-text"> <b>40</b>. A method for determining the presence of or predisposition to a disease associated with altered levels of the polypeptide of <claim-ref idref="US-20020037850-A1-CLM-00001">claim 1</claim-ref> in a first mammalian subject, the method comprising: 
<div class="claim-text">a) measuring the level of expression of the polypeptide in a sample from the first mammalian subject; and </div>
<div class="claim-text">b) comparing the amount of said polypeptide in the sample of step (a) to the amount of the polypeptide present in a control sample from a second mammalian subject known not to have, or not to be predisposed to, said disease, </div>
<div class="claim-text">wherein an alteration in the expression level of the polypeptide in the first subject as compared to the control sample indicates the presence of or predisposition to said disease. </div>
</div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="US-20020037850-A1-CLM-00041" num="41">
<div class="claim-text"> <b>41</b>. A method for determining the presence of or predisposition to a disease associated with altered levels of the nucleic acid molecule of <claim-ref idref="US-20020037850-A1-CLM-00005">claim 5</claim-ref> in a first mammalian subject, the method comprising: 
<div class="claim-text">a) measuring the amount of the nucleic acid in a sample from the first mammalian subject; and </div>
<div class="claim-text">b) comparing the amount of said nucleic acid in the sample of step (a) to the amount of the nucleic acid present in a control sample from a second mammalian subject known not to have or not be predisposed to, the disease; </div>
<div class="claim-text">wherein an alteration in the level of the nucleic acid in the first subject as compared to the control sample indicates the presence of or predisposition to the disease. </div>
</div>
</div>
</div> <div class="claim"> <div class="claim" id="US-20020037850-A1-CLM-00042" num="42">
<div class="claim-text"> <b>42</b>. A method of treating a pathological state in a mammal, the method comprising administering to the mammal a polypeptide in an amount that is sufficient to alleviate the pathological state, wherein the polypeptide is a polypeptide having an amino acid sequence at least 95% identical to a polypeptide comprising the amino acid sequence selected from the group consisting of SEQ ID NO: 2, 4, 6, 8, 10, 12 or 14 or a biologically active fragment thereof. </div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="US-20020037850-A1-CLM-00043" num="43">
<div class="claim-text"> <b>43</b>. A method of treating a pathological state in a mammal, the method comprising administering to the mammal the antibody of <claim-ref idref="US-20020037850-A1-CLM-00011">claim 15</claim-ref> in an amount sufficient to alleviate the pathological state.</div>
</div>
</div> </div>
</div>
</section>
                </article>
            </search-app>
        </body>
    </html>
    