
    <html>
        <body>
            <search-app>
                <article class="result" itemscope="" itemtype="http://schema.org/ScholarlyArticle">
    <h1 itemprop="pageTitle">US8017351B2 - Amylases for pharmaceutical use 
        - Google Patents</h1><section itemprop="abstract" itemscope="">
<h2>Abstract</h2>
<div html="" itemprop="content"><abstract lang="EN" load-source="patent-office" mxw-id="PA88017575">
<div class="abstract" num="p-0001">The pharmaceutical use of amylases related to <i>Bacillus </i>alpha-amylases of SEQ ID NOs: 1-3, optionally in combination with a lipase and/or a protease. Examples of medical indications are: Treatment of digestive disorders, pancreatic exocrine insufficiency (PEI), pancreatitis, cystic fibrosis, diabetes type I, and/or diabetes type II. The amylases of SEQ ID NOs: 1-3 are variants of amylases from <i>Bacillus stearothermophilus, Bacillus licheniformis </i>and <i>Bacillus </i>sp. The amylases of the invention have an improved efficacy in vivo, an improved pH-profile, a high specific activity, and/or an improved starch degradation profile.</div>
</abstract>
</div>
</section><section itemprop="description" itemscope="">
<h2>Description</h2>
<div html="" itemprop="content"><div class="description" lang="EN" load-source="patent-office" mxw-id="PDES44069456">
<heading>CROSS-REFERENCE TO RELATED APPLICATIONS</heading>
<div class="description-paragraph" num="p-0002">This application is a 35 U.S.C. 371 national application of PCT/DK2006/000354 filed Jun. 16, 2006, which claims priority or the benefit under 35 U.S.C. 119 of Danish application no. PA 2005 00931 filed Jun. 16, 2006 and of U.S. provisional application no. 60/694,169 filed Jun. 27, 2005, the contents of which are fully incorporated herein by reference.</div>
<heading>TECHNICAL FIELD</heading>
<div class="description-paragraph" num="p-0003">The present invention relates to the pharmaceutical use of amylases related to certain <i>Bacillus </i>alpha-amylases (SEQ ID NOs: 1-3), optionally in combination with a lipase and/or a protease. Examples of medical indications are: Treatment of digestive disorders, pancreatic exocrine insufficiency (PEI), pancreatitis, cystic fibrosis, diabetes type I, and/or diabetes type II.</div>
<heading>BACKGROUND ART</heading>
<div class="description-paragraph" num="p-0004">Several commercial medicaments in the form of pancreatic enzyme supplements are known for the treatment of pancreatic exocrine insufficiency. The active ingredients of these products are digestive enzymes, mainly amylase, lipase and protease, which are normally produced in the pancreas and excreted to the upper part of the small intestine (the duodenum). The enzymes used in such medicaments derive from bovine or swine pancreas, however there are also products on the market with microbial enzymes, e.g. the product Nortase® which contains a lipase from <i>Rhizopus oryzae</i>, a protease from <i>Aspergillus oryzae</i>, and an amylase from <i>Aspergillus oryzae. </i> </div>
<div class="description-paragraph" num="p-0005">U.S. Pat. No. 5,614,189 (EP 600868) describes the use of, i.a., a lipase derived from <i>Humicola lanuginosa </i>in pancreatic enzyme replacement therapy, for example in the treatment of patients suffering from cystic fibrosis. This lipase is from <i>Humicola lanuginosa </i>DSM 4109 and has the amino acid sequence of amino acids 1-269 of SEQ ID NO: 9 herein.</div>
<div class="description-paragraph" num="p-0006">WO 00/54799 describes the use of physiologically acceptable enzyme mixtures having lipolytic, proteolytic and amylolytic activity in the treatment of diabetes mellitus type I and II.</div>
<div class="description-paragraph" num="p-0007">WO 02/060474 describes the use of a concentrated lipase from <i>Rhizopus delemar</i>, a neutral protease from <i>Aspergillus melleus</i>, and an amylase from <i>Aspergillus oryzae </i>in the treatment of maldigestion.</div>
<div class="description-paragraph" num="p-0008">WO 01/62280 describes the use of a non-fungal lipase crystal crosslinked with a multifunctional crosslinking agent, a protease, and an amylase, wherein the lipase crystal is active at a pH range from about 2.0 to 9.0, for treating or preventing a gastrointestinal disorder in a mammal. A preferred lipase is from <i>Pseudomonas</i>, preferred amylases are from <i>Aspergillus </i>and <i>Bacillus</i>, preferred proteases are bromelain, papain or ficin.</div>
<div class="description-paragraph" num="p-0009">EP 0828509 describes the use of certain acid-stable amylases, optionally in combination with certain acid-stable lipases and/or proteases, in the treatment of exocrine pancreas insufficiency. A preferred amylase is from <i>Aspergillus niger</i>, and preferred lipases are from <i>Rhizopus arrhizus </i>or <i>Rhizopus javanicus. </i> </div>
<div class="description-paragraph" num="p-0010">WO 99/19467 describes certain variants of alpha-amylases derived from <i>Bacillus stearothermophilus, Bacillus licheniformis</i>, and <i>Bacillus amyloliquefaciens</i>, as well as various industrial uses thereof, however not the pharmaceutical use. The sequences of these wildtype <i>Bacillus </i>alpha-amylases designated SEQ ID NOs: 3-5 in WO 99/19467 are included herein as SEQ ID NOs: 10-12, respectively.</div>
<div class="description-paragraph" num="p-0011">EP 0594235 describes a method for preparing purified amylase from a fermentation broth. A pharmaceutical composition is also claimed, and a potential use as digestive aid is mentioned, along with other potential uses. The examples illustrate the claimed purification method for a <i>Bacillus licheniformis </i>alpha-amylase.</div>
<div class="description-paragraph" num="p-0012">WO 2004/078960 discloses a method for identifying at least one T-cell epitope of an amylase and variant amylases comprising at least one alteration in at least one epitope. Many potential uses are enumerated, including the pharmaceutical use. An amylase from <i>Bacillus licheniformis </i>is referred to.</div>
<div class="description-paragraph" num="p-0013">EP 0828509 describes the use of certain acid-stable amylases, optionally in combination with certain acid-stable lipases and/or proteases, in the treatment of exocrine pancreas insufficiency. A preferred amylase is from <i>Aspergillus niger</i>, and preferred lipases are from <i>Rhizopus arrhizus </i>or <i>Rhizopus javanicus. </i> </div>
<div class="description-paragraph" num="p-0014">There is a need in the art for alternative, preferably improved, enzymes for pharmaceutical use.</div>
<heading>SUMMARY OF THE INVENTION</heading>
<div class="description-paragraph" num="p-0015">The present invention provides alternative, preferably improved, enzymes for pharmaceutical use, in particular for the treatment of digestive disorders, pancreatic exocrine insufficiency (PEI), pancreatitis, cystic fibrosis, diabetes type I, and/or diabetes type II. The new enzymes are proteases, amylases, and lipases. Preferably, the enzymes for use according to the invention have an improved efficacy in vivo and/or in vitro; an improved pH-activity profile; an improved specific activity; an improved degradation profile; are active in the presence of bile salts; and/or have a reduced allergenicity.</div>
<div class="description-paragraph" num="p-0016">The present invention relates to an amylase of at least 50% identity to (i) amino acids 1-481 of SEQ ID NO: 1, (ii) amino acids 1-481 of SEQ ID NO: 2, and/or (iii) amino acids 1-483 of SEQ ID NO: 3; for use as a medicament, optionally in combination with a lipase, and/or a protease.</div>
<div class="description-paragraph" num="p-0017">The invention also relates to the use of such amylases for the manufacture of a medicament for the treatment of digestive disorders, PEI, pancreatitis, cystic fibrosis, diabetes type I, and/or diabetes type II, these uses optionally further comprising the use of a lipase, and/or a protease.</div>
<div class="description-paragraph" num="p-0018">The invention furthermore relates to a pharmaceutical composition comprising such amylases, together with at least one pharmaceutically acceptable auxiliary material, optionally including a lipase and/or a protease.</div>
<div class="description-paragraph" num="p-0019">The invention also relates to a method for the treatment of digestive disorders, PEI, pancreatitis (acute and/or chronic), cystic fibrosis, diabetes type I, and/or diabetes type II, by administering a therapeutically effective amount of such amylases, optionally together with a lipase and/or a protease.</div>
<description-of-drawings>
<heading>BRIEF DESCRIPTION OF THE DRAWINGS</heading>
<div class="description-paragraph" num="p-0020"> <figref idrefs="DRAWINGS">FIG. 1</figref> shows a chromatogram representing the degradation profile of an amylase having the amino acid sequence of amino acids 1-486 of SEQ ID NO: 1 (a <i>Bacillus stearothermophilus </i>amylase variant);</div>
<div class="description-paragraph" num="p-0021"> <figref idrefs="DRAWINGS">FIG. 2</figref> shows a chromatogram representing the degradation profile of an amylase from <i>Aspergillus oryzae; </i> </div>
<div class="description-paragraph" num="p-0022"> <figref idrefs="DRAWINGS">FIG. 3</figref> shows a chromatogram representing the degradation profile of Pancreatin;</div>
<div class="description-paragraph" num="p-0023"> <figref idrefs="DRAWINGS">FIG. 4</figref> shows a chromatogram representing the degradation profile of an amylase having the amino acid sequence of amino acids 1-483 of SEQ ID NO: 3 (a <i>Bacillus </i>sp. amylase variant); and</div>
<div class="description-paragraph" num="p-0024"> <figref idrefs="DRAWINGS">FIG. 5</figref> shows a chromatogram representing the degradation profile of an amylase having the amino acid sequence of amino acids 1-481 of SEQ ID NO: 2 (a <i>Bacillus licheniformis </i>amylase variant).</div>
</description-of-drawings>
<heading>DETAILED DESCRIPTION OF THE INVENTION</heading>
<div class="description-paragraph" num="h-0007">Enzymes</div>
<div class="description-paragraph" num="p-0025">The present invention relates to the pharmaceutical use of amylases having at least 50% identity to (i) amino acids 1-481 of SEQ ID NO: 1, (ii) amino acids 1-481 of SEQ ID NO: 2, and/or (iii) amino acids 1-483 of SEQ ID NO: 3; for use as a medicament. The invention also relates to the use of such amylases for the manufacture of a medicament for the treatment of digestive disorders, PEI, pancreatitis (acute and/or chronic), cystic fibrosis, diabetes type I, and/or diabetes type II. The invention furthermore relates to a pharmaceutical composition comprising such amylases, together with at least one pharmaceutically acceptable auxiliary material. The invention furthermore relates to a method for the treatment of the diseases mentioned above by administering a therapeutically effective amount of such amylases.</div>
<div class="description-paragraph" num="p-0026">In what follows, the amylase for use in the compositions, methods and uses of the invention is referred to as the “amylase of the invention.”</div>
<div class="description-paragraph" num="p-0027">In the present context, an amylase is an enzyme that catalyzes the endo-hydrolysis of starch and other linear and branched oligo- and polysaccharides. In a particular embodiment, the amylase for use according to the invention has alpha-amylase activity, viz. catalyzes the endohydrolysis of 1,4-alpha-glucosidic linkages in oligosaccharides and polysaccharides. Alpha-amylases act, e.g., on starch, glycogen and related polysaccharides and oligosaccharides in a random manner, liberating reducing groups in the alpha-configuration.</div>
<div class="description-paragraph" num="p-0028">In a preferred embodiment the amylase of the invention is an alpha-amylase (systematical name: 1,4-alpha-D-glucan glucanohydrolase). In further embodiments, the amylase of the invention belongs to the EC 3.2.1.-group of amylases, such as EC 3.2.1.1 (alpha-amylase), EC 3.2.1.2 (beta-amylase), EC 3.2.1.3 (glucan 1,4-alpha-glucosidase, amyloglucosidase, or glucoamylase), EC 3.2.1.20 (alpha-glucosidase), EC 3.2.1.60 (glucan 1,4-alpha-maltotetraohydrolase), EC 3.2.1.68 (isoamylase), EC 3.2.1.98 (glucan 1,4-alpha-maltohexosidase), or EC 3.2.1.133 (glucan 1,4-alpha-maltohydrolase). In a preferred embodiment, the amylase for use according to the invention can be, or is, classified as belonging to the EC 3.2.1.1 group. The EC numbers refer to Enzyme Nomenclature 1992 from NC-IUBMB, Academic Press, San Diego, Calif., including supplements 1-5 published in Eur. J. Biochem. 1994, 223, 1-5; Eur. J. Biochem. 1995, 232, 1-6; Eur. J. Biochem. 1996, 237, 1-5; Eur. J. Biochem. 1997, 250, 1-6; and Eur. J. Biochem. 1999, 264, 610-650; respectively. The nomenclature is regularly supplemented and updated; see e.g. the World Wide Web at http://www.chem.qmw.ac.uk/iubmb/enzyme/index.html.</div>
<div class="description-paragraph" num="p-0029">In particular embodiments, the amylase of the invention has a degree of identity to any one of (i) amino acids 1-481 of SEQ ID NO: 1, (ii) amino acids 1-481 of SEQ ID NO: 2, and/or (iii) amino acids 1-483 of SEQ ID NO: 3 of at least 51%, 52%, 53%, 54%, 55%, 56%, 57%, 58%, 59%, or at least 60%. In other particular embodiments, the amylase of the invention has a degree of identity to any one of (i) amino acids 1-481 of SEQ ID NO: 1, (ii) amino acids 1-481 of SEQ ID NO: 2, and/or (iii) amino acids 1-483 of SEQ ID NO: 3 of at least 61%, 62%, 63%, 64%, 65%, 66%, 67%, 68%, 69%, or at least 70%. In additional particular embodiments, the amylase of the invention has a degree of identity to any one of (i) amino acids 1-481 of SEQ ID NO: 1, (ii) amino acids 1-481 of SEQ ID NO: 2, and/or (iii) amino acids 1-483 of SEQ ID NO: 3 of at least 71%, 72%, 73%, 74%, 75%, 76%, 77%, 78%, 79%, or at least 80%. In other particular embodiments, the amylase of the invention has a degree of identity to any one of (i) amino acids 1-481 of SEQ ID NO: 1, (ii) amino acids 1-481 of SEQ ID NO: 2, and/or (iii) amino acids 1-483 of SEQ ID NO: 3 of at least 81%, 82%, 83%, 84%, 85%, 86%, 87%, 88%, 89%, or at least 90%. In still further particular embodiments, the amylase of the invention has a degree of identity to any one of (i) amino acids 1-481 of SEQ ID NO: 1, (ii) amino acids 1-481 of SEQ ID NO: 2, and/or (iii) amino acids 1-483 of SEQ ID NO: 3 of at least 91%, 92%, 93%, 94%, 95%, 96%, 97%, 98%, or at least 99%.</div>
<div class="description-paragraph" num="p-0030">In a further particular embodiment, the amylase of the invention comprises at least one substitution, deletion, and/or insertion of one or more amino acids in any one of the sequences of (i) amino acids 1-481 of SEQ ID NO: 1, (ii) amino acids 1-481 of SEQ ID NO: 2, and/or (iii) amino acids 1-483 of SEQ ID NO: 3. Preferably, the amino acid changes are of a minor nature, that is conservative amino acid substitutions or insertions that do not significantly affect the folding and/or activity of the protein; small deletions; small amino- or carboxyl-terminal extensions, such as an amino-terminal methionine residue; a small linker peptide; or a small extension that facilitates purification by changing net charge or another function, such as a poly-histidine tract, an antigenic epitope or a binding domain. In the above context, the term “small” independently designates a number of up to 25 amino acid residues. In preferred embodiments, the term “small” designates up to 24, 23, 22, 21, or up to 20 amino acid residues. In additional preferred embodiments, the term “small” independently designates up to 19, 18, 17, 16, 15, 14, 13, 12, 11, or up to 10 amino acid residues. In further preferred embodiments, the term “small” independently designates up to 9, 8, 7, 6, 5, 4, 3, 2, or up to 1 amino acid residue. In alternative embodiments, the term “small” independently designates up to 40, 39, 38, 37, 36, 35, 34, 33, 32, 31, 30, 29, 28, 27, 26, or up to 25 amino acid residues.</div>
<div class="description-paragraph" num="p-0031">Examples of conservative substitutions are within the group of basic amino acids (arginine, lysine and histidine), acidic amino acids (glutamic acid and aspartic acid), polar amino acids (serine, threonine, glutamine and asparagine), hydrophobic amino acids (leucine, isoleucine, valine and alanine), aromatic amino acids (phenylalanine, tryptophan and tyrosine), and small amino acids (glycine, alanine, proline, serine, threonine, cysteine and methionine).</div>
<div class="description-paragraph" num="p-0032">In the alternative, examples of conservative substitutions are within the group of basic amino acids (arginine, lysine and histidine), acidic amino acids (glutamic acid and aspartic acid), polar amino acids (glutamine and asparagine), hydrophobic amino acids (leucine, isoleucine and valine), aromatic amino acids (phenylalanine, tryptophan and tyrosine), and small amino acids (glycine, alanine, serine, threonine and methionine). Amino acid substitutions which do not generally alter specific activity are known in the art and are described, for example, by H. Neurath and R. L. Hill, 1979, In, The Proteins, Academic Press, New York. The most commonly occurring exchanges are Ala/Ser, Val/Ile, Asp/Glu, Thr/Ser, Ala/Gly, Ala/Thr, Ser/Asn, Ala/Val, Ser/Gly, Tyr/Phe, Ala/Pro, Lys/Arg, Asp/Asn, Leu/Ile, Leu/Val, Ala/Glu, and Asp/Gly.</div>
<div class="description-paragraph" num="p-0033">In additional embodiments, the amylase of the invention has an amino acid sequence which differs by no more than 25, 24, 23, 22, 21, 20, 19, 18, 17, 16, 15, 14, 13, 12, or no more than 11 amino acids from any one of (i) amino acids 1-481 of SEQ ID NO: 1, (ii) amino acids 1-481 of SEQ ID NO: 2, and/or (iii) amino acids 1-483 of SEQ ID NO: 3; or, it differs from any one of (i) amino acids 1-481 of SEQ ID NO: 1, (ii) amino acids 1-481 of SEQ ID NO: 2, and/or (iii) amino acids 1-483 of SEQ ID NO: 3 by no more than 10, 9, 8, 7, 6, 5, 4, 3, 2, or by no more than 1 amino acid. In alternative embodiments, the amylase of the invention has an amino acid sequence which differs by no more than 40, 39, 38, 37, 36, 35, 34, 33, 32, 31, 30, 29, 28, 27, or no more than 26 amino acids from any one of (i) amino acids 1-481 of SEQ ID NO: 1, (ii) amino acids 1-481 of SEQ ID NO: 2, and/or (iii) amino acids 1-483 of SEQ ID NO: 3.</div>
<div class="description-paragraph" num="p-0034">In still further particular embodiments, the amylase of the invention is an allelic variant of any one of the amylases having (i) amino acids 1-481 of SEQ ID NO: 1, (ii) amino acids 1-481 of SEQ ID NO: 2, and/or (iii) amino acids 1-483 of SEQ ID NO: 3, or a fragment thereof that has amylase activity. The amylase of the invention may also be an allelic variant of any one of the amylases having the amino acid sequences of SEQ ID NO: 10, SEQ ID NO: 11, and SEQ ID NO: 12. The term allelic variant denotes any of two or more alternative forms of a gene occupying the same chromosomal locus. Allelic variation arises naturally through mutation, and may result in polymorphism within populations. Gene mutations can be silent (no change in the encoded polypeptide) or may encode polypeptides having altered amino acid sequences. An allelic variant of a polypeptide is a polypeptide encoded by an allelic variant of a gene. The term fragment is defined herein as a polypeptide having one or more amino acids deleted from the amino and/or carboxyl terminus of SEQ ID NO: 1, SEQ ID NO: 2 or SEQ ID NO: 3, preferably having one or more amino acids deleted from the amino and/or carboxyl terminus of amino acids 1-481, 1-484, 1-486, or 1-513 of SEQ ID NO: 1; of amino acids 1-481 of SEQ ID NO: 2, or of amino acids 1-483 of SEQ ID NO: 3. Preferably, a small number of amino acids has been deleted, small being defined as explained above. More preferably, a fragment contains at least 440, 441, 442, 443, 444, 445, 446, 447, 448, 449, 450, 451, 452, 453, or at least 454 amino acid residues. Most preferably, a fragment contains at least 452, 453 454, 455, 456, 457, 458, 459, 460, 461, 462, 463, 464, 465, 466, 467, 468, 469, or at least 470 amino acid residues. Even more preferably, a fragment contains at least 471, 472, 473, 474, 475, 476, 477, 478, 479, or at least 480 amino acid residues.</div>
<div class="description-paragraph" num="p-0035">In summary, one embodiment of the present invention relates to an amylase for pharmaceutical use, wherein a) the amylase comprises an amino acid sequence selected from the group consisting of (i) amino acids 1-481 of SEQ ID NO: 1, (ii) amino acids 1-481 of SEQ ID NO: 2, and/or (iii) amino acids 1-483 of SEQ ID NO: 3; and/or b) the amylase is a variant of an amino acid sequence selected from the group consisting of (i) amino acids 1-481 of SEQ ID NO: 1, (ii) amino acids 1-481 of SEQ ID NO: 2, and/or (iii) amino acids 1-483 of SEQ ID NO: 3, wherein the variant differs from the respective amino acid sequence by no more than twenty-five amino acids, and wherein: (i) the variant comprises at least one substitution, deletion and/or insertion of one or more amino acids as compared to the respective amino acid sequence; and/or (ii) the variant comprises at least one small deletion as compared to the respective amino acid sequence; and/or (iii) the variant comprises at least one small N- or C-terminal extension as compared to the respective amino acid sequence; and/or c) the amylase is an allelic variant of an amylase having amino acids selected from the group consisting of (i) amino acids 1-481 of SEQ ID NO: 1, (ii) amino acids 1-481 of SEQ ID NO: 2, and/or (iii) amino acids 1-483 of SEQ ID NO: 3; and/or d) the amylase is a fragment of an amylase having amino acids selected from the group consisting of (i) amino acids 1-481 of SEQ ID NO: 1, (ii) amino acids 1-481 of SEQ ID NO: 2, and/or (iii) amino acids 1-483 of SEQ ID NO: 3.</div>
<div class="description-paragraph" num="p-0036">In particular, the present invention relates to an amylase for pharmaceutical use, wherein the amylase has an amino acid sequence selected from the group consisting of (i) amino acids 1-481, 1-484, 1-486, or 1-513 of SEQ ID NO: 1, (ii) amino acids 1-481 of SEQ ID NO: 2, and/or (iii) amino acids 1-483 of SEQ ID NO: 3.</div>
<div class="description-paragraph" num="p-0037">In still further particular embodiments of the invention, the amylase is derived from a microorganism, for example from a fungus, or from a bacterium. Examples of bacteria are strains of <i>Bacillus</i>, such as strains of <i>Bacillus amyloliquefaciens, Bacillus circulans, Bacillus halmapalus, Bacillus licheniformis, Bacillus megaterium, Bacillus </i>sp., <i>Bacillus stearothermophilus</i>, and <i>Bacillus subtilis</i>; preferably from strains of <i>Bacillus amyloliquefaciens, Bacillus circulans, Bacillus halmapalus, Bacillus licheniformis, Bacillus megaterium, Bacillus </i>sp., and <i>Bacillus stearothermophilus</i>; most preferably from strains of <i>Bacillus stearothermophilus, Bacillus licheniformis</i>, or <i>Bacillus </i>sp. In this context, the term “derived from” includes enzymes obtainable, or obtained, from wildtype strains; as well as, preferably, variants thereof having at least one substitution, insertion, and/or deletion of at least one amino acid residue. The term variant also includes shufflants, hybrids, chimeric enzymes and consensus enzymes. The variants may have been produced by any method known in the art, such as site-directed mutagenesis, random mutagenesis, consensus derivation processes (EP 897985), and gene shuffling (WO 95/22625, WO 96/00343), etc.</div>
<div class="description-paragraph" num="p-0038">Non-limiting examples of wildtype amylases of the invention are those derived from <i>Bacillus licheniformis</i>, such as Swissprot entry name AMY_BACLI, primary accession number P06278; <i>Bacillus amyloliquefaciens</i>, such as Swissprot entry name AMY_BACAM, primary accession number P00692<i>; Bacillus megaterium</i>, such as Swissprot entry name AMY_BACME, primary accession number P20845<i>; Bacillus circulans</i>, such as Swissprot entry name AMY_BACCI, primary accession number P08137<i>; Bacillus stearothermophilus</i>, such as Swissprot entry name AMY_BACST, primary accession number P06279. Another example is from <i>Bacillus subtilis</i>, such as Swissprot entry name AMY_BACSU, primary accession number P00691. A still further example is the amylase derived from <i>Bacillus halmapalus</i>, amino acids 1-485 of SEQ ID NO: 4. Non-limiting examples of amylase variants for use according to the invention are disclosed in WO 96/23873, WO 99/19467, U.S. Pat. No. 4,933,279, EP 722490, and EP 904360. In particular embodiments, the amylase of the invention (i) is not Swissprot entry name AMY_BACLI, primary accession number P06278; and/or (ii) is not Swissprot entry name AMY_BACST, primary accession number P06279.</div>
<div class="description-paragraph" num="p-0039">Further particular examples of amylases of the invention are the amylases contained in the following commercial products: Clarase, DexLo, GC 262 SP, G-Zyme G990, G-Zyme G995, G-Zyme G997, G-Zyme G998, HTAA, Optimax 7525, Purastar OxAm, Purastar ST, Spezyme AA, Spezyme Alpha, Spezyme BBA, Spezyme Delta AA, Spezyme DBA, Spezyme Ethyl, Spezyme Fred (GC521), Spezyme HPA, and Ultraphlow (all from Genencor); Validase BAA, Validase FAA, Validase HT340L, Valley Thin 340L (all from Valley Research); Avizyme 1500, Dextro 300L, Kleistase, Maltazyme, Maxamyl, Thermozyme, Thermatex, Starzyme HT 120 L, Starzyme Super Conc, and Ultraphlo.</div>
<div class="description-paragraph" num="p-0040">Particularly preferred examples of amylases of the invention are the following: An amylase comprising amino acids 1-481 of SEQ ID NO: 1, such as amylases having amino acids 1-481 of SEQ ID NO: 1, amino acids 1-484 of SEQ ID NO: 1, amino acids 1-486 of SEQ ID NO: 1, or amino acids 1-513 of SEQ ID NO: 1; an amylase having amino acids 1-481 of SEQ ID NO: 2; and an amylase having amino acids 1-483 of SEQ ID NO: 3.</div>
<div class="description-paragraph" num="p-0041">In still further particular embodiments, an additional amylase may be used. Examples of additional amylases are mammalian amylases, and microbial proteases. A preferred mammalian amylase is pancreas extract, e.g. from swine or ox, such as pancreatin. The pancreatin may be used in the form of an uncoated (raw) product, or in the form of a formulated product (enteric coated (to provide resistance against gastric acid), or non-functionally coated (coated, but not to provide resistance against gastric acid)). Pancreatin potentially comprises still further enzymatic active constituents like pancreatic lipase, BSSL (Bile Salt Stimulated Lipase), and/or pancreatic protease. The microbial amylase may, for example, derive from bacterial or fungal strains, such as <i>Bacillus, Pseudomonas, Aspergillus</i>, or <i>Rhizopus</i>. The amylase may in particular be derived from a strain of <i>Aspergillus</i>, such as <i>Aspergillus niger, Aspergillus oryzae </i>or <i>Aspergillus melleus</i>, for example either of the products Amylase A1™ derived from Aspergillus oryzae which is commercially available from Amano Pharmaceuticals, Japan, or Amylase EC™ derived from <i>Aspergillus melleus </i>which is commercially available from Extract-Chemie, Germany.</div>
<div class="description-paragraph" num="p-0042">The amylase of the invention may be used, with or without a lipase as described below, in combination with a protease. The term “protease” is defined herein as an enzyme that hydrolyses peptide bonds. It includes any enzyme belonging to the EC 3.4 enzyme group (including each of the thirteen subclasses thereof, these enzymes being in the following referred to as “belonging to the EC 3.4.-.-group”).</div>
<div class="description-paragraph" num="p-0043">The protease may be a mammalian proteases, or a microbial proteases. A preferred mammalian protease is pancreas extract, e.g. from swine or ox, such as pancreatin. The pancreatin may be used in the form of an uncoated (raw) product, or in the form of a formulated product (enteric coated, or non-functionally coated). Pancreatin potentially comprises still further enzymatic active constituents like pancreatic lipase, BSSL, and/or pancreatic amylase. The microbial protease may be, e.g., based on or derived from bacterial or fungal strains. The protease may in particular be derived from a strain of <i>Aspergillus</i>, such as <i>Aspergillus oryzae </i>or <i>Aspergillus melleus</i>, in particular the product Prozyme 6™ (neutral, alkaline protease EC 3.4.21.63) which is commercially available from Amano Pharmaceuticals, Japan. Examples of bacterial proteases are proteases from <i>Bacillus </i>and <i>Nocardiopsis</i>, such as the <i>Bacillus licheniformis </i>protease having the amino acid sequence of amino acids 1-274 of SEQ ID NO: 5, the <i>Nocardiopsis </i>sp. protease having the amino acid sequence of amino acids 1-188 of SEQ ID NO: 6, or the <i>Nocardiopsis dassonviellei </i>subsp. <i>dassonvillei </i>protease having the amino acid sequence of amino acids 1-188 of SEQ ID NO: 7. The protease of amino acids 1-274 of SEQ ID NO: 5 may, e.g., be prepared as described in DK patent application no. 2005 00930 entitled “Proteases for Pharmaceutical Use” and filed on Jun. 24, 2005 by Solvay Pharmaceuticals GmbH and Novozymes A/S. The proteases of amino acids 1-188 of SEQ ID NO: 6-7 may, e.g., be prepared as described in WO 2001/58276, or in WO 2004/111224.</div>
<div class="description-paragraph" num="p-0044">In a preferred embodiment, the protease is at least 70% identical to either of (i) amino acids 1-274 of SEQ ID NO: 5, (ii) amino acids 1-188 of SEQ ID NO: 6, and/or (iii) amino acids 1-188 of SEQ ID NO: 7. In additional preferred embodiments of either of (i), (ii) or (iii), the degrees of identity is at least 71%, 72%, 73%, 74%, 75%, 76%, 77%, 78%, 79%, 80%, 81%, 82%, 83%, 84%, 85%, 86%, 87%, 88%, 89%, 90%, 91%, 92%, 93%, 94%, 95%, 96%, 97%, 98%, or at least 99%. In alternative embodiments of either of (i), (ii), or (iii), the degrees of identity is at least about 50%, 51%, 52%, 53%, 54%, 55%, 56%, 57%, 58%, 59%, 60%, 61%, 62%, 63%, 64%, 65%, 66%, 67%, 68%, or at least 69%.</div>
<div class="description-paragraph" num="p-0045">The amylase of the invention may be used, with or without a protease as mentioned above, in combination with a lipase. In the present context, a lipase means a carboxylic ester hydrolase EC 3.1.1.-, which includes activities such as EC 3.1.1.3 triacylglycerol lipase, EC 3.1.1.4 phospholipase A1, EC 3.1.1.5 lysophospholipase, EC 3.1.1.26 galactolipase, EC 3.1.1.32 phospholipase A1, EC 3.1.1.73 feruloyl esterase. In a particular embodiment, the lipase is an EC 3.1.1.3 triacylglycerol lipase.</div>
<div class="description-paragraph" num="p-0046">The lipase may be a mammalian lipase, or a microbial lipase. A preferred mammalian lipase is pancreas extract, e.g. from swine or ox, such as pancreatin. The pancreatin may be used in the form of an uncoated (raw) product, or in the form of a formulated product (enteric coated, or non-functionally coated). Pancreatin potentially comprises still further enzymatic active constituents like pancreatic protease, BSSL, and/or pancreatic amylase. The microbial lipase may, for example, be derived from bacterial or fungal strains, such as <i>Bacillus, Pseudomonas, Aspergillus</i>, or <i>Rhizopus</i>. The lipase may in particular be derived from a strain of <i>Rhizopus</i>, such as <i>Rhizopus javanicus, Rhizopus oryzae</i>, or <i>Rhizopus delemar</i>, for example the product Lipase D Amano 2000™ (also designated Lipase D2™) which is commercially available from Amano Pharmaceuticals, Japan.</div>
<div class="description-paragraph" num="p-0047">In further particular embodiments, the lipase is a recombinantly produced microbial lipase, for example derived from a fungus such as <i>Humicola </i>or <i>Rhizomucor</i>, from a yeast such as <i>Candida</i>, or from a bacterium such as <i>Pseudomonas</i>. In a preferred embodiment, the lipase is derived from a strain of <i>Humicola lanuginosa </i>or <i>Rhizomucor miehei. </i> </div>
<div class="description-paragraph" num="p-0048">The <i>Humicola lanuginosa </i>(synonym <i>Thermomyces lanuginosus</i>) lipase (SEQ ID NO: 9) is described in EP 305216, and particular lipase variants are described in, for example, WO 92/05249, WO 92/19726, WO 94/25577, WO 95/22615, WO 97/04079, WO 97/07202, WO 99/42566, WO 00/32758, WO 00/60063, WO 01/83770, WO 02/055679, and WO 02/066622. A preferred lipase variant is a lipase comprising amino acids 1-269, or 2-269, of SEQ ID NO: 8, such as the following: (i) amino acids +1 to +269 of SEQ ID NO: 8, (ii) amino acids −5 to +269 of SEQ ID NO: 8, (iii) amino acids −4 to +269 of SEQ ID NO: 8; (iv) amino acids −3 to +269 of SEQ ID NO: 8; (v) amino acids −2 to +269 of SEQ ID NO: 8; (vi) amino acids −1 to +269 of SEQ ID NO: 8, (vii) amino acids +2 to +269 of SEQ ID NO: 1, as well as (viii) any mixture of two or more of the lipases of (i)-(vii)—and variants thereof. In a particular embodiment, the lipase for use according to the invention is selected from the lipases of (i), (ii), and any mixture of (i) and (ii). Preferred mixtures of (i) and (ii) comprise at least 5%, preferably at least 10%, 20%, 30%, 40%, 50%, 60%, 70%, 80%, 90%, or at least 95% of lipase (i), the percentages being determined by N-terminal sequencing using the Edman method, as described in Example 5 of the PCT application that claims priority from DK patent application no. 2005 00929. Other preferred mixtures are: (a) compositions comprising 35-75%, preferably 40-70%, more preferably 45-65% of lipase (ii); (b) compositions comprising 20-60%, preferably 25-55%, more preferably 30-50%, most preferably 35-47% of lipase (i); (c) compositions comprising up to 30%, preferably up to 25%, more preferably up to 20%, most preferably up to 16% of lipase (vii); and (d) any combination of (a), (b), and/or (c), such as a composition comprising 45-65% of lipase (ii), 35-47% of lipase (i), and up to 16% of lipase (vii).</div>
<div class="description-paragraph" num="p-0049">The lipases of SEQ ID NO: 8 and 9 may, e.g., be prepared on the basis of the basis of the teaching in U.S. Pat. No. 5,869,438 (in which SEQ ID NO: 1 is a DNA sequence encoding the lipase of SEQ ID NO: 9). The lipase of SEQ ID NO: 8 can, e.g., be prepared by recombinant expression in a suitable host cell of a DNA sequence which is a modification of SEQ ID NO:1 of the US patent, the modification reflecting the amino acid differences between SEQ ID NO: 8 and 9 herein. Such modifications can be made by site-directed mutagenesis, as is known in the art.</div>
<div class="description-paragraph" num="p-0050">Still further examples of fungal lipases are the cutinase from <i>Humicola insolens </i>which is described in EP 785994, and the phospholipase from <i>Fusarium oxysporum </i>which is described in EP 869167. Examples of yeast lipases are lipase A and B from <i>Candida antarctica </i>of which lipase A is described in EP 652945, and lipase B is described by, for example, Uppenberg et al in Structure, 2 (1994), 293. An example of a bacterial lipase is the lipase derived from <i>Pseudomonas cepacia</i>, which is described in EP 214761.</div>
<div class="description-paragraph" num="p-0051">In a preferred embodiment, the lipase is at least 70% identical to the lipase of SEQ ID NO: 8, or to amino acids 1-269 thereof. In additional preferred embodiments, the degree of identity to SEQ ID NO: 8, or to amino acids 1-269 thereof, is at least 71%, 72%, 73%, 74%, 75%, 76%, 77%, 78%, 79%, 80%, 81%, 82%, 83%, 84%, 85%, 86%, 87%, 88%, 89%, 90%, 91%, 92%, 93%, 94%, 95%, 96%, 97%, 98%, or at least 99%. In alternative embodiments, the degree of identity to SEQ ID NO: 8, or to amino acids 1-269 thereof, is at least about 50%, 51%, 52%, 53%, 54%, 55%, 56%, 57%, 58%, 59%, 60%, 61%, 62%, 63%, 64%, 65%, 66%, 67%, 68%, or at least 69%.</div>
<div class="description-paragraph" num="p-0052">In a still further preferred embodiment, the lipase, like the mammalian pancreatic lipase, is a 1,3-position specific lipase.</div>
<div class="description-paragraph" num="p-0053">For the purposes of the present invention particularly preferred combinations of enzymes are the following: (i) The protease of amino acids 1-274 of SEQ ID NO: 5 in combination with an amylase comprising amino acids 1-481 of SEQ ID NO: 1 (such as amino acids 1-481, 1-484, 1-486, or 1-513 thereof); (ii) the protease of amino acids 1-274 of SEQ ID NO: 5 in combination with the amylase of SEQ ID NO: 2; (iii) the protease of amino acids 1-274 of SEQ ID NO: 5 in combination with the amylase of SEQ ID NO: 3; (iv) the protease of amino acids 1-188 of SEQ ID NO: 6 in combination with an amylase comprising amino acids 1-481 of SEQ ID NO: 1 (such as amino acids 1-481, 1-484, 1-486, or 1-513 thereof); (v) the protease of amino acids 1-188 of SEQ ID NO: 6 in combination with the amylase of SEQ ID NO: 2; (vi) the protease of amino acids 1-188 of SEQ ID NO: 6 in combination with the amylase of SEQ ID NO: 3; (vii) a lipase comprising amino acids 1-269, or 2-269, of SEQ ID NO: 8 in combination with an amylase comprising amino acids 1-481 of SEQ ID NO: 1 (such as amino acids 1-481, 1-484, 1-486, or 1-513 thereof); (viii) a lipase comprising amino acids 1-269, or 2-269, of SEQ ID NO: 8 in combination with the amylase of SEQ ID NO: 2; (ix) a lipase comprising amino acids 1-269, or 2-269, of SEQ ID NO: 8 in combination with the amylase of SEQ ID NO: 3; (x) the protease of amino acids 1-274 of SEQ ID NO: 5 in combination with an amylase comprising amino acids 1-481 of SEQ ID NO: 1 (such as amino acids 1-481, 1-484, 1-486, or 1-513 thereof) and a lipase comprising amino acids 1-269, or 2-269, of SEQ ID NO: 8; (xi) the protease of amino acids 1-274 of SEQ ID NO: 5 in combination with the amylase of SEQ ID NO: 2 and a lipase comprising amino acids 1-269, or 2-269, of SEQ ID NO: 8; (xii) the protease of amino acids 1-274 of SEQ ID NO: 5 in combination with the amylase of SEQ ID NO: 3 and a lipase comprising amino acids 1-269, or 2-269, of SEQ ID NO: 8); (xiii) the protease of amino acids 1-188 of SEQ ID NO: 6 in combination with an amylase comprising amino acids 1-481 of SEQ ID NO: 1 (such as amino acids 1-481, 1-484, 1-486, or 1-513 thereof) and a lipase comprising amino acids 1-269, or 2-269, of SEQ ID NO: 8; (xiv) the protease of amino acids 1-188 of SEQ ID NO: 6 the amylase of SEQ ID NO: 2 and a lipase comprising amino acids 1-269, or 2-269, of SEQ ID NO: 8; (xv) the protease of amino acids 1-188 of SEQ ID NO: 6 in combination with the amylase of SEQ ID NO: 3 and a lipase comprising amino acids 1-269, or 2-269, of SEQ ID NO: 8; (xvi) a protease having amino acids 1-188 of SEQ ID NO: 7 in combination with an amylase comprising amino acids 1-481 of SEQ ID NO: 1 (such as amino acids 1-481, 1-484, 1-486, or 1-513 thereof); (xvii) a protease having amino acids 1-188 of SEQ ID NO: 7 in combination with an amylase having amino acids 1-481 of SEQ ID NO: 2; (xviii) a protease having amino acids 1-188 of SEQ ID NO: 7 in combination with an amylase having amino acids 1-483 of SEQ ID NO: 3; (xix) a protease having amino acids 1-188 of SEQ ID NO: 7 in combination with an amylase comprising amino acids 1-481 of SEQ ID NO: 1 (such as amino acids 1-481, 1-484, 1-486, or 1-513 thereof) and a lipase comprising amino acids 1-269, or 2-269, of SEQ ID NO: 8; (xx) a protease having amino acids 1-188 of SEQ ID NO: 7 in combination with an amylase having amino acids 1-481 of SEQ ID NO: 2 and a lipase comprising amino acids 1-269, or 2-269, of SEQ ID NO: 8; and (xi) a protease having amino acids 1-188 of SEQ ID NO: 7 in combination with an amylase having amino acids 1-483 of SEQ ID NO: 3 and a lipase comprising amino acids 1-269, or 2-269, of SEQ ID NO: 8.</div>
<div class="description-paragraph" num="p-0054">Accordingly, one embodiment of the present invention relates to an amylase in combination with a lipase and/or a protease according to claim <b>4</b> or <b>5</b>, wherein the amylase is an amylase selected from the group consisting of a) an amylase comprising amino acids 1-481 of SEQ ID NO: 1, b) an amylase having amino acids 1-481 of SEQ ID NO: 2, and c) an amylase having amino acids 1-483 of SEQ ID NO: 3; (ii) the lipase comprises amino acids 2-269 of SEQ ID NO: 8; (iii) the protease is a protease selected from the group consisting of a) a protease having amino acids 1-274 of SEQ ID NO: 5, b) a protease having amino acids 1-188 of SEQ ID NO: 6, and c) a protease having amino acids 1-188 of SEQ ID NO: 7.</div>
<div class="description-paragraph" num="p-0055">Other preferred combinations of enzymes are the following: (i) A protease having at least 50% identity to amino acids 1-274 of SEQ ID NO: 5 in combination with an amylase having at least 50% identity to amino acids 1-481 of SEQ ID NO: 1; (ii) a protease having at least 50% identity to amino acids 1-274 of SEQ ID NO: 5 in combination with an amylase having at least 50% identity to amino acids 1-481 of SEQ ID NO: 2; (iii) a protease having at least 50% identity to amino acids 1-274 of SEQ ID NO: 5 in combination with an amylase having at least 50% identity to amino acids 1-483 of SEQ ID NO: 3; (iv) a protease having at least 50% identity to amino acids 1-188 of SEQ ID NO: 6 in combination with an amylase having at least 50% identity to amino acids 1-481 of SEQ ID NO: 1; (v) a protease having at least 50% identity to amino acids 1-188 of SEQ ID NO: 6 in combination with an amylase having at least 50% identity to amino acids 1-481 of SEQ ID NO: 2; (vi) a protease having at least 50% identity to amino acids 1-188 of SEQ ID NO: 6 in combination with an amylase having at least 50% identity to amino acids 1-483 of SEQ ID NO: 3; (vii) a lipase having at least 50% identity to amino acids 1-269 of SEQ ID NO: 8 in combination with an amylase having at least 50% identity to amino acids 1-481 of SEQ ID NO: 1; (viii) a lipase having at least 50% identity to amino acids 1-269 of SEQ ID NO: 8 in combination with an amylase having at least 50% identity to SEQ ID NO: 2; (ix) a lipase having at least 50% identity to amino acids 1-269 of SEQ ID NO: 8 in combination with an amylase having at least 50% identity to amino acids 1-483 of SEQ ID NO: 3; (x) a protease having at least 50% identity to amino acids 1-274 of SEQ ID NO: 5 in combination with an amylase having at least 50% identity to amino acids 1-481 of SEQ ID NO: 1 and a lipase having at least 50% identity to amino acids 1-269 of SEQ ID NO: 8; (xi) a protease having at least 50% identity to amino acids 1-274 of SEQ ID NO: 5 in combination with an amylase having at least 50% identity to amino acids 1-481 of SEQ ID NO: 2 and a lipase having at least 50% identity to amino acids 1-269 of SEQ ID NO: 8; (xii) a protease having at least 50% identity to amino acids 1-274 of SEQ ID NO: 5 in combination with an amylase having at least 50% identity to amino acids 1-483 of SEQ ID NO: 3 and a lipase having at least 50% identity to amino acids 1-269 of SEQ ID NO: 8); (xiii) a protease having at least 50% identity to amino acids 1-188 of SEQ ID NO: 6 in combination with an amylase having at least 50% identity to amino acids 1-481 of SEQ ID NO: 1 and a lipase having at least 50% identity to amino acids 1-269 of SEQ ID NO: 8; (xiv) a protease having at least 50% identity to amino acids 1-188 of SEQ ID NO: 6 in combination with an amylase having at least 50% identity to amino acids 1-481 of SEQ ID NO: 2 and a lipase having at least 50% identity to amino acids 1-269 of SEQ ID NO: 8; (xv) a protease having at least 50% identity to amino acids 1-188 of SEQ ID NO: 6 in combination with an amylase having at least 50% identity to amino acids 1-483 of SEQ ID NO: 3 and a lipase having at least 50% identity to amino acids 1-269 of SEQ ID NO: 8; (xvi) a protease having at least 50% identity to amino acids 1-188 of SEQ ID NO: 7 in combination with an amylase having at least 50% identity to amino acids 1-481 of SEQ ID NO: 1; (xvii) a protease having at least 50% identity to amino acids 1-188 of SEQ ID NO: 7 in combination with an amylase having at least 50% identity to amino acids 1-481 of SEQ ID NO: 2; (xviii) a protease having at least 50% identity to amino acids 1-188 of SEQ ID NO: 7 in combination with an amylase having at least 50% identity to amino acids 1-483 of SEQ ID NO: 3; (xix) a protease having at least 50% identity to amino acids 1-188 of SEQ ID NO: 7 in combination with an amylase having at least 50% identity to amino acids 1-481 of SEQ ID NO: 1 and a lipase having at least 50% identity to amino acids 1-269 of SEQ ID NO: 8; (xx) a protease having at least 50% identity to amino acids 1-188 of SEQ ID NO: 7 in combination with an amylase having at least 50% identity to amino acids 1-481 of SEQ ID NO: 2 and a lipase having at least 50% identity to amino acids 1-269 of SEQ ID NO: 8; and (xi) a protease having at least 50% identity to amino acids 1-188 of SEQ ID NO: 7 in combination with an amylase having at least 50% identity to amino acids 1-483 of SEQ ID NO: 3 and a lipase having at least 50% identity to amino acids 1-269 of SEQ ID NO: 8. In preferred embodiments of (i)-(xi), each degree of identity is, independently, at least 51%, 52%, 53%, 54%, 55%, 56%, 57%, 58%, 59%, 60%, 61%, 62%, 63%, 64%, 65%, 66%, 67%, 68%, 69%, 70%, 71%, 72%, 73%, 74%, 75%, 76%, 77%, 78%, 79%, 80%, 81%, 82%, 83%, 84%, 85%, 86%, 87%, 88%, 89%, 90%, 91%, 92%, 93%, 94%, 95%, 96%, 97%, 98%, or at least 99%.</div>
<div class="description-paragraph" num="p-0056">Accordingly, one embodiment of the invention relates to an amylase in combination with a lipase and/or a protease according to claim <b>4</b> or <b>5</b>, wherein (i) the amylase is an amylase as defined herein; (ii) the lipase has at least 70% identity to a lipase having amino acids 1-269 of SEQ ID NO: 8; (ii) the protease has at least 70% identity to a protease selected from the group consisting of a) a protease having amino acids 1-274 of SEQ ID NO: 5, b) a protease having amino acids 1-188 of SEQ ID NO: 6, and c) a protease having amino acids 1-188 of SEQ ID NO: 7.</div>
<div class="description-paragraph" num="p-0057">Generally, the amylase, protease, and lipase enzymes (hereinafter “the enzyme(s)”) of the invention may be natural or wild-type enzymes obtained from animals, in particular mammals, for example human or swine enzymes; from plants, or from microorganisms, but also any mutants, variants, fragments etc. thereof exhibiting the desired enzyme activity, as well as synthetic enzymes, such as shuffled, hybrid, or chimeric enzymes, and consensus enzymes.</div>
<div class="description-paragraph" num="p-0058">In a specific embodiment, the enzyme(s) are low-allergenic variants, designed to invoke a reduced immunological response when exposed to animals, including man. The term immunological response is to be understood as any reaction by the immune system of an animal exposed to the enzyme(s). One type of immunological response is an allergic response leading to increased levels of IgE in the exposed animal. Low-allergenic variants may be prepared using techniques known in the art. For example the enzyme(s) may be conjugated with polymer moieties shielding portions or epitopes of the enzyme(s) involved in an immunological response. Conjugation with polymers may involve in vitro chemical coupling of polymer to the enzyme(s), e.g. as described in WO 96/17929, WO 98/30682, WO 98/35026, and/or WO 99/00489. Conjugation may in addition or alternatively thereto involve in vivo coupling of polymers to the enzyme(s). Such conjugation may be achieved by genetic engineering of the nucleotide sequence encoding the enzyme(s), inserting consensus sequences encoding additional glycosylation sites in the enzyme(s) and expressing the enzyme(s) in a host capable of glycosylating the enzyme(s), see e.g. WO 00/26354. Another way of providing low-allergenic variants is genetic engineering of the nucleotide sequence encoding the enzyme(s) so as to cause the enzymes to self-oligomerize, effecting that enzyme monomers may shield the epitopes of other enzyme monomers and thereby lowering the antigenicity of the oligomers. Such products and their preparation is described e.g. in WO 96/16177. Epitopes involved in an immunological response may be identified by various methods such as the phage display method described in WO 00/26230 and WO 01/83559, or the random approach described in EP 561907. Once an epitope has been identified, its amino acid sequence may be altered to produce altered immunological properties of the enzyme(s) by known gene manipulation techniques such as site directed mutagenesis (see e.g. WO 00/26230, WO 00/26354 and/or WO 00/22103) and/or conjugation of a polymer may be done in sufficient proximity to the epitope for the polymer to shield the epitope.</div>
<div class="description-paragraph" num="p-0059">In particular embodiments, the enzyme(s) are (i) stable at pH 2-8, preferably also at pH 3-7, more preferably at pH 4-6; (ii) active at pH 4-9, preferably 4-8; (iii) stable against degradation by pepsin and other digestive proteases (such as pancreas proteases, i.e., mainly trypsin and chymotrypsin); and/or (iv) stable and/or active in the presence of bile salts.</div>
<div class="description-paragraph" num="p-0060">In a preferred embodiment, the amylase of the invention has an activity at pH 7.0 and 37° C. of at least 35% relative to the activity at the pH-optimum and 37° C. More preferably, the activity at pH 7.0 and 37° C. is at least 40, 45, 50, 55, 60, 65, 70, or at least 75% of the activity at the pH-optimum and 37° C. (cf. Table 2 of Example 3).</div>
<div class="description-paragraph" num="p-0061">In another preferred embodiment, the amylase of the invention has an activity at pH 7.0 and 37° C. and in the presence of 5 mM bile salts of at least 25% relative to the activity at the pH-optimum and 37° C. in the absence of bile salts. More preferably, the activity at pH 7.0 and 37° C. and in the presence of 5 mM bile salts is at least 30, 35, 40, 45, 50, 55, 60, or at least 65% of the activity at the pH-optimum and 37° C. in the absence of bile salts (cf. Table 3 of Example 3)</div>
<div class="description-paragraph" num="p-0062">In a still further preferred embodiment, the specific activity of the amylase of the invention, at pH 7.0 and 37° C., is at least 10%, more preferably at least 15, 20, 25, 30, 35, 40, 45, 50, 55, 60, 65, or at least 70%, relative to the specific activity of the amylase of SEQ ID NO: 1 at pH 5.0 and 37° C. (cf. Table 4 of Example 3).</div>
<div class="description-paragraph" num="p-0063">In another preferred embodiment, the specific activity of the amylase of the invention, at pH 7.0 and 37° C. and in the presence of 5 mM bile salts, is at least 10%, more preferably at least 15, 20, 25, 30, 35, 40, 45, 50, 55, 60, 65, 70, or at least 75%, relative to the specific activity of the amylase of SEQ ID NO: 1 at pH 5.0 and 37° C. and in the presence of 5 mM bile salts (cf. Table 5 of Example 3).</div>
<div class="description-paragraph" num="p-0064">The activities referred to in the above preferred embodiments may suitably be determined using a reducing sugar assay, e.g. as described in Example 3, using preferably waxy corn as a substrate. A detailed procedure is described in Example 3.</div>
<div class="description-paragraph" num="p-0065">In a further, independent, particular embodiment, the amylase of the invention has a starch degradation profile with one or more of the following characteristic features: (i) DP1 is detected as a degradation product; (i) the DP4 amount is lower than the amount of DP5; (iii) the DP4 amount is lower than the amount of DP6; and/or (iv) the amount of DP5 and/or DP6 is higher than the amount of DP1. The monomer DP1 is preferably glucose, and the DP2-DP6 products are dimers-hexamers, respectively, of glucose.</div>
<div class="description-paragraph" num="p-0066">The degradation profile may be determined using HPLC as described in Example 4, i.e. after having incubated for 24 hours at a temperature of 60 or 37° C., at a pH of 6.0 or 5.5, preferably using waxy corn as a substrate, more preferably in the presence of Ca<sup>2+</sup>.</div>
<div class="description-paragraph" num="p-0067">The term “in combination with” refers to the combined use according to the invention of the protease, lipase, and/or amylase. The combined use can be simultaneous, overlapping, or sequential, these three terms being generally interpreted in the light of the prescription made by the physician.</div>
<div class="description-paragraph" num="p-0068">The term “simultaneous” refers to circumstances under which the enzymes are active at the same time, for example when they are administered at the same time as one or more separate pharmaceutical products, or if they are administered in one and the same pharmaceutical composition.</div>
<div class="description-paragraph" num="p-0069">The term “sequential” refers to such instances where one and/or two of the enzymes are acting first, and the second and/or third enzyme subsequently. A sequential action can be obtained by administering the enzymes in question as separate pharmaceutical formulations with desired intervals, or as one pharmaceutical composition in which the enzymes in question are differently formulated (compartmentalized), for example with a view to obtaining a different release time, providing an improved product stability, or to optimizing the enzyme dosage.</div>
<div class="description-paragraph" num="p-0070">The term “overlapping” refers to such instances where the enzyme activity periods are neither completely simultaneous nor completely sequential, viz. there is a certain period in which the enzymes are both, or all, active.</div>
<div class="description-paragraph" num="p-0071">The term “a”, for example when used in the context of the enzyme(s) of the invention, means at least one. In particular embodiments, “a” means “one or more,” or “at least one”, which again means one, two, three, four, five etc.</div>
<div class="description-paragraph" num="p-0072">The relatedness between two amino acid sequences is described by the parameter “identity”.</div>
<div class="description-paragraph" num="p-0073">For purposes of the present invention, the alignment of two amino acid sequences is determined by using the Needle program from the EMBOSS package (http://emboss.org) version 2.8.0. The Needle program implements the global alignment algorithm described in Needleman, S. B. and Wunsch, C. D. (1970) J. Mol. Biol. 48, 443-453. The substitution matrix used is BLOSUM62, gap opening penalty is 10, and gap extension penalty is 0.5.</div>
<div class="description-paragraph" num="p-0074">The degree of identity between an amino acid sequence of the present invention (“invention sequence”; e.g. amino acids 1-481 of SEQ ID NO: 1) and a different amino acid sequence (“foreign sequence”; e.g. amino acids 1-514 of SEQ ID NO: 12) is calculated as the number of exact matches in an alignment of the two sequences, divided by the length of the “invention sequence” or the length of the “foreign sequence”, whichever is the shortest. The result is expressed in percent identity.</div>
<div class="description-paragraph" num="p-0075">An exact match occurs when the “invention sequence” and the “foreign sequence” have identical amino acid residues in the same positions of the overlap (in the alignment example below this is represented by “|”). The length of a sequence is the number of amino acid residues in the sequence (e.g. the length of SEQ ID NO: 1 is 481).</div>
<div class="description-paragraph" num="p-0076">In the, purely hypothetical, alignment example below, the overlap is the amino acid sequence “HTWGER-NL” of Sequence 1; or the amino acid sequence “HGWGEDANL” of Sequence 2. In the example a gap is indicated by a “-”.</div>
<div class="description-paragraph" num="h-0008">Hypothetical Alignment Example:</div>
<div class="description-paragraph" num="p-0077">
<chemistry id="CHEM-US-00001" num="00001">
<div class="patent-image"><a href="https://patentimages.storage.googleapis.com/29/07/17/4a77b1731c201b/US08017351-20110913-C00001.png"><img alt="Figure US08017351-20110913-C00001" class="patent-full-image" file="US08017351-20110913-C00001.TIF" he="8.47mm" height="34" id="EMI-C00001" img-content="chem" img-format="tif" inline="no" orientation="portrait" src="https://patentimages.storage.googleapis.com/29/07/17/4a77b1731c201b/US08017351-20110913-C00001.png" wi="50.80mm" width="203"/></a></div>
<attachments>
<attachment attachment-type="cdx" file="US08017351-20110913-C00001.CDX" idref="CHEM-US-00001"> </attachment>
<attachment attachment-type="mol" file="US08017351-20110913-C00001.MOL" idref="CHEM-US-00001"> </attachment>
</attachments>
</chemistry>
</div>
<div class="description-paragraph" num="p-0078">Accordingly, the percentage of identity of Sequence 1 to Sequence 2 is 6/12=0.5, corresponding to 50%.</div>
<div class="description-paragraph" num="p-0079">In a particular embodiment, the percentage of identity of an amino acid sequence of a polypeptide with, or to, amino acids 1-481 of SEQ ID NO: 1 is determined by i) aligning the two amino acid sequences using the Needle program, with the BLOSUM62 substitution matrix, a gap opening penalty of 10, and a gap extension penalty of 0.5; ii) counting the number of exact matches in the alignment; iii) dividing the number of exact matches by the length of the shortest of the two amino acid sequences, and iv) converting the result of the division of iii) into percentage. The percentage of identity to, or with, other sequences of the invention such as amino acids 1-481 of SEQ ID NO: 2, or 1-483 of SEQ ID NO: 3, is calculated in an analogous way.</div>
<div class="description-paragraph" num="p-0080">In the alternative, the degree of identity between two amino acid sequences may be determined by the program “align” which is a Needleman-Wunsch alignment (i.e. a global alignment). The sequences are aligned by the program, using the default scoring matrix BLOSUM50. The penalty for the first residue of a gap is 12, and for further residues of a gap the penalties are 2. The Needleman-Wunsch algorithm is described in Needleman, S. B. and Wunsch, C. D., (1970), Journal of Molecular Biology, 48: 443-453, and the align program by Myers and W. Miller in “Optimal Alignments in Linear Space” CABIOS (computer applications in the biosciences) (1988) 4:11-17. “Align” is part of the FASTA package version v20u6 (see W. R. Pearson and D. J. Lipman (1988), “Improved Tools for Biological Sequence Analysis”, PNAS 85:2444-2448, and W. R. Pearson (1990) “Rapid and Sensitive Sequence Comparison with FASTP and FASTA,” Methods in Enzymology 183:63-98).</div>
<div class="description-paragraph" num="p-0081">The degree of identity between a sample, or test, sequence of any of the enzyme(s) of the invention and a specified sequence may be determined as follows: The two sequences are aligned using the program “align.” The number of perfect matches (“N-perfect-match”) in the alignment is determined (a perfect match means same amino acid residue in same position of the alignment). The common length of the two aligned sequences is also determined, viz. the total number of amino acids in the alignment (the overlap), including trailing and leading gaps created by the alignment, if any (“N-overlap”). The degree of identity is calculated as the ratio between “N-perfect-match” and “N-overlap” (for conversion to percentage identity, multiply by 100).</div>
<div class="description-paragraph" num="p-0082">In an alternative embodiment, the degree of identity between the sample, or test, sequence and a specified sequence may be determined as follows: The sequences are aligned using the program “align.” The number of perfect matches (“N-perfect-match”) in the alignment is determined (a perfect match means same amino acid residue in same position of the alignment). The length of the sample sequence (the number of amino acid residues) is determined (“N-sample”). The degree of identity is calculated as the ratio between “N-perfect-match” and “N-sample” (for conversion to percentage identity, multiply by 100).</div>
<div class="description-paragraph" num="p-0083">In another alternative embodiment, the degree of identity between the sample, or test, sequence and a specified sequence may be determined as follows: The sequences are aligned using the program “align.” The number of perfect matches (“N-perfect-match”) in the alignment is determined (a perfect match means same amino acid residue in same position of the alignment). The length of the specified sequence (the number of amino acid residues) is determined (“N-specified”). The degree of identity is calculated as the ratio between “N-perfect-match” and “N-specified” (for conversion to percentage identity, multiply by 100).</div>
<div class="description-paragraph" num="p-0084">Preferably, the overlap is at least 20% of the specified sequence (“N-overlap” as defined above, divided by the number of the amino acids in the specified sequence (“N-specified”), and multiplied by 100), more preferably at least 25%, 30%, 35%, 40%, 45%, 50%, 55%, 60%, 65%, 70%, 75%, 80%, 85%, 90%, or at least 95%. This means that at least 20% (preferably 25-95%) of the amino acids of the specified sequence end up being included in the overlap, when the sample sequence is aligned to the specified sequence.</div>
<div class="description-paragraph" num="p-0085">In the alternative, the overlap is at least 20% of the specified sequence (“N-overlap” as defined above, divided by “N-sample” as defined above, and multiplied by 100), more preferably at least 25%, 30%, 35%, 40%, 45%, 50%, 55%, 60%, 65%, 70%, 75%, 80%, 85%, 90%, or at least 95%. This means that at least 20% (preferably 25-95%) of the amino acids of the sample sequence end up being included in the overlap, when aligned against the specified sequence.</div>
<div class="description-paragraph" num="p-0086">The activity of the enzyme(s) of the invention can be measured using any suitable assay. Generally, assay-pH and assay-temperature may be adapted to the enzyme in question. Examples of assay-pH-values are pH 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, or 12. Examples of assay-temperatures are 30, 35, 37, 40, 45, 50, 55, 60, 65, 70, 80, 90, or 95° C. Preferred pH values and temperatures are in the physiological range, such as pH values of 4, 5, 6, 7, or 8, and temperatures of 30, 35, 37, or 40° C.</div>
<div class="description-paragraph" num="p-0087">Suitable assays are described in the experimental part, and in DK patent application no. 2005 00929 and 2005 00930 as well as the corresponding PCT-applications.</div>
<div class="description-paragraph" num="p-0088">Other examples are the Ph. Eur. assays for protease, lipase and amylase activity.</div>
<div class="description-paragraph" num="h-0009">Medicament</div>
<div class="description-paragraph" num="p-0089">In the present context, the term “medicament” means a compound, or mixture of compounds, that treats, prevents and/or alleviates the symptoms of disease, preferably treats and/or alleviates. The medicament may be prescribed by a physician, or it may be an over-the-counter product.</div>
<div class="description-paragraph" num="h-0010">Pharmaceutical Compositions</div>
<div class="description-paragraph" num="p-0090">Isolation, purification, and concentration of the enzyme(s) of the invention may be carried out by conventional means. For example, they may be recovered from a fermentation broth by conventional procedures including, but not limited to, centrifugation, filtration, extraction, spray-drying, evaporation, or precipitation, and further purified by a variety of procedures known in the art including, but not limited to, chromatography (e.g., ion exchange, affinity, hydrophobic, chromatofocusing, and size exclusion), electrophoretic procedures (e.g., preparative isoelectric focusing), differential solubility (e.g., ammonium sulphate precipitation), SDS-PAGE, or extraction (see, e.g., Protein Purification, J.-C. Janson and Lars Ryden, editors, VCH Publishers, New York, 1989).</div>
<div class="description-paragraph" num="p-0091">For example, DNA encoding the amylase of the invention (such as SEQ ID NOs: 13-16 encoding SEQ ID NOs: 1-4, respectively) may be recombinantly expressed in a <i>Bacillus </i>host cell as is known in the art, and purified from the fermentation liquid by process steps like centrifugation, filtration, ultrafiltration and, if desired, germ-filtration. Additional, optional, process steps include various kinds of chromatography like, e.g., ion-exchange chromatography and hydrophobic interaction chromatography. A suitable hydrophobic column material is butyl or phenyl, and such columns are commercially available, e.g. from Pharmacia. The pI of the amylases of SEQ ID NO: 1, 2, and 3 are 5.7, 7.3, and 8.8, respectively, and the molecular weights approximately 55 kDa by SDS-PAGE for all three amylases.</div>
<div class="description-paragraph" num="p-0092">In a particular embodiment, concentrated solid or liquid preparations of each of the enzyme(s) are prepared separately. These concentrates may also, at least in part, be separately formulated, as explained in more detail below.</div>
<div class="description-paragraph" num="p-0093">In a further particular embodiment, the enzyme(s) are incorporated in the pharmaceutical compositions of the invention in the form of solid concentrates. The enzyme(s) can be brought into the solid state by various methods as is known in the art. For example, the solid state can be either crystalline, where the enzyme molecules are arranged in a highly ordered form, or a precipitate, where the enzyme molecules are arranged in a less ordered, or disordered, form.</div>
<div class="description-paragraph" num="p-0094">Crystallization may, for example, be carried out at a pH close to the pI of the enzyme(s) and at low conductivity, for example 10 mS/cm or less, as described in EP 691982.</div>
<div class="description-paragraph" num="p-0095">Various precipitation methods are known in the art, including precipitation with salts, such as ammonium sulphate, and/or sodium sulphate; with organic solvents, such as ethanol, and/or isopropanol; or with polymers, such as PEG (Poly Ethylene Glycol). In the alternative, the enzyme(s) can be precipitated from a solution by removing the solvent (typically water) by various methods known in the art, e.g. lyophilization, evaporation (for example at reduced pressure), and/or spray drying.</div>
<div class="description-paragraph" num="p-0096">In a further particular embodiment, the solid concentrate of the enzyme(s) has a content of active enzyme protein of at least 50% (w/w) by reference to the total protein content of the solid concentrate. In still further particular embodiments, the content of active enzyme protein, relative to the total protein content of the solid concentrate is at least 55, 60, 65, 70, 75, 80, 85, 90, or at least 95% (w/w). The protein content can be measured as is known in the art, for example by densitometer scanning of coomassie-stained SDS-PAGE gels, e.g. using a GS-800 calibrated densitometer from BIO-RAD; by using a commercial kit, such as Protein Assay ESL, order no. 1767003, which is commercially available from Roche, or on the basis of the method described in Example 8 of WO 01/58276.</div>
<div class="description-paragraph" num="p-0097">Preferably, the amylase enzyme protein constitutes at least 50%, more preferably at least 55, 60, 65, 70, 75, 80, 85, 90, 92, 94, 95, 96, or at least 97% of the protein spectrum of the solid lipase concentrate for use according to the invention, as measured by densitometer scanning of a coomassie-stained SDS-PAGE gel.</div>
<div class="description-paragraph" num="p-0098">A pharmaceutical composition of the invention comprises the enzyme(s), preferably in the form of concentrated enzyme preparations, more preferably solid concentrates, together with at least one pharmaceutically acceptable auxiliary, or subsidiary, material such as (i) at least one carrier and/or excipient; or (ii) at least one carrier, excipient, diluent, and/or adjuvant. Non-limiting examples of, optional, other ingredients, all pharmaceutically acceptable, are disintegrators, lubricants, buffering agents, moisturizing agents, preservatives, flavouring agents, solvents, solubilizing agents, suspending agents, emulsifiers, stabilizers, propellants, and vehicles.</div>
<div class="description-paragraph" num="p-0099">Generally, depending i.a. on the medical indication in question, the composition of the invention may be designed for all manners of administration known in the art, preferably including enteral administration (through the alimentary canal). Thus, the composition may be in solid, semi-solid, liquid, or gaseous form, such as tablets, capsules, powders, granules, microspheres, ointments, creams, foams, solutions, suppositories, injections, inhalants, gels, microspheres, lotions, and aerosols. The medical practitioner will know to select the most suitable route of administration and of course avoid potentially dangerous or otherwise disadvantageous administration routes.</div>
<div class="description-paragraph" num="p-0100">The following methods and auxiliary materials are therefore also merely exemplary and are in no way limiting.</div>
<div class="description-paragraph" num="p-0101">For solid oral preparations, the enzyme(s) can be used alone or in combination with appropriate additives to make pellets, micropellets, tablets, microtablets, powders, granules or capsules, for example, with conventional carriers, such as lactose, mannitol, corn starch, or potato starch; with excipients or binders, such as crystalline, or microcrystalline, cellulose, cellulose derivatives, acacia, corn starch, or gelatins; with disintegrators, such as corn starch, potato starch, or sodium carboxymethylcellulose; with lubricants, such as carnauba wax, white wax, shellac, waterless colloid silica, polyethylene glycol (PEGs, also known under the term macrogol) from 1500 to 20000, in particular PEG 4000, PEG 6000, PEG 8000, povidone, talc, monolein, or magnesium stearate; and if desired, with diluents, adjuvants, buffering agents, moistening agents, preservatives such as methylparahydroxybenzoate (E218), colouring agents such as titanium dioxide (E171), and flavouring agents such as saccharose, saccharin, orange oil, lemon oil, and vanillin. Oral preparations are examples of preferred preparations for treatment of the medical indication of PEI.</div>
<div class="description-paragraph" num="p-0102">The enzyme(s) can also, quite generally, be formulated into liquid oral preparations, by dissolving, suspending, or emulsifying them in an aqueous solvent such as water, or in non-aqueous solvents such as vegetable or other similar oils, synthetic aliphatic acid glycerides, esters of higher aliphatic acids, propylene glycol, polyethylene glycol such as PEG 4000, or lower alcohols such as linear or ramified C1-C4 alcohols, for example 2-propanol; and if desired, with conventional subsidiary materials or additives such as solubilizers, adjuvants, diluents, isotonic agents, suspending agents, emulsifying agents, stabilizers, and preservatives.</div>
<div class="description-paragraph" num="p-0103">Furthermore, the enzyme(s) can generally be made into suppositories for rectal administration by mixing with a variety of bases such as emulsifying bases or water-soluble bases. The suppository can include vehicles such as cocoa butter, carbowaxes and polyethylene glycols, which melt at body temperature, yet are solidified at room temperature.</div>
<div class="description-paragraph" num="p-0104">The use of liposomes as a delivery vehicle is another method of possible general interest. The lipids may be any useful combination of known liposome forming lipids, including cationic or zwitterionic lipids, such as phosphatidylcholine. The remaining lipid will normally be neutral or acidic lipids, such as cholesterol, phosphatidyl serine, phosphatidyl glycerol, and the like. For preparing the liposomes, the procedure described by Kato et al. (1991) J. Biol. Chem. 266:3361 may be used.</div>
<div class="description-paragraph" num="p-0105">Unit dosage forms for oral or rectal administration such as syrups, elixirs, powders, and suspensions may be provided wherein each dosage unit, for example, teaspoonful, tablespoonful, capsule, tablet or suppository, contains a predetermined amount of the enzyme(s). Similarly, unit dosage forms for injection or intravenous administration may comprise the enzyme(s) in a composition as a solution in sterile water, normal saline, or another pharmaceutically acceptable carrier.</div>
<div class="description-paragraph" num="p-0106">The term “unit dosage form”, as used herein, refers to physically discrete units suitable as unitary dosages for human and animal subjects, each unit containing a predetermined quantity of enzyme(s) in an amount sufficient to produce the desired effect.</div>
<div class="description-paragraph" num="p-0107">In a particular embodiment, the pharmaceutical composition of the invention is for enteral, preferably oral, administration.</div>
<div class="description-paragraph" num="p-0108">In further particular embodiments, the oral composition is (i) a liquid composition containing crystals of the enzyme(s); (ii) a liquid suspension of sediments of (highly) purified enzyme(s); (iii) a gel containing the enzyme(s) in solid or solubilized form; (iv) a liquid suspension of immobilized enzyme(s) or of enzymes adsorbed to particles and the like; or (v) a solid composition in the form of enzyme(s)-containing powder, pellets, granules, or microspheres, if desired in the form of tablets, capsules, or the like, that are optionally coated, for example with an acid-stable coating.</div>
<div class="description-paragraph" num="p-0109">In another particular embodiment of the composition, the enzyme(s) are compartmentalized, viz. separated from each other, for example by means of separate coatings.</div>
<div class="description-paragraph" num="p-0110">In a still further particular embodiment of the composition, the protease is separated from other enzyme components of the composition, such as the lipase, and/or the amylase.</div>
<div class="description-paragraph" num="p-0111">The dosage of the enzyme(s) will vary widely, depending on the specific enzyme(s) to be administered, the frequency of administration, the manner of administration, the severity of the symptoms, and the susceptibility of the subject to side effects, and the like. Some of the specific enzymes may be more potent than others.</div>
<div class="description-paragraph" num="p-0112">Examples of solid oral preparations of the enzyme(s) of the invention comprise: (i) an amylase of at least 50% identity to (i) amino acids 1-481 of SEQ ID NO: 1, (ii) amino acids 1-481 of SEQ ID NO: 2, and/or (iii) amino acids 1-483 of SEQ ID NO: 3; (ii) a protease having at least 70% identity to a protease selected from the group consisting of a) a protease having amino acids 1-274 of SEQ ID NO: 5, b) a protease having amino acids 1-188 of SEQ ID NO: 6, and c) a protease having amino acids 1-188 of SEQ ID NO: 7; and/or (iii) a lipase having at least 70% identity to a lipase having amino acids 1-269 of SEQ ID NO: 8; wherein preferably the anticipated daily clinical dosages of the enzymes of (i), (ii), and (iii) are as follows (all in mg enzyme protein per kg of bodyweight (bw)): For the amylase of (i): 0.001-250, 0.005-100, 0.01-50, or 0.05-10 mg/kg bw; for the protease of (ii): 0.005-500, 0.01-250, 0.05-100, or 0.1-50 mg/kg bw; for the lipase of (iii): 0.01-1000, 0.05-500, 0.1-250, or 0.5-100 mg/kg bw.</div>
<div class="description-paragraph" num="p-0113">A preferred example of solid oral preparations of the enzyme(s) of the invention comprise: (i) an amylase comprising amino acids 1-481 of SEQ ID NO: 1, (ii) a protease comprising amino acids 1-274 of SEQ ID NO: 5, and/or (i) a lipase comprising amino acids 2-269 of SEQ ID NO: 8.</div>
<div class="description-paragraph" num="p-0114">Examples of anticipated daily clinical dosages of the enzymes of (i), (ii), and (iii) are as follows (all in mg enzyme protein per kg of bodyweight (bw)): For the amylase of (i): 0.01-50, 0.05-10, or 0.1-5 mg/kg bw; for the protease of (ii): 0.05-100, 0.1-50, or 0.5-25 mg/kg bw; for the lipase of (iii): 0.1-250, 0.5-100, or 1-50 mg/kg bw.</div>
<div class="description-paragraph" num="p-0115">The amide (peptide) bonds, as well as the amino and carboxy termini, may be modified for greater stability on oral administration. For example, the carboxy terminus may be amidated.</div>
<div class="description-paragraph" num="p-0116">Particular embodiments of pharmaceutical compositions of the invention, suitable for the treatment of digestive disorders, PEI, pancreatitis, cystic fibrosis, diabetes type I, and/or diabetes type II, may be prepared by incorporating the enzyme(s) of the invention into pellets. The pellets may generally comprise from 10-90% (w/w, relative to the dry weight of the resulting pellets) of a physiologically acceptable organic polymer, from 10-90% (w/w, relative to the dry weight of the resulting pellets) of cellulose or a cellulose derivative, and from 80-20% (w/w, relative to the dry weight of the resulting pellets) of the enzyme(s), the total amount of organic polymer, cellulose or cellulose derivative and enzyme(s) making up to 100% in each case.</div>
<div class="description-paragraph" num="p-0117">The physiologically acceptable organic polymer can be selected from the group consisting of polyethylene glycol 1500, polyethylene glycol 2000, polyethylene glycol 3000, polyethylene glycol 4000, polyethylene glycol 6000, polyethylene glycol 8000, polyethylene glycol 10000, polyethylene glycol 20000, hydroxypropyl methylcellulose, polyoxyethylen, copolymers of polyoxyethylen-polyoxypropylen and mixtures of said organic polymers. Polyethylene glycol 4000 is preferred as physiologically acceptable organic polymer.</div>
<div class="description-paragraph" num="p-0118">The cellulose or a cellulose derivative can e.g. be selected from cellulose, cellulose acetate, cellulose fatty acid ester, cellulose nitrates, cellulose ether, carboxymethyl cellulose, ethyl cellulose, hydroxyethyl cellulose, hydroxypropyl cellulose, methyl cellulose, methyl ethylcellulose and methylhydroxypropyl cellulose. Cellulose, in particular microcrystalline cellulose is preferred as cellulose or cellulose derivative.</div>
<div class="description-paragraph" num="p-0119">The resulting pellets may be coated with a suitable enteric coating, other non functional coating or be used directly without such coating. Further, the resulting pellets may be filled in capsules like hard gelatin capsules or gelatin free capsules of a suitable size for therapy of a disorder or disease as described in more detail above. In an embodiment of the invention, pellets produced from different enzyme types, in particular from lipase, protease and/or amylase may be filled into said capsules. While filling the capsules with the different enzyme types, the dosing of the single enzyme types (viz. lipase, protease or amylase) may be adapted to specific needs of a certain indication group or a certain patient subgroup by adding a specified amount of any of lipase, protease and/or amylase to the capsules, i.e. capsules may be produced which vary in their specific ratios of lipase:protease:amylase.</div>
<div class="description-paragraph" num="p-0120">Preferred pharmaceutical compositions of the lipase of the invention are described in WO 2005/092370, in particular formulations comprising the preferred excipients mentioned therein. In a particularly preferred embodiment, the pharmaceutical composition comprises a macrogolglyceride mixture of mono-, di- and tri-acylglycerides and polyethylene glycol (PEG) mono- and di-esters of aliphatic C6-C22 carboxylic acids, and also possibly small proportions of glycerol and free polyethylene glycol.</div>
<div class="description-paragraph" num="p-0121">The polyethylene glycol (PEG) contained in the macrogolglyceride mixtures is preferably PEG which has on average 6 to at most 40 ethylene oxide units per molecule or a molecular weight of between 200 and 2000.</div>
<div class="description-paragraph" num="p-0122">One further aspect of the invention provides for the pharmaceutical composition of the enzyme(s) of the invention to comprise a system consisting of surfactant, co-surfactant and lipophilic phase, the system having an HLB value (Hydrophilic-Lipophilic Balance) greater than or equal to 10 and a melting point greater than or equal to 30° C. In a preferred embodiment, the system has an HLB value of 10 to 16, preferably of 12 to 15, and has a melting point of between 30 and 600° C., preferably between 40 and 500° C. In particular, the system characterised by HLB value and melting point is a mixture of mono-, di- and triacylgylcerides and mono- and diesters of polyethylene glycol (PEG) with aliphatic carboxylic acids with 8 to 20, preferably 8 to 18, carbon atoms, whereby the polyethylene glycol preferably has about 6 to about 32 ethylene oxide units per molecule, and the system optionally contains free glycerin and/or free polyethylene glycol. The HLB value of such a system is preferably regulated by the chain length of the PEG. The melting point of such a system is regulated by the chain length of the fatty acids, the chain length of the PEG and the degree of saturation of the fatty-acid chains, and hence the starting oil for the preparation of the macrogolglyceride mixture.</div>
<div class="description-paragraph" num="p-0123">“Aliphatic C8-C18 carboxylic acids” designates mixtures in which caprylic acid (C8), capric acid (C10), lauric acid (C12), myristic acid (C14), palmitic acid (C16) and stearic acid (C18) are contained in a significant and variable proportion, if these acids are saturated, and the corresponding unsaturated C8-C18 carboxylic acids. The proportions of these fatty acids may vary according to the starting oils.</div>
<div class="description-paragraph" num="p-0124">Such a mixture of mono-, di- and triacylgylcerides and mono- and diesters of polyethylene glycol (PEG) with aliphatic carboxylic acids with 8 to 18 carbon atoms can for example be obtained by a reaction between a polyethylene glycol with a molecular weight of between 200 and 1500 and a starting oil, the starting oil consisting of a triglyceride mixture with fatty acids which are selected from the group containing caprylic acid, capric acid, lauric acid, myristic acid, palmitic acid, stearic acid, oleic acid and linolenic acid, individually or as a mixture. Optionally, the product of such a reaction may also contain small proportions of glycerin and free polyethylene glycol.</div>
<div class="description-paragraph" num="p-0125">Such mixtures are commercially available for example under the trade name Gelucire®. One advantageous embodiment of the invention provides that, of the products known under the trade name Gelucire®, in particular “Gelucire® 50/13” and/or “Gelucire®) 44/14” represent suitable mixtures for use in the pharmaceutical preparations according to the invention.</div>
<div class="description-paragraph" num="p-0126">Gelucire® 50/13 is a mixture with mono-, di- and triacylglycerides and mono- and diesters of polyethylene glycol, with palmitic acid (C16) and stearic acid (C18) at 40% to 50% and 48% to 58%, respectively making up the major proportion of bound fatty acids. The proportion of caprylic acid (C8) and capric acid (C10) is less than 3% in each case, and the proportion of lauric acid (C12) and myristic acid (C14) in each case is less than 5%.</div>
<div class="description-paragraph" num="p-0127">Gelucire® 44/14 is a mixture with mono-, di- and triacylgylcerides and mono- and diesters of polyethylene glycol, the respective proportions of palmitic acid (C16) being 4 to 25%, stearic acid (C18) 5 to 35%, caprylic acid (C8) less than 15%, capric acid (C10) less than 12%, lauric acid (C12) 30 to 50% and myristic acid (C14) 5 to 25%. Gelucire® 44/14 can for example be prepared by an alcoholysis/esterification reaction using palm kernel oil and polyethylene glycol 1500.</div>
<div class="description-paragraph" num="p-0128">A preferred embodiment of the present invention provides for a pharmaceutical composition of the enzyme(s) of the invention which comprises a system containing a mixture of mono-, di- and triacyl-glycerides and polyethylene glycol mono- and diesters of aliphatic C8-C18 carboxylic acids and also possibly small proportions of glycerin and free polyethylene glycol, the system having a melting point between 40° C. and 55° C. and an HLB value in the range between 12 and 15. More preferred, the system has a melting point between 44° C. and 50° C. and an HLB value in the range from 13-14. Alternatively, the system has a melting point around 44° C. and an HLB value of 14, or the system has a melting point around 50° C. and an HLB value of 13.</div>
<div class="description-paragraph" num="h-0011">Methods of Treatment</div>
<div class="description-paragraph" num="p-0129">The amylase of the invention, optionally in combination with a lipase, and/or a protease (the enzyme(s) of the invention), is useful in the therapeutic, and/or prophylactic, treatment of various diseases or disorders in animals. The term “animal” includes all animals, and in particular human beings. Examples of animals are non-ruminants, and ruminants, such as sheep, goat, and cattle, e.g. beef cattle, and cow. In a particular embodiment, the animal is a non-ruminant animal. Non-ruminant animals include mono-gastric animals, e.g. pig (including, but not limited to, piglets, growing pigs, and sows); poultry such as turkey, duck and chicken (including but not limited to broiler chicks, layers); young calves; pets such as cat, and dog; and fish (including but not limited to salmon, trout, tilapia, catfish and carps); and crustaceans (including but not limited to shrimps and prawns). In a particular embodiment the animal is a mammal, more in particular a human being.</div>
<div class="description-paragraph" num="p-0130">For example, the enzyme(s) are useful in the treatment of digestive disorders like maldigestion or dyspepsia that are often caused by a deficient production and/or secretion into the gastrointestinal tract of digestive enzymes normally secreted from, i.a., the stomach, and the pancreas.</div>
<div class="description-paragraph" num="p-0131">Further, the enzyme(s) are particularly useful in the treatment of PEI. PEI can be verified using, i.a., the Borgström test (JOP. J Pancreas (Online) 2002; 3(5):116-125), and it may be caused by diseases and conditions such as pancreatic cancer, pancreatic and/or gastric surgery, e.g. total or partial resection of the pancreas, gastrectomy, post gastrointestinal bypass surgery (e.g. Billroth II gastroenterostomy); chronic pancreatitis; Shwachman Diamond Syndrome; ductal obstruction of the pancreas or common bile duct (e.g. from neoplasm); and/or cystic fibrosis (an inherited disease in which a thick mucus blocks the ducts of the pancreas). The enzyme(s) may also be useful in the treatment of acute pancreatitis.</div>
<div class="description-paragraph" num="p-0132">The effect of the enzyme(s) on digestive disorders can be measured as generally described in EP 0600868, in which Example 2 describes an in vitro digestibility test for measuring lipase stability test under gastric conditions, and Example 3 an in vitro digestibility test for lipase activity in the presence of bile salts. Corresponding tests can be set up for the protease and amylase. Also WO 02/060474 discloses suitable tests, for example (1) an in vitro test for measuring lipid digestion in a swine test feed, and (2) an in vivo trial with pancreas insufficient swine in which the digestibility of fat, protein and starch is measured.</div>
<div class="description-paragraph" num="p-0133">In a particular embodiment, the effect of the amylase of the invention is measured using the in vivo screening test for amylase efficacy of Example 2.</div>
<div class="description-paragraph" num="p-0134">As another example, the enzyme(s) are useful in the treatment of Diabetes mellitus type I, and/or type II, in particular for adjuvant treatment in a diabetes therapy of digestive disorders usually accompanying this disease, with a view to diminishing late complications.</div>
<div class="description-paragraph" num="p-0135">The effect on Diabetes mellitus of the enzyme(s) may be determined by one or more of the methods described in WO 00/54799, for example by controlling the level of glycosylated haemoglobin, the blood glucose level, hypoglycaemic attacks, the status of fat-soluble vitamins like vitamins A, D and E, the required daily dosage of insulin, the body-weight index, and hyper glycaemic periods.</div>
<div class="description-paragraph" num="p-0136">The invention described and claimed herein is not to be limited in scope by the specific embodiments herein disclosed, since these embodiments are intended as illustrations of several aspects of the invention. Any equivalent embodiments are intended to be within the scope of this invention. Indeed, various modifications of the invention in addition to those shown and described herein will become apparent to those skilled in the art from the foregoing description. Such modifications are also intended to fall within the scope of the appended claims. In the case of conflict, the present disclosure including definitions will control.</div>
<div class="description-paragraph" num="p-0137">Various references are cited herein, the disclosures of which are incorporated by reference in their entireties.</div>
<heading>EXAMPLES</heading>
<heading>Example 1</heading>
<heading>Enzyme Assays</heading>
<div class="description-paragraph" num="p-0138">Amylase, protease and lipase activity may be determined using the FIP assays (Fédération Internationale Pharmaceutique), 1 FIP-unit=1 Ph.Eur.-unit (European Pharmacopoeia). These assays are described in: Federation Internationale Pharmaceutique, Scientific Section International Commission for the standardisation of pharmaceutical enzymes. a) “Pharmaceutical Enzymes,” Editors: R. Ruyssen and A. Lauwers, E. Story Scientia, Ghent, Belgium (1978), b) European Pharmacopoeia. See also Deemester et al in Lauwers A, Scharpé S (eds): Pharmaceutical Enzymes, New York, Marcel Dekker, 1997, p. 343-385. Enzyme standards can be procured from the International Commission on Pharmaceutical Enzymes, Centre for Standards, Harelbekestraat 72, B-9000 Ghent.</div>
<div class="description-paragraph" num="h-0015">Amylase FIP Assay</div>
<div class="description-paragraph" num="p-0139">The amylolytic activity of pancreatin was analyzed according to the method published in the European Pharmacopoeia 5.1 against the pancreatin standard supplied by the FIP. For determination of the amylolytic activity of the amylases for use according to the invention, the assay for amylolytic activity of microbial amylases described by the FIP was modified. In principle, starch is hydrolysed by amylase at pH 5.8 and at constant temperature (37.0+/−0.1° C.) in the presence of sodium chloride and calcium chloride. The reducing groups resulting from the hydrolysis react with iodine in alkaline solution and the excess of iodine are titrated with thiosulphate. One unit of amylase is defined as the amount of enzyme, which, under the defined conditions and substrate, hydrolyzes 1 micromol of glycosidic bond per minute.</div>
<div class="description-paragraph" num="h-0016">Reagents</div>
<div class="description-paragraph" num="p-0140">Substrate solution: The substrate is soluble starch (e.g. Merck, No. 101252). The water content of each batch of soluble starch is determined upon opening of the container. To a quantity of soluble starch equivalent to 10.0 g of the dry substrate, add 50 mL of pure (filtration/ion-exchange) water and mix. Add this suspension, whilst stirring continuously, to 800 mL of boiling pure water. Rinse the container several times with successive quantities, each of 10 mL, of pure water and add the washing water to the hot starch solution. Heat to boiling under continuous stirring. Cool to room temperature and dilute to 1000 mL with pure water.</div>
<div class="description-paragraph" num="p-0141">Buffer (acetate) solution pH 5.8 (0.2 mol/L): Dissolve 12 g of acetic acid, 1 g of sodium chloride and 544 mg calcium chloride in about 800 mL of water. Adjust the pH to 5.8 with a sodium hydroxide solution (about 10 N). Dilute to 1000 mL.</div>
<div class="description-paragraph" num="p-0142">Acetic acid, CH<sub>3</sub>COOH, p.a.</div>
<div class="description-paragraph" num="p-0143">Sodium chloride, NaCl, p.a.</div>
<div class="description-paragraph" num="p-0144">Calcium chloride×2H<sub>2</sub>O, p.a.</div>
<div class="description-paragraph" num="p-0145">0.25 N sodium hydroxide</div>
<div class="description-paragraph" num="p-0146">1 N hydrochloric acid</div>
<div class="description-paragraph" num="p-0147">0.1 N iodine solution (e.g. Titrisol (Merck 9910), diluted with pure water)</div>
<div class="description-paragraph" num="p-0148">0.1 N sodium thiosulphate solution</div>
<div class="description-paragraph" num="p-0149">Sulphuric acid (p.a.) 20%: to 4 volume parts water add 1 volume part sulphuric acid 96% carefully.</div>
<div class="description-paragraph" num="p-0150">Fungal amylase reference standard FIP (Batch No. 2; 55310 FIP-U/g, Condition 25° C. without calcium chloride).</div>
<div class="description-paragraph" num="p-0151">Quality control suspension (Fungal Amylase reference standard): Dissolve an accurately weighed quantity of the reference standard so that the solution gives a titration volume of thiosulfate, that is between 2 and 4 mL (e.g. weigh about 12-15 mg of the fungal amylase standard and dissolve in 100 ml buffer solution). The quality control suspension is used only for day to day monitoring of the test system.</div>
<div class="description-paragraph" num="p-0152">Test suspension: Dissolve an accurately weighed quantity of the test sample so that the solution gives a titration volume of thiosulfate, that is between 2 and 4 mL.</div>
<div class="description-paragraph" num="h-0017">Procedure</div>
<div class="description-paragraph" num="p-0153">Test sample: For each determination prepare a reaction solution containing 25.0 mL of substrate solution and 10.0 mL of 0.2 M acetate buffer solution in a 300 mL Erlenmeyer flask. Stopper it with a rubber stopper and place it in a water bath at a constant temperature (37.0+/−0.1° C.). As soon as the substrate mixture has reached a constant temperature, start the reaction by adding 2.0 mL of the test suspension. Shake briefly and incubate for exactly 10 minutes at 37° C. Stop the reaction immediately by adding 4 mL of hydrochloric acid. While stirring add 10 mL of 0.1 N iodine solution, 25 mL of 0.25 N sodium hydroxide and, for rinsing the walls, 20 mL of pure water. Allow the mixture to stand in complete darkness for 15 minutes. Add 4 mL of sulphuric acid and titrate the solution with sodium thiosulphate solution using a microburette. Calculate the average of two determinations.</div>
<div class="description-paragraph" num="p-0154">Quality control sample: Determine the activity of the quality control suspension as described for the test sample in duplicate. Use 2.0 mL of the reference suspension.</div>
<div class="description-paragraph" num="p-0155">Blank values: Prepare blank values of the test and reference suspensions. Repeat the procedure as described above but add the 4 mL of 1 N hydrochloric acid before addition of the test and the reference solution.</div>
<div class="description-paragraph" num="p-0156">Calculate the amylase activity in Units per g of test sample using the following equation:</div>
<div class="description-paragraph" num="p-0157">
<maths id="MATH-US-00001" num="00001">
<math overflow="scroll">
<mrow>
<mfrac>
<mrow>
<mrow>
<mo>(</mo>
<mrow>
<msub>
<mi>n</mi>
<mi>U</mi>
</msub>
<mo>-</mo>
<msub>
<mi>n</mi>
<mi>UB</mi>
</msub>
</mrow>
<mo>)</mo>
</mrow>
<mo>×</mo>
<mn>5</mn>
<mo>×</mo>
<mn>1</mn>
<mo>⁢</mo>
<mstyle>
<mtext>,</mtext>
</mstyle>
<mo>⁢</mo>
<mn>000</mn>
<mo>⁢</mo>
<mstyle>
<mspace height="0.8ex" width="0.8em"> </mspace>
</mstyle>
<mo>⁢</mo>
<mi>test</mi>
<mo>⁢</mo>
<mstyle>
<mspace height="0.8ex" width="0.8em"> </mspace>
</mstyle>
<mo>⁢</mo>
<mrow>
<mi>solution</mi>
<mo>⁢</mo>
<mstyle>
<mspace height="0.8ex" width="0.8em"> </mspace>
</mstyle>
<mo>[</mo>
<mi>ml</mi>
<mo>]</mo>
</mrow>
</mrow>
<mrow>
<mn>2</mn>
<mo>⁢</mo>
<mstyle>
<mspace height="0.8ex" width="0.8em"> </mspace>
</mstyle>
<mo>⁢</mo>
<mi>ml</mi>
<mo>×</mo>
<mi>weighted</mi>
<mo>⁢</mo>
<mstyle>
<mspace height="0.8ex" width="0.8em"> </mspace>
</mstyle>
<mo>⁢</mo>
<mrow>
<mi>sample</mi>
<mo>⁢</mo>
<mstyle>
<mspace height="0.8ex" width="0.8em"> </mspace>
</mstyle>
<mo>[</mo>
<mi>mg</mi>
<mo>]</mo>
</mrow>
</mrow>
</mfrac>
<mo>=</mo>
<mfrac>
<mrow>
<mi>amylase</mi>
<mo>⁢</mo>
<mstyle>
<mspace height="0.8ex" width="0.8em"> </mspace>
</mstyle>
<mo>⁢</mo>
<mi>units</mi>
</mrow>
<mrow>
<mrow>
<mo>[</mo>
<mi>g</mi>
<mo>]</mo>
</mrow>
<mo>⁢</mo>
<mstyle>
<mspace height="0.8ex" width="0.8em"> </mspace>
</mstyle>
<mo>⁢</mo>
<mi>test</mi>
<mo>⁢</mo>
<mstyle>
<mspace height="0.8ex" width="0.8em"> </mspace>
</mstyle>
<mo>⁢</mo>
<mi>sample</mi>
</mrow>
</mfrac>
</mrow>
</math>
</maths>
</div>
<div class="description-paragraph" num="p-0158">nU: consumption of 0.1 N sodium thiosulphate [mL] used in the titration of the test suspension;</div>
<div class="description-paragraph" num="p-0159">nUB: consumption of 0.1 N sodium thiosulphate [mL] used in the blank titration of the test suspension;</div>
<div class="description-paragraph" num="p-0160">The measured activity of the monitoring sample (Fungal reference sample) is 97500 U/g, with a Range of Mean+/−3×SD, in particular with a Range of +/−6000 U/g.</div>
<div class="description-paragraph" num="h-0018">Amylase</div>
<div class="description-paragraph" num="p-0161">Alternatively, the following amylase assay can be used:</div>
<div class="description-paragraph" num="h-0019">Substrate: Phadebas tablets (Pharmacia Diagnostics; cross-linked, insoluble, blue-coloured starch polymer, which is mixed with bovine serum albumin and a buffer substance, and manufactured into tablets)</div>
<div class="description-paragraph" num="p-0162">
</div> <ul>
<li id="ul0001-0001" num="0161">Assay Temperature: 37° C.</li>
<li id="ul0001-0002" num="0162">Assay pH: 4.3 (or 7.0, if desired)</li>
<li id="ul0001-0003" num="0163">Reaction time: 20 min</li>
</ul>
<div class="description-paragraph" num="p-0163">After suspension in water the starch is hydrolyzed by the alpha-amylase, giving soluble blue fragments. The absorbance of the resulting blue solution, measured at 620 nm, is a function of the alpha-amylase activity. One Fungal alpha-Amylase Unit (1 FAU) is the amount of enzyme which breaks down 5.26 g starch (Merck, Amylum soluble Erg. B. 6, Batch 9947275) per hour at the standard assay conditions. A more detailed assay description, APTSMYQI-3207, is available on request from Novozymes A/S, Krogshoejvej 36, DK-2880 Bagsvaerd, Denmark.</div>
<div class="description-paragraph" num="h-0020">Lipase</div>
<div class="description-paragraph" num="p-0164">
</div> <ul>
<li id="ul0002-0001" num="0165">Substrate: para-Nitro-Phenyl (pNP) Valerate</li>
<li id="ul0002-0002" num="0166">Assay pH: 7.7</li>
<li id="ul0002-0003" num="0167">Assay temperature: 40° C.</li>
<li id="ul0002-0004" num="0168">Reaction time: 25 min</li>
</ul>
<div class="description-paragraph" num="p-0165">The digested product with yellow colour has a characteristic absorbance at 405 nm. Its quantity is determined by spectrophotometry. One lipase unit is the amount of enzyme which releases 1 micromole titratable butyric acid per minute under the given assay conditions. A more detailed assay description, AF95/6-GB, is available on request from Novozymes A/S, Krogshoejvej 36, DK-2880 Bagsvaerd, Denmark.</div>
<div class="description-paragraph" num="h-0021">Protease</div>
<div class="description-paragraph" num="p-0166">
</div> <ul>
<li id="ul0003-0001" num="0170">Substrate: Suc-AAPF-pNA (Sigma® S-7388).</li>
<li id="ul0003-0002" num="0171">Assay buffer: 100 mM succinic acid, 100 mM HEPES (Sigma H-3375), 100 mM CHES (Sigma C-2885), 100 mM CABS (Sigma C-5580), 1 mM CaCl<sub>2</sub>, 150 mM KCl, 0.01% Triton® X-100 adjusted to pH 9.0 with HCl or NaOH.</li>
<li id="ul0003-0003" num="0172">Assay temperature: 25° C.</li>
</ul>
<div class="description-paragraph" num="p-0167">300 μl diluted protease sample was mixed with 1.5 ml of the assay buffer and the activity reaction was started by adding 1.5 ml pNA substrate (50 mg dissolved in 1.0 ml DMSO and further diluted 45× with 0.01% Triton® X-100) and, after mixing, the increase in A<sub>405 </sub>was monitored by a spectrophotometer as a measurement of the protease activity. The protease samples were diluted prior to the activity measurement in order to ensure that all activity measurements fell within the linear part of the dose-response curve for the assay.</div>
<heading>Example 2</heading>
<heading>In vivo Screening Test for Amylase Efficacy</heading>
<div class="description-paragraph" num="p-0168">The purified <i>Bacillus </i>amylases of amino acids 1-486 of SEQ ID NO: 1, amino acids 1-481 of SEQ ID NO: 2 and amino acids 1-483 of SEQ ID NO: 3 were tested in female Göttingen minipigs (Ellegaard), together with a purified fungal amylase derived from <i>Aspergillus oryzae</i>, being below 50% identical to any one of SEQ ID NOs: 1-3. Pancreatic Exocrine Insufficiency (PEI) was induced in the minipigs by ligation of the pancreatic duct, and they were also fitted with an ileo-caecal re-entrant cannula, all under halothane anaesthesia and at a weight of about 25 kg, as described in Tabeling et al., J. 1999, Studies on nutrient digestibilities (pre-caecal and total) in pancreatic duct-ligated pigs and the effects of enzyme substitution, J. Anim. Physiol. A. Anim. Nutr. 82: 251-263 (hereinafter referred to as “Tabeling 1999”); and in Gregory et al., J. 1999. Growth and digestion in pancreatic duct ligated pigs, Effect of enzyme supplementation in “Biology of the Pancreas in Growing Animals” (SG Pierzynowski &amp; R. Zabielski eds), Elsevier Science BV, Amsterdam, pp 381-393 (hereinafter referred to as “Gregory et al 1999”). A period of at least 4 weeks was allowed for recovery from surgery, before studies were commenced. Prior to study begin, the PEI status of each pig was confirmed via the stool chymotrypsin test (commercially available from Immundiagnostik AG, Wiesenstrasse 4, D-64625 Bensheim, Germany, with catalogue No. K 6990).</div>
<div class="description-paragraph" num="p-0169">During the studies, the pigs were housed in modified metabolism cages on a 12:12 h light-dark cycle and allowed free access to water and fed two meals/day. The test meal, containing 15.4% protein, 69% starch, and 2.1% fat, had the following composition in g/100 g dry matter: Poultry meat meal 7.1; fishmeal 2.45; casein 4.1; wheat flour 20.65; shelled rice 9.8; potato starch 7.7; maize starch 39.8; cellulose powder 3.0; vitamins 5.3 (as per the nutritional requirements). The pigs were fed 250 g of this test meal mixed with 1 litre water, 0.625 g Cr<sub>2</sub>O<sub>3 </sub>and into which differing amounts of amylases (0, 7500, 18750, 90000 FIP U amylase/meal, see Example 1) were mixed immediately before feeding.</div>
<div class="description-paragraph" num="p-0170">Ileal chyme, collected for a total of 6 h after first appearance of the meal marker in the ileum (green chyme) was frozen at −20° C. immediately after collection, to avoid any bacterial fermentation of the samples. At least one day washout was allowed between separate determinations.</div>
<div class="description-paragraph" num="p-0171">The frozen samples were freeze-dried, milled and analysed for dry matter (DM) and starch. DM was estimated by weight after freeze-drying followed by 8 h incubation at 103° C.; starch was analysed polarimetrically after acid hydrolysis and sugar was estimated gravimetrically. Cr<sub>2</sub>O<sub>3 </sub>was oxidized to chromate and chromium content was calculated via extinction at 365 nm (spectrophotometer), as described by Petry and Rapp in Zeitung für Tierphysiologie (1970), vol. 27, p. 181-189.</div>
<div class="description-paragraph" num="p-0172">Digestibility: Calculation of apparent pre-caecal starch digestibility was made according to the formula, in which Cr<sub>2</sub>O<sub>3 </sub>and starch were expressed as g/100 g dry matter:</div>
<div class="description-paragraph" num="p-0173">
<maths id="MATH-US-00002" num="00002">
<math overflow="scroll">
<mrow>
<mrow>
<mi>Apparent</mi>
<mo>⁢</mo>
<mstyle>
<mspace height="0.8ex" width="0.8em"> </mspace>
</mstyle>
<mo>⁢</mo>
<mi>digestibility</mi>
<mo>⁢</mo>
<mstyle>
<mspace height="0.8ex" width="0.8em"> </mspace>
</mstyle>
<mo>⁢</mo>
<mrow>
<mo>(</mo>
<mi>%</mi>
<mo>)</mo>
</mrow>
</mrow>
<mo>=</mo>
<mrow>
<mn>100</mn>
<mo>-</mo>
<mrow>
<mo>[</mo>
<mrow>
<mfrac>
<mrow>
<mi>%</mi>
<mo>⁢</mo>
<mstyle>
<mspace height="0.8ex" width="0.8em"> </mspace>
</mstyle>
<mo>⁢</mo>
<msub>
<mi>Cr</mi>
<mn>2</mn>
</msub>
<mo>⁢</mo>
<msub>
<mi>O</mi>
<mn>3</mn>
</msub>
<mo>⁢</mo>
<mstyle>
<mspace height="0.8ex" width="0.8em"> </mspace>
</mstyle>
<mo>⁢</mo>
<mi>in</mi>
<mo>⁢</mo>
<mstyle>
<mspace height="0.8ex" width="0.8em"> </mspace>
</mstyle>
<mo>⁢</mo>
<mi>feed</mi>
</mrow>
<mrow>
<mi>%</mi>
<mo>⁢</mo>
<mstyle>
<mspace height="0.8ex" width="0.8em"> </mspace>
</mstyle>
<mo>⁢</mo>
<msub>
<mi>Cr</mi>
<mn>2</mn>
</msub>
<mo>⁢</mo>
<msub>
<mi>O</mi>
<mn>3</mn>
</msub>
<mo>⁢</mo>
<mstyle>
<mspace height="0.8ex" width="0.8em"> </mspace>
</mstyle>
<mo>⁢</mo>
<mi>in</mi>
<mo>⁢</mo>
<mstyle>
<mspace height="0.8ex" width="0.8em"> </mspace>
</mstyle>
<mo>⁢</mo>
<mi>sample</mi>
</mrow>
</mfrac>
<mo>·</mo>
<mfrac>
<mrow>
<mi>%</mi>
<mo>⁢</mo>
<mstyle>
<mspace height="0.8ex" width="0.8em"> </mspace>
</mstyle>
<mo>⁢</mo>
<mi>starch</mi>
<mo>⁢</mo>
<mstyle>
<mspace height="0.8ex" width="0.8em"> </mspace>
</mstyle>
<mo>⁢</mo>
<mi>in</mi>
<mo>⁢</mo>
<mstyle>
<mspace height="0.8ex" width="0.8em"> </mspace>
</mstyle>
<mo>⁢</mo>
<mi>sample</mi>
</mrow>
<mrow>
<mi>%</mi>
<mo>⁢</mo>
<mstyle>
<mspace height="0.8ex" width="0.8em"> </mspace>
</mstyle>
<mo>⁢</mo>
<mi>starch</mi>
<mo>⁢</mo>
<mstyle>
<mspace height="0.8ex" width="0.8em"> </mspace>
</mstyle>
<mo>⁢</mo>
<mi>in</mi>
<mo>⁢</mo>
<mstyle>
<mspace height="0.8ex" width="0.8em"> </mspace>
</mstyle>
<mo>⁢</mo>
<mi>feed</mi>
</mrow>
</mfrac>
<mo>·</mo>
<mn>100</mn>
</mrow>
<mo>]</mo>
</mrow>
</mrow>
</mrow>
</math>
</maths>
</div>
<div class="description-paragraph" num="p-0174">The apparent starch digestibility results are shown in Table 1 below.</div>
<div class="description-paragraph" num="p-0175">
<tables id="TABLE-US-00001" num="00001">
<patent-tables colsep="0" frame="none" pgwide="1" rowsep="0">
<table align="left" class="description-table" cols="5" colsep="0" rowsep="0" width="100%">
<thead>
<tr class="description-tr">
<td class="description-td" colspan="5" nameend="5" namest="1" rowsep="1">TABLE 1</td>
</tr>
<tr class="description-tr">
<td align="center" class="description-td" colspan="5" nameend="5" namest="1" rowsep="1"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">Enzyme</td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">Supplement</td>
<td class="description-td">0 Enzymes</td>
<td class="description-td">7500 FIP U</td>
<td class="description-td">18750 FIP U</td>
<td class="description-td">90000 FIP U</td>
</tr>
<tr class="description-tr">
<td align="center" class="description-td" colspan="5" nameend="5" namest="1" rowsep="1"> </td>
</tr>
</thead>
<tbody><tr class="description-tr">
<td class="description-td">No supplement</td>
<td class="description-td">44.9 +/− 7.6</td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">Pancreatin</td>
<td class="description-td"> </td>
<td class="description-td">64.1 +/− 15.2</td>
<td class="description-td">76.7 +/− 13.2</td>
<td class="description-td">97.2 +/− 2.2</td>
</tr>
<tr class="description-tr">
<td class="description-td">Amylase of SEQ ID NO: 1</td>
<td class="description-td"> </td>
<td class="description-td">87.2 +/− 4.3</td>
<td class="description-td">95.4 +/− 1.6</td>
<td class="description-td">96.4 +/− 3.0</td>
</tr>
<tr class="description-tr">
<td class="description-td">Amylase of SEQ ID NO: 3</td>
<td class="description-td"> </td>
<td class="description-td">89.0 +/− 6.8</td>
<td class="description-td">92.5 +/− 3.8</td>
<td class="description-td">96.8 +/− 2.6</td>
</tr>
<tr class="description-tr">
<td class="description-td">Amylase of SEQ ID NO: 2</td>
<td class="description-td"> </td>
<td class="description-td">93.9 +/− 8.4</td>
<td class="description-td">93.4 +/− 6.8</td>
<td class="description-td">96.7 +/− 2.6</td>
</tr>
<tr class="description-tr">
<td class="description-td">
<i>Aspergillus oryzae</i>
</td>
<td class="description-td"> </td>
<td class="description-td">49.0 +/− 16.2</td>
<td class="description-td">52.6 +/− 7.5</td>
<td class="description-td">59.9 +/− 12.5</td>
</tr>
<tr class="description-tr">
<td class="description-td">amylase</td>
</tr>
<tr class="description-tr">
<td align="center" class="description-td" colspan="5" nameend="5" namest="1" rowsep="1"> </td>
</tr>
<tr class="description-tr">
<td align="left" class="description-td" colspan="5" id="FOO-00001" nameend="5" namest="1">Values are mean ± SD</td>
</tr>
</tbody></table>
</patent-tables>
</tables>
</div>
<div class="description-paragraph" num="p-0176">From the results in Table 1 it is apparent that the amylases of the invention perform much better than the known <i>Aspergillus oryzae </i>amylase and better than known pancreatin preparations. The amylases of the invention caused a strong and dose dependent improvement on starch digestibility, already showing a highly efficient improvement at the lowest dosage tested.</div>
<heading>Example 3</heading>
<heading>Amylase pH Profiles, with and without Bile Salts</heading>
<div class="description-paragraph" num="p-0177">For use in the treatment of PEI and related diseases it is preferable for pancreatic replacement enzymes to have a high activity around pH 6-7 (typical conditions of the upper small intestines). This experiment serves to determine the pH profiles of four alpha-amylases, three bacterial amylases of the invention and a prior art fungal <i>Aspergillus oryzae </i>amylase, with and without added bile salts.</div>
<div class="description-paragraph" num="h-0026">Materials and Methods</div>
<div class="description-paragraph" num="h-0027">Enzymes</div>
<div class="description-paragraph" num="p-0178">The amylases used for this study were the purified <i>Bacillus </i>amylases of amino acids 1-486 of SEQ ID NO: 1 (“SEQ 1”), amino acids 1-481 of SEQ ID NO: 2 (“SEQ 2”) and amino acids 1-483 of SEQ ID NO: 3 (“SEQ 3”), and, for comparison, a purified fungal amylase derived from <i>Aspergillus oryzae </i>(“<i>A. oryzae</i>”), the amylase of the commercial Fungamyl™ amylase product, which is available from Novozymes A/S, Krogshoejvej 36, DK-2880 Bagsvaerd, Denmark.</div>
<div class="description-paragraph" num="h-0028">pH-profile—Reducing Sugar Assay</div>
<div class="description-paragraph" num="p-0179">Enzyme buffer: 50 mM acetate, 50 mM imidazole, 50 mM malonic acid, 1 mM CaCl<sub>2</sub>, 0.01% Triton X-100. Adjust to pH 2.0, 3.0, 4.0, 5.0, 6.0, or 7.0 with HCl/NaOH.</div>
<div class="description-paragraph" num="p-0180">Substrate buffer: 1.5 mg/ml amylopectin (waxy corn, e.g. Waxy corn 04201 from Cerestar, batch WM5671), 50 mM acetate, 50 mM imidazole, 50 mM malonic acid, 1 mM CaCl<sub>2</sub>. Adjust to the desired pH (as above) with HCl/NaOH. Incubate for 5 min at 100° C. The substrate buffer was made with or without 5 mM bile salts (i.e. Sodium taurocholate BRP, lot 2, from the Ph. Eur or FIP, also commercially available from e.g. LGC promochem, 500 g/mol).</div>
<div class="description-paragraph" num="p-0181">The amylase activity was detected by reducing sugar assay. Briefly, 50 μl enzyme (diluted in enzyme buffer so as to fall within the linear range of the assay) was mixed with 100 μl substrate buffer in PCR-MTP (Thermo-Fast® 96, ABgene, cat. no. AB-0600). The MTP's were incubated at 37° C. for 15 min, following which 75 μl stop solution (100 mM p-hydroxybenzoic acid hydrazide, 180 mM K-Na-tartrate, 2% NaOH) was added, and the plates were incubated at 95° C. for 10 min. Then 150 μl from each well was transferred to 96-well MTP, and the absorbance at 410 nm was monitored as a measure of amylase activity.</div>
<div class="description-paragraph" num="h-0029">Results</div>
<div class="description-paragraph" num="p-0182">The results (average of duplicate determinations) are shown in Tables 2-4, below.</div>
<div class="description-paragraph" num="p-0183">Table 2 shows the activity of each enzyme at the pH indicated in the absence of bile salts. For each enzyme, the maximum activity was set to 100%.</div>
<div class="description-paragraph" num="p-0184">Table 3 shows the same as Table 2, but in the presence of 5 mM bile salts.</div>
<div class="description-paragraph" num="p-0185">Table 4 shows the activity of each enzyme per mg enzyme protein at the pH indicated in the absence of bile salts, relative to the maximum enzyme activity measured in this experiment, which was the activity of the SEQ 1 enzyme at pH 5.0 (100%). The activity of each enzyme has accordingly been normalized relative to this activity. The amount of enzyme protein for each enzyme was determined on the basis of the specific activity.</div>
<div class="description-paragraph" num="p-0186">Table 5 shows the same as Table 4, but in the presence of 5 mM bile salts. Here the activity of the SEQ 1 enzyme at pH 5.0 in the presence of 5 mM bile salts is the reference value (100%).</div>
<div class="description-paragraph" num="p-0187">
<tables id="TABLE-US-00002" num="00002">
<patent-tables colsep="0" frame="none" rowsep="0">
<table align="left" class="description-table" cols="1" colsep="0" rowsep="0" width="100%">
<thead>
<tr class="description-tr">
<td class="description-td" colspan="1" nameend="1" namest="1" rowsep="1">TABLE 2</td>
</tr>
</thead>
<tbody><tr class="description-tr">
<td align="center" class="description-td" colspan="1" nameend="1" namest="1" rowsep="1"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">Relative activity without bile salts</td>
</tr>
</tbody></table>
<table align="left" class="description-table" cols="8" colsep="0" rowsep="0" width="100%">
<tbody><tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">Enzyme/pH</td>
<td class="description-td">2</td>
<td class="description-td">3</td>
<td class="description-td">4</td>
<td class="description-td">5</td>
<td class="description-td">6</td>
<td class="description-td">7</td>
</tr>
<tr class="description-tr">
<td align="center" class="description-td" colspan="8" nameend="8" namest="1" rowsep="1"> </td>
</tr>
</tbody></table>
<table align="left" class="description-table" cols="8" colsep="0" rowsep="0" width="100%">
<tbody><tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> <b>A</b>.<b>oryzae</b> </td>
<td class="description-td">0.0</td>
<td class="description-td">0.0</td>
<td class="description-td">77.4</td>
<td class="description-td">93.4</td>
<td class="description-td">100.0</td>
<td class="description-td">25.6</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">SEQ 2</td>
<td class="description-td">0.4</td>
<td class="description-td">0.2</td>
<td class="description-td">7.8</td>
<td class="description-td">51.8</td>
<td class="description-td">100.0</td>
<td class="description-td">76.1</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">SEQ 3</td>
<td class="description-td">0.3</td>
<td class="description-td">0.8</td>
<td class="description-td">2.8</td>
<td class="description-td">22.2</td>
<td class="description-td">79.7</td>
<td class="description-td">100.0</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">SEQ 1</td>
<td class="description-td">0.1</td>
<td class="description-td">1.8</td>
<td class="description-td">29.4</td>
<td class="description-td">100.0</td>
<td class="description-td">86.0</td>
<td class="description-td">71.1</td>
</tr>
<tr class="description-tr">
<td align="center" class="description-td" colspan="8" nameend="8" namest="1" rowsep="1"> </td>
</tr>
</tbody></table>
</patent-tables>
</tables>
</div>
<div class="description-paragraph" num="p-0188">
<tables id="TABLE-US-00003" num="00003">
<patent-tables colsep="0" frame="none" rowsep="0">
<table align="left" class="description-table" cols="1" colsep="0" rowsep="0" width="100%">
<thead>
<tr class="description-tr">
<td class="description-td" colspan="1" nameend="1" namest="1" rowsep="1">TABLE 3</td>
</tr>
</thead>
<tbody><tr class="description-tr">
<td align="center" class="description-td" colspan="1" nameend="1" namest="1" rowsep="1"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">Relative activity with bile salts</td>
</tr>
</tbody></table>
<table align="left" class="description-table" cols="8" colsep="0" rowsep="0" width="100%">
<tbody><tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">Enzyme/pH</td>
<td class="description-td">2</td>
<td class="description-td">3</td>
<td class="description-td">4</td>
<td class="description-td">5</td>
<td class="description-td">6</td>
<td class="description-td">7</td>
</tr>
<tr class="description-tr">
<td align="center" class="description-td" colspan="8" nameend="8" namest="1" rowsep="1"> </td>
</tr>
</tbody></table>
<table align="left" class="description-table" cols="8" colsep="0" rowsep="0" width="100%">
<tbody><tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> <b>A</b>.<b>oryzae</b> </td>
<td class="description-td">0.0</td>
<td class="description-td">0.0</td>
<td class="description-td">53.5</td>
<td class="description-td">71.8</td>
<td class="description-td">68.6</td>
<td class="description-td">16.1</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">SEQ 2</td>
<td class="description-td">0.1</td>
<td class="description-td">0.0</td>
<td class="description-td">0.5</td>
<td class="description-td">37.5</td>
<td class="description-td">85.8</td>
<td class="description-td">66.1</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">SEQ 3</td>
<td class="description-td">0.0</td>
<td class="description-td">0.0</td>
<td class="description-td">0.8</td>
<td class="description-td">2.5</td>
<td class="description-td">61.4</td>
<td class="description-td">78.1</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">SEQ 1</td>
<td class="description-td">0.0</td>
<td class="description-td">0.0</td>
<td class="description-td">10.4*</td>
<td class="description-td">76.0</td>
<td class="description-td">68.6</td>
<td class="description-td">59.7</td>
</tr>
<tr class="description-tr">
<td align="center" class="description-td" colspan="8" nameend="8" namest="1" rowsep="1"> </td>
</tr>
<tr class="description-tr">
<td align="left" class="description-td" colspan="8" id="FOO-00002" nameend="8" namest="1">*One measurement discarded for being clearly erroneous</td>
</tr>
</tbody></table>
</patent-tables>
</tables>
</div>
<div class="description-paragraph" num="p-0189">
<tables id="TABLE-US-00004" num="00004">
<patent-tables colsep="0" frame="none" rowsep="0">
<table align="left" class="description-table" cols="1" colsep="0" rowsep="0" width="100%">
<thead>
<tr class="description-tr">
<td class="description-td" colspan="1" nameend="1" namest="1" rowsep="1">TABLE 4</td>
</tr>
</thead>
<tbody><tr class="description-tr">
<td align="center" class="description-td" colspan="1" nameend="1" namest="1" rowsep="1"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">Normalized absolute activities relative to SEQ, 1 without bile salts</td>
</tr>
</tbody></table>
<table align="left" class="description-table" cols="8" colsep="0" rowsep="0" width="100%">
<tbody><tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">Enzyme/pH</td>
<td class="description-td">2</td>
<td class="description-td">3</td>
<td class="description-td">4</td>
<td class="description-td">5</td>
<td class="description-td">6</td>
<td class="description-td">7</td>
</tr>
<tr class="description-tr">
<td align="center" class="description-td" colspan="8" nameend="8" namest="1" rowsep="1"> </td>
</tr>
</tbody></table>
<table align="left" class="description-table" cols="8" colsep="0" rowsep="0" width="100%">
<tbody><tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> <b>A</b>.<b>oryzae</b> </td>
<td class="description-td">0.0</td>
<td class="description-td">0.0</td>
<td class="description-td">10.9</td>
<td class="description-td">13.2</td>
<td class="description-td">14.1</td>
<td class="description-td">3.6</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">SEQ 2</td>
<td class="description-td">0.1</td>
<td class="description-td">0.1</td>
<td class="description-td">2.1</td>
<td class="description-td">14.0</td>
<td class="description-td">27.1</td>
<td class="description-td">20.6</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">SEQ 3</td>
<td class="description-td">0.1</td>
<td class="description-td">0.4</td>
<td class="description-td">1.4</td>
<td class="description-td">10.7</td>
<td class="description-td">38.3</td>
<td class="description-td">48.0</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">SEQ 1</td>
<td class="description-td">0.1</td>
<td class="description-td">1.8</td>
<td class="description-td">29.4</td>
<td class="description-td">100.0</td>
<td class="description-td">86.0</td>
<td class="description-td">71.1</td>
</tr>
<tr class="description-tr">
<td align="center" class="description-td" colspan="8" nameend="8" namest="1" rowsep="1"> </td>
</tr>
</tbody></table>
</patent-tables>
</tables>
</div>
<div class="description-paragraph" num="p-0190">
<tables id="TABLE-US-00005" num="00005">
<patent-tables colsep="0" frame="none" rowsep="0">
<table align="left" class="description-table" cols="1" colsep="0" rowsep="0" width="100%">
<thead>
<tr class="description-tr">
<td class="description-td" colspan="1" nameend="1" namest="1" rowsep="1">TABLE 5</td>
</tr>
</thead>
<tbody><tr class="description-tr">
<td align="center" class="description-td" colspan="1" nameend="1" namest="1" rowsep="1"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">Normalized absolute activities relative to SEQ 1, with bile salts</td>
</tr>
</tbody></table>
<table align="left" class="description-table" cols="8" colsep="0" rowsep="0" width="100%">
<tbody><tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">Enzyme/pH</td>
<td class="description-td">2</td>
<td class="description-td">3</td>
<td class="description-td">4</td>
<td class="description-td">5</td>
<td class="description-td">6</td>
<td class="description-td">7</td>
</tr>
<tr class="description-tr">
<td align="center" class="description-td" colspan="8" nameend="8" namest="1" rowsep="1"> </td>
</tr>
</tbody></table>
<table align="left" class="description-table" cols="8" colsep="0" rowsep="0" width="100%">
<tbody><tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">
<b>A. oryzae</b>
</td>
<td class="description-td">0.0</td>
<td class="description-td">0.0</td>
<td class="description-td">9.9</td>
<td class="description-td">13.3</td>
<td class="description-td">12.7</td>
<td class="description-td">3.0</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">SEQ 2</td>
<td class="description-td">0.0</td>
<td class="description-td">0.0</td>
<td class="description-td">0.2</td>
<td class="description-td">13.4</td>
<td class="description-td">30.6</td>
<td class="description-td">23.6</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">SEQ 3</td>
<td class="description-td">0.0</td>
<td class="description-td">0.0</td>
<td class="description-td">0.5</td>
<td class="description-td">1.6</td>
<td class="description-td">38.8</td>
<td class="description-td">49.3</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">SEQ 1</td>
<td class="description-td">0.0</td>
<td class="description-td">0.0</td>
<td class="description-td">13.7</td>
<td class="description-td">100.0</td>
<td class="description-td">90.2</td>
<td class="description-td">78.6</td>
</tr>
<tr class="description-tr">
<td align="center" class="description-td" colspan="8" nameend="8" namest="1" rowsep="1"> </td>
</tr>
</tbody></table>
</patent-tables>
</tables>
</div>
<div class="description-paragraph" num="p-0191">These results show that although bile salts seem to slightly reduce the amylase activity, the activity in the presence of 5 mM bile salts is still satisfactory. The results also show that bile salts do not lead to a shift of the pH optimum.</div>
<div class="description-paragraph" num="p-0192">The results furthermore show that each of the <i>Bacillus </i>amylases of the invention all have more than 50% relative activity at pH 7, which is not the case for the comparative fungal amylase.</div>
<div class="description-paragraph" num="p-0193">Finally, Tables 4 and 5 demonstrate that, at least under these conditions, the amylase having amino acids 1-486 of SEQ ID NO: 1 has a significantly higher activity per mg enzyme than all the other amylases tested.</div>
<heading>Example 4</heading>
<heading>Amylase Degradation Profile</heading>
<div class="description-paragraph" num="p-0194">The degradation profile (DP1-10 qualitative fingerprint) of five alpha-amylases was determined by degradation of waxy maize starch followed by analysis on HPLC.</div>
<div class="description-paragraph" num="h-0032">Materials and Methods</div>
<div class="description-paragraph" num="p-0195">The enzymes were purified Bacillus amylases of amino acids 1-486 of SEQ ID NO: 1 (“SEQ 1”), amino acids 1-481 of SEQ ID NO: 2 (“SEQ 2”) and amino acids 1-483 of SEQ ID NO: 3 (“SEQ 3”), and, for comparison, a purified fungal amylase derived from <i>Aspergillus oryzae </i>(“<i>A. oryzae</i>”), the amylase of the commercial Fungamyl™ amylase product, which is available from Novozymes A/S, Krogshoejvej 36, DK-2880 Bagsvaerd, Denmark. Pancreatin was from Solvay Pharmaceuticals.</div>
<div class="description-paragraph" num="p-0196">As substrate a slurry of 50 mg/mL waxy maize starch (e.g. Waxy corn 04201 from Cerestar, Batch: WM5671) was made in 50 mM Na-acetate buffer pH 6.0, 40 ppm Ca<sup>2+</sup>.</div>
<div class="description-paragraph" num="p-0197">Purified enzymes were dosed at a concentration of 0.0073 mg enzyme protein per g of dry substance of the reaction mixture. The amount of amylase enzyme protein was calculated on the basis of the A<sub>280 </sub>values and the amino acid sequences (amino acid compositions) using the principles outlined in S. C. Gill &amp; P. H. von Hippel, Analytical Biochemistry 182, 319-326, (1989). After dosing the enzyme, the reaction was incubated at 60° C. Aliquots were withdrawn after 24 hours and diluted 1:1 in demineralized water and two drops of 1 M HCl was added. The enzymes were inactivated by boiling for 15 min. The samples were filtered through a 0.2 μm filter prior to application on the HPLC fitted with two Aminex HPX 42A (Biorad) columns connected in series and using water as eluent.</div>
<div class="description-paragraph" num="p-0198">100 mg of pancreatin was dissolved in 10 mL 50 mM Na-acetate buffer pH 5.5. 20 ul of this solution was added to 1 mL substrate slurry (50 mg/mL waxy corn starch in 50 mM Na-acetate pH 5.5) and incubated at 37° C. for 24 hours, and then inactivated and filtered as described above prior to application on the HPLC.</div>
<div class="description-paragraph" num="p-0199">In a separate experiment (not shown), the “SEQ 1” amylase was also tested at 37° C. which gave rise to the same degradation profile as at 60° C.</div>
<div class="description-paragraph" num="h-0033">Results</div>
<div class="description-paragraph" num="p-0200">The resulting chromatograms after 24 hours of incubation are shown in <figref idrefs="DRAWINGS">FIGS. 1-5</figref> together with indications as to which compound is represented by which peak in the chromatogram. For the sake of clarity, these indications have been inserted in <figref idrefs="DRAWINGS">FIG. 1</figref> only, but they apply mutatis mutandis to each of the other figures as well. The designation DP stands for Degree of Polymerization. Thus, DP1 designates the monomer, DP2 the dimer, DP3 the trimer, DP4 the tetramer, DP5 the pentamer, DP6 the hexamer, and so forth.</div>
<div class="description-paragraph" num="p-0201">It appears from the figures, that the fungal alpha-amylase (“<i>A. oryzae</i>”) generates DP2, DP3 and DP4 as the main degradation products, whereas it leaves a large fraction non-hydrolysed (large peak at retention time 10 minutes). DP1 was not detected as a degradation product.</div>
<div class="description-paragraph" num="p-0202">The bacterial alpha-amylases on the other hand (“SEQ 1”, “SEQ 2”, “SEQ 3”) in general produce DP1, DP2, DP3 and either of DP5 or DP6 as main products, leaving only a small part of non-hydrolysed substrate. Generally, a low amount of DP4 appears to be produced.</div>
<div class="description-paragraph" num="p-0203">As far as Pancreatin is concerned, it appears to behave like a mixture of the fungal and bacterial amylases described above, i.e. generating small DP products (DP1, DP2 and DP3), almost nothing of DP4, DP5 and DP6, and only leaving a small non-hydrolysed substrate peak.</div>
<heading>Example 5</heading>
<heading>Pharmaceutical Amylase Compositions</heading>
<div class="description-paragraph" num="h-0036">(A) High-strength Pellets</div>
<div class="description-paragraph" num="p-0204">A liquid concentrate was prepared as described in DK 2005 00931 (a germ-filtered ultrafiltrate) of the amylase having amino acids 1-486 of SEQ ID NO: 1. The liquid concentrate was spray-dried. The measured amylase protein content of the spray-dried amylase powder was 37%. 1125 g of the spray-dried amylase powder was dry pre-mixed together with microcrystalline cellulose (450 g) and polyethylene glycol 4000 (Macrogol™ 4000; 675 g) in a commercially available mixer. Isopropyl alcohol 100% (460 g) was added and the resulting wet mass was continued to be thoroughly mixed at room temperature. The homogenized mass was then extruded in a commercially available extruder which was fitted with a piercing die having a hole diameter of 0.8 mm to form cylindrical pellets. The bead temperature was not exceeding 50° C. while extruding. The extrudate produced was rounded to spherical pellets with a commercially available spheronizer by adding the necessary amount of isopropyl alcohol 100% (75.5 g). The pellets were dried at a supply temperature of approximately 40° C. in a commercially available vacuum dryer (from Voetsch). The product temperature did not exceed 45° C. The dried pellets were then separated by using a mechanical sieving machine with 0.7 and 1.4 mm screens. The sieve fractions of ≧0.7 mm and ≦1.4 mm were collected and can be filled in portions of 200 mg pellets each in capsules of size 2. The amylase concentration of the resulting dry pellets was approximately 18.5% (w/w).</div>
<div class="description-paragraph" num="h-0037">(B) Lower-strength Pellets</div>
<div class="description-paragraph" num="p-0205">Similar to the example provided above (A), pellets with a lower content of amylase were produced using 562.5 g of the same spray-dried amylase powder, microcrystalline cellulose (1125 g), polyethylene glycol 4000 (562.5 g), isopropyl alcohol for moistening (700 g) and isopropyl alcohol for rounding (41.6 g). The amylase concentration of the resulting dry pellets was approximately 9.3% (w/w).</div>
<div class="description-paragraph" num="p-0206">The resulting pellets from examples (A) and (B) were tested for amylolytic activity by applying a modified FIP method for microbial amylases. In principle, starch is hydrolysed by amylase at pH 5.8 and at a constant temperature (37.0+/−0.1° C.) in the presence of sodium chloride and calcium chloride. The reducing groups resulting from the hydrolysis react with iodine in alkaline solution and the excess of iodine was titrated with thiosulphate. One unit of amylase is defined as the amount of enzyme, which, under the defined conditions and substrate, hydrolyzes 1 micromol of glycosidic bond per minute. The amylolytic activity in the pellets was found to be approximately 96% relative to the amylolytic activity in the starting powdery amylase material for each of the examples (A) and (B).</div>
<div class="description-paragraph" num="p-0207">The resulting pellets from examples (A) and (B) were then tested for disintegration according to Pharm. Eur. 2.9.1. (Section “Disintegration of tablets and capsules”) (test solution: water—500 mL, 37° C.). The disintegration of the pellets from example (A) was completed within 15 min. The disintegration of the pellets from example (B) was completed within 9 min.</div>
<heading>Example 6</heading>
<heading>Pharmaceutical Compositions of Amylase and Protease</heading>
<div class="description-paragraph" num="p-0208">High-strength pellets containing amylase and protease were prepared as follows:</div>
<div class="description-paragraph" num="p-0209">A germ-filtered liquid concentrate of the protease of amino acids 1-274 of SEQ ID NO: 5 was prepared as described in Example 1 of DK patent application no. 2005 00930 and spray-dried. Spray-dried amylase in powder form (398.5 g) prepared as described in Example 5 was dry pre-mixed together with the spray-dried protease powder (746.5 g, having a measured protease protein content of 58.5%), microcrystalline cellulose (458 g) and polyethylene glycol 4000 (Macrogol™ 4000; 687 g) in a commercially available mixer. Isopropyl alcohol 100% (460 g) was added and the resulting wet mass was continued to be thoroughly mixed at room temperature. The homogenized mass was then extruded in a commercially available extruder which was fitted with a piercing die having a hole diameter of 0.8 mm to form cylindrical pellets. The bead temperature was not exceeding 50° C. while extruding. The extrudate produced was rounded to spherical pellets with a commercially available spheronizer by adding the necessary amount of isopropyl alcohol 100% (58 g). The pellets were dried using a supply temperature of approximately 40° C. in a commercially available vacuum dryer (from Voetsch). The product temperature did not exceed 45° C. The dried pellets were then separated by using a mechanical sieving machine with 0.7 and 1.4 mm screens. The sieve fractions of ≧0.7 mm and ≦1.4 mm were collected and can be filled in portions of 200 mg each in capsules of size 2.</div>
<div class="description-paragraph" num="p-0210">The resulting pellets from were tested for proteolytic and amylolytic activities according to the methods as outlined above. No loss in proteolytic or amylolytic activity was found in the pellets in each case relative to the starting powdery protease or amylase material, respectively.</div>
</div>
</div>
</section><section itemprop="claims" itemscope="">
<h2>Claims (<span itemprop="count">20</span>)</h2>
<div html="" itemprop="content"><div class="claims" lang="EN" load-source="patent-office" mxw-id="PCLM38536326">
<div class="claim"> <div class="claim" id="CLM-00001" num="00001">
<div class="claim-text">1. A method for the treatment of digestive disorders, pancreatic exocrine insufficiency, pancreatitis, cystic fibrosis, diabetes type I, and/or diabetes type II, by administering an effective amount of an amylase having at least 95% identity to (i) amino acids 1-481 of SEQ ID NO: 1, (ii) amino acids 1-481 of SEQ ID NO: 2, and/or (iii) amino acids 1-483 of SEQ ID NO: 3 to an animal in need thereof.</div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="CLM-00002" num="00002">
<div class="claim-text">2. The method of <claim-ref idref="CLM-00001">claim 1</claim-ref>, further comprising administering an effective amount of a lipase or a protease to the animal.</div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="CLM-00003" num="00003">
<div class="claim-text">3. The method of <claim-ref idref="CLM-00001">claim 1</claim-ref>, further comprising administering an effective amount of a lipase and a protease to the animal.</div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="CLM-00004" num="00004">
<div class="claim-text">4. The method of <claim-ref idref="CLM-00002">claim 2</claim-ref>, wherein the protease has at least 95% identity to a protease selected from the group consisting of:
<div class="claim-text">(a) a protease having amino acids 1-274 of SEQ ID NO: 5,</div>
<div class="claim-text">(b) a protease having amino acids 1-188 of SEQ ID NO: 6, and</div>
<div class="claim-text">(c) a protease having amino acids 1-188 of SEQ ID NO: 7.</div>
</div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="CLM-00005" num="00005">
<div class="claim-text">5. The method of <claim-ref idref="CLM-00002">claim 2</claim-ref>, wherein the protease is selected from the group consisting of:
<div class="claim-text">a) a protease comprising amino acids 1-274 of SEQ ID NO: 5,</div>
<div class="claim-text">b) a protease comprising amino acids 1-188 of SEQ ID NO: 6, and</div>
<div class="claim-text">c) a protease comprising amino acids 1-188 of SEQ ID NO: 7.</div>
</div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="CLM-00006" num="00006">
<div class="claim-text">6. The method of <claim-ref idref="CLM-00001">claim 1</claim-ref>, wherein the amylase has an amino acid sequence selected from the group consisting of (i) amino acids 1-481, 1-484, 1-486, or 1-513 of SEQ ID NO: 1, (ii) amino acids 1-481 of SEQ ID NO: 2, and/or (iii) amino acids 1-483 of SEQ ID NO: 3.</div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="CLM-00007" num="00007">
<div class="claim-text">7. The method of <claim-ref idref="CLM-00001">claim 1</claim-ref>, wherein the amylase has an amino acid sequence consisting of amino acids 1-481, 1-484, 1-486, or 1-513 of SEQ ID NO: 1.</div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="CLM-00008" num="00008">
<div class="claim-text">8. The method of <claim-ref idref="CLM-00001">claim 1</claim-ref>, wherein the amylase has an amino acid sequence consisting of amino acids 1-481 of SEQ ID NO: 2.</div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="CLM-00009" num="00009">
<div class="claim-text">9. The method of <claim-ref idref="CLM-00001">claim 1</claim-ref>, wherein the amylase has an amino acid sequence consisting of amino acids amino acids 1-483 of SEQ ID NO: 3.</div>
</div>
</div> <div class="claim"> <div class="claim" id="CLM-00010" num="00010">
<div class="claim-text">10. A method of treating pancreatic exocrine insufficiency comprising administering an effective amount of an amylase having at least 95% identity to (i) amino acids 1-481 of SEQ ID NO: 1, (ii) amino acids 1-481 of SEQ ID NO: 2, and/or (iii) amino acids 1-483 of SEQ ID NO: 3 to an animal in need thereof.</div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="CLM-00011" num="00011">
<div class="claim-text">11. The method of <claim-ref idref="CLM-00010">claim 10</claim-ref>, wherein the amylase has an amino acid sequence selected from the group consisting of (i) amino acids 1-481, 1-484, 1-486, or 1-513 of SEQ ID NO: 1, (ii) amino acids 1-481 of SEQ ID NO: 2, and/or (iii) amino acids 1-483 of SEQ ID NO: 3.</div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="CLM-00012" num="00012">
<div class="claim-text">12. The method of <claim-ref idref="CLM-00010">claim 10</claim-ref>, wherein the amylase has an amino acid sequence consisting of amino acids 1-481, 1-484, 1-486, or 1-513 of SEQ ID NO: 1.</div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="CLM-00013" num="00013">
<div class="claim-text">13. The method of <claim-ref idref="CLM-00010">claim 10</claim-ref>, wherein the amylase has an amino acid sequence consisting of amino acids 1-481 of SEQ ID NO: 2.</div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="CLM-00014" num="00014">
<div class="claim-text">14. The method of <claim-ref idref="CLM-00010">claim 10</claim-ref>, wherein the amylase has an amino acid sequence consisting of amino acids amino acids 1-483 of SEQ ID NO: 3.</div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="CLM-00015" num="00015">
<div class="claim-text">15. The method of <claim-ref idref="CLM-00010">claim 10</claim-ref>, further comprising administering an effective amount of a lipase or a protease.</div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="CLM-00016" num="00016">
<div class="claim-text">16. The method of <claim-ref idref="CLM-00015">claim 15</claim-ref>, wherein the protease has at least 95% identity to a protease selected from the group consisting of:
<div class="claim-text">(a) a protease having amino acids 1-274 of SEQ ID NO: 5,</div>
<div class="claim-text">(b) a protease having amino acids 1-188 of SEQ ID NO: 6, and</div>
<div class="claim-text">(c) a protease having amino acids 1-188 of SEQ ID NO: 7.</div>
</div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="CLM-00017" num="00017">
<div class="claim-text">17. The method of <claim-ref idref="CLM-00015">claim 15</claim-ref>, wherein the protease is selected from the group consisting of:
<div class="claim-text">a) a protease comprising amino acids 1-274 of SEQ ID NO: 5,</div>
<div class="claim-text">b) a protease comprising amino acids 1-188 of SEQ ID NO: 6, and</div>
<div class="claim-text">c) a protease comprising amino acids 1-188 of SEQ ID NO: 7.</div>
</div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="CLM-00018" num="00018">
<div class="claim-text">18. The method of <claim-ref idref="CLM-00001">claim 1</claim-ref>, wherein the animal is human.</div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="CLM-00019" num="00019">
<div class="claim-text">19. The method of <claim-ref idref="CLM-00010">claim 10</claim-ref>, wherein the animal is human.</div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="CLM-00020" num="00020">
<div class="claim-text">20. The method of <claim-ref idref="CLM-00010">claim 10</claim-ref>, further comprising administering an effective amount of a lipase and a protease to the animal.</div>
</div>
</div> </div>
</div>
</section>
                </article>
            </search-app>
        </body>
    </html>
    