
    <html>
        <body>
            <search-app>
                <article class="result" itemscope="" itemtype="http://schema.org/ScholarlyArticle">
    <h1 itemprop="pageTitle">US20030148934A1 - Growth factor modulators 
        - Google Patents</h1><section itemprop="abstract" itemscope="">
<h2>Abstract</h2>
<div html="" itemprop="content"><abstract lang="EN" load-source="patent-office" mxw-id="PA48621877">
<div class="abstract" id="A-0001">The invention provides human growth factor modulators (GFMO) and polynucleotides which identify and encode GFMO. The invention also provides expression vectors, host cells, antibodies, agonists, and antagonists. The invention also provides methods for diagnosing, treating or preventing disorders associated with expression of GFMO. </div>
</abstract>
</div>
</section><section itemprop="description" itemscope="">
<h2>Description</h2>
<div html="" itemprop="content"><ul class="description" lang="EN" load-source="patent-office" mxw-id="PDES68726129">
<summary-of-invention>
<heading>FIELD OF THE INVENTION </heading>
<li> <para-num num="[0001]"> </para-num> <div class="description-line" id="P-0001" num="0001">This invention relates to nucleic acid and amino acid sequences of growth factor modulators and to the use of these sequences in the diagnosis, treatment, and prevention of cancer and fibrotic disorders. <span style="display: none">[0001] </span> </div>
</li> <heading>BACKGROUND OF THE INVENTION </heading>
<li> <para-num num="[0002]"> </para-num> <div class="description-line" id="P-0002" num="0002">Growth factors exert mitogenic affects on cells during development, wound repair, and tissue regeneration. They are also involved in pathologies including fibrotic disorders, such as atherosclerosis, and cancer. Growth factors stimulate cells through cell surface receptors, which are typically associated with tyrosine or serine/threonine kinase activity. In recent years, it has become apparent that several growth factors also bind to a second class of protein. These proteins are proteoglycans associated with the cell surface or the extracellular matrix. Although these binding proteins cannot transmit signals, they appear to modulate the ability of the growth factor to generate a biological response. (Schlessinger, J. et al. (1995) Cell 83:357-360.) <span style="display: none">[0002] </span> </div>
</li> <li> <para-num num="[0003]"> </para-num> <div class="description-line" id="P-0003" num="0003">One family of low affinity growth factor binding protein is the CCN family, which includes connective tissue growth factor (CTGF), Elm1, cef10/cyr61, and neuroblastoma overexpressed gene (nov). The CCN family represents growth factor early response genes. CTGF is induced by TGFβ signaling and appears to be necessary for cell proliferation and type I collagen, fibronectin, and α5 integrin expression in stimulated fibroblasts. Overexpression of CTGF appears to be necessary for wound repair, but may trigger pathological processes such as glomerulosclerosis, lung fibrosis, and liver cirrhosis. CTGF is also expressed at high levels in atherosclerotic, but not normal, blood vessels. Cyr61/CEF10 was identified as an immediate early gene induced in p60<span style="display: none">[0003] </span> <sup>v-src</sup>-transformed chicken embryo fibroblasts (CEFs). It has been shown to potentiate bFGF mitogenic response in fibroblast and endothelial cells, and to promote cell proliferation, migration, and adhesion in NIH 3T3 cells. On the other hand, nov is down regulated in p60<sup>v-src</sup>-transformed CEFs and shows reduced expression levels in Wilms' tumors when compared to normal kidney. Consistent with these observations, nov was found to be growth inhibitory when overexpressed in CEFs. Thus, nov may balance the mitogenic affects of other CCN family members. (Oemar, B. S. and Lüscher, T. F. (1 997) Arterioscler Thromb. Vasc. Biol. 17:1483-1489.) Similarly, Elm1, identified as a protein expressed in low-but not high-metastatic melanoma cells, was found to suppress the in vivo growth and metastatic potential of mouse melanoma cells. (Hashimoto, Y. et al. (1998) J. Exp. Med. 187:289-296.) </div>
</li> <li> <para-num num="[0004]"> </para-num> <div class="description-line" id="P-0004" num="0004">The CCN family is characterized by an absolute conservation of 38 cysteine residues that constitute more than 10% of the total amino acid content. All CCN family members have a signal peptide and are secreted. They also contain four distinct modules, each encoded by a separate exon. The amino terminal half of the molecule consists of an insulin-like growth factor binding domain, common to low affinity IGF binding proteins (IGFBPs), and a von Willebrand factor type C (VWFC) domain. The VWFC domain contains a series of cysteine-rich repeats, which are also found in procollagen and thrombospondin, that are thought to be involved in protein oligomerization. The carboxyl terminal half of CCN family molecules contains a thrombospondin type I repeat, which has been found to interact with sulfated glycoconjugates like heparin, and a cysteine knot motif. The cysteine knot, also found in the growth factors TGFβ, PDGF, and NGF, may be involved in dimerization of protein subunits. (Grotendorst, G. R. (1997) Cytokine Growth Factor Rev. 8:171-179.) <span style="display: none">[0004] </span> </div>
</li> <li> <para-num num="[0005]"> </para-num> <div class="description-line" id="P-0005" num="0005">The FGF-binding protein (FGF-BP) represents another type of growth factor binding protein. The FGF-BP is smaller than the CCN family members and shares little sequence homology. The mouse and human FGF-BPs are approximately 18 kDa secreted peptides having 63% amino acid sequence identity. Expression is restricted to squamous epithelia in humans, though expression is seen in intestine, lung, and ovaries during different developmental stages in mice. Similar to CCN proteins, FGF-BP functions as a modulator of FGF in responsive cell types. The human FGF-BP, HBp17, was found to inhibit the biological activities of both FGF-1 and FGF-2. Expression of FGF-BP in adult mouse skin increased during early stages of carcinogen-induced transformation in vivo and ras-activation in vitro. This is may suggest a role for FGF-BP in tumorigenesis. (Kurtz, A. et al. (1997) Oncogene 14:2671-2681.) The discovery of new growth factor modulators and the polynucleotides encoding them satisfies a need in the art by providing new compositions which are useful in the diagnosis, treatment, and prevention of cancer and fibrotic disorders. <span style="display: none">[0005] </span> </div>
</li> <heading>SUMMARY OF THE INVENTION </heading>
<li> <para-num num="[0006]"> </para-num> <div class="description-line" id="P-0006" num="0006">The invention features substantially purified polypeptides, growth factor modulators, referred to collectively as “GFMO” and individually as “GFMO-1” and “GFMO-2.” In one aspect, the invention provides a substantially purified polypeptide comprising an amino acid sequence selected from the group consisting of SEQ ID NO: 1, SEQ ID NO: 2, a fragment of SEQ ID NO: 1, and a fragment of SEQ ID NO: 2. <span style="display: none">[0006] </span> </div>
</li> <li> <para-num num="[0007]"> </para-num> <div class="description-line" id="P-0007" num="0007">The invention further provides a substantially purified variant having at least 70% amino acid identity to the amino acid sequences of SEQ ID NO: 1 or SEQ ID NO: 2, or to a fragment of either of these sequences. The invention also provides an isolated and purified polynucleotide encoding the polypeptide comprising an amino acid sequence selected from the group consisting of SEQ ID NO: 1, SEQ ID NO: 2, a fragment of SEQ ID NO: 1, and a fragment of SEQ ID NO: 2. The invention also includes an isolated and purified polynucleotide variant having at least 90% polynucleotide seqeunce identity to the polynucleotide encoding the polypeptide comprising an amino acid sequence selected from the group consisting of SEQ ID NO: 1, SEQ ID NO: 2, a fragment of SEQ ID NO: 1, and a fragment of SEQ ID NO: 2. <span style="display: none">[0007] </span> </div>
</li> <li> <para-num num="[0008]"> </para-num> <div class="description-line" id="P-0008" num="0008">Additionally, the invention provides an isolated and purified polynucleotide which hybridizes under stringent conditions to the polynucleotide encoding the polypeptide comprising an amino acid sequence selected from the group consisting of SEQ ID NO: 1, SEQ ID NO: 2, a fragment of SEQ ID NO: 1, and a fragment of SEQ ID NO: 2, as well as an isolated and purified polynucleotide having a sequence which is complementary to the polynucleotide encoding the polypeptide comprising the amino acid sequence selected from the group consisting of SEQ ID NO: 1, SEQ ID NO: 2, a fragment of SEQ ID NO: 1, and a fragment of SEQ ID NO: 2. <span style="display: none">[0008] </span> </div>
</li> <li> <para-num num="[0009]"> </para-num> <div class="description-line" id="P-0009" num="0009">The invention also provides an isolated and purified polynucleotide comprising a polynucleotide sequence selected from the group consisting of SEQ ID NO: 3, SEQ ID NO: 4, a fragment of SEQ ID NO: 3, and a fragment of SEQ ID NO: 4. The invention further provides an isolated and purified polynucleotide variant having at least 70% polynucleotide sequence identity to the polynucleotide sequence comprising a polynucleotide sequence selected from the group consisting of SEQ ID NO: 3, SEQ ID NO: 4, a fragment of SEQ ID NO: 3, and a fragment of SEQ ID NO: 4, as well as an isolated and purified polynucleotide having a sequence which is complementary to the polynucleotide comprising a polynucleotide sequence selected from the group consisting of SEQ ID NO: 3, SEQ ID NO: 4, a fragment of SEQ ID NO: 3, and a fragment of SEQ ID NO: 4. <span style="display: none">[0009] </span> </div>
</li> <li> <para-num num="[0010]"> </para-num> <div class="description-line" id="P-0010" num="0010">The invention further provides an expression vector containing at least a fragment of the polynucleotide encoding the polypeptide comprising an amino acid sequence selected from the group consisting of SEQ ID NO: 1, SEQ ID NO: 2, a fragment of SEQ ID NO: 1, and a fragment of SEQ ID NO: 2. In another aspect, the expression vector is contained within a host cell. <span style="display: none">[0010] </span> </div>
</li> <li> <para-num num="[0011]"> </para-num> <div class="description-line" id="P-0011" num="0011">The invention also provides a method for producing a polypeptide, the method comprising the steps of: (a) culturing the host cell containing an expression vector containing at least a fragment of a polynucleotide encoding the polypeptide under conditions suitable for the expression of the polypeptide; and (b) recovering the polypeptide from the host cell culture. <span style="display: none">[0011] </span> </div>
</li> <li> <para-num num="[0012]"> </para-num> <div class="description-line" id="P-0012" num="0012">The invention also provides a pharmaceutical composition comprising a substantially purified polypeptide having the amino acid sequence selected from the group consisting of SEQ ID NO: 1, SEQ ID NO: 2, a fragment of SEQ ID NO: 1, and a fragment of SEQ ID NO: 2 in conjunction with a suitable pharmaceutical carrier. <span style="display: none">[0012] </span> </div>
</li> <li> <para-num num="[0013]"> </para-num> <div class="description-line" id="P-0013" num="0013">The invention further includes a purified antibody which binds to a polypeptide comprising the sequence of SEQ ID NO: 1, SEQ ID NO: 2, a fragment of SEQ ID NO: 1, or a fragment of SEQ ID NO: 2, as well as a purified agonist and a purified antagonist of the polypeptide. <span style="display: none">[0013] </span> </div>
</li> <li> <para-num num="[0014]"> </para-num> <div class="description-line" id="P-0014" num="0014">The invention also provides a method for treating or preventing a cancer, the method comprising administering to a subject in need of such treatment an effective amount of a pharmaceutical composition comprising a substantially purified polypeptide having an amino acid sequence selected from the group consisting of SEQ ID NO: 1, SEQ ID NO: 2, a fragment of SEQ ID NO: 1, and a fragment of SEQ ID NO: 2. <span style="display: none">[0014] </span> </div>
</li> <li> <para-num num="[0015]"> </para-num> <div class="description-line" id="P-0015" num="0015">The invention also provides a method for treating or preventing a fibrotic disorder, the method comprising administering to a subject in need of such treatment an effective amount of an antagonist of the polypeptide having an amino acid sequence selected from the group consisting of SEQ ID NO: 1, SEQ ID NO: 2, a fragment of SEQ ID NO: 1, and a fragment of SEQ ID NO: 2. <span style="display: none">[0015] </span> </div>
</li> <li> <para-num num="[0016]"> </para-num> <div class="description-line" id="P-0016" num="0016">The invention also provides a method for detecting a polynucleotide, the method comprising the steps of: (a) hybridizing the complement of the polynucleotide sequence encoding the polypeptide comprising the amino acid sequence selected from the group consisting of SEQ ID NO: 1, SEQ ID NO: 2, a fragment of SEQ ID NO: 1, and a fragment of SEQ ID NO: 2 to at least one of the nucleic acids of a biological sample, thereby forming a hybridization complex; and (b) detecting the hybridization complex, wherein the presence of the hybridization complex correlates with the presence of a polynucleotide encoding the polypeptide in the biological sample. In one aspect, the method further comprises amplifying the polynucleotide prior to hybridization.<span style="display: none">[0016] </span> </div>
</li> </summary-of-invention>
<heading>BRIEF DESCRIPTION OF THE FIGURES AND TABLE </heading>
<li> <para-num num="[0017]"> </para-num> <div class="description-line" id="P-0017" num="0017">FIGS. 1A, 1B, <span style="display: none">[0017]</span> <b>1</b>C, and ID show the amino acid sequence (SEQ ID NO: 1) and nucleic acid sequence (SEQ ID NO: 3) of GFMO-1. The alignment was produced using MacDNASIS PRO™ software (Hitachi Software Engineering Co. Ltd., San Bruno, Calif.). </div>
</li> <li> <para-num num="[0018]"> </para-num> <div class="description-line" id="P-0018" num="0018">FIGS. 2A, 2B, and <span style="display: none">[0018]</span> <b>2</b>C show the amino acid sequence (SEQ ID NO: 2) and nucleic acid sequence (SEQ ID NO: 4) of GFMO-2. The alignment was produced using MacDNASIS PRO™ software. </div>
</li> <li> <para-num num="[0019]"> </para-num> <div class="description-line" id="P-0019" num="0019">FIGS. 3A and 3B show the amino acid sequence alignments between GFMO-1 (Incyte Clone 2509339; SEQ ID NO: 1) and mouse Elm1 (GI 2911144; SEQ ID NO: 13), produced using the multisequence alignment program of LASERGENE™ software (DNASTAR Inc, Madison Wis.). <span style="display: none">[0019]</span> </div>
</li> <li> <para-num num="[0020]"> </para-num> <div class="description-line" id="P-0020" num="0020">FIGS. 4A and 4B show the amino acid sequence alignments between GFMO-2 (Incyte Clone 2840746; SEQ ID NO: 2), and mouse FGF-binding protein (GI 1469936; SEQ ID NO: 14), produced using the multisequence alignment program of LASERGENE™ software.<span style="display: none">[0020]</span> </div>
</li> <li> <para-num num="[0021]"> </para-num> <div class="description-line" id="P-0021" num="0021">Table 1 summarizes the software programs, corresponding algorithms, references, and cutoff parameters used to analyze ESTs and full length polynucleotide and amino acid sequences where applicable. <span style="display: none">[0021] </span> </div>
</li> <heading>DESCRIPTION OF THE INVENTION </heading>
<li> <para-num num="[0022]"> </para-num> <div class="description-line" id="P-0022" num="0022">Before the present proteins, nucleotide sequences, and methods are described, it is understood that this invention is not limited to the particular methodology, protocols, cell lines, vectors, and reagents described, as these may vary. It is also to be understood that the terminology used herein is for the purpose of describing particular embodiments only, and is not intended to limit the scope of the present invention which will be limited only by the appended claims. <span style="display: none">[0022] </span> </div>
</li> <li> <para-num num="[0023]"> </para-num> <div class="description-line" id="P-0023" num="0023">It must be noted that as used herein and in the appended claims, the singular forms “a,” “an,” and “the” include plural reference unless the context clearly dictates otherwise. Thus, for example, a reference to “a host cell” includes a plurality of such host cells, and a reference to “an antibody” is a reference to one or more antibodies and equivalents thereof known to those skilled in the art, and so forth. <span style="display: none">[0023] </span> </div>
</li> <li> <para-num num="[0024]"> </para-num> <div class="description-line" id="P-0024" num="0024">Unless defined otherwise, all technical and scientific terms used herein have the same meanings as commonly understood by one of ordinary skill in the art to which this invention belongs. Although any methods and materials similar or equivalent to those described herein can be used in the practice or testing of the present invention, the preferred methods, devices, and materials are now described. All publications mentioned herein are cited for the purpose of describing and disclosing the cell lines, vectors, and methodologies which are reported in the publications and which might be used in connection with the invention. Nothing herein is to be construed as an admission that the invention is not entitled to antedate such disclosure by virtue of prior invention. <span style="display: none">[0024] </span> </div>
</li> <li> <para-num num="[0025]"> </para-num> <div class="description-line" id="P-0025" num="0025">Definitions <span style="display: none">[0025] </span> </div>
</li> <li> <para-num num="[0026]"> </para-num> <div class="description-line" id="P-0026" num="0026">“GFMO,” as used herein, refers to the amino acid sequences, or variant thereof, of substantially purified GFMO obtained from any species, particularly a mammalian species, including bovine, ovine, porcine, murine, equine, and preferably the human species, from any source, whether natural, synthetic, semi-synthetic, or recombinant. <span style="display: none">[0026] </span> </div>
</li> <li> <para-num num="[0027]"> </para-num> <div class="description-line" id="P-0027" num="0027">The term “agonist,” as used herein, refers to a molecule which, when bound to GFMO, increases or prolongs the duration of the effect of GFMO. Agonists may include proteins, nucleic acids, carbohydrates, or any other molecules which bind to and modulate the effect of GFMO. <span style="display: none">[0027] </span> </div>
</li> <li> <para-num num="[0028]"> </para-num> <div class="description-line" id="P-0028" num="0028">An “allelic variant,” as this term is used herein, is an alternative form of the gene encoding GFMO. Allelic variants may result from at least one mutation in the nucleic acid sequence and may result in altered mRNAs or in polypeptides whose structure or function may or may not be altered. Any given natural or recombinant gene may have none, one, or many allelic forms. Common mutational changes which give rise to allelic variants are generally ascribed to natural deletions, additions, or substitutions of nucleotides. Each of these types of changes may occur alone, or in combination with the others, one or more times in a given sequence. <span style="display: none">[0028] </span> </div>
</li> <li> <para-num num="[0029]"> </para-num> <div class="description-line" id="P-0029" num="0029">“Altered” nucleic acid sequences encoding GFMO, as described herein, include those sequences with deletions, insertions, or substitutions of different nucleotides, resulting in a polynucleotide the same as GFMO or a polypeptide with at least one functional characteristic of GFMO. Included within this definition are polymorphisms which may or may not be readily detectable using a particular oligonucleotide probe of the polynucleotide encoding GFMO, and improper or unexpected hybridization to allelic variants, with a locus other than the normal chromosomal locus for the polynucleotide sequence encoding GFMO. The encoded protein may also be “altered,” and may contain deletions, insertions, or substitutions of amino acid residues which produce a silent change and result in a functionally equivalent GFMO. Deliberate amino acid substitutions may be made on the basis of similarity in polarity, charge, solubility, hydrophobicity, hydrophilicity, and/or the amphipathic nature of the residues, as long as the biological or immunological activity of GFMO is retained. For example, negatively charged amino acids may include aspartic acid and glutamic acid, positively charged amino acids may include lysine and arginine, and amino acids with uncharged polar head groups having similar hydrophilicity values may include leucine, isoleucine, and valine; glycine and alanine; asparagine and glutamine; serine and threonine; and phenylalanine and tyrosine. <span style="display: none">[0029] </span> </div>
</li> <li> <para-num num="[0030]"> </para-num> <div class="description-line" id="P-0030" num="0030">The terms “amino acid” or “amino acid sequence,” as used herein, refer to an oligopeptide, peptide, polypeptide, or protein sequence, or a fragment of any of these, and to naturally occurring or synthetic molecules. In this context, “fragments,” “immunogenic fragments,” or “antigenic fragments” refer to fragments of GFMO which are preferably at least 5 to about 15 amino acids in length, most preferably at least 14 amino acids, and which retain some biological activity or immunological activity of GFMO. Where “amino acid sequence” is recited herein to refer to an amino acid sequence of a naturally occurring protein molecule, “amino acid sequence” and like terms are not meant to limit the amino acid sequence to the complete native amino acid sequence associated with the recited protein molecule. <span style="display: none">[0030] </span> </div>
</li> <li> <para-num num="[0031]"> </para-num> <div class="description-line" id="P-0031" num="0031">“Amplification,” as used herein, relates to the production of additional copies of a nucleic acid sequence. Amplification is generally carried out using polymerase chain reaction (PCR) technologies well known in the art. (See, e.g., Dieffenbach, C. W. and G. S. Dveksler (1995) <span style="display: none">[0031] </span> <i>PCR Primer, a Laboratory Manual</i>, Cold Spring Harbor Press, Plainview, N.Y., pp.1-5.) </div>
</li> <li> <para-num num="[0032]"> </para-num> <div class="description-line" id="P-0032" num="0032">The term “antagonist,” as it is used herein, refers to a molecule which, when bound to GFMO, decreases the amount or the duration of the effect of the biological or immunological activity of GFMO. Antagonists may include proteins, nucleic acids, carbohydrates, antibodies, or any other molecules which decrease the effect of GFMO. <span style="display: none">[0032] </span> </div>
</li> <li> <para-num num="[0033]"> </para-num> <div class="description-line" id="P-0033" num="0033">As used herein, the term “antibody” refers to intact molecules as well as to fragments thereof, such as Fab, F(ab′)<span style="display: none">[0033] </span> <sub>2</sub>, and Fv fragments, which are capable of binding the epitopic determinant. Antibodies that bind GFMO polypeptides can be prepared using intact polypeptides or using fragments containing small peptides of interest as the immunizing antigen. The polypeptide or oligopeptide used to immunize an animal (e.g., a mouse, a rat, or a rabbit) can be derived from the translation of RNA, or synthesized chemically, and can be conjugated to a carrier protein if desired. Commonly used carriers that are chemically coupled to peptides include bovine serum albumin, thyroglobulin, and keyhole limpet hemocyanin (KLH). The coupled peptide is then used to immunize the animal. </div>
</li> <li> <para-num num="[0034]"> </para-num> <div class="description-line" id="P-0034" num="0034">The term “antigenic determinant,” as used herein, refers to that fragment of a molecule (i.e., an epitope) that makes contact with a particular antibody. When a protein or a fragment of a protein is used to immunize a host animal, numerous regions of the protein may induce the production of antibodies which bind specifically to antigenic determinants (given regions or three-dimensional structures on the protein). An antigenic determinant may compete with the intact antigen (i.e., the immunogen used to elicit the immune response) for binding to an antibody. <span style="display: none">[0034] </span> </div>
</li> <li> <para-num num="[0035]"> </para-num> <div class="description-line" id="P-0035" num="0035">The term “antisense,” as used herein, refers to any composition containing a nucleic acid sequence which is complementary to the “sense” strand of a specific nucleic acid sequence. Antisense molecules may be produced by any method including synthesis or transcription. Once introduced into a cell, the complementary nucleotides combine with natural sequences produced by the cell to form duplexes and to block either transcription or translation. The designation “negative” can refer to the antisense strand, and the designation “positive” can refer to the sense strand. <span style="display: none">[0035] </span> </div>
</li> <li> <para-num num="[0036]"> </para-num> <div class="description-line" id="P-0036" num="0036">As used herein, the term “biologically active,” refers to a protein having structural, regulatory, or biochemical functions of a naturally occurring molecule. Likewise, “immunologically active” refers to the capability of the natural, recombinant, or synthetic GFMO, or of any oligopeptide thereof, to induce a specific immune response in appropriate animals or cells and to bind with specific antibodies. <span style="display: none">[0036] </span> </div>
</li> <li> <para-num num="[0037]"> </para-num> <div class="description-line" id="P-0037" num="0037">The terms “complementary” or “complementarity,” as used herein, refer to the natural binding of polynucleotides by base pairing. For example, the sequence “5′A-G-T 3′” binds to the complementary sequence “3′T-C-A 5′.” Complementarity between two single-stranded molecules may be “partial,” such that only some of the nucleic acids bind, or it may be “complete,” such that total complementarity exists between the single stranded molecules. The degree of complementarity between nucleic acid strands has significant effects on the efficiency and strength of the hybridization between the nucleic acid strands. This is of particular importance in amplification reactions, which depend upon binding between nucleic acids strands, and in the design and use of peptide nucleic acid (PNA) molecules. <span style="display: none">[0037] </span> </div>
</li> <li> <para-num num="[0038]"> </para-num> <div class="description-line" id="P-0038" num="0038">A “composition comprising a given polynucleotide sequence” or a “composition comprising a given amino acid sequence,” as these terms are used herein, refer broadly to any composition containing the given polynucleotide or amino acid sequence. The composition may comprise a dry formulation or an aqueous solution. Compositions comprising polynucleotide sequences encoding GFMO or fragments of GFMO may be employed as hybridization probes. The probes may be stored in freeze-dried form and may be associated with a stabilizing agent such as a carbohydrate. In hybridizations, the probe may be deployed in an aqueous solution containing salts, e.g., NaCl, detergents, e.g.,sodium dodecyl sulfate (SDS), and other components, e.g., Denhardt's solution, dry milk, salmon sperm DNA, etc. <span style="display: none">[0038] </span> </div>
</li> <li> <para-num num="[0039]"> </para-num> <div class="description-line" id="P-0039" num="0039">“Consensus sequence,” as used herein, refers to a nucleic acid sequence which has been resequenced to resolve uncalled bases, extended using XL-PCR™ (The Perkin-Elmer Corp., Norwalk, Conn.) in the 5′ and/or the 3′ direction, and resequenced, or which has been assembled from the overlapping sequences of more than one Incyte Clone using a computer program for fragment assembly, such as the GELVIEW™ Fragment Assembly system (GCG, Madison, Wis.). Some sequences have been both extended and assembled to produce the consensus sequence. <span style="display: none">[0039] </span> </div>
</li> <li> <para-num num="[0040]"> </para-num> <div class="description-line" id="P-0040" num="0040">As used herein, the term “correlates with expression of a polynucleotide” indicates that the detection of the presence of nucleic acids, the same or related to a nucleic acid sequence encoding GFMO, by Northern analysis is indicative of the presence of nucleic acids encoding GFMO in a sample, and thereby correlates with expression of the transcript from the polynucleotide encoding GFMO. <span style="display: none">[0040] </span> </div>
</li> <li> <para-num num="[0041]"> </para-num> <div class="description-line" id="P-0041" num="0041">A “deletion,” as the term is used herein, refers to a change in the amino acid or nucleotide sequence that results in the absence of one or more amino acid residues or nucleotides. <span style="display: none">[0041] </span> </div>
</li> <li> <para-num num="[0042]"> </para-num> <div class="description-line" id="P-0042" num="0042">The term “derivative,” as used herein, refers to the chemical modification of a polypeptide sequence, or a polynucleotide sequence. Chemical modifications of a polynucleotide sequence can include, for example, replacement of hydrogen by an alkyl, acyl, or amino group. A derivative polynucleotide encodes a polypeptide which retains at least one biological or immunological function of the natural molecule. A derivative polypeptide is one modified by glycosylation, pegylation, or any similar process that retains at least one biological or immunological function of the polypeptide from which it was derived. <span style="display: none">[0042] </span> </div>
</li> <li> <para-num num="[0043]"> </para-num> <div class="description-line" id="P-0043" num="0043">The term “similarity,” as used herein, refers to a degree of complementarity. There may be partial similarity or complete similarity. The word “identity” may substitute for the word “similarity.” A partially complementary sequence that at least partially inhibits an identical sequence from hybridizing to a target nucleic acid is referred to as “substantially similar.” The inhibition of hybridization of the completely complementary sequence to the target sequence may be examined using a hybridization assay (Southern or Northern blot, solution hybridization, and the like) under conditions of reduced stringency. A substantially similar sequence or hybridization probe will compete for and inhibit the binding of a completely similar (identical) sequence to the target sequence under conditions of reduced stringency. This is not to say that conditions of reduced stringency are such that non-specific binding is permitted, as reduced stringency conditions require that the binding of two sequences to one another be a specific (i.e., a selective) interaction. The absence of non-specific binding may be tested by the use of a second target sequence which lacks even a partial degree of complementarity (e.g., less than about 30% similarity or identity). In the absence of non-specific binding, the substantially similar sequence or probe will not hybridize to the second non-complementary target sequence. <span style="display: none">[0043] </span> </div>
</li> <li> <para-num num="[0044]"> </para-num> <div class="description-line" id="P-0044" num="0044">The phrases “percent identity” or “% identity” refer to the percentage of sequence similarity found in a comparison of two or more amino acid or nucleic acid sequences. Percent identity can be determined electronically, e.g., by using the MegAlign™ program (DNASTAR, Inc., Madison Wis.). The MegAlign™ program can create alignments between two or more sequences according to different methods, e.g., the clustal method. (See, e.g., Higgins, D. G. and P. M. Sharp (1988) Gene 73:237-244.) The clustal algorithm groups sequences into clusters by examining the distances between all pairs. The clusters are aligned pairwise and then in groups. The percentage similarity between two amino acid sequences, e.g., sequence A and sequence B, is calculated by dividing the length of sequence A, minus the number of gap residues in sequence A, minus the number of gap residues in sequence B, into the sum of the residue matches between sequence A and sequence B, times one hundred. Gaps of low or of no similarity between the two amino acid sequences are not included in determining percentage similarity. Percent identity between nucleic acid sequences can also be counted or calculated by other methods known in the art, e.g., the Jotun Hein method. (See, e.g., Hein, J. (1990) Methods Enzymol. 183:626-645.) Identity between sequences can also be determined by other methods known in the art, e.g., by varying hybridization conditions. <span style="display: none">[0044] </span> </div>
</li> <li> <para-num num="[0045]"> </para-num> <div class="description-line" id="P-0045" num="0045">“Human artificial chromosomes” (HACs), as described herein, are linear microchromosomes which may contain DNA sequences of about 6 kb to 10 Mb in size, and which contain all of the elements required for stable mitotic chromosome segregation and maintenance. (See, e.g., Harrington, J. J. et al. (1997) Nat Genet. 15:345-355.) <span style="display: none">[0045] </span> </div>
</li> <li> <para-num num="[0046]"> </para-num> <div class="description-line" id="P-0046" num="0046">The term “humanized antibody,” as used herein, refers to antibody molecules in which the amino acid sequence in the non-antigen binding regions has been altered so that the antibody more closely resembles a human antibody, and still retains its original binding ability. <span style="display: none">[0046] </span> </div>
</li> <li> <para-num num="[0047]"> </para-num> <div class="description-line" id="P-0047" num="0047">“Hybridization,” as the term is used herein, refers to any process by which a strand of nucleic acid binds with a complementary strand through base pairing. <span style="display: none">[0047] </span> </div>
</li> <li> <para-num num="[0048]"> </para-num> <div class="description-line" id="P-0048" num="0048">As used herein, the term “hybridization complex” refers to a complex formed between two nucleic acid sequences by virtue of the formation of hydrogen bonds between complementary bases. A hybridization complex may be formed in solution (e.g., Cot or Rot analysis) or formed between one nucleic acid sequence present in solution and another nucleic acid sequence immobilized on a solid support (e.g., paper, membranes, filters, chips, pins or glass slides, or any other appropriate substrate to which cells or their nucleic acids have been fixed). <span style="display: none">[0048] </span> </div>
</li> <li> <para-num num="[0049]"> </para-num> <div class="description-line" id="P-0049" num="0049">The words “insertion” or “addition,” as used herein, refer to changes in an amino acid or nucleotide sequence resulting in the addition of one or more amino acid residues or nucleotides, respectively, to the sequence found in the naturally occurring molecule. <span style="display: none">[0049] </span> </div>
</li> <li> <para-num num="[0050]"> </para-num> <div class="description-line" id="P-0050" num="0050">“Immune response” can refer to conditions associated with inflammation, trauma, immune disorders, or infectious or genetic disease, etc. These conditions can be characterized by expression of various factors, e.g., cytokines, chemokines, and other signaling molecules, which may affect cellular and systemic defense systems. <span style="display: none">[0050] </span> </div>
</li> <li> <para-num num="[0051]"> </para-num> <div class="description-line" id="P-0051" num="0051">The term “microarray,” as used herein, refers to an arrangement of distinct polynucleotides arrayed on a substrate, e.g., paper, nylon or any other type of membrane, filter, chip, glass slide, or any other suitable solid support. <span style="display: none">[0051] </span> </div>
</li> <li> <para-num num="[0052]"> </para-num> <div class="description-line" id="P-0052" num="0052">The terms “element” or “array element” as used herein in a microarray context, refer to hybridizable polynucleotides arranged on the surface of a substrate. <span style="display: none">[0052] </span> </div>
</li> <li> <para-num num="[0053]"> </para-num> <div class="description-line" id="P-0053" num="0053">The term “modulate,” as it appears herein, refers to a change in the activity of GFMO. For example, modulation may cause an increase or a decrease in protein activity, binding characteristics, or any other biological, functional, or immunological properties of GFMO. <span style="display: none">[0053] </span> </div>
</li> <li> <para-num num="[0054]"> </para-num> <div class="description-line" id="P-0054" num="0054">The phrases “nucleic acid” or “nucleic acid sequence,” as used herein, refer to a nucleotide, oligonucleotide, polynucleotide, or any fragment thereof. These phrases also refer to DNA or RNA of genomic or synthetic origin which may be single-stranded or double-stranded and may represent the sense or the antisense strand, to peptide nucleic acid (PNA), or to any DNA-like or RNA-like material. In this context, “fragments” refers to those nucleic acid sequences which, when translated, would produce polypeptides retaining some functional characteristic, e.g., antigenicity, or structural domain characteristic, e.g., ATP-binding site, of the full-length polypeptide. <span style="display: none">[0054] </span> </div>
</li> <li> <para-num num="[0055]"> </para-num> <div class="description-line" id="P-0055" num="0055">The terms “operably associated” or “operably linked,” as used herein, refer to functionally related nucleic acid sequences. A promoter is operably associated or operably linked with a coding sequence if the promoter controls the translation of the encoded polypeptide. While operably associated or operably linked nucleic acid sequences can be contiguous and in the same reading frame, certain genetic elements, e.g., repressor genes, are not contiguously linked to the sequence encoding the polypeptide but still bind to operator sequences that control expression of the polypeptide. <span style="display: none">[0055] </span> </div>
</li> <li> <para-num num="[0056]"> </para-num> <div class="description-line" id="P-0056" num="0056">The term “oligonucleotide,” as used herein, refers to a nucleic acid sequence of at least about 6 nucleotides to 60 nucleotides, preferably about 15 to 30 nucleotides, and most preferably about 20 to 25 nucleotides, which can be used in PCR amplification or in a hybridization assay or microarray. As used herein, the term “oligonucleotide” is substantially equivalent to the terms “amplimer,” “primer,” “oligomer,” and “probe,” as these terms are commonly defined in the art. <span style="display: none">[0056] </span> </div>
</li> <li> <para-num num="[0057]"> </para-num> <div class="description-line" id="P-0057" num="0057">“Peptide nucleic acid” (PNA), as used herein, refers to an antisense molecule or anti-gene agent which comprises an oligonucleotide of at least about 5 nucleotides in length linked to a peptide backbone of amino acid residues ending in lysine. The terminal lysine confers solubility to the composition. PNAs preferentially bind complementary single stranded DNA or RNA and stop transcript elongation, and may be pegylated to extend their lifespan in the cell. (See, e.g., Nielsen, P. E. et al. (1993) Anticancer Drug Des. 8:53-63.) <span style="display: none">[0057] </span> </div>
</li> <li> <para-num num="[0058]"> </para-num> <div class="description-line" id="P-0058" num="0058">The term “sample,” as used herein, is used in its broadest sense. A biological sample suspected of containing nucleic acids encoding GFMO, or fragments thereof, or GFMO itself, may comprise a bodily fluid; an extract from a cell, chromosome, organelle, or membrane isolated from a cell; a cell; genomic DNA, RNA, or cDNA, in solution or bound to a solid support; a tissue; a tissue print; etc. <span style="display: none">[0058] </span> </div>
</li> <li> <para-num num="[0059]"> </para-num> <div class="description-line" id="P-0059" num="0059">As used herein, the terms “specific binding” or “specifically binding” refer to that interaction between a protein or peptide and an agonist, an antibody, or an antagonist. The interaction is dependent upon the presence of a particular structure of the protein, e.g., the antigenic determinant or epitope, recognized by the binding molecule. For example, if an antibody is specific for epitope “A,” the presence of a polypeptide containing the epitope A, or the presence of free unlabeled A, in a reaction containing free labeled A and the antibody will reduce the amount of labeled A that binds to the antibody. <span style="display: none">[0059] </span> </div>
</li> <li> <para-num num="[0060]"> </para-num> <div class="description-line" id="P-0060" num="0060">As used herein, the term “stringent conditions” refers to conditions which permit hybridization between polynucleotides and the claimed polynucleotides. Stringent conditions can be defined by salt concentration, the concentration of organic solvent, e.g., formamide, temperature, and other conditions well known in the art. In particular, stringency can be increased by reducing the concentration of salt, increasing the concentration of formamide, or raising the hybridization temperature. <span style="display: none">[0060] </span> </div>
</li> <li> <para-num num="[0061]"> </para-num> <div class="description-line" id="P-0061" num="0061">The term “substantially purified,” as used herein, refers to nucleic acid or amino acid sequences that are removed from their natural environment and are isolated or separated, and are at least about 60% free, preferably about 75% free, and most preferably about 90% free from other components with which they are naturally associated. <span style="display: none">[0061] </span> </div>
</li> <li> <para-num num="[0062]"> </para-num> <div class="description-line" id="P-0062" num="0062">A “substitution,” as used herein, refers to the replacement of one or more amino acids or nucleotides by different amino acids or nucleotides, respectively. <span style="display: none">[0062] </span> </div>
</li> <li> <para-num num="[0063]"> </para-num> <div class="description-line" id="P-0063" num="0063">“Transformation,” as defined herein, describes a process by which exogenous DNA enters and changes a recipient cell. Transformation may occur under natural or artificial conditions according to various methods well known in the art, and may rely on any known method for the insertion of foreign nucleic acid sequences into a prokaryotic or eukaryotic host cell. The method for transformation is selected based on the type of host cell being transformed and may include, but is not limited to, viral infection, electroporation, heat shock, lipofection, and particle bombardment. The term “transformed” cells includes stably transformed cells in which the inserted DNA is capable of replication either as an autonomously replicating plasmid or as part of the host chromosome, as well as transiently transformed cells which express the inserted DNA or RNA for limited periods of time. <span style="display: none">[0063] </span> </div>
</li> <li> <para-num num="[0064]"> </para-num> <div class="description-line" id="P-0064" num="0064">A “variant” of GFMO polypeptides, as used herein, refers to an amino acid sequence that is altered by one or more amino acid residues. The variant may have “conservative” changes, wherein a substituted amino acid has similar structural or chemical properties (e.g., replacement of leucine with isoleucine). More rarely, a variant may have “nonconservative” changes (e.g., replacement of glycine with tryptophan). Analogous minor variations may also include amino acid deletions or insertions, or both. Guidance in determining which amino acid residues may be substituted, inserted, or deleted without abolishing biological or immunological activity may be found using computer programs well known in the art, for example, LASERGENE™ software. <span style="display: none">[0064] </span> </div>
</li> <li> <para-num num="[0065]"> </para-num> <div class="description-line" id="P-0065" num="0065">The term “variant,” when used in the context of a polynucleotide sequence, may encompass a polynucleotide sequence related to GFMO. This definition may also include, for example, “allelic” (as defined above), “splice,” “species,” or “polymorphic” variants. A splice variant may have significant identity to a reference molecule, but will generally have a greater or lesser number of polynucleotides due to alternate splicing of exons during mRNA processing. The corresponding polypeptide may possess additional functional domains or an absence of domains. Species variants are polynucleotide sequences that vary from one species to another. The resulting polypeptides generally will have significant amino acid identity relative to each other. A polymorphic variant is a variation in the polynucleotide sequence of a particular gene between individuals of a given species. Polymorphic variants also may encompass “single nucleotide polymorphisms” (SNPs) in which the polynucleotide sequence varies by one base. The presence of SNPs may be indicative of, for example, a certain population, a disease state, or a propensity for a disease state. <span style="display: none">[0065] </span> </div>
</li> <li> <para-num num="[0066]"> </para-num> <div class="description-line" id="P-0066" num="0066">The Invention <span style="display: none">[0066] </span> </div>
</li> <li> <para-num num="[0067]"> </para-num> <div class="description-line" id="P-0067" num="0067">The invention is based on the discovery of new human growth factor modulators (GFMO), the polynucleotides encoding GFMO, and the use of these compositions for the diagnosis, treatment, or prevention of cancer and fibrotic disorders. <span style="display: none">[0067] </span> </div>
</li> <li> <para-num num="[0068]"> </para-num> <div class="description-line" id="P-0068" num="0068">Nucleic acids encoding the GFMO-1 of the present invention were first identified in Incyte Clone 2509339 from the sigmoid mesentery tumor tissue cDNA library (CONUTUT) using a computer search, e.g., BLAST, for amino acid sequence alignments. A consensus sequence, SEQ ID NO: 3, was derived from the following overlapping and/or extended nucleic acid sequences (SEQ ID NOs: 5-8): Incyte Clones 2509339H1 and 2509339F6 (CONUTUT01), and shotgun sequences SBCA01417F1 and SBCA02999F1. <span style="display: none">[0068] </span> </div>
</li> <li> <para-num num="[0069]"> </para-num> <div class="description-line" id="P-0069" num="0069">In one embodiment, the invention encompasses a polypeptide comprising the amino acid sequence of SEQ ID NO: 1, as shown in FIGS. 1A, 1B, <span style="display: none">[0069] </span> <b>1</b>C, and <b>1</b>D. GFMO-1 is 354 amino acids in length and has two potential N-glycosylation sites at residues N178 and N308; five potential protein kinase C phosphorylation sites at residues S191, S228, T259, S290, and S324; and an insulin-like growth factor binding protein signature from residue G71 through C86. BLOCKS identifies insulin-like growth factor binding motifs from residues V65 through Y99, and F282 through S310; von Willebrand factor type C motifs from residues C122 through C152, W216 through R230, and S287 through G297; and C-terminal cysteine knot motifs from residues C72 through L98, and C285 through E322. SPScan identifies a potential signal peptide from residue M1 through F17. SigPept identifies a potential signal peptide from residue M1 through G23. PFAM identifies significant sequence identity with cysteine knot proteins and IGF-binding proteins. As shown in FIGS. 3A and 3B, GFMO-1 has chemical and structural similarity with mouse Elm1 (GI 2911144; SEQ ID NO: 13). In particular, GFMO-1 and mouse Elm<b>1</b> share 40% identity, have similar molecular mass (39.3 kDa and 40.7 kDa, respectively), and share CCN protein family IGF-binding, VWFC, and C-terminal cysteine knot motifs. A region of unique sequence in GFMO-1 from about amino acid 184 to about amino acid 190 is encoded by a fragment of SEQ ID NO: 3 from about nucleotide 585 to about nucleotide 605. Northern analysis shows the expression of this sequence in various libraries, at least 75% of which are cancerous and at least 25% of which involve immune response. Of particular note is the expression of GFMO-1 in nervous, endothelial, and connective tissues. </div>
</li> <li> <para-num num="[0070]"> </para-num> <div class="description-line" id="P-0070" num="0070">Nucleic acids encoding the GFMO-2 of the present invention were first identified in Incyte Clone 2840746 from the dorsal root ganglion cDNA library (DRGLNOT01) using a computer search, e.g., BLAST, for amino acid sequence alignments. A consensus sequence, SEQ ID NO: 4, was derived from the following overlapping and/or extended nucleic acid sequences (SEQ ID NOs: 9-12): Incyte Clones 2840746H1 (DRGLNOT01), 861509R6 (BRAITUT03), 2843688T6 (DRGLNOT01), and 866176R1 (BRAITUT03). <span style="display: none">[0070] </span> </div>
</li> <li> <para-num num="[0071]"> </para-num> <div class="description-line" id="P-0071" num="0071">In one embodiment, the invention encompasses a polypeptide comprising the amino acid sequence of SEQ ID NO: 2, as shown in FIGS. 2A, 2B, and <span style="display: none">[0071] </span> <b>2</b>C. GFMO-2 is 223 amino acids in length and has two potential casein kinase II phosphorylation sites at residues T29 and S168; six potential protein kinase C phosphorylation sites at residues T44, S132, S149, T155, T178, and T182; and a potential tyrosine kinase phosphorylation site at residue Y70. SPScan and SigPept identify a potential signal peptide from residue M1 through A21. As shown in FIGS. 4A and 4B, GFMO-2 has chemical and structural similarity with mouse FGF-binding protein (GI 1469936; SEQ ID NO: 14). In particular, GFMO-2 and mouse FGF-binding protein share 16% identity and have similar isoelectric points (8.87 and 9.13, respectively). GFMO-2 and mouse FGF-binding protein also have 8 conserved cysteine residues at C43, C63, C72, C81, C106, C117, C206, and C214, suggesting potential intramolecular disulfide bridging sites. Northern analysis shows the expression of this sequence in various libraries, at least 38% of which are cancerous and at least 22% of which involve immune response. Of particular note is the expression of GFMO-2 in nervous and connective tissues. </div>
</li> <li> <para-num num="[0072]"> </para-num> <div class="description-line" id="P-0072" num="0072">The invention also encompasses GFMO variants. A preferred GFMO variant is one which has at least about 80%, more preferably at least about 90%, and most preferably at least about 95% amino acid sequence identity to the GFMO amino acid sequence, and which contains at least one functional or structural characteristic of GFMO. <span style="display: none">[0072] </span> </div>
</li> <li> <para-num num="[0073]"> </para-num> <div class="description-line" id="P-0073" num="0073">The invention also encompasses polynucleotides which encode GFMO. In a particular embodiment, the invention encompasses a polynucleotide sequence comprising the sequence of SEQ ID NO: 3, as shown in FIGS. 1A, 1B, <span style="display: none">[0073] </span> <b>1</b>C, and <b>1</b>D, which encodes a GFMO-1. In a further embodiment, the invention encompasses the polynucleotide sequence comprising the sequence of SEQ ID NO: 4, as shown in FIGS. 2A, 2B, and <b>2</b>C, which encodes a GFMO-2. </div>
</li> <li> <para-num num="[0074]"> </para-num> <div class="description-line" id="P-0074" num="0074">The invention also encompasses a variant of a polynucleotide sequence encoding GFMO. In particular, such a variant polynucleotide sequence will have at least about 70%, more preferably at least about 85%, and most preferably at least about 95% polynucleotide sequence identity to the polynucleotide sequence encoding GFMO. A particular aspect of the invention encompasses a variant of SEQ ID NO: 3 which has at least about 70%, more preferably at least about 85%, and most preferably at least about 95% polynucleotide sequence identity to SEQ ID NO: 3. The invention further encompasses a polynucleotide variant of SEQ ID NO: 4 having at least about 70%, more preferably at least about 85%, and most preferably at least about 95% polynucleotide sequence identity to SEQ ID NO: 4. Any one of the polynucleotide variants described above can encode an amino acid sequence which contains at least one functional or structural characteristic of GFMO. <span style="display: none">[0074] </span> </div>
</li> <li> <para-num num="[0075]"> </para-num> <div class="description-line" id="P-0075" num="0075">It will be appreciated by those skilled in the art that as a result of the degeneracy of the genetic code, a multitude of polynucleotide sequences encoding GFMO, some bearing minimal similarity to the polynucleotide sequences of any known and naturally occurring gene, may be produced. Thus, the invention contemplates each and every possible variation of polynucleotide sequence that could be made by selecting combinations based on possible codon choices. These combinations are made in accordance with the standard triplet genetic code as applied to the polynucleotide sequence of naturally occurring GFMO, and all such variations are to be considered as being specifically disclosed. <span style="display: none">[0075] </span> </div>
</li> <li> <para-num num="[0076]"> </para-num> <div class="description-line" id="P-0076" num="0076">Although nucleotide sequences which encode GFMO and its variants are preferably capable of hybridizing to the nucleotide sequence of the naturally occurring GFMO under appropriately selected conditions of stringency, it may be advantageous to produce nucleotide sequences encoding GFMO possessing a substantially different codon usage, e.g., inclusion of non-naturally occurring codons. Codons may be selected to increase the rate at which expression of the peptide occurs in a particular prokaryotic or eukaryotic host in accordance with the frequency with which particular codons are utilized by the host. Other reasons for substantially altering the nucleotide sequence encoding GFMO and its derivatives without altering the encoded amino acid sequences include the production of RNA transcripts having more desirable properties, such as a greater half-life, than transcripts produced from the naturally occurring sequence. <span style="display: none">[0076] </span> </div>
</li> <li> <para-num num="[0077]"> </para-num> <div class="description-line" id="P-0077" num="0077">The invention also encompasses production of DNA sequences which encode GFMO and GFMO derivatives, or fragments thereof, entirely by synthetic chemistry. After production, the synthetic sequence may be inserted into any of the many available expression vectors and cell systems using reagents well known in the art. Moreover, synthetic chemistry may be used to introduce mutations into a sequence encoding GFMO or any fragment thereof. <span style="display: none">[0077] </span> </div>
</li> <li> <para-num num="[0078]"> </para-num> <div class="description-line" id="P-0078" num="0078">Also encompassed by the invention are polynucleotide sequences that are capable of hybridizing to the claimed polynucleotide sequences, and, in particular, to those shown in SEQ ID NO: 3, SEQ ID NO: 4, a fragment of SEQ ID NO: 3, or a fragment of SEQ ID NO: 4, under various conditions of stringency. (See, e.g., Wahl, G. M. and S. L. Berger (1987) Methods Enzymol. 152:399-407; Kimmel, A. R. (1987) Methods Enzymol. 152:507-511.) For example, stringent salt concentration will ordinarily be less than about 750 mM NaCl and 75 mM trisodium citrate, preferably less than about 500 mM NaCl and 50 mM trisodium citrate, and most preferably less than about 250 mM NaCl and 25 mM trisodium citrate. Low stringency hybridization can be obtained in the absence of organic solvent, e.g., formamide, while high stringency hybridization can be obtained in the presence of at least about 35% formamide, and most preferably at least about 50% formamide. Stringent temperature conditions will ordinarily include temperatures of at least about 30° C., more preferably of at least about 37° C., and most preferably of at least about 42° C. Varying additional parameters, such as hybridization time, the concentration of detergent, e.g., sodium dodecyl sulfate (SDS), and the inclusion or exclusion of carrier DNA, are well known to those skilled in the art. Various levels of stringency are accomplished by combining these various conditions as needed. In a preferred embodiment, hybridization will occur at 30° C. in 750 mM NaCl, 75 mM trisodium citrate, and 1% SDS. In a more preferred embodiment, hybridization will occur at 37° C. in 500 mM NaCl, 50 mM trisodium citrate, 1% SDS, 35% formamide, and 100 μg/ml denatured salmon sperm DNA (ssDNA). In a most preferred embodiment, hybridization will occur at 42° C. in 250 mM NaCl, 25 mM trisodium citrate, 1% SDS, 50% formamide, and 200 μg/ml ssDNA. Useful variations on these conditions will be readily apparent to those skilled in the art. <span style="display: none">[0078] </span> </div>
</li> <li> <para-num num="[0079]"> </para-num> <div class="description-line" id="P-0079" num="0079">The washing steps which follow hybridization can also vary in stringency. Wash stringency conditions can be defined by salt concentration and by temperature. As above, wash stringency can be increased by decreasing salt concentration or by increasing temperature. For example, stringent salt concentration for the wash steps will preferably be less than about 30 mM NaCl and 3 mM trisodium citrate, and most preferably less than about 15 mM NaCl and 1.5 mM trisodium citrate. Stringent temperature conditions for the wash steps will ordinarily include temperature of at least about 25° C., more preferably of at least about 42° C., and most preferably of at least about 68° C. In a preferred embodiment, wash steps will occur at 25° C. in 30 mM NaCl, 3 mM trisodium citrate, and 0.1% SDS. In a more preferred embodiment, wash steps will occur at 42° C. in 15 mM NaCl, 1.5 mM trisodium citrate, and 0.1% SDS. In a most preferred embodiment, wash steps will occur at 68° C. in 15 mM NaCl, 1.5 mM trisodium citrate, and 0.1% SDS. Additional variations on these conditions will be readily apparent to those skilled in the art. <span style="display: none">[0079] </span> </div>
</li> <li> <para-num num="[0080]"> </para-num> <div class="description-line" id="P-0080" num="0080">Methods for DNA sequencing and analysis are well known in the art. The methods may employ such enzymes as the Klenow fragment of DNA polymerase I, SEQUENASE® (Amersham Pharmacia Biotech Ltd., Uppsala, Sweden), Taq polymerase (The Perkin-Elmer Corp., Norwalk, Conn.), thermostable T7 polymerase (Amersham Pharmacia Biotech Ltd., Uppsala, Sweden), or combinations of polymerases and proofreading exonucleases, such as those found in the ELONGASE™ amplification system (Life Technologies, Inc., Rockville, Md.). Preferably, sequence preparation is automated with machines, e.g., the ABI CATALYST™ 800 (The Perkin-Elmer Corp., Norwalk, Conn.) or MICROLAB® 2200 (Hamilton Co., Reno, Nev.) systems, in combination with thermal cyclers. Sequencing can also be automated, such as by ABI PRISM™ 373 or 377 systems (The Perkin-Elmer Corp., Norwalk, Conn.) or the MEGABACE™ 1000 capillary electrophoresis system (Molecular Dynamics, Inc., Sunnyvale, Calif.). Sequences can be analyzed using computer programs and algorithms well known in the art. (See, e.g., Ausubel, supra, unit 7.7; and Meyers, R. A. (1995) <span style="display: none">[0080] </span> <i>Molecular Biology and Biotechnology</i>, Wiley V C H, Inc, New York, N.Y.) The nucleic acid sequences encoding GFMO may be extended utilizing a partial nucleotide sequence and employing various PCR-based methods known in the art to detect upstream sequences, such as promoters and regulatory elements. For example, one method which may be employed, restriction-site PCR, uses universal and nested primers to amplify unknown sequence from genomic DNA within a cloning vector. (See, e.g., Sarkar, G. (1993) PCR Methods Applic. 2:318-322.) Another method, inverse PCR, uses primers that extend in divergent directions to amplify unknown sequence from a circularized template. The template is derived from restriction fragments comprising a known genomic locus and surrounding sequences. (See, e.g., Triglia, T. et al. (1988) Nucleic Acids Res. 16:8186.) A third method, capture PCR, involves PCR amplification of DNA fragments adjacent to known sequences in human and yeast artificial chromosome DNA. (See, e.g., Lagerstrom, M. et al. (1991) PCR Methods Applic. 1:111-119.) In this method, multiple restriction enzyme digestions and ligations may be used to insert an engineered double-stranded sequence into a region of unknown sequence before performing PCR. Other methods which may be used to retrieve unknown sequences are known in the art. (See, e.g., Parker, J. D. et al. (1991) Nucleic Acids Res. 19:3055-306). Additionally, one may use PCR, nested primers, and PromoterFinder™ libraries to walk genomic DNA (Clontech, Palo Alto, Calif.). This procedure avoids the need to screen libraries and is useful in finding intron/exon junctions. For all PCR-based methods, primers may be designed using commercially available software, such as OLIGO™ 4.06 Primer Analysis software (National Biosciences Inc., Plymouth, Minn.) or another appropriate program, to be about 22 to 30 nucleotides in length, to have a GC content of about 50% or more, and to anneal to the template at temperatures of about 68° C. to 72° C. </div>
</li> <li> <para-num num="[0081]"> </para-num> <div class="description-line" id="P-0081" num="0081">When screening for full-length cDNAs, it is preferable to use libraries that have been size-selected to include larger cDNAs. In addition, random-primed libraries, which often include sequences containing the 5′ regions of genes, are preferable for situations in which an oligo d(T) library does not yield a full-length cDNA. Genomic libraries may be useful for extension of sequence into 5′ non-transcribed regulatory regions. <span style="display: none">[0081] </span> </div>
</li> <li> <para-num num="[0082]"> </para-num> <div class="description-line" id="P-0082" num="0082">Capillary electrophoresis systems which are commercially available may be used to analyze the size or confirm the nucleotide sequence of sequencing or PCR products. In particular, capillary sequencing may employ flowable polymers for electrophoretic separation, four different nucleotide-specific, laser-stimulated fluorescent dyes, and a charge coupled device camera for detection of the emitted wavelengths. Output/light intensity may be converted to electrical signal using appropriate software (e.g., Genotyper™ and Sequence Navigator™, (The Perkin-Elmer Corp., Norwalk, Conn.)), and the entire process from loading of samples to computer analysis and electronic data display may be computer controlled. Capillary electrophoresis is especially preferable for sequencing small DNA fragments which may be present in limited amounts in a particular sample. <span style="display: none">[0082] </span> </div>
</li> <li> <para-num num="[0083]"> </para-num> <div class="description-line" id="P-0083" num="0083">In another embodiment of the invention, polynucleotide sequences or fragments thereof which encode GFMO may be cloned in recombinant DNA molecules that direct expression of GFMO, or fragments or functional equivalents thereof, in appropriate host cells. Due to the inherent degeneracy of the genetic code, other DNA sequences which encode substantially the same or a functionally equivalent amino acid sequence may be produced and used to express GFMO. <span style="display: none">[0083] </span> </div>
</li> <li> <para-num num="[0084]"> </para-num> <div class="description-line" id="P-0084" num="0084">The nucleotide sequences of the present invention can be engineered using methods generally known in the art in order to alter GFMO-encoding sequences for a variety of purposes including, but not limited to, modification of the cloning, processing, and/or expression of the gene product. DNA shuffling by random fragmentation and PCR reassembly of gene fragments and synthetic oligonucleotides may be used to engineer the nucleotide sequences. For example, oligonucleotide-mediated site-directed mutagenesis may be used to introduce mutations that create new restriction sites, alter glycosylation patterns, change codon preference, produce splice variants, and so forth. <span style="display: none">[0084] </span> </div>
</li> <li> <para-num num="[0085]"> </para-num> <div class="description-line" id="P-0085" num="0085">In another embodiment, sequences encoding GFMO may be synthesized, in whole or in part, using chemical methods well known in the art. (See, e.g., Caruthers, M. H. et al. (1980) Nucl. Acids Res. Symp. Ser. 215-223, and Horn, T. et al. (1980) Nucl. Acids Res. Symp. Ser. 225-232.) Alternatively, GFMO itself or a fragment thereof may be synthesized using chemical methods. For example, peptide synthesis can be performed using various solid-phase techniques. (See, e.g., Roberge, J. Y. et al. (1995) Science 269:202-204.) Automated synthesis may be achieved using the ABI 431A Peptide Synthesizer (The Perkin-Elmer Corp., Norwalk, Conn.). Additionally, the amino acid sequence of GFMO, or any part thereof, may be altered during direct synthesis and/or combined with sequences from other proteins, or any part thereof, to produce a variant polypeptide. <span style="display: none">[0085] </span> </div>
</li> <li> <para-num num="[0086]"> </para-num> <div class="description-line" id="P-0086" num="0086">The peptide may be substantially purified by preparative high performance liquid chromatography. (See, e.g, Chiez, R. M. and F. Z. Regnier (1990) Methods Enzymol. 182:392-421.) The composition of the synthetic peptides may be confirmed by amino acid analysis or by sequencing. (See, e.g., Creighton, T. (1984) <span style="display: none">[0086] </span> <i>Proteins, Structures and Molecular Properties</i>, W H Freeman and Co., New York, N.Y.) </div>
</li> <li> <para-num num="[0087]"> </para-num> <div class="description-line" id="P-0087" num="0087">In order to express a biologically active GFMO, the nucleotide sequences encoding GFMO or derivatives thereof may be inserted into an appropriate expression vector, i.e., a vector which contains the necessary elements for transcriptional and translational control of the inserted coding sequence in a suitable host. These elements include regulatory sequences, such as enhancers, constitutive and inducible promoters, and 5′ and 3′ untranslated regions in the vector and in polynucleotide sequences encoding GFMO. Such elements may vary in their strength and specificity. Specific initiation signals may also be used to achieve more efficient translation of sequences encoding GFMO. Such signals include the ATG initiation codon and adjacent sequences, e.g. the Kozak sequence. In cases where sequences encoding GFMO and its initiation codon and upstream regulatory sequences are inserted into the appropriate expression vector, no additional transcriptional or translational control signals may be needed. However, in cases where only coding sequence, or a fragment thereof, is inserted, exogenous translational control signals including an in-frame ATG initiation codon should be provided by the vector. Exogenous translational elements and initiation codons may be of various origins, both natural and synthetic. The efficiency of expression may be enhanced by the inclusion of enhancers appropriate for the particular host cell system used. (See, e.g., Scharf, D. et al. (1994) Results Probl. Cell Differ. 20:125-162.) <span style="display: none">[0087] </span> </div>
</li> <li> <para-num num="[0088]"> </para-num> <div class="description-line" id="P-0088" num="0088">Methods which are well known to those skilled in the art may be used to construct expression vectors containing sequences encoding GFMO and appropriate transcriptional and translational control elements. These methods include in vitro recombinant DNA techniques, synthetic techniques, and in vivo genetic recombination. (See, e.g., Sambrook, J. et al. (1989) <span style="display: none">[0088] </span> <i>Molecular Cloning. A Laboratory Manual</i>, Cold Spring Harbor Press, Plainview, N.Y., ch. 4, 8, and 16-17; and Ausubel, F. M. et al. (1995, and periodic supplements) <i>Current Protocols in Molecular Biology</i>, John Wiley &amp; Sons, New York, N.Y., ch. 9, 13, and 16.) </div>
</li> <li> <para-num num="[0089]"> </para-num> <div class="description-line" id="P-0089" num="0089">A variety of expression vector/host systems may be utilized to contain and express sequences encoding GFMO. These include, but are not limited to, microorganisms such as bacteria transformed with recombinant bacteriophage, plasmid, or cosmid DNA expression vectors; yeast transformed with yeast expression vectors; insect cell systems infected with viral expression vectors (e.g., baculovirus); plant cell systems transformed with viral expression vectors (e.g., cauliflower mosaic virus (CaMV) or tobacco mosaic virus (TMV)) or with bacterial expression vectors (e.g., Ti or pBR322 plasmids); or animal cell systems. The invention is not limited by the host cell employed. <span style="display: none">[0089] </span> </div>
</li> <li> <para-num num="[0090]"> </para-num> <div class="description-line" id="P-0090" num="0090">In bacterial systems, a number of cloning and expression vectors may be selected depending upon the use intended for polynucleotide sequences encoding GFMO. For example, routine cloning, subcloning, and propagation of polynucleotide sequences encoding GFMO can be achieved using a multifunctional <span style="display: none">[0090] </span> <i>E. coli </i>vector such as Bluescript® (Stratagene) or pSport1™ plasmid (GIBCO BRL). Ligation of sequences encoding GFMO into the vector's multiple cloning site disrupts the lacZ gene, allowing a colorimetric screening procedure for identification of transformed bacteria containing recombinant molecules. In addition, these vectors may be useful for in vitro transcription, dideoxy sequencing, single strand rescue with helper phage, and creation of nested deletions in the cloned sequence. (See, e.g., Van Heeke, G. and S. M. Schuster (1989) J. Biol. Chem. 264:5503-5509.) When large quantities of GFMO are needed, e.g. for the production of antibodies, vectors which direct high level expression of GFMO may be used. For example, vectors containing the strong, inducible T5 or T7 bacteriophage promoter may be used. </div>
</li> <li> <para-num num="[0091]"> </para-num> <div class="description-line" id="P-0091" num="0091">Yeast expression systems may be used for production of GFMO. A number of vectors containing constitutive or inducible promoters, such as alpha factor, alcohol oxidase, and PGH, may be used in the yeast <span style="display: none">[0091] </span> <i>Saccharomyces cerevisiae </i>or <i>Pichia pastoris</i>. In addition, such vectors direct either the secretion or intracellular retention of expressed proteins and enable integration of foreign sequences into the host genome for stable propagation. (See, e.g., Ausubel, supra; and Grant et al. (1987) Methods Enzymol. 153:516-54; Scorer, C. A. et al. (1994) Bio/Technology 12:181-184.) </div>
</li> <li> <para-num num="[0092]"> </para-num> <div class="description-line" id="P-0092" num="0092">Plant systems may also be used for expression of GFMO. Transcription of sequences encoding GFMO may be driven viral promoters, e.g., the 35S and 19S promoters of CaMV used alone or in combination with the omega leader sequence from TMV. (Takamatsu, N. (1987) EMBO J. 6:307-311.) Alternatively, plant promoters such as the small subunit of RUBISCO or heat shock promoters may be used. (See, e.g., Coruzzi, G. et al. (1984) EMBO J. 3:1671-1680; Broglie, R. et al. (1984) Science 224:838-843; and Winter, J. et al. (1991) Results Probl. Cell Differ. 17:85-105.) These constructs can be introduced into plant cells by direct DNA transformation or pathogen-mediated transfection. (See, e.g., Hobbs, S. or Murry, L. E. in <span style="display: none">[0092] </span> <i>McGraw Hill Yearbook of Science and Technology </i>(1992) McGraw Hill, New York, N.Y.; pp. 191-196.) </div>
</li> <li> <para-num num="[0093]"> </para-num> <div class="description-line" id="P-0093" num="0093">In mammalian cells, a number of viral-based expression systems may be utilized. In cases where an adenovirus is used as an expression vector, sequences encoding GFMO may be ligated into an adenovirus transcription/translation complex consisting of the late promoter and tripartite leader sequence. Insertion in a non-essential E1 or E3 region of the viral genome may be used to obtain infective virus which expresses GFMO in host cells. (See, e.g., Logan, J. and T. Shenk (1984) Proc. Natl. Acad. Sci. 81:3655-3659.) In addition, transcription enhancers, such as the Rous sarcoma virus (RSV) enhancer, may be used to increase expression in mammalian host cells. SV40 or EBV-based vectors may also be used for high-level protein expression. <span style="display: none">[0093] </span> </div>
</li> <li> <para-num num="[0094]"> </para-num> <div class="description-line" id="P-0094" num="0094">Human artificial chromosomes (HACs) may also be employed to deliver larger fragments of DNA than can be contained in and expressed from a plasmid. HACs of about 6 kb to 10 Mb are constructed and delivered via conventional delivery methods (liposomes, polycationic amino polymers, or vesicles) for therapeutic purposes. <span style="display: none">[0094] </span> </div>
</li> <li> <para-num num="[0095]"> </para-num> <div class="description-line" id="P-0095" num="0095">For long term production of recombinant proteins in mammalian systems, stable expression of GFMO in cell lines is preferred. For example, sequences encoding GFMO can be transformed into cell lines using expression vectors which may contain viral origins of replication and/or endogenous expression elements and a selectable marker gene on the same or on a separate vector. Following the introduction of the vector, cells may be allowed to grow for about 1 to 2 days in enriched media before being switched to selective media. The purpose of the selectable marker is to confer resistance to a selective agent, and its presence allows growth and recovery of cells which successfully express the introduced sequences. Resistant clones of stably transformed cells may be propagated using tissue culture techniques appropriate to the cell type. <span style="display: none">[0095] </span> </div>
</li> <li> <para-num num="[0096]"> </para-num> <div class="description-line" id="P-0096" num="0096">Any number of selection systems may be used to recover transformed cell lines. These include, but are not limited to, the herpes simplex virus thymidine kinase and adenine phosphoribosyltransferase genes, for use in tk<span style="display: none">[0096] </span> <sup>−</sup> or apr<sup>−</sup> cells, respectively. (See, e.g., Wigler, M. et al. (1977) Cell 11:223-232; and Lowy, I. et al. (1980) Cell 22:817-823.) Also, antimetabolite, antibiotic, or herbicide resistance can be used as the basis for selection. For example, dhfr confers resistance to methotrexate; neo confers resistance to the aminoglycosides neomycin and G-418; and als or pat confer resistance to chlorsulfuron and phosphinotricin acetyltransferase, respectively. (See, e.g., Wigler, M. et al. (1980) Proc. Natl. Acad. Sci. 77:3567-3570; Colbere-Garapin, F. et al (1981) J. Mol. Biol. 150:1-14; and Murry, supra.) Additional selectable genes have been described, e.g., trpB and hisD, which alter cellular requirements for metabolites. (See, e.g., Hartman, S. C. and R. C. Mulligan (1988) Proc. Natl. Acad. Sci. 85:8047-8051.) Visible markers, e.g., anthocyanins, green fluorescent proteins (GFP) (Clontech, Palo Alto, Calif.), β glucuronidase and its substrate β-D-glucuronoside, or luciferase and its substrate luciferin may be used. These markers can be used not only to identify transformants, but also to quantify the amount of transient or stable protein expression attributable to a specific vector system. (See, e.g., Rhodes, C. A. et al. (1995) Methods Mol. Biol. 55:121-131.) </div>
</li> <li> <para-num num="[0097]"> </para-num> <div class="description-line" id="P-0097" num="0097">Although the presence/absence of marker gene expression suggests that the gene of interest is also present, the presence and expression of the gene may need to be confirmed. For example, if the sequence encoding GFMO is inserted within a marker gene sequence, transformed cells containing sequences encoding GFMO can be identified by the absence of marker gene function. Alternatively, a marker gene can be placed in tandem with a sequence encoding GFMO under the control of a single promoter. Expression of the marker gene in response to induction or selection usually indicates expression of the tandem gene as well. <span style="display: none">[0097] </span> </div>
</li> <li> <para-num num="[0098]"> </para-num> <div class="description-line" id="P-0098" num="0098">In general, host cells that contain the nucleic acid sequence encoding GFMO and that express GFMO may be identified by a variety of procedures known to those of skill in the art. These procedures include, but are not limited to, DNA-DNA or DNA-RNA hybridizations, PCR amplification, and protein bioassay or immunoassay techniques which include membrane, solution, or chip based technologies for the detection and/or quantification of nucleic acid or protein sequences. <span style="display: none">[0098] </span> </div>
</li> <li> <para-num num="[0099]"> </para-num> <div class="description-line" id="P-0099" num="0099">Immunological methods for detecting and measuring the expression of GFMO using either specific polyclonal or monoclonal antibodies are known in the art. Examples of such techniques include enzyme-linked immunosorbent assays (ELISAs), radioimmunoassays (RIAs), and fluorescence activated cell sorting (FACS). A two-site, monoclonal-based immunoassay utilizing monoclonal antibodies reactive to two non-interfering epitopes on GFMO is preferred, but a competitive binding assay may be employed. These and other assays are well known in the art. (See, e.g., Hampton, R. et al. (1990) <span style="display: none">[0099] </span> <i>Serological Methods, a Laboratory Manual</i>, APS Press, St Paul, Minn., Section IV; Coligan, J. E. et al. (1997 and periodic supplements) <i>Current Protocols in Immunology</i>, Greene Pub. Associates and Wiley-Interscience, New York, N.Y.; and Maddox, D. E. et al. (1983) J. Exp. Med. 158:1211-1216). </div>
</li> <li> <para-num num="[0100]"> </para-num> <div class="description-line" id="P-0100" num="0100">A wide variety of labels and conjugation techniques are known by those skilled in the art and may be used in various nucleic acid and amino acid assays. Means for producing labeled hybridization or PCR probes for detecting sequences related to polynucleotides encoding GFMO include oligolabeling, nick translation, end-labeling, or PCR amplification using a labeled nucleotide. Alternatively, the sequences encoding GFMO, or any fragments thereof, may be cloned into a vector for the production of an mRNA probe. Such vectors are known in the art, are commercially available, and may be used to synthesize RNA probes in vitro by addition of an appropriate RNA polymerase such as T7, T3, or SP6 and labeled nucleotides. These procedures may be conducted using a variety of commercially available kits, such as those provided by Pharmacia &amp; Upjohn (Kalamazoo, Mich.), Promega (Madison, Wis.), and U.S. Biochemical Corp. (Cleveland, Ohio). Suitable reporter molecules or labels which may be used for ease of detection include radionuclides, enzymes, fluorescent, chemiluminescent, or chromogenic agents, as well as substrates, cofactors, inhibitors, magnetic particles, and the like. <span style="display: none">[0100] </span> </div>
</li> <li> <para-num num="[0101]"> </para-num> <div class="description-line" id="P-0101" num="0101">Host cells transformed with nucleotide sequences encoding GFMO may be cultured under conditions suitable for the expression and recovery of the protein from cell culture. The protein produced by a transformed cell may be secreted or retained intracellularly depending on the sequence and/or the vector used. As will be understood by those of skill in the art, expression vectors containing polynucleotides which encode GFMO may be designed to contain signal sequences which direct secretion of GFMO through a prokaryotic or eukaryotic cell membrane. <span style="display: none">[0101] </span> </div>
</li> <li> <para-num num="[0102]"> </para-num> <div class="description-line" id="P-0102" num="0102">In addition, a host cell strain may be chosen for its ability to modulate expression of the inserted sequences or to process the expressed protein in the desired fashion. Such modifications of the polypeptide include, but are not limited to, acetylation, carboxylation, glycosylation, phosphorylation, lipidation, and acylation. Post-translational processing which cleaves a “prepro” form of the protein may also be used to specify protein targeting, folding, and/or activity. Different host cells which have specific cellular machinery and characteristic mechanisms for post-translational activities (e.g., CHO, HeLa, MDCK, HEK293, and W138), are available from the American Type Culture Collection (ATCC, Bethesda, Md.) and may be chosen to ensure the correct modification and processing of the foreign protein. <span style="display: none">[0102] </span> </div>
</li> <li> <para-num num="[0103]"> </para-num> <div class="description-line" id="P-0103" num="0103">In another embodiment of the invention, natural, modified, or recombinant nucleic acid sequences encoding GFMO may be ligated to a heterologous sequence resulting in translation of a fusion protein in any of the aforementioned host systems. For example, a chimeric GFMO protein containing a heterologous moiety that can be recognized by a commercially available antibody may facilitate the screening of peptide libraries for inhibitors of GFMO activity. Heterologous protein and peptide moieties may also facilitate purification of fusion proteins using commercially available affinity matrices. Such moieties include, but are not limited to, glutathione S-transferase (GST), maltose binding protein (MBP), thioredoxin (Trx), calmodulin binding peptide (CBP), 6-His, FLAG, c-myc, and hemagglutinin (HA). GST, MBP, Trx, CBP, and 6-His enable purification of their cognate fusion proteins on immobilized glutathione, maltose, phenylarsine oxide, calmodulin, and metal-chelate resins, respectively. FLAG, c-myc, and hemagglutinin (HA) enable immunoaffinity purification of fusion proteins using commercially available monoclonal and polyclonal antibodies that specifically recognize these epitope tags. A fusion protein may also be engineered to contain a proteolytic cleavage site located between the GFMO encoding sequence and the heterologous protein sequence, so that GFMO may be cleaved away from the heterologous moiety following purification. Methods for fusion protein expression and purification are discussed in Ausubel, F. M. et al. (1995 and periodic supplements) <span style="display: none">[0103] </span> <i>Current Protocols in Molecular Biology</i>, John Wiley &amp; Sons, New York, N.Y., ch 10. A variety of commercially available kits may also be used to facilitate expression and purification of fusion proteins. </div>
</li> <li> <para-num num="[0104]"> </para-num> <div class="description-line" id="P-0104" num="0104">In a further embodiment of the invention, synthesis of radiolabeled GFMO may be achieved in vitro using the TNT™ rabbit reticulocyte lysate or wheat germ extract systems (Promega, Madison, Wis.). These systems couple transcription and translation of protein-coding sequences operably associated with the T7, T3, or SP6 promoters. Translation takes place in the presence of a radiolabeled amino acid precursor, preferably <span style="display: none">[0104] </span> <sup>35</sup>S-methionine. </div>
</li> <li> <para-num num="[0105]"> </para-num> <div class="description-line" id="P-0105" num="0105">Fragments of GFMO may be produced not only by recombinant production, but also by direct peptide synthesis using solid-phase techniques. (See, e.g., Creighton, supra pp. 55-60.) Protein synthesis may be performed by manual techniques or by automation. Automated synthesis may be achieved, for example, using the Applied Biosystems 431 A Peptide Synthesizer (The Perkin-Elmer Corp., Norwalk, Conn.). Various fragments of GFMO may be synthesized separately and then combined to produce the full length molecule. <span style="display: none">[0105] </span> </div>
</li> <li> <para-num num="[0106]"> </para-num> <div class="description-line" id="P-0106" num="0106">Therapeutics <span style="display: none">[0106] </span> </div>
</li> <li> <para-num num="[0107]"> </para-num> <div class="description-line" id="P-0107" num="0107">Chemical and structural similarity exists between GFMO-1 and Elm<span style="display: none">[0107] </span> <b>1</b> from mouse (GI 2911144). In addition, GFMO-1 is expressed in nervous, endothelial, and connective tissues. Therefore, GFMO-1 appears to play a role in cancer and fibrotic disorders. </div>
</li> <li> <para-num num="[0108]"> </para-num> <div class="description-line" id="P-0108" num="0108">Chemical and structural similarity exists between GFMO-2 and FGF-binding protein from mouse (GI 1469936). In addition, GFMO-2 is expressed in nervous and connective tissues. Therefore, GFMO-2 appears to play a role in cancer and fibrotic disorders. <span style="display: none">[0108] </span> </div>
</li> <li> <para-num num="[0109]"> </para-num> <div class="description-line" id="P-0109" num="0109">Therefore, in one embodiment, GFMO or a fragment or derivative thereof may be administered to a subject to treat or prevent a cancer. Such cancers can include, but are not limited to, adenocarcinoma, leukemia, lymphoma, melanoma, myeloma, sarcoma, teratocarcinoma, and, in particular, cancers of the adrenal gland, bladder, bone, bone marrow, brain, breast, cervix, gall bladder, ganglia, gastrointestinal tract, heart, kidney, liver, lung, muscle, ovary, pancreas, parathyroid, penis, prostate, salivary glands, skin, spleen, testis, thymus, thyroid, and uterus. <span style="display: none">[0109] </span> </div>
</li> <li> <para-num num="[0110]"> </para-num> <div class="description-line" id="P-0110" num="0110">In another embodiment, a vector capable of expressing GFMO or a fragment or derivative thereof may be administered to a subject to treat or prevent a cancer including, but not limited to, those described above. <span style="display: none">[0110] </span> </div>
</li> <li> <para-num num="[0111]"> </para-num> <div class="description-line" id="P-0111" num="0111">In a further embodiment, a pharmaceutical composition comprising a substantially purified GFMO in conjunction with a suitable pharmaceutical carrier may be administered to a subject to treat or prevent a cancer including, but not limited to, those provided above. <span style="display: none">[0111] </span> </div>
</li> <li> <para-num num="[0112]"> </para-num> <div class="description-line" id="P-0112" num="0112">In still another embodiment, an agonist which modulates the activity of GFMO may be administered to a subject to treat or prevent a cancer including, but not limited to, those listed above. <span style="display: none">[0112] </span> </div>
</li> <li> <para-num num="[0113]"> </para-num> <div class="description-line" id="P-0113" num="0113">In another embodiment, an antagonist of GFMO may be administered to a subject to treat or prevent a fibrotic disorder. Such fibrotic disorders may include, but are not limited to, atherosclerosis, multiple sclerosis, systemic sclerosis, amyotrophic lateral sclerosis, tuberous sclerosis, arteriosclerosis, neurofibromatosis, myelofibrosis, uterine fibroids, fibrocystic breast disease, chondromyxoid fibroma, fibrous cortical defect, nonossifying fibroma, fibrous dysplasia, fibrosarcoma, malignant fibrous histiocytoma, hepatic fibrosis, dermatofibroma, glomerulosclerosis, fatty hepatocirrhosis, cirrhosis, rheumatoid arthritis, mixed connective tissue disease, idiopathic pulmonary fibrosis, nephronophthisis, and glomerulonephritis. In one aspect, an antibody which specifically binds GFMO may be used directly as an antagonist or indirectly as a targeting or delivery mechanism for bringing a pharmaceutical agent to cells or tissue which express GFMO. <span style="display: none">[0113] </span> </div>
</li> <li> <para-num num="[0114]"> </para-num> <div class="description-line" id="P-0114" num="0114">In an additional embodiment, a vector expressing the complement of the polynucleotide encoding GFMO may be administered to a subject to treat or prevent a fibrotic disorder including, but not limited to, those described above. <span style="display: none">[0114] </span> </div>
</li> <li> <para-num num="[0115]"> </para-num> <div class="description-line" id="P-0115" num="0115">In other embodiments, any of the proteins, antagonists, antibodies, agonists, complementary sequences, or vectors of the invention may be administered in combination with other appropriate therapeutic agents. Selection of the appropriate agents for use in combination therapy may be made by one of ordinary skill in the art, according to conventional pharmaceutical principles. The combination of therapeutic agents may act synergistically to effect the treatment or prevention of the various disorders described above. Using this approach, one may be able to achieve therapeutic efficacy with lower dosages of each agent, thus reducing the potential for adverse side effects. <span style="display: none">[0115] </span> </div>
</li> <li> <para-num num="[0116]"> </para-num> <div class="description-line" id="P-0116" num="0116">An antagonist of GFMO may be produced using methods which are generally known in the art. In particular, purified GFMO may be used to produce antibodies or to screen libraries of pharmaceutical agents to identify those which specifically bind GFMO. Antibodies to GFMO may also be generated using methods that are well known in the art. Such antibodies may include, but are not limited to, polyclonal, monoclonal, chimeric, and single chain antibodies, Fab fragments, and fragments produced by a Fab expression library. Neutralizing antibodies (i.e., those which inhibit dimer formation) are especially preferred for therapeutic use. <span style="display: none">[0116] </span> </div>
</li> <li> <para-num num="[0117]"> </para-num> <div class="description-line" id="P-0117" num="0117">For the production of polyclonal antibodies, various hosts including goats, rabbits, rats, mice, humans, and others may be immunized by injection with GFMO or with any fragment or oligopeptide thereof which has immunogenic properties. Rats and mice are preferred hosts for downstream applications involving monoclonal antibody production. Depending on the host species, various adjuvants may be used to increase immunological response. Such adjuvants include, but are not limited to, Freund's, mineral gels such as aluminum hydroxide, and surface active substances such as lysolecithin, pluronic polyols, polyanions, peptides, oil emulsions, KLH, and dinitrophenol. Among adjuvants used in humans, BCG (bacilli Calmette-Guerin) and <span style="display: none">[0117] </span> <i>Corynebacterium parvum </i>are especially preferable. (For review of methods for antibody production and analysis, see, e.g., Harlow, E. and Lane, D. (1988) <i>Antibodies: A Laboratory Manual</i>, Cold Spring Harbor Laboratory, Cold Spring Harbor, N.Y.) </div>
</li> <li> <para-num num="[0118]"> </para-num> <div class="description-line" id="P-0118" num="0118">It is preferred that the oligopeptides, peptides, or fragments used to induce antibodies to GFMO have an amino acid sequence consisting of at least about 5 amino acids, and, more preferably, of at least about 14 amino acids. It is also preferable that these oligopeptides, peptides, or fragments are identical to a portion of the amino acid sequence of the natural protein and contain the entire amino acid sequence of a small, naturally occurring molecule. Short stretches of GFMO amino acids may be fused with those of another protein, such as KLH, and antibodies to the chimeric molecule may be produced. <span style="display: none">[0118] </span> </div>
</li> <li> <para-num num="[0119]"> </para-num> <div class="description-line" id="P-0119" num="0119">Monoclonal antibodies to GFMO may be prepared using any technique which provides for the production of antibody molecules by continuous cell lines in culture. These include, but are not limited to, the hybridoma technique, the human B-cell hybridoma technique, and the EBV-hybridoma technique. (See, e.g., Kohler, G. et al. (1975) Nature 256:495-497; Kozbor, D. et al. (1 985) J. Immunol. Methods 81:31-42; Cote, R. J. et al. (1983) Proc. Natl. Acad. Sci. 80:2026-2030; and Cole, S. P. et al. (1984) Mol. Cell Biol. 62:109-120.) In addition, techniques developed for the production of “chimeric antibodies,” such as the splicing of mouse antibody genes to human antibody genes to obtain a molecule with appropriate antigen specificity and biological activity, can be used. (See, e.g., Morrison, S. L. et al. (1984) Proc. Natl. Acad. Sci. 81:6851-6855; Neuberger, M. S. et al. (1984) Nature 312:604-608; and Takeda, S. et al. (1985) Nature 314:452-454.) Alternatively, techniques described for the production of single chain antibodies may be adapted, using methods known in the art, to produce GFMO-specific single chain antibodies. Antibodies with related specificity, but of distinct idiotypic composition, may be generated by chain shuffling from random combinatorial immunoglobulin libraries. (See, e.g., Burton D. R. (1991) Proc. Natl. Acad. Sci. 88:10134-10137.) <span style="display: none">[0119] </span> </div>
</li> <li> <para-num num="[0120]"> </para-num> <div class="description-line" id="P-0120" num="0120">Antibodies may also be produced by inducing in vivo production in the lymphocyte population or by screening immunoglobulin libraries or panels of highly specific binding reagents as disclosed in the literature. (See, e.g., Orlandi, R. et al. (1989) Proc. Natl. Acad. Sci. 86: 3833-3837; and Winter, G. et al. (1991) Nature 349:293-299.) Antibody fragments which contain specific binding sites for GFMO may also be generated. For example, such fragments include, but are not limited to, F(ab′)2 fragments produced by pepsin digestion of the antibody molecule and Fab fragments generated by reducing the disulfide bridges of the F(ab′)2 fragments. Alternatively, Fab expression libraries may be constructed to allow rapid and easy identification of monoclonal Fab fragments with the desired specificity. (See, e.g., Huse, W. D. et al. (1989) Science 246:1275-1281.) <span style="display: none">[0120] </span> </div>
</li> <li> <para-num num="[0121]"> </para-num> <div class="description-line" id="P-0121" num="0121">Various immunoassays may be used for screening to identify antibodies having the desired specificity and minimal cross-reactivity. Numerous protocols for competitive binding or immunoradiometric assays using either polyclonal or monoclonal antibodies with established specificities are well known in the art. Such immunoassays typically involve the measurement of complex formation .between GFMO and its specific antibody. A two-site, monoclonal-based immunoassay utilizing monoclonal antibodies reactive to two non-interfering GFMO epitopes is preferred, but a competitive binding assay may also be employed. (Maddox, supra.) <span style="display: none">[0121] </span> </div>
</li> <li> <para-num num="[0122]"> </para-num> <div class="description-line" id="P-0122" num="0122">Various methods such as Scatchard analysis in conjunction with radioimmunoassay techniques may be used to assess the affinity of antibodies for GFMO. Affinity is expressed as an association constant, K<span style="display: none">[0122] </span> <sub>a</sub>, which is defined as the molar concentration of GFMO-antibody complex divided by the molar concentrations of free antigen and free antibody under equilibrium conditions. The K<sub>a </sub>determined for a preparation of polyclonal antibodies, which are heterogeneous in their affinities for multiple GFMO epitopes, represents the average affinity, or avidity, of the antibodies for GFMO. The K<sub>a </sub>determined for a preparation of monoclonal antibodies, which are monospecific for a particular GFMO epitope, represents a true measure of affinity. High-affinity antibody preparations with K<sup>a </sup>ranging from about 10<sup>9 </sup>to 10<sup>12 </sup>L/mole are preferred for use in immunoassays in which the GFMO-antibody complex must withstand rigorous manipulations. Low-affinity antibody preparations with K<sup>a </sup>ranging from about 10<sup>6 </sup>to 10<sup>7 </sup>L/mole are preferred for use in immunopurification and similar procedures which ultimately require dissociation of GFMO, preferably in active form, from the antibody. (Catty, D. (1988) Antibodies. <i>Volume I: A Practical Approach</i>, IRL Press, Washington, D. C.; and Liddell, J. E. and Cryer, A. (1991) <i>A Practical Guide to Monoclonal Antibodies</i>, John Wiley &amp; Sons, New York, N.Y.) </div>
</li> <li> <para-num num="[0123]"> </para-num> <div class="description-line" id="P-0123" num="0123">The titre and avidity of polyclonal antibody preparations may be further evaluated to determine the quality and suitability of such preparations for certain downstream applications. For example, a polyclonal antibody preparation containing at least 1-2 mg specific antibody/ml, preferably 5-10 mg specific antibody/ml, is preferred for use in procedures requiring precipitation of GFMO-antibody complexes. Procedures for evaluating antibody specificity, titer, and avidity, and guidelines for antibody quality and usage in various applications, are generally available. (See, e.g., Catty, supra, and Coligan et al. supra.) <span style="display: none">[0123] </span> </div>
</li> <li> <para-num num="[0124]"> </para-num> <div class="description-line" id="P-0124" num="0124">In another embodiment of the invention, the polynucleotides encoding GFMO, or any fragment or complement thereof, may be used for therapeutic purposes. In one aspect, the complement of the polynucleotide encoding GFMO may be used in situations in which it would be desirable to block the transcription of the mRNA. In particular, cells may be transformed with sequences complementary to polynucleotides encoding GFMO. Thus, complementary molecules or fragments may be used to modulate GFMO activity, or to achieve regulation of gene function. Such technology is now well known in the art, and sense or antisense oligonucleotides or larger fragments can be designed from various locations along the coding or control regions of sequences encoding GFMO. <span style="display: none">[0124] </span> </div>
</li> <li> <para-num num="[0125]"> </para-num> <div class="description-line" id="P-0125" num="0125">Expression vectors derived from retroviruses, adenoviruses, or herpes or vaccinia viruses, or from various bacterial plasmids, may be used for delivery of nucleotide sequences to the targeted organ, tissue, or cell population. Methods which are well known to those skilled in the art can be used to construct vectors to express nucleic acid sequences complementary to the polynucleotides encoding GFMO. (See, e.g., Sambrook, supra; and Ausubel, supra.) <span style="display: none">[0125] </span> </div>
</li> <li> <para-num num="[0126]"> </para-num> <div class="description-line" id="P-0126" num="0126">Genes encoding GFMO can be turned off by transforming a cell or tissue with expression vectors which express high levels of a polynucleotide, or fragment thereof, encoding GFMO. Such constructs may be used to introduce untranslatable sense or antisense sequences into a cell. Even in the absence of integration into the DNA, such vectors may continue to transcribe RNA molecules until they are disabled by endogenous nucleases. Transient expression may last for a month or more with a non-replicating vector, and may last even longer if appropriate replication elements are part of the vector system. <span style="display: none">[0126] </span> </div>
</li> <li> <para-num num="[0127]"> </para-num> <div class="description-line" id="P-0127" num="0127">As mentioned above, modifications of gene expression can be obtained by designing complementary sequences or antisense molecules (DNA, RNA, or PNA) to the control, 5′, or regulatory regions of the gene encoding GFMO. Oligonucleotides derived from the transcription initiation site, e.g., between about positions −10 and +10 from the start site, are preferred. Similarly, inhibition can be achieved using triple helix base-pairing methodology. Triple helix pairing is useful because it causes inhibition of the ability of the double helix to open sufficiently for the binding of polymerases, transcription factors, or regulatory molecules. Recent therapeutic advances using triplex DNA have been described in the literature. (See, e.g., Gee, J. E. et al. (1994) in Huber, B. E. and B. I. Carr, <span style="display: none">[0127] </span> <i>Molecular and Immunologic Approaches</i>, Futura Publishing Co., Mt. Kisco, N.Y., pp. 163-177.) A complementary sequence or antisense molecule may also be designed to block translation of mRNA by preventing the transcript from binding to ribosomes. </div>
</li> <li> <para-num num="[0128]"> </para-num> <div class="description-line" id="P-0128" num="0128">Ribozymes, enzymatic RNA molecules, may also be used to catalyze the specific cleavage of RNA. The mechanism of ribozyme action involves sequence-specific hybridization of the ribozyme molecule to complementary target RNA, followed by endonucleolytic cleavage. For example, engineered hammerhead motif ribozyme molecules may specifically and efficiently catalyze endonucleolytic cleavage of sequences encoding GFMO. <span style="display: none">[0128] </span> </div>
</li> <li> <para-num num="[0129]"> </para-num> <div class="description-line" id="P-0129" num="0129">Specific ribozyme cleavage sites within any potential RNA target are initially identified by scanning the target molecule for ribozyme cleavage sites, including the following sequences: GUA, GUU, and GUC. Once identified, short RNA sequences of between 15 and 20 ribonucleotides, corresponding to the region of the target gene containing the cleavage site, may be evaluated for secondary structural features which may render the oligonucleotide inoperable. The suitability of candidate targets may also be evaluated by testing accessibility to hybridization with complementary oligonucleotides using ribonuclease protection assays. <span style="display: none">[0129] </span> </div>
</li> <li> <para-num num="[0130]"> </para-num> <div class="description-line" id="P-0130" num="0130">Complementary ribonucleic acid molecules and ribozymes of the invention may be prepared by any method known in the art for the synthesis of nucleic acid molecules. These include techniques for chemically synthesizing oligonucleotides such as solid phase phosphoramidite chemical synthesis. Alternatively, RNA molecules may be generated by in vitro and in vivo transcription of DNA sequences encoding GFMO. Such DNA sequences may be incorporated into a wide variety of vectors with suitable RNA polymerase promoters such as T7 or SP6. Alternatively, these cDNA constructs that synthesize complementary RNA, constitutively or inducibly, can be introduced into cell lines, cells, or tissues. <span style="display: none">[0130] </span> </div>
</li> <li> <para-num num="[0131]"> </para-num> <div class="description-line" id="P-0131" num="0131">RNA molecules may be modified to increase intracellular stability and half-life. Possible modifications include, but are not limited to, the addition of flanking sequences at the 5′ and/or 3′ ends of the molecule, or the use of phosphorothioate or 2′ O-methyl rather than phosphodiesterase linkages within the backbone of the molecule. This concept is inherent in the production of PNAs and can be extended in all of these molecules by the inclusion of nontraditional bases such as inosine, queosine, and wybutosine, as well as acetyl-, methyl-, thio-, and similarly modified forms of adenine, cytidine, guanine, thymine, and uridine which are not as easily recognized by endogenous endonucleases. <span style="display: none">[0131] </span> </div>
</li> <li> <para-num num="[0132]"> </para-num> <div class="description-line" id="P-0132" num="0132">Many methods for introducing vectors into cells or tissues are available and equally suitable for use in vivo, in vitro, and ex vivo. For ex vivo therapy, vectors may be introduced into stem cells taken from the patient and clonally propagated for autologous transplant back into that same patient. Delivery by transfection, by liposome injections, or by polycationic amino polymers may be achieved using methods which are well known in the art. (See, e.g., Goldman, C. K. et al. (1997) Nature Biotechnology 15:462-466.) <span style="display: none">[0132] </span> </div>
</li> <li> <para-num num="[0133]"> </para-num> <div class="description-line" id="P-0133" num="0133">Any of the therapeutic methods described above may be applied to any subject in need of such therapy, including, for example, mammals such as dogs, cats, cows, horses, rabbits, monkeys, and most preferably, humans. <span style="display: none">[0133] </span> </div>
</li> <li> <para-num num="[0134]"> </para-num> <div class="description-line" id="P-0134" num="0134">An additional embodiment of the invention relates to the administration of a pharmaceutical or sterile composition, in conjunction with a pharmaceutically acceptable carrier, for any of the therapeutic effects discussed above. Such pharmaceutical compositions may consist of GFMO, antibodies to GFMO, and mimetics, agonists, antagonists, or inhibitors of GFMO. The compositions may be administered alone or in combination with at least one other agent, such as a stabilizing compound, which may be administered in any sterile, biocompatible pharmaceutical carrier including, but not limited to, saline, buffered saline, dextrose, and water. The compositions may be administered to a patient alone, or in combination with other agents, drugs, or hormones. <span style="display: none">[0134] </span> </div>
</li> <li> <para-num num="[0135]"> </para-num> <div class="description-line" id="P-0135" num="0135">The pharmaceutical compositions utilized in this invention may be administered by any number of routes including, but not limited to, oral, intravenous, intramuscular, intra-arterial, intramedullary, intrathecal, intraventricular, transdermal, subcutaneous, intraperitoneal, intranasal, enteral, topical, sublingual, or rectal means. <span style="display: none">[0135] </span> </div>
</li> <li> <para-num num="[0136]"> </para-num> <div class="description-line" id="P-0136" num="0136">In addition to the active ingredients, these pharmaceutical compositions may contain suitable pharmaceutically-acceptable carriers comprising excipients and auxiliaries which facilitate processing of the active compounds into preparations which can be used pharmaceutically. Further details on techniques for formulation and administration may be found in the latest edition of <span style="display: none">[0136] </span> <i>Remington's Pharmaceutical Sciences </i>(Maack Publishing Co., Easton, Pa.). </div>
</li> <li> <para-num num="[0137]"> </para-num> <div class="description-line" id="P-0137" num="0137">Pharmaceutical compositions for oral administration can be formulated using pharmaceutically acceptable carriers well known in the art in dosages suitable for oral administration. Such carriers enable the pharmaceutical compositions to be formulated as tablets, pills, dragees, capsules, liquids, gels, syrups, slurries, suspensions, and the like, for ingestion by the patient. <span style="display: none">[0137] </span> </div>
</li> <li> <para-num num="[0138]"> </para-num> <div class="description-line" id="P-0138" num="0138">Pharmaceutical preparations for oral use can be obtained through combining active compounds with solid excipient and processing the resultant mixture of granules (optionally, after grinding) to obtain tablets or dragee cores. Suitable auxiliaries can be added, if desired. Suitable excipients include carbohydrate or protein fillers, such as sugars, including lactose, sucrose, mannitol, and sorbitol; starch from corn, wheat, rice, potato, or other plants; cellulose, such as methyl cellulose, hydroxypropylmethyl-cellulose, or sodium carboxymethylcellulose; gums, including arabic and tragacanth; and proteins, such as gelatin and collagen. If desired, disintegrating or solubilizing agents may be added, such as the cross-linked polyvinyl pyrrolidone, agar, and alginic acid or a salt thereof, such as sodium alginate. <span style="display: none">[0138] </span> </div>
</li> <li> <para-num num="[0139]"> </para-num> <div class="description-line" id="P-0139" num="0139">Dragee cores may be used in conjunction with suitable coatings, such as concentrated sugar solutions, which may also contain gum arabic, talc, polyvinylpyrrolidone, carbopol gel, polyethylene glycol, and/or titanium dioxide, lacquer solutions, and suitable organic solvents or solvent mixtures. Dyestuffs or pigments may be added to the tablets or dragee coatings for product identification or to characterize the quantity of active compound, i.e., dosage. <span style="display: none">[0139] </span> </div>
</li> <li> <para-num num="[0140]"> </para-num> <div class="description-line" id="P-0140" num="0140">Pharmaceutical preparations which can be used orally include push-fit capsules made of gelatin, as well as soft, sealed capsules made of gelatin and a coating, such as glycerol or sorbitol. Push-fit capsules can contain active ingredients mixed with fillers or binders, such as lactose or starches, lubricants, such as talc or magnesium stearate, and, optionally, stabilizers. In soft capsules, the active compounds may be dissolved or suspended in suitable liquids, such as fatty oils, liquid, or liquid polyethylene glycol with or without stabilizers. <span style="display: none">[0140] </span> </div>
</li> <li> <para-num num="[0141]"> </para-num> <div class="description-line" id="P-0141" num="0141">Pharmaceutical formulations suitable for parenteral administration may be formulated in aqueous solutions, preferably in physiologically compatible buffers such as Hanks's solution, Ringer's solution, or physiologically buffered saline. Aqueous injection suspensions may contain substances which increase the viscosity of the suspension, such as sodium carboxymethyl cellulose, sorbitol, or dextran. Additionally, suspensions of the active compounds may be prepared as appropriate oily injection suspensions. Suitable lipophilic solvents or vehicles include fatty oils, such as sesame oil, or synthetic fatty acid esters, such as ethyl oleate, triglycerides, or liposomes. Non-lipid polycationic amino polymers may also be used for delivery. Optionally, the suspension may also contain suitable stabilizers or agents to increase the solubility of the compounds and allow for the preparation of highly concentrated solutions. <span style="display: none">[0141] </span> </div>
</li> <li> <para-num num="[0142]"> </para-num> <div class="description-line" id="P-0142" num="0142">For topical or nasal administration, penetrants appropriate to the particular barrier to be permeated are used in the formulation. Such penetrants are generally known in the art. <span style="display: none">[0142] </span> </div>
</li> <li> <para-num num="[0143]"> </para-num> <div class="description-line" id="P-0143" num="0143">The pharmaceutical compositions of the present invention may be manufactured in a manner that is known in the art, e.g., by means of conventional mixing, dissolving, granulating, dragee-making, levigating, emulsifying, encapsulating, entrapping, or lyophilizing processes. <span style="display: none">[0143] </span> </div>
</li> <li> <para-num num="[0144]"> </para-num> <div class="description-line" id="P-0144" num="0144">The pharmaceutical composition may be provided as a salt and can be formed with many acids, including but not limited to, hydrochloric, sulfuric, acetic, lactic, tartaric, malic, and succinic acid. Salts tend to be more soluble in aqueous or other protonic solvents than are the corresponding free base forms. In other cases, the preferred preparation may be a lyophilized powder which may contain any or all of the following: 1 mM to 50 mM histidine, 0.1% to 2% sucrose, and 2% to 7% mannitol, at a pH range of 4.5 to 5.5, that is combined with buffer prior to use. <span style="display: none">[0144] </span> </div>
</li> <li> <para-num num="[0145]"> </para-num> <div class="description-line" id="P-0145" num="0145">After pharmaceutical compositions have been prepared, they can be placed in an appropriate container and labeled for treatment of an indicated condition. For administration of GFMO, such labeling would include amount, frequency, and method of administration. <span style="display: none">[0145] </span> </div>
</li> <li> <para-num num="[0146]"> </para-num> <div class="description-line" id="P-0146" num="0146">Pharmaceutical compositions suitable for use in the invention include compositions wherein the active ingredients are contained in an effective amount to achieve the intended purpose. The determination of an effective dose is well within the capability of those skilled in the art. <span style="display: none">[0146] </span> </div>
</li> <li> <para-num num="[0147]"> </para-num> <div class="description-line" id="P-0147" num="0147">For any compound, the therapeutically effective dose can be estimated initially either in cell culture assays, e.g., of neoplastic cells or in animal models such as mice, rats, rabbits, dogs, or pigs. An animal model may also be used to determine the appropriate concentration range and route of administration. Such information can then be used to determine useful doses and routes for administration in humans. <span style="display: none">[0147] </span> </div>
</li> <li> <para-num num="[0148]"> </para-num> <div class="description-line" id="P-0148" num="0148">A therapeutically effective dose refers to that amount of active ingredient, for example GFMO or fragments thereof, antibodies of GFMO, and agonists, antagonists or inhibitors of GFMO, which ameliorates the symptoms or condition. Therapeutic efficacy and toxicity may be determined by standard pharmaceutical procedures in cell cultures or with experimental animals, such as by calculating the ED<span style="display: none">[0148] </span> <sub>50 </sub>(the dose therapeutically effective in 50% of the population) or LD<sub>50 </sub>(the dose lethal to 50% of the population) statistics. The dose ratio of therapeutic to toxic effects is the therapeutic index, and it can be expressed as the ED<sub>50</sub>/LD<sub>50 </sub>ratio. Pharmaceutical compositions which exhibit large therapeutic indices are preferred. The data obtained from cell culture assays and animal studies are used to formulate a range of dosage for human use. The dosage contained in such compositions is preferably within a range of circulating concentrations that includes the ED<sub>50 </sub>with little or no toxicity. The dosage varies within this range depending upon the dosage form employed, the sensitivity of the patient, and the route of administration. </div>
</li> <li> <para-num num="[0149]"> </para-num> <div class="description-line" id="P-0149" num="0149">The exact dosage will be determined by the practitioner, in light of factors related to the subject requiring treatment. Dosage and administration are adjusted to provide sufficient levels of the active moiety or to maintain the desired effect. Factors which may be taken into account include the severity of the disease state, the general health of the subject, the age, weight, and gender of the subject, time and frequency of administration, drug combination(s), reaction sensitivities, and response to therapy. Long-acting pharmaceutical compositions may be administered every 3 to 4 days, every week, or biweekly depending on the half-life and clearance rate of the particular formulation. <span style="display: none">[0149] </span> </div>
</li> <li> <para-num num="[0150]"> </para-num> <div class="description-line" id="P-0150" num="0150">Normal dosage amounts may vary from about 0.1 μg to 100,000 μg, up to a total dose of about 1 gram, depending upon the route of administration. Guidance as to particular dosages and methods of delivery is provided in the literature and generally available to practitioners in the art. Those skilled in the art will employ different formulations for nucleotides than for proteins or their inhibitors. Similarly, delivery of polynucleotides or polypeptides will be specific to particular cells, conditions, locations, etc. <span style="display: none">[0150] </span> </div>
</li> <li> <para-num num="[0151]"> </para-num> <div class="description-line" id="P-0151" num="0151">Diagnostics <span style="display: none">[0151] </span> </div>
</li> <li> <para-num num="[0152]"> </para-num> <div class="description-line" id="P-0152" num="0152">In another embodiment, antibodies which specifically bind GFMO may be used for the diagnosis of disorders characterized by expression of GFMO, or in assays to monitor patients being treated with GFMO or agonists, antagonists, or inhibitors of GFMO. Antibodies useful for diagnostic purposes may be prepared in the same manner as described above for therapeutics. Diagnostic assays for GFMO include methods which utilize the antibody and a label to detect GFMO in human body fluids or in extracts of cells or tissues. The antibodies may be used with or without modification, and may be labeled by covalent or non-covalent attachment of a reporter molecule. A wide variety of reporter molecules, several of which are described above, are known in the art and may be used. <span style="display: none">[0152] </span> </div>
</li> <li> <para-num num="[0153]"> </para-num> <div class="description-line" id="P-0153" num="0153">A variety of protocols for measuring GFMO, including ELISAs, RIAs, and FACS, are known in the art and provide a basis for diagnosing altered or abnormal levels of GFMO expression. Normal or standard values for GFMO expression are established by combining body fluids or cell extracts taken from normal mammalian subjects, preferably human, with antibody to GFMO under conditions suitable for complex formation The amount of standard complex formation may be quantitated by various methods, preferably by photometric means. Quantities of GFMO expressed in subject, control, and disease samples from biopsied tissues are compared with the standard values. Deviation between standard and subject values establishes the parameters for diagnosing disease. <span style="display: none">[0153] </span> </div>
</li> <li> <para-num num="[0154]"> </para-num> <div class="description-line" id="P-0154" num="0154">In another embodiment of the invention, the polynucleotides encoding GFMO may be used for diagnostic purposes. The polynucleotides which may be used include oligonucleotide sequences, complementary RNA and DNA molecules, and PNAs. The polynucleotides may be used to detect and quantitate gene expression in biopsied tissues in which expression of GFMO may be correlated with disease. The diagnostic assay may be used to determine absence, presence, and excess expression of GFMO, and to monitor regulation of GFMO levels during therapeutic intervention. <span style="display: none">[0154] </span> </div>
</li> <li> <para-num num="[0155]"> </para-num> <div class="description-line" id="P-0155" num="0155">In one aspect, hybridization with PCR probes which are capable of detecting polynucleotide sequences, including genomic sequences, encoding GFMO or closely related molecules may be used to identify nucleic acid sequences which encode GFMO. The specificity of the probe, whether it is made from a highly specific region, e.g., the 5′ regulatory region, or from a less specific region, e.g., a conserved motif, and the stringency of the hybridization or amplification (maximal, high, intermediate, or low), will determine whether the probe identifies only naturally occurring sequences encoding GFMO, allelic variants, or related sequences. <span style="display: none">[0155] </span> </div>
</li> <li> <para-num num="[0156]"> </para-num> <div class="description-line" id="P-0156" num="0156">Probes may also be used for the detection of related sequences, and should preferably have at least 50% sequence identity to any of the GFMO encoding sequences. The hybridization probes of the subject invention may be DNA or RNA and may be derived from the sequence of SEQ ID NO: 3, SEQ ID NO: 4, or from genomic sequences including promoters, enhancers, and introns of the GFMO gene. <span style="display: none">[0156] </span> </div>
</li> <li> <para-num num="[0157]"> </para-num> <div class="description-line" id="P-0157" num="0157">Means for producing specific hybridization probes for DNAs encoding GFMO include the cloning of polynucleotide sequences encoding GFMO or GFMO derivatives into vectors for the production of mRNA probes. Such vectors are known in the art, are commercially available, and may be used to synthesize RNA probes in vitro by means of the addition of the appropriate RNA polymerases and the appropriate labeled nucleotides. Hybridization probes may be labeled by a variety of reporter groups, for example, by radionuclides such as <span style="display: none">[0157] </span> <sup>32</sup>p or <sup>35</sup>S, or by enzymatic labels, such as alkaline phosphatase coupled to the probe via avidin/biotin coupling systems, and the like. </div>
</li> <li> <para-num num="[0158]"> </para-num> <div class="description-line" id="P-0158" num="0158">Polynucleotide sequences encoding GFMO may be used for the diagnosis of a disorder associated with expression of GFMO. Examples of such a disorder include, but are not limited to, cancers, such as adenocarcinoma, leukemia, lymphoma, melanoma, myeloma, sarcoma, teratocarcinoma, and, in particular, cancers of the adrenal gland, bladder, bone, bone marrow, brain, breast, cervix, gall bladder, ganglia, gastrointestinal tract, heart, kidney, liver, lung, muscle, ovary, pancreas, parathyroid, penis, prostate, salivary glands, skin, spleen, testis, thymus, thyroid, and uterus; and fibrotic disorders, such as atherosclerosis, multiple sclerosis, systemic sclerosis, amyotrophic lateral sclerosis, tuberous sclerosis, arteriosclerosis, neurofibromatosis, myelofibrosis, uterine fibroids, fibrocystic breast disease, chondromyxoid fibroma, fibrous cortical defect, nonossifying fibroma, fibrous dysplasia, fibrosarcoma, malignant fibrous histiocytoma, hepatic fibrosis, dermatofibroma, glomerulosclerosis, fatty hepatocirrhosis, cirrhosis, rheumatoid arthritis, mixed connective tissue disease, idiopathic pulmonary fibrosis, nephronophthisis, and glomerulonephritis. The polynucleotide sequences encoding GFMO may be used in Southern or Northern analysis, dot blot, or other membrane-based technologies; in PCR technologies; in dipstick, pin, and ELISA assays; and in microarrays utilizing fluids or tissues from patients to detect altered GFMO expression. Such qualitative or quantitative methods are well known in the art. <span style="display: none">[0158] </span> </div>
</li> <li> <para-num num="[0159]"> </para-num> <div class="description-line" id="P-0159" num="0159">In a particular aspect, the nucleotide sequences encoding GFMO may be useful in assays that detect the presence of associated disorders, particularly those mentioned above. The nucleotide sequences encoding GFMO may be labeled by standard methods and added to a fluid or tissue sample from a patient under conditions suitable for the formation of hybridization complexes. After a suitable incubation period, the sample is washed and the signal is quantitated and compared with a standard value. If the amount of signal in the patient sample is significantly altered in comparison to a control sample then the presence of altered levels of nucleotide sequences encoding GFMO in the sample indicates the presence of the associated disorder. Such assays may also be used to evaluate the efficacy of a particular therapeutic treatment regimen in animal studies, in clinical trials, or to monitor the treatment of an individual patient. <span style="display: none">[0159] </span> </div>
</li> <li> <para-num num="[0160]"> </para-num> <div class="description-line" id="P-0160" num="0160">In order to provide a basis for the diagnosis of a disorder associated with expression of GFMO, a normal or standard profile for expression is established. This may be accomplished by combining body fluids or cell extracts taken from normal subjects, either animal or human, with a sequence, or a fragment thereof, encoding GFMO, under conditions suitable for hybridization or amplification. Standard hybridization may be quantified by comparing the values obtained from normal subjects with values from an experiment in which a known amount of a substantially purified polynucleotide is used. Standard values obtained in this manner may be compared with values obtained from samples from patients who are symptomatic for a disorder. Deviation from standard values is used to establish the presence of a disorder. <span style="display: none">[0160] </span> </div>
</li> <li> <para-num num="[0161]"> </para-num> <div class="description-line" id="P-0161" num="0161">Once the presence of a disorder is established and a treatment protocol is initiated, hybridization assays may be repeated on a regular basis to determine if the level of expression in the patient begins to approximate that which is observed in the normal subject. The results obtained from successive assays may be used to show the efficacy of treatment over a period ranging from several days to months. <span style="display: none">[0161] </span> </div>
</li> <li> <para-num num="[0162]"> </para-num> <div class="description-line" id="P-0162" num="0162">With respect to cancer, the presence of a relatively high amount of transcript in biopsied tissue from an individual may indicate a predisposition for the development of the disease, or may provide a means for detecting the disease prior to the appearance of actual clinical symptoms. A more definitive diagnosis of this type may allow health professionals to employ preventative measures or aggressive treatment earlier thereby preventing the development or further progression of the cancer. <span style="display: none">[0162] </span> </div>
</li> <li> <para-num num="[0163]"> </para-num> <div class="description-line" id="P-0163" num="0163">Additional diagnostic uses for oligonucleotides designed from the sequences encoding GFMO may involve the use of PCR. These oligomers may be chemically synthesized, generated enzymatically, or produced in vitro. Oligomers will preferably contain a fragment of a polynucleotide encoding GFMO, or a fragment of a polynucleotide complementary to the polynucleotide encoding GFMO, and will be employed under optimized conditions for identification of a specific gene or condition. Oligomers may also be employed under less stringent conditions for detection or quantitation of closely related DNA or RNA sequences. <span style="display: none">[0163] </span> </div>
</li> <li> <para-num num="[0164]"> </para-num> <div class="description-line" id="P-0164" num="0164">Methods which may also be used to quantitate the expression of GFMO include radiolabeling or biotinylating nucleotides, coamplification of a control nucleic acid, and interpolating results from standard curves. (See, e.g., Melby, P. C. et al. (1993) J. Immunol. Methods 159:235-244; and Duplaa, C. et al. (1993) Anal. Biochem. 229-236.) The speed of quantitation of multiple samples may be accelerated by running the assay in an ELISA format where the oligomer of interest is presented in various dilutions and a spectrophotometric or colorimetric response gives rapid quantitation. <span style="display: none">[0164] </span> </div>
</li> <li> <para-num num="[0165]"> </para-num> <div class="description-line" id="P-0165" num="0165">In further embodiments, oligonucleotides or longer fragments derived from any of the polynucleotide sequences described herein may be used as targets in a microarray. The microarray can be used to monitor the expression level of large numbers of genes simultaneously and to identify genetic variants, mutations, and polymorphisms. This information may be used to determine gene function, to understand the genetic basis of a disorder, to diagnose a disorder, and to develop and monitor the activities of therapeutic agents. <span style="display: none">[0165] </span> </div>
</li> <li> <para-num num="[0166]"> </para-num> <div class="description-line" id="P-0166" num="0166">Microarrays may be prepared, used, and analyzed using methods known in the art. (See, e.g., Brennan, T. M. et al. (1995) U.S. Pat. No. 5,474,796; Schena, M. et al. (1996) Proc. Natl. Acad. Sci. 93:10614-10619; Baldeschweiler et al. (1995) PCT application WO95/251116; Shalon, D. et al. (1995) PCT application W095/35505; Heller, R. A. et al. (1997) Proc. Natl. Acad. Sci. 94:2150-2155; and Heller, M. J. et al. (1997) U.S. Pat. No. 5,605,662.) <span style="display: none">[0166] </span> </div>
</li> <li> <para-num num="[0167]"> </para-num> <div class="description-line" id="P-0167" num="0167">In another embodiment of the invention, nucleic acid sequences encoding GFMO may be used to generate hybridization probes useful in mapping the naturally occurring genomic sequence. The sequences may be mapped to a particular chromosome, to a specific region of a chromosome, or to artificial chromosome constructions, e.g., human artificial chromosomes (HACs), yeast artificial chromosomes (YACs), bacterial artificial chromosomes (BACs), bacterial P1 constructions, or single chromosome cDNA libraries. (See, e.g., Price, C. M. (1993) Blood Rev. 7:127-134; and Trask, B. J. (1991) Trends Genet. 7:149-154.) <span style="display: none">[0167] </span> </div>
</li> <li> <para-num num="[0168]"> </para-num> <div class="description-line" id="P-0168" num="0168">Fluorescent in situ hybridization (FISH) may be correlated with other physical chromosome mapping techniques and genetic map data. (See, e.g., Heinz-Ulrich, et al. (1995) in Meyers, R. A. (ed.) <span style="display: none">[0168] </span> <i>Molecular Biology and Biotechnology</i>, VCH Publishers New York, N.Y., pp. 965-968.) Examples of genetic map data can be found in various scientific journals or at the Online Mendelian Inheritance in Man (OMIM) site. Correlation between the location of the gene encoding GFMO on a physical chromosomal map and a specific disorder, or a predisposition to a specific disorder, may help define the region of DNA associated with that disorder. The nucleotide sequences of the invention may be used to detect differences in gene sequences among normal, carrier, and affected individuals. </div>
</li> <li> <para-num num="[0169]"> </para-num> <div class="description-line" id="P-0169" num="0169">In situ hybridization of chromosomal preparations and physical mapping techniques, such as linkage analysis using established chromosomal markers, may be used for extending genetic maps. Often the placement of a gene on the chromosome of another mammalian species, such as mouse, may reveal associated markers even if the number or arm of a particular human chromosome is not known. New sequences can be assigned to chromosomal arms by physical mapping. This provides valuable information to investigators searching for disease genes using positional cloning or other gene discovery techniques. Once the disease or syndrome has been crudely localized by genetic linkage to a particular genomic region, e.g., ataxia-telangiectasia to 11 q22-23, any sequences mapping to that area may represent associated or regulatory genes for further investigation. (See, e.g., Gatti, R. A. et al. (1988) Nature 336:577-580.) The nucleotide sequence of the subject invention may also be used to detect differences in the chromosomal location due to translocation, inversion, etc., among normal, carrier, or affected individuals. <span style="display: none">[0169] </span> </div>
</li> <li> <para-num num="[0170]"> </para-num> <div class="description-line" id="P-0170" num="0170">In another embodiment of the invention, GFMO, its catalytic or immunogenic fragments, or oligopeptides thereof can be used for screening libraries of compounds in any of a variety of drug screening techniques. The fragment employed in such screening may be free in solution, affixed to a solid support, borne on a cell surface, or located intracellularly. The formation of binding complexes between GFMO and the agent being tested may be measured. <span style="display: none">[0170] </span> </div>
</li> <li> <para-num num="[0171]"> </para-num> <div class="description-line" id="P-0171" num="0171">Another technique for drug screening provides for high throughput screening of compounds having suitable binding affinity to the protein of interest. (See, e.g., Geysen, et al. (1984) PCT application W084/03564.) In this method, large numbers of different small test compounds are synthesized on a solid substrate, such as plastic pins or some other surface. The test compounds are reacted with GFMO, or fragments thereof, and washed. Bound GFMO is then detected by methods well known in the art. Purified GFMO can also be coated directly onto plates for use in the aforementioned drug screening techniques. Alternatively, non-neutralizing antibodies can be used to capture the peptide and immobilize it on a solid support. <span style="display: none">[0171] </span> </div>
</li> <li> <para-num num="[0172]"> </para-num> <div class="description-line" id="P-0172" num="0172">In another embodiment, one may use competitive drug screening assays in which neutralizing antibodies capable of binding GFMO specifically compete with a test compound for binding GFMO. In this manner, antibodies can be used to detect the presence of any peptide which shares one or more antigenic determinants with GFMO. <span style="display: none">[0172] </span> </div>
</li> <li> <para-num num="[0173]"> </para-num> <div class="description-line" id="P-0173" num="0173">In additional embodiments, the nucleotide sequences which encode GFMO may be used in any molecular biology techniques that have yet to be developed, provided the new techniques rely on properties of nucleotide sequences that are currently known, including, but not limited to, such properties as the triplet genetic code and specific base pair interactions. <span style="display: none">[0173] </span> </div>
</li> <li> <para-num num="[0174]"> </para-num> <div class="description-line" id="P-0174" num="0174">The examples below are provided to illustrate the subject invention and are not included for the purpose of limiting the invention. <span style="display: none">[0174] </span> </div>
</li> <heading>EXAMPLES </heading>
<li> <para-num num="[0175]"> </para-num> <div class="description-line" id="P-0175" num="0175">I. cDNA Library Construction <span style="display: none">[0175] </span> </div>
</li> <li> <para-num num="[0176]"> </para-num> <div class="description-line" id="P-0176" num="0176">Conutut01 <span style="display: none">[0176] </span> </div>
</li> <li> <para-num num="[0177]"> </para-num> <div class="description-line" id="P-0177" num="0177">The CONUTUT01 cDNA library was constructed using RNA isolated from sigmoid mesentery tumor tissue obtained from a 61-year old female during a total abdominal hysterectomy and salpingo-oophorectomy with regional lymph node excision. Pathology indicated a metastatic grade 4 malignant mixed mullerian tumor present in the sigmoid mesentery at two sites. <span style="display: none">[0177] </span> </div>
</li> <li> <para-num num="[0178]"> </para-num> <div class="description-line" id="P-0178" num="0178">Drglnot01 <span style="display: none">[0178] </span> </div>
</li> <li> <para-num num="[0179]"> </para-num> <div class="description-line" id="P-0179" num="0179">The DRGLNOT<span style="display: none">[0179] </span> <b>01</b> cDNA library was constructed using RNA isolated from cervical spine dorsal root ganglion tissue obtained from a 32-year-old Caucasian male who died from acute pulmonary edema, acute bronchopneumonia, bilateral pleural effusions, pericardial effusion and malignant lymphoma (natural killer cell type). Patient history included probable cytomegalovirus infection, hepatic congestion and steatosis, splenomegaly, hemorrhagic cystitis, thyroid hemorrhage, and Bell's palsy. Previous surgeries included a colonoscopy, an adenotonsillectomy, and a nasopharyngeal endoscopy and biopsy; treatment included radiation therapy. </div>
</li> <li> <para-num num="[0180]"> </para-num> <div class="description-line" id="P-0180" num="0180">Conutut01 and Drglnot01 <span style="display: none">[0180] </span> </div>
</li> <li> <para-num num="[0181]"> </para-num> <div class="description-line" id="P-0181" num="0181">The frozen tissue was homogenized and lysed in Trizol reagent (Cat. #10296-028; Life Technologies, Inc., Gaithersburg, Md.), a monoplastic solution of phenol and guanidine isothiocyanate, using a Brinkmann Homogenizer Polytron PT-3000 (Brinkmann Instruments, Westbury, N.Y.). After a brief incubation on ice, chloroform was added (1:5 v/v) and the lysate was centrifuged. The upper chloroform layer was removed and the RNA was extracted with isopropanol, resuspended in water, and DNase treated for 25 min at 37° C. The RNA was re-extracted once with acid phenol-chloroform pH 4.7 and precipitated using 0.3M sodium acetate and 2.5 volumes ethanol. Poly(A+) RNA was isolated using the Qiagen Oligotex kit (QIAGEN, Inc., Chatsworth, Calif.). <span style="display: none">[0181] </span> </div>
</li> <li> <para-num num="[0182]"> </para-num> <div class="description-line" id="P-0182" num="0182">Poly(A+) RNA was used for cDNA synthesis and library construction according to the recommended protocols in the SuperScript™ plasmid system (Life Technologies, Inc.). cDNAs were fractionated on a Sepharose CL4B column (catalog #275105, Pharmacia) and those cDNAs exceeding 400 bp were ligated into the pINCY (Incyte Pharmaceuticals, Inc., Palo Alto, Calif.) cloning vector and subsequently transformed into DH5α™ competent cells (Cat. #18258-012, Life Technologies, Inc.). <span style="display: none">[0182] </span> </div>
</li> <li> <para-num num="[0183]"> </para-num> <div class="description-line" id="P-0183" num="0183">II. Isolation of cDNA Clones <span style="display: none">[0183] </span> </div>
</li> <li> <para-num num="[0184]"> </para-num> <div class="description-line" id="P-0184" num="0184">Plasmid DNA was released from the cells and purified using the REAL Prep 96 plasmid kit (Catalog #26173, QIAGEN, Inc.). The recommended protocol was employed except for the following changes: 1) the bacteria were cultured in 1 ml of sterile Terrific Broth (Catalog #22711, Life Technologies, Inc.) with carbenicillin at 25 mg/L and glycerol at 0.4%; 2) after the cultures were incubated for 19 hours, the cells were lysed with 0.3 ml of lysis buffer; and 3) following isopropanol precipitation, the plasmid DNA pellets were resuspended in 0.1 ml of distilled water. The DNA samples were stored at 4° C. <span style="display: none">[0184] </span> </div>
</li> <li> <para-num num="[0185]"> </para-num> <div class="description-line" id="P-0185" num="0185">III. Sequencing and Analysis <span style="display: none">[0185] </span> </div>
</li> <li> <para-num num="[0186]"> </para-num> <div class="description-line" id="P-0186" num="0186">The cDNAs were prepared for sequencing using either an ABI PRISM CATALYST 800 (Perkin-Elmer Applied Biosystems, Foster City, Calif.) or a MICROLAB 2200 (Hamilton Co., Reno, Nev.) sequencing preparation system in combination with Peltier PTC-200 thermal cyclers (MJ Research, Inc., Watertown, Mass.). The cDNAs were sequenced using the ABI PRISM 373 or 377 sequencing systems and ABI protocols, base calling software, and kits (Perkin-Elmer Applied Biosystems). Alternatively, solutions and dyes from Amersham Pharmacia Biotech, Ltd. were used in place of the ABI kits. In some cases, reading frames were determined using standard methods (Ausubel, supra). Some of the cDNA sequences were selected for extension using the techniques disclosed in Example V. <span style="display: none">[0186] </span> </div>
</li> <li> <para-num num="[0187]"> </para-num> <div class="description-line" id="P-0187" num="0187">The polynucleotide sequences derived from cDNA, extension, and shotgun sequencing were assembled and analyzed using a combination of software programs which utilize algorithms well known to those skilled in the art. Table 1 summarizes the software programs used, corresponding algorithms, references, and cutoff parameters used where applicable. The references cited in the third column of Table 1 are incorporated by reference herein. Sequence alignments were also analyzed and produced using MACDNASIS PRO software (Hitachi Software Engineering Co., Ltd. San Bruno, Calif.) and the multisequence alignment program of LASERGENE software (DNASTAR Inc, Madison Wis.). <span style="display: none">[0187] </span> </div>
</li> <li> <para-num num="[0188]"> </para-num> <div class="description-line" id="P-0188" num="0188">The polynucleotide sequences were validated by removing vector, linker, and polyA tail sequences and by masking ambiguous bases, using algorithms and programs based on BLAST, dynamic programing, and dinucleotide nearest neighbor analysis. The sequences were then queried against a selection of public databases such as GenBank primate, rodent, mammalian, vertebrate, and eukaryote databases, and BLOCKS to acquire annotation, using programs based on BLAST, FASTA, and BLIMPS. The sequences were assembled into full length polynucleotide sequences using programs based on Phred, Phrap, and Consed, and were screened for open reading frames using programs based on GeneMark, BLAST, and FASTA. This was followed by translation of the full length polynucleotide sequences to derive the corresponding full length amino acid sequences. These full length polynucleotide and amino acid sequences were subsequently analyzed by querying against databases such as the GenBank databases described above and SwissProt, BLOCKS, PRINTS, PFAM, and Prosite. <span style="display: none">[0188] </span> </div>
</li> <li> <para-num num="[0189]"> </para-num> <div class="description-line" id="P-0189" num="0189">IV. Northern Analysis <span style="display: none">[0189] </span> </div>
</li> <li> <para-num num="[0190]"> </para-num> <div class="description-line" id="P-0190" num="0190">Northern analysis is a laboratory technique used to detect the presence of a transcript of a gene and involves the hybridization of a labeled nucleotide sequence to a membrane on which RNAs from a particular cell type or tissue have been bound. (See, e.g., Sambrook, supra, ch. 7; and Ausubel, supra, ch. 4 and 16.) <span style="display: none">[0190] </span> </div>
</li> <li> <para-num num="[0191]"> </para-num> <div class="description-line" id="P-0191" num="0191">Analogous computer techniques applying BLAST are used to search for identical or related molecules in nucleotide databases such as GenBank or LIFESEQ™ database (Incyte Pharmaceuticals). This analysis is much faster than multiple membrane-based hybridizations. In addition, the sensitivity of the computer search can be modified to determine whether any particular match is categorized as exact or similar. <span style="display: none">[0191] </span> </div>
</li> <li> <para-num num="[0192]"> </para-num> <div class="description-line" id="P-0192" num="0192">The basis of the search is the product score, which is defined as:  
<span style="display: none">[0192] </span> <maths id="MATH-US-00001" num="1"> <math overflow="scroll"> <mfrac> <mrow> <mi>%</mi> <mo></mo> <mstyle> <mtext> </mtext> </mstyle> <mo></mo> <mi>sequence</mi> <mo></mo> <mstyle> <mtext> </mtext> </mstyle> <mo></mo> <mi>identity</mi> <mo>×</mo> <mi>%</mi> <mo></mo> <mstyle> <mtext> </mtext> </mstyle> <mo></mo> <mi>maximum</mi> <mo></mo> <mstyle> <mtext> </mtext> </mstyle> <mo></mo> <mi>BLAST</mi> <mo></mo> <mstyle> <mtext> </mtext> </mstyle> <mo></mo> <mi>score</mi> </mrow> <mn>100</mn> </mfrac> </math> <div class="patent-image"><a href="https://patentimages.storage.googleapis.com/69/88/68/bd4db3997699a7/US20030148934A1-20030807-M00001.png"><img alt="Figure US20030148934A1-20030807-M00001" class="patent-full-image" file="US20030148934A1-20030807-M00001.TIF" he="17.03835" height="68" id="EMI-M00001" img-content="mf" img-format="tif" src="https://patentimages.storage.googleapis.com/69/88/68/bd4db3997699a7/US20030148934A1-20030807-M00001.png" wi="216.027" width="864"/></a></div> <attachments> <attachment attachment-type="nb" file="US20030148934A1-20030807-M00001.NB" idref="MATHEMATICA-00001"> </attachment> </attachments> </maths>
</div>
</li> <li> <para-num num="[0193]"> </para-num> <div class="description-line" id="P-0193" num="0193">The product score takes into account both the degree of similarity between two sequences and the length of the sequence match. For example, with a product score of 40, the match will be exact within a 1% to 2% error, and, with a product score of 70, the match will be exact. Similar molecules are usually identified by selecting those which show product scores between 15 and 40, although lower scores may identify related molecules. <span style="display: none">[0193] </span> </div>
</li> <li> <para-num num="[0194]"> </para-num> <div class="description-line" id="P-0194" num="0194">The results of Northern analysis are reported as a list of libraries in which the transcript encoding GFMO occurs. Abundance and percent abundance are also reported. Abundance directly reflects the number of times a particular transcript is represented in a cDNA library, and percent abundance is abundance divided by the total number of sequences examined in the cDNA library. <span style="display: none">[0194] </span> </div>
</li> <li> <para-num num="[0195]"> </para-num> <div class="description-line" id="P-0195" num="0195">V. Extension of GFMO Encoding Polynucleotides <span style="display: none">[0195] </span> </div>
</li> <li> <para-num num="[0196]"> </para-num> <div class="description-line" id="P-0196" num="0196">Full length nucleic acid sequences of SEQ ID NO: 3 and SEQ ID NO: 4 were produced by extension of an appropriate fragment of the full length molecule, using oligonucleotide primers designed from this fragment. One primer was synthesized to initiate extension of an antisense polynucleotide, and the other was synthesized to initiate extension of a sense polynucleotide. Primers were used to facilitate the extension of the known sequence “outward” generating amplicons containing new unknown nucleotide sequence for the region of interest. The initial primers were designed from the cDNA using OLIGO™ 4.06 (National Biosciences, Plymouth, Minn.), or another appropriate program, to be about 22 to 30 nucleotides in length, to have a GC content of about 50% or more, and to anneal to the target sequence at temperatures of about 68° C. to about 72° C. Any stretch of nucleotides which would result in hairpin structures and primer-primer dimerizations was avoided. <span style="display: none">[0196] </span> </div>
</li> <li> <para-num num="[0197]"> </para-num> <div class="description-line" id="P-0197" num="0197">Selected human cDNA libraries (Life Technologies, Inc.) were used to extend the sequence. If more than one extension is necessary or desired, additional sets of primers are designed to further extend the known region. <span style="display: none">[0197] </span> </div>
</li> <li> <para-num num="[0198]"> </para-num> <div class="description-line" id="P-0198" num="0198">High fidelity amplification was obtained by following the instructions for the XL-PCR™ kit (The Perkin-Elmer Corp., Norwalk, Conn.) and thoroughly mixing the enzyme and reaction mix. PCR was performed using the PTC-200 thermal cycler (MJ Research, Inc., Watertown, Mass.), beginning with 40 pmol of each primer and the recommended concentrations of all other components of the kit, with the following parameters:  
<span style="display: none">[0198] </span> <tables id="TABLE-US-00001" num="1"> <patent-tables colsep="0" frame="none" rowsep="0"> <table align="left" class="description-table" cols="3" colsep="0" rowsep="0" width="100%"> <thead> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td align="center" class="description-td" colspan="3" nameend="2" namest="OFFSET" rowsep="1"> </td> </tr> </thead> <tbody><tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">Step 1</td> <td class="description-td">94° C. for 1 min (initial denaturation)</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">Step 2</td> <td class="description-td">65° C. for 1 min</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">Step 3</td> <td class="description-td">68° C. for 6 min</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">Step 4</td> <td class="description-td">94° C. for 15 sec</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">Step 5</td> <td class="description-td">65° C. for 1 min</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">Step 6</td> <td class="description-td">68° C. for 7 min</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">Step 7</td> <td class="description-td">Repeat steps 4 through 6 for an additional 15 cycles</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">Step 8</td> <td class="description-td">94° C. for 15 sec</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">Step 9</td> <td class="description-td">65° C. for 1 min</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">Step 10</td> <td class="description-td">68° C. for 7:15 min</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">Step 11</td> <td class="description-td">Repeat steps 8 through 10 for an additional 12 cycles</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">Step 12</td> <td class="description-td">72° C. for 8 min</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">Step 13</td> <td class="description-td">4° C. (and holding)</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td align="center" class="description-td" colspan="3" nameend="2" namest="OFFSET" rowsep="1"> </td> </tr> </tbody></table> </patent-tables> </tables>
</div>
</li> <li> <para-num num="[0199]"> </para-num> <div class="description-line" id="P-0199" num="0199">A 5 μl to 10 μl aliquot of the reaction mixture was analyzed by electrophoresis on a low concentration (about 0.6% to 0.8%) agarose mini-gel to determine which reactions were successful in extending the sequence. Bands thought to contain the largest products were excised from the gel, purified using QIAQUICK™ (QIAGEN Inc.), and trimmed of overhangs using Klenow enzyme to facilitate religation and cloning. <span style="display: none">[0199] </span> </div>
</li> <li> <para-num num="[0200]"> </para-num> <div class="description-line" id="P-0200" num="0200">After ethanol precipitation, the products were redissolved in 13 μl of ligation buffer, 1 μl T4-DNA ligase (15 units) and 1 μl T4 polynucleotide kinase were added, and the mixture was incubated at room temperature for 2 to 3 hours, or overnight at 16° C. Competent <span style="display: none">[0200] </span> <i>E. coli </i>cells (in 40 μl of appropriate med were transformed with 3 μl of ligation mixture and cultured in 80 μl of SOC medium. (See, e.g., Sambrook, supra, Appendix A, p. 2.) After incubation for one hour at 37° C., the <i>E. coli </i>mixture was plated on Luria Bertani (LB) agar (See, e.g., Sambrook, supra, Appendix A, p. 1) containing carbenicillin (2×carb). The following day, several colonies were randomly picked from each plate and cultured in 150 μl of liquid LB/2×carb medium placed in an individual well of an appropriate commercially-available sterile 96-well microtiter plate. The following day, 5 μl of each overnight culture was transferred into a non-sterile 96-well plate and, after dilution 1:10 with water, 5 μl from each sample was transferred into a PCR array. </div>
</li> <li> <para-num num="[0201]"> </para-num> <div class="description-line" id="P-0201" num="0201">For PCR amplification, 18 μl of concentrated PCR reaction mix (3.3×) containing 4 units of rTth DNA polymerase, a vector primer, and one or both of the gene specific primers used for the extension reaction were added to each well. Amplification was performed using the following conditions:  
<span style="display: none">[0201] </span> <tables id="TABLE-US-00002" num="2"> <patent-tables colsep="0" frame="none" rowsep="0"> <table align="left" class="description-table" cols="3" colsep="0" rowsep="0" width="100%"> <thead> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td align="center" class="description-td" colspan="3" nameend="2" namest="OFFSET" rowsep="1"> </td> </tr> </thead> <tbody><tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">Step 1</td> <td class="description-td">94° C. for 60 sec</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">Step 2</td> <td class="description-td">94° C. for 20 sec</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">Step 3</td> <td class="description-td">55° C. for 30 sec</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">Step 4</td> <td class="description-td">72° C. for 90 sec</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">Step 5</td> <td class="description-td">Repeat steps 2 through 4 for an additional 29 cycles</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">Step 6</td> <td class="description-td">72° C. for 180 sec</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">Step 7</td> <td class="description-td">4° C. (and holding)</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td align="center" class="description-td" colspan="3" nameend="2" namest="OFFSET" rowsep="1"> </td> </tr> </tbody></table> </patent-tables> </tables>
</div>
</li> <li> <para-num num="[0202]"> </para-num> <div class="description-line" id="P-0202" num="0202">Aliquots of the PCR reactions were run on agarose gels together with molecular weight markers. The sizes of the PCR products were compared to the original partial cDNAs, and appropriate clones were selected, ligated into plasmid, and sequenced. <span style="display: none">[0202] </span> </div>
</li> <li> <para-num num="[0203]"> </para-num> <div class="description-line" id="P-0203" num="0203">In like manner, the nucleotide sequences of SEQ ID NO: 3 and SEQ ID NO: 4 are used to obtain 5′ regulatory sequences using the procedure above, oligonucleotides designed for 5′ extension, and an appropriate genomic library. <span style="display: none">[0203] </span> </div>
</li> <li> <para-num num="[0204]"> </para-num> <div class="description-line" id="P-0204" num="0204">VI. Labeling and Use of Individual Hybridization Probes <span style="display: none">[0204] </span> </div>
</li> <li> <para-num num="[0205]"> </para-num> <div class="description-line" id="P-0205" num="0205">Hybridization probes derived from SEQ ID NO: 3 and SEQ ID NO: 4 are employed to screen cDNAs, genomic DNAS, or mRNAs. Although the labeling of oligonucleotides, consisting of about 20 base pairs, is specifically described, essentially the same procedure is used with larger nucleotide fragments. Oligonucleotides are designed using state-of-the-art software such as OLIGO™ 4.06 software (National Biosciences) and labeled by combining 50 pmol of each oligomer, 250 μCi of [γ-<span style="display: none">[0205] </span> <sup>32</sup>P] adenosine triphosphate (Amersham, Chicago, Ill.), and T4 polynucleotide kinase (DuPont NEN®, Boston, Mass.). The labeled oligonucleotides are substantially purified using a Sephadex™ G-25 superfine size exclusion dextran bead column (Pharmacia &amp; Upjohn, Kalamazoo, Mich.). An aliquot containing 10<sup>7 </sup>counts per minute of the labeled probe is used in a typical membrane-based hybridization analysis of human genomic DNA digested with one of the following endonucleases: Ase I, Bgl II, Eco RI, Pst I, Xbal, or Pvu II (DuPont NEN, Boston, Mass.). </div>
</li> <li> <para-num num="[0206]"> </para-num> <div class="description-line" id="P-0206" num="0206">The DNA from each digest is fractionated on a 0.7% agarose gel and transferred to nylon membranes (Nytran Plus, Schleicher &amp; Schuell, Durham, NH). Hybridization is carried out for 16 hours at 40° C. To remove nonspecific signals, blots are sequentially washed at room temperature under increasingly stringent conditions up to 0.1 ×saline sodium citrate and 0.5% sodium dodecyl sulfate. After XOMAT AR™ film (Kodak, Rochester, N.Y.) is exposed to the blots to film for several hours, hybridization patterns are compared visually. <span style="display: none">[0206] </span> </div>
</li> <li> <para-num num="[0207]"> </para-num> <div class="description-line" id="P-0207" num="0207">VII. Microarrays <span style="display: none">[0207] </span> </div>
</li> <li> <para-num num="[0208]"> </para-num> <div class="description-line" id="P-0208" num="0208">A chemical coupling procedure and an ink jet device can be used to synthesize array elements on the surface of a substrate. (See, e.g., Baldeschweiler, supra.) An array analogous to a dot or slot blot may also be used to arrange and link elements to the surface of a substrate using thermal, UV, chemical, or mechanical bonding procedures. A typical array may be produced by hand or using available methods and machines and contain any appropriate number of elements. After hybridization, nonhybridized probes are removed and a scanner used to determine the levels and patterns of fluorescence. The degree of complementarity and the relative abundance of each probe which hybridizes to an element on the microarray may be assessed through analysis of the scanned images. <span style="display: none">[0208] </span> </div>
</li> <li> <para-num num="[0209]"> </para-num> <div class="description-line" id="P-0209" num="0209">Full-length cDNAs, Expressed Sequence Tags (ESTs), or fragments thereof may comprise the elements of the microarray. Fragments suitable for hybridization can be selected using software well known in the art such as LASERGENE™. Full-length cDNAs, ESTs, or fragments thereof corresponding to one of the nucleotide sequences of the present invention, or selected at random from a cDNA library relevant to the present invention, are arranged on an appropriate substrate, e.g., a glass slide. The cDNA is fixed to the slide using, e.g., UV cross-linking followed by thermal and chemical treatments and subsequent drying. (See, e.g., Schena, M. et al. (1995) Science 270:467-470; and Shalon, D. et al. (1996) Genome Res. 6:639-645.) Fluorescent probes are prepared and used for hybridization to the elements on the substrate. The substrate is analyzed by procedures described above. <span style="display: none">[0209] </span> </div>
</li> <li> <para-num num="[0210]"> </para-num> <div class="description-line" id="P-0210" num="0210">VIII. Complementary Polynucleotides <span style="display: none">[0210] </span> </div>
</li> <li> <para-num num="[0211]"> </para-num> <div class="description-line" id="P-0211" num="0211">Sequences complementary to the GFMO-encoding sequences, or any parts thereof, are used to detect, decrease, or inhibit expression of naturally occurring GFMO. Although use of oligonucleotides comprising from about 15 to 30 base pairs is described, essentially the same procedure is used with smaller or with larger sequence fragments. Appropriate oligonucleotides are designed using OLIGO™ 4.06 software and the coding sequence of GFMO. To inhibit transcription, a complementary oligonucleotide is designed from the most unique 5′ sequence and used to prevent promoter binding to the coding sequence. To inhibit translation, a complementary oligonucleotide is designed to prevent ribosomal binding to the GFMO-encoding transcript. <span style="display: none">[0211] </span> </div>
</li> <li> <para-num num="[0212]"> </para-num> <div class="description-line" id="P-0212" num="0212">IX. Expression of GFMO <span style="display: none">[0212] </span> </div>
</li> <li> <para-num num="[0213]"> </para-num> <div class="description-line" id="P-0213" num="0213">Expression and purification of GFMO is achieved using bacterial or virus-based expression systems. For expression of GFMO in bacteria, cDNA is subcloned into an appropriate vector containing an antibiotic resistance gene and an inducible promoter that directs high levels of cDNA transcription. Examples of such promoters include, but are not limited to, the trp-lac (tac) hybrid promoter and the T5 or T7 bacteriophage promoter in conjunction with the lac operator regulatory element. Recombinant vectors are transformed into suitable bacterial hosts, e.g., BL21(DE3). Antibiotic resistant bacteria express GFMO upon induction with isopropyl beta-D-thiogalactopyranoside (IPTG). Expression of GFMO in eukaryotic cells is achieved by infecting insect or mammalian cell lines with recombinant <span style="display: none">[0213] </span> <i>Autographica californica </i>nuclear polyhedrosis virus (AcMNPV), commonly known as baculovirus. The nonessential polyhedrin gene of baculovirus is replaced with cDNA encoding GFMO by either homologous recombination or bacterial-mediated transposition involving transfer plasmid intermediates. Viral infectivity is maintained and the strong polyhedrin promoter drives high levels of cDNA transcription. Recombinant baculovirus is used to infect <i>Spodoptera frugiperda </i>(Sf9) insect cells in most cases, or human hepatocytes, in some cases. Infection of the latter requires additional genetic modifications to baculovirus. (See Engelhard, E. K. et al. (1994) Proc. Natl. Acad. Sci. USA 91:3224-3227; Sandig, V. et al. (1996) Hum. Gene Ther. 7:1937-1945.) </div>
</li> <li> <para-num num="[0214]"> </para-num> <div class="description-line" id="P-0214" num="0214">In most expression systems, GFMO is synthesized as a fusion protein with, e.g., glutathione S-transferase (GST) or a peptide epitope tag, such as FLAG or 6-His, permitting rapid, single-step, affinity-based purification of recombinant fusion protein from crude cell lysates. GST, a 26-kilodalton enzyme from <span style="display: none">[0214] </span> <i>Schistosoma japonicum</i>, enables the purification of fusion proteins on immobilized glutathione under conditions that maintain protein activity and antigenicity (Pharmacia, Piscataway, N.J.). Following purification, the GST moiety can be proteolytically cleaved from GFMO at specifically engineered sites. FLAG, an 8-amino acid peptide, enables immunoaffinity purification using commercially available monoclonal and polyclonal anti-FLAG antibodies (Eastman Kodak, Rochester, N.Y.). 6-His, a stretch of six consecutive histidine residues, enables purification on metal-chelate resins (QIAGEN Inc, Chatsworth, Calif.). Methods for protein expression and purification are discussed in Ausubel, F. M. et al. (1995 and periodic supplements) <i>Current Protocols in Molecular Biology</i>, John Wiley &amp; Sons, New York, N.Y., ch 10, 16. Purified GFMO obtained by these methods can be used directly in the following activity assay. </div>
</li> <li> <para-num num="[0215]"> </para-num> <div class="description-line" id="P-0215" num="0215">X. Demonstration of GFMO Activity <span style="display: none">[0215] </span> </div>
</li> <li> <para-num num="[0216]"> </para-num> <div class="description-line" id="P-0216" num="0216">The assay for GFMO function is based on there ability to modulate the mitogenic response of cells to growth factors. Mitogenic response is measured by [<span style="display: none">[0216] </span> <sup>3</sup>R]thymidine incorporation into newly synthesized DNA. (Kireeva, M. L. (1996) Mol. Cell. Biol. 16:1326-1334.) Tissue culture cells, such as NIH 3T3 or chicken embryo fibroblasts, are treated with growth factor (basic fibroblast growth factor or transforming growth factor β) and [<sup>3</sup>H]thymidine (1 μCi/ml) in DMEM, 0.2% FBS with or without GFMO. Cells are incubated for 18 h at 37° C., washed with phosphate buffered saline, and fixed with 10% trichloroacetic acid. DNA is dissolved in 0.1 N NaOH, and thymidine incorporation into DNA in GFMO-containing cultures relative to control cells is measured using a scintillation counter. </div>
</li> <li> <para-num num="[0217]"> </para-num> <div class="description-line" id="P-0217" num="0217">XI. Functional Assays <span style="display: none">[0217] </span> </div>
</li> <li> <para-num num="[0218]"> </para-num> <div class="description-line" id="P-0218" num="0218">GFMO function is assessed by expressing the sequences encoding GFMO at physiologically elevated levels in mammalian cell culture systems. cDNA is subcloned into a mammalian expression vector containing a strong promoter that drives high levels of cDNA expression. Vectors of choice include pCMV SPORT™ (Life Technologies, Gaithersburg, Md.) and pCR™ 3.1 (Invitrogen, Carlsbad, Calif., both of which contain the cytomegalovirus promoter. 5-10 μg of recombinant vector are transiently transfected into a human cell line, preferably of endothelial or hematopoietic origin, using either liposome formulations or electroporation. 1-2 μg of an additional plasmid containing sequences encoding a marker protein are co-transfected. Expression of a marker protein provides a means to distinguish transfected cells from nontransfected cells and is a reliable predictor of cDNA expression from the recombinant vector. Marker proteins of choice include, e.g., Green Fluorescent Protein (GFP) (Clontech, Palo Alto, Calif.), CD64, or a CD64-GFP fusion protein. Flow cytometry (FCM), an automated, laser optics-based technique, is used to identify transfected cells expressing GFP or CD64-GFP, and to evaluate properties, for example, their apoptotic state. FCM detects and quantifies the uptake of fluorescent molecules that diagnose events preceding or coincident with cell death. These events include changes in nuclear DNA content as measured by staining of DNA with propidium iodide; changes in cell size and granularity as measured by forward light scatter and 90 degree side light scatter; down-regulation of DNA synthesis as measured by decrease in bromodeoxyuridine uptake; alterations in expression of cell surface and intracellular proteins as measured by reactivity with specific antibodies; and alterations in plasma membrane composition as measured by the binding of fluorescein-conjugated Annexin V protein to the cell surface. Methods in flow cytometry are discussed in Ormerod, M. G. (1994) <span style="display: none">[0218] </span> <i>Flow Cytometry</i>, Oxford, New York, N.Y. </div>
</li> <li> <para-num num="[0219]"> </para-num> <div class="description-line" id="P-0219" num="0219">The influence of GFMO on gene expression can be assessed using highly purified populations of cells transfected with sequences encoding GFMO and either CD64 or CD64-GFP. CD64 and CD64-GFP are expressed on the surface of transfected cells and bind to conserved regions of human immunoglobulin G (IgG). Transfected cells are efficiently separated from nontransfected cells using magnetic beads coated with either human IgG or antibody against CD64 (DYNAL, Lake Success, N.Y.). mRNA can be purified from the cells using methods well known by those of skill in the art. Expression of mRNA encoding GFMO and other genes of interest can be analyzed by Northern analysis or microarray techniques. <span style="display: none">[0219] </span> </div>
</li> <li> <para-num num="[0220]"> </para-num> <div class="description-line" id="P-0220" num="0220">XII. Production of GFMO Specific Antibodies <span style="display: none">[0220] </span> </div>
</li> <li> <para-num num="[0221]"> </para-num> <div class="description-line" id="P-0221" num="0221">GFMO substantially purified using polyacrylamide gel electrophoresis (PAGE)(see, e.g., Harrington, M. G. (1990) Methods Enzymol. 182:488-495), or other purification techniques, is used to immunize rabbits and to produce antibodies using standard protocols. <span style="display: none">[0221] </span> </div>
</li> <li> <para-num num="[0222]"> </para-num> <div class="description-line" id="P-0222" num="0222">Alternatively, the GFMO amino acid sequence is analyzed using LASERGENE™ software (DNASTAR Inc.) to determine regions of high immunogenicity, and a corresponding oligopeptide is synthesized and used to raise antibodies by means known to those of skill in the art. Methods for selection of appropriate epitopes, such as those near the C-terminus or in hydrophilic regions are well described in the art. (See, e.g., Ausubel supra, ch. 11.) <span style="display: none">[0222] </span> </div>
</li> <li> <para-num num="[0223]"> </para-num> <div class="description-line" id="P-0223" num="0223">Typically, oligopeptides 15 residues in length are synthesized using an Applied Biosystems Peptide Synthesizer Model 431 A using fmoc-chemistry and coupled to KLH (Sigma, St. Louis, Mo.) by reaction with N-maleimidobenzoyl-N-hydroxysuccinimide ester (MBS) to increase immunogenicity. (See, e.g., Ausubel supra.) Rabbits are immunized with the oligopeptide-KLH complex in complete Freund's adjuvant. Resulting antisera are tested for antipeptide activity by, for example, binding the peptide to plastic, blocking with 1% BSA, reacting with rabbit antisera, washing,. and reacting with radio-iodinated goat anti-rabbit IgG. <span style="display: none">[0223] </span> </div>
</li> <li> <para-num num="[0224]"> </para-num> <div class="description-line" id="P-0224" num="0224">XIII. Purification of Naturally Occurring GFMO Using Specific Antibodies <span style="display: none">[0224] </span> </div>
</li> <li> <para-num num="[0225]"> </para-num> <div class="description-line" id="P-0225" num="0225">Naturally occurring or recombinant GFMO is substantially purified by immunoaffinity chromatography using antibodies specific for GFMO. An immunoaffinity column is constructed by covalently coupling anti-GFMO antibody to an activated chromatographic resin, such as CNBr-activated Sepharose (Pharmacia &amp; Upjohn). After the coupling, the resin is blocked and washed according to the manufacturer's instructions. <span style="display: none">[0225] </span> </div>
</li> <li> <para-num num="[0226]"> </para-num> <div class="description-line" id="P-0226" num="0226">Media containing GFMO are passed over the immunoaffinity column, and the column is washed under conditions that allow the preferential absorbance of GFMO (e.g., high ionic strength buffers in the presence of detergent). The column is eluted under conditions that disrupt antibody/GFMO binding (e.g., a buffer of pH 2 to pH 3, or a high concentration of a chaotrope, such as urea or thiocyanate ion), and GFMO is collected. <span style="display: none">[0226] </span> </div>
</li> <li> <para-num num="[0227]"> </para-num> <div class="description-line" id="P-0227" num="0227">XIV. Identification of Molecules Which Interact with GFMO <span style="display: none">[0227] </span> </div>
</li> <li> <para-num num="[0228]"> </para-num> <div class="description-line" id="P-0228" num="0228">GFMO, or biologically active fragments thereof, are labeled with 1251 Bolton-Hunter reagent. (See, e.g., Bolton et al. (1973) Biochem. J. 133:529.) Candidate molecules previously arrayed in the wells of a multi-well plate are incubated with the labeled GFMO, washed, and any wells with labeled GFMO complex are assayed. Data obtained using different concentrations of GFMO are used to calculate values for the number, affinity, and association of GFMO with the candidate molecules. <span style="display: none">[0228] </span> </div>
</li> <li> <para-num num="[0229]"> </para-num> <div class="description-line" id="P-0229" num="0229">Various modifications and variations of the described methods and systems of the invention will be apparent to those skilled in the art without departing from the scope and spirit of the invention. Although the invention has been described in connection with specific preferred embodiments, it should be understood that the invention as claimed should not be unduly limited to such specific embodiments. Indeed, various modifications of the described modes for carrying out the invention which are obvious to those skilled in molecular biology or related fields are intended to be within the scope of the following claims.  
<span style="display: none">[0229] </span> <tables id="TABLE-US-00003" num="3"> <patent-tables colsep="0" frame="none" pgwide="1" rowsep="0"> <table align="left" class="description-table" cols="4" colsep="0" rowsep="0" width="100%"> <thead> <tr class="description-tr"> <td align="center" class="description-td" colspan="4" nameend="4" namest="1">TABLE 1</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td align="center" class="description-td" colspan="4" nameend="4" namest="1" rowsep="1"> </td> </tr> <tr class="description-tr"> <td class="description-td">Program</td> <td class="description-td">Description</td> <td class="description-td">Reference</td> <td class="description-td">Parameter Threshold</td> </tr> <tr class="description-tr"> <td align="center" class="description-td" colspan="4" nameend="4" namest="1" rowsep="1"> </td> </tr> </thead> <tbody><tr class="description-tr"> <td class="description-td">ABI FACTURA</td> <td class="description-td">A program that removes vector sequences and</td> <td class="description-td">Perkin-Elmer Applied Biosystems,</td> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">masks ambiguous bases in nucleic acid</td> <td class="description-td">Foster City, CA.</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">sequences.</td> </tr> <tr class="description-tr"> <td class="description-td">ABI/PARACEL FDF</td> <td class="description-td">A Fast Data Finder useful in comparing and</td> <td class="description-td">Perkin-Elmer Applied Biosystems,</td> <td class="description-td">Mismatch &lt;50%</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">annotating amino acid or nucleic acid sequences.</td> <td class="description-td">Foster City, CA; Paracel Inc., Pasadena, CA.</td> </tr> <tr class="description-tr"> <td class="description-td">ABI AutoAssembler</td> <td class="description-td">A program that assembles nucleic acid</td> <td class="description-td">Perkin-Elmer Applied Biosystems,</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">sequences.</td> <td class="description-td">Foster City, CA.</td> </tr> <tr class="description-tr"> <td class="description-td">BLAST</td> <td class="description-td">A Basic Local Alignment Search Tool useful in</td> <td class="description-td">Altschul, S. F. et al. (1990) J. Mol. Biol.</td> <td class="description-td">ESTs: Probability</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">sequence similarity search for amino acid and</td> <td class="description-td">215: 403-410; Altschul, S. F. et al. (1997)</td> <td class="description-td">value = 1.0E−8 or less</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">nucleic acid sequences. BLAST includes five</td> <td class="description-td">Nucleic Acids Res. 25: 3389-3402.</td> <td class="description-td">Full Length sequences:</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">functions: blastp, blastn, blastx, tblastn, and</td> <td class="description-td"> </td> <td class="description-td">Probability value =</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">tblastx.</td> <td class="description-td"> </td> <td class="description-td">1.0E−10 or less</td> </tr> <tr class="description-tr"> <td class="description-td">FASTA</td> <td class="description-td">A Pearson and Lipman algorithm that searches</td> <td class="description-td">Pearson, W. R. and D. J. Lipman (1988) Proc.</td> <td class="description-td">ESTs: fasta E value =</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">for similarity between a query sequence and a</td> <td class="description-td">Natl. Acad Sci. 85: 2444-2448; Pearson, W. R.</td> <td class="description-td">1.06E−6 Assembled</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">group of sequences of the same type. FASTA</td> <td class="description-td">(1990) Methods Enzymol. 183: 63-98; and</td> <td class="description-td">ESTs: fasta Identity =</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">comprises as least five functions: fasta, tfasta,</td> <td class="description-td">Smith, T. F. and M. S. Waterman (1981) Adv.</td> <td class="description-td">95% or greater and</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">fastx, tfastx, and ssearch.</td> <td class="description-td">Appl. Math. 2: 482-489.</td> <td class="description-td">Match length = 200</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td">bases or greater; fastx</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td">E value = 1.0E−8 or</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td">less Full Length</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td">sequences: fastx </td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td">score = 100 or greater</td> </tr> <tr class="description-tr"> <td class="description-td">BLIMPS</td> <td class="description-td">A BLocks IMProved Searcher that matches a</td> <td class="description-td">Henikoff, S and J. G. Henikoff, Nucl. Acid Res.,</td> <td class="description-td">Score = 1000 or</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">sequence against those in BLOCKS and</td> <td class="description-td">19: 6565-72, 1991. J. G. Henikoff and S.</td> <td class="description-td">greater; Ratio of</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">PRINTS databases to search for gene families,</td> <td class="description-td">Henikoff (1996) Methods Enzymol. 266: 88-105;</td> <td class="description-td">Score/Strength = 0.75</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">sequence homology, and structural fingerprint</td> <td class="description-td">and Attwood, T. K. et al. (1997) J. Chem. Inf.</td> <td class="description-td">or larger; and</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">regions.</td> <td class="description-td">Comput. Sci. 37: 417-424.</td> <td class="description-td">Probability</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td">value = 1.0E−3 or less</td> </tr> <tr class="description-tr"> <td class="description-td">PFAM</td> <td class="description-td">A Hidden Markov Models-based application</td> <td class="description-td">Krogh, A. et al. (1994) J. Mol. Biol., 235: 1501-</td> <td class="description-td">Score = 10-50 bits,</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">useful for protein family search.</td> <td class="description-td">1531; Sonnhammer, E. L. L. et al. (1988)</td> <td class="description-td">depending on</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td">Nucleic Acids Res. 26: 320-322.</td> <td class="description-td">individual protein</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td">families</td> </tr> <tr class="description-tr"> <td class="description-td">GeneMark</td> <td class="description-td">A gene prediction algorithm that is bascd on</td> <td class="description-td">Borodovsky, M. and J. McIninch (1993)</td> <td class="description-td">Score = 0.4 or greater</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">inhomogeneous Markov chain models and is</td> <td class="description-td">Computers Chem. 17: 123-133; Blattner, F. R. et</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">useful for DNA sequence analysis and</td> <td class="description-td">al. (1993) Nucleic Acids Res. 21: 5408-5417.</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">particularly for gene prediction.</td> </tr> <tr class="description-tr"> <td class="description-td">ProfileScan</td> <td class="description-td">An algorithm that searches for structural and</td> <td class="description-td">Gribskov, M. et al. (1988) CABIOS 4: 61-66;</td> <td class="description-td">Score = 4.0 or greater</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">sequence motifs in protein sequences that match</td> <td class="description-td">Gribskov, et al. (1989) Methods Enzymol.</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">sequence patterns defined in Prosite.</td> <td class="description-td">183: 146-159; Bairoch, A. et al. (1997) Nucleic</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td">Acids Res. 25: 217-221.</td> </tr> <tr class="description-tr"> <td class="description-td">Phred</td> <td class="description-td">A base-calling algorithm that examines</td> <td class="description-td">Ewing, B. et al. (1998) Genome</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">automated sequencer traces with high sensitivity</td> <td class="description-td">Res. 8: 175-185; Ewing, B. and P.</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">and probability.</td> <td class="description-td">Green (1998) Genome Res. 8: 186-</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td">194.</td> </tr> <tr class="description-tr"> <td class="description-td">Phrap</td> <td class="description-td">A Phils Revised Assembly Program including</td> <td class="description-td">Smith, T. F. and M. S. Waterman (1981) Adv.</td> <td class="description-td">Score = 120 or greater;</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">SWAT and CrossMatch, programs based on</td> <td class="description-td">Appl. Math. 2: 482-489; Smith, T. F. and M. S.</td> <td class="description-td">Match length = 56</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">efficient implementation of the Smith-Waterman</td> <td class="description-td">Waterman (1981) J. Mol. Biol. 147: 195-197;</td> <td class="description-td">or greater</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">algorithm, useful in searching sequence</td> <td class="description-td">and Green, P., University of Washington,</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">homology and assembling DNA sequences.</td> <td class="description-td">Seattle, WA.</td> </tr> <tr class="description-tr"> <td class="description-td">Consed</td> <td class="description-td">A graphical tool for viewing and editing Phrap</td> <td class="description-td">Gordon, D. et al. (1998) Genome</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">assemblies</td> <td class="description-td">Res. 8: 195-202.</td> </tr> <tr class="description-tr"> <td class="description-td">SPScan</td> <td class="description-td">A weight matrix analysis program that scans</td> <td class="description-td">Nielson, H. et al. (1997) Protein Engineering</td> <td class="description-td">Score = 5 or greater</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">protein sequences for the presence of secretory</td> <td class="description-td">10: 1-6; Claverie, J. M. and S. Audic (1997)</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">signal peptides.</td> <td class="description-td">CABIOS 12: 431-439.</td> </tr> <tr class="description-tr"> <td class="description-td">Motifs</td> <td class="description-td">A program that searches amino acid sequences</td> <td class="description-td">Bairoch et al. supra; Wisconsin</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">for patterns that matched those defined in</td> <td class="description-td">Package Program Manual, version</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">Prosite.</td> <td class="description-td">9, page M51-59, Genetics Computer</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td">Group, Madison, WI.</td> </tr> <tr class="description-tr"> <td align="center" class="description-td" colspan="4" nameend="4" namest="1" rowsep="1"> </td> </tr> </tbody></table> </patent-tables> </tables>
</div>
</li> <li> <para-num num="[0230]"> </para-num> <div class="description-line" id="P-0230" num="0230"> <span style="display: none">[0230]</span> </div>
</li> <li> <div class="description-line">
<sequence-cwu id="SEQLST-1">
<number>1</number>
<sequence-list-old-rules>
<s-1>
<s-1-iii>
14 
</s-1-iii>
</s-1>
<s-2 seq-no="1 ">
<s-2-i>
<s-2-i-a>354 amino acids </s-2-i-a>
<s-2-i-b>amino acid </s-2-i-b>
<s-2-i-c>single </s-2-i-c>
<s-2-i-d>linear </s-2-i-d>
</s-2-i>
<s-2-vii>
<s-2-vii-a>CONUTUT01 </s-2-vii-a>
<s-2-vii-b>2509339 </s-2-vii-b>
</s-2-vii>
<s-2-xi> 1  

Met Gln Gly Leu Leu Phe Ser Thr Leu Leu Leu Ala Gly Leu Ala 
                  5                  10                  15 

Gln Phe Cys Cys Arg Val Gln Gly Thr Gly Pro Leu Asp Thr Thr 
                 20                  25                  30 

Pro Glu Gly Arg Pro Gly Glu Val Ser Asp Ala Pro Gln Arg Lys 
                 35                  40                  45 

Gln Phe Cys His Trp Pro Cys Lys Cys Pro Gln Gln Lys Pro Arg 
                 50                  55                  60 

Cys Pro Pro Gly Val Ser Leu Val Arg Asp Gly Cys Gly Cys Cys 
                 65                  70                  75 

Lys Ile Cys Ala Lys Gln Pro Gly Glu Ile Cys Asn Glu Ala Asp 
                 80                  85                  90 

Leu Cys Asp Pro His Lys Gly Leu Tyr Cys Asp Tyr Ser Val Asp 
                 95                 100                 105 

Arg Pro Arg Tyr Glu Thr Gly Val Cys Ala Tyr Leu Val Ala Val 
                110                 115                 120 

Gly Cys Glu Phe Asn Gln Val His Tyr His Asn Gly Gln Val Phe 
                125                 130                 135 

Gln Pro Asn Pro Leu Phe Ser Cys Leu Cys Val Ser Gly Ala Ile 
                140                 145                 150 

Gly Cys Thr Pro Leu Phe Ile Pro Lys Leu Ala Gly Ser His Cys 
                155                 160                 165 

Ser Gly Ala Lys Gly Gly Lys Lys Ser Asp Gln Ser Asn Cys Ser 
                170                 175                 180 

Leu Glu Pro Leu Leu Gln Gln Leu Ser Thr Ser Tyr Lys Thr Met 
                185                 190                 195 

Pro Ala Tyr Arg Asn Leu Pro Leu Ile Trp Lys Lys Lys Cys Leu 
                200                 205                 210 

Val Gln Ala Thr Lys Trp Thr Pro Cys Ser Arg Thr Cys Gly Met 
                215                 220                 225 

Gly Ile Ser Asn Arg Val Thr Asn Glu Asn Ser Asn Cys Glu Met 
                230                 235                 240 

Arg Lys Glu Lys Arg Leu Cys Tyr Ile Gln Pro Cys Asp Ser Asn 
                245                 250                 255 

Ile Leu Lys Thr Ile Lys Ile Pro Lys Gly Lys Thr Cys Gln Pro 
                260                 265                 270 

Thr Phe Gln Leu Ser Lys Ala Glu Lys Phe Val Phe Ser Gly Cys 
                275                 280                 285 

Ser Ser Thr Gln Ser Tyr Lys Pro Thr Phe Cys Gly Ile Cys Leu 
                290                 295                 300 

Asp Lys Arg Cys Cys Ile Pro Asn Lys Ser Lys Met Ile Thr Ile 
                305                 310                 315 

Gln Phe Asp Cys Pro Asn Glu Gly Ser Phe Lys Trp Lys Met Leu 
                320                 325                 330 

Trp Ile Thr Ser Cys Val Cys Gln Arg Asn Cys Arg Glu Pro Gly 
                335                 340                 345 

Asp Ile Phe Ser Glu Leu Lys Ile Leu 
                350 


</s-2-xi>
</s-2>
<s-2 seq-no="2 ">
<s-2-i>
<s-2-i-a>223 amino acids </s-2-i-a>
<s-2-i-b>amino acid </s-2-i-b>
<s-2-i-c>single </s-2-i-c>
<s-2-i-d>linear </s-2-i-d>
</s-2-i>
<s-2-vii>
<s-2-vii-a>DRGLNOT01 </s-2-vii-a>
<s-2-vii-b>2840746 </s-2-vii-b>
</s-2-vii>
<s-2-xi> 2  

Met Lys Phe Val Pro Cys Leu Leu Leu Val Thr Leu Ser Cys Leu 
                  5                  10                  15 

Gly Thr Leu Gly Gln Ala Pro Arg Gln Lys Gln Gly Asn Thr Gly 
                 20                  25                  30 

Glu Glu Phe His Phe Gln Thr Gly Gly Arg Asp Ser Cys Thr Met 
                 35                  40                  45 

Arg Pro Ser Ser Leu Gly Gln Gly Ala Gly Glu Val Trp Leu Arg 
                 50                  55                  60 

Val Asp Cys Arg Asn Thr Asp Gln Thr Tyr Trp Cys Glu Tyr Arg 
                 65                  70                  75 

Gly Gln Pro Ser Met Cys Gln Ala Phe Ala Ala Asp Pro Lys Pro 
                 80                  85                  90 

Tyr Trp Asn Gln Ala Leu Gln Glu Leu Arg Arg Leu His His Ala 
                 95                 100                 105 

Cys Gln Gly Ala Pro Val Leu Arg Pro Ser Val Cys Arg Glu Ala 
                110                 115                 120 

Gly Pro Gln Ala His Met Gln Gln Val Thr Ser Ser Leu Lys Gly 
                125                 130                 135 

Ser Pro Glu Pro Asn Gln Gln Pro Glu Ala Gly Thr Pro Ser Leu 
                140                 145                 150 

Arg Pro Lys Ala Thr Val Lys Leu Thr Glu Ala Thr Gln Leu Gly 
                155                 160                 165 

Lys Asp Ser Met Glu Glu Leu Gly Lys Ala Lys Pro Thr Thr Arg 
                170                 175                 180 

Pro Thr Ala Lys Pro Thr Gln Pro Gly Pro Arg Pro Gly Gly Asn 
                185                 190                 195 

Glu Glu Ala Lys Lys Lys Ala Trp Glu His Cys Trp Lys Pro Phe 
                200                 205                 210 

Gln Ala Leu Cys Ala Phe Leu Ile Ser Phe Phe Arg Gly 
                215                 220 


</s-2-xi>
</s-2>
<s-2 seq-no="3 ">
<s-2-i>
<s-2-i-a>1183 base pairs </s-2-i-a>
<s-2-i-b>nucleic acid </s-2-i-b>
<s-2-i-c>single </s-2-i-c>
<s-2-i-d>linear </s-2-i-d>
</s-2-i>
<s-2-vii>
<s-2-vii-a>CONUTUT01 </s-2-vii-a>
<s-2-vii-b>2509339 </s-2-vii-b>
</s-2-vii>
<s-2-xi> 3  

TCTACCCCTC AGGGTGGCTC CACGGTCCCA GCGACATGCA GGGGCTCCTC TTCTCCACTC     60 

TTCTGCTTGC TGGCCTGGCA CAGTTCTGCT GCAGGGTACA GGGCACTGGA CCATTAGATA    120 

CAACACCTGA AGGAAGGCCT GGAGAAGTGT CAGATGCACC TCAGCGTAAA CAGTTTTGTC    180 

ACTGGCCCTG CAAATGCCCT CAGCAGAAGC CCCGTTGCCC TCCTGGAGTG AGCCTGGTGA    240 

GAGATGGCTG TGGATGCTGT AAAATCTGTG CCAAGCAACC AGGGGAAATC TGCAATGAAG    300 

CTGACCTCTG TGACCCACAC AAAGGGCTGT ATTGTGACTA CTCAGTAGAC AGGCCTAGGT    360 

ACGAGACTGG AGTGTGTGCA TACCTTGTAG CTGTTGGGTG CGAGTTCAAC CAGGTACATT    420 

ATCATAATGG CCAAGTGTTT CAGCCCAACC CCTTGTTCAG CTGCCTCTGT GTGAGTGGGG    480 

CCATTGGATG CACACCTCTG TTCATACCAA AGCTGGCTGG CAGTCACTGC TCTGGAGCTA    540 

AAGGTGGAAA GAAGTCTGAT CAGTCAAACT GTAGCCTGGA ACCATTACTA CAGCAGCTTT    600 

CAACAAGCTA CAAAACAATG CCAGCTTATA GAAATCTCCC ACTTATTTGG AAAAAAAAAT    660 

GTCTTGTGCA AGCAACAAAA TGGACTCCCT GCTCCAGAAC ATGTGGGATG GGAATATCTA    720 

ACAGGGTGAC CAATGAAAAC AGCAACTGTG AAATGAGAAA AGAGAAAAGA CTGTGTTACA    780 

TTCAGCCTTG CGACAGCAAT ATATTAAAGA CAATAAAGAT TCCCAAAGGA AAAACATGCC    840 

AACCTACTTT CCAACTCTCC AAAGCTGAAA AATTTGTCTT TTCTGGATGC TCAAGTACTC    900 

AGAGTTACAA ACCCACTTTT TGTGGAATAT GCTTGGATAA GAGATGCTGT ATCCCTAATA    960 

AGTCTAAAAT GATTACTATT CAATTTGATT GCCCAAATGA GGGGTCATTT AAATGGAAGA   1020 

TGCTGTGGAT TACATCTTGT GTGTGTCAGA GAAACTGCAG AGAACCTGGA GATATATTTT   1080 

CTGAGCTCAA GATTCTGTAA AACCAAGCAA ATGGGGGAAA AGTTAGTCAA TCCTGTCATA   1140 

TAATTAAAAA ATTAGTGAGT TTAAAAAAAA AAAAAAAAAG GGG                     1183 


</s-2-xi>
</s-2>
<s-2 seq-no="4 ">
<s-2-i>
<s-2-i-a>1095 base pairs </s-2-i-a>
<s-2-i-b>nucleic acid </s-2-i-b>
<s-2-i-c>single </s-2-i-c>
<s-2-i-d>linear </s-2-i-d>
</s-2-i>
<s-2-vii>
<s-2-vii-a>DRGLNOT01 </s-2-vii-a>
<s-2-vii-b>2840746 </s-2-vii-b>
</s-2-vii>
<s-2-xi> 4  


TTGCAAGCAA GTTTATCGGA GTATCGCCAT GAAGTTCGTC CCCTGCCTCC TGCTGGTGAC     60 

CTTGTCCTGC CTGGGGACTT TGGGTCAGGC CCCGAGGCAA AAGCAAGGAA ACACTGGGGA    120 

GGAATTCCAT TTCCAGACTG GAGGGAGAGA TTCCTGCACT ATGCGTCCCA GCAGCTTGGG    180 

GCAAGGTGCT GGAGAAGTCT GGCTTCGCGT CGACTGCCGC AACACAGACC AGACCTACTG    240 

GTGTGAGTAC AGGGGGCAGC CCAGCATGTG CCAGGCTTTT GCTGCTGACC CCAAACCTTA    300 

CTGGAATCAA GCCCTGCAGG AGCTGAGGCG CCTTCACCAT GCGTGCCAGG GGGCCCCGGT    360 

GCTTAGGCCA TCCGTGTGCA GGGAGGCTGG ACCCCAGGCC CATATGCAGC AGGTGACTTC    420 

CAGCCTCAAG GGCAGCCCAG AGCCCAACCA GCAGCCTGAG GCTGGGACGC CATCTCTGAG    480 

GCCCAAGGCC ACAGTGAAAC TCACAGAAGC AACACAGCTG GGAAAGGACT CGATGGAAGA    540 

GCTGGGAAAA GCCAAACCCA CCACCCGACC CACAGCCAAA CCTACCCAGC CTGGACCCAG    600 

GCCCGGAGGG AATGAGGAAG CAAAGAAGAA GGCCTGGGAA CATTGTTGGA AACCCTTCCA    660 

GGCCCTGTGC GCCTTTCTCA TCAGCTTCTT CCGAGGGTGA CAGGTGAAAG ACCCCTACAG    720 

ATCTGACCTC TCCCTGACAG ACAACCATCT CTTTTTATAT TATGCCGCTT TCAATCCAAC    780 

GTTCTCACAC TGGAAGAAGA GAGTTTCTAA TCAGATGCAA CGGCCCAAAT TCTTGATCTG    840 

CAGCTTCTCT GAAGTTTGGA AAAGAAACCT TCCTTTCTGG AGTTTGCAGA GTTCAGCAAT    900 

ATGATAGGGA ACAGGTGCTG ATGGGCCCAA GAGTGACAAG CATACACAAC TACTTATTAT    960 

CTGTAGAAGT TTTGCTTTGT TGATCTGAGC CTTCTATGAA AGTTTAAATA TGTAACGCAT   1020 

TCATGAATTT CCAGTGTTCA GTAAATAGCA GCTATGTGTG TGCAAAATAA AAGAATGATT   1080 

TCAGAAAAAA AAAAA                                                    1095 


</s-2-xi>
</s-2>
<s-2 seq-no="5 ">
<s-2-i>
<s-2-i-a>224 base pairs </s-2-i-a>
<s-2-i-b>nucleic acid </s-2-i-b>
<s-2-i-c>single </s-2-i-c>
<s-2-i-d>linear </s-2-i-d>
</s-2-i>
<s-2-vii>
<s-2-vii-a>CONUTUT01 </s-2-vii-a>
<s-2-vii-b>2509339H1 </s-2-vii-b>
</s-2-vii>
<s-2-xi> 5  


TCTACCCCTC AGGGTGGCTC CACGGTCCCA GCGACATGCA GGGGCTCCTC TTCTCCACTC     60 

TTCTGCTTGC TGGCCTGGCA CAGTTCTGCT GCAGGGTACA GGGCACTGGA CCATTAGATA    120 

CAACACCTGA AGGAAGGCCT GGAGAAGTGT CAGATGNACC TCAGCGTAAA CAGTTTTGTC    180 

ACTGGCCCTG CAAATGCCCT CAGCAGAAGC CCCGTTGNCC TCCT                     224 


</s-2-xi>
</s-2>
<s-2 seq-no="6 ">
<s-2-i>
<s-2-i-a>531 base pairs </s-2-i-a>
<s-2-i-b>nucleic acid </s-2-i-b>
<s-2-i-c>single </s-2-i-c>
<s-2-i-d>linear </s-2-i-d>
</s-2-i>
<s-2-vii>
<s-2-vii-a>CONUTUT01 </s-2-vii-a>
<s-2-vii-b>2509339F6 </s-2-vii-b>
</s-2-vii>
<s-2-xi> 6  


TCTACCCCTC ANGGTGGCTC CACGGTCCCA GCGACATGCA GGGGCTCCTC TTCTCCACTC     60 

TTCTGCTTGC TGGCCTGGNA CAGTTCTGCT GCAGGGTACA GGGCACTGGA CCATTAGATA    120 

CAACACCTGA AGGAAGGCCT GGAGAAGTGT CAGATGCACC TCAGCGTAAA CAGTTTTGTC    180 

ACTGGCCCTG CAAATGCCCT CAGCAGAAGC CCCGTTGCCC TCCTGGAGTG AGCCTGGTGA    240 

GAGATGGCTG TGGATGCTGT AAAATCTGTG CCAAGCAACC AGGGGAAATC TGCAATGAAG    300 

CTGACCTCTG TGACCCACAC AAAGGGCTGT ATTGTGACTA CTCAGTAGAC AGGCCTAGGT    360 

ACGAGACTGG AGTGTGTGCA TACTTGTAGC TGTNGGGTGC GATTCAACCA GGTACATTAT    420 

CATAATGGCC AAGTGTTTCA GCCCAAACCC CTTGTTCAGT GCCTCTGTGT GAGTTGGGGC    480 

CATTGGATGC ACACTTCTGT TCATACAAAG CTGGCTTGGC AGTCACTGTT T             531 


</s-2-xi>
</s-2>
<s-2 seq-no="7 ">
<s-2-i>
<s-2-i-a>637 base pairs </s-2-i-a>
<s-2-i-b>nucleic acid </s-2-i-b>
<s-2-i-c>single </s-2-i-c>
<s-2-i-d>linear </s-2-i-d>
</s-2-i>
<s-2-vii>
<s-2-vii-a>N/A </s-2-vii-a>
<s-2-vii-b>SBCA01417F1 </s-2-vii-b>
</s-2-vii>
<s-2-xi> 7  


AGGTCGACTC TAGAGGATCC CCCCATTGGA TGCACACCTC TGTTCATACC AAAGCTGGCT     60 

GGCAGTCACT GCTCTGGAGC TAAAGGTGGA AAGAAGTCTG ATCAGTCAAA CTGTAGCCTG    120 

GAACCATTAC TACAGCAGCT TTCAACAAGC TACAAAACAA TGCCAGCTTA TAGAAATCTC    180 

CCACTTATTT GGAAAAAAAA ATGTCTTGTG CAAGCAACAA AATGGACTCC CTGCTCCAGA    240 

ACATGTGGGA TGGGAATATC TAACAGGGTG ACCAATGAAA ACAGCAACTG TGAAATGAGA    300 

AAAGAGAAAA GACTGTGTTA CATTCAGCCT TGCGACANCA TATATTAAAG ACAATAAAGG    360 

NTCCCCAAGG NAAAACATGC CAACCTACTT TCCAACTCTC CAAAGTGAAA AATTTGGCCT    420 

TTCCGGATGC NCCAGTACCN CAGAGTTTCA AANCCCCCTT TTGGTGGATA TGCTTGGANA    480 

AAGAGATGCC GGTTCCCCNN AATTAAGGGC TTAAAATTGG GTTTACNCCA ATTCCAAATT    540 

TGGGATTGGN CCCNAAAATT GGAAGGGGGN NCNATTTTAA AATTGGGGNA AAATTGCCNC    600 

GTTGGGGANT TAAACAANCC TTGGGGGGGG GGGTCNA                             637 


</s-2-xi>
</s-2>
<s-2 seq-no="8 ">
<s-2-i>
<s-2-i-a>719 base pairs </s-2-i-a>
<s-2-i-b>nucleic acid </s-2-i-b>
<s-2-i-c>single </s-2-i-c>
<s-2-i-d>linear </s-2-i-d>
</s-2-i>
<s-2-vii>
<s-2-vii-a>N/A </s-2-vii-a>
<s-2-vii-b>SBCA02999F1 </s-2-vii-b>
</s-2-vii>
<s-2-xi> 8  

GGTCGACTCT AGAGGATCCC CCCTGGAACC ATTACTACAG CAGCTTTCAA CAAGCTACAA     60 

AACAATGCCA GCTTATAGAA ATCTCCCACT TATTTGGAAA AAAAAATGTC TTGTGCAAGC    120 

AACAAAATGG ACTCCCTGCT CCAGAACATG TGGGATGGGA ATATCTAACA GGGTGACCAA    180 

TGAAAACAGC AACTGTGAAA TGAGAAAAGA GAAAAGACTG TGTTACATTC AGCCTTGCGA    240 

CANAATATAT TAAAGACAAT AAAGATTCCC AAAGGAAAAA CATGCCAACC TACTTTCCAA    300 

CTCTCCAAAG CTGAAAAATT TGTCTTTTCT GGATGCTCAA GTACTCAGAG TTACAAACCC    360 

ACTTTTTGTG GAATATGCTT GGATAAGAGA TGCTGTATCC CTAATAAAGT CTATATGGAT    420 

TACTATTCCA ATTTGANTTG CCCNAAATTG ANGGGGGCCA TTTAAAATTG GGAANGATTG    480 

CCNGGTTGGG AATTTACAAT CCCTTGGGGG GTNGGTTGTT TCAGGGNGGG AAAAACTTGG    540 

CCCNGGAAGG AAANCCCTTN GGGGGGGNTN AANAAATTTT TTNCCNTGAA AGGGGGGGTT    600 

ACCCCGGAGG GCCTNCNGNA AATTTCCGGG TNAAATNCCA ANGGGTCAAA ANAAGNCCGG    660 

GTTTTCCCCC GGGNGGTNNA AAAAATTTGG GTNAATTCCC GGGGNTCCAA NNAAATTTT     719 


</s-2-xi>
</s-2>
<s-2 seq-no="9 ">
<s-2-i>
<s-2-i-a>255 base pairs </s-2-i-a>
<s-2-i-b>nucleic acid </s-2-i-b>
<s-2-i-c>single </s-2-i-c>
<s-2-i-d>linear </s-2-i-d>
</s-2-i>
<s-2-vii>
<s-2-vii-a>DRGLNOT01 </s-2-vii-a>
<s-2-vii-b>2840746H1 </s-2-vii-b>
</s-2-vii>
<s-2-xi> 9  

ANGTCTGGCT TCGCGTCGAC TGCCGCAACA CAGACCAGAC CTACTGGTGT GAGTACAGGG     60 

GGCAGCCCAG CATGTGCCAG GCTTTCGCTG CTGACCCCAA ATCTTACTGG AATCAAGCCC    120 

TGCAGGAGCT GAGGCGCCTT CACCATGCGT GCCAGGGGGC CCCGGTGCTT AGGCCATCCG    180 

TGTGCAGGGA GGCTNGACCC CAGGCCCATA TGCAGCAGGT GACTTCCAGC CTCAAGGGCA    240 

GCCCAGAGCC CAACC                                                     255 


</s-2-xi>
</s-2>
<s-2 seq-no="10 ">
<s-2-i>
<s-2-i-a>572 base pairs </s-2-i-a>
<s-2-i-b>nucleic acid </s-2-i-b>
<s-2-i-c>single </s-2-i-c>
<s-2-i-d>linear </s-2-i-d>
</s-2-i>
<s-2-vii>
<s-2-vii-a>BRAITUT03 </s-2-vii-a>
<s-2-vii-b>861509R6 </s-2-vii-b>
</s-2-vii>
<s-2-xi> 10  

CTTGCAGAGA AAGAGTCTTT TGTGCAGCAC CCTTTAAAGG GTGACTCGTC CCACTTGTGT     60 

TCTCTCTCCT GGNGCAGAGT TGCAAGCAAG TTTATCAGAG TATCGCCATG AAGTTCGTCC    120 

CCTGCCTCCN GCNGGNGACC TTGTCCTGCC TGGGGACTTT GGGTCAGGCC CCGANGCAAA    180 

ANCAAGGAAG CACTGGGGAG GAATTCCATT TCCAGACTGG AGGGAGAGNT TCCTGCACTA    240 

TGCGTCCCAG CAGCTTGGGG CAAGGTGCTG GAGAAGTCTG GCTTCGCGTC GACTGCCGCA    300 

ACACAGACCA GACCTACTGG TGTGAGTACA GGGGGCAGCC CAGCATGTGC CAGGCTTTTG    360 

CTGCTGACCC CAAACCTTAC TGGAATCAAG CCCTGCAGGA GCTGANGGCG CTTCANCAAT    420 

GCGTGCCANG NGGGCCCCCG GTGCNTAAGG CCATTCCGNT GTGCAAGGGA AGGCTTGGAA    480 

CCCCAAGGGN CCCATAATTG CAAGGCAAGG TTGGAANNTT CCAGGGNCTT CAAAAGGGGC    540 

AATTNCCCCA GAAAGCCCCC AAACCCAAGN AA                                  572 


</s-2-xi>
</s-2>
<s-2 seq-no="11 ">
<s-2-i>
<s-2-i-a>637 base pairs </s-2-i-a>
<s-2-i-b>nucleic acid </s-2-i-b>
<s-2-i-c>single </s-2-i-c>
<s-2-i-d>linear </s-2-i-d>
</s-2-i>
<s-2-vii>
<s-2-vii-a>DRGLNOT01 </s-2-vii-a>
<s-2-vii-b>2843688T6 </s-2-vii-b>
</s-2-vii>
<s-2-xi> 11  

CTGAAATCAT TCTTTTATTT TGCACACACA TAGCTGCTAT TTACTGAACA CTGGAAATTC     60 

ATGAATGCGT TACATATTTA AACTTTCATA GAAGGCTCAG ATCAACAAAG CAAAACTTCT    120 

ACAGATAATA AGTAGTTGTG TATGCTTGTC ACTCTTGGGC CCATCAGCAC CTGTTCCCTA    180 

TCATATTGCT GAACTCTGCA AACTCCAGAA AGGAAGGTTT CTTTTCCAAA CTTCAGAGAA    240 

GCTGCAGATC AAGAATTTGG GCCGTTGCAT CTGATTAGAA ACTCTCTTCT TCCAGTGTGA    300 

GAACGTTGGA TTGAAAGCGG CATAATATAA AAAGAGATGG TTGTCTGTCA GGGAGAGGTC    360 

AGATCTGTAG GGGTCTTTCA CCTGTCACCC TCGGAAGAAG CTGATGAGAA AGGCGCACAG    420 

GGCCTGGAAG GGTTTCCAAC AATGTTCCCA GGCCTTCTTC TTTGCTTCCT CATTCCCTCC    480 

GGGCCTGGGT CCAGGCTGGG TAGGTTTGGC TGTGGGTCGG GTGGTGGGTT TGGCTTTTCC    540 

CAGCTCTTCC ATCGAGTCCT TTCCCAGCTG TGTTGCTTCT GTGAGTTTCA CTGTGGCCTT    600 

GGGCCTCAGA GATGGGCGTC CCAGCCTCAG GCTGCTG                             637 


</s-2-xi>
</s-2>
<s-2 seq-no="12 ">
<s-2-i>
<s-2-i-a>437 base pairs </s-2-i-a>
<s-2-i-b>nucleic acid </s-2-i-b>
<s-2-i-c>single </s-2-i-c>
<s-2-i-d>linear </s-2-i-d>
</s-2-i>
<s-2-vii>
<s-2-vii-a>BRAITUT03 </s-2-vii-a>
<s-2-vii-b>866176R1 </s-2-vii-b>
</s-2-vii>
<s-2-xi> 12  

GTGCGCCTTT CTCATCAGCT TCTTCCGAGG GTGACAGGTG AAAGACCCCT ACAGATCTGA     60 

CCTCTCCCTG ACAGACAACC ATCTCTTTTT ATATTATGCC GCTTTCAATC CAACGTTCTC    120 

ACACTGGAAG AAGAGAGTTT CTAATCAGAT GCAACGGCCC AAATTCTTGA TCTGCAGCTT    180 

CTCTGAAGTT TGGAAAAGAA ACCTTCCTTT CTGGAGTTTG CAGAGTTCAG CAATATGATA    240 

GGGAACAGGT GCTAATGGGC CCAAGAGTGA CAAGCATACA CAACTACTTA TTAACNGGTA    300 

GNAGGTTTNG CCTTTGGTGA TTCTTGAGCC TTCCTATNGA AAAGNTTAAA ATATGTAAAC    360 

GCATTCNAGG AATTTCCCNA GGGTTCAGGT AAATAGCAGC NANGGTGNGN NCAAAATAAA    420 

NGAATGATTC CCGNAAA                                                   437 


</s-2-xi>
</s-2>
<s-2 seq-no="13 ">
<s-2-i>
<s-2-i-a>367 amino acids </s-2-i-a>
<s-2-i-b>amino acid </s-2-i-b>
<s-2-i-c>single </s-2-i-c>
<s-2-i-d>linear </s-2-i-d>
</s-2-i>
<s-2-vii>
<s-2-vii-a>GenBank </s-2-vii-a>
<s-2-vii-b>g2911144 </s-2-vii-b>
</s-2-vii>
<s-2-xi> 13  

Met Arg Trp Leu Leu Pro Trp Thr Leu Ala Ala Val Ala Val Leu 
                  5                  10                  15 

Arg Val Gly Asn Ile Leu Ala Thr Ala Leu Ser Pro Thr Pro Thr 
                 20                  25                  30 

Thr Met Thr Phe Thr Pro Ala Pro Leu Glu Glu Thr Thr Thr Arg 
                 35                  40                  45 

Pro Glu Phe Cys Lys Trp Pro Cys Glu Cys Pro Gln Ser Pro Pro 
                 50                  55                  60 

Arg Cys Pro Leu Gly Val Ser Leu Ile Thr Asp Gly Cys Glu Cys 
                 65                  70                  75 

Cys Lys Ile Cys Ala Gln Gln Leu Gly Asp Asn Cys Thr Glu Ala 
                 80                  85                  90 

Ala Ile Cys Asp Pro His Arg Gly Leu Tyr Cys Asp Tyr Ser Gly 
                 95                 100                 105 

Asp Arg Pro Arg Tyr Ala Ile Gly Val Cys Ala Gln Val Val Gly 
                110                 115                 120 

Val Gly Cys Val Leu Asp Gly Val Arg Tyr Thr Asn Gly Glu Ser 
                125                 130                 135 

Phe Gln Pro Asn Cys Arg Tyr Asn Cys Thr Cys Ile Asp Gly Thr 
                140                 145                 150 

Val Gly Cys Thr Pro Leu Cys Leu Ser Pro Arg Pro Pro Arg Leu 
                155                 160                 165 

Trp Cys Arg Gln Pro Arg His Val Arg Val Pro Gly Gln Cys Cys 
                170                 175                 180 

Glu Gln Trp Val Cys Asp Asp Asp Ala Arg Arg Pro Arg Gln Thr 
                185                 190                 195 

Ala Leu Leu Asp Thr Arg Ala Phe Ala Ala Ser Gly Ala Val Glu 
                200                 205                 210 

Gln Arg Tyr Glu Asn Cys Ile Ala Tyr Thr Ser Pro Trp Ser Pro 
                215                 220                 225 

Cys Ser Thr Thr Cys Gly Leu Gly Ile Ser Thr Arg Ile Ser Asn 
                230                 235                 240 

Val Asn Ala Arg Cys Trp Pro Glu Gln Glu Ser Arg Leu Cys Asn 
                245                 250                 255 

Leu Arg Pro Cys Asp Val Asp Ile Gln Leu His Ile Lys Ala Gly 
                260                 265                 270 

Lys Lys Cys Leu Ala Val Tyr Gln Pro Glu Glu Ala Thr Asn Phe 
                275                 280                 285 

Thr Leu Ala Gly Cys Val Ser Thr Arg Thr Tyr Arg Pro Lys Tyr 
                290                 295                 300 

Cys Gly Val Cys Thr Asp Asn Arg Cys Cys Ile Pro Tyr Lys Ser 
                305                 310                 315 

Lys Thr Ile Ser Val Asp Phe Gln Cys Pro Glu Gly Pro Gly Phe 
                320                 325                 330 

Ser Arg Gln Val Leu Trp Ile Asn Ala Cys Phe Cys Asn Leu Ser 
                335                 340                 345 

Cys Arg Asn Pro Asn Asp Ile Phe Ala Asp Leu Glu Ser Tyr Pro 
                350                 355                 360 

Asp Phe Glu Glu Ile Ala Asn 
                365 


</s-2-xi>
</s-2>
<s-2 seq-no="14 ">
<s-2-i>
<s-2-i-a>251 amino acids </s-2-i-a>
<s-2-i-b>amino acid </s-2-i-b>
<s-2-i-c>single </s-2-i-c>
<s-2-i-d>linear </s-2-i-d>
</s-2-i>
<s-2-vii>
<s-2-vii-a>GenBank </s-2-vii-a>
<s-2-vii-b>g1469936 </s-2-vii-b>
</s-2-vii>
<s-2-xi> 14  

Met Arg Leu His Ser Leu Ile Leu Leu Ser Phe Leu Leu Leu Ala 
                  5                  10                  15 

Thr Gln Ala Phe Ser Glu Lys Val Arg Lys Arg Ala Lys Asn Ala 
                 20                  25                  30 

Pro His Ser Thr Ala Glu Glu Gly Val Glu Gly Ser Ala Pro Ser 
                 35                  40                  45 

Leu Gly Lys Ala Gln Asn Lys Gln Arg Ser Arg Thr Ser Lys Ser 
                 50                  55                  60 

Leu Thr His Gly Lys Phe Val Thr Lys Asp Gln Ala Thr Cys Arg 
                 65                  70                  75 

Trp Ala Val Thr Glu Glu Glu Gln Gly Ile Ser Leu Lys Val Gln 
                 80                  85                  90 

Cys Thr Gln Ala Asp Gln Glu Phe Ser Cys Val Phe Ala Gly Asp 
                 95                 100                 105 

Pro Thr Asp Cys Leu Lys His Asp Lys Asp Gln Ile Tyr Trp Lys 
                110                 115                 120 

Gln Val Ala Arg Thr Leu Arg Lys Gln Lys Asn Ile Cys Arg Asp 
                125                 130                 135 

Ala Lys Ser Val Leu Lys Thr Arg Val Cys Arg Lys Arg Phe Pro 
                140                 145                 150 

Glu Ser Asn Leu Lys Leu Val Asn Pro Asn Ala Arg Gly Asn Thr 
                155                 160                 165 

Lys Pro Arg Lys Glu Lys Ala Glu Val Ser Ala Arg Glu His Asn 
                170                 175                 180 

Lys Val Gln Glu Ala Val Ser Thr Glu Pro Asn Arg Ile Lys Glu 
                185                 190                 195 

Asp Ile Thr Leu Asn Pro Ala Ala Thr Gln Thr Met Thr Ile Arg 
                200                 205                 210 

Asp Pro Glu Cys Leu Glu Asp Pro Asp Val Leu Asn Gln Arg Lys 
                215                 220                 225 

Thr Ala Leu Glu Phe Cys Gly Glu Ser Trp Ser Ser Ile Cys Thr 
                230                 235                 240 

Phe Phe Leu Asn Met Leu Gln Ala Thr Ser Cys 
                245                 250
</s-2-xi>
</s-2>
</sequence-list-old-rules>
</sequence-cwu>
</div>
</li> </ul>
</div>
</section><section itemprop="claims" itemscope="">
<h2>Claims (<span itemprop="count">20</span>)</h2>
<div html="" itemprop="content"><div class="claims" lang="EN" load-source="patent-office" mxw-id="PCLM6590244">
<claim-statement>What is claimed is: </claim-statement>
<div class="claim"> <div class="claim" id="US-20030148934-A1-CLM-00001" num="1">
<div class="claim-text"> <b>1</b>. A substantially purified polypeptide comprising an amino acid sequence selected from the group consisting of SEQ ID NO: 1, SEQ ID NO: 2, a fragment of SEQ ID NO: 1, and a fragment of SEQ ID NO: 2. </div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="US-20030148934-A1-CLM-00002" num="2">
<div class="claim-text"> <b>2</b>. A substantially purified variant having at least 90% amino acid identity to the amino acid sequence of <claim-ref idref="US-20030148934-A1-CLM-00001">claim 1</claim-ref>. </div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="US-20030148934-A1-CLM-00003" num="3">
<div class="claim-text"> <b>3</b>. An isolated and purified polynucleotide encoding the polypeptide of <claim-ref idref="US-20030148934-A1-CLM-00001">claim 1</claim-ref>. </div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="US-20030148934-A1-CLM-00004" num="4">
<div class="claim-text"> <b>4</b>. An isolated and purified polynucleotide variant having at least 70% polynucleotide sequence identity to the polynucleotide of <claim-ref idref="US-20030148934-A1-CLM-00003">claim 3</claim-ref>. </div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="US-20030148934-A1-CLM-00005" num="5">
<div class="claim-text"> <b>5</b>. An isolated and purified polynucleotide which hybridizes under stringent conditions to the polynucleotide of <claim-ref idref="US-20030148934-A1-CLM-00003">claim 3</claim-ref>. </div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="US-20030148934-A1-CLM-00006" num="6">
<div class="claim-text"> <b>6</b>. An isolated and purified polynucleotide having a sequence which is complementary to the polynucleotide sequence of <claim-ref idref="US-20030148934-A1-CLM-00003">claim 3</claim-ref>. </div>
</div>
</div> <div class="claim"> <div class="claim" id="US-20030148934-A1-CLM-00007" num="7">
<div class="claim-text"> <b>7</b>. An isolated and purified polynucleotide comprising a polynucleotide sequence selected from the group consisting of SEQ ID NO: 3, SEQ ID NO: 4, a fragment of SEQ ID NO: 3, and a fragment of SEQ ID NO: 4. </div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="US-20030148934-A1-CLM-00008" num="8">
<div class="claim-text"> <b>8</b>. An isolated and purified polynucleotide variant having at least 70% polynucleotide sequence identity to the polynucleotide of <claim-ref idref="US-20030148934-A1-CLM-00007">claim 7</claim-ref>. </div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="US-20030148934-A1-CLM-00009" num="9">
<div class="claim-text"> <b>9</b>. An isolated and purified polynucleotide having a sequence which is complementary to the polynucleotide of <claim-ref idref="US-20030148934-A1-CLM-00007">claim 7</claim-ref>. </div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="US-20030148934-A1-CLM-00010" num="10">
<div class="claim-text"> <b>10</b>. An expression vector comprising at least a fragment of the polynucleotide of <claim-ref idref="US-20030148934-A1-CLM-00003">claim 3</claim-ref>. </div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="US-20030148934-A1-CLM-00011" num="11">
<div class="claim-text"> <b>11</b>. A host cell comprising the expression vector of <claim-ref idref="US-20030148934-A1-CLM-00010">claim 10</claim-ref>. </div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="US-20030148934-A1-CLM-00012" num="12">
<div class="claim-text"> <b>12</b>. A method for producing a polypeptide, the method comprising the steps of: 
<div class="claim-text">a) culturing the host cell of <claim-ref idref="US-20030148934-A1-CLM-00011">claim 11</claim-ref> under conditions suitable for the expression of the polypeptide; and </div> <div class="claim-text">b) recovering the polypeptide from the host cell culture. </div> </div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="US-20030148934-A1-CLM-00013" num="13">
<div class="claim-text"> <b>13</b>. A pharmaceutical composition comprising the polypeptide of <claim-ref idref="US-20030148934-A1-CLM-00001">claim 1</claim-ref> in conjunction with a suitable pharmaceutical carrier. </div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="US-20030148934-A1-CLM-00014" num="14">
<div class="claim-text"> <b>14</b>. A purified antibody which specifically binds to the polypeptide of <claim-ref idref="US-20030148934-A1-CLM-00001">claim 1</claim-ref>. </div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="US-20030148934-A1-CLM-00015" num="15">
<div class="claim-text"> <b>15</b>. A purified agonist of the polypeptide of <claim-ref idref="US-20030148934-A1-CLM-00001">claim 1</claim-ref>. </div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="US-20030148934-A1-CLM-00016" num="16">
<div class="claim-text"> <b>16</b>. A purified antagonist of the polypeptide of <claim-ref idref="US-20030148934-A1-CLM-00001">claim 1</claim-ref>. </div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="US-20030148934-A1-CLM-00017" num="17">
<div class="claim-text"> <b>17</b>. A method for treating or preventing a cancer, the method comprising administering to a subject in need of such treatment an effective amount of the pharmaceutical composition of <claim-ref idref="US-20030148934-A1-CLM-00013">claim 13</claim-ref>. </div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="US-20030148934-A1-CLM-00018" num="18">
<div class="claim-text"> <b>18</b>. A method for treating or preventing a fibrotic disorder, the method comprising administering to a subject in need of such treatment an effective amount of the antagonist of <claim-ref idref="US-20030148934-A1-CLM-00016">claim 16</claim-ref>. </div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="US-20030148934-A1-CLM-00019" num="19">
<div class="claim-text"> <b>19</b>. A method for detecting a polynucleotide, the method comprising the steps of: 
<div class="claim-text">(a) hybridizing the polynucleotide of <claim-ref idref="US-20030148934-A1-CLM-00006">claim 6</claim-ref> to at least one of the nucleic acids in a biological sample, thereby forming a hybridization complex; and </div> <div class="claim-text">(b) detecting the hybridization complex, wherein the presence of the hybridization complex correlates with the presence of the polynucleotide encoding the polypeptide in the biological sample. </div> </div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="US-20030148934-A1-CLM-00020" num="20">
<div class="claim-text"> <b>20</b>. The method of <claim-ref idref="US-20030148934-A1-CLM-00019">claim 19</claim-ref> further comprising amplifying the polynucleotide prior to hybridization.</div>
</div>
</div> </div>
</div>
</section>
                </article>
            </search-app>
        </body>
    </html>
    