
    <html>
        <body>
            <search-app>
                <article class="result" itemscope="" itemtype="http://schema.org/ScholarlyArticle">
    <h1 itemprop="pageTitle">US20070016380A1 - Protein engineering 
        - Google Patents</h1><section itemprop="abstract" itemscope="">
<h2>Abstract</h2>
<div html="" itemprop="content"><abstract lang="EN" load-source="patent-office" mxw-id="PA49577463">
<div class="abstract" id="p-0001" num="0000">A method of protein engineering is provided wherein a searchable computer database is created comprising entries in the form of descriptions of a location and orientation in 3D space of side chains of the constituent amino acid residues of a framework protein. A query is created which corresponds to a description of a location and orientation in 3D space of respective side chains of amino acid residues of a sample protein. The location and orientation in 3D space of constituent side-chains is preferably described as a Cα Cβ vector. The query is used to search the database and thereby identify a hit which corresponds to a framework protein having structural similarity with said sample protein. Framework protein “hits” so identified may be suitable candidates for further modification. A particular advantage of the present invention is that a modified framework protein may display one or more desired characteristics, such as a function similar to or inhibitory of the sample protein. </div>
</abstract>
</div>
</section><section itemprop="description" itemscope="">
<h2>Description</h2>
<div html="" itemprop="content"><ul class="description" lang="EN" load-source="patent-office" mxw-id="PDES13920922">
<heading id="h-0001">FIELD OF THE INVENTION </heading>
<li> <para-num num="[0001]"> </para-num> <div class="description-line" id="p-0002" num="0001"> THIS INVENTION relates to a method of identifying proteins suitable for protein engineering. In particular, the present invention relates to a computer database searching method of identifying proteins according to aspects of three-dimensional structure, and furthermore to the modification of proteins so identified to thereby possess one or more desired characteristics. Although not limited thereto, this invention relates to engineered proteins such as cytokine mimetics. </div>
</li> <heading id="h-0002">BACKGROUND OF THE INVENTION </heading>
<li> <para-num num="[0002]"> </para-num> <div class="description-line" id="p-0003" num="0002"> Proteins are central to life due to their crucial involvement in a variety of biological processes, such as enzyme catalysis of biochemical reactions, control of nucleic acid transcription and replication, hormonal regulation, signal transduction cascades and antigen recognition during immune responses. </div>
</li> <li> <para-num num="[0003]"> </para-num> <div class="description-line" id="p-0004" num="0003"> In many cases, one or more structural regions of a protein are responsible for a particular function, hereinafter referred to as “functional regions”. These regions may constitute the active site of a protein enzyme, the nucleic acid binding domain of a transcription factor, a region of a protein cytokine crucial to binding the specific receptor for that cytokine, or antigen-binding regions of antigen receptors. </div>
</li> <li> <para-num num="[0004]"> </para-num> <div class="description-line" id="p-0005" num="0004"> A functional region of a protein usually comprises one or more amino acids which are required for that particular function, that is, they are essential for that function. </div>
</li> <li> <para-num num="[0005]"> </para-num> <div class="description-line" id="p-0006" num="0005"> In many cases, although these required amino acid residues are topographically proximal to each other, they may be well separated with respect to primary amino acid sequence, that is, they are non-contiguous. In addition, where there is more than one functional region of a protein, these regions may also be topographically proximal, but well separated in terms of primary amino acid sequence. In some cases, however, where there is more than one functional region involved in a particular function, these functional regions may also be topographically well separated. This is a particularly important point with regard to the functional regions of cytokines. </div>
</li> <li> <para-num num="[0006]"> </para-num> <div class="description-line" id="p-0007" num="0006"> “Cytokine” as used herein includes and encompasses soluble protein molecules which have a cognate cell surface receptor, and which are involved in initiating, controlling and otherwise regulating a variety of processes relevant to cell growth, death and differentiation. Cytokines are typically exemplified by interferons (e.g. IFN-γ), interleukins (for example IL-2, IL-4 and IL-6), growth and differentiation factors [e.g. granulocyte colony stimulating factor (G-CSF) and erythropoietin (EPO)] and others such as growth hormone (GH), prolactin, TGF-β, tumour necrosis factor (TNF) and insulin. Each of these molecules is capable of binding a specific receptor and thereby eliciting a particular biological response or set of responses. </div>
</li> <li> <para-num num="[0007]"> </para-num> <div class="description-line" id="p-0008" num="0007"> The fact that a particular function of a protein can be attributed to one or more functional regions of that protein has formed the basis for strategies aimed at modifying a protein by adding or subtracting functional regions to modify the function of that protein. </div>
</li> <li> <para-num num="[0008]"> </para-num> <div class="description-line" id="p-0009" num="0008"> In this regard, the design and engineering of cytokine mimetics has become an area of major importance, as many cytokine-cytokine receptor interactions are central to the regulation of a variety of biological processes. It is envisaged that new mimetics will therefore become important new therapeutic agents that either mimic or inhibit the biological response to cytokine-cytokine receptor interactions. </div>
</li> <li> <para-num num="[0009]"> </para-num> <div class="description-line" id="p-0010" num="0009"> A “mimetic” is a molecule which elicits a biological response either similar to, or more powerful than, that of another molecule (an “agonist”), or inhibits the action of the other molecule (an “antagonist”). The other molecule may be a cytokine, for example. </div>
</li> <li> <para-num num="[0010]"> </para-num> <div class="description-line" id="p-0011" num="0010"> With regard to designing and engineering mimetics based on cytokines, a problem frequently encountered with many engineered mimetics has been that they exhibit short biological half-lives and hence minimal bioavailability and efficacy. In this regard, it has been proposed that small cysteine-rich proteins might be useful as protein “scaffolds” as a basis for engineering mimetics, due to their stability (Vita et al., 1995, Proc. Natl. Acad. Sci. USA 92 6404). These small cysteine-rich proteins comprise a disulfide-bonded core and exposed amino acid side chains at the protein surface (Neilsen et al., 1996, J. Mol. Biol. 263 297). However the full potential of these proteins has not been realized due to the fact that typical prior art strategies for protein engineering have largely been limited to transferring or exchanging contiguous groups of amino acids within individual secondary structural elements, such as loops or helices or β-sheets and no design strategies exist for selecting the most appropriate disulfide-rich candidiate. </div>
</li> <li> <para-num num="[0011]"> </para-num> <div class="description-line" id="p-0012" num="0011"> Examples of such an approach would include: the exchange of secondary structural regions between RNase and angiogenin, either to confer RNase activity on angiogenin (Harper et al., 1989, Biochemistry 28 1875) or angiogenic activity on RNase (Raines et al., 1995, J. Biol. Chem. 27017180); the insertion of elastase inhibition activity into IL-1β by transfer of the protease inhibitor loop of elastase to the IL-1β scaffold (Wolfson et al., 1993, Biochemistry 32 5327); the insertion of a 10 amino acid calcium-binding loop of thermolysin into <i>Bacillus subtilis </i>neutral protease (Toma et al., 1991, Biochemistry 30 97); the insertion of a β-sheet from a snake toxin to replace the β-sheet of charybdotoxin (Drakopolou et al., 1996, J. Biol. Chem. 271 11979); and the incorporation of a β-sheet from carbonic anhydrase into the β-sheet of charybdotoxin (Pierret et al., 1995, J. Med. Chem. 35 2145). </div>
</li> <li> <para-num num="[0012]"> </para-num> <div class="description-line" id="p-0013" num="0012"> Of growing importance in protein engineering has been the use of computer based technology combined with the elucidation of the 3D structures of small molecules and macromolecules. 3D molecular structures are being generated at an increasing rate, such as by X-Ray crystallography and NMR techniques. These 3D features can be stored in generally accessible, searchable databases, such as the BROOKHAVEN database. </div>
</li> <li> <para-num num="[0013]"> </para-num> <div class="description-line" id="p-0014" num="0013"> For the purposes of this specification, a database will comprise a collection of “entries”, each entry corresponding to a representation of an aspect of 3D structure of a framework protein: A framework protein is simply any protein for which a 3D structure exists, either by experimental elucidation or by predictive means such as computer modelling. A framework protein is potentially useful as a scaffold which can be structurally modified for the purposes of imparting a particular function thereto. </div>
</li> <li> <para-num num="[0014]"> </para-num> <div class="description-line" id="p-0015" num="0014"> A “query” refers herein to a representation of an aspect of 3D structure of a protein which exhibits a function of interest. The representation of 3D structure would be in a form suitable for searching a database with the intention of identifying a “hit”. A hit is an entry identified according to the particular query and the algorithm used to perform the search. </div>
</li> <li> <para-num num="[0015]"> </para-num> <div class="description-line" id="p-0016" num="0015"> An important advance in database searching has been made by representing 3D structures in terms of the relationship between atoms located in “distance space”, rather than “Cartesian space” (Jakes &amp; Willett, 1986, J. Mol. Graphics 4 12; Ho &amp; Marshall, 1993, J. Comp. Aided. Mol. Des. 7 3). A location in Cartesian space is defined by three coordinates (x, y, z) which each correspond to a position along three respective axes (X, Y, Z), each axis being oriented at right angles to the other two. </div>
</li> <li> <para-num num="[0016]"> </para-num> <div class="description-line" id="p-0017" num="0016"> A location in distance space, however, is defined by distances between atoms, expressed in the form of a distance matrix, which details the distance between atoms. Distance matrices are therefore coordinate independent, and comparisons between distance matrices can be made without restriction to a particular frame of reference, such as is required using Cartesian coordinates. </div>
</li> <li> <para-num num="[0017]"> </para-num> <div class="description-line" id="p-0018" num="0017"> It is important to emphasise that an arrangement of atoms and its mirror image are described by identical distance matrices. A root mean squared (RMS) difference can be used to alleviate this ambiguity. </div>
</li> <li> <para-num num="[0018]"> </para-num> <div class="description-line" id="p-0019" num="0018"> With regard to the 3D structure of proteins, a simplification of protein structure can be provided by reducing a 3D structure to “Cα-Cβ vectors” as discussed in McKie et al., 1995, Peptides: Chemistry, Structure &amp; Biology p 354-355. A Cα-Cβ vector occupies a location in 3D space, the location being defined by the orientation of the covalent bond between the α carbon and β carbon atoms of an amino acid (Lauri &amp; Bartlett, 1994, J. Comp. Aid. Mol. Des. 8 51). It will be appreciated that each of the 20 naturally-occurring constituent amino acids of a protein (except glycine), possess a Cα-Cβ vector due to the covalent bond between the “central” α carbon and the β carbon of the constituent side chain. </div>
</li> <li> <para-num num="[0019]"> </para-num> <div class="description-line" id="p-0020" num="0019"> For those proteins containing Gly in the database, it is possible to mutate this to Ala to generate the required Cα-Cβ vector for database searching. </div>
</li> <li> <para-num num="[0020]"> </para-num> <div class="description-line" id="p-0021" num="0020"> The usefulness of Cα-Cβ vectors is that they provide a simplification of 3D structure. Therefore, only the amino acid side-chains of a functional region of a protein need be represented by the Cα-Cβ vector map, thereby excluding the substantial portion of the protein(s) not directly involved in that particular function. For the purposes of database searching, Cα-Cβ vectors are ideal, as they constitute the basic 3D structural information needed. </div>
</li> <li> <para-num num="[0021]"> </para-num> <div class="description-line" id="p-0022" num="0021"> After identification of Cα-Cβ vectors corresponding to a protein or a functional region thereof, the parameters that characterize each vector must be stored in a database in such a way that retrieval in response to a query can be made quickly. A number of options are available for suitable representation of Cα-Cβ vectors, whether as a database entry or as a query: 
</div> </li> <ul> <li id="ul0001-0001" num="0022"> (A) as a distance matrix; </li> <li id="ul0001-0002" num="0023"> (B) as a dihedral angle (δ) formed between respective Cα-Cβ vectors; </li> <li id="ul0001-0003" num="0024"> (C) as angles α<sub>1 </sub>and α<sub>2 </sub>formed between respective Cα-Cβ vectors. </li> </ul>
<li> <para-num num="[0025]"> </para-num> <div class="description-line" id="p-0023" num="0025"> A simple explanation of these representations is provided in Lauri &amp; Bartlett, 1994, supra, which is hereinafter incorporated by reference. The key to successful database searching is speed and efficiency. Thus, computer search algorithms have been developed which use a strategy whereby the vast majority of entries in the database are eliminated in a preliminary screening step. </div>
</li> <li> <para-num num="[0026]"> </para-num> <div class="description-line" id="p-0024" num="0026"> These algorithms are demanding of computer resources, and therefore a search is normally effected in two stages: 
</div> </li> <ul> <li id="ul0002-0001" num="0027"> (1) a screening search to eliminate entries that cannot possibly constitute a hit; and </li> <li id="ul0002-0002" num="0028"> (2) an atom-by-atom comparison of a query with each entry not eliminated in (1), to identify one or more hits. </li> </ul>
<li> <para-num num="[0029]"> </para-num> <div class="description-line" id="p-0025" num="0029"> The search in (1) could screen entries based on geometric attributes of the query (Lesk, 1979, Commun. ACM 22 219) interatomic distances and atom types (Jakes &amp; Willett, 1986, supra), aromaticity, hybridization, connectivity, charge, position of lone pair electrons, or centre of mass of ring structures (Sheridan et al., 1989, Proc. Natl Acad. Sci. 86 8165). This screening process would eliminate entries that have no chance of meeting the 3D constraints of the query. </div>
</li> <li> <para-num num="[0030]"> </para-num> <div class="description-line" id="p-0026" num="0030"> This strategy, although quick, requires that for an entry to register as a hit, it must comprise every specified query component. As the number of query components increases, the number of near misses increases and the likelihood of finding a hit decreases. </div>
</li> <li> <para-num num="[0031]"> </para-num> <div class="description-line" id="p-0027" num="0031"> A more useful search strategy which assesses the relative merits of each near miss as well as each hit has recently been provided by the search program FOUNDATION (Ho &amp; Marshall, 1993, supra). FOUNDATION uses a clique-detection algorithm (various algorithms are reviewed and compared in Brint &amp; Willett, 1987, J. Mol. Graphics 5 49 and Brint &amp; Willett, 1987, Chem. Inf. Comput. Sci. 27 152) which searches a 3D database of entries for a user-defined query consisting of the coordinates of various atoms and/or bonds of a 3D structural feature. FOUNDATION identifies all possible entries that contain any combination of a user-specified minimum number of matching atoms and/or bonds as hits. </div>
</li> <li> <para-num num="[0032]"> </para-num> <div class="description-line" id="p-0028" num="0032"> Despite the usefulness of 3D database searching as a means of identifying structurally related proteins, this approach has not been well utilized with respect to engineering proteins with a desired function. </div>
</li> <heading id="h-0003">OBJECT OF THE INVENTION </heading>
<li> <para-num num="[0033]"> </para-num> <div class="description-line" id="p-0029" num="0033"> The present inventors have recognized that 3D database searching is useful for identifying proteins which have one or more desired structural features, such proteins being candidate “frameworks” for the subsequent engineering of proteins with desired characteristics or functions. Furthermore, the present inventors have realized that protein engineering is best achieved by modification of a framework protein to incorporate particular amino acid residues required for a characteristic, property or function, rather than by incorporating entire elements of secondary structure such as loops or helices. This is particularly applicable when functionally important amino acids are scattered throughout a protein and are not confined to particular regions of primary or secondary structure. </div>
</li> <li> <para-num num="[0034]"> </para-num> <div class="description-line" id="p-0030" num="0034"> It is therefore an object of the present invention to provide a novel method of protein engineering. </div>
</li> <heading id="h-0004">SUMMARY OF THE INVENTION </heading>
<li> <para-num num="[0035]"> </para-num> <div class="description-line" id="p-0031" num="0035"> In one aspect, the present invention resides in a method of protein engineering including the steps of: 
</div> </li> <ul> <li id="ul0003-0001" num="0036"> (i) creating a computer database which includes a plurality of entries, each said entry corresponding to a description of a location and orientation in 3D space of side chains of amino acid residues of a framework protein; </li> <li id="ul0003-0002" num="0037"> (ii) creating a query corresponding to a description of a location and orientation in 3D space of respective side chains of two or more amino acid residues of a sample protein which are required for a function of said sample protein; and </li> <li id="ul0003-0003" num="0038"> (iii) searching said database with said query to thereby identify one or more hits wherein at least one of said hits corresponds to a respective said framework protein having structural similarity with said sample protein. </li> </ul>
<li> <para-num num="[0039]"> </para-num> <div class="description-line" id="p-0032" num="0039"> Preferably, the framework protein(s) identified as a hit has greater stability than said sample protein. </div>
</li> <li> <para-num num="[0040]"> </para-num> <div class="description-line" id="p-0033" num="0040"> In one embodiment, the method includes the step of modifying an amino acid sequence of said framework protein which corresponds to a hit, by substituting at least one amino acid residue thereof with at least one amino acid residue of said sample protein to thereby create a modified framework protein having increased structural similarity to said sample protein. </div>
</li> <li> <para-num num="[0041]"> </para-num> <div class="description-line" id="p-0034" num="0041"> Preferably, said at least one amino acid residue of said sample protein represents at least a portion of a functional region of said sample protein. </div>
</li> <li> <para-num num="[0042]"> </para-num> <div class="description-line" id="p-0035" num="0042"> More preferably, at least two of the amino acid residues of said sample protein which substitute amino acid residues of said framwork protein are non-contiguous in primary sequence. </div>
</li> <li> <para-num num="[0043]"> </para-num> <div class="description-line" id="p-0036" num="0043"> Preferably, the framework protein so modified has increased structural similarity to said sample protein. </div>
</li> <li> <para-num num="[0044]"> </para-num> <div class="description-line" id="p-0037" num="0044"> Advantageously, the modified framework protein is capable of exhibiting a function which is either similar to, or inhibitory of, a function of said sample protein. </div>
</li> <li> <para-num num="[0045]"> </para-num> <div class="description-line" id="p-0038" num="0045"> Preferably, the location and orientation of each amino acid side-chain of said framework protein and said sample protein is represented by a Cα-Cβ vector. </div>
</li> <li> <para-num num="[0046]"> </para-num> <div class="description-line" id="p-0039" num="0046"> Preferably, the framework protein is a small cysteine-rich protein capable of internal disulfide bond formation. More preferably, said small cysteine-rich protein comprises 70 amino acids or less, having 2-11 disulfide bonds. </div>
</li> <li> <para-num num="[0047]"> </para-num> <div class="description-line" id="p-0040" num="0047"> In one embodiment, said sample protein is a cytokine selected from the group consisting of GH, IL-4, IL-6 and G-CSF. </div>
</li> <li> <para-num num="[0048]"> </para-num> <div class="description-line" id="p-0041" num="0048"> In a second aspect, the invention provides an engineered protein comprising 70 amino acid residues or less of a framework protein and 2-11 disulfide bonds of said framework protein, together with at least two amino acids of another protein which are non-contiguous in primary sequence and which represent at least a portion of a functional region of said another protein. </div>
</li> <li> <para-num num="[0049]"> </para-num> <div class="description-line" id="p-0042" num="0049"> Preferably, the engineered protein has greater stability than said another protein. </div>
</li> <li> <para-num num="[0050]"> </para-num> <div class="description-line" id="p-0043" num="0050"> More preferably, the engineered protein exhibits a function either similar to, or inhibitory of, said another protein. </div>
</li> <li> <para-num num="[0051]"> </para-num> <div class="description-line" id="p-0044" num="0051"> In one embodiment, said another protein is a cytokine selected from the group consisting of GH, IL-4, IL-6 and G-CSF. </div>
</li> <li> <para-num num="[0052]"> </para-num> <div class="description-line" id="p-0045" num="0052"> In a particular embodiment, the engineered protein has an amino acid sequence selected from the group consisting of the amino acid sequences of SCY01, SCY02, SCY03, ERP01, ERP02, ERP03 and VIBO1. </div>
</li> <li> <para-num num="[0053]"> </para-num> <div class="description-line" id="p-0046" num="0053"> In yet another aspect, the present invention resides in a computer program for searching a protein structure database. </div>
</li> <li> <para-num num="[0054]"> </para-num> <div class="description-line" id="p-0047" num="0054"> In one embodiment, the computer program is for searching a protein database comprising a plurality of entries, each said entry corresponding to a distance matrix representation of two or more Cα-Cβ vectors, said program including the steps of: 
</div> </li> <ul> <li id="ul0004-0001" num="0055"> (i) comparing a query with each said database entry, said query corresponding to a distance matrix representation of two or more Cα-Cβ vectors; and </li> <li id="ul0004-0002" num="0056"> (ii) identifying hits by clique detection, wherein a hit is defined according to a minumum number of Cα-Cβ vector matches between said query and each said entry. </li> </ul>
<li> <para-num num="[0057]"> </para-num> <div class="description-line" id="p-0048" num="0057"> Throughout this specification and claims which follow, unless the context requires otherwise, “comprise”, “comprises” and “comprising” are used inclusively, so that a stated integer or integer group does not exclude other integers or integer groups. </div>
</li> <li> <para-num num="[0058]"> </para-num> <div class="description-line" id="p-0049" num="0058"> It will also be appreciated that throughout this specification and claims, scientific terms are to be given their usual scientific meaning, although certain terms are defined herein to assist interpretation by the skilled person.</div>
</li> <heading id="h-0005">BRIEF DESCRIPTION OF THE FIGURES AND TABLES </heading>
<li> <para-num num="[0059]"> </para-num> <div class="description-line" id="p-0050" num="0059"> Table 1: An example of a query file which defines the query Cα-Cβ vectors, the tolerance for each query atom and the definition of a subset. </div>
</li> <li> <para-num num="[0060]"> </para-num> <div class="description-line" id="p-0051" num="0060"> Table 2: Blood serum stability test results of a solution of SCY01. </div>
</li> <li> <para-num num="[0061]"> </para-num> <div class="description-line" id="p-0052" num="0061"> Table 3: Enzyme stability test results of a solution of SCY01. </div>
</li> <li> <para-num num="[0062]"> </para-num> <div class="description-line" id="p-0053" num="0062"> <figref idrefs="DRAWINGS">FIG. 1</figref>: Amino acid sequences of the hGH high affinity site antagonist framework scyllatoxin, the hGH antagonists SCY01, SCY02, SCY03 and their alignment with the hGH sequence. Disulfide linkages are indicated by lines connecting cysteines. </div>
</li> <li> <para-num num="[0063]"> </para-num> <div class="description-line" id="p-0054" num="0063"> <figref idrefs="DRAWINGS">FIG. 2</figref>: Amino acid sequences for the hGH agonist framework VIB, the engineered molecule VIB01 and the alignment with the hGH sequence. Disulfide linkages are indicated by lines connecting cysteines. </div>
</li> <li> <para-num num="[0064]"> </para-num> <div class="description-line" id="p-0055" num="0064"> <figref idrefs="DRAWINGS">FIG. 3</figref>: Comparision of the hGH structure with hGH agonist molecule VIB01 showing the very high degree of overlap of the alpha helices. </div>
</li> <li> <para-num num="[0065]"> </para-num> <div class="description-line" id="p-0056" num="0065"> <figref idrefs="DRAWINGS">FIG. 4</figref>: Schematic overview of database searching strategy. </div>
</li> <li> <para-num num="[0066]"> </para-num> <div class="description-line" id="p-0057" num="0066"> <figref idrefs="DRAWINGS">FIG. 5</figref>: Two-dimensional depiction of three different representations of a pair of Cα-Cβ vectors: d=interatomic distance as used to construct distance matrices; δ=dihedral angle; α<sub>1 </sub>and α<sub>2 </sub>angles. </div>
</li> <li> <para-num num="[0067]"> </para-num> <div class="description-line" id="p-0058" num="0067"> <figref idrefs="DRAWINGS">FIG. 6</figref>: Circular dichroism spectra of SCY01 showing little change in the structure on temperature changes or on the addition of helix stabilizing agent Trifluroethanol. </div>
</li> <li> <para-num num="[0068]"> </para-num> <div class="description-line" id="p-0059" num="0068"> <figref idrefs="DRAWINGS">FIG. 7</figref>: Structure of the engineered SCY01 molecule shown in comparision with the native scyllatoxin molecule. </div>
</li> <li> <para-num num="[0069]"> </para-num> <div class="description-line" id="p-0060" num="0069"> <figref idrefs="DRAWINGS">FIG. 8</figref>: Biological effect of SCY01 on BaF3 cell proliferation by inhibiting the growth response of the cells to 0.5 ng/mL hGH, but not to 50 U/mL IL-3. </div>
</li> <li> <para-num num="[0070]"> </para-num> <div class="description-line" id="p-0061" num="0070"> <figref idrefs="DRAWINGS">FIG. 9</figref>: Amino acid sequence for the low affinity site hGH anatagonist framework ZDC and the engineered hGH anatagonist ZDC05 and the aligned hGH sequence. Disulfide linkages are indicated by lines connecting cysteines. </div>
</li> <li> <para-num num="[0071]"> </para-num> <div class="description-line" id="p-0062" num="0071"> <figref idrefs="DRAWINGS">FIG. 10</figref>: Circular dichroism spectra of VIB01. </div>
</li> <li> <para-num num="[0072]"> </para-num> <div class="description-line" id="p-0063" num="0072"> <figref idrefs="DRAWINGS">FIG. 11</figref>: Amino acid sequences of the hGH agonist framework ERP, the engineered molecules ERP01, ERP02, ERP03 and their alignment with the hGH sequence. Disulfide linkages are indicated by lines connecting cysteines. </div>
</li> <li> <para-num num="[0073]"> </para-num> <div class="description-line" id="p-0064" num="0073"> <figref idrefs="DRAWINGS">FIG. 12</figref>: Circular dichroism spectra of ERP03 showing little change in the structure on temperature changes or on the addition of helix stabilizing agent Trifluroethanol. </div>
</li> <li> <para-num num="[0074]"> </para-num> <div class="description-line" id="p-0065" num="0074"> <figref idrefs="DRAWINGS">FIG. 13</figref>: Comparison of secondary Hα shifts for ERP01 and ERP03 showing substantially identical structure and disulphide connectivities. The shaded bars show the invarient residues of the native ERP molecule. -▪-=ERP03 δHA; -♦-=ERP δHA. </div>
</li> <li> <para-num num="[0075]"> </para-num> <div class="description-line" id="p-0066" num="0075"> <figref idrefs="DRAWINGS">FIG. 14</figref>: Amino acid sequences of the CD4 frameworks PTA and SCY, the engineered molecules PTA CD4, and SCY CD4 and the alignment with the CD4 sequence. Disulfide linkages are indicated by lines connecting cysteines.</div>
</li> <heading id="h-0006">DETAILED DESCRIPTION OF THE PREFERRED EMBODIMENTS </heading>
<li> <para-num num="[0076]"> </para-num> <div class="description-line" id="p-0067" num="0076"> It will be appreciated that the present invention is predicated, at least in part, on the present inventors' realization that in order to identify framework proteins suitable for further modification by protein engineering, it is advantageous to search databases according to the orientation in 3D space of constituent amino acid side-chains of the framework protein, with respect to constituent amino acid side-chains of the sample protein which is the subject of the query. Framework protein “hits” so identified suitably share similarity, such as in terms of topography and chemistry, to the sample protein “query”, and as such may be suitable candidates for further modification. A particular aspect of the present invention is that a modified framework protein may display one or more desired characteristics, such as increased stability and in some cases a function similar to or inhibitory of the sample protein. </div>
</li> <li> <para-num num="[0077]"> </para-num> <div class="description-line" id="p-0068" num="0077"> Referring to the method of the first-mentioned aspect, preferably, the 3D location and orientation of each amino acid side-chain of said framework protein is represented as a Cα-Cβ vector. </div>
</li> <li> <para-num num="[0078]"> </para-num> <div class="description-line" id="p-0069" num="0078"> Preferably, each said entry corresponds to a description in the form of a distance matrix representation of said Cα-Cβ vectors. </div>
</li> <li> <para-num num="[0079]"> </para-num> <div class="description-line" id="p-0070" num="0079"> Alternatively, said Cα-Cβ vectors may be represented by dihedral angles or α<sub>1 </sub>and α<sub>2 </sub>angles. </div>
</li> <li> <para-num num="[0080]"> </para-num> <div class="description-line" id="p-0071" num="0080"> As used herein, “protein” and “polypeptide” are used interchangeably with regard to amino acid polymers. A “peptide” is a protein which has no more than fifty (50) amino acids. </div>
</li> <li> <para-num num="[0081]"> </para-num> <div class="description-line" id="p-0072" num="0081"> As used herein, a “framework protein” is any protein which exhibits one or more desired structural features which provide advantages which include size, solubility and/or stability. “Stability” in this context includes resistance to degradation by proteolytic enzymes and/or temperature variation and/or resistance to denaturation by chaotropic agents and/or denaturing detergents, changes in pH, pH extremes, and/or REDOX extremes and/or changes. </div>
</li> <li> <para-num num="[0082]"> </para-num> <div class="description-line" id="p-0073" num="0082"> Preferably, the framework protein is a small cysteine-rich protein capable of internal disulfide bond formation. More preferably, said framework protein is a small cysteine-rich protein comprising 70 amino acids or less, having 2-11 disulfide bonds. </div>
</li> <li> <para-num num="[0083]"> </para-num> <div class="description-line" id="p-0074" num="0083"> The amino acids used for creating each said entry may include some or all of the constituent of amino acids of the framework protein. </div>
</li> <li> <para-num num="[0084]"> </para-num> <div class="description-line" id="p-0075" num="0084"> As used herein, a “sample protein” is a protein which has one or more functional characteristics of interest which render it desirable for the purposes of protein engineering. </div>
</li> <li> <para-num num="[0085]"> </para-num> <div class="description-line" id="p-0076" num="0085"> Suitably, the sample protein may be an enzyme, nucleic acid-binding protein, cytokine, antigen, receptor, ion channel, chaperonin, or any protein with a function of interest. </div>
</li> <li> <para-num num="[0086]"> </para-num> <div class="description-line" id="p-0077" num="0086"> In an embodiment, said sample protein is a cytokine selected from the group consisting of GH, IL-4, G-CSF, IL-6 and EPO. </div>
</li> <li> <para-num num="[0087]"> </para-num> <div class="description-line" id="p-0078" num="0087"> Preferably, said function of said sample protein comprises binding a specific receptor to thereby elicit a biological response. However, a variety of other functions are contemplated, such as catalysis, binding cations (Zn<sup>++</sup>, Ca<sup>++</sup>, Mg<sup>++</sup>), transporting ions (e.g. Cl<sup>−</sup>, K<sup>+</sup>, Na<sup>+</sup>), binding lipids, binding nucleic acids as a means of transcriptional regulation or regulating DNA replication, assisting protein folding and transport, and any other function carried out by proteins. </div>
</li> <li> <para-num num="[0088]"> </para-num> <div class="description-line" id="p-0079" num="0088"> With regard to creating a query, it is preferred that the location and orientation in 3D space of each side-chain of said two or more amino acid residues is simplified as a Cα-Cβ vector. </div>
</li> <li> <para-num num="[0089]"> </para-num> <div class="description-line" id="p-0080" num="0089"> Preferably, said query corresponds to a description in the form of a distance matrix representation of Cα-Cβ vectors. However, other representations such as dihedral angles or α<sub>1 </sub>and α<sub>2 </sub>angles may also be applicable. </div>
</li> <li> <para-num num="[0090]"> </para-num> <div class="description-line" id="p-0081" num="0090"> Preferably, said computer program used for searching said database is the VECTRIX program, as will be described in detail hereinafter. VECTRIX incorporates the FOUNDATION algorithm (Ho &amp; Marshall, 1993, supra, which is herein incorporated by reference). Program FOUNDATION searches 3D databases of small organic molecules to identify structures that contain any combination of a user-specified minimum number of matching elements of a user-defined query. It achieves this by first using a distance matrix to define the topography of the query atoms, followed by screening using various query constraints which define the chemical nature of the structure. The topology of the atoms in the structure are again represented using a distance matrix. Structural fragments in the database, whose distance description matches those of the query are identified using graph theory (Gibbons <i>Algorithmic Graph Theory; </i>Cambridge University Press: Cambridge, 1988). </div>
</li> <li> <para-num num="[0091]"> </para-num> <div class="description-line" id="p-0082" num="0091"> In graph theory, a graph is a structure comprised of nodes (vertices) connected by edges. A graph is completely connected when all nodes are connected to one another. A subgraph is any subset of a larger graph. The largest completely connected subgraph of any graph is called a clique. Thus, the query is a completely connected graph, as all interatomic distances are determined in the distance matrix. The task is then to search a structural database to find all cliques that contain at least a user-defined number of matching nodes. </div>
</li> <li> <para-num num="[0092]"> </para-num> <div class="description-line" id="p-0083" num="0092"> There are many clique-finding algorithms. Some of the well known procedures include those by Bonner, 1964, IBM J. Res. Develop., 8 22; Gerhards &amp; Lindenberg, 1981, Computing 27 349 and Bron &amp; Kerbosch, 1973, Commun. ACM 16 575. Computational chemists have adapted these algorithms or implemented similar ideas to facilitate searching for 3D structures within databases (Kuntz et al., 1982, J. Mol. Biol. 161 269; DesJarlais et al., 1988, J. Med. Chem. 31 722; DesJarlais et al., 1990, Proc. Natl. Acad. Sci. 87 6644; Crandell &amp; Smith, 1983, J. Chem. Infr. Comput. Sci. 23 186; Brint &amp; Willett, 1987, J. Mol. Graphics 5 49-56; Kuhl et al., 1984, J. Comput. Chem. 5 24 and Smellie et al., 1991, J. Chem. Inf. Sci. 31 386). </div>
</li> <li> <div class="description-line" id="h-0007" num="0000"> Computer Database Searching </div>
</li> <li> <div class="description-line" id="h-0008" num="0000"> VECTRIX </div>
</li> <li> <para-num num="[0093]"> </para-num> <div class="description-line" id="p-0084" num="0093"> The present inventors have created a program “VECTRIX”, which is a modified version of the clique-detection algorithm in program FOUNDATION as described by Ho &amp; Marshall, 1993, J. Comp. Aided. Mol. Des. 7 3-22. The search procedure is illustrated in Scheme A. The major changes in comparison to Ho &amp; Marshall, 1993, supra include: 
</div> </li> <ul> <li id="ul0005-0001" num="0094"> the query and database structures are both proteins; </li> <li id="ul0005-0002" num="0095"> the query elements are a distance matrix defining the topography of Cα-Cβ vectors, not individual atoms as in FOUNDATION; 
<br/>
 similarly, the database structure is defined as a Cα-Cβ vector distance-matrix and not every atom as in FOUNDATION; 
</li> <li id="ul0005-0003" num="0096"> in FOUNDATION, a pair of atoms in a query is considered to match with a pair of atoms in an entry in the database if the atom-type and the distance between them are matched; in VECTRIX, a pair of Cα-Cβ vectors in a query is considered to match with a pair of Cα-Cβ vectors in an entry in the database if the four distances (Cα<sub>1</sub>-Cα<sub>2</sub>; Cα<sub>1</sub>-Cβ<sub>2</sub>; Cβ<sub>1</sub>-Cα<sub>2</sub>; Cβ<sub>1</sub>-Cβ<sub>2</sub>) between the pairs are matched; and </li> <li id="ul0005-0004" num="0097"> the FOUNDATION program performs the clique detection, steric filtering and subset filtering together and outputs the hits that satisfy the three criteria; by design, the VECTRIX program output all hits that have number of matches greater than or equal to MIN_MATCH. POSTVEC is then used to filter those hits based on steric filtering, a new MIN_MATCH and subset consideration; by separating the clique detection hits and the filtering process, the VECTRIX program is more flexible. </li> </ul>
<li> <para-num num="[0098]"> </para-num> <div class="description-line" id="p-0085" num="0098"> An outline of a program written by the present inventors is shown in Scheme A. </div>
</li> <li> <para-num num="[0099]"> </para-num> <div class="description-line" id="p-0086" num="0099"> The VECTRIX program requires four parameters: (1) query.file (2) database.file; (3) steric.file and (4) MIN_MATCH. The parameters are described in detail below. </div>
</li> <li> <div class="description-line" id="h-0009" num="0000"> (1 ) query.file </div>
</li> <li> <para-num num="[0100]"> </para-num> <div class="description-line" id="p-0087" num="0100"> query.file (for example as in Table 1) contains the definition of the query, the definition of tolerance for each query atom and the definition of SUBSET. The three definitions are described below: </div>
</li> <li> <para-num num="[0101]"> </para-num> <div class="description-line" id="p-0088" num="0101"> Query definition: Prior to running the VECTRIX program, a particular target protein is selected. The target proteins three-dimensional structure must have been determined by experimental or theoretical means well known in the art. The functional amino acids of the target protein must be defined and the Cα-Cβ vectors for those functional residues extracted to the query.file. Table 1 shows the definition of Cα-Cβ vectors of four functional residues. The numbers in column 7, 8 and 9 represent the x, y and z coordinates of the vectors respectively. </div>
</li> <li> <para-num num="[0102]"> </para-num> <div class="description-line" id="p-0089" num="0102"> Tolerance Definition: The tolerance defines the allowable uncertainty in the orientation of each atom. Note that the final tolerance of a vector from atom A to atom B is the sum of the individual tolerance of atom A and B. In Table 1, the tolerances for individual atoms are defined in column 10 to be 0.5 Å, so the tolerance for a distance between two atoms is 1.0 Å. </div>
</li> <li> <para-num num="[0103]"> </para-num> <div class="description-line" id="p-0090" num="0103"> Subset Definition: A list of atoms can be grouped into a SUBSET. The query file allows for the definition of as many SUBSETs as are required. The SUBSET definition will be used in the POSTVEC program to filter the hits to obtain more relevant hits. In Table 1, the 1<sup>st </sup>SUBSET command is defined as subset 1 and it consists of Cα-Cβ vector numbers 1, 3 and 4. The 2<sup>nd </sup>SUBSET command is defined as subset 2 and it consists of Cα-Cβ vector number 2. </div>
</li> <li> <div class="description-line" id="h-0010" num="0000"> (2) database.file </div>
</li> <li> <para-num num="[0104]"> </para-num> <div class="description-line" id="p-0091" num="0104"> database.file contains a list of file names that correspond with the entries constituting the database. </div>
</li> <li> <div class="description-line" id="h-0011" num="0000"> (3) steric.file </div>
</li> <li> <para-num num="[0105]"> </para-num> <div class="description-line" id="p-0092" num="0105"> steric.file contains the coordinates of the grid points representing the ligand or receptor space. There are two forms of steric filtering depending on the availability of 3D structure of a receptor or ligand. If the structure of the receptor is known and a query is from the Cα-Cβ vectors corresponding to the receptor-binding amino acid side chains of a ligand, then a hit must be evaluated in terms of whether it would invade the 3D space accessed by the receptor upon binding a cytokine, for example (receptor-based filtering). Moreover, if the structure of the ligand is known and a query is from the Cα-Cβ vectors corresponding to the receptor-binding amino acid side chains of a ligand, then a hit must be evaluated in terms of whether it would invade the 3D space not occupied by the ligand (ligand-based filtering). The mode is identified in the first line of the ‘steric file’. The first step in our steric filtering algorithm is the calculation of the grid points that represent the ligand or receptor 3D space using the program PREPARE_STERIC_FILTER. The program first defines the limits of the structure via determining the maxima and minima in the x, y and z dimension. Then for each grid points (1 Å apart) within the limit, a xyz coordinate is output to a ‘steric file’ if the point is in steric contact with the receptor or the ligand. </div>
</li> <li> <div class="description-line" id="h-0012" num="0000"> (4) MIN_MATCH </div>
</li> <li> <para-num num="[0106]"> </para-num> <div class="description-line" id="p-0093" num="0106"> MIN_MATCH is an integer defining the minimum number of Cα-Cβ vectors that match between the query and the entry in the database required before VECTRIX will consider a clique as a hit. </div>
</li> <li> <para-num num="[0107]"> </para-num> <div class="description-line" id="p-0094" num="0107"> Having entered the appropriate parameters, the first general step of the VECTRIX program is to calculate the distance matrix of the Cα-Cβ vectors of the query (see SCHEME A). Each database entry is now read in turn and the Cα-Cβ distance matrix of the framework protein is calculated. The clique detection algorithm of Ho &amp; Marshall, 1993, supra, is used to identify geometric matches between the query and the database entry. If no match is found, another database file is read and processed. If a Hit is found, then some further processing is required because clique detection algorithm only finds the entries with Cα-Cβ vectors that match those in the query. It does not check for steric integrity, that is, the structural complementarity that each hit possesses with regard to the 3D space in which it must reside. The VECTRIX program uses the ‘steric file’ to calculate the number of atoms in the hit which invade the receptor space or the non-ligand space depending whether it is in receptor-based or ligand-based filtering mode. Some parts of the framework protein are not essential to binding to the target protein via the ‘matched’ functional residues. The non-essential part includes the side chains that are not in the matches, the N- or C-terminal residues, up to the matched residue or the first cysteine residue. The essential atoms of a residue are the backbone atoms (N, H, CA, HA, C, O) and the side chain atoms that are attached to the CA atom (CB, 1HA and 2HA). The essential residues are between the first and the last cysteine. If no cysteine is found in the protein, the essential residues are defined to be between the first and the last matched residues. The VECTRIX program counts and outputs the number of essential atoms as well as the number of essential atoms that invade the receptor or non-ligand space. Furthermore, for each subset of vector defined in the query file, the VECTRIX program counts and outputs the number of matched vectors in the subset. The results are written to an output file and another database entry is read and the process repeated until the end of the database is reached. </div>
</li> <li> <div class="description-line" id="h-0013" num="0000"> POSTVEC </div>
</li> <li> <para-num num="[0108]"> </para-num> <div class="description-line" id="p-0095" num="0108"> By design, the VECTRIX program outputs all hits that have a number of matches greater than or equal to MIN_MATCH. The POSTVEC program is written for post VECTRIX filtering. The filtering is based on the steric contact, a new number of matches and the count of match in each SUBSET defined in the query.file. The POSTVEC program requires at least three parameters, i.e. </div>
</li> <li> <para-num num="[0109]"> </para-num> <div class="description-line" id="p-0096" num="0109"> postvec vectrix_out.file min_match max_invade_fraction &lt;subset1_num&gt; &lt;subset2_num&gt; . . . &lt;subsetX_num&gt;</div>
</li> <li> <para-num num="[0110]"> </para-num> <div class="description-line" id="p-0097" num="0110"> where: 
</div> </li> <ul> <li id="ul0006-0001" num="0000"> <ul> <li id="ul0007-0001" num="0111"> the vectrix_out.file is the name of the vectrix output file. </li> <li id="ul0007-0002" num="0112"> Min_match represents the new minimum number of matches required. </li> <li id="ul0007-0003" num="0113"> the Max_invade fraction defines the maximum allowable fraction of invasion of receptor/non ligand space. That is, hits are rejected if the fraction of invasion is greater than the max_invade_frac. e.g. 0.1 for 10%. </li> <li id="ul0007-0004" num="0114"> Subset1_num represent the number of matches required for subset1. </li> <li id="ul0007-0005" num="0115"> Subset2_num represent the number of matches required for subset 2. </li> <li id="ul0007-0006" num="0116"> the bracket &lt;&gt; denote optional parameters. That is, Subset parameters are optional, if they are not defined then there is no subset filtering. 
<br/>
 The output of POSTVEC are pdbfiles of the filtered hits. These pdb files are in the same frame of reference as the query files, enabling simple display and comparison. 
<br/>
 Examination of Hits using Insight II 
</li> </ul> </li> </ul>
<li> <para-num num="[0117]"> </para-num> <div class="description-line" id="p-0098" num="0117"> An Insight II macro, EXAMINE_HIT.BCL, was written to enable easy viewing of the hits obtained from POSTVEC. Before using EXAMINE_HIT.BCL, an InsightII .psv file, EXAMINE.PSV, must be created. This file contains the ligand or the receptor in the same reference coordinate as the query vectors. It is used as the background to display the hits. Normally the ligand/receptor are set to dull colours and the query vectors are highlighted with thick lines, Cα coloured red, and Cβ coloured yellow. In Insight II, sourcing the EXAMINE_HIT.BCL file will allow for visualisation of the hits through the next and previous button, or through clicking on the filename of the hit. The hits are displayed together with the query and the receptor/ligand. Steric contacts and matched vectors are highlighted. </div>
</li> <li> <para-num num="[0118]"> </para-num> <div class="description-line" id="p-0099" num="0118"> An alternative representation of the VECTRIX program is shown in Scheme B. </div>
</li> <li> <para-num num="[0119]"> </para-num> <div class="description-line" id="p-0100" num="0119"> Alternatively, other applicable clique detection algorithms are provided by Brint &amp; Willett, 1987, J. Mol. Graphics, supra and Brint &amp; Willett, 1987, Chem. Inf. Comput. Sci, supra, which are hereinafter incorporated by reference. </div>
</li> <li> <para-num num="[0120]"> </para-num> <div class="description-line" id="p-0101" num="0120"> Using a series of automated scripts outlined in Scheme C, the database of small cysteine rich proteins is updated weekly by searching the Brookhaven database for suitable candidates. </div>
</li> <li> <para-num num="[0121]"> </para-num> <div class="description-line" id="p-0102" num="0121"> Suitably, said one or more hits correspond to respective entries identified by said algorithm according to said query. </div>
</li> <li> <para-num num="[0122]"> </para-num> <div class="description-line" id="p-0103" num="0122"> Should there be more than one hit, it is desirable to evaluate and rank each hit. The most important factor in evaluating hits is “steric integrity”, or the 3D structural complementarity of a hit when compared to a query. Several algorithms have been developed which could be utilized for this purpose. Such algorithms would include an algorithm used by the FOUNDATION program, algorithms which check van der Waals overlap of each said hit with said query (Allinger et al., 1972, supra, which is herein incorporated by reference), or algorithms which calculate volume in common and volume of extra space with respect to each said hit and said query (Marshall et al., 1979, supra, which is herein incorporated by reference). </div>
</li> <li> <para-num num="[0123]"> </para-num> <div class="description-line" id="p-0104" num="0123"> It is also contemplated that other algorithms may be useful. For example, simple distance calculations between said hit and said query after superimposition thereof may be used to identify 3D spatial differences therebetween. </div>
</li> <li> <para-num num="[0124]"> </para-num> <div class="description-line" id="p-0105" num="0124"> An outline of the process that is currently used for scoring is given in Scheme D. These procedures post process output data from the POSTVEC program, and these procedures may eventually be incorporated into the program to provide a semi-automated process. In the current filtering process, steps 1 and 2 evaluate the conformational stability of the engineered hit, and step 3 provides optimization of the fit between a receptor and hit. Note that this filtering process is described with reference to scoring hits in terms of their predicted interaction with a receptor eg. a cytokine and cytokine receptor. One skilled in the art will realize that the principles outlined in Scheme D are applicable to any protein-protein interaction. For example, when a crystal structure is not known, scoring procedures can be implemented to ensure that the hit is subsumed by the steric surface of the ligand. </div>
</li> <li> <para-num num="[0125]"> </para-num> <div class="description-line" id="p-0106" num="0125"> It is also envisaged that evaluation and ranking of each said hit may be achieved manually by a person skilled in the art, although this would be a less preferred method, particularly when there is a plurality of hits to be evaluated and ranked. </div>
</li> <li> <para-num num="[0126]"> </para-num> <div class="description-line" id="p-0107" num="0126"> In light of the foregoing, the skilled person will understand that the method of the invention provides framework protein “hits” which may be the subject of further modification. </div>
</li> <li> <para-num num="[0127]"> </para-num> <div class="description-line" id="p-0108" num="0127"> As used herein in this context, a framework protein hit has “structural similarity” to a sample protein by virtue of possessing amino acid sequence similarity, topographical similarity and/or chemical similarity. For example, a framework protein “hit” has a surface topography and/or chemistry which is similar to that of a receptor-binding region of a cytokine. Substitution of framework protein amino acids by sample protein amino acids preferably increases the degree of similarity. </div>
</li> <li> <para-num num="[0128]"> </para-num> <div class="description-line" id="p-0109" num="0128"> Preferably, a framework protein identified as a hit has greater stability than the sample protein. </div>
</li> <li> <para-num num="[0129]"> </para-num> <div class="description-line" id="p-0110" num="0129"> As used herein in this context, “stability” includes resistance to degradation by proteolytic enzymes and/or temperature variation and/or resistance to denaturation by chaotropic agents and/or denaturing detergents, changes in pH, pH extremes, and/or REDOX extremes and/or changes. </div>
</li> <li> <para-num num="[0130]"> </para-num> <div class="description-line" id="p-0111" num="0130"> It will be appreciated that the said two or more amino acids used for creating a query at step (iii) of the method of the invention constitute at least a portion of one or more functional regions of said sample protein. These amino acids may be the same as, or different to, said at least one amino acid used in modifying the hit. </div>
</li> <li> <para-num num="[0131]"> </para-num> <div class="description-line" id="p-0112" num="0131"> In one embodiment, an amino acid sequence of a framework protein which corresponds to a hit is modified by substituting at least one amino acid residue thereof with at least one amino acid residue of said sample protein. Preferably, the said at least one amino acid of the sample protein is/are selected from those required for a function of said sample protein. This engineering process can involve addition, deletion or insertion of amino acids as desired. </div>
</li> <li> <para-num num="[0132]"> </para-num> <div class="description-line" id="p-0113" num="0132"> As already discussed, the purpose of such modification is to impart a particular property, characteristic or function to a framework protein. The method of the invention takes account of the fact that the amino acid residues essential to a particular function will often be non-contiguous with respect to primary sequence. These “scattered” amino acid residues may nevertheless form at least a portion of one or more functional regions, each of which occupies a distinct location and orientation in 3D space. </div>
</li> <li> <para-num num="[0133]"> </para-num> <div class="description-line" id="p-0114" num="0133"> Advantageously, modification of the framework protein hit will be performed so as to effectively “transfer” one or more functional region(s) of the sample protein thereto. Transfer is achieved by incorporating amino acid residues from one or more functional regions (as hereinbefore defined) of the sample protein into an amino acid sequence of a framework protein. Such modification will be performed so as to engineer a protein which incorporates amino acid residues of said one or more functional region(s) appropriately located and oriented in 3D space. </div>
</li> <li> <para-num num="[0134]"> </para-num> <div class="description-line" id="p-0115" num="0134"> In an embodiment, said framework protein is modified to function as a cytokine mimetic. In this regard, modification of a framework protein may be performed so that said framework protein is capable of exhibiting a function similar to that of said sample protein (such as in the case of an agonist), or alternatively, so that it inhibits a function of said sample protein (such as in the case of an antagonist). </div>
</li> <li> <para-num num="[0135]"> </para-num> <div class="description-line" id="p-0116" num="0135"> However, the scope of the present invention extends to engineering proteins with any desired function by substituting amino acid residues of a framework protein. For example, an enzyme might be engineered to catalyze conversion of a substrate, or a transcription factor may be engineered to bind its cognate DNA sequence and to form complexes with other transcription factors necessary to promote transcription. </div>
</li> <li> <para-num num="[0136]"> </para-num> <div class="description-line" id="p-0117" num="0136"> In the case where a cytyokine mimetic is to be engineered, a suitable approach is to modify an amino acid sequence of a framework protein (corresponding to a hit) by substituting amino acid residue(s) thereof with amino acid residue(s) of said cytokine selected from those amino acid residues which are required for binding of said cytokine to a specific receptor. Often, a biological response is elicited by a cytokine binding to two or more receptor molecules, thereby cross-linking said receptor molecules. A cytokine antagonist is therefore engineered by modifying a framework protein to include amino acid residues of a functional region required for binding one receptor molecule but not the other; an agonist is engineered by including amino acid residues of two functional regions, which together are required for binding and cross-linking of two receptor molecules. The functional regions required for binding said two receptor proteins occupy unique locations and orientations in 3D space. Engineering of an agonist therefore requires that the relative 3D location and orientation of each functional region is such that receptor binding and cross-linking is achievable. </div>
</li> <li> <para-num num="[0137]"> </para-num> <div class="description-line" id="p-0118" num="0137"> In addition to direct substitution of amino acid residues of said cytokine selected from those amino acid residues which are required for binding of said cytokine to a specific receptor, several other design processes may be used. In cases where the atomic structure of the sample protein and its receptor are known, de novo design programs such as X-SITE (Laskowski et al., 1996, Journal of Molecular Biology, 175; Bohm, 1992, J. Comput. Aided. Mol. Des. 6 69, which are herein incorporated by reference) may be used to guide engineering of auxilliary binding epitopes into the hit that modulate activity. The auxilliary binding epitopes may be natural or unnatural amino acids that may be conjugated to additional functionality such as protecting groups used in synthetic peptide chemistry. </div>
</li> <li> <para-num num="[0138]"> </para-num> <div class="description-line" id="p-0119" num="0138"> Programs that measure electrostatic similarity of mutated frameworks and the sample protein or electrostatic complementarity of the mutated framework and the sample protein receptor, such as DelPhi (Honig &amp; Nicholls A, 1987, ‘DelPhi’, Computer Program, Department of Biochemistry and Molecular Biophysics Columbia University, which is herein incorporated by reference), may be employed to determine unmutated areas of the mutated framework that may be deleterious to activity. </div>
</li> <li> <para-num num="[0139]"> </para-num> <div class="description-line" id="p-0120" num="0139"> Programs that measure buried surface areas, such as Naccess (Hubbard &amp; Thornton, 1993, ‘NACCESS’, Computer Program, Department of Biochemistry and Molecular Biology, University College London, which is herein incorporated by reference) may be used to analyse and compare the buried surface areas of the sample protein and the mutated framework. </div>
</li> <li> <para-num num="[0140]"> </para-num> <div class="description-line" id="p-0121" num="0140"> Often regions in proteins may be disordered and absent from the X-Ray or NMR structure. When residues are absent in the binding region of the sample protein, techniques such as homology modelling and loop searching may be employed to construct a complete model of the atomic coordinates. </div>
</li> <li> <para-num num="[0141]"> </para-num> <div class="description-line" id="p-0122" num="0141"> Whichever approach is taken, modification of said amino acid sequence of said framework protein requires that considerations of maintaining stereochemical and secondary structural integrity apply. It is therefore important to be able to predict any structural effects induced in said framework protein by such modification. This can be accomplished with algorithms well known to the art as described in Bowie et al., 1991, Science 253 164-170; Luthy et al., 1992, Nature 356 83-85 and Laskowski et al., 1993, J. Appl. Cryst. 26 283-91. </div>
</li> <li> <para-num num="[0142]"> </para-num> <div class="description-line" id="p-0123" num="0142"> Preferably, a modified framework protein would be chemically synthesized. Alternatively, this may be achieved by chemically synthesizing a polynucleotide sequence which encodes an amino acid sequence of said modified framework protein. Techniques applicable to the chemical synthesis of proteins and nucleic acids are well known in the art, and an example of such a technique will be provided hereinafter. </div>
</li> <li> <para-num num="[0143]"> </para-num> <div class="description-line" id="p-0124" num="0143"> Alternatively, a polynucleotide sequence which encodes an amino acid sequence of a framework protein corresponding to said hit may be modified by in vitro mutagenesis techniques, resulting in a modified polynucleotide sequence encoding an amino acid sequence of said modified framework protein. Suitable in vitro mutagenesis techniques are well known in the art, such as described in Chapter 8 CURRENT PROTOCOLS IN MOLECULAR BIOLOGY (Ausubel et al., Eds; John Wiley &amp; Sons Inc., 1995), which is herein incorporated by reference. Phage display is also contemplated, which technique is well known in the art. An exemplary phage display method is provided in Smith et al., 1998, J. Mol. Biol. 277 317, which is herein incorporated by reference. </div>
</li> <li> <para-num num="[0144]"> </para-num> <div class="description-line" id="p-0125" num="0144"> According to one embodiment of the invention, each said entry in the database corresponds to a small cysteine-rich protein of not more than 70 amino acid residues, initially represented in cartesian coordinate form, but subsequently processed into a distance matrix representation of Cα-Cβ vectors prior to searching. Said query is in the form of a distance matrix representation of Cα-Cβ vectors corresponding to amino acid side-chains of said sample protein, said amino acid side-chains being required for high-affinity binding of said sample protein to a receptor protein. In a particular embodiment, the sample protein is selected from group consisting of GH, IL-4, G-CSF and IL-6. </div>
</li> <li> <para-num num="[0145]"> </para-num> <div class="description-line" id="p-0126" num="0145"> In the case where said sample protein is human Growth Hormone (hGH), and said receptor protein is human Growth Hormone Receptor (hGHR), the Cα-Cβ vectors of hGH are a simplification of the 3D location and orientation of the amino acid side-chains of hGH which contact hGHR during high-affinity binding, and are required for such binding. </div>
</li> <li> <para-num num="[0146]"> </para-num> <div class="description-line" id="p-0127" num="0146"> In this case, said small cysteine-rich protein corresponding to a hit is scyllatoxin, the amino acid sequence of which (shown in <figref idrefs="DRAWINGS">FIG. 1</figref>) is modified so that a protein produced with that amino acid sequence is potentially capable of functioning as an hGH antagonist. The particular Cα-Cβ vectors used in the search process were Asp A171; Lys A172; Glu A174; Thr A175; Phe A176; Arg A178; Ile A179; Lys A41; Leu A45; Pro A48; Glu A56; Arg A64; and Gln A68. The particular amino acid residues of hGH incorporated into the amino acid sequence of scyllatoxin were selected from those required for high-affinity binding of hGH to hGHR (as shown above) and which topographically matched with residues of scyllatoxin. Determination of which amino acids of scyllatoxin could be substituted without drastically affecting structural integrity was achieved with the assistance of the INSIGHT II modelling program. </div>
</li> <li> <para-num num="[0147]"> </para-num> <div class="description-line" id="p-0128" num="0147"> The SCY01-SCY03 peptides, designed as potential hGH antagonists, were chemically synthesised with the respective amino acid sequences shown in <figref idrefs="DRAWINGS">FIG. 1</figref>. </div>
</li> <li> <para-num num="[0148]"> </para-num> <div class="description-line" id="p-0129" num="0148"> In another case, said small cysteine-rich protein corresponding to a hit is a marine worm toxin (VIB). Said hit was identified by database searching using a query which comprised Cα-Cβ vectors of the following hGH amino acid residues: Lys A41; Leu A45; Pro A48; Glu A56; Arg A64; Gln A68; Asp A171; Lys A 172; Glu A174; Thr A175; Phe A 176; Arg A178; Ile A179; Arg A8; Leu A9; Asn A12; Leu A15; Arg A16; His A18; Arg A19; Tyr A103; Asp A116; Leu A117; Glu A119; and Thr A123. </div>
</li> <li> <para-num num="[0149]"> </para-num> <div class="description-line" id="p-0130" num="0149"> An amino acid sequence of said hit (VIB) is shown in <figref idrefs="DRAWINGS">FIG. 2</figref>, and an amino acid sequence of proteins engineered by modifying one or more amino acids of said hit (VIB01) is shown in <figref idrefs="DRAWINGS">FIG. 2</figref>. The particular amino acid residues of hGH used to modify said hit were selected from those forming the agonist-binding functional region of hGH as indicated in <figref idrefs="DRAWINGS">FIG. 2</figref>. Overlap between hGH and said marine worm toxin is shown in <figref idrefs="DRAWINGS">FIG. 3</figref>, which serves to emphasize the ability of the method of the invention to identify hits which match cytokine agonist functional regions. </div>
</li> <li> <para-num num="[0150]"> </para-num> <div class="description-line" id="p-0131" num="0150"> The peptides designed according to the hGH agonist regions consitute candidate hGH agonists. </div>
</li> <li> <para-num num="[0151]"> </para-num> <div class="description-line" id="p-0132" num="0151"> In light of the foregoing, it will be understood that the present invention contemplates engineered proteins such as according to the second-mentioned aspect. </div>
</li> <li> <para-num num="[0152]"> </para-num> <div class="description-line" id="p-0133" num="0152"> In one embodiment, the amino acids of said another protein present in the engineered protein represent at least one functional region of said another protein. </div>
</li> <li> <para-num num="[0153]"> </para-num> <div class="description-line" id="p-0134" num="0153"> In another embodiment, the amino acids of said another protein present in the engineered protein represent two functional regions of said another protein. </div>
</li> <li> <para-num num="[0154]"> </para-num> <div class="description-line" id="p-0135" num="0154"> As well as providing amino acids which are non-contiguous in primary sequence, said another protein may also provide amino acids which are contiguous in primary sequence. </div>
</li> <li> <para-num num="[0155]"> </para-num> <div class="description-line" id="p-0136" num="0155"> In one embodiment, the engineered protein has an amino acid sequence selected from the group consisting of SCY01, SCY02, SCY03, ERP01, ERP02, ERP03 and VIBO1. </div>
</li> <li> <para-num num="[0156]"> </para-num> <div class="description-line" id="p-0137" num="0156"> It will also be appreciated that according to both the first and second aspects of the invention, homologs of engineered proteins are contemplated. A person skilled in the art will realize that conservative amino acid substitutions, deletions and additions can be made such that a protein will retain a particular function notwithstanding such changes in amino acid sequence. All such homologs fall within the scope of the invention described herein. </div>
</li> <li> <para-num num="[0157]"> </para-num> <div class="description-line" id="p-0138" num="0157"> In order that the present invention may be understood in more detail, the skilled person is directed to the following non-limiting examples. </div>
</li> <heading id="h-0014">EXAMPLES </heading>
<heading id="h-0015">Example 1 </heading>
<li> <div class="description-line" id="h-0016" num="0000"> Overview of Database Search Strategy </div>
</li> <li> <para-num num="[0158]"> </para-num> <div class="description-line" id="p-0139" num="0158"> A schematic description of the computational approach developed by the present inventors, program VECTRIX, is shown in <figref idrefs="DRAWINGS">FIG. 4</figref>. The first step involves the creation of a library of small cysteine-rich proteins. Currently, 344 such proteins (each with less than 70 amino acid residues) comprising over 3779 experimentally-derived 3D structures have been extracted from the BROOKHAVEN database. However, it would also be feasible to construct databases using theoretically derived features, such as by homology modelling, threading or other techniques known in the field. </div>
</li> <li> <para-num num="[0159]"> </para-num> <div class="description-line" id="p-0140" num="0159"> Each structure is simplified, in turn, into Cα-Cβ vectors (step a), essentially resulting in a database of entries (step b). For the purposes of searching the database, each query is in the form of a distance matrix representation of Cα-Cβ vectors (step c). However, it is possible to represent Cα-Cβ vectors by other means, such as dihedral angles (δ) or α<sub>1 </sub>and α<sub>2 </sub>angles. A simple description of these types of representations with respect to a Cα-Cβ vector pair is shown in <figref idrefs="DRAWINGS">FIG. 5</figref>. </div>
</li> <li> <para-num num="[0160]"> </para-num> <div class="description-line" id="p-0141" num="0160"> The search algorithm compares the distance matrix representing the query Cα-Cβ vectors with the distance matrix representing Cα-Cβ vectors of each entry (step d). Comparison of topographical similarities was chosen because Cα-Cβ vectors are common to all amino acid side chains (except glycine), and are essentially anchored to the backbone. They therefore represent the initial orientation of the amino acid side chain in 3D space, which would probably not undergo significant change upon interaction with another protein. It is envisaged that the extra atoms of the side chain will provide some degree of induced fit during such an interaction. </div>
</li> <li> <para-num num="[0161]"> </para-num> <div class="description-line" id="p-0142" num="0161"> Alternative, more restricted approaches would use secondary structural features such as α-carbon backbone structures, together with suitable algorithms well known in the field (Holm &amp; Sander, 1994, supra; Alexandrov, 1996, supra; Alexandrov &amp; Fisher, 1996, supra; and Oreng, 1994, supra). </div>
</li> <li> <para-num num="[0162]"> </para-num> <div class="description-line" id="p-0143" num="0162"> The intermolecular geometric relationship of Cα-Cβ vectors is compared using the clique-detection algorithm of Ho &amp; Marshall, 1993, supra, which identifies hits according to a user-defined number of minimum vector components. However, other algorithms well known in the art would also be useful in this regard. </div>
</li> <li> <para-num num="[0163]"> </para-num> <div class="description-line" id="p-0144" num="0163"> As a result of step d, one or more hits may be identified. If a single hit is obtained, no ranking is necessary. If the number of hits is small, it may be possible for the skilled person to evaluate and rank each hit individually (step e). If, however, the number of hits is large, such manual comparison would be more difficult, and an automated process is required. </div>
</li> <li> <para-num num="[0164]"> </para-num> <div class="description-line" id="p-0145" num="0164"> The most important factor in evaluating and ranking hits is steric integrity, that is, the structural complementarity that each hit possesses with regard to the 3D space in which it must reside. For example, if the query is in the form of a distance matrix representation of Cα-Cβ vectors corresponding to the receptor-binding amino acid side-chains of a hormone, then a hit must be evaluated in terms of whether it would invade the 3D space accessed by the receptor upon binding the cytokine. Several algorithms have been developed that are useful for this purpose. For example, the FOUNDATION program of Ho &amp; Marshall, 1993, supra uses various flood filling algorithms to define the 3D space occupied by the receptor (as determined from the crystal structure of the receptor), and then uses atom-checking routines to establish whether the atoms of a hit reside in the binding “cavity” of the receptor. Other approaches include placing molecules in a cube containing lattice points and checking the van der Waals overlap of each molecule (Allinger, 1972, In: Pharmacology and the future of Man. Proceedings of the 5th International Conference on Pharmacology pp 57-63). A related method involves the calculation of the volume in common and the volume of extra space of two molecules (Marshall et al., 1979, The Conformational Parameter in Drug Design: The active analog approach. 112 205). </div>
</li> <li> <para-num num="[0165]"> </para-num> <div class="description-line" id="p-0146" num="0165"> It is also possible to use simple distance calculations between query and hit, after the two have been superimposed, to identify if the hit protrudes from the space occupied by the query structure. This is an approach the present inventors have implemented in an algorithm currently being constructed. </div>
</li> <li> <para-num num="[0166]"> </para-num> <div class="description-line" id="p-0147" num="0166"> It is also important to be able to predict any drastic structural effects that may result from amino acid sequence changes when modifying a hit. This will, in part, be achieved by maximizing the degree of amino acid sequence identity of the modified hit with that of the protein (or area of the protein) to which the query corresponded. In addition, the stereochemical and degree of secondary structure disruption of the modified hit can be evaluated using standard algorithms which check protein stereochemistry on an amino acid by amino acid basis. Similarly, secondary structure prediction algorithms can be used to evaluate the potential for an amino acid sequence modification of a hit to disrupt secondary structure. </div>
</li> <li> <para-num num="[0167]"> </para-num> <div class="description-line" id="p-0148" num="0167"> Finally, the present inventors plan to utilize molecular surfaces to compare various physicochemical properties of a query and hit. Charge, electrostatic potential, hydrophobicity, occupancy, and hydrogen bonding potential have all been mapped to protein surfaces, providing detailed comparisons between proteins. A method for quantitating the degree of similarity between two molecular surfaces has been developed, in which a gnomonic projection casts the calculated values of a given property onto a spherical surface (Dasnzinger &amp; Dean, 1985, J. Theor. Biol. 116 215). Two such surfaces can then be superimposed using pairs of corresponding atoms. This algorithm would be very useful for comparing query protein with a hit, to allow fine tuning of amino acid residues of the protein corresponding to the hit, and to improve steric and electrochemical complementarity. </div>
</li> <li> <para-num num="[0168]"> </para-num> <div class="description-line" id="p-0149" num="0168"> Since the database searching algorithms (such as provided by the VECTRIX program) applicable to the method of the invention allow for the identification of partial hits, there is scope for a skilled person to use molecular modelling to identify additional regions on the surface of the protein corresponding to the partial hit for mimicking vectors missed in the database search. This could involve the use of D-amino acids or non-coded amino acids, for example, to achieve better mimicry when engineering a mimetic. </div>
</li> <li> <para-num num="[0169]"> </para-num> <div class="description-line" id="p-0150" num="0169"> In the following examples, the VECTRIX program has been applied to various sample proteins. </div>
</li> <heading id="h-0017">Example 2 </heading>
<li> <div class="description-line" id="h-0018" num="0000"> High Affinity hGH Antagonists </div>
</li> <li> <para-num num="[0170]"> </para-num> <div class="description-line" id="p-0151" num="0170"> Growth hormone (GH) is a pituitary cytokine that regulates many growth processes, such as the growth and differentiation of muscle, bone and cartilage cells. The growth cytokine receptor (GHR) consists of three domains: 
</div> </li> <ul> <li id="ul0008-0001" num="0171"> (i) an extracellular domain that binds GH; </li> <li id="ul0008-0002" num="0172"> (ii) a transmembrane domain; and </li> <li id="ul0008-0003" num="0173"> (iii) a cytoplasmic domain involved in eliciting an intracellular signal upon cytokine binding. </li> </ul>
<li> <para-num num="[0174]"> </para-num> <div class="description-line" id="p-0152" num="0174"> Intracellular signalling occurs as a result of dimerization of separate GHRs following sequential binding of each receptor to a single GH ligand. The first GHR binds to the high affinity site of GH, while the second GHR subsequently binds to this complex. In support of this model, the crystal structure of this complex shows two identical receptor molecules bound to dissimilar sites on a single human GH molecule (hGH; De Vos et al., 1992, Science 255 306). </div>
</li> <li> <para-num num="[0175]"> </para-num> <div class="description-line" id="p-0153" num="0175"> The high affinity site on hGH is concave and buries approximately 1200 Å<sup>2 </sup>of surface area, while the second binding site on hGH buries approximately 900 Å<sup>2 </sup>of surface area. A third region contributing to the stability of the complex comprises an area of 500 Å<sup>2 </sup>buried by the receptor-receptor interaction. </div>
</li> <li> <para-num num="[0176]"> </para-num> <div class="description-line" id="p-0154" num="0176"> The crystal structure also reveals that the actual contact areas of both the high affinity and low affinity sites of hGH are buried upon complexation with the receptors. </div>
</li> <li> <para-num num="[0177]"> </para-num> <div class="description-line" id="p-0155" num="0177"> In developing antagonists of hGH, the present inventors have sought to design molecules that mimic the high-affinity binding of hGH. Mutagenic studies of the amino acid residues within the high affinity binding site showed a dramatic decrease in affinity when certain of these amino acid residues were converted to alanine (Cunningham &amp; Wells, 1993, 234 554). In this regard, of the 31 amino acid residues with buried side-chains, a mere eight (Lys A41; Lys A45; Pro A61; Arg A64; Lys A172; Thr A175; Phe A176; and Arg A178) accounted for approximately 85% of the total change in binding energy resulting from substitution by alanine. A further five residues (Pro A48; Glu A56; Gln A68; Asp A171; and Ile A179) essentially accounted for the remainder of the binding energy. </div>
</li> <li> <para-num num="[0178]"> </para-num> <div class="description-line" id="p-0156" num="0178"> The GH residues currently used in the design of antagonists are: Asp A171; Lys A172; Glu A174; Thr A175; Phe A176; Arg A178; Ile A179; Lys A41; Leu A45; Pro A48; Glu A56; Arg A64; and Gln A68. It is these amino acid residues of hGH which formed the basis of the query for the purposes of database searching. </div>
</li> <li> <para-num num="[0179]"> </para-num> <div class="description-line" id="p-0157" num="0179"> Scyllatoxin (pdb1scy) was returned as a hit framework that matched a maximum of 7 vectors of the hGH high affinity surface. After identification of a hit molecule, molecular modelling studies were used to optimise the hit resulting in the design of SCY01, SCY02 and SCY03. </div>
</li> <li> <para-num num="[0180]"> </para-num> <div class="description-line" id="p-0158" num="0180"> For example, molecular modelling studies (using INSIGHT II) suggested that the C-terminal His of the scyllatoxin-based mimetics could be removed as it does not interact with the receptor. This has advantages when synthesising the target molecule as His have a potential to racemise during peptide assembly. As shown in <figref idrefs="DRAWINGS">FIG. 1</figref>, the mutated framework SCY01 was produced by transfer of 7 matching hGH residues, R167, K168, D171, K172, E174, T175 and F176. Similarly SCY02 was designed by transfer of hGH residues D171, K172, E174, T175, F176, R178 and I179, however the affinity matured hGH mutation E174S was incorporated into SCY02. Similarly, SCY03 incorporated the affinity matured hGH mutations D171S and E174S. In this fashion, several analogues were designed based on a single hit, that incorporated different functional residues and affinity matured residues. </div>
</li> <li> <para-num num="[0181]"> </para-num> <div class="description-line" id="p-0159" num="0181"> In addition, molecular modelling techniques were used to optimise the amino acid functionality that was transferred to the new framework. Using the atomic structure of hGHR, X-SITE (Laskowski et al., 1996, supra) was used to predict binding sites for functional groups that could be incorporated into the hit peptide. Thus SCY13 was developed from SCY02 and SCY03 with the aid of the program X-SITE (Laskowski et al., 1996, supra), to incorporate novel mutations and auxilliary groups. As shown in <figref idrefs="DRAWINGS">FIG. 1</figref>, SCY13 possesses a D171Y mutation, a T175D mutation and an F176E(Fm) mutation. In addition, an N4R mutation in the native scyllatoxin sequence was also incorporated based on the X-SITE (Laskowski et al., 1996, supra) results. These mutations were incorporated to optimise the electrostatic interactions and to increase the bound surface area of the modelled SCY-hGHR complex. </div>
</li> <li> <para-num num="[0182]"> </para-num> <div class="description-line" id="p-0160" num="0182"> Molecular modelling studies indicated that SCY01, SCY02 and SCY03 would bury approximately 700 Å<sup>2 </sup>when bound to hGHR, whilst SCY13 would bury approximately 1000 Å<sup>2 </sup>when bound to hGHR. The modelling program DelPhi (Honig, B. &amp; Nicholls, A. (1987), ‘DelPhi’, Computer Program, Department of Biochemistry and Molecular Biophysics Columbia University) was used to compare the electrostatic potential maps of hGH and SCY peptides, with the conclusion that there was good complementarity between hGH and SCY peptides. </div>
</li> <li> <para-num num="[0183]"> </para-num> <div class="description-line" id="p-0161" num="0183"> The scyllatoxin peptides SCY01-SCY03 and SCY13 (<figref idrefs="DRAWINGS">FIG. 1</figref>) were then synthesised using solid phase techniques (M. Schnolzer et al., 1992, International Journal of Peptide and Protein Research,. 40 180-193) purified and oxidised. The products were fully characterised using mass spectrometry, high performance liquid chromatography (HPLC) and amino acid analysis (AAA). The secondary structure elements of the engineered SCY molecules were determined by circular dichroism on SCY01 and SCY02 (<figref idrefs="DRAWINGS">FIG. 6</figref>). The spectra showed a high helical content consistent with the native SCY fold. In addition, CD indicated that the helical structure was unchanged by addition of helical stabilizing agents such as TFE or destabilizing agents such as Guanidine.HCl or temperature. This emphasises the favourable chemical characteristics of these frameworks. </div>
</li> <li> <para-num num="[0184]"> </para-num> <div class="description-line" id="p-0162" num="0184"> In order to determine that the new engineered SCY framework mimics the structure of the region of GH used as a query, the structure of SCY01 was determined by NMR spectroscopy. As illustrated in <figref idrefs="DRAWINGS">FIG. 7</figref>, we found that their is close conformational overlap (RMS 0.45 Å) between the functional residues on GH and the engineered surface of SCY01. Thus validating the process of selecting a target protein, simplifying the functional epitope into Cα-Cβ vectors, using these as a query to identify new frameworks that match the shape of this query, synthesising, characterising and folding the new engineered framework. The resulting new engineered framework structurally matches that of the functional epitope of the target protein, thus validating the design process. </div>
</li> <li> <para-num num="[0185]"> </para-num> <div class="description-line" id="p-0163" num="0185"> In order to characterise the folding patterns of SCY02 and SCY03 NMR experiments were again carried out. However, this time the secondary shifts were compared (Wishart et al., J. Biomolecular NMR 5 67) between the engineered and native SCY. As expected there is little or no deviation in the CHα or NHα shifts compared to the native SCY molecule indicating the correct fold and disulphide bond connectivity. </div>
</li> <li> <para-num num="[0186]"> </para-num> <div class="description-line" id="p-0164" num="0186"> SCY01 was tested for biological function by bioassay using the BaF3 cell line, which cells normally respond to GH. The results are shown in <figref idrefs="DRAWINGS">FIG. 8</figref>. SCY01 was assayed at various concentrations to check its ability to inhibit BaF3 cell proliferation in response to either 0.5 ng/mL hGH, or as a control, 50 Units/mL IL-3. The calculated K<sub>i </sub>from these experiments was approximately 200 μM, and no inhibitory activity was observed with respect to IL-3 induced proliferation. Thus, SCY01 displayed an inhibitory activity with respect to GH-stimulated proliferation. This biological effect suggests that SCY01 is a candidate for further investigation with regard to it's mechanism of action. </div>
</li> <li> <para-num num="[0187]"> </para-num> <div class="description-line" id="p-0165" num="0187"> The SCY peptides showed extremely good stability in the hGH assay buffer as judged by HPLC of the peptide at various time points after incubation in the assay buffer for up to 72 hrs. Preliminary studies evaluated the bioavailability of SCY01 by exposing it to a variety of proteases (trypsin, chymotrypsin and pepsin) and blood serum proteins as described in MATERIALS AND METHODS. The results of the blood serum stability test are presented in Table 2, and the results of the enzyme stability tests are presented in Table 3. The SCY peptide was found to be stable after 24 hrs in each case, while control peptides were rapidly digested. Thus emphasising the favourable chemical characteristics of disulfide-rich proteins. </div>
</li> <li> <para-num num="[0188]"> </para-num> <div class="description-line" id="p-0166" num="0188"> In this example the present inventors have taken a functional epitope of hGH and successfully engineered it onto a new disulfide-rich framework. This framework has appealing chemical characteristics in terms of bioavailability and bioactivity when compared to macromolecular proteins. </div>
</li> <li> <div class="description-line" id="h-0019" num="0000"> Experimental to Example 2 </div>
</li> <li> <div class="description-line" id="h-0020" num="0000"> Vectrix Results </div>
</li> <li> <para-num num="[0189]"> </para-num> <div class="description-line" id="p-0167" num="0189"> Number of vectors searched: 15—R167, K168, D171, K172, E174, T175, F176, R178, I179, K45, P48, E56, R64, Q68. </div>
</li> <li> <div class="description-line" id="h-0021" num="0000"> Number of Different Frameworks Selected (Name:pdb Code Number Vector Matches): </div>
</li> <li> <div class="description-line" id="h-0022" num="0000"> Scyllatoxin: pdb1scy (7) </div>
</li> <li> <div class="description-line" id="h-0023" num="0000"> Synthesis </div>
</li> <li> <para-num num="[0190]"> </para-num> <div class="description-line" id="p-0168" num="0190"> As described in the General Materials and Methods section. The peptides were fully characterized by mass spectrometry, Reverse Phase High Performance Liquid chromatography (RP-HPLC) and Amino acid analysis (AAA). </div>
</li> <li> <div class="description-line" id="h-0024" num="0000"> Folding </div>
</li> <li> <para-num num="[0191]"> </para-num> <div class="description-line" id="p-0169" num="0191"> The pure reduced peptides SCY 01-03 were folded using 0.1M solution of NH<sub>4</sub>HCO<sub>3 </sub>stirred overnight at RT at a peptide concentration of ˜0.3 μM per ml monitored by HPLC and mass spectrometry. The folded peptide was isolated by preparative HPLC. The correct disulphide connectivity for SCY01 was determined by full structure analysis by NMR. Folding methods using oxidized and reduced glutathione in a ratio of 100:10:1 GSH:GSSG: peptide and published methods using 5 mM GSSG to 0.5 mM GSH in NaPO<sub>4 </sub>buffer pH 7.4 was carried out to give identically folded material. After folding the pure peptide an equivalent yield of peptide was obtained by folding the crude peptide in exactly the same manner. The oxidation of SCY13 was complicated by the Fm group attached to the Glu. SCY13 was oxidised using a 30% TFE solution in the presence of 5 mM GSSG to 0.5 mM GSH in NaPO<sub>4 </sub>buffer pH 7.4. </div>
</li> <li> <div class="description-line" id="h-0025" num="0000"> Circular Dichroism (CD) </div>
</li> <li> <para-num num="[0192]"> </para-num> <div class="description-line" id="p-0170" num="0192"> CD was performed as outlined in the General Materials and Methods section. </div>
</li> <li> <div class="description-line" id="h-0026" num="0000"> NMR </div>
</li> <li> <para-num num="[0193]"> </para-num> <div class="description-line" id="p-0171" num="0193"> The NMR structure of SCY01 and the CHα and NHα connectivities were determined as outlined in the General Materials and Methods section. </div>
</li> <li> <div class="description-line" id="h-0027" num="0000"> Peptide Stability Tests </div>
</li> <li> <div class="description-line" id="h-0028" num="0000"> Stability in Assay Buffer </div>
</li> <li> <para-num num="[0194]"> </para-num> <div class="description-line" id="p-0172" num="0194"> The SCY peptides showed extremely good stability in the hGH assay buffer (RPMI-1640 medium supplemented with 10% (v/v) foetal bovine serum (FBS) and 100 units/mL IL3. The peptides were incubated at 1 mg/ml solutions in the buffer at 37° C. Samples were removed at various time points and HPLC analysis showed the rate of peptide decomposition up to 72 hrs. </div>
</li> <li> <div class="description-line" id="h-0029" num="0000"> Blood Serum </div>
</li> <li> <para-num num="[0195]"> </para-num> <div class="description-line" id="p-0173" num="0195"> Blood was collected in heparinised tubes by venapuncture. The blood was centrifuged at 5000 rpm for 20 mins and the serum decanted. The blood serum was stored at −20° C. A sample of the blood serum (900 μL) was incubated with 100 μL of the stock peptide solution (I mg/mL in H<sub>2</sub>O) at 37° C. and aliquots (100 μL) removed at the required time. A solution of 50% CH<sub>3</sub>CN 0.1% TFA was added to precipitate the blood serum proteins and centrifuged at 13000 rpm for 5 mins. A sample of this solution (100 μL) was analysed by RP-HPLC (Vydac C18 218TP54 250×4.1 mm id 1%/min gradient H<sub>2</sub>O/CH<sub>3</sub>CN 0.1% TFA) to detect peptide digestion. </div>
</li> <li> <div class="description-line" id="h-0030" num="0000"> Enzyme Stability Test </div>
</li> <li> <div class="description-line" id="h-0031" num="0000"> Trypsin </div>
</li> <li> <para-num num="[0196]"> </para-num> <div class="description-line" id="p-0174" num="0196"> To the peptide solution (NH<sub>4</sub>HCO<sub>3</sub>, pH 8.3, 0.87 mg/mL) was added trypsin (5% w:v). Samples were incubated at 37° C. and aliquots removed at 0, 1, 3 and 18 hrs and analysed by RP-HPLC as above. </div>
</li> <li> <div class="description-line" id="h-0032" num="0000"> Chymotrypsin </div>
</li> <li> <para-num num="[0197]"> </para-num> <div class="description-line" id="p-0175" num="0197"> To the stock peptide solution (100 μL) was added 900 μL NH<sub>4</sub>HCO<sub>3 </sub>(pH 8.3). Chymotrypsin was added to 5% w:v and incubated at 37° C. Aliquots were removed at 0 hr. 1 hr and 24 hrs and analysed by RP HPLC. </div>
</li> <li> <div class="description-line" id="h-0033" num="0000"> Pepsin </div>
</li> <li> <para-num num="[0198]"> </para-num> <div class="description-line" id="p-0176" num="0198"> To the stock peptide solution (100 μL) was added H<sub>2</sub>O (800 μL) and 0.1 M HCl (100 μL) to pH 2.2. Pepsin was added to give a 1% w:v solution and incubated at 37° C. Aliquots were removed at 0 h, 1 h and 24 hrs and analysed by RP-HPLC. </div>
</li> <heading id="h-0034">Example 3 </heading>
<li> <div class="description-line" id="h-0035" num="0000"> Growth Hormone—Low Affinity Site </div>
</li> <li> <para-num num="[0199]"> </para-num> <div class="description-line" id="p-0177" num="0199"> The low affinity site of growth hormone comprises at least 12 residues. The Cα-Cβ vectors of these 12 residues were used in a VECTRIX search. Pdb1zdc (ZDC) was returned as the best hit with 9 search vectors matched at 1 Å tolerance. These residues were R8, L9, D11, N12, L15, R16, R19, D116 and E119. Molecular modelling (Insight II) was again used to optimise the hit. It was decided that the R29L (matching L9 of hGH) may disrupt the ZDC fold and this mutation was not incorporated. Furthermore, additional molecular modelling studies suggested that ZDC could match a further 7 residues of hGH. The residues that matched (15 residues—RMSd backbone atoms between hit and hGH—1.46 angstroms) and were incorporated into ZDC05 were, R8, D11, N12, L15, R16, R19, Y111, D112, K115, D116, E118, E119, G120, Q122 and T123. As shown in <figref idrefs="DRAWINGS">FIG. 9</figref>, the mutated framework ZDC05 was produced by transfer of the above 15 matching hGH residues. </div>
</li> <li> <div class="description-line" id="h-0036" num="0000"> Experimental to Example 3 </div>
</li> <li> <div class="description-line" id="h-0037" num="0000"> Vectrix Results. </div>
</li> <li> <para-num num="[0200]"> </para-num> <div class="description-line" id="p-0178" num="0200"> Number of vectors searched: 12—R8, L9, D11, N12, L15, R16, R19, D112, L113, D116, E119, T123. </div>
</li> <li> <div class="description-line" id="h-0038" num="0000"> Number of different matches at 7 or more vector matches: 22 </div>
</li> <li> <div class="description-line" id="h-0039" num="0000"> Number of unique frameworks at 7 or more vector matches: 6 </div>
</li> <li> <div class="description-line" id="h-0040" num="0000"> Number of Different Frameworks Selected (Name:pdb Code Number Vector Matches): </div>
</li> <li> <div class="description-line" id="h-0041" num="0000"> Protein A engineered fragment: pdb1zdc (9) </div>
</li> <heading id="h-0042">Example 4 </heading>
<li> <div class="description-line" id="h-0043" num="0000"> Growth Hormone Agonist I </div>
</li> <li> <para-num num="[0201]"> </para-num> <div class="description-line" id="p-0179" num="0201"> The agonist site of hGH comprises 25 residues. The Cα-Cβ vectors of these 25 residues were used in a VECTRIX search. Pdb1vib (VIB) was returned as the best hit with 8 search vectors matched. These residues were N12, R16, R19, D171, K172, E174, T175 and F176. Molecular modelling determined that VIB could match a further 9 residues of hGH. The residues that matched (17 residues—RMSd backbone atoms between hit and hGH—0.86 angstroms) and incorporated into VIB01 were D11, N12, R16, R19, L20 H21, Q22, L23, F25, R167, K168, D169, D171, K172, E174, T175 and F176. As shown in <figref idrefs="DRAWINGS">FIG. 2</figref>, the mutated framework VIB01 was produced by transfer of the above 17 matching hGH residues. </div>
</li> <li> <para-num num="[0202]"> </para-num> <div class="description-line" id="p-0180" num="0202"> The modelling program Delphi (Honig &amp; Nicholls, 1987, supra) was used to compare the electrostatic potential maps of hGH and the mimics, with the conclusion that there was good complimentarity between hGH and the mimics. </div>
</li> <li> <para-num num="[0203]"> </para-num> <div class="description-line" id="p-0181" num="0203"> With the aid of molecular mechanics forcefield minimisations and molecular dynamics, VIB01 was determined to position the mutated residues in appropriate spatial orientations to mimic hGH and to retain the native fold. </div>
</li> <li> <para-num num="[0204]"> </para-num> <div class="description-line" id="p-0182" num="0204"> The VIB peptide (<figref idrefs="DRAWINGS">FIG. 2</figref>) was synthesised using solid phase techniques (M. Schnolzer et al., International Journal of Peptide and Protein Research, supra), purified and oxidised. The product was fully characterised using mass spectrometry HPLC and AAA. The secondary structure elements of the engineered VIB molecules was checked by circular dichroism as illustrated in <figref idrefs="DRAWINGS">FIG. 10</figref>. The engineered VIB peptide had a very stable structure and shows significant helical character in aqueous conditions. This would be expected as the native fold is a helix loop helix motif. </div>
</li> <li> <para-num num="[0205]"> </para-num> <div class="description-line" id="p-0183" num="0205"> In addition, the VECTRIX search identified peptide ERP as a hit with 7 search vectors matched. These residues were N12, L15, R16, H18, R19, T175 and R178. Molecular modelling determined that ERP could match a further 6 residues of hGH. The residues that matched (13 residues—RMSd backbone atoms between hit and hGH—1.33 angstroms) and were incorporated into ERP01 were R8, D11, N12, M14, L15, R16, H18, R19, E174, T175, F176, R178 and I179. As shown in <figref idrefs="DRAWINGS">FIG. 11</figref>, the mutated framework ERP01 was produced by transfer of the above 13 matching hGH residues. </div>
</li> <li> <para-num num="[0206]"> </para-num> <div class="description-line" id="p-0184" num="0206"> The modelling program DelPhi (Honig &amp; Nicholls, 1987, supra) was used to compare the electrostatic potential maps of hGH and the mimics, with the conclusion that there was good complimentarity between hGH and the mimics. </div>
</li> <li> <para-num num="[0207]"> </para-num> <div class="description-line" id="p-0185" num="0207"> With the aid of molecular mechanics forcefield minimisations and molecular dynamics, ERP01 was determined to position the mutated residues in appropriate spatial orientations to mimic hGH and to retain the native fold. </div>
</li> <li> <para-num num="[0208]"> </para-num> <div class="description-line" id="p-0186" num="0208"> ERP02 differed from ERP01 in containing the hGH affinity matured mutations E174S, I179T and H18D. The G14F mutation (F176 mimic) in ERP01 and ERP02 necessitated two major mutations, S6G and N11G. ERP03 eliminated the G14F mutation and the necessity for these mutations giving a less perturbed sequence. </div>
</li> <li> <para-num num="[0209]"> </para-num> <div class="description-line" id="p-0187" num="0209"> The ERP peptides 01-03 (<figref idrefs="DRAWINGS">FIG. 11</figref>) were synthesised using solid phase techniques (M. Schnolzer et al., International Journal of Peptide and Protein Research, supra), purified and oxidised. The product was fully characterised using mass spectrometry HPLC and AAA. The secondary structure elements of the engineered ERP molecules was checked by circular dichroism on ERP03 (<figref idrefs="DRAWINGS">FIG. 12</figref>). This showed a very high degree of alpha helical character in agreement with the 3 helical bundle structure of the native ERP molecule. </div>
</li> <li> <para-num num="[0210]"> </para-num> <div class="description-line" id="p-0188" num="0210"> NMR of ERP01 and ERP03 was carried out to check that the 3 disulfide bonds have formed correctly. As expected there is only small deviation from the native ERP molecule where the mutations to mimic the hGH molecule are made (<figref idrefs="DRAWINGS">FIG. 13</figref> for ERP03). There is little or no deviation in the CHα or in the NHα shifts compared to the native ERP molecule indicative of the correct folding and disulphide bond connectivity, once again emphasing the ability to engineer new surfaces onto disulfide rich peptides, whilst maintaining the native fold. </div>
</li> <li> <div class="description-line" id="h-0044" num="0000"> Experimental to Example 4: VIB </div>
</li> <li> <div class="description-line" id="h-0045" num="0000"> Vectrix Results </div>
</li> <li> <para-num num="[0211]"> </para-num> <div class="description-line" id="p-0189" num="0211"> Number of vectors searched: 25—R8, L9, N12, L15, R16, H18, R19, K41, L45, P45, E56, R64, Q68, Y103, D116, L117, E119, T123, D171, K172, E174, T175, F176, R178 and I179. </div>
</li> <li> <div class="description-line" id="h-0046" num="0000"> Number of Different Matches </div>
</li> <li> <div class="description-line" id="h-0047" num="0000"> 61292 at minimum 5 vector matches </div>
</li> <li> <div class="description-line" id="h-0048" num="0000"> Number of Unique Frameworks </div>
</li> <li> <div class="description-line" id="h-0049" num="0000"> 10 at minimum 7 vectors, 1 at minimum 8 vectors </div>
</li> <li> <div class="description-line" id="h-0050" num="0000"> Number of Different Frameworks Selected (Name:pdb Code: # Vector Matches) </div>
</li> <li> <div class="description-line" id="h-0051" num="0000"> Marine worm neurotoxin: pdb1vib (8) </div>
</li> <li> <div class="description-line" id="h-0052" num="0000"> Peptide Synthesis </div>
</li> <li> <para-num num="[0212]"> </para-num> <div class="description-line" id="p-0190" num="0212"> Synthesis of the VIB peptides was as described in the General Materials and Methods section. </div>
</li> <li> <div class="description-line" id="h-0053" num="0000"> Oxidation of the VIB Peptides </div>
</li> <li> <para-num num="[0213]"> </para-num> <div class="description-line" id="p-0191" num="0213"> The reduced VIB peptides were oxidsied using the methods outlined for the ERP peptides with 30% TFE solutions and GSSG: GSH oxidation shuttle. </div>
</li> <li> <div class="description-line" id="h-0054" num="0000"> Circular Dichroism </div>
</li> <li> <para-num num="[0214]"> </para-num> <div class="description-line" id="p-0192" num="0214"> CD was performed as outlined in the General Materials and Methods section. </div>
</li> <li> <div class="description-line" id="h-0055" num="0000"> Experimental to Example 4: ERP Molecule </div>
</li> <li> <div class="description-line" id="h-0056" num="0000"> Synthesis of ERP Peptides </div>
</li> <li> <para-num num="[0215]"> </para-num> <div class="description-line" id="p-0193" num="0215"> As described in the General Materials and Methods section. </div>
</li> <li> <div class="description-line" id="h-0057" num="0000"> Folding of ERP Peptides </div>
</li> <li> <para-num num="[0216]"> </para-num> <div class="description-line" id="p-0194" num="0216"> The peptide was dissolved at a low concentration in cold water to which was added trifluoroethanol to 30%. This was cooled at 4° C. for two hours before oxidised and reduced glutathione was added (10:100:1/GSSG:GSH:peptide) then 1M NH<sub>4</sub>HCO<sub>3 </sub>was added to give a 0.1 M solution at pH 8.1. The oxidised peptides were isolated by HPLC. </div>
</li> <li> <div class="description-line" id="h-0058" num="0000"> Circular Dichroism </div>
</li> <li> <para-num num="[0217]"> </para-num> <div class="description-line" id="p-0195" num="0217"> CD was performed as outlined in the General Materials and Methods section. </div>
</li> <li> <div class="description-line" id="h-0059" num="0000"> NMR of ERP01 and 03 </div>
</li> <li> <para-num num="[0218]"> </para-num> <div class="description-line" id="p-0196" num="0218"> The NMR structure of ERP01 and ERP03 and the Cα-Cβ and Cα-NHα connectivities were determined as outlined in the General Materials and Methods section. </div>
</li> <heading id="h-0060">Example 5 </heading>
<li> <div class="description-line" id="h-0061" num="0000"> Interleukin 4 (IL-4) </div>
</li> <li> <para-num num="[0219]"> </para-num> <div class="description-line" id="p-0197" num="0219"> IL-4 is a four helix bundle cytokine that is the basis of the allergic response mechanisms in asthma, rhinitis, conjunctivitis and dermatitis. It plays an important role in the induction of immunoglobulins through the turning on of B-cells that produce Igm, IgE and IgG's. IL-4 associates primarily with the IL-4 alpha receptor which accounts for nearly the complete binding affinity. The IL-4 receptor complex then recruits the common γ chain to form the cell signaling heterodimer. </div>
</li> <li> <para-num num="[0220]"> </para-num> <div class="description-line" id="p-0198" num="0220"> The functional epitope of IL-4 that determines the binding affinity to the receptor α chain has been identified through mutational analysis and from the crystal structure of the recently determined IL-4 and the IL-4Rα complex. (Hage et al., 1999, Cell 97 271) The key binding event involves mainly charged residues from helix A and C of IL-4 particularly Arg88 and Glu9. </div>
</li> <li> <para-num num="[0221]"> </para-num> <div class="description-line" id="p-0199" num="0221"> The 13 amino acid residues of the binding surface of IL-4 were used as a query for program VECTRIX. In this case the database to be searched contained the structure of GCN4, a 31 residue leucine zipper peptide. The GCN4 molecule was identified by the program VECTRIX as a hit. It matched 8 vectors of IL-4 (RMS 0.39 Å). Upon engineering and synthesising this molecule containing these 8 amino acids, an IL-4 agonist is expected with a potency of Kd 106 μM (Dominques et al., 1999, Nat. Struct. Biol. 6 652) </div>
</li> <li> <para-num num="[0222]"> </para-num> <div class="description-line" id="p-0200" num="0222"> An additional molecule ZDC was found that matches 10 vectors. Upon synthesising the engineered framework it will be folded and assayed. </div>
</li> <li> <div class="description-line" id="h-0062" num="0000"> Vectrix Results </div>
</li> <li> <para-num num="[0223]"> </para-num> <div class="description-line" id="p-0201" num="0223"> Number of vectors searched: 13: K77, R81, K84, R85, R88, N89, W91, T13, E9, I5, R53, F82, K12 </div>
</li> <li> <div class="description-line" id="h-0063" num="0000"> Total Number of Different Matches at 7 or More </div>
</li> <li> <div class="description-line" id="h-0064" num="0000"> 396 </div>
</li> <li> <div class="description-line" id="h-0065" num="0000"> Number of Unique Frameworks </div>
</li> <li> <div class="description-line" id="h-0066" num="0000"> 30 </div>
</li> <li> <div class="description-line" id="h-0067" num="0000"> Number of Different Frameworks Selected (Name:pdb Code: # Vector Matches) </div>
</li> <li> <div class="description-line" id="h-0068" num="0000"> GCN4 peptide: pdb1zta (8) </div>
</li> <li> <div class="description-line" id="h-0069" num="0000"> Protein A fragment (engineered): pdb1zdc: (10) </div>
</li> <li> <div class="description-line" id="h-0070" num="0000"> N.B. No molecule selected in the search matched to Arg53. </div>
</li> <heading id="h-0071">Example 6 </heading>
<li> <div class="description-line" id="h-0072" num="0000"> CD4 GP120 </div>
</li> <li> <para-num num="[0224]"> </para-num> <div class="description-line" id="p-0202" num="0224"> The CD4-GP120 interaction is the primary binding event that allows the Human Immunodeficiency Virus (HIV) to enter a cell. The crystal structure of CD4 has been known for some time (Wang et al., 1990, Nature 348 411) but a structure of the CD4 and a highly modified GP120 complex was only solved in June 1998 (Kwong et al., 1998, Nature 393 648). It has been known for some time through mutational analysis of CD4 (Fleury et al., 1991, Cell 66 1037) that the key amino acids involved in binding to GP120 reside on a loop (CDR1) involving the residues 41-47 and the key binding residue Arg59. </div>
</li> <li> <para-num num="[0225]"> </para-num> <div class="description-line" id="p-0203" num="0225"> The Cα-Cβ vectors of these residues were used in a VECTRIX search. Two molecules SCY and PTA (<figref idrefs="DRAWINGS">FIG. 14</figref>) were identifed as potential matches. Both molecules were optimised using a design procedure as described above. </div>
</li> <li> <para-num num="[0226]"> </para-num> <div class="description-line" id="p-0204" num="0226"> The biological activity of SCY is consistent with the studies of Vita et al., 1998, Biopolymer 47 93. </div>
</li> <li> <div class="description-line" id="h-0073" num="0000"> Experimental for Example 6 </div>
</li> <li> <div class="description-line" id="h-0074" num="0000"> Vectrix Results </div>
</li> <li> <para-num num="[0227]"> </para-num> <div class="description-line" id="p-0205" num="0227"> Number of vectors searched: 7: K35, S42, F43, R59, D63, Q40, L44. </div>
</li> <li> <div class="description-line" id="h-0075" num="0000"> Total Number of Different Matches </div>
</li> <li> <div class="description-line" id="h-0076" num="0000"> At 4 or more matches. 409 </div>
</li> <li> <div class="description-line" id="h-0077" num="0000"> Number of Unique Frameworks </div>
</li> <li> <div class="description-line" id="h-0078" num="0000"> 116 </div>
</li> <li> <div class="description-line" id="h-0079" num="0000"> Number of Different Frameworks Selected (Name:pdb Code: # Vector Matches) </div>
</li> <li> <div class="description-line" id="h-0080" num="0000"> Scorpion neurotoxin: pdb2pta ( 5) </div>
</li> <li> <div class="description-line" id="h-0081" num="0000"> Scyllatoxin: pdb1scy: (4) </div>
</li> <li> <div class="description-line" id="h-0082" num="0000"> The scy molecule is only selected in the vectrix search if the absolute requirement of a match with Arg59 is removed. </div>
</li> <li> <div class="description-line" id="h-0083" num="0000"> Synthesis of PTACD4 and SCYCD4 Molecules </div>
</li> <li> <para-num num="[0228]"> </para-num> <div class="description-line" id="p-0206" num="0228"> As described in the General Materials and Methods section. </div>
</li> <li> <div class="description-line" id="h-0084" num="0000"> Oxidation of PTACD4 Molecule </div>
</li> <li> <para-num num="[0229]"> </para-num> <div class="description-line" id="p-0207" num="0229"> The PTA peptide was oxidised by stirring the peptide overnight in 0.1M NH<sub>4</sub>HCO<sub>3 </sub>pH 8.1. The oxidised peptide (2 forms) was recovered by HPLC. Both folded forms were assayed separately. The oxidation of the peptide in different conditions in the presence of glutathione failed to yield folded peptide. </div>
</li> <li> <div class="description-line" id="h-0085" num="0000"> Oxidation of SCY Molecule </div>
</li> <li> <para-num num="[0230]"> </para-num> <div class="description-line" id="p-0208" num="0230"> The SCY CD4 molecule was oxidised using 5 mM GSSG to 0.5 mM GSH in NaPO4 buffer pH 7.4. The oxidised peptide was purified by HPLC. </div>
</li> <li> <div class="description-line" id="h-0086" num="0000"> Biacore Assay </div>
</li> <li> <para-num num="[0231]"> </para-num> <div class="description-line" id="p-0209" num="0231"> GP120 bound to the Biacore chip through NHS coupling onto a CM-5 Biacore chip. CD4 is then passed over the GP120 surface and the degree of binding assessed through both the on rate K<sub>Association </sub>and the off rate k<sub>Dissociation</sub>. CD4 is then equilibrated with the inhibitor ligand and passaged over the GP120. Through the BiaCore module the degree to which the PTA or SCY ligand disrupts the binding of CD4 to the chip is assessed. </div>
</li> <heading id="h-0087">Example 7 </heading>
<li> <div class="description-line" id="h-0088" num="0000"> Interleukin 6 (IL-6) </div>
</li> <li> <para-num num="[0232]"> </para-num> <div class="description-line" id="p-0210" num="0232"> Interleukin 6 (IL-6) is a cytokine that plays an important role in the inflamation cascade, neural development, bone metabolism, hematopoiesis cell proliferation and immune response mechanisms. Interleukin 6 is a 4 helical bundle cytokine that binds to a IL-6 alpha receptor and to a common receptor motif GP130. The IL-6R α subunit does not play a role in intracellular signalling. This is carried out through the ligand dependent dimerisation of the associated GP130 receptor molecule. The full receptor complex is believed to be hexameric with two units each of IL-6, IL-6R and GP130. The pleiotropic effects of IL-6 is thought to come about because of this complex arrangement of the heterotrimeric receptor complex. The interaction sites for both the IL-6Rα and GP130 receptors has been well studied through site specific mutagenisis of both the receptor molecules and the IL-6 molecule. The structure of IL-6 in both solution and crystal forms has been solved and the crystal structure of the GP130 receptor has recently been determined. </div>
</li> <li> <para-num num="[0233]"> </para-num> <div class="description-line" id="p-0211" num="0233"> The IL-6α receptor binding site on IL-6 (termed Site I) is localised primarily to the end of helix D. Two additional sites Site II and III are responsible for the two different GP130 receptor molecules binding. The two GP130 binding sites are spread over a wide area at the opposite end of the molecule to the IL-6 binding site. </div>
</li> <li> <para-num num="[0234]"> </para-num> <div class="description-line" id="p-0212" num="0234"> The IL-6 VECTRIX search described herein pertains only to the II-6α receptor interaction. It does not relate to the GP130 receptor interaction or the multi receptor interactions (though the VECTRIX search has been carried out for these two sites II and III as well). No modeling of the IL-6 residues to any of the hit frameworks has been carried out. A few examples of possible framework targets are listed below. </div>
</li> <li> <div class="description-line" id="h-0089" num="0000"> Vectrix Results </div>
</li> <li> <para-num num="[0235]"> </para-num> <div class="description-line" id="p-0213" num="0235"> Number of vectors searched: 21 Subset1 (Site I) 8 vectors: Subset 2 (Site II and III) 13 vectors. </div>
</li> <li> <div class="description-line" id="h-0090" num="0000"> Number of Different Matches at 8 and Above Matches for Site I </div>
</li> <li> <div class="description-line" id="h-0091" num="0000"> 179 </div>
</li> <li> <div class="description-line" id="h-0092" num="0000"> Number of Unique Frameworks </div>
</li> <li> <div class="description-line" id="h-0093" num="0000"> 29 </div>
</li> <li> <div class="description-line" id="h-0094" num="0000"> Number of Different Frameworks Selected (Name:pdb Code: # Vector Matches) </div>
</li> <li> <div class="description-line" id="h-0095" num="0000"> Protein A fragment (engineered): pdb1zdc: (9) </div>
</li> <li> <div class="description-line" id="h-0096" num="0000"> Moloney murine leukemia virus fragment: Pdb1mof: (10) </div>
</li> <li> <div class="description-line" id="h-0097" num="0000"> Scyllatoxin: pdb1scy: (8) </div>
</li> <heading id="h-0098">Example 8 </heading>
<li> <div class="description-line" id="h-0099" num="0000"> G-CSF </div>
</li> <li> <para-num num="[0236]"> </para-num> <div class="description-line" id="p-0214" num="0236"> Granulocyte Colony Stimulating Factor (G-CSF) is part of the class of 4 helical bundle cytokine or growth factors. It is involved in the promotion of cell proliferation and differentiation leading to the production of mature neutrophils. Its ability to replenish these neutrophils in-vivo makes it an attractive drug target. G-CSF functions through receptor dimerisation of the CSF receptor. There has been alanine scanning mutagenisis carried out on G-CSF to identify the key residues involved in receptor recognition. The crystal structure of G-CSF has been available since 1993 (Hill et al., 1993, Proceedings of the National Acadamy of Science USA 90 5167) and the NMR structure since 1994 (Zink et al., 1994, Biochemistry 33 8453). </div>
</li> <li> <para-num num="[0237]"> </para-num> <div class="description-line" id="p-0215" num="0237"> The VECTRIX search was done with an absolute requirement for a vector matching the critical amino acid Phe 145. However, relatively few hits resulted, presumably due to the restriction of every hit matching the Phe 145 vector. Alterations of this absolute requirement and refinement of the VECTRIX search will lead to a larger number of hits. </div>
</li> <li> <div class="description-line" id="h-0100" num="0000"> Experimental to Example 8 </div>
</li> <li> <div class="description-line" id="h-0101" num="0000"> Vectrix Results </div>
</li> <li> <para-num num="[0238]"> </para-num> <div class="description-line" id="p-0216" num="0238"> Number of vectors searched: 18 </div>
</li> <li> <div class="description-line" id="h-0102" num="0000"> Number of Different Matches </div>
</li> <li> <para-num num="[0239]"> </para-num> <div class="description-line" id="p-0217" num="0239"> 338 </div>
</li> <li> <div class="description-line" id="h-0103" num="0000"> Number of Unique Frameworks </div>
</li> <li> <div class="description-line" id="h-0104" num="0000"> 115 </div>
</li> <li> <div class="description-line" id="h-0105" num="0000"> Number of Different Frameworks Selected (Name:pdb Code: # Vector Matches) </div>
</li> <li> <para-num num="[0240]"> </para-num> <div class="description-line" id="p-0218" num="0240"> Further refinement of the vectrix search is needed before a selection as to probable ligand frameworks. </div>
</li> <heading id="h-0106">GENERAL MATERIALS &amp; METHODS </heading>
<li> <div class="description-line" id="h-0107" num="0000"> Design </div>
</li> <li> <para-num num="[0241]"> </para-num> <div class="description-line" id="p-0219" num="0241"> Database searching and all design steps were carried out on either an R10000 or R12000 SGI Octane workstation. Database searching was performed with VECTRIX. Visualisation and peptide mutations and modifications were performed using Software programs from Biosym/MSI of San Diego-InsightII and Biopolymer respectively. Analysis of electrostatic potential character of the molecules was carried out using Biosym/MSI of San Diego-DelPhi, while surface area calculations were performed with Naccess (Hubbard &amp; Thornton, 1993, ‘NACCESS’, Computer Program, Department of Biochemistry and Molecular Biology, University College London) Molecular mechanics minimisations and molecular dynamics calculations were performed on the mutated frameworks to determine whether the native fold was retained. Programs such as X-SITE (Laskowski et al., 1996 Journal of Molecular Biology, p175-201) were used to add additional functionality to the mutated peptides. </div>
</li> <li> <div class="description-line" id="h-0108" num="0000"> Chemicals and Reagents </div>
</li> <li> <para-num num="[0242]"> </para-num> <div class="description-line" id="p-0220" num="0242"> Trifluoroacetic acid (TFA) dichloromethane (DCM) dimethylformamide (DMF) and disopropylethylamine (DIEA) were from Auspep (Melbourne Australia). 2-(1H-benzotriazol-1-yl)-1,1,3,3-tetramethyl uronium hexafluorophosphate (HBTU) was from Richelieu Biotechnologies (St. Hyacinth, Quebec, Canada). Acetonitrile was from BDH Laboratory Supplies (Poole, U.K.), Diethyl ether from Fluka Biochemicals (Melbourne) and 2-mercaptoethanol from Sigma (St. Louis Mo., USA). Trifluoroethanol from (Aldrich. Milwaukee, Wis., USA). HF was purchased from Boc Gases (Brisbane, Australia) The following Naα-Boc protected L-amino acids Ala, Gly, Ile, Leu, Phe, Pro, Val, Arg (Tos), Asp (OChx), Asn (Xanth), Glu (OChx), His (DNP), Ser (Bzl), Thr (Bzl), Tyr (2BrZ) were purchased either from NovaBiochem (La Jolla, Calif., USA) or Bachem (Switzerland). MBHA polystyrene resin was purchased from Peptide Institute (Kyoto, Japan). </div>
</li> <li> <div class="description-line" id="h-0109" num="0000"> HPLC Methods </div>
</li> <li> <para-num num="[0243]"> </para-num> <div class="description-line" id="p-0221" num="0243"> Analytical and preparative HPLC was carried out using a Waters HPLC system comprised of model 600 solvent delivery system 600E controller and model 484 detector. Vydac C18 and C4 columns analytical (4.6×250 mm id) at a flow rate of 1 ml/min and semi preparative (10×250 mm id) at a flow rate of 3 ml/min and preparative (22×250 mm id) at a flow rate of 8 ml/min were used. All peptides were purified using linear gradients of 0.1% aqueous TFA (solvent A), 90% aqueous to acetonitrile 0.09% TFA (solvent B) </div>
</li> <li> <div class="description-line" id="h-0110" num="0000"> Peptide Synthesis </div>
</li> <li> <para-num num="[0244]"> </para-num> <div class="description-line" id="p-0222" num="0244"> Peptides were synthesized using the rapid manual HBTU in-situ neutralization synthesis techniques (Schnolzer et al., 1992, supra) on a modified ABI 430A peptide synthesizer (Alewood et al., 1997, supra). The peptide was synthesized on a MBHA resin on a 0.2 mmol scale using 0.79 mmol/g NH2 substituted resin. Each amino acid was double coupled using 2 mmol AA 0.48M HBTU (4 ml) and 1 ml DIEA for 10 min each coupling. The Boc group was removed by 2×1 min treatments of TFA with 1 min DMF flow washes of the resin. </div>
</li> <li> <para-num num="[0245]"> </para-num> <div class="description-line" id="p-0223" num="0245"> At the completion of the synthesis the His(DNP) group, if present in a particular sequence, was removed using 20% mercaptoethanol in 10% DIEA/DMF solution 3×30 min treatments. Peptide resin was cleaved using HF with p-cresol and p-thiocresol (90:8:2) as scavangers at −5 to 0° C. for 2 hrs. If Trp(CHO) is present in a sequence, it is removed by treatment with ethanolamine. The HF was removed in vacuo, the peptides triturated with cold diethyl ether (3×50 ml) the precipitated peptide collected then dissolved in 50% acetonitrile with 0.1% TFA to give the crude peptide. The crude peptide (˜80 mg lots) was purified by RPHPLC and fractions collected and analysed by analytical RPHPLC and ESMS. Fractions containing the purified peptide were combined and lyophilised. </div>
</li> <li> <para-num num="[0246]"> </para-num> <div class="description-line" id="p-0224" num="0246"> Mass spectral data were collected using a Perkin Elmer Sciex (Toronto, Canada) API III Biomolecular Mass Analyzer ion-spray mass spectrometer equipped with an ABI 140B solvent delivery system. Raw data was analyzed using the program MassSpec (Perkin Elmer Sciex). Calculated masses were obtained using the program MacProMass (Sunil Vemuri &amp; Terry Lee, City of Hope, Durate, Calif.). </div>
</li> <li> <div class="description-line" id="h-0111" num="0000"> Ultraviolet Circular Dichroism (CD) </div>
</li> <li> <para-num num="[0247]"> </para-num> <div class="description-line" id="p-0225" num="0247"> Far UV-CD spectra were recorded using a Jasco 710 CD spectrometer with associated based PC software. CD spectra are presented as a plot of mean molar ellipticity per residue [θ] deg cm<sup>2 </sup>dmol<sup>−1 </sup>verse wavelength in 0.1 nm increments. The digitised data was ploted using the Kalidagraph program on a Macintosh. All peptide concentrations were determined by quantitative amino acid analysis. </div>
</li> <li> <div class="description-line" id="h-0112" num="0000"> <sup>1</sup>H NMR Spectroscopy </div>
</li> <li> <para-num num="[0248]"> </para-num> <div class="description-line" id="p-0226" num="0248"> All NMR experiments were recorded on a Bruker ARX 500 spectrometer equipped with a Z-gradient unit. Peptide concentration was approximately 3 mM in 95% H<sub>2</sub>O/5% D<sub>2</sub>0 (T=293K). Spectra recorded included NOESY (Kumar et al., 1980, Biochem. Biophys. Res. Comm. 95 1; Jeener et al., 1979, 71 4546) with a mixing time of 400 millisecond, and TOCSY (Bax &amp; Davis, 1985, 65 355) with a mixing time of 85 millisecond. Spectra were run over 5550 Hz with 4K data points, 512 FIDs, 32-64 scans and a recycle delay of 1s. The solvent was suppressed using the WATERGATE sequence (Piotto et al., J. Biomol. NMR, 1992, 2 661) Spectra were processed using UXNMR. FIDS were multiplied by a polynomial function and apodised using a 90° shifted sine-bell function in both dimensions prior to Fourier transformation. Baseline correction using a 5<sup>th </sup>order polynomial was applied and chemical shift values were referenced externally to DSS at 0.00 ppm. The random coil H chemical shift values of Wishart et al., 1995, J. Biomol. NMR 6 135, were used. Spectra were assigned using the methods of Wüthrich et al., 1986, NMR of Proteins and Nucleic Acids. Wiley-Interscience NY. </div>
</li> <li> <div class="description-line" id="h-0113" num="0000"> Growth Hormone Proliferation Assay </div>
</li> <li> <para-num num="[0249]"> </para-num> <div class="description-line" id="p-0227" num="0249"> BaF-B03 cells (a pro B cell line) that stably express the human Growth Hormone Receptor (hGHR) are used in this assay since they are able to elicit a GH-specific response at concentrations as low as 0.1 ng/mL hGH (4.54 pM). These cells also endogenously express the IL3 receptor and require IL3 or GM-CSF to survive in culture. The assay is based on that of Mossman, 1983, J. Immunol. Meth. 65 55, and involves the following procedure: 
</div> </li> <ul> <li id="ul0009-0001" num="0250"> (i) culture cells in RPMI-1640 medium supplemented with 10% (v/v) foetal bovine serum (FBS) and 100 units/mL IL3 under 5% CO<sub>2 </sub>at 37° C. Allow the culture to reach mid-log growth phase; </li> <li id="ul0009-0002" num="0251"> (ii) centrifuge cells at 500×g and wash with PBS to remove IL3 from the culture medium. Repeat the centrifugation and resuspend in 1 mL of RPMI-1640 plus 0.5% (v/v) FBS. Count cells and dilute to a concentration of 8×10<sup>5 </sup>cells/mL in same media; </li> <li id="ul0009-0003" num="0252"> (iii) from a constantly stirred suspension, add 50 μL of cells to each well of two 96 well plates; </li> <li id="ul0009-0004" num="0253"> (iv) prepare stock solutions of the mimetic to be tested at various concentrations such that the final concentration ranges from 100 nM to 100 μM made up in 0.5% FBS media (final volume is 150 μL, therefore stocks should be 3 times final concentration required). Add 50 μL of these solutions to cells in sextuplicate (i.e. A1 to A6 are identical etc.); </li> <li id="ul0009-0005" num="0254"> (v) prepare a stock solution (3 times) of hGH such that the final concentration is 0.5 ng/mL and add 50 mL to each well of one plate. Include one row as a negative control with no cytokine; </li> <li id="ul0009-0006" num="0255"> (vi) prepare a stock solution (3 times) of IL-3 such that the final concentration is 50 units/mL and add 50 μL to each well of the other plate. Include one row as a negative control with no cytokine; </li> <li id="ul0009-0007" num="0256"> (vii) incubate plates with no lids (to prevent uneven evaporation rates) in a vented humidified box under the abovementioned incubation conditions. Allow incubation to continue for 24 hrs; </li> <li id="ul0009-0008" num="0257"> (viii) add 50 μL of 4 mg/mL MTT (3-[4,5-dimethylthiazol-2-yl]-2,5-diphenyltetrazolium bromide) to each well and incubate for a further 3 hrs; </li> <li id="ul0009-0009" num="0258"> (ix) to stop assay, remove from incubator and lyse cells by adding 120 μL of isopropanol and triturating for several seconds per well or until cells are clearly lysed. Allow plate to rest in the dark for 5 minutes before reading; </li> <li id="ul0009-0010" num="0259"> (x) read plate at 595 nm on a microplate reader. Values obtained are directly proportional to cell number (as measured by mitochondrial dehydrogenase levels). </li> </ul>
<heading id="h-0114">CONCLUSIONS </heading>
<li> <para-num num="[0260]"> </para-num> <div class="description-line" id="p-0228" num="0260"> These studies have shown that by engineering small, cysteine-rich proteins, a stable mimetic with high bioavailability can be made with desired biological characteristics, in this case the ability to antagonize the biological action of hGH. Furthermore, the database searching strategy of the present invention has shown that suitable “frameworks” for engineering mimetics can be identified according to aspects of structure which are shared with a sample protein that possesses a function of interest. The framework so identified will advantageously have increased stability compared to the sample protein. Finally, frameworks identified by the method of the invention may be suitable for further amino acid sequence modification so as to impart a function of the sample protein, or a function antagonistic thereto. </div>
</li> <li> <para-num num="[0261]"> </para-num> <div class="description-line" id="p-0229" num="0261"> The present invention therefore provides a new strategy for the engineering of proteins, which strategy is particularly applicable to the engineering of mimetics which may constitute the next generation of therapeutics. </div>
</li> <li> <para-num num="[0262]"> </para-num> <div class="description-line" id="p-0230" num="0262"> It will be understood by the skilled person that the invention is not limited to the particular embodiments described in detail herein, but also includes other embodiments consistent with the broad spririt and scope of the invention.  
<tables id="TABLE-US-00001" num="1"> <patent-tables colsep="0" frame="none" pgwide="1" rowsep="0"> <table align="left" class="description-table" cols="1" colsep="0" rowsep="0" width="100%"> <thead> <tr class="description-tr"> <td align="center" class="description-td" colspan="1" nameend="1" namest="1">TABLE 1</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td align="center" class="description-td" colspan="1" nameend="1" namest="1" rowsep="1"> </td> </tr> <tr class="description-tr"> <td class="description-td">An example of a query file which define the query Cα-Cβ vectors,</td> </tr> <tr class="description-tr"> <td class="description-td">the tolerance for each query atom and the definition of subset</td> </tr> <tr class="description-tr"> <td align="center" class="description-td" colspan="1" nameend="1" namest="1" rowsep="1"> </td> </tr> </thead> <tbody><tr class="description-tr"> <td class="description-td"> </td> </tr> </tbody></table> <table align="left" class="description-table" cols="9" colsep="0" rowsep="0" width="100%"> <tbody><tr class="description-tr"> <td class="description-td">ATOM</td> <td class="description-td">344</td> <td class="description-td">Cα</td> <td class="description-td">LYS A 41</td> <td class="description-td">54.743</td> <td class="description-td">11.420</td> <td class="description-td">29.859</td> <td class="description-td">0.50</td> <td class="description-td">33.97</td> </tr> <tr class="description-tr"> <td class="description-td">ATOM</td> <td class="description-td">347</td> <td class="description-td">Cβ</td> <td class="description-td">LYS A 41</td> <td class="description-td">53.280</td> <td class="description-td">11.410</td> <td class="description-td">30.298</td> <td class="description-td">0.50</td> <td class="description-td">36.33</td> </tr> <tr class="description-tr"> <td class="description-td">ATOM</td> <td class="description-td">382</td> <td class="description-td">Cα</td> <td class="description-td">LEU A 45</td> <td class="description-td">58.116</td> <td class="description-td">17.055</td> <td class="description-td">29.052</td> <td class="description-td">0.50</td> <td class="description-td">30.56</td> </tr> <tr class="description-tr"> <td class="description-td">ATOM</td> <td class="description-td">385</td> <td class="description-td">Cβ</td> <td class="description-td">LEU A 45</td> <td class="description-td">56.870</td> <td class="description-td">17.340</td> <td class="description-td">29.906</td> <td class="description-td">0.50</td> <td class="description-td">27.80</td> </tr> <tr class="description-tr"> <td class="description-td">ATOM</td> <td class="description-td">274</td> <td class="description-td">Cα</td> <td class="description-td">GLU A 119</td> <td class="description-td">43.893</td> <td class="description-td">28.064</td> <td class="description-td">0.887</td> <td class="description-td">0.50</td> <td class="description-td">0.00</td> </tr> <tr class="description-tr"> <td class="description-td">ATOM</td> <td class="description-td">277</td> <td class="description-td">Cβ</td> <td class="description-td">GLU A 119</td> <td class="description-td">43.099</td> <td class="description-td">27.286</td> <td class="description-td">−0.137</td> <td class="description-td">0.50</td> <td class="description-td">0.00</td> </tr> <tr class="description-tr"> <td class="description-td">ATOM</td> <td class="description-td">296</td> <td class="description-td">Cα</td> <td class="description-td">THR A 123</td> <td class="description-td">41.789</td> <td class="description-td">33.792</td> <td class="description-td">1.008</td> <td class="description-td">0.50</td> <td class="description-td">0.00</td> </tr> <tr class="description-tr"> <td class="description-td">ATOM</td> <td class="description-td">299</td> <td class="description-td">Cβ</td> <td class="description-td">THR A 123</td> <td class="description-td">40.586</td> <td class="description-td">32.811</td> <td class="description-td">0.784</td> <td class="description-td">0.50</td> <td class="description-td">0.00</td> </tr> <tr class="description-tr"> <td align="center" class="description-td" colspan="9" nameend="9" namest="1" rowsep="1"> </td> </tr> <tr class="description-tr"> <td align="left" class="description-td" colspan="9" id="FOO-00001" nameend="9" namest="1"> <br/>Subset 1, 3, 4. <br/> </td> </tr> <tr class="description-tr"> <td align="left" class="description-td" colspan="9" id="FOO-00002" nameend="9" namest="1"> <br/>Subset 2. <br/> </td> </tr> </tbody></table> </patent-tables> </tables>
</div>
</li> <li> <para-num num="[0263]"> </para-num> <div class="description-line" id="p-0231" num="0263">
<tables id="TABLE-US-00002" num="2">
<patent-tables colsep="0" frame="none" rowsep="0">
<table align="left" class="description-table" cols="1" colsep="0" rowsep="0" width="100%">
<thead>
<tr class="description-tr">
<td align="center" class="description-td" colspan="1" nameend="1" namest="1">TABLE 2</td>
</tr>
</thead>
<tbody><tr class="description-tr">
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td align="center" class="description-td" colspan="1" nameend="1" namest="1" rowsep="1"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">Blood serum stability test results</td>
</tr>
</tbody></table>
<table align="left" class="description-table" cols="4" colsep="0" rowsep="0" width="100%">
<tbody><tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">0 hr</td>
<td class="description-td">1 hr</td>
<td class="description-td">24 hrs</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td align="center" class="description-td" colspan="4" nameend="3" namest="OFFSET" rowsep="1"> </td>
</tr>
</tbody></table>
<table align="left" class="description-table" cols="4" colsep="0" rowsep="0" width="100%">
<tbody><tr class="description-tr">
<td class="description-td">Control peptide</td>
<td class="description-td">partially digested after 3 mins</td>
<td class="description-td">fully digested</td>
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">SCY01</td>
<td class="description-td">stable</td>
<td class="description-td">stable</td>
<td class="description-td">stable</td>
</tr>
<tr class="description-tr">
<td align="center" class="description-td" colspan="4" nameend="4" namest="1" rowsep="1"> </td>
</tr>
</tbody></table>
</patent-tables>
</tables>
</div>
</li> <li> <para-num num="[0264]"> </para-num> <div class="description-line" id="p-0232" num="0264">
<tables id="TABLE-US-00003" num="3">
<patent-tables colsep="0" frame="none" rowsep="0">
<table align="left" class="description-table" cols="1" colsep="0" rowsep="0" width="100%">
<thead>
<tr class="description-tr">
<td align="center" class="description-td" colspan="1" nameend="1" namest="1">TABLE 3</td>
</tr>
</thead>
<tbody><tr class="description-tr">
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td align="center" class="description-td" colspan="1" nameend="1" namest="1" rowsep="1"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">Enzyme stability test results</td>
</tr>
</tbody></table>
<table align="left" class="description-table" cols="3" colsep="0" rowsep="0" width="100%">
<tbody><tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">Control peptide</td>
<td class="description-td">SCY01</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td align="center" class="description-td" colspan="3" nameend="2" namest="OFFSET" rowsep="1"> </td>
</tr>
</tbody></table>
<table align="left" class="description-table" cols="4" colsep="0" rowsep="0" width="100%">
<tbody><tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">trypsin</td>
<td class="description-td">Digested in 1 hr</td>
<td class="description-td">Stable over 18 hrs</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">α-chymotrypsin</td>
<td class="description-td">Digested in 1 hr</td>
<td class="description-td">Stable over 18 hrs</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">pepsin</td>
<td class="description-td">Digested in 1 hr</td>
<td class="description-td">Stable over 18 hrs</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td align="center" class="description-td" colspan="4" nameend="3" namest="OFFSET" rowsep="1"> </td>
</tr>
</tbody></table>
</patent-tables>
</tables>
</div>
</li> <li> <para-num num="[0265]"> </para-num> <div class="description-line" id="p-0233" num="0265">
<chemistry id="CHEM-US-00001" num="1">
<div class="patent-image"><a href="https://patentimages.storage.googleapis.com/b6/d9/ed/45d8abc197d895/US20070016380A1-20070118-C00001.png"><img alt="Figure US20070016380A1-20070118-C00001" class="patent-full-image" file="US20070016380A1-20070118-C00001.TIF" he="132.67mm" height="531" id="EMI-C00001" img-content="chem" img-format="tif" inline="no" orientation="portrait" src="https://patentimages.storage.googleapis.com/b6/d9/ed/45d8abc197d895/US20070016380A1-20070118-C00001.png" wi="68.83mm" width="275"/></a></div>
<attachments>
<attachment attachment-type="cdx" file="US20070016380A1-20070118-C00001.CDX" idref="CHEM-US-00001"> </attachment>
<attachment attachment-type="mol" file="US20070016380A1-20070118-C00001.MOL" idref="CHEM-US-00001"> </attachment>
</attachments>
</chemistry>
<chemistry id="CHEM-US-00002" num="2">
<div class="patent-image"><a href="https://patentimages.storage.googleapis.com/12/0e/58/fa90b3b7877a67/US20070016380A1-20070118-C00002.png"><img alt="Figure US20070016380A1-20070118-C00002" class="patent-full-image" file="US20070016380A1-20070118-C00002.TIF" he="122.68mm" height="491" id="EMI-C00002" img-content="chem" img-format="tif" inline="no" orientation="portrait" src="https://patentimages.storage.googleapis.com/12/0e/58/fa90b3b7877a67/US20070016380A1-20070118-C00002.png" wi="74.93mm" width="300"/></a></div>
<attachments>
<attachment attachment-type="cdx" file="US20070016380A1-20070118-C00002.CDX" idref="CHEM-US-00002"> </attachment>
<attachment attachment-type="mol" file="US20070016380A1-20070118-C00002.MOL" idref="CHEM-US-00002"> </attachment>
</attachments>
</chemistry>
<chemistry id="CHEM-US-00003" num="3">
<div class="patent-image"><a href="https://patentimages.storage.googleapis.com/8c/a1/0d/491c0341e49329/US20070016380A1-20070118-C00003.png"><img alt="Figure US20070016380A1-20070118-C00003" class="patent-full-image" file="US20070016380A1-20070118-C00003.TIF" he="139.36mm" height="557" id="EMI-C00003" img-content="chem" img-format="tif" inline="no" orientation="portrait" src="https://patentimages.storage.googleapis.com/8c/a1/0d/491c0341e49329/US20070016380A1-20070118-C00003.png" wi="69.93mm" width="280"/></a></div>
<attachments>
<attachment attachment-type="cdx" file="US20070016380A1-20070118-C00003.CDX" idref="CHEM-US-00003"> </attachment>
<attachment attachment-type="mol" file="US20070016380A1-20070118-C00003.MOL" idref="CHEM-US-00003"> </attachment>
</attachments>
</chemistry>
<tables id="TABLE-US-00004" num="4">
<patent-tables colsep="0" frame="none" rowsep="0">
<table align="left" class="description-table" cols="1" colsep="0" rowsep="0" width="100%">
<thead>
<tr class="description-tr">
<td align="center" class="description-td" colspan="1" nameend="1" namest="1">SCHEME D</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td align="center" class="description-td" colspan="1" nameend="1" namest="1" rowsep="1"> </td>
</tr>
</thead>
<tbody><tr class="description-tr">
<td class="description-td">1) Framework hit mutated to sample molecule residues: Using the</td>
</tr>
<tr class="description-tr">
<td class="description-td">Biopolymer module of InsightII, the residues matching the sample</td>
</tr>
<tr class="description-tr">
<td class="description-td">molecule are mutated to the residue type of the sample molecule. Using</td>
</tr>
<tr class="description-tr">
<td class="description-td">the Search_compare module of InsightII, the sidechains of the mutated</td>
</tr>
<tr class="description-tr">
<td class="description-td">residues of the framework hit are flexibly fitted to the sidechains of their</td>
</tr>
<tr class="description-tr">
<td class="description-td">corresponding sample molecule residues to produce a theoretical</td>
</tr>
<tr class="description-tr">
<td class="description-td">‘Bioactive conformation’. A bump-check with the receptor is performed</td>
</tr>
<tr class="description-tr">
<td class="description-td">to identify unmutated sidechain steric clashes with receptor. Investigate</td>
</tr>
<tr class="description-tr">
<td class="description-td">mimicking unmatched functional residues of sample molecule with</td>
</tr>
<tr class="description-tr">
<td class="description-td">unnatural amino acids.</td>
</tr>
<tr class="description-tr">
<td class="description-td">
<chemistry id="CHEM-US-00004" num="4">
<div class="patent-image"><a href="https://patentimages.storage.googleapis.com/ac/fd/2e/193da980048175/US20070016380A1-20070118-C00004.png"><img alt="Figure US20070016380A1-20070118-C00004" class="patent-full-image" file="US20070016380A1-20070118-C00004.TIF" he="8.21mm" height="33" id="EMI-C00004" img-content="chem" img-format="tif" inline="no" orientation="portrait" src="https://patentimages.storage.googleapis.com/ac/fd/2e/193da980048175/US20070016380A1-20070118-C00004.png" wi="63.67mm" width="255"/></a></div>
<attachments>
<attachment attachment-type="cdx" file="US20070016380A1-20070118-C00004.CDX" idref="CHEM-US-00004"> </attachment>
<attachment attachment-type="mol" file="US20070016380A1-20070118-C00004.MOL" idref="CHEM-US-00004"> </attachment>
</attachments>
</chemistry>
</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">2) Conformational stability of theoretical ‘bioactive conformation’: The</td>
</tr>
<tr class="description-tr">
<td class="description-td">‘bioactive conformation’ from above is minimised in a forcefield. The</td>
</tr>
<tr class="description-tr">
<td class="description-td">rmsd of the backbone atoms of the minimised mutated framework hit from</td>
</tr>
<tr class="description-tr">
<td class="description-td">the minimised unmutated framework hit is calculated. If the rmsd &lt;2.0 A,</td>
</tr>
<tr class="description-tr">
<td class="description-td">the conformation is considered accessible.</td>
</tr>
<tr class="description-tr">
<td class="description-td">
<chemistry id="CHEM-US-00005" num="5">
<div class="patent-image"><a href="https://patentimages.storage.googleapis.com/07/64/50/953baa79b904fb/US20070016380A1-20070118-C00005.png"><img alt="Figure US20070016380A1-20070118-C00005" class="patent-full-image" file="US20070016380A1-20070118-C00005.TIF" he="8.21mm" height="33" id="EMI-C00005" img-content="chem" img-format="tif" inline="no" orientation="portrait" src="https://patentimages.storage.googleapis.com/07/64/50/953baa79b904fb/US20070016380A1-20070118-C00005.png" wi="63.75mm" width="255"/></a></div>
<attachments>
<attachment attachment-type="cdx" file="US20070016380A1-20070118-C00005.CDX" idref="CHEM-US-00005"> </attachment>
<attachment attachment-type="mol" file="US20070016380A1-20070118-C00005.MOL" idref="CHEM-US-00005"> </attachment>
</attachments>
</chemistry>
</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">3) Stability of fold: The mutated framework hit and the native framework</td>
</tr>
<tr class="description-tr">
<td class="description-td">hit are subject to molecular dynamics at 300 K. The rmsd of minimised</td>
</tr>
<tr class="description-tr">
<td class="description-td">trajectory intermediates from the original conformer in each dynamics run</td>
</tr>
<tr class="description-tr">
<td class="description-td">are plotted. Unless there is a significantly greater drift in rmsd of the</td>
</tr>
<tr class="description-tr">
<td class="description-td">mutated relative to native framework hit, the fold is considered stable.</td>
</tr>
<tr class="description-tr">
<td class="description-td">
<chemistry id="CHEM-US-00006" num="6">
<div class="patent-image"><a href="https://patentimages.storage.googleapis.com/d9/28/15/b900c386ec20ca/US20070016380A1-20070118-C00006.png"><img alt="Figure US20070016380A1-20070118-C00006" class="patent-full-image" file="US20070016380A1-20070118-C00006.TIF" he="8.21mm" height="33" id="EMI-C00006" img-content="chem" img-format="tif" inline="no" orientation="portrait" src="https://patentimages.storage.googleapis.com/d9/28/15/b900c386ec20ca/US20070016380A1-20070118-C00006.png" wi="63.75mm" width="255"/></a></div>
<attachments>
<attachment attachment-type="cdx" file="US20070016380A1-20070118-C00006.CDX" idref="CHEM-US-00006"> </attachment>
<attachment attachment-type="mol" file="US20070016380A1-20070118-C00006.MOL" idref="CHEM-US-00006"> </attachment>
</attachments>
</chemistry>
</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">4) Electrostatic similarity to target: Electostatic isocontour surfaces are</td>
</tr>
<tr class="description-tr">
<td class="description-td">generated for both mutated framework hit and sample molecule and</td>
</tr>
<tr class="description-tr">
<td class="description-td">compared for similarity. Electrostatic fields are mapped onto solvent</td>
</tr>
<tr class="description-tr">
<td class="description-td">accessible surface of mutated framework hit and sample molecule to</td>
</tr>
<tr class="description-tr">
<td class="description-td">compare electrostatic properties at the contact surface</td>
</tr>
<tr class="description-tr">
<td class="description-td">
<chemistry id="CHEM-US-00007" num="7">
<div class="patent-image"><a href="https://patentimages.storage.googleapis.com/e9/e1/85/10f519803bafdf/US20070016380A1-20070118-C00007.png"><img alt="Figure US20070016380A1-20070118-C00007" class="patent-full-image" file="US20070016380A1-20070118-C00007.TIF" he="8.21mm" height="33" id="EMI-C00007" img-content="chem" img-format="tif" inline="no" orientation="portrait" src="https://patentimages.storage.googleapis.com/e9/e1/85/10f519803bafdf/US20070016380A1-20070118-C00007.png" wi="63.67mm" width="255"/></a></div>
<attachments>
<attachment attachment-type="cdx" file="US20070016380A1-20070118-C00007.CDX" idref="CHEM-US-00007"> </attachment>
<attachment attachment-type="mol" file="US20070016380A1-20070118-C00007.MOL" idref="CHEM-US-00007"> </attachment>
</attachments>
</chemistry>
</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">5) Synthesise</td>
</tr>
<tr class="description-tr">
<td align="center" class="description-td" colspan="1" nameend="1" namest="1" rowsep="1"> </td>
</tr>
<tr class="description-tr">
<td align="left" class="description-td" colspan="1" id="FOO-00003" nameend="1" namest="1"> <br/>Rectify (a) If there are steric clashes, consider the most conserved mutation that removes the bump. <br/> </td>
</tr>
<tr class="description-tr">
<td align="left" class="description-td" colspan="1" id="FOO-00004" nameend="1" namest="1"> <br/>Rectify (b) If the theoretical ‘Bioactive conformation’ is not stable, check the Ramachandran plot of the hit for residues in disallowed regions. Consider stabilizing structure with unnatural amino acids eg alpha-amino isobutyric acid in alpha helix motifs. <br/> </td>
</tr>
</tbody></table>
</patent-tables>
</tables>
</div>
</li> <li> <para-num num="[0266]"> </para-num> <div class="description-line" id="p-0234" num="0266"> </div>
</li> </ul>
</div>
</section><section itemprop="claims" itemscope="">
<h2>Claims (<span itemprop="count">10</span>)</h2>
<div html="" itemprop="content"><div class="claims" lang="EN" load-source="patent-office" mxw-id="PCLM7545830">
<div class="claim"> <div class="claim" id="CLM-01-28" num="01-28">
<div class="claim-text"> <b>1</b>-<b>28</b>. (canceled) </div>
</div>
</div> <div class="claim"> <div class="claim" id="CLM-00029" num="00029">
<div class="claim-text"> <b>29</b>. An engineered protein comprising an amino acid sequence set forth in SEQ ID NO: 1 and at least two amino acid residues of another protein which are non-contiguous in primary sequence and represent at least a portion of a functional region of said another protein. </div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="CLM-00030" num="00030">
<div class="claim-text"> <b>30</b>. The engineered protein of <claim-ref idref="CLM-00029">claim 29</claim-ref>, wherein said another protein is a cytokine. </div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="CLM-00031" num="00031">
<div class="claim-text"> <b>31</b>. The engineered protein of <claim-ref idref="CLM-00029">claim 29</claim-ref>, wherein said another protein is a cytokine receptor. </div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="CLM-00032" num="00032">
<div class="claim-text"> <b>32</b>. The engineered protein of <claim-ref idref="CLM-00031">claim 31</claim-ref>, wherein the functional region of said cytokine receptor is a cytokine binding region. </div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="CLM-00033" num="00033">
<div class="claim-text"> <b>33</b>. The engineered protein of <claim-ref idref="CLM-00030">claim 30</claim-ref>, wherein the cytokine is selected from the group consisting of GH, IL-4, IL-6 and G-CSF. </div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="CLM-00034" num="00034">
<div class="claim-text"> <b>34</b>. The engineered protein of <claim-ref idref="CLM-00033">claim 33</claim-ref>, said engineered protein comprising an amino acid sequence selected from the group consisting of the amino acid sequences set forth in SEQ ID NOS:2-5. </div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="CLM-00035" num="00035">
<div class="claim-text"> <b>35</b>. The engineered protein of <claim-ref idref="CLM-00029">claim 29</claim-ref>, which protein has greater stability than said another protein. </div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="CLM-00036" num="00036">
<div class="claim-text"> <b>36</b>. The engineered protein of <claim-ref idref="CLM-00035">claim 35</claim-ref>, which protein exhibits a function either similar to, or inhibitory of, said another protein. </div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="CLM-00037" num="00037">
<div class="claim-text"> <b>37</b>. The engineered protein of <claim-ref idref="CLM-00036">claim 36</claim-ref>, which engineered protein is a cytokine mimetic.</div>
</div>
</div> </div>
</div>
</section>
                </article>
            </search-app>
        </body>
    </html>
    