
    <html>
        <body>
            <search-app>
                <article class="result" itemscope="" itemtype="http://schema.org/ScholarlyArticle">
    <h1 itemprop="pageTitle">US20180256739A1 - Multivalent Pneumococcal Polysaccharide-Protein Conjugate Composition 
      - Google Patents</h1><section itemprop="abstract" itemscope="">
<h2>Abstract</h2>
<div html="" itemprop="content"><abstract lang="EN" load-source="docdb" mxw-id="PA239911253" source="national office">
<div class="abstract">An immunogenic composition having 13 distinct polysaccharide-protein conjugates and optionally, an aluminum-based adjuvant, is described. Each conjugate contains a capsular polysaccharide prepared from a different serotype of Streptococcus pneumoniae (1, 3, 4, 5, 6A, 6B, 7F, 9V, 14, 18C, 19A, 19F and 23F) conjugated to a carrier protein. The immunogenic composition, formulated as a vaccine, increases coverage against pneumococcal disease in infants and young children globally, and provides coverage for serotypes 6A and 19A that is not dependent on the limitations of serogroup cross-protection. Methods for making an immunogenic conjugate comprising Streptococcus pneumoniae serotype 19A polysaccharide are also provided in which the serotype 19A polysaccharide is co-lyophilized with a carrier protein and conjugation is carried out in dimethyl sulfoxide (DMSO) via a reductive amination mechanism.</div>
</abstract>
</div>
</section><section itemprop="description" itemscope="">
<h2>Description</h2>
<div html="" itemprop="content"><ul class="description" lang="EN" load-source="patent-office" mxw-id="PDES142060134">
<heading id="h-0001">CROSS-REFERENCE TO RELATED APPLICATIONS</heading>
<li> <para-num num="[0001]"> </para-num> <div class="description-line" id="p-0002" num="0001">This application is a continuation-in-part of U.S. application Ser. No. 11/395,593, filed Mar. 31, 2006, which claims the benefit of U.S. Provisional Application No. 60/669,605, filed Apr. 8, 2005, each of which is hereby incorporated in its entirety by reference herein.</div>
</li> <heading id="h-0002">FIELD OF THE INVENTION</heading>
<li> <para-num num="[0002]"> </para-num> <div class="description-line" id="p-0003" num="0002">The present invention relates generally to the field of medicine, and specifically to microbiology, immunology, vaccines and the prevention of infection by a bacterial pathogen by immunization.</div>
</li> <heading id="h-0003">BACKGROUND OF THE INVENTION</heading>
<li> <para-num num="[0003]"> </para-num> <div class="description-line" id="p-0004" num="0003"> <i>Streptococcus pneumoniae </i>is a leading cause of meningitis, pneumonia, and severe invasive disease in infants and young children throughout the world. The multivalent pneumococcal polysaccharide vaccines have been licensed for many years and have proved valuable in preventing pneumococcal disease in elderly adults and high-risk patients. However, infants and young children respond poorly to most pneumococcal polysaccharides. The 7-valent pneumococcal conjugate vaccine (7vPnC, PrevnarÂ®) was the first of its kind demonstrated to be highly immunogenic and effective against invasive disease and otitis media in infants and young children. This vaccine is now approved in many countries around the world. Prevnar contains the capsular polysaccharides from serotypes 4, 6B, 9V, 14, 18C, 19F and 23F, each conjugated to a carrier protein designated CRM<sub>197</sub>. Prevnar covers approximately 80-90%, 60-80%, and 40-80% of invasive pneumococcal disease (IPD) in the US, Europe, and other regions of the world, respectively [1,2]. Surveillance data gathered in the years following Prevnar's introduction has clearly demonstrated a reduction of invasive pneumococcal disease in US infants as expected (<figref idrefs="DRAWINGS">FIG. 1</figref>) [3, 4].</div>
</li> <li> <para-num num="[0004]"> </para-num> <div class="description-line" id="p-0005" num="0004">Surveillance of IPD conducted in US infants prior to the introduction of Prevnar demonstrated that a significant portion of disease due to serogroups 6 and 19 was due to the 6A (approximately one-third) and 19A (approximately one-fourth) serotypes [5,6]. Pneumococcal invasive disease surveillance conducted in the US after licensure, of Prevnar suggests that a large burden of disease is still attributable to serotypes 6A and 19A (<figref idrefs="DRAWINGS">FIG. 1</figref>) [3]. Moreover, these two serotypes account for more cases of invasive disease than serotypes 1, 3, 5, and 7F combined (8.2 vs. 3.3 cases/100,000 children 2 years and under). In addition, serotypes 6A and 19A are associated with high rates of antibiotic resistance (<figref idrefs="DRAWINGS">FIG. 2</figref>) [7,8,9]. While it is possible that serogroup cross-protection will result in a decline of serotype 6A and 19A disease as more children are immunized, there is evidence to suggest that there will be a limit to the decline, and a significant burden of disease due to these serotypes will remain (see below).</div>
</li> <li> <para-num num="[0005]"> </para-num> <div class="description-line" id="p-0006" num="0005">Given the relative burden and importance of invasive pneumococcal disease due to serotypes 1, 3, 5, 6A, 7F, and 19A, adding these serotypes to the Prevnar formulation would increase coverage for invasive disease to &gt;90% in the US and Europe, and as high as 70%-80% in Asia and Latin America. This vaccine would significantly expand coverage beyond that of Prevnar, and provide coverage for 6A and 19A that is not dependent on the limitations of serogroup cross-protection.</div>
</li> <heading id="h-0004">SUMMARY OF THE INVENTION</heading>
<li> <para-num num="[0006]"> </para-num> <div class="description-line" id="p-0007" num="0006">Accordingly, the present invention provides generally a multivalent immunogenic composition comprising 13 distinct polysaccharide-protein conjugates, wherein each of the conjugates contains a capsular polysaccharide from a different serotype of <i>Streptococcus pneumoniae </i>conjugated to a carrier protein, together with a physiologically acceptable vehicle. Optionally, an adjuvant, such as an aluminum-based adjuvant, is included in the formulation. More specifically, the present invention provides a 13-valent pneumococcal conjugate (13vPnC) composition comprising the seven serotypes in the 7vPnC vaccine (4, 6B, 9V, 14, 18C, 19F and 23F) plus six additional serotypes (1, 3, 5, 6A, 7F and 19A).</div>
</li> <li> <para-num num="[0007]"> </para-num> <div class="description-line" id="p-0008" num="0007">The present invention also provides a multivalent immunogenic composition, wherein the capsular polysaccharides are from serotypes 1, 3, 4, 5, 6A, 6B, 7F, 9V, 14, 18C, 19A, 19F and 23F of <i>Streptococcus pneumoniae </i>and the carrier protein is CRM<sub>197</sub>.</div>
</li> <li> <para-num num="[0008]"> </para-num> <div class="description-line" id="p-0009" num="0008">The present invention further provides a multivalent immunogenic composition, wherein the capsular polysaccharides are from serotypes 1, 3, 4, 5, 6A, 6B, 7F, 9v, 14, 18C, 19A, 19F and 23F of <i>Streptococcus pneumoniae</i>, the carrier protein is CRM<sub>197</sub>, and the adjuvant is an aluminum-based adjuvant, such as aluminum phosphate, aluminum sulfate and aluminum hydroxide. In a particular embodiment of the invention, the adjuvant is aluminum phosphate.</div>
</li> <li> <para-num num="[0009]"> </para-num> <div class="description-line" id="p-0010" num="0009">The present invention also provides a multivalent immunogenic composition, comprising polysaccharide-protein conjugates together with a physiologically acceptable vehicle, wherein each of the conjugates comprises a capsular polysaccharide from a different serotype of <i>Streptococcus pneumoniae </i>conjugated to a carrier protein, and the capsular polysaccharides are prepared from serotype 3 and at least one additional serotype.</div>
</li> <li> <para-num num="[0010]"> </para-num> <div class="description-line" id="p-0011" num="0010">In one embodiment of this multivalent immunogenic composition, the additional serotype is selected from the group consisting of serotypes 1, 4, 5, 6A, 6B, 7F, 9V, 14, 18C, 19A, 19F, and 23F. In another embodiment, the carrier protein is CRM<sub>197</sub>. In yet another embodiment, the composition comprises an adjuvant, such as an aluminum-based adjuvant selected from aluminum phosphate, aluminum sulfate and aluminum hydroxide. In a particular embodiment, the adjuvant is aluminum phosphate.</div>
</li> <li> <para-num num="[0011]"> </para-num> <div class="description-line" id="p-0012" num="0011">The present invention also provides a multivalent immunogenic composition, comprising polysaccharide-protein conjugates together with a physiologically acceptable vehicle, wherein each of the conjugates comprises a capsular polysaccharide from a different serotype of <i>Streptococcus pneumoniae </i>conjugated to a carrier protein, and the capsular polysaccharides are prepared from serotypes 4, 6B, 9V, 14, 18C, 19F, 23F and at least one additional serotype.</div>
</li> <li> <para-num num="[0012]"> </para-num> <div class="description-line" id="p-0013" num="0012">In one embodiment of this multivalent immunogenic composition, the additional serotype is selected from the group consisting of serotypes 1, 3, 5, 6A, 7F, and 19A. In another embodiment, the carrier protein is CRM<sub>197</sub>. In yet another embodiment, the composition comprises an adjuvant, such as an aluminum-based adjuvant selected from aluminum phosphate, aluminum sulfate and aluminum hydroxide. In a particular embodiment, the adjuvant is aluminum phosphate.</div>
</li> <li> <para-num num="[0013]"> </para-num> <div class="description-line" id="p-0014" num="0013">The present invention also provides a method of inducing an immune response to a <i>Streptococcus pneumoniae </i>capsular polysaccharide conjugate, comprising administering to a human an immunologically effective amount of any of the immunogenic compositions just described.</div>
</li> <li> <para-num num="[0014]"> </para-num> <div class="description-line" id="p-0015" num="0014">The present invention further provides that any of the immunogenic compositions administered is a single 0.5 mL dose formulated to contain: 2 Î¼g of each saccharide, except for 6B at 4 Î¼g; approximately 29 Î¼g CRM<sub>197 </sub>carrier protein; 0.125 mg of elemental aluminum (0.5 mg aluminum phosphate) adjuvant; and sodium chloride and sodium succinate buffer as excipients.</div>
</li> <li> <para-num num="[0015]"> </para-num> <div class="description-line" id="p-0016" num="0015">Methods for making an immunogenic conjugate comprising <i>Streptococcus pneumoniae </i>serotype 19A (Pn 19A) polysaccharide covalently linked to a carrier protein are also provided. In one embodiment, the method comprises: (i) reacting purified serotype 19A polysaccharide with an oxidizing agent resulting in an activated serotype 19A polysaccharide; (ii) compounding the activated serotype 19A polysaccharide with a carrier protein; (iii) co-lyophilizing the compounded activated serotype 19A polysaccharide and carrier protein; (iv) re-suspending the compounded activated serotype 19A polysaccharide and carrier protein in dimethyl sulfoxide (DMSO); (v) reacting the compounded, activated serotype 19A polysaccharide and carrier protein with a reducing agent resulting in a serotype 19A polysaccharide:carrier protein conjugate; and (vi) capping unreacted aldehydes in the serotype 19A polysaccharide:carrier protein conjugate resulting in an immunogenic conjugate comprising <i>Streptococcus pneumoniae </i>serotype 19A polysaccharide covalently linked to a carrier protein.</div>
</li> <li> <para-num num="[0016]"> </para-num> <div class="description-line" id="p-0017" num="0016">In a further embodiment, the method for making an immunogenic conjugate comprising <i>Streptococcus pneumoniae </i>serotype 19A polysaccharide covalently linked to a carrier protein comprises: (i) reacting purified serotype 19A polysaccharide with sodium periodate resulting in an activated serotype 19A polysaccharide; (ii) adjusting the pH of the activated serotype 19A polysaccharide to 6.5Â±0.2; (iii) compounding the activated serotype 19A polysaccharide with sucrose; (iv) compounding the activated serotype 19A polysaccharide with a CRM<sub>197 </sub>carrier protein at a ratio of 0.8:1; (v) co-lyophilizing the compounded activated serotype 19A polysaccharide and carrier protein; (vi) re-suspending the compounded activated serotype 19A polysaccharide and carrier protein in DMSO; (vii) reacting the compounded, activated serotype 19A polysaccharide and carrier protein with sodium cyanoborohydride resulting in a serotype 19A polysaccharide:carrier protein conjugate; and (viii) capping unreacted aldehydes in the serotype 19A polysaccharide:carrier protein conjugate with sodium borohydride resulting in an immunogenic conjugate comprising <i>Streptococcus pneumoniae </i>serotype 19A polysaccharide covalently linked to a carrier protein.</div>
</li> <li> <para-num num="[0017]"> </para-num> <div class="description-line" id="p-0018" num="0017">Methods for making an immunogenic conjugate comprising a <i>Streptococcus pneumoniae </i>polysaccharide covalently linked to a carrier protein in which the polysaccharide comprises a phosphodiester linkage between repeat units are also provided. In one embodiment, the method comprises: (i) reacting the polysaccharide with an oxidizing agent resulting in an activated polysaccharide; (ii) compounding the activated polysaccharide with a carrier protein; (iii) co-lyophilizing the compounded activated polysaccharide and carrier protein; (iv) re-suspending the compounded activated polysaccharide and carrier protein in DMSO; (v) reacting the compounded, activated polysaccharide and carrier protein with a reducing agent resulting in a polysaccharide:carrier protein conjugate; and (vi) capping unreacted aldehydes in the polysaccharide:carrier protein conjugate resulting in an immunogenic conjugate comprising <i>Streptococcus pneumoniae </i>polysaccharide covalently linked to a carrier protein. In further embodiments of such methods, the polysaccharide comprising a phosphodiester linkage between repeat units is <i>Streptococcus pneumoniae </i>polysaccharide serotype 19A, 19F, 6A, or 6B. In a further embodiment, the carrier protein is CRM<sub>197</sub>.</div>
</li> <description-of-drawings>
<heading id="h-0005">BRIEF DESCRIPTION OF THE DRAWINGS</heading>
<li> <para-num num="[0018]"> </para-num> <div class="description-line" id="p-0019" num="0018"> <figref idrefs="DRAWINGS">FIG. 1</figref> depicts the changes in IPD rates by serotype in US children &lt;2 years of age from baseline (1998/1999) to 2001.</div>
</li> <li> <para-num num="[0019]"> </para-num> <div class="description-line" id="p-0020" num="0019"> <figref idrefs="DRAWINGS">FIG. 2</figref> depicts the distribution of pneumococcal isolates with resistance to penicillin (PCN) in children &lt;5 years of age (1998).</div>
</li> <li> <para-num num="[0020]"> </para-num> <div class="description-line" id="p-0021" num="0020"> <figref idrefs="DRAWINGS">FIG. 3</figref> depicts the reverse cumulative distribution curves (RCDC) of OPA post-third dose results from the D118-P16 Prevnar trial.</div>
</li> </description-of-drawings>
<heading id="h-0006">DETAILED DESCRIPTION OF THE INVENTION</heading>
<heading id="h-0007">Inclusion of Prevnar Serotypes 4, 6B, 9V, 14, 18C, 19F, 23F</heading>
<li> <para-num num="[0021]"> </para-num> <div class="description-line" id="p-0022" num="0021">Data from IPD surveillance between 1995-1998 estimated that the seven serotypes in Prevnar were responsible for around 82% of IPD in children &lt;2 years of age [5]. In Northern California, the site of the efficacy trial, the Prevnar serotypes accounted for 90% of all cases of IPD in infants and young children [10]. Since introduction of the Prevnar vaccine in 2000, there has been a significant decrease in the overall IPD rates due to a decrease in disease due to the vaccine serotypes [3,4]. Therefore, there is no justification at this time to remove any of the Prevnar serotypes from the next generation of pneumococcal conjugate vaccines but rather to add serotypes to obtain wider coverage.</div>
</li> <heading id="h-0008">Inclusion of Serotypes 1, 3, 5 and 7F</heading>
<li> <para-num num="[0022]"> </para-num> <div class="description-line" id="p-0023" num="0022">In the US, the rate of IPD caused by serotype 1 in children under the age of 5 years is &lt;2%, about the same as for each of types 3 and 7F [1,6]. Serotypes 1 and 5 account for higher rates of IPD in US populations at high risk for invasive pneumococcal disease. Specifically, serotype 1 causes 3.5% of IPD in Alaskan native children &lt;2 years of age, and 18% in children 2-4 years of age [11]. Both serotype 1 and serotype 5 significantly cause disease in other parts of the world and in indigenous populations in developed countries [12, 13, 14].</div>
</li> <li> <para-num num="[0023]"> </para-num> <div class="description-line" id="p-0024" num="0023">Serotype 1 may also be associated with more severe disease as compared with other pneumococcal serotypes [15]. This observation is based on the difference in rates of case identification between the US and Europe, and the associated difference in medical practice. Overall, the incidence of IPD is lower in Europe than in the US. However, the percent of IPD caused by serotype 1 in Europe is disproportionately higher than in the US (6-7%, vs. 1-2%, respectively). In Europe, blood cultures are obtained predominantly from hospitalized children. In the US, it is routine medical practice to obtain blood cultures in an outpatient setting from children presenting with fever â¥39Â° C. and elevated white blood cell counts. Given the difference in medical practice, it is postulated that the lower percent of disease caused by serotype 1 in the US may be diluted by higher rates of other serotypes causing milder disease, while the higher percent in Europe reflects more serious disease. In addition, seroepidemiology studies of children with complicated pneumonia demonstrate that serotype 1 is disproportionately represented [16, 17, 18]. This suggests that inclusion of serotype 1 may reduce the amount of severe pneumococcal disease, as well as, contribute to a total reduction in invasive pneumococcal disease.</div>
</li> <li> <para-num num="[0024]"> </para-num> <div class="description-line" id="p-0025" num="0024">The addition of serotypes 3 and 7F will increase coverage against IPD in most areas of the world by approximately 3%-7%, and in Asia by around 9%. Thus, an 11-valent vaccine would cover 50% in Asia and around 80% of IPD in all other regions [1,2]. These serotypes are also important with respect to otitis media coverage [19]. In a multinational study of pneumococcal serotypes causing otitis media, Hausdorff et al found serotype 3 to be the 8th most common middle ear fluid isolate overall [20]. Serotype 3 accounted for up to 8.7% of pneumococcal serotypes associated with otitis media. Thus, the importance of types 3 and 7F in otitis media, as well as in IPD, warrants their inclusion in a pneumococcal conjugate vaccine.</div>
</li> <li> <para-num num="[0025]"> </para-num> <div class="description-line" id="p-0026" num="0025">However, attempts to produce a multivalent pneumococcal conjugate vaccine that exhibits significant immunogenicity with respect to serotype 3 polysaccharides have been unsuccessful. For example, in a study of the immunogenicity and safety of an 11-valent pneumococcal protein D conjugate vaccine (11-Pn-PD), no priming effect was observed for serotype 3 in infants who had received three doses of the vaccine followed by a booster dose of either the same vaccine or a pneumococcal polysaccharide vaccine (Nurkka et al. (2004) <i>Ped. Inf. Dis. J., </i>23:1008-1014). In another study, opsonophagocytic assay (OPA) results from infants who had received doses of 11-Pn-PD failed to show antibody responses for serotype 3 at levels comparable to other tested serotypes (Gatchalian et al., 17<sup>th </sup>Annual Meeting of the Eur. Soc. Paed. Inf. Dis. (ESPID), Poster No. 4, P1A Poster Session 1, Istanbul Turkey, Mar. 27, 2001). In yet another study, which assessed the efficacy of an 11-Pn-PD in the prevention of acute otitis media, the vaccine did not provide protection against episodes caused by serotype 3 (Prymula et al. (2006) Lancet, 367:740-748). Accordingly, a pneumococcal conjugate vaccine comprising capsular polysaccharides from serotype 3 and capable of eliciting an immunogenic response to serotype 3 polysaccharides provides a significant improvement over the existing state of the art.</div>
</li> <heading id="h-0009">Inclusion of Serotypes 6A and 19A</heading>
<li> <para-num num="[0026]"> </para-num> <div class="description-line" id="p-0027" num="0026">a. Epidemiology of Serotypes 6A and 19A</div>
</li> <li> <para-num num="[0027]"> </para-num> <div class="description-line" id="p-0028" num="0027">Surveillance data in the literature suggest that serotypes 6A and 19A account for more invasive pneumococcal disease in US children &lt;2 years of age than serotypes 1, 3, 5, and 7F combined (<figref idrefs="DRAWINGS">FIG. 1</figref>) [1,5]. In addition, these serotypes are commonly associated with antibiotic resistance (<figref idrefs="DRAWINGS">FIG. 2</figref>) and play an important role in otitis media [6, 19, 20]. The ability of the current Prevnar vaccine to protect against disease due to 6A and 19A is not clear. The rationale for inclusion of 6A and 19A components in a 13vPnC vaccine is discussed below.</div>
</li> <li> <div class="description-line" id="p-0029" num="0000">b. Responses to 6A and 19A Induced by 6B and 19F Polysaccharides</div>
</li> <li> <para-num num="[0028]"> </para-num> <div class="description-line" id="p-0030" num="0028">The licensed unconjugated pneumococcal polysaccharide vaccines (for use in persons at least two years of age) have contained 6A or 6B capsular polysaccharide but not both [21]. Immunogenicity data generated at the time of formulation of the 23-valent pneumococcal polysaccharide vaccine demonstrated that a 6B monovalent vaccine induced antibody to both the 6A and 6B capsules. The data from several trials assessing IgG and opsonophagocyic assay (OPA) responses in a variety of populations with free polysaccharide and with pneumococcal conjugate vaccines suggested that IgG responses to 6A are induced by 6B B antigens, but the responses are generally lower, and the OPA activity with 6A organisms is different than with 6B organisms [22, 23, 24, 25]. In addition, subjects responding with high 6B antibody may have little or no activity against 6A.</div>
</li> <li> <para-num num="[0029]"> </para-num> <div class="description-line" id="p-0031" num="0029">In contrast to the chemical composition of the 6A and 6B capsular polysaccharides where there exists a high degree of similarity, the 19A and 19F capsules are quite different due to the presence of two additional side chains in the 19A polysaccharide. Not surprisingly, immune responses measured in human volunteers immunized with 19F polysaccharide vaccine showed that responses to 19F were induced in 80% of subjects, but only 20% of subjects had a response to 19A [26]. Low levels of cross-reactive IgG and OPA responses to serotype 19A after immunization with 19F polysaccharide have also been documented in trials with conjugate vaccines as well [24, 26].</div>
</li> <li> <para-num num="[0030]"> </para-num> <div class="description-line" id="p-0032" num="0030">Internal data on cross-reactive OPA responses to 6A and 19A have been generated from the 7vPnC bridging trial (D118-P16) conducted in US infants (<figref idrefs="DRAWINGS">FIG. 3</figref>). These studies are consistent with the findings of others, and demonstrate induction of cross-reactive functional antibody to 6A polysaccharide after immunization with 6B polysaccharide, although at a lower level, and very little functional antibody to 19A after immunization with 19F.</div>
</li> <heading id="h-0010">Impact of 6B and 19F Immunization on 6A and 19A in Animal Models</heading>
<li> <para-num num="[0031]"> </para-num> <div class="description-line" id="p-0033" num="0031">Animal models have been used to evaluate the potential for cross-protection with polysaccharide immunization. In an otitis media model developed by Giebink et al., chinchillas were immunized with a tetravalent polysaccharide outer membrane protein (OMP) conjugate vaccine (containing 6B, 14, 19F, 23F saccharides) or placebo [27]. In this trial there appeared to be some cross-protection for 6A; however this did not reach statistical significance and the level of protection was lower than with 6B against otitis media. In this same model there was 100% protection against 19F otitis media, but only 17% protection against 19A otitis media.</div>
</li> <li> <para-num num="[0032]"> </para-num> <div class="description-line" id="p-0034" num="0032">Saeland et al. used sera from infants immunized with an 8-valent pneumococcal tetanus conjugate vaccine (containing 6B and 19F) to passively immunize mice prior to an intranasal challenge with 6A organisms, in a lung infection model [28]. Of the 59 serum samples, 53% protected mice against bacteremia with 6B and 37% protected against 6A. Mice passively immunized with sera from infants immunized with four doses of an 11-valent pneumococcal conjugate vaccine (containing 19F conjugated to tetanus toxoid) were given an intranasal challenge with 19A organisms in the same model [29]. Of 100 mice passively immunized and then challenged, 60 mice had no 19A organisms detected in lung tissue, whereas organisms were identified in all mice given saline placebo. However, passive immunization did not protect against challenge with 19F organisms in this model; therefore, the relevance of the model for serogroup 19 is questionable. In general these models provide evidence of some biological impact of 6B immunization on 6A organisms although the effect on the heterologous serotype was not as great as that observed with the homologous serotype. The impact of 19F immunization on 19A organisms is not well understood from these models.</div>
</li> <heading id="h-0011">Impact of 6B and 19F Polysaccharide Conjugate Immunization on 6A and 19A Disease in Efficacy/Effectiveness Trials</heading>
<li> <para-num num="[0033]"> </para-num> <div class="description-line" id="p-0035" num="0033">The number of cases of disease due to the 6B, 6A, 19F and 19A serotypes in 7vPnC and 9vPnC (7vPnC plus serotypes 1 and 5) efficacy trials is noted in Table 1 [30, 10, 31]. The numbers of invasive disease cases are too small to allow any conclusions to be drawn for serotypes 6A and 19A. However, the Finnish otitis media trial generated a large number of pneumococcal isolates [32]. In the per protocol analysis 7vPnC was 84% (95% CI 62%, 93%) efficacious against otitis media due to serotype 6B and 57% (95% CI 24%, 76%) efficacious against otitis media due to serotype 6A (Table 1). In contrast, serotype-specific efficacy with the 7vPnC was not demonstrated for otitis media due to either 19F or 19A.</div>
</li> <li> <div class="description-line" id="p-0036" num="0000">
<tables id="TABLE-US-00001" num="00001">
<patent-tables colsep="0" frame="none" pgwide="1" rowsep="0">
<table align="left" class="description-table" cols="1" colsep="0" rowsep="0" width="100%">
<thead>
<tr class="description-tr">
<td class="description-td" colspan="1" nameend="1" namest="1" rowsep="1">TABLE 1</td>
</tr>
</thead>
<tbody><tr class="description-tr">
<td align="center" class="description-td" colspan="1" nameend="1" namest="1" rowsep="1"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">Cases of Pneumococcal Disease Due to Serotypes 6B, 6A, 19F,</td>
</tr>
<tr class="description-tr">
<td class="description-td">and 19A in Efficacy Trials with the 7vPnC and 9vPnC Vaccines</td>
</tr>
</tbody></table>
<table align="left" class="description-table" cols="5" colsep="0" rowsep="0" width="100%">
<tbody><tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">6B</td>
<td class="description-td">6A</td>
<td class="description-td">19F</td>
<td class="description-td">19A</td>
</tr>
</tbody></table>
<table align="left" class="description-table" cols="9" colsep="0" rowsep="0" width="100%">
<tbody><tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">PnC</td>
<td class="description-td">Contr.</td>
<td class="description-td">PnC</td>
<td class="description-td">Contr.</td>
<td class="description-td">PnC</td>
<td class="description-td">Contr.</td>
<td class="description-td">PnC</td>
<td class="description-td">Contr.</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td align="center" class="description-td" colspan="9" nameend="8" namest="offset" rowsep="1"> </td>
</tr>
</tbody></table>
<table align="left" class="description-table" cols="9" colsep="0" rowsep="0" width="100%">
<tbody><tr class="description-tr">
<td class="description-td">Kaiser Efficacy Trial - 7vPnC</td>
<td class="description-td">1</td>
<td class="description-td">7</td>
<td class="description-td">0</td>
<td class="description-td">1</td>
<td class="description-td">â2*</td>
<td class="description-td">13</td>
<td class="description-td">0</td>
<td class="description-td">1</td>
</tr>
<tr class="description-tr">
<td class="description-td">(ITT)</td>
</tr>
<tr class="description-tr">
<td class="description-td">Navajo Efficacy Trial - 7vPnC</td>
<td class="description-td">0</td>
<td class="description-td">5</td>
<td class="description-td">1</td>
<td class="description-td">0</td>
<td class="description-td">1</td>
<td class="description-td">1</td>
<td class="description-td">1</td>
<td class="description-td">0</td>
</tr>
<tr class="description-tr">
<td class="description-td">(ITT)</td>
</tr>
<tr class="description-tr">
<td class="description-td">South African Efficacy Trial -</td>
<td class="description-td">1</td>
<td class="description-td">2</td>
<td class="description-td">1</td>
<td class="description-td">0</td>
<td class="description-td">0</td>
<td class="description-td">1</td>
<td class="description-td">3</td>
<td class="description-td">1</td>
</tr>
<tr class="description-tr">
<td class="description-td">9vPnC HIV (â) (ITT)</td>
</tr>
<tr class="description-tr">
<td class="description-td">South African Efficacy Trial -</td>
<td class="description-td">1</td>
<td class="description-td">7</td>
<td class="description-td">3</td>
<td class="description-td">10</td>
<td class="description-td">2</td>
<td class="description-td">3</td>
<td class="description-td">2</td>
<td class="description-td">3</td>
</tr>
<tr class="description-tr">
<td class="description-td">9vPnC HIV (+) (ITT)</td>
</tr>
<tr class="description-tr">
<td class="description-td">Finnish Otitis Media Trial -</td>
<td class="description-td">â9*</td>
<td class="description-td">56</td>
<td class="description-td">19*</td>
<td class="description-td">45</td>
<td class="description-td">43â</td>
<td class="description-td">58</td>
<td class="description-td">17</td>
<td class="description-td">26</td>
</tr>
<tr class="description-tr">
<td class="description-td">7vPnC (PP)</td>
</tr>
<tr class="description-tr">
<td align="center" class="description-td" colspan="9" nameend="9" namest="1" rowsep="1"> </td>
</tr>
<tr class="description-tr">
<td align="left" class="description-td" colspan="9" id="FOO-00001" nameend="9" namest="1">*Statistically significant efficacy demonstrated From references 30, 10 and 33, and personal communications</td>
</tr>
<tr class="description-tr">
<td align="left" class="description-td" colspan="9" id="FOO-00002" nameend="9" namest="1">Contr = control</td>
</tr>
<tr class="description-tr">
<td align="left" class="description-td" colspan="9" id="FOO-00003" nameend="9" namest="1">ITT = intention to treat analysis</td>
</tr>
<tr class="description-tr">
<td align="left" class="description-td" colspan="9" id="FOO-00004" nameend="9" namest="1">PP = per protocol analysis</td>
</tr>
</tbody></table>
</patent-tables>
</tables>
</div>
</li> <li> <para-num num="[0034]"> </para-num> <div class="description-line" id="p-0037" num="0034">Post-marketing IPD surveillance data is also available from a case-control trial conducted by the Centers for Disease Control to evaluate the effectiveness of Prevnar [33]. Cases of pneumococcal invasive disease occurring in children 3 to 23 months of age were identified in the surveillance laboratories and matched with three control cases by age and zip code. After obtaining consent, medical and immunization history (subjects were considered immunized if they had received at least one dose of Prevnar) was obtained from parents and medical providers for cases and controls. The preliminary results were presented at the 2003 ICAAC meeting and a summary of the findings for 6B, 19F, 19A and 6A disease is presented in Table 2. These data indicate that Prevnar is able to prevent disease due to 6A, although at a level that may be somewhat lower than serotype 6B disease. These data also indicate that the cross-protection for invasive disease due to 19A is limited.</div>
</li> <li> <div class="description-line" id="p-0038" num="0000">
<tables id="TABLE-US-00002" num="00002">
<patent-tables colsep="0" frame="none" rowsep="0">
<table align="left" class="description-table" cols="1" colsep="0" rowsep="0" width="100%">
<thead>
<tr class="description-tr">
<td class="description-td" colspan="1" nameend="1" namest="1" rowsep="1">TABLE 2</td>
</tr>
</thead>
<tbody><tr class="description-tr">
<td align="center" class="description-td" colspan="1" nameend="1" namest="1" rowsep="1"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">Preliminary results of a Case Control Trial Performed</td>
</tr>
<tr class="description-tr">
<td class="description-td">by the CDC (presented at ICAAC, 2003)</td>
</tr>
</tbody></table>
<table align="left" class="description-table" cols="4" colsep="0" rowsep="0" width="100%">
<tbody><tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">Informative</td>
<td class="description-td">VE*</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">Serotype</td>
<td class="description-td">Sets, n</td>
<td class="description-td">(95% CI)</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td align="center" class="description-td" colspan="4" nameend="3" namest="offset" rowsep="1"> </td>
</tr>
</tbody></table>
<table align="left" class="description-table" cols="4" colsep="0" rowsep="0" width="100%">
<tbody><tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">Vaccine Type, All</td>
<td class="description-td">115</td>
<td class="description-td">94</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">(87, 97)</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">Vaccine Related, All</td>
<td class="description-td">36</td>
<td class="description-td">70</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">(38, 86)</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">Non-Vaccine Type, All</td>
<td class="description-td">43</td>
<td class="description-td">â4</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">(â106, 48)â</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">6B</td>
<td class="description-td">27</td>
<td class="description-td">94</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">(72, 99)</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">19F</td>
<td class="description-td">19</td>
<td class="description-td">73</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">(16, 92)</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">6A</td>
<td class="description-td">15</td>
<td class="description-td">87</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">(53, 97)</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">19A</td>
<td class="description-td">16</td>
<td class="description-td">40</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">(â87, 80)â</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td align="center" class="description-td" colspan="4" nameend="3" namest="offset" rowsep="1"> </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td align="left" class="description-td" colspan="4" id="FOO-00005" nameend="3" namest="offset">*Vaccine effectiveness comparing vaccinated (â¥1 dose) vs. unvaccinated, and adjusted for underlying conditions Reference 40 and personal/confidential communication</td>
</tr>
</tbody></table>
</patent-tables>
</tables>
</div>
</li> <li> <para-num num="[0035]"> </para-num> <div class="description-line" id="p-0039" num="0035">A published analysis [3] of the use of Prevnar also indicated that serotypes 6B and 19F conferred a moderate reduction in IPD caused by serotypes 6A and 19A among children under two years of age (Table 1 in [3]). Disease rates among unimmunized adults caused by serotypes 6A, 9A, 9L, 9N, 18A, 18B, 18F, 19A, 19B, 19C, 23A and 23B (âall vaccine-related serotypesâ) were somewhat reduced (Table 2 in [3]). These data establish that herd immunity from the use of Prevnar in children under two years of age was modest for serotypes 6A and 19A, and provide a basis for the inclusion of serotypes 6A and 19A in the 13vPnC vaccine of this invention.</div>
</li> <heading id="h-0012">Conclusion for Addition of 6A and 19A</heading>
<li> <para-num num="[0036]"> </para-num> <div class="description-line" id="p-0040" num="0036">The post-marketing surveillance data and the case-control study results noted in <figref idrefs="DRAWINGS">FIG. 1</figref> and Table 2 with the 7vPnC vaccine suggest that, consistent with the other information on immune responses and performance in the animals models described above, there may be some cross-protection against 6A disease, but to a lesser extent than to 6B disease. Furthermore, it appears the protection against 19A is limited. Therefore, a 13vPnC vaccine containing serotypes 6A and 19A provides coverage that is not dependent on the limitations of serogroup cross-protection by serotypes 6B and 19F.</div>
</li> <li> <para-num num="[0037]"> </para-num> <div class="description-line" id="p-0041" num="0037">Accordingly, the present invention provides a multivalent immunogenic composition comprising 13 distinct polysaccharide-protein conjugates, wherein each of the conjugates contains a different capsular polysaccharide conjugated to a carrier protein, and wherein the capsular polysaccharides are prepared from serotypes 1, 3, 4, 5, 6A, 6B, 7F, 9V, 14, 18C, 19A, 19F and 23F of <i>Streptococcus pneumoniae</i>, together with a physiologically acceptable vehicle. One such carrier protein is the diphtheria toxoid designated CRM<sub>197</sub>. The immunogenic composition may further comprise an adjuvant, such as an aluminum-based adjuvant, such as aluminum phosphate, aluminum sulfate and aluminum hydroxide.</div>
</li> <li> <para-num num="[0038]"> </para-num> <div class="description-line" id="p-0042" num="0038">Capsular polysaccharides are prepared by standard techniques known to those skilled in the art. In the present invention, capsular polysaccharides are prepared from serotypes 1, 3, 4, 5, 6A, 6B, 7F, 9V, 14, 18C, 19A, 19F and 23F of <i>Streptococcus pneumoniae</i>. These pneumococcal conjugates are prepared by separate processes and formulated into a single dosage formulation. For example, in one embodiment, each pneumococcal polysaccharide serotype is grown in a soy-based medium. The individual polysaccharides are then purified through centrifugation, precipitation, ultra-filtration, and column chromatography. The purified polysaccharides are chemically activated to make the saccharides capable of reacting with the carrier protein.</div>
</li> <li> <para-num num="[0039]"> </para-num> <div class="description-line" id="p-0043" num="0039">Once activated, each capsular polysaccharide is separately conjugated to a carrier protein to form a glycoconjugate. In one embodiment, each capsular polysaccharide is conjugated to the same carrier protein. In this embodiment, the conjugation is effected by reductive amination.</div>
</li> <li> <para-num num="[0040]"> </para-num> <div class="description-line" id="p-0044" num="0040">The chemical activation of the polysaccharides and subsequent conjugation to the carrier protein are achieved by conventional means. See, for example, U.S. Pat. Nos. 4,673,574 and 4,902,506 [34, 35].</div>
</li> <li> <para-num num="[0041]"> </para-num> <div class="description-line" id="p-0045" num="0041">Carrier proteins are preferably proteins that are non-toxic and non-reactogenic and obtainable in sufficient amount and purity. Carrier proteins should be amenable to standard conjugation procedures. In a particular embodiment of the present invention, CRM<sub>197 </sub>is used as the carrier protein.</div>
</li> <li> <para-num num="[0042]"> </para-num> <div class="description-line" id="p-0046" num="0042">CRM<sub>197 </sub>(Wyeth, Sanford, N.C.) is a non-toxic variant (i.e., toxoid) of diphtheria toxin isolated from cultures of <i>Corynebacterium diphtheria </i>strain C7 (Î²197) grown in casamino acids and yeast extract-based medium. CRM<sub>197 </sub>is purified through ultra-filtration, ammonium sulfate precipitation, and ion-exchange chromatography. Alternatively, CRM<sub>197 </sub>is prepared recombinantly in accordance with U.S. Pat. No. 5,614,382, which is hereby incorporated by reference. Other diphtheria toxoids are also suitable for use as carrier proteins.</div>
</li> <li> <para-num num="[0043]"> </para-num> <div class="description-line" id="p-0047" num="0043">Other suitable carrier proteins include inactivated bacterial toxins such as tetanus toxoid, pertussis toxoid, cholera toxoid (e.g., as described in International Patent Application WO2004/083251 [38]), <i>E. coli </i>LT, <i>E. coli </i>ST, and exotoxin A from <i>Pseudomonas aeruginosa</i>. Bacterial outer membrane proteins such as outer membrane complex c (OMPC), porins, transferrin binding proteins, pneumolysin, pneumococcal surface protein A (PspA), pneumococcal adhesin protein (PsaA), C5a peptidase from Group A or Group B <i>streptococcus</i>, or <i>Haemophilus influenzae </i>protein D, can also be used. Other proteins, such as ovalbumin, keyhole limpet hemocyanin (KLH), bovine serum albumin (BSA) or purified protein derivative of tuberculin (PPD) can also be used as carrier proteins.</div>
</li> <li> <para-num num="[0044]"> </para-num> <div class="description-line" id="p-0048" num="0044">After conjugation of the capsular polysaccharide to the carrier protein, the polysaccharide-protein conjugates are purified (enriched with respect to the amount of polysaccharide-protein conjugate) by a variety of techniques. These techniques include concentration/diafiltration operations, precipitation/elution, column chromatography, and depth filtration. See examples below.</div>
</li> <li> <para-num num="[0045]"> </para-num> <div class="description-line" id="p-0049" num="0045">After the individual glycoconjugates are purified, they are compounded to formulate the immunogenic composition of the present invention, which can be used as a vaccine. Formulation of the immunogenic composition of the present invention can be accomplished using art-recognized methods. For instance, the 13 individual pneumococcal conjugates can be formulated with a physiologically acceptable vehicle to prepare the composition. Examples of such vehicles include, but are not limited to, water, buffered saline, polyols (e.g., glycerol, propylene glycol, liquid polyethylene glycol) and dextrose solutions.</div>
</li> <li> <para-num num="[0046]"> </para-num> <div class="description-line" id="p-0050" num="0046">In certain embodiments, the immunogenic composition will comprise one or more adjuvants. As defined herein, an âadjuvantâ is a substance that serves to enhance the immunogenicity of an immunogenic composition of this invention. Thus, adjuvants are often given to boost the immune response and are well known to the skilled artisan. Suitable adjuvants to enhance effectiveness of the composition include, but are not limited to:</div>
</li> <li> <para-num num="[0047]"> </para-num> <div class="description-line" id="p-0051" num="0047">(1) aluminum salts (alum), such as aluminum hydroxide, aluminum phosphate, aluminum sulfate, etc.;</div>
</li> <li> <para-num num="[0048]"> </para-num> <div class="description-line" id="p-0052" num="0048">(2) oil-in-water emulsion formulations (with or without other specific immunostimulating agents such as muramyl peptides (defined below) or bacterial cell wall components), such as, for example,
</div> </li> <ul> <li id="ul0001-0001" num="0000"> <ul> <li id="ul0002-0001" num="0049">(a) MF59 (PCT Publ. No. WO 90/14837), containing 5% Squalene, 0.5% Tween 80, and 0.5% Span 85 (optionally containing various amounts of MTP-PE (see below, although not required)) formulated into submicron particles using a microfluidizer such as Model 110Y microfluidizer (Microfluidics, Newton, Mass.),</li> <li id="ul0002-0002" num="0050">(b) SAF, containing 10% Squalene, 0.4% Tween 80, 5% pluronic-blocked polymer L121, and thr-MDP (see below) either microfluidized into a submicron emulsion or vortexed to generate a larger particle size emulsion, and</li> <li id="ul0002-0003" num="0051">(c) Ribiâ¢ adjuvant system (RAS), (Corixa, Hamilton, Mont.) containing 2% Squalene, 0.2% Tween 80, and one or more bacterial cell wall components from the group consisting of 3-O-deaylated monophosphorylipid A (MPLâ¢) described in U.S. Pat. No. 4,912,094 (Corixa), trehalose dimycolate (TDM), and cell wall skeleton (CWS), preferably MPL+CWS (Detoxm);</li> </ul> </li> </ul>
<li> <para-num num="[0052]"> </para-num> <div class="description-line" id="p-0053" num="0052">(3) saponin adjuvants, such as Quil A or STIMULONâ¢ QS-21 (Antigenics, Framingham, Mass.) (U.S. Pat. No. 5,057,540) may be used or particles generated therefrom such as ISCOMs (immunostimulating complexes);</div>
</li> <li> <para-num num="[0053]"> </para-num> <div class="description-line" id="p-0054" num="0053">(4) bacterial lipopolysaccharides, synthetic lipid A analogs such as aminoalkyl glucosamine phosphate compounds (AGP), or derivatives or analogs thereof, which are available from Corixa, and which are described in U.S. Pat. No. 6,113,918; one such AGP is 2-[(R)-3-Tetradecanoyloxytetradecanoylamino]ethyl 2-Deoxy-4-O-phosphono-3-Oâ[(R)-3-tetradecanoyloxytetradecanoyl]-2-[(R)-3-tetradecanoyloxytetradecanoylamino]-b-D-glucopyranoside, which is also know as 529 (formerly known as RC529), which is formulated as an aqueous form or as a stable emulsion, synthetic polynucleotides such as oligonucleoties containing CpG motif(s) (U.S. Pat. No. 6,207,646);</div>
</li> <li> <para-num num="[0054]"> </para-num> <div class="description-line" id="p-0055" num="0054">(5) cytokines, such as interleukins (e.g., IL-1, IL-2, IL-4, IL-5, IL-6, IL-7, IL-12, IL-15, IL-18, etc.), interferons (e.g., gamma interferon), granulocyte macrophage colony stimulating factor (GM-CSF), macrophage colony stimulating factor (M-CSF), tumor necrosis factor (TNF), costimulatory molecules B7-1 and B7-2, etc.;</div>
</li> <li> <para-num num="[0055]"> </para-num> <div class="description-line" id="p-0056" num="0055">(6) detoxified mutants of a bacterial ADP-ribosylating toxin such as a cholera toxin (CT) either in a wild-type or mutant form, for example, where the glutamic acid at amino acid position 29 is replaced by another amino acid, preferably a histidine, in accordance with published international patent application number WO 00/18434 (see also WO 02/098368 and WO 02/098369), a pertussis toxin (PT), or an <i>E. coli </i>heat-labile toxin (LT), particularly LT-K63, LT-R72, CT-S109, PT-K9/G129 (see, e.g., WO 93/13302 and WO 92/19265); and</div>
</li> <li> <para-num num="[0056]"> </para-num> <div class="description-line" id="p-0057" num="0056">(7) other substances that act as immunostimulating agents to enhance the effectiveness of the composition.</div>
</li> <li> <para-num num="[0057]"> </para-num> <div class="description-line" id="p-0058" num="0057">Muramyl peptides include, but are not limited to, N-acetyl-muramyl-L-threonyl-D-isoglutamine (thr-MDP), N-acetylnormuramyl-L-alanine-2-(1â²-2â² dipalmitoyl-sn-glycero-3-hydroxyphosphoryloxy)-ethylamine (MTP-PE), etc.</div>
</li> <li> <para-num num="[0058]"> </para-num> <div class="description-line" id="p-0059" num="0058">The vaccine formulations of the present invention can be used to protect or treat a human susceptible to pneumococcal infection, by means of administering the vaccine via a systemic or mucosal route. These administrations can include injection via the intramuscular, intraperitoneal, intradermal or subcutaneous routes; or via mucosal administration to the oral/alimentary, respiratory or genitourinary tracts. In one embodiment, intranasal administration is used for the treatment of pneumonia or otitis media (as nasopharyngeal carriage of pneumococci can be more effectively prevented, thus attenuating infection at its earliest stage).</div>
</li> <li> <para-num num="[0059]"> </para-num> <div class="description-line" id="p-0060" num="0059">The amount of conjugate in each vaccine dose is selected as an amount that induces an immunoprotective response without significant, adverse effects. Such amount can vary depending upon the pneumococcal serotype. Generally, each dose will comprise 0.1 to 100 Î¼g of polysaccharide, particularly 0.1 to 10 Î¼g, and more particularly 1 to 5 Î¼g.</div>
</li> <li> <para-num num="[0060]"> </para-num> <div class="description-line" id="p-0061" num="0060">Optimal amounts of components for a particular vaccine can be ascertained by standard studies involving observation of appropriate immune responses in subjects. Following an initial vaccination, subjects can receive one or several booster immunizations adequately spaced.</div>
</li> <li> <para-num num="[0061]"> </para-num> <div class="description-line" id="p-0062" num="0061">In a particular embodiment of the present invention, the 13vPnC vaccine is a sterile liquid formulation of pneumococcal capsular polysaccharides of serotypes 1, 3, 4, 5, 6A, 6B, 7F, 9V, 14, 18C, 19A, 19F, and 23F individually conjugated to CRM<sub>197</sub>. Each 0.5 mL dose is formulated to contain: 2 Î¼g of each saccharide, except for 6B at 4 Î¼g; approximately 29 Î¼g CRM<sub>197 </sub>carrier protein; 0.125 mg of elemental aluminum (0.5 mg aluminum phosphate) adjuvant; and sodium chloride and sodium succinate buffer as excipients. The liquid is filled into single dose syringes without a preservative. After shaking, the vaccine is a homogeneous, white suspension ready for intramuscular administration.</div>
</li> <li> <para-num num="[0062]"> </para-num> <div class="description-line" id="p-0063" num="0062">The choice of dose level for the 13vPnC vaccine is similar to the marketed 7vPnC vaccine (Prevnar). The 2 Î¼g saccharide dose level was selected for all serotypes, except for 6B, which is at 4 Î¼g per dose. The 7vPnC vaccine has shown desirable safety, immunogenicity, and efficacy against IPD in the 2 Î¼g saccharide dose level for serotypes 4, 9V, 14, 18C, 19F and 23F, and at the 4 Î¼g dose for 6B.</div>
</li> <li> <para-num num="[0063]"> </para-num> <div class="description-line" id="p-0064" num="0063">The immunization schedule can follow that designated for the 7vPnC vaccine. For example, the routine schedule for infants and toddlers against invasive disease caused by <i>S. pneumoniae </i>due to the serotypes included in the 13vPnC vaccine is 2, 4, 6 and 12-15 months of age. The compositions of this invention are also suitable for use with older children, adolescents and adults.</div>
</li> <li> <para-num num="[0064]"> </para-num> <div class="description-line" id="p-0065" num="0064">The compositions of this invention may further include one or more additional antigens for use against otitis media caused by infection with other bacteria. Such bacteria include nontypable <i>Haemophlus influenza, Moraxella catarnhalis </i>(formerly known as <i>Branhamela catarrhalis</i>) and <i>Alloiococcus otitidis. </i> </div>
</li> <li> <para-num num="[0065]"> </para-num> <div class="description-line" id="p-0066" num="0065">Examples of nontypable <i>Haemophitis influenzae </i>antigens suitable for inclusion include the P4 protein, also known as protein âeâ (U.S. Pat. No. 5,601,831; International Patent Application WO03/078453), the P6 protein, also known as the PAL or the PBOMP-1 protein (U.S. Pat. No. 5,110,908; International Patent Application WO0100790), the P5 protein (U.S. Reissue Pat. No. 37,741), the <i>Haemophilus </i>adhesion and penetration protein (U.S. Pat. Nos. 6,245,337 and 6,676,948), the LKP tip adhesin protein (U.S. Pat. No. 5,643,725) and the NucA protein (U.S. Pat. No. 6,221,365).</div>
</li> <li> <para-num num="[0066]"> </para-num> <div class="description-line" id="p-0067" num="0066">Examples of <i>Moraxella catarrhalis </i>antigens suitable for inclusion include the UspA2 protein (U.S. Pat. Nos. 5,552,146, 6,310,190), the CD protein (U.S. Pat. No. 5,725,862), the E protein (U.S. Pat. No. 5,948,412) and the 74 kilodalton outer membrane protein (U.S. Pat. No. 6,899,885).</div>
</li> <li> <para-num num="[0067]"> </para-num> <div class="description-line" id="p-0068" num="0067">Examples of <i>Alloiococcus otitidis </i>antigens suitable for inclusion include those identified in International Patent Application WO03/048304.</div>
</li> <li> <para-num num="[0068]"> </para-num> <div class="description-line" id="p-0069" num="0068">The compositions of this invention may also include one or more proteins from <i>Streptococcus pneumoniae</i>. Examples of <i>Streptococcus pneumoniae </i>proteins suitable for inclusion include those identified in International Patent Application WO02/083855, as well as that described in International Patent Application WO02/053761.</div>
</li> <li> <para-num num="[0069]"> </para-num> <div class="description-line" id="p-0070" num="0069">The compositions of this invention may further include one or more proteins from <i>Neisseria meningitidis </i>type B. Examples of <i>Neisseria meningitidis </i>type B proteins suitable for inclusion include those identified in International Patent Applications WO03/063766, WO2004/094596, WO01/85772, WO02/16612 and WO01/87939.</div>
</li> <li> <div class="description-line" id="p-0071" num="0000">Co-Lyophilization and Conjugation Process for <i>S. pneumoniae </i>Serotype 19A Polysaccharide</div>
</li> <li> <para-num num="[0070]"> </para-num> <div class="description-line" id="p-0072" num="0070">Serotype 19A is much more prone to thermal degradation than other <i>S. pneumoniae </i>serotypes due to the presence of phosphodiester linkages between its subunits. In order to improve the conjugation efficiency and to control the stability of the inherently labile serotype 19A polysaccharide, a co-lyophilization and conjugation process in the presence of dimethyl sulfoxide (DMSO) is used in the conjugation of the serotype 19A polysaccharide to the carrier protein CRM<sub>197</sub>. This process provides improved conjugate characteristics in terms of molecular size and the percentage of free saccharide for serotype 19A as compared to the use of conjugation processes involving discrete lyophilization of polysaccharides and carrier proteins in DMSO or processes involving aqueous co-lyophilization of polysaccharides and carrier proteins without DMSO.</div>
</li> <li> <para-num num="[0071]"> </para-num> <div class="description-line" id="p-0073" num="0071">As described in more detail in Example 17 below, conjugation of the serotype 19A polysaccharide to the carrier protein, CRM<sub>197</sub>, is a two reaction-step process. The first step involves periodate oxidation (activation) to generate reactive aldehyde groups on the polysaccharide. The activated polysaccharide is then purified by ultrafiltration to remove saccharide fragments and small molecule reaction by-products. The activated polysaccharide and CRM<sub>197 </sub>are then combined and co-lyophilized with sucrose as a cryoprotectant. The conjugation step is performed in DMSO via a reductive amination mechanism in the presence of sodium cyanoborohydride. Unreacted aldehyde groups are reduced (capped) by the addition of sodium borohydride. The conjugate is then purified to remove unreacted CRM<sub>197 </sub>and saccharide fragments (e.g., by diafiltration versus phosphate buffer followed by buffered saline), giving a final batch concentrate of the glycoconjugate in buffered saline.</div>
</li> <li> <para-num num="[0072]"> </para-num> <div class="description-line" id="p-0074" num="0072">Although a preferred carrier protein within the process described above is the mutated diphtheria toxin CRM<sub>197</sub>, other carrier proteins may be used for conjugation with the serotype 19A polysaccharide within the present methods. Carrier proteins are chosen to increase the immunogenicity of the bound serotype 19A polysaccharide and/or to elicit antibodies against the carrier protein which are diagnostically, analytically and/or therapeutically beneficial. Covalent linking of an antigenic molecule (e.g., a polysaccharide) to a carrier confers enhanced immunogenicity and T-cell dependence (Pozsgay et al. (1999) PNAS, 96:5194-97; Lee et al. (1976) J. Immunol., 116:1711-18; Dintzis et al. (1976) PNAS, 73:3671-75). As described herein, useful carrier proteins include inactivated bacterial toxins such as tetanus toxoid, pertussis toxoid, cholera toxoid (e.g., as described in International Patent Application WO2004/083251), <i>E. coli </i>LT, <i>E. coli </i>ST, and exotoxin A from <i>Pseudomonas aeruginosa</i>. Bacterial outer membrane proteins such as outer membrane complex c, porins, transferrin binding proteins, pneumolysin, pneumococcal surface protein A, pneumococcal adhesin protein, C5a peptidase from Group A or Group B <i>streptococcus</i>, or <i>Haemophilus influenzae </i>protein D, can also be used. Other proteins, such as ovalbumin, keyhole limpet hemocyanin, bovine serum albumin, or purified protein derivative of tuberculin can also be used as carrier proteins.</div>
</li> <li> <para-num num="[0073]"> </para-num> <div class="description-line" id="p-0075" num="0073">Although this co-lyophilization and conjugation process in DMSO is described for use with the serotype 19A polysaccharide, this process may also be used for serotypes that are structurally similar to serotype 19A, such as 6A, 6B, and 19F which also contain phosphodiester linkages between their repeat units.</div>
</li> <li> <para-num num="[0074]"> </para-num> <div class="description-line" id="p-0076" num="0074">The above disclosure generally describes the present invention. A more complete understanding can be obtained by reference to the following specific examples. These examples are described solely for the purpose of illustration and are not intended to limit the scope of the invention.</div>
</li> <heading id="h-0013">EXAMPLES</heading>
<heading id="h-0014">Example 1</heading>
<heading id="h-0015">Preparation of <i>S. Pneumoniae </i>Capsular Polysaccharide Serotype 1</heading>
<heading id="h-0016">Preparation of Master and Working Cell Banks</heading>
<li> <para-num num="[0075]"> </para-num> <div class="description-line" id="p-0077" num="0075"> <i>S. pneumoniae </i>serotype 1 was obtained from the American Type Culture Collection, ATCC, strain 6301. Several generations of seed stocks were created in order to expand the strain and remove components of animal origin (generations F1, F2, and F3). Two additional generations of seed stocks were produced. The first additional generation was made from an F3 vial, and the subsequent generation was made from a vial of the first additional generation. Seed vials were stored frozen (&lt;â70Â° C.) with synthetic glycerol as a cryopreservative. In addition to frozen vials, lyophilized vials were prepared for the F4 generation. For cell bank preparation, all cultures were grown in a soy-based medium. Prior to freezing, cells were concentrated by centrifugation, spent medium was removed, and cell pellets were resuspended in fresh medium containing a cryopreservative, such as synthetic glycerol.</div>
</li> <heading id="h-0017">Fermentation and Harvesting</heading>
<li> <para-num num="[0076]"> </para-num> <div class="description-line" id="p-0078" num="0076">Cultures from the working cell bank were used to inoculate seed bottles containing a soy-based medium. The bottles were incubated at 36Â° C.Â±2Â° C. without agitation until growth requirements were met. A seed bottle was used to inoculate a seed fermentor containing soy-based medium. A pH of about 7.0 was maintained with sterile sodium carbonate solution. After the target optical density was reached, the seed fermentor was used to inoculate the production fermentor containing soy-based medium. The pH was maintained with sterile sodium carbonate solution. The fermentation was terminated after cessation of growth or when the working volume of the fermentor was reached. An appropriate amount of sterile 12% deoxycholate sodium was added to the culture to lyse the bacterial cells and release cell-associated polysaccharide. After lysing, the fermentor contents were cooled. The pH of the lysed culture broth was adjusted to approximately pH 6.6 with acetic acid. The lysate was clarified by continuous flow centrifugation followed by depth filtration and 0.45 Î¼m microfiltration.</div>
</li> <li> <para-num num="[0077]"> </para-num> <div class="description-line" id="p-0079" num="0077">In an alternate process, the fermentation pH of about 7.0 was maintained with 3N NaOH. After the target optical density was reached, the seed fermentor was used to inoculate the production fermentor containing soy-based medium. The pH was maintained with 3N NaOH. The fermentation was terminated after cessation of growth or when the working volume of the fermentor was reached. An appropriate amount of sterile 12% deoxycholate sodium was added to the culture to obtain a 0.12% concentration in the broth, to lyse the bacterial cells and release cell-associated polysaccharide. After lysing, the fermentor contents were held, with agitation, for a time interval between 8 and 24 hours at a temperature between 7Â° C. and 13Â° C., to assure that complete cellular lysis and polysaccharide release had occurred. Agitation during this hold period prevented lysate sediment from settling on the fermentor walls and pH probe, thereby allowing the pH probe integrity to be maintained. Next, the pH of the lysed culture broth was adjusted to approximately pH 5.0 with 50% acetic acid. After a hold time without agitation, for a time interval between 12 and 24 hours at a temperature between 15Â° C. and 25C, a significant portion of the previously soluble proteins dropped out of solution as a solid precipitate with little loss or degradation of the polysaccharide, which remained in solution. The solution with the precipitate was then clarified by continuous flow centrifugation followed by depth filtration and 0.45 Î¼m microfiltration.</div>
</li> <heading id="h-0018">Purification</heading>
<li> <para-num num="[0078]"> </para-num> <div class="description-line" id="p-0080" num="0078">The purification of the pneumococcal polysaccharide consisted of several concentration/diafiltration operations, precipitation/elution, column chromatography, and depth filtration steps. All procedures were performed at room temperature unless otherwise specified.</div>
</li> <li> <para-num num="[0079]"> </para-num> <div class="description-line" id="p-0081" num="0079">Clarified broth from the fermentor cultures of <i>S. pneumoniae </i>serotype 1 were concentrated and diafiltered using a 100 kDa MWCO (kilodalton molecular weight cutoff) filter. Diafiltration was accomplished using sodium phosphate buffer at neutral pH. Diafiltration removed the low molecular weight medium components from the higher molecular weight biopolymers such as nucleic acid, protein and polysaccharide.</div>
</li> <li> <para-num num="[0080]"> </para-num> <div class="description-line" id="p-0082" num="0080">The polysaccharide was precipitated from the concentrated and diafiltered solution by adding hexadecyltrimethyl ammonium bromide (HB) from a stock solution to give a final concentration of 1% HB (w/v). The polysaccharide/HB precipitate was captured on a depth filter and the filtrate was discarded. The polysaccharide precipitate was resolubilized and eluted by recirculating a sodium chloride solution through the precipitate-containing depth filter. The filters were then rinsed with additional sodium chloride solution.</div>
</li> <li> <para-num num="[0081]"> </para-num> <div class="description-line" id="p-0083" num="0081">Sodium iodide (NaI) was added to the polysaccharide solution from a stock NaI solution to achieve a final concentration of 0.5% to precipitate HB. The precipitate was removed by depth filtration. The filtrate contains the target polysaccharide. The precipitation vessel and the filter were rinsed with a NaCl/NaI solution and the rinse was combined, with the partially purified polysaccharide solution. The filter was discarded. The polysaccharide was then filtered through a 0.2 Î¼m filter.</div>
</li> <li> <para-num num="[0082]"> </para-num> <div class="description-line" id="p-0084" num="0082">The polysaccharide solution was concentrated on a 30 kDa MWCO ultrafilter and diafiltered with a sodium chloride solution.</div>
</li> <li> <para-num num="[0083]"> </para-num> <div class="description-line" id="p-0085" num="0083">The partially purified polysaccharide solution was further purified by filtration through a depth filter impregnated with activated carbon. After filtration, the carbon filter was rinsed with a sodium chloride solution. The rinse is combined with the polysaccharide solution, which is then filtered through a 0.2 Î¼m filter.</div>
</li> <li> <para-num num="[0084]"> </para-num> <div class="description-line" id="p-0086" num="0084">The polysaccharide solution was concentrated on a 30 kDa MWCO ultrafilter and adjusted with a 1M sodium phosphate buffer to achieve a final concentration of 0.025 M sodium phosphate. The pH was checked and adjusted to 7.0Â±0.2.</div>
</li> <li> <para-num num="[0085]"> </para-num> <div class="description-line" id="p-0087" num="0085">The ceramic hydroxyapatite (HA) column was equilibrated with sodium phosphate buffer containing sodium chloride to obtain the appropriate conductivity (&lt;15 Î¼S). The polysaccharide solution was then loaded onto the column. Under these conditions, impurities bound to the resin and the polysaccharide was recovered in the flow-through from the column. The polysaccharide solution was filtered through 02 Î¼m inline filters located before and after the column.</div>
</li> <li> <para-num num="[0086]"> </para-num> <div class="description-line" id="p-0088" num="0086">The polysaccharide solution was concentrated using a 30 kDa MWCO filter. The concentrate was then diafiltered with Water for Injection (WFI).</div>
</li> <li> <para-num num="[0087]"> </para-num> <div class="description-line" id="p-0089" num="0087">The diafiltered polysaccharide solution was filtered through a 0.2 Î¼m membrane filter into polypropylene bottles. Samples were removed for release testing and the purified polysaccharide was stored frozen at â25Â°Â±5Â° C.</div>
</li> <heading id="h-0019">Characterization</heading>
<li> <para-num num="[0088]"> </para-num> <div class="description-line" id="p-0090" num="0088">The 1H-NMR data was consistent with the chemical structure by the assignment of signals assigned to the protons of the polysaccharide molecule. The 1H-NMR spectrum showed a series of well-resolved signals (protons from the methyl group) for the quantitation of the O-acetyl functional group in the polysaccharide.</div>
</li> <li> <para-num num="[0089]"> </para-num> <div class="description-line" id="p-0091" num="0089">The identity of the monovalent polysaccharide was confirmed by countercurrent immunoelectrophoresis using specific antisera.</div>
</li> <li> <para-num num="[0090]"> </para-num> <div class="description-line" id="p-0092" num="0090">High performance gel filtration chromatography coupled with refractive index and multiangle laser light scattering (MALLS) detectors was used in conjunction with the sample concentration to calculate the molecular weight.</div>
</li> <li> <para-num num="[0091]"> </para-num> <div class="description-line" id="p-0093" num="0091">Size exclusion chromatography media (CL-4B) was used to profile the relative molecular size distribution of the polysaccharide.</div>
</li> <heading id="h-0020">Example 2</heading>
<heading id="h-0021">Preparation of Serotype 1 Pneumococcal Saccharide-CRM<sub>197 </sub>Conjugate</heading>
<heading id="h-0022">Activation and Conjugation</heading>
<li> <para-num num="[0092]"> </para-num> <div class="description-line" id="p-0094" num="0092">Containers of purified polysaccharide were thawed and combined in a reaction vessel. To the vessel, 0.2 M sodium carbonate, pH 9.0 was added for partial deacetylation (hydrolysis) for 3 hours at 50Â° C. The reaction was cooled to 20Â° C. and neutralization was performed by 0.2 M acetic acid. Oxidation in the presence of sodium periodate was performed by incubation at 2-8Â° C., and the mixture was stirred for 15-21 hours.</div>
</li> <li> <para-num num="[0093]"> </para-num> <div class="description-line" id="p-0095" num="0093">The activation reaction mixture was concentrated and diafiltered 10Ã with 0.9% NaCl using a 30K MWCO membrane. The retentate was 0.2 Î¼m filtered. The activated saccharide was filled into 100 mL glass lyophilization bottles and shell-frozen at â75Â° C. and lyophilized.</div>
</li> <li> <para-num num="[0094]"> </para-num> <div class="description-line" id="p-0096" num="0094">âShell-freezingâ is a method for preparing samples for lyophilization (freeze-drying). Flasks are automatically rotated by motor driven rollers in a refrigerated bath containing alcohol or any other appropriate fluid. A thin coating of product is evenly frozen around the inside âshellâ of a flask, permitting a greater volume of material to be safely processed during each freeze-drying run. These automatic, refrigerated units provide a simple and efficient means of pre-freezing many flasks at a time, producing the desired coatings inside, and providing sufficient surface area for efficient freeze-drying.</div>
</li> <li> <para-num num="[0095]"> </para-num> <div class="description-line" id="p-0097" num="0095">Bottles of lyophilized material were brought to room temperature and resuspended in CRM<sub>197 </sub>solution at a saccharide/protein ratio of 2:1. To the saccharide/protein mixture 1M sodium phosphate buffer was added to a final 0.2M ionic strength and a pH of 7.5, then sodium cyanoborohydride was added. The reaction was incubated at 23Â° C. for 18 hours, followed by a second incubation at 37Â° C. for 72 hours. Following the cyanoborohydride incubations, the reaction mixture was diluted with cold saline followed by the addition of 1M sodium carbonate to adjust the reaction mixture to pH 9.0. Unreacted aldehydes were quenched by addition of sodium borohydride by incubation at 23Â° C. for 3-6 hours.</div>
</li> <li> <para-num num="[0096]"> </para-num> <div class="description-line" id="p-0098" num="0096">The reaction mixture was diluted 2-fold with saline and transferred through a 0.45-5 Î¼m prefilter into a retentate vessel. The reaction mixture is diafiltered 30Ã with 0.15 M phosphate buffer, pH 6, and 20Ã with saline. The retentate was filtered through a 0.2 Î¼m filter.</div>
</li> <li> <para-num num="[0097]"> </para-num> <div class="description-line" id="p-0099" num="0097">The conjugate solution was diluted to a target of 0.5 mg/mL in 0.9% saline, and then sterile filtered into final bulk concentrate (FBC) containers in a Class 100 hood. The conjugate was stored at 2-8Â° C.</div>
</li> <heading id="h-0023">Characterization</heading>
<li> <para-num num="[0098]"> </para-num> <div class="description-line" id="p-0100" num="0098">Size exclusion chromatography media (CL-4B) was used to profile the relative molecular size distribution of the conjugate.</div>
</li> <li> <para-num num="[0099]"> </para-num> <div class="description-line" id="p-0101" num="0099">The identity of the conjugate was confirmed by the slot-blot assay using specific antisera.</div>
</li> <li> <para-num num="[0100]"> </para-num> <div class="description-line" id="p-0102" num="0100">The saccharide and protein concentrations were determined by the uronic acid and Lowry assays, respectively. The ratio of saccharide to protein in the covalently bonded conjugate complex was obtained by the calculation:</div>
</li> <li> <div class="description-line" id="p-0103" num="0000">
<maths id="MATH-US-00001" num="00001">
<math overflow="scroll">
<mrow>
<mi>Ratio</mi>
<mo>=</mo>
<mfrac>
<mrow>
<mi>Î¼</mi>
<mo>î¢</mo>
<mstyle>
<mspace height="0.3ex" width="0.3em"> </mspace>
</mstyle>
<mo>î¢</mo>
<mi>g</mi>
<mo>î¢</mo>
<mstyle>
<mtext>/</mtext>
</mstyle>
<mo>î¢</mo>
<mi>mL</mi>
<mo>î¢</mo>
<mstyle>
<mspace height="0.8ex" width="0.8em"> </mspace>
</mstyle>
<mo>î¢</mo>
<mi>saccharide</mi>
</mrow>
<mrow>
<mi>Î¼</mi>
<mo>î¢</mo>
<mstyle>
<mspace height="0.3ex" width="0.3em"> </mspace>
</mstyle>
<mo>î¢</mo>
<mi>g</mi>
<mo>î¢</mo>
<mstyle>
<mtext>/</mtext>
</mstyle>
<mo>î¢</mo>
<mi>mL</mi>
<mo>î¢</mo>
<mstyle>
<mspace height="0.8ex" width="0.8em"> </mspace>
</mstyle>
<mo>î¢</mo>
<mi>protein</mi>
</mrow>
</mfrac>
</mrow>
</math>
</maths>
</div>
</li> <li> <para-num num="[0101]"> </para-num> <div class="description-line" id="p-0104" num="0101">O-acetyl content was measured by the Hestrin method (Hestrin et. al., J. Biol. Chem. 1949, 180, p. 249). The ratio of O-acetyl concentration to total saccharide concentration gave Î¼moles of O-acetyl per mg of saccharide.</div>
</li> <heading id="h-0024">Example 3</heading>
<heading id="h-0025">Preparation of <i>S. Pneumoniae </i>Capsular Polysaccharide Serotype 3</heading>
<heading id="h-0026">Preparation of Master and Working Cell Banks</heading>
<li> <para-num num="[0102]"> </para-num> <div class="description-line" id="p-0105" num="0102"> <i>S. pneumoniae </i>serotype 3 was obtained from Dr. Robert Austrian, University of Pennsylvania, Philadelphia, Pa. For preparation of the cell bank system, see Example 1.</div>
</li> <heading id="h-0027">Fermentation and Harvesting</heading>
<li> <para-num num="[0103]"> </para-num> <div class="description-line" id="p-0106" num="0103">Cultures from the working cell bank Were used to inoculate seed bottles containing soy-based medium. The bottles were incubated at 36Â° C.Â±2Â° C. without agitation until growth requirements were met. A seed bottle was used to inoculate a seed fermentor containing soy-based medium. A pH of about 7.0 was maintained with sterile sodium carbonate solution. After the target optical density was reached, the seed fermentor was used to inoculate an intermediate seed fermentor. After the target optical density was reached, the intermediate seed fermentor was used to inoculate the production fermentor. The pH was maintained with sterile sodium carbonate solution. The fermentation was terminated after the working volume of the fermentor was reached. An appropriate amount of sterile 12% sodium deoxycholate was added to the culture to lyse the bacterial cells and release cell-associated polysaccharide. After lysing, the fermentor contents were cooled. The pH of the lysed culture broth was adjusted to approximately pH 6.6 with acetic acid. The lysate was clarified by continuous flow centrifugation followed by depth filtration and 0.45 Î¼m microfiltration.</div>
</li> <heading id="h-0028">Purification</heading>
<li> <para-num num="[0104]"> </para-num> <div class="description-line" id="p-0107" num="0104">The purification of the pneumococcal polysaccharide consisted of several concentration/diafiltration operations, precipitation/elution, column chromatography, and depth filtration steps. All procedures were performed at room temperature unless otherwise specified.</div>
</li> <li> <para-num num="[0105]"> </para-num> <div class="description-line" id="p-0108" num="0105">Clarified broth from the fermentor cultures of <i>S. pneumoniae </i>serotype 3 were concentrated and diafiltered using a 100 kDa MWCO filter. Diafiltration was accomplished using sodium phosphate buffer at neutral pH. Diafiltration removed the low molecular weight medium components from the higher molecular weight biopolymers such as nucleic acid, protein and polysaccharide.</div>
</li> <li> <para-num num="[0106]"> </para-num> <div class="description-line" id="p-0109" num="0106">Prior to the addition of hexadecyltrimethyl ammonium bromide (HB), a calculated volume of a NaCl stock solution was added to the concentrated and diafiltered polysaccharide solution to give a final concentration of 0.25 M NaCl. The polysaccharide was then precipitated by adding HB from a stock solution to give a final concentration of 1% HB (w/v). The polysaccharide/HB precipitate was captured on a depth filter and the filtrate was discarded. The polysaccharide precipitate was resolubilized and eluted by recirculating a sodium chloride solution through the precipitate-containing depth filter. The filters were then rinsed with additional sodium chloride solution.</div>
</li> <li> <para-num num="[0107]"> </para-num> <div class="description-line" id="p-0110" num="0107">Sodium iodide (NaI) was added to the polysaccharide solution from a stock NaI solution to achieve a final concentration of 0.5% to precipitate HB. The precipitate was removed by depth filtration. The filtrate contained the target polysaccharide. The precipitation vessel and the filter were rinsed with a NaCl/NaI solution and the rinse was combined with the partially purified polysaccharide solution. The filter was discarded. The polysaccharide was then filtered through a 0.2 Î¼m filter.</div>
</li> <li> <para-num num="[0108]"> </para-num> <div class="description-line" id="p-0111" num="0108">The polysaccharide solution was concentrated on a 30 kDa MWCO ultrafilter and diafiltered with a sodium chloride solution.</div>
</li> <li> <para-num num="[0109]"> </para-num> <div class="description-line" id="p-0112" num="0109">The partially purified polysaccharide solution was further purified by filtration through a depth filter impregnated with activated carbon. After filtration, the carbon filter was rinsed with a sodium chloride solution. The rinse was combined with the polysaccharide solution, which was then filtered through a 0.2 Î¼m filter.</div>
</li> <li> <para-num num="[0110]"> </para-num> <div class="description-line" id="p-0113" num="0110">The polysaccharide solution was concentrated on a 30 kDa MWCO ultrafilter and adjusted with a 1M sodium phosphate buffer to achieve a final concentration of 0.025M sodium phosphate. The pH was checked and adjusted to 7.0Â±0.2.</div>
</li> <li> <para-num num="[0111]"> </para-num> <div class="description-line" id="p-0114" num="0111">The ceramic hydroxyapatite (HA) column was equilibrated with sodium phosphate buffer containing sodium chloride to obtain the appropriate conductivity (15 Î¼S). The polysaccharide solution was then loaded onto the column. Under these conditions, impurities bound to the resin and the polysaccharide was recovered in the flow-through from the column. The polysaccharide was flushed through the column with buffer and was filtered through a 0.2 Î¼m filter.</div>
</li> <li> <para-num num="[0112]"> </para-num> <div class="description-line" id="p-0115" num="0112">The polysaccharide solution was concentrated using a 30 kDa MWCO filter. The concentrate was then diafiltered with WFI.</div>
</li> <li> <para-num num="[0113]"> </para-num> <div class="description-line" id="p-0116" num="0113">The diafiltered polysaccharide solution was filtered through a 0.2 Î¼m membrane filter into stainless steel containers. Samples were removed for release testing and the purified polysaccharide was stored frozen at â25Â°Â±5Â° C.</div>
</li> <heading id="h-0029">Characterization</heading>
<li> <para-num num="[0114]"> </para-num> <div class="description-line" id="p-0117" num="0114">The 1H-NMR data was consistent with the chemical structure by the assignment of signals assigned to the protons of the polysaccharide molecule.</div>
</li> <li> <para-num num="[0115]"> </para-num> <div class="description-line" id="p-0118" num="0115">The identity of the monovalent polysaccharide was confirmed by countercurrent immunoelectrophoresis using specific antisera.</div>
</li> <li> <para-num num="[0116]"> </para-num> <div class="description-line" id="p-0119" num="0116">High performance gel filtration chromatography, coupled with refractive index and multiangle laser light scattering (MALLS) detectors, was used in conjunction with the sample concentration to calculate the molecular weight.</div>
</li> <li> <para-num num="[0117]"> </para-num> <div class="description-line" id="p-0120" num="0117">Size exclusion chromatography media (CL-4B) was used to profile the relative molecular size distribution of the polysaccharide.</div>
</li> <heading id="h-0030">Example 4</heading>
<heading id="h-0031">Preparation of Serotype 3 Pneumococcal Saccharide-CRM<sub>197 </sub>Conjugate</heading>
<heading id="h-0032">Activation and Conjugation</heading>
<li> <para-num num="[0118]"> </para-num> <div class="description-line" id="p-0121" num="0118">Containers of purified serotype 3 saccharide were thawed and combined in a reaction vessel. To the vessel, WFI and 2M acetic acid were added to a final concentration of 0.2M and 2 mg/mL saccharide. The temperature of the solution was raised to 85Â° C. for one hour to hydrolyze the polysaccharide. The reaction was cooled to â¤25Â° C. and 1M magnesium chloride was added to a final concentration of 0.1M. Oxidation in the presence of sodium periodate was performed by incubation for 16-24 hours at 23Â° C.</div>
</li> <li> <para-num num="[0119]"> </para-num> <div class="description-line" id="p-0122" num="0119">The activation reaction mixture was concentrated and diafiltered 10Ã with WFI using a 100K MWCO membrane. The retentate was filtered through a 0.2-Î¼m filter.</div>
</li> <li> <para-num num="[0120]"> </para-num> <div class="description-line" id="p-0123" num="0120">For compounding, 0.2M sodium phosphate, pH 7.0, was added to the activated saccharide to a final concentration of 10 mM and a pH of 6.0-6.5. CRM<sub>197 </sub>carrier protein was mixed with the saccharide solution to a ratio of 2 g of saccharide per 1 g of CRM<sub>197</sub>. The combined saccharide/protein solution was filled into 100 mL glass lyophilization bottles with a 50 mL target fill, shell-frozen at â75Â° C., and lyophilized.</div>
</li> <li> <para-num num="[0121]"> </para-num> <div class="description-line" id="p-0124" num="0121">Bottles of co-lyophilized saccharide/protein material were brought to room temperature and resuspended in 0.1M sodium phosphate buffer, pH 7.0, to a final saccharide concentration of 20 mg/mL. The pH was adjusted to 6.5 and then a 0.5 molar equivalent of sodium cyanoborohydride was added. The reaction was incubated at 37Â° C. for 48 hours. Following the cyanoborohydride incubation, the reaction mixture was diluted with cold 5 mM succinate/0.9% saline buffer. Unreacted aldehydes were quenched by the addition of sodium borohydride and incubation at 23Â° C. for 3-6 hours. The reaction mixture was transferred through a 0.45-5 Î¼m prefilter into a retentate vessel.</div>
</li> <li> <para-num num="[0122]"> </para-num> <div class="description-line" id="p-0125" num="0122">The reaction mixture was diafiltered 30Ã with 0.1M phosphate buffer (pH 9), 20Ã with 0.15M phosphate butter (pH 6), and 20Ã with 5 mM succinate/0.9% saline. The retentate was filtered through a 0.2-Î¼m filter.</div>
</li> <li> <para-num num="[0123]"> </para-num> <div class="description-line" id="p-0126" num="0123">The conjugate solution was diluted to a saccharide target of 0.5 mg/mL, and then sterile filtered into FBC containers in a Class 100 hood. The conjugate was stored at 2-8Â° C.</div>
</li> <heading id="h-0033">Characterization</heading>
<li> <para-num num="[0124]"> </para-num> <div class="description-line" id="p-0127" num="0124">Size exclusion chromatography media (CL-4B) was used to profile the relative molecular size distribution of the conjugate.</div>
</li> <li> <para-num num="[0125]"> </para-num> <div class="description-line" id="p-0128" num="0125">The identity of the conjugate was confirmed by the slot-blot assay using specific antisera.</div>
</li> <li> <para-num num="[0126]"> </para-num> <div class="description-line" id="p-0129" num="0126">The saccharide and protein concentrations were determined by the Anthrone and Lowry assays, respectively. The ratio of saccharide to protein in the covalently bonded conjugate complex was obtained by the calculation:</div>
</li> <li> <div class="description-line" id="p-0130" num="0000">
<maths id="MATH-US-00002" num="00002">
<math overflow="scroll">
<mrow>
<mi>Ratio</mi>
<mo>=</mo>
<mfrac>
<mrow>
<mi>Î¼</mi>
<mo>î¢</mo>
<mstyle>
<mspace height="0.3ex" width="0.3em"> </mspace>
</mstyle>
<mo>î¢</mo>
<mi>g</mi>
<mo>î¢</mo>
<mstyle>
<mtext>/</mtext>
</mstyle>
<mo>î¢</mo>
<mi>mL</mi>
<mo>î¢</mo>
<mstyle>
<mspace height="0.8ex" width="0.8em"> </mspace>
</mstyle>
<mo>î¢</mo>
<mi>saccharide</mi>
</mrow>
<mrow>
<mi>Î¼</mi>
<mo>î¢</mo>
<mstyle>
<mspace height="0.3ex" width="0.3em"> </mspace>
</mstyle>
<mo>î¢</mo>
<mi>g</mi>
<mo>î¢</mo>
<mstyle>
<mtext>/</mtext>
</mstyle>
<mo>î¢</mo>
<mi>mL</mi>
<mo>î¢</mo>
<mstyle>
<mspace height="0.8ex" width="0.8em"> </mspace>
</mstyle>
<mo>î¢</mo>
<mi>protein</mi>
</mrow>
</mfrac>
</mrow>
</math>
</maths>
</div>
</li> <heading id="h-0034">Example 5</heading>
<heading id="h-0035">Preparation of <i>S. Pneumoniae </i>Capsular Polysaccharide Serotype 5</heading>
<li> <para-num num="[0127]"> </para-num> <div class="description-line" id="p-0131" num="0127"> <i>S. pneumoniae </i>serotype 5 was obtained from Dr. Gerald Schiffman of the State University of New York, Brooklyn, N.Y. For preparation of the cell bank system, see Example 1. For fermentation, harvesting, purification and characterization of the polysaccharide, see Example 1.</div>
</li> <heading id="h-0036">Alternate Fermentation Process</heading>
<li> <para-num num="[0128]"> </para-num> <div class="description-line" id="p-0132" num="0128">Cultures from the working cell bank were used to inoculate seed bottles containing a soy-based medium and a 10 mM sterile NaHCO<sub>3 </sub>solution. The bottles were incubated at 36Â° C.Â±2Â° C. without agitation until growth requirements were met. A seed bottle was used to inoculate a seed fermentor containing soy-based medium and a 10 mM sterile NaHCO<sub>3 </sub>solution. A pH of about 7.0 was maintained with 3N NaOH. After the target optical density was reached, the seed fermentor was used to inoculate the production fermentor containing soy-based medium with a 10 mM NaHCO<sub>3 </sub>concentration. The pH was maintained with 3N NaOH. The fermentation was terminated after cessation of growth or when the working volume of the fermentor was reached. An appropriate amount of sterile 12% sodium deoxycholate was added to the culture to obtain a 0.12% concentration in the broth, to lyse the bacterial cells and release cell-associated polysaccharide. After lysing, the fermentor contents were held, with agitation, for a time interval between 8 and 24 hours at a temperature between 7Â° C. and 13Â° C. to assure that complete cellular lysis and polysaccharide release had occurred. Agitation during this hold period prevented lysate sediment from settling on the fermentor walls and pH probe, thereby allowing the pH probe integrity to be maintained. Next, the pH of the lysed culture broth was adjusted to approximately pH 4.5 with 50% acetic acid. After a hold time without agitation, for a time interval between 12 and 24 hours at a temperature between 15Â° C. and 25Â° C., a significant portion of the previously soluble proteins dropped out of solution as a solid precipitate with little loss or degradation of the polysaccharide, which remained in solution. The solution with the precipitate was then clarified by continuous flow centrifugation followed by depth filtration and 0.45 Î¼m microfiltration.</div>
</li> <heading id="h-0037">Example 6</heading>
<heading id="h-0038">Preparation of Serotype 5 Pneumococcal Saccharide-CRM<sub>197 </sub>Conjugate</heading>
<heading id="h-0039">Activation and Conjugation</heading>
<li> <para-num num="[0129]"> </para-num> <div class="description-line" id="p-0133" num="0129">Containers of serotype 5 saccharide were thawed and combined in a reaction vessel. To the vessel, 0.1M sodium acetate, pH 4.7, was added followed by oxidation in the presence of sodium periodate by incubation for 16-22 hours at 23Â° C.</div>
</li> <li> <para-num num="[0130]"> </para-num> <div class="description-line" id="p-0134" num="0130">The activation reaction mixture was concentrated and diafiltered 10Ã with WFI using a 100K MWCO membrane. The retentate was filtered through a 0.2 Î¼m filter.</div>
</li> <li> <para-num num="[0131]"> </para-num> <div class="description-line" id="p-0135" num="0131">The serotype 5 activated saccharide was combined with CRM<sub>197 </sub>at a ratio of 0.8:1. The combined saccharide/protein solution was filled into 100 mL glass lyophilization bottles (50 mL target fill), shell-frozen at â75Â° C., and co-lyophilized.</div>
</li> <li> <para-num num="[0132]"> </para-num> <div class="description-line" id="p-0136" num="0132">Bottles of co-lyophilized material were brought to room temperature and resuspended in 0.1M sodium phosphate, pH 7.5, and sodium cyanoborohydride was added. The reaction was incubated at 30Â° C. for 72 hours, followed by a second addition of cyanoborohydride and incubated at 30Â° C. for 20-28 hours.</div>
</li> <li> <para-num num="[0133]"> </para-num> <div class="description-line" id="p-0137" num="0133">Following the cyanoborohydride incubations, the reaction mixture was diluted 2-fold with saline and transferred through a 0.45-5 Î¼m prefilter into a retentate vessel. The reaction mixture was diafiltered 30Ã with 0.01M phosphate buffer, pH 8, 20Ã with 0.15M phosphate buffer, pH 6, and 20Ã with saline. The retentate was filtered through a 0.2 Î¼m filter.</div>
</li> <li> <para-num num="[0134]"> </para-num> <div class="description-line" id="p-0138" num="0134">The conjugate solution was diluted to a saccharide target of 0.5 mg/mL, and then sterile filtered into FBC containers in a Class 100 hood. The conjugate was stored at 2-8Â° C.</div>
</li> <li> <para-num num="[0135]"> </para-num> <div class="description-line" id="p-0139" num="0135">For the characterization of the conjugate, see Example 2.</div>
</li> <heading id="h-0040">Example 7</heading>
<heading id="h-0041">Preparation of <i>S. Pneumoniae </i>Capsular Polysaccharide Serotype 6A</heading>
<li> <para-num num="[0136]"> </para-num> <div class="description-line" id="p-0140" num="0136"> <i>S. pneumoniae </i>serotype 6A was obtained from Dr. Gerald Schiffman of the State University of New York, Brooklyn, N.Y. For preparation of the cell bank system, see Example 1. For fermentation, harvesting and purification of the polysaccharide, see Example 1, except that during purification, the 30 kDa MWCO concentration step, prior to the chromatography step, is omitted.</div>
</li> <heading id="h-0042">Example 8</heading>
<heading id="h-0043">Preparation of Serotype 6A Pneumococcal Saccharide-CRM<sub>197 </sub>Conjugate</heading>
<heading id="h-0044">Activation and Conjugation</heading>
<li> <para-num num="[0137]"> </para-num> <div class="description-line" id="p-0141" num="0137">Serotype 6A polysaccharide is a high molecular weight polymer that had to be reduced in size prior to oxidation. Containers of serotype 6A saccharide were thawed and combined in a reaction vessel. To the vessel, 2 M acetic acid was added to a final concentration of 0.1 M for hydrolysis for 1.5 hours at 60Â° C. The reaction was cooled to 23Â° C. and neutralization was performed by adjusting the reaction mixture with 1 M NaOH to pH 6. Oxidation in the presence of sodium periodate was performed by incubation at 23Â° C. for 14-22 hours.</div>
</li> <li> <para-num num="[0138]"> </para-num> <div class="description-line" id="p-0142" num="0138">The activation reaction mixture was concentrated and diafiltered 10Ã with WFI using a 100K MWCO membrane. The retentate was filtered through a 0.2 Î¼m filter.</div>
</li> <li> <para-num num="[0139]"> </para-num> <div class="description-line" id="p-0143" num="0139">Serotype 6A was compounded with sucrose and filled into 100 mL glass lyophilization bottles (50 mL target fill) and shell-frozen at â75Â° C. and lyophilized.</div>
</li> <li> <para-num num="[0140]"> </para-num> <div class="description-line" id="p-0144" num="0140">Bottles of lyophilized material were brought to room temperature and resuspended in dimethylsulfoxide (DMSO) at a saccharide/protein ratio of 1:1. After addition of sodium cyanoborohydride, the reaction mixture was incubated at 23Â° C. for 18 hours. Following the cyanoborohydride incubation, the reaction mixture was diluted with cold saline. Unreacted aldehydes were quenched by addition of sodium borohydride by incubation at 23Â° C. for 3-20 hours.</div>
</li> <li> <para-num num="[0141]"> </para-num> <div class="description-line" id="p-0145" num="0141">The diluted reaction mixture was transferred through a 5 Î¼m prefilter into a retentate vessel. The reaction mixture was diafiltered 10Ã with 0.9% NaCl and 30Ã with succinate-buffered NaCl. The retentate was filtered through a 0.2 Î¼m filter.</div>
</li> <li> <para-num num="[0142]"> </para-num> <div class="description-line" id="p-0146" num="0142">The conjugate solution was diluted to a saccharide target of 0.5 mg/mL, and then sterile filtered into FBC containers in a Class 100 hood. The conjugate was stored at 2-8Â° C.</div>
</li> <li> <para-num num="[0143]"> </para-num> <div class="description-line" id="p-0147" num="0143">For the characterization of the conjugate, see Example 2.</div>
</li> <heading id="h-0045">Example 9</heading>
<heading id="h-0046">Preparation of <i>S. Pneumoniae </i>Capsular Polysaccharide Serotype 7F</heading>
<li> <para-num num="[0144]"> </para-num> <div class="description-line" id="p-0148" num="0144"> <i>S. pneumoniae </i>serotype 7F was obtained from Dr. Gerald Schiffman of the State University of New York, Brooklyn, N.Y. For preparation of the cell bank system, and for fermentation and harvesting of the polysaccharide, see Example 3. For an alternate fermentation and harvesting process, see the alternate process described in Example 1.</div>
</li> <heading id="h-0047">Purification</heading>
<li> <para-num num="[0145]"> </para-num> <div class="description-line" id="p-0149" num="0145">The purification of the pneumococcal polysaccharide consisted of several concentration/diafiltration operations, precipitation/elution, column chromatography, and depth filtration steps. All procedures were performed at room temperature unless otherwise specified.</div>
</li> <li> <para-num num="[0146]"> </para-num> <div class="description-line" id="p-0150" num="0146">Clarified broth from fermentor cultures of <i>S. pneumoniae </i>serotype 7F were concentrated and diafiltered using a 100 kDa MWCO filter. Diafiltration was accomplished using sodium phosphate buffer at neutral pH. Diafiltration removed the low molecular weight medium components from the higher molecular weight biopolymers such as nucleic acid, protein and polysaccharide.</div>
</li> <li> <para-num num="[0147]"> </para-num> <div class="description-line" id="p-0151" num="0147">Serotype 7F does not form a precipitate with HB. Instead, impurities were precipitated from the concentrated and diafiltered solution by adding the HB from a stock solution to a final concentration of 1% HB. The precipitate was captured on a depth filter and the filter was discarded. The polysaccharide was contained in the filtrate.</div>
</li> <li> <para-num num="[0148]"> </para-num> <div class="description-line" id="p-0152" num="0148">Sodium iodide (NaI) was added to the polysaccharide solution from a stock NaI solution to achieve a final concentration of 0.5% to precipitate HB. The precipitate was removed by depth filtration. The filtrate contained the target polysaccharide. The precipitation vessel and the filter were rinsed with a NaCl/NaI solution and the rinses were combined with the partially purified polysaccharide solution. The filter was discarded. The polysaccharide was then filtered through a 0.2 Î¼m filter.</div>
</li> <li> <para-num num="[0149]"> </para-num> <div class="description-line" id="p-0153" num="0149">The polysaccharide solution was concentrated on a 30 kDa MWCO ultrafilter and diafiltered with a sodium chloride solution.</div>
</li> <li> <para-num num="[0150]"> </para-num> <div class="description-line" id="p-0154" num="0150">The partially purified polysaccharide solution was further purified by filtration through a depth filter impregnated with activated carbon. After filtration, the carbon filter was rinsed with a sodium chloride solution. The rinse was combined with the polysaccharide solution, which was then filtered through a 0.2 Î¼m filter.</div>
</li> <li> <para-num num="[0151]"> </para-num> <div class="description-line" id="p-0155" num="0151">The polysaccharide solution was concentrated on a 30 kDa MWCO ultrafilter and adjusted with a 1M sodium phosphate buffer to achieve a final concentration of 0.025M sodium phosphate. The pH was checked and adjusted to 7.0Â±0.2.</div>
</li> <li> <para-num num="[0152]"> </para-num> <div class="description-line" id="p-0156" num="0152">The ceramic hydroxyapatite (HA) column was equilibrated with sodium phosphate buffer containing sodium chloride to obtain the appropriate conductivity (15 Î¼S). The polysaccharide solution was then loaded onto the column. Under these conditions, impurities bound to the resin and the polysaccharide was recovered in the flow-through from the column. The polysaccharide was flushed through the column with buffer and was filtered through a 0.2 Î¼m filter.</div>
</li> <li> <para-num num="[0153]"> </para-num> <div class="description-line" id="p-0157" num="0153">The polysaccharide solution was concentrated using a 30 kDa MWCO filter. The concentrate was then diafiltered with WFI.</div>
</li> <li> <para-num num="[0154]"> </para-num> <div class="description-line" id="p-0158" num="0154">The diafiltered polysaccharide solution was filtered through a 0.2 Î¼m membrane filter into stainless steel containers. Samples were removed for release testing and the purified polysaccharide was stored at 2Â°-80C.</div>
</li> <li> <para-num num="[0155]"> </para-num> <div class="description-line" id="p-0159" num="0155">For characterization of the polysaccharide, see Example 3.</div>
</li> <heading id="h-0048">Example 10</heading>
<heading id="h-0049">Preparation of Serotype 7F Pneumococcal Saccharide-CRM<sub>97 </sub>Conjugate</heading>
<heading id="h-0050">Activation and Conjugation</heading>
<li> <para-num num="[0156]"> </para-num> <div class="description-line" id="p-0160" num="0156">Oxidation in the presence of sodium periodate was performed by incubation for 16-24 hrs at 23Â° C.</div>
</li> <li> <para-num num="[0157]"> </para-num> <div class="description-line" id="p-0161" num="0157">The activation reaction mixture was concentrated and diafiltered 10Ã with 10 mM NaOAc, pH 4.5, using a 100K MWCO membrane. The retentate was filtered through a 0.2 Î¼m filter.</div>
</li> <li> <para-num num="[0158]"> </para-num> <div class="description-line" id="p-0162" num="0158">Serotype 7F was filled into 100 mL glass lyophilization bottles (50 mL target fill) and shell-frozen at â75Â° C. and lyophilized.</div>
</li> <li> <para-num num="[0159]"> </para-num> <div class="description-line" id="p-0163" num="0159">Bottles of lyophilized serotype 7F and CRM<sub>197 </sub>were brought to room temperature and resuspended in DMSO at a saccharide/protein ratio of 1.5:1. After the addition of sodium cyanoborohydride, the reaction was incubated at 23Â° C. for 8-10 hours. Unreacted aldehydes were quenched by the addition of sodium borohydride by incubation at 23Â° C. for 16 hours.</div>
</li> <li> <para-num num="[0160]"> </para-num> <div class="description-line" id="p-0164" num="0160">The reaction mixture was diluted 10-fold with cold saline and transferred through a 5 Î¼m prefilter into a retentate vessel. The reaction mixture was diafiltered 10Ã with 0.9% saline and 30Ã with succinate-buffered saline. The retentate was filtered through a 0.2 Î¼m filter.</div>
</li> <li> <para-num num="[0161]"> </para-num> <div class="description-line" id="p-0165" num="0161">The conjugate solution was diluted to a saccharide target of 0.5 mg/mL 0.9% saline, and then sterile filtered into FBC containers in a Class 100 hood. The conjugate was stored at 2-8Â° C.</div>
</li> <li> <para-num num="[0162]"> </para-num> <div class="description-line" id="p-0166" num="0162">For characterization of the conjugate, see Example 4.</div>
</li> <heading id="h-0051">Example 11</heading>
<heading id="h-0052">Preparation of <i>S. Pneumoniae </i>Capsular Polysaccharide Serotype 19A</heading>
<li> <para-num num="[0163]"> </para-num> <div class="description-line" id="p-0167" num="0163"> <i>S. pneumoniae </i>serotype 19A was obtained from Dr. Gerald Schiffman of the State University of New York, Brooklyn, N.Y. For preparation of the cell bank system, see Example 1. For fermentation, harvesting and purification of the polysaccharide, see Example 7. For characterization, see Example 3.</div>
</li> <heading id="h-0053">Example 12</heading>
<heading id="h-0054">Preparation of Serotype 19A Pneumococcal Saccharide-CRM<sub>197 </sub>Conjugate</heading>
<heading id="h-0055">Activation and Conjugation</heading>
<li> <para-num num="[0164]"> </para-num> <div class="description-line" id="p-0168" num="0164">Containers of serotype 19A saccharide were thawed and combined in a reaction vessel. Sodium acetate was added to 10 mM (pH 5.0) and oxidation was carried out in the presence of sodium periodate by incubation for 16-24 hrs at 23Â° C.</div>
</li> <li> <para-num num="[0165]"> </para-num> <div class="description-line" id="p-0169" num="0165">The activation reaction mixture was concentrated and diafiltered 10Ã with 10 mM acetate, pH 5.0, using a 100K MWCO membrane. The retentate was filtered through a 0.2 Î¼m filter.</div>
</li> <li> <para-num num="[0166]"> </para-num> <div class="description-line" id="p-0170" num="0166">The activated saccharide was compounded with sucrose followed by the addition of CRM<sub>197</sub>. The serotype 19A activated saccharide and CRM<sub>197 </sub>mixture (0.8:1 ratio) was filled into 100 mL glass lyophilization bottles (50 mL target fill) and shell-frozen at â75Â° C. and lyophilized.</div>
</li> <li> <para-num num="[0167]"> </para-num> <div class="description-line" id="p-0171" num="0167">Bottles of lyophilized material were brought to room temperature and resuspended in DMSO. To the saccharide/protein mixture, sodium cyanoborohydride (100 mg/ml) was added. The reaction was incubated at 23Â° C. for 15 hours. Following the cyanoborohydride incubation, unreacted aldehydes were quenched by the addition of sodium borohydride by incubation at 23Â° C. for 3-20 hours.</div>
</li> <li> <para-num num="[0168]"> </para-num> <div class="description-line" id="p-0172" num="0168">The reaction mixture was diluted 10-fold with cold saline and transferred through a 5 Î¼m prefilter into a retentate vessel. The reaction mixture was diafiltered 10Ã with 0.9% NaCl, 0.45-Î¼m filtered, and 30Ã with diafiltration using 5 mM succinate/0.9% NaCl buffer, pH 6. The retentate was filtered through a 0.2 Î¼m filter.</div>
</li> <li> <para-num num="[0169]"> </para-num> <div class="description-line" id="p-0173" num="0169">The conjugate solution was diluted to a target of 0.5 mg/mL using 5 mM succinate/0.9% saline, and then sterile filtered into FBC containers in a Class 100 hood. The conjugate was stored at 2-8Â° C.</div>
</li> <li> <para-num num="[0170]"> </para-num> <div class="description-line" id="p-0174" num="0170">For characterization of the conjugate, see Example 4.</div>
</li> <heading id="h-0056">Example 13</heading>
<heading id="h-0057">Preparation of <i>S. Pneumoniae </i>Capsular Polysaccharide Serotypes 4, 6B, 9V, 14, 18C, 19F and 23F</heading>
<heading id="h-0058">Preparation of the <i>S. pneumoniae </i>Seed Culture</heading>
<li> <para-num num="[0171]"> </para-num> <div class="description-line" id="p-0175" num="0171"> <i>S. pneumoniae </i>serotypes 4, 6B, 9V, 18C, 19F and 23F were obtained from Dr. Gerald Schiffman, State University of New York, Brooklyn, N.Y. <i>S. pneumoniae </i>serotype 14 was obtained from the ATCC, strain 6314.</div>
</li> <li> <para-num num="[0172]"> </para-num> <div class="description-line" id="p-0176" num="0172">Separately, one vial of each of the desired serotypes of <i>Streptococcus pneumoniae </i>was used to start a fermentation batch. Two bottles containing a soy-based medium and phenol red were adjusted to a pH range of 7.4Â±0.2 using sodium carbonate, and the required volume of 50% dextrose/1% magnesium sulfate solution was then added to the bottles. The two bottles were inoculated with different amounts of seed. The bottles were incubated at 360Â±2Â° C. until the medium turned yellow. Following incubation, samples were removed from each bottle and tested for optical density (OD) (0.3 to 0.9) and pH (4.6 to 5.5). One of the two bottles was selected for inoculation of the seed fermentor.</div>
</li> <li> <para-num num="[0173]"> </para-num> <div class="description-line" id="p-0177" num="0173">Soy-based medium was transferred to the seed fermentor and sterilized. Then a volume of 50% dextrose/1% magnesium sulfate solution was added to the fermentor. The pH and agitation of the seed fermentor were monitored and controlled (pH 6.7 to 7.4). The temperature was maintained at 360Â±2Â° C. The seed inoculum (bottle) was aseptically connected to the seed fermentor and the inoculum was transferred. The fermentor was maintained in pH control and samples were periodically removed and tested for OD and pH. When the desired OD of 0.5 at 600 nm was reached, the intermediate fermentor was inoculated with the fermentation broth from the seed fermentor.</div>
</li> <li> <para-num num="[0174]"> </para-num> <div class="description-line" id="p-0178" num="0174">Soy-based medium was transferred to the intermediate fermentor and sterilized. Then a volume of 50% dextrose/1% magnesium sulfate solution was added to the fermentor. The pH and agitation of the intermediate fermentor were monitored and controlled (pH 6.7 to 7.4). The temperature was maintained at 360Â±2Â° C. The contents of the seed fermentor were transferred to the intermediate fermentor. The fermentor was maintained in pH control and samples were periodically removed and tested for OD and pH. When the desired OD of 0.5 at 600 nm was reached, the production fermentor was inoculated with the fermentation broth from the intermediate fermentor.</div>
</li> <li> <para-num num="[0175]"> </para-num> <div class="description-line" id="p-0179" num="0175">Soy-based medium was transferred to the production fermentor and sterilized. Then a volume of 50% dextrose/1% magnesium sulfate solution was added to the fermentor. The pH and agitation of the production fermentor were monitored and controlled (pH 6.7 to 7.4). The temperature was maintained at 360Â±2Â° C. The fermentor was maintained in pH control and samples were periodically removed and tested for OD and pH, until the fermentation was complete.</div>
</li> <li> <para-num num="[0176]"> </para-num> <div class="description-line" id="p-0180" num="0176">Deoxycholate sodium was added to the fermentor to a final concentration of approximately 0.12% w/v. The culture was mixed for a minimum of thirty minutes and the temperature set point was reduced to 10Â° C. The culture was incubated overnight and following confirmation of inactivation, the pH of the culture was adjusted to between 6.4 and 6.8, as necessary, with 50% acetic acid. The temperature of the fermentor was increased to 20Â°Â±5Â° C. and the contents were transferred to the clarification hold tank.</div>
</li> <li> <para-num num="[0177]"> </para-num> <div class="description-line" id="p-0181" num="0177">The contents of the clarification hold tank (including the cellular debris) were processed through a centrifuge at a flow rate between 25 and 600 liters per hour (except Serotype 4, wherein the cell debris was discarded and the flow rate tightened to between 25 and 250 liters per hour). Samples of the supernatant were removed and tested for OD. The desired OD during the centrifugation was â¤0.15.</div>
</li> <li> <para-num num="[0178]"> </para-num> <div class="description-line" id="p-0182" num="0178">Initially, the supernatant was recirculated through a depth filter assembly until an OD of 0.05Â±0.03 was achieved. Then the supernatant was passed through the depth filter assembly and through a 0.45 Î¼m membrane filter to the filtrate hold tank.</div>
</li> <li> <para-num num="[0179]"> </para-num> <div class="description-line" id="p-0183" num="0179">Subsequently, the product was transferred through closed pipes to the purification area for processing.</div>
</li> <li> <para-num num="[0180]"> </para-num> <div class="description-line" id="p-0184" num="0180">All of the above operations (centrifugation, filtration and transfer) were performed between 10Â° C. to 30Â° C.</div>
</li> <li> <para-num num="[0181]"> </para-num> <div class="description-line" id="p-0185" num="0181">For an alternate fermentation and harvesting process for serotypes 4 and 6B, see the alternate process described in Example 1.</div>
</li> <heading id="h-0059">Purification</heading>
<li> <para-num num="[0182]"> </para-num> <div class="description-line" id="p-0186" num="0182">The purification of each pneumococcal polysaccharide consisted of several concentration/diafiltration operations, precipitation/elution, column chromatography, and depth filtration steps. All procedures were performed at room temperature unless otherwise specified.</div>
</li> <li> <para-num num="[0183]"> </para-num> <div class="description-line" id="p-0187" num="0183">Clarified broth from the fermentor cultures of the desired <i>S. pneumoniae </i>serotype was concentrated and diafiltered using a 100 kDa MWCO filter. Diafiltration was accomplished using sodium phosphate buffer at pH&lt;9.0. Diafiltration removed the low molecular weight medium components from the higher molecular weight biopolymers such as nucleic acid, protein and polysaccharide.</div>
</li> <li> <para-num num="[0184]"> </para-num> <div class="description-line" id="p-0188" num="0184">The polysaccharide was precipitated from the concentrated and diafiltered solution by adding HB from a stock solution to give a final concentration of 1% HB (w/v) (except Serotype 23F, which had a final concentration of 2.5%). The polysaccharide/HB precipitate was captured on a depth filter and the filtrate was discarded. (Note: Serotype 14 does not precipitate; therefore the filtrate was retained.) The polysaccharide precipitate was resolubilized and eluted by recirculating a sodium chloride solution through the precipitate-containing depth filter. The filters were then rinsed with additional sodium chloride solution.</div>
</li> <li> <para-num num="[0185]"> </para-num> <div class="description-line" id="p-0189" num="0185">Sodium iodide (NaI) was added to the polysaccharide solution from a stock NaI solution to achieve a final concentration of 0.5% to precipitate HB (except for Serotype 6B, which had a final concentration of 0.25%). The precipitate was removed by depth filtration. The filtrate contained the target polysaccharide. The filter was discarded. The polysaccharide was then filtered through a 0.2 Î¼m filter.</div>
</li> <li> <para-num num="[0186]"> </para-num> <div class="description-line" id="p-0190" num="0186">The polysaccharide solution was concentrated on a 30 kDa MWCO ultrafilter and diafiltered with a sodium chloride solution.</div>
</li> <li> <para-num num="[0187]"> </para-num> <div class="description-line" id="p-0191" num="0187">The partially purified polysaccharide solution was further purified by filtration through a depth filter impregnated with activated carbon. After filtration, the carbon filter was rinsed with a sodium chloride solution. The rinse was combined with the polysaccharide solution, which was then filtered through a 0.2 Î¼m filter.</div>
</li> <li> <para-num num="[0188]"> </para-num> <div class="description-line" id="p-0192" num="0188">The polysaccharide solution was concentrated on a 30 kDa MWCO ultrafilter and the filter was rinsed with a sodium chloride solution. The pH was checked and adjusted to 7.0Â±0.3.</div>
</li> <li> <para-num num="[0189]"> </para-num> <div class="description-line" id="p-0193" num="0189">The ceramic hydroxyapatite (HA) column was equilibrated with sodium phosphate buffer containing sodium chloride until the pH is 7.0Â±0.3 and the conductivity was 26Â±4 Î¼S. The polysaccharide solution was then loaded onto the column. Under these conditions, impurities bound to the resin and the polysaccharide was recovered in the flow through from the column. The polysaccharide solution was filtered through a 0.2 Î¼m filter.</div>
</li> <li> <para-num num="[0190]"> </para-num> <div class="description-line" id="p-0194" num="0190">The polysaccharide solution was concentrated using a 30 kDa MWCO filter. The concentrate was then diafiltered with WFI until the conductivity was &lt;15 Î¼S.</div>
</li> <li> <para-num num="[0191]"> </para-num> <div class="description-line" id="p-0195" num="0191">The diafiltered polysaccharide solution was filtered through a 0.2 Î¼m membrane filter into bulk containers and stored at 2-8Â° C.</div>
</li> <heading id="h-0060">Example 14</heading>
<heading id="h-0061">Preparation of Pneumococcal Saccharide-CRM<sub>197 </sub>Conjugates for Serotypes 4, 6B, 9V, 14, 18C, 19F and 23F</heading>
<heading id="h-0062">Activation Process</heading>
<li> <para-num num="[0192]"> </para-num> <div class="description-line" id="p-0196" num="0192">The different serotype saccharides follow different pathways for activation (hydrolysis or no hydrolysis prior to activation) and conjugation (aqueous or DMSO reactions) as described in this example.</div>
</li> <li> <para-num num="[0193]"> </para-num> <div class="description-line" id="p-0197" num="0193">Polysaccharide was transferred from the bulk containers to the reactor vessel. The polysaccharide was then diluted in WFI and sodium phosphate to a final concentration range of 1.6-2.4 mg/mL.</div>
</li> <heading id="h-0063">Step 1.</heading>
<li> <para-num num="[0194]"> </para-num> <div class="description-line" id="p-0198" num="0194">For serotypes 6B, 9V, 14, 19F and 23F, pH was adjusted to pH 6.0Â±0.3.</div>
</li> <li> <para-num num="[0195]"> </para-num> <div class="description-line" id="p-0199" num="0195">For serotype 4, hydrochloric acid (0.01 M final acid concentration) was added and the solution was incubated for 25-35 minutes at 450Â±2Â° C. Hydrolysis was stopped by cooling to 21-25Â° C. and adding 1M sodium phosphate to a target of pH 6.7Â±0.2. An in-process test was done to confirm an appropriate level of depyruvylation.</div>
</li> <li> <para-num num="[0196]"> </para-num> <div class="description-line" id="p-0200" num="0196">For serotype 18C, glacial acetic acid (0.2 M final acid concentration) was added and the solution was incubated for 205-215 minutes at 94Â°Â±2Â° C. Temperature was then decreased to 21-25Â° C. and 1-2 M sodium phosphate was added to a target of 30 pH 6.8Â±0.2.</div>
</li> <heading id="h-0064">Step 2: Periodate Reaction</heading>
<li> <para-num num="[0197]"> </para-num> <div class="description-line" id="p-0201" num="0197">The required sodium periodate molar equivalents for pneumococcal saccharide activation was determined using total saccharide content (except for serotype 4). For serotype 4, a ratio of 0.8-1.2 moles of sodium periodate per mole of saccharide was used. With thorough mixing, the oxidation reaction was allowed to proceed between 16 to 20 hours at 21-25Â° C. for all serotypes except 19F for which the temperature was â¤15Â° C.</div>
</li> <heading id="h-0065">Step 3: Ultrafiltration</heading>
<li> <para-num num="[0198]"> </para-num> <div class="description-line" id="p-0202" num="0198">The oxidized saccharide was concentrated and diafiltered with WFI (0.01 M sodium phosphate buffer pH 6.0 for serotype 19F) on a 100 kDa MWCO ultrafilter (5 kDa ultrafilter for 18C). The permeate was discarded and the retentate was filtered through a 0.22 Î¼m filter.</div>
</li> <heading id="h-0066">Step 4: Lyophilization</heading>
<li> <para-num num="[0199]"> </para-num> <div class="description-line" id="p-0203" num="0199">For serotypes 4, 9V, and 14 the concentrated saccharide was mixed with CRM<sub>197 </sub>carrier protein, filled into glass bottles, shell-frozen and stored at â¤â65Â° C.</div>
</li> <li> <para-num num="[0200]"> </para-num> <div class="description-line" id="p-0204" num="0200">The frozen concentrated saccharide-CRM<sub>197 </sub>was lyophilized and then stored at â25Â°Â±5Â° C.</div>
</li> <li> <para-num num="[0201]"> </para-num> <div class="description-line" id="p-0205" num="0201">For serotypes 6B, 19F, and 23F a specified amount of sucrose was added which was calculated to achieve a 5%Â±3% sucrose concentration in the conjugation reaction mixture. Serotype 18C did not require sucrose addition. The concentrated saccharide was then filled into glass bottles, shell-frozen and stored at â¤â65Â° C. The frozen concentrated saccharide was lyophilized and then stored at â25Â°+5Â° C.</div>
</li> <heading id="h-0067">Conjugation Process</heading>
<li> <para-num num="[0202]"> </para-num> <div class="description-line" id="p-0206" num="0202">Two conjugation processes were used: aqueous conjugation for serotypes 4, 9V, 14 and 18C, and DMSO conjugation for serotypes 6B, 19F and 23F.</div>
</li> <heading id="h-0068">Aqueous Conjugation</heading>
<heading id="h-0069">Step 1: Dissolution</heading>
<li> <para-num num="[0203]"> </para-num> <div class="description-line" id="p-0207" num="0203">For serotypes 4, 9V and 14, the lyophilized activated saccharide-CRM<sub>197 </sub>mixture was thawed and equilibrated at room temperature. The lyophilized activated saccharide-CRM<sub>197 </sub>was then reconstituted in 0.1M sodium phosphate buffer at a typical ratio of:</div>
</li> <li> <para-num num="[0204]"> </para-num> <div class="description-line" id="p-0208" num="0204">1L of buffer per 16-24 g of saccharide for serotype 4 and 9V</div>
</li> <li> <para-num num="[0205]"> </para-num> <div class="description-line" id="p-0209" num="0205">1L of buffer per 6-10 g of saccharide for serotype 14</div>
</li> <li> <para-num num="[0206]"> </para-num> <div class="description-line" id="p-0210" num="0206">The reaction mixture was incubated at 37Â°Â±2Â° C. until total dissolution for the serotype 9V and at 23Â°Â±2Â° C. for serotypes 4 and 14.</div>
</li> <li> <para-num num="[0207]"> </para-num> <div class="description-line" id="p-0211" num="0207">For serotype 18C, the lyophilized saccharide was reconstituted in a solution of CRM<sub>197 </sub>in 1M dibasic sodium phosphate at a typical ratio of 0.11 L of sodium phosphate per 1 L of CRM<sub>197 </sub>solution. The reaction mixture (8-12 g/L saccharide concentration) was incubated at 23Â°Â±2Â° C. until total dissolution.</div>
</li> <li> <para-num num="[0208]"> </para-num> <div class="description-line" id="p-0212" num="0208">The pH was tested as an in-process control at this stage.</div>
</li> <heading id="h-0070">Step 2: Conjugation Reaction</heading>
<li> <para-num num="[0209]"> </para-num> <div class="description-line" id="p-0213" num="0209">For serotypes 4 and 9V, the conjugation reaction was initiated by adding the sodium cyanoborohydride solution (100 mg/mL) to achieve 1.0-1.4 moles sodium cyanoborohydride per mole of saccharide. The reaction mixture was incubated for 44-52 hours at 370Â±2Â° C. The temperature was then reduced to 23Â°Â±2Â° C. and sodium chloride 0.9% was added to the reactor. Sodium borohydride solution (100 mg/mL) was added to achieve. 1.8-2.2 molar equivalents of sodium borohydride per mole saccharide. The mixture was incubated for 3-6 hours at 23Â°Â±2Â° C. The mixture was diluted with sodium chloride 0.9% and the reactor was rinsed. The diluted conjugation mixture was filtered using a 1.2 Î¼m pre-filter into a holding vessel.</div>
</li> <li> <para-num num="[0210]"> </para-num> <div class="description-line" id="p-0214" num="0210">For serotypes 14 and 18C, the conjugation reaction was initiated by adding the cyanoborohydride solution (100 mg/mL) to achieve 1.0-1.4 moles of sodium cyanoborohydride per mole of saccharide. The reaction mixture was incubated for 12-24 hours at 23Â°+2Â° C. The temperature was increased to 37Â°Â±2Â° C. and the reaction was incubated for 72-96 hours. The temperature was then reduced to 23Â°Â±2Â° C. and 0.9% sodium chloride was added to the reactor. Sodium borohydride solution (100 mg/mL) was added to achieve 1.8-2.2 molar equivalents of sodium borohydride per mole of saccharide. The mixture was incubated for 3-6 hours at 23Â°Â±2Â° C. The mixture was diluted with 0.9% sodium chloride and the reactor was rinsed. The diluted conjugation mixture was then filtered using a 1.2 m pre-filter into a holding vessel.</div>
</li> <heading id="h-0071">Step 3: Ultrafiltration 100 kDa</heading>
<li> <para-num num="[0211]"> </para-num> <div class="description-line" id="p-0215" num="0211">The diluted conjugation mixture was concentrated and diafiltrated on a 100 kDa MWCO ultrafilter with either a minimum of 15 volumes (serotype 4) or 40 volumes (serotypes 9V, 14, and 18C) of 0.9% sodium chloride.</div>
</li> <li> <para-num num="[0212]"> </para-num> <div class="description-line" id="p-0216" num="0212">The permeate was discarded.</div>
</li> <li> <para-num num="[0213]"> </para-num> <div class="description-line" id="p-0217" num="0213">For serotype 4, the retentate was filtered through a 0.45 Î¼m filter.</div>
</li> <li> <para-num num="[0214]"> </para-num> <div class="description-line" id="p-0218" num="0214">An in-process control (saccharide content) was performed at this step.</div>
</li> <heading id="h-0072">Step 4: HA Column Purification</heading>
<li> <para-num num="[0215]"> </para-num> <div class="description-line" id="p-0219" num="0215">This step was only performed for the serotype 4 conjugate.</div>
</li> <li> <para-num num="[0216]"> </para-num> <div class="description-line" id="p-0220" num="0216">The HA column was first neutralized using 0.5M sodium phosphate buffer (pH 7.0Â±0.3) and then equilibrated with 0.9% sodium chloride. The filtered retentate (serotype 4) was loaded onto the column at a flow rate of 1.0 L/min. The column was washed with 0.9% sodium chloride at a flow rate of â¤2.0 L/min. The product was then eluted with 0.5M sodium phosphate buffer at a flow rate of â¤2.0 L/min.</div>
</li> <li> <para-num num="[0217]"> </para-num> <div class="description-line" id="p-0221" num="0217">The HA fraction was then concentrated and diafiltered on a 100 kDa MWCO membrane with a minimum of 20 volumes of 0.9% sodium chloride. The permeate was discarded.</div>
</li> <heading id="h-0073">Step 5: Sterile Filtration</heading>
<li> <para-num num="[0218]"> </para-num> <div class="description-line" id="p-0222" num="0218">The retentate after the 100 kDa MWCO diafiltration was filtered through a 0.22 Î¼m filter. In-process controls (saccharide content, free protein, free saccharide and cyanide) were performed on the filtered product. In-process controls on filtered retentate were performed to determine whether additional concentration, diafiltration, and/or dilution were needed to meet FBC targets. These and additional tests were repeated in FBC samples.</div>
</li> <li> <para-num num="[0219]"> </para-num> <div class="description-line" id="p-0223" num="0219">As necessary, the filtered conjugate was diluted with 0.9% sodium chloride in order to achieve a final concentration of less than 0.55 g/L. Release tests for saccharide content, protein content and saccharide:protein ratio were performed at this stage.</div>
</li> <li> <para-num num="[0220]"> </para-num> <div class="description-line" id="p-0224" num="0220">Finally, the conjugate was filtered (0.22 Î¼m) and filled into 10 L stainless steel canisters at a typical quantity of 2.64 g/canister. At this stage, yield, saccharide content, protein content, pH, saccharide:protein ratio and lysine content were performed as in-process controls. Release testing (appearance, free protein, free saccharide, endotoxin, molecular size determination, residual cyanide, saccharide identity, CRM<sub>197 </sub>identity) was performed at this stage.</div>
</li> <heading id="h-0074">DMSO Conjugation</heading>
<heading id="h-0075">Step I: Dissolution</heading>
<li> <para-num num="[0221]"> </para-num> <div class="description-line" id="p-0225" num="0221">The lyophilized activated saccharide serotypes 6B, 19F, 23F and the lyophilized CRM<sub>197 </sub>carrier protein were equilibrated at room temperature and reconstituted in DMSO. The dissolution concentration typically ranged from 2-3 grams of saccharide (2-2.5 g protein) per liter of DMSO.</div>
</li> <heading id="h-0076">Step II: Conjugation Reaction</heading>
<li> <para-num num="[0222]"> </para-num> <div class="description-line" id="p-0226" num="0222">The activated saccharide and CRM<sub>197 </sub>carrier protein were mixed for 60-75 minutes at 23Â°Â±2Â° C. at a ratio range of 0.6 g-1.0 g saccharide/g CRM<sub>197 </sub>for serotypes 6B and 19F or 1.2 to 1.8 g saccharide/g CRM<sub>197 </sub>for serotype 23F.</div>
</li> <li> <para-num num="[0223]"> </para-num> <div class="description-line" id="p-0227" num="0223">The conjugation reaction was initiated by adding the sodium cyanoborohydride solution (100 mg/mL) at a ratio of 0.8-1.2 molar equivalents of sodium cyanoborohydride to one mole activated saccharide. WFI was added to the reaction mixture to a target of 1% (v/v) and the mixture was incubated for over 40 hours at 23Â°Â±2Â° C.</div>
</li> <li> <para-num num="[0224]"> </para-num> <div class="description-line" id="p-0228" num="0224">Sodium borohydride solution, 100 mg/mL (typical 1.8-2.2 molar equivalents sodium borohydride per mole activated saccharide) and WFI (target 5% v/v) were added to the reaction and the mixture was incubated for 3-6 hours at 23Â°Â±2Â° C. This procedure reduced any unreacted aldehydes present on the saccharides. Then the reaction mixture was transferred to a dilution tank containing 0.9% sodium chloride at &lt;15Â° C.</div>
</li> <heading id="h-0077">Step III: 100 kDa Ultrafiltration</heading>
<li> <para-num num="[0225]"> </para-num> <div class="description-line" id="p-0229" num="0225">The diluted conjugate mixture was filtered through a 1.2 Î¼m filter and concentrated and diafiltered on a 100 kDa MWCO membrane with a minimum of 15 volumes of 0.9% sodium chloride (0.01M sodium phosphate/0.05M NaCl buffer was used for serotype 23F). The permeate was discarded. The retentate was filtered through a 0.45 Î¼m filter. An in-process saccharide content sample was taken at this stage.</div>
</li> <heading id="h-0078">Step IV: DEAE Column Purification</heading>
<li> <para-num num="[0226]"> </para-num> <div class="description-line" id="p-0230" num="0226">This step was only performed for serotype 23F.</div>
</li> <li> <para-num num="[0227]"> </para-num> <div class="description-line" id="p-0231" num="0227">The DEAE column was equilibrated with 0.01M sodium phosphate/0.05M sodium chloride buffer. The filtered retentate (serotype 23F) was loaded onto the column and washed with 0.01M sodium phosphate/0.05M sodium chloride buffer. The column was then washed with 0.01M sodium phosphate/0.9% NaCl buffer. The product was then eluted with 0.01M sodium phosphate/0.5M sodium chloride buffer.</div>
</li> <heading id="h-0079">Step V: 100 kDa Ultrafiltration</heading>
<li> <para-num num="[0228]"> </para-num> <div class="description-line" id="p-0232" num="0228">The retentate from 6B and 19F was concentrated and diafiltered with at least 30 volumes of 0.9% sodium chloride. The permeate was discarded.</div>
</li> <li> <para-num num="[0229]"> </para-num> <div class="description-line" id="p-0233" num="0229">The eluate from serotype 23F was concentrated and diafiltered with a minimum of 20 volumes of 0.9% sodium chloride. The permeate was discarded.</div>
</li> <heading id="h-0080">Step VI: Sterile Filtration</heading>
<li> <para-num num="[0230]"> </para-num> <div class="description-line" id="p-0234" num="0230">The retentate after the 100 kDa MWCO diafiltration was filtered through 0.22 Î¼m filter. In-process controls (saccharide content, free protein, free saccharide, residual DMSO and residual cyanide) were performed on the filtered product. In-process controls on filtered retentate were performed to determine whether additional concentration, diafiltration, and/or dilution were needed to meet FBC targets. These and additional tests were repeated in FBC samples.</div>
</li> <li> <para-num num="[0231]"> </para-num> <div class="description-line" id="p-0235" num="0231">As necessary, the filtered conjugate was diluted with 0.9% sodium chloride to achieve a final concentration of less than 0.55 g/L. Release tests for saccharide content, protein content and saccharide:protein ratio were performed at this stage.</div>
</li> <li> <para-num num="[0232]"> </para-num> <div class="description-line" id="p-0236" num="0232">Finally, the conjugate was filtered (0.22 Î¼m) and filled into 10 L stainless steel canisters at a quantity of 2.64 g/canister. At this stage, yield, saccharide content, protein content, pH, saccharide:protein ratio and lysine content were performed as in-process controls. Release testing (appearance, free protein, free saccharide, endotoxin, molecular size determination, residual cyanide, residual DMSO, saccharide identity and CRM<sub>197 </sub>identity) was performed at this stage.</div>
</li> <heading id="h-0081">Example 15</heading>
<heading id="h-0082">Formulation of a Multivalent Pneumococcal Conjugate Vaccine</heading>
<li> <para-num num="[0233]"> </para-num> <div class="description-line" id="p-0237" num="0233">The final bulk concentrates of the 13 conjugates contain 0.85% sodium chloride. Type 3, 6A, 7F and 19A bulk concentrates also contain 5 mM sodium succinate buffer at pH 5.8. The required volumes of bulk concentrates were calculated based on the batch volume and the bulk saccharide concentrations. After 80% of the 0.85% sodium chloride (physiological saline) and the required amount of succinate buffer were added to the pre-labeled formulation vessel, bulk concentrates were added. The preparation was then sterile filtered through a 0.22 Î¼m membrane into a second container by using a Millipore Durapore membrane filter unit. The first container was washed with the remaining 20% of 0.85% sodium chloride and the solution was passed through the same filter and collected into the second container. The formulated bulk was mixed gently during and following the addition of bulk aluminum phosphate. The pH was checked and adjusted if necessary. The formulated bulk product was stored at 2-8Â° C.</div>
</li> <li> <para-num num="[0234]"> </para-num> <div class="description-line" id="p-0238" num="0234">The formulated bulk product was filled into Type 1 borosilicate glass syringes obtained from Becton Dickinson. The vaccine was monitored at regular intervals for turbidity to ensure the uniformity of the filling operation. The filled vaccine (Final Product) was stored at 2-8Â° C.</div>
</li> <heading id="h-0083">Example 16</heading>
<heading id="h-0084">Immunogenicity of the 13-Valent Conjugate Vaccine</heading>
<li> <para-num num="[0235]"> </para-num> <div class="description-line" id="p-0239" num="0235">To date, the preclinical studies performed on the 13vPnC vaccine have been in rabbits. Studies #HT01-0021 and #HT01-0036 were designed to independently examine the effect of chemical conjugation of capsular polysaccharides (PSs) from <i>S. pneumoniae </i>to CRM<sub>197 </sub>and the effect of aluminum phosphate (AlPO<sub>4</sub>) adjuvant on the immune response to the 13vPnC vaccine in rabbits. These effects were characterized by antigen-specific ELISA for serum IgG concentrations and for antibody function by opsonophagocytic assay (OPA).</div>
</li> <heading id="h-0085">Study #HT01-0021</heading>
<li> <para-num num="[0236]"> </para-num> <div class="description-line" id="p-0240" num="0236">Study #HT01-0021 examined the ability of the 13vPnC vaccine with AlPO<sub>4 </sub>adjuvant to elicit vaccine serotype-specific immune responses. The pneumococcal serotypes represented in the 13vPnC vaccine include types 1, 3, 4, 5, 6A, 6B, 7F, 9V, 14, 18C, 19A, 19F and 23F. Secondary objectives included an evaluation of the kinetics and duration of the antibody response. New Zealand White rabbits were immunized intramuscularly at week 0 and week 2 with the planned human clinical dose of each polysaccharide (2 Î¼g of each PS, except 4 Î¼g of 6B) formulated with or without AlPO<sub>4 </sub>(100 Î¼g/dose). Sera were collected at various time points. Serotype specific IgG was measured by ELISA and functional activity was assessed by OPA.</div>
</li> <li> <para-num num="[0237]"> </para-num> <div class="description-line" id="p-0241" num="0237">Table 3 shows the geometric mean titer (GMT) achieved in pooled serum samples, following two doses of the 13vPnC vaccine. A ratio of the IgG GMTs was used to compare responses from week 4 to week 0. These data demonstrate that the inclusion of AlPO<sub>4 </sub>in the 13vPnC formulation elicited higher levels of IgG antibody in comparison to the same vaccine without adjuvant. Although the antibody responses were greater when AlPO<sub>4 </sub>was included in the formulation, these increases were not statistically significant.</div>
</li> <li> <para-num num="[0238]"> </para-num> <div class="description-line" id="p-0242" num="0238">Functional antibody responses were also assessed in rabbits following immunization with the two 13vPnC formulations (Table 4). When comparing vaccine formulations with or without adjuvant, higher OPA GMTs were observed in the 13vPnC+AlPO<sub>4 </sub>vaccine treatment group. OPA titers were detected in week 4 serum pools to all vaccine serotypes in both groups. For the majority of the serotypes, OPA titers measured at week 4 were at least 4-fold higher than those at week 0 (baseline).</div>
</li> <li> <para-num num="[0239]"> </para-num> <div class="description-line" id="p-0243" num="0239">The kinetic responses to each of the 13vPnC vaccine serotypes were evaluated from serum pools of both treatment groups. IgG titers to each serotype were measured from blood draws at week 0 and weeks 1, 2, 3, 4, 8, 12, 26, and 39 and then compared. With the exception of serotype 1, antibody responses in animals receiving adjuvanted vaccine were superior to those that received non-adjuvanted vaccine and peaked at week 2 of the immunization schedule (data not shown).</div>
</li> <li> <para-num num="[0240]"> </para-num> <div class="description-line" id="p-0244" num="0240">Overall, the data indicate that the 13vPnC vaccine formulated with aluminum phosphate is immunogenic in rabbits, eliciting substantial antibody responses to the pneumococcal capsular polysaccharides contained in the vaccine and these responses are associated with functional activity. The responses observed to the seven core serotypes following immunization with 13vPnC+AlPO<sub>4 </sub>are consistent with historical responses of rabbits to the heptavalent formulation.</div>
</li> <li> <div class="description-line" id="p-0245" num="0000">
<tables id="TABLE-US-00003" num="00003">
<patent-tables colsep="0" frame="none" pgwide="1" rowsep="0">
<table align="left" class="description-table" cols="1" colsep="0" rowsep="0" width="100%">
<thead>
<tr class="description-tr">
<td class="description-td" colspan="1" nameend="1" namest="1" rowsep="1">TABLE 3</td>
</tr>
</thead>
<tbody><tr class="description-tr">
<td align="center" class="description-td" colspan="1" nameend="1" namest="1" rowsep="1"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">Rabbit IgG Immune Responses (GMTs) Following Immunization with Two Doses of 13-valent Pneumococcal Glycoconjugate</td>
</tr>
</tbody></table>
<table align="left" class="description-table" cols="4" colsep="0" rowsep="0" width="100%">
<tbody><tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">Diluent with ALPO<sub>4</sub> <sup>a</sup> </td>
<td class="description-td">13vPnC<sup>a</sup> </td>
<td class="description-td">13vPnC + ALPO<sub>4</sub> <sup>a</sup> </td>
</tr>
</tbody></table>
<table align="left" class="description-table" cols="10" colsep="0" rowsep="0" width="100%">
<tbody><tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">Ratio</td>
<td class="description-td"> </td>
<td class="description-td">Week 4</td>
<td class="description-td">Ratio</td>
<td class="description-td"> </td>
<td class="description-td">Week 4</td>
<td class="description-td">Ratio</td>
</tr>
<tr class="description-tr">
<td class="description-td">Serotype</td>
<td class="description-td">Week 0</td>
<td class="description-td">Week 4</td>
<td class="description-td">Wk 4:Wk 0</td>
<td class="description-td">Week 0</td>
<td class="description-td">(95% CI)</td>
<td class="description-td">Wk 4:Wk 0</td>
<td class="description-td">Week 0</td>
<td class="description-td">(95% CI)</td>
<td class="description-td">Wk 4:Wk 0</td>
</tr>
<tr class="description-tr">
<td align="center" class="description-td" colspan="10" nameend="10" namest="1" rowsep="1"> </td>
</tr>
</tbody></table>
<table align="left" class="description-table" cols="10" colsep="0" rowsep="0" width="100%">
<tbody><tr class="description-tr">
<td class="description-td">â1</td>
<td class="description-td">&lt;100</td>
<td class="description-td">&lt;100</td>
<td class="description-td">1.0</td>
<td class="description-td">50</td>
<td class="description-td">â5,926</td>
<td class="description-td">119</td>
<td class="description-td">50</td>
<td class="description-td">11,091</td>
<td class="description-td">222</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">(2,758-12,733)</td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">(5,327-23,093)</td>
</tr>
<tr class="description-tr">
<td class="description-td">â3</td>
<td class="description-td">&lt;100</td>
<td class="description-td">&lt;100</td>
<td class="description-td">1.0</td>
<td class="description-td">50</td>
<td class="description-td">â6,647</td>
<td class="description-td">133</td>
<td class="description-td">58</td>
<td class="description-td">16,443</td>
<td class="description-td">284</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">(2,773-15,932)</td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">(7,096-38,106)</td>
</tr>
<tr class="description-tr">
<td class="description-td">â4</td>
<td class="description-td">&lt;100</td>
<td class="description-td">&lt;100</td>
<td class="description-td">1.0</td>
<td class="description-td">50</td>
<td class="description-td">13,554</td>
<td class="description-td">271</td>
<td class="description-td">50</td>
<td class="description-td">29,183</td>
<td class="description-td">584</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">(8,031-22,875)</td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">(15,342-55,508)â</td>
</tr>
<tr class="description-tr">
<td class="description-td">â5</td>
<td class="description-td">134</td>
<td class="description-td">&lt;100</td>
<td class="description-td">0.4</td>
<td class="description-td">50</td>
<td class="description-td">â5,859</td>
<td class="description-td">117</td>
<td class="description-td">50</td>
<td class="description-td">16,714</td>
<td class="description-td">334</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">(2,450-14,009)</td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">(6,959-40,140)</td>
</tr>
<tr class="description-tr">
<td class="description-td">â6A</td>
<td class="description-td">141</td>
<td class="description-td">&lt;100</td>
<td class="description-td">0.4</td>
<td class="description-td">74</td>
<td class="description-td">22,415</td>
<td class="description-td">303</td>
<td class="description-td">83</td>
<td class="description-td">63,734</td>
<td class="description-td">768</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">(11,987-41,914)â</td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">(21,141-192,146)</td>
</tr>
<tr class="description-tr">
<td class="description-td">â6B</td>
<td class="description-td">&lt;100</td>
<td class="description-td">&lt;100</td>
<td class="description-td">1.0</td>
<td class="description-td">57</td>
<td class="description-td">â8,108</td>
<td class="description-td">142</td>
<td class="description-td">54</td>
<td class="description-td">23,505</td>
<td class="description-td">435</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">(3,564-18,444)</td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">(11,286-48,955)â</td>
</tr>
<tr class="description-tr">
<td class="description-td">â7F</td>
<td class="description-td">3,859</td>
<td class="description-td">579</td>
<td class="description-td">0.2</td>
<td class="description-td">171</td>
<td class="description-td">43,591</td>
<td class="description-td">444</td>
<td class="description-td">143</td>
<td class="description-td">84,888</td>
<td class="description-td">496</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">(26,931-70,557)â</td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">(46,445-155,151)</td>
</tr>
<tr class="description-tr">
<td class="description-td">â9V</td>
<td class="description-td">289</td>
<td class="description-td">995</td>
<td class="description-td">3.4</td>
<td class="description-td">205</td>
<td class="description-td">15,780</td>
<td class="description-td">125</td>
<td class="description-td">208</td>
<td class="description-td"> <sup>â</sup>43,331<sup>b</sup> </td>
<td class="description-td">217</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">(7,193-34,616)</td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">(23,256-71,510)â</td>
</tr>
<tr class="description-tr">
<td class="description-td">14</td>
<td class="description-td">437</td>
<td class="description-td">177</td>
<td class="description-td">0.4</td>
<td class="description-td">61</td>
<td class="description-td">â6,906</td>
<td class="description-td">113</td>
<td class="description-td">70</td>
<td class="description-td">16,076</td>
<td class="description-td">322</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">(3,416-13,962)</td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">(9,649-26,785)</td>
</tr>
<tr class="description-tr">
<td class="description-td">18C</td>
<td class="description-td">&lt;100</td>
<td class="description-td">&lt;100</td>
<td class="description-td">1.0</td>
<td class="description-td">50</td>
<td class="description-td">21,283</td>
<td class="description-td">426</td>
<td class="description-td">50</td>
<td class="description-td">35,040</td>
<td class="description-td">701</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">(15,770-28,725)â</td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">(24,708-49,692)â</td>
</tr>
<tr class="description-tr">
<td class="description-td">19A</td>
<td class="description-td">&lt;100</td>
<td class="description-td">&lt;100</td>
<td class="description-td">1.0</td>
<td class="description-td">121</td>
<td class="description-td">113,599â</td>
<td class="description-td">939</td>
<td class="description-td">144</td>
<td class="description-td">280,976â</td>
<td class="description-td">1,951</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">(54,518-236,707)</td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">(119,587-660,167)â</td>
</tr>
<tr class="description-tr">
<td class="description-td">19F</td>
<td class="description-td">&lt;100</td>
<td class="description-td">&lt;100</td>
<td class="description-td">1.0</td>
<td class="description-td">50</td>
<td class="description-td">14,365</td>
<td class="description-td">287</td>
<td class="description-td">50</td>
<td class="description-td">24,912</td>
<td class="description-td">498</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">(7,346-28,090)</td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">(9,243-67,141)</td>
</tr>
<tr class="description-tr">
<td class="description-td">23F</td>
<td class="description-td">&lt;100</td>
<td class="description-td">&lt;100</td>
<td class="description-td">1.0</td>
<td class="description-td">50</td>
<td class="description-td">â5,323</td>
<td class="description-td">106</td>
<td class="description-td">50</td>
<td class="description-td">15,041</td>
<td class="description-td">301</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">(1,894-14,962)</td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">(4,711-48,018)</td>
</tr>
<tr class="description-tr">
<td align="center" class="description-td" colspan="10" nameend="10" namest="1" rowsep="1"> </td>
</tr>
<tr class="description-tr">
<td align="left" class="description-td" colspan="10" id="FOO-00006" nameend="10" namest="1"> <sup>a</sup>GMTs of pooled sera consisted of equal volumes of serum from each individual rabbit within a group</td>
</tr>
<tr class="description-tr">
<td align="left" class="description-td" colspan="10" id="FOO-00007" nameend="10" namest="1"> <sup>b</sup>Statistically different (p = 0.022) from treatment group without ALPO<sub>4</sub> </td>
</tr>
</tbody></table>
</patent-tables>
</tables>
</div>
</li> <li> <div class="description-line" id="p-0246" num="0000">
<tables id="TABLE-US-00004" num="00004">
<patent-tables colsep="0" frame="none" rowsep="0">
<table align="left" class="description-table" cols="1" colsep="0" rowsep="0" width="100%">
<thead>
<tr class="description-tr">
<td class="description-td" colspan="1" nameend="1" namest="1" rowsep="1">TABLE 4</td>
</tr>
</thead>
<tbody><tr class="description-tr">
<td align="center" class="description-td" colspan="1" nameend="1" namest="1" rowsep="1"> </td>
</tr>
<tr class="description-tr">
<td class="description-td"> <i>S. pneumoniae </i>OPA GMTs for NZW Rabbit Serum</td>
</tr>
<tr class="description-tr">
<td class="description-td">Pools Following Immunization with Two Doses</td>
</tr>
<tr class="description-tr">
<td class="description-td">of 13-valent Pneumococcal Glycoconjugate</td>
</tr>
</tbody></table>
<table align="left" class="description-table" cols="3" colsep="0" rowsep="0" width="100%">
<tbody><tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">13vPnC<sup>a</sup> </td>
<td class="description-td">13vPnC + ALPO<sub>4</sub> <sup>a</sup> </td>
</tr>
</tbody></table>
<table align="left" class="description-table" cols="7" colsep="0" rowsep="0" width="100%">
<tbody><tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">Week</td>
<td class="description-td">Week</td>
<td class="description-td">Ratio</td>
<td class="description-td">Week</td>
<td class="description-td">Week</td>
<td class="description-td">Ratio</td>
</tr>
<tr class="description-tr">
<td class="description-td">Serotype</td>
<td class="description-td">0</td>
<td class="description-td">4</td>
<td class="description-td">Wk 4:Wk 0</td>
<td class="description-td">0</td>
<td class="description-td">4</td>
<td class="description-td">Wk 4:Wk 0</td>
</tr>
<tr class="description-tr">
<td align="center" class="description-td" colspan="7" nameend="7" namest="1" rowsep="1"> </td>
</tr>
</tbody></table>
<table align="left" class="description-table" cols="7" colsep="0" rowsep="0" width="100%">
<tbody><tr class="description-tr">
<td class="description-td">â1</td>
<td class="description-td">&lt;8</td>
<td class="description-td">64</td>
<td class="description-td">16</td>
<td class="description-td">&lt;8</td>
<td class="description-td">64</td>
<td class="description-td">16</td>
</tr>
<tr class="description-tr">
<td class="description-td">â3</td>
<td class="description-td">&lt;8</td>
<td class="description-td">8</td>
<td class="description-td">2</td>
<td class="description-td">&lt;8</td>
<td class="description-td">16</td>
<td class="description-td">4</td>
</tr>
<tr class="description-tr">
<td class="description-td">â4</td>
<td class="description-td">&lt;8</td>
<td class="description-td">16</td>
<td class="description-td">4</td>
<td class="description-td">&lt;8</td>
<td class="description-td">32</td>
<td class="description-td">8</td>
</tr>
<tr class="description-tr">
<td class="description-td">â5</td>
<td class="description-td">&lt;8</td>
<td class="description-td">128</td>
<td class="description-td">32</td>
<td class="description-td">&lt;8</td>
<td class="description-td">512</td>
<td class="description-td">128</td>
</tr>
<tr class="description-tr">
<td class="description-td">â6A</td>
<td class="description-td">8</td>
<td class="description-td">128</td>
<td class="description-td">16</td>
<td class="description-td">8</td>
<td class="description-td">512</td>
<td class="description-td">64</td>
</tr>
<tr class="description-tr">
<td class="description-td">â6B</td>
<td class="description-td">&lt;8</td>
<td class="description-td">256</td>
<td class="description-td">64</td>
<td class="description-td">8</td>
<td class="description-td">1,024</td>
<td class="description-td">128</td>
</tr>
<tr class="description-tr">
<td class="description-td">â7F</td>
<td class="description-td">8</td>
<td class="description-td">64</td>
<td class="description-td">8</td>
<td class="description-td">8</td>
<td class="description-td">128</td>
<td class="description-td">16</td>
</tr>
<tr class="description-tr">
<td class="description-td">â9V</td>
<td class="description-td">8</td>
<td class="description-td">64</td>
<td class="description-td">8</td>
<td class="description-td">8</td>
<td class="description-td">128</td>
<td class="description-td">16</td>
</tr>
<tr class="description-tr">
<td class="description-td">14</td>
<td class="description-td">16</td>
<td class="description-td">32</td>
<td class="description-td">2</td>
<td class="description-td">16</td>
<td class="description-td">32</td>
<td class="description-td">2</td>
</tr>
<tr class="description-tr">
<td class="description-td">18C</td>
<td class="description-td">8</td>
<td class="description-td">256</td>
<td class="description-td">32</td>
<td class="description-td">&lt;8</td>
<td class="description-td">256</td>
<td class="description-td">64</td>
</tr>
<tr class="description-tr">
<td class="description-td">19A</td>
<td class="description-td">&lt;8</td>
<td class="description-td">256</td>
<td class="description-td">64</td>
<td class="description-td">&lt;8</td>
<td class="description-td">1,024</td>
<td class="description-td">256</td>
</tr>
<tr class="description-tr">
<td class="description-td">19F</td>
<td class="description-td">&lt;8</td>
<td class="description-td">128</td>
<td class="description-td">32</td>
<td class="description-td">&lt;8</td>
<td class="description-td">512</td>
<td class="description-td">128</td>
</tr>
<tr class="description-tr">
<td class="description-td">23F</td>
<td class="description-td">8</td>
<td class="description-td">64</td>
<td class="description-td">8</td>
<td class="description-td">&lt;8</td>
<td class="description-td">256</td>
<td class="description-td">64</td>
</tr>
<tr class="description-tr">
<td align="center" class="description-td" colspan="7" nameend="7" namest="1" rowsep="1"> </td>
</tr>
<tr class="description-tr">
<td align="left" class="description-td" colspan="7" id="FOO-00008" nameend="7" namest="1"> <sup>a</sup>Pools consisted of equal volumes of serum from individual rabbits within a treatment group (n = 12)</td>
</tr>
</tbody></table>
</patent-tables>
</tables>
</div>
</li> <heading id="h-0086">Study #HT01-0036</heading>
<li> <para-num num="[0241]"> </para-num> <div class="description-line" id="p-0247" num="0241">Study #HT01-0036 compared rabbit immune responses to the polysaccharides (PSs) contained in the vaccine, after immunization with the 13vPnC vaccine with or without conjugation to the CRM<sub>197 </sub>protein. New Zealand White rabbits were immunized intramuscularly at week 0 and week 2 with a dose of 2.2 Î¼g of each PS (except 4.4 Î¼g of 6B). Animals received one of three vaccine preparations: (a) 13vPnC (PS directly conjugated to CRM<sub>197</sub>), (b) 13vPnPS, (free PS) or (c) 13vPnPS+CRM<sub>197 </sub>(free PS mixed with CRM<sub>197</sub>). All vaccine preparations contained AlPO<sub>4 </sub>as the adjuvant at 125 Î¼g/dose.</div>
</li> <li> <para-num num="[0242]"> </para-num> <div class="description-line" id="p-0248" num="0242">Serotype specific immune responses for all vaccine preparations were evaluated in an IgG ELISA and complement-mediated OPA measuring functional antibody. The immune responses were compared between the treatment groups.</div>
</li> <li> <para-num num="[0243]"> </para-num> <div class="description-line" id="p-0249" num="0243">Table 5 presents GMT data obtained from week 4 bleeds analyzed in antigen specific IgG ELISAs. Additional analyses show the ratio of GMT values at week 4 to week 0. The data indicate that the conjugate vaccine preparation elicited greater serum IgG titers than free PS or free PS+CRM<sub>197 </sub>vaccine. With the exception of <i>S. pneumoniae </i>type 14, the 13vPnC vaccine was able to induce functional antibodies to the representative strains of <i>S. pneumoniae </i>in an OPA (Table 6). After two immunizations with either the 13vPnPS or 13vPnPS+CRM<sub>197 </sub>vaccine, neither could induce OPA titers â¥8-fold at week 4 relative to week 0 for 10 out of the 13 serotypes measured (Table 6).</div>
</li> <li> <para-num num="[0244]"> </para-num> <div class="description-line" id="p-0250" num="0244">In conclusion, these results indicate that conjugation of the 13-valent pneumococcal vaccine polysaccharides produces higher serum IgG titers and overall greater functional antibody activity than seen with free polysaccharide alone or mixed with unconjugated CRM<sub>197</sub>.</div>
</li> <li> <div class="description-line" id="p-0251" num="0000">
<tables id="TABLE-US-00005" num="00005">
<patent-tables colsep="0" frame="none" pgwide="1" rowsep="0">
<table align="left" class="description-table" cols="1" colsep="0" rowsep="0" width="100%">
<thead>
<tr class="description-tr">
<td class="description-td" colspan="1" nameend="1" namest="1" rowsep="1">TABLE 5</td>
</tr>
</thead>
<tbody><tr class="description-tr">
<td align="center" class="description-td" colspan="1" nameend="1" namest="1" rowsep="1"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">Rabbit IgG Responses (GMTs) to PnPS by ELISA Following Immunization</td>
</tr>
<tr class="description-tr">
<td class="description-td">with Two Doses of 13-valent Pneumococcal Glycoconjugate</td>
</tr>
</tbody></table>
<table align="left" class="description-table" cols="4" colsep="0" rowsep="0" width="100%">
<tbody><tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">13vPnPS = CRM<sub>197</sub> </td>
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">13vPnPS (free PS)</td>
<td class="description-td">(PS mixed with CRM<sub>197</sub>)</td>
<td class="description-td">13vPnC</td>
</tr>
</tbody></table>
<table align="left" class="description-table" cols="10" colsep="0" rowsep="0" width="100%">
<tbody><tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">Week 4</td>
<td class="description-td">Ratio</td>
<td class="description-td"> </td>
<td class="description-td">Week 4</td>
<td class="description-td">Ratio</td>
<td class="description-td"> </td>
<td class="description-td">Week 4</td>
<td class="description-td">Ratio</td>
</tr>
<tr class="description-tr">
<td class="description-td">Serotype</td>
<td class="description-td">Week 0</td>
<td class="description-td">(95% CI)</td>
<td class="description-td">Wk 4:Wk 0</td>
<td class="description-td">Week 0</td>
<td class="description-td">(95% CI)</td>
<td class="description-td">Wk 4:Wk 0</td>
<td class="description-td">Week 0</td>
<td class="description-td">(95% CI)</td>
<td class="description-td">Wk 4:Wk 0</td>
</tr>
<tr class="description-tr">
<td align="center" class="description-td" colspan="10" nameend="10" namest="1" rowsep="1"> </td>
</tr>
</tbody></table>
<table align="left" class="description-table" cols="10" colsep="0" rowsep="0" width="100%">
<tbody><tr class="description-tr">
<td class="description-td">â1</td>
<td class="description-td">378</td>
<td class="description-td">2,290ââ</td>
<td class="description-td">5.8</td>
<td class="description-td">395</td>
<td class="description-td">1,959ââ</td>
<td class="description-td">5.0</td>
<td class="description-td">472</td>
<td class="description-td">35,970</td>
<td class="description-td">76.2</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">ââ(843-5,790)</td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">â(809-4,739)</td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">(29,130-44,417)</td>
</tr>
<tr class="description-tr">
<td class="description-td">â3</td>
<td class="description-td">57</td>
<td class="description-td">240</td>
<td class="description-td">4.2</td>
<td class="description-td">89</td>
<td class="description-td">163</td>
<td class="description-td">1.8</td>
<td class="description-td">50</td>
<td class="description-td">10,414</td>
<td class="description-td">208.3</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">â(64-908)</td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">(74-358)</td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">(10,414-16,676)</td>
</tr>
<tr class="description-tr">
<td class="description-td">â4</td>
<td class="description-td">50</td>
<td class="description-td">379</td>
<td class="description-td">7.6</td>
<td class="description-td">50</td>
<td class="description-td">607</td>
<td class="description-td">12.1</td>
<td class="description-td">50</td>
<td class="description-td">12,890</td>
<td class="description-td">257.8</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">(150-959)</td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">â(313-1,178)</td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">â(9,117-18,224)</td>
</tr>
<tr class="description-tr">
<td class="description-td">â5</td>
<td class="description-td">343</td>
<td class="description-td">226</td>
<td class="description-td">4.5</td>
<td class="description-td">50</td>
<td class="description-td">321</td>
<td class="description-td">6.4</td>
<td class="description-td">50</td>
<td class="description-td">35,264</td>
<td class="description-td">705.3</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">(113-450)</td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">(147-701)â</td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">(24,467-50,824)</td>
</tr>
<tr class="description-tr">
<td class="description-td">â6A</td>
<td class="description-td">154</td>
<td class="description-td">466</td>
<td class="description-td">3.0</td>
<td class="description-td">98</td>
<td class="description-td">210</td>
<td class="description-td">2.1</td>
<td class="description-td">163</td>
<td class="description-td">234,245â</td>
<td class="description-td">1,437.1</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">(316-688)</td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">(95-464)</td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">(167,152-328,283)</td>
</tr>
<tr class="description-tr">
<td class="description-td">â6B</td>
<td class="description-td">63</td>
<td class="description-td">727</td>
<td class="description-td">11.6</td>
<td class="description-td">62</td>
<td class="description-td">745</td>
<td class="description-td">12.0</td>
<td class="description-td">131</td>
<td class="description-td">33,599</td>
<td class="description-td">256.5</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">ââ(384-1,375)</td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">â(384-1,440)</td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">(22,934-49,222)</td>
</tr>
<tr class="description-tr">
<td class="description-td">â7F</td>
<td class="description-td">50</td>
<td class="description-td">â61</td>
<td class="description-td">1.2</td>
<td class="description-td">50</td>
<td class="description-td">â72</td>
<td class="description-td">1.4</td>
<td class="description-td">50</td>
<td class="description-td">35,702</td>
<td class="description-td">714.0</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">(39-95)</td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">(47-111)</td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">(24,350-52,347)</td>
</tr>
<tr class="description-tr">
<td class="description-td">â9V</td>
<td class="description-td">50</td>
<td class="description-td">104</td>
<td class="description-td">2.1</td>
<td class="description-td">55</td>
<td class="description-td">169</td>
<td class="description-td">3.0</td>
<td class="description-td">50</td>
<td class="description-td">50,033</td>
<td class="description-td">1,000.7</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">â(48-195)</td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">(74-390)</td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">(34,765-72,007)</td>
</tr>
<tr class="description-tr">
<td class="description-td">14</td>
<td class="description-td">66</td>
<td class="description-td">298</td>
<td class="description-td">4.5</td>
<td class="description-td">50</td>
<td class="description-td">195</td>
<td class="description-td">3.9</td>
<td class="description-td">50</td>
<td class="description-td">20,121</td>
<td class="description-td">402.4</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">(117-757)</td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">(71-535)</td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">(12,087-32,138)</td>
</tr>
<tr class="description-tr">
<td class="description-td">18C</td>
<td class="description-td">89</td>
<td class="description-td">1,555ââ</td>
<td class="description-td">17.5</td>
<td class="description-td">66</td>
<td class="description-td">761</td>
<td class="description-td">11.5</td>
<td class="description-td">101</td>
<td class="description-td">71,451</td>
<td class="description-td">707.4</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">ââ(655-3,688)</td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">â(300-1,935)</td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">â(32,745-124,641)</td>
</tr>
<tr class="description-tr">
<td class="description-td">19A</td>
<td class="description-td">50</td>
<td class="description-td">â89</td>
<td class="description-td">1.8</td>
<td class="description-td">50</td>
<td class="description-td">â80</td>
<td class="description-td">1.6</td>
<td class="description-td">50</td>
<td class="description-td">23,485</td>
<td class="description-td">469.7</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">â(44-179)</td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">(39-163)</td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">(12,857-42,723)</td>
</tr>
<tr class="description-tr">
<td class="description-td">19F</td>
<td class="description-td">61</td>
<td class="description-td">1,362ââ</td>
<td class="description-td">22.3</td>
<td class="description-td">61</td>
<td class="description-td">991</td>
<td class="description-td">16.3</td>
<td class="description-td">67</td>
<td class="description-td">19,358</td>
<td class="description-td">288.9</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">ââ(559-3,317)</td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">â(370-2,654)</td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">(12,553-33,173)</td>
</tr>
<tr class="description-tr">
<td class="description-td">23F</td>
<td class="description-td">73</td>
<td class="description-td">1,085ââ</td>
<td class="description-td">14.9</td>
<td class="description-td">121</td>
<td class="description-td">638</td>
<td class="description-td">5.3</td>
<td class="description-td">68</td>
<td class="description-td">45,972</td>
<td class="description-td">676.1</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">ââ(487-2,420)</td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">â(311-1,311)</td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">(25,134-84,089)</td>
</tr>
<tr class="description-tr">
<td align="center" class="description-td" colspan="10" nameend="10" namest="1" rowsep="1"> </td>
</tr>
</tbody></table>
</patent-tables>
</tables>
</div>
</li> <li> <div class="description-line" id="p-0252" num="0000">
<tables id="TABLE-US-00006" num="00006">
<patent-tables colsep="0" frame="none" pgwide="1" rowsep="0">
<table align="left" class="description-table" cols="1" colsep="0" rowsep="0" width="100%">
<thead>
<tr class="description-tr">
<td class="description-td" colspan="1" nameend="1" namest="1" rowsep="1">TABLE 6</td>
</tr>
</thead>
<tbody><tr class="description-tr">
<td align="center" class="description-td" colspan="1" nameend="1" namest="1" rowsep="1"> </td>
</tr>
<tr class="description-tr">
<td class="description-td"> <i>S. pneumoniae </i>OPA Titers for Rabbit Serum Pools Following Immunization</td>
</tr>
<tr class="description-tr">
<td class="description-td">with Two Doses of 13-valent Pneumococcal Vaccines</td>
</tr>
</tbody></table>
<table align="left" class="description-table" cols="2" colsep="0" rowsep="0" width="100%">
<tbody><tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">OPA Titers</td>
</tr>
</tbody></table>
<table align="left" class="description-table" cols="3" colsep="0" rowsep="0" width="100%">
<tbody><tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">13vPnPS + CRM<sub>197</sub> </td>
<td class="description-td"> </td>
</tr>
</tbody></table>
<table align="left" class="description-table" cols="5" colsep="0" rowsep="0" width="100%">
<tbody><tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">No</td>
<td class="description-td">13vPnPS (free PS)</td>
<td class="description-td">(free PS mixed with CRM<sub>197</sub>)</td>
<td class="description-td">13vPnC</td>
</tr>
</tbody></table>
<table align="left" class="description-table" cols="8" colsep="0" rowsep="0" width="100%">
<tbody><tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">Treatment</td>
<td class="description-td"> </td>
<td class="description-td">Ratio</td>
<td class="description-td"> </td>
<td class="description-td">Ratio</td>
<td class="description-td"> </td>
<td class="description-td">Ratio</td>
</tr>
<tr class="description-tr">
<td class="description-td">Serotype</td>
<td class="description-td">Week 0<sup>a</sup> </td>
<td class="description-td">Week 4</td>
<td class="description-td">Wk 4:Wk 0</td>
<td class="description-td">Week 4</td>
<td class="description-td">Wk 4:Wk 0</td>
<td class="description-td">Week 4</td>
<td class="description-td">Wk 4:Wk 0</td>
</tr>
<tr class="description-tr">
<td align="center" class="description-td" colspan="8" nameend="8" namest="1" rowsep="1"> </td>
</tr>
</tbody></table>
<table align="left" class="description-table" cols="8" colsep="0" rowsep="0" width="100%">
<tbody><tr class="description-tr">
<td class="description-td">â1</td>
<td class="description-td">4</td>
<td class="description-td">16</td>
<td class="description-td">4</td>
<td class="description-td">16</td>
<td class="description-td">4</td>
<td class="description-td">8</td>
<td class="description-td">32</td>
</tr>
<tr class="description-tr">
<td class="description-td">â3</td>
<td class="description-td">4</td>
<td class="description-td">4</td>
<td class="description-td">1</td>
<td class="description-td">4</td>
<td class="description-td">1</td>
<td class="description-td">4</td>
<td class="description-td">8</td>
</tr>
<tr class="description-tr">
<td class="description-td">â4</td>
<td class="description-td">4</td>
<td class="description-td">4</td>
<td class="description-td">1</td>
<td class="description-td">4</td>
<td class="description-td">1</td>
<td class="description-td">4</td>
<td class="description-td">64</td>
</tr>
<tr class="description-tr">
<td class="description-td">â5</td>
<td class="description-td">4</td>
<td class="description-td">32</td>
<td class="description-td">8</td>
<td class="description-td">16</td>
<td class="description-td">4</td>
<td class="description-td">16</td>
<td class="description-td">64</td>
</tr>
<tr class="description-tr">
<td class="description-td">â6A</td>
<td class="description-td">8</td>
<td class="description-td">64</td>
<td class="description-td">8</td>
<td class="description-td">32</td>
<td class="description-td">4</td>
<td class="description-td">32</td>
<td class="description-td">664</td>
</tr>
<tr class="description-tr">
<td class="description-td">â6B</td>
<td class="description-td">8</td>
<td class="description-td">64</td>
<td class="description-td">8</td>
<td class="description-td">32</td>
<td class="description-td">4</td>
<td class="description-td">32</td>
<td class="description-td">32</td>
</tr>
<tr class="description-tr">
<td class="description-td">â7F</td>
<td class="description-td">16</td>
<td class="description-td">32</td>
<td class="description-td">2</td>
<td class="description-td">16</td>
<td class="description-td">1</td>
<td class="description-td">16</td>
<td class="description-td">16</td>
</tr>
<tr class="description-tr">
<td class="description-td">â9V</td>
<td class="description-td">16</td>
<td class="description-td">16</td>
<td class="description-td">1</td>
<td class="description-td">32</td>
<td class="description-td">2</td>
<td class="description-td">32</td>
<td class="description-td">8</td>
</tr>
<tr class="description-tr">
<td class="description-td">14</td>
<td class="description-td">16</td>
<td class="description-td">16</td>
<td class="description-td">1</td>
<td class="description-td">16</td>
<td class="description-td">1</td>
<td class="description-td">16</td>
<td class="description-td">2</td>
</tr>
<tr class="description-tr">
<td class="description-td">18C</td>
<td class="description-td">4</td>
<td class="description-td">16</td>
<td class="description-td">4</td>
<td class="description-td">16</td>
<td class="description-td">4</td>
<td class="description-td">8</td>
<td class="description-td">64</td>
</tr>
<tr class="description-tr">
<td class="description-td">19A</td>
<td class="description-td">8</td>
<td class="description-td">8</td>
<td class="description-td">1</td>
<td class="description-td">8</td>
<td class="description-td">1</td>
<td class="description-td">16</td>
<td class="description-td">64</td>
</tr>
<tr class="description-tr">
<td class="description-td">19F</td>
<td class="description-td">4</td>
<td class="description-td">4</td>
<td class="description-td">1</td>
<td class="description-td">4</td>
<td class="description-td">1</td>
<td class="description-td">8</td>
<td class="description-td">64</td>
</tr>
<tr class="description-tr">
<td class="description-td">23F</td>
<td class="description-td">16</td>
<td class="description-td">32</td>
<td class="description-td">2</td>
<td class="description-td">16</td>
<td class="description-td">1</td>
<td class="description-td">32</td>
<td class="description-td">32</td>
</tr>
<tr class="description-tr">
<td align="center" class="description-td" colspan="8" nameend="8" namest="1" rowsep="1"> </td>
</tr>
<tr class="description-tr">
<td align="left" class="description-td" colspan="8" id="FOO-00009" nameend="8" namest="1"> <sup>a</sup>Used as week 0 values for all groups</td>
</tr>
</tbody></table>
</patent-tables>
</tables>
</div>
</li> <heading id="h-0087">Example 17</heading>
<heading id="h-0088">Alternate Procedure for Serotype 19A Pneumococcal Saccharide-CRM<sub>197 </sub>Conjugation</heading>
<heading id="h-0089">Overview</heading>
<li> <para-num num="[0245]"> </para-num> <div class="description-line" id="p-0253" num="0245">The following example describes a process for making an immunogenic conjugate comprising <i>Streptococcus pneumoniae </i>serotype 19A polysaccharide covalently linked to a carrier protein. In general, following periodate oxidation (activation) to generate reactive aldehyde groups on the polysaccharide, the serotype 19A polysaccharide was co-lyophilized with the carrier protein and conjugation was carried out in dimethyl sulfoxide (DMSO) via a reductive amination mechanism in the presence of sodium cyanoborohydride. As opposed to conjugation processes involving discrete lyophilization of polysaccharides and carrier proteins in DMSO, or processes involving aqueous co-lyophilization of polysaccharides and carrier proteins without DMSO, the following process improvement provided improved conjugation efficiency and control over the stability of serotype 19A polysaccharides as measured by molecular size and the percentage of free saccharide. Although this process is described for the serotype 19A polysaccharide, this process may also be used for serotypes that are structurally similar to serotype 19A, such as 6A, 6B, and 19F which also contain phosphodiester linkages between their repeat units.</div>
</li> <heading id="h-0090">Activation</heading>
<li> <para-num num="[0246]"> </para-num> <div class="description-line" id="p-0254" num="0246">Containers of serotype 19A polysaccharide were thawed and combined in a reaction vessel. Oxidation reactions were performed in 10 mM sodium acetate (pH 5) by the addition of 100 mM sodium acetate buffer (pH 5) at a polysaccharide concentration of 2 mg/mL. Oxidation was carried out in the presence of sodium periodate by incubation at 23Â±2Â° C. for 16-24 hours with 150 rpm mixing.</div>
</li> <heading id="h-0091">Purification of Activated Polysaccharide</heading>
<li> <para-num num="[0247]"> </para-num> <div class="description-line" id="p-0255" num="0247">Concentration and diafiltration of the activated serotype 19A polysaccharide was performed with 30K or 100K MWCO Pall Centramate polysulfone 1 ft<sup>2 </sup>ultrafiltration cassettes. A target membrane challenge of 2 grams of polysaccharide per ft<sup>2 </sup>of membrane area was used for purification of the activated polysaccharide. The ultrafiltration system was equilibrated in 10 mM sodium acetate buffer (pH 5) prior to the addition of the oxidation reaction solution. The oxidation reaction solution was then concentrated and diafiltered against 10 mM sodium acetate (pH 5). The activated polysaccharide was then stored at 5Â±3Â° C. for up to 14 days until the material was compounded for lyophilization.</div>
</li> <heading id="h-0092">Co-Lyophilization and Conjugation Process</heading>
<li> <para-num num="[0248]"> </para-num> <div class="description-line" id="p-0256" num="0248">The pH of the activated serotype 19A polysaccharide was adjusted to pH 6.5Â±0.2 by the addition of dibasic 1M sodium phosphate. The activated polysaccharide was compounded with sucrose at a ratio of 25 grams of sucrose per gram of polysaccharide, followed by compounding with CRM<sub>197 </sub>at a 0.8 saccharide/protein ratio. After shell freezing, the 100 mL glass bottles were placed in â25Â° C. storage.</div>
</li> <li> <para-num num="[0249]"> </para-num> <div class="description-line" id="p-0257" num="0249">The bottles of co-lyophilized polysaccharide/CRM<sub>197 </sub>were removed from the â25Â° C. freezer and allowed to equilibrate to room temperature in a laminar flow hood. The bottles were randomly sampled for moisture analysis. Dissolution of the co-lyophylized activated polysaccharide/CRM<sub>197 </sub>material in DMSO was performed at 2 mg/mL in the lyophilization bottles. The activated polysaccharide/CRM197 DMSO solution was then transferred to the conjugation vessel and stirred for 60-75 min at 23Â±2Â° C. at 70-120 rpm. With stirring, one molar equivalent of sodium cyanoborohydride was then added, followed by 1% WFI. The reaction solution was then stirred at 23Â±2Â° C. for 8-16 hours. One molar equivalent of sodium borohydride was added to the conjugation vessel for the capping reaction and the solution was stirred at 23Â±2Â° C. for 3-16 hours. The reaction solution was then diluted 10-fold in cold (2-8Â° C.) 0.9% sodium chloride.</div>
</li> <li> <para-num num="[0250]"> </para-num> <div class="description-line" id="p-0258" num="0250">The 10-fold diluted reaction solution was passed through a 1.2/5.0 Î¼m filter and then concentrated to 1-2 g/L on an ultrafiltration system equipped with a 300K or 1000K MWCO regenerated cellulose 1 ft<sup>2 </sup>membrane. A filter challenge of 1-2 grams of polysaccharide per ft<sup>2 </sup>of membrane area was used for the purification. A 10Ã diafiltration of the conjugate solution was then performed against 0.9% sodium chloride. Upon membrane cleaning, a 30Ã diafiltration of the conjugate solution was then performed against 5 mM sodium succinate/0.9% sodium chloride buffer (pH 6). The purified conjugate was then passed through a 0.22 Î¼m filter and samples for pre-FBC testing removed. The conjugate solution was then stored at 5Â±3Â° C. for up to 30 days until preparation for the Final Batch Concentrate (FBC).</div>
</li> <heading id="h-0093">Preparation of Final Batch Concentrate</heading>
<li> <para-num num="[0251]"> </para-num> <div class="description-line" id="p-0259" num="0251">The pre-FBC conjugate solution was diluted to a target concentration of 0.5 g/L using 5 mM sodium succinate/0.9% sodium chloride buffer. The solution was mixed with magnetic stirring for approximately 15 minutes and then passed through a 0.22 Î¼m filter into sterile polypropylene FBC bottles. The FBC conjugate solution was then stored at 5Â±3Â° C.</div>
</li> <li> <para-num num="[0252]"> </para-num> <div class="description-line" id="p-0260" num="0252">Characterization of the conjugate was performed as described in Example 4.</div>
</li> <heading id="h-0094">Co-Lyophilization Vs. Discrete Lyophilization in DMSO</heading>
<li> <para-num num="[0253]"> </para-num> <div class="description-line" id="p-0261" num="0253">Twelve 1-6 gram batches of serotype 19A conjugate were produced, with six produced using the co-lyophilization with DMSO method described above and six produced using a discrete lyophilization with DMSO method as described in Example 8 above. Characterization of the conjugate was performed as described in Example 4, including use of size exclusion chromatography media (CL-4B) to profile the relative molecular size distribution of the conjugate and a uronic acid assay to measure saccharide concentration. With respect to the long-term stability of the serotype 19A conjugate, a decrease in molecular size of about 10% with a concomitant increase in free saccharide levels of about 10% is expected over a period of 18 months. Accordingly, in the production of serotype 19A conjugates, a preferred value for conjugate molecular size is about 70% 0.3 Kd, with a preferred free saccharide level of below about 20-25%.</div>
</li> <li> <para-num num="[0254]"> </para-num> <div class="description-line" id="p-0262" num="0254">As shown in Table 7, characterization of the serotype 19A conjugates produced by both methods showed that the co-lyophilization process provided significantly improved conjugate characteristics in terms of both molecular size as well as percentage of free saccharide as compared to the discrete lyophilization process.</div>
</li> <li> <div class="description-line" id="p-0263" num="0000">
<tables id="TABLE-US-00007" num="00007">
<patent-tables colsep="0" frame="none" rowsep="0">
<table align="left" class="description-table" cols="1" colsep="0" rowsep="0" width="100%">
<thead>
<tr class="description-tr">
<td class="description-td" colspan="1" nameend="1" namest="1" rowsep="1">TABLE 7</td>
</tr>
</thead>
<tbody><tr class="description-tr">
<td align="center" class="description-td" colspan="1" nameend="1" namest="1" rowsep="1"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">Comparisons of Key Conjugate Characteristics for Serotype</td>
</tr>
<tr class="description-tr">
<td class="description-td">19A Co-lyophylization in DMSO vs. Discrete Lyophylization</td>
</tr>
</tbody></table>
<table align="left" class="description-table" cols="3" colsep="0" rowsep="0" width="100%">
<tbody><tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">Co-Lyophylization</td>
<td class="description-td">Discrete Lyophylization</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">(n = 6)</td>
<td class="description-td">(n = 6)</td>
</tr>
</tbody></table>
<table align="left" class="description-table" cols="5" colsep="0" rowsep="0" width="100%">
<tbody><tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">Standard</td>
<td class="description-td"> </td>
<td class="description-td">Standard</td>
</tr>
<tr class="description-tr">
<td class="description-td">Characteristic</td>
<td class="description-td">Mean</td>
<td class="description-td">Deviation</td>
<td class="description-td">Mean</td>
<td class="description-td">Deviation</td>
</tr>
<tr class="description-tr">
<td align="center" class="description-td" colspan="5" nameend="5" namest="1" rowsep="1"> </td>
</tr>
</tbody></table>
<table align="left" class="description-table" cols="5" colsep="0" rowsep="0" width="100%">
<tbody><tr class="description-tr">
<td class="description-td">%0.3 Kd (CL-4B)</td>
<td class="description-td">67</td>
<td class="description-td">7.2</td>
<td class="description-td">58</td>
<td class="description-td">13.0</td>
</tr>
<tr class="description-tr">
<td class="description-td">saccharide</td>
</tr>
<tr class="description-tr">
<td class="description-td">Free Saccharide</td>
<td class="description-td">&lt;18</td>
<td class="description-td">&lt;3.5</td>
<td class="description-td">31</td>
<td class="description-td">9.2</td>
</tr>
<tr class="description-tr">
<td class="description-td">(%)</td>
</tr>
<tr class="description-tr">
<td align="center" class="description-td" colspan="5" nameend="5" namest="1" rowsep="1"> </td>
</tr>
</tbody></table>
</patent-tables>
</tables>
</div>
</li> <li> <para-num num="[0255]"> </para-num> <div class="description-line" id="p-0264" num="0255">It should be understood that the foregoing discussion and examples merely present a detailed description of certain embodiments. It therefore should be apparent to those of ordinary skill in the art that various modifications and equivalents can be made without departing from the spirit and scope of the invention.</div>
</li> <li> <para-num num="[0256]"> </para-num> <div class="description-line" id="p-0265" num="0256">All journal articles, other references, patents and patent applications that are identified in this patent application are incorporated by reference in their entirety.</div>
</li> <heading id="h-0095">REFERENCES</heading>
<li> <div class="description-line" id="p-0266" num="0000">
</div> </li> <ul>
<li id="ul0003-0001" num="0257">1. Hausdorff W P, Bryant J, Paradiso P R, Siber G R. Which pneumococcal serogroups cause the most invasive disease: implications for conjugate vaccine formulation and use, part I. <i>Clin Infect Dis </i>2000; 30:100-21.</li>
<li id="ul0003-0002" num="0258">2. Hausdorff W P, Bryant J, Kloek C, Paradiso P R, Siber G R. The contribution of specific pneumococcal serogroups to different disease manifestations: implications for conjugate vaccine formulation and use, part I. <i>Clin Infect Dis </i>2000; 30:122-40.</li>
<li id="ul0003-0003" num="0259">3. Whitney C G, Farley M M, Hadler J, et al. Decline in invasive pneumococcal disease after the introduction of protein-polysaccharide conjugate vaccine. <i>New Engl J Med </i>2003; 348(18):1737-46.</li>
<li id="ul0003-0004" num="0260">4. Black S, Shinefield H. Hansen J, et al. Postlicensure evaluation of the effectiveness of seven valent pneumococcal conjugate vaccine. <i>Pediatr Infect Dis J </i>2001; 20; 11057.</li>
<li id="ul0003-0005" num="0261">5. Robinson K A, Baughman W, Rothrock G, et al. Epidemiology of invasive <i>Streptococcus pneumoniae </i>infections in the United States, 1995-1998: Opportunities for prevention in the conjugate vaccine era. <i>JAMA </i>2001; 285:1729-35.</li>
<li id="ul0003-0006" num="0262">6. Butler J, Breiman R, Lipman H, et al. Serotype distribution of <i>Streptococcus pneumoniae </i>infections among preschool children in the United States, 1978-1994<i>. J Infect Dis </i>1995; 171:885-9.</li>
<li id="ul0003-0007" num="0263">7. Whitney C G, Farley M M, Hadler J, et al. Increasing prevalence of multidrug-resistant <i>Streptococcus pneumoniae </i>in the United States. <i>N Engl J Med </i>2000; 343:1917-24.</li>
<li id="ul0003-0008" num="0264">8. Hofmann J, Cetron M S, Farley M M, et al. The prevalence of drug-resistant <i>Streptococcus pneumoniae </i>in Atlanta. <i>N Engl J Med </i>1995; 333:481-6.</li>
<li id="ul0003-0009" num="0265">9. Joloba M L, Windau A, Bajaksouzian S, Appelbaum P C Hausdorff W P, Jacobs M R. Pneumococcal conjugate vaccine serotypes of <i>Streptococcus pneumoniae </i>isolates and the antimicrobial susceptibility of such isolates in children with otitis media. <i>Clin Infect Dis </i>2001; 33:1489-94.</li>
<li id="ul0003-0010" num="0266">10. Black S, Shinefield H, Fireman B, et al. Efficacy, safety, and immunogenicity of heptavalent pneumococcal conjugate vaccine in children. <i>Pediatr Infect Dis J </i>2000; 19:187-95.</li>
<li id="ul0003-0011" num="0267">11. Rudolph K M, Parkinson A J, Reasonover A L, Bulkow L R, Parks D J, Butler J C. Serotype distribution and antimicrobial resistance patterns of invasive isolates of <i>Streptococcus pneumoniae </i>Alaska, 1991-1998<i>. J Infect Dis </i>2000; 182:490-6.</li>
<li id="ul0003-0012" num="0268">12. Sniadack D H, Schwartz B, Lipman H, et al. Potential interventions for the prevention of childhood pneumonia: geographic and temporal differences in serotype and serogroup distribution of sterile site pneumococcal isolates from children: implications for vaccine strategies. <i>Pediatr Infect Dis J </i>1995; 14:503-10.</li>
<li id="ul0003-0013" num="0269">13. Fagan R L, Hanna J N, Messer R D, Brookes D L, Murphy D M. The epidemiology of invasive pneumococcal disease in children in Far North Queensland. <i>J. Paediatr Child Health </i>2001; 37:571-5.</li>
<li id="ul0003-0014" num="0270">14. Kertesz D A, Di Fabio J L, de Cunto Brandileone M C, et al. Invasive <i>Streptococcus pneumoniae </i>infection in Latin American children: results of the Pan American Health Organization Surveillance Study. <i>Clin Infect Dis </i>1998; 26:1355-61.</li>
<li id="ul0003-0015" num="0271">15. Hausdorff W, Siber G, Paradiso P. Geographical differences in invasive pneumococcal disease rates and serotype frequency in young children. <i>Lancet </i>2001; 357:950-52.</li>
<li id="ul0003-0016" num="0272">16. Buckingham S C, King M D, Miller M L. Incidence and etiologies of complicated parapneumonic effusions in children, 1996 to 2001<i>. Pediatr Infect Dis J </i>2003; 22:499-504.</li>
<li id="ul0003-0017" num="0273">17. Byington C, Spencer L, Johnson T, et al. An epidemiological investigation of a sustained high rate of pediatric parapneumonic empyema: risk factors and microbiological associations. <i>Clin Infect Dis </i>2002; 34:434-40.</li>
<li id="ul0003-0018" num="0274">18. Tan T, Mason E, Wald E, et al. Clinical characteristics with complicated pneumonia caused by <i>Streptococcus pneumoniae. Pediatrics </i>2002; 110:1-6.</li>
<li id="ul0003-0019" num="0275">19. Block S L, Hedrick J, Harrison C J, et al. Pneumococcal serotypes from acute otitis media in rural Kentucky. <i>Pediatr Infect Dis J </i>2002; 21:859-65.</li>
<li id="ul0003-0020" num="0276">20. Hausdorff W P, Yothers G, Dagan R, et al Multinational study of pneumococcal serotypes causing acute otitis media in children. <i>Pediatr Infect Dis J </i>2002; 21:1008-16.</li>
<li id="ul0003-0021" num="0277">21. Robbins J B, Austrian R, Lee C J, et al. Considerations for formulating the second-generation pneumococcal capsular polysaccharide vaccine with emphasis on the cross-reactive types within groups. <i>J Infect Dis </i>1983; 148:1136-59.</li>
<li id="ul0003-0022" num="0278">22. Nahm M H, Olander J V, Magyarlaki M. Identification of cross-reactive antibodies with low opsonophagocytic activity for <i>Streptococcus pneumoniae. J Infect Dis </i>1997; 176:698-703.</li>
<li id="ul0003-0023" num="0279">23. Yu X, Gray B, Chang S, Ward J I, Edwards K M, Nahm M H. Immunity to cross-reactive serotypes induced by pneumococcal conjugate vaccines in infants. <i>J Infect Dis </i>1999; 180:1569-76.</li>
<li id="ul0003-0024" num="0280">24. Vakevainen M, Eklund C, Eskola J, Kayhty H. Cross-reactivity of antibodies to type 6B and 6A polysaccharides of <i>Streptococcus pneumoniae</i>, evoked by pneumococcal conjugate vaccines, in infants. <i>J Infect Dis </i>2001; 184:789-93.</li>
<li id="ul0003-0025" num="0281">25. Ekstrom N, Kilpi T, Lahdenkari M, Lehtonen H, Ahman H, Kayhty, H. Immune response to cross-reacting pneumococcal serotypes 6A6B and 19A19F in the FinOM vaccine trial, Third World of Congress of Pediatric Infectious Diseases, Santiago, Chile, Nov. 19-23, 2003.</li>
<li id="ul0003-0026" num="0282">26. Penn R L, Lewin E B, Douglas R G, Jr., Schiffman G, Lee C J, Robbins J B. Antibody responses in adult volunteers to pneumococcal polysaccharide types 19F and 19A administered singly and in combination. <i>Infect Immun </i>1982; 36:1261-2.</li>
<li id="ul0003-0027" num="0283">27. Giebink G S, Meier J D, Quartey M K, Liebeler C L, Le C T. Immunogenicity and efficacy of <i>Streptococcus pneumoniae </i>polysaccharide-protein conjugate vaccines against homologous and heterologous serotypes in the chinchilla otitis media model. <i>J Infect Dis </i>1996; 173:119-27.</li>
<li id="ul0003-0028" num="0284">28. Saeland E, Jakobsen H, Ingolfsdottir G, Sigurdardottir S T, Jonsdottir I. Serum samples from infants vaccinated with a pneumococcal conjugate vaccine, PncT, protect mice against invasive infection caused by <i>Streptococcus pneumoniae </i>serotypes 6A and 6B. <i>J Infect Dis </i>2001; 183:253-60.</li>
<li id="ul0003-0029" num="0285">29. Jakobsen H, Sigurdsson V D, Sigurdardottir S, Schulz D, Jonsdottir I. Pneumococcal serotype 19F conjugate vaccine induces cross-protective immunity in serotype 19A in a murine pneumococcal pneumonia model. <i>Infect Immun </i>2003; 71:2956-9.</li>
<li id="ul0003-0030" num="0286">30. Klugman K P, Madhi S A, Huebner R E, Kohberger R, Mbelle N, Pierce N. A trial of a 9-valent pneumococcal conjugate vaccine in children with and those without HIV infection. <i>N Engl J Med </i>2003; 349:1341-8.</li>
<li id="ul0003-0031" num="0287">31. O'Brien K L, Moulton L H, Reid R, et al. Efficacy and safety of seven-valent conjugate pneumococcal vaccine in American Indian children: group randomised trial. <i>Lancet </i>2003; 362:355-61.</li>
<li id="ul0003-0032" num="0288">32. Eskola J, Kilpi T, Palmu A, et al. Efficacy of a pneumococcal conjugate vaccine against acute otitis media. <i>N Engl J Med </i>2001: 344:403-9.</li>
<li id="ul0003-0033" num="0289">33. Pilishvili T, Farley M, Vazquez M, Reingold A, Nyquist A, et al. Effectiveness of heptavalent pneumococcal conjugate vaccine in children. Abst G-1079, ICAAC, Chicago, Ill., 2003.</li>
<li id="ul0003-0034" num="0290">34. U.S. Pat. No. 4,673,574.</li>
<li id="ul0003-0035" num="0291">35. U.S. Pat. No. 4,902,506.</li>
</ul>
</ul>
</div>
</section><section itemprop="claims" itemscope="">
<h2>Claims (<span itemprop="count">17</span>)</h2>
<div html="" itemprop="content"><div class="claims" lang="EN" load-source="patent-office" mxw-id="PCLM135690104">
<div class="claim"> <div class="claim" id="CLM-01-15" num="01-15">
<div class="claim-text"> <b>1</b>-<b>15</b>. (canceled)</div>
</div>
</div> <div class="claim"> <div class="claim" id="CLM-00016" num="00016">
<div class="claim-text"> <b>16</b>. A multivalent immunogenic composition, comprising 13 distinct polysaccharide-protein conjugates, together with a physiologically acceptable vehicle, wherein said polysaccharide-protein conjugates consist essentially of individual <i>Streptococcus pneumoniae </i>polysaccharide serotypes 1, 3, 4, 5, 6A, 6B, 7F, 9V, 14, 18C, 19A, 19F, and 23F, and wherein at least one of said serotypes 6A, 6B, 19A, or 19F is covalently linked to a carrier protein according to a method comprising:
<div class="claim-text">(a) reacting purified serotype 6A, 6B, 19A, or 19F polysaccharide with an oxidizing agent resulting in an activated serotype 6A, 6B, 19A, or 19F polysaccharide;</div> <div class="claim-text">(b) compounding the activated serotype 6A, 6B, 19A, or 19F polysaccharide with a carrier protein;</div> <div class="claim-text">(c) co-lyophilizing the compounded activated serotype 6A, 6B, 19A, or 19F polysaccharide and carrier protein;</div> <div class="claim-text">(d) re-suspending the compounded activated serotype 6A, 6B, 19A, or 19F polysaccharide and carrier protein in dimethyl sulfoxide (DMSO);</div> <div class="claim-text">(e) reacting the compounded, activated serotype 6A, 6B, 19A, or 19F polysaccharide and carrier protein with a reducing agent resulting in a serotype 6A, 6B, 19A, or 19F polysaccharide:carrier protein conjugate; and</div> <div class="claim-text">(f) capping unreacted aldehydes in the serotype 6A, 6B, 19A, or 19F polysaccharide:carrier protein conjugate resulting in an immunogenic conjugate comprising <i>Streptococcus pneumoniae </i>serotype 6A, 6B, 19A, or 19F polysaccharide covalently linked to a carrier protein;</div> <div class="claim-text">wherein said carrier protein is CRM197; and</div> <div class="claim-text">wherein step (c) is performed after step (b) and before steps (d) and (e).</div> </div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="CLM-00017" num="00017">
<div class="claim-text"> <b>17</b>. The multivalent immunogenic composition of <claim-ref idref="CLM-00016">claim 16</claim-ref>, wherein the activated serotype 6A, 6B, 19A, or 19F polysaccharide and CRM197 are compounded at a ratio of 0.8:1.</div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="CLM-00018" num="00018">
<div class="claim-text"> <b>18</b>. The multivalent immunogenic composition of <claim-ref idref="CLM-00016">claim 16</claim-ref>, wherein the pH of the activated serotype 6A, 6B, 19A, or 19F polysaccharide is adjusted to 6.5Â±0.2 prior to compounding with the carrier protein.</div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="CLM-00019" num="00019">
<div class="claim-text"> <b>19</b>. The multivalent immunogenic composition of <claim-ref idref="CLM-00016">claim 16</claim-ref>, wherein the activated serotype 6A, 6B, 19A, or 19F polysaccharide is compounded with sucrose prior to compounding with the carrier protein.</div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="CLM-00020" num="00020">
<div class="claim-text"> <b>20</b>. The multivalent immunogenic composition of <claim-ref idref="CLM-00016">claim 16</claim-ref>, further comprising purifying the immunogenic conjugate.</div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="CLM-00021" num="00021">
<div class="claim-text"> <b>21</b>. The multivalent immunogenic composition of <claim-ref idref="CLM-00016">claim 16</claim-ref>, wherein the oxidizing agent is sodium periodate.</div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="CLM-00022" num="00022">
<div class="claim-text"> <b>22</b>. The multivalent immunogenic composition of <claim-ref idref="CLM-00016">claim 16</claim-ref>, wherein the reducing agent is sodium cyanoborohydride.</div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="CLM-00023" num="00023">
<div class="claim-text"> <b>23</b>. The multivalent immunogenic composition of <claim-ref idref="CLM-00016">claim 16</claim-ref>, wherein capping unreacted aldehydes comprises reacting the serotype 6A, 6B, 19A, or 19F polysaccharide:carrier protein conjugate with sodium borohydride.</div>
</div>
</div> <div class="claim"> <div class="claim" id="CLM-00024" num="00024">
<div class="claim-text"> <b>24</b>. A multivalent immunogenic composition, comprising 13 distinct polysaccharide-protein conjugates, together with a physiologically acceptable vehicle, wherein said polysaccharide-protein conjugates consist essentially of individual <i>Streptococcus pneumoniae </i>polysaccharide serotypes 1, 3, 4, 5, 6A, 6B, 7F, 9V, 14, 18C, 19A, 19F, and 23F, and the capsular polysaccharides are individually prepared from a capsular polysaccharide that comprises a phosphodiester linkage between repeat units and at least one additional serotype,
<div class="claim-text">and wherein at least one of said serotypes 6A, 6B, 19A, or 19F capsular polysaccharide that comprises a phosphodiester linkage between repeat units is individually covalently linked to a carrier protein according to a method comprising:</div> <div class="claim-text">(a) reacting purified serotype 6A, 6B, 19A, or 19F polysaccharide with an oxidizing agent resulting in an activated serotype 6A, 6B, 19A, or 19F polysaccharide;</div> <div class="claim-text">(b) compounding the activated serotype 6A, 6B, 19A, or 19F polysaccharide with a carrier protein;</div> <div class="claim-text">(c) co-lyophilizing the compounded activated serotype 6A, 6B, 19A, or 19F polysaccharide and carrier protein;</div> <div class="claim-text">(d) re-suspending the compounded activated serotype 6A, 6B, 19A, or 19F polysaccharide and carrier protein in dimethyl sulfoxide (DMSO);</div> <div class="claim-text">(e) reacting the compounded, activated serotype 6A, 6B, 19A, or 19F polysaccharide and carrier protein with a reducing agent resulting in a serotype 6A, 6B, 19A, or 19F polysaccharide:carrier protein conjugate; and</div> <div class="claim-text">(f) capping unreacted aldehydes in the serotype 6A, 6B, 19A, or 19F polysaccharide:carrier protein conjugate resulting in an immunogenic conjugate comprising <i>Streptococcus pneumoniae </i>serotype 6A, 6B, 19A, or 19F polysaccharide covalently linked to a carrier protein;</div> <div class="claim-text">wherein said carrier protein is CRM197; and</div> <div class="claim-text">wherein step (c) is performed after step (b) and before steps (d) and (e).</div> </div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="CLM-00025" num="00025">
<div class="claim-text"> <b>25</b>. The immunogenic conjugate of <claim-ref idref="CLM-00024">claim 24</claim-ref>, wherein the activated serotype 19A polysaccharide and CRM197 are compounded at a ratio of 0.8:1.</div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="CLM-00026" num="00026">
<div class="claim-text"> <b>26</b>. The immunogenic conjugate of <claim-ref idref="CLM-00024">claim 24</claim-ref>, wherein the pH of the activated serotype 19A polysaccharide is adjusted to 6.5Â±0.2 prior to compounding with the carrier protein.</div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="CLM-00027" num="00027">
<div class="claim-text"> <b>27</b>. The immunogenic conjugate of <claim-ref idref="CLM-00024">claim 24</claim-ref>, wherein the activated serotype 19A polysaccharide is compounded with sucrose prior to compounding with the carrier protein.</div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="CLM-00028" num="00028">
<div class="claim-text"> <b>28</b>. The immunogenic conjugate of <claim-ref idref="CLM-00024">claim 24</claim-ref>, further comprising purifying the immunogenic conjugate.</div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="CLM-00029" num="00029">
<div class="claim-text"> <b>29</b>. The immunogenic conjugate of <claim-ref idref="CLM-00024">claim 24</claim-ref>, wherein the oxidizing agent is sodium periodate.</div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="CLM-00030" num="00030">
<div class="claim-text"> <b>30</b>. The immunogenic conjugate of <claim-ref idref="CLM-00024">claim 24</claim-ref>, wherein the reducing agent is sodium cyanoborohydride.</div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="CLM-00031" num="00031">
<div class="claim-text"> <b>31</b>. The immunogenic conjugate of <claim-ref idref="CLM-00024">claim 24</claim-ref>, wherein capping unreacted aldehydes comprises reacting the serotype 19A polysaccharide:carrier protein conjugate with sodium borohydride.</div>
</div>
</div> </div>
</div>
</section>
                </article>
            </search-app>
        </body>
    </html>
    