
    <html>
        <body>
            <search-app>
                <article class="result" itemscope="" itemtype="http://schema.org/ScholarlyArticle">
    <h1 itemprop="pageTitle">US20190085324A1 - Assays for massively combinatorial perturbation profiling and cellular circuit reconstruction 
      - Google Patents</h1><section itemprop="abstract" itemscope="">
<h2>Abstract</h2>
<div html="" itemprop="content"><abstract lang="EN" load-source="docdb" mxw-id="PA289768882" source="national office">
<div class="abstract">The present invention provides tools and methods for the systematic analysis of genetic interactions, including higher order interactions. The present invention provides tools and methods for combinatorial probing of cellular circuits, for dissecting cellular circuitry, for delineating molecular pathways, and/or for identifying relevant targets for therapeutics development.</div>
</abstract>
</div>
</section><section itemprop="description" itemscope="">
<h2>Description</h2>
<div html="" itemprop="content"><ul class="description" lang="EN" load-source="patent-office" mxw-id="PDES187295534">
<heading id="h-0001">INCORPORATION BY REFERENCE</heading>
<li> <para-num num="[0001]"> </para-num> <div class="description-line" id="p-0002" num="0001">This application is a Continuation-In-Part of International Application Number PCT/US16/59233 filed on Oct. 27, 2016, which published as WO2017/075294 on May 4, 2017 and claims priority and benefit of U.S. provisional application Ser. Nos. 62/247,729 filed Oct. 28, 2015, 62/394,721 filed Sep. 14, 2016, and 62/395,273 filed Sep. 15, 2016.</div>
</li> <li> <para-num num="[0002]"> </para-num> <div class="description-line" id="p-0003" num="0002">Reference is also made to U.S. provisional application Ser. Nos. 62/247,630 filed Oct. 28, 2015, 62/247,656 filed Oct. 28, 2015, and 62/372,393 filed Aug. 9, 2016.</div>
</li> <li> <para-num num="[0003]"> </para-num> <div class="description-line" id="p-0004" num="0003">The foregoing applications, and all documents cited therein or during their prosecution (“appln cited documents”) and all documents cited or referenced in the appln cited documents, and all documents cited or referenced herein (“herein cited documents”), and all documents cited or referenced in herein cited documents, together with any manufacturer's instructions, descriptions, product specifications, and product sheets for any products mentioned herein or in any document incorporated by reference herein, are hereby incorporated herein by reference, and may be employed in the practice of the invention. More specifically, all referenced documents are incorporated by reference to the same extent as if each individual document was specifically and individually indicated to be incorporated by reference.</div>
</li> <heading id="h-0002">FEDERAL FUNDING LEGEND</heading>
<li> <para-num num="[0004]"> </para-num> <div class="description-line" id="p-0005" num="0004">This invention was made with government support under grant numbers HG006193 MH105960, GM102706, CA168370, and DA036858 awarded by the National Institutes of Health. The government has certain rights in the invention.</div>
</li> <heading id="h-0003">FIELD OF THE INVENTION</heading>
<li> <para-num num="[0005]"> </para-num> <div class="description-line" id="p-0006" num="0005">The present invention provides methods and tools for analyzing genetic interactions. The present invention relies on a systematic approach of causing single and combinatorial genome-wide perturbations in cells, with subsequent molecular profiling at the single cell level. Applications include dissection of cell circuitry and delineation of functional or molecular pathways. The present invention is also relevant for therapeutics target discovery.</div>
</li> <heading id="h-0004">BACKGROUND OF THE INVENTION</heading>
<li> <para-num num="[0006]"> </para-num> <div class="description-line" id="p-0007" num="0006">Regulatory circuits in cells process signals, trigger decisions, and orchestrate physiological responses under diverse conditions. Diseases, in turn, arise from circuit malfunctions: one or more components are missing or defective; a key component is over- or under-active. To understand mechanisms underlying disease and develop more effective treatments, it would be highly advantageous to be able to provide a comprehensive picture of all cellular components, to identify the circuits in which they function, and to delineate how these components and circuits are integrated to form cellular responses.</div>
</li> <li> <para-num num="[0007]"> </para-num> <div class="description-line" id="p-0008" num="0007">Genomic research on dissecting cellular circuitry has generally had distinct phases: genomic observations and perturbation of single components.</div>
</li> <li> <para-num num="[0008]"> </para-num> <div class="description-line" id="p-0009" num="0008">Early advances in functional genomics made it possible to observe molecular profiles in different cells. Such global analysis has been very powerful in drawing hypotheses that relate regulators to their targets from statistical correlations. However, it is also very limited: the hypotheses were mostly not tested, and because correlation is not causation, many hypotheses may be found partially or fully incorrect.</div>
</li> <li> <para-num num="[0009]"> </para-num> <div class="description-line" id="p-0010" num="0009">In recent years, efforts were implemented in order to determine causation. Genomic profiles were used to infer a molecular model, on an increasingly large scale, based on genetic manipulations. However, the approach of testing genes individually has limitations: because genes involved in biological circuits have non-linear interactions, one cannot predict how a cellular circuit functions simply by summing up the individual effects. Indeed, biological systems are not linear: the combined effect of multiple factors is not simply the sum of their individual effects. This is a direct outcome of the biochemistry underlying molecular biology, from allosteric protein changes to cooperative binding, and is essential for cells to process complex signals.</div>
</li> <li> <para-num num="[0010]"> </para-num> <div class="description-line" id="p-0011" num="0010">It has remained an insurmountable stumbling block to achieving a quantitative and predictive understanding of circuits on a genomic scale, with far-reaching implications for basic and translational science. For example, despite decades of work, one still cannot predict how the enhancer controlling the transcription of the interferon beta gene (IFNβ) behaves in response to viral and other stimuli. In another example, p38α, a serine/threonine kinase with key roles in inflammation, has been studied for two decades, and yet it remains unclear how it balances control of inflammatory and anti-inflammatory cytokines, and many therapeutics programs launched to target this protein were hampered by unwarranted and unexpected effects. Finally, in genomic studies ranging from yeast to mammals, many molecular events (e.g., transcription factor binding) appear ‘functionally silent’ upon factor perturbation, and only some expression variation is explained with available mechanistic data.</div>
</li> <li> <para-num num="[0011]"> </para-num> <div class="description-line" id="p-0012" num="0011">Citation or identification of any document in this application is not an admission that such document is available as prior art to the present invention.</div>
</li> <heading id="h-0005">SUMMARY OF THE INVENTION</heading>
<li> <para-num num="[0012]"> </para-num> <div class="description-line" id="p-0013" num="0012">A current phase of genomic research on dissecting cellular circuitry involves combinatorial probing of circuits. It would be desirable to provide a combinatorial approach: perturbing multiple components, at a large enough scale that will allow one to reliably reconstruct cellular circuits; for example, simultaneously or at or about the same time or in parallel. Such a combinatorial genomics approach has generally been considered intractable, because it required: (1) the ability to perturb many genes simultaneously (or at or about the same time or in parallel) in the same cell; (2) the ability to readout genomic profiles in individual cells, so that the effect of many perturbations can be assessed in parallel in a pool of cells; and (3) the development of mathematics and computational tools, because even millions of experiments are very few compared to the staggering size of the possible combinatorial space.</div>
</li> <li> <para-num num="[0013]"> </para-num> <div class="description-line" id="p-0014" num="0013">The invention involves Massively Parallel Combinatorial Perturbation Profiling (MCPP) to address or identify genetic interactions. Biological systems are not linear: the combined effect of multiple factors is not simply the sum of their individual effects. This is a direct outcome of the biochemistry underlying molecular biology, from allosteric protein changes to cooperative binding, and is essential for cells to process complex signals. However, heretofore, it has remained an insurmountable stumbling block to quantitative and predictive biology on a genomic scale, with far-reaching implications e.g., from basic research to clinical translation. The invention provides a combinatorial approach: perturbing multiple components simultaneously. Yet, even recent experimental advances in genetics and genomics heretofore have not been enough; for instance, studying all 2-way, 3-way and 4-way interactions would, prior to the instant invention, require performing ˜10<sup>8</sup>, ˜10<sup>12</sup>, and —10<sup>16 </sup>combinations of perturbations, respectively; and, beyond the technological and cost issues, the sheer number of mammalian cells required becomes infeasible because, for example, even if one focused on 100 genes, exhaustive analysis of 5-way interactions would require 70 million perturbations.</div>
</li> <li> <para-num num="[0014]"> </para-num> <div class="description-line" id="p-0015" num="0014">It would be desirable to provide tools and methods for the systematic analysis of genetic interactions, including higher order interactions.</div>
</li> <li> <para-num num="[0015]"> </para-num> <div class="description-line" id="p-0016" num="0015">It would be desirable to provide tools and/or methods for combinatorial probing of cellular circuits, for instance, for dissecting cellular circuitry, for delineating molecular pathways and/or interactions (e.g., intercellular and/or intracellular pathways or interactions), for identifying relevant targets and/or for identifying impact or effect of perturbations or stimuli or mutation; for instance for therapeutics development and/or cellular engineering and/or any cellular manipulation and/or ascertaining internal cell function and/or for bioproduction (e.g., production of antibodies from new sources, expression of products from organisms or cells that previously did not naturally express such products, increasing or decreasing expression of endogenous products, and the like), new plants or animal models.</div>
</li> <li> <para-num num="[0016]"> </para-num> <div class="description-line" id="p-0017" num="0016">The present invention involves cellular circuits (both intracellular and extracellular circuits). For instance, a cellular, e.g., regulatory circuit combines trans inputs (such as the levels and activities of factors, e.g., transcription factors, non-coding RNAs, e.g., regulatory RNAs and signalling molecules) and cis inputs (such as sequences, e.g., regulatory sequences in the promoter and enhancer of a gene); for instance, to determine the level of mRNA produced from a gene.</div>
</li> <li> <para-num num="[0017]"> </para-num> <div class="description-line" id="p-0018" num="0017">Reconstruction of a cellular, e.g., regulatory circuit is to identify inputs, e.g., all identifiable inputs (for example, proteins, non-coding RNAs and cis-regulatory elements), their physical ‘wirings’ (or connections) and the transcriptional functions that they implement; for instance, as to regulation of the level of mRNA.</div>
</li> <li> <para-num num="[0018]"> </para-num> <div class="description-line" id="p-0019" num="0018">A model should address (advantageously simultaneously or in parallel) providing a functional description of the input-output relationships (for example, if regulator A is induced, then target gene B is repressed to a particular extent), and providing a physical description of the circuit (for example, regulator A binds to the promoter of gene B in sequence Y, modifies its chromatin and leads to repression). Networks, e.g., regulatory networks, control complex downstream cellular phenotypes (such as cell death, proliferation and migration).</div>
</li> <li> <para-num num="[0019]"> </para-num> <div class="description-line" id="p-0020" num="0019">Reconstructing the connectivity of a network can be through the monitoring of hundreds to thousands of cellular parameters (massively parallel monitoring or hundreds to thousands of cellular parameters), such as the levels of mRNAs. Hence “massively parallel” can mean undertaking a particular activity hundreds to thousands to millions, e.g., from 100 to 1000 or to 10,000 or to 100,000 or to 1,000,000 or up to 1,000,000,000 times (or as otherwise indicated herein or in figures herewith), in parallel, e.g., simultaneously or at or about the same time. See, e.g., Amit et al., “Strategies to discover regulatory circuits of the mammalian immune system,” NATURE REVIEWS (IMMUNOLOGY) 11: 873-880 (DECEMBER 2011).</div>
</li> <li> <para-num num="[0020]"> </para-num> <div class="description-line" id="p-0021" num="0020">The present invention relates to methods of measuring or determining or inferring RNA levels, e.g., massively parallel measuring or determining or inferring of RNA levels in a single cell or a cellular network or circuit in response to at least one perturbation parameter or advantageously a plurality of perturbation parameters or massively parallel perturbation parameters involving sequencing DNA of a perturbed cell, whereby RNA level and optionally protein level may be determined in the single cell in response to the at least one perturbation parameter or advantageously a plurality of perturbation parameters or massively parallel perturbation parameters. The invention thus may involve a method of inferring or determining or measuring RNA in a single cell or a cellular network or circuit, e.g., massively parallel inferring or determining or measuring of RNA level in a single cell or a cellular network or circuit in response to at least 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22, 23, 24, 25, 26, 27, 28, 29, 30, 31, 32, 33, 34, 35, 36, 37, 38, 39, 40, 41, 42, 43, 44, 45, 46, 47, 48, 49, 50, 51, 52, 53, 54, 55, 56, 57, 58, 59, 60, 61, 62, 63, 64, 65, 66, 67, 68, 69, 70, 71, 72, 73, 74, 75, 76, 77, 78, 79, 80, 81, 82, 83, 84, 85, 86, 87, 88, 89, 90, 91, 92, 93, 94, 95, 96, 97, 98 ,99 or 100 or massively parallel perturbation parameter(s) comprising optionally so perturbing the cell or the cells or each cell of the cellular network or circuit with the perturbation parameter(s) and sequencing of the perturbed cell(s), whereby RNA level(s) and optionally protein level(s) is/are determined in the cell(s) in response to the perturbation parameter(s).</div>
</li> <li> <para-num num="[0021]"> </para-num> <div class="description-line" id="p-0022" num="0021">Genetic screens are used to infer gene function in mammalian cells, but it has remained difficult to assay complex phenotypes—such as genome-wide transcriptional profiles—in large-scale screens. Moreover, it has been traditionally difficult to assay the transcriptional phenotype of genetic perturbations at scale. Preferably, a genomewide scale transcriptome phenotype associated with a perturbation would be possible.</div>
</li> <li> <para-num num="[0022]"> </para-num> <div class="description-line" id="p-0023" num="0022">Here, Applicants developed Perturb-seq, which combines single cell RNA-seq and CRISPR/Cas9 based perturbations identified by unique polyadenylated barcodes to perform many, tens of thousands in certain embodiments, of such assays in a single pooled experiment. By randomly integrating more than one sgRNA in each cell, Perturb-Seq is extended to test transcriptional phenotypes caused by genetic interactions. Applicants develop a computational framework, MIMOSCA (Multi-Input Multi-Output Single Cell Analysis) to identify the regulatory effects of individual perturbations and their combinations at different levels of resolution: from effects on each individual gene to functional signatures to proportional changes in cell types. Applicants demonstrate Perturb-seq by analyzing 200,000 cells across three screens: transcription factors controlling the immune response of dendritic cells to LPS, transcription factors bound in the K562 cell line, and cell cycle regulators in the same cell line. Perturb-Seq accurately identified known regulatory relations, and its individual gene target predictions were validated by ChIP-Seq binding profiles. Applicants posit new functions for regulatory factors affecting cell differentiation, the anti-viral response and mitochondrial function during immune activation, and uncovered an underlying circuit that balances these different programs through positive and negative feedback loops. Using Perturb-Seq Applicants identified genetic interactions including synergistic, buffering and dominant genetic interactions that could not be predicted from individual perturbations alone. Perturb-Seq can be flexibly applied to diverse cell metadata, to customize design and scope of pooled genomic assays.</div>
</li> <li> <para-num num="[0023]"> </para-num> <div class="description-line" id="p-0024" num="0023">Applicants also applied perturb-seq to dissect the mammalian unfolded protein response (UPR). First, Applicants used perturb-seq to build an epistatic map of UPR-mediated transcription. Then, Applicants conducted a genome-scale CRISPRi screen to identify genes whose depletion perturbs ER homeostasis and subjected a subset of our hits (using a ˜100 element sublibrary) to perturb-seq, revealing high precision functional gene clusters. Single-cell analyses revealed both bifurcations in behavior in cells subject to the same perturbation as well as differential activation of the three UPR branches across hits, including an isolated feedback loop between the translocon and the IRE1α branch of the UPR. These studies provide insight into how the three separate sensors of ER homeostasis allow the cell to monitor distinct types of ER stress and, more generally, highlight the ability of perturb-seq to dissect complex cellular responses.</div>
</li> <li> <para-num num="[0024]"> </para-num> <div class="description-line" id="p-0025" num="0024">By combining droplet based single cell transcriptomics with CRISPR-Cas based perturbations, Applicants demonstrate an approach that allow researchers to perform thousands of these assays in a single pooled experiment. This pooled approach represents a 10-fold improvement in cost over current methods for obtaining perturbation transcription profiles. Leveraging the discrete nature of the single cell measurements, the screening approach also has the ability to resolve novel phenotypes such as the effect of a perturbation on cell type composition or cell cycle phase, and filter unperturbed cells whose presence otherwise dilutes the measured effect in population measurements. Finally, by randomly integrating more than one sgRNA in each cell, applicants show the ability to extend the method to test combinatorial effects. Together, these statistical and experimental methods enable researchers to perform large-scale screening of perturbations, including systematic dissection of epistatic effects, using RNA transcription as a phenotype.</div>
</li> <li> <para-num num="[0025]"> </para-num> <div class="description-line" id="p-0026" num="0025">In one aspect, the present invention provides for a method of reconstructing a cellular network or circuit, comprising introducing at least 1, 2, 3, 4 or more single-order or combinatorial perturbations to a plurality of cells in a population of cells, wherein each cell in the plurality of the cells receives at least 1 perturbation; measuring comprising: detecting genomic, genetic, proteomic, epigenetic and/or phenotypic differences in single cells compared to one or more cells that did not receive any perturbation, and detecting the perturbation(s) in single cells; and determining measured differences relevant to the perturbations by applying a model accounting for co-variates to the measured differences, whereby intercellular and/or intracellular networks or circuits are inferred. The measuring in single cells may comprise single cell sequencing. The single cell sequencing may comprise cell barcodes, whereby the cell-of-origin of each RNA is recorded. The single cell sequencing may comprise unique molecular identifiers (UMI), whereby the capture rate of the measured signals, such as transcript copy number or probe binding events, in a single cell is determined. The model may comprise accounting for the capture rate of measured signals, whether the perturbation actually perturbed the cell (phenotypic impact), the presence of subpopulations of either different cells or cell states, and/or analysis of matched cells without any perturbation.</div>
</li> <li> <para-num num="[0026]"> </para-num> <div class="description-line" id="p-0027" num="0026">The single-order or combinatorial perturbations may comprise 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22, 23, 24, 25, 26, 27, 28, 29, 30, 31, 32, 33, 34, 35, 36, 37, 38, 39, 40, 41, 42, 43, 44, 45, 46, 47, 48, 49, 50, 51, 52, 53, 54, 55, 56, 57, 58, 59, 60, 61, 62, 63, 64, 65, 66, 67, 68, 69, 70, 71, 72, 73, 74, 75, 76, 77, 78, 79, 80, 81, 82, 83, 84, 85, 86, 87, 88, 89, 90, 91, 92, 93, 94, 95, 96, 97, 98, 99 or 100 perturbations. The perturbation(s) may target genes in a pathway or intracellular network.</div>
</li> <li> <para-num num="[0027]"> </para-num> <div class="description-line" id="p-0028" num="0027">The measuring may comprise detecting the transcriptome of each of the single cells. The perturbation(s) may comprise one or more genetic perturbation(s). The perturbation(s) may comprise one or more epigenetic or epigenomic perturbation(s). At least one perturbation may be introduced with RNAi- or a CRISPR-Cas system. At least one perturbation may be introduced via a chemical agent, biological agent, an intracellular spatial relationship between two or more cells, an increase or decrease of temperature, addition or subtraction of energy, electromagnetic energy, or ultrasound.</div>
</li> <li> <para-num num="[0028]"> </para-num> <div class="description-line" id="p-0029" num="0028">The cell(s) may comprise a cell in a model non-human organism, a model non-human mammal that expresses a Cas protein, a mouse that expresses a Cas protein, a mouse that expresses Cpf1, a cell in vivo or a cell ex vivo or a cell in vitro. The cell(s) may also comprise a human cell.</div>
</li> <li> <para-num num="[0029]"> </para-num> <div class="description-line" id="p-0030" num="0029">The measuring or measured differences may comprise measuring or measured differences of DNA, RNA, protein or post translational modification; or measuring or measured differences of protein or post translational modification correlated to RNA and/or DNA level(s).</div>
</li> <li> <para-num num="[0030]"> </para-num> <div class="description-line" id="p-0031" num="0030">The perturbing or perturbation(s) may comprise(s) genetic perturbing. The perturbing or perturbation(s) may comprise(s) single-order perturbations. The perturbing or perturbation(s) may comprise(s) combinatorial perturbations. The perturbing or perturbation(s) may comprise gene knock-down, gene knock-out, gene activation, gene insertion, or regulatory element deletion. The perturbing or perturbation(s) may comprise genome-wide perturbation. The perturbing or perturbation(s) may comprise performing CRISPR-Cas-based perturbation. The perturbing or perturbation(s) may comprise performing pooled single or combinatorial CRISPR-Cas-based perturbation with a genome-wide library of sgRNAs. The perturbations may be of a selected group of targets based on similar pathways or network of targets.</div>
</li> <li> <para-num num="[0031]"> </para-num> <div class="description-line" id="p-0032" num="0031">The perturbing or perturbation(s) may comprises performing pooled combinatorial CRISPR-Cas-based perturbation with a genome-wide library of sgRNAs. Each sgRNA may be associated with a unique perturbation barcode. Each sgRNA may be co-delivered with a reporter mRNA comprising the unique perturbation barcode (or sgRNA perturbation barcode).</div>
</li> <li> <para-num num="[0032]"> </para-num> <div class="description-line" id="p-0033" num="0032">The perturbing or perturbation(s) may comprise subjecting the cell to an increase or decrease in temperature. The perturbing or perturbation(s) may comprise subjecting the cell to a chemical agent. The perturbing or perturbation(s) may comprise subjecting the cell to a biological agent. The biological agent may be a toll like receptor agonist or cytokine. The perturbing or perturbation(s) may comprise subjecting the cell to a chemical agent, biological agent and/or temperature increase or decrease across a gradient.</div>
</li> <li> <para-num num="[0033]"> </para-num> <div class="description-line" id="p-0034" num="0033">The cell may be in a microfluidic system. The cell may be in a droplet. The population of cells may be sequenced by using microfluidics to partition each individual cell into a droplet containing a unique barcode, thus allowing a cell barcode to be introduced.</div>
</li> <li> <para-num num="[0034]"> </para-num> <div class="description-line" id="p-0035" num="0034">The perturbing or perturbation(s) may comprise transforming or transducing the cell or a population that includes and from which the cell is isolated with one or more genomic sequence-perturbation constructs that perturbs a genomic sequence in the cell. The sequence-perturbation construct may be a viral vector, preferably a lentivirus vector. The perturbing or perturbation(s) may comprise multiplex transformation or transduction with a plurality of genomic sequence-perturbation constructs.</div>
</li> <li> <para-num num="[0035]"> </para-num> <div class="description-line" id="p-0036" num="0035">In another aspect, or in alternative embodiments of aspects described herein, the present invention provides for a method wherein proteins or transcripts expressed in single cells are determined in response to a perturbation, wherein the proteins or transcripts are detected in the single cells by binding of more than one labeling ligand comprising an oligonucleotide tag, and wherein the oligonucleotide tag comprises a unique constituent identifier (UCI) specific for a target protein or transcript. The single cells may be fixed in discrete particles. The discrete particles may be washed and sorted, such that cell barcodes may be added, e.g. sgRNA perturbation barcodes as described above. The oligonucleotide tag and sgRNA perturbation barcode may comprise a universal ligation handle sequence, whereby a unique cell barcode may be generated by split-pool ligation. The labeling ligand may comprise an oligonucleotide label comprising a regulatory sequence configured for amplification by T7 polymerase. The labeling ligands may comprise oligonucleotide sequences configured to hybridize to a transcript specific region. Not being bound by a theory, both proteins and RNAs may be detected after perturbation. The oligonucleotide label may further comprise a photocleavable linker. The oligonucleotide label may further comprise a restriction enzyme site between the labeling ligand and unique constituent identifier (UCI). The ligation handle may comprise a restriction site for producing an overhang complementary with a first index sequence overhang, and wherein the method further comprises digestion with a restriction enzyme. The ligation handle may comprise a nucleotide sequence complementary with a ligation primer sequence and wherein the overhang complementary with a first index sequence overhang is produced by hybridization of the ligation primer to the ligation handle. The method may further comprise quantitating the relative amount of UCI sequence associated with a first cell to the amount of the same UCI sequence associated with a second cell, whereby the relative differences of a cellular constituent between cell(s) are determined. The labeling ligand may comprise an antibody or an antibody fragment. The antibody fragment may be a nanobody, Fab, Fab′, (Fab′)2, Fv, ScFv, diabody, triabody, tetrabody, Bis-scFv, minibody, Fab2, or Fab3 fragment. The labeling ligand may comprise an aptamer. The labeling ligand may be a nucleotide sequence complementary to a target sequence.</div>
</li> <li> <para-num num="[0036]"> </para-num> <div class="description-line" id="p-0037" num="0036">Single cell sequencing may comprise whole transcriptome amplification.</div>
</li> <li> <para-num num="[0037]"> </para-num> <div class="description-line" id="p-0038" num="0037">The method in aspects of the invention may comprise comparing an RNA profile of the perturbed cell with any mutations in the cell to also correlate phenotypic or transcriptome profile and genotypic profile.</div>
</li> <li> <para-num num="[0038]"> </para-num> <div class="description-line" id="p-0039" num="0038">In another aspect, or in alternative embodiments of aspects described herein, the present invention provides for a method comprising determining genetic interactions by causing a set of P genetic perturbations in single cells of the population of cells, wherein the method comprises: determining, based upon random sampling, a subset of π genetic perturbations from the set of P genetic perturbations; performing said subset of π genetic perturbations in a population of cells; performing single-cell molecular profiling of the population of genetically perturbed cells; inferring, from the results and based upon the random sampling, single-cell molecular profiles for the set of P genetic perturbations in cells. The method may further comprises: from the results, determining genetic interactions. The method may further comprise: confirming genetic interactions determined with additional genetic manipulations.</div>
</li> <li> <para-num num="[0039]"> </para-num> <div class="description-line" id="p-0040" num="0039">The set of P genetic perturbations or said subset of π genetic perturbations may comprise single-order genetic perturbations. The set of P genetic perturbations or said subset of π genetic perturbations may comprise combinatorial genetic perturbations. The genetic perturbation may comprise gene knock-down, gene knock-out, gene activation, gene insertion, or regulatory element deletion.The set of P genetic perturbations or said subset of π genetic perturbations may comprise genome-wide perturbations. The set of P genetic perturbations or said subset of π genetic perturbations may comprise k-order combinations of single genetic perturbations, wherein k is an integer ranging from 2 to 15, and wherein the method comprises determining k-order genetic interactions. The set of P genetic perturbations may comprise combinatorial genetic perturbations, such as k-order combinations of single-order genetic perturbations, wherein k is an integer ranging from 2 to 15, and wherein the method comprises determining j-order genetic interactions, with j&lt;k.</div>
</li> <li> <para-num num="[0040]"> </para-num> <div class="description-line" id="p-0041" num="0040">The method in aspects of this invention may comprise performing RNAi- or CRIPSR-Cas-based perturbation. The method may comprise an array-format or pool-format perturbation. The method may comprise pooled single or combinatorial CRISPR-Cas-based perturbation with a genome-wide library of sgRNAs. The method may comprise pooled combinatorial CRISPR-Cas-based perturbation with a genome-wide library of sgRNAs.</div>
</li> <li> <para-num num="[0041]"> </para-num> <div class="description-line" id="p-0042" num="0041">The random sampling may comprise matrix completion, tensor completion, compressed sensing, or kernel learning. The random sampling may comprise matrix completion, tensor completion, or compressed sensing, and wherein π is of the order of log P.</div>
</li> <li> <para-num num="[0042]"> </para-num> <div class="description-line" id="p-0043" num="0042">The cell may comprise a eukaryotic cell. The eukaryotic cell may comprise a mammalian cell. The mammalian cell may comprise a human cell. The cell may be from a population comprising 10<sup>2 </sup>to 10<sup>8 </sup>cells and DNA or RNA or protein or post translational modification measurements or variables per cell comprise 50 or more.</div>
</li> <li> <para-num num="[0043]"> </para-num> <div class="description-line" id="p-0044" num="0043">The perturbation of the population of cells may be performed in vivo. The perturbation of the population of cells may be performed ex vivo and the population of cells may be adoptively transferred to a subject. The population of cells may comprise tumor cells. The method may comprise a lineage barcode associated with single cells, whereby the lineage or clonality of single cells may be determined.</div>
</li> <li> <para-num num="[0044]"> </para-num> <div class="description-line" id="p-0045" num="0044">The perturbing may be across a library of cells to thereby obtain RNA level and/or optionally protein level, whereby cell-to-cell circuit data at genomic or transcript or expression level is determined. The library of cells may comprise or is from a tissue sample. The tissue sample may comprise or is from a biopsy from a mammalian subject. The mammalian subject may comprise a human subject. The biopsy may be from a tumor. The method may further comprise reconstructing cell-to-cell circuits.</div>
</li> <li> <para-num num="[0045]"> </para-num> <div class="description-line" id="p-0046" num="0045">The method may comprise measuring open chromatin and may comprise fragmenting chromatin inside isolated intact nuclei from a cell, adding universal primers at cutting sites, and uniquely tagging DNA that originated from the cell.</div>
</li> <li> <para-num num="[0046]"> </para-num> <div class="description-line" id="p-0047" num="0046">The method may comprise measuring protein and RNA levels and may comprise CyTOF.</div>
</li> <li> <para-num num="[0047]"> </para-num> <div class="description-line" id="p-0048" num="0047">In another aspect, the present invention provides for a method of determining any combination of protein detection, RNA detection, open chromatin detection, protein-protein interactions, protein-RNA interactions, or protein-DNA interactions comprising any of the preceding methods.</div>
</li> <li> <para-num num="[0048]"> </para-num> <div class="description-line" id="p-0049" num="0048">In another aspect, the present invention provides for a method for screening compounds or agents capable of modifying a cellular network or circuit comprising performing any method as described herein, wherein perturbing further comprises exposing the cell to each compound or agent.</div>
</li> <li> <para-num num="[0049]"> </para-num> <div class="description-line" id="p-0050" num="0049">In another aspect, the present invention provides for a method of identifying a therapeutic, and to a therapeutic identified by the method described herein.</div>
</li> <li> <para-num num="[0050]"> </para-num> <div class="description-line" id="p-0051" num="0050">In another aspect, the present invention provides a method of reconstructing a cellular network or circuit, comprising introducing at least 1, 2, 3, 4 or more single-order or combinatorial perturbations to each cell in a population of cells; measuring genomic, genetic and/or phenotypic differences of each cell and coupling combinatorial peturbations with measured differences to infer intercellular and/or intracellular networks or circuits. The single-order or combinatorial perturbations can comprise 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22, 23, 24, 25, 26, 27, 28, 29, 30, 31, 32, 33, 34, 35, 36, 37, 38, 39, 40, 41, 42, 43, 44, 45, 46, 47, 48, 49, 50, 51, 52, 53, 54, 55, 56, 57, 58, 59, 60, 61, 62, 63, 64, 65, 66, 67, 68, 69, 70, 71, 72, 73, 74, 75, 76, 77, 78, 79, 80, 81, 82, 83, 84, 85, 86, 87, 88, 89, 90, 91, 92, 93, 94, 95, 96, 97, 98 or 99 or massively parallel perturbations. The perturbation(s) can comprise one or more genetic perturbation. The perturbation(s) can comprise one or more epigenetic or epigenomic perturbation. The perturbation can be introduced with RNAi- or a CRISPR-Cas system. For example, reference is also made to Dahlman et al., Nature Biotechnology (2015) doi:10.1038/nbt.3390 Published online 05 Oct. 2015 to allow efficient orthogonal genetic and epigenetic manipulation. Dahlman et al., Nature Biotechnology (2015) doi:10.1038/nbt.3390 have developed a CRISPR-based method that uses catalytically active Cas9 and distinct single guide (sgRNA) constructs to knock out and activate different genes in the same cell. These sgRNAs, with 14- to 15-bp target sequences and MS2 binding loops, can activate gene expression using an active Streptococcus pyogenes Cas9 nuclease, without inducing double-stranded breaks. Dahlman et al., Nature Biotechnology (2015) doi:10.1038/nbt.3390 use these ‘dead RNAs’ to perform orthogonal gene knockout and transcriptional activation in human cells.</div>
</li> <li> <para-num num="[0051]"> </para-num> <div class="description-line" id="p-0052" num="0051">The at least one perturbation can be introduced via a chemical agent, an intracellular spatial relationship between two or more cells, an increase or decrease of temperature, addition or subtraction of energy, electromagnetic energy, or ultrasound. The cell can comprise a cell in a model non-human organism, a model non-human mammal that expresses a Cas protein, a mouse that expresses a Cas protein, a cell in vivo or a cell ex vivo or a cell in vitro. The measuring or measured differences can comprise measuring or measured differences of DNA, RNA, protein or post translational modification; or measuring or measured differences of protein or post translational modification correlated to RNA and/or DNA level(s). The method can include sequencing, and prior to sequencing: perturbing and isolating a single cell with at least one labeling ligand specific for binding at one or more target RNA transcripts, or isolating a single cell with at least one labeling ligand specific for binding at one or more target RNA transcripts and perturbing the cell; and/or lysing the cell under conditions wherein the labeling ligand binds to the target RNA transcript(s).</div>
</li> <li> <para-num num="[0052]"> </para-num> <div class="description-line" id="p-0053" num="0052">The method in aspects of this invention may also include, prior to sequencing perturbing and isolating a single cell with at least one labeling ligand specific for binding at one or more target RNA transcripts, or isolating a single cell with at least one labeling ligand specific for binding at one or more target RNA transcripts and perturbing the cell; and lysing the cell under conditions wherein the labeling ligand binds to the target RNA transcript(s). The perturbing and isolating a single cell may be with at least one labeling ligand specific for binding at one or more target RNA transcripts. The isolating a single cell may be with at least one labeling ligand specific for binding at one or more target RNA transcripts and perturbing the cell.</div>
</li> <li> <para-num num="[0053]"> </para-num> <div class="description-line" id="p-0054" num="0053">The perturbing of the present invention may involve genetic perturbing, single-order genetic perturbations or combinatorial genetic perturbations. The perturbing may also involve gene knock-down, gene knock-out, gene activation, gene insertion or regulatory element deletion. The perturbation may be genome-wide perturbation. The perturbation may be performed by RNAi- or CRISPR-Cas-based perturbation, performed by pooled single or combinatorial CRISPR-Cas-based perturbation with a genome-wide library of sgRNAs or performing pooled combinatorial CRISPR-Cas-based perturbation with a genome-wide library of sgRNAs.</div>
</li> <li> <para-num num="[0054]"> </para-num> <div class="description-line" id="p-0055" num="0054">In another aspect, the methods described herein may be used for a diagnostic assay. In one embodiment, T cells are obtained from a subject and perturb-seq is performed on the cells. In another embodiment, T cells are obtained from a subject and gene expression of single cells is determined. Upon determining gene expression, perturb—seq is performed on a subset of genes differentially expressed. Perturb-seq can imform proper therapies to administer to a subject and can test many targets in a single experiment. In another embodiment, tumor cells are obtained from a subject. The tumor cells may also include cells of the tumor microenvironment, such as immune cells. The cells may be assayed for gene expression and differentially expressed genes can be assayed using the perturb-seq methods described herein. Not being bound by a theory, perturb—seq may allow assaying many targets and perturbations in a single experiement.</div>
</li> <li> <para-num num="[0055]"> </para-num> <div class="description-line" id="p-0056" num="0055">Accordingly, it is an object of the invention not to encompass within the invention any previously known product, process of making the product, or method of using the product such that Applicants reserve the right and hereby disclose a disclaimer of any previously known product, process, or method. It is further noted that the invention does not intend to encompass within the scope of the invention any product, process, or making of the product or method of using the product, which does not meet the written description and enablement requirements of the USPTO (35U.S.C. § 112, first paragraph) or the EPO (Article 83 of the EPC), such that Applicants reserve the right and hereby disclose a disclaimer of any previously described product, process of making the product, or method of using the product. It may be advantageous in the practice of the invention to be in compliance with Art. 53(c) EPC and Rule 28(b) and (c) EPC. All rights to explicitly disclaim any embodiments that are the subject of any granted patent(s) of applicant in the lineage of this application or in any other lineage or in any prior filed application of any third party is explicitly reserved Nothing herein is to be construed as a promise.</div>
</li> <li> <para-num num="[0056]"> </para-num> <div class="description-line" id="p-0057" num="0056">It is noted that in this disclosure and particularly in the claims and/or paragraphs, terms such as “comprises”, “comprised”, “comprising” and the like can have the meaning attributed to it in U.S. Patent law; e.g., they can mean “includes”, “included”, “including”, and the like; and that terms such as “consisting essentially of” and “consists essentially of” have the meaning ascribed to them in U.S. Patent law, e.g., they allow for elements not explicitly recited, but exclude elements that are found in the prior art or that affect a basic or novel characteristic of the invention.</div>
</li> <li> <para-num num="[0057]"> </para-num> <div class="description-line" id="p-0058" num="0057">These and other embodiments are disclosed or are obvious from and encompassed by, the following Detailed Description. For the purpose of clarity and a concise description features are described herein as part of the same or separate embodiments, however, it will be appreciated that the scope of the invention may include embodiments having combinations of all or some of the features described. The various embodiments as described herein for specific aspects of the invention, may be part of any aspect of the invention as described herein.</div>
</li> <description-of-drawings>
<heading id="h-0006">BRIEF DESCRIPTION OF THE DRAWINGS</heading>
<li> <para-num num="[0058]"> </para-num> <div class="description-line" id="p-0059" num="0058">The patent or application file contains at least one drawing executed in color. Copies of this patent or patent application publication with color drawing(s) will be provided by the Office upon request and payment of the necessary fee.</div>
</li> <li> <para-num num="[0059]"> </para-num> <div class="description-line" id="p-0060" num="0059">The following detailed description, given by way of example, but not intended to limit the invention solely to the specific embodiments described, may best be understood in conjunction with the accompanying drawings.</div>
</li> <li> <para-num num="[0060]"> </para-num> <div class="description-line" id="p-0061" num="0060"> <figref idrefs="DRAWINGS">FIG. 1</figref> illustrates a schematic overview of embodiments using hydrogel embedding of single cells, followed by lipid clearing and DNA-tagged antibody labeling. Also shown, are low-throughput and high-throughput readouts.</div>
</li> <li> <para-num num="[0061]"> </para-num> <div class="description-line" id="p-0062" num="0061"> <figref idrefs="DRAWINGS">FIG. 2A-C</figref> illustrates a proof of principle. A) Overlay of transmission light microscopy image epifluorescent measurement of CD51/Alexa488 (green). Black arrow indicates an example of a hydrogel embedded cell, the white arrow indicates an empty hydrogel droplet, staining negative for CD51. B) same cells, stained for genomic DNA DAPI (blue) and intracellular PCNA/Alexa647 (red). C) Strong Pearson correlation (0.98) measured between fluorescence levels of ‘endogenous’ GFP levels from 293/GFP cells, and a detecting anti-GFP antibody conjugated to Alexa647. Similar staining with the BD cytofix/perm kit yielded a Pearson correlation of 0.36.</div>
</li> <li> <para-num num="[0062]"> </para-num> <div class="description-line" id="p-0063" num="0062"> <figref idrefs="DRAWINGS">FIG. 3</figref> illustrates measuring protein levels by staining of aggregations of cellular constituents with high affinity reagents (antibodies) linked to an oligonucleotide with the structure [5′ Amino Modifier]-[˜6bp spacer]-[PhotoCleavable linker]-[˜4bp spacer]-[Illumina PCR primer]-[˜8-16bp UMI]-[˜21bp UCI]-[˜20bp universal ligation handle]. Note: UMI may be omitted in case of incorporation of a UMI in a split and pool index.</div>
</li> <li> <para-num num="[0063]"> </para-num> <div class="description-line" id="p-0064" num="0063"> <figref idrefs="DRAWINGS">FIG. 4</figref> illustrates hybridization of a ligation primer that binds to the universal ligation handle on oligonucleotide label with a sticky end needed for ligation of index A is produced.</div>
</li> <li> <para-num num="[0064]"> </para-num> <div class="description-line" id="p-0065" num="0064"> <figref idrefs="DRAWINGS">FIG. 5</figref> illustrates split-pool ligation using single-cell hydrogel drops as the basic unit and ligation of Index A, B and [C+PCR primer].</div>
</li> <li> <para-num num="[0065]"> </para-num> <div class="description-line" id="p-0066" num="0065"> <figref idrefs="DRAWINGS">FIG. 6</figref> illustrates staining in bulk with adjacent oligo's that hybridize to an RNA transcript or single guide RNA (sgRNA) at sites adjacent to each other.</div>
</li> <li> <para-num num="[0066]"> </para-num> <div class="description-line" id="p-0067" num="0066"> <figref idrefs="DRAWINGS">FIG. 7</figref> illustrates single probe detection of an RNA transcript or sgRNA using a single DNA probe that specifically binds to the target transcript.</div>
</li> <li> <para-num num="[0067]"> </para-num> <div class="description-line" id="p-0068" num="0067"> <figref idrefs="DRAWINGS">FIG. 8</figref> illustrates dual probe detection of an RNA transcript or sgRNA using adjacently binding probes that are ligated, such that only dually detection events are amplified.</div>
</li> <li> <para-num num="[0068]"> </para-num> <div class="description-line" id="p-0069" num="0068"> <figref idrefs="DRAWINGS">FIG. 9</figref> illustrates staining with the ligation primer and performing split-pool ligation with an Index A containing a UMI in index C such that sequencing starts with a random region (improves cluster detection) and ligation primer is no longer separately added, but pre hybridized before staining.</div>
</li> <li> <para-num num="[0069]"> </para-num> <div class="description-line" id="p-0070" num="0069"> <figref idrefs="DRAWINGS">FIG. 10</figref> illustrates an example of the generation of an Index A+UMI.</div>
</li> <li> <para-num num="[0070]"> </para-num> <div class="description-line" id="p-0071" num="0070"> <figref idrefs="DRAWINGS">FIG. 11</figref> illustrates measuring protein-protein complexes by performing a restriction enzyme digestion to generate an oligonucleotide containing two UCI and a compatible end for ligation to an index A for split-pool ligation.</div>
</li> <li> <para-num num="[0071]"> </para-num> <div class="description-line" id="p-0072" num="0071"> <figref idrefs="DRAWINGS">FIG. 12</figref> illustrates oligonucleotide structures for measuring protein-protein complexes. Oligo 1: [5′ Amino Modifier]-[˜6bp spacer]-[PhotoCleavable linker]-[˜4bp spacer]-[Illumina PCR primer]-[˜21bp UCI]-[˜11bp Hybridization sequence 1]. Oligo 2: [5′ Amino Modifier]-[˜6bp spacer]-[RE site for sticky overhang]-[˜21bp UCI]-[˜11bp Hybridization sequence 1 complement].</div>
</li> <li> <para-num num="[0072]"> </para-num> <div class="description-line" id="p-0073" num="0072"> <figref idrefs="DRAWINGS">FIG. 13</figref> illustrates measuring protein-RNA complexes using proximity hybridization. The Final oligonucleotide to sequence contains the UCI protein, UCI RNA and UMI+USI via split-pool ligation protocol.</div>
</li> <li> <para-num num="[0073]"> </para-num> <div class="description-line" id="p-0074" num="0073"> <figref idrefs="DRAWINGS">FIG. 14</figref> illustrates high throughput single-cell ATAC-seq.</div>
</li> <li> <para-num num="[0074]"> </para-num> <div class="description-line" id="p-0075" num="0074"> <figref idrefs="DRAWINGS">FIG. 15</figref> illustrates high throughput single-cell measuring protein-DNA complexes.</div>
</li> <li> <para-num num="[0075]"> </para-num> <div class="description-line" id="p-0076" num="0075"> <figref idrefs="DRAWINGS">FIG. 16</figref> illustrates staining with an antibody bound to an oligonucleotide label and performing split-pool ligation with an Index C containing a UMI in index C such that sequencing starts with a random region (improves cluster detection) and ligation primer is no longer separately added, but pre hybridized before staining.</div>
</li> <li> <para-num num="[0076]"> </para-num> <div class="description-line" id="p-0077" num="0076"> <figref idrefs="DRAWINGS">FIG. 17</figref> illustrates alternative embodiments of measuring RNA levels.</div>
</li> <li> <para-num num="[0077]"> </para-num> <div class="description-line" id="p-0078" num="0077"> <figref idrefs="DRAWINGS">FIG. 18</figref> illustrates the generation of an Index C+UMI.</div>
</li> <li> <para-num num="[0078]"> </para-num> <div class="description-line" id="p-0079" num="0078"> <figref idrefs="DRAWINGS">FIG. 19</figref> illustrates a brightfield microscopy image showing hydrogel droplet encapsulated cells with magnetic particles embedded into the droplets to enable magnetic separation, aiding in clean up and washing steps in multiple reactions. Greatly enhances automation and therefore throughput.</div>
</li> <li> <para-num num="[0079]"> </para-num> <div class="description-line" id="p-0080" num="0079"> <figref idrefs="DRAWINGS">FIG. 20</figref> illustrates a novel probe for detection of complexes consisting of more than 2 cellular constituents at the same time. The probe includes a Unique Location Identifier (ULI). It can be constructed by rolling circle amplification.</div>
</li> <li> <para-num num="[0080]"> </para-num> <div class="description-line" id="p-0081" num="0080"> <figref idrefs="DRAWINGS">FIG. 21</figref> illustrates the overall scheme to measure the proximity of 3 or more proteins, RNA or DNA molecules. The hybridization sequence of the ligand bound oligo binds to the complementary hybridization site on the ULI probe. By extension, each ligand bound oligo incorporates the same ULI. Restriction enzyme digestion generates a 4 bp overhang. Sticky end ligation is used to attach a USI+UMI.</div>
</li> <li> <para-num num="[0081]"> </para-num> <div class="description-line" id="p-0082" num="0081"> <figref idrefs="DRAWINGS">FIG. 22</figref> is an illustrative example of a Design Matrix according to aspects of the present invention.</div>
</li> <li> <para-num num="[0082]"> </para-num> <div class="description-line" id="p-0083" num="0082"> <figref idrefs="DRAWINGS">FIG. 23</figref> is an illustrative example of an Observed Measurement Matrix according to aspects of the present invention.</div>
</li> <li> <para-num num="[0083]"> </para-num> <div class="description-line" id="p-0084" num="0083"> <figref idrefs="DRAWINGS">FIG. 24</figref> is a plot of expected counts versus observed counts for a predetermined set of probes under a certain dilution criteria.</div>
</li> <li> <para-num num="[0084]"> </para-num> <div class="description-line" id="p-0085" num="0084"> <figref idrefs="DRAWINGS">FIG. 25</figref> is a plot of gene aggregations (observed versus expected) across random rows of a measurement vector for a 50 trial experiment.</div>
</li> <li> <para-num num="[0085]"> </para-num> <div class="description-line" id="p-0086" num="0085"> <figref idrefs="DRAWINGS">FIG. 26</figref> is plot of expected counts versus observed counts for a predetermined set of probes under a certain dilution criteria for Purified cDNA High.</div>
</li> <li> <para-num num="[0086]"> </para-num> <div class="description-line" id="p-0087" num="0086"> <figref idrefs="DRAWINGS">FIG. 27</figref> is plot of expected counts versus observed counts for a predetermined set of probes under a certain dilution criteria for unPurified cDNA High.</div>
</li> <li> <para-num num="[0087]"> </para-num> <div class="description-line" id="p-0088" num="0087"> <figref idrefs="DRAWINGS">FIG. 28</figref> is plot of expected counts versus observed counts for a predetermined set of probes under a certain dilution criteria for Purified cDNA Low.</div>
</li> <li> <para-num num="[0088]"> </para-num> <div class="description-line" id="p-0089" num="0088"> <figref idrefs="DRAWINGS">FIG. 29</figref> is plot of expected counts versus observed counts for a predetermined set of probes under a certain dilution criteria for unPurified cDNA Low.</div>
</li> <li> <para-num num="[0089]"> </para-num> <div class="description-line" id="p-0090" num="0089"> <figref idrefs="DRAWINGS">FIG. 30-33</figref> illustrate results of computational simulations where counts are aggregated across all genes from randomly chosen rows.</div>
</li> <li> <para-num num="[0090]"> </para-num> <div class="description-line" id="p-0091" num="0090"> <figref idrefs="DRAWINGS">FIG. 34</figref> illustrates an estimate number of cells (Y axis) needed to capture a given minimal number of cells per subtype (X axis) for BCs (rarest subtype ˜5%), AC, and RGCs (rarest subtype ˜1%).</div>
</li> <li> <para-num num="[0091]"> </para-num> <div class="description-line" id="p-0092" num="0091"> <figref idrefs="DRAWINGS">FIG. 35</figref> illustrates aDrop-Seq. From left: Device; clear separation of a mixture of mouse and human cells; and t-SNE of 44,808 single cells from a mouse retina, distinguishing 39 subpopulations (colors).</div>
</li> <li> <para-num num="[0092]"> </para-num> <div class="description-line" id="p-0093" num="0092"> <figref idrefs="DRAWINGS">FIG. 36</figref> illustrates a Drop-ATAC. Left: schematic. Right: Number of human (x-axis) vs. mouse (y-axis) reads in each barcode in a species cell mixing experiment.</div>
</li> <li> <para-num num="[0093]"> </para-num> <div class="description-line" id="p-0094" num="0093"> <figref idrefs="DRAWINGS">FIG. 37</figref> illustrates aSingle cell ATAC-Seq in DCs.</div>
</li> <li> <para-num num="[0094]"> </para-num> <div class="description-line" id="p-0095" num="0094"> <figref idrefs="DRAWINGS">FIG. 38</figref> illustrates aSingle cell protein+RNA by PEA. Top: PEA: Bottom: PEA distinguishes single THP-1 and K562 cells.</div>
</li> <li> <para-num num="[0095]"> </para-num> <div class="description-line" id="p-0096" num="0095"> <figref idrefs="DRAWINGS">FIG. 39</figref> illustrates a NGS proteomics in hydrogel droplets.</div>
</li> <li> <para-num num="[0096]"> </para-num> <div class="description-line" id="p-0097" num="0096"> <figref idrefs="DRAWINGS">FIG. 40</figref> illustrates a Rapid sequential FISH of 5 genes using five sequential rounds.</div>
</li> <li> <para-num num="[0097]"> </para-num> <div class="description-line" id="p-0098" num="0097"> <figref idrefs="DRAWINGS">FIG. 41</figref> illustrates a Linkage PCR in gel droplets of two CRISPR barcodes.</div>
</li> <li> <para-num num="[0098]"> </para-num> <div class="description-line" id="p-0099" num="0098"> <figref idrefs="DRAWINGS">FIG. 42</figref> illustrates a Proportion of sites (x-axis) where perturbation of bound TF (γ-axis) affects target's expression.</div>
</li> <li> <para-num num="[0099]"> </para-num> <div class="description-line" id="p-0100" num="0099"> <figref idrefs="DRAWINGS">FIG. 43</figref> illustrates a Perturbation model of Th17 cell differentiation.</div>
</li> <li> <para-num num="[0100]"> </para-num> <div class="description-line" id="p-0101" num="0100"> <figref idrefs="DRAWINGS">FIG. 44</figref> illustrates a Genetic (left) vs. molecular (right) models.</div>
</li> <li> <para-num num="[0101]"> </para-num> <div class="description-line" id="p-0102" num="0101"> <figref idrefs="DRAWINGS">FIG. 45</figref> illustrates a Hog network. Each set of incoming edges is quantified for each 1-5-way contribution (not shown).</div>
</li> <li> <para-num num="[0102]"> </para-num> <div class="description-line" id="p-0103" num="0102"> <figref idrefs="DRAWINGS">FIG. 46</figref> illustrates MCPP assays in Cas9-expressing DCs. Cells are infected with a library of sgRNA-expressing lentiviruses that target regulators; DCs are stimulated with LPS; finally, millions of cells are profiled by scRNA-seq or NGS proteomics to monitor changes in gene or protein expression by one or multiple sgRNAs per cell.</div>
</li> <li> <para-num num="[0103]"> </para-num> <div class="description-line" id="p-0104" num="0103"> <figref idrefs="DRAWINGS">FIG. 47</figref> illustrates a CRISPR screen. TLR pathway with screen hits marked (blue). Illustrative flow cytometry staining of Tnf for (shaded) vs. sgRNA controls (empty).</div>
</li> <li> <para-num num="[0104]"> </para-num> <div class="description-line" id="p-0105" num="0104"> <figref idrefs="DRAWINGS">FIG. 48A-G</figref> illustrates Perturb-seq: pooled approach to obtainscreening of transcriptional profiles of perturbations. (A) Overview of approach. From left: A complex pool of lentiviruses, each carrying a guide targeting a specific genes (A-E) in a vector including matched PAPIs is transduced at a given MOI (γ<sub>MOI</sub>) into cells that are subsequently selected (γ<sub>MOI</sub>) through growth and/or fluorescence. Individual cells are profilied with droplet scRNA-seq, to simultaneously recover their RNA profiles and PAPIs. (B) Perturb-Seq vector. A transcribed PAPI is pre-associated with a specific sgRNA, and enriched after WTA using dial-out PCR. (C) Perturb-Seq screens in this study. (D) The observed distribution of number of guides detected per cell (X axis) in BMDCs at 3h post-LPS stimulation. (E) Effect on target. Hif1a transcript expression (Y axis) in cells carrying a an sgRNA targeting Hif1a (sgHif1a-1, right) compared to all other cell (left). Box plots denote [describe all aspects: mean/median? Percentiles, outliers etc]. (F) Distribution of on-target effects in BMDC at 3h post-LPS. Shown is the distribution of on-target expression (X axis) in cells carrying the corresponding targeting guides (blue) and permuted results for a single permutation (green). Rectangle is the 99% confidence interval for the permuted mean. Mean on-target effect of individual guides are in tickmarks, including one outlier exceeding even the permuted data. (G) Linear modeling framework. Applicants fit the coefficients of a (regulatory) matrix (β) to best explain the observed expression profiles of each cell (matrix Y) given the covariates in the design matrix (X), including the observed sgRNAs in each cell and additional cell covariates. See also <figref idrefs="DRAWINGS">FIG. 55</figref>.</div>
</li> <li> <para-num num="[0105]"> </para-num> <div class="description-line" id="p-0106" num="0105"> <figref idrefs="DRAWINGS">FIG. 49A-J</figref> illustrates MIMOSCA: A linear model stratifiesanalysis framework to stratifiy the observed expression variation. (A) A linear modeling framework for continuous phenotypes. A linear model relates a continuous phenotype (gene expression, PC scores, etc; arrow) to a covariate (here guide identity). (B-D) Accounting for differences in cell complexity and state. Scatter plots show the relation for every cell (dot) between the expression levels of the highly expressed Ccl17 gene (B, Y axis) or its residual expression after a model is fit (C, D; Y axis) and the number of transcripts in the same cell (X axis; sum(log(transcripts detected)) in the original data (B), after including the quality measure as a covariate residuals (C) and after also including in addition cell state proportion (D). (E) Effect on cell states. Cartoon illustration of a hypothetical scenario where cells belong to either of two states (red, blue) and perturbation by sgRNA<sub>1 </sub>increases the proportion of cells in one state over the other. (F,G) Accounting for cell states. Shown is the effect on Ccl17 expression (Y axis) in cells with (right box) and without (left box) sg-Rela-3 in the original model (left) and when conditioned on cell state proportions (right). (H) Expectation maximization-like framework to separate effectively wild type cells within the set of cells containing a particular sgRNA. Left: The distribution of number of cells with the sgStat1-3 guide with a given fit (X axis) to the model of the effect of the Stat1-3 perturbation. Right: the distribution of inferred probabilities of an actual perturbation in the same cells. (I) Contribution of each component in the model (Y axis) to the % variance explained (X axis) based on the r<sup>2 </sup>values from cross-validation. (J) Correlation matrix between genes in the residuals of the model. The addition of our covariates reduces the residual structure in the data. See also <figref idrefs="DRAWINGS">FIG. 56</figref>.</div>
</li> <li> <para-num num="[0106]"> </para-num> <div class="description-line" id="p-0107" num="0106"> <figref idrefs="DRAWINGS">FIG. 50A-F</figref> illustrates the analysis of the role of 24 TFs in the response of BMDCs to LPS. (A) Key TF modules in BMDCs at 3h post-LPS. Heat map shows the Pearson correlation (red-blue colorbar) between the coefficients of the regulatory matrix β for every pair of guides (rows, columns), based on a model that does not include the cell state covariates. Yellow rectangles: four key TF modules high (members marked on right). Leftmost column: effect of the guide on-target. (B) Agreement between guides targeting the same gene. Shown is the distribution of correlations between guides targeting the same gene (grey) or different genes (blue). (C) Cell states. Shown are enrichments (−log 10(P-value) for induced (red) and repressed (blue) genes) for GO gene sets (rows) in each of seven cell states (columns) defined for genetically unperturbed BMDC at 3 h post-LPS. Key terms are marked on right. (D) TFs group by their effects on cell state proportion. Show are the Z-score for enrichment (red) or depletion (blue) of guides in cells with each of seven states (columns; as in C). (E, F) Accounting for cell states emphasizes TF-specific effects. (E) Heat map shows the Pearson correlation (red-blue colorbar) between the coefficients of the regulatory matrix β for every pair of guides (rows, columns), based on a model that accounts for cell states as covariates. Yellow rectangles: specific modules all with guides to a single gene (gene marked on right). Leftmost column: effect of the guide on-target. (F) The distribution of correlations between guides targeting the same gene (grey) or different genes (blue), with a model including cell state covariates. See also <figref idrefs="DRAWINGS">FIG. 57</figref>.</div>
</li> <li> <para-num num="[0107]"> </para-num> <div class="description-line" id="p-0108" num="0107"> <figref idrefs="DRAWINGS">FIG. 51A-G</figref> illustrates Perturb-Seq recovers a gene regulatory circuit for BMDCs balancing cell states and immune responses. (A,B) Four TF modules control five transcriptional programs in BMDCs at 3 h post-LPS. (A) Shown is the regulatory coefficient of each guide (labeled and color coded columns) on each gene (rows) based on a model that does not account for cell states as co-variates. Guides and genes are clustered by the similarity of their profiles. Four TF modules (M1-4) and five target programs (P1-5) are marked. Green-white heatmap shows the enrichment in bound targets of each TF based on ChIP-Seq (Garber et al., 2012) in each program (rows). (B) Bi-partite graph, based on (A) of association of TF modules (top) to target programs (bottom). Blue (red) arrows: TF in module activates (inhibits) program (perturbation has the opposite effect). Bottom: TFs from modules that are members of the regulated program (blue: activator of the program; red: repressor of the program). (C,D) TF-TF circuit in BMDCs at 3h post-LPS. (C) Heatmap as in (A) but only showing genes (rows) that encode the TFs targeted by the guides. (D) TF-TF circuit based on the associations in (C). TFs are shown as nodes. TF modules are shadowed in grey. Blue (red) arrows: TF in module activates (inhibits) program (perturbation has the opposite effect). (E-F) ChIP-Seq support for genetic network. (E) Expected effects of perturbation of a TF (green) on a target gene when the factor does not regulate (or bind) the gene (top); directly binds and represses it (middle) or directly binds and activates it (bottom). (F) Regulatory effect of each guide (rows) on its bound target (colorbar) as defined by ChIP-Seq in BMDCs at four time points (columns, 0, 0.5, 1, and 2 h post-LPS). (G) The proportion of bound targets at 120 min post-LPS for the TF targeted by each guide (rows) that are repressed (blue), activated (red) or unaffected (grey) by the perturbation. Asterisks denote significant enrichments (as in F). See also <figref idrefs="DRAWINGS">FIG. 58</figref>.</div>
</li> <li> <para-num num="[0108]"> </para-num> <div class="description-line" id="p-0109" num="0108"> <figref idrefs="DRAWINGS">FIG. 52A-F</figref> illustrates dissecting genetic interactions between perturbations using Perturb-seq. (A) Extending the linear model framework to include higher order polynomial functions as the product of single covariates. (B) Interaction analysis on cell states for BMDCs at 3 h post-LPS. Heatmap shows the enrichment (red) or depletion (blue) of single, pair and triplets of guides (rows) in cells in each of seven states (as in <figref idrefs="DRAWINGS">FIG. 50C</figref>). (C) A three-way genetic interaction enhances cell state 2. Box plot shows the distribution of probabilities of assignment to cell state 2 marginally contributed by the individual perturbation of each of NFKB1, Stat2, and Rel, their pair wise interaction and three-way interaction. (D) Genetic interaction categories. A schematic illustrating the 27 possible schemes of genetic interaction between two genes (A,B), when considering all permutations of significantly positive (red), negative (blue) or no (white) regulatory coefficients marginally associated with each of the two individual guides or their combination. The four key categories of relationships (buffering, synergistic, dominant, and additive) are color coded on left. (E) The distribution genes in each of the 27 categories (rows) for every pair of perturbations assayed for interaction (colunms). Key examples are marked on bottom. (F) Genetic interaction between Rela and Nfkb1 associated with co-binding. Shown are the marginal regulatory coefficients for Rela, Nfkb 1 and their interaction term for each gene (rows) with at least one non-zero coefficient, sorted by the four key categories (color code, left). ChIP-Seq enrichment of individually bound and co-bound targets is shown for each of the groups (right). See also <figref idrefs="DRAWINGS">FIG. 59</figref>.</div>
</li> <li> <para-num num="[0109]"> </para-num> <div class="description-line" id="p-0110" num="0109"> <figref idrefs="DRAWINGS">FIG. 53A-I</figref> illustrates the analysis of the role of 10 non-essential TFs and 14 cell cycle regulators in K562 cells. (A,B) Grouping of TFs by their regulatory coefficients. Shown are the Pearson correlation matrix of the coefficients of the regulatory matrix β of the TFs without including the cell state covariates (A) and after including them (B). (C) Cell states in genetically unperturbed K562 cells. Shown are enrichments (−log 10(P-value) for induced (red) and repressed (blue) genes) for GO gene sets (rows) in each of seven cell states (columns) defined for genetically unperturbed BMIDC at 3h post-LPS. Key terms are marked on right. (D) TFs effects on cell state proportions. Shown are the Z-score for enrichment (red) or depletion (blue) of guides in cells in each of seven states (columns; as in C). (E,F) Agreement of guide effects across time points. Violin plots show the distribution of correlations between guides targeting the same gene (grey), different genes (blue) and a gene and an intergenic region (red) with each of two time points (T1=7d, T2=14d) and across the two time points, in either a model that does not (E) or does (F) include cell state covariates. (G) Pearson correlation matrix of the coefficients of the regulatory matrix β of the cell cycle regulators without including the cell state covariates. (H) The regulatory effect (color bar, defined by the average regulatory coefficients in our model) or the guides targeting each cell cycle gene (rows) on the genes associated with each of six signatures of phases of the cell cycle and apoptosis (columns). (I) Distribution of fitness effects (X axis) across the guides targeting each gene (Y axis) by comparing the number of cells with the guide to its abundance in the initial pool. See also <figref idrefs="DRAWINGS">FIG. 60</figref>.</div>
</li> <li> <para-num num="[0110]"> </para-num> <div class="description-line" id="p-0111" num="0110"> <figref idrefs="DRAWINGS">FIG. 54A-D</figref> illustrates a power analysis and prospects for Perturb-seq. (A,B) Saturation analysis. Shown is the effect of the number of cells (Y axis) and reads (X axis) on our ability to recover a given level of correlation (color bar) with either the per gene transcriptional defects (A) or gene signature effects (e.g., cell type proportions) (B) Applicants observed with our full data. (C) The tradespace of number of cells (X axis) and measurements per cell (Y axis) required for screening scale transcriptome measurements. (D) Extensions of Perturb-Seq. The perturb seq framework can be extended by scaling the number of cells (arrow, left) or by incorporating additional types of cell covariates, such as lineage, marker expression, or temporal tracers.</div>
</li> <li> <para-num num="[0111]"> </para-num> <div class="description-line" id="p-0112" num="0111"> <figref idrefs="DRAWINGS">FIG. 55A-L</figref> illustrates the performance of Perturb-seq (A-E) Log-likelihood functions as a function of detection probability (X axis) and MOI (Y axis) for our zero-truncated zero inflated Poisson distribution for the indicated experiment (label on top). (A-C) Line plots are cumulative distributions for the observed distribution of guides per cell (blue) and the expectation from the maximum likelihood estimate (green). (F) Specificity of PAPI detection. Scatter plot shows the percentage of reads for a PAPI within a given cell (CBC-<b>13</b> cell barcode) (Y axis) as a function of the log 2(reads) it received in that cell (X axis). (G) Effect on target. Cebpb transcript expression (Y axis) in cells carrying an sgRNA targeting Cebpb (sgCebpb_1, right) compared to all other cell (left). Box plots. (I) Relationship between overall mean expression of the on-target gene (X axis) and the observed effect on its expression (Y axis) by the guides that target it, in BMDCs at 3 h post-LPS. (J,K) Relationship between population expression measurements and 10 cell average (top) and 100 cell average for BMDCs (J) and K562 cells (H). (L) Relationship between transcript length (X axis) and the difference between population expression and single cell average expression (Y axis).</div>
</li> <li> <para-num num="[0112]"> </para-num> <div class="description-line" id="p-0113" num="0112"> <figref idrefs="DRAWINGS">FIG. 56A-I</figref> illustrates the performance of MIMOSCA framework. (A-C) Contribution of each component in the model (Y axis) to the % variance explained (X axis) based on the r<sup>2 </sup>values from cross-validation in each of 3 screens (labeled on top). (D-F) Significance of regulatory coefficients. Shown are the distributions of signed logio(FDR) for each of three sgRNAs (marked on top). Values capped at 3; Zero coefficients (due to shrinkage by regularization) have no assigned FDR. (G) Relationship between number of cells/sgRNA (X axis) and number of significant genes for each sgRNA (Y axis). One outlier guide (sgStat1_2) is marked and—while included in all subsequent plots—was not considered in subsequent biological analysis. (H) Computational flowchart for MIMOSCA. Raw input data (orange) is processed into intermediate data types that require little computation (purple) or a more intensive processing (green). Finally, output data (blue) is generated and interpreted biologically. (I) Significant guide effects. Scatter plots show conceptual (left) and real (middle and right) examples of the sum of difference in the sum of the squared residuals between a model fit including a guide covariate and a model not including the guide covariate for positive and negative values. Each dot is a cell. Conceptually (left column), a pure population of perturbed cells that is well separated from the other cells should have few genes that are better fit without the guide covariate (left column, top and middle), whereas a poor performing guide (or a well-performing control) should have few such genes (left column, bottom panel). A guide targeting Rela in BMDC (3h post-LPS) (top middle and top right) has a distinguishable effect compared to a non-targeting guide (below) in a model without cell state covariates (middle) and even when considering cell states (right). Bottom middle and right plots show the mean deviation of each guide (X axis). Red line: the non-targeting control.</div>
</li> <li> <para-num num="[0113]"> </para-num> <div class="description-line" id="p-0114" num="0113"> <figref idrefs="DRAWINGS">FIG. 57A-F</figref> illustrates analysis analyses of the role of 24 TFs in BMDCs. (A) Cell states in BMDCs pre-stimualtion. Shown are enrichments (−log 10(P-value) for induced (red) and repressed (blue) genes) for GO gene sets (rows) in each of four cell states (columns) defined for genetically unperturbed BMDC at 0 h (pre-stimulation). Key terms are marked on right. (B) TFs group by their effects on cell state proportions pre-stimualation. Shown are the Z-score for enrichment (red) or depletion (blue) of guides in cells with each of four states (columns; as in A). (C) Distinct effects pre- and post-stimulation. Boxplots comparing the correlations of regulatory coefficients between guides targeting the same genes within the 3 hr LPS stimulated cells (light blue) or within unstimulated cells (white), and across the two conditions either without modeling cell state co-variates (dark grey) or with modeling cell states (blue). (D) Distribtuion of number of detected transcripts per cell (X axis) for cells harboring guides targeting distinct genes (Y axis). (E) Relationship between the relative abundance of an sgRNA in the input plasmid pool (X axis) and the relative amount of cells with that sgRNA (Y axis). (F) Assessing potential fitness effects. Shown is the distribution of fold changes of sgRNA abundance compared to the input abundance (X axis) for the guides (dots) targeting each gene (Y axis).</div>
</li> <li> <para-num num="[0114]"> </para-num> <div class="description-line" id="p-0115" num="0114"> <figref idrefs="DRAWINGS">FIG. 58A-D</figref> illustrates additional aspects of the regulatory circuitry of BMDCs. (A) Relation between cell states and regulatory programs. Shown is the significance (−log 10(P-value) of the hypergeometric test) of the overlap between the genes in each of the programs P1-P5 (as in <figref idrefs="DRAWINGS">FIG. 51A</figref>) and the genes induced in each of the seven transcriptional states of genetically unperturbed DCs at 3h post-LPS (as in <figref idrefs="DRAWINGS">FIG. 50C</figref>). (B) TF control of transcriptional programs in BMDCs at pre-stimulation (Oh). Shown is the regulatory coefficient of each guide (labeled columns) on each gene (rows) based on a model that does not account for cell states as co-variates. Guides and genes are clustered by the similarity of their profiles. Four target programs (P1-4) are marked. Green-white heatmap shows the enrichment in bound targets of each TF based on ChIP-Seq (Garber et al., 2012) in each program (rows) based on either a lenient, genome-wide background (top) or a strict background (bottom) restricted only to the genes in the four programs. (C) Enrichment in bound genes based on a lenient background. Green-white heatmap shows the enrichment in bound targets of each TF based on ChIP-Seq (Garber et al., 2012) in each program (rows) based on either a lenient, genome-wide background for the same model as in <figref idrefs="DRAWINGS">FIG. 51</figref>. (D) Relation between TFs and transcriptional programs in BMDCs at 3 h post-LPS after accounting for cell states. Shown is the regulatory coefficient of each guide (labeled columns) on each gene (rows) based on a model that accounts for cell states as co-variates. Guides and genes are clustered by the similarity of their profiles. Five target programs (P1-5) are marked (distinct from the programs of <figref idrefs="DRAWINGS">FIG. 51A</figref>).</div>
</li> <li> <para-num num="[0115]"> </para-num> <div class="description-line" id="p-0116" num="0115"> <figref idrefs="DRAWINGS">FIG. 59</figref> illustrates the genetic interactions for K562 TFs.</div>
</li> <li> <para-num num="[0116]"> </para-num> <div class="description-line" id="p-0117" num="0116"> <figref idrefs="DRAWINGS">FIG. 60A-D</figref> illustrates additional anlaysis of the role TFs and cell cycle regulators in K562 cells. (A) Assessing potential fitness effects of TF perturbations in K562 cells. Shown is the distribution of fold changes of sgRNA abundance compared to the input abundance (X axis) for the guides (dots) targeting each gene (Y axis). (B) TF control of transcriptional programs in K562 cells. Shown is the regulatory coefficient of each guide (labeled columns) on each gene (rows) based on a model that either does not (B) or does (C) account for cell states as co-variates. Guides and genes are clustered by the similarity of their profiles. Target programs are marked with key enriched annotations. Green-white heatmap shows the enrichment in bound targets of each TF based on ChIP-Seq in each program (rows) based on either a lenient, genome-wide background (top) or a strict background (bottom) restricted only to the genes in the programs. (C) Cell cycle genes in our screen. Shown is the cell biological classification of our genes (Figure reproduced from Neumann et al). (D) Effects of perturbing cell cycle genes on transcriptional programs in K562 cells. Plot as in (B) but for the cell cycle regulators.</div>
</li> <li> <para-num num="[0117]"> </para-num> <div class="description-line" id="p-0118" num="0117"> <figref idrefs="DRAWINGS">FIG. 61A-L</figref> illustrates a saturation analysis for differential expression, related to <figref idrefs="DRAWINGS">FIG. 54</figref>. (A) Theoretical probability of having a successful perturbation in every target as a function of the number of perturbations (1-p<sup>n</sup>), assuming independence. (B-L) Saturation analysis. Shown is the effect of the fraction of cells (Y axis) and reads (X axis) on our ability to recover a given level of correlation (color bar) with either the PCA scores (B-D), gene signature effects (E-H) or per gene transcriptional defects (I-L) based on the BMDC 3h stimulated screen data. The total number of cells (1.0) is, on average, 300 cells/perturbation and the total number of transcripts per cell is, on average 5,000. PCA scores (as the expression matrix Y) were obtained by projecting the data onto PCs defined by the wildtype cells for the top 11 PCs conditioned on different effect sizes. Effect size units are arbitrary. For gene signatures, effects sizes are in units of average log<sub>2</sub>(UMI) across the gene signature. For an individual gene level, effect sizes are in units of log<sub>2</sub>(UMI).</div>
</li> <li> <para-num num="[0118]"> </para-num> <div class="description-line" id="p-0119" num="0118"> <figref idrefs="DRAWINGS">FIG. 62</figref> illustrates Perturb-Seq analysis of the role of TFs in the response of BMDCs to LPS.</div>
</li> <li> <para-num num="[0119]"> </para-num> <div class="description-line" id="p-0120" num="0119"> <figref idrefs="DRAWINGS">FIG. 63A-D</figref> illustrates Perturb-Seq analysis of the role of cell cycle regulators in K562 cells. (A) Regulation of apoptosis [“virtual FACS”: control, all guides, selected example. (B) Individual and synthetic effects on cell fitness (Livnat model). (C) Individual and synthetic effects on G1/S and G2/M cell states (“virtual FACS” plus Livant model). (D) Transcriptional patterns underlying effect on cell states.</div>
</li> <li> <para-num num="[0120]"> </para-num> <div class="description-line" id="p-0121" num="0120"> <figref idrefs="DRAWINGS">FIG. 64</figref> illustrates Perturb-Seq analysis of the role of non-essential TFs in K562 cells.</div>
</li> <li> <para-num num="[0121]"> </para-num> <div class="description-line" id="p-0122" num="0121"> <figref idrefs="DRAWINGS">FIG. 65</figref> illustrates the prospects for Perturb-Seq.</div>
</li> <li> <para-num num="[0122]"> </para-num> <div class="description-line" id="p-0123" num="0122"> <figref idrefs="DRAWINGS">FIG. 66</figref> illustrates GO terms associated with the following sgRNAs.</div>
</li> <li> <para-num num="[0123]"> </para-num> <div class="description-line" id="p-0124" num="0123"> <figref idrefs="DRAWINGS">FIG. 67</figref> illustrates Gene set (GO/MSigDB) analysis.</div>
</li> <li> <para-num num="[0124]"> </para-num> <div class="description-line" id="p-0125" num="0124"> <figref idrefs="DRAWINGS">FIG. 68</figref> illustrates a graphical embodiment of a perturb-Seq experiment.</div>
</li> <li> <para-num num="[0125]"> </para-num> <div class="description-line" id="p-0126" num="0125"> <figref idrefs="DRAWINGS">FIG. 69</figref> illustrates RT primers that are used to tag the 3′ ends of transcripts.</div>
</li> <li> <para-num num="[0126]"> </para-num> <div class="description-line" id="p-0127" num="0126"> <figref idrefs="DRAWINGS">FIG. 70</figref> illustrates a vector for introducing a guide RNA and the actual transcript insert after capture.</div>
</li> <li> <para-num num="[0127]"> </para-num> <div class="description-line" id="p-0128" num="0127"> <figref idrefs="DRAWINGS">FIG. 71</figref> illustrates a graphical embodiment of a perturb-Seq experiment.</div>
</li> <li> <para-num num="[0128]"> </para-num> <div class="description-line" id="p-0129" num="0128"> <figref idrefs="DRAWINGS">FIG. 72</figref> illustrates that capture primers contain two barcodes specifying cell and molecule identity. Recover guide information is measured by co-expressing a polyadenylated guide barcode from a guide RNA vector.</div>
</li> <li> <para-num num="[0129]"> </para-num> <div class="description-line" id="p-0130" num="0129"> <figref idrefs="DRAWINGS">FIG. 73</figref> illustrates read statistics per cell.</div>
</li> <li> <para-num num="[0130]"> </para-num> <div class="description-line" id="p-0131" num="0130"> <figref idrefs="DRAWINGS">FIG. 74</figref> illustrates guide identification.</div>
</li> <li> <para-num num="[0131]"> </para-num> <div class="description-line" id="p-0132" num="0131"> <figref idrefs="DRAWINGS">FIG. 75</figref> illustrates capture of the guide barcode transcripts during RNAseq and specific amplification.</div>
</li> <li> <para-num num="[0132]"> </para-num> <div class="description-line" id="p-0133" num="0132"> <figref idrefs="DRAWINGS">FIG. 76</figref> illustrates singles, doubles and triplets during droplet sequencing.</div>
</li> <li> <para-num num="[0133]"> </para-num> <div class="description-line" id="p-0134" num="0133"> <figref idrefs="DRAWINGS">FIG. 77</figref> illustrates a graphical embodiment of a perturb-Seq experiment.</div>
</li> <li> <para-num num="[0134]"> </para-num> <div class="description-line" id="p-0135" num="0134"> <figref idrefs="DRAWINGS">FIG. 78</figref> illustrates a triple guide Drop-Seq vector.</div>
</li> <li> <para-num num="[0135]"> </para-num> <div class="description-line" id="p-0136" num="0135"> <figref idrefs="DRAWINGS">FIG. 79</figref> illustrates a triple guide Drop-Seq vector.</div>
</li> <li> <para-num num="[0136]"> </para-num> <div class="description-line" id="p-0137" num="0136"> <figref idrefs="DRAWINGS">FIG. 80</figref> illustrates a comparison of methods for single cell RNA-seq.</div>
</li> <li> <para-num num="[0137]"> </para-num> <div class="description-line" id="p-0138" num="0137"> <figref idrefs="DRAWINGS">FIG. 81A-F</figref> A robust strategy for systematic genetic modifier screens using single cell expression profiling. (A) Schematic of the perturb-seq platform. (1) Cells transduced with a complex pool of CRISPRi guide RNAs (2) are encapsulated with cell lysis buffer and gel beads, which deliver DNA oligos, in droplets using the Chromium™ instrument (10X Genomics). DNA oligos encode cell barcodes (unique to each bead), unique molecular identifiers or UMIs (unique to each bead oligo), and an oligo-dT homopolymer region. (3) After capture, beads dissolve and released oligos prime cellular mRNAs for cDNA synthesis within droplets. (4-6) The cDNA libraries are prepared for deep sequencing in pooled format, and transcripts of particular interest are separately enriched by specific amplification. (B) Method for introducing and identifying perturbations using CRISPR guide RNAs. A lentiviral vector designed for single copy integration into cells expressing Cas9 or dCas9 fused to effector domains (for CRISPRi/a) carries two expression units: (1) a polymerase III-driven sgRNA and (2) a guide barcode selection cassette (top) in reverse orientation. The later transcript can be selectively amplified during library preparation (bottom). (C) Performance of guide barcode capture by specific amplification. The plot shows the distribution of guide barcode UMIs counted per cell in a pilot experiment of ˜5,700 cells containing different perturbations. (D) Performance of perturbation identification. The high coverage of the guide barcode enables efficient identification of perturbation identity, and confident rejection of cell doublets (apparent as multiple guide barcodes attached to a single cell barcode). (E) Characterization of perturb-seq vector by GFP knockdown. GFP+ K562 cells with dCas9-KRAB were transduced with either the perturb-seq vector or the original CRISPRi vector expressing a GFP-targeting sgRNA or a perturb-seq vector expressing a negative control sgRNA. GFP levels were measured by flow cytometry 11 d after transduction. Plotted are kernel density estimates of normalized flow cytometry counts for infected (BFP+) cells. (F) Characterization of perturb-seq vector in pooled format via single-cell RNA-seq. Using identities inferred from the capture of guide barcodes, Applicants computationally separate our pilot perturb-seq experiment into subpopulations of cells containing each sgRNA. The single-cell RNA-seq profiles allow us to assess average knockdown of the gene targeted within each subpopulation (relative to cells containing a control sgRNA). The plot shows three examples.</div>
</li> <li> <para-num num="[0138]"> </para-num> <div class="description-line" id="p-0139" num="0138"> <figref idrefs="DRAWINGS">FIG. 82A-G</figref> Strategy for multiplexed delivery of CRISPR guide RNAs in a single expression vector. (A) Schematic of the unfolded protein response. (B) Schematic of three-guide vector. Three guide RNA expression cassettes (a U6 promoter, a sgRNA targeting sequence, and a sgRNA constant region (cr)) are fused to express the three sgRNAs. The lentiviral backbone is the same as that of the perturb-seq vector. (C) Characterization of initial three-guide vector by GFP knockdown. GFP+ K562 cells with UCOE-dCas9-KRAB were transduced with either the single perturb-seq vector expressing a GFP-targeting sgRNA, an initial three-guide vector expressing a GFP-targeting sgRNA from the hU6 promoter, or a single vector expressing a negative control sgRNA. Cells were selected to purity with puromycin and GFP levels measured by flow cytometry 7 d after transduction. Plotted are kernel density estimates of normalized flow cytometry counts. (D) Design and characterization of constant region variants. Top: Schematic representation of sgRNA constant region, with location and description of changes indicated. Changes that were combined to generate cr2 and cr3 are labeled in orange and purple, respectively. Bottom: RFP levels of <i>E. coli </i>CRISPRi reporter strain expressing a mRFP-targeting sgRNA and the denoted constant region variant. RFP levels, determined by flow cytometry, are plotted as mean fluorescence of three replicates relative to a strain expressing a negative control sgRNA. Error bars denote standard deviations. (E) Characterization of new sgRNA expression cassettes by GFP knockdown. GFP+ K562 cells with dCas9-KRAB were transduced with different constructs expressing a GFP-targeting sgRNA from different U6 promoters and with different constant regions, as indicated. GFP levels were measured by flow cytometry 10 d after transduction. Plotted are kernel density estimates of normalized flow cytometry counts for infected (BFP+) cells. (F) Characterization of final three-guide vector by GFP knockdown. GFP+ K562 cells with UCOE-dCas9-KRAB were transduced with three-guide vectors expressing a GFP-targeting sgRNA from the position indicated in parentheses and two different negative control sgRNAs from the other two positions or a three-guide vector expressing three negative control sgRNAs. Cells were treated as described in (B). Plotted are kernel density estimates of normalized flow cytometry counts. Plot for the single perturb-seq construct is the same as in panel (B). (G) Characterization of the three-guide vector via perturb-seq. Using the three-guide vector, Applicants simultaneously targeted the three main UPR sensors and determined the average reduction in expression (relative to cells containing three negative control guides) via perturb-seq. Each target showed &gt;80% depletion, and depletion was stable across two chemical treatments that induce the UPR. See also <figref idrefs="DRAWINGS">FIG. 88</figref>.</div>
</li> <li> <para-num num="[0139]"> </para-num> <div class="description-line" id="p-0140" num="0139"> <figref idrefs="DRAWINGS">FIG. 83A-F</figref> Epistatic analysis of the three transcriptional arms of the unfolded proteins response using perturb-seq (A) Schematic of perturb-seq epistasis experiment. (B) Unbiased identification and decoupling of single-cell behaviors via low rank independent component analysis. Individual single-cell RNA-seq profiles are noisy, but shared regulation and correlated gene expression mean that patterns exist within the population. To identify these patterns, Applicants computationally construct a low-dimensional approximation of gene expression within the population to remove noise, and then use ICA to identify a small number of independent programs of gene expression. The figure shows this approach applied to our combinatorial knockdowns treated with thapsigargin. Low rank ICA identifies distinct sets of components that vary either across perturbation or across the cell cycle. When the first set of components are projected via t-sne, the cells (each dot) group via perturbation. Conversely when Applicants project the second set, the cells arrange in a circular pattern by cell cycle position. The “bulge” highlighted by the dashed line is enriched for cells that have PERK active. (C) Identification of a PERK- and cell-cycle-dependent subpopulation in thapsigargin-treated cells. The plots show t-sne projections of control (+DMSO) and thapsigargin-treated cells with or without PERK. Low rank ICA identifies a component (IC) that is bimodal within each perturbation subpopulation and marks G1 cells, and that is particularly disenriched in G1 cells within the thapsigargin-treated subpopulation. (D) Cell cycle composition of +DMSO, +Tg, and +Tg cells with PERK depleted. (E) Genetic interactions between G1 and PERK activation. Applicants split each perturbation subpopulation into G1 and non-G1 cells based on the value of IC, and constructed average expression profiles in each condition. Applicants then examined how the 50 genes that most influenced IC varied, exposing both synergistic and antagonistic interactions between progression through G1 and PERK activation. (F) Epistatic interactions among the three branches of the UPR. Applicants isolated 104 genes that strongly varied in our large-scale perturb-seq experiment, and clustered them by their expression pattern within the population. The heatmap shows average expression profiles for each perturbation and chemical treatment. Patterns of induction determine the branch specificity of each gene. The bottom panel shows an unbiased decomposition of the total response into three components obtained via ICA, showing that many ATF6/IRE1α targets have some overlapping regulation. See also <figref idrefs="DRAWINGS">FIG. 89</figref>.</div>
</li> <li> <para-num num="[0140]"> </para-num> <div class="description-line" id="p-0141" num="0140"> <figref idrefs="DRAWINGS">FIG. 84A-G</figref> Genome-scale CRISPRi screening for genetic stresses that perturb the IRE1 branch of the unfolded protein response. (A) Schematic of UPRE (mCh) and constitutive EF1a (GFP) reporter cassettes. (B) K562 reporter cells (cBA011) were transduced with the indicated sgRNAs and treated with 2 μg/mL tunicamycin or DMSO after 4 days. Approximately, 12 hours later these cells were evaluated by flow cytometry. Data is representative of two independent experiments. (C) Schematic of sgRNA screen. K562 cells stably expressing the mCh/GFP reporter cassettes and a dCas9-BFP-KRAB fusion protein were transduced with pooled genome-scale hCRISPRi libraries. Transduced cells were selected, sorted for high and low mCh/GFP ratio, and processed for sequencing of the sgRNA-containing DNA cassettes. (D) Volcano plots of UPRE reporter gene phenotypes and p-values from hCRISPRi-v2 reporter screen. Data generated from negative control sgRNAs are indicated in gray. Screen hits and select genes are indicated in pink and red, respectively. (E) Gene UPRE reporter phenotypes from replicates of hCRISPRi-v2 screen in K562 cells (cBA011). Phenotypes generated from negative control sgRNAs are indicated in gray. (F) UPRE reporter gene phenotypes from hCRISPRi-v2 reporter screen by functional category. Red indicates screen hits. (G) Comparison of mCherry UPRE reporter signal to EF 1a driven GFP in cBA011 cells transduced with 257 sgRNAs targeting 152 hit genes from the hCRISPRi-v2 screen and 3 distinct negative controls. Data represent log2 averages of median fluorescence signals (normalized to untransduced cells) across four possible experiments (n=2-7 technical or experimental replicates), normalized medians from all four experiments are included in the log2 averages for each of the control sgRNAs. See also <figref idrefs="DRAWINGS">FIG. 90</figref>.</div>
</li> <li> <para-num num="[0141]"> </para-num> <div class="description-line" id="p-0142" num="0141"> <figref idrefs="DRAWINGS">FIG. 85A-D</figref> A large-scale perturb-seq experiment interrogating ER homeostasis (A) Functional clustering of hits from perturb-seq analysis of ER homeostasis. Applicants picked ˜100 guides from our genome-wide screen and subjected them to perturb-seq analysis, totaling ˜65,000 single cells. Average expression profiles were created from all cells bearing guides targeting the same gene and hierarchically clustered. The figure shows a heatmap of correlations between expression profiles for all perturbations along with functional annotations. (B) Cell cycle analysis of perturb-seq hits. For each perturbation, the fraction of cells in each cell cycle stage was computationally identified. The figure shows the change in composition induced by each perturbation relative to control cells (containing the NegCtrl-2 guide). (C) Target knockdown efficiency. Average depletion of the sgRNA target was assessed within each subpopulation. Genes targeted by multiple guides have multiple possibly overlapping dots. Error bars are 95% confidence intervals estimated by bootstrapping. (D) Average phenotypes of perturb-seq hits. The UPRE score from the primary genome-wide screen is compared to three computationally derived scores measuring activation of the three branches of the UPR for each perturbation. The final panel is the logio number of genes differentially expressed relative to control cells (containing the NegCtrl-2 guide) measured by the Kolmogorov-Smirnov test at P&lt;0.01. Genes targeted by multiple guides have multiple possibly overlapping dots. See also <figref idrefs="DRAWINGS">FIG. 91</figref>.</div>
</li> <li> <para-num num="[0142]"> </para-num> <div class="description-line" id="p-0143" num="0142"> <figref idrefs="DRAWINGS">FIG. 86A-L</figref> Single-cell information reveals a bifurcated UPR within a population and allows unbiased discovery of UPR-controlled genes (A) Single-cell analysis of HSPA5perturbed cells. The figure shows t-sne plots of guide identity, cell cycle position, and UMI count per cell in HSPA5-perturbed cells and control cells (containing the NegCtrl3 guide). (B) Low rank ICA analysis of HSPA5-perturbed cells identifies two subpopulation-defining independent components. The right panel shows a discretized breakdown of the cells based on applying a threshold to IC1. (C) Branch activation scores in HSPA5-perturbed cells. (D) Expression of UPR genes in HSPA5-perturbed cells. The figure shows the normalized expression matrix of all HSPA5-perturbed cells. Each row is a cell, and each column is a gene in the same order as <figref idrefs="DRAWINGS">FIG. 83F</figref>. The cells have been ordered by increasing value of IC1. (E) Cell cycle composition of HSPA5-perturbed cells. (F) Mean expression of HSPA5 across subpopulations. Error bars are 95% confidence intervals. (G) Strategy for using correlated expression to identify functionally related genes. (H) Unbiased identification of induced gene expression programs in perturb-seq experiment. 200 example genes induced by perturbation of ER homeostasis were identified computationally, and clustered based on their co-expression across the 65,000 cells in the perturb-seq experiment. Top part of figure shows normalized expression across all of the perturbations in the perturb-seq experiment. Bottom part shows normalized expression in epistasis experiment, to assess UPR dependence. Full-size version of figure in <figref idrefs="DRAWINGS">FIG. 92A</figref>. (I) Implicit correlation information in perturb-seq orders the three branches of the UPR. UPR-responsive genes (from <figref idrefs="DRAWINGS">FIG. 83F</figref>) whose expression were altered in the perturb-seq experiment were clustered using single-cell co-expression data from the epistasis experiment, the perturb-seq experiment, and unperturbed control cells. The figure shows the cophenetic correlation coefficients between dendrogram orderings, measuring how similarly genes cluster, along with a visual guide to the movement of major groups. Full version of figure in <figref idrefs="DRAWINGS">FIG. 92B</figref>. (J) Strategy for enriching cells perturbed for a trait of interest. E.g., a group of genes of interest may be induced in only a fraction of the population across many different perturbations. (K) Enriching cells strengthens correlations and identifies related genes. Cells enriched for a set of bait cholesterol biosynthesis genes were identified. Within the enriched population, a group of genes clustered with the bait genes and showed stronger correlated expression than in unperturbed control cells. Right panel shows Reactome annotations and SREBP binding data for the group. (L) Combining correlation information with other types of data. An identical analysis to the previous panel was applied to heat shock genes. The figure shows the correlation matrix among the top hits, and compares this to the most enriched transcription factor binding sites measured by Encode. See also <figref idrefs="DRAWINGS">FIG. 92</figref>.</div>
</li> <li> <para-num num="[0143]"> </para-num> <div class="description-line" id="p-0144" num="0143"> <figref idrefs="DRAWINGS">FIG. 87A-F</figref> Translocon loss-of-function preferentially activates IRE1 UPR signaling. (A) Single-cell analysis of cells depleted for SEC61B in perturb-seq experiment. Panels show the sgRNA identity and IRE1 activation score for each cell. (B) Identical analysis for SEC61A1. (C) RT-PCR probing for XBP1 splicing. (D) K562 cells (cBA011) transduced and sorted for expression of the indicated sgRNAs were collected on the indicated days post transduction for analysis of XBP1 mRNA splicing (top) and expression of SSR2 and CHOP mRNA (bottom). Data represent means relative to ACTB mRNA and normalized to cells transfected with a negative control sgRNA (dotted lines) ±standard error of technical replicates (n=3). (E) K562 cells (cBA010) transduced with the indicated sgRNAs and selected with puromycin were treated with 0.5 μM thapsigargin for 1.5 hours or left untreated 6 days post transduction. Cells were then collected for analysis of XBP1 mRNA splicing and expression of CHOP mRNA. Data represent means relative to ACTB mRNA and normalized to cells transfected with NegCtrl-3±standard error of technical replicates (n=3). (F) A model in which IRE1α actively monitors the function and number of translocons and acts to increase them as needed. See also <figref idrefs="DRAWINGS">FIG. 93</figref>.</div>
</li> <li> <para-num num="[0144]"> </para-num> <div class="description-line" id="p-0145" num="0144"> <figref idrefs="DRAWINGS">FIG. 88A-F</figref> Design and characterization of three-guide vector (related to <figref idrefs="DRAWINGS">FIG. 82</figref>). (A) Characterization of initial three-guide vector by GFP knockdown. GFP+ K562 cells with dCas9-KRAB were transduced with either the single perturb-seq vector expressing a GFP-targeting guide RNA, initial three-guide vectors expressing a GFP-targeting guide RNA from the promoter indicated in parentheses and negative control guide RNAs from the other two promoters, or a single vector expressing a negative control guide RNA. GFP levels were measured by flow cytometry 10 d after transduction. Plotted are kernel density estimates of normalized flow cytometry counts for infected (BFP+) cells. Traces for the single perturb-seq construct and the negative control are the same as in <figref idrefs="DRAWINGS">FIG. 82E</figref>. (B) Characterization of h7SK promoter in the context of the perturb-seq vector. Experiment was conducted as described in (A). Traces for the single perturb-seq construct and the negative control are the same as in <figref idrefs="DRAWINGS">FIG. 82E</figref>. (C) Characterization of GFP+ K562 cells with increased dCas9-KRAB levels. BFP levels report on expression level of the dCas9-BFP-KRAB construct. Increase in dCas9-KRAB is measured by change in BFP fluorescence relative to WT K562 cells. Plotted are kernel density estimates of normalized flow cytometry counts. (D) Crystal structure of Cas9 bound to guide RNA and target DNA (PDB ID code 4OO8 (Nishimasu et al. 2014)), highlighting location of constant region mutations. Cas9 is shown in gray, target ssDNA in yellow, and the guide RNA in orange (targeting region) and cyan (constant region). Constant region bases that were mutated are highlighted in red. (E) Characterization of RNA polymerase III promoters from different mammalian species by GFP knockdown. GFP+ K562 cells with dCas9-KRAB were transduced with vectors expressiong a GFP-targeting guide RNA from the different promoters, in the context of the perturb-seq vector. GFP levels were measured by flow cytometry either 9 d (experiment 1) or 8 d after transduction (experiment 2). % knockdown was calculated after subtracting GFP levels of WT K562 and calculating GFP levels relative to GFP+K562 cells transduced with a negative control vector. Abbreviations: bU6, bovine U6; sU6, sheep U6; buU6, buffalo U6; pU6, pig U6. (F) Cloning strategy for final three-guide vector. In step 1, protospacers are ligated into the individual backbones. In step 2, three guide RNA expression cassettes are amplified by PCR and inserted into the perturb-seq backbone in a single reaction by four-piece Gibson assembly to obtain the final barcoded three-guide vector.</div>
</li> <li> <para-num num="[0145]"> </para-num> <div class="description-line" id="p-0146" num="0145"> <figref idrefs="DRAWINGS">FIG. 89A-B</figref> Perturb-seq analytical pipeline (related to <figref idrefs="DRAWINGS">FIG. 83</figref>). (A) Schematic of the analytical pipeline used in the paper. Each step is explained in the Methods, and each single-cell figure has a dedicated section in the Methods describing its construction. (B) Example analysis of thapsigargin-treated cells, related to <figref idrefs="DRAWINGS">FIG. 83B</figref>. The left panels show t-sne projections of the whole population derived using all differentially expressed genes, as described in the Methods. The middle panels show the 16 independent components found by low rank ICA overlaid on the t-sne plot. The right panels show the average values of the four components identified as varying by perturbation within each of the subpopulations, and the average values of the four components identified as varying through the cell cycle in each cell cycle phase. Further details are in the Methods.</div>
</li> <li> <para-num num="[0146]"> </para-num> <div class="description-line" id="p-0147" num="0146"> <figref idrefs="DRAWINGS">FIG. 90A-D</figref> CRISPRi screens used to select UPR-modulating sgRNAs for perturb-seq (related to <figref idrefs="DRAWINGS">FIG. 84</figref>). (A) K<b>562</b> cells (cBA011) were treated with indicated concentrations of tunicamycin in 0.16% DMSO or DMSO alone. Cells were evaluated by flow cytometry or collected for analysis of XBP1 mRNA splicing at indicated times. Data represent average median fluorescence ±SD (n=3) normalized to DMSO. (B) Comparison of gene phenotypes from the hCRISPRi-v1 and hCRISPRi-v2 screens. Genes chosen for analysis on the perturb-seq platform (83) are indicated in red. (C) Comparison of UPRE reporter gene phenotypes from the hCRISPRi-v2 with gene growth phenotypes from a previously reported genome-scale hCRISPRi-v2 screen (27661255). Select hits are indicated in red. (D) Top eleven annotated functional clusters from DAVID enrichment analysis. Representative names were chosen for each cluster.</div>
</li> <li> <para-num num="[0147]"> </para-num> <div class="description-line" id="p-0148" num="0147"> <figref idrefs="DRAWINGS">FIG. 91A-F</figref> Perturb-seq screen performance (related to <figref idrefs="DRAWINGS">FIG. 85</figref>). (A) Similarity of phenotypes between guides targeting the same gene. Average expression profiles were created for each sgRNA-containing subpopulation, and hierarchically clustered. Guides targeting a common gene are indicated by color. (B) Shift in sgRNA target expression upon depletion. The distribution of expression of each targeted gene is compared between control cells (containing the NegCtrl2 guide) and each sgRNA-containing subpopulation. sgRNAs are ordered by target expression. (C) Homogeneity of knockdown. Applicants computationally separated each sgRNA-containing subpopulation into top- and bottom-third most perturbed cells based on the deviation of their RNA-seq profiles from the distribution of expression seen in control cells (Methods). The plot shows the average difference in percentage knockdown between these two subpopulations for each sgRNA (gray dot), along with a kernel density estimate of the distribution (black). (D) Expression of UPR genes in perturb-seq experiment. The plot shows the average normalized expression within each perturbed subpopulation of all of the genes identified as UPR-responsive in <figref idrefs="DRAWINGS">FIG. 83F</figref>. The thapsigargin data from that figure is repeated to the right for comparison. (E) Alternate scoring system for branch activation. An alternative method of scoring branch activation was developed using independent component analysis. For comparison with <figref idrefs="DRAWINGS">FIG. 85D</figref>. See Methods for details. (F) Performance of random forest scoring system for branch activation. To cross-validate branch activation scores, three alternative scoring systems were prototyped in the epistasis experiment of <figref idrefs="DRAWINGS">FIG. 83</figref>, where branch activation can be inferred from perturbation identity: (1) group score, based on expression of defined gene lists, (2) ICA score, seen in <figref idrefs="DRAWINGS">FIG. 91E</figref>, (3) random forest score, seen in <figref idrefs="DRAWINGS">FIG. 85D</figref>. Cells were defined as “active” for a given branch if they were treated with tunicamycin or thapsigargin, and that branch of the UPR was not depleted. The plots show the distributions of scores in active and inactive cells for each of the three branches and each of the three scoring systems. See Methods for details.</div>
</li> <li> <para-num num="[0148]"> </para-num> <div class="description-line" id="p-0149" num="0148"> <figref idrefs="DRAWINGS">FIG. 92A-B</figref> Functionally clustering genes using single-cell correlation information (related to <figref idrefs="DRAWINGS">FIG. 86</figref>). (A) Full-size version of <figref idrefs="DRAWINGS">FIG. 86H</figref>. (B) Full-size version of <figref idrefs="DRAWINGS">FIG. 86I</figref>.</div>
</li> <li> <para-num num="[0149]"> </para-num> <div class="description-line" id="p-0150" num="0149"> <figref idrefs="DRAWINGS">FIG. 93A-C</figref> Depletion of individual translocon components SEC61A1, SEC61B, or SEC61G upregulate expression of complex partner genes but have distinct growth phenotypes (related to <figref idrefs="DRAWINGS">FIG. 87</figref>). (A) ATF6 and PERK branch activation scores for SEC61B- and SEC61A1-perturbed subpopulations. For comparison with <figref idrefs="DRAWINGS">FIG. 87A</figref>. (B) K562 cells (cBA011) transduced and sorted for expression of the indicated sgRNAs were collected 6 days post transduction for analysis of SEC61A1, SEC61B, SEC61G, and ALG2 expression. Data represent means relative to ACTB mRNA and normalized to cells transfected with NegCtrl-3 ±standard error of technical replicates (n=3). Data from the cells with NegCtrl-1, SEC61A1-2, and SEC61B-2 sgRNAs are represented here and in <figref idrefs="DRAWINGS">FIG. 87D</figref>. (C) Phenotypes for individual sgRNAs targeting SEC61A1, SEC61B, and SEC61G from two growth screens, one reported elsewhere (27661255) and the other conducted in parallel to the first with essentially the same conditions. Data for 10 library negative control sgRNAs were randomly chosen for inclusion. Guides used separately elsewhere are numbered.</div>
</li> </description-of-drawings>
<heading id="h-0007">DETAILED DESCRIPTION OF THE INVENTION</heading>
<li> <para-num num="[0150]"> </para-num> <div class="description-line" id="p-0151" num="0150">Functional genomics has transformed our ability to identify the components of molecular circuits in cells. However, despite over a decade of intensive research, Applicants are still incredibly limited in our ability to predict how perturbing one or more components will affect the cell's phenotype. For example, the recent explosion in studies on the genetic basis of human disease, from genome-wide association studies (Manolio et al., 2009; Shendure and Akey, 2015; Visscher et al., 2012) to rare Mendelian disease (Altshuler et al., 2008; Bamshad et al., 2011; Botstein and Risch, 2003) to cancer genetics (Melnikov et al., 2012) (Lawrence et al., 2014; Martincorena and Campbell, 2015), have identified many disease-associated genes, but the mechanisms by which they contribute to pathogenesis remain largely obscure. It is increasingly clear that a lack of biological understanding of the complex nature of molecular networks severely constrains our progress towards therapeutic intervention (Sawyers, 2004).</div>
</li> <li> <para-num num="[0151]"> </para-num> <div class="description-line" id="p-0152" num="0151">Genomic research on dissecting molecular systems in mammalian cells has proceeded along two main paths. Observational studies use global analysis of molecular profiles in different cells to draw correlational hypotheses that relate regulators to their targets. Conversely, genetic perturbation screens, with RNAi, and more recently CRISPR/Cas9-based genome editing, have opened the way for comprehensive analysis of gene function in mammalian cells in vitro and in vivo. Such screens are designed in either: (1) an individual (“arrayed”) format, where each perturbation is delivered and its effect is measured separately, or in (2) a pooled format, where the effects of perturbations are measured en masse. Pooled phenotypic readouts rely on measuring cell autonomous effects, such growth, drug resistance (Bandyopadhyay et al., 2010; Bassik et al., 2013a; Kampmann et al., 2014; Shalem et al., 2014; Wang et al., 2014), or the expression of a single gene (Melnikov et al., 2012; Parnas et al., 2015; Rajagopal et al., 2016; Smith et al., 2013). In general, pooled screens are more efficient, scalable, and less prone to batch effects than arrayed screens, but have thus far been limited to lower-content readouts. As a result, they are unable to discern between multiple molecular mechanisms that result in similar downstream phenotypes. Such distinctions require a more comprehensive readout, such as the transcriptional profile of a cell. Typically, detailed profiling of ‘hits’ from screens requires individual testing, in a time and labor intensive manner (Parnas et al., 2015).</div>
</li> <li> <para-num num="[0152]"> </para-num> <div class="description-line" id="p-0153" num="0152">Transcriptional profiles are a rich readout of the cell's molecular state, but have been challenging to assess in a screening scale. A few studies tested the transcriptional profiles in a hundred or more individual perturbation experiments, in either follow up studies of pre-validated hits, or in model organisms, such as yeast, where the most recent effort assessed ˜1,500 KO strains individually. Indeed, even signature assays (Amit et al., Science. 2009 9;326(5950):257-63) were only measured in large screens in the context of highly resourced centralized efforts (LINCS program). This “readout barrier” is common to all genetic screening, in vitro or in vivo, from unicellular model organisms to mice or human cells.</div>
</li> <li> <para-num num="[0153]"> </para-num> <div class="description-line" id="p-0154" num="0153">Indeed, it has been challenging to bridge the gap between the use of transcriptional or other rich profiles and the scale of pooled screens. In mammalian cells, a few studies tested transcriptional profiles in a hundred or more individual perturbation experiments, in either follow up studies of pre-validated hits (Berger et al., 2016; Parnas et al., 2015), or in model organisms, such as yeast (Hughes et al., 2000), where a recent effort assessed ˜1,500 knock out (KO) strains individually (Kemmeren et al., 2014). Indeed, even signature assays (Amit et al., 2009) were only performed in large screens in the context of highly resourced centralized efforts (Duan et al., 2014; Lamb et al., 2006). This “readout barrier” is common to all genetic screening, in vitro or in vivo, from unicellular model organisms to mice or human cells.</div>
</li> <li> <para-num num="[0154]"> </para-num> <div class="description-line" id="p-0155" num="0154">Notably, a rich readout—such as a genomic profile—would be particularly important to understand the combined effect of multiple factors, in systems where this effect cannot simply be predicted by the sum of their individual effects. Comprehensive analysis of the effects of pairs of genes on cell viability (“synthetic lethality”) has been performed in yeast (Boone et al., 2007; Costanzo et al., 2010; Tong, 2004). In mammals, the effects of interactions on cell viability (Bassik et al., 2013b; Wong et al., 2016) or cell morphology characteristics (Laufer et al., 2013) were assessed for a small number of selected pairs of genes. Few studies have examined the relationship between a high dimensional phenotype, like gene expression, and pairwise interactions. One study in yeast that explored the marginal transcriptional effect associated with double knockouts of a few transcription factors compared to their respective single knockouts (Capaldi et al., 2008), highlighted the potential power of the approach to determine how a regulatory circuit is wired and functions. Very few studies have examined higher order interactions beyond a limited handful (Haber et al., 2013; Tsumura et al., 2006). To the best of our knowledge, there have been few if any studies coupling a high content readout with scalable generation of cells containing multiple perturbations.</div>
</li> <li> <para-num num="[0155]"> </para-num> <div class="description-line" id="p-0156" num="0155">It has been suggested, by the Applicants (Macosko et al., 2015) and others (Liberali et al., 2014; Macosko et al., 2015; Shendure and Fields, 2016) that scalable methods of transcriptionally assaying perturbations using single cell profiling, if developed, could provide an important new tool in the understanding the functions of genes and circuits.</div>
</li> <li> <para-num num="[0156]"> </para-num> <div class="description-line" id="p-0157" num="0156">Here, applicants address the challenge of scaling the phenotypic measurements in genetic screens by developing Perturb-Seq and provide this ability by developing Perturb-seq, which combines pooled CRISPR screening with a massively parallel single cell RNA-seq readout. Applicants leverage the modularity of sgRNAs in the CRISPR/Cas9 system to perform pooled, multi-locus gene perturbation in mammalian cells (Chen et al., 2015; Cong et al., 2013; Gilbert et al., 2014, 2013; Konermann et al., 2014; Qi et al., 2013; Ran et al., 2015; Zalatan et al., 2015), including in primary cells (Platt et al., 2014). Applicants rely on the scale and single cell nature of massively parallel single cell RNA-seq (Fan et al., 2015; Klein et al., 2015; Macosko et al., 2015) to provide a rich, genome-scale cell autonomous readout for up to millions of cells in a day's work (with appropriate parallelization (Joensson and Andersson Svahn, 2012)). Perturb-Seq uses a CRISPR lentiviral vector, which both delivers an sgRNA to a cell, and reports on the identity of the delivered sgRNA by an expressed barcode on a polyadenylated transcript, captured efficiently by scRNA-seq. Perturb-Seq can be used to assess the effect of single gene perturbations. By increasing the multiplicity of infection, a greater fraction of cells possesses more than one guide, and the approach is more powered to test for epistatic effects. Applicants further developed an integrated computational framework to decipher the effect of individual perturbations and the marginal contributions of pairwise interactions on the level of each expressed gene, gene modules, and global cell states and types.</div>
</li> <li> <para-num num="[0157]"> </para-num> <div class="description-line" id="p-0158" num="0157">Applicants demonstrate the power of Perturb-seq in the context of two distinct biological systems: perturbing key transcription factors (TFs) implicated in cell differentiation and the immune response in bone marrow derived dendritic cells (BMDCs), post-mitotic primary innate immune cells responding to the pathogen component lipopolysaccharide (LPS); and perturbing TFs or(MOI) cell cycle regulators in K562 cells, a rapidly dividing erythroleukemia cell line. Applicants further develop an integrated computational framework to decipher the effect of individual perturbations and the marginal contributions of pairwise interactions on the level of each transcript, transcriptional module, and global cell states. Our framework can be extended to include other high dimensional molecular phenotypes as they become available, as well as incorporate a variety of cell metadata such as lineage information.</div>
</li> <li> <para-num num="[0158]"> </para-num> <div class="description-line" id="p-0159" num="0158">The present invention provides tools and methods for the systematic analysis of genetic interactions, including higher order interactions.</div>
</li> <li> <para-num num="[0159]"> </para-num> <div class="description-line" id="p-0160" num="0159">The present invention provides tools and methods for combinatorial probing of cellular circuits, for dissecting cellular circuitry, for delineating molecular pathways, and/or for identifying relevant targets for therapeutics development.</div>
</li> <li> <para-num num="[0160]"> </para-num> <div class="description-line" id="p-0161" num="0160">The present invention provides tools and methods for pooled screening of perturbations and genome scale readouts in single cells, thus allowing relevant phenotypes to be correlated to specific perturbations.</div>
</li> <li> <para-num num="[0161]"> </para-num> <div class="description-line" id="p-0162" num="0161">In one aspect, the invention provides a method for determining genetic interactions. This method involves causing a set of P genetic perturbations in cells, wherein the method may comprise: determining, based upon random sampling, a subset of π genetic perturbations from the set of P genetic perturbations; performing said subset of π genetic perturbations in a population of cells; performing single-cell molecular profiling of the population of genetically perturbed cells of step; inferring single-cell molecular profiles for the set of P genetic perturbations in cells.</div>
</li> <li> <para-num num="[0162]"> </para-num> <div class="description-line" id="p-0163" num="0162">The population of cells with a plurality of genomic sequence or perturbation conditions involves a plurality of cells and perturbations to be tested and measurements sampled to obtain meaningful data and to infer appropriate circuits. The number of genes perturbed, and how many are perturbed simultaneously (the order of the perturbation, pairs, triplets, etc.) varies. In a tissue with n cell types, the rarest present in m %, how many cells X do you need to sequence so that you have at least Y of the rarest subtype.</div>
</li> <li> <para-num num="[0163]"> </para-num> <div class="description-line" id="p-0164" num="0163">For example, ˜500 cells ensures ≥95% chance of including ≥10 of each cell type, based on the following calculation (<figref idrefs="DRAWINGS">FIG. 34</figref>). Assume the most conservative scenario that of M cell subtypes (for example, 12), all but one having the lowest predicted proportion (for example, p<sub>min</sub>=5%). Assuming that the Central Limit Theorem holds (a reasonable assumption when solving to detect at least 10 cells of each type) the number of cells of each type i, termed T<sub>i</sub>, will distribute as E[T,<sub>i</sub>]=N*p<sub>min </sub>, STDV[T<sub>i</sub>]=√(N*p<sub>min</sub>*(1-p<sub>min</sub>)). The minimal N (total number of cells to profile) can be solved such that all (m−1) subtypes have at least n cells (the last, majority, subtype easily clears this threshold since its proportion is much higher). Applicants confirmed with simulation that the strategy conservatively holds in practice even for n&lt;10, and take a margin of additional (conservative) error, to allow for subsequent failed RNA-Seq experiments (&lt;20-30%, depending on protocol).</div>
</li> <heading id="h-0008">Modelling Genetic Interactions</heading>
<li> <para-num num="[0164]"> </para-num> <div class="description-line" id="p-0165" num="0164">The method of the invention may be used for determining genetic interactions, including modelling and/or analyzing such interactions. Such genetic interactions form part of cellular circuitry, in that the interactions reflect connections of components within one or more cellular pathways. Such pathways may be intracellular pathways or intercellular pathways.</div>
</li> <li> <para-num num="[0165]"> </para-num> <div class="description-line" id="p-0166" num="0165">In some embodiments, the method of the invention may further comprise determining genetic interactions.</div>
</li> <li> <para-num num="[0166]"> </para-num> <div class="description-line" id="p-0167" num="0166">In some embodiments, the method of the invention may further comprise confirming genetic interactions with additional genetic manipulations.</div>
</li> <li> <para-num num="[0167]"> </para-num> <div class="description-line" id="p-0168" num="0167">The method may further comprise a validation step, wherein additional manipulations are performed in order to confirm previously identified genetic interactions. Such validation step may include in vivo or in vitro experiments, such as gene inactivation, gene deletion, gene activation or overexpression, and combinations thereof. Such genetic manipulations may be performed with any genetic tool available in the art, comprising but not limited to RNAi, CRISPR-Cas based gene editing, nucleic acid transfection, etc.</div>
</li> <heading id="h-0009">Genetic Perturbations</heading>
<li> <para-num num="[0168]"> </para-num> <div class="description-line" id="p-0169" num="0168">In one aspect, said set of P genetic perturbations or said subset of π genetic perturbations may comprise single-order genetic perturbations. Within the meaning of the present invention, single-order genetic perturbation means that a given cell undergoes a single genetic perturbation (one perturbation per cell).</div>
</li> <li> <para-num num="[0169]"> </para-num> <div class="description-line" id="p-0170" num="0169">In one aspect, said set of P genetic perturbations or said subset of π genetic perturbations may comprise combinatorial genetic perturbations. Within the meaning of the present invention, combinatorial or higher-order genetic perturbation means that a given cell undergoes a combination of k single-order genetic perturbations (k perturbations per cell), with k&gt;1. In some embodiments, k is an integer ranging from 2 to 15. In some embodiments, k=2, 3, 4, 5, 6, 7, 8, 9 or 10.</div>
</li> <li> <para-num num="[0170]"> </para-num> <div class="description-line" id="p-0171" num="0170">Within the meaning of the present invention, said genetic perturbation may comprise gene knock-down (gene repression or gene inactivation), gene knock-out (gene deletion), gene activation, gene insertion, or regulatory element deletion.</div>
</li> <li> <para-num num="[0171]"> </para-num> <div class="description-line" id="p-0172" num="0171">Combinations of different types of genetic perturbations are also envisioned within the meaning of the present invention. For example, a combination of genetic perturbations may comprise a knock-down for a first gene, combined to an activation of a second gene, etc.</div>
</li> <li> <para-num num="[0172]"> </para-num> <div class="description-line" id="p-0173" num="0172">In one aspect, said set of P genetic perturbations or said subset of π genetic perturbations may comprise genome-wide perturbations. Genome-wide perturbations are genetic perturbations that affect loci across the entire genome. Genome-wide perturbation may include single perturbations of &gt;100, &gt;200, &gt;500, &gt;1,000, &gt;2,500, &gt;5,000, &gt;10,000, &gt;15,000 or &gt;20,000 single genomic loci. The present invention encompasses k-order combinations of genome-wide perturbation.</div>
</li> <li> <para-num num="[0173]"> </para-num> <div class="description-line" id="p-0174" num="0173">In some embodiments, the method may comprise determining k-order genetic interactions.</div>
</li> <li> <para-num num="[0174]"> </para-num> <div class="description-line" id="p-0175" num="0174">In some embodiments, said set of P genetic perturbations may comprise combinatorial genetic perturbations, such as k-order combinations of single-order genetic perturbations, wherein k is an integer ranging from 2 to 15, and step (e) may comprise determining j-order genetic interactions, with j&lt;k. Such embodiments rely on sampling higher-order interactions in order to more efficiently infer lower order ones. Given a limited number of possible assays, one is more powered to determine lower order interactions (e.g., 2-, 3-way) from measuring higher order interactions (e.g., 5-way) than from allotting all assays to the lower order, because any higher order interaction carries some information about all interaction terms up to that order (e.g., in compressed sensing, it informs in convolved form on additional Fourier coefficients). Thus, even if most interactions are low order (2- or 3-way) these embodiments are more powered to detect them.</div>
</li> <heading id="h-0010">CRISPR-Cas Systems</heading>
<li> <para-num num="[0175]"> </para-num> <div class="description-line" id="p-0176" num="0175">In some embodiments, RNAi- or CRISPR-Cas-based perturbation may be performed. Said perturbation may be performed (e.g. “delivered”) in an array-format or pool-format. Some embodiments may comprise pooled single or combinatorial CRISPR-Cas-based perturbation with a genome-wide library of sgRNAs, wherein each sgRNA comprises a unique molecular identifier. In some embodiments, a step may comprise pooled combinatorial CRISPR-Cas-based perturbation with a genome-wide library of sgRNAs, wherein each sgRNA comprises a unique molecular identifier and is co-delivered with a reporter mRNA.</div>
</li> <li> <para-num num="[0176]"> </para-num> <div class="description-line" id="p-0177" num="0176">CRISPR-Cas systems, including CRISPR-Cas9 systems, as used herein, refer to non-naturally occurring systems derived from bacterial Clustered Regularly Interspaced Short Palindromic Repeats loci. These systems generally comprise an enzyme (Cas protein, such as Cas9 protein) and one or more RNAs. Said RNA is a CRISPR RNA and may be an sgRNA. Said RNA and/or said enzyme may be engineered, for example for optimal use in mammalian cells, for optimal delivery therein, for optimal activity therein, for specific uses in gene editing, etc.</div>
</li> <li> <para-num num="[0177]"> </para-num> <div class="description-line" id="p-0178" num="0177">sgRNA refers to a CRISPR single-guide RNA. This RNA is a component of a CRISPR-Cas system. The sequence of the sgRNA determines the target sequence for gene editing, knock-down, knock-out, insertion, etc. For genome-wide approaches, it is possible to design and construct suitable sgRNA libraries. Such sgRNAs may be delivered to cells using vector delivery such as viral vector delivery. Combination of CRISPR-Cas-mediated perturbations may be obtained by delivering multiple sgRNAs within a single cell. This may be achieved in pooled format. In the case of sgRNA viral vector delivery, combined perturbation may be obtained by delivering several sgRNA vectors to the same cell. This may also be achieved in pooled format, and number of combined perturbations in a cell then corresponds to the MOI (multiplicity of infection). Using CRISPR-Cas systems, one may generally implement MOI values of up to 10, 12 or 15.</div>
</li> <li> <para-num num="[0178]"> </para-num> <div class="description-line" id="p-0179" num="0178">The CRISPR-Cas system may be implemented in order to cause massively combinatorial molecular perturbations (MCMP), including single-order and combinatorial genome-wide genetic perturbations.</div>
</li> <li> <para-num num="[0179]"> </para-num> <div class="description-line" id="p-0180" num="0179">CRISPR-Cas-based gene editing allows to perform pooled genome-scale screens with expression readouts in primary cells (A Genome-wide CRISPR Screen in Primary Immune Cells to Dissect Regulatory Networks. Parnas 0., Jovanovic M., Eisenhaure T M., Herbst R H., Dixit A., Ye C J., Przybylski D., Platt R J., Tirosh I., Sanjana N E., Shalem O., Satija R., Raychowdhury R., Mertins P., Carr S A., Zhang F., Hacohen N., Regev A. A Genome-wide CRISPR Screen in Primary Immune Cells to Dissect Regulatory Networks. Cell Jul. 15. (2015) 2015 Jul. 30; 162(3):675-86. doi:10.1016/j.cell.2015.06.059. Epub 2015 Jul. 16).</div>
</li> <li> <para-num num="[0180]"> </para-num> <div class="description-line" id="p-0181" num="0180">In some embodiments, the present invention involves combinatorial perturbations by way of CRISPR-Cas (such as CRISPR-Cas9) assays. In accordance with the present invention, sampling a far-from-exhaustive number of higher order perturbations, when coupled with complex genomic readouts, may suffice to resolve most non-linear relations. Accordingly, in some aspects, the present invention relies on pooled, combinatorial perturbations with genomic readout into Massively Combinatorial Perturbation Profiling (MCPP).</div>
</li> <li> <para-num num="[0181]"> </para-num> <div class="description-line" id="p-0182" num="0181">In some embodiments, the method of the invention may comprise one or more CRISPR-Cas-based assays. Such CRISPR-Cas assays are advantageous for implementing a precise perturbation of genes and their expression levels.</div>
</li> <li> <para-num num="[0182]"> </para-num> <div class="description-line" id="p-0183" num="0182">In some embodiments, CRISPR-Cas systems may be used to knockout protein-coding genes by frameshifts (indels). Embodiments include efficient and specific CRISPR-Cas9 mediated knockout (Gilbert, L. A., Horlbeck, M. A., Adamson, B., Villalta, J. E., Chen, Y., Whitehead, E. H., Guimaraes, C., Panning, B., Ploegh, H. L., Bassik, M. C., Qi, L. S., Kampmann, M. &amp; Weissman, J. S. Genome-Scale CRISPR-Mediated Control of Gene Repression and Activation. Cell. 159, 647-661, doi:10.1016/j.cell.2014.09.029 (2014). PMCID:4253859; Ran, F. A., Cong, L., Yan, W. X., Scott, D. A., Gootenberg, J. S., Kriz, A. J., Zetsche, B., Shalem, O., Wu, X., Makarova, K. S., Koonin, E. V., Sharp, P. A. &amp; Zhang, F. In vivo genome editing using Staphylococcus aureus Cas9. Nature. 520, 186-191, doi:10.1038/nature14299 (2015). PMCID:4393360), including a CRISPR mediated double-nicking to efficiently modify both alleles of a target gene or multiple target loci (Ran, F. A., Hsu, P. D., Lin, C. Y., Gootenberg, J. S., Konermann, S., Trevino, A. E., Scott, D. A., Inoue, A., Matoba, S., Zhang, Y. &amp; Zhang, F. Double nicking by RNA-guided CRISPR Cas9 for enhanced genome editing specificity. Cell. 154, 1380-1389, doi:10.1016/j.cell.2013.08.021 (2013). PMCID:3856256; Wang, H., Yang, H., Shivalila, C. S., Dawlaty, M. M., Cheng, A. W., Zhang, F. &amp; Jaenisch, R. One-step generation of mice carrying mutations in multiple genes by CRISPR-Cas-mediated genome engineering. Cell. 153, 910-918, doi:10.1016/j.cell.2013.04.025 (2013). PMCID:3969854) and implementation of a smaller Cas9 protein for delivery on smaller vectors (Ran, F. A., Cong, L., Yan, W. X., Scott, D. A., Gootenberg, J. S., Kriz, A. J., Zetsche, B., Shalem, O., Wu, X., Makarova, K. S., Koonin, E. V., Sharp, P. A. &amp; Zhang, F. In vivo genome editing using <i>Staphylococcus aureus </i>Cas9. <i>Nature. </i>520, 186-191, doi:10.1038/nature14299 (2015). PMCID:4393360).</div>
</li> <li> <para-num num="[0183]"> </para-num> <div class="description-line" id="p-0184" num="0183">CRISPR-mediated activation or inactivation (CRISPRa/i) systems may be used to activate or inactivate gene transcription. Briefly, a nuclease-dead (deactivated) Cas9 RNA-guided DNA binding domain (dCas9) (Qi, L. S., Larson, M. H., Gilbert, L. A., Doudna, J. A., Weissman, J. S., Arkin, A. P. &amp; Lim, W. A. Repurposing CRISPR as an RNA-guided platform for sequence-specific control of gene expression. Cell. 152, 1173-1183, doi:10.1016/j.cell.2013.02.022 (2013). PMCID:3664290) tethered to transcriptional repressor domains that promote epigenetic silencing (e.g., KRAB) forms a “CRISPRi” (Gilbert, L. A., Larson, M. H., Morsut, L., Liu, Z., Brar, G. A., Torres, S. E., Stern-Ginossar, N., Brandman, O., Whitehead, E. H., Doudna, J. A., Lim, W. A., Weissman, J. S. &amp; Qi, L. S. CRISPR-mediated modular RNA-guided regulation of transcription in eukaryotes. Cell. 154, 442-451, doi:10.1016/j.cell.2013.06.044 (2013). PMCID:3770145; Konermann, S., Brigham, M. D., Trevino, A. E., Hsu, P. D., Heidenreich, M., Cong, L., Platt, R. J., Scott, D. A., Church, G. M. &amp; Zhang, F. Optical control of mammalian endogenous transcription and epigenetic states. <i>Nature. </i>500, 472-476, doi:10.1038/nature12466 (2013). PMCID:3856241) that represses transcription. To use dCas9 as an activator (CRISPRa), a guide RNA may be engineered to carry RNA binding motifs (e.g., MS2) that recruit effector domains fused to RNA-motif binding proteins, increasing transcription (Konermann, S., Brigham, M. D., Trevino, A. E., Joung, J., Abudayyeh, O. O., Barcena, C., Hsu, P. D., Habib, N., Gootenberg, J. S., Nishimasu, H., Nureki, 0. &amp; Zhang, F. Genome-scale transcriptional activation by an engineered CRISPR-Cas9 complex. <i>Nature. </i>517, 583-588, doi:10.1038/nature14136 (2015). PMCID:4420636).</div>
</li> <li> <para-num num="[0184]"> </para-num> <div class="description-line" id="p-0185" num="0184">CRISPR-Cas systems may also be used for the deletion of regulatory elements. To target non-coding elements, pairs of guides may be designed and used to delete regions of a defined size, and tile deletions covering sets of regions in pools. The delivery of two sgRNAs may mediate efficient excision of 500 bp genomic fragments.</div>
</li> <li> <para-num num="[0185]"> </para-num> <div class="description-line" id="p-0186" num="0185">CRISPR-Cas systems may also be used for gene editing, e.g. by RNA-templated homologous recombination. Keskin, H., Shen, Y., Huang, F., Patel, M., Yang, T., Ashley, K., Mazin, A. V. &amp; Storici, F. Transcript-RNA-templated DNA recombination and repair. <i>Nature. </i>515, 436-439, doi:10.1038/nature13682 (2014).</div>
</li> <li> <para-num num="[0186]"> </para-num> <div class="description-line" id="p-0187" num="0186">CRISPR transgenic mice may be used to derive ‘CRISPR-ready’ cells. ‘CRISPR-mice’ are mice where the mouse germ line is engineered to harbor key elements of a CRISPR system, and cells require only the programmable (sgRNA) element to activate the CRISPR-Cas system. CRISPR mice include Cas9-transgenic mice (Platt, R. J., Chen, S., Zhou, Y., Yim, M. J., Swiech, L., Kempton, H. R., Dahlman, J. E., Parnas, O., Eisenhaure, T. M., Jovanovic, M., Graham, D. B., Jhunjhunwala, S., Heidenreich, M., Xavier, R. J., Langer, R., Anderson, D. G., Hacohen, N., Regev, A., Feng, G., Sharp, P. A. &amp; Zhang, F. CRISPR-Cas9 knockin mice for genome editing and cancer modeling. Cell. 159, 440-455, doi:10.1016/j.cell.2014.09.014 (2014). PMCID:4265475; Parnas O., Jovanovic M., Eisenhaure T M., Herbst R H., Dixit A., Ye C J., Przybylski D., Platt R J., Tirosh I., Sanjana N E., Shalem O., Satija R., Raychowdhury R., Mertins P., Carr S A., Zhang F., Hacohen N., Regev A. A Genome-wide CRISPR Screen in Primary Immune Cells to Dissect Regulatory Networks. <i>Cell </i>Jul. 15. (2015) 2015 Jul. 30; 162(3):675-86. doi:10.1016/j.cell.2015.06.059. Epub 2015 Jul. 16).</div>
</li> <li> <para-num num="[0187]"> </para-num> <div class="description-line" id="p-0188" num="0187">CRISPR-Cas based perturbations, including single order or higher order perturbations, may be implemented in pooled format. The perturbation (screen) may be performed with expression readouts or reporter expression readout (genome-wide reporter-based pooled screens).</div>
</li> <li> <para-num num="[0188]"> </para-num> <div class="description-line" id="p-0189" num="0188">CRISPR-Cas functional genomics assays that may be used to cause sets of genetic perturbations are described in Shalem O., Sanjana N E., Zhang F. High-throughput functional genomics using CRISPR-Cas9. <i>Nat Rev Genet. </i>May; 16(5):299-311. (2015). doi: 10.1038/nrg3899. Epub 2015 Apr. 9.</div>
</li> <li> <para-num num="[0189]"> </para-num> <div class="description-line" id="p-0190" num="0189">sgRNA libraries, including genome-wide libraries of sgRNAs, may be designed as described in Parnas O., Jovanovic M., Eisenhaure T M., Herbst R H., Dixit A., Ye C J., Przybylski D., Platt R J., Tirosh I., Sanjana N E., Shalem O., Satija R., Raychowdhury R., Mertins P., Carr SA., Zhang F., Hacohen N., Regev A. A Genome-wide CRISPR Screen in Primary Immune Cells to Dissect Regulatory Networks. <i>Cell </i>Jul. 15. (2015) 2015 Jul. 30; 162(3):675-86. doi: 10.1016/j.cell.2015.06.059. Epub 2015 Jul. 16; Sanjana, N. E., Shalem, O. &amp; Zhang, F. Improved vectors and genome-wide libraries for CRISPR screening. <i>Nat Methods. </i>11, 783-784, doi:10.1038/nmeth.3047 (2014); Shalem, O., Sanjana, N. E., Hartenian, E., Shi, X., Scott, D. A., Mikkelsen, T. S., Heckl, D., Ebert, B. L., Root, D. E., Doench, J. G. &amp; Zhang, F. Genome-scale CRISPR-Cas9 knockout screening in human cells. <i>Science. </i>343, 84-87, doi:10.1126/science.1247005 (2014). PMCID:4089965; Shalem, O., Sanjana, N. E. &amp; Zhang, F. High-throughput functional genomics using CRISPR-Cas9. Nat Rev Genet. 16, 299-311, doi:10.1038/nrg3899 (2015).</div>
</li> <li> <para-num num="[0190]"> </para-num> <div class="description-line" id="p-0191" num="0190">A pooled genome-wide screen for CRISPR-mediated KO (knock-out) may be performed as in Shalem, O., Sanjana, N. E., Hartenian, E., Shi, X., Scott, D. A., Mikkelsen, T. S., Heckl, D., Ebert, B. L., Root, D. E., Doench, J. G. &amp; Zhang, F. Genome-scale CRISPR-Cas9 knockout screening in human cells. <i>Science. </i>343, 84-87, doi:10.1126/science.1247005 (2014). PMCID:4089965.</div>
</li> <li> <para-num num="[0191]"> </para-num> <div class="description-line" id="p-0192" num="0191">An expression marker-based genome-wide CRISPR screen may be performed as in Parnas O., Jovanovic M., Eisenhaure T M., Herbst R H., Dixit A., Ye C J., Przybylski D., Platt R J., Tirosh I., Sanjana N E., Shalem O., Satija R., Raychowdhury R., Mertins P., Carr S A., Zhang F., Hacohen N., Regev A. A Genome-wide CRISPR Screen in Primary Immune Cells to Dissect Regulatory Networks. <i>Cell </i>Jul. 15. (2015) 2015 Jul. 30; 162(3):675-86. doi: 10.1016/j.cell.2015.06.059. Epub 2015 Jul. 16.</div>
</li> <li> <para-num num="[0192]"> </para-num> <div class="description-line" id="p-0193" num="0192">A pooled, genome-scale, CRISPRa screen may be performed as in Konermann, S., Brigham, M. D., Trevino, A. E., Joung, J., Abudayyeh, O. O., Barcena, C., Hsu, P. D., Habib, N., Gootenberg, J. S., Nishimasu, H., Nureki, O. &amp; Zhang, F. Genome-scale transcriptional activation by an engineered CRISPR-Cas9 complex. <i>Nature. </i>517, 583-588, doi:10.1038/nature14136 (2015). PMCID:4420636.</div>
</li> <li> <para-num num="[0193]"> </para-num> <div class="description-line" id="p-0194" num="0193">Pooled combinatorial perturbations may be performed, where the delivered perturbations and impact (molecular profiling) are determined post hoc, in either a conventional readout (e.g., sorting followed by sequencing) or with high-content single cell genomics.</div>
</li> <li> <para-num num="[0194]"> </para-num> <div class="description-line" id="p-0195" num="0194">In some embodiment, the CRISPR-Cas screen is performed by co-delivering multiple sgRNA using virale vector delivery (eg, sgRNA encoding vectors at a relatively high MOI) into cells pre-expressing the Cas9 enzyme to obtain as many higher order combinations as possible. For small sets of ˜5 genes one may generate a combinatorially complete ascertained set of all 32 perturbations.</div>
</li> <li> <para-num num="[0195]"> </para-num> <div class="description-line" id="p-0196" num="0195">To detect which perturbations were co-delivered in pooled bins, several strategies are envisioned: (1) Combining two or more guide barcodes using in situ PCR in PEG hydrogel that restricts the diffusion of double stranded DNA; (2) Split-pool tagging of guide barcodes in hydrogels, such that only guides from the same cell are tagged with the same sequence; (3) FISH of expressed guides for imaging readouts. In each case it is possible to use use an error-correction scheme in the barcodes.</div>
</li> <li> <para-num num="[0196]"> </para-num> <div class="description-line" id="p-0197" num="0196">To detect which perturbations were co-delivered with a single cell genomics readout, it is possible to report the (combinatorial) perturbation in a manner compatible with the full genomic readout. For example one may use an sgRNA vector that also highly expresses a synthetic polyadenylated RNA reporter of the sgRNA barcode. This RNA will be captured along with the cellular mRNA in the transcriptome profiling, eg scRNA-seq (Drop-Seq, see below), or reported by FISH hybridization, such that the same assay ascertains the sgRNAs and their impact on expression (Parnas O., Jovanovic M., Eisenhaure T M., Herbst R H., Dixit A., Ye C J., Przybylski D., Platt R J., Tirosh I., Sanjana N E., Shalem O., Satija R., Raychowdhury R., Mertins P., Carr S A., Zhang F., Hacohen N., Regev A. A Genome-wide CRISPR Screen in Primary Immune Cells to Dissect Regulatory Networks. <i>Cell Jul. </i>15. (2015) 2015 Jul. 30; 162(3):675-86. doi:10.1016/j.cell.2015.06.059. Epub 2015 Jul. 16).</div>
</li> <li> <para-num num="[0197]"> </para-num> <div class="description-line" id="p-0198" num="0197">With respect to general information on CRISPR-Cas Systems, components thereof, and delivery of such components, including methods, materials, delivery vehicles, vectors, particles, AAV, and making and using thereof, including as to amounts and formulations, all useful in the practice of the instant invention, reference is made to: U.S. Pat. Nos. 8,999,641, 8,993,233, 8,945,839, 8,932,814, 8,906,616, 8,895,308, 8,889,418, 8,889,356, 8,871,445, 8,865,406, 8,795,965, 8,771,945 and 8,697,359; US Patent Publications US 2014-0310830 (U.S. application Ser. No. 14/105,031), US 2014-0287938 A1 (U.S. application Ser. No. 14/213,991), US 2014-0273234 A1 (U.S. application Ser. No. 14/293,674), US2014-0273232 A1 (U.S. application Ser. No. 14/290,575), US 2014-0273231 (U.S. application Ser. No. 14/259,420), US 2014-0256046 A1 (U.S. application Ser. No. 14/226,274), US 2014-0248702 A1 (U.S. application Ser. No. 14/258,458), US 2014-0242700 A1 (U.S. application Ser. No. 14/222,930), US 2014-0242699 A1 (U.S. application Ser. No. 14/183,512), US 2014-0242664 A1 (U.S. application Ser. No. 14/104,990), US 2014-0234972 A1 (U.S. application Ser. No. 14/183,471), US 2014-0227787 A1 (U.S. application Ser. No. 14/256,912), US 2014-0189896 A1 (U.S. application Ser. No. 14/105,035), US 2014-0186958 (U.S. application Ser. No. 14/105,017), US 2014-0186919 A1 (U.S. application Ser. No. 14/104,977), US 2014-0186843 A1 (U.S. application Ser. No. 14/104,900), US 2014-0179770 A1 (U.S. application Ser. No. 14/104,837) and US 2014-0179006 A1 (U.S. application Ser. No. 14/183,486), US 2014-0170753 (U.S. application Ser. No. 14/183,429); European Patents EP 2 784 162 B1 and EP 2 771 468 B1; European Patent Applications EP 2 771 468 (EP13818570.7), EP 2 764 103 (EP13824232.6), and EP 2 784 162 (EP14170383.5); and PCT Patent Publications PCT Patent Publications WO 2014/093661 (PCT/US2013/074743), WO 2014/093694 (PCT/US2013/074790), WO 2014/093595 (PCT/US2013/074611), WO 2014/093718 (PCT/US2013/074825), WO 2014/093709 (PCT/US2013/074812), WO 2014/093622 (PCT/US2013/074667), WO 2014/093635 (PCT/US2013/074691), WO 2014/093655 (PCT/US2013/074736), WO 2014/093712 (PCT/US2013/074819), WO2014/093701 (PCT/US2013/074800), WO2014/018423 (PCT/US2013/051418), WO 2014/204723 (PCT/US52014/041790), WO 2014/204724 (PCT/US2014/041800), WO 2014/204725 (PCT/US52014/041803), WO 2014/204726 (PCT/US2014/041804), WO 2014/204727 (PCT/US52014/041806), WO 2014/204728 (PCT/US2014/041808), WO 2014/204729 (PCT/US2014/041809). Reference is also made to U.S. provisional patent applications 61/758,468; 61/802,174; 61/806,375; 61/814,263; 61/819,803 and 61/828,130, filed on Jan. 30, 2013; Mar. 15, 2013; Mar. 28, 2013; Apr. 20, 2013; May 6, 2013 and May 28, 2013 respectively. Reference is also made to U.S. provisional patent application 61/836,123, filed on Jun. 17, 2013. Reference is additionally made to U.S. provisional patent applications 61/835,931, 61/835,936, 61/836,127, 61/836, 101, 61/836,080 and 61/835,973, each filed Jun. 17, 2013. Further reference is made to U.S. provisional patent applications 61/862,468 and 61/862,355 filed on Aug. 5, 2013; 61/871,301 filed on Aug. 28, 2013; 61/960,777 filed on Sep. 25, 2013 and 61/961,980 filed on Oct. 28, 2013. Reference is yet further made to: PCT Patent applications Nos: PCT/US2014/041803, PCT/US52014/041800, PCT/US52014/041809, PCT/US52014/041804 and PCT/US52014/041806, each filed Jun. 10, 2014 6/10/14; PCT/US2014/041808 filed Jun. 11, 2014; and PCT/US2014/62558 filed Oct. 28, 2014, and U.S. Provisional Patent Applications Ser. Nos.: 61/915,150, 61/915,301, 61/915,267 and 61/915,260, each filed Dec. 12, 2013; 61/757,972 and 61/768,959, filed on Jan. 29, 2013 and Feb. 25, 2013; 61/835,936, 61/836,127, 61/836,101, 61/836,080, 61/835,973, and 61/835,931, filed Jun. 17, 2013; 62/010,888 and 62/010,879, both filed Jun. 11, 2014; 62/010,329 and 62/010,441, each filed Jun. 10, 2014; 61/939,228 and 61/939,242, each filed Feb. 12, 2014; 61/980,012, filed Apr. 15,2014; 62/038,358, filed Aug. 17, 2014; 62/054,490, 62/055,484, 62/055,460 and 62/055,487, each filed Sep. 25, 2014; and 62/069,243, filed Oct. 27, 2014. Reference is also made to U.S. provisional patent applications Nos. 62/055,484, 62/055,460, and 62/055,487, filed Sep. 25, 2014; U.S. provisional patent application 61/980,012, filed Apr. 15, 2014; and U.S. provisional patent application 61/939,242 filed Feb. 12, 2014. Reference is made to PCT application designating, inter alia, the United States, application No. PCT/US14/41806, filed Jun. 10, 2014. Reference is made to U.S. provisional patent application 61/930,214 filed on Jan. 22, 2014. Reference is made to U.S. provisional patent applications 61/915,251; 61/915,260 and 61/915,267, each filed on Dec. 12, 2013. Reference is made to U.S. provisional patent application U.S. Ser. No. 61/980,012 filed Apr. 15, 2014. Reference is made to PCT application designating, inter alia, the United States, application No. PCT/US14/41806, filed Jun. 10, 2014. Reference is made to U.S. provisional patent application 61/930,214 filed on Jan. 22, 2014. Reference is made to U.S. provisional patent applications 61/915,251; 61/915,260 and 61/915,267, each filed on Dec. 12, 2013.</div>
</li> <li> <para-num num="[0198]"> </para-num> <div class="description-line" id="p-0199" num="0198">Mention is also made of U.S. application 62/091,455, filed, 12 Dec. 14, PROTECTED GUIDE RNAS (PGRNAS); U.S. application 62/096,708, 24 Dec. 14, PROTECTED GUIDE RNAS (PGRNAS); U.S. application 62/091,462, 12 Dec. 14, DEAD GUIDES FOR CRISPR TRANSCRIPTION FACTORS; U.S. application 62/096,324, 23 Dec. 14, DEAD GUIDES FOR CRISPR TRANSCRIPTION FACTORS; U.S. application 62/091,456, 12 Dec. 14, ESCORTED AND FUNCTIONALIZED GUIDES FOR CRISPR-CAS SYSTEMS; U.S. application 62/091,461, 12 Dec. 14, DELIVERY, USE AND THERAPEUTIC APPLICATIONS OF THE CRISPR-CAS SYSTEMS AND COMPOSITIONS FOR GENOME EDITING AS TO HEMATOPOETIC STEM CELLS (HSCs); U.S. application 62/094,903, 19 Dec. 14, UNBIASED IDENTIFICATION OF DOUBLE-STRAND BREAKS AND GENOMIC REARRANGEMENT BY GENOME-WISE INSERT CAPTURE SEQUENCING; U.S. application 62/096,761, 24 Dec. 14, ENGINEERING OF SYSTEMS, METHODS AND OPTIMIZED ENZYME AND GUIDE SCAFFOLDS FOR SEQUENCE MANIPULATION; U.S. application 62/098,059, 30 Dec. 14, RNA-TARGETING SYSTEM; U.S. application 62/096,656, 24 Dec. 14, CRISPR HAVING OR ASSOCIATED WITH DESTABILIZATION DOMAINS; U.S. application 62/096,697, 24 Dec. 14, CRISPR HAVING OR ASSOCIATED WITH AAV; U.S. application 62/098,158, 30 Dec. 14, ENGINEERED CRISPR COMPLEX INSERTIONAL TARGETING SYSTEMS; U.S. application 62/151,052, 22 Apr. 15, CELLULAR TARGETING FOR EXTRACELLULAR EXOSOMAL REPORTING; U.S. application 62/054,490, 24 Sep. 14, DELIVERY, USE AND THERAPEUTIC APPLICATIONS OF THE CRISPR-CAS SYSTEMS AND COMPOSITIONS FOR TARGETING DISORDERS AND DISEASES USING PARTICLE DELIVERY COMPONENTS; U.S. application 62/055,484, 25 Sep. 14, SYSTEMS, METHODS AND COMPOSITIONS FOR SEQUENCE MANIPULATION WITH OPTIMIZED FUNCTIONAL CRISPR-CAS SYSTEMS; U.S. application 62/087,537, 4 Dec. 14, SYSTEMS, METHODS AND COMPOSITIONS FOR SEQUENCE MANIPULATION WITH OPTIMIZED FUNCTIONAL CRISPR-CAS SYSTEMS; U.S. application 62/054,651, 24 Sep. 14, DELIVERY, USE AND THERAPEUTIC APPLICATIONS OF THE CRISPR-CAS SYSTEMS AND COMPOSITIONS FOR MODELING COMPETITION OF MULTIPLE CANCER MUTATIONS IN VIVO; U.S. application 62/067,886, 23 Oct. 14, DELIVERY, USE AND THERAPEUTIC APPLICATIONS OF THE CRISPR-CAS SYSTEMS AND COMPOSITIONS FOR MODELING COMPETITION OF MULTIPLE CANCER MUTATIONS IN VIVO; U.S. application 62/054,675, 24 Sep. 14, DELIVERY, USE AND THERAPEUTIC APPLICATIONS OF THE CRISPR-CAS SYSTEMS AND COMPOSITIONS IN NEURONAL CELLS/TISSUES; U.S. application 62/054,528, 24 Sep. 14, DELIVERY, USE AND THERAPEUTIC APPLICATIONS OF THE CRISPR-CAS SYSTEMS AND COMPOSITIONS IN IMMUNE DISEASES OR DISORDERS; U.S. application 62/055,454, 25 Sep.14, DELIVERY, USE AND THERAPEUTIC APPLICATIONS OF THE CRISPR-CAS SYSTEMS AND COMPOSITIONS FOR TARGETING DISORDERS AND DISEASES USING CELL PENETRATION PEPTIDES (CPP); U.S. application 62/055,460, 25 Sep14, MULTIFUNCTIONAL-CRISPR COMPLEXES AND/OR OPTIMIZED ENZYME LINKED FUNCTIONAL-CRISPR COMPLEXES; U.S. application 62/087,475, 4 Dec 14, FUNCTIONAL SCREENING WITH OPTIMIZED FUNCTIONAL CRISPR-CAS SYSTEMS; U.S. application 62/055,487, 25 Sep 14, FUNCTIONAL SCREENING WITH OPTIMIZED FUNCTIONAL CRISPR-CAS SYSTEMS; U.S. application 62/087,546, 4 Dec 14, MULTIFUNCTIONAL CRISPR COMPLEXES AND/OR OPTIMIZED ENZYME LINKED FUNCTIONAL-CRISPR COMPLEXES; and U.S. application 62/098,285, 30 Dec. 14, CRISPR MEDIATED IN VIVO MODELING AND GENETIC SCREENING OF TUMOR GROWTH AND METASTASIS.</div>
</li> <li> <para-num num="[0199]"> </para-num> <div class="description-line" id="p-0200" num="0199">Each of these patents, patent publications, and applications, and all documents cited therein or during their prosecution (“appln cited documents”) and all documents cited or referenced in the appln cited documents, together with any instructions, descriptions, product specifications, and product sheets for any products mentioned therein or in any document therein and incorporated by reference herein, are hereby incorporated herein by reference, and may be employed in the practice of the invention. All documents (e.g., these patents, patent publications and applications and the appln cited documents) are incorporated herein by reference to the same extent as if each individual document was specifically and individually indicated to be incorporated by reference.</div>
</li> <li> <para-num num="[0200]"> </para-num> <div class="description-line" id="p-0201" num="0200">Also with respect to general information on CRISPR-Cas Systems, mention is made of the following (also hereby incorporated herein by reference):
</div> </li> <ul> <li id="ul0001-0001" num="0000"> <ul> <li id="ul0002-0001" num="0201">Multiplex genome engineering using CRISPR/Cas systems. Cong, L., Ran, F. A., Cox, D., Lin, S., Barretto, R., Habib, N., Hsu, P. D., Wu, X., Jiang, W., Marraffini, L. A., &amp; Zhang, F. Science February 15; 339(6121):819-23 (2013);</li> <li id="ul0002-0002" num="0202">RNA-guided editing of bacterial genomes using CRISPR-Cas systems. Jiang W., Bikard D., Cox D., Zhang F, Marraffini L A. Nat Biotechnol March;31(3):233-9 (2013);</li> <li id="ul0002-0003" num="0203">One-Step Generation of Mice Carrying Mutations in Multiple Genes by CRISPR/Cas-Mediated Genome Engineering. Wang H., Yang H., Shivalila C S., Dawlaty M M., Cheng A W., Zhang F., Jaenisch R. Cell May 9; 153(4):910-8 (2013);</li> <li id="ul0002-0004" num="0204">Optical control of mammalian endogenous transcription and epigenetic states. Konermann S, Brigham M D, Trevino A E, Hsu P D, Heidenreich M, Cong L, Platt R J, Scott D A, Church G M, Zhang F. <i>Nature. August </i>22; 500(7463):472-6. doi:10.1038/Nature12466. Epub 2013 August 23 (2013);</li> <li id="ul0002-0005" num="0205">Double Nicking by RNA-Guided CRISPR Cas9 for Enhanced Genome Editing Specificity. Ran, F A., Hsu, P D., Lin, C Y., Gootenberg, J S., Konermann, S., Trevino, AE., Scott, D A., Inoue, A., Matoba, S., Zhang, Y., &amp; Zhang, F. Cell August 28. pii: S0092-8674(13)01015-5 (2013-A);</li> <li id="ul0002-0006" num="0206">DNA targeting specificity of RNA-guided Cas9 nucleases. Hsu, P., Scott, D., Weinstein, J., Ran, F A., Konermann, S., Agarwala, V., Li, Y., Fine, E., Wu, X., Shalem, O., Cradick, T J., Marraffini, L A., Bao, G., &amp; Zhang, F. Nat Biotechnol doi:10.1038/nbt.2647 (2013);</li> <li id="ul0002-0007" num="0207">Genome engineering using the CRISPR-Cas9 system. Ran, F A., Hsu, P D., Wright, J., Agarwala, V., Scott, D A., Zhang, F. Nature Protocols November; 8(11):2281-308 (2013-B);</li> <li id="ul0002-0008" num="0208">Genome-Scale CRISPR-Cas9 Knockout Screening in Human Cells. Shalem, O., Sanjana, N E., Hartenian, E., Shi, X., Scott, D A., Mikkelson, T., Heckl, D., Ebert, B L., Root, D E., Doench, J G., Zhang, F. Science December 12. (2013). [Epub ahead of print];</li> <li id="ul0002-0009" num="0209">Crystal structure of cas9 in complex with guide RNA and target DNA. Nishimasu, H., Ran, F A., Hsu, P D., Konermann, S., Shehata, S I., Dohmae, N., Ishitani, R., Zhang, F., Nureki, O. Cell February 27, 156(5):935-49 (2014);</li> <li id="ul0002-0010" num="0210">Genome-wide binding of the CRISPR endonuclease Cas9 in mammalian cells. Wu X., Scott D A., Kriz A J., Chiu A C., Hsu P D., Dadon D B., Cheng A W., Trevino A E., Konermann S., Chen S., Jaenisch R., Zhang F., Sharp P A. Nat Biotechnol. April 20. doi: 10.1038/nbt.2889 (2014);</li> <li id="ul0002-0011" num="0211">CRISPR-Cas9 Knockin Mice for Genome Editing and Cancer Modeling. Platt R J, Chen S, Zhou Y, Yim M J, Swiech L, Kempton H R, Dahlman J E, Parnas O, Eisenhaure T M, Jovanovic M, Graham D B, Jhunjhunwala S, Heidenreich M, Xavier R J, Langer R, Anderson D G, Hacohen N, Regev A, Feng G, Sharp P A, Zhang F. Cell 159(2): 440-455 DOI: 10.1016/j.cell.2014.09.014(2014);</li> <li id="ul0002-0012" num="0212">Development and Applications of CRISPR-Cas9 for Genome Engineering, Hsu P D, Lander E S, Zhang F., Cell. June 5; 157(6):1262-78 (2014).</li> <li id="ul0002-0013" num="0213">Genetic screens in human cells using the CRISPR/Cas9 system, Wang T, Wei J J, Sabatini D M, Lander E S., Science. January 3; 343(6166): 80-84. doi:10.1126/science.1246981 (2014);</li> <li id="ul0002-0014" num="0214">Rational design of highly active sgRNAs for CRISPR-Cas9-mediated gene inactivation, Doench J G, Hartenian E, Graham D B, Tothova Z, Hegde M, Smith I, Sullender M, Ebert B L, Xavier R J, Root D E., (published online 3 Sep. 2014) Nat Biotechnol. December; 32(12):1262-7 (2014);</li> <li id="ul0002-0015" num="0215">In vivo interrogation of gene function in the mammalian brain using CRISPR-Cas9, Swiech L, Heidenreich M, Banerjee A, Habib N, Li Y, Trombetta J, Sur M, Zhang F., (published online 19 Oct. 2014) Nat Biotechnol. January; 33(1):102-6 (2015);</li> <li id="ul0002-0016" num="0216">Genome-scale transcriptional activation by an engineered CRISPR-Cas9 complex, Konermann S, Brigham M D, Trevino A E, Joung J, Abudayyeh OO, Barcena C, Hsu P D, Habib N, Gootenberg J S, Nishimasu H, Nureki O, Zhang F., Nature. January 29; 517(7536):583-8 (2015).</li> <li id="ul0002-0017" num="0217">A split-Cas9 architecture for inducible genome editing and transcription modulation, Zetsche B, Volz S E, Zhang F., (published online 02 Feb. 2015) Nat Biotechnol. February; 33(2):139-42 (2015);</li> <li id="ul0002-0018" num="0218">Genome-wide CRISPR Screen in a Mouse Model of Tumor Growth and Metastasis, Chen S, Sanjana N E, Zheng K, Shalem O, Lee K, Shi X, Scott D A, Song J, Pan J Q, Weissleder R, Lee H, Zhang F, Sharp P A. Cell 160, 1246-1260, Mar. 12, 2015 (multiplex screen in mouse), and</li> <li id="ul0002-0019" num="0219">In vivo genome editing using <i>Staphylococcus aureus </i>Cas9, Ran F A, Cong L, Yan W X, Scott D A, Gootenberg J S, Kriz A J, Zetsche B, Shalem O, Wu X, Makarova K S, Koonin E V, Sharp P A, Zhang F., (published online 01 Apr. 2015), Nature. April 9; 520(7546):186-91 (2015).</li> <li id="ul0002-0020" num="0220">Shalem et al., “High-throughput functional genomics using CRISPR-Cas9,” Nature</li> </ul> </li> </ul>
<li> <para-num num="[0221]"> </para-num> <div class="description-line" id="p-0202" num="0221">Reviews Genetics 16, 299-311 (May 2015).
</div> </li> <ul> <li id="ul0003-0001" num="0000"> <ul> <li id="ul0004-0001" num="0222">Xu et al., “Sequence determinants of improved CRISPR sgRNA design,” Genome Research 25, 1147-1157 (August 2015).</li> <li id="ul0004-0002" num="0223">Parnas et al., “A Genome-wide CRISPR Screen in Primary Immune Cells to Dissect Regulatory Networks,” Cell 162, 675-686 (Jul. 30, 2015).</li> <li id="ul0004-0003" num="0224">Ramanan et al., CRISPR/Cas9 cleavage of viral DNA efficiently suppresses hepatitis B virus,” Scientific Reports 5:10833. doi: 10.1038/srep10833 (Jun. 2, 2015)</li> <li id="ul0004-0004" num="0225">Nishimasu et al., Crystal Structure of Staphylococcus aureus Cas9,” Cell 162, 1113-1126 (Aug. 27, 2015)</li> <li id="ul0004-0005" num="0226">Zetsche et al., “Cpf1 Is a Single RNA-Guided Endonuclease of a Class 2 CRISPR-Cas System, ” Cell 163, 1-13 (Oct. 22, 2015)</li> <li id="ul0004-0006" num="0227">Shmakov et al., “Discovery and Functional Characterization of Diverse Class 2 CRISPR-Cas Systems,” Molecular Cell 60, 1-13 (Available online Oct. 22, 2015)
<br/>
each of which is incorporated herein by reference, may be considered in the practice of the instant invention, and discussed briefly below:
</li> <li id="ul0004-0007" num="0228">Cong et al. engineered type II CRISPR-Cas systems for use in eukaryotic cells based on both Streptococcus thermophilus Cas9 and also Streptococcus pyogenes Cas9 and demonstrated that Cas9 nucleases can be directed by short RNAs to induce precise cleavage of DNA in human and mouse cells. Their study further showed that Cas9 as converted into a nicking enzyme can be used to facilitate homology-directed repair in eukaryotic cells with minimal mutagenic activity. Additionally, their study demonstrated that multiple guide sequences can be encoded into a single CRISPR array to enable simultaneous editing of several at endogenous genomic loci sites within the mammalian genome, demonstrating easy programmability and wide applicability of the RNA-guided nuclease technology. This ability to use RNA to program sequence specific DNA cleavage in cells defined a new class of genome engineering tools. These studies further showed that other CRISPR loci are likely to be transplantable into mammalian cells and can also mediate mammalian genome cleavage. Importantly, it can be envisaged that several aspects of the CRISPR-Cas system can be further improved to increase its efficiency and versatility.</li> <li id="ul0004-0008" num="0229">Jiang et al. used the clustered, regularly interspaced, short palindromic repeats (CRISPR)-associated Cas9 endonuclease complexed with dual-RNAs to introduce precise mutations in the genomes of <i>Streptococcus pneumoniae </i>and <i>Escherichia coli. </i>The approach relied on dual-RNA:Cas9-directed cleavage at the targeted genomic site to kill unmutated cells and circumvents the need for selectable markers or counter-selection systems. The study reported reprogramming dual-RNA:Cas9 specificity by changing the sequence of short CRISPR RNA (crRNA) to make single- and multinucleotide changes carried on editing templates. The study showed that simultaneous use of two crRNAs enabled multiplex mutagenesis. Furthermore, when the approach was used in combination with recombineering, in <i>S. pneumoniae, </i>nearly 100% of cells that were recovered using the described approach contained the desired mutation, and in <i>E. coli, </i>65% that were recovered contained the mutation.</li> <li id="ul0004-0009" num="0230">Wang et al. (2013) used the CRISPR/Cas system for the one-step generation of mice carrying mutations in multiple genes which were traditionally generated in multiple steps by sequential recombination in embryonic stem cells and/or time-consuming intercrossing of mice with a single mutation. The CRISPR/Cas system will greatly accelerate the in vivo study of functionally redundant genes and of epistatic gene interactions.</li> <li id="ul0004-0010" num="0231">Konermann et al. (2013) addressed the need in the art for versatile and robust technologies that enable optical and chemical modulation of DNA-binding domains based CRISPR Cas9 enzyme and also Transcriptional Activator Like Effectors</li> <li id="ul0004-0011" num="0232">Ran et al. (2013-A) described an approach that combined a Cas9 nickase mutant with paired guide RNAs to introduce targeted double-strand breaks. This addresses the issue of the Cas9 nuclease from the microbial CRISPR-Cas system being targeted to specific genomic loci by a guide sequence, which can tolerate certain mismatches to the DNA target and thereby promote undesired off-target mutagenesis. Because individual nicks in the genome are repaired with high fidelity, simultaneous nicking via appropriately offset guide RNAs is required for double-stranded breaks and extends the number of specifically recognized bases for target cleavage. The authors demonstrated that using paired nicking can reduce off-target activity by 50- to 1,500-fold in cell lines and to facilitate gene knockout in mouse zygotes without sacrificing on-target cleavage efficiency. This versatile strategy enables a wide variety of genome editing applications that require high specificity.</li> <li id="ul0004-0012" num="0233">Hsu et al. (2013) characterized SpCas9 targeting specificity in human cells to inform the selection of target sites and avoid off-target effects. The study evaluated &gt;700 guide RNA variants and SpCas9-induced indel mutation levels at &gt;100 predicted genomic off-target loci in 293T and 293FT cells. The authors that SpCas9 tolerates mismatches between guide RNA and target DNA at different positions in a sequence-dependent manner, sensitive to the number, position and distribution of mismatches. The authors further showed that SpCas9-mediated cleavage is unaffected by DNA methylation and that the dosage of SpCas9 and sgRNA can be titrated to minimize off-target modification. Additionally, to facilitate mammalian genome engineering applications, the authors reported providing a web-based software tool to guide the selection and validation of target sequences as well as off-target analyses.</li> <li id="ul0004-0013" num="0234">Ran et al. (2013-B) described a set of tools for Cas9-mediated genome editing via non-homologous end joining (NHEJ) or homology-directed repair (HDR) in mammalian cells, as well as generation of modified cell lines for downstream functional studies. To minimize off-target cleavage, the authors further described a double-nicking strategy using the Cas9 nickase mutant with paired guide RNAs. The protocol provided by the authors experimentally derived guidelines for the selection of target sites, evaluation of cleavage efficiency and analysis of off-target activity. The studies showed that beginning with target design, gene modifications can be achieved within as little as 1-2 weeks, and modified clonal cell lines can be derived within 2-3 weeks.</li> <li id="ul0004-0014" num="0235">Shalem et al. described a new way to interrogate gene function on a genome-wide scale.</li> </ul> </li> </ul>
<li> <para-num num="[0236]"> </para-num> <div class="description-line" id="p-0203" num="0236">Their studies showed that delivery of a genome-scale CRISPR-Cas9 knockout (GeCKO) library targeted 18,080 genes with 64,751 unique guide sequences enabled both negative and positive selection screening in human cells. First, the authors showed use of the GeCKO library to identify genes essential for cell viability in cancer and pluripotent stem cells. Next, in a melanoma model, the authors screened for genes whose loss is involved in resistance to vemurafenib, a therapeutic that inhibits mutant protein kinase BRAF. Their studies showed that the highest-ranking candidates included previously validated genes NF1 and MED12 as well as novel hits NF2, CUL3, TADA2B, and TADA1. The authors observed a high level of consistency between independent guide RNAs targeting the same gene and a high rate of hit confirmation, and thus demonstrated the promise of genome-scale screening with Cas9.
</div> </li> <ul> <li id="ul0005-0001" num="0000"> <ul> <li id="ul0006-0001" num="0237">Nishimasu et al. reported the crystal structure of Streptococcus pyogenes Cas9 in complex with sgRNA and its target DNA at 2.5 A° resolution. The structure revealed a bilobed architecture composed of target recognition and nuclease lobes, accommodating the sgRNA:DNA heteroduplex in a positively charged groove at their interface. Whereas the recognition lobe is essential for binding sgRNA and DNA, the nuclease lobe contains the HNH and RuvC nuclease domains, which are properly positioned for cleavage of the complementary and non-complementary strands of the target DNA, respectively. The nuclease lobe also contains a carboxyl-terminal domain responsible for the interaction with the protospacer adjacent motif (PAM). This high-resolution structure and accompanying functional analyses have revealed the molecular mechanism of RNA-guided DNA targeting by Cas9, thus paving the way for the rational design of new, versatile genome-editing technologies.</li> <li id="ul0006-0002" num="0238">Wu et al. mapped genome-wide binding sites of a catalytically inactive Cas9 (dCas9) from Streptococcus pyogenes loaded with single guide RNAs (sgRNAs) in mouse embryonic stem cells (mESCs). The authors showed that each of the four sgRNAs tested targets dCas9 to between tens and thousands of genomic sites, frequently characterized by a 5-nucleotide seed region in the sgRNA and an NGG protospacer adjacent motif (PAM). Chromatin inaccessibility decreases dCas9 binding to other sites with matching seed sequences; thus 70% of off-target sites are associated with genes. The authors showed that targeted sequencing of 295 dCas9 binding sites in mESCs transfected with catalytically active Cas9 identified only one site mutated above background levels. The authors proposed a two-state model for Cas9 binding and cleavage, in which a seed match triggers binding but extensive pairing with target DNA is required for cleavage.</li> <li id="ul0006-0003" num="0239">Platt et al. established a Cre-dependent Cas9 knockin mouse. The authors demonstrated in vivo as well as ex vivo genome editing using adeno-associated virus (AAV)-, lentivirus-, or particle-mediated delivery of guide RNA in neurons, immune cells, and endothelial cells.</li> <li id="ul0006-0004" num="0240">Hsu et al. (2014) is a review article that discusses generally CRISPR-Cas9 history from yogurt to genome editing, including genetic screening of cells.</li> <li id="ul0006-0005" num="0241">Wang et al. (2014) relates to a pooled, loss-of-function genetic screening approach suitable for both positive and negative selection that uses a genome-scale lentiviral single guide RNA (sgRNA) library.</li> <li id="ul0006-0006" num="0242">Doench et al. created a pool of sgRNAs, tiling across all possible target sites of a panel of six endogenous mouse and three endogenous human genes and quantitatively assessed their ability to produce null alleles of their target gene by antibody staining and flow cytometry. The authors showed that optimization of the PAM improved activity and also provided an on-line tool for designing sgRNAs.</li> <li id="ul0006-0007" num="0243">Swiech et al. demonstrate that AAV-mediated SpCas9 genome editing can enable reverse genetic studies of gene function in the brain.</li> <li id="ul0006-0008" num="0244">Konermann et al. (2015) discusses the ability to attach multiple effector domains, e.g., transcriptional activator, functional and epigenomic regulators at appropriate positions on the guide such as stem or tetraloop with and without linkers.</li> <li id="ul0006-0009" num="0245">Zetsche et al. demonstrates that the Cas9 enzyme can be split into two and hence the assembly of Cas9 for activation can be controlled.</li> <li id="ul0006-0010" num="0246">Chen et al. relates to multiplex screening by demonstrating that a genome-wide in vivo CRISPR-Cas9 screen in mice reveals genes regulating lung metastasis.</li> <li id="ul0006-0011" num="0247">Ran et al. (2015) relates to SaCas9 and its ability to edit genomes and demonstrates that one cannot extrapolate from biochemical assays.</li> <li id="ul0006-0012" num="0248">Shalem et al. (2015) described ways in which catalytically inactive Cas9 (dCas9) fusions are used to synthetically repress (CRISPRi) or activate (CRISPRa) expression, showing. advances using Cas9 for genome-scale screens, including arrayed and pooled screens, knockout approaches that inactivate genomic loci and strategies that modulate transcriptional activity.</li> <li id="ul0006-0013" num="0249">Xu et al. (2015) assessed the DNA sequence features that contribute to single guide RNA (sgRNA) efficiency in CRISPR-based screens. The authors explored efficiency of CRISPR/Cas9 knockout and nucleotide preference at the cleavage site. The authors also found that the sequence preference for CRISPRi/a is substantially different from that for CRISPR/Cas9 knockout.</li> <li id="ul0006-0014" num="0250">Parnas et al. (2015) introduced genome-wide pooled CRISPR-Cas9 libraries into dendritic cells (DCs) to identify genes that control the induction of tumor necrosis factor (Tnf) by bacterial lipopolysaccharide (LPS). Known regulators of Tlr4 signaling and previously unknown candidates were identified and classified into three functional modules with distinct effects on the canonical responses to LPS.</li> <li id="ul0006-0015" num="0251">Ramanan et al (2015) demonstrated cleavage of viral episomal DNA (cccDNA) in infected cells. The HBV genome exists in the nuclei of infected hepatocytes as a 3.2 kb double-stranded episomal DNA species called covalently closed circular DNA (cccDNA), which is a key component in the HBV life cycle whose replication is not inhibited by current therapies. The authors showed that sgRNAs specifically targeting highly conserved regions of HBV robustly suppresses viral replication and depleted cccDNA.</li> <li id="ul0006-0016" num="0252">Nishimasu et al. (2015) reported the crystal structures of SaCas9 in complex with a single guide RNA (sgRNA) and its double-stranded DNA targets, containing the 5′-TTGAAT-3′ PAM and the 5′-TTGGGT-3′ PAM. A structural comparison of SaCas9 with SpCas9 highlighted both structural conservation and divergence, explaining their distinct PAM specificities and orthologous sgRNA recognition.</li> <li id="ul0006-0017" num="0253">Zetsche et al. (2015) reported the characterization of Cpf1, a putative class 2 CRISPR effector. It was demonstrated that Cpf1 mediates robust DNA interference with features distinct from Cas9. Identifying this mechanism of interference broadens our understanding of CRISPR-Cas systems and advances their genome editing applications.</li> <li id="ul0006-0018" num="0254">Shmakov et al. (2015) reported the characterization of three distinct Class 2 CRISPR-Cas systems. The effectors of two of the identified systems, C2c1 and C2c3, contain RuvC like endonuclease domains distantly related to Cpf1. The third system, C2c2, contains an effector with two predicted HEPN RNase domains.</li> </ul> </li> </ul>
<li> <para-num num="[0255]"> </para-num> <div class="description-line" id="p-0204" num="0255">Also, “Dimeric CRISPR RNA-guided Fokl nucleases for highly specific genome editing”, Shengdar Q. Tsai, Nicolas Wyvekens, Cyd Khayter, Jennifer A. Foden, Vishal Thapar, Deepak Reyon, Mathew J. Goodwin, Martin J. Aryee, J. Keith Joung Nature Biotechnology 32(6): 569-77 (2014), relates to dimeric RNA-guided Fokl Nucleases that recognize extended sequences and can edit endogenous genes with high efficiencies in human cells.</div>
</li> <li> <para-num num="[0256]"> </para-num> <div class="description-line" id="p-0205" num="0256">Useful in the practice of the instant invention, reference is made to the article entitled BCL11A enhancer dissection by Cas9-mediated in situ saturating mutagenesis. Canver, M. C., Smith, E. C., Sher, F., Pinello, L., Sanjana, N. E., Shalem, O., Chen, D. D., Schupp, P. G., Vinjamur, D. S., Garcia, S. P., Luc, S., Kurita, R., Nakamura, Y., Fujiwara, Y., Maeda, T., Yuan, G., Zhang, F., Orkin, S. H., &amp; Bauer, D. E. DOI:10.1038/nature15521, published online Sep. 16, 2015, the article is herein incorporated by reference and discussed briefly below:</div>
</li> <li> <para-num num="[0257]"> </para-num> <div class="description-line" id="p-0206" num="0257">Canver et al. describes novel pooled CRISPR-Cas9 guide RNA libraries to perform in situ saturating mutagenesis of the human and mouse BCL11A erythroid enhancers previously identified as an enhancer associated with fetal hemoglobin (HbF) level and whose mouse ortholog is necessary for erythroid BCL11A expression. This approach revealed critical minimal features and discrete vulnerabilities of these enhancers. Through editing of primary human progenitors and mouse transgenesis, the authors validated the BCL11A erythroid enhancer as a target for HbF reinduction. The authors generated a detailed enhancer map that informs therapeutic genome editing.</div>
</li> <li> <para-num num="[0258]"> </para-num> <div class="description-line" id="p-0207" num="0258">Reference is made to Zetsche et al., “Cpf1 Is a Single RNA-Guided Endonuclease of a Class 2 CRISPR-Cas System, ” Cell 163, 1-13 (Oct. 22, 2015) and Shmakov et al., “Discovery and Functional Characterization of Diverse Class 2 CRISPR-Cas Systems,” Molecular Cell 60, 1-13 (Available online Oct. 22, 2015). Zetsche et al. (2015) reported the characterization of Cpf1, a putative class 2 CRISPR effector. It was demonstrated that Cpf1 mediates robust DNA interference with features distinct from Cas9. Identifying this mechanism of interference broadens our understanding of CRISPR-Cas systems and advances their genome editing applications. In certain embodiments, perturb-seq uses Cpf1 as the CRISPR enzyme for introducing perturbations. Not being bound by a theory, Cpf1 does not require Tracr RNA and is a smaller enzyme, thus allowing higher combinatorial perturbations to be tested and for more efficient studies.</div>
</li> <li> <para-num num="[0259]"> </para-num> <div class="description-line" id="p-0208" num="0259">Shmakov et al. (2015) reported the characterization of three distinct Class 2 CRISPR-Cas systems. The effectors of two of the identified systems, C2c1 and C2c3, contain RuvC like endonuclease domains distantly related to Cpf1. The third system, C2c2, contains an effector with two predicted HEPN RNase domains. Mentino is also made of “Dimeric CRISPR RNA-guided Fokl nucleases for highly specific genome editing”, Shengdar Q. Tsai, Nicolas Wyvekens, Cyd Khayter, Jennifer A. Foden, Vishal Thapar, Deepak Reyon, Mathew J. Goodwin, Martin J. Aryee, J. Keith Joung Nature Biotechnology 32(6): 569-77 (2014), which relates to dimeric RNA-guided Fokl Nucleases that recognize extended sequences and can edit endogenous genes with high efficiencies in human cells. In addition, mention is made of PCT application PCT/US14/70057, Attorney Reference 47627.99.2060 and BI-2013/107 entitiled “DELIVERY, USE AND THERAPEUTIC APPLICATIONS OF THE CRISPR-CAS SYSTEMS AND COMPOSITIONS FOR TARGETING DISORDERS AND DISEASES USING PARTICLE DELIVERY COMPONENTS (claiming priority from one or more or all of U.S. provisional patent applications: 62/054,490, filed Sep. 24, 2014; 62/010,441, filed Jun. 10, 2014; and 61/915,118, 61/915,215 and 61/915,148, each filed on Dec. 12, 2013) (”the Particle Delivery PCT″), incorporated herein by reference, with respect to a method of preparing an sgRNA-and-Cas9 protein containing particle comprising admixing a mixture comprising an sgRNA and Cas9 protein (and optionally HDR template) with a mixture comprising or consisting essentially of or consisting of surfactant, phospholipid, biodegradable polymer, lipoprotein and alcohol; and particles from such a process. For example, wherein Cas9 protein and sgRNA were mixed together at a suitable, e.g., 3:1 to 1:3 or 2:1 to 1:2 or 1:1 molar ratio, at a suitable temperature, e.g., 15-30C, e.g., 20-25C, e.g., room temperature, for a suitable time, e.g., 15-45, such as 30 minutes, advantageously in sterile, nuclease free buffer, e.g., 1X PBS. Separately, particle components such as or comprising: a surfactant, e.g., cationic lipid, e.g., 1,2-dioleoyl-3-trimethylammonioum-propane (DOTAP); phospholipid, e.g., dimyristoylphosphatidylcholine (DMPC); biodegradable polymer, such as an ethylene-glycol polymer or PEG, and a lipoprotein, such as a low-density lipoprotein, e.g., cholesterol were dissolved in an alcohol, advantageously a C1-6 alkyl alcohol, such as methanol, ethanol, isopropanol, e.g., 100% ethanol. The two solutions were mixed together to form particles containing the Cas9-sgRNA complexes. Accordingly, sgRNA may be pre-complexed with the Cas9 protein, before formulating the entire complex in a particle. Formulations may be made with a different molar ratio of different components known to promote delivery of nucleic acids into cells (e.g. 1,2-dioleoyl-3-trimethylammonium-propane (DOTAP), 1,2-ditetradecanoyl-sn-glycero-3-phosphocholine (DMPC), polyethylene glycol (PEG), and cholesterol) For example DOTAP: DMPC: PEG: Cholesterol Molar Ratios may be DOTAP 100, DMPC 0, PEG 0, Cholesterol 0; or DOTAP 90, DMPC 0, PEG 10, Cholesterol 0; or DOTAP 90, DMPC 0, PEG 5, Cholesterol 5. DOTAP 100, DMPC 0, PEG 0, Cholesterol 0. That application accordingly comprehends admixing sgRNA, Cas9 protein and components that form a particle; as well as particles from such admixing. Aspects of the instant invention can involve particles; for example, particles using a process analogous to that of the Particle Delivery PCT, e.g., by admixing a mixture comprising sgRNA and/or Cas9 as in the instant invention and components that form a particle, e.g., as in the Particle Delivery PCT, to form a particle and particles from such admixing (or, of course, other particles involving sgRNA and/or Cas9 as in the instant invention). These and other CRISPR-Cas or CRISPR systems can be used in the practice of the invention. Other Perturbations</div>
</li> <li> <para-num num="[0260]"> </para-num> <div class="description-line" id="p-0209" num="0260">The invention also involves perturbing by subjecting the cell to an increase or decrease in temperature. The temperature may range from about 0° C. to about 100° C., advantageously about 10° C., 15° C., 20° C., 25° C., 30° C., 35° C., 40° C., 45° C., 50° C., 55° C., 60° C., 65° C., 70° C., 75° C., 80° C., 85° C., 90° C., 95° C. or 100° C. In another embodiment, the temperature may be closer to a physiological temperature, e.g., about 30° C., 31° C., 32° C., 33° C., 34° C., 35° C., 36° C., 37° C., 38° C., 39° C. or 40° C.</div>
</li> <li> <para-num num="[0261]"> </para-num> <div class="description-line" id="p-0210" num="0261">The invention also involves perturbing by subjecting the cell to a chemical agent. Samples of chemical agents include, but are not limited to, an antibiotic, a small molecule, a hormone, a hormone derivative, a steroid or a steroid derivative.</div>
</li> <li> <para-num num="[0262]"> </para-num> <div class="description-line" id="p-0211" num="0262">The invention also involves perturbing by subjecting the cell to a biological agent. Examples of biological agents may be, but are not limited to cytokines and Toll-like receptor agonists (Amit et al., Science. 2009 9;326(5950):257-63, Chevrier et al., Cell. 2011 Nov. 11; 147(4):853-67); Wang et al., Cell. 2015 Dec. 3; 163(6):1413-27; and Gaublomme et al., Cell. 2015 Dec. 3; 163(6):1400-12).</div>
</li> <li> <para-num num="[0263]"> </para-num> <div class="description-line" id="p-0212" num="0263">In one aspect of the invention the perturbing may be with an energy source such as electromagnetic energy or ultrasound. The electromagnetic energy may be a component of visible light having a wavelength in the range of 450 nm-700 nm. In a preferred embodiment the component of visible light may have a wavelength in the range of 450 nm-500 nm and may be blue light. The blue light may have an intensity of at least 0.2mW/cm<sup>2</sup>, or more preferably at least 4mW/cm<sup>2</sup>. In another embodiment, the component of visible light may have a wavelength in the range of 620-700 nm and is red light.</div>
</li> <li> <para-num num="[0264]"> </para-num> <div class="description-line" id="p-0213" num="0264">The invention also involves perturbing by subjecting the cell to a chemical agent and/or temperature is across a gradient. A biomolecular gradient may be formed, for example, as reviewed in Keenan and Folch, Lab Chip. 2008 January; 8(1): doi:10.1039/b711887b. Biomolecule gradients have been shown to play roles in a wide range of biological processes including development, inflammation, wound healing, and cancer metastasis. Elucidation of these phenomena requires the ability to expose cells to biomolecule gradients that are quantifiable, controllable, and mimic those that are present in vivo.</div>
</li> <li> <para-num num="[0265]"> </para-num> <div class="description-line" id="p-0214" num="0265">A chemical gradient may be formed without requiring fluid flow (see, e.g., Abhyankar et al., Lab Chip, 2006, 6, 389-393). This device consists of a membrane-covered source region and a large volume sink region connected by a microfluidic channel. The high fluidic resistance of the membrane limits fluid flow caused by pressure differences in the system, but allows diffusive transport of a chemical species through the membrane and into the channel. The large volume sink region at the end of the microfluidic channel helps to maintain spatial and temporal stability of the gradient. The chemical gradient in a 0.5 mm region near the sink region experiences a maximum of 10 percent change between the 6 and 24 h data points. Abhyankar et al., Lab Chip, 2006, 6, 389-393 present the theory, design, and characterization of this device and provide an example of neutrophil chemotaxis as proof of concept for future quantitative cell-signaling applications.</div>
</li> <li> <para-num num="[0266]"> </para-num> <div class="description-line" id="p-0215" num="0266">In another embodiment, a gradient may also be introduced with nanowires. In this embodiment, the nanowires do not necessarily introduce a gradient but may introduce other things into the system. A generalized platform for introducing a diverse range of biomolecules into living cells in high-throughput could transform how complex cellular processes are probed and analyzed. Shalek et al., PNAS|Feb. 2, 2010|vol. 107|no. 5 demonstrate spatially localized, efficient, and universal delivery of biomolecules into immortalized and primary mammalian cells using surface-modified vertical silicon nanowires. The method relies on the ability of the silicon nanowires to penetrate a cell's membrane and subsequently release surface-bound molecules directlyminto the cell's cytosol, thus allowing highly efficient delivery of biomolecules without chemical modification or viral packaging. This modality enables one to assess the phenotypic consequences of introducing a broad range of biological effectors (DNAs, RNAs, peptides, proteins, and small molecules) into almost any cell type. Shalek et al., PNAS|Feb. 2, 2010|vol. 107|no. 5 show that this platform can be used to guide neuronal progenitor growth with small molecules, knock down transcript levels by delivering siRNAs, inhibit apoptosis using peptides, and introduce targeted proteins to specific organelles. Shalek et al., PNAS|Feb. 2, 2010|vol. 107|no. 5 further demonstrate codelivery of siRNAs and proteins on a single substrate in a microarray format, highlighting this technology's potential as a robust, monolithic platform for high-throughput, miniaturized bioassays.</div>
</li> <li> <para-num num="[0267]"> </para-num> <div class="description-line" id="p-0216" num="0267">A gradient may be established, for example, in a fluidic device, such as a microfluidic device (see, e.g., Tehranirokh et al., BIOMICROFLUIDICS 7, 051502 (2013)). Microfluidic technology allows dynamic cell culture in microperfusion systems to deliver continuous nutrient supplies for long term cell culture. It offers many opportunities to mimic the cell-cell and cell-extracellular matrix interactions of tissues by creating gradient concentrations of biochemical signals such as growth factors, chemokines, and hormones. Other applications of cell cultivation in microfluidic systems include high resolution cell patterning on a modified substrate with adhesive patterns and the reconstruction of complicated tissue architectures. In the review of Tehranirokh et al., BIOMICROFLUIDICS 7, 051502 (2013), recent advances in microfluidic platforms for cell culturing and proliferation, for both simple monolayer (2D) cell seeding processes and 3D configurations as accurate models of in vivo conditions, are examined. In another embodiment, the fluidic device may be a controlled fluidic device as described below.</div>
</li> <heading id="h-0011">Device for Establishing a Gradient</heading>
<li> <para-num num="[0268]"> </para-num> <div class="description-line" id="p-0217" num="0268">The invention provides a controlled fluidic device for establishing a gradient, particularly a concentration gradient, which may comprise a closed chamber comprising one or more inlet port(s) that deliver two or more different fluids via separate inlet channels and at least one outlet port wherein the location of the one or more inlet port(s) and the at least one outlet port are located so that the flow of fluids can be controlled within the closed chamber and a gradient of the mixture of the two or more fluids from the two or more inlet ports is established.</div>
</li> <li> <para-num num="[0269]"> </para-num> <div class="description-line" id="p-0218" num="0269">In a related aspect, the invention provides a controlled fluidic device wherein the controlled fluidic device is a polygonal plate having an upper surface and a lower surface and a peripheral plate edge having a pre-determined depth. In an embodiment of the invention, the controlled fluidic device may comprise a device wherein at least one of the two fluids includes at least one component for which a gradient could be established.</div>
</li> <li> <para-num num="[0270]"> </para-num> <div class="description-line" id="p-0219" num="0270">In a related aspect, the invention provides a controlled fluidic device which may comprise at least component which includes at least two subcomponents. In an embodiment of the invention, the controlled fluidic device may comprise a closed chamber comprising a chip.</div>
</li> <li> <para-num num="[0271]"> </para-num> <div class="description-line" id="p-0220" num="0271">The invention provides a method of identifying altered chemical resistance in a bacterial population in the controlled fluidic device as described above, the method which may comprise synthesizing a mutant bacterial strain to express fluorescent proteins; introducing a known concentration of the bacterial strain into the closed chamber; administering the two or more different fluids into the closed chamber via the two more inlet ports; isolating DNA from a single cell; purifying DNA from bacteria; sequencing DNA from bacteria; preparing and sequencing a single composite sequence library; wherein wherein identification of alteration in level of expression compared to a baseline gene expression measurement of at least one biomarker is indicative of chemical resistance, and wherein the baseline gene expression measurement is the gene expression measured in the microfluidic well prior to administration of the two or more different fluids.</div>
</li> <li> <para-num num="[0272]"> </para-num> <div class="description-line" id="p-0221" num="0272">The present invention also provides a method of evaluating response in a cell population in the controlled fluidic device as described, the method which may comprise introducing a cell population into the closed chamber; administering the two or more different fluids into the closed chamber via the two more inlet ports such that a concentration gradient is established in the closed chamber; and, measuring the response of the cell population at various concentrations across the concentration gradient.</div>
</li> <li> <para-num num="[0273]"> </para-num> <div class="description-line" id="p-0222" num="0273">In another aspect, the invention provides a method of identifying altered bacterial populations according to the method of evaluating a response, the method comprising: a microfluidic device having a closed chamber having an upper surface and a lower surface and a peripheral plate edge having a predetermined depth; a plurality of microfluidic wells extending from the upper surface of the closed chamber, each well connected to adjacent ones of the plurality of wells by microchannels extending from the upper surface of the plate and extending from a first well to a second well such that the first well is in fluid communication with the second well; wherein the microfluidic device and plurality of wells connected by microchannels creates a chemical concentration gradient in adjacent microfluidic wells wherein one microfluidic well has a different chemical concentration than an adjacent microfluidic well; providing a chemical dye via an inlet port of the closed chamber; providing a chemical via an inlet port of the closed chamber; optionally providing a second chemical via an inlet port of the closed chamber; an outlet port of the closed chamber; and, a peripheral flow channel adjacent a portion of the peripheral plate edge and extending from the inlet port to the outlet port.</div>
</li> <li> <para-num num="[0274]"> </para-num> <div class="description-line" id="p-0223" num="0274">The present invention provides a method of identifying a compound associated with an altered bacterial population as described above, the method comprising: designing a combinatorial library wherein each member of the library comprises at least one pharmacophore associated with the altered gene expression; wherein alteration in level of expression compared to a baseline gene expression measurement of at least one biomarker is indicative of an altered bacterial population; synthesizing a plurality of compounds from said combinatorial library; and, screening said compounds for candidates associated with the altered bacterial population.</div>
</li> <li> <para-num num="[0275]"> </para-num> <div class="description-line" id="p-0224" num="0275">The invention provides an array of controlled microfluidic devices, comprising a plurality of controlled microfluidic devices according to the controlled fluidic device for establishing a gradient, comprising a closed chamber comprising one or more inlet port(s) that deliver two or more different fluids via separate inlet channels and at least one outlet port wherein the location of the one or more inlet port(s) and the at least one outlet port are located so that the flow of fluids can be controlled within the closed chamber and a gradient of the mixture of the two or more fluids from the two or more inlet ports is established.</div>
</li> <heading id="h-0012">Labelling</heading>
<li> <para-num num="[0276]"> </para-num> <div class="description-line" id="p-0225" num="0276">Methods of international patent publication no. WO2014047561 and US patent publication no. 2015/0259674 are contemplated in the present invention.</div>
</li> <li> <para-num num="[0277]"> </para-num> <div class="description-line" id="p-0226" num="0277">The invention also contemplates a labeling ligand which may comprise a unique perturbation identifier (UPI) sequence attached to a perturbation-sequence-capture sequence, and sequencing includes isolating via microbeads comprising a perturbation-sequence-capture-binding-sequence having specific binding affinity for the perturbation-sequence-capture sequence attached to the UPI sequence.</div>
</li> <li> <para-num num="[0278]"> </para-num> <div class="description-line" id="p-0227" num="0278">The UPI sequence may be attached to a universal ligation handle sequence, whereby a unique source identifier USI may be generated by split-pool ligation. The labeling ligand may comprise an oligonucleotide label which may comprise a regulatory sequence configured for amplification by T7 polymerase. The labeling ligands may comprise oligonucleotide sequences configured to hybridize to a transcript specific region. The labeling ligand may comprise an oligonucleotide label, wherein the oligonucleotide label may further comprise a photocleavable linker.</div>
</li> <li> <para-num num="[0279]"> </para-num> <div class="description-line" id="p-0228" num="0279">The oligonucleotide label may further comprise a restriction enzyme site between the labeling ligand and unique constituent identifier (UCI).</div>
</li> <li> <para-num num="[0280]"> </para-num> <div class="description-line" id="p-0229" num="0280">The method may comprise forming discrete unique-identifier-transfer compositions, each of which may comprise the cell and a transfer particle, wherein: (a) an oligonucleotide label further may comprise a capture sequence, and unique constituent identifier (UCI) and capture sequence are together releasably attached to the labeling ligand; the labelling ligand is bound to the target cellular constituent; and, (ca transfer particle may comprise: (i) a capture-binding-sequence having specific binding affinity for the capture sequence attached to the UCI, and, (ii)a unique source identifier (USI) sequence that is unique to each transfer particle.</div>
</li> <li> <para-num num="[0281]"> </para-num> <div class="description-line" id="p-0230" num="0281">In one embodiment, the USI may comprise 4-15 nucleotides.</div>
</li> <li> <para-num num="[0282]"> </para-num> <div class="description-line" id="p-0231" num="0282">In another embodiment, the invention may further comprise releasing the UCI from the labeled ligand, under conditions within the unique-identifier-transfer composition so that the released capture sequence binds to the capture-binding-sequence on the transfer particle, thereby transferring the UCI to the transfer particle.</div>
</li> <li> <para-num num="[0283]"> </para-num> <div class="description-line" id="p-0232" num="0283">In another embodiment, the ligation handle may comprise a restriction site for producing an overhang complementary with a first index sequence overhang, and wherein the method further comprises digestion with a restriction enzyme. In another embodiment, the ligation handle may comprise a nucleotide sequence complementary with a ligation primer sequence and wherein the overhang complementary with a first index sequence overhang is produced by hybridization of the ligation primer to the ligation handle.</div>
</li> <li> <para-num num="[0284]"> </para-num> <div class="description-line" id="p-0233" num="0284">In another embodiment, the invention may further comprise quantitating relative amount of UCI sequence associated with a first cell to the amount of the same UCI sequence associated with a second cell, whereby the relative differences of a cellular constituent between cell(s) are determined.</div>
</li> <li> <para-num num="[0285]"> </para-num> <div class="description-line" id="p-0234" num="0285">In another embodiment, the labeling ligand may comprise an antibody or an antibody fragment, such as but not limited to, a nanobody, Fab, Fab′, (Fab′)2, Fv, ScFv, diabody, triabody, tetrabody, Bis-scFv, minibody, Fab2, or Fab3 fragment.</div>
</li> <li> <para-num num="[0286]"> </para-num> <div class="description-line" id="p-0235" num="0286">In another embodiment, the labeling ligand may comprise an aptamer.</div>
</li> <li> <para-num num="[0287]"> </para-num> <div class="description-line" id="p-0236" num="0287">In another embodiment, the labeling ligand may comprise a nucleotide sequence complementary to a target sequence.</div>
</li> <li> <para-num num="[0288]"> </para-num> <div class="description-line" id="p-0237" num="0288">In another embodiment, the cell or a population includes wherein the cell(s) are a member of a cell population, and the method further comprises transforming or transducing the cell population with one or more genomic sequence-perturbation constructs that perturb a genomic sequence in the cells, wherein each distinct genomic sequence-perturbation construct comprises a unique-perturbation-identified (UPI) sequence unique to that construct. The genomic sequence-perturbation construct may comprises sequence encoding a guide RNA sequence of a CRISPR-Cas targeting system. The method may further comprise multiplex transformation of the population of cells with a plurality of genomic sequence-perturbation constructs. The method may further comprise a UPI sequence attached to a perturbation-sequence-capture sequence, and the transfer particle may comprise a perturbation-sequence-capture-binding-sequence having specific binding affinity for the perturbation-sequence-capture sequence attached to the UPI sequence. The UPI sequence is attached to a universal ligation handle sequence, whereby a USI is generated by split-pool ligation.</div>
</li> <li> <para-num num="[0289]"> </para-num> <div class="description-line" id="p-0238" num="0289">In an advantageous embodiment, agents may be uniquely labeled in a dynamic manner (see, e.g., U.S. provisional patent application serial no. 61/703,884 filed Sep. 21, 2012). The unique labels are, at least in part, nucleic acid in nature, and may be generated by sequentially attaching two or more detectable oligonucleotide tags to each other and each unique label may be associated with a separate agent. A detectable oligonucleotide tag may be an oligonucleotide that may be detected by sequencing of its nucleotide sequence and/or by detecting non-nucleic acid detectable moieties to which it may be attached.</div>
</li> <li> <para-num num="[0290]"> </para-num> <div class="description-line" id="p-0239" num="0290">The oligonucleotide tags may be detectable by virtue of their nucleotide sequence, or by virtue of a non-nucleic acid detectable moiety that is attached to the oligonucleotide such as but not limited to a fluorophore, or by virtue of a combination of their nucleotide sequence and the nonnucleic acid detectable moiety.</div>
</li> <li> <para-num num="[0291]"> </para-num> <div class="description-line" id="p-0240" num="0291">In some embodiments, a detectable oligonucleotide tag may comprise one or more nonoligonucleotide detectable moieties. Examples of detectable moieties may include, but are not limited to, fluorophores, microparticles including quantum dots (Empodocles, et al., Nature 399:126-130, 1999), gold nanoparticles (Reichert et al., Anal. Chem. 72:6025-6029, 2000), microbeads (Lacoste et al., Proc. Natl. Acad. Sci. USA 97(17):9461-9466, 2000), biotin, DNP (dinitrophenyl), fucose, digoxigenin, haptens, and other detectable moieties known to those skilled in the art. In some embodiments, the detectable moieties may be quantum dots. Methods for detecting such moieties are described herein and/or are known in the art.</div>
</li> <li> <para-num num="[0292]"> </para-num> <div class="description-line" id="p-0241" num="0292">Thus, detectable oligonucleotide tags may be, but are not limited to, oligonucleotides which may comprise unique nucleotide sequences, oligonucleotides which may comprise detectable moieties, and oligonucleotides which may comprise both unique nucleotide sequences and detectable moieties.</div>
</li> <li> <para-num num="[0293]"> </para-num> <div class="description-line" id="p-0242" num="0293">A unique label may be produced by sequentially attaching two or more detectable oligonucleotide tags to each other. The detectable tags may be present or provided in a plurality of detectable tags. The same or a different plurality of tags may be used as the source of each detectable tag may be part of a unique label. In other words, a plurality of tags may be subdivided into subsets and single subsets may be used as the source for each tag.</div>
</li> <li> <para-num num="[0294]"> </para-num> <div class="description-line" id="p-0243" num="0294">In some embodiments, one or more other species may be associated with the tags. In particular, nucleic acids released by a lysed cell may be ligated to one or more tags. These may include, for example, chromosomal DNA, RNA transcripts, tRNA, mRNA, mitochondrial DNA, or the like. Such nucleic acids may be sequenced, in addition to sequencing the tags themselves, which may yield information about the nucleic acid profile of the cells, which can be associated with the tags, or the conditions that the corresponding droplet or cell was exposed to.</div>
</li> <heading id="h-0013">Drop-Sequence Methods (“Drop-Seq”)</heading>
<li> <para-num num="[0295]"> </para-num> <div class="description-line" id="p-0244" num="0295">Cells come in different types, sub-types and activity states, which are classify based on their their shape, location, function, or molecular profiles, such as the set of RNAs that they express. RNA profiling is in principle particularly informative, as cells express thousands of different RNAs. Approaches that measure for example the level of every type of RNA have until recently been applied to “homogenized” samples—in which the contents of all the cells are mixed together. Methods to profile the RNA content of tens and hundreds of thousands of individual human cells have been recently developed, including from brain tissues, quickly and inexpensively. To do so, special microfluidic devices have been developed to encapsulate each cell in an individual drop, associate the RNA of each cell with a ‘cell barcode’ unique to that cell/drop, measure the expression level of each RNA with sequencing, and then use the cell barcodes to determine which cell each RNA molecule came from. See, e.g., U.S. 62/048,227 filed Sep. 9, 2014]</div>
</li> <li> <para-num num="[0296]"> </para-num> <div class="description-line" id="p-0245" num="0296">Methods of Macosko et al., 2015, “Highly Parallel Genome-wide Expression Profiling of Individual Cells Using Nanoliter Droplets” Cell 161, 1202-1214; International patent application number PCT/US2015/049178, published as W02016/040476 on Mar. 17, 2016; Klein et al., 2015, “Droplet Barcoding for Single-Cell Transcriptomics Applied to Embryonic Stem Cells” Cell 161, 1187-1201; Zheng, et al., 2016, “Haplotyping germline and cancer genomes with high-throughput linked-read sequencing” Nature Biotechnology 34, 303-311; and International patent publication number WO 2014210353 A2 are contemplated for the present invention, all the contents and disclosure of each of which are herein incorporated by reference in their entirety.</div>
</li> <li> <para-num num="[0297]"> </para-num> <div class="description-line" id="p-0246" num="0297">Microfluidics involves micro-scale devices that handle small volumes of fluids. Because microfluidics may accurately and reproducibly control and dispense small fluid volumes, in particular volumes less than 1 μl, application of microfluidics provides significant cost-savings. The use of microfluidics technology reduces cycle times, shortens time-to-results, and increases throughput. Furthermore, incorporation of microfluidics technology enhances system integration and automation. Microfluidic reactions are generally conducted in microdroplets. The ability to conduct reactions in microdroplets depends on being able to merge different sample fluids and different microdroplets. See, e.g., US Patent Publication No. 20120219947. See also international patent application serial no. PCT/US2014/058637 for disclosure regarding a microfluidic laboratory on a chip.</div>
</li> <li> <para-num num="[0298]"> </para-num> <div class="description-line" id="p-0247" num="0298">Droplet microfluidics offers significant advantages for performing high-throughput screens and sensitive assays. Droplets allow sample volumes to be significantly reduced, leading to concomitant reductions in cost. Manipulation and measurement at kilohertz speeds enable up to 10<sup>8 </sup>discrete biological entities (including, but not limited to, individual cells or organelles) to be screened in a single day. Compartmentalization in droplets increases assay sensitivity by increasing the effective concentration of rare species and decreasing the time required to reach detection thresholds. Droplet microfluidics combines these powerful features to enable currently inaccessible high-throughput screening applications, including single-cell and single-molecule assays. See, e.g., Guo et al., Lab Chip, 2012,12, 2146-2155.</div>
</li> <li> <para-num num="[0299]"> </para-num> <div class="description-line" id="p-0248" num="0299">Drop-Sequence methods and apparatus provides a high-throughput single-cell RNA-Seq and/or targeted nucleic acid profiling (for example, sequencing, quantitative reverse transcription polymerase chain reaction, and the like) where the RNAs from different cells are tagged individually, allowing a single library to be created while retaining the cell identity of each read. A combination of molecular barcoding and emulsion-based microfluidics to isolate, lyse, barcode, and prepare nucleic acids from individual cells in high-throughput is used. Microfluidic devices (for example, fabricated in polydimethylsiloxane), sub-nanoliter reverse emulsion droplets. These droplets are used to co-encapsulate nucleic acids with a barcoded capture bead. Each bead, for example, is uniquely barcoded so that each drop and its contents are distinguishable. The nucleic acids may come from any source known in the art, such as for example, those which come from a single cell, a pair of cells, a cellular lysate, or a solution. The cell is lysed as it is encapsulated in the droplet. To load single cells and barcoded beads into these droplets with Poisson statistics, 100,000 to 10 million such beads are needed to barcode ˜10,000-100,000 cells.</div>
</li> <li> <para-num num="[0300]"> </para-num> <div class="description-line" id="p-0249" num="0300">The invention provides a method for creating a single-cell sequencing library comprising: merging one uniquely barcoded mRNA capture microbead with a single-cell in an emulsion droplet having a diameter of 75-125 μm; lysing the cell to make its RNA accessible for capturing by hybridization onto RNA capture microbead; performing a reverse transcription either inside or outside the emulsion droplet to convert the cell's mRNA to a first strand cDNA that is covalently linked to the mRNA capture microbead; pooling the cDNA-attached microbeads from all cells; and preparing and sequencing a single composite RNA-Seq library.</div>
</li> <li> <para-num num="[0301]"> </para-num> <div class="description-line" id="p-0250" num="0301">The invention provides a method for preparing uniquely barcoded mRNA capture microbeads, which has a unique barcode and diameter suitable for microfluidic devices comprising: 1) performing reverse phosphoramidite synthesis on the surface of the bead in a pool-and-split fashion, such that in each cycle of synthesis the beads are split into four reactions with one of the four canonical nucleotides (T, C, G, or A) or unique oligonucleotides of length two or more bases; 2) repeating this process a large number of times, at least two, and optimally more than twelve, such that, in the latter, there are more than 16 million unique barcodes on the surface of each bead in the pool. (See www.ncbi.nlm.nih.gov/pmc/articles/PMC206447)</div>
</li> <li> <para-num num="[0302]"> </para-num> <div class="description-line" id="p-0251" num="0302">Generally, the invention provides a method for preparing a large number of beads, particles, microbeads, nanoparticles, or the like with unique nucleic acid barcodes comprising performing polynucleotide synthesis on the surface of the beads in a pool-and-split fashion such that in each cycle of synthesis the beads are split into subsets that are subjected to different chemical reactions; and then repeating this split-pool process in two or more cycles, to produce a combinatorially large number of distinct nucleic acid barcodes. Invention further provides performing a polynucleotide synthesis wherein the synthesis may be any type of synthesis known to one of skill in the art for “building” polynucleotide sequences in a step-wise fashion. Examples include, but are not limited to, reverse direction synthesis with phosphoramidite chemistry or forward direction synthesis with phosphoramidite chemistry. Previous and well-known methods synthesize the oligonucleotides separately then “glue” the entire desired sequence onto the bead enzymatically. Applicants present a complexed bead and a novel process for producing these beads where nucleotides are chemically built onto the bead material in a high-throughput manner. Moreover, Applicants generally describe delivering a “packet” of beads which allows one to deliver millions of sequences into separate compartments and then screen all at once.</div>
</li> <li> <para-num num="[0303]"> </para-num> <div class="description-line" id="p-0252" num="0303">The invention further provides an apparatus for creating a single-cell sequencing library via a microfluidic system, comprising: a oil-surfactant inlet comprising a filter and a carrier fluid channel, wherein said carrier fluid channel further comprises a resistor; an inlet for an analyte comprising a filter and a carrier fluid channel, wherein said carrier fluid channel further comprises a resistor; an inlet for mRNA capture microbeads and lysis reagent comprising a filter and a carrier fluid channel, wherein said carrier fluid channel further comprises a resistor; said carrier fluid channels have a carrier fluid flowing therein at an adjustable or predetermined flow rate; wherein each said carrier fluid channels merge at a junction; and said junction being connected to a mixer, which contains an outlet for drops.</div>
</li> <li> <para-num num="[0304]"> </para-num> <div class="description-line" id="p-0253" num="0304">A mixture comprising a plurality of microbeads adorned with combinations of the following elements: bead-specific oligonucleotide barcodes created by the described methods; additional oligonucleotide barcode sequences which vary among the oligonucleotides on an indvidual bead and can therefore be used to differentiate or help identify those individual oligonucleotide molecules; additional oligonucleotide sequences that create substrates for downstream molecular-biological reactions, such as oligo-dT (for reverse transcription of mature mRNAs), specific sequences (for capturing specific portions of the transcriptome, or priming for DNA polymerases and similar enzymes), or random sequences (for priming throughout the transcriptome or genome). In an embodiment, the individual oligonucleotide molecules on the surface of any individual microbead contain all three of these elements, and the third element includes both oligo-dT and a primer sequence.</div>
</li> <li> <para-num num="[0305]"> </para-num> <div class="description-line" id="p-0254" num="0305">Examples of the labeling substance which may be employed include labeling substances known to those skilled in the art, such as fluorescent dyes, enzymes, coenzymes, chemiluminescent substances, and radioactive substances. Specific examples include radioisotopes (e.g., 32P, 14C, 125I, 3H, and 131I), fluorescein, rhodamine, dansyl chloride, umbelliferone, luciferase, peroxidase, alkaline phosphatase, β-galactosidase, β-glucosidase, horseradish peroxidase, glucoamylase, lysozyme, saccharide oxidase, microperoxidase, biotin, and ruthenium. In the case where biotin is employed as a labeling substance, preferably, after addition of a biotin-labeled antibody, streptavidin bound to an enzyme (e.g., peroxidase) is further added.</div>
</li> <li> <para-num num="[0306]"> </para-num> <div class="description-line" id="p-0255" num="0306">Advantageously, the label is a fluorescent label. Examples of fluorescent labels include, but are not limited to, Atto dyes, 4-acetamido-4′-isothiocyanatostilbene-2,2′disulfonic acid; acridine and derivatives: acridine, acridine isothiocyanate; 5-(2′-aminoethyl)aminonaphthalene- 1-sulfonic acid (EDANS); 4-amino-N-[3-vinyl sulfonyl)phenyl]naphthalimide-3,5 di sulfonate; N-(4-anilino-1-naphthyl)maleimide; anthranilamide; BODIPY; Brilliant Yellow; coumarin and derivatives; coumarin, 7-amino-4-methylcoumarin (AMC, Coumarin 120), 7-amino-4-trifluoromethylcouluarin (Coumaran 151); cyanine dyes; cyanosine; 4′,6-diaminidino-2-phenylindole (DAPI); 5′5″-dibromopyrogallol-sulfonaphthalein (Bromopyrogallol Red); 7-diethylamino-3 -(4′-isothiocyanatophenyl)-4-methylcoumarin; diethylenetriamine pentaacetate; 4,4′-diisothiocyanatodihydro-stilbene-2,2′-disulfonic acid; 4,4′-diisothiocyanatostilbene-2,2′-disulfonic acid; 5-[dimethylamino]naphthalene-1-sulfonyl chloride (DNS, dansylchloride); 4-dimethylaminophenylazophenyl-4′-isothiocyanate (DABITC); eosin and derivatives; eosin, eosin isothiocyanate, erythrosin and derivatives; erythrosin B, erythrosin, isothiocyanate; ethidium; fluorescein and derivatives; 5-carboxyfluorescein (FAM), 5-(4,6-dichlorotriazin-2-yl)aminofluorescein (DTAF), 2′,7′-dimethoxy-4′5′-dichloro-6-carboxyfluorescein, fluorescein, fluorescein isothiocyanate, QFITC, (XRITC); fluorescamine; IR144; IR1446; Malachite Green isothiocyanate; 4-methylumbelliferoneortho cresolphthalein; nitrotyrosine; pararosaniline; Phenol Red; B-phycoerythrin; o-phthaldialdehyde; pyrene and derivatives: pyrene, pyrene butyrate, succinimidyl 1-pyrene; butyrate quantum dots; Reactive Red 4 (Cibacron.™. Brilliant Red 3B-A) rhodamine and derivatives: 6-carboxy-X-rhodamine (ROX), 6-carboxyrhodamine (R6G), lissamine rhodamine B sulfonyl chloride rhodamine (Rhod), rhodamine B, rhodamine 123, rhodamine X isothiocyanate, sulforhodamine B, sulforhodamine 101, sulfonyl chloride derivative of sulforhodamine 101 (Texas Red); N,N,N′,N′ tetramethyl-6-carboxyrhodamine (TAMRA); tetramethyl rhodamine; tetramethyl rhodamine isothiocyanate (TRITC); riboflavin; rosolic acid; terbium chelate derivatives; Cy3; Cy5; Cy5.5; Cy7; IRD 700; IRD 800; La Jolta Blue; phthalo cyanine; and naphthalo cyanine.</div>
</li> <li> <para-num num="[0307]"> </para-num> <div class="description-line" id="p-0256" num="0307">The fluorescent label may be a fluorescent protein, such as blue fluorescent protein, cyan fluorescent protein, green fluorescent protein, red fluorescent protein, yellow fluorescent protein or any photoconvertible protein. Colormetric labeling, bioluminescent labeling and/or chemiluminescent labeling may further accomplish labeling. Labeling further may include energy transfer between molecules in the hybridization complex by perturbation analysis, quenching, or electron transport between donor and acceptor molecules, the latter of which may be facilitated by double stranded match hybridization complexes. The fluorescent label may be a perylene or a terrylen. In the alternative, the fluorescent label may be a fluorescent bar code.</div>
</li> <li> <para-num num="[0308]"> </para-num> <div class="description-line" id="p-0257" num="0308">In an advantageous embodiment, the label may be light sensitive, wherein the label is light-activated and/or light cleaves the one or more linkers to release the molecular cargo. The light-activated molecular cargo may be a major light-harvesting complex (LHCII). In another embodiment, the fluorescent label may induce free radical formation.</div>
</li> <li> <para-num num="[0309]"> </para-num> <div class="description-line" id="p-0258" num="0309">The invention described herein enables high throughput and high resolution delivery of reagents to individual emulsion droplets that may contain cells, organelles, nucleic acids, proteins, etc. through the use of monodisperse aqueous droplets that are generated by a microfluidic device as a water-in-oil emulsion. The droplets are carried in a flowing oil phase and stabilized by a surfactant. In one aspect single cells or single organellesor single molecules (proteins, RNA, DNA) are encapsulated into uniform droplets from an aqueous solution/dispersion. In a related aspect, multiple cells or multiple molecules may take the place of single cells or single molecules. The aqueous droplets of volume ranging from 1 pL to 10 nL work as individual reactors. Disclosed embodiments provide 10<sup>4 </sup>to 10<sup>5 </sup>single cells in droplets which can be processed and analyzed in a single run.</div>
</li> <li> <para-num num="[0310]"> </para-num> <div class="description-line" id="p-0259" num="0310">To utilize microdroplets for rapid large-scale chemical screening or complex biological library identification, different species of microdroplets, each containing the specific chemical compounds or biological probes cells or molecular barcodes of interest, have to be generated and combined at the preferred conditions, e.g., mixing ratio, concentration, and order of combination.</div>
</li> <li> <para-num num="[0311]"> </para-num> <div class="description-line" id="p-0260" num="0311">Each species of droplet is introduced at a confluence point in a main microfluidic channel from separate inlet microfluidic channels. Preferably, droplet volumes are chosen by design such that one species is larger than others and moves at a different speed, usually slower than the other species, in the carrier fluid, as disclosed in U.S. Publication No. US 2007/0195127 and International Publication No. WO 2007/089541, each of which are incorporated herein by reference in their entirety. The channel width and length is selected such that faster species of droplets catch up to the slowest species. Size constraints of the channel prevent the faster moving droplets from passing the slower moving droplets resulting in a train of droplets entering a merge zone. Multi-step chemical reactions, biochemical reactions, or assay detection chemistries often require a fixed reaction time before species of different type are added to a reaction. Multi-step reactions are achieved by repeating the process multiple times with a second, third or more confluence points each with a separate merge point. Highly efficient and precise reactions and analysis of reactions are achieved when the frequencies of droplets from the inlet channels are matched to an optimized ratio and the volumes of the species are matched to provide optimized reaction conditions in the combined droplets.</div>
</li> <li> <para-num num="[0312]"> </para-num> <div class="description-line" id="p-0261" num="0312">Fluidic droplets may be screened or sorted within a fluidic system of the invention by altering the flow of the liquid containing the droplets. For instance, in one set of embodiments, a fluidic droplet may be steered or sorted by directing the liquid surrounding the fluidic droplet into a first channel, a second channel, etc. In another set of embodiments, pressure within a fluidic system, for example, within different channels or within different portions of a channel, can be controlled to direct the flow of fluidic droplets. For example, a droplet can be directed toward a channel junction including multiple options for further direction of flow (e.g., directed toward a branch, or fork, in a channel defining optional downstream flow channels). Pressure within one or more of the optional downstream flow channels can be controlled to direct the droplet selectively into one of the channels, and changes in pressure can be effected on the order of the time required for successive droplets to reach the junction, such that the downstream flow path of each successive droplet can be independently controlled. In one arrangement, the expansion and/or contraction of liquid reservoirs may be used to steer or sort a fluidic droplet into a channel, e.g., by causing directed movement of the liquid containing the fluidic droplet. In another embodiment, the expansion and/or contraction of the liquid reservoir may be combined with other flow-controlling devices and methods, e.g., as described herein. Non-limiting examples of devices able to cause the expansion and/or contraction of a liquid reservoir include pistons.</div>
</li> <li> <para-num num="[0313]"> </para-num> <div class="description-line" id="p-0262" num="0313">Key elements for using microfluidic channels to process droplets include: (1) producing droplet of the correct volume, (2) producing droplets at the correct frequency and (3) bringing together a first stream of sample droplets with a second stream of sample droplets in such a way that the frequency of the first stream of sample droplets matches the frequency of the second stream of sample droplets. Preferably, bringing together a stream of sample droplets with a stream of premade library droplets in such a way that the frequency of the library droplets matches the frequency of the sample droplets.</div>
</li> <li> <para-num num="[0314]"> </para-num> <div class="description-line" id="p-0263" num="0314">Methods for producing droplets of a uniform volume at a regular frequency are well known in the art. One method is to generate droplets using hydrodynamic focusing of a dispersed phase fluid and immiscible carrier fluid, such as disclosed in U.S. Publication No. US 2005/0172476 and International Publication No. WO 2004/002627. It is desirable for one of the species introduced at the confluence to be a pre-made library of droplets where the library contains a plurality of reaction conditions, e.g., a library may contain plurality of different compounds at a range of concentrations encapsulated as separate library elements for screening their effect on cells or enzymes, alternatively a library could be composed of a plurality of different primer pairs encapsulated as different library elements for targeted amplification of a collection of loci, alternatively a library could contain a plurality of different antibody species encapsulated as different library elements to perform a plurality of binding assays. The introduction of a library of reaction conditions onto a substrate is achieved by pushing a premade collection of library droplets out of a vial with a drive fluid. The drive fluid is a continuous fluid. The drive fluid may comprise the same substance as the carrier fluid (e.g., a fluorocarbon oil). For example, if a library consists of ten pico-liter droplets is driven into an inlet channel on a microfluidic substrate with a drive fluid at a rate of 10,000 pico-liters per second, then nominally the frequency at which the droplets are expected to enter the confluence point is 1000 per second. However, in practice droplets pack with oil between them that slowly drains. Over time the carrier fluid drains from the library droplets and the number density of the droplets (number/mL) increases. Hence, a simple fixed rate of infusion for the drive fluid does not provide a uniform rate of introduction of the droplets into the microfluidic channel in the substrate. Moreover, library-to-library variations in the mean library droplet volume result in a shift in the frequency of droplet introduction at the confluence point. Thus, the lack of uniformity of droplets that results from sample variation and oil drainage provides another problem to be solved. For example if the nominal droplet volume is expected to be 10 pico-liters in the library, but varies from 9 to 11 pico-liters from library-to-library then a 10,000 pico-liter/second infusion rate will nominally produce a range in frequencies from 900 to 1,100 droplet per second. In short, sample to sample variation in the composition of dispersed phase for droplets made on chip, a tendency for the number density of library droplets to increase over time and library-to-library variations in mean droplet volume severely limit the extent to which frequencies of droplets may be reliably matched at a confluence by simply using fixed infusion rates. In addition, these limitations also have an impact on the extent to which volumes may be reproducibly combined. Combined with typical variations in pump flow rate precision and variations in channel dimensions, systems are severely limited without a means to compensate on a run-to-run basis. The foregoing facts not only illustrate a problem to be solved, but also demonstrate a need for a method of instantaneous regulation of microfluidic control over microdroplets within a microfluidic channel.</div>
</li> <li> <para-num num="[0315]"> </para-num> <div class="description-line" id="p-0264" num="0315">Combinations of surfactant(s) and oils must be developed to facilitate generation, storage, and manipulation of droplets to maintain the unique chemical/biochemical/biological environment within each droplet of a diverse library. Therefore, the surfactant and oil combination must (1) stabilize droplets against uncontrolled coalescence during the drop forming process and subsequent collection and storage, (2) minimize transport of any droplet contents to the oil phase and/or between droplets, and (3) maintain chemical and biological inertness with contents of each droplet (e.g., no adsorption or reaction of encapsulated contents at the oil-water interface, and no adverse effects on biological or chemical constituents in the droplets). In addition to the requirements on the droplet library function and stability, the surfactant-in-oil solution must be coupled with the fluid physics and materials associated with the platform. Specifically, the oil solution must not swell, dissolve, or degrade the materials used to construct the microfluidic chip, and the physical properties of the oil (e.g., viscosity, boiling point, etc.) must be suited for the flow and operating conditions of the platform.</div>
</li> <li> <para-num num="[0316]"> </para-num> <div class="description-line" id="p-0265" num="0316">Droplets formed in oil without surfactant are not stable to permit coalescence, so surfactants must be dissolved in the oil that is used as the continuous phase for the emulsion library. Surfactant molecules are amphiphilic--part of the molecule is oil soluble, and part of the molecule is water soluble. When a water-oil interface is formed at the nozzle of a microfluidic chip for example in the inlet module described herein, surfactant molecules that are dissolved in the oil phase adsorb to the interface. The hydrophilic portion of the molecule resides inside the droplet and the fluorophilic portion of the molecule decorates the exterior of the droplet. The surface tension of a droplet is reduced when the interface is populated with surfactant, so the stability of an emulsion is improved. In addition to stabilizing the droplets against coalescence, the surfactant should be inert to the contents of each droplet and the surfactant should not promote transport of encapsulated components to the oil or other droplets.</div>
</li> <li> <para-num num="[0317]"> </para-num> <div class="description-line" id="p-0266" num="0317">A droplet library may be made up of a number of library elements that are pooled together in a single collection (see, e.g., US Patent Publication No. 2010002241). Libraries may vary in complexity from a single library element to 1015 library elements or more. Each library element may be one or more given components at a fixed concentration. The element may be, but is not limited to, cells, organelles, virus, bacteria, yeast, beads, amino acids, proteins, polypeptides, nucleic acids, polynucleotides or small molecule chemical compounds. The element may contain an identifier such as a label. The terms “droplet library” or “droplet libraries” are also referred to herein as an “emulsion library” or “emulsion libraries.” These terms are used interchangeably throughout the specification.</div>
</li> <li> <para-num num="[0318]"> </para-num> <div class="description-line" id="p-0267" num="0318">A cell library element may include, but is not limited to, hybridomas, B-cells, primary cells, cultured cell lines, cancer cells, stem cells, cells obtained from tissue, or any other cell type. Cellular library elements are prepared by encapsulating a number of cells from one to hundreds of thousands in individual droplets. The number of cells encapsulated is usually given by Poisson statistics from the number density of cells and volume of the droplet. However, in some cases the number deviates from Poisson statistics as described in Edd et al., “Controlled encapsulation of single-cells into monodisperse picolitre drops.” Lab Chip, 8(8): 1262-1264, 2008. The discrete nature of cells allows for libraries to be prepared in mass with a plurality of cellular variants all present in a single starting media and then that media is broken up into individual droplet capsules that contain at most one cell. These individual droplets capsules are then combined or pooled to form a library consisting of unique library elements. Cell division subsequent to, or in some embodiments following, encapsulation produces a clonal library element.</div>
</li> <li> <para-num num="[0319]"> </para-num> <div class="description-line" id="p-0268" num="0319">A bead based library element may contain one or more beads, of a given type and may also contain other reagents, such as antibodies, enzymes or other proteins. In the case where all library elements contain different types of beads, but the same surrounding media, the library elements may all be prepared from a single starting fluid or have a variety of starting fluids. In the case of cellular libraries prepared in mass from a collection of variants, such as genomically modified, yeast or bacteria cells, the library elements will be prepared from a variety of starting fluids.</div>
</li> <li> <para-num num="[0320]"> </para-num> <div class="description-line" id="p-0269" num="0320">Often it is desirable to have exactly one cell per droplet with only a few droplets containing more than one cell when starting with a plurality of cells or yeast or bacteria, engineered to produce variants on a protein. In some cases, variations from Poisson statistics may be achieved to provide an enhanced loading of droplets such that there are more droplets with exactly one cell per droplet and few exceptions of empty droplets or droplets containing more than one cell.</div>
</li> <li> <para-num num="[0321]"> </para-num> <div class="description-line" id="p-0270" num="0321">Examples of droplet libraries are collections of droplets that have different contents, ranging from beads, cells, small molecules, DNA, primers, antibodies. Smaller droplets may be in the order of femtoliter (fL) volume drops, which are especially contemplated with the droplet dispensors. The volume may range from about 5 to about 600 fL. The larger droplets range in size from roughly 0.5 micron to 500 micron in diameter, which corresponds to about 1 pico liter to 1 nano liter. However, droplets may be as small as 5 microns and as large as 500 microns. Preferably, the droplets are at less than 100 microns, about 1 micron to about 100 microns in diameter. The most preferred size is about 20 to 40 microns in diameter (10 to 100 picoliters). The preferred properties examined of droplet libraries include osmotic pressure balance, uniform size, and size ranges.</div>
</li> <li> <para-num num="[0322]"> </para-num> <div class="description-line" id="p-0271" num="0322">The droplets comprised within the emulsion libraries of the present invention may be contained within an immiscible oil which may comprise at least one fluorosurfactant. In some embodiments, the fluorosurfactant comprised within immiscible fluorocarbon oil is a block copolymer consisting of one or more perfluorinated polyether (PFPE) blocks and one or more polyethylene glycol (PEG) blocks. In other embodiments, the fluorosurfactant is a triblock copolymer consisting of a PEG center block covalently bound to two PFPE blocks by amide linking groups. The presence of the fluorosurfactant (similar to uniform size of the droplets in the library) is critical to maintain the stability and integrity of the droplets and is also essential for the subsequent use of the droplets within the library for the various biological and chemical assays described herein. Fluids (e.g., aqueous fluids, immiscible oils, etc.) and other surfactants that may be utilized in the droplet libraries of the present invention are described in greater detail herein.</div>
</li> <li> <para-num num="[0323]"> </para-num> <div class="description-line" id="p-0272" num="0323">The present invention provides an emulsion library which may comprise a plurality of aqueous droplets within an immiscible oil (e.g., fluorocarbon oil) which may comprise at least one fluorosurfactant, wherein each droplet is uniform in size and may comprise the same aqueous fluid and may comprise a different library element. The present invention also provides a method for forming the emulsion library which may comprise providing a single aqueous fluid which may comprise different library elements, encapsulating each library element into an aqueous droplet within an immiscible fluorocarbon oil which may comprise at least one fluorosurfactant, wherein each droplet is uniform in size and may comprise the same aqueous fluid and may comprise a different library element, and pooling the aqueous droplets within an immiscible fluorocarbon oil which may comprise at least one fluorosurfactant, thereby forming an emulsion library.</div>
</li> <li> <para-num num="[0324]"> </para-num> <div class="description-line" id="p-0273" num="0324">For example, in one type of emulsion library, all different types of elements (e.g., cells or beads), may be pooled in a single source contained in the same medium. After the initial pooling, the cells or beads are then encapsulated in droplets to generate a library of droplets wherein each droplet with a different type of bead or cell is a different library element. The dilution of the initial solution enables the encapsulation process. In some embodiments, the droplets formed will either contain a single cell or bead or will not contain anything, i.e., be empty. In other embodiments, the droplets formed will contain multiple copies of a library element. The cells or beads being encapsulated are generally variants on the same type of cell or bead. In one example, the cells may comprise cancer cells of a tissue biopsy, and each cell type is encapsulated to be screened for genomic data or against different drug therapies. Another example is that 10<sup>11 </sup>or 10<sup>15 </sup>different type of bacteria; each having a different plasmid spliced therein, are encapsulated. One example is a bacterial library where each library element grows into a clonal population that secretes a variant on an enzyme.</div>
</li> <li> <para-num num="[0325]"> </para-num> <div class="description-line" id="p-0274" num="0325">In another example, the emulsion library may comprise a plurality of aqueous droplets within an immiscible fluorocarbon oil, wherein a single molecule may be encapsulated, such that there is a single molecule contained within a droplet for every 20-60 droplets produced (e.g., 20, 25, 30, 35, 40, 45, 50, 55, 60 droplets, or any integer in between). Single molecules may be encapsulated by diluting the solution containing the molecules to such a low concentration that the encapsulation of single molecules is enabled. In one specific example, a LacZ plasmid DNA was encapsulated at a concentration of 20 fM after two hours of incubation such that there was about one gene in 40 droplets, where 10 μm droplets were made at 10 kHz per second. Formation of these libraries rely on limiting dilutions.</div>
</li> <li> <para-num num="[0326]"> </para-num> <div class="description-line" id="p-0275" num="0326">Methods of the invention involve forming sample droplets. The droplets are aqueous droplets that are surrounded by an immiscible carrier fluid. Methods of forming such droplets are shown for example in Link et al. (U.S. patent application numbers 2008/0014589, 2008/0003142, and 2010/0137163), Stone et al. (U.S. Pat. No. 7,708,949 and U.S. patent application number 2010/0172803), Anderson et al. (U.S. Pat. No. 7,041,481 and which reissued as RE41,780) and European publication number EP2047910 to Raindance Technologies Inc. The content of each of which is incorporated by reference herein in its entirety.</div>
</li> <li> <para-num num="[0327]"> </para-num> <div class="description-line" id="p-0276" num="0327">In certain embodiments, the carrier fluid may contain one or more additives, such as agents which reduce surface tensions (surfactants). Surfactants can include Tween, Span, fluorosurfactants, and other agents that are soluble in oil relative to water. In some applications, performance is improved by adding a second surfactant to the sample fluid. Surfactants can aid in controlling or optimizing droplet size, flow and uniformity, for example by reducing the shear force needed to extrude or inject droplets into an intersecting channel. This can affect droplet volume and periodicity, or the rate or frequency at which droplets break off into an intersecting channel. Furthermore, the surfactant can serve to stabilize aqueous emulsions in fluorinated oils from coalescing.</div>
</li> <li> <para-num num="[0328]"> </para-num> <div class="description-line" id="p-0277" num="0328">In certain embodiments, the droplets may be surrounded by a surfactant which stabilizes the droplets by reducing the surface tension at the aqueous oil interface. Preferred surfactants that may be added to the carrier fluid include, but are not limited to, surfactants such as sorbitan-based carboxylic acid esters (e.g., the “Span” surfactants, Fluka Chemika), including sorbitan monolaurate (Span 20), sorbitan monopalmitate (Span 40), sorbitan monostearate (Span 60) and sorbitan monooleate (Span 80), and perfluorinated polyethers (e.g., DuPont Krytox 157 FSL, FSM, and/or FSH). Other non-limiting examples of non-ionic surfactants which may be used include polyoxyethylenated alkylphenols (for example, nonyl-, p-dodecyl-, and dinonylphenols), polyoxyethylenated straight chain alcohols, polyoxyethylenated polyoxypropylene glycols, polyoxyethylenated mercaptans, long chain carboxylic acid esters (for example, glyceryl and polyglyceryl esters of natural fatty acids, propylene glycol, sorbitol, polyoxyethylenated sorbitol esters, polyoxyethylene glycol esters, etc.) and alkanolamines (e.g., diethanolamine-fatty acid condensates and isopropanolamine-fatty acid condensates).</div>
</li> <li> <para-num num="[0329]"> </para-num> <div class="description-line" id="p-0278" num="0329">By incorporating a plurality of unique tags into the additional droplets and joining the tags to a solid support designed to be specific to the primary droplet, the conditions that the primary droplet is exposed to may be encoded and recorded. For example, nucleic acid tags can be sequentially ligated to create a sequence reflecting conditions and order of same. Alternatively, the tags can be added independently appended to solid support. Non-limiting examples of a dynamic labeling system that may be used to bioninformatically record information can be found at U.S. Provisional Patent Application entitled “Compositions and Methods for Unique Labeling of Agents” filed Sep. 21, 2012 and Nov. 29, 2012. In this way, two or more droplets may be exposed to a variety of different conditions, where each time a droplet is exposed to a condition, a nucleic acid encoding the condition is added to the droplet each ligated together or to a unique solid support associated with the droplet such that, even if the droplets with different histories are later combined, the conditions of each of the droplets are remain available through the different nucleic acids. Non-limiting examples of methods to evaluate response to exposure to a plurality of conditions can be found at U.S. Provisional Patent Application entitled “Systems and Methods for Droplet Tagging” filed Sep. 21, 2012.</div>
</li> <li> <para-num num="[0330]"> </para-num> <div class="description-line" id="p-0279" num="0330">Applications of the disclosed device may include use for the dynamic generation of molecular barcodes (e.g., DNA oligonucleotides, flurophores, etc.) either independent from or in concert with the controlled delivery of various compounds of interest (drugs, small molecules, siRNA, CRISPR guide RNAs, reagents, etc.). For example, unique molecular barcodes can be created in one array of nozzles while individual compounds or combinations of compounds can be generated by another nozzle array. Barcodes/compounds of interest can then be merged with cell-containing droplets. An electronic record in the form of a computer log file is kept to associate the barcode delivered with the downstream reagent(s) delivered. This methodology makes it possible to efficiently screen a large population of cells for applications such as single-cell drug screening, controlled perturbation of regulatory pathways, etc. The device and techniques of the disclosed invention facilitate efforts to perform studies that require data resolution at the single cell (or single molecule) level and in a cost effective manner. Disclosed embodiments provide a high throughput and high resolution delivery of reagents to individual emulsion droplets that may contain cells, nucleic acids, proteins, etc. through the use of monodisperse aqueous droplets that are generated one by one in a microfluidic chip as a water-in-oil emulsion. Hence, the invention proves advantageous over prior art systems by being able to dynamically track individual cells and droplet treatments/combinations during life cycle experiments. Additional advantages of the disclosed invention provides an ability to create a library of emulsion droplets on demand with the further capability of manipulating the droplets through the disclosed process(es). Disclosed embodiments may, thereby, provide dynamic tracking of the droplets and create a history of droplet deployment and application in a single cell based environment.</div>
</li> <li> <para-num num="[0331]"> </para-num> <div class="description-line" id="p-0280" num="0331">Droplet generation and deployment is produced via a dynamic indexing strategy and in a controlled fashion in accordance with disclosed embodiments of the present invention. Disclosed embodiments of the microfluidic device described herein provides the capability of microdroplets that be processed, analyzed and sorted at a highly efficient rate of several thousand droplets per second, providing a powerful platform which allows rapid screening of millions of distinct compounds, biological probes, proteins or cells either in cellular models of biological mechanisms of disease, or in biochemical, or pharmacological assays.</div>
</li> <heading id="h-0014">Undersampling—A Sampling Based Framework for Genetic Interactions</heading>
<li> <para-num num="[0332]"> </para-num> <div class="description-line" id="p-0281" num="0332">According to the invention, random sampling may comprise matrix completion, tensor completion, compressed sensing, or kernel learning.</div>
</li> <li> <para-num num="[0333]"> </para-num> <div class="description-line" id="p-0282" num="0333">In some aspects, where random sampling comprises matrix completion, tensor completion, or compressed sensing, π may be of the order of log P.</div>
</li> <li> <para-num num="[0334]"> </para-num> <div class="description-line" id="p-0283" num="0334">The invention relies on a random sampling assumption, e.g. that the combinatorial space is sparse and/or of low rank. This assumption is generic and advantageously does not rely on the pre-determination of a (known) set of genetic interactions. This assumption constrains the range or complexity of models, and thus can be used to restrict sampling size (undersampling). Further, as detailed below, the invention relies on the following:: (1) Given a limited number of assays, if one wishes to infer interactions up to an order j, it is advantageous to randomly sample interactions at a higher order k&gt;j, because higher order perturbations maximize the information that can be recovered; and (2) in such a method, one may use a model that accounts for higher order interactions when analyzing lower order ones. For example, it is possible to aim for each perturbation to target k˜5-7 genes at once to estimate/model interactions at lower order j˜3-5.</div>
</li> <li> <para-num num="[0335]"> </para-num> <div class="description-line" id="p-0284" num="0335">Although some experimental methods open the way to test non-linear interactions by high order combinatorial genetic perturbations, exhaustive combinatorial exploration is intractable for anything but 2- or 3-way interactions for a few genes.</div>
</li> <li> <para-num num="[0336]"> </para-num> <div class="description-line" id="p-0285" num="0336">According to the invention, random matrix theory and compressive sensing may be used to re-formulate this as a random sampling problem, developing a new framework from experimental design to model inference, testing and refinement.</div>
</li> <li> <para-num num="[0337]"> </para-num> <div class="description-line" id="p-0286" num="0337">To infer combinatorial models from a dramatic under-sampling of the full high-order combinatorial space with massively combinatorial molecular perturbations (MCPP), one may rely on random matrix theory, compressive sensing and kernel learning.</div>
</li> <li> <para-num num="[0338]"> </para-num> <div class="description-line" id="p-0287" num="0338">According to the invention, it is made possible to model non-linear regulatory functions from genetic manipulations (perturbations).</div>
</li> <li> <para-num num="[0339]"> </para-num> <div class="description-line" id="p-0288" num="0339">One may learn models of higher-order genetic interactions from combinatorial perturbations with single cell profiling. Although the learning problem is underdetermined due to combinatorial explosion (2<sup>m </sup>possible interaction terms among m genes), it can become tractable in the presence of additional structure, including sparsity and smoothness, that constrains the range or complexity of models. One may thus rely on the following: (1) Given a limited number of assays, if one wishes to infer interactions up to an order j, it is advantageous to randomly sample interactions at a higher order k&gt;j, because higher order perturbations maximize the information that can be recovered; and (2) in such a design, one can use a model that accounts for higher order interactions when analyzing lower order ones. One may for example aim for each perturbation to target k˜5-7 genes at once to estimate interactions at j˜3-5.</div>
</li> <li> <para-num num="[0340]"> </para-num> <div class="description-line" id="p-0289" num="0340">Thus the present invention relies on a learning approach that takes multiplex perturbations at a high order n and a complex readout data (e.g., RNA profile) and infers a model of genetic interactions at a lower order (m&lt;n),as well as strategies for experimental design, model testing and refinement.</div>
</li> <li> <para-num num="[0341]"> </para-num> <div class="description-line" id="p-0290" num="0341">If one assumes that genetic interactions are low rank, sparse, or both, then the true number of degrees of freedom is small relative to the complete combinatorial expansion, so that one can infer the full nonlinear landscape with a relatively small random sampling of high-order perturbations, without specific knowledge of which genes are likely to interact. Analysis of prior studies supports the sparsity assumption in yeast (for fitness: Costanzo, M., Baryshnikova, A., Bellay, J., Kim, Y., Spear, E. D., Sevier, C. S., Ding, H., Koh, J. L., Toufighi, K., Mostafavi, S., Prinz, J., St Onge, R. P., VanderSluis, B., Makhnevych, T., Vizeacoumar, F. J., Alizadeh, S., Bahr, S., Brost, R. L., Chen, Y., Cokol, M., Deshpande, R., Li, Z., Lin, Z. Y., Liang, W., Marback, M., Paw, J., San Luis, B. J., Shuteriqi, E., Tong, A. H., van Dyk, N., Wallace, I. M., Whitney, J. A., Weirauch, M. T., Zhong, G., Zhu, H., Houry, W. A., Brudno, M., Ragibizadeh, S., Papp, B., Pal, C., Roth, F. P., Giaever, G., Nislow, C., Troyanskaya, O. G., Bussey, H., Bader, G. D., Gingras, A. C., Morris, Q. D., Kim, P. M., Kaiser, C. A., Myers, C. L., Andrews, B. J. &amp; Boone, C. The genetic landscape of a cell. <i>Science. </i>327, 425-431, doi:10.1126/science.1180823 (2010)), and fly (for 11 imaging phenotypes: Laufer, C., Fischer, B., Billmann, M., Huber, W. &amp; Boutros, M. Mapping genetic interactions in human cancer cells with RNAi and multiparametric phenotyping. <i>Nat Methods. </i>10, 427-431, doi:10.1038/nmeth.2436 (2013)), and to the limited tested extent, mammals (for 60 genes: Bassik, M. C., Kampmann, M., Lebbink, R. J., Wang, S., Hein, M. Y., Poser, I., Weibezahn, J., Horlbeck, M. A., Chen, S., Mann, M., Hyman, A. A., Leproust, E. M., McManus, M. T. &amp; Weissman, J. S. A systematic mammalian genetic interaction map reveals pathways underlying ricin susceptibility. <i>Cell. </i>152, 909-922, doi:10.1016/j.cell2013.01.030 (2013). PMCID:3652613).</div>
</li> <heading id="h-0015">Matrix (Tensor) Completion.</heading>
<li> <para-num num="[0342]"> </para-num> <div class="description-line" id="p-0291" num="0342">All the values of a matrix (tensor) are filled in using a small collection of sampled entries. Applicants hypothesize that the rank of a tensor of higher-order interactions is a fraction of the number of tested genes which is tested by by calculating the rank from a dense sampling of second or third order knockouts from a small collection of genes. If the rank of interactions is limited, then Applicants randomly sample sets of genes to knockout from a larger collection, and fill in the remaining values via nuclear norm regularized least-squares optimization (Candes, E. J. &amp; Plan, Y. Matrix Completion With Noise. Proceedings of the IEEE. 98, 925-936, doi:Doi 10.1109/Jproc.2009.2035722 (2010)). Provable guarantees suggest that if the rank, r, is small relative to the number of genes, n, then m≥O (n<sup>6/5 </sup>r log n) sampled entries are sufficient. However, since these guarantees assume rough uniformity in the loadings of interaction singular vectors, this assumption is unlikely to hold if the interaction matrix is very sparse. In this case, Applicants perform the same random sampling, and simultaneously regularize over both the nuclear norm and the L1 norm of the matrix (Richard, E., Savalle, P. &amp; Vayatis, N. Estimation of Simultaneously Sparse and Low Rank Matrices. arXiv. doi:arXiv:1206.6474).</div>
</li> <heading id="h-0016">Compressed Sensing</heading>
<li> <para-num num="[0343]"> </para-num> <div class="description-line" id="p-0292" num="0343">Here, instead of working with a tensor of interaction terms, Applicants work with a basis that spans all higher order interactions. Each single quantitative phenotype is a real-valued function f(g) on possible genotypes g (the 2<sup>m </sup>possible allelic or knockout states), represented as binary strings of length m. Applicants analyze such Boolean functions using <i>Fourier decomposition </i>(O'Donnell, R. <i>Analysis of boolean functions. </i>(Cambridge University Press, 2014)) f(g)=Σ<sub>bε(0,1)</sub> <sub> <sup2>m</sup2> </sub>{circumflex over (f)}<sub>b</sub>(−1)<sup>b·g</sup>, {circumflex over (f)}<sub>b</sub>=1/2<sup>m</sup>Σ<sub>gε(0,1)</sub> <sub> <sup2>m</sup2> </sub>f(g)(−1)<sup>g·b</sup>, where f is an orthogonal basis indexed by binary strings b, and each Fourier coefficient {circumflex over (f)}<sub>b </sub>precisely quantifies the effect of one possible multi-gene interaction. For example with m=2, {circumflex over (f)}<sub>00 </sub>is the average phenotype; {circumflex over (f)}<sub>10 </sub>is the effect of the first gene KO, marginalized over the genetic background of the second; similarly for {circumflex over (f)}<sub>01</sub>; and {circumflex over (f)}<sub>11 </sub>quantifies the two-way interaction (the extent to which the double KO phenotype differs from that predicted by the sum of the effects of the single KOs). Applicants hypothesize that such genotype-phenotype maps are approximately sparse in the Fourier basis, such that there is a small number, s, of nonzero Fourier coefficients (not known a priori). With perturbations generated only up to a limited order, Applicants obtain a truncated Fourier model, which is a general linear model: the genetic interactions are in the basis functions (encoded into a design matrix), and the response is linear in the unknown Fourier coefficients. Applicants assume most truncated coefficients are negligible. Assuming that the genotype-phenotype maps are approximately sparse in the Fourier basis, Applicants use L1-penalized regression to learn the coefficients of the map from paired genotype-phenotype observations g<sub>i</sub>, f(g<sub>i</sub>) (with uncertainty or noise in both).</div>
</li> <li> <para-num num="[0344]"> </para-num> <div class="description-line" id="p-0293" num="0344">Compressed sensing posits that if Applicants' perturbations are de-coherent under the given basis, then exact recovery is possible with dramatic under-sampling (in the noiseless case) (Candes, E. Compressive sampling. <i>Proceedings of the International Congress of Mathematicians Madrid, </i>Aug. 22-30, 2006. 3, 19, doi:10.4171/022 (2006)), such that a sample size n=C s log p will suffice, where s is the number of effectively nonzero coefficients, p is the magnitude of combinatorial expansion and C depends on noise and experimental design (how the g<sub>1 </sub>are sampled) (Candes, E. Mathematics of sparsity (and few other things). <i>ICM </i>2014 <i>Proceedings, to appear. </i>(2014)). By varying the penalization parameter, Applicants learn sparse structures at different levels of thresholding, and find the level below which the data become insufficient to capture the signal (Hastie, T., Friedman, J. &amp; Tibshirani, R. <i>The elements of statistical learning. </i>Vol. 2 (Springer, 2009)). Applicants explore using a larger penalization parameter on the higher order interaction coefficients, and, with good estimates of single perturbations, even no penalty on the linear terms, or regressing those out first. If each experiment is a Poisson random sampling of KOs, Applicants expect the measurements to have good de-coherence under the Fourier basis, provided the mean number of KO experiments per gene is not too low. If Applicants' assumptions are correct, a soft phase transition in performance as the number of observations crosses a threshold should be observed. Applicants use a small complete dataset or downsampling of a larger more random dataset, to assess if the appropriate transition is observed.</div>
</li> <heading id="h-0017">Kernel Learning.</heading>
<li> <para-num num="[0345]"> </para-num> <div class="description-line" id="p-0294" num="0345">If there is no strict sparsity in the rank or in the coefficients, Applicants build predictive functions of the effects of combinatorial perturbations, using a kernel of experimental similarity. Given m experiments, Applicants define an m x m polynomial kernel, for example, based on the overlap in knockouts between any pair of experiments. Applicants learn a weighted combination of kernel vectors that fits a collection of training data, and use the coefficients to predict the outcome of new experiments. Here, the density of nonlinear interaction terms can be much greater, since Applicants do not directly learn any particular interaction coefficient, but rather a kernelized version of the entire polynomial. Indeed, if the interaction terms are too sparse, kernel learning is unlikely to be successful with under-sampling.</div>
</li> <li> <para-num num="[0346]"> </para-num> <div class="description-line" id="p-0295" num="0346">Applicants analyzed 3-way interaction data measured by overexpression of every 3-way combination of 39 miRNAs and a phenotype of drug resistance, and confirmed substantial sparsity in the data. Applicants analyzed the 5-way interactions affecting expression profiles in response to salt in yeast between the MAPK Hogl (p38 ortholog) and 4 TFs (1, 2, 3, 4, and 5 KO: 32 perturbations). Using a (non-regularized) linear model, Applicants quantified 1- and 2-way interactions, finding diverse non-linearities.</div>
</li> <heading id="h-0018">Analyzing a Cell Population at the Single Cell Level</heading>
<li> <para-num num="[0347]"> </para-num> <div class="description-line" id="p-0296" num="0347">The method according to the invention may comprise a step for single-cell molecular profiling. In some embodiments the step may comprise processing said cell population in order to physically separate cells. In some embodiments the step may comprise single-cell manipulation, e.g. using microfluidics based techniques. In some embodiments the step may comprise reverse emulsion droplet-based single-cell analysis or hydrogel droplet-based single-cell analysis.</div>
</li> <li> <para-num num="[0348]"> </para-num> <div class="description-line" id="p-0297" num="0348">The method of the invention may use microfluidics, e.g. to culture cells in specific combinations, control the spatiotemporal signals they receive, and/or trace and sample them as desired.</div>
</li> <heading id="h-0019">Molecular Profiling at the Single Cell Level</heading>
<li> <para-num num="[0349]"> </para-num> <div class="description-line" id="p-0298" num="0349">The method according to the invention may comprise a step for single-cell molecular profiling. This step may involve analyzing biomolecules quantitatively or semi-quantitatively. The biomolecules may include RNA, mRNA, pre-mRNA, proteins, chromatin or DNA. Said analysis may be performed genome-wide. Said analysis may be coupled (dual or sequential analysis of two or more types of biomolecules).</div>
</li> <li> <para-num num="[0350]"> </para-num> <div class="description-line" id="p-0299" num="0350">In some embodiments the step may comprise single-cell genomic profiling, single-cell RNA profiling, single-cell DNA profiling, single-cell epigenomic profiling, single-cell protein profiling, or single-cell reporter gene expression profiling. Proteins that may be used to alter genomic and epigenomic state are described in Shmakov et al., 2015, Molecular Cell 60, 1-13 and Zetsche et al., 2015, Cell 163, 759-771.</div>
</li> <li> <para-num num="[0351]"> </para-num> <div class="description-line" id="p-0300" num="0351">In some embodiments the step may comprise single-cell RNA abundance analysis, single-cell transcriptome analysis, single-cell exome analysis, single-cell transcription rate analysis, or single-cell RNA degradation rate analysis.</div>
</li> <li> <para-num num="[0352]"> </para-num> <div class="description-line" id="p-0301" num="0352">In some embodiments the step may comprise single-cell DNA abundance analysis, single-cell DNA methylation profiling, single-cell chromatin profiling, single-cell chromatin accessibility profiling, single-cell histone modification profiling, or single-cell chromatin indexing.</div>
</li> <li> <para-num num="[0353]"> </para-num> <div class="description-line" id="p-0302" num="0353">In some embodiments the step may comprise single-cell protein abundance analysis, single-cell post-translational protein modification analysis, or single-cell proteome analysis.</div>
</li> <li> <para-num num="[0354]"> </para-num> <div class="description-line" id="p-0303" num="0354">In some embodiments the step may comprise single-cell mRNA reporter analysis, detection or quantification.</div>
</li> <li> <para-num num="[0355]"> </para-num> <div class="description-line" id="p-0304" num="0355">In some embodiments the step may comprise single-cell dual molecular profiling, such any combination of two amongst single-cell RNA profiling, single-cell DNA profiling, single-cell protein profiling, mRNA reporter analysis.</div>
</li> <li> <para-num num="[0356]"> </para-num> <div class="description-line" id="p-0305" num="0356">The method of the invention may include at the step determining single cell RNA levels. For single cell RNA-Seq (scRNA-Seq), one may use Drop-Seq (Macosko, E. Z., Basu, A., Satija, R., Nemesh, J., Goldman, M., Tirosh, I., Bialas, A. R., Kamitaki, N., Sanes, J. R., Weitz, D. A., Shalek, A. K., Regev, A. &amp; McCarroll, S. A. Highly Parallel Genome-wide Expression Profiling of Individual Cells Using Nanoliter Droplets. <i>Cell. </i>2015 May 21;161(5):1202-14. doi: 10.1016/j.cell.2015.05.002. PMCID:4481139) and variants thereof. This technique relies on reverse-emulsion, early barcoding for analyzing 10<sup>4</sup>-10<sup>6 </sup>cells/experiment at very low cost. Drop-Seq enables to co-encapsulate individual cells with uniquely barcoded mRNA capture beads in reverse emulsion droplets. After lysis and mRNA capture, the emulsion is broken and all beads/cells are processed (RT, library prep) together, deconvolving each cell's profile from bead barcodes. In some embodiments, droplets can compartmentalize hundreds of cells/sec, are stable over time and to heat, and can serve as micro-vessels to add reagents; after RT, barcoded beads are stable and can be sorted or subselected. Sampling noise from shallow read depth is substantially lower than the technical variability between cells (Shalek, A. K., Satija, R., Shuga, J., Trombetta, J. J., Gennert, D., Lu, D., Chen, P., Gertner, R. S., Gaublomme, J. T., Yosef, N., Schwartz, S., Fowler, B., Weaver, S., Wang, J., Wang, X., Ding, R., Raychowdhury, R., Friedman, N., Hacohen, N., Park, H., May, A. P. &amp; Regev, A. Single-cell RNA-seq reveals dynamic paracrine control of cellular variation. <i>Nature. </i>510, 363-369, doi:10.1038/nature13437 (2014). PMCID:4193940.), so one may sufficiently estimate expression with ˜100,000 reads per cell for many applications (especially with a 5′ or 3′-end protocol, Satija, R., Farrell, J. A., Gennert, D., Schier, A. F. &amp; Regev, A. Spatial reconstruction of single-cell gene expression data. <i>Nature biotechnology. </i>33, 495-502, doi:10.1038/nbt.3192 (2015)).</div>
</li> <li> <para-num num="[0357]"> </para-num> <div class="description-line" id="p-0306" num="0357">Single cell RNA may also be analyzed as described in Klein, A. M., Mazutis, L., Akartuna, I., Tallapragada, N., Veres, A., Li, V., Peshkin, L., Weitz, D.A., Kirschner, M. W. Droplet barcoding for single-cell transcriptomics applied to embryonic stem cells. <i>Cell. </i>2015 May 21; 161(5):1187-201. doi: 10.1016/j.cell.2015.04.044. PMCID: 4441768.</div>
</li> <li> <para-num num="[0358]"> </para-num> <div class="description-line" id="p-0307" num="0358">The method of the invention may include determining RNA transcription and degradation rates. One may use RNA metabolically labeled with 4-thiouridine, to measure RNA transcription and degradation rates (Rabani, M., Raychowdhury, R., Jovanovic, M., Rooney, M., Stumpo, D. J., Pauli, A., Hacohen, N., Schier, A. F., Blackshear, P. J., Friedman, N., Amit, I. &amp; Regev, A. High-resolution sequencing and modeling identifies distinct dynamic RNA regulatory strategies. <i>Cell. </i>159, 1698-1710, doi:10.1016/j.cell.2014.11.015 (2014). PMCID:4272607; Rabani, M., Levin, J. Z., Fan, L., Adiconis, X., Raychowdhury, R., Garber, M., Gnirke, A., Nusbaum, C., Hacohen, N., Friedman, N., Amit, I. &amp; Regev, A. Metabolic labeling of RNA uncovers principles of RNA production and degradation dynamics in mammalian cells. <i>Nature biotechnology. </i>29, 436-442, doi:10.1038/nbt.1861 (2011). PMCID:3114636).</div>
</li> <li> <para-num num="[0359]"> </para-num> <div class="description-line" id="p-0308" num="0359">The method of the invention may include a step of determining DNA methylation. One may apply methods for reduced representation (RRBS), targeted capture, and whole genome bisulfate sequencing of DNA methylation from bulk to ultra-low inputs (Chan, M. M., Smith, Z. D., Egli, D., Regev, A. &amp; Meissner, A. Mouse ooplasm confers context-specific reprogramming capacity. Nature genetics. 44, 978-980, doi:10.1038/ng.2382 (2012). PMCID:3432711; Smith, Z. D., Chan, M. M., Humm, K. C., Karnik, R., Mekhoubad, S., Regev, A., Eggan, K. &amp; Meissner, A. DNA methylation dynamics of the human preimplantation embryo. Nature. 511, 611-615, doi:10.1038/nature13581 (2014). PMCID:4178976; Smith, Z. D., Chan, M. M., Mikkelsen, T. S., Gu, H., Gnirke, A., Regev, A. &amp; Meissner, A. A unique regulatory phase of DNA methylation in the early mammalian embryo. Nature. 484, 339-344, doi:10.1038/nature10960 (2012). PMCID:3331945) to single cells.</div>
</li> <li> <para-num num="[0360]"> </para-num> <div class="description-line" id="p-0309" num="0360">The method of the invention may include a step determining Chromatin accessibility. This may be performed by ATAC-Seq. For massively parallel single cell ATAC-Seq one may implement a droplet-based assay. First, in-tube, one may use Tn5 transposase to fragment chromatin inside isolated intact nuclei and add universal primers at cutting sites. Next, in-drop, one may use a high diversity library of barcoded primers to uniquely tag all DNA that originated from the same single cell. Alternatively, one may perform all steps in drop. One may also use a strategy that relies on split pooled nuclei barcoding in plates (Cusanovich, D. A., Daza, R., Adey, A., Pliner, H., Christiansen, L., Gunderson, K. L., Steemers, F. J., Trapnell, C. &amp; Shendure, J. Multiplex single-cell profiling of chromatin accessibility by combinatorial cellular indexing. <i>Science. </i>2015 May 22;348(6237):910-4. doi: 10.1126/science.aab1601. Epub 2015 May 7). Applicants have optimized key steps in a mixture of human and mouse cells, with specificity that exceeds the initial performance of mRNA Drop-Seq. Applicants have also used a Fluidigm C1 protocol (see www.fluidigm.com/products/cl-system) to analyze ˜100 single DCs, closely reproducing ensemble measures, high enrichment in TSSs, and nucleosome-like periodicity.</div>
</li> <li> <para-num num="[0361]"> </para-num> <div class="description-line" id="p-0310" num="0361">ATAC-seq (assay for transposase-accessible chromatin) identifies regions of open chromatin using a hyperactive prokaryotic Tn5-transposase, which preferentially inserts into accessible chromatin and tags the sites with sequencing adaptors (Pott and Lieb Genome Biology (2015) 16:172 DOI 10.1186/s13059-015-0737-7 and Buenrostro J D, Giresi P G, Zaba L C, Chang H Y, Greenleaf W J. Transposition of native chromatin for fast and sensitive epigenomic profiling of open chromatin, DNA-binding proteins and nucleosome position. Nat Methods. 2013;10:1213-128). Two very different approaches were used: one relied on physical isolation of single cells (Buenrostro J D, Wu B, Litzenburger U M, Ruff D, Gonzales M L, Snyder M P, et al. Single-cell chromatin accessibility reveals principles of regulatory variation. Nature. 2015;523:486-90), and the other avoided single-cell reaction volumes by using a two-step combinatorial indexing strategy (Cusanovich D A, Daza R, Adey A, Pliner H A, Christiansen L, Gunderson K L, et al. Epigenetics. Multiplex single-cell profiling of chromatin accessibility by combinatorial cellular indexing. Science. 2015;348:910-4).</div>
</li> <li> <para-num num="[0362]"> </para-num> <div class="description-line" id="p-0311" num="0362">In the indexing scheme, Cusanovich et al. [Cusanovich D A, Daza R, Adey A, Pliner H A, Christiansen L, Gunderson K L, et al. Epigenetics. Multiplex single-cell profiling of chromatin accessibility by combinatorial cellular indexing. Science. 2015;348:910-4] lysed cells, and 2500 nuclei were placed into each well of a 96-well plate. Transposases loaded with unique adaptors were added to each well, creating 96 pools of approximately 2500 nuclei, each pool with distinct barcodes. Nuclei from all of the transposition reactions were mixed, and using a fluorescence-activated cell sorter (FACS) 15-25 nuclei were deposited into each well of a second 96-well plate. Nuclei in each well of this second plate were lysed, and the DNA was amplified using a primer containing a second barcode. The low number of nuclei per well ensured that about 90% of the resulting barcode combinations were unique to a single cell. This combinatorial indexing strategy enabled the recovery of 500-1500 cells with unique tags per experiment. Overall Cusanovich et al. obtained scATAC-seq data from over 15,000 individual cells from mixtures of GM12878 lymphoblastoid cells with HEK293, HL-60, or mouse Patski cells. The number of reads associated with any single cell was very low, varying from 500 to about 70,000 with a median of fewer than 3000 reads per cell.</div>
</li> <li> <para-num num="[0363]"> </para-num> <div class="description-line" id="p-0312" num="0363">Buenrostro et al. [Buenrostro J D, Wu B, Litzenburger U M, Ruff D, Gonzales M L, Snyder M P, et al. Single-cell chromatin accessibility reveals principles of regulatory variation. Nature. 2015;523:486-90] used a programmable microfluidic device (C1, Fluidigm) to isolate single cells and perform ATAC-seq on them in nanoliter reaction chambers. Each nanochamber was analyzed under a microscope to ensure that a single viable cell had been captured. This approach is simple and has the significant advantage of a carefully monitored reaction environment for each individual cell, although the throughput was limited to processing 96 cells in parallel. Buenrostro et al. sampled 1632 cells from eight different cell lines, including GM12878, K562, and H1 cells, and obtained an average of 73,000 reads per cell, about 20 times the number of reads per cell obtained using the barcoding strategy.</div>
</li> <li> <para-num num="[0364]"> </para-num> <div class="description-line" id="p-0313" num="0364">The method of the invention may include a step of determining histone modifications and protein-DNA interactions. One may apply tools that use genomic barcoding to index chromatin prior to immunoprecipitation to enable multiplexed analysis of limited samples and individual cells in a single reaction. For single-cell chromatin profiling, one may use Drop-ChIP where the chromatin of individual cells is barcoded in droplets. Based on the Drop-Seq technique, one may encapsulate single cells, lyse and MNase-digest chromatin, then fuse a second droplet with barcoded oligos, ligate them to the fragmented chromatin, break the emulsion, add carrier chromatin, and carry out ChIP-Seq. this may be performed using a protocol with split-pool barcoding to collect 10<sup>4</sup>-10<sup>5 </sup>single cells/assay.</div>
</li> <li> <para-num num="[0365]"> </para-num> <div class="description-line" id="p-0314" num="0365">ChIP-sequencing, also known as ChIP-seq, is a method used to analyze protein interactions with DNA which may be used with perturbation. ChIP-seq combines chromatin immunoprecipitation (ChIP) with massively parallel DNA sequencing to identify the binding sites of DNA-associated proteins. It can be used to map global binding sites precisely for any protein of interest. ChIP-seq is used primarily to determine how transcription factors and other chromatin-associated proteins influence phenotype-affecting mechanisms. Determining how proteins interact with DNA to regulate gene expression is for understanding many biological processes and disease states. This epigenetic information is complementary to genotype and expression analysis. ChIP-seq technology is as an alternative to ChIP-chip which requires a hybridization array. Specific DNA sites in direct physical interaction with transcription factors and other proteins can be isolated by chromatin immunoprecipitation. ChIP produces a library of target DNA sites bound to a protein of interest in vivo. Massively parallel sequence analyses are used in conjunction with whole-genome sequence databases to analyze the interaction pattern of any protein with DNA, see, e.g., Johnson D S, Mortazavi A et al. (2007) Genome-wide mapping of in vivo protein—DNA interactions. Science 316: 1497-1502, or the pattern of any epigenetic chromatin modifications. This can be applied to the set of ChIP-able proteins and modifications, such as transcription factors, polymerases and transcriptional machinery, structural proteins, protein modifications, and DNA modifications. See, e.g., “Whole Genome Chromatin IP Sequencing,” Illumina, Inc (2010), available at www.illumina.com/Documents/products/datasheets/datasheet chip sequence.pdf (Chromatin Immunoprecipitation with massively parallel sequencing).</div>
</li> <li> <para-num num="[0366]"> </para-num> <div class="description-line" id="p-0315" num="0366">For multiplex analysis of (limited) bulk samples, one may rely on chromatin indexing (MINT-ChIP; iChIP), where MNase-fragmented chromatin is indexed by ligation to a uniquely barcoded adaptor and then pooled and processed in multiplex through all subsequent phases, either with (MINT-ChIP) or without (iChIP: Lara-Astiaso, D., Weiner, A., Lorenzo-Vivas, E., Zaretsky, I., Jaitin, D. A., David, E., Keren-Shaul, H., Mildner, A., Winter, D., Jung, S., Friedman, N. &amp; Amit, I. Immunogenetics. Chromatin state dynamics during blood formation. <i>Science. </i>345, 943-949, doi:10.1126/science.1256271 (2014). PMCID:4412442) carrier chromatin (without adaptors).</div>
</li> <li> <para-num num="[0367]"> </para-num> <div class="description-line" id="p-0316" num="0367">The method of the invention may include a step of determining proteins. Recently developed assays (e.g., CyTOF: Bendall, S. C., Simonds, E. F., Qiu, P., Amir el, A. D., Krutzik, P. O., Finck, R., Bruggner, R. V., Melamed, R., Trejo, A., Ornatsky, O. I., Balderas, R. S., Plevritis, S. K., Sachs, K., Pe'er, D., Tanner, S. D. &amp; Nolan, G. P. Single-cell mass cytometry of differential immune and drug responses across a human hematopoietic continuum. <i>Science. </i>332, 687-696, doi:10.1126/science.1198704 (2011). PMCID:3273988), allow multiplexed, single cell detection of dozens of proteins in millions of cells, but rely on antibodies and cannot yet be combined with DNA readout. Conversely, mass spectrometry (LC-MS/MS) allows quantitative analysis of entire proteomes, but deep analysis requires large amounts of protein/cells. To measure single cell protein levels and post-translational modifications (PTMs), one may use one of three complementary antibody-based assays: (1) standard flow cytometry with a few proteins/PTMs, &gt;10<sup>6 </sup>single cells); (2) CyTOF (Bendall, S. C., Simonds, E. F., Qiu, P., Amir el, A. D., Krutzik, P. O., Finck, R., Bruggner, R. V., Melamed, R., Trejo, A., Ornatsky, O. I., Balderas, R. S., Plevritis, S. K., Sachs, K., Pe'er, D., Tanner, S. D. &amp; Nolan, G. P. Single-cell mass cytometry of differential immune and drug responses across a human hematopoietic continuum. <i>Science. </i>332, 687-696, doi:10.1126/science.1198704 (2011). PMCID:3273988) (heavy metal labeling with multiplex barcoding; ˜30-50 proteins/PTMs, 10<sup>5</sup>-10<sup>6 </sup>single cells); and (3) novel, highly multiplexed, DNA sequencing-based readouts of protein levels (100s proteins/PTMs; 10<sup>6 </sup>cells). For sequencing based readouts, one may use one of two approaches, geared at detecting hundreds of proteins in single cells: Immuno-Seq (when antibodies can be washed out: Niemeyer, C. M., Adler, M. &amp; Wacker, R. Detecting antigens by quantitative immuno-PCR. <i>Nat Protoc. </i>2, 1918-1930, doi:10.1038/nprot.2007.267 (2007)) and proximity extension assays (PEA, when antibodies cannot be washed away: Hammond, M., Nong, R. Y., Ericsson, O., Pardali, K. &amp; Landegren, U. Profiling cellular protein complexes by proximity ligation with dual tag microarray readout. <i>PLoS One. </i>7, e40405, doi:10.1371/journal.pone.0040405 (2012). PMCID:3393744; Nong, R. Y., Wu, D., Yan, J., Hammond, M., Gu, G. J., Kamali-Moghaddam, M., Landegren, U. &amp; Darmanis, S. Solid-phase proximity ligation assays for individual or parallel protein analyses with readout via real-time PCR or sequencing. <i>Nat Protoc. </i>8, 1234-1248, doi:10.1038/nprot.2013.070 (2013); Stahlberg, A., Thomsen, C., Ruff, D. &amp; Aman, P. Quantitative PCR analysis of DNA, RNAs, and proteins in the same single cell. <i>Clin Chem. </i>58, 1682-1691, doi:10.1373/clinchem.2012.191445 (2012).) These use DNA-sequence based encoding, and are compatible with other genomic readouts (e.g., sgRNA barcodes). DNA-sequence tags can be conjugated to antibodies (Janssen, K. P., Knez, K., Spasic, D. &amp; Lammertyn, J. Nucleic acids for ultra-sensitive protein detection. <i>Sensors </i>(Basel). 13, 1353-1384, doi:10.3390/s130101353 (2013). PMCID:3574740), nanobodies (Pardon, E., Laeremans, T., Triest, S., Rasmussen, S. G., Wohlkonig, A., Ruf, A., Muyldermans, S., Hol, W. G., Kobilka, B. K. &amp; Steyaert, J. A general protocol for the generation of Nanobodies for structural biology. <i>Nat Protoc. </i>9, 674-693, doi:10.1038/nprot.2014.039 (2014). PMCID:4297639; Theile, C. S., Witte, M. D., Blom, A. E., Kundrat, L., Ploegh, H. L. &amp; Guimaraes, C. P. Site-specific N-terminal labeling of proteins using sortase-mediated reactions. <i>Nat Protoc. </i>8, 1800-1807, doi:10.1038/nprot.2013.102 (2013). PMCID:3941705) or aptamers (Janssen, K. P., Knez, K., Spasic, D. &amp; Lammertyn, J. Nucleic acids for ultra-sensitive protein detection. <i>Sensors </i>(Basel). 13, 1353-1384, doi:10.3390/s130101353 (2013). PMCID:3574740.). Detection of specific transcripts and proteins in single cells may be performed as well (Frei et al., Highly multiplexed simultaneous detection of RNAs and proteins in single cells. <i>Nat Methods. </i>2016 March; 13(3):269-75.)</div>
</li> <li> <para-num num="[0368]"> </para-num> <div class="description-line" id="p-0317" num="0368">In certain embodiments of the present invention, quantitative measurements of both the copy number and spatial distribution of large fractions of the transcriptome in single cells is measured by multiplexed error-robust fluorescence in situ hybridization (MERFISH) (Moffitt and Zhuang, Chapter One—RNA Imaging with Multiplexed Error-Robust Fluorescence In Situ Hybridization (MERFISH), <i>Methods in Enzymology. </i>Volume 572, 2016, Pages 1-49).</div>
</li> <heading id="h-0020">Multiplex Analysis of Single Cell Constituents</heading>
<li> <para-num num="[0369]"> </para-num> <div class="description-line" id="p-0318" num="0369">It is an object of the present invention to provide a method for the high multiplex analysis of cellular constituents by linking nucleic acids tags to existing ligand binding and/or antibody technologies to enable proteomic or cellular constituent detection and relative quantification by next-generation sequencing (NGS) in single cells or isolated aggregations of cellular constituents. The present invention may combine perturbation of single cells followed by protein analysis in the single cells, Thus, protein expression may be linked to a perturbation.</div>
</li> <li> <para-num num="[0370]"> </para-num> <div class="description-line" id="p-0319" num="0370">It is a further object of the present invention to provide for comparing high multiplex protein data variation between single cells or isolated aggregation of cellular constituents and between different biological conditions (e.g. healthy vs. diseased states; one genetic perturbation vs. another, different genetic backgrounds).</div>
</li> <li> <para-num num="[0371]"> </para-num> <div class="description-line" id="p-0320" num="0371">It is a further object of the present invention to provide massively parallel profiling of all circuit aspects in single cells or isolated aggregations of cellular constituents: from RNA to chromatin organization to protein levels.</div>
</li> <li> <para-num num="[0372]"> </para-num> <div class="description-line" id="p-0321" num="0372">In a first aspect, the present invention provides a method of assaying segregated cellular constituents, comprising: admixing at least one isolated aggregation of cellular constituents with monomers of a polymerizable gel; polymerizing the gel, to embed the cellular constituents in discrete polymer matrices; incubating the cellular constituents embedded in the polymer matrices with one or more labeling ligands with specific binding affinity for one or more target cellular constituents to produce one or more labeled cellular constituents in the polymer matrices, wherein each of the one or more labeling ligand comprises a bound oligonucleotide label comprising a unique constituent identifier (UCI) sequence, and wherein the incubation comprises binding conditions under which the labeling ligand will bind to the cellular constituent within the polymer matrix, and the incubation further comprises washing conditions under which unbound labeling ligands will be washed out of the polymer matrix; and sequencing the oligonucleotide label, whereby detecting the UCI by sequencing indicates the presence of the target cellular constituent.</div>
</li> <li> <para-num num="[0373]"> </para-num> <div class="description-line" id="p-0322" num="0373">Cellular constituents may include any molecule within a cell; i.e. proteins, nucleic acids, or post translational modifications (PTM). The cellular constituent may be a protein, RNA transcript, metabolite, or a DNA molecule. Specific cellular constituents may be proteins, modified proteins, hormones, cytokines, cellular metabolites, or carbohydrates. The isolated aggregation of cellular constituents may be a cell, an extracellular vesicle, an organelle, or an organized subcomponent thereof, including molecular complexes. Isolated aggregations of cellular constituents may include separate organelles of a single cell or separate organelles isolated from a population of cells. Organelles may be for example, mitochondria, nuclei, or cellular vesicles. In one embodiment, a specific type of single cells may be isolated. In one embodiment, immune cells are isolated from a population of cells. Not being bound by a theory, single mitochondria can be purified from a population of cells and the relative amounts of constituents present in each individual mitochondrion may be analyzed. Not being bound by a theory, immune cells may be isolated by a method such as cell sorting and the relative representation of cellular constituents may be determined for each individual cell.</div>
</li> <li> <para-num num="[0374]"> </para-num> <div class="description-line" id="p-0323" num="0374">The step of admixing the isolated aggregation of cellular constituents with monomers may be carried out in an aqueous solution, or in an aqueous aliquot or droplet present in an oil emulsion. The polymer matrix may be a hydrogel. The polymer matrix may be any hydrogel capable of polymerization to create a solid matrix that fixes the cellular constituents and provides a porosity capable of allowing labeling ligands to freely diffuse through the network of pores. The cellular constituents may be further fixed by treating with an aldehyde. The aldehyde may be formaldehyde, paraformaldehyde, or glutaraldehyde. Not being bound by a theory the fixation in a solid matrix prevents the mixing of the cellular constituents between the isolated aggregations of cellular constituents. Not being bound by a theory, capturing cellular constituents in a solid polymer mesh insures that they are physical units that can be ligand and/or antibody stained as a pool and isolated as single cells or isolated aggregates of cellular constituents subsequently. Not being bound by a theory, the fixing of cellular constituents in the polymer matrix allows access to the labeling ligands to intracellular constituents.</div>
</li> <li> <para-num num="[0375]"> </para-num> <div class="description-line" id="p-0324" num="0375">The physical units formed by the polymer matrix may be particles, droplets, or a continuous polymer matrix with discrete regions comprising the isolated aggregates of cellular constituents. Therefore, the polymer matrix may include more than one isolated aggregate of cellular constituents. The polymer matrix may be divided such that isolated aggregates of cellular constituents are separable. The polymer matrix may be separable in that individual particles, droplets, or sections can be isolated. They may be isolated by physical manipulation using a sorting device. The sorting device may use microfluidics. They may be separated by use of dilution or manual manipulation by a user. They may be separated by use any kind of (micro) dissection. The cellular constituents within the polymer matrix may be stained with a dye, or a dye-conjugated ligand indicating the location of individual cellular constituents or cells. The polymer matrix may be punched to isolate a core, wherein each core from the polymer matrix contains a single isolated aggregate of cellular constituents. Not being bound by a theory, the fixation of isolated aggregates of cellular constituents in a matrix allows each isolated aggregate of cellular constituents to be compartmentalized wherein the separate compartments can be treated in a single experimental vessel or container and separated subsequently.</div>
</li> <li> <para-num num="[0376]"> </para-num> <div class="description-line" id="p-0325" num="0376">The labeling ligands are linked with an oligonucleotide label that can be used to determine the identity of the ligand. Each oligonucleotide label may comprise a unique constituent identifier (UCI) that can be used to determine the presence of a cellular constituent. Not being bound by a theory, the availability of unique sequences allows the labeling and detection of a plurality of ligands each for a specific constituent. Not being bound by a theory, the UCI allows a DNA readout for detection of a cellular constituent. The DNA readout may be by any sequencing method or method of amplification, such as by PCR or next generation sequencing. The oligonucleotide label may additionally include a promoter for amplification by an RNA polymerase, such as T7 polymerase. Not being bound by a theory, amplification by T7 polymerase allows amplification of low represented sequences, whereas such sequences may be diluted out by domination of a higher represented sequence during PCR. Not being bound by a theory, the labeling of each labeling ligand with a unique UCI allows the identification of more than ten, or hundred, or thousands of cellular constituents in an isolated aggregation of cellular constituents.</div>
</li> <li> <para-num num="[0377]"> </para-num> <div class="description-line" id="p-0326" num="0377">The method may further comprise segregating the discrete polymer matrices comprising the labeled constituents before the step of sequencing. Segregating the discrete polymer matrices may be by sorting single discrete matrices into separate reaction vessels. Segregating the discrete polymer matrices may be by forming discrete unique-identifier-transfer compositions, each comprising the cellular constituents embedded in a discrete polymer matrix and a transfer particle, wherein: the ligand oligonucleotide label further comprises a capture sequence, and the UCI and capture sequence are together releasably attached to the labeling ligand; the labelling ligand is bound to the target cellular constituent; and, the transfer particle comprises: a capture-binding-sequence having specific binding affinity for the capture sequence attached to the UCI, and, a unique source identifier (USI) sequence that is unique to each transfer particle. The USI of each transfer particle preferably comprises 4-15 nucleotides. The method may further comprise releasing the UCI from the labeled ligand, under conditions within the unique-identifier-transfer composition so that the released capture sequence binds to the capture-binding-sequence on the transfer particle, thereby transferring the UCI to the transfer particle. The transfer particle may be a solid bead. The transfer particle may be a hydrogel bead. The transfer particle may also be used to capture nucleic acids present in a discrete polymer matrix. The nucleic acids may be RNA and/or DNA. Not being bound by a theory the transfer particle may be used to capture both the UCI and the nucleic acids, whereby the source of the bound cellular constituents and nucleic acids can be determined after sequencing.</div>
</li> <li> <para-num num="[0378]"> </para-num> <div class="description-line" id="p-0327" num="0378">The method may further comprise, before the sequencing step, generating a USI for each discrete polymer matrix by a split pool ligation method, wherein the oligonucleotide label further comprises a universal ligation handle (ULH) sequence configured to produce a DNA overhang capable of hybridization to a complementary over hang on a first index nucleotide sequence, wherein the first index nucleotide sequence comprises an overhang complementary to a final index sequence or optionally a middle index sequence, wherein the middle index sequence comprises overhangs complementary to the first index sequence and to the final index sequence or optionally to another middle index sequence and final index sequence, wherein the final index sequence has a single overhang complementary to the preceding index sequence, and wherein the first, middle, and final index sequences are selected from a plurality of unique sequences comprising compatible DNA overhangs and 10 to 30 base pairs of unique sequence.</div>
</li> <li> <para-num num="[0379]"> </para-num> <div class="description-line" id="p-0328" num="0379">The split pool ligation method may comprise: splitting the pool of discrete polymer matrices into separate pools of polymer matrices, each containing a unique first index sequence; ligating the first index sequence to the ligation handle; pooling the discrete polymer matrices; optionally, splitting the pool of discrete polymer matrices into separate pools each containing a unique middle index sequence; ligating the middle index sequence to the first index sequence; and pooling the discrete polymer matrices; optionally, repeating the steps with another middle index sequence; splitting the pool of discrete polymer matrices into pools containing a unique final index sequence; and ligating the final index sequence to the preceding index sequence, whereby each discrete polymer matrix comprises a USI. The USI may have no middle index sequence, one middle index sequence, two middle index sequences, preferably the USI has a first, middle, and final index sequence. Not being bound by a theory, the size of the unique sequences in each index determines the amount included. Not being bound by a theory, the number of indices selected is the amount necessary such that the probability of having identical USI sequences on spate polymer matrices is approaching zero. In an exemplary embodiment, each index includes 192 unique sequences.</div>
</li> <li> <para-num num="[0380]"> </para-num> <div class="description-line" id="p-0329" num="0380">The ligation handle may comprise a restriction site for producing an overhang complementary with a first index sequence overhang, and wherein the method further comprises digestion with a restriction enzyme. The ligation handle may comprise a nucleotide sequence complementary with a ligation primer sequence and wherein the overhang complementary with a first index sequence overhang is produced by hybridization of the ligation primer to the ligation handle. Additionally, the ULH may comprise a dsDNA part that already includes the overhang needed for index ligation.</div>
</li> <li> <para-num num="[0381]"> </para-num> <div class="description-line" id="p-0330" num="0381">The UCI may comprise 4 to 30 nucleotides or 7 to 30 nucleotides, preferably about 21 nucleotides. The oligonucleotide label may further comprise a unique molecular identifier (UMI) sequence. The first, middle, or final index sequence may further comprises a unique molecular identifier (UMI) sequence. The UMI may comprise 4-20 nucleotides. The UMI may comprise 8 to 16 nucleotides.</div>
</li> <li> <para-num num="[0382]"> </para-num> <div class="description-line" id="p-0331" num="0382">The isolated aggregation of cellular constituents may be a cell, an extracellular vesicle, an organelle, or an organized subcomponent thereof</div>
</li> <li> <para-num num="[0383]"> </para-num> <div class="description-line" id="p-0332" num="0383">The sequencing may comprise combining a primer having a unique source identifier (USI) sequence with UCI, so that the USI and UCI sequences are sequenced together, and the USI preferably comprises 20 to 120 nucleotides.</div>
</li> <li> <para-num num="[0384]"> </para-num> <div class="description-line" id="p-0333" num="0384">The step of admixing the isolated aggregation of cellular constituents with monomers may be carried out in an aqueous aliquot or in a droplet formed by an aqueous solution in oil emulsion. The aqueous aliquot may be a separate reaction vessel such as a well in a plate. The droplet may be formed by a microfluidic device. The polymer matrix may be a hydrogel.The method may be a multiplex assay with a plurality of labeling ligands, each labeling ligand have a distinct UCI. The labeling ligand may be non-covalently bound to the target cellular constituent.</div>
</li> <li> <para-num num="[0385]"> </para-num> <div class="description-line" id="p-0334" num="0385">The method may further comprise pooling the oligonucleotide labels comprising a USI from a plurality of polymer matrices and sequencing the pooled UCI sequences and USI sequences. The method may further comprise pooling the oligonucleotide labels comprising a USI and UMI from a plurality of polymer matrices and sequencing the pooled UCI sequences, USI sequences, and UMI sequences.</div>
</li> <li> <para-num num="[0386]"> </para-num> <div class="description-line" id="p-0335" num="0386">The method may further comprise washing the cellular constituents embedded in the polymer matrices to remove selected cellular components from the polymer matrices before incubating the cellular constituents with the labeling ligand. The washing may comprise treating the cellular constituents embedded in the polymer matrices with a detergent so as to remove lipids from the polymer matrices before incubating the cellular constituents with the labeling ligand. The detergent may be an anionic detergent or nonionic detergent. The detergent may be SDS, NP-40, triton X-100, or any other detergent known in the art capable of removing lipids.</div>
</li> <li> <para-num num="[0387]"> </para-num> <div class="description-line" id="p-0336" num="0387">The method may further comprise quantitating the relative amount of the UCI sequence associated with a first aggregation of cellular constituents to the amount of the same UCI sequence associated with a second aggregation of cellular constituents, whereby the relative differences of a cellular constituent between aggregations of cellular constituents are determined. The relative amount may be compared to a control sample. The control sample may have predetermined amounts of cellular constituents. There may be more than one control sample. There may be at least three control samples. The at least three control samples can be used to generate a standard curve upon which all of the other cellular constituents within discrete polymer matrices are compared. The control sample may comprise isolated aggregations of cellular constituents that were untreated as compared relative to isolated aggregations of cellular constituents that were treated with a different condition. Cells may be treated with drugs, small molecules, pathogens, hormones, cytokines, proteins, nucleic acids, virus particles, or grown in different cellular environments. Cells may be isolated from a diseased tissue. The cells from the diseased tissue may be compared to cells from non-diseased tissue. Cells may be treated with systems that knockout, decrease or increase expression of a gene. Cells may be treated with systems that knockout functional elements of a genome. Functional elements include, but are not limited to promoters, enhancers, repressors, centromeres, or telomeres. CRISPR systems may be used.</div>
</li> <li> <para-num num="[0388]"> </para-num> <div class="description-line" id="p-0337" num="0388">The labeling ligand may be an antibody or an antibody fragment. The antibody fragment may be a nanobody, Fab, Fab′, (Fab′)2, Fv, ScFv, diabody, triabody, tetrabody, Bis-scFv, minibody, Fab2, or Fab3 fragment. The labeling ligand may be an aptamer. The labeling ligand may be a nucleotide sequence complementary to a target sequence.</div>
</li> <li> <para-num num="[0389]"> </para-num> <div class="description-line" id="p-0338" num="0389">The method may comprise multiplex binding of two or more labeling ligands to each aggregation of cellular constituents. The two or more distinct labeling ligands may comprise complementary oligonucleotide sequences, so that binding of the labeling ligands to respective target cellular constituents that are in proximity permits the complementary sequences of the distinct ligands to hybridize, forming an amplifiable polynucleotide duplex. The method may further comprise amplifying the polynucleotide duplex to provide an amplified sequence that is a detectable signal that target cellular constituents are in proximity. The complementary oligonucleotide sequences, which serve as a start site for polymerase extension, can either be designed to query proximity of two specific cellular constituents, or it can be designed to be universal, thereby querying interactions between all members of the labeling ligand panel.</div>
</li> <li> <para-num num="[0390]"> </para-num> <div class="description-line" id="p-0339" num="0390">In one embodiment, at least two distinct labeling ligands comprise oligonucleotide sequences configured to be ligated, so that binding of the labeling ligands to respective target cellular constituents that are in proximity permits the oligonucleotide sequences of the distinct ligands to ligate, forming an amplifiable polynucleotide duplex. The method may further comprise amplifying the polynucleotide duplex to provide an amplified sequence that is a detectable signal that target cellular constituents are in proximity.</div>
</li> <li> <para-num num="[0391]"> </para-num> <div class="description-line" id="p-0340" num="0391">One of the labeling ligands may comprise an oligonucleotide label with a restriction enzyme site between the labeling ligand and the UCI, and wherein the method may further comprise treating with a restriction enzyme, whereby the UCI from the labeling ligand is transferred to the oligonucleotide label of the labeling ligand in proximity.</div>
</li> <li> <para-num num="[0392]"> </para-num> <div class="description-line" id="p-0341" num="0392">The method may further comprise labeling the aggregation of cellular constituents by fluorescent in situ hybridization.</div>
</li> <li> <para-num num="[0393]"> </para-num> <div class="description-line" id="p-0342" num="0393">The aggregation of cellular constituents may be a cell that is a member of a cell population. The cell may be transformed or transduced with one or more genomic sequence-perturbation constructs that perturb a genomic sequence in the cells, wherein each distinct genomic sequence-perturbation construct comprises a unique-perturbation-identifier (UPI) sequence unique to that construct. The genomic sequence-perturbation construct may comprise a sequence encoding a guide RNA sequence of a CRISPR-Cas targeting system. The method may further comprise multiplex transformation of the population of cells with a plurality of genomic sequence-perturbation constructs. The UPI sequence may be attached to a perturbation-sequence-capture sequence, and the microbeads may comprise a perturbation-sequence-capture-binding-sequence having specific binding affinity for the perturbation-sequence-capture sequence attached to the UPI sequence. The UPI sequence may be attached to a universal ligation handle sequence, whereby a USI may be generated by split-pool ligation. The method may further comprise multiplex sequencing of the pooled UCI sequences, USI sequences, and UPI sequences.</div>
</li> <li> <para-num num="[0394]"> </para-num> <div class="description-line" id="p-0343" num="0394">The oligonucleotide label may comprise a regulatory sequence configured for amplification by an RNA polymerase, such as T7 polymerase. The labeling ligands may comprise oligonucleotide sequences configured to hybridize to a transcript specific region. The oligonucleotide label may further comprise attachment chemistry, such as an acrylic phosphoramidite modification, whereby the modification allows for incorporation into the polymer matrices upon polymerization. The acrylic phosphoramidite may be Acrydite™ (Eurofins Scientific, Luxembourg). The method may further comprise amplification of the oligonucleotide label and USI by PCR or T7 amplification before sequencing. T7 amplification may be followed by cDNA generation and optionally amplification by PCR. The oligonucleotide label may further comprise at least one spacer sequence, preferably two spacer sequences. The oligonucleotide label may further comprise a photocleavable linker. The oligonucleotide label may further comprise a restriction enzyme site between the labeling ligand and UCI.</div>
</li> <li> <para-num num="[0395]"> </para-num> <div class="description-line" id="p-0344" num="0395">The discrete polymer matrices may be labeled and washed more than once. Discrete polymer matrices may be labeled with a marker specific for a cell type or cell cycle marker or developmental marker, or differentiation marker, or disease marker. The label may be a fluorescent label. The fluorescent label may be used to separate the discrete polymer matrices into distinct groups. The label may be used to identify a certain cell type prior to embedding it into a discrete polymer matrix. The discrete polymer matrices of a distinct group may then be labeled again with labeling ligands that contain an oligonucleotide label of the present invention. After novel information is obtained from the multiplex assay of the present invention, a ‘banked’ population of polymer matrices can be stained for newly identified markers and the population of interest can be sorted (enriched) for, and investigated more deeply.</div>
</li> <li> <para-num num="[0396]"> </para-num> <div class="description-line" id="p-0345" num="0396">In another aspect, the present invention provides a method of determining open chromatin in individual cells comprising: isolating single cells into droplets formed by an aqueous solution in oil emulsions, wherein the droplets further comprise Tn5-transposase loaded with two tagmentation adapters, wherein one adapter is configured for incorporation into a polymer matrix and the second adapter is configured with a ligation handle for generating a USI; incubating the droplets to allow cell lysis and tagmentation of open chromatin; infusing monomers of a polymerizable gel into the droplets; polymerizing the gel, to embed the cellular constituents in discrete polymer matrices; optionally incubating the polymer matrices with one or more labeling ligands with specific binding affinity for one or more target cellular constituents to produce one or more labeled cellular constituents in the polymer matrices, wherein each of the one or more labeling ligand comprises a bound oligonucleotide label comprising a unique constituent identifier (UCI) sequence and a sequence capable of hybridization to the tagmentation adapter configured for incorporation into a polymer matrix, and wherein the incubation comprises binding conditions under which the labeling ligand will bind to the cellular constituent within the polymer matrix and the oligonucleotide label will hybridize to said tagmentation adapter, and wherein the incubation further comprises washing conditions under which unbound labeling ligands will be washed out of the polymer matrix; and extending the genomic DNA and adapter DNA, whereby a continuous DNA strand is generated comprising the adapters, genomic DNA, and DNA overhang; optionally the oligonucleotide label bound to a labeling ligand; generating a USI at the DNA overhang by split-pool ligation; sequencing the continuous DNA strand, whereby open chromatin is determined and optionally the presence of a cellular constituent at a site of open chromatin is determined.</div>
</li> <li> <para-num num="[0397]"> </para-num> <div class="description-line" id="p-0346" num="0397">In another aspect, the present invention provides a method of measuring RNA levels in individual cells comprising: isolating single cells into droplets formed by an aqueous solution in oil emulsions, wherein the droplets comprise at least one labeling ligands specific for binding at one or more target RNA transcripts, wherein the labeling ligands are configured for incorporation into a polymer matrix and comprise a ligation handle for generating a USI; lysing the cells in the droplets under conditions wherein the labeling ligands will bind to the target RNA transcripts; injecting monomers of a polymerizable gel into the droplets; polymerizing the gel, to embed the labeling ligands in discrete polymer matrices; optionally, staining the discrete polymer matrices with at least one additional labeling ligand; generating a USI by split-pool ligation; and sequencing the resulting DNA, whereby RNA levels and optionally protein levels are determined in single cells. The droplets may comprise at least one pair of labeling ligands specific for binding at adjacent sites of one or more target RNA transcripts, wherein each pair of labeling ligands comprises one labeling ligand configured for incorporation into a polymer matrix and one labeling ligand comprising a ligation handle for generating a USI, and wherein the method may further comprise injecting a ligation reaction buffer comprising a ligase that is configured to allow ligation of the pair of labeling ligands if they are hybridized adjacently with single nucleotide resolution on the target RNA transcript, such that off target binding of labeling ligand does not get ligated, and will not be amplified in subsequent steps.</div>
</li> <li> <para-num num="[0398]"> </para-num> <div class="description-line" id="p-0347" num="0398">In another aspect, the present invention provides a method of assaying segregated cellular constituents, comprising: fixing and permeablizing at least one cell; incubating the fixed and permeablized cell(s) with one or more labeling ligands with specific binding affinity for one or more target cellular constituents to produce one or more labeled cell(s), wherein each of the one or more labeling ligands comprise a bound oligonucleotide label comprising a unique constituent identifier (UCI) sequence, and wherein the incubation comprises binding conditions under which the labeling ligand will bind to the cellular constituent within the cell(s), and the incubation further comprises washing conditions under which unbound labeling ligands will be washed from the cell(s); admixing the cell(s) with monomers of a polymerizable gel; isolating single cells into droplets formed by an aqueous solution in oil emulsions; polymerizing the gel, to embed the labeling ligands and other cellular constituents in discrete polymer matrices; optionally, staining the discrete polymer matrices with at least one additional labeling ligand; generating a USI by split-pool ligation; and sequencing the oligonucleotide label, whereby detecting the UCI by sequencing indicates the presence of the target cellular constituent. The labeling ligands in step (b) may comprise at least one pair of labeling ligands specific for binding at adjacent sites of one or more target RNA transcripts, wherein each pair of labeling ligands comprises one labeling ligand configured for incorporation into a polymer matrix and one labeling ligand comprising a ligation handle for generating a USI, and wherein the method further comprises ligating the pair of labeling ligands if they are within proximity after binding to the target RNA transcripts. Any of the preceding methods may comprise polymer matrices wherein they further comprise magnetic particles. In one embodiment, any hydrogel droplet encapsulated aggregations of cellular constituents may further comprise magnetic particles embedded into the droplets. Not being bound by a theory, magnetic particles enable magnetic separation, aiding in clean up and washing steps in multiple reactions. Not being bound by a theory, the use of magnetic particles greatly enhances automation and therefore throughput.</div>
</li> <li> <para-num num="[0399]"> </para-num> <div class="description-line" id="p-0348" num="0399">In another aspect, the present invention provides a method of assaying segregated cellular constituents, comprising: fixing and permeablizing at least one cell; incubating the fixed and permeablized cell(s) with one or more labeling ligands with specific binding affinity for one or more target cellular constituents to produce one or more labeled cell(s), wherein each of the one or more labeling ligands comprise a bound oligonucleotide label comprising a unique constituent identifier (UCI) sequence, and wherein the incubation comprises binding conditions under which the labeling ligand will bind to the cellular constituent within the cell(s), and the incubation further comprises washing conditions under which unbound labeling ligands will be washed from the cell(s); and sequencing the oligonucleotide label, whereby detecting the UCI by sequencing indicates the presence of the target cellular constituent. The cellular constituent may comprise a protein, RNA transcript, or a DNA molecule. The method may further comprise segregating the cell(s) before sequencing. The segregating the cell(s) may comprise sorting the single cell(s) into a separate reaction vessel(s). The segregating the cell(s) may comprise forming discrete unique-identifier-transfer compositions, each comprising a cell and a transfer particle, wherein: the oligonucleotide label further comprises a capture sequence, and the UCI and capture sequence are together releasably attached to the labeling ligand; the labelling ligand is bound to the target cellular constituent; and, the transfer particle comprises: a capture-binding-sequence having specific binding affinity for the capture sequence attached to the UCI, and, a unique source identifier (USI) sequence that is unique to each transfer particle, and the USI preferably comprises 4-15 nucleotides. The method may further comprise releasing the UCI from the labeled ligand, under conditions within the unique-identifier-transfer composition so that the released capture sequence binds to the capture-binding-sequence on the transfer particle, thereby transferring the UCI to the transfer particle. The method may further comprise, before sequencing in step, generating a USI for each cell(s) by a split pool ligation method, wherein the oligonucleotide label further comprises a universal ligation handle (ULH) sequence configured to produce a DNA overhang capable of hybridization to a complementary over hang on a first index nucleotide sequence, wherein the first index nucleotide sequence comprises an overhang complementary to a final index sequence or optionally a middle index sequence, wherein the middle index sequence comprises overhangs complementary to the first index sequence and to the final index sequence or optionally to another middle index sequence and final index sequence, wherein the final index sequence has a single overhang complementary to the preceding index sequence, and wherein the first, middle, and final index sequences are selected from a plurality of unique sequences comprising compatible DNA overhangs and 10 to 30 base pairs of unique sequence. The split pool ligation method may comprise: splitting the pool of cell(s) into separate pools of cell(s), each containing a unique first index sequence; ligating the first index sequence to the ligation handle; pooling the cell(s); optionally, splitting the pool of cell(s) into separate pools each containing a unique middle index sequence; ligating the middle index sequence to the first index sequence; and pooling the cell(s); optionally, repeating with another middle index sequence; splitting the pool of cell(s) into pools containing a unique final index sequence; and ligating the final index sequence to the preceding index sequence, whereby each cell comprises a USI.</div>
</li> <li> <para-num num="[0400]"> </para-num> <div class="description-line" id="p-0349" num="0400">The ligation handle may comprise a restriction site for producing an overhang complementary with a first index sequence overhang, and wherein the method further comprises digestion with a restriction enzyme. The ligation handle may comprise a nucleotide sequence complementary with a ligation primer sequence and wherein the overhang complementary with a first index sequence overhang is produced by hybridization of the ligation primer to the ligation handle.</div>
</li> <li> <para-num num="[0401]"> </para-num> <div class="description-line" id="p-0350" num="0401">The UCI may comprise 4 to 30 nucleotides, or 7 to 30 nucleotides, or about 21 nucleotides. The oligonucleotide label may further comprise a unique molecular identifier (UMI) sequence. The first, middle, or final index sequence may further comprise a unique molecular identifier (UMI) sequence. The UMI may be 4-20 nucleotides. The UMI may be 8 to 16 nucleotides.</div>
</li> <li> <para-num num="[0402]"> </para-num> <div class="description-line" id="p-0351" num="0402">The sequencing may comprise combining a primer having a unique source identifier (USI) sequence with UCI, so that the USI and UCI sequences are sequenced together, and the USI preferably comprises 20 to 120 nucleotides.</div>
</li> <li> <para-num num="[0403]"> </para-num> <div class="description-line" id="p-0352" num="0403">The method may comprise a multiplex assay with a plurality of labeling ligands, each labeling ligand have a distinct UCI. The labeling ligand may be non-covalently bound to the target cellular constituent. The method may further comprise pooling the oligonucleotide labels comprising a USI from a plurality of cells and sequencing the pooled UCI sequences and USI sequences. The method may further comprise pooling the oligonucleotide labels comprising a USI and UMI from a plurality of cells and sequencing the pooled UCI sequences, USI sequences, and UMI sequences. The method may further comprise quantitating the relative amount of the UCI sequence associated with a first cell to the amount of the same UCI sequence associated with a second cell, whereby the relative differences of a cellular constituent between cell(s) are determined.</div>
</li> <li> <para-num num="[0404]"> </para-num> <div class="description-line" id="p-0353" num="0404">The labeling ligand may be an antibody or an antibody fragment. The antibody fragment may be a nanobody, Fab, Fab′, (Fab′)2, Fv, ScFv, diabody, triabody, tetrabody, Bis-scFv, minibody, Fab2, or Fab3 fragment. The labeling ligand may be an aptamer. The labeling ligand may be a nucleotide sequence complementary to a target sequence.</div>
</li> <li> <para-num num="[0405]"> </para-num> <div class="description-line" id="p-0354" num="0405">The method may comprise multiplex binding of two or more labeling ligands to the cellular constituents. At least two distinct labeling ligands may comprise complementary oligonucleotide sequences, so that binding of the labeling ligands to respective target cellular constituents that are in proximity permits the complementary sequences of the distinct ligands to hybridize, forming an amplifiable polynucleotide duplex. The method may further comprise amplifying the polynucleotide duplex to provide an amplified sequence that is a detectable signal that target cellular constituents are in proximity. At least two distinct labeling ligands may comprise oligonucleotide sequences configured to be ligated, so that binding of the labeling ligands to respective target cellular constituents that are in proximity permits the oligonucleotide sequences of the distinct ligands to ligate, forming an amplifiable polynucleotide duplex. The method may further comprise amplifying the polynucleotide duplex to provide an amplified sequence that is a detectable signal that target cellular constituents are in proximity. One of the labeling ligands may comprise a restriction enzyme site between the labeling ligand and the oligonucleotide label, and wherein the method further comprises treating with a restriction enzyme, whereby the UCI from said labeling ligand is transferred to the oligonucleotide label of the labeling ligand in proximity.</div>
</li> <li> <para-num num="[0406]"> </para-num> <div class="description-line" id="p-0355" num="0406">The method may further comprise labeling the cell(s) by fluorescent in situ hybridization.</div>
</li> <li> <para-num num="[0407]"> </para-num> <div class="description-line" id="p-0356" num="0407">The cell(s) may be a member of a cell population, further comprising transforming or transducing the cell population with one or more genomic sequence-perturbation constructs that perturb a genomic sequence in the cells, wherein each distinct genomic sequence-perturbation construct comprises a unique-perturbation-identified (UPI) sequence unique to that construct. The genomic sequence-perturbation construct may comprise a sequence encoding a guide RNA sequence of a CRISPR-Cas targeting system. The method may further comprise multiplex transformation of the population of cells with a plurality of genomic sequence-perturbation constructs. The UPI sequence may be attached to a perturbation-sequence-capture sequence, and the transfer particle may comprise a perturbation-sequence-capture-binding-sequence having specific binding affinity for the perturbation-sequence-capture sequence attached to the UPI sequence. The UPI sequence may be attached to a universal ligation handle sequence, whereby a USI may be generated by split-pool ligation. The method may further comprise multiplex sequencing of the pooled UCI sequences, USI sequences, and UPI sequences.</div>
</li> <li> <para-num num="[0408]"> </para-num> <div class="description-line" id="p-0357" num="0408">In another aspect, the present invention provides a method of determining interactions between 2 or more cellular constituents, comprising: admixing at least one isolated aggregation of cellular constituents with monomers of a polymerizable gel; polymerizing the gel, to embed the cellular constituents in discrete polymer matrices; incubating the cellular constituents embedded in the polymer matrices with one or more labeling ligands with specific binding affinity for one or more target cellular constituents to produce one or more labeled cellular constituents in the polymer matrices, wherein each of the one or more labeling ligands comprise a bound oligonucleotide label comprising a unique constituent identifier (UCI) sequence and a universal hybridization nucleotide sequence, and wherein the incubation comprises binding conditions under which the labeling ligand will bind to the cellular constituent within the polymer matrix, and the incubation further comprises washing conditions under which unbound labeling ligands will be washed out of the polymer matrix; incubating the polymer matrices with at least one Unique Location Index probe, wherein the probe comprises at least two repeating nucleotide sequences, each repeat comprising a restriction enzyme site, a Unique Location Index (ULI) sequence, and a complementary universal hybridization nucleotide sequence, and wherein the incubation comprises binding conditions under which the universal hybridization sequence will hybridize the complementary universal hybridization sequence; extending the oligonucleotide label hybridized to the probe such that the oligo bound to the affinity ligand incorporates the ULI sequence that is unique to that Unique Location Index probe; digestion with a restriction enzyme specific for the site on the probe, sequencing the oligonucleotide label, whereby detecting the same ULI with two or more UCI' s indicates that the cellular constituents were interacting. The ULI sequence may be randomly generated, such that no two ULI sequences are the same. Methods of generating a barcode sequence described herein may be used to generate a ULI. The ULI will be detected with the UCI, such that when multiple cellular constituents are in proximity oligonucleotide labels comprising each UCI and the ULI from a single probe will be generated. Not being bound by a theory, using a plurality of labeling ligands with specificity for a plurality of cellular constituents will allow novel interactions to be determined. The use of polymer matrices allows a stable platform for washing out the unbound labeling ligands before staining with the ULI probes. The cellular constituent may comprise a protein, RNA transcript, or a DNA molecule. The ULI may be 4-30 nucleotides. The ULI may be 8-20 nucleotides.</div>
</li> <li> <para-num num="[0409]"> </para-num> <div class="description-line" id="p-0358" num="0409">The method may further comprise segregating the discrete polymer matrices comprising the labeled constituents before sequencing. The segregating of the discrete polymer matrices may comprise sorting single discrete matrices into separate reaction vessels.</div>
</li> <li> <para-num num="[0410]"> </para-num> <div class="description-line" id="p-0359" num="0410">The method may further comprise, before sequencing, generating a USI for each discrete polymer matrix by a split pool ligation method, wherein the restriction site on the ULI probe is a universal ligation handle (ULH) sequence configured to produce a DNA overhang capable of hybridization to a complementary over hang on a first index nucleotide sequence, wherein the first index nucleotide sequence comprises an overhang complementary to a final index sequence or optionally a middle index sequence, wherein the middle index sequence comprises overhangs complementary to the first index sequence and to the final index sequence or optionally to another middle index sequence and final index sequence, wherein the final index sequence has a single overhang complementary to the preceding index sequence, and wherein the first, middle, and final index sequences are selected from a plurality of unique sequences comprising compatible DNA overhangs and 10 to 30 base pairs of unique sequence. The split pool ligation method may comprise: splitting the pool of discrete polymer matrices into separate pools of polymer matrices, each containing a unique first index sequence; ligating the first index sequence to the ligation handle; pooling the discrete polymer matrices; optionally, splitting the pool of discrete polymer matrices into separate pools each containing a unique middle index sequence; ligating the middle index sequence to the first index sequence; and pooling the discrete polymer matrices; optionally, repeating step (d) with another middle index sequence; splitting the pool of discrete polymer matrices into pools containing a unique final index sequence; and ligating the final index sequence to the preceding index sequence, whereby each discrete polymer matrix comprises a USI.</div>
</li> <li> <para-num num="[0411]"> </para-num> <div class="description-line" id="p-0360" num="0411">The oligonucleotide label may further comprise a unique molecular identifier (UMI) sequence. The first, middle, or final index sequence may further comprise a unique molecular identifier (UMI) sequence. The method may further comprise pooling the oligonucleotide labels comprising a USI, ULI and UMI from a plurality of polymer matrices and sequencing the pooled UCI sequences, USI sequences, ULI sequences, and UMI sequences.</div>
</li> <li> <para-num num="[0412]"> </para-num> <div class="description-line" id="p-0361" num="0412">The aggregation of cellular constituents may be a cell that is a member of a cell population, further comprising transforming or transducing the cell population with one or more genomic sequence-perturbation constructs that perturb a genomic sequence in the cells, wherein each distinct genomic sequence-perturbation construct comprises a unique-perturbation-identified (UPI) sequence unique to that construct.</div>
</li> <li> <para-num num="[0413]"> </para-num> <div class="description-line" id="p-0362" num="0413">In another aspect, the present invention provides a method of determining interactions between 2 or more cellular constituents, comprising: fixing and permeablizing at least one cell; incubating the fixed and permeablized cell(s) with one or more labeling ligands with specific binding affinity for one or more target cellular constituents to produce one or more labeled cell(s), wherein each of the one or more labeling ligands comprise a bound oligonucleotide label comprising a unique constituent identifier (UCI) sequence and a universal hybridization nucleotide sequence, and wherein the incubation comprises binding conditions under which the labeling ligand will bind to the cellular constituent within the cell(s), and the incubation further comprises washing conditions under which unbound labeling ligands will be washed from the polymer cell(s); incubating the cell(s) with at least one Unique Location Index probe, wherein the probe comprises at least two repeating nucleotide sequences, each repeat comprising a restriction enzyme site, a Unique Location Index (ULI) sequence, and a complementary universal hybridization nucleotide sequence, and wherein the incubation comprises binding conditions under which the universal hybridization sequence will hybridize to the complementary universal hybridization sequence; extending the oligonucleotide label hybridized to the probe; digesting with a restriction enzyme specific for the site on the probe; and sequencing the oligonucleotide label, whereby detecting the same ULI with two or more UCI's indicates that the cellular constituents were interacting. The cellular constituent may comprise a protein, RNA transcript, or a DNA molecule. The ULI may be 4-30 nucleotides. The ULI may be 8-20 nucleotides.</div>
</li> <li> <para-num num="[0414]"> </para-num> <div class="description-line" id="p-0363" num="0414">The method may further comprise segregating the cell(s) comprising the labeled constituents before sequencing. The segregating of the cell(s) may comprise sorting single discrete matrices into separate reaction vessels. The method may further comprise, before sequencing, generating a USI for each cell by a split pool ligation method, wherein the restriction site on the ULI probe is a universal ligation handle (ULH) sequence configured to produce a DNA overhang capable of hybridization to a complementary over hang on a first index nucleotide sequence, wherein the first index nucleotide sequence comprises an overhang complementary to a final index sequence or optionally a middle index sequence, wherein the middle index sequence comprises overhangs complementary to the first index sequence and to the final index sequence or optionally to another middle index sequence and final index sequence, wherein the final index sequence has a single overhang complementary to the preceding index sequence, and wherein the first, middle, and final index sequences are selected from a plurality of unique sequences comprising compatible DNA overhangs and 10 to 30 base pairs of unique sequence. The split pool ligation method may comprise: splitting the pool of cells into separate pools of cells, each containing a unique first index sequence; ligating the first index sequence to the ligation handle; pooling the cells; optionally, splitting the pool of cells into separate pools each containing a unique middle index sequence; ligating the middle index sequence to the first index sequence; and pooling the cells; optionally, repeating with another middle index sequence; splitting the pool of cells into pools containing a unique final index sequence; and ligating the final index sequence to the preceding index sequence, whereby each cell comprises a USI.</div>
</li> <li> <para-num num="[0415]"> </para-num> <div class="description-line" id="p-0364" num="0415">The oligonucleotide label may further comprise a unique molecular identifier (UMI) sequence. The first, middle, or final index sequence may further comprise a unique molecular identifier (UMI) sequence. The method may further comprise pooling the oligonucleotide labels comprising a USI, ULI and UMI from a plurality of polymer matrices and sequencing the pooled UCI sequences, USI sequences, ULI sequences, and UMI sequences.</div>
</li> <li> <para-num num="[0416]"> </para-num> <div class="description-line" id="p-0365" num="0416">The cells may be a member of a cell population, further comprising transforming or transducing the cell population with one or more genomic sequence-perturbation constructs that perturb a genomic sequence in the cells, wherein each distinct genomic sequence-perturbation construct comprises a unique-perturbation-identified (UPI) sequence unique to that construct. The perturbation constructs may be any as described herein.</div>
</li> <li> <para-num num="[0417]"> </para-num> <div class="description-line" id="p-0366" num="0417">The oligonucleotide label may comprise a regulatory sequence configured for amplification by T7 polymerase.</div>
</li> <li> <para-num num="[0418]"> </para-num> <div class="description-line" id="p-0367" num="0418">The labeling ligands may comprise oligonucleotide sequences configured to hybridize to a transcript specific region.</div>
</li> <li> <para-num num="[0419]"> </para-num> <div class="description-line" id="p-0368" num="0419">Before sequencing, the method may further comprise: amplification of the oligonucleotide label by PCR; or T7 amplification of the oligonucleotide label followed by subsequent cDNA generation, and optionally amplification by PCR.</div>
</li> <li> <para-num num="[0420]"> </para-num> <div class="description-line" id="p-0369" num="0420">The oligonucleotide label may further comprise at least one spacer sequence. The oligonucleotide label may further comprise a photocleavable linker. The oligonucleotide label may further comprise a restriction enzyme site between the labeling ligand and UCI.</div>
</li> <li> <para-num num="[0421]"> </para-num> <div class="description-line" id="p-0370" num="0421">The oligonucleotide label may comprise one or more iso-dG and/or iso-dC nucleotides. The oligonucleotide labels for hybridization in a proximity assay may comprise one or more iso-dG and/or iso-dC nucleotides. The universal hybridization sequences may comprise one or more iso-dG and/or iso-dC nucleotides. Not being bound by a theory the one or more iso-dG and/or iso-dC nucleotides will increase specificity of hybridization.</div>
</li> <li> <para-num num="[0422]"> </para-num> <div class="description-line" id="p-0371" num="0422">In one embodiment, the oligonucleotide label of any of the methods described herein may comprise one or more iso-dG and/or iso-dC nucleotides. Two complementary sequences may comprise one sequence with iso-dG and the other complementary sequence with iso-dC, whereby the two sequences are capable of hybridizing with each other, but not with sequences containing only dG, dC, dA, and/or dT. The sequence of the oligonucleotide labels for hybridization in a proximity assay may advantageously comprise one or more iso-dG and/or iso-dC nucleotides.</div>
</li> <li> <para-num num="[0423]"> </para-num> <div class="description-line" id="p-0372" num="0423">Any of the methods of the present invention may advantageously be combined for determining any combination of protein detection, RNA detection, open chromatin detection, protein-protein interactions, protein-RNA interactions, or protein-DNA interactions.</div>
</li> <li> <para-num num="[0424]"> </para-num> <div class="description-line" id="p-0373" num="0424">The terms “isolated aggregation of cellular constituents” or “single aggregations of cellular constituents” or “aggregations of cellular constituents” or “aggregations of biologically connected cellular constituents” are used interchangeably and refer to any group of cellular constituents that originate from the same source, that are functionally connected biologically, and that can be isolated individually. Examples may be a cell, an extracellular vesicle, an organelle, or an organized subcomponent thereof. Specific examples may be a nucleus or a mitochondria.</div>
</li> <li> <para-num num="[0425]"> </para-num> <div class="description-line" id="p-0374" num="0425">The term “cellular constituent” refers to any cellular molecule, including but not limited to a protein, nucleic acid, RNA molecule, DNA molecule, or carbohydrate.</div>
</li> <li> <para-num num="[0426]"> </para-num> <div class="description-line" id="p-0375" num="0426">The term “unique molecular identifiers” (UMI) refers to a sequencing linker used in a method that uses molecular tags to detect and quantify unique amplified products. A UMI is used to distinguish effects through a single clone from multiple clones. In preferred embodiments, the amplification is by PCR. A sequencer linker with a random sequence of between 4 and 20 basepairs and an index sequence is added to the 5′ end of the template, which is amplified and sequenced. Sequencing allows for high resolution reads, enabling accurate detection of true variants. As used herein, a “true variant” will be present in every amplified product originating from the original clone as identified by aligning all products with a UMI. Each clone amplified will have a different random UMI that will indicate that the amplified product originated from that clone. Background caused by the fidelity of the amplification process can be eliminated because true variants will be present in all amplified products and background representing random error will only be present in single amplification products (See e.g., Islam S. et al., 2014. Nature Methods No:11, 163-166). Not being bound by a theory, the UMI and UCI's are designed such that assignment to the original can take place despite up to 4-7 errors during amplification or sequencing.</div>
</li> <li> <para-num num="[0427]"> </para-num> <div class="description-line" id="p-0376" num="0427">The term “unique constituent identifier” (UCI) refers to any unique nucleotide sequence linked to a labeling ligand, such that the presence of the sequence indicates the presence of the cellular constituent that the labeling ligand specifically binds. In an exemplary embodiment, the UCI is linked to an antibody for a specific cellular constituent. If the cellular constituent is present in a sample, the antibody will bind and the UCI can be detected. If the cellular constituent is not present in a sample, the antibody will not bind and the UCI will not be detected above background. In another exemplary embodiment, the labeling ligand is an oligonucleotide probe and the cellular constituent is an RNA transcript molecule complementary to the sequence of the oligonucleotide probe. The sequence of the oligonucleotide probe may be the UCI or may comprise an additional UCI sequence to identify the RNA transcript.</div>
</li> <li> <para-num num="[0428]"> </para-num> <div class="description-line" id="p-0377" num="0428">The term “unique source identifier” (USI) refers to a unique nucleotide sequence that is associated with the nucleic acids from a single cell or single isolated aggregation of cellular constituents (source), such that upon sequencing a pool of nucleic acid sequences from more than one cell or isolated aggregation of cellular constituents, the presence of a USI in the sequenced product indicates that a product originated from a single source. USI may be used interchangeably with the term “barcode.”</div>
</li> <li> <para-num num="[0429]"> </para-num> <div class="description-line" id="p-0378" num="0429">The term “unique-amplification-identifier” (UAI) refers to a nucleotide sequence that is only formed only when two or more nucleotide sequences are in close proximity to each other such that they can be ligated. The UAI can be generated using methods described for the proximity ligation assay (PLA) or proximity extension assay (PEA) (Fredriksson S, et al. (2002) Protein detection using proximity-dependent DNA ligation assays. Nature biotechnology 20: 473-477; Gullberg M, et al. (2004) Cytokine detection by antibody-based proximity ligation. Proceedings of the National Academy of Sciences of the United States of America 101: 8420-8424; and Lundberg M, et al. (2011) Homogeneous antibody-based proximity extension assays provide sensitive and specific detection of low-abundant proteins in human blood. Nucleic acids research 39(15): e102). PEA is based on pairs of antibodies that are linked to oligonucleotides having slight affinity to one another (PEA probes). Upon target binding the probes are brought in proximity, and the two oligonucleotides are extended by a DNA polymerase forming the UAI that now acts as a unique surrogate marker for the specific antigen.</div>
</li> <li> <para-num num="[0430]"> </para-num> <div class="description-line" id="p-0379" num="0430">The terms “sticky end,” “overhang” and “DNA overhang” refer to a double stranded DNA having either a 3′ or 5′ single stranded DNA overhang capable of hybridization to another complementary sticky end or DNA overhang.</div>
</li> <li> <para-num num="[0431]"> </para-num> <div class="description-line" id="p-0380" num="0431">The term “hydrogel” refers to any network of polymer chains that are hydrophilic, and sometimes found as a colloidal gel, in which water is the dispersion medium. Hydrogels are highly absorbent (they can contain over 90% water) natural or synthetic polymeric networks. Hydrogels also possess a degree of flexibility very similar to natural tissue, due to their significant water content. Hydrogel may include polyvinyl alcohol, sodium polyacrylate, acrylate polymers, copolymers with an abundance of hydrophilic groups, agarose, methylcellulose, hyaluronan, and other naturally derived polymers.</div>
</li> <li> <para-num num="[0432]"> </para-num> <div class="description-line" id="p-0381" num="0432">The term “tagmentation” refers to a step in the Assay for Transposase Accessible Chromatin using sequencing (ATAC-seq) as described. (See, Buenrostro, J. D., Giresi, P. G., Zaba, L. C., Chang, H. Y., Greenleaf, W. J., Transposition of native chromatin for fast and sensitive epigenomic profiling of open chromatin, DNA-binding proteins and nucleosome position. Nature methods 2013; 10 (12): 1213-1218). Specifically, a hyperactive Tn5 transposase loaded in vitro with adapters for high-throughput DNA sequencing, can simultaneously fragment and tag a genome with sequencing adapters. In one embodiment the adapters are compatible with the methods described herein.</div>
</li> <li> <para-num num="[0433]"> </para-num> <div class="description-line" id="p-0382" num="0433">The present invention may also include barcoding. Barcoding may be performed based on any of the compositions or methods disclosed in patent publication WO 2014047561 A1, Compositions and methods for labeling of agents, incorporated herein in its entirety. In one embodiment each labeling ligand has a barcode (UCI). In one embodiment, a sgRNA has a barcode. In one embodiment the UCI is captured on a bead that includes a barcode sequence (USI). Not being bound by a theory, amplified sequences from single cells or isolated aggregations of cellular constituents can be sequenced together and resolved based on the barcode associated with each USI. Not being bound by a theory, the presence of a labeling ligand can be determined by sequencing of the UCI.</div>
</li> <li> <para-num num="[0434]"> </para-num> <div class="description-line" id="p-0383" num="0434">Barcoding may be performed based on any of the compositions or methods disclosed in patent publication WO 2014047561 A1, Compositions and methods for labeling of agents, incorporated herein in its entirety. In certain embodiments barcoding uses an error correcting scheme (T. K. Moon, Error Correction Coding: Mathematical Methods and Algorithms (Wiley, New York, ed. 1, 2005)). Not being bound by a theory, amplified sequences from single cells can be sequenced together and resolved based on the barcode associated with each cell.</div>
</li> <li> <para-num num="[0435]"> </para-num> <div class="description-line" id="p-0384" num="0435">The term “barcode” as used herein, refers to any unique, non-naturally occurring, nucleic acid sequence that may be used to identify the originating source of a nucleic acid fragment. Such barcodes may be sequences including but not limited to about 20 base pair sequences. Although it is not necessary to understand the mechanism of an invention, it is believed that the barcode sequence provides a high-quality individual read of a barcode associated with a viral vector, labeling ligand, shRNA, sgRNA or cDNA such that multiple species can be sequenced together.</div>
</li> <li> <para-num num="[0436]"> </para-num> <div class="description-line" id="p-0385" num="0436">DNA barcoding is also a taxonomic method that uses a short genetic marker in an organism's DNA to identify it as belonging to a particular species. It differs from molecular phylogeny in that the main goal is not to determine classification but to identify an unknown sample in terms of a known classification. Kress et al., “Use of DNA barcodes to identify flowering plants” Proc. Natl. Acad. Sci. U.S.A. 102(23):8369-8374 (2005). Barcodes are sometimes used in an effort to identify unknown species or assess whether species should be combined or separated. Koch H., “Combining morphology and DNA barcoding resolves the taxonomy of Western Malagasy Liotrigona Moure, 1961” African Invertebrates 51(2): 413-421 (2010); and Seberg et al., “How many loci does it take to DNA barcode a crocus?” PLoS One 4(2):e4598 (2009). Barcoding has been used, for example, for identifying plant leaves even when flowers or fruit are not available, identifying the diet of an animal based on stomach contents or feces, and/or identifying products in commerce (for example, herbal supplements or wood). Soininen et al., “Analysing diet of small herbivores: the efficiency of DNA barcoding coupled with high-throughput pyrosequencing for deciphering the composition of complex plant mixtures” Frontiers in Zoology 6:16 (2009).</div>
</li> <li> <para-num num="[0437]"> </para-num> <div class="description-line" id="p-0386" num="0437">It has been suggested that a desirable locus for DNA barcoding should be standardized so that large databases of sequences for that locus can be developed. Most of the taxa of interest have loci that are sequencable without species-specific PCR primers. CBOL Plant Working Group, “A DNA barcode for land plants” PNAS 106(31):12794-12797 (2009). Further, these putative barcode loci are believed short enough to be easily sequenced with current technology. Kress et al., “DNA barcodes: Genes, genomics, and bioinformatics” PNAS 105(8):2761-2762 (2008). Consequently, these loci would provide a large variation between species in combination with a relatively small amount of variation within a species. Lahaye et al., “DNA barcoding the floras of biodiversity hotspots” Proc Natl Acad Sci USA 105(8):2923-2928 (2008).</div>
</li> <li> <para-num num="[0438]"> </para-num> <div class="description-line" id="p-0387" num="0438">DNA barcoding is based on a relatively simple concept. For example, most eukaryote cells contain mitochondria, and mitochondrial DNA (mtDNA) has a relatively fast mutation rate, which results in significant variation in mtDNA sequences between species and, in principle, a comparatively small variance within species. A 648-bp region of the mitochondrial cytochrome c oxidase subunit 1 (CO1) gene was proposed as a potential ‘barcode’. As of 2009, databases of CO1 sequences included at least 620,000 specimens from over 58,000 species of animals, larger than databases available for any other gene. Ausubel, J., “A botanical macroscope” Proceedings of the National Academy of Sciences 106(31):12569 (2009).</div>
</li> <li> <para-num num="[0439]"> </para-num> <div class="description-line" id="p-0388" num="0439">Software for DNA barcoding requires integration of a field information management system (FIMS), laboratory information management system (LIMS), sequence analysis tools, workflow tracking to connect field data and laboratory data, database submission tools and pipeline automation for scaling up to eco-system scale projects. Geneious Pro can be used for the sequence analysis components, and the two plugins made freely available through the Moorea Biocode Project, the Biocode LIMS and Genbank Submission plugins handle integration with the FIMS, the LIMS, workflow tracking and database submission.</div>
</li> <li> <para-num num="[0440]"> </para-num> <div class="description-line" id="p-0389" num="0440">Additionally, other barcoding designs and tools have been described (see e.g., Birrell et al., (2001) Proc. Natl Acad. Sci. USA 98, 12608-12613; Giaever, et al., (2002) Nature 418, 387-391; Winzeler et al., (1999) Science 285, 901-906; and Xu et al., (2009) Proc Natl Acad Sci USA. February 17; 106(7):2289-94).</div>
</li> <li> <para-num num="[0441]"> </para-num> <div class="description-line" id="p-0390" num="0441">In certain embodiments, sequencing is performed using unique molecular identifiers (UMI). The term “unique molecular identifiers” (UMI) refers to a sequencing linker used in a method that uses molecular tags to detect and quantify unique amplified products. A UMI is used to distinguish effects through a single clone from multiple clones. In preferred embodiments, the amplification is by PCR. A sequencer linker with a random sequence of between 4 and 20 base pairs is added to the 5′ end of the template, which is amplified and sequenced. Sequencing allows for high resolution reads, enabling accurate detection of true variants. As used herein, a “true variant” will be present in every amplified product originating from the original clone as identified by aligning all products with a UMI. Each clone amplified will have a different random UMI that will indicate that the amplified product originated from that clone. Background caused by the fidelity of the amplification process can be eliminated because true variants will be present in all amplified products and background representing random error will only be present in single amplification products (See e.g., Islam S. et al., 2014. Nature Methods No:11, 163-166). Not being bound by a theory, the UMI's are designed such that assignment to the original can take place despite up to 4-7 errors during amplification or sequencing.</div>
</li> <li> <para-num num="[0442]"> </para-num> <div class="description-line" id="p-0391" num="0442">Unique molecular identifiers are a subtype of nucleic acid barcode that can be used, for example, to normalize samples for variable amplification efficiency. For example, in various embodiments, featuring a solid or semisolid support (for example a hydrogel bead), to which nucleic acid barcodes (for example a plurality of barcode sharing the same sequence) are attached, each of the barcodes may be further coupled to a unique molecular identifier, such that every barcode on the particular solid or semisolid support receives a distinct unique molecule identifier. A unique molecular identifier can then be, for example, transferred to a target molecule with the associated barcode, such that the target molecule receives not only a nucleic acid barcode, but also an identifier unique among the identifiers originating from that solid or semisolid support.</div>
</li> <li> <para-num num="[0443]"> </para-num> <div class="description-line" id="p-0392" num="0443">In certain embodiments, multiple displacement amplification (MDA) is used. Multiple displacement amplification (MDA, is a non-PCR-based isothermal method based on the annealing of random hexamers to denatured DNA, followed by strand-displacement synthesis at constant temperature (Blanco et al. J. Biol. Chem. 1989, 264, 8935-8940). It has been applied to samples with small quantities of genomic DNA, leading to the synthesis of high molecular weight DNA with limited sequence representation bias (Lizardi et al. Nature Genetics 1998, 19, 225-232; Dean et al., Proc. Natl. Acad. Sci. U.S.A. 2002, 99, 5261-5266). As DNA is synthesized by strand displacement, a gradually increasing number of priming events occur, forming a network of hyper- branched DNA structures. The reaction can be catalyzed by enzymes such as the Phi29 DNA polymerase or the large fragment of the Bst DNA polymerase. The Phi29 DNA polymerase possesses a proofreading activity resulting in error rates 100 times lower than Taq polymerase (Lasken et al. Trends Biotech. 2003, 21, 531-535).</div>
</li> <li> <para-num num="[0444]"> </para-num> <div class="description-line" id="p-0393" num="0444">The invention provides a method for preparing uniquely barcoded particles. Unique barcode sequences may be generated by a split pool method. The split pool method may include sticky end ligation. Sticky end ligation may include a sticky end ligation handle and separate indexes containing unique sequences capable of hybridizing to a sticky end (see examples). The sticky end may comprise a ssDNA overhang. The over-hang may be 2, 3, 4, 5, 6, 7, 8, preferably 4 bases. The overhang may be generated by a restriction enzyme. Each index may contain a plurality of unique sequences. Each index may contain 10, 20, 30, 40, 50, 60, 70, 80, 90, 100, 200, preferably 192 sequences. In one embodiment there are 2, 3, 4, preferably 3 indexes. A unique barcode sequence is generated by ligation of the first index to the ligation handle, splitting and pooling of the ligated samples, and then addition of the next index also containing sticky ends. The last index preferably has a sticky end for ligation to the previous index. The last index may advantageously include a primer sequence for priming of PCR. Methods of split pooling have been described. In one embodiment the ligation handle is digested with a restriction enzyme to produce a four base overhang. In another embodiment, a ligation primer is hybridized to the ligation handle to generate an at least 4 base overhang that is complementary to an index in the split pool method.</div>
</li> <li> <para-num num="[0445]"> </para-num> <div class="description-line" id="p-0394" num="0445">In one exemplary embodiment, the hydrogel particles or polymer matrices are split into pools, each pool containing a unique index A and each ligation handle is ligated to a sequence in index A. All particles are then pooled and re-split into new pools containing a unique index B. After ligation, all of the particles are pooled again and re-split into new pools containing a unique index C. If each index has 100 unique sequences and for each cycle the particles are split into 100 pools each containing a unique sequence, then after 3 cycles of split and pool ligation, the barcode on any given particle possess the same one of 100<sup>3</sup>=1,000,000 possible barcodes, but different particles have different sequences.</div>
</li> <li> <para-num num="[0446]"> </para-num> <div class="description-line" id="p-0395" num="0446">In another embodiment, single cell or single isolated aggregation of cellular constituent analysis is performed by digital polymerase chain reactions (PCR), e.g., Fluidigm C. Digital polymerase chain reaction (digital PCR, DigitalPCR, dPCR, or dePCR) is a refinement of conventional polymerase chain reaction methods that can be used to directly quantify and clonally amplify nucleic acids including DNA, cDNA or RNA. The key difference between dPCR and traditional PCR lies in that PCR carries out one reaction per single sample and dPCR carries out a single reaction within samples separated into a large number of partitions wherein the reactions are carried out in each partition individually. A sample is partitioned so that individual nucleic acid molecules within the sample are localized and concentrated within many separate regions. The capture or isolation of individual nucleic acid molecules may be effected in micro well plates, capillaries, the dispersed phase of an emulsion, and arrays of miniaturized chambers, as well as on nucleic acid binding surfaces.</div>
</li> <li> <para-num num="[0447]"> </para-num> <div class="description-line" id="p-0396" num="0447">In a preferred embodiment, single cell or single aggregation of cellular constituent analysis is performed using microfluidics. Microfluidics involves micro-scale devices that handle small volumes of fluids. Because microfluidics may accurately and reproducibly control and dispense small fluid volumes, in particular volumes less than 1 μl, application of microfluidics provides significant cost-savings. The use of microfluidics technology reduces cycle times, shortens time-to-results, and increases throughput. Furthermore, incorporation of microfluidics technology enhances system integration and automation. Microfluidic reactions are generally conducted in microdroplets. The ability to conduct reactions in microdroplets depends on being able to merge different sample fluids and different microdroplets. See, e.g., US Patent Publication No. 20120219947 and PCT publication No. WO2014085802 A1.</div>
</li> <li> <para-num num="[0448]"> </para-num> <div class="description-line" id="p-0397" num="0448">Droplet microfluidics offers significant advantages for performing high-throughput screens and sensitive assays. Droplets allow sample volumes to be significantly reduced, leading to concomitant reductions in cost. Manipulation and measurement at kilohertz speeds enable up to 10<sup>8 </sup>samples to be screened in a single day. Compartmentalization in droplets increases assay sensitivity by increasing the effective concentration of rare species and decreasing the time required to reach detection thresholds. Droplet microfluidics combines these powerful features to enable currently inaccessible high-throughput screening applications, including single-cell and single-molecule assays. See, e.g., Guo et al., Lab Chip, 2012,12, 2146-2155.</div>
</li> <li> <para-num num="[0449]"> </para-num> <div class="description-line" id="p-0398" num="0449">The manipulation of fluids to form fluid streams of desired configuration, discontinuous fluid streams, droplets, particles, dispersions, etc., for purposes of fluid delivery, product manufacture, analysis, and the like, is a relatively well-studied art. Microfluidic systems have been described in a variety of contexts, typically in the context of miniaturized laboratory (e.g., clinical) analysis. Other uses have been described as well. For example, WO 2001/89788; WO 2006/040551; U.S. Patent Application Publication No. 2009/0005254; WO 2006/040554; U.S. Patent Application Publication No. 2007/0184489; WO 2004/002627; U.S. Pat. No. 7,708,949; WO 2008/063227; U.S. Patent Application Publication No. 2008/0003142; WO 2004/091763; U.S. Patent Application Publication No. 2006/0163385; WO 2005/021151; U.S. Patent Application Publication No. 2007/0003442; WO 2006/096571; U.S. Patent Application Publication No. 2009/0131543; WO 2007/089541; U.S. Patent Application Publication No. 2007/0195127; WO 2007/081385; U.S. Patent Application Publication No. 2010/0137163; WO 2007/133710; U.S. Patent Application Publication No. 2008/0014589; U.S. Patent Application Publication No. 2014/0256595; and WO 2011/079176. In a preferred embodiment single cell analysis is performed in droplets using methods according to WO 2014085802. Each of these patents and publications is herein incorporated by reference in their entireties for all purposes.</div>
</li> <li> <para-num num="[0450]"> </para-num> <div class="description-line" id="p-0399" num="0450">Single cells or isolated aggregations of cellular constituents may be sorted into separate vessels by dilution of the sample and physical movement, such as pipetting. A machine can control the pipetting and separation. The machine may be a computer controlled robot.</div>
</li> <li> <para-num num="[0451]"> </para-num> <div class="description-line" id="p-0400" num="0451">Microfluidics may also be used to separate the single cells and/or isolated aggregations of cellular constituents. Single cells and/or isolated aggregations of cellular constituents can be separated using microfluidic devices. Microfluidics involves micro-scale devices that handle small volumes of fluids. Because microfluidics may accurately and reproducibly control and dispense small fluid volumes, in particular volumes less than 1 μl, application of microfluidics provides significant cost-savings. The use of microfluidics technology reduces cycle times, shortens time-to-results, and increases throughput. The small volume of microfluidics technology improves amplification and construction of DNA libraries made from single cells and single isolated aggregations of cellular constituents. Furthermore, incorporation of microfluidics technology enhances system integration and automation.</div>
</li> <li> <para-num num="[0452]"> </para-num> <div class="description-line" id="p-0401" num="0452">Single cells and/or single isolated aggregations of cellular constituents of the present invention may be divided into single droplets using a microfluidic device. The single cells and/or single isolated aggregations of cellular constituents in such droplets may be further labeled with a barcode. In this regard reference is made to Macosko et al., 2015, “Highly Parallel Genome-wide Expression Profiling of Individual Cells Using Nanoliter Droplets” Cell 161, 1202-1214 and Klein et al., 2015, “Droplet Barcoding for Single-Cell Transcriptomics Applied to Embryonic Stem Cells” Cell 161, 1187-120,1 all the contents and disclosure of each of which are herein incorporated by reference in their entirety. Not being bound by a theory, the volume size of an aliquot within a droplet may be as small as 1 fL.</div>
</li> <li> <para-num num="[0453]"> </para-num> <div class="description-line" id="p-0402" num="0453">Single cells and/or single aggregations of cellular constituents may be diluted into a physical multi-well plate or a plate free environment. The multi-well assay modules (e.g., plates) may have any number of wells and/or chambers of any size or shape, arranged in any pattern or configuration, and be composed of a variety of different materials. Preferred embodiments of the invention are multi-well assay plates that use industry standard multi-well plate formats for the number, size, shape and configuration of the plate and wells. Examples of standard formats include 96-, 384-, 1536- and 9600-well plates, with the wells configured in two-dimensional arrays. Other formats include single well, two well, six well and twenty-four well and 6144 well plates. Plate free environments of the present invention utilize a single polymerizable gel containing compartmentalized cells and/or isolated aggregations of cellular constituents. In one embodiment, extraction of single cells and/or single isolated aggregations of cellular constituents may be by a mechanical punch. Single cells and/or single isolated aggregations of cellular constituents may be visualized in the gel before a punch.</div>
</li> <li> <para-num num="[0454]"> </para-num> <div class="description-line" id="p-0403" num="0454">In one embodiment, a DNA tag including a protein specific barcode (UCI) is conjugated to detection biomolecules or labeling ligands with high target affinity and low unspecific binding, such as antibodies (Janssen et al., 2013) or nanobodies (Pardon et al., 2014; Theile et al., 2013) or aptamers (Janssen et al., 2013).</div>
</li> <li> <para-num num="[0455]"> </para-num> <div class="description-line" id="p-0404" num="0455">In one embodiment, to ensure proper staining of intracellular and cell surface proteins with, for instance, DNA-tagged antibodies, single cells are embedded in hydrogel droplets. Not being bound by a theory, the hydrogel mesh provides a physical framework, chemically incorporates biomolecules and is permeable to macromolecules such as antibodies (Chung et al., 2013). In one embodiment, to further improve permeability and staining efficiency, lipids are cleared (Chung et al., 2013). Not being bound by a theory, the clearance of the lipids and the porosity of the hydrogel allow for more efficient washing and removal of unspecific antibodies. This higher accuracy of measurement is important for the high multiplex measurements and computational inference of regulatory mechanisms.</div>
</li> <li> <para-num num="[0456]"> </para-num> <div class="description-line" id="p-0405" num="0456">In one embodiment, cells embedded in a hydrogel mesh can be stained with the DNA-tagged antibodies and washed in bulk before isolating the single cells. Once isolated, a cell specific oligonucleotide barcode (USI) can be introduced before subsequent DNA amplification and library preparation steps. Isolating single cells into individual reaction chambers to perform PCR amplification or a proximity ligation/extension assay (Assarsson et al., 2014) can be achieved at modest throughput either by FACS sorting into multi-well plates or microfluidic capture using the Fluidigm C1 (Shalek et al., 2014).</div>
</li> <li> <para-num num="[0457]"> </para-num> <div class="description-line" id="p-0406" num="0457">In one embodiment, for more high throughput processing, a microfluidic chip can be used to capture the hydrogel embedded cells or cellular constituents in nanoliter-sized aqueous droplets (Macosko et al., 2015, “Highly Parallel Genome-wide Expression Profiling of Individual Cells Using Nanoliter Droplets” Cell 161, 1202-1214). In one embodiment, the hydrogel embedded cells or cellular constituents are poisson loaded into microwells (Fan et al., 2015). The aqueous droplets or microwells may be simultaneously loaded with barcoded beads, each of which has oligonucleotides including; a “cell barcode” that is the same across all the primers on the surface of any one bead, but different from the cell barcodes on all other beads; a Unique Molecular Identifier (UMI), different on each primer, that enables sequence reads derived from the same original DNA tag (amplification and PCR duplicates) to be identified computationally (Kivioja et al., 2012); and a capture sequence to bind the oligos (either amplified PCR products or original DNA tags released by proteinase K treatment, or enzymatic/photonic oligo cleavage). Once the beads are loaded, they can be pooled for amplification and library preparation, and sequencing. These beads can take multiple forms, the preferred drop-seq beads are polystyrene, oligo functionalized beads, but alternative beads are possible, such as soft beads (polymer gel based beads), that allow for one on one pairing with cells, as to avoid the poisson loading needed in the described drop-seq scheme. This reduces the amount of cells one needs, and makes it possible to analyze rare cell types or clinical samples only available in low amounts of cells.</div>
</li> <li> <para-num num="[0458]"> </para-num> <div class="description-line" id="p-0407" num="0458">In one embodiment, the present invention provides for the simultaneous detection of proteins and nucleic acids. Nucleic acids can be reverse cross-linked after separation of discrete polymer matrices into separate wells or droplets. The contents of individual wells or droplets may then be sequenced. In one embodiment, crosslinking is reversed by incubating the cross-linked sample in high salt (approximately 200 mM NaCl) at 65° C. for at least 4h.</div>
</li> <li> <para-num num="[0459]"> </para-num> <div class="description-line" id="p-0408" num="0459">In one embodiment, Drop-Seq (Macosko et al., 2015) is used to analyze RNA or DNA in single cells in parallel to the detection of cellular constituents. Drop-Seq is a reverse emulsion, early barcoding method for analyzing 10<sup>4</sup>- 10<sup>6 </sup>cells/experiment at very low cost ($0.06/cell). The Drop-seq method may be used to encapsulate discrete hydrogel matrices in a droplet. The RNA and/or DNA can be reverse cross-linked and the oligonucleotide labels can be removed from the labelling ligand. Capture of RNA, DNA, and oligonucleotide labels on barcoded beads, library preparation, and sequencing is performed as described previously.</div>
</li> <li> <para-num num="[0460]"> </para-num> <div class="description-line" id="p-0409" num="0460">In one embodiment, the detection of proteins or post translational modifications (PTM) is determined by sequencing based readouts. In some embodiments, Immuno-Seq is used when antibodies can be washed out (Niemeyer, C. M., et al., Nat Protoc. 2, 1918-1930 (2007)) and proximity extension assays (PEA) is used when antibodies cannot be washed away (Hammond, M., et al. PLoS One. 7, e40405, (2012); and Stahlberg, A., et al. Clin Chem. 58, 1682-1691 (2012)). These methods use DNA-sequence based encoding, and are compatible with other genomic readouts (e.g., sgRNA barcodes).</div>
</li> <li> <para-num num="[0461]"> </para-num> <div class="description-line" id="p-0410" num="0461">In another embodiment, the detection of proteins embedded in a hydrogel matrix is determined by FACS. Not being bound by a theory, the encapsulation of cellular constituents in a hydrogel matrix and removing lipids provides for improved binding of antibodies to intracellular targets as compared to regular fixation and permeabilization protocols for FACS alone.</div>
</li> <li> <para-num num="[0462]"> </para-num> <div class="description-line" id="p-0411" num="0462">In one embodiment, PEA methods are used for profiling protein-protein or protein-nucleic acid interactions by, respectively, using antibodies against two protein targets (Leuchowius, K. J., et al. Cytometry A. 75, 833-839 (2009)). or replacing one antibody with an oligonucleotide complementary to a sequence of interest (Gustafsdottir, S. M., et al. Proceedings of the National Academy of Sciences of the United States of America. 104, 3067-3072, (2007)).</div>
</li> <li> <para-num num="[0463]"> </para-num> <div class="description-line" id="p-0412" num="0463">In another aspect, the present invention provides screening methods to determine the effect on protein, post translational modifications and cellular constituents of single cells or isolated aggregations of cellular constituents in response to the perturbation of genes or cellular circuits. Perturbation may be knocking down a gene, increasing expression of a gene, mutating a gene, mutating a regulatory sequence, or deleting non-protein-coding DNA.</div>
</li> <li> <para-num num="[0464]"> </para-num> <div class="description-line" id="p-0413" num="0464">In one embodiment, CRISPR/Cas9 may be used to perturb protein-coding genes or non-protein-coding DNA. CRISPR/Cas9 may be used to knockout protein-coding genes by frameshifts, point mutations, inserts, or deletions. An extensive toolbox may be used for efficient and specific CRISPR/Cas9 mediated knockout as described herein, including a double-nicking CRISPR to efficiently modify both alleles of a target gene or multiple target loci and a smaller Cas9 protein for delivery on smaller vectors (Ran, F. A., et al., In vivo genome editing using <i>Staphylococcus aureus </i>Cas9. Nature. 520, 186-191 (2015)). A genome-wide sgRNA mouse library (10 sgRNAs/gene) may also be used in a mouse that expresses a Cas9 protein. The cells of the mouse can then be analyzed using the methods of the present invention.</div>
</li> <li> <para-num num="[0465]"> </para-num> <div class="description-line" id="p-0414" num="0465">In one embodiment, a CRISPR system may be used to activate gene transcription. A nuclease-dead RNA-guided DNA binding domain, dCas9, tethered to transcriptional repressor domains that promote epigenetic silencing (e.g., KRAB) may be used for “CRISPRi” that represses transcription. To use dCas9 as an activator (CRISPRa), a guide RNA is engineered to carry RNA binding motifs (e.g., MS2) that recruit effector domains fused to RNA-motif binding proteins, increasing transcription. A key dendritic cell molecule, p65, may be used as a signal amplifier, but is not required.</div>
</li> <li> <para-num num="[0466]"> </para-num> <div class="description-line" id="p-0415" num="0466">In one embodiment, perturbation is by deletion of regulatory elements. Non-coding elements may be targeted by using pairs of guide RNAs to delete regions of a defined size, and by tiling deletions covering sets of regions in pools.</div>
</li> <li> <para-num num="[0467]"> </para-num> <div class="description-line" id="p-0416" num="0467">In one embodiment, perturbation of genes is by RNAi. The RNAi may be shRNA's targeting genes. The shRNA's may be delivered by any methods known in the art. In one embodiment the shRNA's may be delivered by a viral vector. The viral vector may be a lentivirus.</div>
</li> <li> <para-num num="[0468]"> </para-num> <div class="description-line" id="p-0417" num="0468">In one embodiment, a CRISPR based pooled screen is used. Perturbation may rely on sgRNA expression cassettes that are stably integrated into the genome. The expressed sgRNA may serve as a molecular barcode, reporting the loss of function of the target in a cell. Alternatively, optimized separate barcodes may be co-expressed with the sgRNA, should sgRNAs not be ideal as barcodes. Transduction of cells at a higher multiplicity of infection (MOI) or delivering vectors by transfection at a higher MOI would result in any given cell receiving multiple sgRNA's and allow combinatorial perturbations. In one embodiment, 2, or 3, or 4, or 5, or up to 10 genes, preferably 5-7 genes are perturbed in a single cell.</div>
</li> <li> <para-num num="[0469]"> </para-num> <div class="description-line" id="p-0418" num="0469">In one embodiment, recombinant Cas9 protein and sgRNA is delivered simultaneously to cells with nanowires or the recently developed ‘CellSqueeze’ (Sharei, A., et al. Proceedings of the National Academy of Sciences of the United States of America. 110, 2082-2087, (2013)). Applicants have shown that nanowires can deliver functional proteins, RNA and small molecules alone and in combinations into the cell's cytoplasm, and do not cause toxicity or inappropriate activation and allow the cells to respond normally to signals (Shalek, A. K., et al. Nano Lett. 12, 6498-6504, (2012); Yosef, N., et al. Nature. 496, 461-468, (2013); and Shalek, A. K., et al. Proceedings of the National Academy of Sciences of the United States of America. 107, 1870-1875, (2010)).</div>
</li> <li> <para-num num="[0470]"> </para-num> <div class="description-line" id="p-0419" num="0470">In one embodiment, hybrid measurements or alternative readouts are measured. The alternative readouts may either be stand alone, or hybrid measurements. One alternative readout may be epigenetic measurements. Not being bound by a theory, when biomolecules with functional groups are formaldehyde fixed and bound to the polymer mesh, and membrane and nuclear lipids are cleared, chromosomal DNA is preserved and is accessible for further interrogation. Epigenetic assays that have been applied to single cells may be combined with a perturbation and protein level readout. Not being bound by a theory, the new layers of information aid in understanding of the regulatory mechanisms underpinning cellular behavior. Histone modifications have been measured at specific gene loci at the single cell level (Gomez et al., 2013). This publication uses ISH-PLA (in situ hybridization (ISH), proximity ligation assay (PLA)). They use a biotin modified ISH probe, by binding with streptavidin and an oligo bound anti-streptavadin antibody. As antibodies against multiple histone modifications are readily available, the PLA scheme is applicable to the present invention. Not being bound by a theory, a histone code based on the combination of a plurality of histone modifications determines gene expression at a given locus. Many histone modifications at many genetic loci can be determined simultaneously by replacing the biotin-streptavadin construct by an ISH probe conjugated to a linker (peptide, DNA or nanoparticles, . . . ), followed by another DNA barcode reporting on the genetic locus, and including a binding sequence to the oligo conjugated to the histone modification antibody.</div>
</li> <li> <para-num num="[0471]"> </para-num> <div class="description-line" id="p-0420" num="0471">In one embodiment, chromatin accessibility is determined using a single cell ATAC-seq assay. ATAC-seq offers genome-wide chromatin accessibility of regulatory elements, transcription factor binding and nucleosome positioning.</div>
</li> <li> <para-num num="[0472]"> </para-num> <div class="description-line" id="p-0421" num="0472">In one embodiment, DNA methylation analysis is determined. Cytosine methylation analysis has been analyzed at the single cell level (Kantlehner et al., 2011), as has adenine methylation (Lorthongpanich et al., 2013).</div>
</li> <li> <para-num num="[0473]"> </para-num> <div class="description-line" id="p-0422" num="0473">In one embodiment, the spatial organization of chromosomes is determined. The spatial organization of chromosomes has been found to have fundamental effects on gene expression and cellular function. Single cell measurements (Hi-C) have revealed extensive cell-to-cell heterogeneity in chromosome structure (Nagano et al., 2013). This method can be incorporated into the present invention.</div>
</li> <li> <para-num num="[0474]"> </para-num> <div class="description-line" id="p-0423" num="0474">In one embodiment, protein-protein interactions are measured. In addition to assessing presence and abundance of individual proteins, assays such as Proximity Extension Assay (PEA) allow for assaying the proximity of two proteins. In particular, the present invention allows for probing protein-protein interactions by designing pairs of antibodies for the interacting proteins of interest, such that the oligos conjugated to these antibodies have a binding region, which only bind when the two proteins are in near proximity, and therefore only PCR amplify in this case.</div>
</li> <li> <para-num num="[0475]"> </para-num> <div class="description-line" id="p-0424" num="0475">In one embodiment, protein-DNA interaction measurements are determined. Similar to the modified ISH-PLA described herein, instead of probing histone modifications, one could probe protein (transcription factor) proximity to many specific genetic loci, in a multiplex fashion.</div>
</li> <li> <para-num num="[0476]"> </para-num> <div class="description-line" id="p-0425" num="0476">In one embodiment, fluorescent in situ hybridization methods are used in the present invention. The present invention allows a combined approach where cells can be fluorescently labeled by methods known in the art, and cells of interest can be selected for downstream profiling of cellular constituents. In addition, the assays of the present invention can be combined with in situ hybridization methods such as RNA and DNA FISH.</div>
</li> <li> <para-num num="[0477]"> </para-num> <div class="description-line" id="p-0426" num="0477">In another embodiment, the gelled and cleared cells offer a platform in which any biological agent that is able to be detected by a high affinity and specific counterpart or ligand that can directly or indirectly be conjugated to a DNA molecule could be detected and quantified using the methods of the present invention.</div>
</li> <li> <para-num num="[0478]"> </para-num> <div class="description-line" id="p-0427" num="0478">Releasing the oligo's to be sequenced from their antibody can take a multitude of forms; i.e. in one embodiment, oligo's could be released from their antibodies by digesting all proteins (for instance proteinase K), alternatively, photocleavable linkers could be used, or restriction sites could be included in the oligo sequence to allow for enzymatic restriction and release. In another embodiment, the oligo can stay bound to the antibody, and in situ amplified (i.e. either by PCR, rolling circle amplification or T7 polymerase amplification) and the products of this reaction could be captured and sequenced.</div>
</li> <li> <para-num num="[0479]"> </para-num> <div class="description-line" id="p-0428" num="0479">Similarly, capturing the released oligo's could take a number of forms: in a drop based approach, beads can be loaded with capture oligo's as described herein. Microwells could either be loaded with beads, or their surface could be functionalized with capture oligos from which further amplification could take place. Alternatively, in the scenario where drops are sorted into multiwell plates, or microfluidic reaction chambers such as the Fluidigm C1 system, oligos can be amplified linearly or exponentially, and cellular barcodes and library adapters can be added on during these amplification steps.</div>
</li> <li> <para-num num="[0480]"> </para-num> <div class="description-line" id="p-0429" num="0480">Many different assays have been developed for oligo-barcode based detection of proteins (Janssen et al., 2013) and may be used in the present invention.</div>
</li> <li> <para-num num="[0481]"> </para-num> <div class="description-line" id="p-0430" num="0481">In one embodiment, cells are fixed and monomer infused before capturing them in a droplet. Alternatively, cells or aggregations of constituents are co-flowed with a lysis/monomer solution into a larger diameter drop. In this embodiment, biomolecules from a single cell or isolated aggregation of constituents are spread over a larger volume, which with similar polymer density could increase accessibility for staining.</div>
</li> <li> <para-num num="[0482]"> </para-num> <div class="description-line" id="p-0431" num="0482">The present invention also provides for cell handling before hydrogel polymerization. In one embodiment, cells are fixed and infused with polymer monomers in bulk. Cells may then be segregated and polymerization initiated. Segregation can be by any means described herein. In preferred embodiments, segregation is performed by making single cell drops.</div>
</li> <li> <para-num num="[0483]"> </para-num> <div class="description-line" id="p-0432" num="0483">In another embodiment, biochemical, thermal, or optical treatment on chip of individual cells in reverse emulsion droplets is performed. In this embodiment, polymer monomers may be spiked in microfluidically and optionally fixation reagents. Polymerization of the monomers may then be performed. This allows biochemical, thermal, or optical treatments at the single-cell level. Examples include, but are not limited to: lysis, DNA/RNA fragmentation/tagmentation, dosing with drugs, enzymatic reactions, or any perturbation of the sample before fixation and/or anchoring biomolecules to the polymer mesh upon polymerization.</div>
</li> <li> <para-num num="[0484]"> </para-num> <div class="description-line" id="p-0433" num="0484">In one embodiment, the oligonucleotide label may comprise Iso-deoxyguanosine (iso-dG) and 5-methyl iso-dC (iso-dC). Iso-deoxyguanosine forms a Watson-Crick base pair with 5-methyl iso-dC, but has a different type of hydrogen bonding pattern than those observed for the natural base pairs A:T and C:G. Substitution of a iso-dG: 5-Me-iso-dC base pair for a C:G pair increases the Tm of the resulting duplex by ˜2 deg C per base pair substitution (Switzer, C., et al., Enzymatic incorporation of a new base pair into DNA and RNA. <i>J. Am Chem. Soc. </i>(1989), 111: 8322-8323; and Horn, T., et al., Hybridization properties of the 5-methy-isocytidine/isoguanosine base pair in synthetic oligodeoxynucleotides. Tetrahedron Lett. (1995), 36: 2033-2036). Furthermore, since iso-dG does not pair with dC, iso-dG: 5-Me-iso-dC can function as a stable unnatural base pair that can be used to expand the genetic code. The combination of iso-dG's high selectivity for 5-Me-iso-dC, and the resulting base pair's high thermodynamic stability, make this modified base particular attractive in embodiments of the present invention.</div>
</li> <li> <para-num num="[0485]"> </para-num> <div class="description-line" id="p-0434" num="0485">In one embodiment, iso-dG:5-Me-iso-dC base pairing is used for molecular recognition. The 5-Me-iso-dC:iso-dG base pair may be incorporated into hybridization assays to enhance probe-target specificity and reduce spurious hybridization to non-target sequences. For example, Collins and co-workers significantly improved the sensitivity of a branched DNA quantitative hybridization assay for detecting the HIV POL sequence by incorporating ˜30% 5-Me-iso-dC and iso-dG into the pre-amplifier, branched DNA (bDNA) amplifier and alkaline phosphate probe sequences used in the assay (Collins, M.L, et al. A branched DNA signal amplification assay for quantification of nucleic acid targets below 100 molecules/ml.Nucleic Acids Res. (1997), 25: 2979-2984). Use of this strategy resulted in a significant reduction in non-specific hybridization of the above three sequence types to non-target nucleic acid sequences, and thus less amplification of background. The limits of detection of the assay were improved 10-fold, from &lt;500 HIV molecules/mL to &lt;50 molecules/mL. In preferred embodiments, the present invention utilizes the 5-Me-iso-dC:iso-dG base pair to ensure the correct sequences base pair during hybridization of ligation handle primers and during hybridization of two oligonucleotide labels in proximity assays.</div>
</li> <li> <para-num num="[0486]"> </para-num> <div class="description-line" id="p-0435" num="0486">In another embodiment, iso-dG:5-Me-iso-dC base pairing is used for qPCR and artificially expanded genetic systems. A number of research groups have been working on optimizing PCR amplification on templates containing 5-Me-iso-dC. Such optimization is necessary to enable the full development of artificially expanded genetic systems utilizing an expanded genetic code, thereby allowing for the site-specific incorporation of novel functional components (such as unnatural amino acids) into proteins. In 2004, Johnson and co-workers observed that, by using the Klenow fragment of Taq polymerase (KF-Taq) in PCR, the fidelity of the 5-Me-iso-dC:iso-dG base pair was about 96% per amplification cycle (Johnson, S.C., et al., A third base pair for the polymerase chain reaction: inserting isoC and isoG. Nucleic Acids Resl. (2004), 32: 1937-1941). The limit in fidelity is chiefly due to the ability of iso-dG's 1,2 tautomer to mis-pair with dT. More recently, Sismour and Benner solved this problem by using 2-thio-dT (dT*) in place of dT. dT*pairs with dA, but not with iso-dG (Sismour, A. M.; Benner, S. A. The use of thymidine analogs to improve the replication of an extra DNA base pair: a synthetic biological system. Nucleic Acids Resl. (2005), 33: 5640-5646). Using this artificial base pair system (5-Me-iso-dC:iso-dG, dA:dT*, dC:dG) with KF-Taq, the fidelity in PCR was increased to about 98% per amplification cycle.</div>
</li> <li> <para-num num="[0487]"> </para-num> <div class="description-line" id="p-0436" num="0487">The practice of the present invention employs, unless otherwise indicated, conventional techniques of immunology, biochemistry, chemistry, molecular biology, microbiology, cell biology, genomics and recombinant DNA, which are within the skill of the art. See MOLECULAR CLONING: A LABORATORY MANUAL, 2nd edition (1989) (Sambrook, Fritsch and Maniatis); MOLECULAR CLONING: A LABORATORY MANUAL, 4th edition (2012) (Green and Sambrook); CURRENT PROTOCOLS IN MOLECULAR BIOLOGY (1987) (F. M. Ausubel, et al. eds.); the series METHODS IN ENZYMOLOGY (Academic Press, Inc.); PCR 2: A PRACTICAL APPROACH (1995) (M.J. MacPherson, B. D. Hames and G. R. Taylor eds.); ANTIBODIES, A LABORATORY MANUAL (1988) (Harlow and Lane, eds.); ANTIBODIES A LABORATORY MANUAL, 2nd edition (2013) (E. A. Greenfield ed.); and ANIMAL CELL CULTURE (1987) (R. I. Freshney, ed.).</div>
</li> <li> <para-num num="[0488]"> </para-num> <div class="description-line" id="p-0437" num="0488">The practice of the present invention employs, unless otherwise indicated, conventional techniques for generation of genetically modified mice. See Marten H. Hofker and Jan van Deursen, TRANSGENIC MOUSE METHODS AND PROTOCOLS, 2nd edition (2011).</div>
</li> <li> <para-num num="[0489]"> </para-num> <div class="description-line" id="p-0438" num="0489">The present invention also provides methods applicable to the study of bulk cells and is not limited to single cells. Moreover, the assays described herein are also amenable to regularly fixed and permeabilized cells (i.e. not using polymerization). The proximity assays described herein may be performed on cells without generating discrete polymer matrices. Additionally, detection of cellular constituents utilizing labeling ligands and a sequencing readout may be used to detect low abundant cellular constituents. Not being bound by a theory, the oligonucleotide label may be amplified and increase the signal as compared to antibody readouts known in the art. Moreover, determination of proteins in relation to open chromatin need not be performed in a polymer matrix.</div>
</li> <li> <para-num num="[0490]"> </para-num> <div class="description-line" id="p-0439" num="0490">The present inventions provides advantages over prior assays for detecting proteins and post translation modifications (PTM) in single cells or isolated aggregations of cellular constituents. Standard flow cytometry can be used to detect a few proteins/PTMs in greater than 10<sup>6 </sup>single cells; and CyTOF (heavy metal labeling with multiplex barcoding) can be used to detect ˜30-50 proteins/PTMs in 10<sup>5</sup>-10<sup>6 </sup>single cells. The present invention provides highly multiplexed, DNA sequencing-based readouts of protein/PTM levels of greater than 100′s of proteins/PTMs in greater than 10<sup>6 </sup>cells.</div>
</li> <li> <para-num num="[0491]"> </para-num> <div class="description-line" id="p-0440" num="0491">The present invention advantageously provides a Massively Combinatorial Perturbation Profiling (MCPP) approach. Applicants can perturb vast numbers of combinations of genes, each targeting many circuit components at once. Applicants can use massively-parallel single cell genomics to measure genomic profiles and single cell proteomics to measure protein profiles after each perturbation. Applicants can infer the individual and combinatorial effects at each order, relying on random matrix theory, compressive sensing and kernel learning .</div>
</li> <li> <para-num num="[0492]"> </para-num> <div class="description-line" id="p-0441" num="0492">Biological systems are not linear: the combined effect of multiple factors is not simply the sum of their individual effects. This is a direct outcome of the biochemistry underlying molecular biology, from allosteric protein changes to cooperative binding, and is essential for cells to process complex signals. However, it has remained an insurmountable stumbling block to achieving a quantitative and predictive understanding of circuits on a genomic scale, with far-reaching implications for basic and translational science. Thus, the present invention provides a powerful combination by being able to measure transcriptional chromatin, epigenetic and proteomic changes as a function of genetic perturbation at the single cell level.</div>
</li> <li> <para-num num="[0493]"> </para-num> <div class="description-line" id="p-0442" num="0493">Combinatorial perturbation analyses have measured important genetic interactions, mainly from growth phenotypes in yeast. Mammalian studies have used ricin susceptibility and cell count phenotypes, but none combined large-scale, combinatorial genetic manipulation with complex, quantitative phenotypes, such as proteomic profiles. The single cell resolution readout of both response and perturbation, across many cells, serves as an improved starting point to unravel the function and interaction of the perturbed genes.</div>
</li> <heading id="h-0021">Sparse Coding</heading>
<li> <para-num num="[0494]"> </para-num> <div class="description-line" id="p-0443" num="0494">A simple, flexible, and cost-effective, transcriptome-wide gene-expression profiling solution that does not require measuring individual genes or single cell profiling is desired. This would greatly accelerate the rate of discovery of medically-relevant connections encoded therein by leveraging knowledge of relative abundances of genes to extrapolate underlying cell circuitry</div>
</li> <li> <para-num num="[0495]"> </para-num> <div class="description-line" id="p-0444" num="0495">The present invention relates to genomic informatics and gene-expression profiling. Gene-expression profiles provide complex molecular fingerprints regarding the relative state of a cell or tissue. Similarities in gene-expression profiles between organic states (i.e., for example, normal and diseased cells and/or tissues) provide molecular taxonomies, classification, and diagnostics. Similarities in gene-expression profiles resulting from various external perturbations (i.e., for example, ablation or enforced expression of specific genes, and/or small molecules, and/or environmental changes) reveal functional similarities between these perturbagens, of value in pathway and mechanism-of-action elucidation. Similarities in gene-expression profiles between organic (e.g. disease) and induced (e.g. by small molecule) states may identify clinically-effective therapies.</div>
</li> <li> <para-num num="[0496]"> </para-num> <div class="description-line" id="p-0445" num="0496">To achieve these and other advances and in accordance with the purpose of the present invention, as embodied and broadly described, in one aspect of the present invention, a probe set comprising 100 or more molecules assembled according to a set of random measurement values forming at least one measurement vector, where each molecule comprises a tag for the at least one measurement vector operably linked to a probe for one type of transcript of a plurality of types of transcripts.</div>
</li> <li> <para-num num="[0497]"> </para-num> <div class="description-line" id="p-0446" num="0497">In an aspect of the present invention, the probe set corresponds to a Design Matrix comprising m×n measurement values, where m is a number of measurement vectors and n is the number of types of transcripts. In another aspect, the tag uniquely corresponds to one of the measurement vectors. In an aspect, the molecules of the probe set are single-stranded DNA. In an aspect, the tag is a barcode. In another aspect, the transcript is a gene.</div>
</li> <li> <para-num num="[0498]"> </para-num> <div class="description-line" id="p-0447" num="0498">In an aspect, the number of measurement vectors is approximately 100-30,000. In an aspect, the number of measurement vectors is based on an estimate of system sparsity. In an aspect, the number of measurement vectors is based on the log of the number of types of transcripts. In an aspect the number of measurement vectors is approximately k log (n), where k is an estimate of sparsity. In an aspect, the Design Matrix may be adjusted according to a basis. In an aspect, k is approximately equal to 10. In another aspect, m is less than n. In another aspect, n is greater than 10.</div>
</li> <li> <para-num num="[0499]"> </para-num> <div class="description-line" id="p-0448" num="0499">In another aspect of the invention, a method of measuring relative abundances of transcripts in a pool of samples, comprises generating a Design Matrix comprising mxn measurement values, where m is a number of measurement vectors and n is the number of types of transcripts; generating a probe library corresponding to the Design Matrix, wherein the probe library comprises a collection of molecules assembled according to the measurement values, where each molecule has a tag for one of the measurement vectors operably linked to a probe for one of the types of transcripts; contacting the probe library to the pool of samples, resulting in m measurement results for each sample of the pool of samples; generating an Observed Measurement Matrix M comprising the measurement results for each sample of the pool of samples; and applying a sparse coding solving process to the Observed Measurement Matrix M to learn system matrix S as indicative of relative abundance of the transcripts in each of the samples.</div>
</li> <li> <para-num num="[0500]"> </para-num> <div class="description-line" id="p-0449" num="0500">In another aspect of the present invention, the measurement values in the Design Matrix are independent. In another aspect, the measurement values in the Design Matrix are random.</div>
</li> <li> <para-num num="[0501]"> </para-num> <div class="description-line" id="p-0450" num="0501">In an aspect, the number of measurement vectors is approximately 100-30,000. In an aspect, the number of measurement vectors is based on an estimate of system sparsity. In an aspect, the number of measurement vectors is based on the log of the number of types of transcripts. In an aspect the number of measurement vectors is approximately k log (n), where k is an estimate of sparsity. In an aspect, the Design Matrix may be adjusted according to a basis. In an aspect, k is approximately equal to 10. In another aspect, m is less than n. In another aspect, n is greater than 10.</div>
</li> <li> <para-num num="[0502]"> </para-num> <div class="description-line" id="p-0451" num="0502">In another aspect of the present invention, the tag uniquely corresponds to one of the measurement vectors. In an aspect, the tag is a barcode.</div>
</li> <li> <para-num num="[0503]"> </para-num> <div class="description-line" id="p-0452" num="0503">In another aspect of the present invention, the molecules of the probe library are single-stranded DNA. In an aspect, each molecule in the probe library further comprises a tag for one of the samples.</div>
</li> <li> <para-num num="[0504]"> </para-num> <div class="description-line" id="p-0453" num="0504">In another aspect of the present invention, the contacting includes binding and the contacting may, in some aspects, include hybridization.</div>
</li> <li> <para-num num="[0505]"> </para-num> <div class="description-line" id="p-0454" num="0505">In another aspect of the present invention, the sample is a cell.</div>
</li> <li> <para-num num="[0506]"> </para-num> <div class="description-line" id="p-0455" num="0506">In another aspect of the present invention, generating the Observed Measurement Matrix include hybrid selection and tag quantification. In an aspect, tag quantification includes sequencing.</div>
</li> <li> <para-num num="[0507]"> </para-num> <div class="description-line" id="p-0456" num="0507">In another aspect of the present invention, a method for measuring relative abundances of n biomolecules in a pool of samples, comprises generating a Design Matrix comprising mxn measurement values, where m is a number of measurement vectors and n is the number of types of biomolecules; generating a probe library corresponding to the Design Matrix, wherein the probe library comprises a collection of molecules assembled according to the measurement values, where each molecule has a tag for one of the measurement vectors operably linked to a probe for one of the types of biomolecules; contacting the probe library to the pool of samples, resulting in m measurement results for each sample of the pool of samples; generating an Observed Measurement Matrix M comprising the measurements results for each sample in the pool of samples; and applying a sparse coding solving process to the Observed Measurement Matrix to learn system matrix S as indicative of relative abundance of the biomolecules in each of the samples.</div>
</li> <li> <para-num num="[0508]"> </para-num> <div class="description-line" id="p-0457" num="0508">In another aspect of the present invention, the measurement values in the Design Matrix are independent. In another aspect, the measurement values in the Design Matrix are random.</div>
</li> <li> <para-num num="[0509]"> </para-num> <div class="description-line" id="p-0458" num="0509">In an aspect of the present invention, the biomolecule is a transcript, protein, DNA, non-naturally occurring nucleic acid, peptide. In an aspect of the present invention, the samples include cells, blood, hair, nails, mucus, tissue, feces or urine. In an aspect of the present invention, the probe is a molecule that binds to the biomolecule. In an aspect, the probe is a complex of molecules. In another aspect, the probe comprises an antibody or binding fragment thereof.</div>
</li> <li> <para-num num="[0510]"> </para-num> <div class="description-line" id="p-0459" num="0510">In another aspect of the present invention, the molecules of the probe library are single-stranded DNA. In an aspect, each molecule in the probe library further comprises a tag for one of the samples.</div>
</li> <li> <para-num num="[0511]"> </para-num> <div class="description-line" id="p-0460" num="0511">In another aspect of the present invention, the contacting includes binding and the contacting may, in some aspects, include hybridization.</div>
</li> <li> <para-num num="[0512]"> </para-num> <div class="description-line" id="p-0461" num="0512">In another aspect of the present invention, the tag uniquely corresponds to one of the measurement vectors. In an aspect, the tag is a barcode. In an aspect, each molecule in the probe library further comprises a tag for one of the samples.</div>
</li> <li> <para-num num="[0513]"> </para-num> <div class="description-line" id="p-0462" num="0513">In an aspect, the number of measurement vectors is approximately 1-30,000. In an aspect, the number of measurement vectors is based on an estimate of system sparsity. In an aspect, the number of measurement vectors is based on the log of the number of types of biomolecules. In an aspect the number of measurement vectors is approximately k log (n), where k is an estimate of sparsity. In an aspect, the Design Matrix may be adjusted according to a basis. In an aspect, k is approximately equal to 10. In another aspect, m is less than n. In another aspect, n is greater than 10.</div>
</li> <li> <para-num num="[0514]"> </para-num> <div class="description-line" id="p-0463" num="0514">In another aspect of the present invention, generating the Observed Measurement Matrix include tag quantification. In an aspect, tag quantification includes sequencing.</div>
</li> <li> <para-num num="[0515]"> </para-num> <div class="description-line" id="p-0464" num="0515">The term “device” as used herein, refers to any composition capable of measuring expression levels of transcripts. For example, a device may comprise a solid planar substrate capable of attaching nucleic acids (i.e., an oligonucleotide microarray). Alternatively, a device may comprise a solution-based bead array, wherein nucleic acids are attached to beads and detected using a flow cytometer. Alternatively, a device may comprise a nucleic-acid sequencer.</div>
</li> <li> <para-num num="[0516]"> </para-num> <div class="description-line" id="p-0465" num="0516">The term “probe” as used herein, refers to any molecule capable of attaching and/or binding to a nucleic acid (i.e., for example, a barcode nucleic acid). For example, a capture probe may be an oligonucleotide attached to a bead, wherein the oligonucleotide is at least partially complementary to another oligonucleotide. Alternatively, a capture probe may comprise a polyethylene glycol linker, an antibody, a polyclonal antibody, a monoclonal antibody, an Fab fragment, a biological receptor complex, an enzyme, a hormone, an antigen, and/or a fragment or portion thereof. A probe may be a nucleic acid sequence, the nucleic acid being for example deoxyribonucleic acid (DNA), ribonucleic acid (RNA), peptide nucleic acid (PNA) or other non-naturally occurring nucleic acid.</div>
</li> <li> <para-num num="[0517]"> </para-num> <div class="description-line" id="p-0466" num="0517">The term “Design Matrix” as used herein, refers to a collection of the relative number of times up to which a type of transcript can be counted for a specific type of measurement. The collection may be described as a table and will be so in the following for the sake of the description. However, it is understood that the collection need not be generated as a table but can also be generated in any other form suitable for the measuring relative abundances of transcripts, e.g. a string of data. Each entry in the table is intended to be a randomly generated number, at least initially. In general, each row in the matrix corresponds to a specific type of measurement, while each column in the matrix corresponds to a type of transcripts in the sample pool. Thus, the number of rows in the Design Matrix corresponds to the number of specific types of measurements, while the number of columns in the Design Matrix corresponds to the number of types of transcripts in the sample pool. Of course, one of skill may choose to adjust the Design Matrix to reflect fewer than all available types of measurement and fewer than all types of transcripts in the sample pool. Further, one of skill in the art will appreciate that the Design Matrix itself may be transposed such that the number of rows indicates the number of types of transcripts, while the number of columns indicates the number of types of measurement.</div>
</li> <li> <para-num num="[0518]"> </para-num> <div class="description-line" id="p-0467" num="0518">The term “relative number of times” as used herein, means that if for a type of measurements, the relative number is a<sub>1 </sub>for type of transcripts 1, a<sub>2 </sub>for type of transcripts 2, . . . a<sub>n </sub>for type of transcripts n, then type of transcripts 1 can be counted up to k.a<sub>1 </sub>times, type of transcripts 2 up to k.a<sub>2 </sub>times, . . . type of transcripts n up to k.a<sub>n </sub>times; k being an integer.</div>
</li> <li> <para-num num="[0519]"> </para-num> <div class="description-line" id="p-0468" num="0519">The term “count” and its derivatives as used in the definitions, encompasses any method that yield a measured value indicative of the count.</div>
</li> <li> <para-num num="[0520]"> </para-num> <div class="description-line" id="p-0469" num="0520">The term “Observed Measurement Matrix” as used herein, refers to a collection of measured values of a specific type of measurement for a specific type of sample in the sample pool or portion of sample (such as a cell, or a type of cell). The collection may be described as a table and will be so in the following for the sake of the description. However, it is understood that the collection need not be generated as a table but can also be generated in any other form suitable for the measuring relative abundances of transcripts, e.g. a string of data. In general, each row in the matrix corresponds to a specific type of measurement, while each column in the matrix corresponds to a type of samples in the sample pool or portion of sample. Thus, the number of rows in the Observed Measurement Matrix corresponds to the number of specific types of measurements, while the number of columns in the Observed Measurement Matrix corresponds to the number of types of samples in the sample pool. Of course, one of skill may choose to adjust the Observed Measurement Matrix to reflect fewer than all available types of measurement and fewer than all types of samples in the sample pool. Further, one of skill in the art will appreciate that the Observed Measurement Matrix itself may be transposed such that the number of rows indicates the number of types of sample, while the number of columns indicates the number of types of measurement.</div>
</li> <li> <para-num num="[0521]"> </para-num> <div class="description-line" id="p-0470" num="0521">Correspondingly, any reference to a Matrix used herein, refers to a collection of values, which may be described as a table and will be so in the following for the sake of the description. However, it is understood that the collection need not be generated as a table but can also be generated in any other form suitable for the measuring relative abundances of transcripts, e.g. a string of data. Any reference to a vector (in the mathematical sense) used herein, refers to a collection of values which may be described as row or a column, in some instances row or column of a table, and will be so in the following for the sake of the description. However, it is understood that the collection need not be generated as a row or a column but can also be generated in any other form suitable for the measuring relative abundances of transcripts.</div>
</li> <li> <para-num num="[0522]"> </para-num> <div class="description-line" id="p-0471" num="0522">The term “Connectivity Map” as used herein, refers to a public database of transcriptome-wide gene-expression profiles derived from cultured human cells treated with a plurality of perturbagens, and pattern-matching algorithms for the scoring and identification of significant similarities between those profiles and external gene-expression data, as described by Lamb et al., “The Connectivity Map: using gene-expression signatures to connect small molecules, genes and disease”. Science 313:1929 (2006). Build02 of the Connectivity Map contains 7,056 full-transcriptome gene-expression profiles generated with Affymetrix high-density oligonucleotide microarrays representing the biological effects of 1,309 small-molecule perturbagens, and is available at broadinstitute.org/cmap.</div>
</li> <li> <para-num num="[0523]"> </para-num> <div class="description-line" id="p-0472" num="0523">The term “query signature” as used herein, refers to any set of up- and down-regulated genes between two cellular states (e.g., cells treated with a small molecule versus cells treated with the vehicle in which the small molecule is dissolved) derived from a gene-expression profile that is suitable to query Connectivity Map. For example, a ‘query signature’ may comprise a list of genes differentially expressed in a distinction of interest; (e.g., disease versus normal), as opposed to an ‘expression profile’ that illustrates all genes with their respective expression levels.</div>
</li> <li> <para-num num="[0524]"> </para-num> <div class="description-line" id="p-0473" num="0524">The term “connectivity score” as used herein, refers to a relative measure of the similarity of the biological effects of a perturbagen used to generate a query signature with those of a perturbagen represented in the Connectivity Map based upon the gene-expression profile of a single treatment with that perturbagen. For example, one would expect every treatment instances with vorinostat, a known histone deacetylase (HDAC) inhibitor, to have a high connectivity score with a query signature generated from the effects of treatments with a panel of HDAC inhibitors.</div>
</li> <li> <para-num num="[0525]"> </para-num> <div class="description-line" id="p-0474" num="0525">The term “enrichment score” as used herein, refers to a measure of the similarity of the biological effects of a perturbagen used to generate a query signature with those of a perturbagen represented in the Connectivity Map based upon the gene-expression profiles of multiple independent treatments with that perturbagen.</div>
</li> <li> <para-num num="[0526]"> </para-num> <div class="description-line" id="p-0475" num="0526">The term “small organic molecule” as used herein, refers to any molecule of a size comparable to those organic molecules generally used in pharmaceuticals. The term excludes biological macromolecules (e.g., proteins, nucleic acids, etc.). Preferred small organic molecules range in size from approximately 10 Da up to about 5000 Da, more preferably up to 2000 Da, and most preferably up to about 1000 Da.</div>
</li> <li> <para-num num="[0527]"> </para-num> <div class="description-line" id="p-0476" num="0527">The sample may be a biological sample, for example a blood, buccal, cell, cerebrospinal fluid, mucus, saliva, semen, tissue, tumor, feces, urine, and vaginal sample. It may be obtained from an animal, a plant or a fungus. The animal may be a mammal. The mammal may be a primate. The primate may be a human. In other embodiments, the sample may be an environmental sample, such as water or soil.</div>
</li> <li> <para-num num="[0528]"> </para-num> <div class="description-line" id="p-0477" num="0528">As used herein, the term “sample template” refers to nucleic acid originating from a sample which is analyzed for the presence of a target sequence of interest. In contrast, “background template” is used in reference to nucleic acid other than sample template which may or may not be present in a sample. Background template is most often inadvertent. It may be the result of carryover, or it may be due to the presence of nucleic acid contaminants sought to be purified away from the sample. For example, nucleic acids from organisms other than those to be detected may be present as background in a test sample.</div>
</li> <li> <para-num num="[0529]"> </para-num> <div class="description-line" id="p-0478" num="0529">“Target sequence” is intended to designate either one target sequence or more than one target sequence, i.e. any sequence of interest at which the analysis is aimed. Thus, the sample may comprise more than one target sequence and preferably a plurality of target sequences, the number of which may be 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20 and above.</div>
</li> <li> <para-num num="[0530]"> </para-num> <div class="description-line" id="p-0479" num="0530">The present invention is related to the field of genomic informatics and gene-expression profiling. Gene-expression profiles provide complex molecular fingerprints regarding the relative state of a cell or tissue. Similarities in gene-expression profiles between organic states (i.e., for example, normal and diseased cells and/or tissues) provide molecular taxonomies, classification, and diagnostics. Similarities in gene-expression profiles resulting from various external perturbations (i.e., for example, ablation or enforced expression of specific genes, and/or small molecules, and/or environmental changes) reveal functional similarities between these perturbagens, of value in pathway and mechanism-of-action elucidation. Similarities in gene-expression profiles between organic (e.g. disease) and induced (e.g. by small molecule) states may identify clinically-effective therapies. Improvements described herein allow for the efficient and economical generation of full-transcriptome gene-expression profiles by identifying cluster centroid landmark transcripts that predict the expression levels of other transcripts within the same cluster.</div>
</li> <li> <para-num num="[0531]"> </para-num> <div class="description-line" id="p-0480" num="0531">Some embodiments of the present invention contemplate measuring relative gene abundances of transcripts in a pool of samples to allow genome-wide transcriptional profiling for applications including, but not limited to, disease classification and diagnosis without resort to expensive and laborious microarray technology (i.e., for example, Affymetrix GeneChip microarrays). Other uses include, but are not limited to, generating gene-expression data for use in and with information databases (i.e., for example, connectivity maps). A connectivity map typically may comprise a collection of a large number of gene-expression profiles together with allied pattern-matching software. The collection of profiles is searched with the pattern-matching algorithm for profiles that are similar to gene-expression data derived from a biological state of interest. The utility of this searching and pattern-matching exercise resides in the belief that similar biological states may be identified through the transitory feature of common gene-expression changes. The gene-expression profiles in a connectivity map may be derived from known cellular states, or cells or tissues treated with known chemical or genetic perturbagens. In this mode, the connectivity map is a tool for the functional annotation of the biological state of interest. Alternatively, the connectivity map is populated with gene-expression profiles from cells or tissues treated with previously uncharacterized or novel perturbagens. In this mode, the connectivity map functions as a screening tool. Most often, a connectivity map is populated with profiles of both types. Connectivity maps, in general, establish biologically-relevant connections between disease states, gene-product function, and small-molecule action. In particular, connectivity maps have wide-ranging applications including, but not limited to, functional annotation of unknown genes and biological states, identification of the mode of action or functional class of a small molecule, and the identification of perturbagens that modulate or reverse a disease state towards therapeutic advantage as potential drugs. See Lamb et al, “The Connectivity Map: using gene-expression signatures to connect small molecules, genes and disease” Science 313: 1929-1935 (2006), and Lamb, “The Connectivity Map: a new tool for biomedical research” Nature Reviews Cancer 7: 54-60 (2007). However, the high cost of generating gene-expression profiles severely limits the size and scope of connectivity maps. A connectivity map populated with gene-expression profiles derived from every member of an industrial small-molecule drug-screening library, a saturated combinatorial or diversity-orientated chemical library, a comprehensive collection of crude or purified plant or animal extracts, or from the genetic ablation or forced expression of every gene in a mammalian genome, for example, would be expected to facilitate more, and more profound, biological discoveries than those of existing connectivity maps.</div>
</li> <li> <para-num num="[0532]"> </para-num> <div class="description-line" id="p-0481" num="0532">The present invention contemplates compositions and methods for making and using a transcriptome-wide gene-expression profiling platform that “under samples” the total number of transcripts. Because gene expression is believed to be highly correlated, direct measurement of a small number allows the expression levels of the remainder to be inferred. The present invention, therefore, has the potential to reduce the cost and increase the throughput of full-transcriptome gene-expression profiling relative to the well-known conventional approaches that require all transcripts to be measured.</div>
</li> <li> <para-num num="[0533]"> </para-num> <div class="description-line" id="p-0482" num="0533">Gene expression data are highly structured, such that the expression level of some genes is predictive of the expression level of others. Knowledge that gene expression data are highly structured allows for the assumption that the number of degrees of freedom in the system are small, which allows for assuming that the basis for computation of the relative gene abundances is sparse.</div>
</li> <li> <para-num num="[0534]"> </para-num> <div class="description-line" id="p-0483" num="0534">It is possible to make several biologically motivated assumptions that allow Applicants to recover the nonlinear interaction terms while under-sampling without having any specific knowledge of which genes are likely to interact. In particular, if Applicants assume that genetic interactions are low rank, sparse, or a combination of these, then the true number of degrees of freedom is small relative to the complete combinatorial expansion, which enables Applicants to infer the full nonlinear landscape with a relatively small number of perturbations. Working around these assumptions, analytical theories of matrix completion and compressed sensing may be used to design under-sampled combinatorial perturbation experiments. In addition, a kernel-learning framework may be used to employ under-sampling by building predictive functions of combinatorial perturbations without directly learning any individual interaction coefficient.</div>
</li> <li> <para-num num="[0535]"> </para-num> <div class="description-line" id="p-0484" num="0535">In the framework of matrix completion, or more generally tensor completion, the goal is to fill in all the values of a matrix using a small collection of sampled entries. Applicants hypothesize that the rank of a matrix of second-order interactions, or tensor of higher-order interactions, is a fraction of the number of genes being tested. Applicants can test this idea by explicitly calculating the rank from a dense sampling of second or third order knockouts from a small collection of genes. If the rank of interactions is indeed limited, then Applicants can randomly sample pairs of genes to knockout from a larger collection (in the second order case), and then fill in the remaining values via nuclear norm regularized least-squares optimization [Matrix Completion with Noise. E. Candes and Y. Plan]. Provable guarantees for this recovery suggest that if the rank, r, is small relative to the number of genes, n, then m samples such that</div>
</li> <li> <div class="description-line" id="p-0485" num="0000">
<maths id="MATH-US-00001" num="00001">
<math overflow="scroll">
<mrow>
<mi>m</mi>
<mo>≥</mo>
<mrow>
<mi>O</mi>
<mo></mo>
<mrow>
<mo>(</mo>
<mrow>
<mfrac>
<mn>6</mn>
<mn>5</mn>
</mfrac>
<mo></mo>
<mi>nr</mi>
<mo></mo>
<mstyle>
<mspace height="0.3ex" width="0.3em"> </mspace>
</mstyle>
<mo></mo>
<mrow>
<mi>log</mi>
<mo></mo>
<mrow>
<mo>(</mo>
<mi>n</mi>
<mo>)</mo>
</mrow>
</mrow>
</mrow>
<mo>)</mo>
</mrow>
</mrow>
</mrow>
</math>
</maths>
</div>
</li> <li> <para-num num="[0536]"> </para-num> <div class="description-line" id="p-0486" num="0536">are sufficient. However, these guarantees assume rough uniformity in the loadings of interaction singular vectors, and this assumption is unlikely to hold up if the interaction matrix itself is very sparse. In this case, Applicants can perform the same random sampling scheme, and then simultaneously regularize over both the nuclear norm, and L<sup>1 </sup>norm of the interaction matrix [Estimation of Simultaneously Sparse and Low Rank Matrices. Richard et. al.]. As may be appreciated by one of skill in the art, the “sparsity” of a matrix is a measure of the non-zero elements of a matrix relative to the total number of elements of a matrix. A sparse matrix is a matrix in which most of the elements are zero, which is indicative of loose correlation between systems.</div>
</li> <li> <para-num num="[0537]"> </para-num> <div class="description-line" id="p-0487" num="0537">As may be appreciated by one of skill in the art, the “rank” of a matrix is the maximum number of linearly independent row vectors in the matrix or the maximum number of linearly independent column vectors in the matrix. The rank of a matrix denotes the “information content of the matrix. The lower the rank, the lower is the information content. A basis for a vector space is a collection of vectors that form a set that is linearly independent and that spans the space.</div>
</li> <li> <para-num num="[0538]"> </para-num> <div class="description-line" id="p-0488" num="0538">In the related setting of compressed sensing Applicants can directly leverage the sparse nature of interactions. In this case, instead of working with a tensor of interaction terms, Applicants work with a basis that spans all higher order interactions. This can be, for example, the Fourier basis of higher-order polynomials, in which case Applicants assume that there is a small number, s, of nonzero Fourier coefficients, which are not known a priori, but can accurately capture the effects of any combinatorial perturbation. Then, if Applicants make m random combinatorial knockouts, where m≥O(s log f) and f is the magnitude of combinatorial expansion</div>
</li> <li> <div class="description-line" id="p-0489" num="0000">
<maths id="MATH-US-00002" num="00002">
<math overflow="scroll">
<mrow>
<mrow>
<mo>(</mo>
<mrow>
<mrow>
<mi>e</mi>
<mo>.</mo>
<mi>g</mi>
<mo>.</mo>
<mstyle>
<mspace height="0.8ex" width="0.8em"> </mspace>
</mstyle>
<mo></mo>
<mi>f</mi>
</mrow>
<mo>=</mo>
<mrow>
<mrow>
<mo>(</mo>
<mtable>
<mtr>
<mtd>
<mn>100</mn>
</mtd>
</mtr>
<mtr>
<mtd>
<mn>3</mn>
</mtd>
</mtr>
</mtable>
<mo>)</mo>
</mrow>
<mo>=</mo>
<mrow>
<mrow>
<mo>“</mo>
<mrow>
<mn>100</mn>
<mo></mo>
<mstyle>
<mspace height="0.8ex" width="0.8em"> </mspace>
</mstyle>
<mo></mo>
<mi>choose</mi>
<mo></mo>
<mstyle>
<mspace height="0.8ex" width="0.8em"> </mspace>
</mstyle>
<mo></mo>
<mn>3</mn>
</mrow>
<mo>”</mo>
</mrow>
<mo>=</mo>
<mn>161700</mn>
</mrow>
</mrow>
</mrow>
<mo>)</mo>
</mrow>
<mo>,</mo>
</mrow>
</math>
</maths>
</div>
</li> <li> <div class="description-line" id="p-0490" num="0000">Applicants can recover the nonzero coefficients, and, thus, the set of nonlinear interaction terms, through L<sup>1 </sup>regularized regression. Compressed sensing tells Applicants that if the perturbations are de-coherent under the given basis, then exact recovery is possible with dramatic under-sampling (in the noiseless case) [Compressive sampling. E. Candes.]. Applicants use this measure of de-coherence, then, as a guide in designing combinatorial perturbations. In particular, if each of m experiments includes a Poisson random sampling of knockouts, then Applicants expect the measurements to have good de-coherence under the Fourier basis, provided the mean number of knockouts is not too low (e.g.&gt;1).</div>
</li> <li> <para-num num="[0539]"> </para-num> <div class="description-line" id="p-0491" num="0539">The formulations for matrix completion and compressed sensing each rely on a degree of sparsity—either in the rank or in the coefficients—to achieve recovery with under-sampling. For each of these Applicants can estimate the level of sparsity by performing dense combinatorial perturbations on a small collection of genes, or, in the case of compressed sensing, simply randomly sampling higher-order knockouts. However, Applicants also consider the alternative formulation of kernel learning, which does not have such strict demands on sparsity. In this case Applicants build predictive functions of the effects of combinatorial perturbations using a kernel of experimental similarity. More specifically, assume m experiments with Poisson random sampling of knockouts for each measurement. Then, Applicants can define an m-by-m polynomial kernel, for example, based on the overlap in knockouts between any pair of experiments. If Applicants build such a kernel, and learn the weighted combination of kernel vectors that fits a collection of training data, Applicants can then use these coefficients to predict the outcome of new experiments. In this case the density of nonlinear interaction terms can be much greater, since Applicants do not directly learn any particular interaction coefficient, but rather a kernelized version of the entire polynomial. In fact, if the interaction terms are too sparse, the kernel learning framework is unlikely to be successful with any significant under-sampling. However, together, kernel learning, matrix completion, and compressed sensing represent a complimentary range of approaches for inferring the effects of higher-order, combinatorial perturbations, with different assumptions on the underlying structure of the data for each framework.</div>
</li> <li> <para-num num="[0540]"> </para-num> <div class="description-line" id="p-0492" num="0540">The overall sparsity level of biological networks has been estimated in several ways. By comparing colony sizes between single and double mutants, the genetic landscape of <i>S. cerevisiae </i>has been mapped revealing that &lt;5% of 5.4 million gene pairs interact significantly with respect to fitness. [Costanzo*, Baryshnikova* et. al., 2011] In mammalian cells, a systematic screen for ricin toxicity using a hierarchical approach revealed that among 60 genes with significant individual effects, interactions broadly divided genes into functional complexes and were of a relatively higher density [Bassik et. al., 2013]. A higher dimensional phenotype using imaging between 282 target genes, 20 query genes, and 11 phenotypes found approximately 5,000 significant interactions [Laufer et. al., 2013]. Sparsity with respect to gene expression phenotypes has not been well studied.</div>
</li> <li> <para-num num="[0541]"> </para-num> <div class="description-line" id="p-0493" num="0541">The empirical limits on the number of DSB that might be tenable for an individual cell to survive vary, but rough estimates using imaging of phosphorylation of H2AX and 53BP1 suggest this number could be in the 10-50 range.</div>
</li> <li> <para-num num="[0542]"> </para-num> <div class="description-line" id="p-0494" num="0542">According to embodiments of the present invention, each measurement integrates signal across many genes (for example, all 29 k genes). Thus, the measurements are not sensitive to the stochastic capture of any single gene, but rather to the average signal across a broad range of genes. In this sense the present scheme is robust to technical noise. In addition, probe sets according to the present invention can be designed independently from any existing data. The probes are, in this sense, universally appropriate for any system desired to be measured. This method recognizes that there may be many different signature gene networks across many cell types/tissues, but that these are used to a sparse degree in any single cell. That is, in a single cell the number of active gene networks is small relative to the total number of all existing gene networks. This assumption can be used to under sample the transcriptome, such that a computational analysis is designed to recover signal from sparse systems. It is assumed that the gene networks (or their abstracted analogs) are sparsely used in any single cell, and specific knowledge of when the networks are active, or even the definitions of the networks themselves, are not required. Every gene is represented in every measurement, even if exact correlations between measurements, genes or expressions are not known. By analyzing transcriptomic measurements across many cells or tissues, it is possible to identify structures in the data that reflect the underlying cell circuitry. These same structures can be exploited to recover gene abundances, while dramatically under-sampling the full signal. In particular, if a given cell or tissue has a sparse representation in some basis, then Applicants can use the theories of compressive sensing and sparse coding to guide the design of RNA probe libraries that Applicants believe reduces sequencing requirements nearly 1000-fold.</div>
</li> <li> <para-num num="[0543]"> </para-num> <div class="description-line" id="p-0495" num="0543">To design compressive measurements, Applicants first consider the canonical basis described by Principle Component Analysis (PCA) as an example of an embedding for RNA-Seq data. Suppose, for the sake of argument, that Applicants knew of a universal set of 100 PCs, and that a linear combination of these could be used to describe the gene abundances in any system. By building molecular probes for each PC, Applicants could measure the response along each component in a given sample, and then take the linear combination of these 100 measurements to recover the abundance of every gene. Generally speaking, a basis that is universally appropriate for all systems (in a sense, this is an ultimate goal of biology) is not known. However, there is a surprising result which tells Applicants that making random measurements is universally appropriate for any basis that Applicants are likely to encounter [Compressive sampling. In Proc. Int. Congress of Math., Madrid, Spain, August 2006.]. After making random measurements, and using methods of sparse coding [Efficient sparse coding algorithms. In NIPS, 2006.], Applicants can simultaneously infer the basis and its sparse contributions to the samples in question, and recover the desired signal.</div>
</li> <li> <para-num num="[0544]"> </para-num> <div class="description-line" id="p-0496" num="0544">Measurements may be made via hybridization with targeted gene probes, which are barcoded and sequenced to count hybridization events. In such case, each measurement uses a given collection of probes, and there may be multiple probes per gene. For one of m measurements, a random library may be constructed from a barcode pool, and a pool of gene-specific specific oligos. Each oligo in the barcode pool may contain a universal adapter, a cell/experiment ID, a molecular ID, a measurement ID (one of m total), and a linker region. Gene specific oligos may contain a complimentary linker region, and a target sequence for the given gene. The final probe pool may be created by annealing and extension reactions.</div>
</li> <li> <para-num num="[0545]"> </para-num> <div class="description-line" id="p-0497" num="0545">The choice of probe sets may be a random design. Random probe sets have several advantages. Two random measurements are orthogonal (not correlated) with high probability, which means that each random measurement is effectively measuring something “new.” This is difficult, if not impossible, to ensure when each measurement consists of a single gene, and, thus, random probes do a better job of maximizing the information content of a small number of measurements. Existence of a gene correlation structure is assumed without knowing what it is. The probe sets are substantially universally appropriate. By randomly scattering measurements throughout the transcriptome, it can be ensured that Applicants sample from the relevant structures with high probability. Applicants then employ computational methods of Sparse Coding to learn the basis, or gene expression structure which is appropriate for the sample at hand. Importantly, the learning is integrated into the same process for inferring the full gene expression profile, and does not require a separate set of measurements. The number of types of measurements can be approximated by k log (n), where k is an estimated value of sparsity.</div>
</li> <li> <para-num num="[0546]"> </para-num> <div class="description-line" id="p-0498" num="0546">At least two methods for randomizing the amount of probe for each gene in each library are considered. First, the molecular IDs are synthesized randomly. Each of these IDs are assigned to +1/−1, such that when counting hybridization events for a given measurement Applicants either increment or decrement the total depending on the molecular ID. Thus, Applicants could have the same number of probes for every gene in each measurement, but the sum across molecular IDs for a given gene is a binomial random variable. Alternatively, a Liquid Handling Robot may be used to mix the barcode and gene pools according to a randomly generated matrix. In either case, the probe library is sequenced after construction to determine its composition.</div>
</li> <li> <para-num num="[0547]"> </para-num> <div class="description-line" id="p-0499" num="0547">In the present system, measurements are read out via sequencing, after hybrid selection and amplification. The number of types of measurements is contemplated to be between approximately 100 and 30,000 different types of measurements. The number of types of measurements can be based on an estimate of system sparsity. The number of types of measurements per sample can be, for example, 100. To the extent that there is any knowledge of assumptions about certain measurements or between genes, the number and type of measurements may be adjusted. For each measurement barcode within a cell, Applicants can have gene specific probes for each of ˜29k genes. However, the number of probes for a given gene will vary across each of the 100 measurements, according to random design. Once types of measurements are established, and the number or transcripts to be measured are determined, a Design Matrix having a number of rows corresponding to the number of types of measurements (measurement vectors) and the number of column corresponding to the number of transcripts can be generated. The Design Matrix is populated by “measurement values,” which are random numbers of measurements such that for each type of measurement each transcript can be counted up to a relative number of times that is random and independent of other elements/entries in the Design Matrix. In other words, the relative number of times up to which a certain measurement type can count each respective transcript is random and independent of the relative number of times up to which that certain measurement type can count other transcripts in the sample and the relative number of times up to which a certain transcript can be counted for a certain type of measurement is random and independent of the relative number of times up to which that certain transcript can be counted for other types of measurement. While the measurements may be random and independent, such is not required. Knowledge of some relationships between measurements may allow manipulation of the measurement values to improve calculations. For example, steps according to the present application may be iterated multiple times to refine results by incorporating knowledge gained through a first round of measurements. Similarly, knowledge of relationships between genes, measurements and transcripts may be used to inform and select some measurement values for the Design Matrix, although this is not required. For example, the Design Matrix can be adjusted according to a known basis.</div>
</li> <li> <para-num num="[0548]"> </para-num> <div class="description-line" id="p-0500" num="0548">A probe library corresponding to the Design Matrix may be generated. For example, a probe library comprises 100 or more molecules assembled according to a set of random measurement values forming at least one measurement vector, where each molecule comprises a tag for the at least one measurement vector operably linked to a probe for one type of transcript of a plurality of types of transcripts. The probe library includes a collection of molecules assembled according to the measurement values, where each molecule has a tag, e.g., a barcode, for one of the measurement types operably linked to a probe for one of the types of transcripts. The tag may uniquely correspond to a measurement type/vector. Probes contemplated according to the present invention may be single-stranded DNA, transcript, protein, DNA, non-naturally occurring nucleic acid, peptide, or the like. That is, each amount of probe can thus be adjusted based on the random measurement values in the Design Matrix such that the amount of a specific probe is known relative to the amounts of the other probes based on the Design Matrix.</div>
</li> <li> <para-num num="[0549]"> </para-num> <div class="description-line" id="p-0501" num="0549">While the present example describes cells as the sample for measurement, the sample may include cells, blood, hair, nails, mucus, tissue, feces, urine, body secretion or the like.</div>
</li> <li> <para-num num="[0550]"> </para-num> <div class="description-line" id="p-0502" num="0550">For one application, measurements may be made in single cells, and probe sets may be held as oligos on beads. For example, in one aspect each oligo may hold at least one of a sequencing adapter, a cellular barcode, a measurement barcode, a molecular barcode, and a primary gene-specific probe. cDNA targets may be captured on the primary probes, and then secondary probes, which target sequences immediately adjacent to the primary probes, may be introduced. Ligation of the primary and secondary probes produces a full-length fragment that can be amplified via universal sequences on the 5′ end of the bead-attached oligo, and 3′ end of the free-floating secondary probe. After amplification, Applicants sequence the barcode-containing regions to count the number of molecular events for each cell/measurement barcode combination. Contacting the probe library to the pool of samples thus results in a number of measurement results for each cell of the pool of cells.</div>
</li> <li> <para-num num="[0551]"> </para-num> <div class="description-line" id="p-0503" num="0551">The sample types are not limited to cells and may be any of numerous types of biomolecules. A biomolecule is any molecule that is present in living organisms, such as large macromolecules such as proteins, polysaccharides, lipids, and nucleic acids, as well as small molecules such as primary metabolites, secondary metabolites, and natural products. In an advantage embodiment, the biomolecule is a nucleic acid, such as but not limited to, deoxyribonucleic acid (DNA), ribonucleic acid (RNA), peptide nucleic acid (PNA) or other non-naturally occurring nucleic acid. In another advantageous embodiment, the biomolecule is a protein, such as but not limited to, peptides, antibodies, immunogenic molecules or enzymes.</div>
</li> <li> <para-num num="[0552]"> </para-num> <div class="description-line" id="p-0504" num="0552">An Observed Measurement Matrix may thus be constructed using the measurement results for each cell/sample to populate the elements/entries of the Observed Measurement Matrix. Construction of the Observed Measurement Matrix can include tag quantification, and tag quantification can include sequencing. For example, a relative count is conducted of the tags such that the relative number of times a certain tag is counted per cell is entered into an appropriate element of the Observed Measurement Matrix M. A sparse coding solving process may then be applied to the Observed Measurement Matrix M to learn system matrix S as indicative of relative abundance of the transcripts in each of the samples.</div>
</li> <li> <para-num num="[0553]"> </para-num> <div class="description-line" id="p-0505" num="0553">An example of a sparse coding process can be exemplified by</div>
</li> <li> <div class="description-line" id="p-0506" num="0000">
<maths id="MATH-US-00003" num="00003">
<math overflow="scroll">
<mrow>
<munder>
<mi>M</mi>
<mrow>
<mi>m</mi>
<mo>×</mo>
<mi>c</mi>
</mrow>
</munder>
<mo>=</mo>
<mrow>
<munder>
<mi>DM</mi>
<mrow>
<mi>m</mi>
<mo>×</mo>
<mi>n</mi>
</mrow>
</munder>
<mo>·</mo>
<munder>
<mi>S</mi>
<mrow>
<mi>n</mi>
<mo>×</mo>
<mi>c</mi>
</mrow>
</munder>
</mrow>
</mrow>
</math>
</maths>
</div>
</li> <li> <para-num num="[0554]"> </para-num> <div class="description-line" id="p-0507" num="0554">where m is the number of measurement vectors, c is the number of cell vectors, and n is the number of gene vectors. S is a System Matrix, which can be determined once M and DM are populated (M with actual measurements/counts and DM with random measurement values). S is thus indicative of the relative gene abundances.</div>
</li> <li> <para-num num="[0555]"> </para-num> <div class="description-line" id="p-0508" num="0555">Moreover, once S is known, a Basis Matrix B, can be determined based on knowledge of X, a matrix populated by known contributions of genes n and cells c to the sample.</div>
</li> <li> <div class="description-line" id="p-0509" num="0000">
<maths id="MATH-US-00004" num="00004">
<math overflow="scroll">
<mrow>
<munder>
<mi>S</mi>
<mrow>
<mi>n</mi>
<mo>×</mo>
<mi>c</mi>
</mrow>
</munder>
<mo>=</mo>
<mrow>
<munder>
<mi>B</mi>
<mrow>
<mi>n</mi>
<mo>×</mo>
<mi>n</mi>
</mrow>
</munder>
<mo>·</mo>
<munder>
<mi>X</mi>
<mrow>
<mi>n</mi>
<mo>×</mo>
<mi>c</mi>
</mrow>
</munder>
</mrow>
</mrow>
</math>
</maths>
</div>
</li> <li> <para-num num="[0556]"> </para-num> <div class="description-line" id="p-0510" num="0556">Knowledge of the Basis can then be used to generate new measurement values of the Design Matrix to refine calculation of a new System Matrix S′.</div>
</li> <li> <para-num num="[0557]"> </para-num> <div class="description-line" id="p-0511" num="0557">Testing for linearity of hybrid selection is contemplated, along with dynamic range, by, for example, splitting bulk cDNA prepared from a population of K562 cells. In one split, qRT-PCR is performed for a collection of 25 genes that range from the highest abundance, to being absent. In a second split, hybrid selection is tested on a collection of 24 random probe libraries (3 experimental IDs, each with 8 measurement indices). The goal is to optimize the hybridization conditions for maximal dynamic range, while ensuring that the readout from sequencing after hybridization is linear with regards to qPCR controls, and the amount of probe for a given gene in a given library. The size of the probe library is then be successively increased to 100 genes, then 1000, and finally to the set of all genes. The measurements are more robust with larger panels of genes, since Applicants' readout is effectively the sum of hybridization events across all probes for a given measurement index. In other words, since only barcodes are sequenced the barcodes and not the gene specific regions, the process is not sensitive to the stochastic capture of transcripts for any single gene, but rather to the average capture across all genes. Finally, this design is compatible with single cell technologies, and has indeed been designed with this application in mind. In particular, random probe libraries held on beads in place of barcoded poly-dT oligos are contemplated, but this is just one of many possible constructions.</div>
</li> <li> <para-num num="[0558]"> </para-num> <div class="description-line" id="p-0512" num="0558">In a first round of experiments, Applicants have constructed the random probe sets as follows. One pool of oligos is synthesized to contain a universal sequence, a cellular barcode, a measurement barcode, a molecular barcode, and a linker region. A second pool is synthesized to contain a complimentary linker region, and a gene-specific target. Oligos from the two pools can be annealed, and then extended in the 3′ direction on the barcoded fragment with the Klenow fragment, or T4 DNA polymerase. These can be treated with lambda exonuclease, and purified by size selection for the full-length fragment. This can be done for each gene, and then the fragments can be mixed and pooled according to random design. Alternatively, fragments for each gene in each measurement can be pooled in equal amounts. In this case, Applicants achieve random measurements by assigning each molecular identifier (which was synthesized randomly) to a +1/−1. When integrating the counts for a given measurement, Applicants then increment or decrement the count according to this assignment. Since the molecular barcodes and assignments are made randomly, the sum of +1/−1 assignments for a given gene behave as a random variable across the different measurements. Secondary gene probes can be synthesized independently.</div>
</li> <li> <para-num num="[0559]"> </para-num> <div class="description-line" id="p-0513" num="0559">It is anticipated that probe libraries may be constructed and provided and present methods applied to provide cost-effective gene-expression profiling. Accordingly a product comprising completed probe libraries, or a kit for making the libraries according to principles of the present invention are possible. For some applications it may make sense to focus on a subset of genes, in which case the kit would be particularly appropriate.</div>
</li> <li> <para-num num="[0560]"> </para-num> <div class="description-line" id="p-0514" num="0560">In addition, random probes and sparse coding could be applied as a service. That is, a customer may provide a sample or sample pools to an entity for constructing a probe library according to customer objectives and sample, applying the steps of the methods according to principles of the present invention and providing as its result a report of the relative gene-abundances or gene-expression profile or the like. This could be particularly useful for applications that require the analysis of tens to hundreds of thousands of transcriptional profiles, since Applicants estimate that methods according to principles of the present invention reduce sequencing requirements by 100-1000 fold. For example, when screening for combinatorial effects of small molecules, or when testing for the presence of very rare cell types (such as cancer stem cells), a large number of profiles need to be generated.</div>
</li> <li> <para-num num="[0561]"> </para-num> <div class="description-line" id="p-0515" num="0561">Rapidly advancing experimental methods are opening new ground for the exploration of higher-order genetic interactions. These methods include the CRISPR-Cas system for knockouts, which enables greater degrees of combinatorial perturbations, as well as high-throughput readout techniques such as single-cell RNA-Seq. The ability to sequence the transcriptomes of tens of thousands of cells using droplet microfluidics has recently been demonstrated [Macosko et. al., 2015]. By constructing a polyadenylated RNA barcode that is associated with a CRISPR sgRNA, Applicants was able to couple the experimental scale of microfluidic technologies with perturbation experiments in mammalian cells.</div>
</li> <li> <para-num num="[0562]"> </para-num> <div class="description-line" id="p-0516" num="0562">However, even with these advances, Applicants are limited to full combinatorial samplings of second- or third-order interactions for a small number of genes. For example, for a collection of 30 genes, performing all third-order knockouts is at the boundary of experimental feasibility (30 choose 3=4060 experiments), while a dense sampling of fourth-order terms for the same size collection (30 choose 4=27405), or third-order terms for a larger collection (100 choose 3=161700) is intractable. Therefore, Applicants are motivated to identify schemes that leverage Applicants' ability to perform combinatorial perturbations, while also dramatically under-sampling the full combinatorial space.</div>
</li> <li> <para-num num="[0563]"> </para-num> <div class="description-line" id="p-0517" num="0563">Compressive Sensing the Transcriptome</div>
</li> <li> <para-num num="[0564]"> </para-num> <div class="description-line" id="p-0518" num="0564">1st Trial Run—Analysis</div>
</li> <li> <para-num num="[0565]"> </para-num> <div class="description-line" id="p-0519" num="0565">Probe library creation</div>
</li> <li> <para-num num="[0566]"> </para-num> <div class="description-line" id="p-0520" num="0566">The new protocols generated a much higher percentage of fragments of the correct size (eg 70-99% stitched) as compared with the original L1000 protocol (19-42% stitched). Having larger cDNA input also resulted in a greater percentage of stitched fragments.</div>
</li> <li> <para-num num="[0567]"> </para-num> <div class="description-line" id="p-0521" num="0567">UMI counting</div>
</li> <li> <para-num num="[0568]"> </para-num> <div class="description-line" id="p-0522" num="0568">Correlations of individual gene counts with expectation based on RPKM from ENCODE were generally positive, but not stunning. Results with gene counts from qPCR were largely unchanged. For the new ligation protocols, correlations were generally positive, except for rowl which generally has low (negative) values. For the L1000 ligation protocol correlations were more consistently positive, except for condition 21, which shows negative correlations for many rows/conditions.</div>
</li> <li> <para-num num="[0569]"> </para-num> <div class="description-line" id="p-0523" num="0569">Applicants also considered UMIs aggregated across genes, with randomly chosen rows. This mimics the intended setting. In this case correlations were generally much higher (range 47-79%). The best condition by this measure was 26_27 (new protocol with low ligation, low L/R ratio with undiluted cDNA)</div>
</li> <li> <para-num num="[0570]"> </para-num> <div class="description-line" id="p-0524" num="0570">The normalization strategy used in initial trials is based on sequenced barcode counts from direct ligation of probes, without cDNA targets. Quantifying barcode counts in this way gives an overall estimate of the abundance of each member of the library. In future experiments, applicants will compare the results of this normalization strategy to a normalization based on ligation to a pool of genomic DNA.</div>
</li> <li> <para-num num="[0571]"> </para-num> <div class="description-line" id="p-0525" num="0571">X18 samples (3 FBC vs 6 RBC)</div>
</li> <li> <para-num num="[0572]"> </para-num> <div class="description-line" id="p-0526" num="0572">&lt;P5 barcoding&gt;</div>
</li> <li> <para-num num="[0573]"> </para-num> <div class="description-line" id="p-0527" num="0573">Original L1000: P5-30</div>
</li> <li> <para-num num="[0574]"> </para-num> <div class="description-line" id="p-0528" num="0574">New protocol (high ligation 5u1 mix input for R1, see notes below): P5-25</div>
</li> <li> <para-num num="[0575]"> </para-num> <div class="description-line" id="p-0529" num="0575">New protocol (low ligation lul mix input for R1, see notes below): P5-26</div>
</li> <li> <para-num num="[0576]"> </para-num> <div class="description-line" id="p-0530" num="0576">&lt;P7 barcoding&gt;</div>
</li> <li> <para-num num="[0577]"> </para-num> <div class="description-line" id="p-0531" num="0577">1. High L/R ratio probe mix+5 μL cDNA input (˜50 ng/μL): P7-14</div>
</li> <li> <para-num num="[0578]"> </para-num> <div class="description-line" id="p-0532" num="0578">2. High L/R ratio probe mix+5 μL (⅛ diluted) cDNA input (˜6 ng/μL): P7-21</div>
</li> <li> <para-num num="[0579]"> </para-num> <div class="description-line" id="p-0533" num="0579">3. Low L/R ratio probe mix +5 μL cDNA input (˜50 ng/μL): P7-27</div>
</li> <li> <para-num num="[0580]"> </para-num> <div class="description-line" id="p-0534" num="0580">4. Low L/R ratio probe mix +5 μL (⅛ diluted) cDNA input (˜6 ng/μL): P7-28</div>
</li> <li> <para-num num="[0581]"> </para-num> <div class="description-line" id="p-0535" num="0581">5. High L/R ratio probe mix direct ligation with T7: P7-15</div>
</li> <li> <para-num num="[0582]"> </para-num> <div class="description-line" id="p-0536" num="0582">6. Low L/R ratio probe mix direct ligation with T7: P7-20</div>
</li> <li> <para-num num="[0583]"> </para-num> <div class="description-line" id="p-0537" num="0583">Read Counts</div>
</li> <li> <div class="description-line" id="p-0538" num="0000">
<tables id="TABLE-US-00001" num="00001">
<patent-tables colsep="0" frame="none" rowsep="0">
<table align="left" class="description-table" cols="5" colsep="0" rowsep="0" width="100%">
<thead>
<tr class="description-tr">
<td align="center" class="description-td" colspan="5" nameend="5" namest="1" rowsep="1"> </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">Total</td>
<td class="description-td"> </td>
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">F</td>
<td class="description-td">R</td>
<td class="description-td">Fragments</td>
<td class="description-td">Stitched</td>
<td class="description-td">% Stitched</td>
</tr>
<tr class="description-tr">
<td align="center" class="description-td" colspan="5" nameend="5" namest="1" rowsep="1"> </td>
</tr>
</thead>
<tbody><tr class="description-tr">
<td class="description-td"> </td>
</tr>
</tbody></table>
<table align="left" class="description-table" cols="5" colsep="0" rowsep="0" width="100%">
<tbody><tr class="description-tr">
<td class="description-td">26</td>
<td class="description-td">27</td>
<td class="description-td">799365</td>
<td class="description-td">795574</td>
<td class="description-td">99.5%</td>
</tr>
<tr class="description-tr">
<td class="description-td">25</td>
<td class="description-td">27</td>
<td class="description-td">890612</td>
<td class="description-td">881037</td>
<td class="description-td">98.9%</td>
</tr>
<tr class="description-tr">
<td class="description-td">26</td>
<td class="description-td">20</td>
<td class="description-td">839036</td>
<td class="description-td">825118</td>
<td class="description-td">98.3%</td>
</tr>
<tr class="description-tr">
<td class="description-td">26</td>
<td class="description-td">14</td>
<td class="description-td">401287</td>
<td class="description-td">383828</td>
<td class="description-td">95.6%</td>
</tr>
<tr class="description-tr">
<td class="description-td">26</td>
<td class="description-td">28</td>
<td class="description-td">628901</td>
<td class="description-td">589907</td>
<td class="description-td">93.8%</td>
</tr>
<tr class="description-tr">
<td class="description-td">26</td>
<td class="description-td">21</td>
<td class="description-td">237079</td>
<td class="description-td">217109</td>
<td class="description-td">91.6%</td>
</tr>
<tr class="description-tr">
<td class="description-td">25</td>
<td class="description-td">14</td>
<td class="description-td">124932</td>
<td class="description-td">109626</td>
<td class="description-td">87.7%</td>
</tr>
<tr class="description-tr">
<td class="description-td">26</td>
<td class="description-td">15</td>
<td class="description-td">1067299</td>
<td class="description-td">906534</td>
<td class="description-td">84.9%</td>
</tr>
<tr class="description-tr">
<td class="description-td">25</td>
<td class="description-td">15</td>
<td class="description-td">1033888</td>
<td class="description-td">838764</td>
<td class="description-td">81.1%</td>
</tr>
<tr class="description-tr">
<td class="description-td">25</td>
<td class="description-td">28</td>
<td class="description-td">726668</td>
<td class="description-td">570421</td>
<td class="description-td">78.5%</td>
</tr>
<tr class="description-tr">
<td class="description-td">25</td>
<td class="description-td">20</td>
<td class="description-td">849166</td>
<td class="description-td">628951</td>
<td class="description-td">74.1%</td>
</tr>
<tr class="description-tr">
<td class="description-td">25</td>
<td class="description-td">21</td>
<td class="description-td">65325</td>
<td class="description-td">46374</td>
<td class="description-td">71.0%</td>
</tr>
<tr class="description-tr">
<td class="description-td">30</td>
<td class="description-td">20</td>
<td class="description-td">437882</td>
<td class="description-td">186679</td>
<td class="description-td">42.6%</td>
</tr>
<tr class="description-tr">
<td class="description-td">30</td>
<td class="description-td">14</td>
<td class="description-td">774238</td>
<td class="description-td">257647</td>
<td class="description-td">33.3%</td>
</tr>
<tr class="description-tr">
<td class="description-td">30</td>
<td class="description-td">15</td>
<td class="description-td">938377</td>
<td class="description-td">268906</td>
<td class="description-td">28.7%</td>
</tr>
<tr class="description-tr">
<td class="description-td">30</td>
<td class="description-td">21</td>
<td class="description-td">745495</td>
<td class="description-td">144001</td>
<td class="description-td">19.3%</td>
</tr>
<tr class="description-tr">
<td class="description-td">30</td>
<td class="description-td">28</td>
<td class="description-td">1066525</td>
<td class="description-td">189338</td>
<td class="description-td">17.8%</td>
</tr>
<tr class="description-tr">
<td class="description-td">30</td>
<td class="description-td">27</td>
<td class="description-td">966163</td>
<td class="description-td">149402</td>
<td class="description-td">15.5%</td>
</tr>
<tr class="description-tr">
<td align="center" class="description-td" colspan="5" nameend="5" namest="1" rowsep="1"> </td>
</tr>
</tbody></table>
</patent-tables>
</tables>
</div>
</li> <li> <para-num num="[0584]"> </para-num> <div class="description-line" id="p-0539" num="0584">Correlation with RPKM</div>
</li> <li> <para-num num="[0585]"> </para-num> <div class="description-line" id="p-0540" num="0585">For each read Applicants parse barcodes into structures such as:</div>
</li> <li> <para-num num="[0586]"> </para-num> <div class="description-line" id="p-0541" num="0586">cell1 AGGTCGTA row1 ACTB ACTB</div>
</li> <li> <para-num num="[0587]"> </para-num> <div class="description-line" id="p-0542" num="0587">Stitched reads that do not fit this format are discarded. For direct ligation (no cDNA), Applicants allow the left and right gene barcodes to differ, but for all others Applicants require that the gene barcodes are identical.</div>
</li> <li> <para-num num="[0588]"> </para-num> <div class="description-line" id="p-0543" num="0588">Once parsed, Applicants count the number of UMIs for every gene in each condition and row (8 rows per condition). After normalizing by the number of UMIs in the corresponding direct ligation condition, Applicants can compare the normalized UMI count with actual RPKM for each gene.</div>
</li> <li> <para-num num="[0589]"> </para-num> <div class="description-line" id="p-0544" num="0589">For a given experimental condition, probes were included at 8 dilution levels (rows 1-8). In row 1, probes for all targets were included at approximately the same concentration. In each successive dilution level (rows 2-8) the overall concentration of probes was serially diluted so that the probe concentration in row 8 was approximately 100-fold lower than the concentration in row 1. Thus, correlations of observed versus expected counts within each of the 8 dilution levels of a given experiment reflect the sensitivity of each assay across a 100-fold change in probe concentration. Correlations from these results are included at the end of this document. In <figref idrefs="DRAWINGS">FIG. 3</figref> the results for one example condition (24_26) are depicted in a scatter plot.</div>
</li> <li> <para-num num="[0590]"> </para-num> <div class="description-line" id="p-0545" num="0590">Applicants can also ask how each condition would perform in the random setting where Applicants aggregate counts across all genes. To do this Applicants begin with a single gene, and choose a random row from the given condition. Applicants increment the “observed” count by the number of UMIs for that gene in that row, and increment the “expected” count by the product of the direct ligation UMI count for that gene/row with the RPKM of the gene. This is repeated for every gene such that Applicants have “observed” and “expected” sums across all genes with randomly chosen rows. This was repeated 8 times to generate a vector of random observations and expectations. Below Applicants report the correlation of these vectors for each condition, averaged across 50 random trials. In <figref idrefs="DRAWINGS">FIG. 4</figref> the simulation results for one example condition (24_26) are depicted in a scatter plot.</div>
</li> <li> <div class="description-line" id="p-0546" num="0000">
<tables id="TABLE-US-00002" num="00002">
<patent-tables colsep="0" frame="none" rowsep="0">
<table align="left" class="description-table" cols="3" colsep="0" rowsep="0" width="100%">
<thead>
<tr class="description-tr">
<td class="description-td"> </td>
<td align="center" class="description-td" colspan="3" nameend="2" namest="offset" rowsep="1"> </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">condition</td>
<td class="description-td">correlation</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td align="center" class="description-td" colspan="3" nameend="2" namest="offset" rowsep="1"> </td>
</tr>
</thead>
<tbody><tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">25_21</td>
<td class="description-td">46.6%</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">25_14</td>
<td class="description-td">49.0%</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">26_21</td>
<td class="description-td">52.9%</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">30_14</td>
<td class="description-td">57.8%</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">30_27</td>
<td class="description-td">63.4%</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">30_21</td>
<td class="description-td">65.0%</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">30_28</td>
<td class="description-td">68.9%</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">25_27</td>
<td class="description-td">69.7%</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">26_14</td>
<td class="description-td">70.2%</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">25_28</td>
<td class="description-td">74.1%</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">26_28</td>
<td class="description-td">76.6%</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">26_27</td>
<td class="description-td">79.3%</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td align="center" class="description-td" colspan="3" nameend="2" namest="offset" rowsep="1"> </td>
</tr>
</tbody></table>
</patent-tables>
</tables>
</div>
</li> <li> <para-num num="[0591]"> </para-num> <div class="description-line" id="p-0547" num="0591">Results for individual genes in each condition (condition IDs offset by 1):</div>
</li> <li> <para-num num="[0592]"> </para-num> <div class="description-line" id="p-0548" num="0592">norm.24_13.row1.txt:correlation: 0.101860</div>
</li> <li> <para-num num="[0593]"> </para-num> <div class="description-line" id="p-0549" num="0593">norm.24_13.row2.txt:correlation: 0.198498</div>
</li> <li> <para-num num="[0594]"> </para-num> <div class="description-line" id="p-0550" num="0594">norm.24_13.row3.txt:correlation: 0.125971</div>
</li> <li> <para-num num="[0595]"> </para-num> <div class="description-line" id="p-0551" num="0595">norm.24_13.row5.txt:correlation: 0.184587</div>
</li> <li> <para-num num="[0596]"> </para-num> <div class="description-line" id="p-0552" num="0596">norm.24_13.row6.txt:correlation: −0.065662</div>
</li> <li> <para-num num="[0597]"> </para-num> <div class="description-line" id="p-0553" num="0597">norm.24_13.row7.txt:correlation: 0.548366</div>
</li> <li> <para-num num="[0598]"> </para-num> <div class="description-line" id="p-0554" num="0598">norm.24_13.row8.txt:correlation: 0.188612</div>
</li> <li> <para-num num="[0599]"> </para-num> <div class="description-line" id="p-0555" num="0599">norm.24_20.row1.txt:correlation: 0.018865</div>
</li> <li> <para-num num="[0600]"> </para-num> <div class="description-line" id="p-0556" num="0600">norm.24_20.row2.txt:correlation: 0.011796</div>
</li> <li> <para-num num="[0601]"> </para-num> <div class="description-line" id="p-0557" num="0601">norm.24_20.row3.txt:correlation: −0.067194</div>
</li> <li> <para-num num="[0602]"> </para-num> <div class="description-line" id="p-0558" num="0602">norm.24_20.row5.txt:correlation: −0.220015</div>
</li> <li> <para-num num="[0603]"> </para-num> <div class="description-line" id="p-0559" num="0603">norm.24_20.row6.txt:correlation: −0.157439</div>
</li> <li> <para-num num="[0604]"> </para-num> <div class="description-line" id="p-0560" num="0604">norm.24_20.row7.txt:correlation: 0.364206</div>
</li> <li> <para-num num="[0605]"> </para-num> <div class="description-line" id="p-0561" num="0605">norm.24_20.row8.txt:correlation: 0.080670</div>
</li> <li> <para-num num="[0606]"> </para-num> <div class="description-line" id="p-0562" num="0606">norm.24_26.row1.txt:correlation: 0.372970</div>
</li> <li> <para-num num="[0607]"> </para-num> <div class="description-line" id="p-0563" num="0607">norm.24_26.row2.txt:correlation: 0.778385</div>
</li> <li> <para-num num="[0608]"> </para-num> <div class="description-line" id="p-0564" num="0608">norm.24_26.row3.txt:correlation: 0.794532</div>
</li> <li> <para-num num="[0609]"> </para-num> <div class="description-line" id="p-0565" num="0609">norm.24_26.row5.txt:correlation: 0.501830</div>
</li> <li> <para-num num="[0610]"> </para-num> <div class="description-line" id="p-0566" num="0610">norm.24_26.row6.txt:correlation: 0.340869</div>
</li> <li> <para-num num="[0611]"> </para-num> <div class="description-line" id="p-0567" num="0611">norm.24_26.row7.txt:correlation: 0.562235</div>
</li> <li> <para-num num="[0612]"> </para-num> <div class="description-line" id="p-0568" num="0612">norm.24_26.row8.txt:correlation: 0.576566</div>
</li> <li> <para-num num="[0613]"> </para-num> <div class="description-line" id="p-0569" num="0613">norm.24_27.row1.txt:correlation: 0.381881</div>
</li> <li> <para-num num="[0614]"> </para-num> <div class="description-line" id="p-0570" num="0614">norm.24_27.row2.txt:correlation: 0.771273</div>
</li> <li> <para-num num="[0615]"> </para-num> <div class="description-line" id="p-0571" num="0615">norm.24_27.row3.txt:correlation: 0.684149</div>
</li> <li> <para-num num="[0616]"> </para-num> <div class="description-line" id="p-0572" num="0616">norm.24_27.row5.txt:correlation: 0.445928</div>
</li> <li> <para-num num="[0617]"> </para-num> <div class="description-line" id="p-0573" num="0617">norm.24_27.row6.txt:correlation: 0.170683</div>
</li> <li> <para-num num="[0618]"> </para-num> <div class="description-line" id="p-0574" num="0618">norm.24_27.row7.txt:correlation: 0.543163</div>
</li> <li> <para-num num="[0619]"> </para-num> <div class="description-line" id="p-0575" num="0619">norm.24_27.row8.txt:correlation: 0.530076</div>
</li> <li> <para-num num="[0620]"> </para-num> <div class="description-line" id="p-0576" num="0620">norm.25_13.row1.txt:correlation: 0.309518</div>
</li> <li> <para-num num="[0621]"> </para-num> <div class="description-line" id="p-0577" num="0621">norm.25_13.row2.txt:correlation: 0.490934</div>
</li> <li> <para-num num="[0622]"> </para-num> <div class="description-line" id="p-0578" num="0622">norm.25_13.row3.txt:correlation: 0.349481</div>
</li> <li> <para-num num="[0623]"> </para-num> <div class="description-line" id="p-0579" num="0623">norm.25_13.row5.txt:correlation: 0.397947</div>
</li> <li> <para-num num="[0624]"> </para-num> <div class="description-line" id="p-0580" num="0624">norm.25_13.row6.txt:correlation: 0.272807</div>
</li> <li> <para-num num="[0625]"> </para-num> <div class="description-line" id="p-0581" num="0625">norm.25_13.row7.txt:correlation: —0.158994</div>
</li> <li> <para-num num="[0626]"> </para-num> <div class="description-line" id="p-0582" num="0626">norm.25_13.row8.txt:correlation: 0.256215</div>
</li> <li> <para-num num="[0627]"> </para-num> <div class="description-line" id="p-0583" num="0627">norm.25_20.row1.txt:correlation: 0.141412</div>
</li> <li> <para-num num="[0628]"> </para-num> <div class="description-line" id="p-0584" num="0628">norm.25_20.row2.txt:correlation: 0.275601</div>
</li> <li> <para-num num="[0629]"> </para-num> <div class="description-line" id="p-0585" num="0629">norm.25_20.row3.txt:correlation: —0.170215</div>
</li> <li> <para-num num="[0630]"> </para-num> <div class="description-line" id="p-0586" num="0630">norm.25_20.row5.txt:correlation: —0.115663</div>
</li> <li> <para-num num="[0631]"> </para-num> <div class="description-line" id="p-0587" num="0631">norm.25_20.row6.txt:correlation: —0.306711</div>
</li> <li> <para-num num="[0632]"> </para-num> <div class="description-line" id="p-0588" num="0632">norm.25_20.row7.txt:correlation: —0.590045</div>
</li> <li> <para-num num="[0633]"> </para-num> <div class="description-line" id="p-0589" num="0633">norm.25_20.row8.txt:correlation: —0.327281</div>
</li> <li> <para-num num="[0634]"> </para-num> <div class="description-line" id="p-0590" num="0634">norm.25_26.row1.txt:correlation: 0.513396</div>
</li> <li> <para-num num="[0635]"> </para-num> <div class="description-line" id="p-0591" num="0635">norm.25_26.row2.txt:correlation: 0.135664</div>
</li> <li> <para-num num="[0636]"> </para-num> <div class="description-line" id="p-0592" num="0636">norm.25_26.row3.txt:correlation: 0.493248</div>
</li> <li> <para-num num="[0637]"> </para-num> <div class="description-line" id="p-0593" num="0637">norm.25_26.row5.txt:correlation: 0.592195</div>
</li> <li> <para-num num="[0638]"> </para-num> <div class="description-line" id="p-0594" num="0638">norm.25_26.row6.txt:correlation: 0.649929</div>
</li> <li> <para-num num="[0639]"> </para-num> <div class="description-line" id="p-0595" num="0639">norm.25_26.row7.txt:correlation: 0.638488</div>
</li> <li> <para-num num="[0640]"> </para-num> <div class="description-line" id="p-0596" num="0640">norm.25_26.row8.txt:correlation: 0.592366</div>
</li> <li> <para-num num="[0641]"> </para-num> <div class="description-line" id="p-0597" num="0641">norm.25_27.row1.txt:correlation: 0.435686</div>
</li> <li> <para-num num="[0642]"> </para-num> <div class="description-line" id="p-0598" num="0642">norm.25_27.row2.txt:correlation: −0.043748</div>
</li> <li> <para-num num="[0643]"> </para-num> <div class="description-line" id="p-0599" num="0643">norm.25_27.row3.txt:correlation: 0.249897</div>
</li> <li> <para-num num="[0644]"> </para-num> <div class="description-line" id="p-0600" num="0644">norm.25_27.row5.txt:correlation: 0.615696</div>
</li> <li> <para-num num="[0645]"> </para-num> <div class="description-line" id="p-0601" num="0645">norm.25_27.row6.txt:correlation: 0.591814</div>
</li> <li> <para-num num="[0646]"> </para-num> <div class="description-line" id="p-0602" num="0646">norm.25_27.row7.txt:correlation: 0.550319</div>
</li> <li> <para-num num="[0647]"> </para-num> <div class="description-line" id="p-0603" num="0647">norm.25_27.row8.txt:correlation: 0.433558</div>
</li> <li> <para-num num="[0648]"> </para-num> <div class="description-line" id="p-0604" num="0648">norm.29_13.row1.txt:correlation: −0.351870</div>
</li> <li> <para-num num="[0649]"> </para-num> <div class="description-line" id="p-0605" num="0649">norm.29_13.row2.txt:correlation: 0.144392</div>
</li> <li> <para-num num="[0650]"> </para-num> <div class="description-line" id="p-0606" num="0650">norm.29_13.row3.txt:correlation: 0.057765</div>
</li> <li> <para-num num="[0651]"> </para-num> <div class="description-line" id="p-0607" num="0651">norm.29_13.row5.txt:correlation: 0.393736</div>
</li> <li> <para-num num="[0652]"> </para-num> <div class="description-line" id="p-0608" num="0652">norm.29_13.row6.txt:correlation: 0.031146</div>
</li> <li> <para-num num="[0653]"> </para-num> <div class="description-line" id="p-0609" num="0653">norm.29_13.row7.txt:correlation: 0.043738</div>
</li> <li> <para-num num="[0654]"> </para-num> <div class="description-line" id="p-0610" num="0654">norm.29_13.row8.txt:correlation: 0.382760</div>
</li> <li> <para-num num="[0655]"> </para-num> <div class="description-line" id="p-0611" num="0655">norm.29_20.row1.txt:correlation: −0.092086</div>
</li> <li> <para-num num="[0656]"> </para-num> <div class="description-line" id="p-0612" num="0656">norm.29_20.row2.txt:correlation: 0.040947</div>
</li> <li> <para-num num="[0657]"> </para-num> <div class="description-line" id="p-0613" num="0657">norm.29_20.row3.txt:correlation: 0.265565</div>
</li> <li> <para-num num="[0658]"> </para-num> <div class="description-line" id="p-0614" num="0658">norm.29_20.row5.txt:correlation: 0.589432</div>
</li> <li> <para-num num="[0659]"> </para-num> <div class="description-line" id="p-0615" num="0659">norm.29_20.row6.txt:correlation: 0.117522</div>
</li> <li> <para-num num="[0660]"> </para-num> <div class="description-line" id="p-0616" num="0660">norm.29_20.row7.txt:correlation: 0.386742</div>
</li> <li> <para-num num="[0661]"> </para-num> <div class="description-line" id="p-0617" num="0661">norm.29_20.row8.txt:correlation: 0.557595</div>
</li> <li> <para-num num="[0662]"> </para-num> <div class="description-line" id="p-0618" num="0662">norm.29_26.row1.txt:correlation: −0.218187</div>
</li> <li> <para-num num="[0663]"> </para-num> <div class="description-line" id="p-0619" num="0663">norm.29_26.row2.txt:correlation: 0.578199</div>
</li> <li> <para-num num="[0664]"> </para-num> <div class="description-line" id="p-0620" num="0664">norm.29_26.row3.txt:correlation: 0.549576</div>
</li> <li> <para-num num="[0665]"> </para-num> <div class="description-line" id="p-0621" num="0665">norm.29_26.row5.txt:correlation: 0.131541</div>
</li> <li> <para-num num="[0666]"> </para-num> <div class="description-line" id="p-0622" num="0666">norm.29_26.row6.txt:correlation: 0.271451</div>
</li> <li> <para-num num="[0667]"> </para-num> <div class="description-line" id="p-0623" num="0667">norm.29_26.row7.txt:correlation: 0.429337</div>
</li> <li> <para-num num="[0668]"> </para-num> <div class="description-line" id="p-0624" num="0668">norm.29_26.row8.txt:correlation: 0.189393</div>
</li> <li> <para-num num="[0669]"> </para-num> <div class="description-line" id="p-0625" num="0669">norm.29_27.row1.txt:correlation: −0.246045</div>
</li> <li> <para-num num="[0670]"> </para-num> <div class="description-line" id="p-0626" num="0670">norm.29_27.row2.txt:correlation: 0.449658</div>
</li> <li> <para-num num="[0671]"> </para-num> <div class="description-line" id="p-0627" num="0671">norm.29_27.row3.txt:correlation: 0.425474</div>
</li> <li> <para-num num="[0672]"> </para-num> <div class="description-line" id="p-0628" num="0672">norm.29_27.row5.txt:correlation: 0.376062</div>
</li> <li> <para-num num="[0673]"> </para-num> <div class="description-line" id="p-0629" num="0673">norm.29_27.row6.txt:correlation: −0.444794</div>
</li> <li> <para-num num="[0674]"> </para-num> <div class="description-line" id="p-0630" num="0674">norm.29_27.row7.txt:correlation: −0.125790</div>
</li> <li> <para-num num="[0675]"> </para-num> <div class="description-line" id="p-0631" num="0675">norm.29_27.row8.txt:correlation: 0.399004</div>
</li> <li> <para-num num="[0676]"> </para-num> <div class="description-line" id="p-0632" num="0676">In a second round of optimizations, the probe library was expanded to target 24 genes that range from the most abundant, to non-abundant in the sample. As illustrated in the plots exemplified in <figref idrefs="DRAWINGS">FIGS. 5-8</figref>, correlation values for individual dilution levels (rows 1-8) were significantly higher than initial experiments. Correlation levels may be determined for a probe library and various dilution conditions of that probe library. For example, <figref idrefs="DRAWINGS">FIG. 5</figref> is plot of expected counts versus observed counts for a predetermined set of probes under a range of dilution criteria for the protocol Purified cDNA High. In the experiment exemplified, 24 different genes from most abundant to not present were measured and correlation values determined within a dilution level, where a correlation of closer to 1 indicates a higher degree of correlation. In the first dilution level (row 1), probes for each of 24 genes were included at approximately equal abundance. In the second dilution level (row 2) the overall concentration of probes was reduced by a factor of 2. Each row down to row 8 was successively diluted, so that the concentration of probes in row 8 was approximately 100-fold lower than in row 1. Thus, the 8 correlation values in <figref idrefs="DRAWINGS">FIG. 5</figref> represent the sensitivity of the assay across a 100-fold change in the concentration of gene probes.</div>
</li> <li> <para-num num="[0677]"> </para-num> <div class="description-line" id="p-0633" num="0677">Similarly, <figref idrefs="DRAWINGS">FIG. 27</figref> is plot of expected counts versus observed counts for a predetermined set of probes under a range of dilution criteria for unPurified cDNA High.</div>
</li> <li> <para-num num="[0678]"> </para-num> <div class="description-line" id="p-0634" num="0678"> <figref idrefs="DRAWINGS">FIG. 28</figref> is plot of expected counts versus observed counts for a predetermined set of probes under a range of dilution criteria for Purified cDNA Low.</div>
</li> <li> <para-num num="[0679]"> </para-num> <div class="description-line" id="p-0635" num="0679"> <figref idrefs="DRAWINGS">FIG. 29</figref> is plot of expected counts versus observed counts for a predetermined set of probes under a range of dilution criteria for unPurified cDNA Low.</div>
</li> <li> <para-num num="[0680]"> </para-num> <div class="description-line" id="p-0636" num="0680">The same data can be used to computationally simulate the random setting, where counts are aggregated across all genes from randomly chosen rows, as described above for initial optimizations. Results for these simulations are presented in <figref idrefs="DRAWINGS">FIGS. 30-33</figref>.</div>
</li> <li> <para-num num="[0681]"> </para-num> <div class="description-line" id="p-0637" num="0681">The population of cells with a plurality of genomic sequence or perturbation conditions involves a plurality of cells and perturbations to be tested and measurements sampled to obtain meaningful data and to infer appropriate circuits. The number of genes perturbed, and how many are perturbed simultaneously (the order of the perturbation, pairs, triplets, etc.) varies. In a tissue with n cell types, the rarest present in m %, how many cells X do you need to sequence so that you have at least Y of the rarest subtype.</div>
</li> <li> <para-num num="[0682]"> </para-num> <div class="description-line" id="p-0638" num="0682">For example, ˜500 cells ensures ≥95% chance of including ≥10 of each type, based on the following calculation (<figref idrefs="DRAWINGS">FIG. 34</figref>). Assume the most conservative scenario that of M cell subtypes (for example, 12), all but one having the lowest predicted proportion (for example, p<sub>min</sub>=5%). Assuming that the Central Limit Theorem holds (a reasonable assumption when solving to detect at least 10 cells of each type) the number of cells of each type i, termed T<sub>i, </sub>will distribute as E[T<sub>i</sub>]=N*p<sub>min </sub>, STDV[T<sub>i</sub>]=√(N*p<sub>min</sub>*(1-p<sub>min</sub>)). The minimal N (total number of cells to profile) can be solved such that all (m−1) subtypes have at least n cells (the last, majority, subtype easily clears this threshold since its proportion is much higher). Applicants confirmed with simulation that the strategy conservatively holds in practice even for n&lt;10, and take a margin of additional (conservative) error, to allow for subsequent failed RNA-Seq experiments (&lt;20-30%, depending on protocol).</div>
</li> <li> <para-num num="[0683]"> </para-num> <div class="description-line" id="p-0639" num="0683">Amplification may involve thermocycling or isothermal amplification (such as through the methods RPA or LAMP). Cross-linking may involve overlap-extension PCR or use of ligase to associate multiple amplification products with each other.</div>
</li> <li> <para-num num="[0684]"> </para-num> <div class="description-line" id="p-0640" num="0684">For purpose of this invention, amplification means any method employing a primer and a polymerase capable of replicating a target sequence with reasonable fidelity. Amplification may be carried out by natural or recombinant DNA polymerases such as TaqGold™, T7 DNA polymerase, Klenow fragment of <i>E.coli </i>DNA polymerase, and reverse transcriptase. A preferred amplification method is PCR. In particular, the isolated RNA can be subjected to a reverse transcription assay that is coupled with a quantitative polymerase chain reaction (RT-PCR) in order to quantify the expression level of a sequence associated with a signaling biochemical pathway.</div>
</li> <li> <para-num num="[0685]"> </para-num> <div class="description-line" id="p-0641" num="0685">Detection of the gene expression level can be conducted in real time in an amplification assay. In one aspect, the amplified products can be directly visualized with fluorescent DNA-binding agents including but not limited to DNA intercalators and DNA groove binders. Because the amount of the intercalators incorporated into the double-stranded DNA molecules is typically proportional to the amount of the amplified DNA products, one can conveniently determine the amount of the amplified products by quantifying the fluorescence of the intercalated dye using conventional optical systems in the art. DNA-binding dye suitable for this application include SYBR green, SYBR blue, DAPI, propidium iodine, Hoeste, SYBR gold, ethidium bromide, acridines, proflavine, acridine orange, acriflavine, fluorcoumanin, ellipticine, daunomycin, chloroquine, distamycin D, chromomycin, homidium, mithramycin, ruthenium polypyridyls, anthramycin, and the like.</div>
</li> <li> <para-num num="[0686]"> </para-num> <div class="description-line" id="p-0642" num="0686">In another aspect, other fluorescent labels such as sequence specific probes can be employed in the amplification reaction to facilitate the detection and quantification of the amplified products. Probe-based quantitative amplification relies on the sequence-specific detection of a desired amplified product. It utilizes fluorescent, target-specific probes (e.g., TaqMan® probes) resulting in increased specificity and sensitivity. Methods for performing probe-based quantitative amplification are well established in the art and are taught in U.S. Pat. No. 5,210,015.</div>
</li> <li> <para-num num="[0687]"> </para-num> <div class="description-line" id="p-0643" num="0687">Sequencing may be performed on any high-throughput platform with read-length (either single- or paired-end) sufficient to cover both template and cross-linking event UID's. Methods of sequencing oligonucleotides and nucleic acids are well known in the art (see, e.g., WO93/23564, WO98/28440 and WO98/13523; U.S. Pat. Nos. 5,525,464; 5,202,231; 5,695,940; 4,971,903; 5,902,723; 5,795,782; 5,547,839 and 5,403,708; Sanger et al., Proc. Natl. Acad. Sci. USA 74:5463 (1977); Drmanac et al., Genomics 4:114 (1989); Koster et al., Nature Biotechnology 14:1123 (1996); Hyman, Anal. Biochem. 174:423 (1988); Rosenthal, International Patent Application Publication 761107 (1989); Metzker et al., Nucl. Acids Res. 22:4259 (1994); Jones, Biotechniques 22:938 (1997); Ronaghi et al., Anal. Biochem. 242:84 (1996); Ronaghi et al., Science 281:363 (1998); Nyren et al., Anal. Biochem. 151:504 (1985); Canard and Arzumanov, Gene 11:1 (1994); Dyatkina and Arzumanov, Nucleic Acids Symp Ser 18:117 (1987); Johnson et al., Anal. Biochem.136:192 (1984); and Elgen and Rigler, Proc. Natl. Acad. Sci. USA 91(13):5740 (1994), all of which are expressly incorporated by reference).</div>
</li> <li> <para-num num="[0688]"> </para-num> <div class="description-line" id="p-0644" num="0688">The present invention may be applied to (1) single-cell transcriptomics: cDNA synthesized from mRNA is barcoded and cross-linked during in situ amplification, (2) single-cell proteomics: cDNA or DNA synthesized from RNA- or DNA-tagged antibodies of one or multiple specificities maps the abundance and distributions of different protein-antigens and (3) whole-tissue transcriptomic/proteomic mapping (molecular microscopy or VIPUR microscopy): using the frequency of cross-contamination between cells to determine their physical proximity, and via applications (1) single-cell transcriptomics and (2) single-cell proteomics, determining the global spatial distribution of mRNA, protein, or other biomolecules in a biological sample. This may be used, for example, to screen for anti-cancer/pathogen immunoglobulins (by analyzing co-localization of B-cells and T-cells within affected tissue) for immunotherapy.</div>
</li> <li> <para-num num="[0689]"> </para-num> <div class="description-line" id="p-0645" num="0689">The terms “polynucleotide”, “nucleotide”, “nucleotide sequence”, “nucleic acid” and “oligonucleotide” are used interchangeably. They refer to a polymeric form of nucleotides of any length, either deoxyribonucleotides or ribonucleotides, or analogs thereof. Polynucleotides may have any three dimensional structure, and may perform any function, known or unknown. The following are non-limiting examples of polynucleotides: coding or non-coding regions of a gene or gene fragment, loci (locus) defined from linkage analysis, exons, introns, messenger RNA (mRNA), transfer RNA, ribosomal RNA, short interfering RNA (siRNA), short-hairpin RNA (shRNA), micro-RNA (miRNA), ribozymes, cDNA, recombinant polynucleotides, branched polynucleotides, plasmids, vectors, isolated DNA of any sequence, isolated RNA of any sequence, nucleic acid probes, and primers. The term also encompasses nucleic-acid-like structures with synthetic backbones, see, e.g., Eckstein, 1991; Baserga et al., 1992; Milligan, 1993; WO 97/03211; WO 96/39154; Mata, 1997; Strauss-Soukup, 1997; and Samstag, 1996. A polynucleotide may comprise one or more modified nucleotides, such as methylated nucleotides and nucleotide analogs. If present, modifications to the nucleotide structure may be imparted before or after assembly of the polymer. The sequence of nucleotides may be interrupted by non-nucleotide components. A polynucleotide may be further modified after polymerization, such as by conjugation with a labeling component.</div>
</li> <li> <para-num num="[0690]"> </para-num> <div class="description-line" id="p-0646" num="0690">As used herein the term “variant” should be taken to mean the exhibition of qualities that differ, such as, but not limited to, genetic variations including SNPs, insertion deletion events, and the like.</div>
</li> <li> <para-num num="[0691]"> </para-num> <div class="description-line" id="p-0647" num="0691">The terms “non-naturally occurring” or “engineered” are used interchangeably and indicate the involvement of the hand of man. The terms, when referring to nucleic acid molecules or polypeptides mean that the nucleic acid molecule or the polypeptide is at least substantially free from at least one other component with which they are naturally associated in nature and as found in nature.</div>
</li> <li> <para-num num="[0692]"> </para-num> <div class="description-line" id="p-0648" num="0692">“Complementarity” refers to the ability of a nucleic acid to form hydrogen bond(s) with another nucleic acid sequence by either traditional Watson-Crick or other non-traditional types. A percent complementarity indicates the percentage of residues in a nucleic acid molecule which can form hydrogen bonds (e.g., Watson-Crick base pairing) with a second nucleic acid sequence (e.g., 5, 6, 7, 8, 9, 10 out of 10 being 50%, 60%, 70%, 80%, 90%, and 100% complementary). “Perfectly complementary” means that all the contiguous residues of a nucleic acid sequence hydrogen bond with the same number of contiguous residues in a second nucleic acid sequence. “Substantially complementary” as used herein refers to a degree of complementarity that is at least 60%, 65%, 70%, 75%, 80%, 85%, 90%, 95%, 97%, 98%, 99%, or 100% over a region of 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22, 23, 24, 25, 30, 35, 40, 45, 50, or more nucleotides, or refers to two nucleic acids that hybridize under stringent conditions.</div>
</li> <li> <para-num num="[0693]"> </para-num> <div class="description-line" id="p-0649" num="0693">As used herein, “stringent conditions” for hybridization refer to conditions under which a nucleic acid having complementarity to a target sequence predominantly hybridizes with the target sequence, and substantially does not hybridize to non-target sequences. Stringent conditions are generally sequence-dependent, and vary depending on a number of factors. In general, the longer the sequence, the higher the temperature at which the sequence specifically hybridizes to its target sequence. Non-limiting examples of stringent conditions are described in detail in Tijssen (1993), Laboratory Techniques In Biochemistry And Molecular Biology-Hybridization With Nucleic Acid Probes Part I, Second Chapter “Overview of principles of hybridization and the strategy of nucleic acid probe assay”, Elsevier, N.Y.</div>
</li> <li> <para-num num="[0694]"> </para-num> <div class="description-line" id="p-0650" num="0694">“Hybridization” refers to a reaction in which one or more polynucleotides react to form a complex that is stabilized via hydrogen bonding between the bases of the nucleotide residues. The hydrogen bonding may occur by Watson Crick base pairing, Hoogstein binding, or in any other sequence specific manner. The complex may comprise two strands forming a duplex structure, three or more strands forming a multi stranded complex, a single self-hybridizing strand, or any combination of these. A hybridization reaction may constitute a step in a more extensive process, such as the initiation of PCR, or the cleavage of a polynucleotide by an enzyme. A sequence capable of hybridizing with a given sequence is referred to as the “complement” of the given sequence.</div>
</li> <li> <para-num num="[0695]"> </para-num> <div class="description-line" id="p-0651" num="0695">As used herein, the term “genomic locus” or “locus” (plural loci) is the specific location of a gene or DNA sequence on a chromosome. A “gene” refers to stretches of DNA or RNA that encode a polypeptide or an RNA chain that has functional role to play in an organism and hence is the molecular unit of heredity in living organisms. For the purpose of this invention it may be considered that genes include regions which regulate the production of the gene product, whether or not such regulatory sequences are adjacent to coding and/or transcribed sequences. Accordingly, a gene includes, but is not necessarily limited to, promoter sequences, terminators, translational regulatory sequences such as ribosome binding sites and internal ribosome entry sites, enhancers, silencers, insulators, boundary elements, replication origins, matrix attachment sites and locus control regions.</div>
</li> <li> <para-num num="[0696]"> </para-num> <div class="description-line" id="p-0652" num="0696">As used herein, “expression of a genomic locus” or “gene expression” is the process by which information from a gene is used in the synthesis of a functional gene product. The products of gene expression are often proteins, but in non-protein coding genes such as rRNA genes or tRNA genes, the product is functional RNA. The process of gene expression is used by all known life - eukaryotes (including multicellular organisms), prokaryotes (bacteria and archaea) and viruses to generate functional products to survive. As used herein “expression” of a gene or nucleic acid encompasses not only cellular gene expression, but also the transcription and translation of nucleic acid(s) in cloning systems and in any other context. As used herein, “expression” also refers to the process by which a polynucleotide is transcribed from a DNA template (such as into and mRNA or other RNA transcript) and/or the process by which a transcribed mRNA is subsequently translated into peptides, polypeptides, or proteins. Transcripts and encoded polypeptides may be collectively referred to as “gene product.” If the polynucleotide is derived from genomic DNA, expression may include splicing of the mRNA in a eukaryotic cell.</div>
</li> <li> <para-num num="[0697]"> </para-num> <div class="description-line" id="p-0653" num="0697">As described in aspects of the invention, sequence identity is related to sequence homology. Homology comparisons may be conducted by eye, or more usually, with the aid of readily available sequence comparison programs. These commercially available computer programs may calculate percent (%) homology between two or more sequences and may also calculate the sequence identity shared by two or more amino acid or nucleic acid sequences.</div>
</li> <li> <para-num num="[0698]"> </para-num> <div class="description-line" id="p-0654" num="0698">Sequence homologies may be generated by any of a number of computer programs known in the art, for example BLAST or FASTA, etc. A suitable computer program for carrying out such an alignment is the GCG Wisconsin Bestfit package (University of Wisconsin, U.S.A; Devereux et al., 1984, Nucleic Acids Research 12:387). Examples of other software than may perform sequence comparisons include, but are not limited to, the BLAST package (see Ausubel et al., 1999 ibid—Chapter 18), FASTA (Atschul et al., 1990, J. Mol. Biol., 403-410) and the GENEWORKS suite of comparison tools. Both BLAST and FASTA are available for offline and online searching (see Ausubel et al., 1999 ibid, pages 7-58 to 7-60). However it is preferred to use the GCG Bestfit program. % homology may be calculated over contiguous sequences, i.e., one sequence is aligned with the other sequence and each amino acid or nucleotide in one sequence is directly compared with the corresponding amino acid or nucleotide in the other sequence, one residue at a time. This is called an “ungapped” alignment. Typically, such ungapped alignments are performed only over a relatively short number of residues. Although this is a very simple and consistent method, it fails to take into consideration that, for example, in an otherwise identical pair of sequences, one insertion or deletion may cause the following amino acid residues to be put out of alignment, thus potentially resulting in a large reduction in % homology when a global alignment is performed. Consequently, most sequence comparison methods are designed to produce optimal alignments that take into consideration possible insertions and deletions without unduly penalizing the overall homology or identity score. This is achieved by inserting “gaps” in the sequence alignment to try to maximize local homology or identity. However, these more complex methods assign “gap penalties” to each gap that occurs in the alignment so that, for the same number of identical amino acids, a sequence alignment with as few gaps as possible—reflecting higher relatedness between the two compared sequences—may achieve a higher score than one with many gaps. “Affinity gap costs” are typically used that charge a relatively high cost for the existence of a gap and a smaller penalty for each subsequent residue in the gap. This is the most commonly used gap scoring system. High gap penalties may, of course, produce optimized alignments with fewer gaps. Most alignment programs allow the gap penalties to be modified. However, it is preferred to use the default values when using such software for sequence comparisons. For example, when using the GCG Wisconsin Bestfit package the default gap penalty for amino acid sequences is −12 for a gap and —4 for each extension. Calculation of maximum % homology therefore first requires the production of an optimal alignment, taking into consideration gap penalties. A suitable computer program for carrying out such an alignment is the GCG Wisconsin Bestfit package (Devereux et al., 1984 Nuc. Acids Research 12 p387). Examples of other software than may perform sequence comparisons include, but are not limited to, the BLAST package (see Ausubel et al., 1999 Short Protocols in Molecular Biology, 4th Ed.—Chapter 18), FASTA (Altschul et al., 1990 J. Mol. Biol. 403-410) and the GENEWORKS suite of comparison tools. Both BLAST and FASTA are available for offline and online searching (see Ausubel et al., 1999, Short Protocols in Molecular Biology, pages 7-58 to 7-60). However, for some applications, it is preferred to use the GCG Bestfit program. A new tool, called BLAST 2 Sequences is also available for comparing protein and nucleotide sequences (see FEMS Microbiol Lett. 1999 174(2): 247-50; FEMS Microbiol Lett. 1999 177(1): 187-8 and the website of the National Center for Biotechnology information at the website of the National Institutes for Health). Although the final % homology may be measured in terms of identity, the alignment process itself is typically not based on an all-or-nothing pair comparison. Instead, a scaled similarity score matrix is generally used that assigns scores to each pair-wise comparison based on chemical similarity or evolutionary distance. An example of such a matrix commonly used is the BLOSUM62 matrix—the default matrix for the BLAST suite of programs. GCG Wisconsin programs generally use either the public default values or a custom symbol comparison table, if supplied (see user manual for further details). For some applications, it is preferred to use the public default values for the GCG package, or in the case of other software, the default matrix, such as BLOSUM62.</div>
</li> <li> <para-num num="[0699]"> </para-num> <div class="description-line" id="p-0655" num="0699">Alternatively, percentage homologies may be calculated using the multiple alignment feature in DNASIS™ (Hitachi Software), based on an algorithm, analogous to CLUSTAL (Higgins DG &amp; Sharp PM (1988), Gene 73(1), 237-244). Once the software has produced an optimal alignment, it is possible to calculate % homology, preferably % sequence identity. The software typically does this as part of the sequence comparison and generates a numerical result.</div>
</li> <li> <para-num num="[0700]"> </para-num> <div class="description-line" id="p-0656" num="0700">Embodiments of the invention include sequences (both polynucleotide or polypeptide) which may comprise homologous substitution (substitution and replacement are both used herein to mean the interchange of an existing amino acid residue or nucleotide, with an alternative residue or nucleotide) that may occur i.e., like-for-like substitution in the case of amino acids such as basic for basic, acidic for acidic, polar for polar, etc. Non-homologous substitution may also occur i.e., from one class of residue to another or alternatively involving the inclusion of unnatural amino acids such as ornithine (hereinafter referred to as Z), diaminobutyric acid ornithine (hereinafter referred to as B), norleucine ornithine (hereinafter referred to as O), pyriylalanine, thienylalanine, naphthylalanine and phenylglycine.</div>
</li> <li> <para-num num="[0701]"> </para-num> <div class="description-line" id="p-0657" num="0701">Hybridization can be performed under conditions of various stringency. Suitable hybridization conditions for the practice of the present invention are such that the recognition interaction between the probe and sequences associated with a signaling biochemical pathway is both sufficiently specific and sufficiently stable. Conditions that increase the stringency of a hybridization reaction are widely known and published in the art. See, for example, (Sambrook, et al., (1989); Nonradioactive In Situ Hybridization Application Manual, Boehringer Mannheim, second edition). The hybridization assay can be formed using probes immobilized on any solid support, including but are not limited to nitrocellulose, glass, silicon, and a variety of gene arrays. A preferred hybridization assay is conducted on high-density gene chips as described in U.S. Pat. No. 5,445,934.</div>
</li> <li> <para-num num="[0702]"> </para-num> <div class="description-line" id="p-0658" num="0702">For a convenient detection of the probe-target complexes formed during the hybridization assay, the nucleotide probes are conjugated to a detectable label. Detectable labels suitable for use in the present invention include any composition detectable by photochemical, biochemical, spectroscopic, immunochemical, electrical, optical or chemical means. A wide variety of appropriate detectable labels are known in the art, which include fluorescent or chemiluminescent labels, radioactive isotope labels, enzymatic or other ligands. In preferred embodiments, one will likely desire to employ a fluorescent label or an enzyme tag, such as digoxigenin, B-galactosidase, urease, alkaline phosphatase or peroxidase, avidin/biotin complex.</div>
</li> <li> <para-num num="[0703]"> </para-num> <div class="description-line" id="p-0659" num="0703">The detection methods used to detect or quantify the hybridization intensity will typically depend upon the label selected above. For example, radiolabels may be detected using photographic film or a phosphoimager. Fluorescent markers may be detected and quantified using a photodetector to detect emitted light. Enzymatic labels are typically detected by providing the enzyme with a substrate and measuring the reaction product produced by the action of the enzyme on the substrate; and finally colorimetric labels are detected by simply visualizing the colored label.</div>
</li> <li> <para-num num="[0704]"> </para-num> <div class="description-line" id="p-0660" num="0704">Examples of the labeling substance which may be employed include labeling substances known to those skilled in the art, such as fluorescent dyes, enzymes, coenzymes, chemiluminescent substances, and radioactive substances. Specific examples include radioisotopes (e.g., 32P, 14C, 1251, 3H, and 1311), fluorescein, rhodamine, dansyl chloride, umbelliferone, luciferase, peroxidase, alkaline phosphatase, β-galactosidase, β-glucosidase, horseradish peroxidase, glucoamylase, lysozyme, saccharide oxidase, microperoxidase, biotin, and ruthenium. In the case where biotin is employed as a labeling substance, preferably, after addition of a biotin-labeled antibody, streptavidin bound to an enzyme (e.g., peroxidase) is further added.</div>
</li> <li> <para-num num="[0705]"> </para-num> <div class="description-line" id="p-0661" num="0705">Advantageously, the label is a fluorescent label. Examples of fluorescent labels include, but are not limited to, Atto dyes, 4-acetamido-4′-isothiocyanatostilbene-2,2′disulfonic acid; acridine and derivatives: acridine, acridine isothiocyanate; 5-(2′-aminoethyl)aminonaphthalene-1 -sulfonic acid (EDANS); 4-amino-N-[3-vinylsulfonyl)phenyl]naphthalimide-3 ,5 disulfonate; N-(4-anilino-1-naphthyl)maleimide; anthranilamide; BODIPY; Brilliant Yellow; coumarin and derivatives; coumarin, 7-amino-4-methylcoumarin (AMC, Coumarin 120), 7-amino-4-trifluoromethylcouluarin (Coumaran 151); cyanine dyes; cyanosine; 4′,6-diaminidino-2-phenylindole (DAPI); 5′5″-dibromopyrogallol-sulfonaphthalein (Bromopyrogallol Red); 7-diethylamino-3 -(4′-isothiocyanatophenyl)-4-methylcoumarin; diethylenetriamine pentaacetate; 4,4′-diisothiocyanatodihydro-stilbene-2,2′-disulfonic acid; 4,4′-diisothiocyanatostilbene-2,2′-disulfonic acid; 5-[dimethylamino]naphthalene-1-sulfonyl chloride (DNS, dansylchloride); 4-dimethylaminophenylazophenyl-4′-isothiocyanate (DABITC); eosin and derivatives; eosin, eosin isothiocyanate, erythrosin and derivatives; erythrosin B, erythrosin, isothiocyanate; ethidium; fluorescein and derivatives; 5-carboxyfluorescein (FAM), 5-(4,6-dichlorotriazin-2-yl)aminofluorescein (DTAF), 2′,7′-dimethoxy-4′5′-dichloro-6-carboxyfluorescein, fluorescein, fluorescein isothiocyanate, QFITC, (XRITC); fluorescamine; IR144; IR1446; Malachite Green isothiocyanate; 4-methylumbelliferoneortho cresolphthalein; nitrotyrosine; pararosaniline; Phenol Red; B-phycoerythrin; o-phthaldialdehyde; pyrene and derivatives: pyrene, pyrene butyrate, succinimidyl 1-pyrene; butyrate quantum dots; Reactive Red 4 (Cibacron.™. Brilliant Red 3B-A) rhodamine and derivatives: 6-carboxy-X-rhodamine (ROX), 6-carboxyrhodamine (R6G), lissamine rhodamine B sulfonyl chloride rhodamine (Rhod), rhodamine B, rhodamine 123, rhodamine X isothiocyanate, sulforhodamine B, sulforhodamine 101, sulfonyl chloride derivative of sulforhodamine 101 (Texas Red); N,N,N′,N′ tetramethyl-6-carboxyrhodamine (TAMRA); tetramethyl rhodamine; tetramethyl rhodamine isothiocyanate (TRITC); riboflavin; rosolic acid; terbium chelate derivatives; Cy3; Cy5; Cy5.5; Cy7; IRD 700; IRD 800; La Jolta Blue; phthalo cyanine; and naphthalo cyanine</div>
</li> <li> <para-num num="[0706]"> </para-num> <div class="description-line" id="p-0662" num="0706">The fluorescent label may be a fluorescent protein, such as blue fluorescent protein, cyan fluorescent protein, green fluorescent protein, red fluorescent protein, yellow fluorescent protein or any photoconvertible protein. Colormetric labeling, bioluminescent labeling and/or chemiluminescent labeling may further accomplish labeling. Labeling further may include energy transfer between molecules in the hybridization complex by perturbation analysis, quenching, or electron transport between donor and acceptor molecules, the latter of which may be facilitated by double stranded match hybridization complexes. The fluorescent label may be a perylene or a terrylen. In the alternative, the fluorescent label may be a fluorescent bar code.</div>
</li> <li> <para-num num="[0707]"> </para-num> <div class="description-line" id="p-0663" num="0707">In an advantageous embodiment, the label may be light sensitive, wherein the label is light-activated and/or light cleaves the one or more linkers to release the molecular cargo. The light-activated molecular cargo may be a major light-harvesting complex (LHCII). In another embodiment, the fluorescent label may induce free radical formation.</div>
</li> <li> <para-num num="[0708]"> </para-num> <div class="description-line" id="p-0664" num="0708">In an advantageous embodiment, agents may be uniquely labeled in a dynamic manner (see, e.g., international patent application serial no. PCT/US2013/61182 filed Sep. 23, 2012). The unique labels are, at least in part, nucleic acid in nature, and may be generated by sequentially attaching two or more detectable oligonucleotide tags to each other and each unique label may be associated with a separate agent. A detectable oligonucleotide tag may be an oligonucleotide that may be detected by sequencing of its nucleotide sequence and/or by detecting non-nucleic acid detectable moieties to which it may be attached.</div>
</li> <li> <para-num num="[0709]"> </para-num> <div class="description-line" id="p-0665" num="0709">The oligonucleotide tags may be detectable by virtue of their nucleotide sequence, or by virtue of a non-nucleic acid detectable moiety that is attached to the oligonucleotide such as but not limited to a fluorophore, or by virtue of a combination of their nucleotide sequence and the nonnucleic acid detectable moiety.</div>
</li> <li> <para-num num="[0710]"> </para-num> <div class="description-line" id="p-0666" num="0710">In some embodiments, a detectable oligonucleotide tag may comprise one or more nonoligonucleotide detectable moieties. Examples of detectable moieties may include, but are not limited to, fluorophores, microparticles including quantum dots (Empodocles, et al., Nature 399:126-130, 1999), gold nanoparticles (Reichert et al., Anal. Chem. 72:6025-6029, 2000), biotin, DNP (dinitrophenyl), fucose, digoxigenin, haptens, and other detectable moieties known to those skilled in the art. In some embodiments, the detectable moieties may be quantum dots. Methods for detecting such moieties are described herein and/or are known in the art.</div>
</li> <li> <para-num num="[0711]"> </para-num> <div class="description-line" id="p-0667" num="0711">Thus, detectable oligonucleotide tags may be, but are not limited to, oligonucleotides which may comprise unique nucleotide sequences, oligonucleotides which may comprise detectable moieties, and oligonucleotides which may comprise both unique nucleotide sequences and detectable moieties.</div>
</li> <li> <para-num num="[0712]"> </para-num> <div class="description-line" id="p-0668" num="0712">A unique label may be produced by sequentially attaching two or more detectable oligonucleotide tags to each other. The detectable tags may be present or provided in a plurality of detectable tags. The same or a different plurality of tags may be used as the source of each detectable tag may be part of a unique label. In other words, a plurality of tags may be subdivided into subsets and single subsets may be used as the source for each tag.</div>
</li> <li> <para-num num="[0713]"> </para-num> <div class="description-line" id="p-0669" num="0713">In some embodiments, a detectable oligonucleotide tag may comprise one or more non-oligonucleotide detectable moieties. Examples of detectable moieties include, but are not limited to, fluorophores, microparticles including quantum dots (Empodocles, et al., Nature 399:126-130, 1999), gold nanoparticles (Reichert et al., Anal. Chem. 72:6025-6029, 2000), biotin, DNP (dinitrophenyl), fucose, digoxigenin, haptens, and other detectable moieties known to those skilled in the art. In some embodiments, the detectable moieties are quantum dots. Methods for detecting such moieties are described herein and/or are known in the art.</div>
</li> <li> <para-num num="[0714]"> </para-num> <div class="description-line" id="p-0670" num="0714">Thus, detectable oligonucleotide tags may be, but are not limited to, oligonucleotides which may comprise unique nucleotide sequences, oligonucleotides which may comprise detectable moieties, and oligonucleotides which may comprise both unique nucleotide sequences and detectable moieties.</div>
</li> <li> <para-num num="[0715]"> </para-num> <div class="description-line" id="p-0671" num="0715">A unique nucleotide sequence may be a nucleotide sequence that is different (and thus distinguishable) from the sequence of each detectable oligonucleotide tag in a plurality of detectable oligonucleotide tags. A unique nucleotide sequence may also be a nucleotide sequence that is different (and thus distinguishable) from the sequence of each detectable oligonucleotide tag in a first plurality of detectable oligonucleotide tags but identical to the sequence of at least one detectable oligonucleotide tag in a second plurality of detectable oligonucleotide tags. A unique sequence may differ from other sequences by multiple bases (or base pairs). The multiple bases may be contiguous or non-contiguous. Methods for obtaining nucleotide sequences (e.g., sequencing methods) are described herein and/or are known in the art.</div>
</li> <li> <para-num num="[0716]"> </para-num> <div class="description-line" id="p-0672" num="0716">In some embodiments, detectable oligonucleotide tags comprise one or more of a ligation sequence, a priming sequence, a capture sequence, and a unique sequence (optionally referred to herein as an index sequence). A ligation sequence is a sequence complementary to a second nucleotide sequence which allows for ligation of the detectable oligonucleotide tag to another entity which may comprise the second nucleotide sequence, e.g., another detectable oligonucleotide tag or an oligonucleotide adapter. A priming sequence is a sequence complementary to a primer, e.g., an oligonucleotide primer used for an amplification reaction such as but not limited to PCR. A capture sequence is a sequence capable of being bound by a capture entity. A capture entity may be an oligonucleotide which may comprise a nucleotide sequence complementary to a capture sequence, e.g. a second detectable oligonucleotide tag. A capture entity may also be any other entity capable of binding to the capture sequence, e.g. an antibody, hapten or peptide. An index sequence is a sequence which may comprise a unique nucleotide sequence and/or a detectable moiety as described above.</div>
</li> <li> <para-num num="[0717]"> </para-num> <div class="description-line" id="p-0673" num="0717">“Complementary” is a term which is used to indicate a sufficient degree of complementarity between two nucleotide sequences such that stable and specific binding occurs between one and preferably more bases (or nucleotides, as the terms are used interchangeably herein) of the two sequences. For example, if a nucleotide in a first nucleotide sequence is capable of hydrogen bonding with a nucleotide in second nucleotide sequence, then the bases are considered to be complementary to each other. Complete (i.e., 100%) complementarity between a first nucleotide sequence and a second nucleotide is preferable, but not required for ligation, priming, or capture sequences.</div>
</li> <li> <para-num num="[0718]"> </para-num> <div class="description-line" id="p-0674" num="0718">The present invention also relates to a computer system involved in carrying out the methods of the invention relating to both computations and sequencing.</div>
</li> <li> <para-num num="[0719]"> </para-num> <div class="description-line" id="p-0675" num="0719">A computer system (or digital device) may be used to receive, transmit, display and/or store results, analyze the results, and/or produce a report of the results and analysis. A computer system may be understood as a logical apparatus that can read instructions from media (e.g. software) and/or network port (e.g. from the internet), which can optionally be connected to a server having fixed media. A computer system may comprise one or more of a CPU, disk drives, input devices such as keyboard and/or mouse, and a display (e.g. a monitor). Data communication, such as transmission of instructions or reports, can be achieved through a communication medium to a server at a local or a remote location. The communication medium can include any means of transmitting and/or receiving data. For example, the communication medium can be a network connection, a wireless connection, or an internet connection. Such a connection can provide for communication over the World Wide Web. It is envisioned that data relating to the present invention can be transmitted over such networks or connections (or any other suitable means for transmitting information, including but not limited to mailing a physical report, such as a print-out) for reception and/or for review by a receiver. The receiver can be but is not limited to an individual, or electronic system (e.g. one or more computers, and/or one or more servers).</div>
</li> <li> <para-num num="[0720]"> </para-num> <div class="description-line" id="p-0676" num="0720">In some embodiments, the computer system may comprise one or more processors. Processors may be associated with one or more controllers, calculation units, and/or other units of a computer system, or implanted in firmware as desired. If implemented in software, the routines may be stored in any computer readable memory such as in RAM, ROM, flash memory, a magnetic disk, a laser disk, or other suitable storage medium. Likewise, this software may be delivered to a computing device via any known delivery method including, for example, over a communication channel such as a telephone line, the internet, a wireless connection, etc., or via a transportable medium, such as a computer readable disk, flash drive, etc. The various steps may be implemented as various blocks, operations, tools, modules and techniques which, in turn, may be implemented in hardware, firmware, software, or any combination of hardware, firmware, and/or software. When implemented in hardware, some or all of the blocks, operations, techniques, etc. may be implemented in, for example, a custom integrated circuit (IC), an application specific integrated circuit (ASIC), a field programmable logic array (FPGA), a programmable logic array (PLA), etc.</div>
</li> <li> <para-num num="[0721]"> </para-num> <div class="description-line" id="p-0677" num="0721">A client-server, relational database architecture can be used in embodiments of the invention. A client-server architecture is a network architecture in which each computer or process on the network is either a client or a server. Server computers are typically powerful computers dedicated to managing disk drives (file servers), printers (print servers), or network traffic (network servers). Client computers include PCs (personal computers) or workstations on which users run applications, as well as example output devices as disclosed herein. Client computers rely on server computers for resources, such as files, devices, and even processing power. In some embodiments of the invention, the server computer handles all of the database functionality. The client computer can have software that handles all the front-end data management and can also receive data input from users.</div>
</li> <li> <para-num num="[0722]"> </para-num> <div class="description-line" id="p-0678" num="0722">A machine readable medium which may comprise computer-executable code may take many forms, including but not limited to, a tangible storage medium, a carrier wave medium or physical transmission medium. Non-volatile storage media include, for example, optical or magnetic disks, such as any of the storage devices in any computer(s) or the like, such as may be used to implement the databases, etc. shown in the drawings. Volatile storage media include dynamic memory, such as main memory of such a computer platform. Tangible transmission media include coaxial cables; copper wire and fiber optics, including the wires that comprise a bus within a computer system. Carrier-wave transmission media may take the form of electric or electromagnetic signals, or acoustic or light waves such as those generated during radio frequency (RF) and infrared (IR) data communications. Common forms of computer-readable media therefore include for example: a floppy disk, a flexible disk, hard disk, magnetic tape, any other magnetic medium, a CD-ROM, DVD or DVD-ROM, any other optical medium, punch cards paper tape, any other physical storage medium with patterns of holes, a RAM, a ROM, a PROM and EPROM, a FLASH-EPROM, any other memory chip or cartridge, a carrier wave transporting data or instructions, cables or links transporting such a carrier wave, or any other medium from which a computer may read programming code and/or data. Many of these forms of computer readable media may be involved in carrying one or more sequences of one or more instructions to a processor for execution.</div>
</li> <li> <para-num num="[0723]"> </para-num> <div class="description-line" id="p-0679" num="0723">The subject computer-executable code can be executed on any suitable device which may comprise a processor, including a server, a PC, or a mobile device such as a smartphone or tablet. Any controller or computer optionally includes a monitor, which can be a cathode ray tube (“CRT”) display, a flat panel display (e.g., active matrix liquid crystal display, liquid crystal display, etc.), or others. Computer circuitry is often placed in a box, which includes numerous integrated circuit chips, such as a microprocessor, memory, interface circuits, and others. The box also optionally includes a hard disk drive, a floppy disk drive, a high capacity removable drive such as a writeable CD-ROM, and other common peripheral elements. Inputting devices such as a keyboard, mouse, or touch-sensitive screen, optionally provide for input from a user. The computer can include appropriate software for receiving user instructions, either in the form of user input into a set of parameter fields, e.g., in a GUI, or in the form of preprogrammed instructions, e.g., preprogrammed for a variety of different specific operations.</div>
</li> <li> <para-num num="[0724]"> </para-num> <div class="description-line" id="p-0680" num="0724">The present invention also contemplates multiplex assays. The present invention is especially well suited for multiplex assays. For example, the invention encompasses use of a SureSelect<sup>XT, </sup>SureSelect<sup>XT2 </sup>and SureSelect<sup>QXT </sup>Target Enrichment System for Illumina Multiplexed Sequencing developed by Agilent Technologies (see, e.g., www.agilent.com/genomics/protocolvideos), a SeqCap EZ kit developed by Roche NimbleGen, a TruSeq® Enrichment Kit developed by Illumina and other hybridization-based target enrichment methods and kits that add sample-specific sequence tags either before or after the enrichment step.as well as Illumina HiSeq, MiSeq and NexSeq, Life Technology Ion Torrent. Pacific Biosciences PacBio RSII, Oxford Nanopore MinIon, Promethlon and GridIon and other massively parallel Multiplexed Sequencing Platforms.</div>
</li> <li> <para-num num="[0725]"> </para-num> <div class="description-line" id="p-0681" num="0725">Although the present invention and its advantages have been described in detail, it should be understood that various changes, substitutions and alterations can be made herein without departing from the spirit and scope of the invention as defined in the appended claims.</div>
</li> <li> <para-num num="[0726]"> </para-num> <div class="description-line" id="p-0682" num="0726">The present invention will be further illustrated in the following Examples which are given for illustration purposes only and are not intended to limit the invention in any way.</div>
</li> <li> <para-num num="[0727]"> </para-num> <div class="description-line" id="p-0683" num="0727">PROTOCOL</div>
</li> <li> <para-num num="[0728]"> </para-num> <div class="description-line" id="p-0684" num="0728">Description of Basic Protocol</div>
</li> <li> <para-num num="[0729]"> </para-num> <div class="description-line" id="p-0685" num="0729">Probe library generation</div>
</li> <li> <para-num num="[0730]"> </para-num> <div class="description-line" id="p-0686" num="0730">Each probe library has two groups of probes. One group is called “left probe,” and the other “right probe.” Each of the probes bind a short sequence on the target transcript (or cDNA, or target molecule) and the two binding sites are exactly right next to each other (juxtaposition probe set), so that once bound to target they can be ligated to given a single ligation product.</div>
</li> <li> <para-num num="[0731]"> </para-num> <div class="description-line" id="p-0687" num="0731">Applicants synthesized the two groups probe separately, and one of the probes will have the measurement barcode, the randome molecular index (UMI), and optionally the cell barcodes for single-cell experiments, as shown in <figref idrefs="DRAWINGS">FIG. 13</figref>.</div>
</li> <li> <para-num num="[0732]"> </para-num> <div class="description-line" id="p-0688" num="0732">The probe design in <figref idrefs="DRAWINGS">FIG. 13</figref> is just one group of the probe, in the figure it will be the left side probe, given the 5′-3′ direction. Applicants generate the probe through performing Klenow extension synthesis followed by lamda exonuclease digestion. In detail, Oligo 1 is first synthesized, containing the Universal adapter, Cell label, UMI, SMRI, and a LK (linker) region, followed by the synthesis of Oligo 2, the complementary sequence.</div>
</li> <li> <para-num num="[0733]"> </para-num> <div class="description-line" id="p-0689" num="0733">The left probe combined with the right side probe will be able to bind specifically to the designed location in the target transcript. Applicants then ligate the two probes to form amplification-ready product. This is followed by amplification to add on sequencing adapter. The amplified probes are then purified, quantified, and subjected to next-generation sequencing and analysis, as shown in <figref idrefs="DRAWINGS">FIG. 14</figref>.</div>
</li> <li> <para-num num="[0734]"> </para-num> <div class="description-line" id="p-0690" num="0734">Detailed Experimental Implementation for Bulk RNA</div>
</li> <li> <para-num num="[0735]"> </para-num> <div class="description-line" id="p-0691" num="0735">1. Extract RNA from K562 cells (K562 is a human immortalised myelogenous leukemia cell line)</div>
</li> <li> <para-num num="[0736]"> </para-num> <div class="description-line" id="p-0692" num="0736">2. Prepare cDNA by synthesis, using the RNA as template</div>
</li> <li> <para-num num="[0737]"> </para-num> <div class="description-line" id="p-0693" num="0737">3. Split part for qPCR and RNA-seq (as a reference control)</div>
</li> <li> <para-num num="[0738]"> </para-num> <div class="description-line" id="p-0694" num="0738">4. Split part for probe binding/ligation in the compressive sensing experiment</div>
</li> <li> <para-num num="[0739]"> </para-num> <div class="description-line" id="p-0695" num="0739">5. Synthesize Left and Right Probe and Generate Probe Library Mix</div>
</li> <li> <para-num num="[0740]"> </para-num> <div class="description-line" id="p-0696" num="0740">(1) Synthesize Oligo 1 with the adapter and barcode, and linker region</div>
</li> <li> <para-num num="[0741]"> </para-num> <div class="description-line" id="p-0697" num="0741">(2) Synthesize Oligo 2 with the complementary sequence to the linker region and gene-specific probe</div>
</li> <li> <para-num num="[0742]"> </para-num> <div class="description-line" id="p-0698" num="0742">(3) Anneal Oligo 1 and Oligo 2, and perform Klenow extension to synthesize the complete left side probe. The left side of the probe is either purified by column or used directly without further purification.</div>
</li> <li> <para-num num="[0743]"> </para-num> <div class="description-line" id="p-0699" num="0743">(4) Synthesize Oligo 3 with the right side gene-specific probe sequence, and then phosphorylate the right side probe with T4 Polynucleotide Kinase to phosphorylate the 5′ end of the oligo to render it ligation-ready</div>
</li> <li> <para-num num="[0744]"> </para-num> <div class="description-line" id="p-0700" num="0744">(5) Mix probes according to the design matrix to generate a probe library mix with both left and right probe.</div>
</li> <li> <para-num num="[0745]"> </para-num> <div class="description-line" id="p-0701" num="0745">6. Hybridize the probe library to the cDNA sample using a slow ramping protocol, where the mix is incubated at 95° C. for 5 minutes, then gradually cooled down to room temperature (25° C.). The annealed mixture containing left and right probes bound to the cDNA molecules are then ligated using Taq ligase to generate the ligation products at 45° C. for 2 hours, and the enzymatic activity is heat-inactivated at 65° C. for 20 minutes.</div>
</li> <li> <para-num num="[0746]"> </para-num> <div class="description-line" id="p-0702" num="0746">A control reaction where the probe library mixture is directly ligated with T7 DNA ligase without the addition of cDNA is also performed to obtain the abundance of the probe sets within the library. The ligation step is performed in a similar set-up except cDNA input is substituted with water, and ligation reaction is carried out at room temperature (25° C.) for 2 hours.</div>
</li> <li> <para-num num="[0747]"> </para-num> <div class="description-line" id="p-0703" num="0747">7. Amplify the ligation product using the library-amplification primer, which will also add the next-generation sequencer adapter to complete sequencing-ready library construction. The amplified is purified and size-selected between 200-300 bp and quantified.</div>
</li> <li> <para-num num="[0748]"> </para-num> <div class="description-line" id="p-0704" num="0748">Experimental Design for Implementation for Single Cell Sample</div>
</li> <li> <para-num num="[0749]"> </para-num> <div class="description-line" id="p-0705" num="0749">Each cell is captured and encapsulated in hydrogel droplet, which contains one side of the probe. These single-cell containing droplets are then lysed to allow initial hybridization, barcoded uniquely. Then, the other side of the probe will be added to the system to allow hybridization, ligation, and amplification. The final product are then purified and sequenced (or quantified/detected in other means), and subject to data analysis.</div>
</li> <heading id="h-0022">From Genetic Interactions to Molecular Interactions</heading>
<li> <para-num num="[0750]"> </para-num> <div class="description-line" id="p-0706" num="0750">Analyzing and determining genetic interactions can be used for dissection cellular circuitry. Results obtained using the method of the invention may be combined with further methods in order to probe molecular pathways. This may be achieved by using further perturbation assays (e.g. for validation purposes), by combining with complementary experiments in animal models (e.g. a Cas9 mouse or cell lines derived therefrom), including further genetic, biochemical or cellular testing. The systematic, sparsity-based approach may also be combined with ‘knowledge-based’ (informed) approaches, based on previously identified molecular interactions, such as known components of a given metabolic pathway.</div>
</li> <li> <para-num num="[0751]"> </para-num> <div class="description-line" id="p-0707" num="0751">The method of the invention may also be used to identify groups of cells based on their molecular profiling. The method of the invention may be used to identify possible target for therapeutics efforts.</div>
</li> <li> <para-num num="[0752]"> </para-num> <div class="description-line" id="p-0708" num="0752">The method of the invention may also be used to dissect cellular circuitry in relation with a spatial and/or temporal approach. For examples, the cells in the population may be synchronized with respect to their cell cycling.</div>
</li> <heading id="h-0023">Screening Methods</heading>
<li> <para-num num="[0753]"> </para-num> <div class="description-line" id="p-0709" num="0753">The present invention also envisions screening methods involving the herein described embodiments.</div>
</li> <li> <para-num num="[0754]"> </para-num> <div class="description-line" id="p-0710" num="0754">In one embodiment, the screening involves a a pooled, loss-of-function genetic screening approach suitable for both positive and negative selection that uses a genome-scale lentiviral single-guide RNA (sgRNA) library (see, e.g., Wang et al., Science. 2014 Jan. 3; 343(6166):80-4. doi: 10.1126/science.1246981. Epub 2013 Dec. 12). Briefly, sgRNA expression cassettes were stably integrated into the genome, which enabled a complex mutant pool to be tracked by massively parallel sequencing. A library containing 73,000 sgRNAs was used to generate knockout collections and performed screens in two human cell lines. A screen for resistance to the nucleotide analog 6-thioguanine identified all expected members of the DNA mismatch repair pathway, whereas another for the DNA topoisomerase II (TOP2A) poison etoposide identified TOP2A, as expected, and also cyclin-dependent kinase 6, CDK6. A negative selection screen for essential genes identified numerous gene sets corresponding to fundamental processes. sgRNA efficiency is associated with specific sequence motifs, enabling the prediction of more effective sgRNAs. See also Chen et al., Genome-wide CRISPR Screen in a Mouse Model of Tumor Growth and Metastasis, Cell (2015). DOI: 10.1016/j.cell.2015.02.038.</div>
</li> <li> <para-num num="[0755]"> </para-num> <div class="description-line" id="p-0711" num="0755">The activator screen method of Konermann et al., Nature (2014) doi:10.1038/nature14136 may be applied to the present invention. Systematic interrogation of gene function requires the ability to perturb gene expression in a robust and generalizable manner. Konermann et al. describes structure-guided engineering of a CRISPR-Cas9 complex to mediate efficient transcriptional activation at endogenous genomic loci. Konermann et al. used these engineered Cas9 activation complexes to investigate single-guide RNA (sgRNA) targeting rules for effective transcriptional activation, to demonstrate multiplexed activation of ten genes simultaneously, and to upregulate long intergenic non-coding RNA (lincRNA) transcripts. Konermann et al. also synthesized a library consisting of 70,290 guides targeting all human RefSeq coding isoforms to screen for genes that, upon activation, confer resistance to a BRAF inhibitor. The top hits included genes previously shown to be able to confer resistance, and novel candidates were validated using individual sgRNA and complementary DNA overexpression. A gene signature based on the top screening hits correlated with a gene expression signature of BRAF inhibitor resistance in cell lines and patient-derived samples. These results collectively demonstrate the potential of Cas9-based activators as a powerful genetic perturbation technology.</div>
</li> <li> <para-num num="[0756]"> </para-num> <div class="description-line" id="p-0712" num="0756">The mouse of Platt et al., Cell. 2014 Oct. 9; 159(2):440-55. doi: 10.1016/j.cell.2014.09.014. Epub 2014 Sep. 25 may also be contemplated in the present invention. Platt et al. established a Cre-dependent Cas9 knockin mouse and demonstrated in vivo as well as ex vivo genome editing using adeno-associated virus (AAV)-, lentivirus-, or particle-mediated delivery of guide RNA in neurons, immune cells, and endothelial cells. Using these mice, Platt et al. simultaneously modeled the dynamics of KRAS, p53, and LKB1, the top three significantly mutated genes in lung adenocarcinoma. Delivery of a single AAV vector in the lung generated loss-of-function mutations in p53 and Lkb 1, as well as homology-directed repair-mediated Kras(G12D) mutations, leading to macroscopic tumors of adenocarcinoma pathology.</div>
</li> <li> <para-num num="[0757]"> </para-num> <div class="description-line" id="p-0713" num="0757">The mouse of Platt et al., Cell. 2014 Oct. 9; 159(2):440-55. doi: 10.1016/j.cell.2014.09.014. Epub 2014 Sep. 25, may also be contemplated in the present invention. Platt et al. established a Cre-dependent Cas9 knockin mouse. Platt et al. demonstrated in vivo as well as ex vivo genome editing using adeno-associated virus (AAV)-, lentivirus-, or particle-mediated delivery of guide RNA in neurons, immune cells, and endothelial cells. Using these mice, Platt et al. simultaneously modeled the dynamics of KRAS, p53, and LKB1, the top three significantly mutated genes in lung adenocarcinoma. Delivery of a single AAV vector in the lung generated loss-of-function mutations in p53 and Lkb 1, as well as homology-directed repair-mediated Kras(G12D) mutations, leading to macroscopic tumors of adenocarcinoma pathology.</div>
</li> <li> <para-num num="[0758]"> </para-num> <div class="description-line" id="p-0714" num="0758">Genetic screens infer gene function in mammalian cells, but it has remained difficult to assay complex phenotypes—such as transcriptional profiles—in large screens. Applicants have developed Perturb-seq. Perturb-seq advantageously combines single cell RNA-seq and CRISPR based perturbations identified by unique polyadenylated barcodes to perform many such assays in a pooled experiment. Applicants demonstrate Perturb-seq by analyzing 200,000 cells across three screens: transcription factors involved in the response of dendritic cells to LPS, transcription factors bound in K562 cells, and cell cycle regulators in the same cell line. Perturb-seq accurately identifies individual gene targets, gene signatures, and cell states affected by each perturbation and their genetic interactions. The individual gene target predictions were validated by their known functions and ChIP-Seq. Applicants advantageously posit new functions for regulators affecting cell differentiation, the anti-viral response, and mitochondrial function during immune activation. Perturb-seq can be flexibly applied to diverse cell metadata, to customize design and scope of pooled genomic assays.</div>
</li> <li> <para-num num="[0759]"> </para-num> <div class="description-line" id="p-0715" num="0759">Although the present invention and its advantages have been described in detail, it should be understood that various changes, substitutions and alterations can be made herein without departing from the spirit and scope of the invention as defined in the appended claims.</div>
</li> <li> <para-num num="[0760]"> </para-num> <div class="description-line" id="p-0716" num="0760">The present invention will be further illustrated in the following Examples which are given for illustration purposes only and are not intended to limit the invention in any way.</div>
</li> <heading id="h-0024">EXAMPLES</heading>
<heading id="h-0025">Example 1</heading>
<heading id="h-0026">Combinatorial Genetic Perturbations in Mouse Cells using CRISPR-Cas Mediated Multiple Gene Activation</heading>
<li> <para-num num="[0761]"> </para-num> <div class="description-line" id="p-0717" num="0761">A pooled perturbation assay is performed in mouse cells, using a CRISPR-Cas system, as decribed in Shalem O., Sanjana N E., Zhang F. High-throughput functional genomics using CRISPR-Cas9. Nat Rev Genet. May; 16(5):299-311. (2015). doi: 10.1038/nrg3899. Epub 2015 Apr. 9. The pooled assay is designed to reach MOI values of 3-4, corresponding to orders 3-4 of genetic perturbations, concerning a total of 7,000 target genomic loci.</div>
</li> <li> <para-num num="[0762]"> </para-num> <div class="description-line" id="p-0718" num="0762">An sgRNA library is designed as described in Sanjana, N. E., Shalem, O. &amp; Zhang, F. Improved vectors and genome-wide libraries for CRISPR screening. <i>Nat Methods. </i>11, 783-784, doi:10.1038/nmeth.3047 (2014); Konermann, S., Brigham, M. D., Trevino, A. E., Joung, J., Abudayyeh, O., Barcena, C., Hsu, P. D., Habib, N., Gootenberg, J. S., Nishimasu, H., Nureki, O. &amp; Zhang, F. Genome-scale transcriptional activation by an engineered CRISPR-Cas9 complex. <i>Nature. </i>517, 583-588, doi:10.1038/nature14136 (2015). PMCID:4420636.</div>
</li> <li> <para-num num="[0763]"> </para-num> <div class="description-line" id="p-0719" num="0763">Effects of the genetic perturbations are determined in terms of single-cell RNA profiling. Said RNA profiling is carried out using droplet technology (Drop-Seq technique, as described in Macosko, E. Z., Basu, A., Satija, R., Nemesh, J., Goldman, M., Tirosh, I., Bialas, A. R., Kamitaki, N., Sanes, J. R., Weitz, D. A., Shalek, A. K., Regev, A. &amp; McCarroll, S. A. Highly Parallel Genome-wide Expression Profiling of Individual Cells Using Nanoliter Droplets. <i>Cell. </i>2015 May 21; 161(5):1202-14. doi: 10.1016/j.cell.2015.05.002. PMCID:4481139; or Klein, A. M., Mazutis, L., Akartuna, I., Tallapragada, N., Veres, A., Li, V., Peshkin, L., Weitz, D. A., Kirschner, M. W. Droplet barcoding for single-cell transcriptomics applied to embryonic stem cells. Cell. 2015 May 21; 161(5):1187-201. doi: 10.1016/j.cell.2015.04.044. PMCID: 4441768).</div>
</li> <heading id="h-0027">Example 2</heading>
<heading id="h-0028">Combinatorial Genetic Perturbations in Human Cells Using CRISPR-Cas Mediated Multiple Gene Knock-Outs</heading>
<li> <para-num num="[0764]"> </para-num> <div class="description-line" id="p-0720" num="0764">A pooled perturbation assay is performed in a human using a CRISPR-Cas system, as decribed in Shalem O., Sanjana N E., Zhang F. High-throughput functional genomics using CRISPR-Cas9. Nat Rev Genet. May; 16(5):299-311. (2015). doi: 10.1038/nrg3899. Epub 2015 Apr. 9. The pooled assay is designed to reach MOI values of 2-3, corresponding to orders 2-3 of genetic perturbations regarding a total of 10,000 target genomic loci.</div>
</li> <li> <para-num num="[0765]"> </para-num> <div class="description-line" id="p-0721" num="0765">An sgRNA library is designed as described in Sanjana, N. E., Shalem, O. &amp; Zhang, F. Improved vectors and genome-wide libraries for CRISPR screening. <i>Nat Methods. </i>11, 783-784, doi:10.1038/nmeth.3047 (2014); Shalem, O., Sanjana, N. E., Hartenian, E., Shi, X., Scott, D. A., Mikkelsen, T. S., Heckl, D., Ebert, B. L., Root, D. E., Doench, J. G. &amp; Zhang, F. Genome-scale CRISPR-Cas9 knockout screening in human cells. <i>Science. </i>343, 84-87, doi:10.1126/science.1247005 (2014). PMCID:4089965; Shalem, O., Sanjana, N. E. &amp; Zhang, F.</div>
</li> <li> <para-num num="[0766]"> </para-num> <div class="description-line" id="p-0722" num="0766">High-throughput functional genomics using CRISPR-Cas9. Nat Rev Genet. 16, 299-311, doi:10.1038/nrg3899 (2015).</div>
</li> <li> <para-num num="[0767]"> </para-num> <div class="description-line" id="p-0723" num="0767">Effects of the genetic perturbations are determined in terms of single-cell RNA profiling. Said RNA profiling is carried out using droplet technology (Drop-Seq technique, as described in Macosko, E. Z., Basu, A., Satija, R., Nemesh, J., Goldman, M., Tirosh, I., Bialas, A. R., Kamitaki, N., Sanes, J. R., Weitz, D. A., Shalek, A. K., Regev, A. &amp; McCarroll, S. A. Highly Parallel Genome-wide Expression Profiling of Individual Cells Using Nanoliter Droplets. <i>Cell. </i>2015 May 21; 161(5):1202-14. doi: 10.1016/j.cell.2015.05.002. PMCID:4481139; or Klein, A. M., Mazutis, L., Akartuna, I., Tallapragada, N., Veres, A., Li, V., Peshkin, L., Weitz, D. A., Kirschner, M. W. Droplet barcoding for single-cell transcriptomics applied to embryonic stem cells. <i>Cell. </i>2015 May 21; 161(5):1187-201. doi: 10.1016/j.cell.2015.04.044. PMCID: 4441768).</div>
</li> <heading id="h-0029">Example 3</heading>
<heading id="h-0030">Perturb-Seq: Pooled CRISPR Screens with Single Cell RNA-seq Readout</heading>
<li> <para-num num="[0768]"> </para-num> <div class="description-line" id="p-0724" num="0768">Applicants devised Perturb-seq by combining the design of a pooled CRISPR screen, where the identity of each perturbation is reported by a unique barcode for each guide that is assayed by sequencing, with massively-parallel, droplet-based scRNA-seq, as a rich genomic phenotype for each cell (<figref idrefs="DRAWINGS">FIG. 48A</figref>). In Perturb-seq, the sgRNA barcode is reported by a polyadenylated RNA, which is captured by single cell RNA-seq. Perturb-seq uses a CRISPR lentiviral vector that both delivers an sgRNA to a cell and reports on the identity of the delivered sgRNA by an expressed PolyAdenylated Perturbation Index (PAPI) (<figref idrefs="DRAWINGS">FIG. 48B</figref>). As in a typical pooled genetic screen, applicants first infect cells with a pool of lentiviruses, each carrying a construct for a particular guide. Depending on the multiplicity of infection (MOI), applicants expect a different distribution of the number of lentivirus genomic insertions (and hence sgRNAs) in each cell. At low MOI, the vast majority of cells would have one or no guides. In such a case, Perturb-seq can be used to assess the effect of single gene perturbations. By increasing the multiplicity of infection (At higher MOI), a greater fraction of cells possesses more than one guide. Thus, the MOI is a tunable parameter that, for a given number of cells, determines how powered tests will be to study epistatic effects.</div>
</li> <li> <para-num num="[0769]"> </para-num> <div class="description-line" id="p-0725" num="0769">Next, following growth, differentiation, and/or stimulation, Applicants collect the cells and process them through massively parallel scRNA-seq. This process tags the mRNA of each cell, including the synthetic PAPI, with a unique cell barcode (CBC) and a unique molecular identifier (UMI). The CBC associates the cell's transcriptional profile with the genetic perturbation(s) delivered to this cell, as encoded by the PAPI. Applicants perform Whole Transcriptome Amplification (WTA), library preparation, and sequencing in a single pool. In our current implementation, Applicants rely on a droplet method, which is now commercially available (Zheng et al., 2016) (Methods and Resources), but our design is compatible with additional single-cell RNA-seq methods (Fan et al., 2015; Klein et al., 2015; Macosko et al., 2015), and Applicants have tested it successfully with Drop-seq (Macosko et al., 2015) in both K562 cells and BMDCs, albeit with different gene targets than in the rest of this study (AD, OP, BL, and AR, unpublished data).</div>
</li> <li> <para-num num="[0770]"> </para-num> <div class="description-line" id="p-0726" num="0770">Applicants demonstrate Perturb-seq, in two complementary biological systems: K562 cells, a rapidly growing human erythroleukemia cell line, and bone marrow derived mouse dendritic cells (BMDCs), a post-mitotic, short-lived primary cell, which Applicants analyzed pre-stimulation (0 h) and three hours following stimulation with LPS. Applicants have previously established both systems for genome-wide CRISPR screens, with a readout of either cell viability (in K562 cells (Wang et al., 2015)), or an expression marker (TNF, in BMDCs (Parnas et al., 2015)). In both systems, the Cas9 protein is stably knocked into the cell either in vitro (K562 cells) or in the germline of a transgenic mouse from which the cells are derived (Parnas et al., 2015; Platt et al., 2014).</div>
</li> <li> <para-num num="[0771]"> </para-num> <div class="description-line" id="p-0727" num="0771">Altogether, Applicants performed six Peturb-seq experiments, analyzing 200,000 cells in total (<figref idrefs="DRAWINGS">FIG. 48C</figref>). In K562 cells, Applicants targeted either 14 TFs or 10 cell cycle regulators, in separate pooled experiments (Table S1). For TFs, Applicants performed experiments in both lower and higher MOI, and at two time points (7 and 13 days post infection). In BMDCs, Applicants targeted 24 TFs (Table S1) important for either BMDC cell differentiation or the response to LPS (Amit et al., 2009; Garber et al., 2012), and measured their effects at both pre-stimulation (0 h) and three hours (3 h) post-stimulation. Finally, in each case, Applicants collected reference single cell RNA-seq data from unperturbed cells in separate experiments: 5,400 K562 cells; 2,700 unstimulated BMDCs; and 1,300 BMDCs stimulated with LPS for 3h (Table S1).</div>
</li> <heading id="h-0031">Example 4</heading>
<heading id="h-0032">Perturb-seq Efficiently Detects Expressed RNA Barcodes, Gene Expression Profiles, and Impact on Targeted Genes</heading>
<li> <para-num num="[0772]"> </para-num> <div class="description-line" id="p-0728" num="0772">To couple massively parallel scRNA-seq with a pooled CRISPR/Cas9 screen, applicants developed a lentiviral vector in which a polyadenylated random barcode is associated with an sgRNA and reports on its identity (<figref idrefs="DRAWINGS">FIG. 48C</figref>, Methods and Resources). Applicants first matched each sgRNAs with the identity of its corresponding RNA barcode, by either Sanger or next-generation sequencing (Methods and Resources), and then converted it into lentivirus for pooled transduction. The guide barcode follows the ORF for BFP, allowing us to sort only infected cells. Finally, because accurate detection of sgRNA from single cells is critical for the accuracy of the pooled approach, applicants designed a dial-out PCR approach, which applicants applied following WTA, to further amplify and sequence the barcode (Methods).</div>
</li> <li> <para-num num="[0773]"> </para-num> <div class="description-line" id="p-0729" num="0773">Applicants also developed an enrichment protocol to optimize our ability to detect the sgRNA(s) delivered to each cell. First, Applicants associated each sgRNA with the identity of its corresponding PAPI, by either Sanger or next-generation sequencing (Methods and Resources). Applicants next converted the plasmids into lentiviruses for pooled transduction. The plasmid construct included an ORF encoding a BFP-T2A-Puromycin (<figref idrefs="DRAWINGS">FIG. 48B</figref>), allowing us to select for transduced cells by FACS sorting or by antibiotic selection. Finally, to increase the accuracy of detection of sgRNA from single cells, Applicants designed a dial-out PCR approach, to enrich for the PAPI following WTA (Methods and Resources). Applicants observed low levels of many PAPIs present in many cells (due to cross talk caused by PCR chimeras and ambient RNA) (<figref idrefs="DRAWINGS">FIG. 55F</figref>), and account for it by normalizing the observed number of PAPIs within a cell, and setting a threshold (Methods and Resources).</div>
</li> <li> <para-num num="[0774]"> </para-num> <div class="description-line" id="p-0730" num="0774">Applicants estimated the probability of detection of the PAPI by considering two factors: (1) the initial MOI and (2) the transcript capture efficiency of our scRNA-seq protocol. Applicants reasoned that these should be described well by a generative model, where applicants assume a zero-truncated Poisson distribution for infection with a guide-carrying lentivirus (zeros are truncated by BFP+selection), convolved with a binomial process (for the probability of detection) (Methods). Indeed, the predicted fit was indistinguishable from the observed frequencies of number of guides per cell (<figref idrefs="DRAWINGS">FIG. 55A-C</figref>, bottom panels, KS-test. Using this approach, applicants estimatea 94% (92-96%) detection probability with an approximate initial MOI of 0.63 in the K562 TF pool, a 98% probability with an MOI of 0.35 in the cell cycle pool, and a 60% detection probability with an approximate initial MOI of 1.4 in BMDCs at 3h (<figref idrefs="DRAWINGS">FIG. 55A-E</figref>).The lower detection probability in BMDCs reflects overall lower complexity of these single cell profiles (Table S1), and is an additional source of noise, which Applicants handle in downstream computational analysis. Nevertheless, even in BMDCs, Applicants were able to detect cells with multiple guides per cell (<figref idrefs="DRAWINGS">FIG. 48D</figref>).</div>
</li> <li> <div class="description-line" id="p-0731" num="0000">
<tables id="TABLE-US-00003" num="00003">
<patent-tables colsep="0" frame="none" pgwide="1" rowsep="0">
<table align="left" class="description-table" cols="7" colsep="0" rowsep="0" width="100%">
<thead>
<tr class="description-tr">
<td class="description-td" colspan="7" nameend="7" namest="1" rowsep="1">TABLE S1</td>
</tr>
<tr class="description-tr">
<td align="center" class="description-td" colspan="7" nameend="7" namest="1" rowsep="1"> </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">Average</td>
<td class="description-td">Median</td>
<td class="description-td">Median</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">Number of</td>
<td class="description-td">Number of</td>
<td class="description-td"> </td>
<td class="description-td">PAPI</td>
<td class="description-td">Genes</td>
<td class="description-td">Transcripts</td>
</tr>
<tr class="description-tr">
<td class="description-td">Pool</td>
<td class="description-td">Cells</td>
<td class="description-td">sgRNAs</td>
<td class="description-td">Cells/sgRNA</td>
<td class="description-td">detected/cell</td>
<td class="description-td">Detected</td>
<td class="description-td">Detected</td>
</tr>
<tr class="description-tr">
<td align="center" class="description-td" colspan="7" nameend="7" namest="1" rowsep="1"> </td>
</tr>
</thead>
<tbody><tr class="description-tr">
<td class="description-td"> </td>
</tr>
</tbody></table>
<table align="left" class="description-table" cols="7" colsep="0" rowsep="0" width="100%">
<tbody><tr class="description-tr">
<td class="description-td">PBMDC 3 hr</td>
<td class="description-td">37369</td>
<td class="description-td">57</td>
<td class="description-td">655.5964912</td>
<td class="description-td">1.1</td>
<td class="description-td">1537</td>
<td class="description-td">5074</td>
</tr>
<tr class="description-tr">
<td class="description-td">PBMDC unstim</td>
<td class="description-td">32624</td>
<td class="description-td">57</td>
<td class="description-td">572.3508772</td>
<td class="description-td">1.1</td>
<td class="description-td">988</td>
<td class="description-td">2187</td>
</tr>
<tr class="description-tr">
<td class="description-td">K562 TFs low MOI 7 days</td>
<td class="description-td">33104</td>
<td class="description-td">30</td>
<td class="description-td">1103.466667</td>
<td class="description-td">1.3</td>
<td class="description-td">3015</td>
<td class="description-td">11610</td>
</tr>
<tr class="description-tr">
<td class="description-td">K562 TFs low MOI 13 days</td>
<td class="description-td">19268</td>
<td class="description-td">30</td>
<td class="description-td">642.2666667</td>
<td class="description-td">1.1</td>
<td class="description-td">2952</td>
<td class="description-td">12303</td>
</tr>
<tr class="description-tr">
<td class="description-td">K562 TFs high MOI</td>
<td class="description-td">51898</td>
<td class="description-td">30</td>
<td class="description-td">1729.933333</td>
<td class="description-td">1.9</td>
<td class="description-td">3080</td>
<td class="description-td">13974</td>
</tr>
<tr class="description-tr">
<td class="description-td">K562 cell cycle regulators</td>
<td class="description-td">26020</td>
<td class="description-td">33</td>
<td class="description-td">788.4848485</td>
<td class="description-td">1.2</td>
<td class="description-td">3764</td>
<td class="description-td">11716</td>
</tr>
<tr class="description-tr">
<td class="description-td">Wildtype PBMDC 3 hr</td>
<td class="description-td">1310</td>
<td class="description-td">NA</td>
<td class="description-td">NA</td>
<td class="description-td">NA</td>
<td class="description-td">728</td>
<td class="description-td">2162</td>
</tr>
<tr class="description-tr">
<td class="description-td">Wildtype PBMDC unstim</td>
<td class="description-td">2713</td>
<td class="description-td">NA</td>
<td class="description-td">NA</td>
<td class="description-td">NA</td>
<td class="description-td">585</td>
<td class="description-td">1450</td>
</tr>
<tr class="description-tr">
<td class="description-td">Wildtype K562</td>
<td class="description-td">5409</td>
<td class="description-td">NA</td>
<td class="description-td">NA</td>
<td class="description-td">NA</td>
<td class="description-td">1284</td>
<td class="description-td">3285</td>
</tr>
<tr class="description-tr">
<td class="description-td">TOTALS</td>
<td class="description-td">209715</td>
<td class="description-td">237</td>
<td class="description-td">5492.098884</td>
<td class="description-td">7.7</td>
<td class="description-td">17933</td>
<td class="description-td">63761</td>
</tr>
<tr class="description-tr">
<td class="description-td">AVERAGES</td>
<td class="description-td">23301.66667</td>
<td class="description-td">39.5</td>
<td class="description-td">915.3498139</td>
<td class="description-td">1.283333333</td>
<td class="description-td">1992.555556</td>
<td class="description-td">7084.555556</td>
</tr>
<tr class="description-tr">
<td align="center" class="description-td" colspan="7" nameend="7" namest="1" rowsep="1"> </td>
</tr>
</tbody></table>
</patent-tables>
</tables>
</div>
</li> <li> <para-num num="[0775]"> </para-num> <div class="description-line" id="p-0732" num="0775">Single cell RNA-seq profiles typically represent only a fraction of the molecular complexity of the measured cell's transcriptome (estimated at 2-26% for droplet based approaches (Klein et al., 2015; Macosko et al., 2015; Zheng et al., 2016)). In both systems, averages of approximately 100 single cell droplet profiles recovered the population complexity (<figref idrefs="DRAWINGS">FIG. 55J</figref>,K). (Applicants do note a bias in capture efficiency of small transcripts (&lt;500 nts; <figref idrefs="DRAWINGS">FIG. 55L</figref>), which may be associated with the 3′ library preparation protocol.)</div>
</li> <li> <para-num num="[0776]"> </para-num> <div class="description-line" id="p-0733" num="0776">Finally, Applicants showed that a reduction of the targeted gene's expression is typically associated with the corresponding perturbation (<figref idrefs="DRAWINGS">FIG. 48E</figref>,F, <figref idrefs="DRAWINGS">FIG. 55G-I</figref>). In general, this measure should be a proxy for cutting efficiency, although it is possible that regulatory mechanisms can in some cases cause overexpression of a transcript in the absence of a functional protein. Indeed, the expression of sgRNA targets was significantly reduced from zero (e.g., in BMDCs: <figref idrefs="DRAWINGS">FIG. 48F</figref>,</div>
</li> <li> <para-num num="[0777]"> </para-num> <div class="description-line" id="p-0734" num="0777">p-value&lt;10<sup>−4</sup>, one-sample t-test). Notably, our ability to determine a reduction in expression is strongly related to the expression level of the target in the wild type cells (<figref idrefs="DRAWINGS">FIG. 55I</figref>).</div>
</li> <heading id="h-0033">Example 5</heading>
<heading id="h-0034">A computational model to predict the effect of perturbations on the expression of individual genes and on global cell states</heading>
<li> <para-num num="[0778]"> </para-num> <div class="description-line" id="p-0735" num="0778">Applicants next devised a computational framework, based on a linear model, to estimate the impact of perturbations on gene expression (<figref idrefs="DRAWINGS">FIG. 48G</figref>, <figref idrefs="DRAWINGS">FIG. 49</figref> and <figref idrefs="DRAWINGS">FIG. 56</figref>) (<figref idrefs="DRAWINGS">FIG. 49</figref>). In the basic linear model (<figref idrefs="DRAWINGS">FIG. 49</figref>) (<figref idrefs="DRAWINGS">FIG. 48G and 49A</figref>) the logarithm of, the expression of each gene in a cell is explained as a linear (weighted) combination of the effects of individual sgRNAs on the gene. Thus, the log expression level of each gene in each cell (expression matrix Y, observed) is a product of two matrices: one describing cell attributes—which guides are present in each cell (design matrix X, observed), and the other represents the regulatory effect of each attribute on each gene (coefficient matrix β). This basic model assumes no interactions: that is, the combined effect of two perturbations is simply their weighted sum. The perturb-seq measurements provide matrix Y (expression levels in each cell) and X (guides in each cell; based on analysis of dial-out PCR measurements of PAPIs; Methods and Resources), with good fidelity (<figref idrefs="DRAWINGS">FIG. 48</figref>) (<figref idrefs="DRAWINGS">FIGS. 48 and 55</figref>). Thus, Applicants can fit matrix β to determine the effect of each guide on each gene.</div>
</li> <li> <para-num num="[0779]"> </para-num> <div class="description-line" id="p-0736" num="0779">To fit the coefficient matrix β (effect of each guide on each gene) given the observations, Applicants used elastic net regularization, which contains both L<sub>1 </sub>penalty, to reduce the number of hypotheses tested, and an L<sub>2 </sub>penalty on the magnitude of the coefficients (Methods and Resources). The second regularization is especially important in the case of correlated covariates and noisy data. Notably, Applicants did not provide any other prior knowledge when learning the model: in particular, Applicants did not include which gene is targeted by which guide. After the model is fit, Applicants evaluate the significance of the fit coefficients using a permutation-based approach to calculate significance (Methods and Resources, <figref idrefs="DRAWINGS">FIGS. 56D-G</figref>).</div>
</li> <li> <para-num num="[0780]"> </para-num> <div class="description-line" id="p-0737" num="0780">Next, applicants considered how to handle key confounders that are observed in the data. Applicants account for three key classes of covariates that represent either core biological phenomena or expected confounders (<figref idrefs="DRAWINGS">FIG. 49A-G</figref>): (1) the number of observed UMIs in a cell (“cell quality”); (2) the probability that a perturbation successfully affected the cell; and (3) the presence of distinct subpopulations, of either different cell types or states, in the same sample. Below Applicants describe how each of these aspects manifests in the original data, and how it is handled by our model.</div>
</li> <li> <para-num num="[0781]"> </para-num> <div class="description-line" id="p-0738" num="0781">First, as typical in scRNA-seq experiments (Kharchenko et al., 2014; Shalek et al., 2014; Stegle et al., 2015), scRNA-seq profiles from individual cells in the same experiment can vary substantially in their quality, for example as reflected here by the number of UMIs (RNA barcode) (<figref idrefs="DRAWINGS">FIG. 49</figref>). assigned to the cell. Applicants ability to detect the expression of a given transcript in any given cell depends on both technical and biological factors relating to the total number of transcripts Applicants captured in that cell and the transcript's original expression level in that cell (<figref idrefs="DRAWINGS">FIG. 49B</figref>). To address this confounder in the basic linear model, Applicants added a covariate to our model that represents the sum of the total number of UMIs detected per cell as well the square of this feature (<figref idrefs="DRAWINGS">FIG. 49C</figref>).</div>
</li> <li> <para-num num="[0782]"> </para-num> <div class="description-line" id="p-0739" num="0782">Next, Applicants considered effect of a perturbation (<figref idrefs="DRAWINGS">FIG. 49F</figref>) as well as the the possibility that in some cells even when a guide is confidently detected, it may not have had any phenotypic effect in that specific cell. This could happen, for example, because the CRISPR system produced no mutations, a heterozygous cell for a fully or partially haplosufficient gene, an in-frame mutation, or even if the editing event was too recent to have had a phenotypic impact. Indeed, for most guides, while cells carrying the guide are distinguishable on average from those that do not, there are still cells with reliably detected guides, but with profiles that are similar to those of cells without the guide (e.g., <figref idrefs="DRAWINGS">FIG. 49H</figref>, <figref idrefs="DRAWINGS">FIG. 56I</figref>).</div>
</li> <li> <para-num num="[0783]"> </para-num> <div class="description-line" id="p-0740" num="0783">When profiling RNA from a population of perturbed cells in bulk, this mixture would dilute the true perturbation phenotype. If perturbation effects are strong, it should be possible to distinguish true perturbed cells from unperturbed cells leveraging the single cell nature of our data. The regulatory matrix (β) derived from the observed guide-to-cell assignment, provides an initial assessment of the effects of perturbations. Applicants use this model to revisit each cell and predict whether each of the observed guides had perturbed the cell (<figref idrefs="DRAWINGS">FIG. 4911</figref>, right). Applicants can then re-estimate the model with the corrected perturbation-to-cell assignment. This one iteration can be thought of as approximate Expectation-Minimization applied to the linear model.</div>
</li> <li> <para-num num="[0784]"> </para-num> <div class="description-line" id="p-0741" num="0784">Third, Applicants reasoned that the cells in each of our experiments might belong to one of several states or types, and that those global states can be affected by the perturbations</div>
</li> <li> <para-num num="[0785]"> </para-num> <div class="description-line" id="p-0742" num="0785">(<figref idrefs="DRAWINGS">FIG. 49E</figref>). For example, in K562 cells Applicants expected (Macosko et al., 2015) cells in different phases of the cell cycle, as well as apoptotic cells. In BMDCs, unsupervised clustering shows several distinct subpopulations, including overlap with those previously reported (Helft et al., 2015). Indeed, even after accounting for cell complexity, many transcripts still show two distinct modes of expression (e.g., <figref idrefs="DRAWINGS">FIG. 49C</figref>). Thus, Applicants may want to distinguish between a global impact of a perturbation on cell proportions and a more specific impact within a cell state. To model sub-populations, Applicants first define them using the profiles of the matched, genetically unperturbed, experiments (Methods and Resources), and then incorporate this classification as a set of covariates in our model. Applicants fit the model either with or without these subpopulations covariates. When cell states are fit as covariates, the two modes of expression are no longer observed in the residuals (e.g., <figref idrefs="DRAWINGS">FIG. 49D</figref>), and some of the sgRNA's effect are accounted for (compare <figref idrefs="DRAWINGS">FIG. 49F</figref> and G), suggesting that those perturbations may have primarily affected the proportions of cell types.</div>
</li> <li> <para-num num="[0786]"> </para-num> <div class="description-line" id="p-0743" num="0786">After the model is fit, Applicants can evaluate the significance of the observed coefficients using a permutation based approach to calculate significance (Materials and Methods, <figref idrefs="DRAWINGS">FIGS. 56D-G</figref>).</div>
</li> <li> <para-num num="[0787]"> </para-num> <div class="description-line" id="p-0744" num="0787">Applicants framework, MIMOSCA (<figref idrefs="DRAWINGS">FIG. 4911</figref>), can be readily extended to incorporate non-linear interactions between covariates. In principle, gene perturbations may interact in nonlinear ways with one another (“genetic interactions”), with cell states (cell state specific gene expression changes), or with our quality measures (e.g., changes in cell size). Applicants describe an extension for interactions between perturbations below.</div>
</li> <li> <para-num num="[0788]"> </para-num> <div class="description-line" id="p-0745" num="0788">Example 6</div>
</li> <heading id="h-0035">The Linear Model is Robust, Reproducible and Predictive</heading>
<li> <para-num num="[0789]"> </para-num> <div class="description-line" id="p-0746" num="0789">Applicants assessed the quality of the linear model by its robustness, fit, and biological coherence. Applicants determined the contribution of the four core components of the model (cell complexity, batch, phenotypic effects of perturbations, and cell state), by analyzing the proportion of the variance in the data each explains(FIG. 49I, <figref idrefs="DRAWINGS">FIG. 56A-C</figref>). For example, for stimulated BMDCs, the perturbations themselves explain 5% of the variance in the data, 17% is explained when adding covariates associated with number of transcripts detected, and up to 20% with cell state covariates (<figref idrefs="DRAWINGS">FIG. 49</figref> I). Applicants obtain similar results with the other datasets (<figref idrefs="DRAWINGS">FIG. 56A-C</figref>). This highlights the importance of accounting for covariates to ensure sensitive detection of the relevant perturbation signal (<figref idrefs="DRAWINGS">FIG. 54</figref>). Furthermore, gene-gene correlations in the residuals were significantly reduced as covariates were added (<figref idrefs="DRAWINGS">FIG. 49J</figref>), This suggests that much of the global correlation is mediated through transcripts detected and cell states.</div>
</li> <li> <para-num num="[0790]"> </para-num> <div class="description-line" id="p-0747" num="0790">Considering the multivariate nature of gene expression measurements, Applicants also analyzed the consistency in the effects on genes across cells that all contain a particular perturbation (<figref idrefs="DRAWINGS">FIG. 56I</figref>). Overall, guides targeting genes have stronger and more consistent effects than the non-targeting control guides.</div>
</li> <heading id="h-0036">Example 7</heading>
<heading id="h-0037">Perturb-seq Dissects the Transcriptional Program in the LPS Response in BMDC</heading>
<li> <para-num num="[0791]"> </para-num> <div class="description-line" id="p-0748" num="0791">Applicants next analyzed the effect of each of 24 TFs on the transcriptional response in BMDCs to LPS. Briefly, when DCs are stimulated with LPS they activate the expression of ˜2,000 genes (Amit et al., 2009; Parnas et al., 2015), organized into several key response modules (Parnas et al., 2015; Shalek et al., 2014). Previously, Applicants have implicated dozens of factors in the response, and validated some of their roles by genetic perturbations (Amit et al., 2009; Parnas et al., 2015). In addition, ChIP-seq analysis of 29 transcription factors (19 of them among the 24 Applicants perturbed) at 4 time points during the response (Garber et al., 2012) showed that factors can be distinguished into three categories: (1) pioneer factors that commit the cells to their lineage (CEBPB and Spil/Pu.1); (2) broad binders that prime lineage specific genes, before their stimulus-induced expression (e.g., JunB, Irf4, Atf3); and (3) dynamic factors that activate response-specific genes (e.g., Stat2). Knockdown of some factors, but not all, affect the expression of the genes they bind, suggesting buffering genetic interactions for at least some factors (Garber et al., 2012). The response is not perfectly synchronous, such that at different time points, there is bi-modality across the cell population in the expression of specific modules (Shalek et al., 2013, 2014). Finally, Applicants showed that the cells in a BMDC population may consist of at least two sub-types, and that their relative proportions may shift between experiments (Shalek et al., 2013, 2014). Applicants reasoned that this extensive prior knowledge would help benchmark our results, and that the single cell nature of our screen and computational model would directly address the challenges of heterogeneity.</div>
</li> <li> <para-num num="[0792]"> </para-num> <div class="description-line" id="p-0749" num="0792">Applicants performed Perturb-seq at two time points. Applicants obtained precursor cells from the bone marrow of Cas9 transgenic mice (Platt et al., 2014), and after two days of culturing in GM-CSF, Applicants infected the cells with a pool of lentivirus carrying the Perturb-seq vector with 1-4 guide RNAs targeting each of 24 factors (total of 67 guides) and a non-targeting control. After another seven days, Applicants stimulated the cells with LPS, and collected cells for scRNA-seq at 0 and 3h, profiling 32,624 and 37,369 cells, respectively (Table S1).</div>
</li> <li> <para-num num="[0793]"> </para-num> <div class="description-line" id="p-0750" num="0793">Fitting our regulatory model without considering sub-populations or genetic interactions (<figref idrefs="DRAWINGS">FIGS. 50 and 51</figref>, Methods and Resources), the key predictions—the regulatory effects of guides on genes—were coherent in the sense that guides targeting the same gene typically grouped together by their similar impact on genes in the model (i.e., by the regulatory coefficients, α, in matrix B; <figref idrefs="DRAWINGS">FIG. 51A</figref>), Thus, the correlation in regulatory profiles (α) between guides targeting the same gene is significantly higher than between those targeting different genes (<figref idrefs="DRAWINGS">FIG. 50B</figref>, P&lt;10<sup>−9</sup>, Wilcoxon signed-rank test), even though the model had no knowledge which guides target the same gene. The model also typically shows the repressive effect of guides on the expression of their direct target, although no information was provided to the model on the identity of the gene that it is designed for (<figref idrefs="DRAWINGS">FIG. 50A</figref>, separate column on left).</div>
</li> <li> <para-num num="[0794]"> </para-num> <div class="description-line" id="p-0751" num="0794">Applicants uncovered four key modules of transcription factors (Ml-M4), based on the overall regulatory effects of the different guides as inferred in the regulatory matrix, consistent with their known function (<figref idrefs="DRAWINGS">FIG. 50</figref>) (<figref idrefs="DRAWINGS">FIG. 50A</figref> yellow squares, and 51A): (1) the anti-viral TFs (Stat1, 2; M1); (2) the pioneer factor Cebpb along with the broad binder JunB and the dynamic binders Rela, Stat3 and Hif1a (M2); (3) The dynamic binders Rel and Irf2, along with the broad binder Atf3 (M3); and (4) The pioneer factor Spi1/Pu.1, broad binders Runx1 and Irf4, and the dynamic binder Nfkb1 (M4). While each TF module consists of guides targeting different TFs, the correlation in the regulatory effects of guides targeting the same gene is especially high (e.g., Stat1, Stat2, Cebpb, Rela, Rel, Irf2, Atf3, Spil/Pu.1, Irf4, Nfkb1, Runx1 etc.; <figref idrefs="DRAWINGS">FIG. 51A</figref>, 50B), supporting the specificity of Perturb-seq and suggesting TF specific effects within the context of shared overall programs.</div>
</li> <li> <para-num num="[0795]"> </para-num> <div class="description-line" id="p-0752" num="0795">The four TF modules regulate five major biological programs (P1-5; <figref idrefs="DRAWINGS">FIG. 51A</figref> and B): an anti-viral response (P1), a program of antigen presentation, cytoskeleton and ribosomal protein (RP) expression, including some markers of DC maturation and of “cluster disrupted” cells (e.g., SerpinB6) (P2), mitochondrial function and biogenesis (P3), an interferon gamma program, with chemokines and cytokines important in the response to intracellular pathogens (viruses, bacteria or parasites) (P4), and an inflammatory TNF response (P5). For example, perturbation of Stat1 and Stat2 (M1 module) represses both the anti-viral (P1) program and the interferon gamma program (P4). Perturbation of the M2 module (Cebpb, Rela, and other factors) represses both the inflammatory and IFN gamma modules, consistent with their roles of activators of inflammation, and activates the P2 program of antigen presentation, cytoskeleton and RPs. The M4 module (Spi1, Nfkb1, Runx1, etc.) has an opposite effect to that of M2. The Rel module (M3) has a partially opposing effect to that of the Stat (M1) module, for example inhibiting the antiviral program.</div>
</li> <li> <para-num num="[0796]"> </para-num> <div class="description-line" id="p-0753" num="0796">These results highlight both known functional associations and novel predictions. For example, Stat1 and Stat2 are well known activators of the anti-viral program (Shalek et al., 2013). The predicted repression of the antiviral program by the Rel module, including Irf2 and Atf3, is supported by a recent report in macrophages (Labzin et al., 2015) that Atf3 can act as a direct transcriptional repressor of the interferon beta (IFNβ) gene, that it further modulates the expression of other genes downstream of IFN signaling, and that Atf3-null macrophages have an enhanced clearing of virus.</div>
</li> <li> <para-num num="[0797]"> </para-num> <div class="description-line" id="p-0754" num="0797">Our model also predicted that Stat1 and 2 are repressors of a program of mitochondrial function and biogenesis in BMDCs (<figref idrefs="DRAWINGS">FIG. 51A</figref> and B). BMDCs undergo a shift from mitochondrial to glycolytic metabolism during the LPS response, including mitophagy (Cohen and Sheva, 1998) (Jovanovic et al., 2015) . Recent studies showed that Stat1 inhibits mitochondrial biogenesis in mouse liver (Sisler et al., 2015), and reported Stat2 deficiency in two children as a novel disorder of mitochondrial fission (Shahni et al., 2015). Stat1 and 2 were also reported as physically present in the mitochondria, where they may provide both sensing and signaling functions (Meier and Lamer, 2014).</div>
</li> <li> <para-num num="[0798]"> </para-num> <div class="description-line" id="p-0755" num="0798">The comprehensive model, predicting the effects of each factor on each gene, also provides a detailed view into the way in which DCs control different processes. For example, Gbp2,2b,3,4,5 and 7, a set of IFNγ regulated genes important for the response for intracellular infections by bacteria or parasites, are all positively regulated by Stat1 and 2, and negatively regulated by Rel and Irf2. Stat1 has been reported as an activator of GBPs (Ramsauer et al., 2007) and Rel is reported to bind the Gbp2 promoter (Wei et al., 2008). However, Gbp3,4,5, and 7 are also repressed by M2 (CEBPB, Rela and others), and hence are members of P4, whereas Gbp2 and 2b are not (and are members of P1). This may reflect the distinct roles of different GBPs in infections with different parasites [REF]. Moreover, perturbation of the Ml and M3 TF modules affects not only these effector genes, but also Irf8 and Batf, key transcription factors that control the GBPs (Tussiwand et al., 2012). Irf8 expression is repressed under Stat2 perturbation, suggesting that the impact on the effector genes may be at least partially indirectly mediated through Irf8 perturbation. Moreover, the TF Batf is induced during Stat2 perturbation, which may reflect the cell's attempt at compensation, since Batf can act in a compensatory capacity in the response to intracellular parasites and Leishmania (Tussiwand et al., 2012). Although both IRF8 and BATF regulate differentiation, this is unlikely to be the case here, as the regulatory effect of Stat2 on GBPs, IRF8 and is maintained—and even strengthened—in a model that accounts for cell states and controls for other differentiation effects (below).</div>
</li> <heading id="h-0038">Example 8</heading>
<heading id="h-0039">Opposing Programs of BMDC Differentiation Controlled by Two Modules each with a Distinct Pioneer Factor</heading>
<li> <para-num num="[0799]"> </para-num> <div class="description-line" id="p-0756" num="0799">Two of the TF modules—M2 (Cebpb, JunB, Rela, Stat3 and Hif1a) and M4 (Spi 1, Runx1, Irf4, and Nfkb1) had diametrically opposed effects on broad programs: P2 was repressed by factors in M2 and induced by those in M4, whereas P4 and P5 were both induced by M2 factors and repressed by M4 factors. (P4 is regulated by Stats whereas P5 is not, as Applicants discuss below.) The P4 and P5 programs each reflect key aspects of the DC response to LPS and pathogens. Conversely, the P2 program is less expected: in addition to genes for antigen presentation (MHC II system) and cytoskeleton proteins (important for DC migration after maturation), it is also enriched for ribosomal proteins. P2 also includes SerpinB6 and CD86: Applicants have previously associated SerpinB6 with “cluster disrupted cells” (Shalek et al., 2014), a sub-population in BMDCs that appears to express some maturation genes even prior to stimulation (Shalek et al., 2014) and refs within]; CD86 is known to be associated with DC maturation (Helft et al., 2015). The expression of ribosomal, cytoskeletal and MHC II proteins has been previously reported to be induced in pre-DCs along DC differentiation from progenitors [REF: PMID 26054720 and Biorxiv: biorxiv.org/content/biorxiv/early/2016/10/07/079509.full.pdf]. Several genes expressed by these pre-DCs, but not earlier progenitors are members of P2, including genes associated with early pre-DCs (1g1as3, Itgax), actin polymerization (Crip1), and late-pre-DCs (Cd74, H2-Ab 1,H2-AA, H2-Eb1). Furthermore, a recent study showed that protein synthesis is reduced upon full differentiation [Biorxiv: biorxiv.org/content/biorxiv/early/2016/10/07/079509.full.pdf]. Thus, P2 may reflect a distinct cell state or type, for example, maybe cells that are either less differentiated or abortive or arrested in their differentiation ex vivo.</div>
</li> <li> <para-num num="[0800]"> </para-num> <div class="description-line" id="p-0757" num="0800">Applicants reasoned these effects may thus be best interpreted by a global impact of some of the perturbed TFs on set of possible BMDC-like cell types or states, such that an increase at one cell type under a given perturbation, would come at the proportional decrease of others. Indeed, the BMDCs population Applicants study do have substantial heterogeneity, including the presence of distinct sub-types, such as “cluster-disrupted” cells among the Cd11c<sup>+</sup> cells (Shalek et al., 2013, 2014) as well as ˜10% of the cells that are Cd11c<sup>−</sup> (Amit 2009, Garber 2012]). For example, if a perturbation is detrimental to proper derivation of BMDCs ex vivo, it would lead to an increase in the number of cells with the active P2 program, over those that assume P4 and/or P5.</div>
</li> <li> <para-num num="[0801]"> </para-num> <div class="description-line" id="p-0758" num="0801">To test for this possibility, Applicants first used unsupervised analysis to identify a set of seven cell clusters present in 1,310 wild type cells stimulated with LPS (<figref idrefs="DRAWINGS">FIG. 50C</figref>, Methods and Resources). The clusters are significantly associated with the five programs identified by genetic perturbations, with distinct enrichments for P2 (cluster 2, 5 and 6), P3 (cluster 1) and P4/5 (clusters 0, 1, 3, 4) (<figref idrefs="DRAWINGS">FIG. 58A</figref>). This supports the hypothesis that P2 and P4/5 are states that exist in different cells in the somewhat heterogeneous BMDC population, absent perturbation, but the probability that a cell assumes any of these states over the other options may be impacted by at least some of the TF perturbations. Applicants next tested the association of each guide or targeted gene with each state by either predicting the probability that a guide associates with a state, or assigning cells to states and testing for their enrichment in the state (<figref idrefs="DRAWINGS">FIG. 50D</figref>, Methods and Resources).</div>
</li> <li> <para-num num="[0802]"> </para-num> <div class="description-line" id="p-0759" num="0802">Both methods show that cells perturbed for specific TFs are significantly associated with one or more of the cell state clusters, compared to the expected proportion of cells in that state when considering their frequencies in the WT cells?/non-targeted cells? in the same pool. In particular, cells perturbed for Cebpb, and to a lesser extent Rela or JunB, all from M2, are enriched in one or more cell clusters matching P2 and concomitantly depleted in cell clusters matching P5, whereas those perturbed for Spi1, Irf4, Nfkb1, or (to a lesser extent) Runx1, have the opposite pattern (<figref idrefs="DRAWINGS">FIG. 50D</figref>).</div>
</li> <li> <para-num num="[0803]"> </para-num> <div class="description-line" id="p-0760" num="0803">This suggests a model where factors in M2 and M4—each involving a different pioneer factor (Garber et al., 2012)—CEBPB and Spil, respectively—have mutually opposing effects on DC differentiation programs. In this model, M2 promotes differentiation, leading to LPS-responsive programs (P4 and P5), whereas M4 promotes a mutually exclusive cell state that is either less differentiated, or less productive (P2). Importantly, the two states are present in different cells in a BMDC population absent genetic perturbation. Furthermore, consistent with this model, Applicants observed the two TF modules and their effects with the same pattern prior to stimulation with LPS, when the other programs are largely not detected (<figref idrefs="DRAWINGS">FIG. 57A</figref>,B and 58B).</div>
</li> <heading id="h-0040">Example 9</heading>
<heading id="h-0041">Diametrically Opposed Programs are Generated by a Regulatory Circuit with Self-Reinforcing and Mutually Inhibitory Transcriptional Modules</heading>
<li> <para-num num="[0804]"> </para-num> <div class="description-line" id="p-0761" num="0804">Applicants next asked how is the transcriptional circuit wired to control these mutually exclusive states or types. Examining the genes in the key transcriptional programs P2 and P5 Applicants noted that they include as members their key positive and negative regulators (<figref idrefs="DRAWINGS">FIG. 51B</figref>, bottom, blue and red gene names, respectively). This would lead to both positive (reinforcing) and negative (inhibiting) feedback loops that can support the diametrically opposed, mutually exclusive behavior of M2 and M4 (<figref idrefs="DRAWINGS">FIG. 51B</figref>) and could explain how disrupting one of the factors in a TF module would ‘flip’ the cell to the other state. Specifically, IRF4, a TF in M4 is a target gene in P2, a program that M2 positively controls, generating a positive feedback loop. Conversely, Stat3, another target genes in P2, is a member of M2, generating a negative feedback loop. Similarly, Cebpb and Hif1a of M2 are members of its positive target program M5, but so is Spi1, a member of its negative regulator module, M4. The same organizational principle is present in P1, the antiviral program controlled by Ml (positively) and M3 (negatively): Stat1 is a member of the program (based on its positive regulation by Stat2; by definition, our genetic experiment cannot ascertain its auto-regulation), but so are Irf2 and Atf3, members of M3. This architecture—observed in other immune cell systems (Yosef et al., 2013); reviewed in (Yosef and Regev, 2016) may be a general principle in the DC network, resulting in balanced regulation of the cell's diverse functions, for example based on functional demands (Okabe and Medzhitov, 2014).</div>
</li> <li> <para-num num="[0805]"> </para-num> <div class="description-line" id="p-0762" num="0805">Applicants further sharpened the details of this balanced architecture by constructing a gene level genetic circuit based on the individual significant effects of the key transcription factors on each other (<figref idrefs="DRAWINGS">FIG. 51C</figref> and D). Most of the TF modules (<figref idrefs="DRAWINGS">FIG. 51D</figref>, shaded areas), have internally reinforcing positive regulation of one module TF by another (e.g., Hifl a and</div>
</li> <li> <para-num num="[0806]"> </para-num> <div class="description-line" id="p-0763" num="0806">Cebpb by each other and by JunB, Stat3, Rela (M2); Stat1 and Stat2 (M1); Spil by Irf4 and Irf4 by NfKB1 and Runx1 (M4)), and negative regulation between modules (e.g., ebpb and HIF1 a in M2 negatively regulated by Runxl and NFKB1 in M4; Rela in M4 negatively regulates Irf2 in M2; Stat3 and Rela in M4 repress Rel in M3). Importantly, there is a regulatory asymmetry in the network: some factors (e.g., Nfkb1, Rela, Runx1) regulate many others, but are not regulated themselves, whereas a few factors—Cebpb, HiflA, Spil, Irf4, Rel, Atf3—are particularly highly regulated by many others (˜6 incoming edges compared to ˜1 for the other factors). These latter factors are typically themselves members of key transcriptional programs, as noted above, and thus would account for broad-impact feedback loops.</div>
</li> <heading id="h-0042">Example 10</heading>
<heading id="h-0043">The Genetic Circuit is Strongly Supported by the Binding Patterns of Transcription Factors</heading>
<li> <para-num num="[0807]"> </para-num> <div class="description-line" id="p-0764" num="0807">To support the genetic model and relate it to underlying mechanisms, Applicants compared the effects of the individual factors to their binding patterns, which Applicants previously measured for 22 of the 24 TFs by ChIP-Seq at bulk populations of BMDCs at 0, 30, 60, and 120 min following stimulation with LPS (Garber et al., 2012). Applicants tested whether the bound targets of each individual TF (by ChIP-Seq) are more likely to be negatively or positively regulated by the TF in our model than genes that are not bound by the factor (<figref idrefs="DRAWINGS">FIG. 51E</figref>, Methods and Resources).</div>
</li> <li> <para-num num="[0808]"> </para-num> <div class="description-line" id="p-0765" num="0808">The negative effects of a TF perturbation on individual genes are strongly supported by corresponding binding of the TF to these genes' regulatory regions, and consistent with the TF's known role as an activator (<figref idrefs="DRAWINGS">FIG. 51F</figref>,G). In particular, genes bound by Rela and Cebpb, both prior to LPS stimulation and at each subsequent time point, are enriched for negative regulatory effects of the guides perturbing these factors. This effect on targets bound across the time course is consistent with the role of these factors as transcriptional activators and with the strong effect perturbing Cebpb and Rela on cell sub-populations at both unstimulated cells (<figref idrefs="DRAWINGS">FIG. 57B</figref>) and at 3 h post LPS (<figref idrefs="DRAWINGS">FIG. 50D</figref>). Furthermore, genes bound by Stat1 and Stat2 at 2 h post LPS, but not earlier, are enriched for negative regulatory effects of the guides perturbing these factors. This is consistent with the role of Stat1 and 2 as activators and with the fact that they only bind most of their targets later in the response (Garber et al., 2012).</div>
</li> <li> <para-num num="[0809]"> </para-num> <div class="description-line" id="p-0766" num="0809">Similarly, the analysis strongly supports the role of factors as repressors, as the genes that they bind are enriched for positive regulatory effects of the perturbing guides (FIG. <b>51</b>F,G). These include Irf2, Atf3, and Nfkb1. Irf2 is bound to its targets both pre- and post-LPS stimulation, and its perturbation indeed induces their expression even at time 0 (<figref idrefs="DRAWINGS">FIG. 58B</figref>), both in our current study and in an earlier one using RNAi (Garber et al., 2012). The targets bound by Atf3 and preferentially induced by its perturbation are enriched for anti-viral genes (e.g., P&lt;1.32×10<sup>−6</sup>), directly supporting our model's prediction that Atf3 is a repressor of the antiviral response, and related observations in macrophages (Labzin et al., 2015), as discussed above. Finally, Nfkb1 (encoding p50) acts as a repressor for its known bound targets, suggesting that its major impact might be as a p50-p50 homodimer, a known transcriptional repressor that recruits HDAC-1 (Cartwright et al., 2016; Elsharkawy et al., 2010), rather than as part of the p65 (Rela)-p50 heterodimer, which is a transcriptional activator (Rela perturbation represses its bound target, consistent with p65′s role in activating NFkB complexes.)Applicants return to the relationship between Nfkb1 and Rela when Applicants address genetic interactions below.</div>
</li> <li> <para-num num="[0810]"> </para-num> <div class="description-line" id="p-0767" num="0810">The distinct binding patterns of many of the TFs support their specific and direct associations with three of the regulatory programs: P1, P4 and P5 (<figref idrefs="DRAWINGS">FIG. 51A</figref>,B). Compared to a stringent background of only those genes in any of the programs (rather than a genome-wide background), genes bound by the Ml factors, Stat1 and 2, are enriched in P1 and P4 (both positively regulated by Ml); genes bound by Atf3 and Irf2 (from M3) are enriched in P1 (negatively regulated by M3, and consistent with the role of the two factors as repressors; above); genes bound by Atf3 are also enriched in P4 (including key targets such as IL-6, where Atf3 is known to act as a repressor in the TLR4/LPS response in macrophages [PMID: 16688168]); Stat3 and Rela targets are enriched in P4, and Cebpb targets are enriched in P5, both programs positively regulated by their module M2. M4 factors Irf4 and Runxl, and to a lesser extent NFKB1, are enriched for bound targets in P4, and NFKB1 targets are also weakly enriched in its repressed target program P5.</div>
</li> <li> <para-num num="[0811]"> </para-num> <div class="description-line" id="p-0768" num="0811">The remaining two programs do not show such enrichments for bound TF targets. The mitochondrial P3 program is not enriched for targets bound by any of the factors, even when comparing to a more lenient, genome-wide background (<figref idrefs="DRAWINGS">FIG. 58C</figref>). This supports a model where Stat1 and 2 do not regulate these genes directly, but possibly through another function, such as mitochondrial signals (Meier and Lamer, 2014). This is further consistent with the fact that Stat1 and 2 perturbation affects the mitochondrial module even pre-stimulation, when Stat1 and 2 are not bound to promoters (Garber et al., 2012) and that they repress the module (while act as transcriptional activators). The P2 genes, however, are enriched for binding by many key factors when comparing to a genome-wide background genes (<figref idrefs="DRAWINGS">FIG. 58C</figref>). These include Cebpb and Spil, but also many other factors that do not regulate the program. One possibility is that binding by some of these factors (e.g., Spil in M4) does directly activate this module. Another is that the P2 program and associated cell state/type are the outcome of the competing positive and negative effects of M2 and M4 on P4/P5, such that it is an alternative default or arrested state taken by the cell instead of the differentiated or responsive state P4/5. Finally, in some cases the bound targets of TFs (e.g., Rela, SPI1) are enriched (relative to the stringent background) in programs (e.g., P1) where their perturbation does not appear to have a substantial effect on the target genes (<figref idrefs="DRAWINGS">FIG. 51A</figref> and 51G).</div>
</li> <heading id="h-0044">Example 11</heading>
<heading id="h-0045">TF-Specific Programs are Revealed Once Accounting for the Global Effects on Sub-Populations</heading>
<li> <para-num num="[0812]"> </para-num> <div class="description-line" id="p-0769" num="0812">The presence of a small number of TF modules that share global effects on the one hand, and of factor-specific effects on the other hand, motivated us to distinguish global effects on the relative proportions of sub-populations, from effects of TFs that are independent of the context of a cell type or its global state. Applicants also reasoned that absent such distinction, the unique contribution of individual TFs may be masked by entire TF modules with similar global effects. Applicants therefore introduced the assignment of the perturbed cells into the seven states (subpopulations, <figref idrefs="DRAWINGS">FIG. 50C</figref>) as covariates in the model. Once Applicants explicitly modeled these global cell states, guides targeting the same TF grouped particularly tightly together (Stat2, Stat1; Irf2, NFkB and Irf4; Rel; Rela; Irfl; and Irf2 and Egr2; <figref idrefs="DRAWINGS">FIG. 50E</figref>,F and <figref idrefs="DRAWINGS">FIG. 58D</figref>), compared to their membership in larger modules of guides with similar global profiles observed when global states were not explicitly modeled (<figref idrefs="DRAWINGS">FIG. 50A</figref>,B).</div>
</li> <li> <para-num num="[0813]"> </para-num> <div class="description-line" id="p-0770" num="0813">Rather than emphasizing the regulatory system controlling the two major cell types/states (which were now reflected in the covariates), the resulting model (<figref idrefs="DRAWINGS">FIG. 58D</figref>) highlighted subtler effects unique to one or a few factors. For example, in this model, Applicants distinguished a positive regulatory effect of Runxl and Ctcf, and a negative effect of Rel, on the expression of the antigen presentation (MHCII) system, which was not detectable before. Similarly, Applicants found a strong repressive effect of Irf2 (consistent with its role as a repressor) on the IFN gamma response. Importantly, the two models are complementary: the first emphasized global phenomena, and uncovered a densely inter-connected transcriptional circuit that controls the mutually exclusive choice between two global states or types and allows to maintain both in a mixed population. The second controlled for this major phenomenon, and thus uncovered additional, more specific effects that are independent of (and may be masked by) global cell states.</div>
</li> <heading id="h-0046">Example 12</heading>
<heading id="h-0047">Genetic Interactions Between TFs Affect Gene Expression and Global Cell States</heading>
<li> <para-num num="[0814]"> </para-num> <div class="description-line" id="p-0771" num="0814">The network Applicants infer from observing the impact of single gene perturbations is densely interconnected. Multiple TFs often impact the same gene and program (<figref idrefs="DRAWINGS">FIGS. 50 and 51</figref>), but can either have the same, opposite, or no effect on transcription when perturbed individually. These effects can be direct when the two transcription factors bind the target gene Applicants leveraged the fact that in each pooled Perturb-Seq experiment Applicants obtained cells containing more than one guide (e.g., <figref idrefs="DRAWINGS">FIG. 48D</figref>), to analyze the extent to which the observed transcriptional defect (on individual gene targets or on global cell states) in cells where two genes are perturbed deviates from what Applicants would expect based on a linear supposition (<figref idrefs="DRAWINGS">FIG. 52</figref>).</div>
</li> <li> <para-num num="[0815]"> </para-num> <div class="description-line" id="p-0772" num="0815">First, Applicants found several significant (non-additive) interaction terms for the extent to which perturbing more than one gene affected the probability of being in one of our seven cell states: that is the proportion of cells observed with the combination of guides was higher or lower than expected given the proportion of cells observed with the individual guides (<figref idrefs="DRAWINGS">FIG. 52B</figref>). For example (<figref idrefs="DRAWINGS">FIG. 52C</figref>), Runxl-NFKB1 are depleted from cells in state 4 (related to P4/5), even though each alone is either enriched (Runxl) or unchanged (NFKB1) in proportion in this state. This may be consistent with a recent report that co-expression of NFKB1 (p50) and Runxl in macrophages enhances the expression of inflammatory genes (such as Il lb from P5) [PMID 27573239]. Applicants even uncovered three-way genetic interactions, such as one between Nfkb1, Rel, and Stat2 (<figref idrefs="DRAWINGS">FIG. 52C</figref>), where cells with a 3-way perturbation have a higher probability of assuming cell state 2 (partially corresponding to the P2 program), even though neither cells carrying single nor double perturbations show a related effect. Two other 3-way combinations involving NFKB1 and Stat2 are each enriched in a different cell state: NFKB1-Stat2-Spi 1 is enriched in cell state 1 (corresponding to P3, mitochondrial biogenesis), and NFKB1-Stat2-Stat3 is enriched in cell state 3 (corresponding to P5, the inflammatory response).</div>
</li> <li> <para-num num="[0816]"> </para-num> <div class="description-line" id="p-0773" num="0816">Next, Applicants assessed the effect of genetic interactions on the expression of each gene, by learning a new model, expanding the original one to include interaction terms with two genes per term (<figref idrefs="DRAWINGS">FIG. 52A</figref>, Methods and Resources). Then, Applicants stratified the genes by each of the 27 possible categories when considering the negative, positive, or no effect of each member of the pair and their interaction (<figref idrefs="DRAWINGS">FIG. 52D</figref>,E). The 27 categories in turn belong to four major classes: additive (no interaction), synergistic, buffering, or dominant (when the two factors have opposing effects, and the interaction term enhances one of them). Finally, Applicants assessed for each pair of TFs the relative proportion of target genes that are in each of the 27 categories (<figref idrefs="DRAWINGS">FIG. 52E</figref>).</div>
</li> <li> <para-num num="[0817]"> </para-num> <div class="description-line" id="p-0774" num="0817">While some TFs participate in genetic interactions that impact many target of genes, others mostly have additive effects. In particular, most TF pairs involving one of Runxl, Irf1, Irf2, or Irf4 has mostly additive effects. In contrast, interactions that include combinations of Stat1, Stat2, Stat3, Rela, Nfkb1 and Spi 1 tend to be highly enriched for interactions: they had non-additive effects on a large proportion of their targets. Surprisingly, only a subset of these, all involving Nfkb 1 (Stat1-Nfkb 1, Stat3 -Nfkb1, Rel-Nfkb 1, Spi 1-Nfkb 1) contain buffering interactions (<figref idrefs="DRAWINGS">FIG. 52E</figref>).</div>
</li> <li> <para-num num="[0818]"> </para-num> <div class="description-line" id="p-0775" num="0818">The gene-level analysis of genetic interactions allows us to relate the presence (or absence) of interactions to the molecular mechanisms by which TFs act on their targets, as is illustrated in the case of NFKB1 and Rela (<figref idrefs="DRAWINGS">FIG. 52F</figref>). As noted herein, NFKB1 and Rela individually have opposing effects on the genes in both P4 and P5 (the programs reflecting the response to intracellular parasites and the inflammatory response): NFKB1 acts as a repressor, and Rela as an activator, ChIP-Seq profiles indicates that these affected genes are directly bound by the respective factors. The model with interaction terms shows that this group of genes (<figref idrefs="DRAWINGS">FIG. 52F</figref>, hatched boxes) splits in two: in some genes, the joint perturbation of Rela and NFKB1 leads to an additive effect (no interaction: final gene expression is a sum of the two opposing effects), in the others, there is a dominant interaction, such that the observed expression reflects the effect of NFKB1 over that of Rela. While both sets belong to the same broad programs (<figref idrefs="DRAWINGS">FIG. 52F</figref>, hatched boxes), and both are enriched for ChIP targets of both NFKB1 and Rela (<figref idrefs="DRAWINGS">FIG. 52F</figref>, right panel), only the set with the dominant interaction are co-bound by both factors (<figref idrefs="DRAWINGS">FIG. 52F</figref>, right panel). Applicants observed a very similar phenomenon—where genetic interactions between two TFs is present only in those genes that they co-bind (but not in other genes from the same program, despite their similar individual effects)-<b>13</b> for additional pairs of TFs (e.g., buffering interactions for Runxl-Rel and Irf4-Nfkb1 for P4 and P5 targets) (<figref idrefs="DRAWINGS">FIG. 59</figref>).</div>
</li> <heading id="h-0048">Example 13</heading>
<heading id="h-0049">Global Transcriptional Modules and Specific Effects of TFs in K562 Cells</heading>
<li> <para-num num="[0819]"> </para-num> <div class="description-line" id="p-0776" num="0819">To explore the generality of our approach, Applicants also performed Perturb-seq targeting TFs in K562 cells, a rapidly proliferating cell line that is not synchronized by a stimulus. The TFs were chosen to be not essential for cell viability, but are known to bind targets in K562 cells. Fitting a linear model without cell state co-variates, the TFs partition into two global modules by the correlations in their coefficients: one including EGR1, CREB1, and ETS1, and the other including GABPA, NR2C2, and ELF1, as well as a distinct response to YY1 (<figref idrefs="DRAWINGS">FIG. 53A</figref> and <figref idrefs="DRAWINGS">FIG. 60</figref>). Although some guides partitioned inconsistently between the two modules, there was an overall significant consistency between guides to the same gene, both within an experiment (<figref idrefs="DRAWINGS">FIG. 60</figref>), and across experiments conducted at different time points (<figref idrefs="DRAWINGS">FIG. 53E</figref>), as tested by growing a subset of the cells used for the promoter experiments an additional six days (<figref idrefs="DRAWINGS">FIG. 53E</figref>). (Such a later time point is not possible to study in BMDCs, a primary short-lived cell culture.)</div>
</li> <li> <para-num num="[0820]"> </para-num> <div class="description-line" id="p-0777" num="0820">As in BMDCs, Applicants explored the role of global cell states. Initial unsupervised clustering partitioned the cells to 9 “states”, reflecting biological processes such as G2/M transition, mitotic cell cycle, anaphase promoting complex, the ribosome, cholesterol biosynthesis, and hemoglobin complex (<figref idrefs="DRAWINGS">FIG. 53C</figref>). Applicants tested each perturbation for enrichment in each state (<figref idrefs="DRAWINGS">FIG. 53D</figref>) and found significant associations. For example, cells perturbed in EGR1 are depleted from State 6, enriched for hemoglobin, consistent with EGR1′s known role in this process (PMID 19597182). In another example, cells perturbed in YY1 are enriched from State 5 (enriched for cholestrerol biosynthesis genes), consistent with the role of YY1 as a repressor of key genes in this process (PMID: 17925399, 26310456, 11145566).</div>
</li> <li> <para-num num="[0821]"> </para-num> <div class="description-line" id="p-0778" num="0821">Next, Applicants fitted a model that accounted for cell states. Applicants reasoned that, unlike BMDCs, many of the states are in fact continuous (e.g., phase in the cell cycle) rather than discrete (e.g., cell types). Applicants therefore performed PCA on the unperturbed dataset, scored the cells from the Perturb-Seq experiment against those principle component (PC) scores, and introduced these state PC scores as covariates to the linear model. In the resulting model (<figref idrefs="DRAWINGS">FIG. 53B</figref>) the two global modules are far less correlated, but individual guides to the same gene are much more consistent in their effects (<figref idrefs="DRAWINGS">FIGS. 53B</figref>,F), suggesting Applicants have controlled for the shared program. This higher consistency is also observed when comparing across experiments and time points (<figref idrefs="DRAWINGS">FIG. 53F</figref>), suggesting that TF-specific effects can be reproducibly identified even if global states and cell proportions change over time.</div>
</li> <li> <para-num num="[0822]"> </para-num> <div class="description-line" id="p-0779" num="0822">Applicants used the model that accounts for cell states to explore the individual functions of TFs. For example, Applicants find a distinctive repression of mitochondrial functions upon perturbation of GABPA (<figref idrefs="DRAWINGS">FIG. 60Y</figref>) consistent with its role as a required regulator of mitochondrial biogenesis (P&lt;2.05×10<sup>−9</sup>) (PMID 24958105) and weakly supported by ChIP-Seq binding [REF]. YY1 perturbation represses oxidative phosphorylation (P&lt;1.11×10<sup>−11</sup>) (known to be regulated by YY1; PMID 12525853, and enriched for its binding by ChIP-Seq [REF]) and induces an NFkB innate immune response (P&lt;2.82×10<sup>−11</sup>) (PMID 22065573).</div>
</li> <heading id="h-0050">Example 14</heading>
<heading id="h-0051">Perturbations of Cell Cycle Regulators Reveal Distinct Profiles Associated with Similar Fitness Effects and Mitotic Arrest</heading>
<li> <para-num num="[0823]"> </para-num> <div class="description-line" id="p-0780" num="0823">Individual cells in a rapidly dividing cell line, such as K562 cells, naturally vary in their cell cycle state. This continuous spectrum is readily detected in scRNA-seq profiles (Buettner et al., 2015; Kowalczyk et al., 2015; Macosko et al., 2015). Factors that impact the cell cycle can be discovered by their impact on morphological features or key marker genes, but the fact that perturbation of two genes has the same overall phenotypic endpoint (e.g., arrest at a particular phase) does not necessarily reflect identical underlying mechanisms.</div>
</li> <li> <para-num num="[0824]"> </para-num> <div class="description-line" id="p-0781" num="0824">To address this question, Applicants targeted in K562 cells 13 genes (with 33 guides) that were previously identified by a mitotic arrest phenotype in a genome-wide arrayed imaging screen in HeLa cells (Neumann et al., 2010) (Table Si). The genes were associated with diverse phenotypes, ranging from a reduction in migration speed to “grape-like” cells with clusters of micronuclei (<figref idrefs="DRAWINGS">FIG. 60</figref>, reproduced from (Neumann et al., 2010)).</div>
</li> <li> <para-num num="[0825]"> </para-num> <div class="description-line" id="p-0782" num="0825">First, Applicants used the preponderance of cells with each PAPI to determine the fitness effects of each gene. Unlike BMDCs, where fitness effects are mild, K562 cells are rapidly proliferating and our cell cycle targets are known to affect proliferation (Neumann et al., 2010). Applicants assessed the number of cells carrying each PAPI relative to their expected frequency given the input pool with either of two complementary Bayesian probabilistic models that compute the expected probability of each guide in the resulting cell population (Methods and Resources). Applicants identified a strong proliferative advantage conferred by targeting</div>
</li> <li> <para-num num="[0826]"> </para-num> <div class="description-line" id="p-0783" num="0826">PTGER2, CABP7 and CIT, while a subset of guides targeting genes such as AURKA, TOR1AIP1, and RACGAP1 show proliferative disadvantage (<figref idrefs="DRAWINGS">FIG. 53I</figref>). (Similar analysis of the TFs in K562 cells showed a proliferative disadvantage of ELK1 and EGR1, whereas —as expected—there were few fitness effects in DCs). Surviving cells may be enriched for in-frame mutations or off target (yet compensatory) effects.</div>
</li> <li> <para-num num="[0827]"> </para-num> <div class="description-line" id="p-0784" num="0827">Applicants next asked whether these fitness effects can be explained by the genes and processes regulated by the perturbation of each gene. Applicants collapsed all guides for a given gene into a single covariate to increase our ability to discern the true impact of a gene. Applicants found distinct processes affected by those factors with positive and negative effects on fitness, but also two different routes to increased fitness. First, supervised analysis using signature gene sets for the cell cycle phases (Macosko et al., 2015), showed that perturbation of the three Aurora Kinase genes (A, B, and C) and of TOR1AIP1 (which decrease fitness) are all associated with an increase in G2/M and M signature expression (<figref idrefs="DRAWINGS">FIG. 5311</figref>). Conversely, perturbation of CABP7 and PTGER2—which increase fitness—are associated with a strong opposite effect: decrease in G2/M and M signature expression and increase in the M/G1 signature (<figref idrefs="DRAWINGS">FIG. 5311</figref>). Notably, perturbation of CIT, which also increases fitness, has an increase in Gl/S and S cell cycle states, highlighting a different route to an overall phenotype of increased fitness.</div>
</li> <li> <para-num num="[0828]"> </para-num> <div class="description-line" id="p-0785" num="0828">Furthermore, signature-free analysis of our perturbation by our gene-level model (<figref idrefs="DRAWINGS">FIG. 60</figref>) shows that perturbation of CABP7 strongly induces a program of mitochondrial respiration and biogenesis (P&lt;1.59×10<sup>−11</sup>), NFkB signaling (P&lt;2.45×10<sup>−6</sup>), and mitotic division (P&lt;2.45×10<sup>−9</sup>), consistent with the fitness advantage. Perturbation of CIT and PTGER2 repressed these global programs and instead induced the expression of genes of other phases, especially 11 histone genes induced by CIT.</div>
</li> <li> <para-num num="[0829]"> </para-num> <div class="description-line" id="p-0786" num="0829">Notably, the overall partitioning of factors based on their global effects on gene expression also mostly followed their previous reported groupings based on cell biological features in HeLa cells (<figref idrefs="DRAWINGS">FIG. 60</figref>, reproduced from (Neumann et al., 2010)): (1) CIT, PTGER2 and RACGAP1 (binuclear phenotype); (2) Cenpe and Arhgef17 (grape-like phenotype and mitotic delay); and (3) Aurorak kinases A, B, and C (proliferation and migration defects). A notable exception is CABP7, which despite a similar binuclear phenotype to that of CIT, PTGER2 and RACGAP1 has a distinct transcriptional phenotype, as described above (<figref idrefs="DRAWINGS">FIG. 60</figref>).</div>
</li> <li> <para-num num="[0830]"> </para-num> <div class="description-line" id="p-0787" num="0830">Finally, when examining the nonlinear effects associated with multiple perturbations, Applicants found several significant interactions (<figref idrefs="DRAWINGS">FIG. 60</figref>), and noted a significant nonlinear decrease in genes detected when more than three cell cycle genes are perturbed (<figref idrefs="DRAWINGS">FIG. 60D</figref>).</div>
</li> <li> <para-num num="[0831]"> </para-num> <div class="description-line" id="p-0788" num="0831">Fitness considers individual guides and synthetic lethality (Livnat model). The effect on G1/S, G2/M cell states is shown (<figref idrefs="DRAWINGS">FIG. 51C</figref>), “virtual FACS” plus Livant model. <figref idrefs="DRAWINGS">FIG. 51D</figref> shows the underlying transcriptional patterns, such that there is a correlation between imaging phenotype and transcription phenotype, such that there is a different expression signature with same imaging phenotype.</div>
</li> <heading id="h-0052">Example 15</heading>
<heading id="h-0053">A Brief Guide to the Miserly: The Effects of Perturbations on Gene Signatures and PC Scores can be Recovered with Substantially Lower Numbers of Cells and Reads</heading>
<li> <para-num num="[0832]"> </para-num> <div class="description-line" id="p-0789" num="0832">The regulatory coefficients associated with the perturbations are highly structured in each of the studied systems (<figref idrefs="DRAWINGS">FIG. 50-53</figref>), consistent with the modular organization of genes regulatory systems (Heimberg et al., 2016; Kemmeren et al., 2014; Sokolov et al., 2016). Thus, sets of target genes are affected in coherent ways across sets of perturbations. Satisfyingly, these gene sets are enriched for members of coherent biological pathways (<figref idrefs="DRAWINGS">FIG. 50-53</figref>).</div>
</li> <li> <para-num num="[0833]"> </para-num> <div class="description-line" id="p-0790" num="0833">Applicants thus reasoned that recovery of effects at a pathway level should be more robust—and could be detected even if a substantially smaller number of cells is analyzed and/or a smaller number of reads is sequenced per cell. Such a scenario would be highly favorable for researchers who may often desire to employ Perturb-seq to determine key phenotypes (as assayed by their signatures) over fine regulatory connections (such as the targets of each TF, or the wiring and logic of a TF circuit). Applicants thus quantified our ability to detect gene level regulation (<figref idrefs="DRAWINGS">FIG. 54A</figref>) vs. signature or state level regulation (<figref idrefs="DRAWINGS">FIG. 54B</figref>), when Applicants down-sample cells and reads/cell across our dataset (Applicants had sequenced each library to near-saturation to permit such analysis; Methods and Resources). In particular, Applicants considered signatures quantified in two ways: (1) cell state changes, as defined by our clusters on unperturbed cells; and (2) previously described pathways, such as the antiviral response in BMDCs (Shalek et al., 2014) (Methods and Resources).</div>
</li> <li> <para-num num="[0834]"> </para-num> <div class="description-line" id="p-0791" num="0834">Applicants found that the number of cells and reads needed to detect signature level effects (<figref idrefs="DRAWINGS">FIG. 54B</figref>, 61B-H) is indeed substantially lower than that needed to detect individual effects on target genes (<figref idrefs="DRAWINGS">FIG. 54A</figref>, and <figref idrefs="DRAWINGS">FIG. 61I-L</figref>). In fact, there is almost no change in the observed effects on signatures as Applicants down-sample to approximately half the transcripts and cells in our full dataset (148 cells/perturbation (32-671); 780 transcripts); and a Spearman correlation of 0.5 between the effects is still discernable with just 10% of the cells and 25% of the transcripts (30 cells/perturbation (6-134), 390 transcripts). In contrast, the scale required to reach confident estimates in transcriptional effects on individual target genes is higher (-100-200 cells, 500+transcripts). These estimates—approximately consistent across our three systems (primary cell and cell lines; TFs and cell cycle regulators)—should provide helpful guidelines for future researchers using Perturb-seq.</div>
</li> <heading id="h-0054">Example 16</heading>
<li> <para-num num="[0835]"> </para-num> <div class="description-line" id="p-0792" num="0835">Turning to <figref idrefs="DRAWINGS">FIG. 66</figref>, GO terms associated with the following sgRNAs: c_sgCIT_1, c_sgRACGAP1_9, c_sgPTGER2_3, c_sgPTGER2_4, c_sgOGG1_4, c_sgECT2_3, c_sgCIT_7, c_sgAURKB_4, c_sgRACGAP1_3, c_sgAURKB_6, c_sgTOR1AIP1_1, c_sgAURKA_3, c_sgOGG1_2, c_sgCENPE_4, c_sgPTGER2_2, c_INTERGENIC216151, c_INTERGENIC345439, c_sgARHGEF17_4, c_sgECT2_2, c_sgCENPE_1, c_sgOGG1_3, c_sgCENPE_2, c_sgPTGER2_1, c_sgCEP55_1, c_INTERGENIC393453, c_sgAURKC_1, c_sgAURKC_7, c_sgCEP55_4, c_sgCABP7_1, c_sgCABP7_4, c_sgCABP7_2, c_sgARHGEF17_1 are shown. The Table below describes the labeling of the figure.</div>
</li> <li> <div class="description-line" id="p-0793" num="0000">
<tables id="TABLE-US-00004" num="00004">
<patent-tables colsep="0" frame="none" pgwide="1" rowsep="0">
<table align="left" class="description-table" cols="2" colsep="0" rowsep="0" width="100%">
<thead>
<tr class="description-tr">
<td align="center" class="description-td" colspan="2" nameend="2" namest="1" rowsep="1"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">X-axis left to right</td>
<td class="description-td">y-axis bottom to top</td>
</tr>
<tr class="description-tr">
<td align="center" class="description-td" colspan="2" nameend="2" namest="1" rowsep="1"> </td>
</tr>
</thead>
<tbody><tr class="description-tr">
<td class="description-td">c_sgCIT_1</td>
<td class="description-td">chemical_synaptic_transmission</td>
</tr>
<tr class="description-tr">
<td class="description-td">c_sgRACGAP1_9</td>
<td class="description-td">double-strand_break_repair_via_homologous_recombination</td>
</tr>
<tr class="description-tr">
<td class="description-td">c_sgPTGER2_3</td>
<td class="description-td">ribosomal_small_subunit_biogenesis</td>
</tr>
<tr class="description-tr">
<td class="description-td">c_sgPTGER2_4</td>
<td class="description-td">ribosomal_large_subunit_assembly</td>
</tr>
<tr class="description-tr">
<td class="description-td">c_sgOGG1_4</td>
<td class="description-td">anchored_component_of_membrane</td>
</tr>
<tr class="description-tr">
<td class="description-td">c_sgECT2_3</td>
<td class="description-td">chromosome</td>
</tr>
<tr class="description-tr">
<td class="description-td">c_sgCIT_7</td>
<td class="description-td">error-prone_translesion_synthesis</td>
</tr>
<tr class="description-tr">
<td class="description-td">c_sgAURKB_4</td>
<td class="description-td">positive_regulation_of_viral_transcription</td>
</tr>
<tr class="description-tr">
<td class="description-td">c_sgRACGAP1_3</td>
<td class="description-td">transmembrane_signaling_receptor_activity</td>
</tr>
<tr class="description-tr">
<td class="description-td">c_sgAURKB_6</td>
<td class="description-td">oxidative_phosphorylation</td>
</tr>
<tr class="description-tr">
<td class="description-td">c_sgTOR1AIP1_1</td>
<td class="description-td">ribosomal_large_subunit_biogenesis</td>
</tr>
<tr class="description-tr">
<td class="description-td">c_sgAURKA_3</td>
<td class="description-td">exon-exon_junction_complex</td>
</tr>
<tr class="description-tr">
<td class="description-td">c_sgOGG1_2</td>
<td class="description-td">serine-type_endopeptidase_activity</td>
</tr>
<tr class="description-tr">
<td class="description-td">c_sgCENPE_4</td>
<td class="description-td">heparin_binding</td>
</tr>
<tr class="description-tr">
<td class="description-td">c_sgPTGER2_2</td>
<td class="description-td">hemoglobin_complex</td>
</tr>
<tr class="description-tr">
<td class="description-td">c_INTERGENIC216151</td>
<td class="description-td">regulation_of_ion_transmembrane_transport</td>
</tr>
<tr class="description-tr">
<td class="description-td">c_INTERGENIC345439</td>
<td class="description-td">visual_perception</td>
</tr>
<tr class="description-tr">
<td class="description-td">c_sgARHGEF17_4</td>
<td class="description-td">potassium_ion_transmembrane_transport</td>
</tr>
<tr class="description-tr">
<td class="description-td">c_sgECT2_2</td>
<td class="description-td">growth_factor_activity</td>
</tr>
<tr class="description-tr">
<td class="description-td">c_sgCENPE_1</td>
<td class="description-td">7-methylguanosine_mRNA_capping</td>
</tr>
<tr class="description-tr">
<td class="description-td">c_sgOGG1_3</td>
<td class="description-td">snRNA_transcription_from_RNA_polymerase_II_promoter</td>
</tr>
<tr class="description-tr">
<td class="description-td">c_sgCENPE_2</td>
<td class="description-td">chromosome<sub>——</sub>centromeric_region</td>
</tr>
<tr class="description-tr">
<td class="description-td">c_sgPTGER2_1</td>
<td class="description-td">chromosome_segregation</td>
</tr>
<tr class="description-tr">
<td class="description-td">c_sgCEP55_1</td>
<td class="description-td">endosomal_transport</td>
</tr>
<tr class="description-tr">
<td class="description-td">c_INTERGENIC393453</td>
<td class="description-td">mitotic_metaphase_plate_congression</td>
</tr>
<tr class="description-tr">
<td class="description-td">c_sgAURKC_1</td>
<td class="description-td">aerobic_respiration</td>
</tr>
<tr class="description-tr">
<td class="description-td">c_sgAURKC_7</td>
<td class="description-td">viral_life_cycle</td>
</tr>
<tr class="description-tr">
<td class="description-td">c_sgCEP55_4</td>
<td class="description-td">mitochondrion_organization</td>
</tr>
<tr class="description-tr">
<td class="description-td">c_sgCABP7_1</td>
<td class="description-td">DNA_duplex_unwinding</td>
</tr>
<tr class="description-tr">
<td class="description-td">c_sgCABP7_4</td>
<td class="description-td">postsynaptic_membrane</td>
</tr>
<tr class="description-tr">
<td class="description-td">c_sgCABP7_2</td>
<td class="description-td">small_nuclear_ribonucleoprotein_complex</td>
</tr>
<tr class="description-tr">
<td class="description-td">c_sgARHGEF17_1</td>
<td class="description-td">telomere_maintenance_via_recombination</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">DNA_replication_initiation</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">cell_proliferation</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">cell_cycle</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">proteasome_regulatory_particle</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">proteasome_regulatory_particle<sub>——</sub>base_subcomplex</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">COPII_vesicle_coating</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">formation_of_translation_preinitiation_complex</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">thioredoxin_peroxidase_activity</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">double-stranded_RNA_binding</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">intracellular_protein_transport</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">toxin_transport</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">nuclear_matrix</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">vesicle</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">nucleotide-excision_repair<sub>——</sub>DNA_incision<sub>——</sub>5′-to_lesion</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">nucleotide-excision_repair<sub>——</sub>DNA_incision</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">error-free_translesion_synthesis</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">signal_transduction</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">mitochondrial_ribosome</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">nucleosomal_DNA_binding</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">DNA_damage_response<sub>——</sub>detection_of_DNA_damage</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">damaged_DNA_binding</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">G1_S_transition_of_mitotic_cell_cycle</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">mitochondrial_proton-transporting_ATP_synthase_complex</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">ATP_biosynthetic_process</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">mitochondrial_nucleoid</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">mitochondrial_small_ribosomal_subunit</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">mitochondrial_translation</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">mitochondrial_large_ribosomal_subunit</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">mitochondrial_intermembrane_space</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">gene_expression</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">nuclear_import</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">SMN-Sm_protein_complex</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">spliceosomal_complex_assembly</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">protein_peptidyl-prolyl_isomerization</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">peptidyl-prolyl_cis-trans_isomerase_activity</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">U5_snRNP</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">ribonucleoprotein_complex_binding</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">RNA_splicing<sub>——</sub>via_transesterification_reactions</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">U4_U6_x_U5_tri-snRNP_complex</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">preribosome<sub>——</sub>large_subunit_precursor</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">maturation_of_SSU-rRNA_from_tricistronic_rRNA_transcript_(SSU-rRNA<sub>——</sub>5.8S_small-subunit_processome</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">positive_regulation_of_GTPase_activity</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">response_to_unfolded_protein</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">tRNA_binding</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">GTPase_activity</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">GTP_binding</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">GDP_binding</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">receptor_complex</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">tricarboxylic_acid_cycle</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">cell_redox_homeostasis</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">NAD_binding</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">electron_carrier_activity</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">regulation_of_translational_initiation</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">nucleotide-excision_repair<sub>——</sub>DNA_damage_recognition</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">spliceosomal_snRNP_assembly</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">U2_snRNP</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">precatalytic_spliceosome</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">chromatin</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">methylosome</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">U1_snRNP</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">proteasome_assembly</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">DNA_recombination</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">telomere_maintenance</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">proteasome_core_complex<sub>——</sub>alpha-subunit_complex</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">positive_regulation_of_protein_localization_to_Cajal_body</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">chaperonin-containing_T-complex</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">zona_pellucida_receptor_complex</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">positive_regulation_of_telomere_maintenance_via_telomerase</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">positive_regulation_of_establishment_of_protein_localization_to_telomere</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">positive_regulation_of_telomerase_RNA_localization_to_Cajal_body</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">tRNA_aminoacylation_for_protein_translation</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">negative_regulation_of_apoptotic_process</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">proteasome_accessory_complex</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">ATPase_activity</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">ubiquitin-dependent_protein_catabolic_process</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">identical_protein_binding</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">endoplasmic_reticulum-Golgi_intermediate_compartment</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">protein_stabilization</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">sister_chromatid_cohesion</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">condensed_chromosome_kinetochore</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">protein_transport</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">spindle_pole</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">kinetochore</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">DNA_damage_response<sub>——</sub>signal_transduction_by_p53_class_mediator_resulting_retrograde_vesicle-</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">mediated_transport<sub>——</sub>Golgi_to_ER</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">ER_to_Golgi_vesicle-mediated_transport</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">transcription_coactivator_activity</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">protein_complex_binding</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">intracellular_membrane-bounded_organelle</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">regulation_of_signal_transduction_by_p53_class_mediator</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">regulation_of_cellular_response_to_heat</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">mitotic_nuclear_envelope_disassembly</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">intracellular_transport_of_virus</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">protein_sumoylation</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">RNA_processing</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">ER-associated_ubiquitin-dependent_protein_catabolic_process</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">nuclear_envelope</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">covalent_chromatin_modification</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">nuclear_membrane</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">protein_complex</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">nuclear_chromatin</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">spindle</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">G2_M_transition_of_mitotic_cell_cycle</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">ubiquitin_protein_ligase_binding</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">midbody</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">endoplasmic_reticulum_membrane</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">endoplasmic_reticulum</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">ATP_binding</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">chromatin_binding</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">protein_kinase_binding</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">regulation_of_transcription_from_RNA_polymerase_II_promoter</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">ligase_activity</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">enzyme_binding</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">Golgi_apparatus</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">ubiquitin_protein_ligase_activity</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">ubiquitin-protein_transferase_activity</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">transcription_initiation_from_RNA_polymerase_II_promoter</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">cellular_response_to_DNA_damage_stimulus</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">DNA_binding</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">transcription<sub>——</sub>DNA-templated</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">negative_regulation_of_transcription_from_RNA_polymerase_II_promoter</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">transcription_factor_binding</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">zinc_ion_binding</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">focal_adhesion</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">cytokine_activity</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">cell_surface</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">ribosome</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">small_ribosomal_subunit</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">cytoplasmic_translation</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">nucleus</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">protein_binding</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">nucleoplasm</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">extracellular_region</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">poly(A)_RNA_binding</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">cytosol</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">cytoplasm</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">membrane</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">nucleolus</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">integral_component_of_plasma_membrane</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">G-protein_coupled_receptor_activity</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">G-protein_coupled_receptor_signaling_pathway</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">extracellular_space</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">mitochondrion</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">plasma_membrane</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">SRP-dependent_cotranslational_protein_targeting_to_membrane</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">nuclear-transcribed_mRNA_catabolic_process<sub>——</sub>nonsense-mediated_decay</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">viral_transcription</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">cytosolic_small_ribosomal_subunit</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">cytosolic_large_ribosomal_subunit</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">translation</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">translational_initiation</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">rRNA_processing</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">structural_constituent_of_ribosome</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">homophilic_cell_adhesion_via_plasma_membrane_adhesion_molecules</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">cell_adhesion</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">calcium_ion_binding</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">proteinaceous_extracellular_matrix</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">cell_division</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">mitotic_nuclear_division</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">centrosome</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">cell-cell_adherens_junction</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">cell-cell_adhesion</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">cadherin_binding_involved_in_cell-cell_adhesion</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">viral_process</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">nucleotide_binding</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">mRNA_processing</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">nuclear_speck</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">termination_of_RNA_polymerase_II_transcription</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">mRNA_3′-end_processing</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">nuclear_chromosome<sub>——</sub>telomeric_region</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">mRNA_binding</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">single-stranded_DNA_binding</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">RNA_export_from_nucleus</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">transcription-coupled_nucleotide-excision_repair</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">external_side_of_plasma_membrane</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">U12-type_spliceosomal_complex</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">extracellular_matrix_organization</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">intracellular_ribonucleoprotein_complex</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">transcription_elongation_from_RNA_polymerase_II_promoter</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">mitochondrial_respiratory_chain_complex_I</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">mitochondrial_electron_transport<sub>——</sub>NADH_to_ubiquinone</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">NADH_dehydrogenase_(ubiquinone)_activity</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">mitochondrial_respiratory_chain_complex_I_assembly</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">cell-cell_signaling</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">DNA_repair</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">DNA_replication</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">immune_response</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">inflammatory_response</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">mRNA_export_from_nucleus</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">mRNA_splicing<sub>——</sub>via_spliceosome</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">RNA_splicing</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">mitochondrial_inner_membrane</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">mitochondrial_matrix</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">spliceosomal_complex</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">mitochondrial_translational_elongation</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">mitochondrial_translational_termination</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">RNA_binding</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">integral_component_of_membrane</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">negative_regulation_of_ubiquitin-protein_ligase_activity_involved_in_mitotic_cell_anaphase-</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">promoting_complex-dependent_catabolic_process</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">positive_regulation_of_ubiquitin-protein_ligase_activity_involved_in_regulation_of_proteasome_complex</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">NIK_NF-kappaB_signaling</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">regulation_of_cellular_amino_acid_metabolic_process</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">catalytic_step_2_spliceosome</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">regulation_of_mRNA_stability</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">proteasome-mediated_ubiquitin-dependent_protein_catabolic_process</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">protein_polyubiquitination</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">extracellular_exosome</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">stimulatory_C-type_lectin_receptor_signaling_pathway</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">Fc-epsilon_receptor_signaling_pathway</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">Wnt_signaling_pathway<sub>——</sub>planar_cell_polarity_pathway</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">antigen_processing_and_presentation_of_exogenous_peptide_antigen_via_MHC_negative_regulation_of_canon-</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">ical_Wnt_signaling_pathway</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">T_cell_receptor_signaling_pathway</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">positive_regulation_of_canonical_Wnt_signaling_pathway</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">tumor_necrosis_factor-mediated_signaling_pathway</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">threonine-type_endopeptidase_activity</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">proteasome_core_complex</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">translation_initiation_factor_activity</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">protein_folding</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">unfolded_protein_binding</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">myelin_sheath</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">melanosome</td>
</tr>
<tr class="description-tr">
<td align="center" class="description-td" colspan="2" nameend="2" namest="1" rowsep="1"> </td>
</tr>
</tbody></table>
</patent-tables>
</tables>
</div>
</li> <li> <para-num num="[0836]"> </para-num> <div class="description-line" id="p-0794" num="0836">Whereas the nature of the experimental system minimizes batch effects when comparing perturbations, as all cells are grown in one pool, applicants sought to determine to what extent the effects observed are reproducible across when compared to a later time point. A subset of the cells used for the promoter experiments were grown for seven more days, and then assayed for expression in the same manner.</div>
</li> <li> <para-num num="[0837]"> </para-num> <div class="description-line" id="p-0795" num="0837">Reports suggest that CRISPRi uniformly reduces expression of a gene across all cells, while the CRISPR-KO system creates an assortment of heterogeneous indel mutations. In order to compare these two systems, applicants perturbed three of the same genes using four sgRNAs per gene with the CRISPRi system.</div>
</li> <li> <para-num num="[0838]"> </para-num> <div class="description-line" id="p-0796" num="0838">Example of cost for conditions and cells:</div>
</li> <li> <div class="description-line" id="p-0797" num="0000">
<tables id="TABLE-US-00005" num="00005">
<patent-tables colsep="0" frame="none" rowsep="0">
<table align="left" class="description-table" cols="5" colsep="0" rowsep="0" width="100%">
<thead>
<tr class="description-tr">
<td align="center" class="description-td" colspan="5" nameend="5" namest="1" rowsep="1"> </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">Conditions -</td>
<td class="description-td">Conditions -</td>
<td class="description-td"> </td>
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">low</td>
<td class="description-td">high</td>
<td class="description-td">Cost ($) -</td>
<td class="description-td">Cost -high</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">(800 cells/</td>
<td class="description-td">(80 cells/</td>
<td class="description-td">low (1000</td>
<td class="description-td">(5,000</td>
</tr>
<tr class="description-tr">
<td class="description-td">Cells</td>
<td class="description-td">condition)</td>
<td class="description-td">condition)</td>
<td class="description-td">reads/cell)</td>
<td class="description-td">reads/cell)</td>
</tr>
<tr class="description-tr">
<td align="center" class="description-td" colspan="5" nameend="5" namest="1" rowsep="1"> </td>
</tr>
</thead>
<tbody><tr class="description-tr">
<td class="description-td"> </td>
</tr>
</tbody></table>
<table align="left" class="description-table" cols="5" colsep="0" rowsep="0" width="100%">
<tbody><tr class="description-tr">
<td class="description-td">10,000</td>
<td class="description-td">12.5</td>
<td class="description-td">XXX</td>
<td class="description-td">250</td>
<td class="description-td">1,250</td>
</tr>
<tr class="description-tr">
<td class="description-td">50,000</td>
<td class="description-td">62.5</td>
<td class="description-td">  625*</td>
<td class="description-td">1,250</td>
<td class="description-td">6,250</td>
</tr>
<tr class="description-tr">
<td class="description-td">100,000</td>
<td class="description-td">125  </td>
<td class="description-td">1,250*</td>
<td class="description-td">2,500</td>
<td class="description-td">12,000</td>
</tr>
<tr class="description-tr">
<td class="description-td">500,000</td>
<td class="description-td">625*  </td>
<td class="description-td">6,250*</td>
<td class="description-td">12,500</td>
<td class="description-td">62,500</td>
</tr>
<tr class="description-tr">
<td class="description-td">1,000,000</td>
<td class="description-td">1,250*  </td>
<td class="description-td">12,500* </td>
<td class="description-td">25,000</td>
<td class="description-td">125,000</td>
</tr>
<tr class="description-tr">
<td class="description-td">10,000,000</td>
<td class="description-td">12,500*  </td>
<td class="description-td">125,000* </td>
<td class="description-td">250,000</td>
<td class="description-td">1.25</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">million</td>
</tr>
<tr class="description-tr">
<td align="center" class="description-td" colspan="5" nameend="5" namest="1" rowsep="1"> </td>
</tr>
<tr class="description-tr">
<td align="left" class="description-td" colspan="5" id="FOO-00001" nameend="5" namest="1">*significantly more than previously feasible (100's of conditions)</td>
</tr>
</tbody></table>
</patent-tables>
</tables>
</div>
</li> <heading id="h-0055">Example 17</heading>
<heading id="h-0056">Vectors for Capturing Guide RNAs in Single Cells and RT Primers for Labelling Transcripts</heading>
<li> <para-num num="[0839]"> </para-num> <div class="description-line" id="p-0798" num="0839">Assays typically measure either a great number of cells, or a great number of genes. RNA-seq allows for the measuring of the expression of every gene in the genome, but can only get average behavior. High content microscopy/flow cytometry allows for the measuring of many individual cells, but only a handful of reporters. Single-cell resolution exposes new phenotypes and the present invention allows for determining rich phenotypes for each perturbation, thus enabling mechanistic understanding and functional clustering (<figref idrefs="DRAWINGS">FIG. 68</figref>). Before the present invention guide RNAs could not be easily captured. Applicants developed a novel vector for measuring the guide RNA in a single cell that may be distinguished from guide RNAs present in the pool of cells of the present invention. The vector relies on the capture of a reporter transcript having a unique guide RNA specific barcode (<figref idrefs="DRAWINGS">FIG. 70</figref>).</div>
</li> <li> <para-num num="[0840]"> </para-num> <div class="description-line" id="p-0799" num="0840">To build a highly parallel platform for conducting functional genetic studies with complex phenotypes (defined by single cell expression profiling), Applicants developed a simple method for encoding the identity of genetic perturbations into the DNA of individual cells (<figref idrefs="DRAWINGS">FIG. 81A-B</figref>). These DNA “barcodes” can be expressed as mRNAs and, therefore, captured during single-cell RNA-sequencing library preparation (<figref idrefs="DRAWINGS">FIG. 81A-B</figref>). This strategy is broadly applicable because our barcode design allows the encoding of different pieces of information; however, here Applicants focus on the case where the barcode encodes the identity of a perturbation made using CRISPRi/a (23849981, 25307932). In this system, CRISPR guide RNAs (sgRNAs) are delivered to cells using a lentiviral delivery vector, called the perturb-seq vector, that carries tandem expression cassettes: one encoding an sgRNA under control of a polymerase III-driven promoter and the other encoding a polymerase II-driven mRNA (<figref idrefs="DRAWINGS">FIG. 81B</figref>). This mRNA carries a guide-mapping barcode, or “guide barcode,” that uniquely identifies the sgRNA in the former expression cassette, as well as two cell selection markers, blue fluorescent protein (BFP) and puromycin (PURO). Because the majority of single cell RNA sequencing platforms rely on polyadenyation-mediated priming for reverse transcription and capture only the 3′ end of message RNAs in sequencing amplicons, Applicants engineered our barcode expression transcript to terminate with a strong polyadenylation signal sequence (BGH polyadenylation signal cloned from pcDNA3.1(+)) and placed the sgRNA-mapping barcode adjacent to this sequence. To incorporate this design into a functional lentiviral delivery system and prevent the BGH polyadeylation sequence from disrupting provirus expression, both the guide and barcode expression cassettes were encoded in reverse orientation with respect to the lentiviral genome. After incorporation into a target cell genome, the expressed barcode transcript can be captured by any oligo-dT-based protocol for cDNA construction. Specific amplification of the barcode transcript can then be used to enrich this important mapping information within RNA-sequencing libraries.</div>
</li> <li> <para-num num="[0841]"> </para-num> <div class="description-line" id="p-0800" num="0841">In the Drop-seq method beads are used to deliver RT primers (<figref idrefs="DRAWINGS">FIG. 70</figref>), which are used to tag the 3′ ends of transcripts with: (1) Cell barcode, a sequence common to all primers on that bead, used to group transcripts after sequencing; and (2) UMI (<u>U</u>nique <u>M</u>olecular <u>I</u>dentifier), a random sequence used to identify duplicates introduced during PCR amplification. Therefore, all of the transcripts in a cell, including the reporter transcript are tagged. The method allows one to introduce diverse perturbations using CRISPRi/a into single cells, pool, and profile responses by droplet RNAseq. Perturb-seq in droplets relies on 3 barcodes to map single cell transcriptomes and analyze expression (<figref idrefs="DRAWINGS">FIG. 72</figref>). Vectors can also be used having multiple guide RNAs. In one embodiment, three guides are expressed from a single vector (<figref idrefs="DRAWINGS">FIG. 78, 79</figref>).</div>
</li> <li> <para-num num="[0842]"> </para-num> <div class="description-line" id="p-0801" num="0842">Applicants performed pilot experiments with eight cell lines containing transcription factor knockdowns by CRISPRi, ˜6,000 cells total. The targets had minimal growth defects, but still showed phenotypes in some screens. One control guide at 3-fold higher representation was used in addition to guides for seven transcription factors: EP300, BHLHE40, CREB1, DDIT3, ZNF326, SNAI1 SPI1. Two sequencing runs were performed on the same library. A HiSeq 2500 rapid run resulted in 132 million reads total, ˜22,900 reads per cell, ˜10,000 UMIs per cell, and had 2.29X coverage. A HiSeq 4000 lane run resulted in 300 million reads, ˜52,000 reads per cell, ˜17,000 UMIs per cell, and had 3X coverage. The merged data set resulted in 75,000 reads per cell, <b>18</b> 20,700 UMIs per cell, and 3.6X coverage.</div>
</li> <li> <para-num num="[0843]"> </para-num> <div class="description-line" id="p-0802" num="0843">A dial-out PCR approach was designed for guide identification (<figref idrefs="DRAWINGS">FIG. 81</figref>). Transcripts containing guide barcodes are specifically amplified by PCR and prepared as separate sequencing libraries. Sequencing coverage is high (16.5 reads/UMI), so almost every molecule is detected (<figref idrefs="DRAWINGS">FIG. 74</figref>). Applicants also capture the guide barcode transcripts during RNAseq and the identities agree, but are less certain (<figref idrefs="DRAWINGS">FIG. 75</figref>).</div>
</li> <li> <para-num num="[0844]"> </para-num> <div class="description-line" id="p-0803" num="0844">Guide identification is done by consensus because most cells have a clearly dominant guide. When two guides show similar abundance, that cell barcode is ignored as a cell doublet.</div>
</li> <li> <div class="description-line" id="p-0804" num="0000">
<tables id="TABLE-US-00006" num="00006">
<patent-tables colsep="0" frame="none" pgwide="1" rowsep="0">
<table align="left" class="description-table" cols="6" colsep="0" rowsep="0" width="100%">
<thead>
<tr class="description-tr">
<td align="center" class="description-td" colspan="6" nameend="6" namest="1" rowsep="1"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">cell BC</td>
<td class="description-td">guide identity</td>
<td class="description-td">reed count</td>
<td class="description-td">UMI count</td>
<td class="description-td">coverage</td>
<td class="description-td">good coverage</td>
</tr>
<tr class="description-tr">
<td align="center" class="description-td" colspan="6" nameend="6" namest="1" rowsep="1"> </td>
</tr>
</thead>
<tbody><tr class="description-tr">
<td class="description-td"> </td>
</tr>
</tbody></table>
<table align="left" class="description-table" cols="6" colsep="0" rowsep="0" width="100%">
<tbody><tr class="description-tr">
<td class="description-td">AGCCTCACAGGCGA-1</td>
<td class="description-td">*</td>
<td class="description-td">132</td>
<td class="description-td">86</td>
<td class="description-td">1.534884</td>
<td class="description-td">False</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">EP300_pDS268</td>
<td class="description-td">3888</td>
<td class="description-td">243</td>
<td class="description-td">16.000000</td>
<td class="description-td">True</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">ZNF326_pDS262</td>
<td class="description-td">629</td>
<td class="description-td">37</td>
<td class="description-td">17.000000</td>
<td class="description-td">True</td>
</tr>
<tr class="description-tr">
<td class="description-td">ATAGCGTGCAACCA-1</td>
<td class="description-td">*</td>
<td class="description-td">87</td>
<td class="description-td">67</td>
<td class="description-td">1.298507</td>
<td class="description-td">False</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">62(mod)_pBA581</td>
<td class="description-td">3906</td>
<td class="description-td">213</td>
<td class="description-td">18.338028</td>
<td class="description-td">True</td>
</tr>
<tr class="description-tr">
<td align="center" class="description-td" colspan="6" nameend="6" namest="1" rowsep="1"> </td>
</tr>
</tbody></table>
</patent-tables>
</tables>
</div>
</li> <li> <para-num num="[0845]"> </para-num> <div class="description-line" id="p-0805" num="0845">Guide identification is illustrated the following in test experiment.</div>
</li> <li> <div class="description-line" id="p-0806" num="0000">
<tables id="TABLE-US-00007" num="00007">
<patent-tables colsep="0" frame="none" rowsep="0">
<table align="left" class="description-table" cols="4" colsep="0" rowsep="0" width="100%">
<thead>
<tr class="description-tr">
<td align="center" class="description-td" colspan="4" nameend="4" namest="1" rowsep="1"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">rank</td>
<td class="description-td">guide identity</td>
<td class="description-td">number of cells</td>
<td class="description-td">percentage</td>
</tr>
<tr class="description-tr">
<td align="center" class="description-td" colspan="4" nameend="4" namest="1" rowsep="1"> </td>
</tr>
</thead>
<tbody><tr class="description-tr">
<td class="description-td"> </td>
</tr>
</tbody></table>
<table align="left" class="description-table" cols="4" colsep="0" rowsep="0" width="100%">
<tbody><tr class="description-tr">
<td class="description-td">1</td>
<td class="description-td">62(mod)_pBA581</td>
<td class="description-td">1552</td>
<td class="description-td">28.22</td>
</tr>
<tr class="description-tr">
<td class="description-td">2</td>
<td class="description-td">SPI1_pDS255</td>
<td class="description-td">600</td>
<td class="description-td">10.91</td>
</tr>
<tr class="description-tr">
<td class="description-td">3</td>
<td class="description-td">EP300_pDS268</td>
<td class="description-td">525</td>
<td class="description-td">9.55</td>
</tr>
<tr class="description-tr">
<td class="description-td">4</td>
<td class="description-td">ZNF326_pDS262</td>
<td class="description-td">492</td>
<td class="description-td">8.95</td>
</tr>
<tr class="description-tr">
<td class="description-td">5</td>
<td class="description-td">SNAI1_pDS266</td>
<td class="description-td">477</td>
<td class="description-td">8.67</td>
</tr>
<tr class="description-tr">
<td class="description-td">6</td>
<td class="description-td">BHLHE40_pDS258</td>
<td class="description-td">472</td>
<td class="description-td">8.58</td>
</tr>
<tr class="description-tr">
<td class="description-td">7</td>
<td class="description-td">CREB1_pDS269</td>
<td class="description-td">436</td>
<td class="description-td">7.93</td>
</tr>
<tr class="description-tr">
<td class="description-td">8</td>
<td class="description-td">DDIT3_pDS263</td>
<td class="description-td">410</td>
<td class="description-td">7.45</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">*</td>
<td class="description-td">4</td>
<td class="description-td">0.07</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">Multiplets</td>
<td class="description-td">353</td>
<td class="description-td">6.42</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">Total uniquely identified</td>
<td class="description-td">4964</td>
<td class="description-td">90.25</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">Total unidentifiable</td>
<td class="description-td">183</td>
<td class="description-td">3.33</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">Total number of cells</td>
<td class="description-td">5500</td>
<td class="description-td">100.00</td>
</tr>
<tr class="description-tr">
<td align="center" class="description-td" colspan="4" nameend="4" namest="1" rowsep="1"> </td>
</tr>
</tbody></table>
</patent-tables>
</tables>
</div>
</li> <li> <para-num num="[0846]"> </para-num> <div class="description-line" id="p-0807" num="0846">The guide barcodes also permit overloading (<figref idrefs="DRAWINGS">FIG. 76</figref>). If barcodes are sufficiently diverse and recoverable with high confidence, then one can overload and discard doubles, triples, etc. The peak loading possible is 37.8%, although there is a “sweet spot” with sequencing costs.</div>
</li> <li> <para-num num="[0847]"> </para-num> <div class="description-line" id="p-0808" num="0847">Single cell sequencing may be performed by any method known in the art and each has advantages and disadvantages (<figref idrefs="DRAWINGS">FIG. 80</figref>).</div>
</li> <heading id="h-0057">Example 18</heading>
<heading id="h-0058">Discussion</heading>
<li> <para-num num="[0848]"> </para-num> <div class="description-line" id="p-0809" num="0848">Here, Applicants developed Perturb-seq a method to analyze the transcriptional defect associated with genetic manipulations as a readout of the cell's phenotype and the architecture of its molecular circuitry—from individual genes to specific processes to global states or types. Perturb-seq dramatically decreases the time and cost limitations associated with assaying the effects of large numbers of perturbations. By decomposing expression into the effects due to technical vs. biological covariates, Applicants recover expected expression changes and uncover new effects, from the effect on individual genes to cell states. Because of the single cell nature of the experiment, Perturb-seq can analyze heterogeneity in cell populations—distinguishing transcriptional effects associated with changes in cell state proportions from transcriptional responses within states.</div>
</li> <li> <para-num num="[0849]"> </para-num> <div class="description-line" id="p-0810" num="0849">The present invention shows novel methods to improve scalability, single cell phenotypes associated with perturbations, and interactions. As both the cost of both sequencing and high throughput library preparation methods decrease, the experimental and computational methods applicants have outlined should become versatile tools in armamentarium of the systems biologist. By sequencing shallowly across large numbers of cells, major transcriptional defects can be readily identified and, by sequencing deeper, smaller changes become apparent. In this invention, applicants exceeded the necessary scale to provide the community with a necessary resource to design future experiments. Excitingly, the signature analysis applicants describe, with as few cells as the number of cells that were analyzed in this work, could be used to gain a broad survey of transcriptional phenotypes across thousands of genes.</div>
</li> <li> <para-num num="[0850]"> </para-num> <div class="description-line" id="p-0811" num="0850">In certain examples, a diverse set of RNA barcodes can be used as an error correcting mechanism when more than one guide is present in a cell and sufficient clonality exists in the population. This feature can be leveraged in future studies to more intensively study the effects of combinations of perturbations and condition our regression framework on clonal similarity. Greater than 1 guide per cell can inform measurements of 1 guide per cell. Greater than 1 cell/drop can reduce costs by an order of magnitude.</div>
</li> <heading id="h-0059">Example 19</heading>
<heading id="h-0060">Perturb-seq and MIMSOCA Correctly Determine Broad Phenotypic and Gene Specific Effects of Individual Perturbations and Genetic Interactions</heading>
<li> <para-num num="[0851]"> </para-num> <div class="description-line" id="p-0812" num="0851">Perturb-Seq analysis of a set of TFs in BMDCs demonstrated the accuracy and resolution of our approach, as well as made specific new discoveries in this cell type. The grouping of individual guides and sets of TFs into TF modules, and many of the genes and processes that perturb-seq and our MIMOSCA framework predicted they regulate were coherent with respect to the extensive knowledge in the system. The specific regulatory connections were strongly supported by ChIP-Seq data. In addition, Applicants uncovered new and emerging regulatory connections, such as between Stat1 and 2 and mitochondrial biogenesis and Atf3, Irf2 and Rel and anti-viral responses, and showed that most programs induce both positive and negative feedback loops.</div>
</li> <li> <para-num num="[0852]"> </para-num> <div class="description-line" id="p-0813" num="0852">Because this novel approach supports such detailed dissection without pre-sorting the cells based on specific marker or state, Applicants could uncover two diametrically opposed programs that govern two global cell states, which precede stimulation with LPS, and may reflect distinct or mutually exclusive differentiation programs. The specific functional nature of “cluster disrupted cells” has been debated, including a suggestion that the two states represent dendritic cells (“cluster disrupted”; here, P2) and macrophages (“non cluster disrupted”, here P4/5). Our result suggest that both states not only co-exist in unperturbed cells, but are coupled through a circuit with a specific set of positive and negative feedback loops, such that genetic perturbations (in single TFs or their combinations) that enhance one state inhibit the other and vice versa. The specific genes regulated in P2 include many recently highlighted in pre-DC cells, and may thus represent a less differentiated, arrested or abortive state.</div>
</li> <heading id="h-0061">Example 20</heading>
<heading id="h-0062">Alternative Embodiments to Enhance the Precision and Facility of Perturb-Seq</heading>
<li> <para-num num="[0853]"> </para-num> <div class="description-line" id="p-0814" num="0853">Several enhancements can further improve Perturb-Seq. First, including a veritable cornucopia of sgRNAs that are non-targeting or target intergenic regions along with nontransduced controls in the same experiment would help create a stronger reference from which to evaluate the effect of perturbations.</div>
</li> <li> <para-num num="[0854]"> </para-num> <div class="description-line" id="p-0815" num="0854">Second, an important advantage of using Perturb-seq is that higher order interactions can be reached without further need to generate complex reagents (such as new vectors). The same experiment used for a single perturbation, due to the Poisson loading of perturbation per cell, can also uncover the genetic interactions between the perturbed genes. However, the number of sgRNAs present in each cell can vary over a large range. Applicants can leverage the cassette structure of the endogenous bacterial Cas9 crRNAs (Makarova et al., 2011) and the ability of Cpf1 to autonomously process an entire array, and deliver several sgRNAs (or an unprocessed array) on one construct (Kabadi et al., 2014; Wong et al., 2016; Zetsche et al., 2015, 2016). Such a system has advantages, such as having all members of the cassette be jointly detected or not, but depending on implementation disadvantages as well, such as cloning difficulty, and oligonucleotide synthesis scale.</div>
</li> <li> <para-num num="[0855]"> </para-num> <div class="description-line" id="p-0816" num="0855">At its current scale, given the required number of cells and depth, and with reasonable estimates for the current cost of single cell assays (<figref idrefs="DRAWINGS">FIG. 54C</figref>), Perturb-seq can be readily applied for targeted screens of a subset of genes of interest and their interactions, as Applicants have done here for TFs and cell cycle regulators. In some systems, growth or marker based screens may first be performed to identify this subset upstream of Perturb-seq (as done for the unfolded protein response described herein as Applicants have done earlier in BMDCs (Parnas et al., 2015)). The scale of Perturb-seq, will increase as both the cost of both sequencing and of high throughput scRNA-seq preparation methods continues to decrease (<figref idrefs="DRAWINGS">FIG. 54C</figref>).</div>
</li> <li> <para-num num="[0856]"> </para-num> <div class="description-line" id="p-0817" num="0856">A researcher can balance the key parameters of the screens based on their wish to focus on major transcriptional phenotypes (cell states or signatures) vs. effects on individual genes' expression, by varying the number of surveyed cells and the sequencing depth. Major transcriptional defects can be readily identified when sequencing shallowly across large numbers of cells; by sequencing deeper (and more cells per perturbation), finer effects become readily apparent. Applicants purposefully exceeded in this study the sample size and sequencing depth required to reach saturation, thus providing the community with a necessary resource to design future experiments. Surprisingly, the signature analysis Applicants describe showed that with as few as ˜10 cells per perturbation, a broad survey of transcriptional phenotypes across thousands of perturbations can be performed (<figref idrefs="DRAWINGS">FIG. 54B</figref>).</div>
</li> <li> <para-num num="[0857]"> </para-num> <div class="description-line" id="p-0818" num="0857">As cell numbers increase towards genome-wide or large combinatorial screens, Applicants will also need a concomitant increase in the bandwidth of computational methods. In our analysis framework, MIMOSCA (<figref idrefs="DRAWINGS">FIG. 5611</figref>), Applicants used a range of efficient algorithms, such as Facebook's Fast PCA implementation (research.facebook.com/blog/fast-randomized-svd), Infomap clustering (Rosvall and Bergstrom, 2008), parallelizable regression frameworks, and binned permutation significance testing, foreseeing a need for scalability towards millions of cells. In future extensions, when the total number of perturbations is larger, expression matrices can be stored in chunked form, split by perturbation identity, and the relevant subsets can be read, combined with control cells, and analyzed in parallel.</div>
</li> <heading id="h-0063">Example 21</heading>
<heading id="h-0064">Challenges and Opportunities for Understanding the Vast Space of Possible Genetic Interactions</heading>
<li> <para-num num="[0858]"> </para-num> <div class="description-line" id="p-0819" num="0858">By considering interactions between the covariates, Applicants showed that Perturb-seq can dissect molecular effects at unprecedented resolution. A key extension is to screen the combinatorial interaction space between perturbations (Hartl, 2014; Phillips, 2008; Sackton and Hartl, 2016; Shao et al., 2008; Zuk et al., 2012), to understand genetic interactions not only between pairs of genes but even at higher order, such as the three-way interaction between NFKB1-Rel-Stat2 Applicants uncovered in BMDC (<figref idrefs="DRAWINGS">FIG. 52C</figref>)). This can also help uncover global principles of the genetic architecture of a system, such as how robust it is and what is the characteristic number of permissible perturbations (as in <figref idrefs="DRAWINGS">FIG. 60</figref>).</div>
</li> <li> <para-num num="[0859]"> </para-num> <div class="description-line" id="p-0820" num="0859">However, there are several major hurdles to achieve this ambitious goal. Both the probability of detecting all perturbations and the probability of all perturbations resulting in an effect scale exponentially with the order of perturbation (<figref idrefs="DRAWINGS">FIG. 61A</figref>). For example, when examining cells containing three sgRNAs, assuming half of the cells contain homozygous mutations, and all sgRNAs are independent, the probability of all targets being fully perturbed is one-eighth. Overcoming this source of noise will be a major challenge. However, the alternating-descent framework Applicants described (<figref idrefs="DRAWINGS">FIG. 82</figref>) and future improvements to it can help to deconvolve mixtures of knockouts, initialized by the observed assignments between cells and perturbations, and is potentially scalable to higher order interaction terms.</div>
</li> <li> <para-num num="[0860]"> </para-num> <div class="description-line" id="p-0821" num="0860">More critically, while Perturb-seq significantly reduces the time and cost required for transcriptional assays of many perturbations, these limiting factors still scale linearly with the number of perturbations assayed. The size of the combinatorially complete parameter space, p, scales as 2<sup>g</sup>, where g is the number of perturbations. For any sizeable g, p is enormous; far more enormous, in fact, that the number of cells available in any imaginable setting, regardless of costs, efforts or other technological improvements.</div>
</li> <li> <para-num num="[0861]"> </para-num> <div class="description-line" id="p-0822" num="0861">Applicants hypothesize however, that an alternative strategy should combine a substantial under-sampling of this vast space, with appropriate analytical approaches—inspired by both classical and recent methods (Beerenwinkel et al., 2007; Carter et al., 2007; Jin et al., 2006; Weinberger, 1991), and should aid greatly in our endeavor to infer the impact of interactions even when p—the number of possible combinations—is much larger than n, the number of samples. Applicants are motivated in pursuing this framing by two assumptions on biomolecular systems, such as the circuits that control gene expression: (1) modularity, as Applicants have shown in this study for both the perturbations and the gene targets, and has been well-established in many genomic and genetic studies (e.g., PMID 14973197, 12134151, 15448693, 27708008); and (2) sparsity, for example, such that Applicants expect most genes not to have genetic interactions (this is supported by those studies—almost entirely in yeast—of pairwise genetic interactions; PMID 27708008). Under these two assumptions, it may be possible, for example, to perform a subset of experiments, when most cells receive a relatively large number of perturbations (e.g., 5) and infer both partially observed and unobserved interactions at a lower order (e.g., 2 or 3).</div>
</li> <li> <para-num num="[0862]"> </para-num> <div class="description-line" id="p-0823" num="0862">Perturb-seq was experimentally designed with such future studies in mind, and our initial framework already started to rely on such assumptions. One example is the L<sub>1 </sub>penalty in the elastic net, in our regression framework. A similar L<sub>1 </sub>penalty as Applicants apply to the coefficients of B could in principle be also applied to its rank (Candès and Recht, 2009); considering the group structure in our pathway analysis of perturbations, this may represent a reasonable constraint. More technically, when attempting to fit the combinatorial interaction space, our approach is consistent with the general machine learning framework of finding a function f(x) that minimizes a loss function to best explain our observations y. Applicants effectively search for a sparse polynomial of the form y=x<sub>1</sub>+x<sub>2</sub>+x<sub>3</sub>+ . . . + x<sub>1</sub>x<sub>2</sub>+x<sub>1</sub>x<sub>2</sub>x<sub>3 </sub>. . . to best explain our observations.</div>
</li> <heading id="h-0065">Example 22</heading>
<heading id="h-0066">Perturb-Seq is a First Example of a General Framework to Combine Rich Readout with Cellular Metadata</heading>
<li> <para-num num="[0863]"> </para-num> <div class="description-line" id="p-0824" num="0863">The current implementation of Perturb-Seq can be readily extended to other traces of the cell's history and to additional rich readouts. First, other CRISPR-based perturbations are readily compatible with Perturb-Seq, including CRISPRi, CRISPRa, and alternative editors such as CRISPR/Cpf1. In particular, Modeling SNPs associated with common diseases or variants mutated in cancers requires efficient homologous recombination followed by cloning of edited cells, but this obstacle may be reduced by the ability to deconvolve wildtype and perturbed cells based on expression signature, as Applicants have done here, or by assays that will directly detect the edited variant (through DNA+RNA assay or through its encoded transcript) in each cell in the pool.</div>
</li> <li> <para-num num="[0864]"> </para-num> <div class="description-line" id="p-0825" num="0864">The RNA barcodes (PAPIs) used in this study served to identify perturbations, but can also be used to mark cells derived from a common ancestor for the purposes of lineage tracing (<figref idrefs="DRAWINGS">FIG. 54D</figref>), thus relating cell lineage and cell fate maps. For example the PRISM pool contains a set of barcoded cell lines (Yu et al., 2016). Their expressed RNA barcodes could be combined with Perturb-seq to enable pooled measurements of the transcription effect of different perturbations across cell lines by fitting an interaction term between the cell line and the perturbation.</div>
</li> <li> <para-num num="[0865]"> </para-num> <div class="description-line" id="p-0826" num="0865">Other measurement platforms may be compatible within the experimental and computational framework Applicants describe. For example, multiplex PCR (Fan et al., 2015) can be used to dial-out a subset of transcripts on which to obtain more reliable information (similar to our PAPI protocol). Protein measurements across millions of cells may be feasible by fusing FISH probes to heavy metal and using CyTOF as the readout technique. It may also be possible to devise paracrine assays by co-culturing perturbed cells in droplets and merging the droplets with a massively parallel single cell sequencing technology.</div>
</li> <li> <para-num num="[0866]"> </para-num> <div class="description-line" id="p-0827" num="0866">The experiments and analysis provide a guide the design of future experiments that leverage the scale and tunable resolution possible by combining high throughput single cell sequencing and pooled screens. By rapidly and simultaneously testing causal relations between many genetic manipulations and the resulting molecular phenotypes, Applicants aspired to help bridge the gap between genetic screening and molecular follow up experiments and to provide a valuable tool to the systems biologist. In doing so, Applicants hope to facilitate casual research questions that infer mechanisms by which genotypes result in specific phenotypes.</div>
</li> <heading id="h-0067">Example 23</heading>
<heading id="h-0068">A Robust Strategy for Encoding Information in RNA Transcripts that Allows Parallel Profiling of Separately Perturbed Cells by Droplet-Based Single-Cell RNA-Sequencing</heading>
<li> <para-num num="[0867]"> </para-num> <div class="description-line" id="p-0828" num="0867">A particularly fascinating example of a complex transcriptional response is the mammalian unfolded protein response (UPR), a branched pathway critical for maintaining homeostasis of the endoplasmic reticulum (ER). In metazoans, this response is coordinated by three distinct ER transmembrane sensor proteins (IRE1α, ATF6, and PERK), which activate three individual transcription factors (XBP1, the N-terminal cleavage product of ATF6, and ATF4, respectively) to promote survival or, when ER stress cannot be corrected, trigger cell death pathways (Ron, Walter 2007, Walter, Ron 2011, Hetz 2012). The molecular signaling events downstream of each UPR sensor protein are discrete. The resulting transcriptional responses, however, are overlapping and integrated and, therefore, the response to any given stress is likely to be heterogenous. Among stresses that can activate the UPR are potentially deleterious changes to protein folding, as well as disruption of normal nutrient flux, redox potential, calcium homeostasis, hormone stimulation, lipid biosynthesis, and membrane integrity. Considering the diversity of inputs monitored by the UPR and the necessary plasticity in outcome, comprehensive characterization of this system in mammalian cells requires both unbiased profiling of the physiological stresses that activate the sensors and delineation of the complex transcriptional phenotypes for each input.</div>
</li> <li> <para-num num="[0868]"> </para-num> <div class="description-line" id="p-0829" num="0868">Here, Applicants also apply perturb-seq to the systematic analysis of the mammalian UPR. To independently manipulate the three branches of the UPR, Applicants first developed a programmable strategy for simultaneously depleting up to three genes with high efficacy. Applicants then used perturb-seq with combinatorial depletion of the UPR sensors to delineate the distinct transcriptional programs of the UPR and the epistatic relationships among the three UPR branches. Next, Applicants used a two-tiered approach to interrogate the biological systems monitored by the UPR. Applicants identified hundreds of genes that contribute to ER homeostasis from a genome-wide CRISPRi screen and then applied perturb-seq to interrogate a diverse subset of these genes with single-cell resolution. These experiments allowed us to systematically define the functional relationships between genes with high precision and to dissect the complex, partially overlapping transcriptional responses to ER stress. Furthermore, analysis of the single-cell responses revealed bifurcation of the UPR branches at two levels: among individual cells subject to the same perturbation and at the population level, where differential activation of the three UPR branches occurred across perturbations. The latter indicated a dedicated feedback loop that enables a single arm of the UPR (the IRE1α/XBP1 branch) to specifically monitor the integrity of the protein translocation machinery. These data, alongside the parallel analysis of dendritic cell activation described herein, demonstrate the ability of perturb-seq to provide rich biological insights and systematically dissect complex biological responses.</div>
</li> <li> <para-num num="[0869]"> </para-num> <div class="description-line" id="p-0830" num="0869">The recent introduction of scalable, droplet-based approaches for single-cell gene expression profiling and the development of commercial devices (10X Genomics) have made measurement of complex phenotypes highly parallelizable (Macosko et al. 2015, Klein et al. 2015, Zheng et al. 2016). These methods incorporate two barcoding strategies that allow deconvolution of pooled RNA-seq data into single-cell transcriptomes. A cell barcode, affixed to cDNA molecules during reverse transcription of mRNAs from a single cell, is used to computationally link data from individual cells, and a similarly appended unique molecular identifier (UMI) collapses information gathered from individual mRNA molecules to allow molecular counting (<figref idrefs="DRAWINGS">FIG. 81A</figref>). A current limitation of these methods, however, is that they rely on oligo-dT priming during cDNA synthesis to capture mRNA identity and therefore report only on polyadenylated RNA transcripts. To enable the recording of other types of information, Applicants built a platform that allows genetically encoded barcodes to be expressed as polyadenlyated transcripts and captured alongside endogenous mRNAs (<figref idrefs="DRAWINGS">FIG. 81A</figref>). These barcodes can be used to mark specific cell perturbations (e.g., the identity of a Cas9-targeting single guide RNA, sgRNA), and thus, allow complex pools of cells to be evaluated by single-cell RNA-sequencing on existing droplet-based platforms.</div>
</li> <li> <para-num num="[0870]"> </para-num> <div class="description-line" id="p-0831" num="0870">Central to this approach is our barcode delivery construct, a third generation lentiviral vector carrying a polymerase II-driven barcode expression cassette that terminates with a strong polyadenylation signal (PAS) (<figref idrefs="DRAWINGS">FIG. 81B</figref>) (Goodwin, Rottman 1992). To ensure that the PAS of this internal cassette does not disrupt transcription of the lentiviral genome, and therefore lentivirus production, the entire barcode expression cassette was placed in reverse orientation with respect to transcription initiated by the upstream hybrid long terminal repeat (LTR) promoter. Within this region, the PAS was encoded directly adjacent to the barcode site (127 nucleotides separate the barcode site from the AAUAAA hexanucleotide), a placement designed to favor faithful transmission of barcode sequences into single-cell RNA-sequencing libraries, which typically capture and count only the 3′ ends of polyadenylated transcripts. To test barcode capture, Applicants performed single-cell RNA sequencing using the Chromium™ Single Cell 3′ Solution (10X Genomics) on a pool of individually transduced chronic myeloid leukemia cells (K562) carrying 8 distinct barcodes. To ensure robust barcode capture with minimal sequencing depth, Applicants developed a protocol to specifically amplify barcode transcripts (<figref idrefs="DRAWINGS">FIG. 81A</figref>). In a pilot experiment of ˜5,700 cells, Applicants observed a median of 45 independent observations (marked by unique molecular identifiers) of the “guide barcodes” per cell using this approach, allowing us to uniquely infer a guide barcode for 92.2% of cells (<figref idrefs="DRAWINGS">FIG. 81C</figref>,D). The guide barcodes also allowed us to discard droplets that fortuitously received more than one cell (marked by the presence of two or more distinct guide barcodes assigned to one cell barcodes) (<figref idrefs="DRAWINGS">FIG. 81D</figref>).</div>
</li> <li> <para-num num="[0871]"> </para-num> <div class="description-line" id="p-0832" num="0871">A practical way to build complex pools of cells experiencing different perturbations is by variably modulating gene expression, and the advent of CRISPR-based genetic approaches has revolutionized the specificity with which this can be done in mammalian cells (Shalem, Sanjana &amp; Zhang 2015). By including an sgRNA expression cassette in our barcode delivery vector, Applicants tailored our barcoding system to the study of complex CRISPR-based phenotypes (<figref idrefs="DRAWINGS">FIG. 81B</figref>). To begin validating this “perturb-seq” approach, Applicants confirmed that sgRNA expression from our barcoded construct, the “perturb-seq vector,” was capable of generating robust CRISPRi-mediated gene depletion phenotypes (<figref idrefs="DRAWINGS">FIG. 81E</figref>). Specifically, Applicants tested an EGFP-targeting sgRNA (programmed with a previously validated protospacer, EGFP-NT2 (Gilbert et al. 2013)) in the context of our perturb-seq vector for its ability to repress expression of a genomically integrated GFP reporter. Guide activity from this construct was robust and comparable to that from a previously validated sgRNA expression construct (Gilbert et al. 2014) (95.4% and 96.2% repression of GFP fluorescence, respectively). Importantly, this experiment also demonstrated that, consistent with previous findings (Gilbert et al. 2013), CRISPRi mediates highly homogenous protein depletion across a population of perturbed cells. This ability carried over to pooled experiments, in which the guide barcode can be used computationally to identify the subpopulations of cells bearing particular CRISPR perturbations. Within each subpopulation, the RNA-seq profiles always revealed knockdown of the expected target, as seen in example perturbations from our pilot experiment (<figref idrefs="DRAWINGS">FIG. 81F</figref>) demonstrating our ability to genetically encode and reliably capture the identity of CRISPRi-mediated perturbations in pooled format with single-cell resolution.</div>
</li> <heading id="h-0069">Example 24</heading>
<heading id="h-0070">A Strategy for Multiplexed sgRNA Delivery that Allows Systematic and Simultaneous Induction of Robust and Homogeneous Genetic Perturbations</heading>
<li> <para-num num="[0872]"> </para-num> <div class="description-line" id="p-0833" num="0872">Having established the potential for perturb-seq as a high content screening platform, Applicants sought to interrogate the unfolded protein response (UPR). Although well studied at the level of mechanistic signaling and bulk population behavior (Ron, Walter 2007, Walter, Ron 2011, Hetz 2012), much remains unknown about how individual cells execute the UPR. Of particular interest is the potential for cell-to-cell heterogeneities in activation of the metazoan UPR, because, unlike the linear pathway in yeast, this branched stress response integrates distinct and separable signaling events to coordinate downstream transcriptional programs (<figref idrefs="DRAWINGS">FIG. 82A</figref>). Specifically, there are 3 signaling arms of the metazoan UPR, each controlled by a different ER transmembrane sensor protein. These sensor proteins are IRE1α, PERK, and ATF6 (Walter, Ron 2011). Briefly, IRE 1α is an endoribonuclease that, upon sensing ER stress, mediates noncanonical splicing of XBP1 mRNA and facilitates stabilized expression of the active XBP1 transcription factor (XBP1s). PERK is a kinase that, upon activation, phosphorylates the alpha subunit of translation initiation factor eIF2 (eIF2α). This inhibitory phosphorylation event generally suppresses translation but paradoxically promotes translation of a small group of mRNAs, including one encoding the ATF4 transcription factor. Lastly, under stress conditions, ATF6 is targeted to the Golgi where proteolytic cleavage releases a cytosolic transcription factor domain. Once activated, XBP1, ATF4 and cleaved ATF6 translocate into the nucleus to initiate transcription.</div>
</li> <li> <para-num num="[0873]"> </para-num> <div class="description-line" id="p-0834" num="0873">Applicants sought to systematically interrogate the integrated programs of UPR-mediated transcription using perturb-seq with two motivations: first, to delineate the IRE1α, PERK, and ATF6 transcriptional regulons, which is inherently challenging because of cross-regulation between branches, and second, to evaluate both cell-to-cell and perturbation-to-perturbation heterogeneity in UPR activation. To accomplish our first goal, Applicants had to expand the perturb-seq platform for the systematic analysis of higher-order genetic interaction. Therefore, Applicants developed an approach that allows combinations of genes to be robustly and homogeneously perturbed in a single cell (<figref idrefs="DRAWINGS">FIG. 82B</figref>).</div>
</li> <li> <para-num num="[0874]"> </para-num> <div class="description-line" id="p-0835" num="0874">Previous efforts to multiplex genetic perturbations by expressing multiple sgRNAs for Cas9 or the alternative programmable nuclease Cpf1 have had limited success achieving consistent and uniform perturbation of multiple genes simultaneously (Kabadi et al. 2014, Nissim et al. 2014, Zetsche et al. 2016). To begin addressing this issue, Applicants adapted a previously reported strategy (Kabadi et al. 2014) of fusing three sgRNA expression cassettes (each composed of an RNA polymerase III promoter, the sgRNA targeting sequence, and the sgRNA constant region) into a lentiviral vector carrying our reverse barcode expression cassette (<figref idrefs="DRAWINGS">FIG. 82B</figref>). To minimize recombination at repetitive nucleotide sequences during lentiviral transduction (Sack et al. 2016, Smyth, Davenport &amp; Mak 2012), Applicants used three different RNA polymerase III promoters (initially Applicants used a modified mouse U6 promoter (mU6), a modified human U6 promoter (hU6), and a modified human 7SK promoter (h7SK), but replaced h7SK with a more potent modified bovine U6 promoter (bU6) in later work). To evaluate the activity of our initial three-guide expression vectors, Applicants assembled three test constructs that each express EGFP-NT2 from one of the promoters (and negative control sgRNAs from the other two) and monitored GFP depletion in K562 cells using flow cytometry (<figref idrefs="DRAWINGS">FIG. 88A</figref>). Applicants found that the h7SK promoter generally performed poorly in the context of our perturb-seq vector (<figref idrefs="DRAWINGS">FIG. 88B</figref>). More importantly, however, cells transduced with our first generation three-guide vectors expressing EGFP-NT2 from the hU6 promoter or the mU6 promoter partitioned into two distinct subpopulations with either strong depletion (&gt;90%) or no depletion of GFP (<figref idrefs="DRAWINGS">FIG. 88A</figref>). To exclude the possibility that dCas9-KRAB expression levels limited the activity of the three-guide vectors, Applicants engineered a GFP+K562 cell line with increased levels of dCas9-KRAB expression (<figref idrefs="DRAWINGS">FIG. 88C</figref>, see Methods). When transduced with a three-guide vector expressing EGFP-NT2, these cells still exhibited bimodal GFP expression (<figref idrefs="DRAWINGS">FIG. 82C</figref>). A likely source of this inactive population is intramolecular recombination within the 93-nt constant regions, which would lead to elimination of one or two sgRNA expression cassettes (Sack et al. 2016).</div>
</li> <li> <para-num num="[0875]"> </para-num> <div class="description-line" id="p-0836" num="0875">Consequently, Applicants sought to engineer modified sgRNA constant regions that were functionally equivalent but distinct enough in sequence to suppress recombination. Guided by crystal structures of Cas9 (Nishimasu et al. 2014, Anders et al. 2014, Jiang et al. 2015), literature reports of modified constant regions (Briner et al. 2014, Dang et al. 2015), and sequence conservation of tracrRNAs in <i>Streptococcus </i>species (Briner et al. 2014), Applicants identified bases that seemed amenable to mutation and designed a panel of variant constant regions (<figref idrefs="DRAWINGS">FIGS. 82D</figref>, 88D). Almost all constant region variants retained strong CRISPRi activity as indicated by a 97-99% reduction in mRFP levels using a bacterial CRISPRi assay suggesting that the introduced mutations do not disrupt sgRNA:Cas9 binding (<figref idrefs="DRAWINGS">FIG. 82D</figref>). By combining sets of neutral mutations, Applicants obtained two new constant regions (cr2 and cr3) that share at most 20 bases of continuous sequence homology with each other and the original constant region. Both cr2 and cr3 exhibit strong CRISPRi activity in <i>E. coli </i>(<figref idrefs="DRAWINGS">FIG. 82D</figref>).</div>
</li> <li> <para-num num="[0876]"> </para-num> <div class="description-line" id="p-0837" num="0876">To test the activity of cr2 and cr3 in human cells, Applicants generated sgRNA expression cassettes containing the mU6 promoter paired with cr2 and the hU6 promoter paired with cr3 in the context of our perturb-seq vector. When used to express EGFP-NT2, both cassettes yielded GFP depletion that was indistinguishable from that of the original perturb-seq vector in K562 cells (<figref idrefs="DRAWINGS">FIG. 82E</figref>). Finally, to identify a third functional RNA polymerase III promoter, Applicants cloned a panel of U6 promoters from different mammalian species into our perturb-seq construct with the original constant region and tested them in our GFP depletion assay. Several U6 promoters mediated strong CRISPRi activity (<figref idrefs="DRAWINGS">FIG. 88E</figref>), with the bovine U6-2 promoter (bU6) (Lambeth et al. 2006) having the largest effect.</div>
</li> <li> <para-num num="[0877]"> </para-num> <div class="description-line" id="p-0838" num="0877">Using our three sgRNA expression cassettes (bU6-2 with original cr (cr1), mU6 with cr2, hU6 with cr3), Applicants then assembled our second iteration of three-guide expression vectors (<figref idrefs="DRAWINGS">FIGS. 82B</figref>, <figref idrefs="DRAWINGS">FIG. 88F</figref>) and tested them in our GFP depletion assay. GFP+ K562 cells carrying dCas9-KRAB (higher expression levels) transduced with three-guide vectors expressing EGFP-NT2 from any of the three promoters exhibited near-uniform and strong knockdown of GFP (96-97%), nearly identical to that mediated by our single perturb-seq vector (<figref idrefs="DRAWINGS">FIG. 82F</figref>). These data establish that recombination at the constant regions during lentiviral transduction is an important limitation when fusing multiple sgRNA expression cassettes and that new highly active sgRNAs can be engineered to suppress recombination. Our final three-guide vectors, which use three different RNA polymerase III promoters and three different constant regions, can be faithfully transduced into cells alongside a single perturb-seq barcode to mediate robust depletion of targeted genes.</div>
</li> <heading id="h-0071">Example 25</heading>
<heading id="h-0072">Systematic Delineation of the Transcriptional Responses Controlled by the Unfolded Protein Response Using Perturb-Seq</heading>
<li> <para-num num="[0878]"> </para-num> <div class="description-line" id="p-0839" num="0878">Applicants next sought to use the perturb-seq approach together with our ability to simultaneously repress expression of multiple genes to systematically explore the three branches of the mammalian UPR (<figref idrefs="DRAWINGS">FIG. 83A</figref>). Using our triple sgRNA expression vector, Applicants transduced sgRNAs targeting each UPR sensor gene (ERN1 encoding IRE1α, EIF2AK3 encoding PERK, or ATF6) in all possible combinations (i.e., single, double, or triple depletions) into K562 cells. Applicants chose K562 because they have a robust UPR and as suspension cells avoid heterogeneity due to difference in their microenvironment (Snijder et al. 2009). Transduced cells were then pooled, sorted for BFP expression (a marker for sgRNA delivery), and after 2 days of growth, cells were treated with thapsigargin, a non-competitive inhibitor of ER calcium pumps, or tunicamycin, an inhibitor of N-linked glycosylation, well-known pharmacological inducers of the UPR. Control cells were treated with DMSO. Applicants sequenced transcriptomes of ˜15,000 cells, of which 5% were called as multiplets and 5% were unidentifiable, to a depth of ˜120,000 reads per cell, and observed a median of ˜25,000 UMIs per cell arising from ˜4,300 unique genes. ERN1, EIF2AK3, and ATF6 were depleted &gt;80% when targeted in any combination, and across the three chemical treatments (<figref idrefs="DRAWINGS">FIG. 82G</figref>).</div>
</li> <li> <para-num num="[0879]"> </para-num> <div class="description-line" id="p-0840" num="0879">Applicants then set out to construct an analytical approach for finding statistically robust features within the data (<figref idrefs="DRAWINGS">FIG. 83B, 89A</figref>). Every perturb-seq experiment generates an extremely rich dataset, but single-cell RNA-seq data are both intrinsically noisy (due to efficiency limitations during library preparation and natural stochasticity of transcription at the single-cell level) and of very high-dimension (Gran, van Oudenaarden 2015). The latter problem is usually overcome using unsupervised dimensionality reduction methods like principal components analysis (PCA). As many genes share common regulation, cellular behavior is intrinsically low-dimensional (Heimberg et al. 2016, Pollen et al. 2014), thus cellular behavior can be effectively summarized in terms of tens of aggregated effects instead of thousands of individual genes. A limitation of PCA is that it can be arbitrarily corrupted by outliers, and principal components often do not have clear meanings. In our hands, PCA was strongly influenced by clear outliers, such as apoptotic cells (data not shown).</div>
</li> <li> <para-num num="[0880]"> </para-num> <div class="description-line" id="p-0841" num="0880">To mitigate these problems Applicants developed an analytical pipeline, Applicants term low rank independent component analysis (LRICA), which proceeds in two stages (See Methods). Applicants first leverage recent advances in the mathematics of sparse matrices (Candès et al. 2011, Lin, Chen &amp; Ma 2010) to directly decompose the observed gene expression matrix (X) into a low-rank matrix (L), representing the low-dimensional dynamics shared across the population, and a sparse matrix (S), capturing noise and effects that are highly variable between cells:</div>
</li> <li> <div class="description-line" id="p-0842" num="0000">
<br/>
<i>X=L+S </i>
</div>
</li> <li> <para-num num="[0881]"> </para-num> <div class="description-line" id="p-0843" num="0881">The low rank matrix contains the major dynamic trends in the population. Applicants identified these trends using independent component analysis (ICA, Methods), which deconstructs the gene expression pattern in each cell into a linear sum of components that are maximally independent of each other (instead of uncorrelated, as in PCA). The components aid interpretation in two ways: by determining which cells are high or low for a particular component, Applicants can identify subpopulations, and by asking which genes maximally influence a component, Applicants can identify groups of related genes driving a particular behavior.</div>
</li> <li> <para-num num="[0882]"> </para-num> <div class="description-line" id="p-0844" num="0882">Applicants applied LRICA to our thapsigargin-treated population. Four components varied across the different perturbations, including three that tracked the presence of PERK, IRE1α, and ATF6 (Methods, <figref idrefs="DRAWINGS">FIG. 89B</figref>). When projected to two dimensions using t-distributed stochastic neighbor embedding (t-sne) (Van Der Maaten 2014), cells bearing a particular perturbation all grouped together, and biologically reasonable groups of perturbations also clustered in proximity to each other. Thus, the identity of the perturbation contained in each cell was distinguishable, which further validated our triple knockdown strategy. The same analysis applied to the four components that varied across the cell cycle arranged the cells in a circular pattern ordered correctly by cell cycle phase. Thus, LRICA identified and decomposed the two largest effects causing variation in the population in an unbiased way and also computationally decoupled them from each other, enabling them to be studied independently.</div>
</li> <li> <para-num num="[0883]"> </para-num> <div class="description-line" id="p-0845" num="0883">Applicants did observe a small but prominent interaction between the two effects, apparent in a “bulge” seen in the cell cycle analysis (highlighted in <figref idrefs="DRAWINGS">FIG. 83B</figref>) where cells bearing certain perturbations concentrated. Closer examination of a more targeted set of three perturbations suggested that this interaction was caused by PERK-dependent cell cycle arrest in G1 caused by thapsigargin treatment (<figref idrefs="DRAWINGS">FIGS. 83C</figref>,D) (Hamanaka et al. 2005). A particular component (right panel of <figref idrefs="DRAWINGS">FIG. 83C</figref>) was bimodal among the cells bearing each perturbation. Defining the cells with that component low as “G1 cells” (cf. middle and right panels of <figref idrefs="DRAWINGS">FIG. 83C</figref>), Applicants looked at the top fifty genes influencing the component (<figref idrefs="DRAWINGS">FIG. 83E</figref>). This list included a set of factors associated with cell cycle progression and the UPR. Intriguingly, it highlighted epistatic interactions between the two effects of PERK-dependent activation of the UPR and progression through G1, showing for example how for some genes the two programs cancel each other out, while for others they may act synergistically. A notable example of the latter was the expression of MYC, which has previously been identified as being thapsigargin-induced (Liang et al. 2006). Our data here show that MYC induction is most strongly associated with the G1-arrested subpopulation (<figref idrefs="DRAWINGS">FIG. 83E</figref>).</div>
</li> <li> <para-num num="[0884]"> </para-num> <div class="description-line" id="p-0846" num="0884">Given our success in decomposing independent effects, Applicants next turned to decomposing the distinct branches of the UPR itself. Applicants identified a set of genes that were most robustly induced in response to both thapsigargin and tunicamycin treatment and used hierarchical clustering to order them based on their pattern of co-expression in the 15,000 cells within our experiment (see Methods). Applicants constructed synthetic bulk RNA-seq profiles by averaging together all cells bearing the same perturbation. When these profiles were ordered according to our clustering, clear patterns of regulatory control were apparent (<figref idrefs="DRAWINGS">FIG. 83F</figref>). To estimate regulatory overlap, Applicants decomposed the changes across bulk responses using ICA (bottom of <figref idrefs="DRAWINGS">FIG. 83F</figref>, Methods). PERK/ATF4 had the largest regulon in our experiment, and had many targets uniquely under its control. ATF6 and IRE1α showed more overlap in their set of regulatory outputs, consistent with their common transcriptional regulatory mechanism. Of the two, IRE1α had more specific targets, notably components of the translocon (e.g. SEC61A1, SEC61B), which is consistent with previous reports (Shoulders et al. 2013), but ATF6 had stronger activating effects on common targets (<figref idrefs="DRAWINGS">FIG. 83F</figref>). Many genes showed some sensitivity to all branches, particularly a group of very high abundance core stress response genes (HSPA5, HERPUD1, SDFL21). Our epistasis experiment thus defined and decoupled the three overlapping branches of the mammalian UPR, both at the bulk level and within single cells.</div>
</li> <heading id="h-0073">Example 26</heading>
<heading id="h-0074">Unbiased Genome-Scale CRISPRi Screens Identify a Library of Genetic Perturbations that Induce the UPR</heading>
<li> <para-num num="[0885]"> </para-num> <div class="description-line" id="p-0847" num="0885">Having systematically mapped the transcriptional programs controlled by each of the three UPR sensors, Applicants next sought to leverage this information and our platform to evaluate what role each branch plays in responding to different types of perturbations. A principal advantage of perturb-seq is the ability to evaluate transcriptional phenotypes for many genetic modifications in a single, internally controlled experiment. Thus Applicants can ask fundamental questions about how transcriptional pathways differentially respond to diverse and physiologically relevant inputs, and importantly, Applicants can do this in cells otherwise exposed to identical conditions. Applicants employed a two-tiered approach to use perturb-seq to systematically explore transcriptional pathways, such as the UPR. First, Applicants performed a genome-scale CRISPRi screen to identify genes that are important for ER homeostasis, as evidenced by induction of the UPR upon depletion. Applicants then applied perturb-seq to a diverse sub-library targeting the strongest hits to better understand their role.</div>
</li> <li> <para-num num="[0886]"> </para-num> <div class="description-line" id="p-0848" num="0886">To begin, Applicants constructed a fluorescence reporter of IRE1α activation. This reporter drives mCherry expression from a minimal c-fos promoter controlled by 5 IRE1α-responsive unfolded protein response elements (UPREs) (Wang et al. 2000) (<figref idrefs="DRAWINGS">FIG. 84A</figref>). The reporter was transduced into K562 cells stably expressing a dCas9-KRAB fusion protein and a clonally derived line (cBA010), which demonstrated strong reporter activation (<figref idrefs="DRAWINGS">FIG. 84B</figref>), was selected. To control for general effects on gene expression during screening, this cBA010 parental line was subsequently transduced with a constitutive GFP reporter (driven by the EF1α promoter) (<figref idrefs="DRAWINGS">FIG. 84A</figref>). Importantly, when treated with tunicamycin, these cells (cBA011) demonstrated XBP1-dependent mCherry induction (maximally 16-fold), which occurred subsequent to endogenous XBP1 splicing (<figref idrefs="DRAWINGS">FIGS. 84B</figref>, 90A). As expected, Applicants observed no similar induction of GFP.</div>
</li> <li> <para-num num="[0887]"> </para-num> <div class="description-line" id="p-0849" num="0887">Using our reporter cell line, Applicants separately screened two genome-scale CRISPRi libraries, our first generation library (hCRISPRi-v1) and our recently described second generation library (hCRISPRi-v2) (<figref idrefs="DRAWINGS">FIG. 84C</figref>) (Gilbert et al. 2014, Horlbeck et al. 2016). Briefly, cBA011 K562 cells were transduced with each library. After 8 days of growth, the cells were separated by fluorescence-activated cell sorting into bins according to the ratiometric reporter signal (mCherry/GFP). Cells in the top and bottom thirds of the reporter distribution were collected and processed to measure the frequency of sgRNAs contained within each bin by deep sequencing. Using these data, Applicants calculated the loge sgRNA enrichments in the top mCherry/GFP bin over the bottom bin, and defined gene-level reporter signal phenotypes as the average of the 3 most active sgRNAs targeting each gene. Our hCRISPRi-v2 screen identified 397 hit genes with high mCherry/GFP, indicative of UPR activation (<figref idrefs="DRAWINGS">FIG. 84D</figref>). Importantly, phenotypes were reproducible between replicates of this screen and minimal correlation was observed between hit phenotypes and previously calculated gene growth phenotypes (Spearman R=−0.2) (<figref idrefs="DRAWINGS">FIG. 84E, 90B</figref>).</div>
</li> <li> <para-num num="[0888]"> </para-num> <div class="description-line" id="p-0850" num="0888">Our first generation CRISPRi library targets 15,977 genes (20,899 transcriptional start sites, TSSs) with 10 sgRNAs per TSS, and our hCRISPRi-v2 library targets 18,905 genes at 20,526 TSSs. hCRISPRi-v2 incorporates several improvements over hCRISPRi-v1, including a greater frequency of highly active sgRNAs and the flexibility to screen with either 10 sgRNAs per gene or a similarly active half-library containing the top 5 predicted sgRNAs for each gene (Horlbeck et al. 2016). Practically, the compact hCRISPRi-v2 library allowed us to maintain higher screen representation through the flow cytometer with similar sorting times (screen representations: hCRISPRi-v1 ˜450, hCRISPRi-v2 ˜600). Encouragingly, of the 141 hits from the hCRISPRi-v1 screen, 103 reproduced from screening the hCRISPRi-v2 library (Fisher's Exact p-value=8.97e-138) (<figref idrefs="DRAWINGS">FIG. 90C</figref>).</div>
</li> <li> <para-num num="[0889]"> </para-num> <div class="description-line" id="p-0851" num="0889">Among hits from the hCRISPRi-v2 screen are well-characterized regulators of protein folding in the ER, most notably HSPA5, which encodes the major ER Hsp70 chaperone BiP (<figref idrefs="DRAWINGS">FIG. 84F</figref>). Consistent with results from a similar screen in yeast (Jonikas et al. 2009), our hits also featured genes functionally involved in N-linked glycosylation, including components of the oligosaccharyltransferase (OST) and the dolichol-linked oligosaccharide biosynthesis pathway, ER-associated degradation (ERAD), and protein trafficking. Additionally, genes involved in protein targeting to the ER were enriched among hits (Fisher's Exact p-value=2.65e-09). This gene set consists of genes encoding components of the signal recognition particle (SRP), the SRP receptor, and the signal peptidase complex. Three out four of the translocon-associated protein complex (TRAP) scored; and strikingly, among the 7 hits with the strongest phenotype were all three genes that encode the ER protein-conducting channel or PCC (SEC61A1, SEC61B, SEC61G) (<figref idrefs="DRAWINGS">FIG. 84D</figref>,F). The phenotypes of SRP targeting factors and PCC were surprising because recent reports have shown that SRP-mediated recruitment to the ER and IRE1α binding to PCC subunits are required for maximal pathway activation in response to exogenous stress (Plumb et al. 2015, Kanda et al. 2016). Satisfyingly, targeting of both ERN<sub>1 </sub>and XBP1 was observed to decrease reporter signal in the screen (<figref idrefs="DRAWINGS">FIG. 84D</figref>).</div>
</li> <li> <para-num num="[0890]"> </para-num> <div class="description-line" id="p-0852" num="0890">Of note, genes with biological functions unrelated or indirectly related to ER function also scored as UPR hits, some of which are distinct from functional classes seen in the analogous systematic yeast studies (Jonikas et al. 2009). Specifically, sets of genes that control general translation, transcription, and, perhaps most intriguingly, mitochondrial function were enriched among hits (<figref idrefs="DRAWINGS">FIG. 84F, 90D</figref>). While disruption of the functions of these globally important genes may induce the UPR through impairment of ER function, it is also possible that such broad hits, especially the transcriptional components, represent perturbations of our reporter system. During individual testing of 257 sgRNAs targeting 152 select hit genes, Applicants confirmed that a majority induced reporter signaling from the UPRE reporter; however, for some sgRNAs, notably ones targeting the mediator transcriptional complex, Applicants found that GFP levels were also reduced, inflating the mCherry/GFP ratio (<figref idrefs="DRAWINGS">FIG. 84G</figref>). Moreover, from the magnitude of the phenotypes alone, Applicants cannot distinguish among the various mechanisms for disrupting ER homeostasis because perturbation of functionally diverse genes frequently resulted in similar levels of reporter induction whereas perturbation of genes with shared function, as in the case of ER protein targeting genes, often resulted in a range of phenotypes (<figref idrefs="DRAWINGS">FIG. 84F</figref>). Cumulatively, these data highlight the ability of the pooled screens to robustly identify diverse factors important for ER homeostasis but the challenges of making systematic conclusions about gene function from such simple phenotypic readouts.</div>
</li> <heading id="h-0075">Example 27</heading>
<heading id="h-0076">Perturb-Seq Analysis of a UPR-Inducing CRISPRi Sub-Library Demonstrates Functional Relationships between Genes and Differential Activation of UPR Transcriptional Programs in Response to Distinct Forms of Stress</heading>
<li> <para-num num="[0891]"> </para-num> <div class="description-line" id="p-0853" num="0891">Our next goal was to systematically characterize the role of these different classes of genes. Applicants hypothesized that a mechanistically diverse set of perturbations to ER homeostasis might reveal a richer picture of the underlying structure of its regulation than is possible with a small set of strong chemical perturbants (e.g., thapsigargin and tunicamycin). Applicants thus applied perturb-seq to a CRISPRi sub-library of 91 sgRNAs targeting 83 genes, including many of our strongest hits, and 2 negative controls (<figref idrefs="DRAWINGS">FIG. 90C</figref>). To test platform scalability, sgRNAs were delivered via pooled transduction in this experiment. Applicants then subjected the UPR-inducing CRISPRi sub-library to profiling via perturb-seq, collecting ˜65,000 cells and sequencing to an average depth of ˜67,000 reads per cell, which collapsed to ˜15,300 UMIs arising from ˜3,700 unique genes per cell. Approximately 50,500 cells were called a containing unique sgRNA, while <b>18</b> 9,000 contained multiple sgRNAs (either due to double infection during our pooled transduction or encapsulation of multiple cells during emulsion generation). ˜6,000 cells were unidentifiable. All of the expected sgRNAs were detected, with nearly even representation across sgRNAs that were pooled equally during viral titering (457±108 cells per sgRNA, mean±standard deviation).</div>
</li> <li> <para-num num="[0892]"> </para-num> <div class="description-line" id="p-0854" num="0892">Applicants then constructed synthetic bulk expression profiles by averaging normalized expression across cells containing a given sgRNA. Hierarchical clustering of these average profiles revealed that sgRNAs targeting the same gene always clustered together (<figref idrefs="DRAWINGS">FIG. 91A</figref>), and the level of knockdown of guides targeting the same gene was similar (<figref idrefs="DRAWINGS">FIG. 85C</figref>). Knockdown was robust, with a median 90% depletion of the guide target (<figref idrefs="DRAWINGS">FIG. 85C</figref>). Knockdown appeared as a shift of the entire distribution of expression, rather than a bifurcation into perturbed and unperturbed subpopulations (<figref idrefs="DRAWINGS">FIG. 91B</figref>). To assess homogeneity, Applicants computationally assigned each cell a score measuring the deviation of its expression profile from the distribution of expression seen in control cells. Comparing the most perturbed cells (top third) to the least perturbed (bottom third) showed a median difference in knockdown of 8% (<figref idrefs="DRAWINGS">FIG. 91C</figref>). These findings establish the general robustness of the CRISPRi reagents in producing uniform knockdowns as well as the ability of the barcoding scheme to accurately assign sgRNAs to the appropriate cells. Given the similarity in phenotypes between sgRNAs targeting the same gene, in subsequent analyses Applicants grouped cells by sgRNA target, rather than by sgRNA.</div>
</li> <li> <para-num num="[0893]"> </para-num> <div class="description-line" id="p-0855" num="0893">At the level of bulk analysis, perturb-seq offers a simple and robust platform for highly parallel RNA-seq. Since sgRNAs can be introduced in pooled format, variability from exogenous factors is minimized, and the loss of information from shallow capture and sequencing is reduced once many cells are averaged together. These bulk profiles are a rich phenotypic fingerprint that can be used to understand how different perturbations relate to each other (Parnas et al. 2015). For example, hierarchical clustering of bulk profiles for each targeted gene revealed high precision functional clusters (apparent as lines on the diagonal in <figref idrefs="DRAWINGS">FIG. 85A</figref>), consistent with known protein-protein interactions such as the oligosaccharyltransferase complex (DDOST/DAD1 /OST4) and Triple T complex (TELO2/TTI1/TTI2), as well as factors involved in the ubiquitylation reactions of ERAD (SYVN1/SEL1L), SRP-mediated protein targeting (SRP68/SRPRB/SRPR), protein trafficking (TMED2/TMED10), and UFMylation (UFML1/UFM1/DDRGK1). Genes also clustered by role, including ER-Golgi trafficking (the YIPF5/IER3IP1 complex clustering with SCYL1 and TMEM167A). Interestingly, genes encoding particular translation initiation factors, the EIF2B complex (EIF2B2/EIF2B3/EIF2B4) and eIF1α (EIF21), were part of a large, highly correlated cluster. Similar to inhibition of eIF1α, EIF2B loss-of-function is known to induce ATF4 signaling (Fogli, Boespflug-Tanguy 2006). This presents the possibility that the repression of other genes within this cluster also induce ATF4. Indeed, within this large cluster Applicants observed smaller clusters of genes whose common roles are monitored by ATF4, including a set of aminoacyl tRNA synthetases (IARS2/SARS/FARSB/HARS/QARS/CARS/AARS/MARS/DARS/TARS) and mitochondrial genes (TIMM23/SAW50/ATP5B/MRPL39 and DNAJC19/HSPA9/ IARS2) (Bao et al. 2016, Tyynismaa et al. 2010).</div>
</li> <li> <para-num num="[0894]"> </para-num> <div class="description-line" id="p-0856" num="0894">An advantage of perturb-seq is that these bulk properties can then be examined at the single-cell level to better understand the phenotypes. For example, decomposing the populations by cell cycle position (<figref idrefs="DRAWINGS">FIG. 85B</figref>) revealed that some perturbations, such as many of the aminoacyl tRNA synthetases, elicited an accumulation of cells in G2. In contrast, CAD, a gene essential for pyrimidine synthesis, elicited a clear accumulation of cells in S phase.</div>
</li> <li> <para-num num="[0895]"> </para-num> <div class="description-line" id="p-0857" num="0895">Applicants next sought to analyze how individual hits effect activation of the different branches of the UPR. As single-cell data are prone to dropouts, scoring systems based on defined linear combinations of gene expression are prone to wide variation. Applicants thus adopted a data-driven strategy, and trained random forest regressors to score branch activation using the cells in our epistasis experiment, in which the branches are definitively separated, as training data (Methods). This scoring method had superior accuracy compared to defined gene lists and superpositions derived from ICA (Methods, <figref idrefs="DRAWINGS">FIG. 91F</figref>).</div>
</li> <li> <para-num num="[0896]"> </para-num> <div class="description-line" id="p-0858" num="0896">Branch activation scores (<figref idrefs="DRAWINGS">FIG. 85D</figref>) showed that hits from the screen elicited activation of all three branches of the UPR with clear correlations in activation among functionally related groups of genes. Perhaps most interestingly, different groups of perturbations elicited differential activation of the three branches. For example, repression of HSPA5, which encodes the core component of the UPR and central ER chaperone BiP, robustly activated all three branches. The amino acid synthetases activated both IRE1α and ATF4 branches of the UPR, with the latter possibly influenced by GCN2 signaling, which is responsive to amino acid stress and, like PERK, inactivates eIF2α. (Because our training data was experimentally obtained using PERK depletion in the presence of ER stress, Applicants score activation of the corresponding signature with a “PERK score.” However, the activity of four separate kinases that upregulate ATF4 may formally contribute to these effects). Finally, all three components of the PCC (SEC61A1/SEC61G/SEC61B) appeared to selectively activate only the IRE1α branch. Comparison with alternate scoring methods and expression of branch-specific genes Applicants identified in the epistasis experiment showed good agreement with these calls (<figref idrefs="DRAWINGS">FIG. 91D</figref>,E). Thus at the bulk level, our data reveal how different genetic perturbations can selectively activate the different branches of the UPR.</div>
</li> <heading id="h-0077">Example 28</heading>
<heading id="h-0078">Single-Cell Expression Profiling of Cells Perturbed by HSPA5 Depletion Uncovers a Bifurcated Response</heading>
<li> <para-num num="[0897]"> </para-num> <div class="description-line" id="p-0859" num="0897">The observation of differential activation of the different UPR branches in bulk raises an immediate question: do single cells coordinately carry out an identical response to a given perturbation, or might the branches also operate independently at the single-cell level? To explore this issue, Applicants examined cells depleted of BiP, where all three branches of the UPR are active.</div>
</li> <li> <para-num num="[0898]"> </para-num> <div class="description-line" id="p-0860" num="0898">When compared to cells containing control sgRNAs, cells transduced with HSPA5-targeting sgRNAs were readily distinguishable as a distinct population (<figref idrefs="DRAWINGS">FIG. 86A</figref>), and had markedly different patterns of gene expression (˜2,100 genes differentially expressed at P&lt;0.01). Using LRICA, Applicants decomposed these differences into sixteen independent components. Several of these reflected cell cycle effects, reflecting the altered composition seen previously (<figref idrefs="DRAWINGS">FIG. 85B</figref>).</div>
</li> <li> <para-num num="[0899]"> </para-num> <div class="description-line" id="p-0861" num="0899">Two of the independent components (IC<b>1</b> and IC<b>2</b>) varied substantially between control and HSPA5-perturbed cells (<figref idrefs="DRAWINGS">FIG. 86B</figref>), and examination of the genes that most strongly affected these components showed they were driven by genes Applicants had previously identified as under UPR control. Comparing these hypothesis-free results to the branch activation scores given by our random forest regressors (<figref idrefs="DRAWINGS">FIG. 86C</figref>) showed that our analysis pipeline had independently discovered a subpopulation structure within the HSPA5-perturbed cells in which ATF6 alone was activated (IC1 and IC2 both low), ATF6 and IRE1α were activated (IC1 low, IC2 high), or ATF6, IRE1α, and ATF4 were simultaneously activated (IC1 high, IC2 low). Indeed, when Applicants ordered the cells by the value of IC1 and examined the expression of UPR-induced genes (as defined in <figref idrefs="DRAWINGS">FIG. 83F</figref>), the trends defining these subpopulations were apparent (<figref idrefs="DRAWINGS">FIG. 86D</figref>).</div>
</li> <li> <para-num num="[0900]"> </para-num> <div class="description-line" id="p-0862" num="0900">Of particular note was the switch-like induction of the ATF4 branch, revealing that these cells represented a discrete subpopulation. Importantly, these differences did not reflect differences in BiP depletion, as the subpopulations with IC1 low and high (<figref idrefs="DRAWINGS">FIGS. 86B</figref>,D) had indistinguishable expression of HSPA5 (<figref idrefs="DRAWINGS">FIG. 86E</figref>). They were distinguishable, however, in that the ATF4-induced subpopulation had a substantially altered cell cycle distribution, with many cells accumulating in G2 compared to control cells or HSPA5-perturbed cells without ATF4 induced (<figref idrefs="DRAWINGS">FIG. 86F</figref>). These results thus reveal that the phenotype observed at bulk level does not accurately capture the biology of single cells, and that the UPR can be executed in markedly different ways within an apparently homogeneous population.</div>
</li> <heading id="h-0079">Example 29</heading>
<heading id="h-0080">Gene-Gene Covariance Analysis of Perturb-Seq Data Reveals Transcriptional Regulons</heading>
<li> <para-num num="[0901]"> </para-num> <div class="description-line" id="p-0863" num="0901">Perturb-seq offers substantial insights into the biology of individual cells, but also reveals the genes driving those behaviors. <figref idrefs="DRAWINGS">FIG. 86D</figref> underscores a key point: correlated up- or downregulation of genes can be a signature of shared regulation (Klein et al. 2015). As a single perturb-seq experiment provides tens of thousands of data points for each gene, these coordinated motions provide a rich phenotype that can be used to identify related genes by looking for correlated patterns of expression (<figref idrefs="DRAWINGS">FIG. 86G</figref>).</div>
</li> <li> <para-num num="[0902]"> </para-num> <div class="description-line" id="p-0864" num="0902">For example, Applicants identified 200 genes that were perturbation-induced and highly variable across the 50,500 cells in our perturb-seq experiment. When these genes were clustered based on their patterns of co-expression, obvious functional groupings appeared, including all three branches of the UPR (<figref idrefs="DRAWINGS">FIGS. 86H</figref> and 92A). Indeed, when Applicants clustered our defined list of UPR-induced genes (from <figref idrefs="DRAWINGS">FIG. 83F</figref>) using co-expression information from the epistasis experiment or the perturb-seq experiment, Applicants uncovered similar clusterings (cophenetic correlation 0.81, compared to 0.13 when control cells were used to cluster the genes), with differences tending to involve targets with overlapping regulation by ATF6 and IRE1α (<figref idrefs="DRAWINGS">FIG. 861</figref>, <figref idrefs="DRAWINGS">FIG. 92B</figref>). The similarity between experiments indicates that much of the regulatory interactions between these genes is implicitly present in the perturb-seq experiment, and also suggests that the organization of the UPR is preserved between the very strong chemical inducers used in the epistasis experiment and the far more varied genetic perturbations used here.</div>
</li> <li> <para-num num="[0903]"> </para-num> <div class="description-line" id="p-0865" num="0903">A key advantage of perturb-seq is that the perturbations tend to elevate correlations above the levels induced by natural variation. These correlations can be further enhanced by a “fishing” strategy, where the population is subsampled to enrich for cells perturbed for a given group of genes. Related genes can then be discovered by how well their expression correlates with the original group (<figref idrefs="DRAWINGS">FIG. 86J</figref>,K, Methods). For example, our preliminary analysis (<figref idrefs="DRAWINGS">FIG. 86H</figref>) identified a five gene cluster of cholesterol biosynthesis genes with correlated expression. When Applicants confined our gene clustering analysis to the roughly ˜9,000 cells most perturbed for this cluster, Applicants saw a general strengthening of correlations, and the core group now clustered with a group of genes related to cholesterol biosynthesis (<figref idrefs="DRAWINGS">FIG. 86K</figref>). Similarly, when Applicants selected out the cells most perturbed for the heat shock proteins HSPAJA and HSPAJB, the genes that clustered nearby were essentially all implicated in the heat shock response based on Reactome annotation, HSF1 binding profiles from Encode, or literature searches (<figref idrefs="DRAWINGS">FIG. 86L</figref>). When Applicants expanded the analysis to look at nearby gene clusters, Applicants noticed some apparent structure in co-expression (<figref idrefs="DRAWINGS">FIG. 86L</figref>). Intriguingly, the correlation structure seemed somewhat explainable in terms of the most enriched transcription factor binding sites among the gene set, with the middle group of genes for example linked to the first group by a possible regulatory overlap between HSF1 and KAT2A (<figref idrefs="DRAWINGS">FIG. 86L</figref>). Though these demonstrations are not definitive proof, they suggest that gene-gene correlation information from perturb-seq, potentially constrained by other sources of information like transcription factor binding, may enable automated functional clustering of genes of unknown function.</div>
</li> <heading id="h-0081">Example 30</heading>
<heading id="h-0082">Preferential Activation of the IRE1α-Controlled Arm of the UPR in Response to Translocon Dysfunction Initiates Homeostatic Feedback</heading>
<li> <para-num num="[0904]"> </para-num> <div class="description-line" id="p-0866" num="0904">Among the genes targeted in the perturb-seq screen, the PCC genes (SEC61A1, SEC61G, and SEC61B) were perhaps the most intriguing outliers, in that they each displayed a marked preference for activation of the IRE1 branch (<figref idrefs="DRAWINGS">FIG. 85D</figref>), with widespread IRE1α activation and little or no activation of the other branches (<figref idrefs="DRAWINGS">FIG. 87A, 87B, 93A</figref>). To confirm that our single-cell data were accurately calling IRE1α activation, Applicants directly probed for XBP1 splicing using RT-PCR (Calfon et al. 2002) (<figref idrefs="DRAWINGS">FIG. 87C</figref>). Depletion of all three PCC coding genes induced XBP1 splicing at levels consistent with the single-cell data and to a degree at or above that provoked by targeting HSPA5, a perturbation that induces all three branches of the UPR. Additionally, a time course experiment showed that depletion of PCC components led to sustained XBP1 splicing over the course of days. Expression of SSR2, a translocon-associated protein and strongly selective target of IRE1α (<figref idrefs="DRAWINGS">FIG. 87D</figref>, and cf. <figref idrefs="DRAWINGS">FIG. 83F</figref>), was also maintained over the same period. These results were in marked contrast to the transient XBP1 splicing seen when cells were challenged with chemical stress (<figref idrefs="DRAWINGS">FIG. 90A</figref>), which diminished on the scale of hours, consistent with previous reports (Lin et al. 2007).</div>
</li> <li> <para-num num="[0905]"> </para-num> <div class="description-line" id="p-0867" num="0905">The kinetics and magnitude of IRE1 activation in response to PCC gene repression varied with respect to the subunit targeted, which may reflect the fact that depletion of these factors also have variable effects on cell growth. SEC61A1 is an essential gene with strong growth phenotypes in both CRISPRi and CRISPR cutting cell viability screens (Gilbert et al. 2014, Wang et al. 2015), however across both analyses, SEC61B depletion demonstrated at most a relatively mild effect, which included analysis of a CRISPRi sgRNA (SEC61B-1), for which Applicants independently validated mRNA depletion (<figref idrefs="DRAWINGS">FIG. 90B, 93B</figref>,C). Concordantly, Applicants observed a limited increase in CHOP expression upon depletion of SEC61A1 (<figref idrefs="DRAWINGS">FIG. 87D</figref>), which is indicative of ATF4 induction. The single-cell analyses nonetheless revealed that the majority of cells only induced the IRE1α branch (<figref idrefs="DRAWINGS">FIG. 87A, 93A</figref>), suggesting that the increase in CHOP expression might reflect general toxicity. Strikingly, depletion of SEC61B, which causes only mild growth defects, showed little to no CHOP induction over ten days of growth, while showing sustained markers of IRE1α induction (<figref idrefs="DRAWINGS">FIG. 87D</figref>). Applicants note that SEC61B appears to share a co-regulated promoter region with ALG2, a gene that functions in N-linked glycosylation, and that Applicants cannot formally separate the effects of repressing these genes (<figref idrefs="DRAWINGS">FIG. 93B</figref>). Nonetheless, the consistent phenotypes of SEC61A1, SEC61B, and SEC61G repression suggest that genetic disruption of the PCC elicited selective activation of the IRE 1α branch. An alternative explanation for these observations is that depletion of PCC subunits induces general stress but impairs activation of the other branches of the UPR. ATF4 activation was intact, however, as cells transduced with sgRNAs targeting SEC61A1, SEC61B, or SEC61G showed robust CHOP induction in response to thapsigargin treatment (<figref idrefs="DRAWINGS">FIG. 87E</figref>). Additionally, without stress these cells showed strong XBP1 splicing and low levels of CHOP as compared to cells transduced with an HSPA5 targeting sgRNA, which demonstrated strong basal CHOP induction (<figref idrefs="DRAWINGS">FIG. 87C</figref>,E). Collectively, these results confirm that depletion of PCC subunits selectively induces the IRE1α branch.</div>
</li> <li> <para-num num="[0906]"> </para-num> <div class="description-line" id="p-0868" num="0906">Taken together with our single-cell epistasis data, these experiments suggest a selective role for the IRE1α branch of the UPR in monitoring PCC function. Many of the strongest and most selective IRE1α transcriptional targets in the epistasis experiment were PCC-associated genes (<figref idrefs="DRAWINGS">FIG. 83F</figref>), and by qPCR analysis, Applicants confirmed upregulation of the two other PCC coding genes in response to depletion of SEC61A1 or SEC61B (<figref idrefs="DRAWINGS">FIG. 93B</figref>). Conversely, PCC coding and translocon-associated genes were hits in our unbiased genome-wide screen for IRE1α activation and also showed preferential IRE1α pathway activation in our perturb-seq analysis. Given that the translocon has been shown to selectively interact with IRE1α (and not ATF6 or PERK) (Plumb et al. 2015, Kanda et al. 2016), these results suggest a model in which IRE1α actively monitors the function and number of translocons and acts to increase them as needed (<figref idrefs="DRAWINGS">FIG. 87F</figref>).</div>
</li> <heading id="h-0083">Example 31</heading>
<heading id="h-0084">Discussion</heading>
<li> <para-num num="[0907]"> </para-num> <div class="description-line" id="p-0869" num="0907">Here Applicants present perturb-seq, an experimental and analytical platform that enables multiplexed profiling of many perturbations with single-cell resolution, and used it to systematically dissect the mammalian unfolded protein response (UPR). Perturb-seq is enabled by a strategy Applicants developed for robustly encoding the identity of a perturbation in an expressed transcript. Because this information is specifically encoded in the genome of cells bearing a particular perturbation, different perturbations can be interrogated together in pooled format via massively parallel droplet-based single-cell RNA-seq. This approach fills a gap in our ability to interrogate biological systems, allowing measurement of rich, unbiased phenotypes for many perturbations in one experiment.</div>
</li> <li> <para-num num="[0908]"> </para-num> <div class="description-line" id="p-0870" num="0908">Perturb-seq addresses a pressing problem in modern genetics: with genome-wide screens now routine (Shalem, Sanjana &amp; Zhang 2015), strategies must be developed to rapidly and systematically understand hits. Perturb-seq enables this in two ways. First and most simply, it is a powerful platform for highly parallelized RNA-seq in homogeneous experimental conditions. Combined with our approach for combinatorial genetic perturbations, perturb-seq has the potential to interrogate tens of thousands of distinct combinations of perturbations in one pool. On the other end, the single-cell nature of the data provides deep and precise insights into how cells behave. Indeed the analytical approaches Applicants have developed provide general tools for exploiting single-cell data to identify co-regulated genes in a hypothesis-free manner, reveal discrete subpopulations that make bulk phenotypes misleading, and separate specific effects from confounding phenomena like the cell cycle.</div>
</li> <li> <para-num num="[0909]"> </para-num> <div class="description-line" id="p-0871" num="0909">Applicants here focused on genetic perturbations using CRISPRi, but the platform can be used to encode the identity of a wide range of perturbations or changes, such as CRISPR cutting-mediated loss of function (Wang et al. 2015, Shalem et al. 2014), gene activation (Tanenbaum et al. 2014, Konermann et al. 2015, Chavez et al. 2015), targeted mutation (Komor et al. 2016), or the identity of a particular variant in a library of systematically mutagenized proteins. Applicants have shown that CRISPRi gives strong, highly homogeneous, and simultaneous depletion of up to three targets and enables the study of essential genes. It also has some potential advantages when scaling to high-order combinations relative to CRISPR cutting, as genetic variability (e.g., partially functional alleles due to in-frame indels (Shi et al. 2015)) and non-specific toxicity due to DNA cutting (Wang et al. 2015, Aguirre et al. 2016, Munoz et al. 2016, Horlbeck et al. 2016) both increase with the number of cut sites. On the other hand, CRISPR cutting provides complementary advantages including allowing domain-level analysis (Shi et al. 2015) and the generation of true null alleles (Shalem et al. 2014).</div>
</li> <li> <para-num num="[0910]"> </para-num> <div class="description-line" id="p-0872" num="0910">Perturb-seq is already a robust platform for in depth functional analysis of hit genes from screens. Scaling it further to genome-level coverage requires overcoming some obstacles, but none appear intractable. Cost per cell is a major consideration, but this will decline as technologies mature. With current techniques (Zheng et al. 2016), ˜50,000 cells can be profiled in ten minutes. Additionally, perturb-seq will enable higher loading, which usually has the undesirable effect of increasing the frequency at which multiple cells are encapsulated per droplet, because our guide barcodes allow computational removal of cell doublets. Sequencing costs are high, but can be mitigated at the library preparation stage through specific amplification of select targets, as demonstrated with our barcode capture, or by depletion of uninteresting high abundance genes (Gu et al. 2016). A more subtle point is that provirus recombination during lentivirus transduction can potentially scramble barcode identities during standard methods of pooled library delivery (Sack et al. 2016). Applicants took careful steps to avoid these problems (see Methods) and expect that altered library preparation protocols will circumvent this issue.</div>
</li> <li> <para-num num="[0911]"> </para-num> <div class="description-line" id="p-0873" num="0911">By far the biggest barrier Applicants anticipate is on the analytical side. Perturb-seq generates massive amounts of data, but these data are intrinsically noisy due both to the low capture efficiencies of single-cell RNA-seq protocols and more fundamentally, the intrinsic stochasticity of transcription at the single-cell level. Rigorously extracting meaningful conclusions will require new tools and algorithms (Satija et al. 2015, Heimberg et al. 2016, Trapnell 2015, Grun, van Oudenaarden 2015), as well as new statistical approaches for assessing the significance of claims. Applicants have made some progress on this front, and showed how single-cell data can be used for example to objectively decouple the branches of the UPR, uncover subtle subpopulations within cells of the same type, and infer larger programs of gene expression using correlated expression. Along with previous successes (Macosko et al. 2015, Klein et al. 2015, Jaitin et al. 2014), and the novel analytical approaches presented in an accompanying manuscript (Dixit et al., co-submitted manuscript), Applicants anticipate that large-scale analyses of single cell behavior will enable systematic discovery of the complex and overlapping regulatory programs at work within cells. As perturb-seq scales, it will present computational challenges on the same scale as other “big data” fields.</div>
</li> <li> <para-num num="[0912]"> </para-num> <div class="description-line" id="p-0874" num="0912">Our experiments also provide important insights into our understanding of the logic used by the mammalian UPR to sense and respond appropriately to the diversity of challenges faced by the ER. A central question in the field of ER homeostasis is why metazoan cells have evolved to have three independent and mechanistically distinct sensors of protein misfolding in the ER. As expected from previous work (Acosta-Alvear et al. 2007, Han et al. 2013, Lee, Iwakoshi &amp; Glimcher 2003, Shoulders et al. 2013), epistasis analysis using combinatorial depletion of the three main regulators of the UPR revealed both distinct and overlapping programs of gene expression controlled by PERK, ATF6, and IRE1α. The clean divisions Applicants observed at the single-cell level highlight the importance of performing these experiments in a pooled format, as sensitive phenotypes like stress responses might otherwise be overwhelmed by experimental variation. PERK/ATF4 had the largest regulon in our data, while ATF6 and IRE1α had more overlapping targets, with IRE1 having more unique targets such as the translocon. Additionally, Applicants observed that the three UPR branches were able to operate independently both at the bulk level, as different genetic perturbations showed differential activation of the three branches, and at the single-cell level.</div>
</li> <li> <para-num num="[0913]"> </para-num> <div class="description-line" id="p-0875" num="0913">Our genome-wide screen identified a broad diversity of genetic perturbations that activate signaling from the IRE1α branch of the UPR, including some categories of genes that were not expected from analogous yeast screens (Jonikas et al. 2009). Subjecting these hits to perturb-seq showed that the screen in fact hit all three branches of the UPR, and that genes with similar functional roles induced the UPR in similar ways. Intriguingly, Applicants also found that the signatures of the three branches of the UPR were implicit in the correlations seen in the perturb-seq experiment, suggesting that in principle the hypothesis-driven epistasis experiment could be dispensable. Thus, perturb-seq could enable identification and characterization of novel pathways for which such a priori information is not available. The single-cell analysis also revealed a remarkable bifurcation of UPR branch activation within cells subject to identical perturbations. In particular, depletion of HSPA5 resulted in some cells primarily inducing ATF6, others both IRE1α and ATF6, and a distinct subset inducing all three UPR branches simultaneously. This example illustrates the utility of single-cell data: bulk RNA-seq would in this case describe an average state that no cell actually occupies, just as it obscures the effect of other factors like the cell cycle. As all HSPA5-perturbed cells were treated identically, the cause of such marked differences in behavior remains in question. The simplest answer, variability in knockdown efficiency, does not appear to play a role, though the time course of HSPA5 depletion might. But this possibility suggests a real, interesting biological hypothesis, namely that there is a temporal ordering to the activation of the three pathways in HSPA5-depleted cells (Lin et al. 2007).</div>
</li> <li> <para-num num="[0914]"> </para-num> <div class="description-line" id="p-0876" num="0914">Applicants also found that different perturbations show clear differences in their propensity to activate the different branches of the UPR. Particularly intriguing in this regard were the three components of the protein-conducting channel encoded by SEC61A1, SEC61G, and SEC61B, repression of which selectively activated the IRE1α branch of the UPR. These results are notable in light of recent studies suggesting that IRE1α acts in proximity to the translocon (Plumb et al. 2015, Kanda et al. 2016). Given that Applicants and others have observed regulation of translocon expression to be uniquely under IRE1α control (Shoulders et al. 2013), this suggests a feedback model in which IRE1α actively monitors the state of translocation. Interestingly, isolated IRE1α induction would enable repair to or upregulation of the translocation machinery without a broader UPR induction, preventing potentially inappropriate responses including cell death (Hetz 2012). Additionally, further study of selective UPR induction using genetic perturbations, as Applicants have shown here, could also inform efforts to pharmacologically manipulate the UPR to enhance specific repair and prosurvival pathways (Shoulders et al. 2013). More broadly, this provides a concrete example of how the three distinct sensors of ER homeostasis allow the cell to monitor distinct types of ER stress and provide nuanced responses tailored to the specific challenge.</div>
</li> <li> <para-num num="[0915]"> </para-num> <div class="description-line" id="p-0877" num="0915">Applicants believe that their study of the mammalian UPR serves as a blueprint for the study of complex and overlapping transcriptional networks, in which a primary genome-wide screen serves as the input to more detailed analysis via perturb-seq. Applicants independently uncovered both well-known features of the UPR and new and unexpected findings. As the technology scales, it should be possible to conduct saturating analyses of every major pathway. Our success here, and the parallel success in understanding dendritic cell activation described herein, speak well to potential of the perturb-seq approach to become a standard strategy for understanding regulatory interactions in the cell.</div>
</li> <heading id="h-0085">Materials and Methods</heading>
<li> <para-num num="[0916]"> </para-num> <div class="description-line" id="p-0878" num="0916">Pool construction and transduction. A lentivirus backbone was constructed containing antiparallel cassettes of a PolIII promoter for sgRNA construction and a PolII promoter for expression of a selection marker and a polyadenylated RNA barcode. This construction avoided early termination of the viral transcript. A diverse library of RNA barcodes was inserted (approximately 1,000). Finally, sgRNAs were inserted. Association between RNA barcodes and sgRNAs was performed using Sanger sequencing. Lentivirus was made using 293T cells transfected with pBA439, and pMD2.G (Addgene 12259) at a 10:10:1 ratio, using Lipofectamine LTX and plus reagents according to the manufacturer's instructions.</div>
</li> <li> <para-num num="[0917]"> </para-num> <div class="description-line" id="p-0879" num="0917">For the mouse experiments, BMDCs were isolated from 6- to 8-week-old constitutive Cas9-expressing female mice and used as described previously (Platt et al., 2014). Cells were infected with the pooled lentiviral library at an MOI of approximately 0.2 at day 2. At day 9, BMDCs were stimulated with 100 ng/ml LPS. Cells were harvested for library preparation at 0hr and 3 hr post stimulation.</div>
</li> <li> <para-num num="[0918]"> </para-num> <div class="description-line" id="p-0880" num="0918">For the human experiments, k562 transduced with Cas9 were purified by high GFP expression. These cells were transduced with the pooled lentiviral library at an MOI of approximately 0.2 for most experiments and an MOI of 1 for the high MOI experiment. Cells were harvested for library preparation 7 days post transduction for most experiments and 14 days post transduction for the second time point promoters experiment.</div>
</li> <li> <para-num num="[0919]"> </para-num> <div class="description-line" id="p-0881" num="0919">Library preparation. After cells were collected they were processed into single cell RNA-seq libraries following the manufacturer's protocol of the commercial 10X Chromium system.</div>
</li> <li> <para-num num="[0920]"> </para-num> <div class="description-line" id="p-0882" num="0920">Bone marrow derived dendritic cells (BMDCs). All animal protocols were reviewed and approved by the MIT/Whitehead Institute/Broad Institute Committee on Animal Care (CAC protocol 0609-058-12). To obtain sufficient number of cells, Applicants implemented a modified version of the DCs isolation protocol as previously described (Amit et al. 2009; Chevrier et al. 2011; Garber et al. 2012; Lutz et al. 1999; Rabani et al. 2011). Briefly, for all CRISPR knockout experiments six- to eight-week old constitutive Cas9-expressing female mice were used as described previously (Platt et al. 2014). RPMI medium (Invitrogen) supplemented with 10% heat inactivated FBS (Invitrogen), β-mercaptoethanol (50μM, Invitrogen), L-glutamine (2mM, VWR), penicillin/streptomycin (100 U/ml, VWR), MEM non-essential amino acids (1X, VWR), HEPES (10 mM, VWR), sodium pyruvate (1 mM, VWR), and GM-CSF (20 ng/ml; Peprotech) was used throughout the study. At day 0, cells were collected from femora and tibiae and plated in 100 mm non tissue culture treated plastic dishes using 10 ml medium per plate at concentration of 2×10<sup>5</sup>/ml. At day 2, cells were fed with another 10m1 of medium per dish. At day 5, 12 ml of the medium were carefully removed (to avoid removal of cells) and 10m1 of fresh medium were added back to the original dish. Cells were fed with another 5 ml medium at day 7. At day 8, all non-adherent and loosely bound cells were collected and harvested by centrifugation. Cells were then re-suspended with medium, plated at a concentration of 10×10<sup>6 </sup>cells in 10 ml medium per 100 mm dish. At day 9, cells were stimulated for with LPS (100 ng/ml, rough, ultrapure <i>E. coli </i>K12 strain, Invitrogen) and harvested. Cells were always plated at concentration of 2×10<sup>5</sup>/ml at day 0. Cells were harvested post stimulation after 0 hr or 3 hr and cells from cultures that contained 10% BFP positive cells were sorted for BFP+ and GFP+ (contain CAS9).</div>
</li> <li> <para-num num="[0921]"> </para-num> <div class="description-line" id="p-0883" num="0921">Cell Culture. Human K562 cells were purchased; murine BMDC were isolated from 6- to 8-week-old constitutive Cas9-expressing female mice.</div>
</li> <li> <para-num num="[0922]"> </para-num> <div class="description-line" id="p-0884" num="0922">K562 cells were grown to a confluence of 30-60% and spun down at 300×g for 5 min. The supernatant was removed, and cells were resuspended in 1 mL of 1×PBS+0.2% BSA (Sigma cat #A8806) and re-spun at 300×g for 3 min. The supernatant was again removed, and the cells re- suspended in 1 mL of 1×PBS, passed through a 40-micron cell strainer (Falcon, VWR cat #21008- 949), and counted. For 10X, cells were diluted to the final concentration in lx PBS+200 μg/mL BSA (NEB, cat # B9000S).</div>
</li> <li> <para-num num="[0923]"> </para-num> <div class="description-line" id="p-0885" num="0923">K562 cell cultures. Applicants used transgenic K562 cells constitutively expressing Cas9. K562 cells were transduced using several titers of virus and cells where spin infected in 2000 rpm for 30 min. For the low MOI experiment Applicants used cultures that contained 10% BFP+ and for the high MOI 50% BFP+. Cells were grown in RPMI Medium 1640+GlutaMAX (ThermoFisher)+10% heat inactivated FBS (Invitrogen). Cells were grown to a confluence of 30-60% and spun down at 300×g for 5 min. The supernatant was removed, and cells were suspended in 5 mL of 1×PBS+ 0.2% BSA (Sigma cat #A8806) for sorting: BFP+ and GFP+ (CAS9 expressing) cells were sorted. Cells were harvested for library preparation 7 days post transduction for most experiments and 13 days post transduction for the second time point of the TF pool experiment. After sorting BFP+ GFP+ cells passed through a 40-micron cell strainer (Falcon, VWR cat #21008- 949), wash twice and counted.</div>
</li> <li> <para-num num="[0924]"> </para-num> <div class="description-line" id="p-0886" num="0924">Library preparation. After cells were collected, they were processed into single cell RNA-seq libraries following the manufacturer's protocol of the commercial 10X Chromium system.</div>
</li> <li> <para-num num="[0925]"> </para-num> <div class="description-line" id="p-0887" num="0925">Single cell library preparation. Prior to analysis, cells were diluted to the final concentration in 1×PBS +200 μg/mL BSA (NEB, cat # B9000S). Sorted cells (BMDCs or K562 cells) where loaded on the 10X Chromium system (Zheng et al., 2016) (8,000 cells/channel) and single cell RNA-seq libraries were generated following the manufacturer's instructions.</div>
</li> <li> <para-num num="[0926]"> </para-num> <div class="description-line" id="p-0888" num="0926">Following WTA, a fraction of the WTA was used to amplify sgRNA barcode using a dial-out PCR strategy with the primer sequences below (the full primer sequence is a concatenation of the columns). The template material was approximately 5 ng of WTA libraries. 25 cycles of PCR were performed using one of the dial-out primers below with the P7 Illumina reverse primer.</div>
</li> <li> <para-num num="[0927]"> </para-num> <div class="description-line" id="p-0889" num="0927">Preparation of RNA barcode sub-libraries. In order obtain high confidence information about the identities of which sgRNAs were present in each cell, a dial-out PCR strategy was performed using the primer sequences below (the full primer sequence is a concatenation of the columns). The template material was approximately 5 ng of whole transcriptome amplification libraries obtained after a 10X library, 25 cycles of PCR was performed using one of the dial-out primers below with the a P7 Ilumina primer reverse primer.</div>
</li> <li> <para-num num="[0928]"> </para-num> <div class="description-line" id="p-0890" num="0928">Primer sequences:</div>
</li> <li> <div class="description-line" id="p-0891" num="0000">
<tables id="TABLE-US-00008" num="00008">
<patent-tables colsep="0" frame="none" rowsep="0" tabstyle="monospace">
<table align="left" class="description-table" cols="4" colsep="0" rowsep="0" width="100%">
<thead>
<tr class="description-tr">
<td align="center" class="description-td" colspan="4" nameend="4" namest="1" rowsep="1"> </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">Primer</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">(BFP location</td>
</tr>
<tr class="description-tr">
<td class="description-td">P5</td>
<td class="description-td">Barcode</td>
<td class="description-td">R1</td>
<td class="description-td">of pBA439)</td>
</tr>
<tr class="description-tr">
<td align="center" class="description-td" colspan="4" nameend="4" namest="1" rowsep="1"> </td>
</tr>
</thead>
<tbody><tr class="description-tr">
<td class="description-td">AATGATACGGC</td>
<td class="description-td">NNNNNNN</td>
<td class="description-td">TCGTCGGCAGCG</td>
<td class="description-td">TAGCAAACTGGGG</td>
</tr>
<tr class="description-tr">
<td class="description-td">GACCACCGAGA</td>
<td class="description-td">N</td>
<td class="description-td">TCAGATGTGTAT</td>
<td class="description-td">CACAAGC</td>
</tr>
<tr class="description-tr">
<td class="description-td">TCTACAC</td>
<td class="description-td"> </td>
<td class="description-td">AAGAGACAG</td>
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">AATGATACGGC</td>
<td class="description-td">TCGCCTT</td>
<td class="description-td">TCGTCGGCAGCG</td>
<td class="description-td">TAGCAAACTGGGG</td>
</tr>
<tr class="description-tr">
<td class="description-td">GACCACCGAGA</td>
<td class="description-td">A</td>
<td class="description-td">TCAGATGTGTAT</td>
<td class="description-td">CACAAGC</td>
</tr>
<tr class="description-tr">
<td class="description-td">TCTACAC</td>
<td class="description-td"> </td>
<td class="description-td">AAGAGACAG</td>
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">AATGATACGGC</td>
<td class="description-td">CTAGTAC</td>
<td class="description-td">TCGTCGGCAGCG</td>
<td class="description-td">TAGCAAACTGGGG</td>
</tr>
<tr class="description-tr">
<td class="description-td">GACCACCGAGA</td>
<td class="description-td">G</td>
<td class="description-td">TCAGATGTGTAT</td>
<td class="description-td">CACAAGC</td>
</tr>
<tr class="description-tr">
<td class="description-td">TCTACAC</td>
<td class="description-td"> </td>
<td class="description-td">AAGAGACAG</td>
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">AATGATACGGC</td>
<td class="description-td">TTCTGCC</td>
<td class="description-td">TCGTCGGCAGCG</td>
<td class="description-td">TAGCAAACTGGGG</td>
</tr>
<tr class="description-tr">
<td class="description-td">GACCACCGAGA</td>
<td class="description-td">T</td>
<td class="description-td">TCAGATGTGTAT</td>
<td class="description-td">CACAAGC</td>
</tr>
<tr class="description-tr">
<td class="description-td">TCTACAC</td>
<td class="description-td"> </td>
<td class="description-td">AAGAGACAG</td>
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">AATGATACGGC</td>
<td class="description-td">GCTCAGG</td>
<td class="description-td">TCGTCGGCAGCG</td>
<td class="description-td">TAGCAAACTGGGG</td>
</tr>
<tr class="description-tr">
<td class="description-td">GACCACCGAGA</td>
<td class="description-td">A</td>
<td class="description-td">TCAGATGTGTAT</td>
<td class="description-td">CACAAGC</td>
</tr>
<tr class="description-tr">
<td class="description-td">TCTACAC</td>
<td class="description-td"> </td>
<td class="description-td">AAGAGACAG</td>
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">AATGATACGGC</td>
<td class="description-td">CATGCCT</td>
<td class="description-td">TCGTCGGCAGCG</td>
<td class="description-td">TAGCAAACTGGGG</td>
</tr>
<tr class="description-tr">
<td class="description-td">GACCACCGAGA</td>
<td class="description-td">A</td>
<td class="description-td">TCAGATGTGTAT</td>
<td class="description-td">CACAAGC</td>
</tr>
<tr class="description-tr">
<td class="description-td">TCTACAC</td>
<td class="description-td"> </td>
<td class="description-td">AAGAGACAG</td>
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">AATGATACGGC</td>
<td class="description-td">GTAGAGA</td>
<td class="description-td">TCGTCGGCAGCG</td>
<td class="description-td">TAGCAAACTGGGG</td>
</tr>
<tr class="description-tr">
<td class="description-td">GACCACCGAGA</td>
<td class="description-td">G</td>
<td class="description-td">TCAGATGTGTAT</td>
<td class="description-td">CACAAGC</td>
</tr>
<tr class="description-tr">
<td class="description-td">TCTACAC</td>
<td class="description-td"> </td>
<td class="description-td">AAGAGACAG</td>
</tr>
<tr class="description-tr">
<td align="center" class="description-td" colspan="4" nameend="4" namest="1" rowsep="1"> </td>
</tr>
</tbody></table>
</patent-tables>
</tables>
</div>
</li> <li> <para-num num="[0929]"> </para-num> <div class="description-line" id="p-0892" num="0929">P7 Ilumina Reverse Primer:</div>
</li> <li> <div class="description-line" id="p-0893" num="0000">
<tables id="TABLE-US-00009" num="00009">
<patent-tables colsep="0" frame="none" rowsep="0" tabstyle="monospace">
<table align="left" class="description-table" cols="2" colsep="0" rowsep="0" width="100%">
<tbody><tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">CAAGCAGAAGACGGCATACGAGAT</td>
</tr>
</tbody></table>
</patent-tables>
</tables>
</div>
</li> <li> <para-num num="[0930]"> </para-num> <div class="description-line" id="p-0894" num="0930">The PCR protocol is as follows:</div>
</li> <li> <div class="description-line" id="p-0895" num="0000">
<tables id="TABLE-US-00010" num="00010">
<patent-tables colsep="0" frame="none" rowsep="0">
<table align="left" class="description-table" cols="2" colsep="0" rowsep="0" width="100%">
<thead>
<tr class="description-tr">
<td class="description-td"> </td>
<td align="center" class="description-td" colspan="2" nameend="1" namest="offset" rowsep="1"> </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">1 rxn</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td align="center" class="description-td" colspan="2" nameend="1" namest="offset" rowsep="1"> </td>
</tr>
</thead>
<tbody><tr class="description-tr">
<td class="description-td"> </td>
</tr>
</tbody></table>
<table align="left" class="description-table" cols="3" colsep="0" rowsep="0" width="100%">
<tbody><tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">Q5 2X master mix</td>
<td class="description-td">25</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">P7 primer @10 uM</td>
<td class="description-td">1.25</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">pBA439_rev @ 10 uM</td>
<td class="description-td">1.25</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">Template (5 ng total)</td>
<td class="description-td">x</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">water (qs up to final rxn of 50 ul)</td>
<td class="description-td">y</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">total</td>
<td class="description-td">50</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td align="center" class="description-td" colspan="3" nameend="2" namest="offset" rowsep="1"> </td>
</tr>
</tbody></table>
</patent-tables>
</tables>
</div>
</li> <li> <div class="description-line" id="p-0896" num="0000">
<tables id="TABLE-US-00011" num="00011">
<patent-tables colsep="0" frame="none" rowsep="0">
<table align="left" class="description-table" cols="4" colsep="0" rowsep="0" width="100%">
<thead>
<tr class="description-tr">
<td class="description-td"> </td>
<td align="center" class="description-td" colspan="4" nameend="3" namest="offset" rowsep="1"> </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">Temp</td>
<td class="description-td">time</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td align="center" class="description-td" colspan="4" nameend="3" namest="offset" rowsep="1"> </td>
</tr>
</thead>
<tbody><tr class="description-tr">
<td class="description-td"> </td>
</tr>
</tbody></table>
<table align="left" class="description-table" cols="5" colsep="0" rowsep="0" width="100%">
<tbody><tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">98 C.</td>
<td class="description-td">10</td>
<td class="description-td">s</td>
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">98 C.</td>
<td class="description-td">2</td>
<td class="description-td">s</td>
<td class="description-td">repeat for 25</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">65 C.</td>
<td class="description-td">5</td>
<td class="description-td">s</td>
<td class="description-td">cycles</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">72 C.</td>
<td class="description-td">10</td>
<td class="description-td">s</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">72 C.</td>
<td class="description-td">1</td>
<td class="description-td">min</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td align="center" class="description-td" colspan="5" nameend="4" namest="offset" rowsep="1"> </td>
</tr>
</tbody></table>
</patent-tables>
</tables>
</div>
</li> <li> <para-num num="[0931]"> </para-num> <div class="description-line" id="p-0897" num="0931">The resulting PCR product was purified with Agencourt AMPure XP beads at 0.7× and sequenced.</div>
</li> <li> <para-num num="[0932]"> </para-num> <div class="description-line" id="p-0898" num="0932">Read alignment and Generation of Expression Matrix. A digital expression matrix was obtained for each experiment using 10X's Cellranger pipeline with default parameters. The software uses STAR for alignment. All subsequence analysis information is available and maintained in current form in the following Git repository and documentation available through ReadtheDocs.</div>
</li> <li> <para-num num="[0933]"> </para-num> <div class="description-line" id="p-0899" num="0933">Alignment of initial abundances. Applicants compiled a dictionary of RNA barcodes associated with sgRNA from Sanger sequencing each part of the construct separately. Applicants devised an alternative cloning protocol for larger libraries to enable NGS pairing of sgRNA/RNA barcodes, but did not use it for any of the results in this paper. The initial abundance of each plasmid in the pool was quantified using NGS of the RNA barcode. The RNA barcode/sgRNA dictionary was used to convert the readout into a relative abundance estimate of sgRNA in the initial pool.</div>
</li> <li> <para-num num="[0934]"> </para-num> <div class="description-line" id="p-0900" num="0934">Alignment of Cell barcode/RNA barcodes libraries. In order to associate cell barcodes with guides, the same RNA barcode/sgRNA dictionary was used. Then NGS sequencing may have contained a complex library of molecules such as a full RNA-seq library reads. Paired end reads containing a cell barcode UMI on one side and an RNA barcode on another side were isolated and collapsed into unique molecules by first demultiplexing a sequencing run using bc12fastq2 with the followign options—create-fastq-for-index-reads—barcode-mismatches 1—no-lane-splitting—mask-short-adapter- reads 5—minimum-trimmed-read-length 5 using a sample-sheet containing one line with polyG tracts in the index read columns (see below). The resulting Undetermined fastq files were split using kentools into two folders called splitl and split2 containing chunked R1 and R2 reads respectively.</div>
</li> <li> <div class="description-line" id="p-0901" num="0000">
<tables id="TABLE-US-00012" num="00012">
<patent-tables colsep="0" frame="none" pgwide="1" rowsep="0" tabstyle="monospace">
<table align="left" class="description-table" cols="6" colsep="0" rowsep="0" width="100%">
<thead>
<tr class="description-tr">
<td align="center" class="description-td" colspan="6" nameend="6" namest="1" rowsep="1"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">Sample_ID</td>
<td class="description-td">Sample_Name</td>
<td class="description-td">I7_Index_ID</td>
<td class="description-td">index</td>
<td class="description-td">I5_Index_ID</td>
<td class="description-td">index2</td>
</tr>
<tr class="description-tr">
<td align="center" class="description-td" colspan="6" nameend="6" namest="1" rowsep="1"> </td>
</tr>
</thead>
<tbody><tr class="description-tr">
<td class="description-td">1</td>
<td class="description-td">placeholder</td>
<td class="description-td">silly1</td>
<td class="description-td">GGGGGGG</td>
<td class="description-td">silly2</td>
<td class="description-td">GGGGGGGG</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">GGGGGGG</td>
</tr>
<tr class="description-tr">
<td align="center" class="description-td" colspan="6" nameend="6" namest="1" rowsep="1"> </td>
</tr>
</tbody></table>
</patent-tables>
</tables>
</div>
</li> <li> <para-num num="[0935]"> </para-num> <div class="description-line" id="p-0902" num="0935">Then the reads were concatenated and RNA barcode reads isolated using the following UNIX commands.</div>
</li> <li> <div class="description-line" id="p-0903" num="0000">
<tables id="TABLE-US-00013" num="00013">
<patent-tables colsep="0" frame="none" rowsep="0" tabstyle="monospace">
<table align="left" class="description-table" cols="2" colsep="0" rowsep="0" width="100%">
<tbody><tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">paste ./split1/$f1 ./split2/$f2 | awk</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">‘{if (NR%4==1) printf($2“\t”); else if</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">(NR%4==2)printf($0“\n”)}’ | grep</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">GGCACAAGCTTAATTAAGAATT</td>
</tr>
</tbody></table>
</patent-tables>
</tables>
</div>
</li> <li> <para-num num="[0936]"> </para-num> <div class="description-line" id="p-0904" num="0936">with GGCACAAGCTTAATTAAGAATT corresponding to a constant sequence within the RNA barcode, and $f1 and $f2 corresponding to input arguments containing the read1_splitfile and read2_splitfile respectively and creating an output split file.</div>
</li> <li> <para-num num="[0937]"> </para-num> <div class="description-line" id="p-0905" num="0937">All split files containing a particular index barcode are finally concatenated and collapsed to unique molecules using the following command.</div>
</li> <li> <div class="description-line" id="p-0906" num="0000">cat *${inputbc}.txt|sort|uniq-c|sort-k1, 1 g|awk ‘{print $1″\t″$2″\t″$3″-″$4}’</div>
</li> <li> <para-num num="[0938]"> </para-num> <div class="description-line" id="p-0907" num="0938">the result file format looks as follows (grey highlighting for actual barcode portion of RNA barcode read):</div>
</li> <li> <div class="description-line" id="p-0908" num="0000">
<tables id="TABLE-US-00014" num="00014">
<patent-tables colsep="0" frame="none" rowsep="0" tabstyle="monospace">
<table align="left" class="description-table" cols="1" colsep="0" rowsep="0" width="100%">
<tbody><tr class="description-tr">
<td class="description-td">2791:N:0:</td>
</tr>
<tr class="description-tr">
<td class="description-td">GGTGATACCTCATT + TCGCATAA</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">CGCAAACTGGGGCACAAGCTTAATTAAGAATTCGATCAACGCAGAGACGG</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">CCTAGGCGTATAAGT</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">2821:N:0:</td>
</tr>
<tr class="description-tr">
<td class="description-td">TCGAGCCTTATGGC + TCGCATAA</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">CGCAAACTGGGGCACAAGCTTAATTAAGAATTGCTTGACTCGTTAGCGAG</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">CCTAGCACGATACCG</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">2861:N:0:</td>
</tr>
<tr class="description-tr">
<td class="description-td">ACTCGAGAGTTCGA + TCGCATAA</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">CGCAAACTGGGGCACAAGCTTAATTAAGAATTCGATCAACGCAGAGACGG</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">CCTAGGTCTAATGAA</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">2941:N:0:</td>
</tr>
<tr class="description-tr">
<td class="description-td">GCACGTCTACTAGC + TCGCATAA</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">CGCAAACTGGGGCACAAGCTTAATTAAGAATTCTAACTCAGCGACTGGAG</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">CCTAGCATGTGCCCG</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">2971:N:0:</td>
</tr>
<tr class="description-tr">
<td class="description-td">ATAGATTGTCCGAA + TCGCATAA</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">CGCAAACTGGGGCACAAGCTTAATTAAGAATTGCTTGACTCGTTAGCGAG</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">CCTAGAAAAGGATGG</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">2991:N:0:</td>
</tr>
<tr class="description-tr">
<td class="description-td">GAGCAGGAGCTATG + TCGCATAA</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">CGCAAACTGGGGCACAAGCTTAATTAAGAATTAAACCCTCACTGCCGACG</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">CCTAGACCTGTTACG</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">3331:N:0:</td>
</tr>
<tr class="description-tr">
<td class="description-td">GGAGGCCTGTTACG + TCGCATAA</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">CGCAAACTGGGGCACAAGCTTAATTAAGAATTAGGGCTTGCAGTGCACGG</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">CCTAGAACGTCAAGA</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">3391:N:0:</td>
</tr>
<tr class="description-tr">
<td class="description-td">CGACTCACGTTCAG + TCGCATAA</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">CGCAAACTGGGGCACAAGCTTAATTAAGAATTCGATCAACGCAGAGACGG</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">CCTAGCAGACTGGGC</td>
</tr>
</tbody></table>
</patent-tables>
</tables>
</div>
</li> <li> <para-num num="[0939]"> </para-num> <div class="description-line" id="p-0909" num="0939">Counts 1:N:0:Cellbarcode+Index/SampleBarcode RNAbarcoderead-UMI</div>
</li> <li> <para-num num="[0940]"> </para-num> <div class="description-line" id="p-0910" num="0940">The resulting file along with the preassociated RNA barcode, sgRNA dictionary is parsed using a custom python script to create a new table of probability estimates of which sgRNA are present in each cell. The probability estimate is thresholded to create a dictionary of which cell barcodes contain which sgRNAs.</div>
</li> <li> <para-num num="[0941]"> </para-num> <div class="description-line" id="p-0911" num="0941">Fit of distribution of guides per cell. The distribution of guides per cell was fit using a maximum likelihood approach.</div>
</li> <li> <para-num num="[0942]"> </para-num> <div class="description-line" id="p-0912" num="0942">Linear model. To fit the linear model Applicants compiled our covariate matrix X and our expression matrix (or one of the continuous covariates; as done for some assignments of cell states; below) as our matrix Y. Applicants then used the Python implementation of elastic net with the following parameters, to fit our model. sklearn.linear_model.ElasticNet(11_ratio=0.5,alpha=0.0005,max_iter=10000)</div>
</li> <li> <para-num num="[0943]"> </para-num> <div class="description-line" id="p-0913" num="0943">Alternating descent fit of perturbation probability. To account for the contribution of unperturbed cells in the population containing a particular sgRNA, Applicants constructed an approach in which the presence of sgRNA in a given cell was converted into a probability measure of that sgRNA having a phenotypic effect on the cell, as follows:</div>
</li> <li> <para-num num="[0944]"> </para-num> <div class="description-line" id="p-0914" num="0944">First, Applicants fit using the multivariate regression on</div>
</li> <heading id="h-0086">Y=X</heading>
<li> <para-num num="[0945]"> </para-num> <div class="description-line" id="p-0915" num="0945">
<maths id="MATH-US-00005" num="00005">
<math overflow="scroll">
<mrow>
<mstyle>
<mspace height="4.2ex" width="4.2em"> </mspace>
</mstyle>
<mo></mo>
<mrow>
<mrow>
<mi>LL</mi>
<mo></mo>
<mrow>
<mo>(</mo>
<mi>y</mi>
<mo>)</mo>
</mrow>
</mrow>
<mo>=</mo>
<mrow>
<mrow>
<mi>log</mi>
<mo></mo>
<mrow>
<mo>(</mo>
<mfrac>
<mrow>
<mfrac>
<mn>1</mn>
<mrow>
<mn>2</mn>
<mo></mo>
<mi>πσ</mi>
</mrow>
</mfrac>
<mo></mo>
<msup>
<mi>e</mi>
<mfrac>
<msup>
<mrow>
<mo>(</mo>
<mrow>
<mi>y</mi>
<mo>-</mo>
<msub>
<mi>μ</mi>
<mn>1</mn>
</msub>
</mrow>
<mo>)</mo>
</mrow>
<mn>2</mn>
</msup>
<mrow>
<mn>2</mn>
<mo></mo>
<mi>σ</mi>
</mrow>
</mfrac>
</msup>
</mrow>
<mrow>
<mfrac>
<mn>1</mn>
<mrow>
<mn>2</mn>
<mo></mo>
<mi>πσ</mi>
</mrow>
</mfrac>
<mo></mo>
<msup>
<mi>e</mi>
<mfrac>
<msup>
<mrow>
<mo>(</mo>
<mrow>
<mi>y</mi>
<mo>-</mo>
<msub>
<mi>μ</mi>
<mn>0</mn>
</msub>
</mrow>
<mo>)</mo>
</mrow>
<mn>2</mn>
</msup>
<mrow>
<mn>2</mn>
<mo></mo>
<mi>σ</mi>
</mrow>
</mfrac>
</msup>
</mrow>
</mfrac>
<mo>)</mo>
</mrow>
</mrow>
<mo>=</mo>
<mfrac>
<mrow>
<msup>
<mrow>
<mo>(</mo>
<mrow>
<mi>y</mi>
<mo>-</mo>
<msub>
<mi>μ</mi>
<mn>0</mn>
</msub>
</mrow>
<mo>)</mo>
</mrow>
<mn>2</mn>
</msup>
<mo>-</mo>
<msup>
<mrow>
<mo>(</mo>
<mrow>
<mi>y</mi>
<mo>-</mo>
<msub>
<mi>μ</mi>
<mn>1</mn>
</msub>
</mrow>
<mo>)</mo>
</mrow>
<mn>2</mn>
</msup>
</mrow>
<mrow>
<mn>2</mn>
<mo></mo>
<mi>σ</mi>
</mrow>
</mfrac>
</mrow>
</mrow>
</mrow>
</math>
</maths>
<maths id="MATH-US-00005-2" num="00005.2">
<math overflow="scroll">
<mrow>
<mrow>
<mi>P</mi>
<mo></mo>
<mrow>
<mo>(</mo>
<mrow>
<msub>
<mi>X</mi>
<mi>j</mi>
</msub>
<mo>=</mo>
<mrow>
<mn>1</mn>
<mo>|</mo>
<mi>Y</mi>
</mrow>
</mrow>
<mo>)</mo>
</mrow>
</mrow>
<mo>=</mo>
<mrow>
<mfrac>
<mrow>
<mrow>
<mi>P</mi>
<mo></mo>
<mrow>
<mo>(</mo>
<mrow>
<mrow>
<mi>Y</mi>
<mo>|</mo>
<msub>
<mi>X</mi>
<mi>j</mi>
</msub>
</mrow>
<mo>=</mo>
<mn>1</mn>
</mrow>
<mo>)</mo>
</mrow>
</mrow>
<mo></mo>
<mrow>
<mi>P</mi>
<mo></mo>
<mrow>
<mo>(</mo>
<mrow>
<msub>
<mi>X</mi>
<mi>j</mi>
</msub>
<mo>=</mo>
<mn>1</mn>
</mrow>
<mo>)</mo>
</mrow>
</mrow>
</mrow>
<mrow>
<mi>P</mi>
<mo></mo>
<mrow>
<mo>(</mo>
<mi>Y</mi>
<mo>)</mo>
</mrow>
</mrow>
</mfrac>
<mo>=</mo>
<mrow>
<mfrac>
<mrow>
<mrow>
<mi>P</mi>
<mo></mo>
<mrow>
<mo>(</mo>
<mrow>
<mrow>
<mi>Y</mi>
<mo>|</mo>
<msub>
<mi>X</mi>
<mi>j</mi>
</msub>
</mrow>
<mo>=</mo>
<mn>1</mn>
</mrow>
<mo>)</mo>
</mrow>
</mrow>
<mo></mo>
<mrow>
<mi>P</mi>
<mo></mo>
<mrow>
<mo>(</mo>
<mrow>
<msub>
<mi>X</mi>
<mi>j</mi>
</msub>
<mo>=</mo>
<mn>1</mn>
</mrow>
<mo>)</mo>
</mrow>
</mrow>
</mrow>
<mrow>
<mrow>
<mrow>
<mi>P</mi>
<mo></mo>
<mrow>
<mo>(</mo>
<mrow>
<mrow>
<mi>Y</mi>
<mo>|</mo>
<msub>
<mi>X</mi>
<mi>j</mi>
</msub>
</mrow>
<mo>=</mo>
<mn>0</mn>
</mrow>
<mo>)</mo>
</mrow>
</mrow>
<mo></mo>
<mrow>
<mi>P</mi>
<mo></mo>
<mrow>
<mo>(</mo>
<mrow>
<msub>
<mi>X</mi>
<mi>j</mi>
</msub>
<mo>=</mo>
<mn>0</mn>
</mrow>
<mo>)</mo>
</mrow>
</mrow>
</mrow>
<mo>+</mo>
<mrow>
<mrow>
<mi>P</mi>
<mo></mo>
<mrow>
<mo>(</mo>
<mrow>
<mrow>
<mi>Y</mi>
<mo>|</mo>
<msub>
<mi>X</mi>
<mi>j</mi>
</msub>
</mrow>
<mo>=</mo>
<mn>1</mn>
</mrow>
<mo>)</mo>
</mrow>
</mrow>
<mo></mo>
<mrow>
<mi>P</mi>
<mo></mo>
<mrow>
<mo>(</mo>
<mrow>
<msub>
<mi>X</mi>
<mi>j</mi>
</msub>
<mo>=</mo>
<mn>1</mn>
</mrow>
<mo>)</mo>
</mrow>
</mrow>
</mrow>
</mrow>
</mfrac>
<mo>=</mo>
<mfrac>
<mn>1</mn>
<mrow>
<mn>1</mn>
<mo>+</mo>
<msup>
<mi>e</mi>
<mrow>
<mo>-</mo>
<mrow>
<mi>LL</mi>
<mo></mo>
<mrow>
<mo>(</mo>
<mi>Y</mi>
<mo>)</mo>
</mrow>
</mrow>
</mrow>
</msup>
</mrow>
</mfrac>
</mrow>
</mrow>
</mrow>
</math>
</maths>
</div>
</li> <li> <para-num num="[0946]"> </para-num> <div class="description-line" id="p-0916" num="0946">Next, Applicants evaluate the fit with the guide covariate set to 0, Xo</div>
</li> <li> <div class="description-line" id="p-0917" num="0000">
<maths id="MATH-US-00006" num="00006">
<math overflow="scroll">
<mrow>
<mrow>
<mrow>
<mi>P</mi>
<mo></mo>
<mrow>
<mo>(</mo>
<mrow>
<msub>
<mi>X</mi>
<mi>j</mi>
</msub>
<mo>=</mo>
<mn>1</mn>
</mrow>
<mo>)</mo>
</mrow>
</mrow>
<mo>=</mo>
<mrow>
<mi>logistic</mi>
<mo></mo>
<mrow>
<mo>(</mo>
<mrow>
<msup>
<mrow>
<msub>
<mi>Σ</mi>
<mi>i</mi>
</msub>
<mo></mo>
<mrow>
<mo>[</mo>
<mrow>
<msub>
<mi>Y</mi>
<mi>ij</mi>
</msub>
<mo>-</mo>
<mrow>
<msub>
<mi>X</mi>
<mi>o</mi>
</msub>
<mo></mo>
<msub>
<mi>β</mi>
<mi>l</mi>
</msub>
</mrow>
</mrow>
<mo>]</mo>
</mrow>
</mrow>
<mn>2</mn>
</msup>
<mo>-</mo>
<msup>
<mrow>
<mo>[</mo>
<mrow>
<msub>
<mi>Y</mi>
<mi>ij</mi>
</msub>
<mo>-</mo>
<mover>
<msub>
<mi>Y</mi>
<mi>ij</mi>
</msub>
<mo>~</mo>
</mover>
</mrow>
<mo>]</mo>
</mrow>
<mn>2</mn>
</msup>
</mrow>
<mo>)</mo>
</mrow>
</mrow>
</mrow>
<mo>;</mo>
<mstyle>
<mspace height="0.8ex" width="0.8em"> </mspace>
</mstyle>
<mo></mo>
<mi>where</mi>
</mrow>
</math>
</maths>
<maths id="MATH-US-00006-2" num="00006.2">
<math overflow="scroll">
<mrow>
<mrow>
<mi>logistic</mi>
<mo></mo>
<mrow>
<mo>(</mo>
<mi>x</mi>
<mo>)</mo>
</mrow>
</mrow>
<mo>=</mo>
<mfrac>
<mn>1</mn>
<mrow>
<mn>1</mn>
<mo>+</mo>
<msup>
<mi>e</mi>
<mrow>
<mo>-</mo>
<mi>x</mi>
</mrow>
</msup>
</mrow>
</mfrac>
</mrow>
</math>
</maths>
</div>
</li> <li> <para-num num="[0947]"> </para-num> <div class="description-line" id="p-0918" num="0947">Finally, Applicants use the new covariate matrix X<sub>M </sub>to recompute β<sub>M</sub>.</div>
</li> <li> <para-num num="[0948]"> </para-num> <div class="description-line" id="p-0919" num="0948">Significance testing for coefficients of linear model. Applicants devised a permutation strategy to obtain an empirical null distribution of the coefficients associated with our sgRNA effects. Specifically, Applicants randomized the guide assignments to cells (such that co-occurrence between guides was preserved) and the linear model was recomputed with all other covariates being held constant. Applicants repeated this ten times. Applicants noticed that three significant as-yet latent factors impacted the empirical null distribution of coefficients: (1) the mean expression level of a gene; (2) the variance in expression of a gene; and (3) the number of cells a particular sgRNA was present in.</div>
</li> <li> <para-num num="[0949]"> </para-num> <div class="description-line" id="p-0920" num="0949">To control for these factors when assessing significance, each empirical null coefficient's value was assigned a point in 4-space: [Gene mean, gene variance, number of cells, value]. Applicants then estimated the multivariate density using a binning approach (np.histogramdd in Python). True nonzero coefficients were evaluated for significance relative to a matched set of bins (to create an empirical conditional cumulative probability distribution) conditioned on the first three factors.</div>
</li> <li> <para-num num="[0950]"> </para-num> <div class="description-line" id="p-0921" num="0950">A less stringent null was generated by obtaining a permuted distribution of coefficients based on a permuted cell to guide assignment, but only calculating the significance on a per guide basis without consideration of the mean expression level or variance of the gene being considered.</div>
</li> <li> <para-num num="[0951]"> </para-num> <div class="description-line" id="p-0922" num="0951">In both cases, Applicants used a Benjamini-Hochberg procedure to control for multiple hypothesis testing.</div>
</li> <li> <para-num num="[0952]"> </para-num> <div class="description-line" id="p-0923" num="0952">Residuals analysis. To determine the marginal effect of each covariate in explaining the observed gene expression variation, Applicants estimated the model R<sup>2 </sup>by cross-validated (trained on 80% of the data and tested against 20%) for the addition of each of the covariates.</div>
</li> <li> <para-num num="[0953]"> </para-num> <div class="description-line" id="p-0924" num="0953">To determine the extent to which our covariates explained the major axes of variation in our data, Applicants decomposed the residuals using the same randomized PCA approach described in the Definition of Cell States section. Two major metrics were evaluated: (1) the eigenvalue distribution, and (2) the extent to which the top loadings were enriched for biological terms.</div>
</li> <li> <para-num num="[0954]"> </para-num> <div class="description-line" id="p-0925" num="0954">Cross validated R<sup>2</sup>. To estimate the generalizability of the model, Applicants determined a cross-validated R<sup>2 </sup>by training our model on 80% of our data and determining the fit on the remaining 20%.</div>
</li> <li> <para-num num="[0955]"> </para-num> <div class="description-line" id="p-0926" num="0955">Definition of cell states. Cell states were defined using cells that did not have sgRNAs introduced. Starting from an expression matrix Y, variable genes are selected based on fitting a nonparametric loess regression (using a moving window of 25 percent of the data) to the relationship between the average expression of a gene and its respective coefficient of variation (after normalizing each cell for complexity). Genes with high residuals (i.e more variable than genes at comprable expression levels) were selected (approximately 1000 genes).</div>
</li> <li> <para-num num="[0956]"> </para-num> <div class="description-line" id="p-0927" num="0956">Next, the cell normalized expression matrix (the sum of the number of transcripts is renormalized to 10,000) and then log 2 transformed with a pseudocount of 1. The final normalization is Z-transforming the genes of the matrix. Randomized PCA is performed on the Z transformed expression matrix using Facebook's implementation through the python package fbpca retaining the top 50 components.</div>
</li> <li> <para-num num="[0957]"> </para-num> <div class="description-line" id="p-0928" num="0957">A combination of the elbow method looking at the eigenvalue gap of each component, GO enrichment of each component using jackstraw (cite Storey paper), and a PC robustness analysis (in which increasing amounts of random noise is added to the data and the stability of each principal component with respect to the original components is evaluated) was used to determine the number of principal components (which in general was approximately 10)</div>
</li> <li> <para-num num="[0958]"> </para-num> <div class="description-line" id="p-0929" num="0958">A two dimensional visualization of the relationship between cells was performed using TSNE. Clustering was performed using Infomap with k refined so that slightly more clusters are created than one would expect. Clusters are subsequently merged in an iterative fashion that have fewer than 100 differentially expressed genes between them.</div>
</li> <li> <para-num num="[0959]"> </para-num> <div class="description-line" id="p-0930" num="0959">Differential expression is evaluated using a Welch's t-test on the Z-transformed values between each cluster and the rest of the cells.</div>
</li> <li> <para-num num="[0960]"> </para-num> <div class="description-line" id="p-0931" num="0960">A Benjamini-Hochberg procedure was used to control for multiple hypothesis testing.</div>
</li> <li> <para-num num="[0961]"> </para-num> <div class="description-line" id="p-0932" num="0961">The clusters are evaluated for GO enrichment using FDR corrected p-values (see Interpretation of Results).</div>
</li> <li> <para-num num="[0962]"> </para-num> <div class="description-line" id="p-0933" num="0962">Relation of perturbed cells to unperturbed states. In order to define the relationship between the cell states in the unperturbed cells and the perturbed cells, applicants projected the perturbed cells onto the same significant principal component vectors derived from the unperturbed cells. The projection onto these components was used as a covariate by itself, especially with the K562 cells, where the major axes of variation such as cell cycle describe somewhat continuous processes. Alternatively, for the PBMDCs, discrete cell types are readily discernable. As such, a random forest classifier was trained using class labels obtained by the merged Infomap clusters with features consisting of PC scores. from sklearn.ensemble import RandomForestClassifier clf=RandomForestClassifier(n_estimators=100,n_job s=−1,oob_score=True,class_weight=‘balanced’)</div>
</li> <li> <para-num num="[0963]"> </para-num> <div class="description-line" id="p-0934" num="0963">The out of bag probability estimates were used to generate ROC curves to determine the sensitivity and specificity of classification per cluster.</div>
</li> <li> <para-num num="[0964]"> </para-num> <div class="description-line" id="p-0935" num="0964">Finally, the random forest was applied to the projected PC scores of the perturbed cells to obtain class membership estimates.</div>
</li> <li> <para-num num="[0965]"> </para-num> <div class="description-line" id="p-0936" num="0965">Testing for significant association of guides with cell states. Applicants used several tests to assess guide-state associations and our reported results are coherent across the tests.</div>
</li> <li> <para-num num="[0966]"> </para-num> <div class="description-line" id="p-0937" num="0966">In a continuous test Applicants used the Wilcoxon-ranksum test to examine if the cells with a specific guide have significantly higher or lower cell-state scores compared to the cells with non-targeting or intergenic guides. The test is performed separately for each batch (to avoid batch-effects), and then the p-values are merged using Fisher's combined probability test.</div>
</li> <li> <para-num num="[0967]"> </para-num> <div class="description-line" id="p-0938" num="0967">In a discrete test, each cell is assigned with a single state (the one it scored highest in). Cells with low cell-state scores (&lt;0.5) are classified as unclustered. For each guide-state pair &lt;g,s&gt;Applicants examine if the number of cells with guide g in state s is significantly larger or smaller than expected (using a binomial test). The expected is based on the probability of the cell-state in the cells with intergenic/non-targeting guides.</div>
</li> <li> <para-num num="[0968]"> </para-num> <div class="description-line" id="p-0939" num="0968">A guide-cell association is defined as significant if its p-value &lt;0.05 after correcting for multiple hypotheses testing using the Benjamini-Hochberg procedure.</div>
</li> <li> <para-num num="[0969]"> </para-num> <div class="description-line" id="p-0940" num="0969">Tests of sgRNA effect on Covariate. In order to evaluate the effect of a sgRNA on a covariate such as genes detected, transcripts detected, or cell state, an Welch's t-test is performed between the covariate and the set of control sgRNAs.</div>
</li> <li> <para-num num="[0970]"> </para-num> <div class="description-line" id="p-0941" num="0970">Expectation Maximization. In order to account for the contribution of unperturbed cells in the population containing a particular sgRNA, an expectation maximization approach was constructed in which the presence of sgRNA in a given cell was converted into a probability measure. The approach is as follows:</div>
</li> <li> <para-num num="[0971]"> </para-num> <div class="description-line" id="p-0942" num="0971">First, fit β using the multivariate regression on</div>
</li> <li> <para-num num="[0972]"> </para-num> <div class="description-line" id="p-0943" num="0972">Y=Xβ</div>
</li> <li> <para-num num="[0973]"> </para-num> <div class="description-line" id="p-0944" num="0973">Then evaluate the fit with guide covariate set to 0 X<sub>x </sub> </div>
</li> <li> <div class="description-line" id="p-0945" num="0000">
<maths id="MATH-US-00007" num="00007">
<math overflow="scroll">
<mrow>
<mrow>
<mrow>
<mi>P</mi>
<mo></mo>
<mrow>
<mo>(</mo>
<mrow>
<msub>
<mi>X</mi>
<mi>j</mi>
</msub>
<mo>=</mo>
<mn>1</mn>
</mrow>
<mo>)</mo>
</mrow>
</mrow>
<mo>=</mo>
<mrow>
<mi>logistic</mi>
<mo></mo>
<mrow>
<mo>(</mo>
<mrow>
<msup>
<mrow>
<msub>
<mi>Σ</mi>
<mi>i</mi>
</msub>
<mo></mo>
<mstyle>
<mspace height="0.8ex" width="0.8em"> </mspace>
</mstyle>
<mo>[</mo>
<mrow>
<msub>
<mi>Y</mi>
<mi>ij</mi>
</msub>
<mo>-</mo>
<mrow>
<msub>
<mi>X</mi>
<mi>o</mi>
</msub>
<mo></mo>
<mi>β</mi>
</mrow>
</mrow>
<mo>]</mo>
</mrow>
<mn>2</mn>
</msup>
<mo>-</mo>
<msup>
<mrow>
<mo>[</mo>
<mrow>
<msub>
<mi>Y</mi>
<mi>ij</mi>
</msub>
<mo>-</mo>
<mover>
<msub>
<mi>Y</mi>
<mi>ij</mi>
</msub>
<mo>~</mo>
</mover>
</mrow>
<mo>]</mo>
</mrow>
<mn>2</mn>
</msup>
</mrow>
<mo>)</mo>
</mrow>
</mrow>
</mrow>
<mo>;</mo>
<mstyle>
<mspace height="0.8ex" width="0.8em"> </mspace>
</mstyle>
<mo></mo>
<mi>where</mi>
</mrow>
</math>
</maths>
<maths id="MATH-US-00007-2" num="00007.2">
<math overflow="scroll">
<mrow>
<mrow>
<mi>logistic</mi>
<mo></mo>
<mrow>
<mo>(</mo>
<mi>x</mi>
<mo>)</mo>
</mrow>
</mrow>
<mo>=</mo>
<mfrac>
<mn>1</mn>
<mrow>
<mn>1</mn>
<mo>+</mo>
<msup>
<mi>e</mi>
<mrow>
<mo>-</mo>
<mi>x</mi>
</mrow>
</msup>
</mrow>
</mfrac>
</mrow>
</math>
</maths>
</div>
</li> <li> <div class="description-line" id="p-0946" num="0000">Finally, the new covariate matrix X<sub>M </sub>is used to recompute β<sub>M </sub> </div>
</li> <li> <para-num num="[0974]"> </para-num> <div class="description-line" id="p-0947" num="0974">Significance Testing for Coefficients of Linear Model. A permutation strategy was devised to get an empirical null distribution of the coefficients associated with our sgRNA effects. Specifically, the guide assignments to cells were randomized (such that co-occurrence between guides was preserved) and the linear model was recomputed with all other covariates being held constant ten times. Applicants noticed that three significant as-yet latent factors impacted the empirical null distribution of coefficients: 1) the mean expression level of a gene 2) the variance in expression of a gene 3) the number of cells a particular sgRNA was present in.</div>
</li> <li> <para-num num="[0975]"> </para-num> <div class="description-line" id="p-0948" num="0975">In order to control for these factors in assessing significance, each empirical null coefficient's value was assigned a point in 4-space: [Gene mean, Gene variance, number of cells, value] A multivariate density estimation was subsequently performed using a binning approach (np.histogramdd in Python). True coefficients were evaluated for significance relative to a matched set of bins (to create an empirical conditional cumulative probability distribution) conditioned on the first three factors.</div>
</li> <li> <para-num num="[0976]"> </para-num> <div class="description-line" id="p-0949" num="0976">A Benjamini-Hochberg procedure was used to control for multiple hypothesis testing.</div>
</li> <li> <para-num num="[0977]"> </para-num> <div class="description-line" id="p-0950" num="0977">Residuals Analysis. In order to determine the marginal effect of each covariate in explaining the observed gene expression variation a cross-validated estimation (trained on 80% of the data and tested against 20%) of the model R<sup>2 </sup>was obtained for the addition of each of the covariates.</div>
</li> <li> <para-num num="[0978]"> </para-num> <div class="description-line" id="p-0951" num="0978">In order to determine the extent to which our covariates explained the major axes of variation in our data, applicants decomposed the residuals using the same randomized PCA approach described in the Definition of Cell States section. Two major metrics were evaluated: 1) the eigenvalue distribution 2) the extent to which the top loadings were enriched for biological terms using the aforementioned approach.</div>
</li> <li> <para-num num="[0979]"> </para-num> <div class="description-line" id="p-0952" num="0979">Interpretation of Results. GO analysis was performed using the goatools python package with an FDR threshold of 0.1.</div>
</li> <li> <para-num num="[0980]"> </para-num> <div class="description-line" id="p-0953" num="0980">Power Analysis/Design of Experiments. Power analysis was performed to determine as a function of observed effect size and baseline expression of a gene, how many cells are required to observe a signal.</div>
</li> <li> <para-num num="[0981]"> </para-num> <div class="description-line" id="p-0954" num="0981">Fitness effects of sgRNAs. Applicants used several approaches to assess the fitness effects of sgRNAs and all yielded qualitatively comparable results.</div>
</li> <li> <para-num num="[0982]"> </para-num> <div class="description-line" id="p-0955" num="0982">In the most straightfoward approach, Applicants obtained estimates of the initial abundances of each sgRNA in the pool. The initial abundance of each plasmid in the pool was quantified using NGS of the RNA barcode. The RNA barcode / sgRNA dictionary was used to convert the readout into a relative abundance estimate of sgRNA in the initial pool. Then, Applicants calculated the fold change of the observed abundance of cells containing a particular sgRNA compared to its respective abundance in the original pool.</div>
</li> <li> <para-num num="[0983]"> </para-num> <div class="description-line" id="p-0956" num="0983">In an alternative approach, Applicants developed two complementary Bayesian probabilistic models that compute the expected probability of each guide in the resulting cell population. The first model assumes that none of the guides affect cell fitness and is suitable in situations where the perturbations are unlikely to effect cell survival or proliferations, as in BMDC TFs. The expected guide frequency is modeled as a function of the MOI, the guide frequency in the initial library, and the guide detection rate, such that multiple infections are considered to be independent Poisson processes. Applicants applied this model and examined if the observed guide frequencies deviate from the expected ones. Indeed, in K562 cells the observed guide distribution deviates significantly from the expected distribution, indicating that certain perturbations affect cell fitness. Applicants then extended the probabilistic model to account for these effects by introducing fitness parameters, which are set by solving a maximum likelihood optimization problem. In this setting Applicants model the fitness effects of multiple perturbations as additive, and identify perturbations that show different co-occurrence frequencies as indicators of genetic interactions.</div>
</li> <li> <para-num num="[0984]"> </para-num> <div class="description-line" id="p-0957" num="0984">In both Bayesian models the infection, selection, and detection processes are modeled such that: (1) the guide detection rate, denoted as d, is the same for all guides, (2) there are no falsely detected guides, (3) the infection with different guides is independent and, (4) due to BFP selection, denoted as B, all cells have at least one guide by the end of the screen. Let M, f<sub>g</sub>, and d<sub>g </sub>denote the MOI, the frequency of guide g in the initial pool of the screen, and the event of detecting guide g, respectively. Applicants model the infection as a Poisson process, such that i<sub>g </sub>denotes the number of times the cell was infected with guide g, i<sub>g</sub>{tilde over ( )}Poisson(f<sub>g</sub>·M). Let I<sub>c,g</sub>, N<sub>0</sub>, and N<sub>g </sub>be the indicator that cell c was detected with guide g in the screen, the number of cells in which no guide was detected, and the number of cells in which guide g was detected, respectively.</div>
</li> <li> <para-num num="[0985]"> </para-num> <div class="description-line" id="p-0958" num="0985">To find the most likely parameters and compute the expected guide probabilities without fitness effects Applicants solve the following maximum likelihood optimization problem.</div>
</li> <li> <div class="description-line" id="p-0959" num="0000">
<maths id="MATH-US-00008" num="00008">
<math overflow="scroll">
<mrow>
<mstyle>
<mspace height="4.4ex" width="4.4em"> </mspace>
</mstyle>
<mo></mo>
<mrow>
<mrow>
<mi>L</mi>
<mo></mo>
<mrow>
<mo>(</mo>
<mrow>
<mi>M</mi>
<mo>,</mo>
<msub>
<mi>p</mi>
<mi>d</mi>
</msub>
</mrow>
<mo>)</mo>
</mrow>
</mrow>
<mo>=</mo>
<mrow>
<munder>
<mo>∏</mo>
<mi>c</mi>
</munder>
<mo></mo>
<mrow>
<mi>P</mi>
<mo></mo>
<mrow>
<mo>(</mo>
<mrow>
<msub>
<mi>d</mi>
<mn>1</mn>
</msub>
<mo>,</mo>
<mi>…</mi>
<mo></mo>
<mstyle>
<mspace height="0.8ex" width="0.8em"> </mspace>
</mstyle>
<mo>,</mo>
<msub>
<mi>d</mi>
<mi>k</mi>
</msub>
<mo>,</mo>
<mi>B</mi>
</mrow>
<mo>)</mo>
</mrow>
</mrow>
</mrow>
</mrow>
</mrow>
</math>
</maths>
<maths id="MATH-US-00008-2" num="00008.2">
<math overflow="scroll">
<mtable>
<mtr>
<mtd>
<mrow>
<mstyle>
<mspace height="4.4ex" width="4.4em"> </mspace>
</mstyle>
<mo></mo>
<mrow>
<mrow>
<mi>P</mi>
<mo></mo>
<mrow>
<mo>(</mo>
<mrow>
<msub>
<mi>d</mi>
<mn>1</mn>
</msub>
<mo>,</mo>
<mi>…</mi>
<mo></mo>
<mstyle>
<mspace height="0.8ex" width="0.8em"> </mspace>
</mstyle>
<mo>,</mo>
<msub>
<mi>d</mi>
<mi>k</mi>
</msub>
<mo>,</mo>
<mi>B</mi>
</mrow>
<mo>)</mo>
</mrow>
</mrow>
<mo>=</mo>
<mi> </mi>
<mo></mo>
<mrow>
<mrow>
<mi>P</mi>
<mo></mo>
<mrow>
<mo>(</mo>
<mrow>
<mrow>
<mi>B</mi>
<mo>|</mo>
<msub>
<mi>d</mi>
<mn>1</mn>
</msub>
</mrow>
<mo>,</mo>
<mi>…</mi>
<mo></mo>
<mstyle>
<mspace height="0.8ex" width="0.8em"> </mspace>
</mstyle>
<mo>,</mo>
<msub>
<mi>d</mi>
<mi>k</mi>
</msub>
</mrow>
<mo>)</mo>
</mrow>
</mrow>
<mo></mo>
<mrow>
<mi>P</mi>
<mo></mo>
<mrow>
<mo>(</mo>
<mrow>
<msub>
<mi>d</mi>
<mn>1</mn>
</msub>
<mo>,</mo>
<mi>…</mi>
<mo></mo>
<mstyle>
<mspace height="0.8ex" width="0.8em"> </mspace>
</mstyle>
<mo>,</mo>
<msub>
<mi>d</mi>
<mi>k</mi>
</msub>
</mrow>
<mo>)</mo>
</mrow>
</mrow>
</mrow>
</mrow>
</mrow>
</mtd>
</mtr>
<mtr>
<mtd>
<mrow>
<mo>=</mo>
<mi> </mi>
<mo></mo>
<mrow>
<mo>{</mo>
<mtable>
<mtr>
<mtd>
<mrow>
<mi>P</mi>
<mo></mo>
<mrow>
<mo>(</mo>
<mrow>
<msub>
<mi>d</mi>
<mn>1</mn>
</msub>
<mo>,</mo>
<mi>…</mi>
<mo></mo>
<mstyle>
<mspace height="0.8ex" width="0.8em"> </mspace>
</mstyle>
<mo>,</mo>
<msub>
<mi>d</mi>
<mi>k</mi>
</msub>
</mrow>
<mo>)</mo>
</mrow>
</mrow>
</mtd>
<mtd>
<mrow>
<mrow>
<mi>if</mi>
<mo></mo>
<mstyle>
<mspace height="0.8ex" width="0.8em"> </mspace>
</mstyle>
<mo></mo>
<mrow>
<munder>
<mo>∑</mo>
<mi>g</mi>
</munder>
<mo></mo>
<msub>
<mi>d</mi>
<mi>g</mi>
</msub>
</mrow>
</mrow>
<mo>&gt;</mo>
<mn>0</mn>
</mrow>
</mtd>
</mtr>
<mtr>
<mtd>
<mrow>
<mrow>
<mi>P</mi>
<mo></mo>
<mrow>
<mo>(</mo>
<mrow>
<mrow>
<mi>B</mi>
<mo>|</mo>
<mrow>
<msub>
<mo>∀</mo>
<mi>g</mi>
</msub>
<mo></mo>
<msub>
<mi>d</mi>
<mi>g</mi>
</msub>
</mrow>
</mrow>
<mo>=</mo>
<mn>0</mn>
</mrow>
<mo>)</mo>
</mrow>
</mrow>
<mo></mo>
<mrow>
<mi>P</mi>
<mo></mo>
<mrow>
<mo>(</mo>
<mrow>
<mrow>
<msub>
<mo>∀</mo>
<mi>g</mi>
</msub>
<mo></mo>
<msub>
<mi>d</mi>
<mi>g</mi>
</msub>
</mrow>
<mo>=</mo>
<mn>0</mn>
</mrow>
<mo>)</mo>
</mrow>
</mrow>
</mrow>
</mtd>
<mtd>
<mi>otherwise</mi>
</mtd>
</mtr>
</mtable>
</mrow>
</mrow>
</mtd>
</mtr>
</mtable>
</math>
</maths>
<maths id="MATH-US-00008-3" num="00008.3">
<math overflow="scroll">
<mrow>
<mrow>
<mi>L</mi>
<mo></mo>
<mrow>
<mo>(</mo>
<mrow>
<mi>M</mi>
<mo>,</mo>
<msub>
<mi>p</mi>
<mi>d</mi>
</msub>
</mrow>
<mo>)</mo>
</mrow>
</mrow>
<mo>=</mo>
<mrow>
<mrow>
<msub>
<mi>Π</mi>
<mi>c</mi>
</msub>
<mo></mo>
<mrow>
<mo>[</mo>
<mrow>
<msup>
<mrow>
<mi>P</mi>
<mo></mo>
<mrow>
<mo>(</mo>
<mrow>
<mrow>
<mi>B</mi>
<mo>|</mo>
<mrow>
<msub>
<mo>∀</mo>
<mi>g</mi>
</msub>
<mo></mo>
<msub>
<mi>d</mi>
<mi>g</mi>
</msub>
</mrow>
</mrow>
<mo>=</mo>
<mn>0</mn>
</mrow>
<mo>)</mo>
</mrow>
</mrow>
<msub>
<mi>I</mi>
<mrow>
<mrow>
<msub>
<mo>∀</mo>
<mi>g</mi>
</msub>
<mo></mo>
<msub>
<mi>I</mi>
<mrow>
<mi>c</mi>
<mo>,</mo>
<mi>g</mi>
</mrow>
</msub>
</mrow>
<mo>=</mo>
<mn>0</mn>
</mrow>
</msub>
</msup>
<mo></mo>
<msub>
<mi>Π</mi>
<mi>g</mi>
</msub>
<mo></mo>
<mrow>
<msup>
<mrow>
<mi>P</mi>
<mo></mo>
<mrow>
<mo>(</mo>
<mrow>
<msub>
<mi>d</mi>
<mi>g</mi>
</msub>
<mo>=</mo>
<mn>1</mn>
</mrow>
<mo>)</mo>
</mrow>
</mrow>
<msub>
<mi>I</mi>
<mrow>
<mi>c</mi>
<mo>,</mo>
<mi>g</mi>
</mrow>
</msub>
</msup>
<mo>·</mo>
<msup>
<mrow>
<mi>P</mi>
<mo></mo>
<mrow>
<mo>(</mo>
<mrow>
<msub>
<mi>d</mi>
<mi>g</mi>
</msub>
<mo>=</mo>
<mn>0</mn>
</mrow>
<mo>)</mo>
</mrow>
</mrow>
<mrow>
<mn>1</mn>
<mo>-</mo>
<msub>
<mi>I</mi>
<mrow>
<mi>c</mi>
<mo>,</mo>
<mi>g</mi>
</mrow>
</msub>
</mrow>
</msup>
</mrow>
</mrow>
<mo>]</mo>
</mrow>
</mrow>
<mo>=</mo>
<mrow>
<mrow>
<msup>
<mrow>
<mi>P</mi>
<mo></mo>
<mrow>
<mo>(</mo>
<mrow>
<mrow>
<mi>B</mi>
<mo>|</mo>
<mrow>
<msub>
<mo>∀</mo>
<mi>g</mi>
</msub>
<mo></mo>
<msub>
<mi>d</mi>
<mi>g</mi>
</msub>
</mrow>
</mrow>
<mo>=</mo>
<mn>0</mn>
</mrow>
<mo>)</mo>
</mrow>
</mrow>
<msub>
<mi>N</mi>
<mn>0</mn>
</msub>
</msup>
<mo></mo>
<mrow>
<mo>[</mo>
<mrow>
<msub>
<mi>Π</mi>
<mi>g</mi>
</msub>
<mo></mo>
<mrow>
<msup>
<mrow>
<mi>P</mi>
<mo></mo>
<mrow>
<mo>(</mo>
<mrow>
<msub>
<mi>d</mi>
<mi>g</mi>
</msub>
<mo>=</mo>
<mn>1</mn>
</mrow>
<mo>)</mo>
</mrow>
</mrow>
<msub>
<mi>N</mi>
<mi>g</mi>
</msub>
</msup>
<mo>·</mo>
<msup>
<mrow>
<mi>P</mi>
<mo></mo>
<mrow>
<mo>(</mo>
<mrow>
<msub>
<mi>d</mi>
<mi>g</mi>
</msub>
<mo>=</mo>
<mn>0</mn>
</mrow>
<mo>)</mo>
</mrow>
</mrow>
<mrow>
<mi>N</mi>
<mo>-</mo>
<msub>
<mi>N</mi>
<mi>g</mi>
</msub>
</mrow>
</msup>
</mrow>
</mrow>
<mo>]</mo>
</mrow>
</mrow>
<mo>=</mo>
<mrow>
<msup>
<mrow>
<mo>(</mo>
<mrow>
<mn>1</mn>
<mo>-</mo>
<msup>
<mi>e</mi>
<mrow>
<mo>-</mo>
<mrow>
<mi>M</mi>
<mo></mo>
<mrow>
<mo>(</mo>
<mrow>
<mn>1</mn>
<mo>-</mo>
<msub>
<mi>p</mi>
<mi>d</mi>
</msub>
</mrow>
<mo>)</mo>
</mrow>
</mrow>
</mrow>
</msup>
</mrow>
<mo>)</mo>
</mrow>
<msub>
<mi>N</mi>
<mn>0</mn>
</msub>
</msup>
<mo></mo>
<mrow>
<mo>[</mo>
<mrow>
<msub>
<mi>Π</mi>
<mi>g</mi>
</msub>
<mo></mo>
<mrow>
<msup>
<mrow>
<mi>P</mi>
<mo></mo>
<mrow>
<mo>(</mo>
<mrow>
<msub>
<mi>d</mi>
<mi>g</mi>
</msub>
<mo>=</mo>
<mn>1</mn>
</mrow>
<mo>)</mo>
</mrow>
</mrow>
<msub>
<mi>N</mi>
<mi>g</mi>
</msub>
</msup>
<mo>·</mo>
<msup>
<mrow>
<mi>P</mi>
<mo></mo>
<mrow>
<mo>(</mo>
<mrow>
<msub>
<mi>d</mi>
<mi>g</mi>
</msub>
<mo>=</mo>
<mn>0</mn>
</mrow>
<mo>)</mo>
</mrow>
</mrow>
<mrow>
<mi>N</mi>
<mo>-</mo>
<msub>
<mi>N</mi>
<mi>g</mi>
</msub>
</mrow>
</msup>
</mrow>
</mrow>
<mo>]</mo>
</mrow>
</mrow>
</mrow>
</mrow>
</mrow>
</math>
</maths>
</div>
</li> <li> <para-num num="[0986]"> </para-num> <div class="description-line" id="p-0960" num="0986">The log likelihood is</div>
</li> <li> <div class="description-line" id="p-0961" num="0000">
<maths id="MATH-US-00009" num="00009">
<math overflow="scroll">
<mrow>
<mrow>
<mi>log</mi>
<mo></mo>
<mrow>
<mo>[</mo>
<mrow>
<mi>L</mi>
<mo></mo>
<mrow>
<mo>(</mo>
<mrow>
<mi>M</mi>
<mo>,</mo>
<msub>
<mi>p</mi>
<mi>d</mi>
</msub>
</mrow>
<mo>)</mo>
</mrow>
</mrow>
<mo>]</mo>
</mrow>
</mrow>
<mo>=</mo>
<mrow>
<mrow>
<mrow>
<msub>
<mi>N</mi>
<mn>0</mn>
</msub>
<mo></mo>
<mrow>
<mi>log</mi>
<mo></mo>
<mrow>
<mo>(</mo>
<mrow>
<mn>1</mn>
<mo>-</mo>
<msup>
<mi>e</mi>
<mrow>
<mo>-</mo>
<mrow>
<mi>M</mi>
<mo></mo>
<mrow>
<mo>(</mo>
<mrow>
<mn>1</mn>
<mo>-</mo>
<msub>
<mi>p</mi>
<mi>d</mi>
</msub>
</mrow>
<mo>)</mo>
</mrow>
</mrow>
</mrow>
</msup>
</mrow>
<mo>)</mo>
</mrow>
</mrow>
</mrow>
<mo>+</mo>
<mrow>
<munder>
<mo>∑</mo>
<mi>g</mi>
</munder>
<mo></mo>
<mrow>
<mo>{</mo>
<mrow>
<mrow>
<msub>
<mi>N</mi>
<mi>g</mi>
</msub>
<mo></mo>
<mrow>
<mi>log</mi>
<mo></mo>
<mrow>
<mo>[</mo>
<mrow>
<mi>P</mi>
<mo></mo>
<mrow>
<mo>(</mo>
<mrow>
<msub>
<mi>d</mi>
<mi>g</mi>
</msub>
<mo>=</mo>
<mn>1</mn>
</mrow>
<mo>)</mo>
</mrow>
</mrow>
<mo>]</mo>
</mrow>
</mrow>
</mrow>
<mo>+</mo>
<mrow>
<mrow>
<mo>(</mo>
<mrow>
<mi>N</mi>
<mo>-</mo>
<msub>
<mi>N</mi>
<mi>g</mi>
</msub>
</mrow>
<mo>)</mo>
</mrow>
<mo></mo>
<mrow>
<mi>log</mi>
<mo></mo>
<mrow>
<mo>[</mo>
<mrow>
<mo>(</mo>
<mrow>
<msub>
<mi>d</mi>
<mi>g</mi>
</msub>
<mo>=</mo>
<mn>0</mn>
</mrow>
<mo>)</mo>
</mrow>
<mo>]</mo>
</mrow>
</mrow>
</mrow>
</mrow>
<mo>}</mo>
</mrow>
</mrow>
</mrow>
<mo>=</mo>
<mrow>
<mrow>
<msub>
<mi>N</mi>
<mn>0</mn>
</msub>
<mo></mo>
<mrow>
<mi>log</mi>
<mo></mo>
<mrow>
<mo>(</mo>
<mrow>
<mn>1</mn>
<mo>-</mo>
<msup>
<mi>e</mi>
<mrow>
<mo>-</mo>
<mrow>
<mi>M</mi>
<mo></mo>
<mrow>
<mo>(</mo>
<mrow>
<mn>1</mn>
<mo>-</mo>
<msub>
<mi>p</mi>
<mi>d</mi>
</msub>
</mrow>
<mo>)</mo>
</mrow>
</mrow>
</mrow>
</msup>
</mrow>
<mo>)</mo>
</mrow>
</mrow>
</mrow>
<mo>+</mo>
<mrow>
<munder>
<mo>∑</mo>
<mi>g</mi>
</munder>
<mo></mo>
<mrow>
<mo>{</mo>
<mrow>
<mrow>
<msub>
<mi>N</mi>
<mi>g</mi>
</msub>
<mo></mo>
<mrow>
<mi>log</mi>
<mo></mo>
<mrow>
<mo>(</mo>
<mrow>
<mn>1</mn>
<mo>-</mo>
<msup>
<mi>e</mi>
<mrow>
<mrow>
<mo>-</mo>
<msub>
<mi>f</mi>
<mi>g</mi>
</msub>
</mrow>
<mo></mo>
<msub>
<mi>p</mi>
<mi>d</mi>
</msub>
<mo></mo>
<mi>M</mi>
</mrow>
</msup>
</mrow>
<mo>)</mo>
</mrow>
</mrow>
</mrow>
<mo>-</mo>
<mrow>
<mrow>
<mo>(</mo>
<mrow>
<mi>N</mi>
<mo>-</mo>
<msub>
<mi>N</mi>
<mi>g</mi>
</msub>
</mrow>
<mo>)</mo>
</mrow>
<mo></mo>
<msub>
<mi>f</mi>
<mi>g</mi>
</msub>
<mo></mo>
<msub>
<mi>p</mi>
<mi>d</mi>
</msub>
<mo></mo>
<mi>M</mi>
</mrow>
</mrow>
<mo>}</mo>
</mrow>
</mrow>
</mrow>
</mrow>
</mrow>
</math>
</maths>
</div>
</li> <li> <para-num num="[0987]"> </para-num> <div class="description-line" id="p-0962" num="0987">Below Applicants provide the equations used to compute the maximum likelihood function above.</div>
</li> <li> <div class="description-line" id="p-0963" num="0000"> <br/> <i>P</i>(<i>d</i> <sub>g</sub>=0)=<i>e</i> <sup>−fgMpd </sup>  (1.1)
</div>
</li> <li> <para-num num="[0988]"> </para-num> <div class="description-line" id="p-0964" num="0988">Based on the moment-generating function</div>
</li> <li> <div class="description-line" id="p-0965" num="0000">
<maths id="MATH-US-00010" num="00010">
<math overflow="scroll">
<mrow>
<mstyle>
<mspace height="4.4ex" width="4.4em"> </mspace>
</mstyle>
<mo></mo>
<mrow>
<mrow>
<mrow>
<mi>E</mi>
<mo></mo>
<mrow>
<mo>[</mo>
<msup>
<mi>e</mi>
<mi>tX</mi>
</msup>
<mo>]</mo>
</mrow>
</mrow>
<mo>=</mo>
<msup>
<mi>e</mi>
<mrow>
<mi>λ</mi>
<mo></mo>
<mrow>
<mo>(</mo>
<mrow>
<msup>
<mi>e</mi>
<mi>t</mi>
</msup>
<mo>-</mo>
<mn>1</mn>
</mrow>
<mo>)</mo>
</mrow>
</mrow>
</msup>
</mrow>
<mo>,</mo>
<mrow>
<mrow>
<mi>where</mi>
<mo></mo>
<mstyle>
<mspace height="0.8ex" width="0.8em"> </mspace>
</mstyle>
<mo></mo>
<mrow>
<mi>X</mi>
<mo>~</mo>
<mrow>
<mi>Poisson</mi>
<mo></mo>
<mrow>
<mo>(</mo>
<mi>λ</mi>
<mo>)</mo>
</mrow>
</mrow>
</mrow>
<mo></mo>
<mstyle>
<mspace height="0.8ex" width="0.8em"> </mspace>
</mstyle>
<mo></mo>
<mi>and</mi>
<mo></mo>
<mstyle>
<mspace height="0.8ex" width="0.8em"> </mspace>
</mstyle>
<mo></mo>
<mi>t</mi>
</mrow>
<mo>∈</mo>
<mi>ℝ</mi>
</mrow>
</mrow>
</mrow>
</math>
</maths>
<maths id="MATH-US-00010-2" num="00010.2">
<math overflow="scroll">
<mrow>
<mstyle>
<mspace height="4.4ex" width="4.4em"> </mspace>
</mstyle>
<mo></mo>
<mrow>
<mrow>
<mi>t</mi>
<mo>=</mo>
<mrow>
<mi>ln</mi>
<mo></mo>
<mstyle>
<mspace height="0.6ex" width="0.6em"> </mspace>
</mstyle>
<mo></mo>
<mi>q</mi>
</mrow>
</mrow>
<mo>,</mo>
<mrow>
<mi>λ</mi>
<mo>=</mo>
<mrow>
<msub>
<mi>f</mi>
<mi>g</mi>
</msub>
<mo>·</mo>
<mi>M</mi>
</mrow>
</mrow>
<mo>,</mo>
<mrow>
<mrow>
<mi>E</mi>
<mo></mo>
<mrow>
<mo>[</mo>
<msup>
<mi>q</mi>
<msub>
<mi>i</mi>
<mi>g</mi>
</msub>
</msup>
<mo>]</mo>
</mrow>
</mrow>
<mo>=</mo>
<mrow>
<mrow>
<mi>E</mi>
<mo></mo>
<mrow>
<mo>[</mo>
<msup>
<mi>e</mi>
<mrow>
<mrow>
<msub>
<mi>i</mi>
<mi>g</mi>
</msub>
<mo>·</mo>
<mi>l</mi>
</mrow>
<mo></mo>
<mstyle>
<mspace height="0.3ex" width="0.3em"> </mspace>
</mstyle>
<mo></mo>
<mi>n</mi>
<mo></mo>
<mstyle>
<mspace height="0.6ex" width="0.6em"> </mspace>
</mstyle>
<mo></mo>
<mi>q</mi>
</mrow>
</msup>
<mo>]</mo>
</mrow>
</mrow>
<mo>=</mo>
<mrow>
<msup>
<mi>e</mi>
<mrow>
<msub>
<mi>f</mi>
<mi>g</mi>
</msub>
<mo></mo>
<mrow>
<mi>M</mi>
<mo></mo>
<mrow>
<mo>(</mo>
<mrow>
<mi>q</mi>
<mo>-</mo>
<mn>1</mn>
</mrow>
<mo>)</mo>
</mrow>
</mrow>
</mrow>
</msup>
<mo>=</mo>
<msup>
<mi>e</mi>
<mrow>
<mrow>
<mo>-</mo>
<msub>
<mi>f</mi>
<mi>g</mi>
</msub>
</mrow>
<mo></mo>
<msub>
<mi>Mp</mi>
<mi>d</mi>
</msub>
</mrow>
</msup>
</mrow>
</mrow>
</mrow>
</mrow>
</mrow>
</math>
</maths>
<maths id="MATH-US-00010-3" num="00010.3">
<math overflow="scroll">
<mrow>
<mstyle>
<mspace height="4.4ex" width="4.4em"> </mspace>
</mstyle>
<mo></mo>
<mrow>
<mrow>
<mrow>
<mo>(</mo>
<mn>1.2</mn>
<mo>)</mo>
</mrow>
<mo></mo>
<mstyle>
<mspace height="0.6ex" width="0.6em"> </mspace>
</mstyle>
<mo></mo>
<mrow>
<mi>P</mi>
<mo></mo>
<mrow>
<mo>(</mo>
<mi>B</mi>
<mo>)</mo>
</mrow>
</mrow>
</mrow>
<mo>=</mo>
<mrow>
<mn>1</mn>
<mo>-</mo>
<msup>
<mi>e</mi>
<mrow>
<mo>-</mo>
<mi>M</mi>
</mrow>
</msup>
</mrow>
</mrow>
</mrow>
</math>
</maths>
<maths id="MATH-US-00010-4" num="00010.4">
<math overflow="scroll">
<mrow>
<mstyle>
<mspace height="4.4ex" width="4.4em"> </mspace>
</mstyle>
<mo></mo>
<mrow>
<mrow>
<mi>P</mi>
<mo></mo>
<mrow>
<mo>(</mo>
<mi>B</mi>
<mo>)</mo>
</mrow>
</mrow>
<mo>=</mo>
<mrow>
<mn>1</mn>
<mo>-</mo>
<mrow>
<mi>P</mi>
<mo></mo>
<mrow>
<mo>(</mo>
<mrow>
<mo></mo>
<mi>B</mi>
</mrow>
<mo>)</mo>
</mrow>
</mrow>
</mrow>
</mrow>
</mrow>
</math>
</maths>
<maths id="MATH-US-00010-5" num="00010.5">
<math overflow="scroll">
<mrow>
<mrow>
<mi>P</mi>
<mo></mo>
<mrow>
<mo>(</mo>
<mrow>
<mo></mo>
<mi>B</mi>
</mrow>
<mo>)</mo>
</mrow>
</mrow>
<mo>=</mo>
<mrow>
<mrow>
<mi>P</mi>
<mo></mo>
<mrow>
<mo>(</mo>
<mrow>
<mrow>
<msub>
<mo>∀</mo>
<mi>g</mi>
</msub>
<mo></mo>
<msub>
<mi>I</mi>
<mi>g</mi>
</msub>
</mrow>
<mo>=</mo>
<mn>0</mn>
</mrow>
<mo>)</mo>
</mrow>
</mrow>
<mo>=</mo>
<mrow>
<mrow>
<munder>
<mo>∏</mo>
<mi>g</mi>
</munder>
<mo></mo>
<mrow>
<mi>P</mi>
<mo></mo>
<mrow>
<mo>(</mo>
<mrow>
<msub>
<mi>I</mi>
<mi>g</mi>
</msub>
<mo>=</mo>
<mn>0</mn>
</mrow>
<mo>)</mo>
</mrow>
</mrow>
</mrow>
<mo>=</mo>
<mrow>
<mrow>
<munder>
<mo>∏</mo>
<mi>g</mi>
</munder>
<mo></mo>
<msup>
<mi>e</mi>
<mrow>
<mo>-</mo>
<msub>
<mi>Mf</mi>
<mi>g</mi>
</msub>
</mrow>
</msup>
</mrow>
<mo>=</mo>
<mrow>
<msup>
<mi>e</mi>
<mrow>
<mrow>
<mo>-</mo>
<msub>
<mi>MΣ</mi>
<mi>g</mi>
</msub>
</mrow>
<mo></mo>
<msub>
<mi>f</mi>
<mi>g</mi>
</msub>
</mrow>
</msup>
<mo>=</mo>
<msup>
<mi>e</mi>
<mrow>
<mo>-</mo>
<mi>M</mi>
</mrow>
</msup>
</mrow>
</mrow>
</mrow>
</mrow>
</mrow>
</math>
</maths>
<maths id="MATH-US-00010-6" num="00010.6">
<math overflow="scroll">
<mrow>
<mstyle>
<mspace height="4.4ex" width="4.4em"> </mspace>
</mstyle>
<mo></mo>
<mrow>
<mrow>
<mrow>
<mo>(</mo>
<mn>1.3</mn>
<mo>)</mo>
</mrow>
<mo></mo>
<mstyle>
<mspace height="0.6ex" width="0.6em"> </mspace>
</mstyle>
<mo></mo>
<mrow>
<mi>P</mi>
<mo></mo>
<mrow>
<mo>(</mo>
<mrow>
<mrow>
<mi>B</mi>
<mo>|</mo>
<mrow>
<msub>
<mo>∀</mo>
<mi>g</mi>
</msub>
<mo></mo>
<msub>
<mi>d</mi>
<mi>g</mi>
</msub>
</mrow>
</mrow>
<mo>=</mo>
<mn>0</mn>
</mrow>
<mo>)</mo>
</mrow>
</mrow>
</mrow>
<mo>=</mo>
<mrow>
<mn>1</mn>
<mo>-</mo>
<msup>
<mi>e</mi>
<mrow>
<mo>-</mo>
<mrow>
<mi>M</mi>
<mo></mo>
<mrow>
<mo>(</mo>
<mrow>
<mn>1</mn>
<mo>-</mo>
<msub>
<mi>p</mi>
<mi>d</mi>
</msub>
</mrow>
<mo>)</mo>
</mrow>
</mrow>
</mrow>
</msup>
</mrow>
</mrow>
</mrow>
</math>
</maths>
<maths id="MATH-US-00010-7" num="00010.7">
<math overflow="scroll">
<mrow>
<mstyle>
<mspace height="4.4ex" width="4.4em"> </mspace>
</mstyle>
<mo></mo>
<mrow>
<mrow>
<mi>P</mi>
<mo></mo>
<mrow>
<mo>(</mo>
<mrow>
<mrow>
<mi>B</mi>
<mo>|</mo>
<mrow>
<msub>
<mo>∀</mo>
<mi>g</mi>
</msub>
<mo></mo>
<msub>
<mi>d</mi>
<mi>g</mi>
</msub>
</mrow>
</mrow>
<mo>=</mo>
<mn>0</mn>
</mrow>
<mo>)</mo>
</mrow>
</mrow>
<mo>=</mo>
<mrow>
<mn>1</mn>
<mo>-</mo>
<mrow>
<mi>P</mi>
<mo></mo>
<mrow>
<mo>(</mo>
<mrow>
<mrow>
<mrow>
<mo></mo>
<mi>B</mi>
</mrow>
<mo>|</mo>
<mrow>
<msub>
<mo>∀</mo>
<mi>g</mi>
</msub>
<mo></mo>
<msub>
<mi>d</mi>
<mi>g</mi>
</msub>
</mrow>
</mrow>
<mo>=</mo>
<mn>0</mn>
</mrow>
<mo>)</mo>
</mrow>
</mrow>
</mrow>
</mrow>
</mrow>
</math>
</maths>
<maths id="MATH-US-00010-8" num="00010.8">
<math overflow="scroll">
<mrow>
<mrow>
<mi>P</mi>
<mo></mo>
<mrow>
<mo>(</mo>
<mrow>
<mrow>
<mrow>
<mo></mo>
<mi>B</mi>
</mrow>
<mo>|</mo>
<mrow>
<msub>
<mo>∀</mo>
<mi>g</mi>
</msub>
<mo></mo>
<msub>
<mi>d</mi>
<mi>g</mi>
</msub>
</mrow>
</mrow>
<mo>=</mo>
<mn>0</mn>
</mrow>
<mo>)</mo>
</mrow>
</mrow>
<mo>=</mo>
<mrow>
<mrow>
<mi>P</mi>
<mo></mo>
<mrow>
<mo>(</mo>
<mrow>
<mrow>
<msub>
<mo>∀</mo>
<mi>g</mi>
</msub>
<mo></mo>
<msub>
<mi>I</mi>
<mi>g</mi>
</msub>
</mrow>
<mo>=</mo>
<mrow>
<mrow>
<mn>0</mn>
<mo>|</mo>
<mrow>
<msub>
<mo>∀</mo>
<mi>g</mi>
</msub>
<mo></mo>
<msub>
<mi>d</mi>
<mi>g</mi>
</msub>
</mrow>
</mrow>
<mo>=</mo>
<mn>0</mn>
</mrow>
</mrow>
<mo>)</mo>
</mrow>
</mrow>
<mo>=</mo>
<mrow>
<mrow>
<msub>
<mi>Π</mi>
<mi>g</mi>
</msub>
<mo></mo>
<mrow>
<mi>P</mi>
<mo></mo>
<mrow>
<mo>(</mo>
<mrow>
<msub>
<mi>I</mi>
<mi>g</mi>
</msub>
<mo>=</mo>
<mrow>
<mrow>
<mn>0</mn>
<mo>|</mo>
<msub>
<mi>d</mi>
<mi>g</mi>
</msub>
</mrow>
<mo>=</mo>
<mn>0</mn>
</mrow>
</mrow>
<mo>)</mo>
</mrow>
</mrow>
</mrow>
<mo>=</mo>
<mrow>
<mrow>
<msub>
<mi>Π</mi>
<mi>g</mi>
</msub>
<mo></mo>
<mfrac>
<mrow>
<mi>P</mi>
<mo></mo>
<mrow>
<mo>(</mo>
<mrow>
<msub>
<mi>d</mi>
<mi>g</mi>
</msub>
<mo>=</mo>
<mrow>
<mrow>
<mn>0</mn>
<mo>|</mo>
<msub>
<mi>I</mi>
<mi>g</mi>
</msub>
</mrow>
<mo>=</mo>
<mn>0</mn>
</mrow>
</mrow>
<mo>)</mo>
</mrow>
<mo></mo>
<mrow>
<mi>P</mi>
<mo></mo>
<mrow>
<mo>(</mo>
<mrow>
<msub>
<mi>I</mi>
<mi>g</mi>
</msub>
<mo>=</mo>
<mn>0</mn>
</mrow>
<mo>)</mo>
</mrow>
</mrow>
</mrow>
<mrow>
<mi>P</mi>
<mo></mo>
<mrow>
<mo>(</mo>
<mrow>
<msub>
<mi>d</mi>
<mi>g</mi>
</msub>
<mo>=</mo>
<mn>0</mn>
</mrow>
<mo>)</mo>
</mrow>
</mrow>
</mfrac>
</mrow>
<mo>=</mo>
<mrow>
<mrow>
<msub>
<mi>Π</mi>
<mi>g</mi>
</msub>
<mo></mo>
<mfrac>
<mrow>
<mn>1</mn>
<mo>·</mo>
<msup>
<mi>e</mi>
<mrow>
<mo>-</mo>
<msub>
<mi>Mf</mi>
<mi>g</mi>
</msub>
</mrow>
</msup>
</mrow>
<msup>
<mi>e</mi>
<mrow>
<mrow>
<mo>-</mo>
<msub>
<mi>f</mi>
<mi>g</mi>
</msub>
</mrow>
<mo></mo>
<msub>
<mi>Mp</mi>
<mi>d</mi>
</msub>
</mrow>
</msup>
</mfrac>
</mrow>
<mo>=</mo>
<mrow>
<mrow>
<msub>
<mi>Π</mi>
<mi>g</mi>
</msub>
<mo></mo>
<msup>
<mi>e</mi>
<mrow>
<mo>-</mo>
<mrow>
<msub>
<mi>Mf</mi>
<mi>g</mi>
</msub>
<mo></mo>
<mrow>
<mo>(</mo>
<mrow>
<mn>1</mn>
<mo>-</mo>
<msub>
<mi>p</mi>
<mi>d</mi>
</msub>
</mrow>
<mo>)</mo>
</mrow>
</mrow>
</mrow>
</msup>
</mrow>
<mo>=</mo>
<mrow>
<msup>
<mi>e</mi>
<mrow>
<mrow>
<mo>-</mo>
<mrow>
<mi>M</mi>
<mo></mo>
<mrow>
<mo>(</mo>
<mrow>
<mn>1</mn>
<mo>-</mo>
<msub>
<mi>p</mi>
<mi>d</mi>
</msub>
</mrow>
<mo>)</mo>
</mrow>
</mrow>
</mrow>
<mo></mo>
<msub>
<mi>Σ</mi>
<mi>g</mi>
</msub>
<mo></mo>
<msub>
<mi>f</mi>
<mi>g</mi>
</msub>
</mrow>
</msup>
<mo>=</mo>
<msup>
<mi>e</mi>
<mrow>
<mo>-</mo>
<mrow>
<mi>M</mi>
<mo></mo>
<mrow>
<mo>(</mo>
<mrow>
<mn>1</mn>
<mo>-</mo>
<msub>
<mi>p</mi>
<mi>d</mi>
</msub>
</mrow>
<mo>)</mo>
</mrow>
</mrow>
</mrow>
</msup>
</mrow>
</mrow>
</mrow>
</mrow>
</mrow>
</mrow>
</mrow>
</math>
</maths>
</div>
</li> <li> <para-num num="[0989]"> </para-num> <div class="description-line" id="p-0966" num="0989">To account for fitness effects Applicants introduce a fitness parameter for each guide, denoted <b>8</b> <sub>g</sub>, and define the fitness normalization factor Z, such that</div>
</li> <li> <div class="description-line" id="p-0967" num="0000">
<maths id="MATH-US-00011" num="00011">
<math overflow="scroll">
<mrow>
<mstyle>
<mspace height="4.4ex" width="4.4em"> </mspace>
</mstyle>
<mo></mo>
<mrow>
<mi>Z</mi>
<mo>=</mo>
<mrow>
<mrow>
<msub>
<mi>E</mi>
<mrow>
<mi>P</mi>
<mo></mo>
<mrow>
<mo>(</mo>
<msub>
<mi>i</mi>
<mi>g</mi>
</msub>
<mo>)</mo>
</mrow>
</mrow>
</msub>
<mo></mo>
<mrow>
<mo>[</mo>
<msub>
<mi>δ</mi>
<mi>g</mi>
</msub>
<mo>]</mo>
</mrow>
</mrow>
<mo>=</mo>
<mrow>
<mrow>
<msub>
<mi>δ</mi>
<mn>0</mn>
</msub>
<mo></mo>
<mrow>
<munder>
<mo>∏</mo>
<mi>g</mi>
</munder>
<mo></mo>
<mrow>
<mo>(</mo>
<mrow>
<mrow>
<msup>
<mi>e</mi>
<mrow>
<mrow>
<mo>-</mo>
<msub>
<mi>f</mi>
<mi>g</mi>
</msub>
</mrow>
<mo></mo>
<mi>M</mi>
</mrow>
</msup>
<mo></mo>
<mrow>
<mo>(</mo>
<mrow>
<mn>1</mn>
<mo>-</mo>
<msub>
<mi>δ</mi>
<mi>g</mi>
</msub>
</mrow>
<mo>)</mo>
</mrow>
</mrow>
<mo>+</mo>
<msub>
<mi>δ</mi>
<mi>g</mi>
</msub>
</mrow>
<mo>)</mo>
</mrow>
</mrow>
</mrow>
<mo>=</mo>
<mrow>
<msub>
<mi>δ</mi>
<mn>0</mn>
</msub>
<mo></mo>
<mrow>
<munder>
<mo>∏</mo>
<mi>g</mi>
</munder>
<mo></mo>
<msub>
<mi>α</mi>
<mi>g</mi>
</msub>
</mrow>
</mrow>
</mrow>
</mrow>
</mrow>
</mrow>
</math>
</maths>
<maths id="MATH-US-00011-2" num="00011.2">
<math overflow="scroll">
<mrow>
<mrow>
<mi>Z</mi>
<mo>=</mo>
<mrow>
<mrow>
<msub>
<mi>E</mi>
<mrow>
<mi>P</mi>
<mo></mo>
<mrow>
<mo>(</mo>
<msub>
<mi>i</mi>
<mi>g</mi>
</msub>
<mo>)</mo>
</mrow>
</mrow>
</msub>
<mo></mo>
<mrow>
<mo>[</mo>
<msub>
<mi>δ</mi>
<mi>g</mi>
</msub>
<mo>]</mo>
</mrow>
</mrow>
<mo>=</mo>
<mrow>
<mrow>
<munder>
<mo>∑</mo>
<mrow>
<mo>(</mo>
<mrow>
<msub>
<mi>n</mi>
<mn>2</mn>
</msub>
<mo>,</mo>
<mi>…</mi>
<mo></mo>
<mstyle>
<mspace height="0.3ex" width="0.3em"> </mspace>
</mstyle>
<mo>,</mo>
<msub>
<mi>n</mi>
<mi>k</mi>
</msub>
</mrow>
<mo>)</mo>
</mrow>
</munder>
<mo></mo>
<mrow>
<mrow>
<mi>P</mi>
<mo></mo>
<mrow>
<mo>(</mo>
<mrow>
<mrow>
<msub>
<mi>i</mi>
<mn>1</mn>
</msub>
<mo>=</mo>
<msub>
<mi>n</mi>
<mn>1</mn>
</msub>
</mrow>
<mo>,</mo>
<mstyle>
<mspace height="0.3ex" width="0.3em"> </mspace>
</mstyle>
<mo></mo>
<mi>…</mi>
<mo></mo>
<mstyle>
<mspace height="0.6ex" width="0.6em"> </mspace>
</mstyle>
<mo>,</mo>
<mrow>
<msub>
<mi>i</mi>
<mi>k</mi>
</msub>
<mo>=</mo>
<msub>
<mi>n</mi>
<mi>k</mi>
</msub>
</mrow>
</mrow>
<mo>)</mo>
</mrow>
</mrow>
<mo></mo>
<msub>
<mi>δ</mi>
<mn>0</mn>
</msub>
<mo></mo>
<msubsup>
<mi>δ</mi>
<mn>1</mn>
<mrow>
<msub>
<mi>n</mi>
<mn>1</mn>
</msub>
<mo>&gt;</mo>
<mn>0</mn>
</mrow>
</msubsup>
<mo></mo>
<mstyle>
<mspace height="0.8ex" width="0.8em"> </mspace>
</mstyle>
<mo></mo>
<mi>…</mi>
<mo></mo>
<mstyle>
<mspace height="0.8ex" width="0.8em"> </mspace>
</mstyle>
<mo></mo>
<msubsup>
<mi>δ</mi>
<mi>k</mi>
<mrow>
<msub>
<mi>i</mi>
<mi>k</mi>
</msub>
<mo>&gt;</mo>
<mn>0</mn>
</mrow>
</msubsup>
</mrow>
</mrow>
<mo>=</mo>
<mrow>
<mrow>
<msub>
<mi>δ</mi>
<mn>0</mn>
</msub>
<mo></mo>
<mrow>
<munder>
<mo>∑</mo>
<mrow>
<mo>(</mo>
<mrow>
<msub>
<mi>n</mi>
<mn>2</mn>
</msub>
<mo>,</mo>
<mi>…</mi>
<mo></mo>
<mstyle>
<mspace height="0.6ex" width="0.6em"> </mspace>
</mstyle>
<mo>,</mo>
<msub>
<mi>n</mi>
<mi>k</mi>
</msub>
</mrow>
<mo>)</mo>
</mrow>
</munder>
<mo></mo>
<mrow>
<mo>[</mo>
<mrow>
<mrow>
<mrow>
<mi>P</mi>
<mo></mo>
<mrow>
<mo>(</mo>
<mrow>
<mrow>
<msub>
<mi>i</mi>
<mn>1</mn>
</msub>
<mo>=</mo>
<mn>0</mn>
</mrow>
<mo>,</mo>
<mrow>
<msub>
<mi>i</mi>
<mn>2</mn>
</msub>
<mo>=</mo>
<mrow>
<msub>
<mi>n</mi>
<mn>2</mn>
</msub>
<mo></mo>
<mstyle>
<mspace height="0.8ex" width="0.8em"> </mspace>
</mstyle>
<mo></mo>
<mi>…</mi>
</mrow>
</mrow>
<mo></mo>
<mstyle>
<mspace height="0.8ex" width="0.8em"> </mspace>
</mstyle>
<mo>,</mo>
<mrow>
<msub>
<mi>i</mi>
<mi>k</mi>
</msub>
<mo>=</mo>
<msub>
<mi>n</mi>
<mi>k</mi>
</msub>
</mrow>
</mrow>
<mo>)</mo>
</mrow>
</mrow>
<mo></mo>
<msubsup>
<mi>δ</mi>
<mn>2</mn>
<mrow>
<msub>
<mi>i</mi>
<mn>2</mn>
</msub>
<mo>&gt;</mo>
<mn>0</mn>
</mrow>
</msubsup>
<mo></mo>
<mstyle>
<mspace height="0.8ex" width="0.8em"> </mspace>
</mstyle>
<mo></mo>
<mi>…</mi>
<mo></mo>
<mstyle>
<mspace height="0.8ex" width="0.8em"> </mspace>
</mstyle>
<mo></mo>
<msubsup>
<mi>δ</mi>
<mi>k</mi>
<mrow>
<msub>
<mi>i</mi>
<mi>k</mi>
</msub>
<mo>&gt;</mo>
<mn>0</mn>
</mrow>
</msubsup>
</mrow>
<mo>+</mo>
<mrow>
<mrow>
<mi>P</mi>
<mo></mo>
<mrow>
<mo>(</mo>
<mrow>
<mrow>
<msub>
<mi>i</mi>
<mn>1</mn>
</msub>
<mo>&gt;</mo>
<mn>0</mn>
</mrow>
<mo>,</mo>
<mrow>
<msub>
<mi>i</mi>
<mn>2</mn>
</msub>
<mo>=</mo>
<mrow>
<msub>
<mi>n</mi>
<mn>2</mn>
</msub>
<mo></mo>
<mstyle>
<mspace height="0.8ex" width="0.8em"> </mspace>
</mstyle>
<mo></mo>
<mi>…</mi>
</mrow>
</mrow>
<mo></mo>
<mstyle>
<mspace height="0.8ex" width="0.8em"> </mspace>
</mstyle>
<mo>,</mo>
<msub>
<mi>i</mi>
<mi>k</mi>
</msub>
</mrow>
<mo>)</mo>
</mrow>
</mrow>
<mo></mo>
<msub>
<mi>δ</mi>
<mn>1</mn>
</msub>
<mo></mo>
<msubsup>
<mi>δ</mi>
<mn>2</mn>
<mrow>
<msub>
<mi>i</mi>
<mn>2</mn>
</msub>
<mo>&gt;</mo>
<mn>0</mn>
</mrow>
</msubsup>
<mo></mo>
<mstyle>
<mspace height="0.8ex" width="0.8em"> </mspace>
</mstyle>
<mo></mo>
<mi>…</mi>
<mo></mo>
<mstyle>
<mspace height="0.8ex" width="0.8em"> </mspace>
</mstyle>
<mo></mo>
<msubsup>
<mi>δ</mi>
<mi>k</mi>
<mrow>
<msub>
<mi>i</mi>
<mi>k</mi>
</msub>
<mo>&gt;</mo>
<mn>0</mn>
</mrow>
</msubsup>
</mrow>
</mrow>
<mo>]</mo>
</mrow>
</mrow>
</mrow>
<mo>=</mo>
<mrow>
<mrow>
<msub>
<mi>δ</mi>
<mn>0</mn>
</msub>
<mo></mo>
<mrow>
<munder>
<mo>∑</mo>
<mrow>
<mo>(</mo>
<mrow>
<msub>
<mi>n</mi>
<mn>2</mn>
</msub>
<mo>,</mo>
<mstyle>
<mspace height="0.3ex" width="0.3em"> </mspace>
</mstyle>
<mo></mo>
<mi>…</mi>
<mo></mo>
<mstyle>
<mspace height="0.3ex" width="0.3em"> </mspace>
</mstyle>
<mo>,</mo>
<msub>
<mi>n</mi>
<mi>k</mi>
</msub>
</mrow>
<mo>)</mo>
</mrow>
</munder>
<mo></mo>
<mrow>
<mrow>
<mo>(</mo>
<mrow>
<mrow>
<mi>P</mi>
<mo></mo>
<mrow>
<mo>(</mo>
<mrow>
<msub>
<mi>i</mi>
<mn>1</mn>
</msub>
<mo>=</mo>
<mn>0</mn>
</mrow>
<mo>)</mo>
</mrow>
</mrow>
<mo>+</mo>
<mrow>
<mrow>
<mi>P</mi>
<mo></mo>
<mrow>
<mo>(</mo>
<mrow>
<msub>
<mi>i</mi>
<mn>1</mn>
</msub>
<mo>&gt;</mo>
<mn>0</mn>
</mrow>
<mo>)</mo>
</mrow>
</mrow>
<mo></mo>
<msub>
<mi>δ</mi>
<mn>1</mn>
</msub>
</mrow>
</mrow>
<mo>)</mo>
</mrow>
<mo></mo>
<mrow>
<munder>
<mo>∏</mo>
<mrow>
<mn>2</mn>
<mo>≤</mo>
<mi>g</mi>
<mo>≤</mo>
<mi>k</mi>
</mrow>
</munder>
<mo></mo>
<mrow>
<mrow>
<mi>P</mi>
<mo></mo>
<mrow>
<mo>(</mo>
<mrow>
<msub>
<mi>i</mi>
<mi>g</mi>
</msub>
<mo>=</mo>
<msub>
<mi>n</mi>
<mi>g</mi>
</msub>
</mrow>
<mo>)</mo>
</mrow>
</mrow>
<mo></mo>
<msubsup>
<mi>δ</mi>
<mn>1</mn>
<mrow>
<msub>
<mi>i</mi>
<mi>g</mi>
</msub>
<mo>&gt;</mo>
<mn>0</mn>
</mrow>
</msubsup>
</mrow>
</mrow>
</mrow>
</mrow>
</mrow>
<mo>=</mo>
<mrow>
<mrow>
<msub>
<mi>δ</mi>
<mn>0</mn>
</msub>
<mo></mo>
<mrow>
<munder>
<mo>∏</mo>
<mi>g</mi>
</munder>
<mo></mo>
<mrow>
<mo>(</mo>
<mrow>
<mrow>
<mi>P</mi>
<mo></mo>
<mrow>
<mo>(</mo>
<mrow>
<msub>
<mi>i</mi>
<mi>g</mi>
</msub>
<mo>=</mo>
<mn>0</mn>
</mrow>
<mo>)</mo>
</mrow>
</mrow>
<mo>+</mo>
<mrow>
<mrow>
<mi>P</mi>
<mo></mo>
<mrow>
<mo>(</mo>
<mrow>
<msub>
<mi>i</mi>
<mi>g</mi>
</msub>
<mo>&gt;</mo>
<mn>0</mn>
</mrow>
<mo>)</mo>
</mrow>
</mrow>
<mo></mo>
<msub>
<mi>δ</mi>
<mi>g</mi>
</msub>
</mrow>
</mrow>
<mo>)</mo>
</mrow>
</mrow>
</mrow>
<mo>=</mo>
<mrow>
<mrow>
<msub>
<mi>δ</mi>
<mn>0</mn>
</msub>
<mo></mo>
<mrow>
<munder>
<mo>∏</mo>
<mi>g</mi>
</munder>
<mo></mo>
<mrow>
<mo>(</mo>
<mrow>
<mrow>
<msup>
<mi>e</mi>
<mrow>
<mrow>
<mo>-</mo>
<msub>
<mi>f</mi>
<mi>g</mi>
</msub>
</mrow>
<mo></mo>
<mi>M</mi>
</mrow>
</msup>
<mo></mo>
<mrow>
<mo>(</mo>
<mrow>
<mn>1</mn>
<mo>-</mo>
<msub>
<mi>δ</mi>
<mi>g</mi>
</msub>
</mrow>
<mo>)</mo>
</mrow>
</mrow>
<mo>+</mo>
<msub>
<mi>δ</mi>
<mi>g</mi>
</msub>
</mrow>
<mo>)</mo>
</mrow>
</mrow>
</mrow>
<mo>=</mo>
<mrow>
<mrow>
<msub>
<mi>δ</mi>
<mn>0</mn>
</msub>
<mo></mo>
<mrow>
<munder>
<mo>∏</mo>
<mi>g</mi>
</munder>
<mo></mo>
<mrow>
<msub>
<mi>α</mi>
<mi>g</mi>
</msub>
<mo></mo>
<mstyle>
<mtext>
</mtext>
</mstyle>
<mo></mo>
<mstyle>
<mspace height="4.4ex" width="4.4em"> </mspace>
</mstyle>
<mo></mo>
<mi>Where</mi>
<mo></mo>
<mstyle>
<mspace height="0.8ex" width="0.8em"> </mspace>
</mstyle>
<mo></mo>
<msub>
<mi>α</mi>
<mi>g</mi>
</msub>
</mrow>
</mrow>
</mrow>
<mo>=</mo>
<mrow>
<mrow>
<msup>
<mi>e</mi>
<mrow>
<mrow>
<mo>-</mo>
<msub>
<mi>f</mi>
<mi>g</mi>
</msub>
</mrow>
<mo></mo>
<mi>M</mi>
</mrow>
</msup>
<mo></mo>
<mrow>
<mo>(</mo>
<mrow>
<mn>1</mn>
<mo>-</mo>
<msub>
<mi>δ</mi>
<mi>g</mi>
</msub>
</mrow>
<mo>)</mo>
</mrow>
</mrow>
<mo>+</mo>
<msub>
<mi>δ</mi>
<mi>g</mi>
</msub>
</mrow>
</mrow>
</mrow>
</mrow>
</mrow>
</mrow>
</mrow>
</mrow>
</mrow>
<mo></mo>
<mstyle>
<mspace height="0.3ex" width="0.3em"> </mspace>
</mstyle>
</mrow>
</math>
</maths>
</div>
</li> <li> <para-num num="[0990]"> </para-num> <div class="description-line" id="p-0968" num="0990">Applicants adjust the probability that a cell that was infected i<sub>g </sub>times with guide g to account for the fitness effects.</div>
</li> <li> <div class="description-line" id="p-0969" num="0000">
<maths id="MATH-US-00012" num="00012">
<math overflow="scroll">
<mrow>
<mstyle>
<mspace height="4.4ex" width="4.4em"> </mspace>
</mstyle>
<mo></mo>
<mrow>
<mrow>
<msub>
<mi>P</mi>
<mi>f</mi>
</msub>
<mo></mo>
<mrow>
<mo>(</mo>
<mrow>
<msub>
<mi>i</mi>
<mi>g</mi>
</msub>
<mo>=</mo>
<msub>
<mi>n</mi>
<mi>g</mi>
</msub>
</mrow>
<mo>)</mo>
</mrow>
</mrow>
<mo>=</mo>
<mfrac>
<mrow>
<msubsup>
<mi>δ</mi>
<mi>g</mi>
<mrow>
<msub>
<mi>n</mi>
<mi>g</mi>
</msub>
<mo>&gt;</mo>
<mn>0</mn>
</mrow>
</msubsup>
<mo></mo>
<mrow>
<mi>P</mi>
<mo></mo>
<mrow>
<mo>(</mo>
<mrow>
<msub>
<mi>i</mi>
<mi>g</mi>
</msub>
<mo>=</mo>
<msub>
<mi>n</mi>
<mi>g</mi>
</msub>
</mrow>
<mo>)</mo>
</mrow>
</mrow>
</mrow>
<msub>
<mi>α</mi>
<mi>g</mi>
</msub>
</mfrac>
</mrow>
</mrow>
</math>
</maths>
<maths id="MATH-US-00012-2" num="00012.2">
<math overflow="scroll">
<mrow>
<mrow>
<msub>
<mi>P</mi>
<mi>f</mi>
</msub>
<mo></mo>
<mrow>
<mo>(</mo>
<mrow>
<msub>
<mi>i</mi>
<mn>1</mn>
</msub>
<mo>=</mo>
<msub>
<mi>n</mi>
<mn>1</mn>
</msub>
</mrow>
<mo>)</mo>
</mrow>
</mrow>
<mo>=</mo>
<mrow>
<mrow>
<mfrac>
<mn>1</mn>
<mi>Z</mi>
</mfrac>
<mo></mo>
<mrow>
<munder>
<mo>∑</mo>
<mrow>
<mo>(</mo>
<mrow>
<msub>
<mi>n</mi>
<mn>2</mn>
</msub>
<mo>,</mo>
<mstyle>
<mspace height="0.3ex" width="0.3em"> </mspace>
</mstyle>
<mo></mo>
<mi>…</mi>
<mo></mo>
<mstyle>
<mspace height="0.6ex" width="0.6em"> </mspace>
</mstyle>
<mo>,</mo>
<msub>
<mi>n</mi>
<mi>k</mi>
</msub>
</mrow>
<mo>)</mo>
</mrow>
</munder>
<mo></mo>
<mrow>
<mrow>
<mi>P</mi>
<mo></mo>
<mrow>
<mo>(</mo>
<mrow>
<mrow>
<msub>
<mi>i</mi>
<mn>1</mn>
</msub>
<mo>=</mo>
<msub>
<mi>n</mi>
<mn>1</mn>
</msub>
</mrow>
<mo>,</mo>
<mrow>
<msub>
<mi>i</mi>
<mn>2</mn>
</msub>
<mo>=</mo>
<mrow>
<msub>
<mi>n</mi>
<mn>2</mn>
</msub>
<mo></mo>
<mstyle>
<mspace height="0.8ex" width="0.8em"> </mspace>
</mstyle>
<mo></mo>
<mi>…</mi>
</mrow>
</mrow>
<mo></mo>
<mstyle>
<mspace height="0.8ex" width="0.8em"> </mspace>
</mstyle>
<mo>,</mo>
<mrow>
<msub>
<mi>i</mi>
<mi>k</mi>
</msub>
<mo>=</mo>
<msub>
<mi>n</mi>
<mi>k</mi>
</msub>
</mrow>
</mrow>
<mo>)</mo>
</mrow>
</mrow>
<mo></mo>
<msub>
<mi>δ</mi>
<mn>0</mn>
</msub>
<mo></mo>
<msubsup>
<mi>δ</mi>
<mn>1</mn>
<mrow>
<msub>
<mi>n</mi>
<mn>1</mn>
</msub>
<mo>&gt;</mo>
<mn>0</mn>
</mrow>
</msubsup>
<mo></mo>
<mstyle>
<mspace height="0.8ex" width="0.8em"> </mspace>
</mstyle>
<mo></mo>
<mi>…</mi>
<mo></mo>
<mstyle>
<mspace height="0.8ex" width="0.8em"> </mspace>
</mstyle>
<mo></mo>
<msubsup>
<mi>δ</mi>
<mi>k</mi>
<mrow>
<msub>
<mi>i</mi>
<mi>k</mi>
</msub>
<mo>&gt;</mo>
<mn>0</mn>
</mrow>
</msubsup>
</mrow>
</mrow>
</mrow>
<mo>=</mo>
<mrow>
<mrow>
<mfrac>
<mn>1</mn>
<mrow>
<msub>
<mi>Π</mi>
<mi>g</mi>
</msub>
<mo></mo>
<mrow>
<mo>(</mo>
<mrow>
<mrow>
<msup>
<mi>e</mi>
<mrow>
<mrow>
<mo>-</mo>
<msub>
<mi>f</mi>
<mi>g</mi>
</msub>
</mrow>
<mo></mo>
<mi>M</mi>
</mrow>
</msup>
<mo></mo>
<mrow>
<mo>(</mo>
<mrow>
<mn>1</mn>
<mo>-</mo>
<msub>
<mi>δ</mi>
<mi>g</mi>
</msub>
</mrow>
<mo>)</mo>
</mrow>
</mrow>
<mo>+</mo>
<msub>
<mi>δ</mi>
<mi>g</mi>
</msub>
</mrow>
<mo>)</mo>
</mrow>
</mrow>
</mfrac>
<mo></mo>
<msubsup>
<mi>δ</mi>
<mn>1</mn>
<mrow>
<msub>
<mi>n</mi>
<mn>1</mn>
</msub>
<mo>&gt;</mo>
<mn>0</mn>
</mrow>
</msubsup>
<mo></mo>
<mrow>
<mi>P</mi>
<mo></mo>
<mrow>
<mo>(</mo>
<mrow>
<msub>
<mi>i</mi>
<mn>1</mn>
</msub>
<mo>=</mo>
<msub>
<mi>n</mi>
<mn>1</mn>
</msub>
</mrow>
<mo>)</mo>
</mrow>
</mrow>
<mo></mo>
<mrow>
<munder>
<mo>∏</mo>
<mrow>
<mi>g</mi>
<mo>≠</mo>
<mn>1</mn>
</mrow>
</munder>
<mo></mo>
<mrow>
<mo>(</mo>
<mrow>
<mrow>
<msup>
<mi>e</mi>
<mrow>
<mrow>
<mo>-</mo>
<msub>
<mi>f</mi>
<mi>g</mi>
</msub>
</mrow>
<mo></mo>
<mi>M</mi>
</mrow>
</msup>
<mo></mo>
<mrow>
<mo>(</mo>
<mrow>
<mn>1</mn>
<mo>-</mo>
<msub>
<mi>δ</mi>
<mi>g</mi>
</msub>
</mrow>
<mo>)</mo>
</mrow>
</mrow>
<mo>+</mo>
<msub>
<mi>δ</mi>
<mi>g</mi>
</msub>
</mrow>
<mo>)</mo>
</mrow>
</mrow>
</mrow>
<mo>=</mo>
<mfrac>
<mrow>
<msubsup>
<mi>δ</mi>
<mn>1</mn>
<mrow>
<msub>
<mi>n</mi>
<mn>1</mn>
</msub>
<mo>&gt;</mo>
<mn>0</mn>
</mrow>
</msubsup>
<mo></mo>
<mrow>
<mi>P</mi>
<mo></mo>
<mrow>
<mo>(</mo>
<mrow>
<msub>
<mi>i</mi>
<mn>1</mn>
</msub>
<mo>=</mo>
<msub>
<mi>n</mi>
<mn>1</mn>
</msub>
</mrow>
<mo>)</mo>
</mrow>
</mrow>
</mrow>
<mrow>
<mrow>
<msup>
<mi>e</mi>
<mrow>
<mrow>
<mo>-</mo>
<msub>
<mi>f</mi>
<mn>1</mn>
</msub>
</mrow>
<mo></mo>
<mi>M</mi>
</mrow>
</msup>
<mo></mo>
<mrow>
<mo>(</mo>
<mrow>
<mn>1</mn>
<mo>-</mo>
<msub>
<mi>δ</mi>
<mn>1</mn>
</msub>
</mrow>
<mo>)</mo>
</mrow>
</mrow>
<mo>+</mo>
<msub>
<mi>δ</mi>
<mn>1</mn>
</msub>
</mrow>
</mfrac>
</mrow>
</mrow>
</mrow>
</math>
</maths>
</div>
</li> <li> <para-num num="[0991]"> </para-num> <div class="description-line" id="p-0970" num="0991">The probability that guide g was not detected in a specific cell is now</div>
</li> <li> <div class="description-line" id="p-0971" num="0000">
<maths id="MATH-US-00013" num="00013">
<math overflow="scroll">
<mrow>
<mrow>
<mi>P</mi>
<mo></mo>
<mrow>
<mo>(</mo>
<mrow>
<msub>
<mi>d</mi>
<mi>g</mi>
</msub>
<mo>=</mo>
<mn>0</mn>
</mrow>
<mo>)</mo>
</mrow>
</mrow>
<mo>=</mo>
<mfrac>
<mrow>
<mrow>
<msup>
<mi>e</mi>
<mrow>
<mrow>
<mo>-</mo>
<msub>
<mi>f</mi>
<mi>g</mi>
</msub>
</mrow>
<mo></mo>
<mi>M</mi>
</mrow>
</msup>
<mo></mo>
<mrow>
<mo>(</mo>
<mrow>
<mn>1</mn>
<mo>-</mo>
<msub>
<mi>δ</mi>
<mi>g</mi>
</msub>
</mrow>
<mo>)</mo>
</mrow>
</mrow>
<mo>+</mo>
<mrow>
<msub>
<mi>δ</mi>
<mi>g</mi>
</msub>
<mo></mo>
<msup>
<mi>e</mi>
<mrow>
<mrow>
<mo>-</mo>
<msub>
<mi>f</mi>
<mi>g</mi>
</msub>
</mrow>
<mo></mo>
<msub>
<mi>Mp</mi>
<mi>d</mi>
</msub>
</mrow>
</msup>
</mrow>
</mrow>
<mrow>
<mrow>
<msup>
<mi>e</mi>
<mrow>
<mrow>
<mo>-</mo>
<msub>
<mi>f</mi>
<mi>g</mi>
</msub>
</mrow>
<mo></mo>
<mi>M</mi>
</mrow>
</msup>
<mo></mo>
<mrow>
<mo>(</mo>
<mrow>
<mn>1</mn>
<mo>-</mo>
<msub>
<mi>δ</mi>
<mi>g</mi>
</msub>
</mrow>
<mo>)</mo>
</mrow>
</mrow>
<mo>+</mo>
<msub>
<mi>δ</mi>
<mi>g</mi>
</msub>
</mrow>
</mfrac>
</mrow>
</math>
</maths>
<maths id="MATH-US-00013-2" num="00013.2">
<math overflow="scroll">
<mrow>
<mrow>
<mi>P</mi>
<mo></mo>
<mrow>
<mo>(</mo>
<mrow>
<msub>
<mi>d</mi>
<mi>g</mi>
</msub>
<mo>=</mo>
<mrow>
<mn>0</mn>
<mo>|</mo>
<msub>
<mi>i</mi>
<mi>g</mi>
</msub>
</mrow>
</mrow>
<mo>)</mo>
</mrow>
</mrow>
<mo>=</mo>
<mrow>
<msup>
<mrow>
<mo>(</mo>
<mrow>
<mn>1</mn>
<mo>-</mo>
<msub>
<mi>p</mi>
<mi>d</mi>
</msub>
</mrow>
<mo>)</mo>
</mrow>
<msub>
<mi>i</mi>
<mi>g</mi>
</msub>
</msup>
<mo>=</mo>
<msup>
<mi>q</mi>
<msub>
<mi>i</mi>
<mi>g</mi>
</msub>
</msup>
</mrow>
</mrow>
</math>
</maths>
<maths id="MATH-US-00013-3" num="00013.3">
<math overflow="scroll">
<mtable>
<mtr>
<mtd>
<mrow>
<mrow>
<msub>
<mi>P</mi>
<mi>f</mi>
</msub>
<mo></mo>
<mrow>
<mo>(</mo>
<mrow>
<msub>
<mi>d</mi>
<mi>g</mi>
</msub>
<mo>=</mo>
<mn>0</mn>
</mrow>
<mo>)</mo>
</mrow>
</mrow>
<mo>=</mo>
<mi> </mi>
<mo></mo>
<mrow>
<munder>
<mo>∑</mo>
<mi>n</mi>
</munder>
<mo></mo>
<mrow>
<msup>
<mi>q</mi>
<mi>n</mi>
</msup>
<mo></mo>
<mrow>
<msub>
<mi>P</mi>
<mi>f</mi>
</msub>
<mo></mo>
<mrow>
<mo>(</mo>
<mrow>
<msub>
<mi>i</mi>
<mi>g</mi>
</msub>
<mo>=</mo>
<mi>n</mi>
</mrow>
<mo>)</mo>
</mrow>
</mrow>
</mrow>
</mrow>
</mrow>
</mtd>
</mtr>
<mtr>
<mtd>
<mrow>
<mo>=</mo>
<mi> </mi>
<mo></mo>
<mrow>
<munder>
<mo>∑</mo>
<mi>n</mi>
</munder>
<mo></mo>
<mrow>
<msup>
<mi>q</mi>
<mi>n</mi>
</msup>
<mo></mo>
<mfrac>
<mrow>
<msubsup>
<mi>δ</mi>
<mi>g</mi>
<mrow>
<mi>n</mi>
<mo>&gt;</mo>
<mn>0</mn>
</mrow>
</msubsup>
<mo></mo>
<mrow>
<mi>P</mi>
<mo></mo>
<mrow>
<mo>(</mo>
<mrow>
<msub>
<mi>i</mi>
<mi>g</mi>
</msub>
<mo>=</mo>
<mi>n</mi>
</mrow>
<mo>)</mo>
</mrow>
</mrow>
</mrow>
<msub>
<mi>α</mi>
<mi>g</mi>
</msub>
</mfrac>
</mrow>
</mrow>
</mrow>
</mtd>
</mtr>
<mtr>
<mtd>
<mrow>
<mo>=</mo>
<mi> </mi>
<mo></mo>
<mrow>
<mfrac>
<mn>1</mn>
<msub>
<mi>α</mi>
<mi>g</mi>
</msub>
</mfrac>
<mo></mo>
<mrow>
<munder>
<mo>∑</mo>
<mi>n</mi>
</munder>
<mo></mo>
<mrow>
<msup>
<mi>q</mi>
<mi>n</mi>
</msup>
<mo></mo>
<msubsup>
<mi>δ</mi>
<mi>g</mi>
<mrow>
<mi>n</mi>
<mo>&gt;</mo>
<mn>0</mn>
</mrow>
</msubsup>
<mo></mo>
<mrow>
<mi>P</mi>
<mo></mo>
<mrow>
<mo>(</mo>
<mrow>
<msub>
<mi>i</mi>
<mi>g</mi>
</msub>
<mo>=</mo>
<mi>n</mi>
</mrow>
<mo>)</mo>
</mrow>
</mrow>
</mrow>
</mrow>
</mrow>
</mrow>
</mtd>
</mtr>
<mtr>
<mtd>
<mrow>
<mo>=</mo>
<mi> </mi>
<mo></mo>
<mrow>
<mfrac>
<mn>1</mn>
<msub>
<mi>α</mi>
<mi>g</mi>
</msub>
</mfrac>
<mo>[</mo>
<mrow>
<mrow>
<mi>P</mi>
<mo></mo>
<mrow>
<mo>(</mo>
<mrow>
<msub>
<mi>i</mi>
<mi>g</mi>
</msub>
<mo>=</mo>
<mn>0</mn>
</mrow>
<mo>)</mo>
</mrow>
</mrow>
<mo>-</mo>
<mrow>
<msub>
<mi>δ</mi>
<mi>g</mi>
</msub>
<mo></mo>
<mrow>
<mi>P</mi>
<mo></mo>
<mrow>
<mo>(</mo>
<mrow>
<msub>
<mi>i</mi>
<mi>g</mi>
</msub>
<mo>=</mo>
<mn>0</mn>
</mrow>
<mo>)</mo>
</mrow>
</mrow>
</mrow>
<mo>+</mo>
<mrow>
<msub>
<mi>δ</mi>
<mi>g</mi>
</msub>
<mo></mo>
<mrow>
<munder>
<mo>∑</mo>
<mi>n</mi>
</munder>
<mo></mo>
<mrow>
<msup>
<mi>q</mi>
<mi>n</mi>
</msup>
<mo></mo>
<mrow>
<mi>P</mi>
<mo></mo>
<mrow>
<mo>(</mo>
<mrow>
<msub>
<mi>i</mi>
<mi>g</mi>
</msub>
<mo>=</mo>
<mi>n</mi>
</mrow>
<mo>)</mo>
</mrow>
</mrow>
</mrow>
</mrow>
</mrow>
</mrow>
<mo>]</mo>
</mrow>
</mrow>
</mtd>
</mtr>
<mtr>
<mtd>
<mrow>
<mo>=</mo>
<mi> </mi>
<mo></mo>
<mrow>
<mfrac>
<mn>1</mn>
<msub>
<mi>α</mi>
<mi>g</mi>
</msub>
</mfrac>
<mo></mo>
<mrow>
<mo>[</mo>
<mrow>
<mrow>
<msup>
<mi>e</mi>
<mrow>
<mrow>
<mo>-</mo>
<msub>
<mi>f</mi>
<mi>g</mi>
</msub>
</mrow>
<mo></mo>
<mi>M</mi>
</mrow>
</msup>
<mo></mo>
<mrow>
<mo>(</mo>
<mrow>
<mn>1</mn>
<mo>-</mo>
<msub>
<mi>δ</mi>
<mi>g</mi>
</msub>
</mrow>
<mo>)</mo>
</mrow>
</mrow>
<mo>+</mo>
<mrow>
<msub>
<mi>δ</mi>
<mi>g</mi>
</msub>
<mo></mo>
<msup>
<mi>e</mi>
<mrow>
<mrow>
<mo>-</mo>
<msub>
<mi>f</mi>
<mi>g</mi>
</msub>
</mrow>
<mo></mo>
<msub>
<mi>Mp</mi>
<mi>d</mi>
</msub>
</mrow>
</msup>
</mrow>
</mrow>
<mo>]</mo>
</mrow>
</mrow>
</mrow>
</mtd>
</mtr>
<mtr>
<mtd>
<mrow>
<mo>=</mo>
<mi> </mi>
<mo></mo>
<mfrac>
<mrow>
<mrow>
<msup>
<mi>e</mi>
<mrow>
<mrow>
<mo>-</mo>
<msub>
<mi>f</mi>
<mi>g</mi>
</msub>
</mrow>
<mo></mo>
<mi>M</mi>
</mrow>
</msup>
<mo></mo>
<mrow>
<mo>(</mo>
<mrow>
<mn>1</mn>
<mo>-</mo>
<msub>
<mi>δ</mi>
<mi>g</mi>
</msub>
</mrow>
<mo>)</mo>
</mrow>
</mrow>
<mo>+</mo>
<mrow>
<msub>
<mi>δ</mi>
<mi>g</mi>
</msub>
<mo></mo>
<msup>
<mi>e</mi>
<mrow>
<mrow>
<mo>-</mo>
<msub>
<mi>f</mi>
<mi>g</mi>
</msub>
</mrow>
<mo></mo>
<msub>
<mi>Mp</mi>
<mi>d</mi>
</msub>
</mrow>
</msup>
</mrow>
</mrow>
<mrow>
<mrow>
<msup>
<mi>e</mi>
<mrow>
<mrow>
<mo>-</mo>
<msub>
<mi>f</mi>
<mi>g</mi>
</msub>
</mrow>
<mo></mo>
<mi>M</mi>
</mrow>
</msup>
<mo></mo>
<mrow>
<mo>(</mo>
<mrow>
<mn>1</mn>
<mo>-</mo>
<msub>
<mi>δ</mi>
<mi>g</mi>
</msub>
</mrow>
<mo>)</mo>
</mrow>
</mrow>
<mo>+</mo>
<msub>
<mi>δ</mi>
<mi>g</mi>
</msub>
</mrow>
</mfrac>
</mrow>
</mtd>
</mtr>
</mtable>
</math>
</maths>
</div>
</li> <li> <para-num num="[0992]"> </para-num> <div class="description-line" id="p-0972" num="0992">The probability of detecting guide g in a cell</div>
</li> <li> <div class="description-line" id="p-0973" num="0000">
<maths id="MATH-US-00014" num="00014">
<math overflow="scroll">
<mrow>
<mrow>
<msub>
<mi>P</mi>
<mi>f</mi>
</msub>
<mo></mo>
<mrow>
<mo>(</mo>
<mrow>
<msub>
<mi>d</mi>
<mi>g</mi>
</msub>
<mo>=</mo>
<mn>1</mn>
</mrow>
<mo>)</mo>
</mrow>
</mrow>
<mo>=</mo>
<mfrac>
<mrow>
<msub>
<mi>δ</mi>
<mi>g</mi>
</msub>
<mo></mo>
<mrow>
<mo>(</mo>
<mrow>
<mn>1</mn>
<mo>-</mo>
<msup>
<mi>e</mi>
<mrow>
<mrow>
<mo>-</mo>
<msub>
<mi>f</mi>
<mi>g</mi>
</msub>
</mrow>
<mo></mo>
<msub>
<mi>Mp</mi>
<mi>d</mi>
</msub>
</mrow>
</msup>
</mrow>
<mo>)</mo>
</mrow>
</mrow>
<msub>
<mi>α</mi>
<mi>g</mi>
</msub>
</mfrac>
</mrow>
</math>
</maths>
</div>
</li> <li> <para-num num="[0993]"> </para-num> <div class="description-line" id="p-0974" num="0993">The probability that a cell survived BFP selection given that none of the guides was detected</div>
</li> <li> <div class="description-line" id="p-0975" num="0000">
<maths id="MATH-US-00015" num="00015">
<math overflow="scroll">
<mrow>
<mstyle>
<mspace height="4.4ex" width="4.4em"> </mspace>
</mstyle>
<mo></mo>
<mrow>
<mrow>
<mi>P</mi>
<mo></mo>
<mrow>
<mo>(</mo>
<mrow>
<mrow>
<mi>B</mi>
<mo>|</mo>
<mrow>
<msub>
<mo>∀</mo>
<mi>g</mi>
</msub>
<mo></mo>
<msub>
<mi>d</mi>
<mi>g</mi>
</msub>
</mrow>
</mrow>
<mo>=</mo>
<mn>0</mn>
</mrow>
<mo>)</mo>
</mrow>
</mrow>
<mo>=</mo>
<mrow>
<mn>1</mn>
<mo>-</mo>
<mfrac>
<mn>1</mn>
<mrow>
<msub>
<mi>Π</mi>
<mi>g</mi>
</msub>
<mo></mo>
<mrow>
<mo>(</mo>
<mrow>
<mn>1</mn>
<mo>-</mo>
<msub>
<mi>δ</mi>
<mi>g</mi>
</msub>
<mo>+</mo>
<mrow>
<msub>
<mi>δ</mi>
<mi>g</mi>
</msub>
<mo></mo>
<msup>
<mi>e</mi>
<mrow>
<mrow>
<mo>-</mo>
<msub>
<mi>f</mi>
<mi>g</mi>
</msub>
</mrow>
<mo></mo>
<mrow>
<mi>M</mi>
<mo></mo>
<mrow>
<mo>(</mo>
<mrow>
<msub>
<mi>p</mi>
<mi>d</mi>
</msub>
<mo>-</mo>
<mn>1</mn>
</mrow>
<mo>)</mo>
</mrow>
</mrow>
</mrow>
</msup>
</mrow>
</mrow>
<mo>)</mo>
</mrow>
</mrow>
</mfrac>
</mrow>
</mrow>
</mrow>
</math>
</maths>
<maths id="MATH-US-00015-2" num="00015.2">
<math overflow="scroll">
<mrow>
<mrow>
<mi>P</mi>
<mo></mo>
<mrow>
<mo>(</mo>
<mrow>
<mrow>
<mrow>
<mo></mo>
<mi>B</mi>
</mrow>
<mo>|</mo>
<mrow>
<msub>
<mo>∀</mo>
<mi>g</mi>
</msub>
<mo></mo>
<msub>
<mi>d</mi>
<mi>g</mi>
</msub>
</mrow>
</mrow>
<mo>=</mo>
<mn>0</mn>
</mrow>
<mo>)</mo>
</mrow>
</mrow>
<mo>=</mo>
<mrow>
<mrow>
<mi>P</mi>
<mo></mo>
<mrow>
<mo>(</mo>
<mrow>
<mrow>
<msub>
<mo>∀</mo>
<mi>g</mi>
</msub>
<mo></mo>
<msub>
<mi>i</mi>
<mi>g</mi>
</msub>
</mrow>
<mo>=</mo>
<mrow>
<mrow>
<mn>0</mn>
<mo>|</mo>
<mrow>
<msub>
<mo>∀</mo>
<mi>g</mi>
</msub>
<mo></mo>
<msub>
<mi>d</mi>
<mi>g</mi>
</msub>
</mrow>
</mrow>
<mo>=</mo>
<mn>0</mn>
</mrow>
</mrow>
<mo>)</mo>
</mrow>
</mrow>
<mo>=</mo>
<mfrac>
<mrow>
<mover>
<mrow>
<mi>P</mi>
<mo></mo>
<mrow>
<mo>(</mo>
<mrow>
<mrow>
<msub>
<mo>∀</mo>
<mi>g</mi>
</msub>
<mo></mo>
<msub>
<mi>d</mi>
<mi>g</mi>
</msub>
</mrow>
<mo>=</mo>
<mrow>
<mrow>
<mn>0</mn>
<mo>|</mo>
<mrow>
<msub>
<mo>∀</mo>
<mi>g</mi>
</msub>
<mo></mo>
<msub>
<mi>i</mi>
<mi>g</mi>
</msub>
</mrow>
</mrow>
<mo>=</mo>
<mn>0</mn>
</mrow>
</mrow>
<mo>)</mo>
</mrow>
</mrow>
<mover>
<mi></mi>
<mn>1</mn>
</mover>
</mover>
<mo></mo>
<mrow>
<mi>P</mi>
<mo></mo>
<mrow>
<mo>(</mo>
<mrow>
<mrow>
<msub>
<mo>∀</mo>
<mi>g</mi>
</msub>
<mo></mo>
<msub>
<mi>i</mi>
<mi>g</mi>
</msub>
</mrow>
<mo>=</mo>
<mn>0</mn>
</mrow>
<mo>)</mo>
</mrow>
</mrow>
</mrow>
<mrow>
<mi>P</mi>
<mo></mo>
<mrow>
<mo>(</mo>
<mrow>
<mrow>
<msub>
<mo>∀</mo>
<mi>g</mi>
</msub>
<mo></mo>
<msub>
<mi>d</mi>
<mi>g</mi>
</msub>
</mrow>
<mo>=</mo>
<mn>0</mn>
</mrow>
<mo>)</mo>
</mrow>
</mrow>
</mfrac>
</mrow>
</mrow>
</math>
</maths>
<maths id="MATH-US-00015-3" num="00015.3">
<math overflow="scroll">
<mrow>
<mrow>
<munder>
<mo>∏</mo>
<mi>g</mi>
</munder>
<mo></mo>
<mfrac>
<mfrac>
<mrow>
<mi>P</mi>
<mo></mo>
<mrow>
<mo>(</mo>
<mrow>
<msub>
<mi>i</mi>
<mi>g</mi>
</msub>
<mo>=</mo>
<mn>0</mn>
</mrow>
<mo>)</mo>
</mrow>
</mrow>
<msub>
<mi>α</mi>
<mi>g</mi>
</msub>
</mfrac>
<mrow>
<mfrac>
<mn>1</mn>
<msub>
<mi>α</mi>
<mi>g</mi>
</msub>
</mfrac>
<mo></mo>
<mrow>
<mo>[</mo>
<mrow>
<mrow>
<msup>
<mi>e</mi>
<mrow>
<mrow>
<mo>-</mo>
<msub>
<mi>f</mi>
<mi>g</mi>
</msub>
</mrow>
<mo></mo>
<mi>M</mi>
</mrow>
</msup>
<mo></mo>
<mrow>
<mo>(</mo>
<mrow>
<mn>1</mn>
<mo>-</mo>
<msub>
<mi>δ</mi>
<mi>g</mi>
</msub>
</mrow>
<mo>)</mo>
</mrow>
</mrow>
<mo>+</mo>
<mrow>
<msub>
<mi>δ</mi>
<mi>g</mi>
</msub>
<mo></mo>
<msup>
<mi>e</mi>
<mrow>
<mrow>
<mo>-</mo>
<msub>
<mi>f</mi>
<mi>g</mi>
</msub>
</mrow>
<mo></mo>
<msub>
<mi>Mp</mi>
<mi>d</mi>
</msub>
</mrow>
</msup>
</mrow>
</mrow>
<mo>]</mo>
</mrow>
</mrow>
</mfrac>
</mrow>
<mo>=</mo>
<mrow>
<mrow>
<munder>
<mo>∏</mo>
<mi>g</mi>
</munder>
<mo></mo>
<mfrac>
<msup>
<mi>e</mi>
<mrow>
<mrow>
<mo>-</mo>
<msub>
<mi>f</mi>
<mi>g</mi>
</msub>
</mrow>
<mo></mo>
<mi>M</mi>
</mrow>
</msup>
<mrow>
<mrow>
<msup>
<mi>e</mi>
<mrow>
<mrow>
<mo>-</mo>
<msub>
<mi>f</mi>
<mi>g</mi>
</msub>
</mrow>
<mo></mo>
<mi>M</mi>
</mrow>
</msup>
<mo></mo>
<mrow>
<mo>(</mo>
<mrow>
<mn>1</mn>
<mo>-</mo>
<msub>
<mi>δ</mi>
<mi>g</mi>
</msub>
</mrow>
<mo>)</mo>
</mrow>
</mrow>
<mo>+</mo>
<mrow>
<msub>
<mi>δ</mi>
<mi>g</mi>
</msub>
<mo></mo>
<msup>
<mi>e</mi>
<mrow>
<mrow>
<mo>-</mo>
<msub>
<mi>f</mi>
<mi>g</mi>
</msub>
</mrow>
<mo></mo>
<msub>
<mi>Mp</mi>
<mi>d</mi>
</msub>
</mrow>
</msup>
</mrow>
</mrow>
</mfrac>
</mrow>
<mo>=</mo>
<mrow>
<mrow>
<munder>
<mo>∏</mo>
<mi>g</mi>
</munder>
<mo></mo>
<mfrac>
<msup>
<mi>e</mi>
<mrow>
<mrow>
<mo>-</mo>
<msub>
<mi>f</mi>
<mi>g</mi>
</msub>
</mrow>
<mo></mo>
<mi>M</mi>
</mrow>
</msup>
<mrow>
<msup>
<mi>e</mi>
<mrow>
<mrow>
<mo>-</mo>
<msub>
<mi>f</mi>
<mi>g</mi>
</msub>
</mrow>
<mo></mo>
<mi>M</mi>
</mrow>
</msup>
<mo></mo>
<mrow>
<mo>(</mo>
<mrow>
<mn>1</mn>
<mo>-</mo>
<msub>
<mi>δ</mi>
<mi>g</mi>
</msub>
<mo>+</mo>
<mrow>
<msub>
<mi>δ</mi>
<mi>g</mi>
</msub>
<mo></mo>
<msup>
<mi>e</mi>
<mrow>
<mrow>
<mo>-</mo>
<msub>
<mi>f</mi>
<mi>g</mi>
</msub>
</mrow>
<mo></mo>
<mrow>
<mi>M</mi>
<mo></mo>
<mrow>
<mo>(</mo>
<mrow>
<msub>
<mi>p</mi>
<mi>d</mi>
</msub>
<mo>-</mo>
<mn>1</mn>
</mrow>
<mo>)</mo>
</mrow>
</mrow>
</mrow>
</msup>
</mrow>
</mrow>
<mo>)</mo>
</mrow>
</mrow>
</mfrac>
</mrow>
<mo>=</mo>
<mfrac>
<mn>1</mn>
<mrow>
<msub>
<mi>Π</mi>
<mi>g</mi>
</msub>
<mo></mo>
<mrow>
<mo>(</mo>
<mrow>
<mn>1</mn>
<mo>-</mo>
<msub>
<mi>δ</mi>
<mi>g</mi>
</msub>
<mo>+</mo>
<mrow>
<msub>
<mi>δ</mi>
<mi>g</mi>
</msub>
<mo></mo>
<msup>
<mi>e</mi>
<mrow>
<mrow>
<mo>-</mo>
<msub>
<mi>f</mi>
<mi>g</mi>
</msub>
</mrow>
<mo></mo>
<mrow>
<mi>M</mi>
<mo></mo>
<mrow>
<mo>(</mo>
<mrow>
<msub>
<mi>p</mi>
<mi>d</mi>
</msub>
<mo>-</mo>
<mn>1</mn>
</mrow>
<mo>)</mo>
</mrow>
</mrow>
</mrow>
</msup>
</mrow>
</mrow>
<mo>)</mo>
</mrow>
</mrow>
</mfrac>
</mrow>
</mrow>
</mrow>
</math>
</maths>
</div>
</li> <li> <para-num num="[0994]"> </para-num> <div class="description-line" id="p-0976" num="0994">The probability of the BFP selection</div>
</li> <li> <div class="description-line" id="p-0977" num="0000">
<maths id="MATH-US-00016" num="00016">
<math overflow="scroll">
<mrow>
<mrow>
<mi>P</mi>
<mo></mo>
<mrow>
<mo>(</mo>
<mi>B</mi>
<mo>)</mo>
</mrow>
</mrow>
<mo>=</mo>
<mrow>
<mn>1</mn>
<mo>-</mo>
<mfrac>
<msup>
<mi>e</mi>
<mrow>
<mo>-</mo>
<mi>M</mi>
</mrow>
</msup>
<mrow>
<mrow>
<msub>
<mi>Π</mi>
<mi>g</mi>
</msub>
<mo></mo>
<mrow>
<msup>
<mi>e</mi>
<mrow>
<mrow>
<mo>-</mo>
<msub>
<mi>f</mi>
<mi>g</mi>
</msub>
</mrow>
<mo></mo>
<mi>M</mi>
</mrow>
</msup>
<mo></mo>
<mrow>
<mo>(</mo>
<mrow>
<mn>1</mn>
<mo>-</mo>
<msub>
<mi>δ</mi>
<mi>g</mi>
</msub>
</mrow>
<mo>)</mo>
</mrow>
</mrow>
</mrow>
<mo>+</mo>
<msub>
<mi>δ</mi>
<mi>g</mi>
</msub>
</mrow>
</mfrac>
</mrow>
</mrow>
</math>
</maths>
<maths id="MATH-US-00016-2" num="00016.2">
<math overflow="scroll">
<mtable>
<mtr>
<mtd>
<mrow>
<mrow>
<mi>P</mi>
<mo></mo>
<mrow>
<mo>(</mo>
<mrow>
<mo></mo>
<mi>B</mi>
</mrow>
<mo>)</mo>
</mrow>
</mrow>
<mo>=</mo>
<mi> </mi>
<mo></mo>
<mrow>
<mi>P</mi>
<mo></mo>
<mrow>
<mo>(</mo>
<mrow>
<mrow>
<msub>
<mo>∀</mo>
<mi>g</mi>
</msub>
<mo></mo>
<msub>
<mi>i</mi>
<mi>g</mi>
</msub>
</mrow>
<mo>=</mo>
<mn>0</mn>
</mrow>
<mo>)</mo>
</mrow>
</mrow>
</mrow>
</mtd>
</mtr>
<mtr>
<mtd>
<mrow>
<mo>=</mo>
<mi> </mi>
<mo></mo>
<mrow>
<munder>
<mo>∏</mo>
<mi>g</mi>
</munder>
<mo></mo>
<mfrac>
<mrow>
<mi>P</mi>
<mo></mo>
<mrow>
<mo>(</mo>
<mrow>
<msub>
<mi>i</mi>
<mi>g</mi>
</msub>
<mo>=</mo>
<msub>
<mi>n</mi>
<mn>1</mn>
</msub>
</mrow>
<mo>)</mo>
</mrow>
</mrow>
<mrow>
<mrow>
<msup>
<mi>e</mi>
<mrow>
<mrow>
<mo>-</mo>
<msub>
<mi>f</mi>
<mi>g</mi>
</msub>
</mrow>
<mo></mo>
<mi>M</mi>
</mrow>
</msup>
<mo></mo>
<mrow>
<mo>(</mo>
<mrow>
<mn>1</mn>
<mo>-</mo>
<msub>
<mi>δ</mi>
<mi>g</mi>
</msub>
</mrow>
<mo>)</mo>
</mrow>
</mrow>
<mo>+</mo>
<msub>
<mi>δ</mi>
<mi>g</mi>
</msub>
</mrow>
</mfrac>
</mrow>
</mrow>
</mtd>
</mtr>
<mtr>
<mtd>
<mrow>
<mo>=</mo>
<mi> </mi>
<mo></mo>
<mfrac>
<msup>
<mi>e</mi>
<mrow>
<mo>-</mo>
<mi>M</mi>
</mrow>
</msup>
<mrow>
<mrow>
<msub>
<mi>Π</mi>
<mi>g</mi>
</msub>
<mo></mo>
<mrow>
<msup>
<mi>e</mi>
<mrow>
<mrow>
<mo>-</mo>
<msub>
<mi>f</mi>
<mi>g</mi>
</msub>
</mrow>
<mo></mo>
<mi>M</mi>
</mrow>
</msup>
<mo></mo>
<mrow>
<mo>(</mo>
<mrow>
<mn>1</mn>
<mo>-</mo>
<msub>
<mi>δ</mi>
<mi>g</mi>
</msub>
</mrow>
<mo>)</mo>
</mrow>
</mrow>
</mrow>
<mo>+</mo>
<msub>
<mi>δ</mi>
<mi>g</mi>
</msub>
</mrow>
</mfrac>
</mrow>
</mtd>
</mtr>
</mtable>
</math>
</maths>
</div>
</li> <li> <para-num num="[0995]"> </para-num> <div class="description-line" id="p-0978" num="0995">The new maximum likelihood function that accounts for the fitness parameters is</div>
</li> <li> <div class="description-line" id="p-0979" num="0000">
<maths id="MATH-US-00017" num="00017">
<math overflow="scroll">
<mrow>
<mrow>
<mi>log</mi>
<mo></mo>
<mrow>
<mo>[</mo>
<mrow>
<mi>L</mi>
<mo></mo>
<mrow>
<mo>(</mo>
<mrow>
<mi>M</mi>
<mo>,</mo>
<msub>
<mi>p</mi>
<mi>d</mi>
</msub>
<mo>,</mo>
<mi>δ</mi>
</mrow>
<mo>)</mo>
</mrow>
</mrow>
<mo>]</mo>
</mrow>
</mrow>
<mo>=</mo>
<mrow>
<mrow>
<msub>
<mi>N</mi>
<mn>0</mn>
</msub>
<mo></mo>
<mi>log</mi>
<mo></mo>
<mstyle>
<mspace height="0.3ex" width="0.3em"> </mspace>
</mstyle>
<mo></mo>
<mrow>
<mi>P</mi>
<mo></mo>
<mrow>
<mo>(</mo>
<mrow>
<mrow>
<mi>B</mi>
<mo>|</mo>
<mrow>
<msub>
<mo>∀</mo>
<mi>g</mi>
</msub>
<mo></mo>
<msub>
<mi>d</mi>
<mi>g</mi>
</msub>
</mrow>
</mrow>
<mo>=</mo>
<mn>0</mn>
</mrow>
<mo>)</mo>
</mrow>
</mrow>
</mrow>
<mo>+</mo>
<mover>
<mrow>
<munder>
<mo>∑</mo>
<mi>g</mi>
</munder>
<mo></mo>
<mrow>
<mo>{</mo>
<mrow>
<mrow>
<msub>
<mi>N</mi>
<mi>g</mi>
</msub>
<mo></mo>
<mrow>
<mi>log</mi>
<mo></mo>
<mrow>
<mo>[</mo>
<mrow>
<mi>P</mi>
<mo></mo>
<mrow>
<mo>(</mo>
<mrow>
<msub>
<mi>d</mi>
<mi>g</mi>
</msub>
<mo>=</mo>
<mn>1</mn>
</mrow>
<mo>)</mo>
</mrow>
</mrow>
<mo>]</mo>
</mrow>
</mrow>
</mrow>
<mo>+</mo>
<mrow>
<mrow>
<mo>(</mo>
<mrow>
<mi>N</mi>
<mo>-</mo>
<msub>
<mi>N</mi>
<mi>g</mi>
</msub>
</mrow>
<mo>)</mo>
</mrow>
<mo></mo>
<mrow>
<mi>log</mi>
<mo></mo>
<mrow>
<mo>[</mo>
<mrow>
<mi>P</mi>
<mo></mo>
<mrow>
<mo>(</mo>
<mrow>
<msub>
<mi>d</mi>
<mi>g</mi>
</msub>
<mo>=</mo>
<mn>0</mn>
</mrow>
<mo>)</mo>
</mrow>
</mrow>
<mo>]</mo>
</mrow>
</mrow>
</mrow>
</mrow>
<mo>}</mo>
</mrow>
</mrow>
<mover>
<mi></mi>
<mi>γ</mi>
</mover>
</mover>
</mrow>
</mrow>
</math>
</maths>
<maths id="MATH-US-00017-2" num="00017.2">
<math overflow="scroll">
<mrow>
<mi>γ</mi>
<mo>=</mo>
<mrow>
<mrow>
<munder>
<mo>∑</mo>
<mi>g</mi>
</munder>
<mo></mo>
<mrow>
<mo>{</mo>
<mrow>
<mrow>
<msub>
<mi>N</mi>
<mi>g</mi>
</msub>
<mo></mo>
<mi>log</mi>
<mo></mo>
<mstyle>
<mspace height="0.3ex" width="0.3em"> </mspace>
</mstyle>
<mo></mo>
<mfrac>
<mrow>
<msub>
<mi>δ</mi>
<mi>g</mi>
</msub>
<mo></mo>
<mrow>
<mo>(</mo>
<mrow>
<mn>1</mn>
<mo>-</mo>
<msup>
<mi>e</mi>
<mrow>
<mrow>
<mo>-</mo>
<msub>
<mi>f</mi>
<mi>g</mi>
</msub>
</mrow>
<mo></mo>
<msub>
<mi>Mp</mi>
<mi>d</mi>
</msub>
</mrow>
</msup>
</mrow>
<mo>)</mo>
</mrow>
</mrow>
<msub>
<mi>α</mi>
<mi>g</mi>
</msub>
</mfrac>
</mrow>
<mo>+</mo>
<mrow>
<mrow>
<mo>(</mo>
<mrow>
<mi>N</mi>
<mo>-</mo>
<msub>
<mi>N</mi>
<mi>g</mi>
</msub>
</mrow>
<mo>)</mo>
</mrow>
<mo></mo>
<mi>log</mi>
<mo></mo>
<mstyle>
<mspace height="0.3ex" width="0.3em"> </mspace>
</mstyle>
<mo></mo>
<mfrac>
<mrow>
<mrow>
<msup>
<mi>e</mi>
<mrow>
<mrow>
<mo>-</mo>
<msub>
<mi>f</mi>
<mi>g</mi>
</msub>
</mrow>
<mo></mo>
<mi>M</mi>
</mrow>
</msup>
<mo></mo>
<mrow>
<mo>(</mo>
<mrow>
<mn>1</mn>
<mo>-</mo>
<msub>
<mi>δ</mi>
<mi>g</mi>
</msub>
</mrow>
<mo>)</mo>
</mrow>
</mrow>
<mo>+</mo>
<mrow>
<msub>
<mi>δ</mi>
<mi>g</mi>
</msub>
<mo></mo>
<msup>
<mi>e</mi>
<mrow>
<mrow>
<mo>-</mo>
<msub>
<mi>f</mi>
<mi>g</mi>
</msub>
</mrow>
<mo></mo>
<msub>
<mi>Mp</mi>
<mi>d</mi>
</msub>
</mrow>
</msup>
</mrow>
</mrow>
<msub>
<mi>α</mi>
<mi>g</mi>
</msub>
</mfrac>
</mrow>
</mrow>
<mo>}</mo>
</mrow>
</mrow>
<mo>=</mo>
<mrow>
<mrow>
<munder>
<mo>∑</mo>
<mi>g</mi>
</munder>
<mo></mo>
<mrow>
<mo>{</mo>
<mrow>
<mrow>
<msub>
<mi>N</mi>
<mi>g</mi>
</msub>
<mo></mo>
<mrow>
<mi>log</mi>
<mo></mo>
<mrow>
<mo>[</mo>
<mrow>
<msub>
<mi>δ</mi>
<mi>g</mi>
</msub>
<mo></mo>
<mrow>
<mo>(</mo>
<mrow>
<mn>1</mn>
<mo>-</mo>
<msup>
<mi>e</mi>
<mrow>
<mrow>
<mo>-</mo>
<msub>
<mi>f</mi>
<mi>g</mi>
</msub>
</mrow>
<mo></mo>
<msub>
<mi>Mp</mi>
<mi>d</mi>
</msub>
</mrow>
</msup>
</mrow>
<mo>)</mo>
</mrow>
</mrow>
<mo>]</mo>
</mrow>
</mrow>
</mrow>
<mo>-</mo>
<mrow>
<msub>
<mi>N</mi>
<mi>g</mi>
</msub>
<mo></mo>
<mi>log</mi>
<mo></mo>
<mstyle>
<mspace height="0.3ex" width="0.3em"> </mspace>
</mstyle>
<mo></mo>
<msub>
<mi>α</mi>
<mi>g</mi>
</msub>
</mrow>
<mo>+</mo>
<mrow>
<mrow>
<mo>(</mo>
<mrow>
<mi>N</mi>
<mo>-</mo>
<msub>
<mi>N</mi>
<mi>g</mi>
</msub>
</mrow>
<mo>)</mo>
</mrow>
<mo></mo>
<mrow>
<mi>log</mi>
<mo></mo>
<mrow>
<mo>(</mo>
<mrow>
<mrow>
<msup>
<mi>e</mi>
<mrow>
<mrow>
<mo>-</mo>
<msub>
<mi>f</mi>
<mi>g</mi>
</msub>
</mrow>
<mo></mo>
<mi>M</mi>
</mrow>
</msup>
<mo></mo>
<mrow>
<mo>(</mo>
<mrow>
<mn>1</mn>
<mo>-</mo>
<msub>
<mi>δ</mi>
<mi>g</mi>
</msub>
</mrow>
<mo>)</mo>
</mrow>
</mrow>
<mo>+</mo>
<mrow>
<msub>
<mi>δ</mi>
<mi>g</mi>
</msub>
<mo></mo>
<msup>
<mi>e</mi>
<mrow>
<mrow>
<mo>-</mo>
<msub>
<mi>f</mi>
<mi>g</mi>
</msub>
</mrow>
<mo></mo>
<msub>
<mi>Mp</mi>
<mi>d</mi>
</msub>
</mrow>
</msup>
</mrow>
</mrow>
<mo>)</mo>
</mrow>
</mrow>
</mrow>
<mo>-</mo>
<mrow>
<mrow>
<mo>(</mo>
<mrow>
<mi>N</mi>
<mo>-</mo>
<msub>
<mi>N</mi>
<mi>g</mi>
</msub>
</mrow>
<mo>)</mo>
</mrow>
<mo></mo>
<mi>log</mi>
<mo></mo>
<mstyle>
<mspace height="0.3ex" width="0.3em"> </mspace>
</mstyle>
<mo></mo>
<msub>
<mi>α</mi>
<mi>g</mi>
</msub>
</mrow>
</mrow>
<mo>}</mo>
</mrow>
</mrow>
<mo>=</mo>
<mrow>
<munder>
<mo>∑</mo>
<mi>g</mi>
</munder>
<mo></mo>
<mrow>
<mo>{</mo>
<mrow>
<mrow>
<msub>
<mi>N</mi>
<mi>g</mi>
</msub>
<mo></mo>
<mrow>
<mi>log</mi>
<mo></mo>
<mrow>
<mo>(</mo>
<mrow>
<mn>1</mn>
<mo>-</mo>
<msup>
<mi>e</mi>
<mrow>
<mrow>
<mo>-</mo>
<msub>
<mi>f</mi>
<mi>g</mi>
</msub>
</mrow>
<mo></mo>
<msub>
<mi>Mp</mi>
<mi>d</mi>
</msub>
</mrow>
</msup>
</mrow>
<mo>)</mo>
</mrow>
</mrow>
</mrow>
<mo>+</mo>
<mrow>
<msub>
<mi>N</mi>
<mi>g</mi>
</msub>
<mo></mo>
<mi>log</mi>
<mo></mo>
<mstyle>
<mspace height="0.3ex" width="0.3em"> </mspace>
</mstyle>
<mo></mo>
<msub>
<mi>δ</mi>
<mi>g</mi>
</msub>
</mrow>
<mo>+</mo>
<mrow>
<mrow>
<mo>(</mo>
<mrow>
<mi>N</mi>
<mo>-</mo>
<msub>
<mi>N</mi>
<mi>g</mi>
</msub>
</mrow>
<mo>)</mo>
</mrow>
<mo></mo>
<mrow>
<mi>log</mi>
<mo></mo>
<mrow>
<mo>(</mo>
<mrow>
<mrow>
<msup>
<mi>e</mi>
<mrow>
<mrow>
<mo>-</mo>
<msub>
<mi>f</mi>
<mi>g</mi>
</msub>
</mrow>
<mo></mo>
<mi>M</mi>
</mrow>
</msup>
<mo></mo>
<mrow>
<mo>(</mo>
<mrow>
<mn>1</mn>
<mo>-</mo>
<msub>
<mi>δ</mi>
<mi>g</mi>
</msub>
</mrow>
<mo>)</mo>
</mrow>
</mrow>
<mo>+</mo>
<mrow>
<msub>
<mi>δ</mi>
<mi>g</mi>
</msub>
<mo></mo>
<msup>
<mi>e</mi>
<mrow>
<mrow>
<mo>-</mo>
<msub>
<mi>f</mi>
<mi>g</mi>
</msub>
</mrow>
<mo></mo>
<msub>
<mi>Mp</mi>
<mi>d</mi>
</msub>
</mrow>
</msup>
</mrow>
</mrow>
<mo>)</mo>
</mrow>
</mrow>
</mrow>
<mo>-</mo>
<mrow>
<mi>N</mi>
<mo></mo>
<mstyle>
<mspace height="0.3ex" width="0.3em"> </mspace>
</mstyle>
<mo></mo>
<mi>log</mi>
<mo></mo>
<mstyle>
<mspace height="0.3ex" width="0.3em"> </mspace>
</mstyle>
<mo></mo>
<msub>
<mi>α</mi>
<mi>g</mi>
</msub>
</mrow>
</mrow>
<mo>}</mo>
</mrow>
</mrow>
</mrow>
</mrow>
</mrow>
</math>
</maths>
<maths id="MATH-US-00017-3" num="00017.3">
<math overflow="scroll">
<mrow>
<mrow>
<mi>log</mi>
<mo></mo>
<mrow>
<mo>[</mo>
<mrow>
<mi>L</mi>
<mo></mo>
<mrow>
<mo>(</mo>
<mrow>
<mi>M</mi>
<mo>,</mo>
<msub>
<mi>p</mi>
<mi>d</mi>
</msub>
<mo>,</mo>
<mi>δ</mi>
</mrow>
<mo>)</mo>
</mrow>
</mrow>
<mo>]</mo>
</mrow>
</mrow>
<mo>=</mo>
<mrow>
<mrow>
<msub>
<mi>N</mi>
<mn>0</mn>
</msub>
<mo></mo>
<mrow>
<mi>log</mi>
<mo>(</mo>
<mrow>
<mn>1</mn>
<mo>-</mo>
<mfrac>
<mn>1</mn>
<mrow>
<msub>
<mi>Π</mi>
<mi>g</mi>
</msub>
<mo></mo>
<mrow>
<mo>(</mo>
<mrow>
<mn>1</mn>
<mo>-</mo>
<msub>
<mi>δ</mi>
<mi>g</mi>
</msub>
<mo>+</mo>
<mrow>
<msub>
<mi>δ</mi>
<mi>g</mi>
</msub>
<mo></mo>
<msup>
<mi>e</mi>
<mrow>
<msub>
<mi>f</mi>
<mi>g</mi>
</msub>
<mo></mo>
<mrow>
<mi>M</mi>
<mo></mo>
<mrow>
<mo>(</mo>
<mrow>
<mn>1</mn>
<mo>-</mo>
<msub>
<mi>p</mi>
<mi>d</mi>
</msub>
</mrow>
<mo>)</mo>
</mrow>
</mrow>
</mrow>
</msup>
</mrow>
</mrow>
<mo>)</mo>
</mrow>
</mrow>
</mfrac>
</mrow>
<mo>)</mo>
</mrow>
</mrow>
<mo>+</mo>
<mrow>
<munder>
<mo>∑</mo>
<mi>g</mi>
</munder>
<mo></mo>
<mrow>
<mo>{</mo>
<mrow>
<mrow>
<msub>
<mi>N</mi>
<mi>g</mi>
</msub>
<mo></mo>
<mrow>
<mi>log</mi>
<mo></mo>
<mrow>
<mo>(</mo>
<mrow>
<mn>1</mn>
<mo>-</mo>
<msup>
<mi>e</mi>
<mrow>
<mrow>
<mo>-</mo>
<msub>
<mi>f</mi>
<mi>g</mi>
</msub>
</mrow>
<mo></mo>
<msub>
<mi>Mp</mi>
<mi>d</mi>
</msub>
</mrow>
</msup>
</mrow>
<mo>)</mo>
</mrow>
</mrow>
</mrow>
<mo>+</mo>
<mrow>
<msub>
<mi>N</mi>
<mi>g</mi>
</msub>
<mo></mo>
<mi>log</mi>
<mo></mo>
<mstyle>
<mspace height="0.3ex" width="0.3em"> </mspace>
</mstyle>
<mo></mo>
<msub>
<mi>δ</mi>
<mi>g</mi>
</msub>
</mrow>
<mo>+</mo>
<mrow>
<mrow>
<mo>(</mo>
<mrow>
<mi>N</mi>
<mo>-</mo>
<msub>
<mi>N</mi>
<mi>g</mi>
</msub>
</mrow>
<mo>)</mo>
</mrow>
<mo></mo>
<mrow>
<mi>log</mi>
<mo></mo>
<mrow>
<mo>(</mo>
<mrow>
<mrow>
<msup>
<mi>e</mi>
<mrow>
<mrow>
<mo>-</mo>
<msub>
<mi>f</mi>
<mi>g</mi>
</msub>
</mrow>
<mo></mo>
<mi>M</mi>
</mrow>
</msup>
<mo></mo>
<mrow>
<mo>(</mo>
<mrow>
<mn>1</mn>
<mo>-</mo>
<msub>
<mi>δ</mi>
<mi>g</mi>
</msub>
</mrow>
<mo>)</mo>
</mrow>
</mrow>
<mo>+</mo>
<mrow>
<msub>
<mi>δ</mi>
<mi>g</mi>
</msub>
<mo></mo>
<msup>
<mi>e</mi>
<mrow>
<mrow>
<mo>-</mo>
<msub>
<mi>f</mi>
<mi>g</mi>
</msub>
</mrow>
<mo></mo>
<msub>
<mi>Mp</mi>
<mi>d</mi>
</msub>
</mrow>
</msup>
</mrow>
</mrow>
<mo>)</mo>
</mrow>
</mrow>
</mrow>
<mo>-</mo>
<mrow>
<mi>N</mi>
<mo></mo>
<mstyle>
<mspace height="0.3ex" width="0.3em"> </mspace>
</mstyle>
<mo></mo>
<mi>log</mi>
<mo></mo>
<mstyle>
<mspace height="0.3ex" width="0.3em"> </mspace>
</mstyle>
<mo></mo>
<msub>
<mi>α</mi>
<mi>g</mi>
</msub>
</mrow>
</mrow>
<mo>}</mo>
</mrow>
</mrow>
</mrow>
</mrow>
</math>
</maths>
</div>
</li> <li> <para-num num="[0996]"> </para-num> <div class="description-line" id="p-0980" num="0996">Analysis of perturbation effects on individual genes and gene modules. The most variable genes from each Perturb-Seq experiment were filtered by using the jackstraw approach (Chung and Storey, 2015) to identify the most significant genes (q-value &lt;0.01) in the top 20 PCs of the coefficient matrix. The genes were then clustered using k-means clustering by their coefficients. Optimal k was chosen by visual inspection of clustering results. Gene ontology (GO) enrichment analysis was performed on each cluster using goatools (Tang et al., 2015) with FDR threshold of 0.05.</div>
</li> <li> <para-num num="[0997]"> </para-num> <div class="description-line" id="p-0981" num="0997">Comparison to ChIP-seq binding profiles. Applicants analyzed assignments of TF binding in gene promoters in BMDCs following LPS stimulation across four time points (0 minutes, 30 minutes, 60 minutes, and 120 minutes) (Garber et al., 2012). To test for significant binding, two tests were used. First, the regulatory coefficients of bound genes were compared to those of unbound genes using a non-parametric Mann-Whitney test to identify significantly different means. Because of the possibility that this significance was driven by skewed covariates from unbound genes, Applicants also tested whether the coefficients of bound genes significantly deviated from 0, using the non-parametric one-sample Wilcox test. Finally, because TFs could both activate and repress genes, Applicants examined the number of bound genes significantly up- or down-regulated. To do this, Applicants used the distribution of covariates of unbound genes to define thresholds at the 5<sup>th </sup>percentile of lowest negative and highest positive coefficients. Any bound genes with coefficients that surpassed the thresholds were considered significant. Between the set of genes with significant positive and negative coefficients, Applicants used the larger set to infer whether the transcription factor was activating or inhibiting gene expression. Only BMDC expressed genes were considered in all ChIP-Seq analysis.</div>
</li> <li> <para-num num="[0998]"> </para-num> <div class="description-line" id="p-0982" num="0998">Power analysis and experimental design considerations. Power analysis was performed to determine how many cells are required to observe a signal as a function of observed effect size and baseline expression of a gene.</div>
</li> <li> <para-num num="[0999]"> </para-num> <div class="description-line" id="p-0983" num="0999">As an estimate of required read depth, Applicants downsampled at theUMI level. For example, for a vector of gene expression for a cell with the following values: [2,0,1,6] Applicants convert it into the following vector [1,1,3,4,4,4,4,4,4] on which downsampling is performed with equal probability without replacement. It is reconstructed into the original probability space by binning the observed integer counts. Applicants also downsampled cells are without replacement from our observed set.</div>
</li> <li> <para-num num="[1000]"> </para-num> <div class="description-line" id="p-0984" num="1000">Applicants performed our regression analysis on the downsampled expression matrix for various amounts of downsampling and recomputed resulting regulatory matrix. For each level of downsampling, 10 instances are averaged. Applicants compared between the original regulatory matrix and the matrix that results after downsampling using either a Pearson or Spearman correlation.</div>
</li> <li> <para-num num="[1001]"> </para-num> <div class="description-line" id="p-0985" num="1001">Plasmid design and construction The perturb-seq expression vector (pBA439) was derived from a previously described CRISPRi expression vector (herein referred to as the “original CRISPRi expression vector”) (Addgene plasmid #60955) (Gilbert et al. 2014). To construct pBA439, the mU6-sgRNA-EF1a-PURO-BFP region from the original CRISPRi vector and a BGH polyadenylation sequence amplified by PCR from pcDNA3.1(+) (Invitrogen, V790-20) were inserted in reverse origination between the Xbal and EcoRI sites of the original CRISPRi expression vector. A random 18 nucleotide barcode was then inserted between the BFP and BGH polyA sequences (using disrupted EcoRI and AvrII sites) by Gibson assembly to construct the perturb-seq expression library (pBA571). The perturb-seq library was prepared with an estimated barcode diversity of &gt;100,000 essentially as previously described (Kampmann, Bassik &amp; Weissman 2014). Guide RNA protospacer sequences were individually cloned into both the original CRISPRi expression vector and the pBA571 library (between the BstXI and BlpI sites) by ligation. Each vector was then verified by Sanger sequencing of the protospacer and, if applicable, its corresponding barcode. Final guide expression vectors containing barcodes that introduced the conserved polyadenylation signal AATAAA were discarded. To construct pMH0001, a minimal ubiquitous chromatin opening element (UCOE) (Müller-Kuller et al. 2015) was inserted upstream of the SFFV promoter in the lentiviral dCas9-BFP-KRAB expression vector (Gilbert et al. 2014).</div>
</li> <li> <para-num num="[1002]"> </para-num> <div class="description-line" id="p-0986" num="1002">The UPRE reporter was built into a backbone for lentiviral expression that has been previously described (Addgene plasmid #44012) (Meerbrey et al. 2011). This parental vector was digested with Agel and religated to remove unwanted functional cassettes, and the UPRE promoter region or EF1a promoter were inserted between the BamHI and Xhol site of the resulting product. The UPRE promoter region contains 5 UPR elements (UPREs, 5′-TGACGTGG-3′) upstream of the c-fos minimal promoter (−53 to +45 of the human c-fos promoter) (Wang et al. 2000). Lastly, mCherry and sfGFP were cloned adjacent to UPRE and EF1a promoters, respectively (into an Hpal site). The resulting vectors are pBA407 (UPRE-mCh-Ubc-Neo) and pBA409 (EF1a-sfGFP-Ubc-Neo). [001006] For testing of constant region variants in K562 cells, constant region variants fused to a GFP-targeting protospacer (EGFP-NT2, sequence GACCAGGATGGGCACCACCC) or a negative control protospacer were PCR-amplified and inserted into BstXI/XhoI-digested pBA439 (perturb-seq expression vector) by Gibson assembly. For testing of U6 promoters, U6 promoters from cow (bU6-2, GenBank DQ150531 and bU6-3, GenBank DQ150532 (Lambeth et al. 2006)), sheep (sU6-1, GenBank HM641427 and sU6-2, GenBank HM641426 (Hu et al. 2011)), buffalo (buU6, GenBank JN417659 (Zhang et al. 2014)), and pig (pU6, GenBank EU520423 (Chuang et al. 2009)) spanning ˜400-500 bp upstream of the TSS, modified to contain a BstXI site at the TSS, and fused to EGFP-NT2 and the original constant region were obtained as synthetic DNA segments (Integrated DNA technologies) and inserted into Hpal/Xhol-digested pBA439 (perturb-seq expression vector) by Gibson assembly.</div>
</li> <li> <para-num num="[1003]"> </para-num> <div class="description-line" id="p-0987" num="1003">Three-guide vectors were assembled by a two-step cloning procedure (<figref idrefs="DRAWINGS">FIG. 88F</figref>). First, guide RNAs to be included were cloned into the corresponding single guide RNA expression vectors. Briefly, complementary oligonucleotides (Integrated DNA Technologies) containing the protospacer sequence and ligation overhangs were annealed and ligated into BstXI/B1pI-digested guide RNA expression vectors containing specific primer binding sites flanking the guide RNA expression cassette. The three-guide RNA expression cassettes were then PCR-amplified and assembled into Hpal/Xhol-digested pBA571 (perturb-seq expression library) by a single four-piece Gibson assembly step. Vectors were validated and barcodes were determined as described above. For three-guide vectors targeting the UPR branches, the bU6, mU6, and hU6 cassettes were designed to either express an sgRNA targeting ATF6, EIF2AK3 (PERK), or ERN1 (IRE1α), respectively, or a non-targeting negative control sgRNA. The following protospacer sequences were used: ATF6-targeting, gGGGATCTGAGAATGTACCA; EIF2AK3-targeting, gCGGGCTGAGACGTGGCCAG; ERN1/-targeting, gAGAACTGACTAGGCAGCGG; non-targeting sgRNA in bU6 cassette, gACGACTAGTTAGGCGTGTA; non-targeting sgRNA in mU6 cassette, gGCCAAACGTGCCCTGACGG; non-targeting sgRNA in hU6 cassette, gCCTTGGCTAAACCGCTCCC.</div>
</li> <li> <para-num num="[1004]"> </para-num> <div class="description-line" id="p-0988" num="1004">Cell culture, DNA transfections, viral production, and construction of reporter cell lines K562 cells were grown in RPMI-1640 with 25mM HEPES, 2.0 g/L NaHCO3, 0.3 g/L L-Glutamine supplemented with 10% FBS, 2 mM glutamine, 100 units/mL penicillin and 100 μg/mL streptomycin. HEK293T cells were grown in Dulbecco's modified eagle medium (DMEM) in 10% FBS, 100 units/mL penicillin and 100 μg/mL streptomycin. Cells were treated with tunicamycin or thapsigargin (Sigma, T9033) solubilized in DMSO. Lentivirus was produced by transfecting HEK293T with standard packaging vectors using TranslT®-LTI Transfection Reagent (Mirus, MIR 2306). Viral supernatant was harvested at least 2 days after transfection and filtered through a PVDF syringe filter and/or frozen prior to infection.</div>
</li> <li> <para-num num="[1005]"> </para-num> <div class="description-line" id="p-0989" num="1005">To construct the UPRE reporter cell line, K562 cells stably expressing dCas9-KRAB (Gilbert et al. 2014), originally constructed from K562 cells obtained from ATCC 536 (RRID:CVCL_0004), were stably transduced with pBA407 and selected in media supplemented with 500 μg/mL Geneticin (Gibco, 10131-035). The clonal line cBA010 was then selected by limiting dilution. cBA011 is a derivative of cBA010 containing pBA409. cBA011 was made by stable transduction and selection of GFP positive cells using fluorescence activated cell sorting on a BD FACSAria2. The GFP reporter cell line was constructed by infecting K562 cells stably expressing dCas9-KRAB with a Murine Stem Cell Virus (MSCV) retrovirus expressing GFP from the SV40 promoter. MSCV retrovirus was produced by transfecting amphotropic Phoenix packaging cell lines with standard packaging vectors. K562 cells stably expressing GFP were sorted to purity by flow cytometry using a BD FACS Aria2. To construct the GFP+ K562 UCOE-dCas9-KRAB cell line, the GFP reporter cell line was transduced with pMH0001 at a multiplicity of infection of ˜3. Transduced cells were sorted for BFP expression (top 33%) by flow cytometry on a BD FACS Aria2. BFP fluorescence was monitored for several generations and found to be stable.</div>
</li> <li> <para-num num="[1006]"> </para-num> <div class="description-line" id="p-0990" num="1006">Design and cloning of constant region variants for testing in <i>E. coli. </i>Constant region bases to mutate were identified by inspection of the crystal structure of Cas9 bound to guide RNA and target DNA (<figref idrefs="DRAWINGS">FIG. 88D</figref>, PDB ID codes 4OO8, 4UN3, 4ZTO (Nishimasu et al. 2014, Anders et al. 2014, Jiang et al. 2015)). Bases that did not form direct contacts with Cas9 or with other nucleotides of the constant region were deemed amenable for mutation. If applicable, sequence conservation patterns of the base in crRNAs/tracrRNAs of <i>Streptococcus </i>species and previous reports of constant regions carrying changes at the base (Briner et al. 2014, Dang et al. 2015) were used to determine the type of mutation. In this fashion, 15 constant region variants with mutations in different parts of the constant region were designed (<figref idrefs="DRAWINGS">FIG. 82D</figref>). The most diverse constant region variants cr2 and cr3 were designed by combining multiple individual mutations (<figref idrefs="DRAWINGS">FIG. 82D</figref>).</div>
</li> <li> <para-num num="[1007]"> </para-num> <div class="description-line" id="p-0991" num="1007">To rapidly assess the activity of the variant constant regions, the variants were fused to a mRFP-targeting guide RNA (mRFP-NT1, sequence AACTTTCAGTTTAGCGGTCT) (Qi et al. 2013) and tested in an <i>E. coli </i>CRISPRi reporter strain for knockdown of mRFP (see below). To eliminate variability from copy number variation, guide RNAs were cloned into a plasmid for site-specific integration into the <i>E. coli </i>genome at attL and expressed from single copy from an IPTG-inducible P<sub>LlacO-1 </sub>promoter (Lutz, Bujard 1997). To construct the integrating guide RNA expression plasmid, a guide RNA expression cassette was first PCR-amplified from pgRNA-bacteria Addgene plasmid #44251) (Qi et al. 2013), modified to be flanked by the strong synthetic terminators L3S3P22 and L3S2P21 (Chen et al. 2013) and inserted into pCAH63 (Haldimann, Wanner 2001) at the ClaI/Nhel sites. The constitutive promoter from pgRNA-bacteria was replaced with the IPTG-inducible P<sub>LlacO-1 </sub>promoter, generating pCs-550r. Then, pCs-550r was further modified to include the constant region used in mammalian CRISPRi (Gilbert et al. 2014), PCR-amplified with an mRFP-targeting protospacer and inserted into pCs-550r at the Spel and Kpnl sites to generate pMJ020. Finally, constant region variants 1-15 as well as cr2 and cr3 were cloned into pMJ020 by inverse PCR with mutations encoded in primer overhangs, by site-directed mutagenesis following standard procedures, or by insertion of a synthetic DNA segment encoding the constant region (Integrated DNA Technologies) into Spel/Kpnl-digested pMJ020 by Gibson assembly.</div>
</li> <li> <para-num num="[1008]"> </para-num> <div class="description-line" id="p-0992" num="1008">Construction of <i>E. coli </i>CRISPRi reporter strain and testing of constant region variants The <i>E. coli </i>CRISPRi reporter strain was constructed by sequential insertion of a construct for IPTG-inducible expression of dCas9, a construct for constitutive expression of mRFP, and a construct for IPTG-inducible guide RNA expression (described above) into the <i>E. coli </i>genome. First, a laclq-t0 P<sub>LlacO-1</sub>dCas9 cassette (laclq for strong expression of the Lac repressor; t0, a transcription terminator; P<sub>LlacO-1</sub>-dCas9; for IPTG-inducible expression of <i>S. pyogenes </i>D10A/H840A Cas9 (dCas9)) was inserted into the chromosome of <i>E. coli </i>BW25113 at +19 attL via of lambda Red recombinase-mediated recombineering (Thomason et al. 2014). Then, a nfsA::mRFP-kan cassette for expression of mRFP from the J23119 promoter, a strong synthetic constitutive promoter from the Anderson promoter collection (http://parts.igem.org/Promoters/Catalog/Anderson), was inserted into an <i>E. coli </i>MG1655-derived strain by lambda Red recombinase-mediated recombineering as described previously (Qi et al. 2013), and moved from the MG1655-derived strain into the dCas9-expressing BW25113 strain by P1 transduction and selection on kanamycin following a published protocol (Thomason, Costantino &amp; Court 2007). Plasmids for expression of mRFP-NT1 with the different constant region variants were integrated into the dCas9- and mRFP-expressing strain at attL using the helper plasmid pINT-ts (Haldimann, Wanner 2001), selecting for chloramphenicol resistance.</div>
</li> <li> <para-num num="[1009]"> </para-num> <div class="description-line" id="p-0993" num="1009">Single colonies of strains with the integrated guide RNA expression plasmids were inoculated into LB and grown overnight in deep 96-well blocks at 37 ° C. with shaking at 900 rpm. Stationary-phase cultures were back-diluted 1:30 and grown into mid-exponential phase, at which point they were back-diluted 1:10000 into LB with 1 mM IPTG for induction of guide RNA and dCas9 expression. Induced cultures were grown at 37 ° C. with shaking until OD<sub>600 nm </sub>reached ˜0.4-0.7 (approximately 5 hrs), at which point they were diluted 1:30 in PBS in a 96-well plate. RFP fluorescence was recorded on a LSR-II flow cytometer (BD Biosciences) equipped with a 96-well high-throughput sampler. Each experiment was carried out using three individual colonies for each constant region variant. RFP levels were normalized to those of a strain expressing a non-targeting guide RNA.</div>
</li> <li> <para-num num="[1010]"> </para-num> <div class="description-line" id="p-0994" num="1010">Testing of single- and three-guide vectors in K562 cells by GFP knockdown Vectors for expression of EGFP-NT2 in different contexts were delivered into GFP+K562 cells with dCas9-KRAB or with UCOE-dCas9-KRAB by lentiviral transduction at MOI of 0.1-0.5. For all experiments using GFP+K562 cells with UCOE-dCas9-KRAB, transduced cells were allowed to recover for 2 d, then selected to purity using 2 μg/mL puromycin for 3 d, and allowed to recover for another 2 d before GFP levels were recorded by flow cytometry on a LSR-II flow cytometer (BD Biosciences). For experiments involving only GFP+ K562 cells with dCas9-KRAB, cells were grown out for 7-9 d after transduction and GFP levels were recorded by flow cytometry, using BFP expression to gate for transduced cells. Flow cytometry data were analyzed using FlowCytometryTools v0.4.5 (http://eyurtsev.github.io/FlowCytometryTools/). For plotting, flow cytometry events were normalized to population size and the histograms were smoothened by kernel density estimation. For estimating knockdowns, GFP levels of wild-type (GFP—) K562 cells were subtracted. [001015] Perturb-seq screening Viruses were individually packaged and harvested in preparation for perturb-seq screening. Individual packaging of the lentivirus and pooling at the step of virus or cells was done to avoid intermolecular recombination of proviral genomes and to ensure maintenance of paired barcode-sgRNA coupling (Sack et al. 2016). For the pilot experiment represented in <figref idrefs="DRAWINGS">FIG. 81C</figref>, 1D, 1F, cBA010 cells were individually spinfected with virus (at 33° C. for 2 hours at 1000 xg) in media supplemented with 8 μg/mL polybrene; 5 hours post spinfection, virus was removed by centrifugation and cells were resuspended in fresh media. Three days later, a transduction efficiency of 20-30%, as determined by percentage of BFP positive (BFP+) cells, was measured by flow cytometry and cells were pooled with equal numbers of guide RNA-containing (BFP+) cells. Control cells were included in the pool at 3-fold coverage. Pooled cells were then grown in the presence of puromycin (3 μg/mL) for 5 additional days. Seven days post transduction cells were sorted on a BD FACSAria2 to near purity and eight days post transduction the sorted cells were separated into droplet emulsion using the Chromium™ Single Cell 3′ Solution according to manufacturer's instructions (10X Genomics).</div>
</li> <li> <para-num num="[1011]"> </para-num> <div class="description-line" id="p-0995" num="1011">For the perturb-seq epistasis experiment, seven three-guide vectors targeting every possible combination of ATF6, IRE1, and PERK as well as two independent three-guide vectors with three negative control guide RNAs and different barcodes were individually packaged into lentiviruses. Freshly produced (i.e. not frozen) lentiviruses were spinfected into cBA007 cells (at 33° C. for 2 h at 1000×g) in media supplemented with 8 μg/mL polybrene. The virus was removed by centrifugation and cells were resuspended in fresh media. Three days after infection, transduction efficiencies of 5-10% were measured by flow cytometry. Cells were combined into a pool with equal numbers of transduced (BFP+) cells for each vector (resulting in 2-fold excess of negative control vectors) and the combined cells were then sorted on a BD FACS Aria2 to near purity. To limit heterogenous effects of cell microenvironments caused by cell settling, the sorted cells were grown with continuous agitation on an orbital shaker. Five days after infection, the pooled and sorted cells were split into three populations, which were treated as follows: 1) DMSO control treatment for 6 hr; 2) treatment with 4 μg/mL tunicamycin for 6 hr; and 3) treatment with 100 nM thapsigargin for 4 hr. At the end of the treatment, the cells were separated into droplet emulsion using the Chromium™ Single Cell 3′ Solution according to manufacturer's instructions (10X Genomics). Cells loaded onto the device were 90.4%, 87.9%, and 85.3% viable for the different treatment conditions, respectively.</div>
</li> <li> <para-num num="[1012]"> </para-num> <div class="description-line" id="p-0996" num="1012">For the large-scale perturb-seq screen of UPR-inducing guide RNAs, viruses were individually titered by test infections into cBA011 cells and then pooled evenly. To account for varied effects on cell viability across the guide RNA sub-library and minimize cell number difference, pooling titers were determined by the percentage of BFP+ cells remaining 6 days post transduction. Two negative control guides were included, NegCtrl-2 and NegCtrl-3. NegCtrl-2 and select guides (those encoded by pDS002, pDS017, pDS026, pDS032, pDS033, pDS044, pDS052, pDS088, pDS091, pDS160, pDS186) were included at higher representation within the lentivirus pool, 8-fold and 2-fold, respectively. The lentivirus library pool was then used to infect cBA010 cells (performed by spinfection at 33° C. for 3 hours at 1000xg) so that a single pooled cell population with all perturbations would be carried though subsequent steps. Post centrifugation, cells were immediately removed from virus and transferred to a spinner flask for growth in fresh media. Three days later, a transduction efficiency of 15% was measured by flow cytometry and BFP+ cells were sorted to near purity on a BD FACSAria2. To limit heterogenous effects of cell microenvironments caused by cell settling, the sorted cells were grown with continuous agitation on an orbital shaker. Approximately seven days post transduction cells were separated into droplet emulsion using the Chromium™ Single Cell 3′ Solution across two separate runs totaling 10 lanes on the device according to manufacturer's instructions (10X Genomics). Cells loaded onto the device were 92% BFP+and 93-94% viable, as determined by flow cytometry. [001018] For all perturb-seq experiments single-cell RNA-seq libraries were prepared according to the Single Cell 3′ Reagent Kits User Guide (10X Genomics). However, this protocol produces libraries that are not compatible with the HiSeq 4000 due to the presence of some sort of toxic byproducts that it is uniquely sensitive to. To remove this issue, Applicants implemented a short cleanup protocol taking place after library preparation. 120-200 ng of library material was split into parallel PCR reactions containing 0.3 μM each of the Illumina P5 and P7 primers, and amplified using Kapa HiFi ReadyMix according to the following protocol: (1) 95° C. 80 sec (2) 98° C. 20 sec/65° C. 30 sec/72° C. 20 sec for 6 cycles (3) 72° C. 1 min. PCR products were then SPRI-purified at 1X rati, repooled during elution, and then fragments of length 350-525 bp were selected using the BluePippin (Sage Science). [001019] Genome-scale CRISPRi screening Reporter screens were conducted using protocols similar to those previously described (Gilbert et al. 2014, Sidrauski et al. 2015). The hCRISPRi-v1 (Gilbert et al. 2014) or the compact (5 sgRNA/gene) hCRISPRi-v2 (Horlbeck et al. 2016) sgRNA libraries were transduced into cBA011 cells at an MOI&lt;1 (percent BFP+ cells was ˜45% and 26%, respectively). For the hCRISPRi-v1 screen, cells were grown in spinner flasks for 2 days without selection, followed by 3 days of selection with 1 μg/mL puromycin. Screen replicates were split post infection and carried separately throughout the remainder of the experiment. One replicate arm of the hCRISPRi-v1 screen was carried with media supplemented with 88-150 nM ISRIB throughout, although differences observed between the replicates at the level of both sgRNAs and genes (Table S3) were negligible. For the hCRISPRi-v2 screen, cells were grown in spinner flasks for 2 days without selection, followed by 5 days of selection with 1-3 μg/mL puromycin. Screen replicates were split into separate spinner flasks on day 3. For both screens, cells were separated into those with the highest (˜28-33%) and lowest (˜30-35%) mCherry/GFP ratio eight days post transduction by fluorescence-activated cell sorting. Cell pellets were frozen after collection. Approximately 23-30 million cells were collected per bin during screening of the hCRISPRi-v1 library (a representation of ˜450) and 19-22 million cells per bin for hCRISPRi-v2 (a representation of ˜600). Genomic DNA was isolated from frozen cells essentially as described previously (hCRISPRi-v2, 18239126) and the sgRNA-encoded regions were enriched, amplified, and prepared for sequencing (Gilbert et al. 2014).</div>
</li> <li> <para-num num="[1013]"> </para-num> <div class="description-line" id="p-0997" num="1013">Sequenced protospacer sequences were aligned and data were processed as described (Gilbert et al. 2014, Horlbeck et al. 2016) with custom Python scripts (available at https://github.com/mhorlbeck/ScreenProcessing). Reporter phenotypes (referred to as Reporter signal) for library sgRNAs were calculated as the log 2 enrichment of sgRNA sequences identified within the high mCherry/GFP cells over the low mCherry/GFP cells. Phenotypes for each transcription start site were then calculated as the average reporter phenotype of the 3 sgRNAs with the strongest phenotype by absolute value (most active sgRNAs). Mann-Whitney test p-values were calculated by comparing all sgRNAs targeting a given TSS to the full set of negative control sgRNAs. For data presented in <figref idrefs="DRAWINGS">FIGS. 84D-F</figref> and <b>90</b>B,C, genes with multiple targeted TSSs were collapsed such that only the TSS with the lowest p-value was used. Screen hits were defined as those genes (or separately those TSSs) with a discriminate score, defined as the absolute value of a calculated reporter phenotype over the standard deviation of all evaluated phenotypes multiplied by the log 10 of the Mann-Whitney p-value for given candidate, greater than 7. Growth screen data in <figref idrefs="DRAWINGS">FIG. 90C</figref> has been reported elsewhere, and the unreported screen was conducted in parallel essential as described (Horlbeck et al. 2016). Gene ontology analysis was conducting using select databases (GOTERM_BP FAT, GOTERM_CC_FAT, GOTERM_NIF FAT, KEGG_PATHWAY) and hits (calculated from all TSSs) with a phenotype of greater than 1 using DAVID Bioinformatic Resources 6.8 Beta (https://david-d.ncifcrf.gov/) with (Huang, Sherman &amp; Lempicki 2009a, Huang, Sherman &amp; Lempicki 2009b). Biological classifications reported in <figref idrefs="DRAWINGS">FIG. 84F-G</figref> were manually assembled from the literature and using resources from the HUGO Gene Nomenclature Committee (www.genenames.org), AmiGO, the GO Consortium's annotation and ontology toolkit (Carbon et al. 2009) (http://amigo.geneontology.org), DAVID Bioinformatic Resources (https://david.ncifcrf.gov) (Huang, Sherman &amp; Lempicki 2009a, Huang, Sherman &amp; Lempicki 2009b).</div>
</li> <li> <para-num num="[1014]"> </para-num> <div class="description-line" id="p-0998" num="1014">Individual re-test of sgRNA reporter phenotypes Viruses were individually packaged and harvested as described above. UPRE reporter-containing K562 cells (cBA011) cells were infected with thawed virus. Additionally, parental K562 dCas9-KRAB cells (Gilbert et al. 2014) were transduced with negative controls. Flow cytometer readings of the mCherry UPRE signal and GFP EF1a signal were taken periodically and 8 days post transduction. Median fluorescence signals were analyzed by subtracting an average background signal from control-transduced K562 dCas9-KRAB cells and normalizing the mCherry, GFP, or mCherry/GFP measurement from guide-containing cells (as determined by BFP fluorescence) in each well to untransduced cells. Data from wells with fewer than 500 transduced or untransduced cells or with lower than expected BFP signal (3 standard deviations below the mean of BFP median from all other experimental wells) were systematically discarded from further analysis. For experiments where a flow cytometer reading was taken on the second day post transduction, data was also filtered for a minimum day 2 viability percentage. [001022] RT-qPCR and semi-quantitative PCR for XBP1 mRNA splicing Cells were harvested and total RNA was isolated using TRIzol® Reagent (ThermoFisher Scientific, 15596-018) and Phase Lock Gel tubes (VWR, 10052-170) or NucleoSpin® RNA (Macherey-Nagel, 740955.50) essentially according to manufacturers' instructions. RNA prepared by TRIzol® extraction was treated with TURBO™ DNase (ThermoFisher Scientific). RNA was converted to cDNA using SuperScript® II or SuperScript® III Reverse Transcriptase (ThermoFisher Scientific) under standard conditions with oligo(dT) primers or random hexamers with or without RNaseOUT™ Recombinant Ribonuclease Inhibitor (ThermoFisher Scientific). Quantitative PCR reactions were prepared with 1X master mix containing 1x Colorless GoTaq® Reaction Buffer (Promega, M792A), MgC12 (0.7 mM), dNTPs (0.2 mM each), primers (0.75 μM each), and 1000X SYBR Green with GoTaq® DNA polymerase (Promega, M830B) in 22 μL reactions. Reactions were run on a LightCycler® 480 Instrument (Roche). Semi-quantitative XBP1-specific PCR reactions were prepared with 2μL of cDNA diluted 1:10 using a master mix containing 0.9X Colorless GoTaq® Reaction Buffer (Promega, M792A), dNTPs (0.23 mM each), primers (0.45 μM each) with GoTaq® DNA polymerase (Promega, M830B) in 22.1 μL reactions. These reactions were run on a standard thermocycler program with 30 second at 60.5° C. for annealing and 28 cycles. PCR products were visualized on 8% TBE gels.</div>
</li> <li> <para-num num="[1015]"> </para-num> <div class="description-line" id="p-0999" num="1015">Quantification and Statistical Analysis. Applicants will first provide an overview of the methods used, and then describe their specific application to each figure.</div>
</li> <li> <para-num num="[1016]"> </para-num> <div class="description-line" id="p-1000" num="1016">Pipeline overview All analysis was performed in Python, using a combination of Numpy, Pandas, scikit-learn, and a custom-made perturb-seq library. The general outline is presented in <figref idrefs="DRAWINGS">FIG. 89A</figref>, and Applicants will outline the steps below.</div>
</li> <li> <para-num num="[1017]"> </para-num> <div class="description-line" id="p-1001" num="1017">Sequencing: Reads from 10X single-cell RNAseq experiments were aligned and collapsed to unique molecular identifier (UMI) counts using 10X's cellranger software (version 1.1). The result is a large digital expression matrix with cell barcodes as rows and gene identities as columns.</div>
</li> <li> <para-num num="[1018]"> </para-num> <div class="description-line" id="p-1002" num="1018">Perturbation identity mapping Specifically amplified guide barcode libraries were described as above and either sequenced as spike-ins or independently. The specific amplification strategy Applicants used preserved the 3′ end of the transcript (and thus the cell barcode and UMI of a given captured molecule) and introduced an Illumina read 1 primer upstream of the guide barcode sequence. These reads were aligned using bowtie (flags: -v2 -q -m1) to a library of expected barcode sequences. Applicants then collapsed all reads with common cell barcode, UMI, and read identity (as some reads were not mapped by bowtie due to low quality scores) to produce a table consisting of possible guide identities for each cell, and the number of molecules attributing a given guide identity to that cell. Applicants defined the coverage of a given proposed identity as the number of reads divided by the number of UMIs, and defined a proposed identity as having good coverage if it: (1) had a coverage level at or above the mean coverage level minus two times the standard deviation in coverage (2) had at least 50 raw reads and (3) had at least 3 UMIs. Any cell that had only a single identity that met these criteria was assigned that perturbation identity. Any cell that had two or more identities meeting these criteria was assigned as a multiple (either a multiple infection, or a multiple encapsulation during emulsion generation). Any cell that had no identities meeting these criteria was assigned as unidentifiable.</div>
</li> <li> <para-num num="[1019]"> </para-num> <div class="description-line" id="p-1003" num="1019">Expression normalization To normalize for differences in sequencing capture and coverage across emulsion droplets, Applicants rescaled all cells to have the median number of total UMIs (i.e. each row of the raw digital expression matrix is normalized to the same sum). Expression of each gene was then z-normalized with respect to the mean and standard deviation of that gene in the control (unperturbed) population:</div>
</li> <li> <div class="description-line" id="p-1004" num="0000">
<maths id="MATH-US-00018" num="00018">
<math overflow="scroll">
<mrow>
<msub>
<mi>x</mi>
<mi>normlized</mi>
</msub>
<mo>=</mo>
<mfrac>
<mrow>
<mi>x</mi>
<mo>-</mo>
<msub>
<mi>μ</mi>
<mi>control</mi>
</msub>
</mrow>
<msub>
<mi>σ</mi>
<mi>control</mi>
</msub>
</mfrac>
</mrow>
</math>
</maths>
</div>
</li> <li> <para-num num="[1020]"> </para-num> <div class="description-line" id="p-1005" num="1020">This normalization means that control cells always have mean normalized expression of 0 for all genes and standard deviation 1, so that the units of expression are “standard deviations above/below the control distribution.”</div>
</li> <li> <para-num num="[1021]"> </para-num> <div class="description-line" id="p-1006" num="1021">In the epistasis experiment, the control population was the DMSO-treated cells. In the perturb-seq experiment, they were the cells containing the NegCtrl-2 guide. In the perturb-seq experiment, the mixed population was run in ten separate pools that were treated independently during library preparation (corresponding to lanes on the 10X Chromium instrument and on the Illumina sequencer). To avoid any lane-dependent batch effects, cells were normalized to the control cell distribution within the same lane. [001030] Low cell count/inviable cell removal While developing the low rank ICA method described below, Applicants observed that all experiments always contained two subpopulations that were peculiar in that they contained roughly equal membership from all perturbations. Further investigation showed that these were a group of cells with systematically lower total UMI counts (visible as a small second mode in the distribution of total UMIs per cell) and a group of cells that contained markers of activation of apoptotic programs. Applicants attributed the first population to partly failed RNAseq library preparation occurring in a small number of emulsion droplets, and the second to inviable cells (which Applicants knew were present in the cells placed used in the 10X experiment). Neither population composed more than a few percent of the total number of cells. Though low rank ICA always isolated these in an unbiased way, Applicants generally excluded them from analysis. The low UMI count cells were simply removed using a threshold. To remove the apoptotic cells, Applicants trained a random forest regressor (described in more detail below in the section on UPR branch activation scoring below) to recognize them using the cells in our epistasis experiment as training data.</div>
</li> <li> <para-num num="[1022]"> </para-num> <div class="description-line" id="p-1007" num="1022">Identification of differentially expressed genes The end result of the previous steps is a normalized gene expression matrix where each cell has been assigned a perturbation identity. In general, Applicants were interested in analyzing differences between populations, and used two distinct strategies for isolating interesting genes.</div>
</li> <li> <para-num num="[1023]"> </para-num> <div class="description-line" id="p-1008" num="1023">Kolmogorov-Smirnov test/metric: The Kolmogorov-Smirnov test is a nonparametric test for equality of probability distributions based on a metric defined on their cumulative distribution functions. Specifically, if F<sub>perturbed </sub>and F<sub>control </sub>are the CDFs for a given gene in the perturbed and control distribution, the test statistic is</div>
</li> <li> <div class="description-line" id="p-1009" num="0000">
<maths id="MATH-US-00019" num="00019">
<math overflow="scroll">
<mrow>
<mi>D</mi>
<mo>=</mo>
<mrow>
<munder>
<mi>sup</mi>
<mi>x</mi>
</munder>
<mo></mo>
<mrow>
<mo></mo>
<mrow>
<mrow>
<msub>
<mi>F</mi>
<mi>perturbed</mi>
</msub>
<mo></mo>
<mrow>
<mo>(</mo>
<mi>x</mi>
<mo>)</mo>
</mrow>
</mrow>
<mo>-</mo>
<mrow>
<msub>
<mi>F</mi>
<mi>control</mi>
</msub>
<mo></mo>
<mrow>
<mo>(</mo>
<mi>x</mi>
<mo>)</mo>
</mrow>
</mrow>
</mrow>
<mo></mo>
</mrow>
</mrow>
</mrow>
</math>
</maths>
</div>
</li> <li> <para-num num="[1024]"> </para-num> <div class="description-line" id="p-1010" num="1024">This can be assigned a p-value in a standard way. However, the large scale of single-cell data means that many genes were often significantly perturbed without being interestingly perturbed, simply because of small differences detected by great sampling depth. Thus in some cases Applicants placed a direct threshold on the test statistic D itself, which ensured that changes were both significant (in the statistical sense) and also of reasonable magnitude, as it is valid metric on the space of CDFs.</div>
</li> <li> <para-num num="[1025]"> </para-num> <div class="description-line" id="p-1011" num="1025">Random forest classifier An advantage of perturb-seq is that the cell populations are known, which means that supervised learning methods can be brought to bear. Our strategy here was motivated by the idea that a gene is likely important for a given perturbation if its expression level can be used to accurately predict that perturbation's identity. This idea is particularly useful when many perturbations are being compared, as what you want then are the genes that best distinguish all of the perturbations from each other. [001035] To leverage this idea, Applicants used random forest classifiers. Given a set of perturbations, Applicants would train a random forest classifier to predict perturbation identity using a subset of genes. Specifically, Applicants used the implementation of extremely randomized trees implemented in scikit-learn, generally with 1000 trees in the forest. Applicants performed a two-stage fitting process for a given number of desired features N<sub>genes</sub>. First, Applicants set aside 20% of the cells. The remaining 80% were used to train a random forest classifier (usually with 1000 estimators) to predict the perturbation identity using the normalized expression profile for each cell (with some threshold on gene expression level) as the set of features. The random forest assigns importances to features during training based on their predictive value, and Applicants would then take the top N<sub>genes </sub>sorted by importance as the set of most informative genes. To evaluate how informative these genes were, Applicants would then retrain the classifier using only these genes, and predict the perturbation present in the 20% of cells Applicants had initially set aside. For sets of perturbations with large differences, Applicants routinely saw accuracies of 80-90%. The genes chosen by the random forest essentially always showed marked differences by the Kolmogorov-Smirnov approach outlined above, and the forests had the advantages that they scaled to an arbitrary number of perturbations, and that the selected genes were known to vary informatively across perturbations instead of simply having a difference in distribution.</div>
</li> <li> <para-num num="[1026]"> </para-num> <div class="description-line" id="p-1012" num="1026">Low rank ICA Single-cell data are intrinsically very noisy, either due to real biological variation or problems in capture efficiency. As described in the main text, these effects can affect the sensitivity of methods like principal components analysis, which is intrinsically variance-maximizing and hence very sensitive to outliers. To isolate larger trends within the data, Applicants developed a simple two-step approach called low rank ICA. The first step consists of isolating a low rank approximation of the dynamics within the experiment. To do this, Applicants used Robust PCA (Candès et al. 2011), which seeks a decomposition of the form</div>
</li> <li> <div class="description-line" id="p-1013" num="0000">
<br/>
<i>X=L+S </i>
</div>
</li> <li> <para-num num="[1027]"> </para-num> <div class="description-line" id="p-1014" num="1027">where X is the normalized expression matrix, L is a low rank matrix, and S is a sparse matrix (most entries are zero). Specifically, Robust PCA solves the optimization problem</div>
</li> <li> <div class="description-line" id="p-1015" num="0000">
<maths id="MATH-US-00020" num="00020">
<math overflow="scroll">
<mrow>
<mrow>
<mrow>
<munder>
<mi>min</mi>
<mrow>
<mi>L</mi>
<mo>,</mo>
<mi>S</mi>
</mrow>
</munder>
<mo></mo>
<msub>
<mrow>
<mo></mo>
<mi>L</mi>
<mo></mo>
</mrow>
<mo>*</mo>
</msub>
</mrow>
<mo>+</mo>
<mrow>
<mi>λ</mi>
<mo></mo>
<msub>
<mrow>
<mo></mo>
<mi>S</mi>
<mo></mo>
</mrow>
<mn>1</mn>
</msub>
<mo></mo>
<mstyle>
<mspace height="0.8ex" width="0.8em"> </mspace>
</mstyle>
<mo></mo>
<mi>subject</mi>
<mo></mo>
<mstyle>
<mspace height="0.8ex" width="0.8em"> </mspace>
</mstyle>
<mo></mo>
<mi>to</mi>
<mo></mo>
<mstyle>
<mspace height="0.8ex" width="0.8em"> </mspace>
</mstyle>
<mo></mo>
<mi>X</mi>
</mrow>
</mrow>
<mo>=</mo>
<mrow>
<mi>L</mi>
<mo>+</mo>
<mi>S</mi>
</mrow>
</mrow>
</math>
</maths>
</div>
</li> <li> <para-num num="[1028]"> </para-num> <div class="description-line" id="p-1016" num="1028">where ∥·∥<sub>* </sub>is the nuclear norm (sum of singular values) and ∥×∥<sub>1 </sub>is the sum of the absolute values of the entries of the matrix. These constraints naturally induce L to be low rank, and S to be sparse. In implementations, Applicants used the augmented Lagrangian multiplier method (Lin, Chen &amp; Ma 2010), which was fast and efficient.</div>
</li> <li> <para-num num="[1029]"> </para-num> <div class="description-line" id="p-1017" num="1029">Applicants should note that our interpretation of this optimization problem is slightly different from that seen in some other instances, where S is regarded as capturing noise corrupting the “true” dynamics seen in L. In single-cell data the “noise” may actually be biological in origin, but our primary intent is to isolate the low rank approximation L, which is effectively a smoothed version of the population's dynamics that leaves major trends intact. The advantage of the decomposition of course is that the S matrix is still available afterward, and it may in fact carry useful information about highly stochastic processes within the population.</div>
</li> <li> <para-num num="[1030]"> </para-num> <div class="description-line" id="p-1018" num="1030">Our next goal was to isolate the major trends within the low rank dynamics of the population. To do this Applicants applied independent components analysis (ICA). ICA posits a model in which the expression of a given gene (y<sub>j</sub>) can be decomposed as a linear sum of various effects (s<sub>1 </sub>to s<sub>n</sub>) that are statistically independent of each other:</div>
</li> <li> <div class="description-line" id="p-1019" num="0000">
<br/>
<i>y</i>
<sub>1</sub>
<i>=a</i>
<sub>j</sub>
<i>s</i>
<sub>1</sub>
<i>+a</i>
<sub>j2</sub>
<i>s</i>
<sub>2</sub>
<i>+ . . . + a</i>
<sub>jn</sub>
<i>s</i>
<sub>n </sub>
</div>
</li> <li> <para-num num="[1031]"> </para-num> <div class="description-line" id="p-1020" num="1031">Solving this problem is beyond the scope of this section, but our interest lies primarily in the vector version of this formula,</div>
</li> <li> <div class="description-line" id="p-1021" num="0000"> <br/>y+As</div>
</li> <li> <para-num num="[1032]"> </para-num> <div class="description-line" id="p-1022" num="1032">in which a cell's expression profile y (over all genes) is viewed as a linear sum of independent effects, and the equivalent matrix version</div>
</li> <li> <div class="description-line" id="p-1023" num="0000"> <br/>Y=AS</div>
</li> <li> <para-num num="[1033]"> </para-num> <div class="description-line" id="p-1024" num="1033">in which Applicants decompose all of the dynamics of the cells within our population (the columns of Y) into sums of independent components (ICs). The matrix A above is called the mixing matrix, and in our context describes which genes contribute to which effects. As noted above, the key difference in this case from decompositions like principal components analysis is that the s components are derived in a way to make them as statistically independent as possible. Once the matrix A is estimated, Applicants can then “unmix” the dynamics of each cell in the population by applying the inverse operation (denoted here by W) to its expression profile:</div>
</li> <li> <div class="description-line" id="p-1025" num="0000"> <br/>s=Wy</div>
</li> <li> <para-num num="[1034]"> </para-num> <div class="description-line" id="p-1026" num="1034">This yields a low-dimensional description of what each cell is doing in terms of the independent factors given by s.</div>
</li> <li> <para-num num="[1035]"> </para-num> <div class="description-line" id="p-1027" num="1035">In our case Applicants apply ICA to the low rank matrix L, i.e. Y=L<sup>T </sup>above. Thus Applicants try to separate the population's low rank dynamics into independent factors. As the ICA minimization problem posed in the strongest form cannot practically be solved, different algorithms will give somewhat different answers based on the tradeoffs they make. After trying several methods, Applicants settled on the ProDenICA algorithm (Friedman, Hastie &amp; Tibshirani 2001), which Applicants found to frequently give the highest quality components.</div>
</li> <li> <para-num num="[1036]"> </para-num> <div class="description-line" id="p-1028" num="1036">In general Applicants applied low rank ICA in two ways. First, it can be used to partition cells into subpopulations. Strong trends often lead to independent components that are bimodal, so simply thresholding the value of a component is a means of clustering. Applicants note however that an advantage of this method of subpopulation identification is that it can also identify continuous trends, rather than enforcing discrete categories that may not exist like in other methods of clustering. Secondly, the mixing matrix A is very informative, as it determines the extent to which each gene contributes to a given component. This can be useful both in understanding what the component is measuring (if the most heavily weighted genes have a clear common function) and in identifying groups of genes that are co-expressed in an unbiased way.</div>
</li> <li> <para-num num="[1037]"> </para-num> <div class="description-line" id="p-1029" num="1037">Interpretation of independent components does have some caveats. First, they have no natural sign (so an “enriched” effect may appear as a low value of an independent component) or scale: thus there is no natural order where the first IC is somehow more informative than the next, consistent with the fact that they are meant to represent independent effects. Applicants do note that one pragmatic solution is to order the components by the norm of the corresponding column in the mixing matrix, which tends to place the most interesting components first.</div>
</li> <li> <para-num num="[1038]"> </para-num> <div class="description-line" id="p-1030" num="1038">t-sne visualization To obtain two-dimensional projections of the population's dynamics, Applicants first further reduce the dimensionality of the low rank matrix L using classical PCA (with the number of components determined from a scree plot), and then further reduce thse components via t-distributed stochastic neighbor embedding. Applicants occasionally directly visualize the ICs in this way as well, but because they lack intrinsic scale like principal components, dominant effects can be crowded out by minor ones.</div>
</li> <li> <para-num num="[1039]"> </para-num> <div class="description-line" id="p-1031" num="1039">Hierarchical clustering of genes Several of the analyses in the paper use single-cell co-expression information to cluster genes. For a given list of genes, Applicants perform this clustering by first calculating the gene-gene correlation matrix p over all cells in the population. This is then converted to a dissimilarity matrix π via the transformation π=√{square root over (2(1−p))}. dissimilarity matrix is then clustered using Ward's method. For visualization purposes, Applicants then apply the optimal leaf ordering algorithm in MATLAB. This reorders the leaves in the dendrogram by flipping tree branches to maximize the similarity between adjacent leaves, but without dividing any branches (i.e. the clustering is unchanged, but the dendrogram ordering is in some sense optimal). Applicants then reorder the columns and rows of the correlation matrix via the resulting ordering, so that groups of genes with correlated expression appear as blocks along the diagonal.</div>
</li> <li> <para-num num="[1040]"> </para-num> <div class="description-line" id="p-1032" num="1040">Cell cycle position Applicants used an approach previously described, in which the expression of sets of experimentally-derived genes specific for each cell cycle phase is used for each cell to score cell cycle phase (Macosko et al. 2015).</div>
</li> <li> <para-num num="[1041]"> </para-num> <div class="description-line" id="p-1033" num="1041">Average expression profiles Applicants often create synthetic bulk profiles for different populations. These are created by averaging the normalized expression profile of each cell within that population together.</div>
</li> <li> <para-num num="[1042]"> </para-num> <div class="description-line" id="p-1034" num="1042">Analytical steps for each figure. Applicants now describe the analysis behind each figure in the paper, with references as necessary to the above sections.</div>
</li> <li> <para-num num="[1043]"> </para-num> <div class="description-line" id="p-1035" num="1043">Single-cell analysis in <figref idrefs="DRAWINGS">FIG. 83</figref>. Applicants formed a population consisting of cells treated with 100 nM thapsigargin in each of our 8 genetic backgrounds, along with DMSO-treated control cells. As outlined in the “Low cell count/inviable cell removal” section, Applicants removed cells with substantially lower than average UMI counts or that scored strongly for inviability markers from analysis, as these groups partitioned away from the rest of the population in preliminary analyses. For each perturbation, Applicants then looked for genes that were differentially expressed relative to the control, as described in the “Identification of differentially expressed genes.” Applicants made a list of all genes that had a mean expression of at least 0.5 UMI per cell in the population and for which the Kolmogorov-Smirnov test statistic D&gt;0.15 in at least one perturbation. This led to a group of 1,711 differentially expressed genes. Applicants formed a reduced gene expression matrix containing only these genes, and performed low rank ICA to reduce the population's dynamics therein to 16 ICs (<figref idrefs="DRAWINGS">FIG. 89B</figref>). Applicants examined the raw trends in the population by reducing the low rank matrix to 16 components via PCA (16 components) and then to two dimensions via t-sne, revealing a general breakdown by perturbation and by cell cycle within each perturbation (<figref idrefs="DRAWINGS">FIG. 89B</figref>). Applicants then looked for ICs whose average value varied either across the perturbation, or across the cell cycle position. For each category, four components showed clear trends at the average level and in the t-sne plots (<figref idrefs="DRAWINGS">FIG. 89B</figref>). For example, several of the components clearly showed the expected epistasis patterns for PERK, ATF6, and IRE1 (<figref idrefs="DRAWINGS">FIG. 89B</figref>). The plots made in <figref idrefs="DRAWINGS">FIG. 83B</figref> of the main text were then made by furthering reducing only the ICs that varied across perturbation (IC1-IC4 in <figref idrefs="DRAWINGS">FIG. 89B</figref>) or across the cell cycle (IC5-IC8 in <figref idrefs="DRAWINGS">FIG. 89B</figref>) to two dimensions using t-sne. (I.e., Applicants constructed matrices with cells as rows and the given ICs as columns and reduced those matrices to two dimensions with t-sne.) To make the plots in <figref idrefs="DRAWINGS">FIG. 83C</figref>, Applicants then subsampled our population to only look at cells treated with thapsigargin with or without depletion of PERK, and the DMSO-treated control. Applicants applied the same methodology as above, though with 12 ICs instead of 16. The “G1 cell” IC described in the main text was bimodal within each subpopulation (see inset in right panel of <figref idrefs="DRAWINGS">FIG. 83C</figref>), but with varying distances between the two modes (note that the IC takes a substantially lower value in the +Tg population than in any of the others, <figref idrefs="DRAWINGS">FIG. 83C</figref>). Applicants split each population at the location of the mode. The cell cycle position histograms were made as described above. To make <figref idrefs="DRAWINGS">FIG. 83E</figref>, Applicants took the 25 genes that most positively influenced the IC and the 25 genes that most negatively influenced the IC (by sorting the mixing matrix column for that IC by coefficient value) and then clustered them based on coexpression as described in the “Hierarchical clustering of genes” section. The meaning of each cluster was discerned by the pattern of up- and down-regulation observed within.</div>
</li> <li> <para-num num="[1044]"> </para-num> <div class="description-line" id="p-1036" num="1044">Branch epistasis analysis in <figref idrefs="DRAWINGS">FIG. 83F</figref>. Applicants created two populations: (1) consisting of cells treated with 100 nM thapsigargin in each of our 8 genetic backgrounds, along with DMSO-treated control cells, or (2) consisting of cells treated with 4 μg/mL tunicamycin in each of our 8 genetic backgrounds, along with DMSO-treated control cells. To identify informative differentially regulated genes, Applicants used the random forest classifier method described in the “Identification of differentially expressed genes,” limiting the random forest to pick 100 genes for each of the two populations. Applicants then combined these two lists in one and discarded any duplicate genes. Applicants created average profiles of expression of these genes for each of the nine conditions present in the two populations, as visualized in <figref idrefs="DRAWINGS">FIG. 83F</figref>. The average epistatic phenotype of a gene can then be viewed as a 9-vector in either the thapsigargin- or tunicamycin-treated populations. Applicants discarded any genes where the correlation between these two conditions was less than 0.9, as Applicants were only interested in factors that showed the same regulation in response to both condtions. The end result was the 104 genes presented in <figref idrefs="DRAWINGS">FIG. 83F</figref>. These were then clustered based on their co-expression pattern as described in the “Hierarchical clustering of genes” section, with the exception that Spearman correlation was used instead of Pearson correlation (to emphasize the large shifts in expression across the population). Rough meanings were ascribed to clusters based on the average pattern of gene expression across perturbations, but Applicants emphasize that many targets show some degree of cross-regulation. To assess this in an unbiased way, Applicants constructed a matrix consisting of the average expression of the 104 assayed genes across the 17 unique conditions present in the experiment, and reduced it to four independent components using FastICA. Three of the components clearly corresponded to ATF6, IRE1, and PERK perturbations, as they showed banded patterns in the reduced matrix matching the pattern of epistasis for those regulators seen in <figref idrefs="DRAWINGS">FIG. 83F</figref> (e.g. the PERK component was high in all PERK conditions, and low everywhere else). The fourth component was low in the DMSO and all tunicamycin-treated conditions, and high in the thapsigargin-treated condition, so Applicants discarded it as representing the difference between the two perturbations. The panel at the bottom of <figref idrefs="DRAWINGS">FIG. 83F</figref> plots the mixing matrix coefficients for each gene in the indicated component, and thus determines how much that gene affects that component's value.</div>
</li> <li> <para-num num="[1045]"> </para-num> <div class="description-line" id="p-1037" num="1045">Genome-wide CRISPRi screen in <figref idrefs="DRAWINGS">FIG. 84</figref>. Analysis of the screen is described above along-side the experimental details above.</div>
</li> <li> <para-num num="[1046]"> </para-num> <div class="description-line" id="p-1038" num="1046">Clustering of guides and perturbations in <figref idrefs="DRAWINGS">FIG. 85</figref>. Applicants first split our large perturb-seq population into subpopulations based on guide identity and created average expression profiles (see “Average expression profiles” section) for each perturbations of all genes with mean representation &gt;1 UMI per cell. Applicants calculated the perturbation-perturbation correlation matrix between all average expression profiles, and then clustering it using the same methodology described in the “Hierarchical clustering of genes.” The ordering is seen in <figref idrefs="DRAWINGS">FIG. 91A</figref>. Because guides targeting the same gene behaved similarly in this analysis, in subsequent analyses Applicants instead split the population into subpopulations based on guide target (thus merging subpopulations that had different guides that targeted the same gene). Applicants clustered these profiles using the same criteria, and optimally ordered the resulting dendrogram and correlation matrix (as described in “Hierarchical clustering of genes”) to produce <figref idrefs="DRAWINGS">FIG. 85A</figref>.</div>
</li> <li> <para-num num="[1047]"> </para-num> <div class="description-line" id="p-1039" num="1047">Assessing guide homogeneity and knockdown in <figref idrefs="DRAWINGS">FIG. 85</figref>. Most guide targets were too low abundance to interrogate directly at single-cell resolution. Applicants first directly visualized the shift in guide target expression induced by the guide, comparing the distribution of expression in control cells to cells perturbed for a given target (<figref idrefs="DRAWINGS">FIG. 91B</figref>). Applicants calculated mean knockdown per guide (<figref idrefs="DRAWINGS">FIG. 85C</figref>), and assigned 95% confidence intervals to our estimates via bootstrapping.</div>
</li> <li> <para-num num="[1048]"> </para-num> <div class="description-line" id="p-1040" num="1048">Applicants also attempted to assess to what extent knockdown varied throughout the population based on phenotype. To do this, Applicants needed an unbiased means of assessing deviation in behavior from the control cells. Applicants leveraged a method called OneClassSVM (Scholkopf et al. 2001) which is a means of novelty detection. Given a set of training exemplars, a OneClassSVM learns an estimate of how those points are distributed (potentially in a high-dimensional space). When given new observations, the OneClassSVM then estimates how likely it is that those observations came from the same distribution as the training set, or if they are outliers (potentially novel). In our case Applicants trained the OneClassSVM using control cells, and thus scored the extent to which perturbed cells scored as outliers, or if they fell within the expected range of behavior for unperturbed cells. Specifically, for each guide target, Applicants performed the following algorithm:
</div> </li> <ul> <li id="ul0007-0001" num="0000"> <ul> <li id="ul0008-0001" num="1049">1.Form a population of all cells perturbed for that target, and an equal number of randomly sampled control cells.</li> <li id="ul0008-0002" num="1050">2.Find all genes that are expressed at an average level of 0.5 UMI per cell or higher and that are differentially expressed between control and perturbed cells by the Kolmogorov-Smimov test (as described in “Identification of differentially expressed genes”) at P &lt;0.01.</li> <li id="ul0008-0003" num="1051">3.Form a reduced gene expression matrix consisting only of the differentially expressed genes. Create a low-dimensional picture of the dynamics within this population by reducing this matrix to 8 dimensions via PCA.</li> <li id="ul0008-0004" num="1052">4.To form an estimate of “normal” behavior, train a OneClassSVM model to estimate the support of the control cells in this 8-dimensional space. The model was trained assuming a contamination rate with outliers of 5%.</li> <li id="ul0008-0005" num="1053">5. Score each cell in the perturbed population using the OneClassSVM model to estimate the extent it deviates from control behavior.</li> </ul> </li> </ul>
<li> <para-num num="[1054]"> </para-num> <div class="description-line" id="p-1041" num="1054">These scores generally assigned most or all of the perturbed cells outlier status, except in guides where very few genes were perturbed to begin with (bottom panel of <figref idrefs="DRAWINGS">FIG. 85D</figref>). Ordering the cells by score, Applicants split each perturbed cell population into top third and bottom thirds (i.e. the most and least perturbed cells) and assessed the difference in average knockdown in each of these populations (<figref idrefs="DRAWINGS">FIG. 91C</figref>), with a difference of ˜8% on average.</div>
</li> <li> <para-num num="[1055]"> </para-num> <div class="description-line" id="p-1042" num="1055">Applicants also reported the number of differentially expressed genes measured above in the bottom panel of <figref idrefs="DRAWINGS">FIG. 85D</figref>.</div>
</li> <li> <para-num num="[1056]"> </para-num> <div class="description-line" id="p-1043" num="1056">Scoring branch activation in <figref idrefs="DRAWINGS">FIG. 85D</figref>. As outlined in the main text, Applicants adopted a data-driven strategy to score activation of each of the UPR branches using the epistasis experiment as training data. To do this, Applicants assigned the label “ATF6 active”, “IRE1 active”, or “PERK active” to each cell in the epistasis experiment based on whether a given branch was present (i.e. not depleted) and induced (tunicamycin or thapsigargin had been added). For example, cells treated with thapsigargin and depleted for IRE1 would have ATF6 and PERK active, but not IRE1. Applicants converted these labels to scores of 0, for inactive, and 1, for active, and then trained three random forest regressors to predict activation of each branch. The training strategy was the same as outlined in the “Identification of differentially expressed genes” section: each cell was regarded as a training data point, with every gene of mean &gt;1 UMI initially regarded as a possible feature for predicting branch activation. Each regressor was constrained to use the top 25 genes for predicting branch activation, as Applicants found no performance improvement when more genes were included. The genes isolated as most important by the three regressors for scoring activation of the three branches all appear in the epistasis analysis in <figref idrefs="DRAWINGS">FIG. 83F</figref>.</div>
</li> <li> <para-num num="[1057]"> </para-num> <div class="description-line" id="p-1044" num="1057">To assess performance, Applicants compared this approach to scoring based on two other strategies:
</div> </li> <ul> <li id="ul0009-0001" num="0000"> <ul> <li id="ul0010-0001" num="1058">1.Gene list approach: A list of hand-picked branch-specific genes were chosen from <figref idrefs="DRAWINGS">FIG. 83F</figref>, and a score was defined as the sum of the normalized expression of those genes.</li> <li id="ul0010-0002" num="1059">2.ICA approach: To allow for more complicated logic than simple sums, Applicants used the</li> </ul> </li> </ul>
<li> <para-num num="[1060]"> </para-num> <div class="description-line" id="p-1045" num="1060">ICA decomposition seen in <figref idrefs="DRAWINGS">FIG. 83F</figref> and computed the value of each IC for every cell in the population. [001063] With each scoring system, Applicants subtracted the median of the DMSO-treated control cells' and thresholded all cells with negative scores to zero. Applicants then assessed the overlap of score distributions between cells expected to have a given branch active or inactive. Both the gene list approach and the ICA approach performed worse than the random forests in this analysis (<figref idrefs="DRAWINGS">FIG. 91F</figref>). [001064] The branch scores seen in <figref idrefs="DRAWINGS">FIG. 85D</figref> are the result of applying the random forest regressor scoring system to each cell in the perturb-seq experiment, and then averaging the results within cells knocked down for the same gene. The average scores assigned by the ICA method agree well (cf. <figref idrefs="DRAWINGS">FIG. 85D</figref> and <figref idrefs="DRAWINGS">FIG. 91E</figref>).</div>
</li> <li> <para-num num="[1061]"> </para-num> <div class="description-line" id="p-1046" num="1061">Single-cell analysis in <figref idrefs="DRAWINGS">FIG. 86</figref>. Applicants formed a population of cells containing either our two guides targeting HSPA5, or the NegCtrl2 guide. Applicants found all genes that had mean abundance &gt;0.5 UMI per cell and that were differentially expressed between the two populations by Kolmogorov-Smirnov test (P &lt;0.01), resulting in ˜2,100 genes. Applicants formed a reduced gene expression matrix consisting only of these genes and applied low rank ICA to reduce the population's dynamics therein to 12 ICs. The t-sne plots were made by reducing the low rank matrix to 16 components using PCA and then applying t-sne (see “t-sne visualization” section). Branch activation scores in <figref idrefs="DRAWINGS">FIG. 86C</figref> were assigned as described above in the “Scoring branch activation in <figref idrefs="DRAWINGS">FIG. 85D</figref>” section.</div>
</li> <li> <para-num num="[1062]"> </para-num> <div class="description-line" id="p-1047" num="1062">Two ICs varied substantially in average value between the control and perturbed cells (<figref idrefs="DRAWINGS">FIG. 86B</figref>). The first, IC1, had a two-phase distribution in which all control cells and the majority of HSPA5-perturbed cells fell in the large lower peak, and a subpopulation of HSPA5-perturbed cells fell into a long tail of higher values (<figref idrefs="DRAWINGS">FIG. 86B</figref>). Applicants defined the HSPA5 IC1 HIGH cells to be the ones that fell within this tail (<figref idrefs="DRAWINGS">FIG. 86B</figref>). <figref idrefs="DRAWINGS">FIG. 86D</figref> shows the normalized expression of genes found in our epistasis analysis (<figref idrefs="DRAWINGS">FIG. 83F</figref>) as columns, and the HSPA5-perturbed cells as rows, ordered by increasing IC1. <figref idrefs="DRAWINGS">FIG. 86E</figref> was created simply by averaging the expression of HSPA5 within the subpopulations defined in <figref idrefs="DRAWINGS">FIG. 86B</figref>. <figref idrefs="DRAWINGS">FIG. 86F</figref> was created using the cell cycle positions called in the “Cell cycle position” section.</div>
</li> <li> <para-num num="[1063]"> </para-num> <div class="description-line" id="p-1048" num="1063">Gene clustering analysis in <figref idrefs="DRAWINGS">FIG. 86H</figref>. Applicants first needed an unbiased approach to find programs of gene expression induced in the perturb-seq experiment. To do this Applicants separated the population into control cells (containing our two control guides) and perturbed cells (containing any guide). Applicants constructed average expression profiles (see “Average expression profiles” section) of each, and then restricted our analysis to genes of mean expression &gt;0.5 UMI per cell on average in the perturbed population, and whose normalized expression was &gt;0.5. (Control cells by definition have mean normalized expression 0 for all genes, see “Expression normalization” section.) Applicants then used a random forest classifier approach to select 200 of these induced genes that varied informatively across all of the perturbations in the perturb-seq experiment (see “Identification of differentially expressed genes” section). The genes were then clustered based on their co-expression throughout the population, with the dendrogram leaves optimally reordered (see “Hierarchical clustering of genes” section). Our assumption was that many of these “induced genes” were involved in the unfolded protein response. Applicants evaluated UPR dependence by examining the expression pattern of the induced genes within thapsigargin- and tunicamycin-treated cells (<figref idrefs="DRAWINGS">FIG. 92B</figref>). Applicants also assigned identities to some other clusters based on clear functional connections (as seen in <figref idrefs="DRAWINGS">FIG. 86H</figref>).</div>
</li> <li> <para-num num="[1064]"> </para-num> <div class="description-line" id="p-1049" num="1064">Comparison of clustering of UPR genes in <figref idrefs="DRAWINGS">FIG. 86I</figref> and <figref idrefs="DRAWINGS">FIG. 92A</figref>. As many UPR genes fell out of the previous analysis, Applicants wanted to evaluate the ability to go the opposite direction, and cluster known interactions. Applicants thus reexamined the list of UPR-regulated genes found in <figref idrefs="DRAWINGS">FIG. 83F</figref>. Applicants separated the perturb-seq population into control cells (containing our two control guides) and perturbed cells (containing any guide). Applicants constructed average expression profiles (see “Average expression profiles” section) of each, and then restricted our analysis to the UPR-regulated genes that showed the same pattern of induction or repression in the perturbed cell population as they did in the cells treated with thapsigargin in the epistasis experiment that had all branches of the UPR intact. Applicants then performed hierarchical clustering of these genes (see “Hierarchical clustering of genes” section) using co-expression information from either (1) all cells in the epistasis experiment, (2) all cells in the perturb-seq, and (3) only control cells in the perturb-seq experiment. Applicants assessed the similarity among clusterings using the cophenetic correlation coefficient, i.e. the correlation coefficient between dendrogram distances taken over all possible pairs of genes. Closeness in cophenetic correlation thus implies that the dendrograms tend to place the same genes close to each other. The figure is meant only as a visual aid, as the cophenetic correlation relies on information beyond the linear order. The genes were roughly grouped based on their epistasis pattern in the epistasis experiment (as in <figref idrefs="DRAWINGS">FIG. 83F</figref>), and then color was preserved as they were shuffled by the other two clusterings.</div>
</li> <li> <para-num num="[1065]"> </para-num> <div class="description-line" id="p-1050" num="1065">Enrichment of cholesterol genes in <figref idrefs="DRAWINGS">FIG. 86K</figref>. Our unbiased analysis in <figref idrefs="DRAWINGS">FIG. 86H</figref> contained a cluster of genes involved in cholesterol biosynthesis: ACAT2, FDPS, FADS1, INSIG1, TMEM97. Applicants made a “cholesterol score” by summing the normalized expression of this group of genes in each cell, and then created a subpopulation containing (1) cells with cholesterol scores at or above the 95% of the control cell population and (2) control cells. This gave ˜9,000 cells. Within this subpopulation, Applicants then correlated the cholesterol score with the normalized expression of all genes with mean &gt;0.25 UMI per cell. Applicants then selected all genes that had a correlation of 0.15 or higher with the cholesterol score for further analysis. Applicants clustered the genes by co-expression within the population (see “Hierarchical clustering of genes” section), and then selected a group of 23 genes that clustered together with the original five and that appeared as a distinct block on the diagonal of the gene-gene correlation matrix. To demonstrate the improvement in correlation obtained by this “fishing” approach, Applicants compared correlation matrices composed of these 23 genes and 23 random genes of similar average abundance between our enriched population, and control cells (seen in <figref idrefs="DRAWINGS">FIG. 86K</figref>). Finally, Applicants used Enrichr (Kuleshov et al. 2016) to obtain Reactome annotations and Encode SREBP binding state. Note that some of the genes that don't have annotations nevertheless are almost certainly cholesterol-related, such as the lncRNA RP11-660L16 which is directly next to DHC7R.</div>
</li> <li> <para-num num="[1066]"> </para-num> <div class="description-line" id="p-1051" num="1066">Enrichment of heat shock genes in <figref idrefs="DRAWINGS">FIG. 86L</figref>. Applicants followed an identical approach to the above, except starting with the genes HSPA1A and HSPA1B. In this case Applicants present all of the genes that had correlations of 0.15 or greater. Applicants again used Enrichr to find the top 3 most enriched transcription factor binding sites among the set of genes, as presented in <figref idrefs="DRAWINGS">FIG. 6L</figref>.</div>
</li> <li> <para-num num="[1067]"> </para-num> <div class="description-line" id="p-1052" num="1067">Single-cell analysis in <figref idrefs="DRAWINGS">FIG. 87</figref>. Applicants formed populations of cells containing guides targeting either SEC61A1 or SEC61B, along with cells containing the NegCtrl2 guide. Applicants found all genes that had mean abundance &gt;0.5 UMI per cell and that were differentially expressed between the two populations by Kolmogorov-Smirnov test setting a threshold of D&gt;0.15 for SEC61A1, and D&gt;0.1 for SEC61B, which is a weaker perturbation (see “Identification of differentially expressed genes” section). The different thresholds were chosen largely for esthetic reasons: lowering the threshold with SEC61A1, which is a strong perturbation, resulted in the inclusion of a number of cell cycle genes that caused the control population to fragment into subpopulations by cell cycle phase, which Applicants felt was distracting. In each case Applicants formed a reduced gene expression matrix consisting only of differentially expressed genes, then applied robust PCA (see “Low rank ICA” section) to these matrices, and then visualized the dynamics using t-sne plots generated using the first 16 principal components (see “t-sne visualization” section). Branch activation scores in <figref idrefs="DRAWINGS">FIG. 87A</figref>,B, S<b>6</b>A were assigned as described above in the “Scoring branch activation in <figref idrefs="DRAWINGS">FIG. 85D</figref>” section.</div>
</li> <heading id="h-0087">REFERENCES</heading>
<li> <para-num num="[1068]"> </para-num> <div class="description-line" id="p-1053" num="1068">Acosta-Alvear, D., Zhou, Y., Blais, A., Tsikitis, M., Lents, N. H., Arias, C., Lennon, C. J., Kluger, Y. &amp; Dynlacht, B. D. 2007, “XBP1 controls diverse cell type- and condition-specific transcriptional regulatory networks”, <i>Molecular Cell, </i>vol. 27, no. 1, pp. 53-66.</div>
</li> <li> <para-num num="[1069]"> </para-num> <div class="description-line" id="p-1054" num="1069">Adiconis, X., Borges-Rivera, D., Satija, R., DeLuca, D. S., Busby, M. A., Berlin, A. M., Sivachenko, A., Thompson, D. A., Wysoker, A., Fennell, T., Gnirke, A., Pochet, N., Regev, A. &amp; Levin, J. Z. Comparative analysis of RNA sequencing methods for degraded or low-input samples. <i>Nat Methods. </i>10, 623-629, doi:10.1038/nmeth.2483 (2013). PMCID:3821180.</div>
</li> <li> <para-num num="[1070]"> </para-num> <div class="description-line" id="p-1055" num="1070">Aguirre, A. J., Meyers, R. M., Weir, B. A., Vazquez, F., Zhang, C., Ben-David, U., Cook, A., Ha, G., Harrington, W. F., Doshi, M. B., et al 2016, “Genomic Copy Number Dictates a Gene-Independent Cell Response to CRISPR/Cas9 Targeting”, <i>Cancer Discovery, </i>vol. 6, no. 8, pp. 914-929.</div>
</li> <li> <para-num num="[1071]"> </para-num> <div class="description-line" id="p-1056" num="1071">Altshuler, D., Daly, M. J., and Lander, E. S. (2008). Genetic Mapping in Human Disease. Science (80-.). 322, 881-888.</div>
</li> <li> <para-num num="[1072]"> </para-num> <div class="description-line" id="p-1057" num="1072">Amit, I., Garber, M., Chevrier, N., Leite, A. P., Donner, Y., Eisenhaure, T., Guttman, M., Grenier, J. K., Li, W., Zuk, O., et al. (2009). Unbiased Reconstruction of a Mammalian Transcriptional Network Mediating Pathogen Responses. Science (80-.). 326, 257-263.</div>
</li> <li> <para-num num="[1073]"> </para-num> <div class="description-line" id="p-1058" num="1073">Anders, C., Niewoehner, O., Duerst, A. &amp; Jinek, M. 2014, “Structural basis of PAM-dependent target DNA recognition by the Cas9 endonuclease”, <i>Nature, </i>vol. 513, no. 7519, pp. 569-573.</div>
</li> <li> <para-num num="[1074]"> </para-num> <div class="description-line" id="p-1059" num="1074">Assarsson, E., Lundberg, M., Holmquist, G., Bjorkesten, J., Thorsen, S. B., Ekman, D., Eriksson, A., Rennel Dickens, E., Ohlsson, S., Edfeldt, G., et al. (2014). Homogenous 96-plex PEA immunoassay exhibiting high sensitivity, specificity, and excellent scalability. PloS one 9, e95192.</div>
</li> <li> <para-num num="[1075]"> </para-num> <div class="description-line" id="p-1060" num="1075">Bamshad, M. J., Ng, S. B., Bigham, A. W., Tabor, H. K., Emond, M. J., Nickerson, D. A., and Shendure, J. (2011). Exome sequencing as a tool for Mendelian disease gene discovery. Nat. Rev. Genet. 12, 745-755.</div>
</li> <li> <para-num num="[1076]"> </para-num> <div class="description-line" id="p-1061" num="1076">Bandyopadhyay, S., Mehta, M., Kuo, D., Sung, M.-K., Chuang, R., Jaehnig, E. J., Bodenmiller, B., Licon, K., Copeland, W., Shales, M., et al. (2010). Rewiring of Genetic Networks in Response to DNA Damage. Science (80-.). 330, 1385-1389.</div>
</li> <li> <para-num num="[1077]"> </para-num> <div class="description-line" id="p-1062" num="1077">Bao, X. R., Ong, S., Goldberger, O., Peng, J., Sharma, R., Thompson, D. A., Vafai, S. B., Cox, A. G., Marutani, E., Ichinose, F., et al 2016, “Mitochondrial dysfunction remodels one-carbon metabolism in human cells”, <i>eLife, </i>vol. 5.</div>
</li> <li> <para-num num="[1078]"> </para-num> <div class="description-line" id="p-1063" num="1078">Bassik, M. C., Kampmann, M., Lebbink, R. J., Wang, S., Hein, M. Y., Poser, I., Weibezahn, J., Horlbeck, M. a, Chen, S., Mann, M., et al. (2013a). A systematic mammalian genetic interaction map reveals pathways underlying ricin susceptibility. Cell 152, 909-922.</div>
</li> <li> <para-num num="[1079]"> </para-num> <div class="description-line" id="p-1064" num="1079">Bassik, M. C., Kampmann, M., Lebbink, R. J., Wang, S., Hein, M. Y., Poser, I., Weibezahn, J., Horlbeck, M. A., Chen, S., Mann, M., et al. (2013b). A Systematic Mammalian Genetic Interaction Map Reveals Pathways Underlying Ricin Susceptibility. Cell 152, 909-922.</div>
</li> <li> <para-num num="[1080]"> </para-num> <div class="description-line" id="p-1065" num="1080">Beerenwinkel, N., Pachter, L., and Sturmfels, B. (2007). Epistasis and Shapes of Fitness Landscapes. Stat. Sin. 17,1317-1342.</div>
</li> <li> <para-num num="[1081]"> </para-num> <div class="description-line" id="p-1066" num="1081">Bendall, S. C., Simonds, E. F., Qiu, P., Amir el, A. D., Krutzik, P. O., Finck, R., Bruggner, R. V., Melamed, R., Trejo, A., Ornatsky, O.I., et al. (2011). Single-cell mass cytometry of differential immune and drug responses across a human hematopoietic continuum. Science 332, 687-696.</div>
</li> <li> <para-num num="[1082]"> </para-num> <div class="description-line" id="p-1067" num="1082">Berger, A. H., Brooks, A. N., Wu, X., Shrestha, Y., Chouinard, C., Piccioni, F., Bagul, M., Kamburov, A., Imielinski, M., Hogstrom, L., et al. (2016). High-throughput Phenotyping of Lung Cancer Somatic Mutations. Cancer Cell 0, 248-249.</div>
</li> <li> <para-num num="[1083]"> </para-num> <div class="description-line" id="p-1068" num="1083">Blecher-Gonen, R., Barnett-Itzhaki, Z., Jaitin, D., Amann-Zalcenstein, D., Lara-Astiaso, D. &amp; Amit, I. High-throughput chromatin immunoprecipitation for genome-wide mapping of in vivo protein-DNA interactions and epigenomic states. <i>Nat Protoc. </i>8, 539-554, doi:10.1038/nprot.2013.023 (2013).</div>
</li> <li> <para-num num="[1084]"> </para-num> <div class="description-line" id="p-1069" num="1084">Bochkis, I. M., Przybylski, D., Chen, J. &amp; Regev, A. Changes in nucleosome occupancy associated with metabolic alterations in aged mammalian liver. <i>Cell reports. </i>9, 996-1006, doi:10.1016/j.celrep.2014.09.048 (2014). PMCID:4250828.</div>
</li> <li> <para-num num="[1085]"> </para-num> <div class="description-line" id="p-1070" num="1085">Boone, C., Bussey, H., and Andrews, B. J. (2007). Exploring genetic interactions and networks with yeast. 8, 437-449.</div>
</li> <li> <para-num num="[1086]"> </para-num> <div class="description-line" id="p-1071" num="1086">Bornstein, C., Winter, D., Barnett-Itzhaki, Z., David, E., Kadri, S., Garber, M. &amp; Amit, I. A negative feedback loop of transcription factors specifies alternative dendritic cell chromatin States. <i>Mol Cell. </i>56, 749-762, doi:10.1016/j.molcel.2014.10.014 (2014). PMCID:4412443.</div>
</li> <li> <para-num num="[1087]"> </para-num> <div class="description-line" id="p-1072" num="1087">Botstein, D., and Risch, N. (2003). Discovering genotypes underlying human phenotypes: past successes for mendelian disease, future approaches for complex disease. Nat. Genet. 33, 228-237.</div>
</li> <li> <para-num num="[1088]"> </para-num> <div class="description-line" id="p-1073" num="1088">Briner, A. E., Donohoue, P. D., Gomaa, A. A., Selle, K., Slorach, E. M., Nye, C. H., Haurwitz, R. E., Beisel, C. L., May, A. P. &amp; Barrangou, R. 2014, “Guide RNA functional modules direct Cas9 activity and orthogonality”, <i>Molecular Cell, </i>vol. 56, no. 2, pp. 333-339.</div>
</li> <li> <para-num num="[1089]"> </para-num> <div class="description-line" id="p-1074" num="1089">Buettner, F., Natarajan, K. N., Casale, F. P., Proserpio, V., Scialdone, A., Theis, F. J., Teichmann, S. A., Marioni, J. C., and Stegle, O. (2015). Computational analysis of cell-to-cell heterogeneity in single-cell RNA-sequencing data reveals hidden subpopulations of cells. Nat. Biotechnol. 33, 155-160.</div>
</li> <li> <para-num num="[1090]"> </para-num> <div class="description-line" id="p-1075" num="1090">Cabili, M. N., Dunagin, M. C., McClanahan, P. D., Biaesch, A., Padovan-Merhar, O., Regev, A., Rinn, J. L. &amp; Raj, A. Localization and abundance analysis of human lncRNAs at single-cell and single-molecule resolution. <i>Genome Biol. </i>16, 20, doi:10.1186/s13059-015-0586-4 (2015). PMCID:4369099.</div>
</li> <li> <para-num num="[1091]"> </para-num> <div class="description-line" id="p-1076" num="1091">Cabili, M. N., Trapnell, C., Goff, L., Koziol, M., Tazon-Vega, B., Regev, A. &amp; Rinn, J. L. Integrative annotation of human large intergenic noncoding RNAs reveals global properties and specific subclasses. <i>Genes Dev. </i>25, 1915-1927, doi:10.1101/gad.17446611 (2011). PMCID:3185964.</div>
</li> <li> <para-num num="[1092]"> </para-num> <div class="description-line" id="p-1077" num="1092">Calfon, M., Zeng, H., Urano, F., Till, J. H., Hubbard, S. R., Harding, H. P., Clark, S. G. &amp; Ron, D. 2002, “IRE1 couples endoplasmic reticulum load to secretory capacity by processing the XBP-1 mRNA”, <i>Nature, </i>vol. 415, no. 6867, pp. 92-96.</div>
</li> <li> <para-num num="[1093]"> </para-num> <div class="description-line" id="p-1078" num="1093">Candès, E. J., and Recht, B. (2009). Exact Matrix Completion via Convex Optimization. Found. Comput. Math. 9, 717-772.</div>
</li> <li> <para-num num="[1094]"> </para-num> <div class="description-line" id="p-1079" num="1094">Candès, E. J., Li, X., Ma, Y. &amp; Wright, J. 2011, “Robust principal component analysis?”, <i>Journal of the ACM </i>(<i>JACM</i>), vol. 58, no. 3, pp. 11.</div>
</li> <li> <para-num num="[1095]"> </para-num> <div class="description-line" id="p-1080" num="1095">Capaldi, A. P., Kaplan, T., Liu, Y., Habib, N., Regev, A., Friedman, N., and O'Shea, E. K. (2008). Structure and function of a transcriptional network activated by the MAPK Hogl. Nat. Genet. 40, 1300-1306.</div>
</li> <li> <para-num num="[1096]"> </para-num> <div class="description-line" id="p-1081" num="1096">Carter, G. W., Prinz, S., Neou, C., Shelby, J. P., Marzolf, B., Thorsson, V., and Galitski, T. (2007). Prediction of phenotype and gene expression for combinations of mutations. Mol. Syst. Biol. 3, 96.</div>
</li> <li> <para-num num="[1097]"> </para-num> <div class="description-line" id="p-1082" num="1097">Carbon, S., Ireland, A., Mungall, C. J., Shu, S., Marshall, B. &amp; Lewis, S. 2009, “AmiGO: online access to ontology and annotation data”, <i>Bioinformatics </i>(<i>Oxford, England</i>), vol. 25, no. 2, pp. 288-289.</div>
</li> <li> <para-num num="[1098]"> </para-num> <div class="description-line" id="p-1083" num="1098">Cartwright, T., Perkins, N. D., and L Wilson, C. (2016). NFKB1: a suppressor of inflammation, ageing and cancer. FEBS J. 283, 1812-1822.</div>
</li> <li> <para-num num="[1099]"> </para-num> <div class="description-line" id="p-1084" num="1099">Chan, M. M., Smith, Z. D., Egli, D., Regev, A. &amp; Meissner, A. Mouse ooplasm confers context-specific reprogramming capacity. <i>Nature genetics. </i>44, 978-980, doi:10.1038/ng.2382 (2012). PMCID:3432711.</div>
</li> <li> <para-num num="[1100]"> </para-num> <div class="description-line" id="p-1085" num="1100">Chavez, A., Scheiman, J., Vora, S., Pruitt, B. W., Tuttle, M., P R Iyer, E., Lin, S., Kiani, S., Guzman, C. D., Wiegand, D. J., et al 2015, “Highly efficient Cas9-mediated transcriptional programming”, <i>Nature Methods, </i>vol. 12, no. 4, pp. 326-328.</div>
</li> <li> <para-num num="[1101]"> </para-num> <div class="description-line" id="p-1086" num="1101">Chen, Y., Liu, P., Nielsen, A. A. K., Brophy, J. A. N., Clancy, K., Peterson, T. &amp; Voigt, C. A. 2013, “Characterization of 582 natural and synthetic terminators and quantification of their design constraints”, Nature Methods, vol. 10, no. 7, pp. 659-664.</div>
</li> <li> <para-num num="[1102]"> </para-num> <div class="description-line" id="p-1087" num="1102">Chen, S., Sanjana, N. E., Zheng, K., Shalem, O., Lee, K., Shi, X., Scott, D. A., Song, J., Pan, J. Q., Weissleder, R., et al. (2015). Genome-wide CRISPR Screen in a Mouse Model of Tumor Growth and Metastasis. Cell 160, 1246-1260.</div>
</li> <li> <para-num num="[1103]"> </para-num> <div class="description-line" id="p-1088" num="1103">Cheng, C. S., Rai, K., Garber, M., Hollinger, A., Robbins, D., Anderson, S., Macbeth, A., Tzou, A., Carneiro, M. O., Raychowdhury, R., Russ, C., Hacohen, N., Gershenwald, J. E., Lennon, N., Nusbaum, C., Chin, L., Regev, A. &amp; Amit, I. Semiconductor-based DNA sequencing of histone modification states. <i>Nat Commun. </i>4, 2672, doi:10.1038/ncomms3672 (2013). PMCID:3917140.</div>
</li> <li> <para-num num="[1104]"> </para-num> <div class="description-line" id="p-1089" num="1104">Chevrier, N., Mertins, P., Artyomov, M. N., Shalek, A. K., Iannacone, M., Ciaccio, M. F., Gat-Viks, I., Tonti, E., DeGrace, M. M., Clauser, K. R., Garber, M., Eisenhaure, T. M., Yosef, N., Robinson, J., Sutton, A., Andersen, M. S., Root, D. E., von Andrian, U., Jones, R. B., Park, H., Carr, S. A., Regev, A., Amit, I. &amp; Hacohen, N. Systematic discovery of TLR signaling components delineates viral-sensing circuits. <i>Cell. </i>147, 853-867, doi:10.1016/j.cell.2011.10.022 (2011). PMCID:3809888.</div>
</li> <li> <para-num num="[1105]"> </para-num> <div class="description-line" id="p-1090" num="1105">Chuang, C., Lee, K., Fan, C. &amp; Su, Y. 2009, “Porcine type III RNA polymerase III promoters for short hairpin RNA expression”, <i>Animal Biotechnology, </i>vol. 20, no. 1, pp. 34-39.</div>
</li> <li> <para-num num="[1106]"> </para-num> <div class="description-line" id="p-1091" num="1106">Chung, K., Wallace, J., Kim, S. Y., Kalyanasundaram, S., Andalman, A. S., Davidson, T. J., Mirzabekov, J. J., Zalocusky, K. A., Mattis, J., Denisin, A. K., et al. (2013). Structural and molecular interrogation of intact biological systems. Nature 497, 332-337.</div>
</li> <li> <para-num num="[1107]"> </para-num> <div class="description-line" id="p-1092" num="1107">Chung, N. C., and Storey, J. D. (2015). Statistical significance of variables driving systematic variation in high-dimensional data. Bioinformatics 31, 545-554.</div>
</li> <li> <para-num num="[1108]"> </para-num> <div class="description-line" id="p-1093" num="1108">Cohen, A., and Sheva, B.- (1998). HIDDEN MARKOV MODELS IN BIOMEDICAL SIGNAL PROCESSING. 20, 1145-1150.</div>
</li> <li> <para-num num="[1109]"> </para-num> <div class="description-line" id="p-1094" num="1109">Cong, L., Ran, F. A., Cox, D., Lin, S., Barretto, R., Habib, N., Hsu, P. D., Wu, X., Jiang, W., Marraffini, L. a, et al. (2013). Multiplex genome engineering using CRISPR/Cas systems. Science 339, 819-823.</div>
</li> <li> <para-num num="[1110]"> </para-num> <div class="description-line" id="p-1095" num="1110">Costanzo, M., Baryshnikova, A., Bellay, J., Kim, Y., Spear, E. D., Sevier, C. S., Ding, H., Koh, J. L. Y., Toufighi, K., Mostafavi, S., et al. (2010). The Genetic Landscape of a Cell. Science (80-.). 327, 425-431.</div>
</li> <li> <para-num num="[1111]"> </para-num> <div class="description-line" id="p-1096" num="1111">Dang, Y., Jia, G., Choi, J., Ma, H., Anaya, E., Ye, C., Shankar, P. &amp; Wu, H. 2015, “Optimizing sgRNA structure to improve CRISPR-Cas9 knockout efficiency”, <i>Genome Biology, </i>vol. 16, pp. 280.</div>
</li> <li> <para-num num="[1112]"> </para-num> <div class="description-line" id="p-1097" num="1112">Duan, Q., Flynn, C., Niepel, M., Hafner, M., Muhlich, J. L., Fernandez, N. F., Rouillard, A. D., Tan, C. M., Chen, E. Y., Golub, T. R., et al. (2014). LINCS Canvas Browser: interactive web app to query, browse and interrogate LINCS L1000 gene expression signatures. Nucleic Acids Res. 42, W449-W460. Elsharkawy, A. M., Oakley, F., Lin, F., Packham, G., Mann, D. A., and Mann, J. (2010). The NF-kappaB p50:p50:HDAC-1 repressor complex orchestrates transcriptional inhibition of multiple pro-inflammatory genes. J. Hepatol. 53, 519-527.</div>
</li> <li> <para-num num="[1113]"> </para-num> <div class="description-line" id="p-1098" num="1113">Engreitz, J. M., Pandya-Jones, A., McDonel, P., Shishkin, A., Sirokman, K., Surka, C., Kadri, S., Xing, J., Goren, A., Lander, E. S., Plath, K. &amp; Guttman, M. The Xist lncRNA exploits three-dimensional genome architecture to spread across the X chromosome. <i>Science. </i>341, 1237973, doi:10.1126/science.1237973 (2013). PMCID:3778663.</div>
</li> <li> <para-num num="[1114]"> </para-num> <div class="description-line" id="p-1099" num="1114">Fan, H. C., Fu, G. K., and Fodor, S. P. a. (2015). Combinatorial labeling of single cells for gene expression cytometry. Science. 347, 1258367-1258367.</div>
</li> <li> <para-num num="[1115]"> </para-num> <div class="description-line" id="p-1100" num="1115">Fogli, A. &amp; Boespflug-Tanguy, O. 2006, “The large spectrum of eIF2B-related diseases”, <i>Biochemical Society Transactions, </i>vol. 34, no. Pt 1, pp. 22-29.</div>
</li> <li> <para-num num="[1116]"> </para-num> <div class="description-line" id="p-1101" num="1116">Friedman, J., Hastie, T. &amp; Tibshirani, R. 2001, The elements of statistical learning, Springer series in statistics Springer, Berlin.</div>
</li> <li> <para-num num="[1117]"> </para-num> <div class="description-line" id="p-1102" num="1117">Galonska, C., Smith, Z. D. &amp; Meissner, A. In Vivo and in vitro dynamics of undifferentiated embryonic cell transcription factor 1. <i>Stem Cell Reports. </i>2, 245-252, doi:10.1016/j.stemcr.2014.01.007 (2014). PMCID:3964277.</div>
</li> <li> <para-num num="[1118]"> </para-num> <div class="description-line" id="p-1103" num="1118">Garber, M., Yosef, N., Goren, A., Raychowdhury, R., Thielke, A., Guttman, M., Robinson, J., Minie, B., Chevrier, N., Itzhaki, Z., et al. (2012). A High-Throughput Chromatin Immunoprecipitation Approach Reveals Principles of Dynamic Gene Regulation in Mammals. Mol. Cell 47, 810-822.</div>
</li> <li> <para-num num="[1119]"> </para-num> <div class="description-line" id="p-1104" num="1119">Gat-Viks, I., Chevrier, N., Wilentzik, R., Eisenhaure, T., Raychowdhury, R., Steuerman, Y., Shalek, A. K., Hacohen, N., Amit, I. &amp; Regev, A. Deciphering molecular circuits from genetic variation underlying transcriptional responsiveness to stimuli. <i>Nature biotechnology. </i>31, 342-349, doi:10.1038/nbt.2519 (2013). PMCID:3622156.</div>
</li> <li> <para-num num="[1120]"> </para-num> <div class="description-line" id="p-1105" num="1120">Gilbert, L. A., Horlbeck, M. A., Adamson, B., Villalta, J. E., Chen, Y., Whitehead, E. H., Guimaraes, C., Panning, B., Ploegh, H. L., Bassik, M. C., et al. (2014). Genome-Scale CRISPR-Mediated Control of Gene Repression and Activation. Cell 159, 647-661.</div>
</li> <li> <para-num num="[1121]"> </para-num> <div class="description-line" id="p-1106" num="1121">Gilbert, L. a, Larson, M. H., Morsut, L., Liu, Z., Brar, G. a, Torres, S. E., Stern-Ginossar, N., Brandman, O., Whitehead, E. H., Doudna, J. a, et al. (2013). CRISPR-mediated modular RNA-guided regulation of transcription in eukaryotes. Cell 154, 442-451.</div>
</li> <li> <para-num num="[1122]"> </para-num> <div class="description-line" id="p-1107" num="1122">Gomez, D., Shankman, L. S., Nguyen, A. T., and Owens, G. K. (2013). Detection of histone modifications at specific gene loci in single cells in histological sections. Nature methods 10, 171-177.</div>
</li> <li> <para-num num="[1123]"> </para-num> <div class="description-line" id="p-1108" num="1123">Goodwin, E. C. &amp; Rottman, F. M. 1992, “The 3′-flanking sequence of the bovine growth hormone gene contains novel elements required for efficient and accurate polyadenylation”, <i>The Journal of Biological Chemistry, </i>vol. 267, no. 23, pp. 16330-16334.</div>
</li> <li> <para-num num="[1124]"> </para-num> <div class="description-line" id="p-1109" num="1124">Grün, D. &amp; van Oudenaarden, A. 2015, “Design and Analysis of Single-Cell Sequencing Experiments”, Cell, vol. 163, no. 4, pp. 799-810.</div>
</li> <li> <para-num num="[1125]"> </para-num> <div class="description-line" id="p-1110" num="1125">Gu, W., Crawford, E. D., O'Donovan, B. D., Wilson, M. R., Chow, E. D., Retallack, H. &amp; DeRisi, J. L. 2016, “Depletion of Abundant Sequences by Hybridization (DASH): using Cas9 to remove unwanted high-abundance species in sequencing libraries and molecular counting applications”, <i>Genome Biology, </i>vol. 17, pp. 41.</div>
</li> <li> <para-num num="[1126]"> </para-num> <div class="description-line" id="p-1111" num="1126">Guttman, M., Donaghey, J., Carey, B. W., Garber, M., Grenier, J. K., Munson, G., Young, G., Lucas, A. B., Ach, R., Bruhn, L., Yang, X., Amit, I., Meissner, A., Regev, A., Rinn, J. L., Root, D. E. &amp; Lander, E. S. lincRNAs act in the circuitry controlling pluripotency and differentiation. <i>Nature. </i>477, 295-300, doi:10.1038/nature10398 (2011). PMCID:3175327.</div>
</li> <li> <para-num num="[1127]"> </para-num> <div class="description-line" id="p-1112" num="1127">Haas, B. J., Papanicolaou, A., Yassour, M., Grabherr, M., Blood, P. D., Bowden, J., Couger, M. B., Eccles, D., Li, B., Lieber, M., Macmanes, M. D., Ott, M., Orvis, J., Pochet, N., Strozzi, F., Weeks, N., Westerman, R., William, T., Dewey, C. N., Henschel, R., Leduc, R. D., Friedman, N. &amp; Regev, A. De novo transcript sequence reconstruction from RNA-seq using the Trinity platform for reference generation and analysis. <i>Nat Protoc. </i>8, 1494-1512, doi:10.1038/nprot.2013.084 (2013). PMCID:3875132.</div>
</li> <li> <para-num num="[1128]"> </para-num> <div class="description-line" id="p-1113" num="1128">Haber, J. E., Braberg, H., Wu, Q., Alexander, R., Haase, J., Ryan, C., Lipkin-Moore, Z., Franks-Skiba, K. E., Johnson, T., Shales, M., et al. (2013). Systematic triple-mutant analysis uncovers functional connectivity between pathways involved in chromosome regulation. Cell Rep. 3, 2168-2178.</div>
</li> <li> <para-num num="[1129]"> </para-num> <div class="description-line" id="p-1114" num="1129">Hacisuleyman, E., Goff, L. A., Trapnell, C., Williams, A., Henao-Mejia, J., Sun, L., McClanahan, P., Hendrickson, D. G., Sauvageau, M., Kelley, D. R., Morse, M., Engreitz, J., Lander, E. S., Guttman, M., Lodish, H. F., Flavell, R., Raj, A. &amp; Rinn, J. L. Topological organization of multichromosomal regions by the long intergenic noncoding RNA Firre. <i>Nat Struct Mol Biol. </i>21, 198-206, doi:10.1038/nsmb.2764 (2014). PMCID:3950333.</div>
</li> <li> <para-num num="[1130]"> </para-num> <div class="description-line" id="p-1115" num="1130">Haldimann, A. &amp; Wanner, B. L. 2001, “Conditional-replication, integration, excision, and retrieval plasmid-host systems for gene structure-function studies of bacteria”, <i>Journal of Bacteriology, </i>vol. 183, no. 21, pp. 6384-6393.</div>
</li> <li> <para-num num="[1131]"> </para-num> <div class="description-line" id="p-1116" num="1131">Hamanaka, R. B., Bennett, B. S., Cullinan, S. B. &amp; Diehl, J. A. 2005, “PERK and GCN2 Contribute to eIF2α Phosphorylation and Cell Cycle Arrest after Activation of the Unfolded Protein Response Pathway”, <i>Molecular Biology of the Cell, </i>vol. 16, no. 12, pp. 5493-5501.</div>
</li> <li> <para-num num="[1132]"> </para-num> <div class="description-line" id="p-1117" num="1132">Han, J., Back, S. H., Hur, J., Lin, Y., Gildersleeve, R., Shan, J., Yuan, C. L., Krokowski, D., Wang, S., Hatzoglou, M., et al 2013, “ER-stress-induced transcriptional regulation increases protein synthesis leading to cell death”, <i>Nature Cell Biology, </i>vol. 15, no. 5, pp. 481-490.</div>
</li> <li> <para-num num="[1133]"> </para-num> <div class="description-line" id="p-1118" num="1133">Hartl, D. L. (2014). What can Applicants learn from fitness landscapes? Curr. Opin. Microbiol. 21, 51-57.</div>
</li> <li> <para-num num="[1134]"> </para-num> <div class="description-line" id="p-1119" num="1134">Heckl, D., Kowalczyk, M. S., Yudovich, D., Belizaire, R., Puram, R. V., McConkey, M. E., Thielke, A., Aster, J. C., Regev, A. &amp; Ebert, B. L. Generation of mouse models of myeloid malignancy with combinatorial genetic lesions using CRISPR-Cas9 genome editing. <i>Nature biotechnology. </i>32,941-946, doi:10.1038/nbt.2951 (2014). PMCID:4160386.</div>
</li> <li> <para-num num="[1135]"> </para-num> <div class="description-line" id="p-1120" num="1135">Heimberg, G., Bhatnagar, R., El-Samad, H., and Thomson, M. (2016). Low Dimensionality in Gene Expression Data Enables the Accurate Extraction of Transcriptional Programs from Shallow Sequencing. Cell Syst. 2, 239-250.</div>
</li> <li> <para-num num="[1136]"> </para-num> <div class="description-line" id="p-1121" num="1136">Helft, J., Böttcher, J., Chakravarty, P., Zelenay, S., Huotari, J., Schraml, B. U., Goubau, D., and Reis e Sousa, C. (2015). GM-CSF Mouse Bone Marrow Cultures Comprise a Heterogeneous Population of CD11c+MHCII+ Macrophages and Dendritic Cells. Immunity 42, 1197-1211.</div>
</li> <li> <para-num num="[1137]"> </para-num> <div class="description-line" id="p-1122" num="1137">Hetz, C. 2012, “The unfolded protein response: controlling cell fate decisions under ER stress and beyond”, <i>Nature Reviews. Molecular Cell Biology, </i>vol. 13, no. 2, pp. 89-102.</div>
</li> <li> <para-num num="[1138]"> </para-num> <div class="description-line" id="p-1123" num="1138">Horlbeck, M. A., Gilbert, L. A., Villalta, J. E., Adamson, B., Pak, R. A., Chen, Y., Fields, A. P., Park, C. Y., Corn, J. E. &amp; Kampmann, M. 2016, “Compact and highly active next-generation libraries for CRISPR-mediated gene repression and activation”, <i>eLife, </i>vol. 5, pp. e19760.</div>
</li> <li> <para-num num="[1139]"> </para-num> <div class="description-line" id="p-1124" num="1139">Hu, S., Ni, W., Hazi, W., Zhang, H., Zhang, N., Meng, R. &amp; Chen, C. 2011, “Cloning and functional analysis of sheep U6 promoters”, <i>Animal Biotechnology, </i>vol. 22, no. 3, pp. 170-174.</div>
</li> <li> <para-num num="[1140]"> </para-num> <div class="description-line" id="p-1125" num="1140">Huang, D. W., Sherman, B. T. &amp; Lempicki, R. A. 2009a, “Bioinformatics enrichment tools: paths toward the comprehensive functional analysis of large gene lists”, <i>Nucleic Acids Research, </i>vol. 37, no. 1, pp. 1-13.</div>
</li> <li> <para-num num="[1141]"> </para-num> <div class="description-line" id="p-1126" num="1141">Huang, D. W., Sherman, B. T. &amp; Lempicki, R. A. 2009b, “Systematic and integrative analysis of large gene lists using DAVID bioinformatics resources”, <i>Nature Protocols, </i>vol. 4, no. 1, pp. 44-57.</div>
</li> <li> <para-num num="[1142]"> </para-num> <div class="description-line" id="p-1127" num="1142">Hughes, T. R., Marton, M. J., Jones, A. R., Roberts, C. J., Stoughton, R., Armour, C. D., Bennett, H. a, Coffey, E., Dai, H., He, Y. D., et al. (2000). Functional Discovery via a Compendium of Expression Profiles. Cell 102, 109-126.</div>
</li> <li> <para-num num="[1143]"> </para-num> <div class="description-line" id="p-1128" num="1143">Jaitin, D. A., Kenigsberg, E., Keren-Shaul, H., Elefant, N., Paul, F., Zaretsky, I., Mildner, A., Cohen, N., Jung, S., Tanay, A. &amp; Amit, I. Massively parallel single-cell RNA-seq for marker-free decomposition of tissues into cell types. <i>Science. </i>343, 776-779, doi:10.1126/science.1247651 (2014). PMCID:4412462.</div>
</li> <li> <para-num num="[1144]"> </para-num> <div class="description-line" id="p-1129" num="1144">Janssen, K. P., Knez, K., Spasic, D., and Lammertyn, J. (2013). Nucleic acids for ultra-sensitive protein detection. Sensors 13,1353-1384.</div>
</li> <li> <para-num num="[1145]"> </para-num> <div class="description-line" id="p-1130" num="1145">Jiang, F., Zhou, K., Ma, L., Gressel, S. &amp; Doudna, J. A. 2015, “STRUCTURAL BIOLOGY. A Cas9-guide RNA complex preorganized for target DNA recognition”, <i>Science </i>(New York, N.Y.), vol. 348, no. 6242, pp. 1477-1481.</div>
</li> <li> <para-num num="[1146]"> </para-num> <div class="description-line" id="p-1131" num="1146">Jin, F., Hazbun, T., Michaud, G. A., Salcius, M., Predki, P. F., Fields, S., and Huang, J. (2006). A pooling-deconvolution strategy for biological network elucidation. Nat Methods 3, 183-189.</div>
</li> <li> <para-num num="[1147]"> </para-num> <div class="description-line" id="p-1132" num="1147">Joensson, H. N., and Andersson Svahn, H. (2012). Droplet Microfluidics-A Tool for Single-Cell Analysis. Angew. Chemie Int. Ed. 51, 12176-12192.</div>
</li> <li> <para-num num="[1148]"> </para-num> <div class="description-line" id="p-1133" num="1148">Jonikas, M. C., Collins, S. R., Denic, V., Oh, E., Quan, E. M., Schmid, V., Weibezahn, J., Schwappach, B., Walter, P., Weissman, J. S., et al 2009, “Comprehensive characterization of genes required for protein folding in the endoplasmic reticulum”, <i>Science </i>(New York, N.Y.), vol. 323, no. 5922, pp. 1693-1697.</div>
</li> <li> <para-num num="[1149]"> </para-num> <div class="description-line" id="p-1134" num="1149">Jovanovic, M., Rooney, M. S., Mertins, P., Przybylski, D., Chevrier, N., Satija, R., Rodriguez, E. H., Fields, A. P., Schwartz, S., Raychowdhury, R., Mumbach, M. R., Eisenhaure, T., Rabani, M., Gennert, D., Lu, D., Delorey, T., Weissman, J. S., Carr, S. A., Hacohen, N. &amp; Regev, A. Dynamic profiling of the protein life cycle in response to pathogens. <i>Science. </i>347, 1259038, doi:10.1126/science.1259038 (2015). PMCID:PMC Journal—In Process.</div>
</li> <li> <para-num num="[1150]"> </para-num> <div class="description-line" id="p-1135" num="1150">Jovanovic, M., Rooney, M. S., Mertins, P., Przybylski, D., Chevrier, N., Satija, R., Rodriguez, E. H., Fields, A. P., Schwartz, S., Raychowdhury, R., et al. (2015). Dynamic profiling of the protein life cycle in response to pathogens. Science (80-.). 347, 1259038-1259038.</div>
</li> <li> <para-num num="[1151]"> </para-num> <div class="description-line" id="p-1136" num="1151">Kabadi, A. M., Ousterout, D. G., Hilton, I. B., and Gersbach, C. A. (2014). Multiplex CRISPR/Cas9-based genome engineering from a single lentiviral vector. Nucleic Acids Res. 42, 1-11.</div>
</li> <li> <para-num num="[1152]"> </para-num> <div class="description-line" id="p-1137" num="1152">Kampmann, M., Bassik, M. C., and Weissman, J. S. (2014). Functional genomics platform for pooled screening and generation of mammalian genetic interaction maps. Nat. Protoc. 9, 1825-1847.</div>
</li> <li> <para-num num="[1153]"> </para-num> <div class="description-line" id="p-1138" num="1153">Kanda, S., Yanagitani, K., Yokota, Y., Esaki, Y. &amp; Kohno, K. 2016, “Autonomous translational pausing is required for XBP1u mRNA recruitment to the ER via the SRP pathway”, <i>Proceedings of the National Academy of Sciences of the United States of America, </i>vol. 113, no. 40, pp. E5895.</div>
</li> <li> <para-num num="[1154]"> </para-num> <div class="description-line" id="p-1139" num="1154">Kantlehner, M., Kirchner, R., Hartmann, P., Ellwart, J. W., Alunni-Fabbroni, M., and Schumacher, A. (2011). A high-throughput DNA methylation analysis of a single cell. Nucleic acids research 39, e44.</div>
</li> <li> <para-num num="[1155]"> </para-num> <div class="description-line" id="p-1140" num="1155">Kearns, N. A., Genga, R. M., Enuameh, M. S., Garber, M., Wolfe, S. A. &amp; Maehr, R. Cas9 effector-mediated regulation of transcription and differentiation in human pluripotent stem cells. <i>Development. </i>141,219-223, doi:10.1242/dev.103341 (2014). PMCID:3865759.</div>
</li> <li> <para-num num="[1156]"> </para-num> <div class="description-line" id="p-1141" num="1156">Kelley, D. &amp; Rinn, J. Transposable elements reveal a stem cell-specific class of long noncoding RNAs. <i>Genome Biol. </i>13, R107, doi:10.1186/gb-2012-13-11-r107 (2012). PMCID:3580499.</div>
</li> <li> <para-num num="[1157]"> </para-num> <div class="description-line" id="p-1142" num="1157">Kemmeren, P., Sameith, K., Van De Pasch, L. A. L., Benschop, J. J., Lenstra, T. L., Margaritis, T., O′Duibhir, E., Apweiler, E., Van Wageningen, S., Ko, C.W., et al. (2014). Large-scale genetic perturbations reveal regulatory networks and an abundance of gene-specific repressors. Cell 157, 740-752.</div>
</li> <li> <para-num num="[1158]"> </para-num> <div class="description-line" id="p-1143" num="1158">Kharchenko, P. V, Silberstein, L., and Scadden, D. T. (2014). Bayesian approach to single-cell differential expression analysis. Nat. Methods 11, 740-742.</div>
</li> <li> <para-num num="[1159]"> </para-num> <div class="description-line" id="p-1144" num="1159">Klein, A. M., Mazutis, L., Akartuna, I., Tallapragada, N., Veres, A., Li, V., Peshkin, L., Weitz, D. A., and Kirschner, M. W. (2015). Droplet barcoding for single-cell transcriptomics applied to embryonic stem cells. Cell 161, 1187-1201.</div>
</li> <li> <para-num num="[1160]"> </para-num> <div class="description-line" id="p-1145" num="1160">Komor, A. C., Kim, Y. B., Packer, M. S., Zuris, J. A. &amp; Liu, D. R. 2016, “Programmable editing of a target base in genomic DNA without double-stranded DNA cleavage”, <i>Nature, </i>vol. 533, no. 7603, pp. 420-424.</div>
</li> <li> <para-num num="[1161]"> </para-num> <div class="description-line" id="p-1146" num="1161">Konermann, S., Brigham, M. D., Trevino, A. E., Joung, J., Abudayyeh, O. O., Barcena, C., Hsu, P. D., Habib, N., Gootenberg, J. S., Nishimasu, H., et al. (2014). Genome-scale transcriptional activation by an engineered CRISPR-Cas9 complex. Nature 517, 583-588.</div>
</li> <li> <para-num num="[1162]"> </para-num> <div class="description-line" id="p-1147" num="1162">Kowalczyk, M. S., Tirosh, I., Heckl, D., Rao, T. N., Dixit, A., Haas, B. J., Schneider, R. K., Wagers, A. J., Ebert, B. L., and Regev, A. (2015). Single-cell RNA-seq reveals changes in cell cycle and differentiation programs upon aging of hematopoietic stem cells. Genome Res. 25, 1860-1872.</div>
</li> <li> <para-num num="[1163]"> </para-num> <div class="description-line" id="p-1148" num="1163">Kuleshov, M. V., Jones, M. R., Rouillard, A. D., Fernandez, N. F., Duan, Q., Wang, Z., Koplev, S., Jenkins, S. L., Jagodnik, K. M., Lachmann, A., et al 2016, “Enrichr: a comprehensive gene set enrichment analysis web server 2016 update”, <i>Nucleic Acids Research, </i>vol. 44, no. W1, pp. 90.</div>
</li> <li> <para-num num="[1164]"> </para-num> <div class="description-line" id="p-1149" num="1164">Kumar, R. M., Cahan, P., Shalek, A. K., Satija, R., DaleyKeyser, A. J., Li, H., Zhang, J., Pardee, K., Gennert, D., Trombetta, J. J., Ferrante, T. C., Regev, A., Daley, G. Q. &amp; Collins, J. J. Deconstructing transcriptional heterogeneity in pluripotent stem cells. <i>Nature. </i>516, 56-61, doi:10.1038/nature13920 (2014). PMCID:4256722.</div>
</li> <li> <para-num num="[1165]"> </para-num> <div class="description-line" id="p-1150" num="1165">Labzin, L. I., Schmidt, S. V, Masters, S. L., Beyer, M., Krebs, W., Klee, K., Stahl, R., Lütjohann, D., Schultze, J. L., Latz, E., et al. (2015). ATF3 Is a Key Regulator of Macrophage IFN Responses. J. Immunol. 195, 4446-4455.</div>
</li> <li> <para-num num="[1166]"> </para-num> <div class="description-line" id="p-1151" num="1166">Lamb, J., Crawford, E. D., Peck, D., Modell, J. W., Blat, I. C., Wrobel, M. J., Lerner, J., Brunet, J., Subramanian, A., Ross, K. N., et al. (2006). The Connectivity Map: Using. Science (80-.). 313, 1929-1935.</div>
</li> <li> <para-num num="[1167]"> </para-num> <div class="description-line" id="p-1152" num="1167">Lambeth, L. S., Wise, T. G., Moore, R. J., Muralitharan, M. S. &amp; Doran, T. J. 2006, “Comparison of bovine RNA polymerase III promoters for short hairpin RNA expression”, Animal Genetics, vol. 37, no. 4, pp. 369-372.</div>
</li> <li> <para-num num="[1168]"> </para-num> <div class="description-line" id="p-1153" num="1168">Lara-Astiaso, D., Weiner, A., Lorenzo-Vivas, E., Zaretsky, I., Jaitin, D. A., David, E., Keren-Shaul, H., Mildner, A., Winter, D., Jung, S., Friedman, N. &amp; Amit, I. Immunogenetics. Chromatin state dynamics during blood formation. Science. 345, 943-949, doi:10.1126/science.1256271 (2014). PMCID:4412442.</div>
</li> <li> <para-num num="[1169]"> </para-num> <div class="description-line" id="p-1154" num="1169">Laufer, C., Fischer, B., Billmann, M., Huber, W., and Boutros, M. (2013). Mapping genetic interactions in human cancer cells with RNAi and multiparametric phenotyping. Nat. Methods 10, 427-431.</div>
</li> <li> <para-num num="[1170]"> </para-num> <div class="description-line" id="p-1155" num="1170">Lawrence, M. S., Stojanov, P., Mermel, C. H., Robinson, J. T., Garraway, L. A., Golub, T. R., Meyerson, M., Gabriel, S. B., Lander, E. S., and Getz, G. (2014). Discovery and saturation analysis of cancer genes across 21 tumour types. Nature 505, 495-501.</div>
</li> <li> <para-num num="[1171]"> </para-num> <div class="description-line" id="p-1156" num="1171">Lee, A., Iwakoshi, N. N. &amp; Glimcher, L. H. 2003, “XBP-1 regulates a subset of endoplasmic reticulum resident chaperone genes in the unfolded protein response”, <i>Molecular and Cellular Biology,</i>vol. 23, no. 21, pp. 7448-7459.</div>
</li> <li> <para-num num="[1172]"> </para-num> <div class="description-line" id="p-1157" num="1172">Lee, M. N., Ye, C., Villani, A. C., Raj, T., Li, W., Eisenhaure, T. M., Imboywa, S. H., Chipendo, P. I., Ran, F. A., Slowikowski, K., Ward, L. D., Raddassi, K., McCabe, C., Lee, M. H., Frohlich, I. Y., Hafler, D. A., Kellis, M., Raychaudhuri, S., Zhang, F., Stranger, B. E., Benoist, C. O., De Jager, P. L., Regev, A. &amp; Hacohen, N. Common genetic variants modulate pathogen-sensing responses in human dendritic cells. <i>Science. </i>343, 1246980, doi:10.1126/science.1246980 (2014). PMCID:4124741.</div>
</li> <li> <para-num num="[1173]"> </para-num> <div class="description-line" id="p-1158" num="1173">Liang, S., Zhang, W., McGrath, B. C., Zhang, P. &amp; Cavener, D. R. 2006, “PERK (eIF2alpha kinase) is required to activate the stress-activated MAPKs and induce the expression of immediate-early genes upon disruption of ER calcium homoeostasis”, <i>The Biochemical Journal, </i>vol. 393, no. Pt 1, pp. 201-209.</div>
</li> <li> <para-num num="[1174]"> </para-num> <div class="description-line" id="p-1159" num="1174">Liberali, P., Snijder, B., and Pelkmans, L. (2014). Single-cell and multivariate approaches in genetic perturbation screens. Nat. Rev. <i>Genet. </i>16, 18-32.</div>
</li> <li> <para-num num="[1175]"> </para-num> <div class="description-line" id="p-1160" num="1175">Lin, J. H., Li, H., Yasumura, D., Cohen, H. R., Zhang, C., Panning, B., Shokat, K. M., Lavail, M. M. &amp; Walter, P. 2007, “IRE1 signaling affects cell fate during the unfolded protein response”, <i>Science </i>(New York, N.Y.), vol. 318, no. 5852, pp. 944-949.</div>
</li> <li> <para-num num="[1176]"> </para-num> <div class="description-line" id="p-1161" num="1176">Lin, Z., Chen, M. &amp; Ma, Y. 2010, “The augmented lagrange multiplier method for exact recovery of corrupted low-rank matrices”, arXiv preprint arXiv: 1009.5055.</div>
</li> <li> <para-num num="[1177]"> </para-num> <div class="description-line" id="p-1162" num="1177">Lorthongpanich, C., Cheow, L. F., Balu, S., Quake, S. R., Knowles, B. B., Burkholder, W. F., Solter, D., and Messerschmidt, D. M. (2013). Single-cell DNA-methylation analysis reveals epigenetic chimerism in preimplantation embryos. <i>Science </i>341, 1110-1112.</div>
</li> <li> <para-num num="[1178]"> </para-num> <div class="description-line" id="p-1163" num="1178">Lutz, R. &amp; Bujard, H. 1997, “Independent and tight regulation of transcriptional units in <i>Escherichia coli </i>via the LacR/O, the TetR/O and AraC/I1-I2 regulatory elements”, <i>Nucleic Acids Research, </i>vol. 25, no. 6, pp. 1203-1210.</div>
</li> <li> <para-num num="[1179]"> </para-num> <div class="description-line" id="p-1164" num="1179">Macosko, E. Z., Basu, A., Satija, R., Nemesh, J., Shekhar, K., Goldman, M., Tirosh, I., Bialas, A. R., Kamitaki, N., Martersteck, E. M., et al. (2015). Highly Parallel Genome-wide Expression Profiling of Individual Cells Using Nanoliter Droplets. Cell 161, 1202-1214.</div>
</li> <li> <para-num num="[1180]"> </para-num> <div class="description-line" id="p-1165" num="1180">Makarova, K. S., Haft, D. H., Barrangou, R., Brouns, S. J. J., Charpentier, E., Horvath, P., Moineau, S., Mojica, F. J. M., Wolf, Y. I., Yakunin, A. F., et al. (2011). Evolution and Classification of the CRISPR-Cas Systems. Nat. Rev. Microbiol. 9, 467-477.</div>
</li> <li> <para-num num="[1181]"> </para-num> <div class="description-line" id="p-1166" num="1181">Manolio, T. A., Collins, F. S., Cox, N. J., Goldstein, D. B., Hindorff, L. A., Hunter, D. J., McCarthy, M. I., Ramos, E. M., Cardon, L. R., Chakravarti, A., et al. (2009). Finding the missing heritability of complex diseases. Nature 461, 747-753.</div>
</li> <li> <para-num num="[1182]"> </para-num> <div class="description-line" id="p-1167" num="1182">Martincorena, I., and Campbell, P. J. (2015). Somatic mutation in cancer and normal cells. Science (80-.). 349, 1483-1489.</div>
</li> <li> <para-num num="[1183]"> </para-num> <div class="description-line" id="p-1168" num="1183">Meerbrey, K. L., Hu, G., Kessler, J. D., Roarty, K., Li, M. Z., Fang, J. E., Herschkowitz, J. I., Burrows, A. E., Ciccia, A., Sun, T., et al 2011, “The pINDUCER lentiviral toolkit for inducible RNA interference in vitro and in vivo”, <i>Proceedings of the National Academy of Sciences of the United States of America, </i>vol. 108, no. 9, pp. 3665-3670.</div>
</li> <li> <para-num num="[1184]"> </para-num> <div class="description-line" id="p-1169" num="1184">Meier, J. A., and Lamer, A. C. (2014). Toward a new STATe: the role of STATs in mitochondrial function. Semin. Immunol. 26, 20-28.</div>
</li> <li> <para-num num="[1185]"> </para-num> <div class="description-line" id="p-1170" num="1185">Melnikov, A., Murugan, A., Zhang, X., Tesileanu, T., Wang, L., Rogov, P., Feizi, S., Gnirke, A., Callan, C. G., Kinney, J. B., et al. (2012). Systematic dissection and optimization of inducible enhancers in human cells using a massively parallel reporter assay. Nat. Biotechnol. 30, 271-277.</div>
</li> <li> <para-num num="[1186]"> </para-num> <div class="description-line" id="p-1171" num="1186">Muller-Kuller, U., Ackermann, M., Kolodziej, S., Brendel, C., Fritsch, J., Lachmann, N., Kunkel, H., Lausen, J., Schambach, A., Moritz, T., et al 2015, “A minimal ubiquitous chromatin opening element (UCOE) effectively prevents silencing of juxtaposed heterologous promoters by epigenetic remodeling in multipotent and pluripotent stem cells”, <i>Nucleic Acids Research, </i>pp. gkv019.</div>
</li> <li> <para-num num="[1187]"> </para-num> <div class="description-line" id="p-1172" num="1187">Munoz, D. M., Cassiani, P. J., Li, L., Billy, E., Korn, J. M., Jones, M. D., Golji, J., Ruddy, D. A., Yu, K., McAllister, G., et al 2016, “CRISPR Screens Provide a Comprehensive Assessment of Cancer Vulnerabilities but Generate False-Positive Hits for Highly Amplified Genomic Regions”, <i>Cancer Discovery, </i>vol. 6, no. 8, pp. 900-913.</div>
</li> <li> <para-num num="[1188]"> </para-num> <div class="description-line" id="p-1173" num="1188">Na, Y. R., Kim, S. Y., Gaublomme, J. T., Shalek, A. K., Jorgolli, M., Park, H. &amp; Yang, E. G. Probing enzymatic activity inside living cells using a nanowire-cell “sandwich” assay. <i>Nano Lett. </i>13,153-158, doi:10.1021/n13037068 (2013). PMCID:3541459.</div>
</li> <li> <para-num num="[1189]"> </para-num> <div class="description-line" id="p-1174" num="1189">Nagano, T., Lubling, Y., Stevens, T. J., Schoenfelder, S., Yaffe, E., Dean, W., Laue, E. D., Tanay, A., and Fraser, P. (2013). Single-cell Hi-C reveals cell-to-cell variability in chromosome structure. Nature 502, 59-64.</div>
</li> <li> <para-num num="[1190]"> </para-num> <div class="description-line" id="p-1175" num="1190">Neumann, B., Walter, T., Hériché, J. -K., Bulkescher, J., Erfle, H., Conrad, C., Rogers, P., Poser, I., Held, M., Liebel, U., et al. (2010). Phenotypic profiling of the human genome by time-lapse microscopy reveals cell division genes. Nature 464, 721-727.</div>
</li> <li> <para-num num="[1191]"> </para-num> <div class="description-line" id="p-1176" num="1191">Nishimasu, H., Ran, F. A., Hsu, P. D., Konermann, S., Shehata, S. I., Dohmae, N., Ishitani, R., Zhang, F. &amp; Nureki, O. 2014, “Crystal structure of Cas9 in complex with guide RNA and target DNA”, <i>Cell, </i>vol. 156, no. 5, pp. 935-949.</div>
</li> <li> <para-num num="[1192]"> </para-num> <div class="description-line" id="p-1177" num="1192">Nissim, L., Perli, S. D., Fridkin, A., Perez-Pinera, P. &amp; Lu, T. K. 2014, “Multiplexed and programmable regulation of gene networks with an integrated RNA and CRISPR/Cas toolkit in human cells”, <i>Molecular Cell, </i>vol. 54, no. 4, pp. 698-710.</div>
</li> <li> <para-num num="[1193]"> </para-num> <div class="description-line" id="p-1178" num="1193">Okabe, Y., and Medzhitov, R. (2014). Tissue-Specific Signals Control Reversible Program of Localization and Functional Polarization of Macrophages. Cell 157, 832-844.</div>
</li> <li> <para-num num="[1194]"> </para-num> <div class="description-line" id="p-1179" num="1194">Pardon, E., Laeremans, T., Triest, S., Rasmussen, S. G., Wohlkonig, A., Ruf, A., Muyldermans, S., Hol, W. G., Kobilka, B. K., and Steyaert, J. (2014). A general protocol for the generation of Nanobodies for structural biology. Nature protocols 9, 674-693.</div>
</li> <li> <para-num num="[1195]"> </para-num> <div class="description-line" id="p-1180" num="1195">Parnas, O., Jovanovic, M., Eisenhaure, T. M., Herbst, R. H., Dixit, A., Ye, C. J., Przybylski, D., Platt, R. J., Tirosh, I., Sanjana, N. E., et al. (2015). A Genome-wide CRISPR Screen in Primary Immune Cells to Dissect Regulatory Networks. Cell 162, 675-686.</div>
</li> <li> <para-num num="[1196]"> </para-num> <div class="description-line" id="p-1181" num="1196">Perfetto, S. P., Chattopadhyay, P. K., and Roederer, M. (2004). Seventeen-colour flow cytometry: unravelling the immune system. Nature reviews Immunology 4, 648-655.</div>
</li> <li> <para-num num="[1197]"> </para-num> <div class="description-line" id="p-1182" num="1197">Phillips, P. C. (2008). Epistasis—the essential role of gene interactions in the structure and evolution of genetic systems.</div>
</li> <li> <para-num num="[1198]"> </para-num> <div class="description-line" id="p-1183" num="1198">Platt, R. J., Chen, S., Zhou, Y., Yim, M. J., Swiech, L., Kempton, H. R., Dahlman, J. E., Parnas, O., Eisenhaure, T. M., Jovanovic, M., et al. (2014). CRISPR-Cas9 Knockin Mice for Genome Editing and Cancer Modeling. Cell 159, 440-455.</div>
</li> <li> <para-num num="[1199]"> </para-num> <div class="description-line" id="p-1184" num="1199">Plumb, R., Zhang, Z., Appathurai, S. &amp; Mariappan, M. 2015, “A functional link between the co-translational protein translocation pathway and the UPR”, <i>eLife, </i>vol. 4.</div>
</li> <li> <para-num num="[1200]"> </para-num> <div class="description-line" id="p-1185" num="1200">Pollen, A. A., Nowakowski, T. J., Shuga, J., Wang, X., Leyrat, A. A., Lui, J. H., Li, N., Szpankowski, L., Fowler, B., Chen, P., et al 2014, “Low-coverage single-cell mRNA sequencing reveals cellular heterogeneity and activated signaling pathways in developing cerebral cortex”, Nature Biotechnology, vol. 32, no. 10, pp. 1053-1058.</div>
</li> <li> <para-num num="[1201]"> </para-num> <div class="description-line" id="p-1186" num="1201">Qi, L. S., Larson, M. H., Gilbert, L. A., Doudna, J. A., Weissman, J. S., Arkin, A. P., and Lim, W. A. (2013). Repurposing CRISPR as an RNA-Guided Platform for Sequence-Specific Control of Gene Expression. Cell 152, 1173-1183.</div>
</li> <li> <para-num num="[1202]"> </para-num> <div class="description-line" id="p-1187" num="1202">Rabani, M., Raychowdhury, R., Jovanovic, M., Rooney, M., Stumpo, D. J., Pauli, A., Hacohen, N., Schier, A. F., Blackshear, P. J., Friedman, N., Amit, I. &amp; Regev, A. High-resolution sequencing and modeling identifies distinct dynamic RNA regulatory strategies. <i>Cell. </i>159, 1698-1710, doi:10.1016/j.cell2014.11.015 (2014). PMCID:4272607.</div>
</li> <li> <para-num num="[1203]"> </para-num> <div class="description-line" id="p-1188" num="1203">Rajagopal, N., Srinivasan, S., Kooshesh, K., Guo, Y., Edwards, M. D., Banerjee, B., Syed, T., Emons, B. J. M., Gifford, D. K., and Sherwood, R. I. (2016). High-throughput mapping of regulatory DNA. Nat. Biotechnol. 34, 167-174.</div>
</li> <li> <para-num num="[1204]"> </para-num> <div class="description-line" id="p-1189" num="1204">Ramsauer, K., Farlik, M., Zupkovitz, G., Seiser, C., Kroger, A., Hauser, H., and Decker, T. (2007). Distinct modes of action applied by transcription factors STAT1 and IRF1 to initiate transcription of the IFN-gamma-inducible gbp2 gene. Proc. Natl. Acad. Sci. U.S.A. 104, 2849-2854.</div>
</li> <li> <para-num num="[1205]"> </para-num> <div class="description-line" id="p-1190" num="1205">Ram, O., Goren, A., Amit, I., Shoresh, N., Yosef, N., Ernst, J., Kellis, M., Gymrek, M., Issner, R., Coyne, M., Durham, T., Zhang, X., Donaghey, J., Epstein, C. B., Regev, A. &amp; Bernstein, B. E. Combinatorial patterning of chromatin regulators uncovered by genome-wide location analysis in human cells. <i>Cell. </i>147, 1628-1639, doi:10.1016/j.cell.2011.09.057 (2011). PMCID:3312319.</div>
</li> <li> <para-num num="[1206]"> </para-num> <div class="description-line" id="p-1191" num="1206">Ran, F. A., Cong, L., Yan, W. X., Scott, D. A., Gootenberg, J.S., Kriz, A. J., Zetsche, B., Shalem, O., Wu, X., Makarova, K. S., et al. (2015). In vivo genome editing using <i>Staphylococcus aureus </i>Cas9. Nature 520, 186-191.</div>
</li> <li> <para-num num="[1207]"> </para-num> <div class="description-line" id="p-1192" num="1207">Ron, D. &amp; Walter, P. 2007, “Signal integration in the endoplasmic reticulum unfolded protein response”, <i>Nature Reviews. Molecular Cell Biology, </i>vol. 8, no. 7, pp. 519-529.</div>
</li> <li> <para-num num="[1208]"> </para-num> <div class="description-line" id="p-1193" num="1208">Rosvall, M., and Bergstrom, C. T. (2008). Maps of random walks on complex networks reveal community structure. Proc. Natl. Acad. Sci. 105, 1118-1123.</div>
</li> <li> <para-num num="[1209]"> </para-num> <div class="description-line" id="p-1194" num="1209">Sack, L. M., Davoli, T., Xu, Q., Li, M. Z. &amp; Elledge, S. J. 2016, “Sources of Error in Mammalian Genetic Screens”, <i>G</i>3 (<i>Bethesda, Md.</i>), vol. 6, no. 9, pp. 2781-2790.</div>
</li> <li> <para-num num="[1210]"> </para-num> <div class="description-line" id="p-1195" num="1210">Sackton, T. B., and Hartl, D. L. (2016). Perspective Genotypic Context and Epistasis in Individuals and Populations. Cell 166, 279-287.</div>
</li> <li> <para-num num="[1211]"> </para-num> <div class="description-line" id="p-1196" num="1211">Satija, R., Farrell, J. A., Gennert, D., Schier, A. F. &amp; Regev, A. Spatial reconstruction of single-cell gene expression data. <i>Nature biotechnology. </i>33, 495-502, doi:10.1038/nbt.3192 (2015).</div>
</li> <li> <para-num num="[1212]"> </para-num> <div class="description-line" id="p-1197" num="1212">Sauvageau, M., Goff, L. A., Lodato, S., Bonev, B., Groff, A. F., Gerhardinger, C., Sanchez-Gomez, D. B., Hacisuleyman, E., Li, E., Spence, M., Liapis, S. C., Mallard, W., Morse, M., Swerdel, M. R., D'Ecclessis, M. F., Moore, J. C., Lai, V., Gong, G., Yancopoulos, G. D., Frendewey, D., Kellis, M., Hart, R. P., Valenzuela, D. M., Arlotta, P. &amp; Rinn, J. L. Multiple knockout mouse models reveal lincRNAs are required for life and brain development. <i>eLife. </i>2, e01749, doi:10.7554/eLife.01749 (2013). PMCID:3874104.</div>
</li> <li> <para-num num="[1213]"> </para-num> <div class="description-line" id="p-1198" num="1213">Sawyers, C. (2004). Targeted cancer therapy. Nature 432, 294-297.</div>
</li> <li> <para-num num="[1214]"> </para-num> <div class="description-line" id="p-1199" num="1214">Scholkopf, B., Platt, J. C., Shawe-Taylor, J., Smola, A. J. &amp; Williamson, R. C. 2001, “Estimating the support of a high-dimensional distribution”, <i>Neural computation, </i>vol. 13, no. 7, pp. 1443-1471.</div>
</li> <li> <para-num num="[1215]"> </para-num> <div class="description-line" id="p-1200" num="1215">Schwartz, S., Agarwala, S. D., Mumbach, M. R., Jovanovic, M., Mertins, P., Shishkin, A., Tabach, Y., Mikkelsen, T. S., Satija, R., Ruvkun, G., Carr, S. A., Lander, E. S., Fink, G. R. &amp; Regev, A. High-resolution mapping reveals a conserved, widespread, dynamic mRNA methylation program in yeast meiosis. <i>Cell. </i>155, 1409-1421, doi:10.1016/j.cell2013.10.047 (2013). PMCID:3956118.</div>
</li> <li> <para-num num="[1216]"> </para-num> <div class="description-line" id="p-1201" num="1216">Schwartz, S., Bernstein, D. A., Mumbach, M. R., Jovanovic, M., Herbst, R. H., Leon-Ricardo, B. X., Engreitz, J. M., Guttman, M., Satija, R., Lander, E. S., Fink, G. &amp; Regev, A. Transcriptome-wide mapping reveals widespread dynamic-regulated pseudouridylation of ncRNA and mRNA. <i>Cell. </i>159, 148-162, doi:10.1016/j.cell.2014.08.028 (2014). PMCID:4180118.</div>
</li> <li> <para-num num="[1217]"> </para-num> <div class="description-line" id="p-1202" num="1217">Schwartz, S., Mumbach, M. R., Jovanovic, M., Wang, T., Maciag, K., Bushkin, G. G., Mertins, P., Ter-Ovanesyan, D., Habib, N., Cacchiarelli, D., Sanjana, N. E., Freinkman, E., Pacold, M. E., Satija, R., Mikkelsen, T. S., Hacohen, N., Zhang, F., Carr, S. A., Lander, E. S. &amp; Regev, A. Perturbation of m6A writers reveals two distinct classes of mRNA methylation at internal and 5′ sites. <i>Cell reports. </i>8, 284-296, doi:10.1016/j.celrep.2014.05.048 (2014). PMCID:4142486.</div>
</li> <li> <para-num num="[1218]"> </para-num> <div class="description-line" id="p-1203" num="1218">Shahni, R., Cale, C. M., Anderson, G., Osellame, L. D., Hambleton, S., Jacques, T. S., Wedatilake, Y., Taanman, J.-W., Chan, E., Qasim, W., et al. (2015). Signal transducer and activator of transcription 2 deficiency is a novel disorder of mitochondrial fission. Brain 138, 2834-2846.</div>
</li> <li> <para-num num="[1219]"> </para-num> <div class="description-line" id="p-1204" num="1219">Shakya, A., Callister, C., Goren, A., Yosef, N., Garg, N., Khoddami, V., Nix, D., Regev, A. &amp; Tantin, D. Pluripotency transcription factor Oct4 mediates stepwise nucleosome demethylation and depletion. <i>Mol Cell Biol. </i>35,1014-1025, doi:10.1128/MCB.01105-14 (2015). PMCID:4333097.</div>
</li> <li> <para-num num="[1220]"> </para-num> <div class="description-line" id="p-1205" num="1220">Shalek, A. K., Gaublomme, J. T., Wang, L., Yosef, N., Chevrier, N., Andersen, M. S., Robinson, J. T., Pochet, N., Neuberg, D., Gertner, R. S., Amit, I., Brown, J. R., Hacohen, N., Regev, A., Wu, C. J. &amp; Park, H. Nanowire-mediated delivery enables functional interrogation of primary immune cells: application to the analysis of chronic lymphocytic leukemia. <i>Nano Lett. </i>12,6498-6504, doi:10.1021/n13042917 (2012). PMCID:3573729.</div>
</li> <li> <para-num num="[1221]"> </para-num> <div class="description-line" id="p-1206" num="1221">Shalek, A. K., Satija, R., Adiconis, X., Gertner, R. S., Gaublomme, J. T., Raychowdhury, R., Schwartz, S., Yosef, N., Malboeuf, C., Lu, D., et al. (2013). Single-cell transcriptomics reveals bimodality in expression and splicing in immune cells. Nature 498, 236-240.</div>
</li> <li> <para-num num="[1222]"> </para-num> <div class="description-line" id="p-1207" num="1222">Shalek, A. K., Satija, R., Shuga, J., Trombetta, J. J., Gennert, D., Lu, D., Chen, P., Gertner, R. S., Gaublomme, J. T., Yosef, N., et al. (2014). Single-cell RNA-seq reveals dynamic paracrine control of cellular variation. Nature 510, 363-369.</div>
</li> <li> <para-num num="[1223]"> </para-num> <div class="description-line" id="p-1208" num="1223">Shalem, O., Sanjana, N. E., Hartenian, E., Shi, X., Scott, D. A., Mikkelsen, T. S., Heckl, D., Ebert, B. L., Root, D. E., Doench, J. G., et al. (2014). Genome-scale CRISPR-Cas9 knockout screening in human cells. Science 343, 84-87.</div>
</li> <li> <para-num num="[1224]"> </para-num> <div class="description-line" id="p-1209" num="1224">Shalem, O., Sanjana, N. E. &amp; Zhang, F. 2015, “High-throughput functional genomics using CRISPR-Cas9<i>”, Nature Reviews Genetics, </i>vol. 16, no. 5, pp. 299-311.</div>
</li> <li> <para-num num="[1225]"> </para-num> <div class="description-line" id="p-1210" num="1225">Shao, H., Burrage, L. C., Sinasac, D. S., Hill, A. E., Ernest, S. R., O'Brien, W., Courtland, H.-W., Jepsen, K. J., Kirby, A., Kulbokas, E. J., et al. (2008). Genetic architecture of complex traits: large phenotypic effects and pervasive epistasis. Proc. Natl. Acad. Sci. U.S.A. 105, 19910-19914.</div>
</li> <li> <para-num num="[1226]"> </para-num> <div class="description-line" id="p-1211" num="1226">Shendure, J., and Akey, J. M. (2015). The origins, determinants, and consequences of human mutations. Science (80-.). 349, 1478-1483.</div>
</li> <li> <para-num num="[1227]"> </para-num> <div class="description-line" id="p-1212" num="1227">Shendure, J., and Fields, S. (2016). Massively Parallel Genetics. Genetics 203, 617-619.</div>
</li> <li> <para-num num="[1228]"> </para-num> <div class="description-line" id="p-1213" num="1228">Shi, J., Wang, E., Milazzo, J. P., Wang, Z., Kinney, J. B. &amp; Vakoc, C. R. 2015, “Discovery of cancer drug targets by CRISPR-Cas9 screening of protein domains”, <i>Nature biotechnology, </i>vol. 33, no. 6, pp. 661-667.</div>
</li> <li> <para-num num="[1229]"> </para-num> <div class="description-line" id="p-1214" num="1229">Shoulders, M. D., Ryno, L. M., Genereux, J. C., Moresco, J. J., Tu, P. G., Wu, C., Yates, J. R., Su, A. I., Kelly, J. W. &amp; Wiseman, R. L. 2013, “Stress-independent activation of XBP1s and/or ATF6 reveals three functionally diverse ER proteostasis environments”, <i>Cell Reports, </i>vol. 3, no. 4, pp. 1279-1292.</div>
</li> <li> <para-num num="[1230]"> </para-num> <div class="description-line" id="p-1215" num="1230">Sidrauski, C., Tsai, J. C., Kampmann, M., Hearn, B. R., Vedantham, P., Jaishankar, P., Sokabe, M., Mendez, A. S., Newton, B. W., Tang, E. L., et al 2015, “Pharmacological dimerization and activation of the exchange factor eIF2B antagonizes the integrated stress response”, <i>eLife, </i>vol. 4, pp. e07314.</div>
</li> <li> <para-num num="[1231]"> </para-num> <div class="description-line" id="p-1216" num="1231">Sisler, J. D., Morgan, M., Raje, V., Grande, R. C., Derecka, M., Meier, J., Cantwell, M., Szczepanek, K., Korzun, W. J., Lesnefsky, E. J., et al. (2015). The Signal Transducer and Activator of Transcription 1 (STAT1) Inhibits Mitochondrial Biogenesis in Liver and Fatty Acid Oxidation in Adipocytes. PLoS One 10, e0144444.</div>
</li> <li> <para-num num="[1232]"> </para-num> <div class="description-line" id="p-1217" num="1232">Smith, R. P., Taher, L., Patwardhan, R. P., Kim, M. J., Inoue, F., Shendure, J., Ovcharenko, I., and Ahituv, N. (2013). Massively parallel decoding of mammalian regulatory sequences supports a flexible organizational model. Nat. Genet. 45, 1021-1028.</div>
</li> <li> <para-num num="[1233]"> </para-num> <div class="description-line" id="p-1218" num="1233">Smith, Z. D., Chan, M. M., Humm, K. C., Karnik, R., Mekhoubad, S., Regev, A., Eggan, K. &amp; Meissner, A. DNA methylation dynamics of the human preimplantation embryo. <i>Nature. </i>511,611-615, doi:10.1038/nature13581 (2014). PMCID:4178976.</div>
</li> <li> <para-num num="[1234]"> </para-num> <div class="description-line" id="p-1219" num="1234">Smith, Z. D., Chan, M. M., Mikkelsen, T. S., Gu, H., Gnirke, A., Regev, A. &amp; Meissner, A. A unique regulatory phase of DNA methylation in the early mammalian embryo. <i>Nature. </i>484, 339-344, doi:10.1038/nature10960 (2012). PMCID:3331945.</div>
</li> <li> <para-num num="[1235]"> </para-num> <div class="description-line" id="p-1220" num="1235">Smyth, R. P., Davenport, M. P. &amp; Mak, J. 2012, “The origin of genetic diversity in HIV-1<i>”, Virus Research, </i>vol. 169, no. 2, pp. 415-429.</div>
</li> <li> <para-num num="[1236]"> </para-num> <div class="description-line" id="p-1221" num="1236">Snijder, B., Sacher, R., Ramo, P., Damm, E., Liberali, P. &amp; Pelkmans, L. 2009, “Population context determines cell-to-cell variability in endocytosis and virus infection”, <i>Nature, </i>vol. 461, no. 7263, pp. 520-523.</div>
</li> <li> <para-num num="[1237]"> </para-num> <div class="description-line" id="p-1222" num="1237">Sokolov, A., Carlin, D. E., Paull, E. O., Baertsch, R., and Stuart, J. M. (2016). Pathway-Based Genomics Prediction using Generalized Elastic Net. PLOS Comput. Biol. 12, e1004790.</div>
</li> <li> <para-num num="[1238]"> </para-num> <div class="description-line" id="p-1223" num="1238">Stegle, O., Teichmann, S. A., and Marioni, J. C. (2015). Computational and analytical challenges in single-cell transcriptomics. Nat. Rev. Genet. 16, 133-145.</div>
</li> <li> <para-num num="[1239]"> </para-num> <div class="description-line" id="p-1224" num="1239">Sun, L., Goff, L. A., Trapnell, C., Alexander, R., Lo, K. A., Hacisuleyman, E., Sauvageau, M., Tazon-Vega, B., Kelley, D. R., Hendrickson, D. G., Yuan, B., Kellis, M., Lodish, H. F. &amp; Rinn, J. L. Long noncoding RNAs regulate adipogenesis. <i>Proceedings of the National Academy of Sciences of the United States of America. </i>110, 3387-3392, doi:10.1073/pnas.1222643110 (2013). PMCID:3587215.</div>
</li> <li> <para-num num="[1240]"> </para-num> <div class="description-line" id="p-1225" num="1240">Tanenbaum, M. E., Gilbert, L. A., Qi, L. S., Weissman, J. S. &amp; Vale, R. D. 2014, “A protein-tagging system for signal amplification in gene expression and fluorescence imaging”, <i>Cell, </i>vol. 159, no. 3, pp. 635-646.</div>
</li> <li> <para-num num="[1241]"> </para-num> <div class="description-line" id="p-1226" num="1241">Tang, H., Klopfenstein, D., Pedersen, B., Flick, P., Sato, K., Ramirez, F., Yunes, J., and Mungall, C. (2015). GOATOOLS: Tools for Gene Ontology.</div>
</li> <li> <para-num num="[1242]"> </para-num> <div class="description-line" id="p-1227" num="1242">Theile, C. S., Witte, M. D., Blom, A. E., Kundrat, L., Ploegh, H. L., and Guimaraes, C. P. (2013). Site-specific N-terminal labeling of proteins using sortase-mediated reactions. Nature protocols 8, 1800-1807.</div>
</li> <li> <para-num num="[1243]"> </para-num> <div class="description-line" id="p-1228" num="1243">Thomason, L. C., Costantino, N. &amp; Court, D. L. 2007, <i>“E. coli </i>genome manipulation by P1 transduction”, <i>Current Protocols in Molecular Biology, </i>vol. Chapter 1, pp. Unit 1.17.</div>
</li> <li> <para-num num="[1244]"> </para-num> <div class="description-line" id="p-1229" num="1244">Thomason, L. C., Sawitzke, J. A., Li, X., Costantino, N. &amp; Court, D. L. 2014, “Recombineering: genetic engineering in bacteria using homologous recombination”, <i>Current Protocols in Molecular Biology, </i>vol. 106, pp. 39.</div>
</li> <li> <para-num num="[1245]"> </para-num> <div class="description-line" id="p-1230" num="1245">Tong, A. H. Y. (2004). Global Mapping of the Yeast Genetic Interaction Network. Science (80-.). 303, 808-813.</div>
</li> <li> <para-num num="[1246]"> </para-num> <div class="description-line" id="p-1231" num="1246">Trapnell, C. 2015, “Defining cell types and states with single-cell genomics”, <i>Genome Research, </i>vol. 25, no. 10, pp. 1491-1498.</div>
</li> <li> <para-num num="[1247]"> </para-num> <div class="description-line" id="p-1232" num="1247">Trapnell, C., Cacchiarelli, D., Grimsby, J., Pokharel, P., Li, S., Morse, M., Lennon, N. J., Livak, K. J., Mikkelsen, T. S. &amp; Rinn, J. L. The dynamics and regulators of cell fate decisions are revealed by pseudotemporal ordering of single cells. <i>Nature biotechnology. </i>32, 381-386, doi:10.1038/nbt.2859 (2014). PMCID:4122333.</div>
</li> <li> <para-num num="[1248]"> </para-num> <div class="description-line" id="p-1233" num="1248">Trapnell, C., Hendrickson, D. G., Sauvageau, M., Goff, L., Rinn, J. L. &amp; Pachter, L. Differential analysis of gene regulation at transcript resolution with RNA-seq. <i>Nature biotechnology. </i>31, 46-53, doi:10.1038/nbt.2450 (2013). PMCID:3869392.</div>
</li> <li> <para-num num="[1249]"> </para-num> <div class="description-line" id="p-1234" num="1249">Trombetta, J. J., Gennert, D., Lu, D., Satija, R., Shalek, A. K. &amp; Regev, A. Preparation of Single-Cell RNA-Seq Libraries for Next Generation Sequencing. <i>Curr Protoc Mol Biol. </i>107, 4 22 21-24 22 17, doi:10.1002/0471142727.mb0422s107 (2014). PMCID:4338574.</div>
</li> <li> <para-num num="[1250]"> </para-num> <div class="description-line" id="p-1235" num="1250">Tsumura, A., Hayakawa, T., Kumaki, Y., Takebayashi, S., Sakaue, M., Matsuoka, C., Shimotohno, K., Ishikawa, F., Li, E., Ueda, H.R., et al. (2006). Maintenance of self-renewal ability of mouse embryonic stem cells in the absence of DNA methyltransferases Dnmt1, Dnmt3a and Dnmt3b. Genes to Cells 11, 805-814.</div>
</li> <li> <para-num num="[1251]"> </para-num> <div class="description-line" id="p-1236" num="1251">Tussiwand, R., Lee, W.-L., Murphy, T. L., Mashayekhi, M., K C, W., Albring, J. C., Satpathy, A. T., Rotondo, J. A., Edelson, B. T., Kretzer, N. M., et al. (2012). Compensatory dendritic cell development mediated by BATF-IRF interactions. Nature 490, 502-507.</div>
</li> <li> <para-num num="[1252]"> </para-num> <div class="description-line" id="p-1237" num="1252">Tyynismaa, H., Carroll, C. J., Raimundo, N., Ahola-Erkkilä, S., Wenz, T., Ruhanen, H., Guse, K., Hemminki, A., Peltola-Mjøsund, K. E., Tulkki, V., et al 2010, “Mitochondrial myopathy induces a starvation-like response”, <i>Human Molecular Genetics, </i>vol. 19, no. 20, pp. 3948-3958.</div>
</li> <li> <para-num num="[1253]"> </para-num> <div class="description-line" id="p-1238" num="1253">Van Der Maaten, L. 2014, “Accelerating t-SNE using tree-based algorithms.”, <i>Journal of machine learning research, </i>vol. 15, no. 1, pp. 3221-3245.</div>
</li> <li> <para-num num="[1254]"> </para-num> <div class="description-line" id="p-1239" num="1254">Visscher, P. M., Brown, M. A., McCarthy, M. I., and Yang, J. (2012). Five Years of GWAS Discovery. Am. J. Hum. Genet. 90, 7-24.</div>
</li> <li> <para-num num="[1255]"> </para-num> <div class="description-line" id="p-1240" num="1255">Walter, P. &amp; Ron, D. 2011, “The unfolded protein response: from stress pathway to homeostatic regulation”, <i>Science </i>(New York, N.Y.), vol. 334, no. 6059, pp. 1081-1086.</div>
</li> <li> <para-num num="[1256]"> </para-num> <div class="description-line" id="p-1241" num="1256">Wang, L., Shalek, A. K., Lawrence, M., Ding, R., Gaublomme, J. T., Pochet, N., Stojanov, P., Sougnez, C., Shukla, S. A., Stevenson, K. E., Zhang, W., Wong, J., Sievers, Q. L., MacDonald, B. T., Vartanov, A. R., Goldstein, N. R., Neuberg, D., He, X., Lander, E., Hacohen, N., Regev, A., Getz, G., Brown, J. R., Park, H. &amp; Wu, C. J. Somatic mutation as a mechanism of Wnt/beta-catenin pathway activation in CLL. <i>Blood. </i>124, 1089-1098, doi:10.1182/blood-2014-01-552067 (2014). PMCID:4133483.</div>
</li> <li> <para-num num="[1257]"> </para-num> <div class="description-line" id="p-1242" num="1257">Wang, T., Wei, J. J., Sabatini, D. M., and Lander, E. S. (2014). Genetic screens in human cells using the CRISPR-Cas9 system. Science 343, 80-84.</div>
</li> <li> <para-num num="[1258]"> </para-num> <div class="description-line" id="p-1243" num="1258">Wang, T., Birsoy, K., Hughes, N. W., Krupczak, K. M., Post, Y., Wei, J. J., Lander, E. S., and Sabatini, D. M. (2015). Identification and characterization of essential genes in the human genome. Science (80-.). 350, 1096-1101.</div>
</li> <li> <para-num num="[1259]"> </para-num> <div class="description-line" id="p-1244" num="1259">Wang, Y., Shen, J., Arenzana, N., Tirasophon, W., Kaufman, R.J. &amp; Prywes, R. 2000, “Activation of ATF6 and an ATF6 DNA binding site by the endoplasmic reticulum stress response”, <i>The Journal of Biological Chemistry, </i>vol. 275, no. 35, pp. 27013-27020.</div>
</li> <li> <para-num num="[1260]"> </para-num> <div class="description-line" id="p-1245" num="1260">Wei, L., Fan, M., Xu, L., Heinrich, K., Berry, M. W., Homayouni, R., and Pfeffer, L. M. (2008). Bioinformatic analysis reveals cRel as a regulator of a subset of interferon-stimulated genes. J. Interferon Cytokine Res. 28, 541-551.</div>
</li> <li> <para-num num="[1261]"> </para-num> <div class="description-line" id="p-1246" num="1261">Washietl, S., Kellis, M. &amp; Garber, M. Evolutionary dynamics and tissue specificity of human long noncoding RNAs in six mammals. <i>Genome Res. </i>24, 616-628, doi:10.1101/gr.165035.113 (2014). PMCID:3975061.</div>
</li> <li> <para-num num="[1262]"> </para-num> <div class="description-line" id="p-1247" num="1262">Weinberger, E. D. (1991). Fourier and Taylor series on fitness landscapes. Biol. Cybern. 65, 321-330.</div>
</li> <li> <para-num num="[1263]"> </para-num> <div class="description-line" id="p-1248" num="1263">Wong, A. S. L., Choi, G. C. G., Cui, C. H., Pregernig, G., Milani, P., Adam, M., Perli, S. D., Kazer, S. W., Gaillard, A., Hermann, M., et al. (2016). Multiplexed barcoded CRISPR-Cas9 screening enabled by CombiGEM. Proc. Natl. Acad. Sci. 113, 2544-2549.</div>
</li> <li> <para-num num="[1264]"> </para-num> <div class="description-line" id="p-1249" num="1264">Wu, C., Yosef, N., Thalhamer, T., Zhu, C., Xiao, S., Kishi, Y., Regev, A. &amp; Kuchroo, V. K. Induction of pathogenic TH17 cells by inducible salt-sensing kinase SGK1. <i>Nature. </i>496,513-517, doi:10.1038/nature11984 (2013). PMCID:3637879.</div>
</li> <li> <para-num num="[1265]"> </para-num> <div class="description-line" id="p-1250" num="1265">Yosef, N., and Regev, A. (2016). Writ large: Genomic dissection of the effect of cellular environment on immune response. Science (80-.). 354, 64-68.</div>
</li> <li> <para-num num="[1266]"> </para-num> <div class="description-line" id="p-1251" num="1266">Yosef, N., Shalek, A. K., Gaublomme, J. T., Jin, H., Lee, Y., Awasthi, A., Wu, C., Karwacz, K., Xiao, S., Jorgolli, M., Gennert, D., Satija, R., Shakya, A., Lu, D. Y., Trombetta, J. J., Pillai, M. R., Ratcliffe, P. J., Coleman, M. L., Bix, M., Tantin, D., Park, H., Kuchroo, V. K. &amp; Regev, A. Dynamic regulatory network controlling TH17 cell differentiation. Nature. 496,461-468, doi:10.1038/nature11981 (2013). PMCID:3637864.</div>
</li> <li> <para-num num="[1267]"> </para-num> <div class="description-line" id="p-1252" num="1267">Yu, C., Mannan, A. M., Yvone, G. M., Ross, K. N., Zhang, Y.-L., Marton, M. A., Taylor, B. R., Crenshaw, A., Gould, J. Z., Tamayo, P., et al. (2016). High-throughput identification of genotype-specific cancer vulnerabilities in mixtures of barcoded tumor cell lines. Nat. Biotechnol. 34, 419-423.</div>
</li> <li> <para-num num="[1268]"> </para-num> <div class="description-line" id="p-1253" num="1268">Zalatan, J. G., Lee, M. E., Almeida, R., Gilbert, L. A., Whitehead, E. H., La Russa, M., Tsai, J. C., Weissman, J. S., Dueber, J. E., Qi, L. S., et al. (2015). Engineering Complex Synthetic Transcriptional Programs with CRISPR RNA Scaffolds. Cell 160, 339-350.</div>
</li> <li> <para-num num="[1269]"> </para-num> <div class="description-line" id="p-1254" num="1269">Zetsche, B., Gootenberg, J. S., Abudayyeh, O.O., Slaymaker, I. M., Makarova, K. S., Essletzbichler, P., Volz, S. E., Joung, J., van der Oost, J., Regev, A., et al. (2015). Cpf1 Is a Single RNA-Guided Endonuclease of a Class 2 CRISPR-Cas System. Cell 163, 759-771.</div>
</li> <li> <para-num num="[1270]"> </para-num> <div class="description-line" id="p-1255" num="1270">Zetsche, B., Heidenreich, M., Mohanraju, P., Fedorova, I., Kneppers, J., DeGennaro, E.M., Winblad, N., Choudhury, S.R., Abudayyeh, O. O., Gootenberg, J. S., et al 2016, “Multiplex gene editing by CRISPR-Cpf1 through autonomous processing of a single crRNA array”, <i>bioRxiv. </i> </div>
</li> <li> <para-num num="[1271]"> </para-num> <div class="description-line" id="p-1256" num="1271">Zhang, X., Liu, Q., Luo, C., Deng, Y., Cui, K. &amp; Shi, D. 2014, “Identification and characterization of buffalo 7SK and U6 pol III promoters and application for expression of short hairpin RNAs”, <i>International Journal of Molecular Sciences, </i>vol. 15, no. 2, pp. 2596-2607.</div>
</li> <li> <para-num num="[1272]"> </para-num> <div class="description-line" id="p-1257" num="1272">Zheng, G. X. Y., Terry, J. M., Belgrader, P., Ryvkin, P., Bent, Z. W., Wilson, R., Ziraldo, S. B., Wheeler, T. D., McDermott, G. P., Zhu, J., et al 2016, “Massively parallel digital transcriptional profiling of single cells”, <i>bioRxiv. </i> </div>
</li> <li> <para-num num="[1273]"> </para-num> <div class="description-line" id="p-1258" num="1273">Zuk, O. Hechter, E., Sunyaev, S. R., and Lander, E. S. (2012). The mystery of missing heritability: Genetic interactions create phantom heritability. Proc. Natl. Acad. Sci. U.S.A. 109, 1193-1198.</div>
</li> <li> <para-num num="[1274]"> </para-num> <div class="description-line" id="p-1259" num="1274">The invention is further described by the following numbered paragraphs:
</div> </li> <ul> <li id="ul0011-0001" num="1275">1. A method of reconstructing a cellular network or circuit, comprising introducing at least 1, 2, 3, 4 or more single-order or combinatorial perturbations to each cell in a population of cells;</li> <li id="ul0011-0002" num="1276">measuring genomic, genetic and/or phenotypic differences of each cell and coupling combinatorial peturbations with measured differences to infer intercellular and/or intracellular networks or circuits.</li> <li id="ul0011-0003" num="1277">2. The method of paragraph 1, wherein the single-order or combinatorial perturbations comprise 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22, 23, 24, 25, 26, 27, 28, 29, 30, 31, 32, 33, 34, 35, 36, 37, 38, 39, 40, 41, 42, 43, 44, 45, 46, 47, 48, 49, 50, 51, 52, 53, 54, 55, 56, 57, 58, 59, 60, 61, 62, 63, 64, 65, 66, 67, 68, 69, 70, 71, 72, 73, 74, 75, 76, 77, 78, 79, 80, 81, 82, 83, 84, 85, 86, 87, 88, 89, 90, 91, 92, 93, 94, 95, 96, 97, 98 or 99 or massively parallel perturbations.</li> <li id="ul0011-0004" num="1278">3. The method of paragraph 1 or 2 wherein the perturbation(s) comprise one or more genetic perturbation.</li> <li id="ul0011-0005" num="1279">4. The method of paragraph 1, 2 or 3 wherein the perturbation(s) comprise one or more epigenetic or epigenomic perturbation.</li> <li id="ul0011-0006" num="1280">5. The method of any of the preceding paragraphs wherein at least one perturbation is introduced with RNAi- or a CRISPR-Cas system.</li> <li id="ul0011-0007" num="1281">6. The method of any of the preceding paragraphs wherein at least one perturbation is introduced via a chemical agent, an intracellular spatial relationship between two or more cells, an increase or decrease of temperature, addition or subtraction of energy, electromagnetic energy, or ultrasound.</li> <li id="ul0011-0008" num="1282">7. The method of any one of the preceding paragraphs wherein the cell comprises a cell in a model non-human organism, a model non-human mammal that expresses a Cas protein, a mouse that expresses a Cas protein, a cell in vivo or a cell ex vivo or a cell in vitro.</li> <li id="ul0011-0009" num="1283">8. The method of any one of the preceding paragraphs wherein measuring or measured differences comprises measuring or measured differences of DNA, RNA, protein or post translational modification; or measuring or measured differences of protein or post translational modification correlated to RNA and/or DNA level(s).</li> <li id="ul0011-0010" num="1284">9. The method of any preceding paragraph wherein the method includes sequencing, and prior to sequencing:</li> <li id="ul0011-0011" num="1285">(a) perturbing and isolating a single cell with at least one labeling ligand specific for binding at one or more target RNA transcripts, or isolating a single cell with at least one labeling ligand specific for binding at one or more target RNA transcripts and perturbing the cell; and/or</li> <li id="ul0011-0012" num="1286">(b) lysing the cell under conditions wherein the labeling ligand binds to the target RNA transcript(s).</li> <li id="ul0011-0013" num="1287">10. The method of paragraph 9 wherein step (a) comprises perturbing and isolating a single cell with at least one labeling ligand specific for binding at one or more target RNA transcripts.</li> <li id="ul0011-0014" num="1288">11. The method of paragraph 9 wherein step (a) comprises isolating a single cell with at least one labeling ligand specific for binding at one or more target RNA transcripts and perturbing the cell.</li> <li id="ul0011-0015" num="1289">12. The method of any preceding paragraph wherein perturbing or perturbation(s) comprise(s) genetic perturbing.</li> <li id="ul0011-0016" num="1290">13. The method of any preceding paragraph wherein perturbing or perturbation(s) comprise(s) single-order perturbations.</li> <li id="ul0011-0017" num="1291">14. The method of any preceding paragraph wherein perturbing or perturbation(s) comprise(s) combinatorial perturbations.</li> <li id="ul0011-0018" num="1292">15. The method of any preceding paragraph wherein perturbing or perturbation(s) comprises gene knock-down, gene knock-out, gene activation, gene insertion, or regulatory element deletion.</li> <li id="ul0011-0019" num="1293">16. The method of any preceding paragraph wherein perturbing or perturbation(s) comprises genome-wide perturbation.</li> <li id="ul0011-0020" num="1294">17. The method of any preceding paragraph wherein perturbing or perturbation(s) comprises performing CRISPR-Cas-based perturbation.</li> <li id="ul0011-0021" num="1295">18. The method of any preceding paragraph wherein perturbing or perturbation(s) comprises performing pooled single or combinatorial CRISPR-Cas-based perturbation with a genome-wide library of sgRNAs.</li> <li id="ul0011-0022" num="1296">19. The method of any preceding paragraph wherein perturbing or perturbation(s) comprises performing pooled combinatorial CRISPR-Cas-based perturbation with a genome-wide library of sgRNAs.</li> <li id="ul0011-0023" num="1297">20. The method of any one of paragraphs 18 or 19, wherein each sgRNA comprises a unique molecular identifier.</li> <li id="ul0011-0024" num="1298">21. The method of any one of paragraphs 18 or 19, wherein each sgRNA is co-delivered with a reporter mRNA.</li> <li id="ul0011-0025" num="1299">22. The method of any preceding paragraph wherein perturbing or perturbation(s) comprises subjecting the cell to an increase or decrease in temperature.</li> <li id="ul0011-0026" num="1300">23. The method of any preceding paragraph wherein perturbing or perturbation(s) comprises subjecting the cell to a chemical agent.</li> <li id="ul0011-0027" num="1301">24. The method of any preceding paragraph wherein perturbing or perturbation(s) comprises subjecting the cell to a chemical agent and/or temperature increase or decrease across a gradient.</li> <li id="ul0011-0028" num="1302">25. The method of any preceding paragraph wherein the cell is in a microfluidic system.</li> <li id="ul0011-0029" num="1303">26. The method of any preceding paragraph wherein the cell is in a droplet.</li> <li id="ul0011-0030" num="1304">27. The method of any preceding paragraph wherein perturbing or perturbation(s) comprises transforming or transducing the cell or a population that includes and from which the cell is isolated with one or more genomic sequence-perturbation constructs that perturbs a genomic sequence in the cell.</li> <li id="ul0011-0031" num="1305">28. The method of any preceding paragraph wherein perturbing or perturbation(s) comprises multiplex transformation with a plurality of genomic sequence-perturbation constructs.</li> <li id="ul0011-0032" num="1306">29. The method of any one of the preceding paragraphs, wherein the labeling ligand comprises a unique perturbation identifier (UPI) sequence attached to a perturbation-sequence-capture sequence, and sequencing includes isolating via microbeads comprising a perturbation-sequence-capture-binding-sequence having specific binding affinity for the perturbation-sequence-capture sequence attached to the UPI sequence.</li> <li id="ul0011-0033" num="1307">30. The method of paragraph 29, wherein the UPI sequence is attached to a universal ligation handle sequence, whereby a unique source identifier USI may be generated by split-pool ligation.</li> <li id="ul0011-0034" num="1308">31. The method of any preceding paragraph wherein the labeling ligand comprises an oligonucleotide label comprising a regulatory sequence configured for amplification by T7 polymerase.</li> <li id="ul0011-0035" num="1309">32. The method of any preceding paragraph wherein the labeling ligands comprises oligonucleotide sequences configured to hybridize to a transcript specific region.</li> <li id="ul0011-0036" num="1310">33. The method of any preceding paragraph wherein the labeling ligands comprises an oligonucleotide label.</li> <li id="ul0011-0037" num="1311">34. The method of paragraph 33, wherein the oligonucleotide label further comprises a photocleavable linker.</li> <li id="ul0011-0038" num="1312">35. The method of paragraph 33 or 34 wherein the oligonucleotide label further comprises a restriction enzyme site between the labeling ligand and unique constituent identifier (UCI).</li> <li id="ul0011-0039" num="1313">36. The method of any preceding paragraph, wherein isolating comprises forming discrete unique-identifier-transfer compositions, each comprising the cell and a transfer particle, wherein:</li> <li id="ul0011-0040" num="1314">(a)an oligonucleotide label further comprises a capture sequence, and unique constituent identifier (UCI) and capture sequence are together releasably attached to the labeling ligand;</li> <li id="ul0011-0041" num="1315">(b)the labelling ligand is bound to the target cellular constituent; and,</li> <li id="ul0011-0042" num="1316">(c)a transfer particle comprises:</li> <li id="ul0011-0043" num="1317">(i)a capture-binding-sequence having specific binding affinity for the capture sequence attached to the UCI, and,</li> <li id="ul0011-0044" num="1318">(ii) a unique source identifier (USI) sequence that is unique to each transfer particle.</li> <li id="ul0011-0045" num="1319">37. The method of paragraph 36 wherein, the USI comprises 4-15 nucleotides.</li> <li id="ul0011-0046" num="1320">38. The method of paragraph 36 or 37, further comprising releasing the UCI from the labeled ligand, under conditions within the unique-identifier-transfer composition so that the released capture sequence binds to the capture-binding-sequence on the transfer particle, thereby transferring the UCI to the transfer particle.</li> <li id="ul0011-0047" num="1321">39. The method of any preceding paragraph wherein the ligation handle comprises a restriction site for producing an overhang complementary with a first index sequence overhang, and wherein the method further comprises digestion with a restriction enzyme.</li> <li id="ul0011-0048" num="1322">40. The method of method of any preceding paragraph wherein wherein the ligation handle comprises a nucleotide sequence complementary with a ligation primer sequence and wherein the overhang complementary with a first index sequence overhang is produced by hybridization of the ligation primer to the ligation handle.</li> <li id="ul0011-0049" num="1323">41. The method of any one of the preceding paragraphs further comprising quantitating relative amount of UCI sequence associated with a first cell to the amount of the same UCI sequence associated with a second cell, whereby the relative differences of a cellular constituent between cell(s) are determined.</li> <li id="ul0011-0050" num="1324">42. The method of any one of the preceding paragraphs wherein the labeling ligand comprises an antibody or an antibody fragment.</li> <li id="ul0011-0051" num="1325">43. The method of any one of the preceding paragraphs wherein the antibody fragment is a nanobody, Fab, Fab′, (Fab′)2, Fv, ScFv, diabody, triabody, tetrabody, Bis-scFv, minibody, Fab2, or Fab3 fragment.</li> <li id="ul0011-0052" num="1326">44. The method of any preceding paragraph wherein the labeling ligand comprises an aptamer.</li> <li id="ul0011-0053" num="1327">45. The method of any preceding paragraph wherein the labeling ligand is a nucleotide sequence complementary to a target sequence.</li> <li id="ul0011-0054" num="1328">46. The method of any method of any preceding paragraph wherein the cell or a population that includes wherein the cell(s) are a member of a cell population, and the method further comprises transforming or transducing the cell population with one or more genomic sequence-perturbation constructs that perturb a genomic sequence in the cells, wherein each distinct genomic sequence-perturbation construct comprises a unique-perturbation-identified (UPI) sequence unique to that construct.</li> <li id="ul0011-0055" num="1329">47. The method of paragraph 46, wherein the genomic sequence-perturbation construct comprises a sequence encoding a guide RNA sequence of a CRISPR-Cas targeting system.</li> <li id="ul0011-0056" num="1330">48. The method of paragraph 46 or 47, further comprising multiplex transformation of the population of cells with a plurality of genomic sequence-perturbation constructs.</li> <li id="ul0011-0057" num="1331">49. The method of any one of paragraphs 46, 47 or 48, wherein the UPI sequence is attached to a perturbation-sequence-capture sequence, and the transfer particle comprise a perturbation-sequence-capture-binding-sequence having specific binding affinity for the perturbation-sequence-capture sequence attached to the UPI sequence.</li> <li id="ul0011-0058" num="1332">50. The method of any one of paragraphs 46 or 47, wherein the UPI sequence is attached to a universal ligation handle sequence, whereby a USI is generated by split-pool ligation.</li> <li id="ul0011-0059" num="1333">51. A method of determining any combination of protein detection, RNA detection, open chromatin detection, protein-protein interactions, protein-RNA interactions, or protein-DNA interactions comprising any of the preceding methods.</li> <li id="ul0011-0060" num="1334">52. A method according to any of the preceding paragraphs including whole transcriptome amplification.</li> <li id="ul0011-0061" num="1335">53. The method according to any one of the preceding paragraphs including comparing an RNA profile of the perturbed cell with any mutations in the cell to also correlate phenotypic or transcriptome profile and genotypic profile.</li> <li id="ul0011-0062" num="1336">54. A method for determining genetic interactions by causing a set of P genetic perturbations in cells,</li> <li id="ul0011-0063" num="1337">wherein the method comprises:
    <ul> <li id="ul0012-0001" num="1338">(a) Determining, based upon random sampling, a subset of π genetic perturbations from the set of P genetic perturbations;</li> <li id="ul0012-0002" num="1339">(b) Performing said subset of π genetic perturbations in a population of cells;</li> <li id="ul0012-0003" num="1340">(c) Performing single-cell molecular profiling of the population of genetically perturbed cells of step (b);</li> <li id="ul0012-0004" num="1341">(d) Inferring, from the results of step (c) and based upon the random sampling of step (a), single-cell molecular profiles for the set of P genetic perturbations in cells.</li> </ul>
</li> <li id="ul0011-0064" num="1342">55.The method according to paragraph 54,</li> <li id="ul0011-0065" num="1343">wherein the method further comprises:
    <ul> <li id="ul0013-0001" num="1344">(e) From the results of step (d), determining genetic interactions.</li> </ul>
</li> <li id="ul0011-0066" num="1345">56. The method according to paragraph 54,</li> <li id="ul0011-0067" num="1346">wherein the method further comprises:
    <ul> <li id="ul0014-0001" num="1347">(f) Confirming genetic interactions determined at step (e) with additional genetic manipulations.</li> </ul>
</li> <li id="ul0011-0068" num="1348">57. The method according to any one of paragraphs 54-56,</li> <li id="ul0011-0069" num="1349">wherein said set of P genetic perturbations or said subset of π genetic perturbations comprises single-order genetic perturbations.</li> <li id="ul0011-0070" num="1350">58. The method according to any one of paragraphs 54-57,</li> <li id="ul0011-0071" num="1351">wherein said set of P genetic perturbations or said subset of π genetic perturbations comprises combinatorial genetic perturbations.</li> <li id="ul0011-0072" num="1352">59. The method according to any one of paragraphs 54-58,</li> <li id="ul0011-0073" num="1353">wherein said genetic perturbation comprises gene knock-down, gene knock-out, gene activation, gene insertion, or regulatory element deletion.</li> <li id="ul0011-0074" num="1354">60. The method according to any one of paragraphs 54-59,</li> <li id="ul0011-0075" num="1355">wherein said set of P genetic perturbations or said subset of π genetic perturbations comprises genome-wide perturbations.</li> <li id="ul0011-0076" num="1356">61. The method according to any one of paragraphs 54-60,</li> <li id="ul0011-0077" num="1357">wherein said set of P genetic perturbations or said subset of π genetic perturbations comprises k-order combinations of single genetic perturbations, wherein k is an integer ranging from 2 to 15, and</li> <li id="ul0011-0078" num="1358">wherein step (e) comprises determining k-order genetic interactions.</li> <li id="ul0011-0079" num="1359">62. The method according to any one of paragraphs 54-61,</li> <li id="ul0011-0080" num="1360">wherein said set of P genetic perturbations comprises combinatorial genetic perturbations, such as k-order combinations of single-order genetic perturbations, wherein k is an integer ranging from 2 to 15, and</li> <li id="ul0011-0081" num="1361">wherein step (e) comprises determining j -order genetic interactions, with j&lt;k.</li> <li id="ul0011-0082" num="1362">63. The method according to any one of paragraphs 54-62,</li> <li id="ul0011-0083" num="1363">wherein step (b) comprises performing RNAi- or CRIPSR-Cas-based perturbation.</li> <li id="ul0011-0084" num="1364">64. The method according to any one of paragraphs 54-63,</li> <li id="ul0011-0085" num="1365">wherein step (b) comprises array-format or pool-format perturbation.</li> <li id="ul0011-0086" num="1366">65. The method according to any one of paragraphs 54-64,</li> <li id="ul0011-0087" num="1367">wherein step (b) comprises pooled single or combinatorial CRISPR-Cas-based perturbation with a genome-wide library of sgRNAs.</li> <li id="ul0011-0088" num="1368">66. The method according to any one of paragraphs 54-65,</li> <li id="ul0011-0089" num="1369">wherein step (b) comprises pooled combinatorial CRISPR-Cas-based perturbation with a genome-wide library of sgRNAs.</li> <li id="ul0011-0090" num="1370">67. The method of paragraph 65 or 66, wherein each sgRNA comprises a unique molecular identifier.</li> <li id="ul0011-0091" num="1371">68. The method of paragraph 65, 66 or 67, wherein each sgRNA is co-delivered with a reporter mRNA.</li> <li id="ul0011-0092" num="1372">69. The method according to any one of paragraphs 54-68,</li> <li id="ul0011-0093" num="1373">wherein random sampling comprises matrix completion, tensor completion, compressed sensing, or kernel learning.</li> <li id="ul0011-0094" num="1374">70. The method according to any one of paragraphs 54-69,</li> <li id="ul0011-0095" num="1375">wherein random sampling comprises matrix completion, tensor completion, or compressed sensing, and</li> <li id="ul0011-0096" num="1376">wherein π is of the order of log P.</li> <li id="ul0011-0097" num="1377">71. The method of any one of the preceding paragraphs, wherein the cell comprises a eukaryotic cell.</li> <li id="ul0011-0098" num="1378">72. The method of paragraph 71, wherein the eukaryotic cell comprises a mammalian cell.</li> <li id="ul0011-0099" num="1379">73. The method of paragraph 72, wherein the mammalian cell comprises a human cell.</li> <li id="ul0011-0100" num="1380">74. The method of any one of the preceding paragraphs, wherein the cell is from a population comprising 10<sup>2 </sup>to 10<sup>8 </sup>cells and/or DNA or RNA or protein or post translational modification measurements or variables per cell comprise 50 or more.</li> <li id="ul0011-0101" num="1381">75. A method for screening a collection of compounds or agents comprising performing a method as paragraphed in any of the preceding paragraphs wherein perturbing comprises exposing the cell to each compound or agent.</li> <li id="ul0011-0102" num="1382">76. A therapeutic identified by the method of paragraph 75.</li> <li id="ul0011-0103" num="1383">77. The method of any one of the preceding paragraphs wherein the perturbing is across a library of cells to thereby obtain RNA level and optionally protein level cell-to-cell circuit data at genomic or transcript or expression level.</li> <li id="ul0011-0104" num="1384">78. The method of paragraph 77, wherein the library of cells comprises or is from a tissue sample.</li> <li id="ul0011-0105" num="1385">79. The method of paragraph 78, wherein the tissue sample comprises or is from a biopsy from a mammalian subject.</li> <li id="ul0011-0106" num="1386">80. The method of paragraph 79, wherein the mammalian subject comprises a human subject.</li> <li id="ul0011-0107" num="1387">81. The method of paragraph 80, wherein the biopsy is from a tumor.</li> <li id="ul0011-0108" num="1388">82. The method of any one of paragraphs 77 to 81, further comprising reconstructing cell-to-cell circuits.</li> <li id="ul0011-0109" num="1389">83.The method of any preceding paragraph wherein open chromatin detection includes fragmenting chromatin inside isolated intact nuclei from a cell, adding universal primers at cutting sites, and uniquely tagging DNA that originated from the cell.</li> </ul>
<li> <para-num num="[1390]"> </para-num> <div class="description-line" id="p-1260" num="1390">The invention is further described by the following numbered statement:</div>
</li> <li> <para-num num="[1391]"> </para-num> <div class="description-line" id="p-1261" num="1391">A method of reconstructing a cellular network or circuit, comprising:
</div> </li> <ul> <li id="ul0015-0001" num="1392">(a)introducing at least 1, 2, 3, 4 or more single-order or combinatorial perturbations to a plurality of cells in a population of cells, wherein each cell in the plurality of the cells receives at least 1 perturbation; and</li> <li id="ul0015-0002" num="1393">(b)measuring comprising:</li> <li id="ul0015-0003" num="1394">(i)detecting genomic, genetic, proteomic, epigenetic and/or phenotypic differences in single cells compared to one or more cells that did not receive any perturbation, and</li> <li id="ul0015-0004" num="1395">(ii)detecting the perturbation(s) in single cells.</li> </ul>
<li> <para-num num="[1396]"> </para-num> <div class="description-line" id="p-1262" num="1396">Having thus described in detail preferred embodiments of the present invention, it is to be understood that the invention defined by the above paragraphs is not to be limited to particular details set forth in the above description as many apparent variations thereof are possible without departing from the spirit or scope of the present invention.</div>
</li> </ul>
</div>
</section><section itemprop="claims" itemscope="">
<h2>Claims (<span itemprop="count">83</span>)</h2>
<div html="" itemprop="content"><div class="claims" lang="EN" load-source="patent-office" mxw-id="PCLM181065773">
<div class="claim"> <div class="claim" id="CLM-00001" num="00001">
<div class="claim-text"> <b>1</b>. A method of reconstructing a cellular network or circuit, comprising
<div class="claim-text">(a) introducing at least 1, 2, 3, 4 or more single-order or combinatorial perturbations to a plurality of cells in a population of cells, wherein each cell in the plurality of the cells receives at least 1 perturbation;</div> <div class="claim-text">(b) measuring comprising:
<div class="claim-text">detecting genomic, genetic, proteomic, epigenetic and/or phenotypic differences in single cells compared to one or more cells that did not receive any perturbation, and</div>
<div class="claim-text">(ii) detecting the perturbation(s) in single cells; and</div>
</div> <div class="claim-text">(c) determining measured differences relevant to the perturbations by applying a model accounting for co-variates to the measured differences, whereby intercellular and/or intracellular networks or circuits are inferred.</div> </div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="CLM-00002" num="00002">
<div class="claim-text"> <b>2</b>. The method of <claim-ref idref="CLM-00001">claim 1</claim-ref>, wherein the measuring in single cells comprises single cell sequencing, preferably,
<div class="claim-text">wherein the measuring comprises detecting the transcriptome of each of the single cells; and/or</div> <div class="claim-text">wherein single cell sequencing comprises cell barcodes, whereby the cell-of-origin of each RNA is recorded; and/or</div> <div class="claim-text">wherein the single cell sequencing comprises unique molecular identifiers (UMI), whereby the capture rate of the measured signals, such as transcript copy number or probe binding events, in a single cell is determined; and/or</div> <div class="claim-text">wherein single cell sequencing comprises whole transcriptome amplification.</div> </div>
</div>
</div> <div class="claim"> <div class="claim" id="CLM-00003" num="00003">
<div class="claim-text"> <b>3</b>. (canceled)</div>
</div>
</div> <div class="claim"> <div class="claim" id="CLM-00004" num="00004">
<div class="claim-text"> <b>4</b>. (canceled)</div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="CLM-00005" num="00005">
<div class="claim-text"> <b>5</b>. The method of <claim-ref idref="CLM-00001">claim 1</claim-ref>, wherein the model comprises accounting for the capture rate of measured signals, whether the perturbation actually perturbed the cell (phenotypic impact), the presence of subpopulations of either different cells or cell states, and/or analysis of matched cells without any perturbation.</div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="CLM-00006" num="00006">
<div class="claim-text"> <b>6</b>. The method of <claim-ref idref="CLM-00001">claim 1</claim-ref>, wherein the single-order or combinatorial perturbations comprise 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22, 23, 24, 25, 26, 27, 28, 29, 30, 31, 32, 33, 34, 35, 36, 37, 38, 39, 40, 41, 42, 43, 44, 45, 46, 47, 48, 49, 50, 51, 52, 53, 54, 55, 56, 57, 58, 59, 60, 61, 62, 63, 64, 65, 66, 67, 68, 69, 70, 71, 72, 73, 74, 75, 76, 77, 78, 79, 80, 81, 82, 83, 84, 85, 86, 87, 88, 89, 90, 91, 92, 93, 94, 95, 96, 97, 98, 99 or 100 perturbations; and/or
<div class="claim-text">wherein the perturbation(s) target genes in a pathway or intracellular network.</div> </div>
</div>
</div> <div class="claim"> <div class="claim" id="CLM-00007" num="00007">
<div class="claim-text"> <b>7</b>. (canceled)</div>
</div>
</div> <div class="claim"> <div class="claim" id="CLM-00008" num="00008">
<div class="claim-text"> <b>8</b>. (canceled)</div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="CLM-00009" num="00009">
<div class="claim-text"> <b>9</b>. The method of <claim-ref idref="CLM-00001">claim 1</claim-ref>, wherein the perturbation(s) comprise one or more genetic perturbations, preferably, gene knock-down, gene knock-out, gene activation, gene insertion, or regulatory element deletion; and/or
<div class="claim-text">one or more epigenetic or epigenomic perturbations; and/or</div> <div class="claim-text">genome-wide perturbation.</div> </div>
</div>
</div> <div class="claim"> <div class="claim" id="CLM-00010" num="00010">
<div class="claim-text"> <b>10</b>. (canceled)</div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="CLM-00011" num="00011">
<div class="claim-text"> <b>11</b>. The method of <claim-ref idref="CLM-00001">claim 1</claim-ref>, wherein at least one perturbation is introduced with RNAi- or a CRISPR-Cas system; and/or
<div class="claim-text">wherein at least one perturbation is introduced via a chemical agent, biological agent, an intracellular spatial relationship between two or more cells, an increase or decrease of temperature, addition or subtraction of energy, electromagnetic energy, or ultrasound.</div> </div>
</div>
</div> <div class="claim"> <div class="claim" id="CLM-00012" num="00012">
<div class="claim-text"> <b>12</b>. (canceled)</div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="CLM-00013" num="00013">
<div class="claim-text"> <b>13</b>. The method of <claim-ref idref="CLM-00001">claim 1</claim-ref> wherein the cell comprises a cell in a model non-human organism, a model non-human mammal that expresses a Cas protein, a mouse that expresses a Cas protein, a cell in vivo or a cell ex vivo or a cell in vitro.</div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="CLM-00014" num="00014">
<div class="claim-text"> <b>14</b>. The method of <claim-ref idref="CLM-00001">claim 1</claim-ref> wherein measuring or measured differences comprises measuring or measured differences of DNA, RNA, protein or post translational modification; or measuring or measured differences of protein or post translational modification correlated to RNA and/or DNA level(s).</div>
</div>
</div> <div class="claim"> <div class="claim" id="CLM-00015" num="00015">
<div class="claim-text"> <b>15</b>. (canceled)</div>
</div>
</div> <div class="claim"> <div class="claim" id="CLM-00016" num="00016">
<div class="claim-text"> <b>16</b>. (canceled)</div>
</div>
</div> <div class="claim"> <div class="claim" id="CLM-00017" num="00017">
<div class="claim-text"> <b>17</b>. (canceled)</div>
</div>
</div> <div class="claim"> <div class="claim" id="CLM-00018" num="00018">
<div class="claim-text"> <b>18</b>. (canceled)</div>
</div>
</div> <div class="claim"> <div class="claim" id="CLM-00019" num="00019">
<div class="claim-text"> <b>19</b>. (canceled)</div>
</div>
</div> <div class="claim"> <div class="claim" id="CLM-00020" num="00020">
<div class="claim-text"> <b>20</b>. (canceled)</div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="CLM-00021" num="00021">
<div class="claim-text"> <b>21</b>. The method of <claim-ref idref="CLM-00001">claim 1</claim-ref>, wherein perturbing or perturbation(s) comprises performing pooled single or combinatorial perturbation, preferably, CRISPR-Cas-based perturbation with a library of sgRNAs.</div>
</div>
</div> <div class="claim"> <div class="claim" id="CLM-00022" num="00022">
<div class="claim-text"> <b>22</b>. (canceled)</div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="CLM-00023" num="00023">
<div class="claim-text"> <b>23</b>. The method of <claim-ref idref="CLM-00021">claim 21</claim-ref>, wherein each sgRNA is associated with a unique perturbation barcode, preferably, wherein each sgRNA is co-delivered with a reporter mRNA comprising the unique perturbation barcode.</div>
</div>
</div> <div class="claim"> <div class="claim" id="CLM-00024" num="00024">
<div class="claim-text"> <b>24</b>. (canceled)</div>
</div>
</div> <div class="claim"> <div class="claim" id="CLM-00025" num="00025">
<div class="claim-text"> <b>25</b>. (canceled)</div>
</div>
</div> <div class="claim"> <div class="claim" id="CLM-00026" num="00026">
<div class="claim-text"> <b>26</b>. (canceled)</div>
</div>
</div> <div class="claim"> <div class="claim" id="CLM-00027" num="00027">
<div class="claim-text"> <b>27</b>. (canceled)</div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="CLM-00028" num="00028">
<div class="claim-text"> <b>28</b>. The method of <claim-ref idref="CLM-00011">claim 11</claim-ref>, wherein the biological agent is a toll like receptor agonist or cytokine.</div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="CLM-00029" num="00029">
<div class="claim-text"> <b>29</b>. The method of <claim-ref idref="CLM-00011">claim 11</claim-ref>, wherein perturbing or perturbation(s) comprises subjecting the cell to a temperature increase or decrease across a gradient.</div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="CLM-00030" num="00030">
<div class="claim-text"> <b>30</b>. The method of <claim-ref idref="CLM-00001">claim 1</claim-ref>, wherein the cell is in a microfluidic system; and/or wherein the cell is in a droplet.</div>
</div>
</div> <div class="claim"> <div class="claim" id="CLM-00031" num="00031">
<div class="claim-text"> <b>31</b>. (canceled)</div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="CLM-00032" num="00032">
<div class="claim-text"> <b>32</b>. The method of <claim-ref idref="CLM-00001">claim 1</claim-ref>, wherein perturbing or perturbation(s) comprises transforming or transducing the cell or a population that includes and from which the cell is isolated with one or more genomic sequence-perturbation constructs that perturbs a genomic sequence in the cell; or wherein perturbing or perturbation(s) comprises multiplex transformation with a plurality of genomic sequence-perturbation constructs.</div>
</div>
</div> <div class="claim"> <div class="claim" id="CLM-00033" num="00033">
<div class="claim-text"> <b>33</b>. (canceled)</div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="CLM-00034" num="00034">
<div class="claim-text"> <b>34</b>. The method of <claim-ref idref="CLM-00001">claim 1</claim-ref>, wherein proteins or transcripts expressed in single cells are determined in response to a perturbation, wherein the proteins or transcripts are detected in the single cells by binding of more than one labeling ligand comprising an oligonucleotide tag, wherein the oligonucleotide tag comprises a unique constituent identifier (UCI) specific for a target protein or transcript, optionally,
<div class="claim-text">wherein single cells are fixed in discrete particles; and/or</div> <div class="claim-text">wherein the oligonucleotide tag and sgRNA perturbation barcode comprises a universal ligation handle sequence, whereby a unique cell barcode may be generated by split-pool ligation; and/or</div> <div class="claim-text">wherein the labeling ligand comprises an oligonucleotide label comprising a regulatory sequence configured for amplification by T7 polymerase; and/or</div> <div class="claim-text">wherein the labeling ligands comprise oligonucleotide sequences configured to hybridize to a transcript specific region; and/or</div> <div class="claim-text">wherein the oligonucleotide label further comprises a photocleavable linker; and/or</div> <div class="claim-text">wherein the oligonucleotide label further comprises a restriction enzyme site between the labeling ligand and unique constituent identifier (UCI); and/or</div> <div class="claim-text">further comprising quantitating relative amount of UCI sequence associated with a first cell to the amount of the same UCI sequence associated with a second cell, whereby the relative differences of a cellular constituent between cell(s) are determined.</div> </div>
</div>
</div> <div class="claim"> <div class="claim" id="CLM-00035" num="00035">
<div class="claim-text"> <b>35</b>. (canceled)</div>
</div>
</div> <div class="claim"> <div class="claim" id="CLM-00036" num="00036">
<div class="claim-text"> <b>36</b>. (canceled)</div>
</div>
</div> <div class="claim"> <div class="claim" id="CLM-00037" num="00037">
<div class="claim-text"> <b>37</b>. (canceled)</div>
</div>
</div> <div class="claim"> <div class="claim" id="CLM-00038" num="00038">
<div class="claim-text"> <b>38</b>. (canceled)</div>
</div>
</div> <div class="claim"> <div class="claim" id="CLM-00039" num="00039">
<div class="claim-text"> <b>39</b>. (canceled)</div>
</div>
</div> <div class="claim"> <div class="claim" id="CLM-00040" num="00040">
<div class="claim-text"> <b>40</b>. (canceled)</div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="CLM-00041" num="00041">
<div class="claim-text"> <b>41</b>. The method of <claim-ref idref="CLM-00034">claim 34</claim-ref>, wherein the ligation handle comprises a restriction site for producing an overhang complementary with a first index sequence overhang, and wherein the method further comprises digestion with a restriction enzyme; or
<div class="claim-text">wherein the ligation handle comprises a nucleotide sequence complementary with a ligation primer sequence and wherein the overhang complementary with a first index sequence overhang is produced by hybridization of the ligation primer to the ligation handle.</div> </div>
</div>
</div> <div class="claim"> <div class="claim" id="CLM-00042" num="00042">
<div class="claim-text"> <b>42</b>. (canceled)</div>
</div>
</div> <div class="claim"> <div class="claim" id="CLM-00043" num="00043">
<div class="claim-text"> <b>43</b>. (canceled)</div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="CLM-00044" num="00044">
<div class="claim-text"> <b>44</b>. The method of <claim-ref idref="CLM-00034">claim 34</claim-ref>, wherein the labeling ligand comprises an antibody or an antibody fragment, preferably, wherein the antibody fragment is a nanobody, Fab, Fab′, (Fab′)2, Fv, ScFv, diabody, triabody, tetrabody, Bis-scFv, minibody, Fab2, or Fab3 fragment or
<div class="claim-text">wherein the labeling ligand comprises an aptamer, or</div>
<div class="claim-text">wherein the labeling ligand is a nucleotide sequence complementary to a target sequence.</div>
</div>
</div>
</div> <div class="claim"> <div class="claim" id="CLM-00045" num="00045">
<div class="claim-text"> <b>45</b>. (canceled)</div>
</div>
</div> <div class="claim"> <div class="claim" id="CLM-00046" num="00046">
<div class="claim-text"> <b>46</b>. (canceled)</div>
</div>
</div> <div class="claim"> <div class="claim" id="CLM-00047" num="00047">
<div class="claim-text"> <b>47</b>. (canceled)</div>
</div>
</div> <div class="claim"> <div class="claim" id="CLM-00048" num="00048">
<div class="claim-text"> <b>48</b>. (canceled)</div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="CLM-00049" num="00049">
<div class="claim-text"> <b>49</b>. The method according to <claim-ref idref="CLM-00001">claim 1</claim-ref> including comparing an RNA profile of the perturbed cell with any mutations in the cell to also correlate phenotypic or transcriptome profile and genotypic profile.</div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="CLM-00050" num="00050">
<div class="claim-text"> <b>50</b>. The method according to <claim-ref idref="CLM-00001">claim 1</claim-ref>, further comprising determining genetic interactions by causing a set of P genetic perturbations in single cells of the population of cells,
<div class="claim-text">wherein the method comprises:
<div class="claim-text">(a) determining, based upon random sampling, a subset of π genetic perturbations from the set of P genetic perturbations;</div>
<div class="claim-text">(b) performing said subset of π genetic perturbations in a population of cells;</div>
<div class="claim-text">(c) performing single-cell molecular profiling of the population of genetically perturbed cells of step (b);</div>
<div class="claim-text">(d) inferring, from the results of step (c) and based upon the random sampling of step (a), single-cell molecular profiles for the set of P genetic perturbations in cells; and optionally,</div>
<div class="claim-text">(e) from the results of step (d), determining genetic interactions; and further optionally,</div>
<div class="claim-text">(f) confirming genetic interactions determined at step (e) with additional genetic manipulations.</div>
</div> </div>
</div>
</div> <div class="claim"> <div class="claim" id="CLM-00051" num="00051">
<div class="claim-text"> <b>51</b>. (canceled)</div>
</div>
</div> <div class="claim"> <div class="claim" id="CLM-00052" num="00052">
<div class="claim-text"> <b>52</b>. (canceled)</div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="CLM-00053" num="00053">
<div class="claim-text"> <b>53</b>. The method according to <claim-ref idref="CLM-00050">claim 50</claim-ref>, wherein said set of P genetic perturbations or said subset of π genetic perturbations comprises single-order genetic perturbations; or
<div class="claim-text">wherein said set of P genetic perturbations or said subset of π genetic perturbations comprises combinatorial genetic perturbations; or</div>
<div class="claim-text">wherein said set of P genetic perturbations or said subset of π genetic perturbations comprises genome-wide perturbations; or</div>
<div class="claim-text">wherein said set of P genetic perturbations or said subset of π genetic perturbations comprises k-order combinations of single genetic perturbations, wherein k is an integer ranging from 2 to 15, and wherein step (e) comprises determining k-order genetic interactions; or</div>
<div class="claim-text">wherein said set of P genetic perturbations comprises combinatorial genetic perturbations, such as k-order combinations of single-order genetic perturbations, wherein k is an integer ranging from 2 to 15, and wherein step (e) comprises determining j—order genetic interactions, with j&lt;k.</div>
</div>
</div>
</div> <div class="claim"> <div class="claim" id="CLM-00054" num="00054">
<div class="claim-text"> <b>54</b>. (canceled)</div>
</div>
</div> <div class="claim"> <div class="claim" id="CLM-00055" num="00055">
<div class="claim-text"> <b>55</b>. (canceled)</div>
</div>
</div> <div class="claim"> <div class="claim" id="CLM-00056" num="00056">
<div class="claim-text"> <b>56</b>. (canceled)</div>
</div>
</div> <div class="claim"> <div class="claim" id="CLM-00057" num="00057">
<div class="claim-text"> <b>57</b>. (canceled)</div>
</div>
</div> <div class="claim"> <div class="claim" id="CLM-00058" num="00058">
<div class="claim-text"> <b>58</b>. (canceled)</div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="CLM-00059" num="00059">
<div class="claim-text"> <b>59</b>. The method according to <claim-ref idref="CLM-00050">claim 50</claim-ref>, wherein step (b) comprises performing RNAi- or CRIPSR-Cas-based perturbation; and/or
<div class="claim-text">wherein step (b) comprises array-format or pool-format perturbation; and/or</div> <div class="claim-text">wherein step (b) comprises pooled single or combinatorial CRISPR-Cas-based perturbation with a genome-wide library of sgRNAs; and/or</div> <div class="claim-text">wherein step (b) comprises pooled combinatorial CRISPR-Cas-based perturbation with a genome-wide library of sgRNAs.</div> </div>
</div>
</div> <div class="claim"> <div class="claim" id="CLM-00060" num="00060">
<div class="claim-text"> <b>60</b>. (canceled)</div>
</div>
</div> <div class="claim"> <div class="claim" id="CLM-00061" num="00061">
<div class="claim-text"> <b>61</b>. (canceled)</div>
</div>
</div> <div class="claim"> <div class="claim" id="CLM-00062" num="00062">
<div class="claim-text"> <b>62</b>. (canceled)</div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="CLM-00063" num="00063">
<div class="claim-text"> <b>63</b>. The method according to <claim-ref idref="CLM-00050">claim 50</claim-ref>, wherein random sampling comprises matrix completion, tensor completion, compressed sensing, or kernel learning; or
<div class="claim-text">wherein random sampling comprises matrix completion, tensor completion, or compressed sensing, and wherein π is of the order of log P.</div> </div>
</div>
</div> <div class="claim"> <div class="claim" id="CLM-00064" num="00064">
<div class="claim-text"> <b>64</b>. (canceled)</div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="CLM-00065" num="00065">
<div class="claim-text"> <b>65</b>. The method of <claim-ref idref="CLM-00001">claim 1</claim-ref>, wherein the cell comprises a eukaryotic cell, preferably, wherein the eukaryotic cell comprises a mammalian cell, more preferably, wherein the mammalian cell comprises a human cell.</div>
</div>
</div> <div class="claim"> <div class="claim" id="CLM-00066" num="00066">
<div class="claim-text"> <b>66</b>. (canceled)</div>
</div>
</div> <div class="claim"> <div class="claim" id="CLM-00067" num="00067">
<div class="claim-text"> <b>67</b>. (canceled)</div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="CLM-00068" num="00068">
<div class="claim-text"> <b>68</b>. The method of <claim-ref idref="CLM-00001">claim 1</claim-ref>, wherein the cell is from a population comprising 10<sup>2 </sup>to 10<sup>8 </sup>cells and DNA or RNA or protein or post translational modification measurements or variables per cell comprise 50 or more.</div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="CLM-00069" num="00069">
<div class="claim-text"> <b>69</b>. The method of <claim-ref idref="CLM-00001">claim 1</claim-ref>, wherein perturbation of the population of cells is performed in vivo<sup>. </sup>or
<div class="claim-text">wherein perturbation of the population of cells is performed ex vivo and the population of cells is adoptively transferred to a subject.</div> </div>
</div>
</div> <div class="claim"> <div class="claim" id="CLM-00070" num="00070">
<div class="claim-text"> <b>70</b>. (canceled)</div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="CLM-00071" num="00071">
<div class="claim-text"> <b>71</b>. The method of <claim-ref idref="CLM-00069">claim 69</claim-ref>, wherein the population of cells comprise tumor cells; and/or wherein a lineage barcode is associated with single cells, whereby the lineage or clonality of single cells may be determined.</div>
</div>
</div> <div class="claim"> <div class="claim" id="CLM-00072" num="00072">
<div class="claim-text"> <b>72</b>. (canceled)</div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="CLM-00073" num="00073">
<div class="claim-text"> <b>73</b>. The method of <claim-ref idref="CLM-00001">claim 1</claim-ref> wherein the perturbing is across a library of cells to thereby obtain RNA level and optionally protein level, whereby cell-to-cell circuit data at genomic or transcript or expression level is determined; preferably,
<div class="claim-text">wherein the library of cells comprises or is from a tissue sample, more preferably,</div>
<div class="claim-text">wherein the tissue sample comprises or is from a biopsy from a mammalian subject,</div>
<div class="claim-text">wherein the mammalian subject comprises a human subject, and</div>
<div class="claim-text">wherein the biopsy is from a tumor.</div>
</div>
</div>
</div> <div class="claim"> <div class="claim" id="CLM-00074" num="00074">
<div class="claim-text"> <b>74</b>. (canceled)</div>
</div>
</div> <div class="claim"> <div class="claim" id="CLM-00075" num="00075">
<div class="claim-text"> <b>75</b>. (canceled)</div>
</div>
</div> <div class="claim"> <div class="claim" id="CLM-00076" num="00076">
<div class="claim-text"> <b>76</b>. (canceled)</div>
</div>
</div> <div class="claim"> <div class="claim" id="CLM-00077" num="00077">
<div class="claim-text"> <b>77</b>. (canceled)</div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="CLM-00078" num="00078">
<div class="claim-text"> <b>78</b>. The method of <claim-ref idref="CLM-00073">claim 73</claim-ref>, further comprising reconstructing cell-to-cell circuits.</div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="CLM-00079" num="00079">
<div class="claim-text"> <b>79</b>. The method of <claim-ref idref="CLM-00001">claim 1</claim-ref>, wherein open chromatin is measured and comprises fragmenting chromatin inside isolated intact nuclei from a cell, adding universal primers at cutting sites, and uniquely tagging DNA that originated from the cell; and/or
<div class="claim-text">wherein measuring protein and RNA levels comprises CyTOF.</div> </div>
</div>
</div> <div class="claim"> <div class="claim" id="CLM-00080" num="00080">
<div class="claim-text"> <b>80</b>. (canceled)</div>
</div>
</div> <div class="claim"> <div class="claim" id="CLM-00081" num="00081">
<div class="claim-text"> <b>81</b>. (canceled)</div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="CLM-00082" num="00082">
<div class="claim-text"> <b>82</b>. A method for screening compounds or agents capable of modifying a cellular network or circuit comprising performing a method as claimed in any of the preceding claims, wherein perturbing further comprises exposing the cell to each compound or agent.</div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="CLM-00083" num="00083">
<div class="claim-text"> <b>83</b>. A therapeutic identified by the method of <claim-ref idref="CLM-00082">claim 82</claim-ref>.</div>
</div>
</div> </div>
</div>
</section>
                </article>
            </search-app>
        </body>
    </html>
    