
    <html>
        <body>
            <search-app>
                <article class="result" itemscope="" itemtype="http://schema.org/ScholarlyArticle">
    <h1 itemprop="pageTitle">US20200399307A1 - Methods of purifying polypeptides 
      - Google Patents</h1><section itemprop="abstract" itemscope="">
<h2>Abstract</h2>
<div html="" itemprop="content"><abstract lang="EN" load-source="patent-office" mxw-id="PA427363187">
<div class="abstract" id="p-0001" num="0000">The present invention provides methods for purifying a polypeptide from a composition comprising the polypeptide and at least one contaminant and formulations comprising the polypeptide purified by the methods. The methods for purifying include cation exchange material and/or mixed mode material.</div>
</abstract>
</div>
</section><section itemprop="description" itemscope="">
<h2>Description</h2>
<div html="" itemprop="content"><ul class="description" lang="EN" load-source="patent-office" mxw-id="PDES282588928">
<heading id="h-0001">CROSS-REFERENCE TO RELATED APPLICATIONS</heading>
<li> <para-num num="[0001]"> </para-num> <div class="description-line" id="p-0002" num="0001">This application is a continuation of U.S. patent application Ser. No. 16/167,364, filed on Oct. 22, 2018, now abandoned, which is a continuation of U.S. patent application Ser. No. 13/682,620, filed on Nov. 20, 2012, now abandoned, which is a continuation of International Patent Application No. PCT/US2011/037977, filed May 25, 2011, which claims priority benefit to U.S. Provisional Patent Application No. 61/348,143 filed May 25, 2010, the content of each of which is hereby incorporated herein by reference in its entirety.</div>
</li> <heading id="h-0002">FIELD OF THE INVENTION</heading>
<li> <para-num num="[0002]"> </para-num> <div class="description-line" id="p-0003" num="0002">The present invention provides methods for purifying a polypeptide from a composition comprising the polypeptide and at least one contaminant and formulations comprising the polypeptide purified by the methods.</div>
</li> <heading id="h-0003">BACKGROUND OF THE INVENTION</heading>
<li> <para-num num="[0003]"> </para-num> <div class="description-line" id="p-0004" num="0003">The large-scale, economic purification of polypeptides is increasingly an important problem for the biotechnology industry. Generally, polypeptides are produced by cell culture, using either mammalian or bacterial cell lines engineered to produce the polypeptide of interest by insertion of a recombinant plasmid containing the gene for that polypeptide. Since the cell lines used are living organisms, they must be fed with a complex growth medium, containing sugars, amino acids, and growth factors, usually supplied from preparations of animal serum. It is desirable to separate the polypeptide of interest from a mixture of compounds fed to the cells and from the by-products of the cells themselves.</div>
</li> <li> <para-num num="[0004]"> </para-num> <div class="description-line" id="p-0005" num="0004">The separation of the polypeptide of interest from other products produced by the cell is usually attempted using a combination of different chromatography techniques. These techniques separate mixtures of polypeptides on the basis of their charge, degree of hydrophobicity, size, or the specific interaction between the polypeptide of interest and an immobilized capture agent. Several different chromatography resins are available for each of these techniques, allowing accurate tailoring of the purification scheme to the particular polypeptide involved. The essence of each of these separation methods is that polypeptides can be caused either to move at different rates down a long column, achieving a physical separation that increases as they pass further down the column, or to adhere selectively to the separation medium, being then differentially eluted by different solvents. In some cases, the polypeptide of interest is separated from impurities when the impurities specifically adhere to the column, and the polypeptide of interest does not, that is, the polypeptide of interest is present in the “flow-through.”</div>
</li> <li> <para-num num="[0005]"> </para-num> <div class="description-line" id="p-0006" num="0005">The large-scale, cost-effective purification of a polypeptide to sufficient purity for use as a human therapeutic remains a formidable challenge.</div>
</li> <heading id="h-0004">BRIEF SUMMARY</heading>
<li> <para-num num="[0006]"> </para-num> <div class="description-line" id="p-0007" num="0006">Provided herein are methods for purifying a polypeptide from a composition comprising the polypeptide and at least one contaminant, wherein the method comprises either (i) or (ii): (i) sequential steps of (a) loading the composition onto a cation exchange material at a loading density of greater than about 150 g/L of cation exchange material; and (b) loading a composition recovered from the cation exchange material onto a mixed mode material; or (ii) sequential steps of (a) loading the composition onto a mixed mode material; and (b) loading a composition recovered from mixed mode material onto a cation exchange material at a loading density of greater than about 150 g/L of cation exchange material.</div>
</li> <li> <para-num num="[0007]"> </para-num> <div class="description-line" id="p-0008" num="0007">In some embodiments of any of the methods, the polypeptide has a pI of between about 6 and about 10. In some embodiments, the polypeptide has a pI of between about 7 and about 9.</div>
</li> <li> <para-num num="[0008]"> </para-num> <div class="description-line" id="p-0009" num="0008">In some embodiments of any of the methods, the polypeptide is an antibody or an immunoadhesin. In some embodiments, the polypeptide is an immunoadhesin. In some embodiments, the polypeptide is an antibody. In some embodiments, the antibody is a monoclonal antibody. In some embodiments, the monoclonal antibody is a chimeric antibody, humanized antibody, or human antibody. In some embodiments, the monoclonal antibody is an IgG monoclonal antibody. In some embodiments, the antibody is an antigen binding fragment. In some embodiments, the antigen binding fragment is selected from the group consisting of a Fab fragment, a Fab′ fragment, a F(ab′)<sub>2 </sub>fragment, a scFv, a Fv, and a diabody.</div>
</li> <li> <para-num num="[0009]"> </para-num> <div class="description-line" id="p-0010" num="0009">In some embodiments of any of the methods, the at least one contaminant is any one or more of Chinese Hamster Ovary Protein (CHOP), leached protein A, DNA, aggregated protein, cell culture media component, gentamicin, and viral contaminant</div>
</li> <li> <para-num num="[0010]"> </para-num> <div class="description-line" id="p-0011" num="0010">In some embodiments of any of the methods, the sequential steps in (i) and/or (ii) are continuous. In some embodiments of any of the methods, the sequential steps in (i) and/or (ii) are discontinuous. In some embodiments of any of the methods, the method is (i). In some embodiments of any of the methods, the method is (ii).</div>
</li> <li> <para-num num="[0011]"> </para-num> <div class="description-line" id="p-0012" num="0011">In some embodiments of any of the methods, the loading density is between about 150 g/L and about 2000 g/L. In some embodiments, the density is between about 150 g/L and about 1000 g/L. In some embodiments, the density is between about 500 g/L and about 700 g/L</div>
</li> <li> <para-num num="[0012]"> </para-num> <div class="description-line" id="p-0013" num="0012">In some embodiments of any of the methods, the cation exchange material comprises a carboxylic acid functional group or a sulfonic acid functional group. In some embodiments, the functional group is sulphopropyl, sulfoethyl, sulfoisobutyl, or caboxyl. In some embodiments, the cation exchange material is a membrane, a monolith, or resin particles. In some embodiments, the cation exchange material is a resin. In some embodiments, the cation exchange material is Mustang S, Sartobind S, SO3 Monolith, S Ceramic HyperD, Poros HS50, Poros HS20, sulphopropyl-Sepharose Fast Flow (SPSFF), SP-Sepharose XL (SPXL), CM Sepharose Fast Flow, Capto S, Fractogel Se HiCap, Fractogel SO3, or Fractogel COO. In some embodiments, the cation exchange material is Poros HS50.</div>
</li> <li> <para-num num="[0013]"> </para-num> <div class="description-line" id="p-0014" num="0013">In some embodiments of any of the methods, the mixed mode material comprises functional groups capable of anionic exchange and hydrophobic interactions. In some embodiments, the mixed mode material is Capto-Adhere resin, MEP HyperCel resin, HEA HyperCel resin, PPA HyperCel resin, or ChromaSorb membrane. In some embodiments, the mixed mode material is Capto-Adhere resin.</div>
</li> <li> <para-num num="[0014]"> </para-num> <div class="description-line" id="p-0015" num="0014">In some embodiments of any of the methods, the method comprises use of an equilibration buffer, a wash buffer, and/or a loading buffer with the cation exchange material and/or anion exchange material, and the conductivity of the equilibration buffer, the wash buffer, and/or the loading buffer is between about 2 mS/cm to about 25 mS/cm. In some embodiments, the conductivity of the equilibration buffer, the wash buffer, and/or the loading buffer is between about 3 mS/cm and 8 mS/cm.</div>
</li> <li> <para-num num="[0015]"> </para-num> <div class="description-line" id="p-0016" num="0015">In some embodiments of any of the methods, the method comprises use of an equilibration buffer, a wash buffer, and/or a loading buffer with the cation exchange material and/or the anion exchange material, and the pH of the equilibration buffer, the wash buffer, and/or the loading buffer is between about 4.5 and about 6.5.</div>
</li> <li> <para-num num="[0016]"> </para-num> <div class="description-line" id="p-0017" num="0016">In some embodiments of any of the methods, the equilibration buffer, the wash buffer, and/or the loading buffer with the cation exchange material and/or the anion exchange material are the same. In some embodiments of any of the methods, the equilibration buffer, the wash buffer, and/or the loading buffer with the cation exchange material and/or the anion exchange material are the different.</div>
</li> <li> <para-num num="[0017]"> </para-num> <div class="description-line" id="p-0018" num="0017">In some embodiments of any of the methods, the method further comprising subjecting the composition comprising the polypeptide to one or more further purification steps either before or after steps (a) and (b). In some embodiments of any of the methods, the method further comprises recovering the purified polypeptide. In some embodiments of any of the methods, the method further comprises combining the purified polypeptide with a pharmaceutically acceptable carrier.</div>
</li> <description-of-drawings>
<heading id="h-0005">BRIEF DESCRIPTION OF THE DRAWINGS</heading>
<li> <para-num num="[0018]"> </para-num> <div class="description-line" id="p-0019" num="0018"> <figref idrefs="DRAWINGS">FIGS. 1A-1D</figref> show the chromatograms using Poros HS50, SPSFF, SO3 Monolith, and Mustang S for the purification of anti-CD11a antibody.</div>
</li> <li> <para-num num="[0019]"> </para-num> <div class="description-line" id="p-0020" num="0019"> <figref idrefs="DRAWINGS">FIG. 2</figref> shows C/C<sub>0 </sub>(the Mab (“monomer antibody”) concentration) and C/C<sub>0 </sub>(the Chinese Hamster Ovary Protein (“CHOP”) concentration) with varying amount of the product comprising anti-CD11a antibody collected (g/L CV or MV) using SPSFF, Poros HS50, Mustang S, and SO3 monolith. C is the Mab or CHOP concentration in the collected fraction and C<sub>0 </sub>is the Mab or CHOP concentration in the load.</div>
</li> <li> <para-num num="[0020]"> </para-num> <div class="description-line" id="p-0021" num="0020"> <figref idrefs="DRAWINGS">FIG. 3</figref> shows C/C<sub>0 </sub>(Mab concentration) and C/C<sub>0 </sub>(the high molecular weight (“HMW”) concentration) with varying amount of the product comprising anti-CD11a antibody collected (g/L CV or MV) using SO3 monolith, Mustang S, SPSFF, and Poros HS50. C is the Mab or HMW percentage in the collected fraction and C<sub>0 </sub>is the Mab or HMW percentage in the load.</div>
</li> <li> <para-num num="[0021]"> </para-num> <div class="description-line" id="p-0022" num="0021"> <figref idrefs="DRAWINGS">FIGS. 4A-4D</figref> show C/C<sub>0 </sub>(Mab concentration), C/C<sub>0 </sub>(HMW1 concentration) and C/C<sub>0 </sub>(HMW2 concentration) with varying amount of the product comprising anti-CD11a antibody collected (g/L CV or MV) using SO3 monolith, Mustang S, SPSFF, and Poros HS50.</div>
</li> <li> <para-num num="[0022]"> </para-num> <div class="description-line" id="p-0023" num="0022"> <figref idrefs="DRAWINGS">FIG. 5A</figref> shows the chromatogram of HMWs and Mab in the product comprising anti-CD11a antibody loaded using Poros HS50; <figref idrefs="DRAWINGS">FIG. 5B</figref> shows the chromatogram of HMWs and Mab in the elution pool using Poros HS50 (the inner figure shows an enlarged section of the peaks); <figref idrefs="DRAWINGS">FIG. 5C</figref> shows the accumulated HMW % in the flow through (“FT”) pool and the HMW % in the FT fractions with varying amount of the product comprising anti-CD11a antibody collected (g/L CV) using Poros HS50.</div>
</li> <li> <para-num num="[0023]"> </para-num> <div class="description-line" id="p-0024" num="0023"> <figref idrefs="DRAWINGS">FIG. 6A</figref> shows the chromatograms of the FT pool with varying amount of the product comprising anti-CD11a antibody collected using SPSFF column; <figref idrefs="DRAWINGS">FIG. 6B</figref> shows the accumulated HMW % in the FT pool and the HMW % in the FT fractions with varying amount of the product comprising anti-CD11a antibody collected (mg/mL CV) using SPSFF; <figref idrefs="DRAWINGS">FIG. 6C</figref> shows HMW % with varying amount of the product comprising anti-CD11a antibody collected (mg/mL) using SPSFF.</div>
</li> <li> <para-num num="[0024]"> </para-num> <div class="description-line" id="p-0025" num="0024"> <figref idrefs="DRAWINGS">FIGS. 7A-7D</figref> show the chromatograms using Poros HS50, SPSFF, SO3 Monolith, and Mustang S for the purification of anti-VEGF antibody.</div>
</li> <li> <para-num num="[0025]"> </para-num> <div class="description-line" id="p-0026" num="0025"> <figref idrefs="DRAWINGS">FIG. 8</figref> shows C/C<sub>0 </sub>(Mab concentration) of anti-VEGF antibody with varying amount of the product loaded (g/L CV or MV) using Poros HS50, Mustang S, SO3 monolith, and SPSFF.</div>
</li> <li> <para-num num="[0026]"> </para-num> <div class="description-line" id="p-0027" num="0026"> <figref idrefs="DRAWINGS">FIG. 9</figref> shows C/C<sub>0 </sub>(CHOP concentration) with varying amount of the product comprising anti-VEGF antibody collected (g/L CV or MV) using SPSFF, Sartobind S, Poros HS50, Mustang S, and SO3 monolith.</div>
</li> <li> <para-num num="[0027]"> </para-num> <div class="description-line" id="p-0028" num="0027"> <figref idrefs="DRAWINGS">FIG. 10</figref> shows the amount of DNA (pg/mg) with varying amount of the product comprising anti-VEGF antibody collected (g/L CV or MV) using SPSFF, Poros HS50, Mustang S, and SO3 monolith.</div>
</li> <li> <para-num num="[0028]"> </para-num> <div class="description-line" id="p-0029" num="0028"> <figref idrefs="DRAWINGS">FIG. 11</figref> shows C/C<sub>0 </sub>(HMW concentration) with varying amount of the product comprising anti-VEGF antibody collected (g/L CV or MV) using SPSFF, Sartobind S, Poros HS50 with dilute Mab, Poros HS50, Mustang S, and SO3 monolith.</div>
</li> <li> <para-num num="[0029]"> </para-num> <div class="description-line" id="p-0030" num="0029"> <figref idrefs="DRAWINGS">FIGS. 12A-12B</figref> show C/C<sub>0 </sub>(HMW1 concentration) and C/C<sub>0 </sub>(HMW 2 concentration) with varying amount of the product comprising anti-VEGF antibody collected (g/L CV or MV) using SPSFF, Poros HS50 with dilute Mab, Poros HS50, Mustang S, and SO3 monolith.</div>
</li> <li> <para-num num="[0030]"> </para-num> <div class="description-line" id="p-0031" num="0030"> <figref idrefs="DRAWINGS">FIGS. 13A-13E</figref> show the % HMW bound to the resin (Poros HS50, SE HiCap, SPSFF, SPXL, and Capto S) using the product comprising anti-CD20 antibody under various pHs and salt concentrations.</div>
</li> <li> <para-num num="[0031]"> </para-num> <div class="description-line" id="p-0032" num="0031"> <figref idrefs="DRAWINGS">FIGS. 14A-14E</figref> show the % CHOP bound to the resin (Poros HS50, SE HiCap, SPSFF, SPXL, and Capto S) using the product comprising anti-CD20 antibody under various pHs and salt concentrations.</div>
</li> <li> <para-num num="[0032]"> </para-num> <div class="description-line" id="p-0033" num="0032"> <figref idrefs="DRAWINGS">FIG. 15</figref> shows C/C<sub>0 </sub>(HMW concentration %) with varying amount of the product comprising anti-CD20 antibody collected (g/L CV) using Poros HS50 and Capto S.</div>
</li> <li> <para-num num="[0033]"> </para-num> <div class="description-line" id="p-0034" num="0033"> <figref idrefs="DRAWINGS">FIG. 16</figref> shows accumulated HMW (%) with varying amount of the product comprising anti-CD20 antibody collected (g/L CV) using Poros HS50 and Capto S.</div>
</li> <li> <para-num num="[0034]"> </para-num> <div class="description-line" id="p-0035" num="0034"> <figref idrefs="DRAWINGS">FIG. 17</figref> shows C/C<sub>0 </sub>(CHOP concentration) with varying amount of the product comprising anti-CD20 antibody collected (g/L CV) using Poros HS50 and Capto S.</div>
</li> <li> <para-num num="[0035]"> </para-num> <div class="description-line" id="p-0036" num="0035"> <figref idrefs="DRAWINGS">FIG. 18</figref> shows accumulated CHOP (ppm) with varying amount of the product comprising anti-CD20 antibody collected (g/L CV) using Poros HS50 and Capto S.</div>
</li> <li> <para-num num="[0036]"> </para-num> <div class="description-line" id="p-0037" num="0036"> <figref idrefs="DRAWINGS">FIGS. 19A-19B</figref> show the chromatograms of the UV trace obtained at 280 nm plotted over run time (<figref idrefs="DRAWINGS">FIG. 19A</figref>) and over product loading volume (<figref idrefs="DRAWINGS">FIG. 19B</figref>) using SPSFF under various flow rates for the purification of anti-VEGF antibody.</div>
</li> <li> <para-num num="[0037]"> </para-num> <div class="description-line" id="p-0038" num="0037"> <figref idrefs="DRAWINGS">FIG. 20</figref> shows C/C<sub>0 </sub>(Mab concentration) and C/C<sub>0 </sub>(CHOP concentration) with varying amount of the product comprising anti-VEGF antibody collected (mg/mL CV) using SPSFF under various flow rates.</div>
</li> <li> <para-num num="[0038]"> </para-num> <div class="description-line" id="p-0039" num="0038"> <figref idrefs="DRAWINGS">FIG. 21</figref> shows the amount of DNA (pg/mg) with varying amount of the product comprising anti-VEGF antibody collected (mg/mL CV) using SPSFF under various flow rates.</div>
</li> <li> <para-num num="[0039]"> </para-num> <div class="description-line" id="p-0040" num="0039"> <figref idrefs="DRAWINGS">FIG. 22</figref> shows C/C<sub>0 </sub>(HMW concentration) with varying amount of the product comprising anti-VEGF antibody collected (g/L CV) using SPSFF under various flow rates.</div>
</li> <li> <para-num num="[0040]"> </para-num> <div class="description-line" id="p-0041" num="0040"> <figref idrefs="DRAWINGS">FIG. 23</figref> shows C/C<sub>0 </sub>(HMW concentration (%)), C/C<sub>0 </sub>(CHOP concentration (ppm)), and the amount of DNA (pg/mg) with varying amount of the product comprising anti-VEGF antibody collected (g/L CV) using Poros HS50 under various flow rates.</div>
</li> <li> <para-num num="[0041]"> </para-num> <div class="description-line" id="p-0042" num="0041"> <figref idrefs="DRAWINGS">FIG. 24</figref> shows the chromatograms using Poros HS50 under various loading conductivities for the purification of anti-VEGF antibody.</div>
</li> <li> <para-num num="[0042]"> </para-num> <div class="description-line" id="p-0043" num="0042"> <figref idrefs="DRAWINGS">FIGS. 25A-25B</figref> show the chromatograms of the eluate from the elution (P1 peak) and the eluate from the cleaning (P2 peak) using Poros HS50 loaded with the product comprising anti-VEGF antibody under various loading conductivities.</div>
</li> <li> <para-num num="[0043]"> </para-num> <div class="description-line" id="p-0044" num="0043"> <figref idrefs="DRAWINGS">FIG. 26</figref> shows the amount of CHOP in fraction (ppm) with varying amount of CHOP loaded (ug/mL CV) using Poros HS50 under various loading conductivities for the purification of anti-VEGF antibody.</div>
</li> <li> <para-num num="[0044]"> </para-num> <div class="description-line" id="p-0045" num="0044"> <figref idrefs="DRAWINGS">FIG. 27</figref> shows C/C<sub>0 </sub>(HMW concentration) with varying amount of the product comprising anti-VEGF antibody collected (mg/mL CV) using Poros HS50 under various loading conductivities.</div>
</li> <li> <para-num num="[0045]"> </para-num> <div class="description-line" id="p-0046" num="0045"> <figref idrefs="DRAWINGS">FIGS. 28A-28B</figref> show the amount of DNA (pg/ml) with varying amount of the product comprising anti-VEGF antibody collected (g/L CV) under various loading conductivities and various flow rates (different amount of DNA in the load (pg/mg) for different loading conductivity).</div>
</li> <li> <para-num num="[0046]"> </para-num> <div class="description-line" id="p-0047" num="0046"> <figref idrefs="DRAWINGS">FIG. 29</figref> shows the amount of DNA (pg/mL) and antibody concentration with varying amount of the product comprising anti-VEGF antibody collected (mg/mL CV) using Poros HS50 eluted with a linear salt gradient elution.</div>
</li> <li> <para-num num="[0047]"> </para-num> <div class="description-line" id="p-0048" num="0047"> <figref idrefs="DRAWINGS">FIG. 30</figref> shows C/C<sub>0 </sub>(HMW concentration (%)), C/C<sub>0 </sub>(CHOP concentration (ppm)), and the amount of DNA (pg/mg) with varying amount of the product comprising anti-VEGF antibody loaded (g/L CV) using Poros HS50 with the bed height of 4.6 cm or 14.2 cm.</div>
</li> <li> <para-num num="[0048]"> </para-num> <div class="description-line" id="p-0049" num="0048"> <figref idrefs="DRAWINGS">FIGS. 31A-31F</figref> show the % Mab recovery for anti-VEGF antibody (in FT fraction), anti-CD11a antibody, and anti-CD20 antibody, using Capto Adhere resin under various pHs and conductivities (with either NaAC or glycine HCl as buffering salt).</div>
</li> <li> <para-num num="[0049]"> </para-num> <div class="description-line" id="p-0050" num="0049"> <figref idrefs="DRAWINGS">FIGS. 32A-32F</figref> show the % HMW bound using Capto Adhere resin under various pHs and conductivities (with either NaAC or glycine HCl as buffering salt) for anti-VEGF antibody, anti-CD11a antibody, and anti-CD20 antibody.</div>
</li> <li> <para-num num="[0050]"> </para-num> <div class="description-line" id="p-0051" num="0050"> <figref idrefs="DRAWINGS">FIGS. 33A-33F</figref> show the % CHOP bound using Capto Adhere resin under various pHs and conductivities (with either NaAC or glycine HCl as buffering salt) for anti-VEGF antibody, anti-CD11a antibody, and anti-CD20 antibody.</div>
</li> <li> <para-num num="[0051]"> </para-num> <div class="description-line" id="p-0052" num="0051"> <figref idrefs="DRAWINGS">FIG. 34</figref> shows the chromatograms using the coupled Capto Adhere column and Poros HS50 column for the purification of anti-CD11a antibody.</div>
</li> </description-of-drawings>
<heading id="h-0006">DETAILED DESCRIPTION OF THE INVENTION</heading>
<heading id="h-0007">I. Definitions</heading>
<li> <para-num num="[0052]"> </para-num> <div class="description-line" id="p-0053" num="0052">The term “polypeptide” or “protein” is meant a sequence of amino acids for which the chain length is sufficient to produce the higher levels of tertiary and/or quaternary structure. Thus, proteins are distinguished from “peptides” which are also amino acid-based molecules that do not have such structure. Typically, a protein for use herein will have a molecular weight of at least about 5-20 kD, alternatively at least about 15-20 kD, preferably at least about 20 kD. “Peptide” is meant a sequence of amino acids that generally does not exhibit a higher level of tertiary and/or quaternary structure. Peptides generally have a molecular weight of less than about 5 kD.</div>
</li> <li> <para-num num="[0053]"> </para-num> <div class="description-line" id="p-0054" num="0053">Examples of polypeptides encompassed within the definition herein include mammalian proteins, such as, e.g., renin; a growth hormone, including human growth hormone and bovine growth hormone; growth hormone releasing factor; parathyroid hormone; thyroid stimulating hormone; lipoproteins; alpha-1-antitrypsin; insulin A-chain; insulin B-chain; proinsulin; follicle stimulating hormone; calcitonin; luteinizing hormone; glucagon; clotting factors such as factor VIIIC, factor IX, tissue factor, and von Willebrands factor; anti-clotting factors such as Protein C; atrial natriuretic factor; lung surfactant; a plasminogen activator, such as urokinase or human urine or tissue-type plasminogen activator (t-PA); bombesin; thrombin; hemopoietic growth factor; tumor necrosis factor-alpha and -beta; enkephalinase; RANTES (regulated on activation normally T-cell expressed and secreted); human macrophage inflammatory protein (MIP-1-alpha); a serum albumin such as human serum albumin; Muellerian-inhibiting substance; relaxin A-chain; relaxin B-chain; prorelaxin; mouse gonadotropin-associated peptide; a microbial protein, such as beta-lactamase; DNase; IgE; a cytotoxic T-lymphocyte associated antigen (CTLA), such as CTLA-4; inhibin; activin; vascular endothelial growth factor (VEGF); receptors for hormones or growth factors; protein A or D; rheumatoid factors; a neurotrophic factor such as bone-derived neurotrophic factor (BDNF), neurotrophin-3, -4, -5, or -6 (NT-3, NT-4, NT-5, or NT-6), or a nerve growth factor such as NGF-b; platelet-derived growth factor (PDGF); fibroblast growth factor such as aFGF and bFGF; epidermal growth factor (EGF); transforming growth factor (TGF) such as TGF-alpha and TGF-beta, including TGF-β1, TGF-β2, TGF-β3, TGF-β4, or TGF-β5; insulin-like growth factor-I and -II (IGF-I and IGF-II); des(1-3)-IGF-I (brain IGF-I), insulin-like growth factor binding proteins (IGFBPs); CD proteins such as CD3, CD4, CD8, CD19 and CD20; erythropoietin; osteoinductive factors; immunotoxins; a bone morphogenetic protein (BMP); an interferon such as interferon-alpha, -beta, and -gamma; colony stimulating factors (CSFs), e.g., M-CSF, GM-CSF, and G-CSF; interleukins (ILs), e.g., IL-1 to IL-10; superoxide dismutase; T-cell receptors; surface membrane proteins; decay accelerating factor; viral antigen such as, for example, a portion of the AIDS envelope; transport proteins; homing receptors; addressins; regulatory proteins; integrins such as CD11a, CD11b, CD11c, CD18, an ICAM, VLA-4 and VCAM; a tumor associated antigen such as CA125 (ovarian cancer antigen) or HER2, HER3 or HER4 receptor; immunoadhesins; and fragments and/or variants of any of the above-listed proteins as well as antibodies, including antibody fragments, binding to a protein, including, for example, any of the above-listed proteins.</div>
</li> <li> <para-num num="[0054]"> </para-num> <div class="description-line" id="p-0055" num="0054">“Purified” polypeptide (e.g., antibody) means that the polypeptide has been increased in purity, such that it exists in a form that is more pure than it exists in its natural environment and/or when initially synthesized and/or amplified under laboratory conditions. Purity is a relative term and does not necessarily mean absolute purity.</div>
</li> <li> <para-num num="[0055]"> </para-num> <div class="description-line" id="p-0056" num="0055">The term “epitope tagged” when used herein refers to a chimeric polypeptide comprising a polypeptide fused to a “tag polypeptide.” The tag polypeptide has enough residues to provide an epitope against which an antibody can be made, yet is short enough such that it does not interfere with activity of the polypeptide to which it is fused. The tag polypeptide preferably also is fairly unique so that the antibody does not substantially cross-react with other epitopes. Suitable tag polypeptides generally have at least six amino acid residues and usually between about 8 and 50 amino acid residues (preferably, between about 10 and 20 amino acid residues).</div>
</li> <li> <para-num num="[0056]"> </para-num> <div class="description-line" id="p-0057" num="0056">“Active” or “activity” for the purposes herein refers to form(s) of a polypeptide which retain a biological and/or an immunological activity of native or naturally-occurring polypeptide, wherein “biological” activity refers to a biological function (either inhibitory or stimulatory) caused by a native or naturally-occurring polypeptide other than the ability to induce the production of an antibody against an antigenic epitope possessed by a native or naturally-occurring polypeptide and an “immunological” activity refers to the ability to induce the production of an antibody against an antigenic epitope possessed by a native or naturally-occurring polypeptide.</div>
</li> <li> <para-num num="[0057]"> </para-num> <div class="description-line" id="p-0058" num="0057">The term “antagonist” is used in the broadest sense, and includes any molecule that partially or fully blocks, inhibits, or neutralizes a biological activity of a native polypeptide. In a similar manner, the term “agonist” is used in the broadest sense and includes any molecule that mimics a biological activity of a native polypeptide. Suitable agonist or antagonist molecules specifically include agonist or antagonist antibodies or antibody fragments, fragments or amino acid sequence variants of native polypeptides, etc. Methods for identifying agonists or antagonists of a polypeptide may comprise contacting a polypeptide with a candidate agonist or antagonist molecule and measuring a detectable change in one or more biological activities normally associated with the polypeptide.</div>
</li> <li> <para-num num="[0058]"> </para-num> <div class="description-line" id="p-0059" num="0058">“Complement dependent cytotoxicity” or “CDC” refer to the ability of a molecule to lyse a target in the presence of complement. The complement activation pathway is initiated by the binding of the first component of the complement system (C1q) to a molecule (e.g. polypeptide (e.g., an antibody)) complexed with a cognate antigen. To assess complement activation, a CDC assay, e.g. as described in Gazzano-Santoro et al., <i>J. Immunol. Methods </i>202:163 (1996), may be performed.</div>
</li> <li> <para-num num="[0059]"> </para-num> <div class="description-line" id="p-0060" num="0059">A polypeptide “which binds” an antigen of interest, e.g. a tumor-associated polypeptide antigen target, is one that binds the antigen with sufficient affinity such that the polypeptide is useful as a diagnostic and/or therapeutic agent in targeting a cell or tissue expressing the antigen, and does not significantly cross-react with other polypeptides. In such embodiments, the extent of binding of the polypeptide to a “non-target” polypeptide will be less than about 10% of the binding of the polypeptide to its particular target polypeptide as determined by fluorescence activated cell sorting (FACS) analysis or radioimmunoprecipitation (RIA).</div>
</li> <li> <para-num num="[0060]"> </para-num> <div class="description-line" id="p-0061" num="0060">With regard to the binding of a polypeptide to a target molecule, the term “specific binding” or “specifically binds to” or is “specific for” a particular polypeptide or an epitope on a particular polypeptide target means binding that is measurably different from a non-specific interaction. Specific binding can be measured, for example, by determining binding of a molecule compared to binding of a control molecule, which generally is a molecule of similar structure that does not have binding activity. For example, specific binding can be determined by competition with a control molecule that is similar to the target, for example, an excess of non-labeled target. In this case, specific binding is indicated if the binding of the labeled target to a probe is competitively inhibited by excess unlabeled target.</div>
</li> <li> <para-num num="[0061]"> </para-num> <div class="description-line" id="p-0062" num="0061">A polypeptide that “inhibits the growth of tumor cells” or a “growth inhibitory” polypeptide is one which results in measurable growth inhibition of cancer cells. In one embodiment, growth inhibition can be measured at a polypeptide concentration of about 0.1 to about 30 μg/ml or about 0.5 nM to about 200 nM in cell culture, where the growth inhibition is determined 1-10 days after exposure of the tumor cells to the polypeptide. The polypeptide is growth inhibitory in vivo if administration of the polypeptide at about 1 μg/kg to about 100 mg/kg body weight results in reduction in tumor size or tumor cell proliferation within about 5 days to about 3 months from the first administration of the polypeptide, preferably within about 5 to about 30 days.</div>
</li> <li> <para-num num="[0062]"> </para-num> <div class="description-line" id="p-0063" num="0062">A polypeptide which “induces apoptosis” is one which induces programmed cell death as determined by binding of annexin V, fragmentation of DNA, cell shrinkage, dilation of endoplasmic reticulum, cell fragmentation, and/or formation of membrane vesicles (called apoptotic bodies). Preferably the cell is a tumor cell, e.g., a prostate, breast, ovarian, stomach, endometrial, lung, kidney, colon, bladder cell. Various methods are available for evaluating the cellular events associated with apoptosis. For example, phosphatidyl serine (PS) translocation can be measured by annexin binding; DNA fragmentation can be evaluated through DNA laddering; and nuclear/chromatin condensation along with DNA fragmentation can be evaluated by any increase in hypodiploid cells. Preferably, the polypeptide which induces apoptosis is one which results in about 2 to about 50 fold, preferably about 5 to about 50 fold, and most preferably about 10 to about 50 fold, induction of annexin binding relative to untreated cell in an annexin binding assay.</div>
</li> <li> <para-num num="[0063]"> </para-num> <div class="description-line" id="p-0064" num="0063">A polypeptide which “induces cell death” is one which causes a viable cell to become nonviable. Preferably, the cell is a cancer cell, e.g., a breast, ovarian, stomach, endometrial, salivary gland, lung, kidney, colon, thyroid, pancreatic or bladder cell. Cell death in vitro may be determined in the absence of complement and immune effector cells to distinguish cell death induced by antibody-dependent cell-mediated cytotoxicity (ADCC) or complement dependent cytotoxicity (CDC). Thus, the assay for cell death may be performed using heat inactivated serum (i.e., in the absence of complement) and in the absence of immune effector cells. To determine whether the polypeptide is able to induce cell death, loss of membrane integrity as evaluated by uptake of propidium iodide (PI), trypan blue (see Moore et al. <i>Cytotechnology </i>17:1-11 (1995)) or 7AAD can be assessed relative to untreated cells.</div>
</li> <li> <para-num num="[0064]"> </para-num> <div class="description-line" id="p-0065" num="0064">The term “antibody” herein is used in the broadest sense and specifically covers monoclonal antibodies, polyclonal antibodies, multispecific antibodies (e.g. bispecific antibodies) formed from at least two intact antibodies, and antibody fragments so long as they exhibit the desired biological activity. The term “immunoglobulin” (Ig) is used interchangeable with antibody herein.</div>
</li> <li> <para-num num="[0065]"> </para-num> <div class="description-line" id="p-0066" num="0065">Antibodies are naturally occurring immunoglobulin molecules which have varying structures, all based upon the immunoglobulin fold. For example, IgG antibodies have two “heavy” chains and two “light” chains that are disulphide-bonded to form a functional antibody. Each heavy and light chain itself comprises a “constant” (C) and a “variable” (V) region. The V regions determine the antigen binding specificity of the antibody, whilst the C regions provide structural support and function in non-antigen-specific interactions with immune effectors. The antigen binding specificity of an antibody or antigen-binding fragment of an antibody is the ability of an antibody to specifically bind to a particular antigen.</div>
</li> <li> <para-num num="[0066]"> </para-num> <div class="description-line" id="p-0067" num="0066">The antigen binding specificity of an antibody is determined by the structural characteristics of the V region. The variability is not evenly distributed across the 110-amino acid span of the variable domains. Instead, the V regions consist of relatively invariant stretches called framework regions (FRs) of 15-30 amino acids separated by shorter regions of extreme variability called “hypervariable regions” that are each 9-12 amino acids long. The variable domains of native heavy and light chains each comprise four FRs, largely adopting a β-sheet configuration, connected by three hypervariable regions, which form loops connecting, and in some cases forming part of, the β-sheet structure. The hypervariable regions in each chain are held together in close proximity by the FRs and, with the hypervariable regions from the other chain, contribute to the formation of the antigen-binding site of antibodies (see Kabat et al., <i>Sequences of Proteins of Immunological Interest, </i>5th Ed. Public Health Service, National Institutes of Health, Bethesda, Md. (1991)). The constant domains are not involved directly in binding an antibody to an antigen, but exhibit various effector functions, such as participation of the antibody in antibody dependent cellular cytotoxicity (ADCC).</div>
</li> <li> <para-num num="[0067]"> </para-num> <div class="description-line" id="p-0068" num="0067">Each V region typically comprises three complementarity determining regions (“CDRs”, each of which contains a “hypervariable loop”), and four framework regions. An antibody binding site, the minimal structural unit required to bind with substantial affinity to a particular desired antigen, will therefore typically include the three CDRs, and at least three, preferably four, framework regions interspersed there between to hold and present the CDRs in the appropriate conformation. Classical four chain antibodies have antigen binding sites which are defined by V<sub>H </sub>and V<sub>L </sub>domains in cooperation. Certain antibodies, such as camel and shark antibodies, lack light chains and rely on binding sites formed by heavy chains only. Single domain engineered immunoglobulins can be prepared in which the binding sites are formed by heavy chains or light chains alone, in absence of cooperation between V<sub>H </sub>and V<sub>L</sub>.</div>
</li> <li> <para-num num="[0068]"> </para-num> <div class="description-line" id="p-0069" num="0068">The term “variable” refers to the fact that certain portions of the variable domains differ extensively in sequence among antibodies and are used in the binding and specificity of each particular antibody for its particular antigen. However, the variability is not evenly distributed throughout the variable domains of antibodies. It is concentrated in three segments called hypervariable regions both in the light chain and the heavy chain variable domains. The more highly conserved portions of variable domains are called the framework regions (FRs). The variable domains of native heavy and light chains each comprise four FRs, largely adopting a β-sheet configuration, connected by three hypervariable regions, which form loops connecting, and in some cases forming part of, the β-sheet structure. The hypervariable regions in each chain are held together in close proximity by the FRs and, with the hypervariable regions from the other chain, contribute to the formation of the antigen-binding site of antibodies (see Kabat et al., <i>Sequences of Proteins of Immunological Interest, </i>5th Ed. Public Health Service, National Institutes of Health, Bethesda, Md. (1991)). The constant domains are not involved directly in binding an antibody to an antigen, but exhibit various effector functions, such as participation of the antibody in antibody dependent cellular cytotoxicity (ADCC).</div>
</li> <li> <para-num num="[0069]"> </para-num> <div class="description-line" id="p-0070" num="0069">The term “hypervariable region” when used herein refers to the amino acid residues of an antibody that are responsible for antigen binding. The hypervariable region may comprise amino acid residues from a “complementarity determining region” or “CDR” (e.g., around about residues 24-34 (L1), 50-56 (L2) and 89-97 (L3) in the V<sub>L</sub>, and around about 31-35B (H1), 50-65 (H2) and 95-102 (H3) in the V<sub>H </sub>(Kabat et al., <i>Sequences of Proteins of Immunological Interest, </i>5th Ed. Public Health Service, National Institutes of Health, Bethesda, Md. (1991)) and/or those residues from a “hypervariable loop” (e.g. residues 26-32 (L1), 50-52 (L2) and 91-96 (L3) in the V<sub>L</sub>, and 26-32 (H1), 52A-55 (H2) and 96-101 (H3) in the V<sub>H </sub>(Chothia and Lesk <i>J. Mol. Biol. </i>196:901-917 (1987)).</div>
</li> <li> <para-num num="[0070]"> </para-num> <div class="description-line" id="p-0071" num="0070">“Framework” or “FR” residues are those variable domain residues other than the hypervariable region residues as herein defined.</div>
</li> <li> <para-num num="[0071]"> </para-num> <div class="description-line" id="p-0072" num="0071">“Antibody fragments” comprise a portion of an intact antibody, preferably comprising the antigen binding region thereof. Examples of antibody fragments include Fab, Fab′, F(ab′)<sub>2</sub>, and Fv fragments; diabodies; linear antibodies; single-chain antibody molecules; and multispecific antibodies formed from antibody fragments.</div>
</li> <li> <para-num num="[0072]"> </para-num> <div class="description-line" id="p-0073" num="0072">Papain digestion of antibodies produces two identical antigen-binding fragments, called “Fab” fragments, each with a single antigen-binding site, and a residual “Fc” fragment, whose name reflects its ability to crystallize readily. Pepsin treatment yields an F(ab′)<sub>2 </sub>fragment that has two antigen-binding sites and is still capable of cross-linking antigen.</div>
</li> <li> <para-num num="[0073]"> </para-num> <div class="description-line" id="p-0074" num="0073">“Fv” is the minimum antibody fragment that contains a complete antigen-recognition and antigen-binding site. This region consists of a dimer of one heavy chain and one light chain variable domain in tight, non-covalent association. It is in this configuration that the three hypervariable regions of each variable domain interact to define an antigen-binding site on the surface of the V<sub>H</sub>-V<sub>L </sub>dimer. Collectively, the six hypervariable regions confer antigen-binding specificity to the antibody. However, even a single variable domain (or half of an Fv comprising only three hypervariable regions specific for an antigen) has the ability to recognize and bind antigen, although at a lower affinity than the entire binding site.</div>
</li> <li> <para-num num="[0074]"> </para-num> <div class="description-line" id="p-0075" num="0074">The Fab fragment also contains the constant domain of the light chain and the first constant domain (CH1) of the heavy chain. Fab′ fragments differ from Fab fragments by the addition of a few residues at the carboxy terminus of the heavy chain CH1 domain including one or more cysteines from the antibody hinge region. Fab′-SH is the designation herein for Fab′ in which the cysteine residue(s) of the constant domains bear at least one free thiol group. F(ab′)2 antibody fragments originally were produced as pairs of Fab′ fragments that have hinge cysteines between them. Other chemical couplings of antibody fragments are also known.</div>
</li> <li> <para-num num="[0075]"> </para-num> <div class="description-line" id="p-0076" num="0075">The “light chains” of antibodies (immunoglobulins) from any vertebrate species can be assigned to one of two clearly distinct types, called kappa (κ) and lambda (λ), based on the amino acid sequences of their constant domains</div>
</li> <li> <para-num num="[0076]"> </para-num> <div class="description-line" id="p-0077" num="0076">Depending on the amino acid sequence of the constant domain of their heavy chains, antibodies can be assigned to different classes. There are five major classes of intact antibodies: IgA, IgD, IgE, IgG, and IgM, and several of these may be further divided into subclasses (isotypes), e.g., IgG1, IgG2, IgG3, IgG4, IgA, and IgA2. The heavy chain constant domains that correspond to the different classes of antibodies are called α, δ, ε, γ, and μ, respectively. The subunit structures and three-dimensional configurations of different classes of immunoglobulins are well known.</div>
</li> <li> <para-num num="[0077]"> </para-num> <div class="description-line" id="p-0078" num="0077">“Single-chain Fv” or “scFv” antibody fragments comprise the V<sub>H </sub>and V<sub>L </sub>domains of antibody, wherein these domains are present in a single polypeptide chain. In some embodiments, the Fv polypeptide further comprises a polypeptide linker between the V<sub>H </sub>and V<sub>L </sub>domains that enables the scFv to form the desired structure for antigen binding. For a review of scFv see Plückthun in <i>The Pharmacology of Monoclonal Antibodies</i>, vol. 113, Rosenburg and Moore eds., Springer-Verlag, New York, pp. 269-315 (1994).</div>
</li> <li> <para-num num="[0078]"> </para-num> <div class="description-line" id="p-0079" num="0078">The term “diabodies” refers to small antibody fragments with two antigen-binding sites, which fragments comprise a heavy chain variable domain (V<sub>H</sub>) connected to a light chain variable domain (V<sub>L</sub>) in the same polypeptide chain (V<sub>H</sub>-V<sub>L</sub>). By using a linker that is too short to allow pairing between the two domains on the same chain, the domains are forced to pair with the complementary domains of another chain and create two antigen-binding sites. Diabodies are described more fully in, for example, EP 404,097; WO 93/11161; and Hollinger et al., <i>Proc. Natl. Acad. Sci. USA, </i>90:6444-6448 (1993).</div>
</li> <li> <para-num num="[0079]"> </para-num> <div class="description-line" id="p-0080" num="0079">The term “monoclonal antibody” as used herein refers to an antibody obtained from a population of substantially homogeneous antibodies, i.e., the individual antibodies comprising the population are identical and/or bind the same epitope, except for possible variants that may arise during production of the monoclonal antibody, such variants generally being present in minor amounts. In contrast to polyclonal antibody preparations that typically include different antibodies directed against different determinants (epitopes), each monoclonal antibody is directed against a single determinant on the antigen. In addition to their specificity, the monoclonal antibodies are advantageous in that they are uncontaminated by other immunoglobulins. The modifier “monoclonal” indicates the character of the antibody as being obtained from a substantially homogeneous population of antibodies, and is not to be construed as requiring production of the antibody by any particular method. For example, the monoclonal antibodies to be used in accordance with the methods provided herein may be made by the hybridoma method first described by Kohler et al., <i>Nature </i>256:495 (1975), or may be made by recombinant DNA methods (see, e.g., U.S. Pat. No. 4,816,567). The “monoclonal antibodies” may also be isolated from phage antibody libraries using the techniques described in Clackson et al., <i>Nature </i>352:624-628 (1991) and Marks et al., <i>J. Mol. Biol. </i>222:581-597 (1991), for example.</div>
</li> <li> <para-num num="[0080]"> </para-num> <div class="description-line" id="p-0081" num="0080">The monoclonal antibodies herein specifically include “chimeric” antibodies (immunoglobulins) in which a portion of the heavy and/or light chain is identical with or homologous to corresponding sequences in antibodies derived from a particular species or belonging to a particular antibody class or subclass, while the remainder of the chain(s) is identical with or homologous to corresponding sequences in antibodies derived from another species or belonging to another antibody class or subclass, as well as fragments of such antibodies, so long as they exhibit the desired biological activity (U.S. Pat. No. 4,816,567; Morrison et al., <i>Proc. Natl. Acad. Sci. USA </i>81:6851-6855 (1984)). Chimeric antibodies of interest herein include “primatized” antibodies comprising variable domain antigen-binding sequences derived from a non-human primate (e.g. Old World Monkey, such as baboon, rhesus or cynomolgus monkey) and human constant region sequences (U.S. Pat. No. 5,693,780).</div>
</li> <li> <para-num num="[0081]"> </para-num> <div class="description-line" id="p-0082" num="0081">“Humanized” forms of non-human (e.g., murine) antibodies are chimeric antibodies that contain minimal sequence derived from non-human immunoglobulin. For the most part, humanized antibodies are human immunoglobulins (recipient antibody) in which residues from a hypervariable region of the recipient are replaced by residues from a hypervariable region of a non-human species (donor antibody) such as mouse, rat, rabbit or nonhuman primate having the desired specificity, affinity, and capacity. In some instances, framework region (FR) residues of the human immunoglobulin are replaced by corresponding non-human residues. Furthermore, humanized antibodies may comprise residues that are not found in the recipient antibody or in the donor antibody. These modifications are made to further refine antibody performance. In general, the humanized antibody will comprise substantially all of at least one, and typically two, variable domains, in which all or substantially all of the hypervariable loops correspond to those of a non-human immunoglobulin and all or substantially all of the FRs are those of a human immunoglobulin sequence, except for FR substitution(s) as noted above. The humanized antibody optionally also will comprise at least a portion of an immunoglobulin constant region, typically that of a human immunoglobulin. For further details, see Jones et al., <i>Nature </i>321:522-525 (1986); Riechmann et al., <i>Nature </i>332:323-329 (1988); and Presta, <i>Curr. Op. Struct. Biol. </i>2:593-596 (1992).</div>
</li> <li> <para-num num="[0082]"> </para-num> <div class="description-line" id="p-0083" num="0082">For the purposes herein, an “intact antibody” is one comprising heavy and light variable domains as well as an Fc region. The constant domains may be native sequence constant domains (e.g. human native sequence constant domains) or amino acid sequence variant thereof. Preferably, the intact antibody has one or more effector functions.</div>
</li> <li> <para-num num="[0083]"> </para-num> <div class="description-line" id="p-0084" num="0083">“Native antibodies” are usually heterotetrameric glycoproteins of about 150,000 daltons, composed of two identical light (L) chains and two identical heavy (H) chains. Each light chain is linked to a heavy chain by one covalent disulfide bond, while the number of disulfide linkages varies among the heavy chains of different immunoglobulin isotypes. Each heavy and light chain also has regularly spaced intrachain disulfide bridges. Each heavy chain has at one end a variable domain (V<sub>H</sub>) followed by a number of constant domains. Each light chain has a variable domain at one end (V<sub>L</sub>) and a constant domain at its other end; the constant domain of the light chain is aligned with the first constant domain of the heavy chain, and the light chain variable domain is aligned with the variable domain of the heavy chain Particular amino acid residues are believed to form an interface between the light chain and heavy chain variable domains.</div>
</li> <li> <para-num num="[0084]"> </para-num> <div class="description-line" id="p-0085" num="0084">A “naked antibody” is an antibody (as herein defined) that is not conjugated to a heterologous molecule, such as a cytotoxic moiety or radiolabel.</div>
</li> <li> <para-num num="[0085]"> </para-num> <div class="description-line" id="p-0086" num="0085">In some embodiments, antibody “effector functions” refer to those biological activities attributable to the Fc region (a native sequence Fc region or amino acid sequence variant Fc region) of an antibody, and vary with the antibody isotype. Examples of antibody effector functions include: C1q binding and complement dependent cytotoxicity; Fc receptor binding; antibody-dependent cell-mediated cytotoxicity (ADCC); phagocytosis; down regulation of cell surface receptors.</div>
</li> <li> <para-num num="[0086]"> </para-num> <div class="description-line" id="p-0087" num="0086">“Antibody-dependent cell-mediated cytotoxicity” and “ADCC” refer to a cell-mediated reaction in which nonspecific cytotoxic cells that express Fc receptors (FcRs) (e.g. Natural Killer (NK) cells, neutrophils, and macrophages) recognize bound antibody on a target cell and subsequently cause lysis of the target cell. The primary cells for mediating ADCC, NK cells, express FcγRIII only, whereas monocytes express FcγR1, FcγRII and FcγRIII. FcR expression on hematopoietic cells in summarized is Table 3 on page 464 of Ravetch and Kinet, <i>Annu. Rev. Immunol </i>9:457-92 (1991). To assess ADCC activity of a molecule of interest, an in vitro ADCC assay, such as that described in U.S. Pat. No. 5,500,362 or 5,821,337 may be performed. Useful effector cells for such assays include peripheral blood mononuclear cells (PBMC) and Natural Killer (NK) cells. Alternatively, or additionally, ADCC activity of the molecule of interest may be assessed in vivo, e.g., in a animal model such as that disclosed in Clynes et al., <i>PNAS </i>(<i>USA</i>) 95:652-656 (1998).</div>
</li> <li> <para-num num="[0087]"> </para-num> <div class="description-line" id="p-0088" num="0087">“Human effector cells” are leukocytes that express one or more FcRs and perform effector functions. In some embodiments, the cells express at least FcγRIII and carry out ADCC effector function. Examples of human leukocytes that mediate ADCC include peripheral blood mononuclear cells (PBMC), natural killer (NK) cells, monocytes, cytotoxic T cells and neutrophils; with PBMCs and NK cells being preferred.</div>
</li> <li> <para-num num="[0088]"> </para-num> <div class="description-line" id="p-0089" num="0088">The terms “Fc receptor” or “FcR” are used to describe a receptor that binds to the Fc region of an antibody. In some embodiments, the FcR is a native sequence human FcR. Moreover, a preferred FcR is one that binds an IgG antibody (a gamma receptor) and includes receptors of the FcγRI, FcγRII, and Fcγ RIII subclasses, including allelic variants and alternatively spliced forms of these receptors. FcγRII receptors include FcγRIIA (an “activating receptor”) and FcγRIIB (an “inhibiting receptor”), which have similar amino acid sequences that differ primarily in the cytoplasmic domains thereof. Activating receptor FcγRIIA contains an immunoreceptor tyrosine-based activation motif (ITAM) in its cytoplasmic domain. Inhibiting receptor FcγRIIB contains an immunoreceptor tyrosine-based inhibition motif (ITIM) in its cytoplasmic domain. (see Daëron, <i>Annu. Rev. Immunol. </i>15:203-234 (1997)). FcRs are reviewed in Ravetch and Kinet, <i>Annu. Rev. Immunol </i>9:457-92 (1991); Capel et al., <i>Immunomethods </i>4:25-34 (1994); and de Haas et al., <i>J. Lab. Clin. Med. </i>126:330-41 (1995). Other FcRs, including those to be identified in the future, are encompassed by the term “FcR” herein. The term also includes the neonatal receptor, FcRn, which is responsible for the transfer of maternal IgGs to the fetus (Guyer et al., <i>J. Immunol. </i>117:587 (1976) and Kim et al., <i>J. Immunol. </i>24:249 (1994)).</div>
</li> <li> <para-num num="[0089]"> </para-num> <div class="description-line" id="p-0090" num="0089">The term “sequential” as used herein refers to having no chromatography step between step (a) and (b) of the method.</div>
</li> <li> <para-num num="[0090]"> </para-num> <div class="description-line" id="p-0091" num="0090">The term “continuous” as used herein refers to having the cation exchange material and the mixed mode material either directly connected or some other mechanism which allows for continuous flow between the cation exchange material and the mixed mode material.</div>
</li> <li> <para-num num="[0091]"> </para-num> <div class="description-line" id="p-0092" num="0091">“Contaminants” refer to materials that are different from the desired polypeptide product. The contaminant includes, without limitation: host cell materials, such as CHOP; leached Protein A; nucleic acid; a variant, fragment, aggregate or derivative of the desired polypeptide; another polypeptide; endotoxin; viral contaminant; cell culture media component, etc.</div>
</li> <li> <para-num num="[0092]"> </para-num> <div class="description-line" id="p-0093" num="0092">Reference to “about” a value or parameter herein includes (and describes) variations that are directed to that value or parameter per se. For example, description referring to “about X” includes description of “X”.</div>
</li> <li> <para-num num="[0093]"> </para-num> <div class="description-line" id="p-0094" num="0093">As used herein and in the appended claims, the singular forms “a,” “or,” and “the” include plural referents unless the context clearly dictates otherwise. It is understood that aspects and variations of the invention described herein include “consisting” and/or “consisting essentially of” aspects and variations.</div>
</li> <heading id="h-0008">II. Methods of Purification</heading>
<li> <para-num num="[0094]"> </para-num> <div class="description-line" id="p-0095" num="0094">Provided herein are methods for purifying a polypeptide from a composition comprising the polypeptide and at least one contaminant. In particular, the methods comprise using an overloaded cation exchange material. For example, the methods comprise loading onto a cation exchange material at a loading density of greater than about 150 g/L of cation exchange material.</div>
</li> <li> <para-num num="[0095]"> </para-num> <div class="description-line" id="p-0096" num="0095">The methods of purifying provided herein may further comprise loading onto a mixed mode material. For example, in some embodiments, the methods comprise the sequential steps of (a) loading the composition onto a cation exchange material at a loading density of greater than about 150 g/L of cation exchange material; and (b) loading a composition recovered from the cation exchange material onto a mixed mode material. In another example, in some embodiments, the methods comprise the sequential steps of (a) loading the composition onto a mixed mode material; and (b) loading a composition recovered from the mixed mode material onto a cation exchange material at a loading density of greater than about 150 g/L of cation exchange material. In some embodiments of any of the methods described herein, the sequential steps are continuous. In some embodiments of any of the methods described herein, the sequential steps are discontinuous. In some embodiments, the continuous purification utilizes the same flow rate, conductivity, and/or pH.</div>
</li> <li> <para-num num="[0096]"> </para-num> <div class="description-line" id="p-0097" num="0096">The methods described above may further comprise the step of loading onto a Protein A affinity chromatography material. The step of loading onto a Protein A affinity chromatography material is generally, but not necessarily, performed before the other chromatography step(s). In some embodiments, the step of loading onto a Protein A affinity chromatography material may be combined with the sequential steps of overloaded cation exchange and mixed mode, in any order, chromatography. In some embodiments, the sequential steps are continuous. In some embodiments, the continuous purification utilizes the same flow rate, conductivity, and/or pH.</div>
</li> <li> <para-num num="[0097]"> </para-num> <div class="description-line" id="p-0098" num="0097">Cation exchange material is a solid phase that is negatively charged and has free cations for exchange with cations in an aqueous solution passed over or through the solid phase. In some embodiments of any of the methods described herein, the cation exchange material may be a membrane, a monolith, or resin. In a preferred embodiment, the cation exchange material may be a resin. The cation exchange material may comprise a carboxylic acid functional group or a sulfonic acid functional group such as, but not limited to, sulfonate, carboxylic, carboxymethyl sulfonic acid, sulfoisobutyl, sulfoethyl, carboxyl, sulphopropyl, sulphonyl, sulphoxyethyl, or orthophosphate.</div>
</li> <li> <para-num num="[0098]"> </para-num> <div class="description-line" id="p-0099" num="0098">In some embodiments of any of the methods described herein, the cation exchange material may utilize a conventional chromatography material or a convective chromatography material. The conventional chromatography materials include, for example, perfusive materials (e.g., poly(styrene-divinylbenzene) resin) and diffusive materials (e.g., cross-linked agarose resin). In some embodiments, the poly(styrene-divinylbenzene) resin can be Poros HS resin. The Poros HS resin can be Poros HS 50 μm or Poros HS 20 μm particles. In some embodiments, the cross-linked agarose resin may be sulphopropyl-Sepharose Fast Flow (“SPSFF”) resin. The convective chromatography material may be a membrane (e.g., polyethersulfone) or monolith material (e.g. cross-linked polymer). The polyethersulfone membrane may be Mustang S. The cross-linked polymer monolith material may be cross-linked poly(glycidyl methacrylate-co-ethylene dimethacrylate), e.g., monolith SO3.</div>
</li> <li> <para-num num="[0099]"> </para-num> <div class="description-line" id="p-0100" num="0099">Examples of cation exchange materials are known in the art include, but are not limited to Mustang S, Sartobind S, SO3 Monolith, S Ceramic HyperD, Poros HS50, Poros HS20, SPSFF, SP-Sepharose XL (SPXL), CM Sepharose Fast Flow, Capto S, Fractogel Se HiCap, Fractogel SO3, or Fractogel COO. In some embodiments of any of the methods described herein, the cation exchange material is Poros HS50. In some embodiments, the Poros HS resin may be Poros HS 50 μm or Poros HS 20 μm particles.</div>
</li> <li> <para-num num="[0100]"> </para-num> <div class="description-line" id="p-0101" num="0100">In some embodiments of any of the methods described herein, the mixed mode material comprises functional groups capable of one of more of the following functionalities: anionic exchange, hydrogen bonding, and hydrophobic interactions. In some embodiments, the mixed mode material comprises functional groups capable of anionic exchange and hydrophobic interactions. The mixed mode material may contain N-benzyl-N-methyl ethanol amine, 4-mercapto-ethyl-pyridine, hexylamine, or phenylpropylamine as ligand or contain cross-linked polyallylamine. Examples of the mixed mode materials include Capto-Adhere, MEP HyperCel, HEA HyperCel or PPA HyperCel resin or ChromaSorb membrane. In some embodiments, the mixed mode material is Capto-Adhere resin.</div>
</li> <li> <para-num num="[0101]"> </para-num> <div class="description-line" id="p-0102" num="0101">In some embodiments, provided herein are methods for purifying a polypeptide from a composition comprising the polypeptide and at least one contaminant, wherein the method comprises either (i) or (ii): (i) sequential steps of (a) loading the composition onto Poros HS50 at a loading density of greater than about 150 g/L of resin; and (b) loading a composition recovered from the Poros HS50 onto Capto-Adhere; or (ii) sequential steps of (a) loading the composition onto Capto-Adhere; and (b) loading a composition recovered from Capto-Adhere onto Poros HS50 at a loading density of greater than about 150 g/L of resin.</div>
</li> <li> <para-num num="[0102]"> </para-num> <div class="description-line" id="p-0103" num="0102">In some embodiments of any of the methods described herein, the composition is loaded onto a cation exchange material at a loading density of greater than about any of 150 g/L, 200 g/L, 300 g/L, 400 g/L, 500 g/L, 550 g/L, 600 g/L, 650 g/L, 700 g/L, 800 g/L, 900 g/L, or 1000 g/L of cation exchange material. The composition may be loaded onto a cation exchange material at a loading density of between about any of 150 g/L and 2000 g/L, 150 g/L and 1500 g/L, 150 g/L and 1000 g/L, 200 g/L and 1500 g/L, 300 g/L and 1500 g/L, 400 g/L and 1000 g/L, or 500 g/L and 1000 g/L of cation exchange material. In some embodiments, the composition is loaded onto a cation exchange material at a loading density of about any of 150 g/L, 300 g/L, 500 g/L, 550 g/L, 600 g/L, 650 g/L, 700 g/L, 800 g/L, 850 g/L, 900 g/L, 1000 g/L, 1500 g/L, or 2000 g/L of cation exchange material.</div>
</li> <li> <para-num num="[0103]"> </para-num> <div class="description-line" id="p-0104" num="0103">In some embodiments of any of the methods described herein, the composition is loaded onto a mixed mode material at a loading density of greater than about any of 25 g/L, 50 g/L, 75 g/L, 100 g/L, 150 g/L, 200 g/L, 300 g/L, 400 g/L, 500 g/L, or 550 g/L of mixed mode material. The composition is loaded onto a mixed mode material at a loading density of between about any of 25 g/L and 1000 g/L, 25 g/L and 700 g/L, or 25 g/L and 500 g/L of mixed mode material.</div>
</li> <li> <para-num num="[0104]"> </para-num> <div class="description-line" id="p-0105" num="0104">Various buffers which can be employed depending, for example, on the desired pH of the buffer, the desired conductivity of the buffer, the characteristics of the protein of interest, and the purification method. In some embodiments of any of the methods described herein, the methods comprise using a buffer. The buffer can be a loading buffer, an equilibration buffer, or a wash buffer. In some embodiments, one or more of the loading buffer, the equilibration buffer, and/or the wash buffer are the same. In some embodiments, the loading buffer, the equilibration buffer, and/or the wash buffer are different. In some embodiments of any of the methods described herein, the buffer comprises a salt. The buffer may comprise sodium chloride, sodium acetate, or a mixture thereof. In some embodiments, the buffer is a sodium chloride buffer. In some embodiments, the buffer is a sodium acetate buffer.</div>
</li> <li> <para-num num="[0105]"> </para-num> <div class="description-line" id="p-0106" num="0105">Load, as used herein, is the composition loaded onto a chromatography material. Loading buffer is the buffer used to load the composition comprising the polypeptide of interest onto a chromatography material. The chromatography material may be equilibrated with an equilibration buffer prior to loading the composition which is to be purified. The wash buffer is used after loading the composition onto a chromatography material to elute the polypeptide of interest from the solid phase.</div>
</li> <li> <para-num num="[0106]"> </para-num> <div class="description-line" id="p-0107" num="0106">Conductivity refers to the ability of an aqueous solution to conduct an electric current between two electrodes. In solution, the current flows by ion transport. Therefore, with an increasing amount of ions present in the aqueous solution, the solution will have a higher conductivity. The basic unit of measure for conductivity is the Siemen (or mho), mho (mS/cm), and can be measured using a conductivity meter, such as various models of Orion conductivity meters. Since electrolytic conductivity is the capacity of ions in a solution to carry electrical current, the conductivity of a solution may be altered by changing the concentration of ions therein. For example, the concentration of a buffering agent and/or the concentration of a salt (e.g. sodium chloride, sodium acetate, or potassium chloride) in the solution may be altered in order to achieve the desired conductivity. Preferably, the salt concentration of the various buffers is modified to achieve the desired conductivity.</div>
</li> <li> <para-num num="[0107]"> </para-num> <div class="description-line" id="p-0108" num="0107">In some embodiments of any of the methods described herein, the conductivity has a conductivity of greater than about any of 2 mS/cm, 5 mS/cm, 7.5 mS/cm, or 10 mS/cm. The conductivity may be between about any of 2 mS/cm and 25 mS/cm, 2 mS/cm and 10 mS/cm, 3 mS/cm and 8 mS/cm, 2 mS/cm and 6 mS/cm, 4 mS/cm and 6 mS/cm, or 2 mS/cm and 4 mS/cm. In some embodiments, the conductivity is about any of 2 mS/cm, 3 mS/cm, 4 mS/cm, 5 mS/cm, 6 mS/cm, 8 mS/cm, or 10 mS/cm. In one aspect, the conductivity is the conductivity of the loading buffer, the equilibration buffer, and/or the wash buffer. In some embodiments, the conductivity of one or more of the loading buffer, the equilibration buffer, and the wash buffer are the same. In some embodiments, the conductivity of the loading buffer is different from the conductivity of the wash buffer and/or equilibration buffer.</div>
</li> <li> <para-num num="[0108]"> </para-num> <div class="description-line" id="p-0109" num="0108">In some embodiments of any of the methods described herein, the buffer has a pH of less than about any of 10, 9, 8, 7, or 6. The buffer may have a pH of between about any of 3 and 10, 4 and 8, 4 and 6, or 5 and 6. In some embodiments, the pH is about any of 4, 4.5, 5, 5.5, 6, 6.5, 7, 7.5, or 8. The pH can be the pH of the loading buffer, the equilibration buffer, or the wash buffer. In some embodiments, the pH of one or more of the loading buffer, the equilibration buffer, and/or the wash buffer are the same. In some embodiments, the pH of the loading buffer is different from the pH of the equilibration buffer and/or the wash buffer.</div>
</li> <li> <para-num num="[0109]"> </para-num> <div class="description-line" id="p-0110" num="0109">In some embodiments of any of the methods described herein, the flow rate is less than about any of 50 CV/hr, 40 CV/hr, or 30 CV/hr. The flow rate may be between about any of 5 CV/hr and 50 CV/hr, 10 CV/hr and 40 CV/hr, or 18 CV/hr and 36 CV/hr. In some embodiments, the flow rate is about any of 9 CV/hr, 18 CV/hr, 25 CV/hr, 30 CV/hr, 36 CV/hr, or 40 CV/hr. In some embodiments of any of the methods described herein, the flow rate is less than about any of 100 cm/hr, 75 cm/hr, or 50 cm/hr. The flow rate may be between about any of 25 cm/hr and 150 cm/hr, 25 cm/hr and 100 cm/hr, 50 cm/hr and 100 cm/hr, or 65 cm/hr and 85 cm/hr. The flow rate can be the flow rate over the cation exchange material or the flow rate over the mixed mode material. In some embodiments, the flow rate over the cation exchange material is the same as the flow rate over the mixed mode material. In some embodiments, the flow rate over the cation exchange material is different from the flow rate over the mixed mode material.</div>
</li> <li> <para-num num="[0110]"> </para-num> <div class="description-line" id="p-0111" num="0110">Bed height is the height of chromatography material used. In some embodiments of any of the method described herein, the bed height is greater than about any of 3 cm, 10 cm, or 15 cm. The bed height may be between about any of 3 cm and 35 cm, 5 cm and 15 cm, 3 cm and 10 cm, or 5 cm and 8 cm. In some embodiments, the bed height is about any of 3 cm, 5 cm, 10 cm, or 15 cm. In some embodiments, the bed height of the cation exchange material is the same as the bed height of the mixed mode material. In some embodiments, the bed height of the cation exchange material is different from the bed height of the mixed mode material.</div>
</li> <li> <para-num num="[0111]"> </para-num> <div class="description-line" id="p-0112" num="0111">In some embodiments of any of the methods described herein, the at least one contaminant is any one or more of CHOP, leached protein A, DNA, aggregated protein, cell culture media component, gentamicin, and viral contaminant.</div>
</li> <li> <para-num num="[0112]"> </para-num> <div class="description-line" id="p-0113" num="0112">CHOP are proteins from host cells, i.e., Chinese Hamster Ovary Proteins. The amount of CHOP may be measured by enzyme-linked immunosorbent assay (“ELISA”) or Meso Scale Discovery (“MSO”). In some embodiments of any of the methods described herein, the amount of CHOP is reduced by greater than about any of 10%, 20%, 30%, 40%, 50%, 60%, 70%, 80%, 90%, or 95%. The amount of CHOP may be reduced by between about any of 10% and 99%, 30% and 95%, 30% and 99%, 50% and 95%, 50% and 99%, 75% and 99%, or 85% and 99%. In some embodiments, the amount of CHOP is reduced by about any of 10%, 20%, 30%, 40%, 50%, 60%, 70%, 80%, 85%, 90%, 95%, or 98%. In some embodiments, the reduction is determined by comparing the amount of CHOP in the composition recovered from a purification step(s) to the amount of CHOP in the composition before the purification step(s).</div>
</li> <li> <para-num num="[0113]"> </para-num> <div class="description-line" id="p-0114" num="0113">Aggregated polypeptide can be high molecular weight (HMW) protein. In some embodiments, the aggregated polypeptide is multimers of the polypeptide of interest. The HMW may be a dimmer, up to 8× monomer, or larger of the polypeptide of interest. Methods of measuring aggregated protein (e.g., HMW) are known in the art and described in the examples section. In some embodiments of any of the methods described herein, the amount of aggregated protein is reduced by greater than about any of 5%, 10%, 20%, 30%, 40%, 50%, 60%, 70%, 80%, 90%, or 95%. The amount of aggregated protein may be reduced by between about any of 10% and 99%, 30% and 95%, 30% and 99%, 50% and 95%, 50% and 99%, 75% and 99%, or 85% and 99%. The amount of aggregated protein may be reduced by about any of 5%, 10%, 20%, 30%, 40%, 50%, 60%, 70%, 80%, 90%, or 95%. In some embodiments, the reduction is determined by comparing the amount of aggregated protein (e.g., HMW) in the composition recovered from a purification step(s) to the amount of aggregated protein (e.g., HMW) in the composition before the purification step(s).</div>
</li> <li> <para-num num="[0114]"> </para-num> <div class="description-line" id="p-0115" num="0114">Leached Protein A is Protein A detached or washed from a solid phase to which it is bound. For example, leached Protein A can be leached from Protein A chromatography column. The amount of Protein A may be measured, for example, by ELISA. In some embodiments of any of the methods described herein, the amount of leached Protein A is reduced by greater than about any of 10%, 20%, 30%, 40%, 50%, 60%, 70%, 80%, or 90%. The amount of leached Protein A may be reduced by between about any of 10% and 99%, 30% and 95%, 30% and 99%, 50% and 95%, 50% and 99%, 75% and 99%, or 85% and 99%. In some embodiments, the amount of leached Protein A is reduced by about any of 10%, 20%, 30%, 40%, 50%, 60%, 70%, 80%, 90%, or 95%. In some embodiments, the reduction is determined by comparing the amount of leached Protein A in the composition recovered from a purification step(s) to the amount of leached Protein A in the composition before the purification step(s).</div>
</li> <li> <para-num num="[0115]"> </para-num> <div class="description-line" id="p-0116" num="0115">Methods of measuring DNA such as CHO cell DNA are known in the art and described in the examples section. In some embodiments of any of the methods described herein, the amount of DNA is reduced by greater than about any of 10%, 20%, 30%, 40%, 50%, 60%, 70%, 80%, or 90%. The amount of DNA may be reduced by between about any of 10% and 99%, 30% and 95%, 30% and 99%, 50% and 95%, 50% and 99%, 75% and 99%, or 85% and 99%. The amount of DNA may be reduced by about any of 10%, 20%, 30%, 40%, 50%, 60%, 70%, 80%, 90%, 95%, or 99%. In some embodiments, the reduction is determined by comparing the amount of DNA in the composition recovered from a purification step(s) to the amount of DNA in the composition before the purification step(s).</div>
</li> <li> <para-num num="[0116]"> </para-num> <div class="description-line" id="p-0117" num="0116">Cell culture media component refers to a component present in a cell culture media. A cell culture media may be a cell culture media at the time of harvesting cells. In some embodiments, the cell culture media component is gentamicin. The amount of gentamicin may be measured by ELISA. In some embodiments of any of the methods described herein, the amount of cell culture media component is reduced by greater than about any of 10%, 20%, 30%, 40%, 50%, 60%, 70%, 80%, or 90%. The amount of cell culture media component may be reduced by between about any of 10% and 99%, 30% and 95%, 30% and 99%, 50% and 95%, 50% and 99%, 75% and 99%, or 85% and 99%. In some embodiments, the amount of cell culture media component is reduced by about any of 10%, 20%, 30%, 40%, 50%, 60%, 70%, 80%, 90%, 95%, or 98%. In some embodiments, the reduction is determined by comparing the amount of cell culture media component in the composition recovered from a purification step(s) to the amount of cell culture media component in the composition before the purification step(s).</div>
</li> <li> <para-num num="[0117]"> </para-num> <div class="description-line" id="p-0118" num="0117">In some embodiments of any of the methods described herein, the methods may further comprise one or more purification steps either prior to, or after, any of the chromatography steps described herein. In some embodiments, the methods further comprise subjecting the composition comprising the polypeptide to one or more further purification steps either before or after steps (a) and (b). Other purification procedures include, for example, hydroxylapatite chromatography; gel filtration chromatography; affinity chromatography; gel electrophoresis; dialysis; ethanol precipitation; reverse phase HPLC; chromatography on silica; chromatofocusing; SDS-PAGE; ammonium sulfate precipitation; and metal chelating columns to bind epitope-tagged forms of the polypeptide.</div>
</li> <li> <para-num num="[0118]"> </para-num> <div class="description-line" id="p-0119" num="0118">In some embodiments of any of the methods described herein, the methods further comprise recovering the purified polypeptide. In some embodiments, the purified polypeptide is recovered from any of the purification steps described herein. The chromatography step may be cation exchange chromatography, mixed mode chromatography, or Protein A chromatography.</div>
</li> <li> <para-num num="[0119]"> </para-num> <div class="description-line" id="p-0120" num="0119">In some embodiments of any of the methods described herein, the methods further comprise combining the purified polypeptide of the methods of purification with a pharmaceutically acceptable carrier.</div>
</li> <heading id="h-0009">III. Polypeptides</heading>
<li> <para-num num="[0120]"> </para-num> <div class="description-line" id="p-0121" num="0120">Polypeptides are provided for use in any of the methods of purifying polypeptides and formulations comprising the polypeptides purified by the methods described herein.</div>
</li> <li> <para-num num="[0121]"> </para-num> <div class="description-line" id="p-0122" num="0121">In some embodiments, the polypeptide is a therapeutic polypeptide. The therapeutic polypeptide may inhibit the growth of tumor cells, induce apoptosis, and/or induce cell death. In some embodiments, the polypeptide is an antagonist. In some embodiments, the polypeptide is an agonist. In some embodiments, the polypeptide is an antibody.</div>
</li> <li> <para-num num="[0122]"> </para-num> <div class="description-line" id="p-0123" num="0122">In some embodiments, the polypeptide has a molecular weight of greater than about any of 5,000 Daltons, 10,000 Daltons, 15,000 Daltons, 25,000 Daltons, 50,000 Daltons, 75,000 Daltons, 100,000 Dalton, 125,000 Daltons, or 150,000 Daltons. The polypeptide may have a molecular weight between about any of 50,000 Daltons to 200,000 Daltons or 100,000 Daltons to 200,000 Daltons. Alternatively, the polypeptide for use herein may have a molecular weight of about 120,000 Daltons or about 25,000 Daltons.</div>
</li> <li> <para-num num="[0123]"> </para-num> <div class="description-line" id="p-0124" num="0123">pI is the isoelectric point and is the pH at which a particular molecule or surface carries no net electrical charge. In some embodiments of any of the methods described herein, the pI of the polypeptide may be between about any of 6 to 10, 7 to 9, or 8 to 9. In some embodiments, the polypeptide has a pI of about any of 6, 7, 7.5, 8, 8.5, 9, 9.5, or 10.</div>
</li> <li> <para-num num="[0124]"> </para-num> <div class="description-line" id="p-0125" num="0124">The polypeptides to be purified using the methods described herein is generally produced using recombinant techniques. Methods for producing recombinant proteins are described, e.g., in U.S. Pat. Nos. 5,534,615 and 4,816,567, specifically incorporated herein by reference. In some embodiments, the protein of interest is produced in a CHO cell (see, e.g. WO 94/11026). When using recombinant techniques, the polypeptides can be produced intracellularly, in the periplasmic space, or directly secreted into the medium.</div>
</li> <li> <para-num num="[0125]"> </para-num> <div class="description-line" id="p-0126" num="0125">The polypeptides may be recovered from culture medium or from host cell lysates. Cells employed in expression of the polypeptides can be disrupted by various physical or chemical means, such as freeze-thaw cycling, sonication, mechanical disruption, or cell lysing agents. If the polypeptide is produced intracellularly, as a first step, the particulate debris, either host cells or lysed fragments, are removed, for example, by centrifugation or ultrafiltration. Carter et al., <i>Bio/Technology </i>10: 163-167 (1992) describe a procedure for isolating polypeptides which are secreted to the periplasmic space of <i>E. coli</i>. Briefly, cell paste is thawed in the presence of sodium acetate (pH 3.5), EDTA, and phenylmethylsulfonylfluoride (PMSF) over about 30 min Cell debris can be removed by centrifugation. Where the polypeptide is secreted into the medium, supernatants from such expression systems are generally first concentrated using a commercially available polypeptide concentration filter, for example, an Amicon or Millipore Pellicon ultrafiltration unit. A protease inhibitor such as PMSF may be included in any of the foregoing steps to inhibit proteolysis and antibiotics may be included to prevent the growth of adventitious contaminants.</div>
</li> <heading id="h-0010">(A) Antibodies</heading>
<li> <para-num num="[0126]"> </para-num> <div class="description-line" id="p-0127" num="0126">In some embodiments of any of the methods described herein, the polypeptide for use in any of the methods of purifying polypeptides and formulations comprising the polypeptides purified by the methods described herein is an antibody.</div>
</li> <li> <para-num num="[0127]"> </para-num> <div class="description-line" id="p-0128" num="0127">Molecular targets for antibodies include CD proteins and their ligands, such as, but not limited to: (i) CD3, CD4, CD8, CD19, CD11a, CD20, CD22, CD34, CD40, CD79a (CD79a), and CD79β (CD79b); (ii) members of the ErbB receptor family such as the EGF receptor, HER2, HER3 or HER4 receptor; (iii) cell adhesion molecules such as LFA-1, Mac1, p150,95, VLA-4, ICAM-1, VCAM and □v/□3 integrin, including either alpha or beta subunits thereof (e.g., anti-CD11a, anti-CD18 or anti-CD11b antibodies); (iv) growth factors such as VEGF; IgE; blood group antigens; flk2/flt3 receptor; obesity (OB) receptor; mpl receptor; CTLA-4; protein C, BR3, c-met, tissue factor, □7 etc; and (v) cell surface and transmembrane tumor-associated antigens (TAA), such as those described in U.S. Pat. No. 7,521,541.</div>
</li> <li> <para-num num="[0128]"> </para-num> <div class="description-line" id="p-0129" num="0128">Other exemplary antibodies include those selected from, and without limitation, anti-estrogen receptor antibody, anti-progesterone receptor antibody, anti-p53 antibody, anti-HER-2/neu antibody, anti-EGFR antibody, anti-cathepsin D antibody, anti-Bcl-2 antibody, anti-E-cadherin antibody, anti-CA125 antibody, anti-CA15-3 antibody, anti-CA19-9 antibody, anti-c-erbB-2 antibody, anti-P-glycoprotein antibody, anti-CEA antibody, anti-retinoblastoma protein antibody, anti-ras oncoprotein antibody, anti-Lewis X antibody, anti-Ki-67 antibody, anti-PCNA antibody, anti-CD3 antibody, anti-CD4 antibody, anti-CD5 antibody, anti-CD7 antibody, anti-CD8 antibody, anti-CD9/p24 antibody, anti-CD10 antibody, anti-CD11a antibody, anti-CD11c antibody, anti-CD13 antibody, anti-CD14 antibody, anti-CD15 antibody, anti-CD19 antibody, anti-CD20 antibody, anti-CD22 antibody, anti-CD23 antibody, anti-CD30 antibody, anti-CD31 antibody, anti-CD33 antibody, anti-CD34 antibody, anti-CD35 antibody, anti-CD38 antibody, anti-CD41 antibody, anti-LCA/CD45 antibody, anti-CD45RO antibody, anti-CD45RA antibody, anti-CD39 antibody, anti-CD100 antibody, anti-CD95/Fas antibody, anti-CD99 antibody, anti-CD106 antibody, anti-ubiquitin antibody, anti-CD71 antibody, anti-c-myc antibody, anti-cytokeratins antibody, anti-vimentins antibody, anti-HPV proteins antibody, anti-kappa light chains antibody, anti-lambda light chains antibody, anti-melanosomes antibody, anti-prostate specific antigen antibody, anti-S-100 antibody, anti-tau antigen antibody, anti-fibrin antibody, anti-keratins antibody and anti-Tn-antigen antibody.</div>
</li> <heading id="h-0011">(i) Polyclonal Antibodies</heading>
<li> <para-num num="[0129]"> </para-num> <div class="description-line" id="p-0130" num="0129">In some embodiments, the antibodies are polyclonal antibodies. Polyclonal antibodies are preferably raised in animals by multiple subcutaneous (sc) or intraperitoneal (ip) injections of the relevant antigen and an adjuvant. It may be useful to conjugate the relevant antigen to a polypeptide that is immunogenic in the species to be immunized, e.g., keyhole limpet hemocyanin, serum albumin, bovine thyroglobulin, or soybean trypsin inhibitor using a bifunctional or derivatizing agent, for example, maleimidobenzoyl sulfosuccinimide ester (conjugation through cysteine residues), N-hydroxysuccinimide (through lysine residues), glutaraldehyde, succinic anhydride, SOCl<sub>2</sub>, or R<sup>1</sup>N═C═NR, where R and R′ are different alkyl groups.</div>
</li> <li> <para-num num="[0130]"> </para-num> <div class="description-line" id="p-0131" num="0130">Animals are immunized against the antigen, immunogenic conjugates, or derivatives by combining, e.g., 100 μg or 5 μg of the polypeptide or conjugate (for rabbits or mice, respectively) with 3 volumes of Freund's complete adjuvant and injecting the solution intradermally at multiple sites. One month later the animals are boosted with ⅕ to 1/10 the original amount of peptide or conjugate in Freund's complete adjuvant by subcutaneous injection at multiple sites. Seven to 14 days later the animals are bled and the serum is assayed for antibody titer. Animals are boosted until the titer plateaus. In some embodiments, the animal is boosted with the conjugate of the same antigen, but conjugated to a different polypeptide and/or through a different cross-linking reagent. Conjugates also can be made in recombinant cell culture as polypeptide fusions. Also, aggregating agents such as alum are suitably used to enhance the immune response.</div>
</li> <li> <div class="description-line" id="p-0132" num="0000">(iii) Monoclonal Antibodies</div>
</li> <li> <para-num num="[0131]"> </para-num> <div class="description-line" id="p-0133" num="0131">In some embodiments, the antibodies are monoclonal antibodies. Monoclonal antibodies are obtained from a population of substantially homogeneous antibodies, i.e., the individual antibodies comprising the population are identical and/or bind the same epitope except for possible variants that arise during production of the monoclonal antibody, such variants generally being present in minor amounts. Thus, the modifier “monoclonal” indicates the character of the antibody as not being a mixture of discrete or polyclonal antibodies.</div>
</li> <li> <para-num num="[0132]"> </para-num> <div class="description-line" id="p-0134" num="0132">For example, the monoclonal antibodies may be made using the hybridoma method first described by Kohler et al., <i>Nature </i>256:495 (1975), or may be made by recombinant DNA methods (U.S. Pat. No. 4,816,567).</div>
</li> <li> <para-num num="[0133]"> </para-num> <div class="description-line" id="p-0135" num="0133">In the hybridoma method, a mouse or other appropriate host animal, such as a hamster, is immunized as herein described to elicit lymphocytes that produce or are capable of producing antibodies that will specifically bind to the polypeptide used for immunization. Alternatively, lymphocytes may be immunized in vitro. Lymphocytes then are fused with myeloma cells using a suitable fusing agent, such as polyethylene glycol, to form a hybridoma cell (Goding, Monoclonal Antibodies: Principles and Practice, pp. 59-103 (Academic Press, 1986)).</div>
</li> <li> <para-num num="[0134]"> </para-num> <div class="description-line" id="p-0136" num="0134">The hybridoma cells thus prepared are seeded and grown in a suitable culture medium that preferably contains one or more substances that inhibit the growth or survival of the unfused, parental myeloma cells. For example, if the parental myeloma cells lack the enzyme hypoxanthine guanine phosphoribosyl transferase (HGPRT or HPRT), the culture medium for the hybridomas typically will include hypoxanthine, aminopterin, and thymidine (HAT medium), which substances prevent the growth of HGPRT-deficient cells.</div>
</li> <li> <para-num num="[0135]"> </para-num> <div class="description-line" id="p-0137" num="0135">In some embodiments, the myeloma cells are those that fuse efficiently, support stable high-level production of antibody by the selected antibody-producing cells, and are sensitive to a medium such as HAT medium. Among these, in some embodiments, the myeloma cell lines are murine myeloma lines, such as those derived from MOPC-21 and MPC-11 mouse tumors available from the Salk Institute Cell Distribution Center, San Diego, Calif. USA, and SP-2 or X63-Ag8-653 cells available from the American Type Culture Collection, Rockville, Md. USA. Human myeloma and mouse-human heteromyeloma cell lines also have been described for the production of human monoclonal antibodies (Kozbor, <i>J. Immunol. </i>133:3001 (1984); Brodeur et al., <i>Monoclonal Antibody Production Techniques and Applications </i>pp. 51-63 (Marcel Dekker, Inc., New York, 1987)).</div>
</li> <li> <para-num num="[0136]"> </para-num> <div class="description-line" id="p-0138" num="0136">Culture medium in which hybridoma cells are growing is assayed for production of monoclonal antibodies directed against the antigen. In some embodiments, the binding specificity of monoclonal antibodies produced by hybridoma cells is determined by immunoprecipitation or by an in vitro binding assay, such as radioimmunoassay (RIA) or enzyme-linked immunoabsorbent assay (ELISA).</div>
</li> <li> <para-num num="[0137]"> </para-num> <div class="description-line" id="p-0139" num="0137">The binding affinity of the monoclonal antibody can, for example, be determined by the Scatchard analysis of Munson et al., <i>Anal. Biochem. </i>107:220 (1980).</div>
</li> <li> <para-num num="[0138]"> </para-num> <div class="description-line" id="p-0140" num="0138">After hybridoma cells are identified that produce antibodies of the desired specificity, affinity, and/or activity, the clones may be subcloned by limiting dilution procedures and grown by standard methods (Goding, <i>Monoclonal Antibodies: Principles and Practice </i>pp. 59-103 (Academic Press, 1986)). Suitable culture media for this purpose include, for example, D-MEM or RPMI-1640 medium. In addition, the hybridoma cells may be grown in vivo as ascites tumors in an animal</div>
</li> <li> <para-num num="[0139]"> </para-num> <div class="description-line" id="p-0141" num="0139">The monoclonal antibodies secreted by the subclones are suitably separated from the culture medium, ascites fluid, or serum by conventional immunoglobulin purification procedures such as, for example, polypeptide A-Sepharose, hydroxylapatite chromatography, gel electrophoresis, dialysis, or affinity chromatography.</div>
</li> <li> <para-num num="[0140]"> </para-num> <div class="description-line" id="p-0142" num="0140">DNA encoding the monoclonal antibodies is readily isolated and sequenced using conventional procedures (e.g., by using oligonucleotide probes that are capable of binding specifically to genes encoding the heavy and light chains of murine antibodies). In some embodiments, the hybridoma cells serve as a source of such DNA. Once isolated, the DNA may be placed into expression vectors, which are then transfected into host cells such as <i>E. coli </i>cells, simian COS cells, Chinese Hamster Ovary (CHO) cells, or myeloma cells that do not otherwise produce immunoglobulin polypeptide, to obtain the synthesis of monoclonal antibodies in the recombinant host cells. Review articles on recombinant expression in bacteria of DNA encoding the antibody include Skerra et al., <i>Curr. Opinion in Immunol. </i>5:256-262 (1993) and Plückthun, <i>Immunol. Revs., </i>130:151-188 (1992).</div>
</li> <li> <para-num num="[0141]"> </para-num> <div class="description-line" id="p-0143" num="0141">In a further embodiment, antibodies or antibody fragments can be isolated from antibody phage libraries generated using the techniques described in McCafferty et al., <i>Nature </i>348:552-554 (1990). Clackson et al., <i>Nature </i>352:624-628 (1991) and Marks et al., <i>J. Mol. Biol. </i>222:581-597 (1991) describe the isolation of murine and human antibodies, respectively, using phage libraries. Subsequent publications describe the production of high affinity (nM range) human antibodies by chain shuffling (Marks et al., <i>Bio/Technology </i>10:779-783 (1992)), as well as combinatorial infection and in vivo recombination as a strategy for constructing very large phage libraries (Waterhouse et al., <i>Nuc. Acids. Res. </i>21:2265-2266 (1993)). Thus, these techniques are viable alternatives to traditional monoclonal antibody hybridoma techniques for isolation of monoclonal antibodies.</div>
</li> <li> <para-num num="[0142]"> </para-num> <div class="description-line" id="p-0144" num="0142">The DNA also may be modified, for example, by substituting the coding sequence for human heavy- and light chain constant domains in place of the homologous murine sequences (U.S. Pat. No. 4,816,567; Morrison et al., <i>Proc. Natl Acad. Sci. USA </i>81:6851 (1984)), or by covalently joining to the immunoglobulin coding sequence all or part of the coding sequence for a non-immunoglobulin polypeptide.</div>
</li> <li> <para-num num="[0143]"> </para-num> <div class="description-line" id="p-0145" num="0143">Typically such non-immunoglobulin polypeptides are substituted for the constant domains of an antibody, or they are substituted for the variable domains of one antigen-combining site of an antibody to create a chimeric bivalent antibody comprising one antigen-combining site having specificity for an antigen and another antigen-combining site having specificity for a different antigen.</div>
</li> <li> <para-num num="[0144]"> </para-num> <div class="description-line" id="p-0146" num="0144">In some embodiments of any of the methods described herein, the antibody is IgA, IgD, IgE, IgG, or IgM. In some embodiments, the antibody is an IgG monoclonal antibody.</div>
</li> <heading id="h-0012">(iv) Humanized Antibodies</heading>
<li> <para-num num="[0145]"> </para-num> <div class="description-line" id="p-0147" num="0145">In some embodiments, the antibody is a humanized antibody. Methods for humanizing non-human antibodies have been described in the art. In some embodiments, a humanized antibody has one or more amino acid residues introduced into it from a source that is non-human. These non-human amino acid residues are often referred to as “import” residues, which are typically taken from an “import” variable domain. Humanization can be essentially performed following the method of Winter and co-workers (Jones et al., <i>Nature </i>321:522-525 (1986); Riechmann et al., <i>Nature </i>332:323-327 (1988); Verhoeyen et al., <i>Science </i>239:1534-1536 (1988)), by substituting hypervariable region sequences for the corresponding sequences of a human antibody. Accordingly, such “humanized” antibodies are chimeric antibodies (U.S. Pat. No. 4,816,567) wherein substantially less than an intact human variable domain has been substituted by the corresponding sequence from a non-human species. In practice, humanized antibodies are typically human antibodies in which some hypervariable region residues and possibly some FR residues are substituted by residues from analogous sites in rodent antibodies.</div>
</li> <li> <para-num num="[0146]"> </para-num> <div class="description-line" id="p-0148" num="0146">The choice of human variable domains, both light and heavy, to be used in making the humanized antibodies is very important to reduce antigenicity. According to the so-called “best-fit” method, the sequence of the variable domain of a rodent antibody is screened against the entire library of known human variable-domain sequences. The human sequence that is closest to that of the rodent is then accepted as the human framework region (FR) for the humanized antibody (Sims et al., <i>J. Immunol. </i>151:2296 (1993); Chothia et al., <i>J. Mol. Biol. </i>196:901 (1987)). Another method uses a particular framework region derived from the consensus sequence of all human antibodies of a particular subgroup of light or heavy chain variable regions. The same framework may be used for several different humanized antibodies (Carter et al., <i>Proc. Natl. Acad. Sci. USA </i>89:4285 (1992); Presta et al., <i>J. Immunol. </i>151:2623 (1993)).</div>
</li> <li> <para-num num="[0147]"> </para-num> <div class="description-line" id="p-0149" num="0147">It is further important that antibodies be humanized with retention of high affinity for the antigen and other favorable biological properties. To achieve this goal, in some embodiments of the methods, humanized antibodies are prepared by a process of analysis of the parental sequences and various conceptual humanized products using three-dimensional models of the parental and humanized sequences. Three-dimensional immunoglobulin models are commonly available and are familiar to those skilled in the art. Computer programs are available that illustrate and display probable three-dimensional conformational structures of selected candidate immunoglobulin sequences. Inspection of these displays permits analysis of the likely role of the residues in the functioning of the candidate immunoglobulin sequence, i.e., the analysis of residues that influence the ability of the candidate immunoglobulin to bind its antigen. In this way, FR residues can be selected and combined from the recipient and import sequences so that the desired antibody characteristic, such as increased affinity for the target antigen(s), is achieved. In general, the hypervariable region residues are directly and most substantially involved in influencing antigen binding.</div>
</li> <heading id="h-0013">(v) Human Antibodies</heading>
<li> <para-num num="[0148]"> </para-num> <div class="description-line" id="p-0150" num="0148">In some embodiments, the antibody is a human antibody. As an alternative to humanization, human antibodies can be generated. For example, it is now possible to produce transgenic animals (e.g., mice) that are capable, upon immunization, of producing a full repertoire of human antibodies in the absence of endogenous immunoglobulin production. For example, it has been described that the homozygous deletion of the antibody heavy chain joining region (J<sub>H</sub>) gene in chimeric and germ-line mutant mice results in complete inhibition of endogenous antibody production. Transfer of the human germ-line immunoglobulin gene array in such germ-line mutant mice will result in the production of human antibodies upon antigen challenge. See, e.g., Jakobovits et al., <i>Proc. Natl. Acad. Sci. USA </i>90:2551 (1993); Jakobovits et al., <i>Nature </i>362:255-258 (1993); Bruggermann et al., <i>Year in Immuno. </i>7:33 (1993); and U.S. Pat. Nos. 5,591,669; 5,589,369; and 5,545,807.</div>
</li> <li> <para-num num="[0149]"> </para-num> <div class="description-line" id="p-0151" num="0149">Alternatively, phage display technology (McCafferty et al., <i>Nature </i>348:552-553 (1990)) can be used to produce human antibodies and antibody fragments in vitro, from immunoglobulin variable (V) domain gene repertoires from unimmunized donors. According to this technique, antibody V domain genes are cloned in-frame into either a major or minor coat polypeptide gene of a filamentous bacteriophage, such as M13 or fd, and displayed as functional antibody fragments on the surface of the phage particle. Because the filamentous particle contains a single-stranded DNA copy of the phage genome, selections based on the functional properties of the antibody also result in selection of the gene encoding the antibody exhibiting those properties. Thus, the phage mimics some of the properties of the B cell. Phage display can be performed in a variety of formats; for their review see, e.g., Johnson, Kevin S. and Chiswell, David J., Current Opinion in Structural Biology 3:564-571 (1993). Several sources of V-gene segments can be used for phage display. Clackson et al., <i>Nature </i>352:624-628 (1991) isolated a diverse array of anti-oxazolone antibodies from a small random combinatorial library of V genes derived from the spleens of immunized mice. A repertoire of V genes from unimmunized human donors can be constructed and antibodies to a diverse array of antigens (including self-antigens) can be isolated essentially following the techniques described by Marks et al., <i>J. Mol. Biol. </i>222:581-597 (1991), or Griffith et al., <i>EMKO J. </i>12:725-734 (1993). See also, U.S. Pat. Nos. 5,565,332 and 5,573,905.</div>
</li> <li> <para-num num="[0150]"> </para-num> <div class="description-line" id="p-0152" num="0150">Human antibodies may also be generated by in vitro activated B cells (see U.S. Pat. Nos. 5,567,610 and 5,229,275).</div>
</li> <heading id="h-0014">(vi) Antibody Fragments</heading>
<li> <para-num num="[0151]"> </para-num> <div class="description-line" id="p-0153" num="0151">In some embodiments, the antibody is an antibody fragment. Various techniques have been developed for the production of antibody fragments. Traditionally, these fragments were derived via proteolytic digestion of intact antibodies (see, e.g., Morimoto et al., <i>Journal of Biochemical and Biophysical Methods </i>24:107-117 (1992) and Brennan et al., <i>Science </i>229:81 (1985)). However, these fragments can now be produced directly by recombinant host cells. For example, the antibody fragments can be isolated from the antibody phage libraries discussed above. Alternatively, Fab′-SH fragments can be directly recovered from <i>E. coli </i>and chemically coupled to form F(ab′)<sub>2 </sub>fragments (Carter et al., <i>Bio/Technology </i>10:163-167 (1992)). According to another approach, F(ab′)<sub>2 </sub>fragments can be isolated directly from recombinant host cell culture. Other techniques for the production of antibody fragments will be apparent to the skilled practitioner. In other embodiments, the antibody of choice is a single chain Fv fragment (scFv). See WO 93/16185; U.S. Pat. Nos. 5,571,894; and 5,587,458. The antibody fragment may also be a “linear antibody,” e.g., as described in U.S. Pat. No. 5,641,870 for example Such linear antibody fragments may be monospecific or bispecific.</div>
</li> <li> <para-num num="[0152]"> </para-num> <div class="description-line" id="p-0154" num="0152">In some embodiments, fragments of the antibodies described herein are provided. In some embodiments, the antibody fragment is an antigen binding fragment. In some embodiments, the antigen binding fragment is selected from the group consisting of a Fab fragment, a Fab′ fragment, a F(ab′)2 fragment, a scFv, a Fv, and a diabody.</div>
</li> <li> <div class="description-line" id="p-0155" num="0000">(vii) Bispecific Antibodies</div>
</li> <li> <para-num num="[0153]"> </para-num> <div class="description-line" id="p-0156" num="0153">In some embodiments, the antibody is a bispecific antibody. Bispecific antibodies are antibodies that have binding specificities for at least two different epitopes. Exemplary bispecific antibodies may bind to two different epitopes. Alternatively, a bispecific antibody binding arm may be combined with an arm that binds to a triggering molecule on a leukocyte such as a T-cell receptor molecule (e.g. CD2 or CD3), or Fc receptors for IgG (FcγR), such as FcγRI (CD64), FcγRII (CD32) and FcγRIII (CD16) so as to focus cellular defense mechanisms to the cell. Bispecific antibodies can be prepared as full length antibodies or antibody fragments (e.g. F(ab′)<sub>2</sub>bispecific antibodies).</div>
</li> <li> <para-num num="[0154]"> </para-num> <div class="description-line" id="p-0157" num="0154">Methods for making bispecific antibodies are known in the art. Traditional production of full length bispecific antibodies is based on the coexpression of two immunoglobulin heavy chain-light chain pairs, where the two chains have different specificities (Millstein et al., <i>Nature </i>305:537-539 (1983)). Because of the random assortment of immunoglobulin heavy and light chains, these hybridomas (quadromas) produce a potential mixture of 10 different antibody molecules, of which only one has the correct bispecific structure. Purification of the correct molecule, which is usually done by affinity chromatography steps, is rather cumbersome, and the product yields are low. Similar procedures are disclosed in WO 93/08829, and in Traunecker et al., <i>EMBO J., </i>10:3655-3659 (1991).</div>
</li> <li> <para-num num="[0155]"> </para-num> <div class="description-line" id="p-0158" num="0155">According to a different approach, antibody variable domains with the desired binding specificities (antibody-antigen combining sites) are fused to immunoglobulin constant domain sequences. In some embodiments, the fusion is with an immunoglobulin heavy chain constant domain, comprising at least part of the hinge, CH2, and CH3 regions. In some embodiments, the first heavy chain constant region (CH1) containing the site necessary for light chain binding, present in at least one of the fusions. DNAs encoding the immunoglobulin heavy chain fusions and, if desired, the immunoglobulin light chain, are inserted into separate expression vectors, and are co-transfected into a suitable host organism. This provides for great flexibility in adjusting the mutual proportions of the three polypeptide fragments in embodiments when unequal ratios of the three polypeptide chains used in the construction provide the optimum yields. It is, however, possible to insert the coding sequences for two or all three polypeptide chains in one expression vector when the expression of at least two polypeptide chains in equal ratios results in high yields or when the ratios are of no particular significance.</div>
</li> <li> <para-num num="[0156]"> </para-num> <div class="description-line" id="p-0159" num="0156">In some embodiments of this approach, the bispecific antibodies are composed of a hybrid immunoglobulin heavy chain with a first binding specificity in one arm, and a hybrid immunoglobulin heavy chain-light chain pair (providing a second binding specificity) in the other arm. It was found that this asymmetric structure facilitates the separation of the desired bispecific compound from unwanted immunoglobulin chain combinations, as the presence of an immunoglobulin light chain in only one half of the bispecific molecule provides for a facile way of separation. This approach is disclosed in WO 94/04690. For further details of generating bispecific antibodies see, for example, Suresh et al., <i>Methods in Enzymology </i>121:210 (1986).</div>
</li> <li> <para-num num="[0157]"> </para-num> <div class="description-line" id="p-0160" num="0157">According to another approach described in U.S. Pat. No. 5,731,168, the interface between a pair of antibody molecules can be engineered to maximize the percentage of heterodimers that are recovered from recombinant cell culture. In some embodiments, the interface comprises at least a part of the C<sub>H</sub>3 domain of an antibody constant domain. In this method, one or more small amino acid side chains from the interface of the first antibody molecule are replaced with larger side chains (e.g. tyrosine or tryptophan). Compensatory “cavities” of identical or similar size to the large side chain(s) are created on the interface of the second antibody molecule by replacing large amino acid side chains with smaller ones (e.g. alanine or threonine). This provides a mechanism for increasing the yield of the heterodimer over other unwanted end-products such as homodimers.</div>
</li> <li> <para-num num="[0158]"> </para-num> <div class="description-line" id="p-0161" num="0158">Bispecific antibodies include cross-linked or “heteroconjugate” antibodies. For example, one of the antibodies in the heteroconjugate can be coupled to avidin, the other to biotin. Such antibodies have, for example, been proposed to target immune system cells to unwanted cells (U.S. Pat. No. 4,676,980), and for treatment of HIV infection (WO 91/00360, WO 92/200373, and EP 03089). Heteroconjugate antibodies may be made using any convenient cross-linking methods. Suitable cross-linking agents are well known in the art, and are disclosed in U.S. Pat. No. 4,676,980, along with a number of cross-linking techniques.</div>
</li> <li> <para-num num="[0159]"> </para-num> <div class="description-line" id="p-0162" num="0159">Techniques for generating bispecific antibodies from antibody fragments have also been described in the literature. For example, bispecific antibodies can be prepared using chemical linkage. Brennan et al., <i>Science </i>229: 81 (1985) describe a procedure wherein intact antibodies are proteolytically cleaved to generate F(ab′)2 fragments. These fragments are reduced in the presence of the dithiol complexing agent sodium arsenite to stabilize vicinal dithiols and prevent intermolecular disulfide formation. The Fab′ fragments generated are then converted to thionitrobenzoate (TNB) derivatives. One of the Fab′-TNB derivatives is then reconverted to the Fab′-thiol by reduction with mercaptoethylamine and is mixed with an equimolar amount of the other Fab′-TNB derivative to form the bispecific antibody. The bispecific antibodies produced can be used as agents for the selective immobilization of enzymes.</div>
</li> <li> <para-num num="[0160]"> </para-num> <div class="description-line" id="p-0163" num="0160">Various techniques for making and isolating bispecific antibody fragments directly from recombinant cell culture have also been described. For example, bispecific antibodies have been produced using leucine zippers. Kostelny et al., <i>J. Immunol. </i>148(5): 1547-1553 (1992). The leucine zipper peptides from the Fos and Jun proteins were linked to the Fab′ portions of two different antibodies by gene fusion. The antibody homodimers were reduced at the hinge region to form monomers and then re-oxidized to form the antibody heterodimers. This method can also be utilized for the production of antibody homodimers. The “diabody” technology described by Hollinger et al., <i>Proc. Natl. Acad. Sci. USA </i>90:6444-6448 (1993) has provided an alternative mechanism for making bispecific antibody fragments. The fragments comprise a heavy chain variable domain (V<sub>H</sub>) connected to a light chain variable domain (V<sub>L</sub>) by a linker that is too short to allow pairing between the two domains on the same chain. Accordingly, the V<sub>H </sub>and V<sub>L </sub>domains of one fragment are forced to pair with the complementary V<sub>L </sub>and V<sub>H </sub>domains of another fragment, thereby forming two antigen-binding sites. Another strategy for making bispecific antibody fragments by the use of single-chain Fv (sFv) dimers has also been reported. See Gruber et al., <i>J. Immunol. </i>152:5368 (1994).</div>
</li> <li> <para-num num="[0161]"> </para-num> <div class="description-line" id="p-0164" num="0161">Antibodies with more than two valencies are contemplated. For example, trispecific antibodies can be prepared. Tutt et al., <i>J. Immunol. </i>147: 60 (1991).</div>
</li> <li> <div class="description-line" id="p-0165" num="0000">(viii) Multivalent Antibodies</div>
</li> <li> <para-num num="[0162]"> </para-num> <div class="description-line" id="p-0166" num="0162">In some embodiments, the antibodies are multivalent antibodies. A multivalent antibody may be internalized (and/or catabolized) faster than a bivalent antibody by a cell expressing an antigen to which the antibodies bind. The antibodies provided herein can be multivalent antibodies (which are other than of the IgM class) with three or more antigen binding sites (e.g., tetravalent antibodies), which can be readily produced by recombinant expression of nucleic acid encoding the polypeptide chains of the antibody. The multivalent antibody can comprise a dimerization domain and three or more antigen binding sites. The preferred dimerization domain comprises (or consists of) an Fc region or a hinge region. In this scenario, the antibody will comprise an Fc region and three or more antigen binding sites amino-terminal to the Fc region. The preferred multivalent antibody herein comprises (or consists of) three to about eight, but preferably four, antigen binding sites. The multivalent antibody comprises at least one polypeptide chain (and preferably two polypeptide chains), wherein the polypeptide chain(s) comprise two or more variable domains. For instance, the polypeptide chain(s) may comprise VD1-(X1)n-VD2-(X2) n-Fc, wherein VD1 is a first variable domain, VD2 is a second variable domain, Fc is one polypeptide chain of an Fc region, X1 and X2 represent an amino acid or polypeptide, and n is 0 or 1. For instance, the polypeptide chain(s) may comprise: VH-CH1-flexible linker-VH-CH1-Fc region chain; or VH-CH1-VH-CH1-Fc region chain. The multivalent antibody herein preferably further comprises at least two (and preferably four) light chain variable domain polypeptides. The multivalent antibody herein may, for instance, comprise from about two to about eight light chain variable domain polypeptides. The light chain variable domain polypeptides contemplated here comprise a light chain variable domain and, optionally, further comprise a CL domain.</div>
</li> <heading id="h-0015">(ix) Other Antibody Modifications</heading>
<li> <para-num num="[0163]"> </para-num> <div class="description-line" id="p-0167" num="0163">It may be desirable to modify the antibody provided herein with respect to effector function, e.g., so as to enhance antigen-dependent cell-mediated cyotoxicity (ADCC) and/or complement dependent cytotoxicity (CDC) of the antibody. This may be achieved by introducing one or more amino acid substitutions in an Fc region of the antibody. Alternatively or additionally, cysteine residue(s) may be introduced in the Fc region, thereby allowing interchain disulfide bond formation in this region. The homodimeric antibody thus generated may have improved internalization capability and/or increased complement-mediated cell killing and antibody-dependent cellular cytotoxicity (ADCC). See Caron et al., <i>J. Exp Med. </i>176:1191-1195 (1992) and Shopes, B. J., <i>Immunol. </i>148:2918-2922 (1992). Homodimeric antibodies with enhanced anti-tumor activity may also be prepared using heterobifunctional cross-linkers as described in Wolff et al., <i>Cancer Research </i>53:2560-2565 (1993). Alternatively, an antibody can be engineered which has dual Fc regions and may thereby have enhanced complement mediated lysis and ADCC capabilities. See Stevenson et al., <i>Anti</i>-<i>Cancer Drug Design </i>3:219-230 (1989).</div>
</li> <li> <para-num num="[0164]"> </para-num> <div class="description-line" id="p-0168" num="0164">For increasing serum half the serum half life of the antibody, amino acid alterations can be made in the antibody as described in US 2006/0067930, which is hereby incorporated by reference in its entirety.</div>
</li> <heading id="h-0016">(B) Polypeptide Variants and Modifications</heading>
<li> <para-num num="[0165]"> </para-num> <div class="description-line" id="p-0169" num="0165">Amino acid sequence modification(s) of the polypeptides, including antibodies, described herein may be used in the methods of purifying polypeptides (e.g., antibodies) described herein.</div>
</li> <heading id="h-0017">(i) Variant Polypeptides</heading>
<li> <para-num num="[0166]"> </para-num> <div class="description-line" id="p-0170" num="0166">“Polypeptide variant” means a polypeptide, preferably an active polypeptide, as defined herein having at least about 80% amino acid sequence identity with a full-length native sequence of the polypeptide, a polypeptide sequence lacking the signal peptide, an extracellular domain of a polypeptide, with or without the signal peptide. Such polypeptide variants include, for instance, polypeptides wherein one or more amino acid residues are added, or deleted, at the N or C-terminus of the full-length native amino acid sequence. Ordinarily, a TAT polypeptide variant will have at least about 80% amino acid sequence identity, alternatively at least about any of 85%, 90%, 95%, 96%, 97%, 98%, or 99% amino acid sequence identity, to a full-length native sequence polypeptide sequence, a polypeptide sequence lacking the signal peptide, an extracellular domain of a polypeptide, with or without the signal peptide. Optionally, variant polypeptides will have no more than one conservative amino acid substitution as compared to the native polypeptide sequence, alternatively no more than about any of 2, 3, 4, 5, 6, 7, 8, 9, or 10 conservative amino acid substitution as compared to the native polypeptide sequence.</div>
</li> <li> <para-num num="[0167]"> </para-num> <div class="description-line" id="p-0171" num="0167">The variant polypeptide may be truncated at the N-terminus or C-terminus, or may lack internal residues, for example, when compared with a full length native polypeptide. Certain variant polypeptides may lack amino acid residues that are not essential for a desired biological activity. These variant polypeptides with truncations, deletions, and insertions may be prepared by any of a number of conventional techniques. Desired variant polypeptides may be chemically synthesized. Another suitable technique involves isolating and amplifying a nucleic acid fragment encoding a desired variant polypeptide, by polymerase chain reaction (PCR). Oligonucleotides that define the desired termini of the nucleic acid fragment are employed at the 5′ and 3′ primers in the PCR. Preferably, variant polypeptides share at least one biological and/or immunological activity with the native polypeptide disclosed herein.</div>
</li> <li> <para-num num="[0168]"> </para-num> <div class="description-line" id="p-0172" num="0168">Amino acid sequence insertions include amino- and/or carboxyl-terminal fusions ranging in length from one residue to polypeptides containing a hundred or more residues, as well as intrasequence insertions of single or multiple amino acid residues. Examples of terminal insertions include an antibody with an N-terminal methionyl residue or the antibody fused to a cytotoxic polypeptide. Other insertional variants of the antibody molecule include the fusion to the N- or C-terminus of the antibody to an enzyme or a polypeptide which increases the serum half-life of the antibody.</div>
</li> <li> <para-num num="[0169]"> </para-num> <div class="description-line" id="p-0173" num="0169">For example, it may be desirable to improve the binding affinity and/or other biological properties of the polypeptide Amino acid sequence variants of the polypeptide are prepared by introducing appropriate nucleotide changes into the antibody nucleic acid, or by peptide synthesis. Such modifications include, for example, deletions from, and/or insertions into and/or substitutions of, residues within the amino acid sequences of the polypeptide. Any combination of deletion, insertion, and substitution is made to arrive at the final construct, provided that the final construct possesses the desired characteristics. The amino acid changes also may alter post-translational processes of the polypeptide (e.g., antibody), such as changing the number or position of glycosylation sites.</div>
</li> <li> <para-num num="[0170]"> </para-num> <div class="description-line" id="p-0174" num="0170">Guidance in determining which amino acid residue may be inserted, substituted or deleted without adversely affecting the desired activity may be found by comparing the sequence of the polypeptide with that of homologous known polypeptide molecules and minimizing the number of amino acid sequence changes made in regions of high homology.</div>
</li> <li> <para-num num="[0171]"> </para-num> <div class="description-line" id="p-0175" num="0171">A useful method for identification of certain residues or regions of the polypeptide (e.g., antibody) that are preferred locations for mutagenesis is called “alanine scanning mutagenesis” as described by Cunningham and Wells, Science 244:1081-1085 (1989). Here, a residue or group of target residues are identified (e.g., charged residues such as Arg, Asp, His, Lys, and Glu) and replaced by a neutral or negatively charged amino acid (most preferably Alanine or Polyalanine) to affect the interaction of the amino acids with antigen. Those amino acid locations demonstrating functional sensitivity to the substitutions then are refined by introducing further or other variants at, or for, the sites of substitution. Thus, while the site for introducing an amino acid sequence variation is predetermined, the nature of the mutation per se need not be predetermined. For example, to analyze the performance of a mutation at a given site, ala scanning or random mutagenesis is conducted at the target codon or region and the expressed antibody variants are screened for the desired activity.</div>
</li> <li> <para-num num="[0172]"> </para-num> <div class="description-line" id="p-0176" num="0172">Another type of variant is an amino acid substitution variant. These variants have at least one amino acid residue in the antibody molecule replaced by a different residue. The sites of greatest interest for substitutional mutagenesis include the hypervariable regions, but FR alterations are also contemplated. Conservative substitutions are shown in the Table 1 below under the heading of “preferred substitutions.” If such substitutions result in a change in biological activity, then more substantial changes, denominated “exemplary substitutions” in the Table 1, or as further described below in reference to amino acid classes, may be introduced and the products screened.</div>
</li> <li> <div class="description-line" id="p-0177" num="0000">
<tables id="TABLE-US-00001" num="00001">
<patent-tables colsep="0" frame="none" rowsep="0">
<table align="left" class="description-table" cols="3" colsep="0" rowsep="0" width="100%">
<thead>
<tr class="description-tr">
<td class="description-td" colspan="3" nameend="3" namest="1" rowsep="1">TABLE 1</td>
</tr>
<tr class="description-tr">
<td align="center" class="description-td" colspan="3" nameend="3" namest="1" rowsep="1"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">Original</td>
<td class="description-td">Exemplary</td>
<td class="description-td">Preferred</td>
</tr>
<tr class="description-tr">
<td class="description-td">Residue</td>
<td class="description-td">Substitutions</td>
<td class="description-td">Substitutions</td>
</tr>
<tr class="description-tr">
<td align="center" class="description-td" colspan="3" nameend="3" namest="1" rowsep="1"> </td>
</tr>
</thead>
<tbody><tr class="description-tr">
<td class="description-td">Ala (A)</td>
<td class="description-td">Val; Leu; Ile</td>
<td class="description-td">Val</td>
</tr>
<tr class="description-tr">
<td class="description-td">Arg (R)</td>
<td class="description-td">Lys; Gln; Asn</td>
<td class="description-td">Lys</td>
</tr>
<tr class="description-tr">
<td class="description-td">Asn (N)</td>
<td class="description-td">Gln; His; Asp, Lys; Arg</td>
<td class="description-td">Gln</td>
</tr>
<tr class="description-tr">
<td class="description-td">Asp (D)</td>
<td class="description-td">Glu; Asn</td>
<td class="description-td">Glu</td>
</tr>
<tr class="description-tr">
<td class="description-td">Cys (C)</td>
<td class="description-td">Ser; Ala</td>
<td class="description-td">Ser</td>
</tr>
<tr class="description-tr">
<td class="description-td">Gln (Q)</td>
<td class="description-td">Asn; Glu</td>
<td class="description-td">Asn</td>
</tr>
<tr class="description-tr">
<td class="description-td">Glu (E)</td>
<td class="description-td">Asp; Gln</td>
<td class="description-td">Asp</td>
</tr>
<tr class="description-tr">
<td class="description-td">Gly (G)</td>
<td class="description-td">Ala</td>
<td class="description-td">Ala</td>
</tr>
<tr class="description-tr">
<td class="description-td">His (H)</td>
<td class="description-td">Asn; Gln; Lys; Arg</td>
<td class="description-td">Arg</td>
</tr>
<tr class="description-tr">
<td class="description-td">Ile (I)</td>
<td class="description-td">Leu; Val; Met; Ala; Phe; Norleucine</td>
<td class="description-td">Leu</td>
</tr>
<tr class="description-tr">
<td class="description-td">Leu (L)</td>
<td class="description-td">Norleucine; Ile; Val; Met; Ala; Phe</td>
<td class="description-td">Ile</td>
</tr>
<tr class="description-tr">
<td class="description-td">Lys (K)</td>
<td class="description-td">Arg; Gln; Asn</td>
<td class="description-td">Arg</td>
</tr>
<tr class="description-tr">
<td class="description-td">Met (M)</td>
<td class="description-td">Leu; Phe; Ile</td>
<td class="description-td">Leu</td>
</tr>
<tr class="description-tr">
<td class="description-td">Phe (F)</td>
<td class="description-td">Trp; Leu; Val; Ile; Ala; Tyr</td>
<td class="description-td">Tyr</td>
</tr>
<tr class="description-tr">
<td class="description-td">Pro (P)</td>
<td class="description-td">Ala</td>
<td class="description-td">Ala</td>
</tr>
<tr class="description-tr">
<td class="description-td">Ser (S)</td>
<td class="description-td">Thr</td>
<td class="description-td">Thr</td>
</tr>
<tr class="description-tr">
<td class="description-td">Thr (T)</td>
<td class="description-td">Val; Ser</td>
<td class="description-td">Ser</td>
</tr>
<tr class="description-tr">
<td class="description-td">Trp (W)</td>
<td class="description-td">Tyr; Phe</td>
<td class="description-td">Tyr</td>
</tr>
<tr class="description-tr">
<td class="description-td">Tyr (Y)</td>
<td class="description-td">Trp; Phe; Thr; Ser</td>
<td class="description-td">Phe</td>
</tr>
<tr class="description-tr">
<td class="description-td">Val (V)</td>
<td class="description-td">Ile; Leu; Met; Phe; Ala; Norleucine</td>
<td class="description-td">Leu</td>
</tr>
<tr class="description-tr">
<td align="center" class="description-td" colspan="3" nameend="3" namest="1" rowsep="1"> </td>
</tr>
</tbody></table>
</patent-tables>
</tables>
</div>
</li> <li> <para-num num="[0173]"> </para-num> <div class="description-line" id="p-0178" num="0173">Substantial modifications in the biological properties of the polypeptide are accomplished by selecting substitutions that differ significantly in their effect on maintaining (a) the structure of the polypeptide backbone in the area of the substitution, for example, as a sheet or helical conformation, (b) the charge or hydrophobicity of the molecule at the target site, or (c) the bulk of the side chain Amino acids may be grouped according to similarities in the properties of their side chains (in A. L. Lehninger, Biochemistry second ed., pp. 73-75, Worth Publishers, New York (1975)):</div>
</li> <li> <para-num num="[0174]"> </para-num> <div class="description-line" id="p-0179" num="0174">(1) non-polar: Ala (A), Val (V), Leu (L), Ile (I), Pro (P), Phe (F), Trp (W), Met (M)</div>
</li> <li> <para-num num="[0175]"> </para-num> <div class="description-line" id="p-0180" num="0175">(2) uncharged polar: Gly (G), Ser (S), Thr (T), Cys (C), Tyr (Y), Asn (N), Gln (Q)</div>
</li> <li> <para-num num="[0176]"> </para-num> <div class="description-line" id="p-0181" num="0176">(3) acidic: Asp (D), Glu (E)</div>
</li> <li> <para-num num="[0177]"> </para-num> <div class="description-line" id="p-0182" num="0177">(4) basic: Lys (K), Arg (R), His(H)</div>
</li> <li> <para-num num="[0178]"> </para-num> <div class="description-line" id="p-0183" num="0178">Alternatively, naturally occurring residues may be divided into groups based on common side-chain properties:</div>
</li> <li> <para-num num="[0179]"> </para-num> <div class="description-line" id="p-0184" num="0179">(1) hydrophobic: Norleucine, Met, Ala, Val, Leu, Ile;</div>
</li> <li> <para-num num="[0180]"> </para-num> <div class="description-line" id="p-0185" num="0180">(2) neutral hydrophilic: Cys, Ser, Thr, Asn, Gln;</div>
</li> <li> <para-num num="[0181]"> </para-num> <div class="description-line" id="p-0186" num="0181">(3) acidic: Asp, Glu;</div>
</li> <li> <para-num num="[0182]"> </para-num> <div class="description-line" id="p-0187" num="0182">(4) basic: His, Lys, Arg;</div>
</li> <li> <para-num num="[0183]"> </para-num> <div class="description-line" id="p-0188" num="0183">(5) residues that influence chain orientation: Gly, Pro;</div>
</li> <li> <para-num num="[0184]"> </para-num> <div class="description-line" id="p-0189" num="0184">(6) aromatic: Trp, Tyr, Phe.</div>
</li> <li> <para-num num="[0185]"> </para-num> <div class="description-line" id="p-0190" num="0185">Non-conservative substitutions will entail exchanging a member of one of these classes for another class.</div>
</li> <li> <para-num num="[0186]"> </para-num> <div class="description-line" id="p-0191" num="0186">Any cysteine residue not involved in maintaining the proper conformation of the antibody also may be substituted, generally with serine, to improve the oxidative stability of the molecule and prevent aberrant crosslinking. Conversely, cysteine bond(s) may be added to the polypeptide to improve its stability (particularly where the antibody is an antibody fragment such as an Fv fragment).</div>
</li> <li> <para-num num="[0187]"> </para-num> <div class="description-line" id="p-0192" num="0187">A particularly preferred type of substitutional variant involves substituting one or more hypervariable region residues of a parent antibody (e.g., a humanized antibody). Generally, the resulting variant(s) selected for further development will have improved biological properties relative to the parent antibody from which they are generated. A convenient way for generating such substitutional variants involves affinity maturation using phage display. Briefly, several hypervariable region sites (e.g., 6-7 sites) are mutated to generate all possible amino substitutions at each site. The antibody variants thus generated are displayed in a monovalent fashion from filamentous phage particles as fusions to the gene III product of M13 packaged within each particle. The phage-displayed variants are then screened for their biological activity (e.g., binding affinity) as herein disclosed. In order to identify candidate hypervariable region sites for modification, alanine scanning mutagenesis can be performed to identify hypervariable region residues contributing significantly to antigen binding. Alternatively, or additionally, it may be beneficial to analyze a crystal structure of the antigen-antibody complex to identify contact points between the antibody and target. Such contact residues and neighboring residues are candidates for substitution according to the techniques elaborated herein. Once such variants are generated, the panel of variants is subjected to screening as described herein and antibodies with superior properties in one or more relevant assays may be selected for further development.</div>
</li> <li> <para-num num="[0188]"> </para-num> <div class="description-line" id="p-0193" num="0188">Another type of amino acid variant of the polyeptide alters the original glycosylation pattern of the antibody. The polypeptide may comprise non-amino acid moieties. For example, the polypeptide may be glycosylated. Such glycosylation may occur naturally during expression of the polypeptide in the host cell or host organism, or may be a deliberate modification arising from human intervention. By altering is meant deleting one or more carbohydrate moieties found in the polypeptide, and/or adding one or more glycosylation sites that are not present in the polypeptide.</div>
</li> <li> <para-num num="[0189]"> </para-num> <div class="description-line" id="p-0194" num="0189">Glycosylation of polypeptide is typically either N-linked or O-linked. N-linked refers to the attachment of the carbohydrate moiety to the side chain of an asparagine residue. The tripeptide sequences asparagine-X-serine and asparagine-X-threonine, where X is any amino acid except proline, are the recognition sequences for enzymatic attachment of the carbohydrate moiety to the asparagine side chain Thus, the presence of either of these tripeptide sequences in a polypeptide creates a potential glycosylation site. O-linked glycosylation refers to the attachment of one of the sugars N-aceylgalactosamine, galactose, or xylose to a hydroxyamino acid, most commonly serine or threonine, although 5-hydroxyproline or 5-hydroxylysine may also be used.</div>
</li> <li> <para-num num="[0190]"> </para-num> <div class="description-line" id="p-0195" num="0190">Addition of glycosylation sites to the polypeptide is conveniently accomplished by altering the amino acid sequence such that it contains one or more of the above-described tripeptide sequences (for N-linked glycosylation sites). The alteration may also be made by the addition of, or substitution by, one or more serine or threonine residues to the sequence of the original antibody (for O-linked glycosylation sites).</div>
</li> <li> <para-num num="[0191]"> </para-num> <div class="description-line" id="p-0196" num="0191">Removal of carbohydrate moieties present on the polypeptide may be accomplished chemically or enzymatically or by mutational substitution of codons encoding for amino acid residues that serve as targets for glycosylation. Enzymatic cleavage of carbohydrate moieties on polypeptides can be achieved by the use of a variety of endo- and exo-glycosidases.</div>
</li> <li> <para-num num="[0192]"> </para-num> <div class="description-line" id="p-0197" num="0192">Other modifications include deamidation of glutaminyl and asparaginyl residues to the corresponding glutamyl and aspartyl residues, respectively, hydroxylation of proline and lysine, phosphorylation of hydroxyl groups of seryl or threonyl residues, methylation of the a-amino groups of lysine, arginine, and histidine side chains, acetylation of the N-terminal amine, and amidation of any C-terminal carboxyl group.</div>
</li> <heading id="h-0018">(ii) Chimeric Polypeptides</heading>
<li> <para-num num="[0193]"> </para-num> <div class="description-line" id="p-0198" num="0193">The polypeptide described herein may be modified in a way to form chimeric molecules comprising the polypeptide fused to another, heterologous polypeptide or amino acid sequence. In some embodiments, a chimeric molecule comprises a fusion of the polypeptide with a tag polypeptide which provides an epitope to which an anti-tag antibody can selectively bind. The epitope tag is generally placed at the amino- or carboxyl-terminus of the polypeptide. The presence of such epitope-tagged forms of the polypeptide can be detected using an antibody against the tag polypeptide. Also, provision of the epitope tag enables the polypeptide to be readily purified by affinity purification using an anti-tag antibody or another type of affinity matrix that binds to the epitope tag.</div>
</li> <li> <para-num num="[0194]"> </para-num> <div class="description-line" id="p-0199" num="0194">In an alternative embodiment, the chimeric molecule may comprise a fusion of the polypeptide with an immunoglobulin or a particular region of an immunoglobulin. A bivalent form of the chimeric molecule is referred to as an “immunoadhesin.”</div>
</li> <li> <para-num num="[0195]"> </para-num> <div class="description-line" id="p-0200" num="0195">As used herein, the term “immunoadhesin” designates antibody-like molecules which combine the binding specificity of a heterologous polypeptide with the effector functions of immunoglobulin constant domains. Structurally, the immunoadhesins comprise a fusion of an amino acid sequence with the desired binding specificity which is other than the antigen recognition and binding site of an antibody (i.e., is “heterologous”), and an immunoglobulin constant domain sequence. The adhesin part of an immunoadhesin molecule typically is a contiguous amino acid sequence comprising at least the binding site of a receptor or a ligand. The immunoglobulin constant domain sequence in the immunoadhesin may be obtained from any immunoglobulin, such as IgG-1, IgG-2, IgG-3, or IgG-4 subtypes, IgA (including IgA-1 and IgA-2), IgE, IgD or IgM.</div>
</li> <li> <para-num num="[0196]"> </para-num> <div class="description-line" id="p-0201" num="0196">The Ig fusions preferably include the substitution of a soluble (transmembrane domain deleted or inactivated) form of a polypeptide in place of at least one variable region within an Ig molecule. In a particularly preferred embodiment, the immunoglobulin fusion includes the hinge, CH<sub>2 </sub>and CH<sub>3</sub>, or the hinge, CH<sub>1</sub>, CH<sub>2 </sub>and CH<sub>3 </sub>regions of an IgG1 molecule.</div>
</li> <li> <div class="description-line" id="p-0202" num="0000">(iii) Polypeptide Conjugates</div>
</li> <li> <para-num num="[0197]"> </para-num> <div class="description-line" id="p-0203" num="0197">The polypeptide for use in polypeptide formulations may be conjugated to a cytotoxic agent such as a chemotherapeutic agent, a growth inhibitory agent, a toxin (e.g., an enzymatically active toxin of bacterial, fungal, plant, or animal origin, or fragments thereof), or a radioactive isotope (i.e., a radioconjugate).</div>
</li> <li> <para-num num="[0198]"> </para-num> <div class="description-line" id="p-0204" num="0198">Chemotherapeutic agents useful in the generation of such conjugates can be used. In addition, enzymatically active toxins and fragments thereof that can be used include diphtheria A chain, nonbinding active fragments of diphtheria toxin, exotoxin A chain (from <i>Pseudomonas aeruginosa</i>), ricin A chain, abrin A chain, modeccin A chain, alpha-sarcin, <i>Aleurites fordii </i>proteins, dianthin proteins, <i>Phytolaca americana </i>proteins (PAPI, PAPII, and PAP-S), <i>Momordica charantia </i>inhibitor, curcin, crotin, <i>Sapaonaria officinalis </i>inhibitor, gelonin, mitogellin, restrictocin, phenomycin, enomycin, and the tricothecenes. A variety of radionuclides are available for the production of radioconjugated polypeptides. Examples include <sup>212</sup>Bi, <sup>131</sup>I, <sup>131</sup>In, <sup>90</sup>Y, and <sup>186</sup>Re. Conjugates of the polypeptide and cytotoxic agent are made using a variety of bifunctional protein-coupling agents such as N-succinimidyl-3-(2-pyridyldithiol) propionate (SPDP), iminothiolane (IT), bifunctional derivatives of imidoesters (such as dimethyl adipimidate HCL), active esters (such as disuccinimidyl suberate), aldehydes (such as glutareldehyde), bis-azido compounds (such as bis(p-azidobenzoyl) hexanediamine), bis-diazonium derivatives (such as bis-(p-diazoniumbenzoyl)-ethylenediamine), diisocyanates (such as tolyene 2,6-diisocyanate), and bis-active fluorine compounds (such as 1,5-difluoro-2,4-dinitrobenzene). For example, a ricin immunotoxin can be prepared as described in Vitetta et al., <i>Science </i>238: 1098 (1987). Carbon-14-labeled 1-isothiocyanatobenzyl-3-methyldiethylene triaminepentaacetic acid (MX-DTPA) is an exemplary chelating agent for conjugation of radionucleotide to the polypeptide.</div>
</li> <li> <para-num num="[0199]"> </para-num> <div class="description-line" id="p-0205" num="0199">Conjugates of a polypeptide and one or more small molecule toxins, such as a calicheamicin, maytansinoids, a trichothene, and CC1065, and the derivatives of these toxins that have toxin activity, are also contemplated herein.</div>
</li> <li> <para-num num="[0200]"> </para-num> <div class="description-line" id="p-0206" num="0200">Maytansinoids are mitototic inhibitors which act by inhibiting tubulin polymerization. Maytansine was first isolated from the east African shrub Maytenus serrata. Subsequently, it was discovered that certain microbes also produce maytansinoids, such as maytansinol and C-3 maytansinol esters. Synthetic maytansinol and derivatives and analogues thereof are also contemplated. There are many linking groups known in the art for making polypeptide-maytansinoid conjugates, including, for example, those disclosed in U.S. Pat. No. 5,208,020. The linking groups include disufide groups, thioether groups, acid labile groups, photolabile groups, peptidase labile groups, or esterase labile groups, as disclosed in the above-identified patents, disulfide and thioether groups being preferred.</div>
</li> <li> <para-num num="[0201]"> </para-num> <div class="description-line" id="p-0207" num="0201">The linker may be attached to the maytansinoid molecule at various positions, depending on the type of the link. For example, an ester linkage may be formed by reaction with a hydroxyl group using conventional coupling techniques. The reaction may occur at the C-3 position having a hydroxyl group, the C-14 position modified with hyrdoxymethyl, the C-15 position modified with a hydroxyl group, and the C-20 position having a hydroxyl group. In a preferred embodiment, the linkage is formed at the C-3 position of maytansinol or a maytansinol analogue.</div>
</li> <li> <para-num num="[0202]"> </para-num> <div class="description-line" id="p-0208" num="0202">Another conjugate of interest comprises a polypeptide conjugated to one or more calicheamicin molecules. The calicheamicin family of antibiotics are capable of producing double-stranded DNA breaks at sub-picomolar concentrations. For the preparation of conjugates of the calicheamicin family, see, e.g., U.S. Pat. No. 5,712,374. Structural analogues of calicheamicin which may be used include, but are not limited to, γ<sub>1</sub> <sup>1</sup>, α<sub>2</sub> <sup>1</sup>, α<sub>3</sub> <sup>1</sup>, N-acetyl-γ<sub>1</sub> <sup>1</sup>, PSAG and θ<sub>1</sub> <sup>1</sup>. Another anti-tumor drug that the antibody can be conjugated is QFA which is an antifolate. Both calicheamicin and QFA have intracellular sites of action and do not readily cross the plasma membrane. Therefore, cellular uptake of these agents through polypeptide (e.g., antibody) mediated internalization greatly enhances their cytotoxic effects.</div>
</li> <li> <para-num num="[0203]"> </para-num> <div class="description-line" id="p-0209" num="0203">Other antitumor agents that can be conjugated to the polypeptides described herein include BCNU, streptozoicin, vincristine and 5-fluorouracil, the family of agents known collectively LL-E33288 complex, as well as esperamicins.</div>
</li> <li> <para-num num="[0204]"> </para-num> <div class="description-line" id="p-0210" num="0204">In some embodiments, the polypeptide may be a conjugate between a polypeptide and a compound with nucleolytic activity (e.g., a ribonuclease or a DNA endonuclease such as a deoxyribonuclease; DNase).</div>
</li> <li> <para-num num="[0205]"> </para-num> <div class="description-line" id="p-0211" num="0205">In yet another embodiment, the polypeptide (e.g., antibody) may be conjugated to a “receptor” (such streptavidin) for utilization in tumor pre-targeting wherein the polypeptide receptor conjugate is administered to the patient, followed by removal of unbound conjugate from the circulation using a clearing agent and then administration of a “ligand” (e.g., avidin) which is conjugated to a cytotoxic agent (e.g., a radionucleotide).</div>
</li> <li> <para-num num="[0206]"> </para-num> <div class="description-line" id="p-0212" num="0206">In some embodiments, the polypeptide may be conjugated to a prodrug-activating enzyme which converts a prodrug (e.g., a peptidyl chemotherapeutic agent) to an active anti-cancer drug. The enzyme component of the immunoconjugate includes any enzyme capable of acting on a prodrug in such a way so as to covert it into its more active, cytotoxic form.</div>
</li> <li> <para-num num="[0207]"> </para-num> <div class="description-line" id="p-0213" num="0207">Enzymes that are useful include, but are not limited to, alkaline phosphatase useful for converting phosphate-containing prodrugs into free drugs; arylsulfatase useful for converting sulfate-containing prodrugs into free drugs; cytosine deaminase useful for converting non-toxic 5-fluorocytosine into the anti-cancer drug, 5-fluorouracil; proteases, such as serratia protease, thermolysin, subtilisin, carboxypeptidases and cathepsins (such as cathepsins B and L), that are useful for converting peptide-containing prodrugs into free drugs; D-alanylcarboxypeptidases, useful for converting prodrugs that contain D-amino acid substituents; carbohydrate-cleaving enzymes such as β-galactosidase and neuraminidase useful for converting glycosylated prodrugs into free drugs; β-lactamase useful for converting drugs derivatized with β-lactams into free drugs; and penicillin amidases, such as penicillin V amidase or penicillin G amidase, useful for converting drugs derivatized at their amine nitrogens with phenoxyacetyl or phenylacetyl groups, respectively, into free drugs. Alternatively, antibodies with enzymatic activity, also known in the art as “abzymes”, can be used to convert the prodrugs into free active drugs.</div>
</li> <heading id="h-0019">(iv) Other</heading>
<li> <para-num num="[0208]"> </para-num> <div class="description-line" id="p-0214" num="0208">Another type of covalent modification of the polypeptide comprises linking the polypeptide to one of a variety of nonproteinaceous polymers, e.g., polyethylene glycol, polypropylene glycol, polyoxyalkylenes, or copolymers of polyethylene glycol and polypropylene glycol. The polypeptide also may be entrapped in microcapsules prepared, for example, by coacervation techniques or by interfacial polymerization (for example, hydroxymethylcellulose or gelatin-microcapsules and poly-(methylmethacylate) microcapsules, respectively), in colloidal drug delivery systems (for example, liposomes, albumin microspheres, microemulsions, nano-particles and nanocapsules), or in macroemulsions. Such techniques are disclosed in Remington's Pharmaceutical Sciences, 18th edition, Gennaro, A. R., Ed., (1990).</div>
</li> <heading id="h-0020">IV. Obtaining Polypeptides for Use in the Formulations and Methods</heading>
<li> <para-num num="[0209]"> </para-num> <div class="description-line" id="p-0215" num="0209">The polypeptides used in the methods of purification described herein may be obtained using methods well-known in the art, including the recombination methods. The following sections provide guidance regarding these methods.</div>
</li> <heading id="h-0021">(A) Polynucleotides</heading>
<li> <para-num num="[0210]"> </para-num> <div class="description-line" id="p-0216" num="0210">“Polynucleotide,” or “nucleic acid,” as used interchangeably herein, refer to polymers of nucleotides of any length, and include DNA and RNA.</div>
</li> <li> <para-num num="[0211]"> </para-num> <div class="description-line" id="p-0217" num="0211">Polynucleotides encoding polypeptides may be obtained from any source including, but not limited to, a cDNA library prepared from tissue believed to possess the polypeptide mRNA and to express it at a detectable level. Accordingly, polynucleotides encoding polypeptide can be conveniently obtained from a cDNA library prepared from human tissue. The polypeptide-encoding gene may also be obtained from a genomic library or by known synthetic procedures (e.g., automated nucleic acid synthesis).</div>
</li> <li> <para-num num="[0212]"> </para-num> <div class="description-line" id="p-0218" num="0212">For example, the polynucleotide may encode an entire immunoglobulin molecule chain, such as a light chain or a heavy chain. A complete heavy chain includes not only a heavy chain variable region (V<sub>H</sub>) but also a heavy chain constant region (C<sub>H</sub>), which typically will comprise three constant domains: C<sub>H</sub>1, C<sub>H</sub>2 and C<sub>H</sub>3; and a “hinge” region. In some situations, the presence of a constant region is desirable.</div>
</li> <li> <para-num num="[0213]"> </para-num> <div class="description-line" id="p-0219" num="0213">Other polypeptides which may be encoded by the polynucleotide include antigen-binding antibody fragments such as single domain antibodies (“dAbs”), Fv, scFv, Fab′ and F(ab′)2 and “minibodies.” Minibodies are (typically) bivalent antibody fragments from which the C<sub>H</sub>1 and C<sub>K </sub>or C<sub>L </sub>domain has been excised. As minibodies are smaller than conventional antibodies they should achieve better tissue penetration in clinical/diagnostic use, but being bivalent they should retain higher binding affinity than monovalent antibody fragments, such as dAbs. Accordingly, unless the context dictates otherwise, the term “antibody” as used herein encompasses not only whole antibody molecules but also antigen-binding antibody fragments of the type discussed above. Preferably each framework region present in the encoded polypeptide will comprise at least one amino acid substitution relative to the corresponding human acceptor framework. Thus, for example, the framework regions may comprise, in total, three, four, five, six, seven, eight, nine, ten, eleven, twelve, thirteen, fourteen, or fifteen amino acid substitutions relative to the acceptor framework regions.</div>
</li> <li> <para-num num="[0214]"> </para-num> <div class="description-line" id="p-0220" num="0214">Suitably, the polynucleotides described herein may be isolated and/or purified. In some embodiments, the polynucleotides are isolated polynucleotides.</div>
</li> <li> <para-num num="[0215]"> </para-num> <div class="description-line" id="p-0221" num="0215">The term “isolated polynucleotide” is intended to indicate that the molecule is removed or separated from its normal or natural environment or has been produced in such a way that it is not present in its normal or natural environment. In some embodiments, the polynucleotides are purified polynucleotides. The term purified is intended to indicate that at least some contaminating molecules or substances have been removed.</div>
</li> <li> <para-num num="[0216]"> </para-num> <div class="description-line" id="p-0222" num="0216">Suitably, the polynucleotides are substantially purified, such that the relevant polynucleotides constitutes the dominant (i.e., most abundant) polynucleotides present in a composition.</div>
</li> <li> <para-num num="[0217]"> </para-num> <div class="description-line" id="p-0223" num="0217">Recombinant nucleic acids comprising an insert coding for a heavy chain variable domain and/or for a light chain variable domain may be used in the methods as described herein. By definition such nucleic acids comprise coding single stranded nucleic acids, double stranded nucleic acids consisting of said coding nucleic acids and of complementary nucleic acids thereto, or these complementary (single stranded) nucleic acids themselves.</div>
</li> <li> <para-num num="[0218]"> </para-num> <div class="description-line" id="p-0224" num="0218">Modification(s) may also be made outside the heavy chain variable domain and/or of the light chain variable domain of the antibody. Such a mutant nucleic acid may be a silent mutant wherein one or more nucleotides are replaced by other nucleotides with the new codons coding for the same amino acid(s). Such a mutant sequence may be a degenerate sequence. Degenerate sequences are degenerated within the meaning of the genetic code in that an unlimited number of nucleotides are replaced by other nucleotides without resulting in a change of the amino acid sequence originally encoded. Such degenerated sequences may be useful due to their different restriction sites and/or frequency of particular codons which are preferred by the specific host, particularly yeast, bacterial or mammalian cells, to obtain an optimal expression of the heavy chain variable domain and/or the light chain variable domain.</div>
</li> <li> <para-num num="[0219]"> </para-num> <div class="description-line" id="p-0225" num="0219">Sequences having a degree of sequence identity or sequence homology with amino acid sequence(s) of a polypeptide having the specific properties defined herein or of any nucleotide sequence encoding such a polypeptide (hereinafter referred to as a “homologous sequence(s)”). Here, the term “homologue” means an entity having a certain homology with the subject amino acid sequences and the subject nucleotide sequences. Here, the term “homology” can be equated with “identity.”</div>
</li> <li> <para-num num="[0220]"> </para-num> <div class="description-line" id="p-0226" num="0220">In some embodiments, homologous amino acid sequence and/or nucleotide sequence should encode a polypeptide which retains the functional activity and/or enhances the activity of the antibody. In some embodiments, homologous sequence is taken to include an amino acid sequence which may be at least about any of 75, 85, or 90% identical, preferably at least about 95 or 98% identical to the subject sequence. Typically, the homologues will comprise the same active sites etc. as the subject amino acid sequence. Although homology can also be considered in terms of similarity (i.e., amino acid residues having similar chemical properties/functions). In some embodiments, it is preferred to express homology in terms of sequence identity.</div>
</li> <li> <para-num num="[0221]"> </para-num> <div class="description-line" id="p-0227" num="0221">In the present context, a homologous sequence is taken to include a nucleotide sequence which may be at least about any of 75, 85, or 90% identical, preferably at least about 95 or 98% identical to a nucleotide sequence encoding a polypeptide described herein (the subject sequence). Typically, the homologues will comprise the same sequences that code for the active sites etc. as the subject sequence. Although homology can also be considered in terms of similarity (i.e., amino acid residues having similar chemical properties/functions). In some embodiments, it is preferred to express homology in terms of sequence identity.</div>
</li> <li> <para-num num="[0222]"> </para-num> <div class="description-line" id="p-0228" num="0222">These methods include, but are not limited to, isolation from a natural source (in the case of naturally occurring amino acid sequence variants) or preparation by oligonucleotide-mediated (or site-directed) mutagenesis, PCR mutagenesis, and cassette mutagenesis of an earlier prepared variant or a non-variant version of the polypeptide.</div>
</li> <heading id="h-0022">(B) Expression of Polynucleotides</heading>
<li> <para-num num="[0223]"> </para-num> <div class="description-line" id="p-0229" num="0223">The description below relates primarily to production of polypeptides by culturing cells transformed or transfected with a vector containing polypeptide-encoding polynucleotides. It is, of course, contemplated that alternative methods, which are well known in the art, may be employed to prepare polypeptides. For instance, the appropriate amino acid sequence, or portions thereof, may be produced by direct peptide synthesis using solid-phase techniques (see, e.g., Stewart et al., <i>Solid</i>-<i>Phase Peptide Synthesis </i>W.H. Freeman Co., San Francisco, Calif. (1969); Merrifield, <i>J. Am. Chem. Soc. </i>85:2149-2154 (1963)). In vitro protein synthesis may be performed using manual techniques or by automation. Automated synthesis may be accomplished, for instance, using an Applied Biosystems Peptide Synthesizer (Foster City, Calif.) using manufacturer's instructions. Various portions of the polypeptide may be chemically synthesized separately and combined using chemical or enzymatic methods to produce the desired polypeptide.</div>
</li> <li> <para-num num="[0224]"> </para-num> <div class="description-line" id="p-0230" num="0224">Polynucleotides as described herein are inserted into an expression vector(s) for production of the polypeptides. The term “control sequences” refers to DNA sequences necessary for the expression of an operably linked coding sequence in a particular host organism. The control sequences include, but are not limited to, promoters (e.g., naturally-associated or heterologous promoters), signal sequences, enhancer elements, and transcription termination sequences.</div>
</li> <li> <para-num num="[0225]"> </para-num> <div class="description-line" id="p-0231" num="0225">A polynucleotide is “operably linked” when it is placed into a functional relationship with another polynucleotide sequence. For example, nucleic acids for a presequence or secretory leader is operably linked to nucleic acids for a polypeptide if it is expressed as a preprotein that participates in the secretion of the polypeptide; a promoter or enhancer is operably linked to a coding sequence if it affects the transcription of the sequence; or a ribosome binding site is operably linked to a coding sequence if it is positioned so as to facilitate translation. Generally, “operably linked” means that the nucleic acid sequences being linked are contiguous, and, in the case of a secretory leader, contiguous and in reading phase. However, enhancers do not have to be contiguous. Linking is accomplished by ligation at convenient restriction sites. If such sites do not exist, the synthetic oligonucleotide adaptors or linkers are used in accordance with conventional practice.</div>
</li> <li> <para-num num="[0226]"> </para-num> <div class="description-line" id="p-0232" num="0226">For antibodies, the light and heavy chains can be cloned in the same or different expression vectors. The nucleic acid segments encoding immunoglobulin chains are operably linked to control sequences in the expression vector(s) that ensure the expression of immunoglobulin polypeptides.</div>
</li> <li> <para-num num="[0227]"> </para-num> <div class="description-line" id="p-0233" num="0227">Selection Gene Component—</div>
</li> <li> <para-num num="[0228]"> </para-num> <div class="description-line" id="p-0234" num="0228">Commonly, expression vectors contain selection markers (e.g., ampicillin-resistance, hygromycin-resistance, tetracycline resistance, kanamycin resistance or neomycin resistance) to permit detection of those cells transformed with the desired DNA sequences (see, e.g., U.S. Pat. No. 4,704,362). In some embodiments, selection genes encode proteins that (a) confer resistance to antibiotics or other toxins, e.g., ampicillin, neomycin, methotrexate, or tetracycline, (b) complement auxotrophic deficiencies, or (c) supply critical nutrients not available from complex media, e.g., the gene encoding D-alanine racemase for Bacilli.</div>
</li> <li> <para-num num="[0229]"> </para-num> <div class="description-line" id="p-0235" num="0229">One example of a selection scheme utilizes a drug to arrest growth of a host cell. Those cells that are successfully transformed with a heterologous gene produce a protein conferring drug resistance and thus survive the selection regimen. Examples of such dominant selection use the drugs neomycin, mycophenolic acid and hygromycin.</div>
</li> <li> <para-num num="[0230]"> </para-num> <div class="description-line" id="p-0236" num="0230">Another example of suitable selectable markers for mammalian cells are those that enable the identification of cells competent to take up the nucleic acid encoding antibodies described herein, such as DHFR, thymidine kinase, metallothionein-I and -III, preferably primate metallothionein genes, adenosine deaminase, ornithine decarboxylase, etc.</div>
</li> <li> <para-num num="[0231]"> </para-num> <div class="description-line" id="p-0237" num="0231">For example, cells transformed with the DHFR selection gene are first identified by culturing all of the transformants in a culture medium that contains methotrexate (Mtx), a competitive antagonist of DHFR. An appropriate host cell when wild-type DHFR is employed is the Chinese hamster ovary (CHO) cell line deficient in DHFR activity (e.g., ATCC CRL-9096).</div>
</li> <li> <para-num num="[0232]"> </para-num> <div class="description-line" id="p-0238" num="0232">Alternatively, host cells (particularly wild-type hosts that contain endogenous DHFR) transformed or co-transformed with DNA sequences encoding a polypeptide described herein, wild-type DHFR protein, and another selectable marker such as aminoglycoside 3′-phosphotransferase (APH) can be selected by cell growth in medium containing a selection agent for the selectable marker such as an aminoglycosidic antibiotic, e.g., kanamycin, neomycin, or G418. See U.S. Pat. No. 4,965,199.</div>
</li> <li> <para-num num="[0233]"> </para-num> <div class="description-line" id="p-0239" num="0233">A suitable selection gene for use in yeast is the trp1 gene present in the yeast plasmid YRp7 (Stinchcomb et al., <i>Nature </i>282:39 (1979)). The trp1 gene provides a selection marker for a mutant strain of yeast lacking the ability to grow in tryptophan, for example, ATCC No. 44076 or PEP4-1. Jones, <i>Genetics </i>85:12 (1977). The presence of the trp1 lesion in the yeast host cell genome then provides an effective environment for detecting transformation by growth in the absence of tryptophan. Similarly, Leu2-deficient yeast strains (ATCC 20,622 or 38,626) are complemented by known plasmids bearing the Leu2 gene.</div>
</li> <li> <para-num num="[0234]"> </para-num> <div class="description-line" id="p-0240" num="0234">In addition, vectors derived from the 1.6 μm circular plasmid pKD1 can be used for transformation of <i>Kluyveromyces </i>yeasts. Alternatively, an expression system for large-scale production of recombinant calf chymosin was reported for <i>K. lactis</i>. Van den Berg, Bio/Technology 8:135 (1990). Stable multi-copy expression vectors for secretion of mature recombinant human serum albumin by industrial strains of <i>Kluyveromyces </i>have also been disclosed. Fleer et al., <i>Bio/Technology </i>9:968-975 (1991).</div>
</li> <li> <para-num num="[0235]"> </para-num> <div class="description-line" id="p-0241" num="0235">Signal Sequence Component—</div>
</li> <li> <para-num num="[0236]"> </para-num> <div class="description-line" id="p-0242" num="0236">The polypeptides may be produced recombinantly not only directly, but also as a fusion polypeptide with a heterologous polypeptide, which is preferably a signal sequence or other polypeptide having a specific cleavage site at the N-terminus of the mature polypeptide. The heterologous signal sequence selected preferably is one that is recognized and processed (i.e., cleaved by a signal peptidase) by the host cell. A signal sequence can be substituted with a prokaryotic signal sequence selected, for example, from the group of the alkaline phosphatase, penicillinase, 1 pp, or heat-stable enterotoxin II leaders. For yeast secretion the native signal sequence may be substituted by, e.g., the yeast invertase leader, a factor leader (including <i>Saccharomyces </i>and <i>Kluyveromyces </i>α-factor leaders), or acid phosphatase leader, the <i>C. albicans </i>glucoamylase leader, or the signal described in WO 90/13646. In mammalian cell expression, mammalian signal sequences as well as viral secretory leaders, for example, the herpes simplex gD signal, are available.</div>
</li> <li> <para-num num="[0237]"> </para-num> <div class="description-line" id="p-0243" num="0237">The nucleic acid sequence for such precursor region is ligated in reading frame to the nucleic acid sequence encoding the polypeptide described herein.</div>
</li> <li> <para-num num="[0238]"> </para-num> <div class="description-line" id="p-0244" num="0238">Origin of Replication—</div>
</li> <li> <para-num num="[0239]"> </para-num> <div class="description-line" id="p-0245" num="0239">Both expression and cloning vectors contain a polynucleotide sequence that enables the vector to replicate in one or more selected host cells. Generally, in cloning vectors this sequence is one that enables the vector to replicate independently of the host chromosomal DNA, and includes origins of replication or autonomously replicating sequences. Such sequences are well known for a variety of bacteria, yeast, and viruses. The origin of replication from the plasmid pBR322 is suitable for most Gram-negative bacteria, the 2μ plasmid origin is suitable for yeast, and various viral origins (SV40, polyoma, adenovirus, VSV or BPV) are useful for cloning vectors in mammalian cells. Generally, the origin of replication component is not needed for mammalian expression vectors (the SV40 origin may typically be used only because it contains the early promoter).</div>
</li> <li> <para-num num="[0240]"> </para-num> <div class="description-line" id="p-0246" num="0240">Promoter Component—</div>
</li> <li> <para-num num="[0241]"> </para-num> <div class="description-line" id="p-0247" num="0241">Expression and cloning vectors usually contain a promoter that is recognized by the host organism and is operably linked to the nucleic acid encoding the polypeptide. Promoters suitable for use with prokaryotic hosts include the phoA promoter, β-lactamase and lactose promoter systems, alkaline phosphatase promoter, a tryptophan (trp) promoter system, and hybrid promoters such as the tac promoter. However, other known bacterial promoters are suitable. Promoters for use in bacterial systems also will contain a Shine-Dalgarno (S.D.) sequence operably linked to the DNA encoding the polypeptide.</div>
</li> <li> <para-num num="[0242]"> </para-num> <div class="description-line" id="p-0248" num="0242">Suitably, the expression control sequences are eukaryotic promoter systems in vectors capable of transforming or transfecting eukaryotic host cells (e.g., COS cells—such as COS 7 cells—or CHO cells). Once the vector has been incorporated into the appropriate host, the host is maintained under conditions suitable for high level expression of the nucleotide sequences, and the collection and purification of the cross-reacting antibodies.</div>
</li> <li> <para-num num="[0243]"> </para-num> <div class="description-line" id="p-0249" num="0243">Promoter sequences are known for eukaryotes. Virtually all eukaryotic genes have an AT-rich region located approximately 25 to 30 bases upstream from the site where transcription is initiated. Another sequence found 70 to 80 bases upstream from the start of transcription of many genes is a CNCAAT region where N may be any nucleotide. At the 3′ end of most eukaryotic genes is an AATAAA sequence that may be the signal for addition of the poly A tail to the 3′ end of the coding sequence. All of these sequences are suitably inserted into eukaryotic expression vectors.</div>
</li> <li> <para-num num="[0244]"> </para-num> <div class="description-line" id="p-0250" num="0244">Examples of suitable promoter sequences for use with yeast hosts include the promoters for 3-phosphoglycerate kinase or other glycolytic enzymes, such as enolase, glyceraldehyde-3-phosphate dehydrogenase, hexokinase, pyruvate decarboxylase, phosphofructokinase, glucose-6-phosphate isomerase, 3-phosphoglycerate mutase, pyruvate kinase, triosephosphate isomerase, phosphoglucose isomerase, and glucokinase.</div>
</li> <li> <para-num num="[0245]"> </para-num> <div class="description-line" id="p-0251" num="0245">Other yeast promoters, which are inducible promoters having the additional advantage of transcription controlled by growth conditions, are the promoter regions for alcohol dehydrogenase 2, isocytochrome C, acid phosphatase, degradative enzymes associated with nitrogen metabolism, metallothionein, glyceraldehyde-3-phosphate dehydrogenase, and enzymes responsible for maltose and galactose utilization. Suitable vectors and promoters for use in yeast expression are further described in EP 73,657. Yeast enhancers also are advantageously used with yeast promoters.</div>
</li> <li> <para-num num="[0246]"> </para-num> <div class="description-line" id="p-0252" num="0246">The transcription of the polypeptides described herein from vectors in mammalian host cells is controlled, for example, by promoters obtained from the genomes of viruses such as polyoma virus, fowlpox virus, adenovirus (such as Adenovirus 2), bovine papilloma virus, avian sarcoma virus, cytomegalovirus, a retrovirus, hepatitis-B virus and most preferably Simian Virus 40 (SV40), from heterologous mammalian promoters, e.g., the actin promoter or an immunoglobulin promoter, from heat-shock promoters, provided such promoters are compatible with the host cell systems.</div>
</li> <li> <para-num num="[0247]"> </para-num> <div class="description-line" id="p-0253" num="0247">The early and late promoters of the SV40 virus are conveniently obtained as an SV40 restriction fragment that also contains the SV40 viral origin of replication. The immediate early promoter of the human cytomegalovirus is conveniently obtained as a HindIII E restriction fragment. A system for expressing DNA in mammalian hosts using the bovine papilloma virus as a vector is disclosed in U.S. Pat. No. 4,419,446. A modification of this system is described in U.S. Pat. No. 4,601,978. See also Reyes et al., <i>Nature </i>297:598-601 (1982) on expression of human β-interferon cDNA in mouse cells under the control of a thymidine kinase promoter from herpes simplex virus. Alternatively, the Rous Sarcoma Virus long terminal repeat can be used as the promoter.</div>
</li> <li> <para-num num="[0248]"> </para-num> <div class="description-line" id="p-0254" num="0248">Enhancer Element Component—</div>
</li> <li> <para-num num="[0249]"> </para-num> <div class="description-line" id="p-0255" num="0249">Transcription of a DNA encoding the polypeptide described herein by higher eukaryotes is often increased by inserting an enhancer sequence into the vector. Many enhancer sequences are now known from mammalian genes (globin, elastase, albumin, α-fetoprotein, and insulin). Typically, however, one will use an enhancer from a eukaryotic cell virus. Examples include the SV40 enhancer on the late side of the replication origin (bp 100-270), the cytomegalovirus early promoter enhancer, the polyoma enhancer on the late side of the replication origin, and adenovirus enhancers. See also Yaniv, <i>Nature </i>297:17-18 (1982) on enhancing elements for activation of eukaryotic promoters. The enhancer may be spliced into the vector at a position 5′ or 3′ to the polypeptide-encoding sequence, but is preferably located at a site 5′ from the promoter.</div>
</li> <li> <para-num num="[0250]"> </para-num> <div class="description-line" id="p-0256" num="0250">Transcription Termination Component—</div>
</li> <li> <para-num num="[0251]"> </para-num> <div class="description-line" id="p-0257" num="0251">Expression vectors used in eukaryotic host cells (yeast, fungi, insect, plant, animal, human, or nucleated cells from other multicellular organisms) will also contain sequences necessary for the termination of transcription and for stabilizing the mRNA. Such sequences are commonly available from the 5′ and, occasionally 3′, untranslated regions of eukaryotic or viral DNAs or cDNAs. One useful transcription termination component is the bovine growth hormone polyadenylation region. See WO94/11026 and the expression vector disclosed therein.</div>
</li> <li> <para-num num="[0252]"> </para-num> <div class="description-line" id="p-0258" num="0252">The vectors containing the polynucleotide sequences (e.g., the variable heavy and/or variable light chain encoding sequences and optional expression control sequences) can be transferred into a host cell by well-known methods, which vary depending on the type of cellular host. For example, calcium chloride transfection is commonly utilized for prokaryotic cells, whereas calcium phosphate treatment, electroporation, lipofection, biolistics or viral-based transfection may be used for other cellular hosts. (See generally Sambrook et al., <i>Molecular Cloning: A Laboratory Manual </i>(Cold Spring Harbor Press, 2nd ed., 1989). Other methods used to transform mammalian cells include the use of polybrene, protoplast fusion, liposomes, electroporation, and microinjection. For production of transgenic animals, transgenes can be microinjected into fertilized oocytes, or can be incorporated into the genome of embryonic stem cells, and the nuclei of such cells transferred into enucleated oocytes.</div>
</li> <li> <para-num num="[0253]"> </para-num> <div class="description-line" id="p-0259" num="0253">When heavy and light chains are cloned on separate expression vectors, the vectors are co-transfected to obtain expression and assembly of intact immunoglobulins. Once expressed, the whole antibodies, their dimers, individual light and heavy chains, or other immunoglobulin forms can be purified according to standard procedures of the art, including ammonium sulfate precipitation, affinity columns, column chromatography, HPLC purification, gel electrophoresis and the like (see generally Scopes, <i>Protein Purification </i>(Springer-Verlag, N.Y., (1982)). Substantially pure immunoglobulins of at least about 90 to 95% homogeneity are preferred, and about 98 to 99% or more homogeneity is most preferred, for pharmaceutical uses.</div>
</li> <heading id="h-0023">(C) Constructs</heading>
<li> <para-num num="[0254]"> </para-num> <div class="description-line" id="p-0260" num="0254">Typically the construct will be an expression vector allowing expression, in a suitable host, of the polypeptide(s) encoded by the polynucleotide. The construct may comprise, for example, one or more of the following: a promoter active in the host; one or more regulatory sequences, such as enhancers; an origin of replication; and a marker, preferably a selectable marker. The host may be a eukaryotic or prokaryotic host, although eukaryotic (and especially mammalian) hosts may be preferred. The selection of suitable promoters will obviously depend to some extent on the host cell used, but may include promoters from human viruses such as HSV, SV40, RSV and the like. Numerous promoters are known to those skilled in the art.</div>
</li> <li> <para-num num="[0255]"> </para-num> <div class="description-line" id="p-0261" num="0255">The construct may comprise a polynucleotide which encodes a polypeptide comprising three light chain hypervariable loops or three heavy chain hypervariable loops. Alternatively the polynucleotide may encode a polypeptide comprising three heavy chain hypervariable loops and three light chain hypervariable loops joined by a suitably flexible linker of appropriate length. Another possibility is that a single construct may comprise a polynucleotide encoding two separate polypeptides—one comprising the light chain loops and one comprising the heavy chain loops. The separate polypeptides may be independently expressed or may form part of a single common operon.</div>
</li> <li> <para-num num="[0256]"> </para-num> <div class="description-line" id="p-0262" num="0256">The construct may comprise one or more regulatory features, such as an enhancer, an origin of replication, and one or more markers (selectable or otherwise). The construct may take the form of a plasmid, a yeast artificial chromosome, a yeast mini-chromosome, or be integrated into all or part of the genome of a virus, especially an attenuated virus or similar which is non-pathogenic for humans.</div>
</li> <li> <para-num num="[0257]"> </para-num> <div class="description-line" id="p-0263" num="0257">The construct may be conveniently formulated for safe administration to a mammalian, preferably human, subject. Typically, they will be provided in a plurality of aliquots, each aliquot containing sufficient construct for effective immunization of at least one normal adult human subject.</div>
</li> <li> <para-num num="[0258]"> </para-num> <div class="description-line" id="p-0264" num="0258">The construct may be provided in liquid or solid form, preferably as a freeze-dried powder which, typically, is rehydrated with a sterile aqueous liquid prior to use.</div>
</li> <li> <para-num num="[0259]"> </para-num> <div class="description-line" id="p-0265" num="0259">The construct may be formulated with an adjuvant or other component which has the effect of increasing the immune response of the subject (e.g., as measured by specific antibody titer) in response to administration of the construct.</div>
</li> <heading id="h-0024">(D) Vectors</heading>
<li> <para-num num="[0260]"> </para-num> <div class="description-line" id="p-0266" num="0260">The term “vector” includes expression vectors and transformation vectors and shuttle vectors.</div>
</li> <li> <para-num num="[0261]"> </para-num> <div class="description-line" id="p-0267" num="0261">The term “expression vector” means a construct capable of in vivo or in vitro expression.</div>
</li> <li> <para-num num="[0262]"> </para-num> <div class="description-line" id="p-0268" num="0262">The term “transformation vector” means a construct capable of being transferred from one entity to another entity—which may be of the species or may be of a different species. If the construct is capable of being transferred from one species to another—such as from an <i>Escherichia coli </i>plasmid to a bacterium, such as of the genus <i>Bacillus</i>, then the transformation vector is sometimes called a “shuttle vector”. It may even be a construct capable of being transferred from an <i>E. coli </i>plasmid to an <i>Agrobacterium </i>to a plant.</div>
</li> <li> <para-num num="[0263]"> </para-num> <div class="description-line" id="p-0269" num="0263">Vectors may be transformed into a suitable host cell as described below to provide for expression of a polypeptide. Various vectors are publicly available. The vector may, for example, be in the form of a plasmid, cosmid, viral particle, or phage. The appropriate nucleic acid sequence may be inserted into the vector by a variety of procedures. In general, DNA is inserted into an appropriate restriction endonuclease site(s) using techniques known in the art. Construction of suitable vectors containing one or more of these components employs standard ligation techniques which are known to the skilled artisan.</div>
</li> <li> <para-num num="[0264]"> </para-num> <div class="description-line" id="p-0270" num="0264">The vectors may be for example, plasmid, virus or phage vectors provided with an origin of replication, optionally a promoter for the expression of the said polynucleotide and optionally a regulator of the promoter. Vectors may contain one or more selectable marker genes which are well known in the art.</div>
</li> <li> <para-num num="[0265]"> </para-num> <div class="description-line" id="p-0271" num="0265">These expression vectors are typically replicable in the host organisms either as episomes or as an integral part of the host chromosomal DNA.</div>
</li> <heading id="h-0025">(E) Host Cells</heading>
<li> <para-num num="[0266]"> </para-num> <div class="description-line" id="p-0272" num="0266">The host cell may be a bacterium, a yeast or other fungal cell, insect cell, a plant cell, or a mammalian cell, for example.</div>
</li> <li> <para-num num="[0267]"> </para-num> <div class="description-line" id="p-0273" num="0267">A transgenic multicellular host organism which has been genetically manipulated may be used to produce a polypeptide. The organism may be, for example, a transgenic mammalian organism (e.g., a transgenic goat or mouse line).</div>
</li> <li> <para-num num="[0268]"> </para-num> <div class="description-line" id="p-0274" num="0268">Suitable prokaryotes include but are not limited to eubacteria, such as Gram-negative or Gram-positive organisms, for example, Enterobacteriaceae such as <i>E. coli</i>. Various <i>E. coli </i>strains are publicly available, such as <i>E. coli </i>K12 strain MM294 (ATCC 31,446); <i>E. coli </i>X1776 (ATCC 31,537); <i>E. coli </i>strain W3110 (ATCC 27,325) and K5 772 (ATCC 53,635). Other suitable prokaryotic host cells include Enterobacteriaceae such as <i>Escherichia</i>, e.g., <i>E. coli, Enterobacter, Erwinia, Klebsiella, Proteus, Salmonella</i>, e.g., <i>Salmonella typhimurium, Serratia</i>, e.g., <i>Serratia marcescans</i>, and <i>Shigella</i>, as well as <i>Bacilli </i>such as <i>B. subtilis </i>and <i>B. licheniformis </i>(e.g., <i>B. licheniformis </i>41P), <i>Pseudomonas </i>such as <i>P. aeruginosa</i>, and <i>Streptomyces</i>. These examples are illustrative rather than limiting. Strain W3110 is one particularly preferred host or parent host because it is a common host strain for recombinant polynucleotide product fermentations. Preferably, the host cell secretes minimal amounts of proteolytic enzymes. For example, strain W3110 may be modified to effect a genetic mutation in the genes encoding polypeptides endogenous to the host, with examples of such hosts including <i>E. coli </i>W3110 strain 1A2, which has the complete genotype tonA; <i>E. coli </i>W3110 strain 9E4, which has the complete genotype tonA ptr3; <i>E. coli </i>W3110 strain 27C7 (ATCC 55,244), which has the complete genotype tonA ptr3 phoA E15 (argF-lac)169 degP ompT kan′; <i>E. coli </i>W3110 strain 37D6, which has the complete genotype tonA ptr3 phoA E15 (argF-lac)169 degP ompT rbs7 ilvG kan′; <i>E. coli </i>W3110 strain 40B4, which is strain 37D6 with a non-kanamycin resistant degP deletion mutation; and an <i>E. coli </i>strain having mutant periplasmic protease. Alternatively, in vitro methods of cloning, e.g., PCR or other nucleic acid polymerase reactions, are suitable.</div>
</li> <li> <para-num num="[0269]"> </para-num> <div class="description-line" id="p-0275" num="0269">In these prokaryotic hosts, one can make expression vectors, which will typically contain expression control sequences compatible with the host cell (e.g., an origin of replication). In addition, any number of a variety of well-known promoters will be present, such as the lactose promoter system, a tryptophan (trp) promoter system, a beta-lactamase promoter system, or a promoter system from phage lambda. The promoters will typically control expression, optionally with an operator sequence, and have ribosome binding site sequences and the like, for initiating and completing transcription and translation.</div>
</li> <li> <para-num num="[0270]"> </para-num> <div class="description-line" id="p-0276" num="0270">Eukaryotic microbes may be used for expression. Eukaryotic microbes such as filamentous fingi or yeast are suitable cloning or expression hosts for polypeptide-encoding vectors. <i>Saccharomyces cerevisiae </i>is a commonly used lower eukaryotic host microorganism. Others include <i>Schizosaccharomyces pombe; Kluyveromyces </i>hosts such as, e.g., <i>K. lactis </i>(MW98-8C, CBS683, CBS4574), <i>K. fragilis </i>(ATCC 12,424), <i>K. bulgaricus </i>(ATCC 16,045), <i>K. wickeramii </i>(ATCC 24,178), <i>K. waltii </i>(ATCC 56,500), K <i>drosophilarum </i>(ATCC 36,906), <i>K. thermotolerans</i>, and <i>K marxianus; yarrowia </i>(EP 402,226); <i>Pichia </i>pastor's; <i>Candida; Trichoderma </i>reesia; <i>Neurospora crassa; Schwanniomyces </i>such as <i>Schwanniomyces occidentalis</i>; and filamentous fungi such as, e.g., <i>Neurospora, Penicillium, Tolypocladium</i>, and <i>Aspergillus </i>hosts such as <i>A. nidulans</i>, and <i>A. niger</i>. Methylotropic yeasts are suitable herein and include, but are not limited to, yeast capable of growth on methanol selected from the genera consisting of <i>Hansenula, Candida, Kloeckera, Pichia, Saccharomyces, Torulopsis</i>, and <i>Rhodotorula. Saccharomyces </i>is a preferred yeast host, with suitable vectors having expression control sequences (e.g., promoters), an origin of replication, termination sequences and the like as desired. Typical promoters include 3-phosphoglycerate kinase and other glycolytic enzymes. Inducible yeast promoters include, among others, promoters from alcohol dehydrogenase, isocytochrome C, and enzymes responsible for maltose and galactose utilization.</div>
</li> <li> <para-num num="[0271]"> </para-num> <div class="description-line" id="p-0277" num="0271">In addition to microorganisms, mammalian tissue cell culture may also be used to express and produce the polypeptides as described herein and in some instances are preferred (See Winnacker, <i>From Genes to Clones </i>VCH Publishers, N.Y., N.Y. (1987). For some embodiments, eukaryotic cells may be preferred, because a number of suitable host cell lines capable of secreting heterologous polypeptides (e.g., intact immunoglobulins) have been developed in the art, and include CHO cell lines, various Cos cell lines, HeLa cells, preferably, myeloma cell lines, or transformed B-cells or hybridomas. In some embodiments, the mammalian host cell is a CHO cell.</div>
</li> <li> <para-num num="[0272]"> </para-num> <div class="description-line" id="p-0278" num="0272">In some embodiments, the host cell is a vertebrate host cell. Examples of useful mammalian host cell lines are monkey kidney CV1 line transformed by SV40 (COS-7, ATCC CRL 1651); human embryonic kidney line (293 or 293 cells subcloned for growth in suspension culture); baby hamster kidney cells (BHK, ATCC CCL 10); Chinese hamster ovary cells/-DHFR(CHO or CHO-DP-12 line); mouse sertoli cells; monkey kidney cells (CV1 ATCC CCL 70); African green monkey kidney cells (VERO-76, ATCC CRL-1587); human cervical carcinoma cells (HELA, ATCC CCL 2); canine kidney cells (MDCK, ATCC CCL 34); buffalo rat liver cells (BRL 3A, ATCC CRL 1442); human lung cells (W138, ATCC CCL 75); human liver cells (Hep G2, HB 8065); mouse mammary tumor (MMT 060562, ATCC CCL51); TRI cells; MRC 5 cells; FS4 cells; and a human hepatoma line (Hep G2).</div>
</li> <li> <para-num num="[0273]"> </para-num> <div class="description-line" id="p-0279" num="0273">Alternatively, polypeptide-coding sequences can be incorporated into transgenes for introduction into the genome of a transgenic animal and subsequent expression in the milk of the transgenic animal. Suitable transgenes include coding sequences for light and/or heavy chains in operable linkage with a promoter and enhancer from a mammary gland specific gene, such as casein or beta lactoglobulin.</div>
</li> <li> <para-num num="[0274]"> </para-num> <div class="description-line" id="p-0280" num="0274">Alternatively, the antibodies described herein can be produced in transgenic plants (e.g., tobacco, maize, soybean and alfalfa). Improved ‘plantibody’ vectors (Hendy et al., <i>J. Immunol. Methods </i>231:137-146 (1999)) and purification strategies coupled with an increase in transformable crop species render such methods a practical and efficient means of producing recombinant immunoglobulins not only for human and animal therapy, but for industrial applications as well (e.g., catalytic antibodies). Moreover, plant produced antibodies have been shown to be safe and effective and avoid the use of animal-derived materials. Further, the differences in glycosylation patterns of plant and mammalian cell-produced antibodies have little or no effect on antigen binding or specificity. In addition, no evidence of toxicity or HAMA has been observed in patients receiving topical oral application of a plant-derived secretory dimeric IgA antibody (see Larrick et al., <i>Res. Immunol. </i>149:603-608 (1998)).</div>
</li> <li> <para-num num="[0275]"> </para-num> <div class="description-line" id="p-0281" num="0275">Host cells are transfected or transformed with expression or cloning vectors described herein for polypeptide production and cultured in conventional nutrient media modified as appropriate for inducing promoters, selecting transformants, or amplifying the genes encoding the desired sequences. The culture conditions, such as media, temperature, pH and the like, can be selected by the skilled artisan without undue experimentation. In general, principles, protocols, and practical techniques for maximizing the productivity of cell cultures can be found in <i>Mammalian Cell Biotechnology: a Practical Approach </i>M. Butler, ed. (IRL Press, 1991).</div>
</li> <li> <para-num num="[0276]"> </para-num> <div class="description-line" id="p-0282" num="0276">Methods of eukaryotic cell transfection and prokaryotic cell transformation are known to the ordinarily skilled artisan, for example, CaCl<sub>2</sub>), CaPO<sub>4</sub>, liposome-mediated and electroporation. Depending on the host cell used, transformation is performed using standard techniques appropriate to such cells. The calcium treatment employing calcium chloride or electroporation is generally used for prokaryotes. Infection with <i>Agrobacterium tumefaciens </i>is used for transformation of certain plant cells, as described by Shaw et al., <i>Gene </i>23:315 (1983) and WO 89/05859 published 29 Jun. 1989. For mammalian cells without such cell walls, the calcium phosphate precipitation method of Graham and van der Eb, Virology 52:456-457 (1978) can be employed. General aspects of mammalian cell host system transfections have been described in U.S. Pat. No. 4,399,216. Transformations into yeast are typically carried out according to the method of Van Solingen et al., <i>J. Bact. </i>130:946 (1977) and Hsiao et al., <i>Proc. Natl. Acad. Sci</i>. (<i>USA</i>) 76:3829 (1979). However, other methods for introducing DNA into cells, such as by nuclear microinjection, electroporation, bacterial protoplast fusion with intact cells, or polycations, e.g., polybrene, polyornithine, may also be used. For various techniques for transforming mammalian cells, see Keown et al., <i>Methods in Enzymology </i>185:527-537 (1990) and Mansour et al., <i>Nature </i>336:348-352 (1988).</div>
</li> <li> <para-num num="[0277]"> </para-num> <div class="description-line" id="p-0283" num="0277">Polypeptides, e.g., antibodies, can be produced in bacteria, in particular when glycosylation and Fc effector function are not needed, such as when the therapeutic antibody is conjugated to a cytotoxic agent (e.g., a toxin) and the immunoconjugate by itself shows effectiveness in tumor cell destruction. Full length antibodies have greater half life in circulation. Production in <i>E. coli </i>is faster and more cost efficient. For expression of polypeptides in bacteria, see, e.g., U.S. Pat. No. 5,840,523, which describes translation initiation region (TIR) and signal sequences for optimizing expression and secretion, these patents incorporated herein by reference. After expression, the antibody is isolated from the <i>E. coli </i>cell paste in a soluble fraction and can be purified through, e.g., a protein A or G column depending on the isotype. Final purification can be carried out similar to the process for purifying antibody expressed e.g., in CHO cells.</div>
</li> <li> <para-num num="[0278]"> </para-num> <div class="description-line" id="p-0284" num="0278">Suitable host cells for the expression of glycosylated polypeptides described herein are derived from multicellular organisms. Examples of invertebrate cells include plant and insect cells. Numerous baculoviral strains and variants and corresponding permissive insect host cells from hosts such as <i>Spodoptera frugiperda </i>(caterpillar), <i>Aedes aegypti </i>(mosquito), <i>Aedes albopictus </i>(mosquito), <i>Drosophila melanogaster </i>(fruitfly), and <i>Bombyx mori </i>have been identified. A variety of viral strains for transfection are publicly available, e.g., the L-1 variant of <i>Autographa californica </i>NPV and the Bm-5 strain of <i>Bombyx mori </i>NPV, and such viruses may be used as the virus herein, particularly for transfection of <i>Spodoptera frugiperda </i>cells.</div>
</li> <li> <para-num num="[0279]"> </para-num> <div class="description-line" id="p-0285" num="0279">The host cells used to produce the polypeptide may be cultured in a variety of media. Commercially available media such as Ham's F10 (Sigma), Minimal Essential Medium (MEM), (Sigma), RPMI-1640 (Sigma), and Dulbecco's Modified Eagle's Medium ((DMEM), Sigma) are suitable for culturing the host cells. Any of these media may be supplemented as necessary with hormones and/or other growth factors (such as insulin, transferrin, or epidermal growth factor), salts (such as sodium chloride, calcium, magnesium, and phosphate), buffers (such as HEPES), nucleotides (such as adenosine and thymidine), antibiotics (such as GENTAMYCIN™ drug), trace elements (defined as inorganic compounds usually present at final concentrations in the micromolar range), and glucose or an equivalent energy source. Any other necessary supplements may also be included at appropriate concentrations that would be known to those skilled in the art. The culture conditions, such as temperature, pH, and the like, are those previously used with the host cell selected for expression, and will be apparent to the ordinarily skilled artisan.</div>
</li> <heading id="h-0026">V. Formulations and Methods of Making of the Formulations</heading>
<li> <para-num num="[0280]"> </para-num> <div class="description-line" id="p-0286" num="0280">Provided herein are also formulations and methods of making the formulation comprising the polypeptides (e.g., antibodies) purified by the methods described herein. For example, the purified polypeptide may be combined with a pharmaceutically acceptable carrier.</div>
</li> <li> <para-num num="[0281]"> </para-num> <div class="description-line" id="p-0287" num="0281">The polypeptide formulations in some embodiments may be prepared for storage by mixing a polypeptide having the desired degree of purity with optional pharmaceutically acceptable carriers, excipients or stabilizers (Remington's Pharmaceutical Sciences 16th edition, Osol, A. Ed. (1980)), in the form of lyophilized formulations or aqueous solutions.</div>
</li> <li> <para-num num="[0282]"> </para-num> <div class="description-line" id="p-0288" num="0282">“Carriers” as used herein include pharmaceutically acceptable carriers, excipients, or stabilizers which are nontoxic to the cell or mammal being exposed thereto at the dosages and concentrations employed. Often the physiologically acceptable carrier is an aqueous pH buffered solution.</div>
</li> <li> <para-num num="[0283]"> </para-num> <div class="description-line" id="p-0289" num="0283">Acceptable carriers, excipients, or stabilizers are nontoxic to recipients at the dosages and concentrations employed, and include buffers such as phosphate, citrate, and other organic acids; antioxidants including ascorbic acid and methionine; preservatives (such as octadecyldimethylbenzyl ammonium chloride; hexamethonium chloride; benzalkonium chloride, benzethonium chloride; phenol, butyl or benzyl alcohol; alkyl parabens such as methyl or propyl paraben; catechol; resorcinol; cyclohexanol; 3-pentanol; and m-cresol); low molecular weight (less than about 10 residues) polypeptides; proteins, such as serum albumin, gelatin, or immunoglobulins; hydrophilic polymers such as polyvinylpyrrolidone; amino acids such as glycine, glutamine, asparagine, histidine, arginine, or lysine; monosaccharides, disaccharides, and other carbohydrates including glucose, mannose, or dextrins; chelating agents such as EDTA; sugars such as sucrose, mannitol, trehalose or sorbitol; salt-forming counter-ions such as sodium; metal complexes (e.g. Zn-protein complexes); and/or non-ionic surfactants such as TWEEN™, PLURONICS™ or polyethylene glycol (PEG).</div>
</li> <li> <para-num num="[0284]"> </para-num> <div class="description-line" id="p-0290" num="0284">In some embodiments, the polypeptide in the polypeptide formulation maintains functional activity.</div>
</li> <li> <para-num num="[0285]"> </para-num> <div class="description-line" id="p-0291" num="0285">The formulations to be used for in vivo administration must be sterile. This is readily accomplished by filtration through sterile filtration membranes.</div>
</li> <li> <para-num num="[0286]"> </para-num> <div class="description-line" id="p-0292" num="0286">The formulations herein may also contain more than one active compound as necessary for the particular indication being treated, preferably those with complementary activities that do not adversely affect each other. For example, in addition to a polypeptide, it may be desirable to include in the one formulation, an additional polypeptide (e.g., antibody). Alternatively, or additionally, the composition may further comprise a chemotherapeutic agent, cytotoxic agent, cytokine, growth inhibitory agent, anti-hormonal agent, and/or cardioprotectant. Such molecules are suitably present in combination in amounts that are effective for the purpose intended.</div>
</li> <heading id="h-0027">V. Articles of Manufacture</heading>
<li> <para-num num="[0287]"> </para-num> <div class="description-line" id="p-0293" num="0287">The polypeptides purified by the methods described herein and/or formulations comprising the polypeptides purified by the methods described herein may be contained within an article of manufacture. The article of manufacture may comprise a container containing the polypeptide and/or the polypeptide formulation. Preferably, the article of manufacture comprises: (a) a container comprising a composition comprising the polypeptide and/or the polypeptide formulation described herein within the container; and (b) a package insert with instructions for administering the formulation to a subject.</div>
</li> <li> <para-num num="[0288]"> </para-num> <div class="description-line" id="p-0294" num="0288">The article of manufacture comprises a container and a label or package insert on or associated with the container. Suitable containers include, for example, bottles, vials, syringes, etc. The containers may be formed from a variety of materials such as glass or plastic. The container holds or contains a formulation and may have a sterile access port (for example the container may be an intravenous solution bag or a vial having a stopper pierceable by a hypodermic injection needle). At least one active agent in the composition is the polypeptide. The label or package insert indicates that the composition's use in a subject with specific guidance regarding dosing amounts and intervals of polypeptide and any other drug being provided. The article of manufacture may further include other materials desirable from a commercial and user standpoint, including other buffers, diluents, filters, needles, and syringes. In some embodiments, the container is a syringe. In some embodiments, the syringe is further contained within an injection device. In some embodiments, the injection device is an autoinjector.</div>
</li> <li> <para-num num="[0289]"> </para-num> <div class="description-line" id="p-0295" num="0289">A “package insert” is used to refer to instructions customarily included in commercial packages of therapeutic products, that contain information about the indications, usage, dosage, administration, contraindications, other therapeutic products to be combined with the packaged product, and/or warnings concerning the use of such therapeutic products.</div>
</li> <li> <para-num num="[0290]"> </para-num> <div class="description-line" id="p-0296" num="0290">Further details of the invention are illustrated by the following non-limiting Examples. The disclosures of all references in the specification are expressly incorporated herein by reference.</div>
</li> <heading id="h-0028">EXAMPLES</heading>
<li> <para-num num="[0291]"> </para-num> <div class="description-line" id="p-0297" num="0291">The examples below are intended to be purely exemplary of the invention and should therefore not be considered to limit the invention in any way. The following examples and detailed description are offered by way of illustration and not by way of limitation.</div>
</li> <heading id="h-0029">Example 1—Selection of Overloaded Cation Exchange Materials</heading>
<li> <para-num num="[0292]"> </para-num> <div class="description-line" id="p-0298" num="0292">This example describes a cation exchange chromatography process for purifying a recombinant humanized anti-vascular endothelial growth factor (VEGF) antibody, a recombinant humanized anti-CD11a antibody, and a recombinant chimeric anti-CD20 antibody. The structures of anti-VEGF antibody, anti-CD11a antibody, and anti-CD20 antibody were disclosed in U.S. Pat. Nos. 7,169,901, and 6,703,018, and 5,736,137, respectively, each of which is expressly incorporated herein by reference.</div>
</li> <heading id="h-0030">Materials and Methods</heading>
<li> <para-num num="[0293]"> </para-num> <div class="description-line" id="p-0299" num="0293">Purification Methods</div>
</li> <li> <para-num num="[0294]"> </para-num> <div class="description-line" id="p-0300" num="0294">The cell culture fluid containing monoclonal antibody produced in Chinese hamster ovary cells was processed by continuous centrifuge to remove cellular debris, and further clarified through filtration with depth filters and 0.2 um filter. The harvested cell culture fluids were purified through protein A chromatography. The loading and washing conditions were at neutral pH range and elution was carried out at low pH range during the chromatography. pH and conductivity of the protein pools obtained from the protein A column were adjusted to the loading conditions used for cation exchange chromatography and then 0.22 um filtered. The same pH and conductivity were used for the cation exchange chromatography loading conditions, equilibration conditions, and initial wash conditions. The filtered and equilibrated protein pools were used as the product for subsequent cation exchange chromatography. Concentration of product (g/L) was determined by absorbance 280 nm. Specific cation exchange chromatography conditions were performed as indicated in the Experimental Procedures and Results section below.</div>
</li> <li> <para-num num="[0295]"> </para-num> <div class="description-line" id="p-0301" num="0295">Assays</div>
</li> <li> <para-num num="[0296]"> </para-num> <div class="description-line" id="p-0302" num="0296">Assays for all examples were performed as indicated below.</div>
</li> <li> <para-num num="[0297]"> </para-num> <div class="description-line" id="p-0303" num="0297">Antibody Concentration Assay</div>
</li> <li> <para-num num="[0298]"> </para-num> <div class="description-line" id="p-0304" num="0298">Antibody concentration in harvested cell culture fluid (HCCF) was determined using a Poros protein A HPLC assay (Applied Biosystems, Foster City, Calif.). The column was operated at a flow rate of 2.0 mL/min where it was equilibrated in sodium phosphate/sodium chloride buffer at pH 6.3 and eluted with acetic acid/glycine solution at pH 2.5. Absorbance at 280 nm was monitored and the elution peak area used to quantify antibody concentration from a standard curve. Antibody in purified pools obtained through protein A affinity or ion exchange chromatography was determined by absorbance at 280 nm (with absorbance at 320 nm subtracted to correct for light scattering), using a spectrophotometer with 10 mm path length flow cell. Antibody concentration was calculated as [(absorbance at 280 nm−absorbance at 320 nm)×dilution]/extinction coefficient.</div>
</li> <li> <para-num num="[0299]"> </para-num> <div class="description-line" id="p-0305" num="0299">CHOP Assay</div>
</li> <li> <para-num num="[0300]"> </para-num> <div class="description-line" id="p-0306" num="0300">Two CHOP assays, enzyme linked immunosorbent assay (ELISA) and Meso Scale Discovery (MSD) assays, were used.</div>
</li> <li> <para-num num="[0301]"> </para-num> <div class="description-line" id="p-0307" num="0301">CHOP ELISA Assay—</div>
</li> <li> <para-num num="[0302]"> </para-num> <div class="description-line" id="p-0308" num="0302">Samples from selected runs were submitted to an assay group that performed a standard and validated ELISA to quantitate the levels of CHOP. Affinity-purified goat anti-CHOP antibodies were immobilized on microtiter plate wells. Dilutions of the samples containing CHOP, standards, and controls, were incubated in the wells, followed by incubation with goat anti-CHOP antibodies conjugated to horseradish peroxidase. The horseradish peroxidase enzymatic activity was detected with o-phenylenediamine dihydrochloride. The CHOP was quantitated by reading absorbance at 492 nm in a microtiter plate reader. A computer curve-fitting program was used to generate the standard curve and automatically calculate the sample concentration. The assay range for the ELISA was typically 5 ng/mL to 320 ng/mL, and results were standardized to ppm for pool comparisons.</div>
</li> <li> <para-num num="[0303]"> </para-num> <div class="description-line" id="p-0309" num="0303">CHOP MSD Assay—</div>
</li> <li> <para-num num="[0304]"> </para-num> <div class="description-line" id="p-0310" num="0304">Samples from selected runs were submitted to an assay group that performed a standard and validated MSD assay to quantitate the levels of CHOP Affinity-purified goat anti-CHOP antibodies were immobilized on microtiter plate wells. Dilutions of the samples containing CHOP, standards, and controls, were incubated in the wells, followed by incubation with goat anti-CHOP antibodies conjugated to MSD SULFO-TAG NHS-ESTER. The NHS-ESTER has an amine-reactive linker, N-hydroxysuccinimide ester, which readily couples to primary amine groups of proteins. The CHOP concentration was measured after addition of the MSD Read Buffer and applying electricity to the bottom of the plate. The labeled detection antibodies interacted with the tripropylamine-containing Read Buffer to emit light upon electrochemical stimulation initiated at the electrode surfaces of Multi-Array microplates at 620 nm and was measured by a CCD camera. The concentration was then obtained by inverse regression on the standard curve. The assay range for the assay was 5 ng/mL to 1530 ng/mL, and results were standardized to ppm for pool comparison.</div>
</li> <li> <para-num num="[0305]"> </para-num> <div class="description-line" id="p-0311" num="0305">Protein A Assay</div>
</li> <li> <para-num num="[0306]"> </para-num> <div class="description-line" id="p-0312" num="0306">Leached Protein A in samples where Protein A chromatography has been used for the recovery process was detected. The level of Protein-A was determined by a sandwich Protein-A ELISA. Chicken anti-staphylococcal protein A antibodies were immobilized on microtiter plate wells. The sample treatment procedure included sample dilution and then dissociation of the Protein A/IgG complex using microwave assisted heating as a pretreatment step before running the samples on a sandwich ELISA. Protein A, if present in the sample, bound the coat antibody. Bound protein A was detected with horseradish peroxidase conjugated anti-protein antibodies. Horseradish peroxidase enzymatic activity was quantified with a 2 component TMB substrate solution which produces a colorimetric signal.</div>
</li> <li> <para-num num="[0307]"> </para-num> <div class="description-line" id="p-0313" num="0307">Gentamicin Assay</div>
</li> <li> <para-num num="[0308]"> </para-num> <div class="description-line" id="p-0314" num="0308">A competitive ELISA was used to determine gentamicin concentrations in all of the pools. Goat polyclonal antibodies to gentamicin-BSA were immobilized on microtiter platre wells. Gentamicin competed with biotinylated-gentamicin for binding to the antibodies. The amount of bound biotin-labeled gentamicin was detected with the addition of horseradish peroxidase-streptavidin and o-phenylendiamine substract. Samples were serially diluted in assay diluent so that the absorbance reading fell within the quantifiable range of the assay (0.37-90 ng/mL).</div>
</li> <li> <para-num num="[0309]"> </para-num> <div class="description-line" id="p-0315" num="0309">Size Exclusion Chromatography</div>
</li> <li> <para-num num="[0310]"> </para-num> <div class="description-line" id="p-0316" num="0310">The size heterogeneity of the monoclonal antibodies was determined by a high-performance size exclusion chromatography assay. It employed a TSK-GEL G3000SWXL column (7.8 mm×300 mm, Tosoh Bioscience, Montgomeryville, Pa.) to separate monomer and higher molecular weight species (HMWs). The column was operated at a flow rate of 0.3 mL/min using a 200 mM potassium phosphate, 250 mM potassium chloride pH 6.2 mobile phase. 20<sub>j</sub>tg of antibody was injected for each sample. Absorbance at 280 nm was used to monitor the separation of monomer and HMWs. Percentages of monomer and HMWs were calculated based on their peak areas.</div>
</li> <li> <para-num num="[0311]"> </para-num> <div class="description-line" id="p-0317" num="0311">DNA Assay</div>
</li> <li> <para-num num="[0312]"> </para-num> <div class="description-line" id="p-0318" num="0312">Taqman PCR for CHO cell DNA assay uses real-time PCR to detect and quantify CHO DNA in product samples. DNA from samples and controls are first extracted using Qiagen's Virus Biorobot kit. The extracted samples, controls, and standard DNA, are subject to TaqMan real time Polymerase chain reaction (PCR) using PCR primers and probe in a 96-well plate with ABI's sequence detection system. The primers are defined by a 110 base pair segment of a repetitive DNA sequence in the <i>Cricetulus griseus </i>genome. The probe was labeled with a fluorescent reporter dye at 5′ end and a quencher dye at the 3′ end. When the probe is intact, the emission spectrum of the reporter is suppressed by the quencher. The 5′ nuclease activity of polymerase hydrolyzes the probe and releases the report, which results in an increase in fluorescence emission. The sequence detector quantifies the amplified product in direct proportion to the increase in fluorescence emission measured continuously during the DNA amplification. Cycle numbers at which DNA has amplified past the threshold (CT) are calculated for the standard curve. A standard curve ranging 1 pg/mL—10,000 pg/mL is generated, which is used for quantifying DNA in samples.</div>
</li> <heading id="h-0031">Experimental Procedures and Results</heading>
<li> <para-num num="[0313]"> </para-num> <div class="description-line" id="p-0319" num="0313">Purification of Anti-CD11a Antibody</div>
</li> <li> <para-num num="[0314]"> </para-num> <div class="description-line" id="p-0320" num="0314">Three types of cation exchange materials were evaluated in this study: resin, membrane, and monolith. The overloaded cation exchange purification processes of anti-CD11a antibody using these three cation exchange materials were evaluated with respect to process performance (e.g., impurities removal, step yield, and product quality).</div>
</li> <li> <para-num num="[0315]"> </para-num> <div class="description-line" id="p-0321" num="0315">The chromatograms of the three types of cation exchange materials, resin (Poros HS50 column (0.66 cm inner diameter (“i.d.”)×5 cm, 15.1 CV/hr) and SP Sepharose FF (“SPSFF”) (0.66 cm i.d.×4.8 cm, 15.7 CV/hr)), monolith (SO3 Monolith (0.34 mL, 353 MV/hr)), and membrane (Mustang S (0.18 mL, 333 MV/hr)), loaded with the product up to 1000 g/L CV (column volume) or MV (membrane or monolith volume), were run at pH 5.5 and 3 mS/cm. See <figref idrefs="DRAWINGS">FIGS. 1A-1D</figref>.</div>
</li> <li> <para-num num="[0316]"> </para-num> <div class="description-line" id="p-0322" num="0316">The performance of the three types of cation exchange materials, resin (Poros HS50 column (15.1 CV/hr or 75 cm/hr) and SPSFF column (15.7 CV/hr or 75 cm/hr)), membrane (Mustang S (333 MV/hr)), and monolith (SO3 Monolith (353 MV/hr)) with varying amount of the product loaded up to 1000 g/L CV or MV was evaluated at pH 5.5 and 3 mS/cm. As shown in <figref idrefs="DRAWINGS">FIG. 2</figref>, there was an over 93% monomer anti-CD11a antibody recovery for Mustang S, Monolith SO3, and Poros HS50 and about 88% monomer anti-CD11a antibody recovery for SPSFF.</div>
</li> <li> <para-num num="[0317]"> </para-num> <div class="description-line" id="p-0323" num="0317">The ability of the three types of cation exchange materials, resin (Poros HS50 column (15.1 CV/hr or 75 cm/hr) and SPSFF column (15.7 CV/hr or 75 cm/hr)), membrane (Mustang S (333 MV/hr)), and monolith (SO3 Monolith (353 MV/hr)) with varying amount of the product loaded up to 1000 g/L CV or MV to remove CHOP was evaluated at pH 5.5 and 3 mS/cm. As shown in <figref idrefs="DRAWINGS">FIG. 2</figref>, the monolith material, SO3 Monolith, was better at removing CHOP than the other two cation exchange materials. The membrane (Mustang S) and the Poros HS50 column were similar and better than the SPSFF column in removing CHOP. See <figref idrefs="DRAWINGS">FIG. 2</figref>.</div>
</li> <li> <para-num num="[0318]"> </para-num> <div class="description-line" id="p-0324" num="0318">The ability of the three types of cation exchange materials, resin (Poros HS50 column (15.1 CV/hr or 75 cm/hr) and SPSFF column (15.7 CV/hr or 75 cm/hr)), membrane (Mustang S (333 MV/hr)), and monolith (SO3 Monolith (353 MV/hr)) with varying amount of the product loaded up to 1000 g/L CV or MV to remove HMW was evaluated at pH 5.5 and 3 mS/cm. As shown in <figref idrefs="DRAWINGS">FIG. 3</figref>, the resin cation exchange material, Poros HS50, was most effective in removing HMW, followed by SPSFF. Further, Mustang S was better than SO3 monolith in removing HMW. See <figref idrefs="DRAWINGS">FIG. 3</figref>.</div>
</li> <li> <para-num num="[0319]"> </para-num> <div class="description-line" id="p-0325" num="0319">The ability of the cation exchange materials, resin (Poros HS50 column and SPSFF column), membrane (Mustang S), and monolith (SO3 Monolith) with varying amount of the product loaded up to 1000 g/L CV or MV to remove HMW1 and HMW2 was also evaluated at pH 5.5 and 3 mS/cm. As shown in <figref idrefs="DRAWINGS">FIGS. 4A-4D</figref>, Poros HS50 and Mustang S were more effective in removing HMW2.</div>
</li> <li> <para-num num="[0320]"> </para-num> <div class="description-line" id="p-0326" num="0320">The ability of Poros HS50 (15.1 CV/hr or 75 cm/hr) to remove HMW1 and HMW2 with the product loaded at varying amount up to 1000 mg/mL CV was examined at pH 5.5 and 3 mS/cm. The HMW was 6.26% and 60% in the load and in the elution pool, respectively. As shown in <figref idrefs="DRAWINGS">FIGS. 5A-5C</figref>, Poros HS50 was effective in removing both HMW1 and HMW2.</div>
</li> <li> <para-num num="[0321]"> </para-num> <div class="description-line" id="p-0327" num="0321">The ability of SPSFF to remove HMW was evaluated with the product loaded at varying amount of product up to about 800 g/L CV at pH 5.5 and 3 mS/cm. <figref idrefs="DRAWINGS">FIGS. 6A-6C</figref> shows the chromatograms and the HMW removal.</div>
</li> <li> <para-num num="[0322]"> </para-num> <div class="description-line" id="p-0328" num="0322">The performance of the three types of cation exchange materials, resin (Poros HS50 column and SPSFF column), membrane (Mustang S), and monolith (SO3 Monolith) were summarized in Table 2. The commercial process for purifying anti-CD11a used the bind-elute mode and the loading density of 15-44 mg/mL CV. As shown in Table 2, SO3 Monolith was most effective in removing CHOP and Poros HS resin was most effective in removing HMWs. The overloaded Poros HS process was similar in HMW and CHOP removals as the commercial SPSFF process. See Table 2.</div>
</li> <li> <div class="description-line" id="p-0329" num="0000">
<tables id="TABLE-US-00002" num="00002">
<patent-tables colsep="0" frame="none" pgwide="1" rowsep="0">
<table align="left" class="description-table" cols="2" colsep="0" rowsep="0" width="100%">
<thead>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td" colspan="2" nameend="1" namest="offset" rowsep="1">TABLE 2</td>
</tr>
</thead>
<tbody><tr class="description-tr">
<td class="description-td"> </td>
<td align="center" class="description-td" colspan="2" nameend="1" namest="offset" rowsep="1"> </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">In CEX pool</td>
</tr>
</tbody></table>
<table align="left" class="description-table" cols="7" colsep="0" rowsep="0" width="100%">
<tbody><tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">Commercial</td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">SPSFF</td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">SO<sub>3</sub> </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">In load (g/L)</td>
<td class="description-td">process</td>
<td class="description-td">Poros HS</td>
<td class="description-td">SPSFF</td>
<td class="description-td">Mustang S</td>
<td class="description-td">monolith</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td align="center" class="description-td" colspan="7" nameend="6" namest="offset" rowsep="1"> </td>
</tr>
</tbody></table>
<table align="left" class="description-table" cols="7" colsep="0" rowsep="0" width="100%">
<tbody><tr class="description-tr">
<td class="description-td">CHOP</td>
<td class="description-td">600-900 in the</td>
<td class="description-td">72-183</td>
<td class="description-td">&lt;150</td>
<td class="description-td">&lt;400</td>
<td class="description-td">&lt;150</td>
<td class="description-td">&lt;60</td>
</tr>
<tr class="description-tr">
<td class="description-td">ppm</td>
<td class="description-td">commercial</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">process;</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">600 in the</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">present study</td>
</tr>
<tr class="description-tr">
<td class="description-td">HMW</td>
<td class="description-td">~5 in the</td>
<td class="description-td">0.2</td>
<td class="description-td">0.4</td>
<td class="description-td">1</td>
<td class="description-td">2.7</td>
<td class="description-td">4.7</td>
</tr>
<tr class="description-tr">
<td class="description-td">%</td>
<td class="description-td">commercial</td>
<td class="description-td"> </td>
<td class="description-td">500 g/L CV</td>
<td class="description-td">500 g/L CV</td>
<td class="description-td">500 g/L MV</td>
<td class="description-td">500 g/L MV</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">process;</td>
<td class="description-td"> </td>
<td class="description-td">collected</td>
<td class="description-td">collected</td>
<td class="description-td">collected</td>
<td class="description-td">collected</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">6.2 in the</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">present study</td>
</tr>
<tr class="description-tr">
<td align="center" class="description-td" colspan="7" nameend="7" namest="1" rowsep="1"> </td>
</tr>
</tbody></table>
</patent-tables>
</tables>
</div>
</li> <li> <para-num num="[0323]"> </para-num> <div class="description-line" id="p-0330" num="0323">Purification of Anti-VEGF Antibody</div>
</li> <li> <para-num num="[0324]"> </para-num> <div class="description-line" id="p-0331" num="0324">Three types of cation exchange materials were evaluated in this study: resin, membrane, and monolith. The overloaded cation exchange purification processes of anti-VEGF antibody using these three cation exchange materials were evaluated with respect to process performance (e.g., impurities removal, step yield, and product quality).</div>
</li> <li> <para-num num="[0325]"> </para-num> <div class="description-line" id="p-0332" num="0325">The chromatograms of the three types of cation exchange materials, resin (Poros HS50 column (0.66 cm i.d.×5.4 cm, 18.5 CV/hr) and SPSFF column (0.66 cm i.d.×5.5 cm, 18.5 CV/hr)), monolith (SO3 Monolith (0.34 mL, 176 MV/hr)), and membrane (Mustang S (0.35 mL, 171 MV/hr)), loaded with the product close to 1000 g/L CV or MV, were run at pH 5.5 and 5 mS/cm. See <figref idrefs="DRAWINGS">FIGS. 7A-7D</figref>.</div>
</li> <li> <para-num num="[0326]"> </para-num> <div class="description-line" id="p-0333" num="0326">The performance of the three types of cation exchange materials, resin (Poros HS50 column (1.8 mL) and SPSFF column (1.8 mL)), membrane (Mustang S (0.35 mL)), and monolith (SO3 Monolith (0.34 mL)), with varying amount of the product loaded between 0 and 1000 g/L CV or MV was evaluated at pH 5.5 and 5 mS/cm. For the membrane and monolithic chromatography, the flow rate used was 100-400 MV/hr. For the resins, the flow rate used was between 50-200 cm/hr (9-36 CV/hr). There was an over 90% monomer anti-VEGF antibody recovery using all cation exchange materials as shown in <figref idrefs="DRAWINGS">FIG. 8</figref>.</div>
</li> <li> <para-num num="[0327]"> </para-num> <div class="description-line" id="p-0334" num="0327">The ability of the three types of cation exchange materials, resin (Poros HS50 column (18 CV/hr and 1.7 mL) and SPSFF column (18 CV/hr and 1.7 mL)), membrane (Mustang S (171 MV/hr) and Sartobind S (120 MV/hr)), and monolith (SO3 Monolith (176 MV/hr)), with varying amount of the product loaded between 0 and 1000 g/L CV or MV to remove CHOP was valuated at pH 5.5 and 5 mS/cm. As shown in <figref idrefs="DRAWINGS">FIG. 9</figref>, the monolith, SO3 Monolith, was better at removing CHOP than the other two cation exchange materials. The resin (Poros HS50 column and SPSFF column) and the membranes (Mustang S and Sartobind S) cation exchange materials were similar in removing CHOP. See <figref idrefs="DRAWINGS">FIG. 9</figref>.</div>
</li> <li> <para-num num="[0328]"> </para-num> <div class="description-line" id="p-0335" num="0328">The ability of the three types of cation exchange materials, resin (Poros HS50 column (18 CV/hr) and SPSFF column (18 CV/hr)), membrane (Mustang S (171 CV/hr)), and monolith (SO3 Monolith (176 CV/hr)), with varying amount of the product loaded between 0 and 1000 g/L CV or MV to remove DNA was evaluated at pH 5.5 and 5 mS/cm. The resin cation exchange material Poros HS50 was most effective in removing DNA. See <figref idrefs="DRAWINGS">FIG. 10</figref>.</div>
</li> <li> <para-num num="[0329]"> </para-num> <div class="description-line" id="p-0336" num="0329">The ability of the three types of cation exchange materials, resin (Poros HS50 column (18 CV/hr), Poros HS50 column (18 CV/hr) with 1:2 diluted monomer antibody (“Mab”), and SPSFF column (18 CV/hr)), membrane (Mustang S (176 MV/hr) and Sartobind S (120 MV/hr)), and monolith (SO3 Monolith (171 MV/hr)), with varying amount of the product loaded between 0 and 1000 g/L CV or MV to remove HMW was valuated at pH 5.5 and 5 mS/cm. As shown in <figref idrefs="DRAWINGS">FIG. 11</figref>, the resin cation exchange material, Poros HS50, was most effective in HMW removal followed by SPSFF. Further, Mustang S was better than SO3 monolith in removing HMW. See <figref idrefs="DRAWINGS">FIG. 11</figref>.</div>
</li> <li> <para-num num="[0330]"> </para-num> <div class="description-line" id="p-0337" num="0330">The ability of the cation exchange materials, resin (Poros HS50 column(18 CV/hr), Poros HS50 column (18 CV/hr) with diluted Mab, and SPSFF column (18 CV/hr)), membrane (Mustang S (176 MV/hr)), and monolith (SO3 Monolith (171 MV/hr)) with varying amount of the product loaded between 0 and 1000 g/L CV or MV to remove HMW1 and HMW2 was also evaluated at pH 5.5 and 5 mS/cm. As shown in <figref idrefs="DRAWINGS">FIGS. 12A-12B</figref>, the HMW2 removal was generally more effective than the HMW1 removal. See <figref idrefs="DRAWINGS">FIGS. 12A-12B</figref>.</div>
</li> <li> <para-num num="[0331]"> </para-num> <div class="description-line" id="p-0338" num="0331">The dynamic binding capacity (“DBC”) of the three types of cation exchange materials, resin (SPSFF column (1.88 mL) and Poros HS50 column (1.85 mL)), membrane (Mustang S coin (0.35 mL) and Mustang S (Acrodisc 0.18 mL)), and monolith (SO3 Monolith disk (0.34 mL-1 disk) and SO3 Monolith disk (0.68 mL-2 disks)) at pH 5.5 and 5 mS/cm using multiple flow rates was evaluated. As described and shown in Table 3, the cation exchange resin had better Mab binding capacities than the membrane and monolith. In general, the Mab binding capacity of the cation exchange materials correlated with their ability of removing HMW. See <figref idrefs="DRAWINGS">FIG. 11</figref> and Table 3. Poros HS50 showed better mass transport than SPSFF. See Table 3. Poros HS50 showed a higher DBC than Mustang S and Monolith SO3. See Table 3.</div>
</li> <li> <div class="description-line" id="p-0339" num="0000">
<tables id="TABLE-US-00003" num="00003">
<patent-tables colsep="0" frame="none" rowsep="0">
<table align="left" class="description-table" cols="5" colsep="0" rowsep="0" width="100%">
<thead>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td" colspan="5" nameend="4" namest="offset" rowsep="1">TABLE 3</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td align="center" class="description-td" colspan="5" nameend="4" namest="offset" rowsep="1"> </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">DBC at</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">Flow rate</td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">5% BT</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td align="center" class="description-td" colspan="5" nameend="4" namest="offset" rowsep="1"> </td>
</tr>
</thead>
<tbody><tr class="description-tr">
<td class="description-td"> </td>
</tr>
</tbody></table>
<table align="left" class="description-table" cols="1" colsep="0" rowsep="0" width="100%">
<tbody><tr class="description-tr">
<td class="description-td">SPSFF (0.66 cm i.d. × 5.5 cm = 1.88 mL)</td>
</tr>
</tbody></table>
<table align="left" class="description-table" cols="5" colsep="0" rowsep="0" width="100%">
<tbody><tr class="description-tr">
<td class="description-td">9.1</td>
<td class="description-td">CV/hr</td>
<td class="description-td">50</td>
<td class="description-td">cm/hr</td>
<td class="description-td">67.2</td>
</tr>
<tr class="description-tr">
<td class="description-td">18.2</td>
<td class="description-td">CV/hr</td>
<td class="description-td">100</td>
<td class="description-td">cm/hr</td>
<td class="description-td">52.6</td>
</tr>
<tr class="description-tr">
<td class="description-td">36.4</td>
<td class="description-td">CV/hr</td>
<td class="description-td">200</td>
<td class="description-td">cm/hr</td>
<td class="description-td">41.0</td>
</tr>
<tr class="description-tr">
<td class="description-td">54.5</td>
<td class="description-td">CV/hr</td>
<td class="description-td">300</td>
<td class="description-td">cm/hr</td>
<td class="description-td">30.7</td>
</tr>
</tbody></table>
<table align="left" class="description-table" cols="1" colsep="0" rowsep="0" width="100%">
<tbody><tr class="description-tr">
<td class="description-td">Poros 50HS (0.66 cm i.d. × 5.4 cm = 1.85 mL)</td>
</tr>
</tbody></table>
<table align="left" class="description-table" cols="5" colsep="0" rowsep="0" width="100%">
<tbody><tr class="description-tr">
<td class="description-td">9.3</td>
<td class="description-td">CV/hr</td>
<td class="description-td">50</td>
<td class="description-td">cm/hr</td>
<td class="description-td">55.7</td>
</tr>
<tr class="description-tr">
<td class="description-td">18.5</td>
<td class="description-td">CV/hr</td>
<td class="description-td">100</td>
<td class="description-td">cm/hr</td>
<td class="description-td">51.4</td>
</tr>
<tr class="description-tr">
<td class="description-td">37</td>
<td class="description-td">CV/hr</td>
<td class="description-td">200</td>
<td class="description-td">cm/hr</td>
<td class="description-td">47.4</td>
</tr>
<tr class="description-tr">
<td class="description-td">55.6</td>
<td class="description-td">CV/hr</td>
<td class="description-td">300</td>
<td class="description-td">cm/hr</td>
<td class="description-td">44.3</td>
</tr>
</tbody></table>
<table align="left" class="description-table" cols="1" colsep="0" rowsep="0" width="100%">
<tbody><tr class="description-tr">
<td class="description-td">Mustang S coin 0.35 mL 21.9</td>
</tr>
</tbody></table>
<table align="left" class="description-table" cols="5" colsep="0" rowsep="0" width="100%">
<tbody><tr class="description-tr">
<td class="description-td">171.4</td>
<td class="description-td">CV/hr</td>
<td class="description-td">1</td>
<td class="description-td">mL/min</td>
<td class="description-td"> </td>
</tr>
</tbody></table>
<table align="left" class="description-table" cols="1" colsep="0" rowsep="0" width="100%">
<tbody><tr class="description-tr">
<td class="description-td">Mustang S (Acrodisc) 0.18 mL 20.6</td>
</tr>
</tbody></table>
<table align="left" class="description-table" cols="5" colsep="0" rowsep="0" width="100%">
<tbody><tr class="description-tr">
<td class="description-td">171.4</td>
<td class="description-td">CV/hr</td>
<td class="description-td">1</td>
<td class="description-td">mL/min</td>
<td class="description-td"> </td>
</tr>
</tbody></table>
<table align="left" class="description-table" cols="1" colsep="0" rowsep="0" width="100%">
<tbody><tr class="description-tr">
<td class="description-td">Monolith SO3 disk (0.34 mL - 1 disk) 17.5</td>
</tr>
</tbody></table>
<table align="left" class="description-table" cols="5" colsep="0" rowsep="0" width="100%">
<tbody><tr class="description-tr">
<td class="description-td">88.2</td>
<td class="description-td">CV/hr</td>
<td class="description-td">0.5</td>
<td class="description-td">mL/min</td>
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">176.5</td>
<td class="description-td">CV/hr</td>
<td class="description-td">1</td>
<td class="description-td">mL/min</td>
<td class="description-td">17.8</td>
</tr>
</tbody></table>
<table align="left" class="description-table" cols="1" colsep="0" rowsep="0" width="100%">
<tbody><tr class="description-tr">
<td class="description-td">Monolith SO3 disk (0.68 mL - 2 disk) 17.6</td>
</tr>
</tbody></table>
<table align="left" class="description-table" cols="5" colsep="0" rowsep="0" width="100%">
<tbody><tr class="description-tr">
<td class="description-td">88.2</td>
<td class="description-td">CV/hr</td>
<td class="description-td">1</td>
<td class="description-td">mL/min</td>
</tr>
<tr class="description-tr">
<td align="center" class="description-td" colspan="5" nameend="5" namest="1" rowsep="1"> </td>
</tr>
</tbody></table>
</patent-tables>
</tables>
</div>
</li> <li> <para-num num="[0332]"> </para-num> <div class="description-line" id="p-0340" num="0332">Purification of Anti-CD20 Antibody</div>
</li> <li> <para-num num="[0333]"> </para-num> <div class="description-line" id="p-0341" num="0333">The ability of various types of cation exchange resin (Poros HS50, SE HiCap, SPSFF, SPXL and Capto S) to remove HMW was evaluated using high throughput 96-well plates and batch binding mode with the product loaded at 70-90 mg/mL resin under various pHs and salt concentrations for the purification of an anti-CD20 antibody. The HMW removal was evaluated by the % HMW bound on the resin. As shown in <figref idrefs="DRAWINGS">FIGS. 13A-13E</figref>, Poros HS50 was most effective in removing HMW, followed by SE HiCap. Further, SPSFF was better than SPXL in removing HMW. See <figref idrefs="DRAWINGS">FIGS. 13A-13E</figref>. Moreover, SPXL was similar to Capto S in removing HMW. See <figref idrefs="DRAWINGS">FIGS. 13A-13E</figref>.</div>
</li> <li> <para-num num="[0334]"> </para-num> <div class="description-line" id="p-0342" num="0334">The ability of various types of resin (Poros HS50, SE HiCap, SPSFF, SPXL and Capto S) to remove CHOP was evaluated using high throughput 96-well plates with the product loaded at 70-90 mg/mL resin under various pHs and salt concentrations. The CHOP removal was evaluated by the % CHOP bound on the resin. As shown in <figref idrefs="DRAWINGS">FIGS. 14A-14E</figref>, Poros HS50 was most effective in removing CHOP, followed by SE HiCap. Further, SPSFF was better than SPXL in removing CHOP. See <figref idrefs="DRAWINGS">FIGS. 14A-14E</figref>. Moreover, SPXL was better than Capto S in removing CHOP. See <figref idrefs="DRAWINGS">FIGS. 14A-14E</figref>.</div>
</li> <li> <para-num num="[0335]"> </para-num> <div class="description-line" id="p-0343" num="0335">The ability of Poros HS50 and Capto S resins to remove HMW was further examined using column chromatography with the varying amount of the product loaded at 5 mS/cm and different pHs (Poros HS50 at pH 5.5 and pH 6 and Capto S at pH 5 and pH 5.5). As shown in <figref idrefs="DRAWINGS">FIG. 15</figref>, Poros HS50 was better than Capto S in binding HMWs. The accumulated HMW % was lower in the collected product pools from the runs on Poros HS50 resin than in the collected product pools from the runs on Capto S resin. See <figref idrefs="DRAWINGS">FIG. 16</figref>.</div>
</li> <li> <para-num num="[0336]"> </para-num> <div class="description-line" id="p-0344" num="0336">The ability of Poros HS50 and Capto S resins to remove CHOP was also examined using column chromatography with the varying amount of the product loaded at 5 mS/cm and different pHs (Poros HS50 at pH 5.5 and pH 6 and Capto S at pH 5 and pH 5.5). As shown in <figref idrefs="DRAWINGS">FIG. 17</figref>, Poros HS50 was better than Capto S in binding CHOP. The accumulated CHOP was lower in the collected product pools from the runs on Poros HS50 resin than in the collected product pools from the runs on Capto S resin. See <figref idrefs="DRAWINGS">FIG. 18</figref>.</div>
</li> <heading id="h-0032">Example 2—Overloaded Cation Exchange Chromatography Using Various Purification Conditions</heading>
<li> <para-num num="[0337]"> </para-num> <div class="description-line" id="p-0345" num="0337">This example describes an overloaded cation exchange chromatography process for purifying antibodies.</div>
</li> <heading id="h-0033">Materials and Methods</heading>
<li> <para-num num="[0338]"> </para-num> <div class="description-line" id="p-0346" num="0338">Purification Methods</div>
</li> <li> <para-num num="[0339]"> </para-num> <div class="description-line" id="p-0347" num="0339">The filtered and equilibrated protein pools were prepared as described in Example 1 and used as the product for the cation exchange chromatography.</div>
</li> <li> <para-num num="[0340]"> </para-num> <div class="description-line" id="p-0348" num="0340">The columns were packed with Poros HS50 or SPSFF or Capto S resin, respectively. The column dimensions were 0.66 cm i.d. by 5 to 15 cm bed height. The flow rates used were 50-200 cm per hour (9-36 column volume (CV) per hour). Chromatography was monitored at 280 nm and performed at room temperature.</div>
</li> <li> <para-num num="[0341]"> </para-num> <div class="description-line" id="p-0349" num="0341">To avoid exclusion of antibody from chromatography resin, for example, the optimized loading conductivity is 4-6 mS/cm at pH 5-6, under which the maximum dynamic binding capacity can be obtained. The conductivity of 5-6 mS/cm and pH of 5.0-5.5 was used at the loading condition on the cation exchange columns in the first group study.</div>
</li> <li> <para-num num="[0342]"> </para-num> <div class="description-line" id="p-0350" num="0342">The product as determined by 280 nm was loaded up to 1000 g/L of column volume during each run. A wash step using the matrix equilibration buffer was applied immediately following the loading step until the UV trace returned to the baseline. An elution step with a higher salt concentration solution was used to strip out the bound species from the matrix during each chromatographic run. The bound species contained unwanted HMW species, DNA, CHOP, and other impurities. Columns were cleaned with 0.5N NaOH solution after each run and stored in 0.1N NaOH solution.</div>
</li> <li> <para-num num="[0343]"> </para-num> <div class="description-line" id="p-0351" num="0343">During the loading and washing steps, the flow through from each run was fractionated and collected in appropriate size comical tubes. The protein concentration in each fraction was measured using UV-Vis spectrophotometer. The impurity contents such as host cell protein, DNA, leached protein A, and gentamicin were determined in selected fractions to cover the entire loading and washing steps as discussed above.</div>
</li> <heading id="h-0034">Experimental Procedures and Results</heading>
<li> <para-num num="[0344]"> </para-num> <div class="description-line" id="p-0352" num="0344">Purification of Anti-VEGF Antibody using SPSFF</div>
</li> <li> <para-num num="[0345]"> </para-num> <div class="description-line" id="p-0353" num="0345">The chromatograms of SPSFF (0.66 cm id×5.5 cm bed height) with the product loading amount of up to 1000 mg/mL CV were run under various flow rates (9 CV/hr, 8 CV/hr, and 36 CV/hr) at pH 5.5, 5 mS/cm in NaAC. See <figref idrefs="DRAWINGS">FIGS. 19A-19B</figref>.</div>
</li> <li> <para-num num="[0346]"> </para-num> <div class="description-line" id="p-0354" num="0346">The effects of various flow rates (9 CV/hr, 18 CV/hr, and 36 CV/hr) on CHOP, DNA and HMW removals for SPSFF with varying amount of the product loaded up to 1000 g/L CV were evaluated at pH 5.5 and 5 mS/cm. As shown in <figref idrefs="DRAWINGS">FIG. 20</figref>, the CHOP removal was not significantly affected by the flow rates tested. However, as shown in <figref idrefs="DRAWINGS">FIGS. 21 and 22</figref>, the DNA and HMW removals were affected by the flow rates tested.</div>
</li> <li> <para-num num="[0347]"> </para-num> <div class="description-line" id="p-0355" num="0347">Purification of Anti-VEGF Antibody using Poros HS50</div>
</li> <li> <para-num num="[0348]"> </para-num> <div class="description-line" id="p-0356" num="0348">The effects of various flow rates (18 CV/hr and 36 CV/hr) on CHOP, DNA and HMW removals for Poros HS50 with varying amount of the product loaded up to 1000 g/L CV were evaluated at pH 5.5 and 5 mS/cm. As shown in <figref idrefs="DRAWINGS">FIG. 23</figref>, the HMW, CHOP and DNA removals were not significantly affected by the flow rates tested.</div>
</li> <li> <para-num num="[0349]"> </para-num> <div class="description-line" id="p-0357" num="0349">The Mab binding ability of Poros HS50 was evaluated at various loading conductivities (3 mS/cm, 5 mS/cm, 8 mS/cm, 13 mS/cm, and 18 mS/cm) at pH 5.5 and 18 CV/hr using varying amount of the product loaded up to 1000 mg/mL CV. As shown in <figref idrefs="DRAWINGS">FIG. 24</figref>, a higher loading conductivity correlated with a lower Mab binding capacity.</div>
</li> <li> <para-num num="[0350]"> </para-num> <div class="description-line" id="p-0358" num="0350">The chromatograms of Poros HS50 were run under various loading conductivities (5 mS/cm, 8 mS/cm, 13 mS/cm, and 18 mS/cm) at pH 5.5. P1 shows the elution peak with 350 mM NaAC and P2 shows the cleaning peak with 0.5 N NaOH. See <figref idrefs="DRAWINGS">FIGS. 25A-25B</figref>. As shown in <figref idrefs="DRAWINGS">FIGS. 25A-25B</figref>, there was typical binding to Poros HS50 under the loading conductivities of 5 mS/cm and 8 mS/cm. Further, there was partial binding at 13 mS/cm. See <figref idrefs="DRAWINGS">FIGS. 25A-25B</figref>. At 18 mS/cm, there was typical flow through. See <figref idrefs="DRAWINGS">FIGS. 25A-25B</figref>.</div>
</li> <li> <para-num num="[0351]"> </para-num> <div class="description-line" id="p-0359" num="0351">The ability of Poros HS50 in CHOP removal was examined at pH 5.5 under various conductivity conditions (3 mS/cm, 5 mS/cm, 5 mS/cm and loaded with ˜2 fold diluted CHOP, 8 mS/cm, 13 mS/cm, and 18 mS/cm) with varying amount of the product loaded up to 1000 mg/mL CV at 18 CV/hr. As shown in <figref idrefs="DRAWINGS">FIG. 26</figref>, the CHOP removal was more effective under lower loading conductivity.</div>
</li> <li> <para-num num="[0352]"> </para-num> <div class="description-line" id="p-0360" num="0352">The ability of Poros HS50 in HMW removal was evaluated at pH 5.5 under various loading conductivities (3 mS/cm, 5 mS/cm, 5 mS/cm and ˜2 fold diluted Mab at 2.37 mg/ml, 8 mS/cm, 13 mS/cm, and 18 mS/cm) with varying amount of the product loaded up to 1000 g/L CV and 18 CV/hr. As shown in <figref idrefs="DRAWINGS">FIG. 27</figref>, at 3 mS/cm, there was a lower amount of HMW bound to Poros HS50 at the beginning of the process. At 13 mS/cm, a higher amount of HMW was bound to Poros HS50 at the beginning of the process and the amount of HMW bound declined over time. See <figref idrefs="DRAWINGS">FIG. 27</figref>.</div>
</li> <li> <para-num num="[0353]"> </para-num> <div class="description-line" id="p-0361" num="0353">The ability of Poros HS50 in DNA removal with varying amount of the product loaded up to 1000 mg/mL CV was evaluated under various loading conductivities and flow rates (3 mS/cm and 18 CV/hr, 5 mS/cm and 18 CV/hr, 5 mS/cm and 36 CV/hr, 8 mS/cm and 18 CV/hr, 13 mS/cm and 18 CV/hr, and 18 mS/cm and 18 CV/hr) at pH 5.5. The amounts of DNA in the load were about 1000 pg/mg, 1000 pg/mg, 7000 pg/mg, 16821 pg/mg, and 18566 pg/mg for the loading conductivity conditions of 3 mS/cm, 5 mS/cm, 8 mS/cm, 13 mS/cm, and 18 mS/cm, respectively. As shown in <figref idrefs="DRAWINGS">FIGS. 28A and 28B</figref>, Poros HS50 was able to bind DNA under the loading conductivities tested.</div>
</li> <li> <para-num num="[0354]"> </para-num> <div class="description-line" id="p-0362" num="0354">The ability of Poros HS50 to bind DNA with the product loaded at 300 mg/mL CV was also evaluated at pH 5.5 with the gradient elution. As shown in <figref idrefs="DRAWINGS">FIG. 29</figref>, DNA was bound to the column at pH 5.5 and 5 mS/cm and eluted off the column during the linear salt gradient elution.</div>
</li> <li> <para-num num="[0355]"> </para-num> <div class="description-line" id="p-0363" num="0355">The ability of Poros HS50 (1.88 mL resin) to remove gentamicin was evaluated at pH 5.5, 5 mS/cm, and 18 CV/hr with varying amount of the product loaded up to about 990 g/L CV. As shown in Table 4, Poros HS50 was able to remove gentamicin in the overloaded process.</div>
</li> <li> <div class="description-line" id="p-0364" num="0000">
<tables id="TABLE-US-00004" num="00004">
<patent-tables colsep="0" frame="none" rowsep="0">
<table align="left" class="description-table" cols="5" colsep="0" rowsep="0" width="100%">
<thead>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td" colspan="5" nameend="4" namest="offset" rowsep="1">TABLE 4</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td align="center" class="description-td" colspan="5" nameend="4" namest="offset" rowsep="1"> </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">Sample</td>
<td class="description-td">Product Collected</td>
<td class="description-td">Gentamicin</td>
<td class="description-td">Gentamicin</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">ID</td>
<td class="description-td">(g/L)</td>
<td class="description-td">(pg/mL)</td>
<td class="description-td">(pg/mg)</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td align="center" class="description-td" colspan="5" nameend="4" namest="offset" rowsep="1"> </td>
</tr>
</thead>
<tbody><tr class="description-tr">
<td class="description-td"> </td>
</tr>
</tbody></table>
<table align="left" class="description-table" cols="5" colsep="0" rowsep="0" width="100%">
<tbody><tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">Load</td>
<td class="description-td">—</td>
<td class="description-td">32.875</td>
<td class="description-td">7.15</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">F5</td>
<td class="description-td"> 65</td>
<td class="description-td">&lt;0.37</td>
<td class="description-td">&lt;0.08</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">F15</td>
<td class="description-td">331</td>
<td class="description-td">&lt;0.37</td>
<td class="description-td">&lt;0.08</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">F30</td>
<td class="description-td">727</td>
<td class="description-td">&lt;0.37</td>
<td class="description-td">&lt;0.08</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">F35</td>
<td class="description-td">860</td>
<td class="description-td">&lt;0.37</td>
<td class="description-td">&lt;0.08</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">Eluate</td>
<td class="description-td">—</td>
<td class="description-td">321.25</td>
<td class="description-td">20</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td align="center" class="description-td" colspan="5" nameend="4" namest="offset" rowsep="1"> </td>
</tr>
</tbody></table>
</patent-tables>
</tables>
</div>
</li> <li> <para-num num="[0356]"> </para-num> <div class="description-line" id="p-0365" num="0356">The ability of Poros HS50 (1.88 mL resin) to remove Protein A was evaluated at pH 5.5, 5 mS/cm, and 18 CV/hr with varying amount of the product loaded up to about 990 g/L CV. As shown in Table 5, Poros HS50 was able to remove Protein A leachate in the overloaded process. The Protein A broke through from the column between 727 and 860 g collected/L CV. See Table 5.</div>
</li> <li> <div class="description-line" id="p-0366" num="0000">
<tables id="TABLE-US-00005" num="00005">
<patent-tables colsep="0" frame="none" rowsep="0">
<table align="left" class="description-table" cols="5" colsep="0" rowsep="0" width="100%">
<thead>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td" colspan="5" nameend="4" namest="offset" rowsep="1">TABLE 5</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td align="center" class="description-td" colspan="5" nameend="4" namest="offset" rowsep="1"> </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">Product Collected</td>
<td class="description-td">Protein A</td>
<td class="description-td">Protein A</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">Sample ID</td>
<td class="description-td">(g/L)</td>
<td class="description-td">(ng/mL)</td>
<td class="description-td">(ng/mg)</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td align="center" class="description-td" colspan="5" nameend="4" namest="offset" rowsep="1"> </td>
</tr>
</thead>
<tbody><tr class="description-tr">
<td class="description-td"> </td>
</tr>
</tbody></table>
<table align="left" class="description-table" cols="5" colsep="0" rowsep="0" width="100%">
<tbody><tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">Load</td>
<td class="description-td">—</td>
<td class="description-td">22.17</td>
<td class="description-td">4.82</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">F5</td>
<td class="description-td"> 65</td>
<td class="description-td">&lt;9.8</td>
<td class="description-td">&lt;2.18</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">F15</td>
<td class="description-td">331</td>
<td class="description-td">&lt;9.8</td>
<td class="description-td">&lt;2.17</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">F22</td>
<td class="description-td">517</td>
<td class="description-td">&lt;9.8</td>
<td class="description-td">&lt;2.18</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">F30</td>
<td class="description-td">727</td>
<td class="description-td">&lt;9.8</td>
<td class="description-td">&lt;2.20</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">F35</td>
<td class="description-td">860</td>
<td class="description-td">13.53</td>
<td class="description-td">2.99</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">Eluate</td>
<td class="description-td">—</td>
<td class="description-td">1324.35</td>
<td class="description-td">82.41</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td align="center" class="description-td" colspan="5" nameend="4" namest="offset" rowsep="1"> </td>
</tr>
</tbody></table>
</patent-tables>
</tables>
</div>
</li> <li> <para-num num="[0357]"> </para-num> <div class="description-line" id="p-0367" num="0357">The ability of Poros HS50 to remove HMW, CHOP and DNA under different column bed heights (4.6 cm bed height and 14.2 cm bed height) was evaluated at pH 5.5, 5 mS/cm, and 18 CV/hr with varying amount of the product loaded up to 1000 mg/mL CV. As shown in <figref idrefs="DRAWINGS">FIG. 30</figref>, the column bed heights tested (4.6 cm and 14.2 cm) had no significant effects on HMW, CHOP, or DNA removal. Further, the column bed heights of 4.6 cm and 14.2 cm kept the same residence time (3 minutes).</div>
</li> <li> <para-num num="[0358]"> </para-num> <div class="description-line" id="p-0368" num="0358">The distribution of charged variants in the elution fractions from the Poros HS50 column (1.88 mL) was evaluated at pH 5.5, 5 mS/cm, and 18 CV/hr, with varying amount of the product loaded up to 1000 g/L CV. The basic variants were treated with CPB (Carboxypeptidase B) prior to the performing the charged variant analysis. As shown in Table 6, the distribution of charged variants (the acidic and basic variants) and main variants in the final pool was not substantially changed. There was a higher percentage of acidic variant in the flow through at the low loading amount. See Table 6. Poros HS50 was able to bind to some basic variant. See Table 6.</div>
</li> <li> <div class="description-line" id="p-0369" num="0000">
<tables id="TABLE-US-00006" num="00006">
<patent-tables colsep="0" frame="none" rowsep="0">
<table align="left" class="description-table" cols="4" colsep="0" rowsep="0" width="100%">
<thead>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td" colspan="4" nameend="3" namest="offset" rowsep="1">TABLE 6</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td align="center" class="description-td" colspan="4" nameend="3" namest="offset" rowsep="1"> </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">Acidic (%)</td>
<td class="description-td">Main (%)</td>
<td class="description-td">Basic (%)</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td align="center" class="description-td" colspan="4" nameend="3" namest="offset" rowsep="1"> </td>
</tr>
</thead>
<tbody><tr class="description-tr">
<td class="description-td"> </td>
</tr>
</tbody></table>
<table align="left" class="description-table" cols="4" colsep="0" rowsep="0" width="100%">
<tbody><tr class="description-tr">
<td class="description-td">Load</td>
<td class="description-td">28.54</td>
<td class="description-td">60.91</td>
<td class="description-td">10.54</td>
</tr>
<tr class="description-tr">
<td class="description-td">Collected Mab per CV in</td>
</tr>
<tr class="description-tr">
<td class="description-td">fraction (g/L cv)</td>
</tr>
<tr class="description-tr">
<td class="description-td"> 65</td>
<td class="description-td">32.87</td>
<td class="description-td">62.76</td>
<td class="description-td">4.37</td>
</tr>
<tr class="description-tr">
<td class="description-td">331</td>
<td class="description-td">29.78</td>
<td class="description-td">59.21</td>
<td class="description-td">11.01</td>
</tr>
<tr class="description-tr">
<td class="description-td">727</td>
<td class="description-td">29.53</td>
<td class="description-td">58.05</td>
<td class="description-td">12.42</td>
</tr>
<tr class="description-tr">
<td class="description-td">833</td>
<td class="description-td">29.62</td>
<td class="description-td">57.6</td>
<td class="description-td">12.78</td>
</tr>
<tr class="description-tr">
<td class="description-td">Elute Pool (60.8 mg/mL CV)</td>
<td class="description-td">8.36</td>
<td class="description-td">32.42</td>
<td class="description-td">59.22</td>
</tr>
<tr class="description-tr">
<td align="center" class="description-td" colspan="4" nameend="4" namest="1" rowsep="1"> </td>
</tr>
</tbody></table>
</patent-tables>
</tables>
</div>
</li> <heading id="h-0035">Example 3—Purification of Polypeptides Using Mixed Mode Chromatography</heading>
<li> <para-num num="[0359]"> </para-num> <div class="description-line" id="p-0370" num="0359">This example describes a mixed mode chromatography process for purifying anti-VEGF antibody, anti-CD11a antibody, and anti-CD20 antibody.</div>
</li> <heading id="h-0036">Materials and Methods</heading>
<li> <para-num num="[0360]"> </para-num> <div class="description-line" id="p-0371" num="0360">Purification Methods</div>
</li> <li> <para-num num="[0361]"> </para-num> <div class="description-line" id="p-0372" num="0361">The filtered and equilibrated protein pools were prepared as described in Example 1 for subsequent mixed mode chromatography. Pools were adjusted to the pH of 5 to 8.5 with 1.5 M Tris base or 2 N glacial acetic acid. Chloride concentrations were adjusted to 0 mM to 250 mM with the addition of 3M NaCl. Specific mixed mode chromatography conditions were performed as indicated in the Experimental Procedures and Results section below.</div>
</li> <heading id="h-0037">Experimental Procedures and Results</heading>
<li> <para-num num="[0362]"> </para-num> <div class="description-line" id="p-0373" num="0362">The performance of the mixed-mode resin, e.g., Capto Adhere, was evaluated using multiple antibodies, anti-VEGF antibody (70 mg/ml resin), anti-CD11a antibody (90 mg/ml resin), and anti-CD20 antibody (90 mg/ml resin) under various pHs and salt conditions in a high throughput screening experiment. The amount of CHOP (ppm) and % HMW in the filtered and equilibrated protein A purified pools was 497 and 7.6 (for anti-VEGF antibody), 820 and 6.4 (for anti-CD11a antibody), and 4720 and 3.5 (for anti-CD20 antibody). <figref idrefs="DRAWINGS">FIGS. 31A-31F, 32A-32F, and 33A-33F</figref> show the results for the Mab recovery, the binding of HMW to the resin, and the binding of CHOP to the resin, respectively, under various pHs and salt conditions. As shown in <figref idrefs="DRAWINGS">FIGS. 31A-31F</figref>, increasing the pH and conductivity increased the binding of Mab to Capto Adhere resin for anti-VEGF antibody and anti-CD20 antibody. The pH had significant effects on the ability of Capto Adhere to bind anti-CD11a antibody. See <figref idrefs="DRAWINGS">FIGS. 31A-31F</figref>. Increasing the pH increased the percentage of HMW species bound to the Cato Adhere resin for anti-VEGF antibody, anti-CD11a antibody, and anti-CD20 antibody. See <figref idrefs="DRAWINGS">FIGS. 32A-32F</figref>. The pH had more significant effects than the conductivity on the ability of Capto Adhere to bind HMWs. See <figref idrefs="DRAWINGS">FIGS. 32A-32F</figref>. Capto Adhere resin was able to bind CHOP under the broader pH and conductivity ranges. As shown in <figref idrefs="DRAWINGS">FIGS. 33A-33F</figref>, more than 80% of CHOP was bound to the resin for the majority of the contour plots for anti-VEGF antibody, anti-CD11a antibody, and anti-CD20 antibody.</div>
</li> <li> <para-num num="[0363]"> </para-num> <div class="description-line" id="p-0374" num="0363">The performance of Capto Adhere in the high throughput screening experiment and in the column chromatography for the purification of anti-VEGF antibody is summarized in Table 7. In general, the observations made in the high throughput experiment correlated with the results obtained from the column chromatography experiment.</div>
</li> <li> <div class="description-line" id="p-0375" num="0000">
<tables id="TABLE-US-00007" num="00007">
<patent-tables colsep="0" frame="none" pgwide="1" rowsep="0">
<table align="left" class="description-table" cols="6" colsep="0" rowsep="0" width="100%">
<thead>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td" colspan="6" nameend="5" namest="offset" rowsep="1">TABLE 7</td>
</tr>
</thead>
<tbody><tr class="description-tr">
<td class="description-td"> </td>
<td align="center" class="description-td" colspan="6" nameend="5" namest="offset" rowsep="1"> </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">% HMW</td>
<td class="description-td">CHOP (ppm)</td>
<td class="description-td">% CHOP</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">% Mab in FT</td>
<td class="description-td">% HMW in FT</td>
<td class="description-td">Bound</td>
<td class="description-td">in FT</td>
<td class="description-td">Bound</td>
</tr>
</tbody></table>
<table align="left" class="description-table" cols="12" colsep="0" rowsep="0" width="100%">
<tbody><tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">Well</td>
<td class="description-td"> </td>
<td class="description-td">Well</td>
<td class="description-td"> </td>
<td class="description-td">Well</td>
<td class="description-td"> </td>
<td class="description-td">Well</td>
<td class="description-td"> </td>
<td class="description-td">Well</td>
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">pH</td>
<td class="description-td">Condo</td>
<td class="description-td">Plate</td>
<td class="description-td">Column</td>
<td class="description-td">Plate</td>
<td class="description-td">Column</td>
<td class="description-td">Plate</td>
<td class="description-td">Column</td>
<td class="description-td">Plate</td>
<td class="description-td">Column</td>
<td class="description-td">Plate</td>
<td class="description-td">Column</td>
</tr>
<tr class="description-tr">
<td align="center" class="description-td" colspan="12" nameend="12" namest="1" rowsep="1"> </td>
</tr>
</tbody></table>
<table align="left" class="description-table" cols="12" colsep="0" rowsep="0" width="100%">
<tbody><tr class="description-tr">
<td class="description-td">5.0</td>
<td class="description-td">5.0</td>
<td class="description-td">93</td>
<td class="description-td">100</td>
<td class="description-td">5.9</td>
<td class="description-td">5.1</td>
<td class="description-td">−13.8</td>
<td class="description-td">7.8</td>
<td class="description-td">23.0</td>
<td class="description-td">36.0</td>
<td class="description-td">93.4</td>
<td class="description-td">93.0</td>
</tr>
<tr class="description-tr">
<td class="description-td">7.0</td>
<td class="description-td">5.0</td>
<td class="description-td">67</td>
<td class="description-td">76</td>
<td class="description-td">4.4</td>
<td class="description-td">4.6</td>
<td class="description-td">35.4</td>
<td class="description-td">37.4</td>
<td class="description-td">10.4</td>
<td class="description-td">12.8</td>
<td class="description-td">97.4</td>
<td class="description-td">96.9</td>
</tr>
<tr class="description-tr">
<td class="description-td">8.0</td>
<td class="description-td">5.0</td>
<td class="description-td">27</td>
<td class="description-td">28</td>
<td class="description-td">1.2</td>
<td class="description-td">1.7</td>
<td class="description-td">83.3</td>
<td class="description-td">77.9</td>
<td class="description-td">17.3</td>
<td class="description-td">31.3</td>
<td class="description-td">95.3</td>
<td class="description-td">92.7</td>
</tr>
<tr class="description-tr">
<td class="description-td">8.0</td>
<td class="description-td">19.4</td>
<td class="description-td">23</td>
<td class="description-td">21</td>
<td class="description-td">1.9</td>
<td class="description-td">6.9</td>
<td class="description-td">71.0</td>
<td class="description-td">40.1</td>
<td class="description-td">71.1</td>
<td class="description-td">154.9</td>
<td class="description-td">79.6</td>
<td class="description-td">70.9</td>
</tr>
<tr class="description-tr">
<td class="description-td">8.5</td>
<td class="description-td">5.0</td>
<td class="description-td">13</td>
<td class="description-td">22</td>
<td class="description-td">0.9</td>
<td class="description-td">2.3</td>
<td class="description-td">86.0</td>
<td class="description-td">73.9</td>
<td class="description-td">23.7</td>
<td class="description-td">85.0</td>
<td class="description-td">93.4</td>
<td class="description-td">83.9</td>
</tr>
<tr class="description-tr">
<td align="center" class="description-td" colspan="12" nameend="12" namest="1" rowsep="1"> </td>
</tr>
</tbody></table>
</patent-tables>
</tables>
</div>
</li> <li> <para-num num="[0364]"> </para-num> <div class="description-line" id="p-0376" num="0364">The ability of Capto Adhere to remove CHOP and HMW was also examined in the column chromatography for the purification of anti-CD11a at pH 5.5 and 5 mS/cm. Capto Adhere resin was able to reduce CHOP levels to below 20 ppm when the product was loaded at about 700 mg/mL resin; however, Capto Adhere was not able to reduce % HMW. Data not shown.</div>
</li> <heading id="h-0038">Example 4—Purification of Anti-CD11a Antibody Using a Combination of Overloaded Cation Exchange Chromatography and Standard Anion Exchange Chromatography</heading>
<li> <para-num num="[0365]"> </para-num> <div class="description-line" id="p-0377" num="0365">This example describes a process using a combination of overloaded cation exchange chromatography and standard anion exchange chromatography for purifying a recombinant humanized CD11a antibody.</div>
</li> <heading id="h-0039">Materials and Methods</heading>
<li> <para-num num="[0366]"> </para-num> <div class="description-line" id="p-0378" num="0366">The cell culture fluid containing monoclonal antibody produced in Chinese hamster ovary cells was processed by continuous centrifuge to remove cellular debris, and further clarified through filtration with depth filters and 0.2 um filter. The anti-CD11a antibody was purified by either a) development run overloaded cation exchange: (i) protein A purification (Prosep vA), (ii) cation exchange purification (CEX) (Poros HS50, in overloaded mode), and (ii) anion exchange (AEX) (QSFF), or (b) commercial process (i) protein A purification (Prosep vA), (ii) CEX (SPSFF), and (ii) AEX (QSFF). The running conditions are listed in Table 8.</div>
</li> <li> <div class="description-line" id="p-0379" num="0000">
<tables id="TABLE-US-00008" num="00008">
<patent-tables colsep="0" frame="none" pgwide="1" rowsep="0">
<table align="left" class="description-table" cols="3" colsep="0" rowsep="0" width="100%">
<thead>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td" colspan="3" nameend="2" namest="offset" rowsep="1">TABLE 8</td>
</tr>
</thead>
<tbody><tr class="description-tr">
<td class="description-td"> </td>
<td align="center" class="description-td" colspan="3" nameend="2" namest="offset" rowsep="1"> </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">Commercial Process</td>
<td class="description-td">Development Run Overloaded CEX</td>
</tr>
</tbody></table>
<table align="left" class="description-table" cols="7" colsep="0" rowsep="0" width="100%">
<tbody><tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">Load</td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">Load</td>
</tr>
<tr class="description-tr">
<td class="description-td">Purification</td>
<td class="description-td">Operating</td>
<td class="description-td">Operating</td>
<td class="description-td">Density</td>
<td class="description-td">Operating</td>
<td class="description-td">Operating</td>
<td class="description-td">Density</td>
</tr>
<tr class="description-tr">
<td class="description-td">Step</td>
<td class="description-td">Mode</td>
<td class="description-td">Condition</td>
<td class="description-td">(g/L CV)</td>
<td class="description-td">Mode</td>
<td class="description-td">Condition</td>
<td class="description-td">(2/L CV)</td>
</tr>
<tr class="description-tr">
<td align="center" class="description-td" colspan="7" nameend="7" namest="1" rowsep="1"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">Protein A</td>
<td class="description-td">B/E</td>
<td class="description-td">STD</td>
<td class="description-td"> 8-20</td>
<td class="description-td">B/E</td>
<td class="description-td">STD</td>
<td class="description-td">8-20</td>
</tr>
<tr class="description-tr">
<td class="description-td">CEX</td>
<td class="description-td">B/E</td>
<td class="description-td">pH 5.5,</td>
<td class="description-td">15-40</td>
<td class="description-td">Overloaded</td>
<td class="description-td">pH 5.5,</td>
<td class="description-td">600</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">load-</td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">5 mS/cm</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">4.7 mS/cm,</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">elution-</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">13.25 mS/cm</td>
</tr>
<tr class="description-tr">
<td class="description-td">AEX</td>
<td class="description-td">FT</td>
<td class="description-td">pH 8.0,</td>
<td class="description-td">15-70</td>
<td class="description-td">FT</td>
<td class="description-td">pH 8.0,</td>
<td class="description-td">70</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">6.9 mS/cm</td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">5 mS/cm</td>
</tr>
<tr class="description-tr">
<td align="center" class="description-td" colspan="7" nameend="7" namest="1" rowsep="1"> </td>
</tr>
</tbody></table>
</patent-tables>
</tables>
</div>
</li> <heading id="h-0040">Experimental Procedures and Results</heading>
<li> <para-num num="[0367]"> </para-num> <div class="description-line" id="p-0380" num="0367">The performance of the process combining the overloaded CEX with the AEX was evaluated with respect to the CHOP removal and Mab recovery. The results are summarized in Table 9.</div>
</li> <li> <div class="description-line" id="p-0381" num="0000">
<tables id="TABLE-US-00009" num="00009">
<patent-tables colsep="0" frame="none" rowsep="0">
<table align="left" class="description-table" cols="3" colsep="0" rowsep="0" width="100%">
<thead>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td" colspan="3" nameend="2" namest="offset" rowsep="1">TABLE 9</td>
</tr>
</thead>
<tbody><tr class="description-tr">
<td class="description-td"> </td>
<td align="center" class="description-td" colspan="3" nameend="2" namest="offset" rowsep="1"> </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">Commercial Process</td>
<td class="description-td">Development Run Overloaded CEX</td>
</tr>
</tbody></table>
<table align="left" class="description-table" cols="6" colsep="0" rowsep="0" width="100%">
<tbody><tr class="description-tr">
<td class="description-td">Purification</td>
<td class="description-td">CHOP</td>
<td class="description-td">Monomer</td>
<td class="description-td">CHOP</td>
<td class="description-td">Monomer</td>
<td class="description-td">Yield</td>
</tr>
<tr class="description-tr">
<td class="description-td">Step</td>
<td class="description-td">(ng/mg)</td>
<td class="description-td">(%)</td>
<td class="description-td">(ng/mg)</td>
<td class="description-td">(%)</td>
<td class="description-td">(%)</td>
</tr>
<tr class="description-tr">
<td align="center" class="description-td" colspan="6" nameend="6" namest="1" rowsep="1"> </td>
</tr>
</tbody></table>
<table align="left" class="description-table" cols="6" colsep="0" rowsep="0" width="100%">
<tbody><tr class="description-tr">
<td class="description-td">Protein A</td>
<td class="description-td">677-939</td>
<td class="description-td">94.7-95.9</td>
<td class="description-td">995</td>
<td class="description-td">93.4</td>
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">CEX</td>
<td class="description-td">116-183</td>
<td class="description-td">99.3-99.8</td>
<td class="description-td">158</td>
<td class="description-td">99.7</td>
<td class="description-td">90</td>
</tr>
<tr class="description-tr">
<td class="description-td">AEX</td>
<td class="description-td">&lt;22.2</td>
<td class="description-td">99.7-99.8</td>
<td class="description-td">2</td>
<td class="description-td">99.7</td>
<td class="description-td">95</td>
</tr>
<tr class="description-tr">
<td class="description-td">Bulk</td>
<td class="description-td">&lt;1.48</td>
<td class="description-td">99.6-99.7</td>
</tr>
<tr class="description-tr">
<td align="center" class="description-td" colspan="6" nameend="6" namest="1" rowsep="1"> </td>
</tr>
</tbody></table>
</patent-tables>
</tables>
</div>
</li> <li> <para-num num="[0368]"> </para-num> <div class="description-line" id="p-0382" num="0368">The ability of Poros HS50 and QSFF to remove HMW, CHOP, gentamicin, DNA, and leached Protein A was evaluated under two conditions: (a) Poros HS50 at pH 5.5, 3.5 mS/cm and QSFF at pH 8.0, 3.5 mS/cm; and (b) Poros HS 50 at pH 5.5, 5.0 mS/cm and QSFF at pH 8.0, 5.0 mS/cm. The results for conditions (a) and (b) are summarized in Table 10 and Table 11, respectively. Gentamicin, DNA, and leached Protein A were removed to below detection limits. There was better CHOP removal under condition (a) than under condition (b). There was better HMW removal under condition (b) than under condition (a). The product yield was at least 90% for each chromatography step.</div>
</li> <li> <div class="description-line" id="p-0383" num="0000">
<tables id="TABLE-US-00010" num="00010">
<patent-tables colsep="0" frame="none" pgwide="1" rowsep="0">
<table align="left" class="description-table" cols="6" colsep="0" rowsep="0" width="100%">
<thead>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td" colspan="6" nameend="5" namest="offset" rowsep="1">TABLE 10</td>
</tr>
</thead>
<tbody><tr class="description-tr">
<td class="description-td"> </td>
<td align="center" class="description-td" colspan="6" nameend="5" namest="offset" rowsep="1"> </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">CHOP</td>
<td class="description-td">Genamicin</td>
<td class="description-td"> </td>
<td class="description-td">Leached ProA</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">HMV (%)</td>
<td class="description-td">(ppm)</td>
<td class="description-td">(ng/mg)</td>
<td class="description-td">DNA (ng/mg)</td>
<td class="description-td">(ng/mg)</td>
</tr>
</tbody></table>
<table align="left" class="description-table" cols="11" colsep="0" rowsep="0" width="100%">
<tbody><tr class="description-tr">
<td class="description-td">Step</td>
<td class="description-td">Load</td>
<td class="description-td">Pool</td>
<td class="description-td">Load</td>
<td class="description-td">Pool</td>
<td class="description-td">Load</td>
<td class="description-td">Pool</td>
<td class="description-td">Load</td>
<td class="description-td">Pool</td>
<td class="description-td">Load</td>
<td class="description-td">Pool</td>
</tr>
<tr class="description-tr">
<td align="center" class="description-td" colspan="11" nameend="11" namest="1" rowsep="1"> </td>
</tr>
</tbody></table>
<table align="left" class="description-table" cols="11" colsep="0" rowsep="0" width="100%">
<tbody><tr class="description-tr">
<td class="description-td">POROS</td>
<td class="description-td">6.4</td>
<td class="description-td">1.42</td>
<td class="description-td">1168</td>
<td class="description-td">55</td>
<td class="description-td">9.1</td>
<td class="description-td">&lt;0.1</td>
<td class="description-td">0.3</td>
<td class="description-td">0.3</td>
<td class="description-td">25</td>
<td class="description-td">&lt;2</td>
</tr>
<tr class="description-tr">
<td class="description-td">HS50</td>
</tr>
<tr class="description-tr">
<td class="description-td">QSFF</td>
<td class="description-td">1.42</td>
<td class="description-td">1.41</td>
<td class="description-td">55</td>
<td class="description-td">2</td>
<td class="description-td">&lt;0.1</td>
<td class="description-td">&lt;0.1</td>
<td class="description-td">0.5</td>
<td class="description-td">0</td>
<td class="description-td">&lt;2</td>
<td class="description-td">&lt;2</td>
</tr>
<tr class="description-tr">
<td align="center" class="description-td" colspan="11" nameend="11" namest="1" rowsep="1"> </td>
</tr>
</tbody></table>
</patent-tables>
</tables>
</div>
</li> <li> <div class="description-line" id="p-0384" num="0000">
<tables id="TABLE-US-00011" num="00011">
<patent-tables colsep="0" frame="none" pgwide="1" rowsep="0">
<table align="left" class="description-table" cols="6" colsep="0" rowsep="0" width="100%">
<thead>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td" colspan="6" nameend="5" namest="offset" rowsep="1">TABLE 11</td>
</tr>
</thead>
<tbody><tr class="description-tr">
<td class="description-td"> </td>
<td align="center" class="description-td" colspan="6" nameend="5" namest="offset" rowsep="1"> </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">CHOP</td>
<td class="description-td">Gentamicin</td>
<td class="description-td"> </td>
<td class="description-td">Leached ProA</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">HMV (%)</td>
<td class="description-td">(ppm)</td>
<td class="description-td">(ng/mg)</td>
<td class="description-td">DNA (ng/mg)</td>
<td class="description-td">(ng/mg)</td>
</tr>
</tbody></table>
<table align="left" class="description-table" cols="11" colsep="0" rowsep="0" width="100%">
<tbody><tr class="description-tr">
<td class="description-td">Step</td>
<td class="description-td">Load</td>
<td class="description-td">Pool</td>
<td class="description-td">Load</td>
<td class="description-td">Pool</td>
<td class="description-td">Load</td>
<td class="description-td">Pool</td>
<td class="description-td">Load</td>
<td class="description-td">Pool</td>
<td class="description-td">Load</td>
<td class="description-td">Pool</td>
</tr>
<tr class="description-tr">
<td align="center" class="description-td" colspan="11" nameend="11" namest="1" rowsep="1"> </td>
</tr>
</tbody></table>
<table align="left" class="description-table" cols="11" colsep="0" rowsep="0" width="100%">
<tbody><tr class="description-tr">
<td class="description-td">POROS</td>
<td class="description-td">6.59</td>
<td class="description-td">0.28</td>
<td class="description-td">664</td>
<td class="description-td">133</td>
<td class="description-td">5</td>
<td class="description-td">&lt;0.1</td>
<td class="description-td">0.3</td>
<td class="description-td">&lt;LTR</td>
<td class="description-td">21</td>
<td class="description-td">&lt;2</td>
</tr>
<tr class="description-tr">
<td class="description-td">HS50</td>
</tr>
<tr class="description-tr">
<td class="description-td">QSFF</td>
<td class="description-td">0.28</td>
<td class="description-td">0.3</td>
<td class="description-td">133</td>
<td class="description-td">2</td>
<td class="description-td">&lt;0.1</td>
<td class="description-td">&lt;0.1</td>
<td class="description-td">&lt;LTR</td>
<td class="description-td">&lt;LTR</td>
<td class="description-td">&lt;2</td>
<td class="description-td">&lt;2</td>
</tr>
<tr class="description-tr">
<td align="center" class="description-td" colspan="11" nameend="11" namest="1" rowsep="1"> </td>
</tr>
</tbody></table>
</patent-tables>
</tables>
</div>
</li> <heading id="h-0041">Example 5—Purification Using a Combination of Overloaded Cation Exchange Chromatography and Mixed Mode Chromatography</heading>
<li> <para-num num="[0369]"> </para-num> <div class="description-line" id="p-0385" num="0369">This example describes a process using a combination of overloaded cation exchange chromatography and mixed mode chromatography for purifying an anti-CD11a antibody and anti-CD20 antibody.</div>
</li> <heading id="h-0042">Materials and Methods</heading>
<li> <para-num num="[0370]"> </para-num> <div class="description-line" id="p-0386" num="0370">The filtered and equilibrated protein pools were prepared as described in Example 1 for subsequent cation exchange chromatography and/or mixed mode chromatography. Specific cation exchange chromatography and/or mixed mode chromatography conditions were performed as indicated in the Experimental Procedures and Results section below.</div>
</li> <heading id="h-0043">Experimental Procedures and Results</heading>
<li> <para-num num="[0371]"> </para-num> <div class="description-line" id="p-0387" num="0371">Anti-CD11a Purification</div>
</li> <li> <para-num num="[0372]"> </para-num> <div class="description-line" id="p-0388" num="0372">Experiment 1—</div>
</li> <li> <para-num num="[0373]"> </para-num> <div class="description-line" id="p-0389" num="0373">The filtered and equilibrated product described above was purified using an overloaded Poros HS50 column and a Capto Adhere column with approximately ˜600 g/L CV of product loaded for Poros HS50 and about ˜100 g/L CV for Capto Adhere at pH 5.5, 5 mS/cm, and 100 cm/hr. The experiment was performed in two different sequences: a) Sequence A: (i) Poros HS50 column (0.66 cm×5 cm) and (ii) Capto Adhere (1.5 cm×20 cm) or b) Sequence B (i) Capto Adhere (0.66 cm×20 cm) and (ii) Poros HS50 column (0.66 cm×5 cm). For Sequence A, the loading densities for Poros HS50 column and Capto Adhere column were 600 mg/mL resin and 106 mg/mL resin, respectively. For Sequence B, the loading densities for Capto Adhere column and Poros HS50 column were 100 mg/mL resin and 569 mg/mL resin, respectively.</div>
</li> <li> <para-num num="[0374]"> </para-num> <div class="description-line" id="p-0390" num="0374">The results on impurity removals are summarized in Table 12 for Sequence A and Table 13 Sequence B. The product recovery yields and the results on impurity removals for both of the procedures were met commercial process purity and yield.</div>
</li> <li> <div class="description-line" id="p-0391" num="0000">
<tables id="TABLE-US-00012" num="00012">
<patent-tables colsep="0" frame="none" pgwide="1" rowsep="0">
<table align="left" class="description-table" cols="1" colsep="0" rowsep="0" width="100%">
<thead>
<tr class="description-tr">
<td class="description-td" colspan="1" nameend="1" namest="1" rowsep="1">TABLE 12</td>
</tr>
<tr class="description-tr">
<td align="center" class="description-td" colspan="1" nameend="1" namest="1" rowsep="1"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">Sequence A.</td>
</tr>
<tr class="description-tr">
<td align="center" class="description-td" colspan="1" nameend="1" namest="1" rowsep="1"> </td>
</tr>
</thead>
<tbody><tr class="description-tr">
<td class="description-td"> </td>
</tr>
</tbody></table>
<table align="left" class="description-table" cols="3" colsep="0" rowsep="0" width="100%">
<tbody><tr class="description-tr">
<td class="description-td">Step</td>
<td class="description-td">CHOP Conc (ppm)</td>
<td class="description-td">% HMW</td>
</tr>
</tbody></table>
<table align="left" class="description-table" cols="6" colsep="0" rowsep="0" width="100%">
<tbody><tr class="description-tr">
<td class="description-td">Description</td>
<td class="description-td">Step Recovery</td>
<td class="description-td">Load</td>
<td class="description-td">Pool</td>
<td class="description-td">Load</td>
<td class="description-td">Pool</td>
</tr>
<tr class="description-tr">
<td align="center" class="description-td" colspan="6" nameend="6" namest="1" rowsep="1"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">Poros HS</td>
<td class="description-td">96</td>
<td class="description-td">659</td>
<td class="description-td">110</td>
<td class="description-td">6.4</td>
<td class="description-td">0.5</td>
</tr>
<tr class="description-tr">
<td class="description-td">Capto Adhere</td>
<td class="description-td">100</td>
<td class="description-td">110</td>
<td class="description-td">3</td>
<td class="description-td">0.5</td>
<td class="description-td">0.4</td>
</tr>
<tr class="description-tr">
<td align="center" class="description-td" colspan="6" nameend="6" namest="1" rowsep="1"> </td>
</tr>
</tbody></table>
<table align="left" class="description-table" cols="4" colsep="0" rowsep="0" width="100%">
<tbody><tr class="description-tr">
<td class="description-td">Step</td>
<td class="description-td">DNA (pg/mg)</td>
<td class="description-td">Leached ProA (ppm)</td>
<td class="description-td">Gentamicin (ppm)</td>
</tr>
</tbody></table>
<table align="left" class="description-table" cols="7" colsep="0" rowsep="0" width="100%">
<tbody><tr class="description-tr">
<td class="description-td">Description</td>
<td class="description-td">Load</td>
<td class="description-td">Pool</td>
<td class="description-td">Load</td>
<td class="description-td">Pool</td>
<td class="description-td">Load</td>
<td class="description-td">Pool</td>
</tr>
<tr class="description-tr">
<td align="center" class="description-td" colspan="7" nameend="7" namest="1" rowsep="1"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">Poros HS</td>
<td class="description-td">137.6</td>
<td class="description-td">&lt;2</td>
<td class="description-td">27</td>
<td class="description-td">&lt;2</td>
<td class="description-td">46.8</td>
<td class="description-td">&lt;1</td>
</tr>
<tr class="description-tr">
<td class="description-td">Capto Adhere</td>
<td class="description-td">&lt;2</td>
<td class="description-td">&lt;2</td>
<td class="description-td">&lt;2</td>
<td class="description-td">&lt;2</td>
<td class="description-td">&lt;1</td>
<td class="description-td">&lt;1</td>
</tr>
<tr class="description-tr">
<td align="center" class="description-td" colspan="7" nameend="7" namest="1" rowsep="1"> </td>
</tr>
</tbody></table>
</patent-tables>
</tables>
</div>
</li> <li> <div class="description-line" id="p-0392" num="0000">
<tables id="TABLE-US-00013" num="00013">
<patent-tables colsep="0" frame="none" pgwide="1" rowsep="0">
<table align="left" class="description-table" cols="1" colsep="0" rowsep="0" width="100%">
<thead>
<tr class="description-tr">
<td class="description-td" colspan="1" nameend="1" namest="1" rowsep="1">TABLE 13</td>
</tr>
<tr class="description-tr">
<td align="center" class="description-td" colspan="1" nameend="1" namest="1" rowsep="1"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">Sequence B.</td>
</tr>
<tr class="description-tr">
<td align="center" class="description-td" colspan="1" nameend="1" namest="1" rowsep="1"> </td>
</tr>
</thead>
<tbody><tr class="description-tr">
<td class="description-td"> </td>
</tr>
</tbody></table>
<table align="left" class="description-table" cols="3" colsep="0" rowsep="0" width="100%">
<tbody><tr class="description-tr">
<td class="description-td">Step</td>
<td class="description-td">CHOP Conc (ppm)</td>
<td class="description-td">% HMW</td>
</tr>
</tbody></table>
<table align="left" class="description-table" cols="6" colsep="0" rowsep="0" width="100%">
<tbody><tr class="description-tr">
<td class="description-td">Description</td>
<td class="description-td">Step Recovery</td>
<td class="description-td">Load</td>
<td class="description-td">Pool</td>
<td class="description-td">Load</td>
<td class="description-td">Pool</td>
</tr>
<tr class="description-tr">
<td align="center" class="description-td" colspan="6" nameend="6" namest="1" rowsep="1"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">Capto Adhere</td>
<td class="description-td">102</td>
<td class="description-td">659</td>
<td class="description-td">16</td>
<td class="description-td">6.4</td>
<td class="description-td">6.2</td>
</tr>
<tr class="description-tr">
<td class="description-td">Poros HS</td>
<td class="description-td">90</td>
<td class="description-td">16</td>
<td class="description-td">2</td>
<td class="description-td">6.2</td>
<td class="description-td">0.3</td>
</tr>
<tr class="description-tr">
<td align="center" class="description-td" colspan="6" nameend="6" namest="1" rowsep="1"> </td>
</tr>
</tbody></table>
<table align="left" class="description-table" cols="4" colsep="0" rowsep="0" width="100%">
<tbody><tr class="description-tr">
<td class="description-td">Step</td>
<td class="description-td">DNA (pg/mg)</td>
<td class="description-td">Leached ProA (ppm)</td>
<td class="description-td">Gentamicin (ppm)</td>
</tr>
</tbody></table>
<table align="left" class="description-table" cols="7" colsep="0" rowsep="0" width="100%">
<tbody><tr class="description-tr">
<td class="description-td">Description</td>
<td class="description-td">Load</td>
<td class="description-td">Pool</td>
<td class="description-td">Load</td>
<td class="description-td">Pool</td>
<td class="description-td">Load</td>
<td class="description-td">Pool</td>
</tr>
<tr class="description-tr">
<td align="center" class="description-td" colspan="7" nameend="7" namest="1" rowsep="1"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">Capto Adhere</td>
<td class="description-td">137.6</td>
<td class="description-td">1.1</td>
<td class="description-td">27</td>
<td class="description-td">&lt;2</td>
<td class="description-td">46.8</td>
<td class="description-td">&lt;1</td>
</tr>
<tr class="description-tr">
<td class="description-td">Poros HS</td>
<td class="description-td">1.1</td>
<td class="description-td">&lt;2</td>
<td class="description-td">&lt;2</td>
<td class="description-td">&lt;2</td>
<td class="description-td">&lt;1</td>
<td class="description-td">&lt;1</td>
</tr>
<tr class="description-tr">
<td align="center" class="description-td" colspan="7" nameend="7" namest="1" rowsep="1"> </td>
</tr>
</tbody></table>
</patent-tables>
</tables>
</div>
</li> <li> <para-num num="[0375]"> </para-num> <div class="description-line" id="p-0393" num="0375">Experiment 2—</div>
</li> <li> <para-num num="[0376]"> </para-num> <div class="description-line" id="p-0394" num="0376">The filtered and equilibrated product described above was continuously purified using Capto Adhere (0.66 cm×5.2 cm) then Poros HS50 column (0.66 cm×4.8 cm) with approximately 566 g/L CV of product loaded for Poros HS50 and about 614 g/L CV for Capto Adhere at pH5.5, 5 mS/cm, and 100 cm/hr under the conditions recited in the table below.</div>
</li> <li> <div class="description-line" id="p-0395" num="0000">
<tables id="TABLE-US-00014" num="00014">
<patent-tables colsep="0" frame="none" rowsep="0">
<table align="left" class="description-table" cols="1" colsep="0" rowsep="0" width="100%">
<thead>
<tr class="description-tr">
<td class="description-td" colspan="1" nameend="1" namest="1" rowsep="1">TABLE 14</td>
</tr>
</thead>
<tbody><tr class="description-tr">
<td align="center" class="description-td" colspan="1" nameend="1" namest="1" rowsep="1"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">Chromatographic conditions used in the continuous</td>
</tr>
<tr class="description-tr">
<td class="description-td">processing of anti-CD11a antibody.</td>
</tr>
</tbody></table>
<table align="left" class="description-table" cols="3" colsep="0" rowsep="0" width="100%">
<tbody><tr class="description-tr">
<td class="description-td">Phase</td>
<td class="description-td">Buffer/Solution</td>
<td class="description-td">Process Parameter</td>
</tr>
<tr class="description-tr">
<td align="center" class="description-td" colspan="3" nameend="3" namest="1" rowsep="1"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">Equilibration</td>
<td class="description-td">77 mM NaAc,</td>
<td class="description-td">5 CV of Poros HS50 or</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">pH 5.5,</td>
<td class="description-td">Capto Adhere resin</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">5 mS/cm</td>
</tr>
<tr class="description-tr">
<td class="description-td">Load</td>
<td class="description-td">Conditioned Protein A</td>
<td class="description-td">Load up to 614 g/L</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">pool (pH 5.5 &amp;</td>
<td class="description-td">Poros HS50 resin;</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">5 mS/cm)</td>
<td class="description-td">start to pooling at OD280 ≥0.5;</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">end pooling at OD280 ≤0.5</td>
</tr>
<tr class="description-tr">
<td class="description-td">Wash</td>
<td class="description-td">77 mM NaAc, pH 5.5,</td>
<td class="description-td">≥2 CV of Poros HS50 or</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">5 mS/cm</td>
<td class="description-td">Capto Adhere column</td>
</tr>
<tr class="description-tr">
<td class="description-td">Strip 1</td>
<td class="description-td">2M NaCl</td>
<td class="description-td">5 CV of Poros HS50 or</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">Capto Adhere resin</td>
</tr>
<tr class="description-tr">
<td class="description-td">Strip 2</td>
<td class="description-td">0.15M acetate buffer,</td>
<td class="description-td">5 CV of Poros HS50 or</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">pH 2.8</td>
<td class="description-td">Capto Adhere resin</td>
</tr>
<tr class="description-tr">
<td class="description-td">Sanitization</td>
<td class="description-td">0.5N NaOH</td>
<td class="description-td">4 CV of Poros HS50 or</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">Capto Adhere column</td>
</tr>
<tr class="description-tr">
<td class="description-td">Storage</td>
<td class="description-td">0.1N NaOH</td>
<td class="description-td">3 CV of Poros HS50 or</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">Capto Adhere resin</td>
</tr>
<tr class="description-tr">
<td align="center" class="description-td" colspan="3" nameend="3" namest="1" rowsep="1"> </td>
</tr>
</tbody></table>
</patent-tables>
</tables>
</div>
</li> <li> <para-num num="[0377]"> </para-num> <div class="description-line" id="p-0396" num="0377">The chromatogram results for the continuous process are shown in <figref idrefs="DRAWINGS">FIG. 34</figref>. The continuous processing resulted in the Mab recovery yield of about 90%. Data not shown. In addition, there was 617 ppm CHOP in the load but only 3.4 ppm in the pool, there was 6.37% HMW in the load but only 0.6% in the pool (90% HMW removal), and DNA, Protein A leachate, and gentamicin was below detection limits.</div>
</li> <li> <para-num num="[0378]"> </para-num> <div class="description-line" id="p-0397" num="0378">Anti-CD20 Purification</div>
</li> <li> <para-num num="[0379]"> </para-num> <div class="description-line" id="p-0398" num="0379">Experiment 1—</div>
</li> <li> <para-num num="[0380]"> </para-num> <div class="description-line" id="p-0399" num="0380">The product pool from Protein A chromatography was purified using Poros HS50 column (0.66 cm×5 cm) with approximately 600 g/L CV of product loaded and Capto Adhere (0.66 cm×8 cm) with approximately 316 g/L CV for Capto Adhere at pH 5.5, 5 mS/cm, and 100 cm/hr.</div>
</li> <li> <para-num num="[0381]"> </para-num> <div class="description-line" id="p-0400" num="0381">Results are provided in Table 15 below. The overloaded Poros HS resulted in ˜50% reduction in % HMW and 98% CHOP reduction. Poros HS50 was the best resin compared to SE Hicap, SPFF, SPXL and Capto S in reducing % HMW (data not shown). Capto Adhere resulted in no additional % HMW reduction, but a 99% CHOP reduction (CHOP break through was observed).</div>
</li> <li> <div class="description-line" id="p-0401" num="0000">
<tables id="TABLE-US-00015" num="00015">
<patent-tables colsep="0" frame="none" rowsep="0">
<table align="left" class="description-table" cols="6" colsep="0" rowsep="0" width="100%">
<thead>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td" colspan="6" nameend="5" namest="offset" rowsep="1">TABLE 15</td>
</tr>
</thead>
<tbody><tr class="description-tr">
<td class="description-td"> </td>
<td align="center" class="description-td" colspan="6" nameend="5" namest="offset" rowsep="1"> </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">Step</td>
<td class="description-td">% HMW</td>
<td class="description-td"> </td>
<td class="description-td">CHOP (ppm)</td>
<td class="description-td"> </td>
</tr>
</tbody></table>
<table align="left" class="description-table" cols="6" colsep="0" rowsep="0" width="100%">
<tbody><tr class="description-tr">
<td class="description-td">Resin</td>
<td class="description-td">Recovery</td>
<td class="description-td">Load</td>
<td class="description-td">Pool</td>
<td class="description-td">Load</td>
<td class="description-td">Pool</td>
</tr>
<tr class="description-tr">
<td align="center" class="description-td" colspan="6" nameend="6" namest="1" rowsep="1"> </td>
</tr>
</tbody></table>
<table align="left" class="description-table" cols="6" colsep="0" rowsep="0" width="100%">
<tbody><tr class="description-tr">
<td class="description-td">Poros 50 HS</td>
<td class="description-td">90</td>
<td class="description-td">3.8</td>
<td class="description-td">1.7</td>
<td class="description-td">3812</td>
<td class="description-td">72</td>
</tr>
<tr class="description-tr">
<td class="description-td">Capto Adhere</td>
<td class="description-td">100</td>
<td class="description-td">3.5</td>
<td class="description-td">3.4</td>
<td class="description-td">4722</td>
<td class="description-td">49</td>
</tr>
<tr class="description-tr">
<td align="center" class="description-td" colspan="6" nameend="6" namest="1" rowsep="1"> </td>
</tr>
</tbody></table>
</patent-tables>
</tables>
</div>
</li> <li> <para-num num="[0382]"> </para-num> <div class="description-line" id="p-0402" num="0382">Experiment 2—</div>
</li> <li> <para-num num="[0383]"> </para-num> <div class="description-line" id="p-0403" num="0383">The filtered and equilibrated product described above was continuously purified using an overloaded Poros HS50 column (0.66 cm×4.8 cm) and a Capto Adhere (0.66 cm×7 cm) column with approximately ˜600 g/L CV of product loaded for Poros HS50 and about ˜340 g/L CV for Capto Adhere at pH 5.5, 5 mS/cm, and 100 cm/hr. The experiment was performed in two different sequences: a) Sequence A: (i) Poros HS50 column and (ii) Capto Adhere or b) Sequence B (i) Capto Adhere and (ii) Poros HS50 column under the conditions recited in the table below.</div>
</li> <li> <div class="description-line" id="p-0404" num="0000">
<tables id="TABLE-US-00016" num="00016">
<patent-tables colsep="0" frame="none" rowsep="0">
<table align="left" class="description-table" cols="1" colsep="0" rowsep="0" width="100%">
<thead>
<tr class="description-tr">
<td class="description-td" colspan="1" nameend="1" namest="1" rowsep="1">TABLE 16</td>
</tr>
</thead>
<tbody><tr class="description-tr">
<td align="center" class="description-td" colspan="1" nameend="1" namest="1" rowsep="1"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">Chromatographic conditions used in continuous</td>
</tr>
<tr class="description-tr">
<td class="description-td">processing of anti-CD20 antibody.</td>
</tr>
</tbody></table>
<table align="left" class="description-table" cols="3" colsep="0" rowsep="0" width="100%">
<tbody><tr class="description-tr">
<td class="description-td">Phase</td>
<td class="description-td">Buffer/Solution</td>
<td class="description-td">Process Parameter</td>
</tr>
<tr class="description-tr">
<td align="center" class="description-td" colspan="3" nameend="3" namest="1" rowsep="1"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">Equilibration</td>
<td class="description-td">77 mM NaAc,</td>
<td class="description-td">5 CV of Poros HS50 or</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">pH 5.5,</td>
<td class="description-td">Capto Adhere resin</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">5 mS/cm</td>
<td class="description-td">Load up to 600 g/L</td>
</tr>
<tr class="description-tr">
<td class="description-td">Load</td>
<td class="description-td">Conditioned Protein A</td>
<td class="description-td">Poros HS50 resin</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">pool at pH 5.5 &amp;</td>
<td class="description-td">(it equals to 340 g/L</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">5 mS/cm</td>
<td class="description-td">Capto Adhere resin);</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">start to pooling at OD280 ≥0.5;</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">end pooling at OD280 ≤0.5</td>
</tr>
<tr class="description-tr">
<td class="description-td">Wash</td>
<td class="description-td">~80 mM NaAc,</td>
<td class="description-td">≥2 CV of</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">pH 5.5,</td>
<td class="description-td">Poros HS50 or</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">5 mS/cm</td>
<td class="description-td">Capto Adhere column</td>
</tr>
<tr class="description-tr">
<td class="description-td">Strip 1</td>
<td class="description-td">2M NaCl</td>
<td class="description-td">5 CV of Poros HS50 or</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">Capto Adhere resin</td>
</tr>
<tr class="description-tr">
<td class="description-td">Strip 2</td>
<td class="description-td">0.15M acetate buffer,</td>
<td class="description-td">5 CV of Poros HS50 or</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">pH 2.8</td>
<td class="description-td">Capto Adhere resin</td>
</tr>
<tr class="description-tr">
<td class="description-td">Sanitization</td>
<td class="description-td">0.5N NaOH</td>
<td class="description-td">4 CV of Poros HS50 or</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">Capto Adhere column</td>
</tr>
<tr class="description-tr">
<td class="description-td">Storage</td>
<td class="description-td">0.1N NaOH</td>
<td class="description-td">3 CV of Poros HS50 or</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">Capto Adhere resin</td>
</tr>
<tr class="description-tr">
<td align="center" class="description-td" colspan="3" nameend="3" namest="1" rowsep="1"> </td>
</tr>
</tbody></table>
</patent-tables>
</tables>
</div>
</li> <li> <para-num num="[0384]"> </para-num> <div class="description-line" id="p-0405" num="0384">Results are provided in Table 17 below. The continuous processing resulted in the Mab recovery yield of about 90% (Poros HS50 then Capto Adhere) and about 87% (Capto Adhere then Poros HS50). The overloaded Poros HS resulted in ˜59% reduction in % HMW, CHOP was reduced to less than 10 ppm, and DNA, Protein A leachate, gentamicin were below detectable limits.</div>
</li> <li> <div class="description-line" id="p-0406" num="0000">
<tables id="TABLE-US-00017" num="00017">
<patent-tables colsep="0" frame="none" rowsep="0">
<table align="left" class="description-table" cols="4" colsep="0" rowsep="0" width="100%">
<thead>
<tr class="description-tr">
<td class="description-td" colspan="4" nameend="4" namest="1" rowsep="1">TABLE 17</td>
</tr>
</thead>
<tbody><tr class="description-tr">
<td align="center" class="description-td" colspan="4" nameend="4" namest="1" rowsep="1"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">Step</td>
<td class="description-td">Step</td>
<td class="description-td">% HMW</td>
<td class="description-td">CHOP Conc (ppm)</td>
</tr>
</tbody></table>
<table align="left" class="description-table" cols="6" colsep="0" rowsep="0" width="100%">
<tbody><tr class="description-tr">
<td class="description-td">Order</td>
<td class="description-td">Recovery (%)</td>
<td class="description-td">Load</td>
<td class="description-td">Pool</td>
<td class="description-td">Load</td>
<td class="description-td">Pool</td>
</tr>
<tr class="description-tr">
<td align="center" class="description-td" colspan="6" nameend="6" namest="1" rowsep="1"> </td>
</tr>
</tbody></table>
<table align="left" class="description-table" cols="6" colsep="0" rowsep="0" width="100%">
<tbody><tr class="description-tr">
<td class="description-td">Poros HS/</td>
<td class="description-td">92</td>
<td class="description-td">3.7</td>
<td class="description-td">1.5</td>
<td class="description-td">5607</td>
<td class="description-td">5</td>
</tr>
<tr class="description-tr">
<td class="description-td">Capto Adhere</td>
</tr>
<tr class="description-tr">
<td class="description-td">Capto Adhere/</td>
<td class="description-td">87</td>
<td class="description-td">3.7</td>
<td class="description-td">1.5</td>
<td class="description-td">4797</td>
<td class="description-td">6</td>
</tr>
<tr class="description-tr">
<td class="description-td">Poros HS</td>
</tr>
<tr class="description-tr">
<td align="center" class="description-td" colspan="6" nameend="6" namest="1" rowsep="1"> </td>
</tr>
</tbody></table>
</patent-tables>
</tables>
</div>
</li> </ul>
</div>
</section><section itemprop="claims" itemscope="">
<h2>Claims (<span itemprop="count">32</span>)</h2>
<div html="" itemprop="content"><div class="claims" lang="EN" load-source="patent-office" mxw-id="PCLM278098532">
<div class="claim"> <div class="claim" id="CLM-00001" num="00001">
<div class="claim-text"> <b>1</b>. A method for purifying a antibody or immunoadhesin from a composition comprising the antibody or immunoadhesin and at least one contaminant, wherein the method comprises either (i) or (ii):
<div class="claim-text">(i) sequential steps of (a) loading the composition onto a cation exchange material at a loading density of greater than about 150 g/L of cation exchange material; and (b) loading a composition recovered from the cation exchange material onto a mixed mode material; or</div> <div class="claim-text">(ii) sequential steps of (a) loading the composition onto a mixed mode material; and (b) loading a composition recovered from mixed mode material onto a cation exchange material at a loading density of greater than about 150 g/L of cation exchange material;</div> <div class="claim-text">wherein the cation exchange material is resin particles.</div> </div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="CLM-00002" num="00002">
<div class="claim-text"> <b>2</b>. The method of <claim-ref idref="CLM-00001">claim 1</claim-ref>, wherein the antibody or immunoadhesin has a pI of between about 6 and about 10.</div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="CLM-00003" num="00003">
<div class="claim-text"> <b>3</b>. The method of <claim-ref idref="CLM-00002">claim 2</claim-ref>, wherein the antibody or immunoadhesin has a pI of between about 7 and about 9.</div>
</div>
</div> <div class="claim"> <div class="claim" id="CLM-00004" num="00004">
<div class="claim-text"> <b>4</b>. (canceled)</div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="CLM-00005" num="00005">
<div class="claim-text"> <b>5</b>. The method of <claim-ref idref="CLM-00001">claim 1</claim-ref>, wherein the antibody or immunoadhesin is an immunoadhesin.</div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="CLM-00006" num="00006">
<div class="claim-text"> <b>6</b>. The method of <claim-ref idref="CLM-00001">claim 1</claim-ref>, wherein the antibody or immunoadhesin is an antibody.</div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="CLM-00007" num="00007">
<div class="claim-text"> <b>7</b>. The method of <claim-ref idref="CLM-00006">claim 6</claim-ref>, wherein the antibody is a monoclonal antibody.</div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="CLM-00008" num="00008">
<div class="claim-text"> <b>8</b>. The method of <claim-ref idref="CLM-00007">claim 7</claim-ref>, wherein the monoclonal antibody is a chimeric antibody, humanized antibody, or human antibody.</div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="CLM-00009" num="00009">
<div class="claim-text"> <b>9</b>. The method of <claim-ref idref="CLM-00007">claim 7</claim-ref>, wherein the monoclonal antibody is an IgG monoclonal antibody.</div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="CLM-00010" num="00010">
<div class="claim-text"> <b>10</b>. The method of <claim-ref idref="CLM-00006">claim 6</claim-ref>, wherein the antibody is an antigen binding fragment.</div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="CLM-00011" num="00011">
<div class="claim-text"> <b>11</b>. The method of <claim-ref idref="CLM-00010">claim 10</claim-ref>, wherein the antigen binding fragment is selected from the group consisting of a Fab fragment, a Fab′ fragment, a F(ab′)<sub>2 </sub>fragment, a scFv, a Fv, and a diabody.</div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="CLM-00012" num="00012">
<div class="claim-text"> <b>12</b>. The method of <claim-ref idref="CLM-00001">claim 1</claim-ref>, wherein the at least one contaminant is any one or more of Chinese Hamster Ovary Protein (CHOP), leached protein A, DNA, aggregated protein, cell culture media component, gentamicin, and viral contaminant.</div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="CLM-00013" num="00013">
<div class="claim-text"> <b>13</b>. The method of <claim-ref idref="CLM-00001">claim 1</claim-ref>, wherein the sequential steps in (i) and/or (ii) are continuous.</div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="CLM-00014" num="00014">
<div class="claim-text"> <b>14</b>. The method of <claim-ref idref="CLM-00001">claim 1</claim-ref>, wherein the method is (i).</div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="CLM-00015" num="00015">
<div class="claim-text"> <b>15</b>. The method of <claim-ref idref="CLM-00001">claim 1</claim-ref>, wherein the method is (ii).</div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="CLM-00016" num="00016">
<div class="claim-text"> <b>16</b>. The method of <claim-ref idref="CLM-00001">claim 1</claim-ref>, wherein the loading density is between about 150 g/L and about 2000 g/L of cation exchange material.</div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="CLM-00017" num="00017">
<div class="claim-text"> <b>17</b>. The method of <claim-ref idref="CLM-00016">claim 16</claim-ref>, wherein the density is between about 500 g/L and about 1000 g/L of cation exchange material.</div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="CLM-00018" num="00018">
<div class="claim-text"> <b>18</b>. The method of <claim-ref idref="CLM-00001">claim 1</claim-ref>, wherein the cation exchange material comprises a carboxylic acid functional group or a sulfonic acid functional group.</div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="CLM-00019" num="00019">
<div class="claim-text"> <b>19</b>. The method of <claim-ref idref="CLM-00018">claim 18</claim-ref>, wherein the functional group is sulphopropyl, sulfoethyl, sulfoisobutyl, or caboxyl.</div>
</div>
</div> <div class="claim"> <div class="claim" id="CLM-00020" num="00020">
<div class="claim-text"> <b>20</b>. (canceled)</div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="CLM-00021" num="00021">
<div class="claim-text"> <b>21</b>. The method of <claim-ref idref="CLM-00020">claim 20</claim-ref>, wherein the cation exchange material is resin.</div>
</div>
</div> <div class="claim"> <div class="claim" id="CLM-00022" num="00022">
<div class="claim-text"> <b>22</b>. (canceled)</div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="CLM-00023" num="00023">
<div class="claim-text"> <b>23</b>. The method of <claim-ref idref="CLM-00001">claim 1</claim-ref>, wherein the mixed mode material comprises functional groups capable of anionic exchange and hydrophobic interactions.</div>
</div>
</div> <div class="claim"> <div class="claim" id="CLM-00024" num="00024">
<div class="claim-text"> <b>24</b>. (canceled)</div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="CLM-00025" num="00025">
<div class="claim-text"> <b>25</b>. The method of <claim-ref idref="CLM-00001">claim 1</claim-ref>, wherein the method comprises use of an equilibration buffer, a wash buffer, and/or a loading buffer with the cation exchange material and/or mixed mode material, and the conductivity of the equilibration buffer, the wash buffer, and/or the loading buffer is between about 2 mS/cm to about 25 mS/cm.</div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="CLM-00026" num="00026">
<div class="claim-text"> <b>26</b>. The method of <claim-ref idref="CLM-00025">claim 25</claim-ref>, wherein the conductivity of the equilibration buffer, the wash buffer, and/or the loading buffer is between about 3 mS/cm and 8 mS/cm.</div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="CLM-00027" num="00027">
<div class="claim-text"> <b>27</b>. The method of <claim-ref idref="CLM-00001">claim 1</claim-ref>, wherein the method comprises use of an equilibration buffer, a wash buffer, and/or a loading buffer with the cation exchange material and/or the mixed mode material, and the pH of the equilibration buffer, the wash buffer, and/or the loading buffer is between about 4.5 and about 6.5.</div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="CLM-00028" num="00028">
<div class="claim-text"> <b>28</b>. The method of <claim-ref idref="CLM-00025">claim 25</claim-ref>, wherein the equilibration buffer, the wash buffer, and/or the loading buffer with the cation exchange material and/or the mixed mode material are the same.</div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="CLM-00029" num="00029">
<div class="claim-text"> <b>29</b>. The method of <claim-ref idref="CLM-00001">claim 1</claim-ref> further comprises subjecting the composition comprising the antibody or immunoadhesin to one or more further purification steps either before or after steps (a) and (b).</div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="CLM-00030" num="00030">
<div class="claim-text"> <b>30</b>. The method of <claim-ref idref="CLM-00001">claim 1</claim-ref> further comprising recovering the purified antibody or immunoadhesin.</div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="CLM-00031" num="00031">
<div class="claim-text"> <b>31</b>. The method of <claim-ref idref="CLM-00030">claim 30</claim-ref> further comprising combining the purified antibody or immunoadhesin with a pharmaceutically acceptable carrier.</div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="CLM-00032" num="00032">
<div class="claim-text"> <b>32</b>. The method of <claim-ref idref="CLM-00006">claim 6</claim-ref>, wherein the antibody is an anti-CD20 antibody, an anti-CD11a antibody or an anti-VEGF antibody.</div>
</div>
</div> </div>
</div>
</section>
                </article>
            </search-app>
        </body>
    </html>
    