
    <html>
        <body>
            <search-app>
                <article class="result" itemscope="" itemtype="http://schema.org/ScholarlyArticle">
    <h1 itemprop="pageTitle">US20200291478A1 - Identification of polymorphic sequences in mixtures of genomic dna 
      - Google Patents</h1><section itemprop="abstract" itemscope="">
<h2>Abstract</h2>
<div html="" itemprop="content"><abstract lang="EN" load-source="docdb" mxw-id="PA414486986" source="national office">
<div class="abstract">The present invention relates to methods comprising identifying polymorphisms in samples comprising mixtures of genomes, and for determining and/or monitoring the presence or absence of disorders associated with the identified polymorphisms.</div>
</abstract>
</div>
</section><section itemprop="description" itemscope="">
<h2>Description</h2>
<div html="" itemprop="content"><ul class="description" lang="EN" load-source="patent-office" mxw-id="PDES273858694">
<heading id="h-0001">CROSS-REFERENCE</heading>
<li> <para-num num="[0001]"> </para-num> <div class="description-line" id="p-0002" num="0001">This application is a continuation of application Ser. No. 13/009,718, filed on Jan. 19, 2011, which is a continuation-in-part of application Ser. No. 12/958,347 filed on Dec. 1, 2010, which application is incorporated by reference in its entirety. This application claims priority to U.S. Provisional Application Ser. No. 61/296,358, filed on Jan. 19, 2010; and U.S. Provisional Application Ser. No. 61/360,837, filed on Jul. 1, 2010, which applications are incorporated herein by reference in their entirety.</div>
</li> <heading id="h-0002">FIELD OF THE INVENTION</heading>
<li> <para-num num="[0002]"> </para-num> <div class="description-line" id="p-0003" num="0002">The invention is applicable to the field of medical diagnostics and particularly relates to whole genome sequencing methods for identifying polymorphisms in samples comprising mixtures of genomes.</div>
</li> <heading id="h-0003">BACKGROUND OF THE INVENTION</heading>
<li> <para-num num="[0003]"> </para-num> <div class="description-line" id="p-0004" num="0003">Prenatal screening and diagnosis are a routine part of antenatal care. Currently, prenatal diagnosis of genetic and chromosomal conditions involves invasive testing, such as amniocentesis or chorionic villus sampling (CVS), performed from 11 weeks gestation and carrying a ˜1% risk of miscarriage. The existence of circulating cell-free DNA in maternal blood (Lo et al., Lancet 350:485-487 [1997]) is being exploited for developing noninvasive processes that use fetal nucleic acids from a maternal peripheral blood sample to determine fetal chromosomal aneuploidies e.g. trisomy 21 (Fan H C and Quake S R Anal Chem 79:7576-7579 [2007]; Fan et al., Proc Natl Acad Sci 105:16266-16271 [2008]). These methods offer an alternative and safer source of fetal genetic material for prenatal diagnosis, and could effectively pronounce the end of invasive procedures.</div>
</li> <li> <para-num num="[0004]"> </para-num> <div class="description-line" id="p-0005" num="0004">Next Generation Sequencing (NGS) technologies have been used to determine entire human genome sequences (Levy et al. PLoS Biol 55, e254 [2007]; Wheeler et al. Nature 452:872-876 [2008]; Bentley et al., Nature 456:53-59 [2008]), and a broad interest exists in using NGS technologies for whole genome sequencing (WGS) to better understand human genetic variation and genome-related diseases, and ultimately to guide discoveries and decisions about the health of individuals. The extensive public genome-wide database of patterns of common human sequence variation provided by the International HapMap Project, and the increasing accessibility to whole genome sequencing technologies, will lead to the identification of new therapeutic targets and the development of targeted interventions for an individuals' medical care.</div>
</li> <li> <para-num num="[0005]"> </para-num> <div class="description-line" id="p-0006" num="0005">An additional need that remains is for identifying the disorder-associated genetic variations when two or more individual genomes are intermixed in a clinical sample e.g. mixtures of fetal and maternal genomes in biological fluid samples obtained from the mother, and mixtures of euploid and aneuploid genomes derived from cells of cancer patients.</div>
</li> <li> <para-num num="[0006]"> </para-num> <div class="description-line" id="p-0007" num="0006">The present invention addresses the need by providing a method for identifying polymorphisms in mixtures of genomes present in samples that can be obtained by noninvasive means. The method can be used for the non-invasive identification of multiple disease-associated polymorphisms in a variety of fields including but not limited to prenatal diagnostics, and oncology.</div>
</li> <heading id="h-0004">SUMMARY OF THE INVENTION</heading>
<li> <para-num num="[0007]"> </para-num> <div class="description-line" id="p-0008" num="0007">The present invention relates to methods comprising whole genome sequencing for identifying polymorphisms in samples comprising mixtures of genomes, and for determining and/or monitoring the presence or absence of disorders associated with the identified polymorphisms.</div>
</li> <li> <para-num num="[0008]"> </para-num> <div class="description-line" id="p-0009" num="0008">In one embodiment, the invention provides a method for identifying multiple polymorphisms in a first genome of a blood sample comprising a mixture of cfDNA of a first and a second genome, comprising: (a) whole genome sequencing at least a portion of the mixture of cfDNA, thereby obtaining a plurality of sequence tags, wherein the mixture is unenriched for the multiple polymorphisms; (b) comparing the sequence of the plurality of tags to the sequence of multiple reference polymorphisms; (c) identifying the multiple polymorphisms in the first and second genome of the mixture; and (d) associating the multiple polymorphisms identified in step (c) with the first and second genome, thereby identifying the multiple polymorphisms in the first genome of the mixture. The blood sample can be unenriched for polymorphic target sequences in the mixture of first and second genomes. Sequencing is massively parallel sequencing of clonally amplified cfDNA molecules or of single cfDNA molecules. In some embodiments, sequencing is performed using massively parallel sequencing-by-synthesis with reversible dye terminators. In other embodiments, sequencing is performed using massively parallel sequencing-by-ligation. In some embodiments, the sample is a plasma sample.</div>
</li> <li> <para-num num="[0009]"> </para-num> <div class="description-line" id="p-0010" num="0009">In another embodiment, the method for identifying multiple polymorphisms in a first genome of a blood sample comprising a mixture of cfDNA of a first and a second genome, comprises: (a) whole genome sequencing at least a portion of the mixture of cfDNA, thereby obtaining a plurality of sequence tags, wherein the mixture is unenriched for the multiple polymorphisms; (b) comparing the sequence of the plurality of tags to the sequence of multiple reference polymorphisms; (c) identifying the multiple polymorphisms in the first and second genome of the mixture; and (d) associating the multiple polymorphisms identified in step (c) with the first and second genome, thereby identifying the multiple polymorphisms in the first genome of the mixture. Step (c) comprises genotyping the second genome in a sample that is substantially free of the first genome. The blood sample can be unenriched for polymorphic target sequences in the mixture of first and second genomes. Sequencing is massively parallel sequencing of clonally amplified cfDNA molecules or of single cfDNA molecules. In some embodiments, sequencing is performed using massively parallel sequencing-by-synthesis with reversible dye terminators. In other embodiments, sequencing is performed using massively parallel sequencing-by-ligation. In some embodiments, the sample is a plasma sample.</div>
</li> <li> <para-num num="[0010]"> </para-num> <div class="description-line" id="p-0011" num="0010">In another embodiment, the method for identifying multiple polymorphisms in a first genome of a blood sample comprising a mixture of cfDNA of a first and a second genome, comprises: (a) whole genome sequencing at least a portion of the mixture of cfDNA, thereby obtaining a plurality of sequence tags, wherein the mixture is unenriched for the multiple polymorphisms; (b) comparing the sequence of the plurality of tags to the sequence of multiple reference polymorphisms; (c) identifying the multiple polymorphisms in the first and second genome of the mixture; and (d) associating the multiple polymorphisms identified in step (c) with the first and second genome, thereby identifying the multiple polymorphisms in the first genome of the mixture, wherein step (c) comprises counting sequence tags mapped to the multiple reference polymorphisms. The blood sample can be unenriched for polymorphic target sequences in the mixture of first and second genomes. Sequencing is massively parallel sequencing of clonally amplified cfDNA molecules or of single cfDNA molecules. In some embodiments, sequencing is performed using massively parallel sequencing-by-synthesis with reversible dye terminators. In other embodiments, sequencing is performed using massively parallel sequencing-by-ligation. In some embodiments, the sample is a plasma sample.</div>
</li> <li> <para-num num="[0011]"> </para-num> <div class="description-line" id="p-0012" num="0011">In another embodiment, the method for identifying multiple polymorphisms in a first genome of a blood sample comprising a mixture of cfDNA of a first and a second genome, comprises: (a) whole genome sequencing at least a portion of the mixture of cfDNA, thereby obtaining a plurality of sequence tags, wherein the mixture is unenriched for the multiple polymorphisms; (b) comparing the sequence of the plurality of tags to the sequence of multiple reference polymorphisms; (c) identifying the multiple polymorphisms in the first and second genome of the mixture; and (d) associating the multiple polymorphisms identified in step (c) with the first and second genome, thereby identifying the multiple polymorphisms in the first genome of the mixture, wherein the multiple polymorphisms in the first genome are associated with at least one disorder. The blood sample can be unenriched for polymorphic target sequences in the mixture of first and second genomes. Sequencing is massively parallel sequencing of clonally amplified cfDNA molecules or of single cfDNA molecules. In some embodiments, sequencing is performed using massively parallel sequencing-by-synthesis with reversible dye terminators. In other embodiments, sequencing is performed using massively parallel sequencing-by-ligation. In some embodiments, the sample is a plasma sample.</div>
</li> <li> <para-num num="[0012]"> </para-num> <div class="description-line" id="p-0013" num="0012">In another embodiment, the method for identifying multiple polymorphisms in a first genome of a blood sample comprising a mixture of cfDNA of a first and a second genome, comprises: (a) whole genome sequencing at least a portion of the mixture of cfDNA, thereby obtaining a plurality of sequence tags, wherein the mixture is unenriched for the multiple polymorphisms; (b) comparing the sequence of the plurality of tags to the sequence of multiple reference polymorphisms; (c) identifying the multiple polymorphisms in the first and second genome of the mixture; and (d) associating the multiple polymorphisms identified in step (c) with the first and second genome, wherein step (c) comprises genotyping said second genome in a sample that is substantially free of said first genome, and wherein the multiple polymorphisms in the first genome are associated with at least one disorder. The blood sample can be unenriched for polymorphic target sequences in the mixture of first and second genomes. Sequencing is massively parallel sequencing of clonally amplified cfDNA molecules or of single cfDNA molecules. In some embodiments, sequencing is performed using massively parallel sequencing-by-synthesis with reversible dye terminators. In other embodiments, sequencing is performed using massively parallel sequencing-by-ligation. In some embodiments, the sample is a plasma sample.</div>
</li> <li> <para-num num="[0013]"> </para-num> <div class="description-line" id="p-0014" num="0013">In another embodiment, the method for identifying multiple polymorphisms in a first genome of a blood sample comprising a mixture of cfDNA of a first and a second genome, comprises: (a) whole genome sequencing at least a portion of the mixture of cfDNA, thereby obtaining a plurality of sequence tags, wherein the mixture is unenriched for the multiple polymorphisms; (b) comparing the sequence of the plurality of tags to the sequence of multiple reference polymorphisms; (c) identifying the multiple polymorphisms in the first and second genome of the mixture; and (d) associating the multiple polymorphisms identified in step (c) with the first and second genome, thereby identifying the multiple polymorphisms in the first genome of the mixture, wherein step (c) comprises counting sequence tags mapped to the multiple reference polymorphisms, and wherein the multiple polymorphisms in the first genome are associated with at least one disorder. The blood sample can be unenriched for polymorphic target sequences in the mixture of first and second genomes. Sequencing is massively parallel sequencing of clonally amplified cfDNA molecules or of single cfDNA molecules. In some embodiments, sequencing is performed using massively parallel sequencing-by-synthesis with reversible dye terminators. In other embodiments, sequencing is performed using massively parallel sequencing-by-ligation. In some embodiments, the sample is a plasma sample.</div>
</li> <li> <para-num num="[0014]"> </para-num> <div class="description-line" id="p-0015" num="0014">In another embodiment, the method for identifying multiple polymorphisms in a first genome of a blood sample comprising a mixture of cfDNA of a first and a second genome, comprises: (a) whole genome sequencing at least a portion of the mixture of cfDNA, thereby obtaining a plurality of sequence tags, wherein the mixture is unenriched for the multiple polymorphisms; (b) comparing the sequence of the plurality of tags to the sequence of multiple reference polymorphisms; (c) identifying the multiple polymorphisms in the first and second genome of the mixture; and (d) associating the multiple polymorphisms identified in step (c) with the first and second genome, thereby identifying the multiple polymorphisms in the first genome of the mixture. The blood sample is obtained from a pregnant woman, and it can be unenriched for cfDNA of the first or second genome. The first genome is a fetal genome and the second genome is a maternal genome. Optionally, the method further comprises identifying the multiple polymorphisms in a paternal genome. Sequencing is massively parallel sequencing of clonally amplified cfDNA molecules or of single cfDNA molecules. In some embodiments, sequencing is performed using massively parallel sequencing-by-synthesis with reversible dye terminators. In other embodiments, sequencing is performed using massively parallel sequencing-by-ligation. In some embodiments, the sample is a plasma sample.</div>
</li> <li> <para-num num="[0015]"> </para-num> <div class="description-line" id="p-0016" num="0015">In another embodiment, the method for identifying multiple polymorphisms in a first genome of a blood sample comprising a mixture of cfDNA of a first and a second genome, comprises: (a) whole genome sequencing at least a portion of the mixture of cfDNA, thereby obtaining a plurality of sequence tags, wherein the mixture is unenriched for the multiple polymorphisms; (b) comparing the sequence of the plurality of tags to the sequence of multiple reference polymorphisms; (c) identifying the multiple polymorphisms in the first and second genome of the mixture; and (d) associating the multiple polymorphisms identified in step (c) with the first and second genome, thereby identifying the multiple polymorphisms in the first genome of the mixture, wherein step (c) comprises genotyping the second genome in a sample that is substantially free of the first genome. The blood sample is obtained from a pregnant woman, and it can be unenriched for cfDNA of the first or second genome. The first genome is a fetal genome and the second genome is a maternal genome. Optionally, the method further comprises identifying the multiple polymorphisms in a paternal genome. Sequencing is massively parallel sequencing of clonally amplified cfDNA molecules or of single cfDNA molecules. In some embodiments, sequencing is performed using massively parallel sequencing-by-synthesis with reversible dye terminators. In other embodiments, sequencing is performed using massively parallel sequencing-by-ligation. In some embodiments, the sample is a plasma sample.</div>
</li> <li> <para-num num="[0016]"> </para-num> <div class="description-line" id="p-0017" num="0016">In another embodiment, the method for identifying multiple polymorphisms in a first genome of a blood sample comprising a mixture of cfDNA of a first and a second genome, comprises: (a) whole genome sequencing at least a portion of the mixture of cfDNA, thereby obtaining a plurality of sequence tags, wherein the mixture is unenriched for the multiple polymorphisms; (b) comparing the sequence of the plurality of tags to the sequence of multiple reference polymorphisms; (c) identifying the multiple polymorphisms in the first and second genome of the mixture; and (d) associating the multiple polymorphisms identified in step (c) with the first and second genome, thereby identifying the multiple polymorphisms in the first genome of the mixture, wherein step (c) comprises counting sequence tags mapped to the multiple reference polymorphisms. The blood sample is obtained from a pregnant woman, and it can be unenriched for cfDNA of the first or second genome. The first genome is a fetal genome and the second genome is a maternal genome. Optionally, the method further comprises identifying the multiple polymorphisms in a paternal genome. Sequencing is massively parallel sequencing of clonally amplified cfDNA molecules or of single cfDNA molecules. In some embodiments, sequencing is performed using massively parallel sequencing-by-synthesis with reversible dye terminators. In other embodiments, sequencing is performed using massively parallel sequencing-by-ligation. In some embodiments, the sample is a plasma sample.</div>
</li> <li> <para-num num="[0017]"> </para-num> <div class="description-line" id="p-0018" num="0017">In another embodiment, the method for identifying multiple polymorphisms in a first genome of a blood sample comprising a mixture of cfDNA of a first and a second genome, comprises: (a) whole genome sequencing at least a portion of the mixture of cfDNA, thereby obtaining a plurality of sequence tags, wherein the mixture is unenriched for the multiple polymorphisms; (b) comparing the sequence of the plurality of tags to the sequence of multiple reference polymorphisms; (c) identifying the multiple polymorphisms in the first and second genome of the mixture; and (d) associating the multiple polymorphisms identified in step (c) with the first and second genome, thereby identifying the multiple polymorphisms in the first genome of the mixture. The sample is a blood sample obtained from a subject that is known or suspected of having cancer. The blood sample can be unenriched for polymorphic target sequences in the mixture of first and second genomes. Sequencing is massively parallel sequencing of clonally amplified cfDNA molecules or of single cfDNA molecules. In some embodiments, sequencing is performed using massively parallel sequencing-by-synthesis with reversible dye terminators. In other embodiments, sequencing is performed using massively parallel sequencing-by-ligation. In some embodiments, the sample is a plasma sample.</div>
</li> <li> <para-num num="[0018]"> </para-num> <div class="description-line" id="p-0019" num="0018">In another embodiment, the method for identifying multiple polymorphisms in a first genome of a blood sample comprising a mixture of cfDNA of a first and a second genome, comprises: (a) whole genome sequencing at least a portion of the mixture of cfDNA, thereby obtaining a plurality of sequence tags, wherein the mixture is unenriched for the multiple polymorphisms; (b) comparing the sequence of the plurality of tags to the sequence of multiple reference polymorphisms; (c) identifying the multiple polymorphisms in the first and second genome of the mixture; and (d) associating the multiple polymorphisms identified in step (c) with the first and second genome, thereby identifying the multiple polymorphisms in the first genome of the mixture, wherein step (c) comprises genotyping the second genome in a sample that is substantially free of the first genome. The sample is a blood sample obtained from a subject that is known or suspected of having cancer. The blood sample can be unenriched for polymorphic target sequences in the mixture of first and second genomes. Sequencing is massively parallel sequencing of clonally amplified cfDNA molecules or of single cfDNA molecules. In some embodiments, sequencing is performed using massively parallel sequencing-by-synthesis with reversible dye terminators. In other embodiments, sequencing is performed using massively parallel sequencing-by-ligation. In some embodiments, the sample is a plasma sample.</div>
</li> <li> <para-num num="[0019]"> </para-num> <div class="description-line" id="p-0020" num="0019">In another embodiment, the method for identifying multiple polymorphisms in a first genome of a blood sample comprising a mixture of cfDNA of a first and a second genome, comprises: (a) whole genome sequencing at least a portion of the mixture of cfDNA, thereby obtaining a plurality of sequence tags, wherein the mixture is unenriched for the multiple polymorphisms; (b) comparing the sequence of the plurality of tags to the sequence of multiple reference polymorphisms; (c) identifying the multiple polymorphisms in the first and second genome of the mixture; and (d) associating the multiple polymorphisms identified in step (c) with the first and second genome, thereby identifying the multiple polymorphisms in the first genome of the mixture, wherein step (c) comprises counting sequence tags mapped to the multiple reference polymorphisms. The sample is a blood sample obtained from a subject that is known or suspected of having cancer. The blood sample can be unenriched for polymorphic target sequences in the mixture of first and second genomes. Sequencing is massively parallel sequencing of clonally amplified cfDNA molecules or of single cfDNA molecules. In some embodiments, sequencing is performed using massively parallel sequencing-by-synthesis with reversible dye terminators. In other embodiments, sequencing is performed using massively parallel sequencing-by-ligation. In some embodiments, the sample is a plasma sample.</div>
</li> <li> <para-num num="[0020]"> </para-num> <div class="description-line" id="p-0021" num="0020">In another embodiment, the invention provides a method for determining the presence or absence of multiple disorders in a blood sample comprising a mixture of cfDNA of a first and a second genome, comprising: (a) whole genome sequencing at least a portion of the mixture of cfDNA, thereby providing a plurality of sequence tags; (b) identifying multiple polymorphisms in the plurality of sequence tags, wherein the multiple polymorphisms are associated with the number of disorders; and (c) associating the multiple polymorphisms with the first and/or second genome in the mixture, wherein the mixture is unenriched for the multiple polymorphisms. Sequencing is massively parallel sequencing of clonally amplified cfDNA molecules or of single cfDNA molecules. In some embodiments, sequencing is performed using massively parallel sequencing-by-synthesis with reversible dye terminators. In other embodiments, sequencing is performed using massively parallel sequencing-by-ligation. In some embodiments, the sample is a plasma sample.</div>
</li> <li> <para-num num="[0021]"> </para-num> <div class="description-line" id="p-0022" num="0021">In another embodiment, the method for determining the presence or absence of multiple disorders in a blood sample comprising a mixture of cfDNA of a first and a second genome, comprises: (a) whole genome sequencing at least a portion of the mixture of cfDNA, thereby providing a plurality of sequence tags; (b) identifying multiple polymorphisms in the plurality of sequence tags, wherein the multiple polymorphisms are associated with the number of disorders; and (c) associating the multiple polymorphisms with the first and/or second genome in the mixture, wherein the mixture is unenriched for the multiple polymorphisms. The method further comprises comparing the sequence of the plurality of tags to the sequence of multiple reference polymorphisms, thereby identifying the multiple polymorphisms in the mixture of cfDNA. Sequencing is massively parallel sequencing of clonally amplified cfDNA molecules or of single cfDNA molecules. In some embodiments, sequencing is performed using massively parallel sequencing-by-synthesis with reversible dye terminators. In other embodiments, sequencing is performed using massively parallel sequencing-by-ligation. In some embodiments, the sample is a plasma sample.</div>
</li> <li> <para-num num="[0022]"> </para-num> <div class="description-line" id="p-0023" num="0022">In another embodiment, the method for determining the presence or absence of multiple disorders in a blood sample comprising a mixture of cfDNA of a first and a second genome, comprises: (a) whole genome sequencing at least a portion of the mixture of cfDNA, thereby providing a plurality of sequence tags; (b) identifying multiple polymorphisms in the plurality of sequence tags, wherein the multiple polymorphisms are associated with the number of disorders; and (c) associating the multiple polymorphisms with the first and/or second genome in the mixture, wherein the mixture is unenriched for the multiple polymorphisms. In some embodiments, step (b) comprises counting sequence tags mapped to the multiple polymorphisms. Alternatively, step (b) comprises genotyping the second genome in a sample that is substantially free of the first genome. Sequencing is massively parallel sequencing of clonally amplified cfDNA molecules or of single cfDNA molecules. In some embodiments, sequencing is performed using massively parallel sequencing-by-synthesis with reversible dye terminators. In other embodiments, sequencing is performed using massively parallel sequencing-by-ligation. In some embodiments, the sample is a plasma sample.</div>
</li> <li> <para-num num="[0023]"> </para-num> <div class="description-line" id="p-0024" num="0023">In another embodiment, the method for determining the presence or absence of multiple disorders in a blood sample comprising a mixture of cfDNA of a first and a second genome, comprises: (a) whole genome sequencing at least a portion of the mixture of cfDNA, thereby providing a plurality of sequence tags; (b) identifying multiple polymorphisms in the plurality of sequence tags, wherein the multiple polymorphisms are associated with the number of disorders; and (c) associating the multiple polymorphisms with the first and/or second genome in the mixture, wherein the mixture is unenriched for the multiple polymorphisms, wherein the first genome is a genome of an unaffected cell and the second genome is a genome from an affected cell. Sequencing is massively parallel sequencing of clonally amplified cfDNA molecules or of single cfDNA molecules. In some embodiments, sequencing is performed using massively parallel sequencing-by-synthesis with reversible dye terminators. In other embodiments, sequencing is performed using massively parallel sequencing-by-ligation. In some embodiments, the sample is a plasma sample.</div>
</li> <li> <para-num num="[0024]"> </para-num> <div class="description-line" id="p-0025" num="0024">In another embodiment, the method for determining the presence or absence of multiple disorders in a blood sample comprising a mixture of cfDNA of a first and a second genome, comprises: (a) whole genome sequencing at least a portion of the mixture of cfDNA, thereby providing a plurality of sequence tags; (b) identifying multiple polymorphisms in the plurality of sequence tags, wherein the multiple polymorphisms are associated with the number of disorders; and (c) associating the multiple polymorphisms with the first and/or second genome in the mixture, wherein the mixture is unenriched for the multiple polymorphisms, and wherein the first genome is a genome of an unaffected cell and the second genome is a genome from an affected cell. The method further comprises comparing the sequence of the plurality of tags to the sequence of multiple reference polymorphisms, thereby identifying the multiple polymorphisms in the mixture of cfDNA. Sequencing is massively parallel sequencing of clonally amplified cfDNA molecules or of single cfDNA molecules. In some embodiments, sequencing is performed using massively parallel sequencing-by-synthesis with reversible dye terminators. In other embodiments, sequencing is performed using massively parallel sequencing-by-ligation. In some embodiments, the sample is a plasma sample. Sequencing is massively parallel sequencing of clonally amplified cfDNA molecules or of single cfDNA molecules. In some embodiments, sequencing is performed using massively parallel sequencing-by-synthesis with reversible dye terminators. In other embodiments, sequencing is performed using massively parallel sequencing-by-ligation. In some embodiments, the sample is a plasma sample.</div>
</li> <li> <para-num num="[0025]"> </para-num> <div class="description-line" id="p-0026" num="0025">In another embodiment, the method for determining the presence or absence of multiple disorders in a blood sample comprising a mixture of cfDNA of a first and a second genome, comprises: (a) whole genome sequencing at least a portion of the mixture of cfDNA, thereby providing a plurality of sequence tags; (b) identifying multiple polymorphisms in the plurality of sequence tags, wherein the multiple polymorphisms are associated with the number of disorders; and (c) associating the multiple polymorphisms with the first and/or second genome in the mixture, wherein the mixture is unenriched for the multiple polymorphisms, and wherein the first genome is a genome of an unaffected cell and the second genome is a genome from an affected cell. In some embodiments, step (b) comprises counting sequence tags mapped to the multiple polymorphisms. Alternatively, step (b) comprises genotyping the second genome in a sample that is substantially free of the first genome. Sequencing is massively parallel sequencing of clonally amplified cfDNA molecules or of single cfDNA molecules. In some embodiments, sequencing is performed using massively parallel sequencing-by-synthesis with reversible dye terminators. In other embodiments, sequencing is performed using massively parallel sequencing-by-ligation. In some embodiments, the sample is a plasma sample.</div>
</li> <li> <para-num num="[0026]"> </para-num> <div class="description-line" id="p-0027" num="0026">In another embodiment, the method for determining the presence or absence of multiple disorders in a blood sample comprising a mixture of cfDNA of a first and a second genome, comprises: (a) whole genome sequencing at least a portion of the mixture of cfDNA, thereby providing a plurality of sequence tags; (b) identifying multiple polymorphisms in the plurality of sequence tags, wherein the multiple polymorphisms are associated with the number of disorders; and (c) associating the multiple polymorphisms with the first and/or second genome in the mixture, wherein the mixture is unenriched for the multiple polymorphisms. In some embodiments the disorder is a monogenic disorder. Alternatively, the disorder is a polygenic disorder. In other embodiments, the disorder is cancer. Sequencing is massively parallel sequencing of clonally amplified cfDNA molecules or of single cfDNA molecules. In some embodiments, sequencing is performed using massively parallel sequencing-by-synthesis with reversible dye terminators. In other embodiments, sequencing is performed using massively parallel sequencing-by-ligation. In some embodiments, the sample is a plasma sample.</div>
</li> <li> <para-num num="[0027]"> </para-num> <div class="description-line" id="p-0028" num="0027">In another embodiment, the method for determining the presence or absence of multiple disorders in a blood sample comprising a mixture of cfDNA of a first and a second genome, comprises: (a) whole genome sequencing at least a portion of the mixture of cfDNA, thereby providing a plurality of sequence tags; (b) identifying multiple polymorphisms in the plurality of sequence tags, wherein the multiple polymorphisms are associated with the number of disorders; and (c) associating the multiple polymorphisms with the first and/or second genome in the mixture, wherein the mixture is unenriched for the multiple polymorphisms. The method further comprises comparing the sequence of the plurality of tags to the sequence of multiple reference polymorphisms, thereby identifying the multiple polymorphisms in the mixture of cfDNA. In some embodiments the disorder is a monogenic disorder. Alternatively, the disorder is a polygenic disorder. In other embodiments, the disorder is cancer. Sequencing is massively parallel sequencing of clonally amplified cfDNA molecules or of single cfDNA molecules. In some embodiments, sequencing is performed using massively parallel sequencing-by-synthesis with reversible dye terminators. In other embodiments, sequencing is performed using massively parallel sequencing-by-ligation. In some embodiments, the sample is a plasma sample.</div>
</li> <li> <para-num num="[0028]"> </para-num> <div class="description-line" id="p-0029" num="0028">In another embodiment, the method for determining the presence or absence of multiple disorders in a blood sample comprising a mixture of cfDNA of a first and a second genome, comprises: (a) whole genome sequencing at least a portion of the mixture of cfDNA, thereby providing a plurality of sequence tags; (b) identifying multiple polymorphisms in the plurality of sequence tags, wherein the multiple polymorphisms are associated with the number of disorders; and (c) associating the multiple polymorphisms with the first and/or second genome in the mixture, wherein the mixture is unenriched for the multiple polymorphisms. In some embodiments, step (b) comprises counting sequence tags mapped to the multiple polymorphisms. Alternatively, step (b) comprises genotyping the second genome in a sample that is substantially free of the first genome. In some embodiments the disorder is a monogenic disorder. Alternatively, the disorder is a polygenic disorder. In other embodiments, the disorder is cancer. Sequencing is massively parallel sequencing of clonally amplified cfDNA molecules or of single cfDNA molecules. In some embodiments, sequencing is performed using massively parallel sequencing-by-synthesis with reversible dye terminators. In other embodiments, sequencing is performed using massively parallel sequencing-by-ligation. In some embodiments, the sample is a plasma sample.</div>
</li> <li> <para-num num="[0029]"> </para-num> <div class="description-line" id="p-0030" num="0029">In another embodiment, the method for determining the presence or absence of multiple disorders in a blood sample comprising a mixture of cfDNA of a first and a second genome, comprises: (a) whole genome sequencing at least a portion of the mixture of cfDNA, thereby providing a plurality of sequence tags; (b) identifying multiple polymorphisms in the plurality of sequence tags, wherein the multiple polymorphisms are associated with the number of disorders; and (c) associating the multiple polymorphisms with the first and/or second genome in the mixture, wherein the mixture is unenriched for the multiple polymorphisms. The first genome is a fetal genome and the second genome is a maternal genome. Optionally, the method further comprises determining the presence or absence of the multiple polymorphisms in a cellular paternal genome. Sequencing is massively parallel sequencing of clonally amplified cfDNA molecules or of single cfDNA molecules. In some embodiments, sequencing is performed using massively parallel sequencing-by-synthesis with reversible dye terminators. In other embodiments, sequencing is performed using massively parallel sequencing-by-ligation. In some embodiments, the sample is a plasma sample.</div>
</li> <li> <para-num num="[0030]"> </para-num> <div class="description-line" id="p-0031" num="0030">In another embodiment, the method for determining the presence or absence of multiple disorders in a blood sample comprising a mixture of cfDNA of a first and a second genome, comprises: (a) whole genome sequencing at least a portion of the mixture of cfDNA, thereby providing a plurality of sequence tags; (b) identifying multiple polymorphisms in the plurality of sequence tags, wherein the multiple polymorphisms are associated with the number of disorders; and (c) associating the multiple polymorphisms with the first and/or second genome in the mixture, wherein the mixture is unenriched for the multiple polymorphisms. The method further comprises comparing the sequence of the plurality of tags to the sequence of multiple reference polymorphisms, thereby identifying the multiple polymorphisms in the mixture of cfDNA. The first genome is a fetal genome and the second genome is a maternal genome. Optionally, the method further comprises determining the presence or absence of the multiple polymorphisms in a cellular paternal genome. Sequencing is massively parallel sequencing of clonally amplified cfDNA molecules or of single cfDNA molecules. In some embodiments, sequencing is performed using massively parallel sequencing-by-synthesis with reversible dye terminators. In other embodiments, sequencing is performed using massively parallel sequencing-by-ligation. In some embodiments, the sample is a plasma sample.</div>
</li> <li> <para-num num="[0031]"> </para-num> <div class="description-line" id="p-0032" num="0031">In another embodiment, the method for determining the presence or absence of multiple disorders in a blood sample comprising a mixture of cfDNA of a first and a second genome, comprises: (a) genome sequencing at least a portion of the mixture of cfDNA, thereby providing a plurality of sequence tags; (b) identifying multiple polymorphisms in the plurality of sequence tags, wherein the multiple polymorphisms are associated with the number of disorders; and (c) associating the multiple polymorphisms with the first and/or second genome in the mixture, wherein the mixture is unenriched for the multiple polymorphisms. In some embodiments, step (b) comprises counting sequence tags mapped to the multiple polymorphisms. Alternatively, step (b) comprises genotyping the second genome in a sample that is substantially free of the first genome. The first genome is a fetal genome and the second genome is a maternal genome. Optionally, the method further comprises determining the presence or absence of the multiple polymorphisms in a cellular paternal genome. Sequencing is massively parallel sequencing of clonally amplified cfDNA molecules or of single cfDNA molecules. In some embodiments, sequencing is performed using massively parallel sequencing-by-synthesis with reversible dye terminators. In other embodiments, sequencing is performed using massively parallel sequencing-by-ligation. In some embodiments, the sample is a plasma sample.</div>
</li> <li> <para-num num="[0032]"> </para-num> <div class="description-line" id="p-0033" num="0032">In another embodiment, the invention provides a method for determining the presence or absence of multiple fetal disorders in a maternal blood sample comprising a mixture of fetal and maternal cfDNA, the method comprising: (a) whole genome sequencing at least a portion of the mixture of fetal and maternal cfDNA, thereby obtaining a plurality of sequence tags; (b) identifying multiple fetal polymorphisms in the plurality of sequence tags, thereby determining multiple fetal haplotypes associated with the multiple fetal disorders; and (c) determining the presence or absence of the at least one fetal disorder. Optionally, the method further comprises sequencing paternal cellular DNA. Sequencing is massively parallel sequencing of clonally amplified cfDNA molecules or of single cfDNA molecules. In some embodiments, sequencing is performed using massively parallel sequencing-by-synthesis with reversible dye terminators. In other embodiments, sequencing is performed using massively parallel sequencing-by-ligation. In some embodiments, the sample is a plasma sample.</div>
</li> <li> <para-num num="[0033]"> </para-num> <div class="description-line" id="p-0034" num="0033">In another embodiment, the method for determining the presence or absence of multiple fetal disorders in a maternal blood sample comprising a mixture of fetal and maternal cfDNA, comprises: (a) whole genome sequencing at least a portion of the mixture of fetal and maternal cfDNA, thereby obtaining a plurality of sequence tags; (b) identifying multiple fetal polymorphisms in the plurality of sequence tags, thereby determining multiple fetal haplotypes associated with the multiple fetal disorders; and (c) determining the presence or absence of the at least one fetal disorder. Optionally, the method further comprises sequencing paternal cellular DNA. In some embodiments, step (b) comprises genotyping maternal cellular DNA thereby identifying multiple polymorphisms in the maternal cellular DNA. Alternatively, step (b) comprises counting tags mapped to the fetal and maternal polymorphisms in the mixture. Sequencing is massively parallel sequencing of clonally amplified cfDNA molecules or of single cfDNA molecules. In some embodiments, sequencing is performed using massively parallel sequencing-by-synthesis with reversible dye terminators. In other embodiments, sequencing is performed using massively parallel sequencing-by-ligation. In some embodiments, the sample is a plasma sample.</div>
</li> <li> <para-num num="[0034]"> </para-num> <div class="description-line" id="p-0035" num="0034">In another embodiment, the method for determining the presence or absence of multiple fetal disorders in a maternal blood sample comprising a mixture of fetal and maternal cfDNA, comprises: (a) whole genome sequencing at least a portion of the mixture of fetal and maternal cfDNA, thereby obtaining a plurality of sequence tags; (b) identifying multiple fetal polymorphisms in the plurality of sequence tags, thereby determining multiple fetal haplotypes associated with the multiple fetal disorders; and (c) determining the presence or absence of the at least one fetal disorder. Optionally, the method further comprises sequencing paternal cellular DNA. Each of the multiple fetal polymorphisms comprises at least one SNP, a tandem SNP or an STR. In some embodiments, the at least one SNP is a tag SNP. In other embodiments, the at least one STR is a tag STR. Sequencing is massively parallel sequencing of clonally amplified cfDNA molecules or of single cfDNA molecules. In some embodiments, sequencing is performed using massively parallel sequencing-by-synthesis with reversible dye terminators. In other embodiments, sequencing is performed using massively parallel sequencing-by-ligation. In some embodiments, the sample is a plasma sample.</div>
</li> <li> <para-num num="[0035]"> </para-num> <div class="description-line" id="p-0036" num="0035">In another embodiment, the method for determining the presence or absence of multiple fetal disorders in a maternal blood sample comprising a mixture of fetal and maternal cfDNA, comprises: (a) whole genome sequencing at least a portion of the mixture of fetal and maternal cfDNA, thereby obtaining a plurality of sequence tags; (b) identifying multiple fetal polymorphisms in the plurality of sequence tags, thereby determining multiple fetal haplotypes associated with the multiple fetal disorders; and (c) determining the presence or absence of the at least one fetal disorder, wherein step (b) comprises genotyping maternal cellular DNA thereby identifying multiple polymorphisms in the maternal cellular DNA. Each of the multiple fetal polymorphisms comprises at least one SNP, a tandem SNP or an STR. In some embodiments, the at least one SNP is a tag SNP. In other embodiments, the at least one STR is a tag STR. Optionally, the method further comprises sequencing paternal cellular DNA. Sequencing is massively parallel sequencing of clonally amplified cfDNA molecules or of single cfDNA molecules. In some embodiments, sequencing is performed using massively parallel sequencing-by-synthesis with reversible dye terminators. In other embodiments, sequencing is performed using massively parallel sequencing-by-ligation. In some embodiments, the sample is a plasma sample.</div>
</li> <li> <para-num num="[0036]"> </para-num> <div class="description-line" id="p-0037" num="0036">In another embodiment, the method for determining the presence or absence of multiple fetal disorders in a maternal blood sample comprising a mixture of fetal and maternal cfDNA, comprises: (a) whole genome sequencing at least a portion of the mixture of fetal and maternal cfDNA, thereby obtaining a plurality of sequence tags; (b) identifying multiple fetal polymorphisms in the plurality of sequence tags, thereby determining multiple fetal haplotypes associated with the multiple fetal disorders; and (c) determining the presence or absence of the at least one fetal disorder, wherein step (b) comprises counting tags mapped to the fetal and maternal polymorphisms in the mixture. Each of the multiple fetal polymorphisms comprises at least one SNP, a tandem SNP or an STR. In some embodiments, the at least one SNP is a tag SNP. In other embodiments, the at least one STR is a tag STR. Optionally, the method further comprises sequencing paternal cellular DNA. Sequencing is massively parallel sequencing of clonally amplified cfDNA molecules or of single cfDNA molecules. In some embodiments, sequencing is performed using massively parallel sequencing-by-synthesis with reversible dye terminators. In other embodiments, sequencing is performed using massively parallel sequencing-by-ligation. In some embodiments, the sample is a plasma sample.</div>
</li> <li> <para-num num="[0037]"> </para-num> <div class="description-line" id="p-0038" num="0037">Although the examples herein concern humans and the language is primarily directed to human concerns, the concept of this invention is applicable to genomes from any plant or animal.</div>
</li> <heading id="h-0005">INCORPORATION BY REFERENCE</heading>
<li> <para-num num="[0038]"> </para-num> <div class="description-line" id="p-0039" num="0038">All patents, patent applications, and other publications, including all sequences disclosed within these references, referred to herein are expressly incorporated by reference, to the same extent as if each individual publication, patent or patent application was specifically and individually indicated to be incorporated by reference. All documents cited are, in relevant part, incorporated herein by reference. However, the citation of any document is not to be construed as an admission that it is prior art with respect to the present invention.</div>
</li> <description-of-drawings>
<heading id="h-0006">BRIEF DESCRIPTION OF THE DRAWINGS</heading>
<li> <para-num num="[0039]"> </para-num> <div class="description-line" id="p-0040" num="0039">The novel features of the invention are set forth with particularity in the appended claims. A better understanding of the features and advantages of the present invention will be obtained by reference to the following detailed description that sets forth illustrative embodiments, in which the principles of the invention are utilized, and the accompanying drawings of which:</div>
</li> <li> <para-num num="[0040]"> </para-num> <div class="description-line" id="p-0041" num="0040">The novel features of the invention are set forth with particularity in the appended claims. A better understanding of the features and advantages of the present invention will be obtained by reference to the following detailed description that sets forth illustrative embodiments, in which the principles of the invention are utilized, and the accompanying drawings of which:</div>
</li> <li> <para-num num="[0041]"> </para-num> <div class="description-line" id="p-0042" num="0041"> <figref idrefs="DRAWINGS">FIG. 1</figref> shows the percent coverage of individual chromosomes determined by whole genome sequencing of a combination of maternal samples comprising a mixture of fetal and maternal cfDNA as a function of depth. A maximum depth of 1 represents 100% coverage of each chromosome.</div>
</li> <li> <para-num num="[0042]"> </para-num> <div class="description-line" id="p-0043" num="0042"> <figref idrefs="DRAWINGS">FIG. 2</figref> shows the number of SNPs identified from the sequencing data shown in <figref idrefs="DRAWINGS">FIG. 1</figref> that were mapped to each of the chromosomes. The SNPs that were counted were not differentiated between the maternal and fetal genome i.e. all counted SNPs had the same polymorphic sequence.</div>
</li> <li> <para-num num="[0043]"> </para-num> <div class="description-line" id="p-0044" num="0043"> <figref idrefs="DRAWINGS">FIG. 3</figref> shows the predicted genome coverage assuming the presence of at least one genome in the sample and using the whole genome sequencing data shown in <figref idrefs="DRAWINGS">FIG. 1</figref>. Cumulative (●) and incremental (▪) genome coverage are shown.</div>
</li> <li> <para-num num="[0044]"> </para-num> <div class="description-line" id="p-0045" num="0044"> <figref idrefs="DRAWINGS">FIG. 4</figref> is a bar diagram showing the identification of fetal and maternal polymorphic sequences (SNPs) used to determine fetal fraction in a test sample. The total number of sequence reads (Y-axis) mapped to the SNP sequences identified by rs numbers (X-axis), and the relative level of fetal nucleic acids (*) are shown.</div>
</li> <li> <para-num num="[0045]"> </para-num> <div class="description-line" id="p-0046" num="0045"> <figref idrefs="DRAWINGS">FIG. 5</figref> is a flowchart outlining alternative methods used for determining fetal fraction by massively parallel sequencing as described in Examples 6-9.</div>
</li> <li> <para-num num="[0046]"> </para-num> <div class="description-line" id="p-0047" num="0046"> <figref idrefs="DRAWINGS">FIG. 6</figref> shows sequence doses (Y-axis) for a segment of human chromosome 11 (81000082-103000103 bp) determined from sequencing cfDNA extracted from a set of 7 qualified samples (0) obtained and 1 test sample (♦) from pregnant human subjects. A sample from a subject carrying a fetus with a partial aneuploidy of chromosome 11 (♦) was identified.</div>
</li> </description-of-drawings>
<heading id="h-0007">DETAILED DESCRIPTION OF THE INVENTION</heading>
<li> <para-num num="[0047]"> </para-num> <div class="description-line" id="p-0048" num="0047">The present invention relates to methods comprising whole genome sequencing for identifying polymorphisms in samples comprising mixtures of genomes, and for determining and/or monitoring the presence or absence of disorders associated with the identified polymorphisms.</div>
</li> <li> <para-num num="[0048]"> </para-num> <div class="description-line" id="p-0049" num="0048">Unless otherwise indicated, the practice of the present invention involves conventional techniques commonly used in molecular biology, microbiology, protein purification, protein engineering, protein and DNA sequencing, and recombinant DNA fields, which are within the skill of the art. Such techniques are known to those of skill in the art and are described in numerous standard texts and reference works. All patents, patent applications, articles and publications mentioned herein are hereby expressly incorporated herein by reference in their entirety.</div>
</li> <li> <para-num num="[0049]"> </para-num> <div class="description-line" id="p-0050" num="0049">Numeric ranges are inclusive of the numbers defining the range. It is intended that every maximum numerical limitation given throughout this specification includes every lower numerical limitation, as if such lower numerical limitations were expressly written herein. Every minimum numerical limitation given throughout this specification will include every higher numerical limitation, as if such higher numerical limitations were expressly written herein. Every numerical range given throughout this specification will include every narrower numerical range that falls within such broader numerical range, as if such narrower numerical ranges were all expressly written herein.</div>
</li> <li> <para-num num="[0050]"> </para-num> <div class="description-line" id="p-0051" num="0050">The headings provided herein are not limitations of the various aspects or embodiments of the invention which can be had by reference to the Specification as a whole. Accordingly, as indicated above, the terms defined immediately below are more fully defined by reference to the specification as a whole.</div>
</li> <li> <para-num num="[0051]"> </para-num> <div class="description-line" id="p-0052" num="0051">Unless defined otherwise herein, all technical and scientific terms used herein have the same meaning as commonly understood by one of ordinary skill in the art to which this invention belongs. Various scientific dictionaries that include the terms included herein are well known and available to those in the art. Although any methods and materials similar or equivalent to those described herein find use in the practice or testing of the present invention, some preferred methods and materials are described. Accordingly, the terms defined immediately below are more fully described by reference to the Specification as a whole. It is to be understood that this invention is not limited to the particular methodology, protocols, and reagents described, as these may vary, depending upon the context they are used by those of skill in the art.</div>
</li> <heading id="h-0008">Definitions</heading>
<li> <para-num num="[0052]"> </para-num> <div class="description-line" id="p-0053" num="0052">As used herein, the singular terms “a”, “an,” and “the” include the plural reference unless the context clearly indicates otherwise. Unless otherwise indicated, nucleic acids are written left to right in 5′ to 3′ orientation and amino acid sequences are written left to right in amino to carboxy orientation, respectively. The term “assessing” herein refers to characterizing the status of a chromosomal aneuploidy by one of three types of calls: “normal”, “affected”, and “no-call”. For example, in the presence of trisomy the “normal” call is determined by the value of a parameter e.g. a test chromosome dose that is below a user-defined threshold of reliability, the “affected” call is determined by a parameter e.g. a test chromosome dose, that is above a user-defined threshold of reliability, and the “no-call” result is determined by a parameter e.g. a test chromosome dose, that lies between the a user-defined thresholds of reliability for making a “normal” or an “affected” call.</div>
</li> <li> <para-num num="[0053]"> </para-num> <div class="description-line" id="p-0054" num="0053">The terms “polymorphism” and “polymorphic sequences” are herein used interchangeably to refer to the occurrence of two or more genetically determined alternative sequences or alleles in a population. Each divergent sequence is termed an allele, and can be part of a gene or located within an intergenic or non-genic sequence. A diallelic polymorphism has two alleles, and a triallelic polymorphism has three alleles. Diploid organisms can contain two alleles and may be homozygous or heterozygous for allelic forms. The first identified allelic form is arbitrarily designated the reference form or allele; other allelic forms are designated as alternative or variant alleles. The most frequently occurring allelic form in a selected population is typically referred to as the wild-type form. Polymorphisms encompass sequence differences that include single nucleotide polymorphisms (SNPs), tandem SNPs, small-scale multi-base deletions or insertions, called IN-DELS (also called deletion insertion polymorphisms or DIPs), Multi-Nucleotide Polymorphisms (MNPs), Short Tandem Repeats (STRs), restriction fragment length polymorphism (RFLP), deletions, including microdeletions, insertions, including microinsertions, duplications, inversions, translocations, multiplications, complex multi-site variants, copy number variations (CNV), and polymorphisms comprising any other change of sequence in a chromosome. Differences in genomic sequences include combinations of polymorphisms. For example, polymorphisms can encompass the combination of one or more SNPs and one or more STR. Polymorphisms can be indicative, informative, or both. For example, indicative polymorphisms indicate the presence of fetal cell-free DNA in a maternal sample. Informative polymorphisms yield information about the fetus—for example, the presence or absence of a disorder, abnormality, or any other biological information such as the stage of gestation or gender.</div>
</li> <li> <para-num num="[0054]"> </para-num> <div class="description-line" id="p-0055" num="0054">The term “copy number variation” herein refers to variation in the number of copies of a nucleic acid sequence that is 1 kb or larger present in a test sample in comparison with the copy number of the nucleic acid sequence present in a qualified sample. A “copy number variant” refers to the 1 kb or larger sequence of nucleic acid in which copy-number differences are found by comparison of a sequence of interest in test sample with that present in a qualified sample. Copy number variants/variations include deletions, including microdeletions, insertions, including microinsertions, duplications, multiplications, inversions, translocations and complex multi-site variants. CNV encompass chromosomal aneuploidies and partial aneuplodies.</div>
</li> <li> <para-num num="[0055]"> </para-num> <div class="description-line" id="p-0056" num="0055">The term “single nucleotide polymorphism (SNP)” refers to a single base (nucleotide) polymorphism in a DNA sequence among individuals in a population. A SNP may be present within coding sequences of genes, non-coding regions of genes, or in the intergenic regions between genes. SNPs within a coding sequence will not necessarily change the amino acid sequence of the protein that is produced, due to degeneracy of the genetic code. A SNP in which both forms lead to the same polypeptide sequence is termed “synonymous” (sometimes called a silent mutation)—if a different polypeptide sequence is produced they are “nonsynonymous”. A nonsynonymous change may either be missense or “nonsense”, where a missense change results in a different amino acid, while a nonsense change results in a premature stop codon. SNPs can also arise from a deletion of a nucleotide or an insertion of a nucleotide relative to a reference allele. Single nucleotide polymorphisms (SNPs) are positions at which two alternative bases occur at appreciable frequency (&gt;1%) in the human population, and are the most common type of human genetic variation.</div>
</li> <li> <para-num num="[0056]"> </para-num> <div class="description-line" id="p-0057" num="0056">The term “tandem SNPs” herein refers to two or more SNPs that are present within a polymorphic target nucleic acid sequence.</div>
</li> <li> <para-num num="[0057]"> </para-num> <div class="description-line" id="p-0058" num="0057">The term “short tandem repeat” or “STR” herein refers to a class of polymorphisms that occurs when a pattern of two or more nucleotides are repeated and the repeated sequences are directly adjacent to each other. The pattern can range in length from 2 to 10 base pairs (bp) (for example (CATG)n in a genomic region) and is typically in the non-coding intron region. By examining several STR loci and counting how many repeats of a specific STR sequence there are at a given locus, it is possible to create a unique genetic profile of an individual.</div>
</li> <li> <para-num num="[0058]"> </para-num> <div class="description-line" id="p-0059" num="0058">The term “sample” herein refers to a biological specimen comprising a mixture of two or more genomes. Biological specimens include tissue samples, biological fluid samples, or cell samples.</div>
</li> <li> <para-num num="[0059]"> </para-num> <div class="description-line" id="p-0060" num="0059">The term “maternal sample” unless otherwise specified, herein refers to a biological sample obtained from a pregnant subject that comprises a mixture of fetal and maternal nucleic acids e.g. cfDNA.</div>
</li> <li> <para-num num="[0060]"> </para-num> <div class="description-line" id="p-0061" num="0060">As used herein, the terms “aligned”, “alignment”, or “aligning” refer to one or more sequences that are identified as a match in terms of the order of their nucleic acid molecules to a known sequence from a reference genome. Such alignment can be done manually or by a computer algorithm, examples including the Efficient Local Alignment of Nucleotide Data (ELAND) computer program distributed as part of the Illumina Genomics Analysis pipeline. The matching of a sequence read in aligning can be a 100% sequence match or less than 100% (non-perfect match).</div>
</li> <li> <para-num num="[0061]"> </para-num> <div class="description-line" id="p-0062" num="0061">The term “sequence tag” is herein used interchangeably with the term “mapped sequence tag” to refer to a sequence read that has been specifically assigned i.e. mapped, to a larger sequence e.g. a reference genome, by alignment. Mapped sequence tags are uniquely mapped to a reference genome i.e. they are assigned to a single location to the reference genome. Tags that can be mapped to more than one location on a reference genome i.e. tags that do not map uniquely, are not included in the analysis.</div>
</li> <li> <para-num num="[0062]"> </para-num> <div class="description-line" id="p-0063" num="0062">The term “read” refers to a DNA sequence of sufficient length (e.g., at least about 30 bp) that can be used to identify a larger sequence or region, e.g. that can be aligned and specifically assigned to a chromosome or genomic region or gene.</div>
</li> <li> <para-num num="[0063]"> </para-num> <div class="description-line" id="p-0064" num="0063">The term “biological fluid” herein refers to a liquid taken from a biological source and includes, for example, blood, serum, plasma, sputum, lavage fluid, cerebrospinal fluid, urine, semen, sweat, tears, saliva, and the like. As used herein, the terms “blood,” “plasma” and “serum” expressly encompass fractions or processed portions thereof. Similarly, where a sample is taken from a biopsy, swab, smear, etc., the “sample” expressly encompasses a processed fraction or portion derived from the biopsy, swab, smear, etc.</div>
</li> <li> <para-num num="[0064]"> </para-num> <div class="description-line" id="p-0065" num="0064">The term “transrenal-DNA (Tr-DNA)” herein refers to DNA that has crosses the kidney barrier and is found in urine.</div>
</li> <li> <para-num num="[0065]"> </para-num> <div class="description-line" id="p-0066" num="0065">The term “mixture” is used herein to refer to a mixture of nucleic acids e.g. DNA, that are present in a sample and are derived from two or more genomes. A non-limiting example of a mixture of nucleic acids is a maternal blood sample that comprises a mixture of fetal and maternal genomic cell-free DNA.</div>
</li> <li> <para-num num="[0066]"> </para-num> <div class="description-line" id="p-0067" num="0066">The term “Whole Genome Sequencing (WGS)” herein refers to a process whereby the sequence of the entire genome of an organism, for example, humans, dogs, mice, viruses or bacteria can be determined. It is not necessary that the entire genome actually be sequenced. The WGS methods of the invention are those sequencing methods that when applied to a sample of genomic DNA are capable of obtaining the sequence of the entire genome. Whole genome sequencing can be performed using any NGS technology as described herein.</div>
</li> <li> <para-num num="[0067]"> </para-num> <div class="description-line" id="p-0068" num="0067">The term “sequencing” herein refers to a method for determining the nucleotide sequence of a polynucleotide e.g. genomic DNA. Preferably, sequencing methods include as non-limiting examples next generation sequencing (NGS) methods, (NGS) in which clonally amplified DNA templates or single DNA molecules are sequenced in a massively parallel fashion (e.g. as described in Volkerding et al. Clin Chem 55:641-658 [2009]; Metzker M Nature Rev 11:31-46 [2010]).</div>
</li> <li> <para-num num="[0068]"> </para-num> <div class="description-line" id="p-0069" num="0068">The term “at least as portion” herein refers to a fraction of the total genomic DNA present in a sample comprising a mixture of DNA from two different genomes that represents at least one genome equivalent. For example, “at least a portion” of cfDNA in a blood sample comprising a mixture of cfDNA derived from two different genomes refers the fraction of the total cfDNA in the mixture that represents at least one copy of one of the two genomes.</div>
</li> <li> <para-num num="[0069]"> </para-num> <div class="description-line" id="p-0070" num="0069">The term “genome equivalent” herein refers to the amount of DNA necessary to be present in a sample to guarantee that all genes will be present. This number increases with the total genome size of an organism and can be calculated by converting the size of a genome in base pairs to micrograms of DNA.</div>
</li> <li> <para-num num="[0070]"> </para-num> <div class="description-line" id="p-0071" num="0070">The term “cellular” when used in reference to a genome or genomic DNA refers to the genomic content of a cell that can be artificially extracted by methods known in the art.</div>
</li> <li> <para-num num="[0071]"> </para-num> <div class="description-line" id="p-0072" num="0071">The term “cell-free” when used in reference to a genome or genomic DNA refers to the genomic content of a cell that is naturally released from cells by biological process such as necrosis and apoptosis.</div>
</li> <li> <para-num num="[0072]"> </para-num> <div class="description-line" id="p-0073" num="0072">The term “sequence tag” is herein used interchangeably with the term “mapped sequence tag” to refer to a sequence read that has been specifically assigned i.e. mapped, to a larger sequence e.g. a reference genome, by alignment. Mapped sequence tags are uniquely mapped to a reference genome i.e. they are assigned to a single location to the reference genome. Tags that can be mapped to more than one location on a reference genome i.e. tags that do not map uniquely, are not included in the analysis.</div>
</li> <li> <para-num num="[0073]"> </para-num> <div class="description-line" id="p-0074" num="0073">The term “plurality” is used herein in reference to a number of sequence tags that is sufficient to identify sequence differences that exist between two genomes. Sequence differences include polymorphisms found in genomes among subjects of the same species e.g. fetal and maternal genomes. The number of sequence tags that constitutes the plurality of sequence tags depends on the size of the genomes and the sequencing technology used to identify the differences. The plurality of sequence tags sufficient to identify sequence differences between two genomes will be greater when using sequencing technology that results in shorter tags e.g. 36 base pair tags, than when using sequencing technology that provides 500 base pair long tags. For example, X 36 base pair sequence tags and Y 500 base pair sequence tags are needed respectively, to encompass all of the approximately 3×10<sup>9 </sup>base pairs of the human genome.</div>
</li> <li> <para-num num="[0074]"> </para-num> <div class="description-line" id="p-0075" num="0074">The term “enriched” is used herein in reference to one or more sequences present in a sample comprising a mixture of genomic DNA that have been amplified to augment the number of copies of the one or more sequences in the sample. For example, a sample comprising a mixture of genomic DNA can be enriched for specific polymorphic target sequences. Alternatively, a sample comprising a mixture of genomic DNA can be enriched non-specifically for sequences contained in the mixture e.g. all the genomic DNA sequences can be amplified to enrich the sample.</div>
</li> <li> <para-num num="[0075]"> </para-num> <div class="description-line" id="p-0076" num="0075">The terms “polymorphic target nucleic acid”, “polymorphic sequence”, “polymorphic target nucleic acid sequence” and “polymorphic nucleic acid” are used interchangeably herein to refer to a nucleic acid sequence e.g. a DNA sequence, that comprises one or more polymorphic sites i.e. polymorphisms.</div>
</li> <li> <para-num num="[0076]"> </para-num> <div class="description-line" id="p-0077" num="0076">The term “sequence tag density” herein refers to the number of sequence reads that are mapped to a reference genome sequence.</div>
</li> <li> <para-num num="[0077]"> </para-num> <div class="description-line" id="p-0078" num="0077">The term “sequence dose” herein refers to a parameter that relates the sequence tag density of a sequence of interest to the tag density of a normalizing sequence. A “test sequence dose” is a parameter that relates the sequence tag density of a sequence of interest determined in a test sample. Similarly, a “qualified sequence dose” is a parameter that relates the sequence tag density of a sequence of interest to that of a normalizing sequence determined in a qualified sample.</div>
</li> <li> <para-num num="[0078]"> </para-num> <div class="description-line" id="p-0079" num="0078">The term “normalizing sequence” herein refers to a sequence that displays a variability in the number of sequence tags that are mapped to it among samples and sequencing runs that best approximates that of the sequence of interest for which it is used as a normalizing parameter, and that can best differentiate an affected sample from one or more unaffected samples. A “normalizing chromosome” is an example of a “normalizing sequence”.</div>
</li> <li> <para-num num="[0079]"> </para-num> <div class="description-line" id="p-0080" num="0079">The term “sequence of interest” herein refers to a nucleic acid sequence that is associated with a difference in sequence representation in healthy versus diseased individuals. A “qualified sequence of interest” is a sequence of interest in a qualified sample. A “test sequence of interest” is a sequence of interest in a test sample.</div>
</li> <li> <para-num num="[0080]"> </para-num> <div class="description-line" id="p-0081" num="0080">The term “qualified sample” herein refers to a sample comprising a mixture of nucleic acids that are present in a sample that is normal e.g. not aneuploid, for the sequence of interest.</div>
</li> <li> <para-num num="[0081]"> </para-num> <div class="description-line" id="p-0082" num="0081">The term “test sample” herein refers to a sample comprising a mixture of nucleic acids comprising at least one nucleic acid sequence that is suspected of having undergone variation. Nucleic acids present in a test sample are referred to as “test nucleic acids”.</div>
</li> <li> <para-num num="[0082]"> </para-num> <div class="description-line" id="p-0083" num="0082">The term “unenriched” is used herein in reference to a mixture of genomic DNA that has not been specifically or non-specifically enriched for sequence(s) comprised in the mixture.</div>
</li> <li> <para-num num="[0083]"> </para-num> <div class="description-line" id="p-0084" num="0083">As used herein, the term “enriched library” herein refers to a sequencing library comprising amplified polymorphic target nucleic acid sequences. An example of an enriched library is a sequencing library comprising naturally-occurring cfDNA sequences and amplified target nucleic acid sequences. An “unenriched library” herein refers to a sequencing library that does not comprise i.e. a library generated from naturally-occurring cfDNA sequences. A “polymorphic target nucleic acid library” is a library generated from amplified target nucleic acids”.</div>
</li> <li> <para-num num="[0084]"> </para-num> <div class="description-line" id="p-0085" num="0084">As used herein, the term “purified” refers to material (e.g., an isolated polynucleotide) that is in a relatively pure state, e.g., at least about 80% pure, at least about 85% pure, at least about 90% pure, at least about 95% pure, at least about 98% pure, or even at least about 99% pure.</div>
</li> <li> <para-num num="[0085]"> </para-num> <div class="description-line" id="p-0086" num="0085">As used herein, the term “substantially cell free” encompasses preparations of the desired sample from which components that are normally associated with it are removed. For example, a plasma sample is rendered essentially cell free by removing blood cells e.g. red cells, which are normally associated with it.</div>
</li> <li> <para-num num="[0086]"> </para-num> <div class="description-line" id="p-0087" num="0086">The term “selective” when used in reference to whole genome amplification refers to the amplification of one of the two genomes present in the sample. For example, selective non-specific enrichment of genomic DNA in a sample comprising DNA molecules derived from two genomes refers to the whole genome enrichment of a selected i.e. one of two, genomes.</div>
</li> <li> <para-num num="[0087]"> </para-num> <div class="description-line" id="p-0088" num="0087">The term “whole genome amplification” herein refers to a process whereby DNA sequences present in a sample are amplified to provide multiple copies of the genome that the sequences represent.</div>
</li> <li> <para-num num="[0088]"> </para-num> <div class="description-line" id="p-0089" num="0088">The term “polymorphic reference sequences” herein refers to known variant sequences that predominate in a natural population. Polymorphic reference sequences can also refer to variant sequences present in a genome present in a sample comprising a mixture of two or more genomes, to which polymorphisms of other genomes are compared.</div>
</li> <li> <para-num num="[0089]"> </para-num> <div class="description-line" id="p-0090" num="0089">The term “associated” is used herein in reference to sequence differences e.g. polymorphisms, that are known to be linked to a disorder. The association of one or more polymorphisms with a disorder can result in the disorder or can represent the genetic predisposition i.e. risk, of developing the disorder.</div>
</li> <li> <para-num num="[0090]"> </para-num> <div class="description-line" id="p-0091" num="0090">The terms “disorder” and “genetic disorders” are used herein interchangeably to refer to conditions or diseases that are caused in whole or in part by alterations in genes or chromosomes. The alterations in genes or chromosomes can be inherited, or can be the result of external factors such as infectious diseases. Disorders encompass single gene disorders including autosomal dominant, autosomal recessive, X-linked dominant, X-linked recessive, Y-linked, and polygenic disorders.</div>
</li> <li> <para-num num="[0091]"> </para-num> <div class="description-line" id="p-0092" num="0091">The term “unaffected” when used in reference to a cell refers to a cell having a genome devoid of variations known or suspected to be associated with a genetic disorder. Conversely, an “affected” cell is a cell having a genome comprising one or more variations e.g. polymorphisms, that are known or suspected to be associated with a genetic disorder.</div>
</li> <li> <para-num num="[0092]"> </para-num> <div class="description-line" id="p-0093" num="0092">The term “allele” herein refers to a sequence variant of a genetic sequence. For purposes of this application, alleles can but need not be located within a gene sequence. Alleles can be identified with respect to one or more polymorphic positions such as SNPs, while the rest of the gene sequence can remain unspecified. For example, an allele may be defined by the nucleotide present at a single SNP, or by the nucleotides present at a plurality of SNPs.</div>
</li> <li> <para-num num="[0093]"> </para-num> <div class="description-line" id="p-0094" num="0093">The term “monogenic disorder” herein refers to diseases that result from modifications in a single gene occurring in all cells of the body, and can be classified as dominant, recessive, X-linked, Y-linked and mitochondrial diseases. Monogenic disorders include pure genetic diseases that are caused by a single error in a single gene in the human DNA.</div>
</li> <li> <para-num num="[0094]"> </para-num> <div class="description-line" id="p-0095" num="0094">The term “polygenic disorder” herein refers to a genetic disorder resulting from the combined action of alleles of more than one gene (e.g., heart disease, diabetes, and some cancers). Although such disorders are inherited, they depend on the simultaneous presence of several alleles; thus the hereditary patterns usually are more complex than those of single-gene disorders.</div>
</li> <li> <para-num num="[0095]"> </para-num> <div class="description-line" id="p-0096" num="0095">The term “haplotype” refers to a DNA sequence comprising one or more polymorphisms of interest contained on a subregion of a single chromosome of an individual. The polymorphisms of a haplotype can be of the same type e.g. all SNPs, or can be a combination of two or more types of polymorphisms e.g. a combinations of SNPs and STRs. A haplotype can refer to a set of polymorphisms in a single gene, an intergenic sequence, or in larger sequences including both gene and intergenic sequences, e.g., a collection of genes, or of genes and intergenic sequences. For example, a haplotype can refer to a set of polymorphisms in the regulation of complement activation (RCA) locus, which includes gene sequences for complement factor H(CFH), FHR3, FHR1, FHR4, FHR2, FHR5, and F13B and intergenic sequences (i.e., intervening intergenic sequences, upstream sequences, and downstream sequences that are in linkage disequilibrium with polymorphisms in the genic region). A haplotype, for instance, can be a set of maternally inherited alleles, or a set of paternally inherited alleles, at any locus.</div>
</li> <li> <para-num num="[0096]"> </para-num> <div class="description-line" id="p-0097" num="0096">The term “haplotyping” herein refers to a process for determining one or more haplotypes in an individual and includes use of family pedigrees, molecular techniques and/or statistical inference. Preferably, haplotypes are determined by sequencing using next generation sequencing technologies.</div>
</li> <li> <para-num num="[0097]"> </para-num> <div class="description-line" id="p-0098" num="0097">The term “haplotype block” herein refers to sites of closely located SNPs which are inherited in blocks. A haplotype block includes a group of SNP locations that do not appear to recombine independently and that can be grouped together. Regions corresponding to blocks have a few common haplotypes which account for a large proportion of chromosomes. Identification of haplotype blocks is a way of examining the extent of linkage disequilibrium (LD) in the genome. The “Hap-Map” project (see the internet at the Hap-Map website) describes the mapping of haplotype blocks in the human genome.</div>
</li> <li> <para-num num="[0098]"> </para-num> <div class="description-line" id="p-0099" num="0098">The term “tag polymorphism” herein refers to a polymorphism that by itself or in combination with additional polymorphisms indicates the presence of a specific haplotype, or of one member of a group of haplotypes. Examples of tag polymorphisms include without limitation tag SNPs and tag STRs. The haplotype or haplotypes can indicate a genetic factor is associated with risk for disease, thus for example, a tag SNP or combination of tag SNPs indicates the presence or absence of risk factors for disease. A “tag SNP” and a “tag STR” are a representative single nucleotide polymorphism (SNP) and a representative STR, respectively, in a region of the genome with high linkage disequilibrium (the non-random association of alleles at two or more loci) that is associated with a disease. Detecting the presence of at least one tag SNP and/or STR means detecting the presence of a risk allele of a tag SNP and/or STR.</div>
</li> <li> <para-num num="[0099]"> </para-num> <div class="description-line" id="p-0100" num="0099">The term “linkage” herein refers to an association of two or more loci at positions on the same chromosome, such that recombination between the two loci is reduced to a proportion significantly less than 50%. The term linkage can also be used in reference to the association between one or more loci and a trait if an allele (or alleles) and the trait, or absence thereof, are observed together in significantly greater than 50% of occurrences. A linkage group is a set of loci, in which all members are linked either directly or indirectly to all other members of the set.</div>
</li> <li> <para-num num="[0100]"> </para-num> <div class="description-line" id="p-0101" num="0100">The term “Linkage Disequilibrium” herein refers to the co-occurrence of two genetic loci (e.g., markers) at a frequency greater than expected for independent loci based on the allele frequencies. Linkage disequilibrium (LD) typically occurs when two loci are located close together on the same chromosome. When alleles of two genetic loci (such as a marker locus and a causal locus) are in strong LD, the allele observed at one locus (such as a marker locus) is predictive of the allele found at the other locus (for example, a causal locus contributing to a phenotypic trait). The linkage disequilibrium (LD) measure r.sup.2 (the squared correlation coefficient) can be used to evaluate how for example SNPs are related on a haplotype block.</div>
</li> <li> <para-num num="[0101]"> </para-num> <div class="description-line" id="p-0102" num="0101">The term “depth” when used in reference to sequencing of a chromosome, herein refers to the relative coverage of a chromosome by sequencing tags. A maximum depth of 1 indicates that 100% of the sequenced chromosome was covered by sequencing tags.</div>
</li> <li> <para-num num="[0102]"> </para-num> <div class="description-line" id="p-0103" num="0102">The term “coverage” when used in reference to the sequencing of a chromosome, herein refers to the amount of a chromosome sequence that is included in the sequencing tags that map to the chromosome. For example, a 1× coverage of a chromosome means that a sufficient number of sequence tags have been obtained to span the entire chromosome once.</div>
</li> <li> <para-num num="[0103]"> </para-num> <div class="description-line" id="p-0104" num="0103">The “subject” as used in the specification refers to any organism with at least diploid genome including, but not limited to worms, fish, insects, plants, murine and other mammals including domestic animals such as cows, horse, dogs, cats, and, most preferably humans.</div>
</li> <li> <para-num num="[0104]"> </para-num> <div class="description-line" id="p-0105" num="0104">The term “polymorphic target sequence” herein refers to a polymorphic sequence i.e. a sequence that comprises a polymorphism, that is selected for amplification in a sample comprising a mixture of two genomes. A “polymorphic target sequence of interest” herein refers to a selected polymorphic sequence that is known to be associated with a disorder.</div>
</li> <li> <para-num num="[0105]"> </para-num> <div class="description-line" id="p-0106" num="0105">The term “whole genome amplification” or “WGA” as used herein generally refers to a method for amplification of a limited DNA sample in a non-specific manner in order to generate a new sample that is indistinguishable from the original but with a higher DNA concentration. The ideal whole genome amplification technique would amplify a sample up to a microgram level while maintaining the original sequence representation. Degenerate oligonucleotide-primed PCR (DOP), primer extension PCR technique (PEP) and multiple displacement amplification (MDA) are examples of whole genome amplification methods.</div>
</li> <li> <para-num num="[0106]"> </para-num> <div class="description-line" id="p-0107" num="0106">Non-limiting examples of information that can be gained from identifying polymorphisms in samples comprising mixtures of two genomes according to the methods of the invention include (i) determining the presence of two genomes that are derived from two subjects of the same species, e.g. fetal and maternal genomes, (ii) determining the presence of two genomes each derived from the same subject, and contributed by unaffected cells and cells that have undergone malignant transformations e.g. cancer, respectively, and (iii) determining the presence of two genomes derived from the same subject, and contributed to the sample by the subject and donor cells e.g. transplant patient.</div>
</li> <li> <para-num num="[0107]"> </para-num> <div class="description-line" id="p-0108" num="0107">Accordingly, the present invention can be used for applications, including, but not limited to, identifying polymorphisms in maternal samples for determining the presence or absence of a genetic disease in a fetus, identifying polymorphisms in a sample from an individual to determine the presence or absence of a disease e.g. cancer, and/or the status of the disease, and identifying polymorphisms in a sample from an individual to determine the presence or absence of graft versus host disease (GVHD). Additional uses of the method include identifying polymorphisms in mixed samples to determine the contribution of individuals in forensic analyses.</div>
</li> <li> <para-num num="[0108]"> </para-num> <div class="description-line" id="p-0109" num="0108">Differences in genomic sequences encompass polymorphisms including-without limitation single nucleotide polymorphisms (SNPs), tandem SNPs, small-scale multi-base deletions or insertions, called IN-DELS (also called deletion insertion polymorphisms or DIPs), Multi-Nucleotide Polymorphisms (MNPs) Short Tandem Repeats (STRs), restriction fragment length polymorphism (RFLP), deletions, including microdeletions, insertions, including microinsertions, duplications, inversions, translocations, multiplications, complex multi-site variants, copy number variations (CNV), and polymorphisms comprising any other change of sequence in a chromosome. In some embodiments, polymorphisms that are identified according to the method of the invention are SNPs and/or STRs. SNP polymorphisms can be single SNP, tandem SNPs. Single SNPs include individual SNPs, and tag SNPs i.e. SNPs present in a haplotype, and/or a haplotype block. Differences in genomic sequences include combinations of polymorphisms. For example, differences in genomic sequences can be detected by comparison of a combination of polymorphic sequences comprising one or more SNPs and one or more STRs. Any polymorphic site that can be encompassed by the reads generated by the sequencing methods described herein can be used to identify polymorphic sequences in samples comprising DNA of different genomes. Polymorphic sequences useful for practicing the methods of the invention are available from a variety of publicly accessible databases, which are continuously expanding.</div>
</li> <li> <para-num num="[0109]"> </para-num> <div class="description-line" id="p-0110" num="0109">For example, useful databases include without limitation Human SNP Database at world wide web address wi.mit.edu, NCBI dbSNP Home Page at world wide web address ncbi.nlm.nih.gov, world wide web address lifesciences.perkinelmer.com, Celera Human SNP database at world wide web address celera.com, the SNP Database of the Genome Analysis Group (GAN) at world wide web address gan.iarc.fr, ATCC short tandem repeat (STR) database at world wide web address atcc.org, and the HapMap database at world wide web address hapmap.org.</div>
</li> <li> <para-num num="[0110]"> </para-num> <div class="description-line" id="p-0111" num="0110">Samples</div>
</li> <li> <para-num num="[0111]"> </para-num> <div class="description-line" id="p-0112" num="0111">Samples that are used for identifying polymorphisms in mixtures of two genomes comprise genomic DNA that is cellular or cell-free. Cellular DNA is derived from whole cells by manually or mechanically extracting the genomic DNA from whole cells of the same or of differing genetic compositions. Cellular DNA can be derived for example, from whole cells of the same genetic composition derived from one subject, from a mixture of whole cells of different subjects, or from a mixture of whole cells that differ in genetic composition that are derived from one subject. Methods for extracting genomic DNA from whole cells are known in the art, and differ depending upon the nature of the source. In some instances, it can be advantageous to fragment the cellular genomic DNA. Fragmentation can be random, or it can be specific, as achieved, for example, using restriction endonuclease digestion. Methods for random fragmentation are well known in the art, and include, for example, limited DNAse digestion, alkali treatment and physical shearing. In other embodiments, the sample nucleic acids are obtained as cellular genomic DNA, which is subjected to fragmentation into fragments of approximately 500 or more base pairs, and to which next generation sequencing (NGS) methods can be readily applied. In some embodiments, cellular genomic DNA is obtained to identify polymorphisms of a single genome. For example, cellular genomic DNA is obtained from a sample that contains only cells of the mother i.e. the sample is free of fetal genomic sequences. Identification of polymorphisms from a single genome e.g. maternal only genome, can be used in a comparison with polymorphisms identified in a mixture of fetal and maternal genomes to identify the fetal polymorphisms.</div>
</li> <li> <para-num num="[0112]"> </para-num> <div class="description-line" id="p-0113" num="0112">Cell-free DNA is genomic DNA that naturally occurs as a mixture of genomic fragments typically found in biological fluids e.g. blood, of a subject. The genomic mixture can be derived from cells that naturally rupture to release their genomic content by biological processes e.g. apoptosis. A sample of cfDNA can comprise cfDNA derived from a mixture of cells of different subjects of the same species, from a mixture of cells from one subject that differ in genetic composition, or from a mixture of cells from different species e.g. a subject. According to the method of the invention, DNA sequences of the genomes comprised in the mixture of cellular and/or cell-free DNA can be determined to identify the origin of the nucleic acids.</div>
</li> <li> <para-num num="[0113]"> </para-num> <div class="description-line" id="p-0114" num="0113">In some embodiments of the invention it is advantageous to obtain cell-free nucleic acids e.g. cell-free DNA (cfDNA). Cell-free nucleic acids, including cell-free DNA, can be obtained by various methods known in the art from biological samples including but not limited to plasma, serum and urine (Fan et al., Proc Natl Acad Sci 105:16266-16271 [2008]; Koide et al., Prenatal Diagnosis 25:604-607 [2005]; Chen et al., Nature Med. 2: 1033-1035 [1996]; Lo et al., Lancet 350: 485-487 [1997]; Botezatu et al., Clin Chem. 46: 1078-1084, 2000; and Su et al., J Mol. Diagn. 6: 101-107 [2004]). To separate cfDNA from cells, fractionation, centrifugation (e.g., density gradient centrifugation), DNA-specific precipitation, or high-throughput cell sorting and/or separation methods can be used. Commercially available kits for manual and automated separation of cfDNA are available (Roche Diagnostics, Indianapolis, Ind., Qiagen, Valencia, Calif., Macherey-Nagel, Duren, Del.).</div>
</li> <li> <para-num num="[0114]"> </para-num> <div class="description-line" id="p-0115" num="0114">The cfDNA present in the sample can be enriched specifically or non-specifically. Non-specific enrichment of sample DNA refers to the whole genome amplification of the genomic DNA fragments of the sample that can be used to increase the level of the sample DNA prior to identifying polymorphisms by NGS sequencing. Non-specific enrichment can be the selective enrichment of one of the two genomes present in the sample. For example, non-specific enrichment can be selective of the fetal genome in a maternal sample, which can be obtained by known methods to increase the relative proportion of fetal to maternal DNA in a sample. Alternatively, non-specific enrichment can be the non-selective amplification of both genomes present in the sample. For example, non-specific amplification can be of fetal and maternal DNA in a sample comprising a mixture of DNA from the fetal and maternal genomes. Methods for whole genome amplification are known in the art. Degenerate oligonucleotide-primed PCR (DOP), primer extension PCR technique (PEP) and multiple displacement amplification (MDA), are examples of whole genome amplification methods. In some embodiments, the sample comprising the mixture of cfDNA from different genomes is unenriched for cfDNA of the genomes present in the mixture. In other embodiments, the sample comprising the mixture of cfDNA from different genomes is non-specifically enriched for any one of the genomes present in the sample.</div>
</li> <li> <para-num num="[0115]"> </para-num> <div class="description-line" id="p-0116" num="0115">Specific enrichment refers to the enrichment of a genomic sample for specific sequences e.g. polymorphic target sequences, which are selected for amplification prior to sequencing the DNA sample. Polymorphic target nucleic acid sequences comprising e.g. STRs, can be selected, amplified and combined with the cellular or cell free DNA present in a mixture of genomes to simultaneously determine fetal fraction, and aneuploidy as provided in the examples. Whole genome sequencing for identifying polymorphisms associated with disorders as provided by the present method can be performed in conjunction with other prenatal diagnostic methods e.g. methods for determining aneuploidy. In some embodiments, the identification of polymorphisms according to the present method does not include specifically enriching polymorphic target sequences.</div>
</li> <li> <para-num num="[0116]"> </para-num> <div class="description-line" id="p-0117" num="0116">The sample comprising the mixture of nucleic acids to which the methods described herein are applied is a biological sample such as a tissue sample, a biological fluid sample, or a cell sample. In some embodiments, the mixture of nucleic acids is purified or isolated from the biological sample by any one of the known methods. A biological fluid sample includes, as non-limiting examples, blood, plasma, serum, sweat, tears, sputum, urine, sputum, ear flow, lymph, interstitial fluid, saliva, cerebrospinal fluid, ravages, bone marrow suspension, vaginal flow, transcervical lavage, brain fluid, ascites, milk, secretions of the respiratory, intestinal and genitourinary tracts, amniotic fluid and leukophoresis samples. In some embodiments, the sample is a sample that is easily obtainable by non-invasive procedures e.g. blood, plasma, serum, sweat, tears, sputum, urine, sputum, ear flow, and saliva. Preferably, the biological sample is a peripheral blood sample, or the plasma and serum fractions. In other embodiments, the biological sample is a swab or smear, a biopsy specimen, or a cell culture. In another embodiment, the sample is a mixture of two or more biological samples e.g. a biological sample can comprise two or more of a biological fluid sample, a tissue sample, and a cell culture sample. As used herein, the terms “blood,” “plasma” and “serum” expressly encompass fractions or processed portions thereof. Similarly, where a sample is taken from a biopsy, swab, smear, etc., the “sample” expressly encompasses a processed fraction or portion derived from the biopsy, swab, smear, etc.</div>
</li> <li> <para-num num="[0117]"> </para-num> <div class="description-line" id="p-0118" num="0117">In some embodiments, samples can be obtained from sources, including, but not limited to, samples from different individuals, different developmental stages of the same or different individuals, different diseased individuals (e.g., individuals with cancer or suspected of having a genetic disorder), normal individuals, samples obtained at different stages of a disease in an individual, samples obtained from an individual subjected to different treatments for a disease, samples from individuals subjected to different environmental factors, or individuals with predisposition to a pathology, individuals with exposure to a pathogen such as an infectious disease agent (e.g., HIV), and individuals who are recipients of donor cells, tissues and/or organs. In one embodiment, the sample is a maternal sample that is obtained from a pregnant female, for example a pregnant woman. In this instance, the sample can be analyzed using the methods described herein to provide a prenatal diagnosis of potential fetal disorders. Unless otherwise specified, a maternal sample comprises a mixture of fetal and maternal DNA e.g. cfDNA. In some embodiments, the maternal sample is a biological fluid sample e.g. blood sample. In other embodiments, the maternal sample is a purified cfDNA sample.</div>
</li> <li> <para-num num="[0118]"> </para-num> <div class="description-line" id="p-0119" num="0118">Samples can also be obtained from in vitro cultured tissues, cells, or other polynucleotide-containing sources. The cultured samples can be taken from sources including, but not limited to, cultures (e.g., tissue or cells) maintained in different media and conditions (e.g., pH, pressure, or temperature), cultures (e.g., tissue or cells) maintained for different periods of length, cultures (e.g., tissue or cells) treated with different factors or reagents (e.g., a drug candidate, or a modulator), or cultures of different types of tissue or cells.</div>
</li> <li> <para-num num="[0119]"> </para-num> <div class="description-line" id="p-0120" num="0119">Sequencing Methods</div>
</li> <li> <para-num num="[0120]"> </para-num> <div class="description-line" id="p-0121" num="0120">Differences in polymorphic sequences present in samples comprising a mixture of DNA e.g. cfDNA from two different genomes are identified by whole genome sequencing that employs next generation sequencing technologies (NGS) in which clonally amplified DNA templates or single DNA molecules are sequenced in a massively parallel fashion (e.g. as described in Volkerding et al. Clin Chem 55:641-658 [2009]; Metzker M Nature Rev 11:31-46 [2010]). In addition to high-throughput sequence information, NGS provides digital quantitative information, in that each sequence read is a countable “sequence tag” representing an individual clonal DNA template or a single DNA molecule. The sequencing technologies of NGS include pyrosequencing, sequencing-by-synthesis with reversible dye terminators, sequencing by oligonucleotide probe ligation and real time sequencing.</div>
</li> <li> <para-num num="[0121]"> </para-num> <div class="description-line" id="p-0122" num="0121">Some of the sequencing technologies are available commercially, such as the sequencing-by-hybridization platform from Affymetrix Inc. (Sunnyvale, Calif.) and the sequencing-by-synthesis platforms from 454 Life Sciences (Bradford, Conn.), Illumina/Solexa (Hayward, Calif.) and Helicos Biosciences (Cambridge, Mass.), and the sequencing-by-ligation platform from Applied Biosystems (Foster City, Calif.), as described below. In addition to the single molecule sequencing performed using sequencing-by-synthesis of Helicos Biosciences, other single molecule sequencing technologies are encompassed by the method of the invention and include the SMRT™ technology of Pacific Biosciences, the Ion Torrent™ technology, and nanopore sequencing being developed for example, by Oxford Nanopore Technologies.</div>
</li> <li> <para-num num="[0122]"> </para-num> <div class="description-line" id="p-0123" num="0122">While the automated Sanger method is considered as a ‘first generation’ technology, Sanger sequencing including the automated Sanger sequencing, can also be employed by the method of the invention. Additional sequencing methods that comprise the use of developing nucleic acid imaging technologies e.g. atomic force microscopy (AFM) or transmission electron microscopy (TEM), are also encompassed by the method of the invention. Exemplary sequencing technologies are described below.</div>
</li> <li> <para-num num="[0123]"> </para-num> <div class="description-line" id="p-0124" num="0123">In one embodiment, the DNA sequencing technology that is used in the method of the invention is the Helicos True Single Molecule Sequencing (tSMS) (e.g. as described in Harris T. D. et al., Science 320:106-109 [2008]). In the tSMS technique, a DNA sample is cleaved into strands of approximately 100 to 200 nucleotides, and a polyA sequence is added to the 3′ end of each DNA strand. Each strand is labeled by the addition of a fluorescently labeled adenosine nucleotide. The DNA strands are then hybridized to a flow cell, which contains millions of oligo-T capture sites that are immobilized to the flow cell surface. The templates can be at a density of about 100 million templates/cm<sup>2</sup>. The flow cell is then loaded into an instrument, e.g., HeliScope™ sequencer, and a laser illuminates the surface of the flow cell, revealing the position of each template. A CCD camera can map the position of the templates on the flow cell surface. The template fluorescent label is then cleaved and washed away. The sequencing reaction begins by introducing a DNA polymerase and a fluorescently labeled nucleotide. The oligo-T nucleic acid serves as a primer. The polymerase incorporates the labeled nucleotides to the primer in a template directed manner. The polymerase and unincorporated nucleotides are removed. The templates that have directed incorporation of the fluorescently labeled nucleotide are discerned by imaging the flow cell surface. After imaging, a cleavage step removes the fluorescent label, and the process is repeated with other fluorescently labeled nucleotides until the desired read length is achieved. Sequence information is collected with each nucleotide addition step. Whole genome sequencing by single molecule sequencing technologies excludes PCR-based amplification in the preparation of the sequencing libraries, and the directness of sample preparation allows for direct measurement of the sample, rather than measurement of copies of that sample.</div>
</li> <li> <para-num num="[0124]"> </para-num> <div class="description-line" id="p-0125" num="0124">In one embodiment, the DNA sequencing technology that is used in the method of the invention is the 454 sequencing (Roche) (e.g. as described in Margulies, M. et al. Nature 437:376-380 [2005]). 454 sequencing involves two steps. In the first step, DNA is sheared into fragments of approximately 300-800 base pairs, and the fragments are blunt-ended. Oligonucleotide adaptors are then ligated to the ends of the fragments. The adaptors serve as primers for amplification and sequencing of the fragments. The fragments can be attached to DNA capture beads, e.g., streptavidin-coated beads using, e.g., Adaptor B, which contains 5′-biotin tag. The fragments attached to the beads are PCR amplified within droplets of an oil-water emulsion. The result is multiple copies of clonally amplified DNA fragments on each bead. In the second step, the beads are captured in wells (pico-liter sized). Pyrosequencing is performed on each DNA fragment in parallel. Addition of one or more nucleotides generates a light signal that is recorded by a CCD camera in a sequencing instrument. The signal strength is proportional to the number of nucleotides incorporated. Pyrosequencing makes use of pyrophosphate (PPi) which is released upon nucleotide addition. PPi is converted to ATP by ATP sulfurylase in the presence of adenosine 5′ phosphosulfate. Luciferase uses ATP to convert luciferin to oxyluciferin, and this reaction generates light that is discerned and analyzed.</div>
</li> <li> <para-num num="[0125]"> </para-num> <div class="description-line" id="p-0126" num="0125">In one embodiment, the DNA sequencing technology that is used in the method of the invention is the SOLiD™ technology (Applied Biosystems). In SOLiD™ sequencing-by-ligation, genomic DNA is sheared into fragments, and adaptors are attached to the 5′ and 3′ ends of the fragments to generate a fragment library. Alternatively, internal adaptors can be introduced by ligating adaptors to the 5′ and 3′ ends of the fragments, circularizing the fragments, digesting the circularized fragment to generate an internal adaptor, and attaching adaptors to the 5′ and 3′ ends of the resulting fragments to generate a mate-paired library. Next, clonal bead populations are prepared in microreactors containing beads, primers, template, and PCR components. Following PCR, the templates are denatured and beads are enriched to separate the beads with extended templates. Templates on the selected beads are subjected to a 3′ modification that permits bonding to a glass slide. The sequence can be determined by sequential hybridization and ligation of partially random oligonucleotides with a central determined base (or pair of bases) that is identified by a specific fluorophore. After a color is recorded, the ligated oligonucleotide is cleaved and removed and the process is then repeated.</div>
</li> <li> <para-num num="[0126]"> </para-num> <div class="description-line" id="p-0127" num="0126">In one embodiment, the DNA sequencing technology that is used in the method of the invention is the single molecule, real-time (SMRT™) sequencing technology of Pacific Biosciences. In SMRT sequencing, the continuous incorporation of dye-labeled nucleotides is imaged during DNA synthesis. Single DNA polymerase molecules are attached to the bottom surface of individual zero-mode wavelength identifiers (ZMW identifiers) that obtain sequence information while phospolinked nucleotides are being incorporated into the growing primer strand. A ZMW is a confinement structure which enables observation of incorporation of a single nucleotide by DNA polymerase against the background of fluorescent nucleotides that rapidly diffuse in an out of the ZMW (in microseconds). It takes several milliseconds to incorporate a nucleotide into a growing strand. During this time, the fluorescent label is excited and produces a fluorescent signal, and the fluorescent tag is cleaved off. Identification of the corresponding fluorescence of the dye indicates which base was incorporated. The process is repeated.</div>
</li> <li> <para-num num="[0127]"> </para-num> <div class="description-line" id="p-0128" num="0127">In one embodiment, the DNA sequencing technology that is used in the method of the invention is nanopore sequencing (e.g. as described in Soni G V and Meller A. Clin Chem 53: 1996-2001 [2007]). Nanopore sequencing DNA analysis techniques are being industrially developed by a number of companies, including Oxford Nanopore Technologies (Oxford, United Kingdom). Nanopore sequencing is a single-molecule sequencing technology whereby a single molecule of DNA is sequenced directly as it passes through a nanopore. A nanopore is a small hole, of the order of 1 nanometer in diameter. Immersion of a nanopore in a conducting fluid and application of a potential (voltage) across it results in a slight electrical current due to conduction of ions through the nanopore. The amount of current which flows is sensitive to the size and shape of the nanopore. As a DNA molecule passes through a nanopore, each nucleotide on the DNA molecule obstructs the nanopore to a different degree, changing the magnitude of the current through the nanopore in different degrees. Thus, this change in the current as the DNA molecule passes through the nanopore represents a reading of the DNA sequence.</div>
</li> <li> <para-num num="[0128]"> </para-num> <div class="description-line" id="p-0129" num="0128">In one embodiment, the DNA sequencing technology that is used in the method of the invention is the chemical-sensitive field effect transistor (chemFET) array (e.g., as described in U.S. Patent Application Publication No. 20090026082). In one example of the technique, DNA molecules can be placed into reaction chambers, and the template molecules can be hybridized to a sequencing primer bound to a polymerase. Incorporation of one or more triphosphates into a new nucleic acid strand at the 3′ end of the sequencing primer can be discerned by a change in current by a chemFET. An array can have multiple chemFET sensors. In another example, single nucleic acids can be attached to beads, and the nucleic acids can be amplified on the bead, and the individual beads can be transferred to individual reaction chambers on a chemFET array, with each chamber having a chemFET sensor, and the nucleic acids can be sequenced.</div>
</li> <li> <para-num num="[0129]"> </para-num> <div class="description-line" id="p-0130" num="0129">In one embodiment, the DNA sequencing technology that is used in the method of the invention is the Halcyon Molecular's method that uses transmission electron microscopy (TEM). The method, termed Individual Molecule Placement Rapid Nano Transfer (IMPRNT), comprises utilizing single atom resolution transmission electron microscope imaging of high-molecular weight (150 kb or greater) DNA selectively labeled with heavy atom markers and arranging these molecules on ultra-thin films in ultra-dense (3 nm strand-to-strand) parallel arrays with consistent base-to-base spacing. The electron microscope is used to image the molecules on the films to determine the position of the heavy atom markers and to extract base sequence information from the DNA. The method is further described in PCT patent publication WO 2009/046445. The method allows for sequencing complete human genomes in less than ten minutes.</div>
</li> <li> <para-num num="[0130]"> </para-num> <div class="description-line" id="p-0131" num="0130">In one embodiment, the DNA sequencing technology is the Ion Torrent single molecule sequencing, which pairs semiconductor technology with a simple sequencing chemistry to directly translate chemically encoded information (A, C, G, T) into digital information (0, 1) on a semiconductor chip. In nature, when a nucleotide is incorporated into a strand of DNA by a polymerase, a hydrogen ion is released as a byproduct. Ion Torrent uses a high-density array of micro-machined wells to perform this biochemical process in a massively parallel way. Each well holds a different DNA molecule. Beneath the wells is an ion-sensitive layer and beneath that an ion sensor. When a nucleotide, for example a C, is added to a DNA template and is then incorporated into a strand of DNA, a hydrogen ion will be released. The charge from that ion will change the pH of the solution, which can be identified by Ion Torrent's ion sensor. The sequencer-essentially the world's smallest solid-state pH meter-calls the base, going directly from chemical information to digital information. The Ion personal Genome Machine (PGM™) sequencer then sequentially floods the chip with one nucleotide after another. If the next nucleotide that floods the chip is not a match. No voltage change will be recorded and no base will be called. If there are two identical bases on the DNA strand, the voltage will be double, and the chip will record two identical bases called. Direct identification allows recordation of nucleotide incorporation in seconds.</div>
</li> <li> <para-num num="[0131]"> </para-num> <div class="description-line" id="p-0132" num="0131">In one embodiment, the method employs massively parallel sequencing of millions of DNA fragments using Illumina's sequencing-by-synthesis and reversible terminator-based sequencing chemistry (e.g. as described in Bentley et al., Nature 6:53-59 [2009]). Template DNA can be genomic DNA e.g. cfDNA. In some embodiments, genomic DNA from isolated cells is used as the template, and it is fragmented into lengths of several hundred base pairs. In other embodiments, cfDNA is used as the template, and fragmentation is not required as cfDNA exists as short fragments. For example fetal cfDNA circulates in the bloodstream as fragments of &lt;300 bp, and maternal cfDNA has been estimated to circulate as fragments of between about 0.5 and 1 Kb (Li et al., Clin Chem, 50: 1002-1011 [2004]). Illumina's sequencing technology relies on the attachment of fragmented genomic DNA to a planar, optically transparent surface on which oligonucleotide anchors are bound. Template DNA is end-repaired to generate 5′-phosphorylated blunt ends, and the polymerase activity of Klenow fragment is used to add a single A base to the 3′ end of the blunt phosphorylated DNA fragments. This addition prepares the DNA fragments for ligation to oligonucleotide adapters, which have an overhang of a single T base at their 3′ end to increase ligation efficiency. The adapter oligonucleotides are complementary to the flow-cell anchors. Under limiting-dilution conditions, adapter-modified, single-stranded template DNA is added to the flow cell and immobilized by hybridization to the anchors. Attached DNA fragments are extended and bridge amplified to create an ultra-high density sequencing flow cell with hundreds of millions of clusters, each containing ˜1,000 copies of the same template. In one embodiment, the randomly fragmented genomic DNA e.g. cfDNA, is amplified using PCR before it is subjected to cluster amplification. Alternatively, an amplification-free genomic library preparation is used, and the randomly fragmented genomic DNA e.g. cfDNA is enriched using the cluster amplification alone (Kozarewa et al., Nature Methods 6:291-295 [2009]). The templates are sequenced using a robust four-color DNA sequencing-by-synthesis technology that employs reversible terminators with removable fluorescent dyes. High-sensitivity fluorescence identification is achieved using laser excitation and total internal reflection optics. Short sequence reads of about 20-40 bp e.g. 36 bp, are aligned against a repeat-masked reference genome and genetic differences are called using specially developed data analysis pipeline software. After completion of the first read, the templates can be regenerated in situ to enable a second read from the opposite end of the fragments. Thus, either single-end or paired end sequencing of the DNA fragments is used according to the method. Partial sequencing of DNA fragments present in the sample is performed, and sequence tags comprising reads of predetermined length e.g. 36 bp, are mapped to a known reference genome. In some embodiments, the mapped tags are counted.</div>
</li> <li> <para-num num="[0132]"> </para-num> <div class="description-line" id="p-0133" num="0132">The length of the sequence read is associated with the particular sequencing technology. NGS methods provide sequence reads that vary in size from tens to hundreds of base pairs. In some embodiments of the method described herein, the sequence reads are about 20 bp, about 25 bp, about 30 bp, about 35 bp, about 40 bp, about 45 bp, about 50 bp, about 55 bp, about 60 bp, about 65 bp, about 70 bp, about 75 bp, about 80 bp, about 85 bp, about 90 bp, 5 bp, a bout 100 bp, about 110 bp, about 120 bp, about 130, about 140 bp, about 150 bp, about 200 bp, about 250 bp, about 300 bp, about 350 bp, about 400 bp, about 450 bp, or about 500 bp. It is expected that technological advances will enable single-end reads of greater than 500 bp enabling for reads of greater than about 1000 bp when paired end reads are generated. In one embodiment, the sequence reads are 36 bp. Other sequencing methods that can be employed by the method of the invention include the single molecule sequencing methods that can sequence nucleic acids molecules &gt;5000 bp. The massive quantity of sequence output is transferred by an analysis pipeline that transforms primary imaging output from the sequencer into strings of bases. A package of integrated algorithms performs the core primary data transformation steps: image analysis, intensity scoring, base calling, and alignment.</div>
</li> <li> <para-num num="[0133]"> </para-num> <div class="description-line" id="p-0134" num="0133">In one embodiment, partial sequencing of DNA fragments present in the sample is performed, and sequence tags comprising reads of predetermined length e.g. 36 bp, are mapped to a known reference genome.</div>
</li> <li> <para-num num="[0134]"> </para-num> <div class="description-line" id="p-0135" num="0134">In another embodiment, the mapped tags are counted. Only sequence reads that uniquely align to the reference genome are counted as sequence tags. In one embodiment, the reference genome is the human reference genome NCBI36/hg18 sequence, which is available on the world wide web at genome.ucsc.edu/cgi-bin/hgGateway?org=Human&amp;db=hg18&amp;hgsid=166260105). Other sources of public sequence information include GenBank, dbEST, dbSTS, EMBL (the European Molecular Biology Laboratory), and the DDBJ (the DNA Databank of Japan). In another embodiment, the reference genome comprises the human reference genome NCBI36/hg18 sequence and/or an artificial target sequences genome, which consists of polymorphic target sequences e.g. a SNP genome consisting of polymorphic sequences known to be associated with a disorder. In yet another embodiment, the reference genome is an artificial target sequence genome comprising polymorphic target sequences e.g. SNP sequences consisting of polymorphic sequences known to be associated with a disorder.</div>
</li> <li> <para-num num="[0135]"> </para-num> <div class="description-line" id="p-0136" num="0135">Mapping of the sequence tags is achieved by comparing the sequence of the tag with the sequence of the reference genome to determine the chromosomal origin of the sequenced nucleic acid (e.g. cfDNA) molecule, and specific genetic sequence information is not needed. A number of computer algorithms are available for aligning sequences, including without limitation BLAST (Altschul et al., 1990), BLITZ (MPsrch) (Sturrock &amp; Collins, 1993), FASTA (Person &amp; Lipman, 1988), BOWTIE (Langmead et al., Genome Biology 10:R25.1-R25.10 [2009]), or ELAND (Illumina, Inc., San Diego, Calif., USA). In one embodiment, one end of the clonally expanded copies of the plasma cfDNA molecules is sequenced and processed by bioinformatic alignment analysis for the Illumina Genome Analyzer, which uses the Efficient Large-Scale Alignment of Nucleotide Databases (ELAND) software. Analysis of sequencing information for the identification of polymorphic sequences may allow for a small degree of mismatch (0-2 mismatches per sequence tag) to account for minor polymorphisms that may exist between the reference genome and the genomes in the mixed sample. Analysis of sequencing information for the determination of fetal fraction may allow for a small degree of mismatch depending on the polymorphic sequence. For example, a small degree of mismatch may be allowed if the polymorphic sequence is an STR. In cases when the polymorphic sequence is a SNP, all sequence that match exactly to either of the two alleles at the SNP site are counted first and filtered from the remaining reads, for which a small degree of mismatch may be allowed.</div>
</li> <li> <para-num num="[0136]"> </para-num> <div class="description-line" id="p-0137" num="0136">Only sequence reads that uniquely align to a reference genome are counted as sequence tags. In one embodiment, the reference genome is an artificial target sequences genome that comprises the sequences of the polymorphic target nucleic acids e.g. SNPs. In one embodiment, the reference genome is an artificial SNP reference genome. In another r embodiment, the reference genome is an artificial STR reference genome. In yet another embodiment, the reference genome is an artificial tandem-STR reference genome. Artificial reference genomes can be compiled using the sequences of the target polymorphic nucleic acids. Artificial reference genomes can comprise polymorphic target sequence each comprising one or more different types of polymorphic sequences. For example, an artificial reference genome can comprise polymorphic sequences comprising SNP alleles and/or STRs.</div>
</li> <li> <para-num num="[0137]"> </para-num> <div class="description-line" id="p-0138" num="0137">Sequencing Library Preparation</div>
</li> <li> <para-num num="[0138]"> </para-num> <div class="description-line" id="p-0139" num="0138">Next-generation DNA sequencers, such as the 454-FLX (Roche; at web address 454.com), the SOLiD™3 (Applied Biosystems; at web address solid.appliedbiosystems.com), and the Genome Analyzer (Illumina; http://www.illumina.com/pages.ilmn?ID=204) have transformed the landscape of genetics through their ability to produce hundreds of megabases of sequence information in a single run.</div>
</li> <li> <para-num num="[0139]"> </para-num> <div class="description-line" id="p-0140" num="0139">Sequencing methods require the preparation of sequencing libraries. Sequencing library preparation involves the production of a random collection of adapter-modified DNA fragments, which are ready to be sequenced. Sequencing libraries of polynucleotides can be prepared from DNA or RNA, including equivalents, analogs of either DNA or cDNA, that is complementary or copy DNA produced from an RNA template, for example by the action of reverse transcriptase. The polynucleotides may originate in double-stranded DNA (dsDNA) form (e.g. genomic DNA fragments, PCR and amplification products) or polynucleotides that may have originated in single-stranded form, as DNA or RNA, and been converted to dsDNA form. By way of example, mRNA molecules may be copied into double-stranded cDNAs suitable for use in preparing a sequencing library. The precise sequence of the primary polynucleotide molecules is generally not material to the method of library preparation, and may be known or unknown. In one embodiment, the polynucleotide molecules are DNA molecules. More particularly, the polynucleotide molecules represent the entire genetic complement of an organism, and are genomic DNA molecules e.g. cfDNA molecules, which include both intron and exon sequence (coding sequence), as well as non-coding regulatory sequences such as promoter and enhancer sequences. Still yet more particularly, the primary polynucleotide molecules are human genomic DNA molecules e.g. cfDNA molecules present in peripheral blood of a pregnant subject. Preparation of sequencing libraries for some NGS sequencing platforms require that the polynucleotides be of a specific range of fragment sizes e.g. 0-1200 bp. Therefore, fragmentation of polynucleotides e.g. genomic DNA may be required. cfDNA exists as fragments of &lt;300 base pairs. Therefore, fragmentation of cfDNA is not necessary for generating a sequencing library using cfDNA samples. Fragmentation of polynucleotide molecules by mechanical means e.g. nebulization, sonication and hydroshear, results in fragments with a heterogeneous mix of blunt and 3′- and 5′-overhanging ends. Whether polynucleotides are forcibly fragmented or naturally exists as fragments, they are converted to blunt-ended DNA having 5-phosphates and 3′-hydroxyl.</div>
</li> <li> <para-num num="[0140]"> </para-num> <div class="description-line" id="p-0141" num="0140">Typically, the fragment ends are end-repaired i.e. blunt-ended using methods or kits known in the art. The blunt-ended fragments can be phosphorylated by enzymatic treatment, for example using polynucleotide kinase. In some embodiments, a single deoxynucleotide e.g. deoxyadenosine (A) is added to the 3′-ends of the polynucleotides, for example, by the activity of certain types of DNA polymerase such as Taq polymerase or Klenow exo minus polymerase. dA-tailed products are compatible with ‘T’ overhang present on the 3′ terminus of each duplex region of adaptors to which they are ligated in a subsequent step. dA-tailing prevents self-ligation of both of the blunt-ended polynucleotide such that there is a bias towards formation of the adaptor-ligated sequences. The dA-tailed polynucleotides are ligated to double-stranded adaptor polynucleotides sequences. The same adaptor can be used for both ends of the polynucleotide, or two sets of adaptors can be utilized. Ligation methods are known in the art and utilize ligase enzymes such as DNA ligase to covalently link the adaptor to the d-A-tailed polynucleotide. The adaptor may contain a 5′-phosphate moiety to facilitate ligation to the target 3′-OH. The dA-tailed polynucleotide contains a 5′-phosphate moiety, either residual from the shearing process, or added using an enzymatic treatment step, and has been end repaired, and optionally extended by an overhanging base or bases, to give a 3′-OH suitable for ligation. The products of the ligation reaction are purified to remove unligated adaptors, adaptors that may have ligated to one another, and to select a size range of templates for cluster generation, which can be preceded by an amplification e.g. a PCR amplification. Purification of the ligation products can be obtained by methods including gel electrophoresis and solid-phase reversible immobilization (SPRI).</div>
</li> <li> <para-num num="[0141]"> </para-num> <div class="description-line" id="p-0142" num="0141">Standard protocols e.g. protocols for sequencing using, for example, the Illumina platform, instruct users to purify the end-repaired products prior to dA-tailing, and to purify the dA-tailing products prior to the adaptor-ligating steps of the library preparation. Purification of the end-repaired products and dA-tailed products remove enzymes, buffers, salts and the like to provide favorable reaction conditions for the subsequent enzymatic step. In one embodiment, the steps of end-repairing, dA-tailing and adaptor ligating exclude the purification steps. Thus, in one embodiment, the method of the invention encompasses preparing a sequencing library that comprises the consecutive steps of end-repairing, dA-tailing and adaptor-ligating (see pending U.S. patent application Ser. No. 12/958,353). In embodiments for preparing sequencing libraries that do not require the dA-tailing step, e.g. protocols for sequencing using Roche 454 and SOLID™3 platforms, the steps of end-repairing and adaptor-ligating exclude the purification step of the end-repaired products prior to the adaptor-ligating.</div>
</li> <li> <para-num num="[0142]"> </para-num> <div class="description-line" id="p-0143" num="0142">In a next step of one embodiment of the method, an amplification reaction is prepared. The amplification step introduces to the adaptor ligated template molecules the oligonucleotide sequences required for hybridization to the flow cell. The contents of an amplification reaction are known by one skilled in the art and include appropriate substrates (such as dNTPs), enzymes (e.g. a DNA polymerase) and buffer components required for an amplification reaction. Optionally, amplification of adaptor-ligated polynucleotides can be omitted. Generally amplification reactions require at least two amplification primers i.e. primer oligonucleotides, which may be identical, and include an ‘adaptor-specific portion’, capable of annealing to a primer-binding sequence in the polynucleotide molecule to be amplified (or the complement thereof if the template is viewed as a single strand) during the annealing step. Once formed, the library of templates prepared according to the methods described above can be used for solid-phase nucleic acid amplification. The term ‘solid-phase amplification’ as used herein refers to any nucleic acid amplification reaction carried out on or in association with a solid support such that all or a portion of the amplified products are immobilized on the solid support as they are formed. In particular, the term encompasses solid-phase polymerase chain reaction (solid-phase PCR) and solid phase isothermal amplification which are reactions analogous to standard solution phase amplification, except that one or both of the forward and reverse amplification primers is/are immobilized on the solid support. Solid phase PCR covers systems such as emulsions, wherein one primer is anchored to a bead and the other is in free solution, and colony formation in solid phase gel matrices wherein one primer is anchored to the surface, and one is in free solution. Following amplification, and sequencing libraries can be analyzed by microfluidic capillary electrophoresis to ensure that the library is free of adaptor dimers or single stranded DNA. The library of template polynucleotide molecules is particularly suitable for use in solid phase sequencing methods. In addition to providing templates for solid-phase sequencing and solid-phase PCR, library templates provide templates for whole genome amplification.</div>
</li> <li> <para-num num="[0143]"> </para-num> <div class="description-line" id="p-0144" num="0143">In one embodiment, the library of adaptor-ligated polynucleotides is subjected to massively parallel sequencing, which includes techniques for sequencing millions of fragments of nucleic acids, e.g., using attachment of randomly fragmented genomic DNA to a planar, optically transparent surface and solid phase amplification to create a high density sequencing flow cell with millions of clusters. Clustered arrays can be prepared using either a process of thermocycling, as described in patent WO9844151, or a process whereby the temperature is maintained as a constant, and the cycles of extension and denaturing are performed using changes of reagents. The Solexa/Illumina method referred to herein relies on the attachment of randomly fragmented genomic DNA to a planar, optically transparent surface. Attached DNA fragments are extended and bridge amplified to create an ultra-high density sequencing flow cell with millions of clusters each containing thousands of copies of the same template (WO 00/18957 and WO 98/44151). The cluster templates are sequenced using a robust four-color DNA sequencing-by-synthesis technology that employs reversible terminators with removable fluorescent dyes. Alternatively, the library may be amplified on beads wherein each bead contains a forward and reverse amplification primer.</div>
</li> <li> <para-num num="[0144]"> </para-num> <div class="description-line" id="p-0145" num="0144">Sequencing of the amplified libraries can be carried out using any suitable sequencing technique as described herein. In one embodiment, sequencing is massively parallel sequencing using sequencing-by-synthesis with reversible dye terminators. In other embodiments, sequencing is massively parallel sequencing using sequencing-by-ligation. In other embodiments, sequencing is single molecule sequencing.</div>
</li> <li> <para-num num="[0145]"> </para-num> <div class="description-line" id="p-0146" num="0145">Identification of Polymorphisms by WGS</div>
</li> <li> <para-num num="[0146]"> </para-num> <div class="description-line" id="p-0147" num="0146">Polymorphisms present in the sample comprising a mixture of DNA from two genomes can be identified by WGS of at least a portion of the DNA molecules contained in the sample.</div>
</li> <li> <para-num num="[0147]"> </para-num> <div class="description-line" id="p-0148" num="0147">Prior to sequencing, the mixture of DNA e.g. purified mixture of fetal and maternal cfDNA, is modified to prepare a sequencing library to generate single-end sequence reads of between 20 and 40 bp e.g. 36 bp, which are aligned to a reference genome, e.g. hg18. In some embodiments, the sequence reads comprise about 20 bp, about 25 bp, about 30 bp, about 35 bp, about 40 bp, about 45 bp, about 50 bp, about 55 bp, about 60 bp, about 65 bp, about 70 bp, about 75 bp, about 80 bp, about 85 bp, about 90 bp, about 95 bp, about 100 bp, about 110 bp, about 120 bp, about 130 bp, about 140 bp, about 150 bp, about 200 bp, about 250 bp, about 300 bp, about 350 bp, about 400 bp, about 450 bp, or about 500 bp. It is expected that technological advances will enable single-end reads of greater than 500 bp enabling for reads of greater than about 1000 bp when paired end reads are generated. In one embodiment, the sequence reads comprise 36 bp. In another embodiment, paired-end sequencing is used. Paired-end sequencing is a simple modification of the single end read DNA library preparation that facilitates reading both the forward and reverse template strands of each cluster during one paired-end read. In addition to sequence information, both reads contain long range positional information. Sequence reads are aligned to a human reference genome, and the reads that are uniquely mapped to the human reference genome are identified as sequence tags. In one embodiment, at least about 3×10<sup>6 </sup>qualified sequence tags, at least about 5×10<sup>6 </sup>qualified sequence tags, at least about 8×10<sup>6 </sup>qualified sequence tags, at least about 10×10<sup>6 </sup>qualified sequence tags, at least about 15×10<sup>6 </sup>qualified sequence tags, at least about 20×10<sup>6 </sup>qualified sequence tags, at least about 30×10<sup>6 </sup>qualified sequence tags, at least about 40×10<sup>6 </sup>qualified sequence tags, at least about 50×10<sup>6 </sup>qualified sequence tags, at least about 50×10<sup>6 </sup>qualified sequence tags, at least about 60×10<sup>6 </sup>qualified sequence tags, at least about 70×10<sup>6 </sup>qualified sequence tags, at least about 80×10<sup>6 </sup>qualified sequence tags, at least about 90×10<sup>6 </sup>qualified sequence tags, at least about 100×10<sup>6 </sup>qualified sequence tags, at least about 150×10<sup>6 </sup>qualified sequence tags, at least about 200×10<sup>6 </sup>qualified sequence tags, at least about 250×10<sup>6 </sup>qualified sequence tags, at least about 300×10<sup>6 </sup>qualified sequence tags, or more qualified sequence tags comprising between 20 and 40 bp reads are obtained from reads that map uniquely to a reference genome. The number of tags required to encompass an entire genome depends on the length of the reads. It is estimated that the human genome comprises at least 10 million SNPs. Therefore, a sufficient number of tags that encompass at least 10 million polymorphisms will need to be generated to identify polymorphisms using whole genome sequencing according to the present method.</div>
</li> <li> <para-num num="[0148]"> </para-num> <div class="description-line" id="p-0149" num="0148">Polymorphic sequences located at multiple polymorphic sites are identified by the sequences of the tags, and sequence differences at each of the multiple polymorphic sites (loci) can be analyzed to allocate the variant sequences to one of the two genomes from which the sequenced DNA is derived. Identification of the multiple polymorphic sequences is performed by aligning the sequence tags to multiple reference polymorphic sequences, which are known sequences that are available from various databases as described herein. For example, identification of polymorphic sequences comprising SNPs can be performed by comparing the sequences of the tags to the sequences of known SNPs provided in databases including, but not limited to Human SNP Database at world wide web address wi.mit.edu, NCBI dbSNP Home Page at world wide web address ncbi.nlm.nih.gov, world wide web address lifesciences.perkinelmer.com, Celera Human SNP database at world wide web address celera.com, the SNP Database of the Genome Analysis Group (GAN) at world wide web address gan.iarc.fr. To identify tandem SNP sequences, the International HapMap Consortium database can be searched (The International HapMap Project, Nature 426:789-796 [2003]). The database is available on the world wide web at hapmap.org. Identification of polymorphic sequences comprising STRs can be performed by comparing the sequences of the tags to the sequences of known STRs provided for example in the ATCC short tandem repeat (STR) database at world wide web address atcc.org.</div>
</li> <li> <para-num num="[0149]"> </para-num> <div class="description-line" id="p-0150" num="0149">In one embodiment, the sample comprising the mixture of DNA molecules derived from two genomes is unenriched for polymorphic target sequences. In another embodiment, the sample comprising the mixture of DNA molecules derived from two different genomes is non-specifically enriched for the whole genome sequences prior to whole genome sequencing i.e. whole genome amplification is performed prior to sequencing.</div>
</li> <li> <para-num num="[0150]"> </para-num> <div class="description-line" id="p-0151" num="0150">In one embodiment, the method can be used for identifying multiple polymorphisms in a first genome present in a blood sample that comprises a mixture of cfDNA molecules contributed to the sample by two different genomes. For example, the blood sample is a maternal blood sample that comprises cfDNA molecules derived from the fetal and the maternal genomes. Preferably, the plasma fraction is obtained from the whole blood sample, and the cfDNA contained in the plasma fraction is purified using known methods prior to whole genome sequencing. In some embodiments, the purified cfDNA sample comprising the mixture of fetal and maternal cfDNA is unenriched for either the fetal or the maternal genome.</div>
</li> <li> <para-num num="[0151]"> </para-num> <div class="description-line" id="p-0152" num="0151">Enrichment of specific polymorphic sequences i.e. polymorphic target sequences can be performed for example, using sequence-specific primers to amplify polymorphic sequences of interest. In one embodiment, polymorphic sequences of interest include sequences known to be associated with a disorder. Polymorphic target sequences of interest include any one or a combination of the polymorphisms described herein. In one embodiment, the polymorphic target sequences of interest are associated with monogenic and/or polygenic disorders. In one embodiment, polymorphic sequences are specifically amplified in a maternal sample comprising a mixture of fetal and maternal cfDNA for identifying targeted polymorphic sequences that are associated with a disorder. In some embodiments, the polymorphic sequences are associated with a disorder in the fetus.</div>
</li> <li> <para-num num="[0152]"> </para-num> <div class="description-line" id="p-0153" num="0152">In one embodiment, identification of polymorphic sequences contributed by a first genome present in a sample comprising a mixture of two genomes is performed by determining the sequence at multiple polymorphic sites in a first sample containing DNA molecules derived essentially from only a second genome, determining the sequence at the corresponding multiple polymorphic sites in a second sample containing a mixture of DNA molecules derived from a first and a second genome, and comparing the polymorphic sequences determined in both samples thereby identifying multiple polymorphisms in a first genome of a sample comprising a mixture of two genomes. For example, identification of polymorphic sequences contributed by a fetal genome i.e. first genome, is performed by determining the sequence at multiple polymorphic sites in a maternal buffy coat sample i.e. a sample containing DNA molecules derived essentially from only a second genome, determining the sequence at the corresponding multiple polymorphic sites in a purified plasma sample i.e. a second sample containing a mixture of cfDNA molecules derived from the fetal and the maternal genomes, and comparing the polymorphic sequences determined in both samples to identify multiple fetal polymorphisms.</div>
</li> <li> <para-num num="[0153]"> </para-num> <div class="description-line" id="p-0154" num="0153">In another embodiment, identification of polymorphic sequences contributed by the fetal genome to the mixture of fetal and maternal nucleic acids in a maternal sample can be performed without genotyping the maternal alleles in a maternal sample that is substantially free of fetal nucleic acids. Identification of the fetal polymorphic sequences can be made based on the identification and quantification of different sequences i.e. alleles, at multiple polymorphic sites. As is shown in Example 1, whole genome sequencing of a maternal plasma cfDNA sample comprising a mixture of fetal and maternal cfDNA molecules reveals that fetal polymorphic sequences e.g. SNPs, that are present in the maternal sample are present at a level that is proportional to the level of fetal cfDNA relative to that of the maternal cfDNA. In some embodiments, the method identifies multiple polymorphisms in a sample comprising a mixture of cfDNA of two genomes without specifically amplifying target polymorphic sequences of interest. In some embodiments, identification of polymorphic sequences can be performed in samples that are specifically enriched for target polymorphic sequences. Examples 6, 7, 8, and 9 show that the sequence tags that map to specific polymorphic sequences can be quantified, that the polymorphic sequences pertaining to the fetal and the maternal genomes can be identified, and that the relative level of fetal and maternal tags can be used to determine the fetal fraction. In particular, the relative contribution of a polymorphic allele derived from the fetal genome to the contribution of the corresponding polymorphic allele derived from the maternal genome is indicative of the origin of the sequence i.e. whether the polymorphic sequence is fetal or maternal in origin. Informative polymorphic sites e.g. SNP, are identified by the difference in the allelic sequences and the amount of each of the possible alleles. Fetal cfDNA is present at a concentration that can be up to about 30% of the maternal cfDNA (Liao et al., Clin Chem 57:1 [2011]). Thus, the presence of a minor contribution of an allele to the mixture of fetal and maternal nucleic acids relative to the major contribution of the maternal allele can be assigned to the fetus. Alleles that are derived from the maternal genome are herein referred to as major alleles, and alleles that are derived from the fetal genome are herein referred to as minor alleles. Alleles that are represented by similar levels of mapped sequence tags represent maternal alleles. <figref idrefs="DRAWINGS">FIG. 4</figref> provides exemplary data showing that informative SNPs are discerned from the single nucleotide change at a polymorphic site, and fetal alleles are discerned by their relative minor contribution to the mixture of fetal and maternal nucleic acids in the sample when compared to the major contribution to the mixture by the maternal nucleic acids. The relative abundance of fetal cfDNA in the maternal sample can be determined as a parameter of the total number of unique sequence tags mapped to the target nucleic acid sequence on a reference genome for each of the two alleles of the predetermined polymorphic site. Quantification of informative polymorphisms e.g. SNPs, can be used to determine the fraction of fetal DNA in a mixture of fetal and maternal DNA. Similarly, quantification of informative polymorphic alleles can be used to determine the level of a first genome and a second genome in a sample comprising a mixture of the two genomes.</div>
</li> <li> <para-num num="[0154]"> </para-num> <div class="description-line" id="p-0155" num="0154">In one embodiment, the multiple polymorphic sites are located on chromosomes other than 21, 18, 13, or Y. In another embodiment, the multiple polymorphic sites are located on a plurality of different chromosomes.</div>
</li> <li> <para-num num="[0155]"> </para-num> <div class="description-line" id="p-0156" num="0155">In one embodiment, a paternal sample can be used to determine the presence or absence of a paternally-inherited fetal nucleic acid in a sample. The paternal sample can be, for example, a whole blood sample, a plasma sample, a serum sample, or a buffy coat sample. In another embodiment, a paternal sample is not used to determine the presence or absence of a paternally-inherited fetal nucleic acid in a sample. The sample can be enriched for specific sequences prior to genotyping.</div>
</li> <li> <para-num num="[0156]"> </para-num> <div class="description-line" id="p-0157" num="0156">In one embodiment, identification of polymorphic sequences in the mixture of fetal and maternal cfDNA molecules does not involve investigating the methylation status of nucleic acids in the sample. In another embodiment, identification of polymorphic sequences in the mixture of fetal and maternal cfDNA molecules does not involve use of Y-chromosome sequence.</div>
</li> <li> <para-num num="[0157]"> </para-num> <div class="description-line" id="p-0158" num="0157">The number of polymorphisms that can be genotyped in a sample can be at least 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22, 23, 24, 25, 26, 27, 28, 29, 30, 35, 40, 45, 50, 55, 60, 65, 70, 75, 80, 85, 90, 95, 100, 200, 300, 400, 500, 600, 700, 800, 900, 1000 or more. For example, it is estimated that the human genome comprises at least about 10 million SNPs. Therefore, the number of polymorphisms that can be genotypes in a sample from a human subject can be at least about 10 million SNPs, as well as many other types of polymorphisms that are present in any one human genome. In some embodiments, identification of multiple polymorphisms in a first genome of a sample comprising a mixture of DNA e.g. cfDNA, of a first and a second genome is performed by whole genome sequencing using a NGS method as described herein. In some embodiments, the whole genome sequencing method is an NGS method that identifies the polymorphic sequences by massively parallel sequencing clonally amplified nucleic acid molecules or by massively parallel sequencing of single nucleic acid molecules i.e. single molecule sequencing.</div>
</li> <li> <para-num num="[0158]"> </para-num> <div class="description-line" id="p-0159" num="0158">Applications to Disorders</div>
</li> <li> <para-num num="[0159]"> </para-num> <div class="description-line" id="p-0160" num="0159">In one embodiment, the invention provides a method for determining the presence or absence of a number of disorders in a blood sample comprising a mixture of cfDNA of a first and a second genome. Determination of the presence or absence of multiple disorders comprises (a) whole genome sequencing at least a portion of the mixture of cfDNA to obtain a plurality of sequence tags; (b) determining in the plurality of sequence tags the presence or absence of multiple polymorphisms that are associated with the number of disorders, and (c) associating the multiple polymorphisms with the first and/or second genome in the mixture, wherein the mixture is unenriched for the multiple polymorphisms. Identification of the multiple polymorphisms in the mixture of cfDNA is performed by comparing the sequence of the mapped tags obtained by the whole genome sequencing method to multiple reference polymorphisms, as described herein. In one embodiment, the first genome is a fetal genome, and a second genome is a maternal genome. In another embodiment, the first genome is a genome of an unaffected cell and the second genome is a genome from an affected cell e.g. a cancer cell. In some embodiments, the affected and unaffected cells are derived from the same subject. For example, the affected cell can be a cell whose genome has been altered by a disorder. In some embodiments the disorder is a monogenic disorder. In other embodiments, the disorder is a polygenic disorder. Disorders can be identified by a single polymorphism e.g. a tag SNP, or by multiple polymorphisms present in a haplotype. In some embodiments, the multiple polymorphisms identified according to the present method are present in a haplotype block.</div>
</li> <li> <para-num num="[0160]"> </para-num> <div class="description-line" id="p-0161" num="0160">The disorders that can be identified by the present method are genetic disorders, which are illnesses caused at least in part by abnormalities in genes or chromosomes. Disorders identified by the present method include monogenic i.e. single gene, disorders and polygenic i.e. complex disorders. Single gene disorders include autosomal dominant, autosomal recessive, X-linked dominant, X-linked recessive, and Y-linked.</div>
</li> <li> <para-num num="[0161]"> </para-num> <div class="description-line" id="p-0162" num="0161">In autosomal dominant disorders, only one mutated copy of the gene will be necessary for a person to be affected by the disorder. Typically, an affected subject has one affected parent, and there is a 50% chance that the offspring will inherit the mutated gene. Conditions that are autosomal dominant sometimes have reduced penetrance, which means that although only one mutated copy is needed, not all individuals who inherit that mutation go on to develop the disease. Examples of autosomal dominant disorders that can be identified by the present method include without limitation familial hypercholesterolemia, hereditary spherocytosis, Marfan syndrome, neurofibromatosis type 1, hereditary nonpolyposis colorectal cancer, and hereditary multiple exostoses, and Huntington disease.</div>
</li> <li> <para-num num="[0162]"> </para-num> <div class="description-line" id="p-0163" num="0162">Autosomal recessive disorders detected by the present method include Sickle cell anemia, Cystic fibrosis, Tay-Sachs disease, Tay-Sachs disease, Mucopolysaccharidoses, Glycogen storage diseases, and Galactosemia. X-linked disorders detected by the present method include Duchenne muscular dystrophy and hemophilia. In autosomal recessive disorders, two copies of the gene must be mutated for a subject to be affected by an autosomal recessive disorder. An affected subject usually has unaffected parents who each carry a single copy of the mutated gene (and are referred to as carriers). Two unaffected people who each carry one copy of the mutated gene have a 25% chance with each pregnancy of having a child affected by the disorder. Examples of this type of disorder that can be identified by the present method include are cystic fibrosis, sickle-cell disease, Tay-Sachs disease, Niemann-Pick disease, spinal muscular atrophy, and Roberts syndrome. Certain other phenotypes, such as wet versus dry earwax, are also determined in an autosomal recessive fashion. X-linked dominant disorders are caused by mutations in genes on the X chromosome. Only a few disorders have this inheritance pattern, with a prime example being X-linked hypophosphatemic rickets. Males and females are both affected in these disorders, with males typically being more severely affected than females. Some X-linked dominant conditions such as Rett syndrome, incontinentia pigmenti type 2 and Aicardi syndrome are usually fatal in males, and are therefore predominantly seen in females. Exceptions to this finding are extremely rare cases in which boys with Klinefelter syndrome (47,XXY) also inherit an X-linked dominant condition and exhibit symptoms more similar to those of a female in terms of disease severity. The chance of passing on an X-linked dominant disorder differs between men and women. The sons of a man with an X-linked dominant disorder will all be unaffected (since they receive their father's Y chromosome), and his daughters will all inherit the condition. A woman with an X-linked dominant disorder has a 50% chance of having an affected fetus with each pregnancy, although it should be noted that in cases such as incontinentia pigmenti only female offspring are generally viable. In addition, although these conditions do not alter fertility per se, individuals with Rett syndrome or Aicardi syndrome rarely reproduce.</div>
</li> <li> <para-num num="[0163]"> </para-num> <div class="description-line" id="p-0164" num="0163">The present method can also be used to identify polymorphisms associated with X-linked disorders. X-linked recessive conditions are also caused by mutations in genes on the X chromosome. Males are more frequently affected than females, and the chance of passing on the disorder differs between men and women. The sons of a man with an X-linked recessive disorder will not be affected, and his daughters will carry one copy of the mutated gene. A woman who is a carrier of an X-linked recessive disorder (X<sup>R</sup>X<sup>r</sup>) has a 50% chance of having sons who are affected and a 50% chance of having daughters who carry one copy of the mutated gene and are therefore carriers. X-linked recessive conditions include without limitation the serious diseases Hemophilia A, Duchenne muscular dystrophy, and Lesch-Nyhan syndrome as well as common and less serious conditions such as male pattern baldness and red-green color blindness. X-linked recessive conditions can sometimes manifest in females due to skewed X-inactivation or monosomy X (Turner syndrome).</div>
</li> <li> <para-num num="[0164]"> </para-num> <div class="description-line" id="p-0165" num="0164">Y-linked disorders are caused by mutations on the Y chromosome. Because males inherit a Y chromosome from their fathers, every son of an affected father will be affected. Because females inherit an X chromosome from their fathers, female offspring of affected fathers are never affected. Since the Y chromosome is relatively small and contains very few genes, there are relatively few Y-linked disorders. Often the symptoms include infertility, which may be circumvented with the help of some fertility treatments. Examples are male infertility and hypertrichosis pinnae.</div>
</li> <li> <para-num num="[0165]"> </para-num> <div class="description-line" id="p-0166" num="0165">The present method can also be used to identify polymorphisms associated with genetic disorders that are complex, multifactorial, or polygenic, meaning that they are likely associated with the effects of multiple genes in combination with lifestyle and environmental factors. Multifactorial disorders include for example, heart disease and diabetes. Although complex disorders often cluster in families, they do not have a clear-cut pattern of inheritance. On a pedigree, polygenic diseases do tend to “run in families”, but the inheritance does is not simple as is with Mendelian diseases. Strong environmental components are associated with many complex disorders e.g., blood pressure. The present method can be used to identify polymorphisms that are associated with polygenic disorders including but not limited to asthma, autoimmune diseases such as multiple sclerosis, cancers, ciliopathies, cleft palate, diabetes, heart disease, hypertension, inflammatory bowel disease, mental retardation, mood disorder, obesity, refractive error, and infertility. In some embodiments, the polymorphisms are SNPs. In other embodiments, the polymorphisms are STRs. In yet other embodiments, the polymorphisms are a combination of SNPs and STRs.</div>
</li> <li> <para-num num="[0166]"> </para-num> <div class="description-line" id="p-0167" num="0166">In one embodiment, identification of the polymorphic sequences associated with disorders comprises sequencing at least a portion of the cellular genome corresponding to the second genome in the mixture of cfDNA. Identification of polymorphic sequences contributed by a first genome is performed by determining the sequence at multiple polymorphic sites in a first sample containing DNA molecules derived essentially from only a second genome, determining the sequence at the corresponding multiple polymorphic sites in a second sample containing a mixture of DNA molecules derived from a first and a second genome, and comparing the polymorphic sequences determined in both samples thereby identifying multiple polymorphisms in a first genome of a sample comprising a mixture of two genomes. For example, identification of polymorphic sequences contributed by a fetal genome i.e. first genome, is performed by determining the sequence at multiple polymorphic sites in a maternal buffy coat sample i.e. a sample containing DNA molecules derived essentially from only a second genome, determining the sequence at the corresponding multiple polymorphic sites in a purified plasma sample i.e. a second sample containing a mixture of cfDNA molecules derived from the fetal and the maternal genomes, and comparing the polymorphic sequences determined in both samples to identify multiple fetal polymorphisms. In one embodiment, the first genome is a fetal genome, and a second genome is a maternal genome. In another embodiment, the first genome is a genome of an unaffected cell and the second genome is a genome from an affected cell. In some embodiments, the affected and unaffected cells are derived from the same subject. For example, the affected cell can be a cell whose genome has been altered by a disorder. In some embodiments the disorder is a monogenic disorder. In other embodiments, the disorder is a polygenic disorder. Disorders can be identified by a single polymorphism, or by multiple polymorphisms present in a haplotype. In some embodiments, the multiple polymorphisms identified according to the present method are present in a haplotype block.</div>
</li> <li> <para-num num="[0167]"> </para-num> <div class="description-line" id="p-0168" num="0167">In one embodiment, the present invention provides methods for detecting cancer in a patient, comprising: providing a sample from a patient comprising a mixture of genomes derived from normal i.e. unaffected, and cancerous i.e. affected, cells; and identifying multiple polymorphisms associated with cancer. In some embodiments, the sample is selected from blood, plasma, serum and urine. In some embodiments, the sample is a plasma sample. In other embodiments, the sample is a urine sample.</div>
</li> <li> <para-num num="[0168]"> </para-num> <div class="description-line" id="p-0169" num="0168">In one embodiment, identifying multiple polymorphisms associated with cancer comprises enriching the DNA in the sample for polymorphic target sequences. In other embodiments, enrichment of the sample for polymorphic target sequences is not performed. In some embodiments, identifying multiple polymorphisms associated with cancer comprises quantifying the number of copies of the polymorphic sequence.</div>
</li> <li> <para-num num="[0169]"> </para-num> <div class="description-line" id="p-0170" num="0169">Cancers that can be identified and/or monitored according to the method of the invention include solid tumors, as well as, hematologic tumors and/or malignancies. Various cancers to be treated include sarcomas, carcinomas, and adenocarcinomas not limited to breast cancer, lung cancer, colorectal cancer, pancreatic cancer, ovarian cancer, prostate cancer, renal carcinoma, hepatoma, brain cancer, melanoma, multiple myeloma, lymphoma, Hodgkin's lymphoma, non-Hodgkin's lymphoma, childhood lymphomas, and lymphomas of lymphocytic and cutaneous origin, leukemia, childhood leukemia, hairy-cell leukemia, acute lymphocytic leukemia, acute myelocytic leukemia, chronic lymphocytic leukemia, chronic myelocytic leukemia, chronic myelogenous leukemia, and mast cell leukemia, myeloid neoplasms, mast cell neoplasms, hematologic tumor, and lymphoid tumor, including metastatic lesions in other tissues or organs distant from the primary tumor site.</div>
</li> <li> <para-num num="[0170]"> </para-num> <div class="description-line" id="p-0171" num="0170">The methods of the present invention are useful, for example, in diagnosing or determining a prognosis in a disease condition known to be associated with a specific haplotype(s), to determine novel haplotypes, and to detect haplotype associations with responsiveness to pharmaceuticals. The association of multiple polymorphic sequences with multiple disorders can be determined from the identity of a single polymorphic sequence for each of the multiple disorders. Alternatively, association of multiple polymorphic sequences with multiple disorders can be determined from the identity of multiple polymorphic sequences for each of the multiple disorders.</div>
</li> <li> <para-num num="[0171]"> </para-num> <div class="description-line" id="p-0172" num="0171">Conventional genotyping techniques have been limited to identifying polymorphisms in short genomic regions of a few kilobases, and identification of haplotypes has relied on family data and statistical estimation using computational algorithms. Whole genome sequencing enables the identification of haplotypes by directly identifying the polymorphisms on a genome. The identification of the haplotypes according to the method is not limited by the intervening distance between polymorphisms. In some embodiments, the method further comprises whole genome sequencing maternal cellular DNA. Maternal cellular DNA can be obtained from a biological sample devoid of fetal genomic DNA. For example, maternal DNA can be obtained from the buffy coat layer of a maternal blood. Haplotypes comprising a plurality of polymorphic sequences that span entire chromosomes can be determined according the present method. In one embodiment, the fetal haplotypes are compared to known disorder-associated haplotypes, and based on a match of the fetal haplotype with any one of the known disorder-associated haplotypes indicates that the fetus has the disorder or that the fetus is susceptible for the disorder. Fetal haplotypes can also be compared to haplotypes associated with treatment responsiveness or unresponsiveness of the specific polymorphism. Comparison of the identified fetal haplotypes to known haplotype databases allow for the diagnosis and/or prognosis of a disorder. Any biological sample comprising a mixture of fetal and maternal cfDNA can be used to determine the presence or absence of the fetal disorder. Preferably, the biological sample is selected from blood, or fractions thereof including plasma, or urine. In one embodiment, the biological sample is a blood sample. In another embodiment, the biological sample is a plasma sample. In yet another embodiment, the biological sample is a urine sample.</div>
</li> <li> <para-num num="[0172]"> </para-num> <div class="description-line" id="p-0173" num="0172">In one embodiment, the invention provides a method for determining the presence or absence of multiple fetal disorders, comprising (a) obtaining a maternal blood sample comprising a cell-free mixture of fetal and maternal DNA, (b) whole genome sequencing at least a portion of the cell-free mixture of fetal and maternal DNA, thereby obtaining a plurality of sequence tags; (c) determining multiple fetal polymorphisms in the sequence tags, and (d) determining the presence or absence of multiple fetal disorders. Examples of multiple fetal disorders that can be identified according to the present method include monogenic and polygenic disorders described herein.</div>
</li> <li> <para-num num="[0173]"> </para-num> <div class="description-line" id="p-0174" num="0173">In one embodiment, the invention provides a method for determining the presence or absence of multiple fetal disorders that comprises identifying multiple fetal polymorphisms associated with multiple disorders related haplotypes. In some embodiments, each of the haplotypes comprises at least at least two, at least three, at least four, at least five, at least ten or at least fifteen different tag polymorphisms. The tag polymorphisms present in the haplotype can be of the same type of polymorphism e.g. all tag SNP polymorphisms, or can be a combination of polymorphisms e.g. tag SNPs and tag deletions. In one embodiment, the polymorphisms are tag SNPs. In another embodiment, the polymorphisms are tag STRs. In yet another embodiment, the polymorphisms are a combination of tag SNPs and tag STRs. The tag polymorphisms can be in coding and/or non-coding regions of the genome. Identification of the polymorphisms is performed by whole genome sequencing using NGS technologies as described herein.</div>
</li> <li> <para-num num="[0174]"> </para-num> <div class="description-line" id="p-0175" num="0174">The invention provides a method for identifying copy number variations (CNV) as polymorphisms of a sequence of interest in a test sample that comprises a mixture of nucleic acids derived from two different genomes, and which are known or are suspected to differ in the amount of one or more sequence of interest. Copy number variations determined by the method of the invention include gains or losses of entire chromosomes, alterations involving very large chromosomal segments that are microscopically visible, and an abundance of sub-microscopic copy number variation of DNA segments ranging from kilobases (kb) to megabases (Mb) in size.</div>
</li> <li> <para-num num="[0175]"> </para-num> <div class="description-line" id="p-0176" num="0175">CNV in the human genome significantly influence human diversity and predisposition to disease (Redon et al., Nature 23:444-454 [2006], Shaikh et al. Genome Res 19:1682-1690 [2009]). CNVs have been known to contribute to genetic disease through different mechanisms, resulting in either imbalance of gene dosage or gene disruption in most cases. In addition to their direct correlation with genetic disorders, CNVs are known to mediate phenotypic changes that can be deleterious. Recently, several studies have reported an increased burden of rare or de novo CNVs in complex disorders such as Autism, ADHD, and schizophrenia as compared to normal controls, highlighting the potential pathogenicity of rare or unique CNVs (Sebat et al., 316:445-449 [2007]; Walsh et al., Science 320:539-543 [2008]). CNV arise from genomic rearrangements, primarily owing to deletion, duplication, insertion, and unbalanced translocation events.</div>
</li> <li> <para-num num="[0176]"> </para-num> <div class="description-line" id="p-0177" num="0176">Embodiments of the invention provide for a method to assess copy number variation of a sequence of interest e.g. a clinically-relevant sequence, in a test sample that comprises a mixture of nucleic acids derived from two different genomes, and which are known or are suspected to differ in the amount of one or more sequence of interest. The mixture of nucleic acids is derived from two or more types of cells. In one embodiment, the mixture of nucleic acids is derived from normal and cancerous cells derived from a subject suffering from a medical condition e.g. cancer.</div>
</li> <li> <para-num num="[0177]"> </para-num> <div class="description-line" id="p-0178" num="0177">It is believed that many solid tumors, such as breast cancer, progress from initiation to metastasis through the accumulation of several genetic aberrations. [Sato et al., Cancer Res., 50: 7184-7189 [1990]; Jongsma et al., J Clin Pathol: Mol Path 55:305-309 [2002])]. Such genetic aberrations, as they accumulate, may confer proliferative advantages, genetic instability and the attendant ability to evolve drug resistance rapidly, and enhanced angiogenesis, proteolysis and metastasis. The genetic aberrations may affect either recessive “tumor suppressor genes” or dominantly acting oncogenes. Deletions and recombination leading to loss of heterozygosity (LOH) are believed to play a major role in tumor progression by uncovering mutated tumor suppressor alleles.</div>
</li> <li> <para-num num="[0178]"> </para-num> <div class="description-line" id="p-0179" num="0178">cfDNA has been found in the circulation of patients diagnosed with malignancies including but not limited to lung cancer (Pathak et al. Clin Chem 52:1833-1842 [2006]), prostate cancer (Schwartzenbach et al. Clin Cancer Res 15:1032-8 [2009]), and breast cancer (Schwartzenbach et al. available online at breast-cancer-research.com/content/11/5/R71 [2009]). Identification of genomic instabilities associated with cancers that can be determined in the circulating cfDNA in cancer patients is a potential diagnostic and prognostic tool. In one embodiment, the method of the invention assesses CNV of a sequence of interest in a sample comprising a mixture of nucleic acids derived from a subject that is suspected or is known to have cancer e.g. carcinoma, sarcoma, lymphoma, leukemia, germ cell tumors and blastoma. In one embodiment, the sample is a plasma sample derived (processes) from peripheral blood and that comprises a mixture of cfDNA derived from normal and cancerous cells. In another embodiment, the biological sample that is needed to determine whether a CNV is present is derived from a mixture of cancerous and non-cancerous cells from other biological fluids including but not limited to serum, sweat, tears, sputum, urine, sputum, ear flow, lymph, saliva, cerebrospinal fluid, ravages, bone marrow suspension, vaginal flow, transcervical lavage, brain fluid, ascites, milk, secretions of the respiratory, intestinal and genitourinary tracts, and leukophoresis samples, or in tissue biopsies, swabs or smears.</div>
</li> <li> <para-num num="[0179]"> </para-num> <div class="description-line" id="p-0180" num="0179">The sequence of interest is a nucleic acid sequence that is known or is suspected to play a role in the development and/or progression of the cancer. Examples of a sequence of interest include nucleic acids sequences that are amplified or deleted in cancerous cells as described in the following.</div>
</li> <li> <para-num num="[0180]"> </para-num> <div class="description-line" id="p-0181" num="0180">Dominantly acting genes associated with human solid tumors typically exert their effect by overexpression or altered expression. Gene amplification is a common mechanism leading to upregulation of gene expression. Evidence from cytogenetic studies indicates that significant amplification occurs in over 50% of human breast cancers. Most notably, the amplification of the proto-oncogene human epidermal growth factor receptor 2 (HER2) located on chromosome 17 (17(17q21-q22)), results in overexpression of HER2 receptors on the cell surface leading to excessive and dysregulated signaling in breast cancer and other malignancies (Park et al., Clinical Breast Cancer 8:392-401 [2008]). A variety of oncogenes have been found to be amplified in other human malignancies. Examples of the amplification of cellular oncogenes in human tumors include amplifications of: c-myc in promyelocytic leukemia cell line HL60, and in small-cell lung carcinoma cell lines, N-myc in primary neuroblastomas (stages III and IV), neuroblastoma cell lines, retinoblastoma cell line and primary tumors, and small-cell lung carcinoma lines and tumors, L-myc in small-cell lung carcinoma cell lines and tumors, c-myb in acute myeloid leukemia and in colon carcinoma cell lines, c-erbb in epidermoid carcinoma cell, and primary gliomas, c-K-ras-2 in primary carcinomas of lung, colon, bladder, and rectum, N-ras in mammary carcinoma cell line (Varmus H., Ann Rev Genetics 18: 553-612 (1984) [cited in Watson et al., Molecular Biology of the Gene (4th ed.; Benjamin/Cummings Publishing Co. 1987)].</div>
</li> <li> <para-num num="[0181]"> </para-num> <div class="description-line" id="p-0182" num="0181">Chromosomal deletions involving tumor suppressor genes may play an important role in the development and progression of solid tumors. The retinoblastoma tumor suppressor gene (Rb-1), located in chromosome 13q14, is the most extensively characterized tumor suppressor gene. The Rb-1 gene product, a 105 kDa nuclear phosphoprotein, apparently plays an important role in cell cycle regulation (Howe et al., Proc Natl Acad Sci (USA) 87:5883-5887 [1990]). Altered or lost expression of the Rb protein is caused by inactivation of both gene alleles either through a point mutation or a chromosomal deletion. Rb-i gene alterations have been found to be present not only in retinoblastomas but also in other malignancies such as osteosarcomas, small cell lung cancer (Rygaard et al., Cancer Res 50: 5312-5317 [1990)]) and breast cancer. Restriction fragment length polymorphism (RFLP) studies have indicated that such tumor types have frequently lost heterozygosity at 13q suggesting that one of the Rb-1 gene alleles has been lost due to a gross chromosomal deletion (Bowcock et al., Am J Hum Genet, 46: 12 [1990]). Chromosome 1 abnormalities including duplications, deletions and unbalanced translocations involving chromosome 6 and other partner chromosomes indicate that regions of chromosome 1, in particular 1q21-1q32 and 1p11-13, might harbor oncogenes or tumor suppressor genes that are pathogenetically relevant to both chronic and advanced phases of myeloproliferative neoplasms (Caramazza et al., Eur J Hemato184:191-200 [2010]). Myeloproliferative neoplasms are also associated with deletions of chromosome 5. Complete loss or interstitial deletions of chromosome 5 are the most common karyotypic abnormality in myelodysplastic syndromes (MDSs). Isolated del(5q)/5q-MDS patients have a more favorable prognosis than those with additional karyotypic defects, who tend to develop myeloproliferative neoplasms (MPNs) and acute myeloid leukemia. The frequency of unbalanced chromosome 5 deletions has led to the idea that 5q harbors one or more tumor-suppressor genes that have fundamental roles in the growth control of hematopoietic stem/progenitor cells (HSCs/HPCs). Cytogenetic mapping of commonly deleted regions (CDRs) centered on 5q31 and 5q32 identified candidate tumor-suppressor genes, including the ribosomal subunit RPS 14, the transcription factor Egr1/Krox20 and the cytoskeletal remodeling protein, alpha-catenin (Eisenmann et al., Oncogene 28:3429-3441 [2009]). Cytogenetic and allelotyping studies of fresh tumours and tumour cell lines have shown that allelic loss from several distinct regions on chromosome 3p, including 3p25, 3p21-22, 3p21.3, 3p12-13 and 3p14, are the earliest and most frequent genomic abnormalities involved in a wide spectrum of major epithelial cancers of lung, breast, kidney, head and neck, ovary, cervix, colon, pancreas, esophagous, bladder and other organs. Several tumor suppressor genes have been mapped to the chromosome 3p region, and are thought that interstitial deletions or promoter hypermethylation precede the loss of the 3p or the entire chromosome 3 in the development of carcinomas (Angeloni D., Briefings Functional Genomics 6:19-39 [2007]).</div>
</li> <li> <para-num num="[0182]"> </para-num> <div class="description-line" id="p-0183" num="0182">Newborns and children with Down syndrome (DS) often present with congenital transient leukemia and have an increased risk of acute myeloid leukemia and acute lymphoblastic leukemia. Chromosome 21, harboring about 300 genes, may be involved in numerous structural aberrations, e.g., translocations, deletions, and amplifications, in leukemias, lymphomas, and solid tumors. Moreover, genes located on chromosome 21 have been identified that play an important role in tumorigenesis. Somatic numerical as well as structural chromosome 21 aberrations are associated with leukemias, and specific genes including RUNX1, TMPRSS2, and TFF, which are located in 21q, play a role in tumorigenesis (Fonatsch C Gene Chromosomes Cancer 49:497-508 [2010]).</div>
</li> <li> <para-num num="[0183]"> </para-num> <div class="description-line" id="p-0184" num="0183">In one embodiment, the method provides a means to assess the association between gene amplification and the extent of tumor evolution. Correlation between amplification and/or deletion and stage or grade of a cancer may be prognostically important because such information may contribute to the definition of a genetically based tumor grade that would better predict the future course of disease with more advanced tumors having the worst prognosis. In addition, information about early amplification and/or deletion events may be useful in associating those events as predictors of subsequent disease progression. Gene amplification and deletions as identified by the method can be associated with other known parameters such as tumor grade, histology, Brd/Urd labeling index, hormonal status, nodal involvement, tumor size, survival duration and other tumor properties available from epidemiological and biostatistical studies. For example, tumor DNA to be tested by the method could include atypical hyperplasia, ductal carcinoma in situ, stage I-III cancer and metastatic lymph nodes in order to permit the identification of associations between amplifications and deletions and stage. The associations made may make possible effective therapeutic intervention. For example, consistently amplified regions may contain an overexpressed gene, the product of which may be able to be attacked therapeutically (for example, the growth factor receptor tyrosine kinase, p185<sup>HER2</sup>).</div>
</li> <li> <para-num num="[0184]"> </para-num> <div class="description-line" id="p-0185" num="0184">The method can be used to identify amplification and/or deletion events that are associated with drug resistance by determining the copy number variation of nucleic acids from primary cancers to those of cells that have metastasized to other sites. If gene amplification and/or deletion is a manifestation of karyotypic instability that allows rapid development of drug resistance, more amplification and/or deletion in primary tumors from chemoresistant patients than in tumors in chemosensitive patients would be expected. For example, if amplification of specific genes is responsible for the development of drug resistance, regions surrounding those genes would be expected to be amplified consistently in tumor cells from pleural effusions of chemoresistant patients but not in the primary tumors. Discovery of associations between gene amplification and/or deletion and the development of drug resistance may allow the identification of patients that will or will not benefit from adjuvant therapy.</div>
</li> <li> <para-num num="[0185]"> </para-num> <div class="description-line" id="p-0186" num="0185">In other embodiments, the present method can be used to identify polymorphisms associated with trinucleotide repeat disorders, which are a set of genetic disorders caused by trinucleotide repeat expansion. Trinucleotide expansions are a subset of unstable microsatellite repeats that occur throughout all genomic sequences. If the repeat is present in a healthy gene, a dynamic mutation may increase the repeat count and result in a defective gene. In one embodiment, the method can be used to identify trinucleotide repeats associated with fragile X syndrome. The long arm of the X chromosome of patients suffering from fragile X syndrome can contain from 230 to 4000 CGG, as compared with 60 to 230 repeats in carriers and 5 to 54 repeats in unaffected individuals. The chromosomal instability resulting from this trinucleotide expansion presents clinically as mental retardation, distinctive facial features, and macroorchidism in males. The second, related DNA-triplet repeat disease, fragile X-E syndrome, was also identified on the X chromosome, but was found to be the result of an expanded CCG repeat. The present method can identify trinucleotide repeats associated with other repeat expansion disorders including Categories I, II and III. Category I disorders include Huntington's disease (HD) and the spinocerebellar ataxias that are caused by a CAG repeat expansion in protein-coding portions of specific genes. Category II expansions tend to be more phenotypically diverse with heterogeneous expansions that are generally small in magnitude, but also found in the exons of genes. Category III includes fragile X syndrome, myotonic dystrophy, two of the spinocerebellar ataxias, juvenile myoclonic epilepsy and friereich's ataxia. These diseases are characterized by typically much larger repeat expansions than the first two groups, and the repeats are located outside of the protein-coding regions of the genes.</div>
</li> <li> <para-num num="[0186]"> </para-num> <div class="description-line" id="p-0187" num="0186">In other embodiments, the present method can identify CAG trinucleotide repeats associated with at least ten neurologic disorders known to be caused by an increased number of CAG repeats, typically in coding regions of otherwise unrelated proteins. During protein synthesis, the expanded CAG repeats are translated into a series of uninterrupted glutamine residues forming what is known as a polyglutamine tract (“polyQ”). Such polyglutamine tracts may be subject to increased aggregation. These disorders are characterized by autosomal dominant mode of inheritance (with the exception of spino-bulbar muscular atrophy which shows X-linked inheritance), midlife onset, a progressive course, and a correlation of the number of CAG repeats with the severity of disease and the age at onset. Causative genes are widely expressed in all of the known polyglutamine diseases. A common symptom of PolyQ diseases is characterized by a progressive degeneration of nerve cells usually affecting people later in life. Although these diseases share the same repeated codon (CAG) and some symptoms, the repeats for the different polyglutamine diseases occur on different chromosomes. Examples of polyQ disorders that can be identified by the present method include without limitation DRPLA (Dentatorubropallidoluysian atrophy), HD (Huntington's disease), SBMA (Spinobulbar muscular atrophy or Kennedy disease), SCA1 (Spinocerebellar ataxia Type 1), SCA2 (Spinocerebellar ataxia Type 2), SCA3 (Spinocerebellar ataxia Type 3 or Machado-Joseph disease), SCA6 (Spinocerebellar ataxia Type 6), SCA7 (Spinocerebellar ataxia Type 7), SCA17 (Spinocerebellar ataxia Type 17). Examples of non-polyQ disorders that can be identified by the present method include FRAXA (Fragile X syndrome), FXTAS (Fragile X-associated tremor/ataxia syndrome), FRAXE (Fragile XE mental retardation), FRDA (Friedreich's ataxia), DM (Myotonic dystrophy), SCA8 (Spinocerebellar ataxia Type 8), SCA12 (Spinocerebellar ataxia Type 12).</div>
</li> <li> <para-num num="[0187]"> </para-num> <div class="description-line" id="p-0188" num="0187">The present method used for identifying polymorphisms associated with multiple can be determined can be used in conjunction with methods for the determination of aneuploidy such as the methods described in U.S. Patent Application Publication Nos. US 2007/0202525A1; US2010/0112575A1, US 2009/0087847A1; US2009/0029377A1; US 2008/0220422A1; US2008/0138809A1, US2008/0153090A1, US2010/0138165, US 2010/0112590, U.S. Pat. No. 7,645,576, Fan et al., PNAS 105:16266-16271 [2008]; Chiu et al. PNAS 105:20458-20463 [2008]; Chiu et al., BMJ 2011; 342:c7401 [2011]; and pending U.S. patent application Ser. No. 12/958,352 “Method for Determining Copy Number Variations”, and Ser. No. 12/958,353 “Sequencing Method sand Compositions for Prenatal Diagnoses”, which are herein incorporated by reference in their entirety. The method can also be combined with assays for determining other prenatal conditions associated with the mother and/or the fetus. For example, the method can be used in conjunction with prenatal analyses, for example, as described in U.S. Patent Application Publication Nos. US2010/0112590A1, US2009/0162842A1, US2007/0207466A1, and US2001/0051341A1, all of which are incorporated by reference in their entirety.</div>
</li> <li> <para-num num="[0188]"> </para-num> <div class="description-line" id="p-0189" num="0188">The present invention is described in further detail in the following Examples which are not in any way intended to limit the scope of the invention as claimed. The attached Figures are meant to be considered as integral parts of the specification and description of the invention. The following examples are offered to illustrate, but not to limit the claimed invention.</div>
</li> <heading id="h-0009">EXAMPLES</heading>
<heading id="h-0010">Example 1</heading>
<heading id="h-0011">Plasma Cell-Free DNA Comprises an Entire Genome</heading>
<li> <para-num num="[0189]"> </para-num> <div class="description-line" id="p-0190" num="0189">To demonstrate that plasma cfDNA comprises sequences that cover the entire human genome, experimental information obtained from sequencing cfDNA from maternal plasma samples was compared to the expected percent genome covered by a given number of sequence tags of a given tag length.</div>
</li> <li> <para-num num="[0190]"> </para-num> <div class="description-line" id="p-0191" num="0190">Experimental Data: Experimental Determination of Genome Coverage by Sequencing cfDNA from Maternal Plasma.</div>
</li> <li> <para-num num="[0191]"> </para-num> <div class="description-line" id="p-0192" num="0191">The percent genome coverage was determined experimentally by sequencing cfDNA that was purified from the plasma fraction of a peripheral blood sample from 24 women each pregnant with an unaffected fetus identified as karyotypically normal (either 46, XX or 46, XY). DNA was extracted from plasma of peripheral blood samples using the QIAamp DNA Mini kit (Qiagen) with modifications to the manufacturer's instruction. 4.5 ml-5 ml of cell-free plasma was transferred into a 50 mL conical tube. PBS was added to bring the sample to 5 mL total volume, and the sample was treated with 500 μL Qiagen Protease solution. The sample was mixed and 5 mL Buffer AL was added to the sample solution. The sample was vortexed at 2500 RPM for 10 seconds, and incubated at 56° C. for 12 minutes. 5 mL 100% ethanol was added to the sample solution, and the solution was vortexed at 2500 RPM for 5 seconds. 590 μL aliquots of sample solution were transferred to each of 6 mini columns. The columns were centrifuged at 8,000 RPM for 30 seconds. The process was repeated three more times. On 4th and final spin, even out all column volumes were evened out, and the columns were spun for 1 minute. 500 μL wash AW1 was pipetted onto columns, and the columns were spun at 8,000 RPM for 1 minute. The columns were transferred to Qiagen collection tubes, and washed with 500 μL of wash buffer by centrifugation at 14,000 RPM for 3 minutes. The columns were transferred to new clean Qiagen collection tubes, and centrifuged for an additional one minute. 55 μL of H2O or Elution Buffer (EB) was added to each of the columns, which were incubated at room temperature for 5 minutes. Purified cfDNA samples were eluted by centrifugation at 8,000 RPM for 2 minutes and stored at −80° C.</div>
</li> <li> <para-num num="[0192]"> </para-num> <div class="description-line" id="p-0193" num="0192">cfDNA was purified from plasma using reagents contained in the QIAamp DNA Mini kit (Qiagen) essentially according to the method used for extracting genomic DNA. 500 μL Qiagen Protease were added to 4.5 ml-5 ml of cell-free plasma, and mixed for 5 seconds. 5 mL Buffer AL were added to the sample solution, mixed for 10 seconds, and the mixture was incubated in sealed tubes 56° C. for 12 minutes. 5 mL 100% ethanol were added to the sample solution, the sample was mixed for 5 seconds, and 590 μL of sample solution was applied to each of 6 mini columns. The columns were centrifuged at 8,000 RPM for 30 seconds. Aliquots of the remaining sample solution were applied to the six mini columns as described. The columns were washed with 500 μL AW1, centrifuged at 8,000 RPM for 1 minute, transferred to a new Qiagen collection tube, washed with 500 μL of wash AW2, centrifuged at 14,000 RPM for 3 minutes, and again at 14,000 RPM for 1 minute. The columns were transferred to a sample collection tube. 55 μL of H2O or Elution Buffer (EB) was applied to the columns, and the columns were incubated at room temperature for 5 minutes before being centrifuged at 8,000 RPM for 2 minutes. The eluted samples were eluted and stored at −80° C. The purified cfDNA was sent to a service provider (Prognosis Biosciences, La Jolla, Calif.) for preparation of the sequencing libraries and for sequencing using the Illumina GAII (Illumina).</div>
</li> <li> <para-num num="[0193]"> </para-num> <div class="description-line" id="p-0194" num="0193">Image analysis of the sequencing information comprised in 36 bp reads was carried out at Prognosys Biosciences, utilizing the Integrated Primary Analysis and Reporting software (IPAR v1.3), and results were provided as the Data Folder portion of the Run Folder output (See Genome Analyzer Pipeline v1.3 and Consensus Assessment of Sequence and Variation (CASAVA) v1.0 Software User Guide from Illumina). The standard bustard.py script was used to generate base calls with the GERALD module to do sequence alignments to the reference human genome hg18. The 24 export.txt (aligned sequence tag) files each comprising about 5×10<sup>6 </sup>tags were utilized as input into the (CASAVA) software package, treated as if they came from a single individual, and analyzed as a genome build while including examination of sequence differences between the combined sample and the reference genome sequence (in this case hg18). CASAVA provided coverage/depth statistics as well as SNP statistics for individual chromosomes.</div>
</li> <li> <para-num num="[0194]"> </para-num> <div class="description-line" id="p-0195" num="0194">The relative coverage of individual chromosomes by the sequence tags is shown in <figref idrefs="DRAWINGS">FIG. 1</figref> as a function of depth, wherein a maximum depth of 1 represents 100% coverage of each chromosome. <figref idrefs="DRAWINGS">FIG. 2</figref> shows the number of SNPs that were identified and mapped to each of the chromosomes from the genome build (<figref idrefs="DRAWINGS">FIG. 1</figref>) compared to the reference sequences. The SNPs that were counted were not differentiated between the maternal and fetal genome i.e. all counted SNPs had the same polymorphic sequence.</div>
</li> <li> <para-num num="[0195]"> </para-num> <div class="description-line" id="p-0196" num="0195">Predicted Data: Predicted Genome Coverage of cfDNA in Maternal Plasma.</div>
</li> <li> <para-num num="[0196]"> </para-num> <div class="description-line" id="p-0197" num="0196">Given the limited genome coverage obtained from sequencing information of the 24 samples, we examined the experimental results in light of the theoretical expectation. Given a specified genome length, number of sequence tags (mapped chromosome reads), and tag length, the expected proportion (or percentage) of genome coverage was calculated by considering the probability density function for the binomial distribution</div>
</li> <li> <div class="description-line" id="p-0198" num="0000"> <br/> <i>f</i>(<i>k|n,p</i>)=<i>Pr</i>(<i>K=k</i>)=(<sub>k</sub> <sup>n</sup>)<i>p</i> <sup>k</sup>(1−<i>p</i>)<sup>n-k </sup>
</div>
</li> <li> <para-num num="[0197]"> </para-num> <div class="description-line" id="p-0199" num="0197">where k is the number of tags at a given position, n is the number of tags mapped to the genome, and p is the probability of a tag mapping to a given position. Herein, we estimated that p=tag length/genome length.</div>
</li> <li> <para-num num="[0198]"> </para-num> <div class="description-line" id="p-0200" num="0198">Assuming the experimental conditions used in sequencing cfDNA in maternal plasma i.e. that each sequence tag is 30 bases; that 5×10<sup>6 </sup>tags are obtained in a single sequencing event; that the total number of tags are determined from 24 sequencing events; and assuming that the entire genome of approximately 3×10<sup>9 </sup>bases is present in the, the distribution of (5×10<sup>6</sup>)(24)=1.2×10<sup>8 </sup>tags across a single genome was analyzed and shown in <figref idrefs="DRAWINGS">FIG. 3</figref>.</div>
</li> <li> <para-num num="[0199]"> </para-num> <div class="description-line" id="p-0201" num="0199"> <figref idrefs="DRAWINGS">FIG. 3</figref> shows that approximately 30% of the genome is predicted not to be covered by at least one of the 1.2×10<sup>8 </sup>tags, indicating that approximately 70% of the genome is expected to be mapped at least once. Approximately 95% of the mapped genome sites had a tag depth not greater than 3, indicating that most sites were covered by one, two, or three tags.</div>
</li> <li> <para-num num="[0200]"> </para-num> <div class="description-line" id="p-0202" num="0200">The percent genome coverage related to the tag depth i.e. the number of tags that mapped to the same site on the genome was tabulated as follows (Table 1), where the percent expected proportion of the genome that is covered per tag depth is (1−Expected Proportion of Genome Not Covered)×100.</div>
</li> <li> <div class="description-line" id="p-0203" num="0000">
<tables id="TABLE-US-00001" num="00001">
<patent-tables colsep="0" frame="none" rowsep="0">
<table align="left" class="description-table" cols="1" colsep="0" rowsep="0" width="100%">
<thead>
<tr class="description-tr">
<td class="description-td" colspan="1" nameend="1" namest="1" rowsep="1">TABLE 1</td>
</tr>
</thead>
<tbody><tr class="description-tr">
<td align="center" class="description-td" colspan="1" nameend="1" namest="1" rowsep="1"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">Genome Tag Coverage</td>
</tr>
</tbody></table>
<table align="left" class="description-table" cols="3" colsep="0" rowsep="0" width="100%">
<tbody><tr class="description-tr">
<td class="description-td">Coverage</td>
<td class="description-td">Expected Proportion of</td>
<td class="description-td">Expected Proportion of</td>
</tr>
<tr class="description-tr">
<td class="description-td">(tag depth)</td>
<td class="description-td">Genome Not Covered</td>
<td class="description-td">Genome Covered (%)</td>
</tr>
<tr class="description-tr">
<td align="center" class="description-td" colspan="3" nameend="3" namest="1" rowsep="1"> </td>
</tr>
</tbody></table>
<table align="left" class="description-table" cols="3" colsep="0" rowsep="0" width="100%">
<tbody><tr class="description-tr">
<td class="description-td">0</td>
<td class="description-td">0.301</td>
<td class="description-td">69.9</td>
</tr>
<tr class="description-tr">
<td class="description-td">1</td>
<td class="description-td">0.361</td>
<td class="description-td">63.9</td>
</tr>
<tr class="description-tr">
<td class="description-td">2</td>
<td class="description-td">0.217</td>
<td class="description-td">78.3</td>
</tr>
<tr class="description-tr">
<td class="description-td">3</td>
<td class="description-td">0.087</td>
<td class="description-td">91.3</td>
</tr>
<tr class="description-tr">
<td class="description-td">4</td>
<td class="description-td">0.026</td>
<td class="description-td">97.4</td>
</tr>
<tr class="description-tr">
<td class="description-td">5</td>
<td class="description-td">0.006</td>
<td class="description-td">99.4</td>
</tr>
<tr class="description-tr">
<td class="description-td">6</td>
<td class="description-td">0.001</td>
<td class="description-td">99.9</td>
</tr>
<tr class="description-tr">
<td class="description-td">7</td>
<td class="description-td">0.000</td>
<td class="description-td">100</td>
</tr>
<tr class="description-tr">
<td class="description-td">8</td>
<td class="description-td">0.000</td>
<td class="description-td">100</td>
</tr>
<tr class="description-tr">
<td class="description-td">9</td>
<td class="description-td">0.000</td>
<td class="description-td">100</td>
</tr>
<tr class="description-tr">
<td class="description-td">10</td>
<td class="description-td">0.000</td>
<td class="description-td">100</td>
</tr>
<tr class="description-tr">
<td align="center" class="description-td" colspan="3" nameend="3" namest="1" rowsep="1"> </td>
</tr>
</tbody></table>
</patent-tables>
</tables>
</div>
</li> <li> <para-num num="[0201]"> </para-num> <div class="description-line" id="p-0204" num="0201">The results show that genome coverage determined from the experimental results corresponds with that of the expected coverage, if the entire genome is available in the cfDNA sample. The result implies that even in experiments with limited coverage the cfDNA sample comprises sequences that span an entire human genome. In addition, the experimental data shows that polymorphisms e.g. SNPs can be identified by whole genome sequencing.</div>
</li> <li> <para-num num="[0202]"> </para-num> <div class="description-line" id="p-0205" num="0202">Having determined that polymorphisms can be identified in a sample comprising a mixture of fetal and maternal cfDNA, polymorphisms including SNPs and STRs were identified in maternal samples comprising fetal and maternal cfDNA for determining fetal fraction. Methods that use the identified polymorphisms for the simultaneous determination of fetal fraction and aneuploidy are provided below and in Pending U.S. patent application Ser. No. 12/958,347 entitled “Methods for Determining Fraction of Fetal Nucleic Acids in Maternal Samples”; Ser. No. 12/958,352 entitled “Method for Determining Copy Number Variations”; Ser. No. 12/958,356 entitled “Simultaneous Determination of Aneuploidy and Fetal Fraction”, and Ser. No. 12/958,353 entitled “Methods and Compositions for Prenatal Diagnoses”, all filed on Dec. 1, 2010, which are herein incorporated by reference in their entirety.</div>
</li> <heading id="h-0012">Example 2</heading>
<heading id="h-0013">Identification of SNPs by TaqMan Real-Time PCR in Maternal Samples Comprising a Mixture of Fetal and Maternal cfDNA: Use for Determining Fetal Fraction</heading>
<li> <para-num num="[0203]"> </para-num> <div class="description-line" id="p-0206" num="0203">Having determined that polymorphisms can be identified in a sample comprising a mixture of fetal and maternal cfDNA, initial experiments were performed to identify informative SNPs by TaqMan Real-Time PCR in plasma samples of pregnant women, and for determining the fraction of fetal cfDNA.</div>
</li> <li> <para-num num="[0204]"> </para-num> <div class="description-line" id="p-0207" num="0204">Sample Processing and DNA Extraction</div>
</li> <li> <para-num num="[0205]"> </para-num> <div class="description-line" id="p-0208" num="0205">Peripheral blood drawn from each subject was collected in ACD tubes. One tube of blood sample (approximately 6-9 mL/tube) was transferred into one 15-mL low speed centrifuge tube. Blood was centrifuged at 2640 rpm, 4° C. for 10 min using Beckman Allegra 6 R centrifuge and rotor model GA 3.8. The buffy coat was collected and used as a source of genomic DNA. The upper plasma layer was collected and used for preparing cfDNA.</div>
</li> <li> <para-num num="[0206]"> </para-num> <div class="description-line" id="p-0209" num="0206">Genomic DNA was extracted from 1 ml of whole blood or buffy coat sample using QIAamp DNA Blood Mini kit (Qiagen) with modifications to the manufacturer's instructions as follows. One hundred μl of Qiagen Protease was added to 1 ml sample, and mixed for 10 seconds. One milliliter of Buffer AL was added to the sample, mixed for 20 seconds, and the mixture was incubated in sealed tubes at 56° C. for 10 minutes. One milliter of 100% ethanol was added to the sample solution, the sample was mixed for 15 seconds, and 620 μL of sample solution was applied to each of 2 mini columns. The columns were centrifuged at 8,000 RPM for 1 minute, washed with 500 μL AW1 solution, centrifuged at 8,000 RPM for 1 minute, transferred to a new Qiagen collection tube, washed with 500 μL of AW2, centrifuged once at 14,000 RPM for 3 minutes, and again at 14,000 RPM for 1 minute. The columns were transferred to sample collection tube. 200 μL of H<sub>2</sub>O was applied to the columns, and the columns were incubated at room temperature for 5 minutes before being centrifuged at 8,000 RPM for 2 minutes. The eluted samples were stored in one cryostorage tube at −80° C.</div>
</li> <li> <para-num num="[0207]"> </para-num> <div class="description-line" id="p-0210" num="0207">Cell-free DNA was extracted from cell-free plasma as described in Example 1.</div>
</li> <li> <para-num num="[0208]"> </para-num> <div class="description-line" id="p-0211" num="0208">Identification of Maternal and Fetal Genotype Using SNP Alleles by TaqMan Real-Time PCR Analysis</div>
</li> <li> <para-num num="[0209]"> </para-num> <div class="description-line" id="p-0212" num="0209">A SNP panel (Table 2, Pakstis et al., Hum Genet. 121:305-317 [2007]) was used to determine the maternal and fetal SNP genotypes. TaqMan amplification reactions were set up in a reaction volume of 25 μl. Each reaction contained 1× TaqMan Universal PCR master mix, 0.9 μM forward and reverse primers and 0.2 μM probe, 1 ng of genomic DNA to determine the maternal genotypes or 0.3-1.5 ng of cell-free DNA to determine the fetal genotypes. Thermal cycling was 10 min at 95° C. and then 40 cycles of 95° C. for 15 seconds and 60° C. for 1 min. Amplification data was collected using 7500 Sequence Detector and then analyzed using the Sequence Detection System Software (Applied Biosystems).</div>
</li> <li> <para-num num="[0210]"> </para-num> <div class="description-line" id="p-0213" num="0210">SNPs present in maternal DNA were genotyped for the first 10 SNPs in the panel listed in Table 2. The maternal origin was verified by confirming the absence of an SRY (Sex-determination region on Y chromosome) allele. Subsequently, a sample of genotype matched cell-free DNA containing cell-free fetal DNA, was interrogated for the same 10 SNPs. Informative SNP alleles were detected, and used to determine the percentage of fetal fraction in the total cell-free DNA.</div>
</li> <li> <div class="description-line" id="p-0214" num="0000">
<tables id="TABLE-US-00002" num="00002">
<patent-tables colsep="0" frame="none" rowsep="0">
<table align="left" class="description-table" cols="1" colsep="0" rowsep="0" width="100%">
<thead>
<tr class="description-tr">
<td class="description-td" colspan="1" nameend="1" namest="1" rowsep="1">TABLE 2</td>
</tr>
</thead>
<tbody><tr class="description-tr">
<td align="center" class="description-td" colspan="1" nameend="1" namest="1" rowsep="1"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">SNP panel used for TaqMan real-time PCR genotyping</td>
</tr>
</tbody></table>
<table align="left" class="description-table" cols="4" colsep="0" rowsep="0" width="100%">
<tbody><tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">NCBI SNP</td>
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">SNP #</td>
<td class="description-td">Assay ID</td>
<td class="description-td">Reference</td>
<td class="description-td">Cytogenetic Band</td>
</tr>
<tr class="description-tr">
<td align="center" class="description-td" colspan="4" nameend="4" namest="1" rowsep="1"> </td>
</tr>
</tbody></table>
<table align="left" class="description-table" cols="4" colsep="0" rowsep="0" width="100%">
<tbody><tr class="description-tr">
<td class="description-td">1</td>
<td class="description-td">C<sub>———</sub>3032822_1<sub>—</sub> </td>
<td class="description-td">rs315791</td>
<td class="description-td">5q35.1c</td>
</tr>
<tr class="description-tr">
<td class="description-td">2</td>
<td class="description-td">C<sub>———</sub>2822618_10</td>
<td class="description-td">rs3780962</td>
<td class="description-td">10p13a</td>
</tr>
<tr class="description-tr">
<td class="description-td">3</td>
<td class="description-td">C<sub>———</sub>7538108_10</td>
<td class="description-td">rs1410059</td>
<td class="description-td">10q23.33d</td>
</tr>
<tr class="description-tr">
<td class="description-td">4</td>
<td class="description-td">C<sub>———</sub>8263011_10</td>
<td class="description-td">rs279844</td>
<td class="description-td">4p12b</td>
</tr>
<tr class="description-tr">
<td class="description-td">5</td>
<td class="description-td">C<sub>———</sub>3153696_10</td>
<td class="description-td">rs338882</td>
<td class="description-td">5q35.3d</td>
</tr>
<tr class="description-tr">
<td class="description-td">6</td>
<td class="description-td">C<sub>———</sub>1371205_10</td>
<td class="description-td">rs9951171</td>
<td class="description-td">18p11.22b</td>
</tr>
<tr class="description-tr">
<td class="description-td">7</td>
<td class="description-td">C<sub>———</sub>2515223_10</td>
<td class="description-td">rs214955</td>
<td class="description-td">6q25.2a</td>
</tr>
<tr class="description-tr">
<td class="description-td">8</td>
<td class="description-td">C<sub>——</sub>25749280_10</td>
<td class="description-td">rs6444724</td>
<td class="description-td">3q29b</td>
</tr>
<tr class="description-tr">
<td class="description-td">9</td>
<td class="description-td">C<sub>——</sub> <sub>——</sub>411273_10</td>
<td class="description-td">rs2503107</td>
<td class="description-td">6q22.33a</td>
</tr>
<tr class="description-tr">
<td class="description-td">10</td>
<td class="description-td">C<sub>———</sub>2572254_10</td>
<td class="description-td">rs1019029</td>
<td class="description-td">7p21.2a</td>
</tr>
<tr class="description-tr">
<td class="description-td">11</td>
<td class="description-td">C<sub>———</sub>1732269_10</td>
<td class="description-td">rs1413212</td>
<td class="description-td">1q43f</td>
</tr>
<tr class="description-tr">
<td class="description-td">12</td>
<td class="description-td">C<sub>———</sub>8953333_10</td>
<td class="description-td">rs1031825</td>
<td class="description-td">20p13a</td>
</tr>
<tr class="description-tr">
<td class="description-td">13</td>
<td class="description-td">C<sub>———</sub>7539584_10</td>
<td class="description-td">rs891700</td>
<td class="description-td">1q43c</td>
</tr>
<tr class="description-tr">
<td class="description-td">14</td>
<td class="description-td">C<sub>———</sub>2203431_10</td>
<td class="description-td">rs1005533</td>
<td class="description-td">20q12b</td>
</tr>
<tr class="description-tr">
<td class="description-td">15</td>
<td class="description-td">C<sub>———</sub>2528441_20</td>
<td class="description-td">rs2831700</td>
<td class="description-td">21q21.3b</td>
</tr>
<tr class="description-tr">
<td class="description-td">16</td>
<td class="description-td">C<sub>———</sub>3084646_10</td>
<td class="description-td">rs354439</td>
<td class="description-td">13q33.2b</td>
</tr>
<tr class="description-tr">
<td class="description-td">17</td>
<td class="description-td">C<sub>——</sub>12098080_10</td>
<td class="description-td">rs1979255</td>
<td class="description-td">4q35.2d</td>
</tr>
<tr class="description-tr">
<td class="description-td">18</td>
<td class="description-td">C<sub>———</sub>2120263_10</td>
<td class="description-td">rs1454361</td>
<td class="description-td">14q12a</td>
</tr>
<tr class="description-tr">
<td class="description-td">19</td>
<td class="description-td">C<sub>——</sub>29375514_10</td>
<td class="description-td">rs8037429</td>
<td class="description-td">15q21.3a</td>
</tr>
<tr class="description-tr">
<td class="description-td">20</td>
<td class="description-td">C<sub>———</sub>9630073_10</td>
<td class="description-td">rs1490413</td>
<td class="description-td">1p36.32b</td>
</tr>
<tr class="description-tr">
<td align="center" class="description-td" colspan="4" nameend="4" namest="1" rowsep="1"> </td>
</tr>
</tbody></table>
</patent-tables>
</tables>
</div>
</li> <li> <para-num num="[0211]"> </para-num> <div class="description-line" id="p-0215" num="0211">In Table 2 above, Assay ID number refers to SNPs that are commercially available through Applied Biosystems Assay ID. Context sequences are sequences that flank the 2 SNP alleles, which are written in bold and underlined. A dual-labeled fluorescent TaqMan probe for each SNP alleles was used. The 1st of the two alleles was labeled with VIC and the 2nd allele was labeled with FAM. Cytogenetic band refers to the chromosome locations of the SNPs. Forward and reverse primers, and labeled probes were synthesized by Applied Biosystems for each of the SNP sequences provided in Table 1 above.</div>
</li> <li> <para-num num="[0212]"> </para-num> <div class="description-line" id="p-0216" num="0212">SNP Genotyping of Maternal DNA</div>
</li> <li> <para-num num="[0213]"> </para-num> <div class="description-line" id="p-0217" num="0213">Maternal genotypes were determined for the 10 SNPs in the panel given in Table 2 above using TaqMan real-time PCR. All 10 SNPs were genotyped in genomic DNA extracted from buffy coat samples and whole blood samples, and the SRY allele was not detected in any of the samples (SRY %=0%), confirming that the origin of the alleles is maternal. These results demonstrate that maternal genotype can be determined from the genomic DNA extracted from whole blood or buffy coat.</div>
</li> <li> <para-num num="[0214]"> </para-num> <div class="description-line" id="p-0218" num="0214">SNP Genotyping of Fetal DNA</div>
</li> <li> <para-num num="[0215]"> </para-num> <div class="description-line" id="p-0219" num="0215">Fetal genotypes were determined for SNPs 1-10 in the panel of Table 2 above. All 10 SNPs were genotyped in cfDNA extracted from plasma samples as described above. Informative alleles were identified by comparing the genotypes determined in the cfDNA samples with the corresponding genotypes determined in the genomic DNA samples obtained from the pregnant subjects. The genotypes of 5 exemplary maternal and fetal alleles are given in Table 3 below. The informative fetal SNP alleles are highlighted and underlined.</div>
</li> <li> <div class="description-line" id="p-0220" num="0000">
<tables id="TABLE-US-00003" num="00003">
<patent-tables colsep="0" frame="none" pgwide="1" rowsep="0">
<table align="left" class="description-table" cols="1" colsep="0" rowsep="0" width="100%">
<thead>
<tr class="description-tr">
<td class="description-td" colspan="1" nameend="1" namest="1" rowsep="1">TABLE 3</td>
</tr>
</thead>
<tbody><tr class="description-tr">
<td align="center" class="description-td" colspan="1" nameend="1" namest="1" rowsep="1"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">SNP genotypes identified in genomic and cfDNA samples</td>
</tr>
</tbody></table>
<table align="left" class="description-table" cols="11" colsep="0" rowsep="0" width="100%">
<tbody><tr class="description-tr">
<td class="description-td">Sample ID</td>
<td class="description-td">SNP1</td>
<td class="description-td">SNP2</td>
<td class="description-td">SNP3</td>
<td class="description-td">SNP4</td>
<td class="description-td">SNP5</td>
<td class="description-td">SNP6</td>
<td class="description-td">SNP7</td>
<td class="description-td">SNP8</td>
<td class="description-td">SNP9</td>
<td class="description-td">SNP10</td>
</tr>
<tr class="description-tr">
<td align="center" class="description-td" colspan="11" nameend="11" namest="1" rowsep="1"> </td>
</tr>
</tbody></table>
<table align="left" class="description-table" cols="12" colsep="0" rowsep="0" width="100%">
<tbody><tr class="description-tr">
<td class="description-td">24181</td>
<td class="description-td">Maternal</td>
<td class="description-td">A/A</td>
<td class="description-td">G/G</td>
<td class="description-td">C/T</td>
<td class="description-td">A/T</td>
<td class="description-td">A/G</td>
<td class="description-td">A/G</td>
<td class="description-td">C/T</td>
<td class="description-td">C/T</td>
<td class="description-td">A/C</td>
<td class="description-td">G/G</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">(buffy</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">coat)</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">Fetal</td>
<td class="description-td">A/A</td>
<td class="description-td">G/G</td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">
<u>
<b>A/G</b>
</u>
</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">(cfDNA)</td>
</tr>
<tr class="description-tr">
<td class="description-td">31205</td>
<td class="description-td">Maternal</td>
<td class="description-td">C/C</td>
<td class="description-td">G/G</td>
<td class="description-td">C/C</td>
<td class="description-td">A/T</td>
<td class="description-td">A/G</td>
<td class="description-td">A/G</td>
<td class="description-td">T/T</td>
<td class="description-td">C/T</td>
<td class="description-td">A/C</td>
<td class="description-td">G/G</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">(buffy</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">coat)</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">Fetal</td>
<td class="description-td">C/C</td>
<td class="description-td"> </td>
<td class="description-td">
<u>
<b>C/T</b>
</u>
</td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">G/G</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">(cfDNA)</td>
</tr>
<tr class="description-tr">
<td class="description-td">51212</td>
<td class="description-td">Maternal</td>
<td class="description-td">C/C</td>
<td class="description-td">A/G</td>
<td class="description-td">T/T</td>
<td class="description-td">A/A</td>
<td class="description-td">A/G</td>
<td class="description-td">A/G</td>
<td class="description-td">C/T</td>
<td class="description-td">T/T</td>
<td class="description-td">A/C</td>
<td class="description-td">G/G</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">(buffy</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">coat)</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">Fetal</td>
<td class="description-td">C/C</td>
<td class="description-td"> </td>
<td class="description-td">
<u>
<b>C/T</b>
</u>
</td>
<td class="description-td">A/A</td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">
<u>
<b>C/T</b>
</u>
</td>
<td class="description-td"> </td>
<td class="description-td">G/G</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">(cfDNA)</td>
</tr>
<tr class="description-tr">
<td class="description-td">ABRSSJA5516</td>
<td class="description-td">Maternal</td>
<td class="description-td">C/C</td>
<td class="description-td">A/G</td>
<td class="description-td">C/T</td>
<td class="description-td">T/T</td>
<td class="description-td">A/G</td>
<td class="description-td">A/G</td>
<td class="description-td">C/T</td>
<td class="description-td">C/C</td>
<td class="description-td">A/C</td>
<td class="description-td">A/G</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">(buffy</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">coat)</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">Fetal</td>
<td class="description-td">C/C</td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">T/T</td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">
<u>
<b>C/T</b>
</u>
</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">(cfDNA)</td>
</tr>
<tr class="description-tr">
<td class="description-td">ABRSJA5517</td>
<td class="description-td">Maternal</td>
<td class="description-td">A/C</td>
<td class="description-td">G/G</td>
<td class="description-td">C/T</td>
<td class="description-td">A/T</td>
<td class="description-td">A/A</td>
<td class="description-td">A/G</td>
<td class="description-td">C/T</td>
<td class="description-td">C/C</td>
<td class="description-td">A/C</td>
<td class="description-td">A/G</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">(buffy</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">coat)</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">Fetal</td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">
<u>
<b>A/G</b>
</u>
</td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">C/C</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">(cfDNA)</td>
</tr>
<tr class="description-tr">
<td align="center" class="description-td" colspan="12" nameend="12" namest="1" rowsep="1"> </td>
</tr>
</tbody></table>
</patent-tables>
</tables>
</div>
</li> <li> <para-num num="[0216]"> </para-num> <div class="description-line" id="p-0221" num="0216">Determination of Fetal Fraction</div>
</li> <li> <para-num num="[0217]"> </para-num> <div class="description-line" id="p-0222" num="0217">Male and female fetal fractions were determined by quantifying the identified informative SNP alleles using the Fluidigm™ dPCR platform. Simplex digital PCR was performed on the BioMark System using the 12.765 Digital Arrays (Fluidigm, South San Francisco). The real-time PCR amplification was performed using Data Collection Software. The digital array thermocycling conditions consisted of 2 min at 50° C., 10 min at 95° C., and followed by 45 cycles of a two-step thermal profile including denaturation at 95° C. for 15 seconds and combined annealing and extension at 57° C. for 60 seconds. Following amplification, digital raw data were processed by the BioMark Digital PCR Analysis software using a manually selected threshold to remove background signals and target a Ct range of 20 to 35. For SNP fetal fraction analysis, the final reaction mix for each digital panel comprised 1×TaqMan Genotyping Master Mix (Applied Biosystems), 1×sample loading reagent (Fluidigm), relevant SNP forward and reverse primers and probes, and cell-free DNA (100-300 pg). Microfluidic digital PCR (Fluidigm) was used to quantify the amount of total (fetal and maternal) and fetal DNA using TaqMan assays targeting at the EIF2C1 locus on chromosome 1 and the SRY locus on chromosome Y (Fan H C et al. PNAS 105:16266-16271 [2008]), respectively. Probes and primers used for the EIF2C1 locus and SRY locus are given in Table 4 below.</div>
</li> <li> <div class="description-line" id="p-0223" num="0000">
<tables id="TABLE-US-00004" num="00004">
<patent-tables colsep="0" frame="none" pgwide="1" rowsep="0" tabstyle="monospace">
<table align="left" class="description-table" cols="1" colsep="0" rowsep="0" width="100%">
<thead>
<tr class="description-tr">
<td class="description-td" colspan="1" nameend="1" namest="1" rowsep="1">TABLE 4</td>
</tr>
</thead>
<tbody><tr class="description-tr">
<td align="center" class="description-td" colspan="1" nameend="1" namest="1" rowsep="1"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">Probes and Primers for the EIF2C1 and SRY Loci</td>
</tr>
</tbody></table>
<table align="left" class="description-table" cols="4" colsep="0" rowsep="0" width="100%">
<tbody><tr class="description-tr">
<td class="description-td">Locus</td>
<td class="description-td">Forward Primer</td>
<td class="description-td">Reverse Primer</td>
<td class="description-td">Probe</td>
</tr>
<tr class="description-tr">
<td align="center" class="description-td" colspan="4" nameend="4" namest="1" rowsep="1"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">EIF2C1</td>
<td class="description-td">GTTCGGCTTTCACCAGTCT</td>
<td class="description-td">CTCCATAGCTCTCCCCACT</td>
<td class="description-td">VIC-CGCCCTGCCATGTGGAA-</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">(SEQ ID NO: 312)</td>
<td class="description-td">(SEQ ID NO: 313)</td>
<td class="description-td">MGB (SEQ ID NO: 314)</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">SRY</td>
<td class="description-td">CGCTTAACATAGCAGAAGCA</td>
<td class="description-td">AGTTTCGAACTCTGGCACCT</td>
<td class="description-td">FAM-</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">(SEQ ID NO: 315)</td>
<td class="description-td">(SEQ ID NO: 316)</td>
<td class="description-td">TGTCGCACTCTCCTTGTTTTTGACA</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">MGB (SEQ ID NO: 317)</td>
</tr>
<tr class="description-tr">
<td align="center" class="description-td" colspan="4" nameend="4" namest="1" rowsep="1"> </td>
</tr>
</tbody></table>
</patent-tables>
</tables>
</div>
</li> <li> <para-num num="[0218]"> </para-num> <div class="description-line" id="p-0224" num="0218">Fetal DNA fractions were determined using following formula:</div>
</li> <li> <div class="description-line" id="p-0225" num="0000"> <br/>fetal DNA Fraction=(fetal informative allele counts)/((maternal allele counts+fetal informative counts)/2)</div>
</li> <li> <para-num num="[0219]"> </para-num> <div class="description-line" id="p-0226" num="0219">The results shown in Table 5 below, demonstrate that fetal fraction can be determined in maternal samples of subjects carrying male or female fetuses by identifying and quantifying SNP alleles using PCR-based methods.</div>
</li> <li> <div class="description-line" id="p-0227" num="0000">
<tables id="TABLE-US-00005" num="00005">
<patent-tables colsep="0" frame="none" rowsep="0">
<table align="left" class="description-table" cols="1" colsep="0" rowsep="0" width="100%">
<thead>
<tr class="description-tr">
<td class="description-td" colspan="1" nameend="1" namest="1" rowsep="1">TABLE 5</td>
</tr>
</thead>
<tbody><tr class="description-tr">
<td align="center" class="description-td" colspan="1" nameend="1" namest="1" rowsep="1"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">Fetal Fraction of male and female fetuses determined</td>
</tr>
<tr class="description-tr">
<td class="description-td">by identification and quantification of SNPs</td>
</tr>
</tbody></table>
<table align="left" class="description-table" cols="2" colsep="0" rowsep="0" width="100%">
<tbody><tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">Fetal Fraction</td>
</tr>
</tbody></table>
<table align="left" class="description-table" cols="5" colsep="0" rowsep="0" width="100%">
<tbody><tr class="description-tr">
<td class="description-td">Sample ID</td>
<td class="description-td">Fetal sex</td>
<td class="description-td">Gestation (weeks)</td>
<td class="description-td">SNP %</td>
<td class="description-td">SRY %</td>
</tr>
<tr class="description-tr">
<td align="center" class="description-td" colspan="5" nameend="5" namest="1" rowsep="1"> </td>
</tr>
</tbody></table>
<table align="left" class="description-table" cols="5" colsep="0" rowsep="0" width="100%">
<tbody><tr class="description-tr">
<td class="description-td">24181</td>
<td class="description-td">Female</td>
<td class="description-td">11</td>
<td class="description-td">6.48</td>
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">51212</td>
<td class="description-td">Male</td>
<td class="description-td">18</td>
<td class="description-td">12.92</td>
<td class="description-td">7.43</td>
</tr>
<tr class="description-tr">
<td class="description-td">ABRSSJA5516</td>
<td class="description-td">Male</td>
<td class="description-td">12</td>
<td class="description-td">6.61</td>
<td class="description-td">5.87</td>
</tr>
<tr class="description-tr">
<td align="center" class="description-td" colspan="5" nameend="5" namest="1" rowsep="1"> </td>
</tr>
</tbody></table>
</patent-tables>
</tables>
</div>
</li> <heading id="h-0014">Example 3</heading>
<heading id="h-0015">Specific Enrichment of Nucleic Acids and Sensitivity of SNP Detection by Real-Time PCR</heading>
<li> <para-num num="[0220]"> </para-num> <div class="description-line" id="p-0228" num="0220">To increase the sensitivity of detection of SNPs in a maternal sample, cfDNA was pre-amplified prior to performing real-time PCR and/or dPCR as described above.</div>
</li> <li> <para-num num="[0221]"> </para-num> <div class="description-line" id="p-0229" num="0221">Template cell free DNA (250-850 pg) was amplified by PCR for 14 cycles as follows. 250-850 pg template cfDNA contained in 12.5 μl was combined with 12.5 μl of a 0.2× Pool SNP 1-20 assay mix containing the forward and reverse primers, and the corresponding probes for identifying the SNPs in Table 2, and 25 μl of 2× TaqMan PreAmp Master Mix. The cfDNA was denatured at 95° C. for 10 minutes, and amplified for 14 cycles of denaturation at 95° C. for 15 seconds and combined annealing and extension at 60° C. for 4 minutes.</div>
</li> <li> <para-num num="[0222]"> </para-num> <div class="description-line" id="p-0230" num="0222">The effect of preamplifying cfDNA prior to detecting informative SNPs was first tested in artificial samples of cfDNA obtained from non-pregnant subjects were spiked to contain 3%, 5% and 10% amplified products, and analyzed using ABI Real-Time TaqMan PCR. The amplified nucleic acids were diluted, and the identity of alleles at polymorphic loci was determined using ABI Real-Time TaqMan PCR. The results shown in Table 6 below indicate that preamplification of nucleic acid sequences comprising the SNPs increases the number of SNPs that can be identified in the cfDNA sample.</div>
</li> <li> <div class="description-line" id="p-0231" num="0000">
<tables id="TABLE-US-00006" num="00006">
<patent-tables colsep="0" frame="none" rowsep="0">
<table align="left" class="description-table" cols="1" colsep="0" rowsep="0" width="100%">
<thead>
<tr class="description-tr">
<td class="description-td" colspan="1" nameend="1" namest="1" rowsep="1">TABLE 6</td>
</tr>
</thead>
<tbody><tr class="description-tr">
<td align="center" class="description-td" colspan="1" nameend="1" namest="1" rowsep="1"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">Identification of SNPs in Artificial cfDNA Samples</td>
</tr>
</tbody></table>
<table align="left" class="description-table" cols="3" colsep="0" rowsep="0" width="100%">
<tbody><tr class="description-tr">
<td class="description-td">Sample ID</td>
<td class="description-td">Without Preamplification</td>
<td class="description-td">With Preamplification</td>
</tr>
<tr class="description-tr">
<td align="center" class="description-td" colspan="3" nameend="3" namest="1" rowsep="1"> </td>
</tr>
</tbody></table>
<table align="left" class="description-table" cols="7" colsep="0" rowsep="0" width="100%">
<tbody><tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">SNP6</td>
<td class="description-td">SNP10</td>
<td class="description-td"> </td>
<td class="description-td">SNP6</td>
<td class="description-td">SNP10</td>
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td align="center" class="description-td" colspan="7" nameend="7" namest="1" rowsep="1"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">SJA01</td>
<td class="description-td">G/G</td>
<td class="description-td">A/A</td>
</tr>
<tr class="description-tr">
<td class="description-td">ART5</td>
<td class="description-td">A/G</td>
<td class="description-td">A/G</td>
</tr>
<tr class="description-tr">
<td class="description-td">ART5 3%</td>
<td class="description-td">
<u>
<b>A/G</b>
</u>
</td>
<td class="description-td">
<u>
<b>A/G</b>
</u>
</td>
<td class="description-td"> </td>
<td class="description-td">
<u>
<b>A/G</b>
</u>
</td>
<td class="description-td">
<u>
<b>A/G</b>
</u>
</td>
</tr>
<tr class="description-tr">
<td class="description-td">ART5 5%</td>
<td class="description-td">
<u>
<b>A/G</b>
</u>
</td>
<td class="description-td">
<u>
<b>A/G</b>
</u>
</td>
<td class="description-td"> </td>
<td class="description-td">
<u>
<b>A/G</b>
</u>
</td>
<td class="description-td">
<u>
<b>A/G</b>
</u>
</td>
</tr>
<tr class="description-tr">
<td class="description-td">ART5 10%</td>
<td class="description-td">
<u>
<b>A/G</b>
</u>
</td>
<td class="description-td">
<u>
<b>A/G</b>
</u>
</td>
<td class="description-td"> </td>
<td class="description-td">
<u>
<b>A/G</b>
</u>
</td>
<td class="description-td">
<u>
<b>A/G</b>
</u>
</td>
</tr>
<tr class="description-tr">
<td align="center" class="description-td" colspan="7" nameend="7" namest="1" rowsep="1"> </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">SNP8</td>
<td class="description-td">SNP10</td>
<td class="description-td"> </td>
<td class="description-td">SNP8</td>
<td class="description-td">SNP10</td>
</tr>
<tr class="description-tr">
<td align="center" class="description-td" colspan="7" nameend="7" namest="1" rowsep="1"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">ART7</td>
<td class="description-td">C/C</td>
<td class="description-td">G/G</td>
</tr>
<tr class="description-tr">
<td class="description-td">ART5</td>
<td class="description-td">C/T</td>
<td class="description-td">A/G</td>
</tr>
<tr class="description-tr">
<td class="description-td">ART5 3%</td>
<td class="description-td">C/C</td>
<td class="description-td">G/G</td>
<td class="description-td"> </td>
<td class="description-td">
<u>
<b>C/T</b>
</u>
</td>
<td class="description-td">
<u>
<b>A/G</b>
</u>
</td>
</tr>
<tr class="description-tr">
<td class="description-td">ART5 5%</td>
<td class="description-td">
<u>
<b>C/T</b>
</u>
</td>
<td class="description-td">G/G</td>
<td class="description-td"> </td>
<td class="description-td">
<u>
<b>C/T</b>
</u>
</td>
<td class="description-td">
<u>
<b>A/G</b>
</u>
</td>
</tr>
<tr class="description-tr">
<td class="description-td">ART5 10%</td>
<td class="description-td">C/C</td>
<td class="description-td">G/G</td>
<td class="description-td"> </td>
<td class="description-td">
<u>
<b>C/T</b>
</u>
</td>
<td class="description-td">
<u>
<b>A/G</b>
</u>
</td>
</tr>
<tr class="description-tr">
<td align="center" class="description-td" colspan="7" nameend="7" namest="1" rowsep="1"> </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">SNP1</td>
<td class="description-td">SNP5</td>
<td class="description-td">SNP9</td>
<td class="description-td">SNP1</td>
<td class="description-td">SNP5</td>
<td class="description-td">SNP9</td>
</tr>
<tr class="description-tr">
<td align="center" class="description-td" colspan="7" nameend="7" namest="1" rowsep="1"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">ART5</td>
<td class="description-td">A/A</td>
<td class="description-td">A/A</td>
<td class="description-td">C/C</td>
</tr>
<tr class="description-tr">
<td class="description-td">ART6</td>
<td class="description-td">A/C</td>
<td class="description-td">A/G</td>
<td class="description-td">A/C</td>
</tr>
<tr class="description-tr">
<td class="description-td">ART6 3%</td>
<td class="description-td">A/A</td>
<td class="description-td">A/A</td>
<td class="description-td">
<u>
<b>A/C</b>
</u>
</td>
<td class="description-td">
<u>
<b>A/C</b>
</u>
</td>
<td class="description-td">
<u>
<b>A/G</b>
</u>
</td>
<td class="description-td">
<u>
<b>A/C</b>
</u>
</td>
</tr>
<tr class="description-tr">
<td class="description-td">ART6 5%</td>
<td class="description-td">A/A</td>
<td class="description-td">
<u>
<b>A/G</b>
</u>
</td>
<td class="description-td">
<u>
<b>A/C</b>
</u>
</td>
<td class="description-td">
<u>
<b>A/C</b>
</u>
</td>
<td class="description-td">
<u>
<b>A/G</b>
</u>
</td>
<td class="description-td">
<u>
<b>A/C</b>
</u>
</td>
</tr>
<tr class="description-tr">
<td class="description-td">ART6 10%</td>
<td class="description-td">
<u>
<b>A/C</b>
</u>
</td>
<td class="description-td">
<u>
<b>A/G</b>
</u>
</td>
<td class="description-td">
<u>
<b>A/C</b>
</u>
</td>
<td class="description-td">
<u>
<b>A/C</b>
</u>
</td>
<td class="description-td">
<u>
<b>A/G</b>
</u>
</td>
<td class="description-td">
<u>
<b>A/C</b>
</u>
</td>
</tr>
<tr class="description-tr">
<td align="center" class="description-td" colspan="7" nameend="7" namest="1" rowsep="1"> </td>
</tr>
</tbody></table>
</patent-tables>
</tables>
</div>
</li> <li> <para-num num="[0223]"> </para-num> <div class="description-line" id="p-0232" num="0223">The effect of preamplifying cfDNA prior to detecting informative SNPs chosen from SNPs 1-10 (Table 2 above) was tested in 9 cfDNA samples obtained from pregnant subjects. Four of the 9 samples were further tested for SNPs 11-20 (Table 8).</div>
</li> <li> <para-num num="[0224]"> </para-num> <div class="description-line" id="p-0233" num="0224">The results shown in Table 7 below show that preamplification i.e. enrichment, of cfDNA for polymorphic target sequences enhances the sensitivity of detection of SNPs in maternal cfDNA samples. Fetal informative SNPs were detected in 9 (100%) of the preamplified samples from pregnant subjects, while fetal informative SNPs were detected in only 3 (30%) of the 9 sample in the absence of preamplification.</div>
</li> <li> <div class="description-line" id="p-0234" num="0000">
<tables id="TABLE-US-00007" num="00007">
<patent-tables colsep="0" frame="none" pgwide="1" rowsep="0">
<table align="left" class="description-table" cols="1" colsep="0" rowsep="0" width="100%">
<thead>
<tr class="description-tr">
<td class="description-td" colspan="1" nameend="1" namest="1" rowsep="1">TABLE 7</td>
</tr>
</thead>
<tbody><tr class="description-tr">
<td align="center" class="description-td" colspan="1" nameend="1" namest="1" rowsep="1"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">Identification of SNPs in Maternal cfDNA samples:</td>
</tr>
<tr class="description-tr">
<td class="description-td">Effect of Preamplification of SNP Sequences</td>
</tr>
</tbody></table>
<table align="left" class="description-table" cols="3" colsep="0" rowsep="0" width="100%">
<tbody><tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">Without Preamplification</td>
<td class="description-td">With Preamplification</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td align="center" class="description-td" colspan="3" nameend="2" namest="offset" rowsep="1"> </td>
</tr>
</tbody></table>
<table align="left" class="description-table" cols="9" colsep="0" rowsep="0" width="100%">
<tbody><tr class="description-tr">
<td class="description-td">Sample ID</td>
<td class="description-td">SNP 1</td>
<td class="description-td">SNP 5</td>
<td class="description-td">SNP 7</td>
<td class="description-td">SNP 10</td>
<td class="description-td">SNP 1</td>
<td class="description-td">SNP 5</td>
<td class="description-td">SNP 7</td>
<td class="description-td">SNP 10</td>
</tr>
<tr class="description-tr">
<td align="center" class="description-td" colspan="9" nameend="9" namest="1" rowsep="1"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">ABRSAC2394 gDNA</td>
<td class="description-td">A/A</td>
<td class="description-td">A/A</td>
<td class="description-td">T/T</td>
<td class="description-td">G/G</td>
<td class="description-td">A/A</td>
<td class="description-td">A/A</td>
<td class="description-td">T/T</td>
<td class="description-td">G/G</td>
</tr>
<tr class="description-tr">
<td class="description-td">ABRSAC2394 cfDNA</td>
<td class="description-td">A/A</td>
<td class="description-td">A/A</td>
<td class="description-td">T/T</td>
<td class="description-td">G/G</td>
<td class="description-td">A/A</td>
<td class="description-td">
<u>
<b>A/G</b>
</u>
</td>
<td class="description-td">T/T</td>
<td class="description-td">
<u>
<b>A/G</b>
</u>
</td>
</tr>
<tr class="description-tr">
<td align="center" class="description-td" colspan="9" nameend="9" namest="1" rowsep="1"> </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">SNP 3</td>
<td class="description-td">SNP 4</td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">SNP 3</td>
<td class="description-td">SNP 4</td>
</tr>
<tr class="description-tr">
<td align="center" class="description-td" colspan="9" nameend="9" namest="1" rowsep="1"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">ABRCMD3437 gDNA</td>
<td class="description-td">C/C</td>
<td class="description-td">A/A</td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">C/C</td>
<td class="description-td">A/A</td>
</tr>
<tr class="description-tr">
<td class="description-td">ABRCMD3437 cfDNA</td>
<td class="description-td">C/C</td>
<td class="description-td">A/A</td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">C/C</td>
<td class="description-td">
<u>
<b>A/T</b>
</u>
</td>
</tr>
<tr class="description-tr">
<td align="center" class="description-td" colspan="9" nameend="9" namest="1" rowsep="1"> </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">SNP 2</td>
<td class="description-td">SNP 4</td>
<td class="description-td">SNP 5</td>
<td class="description-td">SNP 9</td>
<td class="description-td">SNP 2</td>
<td class="description-td">SNP 4</td>
<td class="description-td">SNP 5</td>
<td class="description-td">SNP 9</td>
</tr>
<tr class="description-tr">
<td align="center" class="description-td" colspan="9" nameend="9" namest="1" rowsep="1"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">ABRFRA4675 gDNA</td>
<td class="description-td">G/G</td>
<td class="description-td">T/T</td>
<td class="description-td">A/A</td>
<td class="description-td">A/A</td>
<td class="description-td">G/G</td>
<td class="description-td">T/T</td>
<td class="description-td">A/A</td>
<td class="description-td">A/A</td>
</tr>
<tr class="description-tr">
<td class="description-td">ABRFRA4675 cfDNA</td>
<td class="description-td">G/G</td>
<td class="description-td">T/T</td>
<td class="description-td">A/A</td>
<td class="description-td">A/A</td>
<td class="description-td">G/G</td>
<td class="description-td">T/T</td>
<td class="description-td">
<u>
<b>A/G</b>
</u>
</td>
<td class="description-td">
<u>
<b>A/C</b>
</u>
</td>
</tr>
<tr class="description-tr">
<td align="center" class="description-td" colspan="9" nameend="9" namest="1" rowsep="1"> </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">SNP 2</td>
<td class="description-td">SNP 4</td>
<td class="description-td">SNP 7</td>
<td class="description-td"> </td>
<td class="description-td">SNP 2</td>
<td class="description-td">SNP 4</td>
<td class="description-td">SNP 7</td>
</tr>
<tr class="description-tr">
<td align="center" class="description-td" colspan="9" nameend="9" namest="1" rowsep="1"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">ABRSAC9818 gDNA</td>
<td class="description-td">G/G</td>
<td class="description-td">T/T</td>
<td class="description-td">T/T</td>
<td class="description-td"> </td>
<td class="description-td">G/G</td>
<td class="description-td">T/T</td>
<td class="description-td">T/T</td>
</tr>
<tr class="description-tr">
<td class="description-td">ABRSAC9818 cfDNA</td>
<td class="description-td">G/G</td>
<td class="description-td">T/T</td>
<td class="description-td">T/T</td>
<td class="description-td"> </td>
<td class="description-td">G/G</td>
<td class="description-td">
<u>
<b>A/T</b>
</u>
</td>
<td class="description-td">T/T</td>
</tr>
<tr class="description-tr">
<td align="center" class="description-td" colspan="9" nameend="9" namest="1" rowsep="1"> </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">SNP 2</td>
<td class="description-td">SNP 4</td>
<td class="description-td">SNP 5</td>
<td class="description-td"> </td>
<td class="description-td">SNP 2</td>
<td class="description-td">SNP 4</td>
<td class="description-td">SNP 5</td>
</tr>
<tr class="description-tr">
<td align="center" class="description-td" colspan="9" nameend="9" namest="1" rowsep="1"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">ABRSAC9830 gDNA</td>
<td class="description-td">G/G</td>
<td class="description-td">A/A</td>
<td class="description-td">A/A</td>
<td class="description-td"> </td>
<td class="description-td">G/G</td>
<td class="description-td">A/A</td>
<td class="description-td">A/A</td>
</tr>
<tr class="description-tr">
<td class="description-td">ABRSAC9830 gDNA</td>
<td class="description-td">G/G</td>
<td class="description-td">A/A</td>
<td class="description-td">A/A</td>
<td class="description-td"> </td>
<td class="description-td">G/G</td>
<td class="description-td">
<u>
<b>A/T</b>
</u>
</td>
<td class="description-td">
<u>
<b>A/G</b>
</u>
</td>
</tr>
<tr class="description-tr">
<td align="center" class="description-td" colspan="9" nameend="9" namest="1" rowsep="1"> </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">SNP 2</td>
<td class="description-td">SNP 3</td>
<td class="description-td">SNP 7</td>
<td class="description-td">SNP 8</td>
<td class="description-td">SNP 2</td>
<td class="description-td">SNP 3</td>
<td class="description-td">SNP 7</td>
<td class="description-td">SNP 8</td>
</tr>
<tr class="description-tr">
<td align="center" class="description-td" colspan="9" nameend="9" namest="1" rowsep="1"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">ABRCMD3362 gDNA</td>
<td class="description-td">G/G</td>
<td class="description-td">T/T</td>
<td class="description-td">C/C</td>
<td class="description-td">T/T</td>
<td class="description-td">G/G</td>
<td class="description-td">T/T</td>
<td class="description-td">C/C</td>
<td class="description-td">T/T</td>
</tr>
<tr class="description-tr">
<td class="description-td">ABRCMD3362 cfDNA</td>
<td class="description-td">G/G</td>
<td class="description-td">T/T</td>
<td class="description-td">C/C</td>
<td class="description-td">
<u>
<b>C/T</b>
</u>
</td>
<td class="description-td">
<u>
<b>A/G</b>
</u>
</td>
<td class="description-td">
<u>
<b>C/T</b>
</u>
</td>
<td class="description-td">C/C</td>
<td class="description-td">
<u>
<b>C/T</b>
</u>
</td>
</tr>
<tr class="description-tr">
<td align="center" class="description-td" colspan="9" nameend="9" namest="1" rowsep="1"> </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">SNP 1</td>
<td class="description-td">SNP 2</td>
<td class="description-td">SNP 3</td>
<td class="description-td">SNP 4</td>
<td class="description-td">SNP 1</td>
<td class="description-td">SNP 2</td>
<td class="description-td">SNP 3</td>
<td class="description-td">SNP 4</td>
</tr>
<tr class="description-tr">
<td align="center" class="description-td" colspan="9" nameend="9" namest="1" rowsep="1"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">ABRSJA5500 gDNA</td>
<td class="description-td">C/C</td>
<td class="description-td">G/G</td>
<td class="description-td">C/C</td>
<td class="description-td">A/A</td>
<td class="description-td">C/C</td>
<td class="description-td">G/G</td>
<td class="description-td">C/C</td>
<td class="description-td">A/A</td>
</tr>
<tr class="description-tr">
<td class="description-td">ABRSJA5500 cfDNA</td>
<td class="description-td">
<u>
<b>A/C</b>
</u>
</td>
<td class="description-td">G/G</td>
<td class="description-td">C/C</td>
<td class="description-td">A/A</td>
<td class="description-td">
<u>
<b>A/C</b>
</u>
</td>
<td class="description-td">
<u>
<b>A/G</b>
</u>
</td>
<td class="description-td">C/C</td>
<td class="description-td">
<u>
<b>A/T</b>
</u>
</td>
</tr>
<tr class="description-tr">
<td align="center" class="description-td" colspan="9" nameend="9" namest="1" rowsep="1"> </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">SNP 1</td>
<td class="description-td">SNP 4</td>
<td class="description-td">SNP 8</td>
<td class="description-td"> </td>
<td class="description-td">SNP 1</td>
<td class="description-td">SNP 4</td>
<td class="description-td">SNP 8</td>
</tr>
<tr class="description-tr">
<td align="center" class="description-td" colspan="9" nameend="9" namest="1" rowsep="1"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">ABRSJA5516 gDNA</td>
<td class="description-td">C/C</td>
<td class="description-td">T/T</td>
<td class="description-td">C/C</td>
<td class="description-td"> </td>
<td class="description-td">C/C</td>
<td class="description-td">T/T</td>
<td class="description-td">C/C</td>
</tr>
<tr class="description-tr">
<td class="description-td">ABRSJA5516 cfDNA</td>
<td class="description-td">C/C</td>
<td class="description-td">T/T</td>
<td class="description-td">
<u>
<b>C/T</b>
</u>
</td>
<td class="description-td"> </td>
<td class="description-td">
<u>
<b>A/C</b>
</u>
</td>
<td class="description-td">
<u>
<b>A/T</b>
</u>
</td>
<td class="description-td">
<u>
<b>C/T</b>
</u>
</td>
</tr>
<tr class="description-tr">
<td align="center" class="description-td" colspan="9" nameend="9" namest="1" rowsep="1"> </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">SNP 2</td>
<td class="description-td">SNP 5</td>
<td class="description-td">SNP 8</td>
<td class="description-td"> </td>
<td class="description-td">SNP 2</td>
<td class="description-td">SNP 5</td>
<td class="description-td">SNP 8</td>
</tr>
<tr class="description-tr">
<td align="center" class="description-td" colspan="9" nameend="9" namest="1" rowsep="1"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">ABRSJA5517 gDNA</td>
<td class="description-td">G/G</td>
<td class="description-td">A/A</td>
<td class="description-td">C/C</td>
<td class="description-td"> </td>
<td class="description-td">G/G</td>
<td class="description-td">A/A</td>
<td class="description-td">C/C</td>
</tr>
<tr class="description-tr">
<td class="description-td">ABRSJA5517 cfDNA</td>
<td class="description-td">G/G</td>
<td class="description-td">
<u>
<b>A/G</b>
</u>
</td>
<td class="description-td">C/C</td>
<td class="description-td"> </td>
<td class="description-td">
<u>
<b>A/G</b>
</u>
</td>
<td class="description-td">
<u>
<b>A/G</b>
</u>
</td>
<td class="description-td">
<u>
<b>C/T</b>
</u>
</td>
</tr>
<tr class="description-tr">
<td align="center" class="description-td" colspan="9" nameend="9" namest="1" rowsep="1"> </td>
</tr>
</tbody></table>
</patent-tables>
</tables>
</div>
</li> <li> <div class="description-line" id="p-0235" num="0000">
<tables id="TABLE-US-00008" num="00008">
<patent-tables colsep="0" frame="none" rowsep="0">
<table align="left" class="description-table" cols="1" colsep="0" rowsep="0" width="100%">
<thead>
<tr class="description-tr">
<td class="description-td" colspan="1" nameend="1" namest="1" rowsep="1">TABLE 8</td>
</tr>
<tr class="description-tr">
<td align="center" class="description-td" colspan="1" nameend="1" namest="1" rowsep="1"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">Detection of Additional SNPs Following Preamplification</td>
</tr>
<tr class="description-tr">
<td align="center" class="description-td" colspan="1" nameend="1" namest="1" rowsep="1"> </td>
</tr>
</thead>
<tbody><tr class="description-tr">
<td class="description-td"> </td>
</tr>
</tbody></table>
<table align="left" class="description-table" cols="6" colsep="0" rowsep="0" width="100%">
<tbody><tr class="description-tr">
<td class="description-td">Sample ID</td>
<td class="description-td">SNP13</td>
<td class="description-td">SNP14</td>
<td class="description-td">SNP16</td>
<td class="description-td">SNP19</td>
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td align="center" class="description-td" colspan="6" nameend="6" namest="1" rowsep="1"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">ABRSAC2394 gDNA</td>
<td class="description-td">G/G</td>
<td class="description-td">A/A</td>
<td class="description-td">A/A</td>
<td class="description-td">T/T</td>
</tr>
<tr class="description-tr">
<td class="description-td">ABRSAC2394 cfDNA</td>
<td class="description-td">G/G</td>
<td class="description-td">A/G</td>
<td class="description-td">A/T</td>
<td class="description-td">
<u>
<b>C/T</b>
</u>
</td>
</tr>
<tr class="description-tr">
<td align="center" class="description-td" colspan="6" nameend="6" namest="1" rowsep="1"> </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">SNP13</td>
<td class="description-td">SNP14</td>
<td class="description-td">SNP15</td>
<td class="description-td">SNP18</td>
<td class="description-td">SNP19</td>
</tr>
<tr class="description-tr">
<td align="center" class="description-td" colspan="6" nameend="6" namest="1" rowsep="1"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">ABRCM3400 gDNA</td>
<td class="description-td">A/A</td>
<td class="description-td">A/A</td>
<td class="description-td">G/G</td>
<td class="description-td">T/T</td>
<td class="description-td">T/T</td>
</tr>
<tr class="description-tr">
<td class="description-td">ABRCM3400 cfDNA</td>
<td class="description-td">
<u>
<b>A/G</b>
</u>
</td>
<td class="description-td">
<u>
<b>A/G</b>
</u>
</td>
<td class="description-td">
<u>
<b>A/G</b>
</u>
</td>
<td class="description-td">T/T</td>
<td class="description-td">
<u>
<b>C/T</b>
</u>
</td>
</tr>
<tr class="description-tr">
<td align="center" class="description-td" colspan="6" nameend="6" namest="1" rowsep="1"> </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">SNP18</td>
<td class="description-td">SNP19</td>
</tr>
<tr class="description-tr">
<td align="center" class="description-td" colspan="6" nameend="6" namest="1" rowsep="1"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">ABRCMD3437 gDNA</td>
<td class="description-td">T/T</td>
<td class="description-td">C/C</td>
</tr>
<tr class="description-tr">
<td class="description-td">ABRCMD3437 cfDNA</td>
<td class="description-td">
<u>
<b>A/T</b>
</u>
</td>
<td class="description-td">
<u>
<b>C/T</b>
</u>
</td>
</tr>
<tr class="description-tr">
<td align="center" class="description-td" colspan="6" nameend="6" namest="1" rowsep="1"> </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">SNP15</td>
<td class="description-td">SNP19</td>
<td class="description-td">SNP20</td>
</tr>
<tr class="description-tr">
<td align="center" class="description-td" colspan="6" nameend="6" namest="1" rowsep="1"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">ABRFRA4675 gDNA</td>
<td class="description-td">G/G</td>
<td class="description-td">C/C</td>
<td class="description-td">G/G</td>
</tr>
<tr class="description-tr">
<td class="description-td">ABRFRA4675 cfDNA</td>
<td class="description-td">
<u>
<b>A/G</b>
</u>
</td>
<td class="description-td">
<b>C/T</b>
</td>
<td class="description-td">
<u>
<b>A/G</b>
</u>
</td>
</tr>
<tr class="description-tr">
<td align="center" class="description-td" colspan="6" nameend="6" namest="1" rowsep="1"> </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">SNP14</td>
<td class="description-td">SNP15</td>
<td class="description-td">SNP17</td>
<td class="description-td">SNP19</td>
</tr>
<tr class="description-tr">
<td align="center" class="description-td" colspan="6" nameend="6" namest="1" rowsep="1"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">ABRSAC9818 gDNA</td>
<td class="description-td">A/A</td>
<td class="description-td">G/G</td>
<td class="description-td">G/G</td>
<td class="description-td">C/C</td>
</tr>
<tr class="description-tr">
<td class="description-td">ABRSAC9818 cfDNA</td>
<td class="description-td">
<u>
<b>A/G</b>
</u>
</td>
<td class="description-td">
<u>
<b>A/G</b>
</u>
</td>
<td class="description-td">G/G</td>
<td class="description-td">
<u>
<b>C/T</b>
</u>
</td>
</tr>
<tr class="description-tr">
<td align="center" class="description-td" colspan="6" nameend="6" namest="1" rowsep="1"> </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">SNP14</td>
<td class="description-td">SNP15</td>
</tr>
<tr class="description-tr">
<td align="center" class="description-td" colspan="6" nameend="6" namest="1" rowsep="1"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">ABRSAC9825 gDNA</td>
<td class="description-td">A/A</td>
<td class="description-td">A/A</td>
</tr>
<tr class="description-tr">
<td class="description-td">ABRSAC9825 gDNA</td>
<td class="description-td">
<u>
<b>A/T</b>
</u>
</td>
<td class="description-td">A/A</td>
</tr>
<tr class="description-tr">
<td align="center" class="description-td" colspan="6" nameend="6" namest="1" rowsep="1"> </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">SNP12</td>
<td class="description-td">SNP17</td>
<td class="description-td">SNP18</td>
<td class="description-td">SNP19</td>
</tr>
<tr class="description-tr">
<td align="center" class="description-td" colspan="6" nameend="6" namest="1" rowsep="1"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">ABRSAC9829 gDNA</td>
<td class="description-td">A/A</td>
<td class="description-td">C/C</td>
<td class="description-td">A/A</td>
<td class="description-td">C/C</td>
</tr>
<tr class="description-tr">
<td class="description-td">ABRSAC9829 cfDNA</td>
<td class="description-td">
<u>
<b>A/C</b>
</u>
</td>
<td class="description-td">C/C</td>
<td class="description-td">A/A</td>
<td class="description-td">
<u>
<b>C/T</b>
</u>
</td>
</tr>
<tr class="description-tr">
<td align="center" class="description-td" colspan="6" nameend="6" namest="1" rowsep="1"> </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">SNP13</td>
<td class="description-td">SNP15</td>
<td class="description-td">SNP17</td>
<td class="description-td">SNP18</td>
<td class="description-td">SNP19</td>
</tr>
<tr class="description-tr">
<td align="center" class="description-td" colspan="6" nameend="6" namest="1" rowsep="1"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">ABRSAC9830 gDNA</td>
<td class="description-td">G/G</td>
<td class="description-td">A/A</td>
<td class="description-td">G/G</td>
<td class="description-td">A/A</td>
<td class="description-td">T/T</td>
</tr>
<tr class="description-tr">
<td class="description-td">ABRSAC9830 cfDNA</td>
<td class="description-td">
<u>
<b>A/G</b>
</u>
</td>
<td class="description-td">
<u>
<b>A/G</b>
</u>
</td>
<td class="description-td">G/G</td>
<td class="description-td">A/A</td>
<td class="description-td">
<u>
<b>C/T</b>
</u>
</td>
</tr>
<tr class="description-tr">
<td align="center" class="description-td" colspan="6" nameend="6" namest="1" rowsep="1"> </td>
</tr>
</tbody></table>
</patent-tables>
</tables>
</div>
</li> <heading id="h-0016">Example 4</heading>
<heading id="h-0017">Identification of Polymorphisms by Massively Parallel Sequencing of Samples Comprising a Mixture of Fetal and Maternal cfDNA: Sample Processing, Preparation of Sequencing Libraries, Sequencing, and Analysis of Sequencing Data</heading>
<li> <para-num num="[0225]"> </para-num> <div class="description-line" id="p-0236" num="0225">(a) Sample Processing and cfDNA Extraction</div>
</li> <li> <para-num num="[0226]"> </para-num> <div class="description-line" id="p-0237" num="0226">Peripheral blood samples were collected from pregnant women in their first or second trimester of pregnancy and who were deemed at risk for fetal aneuploidy. Informed consent was obtained from each participant prior to the blood draw. Blood was collected before amniocentesis or chorionic villus sampling. Karyotype analysis was performed using the chorionic villus or amniocentesis samples to confirm fetal karyotype.</div>
</li> <li> <para-num num="[0227]"> </para-num> <div class="description-line" id="p-0238" num="0227">Peripheral blood drawn from each subject was collected in ACD tubes. One tube of blood sample (approximately 6-9 mL/tube) was transferred into one 15-mL low speed centrifuge tube. Blood was centrifuged at 2640 rpm, 4° C. for 10 min using Beckman Allegra 6 R centrifuge and rotor model GA 3.8.</div>
</li> <li> <para-num num="[0228]"> </para-num> <div class="description-line" id="p-0239" num="0228">For cell-free plasma extraction, the upper plasma layer was transferred to a 15-ml high speed centrifuge tube and centrifuged at 16000×g, 4° C. for 10 min using Beckman Coulter Avanti J-E centrifuge, and JA-14 rotor. The two centrifugation steps were performed within 72 h after blood collection. Cell-free plasma comprising cfDNA was stored at −80° C. and thawed only once before amplification of plasma cfDNA or for purification of cfDNA.</div>
</li> <li> <para-num num="[0229]"> </para-num> <div class="description-line" id="p-0240" num="0229">Purified cell-free DNA (cfDNA) was extracted from cell-free plasma using the QIAamp Blood DNA Mini kit (Qiagen) essentially according to the manufacturer's instruction. One milliliter of buffer AL and 100 μl of Protease solution were added to 1 ml of plasma. The mixture was incubated for 15 minutes at 56° C. One milliliter of 100% ethanol was added to the plasma digest. The resulting mixture was transferred to QIAamp mini columns that were assembled with VacValves and VacConnectors provided in the QIAvac 24 Plus column assembly (Qiagen). Vacuum was applied to the samples, and the cfDNA retained on the column filters was washed under vacuum with 750 μl of buffer AW1, followed by a second wash with 750 μl of buffer AW24. The column was centrifuged at 14,000 RPM for 5 minutes to remove any residual buffer from the filter. The cfDNA was eluted with buffer AE by centrifugation at 14,000 RPM, and the concentration determined using Qubit™ Quantitation Platform (Invitrogen).</div>
</li> <li> <para-num num="[0230]"> </para-num> <div class="description-line" id="p-0241" num="0230">(b) Preparation of Sequencing Libraries</div>
</li> <li> <para-num num="[0231]"> </para-num> <div class="description-line" id="p-0242" num="0231">All sequencing libraries i.e. target, primary and enriched libraries, were prepared from approximately 2 ng of purified cfDNA that was extracted from maternal plasma. Library preparation was performed using reagents of the NEBNext™ DNA Sample Prep DNA Reagent Set 1 (Part No. E6000L; New England Biolabs, Ipswich, Mass.) for Illumina® as follows. Because cell-free plasma DNA is fragmented in nature, no further fragmentation by nebulization or sonication was done on the plasma DNA samples. The overhangs of approximately 2 ng purified cfDNA fragments contained in 40 μl were converted into phosphorylated blunt ends according to the NEBNext® End Repair Module by incubating in a 1.5 ml microfuge tube the cfDNA with 5 μl 10× phosphorylation buffer, 2 μl deoxynucleotide solution mix (10 mM each dNTP), 1 μl of a 1:5 dilution of DNA Polymerase I, 1 μl T4 DNA Polymerase and 1 μl T4 Polynucleotide Kinase provided in the NEBNext™ DNA Sample Prep DNA Reagent Set 1 for 15 minutes at 20° C. The enzymes were then heat inactivated by incubating the reaction mixture at 75° C. for 5 minutes. The mixture was cooled to 4° C., and dA tailing of the blunt-ended DNA was accomplished using 10 μl of the dA-tailing master mix containing the Klenow fragment (3′ to 5′ exo minus) (NEBNext™ DNA Sample Prep DNA Reagent Set 1), and incubating for 15 minutes at 37° C. Subsequently, the Klenow fragment was heat inactivated by incubating the reaction mixture at 75° C. for 5 minutes. Following the inactivation of the Klenow fragment, 11 μl of a 1:5 dilution of Illumina Genomic Adaptor Oligo Mix (Part No. 1000521; Illumina Inc., Hayward, Calif.) was used to ligate the Illumina adaptors (Non-Index Y-Adaptors) to the dA-tailed DNA using 4 μl of the T4 DNA ligase provided in the NEBNext™ DNA Sample Prep DNA Reagent Set 1, by incubating the reaction mixture for 15 minutes at 25° C. The mixture was cooled to 4° C., and the adaptor-ligated cfDNA was purified from unligated adaptors, adaptor dimers, and other reagents using magnetic beads provided in the Agencourt AMPure XP PCR purification system (Part No. A63881; Beckman Coulter Genomics, Danvers, Mass.). Eighteen cycles of PCR were performed to selectively enrich adaptor-ligated cfDNA using Phusion® High-Fidelity Master Mix (Finnzymes, Woburn, Mass.) and Illumina's PCR primers complementary to the adaptors (Part No. 1000537 and 1000537). The adaptor-ligated DNA was subjected to PCR (98° C. for 30 seconds; 18 cycles of 98° C. for 10 seconds, 65° C. for 30 seconds, and 72° C. for 30 seconds; final extension at 72° C. for 5 minutes, and hold at 4° C.) using Illumina Genomic PCR Primers (Part Nos. 100537 and 1000538) and the Phusion HF PCR Master Mix provided in the NEBNext™ DNA Sample Prep DNA Reagent Set 1, according to the manufacturer's instructions. The amplified product was purified using the Agencourt AMPure XP PCR purification system (Agencourt Bioscience Corporation, Beverly, Mass.) according to the manufacturer's instructions available at www.beckmangenomics.com/products/AMPureXPProtocol_000387v001.pdf. The purified amplified product was eluted in 40 μl of Qiagen EB Buffer, and the concentration and size distribution of the amplified libraries was analyzed using the Agilent DNA 1000 Kit for the 2100 Bioanalyzer (Agilent technologies Inc., Santa Clara, Calif.).</div>
</li> <li> <para-num num="[0232]"> </para-num> <div class="description-line" id="p-0243" num="0232">(c) Sequencing</div>
</li> <li> <para-num num="[0233]"> </para-num> <div class="description-line" id="p-0244" num="0233">Sequencing of library DNA was performed using the Genome Analyzer II (Illumina Inc., San Diego, Calif., USA) according to standard manufacturer protocols. Copies of the protocol for whole genome sequencing using Illumina/Solexa technology may be found at BioTechniques® Protocol Guide 2007 Published December 2006: p 29, and on the world wide web at biotechniques.com/default.asp?page=protocol&amp;subsection=article_display&amp;id=112378. The DNA library was diluted to InM and denatured. Library DNA (5 pM) was subjected to cluster amplification according to the procedure described in Illumina's Cluster Station User Guide and Cluster Station Operations Guide, available on the world wide web at illumina.com/systems/genome analyzer/cluster_station.ilmn. The amplified DNA was sequenced using Illumina's Genome Analyzer II to obtain single-end reads of 36 bp. Only about 30 bp of random sequence information are needed to identify a sequence as belonging to a specific human chromosome. Longer sequences can uniquely identify more particular targets. In the present case, a large number of 36 bp reads were obtained, covering approximately 10% of the genome.</div>
</li> <li> <para-num num="[0234]"> </para-num> <div class="description-line" id="p-0245" num="0234">(d) Analysis of Sequencing Data for the Determination of Fetal Fraction</div>
</li> <li> <para-num num="[0235]"> </para-num> <div class="description-line" id="p-0246" num="0235">Upon completion of sequencing of the sample, the Illumina “Sequencer Control Software” transferred image and base call files to a Unix server running the Illumina “Genome Analyzer Pipeline” software version 1.51. The 36 bp reads were aligned to an artificial reference genome e.g. a SNP genome, using the BOWTIE program. The artificial reference genome was identified as the grouping of the polymorphic DNA sequences that encompass the alleles comprised in the polymorphic target sequences. For example, the artificial reference genome is a SNP genome comprising SEQ ID NOs: 1-56. Only reads that mapped uniquely to the artificial genome were used for the analysis of fetal fraction. Reads that matched perfectly to the SNP genome were counted as tags and filtered. Of the remaining reads, only reads having one or two mismatches were counted as tags and included in the analysis. Tags mapped to each of the polymorphic alleles were counted, and the fetal fraction was determined as a percent of the ratio of the number of tags mapped to the major allele i.e. maternal allele, and the number of tags mapped to the minor allele i.e. fetal allele.</div>
</li> <heading id="h-0018">Example 5</heading>
<heading id="h-0019">Selection of Autosomal SNPs for Identification by Massively Parallel Sequencing</heading>
<li> <para-num num="[0236]"> </para-num> <div class="description-line" id="p-0247" num="0236">A set of 28 autosomal SNPs were selected from a list of 92 SNPs (Pakstis et al., Hum Genet 127:315-324 [2010]) and from Applied Biosystems by Life Technologies™ (Carlsbad, Calif.) at world wide web address appliedbiosystems.com, and validated for use in multiplexed PCR amplification. Primers were designed to hybridize to a sequence close to the SNPs site on the cfDNA to ensure that it be included in the 36 bp read generated from the massively parallel sequencing on the Illumina Analyzer GII, and to generate amplicons of sufficient length to undergo bridge-amplification during cluster formation. Thus, primers were designed to generate amplicons that were at least 110 bp, which when combined with the universal adaptors (Illumina Inc., San Diego, Calif.) used for cluster amplification, resulted in DNA molecules of at least 200 bp. Primer sequences were identified, and primer sets i.e. forward and reverse primers, were synthesized by Integrated DNA Technologies (San Diego, Calif.), and stored as a 1 μM solution to be used for amplifying polymorphic target sequences as described in Examples 6-9. Table 8 provides the RefSNP (rs) accession ID numbers, the primers used for amplifying the target cfDNA sequence, and the sequences of the amplicons comprising the possible SNP alleles that would be generated using the primers. The SNPs given in Table 1 were used for the simultaneous amplification of 13 target sequences in a multiplexed assay. The panel provided in Table 9 is an exemplary SNP panel. Fewer or more SNPs can be employed to enrich the fetal and maternal DNA for polymorphic target nucleic acids. Additional SNPs that can be used include the SNPs given in Table 10. The SNP alleles are shown in bold and are underlined. Other additional SNPs that can be used to determine fetal fraction according to the present method include rs315791, rs3780962, rs1410059, rs279844, rs38882, rs9951171, rs214955, rs6444724, rs2503107, rs1019029, rs1413212, rs1031825, rs891700, rs1005533, rs2831700, rs354439, rs1979255, rs1454361, rs8037429, and rs1490413, which have been analyzed for determining fetal fraction by TaqMan PCR, and are disclosed in U.S. Provisional applications 61/296,358 and 61/360,837.</div>
</li> <li> <div class="description-line" id="p-0248" num="0000">
<tables id="TABLE-US-00009" num="00009">
<patent-tables colsep="0" frame="none" pgwide="1" rowsep="0" tabstyle="monospace">
<table align="left" class="description-table" cols="1" colsep="0" rowsep="0" width="100%">
<thead>
<tr class="description-tr">
<td class="description-td" colspan="1" nameend="1" namest="1" rowsep="1">TABLE 9</td>
</tr>
</thead>
<tbody><tr class="description-tr">
<td align="center" class="description-td" colspan="1" nameend="1" namest="1" rowsep="1"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">SNP Panel for the Determination of Fetal Fraction</td>
</tr>
</tbody></table>
<table align="left" class="description-table" cols="6" colsep="0" rowsep="0" width="100%">
<tbody><tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">Forward Primer</td>
<td class="description-td">Reverse Primer</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">Sequence,</td>
<td class="description-td">Sequence,</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">Amplicon:</td>
<td class="description-td">Amplicon:</td>
<td class="description-td">name and</td>
<td class="description-td">name and </td>
</tr>
<tr class="description-tr">
<td class="description-td">SNP ID</td>
<td class="description-td">Chr</td>
<td class="description-td">Allele 1</td>
<td class="description-td">Allele 2</td>
<td class="description-td">SEQ ID NO:</td>
<td class="description-td">SEQ ID NO:</td>
</tr>
<tr class="description-tr">
<td align="center" class="description-td" colspan="6" nameend="6" namest="1" rowsep="1"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">rs560681</td>
<td class="description-td"> 1</td>
<td class="description-td">CACATGCACAGCC</td>
<td class="description-td">CACATGCACAGCC</td>
<td class="description-td">CACATGC</td>
<td class="description-td">CCCCAAGGT</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">AGCAACCCTGTCA</td>
<td class="description-td">AGCAACCCTGTCA</td>
<td class="description-td">ACAGCCA</td>
<td class="description-td">CCTGTGACCT</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">GCAGGAGTTCCCA</td>
<td class="description-td">GCAGGAGTTCCCA</td>
<td class="description-td">GCAACCC</td>
<td class="description-td">GAGT</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">CCAGTTTCTTTCTG</td>
<td class="description-td">CCAGTTTCTTTCTG</td>
<td class="description-td">(rs560681_</td>
<td class="description-td">(rs560681_C1_</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">AGAACATCTGTTC</td>
<td class="description-td">AGAACATCTGTTC</td>
<td class="description-td">C1_1_F;</td>
<td class="description-td">1_R; SEQ ID</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">AGGTTTCTCTCCAT</td>
<td class="description-td">AGGTTTCTCTCCAT</td>
<td class="description-td">SEQ ID</td>
<td class="description-td">NO: 58)</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">CTCT<u> <b>A</b> </u>TTTACTCAG</td>
<td class="description-td">CTCT<u> <b>G</b> </u>TTTACTCA</td>
<td class="description-td">NO: 57)</td>
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">GTCACAGGACCTT</td>
<td class="description-td">GGTCACAGGACCT</td>
<td class="description-td"> </td>
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">GGGG (SEQ ID</td>
<td class="description-td">TGGGG (SEQ ID</td>
<td class="description-td"> </td>
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">NO: 1)</td>
<td class="description-td">NO: 2)</td>
<td class="description-td"> </td>
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">rs1109037</td>
<td class="description-td"> 2</td>
<td class="description-td">TGAGGAAGTGAGG</td>
<td class="description-td">TGAGGAAGTGAGG</td>
<td class="description-td">TGAGGAA</td>
<td class="description-td">TGCCAGTGC</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">CTCAGAGGGTAAG</td>
<td class="description-td">CTCAGAGGGTAAG</td>
<td class="description-td">GTGAGGC</td>
<td class="description-td">GAGATGAAA</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">AAACTTTGTCACA</td>
<td class="description-td">AAACTTTGTCACA</td>
<td class="description-td">TCAGAGG</td>
<td class="description-td">GTCTTT</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">GAGCTGGTGGTGA</td>
<td class="description-td">GAGCTGGTGGTGA</td>
<td class="description-td">GT</td>
<td class="description-td">(rs110937_C2_</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">GGGTGGAGATTTT</td>
<td class="description-td">GGGTGGAGATTTT</td>
<td class="description-td">(rs110937_</td>
<td class="description-td">1_R; SEQ ID</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">ACACTCCCTGCCT</td>
<td class="description-td">ACACTCCCTGCCT</td>
<td class="description-td">C2_1_F;</td>
<td class="description-td">NO: 60)</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">CCCACACCAGTTT</td>
<td class="description-td">CCCACACCAGTTT</td>
<td class="description-td">SEQ ID</td>
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">CTCC<u> <b>A</b> </u>GAGTGGAA</td>
<td class="description-td">CTCC<u> <b>G</b> </u>GAGTGGAA</td>
<td class="description-td">NO: 59)</td>
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">AGACTTTCATCTC</td>
<td class="description-td">AGACTTTCATCTC</td>
<td class="description-td"> </td>
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">GCACTGGCA (SEQ</td>
<td class="description-td">GCACTGGCA (SEQ</td>
<td class="description-td"> </td>
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">ID NO: 3)</td>
<td class="description-td">ID NO: 4)</td>
<td class="description-td"> </td>
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">rs9866013</td>
<td class="description-td"> 3</td>
<td class="description-td">GTGCCTTCAGAAC</td>
<td class="description-td">GTGCCTTCAGAAC</td>
<td class="description-td">GTGCCTT</td>
<td class="description-td">TCCCATCCCA</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">CTTTGAGATCTGA</td>
<td class="description-td">CTTTGAGATCTGA</td>
<td class="description-td">CAGAACC</td>
<td class="description-td">CCAGCCACC</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">TTCTATTTTTAAAG</td>
<td class="description-td">TTCTATTTTTAAAG</td>
<td class="description-td">TTTGAGA</td>
<td class="description-td">C</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">CTTCTTAGAAGAG</td>
<td class="description-td">CTTCTTAGAAGAG</td>
<td class="description-td">TCTGAT</td>
<td class="description-td">(rs9866013_C3_</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">AGATTGCAAAGTG</td>
<td class="description-td">AGATTGCAAAGTG</td>
<td class="description-td">(rs9866013_</td>
<td class="description-td">1_R; SEQ ID</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">GGTTGTTTCTCTAG</td>
<td class="description-td">GGTTGTTTCTCTAG</td>
<td class="description-td">C3_1_F;</td>
<td class="description-td">NO: 62)</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">CCAGACAGGGCAG</td>
<td class="description-td">CCAGACAGGGCAG</td>
<td class="description-td">SEQ ID</td>
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">G<u> <b>C</b> </u>AAATAGGGGTG</td>
<td class="description-td">G<u> <b>T</b> </u>AAATAGGGGTG</td>
<td class="description-td">NO: 61)</td>
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">GCTGGTGGGATGG</td>
<td class="description-td">GCTGGTGGGATGG</td>
<td class="description-td"> </td>
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">GA (SEQ ID</td>
<td class="description-td">GA (SEQ ID NO: 6)</td>
<td class="description-td"> </td>
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">NO: 5)</td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">rs13182883</td>
<td class="description-td">5</td>
<td class="description-td">AGGTGTGTCTCTC</td>
<td class="description-td">AGGTGTGTCTCTC</td>
<td class="description-td">AGGTGTG</td>
<td class="description-td">CCTTTGTCCC</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">TTTTGTGAGGGGA</td>
<td class="description-td">TTTTGTGAGGGGA</td>
<td class="description-td">TCTCTCTT</td>
<td class="description-td">ACCTCCCCAC</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">GGGGTCCCTTCTG</td>
<td class="description-td">GGGGTCCCTTCTG</td>
<td class="description-td">TTGTGAG</td>
<td class="description-td">C</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">GCCTAGTAGAGGG</td>
<td class="description-td">GCCTAGTAGAGGG</td>
<td class="description-td">GGG</td>
<td class="description-td">(rs13182883_C5_</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">CCTGGCCTGCAGT</td>
<td class="description-td">CCTGGCCTGCAGT</td>
<td class="description-td">(rs13182883_</td>
<td class="description-td">1_R; SEQ ID</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">GAGCATTCAAATC</td>
<td class="description-td">GAGCATTCAAATC</td>
<td class="description-td">C5_1_F;</td>
<td class="description-td">NO: 64)</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">CTC<u> <b>A</b> </u>AGGAACAGG</td>
<td class="description-td">CTC<u> <b>G</b> </u>AGGAACAGG</td>
<td class="description-td">SEQ ID</td>
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">GTGGGGAGGTGGG</td>
<td class="description-td">GTGGGGAGGTGGG</td>
<td class="description-td">NO: 63)</td>
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">ACAAAGG (SEQ ID</td>
<td class="description-td">ACAAAGG (SEQ ID</td>
<td class="description-td"> </td>
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">NO: 7)</td>
<td class="description-td">NO: 8)</td>
<td class="description-td"> </td>
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">rs13218440</td>
<td class="description-td">6</td>
<td class="description-td">CCTCGCCTACTGT</td>
<td class="description-td">CCTCGCCTACTGT</td>
<td class="description-td">CCTCGCC</td>
<td class="description-td">CCATCCCAGC</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">GCTGTTTCTAACC</td>
<td class="description-td">GCTGTTTCTAACC</td>
<td class="description-td">TACTGTG</td>
<td class="description-td">TGAGTATTCC</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">ATCATGCTTTTCCC</td>
<td class="description-td">ATCATGCTTTTCCC</td>
<td class="description-td">CTGTTTCT</td>
<td class="description-td">AGGAG</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">TGAATCTCTTGAG</td>
<td class="description-td">TGAATCTCTTGAG</td>
<td class="description-td">AACC</td>
<td class="description-td">(rs13218440_C6_</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">TCTTTTTCTGCTGT</td>
<td class="description-td">TCTTTTTCTGCTGT</td>
<td class="description-td">(rs13218440_</td>
<td class="description-td">1_R; SEQ ID</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">GGACTGAAACTTG</td>
<td class="description-td">GGACTGAAACTTG</td>
<td class="description-td">C6_1_F;</td>
<td class="description-td">NO: 66)</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">ATCCTGAGATTCA</td>
<td class="description-td">ATCCTGAGATTCA</td>
<td class="description-td">SEQ ID</td>
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">CCTCTAGTCCCTCT</td>
<td class="description-td">CCTCTAGTCCCTCT</td>
<td class="description-td">NO: 65)</td>
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">G<u> <b>A</b> </u>GCAGCCTCCTG</td>
<td class="description-td">G<u> <b>G</b> </u>GCAGCCTCCTG</td>
<td class="description-td"> </td>
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">GAATACTCAGCTG</td>
<td class="description-td">GAATACTCAGCTG</td>
<td class="description-td"> </td>
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">GGATGG (SEQ ID</td>
<td class="description-td">GGATGG (SEQ ID</td>
<td class="description-td"> </td>
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">NO: 9)</td>
<td class="description-td">NO: 10)</td>
<td class="description-td"> </td>
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">rs7041158</td>
<td class="description-td">9</td>
<td class="description-td">AATTGCAATGGTG</td>
<td class="description-td">AATTGCAATGGTG</td>
<td class="description-td">AATTGCA</td>
<td class="description-td">CCAGTGAGA</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">AGAGGTTGATGGT</td>
<td class="description-td">AGAGGTTGATGGT</td>
<td class="description-td">ATGGTGA</td>
<td class="description-td">AGTGTCTTGG</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">AAAATCAAACGGA</td>
<td class="description-td">AAAATCAAACGGA</td>
<td class="description-td">GAGGTTG</td>
<td class="description-td">GTTGG (SEQ</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">ACTTGTTATTTTGT</td>
<td class="description-td">ACTTGTTATTTTGT</td>
<td class="description-td">ATGGT</td>
<td class="description-td">ID NO: 68)</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">CATTCTGATGGAC</td>
<td class="description-td">CATTCTGATGGAC</td>
<td class="description-td">(SEQ ID</td>
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">TGGAACTGAGGAT</td>
<td class="description-td">TGGAACTGAGGAT</td>
<td class="description-td">NO: 67)</td>
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">TTTCAATTTCCT<u> <b>C</b> </u>T</td>
<td class="description-td">TTTCAATTTCCT<u> <b>T</b> </u>T</td>
<td class="description-td"> </td>
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">CCAACCCAAGACA</td>
<td class="description-td">CCAACCCAAGACA</td>
<td class="description-td"> </td>
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">CTTCTCACTGG</td>
<td class="description-td">CTTCTCACTGG</td>
<td class="description-td"> </td>
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">(SEQ ID NO: 11)</td>
<td class="description-td">(SEQ ID NO: 12)</td>
<td class="description-td"> </td>
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">rs740598</td>
<td class="description-td">10</td>
<td class="description-td">GAAATGCCTTCTC</td>
<td class="description-td">GAAATGCCTTCTC</td>
<td class="description-td">GAAATGC</td>
<td class="description-td">GGTTTGAGC</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">AGGTAATGGAAGG</td>
<td class="description-td">AGGTAATGGAAGG</td>
<td class="description-td">CTTCTCA</td>
<td class="description-td">AGTTCTGAG</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">TTATCCAAATATTT</td>
<td class="description-td">TTATCCAAATATTT</td>
<td class="description-td">GGTAATG</td>
<td class="description-td">AATGTGGCT</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">TTCGTAAGTATTTC</td>
<td class="description-td">TTCGTAAGTATTTC</td>
<td class="description-td">GAAGGT</td>
<td class="description-td">(SEQ ID</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">AAATAGCAATGGC</td>
<td class="description-td">AAATAGCAATGGC</td>
<td class="description-td">(SEQ ID</td>
<td class="description-td">NO: 70)</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">TCGTCTATGGTTA</td>
<td class="description-td">TCGTCTATGGTTA</td>
<td class="description-td">NO: 69)</td>
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">GTCTC<u> <b>A</b> </u>CAGCCAC</td>
<td class="description-td">GTCTC<u> <b>G</b> </u>CAGCCAC</td>
<td class="description-td"> </td>
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">ATTCTCAGAACTG</td>
<td class="description-td">ATTCTCAGAACTG</td>
<td class="description-td"> </td>
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">CTCAAACC (SEQ</td>
<td class="description-td">CTCAAACC (SEQ ID</td>
<td class="description-td"> </td>
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">ID NO: 13)</td>
<td class="description-td">NO: 14)</td>
<td class="description-td"> </td>
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">rs10773760</td>
<td class="description-td">12</td>
<td class="description-td">ACCCAAAACACTG</td>
<td class="description-td">ACCCAAAACACTG</td>
<td class="description-td">ACCCAAA</td>
<td class="description-td">CCCTTATCTG</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">GAGGGGCCTCTTC</td>
<td class="description-td">GAGGGGCCTCTTC</td>
<td class="description-td">ACACTGG</td>
<td class="description-td">CTATGTGGCA</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">TCATTTTCGGTAG</td>
<td class="description-td">TCATTTTCGGTAG</td>
<td class="description-td">AGGGGCC</td>
<td class="description-td">TACTTGG</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">ACTGCAAGTGTTA</td>
<td class="description-td">ACTGCAAGTGTTA</td>
<td class="description-td">T (SEQ ID</td>
<td class="description-td">(SEQ ID</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">GCCGTCGGGACCA</td>
<td class="description-td">GCCGTCGGGACCA</td>
<td class="description-td">NO: 71)</td>
<td class="description-td">NO: 72)</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">GCTTCTGTCTGGA</td>
<td class="description-td">GCTTCTGTCTGGA</td>
<td class="description-td"> </td>
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">AGTTCGTCAAATT</td>
<td class="description-td">AGTTCGTCAAATT</td>
<td class="description-td"> </td>
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">GCAGTTA<u> <b>A</b> </u>GTCCA</td>
<td class="description-td">GCAGTTA<u> <b>G</b> </u>GTCCA</td>
<td class="description-td"> </td>
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">AGTATGCCACATA</td>
<td class="description-td">AGTATGCCACATA</td>
<td class="description-td"> </td>
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">GCAGATAAGGG</td>
<td class="description-td">GCAGATAAGGG</td>
<td class="description-td"> </td>
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">(SEQ ID NO: 15)</td>
<td class="description-td">(SEQ ID NO: 16)</td>
<td class="description-td"> </td>
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">rs4530059</td>
<td class="description-td">14</td>
<td class="description-td">GCACCAGAATTTA</td>
<td class="description-td">GCACCAGAATTTA</td>
<td class="description-td">GCACCAG</td>
<td class="description-td">GCACCTGAC</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">AACAACGCTGACA</td>
<td class="description-td">AACAACGCTGACA</td>
<td class="description-td">AATTTAA</td>
<td class="description-td">AGGCACATC</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">ATAAATATGCAGT</td>
<td class="description-td">ATAAATATGCAGT</td>
<td class="description-td">ACAACGC</td>
<td class="description-td">AGCG (SEQ ID</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">CGATGATGACTTC</td>
<td class="description-td">CGATGATGACTTC</td>
<td class="description-td">TGACAA</td>
<td class="description-td">NO: 74)</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">CCAGAGCTCCAGA</td>
<td class="description-td">CCAGAGCTCCAGA</td>
<td class="description-td">(SEQ ID</td>
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">AGCAACTCCAGCA</td>
<td class="description-td">AGCAACTCCAGCA</td>
<td class="description-td">NO: 73)</td>
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">CAC<u> <b>A</b> </u>GAGAGGCGC</td>
<td class="description-td">CAC<u> <b>G</b> </u>GAGAGGCGC</td>
<td class="description-td"> </td>
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">TGATGTGCCTGTC</td>
<td class="description-td">TGATGTGCCTGTC</td>
<td class="description-td"> </td>
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">AGGTGC (SEQ ID</td>
<td class="description-td">AGGTGC (SEQ ID</td>
<td class="description-td"> </td>
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">NO: 17)</td>
<td class="description-td">NO: 18)</td>
<td class="description-td"> </td>
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">rs7205345</td>
<td class="description-td">16</td>
<td class="description-td">TGACTGTATACCC</td>
<td class="description-td">TGACTGTATACCC</td>
<td class="description-td">TGACTGT</td>
<td class="description-td">GCACTAAGG</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">CAGGTGCACCCTT</td>
<td class="description-td">CAGGTGCACCCTT</td>
<td class="description-td">ATACCCC</td>
<td class="description-td">ATGTGGAAG</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">GGGTCATCTCTAT</td>
<td class="description-td">GGGTCATCTCTAT</td>
<td class="description-td">AGGTGCA</td>
<td class="description-td">TCTAGTGTG</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">CATAGAACTTATC</td>
<td class="description-td">CATAGAACTTATC</td>
<td class="description-td">CCC (SEQ</td>
<td class="description-td">(SEQ ID</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">TCACAGAGTATAA</td>
<td class="description-td">TCACAGAGTATAA</td>
<td class="description-td">ID NO: 75)</td>
<td class="description-td">NO: 76)</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">GAGCTGATTTCTG</td>
<td class="description-td">GAGCTGATTTCTG</td>
<td class="description-td"> </td>
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">TGTCTGCCT<u> <b>C</b> </u>TCA</td>
<td class="description-td">TGTCTGCCT<u> <b>G</b> </u>TCA</td>
<td class="description-td"> </td>
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">CACTAGACTTCCA</td>
<td class="description-td">CACTAGACTTCCA</td>
<td class="description-td"> </td>
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">CATCCTTAGTGC</td>
<td class="description-td">CATCCTTAGTGC</td>
<td class="description-td"> </td>
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">(SEQ ID NO: 19)</td>
<td class="description-td">(SEQ ID NO: 20)</td>
<td class="description-td"> </td>
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">rs8078417</td>
<td class="description-td">17</td>
<td class="description-td">TGTACGTGGTCAC</td>
<td class="description-td">TGTACGTGGTCAC</td>
<td class="description-td">TGTACGT</td>
<td class="description-td">AGTGTGAGA</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">CAGGGGACGCCTG</td>
<td class="description-td">CAGGGGACGCCTG</td>
<td class="description-td">GGTCACC</td>
<td class="description-td">AGAGCCTCA</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">GCGCTGCGAGGGA</td>
<td class="description-td">GCGCTGCGAGGGA</td>
<td class="description-td">AGGGGAC</td>
<td class="description-td">AGGACAGC</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">GGCCCCGAGCCTC</td>
<td class="description-td">GGCCCCGAGCCTC</td>
<td class="description-td">G (SEQ ID</td>
<td class="description-td">(SEQ ID</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">GTGCCCCCGTGAA</td>
<td class="description-td">GTGCCCCCGTGAA</td>
<td class="description-td">NO: 77)</td>
<td class="description-td">NO: 78)</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">GCTTCAGCTCCCC</td>
<td class="description-td">GCTTCAGCTCCCC</td>
<td class="description-td"> </td>
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">TCCC<u> <b>C</b> </u>GGCTGTCC</td>
<td class="description-td">TCCC<u> <b>T</b> </u>GGCTGTCC</td>
<td class="description-td"> </td>
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">TTGAGGCTCTTCTC</td>
<td class="description-td">TTGAGGCTCTTCTC</td>
<td class="description-td"> </td>
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">ACACT (SEQ ID</td>
<td class="description-td">ACACT (SEQ ID</td>
<td class="description-td"> </td>
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">NO: 21)</td>
<td class="description-td">NO: 22)</td>
<td class="description-td"> </td>
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">rs576261</td>
<td class="description-td">19</td>
<td class="description-td">CAGTGGACCCTGC</td>
<td class="description-td">CAGTGGACCCTGC</td>
<td class="description-td">CAGTGGA</td>
<td class="description-td">GTGGCAAAG</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">TGCACCTTTCCTCC</td>
<td class="description-td">TGCACCTTTCCTCC</td>
<td class="description-td">CCCTGCT</td>
<td class="description-td">GAGAGAGTT</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">CCTCCCATCAACC</td>
<td class="description-td">CCTCCCATCAACC</td>
<td class="description-td">GCACCTT</td>
<td class="description-td">GTGAGG (SEQ</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">TCTTTTGTGCCTCC</td>
<td class="description-td">TCTTTTGTGCCTCC</td>
<td class="description-td">(SEQ ID</td>
<td class="description-td">ID NO: 80)</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">CCCTCCGTGTACC</td>
<td class="description-td">CCCTCCGTGTACC</td>
<td class="description-td">NO: 79)</td>
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">ACCTTCTCTGTCAC</td>
<td class="description-td">ACCTTCTCTGTCAC</td>
<td class="description-td"> </td>
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">CA<u> <b>A</b> </u>CCCTGGCCTC</td>
<td class="description-td">CA<u> <b>C</b> </u>CCCTGGCCTC</td>
<td class="description-td"> </td>
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">ACAACTCTCTCCTT</td>
<td class="description-td">ACAACTCTCTCCTT</td>
<td class="description-td"> </td>
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">TGCCAC (SEQ ID</td>
<td class="description-td">TGCCAC (SEQ ID</td>
<td class="description-td"> </td>
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">NO: 23)</td>
<td class="description-td">NO: 24)</td>
<td class="description-td"> </td>
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">rs2567608</td>
<td class="description-td">20</td>
<td class="description-td">CAGTGGCATAGTA</td>
<td class="description-td">CAGTGGCATAGTA</td>
<td class="description-td">CAGTGGC</td>
<td class="description-td">CCTCTCCGAC</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">GTCCAGGGGCTCC</td>
<td class="description-td">GTCCAGGGGCTCC</td>
<td class="description-td">ATAGTAG</td>
<td class="description-td">AACTTCCGCC</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">TCCTCAGCACCTC</td>
<td class="description-td">TCCTCAGCACCTC</td>
<td class="description-td">TCCAGGG</td>
<td class="description-td">G (SEQ ID</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">CAGCACCTTCCAG</td>
<td class="description-td">CAGCACCTTCCAG</td>
<td class="description-td">GCT (SEQ</td>
<td class="description-td">NO: 82)</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">GAGGCAGCAGCGC</td>
<td class="description-td">GAGGCAGCAGCGC</td>
<td class="description-td">ID NO: 81)</td>
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">AGGCAGAGAACCC</td>
<td class="description-td">AGGCAGAGAACCC</td>
<td class="description-td"> </td>
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">GCTGGAAG<u> <b>A</b> </u>ATCG</td>
<td class="description-td">GCTGGAAG<u> <b>G</b> </u>ATCG</td>
<td class="description-td"> </td>
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">GCGGAAGTTGTCG</td>
<td class="description-td">GCGGAAGTTGTCG</td>
<td class="description-td"> </td>
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">GAGAGG (SEQ ID</td>
<td class="description-td">GAGAGG (SEQ ID</td>
<td class="description-td"> </td>
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">NO: 25)</td>
<td class="description-td">NO: 26)</td>
</tr>
<tr class="description-tr">
<td align="center" class="description-td" colspan="6" nameend="6" namest="1" rowsep="1"> </td>
</tr>
</tbody></table>
</patent-tables>
</tables>
</div>
</li> <li> <div class="description-line" id="p-0249" num="0000">
<tables id="TABLE-US-00010" num="00010">
<patent-tables colsep="0" frame="none" pgwide="1" rowsep="0" tabstyle="monospace">
<table align="left" class="description-table" cols="1" colsep="0" rowsep="0" width="100%">
<thead>
<tr class="description-tr">
<td class="description-td" colspan="1" nameend="1" namest="1" rowsep="1">TABLE 10</td>
</tr>
</thead>
<tbody><tr class="description-tr">
<td align="center" class="description-td" colspan="1" nameend="1" namest="1" rowsep="1"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">Additional SNPs for the Determination of Fetal Fraction</td>
</tr>
</tbody></table>
<table align="left" class="description-table" cols="6" colsep="0" rowsep="0" width="100%">
<tbody><tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">Forward Primer</td>
<td class="description-td">Reverse Primer</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">Sequence,</td>
<td class="description-td">Sequence,</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">Amplicon:</td>
<td class="description-td">Amplicon:</td>
<td class="description-td">name and</td>
<td class="description-td">name and </td>
</tr>
<tr class="description-tr">
<td class="description-td">SNP ID</td>
<td class="description-td">Chr</td>
<td class="description-td">Allele 1</td>
<td class="description-td">Allele 2</td>
<td class="description-td">SEQ ID NO:</td>
<td class="description-td">SEQ ID NO:</td>
</tr>
<tr class="description-tr">
<td align="center" class="description-td" colspan="6" nameend="6" namest="1" rowsep="1"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">rs430046</td>
<td class="description-td">16</td>
<td class="description-td">AGGTCTGGGGGC</td>
<td class="description-td">AGGTCTGGGGGCC</td>
<td class="description-td">AGGTCTG</td>
<td class="description-td">TCCTCCCATT</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">CGCTGAATGCCA</td>
<td class="description-td">GCTGAATGCCAAG</td>
<td class="description-td">GGGGCCG</td>
<td class="description-td">AAACCCAGC</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">AGCTGGGAATCTT</td>
<td class="description-td">CTGGGAATCTTAAA</td>
<td class="description-td">CTGAAT</td>
<td class="description-td">ACCT</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">AAATGTTAAGGA</td>
<td class="description-td">TGTTAAGGAACAA</td>
<td class="description-td">(rs430046_</td>
<td class="description-td">(rs430046_C1_</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">ACAAGGTCATAC</td>
<td class="description-td">GGTCATACAATGA</td>
<td class="description-td">C1_1_F;</td>
<td class="description-td">1_R; SEQ ID</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">AATGAATGGTGT</td>
<td class="description-td">ATGGTGTGATGTAA</td>
<td class="description-td">SEQ ID</td>
<td class="description-td">NO: 84)</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">GATGTAAAAGCT</td>
<td class="description-td">AAGCTTGGGAGGT</td>
<td class="description-td">NO: 83)</td>
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">TGGGAGGTGATTT</td>
<td class="description-td">GATTT<u> <b>T</b> </u>TGAGGGTA</td>
<td class="description-td"> </td>
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> <u> <b>C</b> </u>TGAGGGTAGGT</td>
<td class="description-td">GGTGCTGGGTTTAA</td>
<td class="description-td"> </td>
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">GCTGGGTTTAATG</td>
<td class="description-td">TGGGAGGA (SEQ ID</td>
<td class="description-td"> </td>
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">GGAGGA (SEQ ID</td>
<td class="description-td">NO: 28)</td>
<td class="description-td"> </td>
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">NO: 27)</td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">rs9951171</td>
<td class="description-td">18</td>
<td class="description-td">ACGGTTCTGTCCT</td>
<td class="description-td">ACGGTTCTGTCCTG</td>
<td class="description-td">ACGGTTC</td>
<td class="description-td">CCTGTTCACT</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">GTAGGGGAGAAA</td>
<td class="description-td">TAGGGGAGAAAAG</td>
<td class="description-td">TGTCCTG</td>
<td class="description-td">TGTGGCAGG</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">AGTCCTCGTTGTT</td>
<td class="description-td">TCCTCGTTGTTCCT</td>
<td class="description-td">TAGGGGA</td>
<td class="description-td">GCA</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">CCTCTGGGATGCA</td>
<td class="description-td">CTGGGATGCAACAT</td>
<td class="description-td">GA</td>
<td class="description-td">(rs9951171_C1_</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">ACATGAGAGAGC</td>
<td class="description-td">GAGAGAGCAGCAC</td>
<td class="description-td">(rs9951171_</td>
<td class="description-td">1_R; SEQ ID</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">AGCACACTGAGG</td>
<td class="description-td">ACTGAGGCTTTATG</td>
<td class="description-td">C1_1_F;</td>
<td class="description-td">NO: 86)</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">CTTTATGG<u> <b>A</b> </u>TTGC</td>
<td class="description-td">G<u> <b>G</b> </u>TTGCCCTGCCAC</td>
<td class="description-td">SEQ ID</td>
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">CCTGCCACAAGT</td>
<td class="description-td">AAGTGAACAGG</td>
<td class="description-td">NO: 85)</td>
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">GAACAGG (SEQ</td>
<td class="description-td">(SEQ ID NO: 30)</td>
<td class="description-td"> </td>
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">ID NO: 29)</td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">rs338882</td>
<td class="description-td"> 5</td>
<td class="description-td">GCGCAGTCAGAT</td>
<td class="description-td">GCGCAGTCAGATG</td>
<td class="description-td">GCGCAGT</td>
<td class="description-td">TCCAGCCCTT</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">GGGCGTGCTGGC</td>
<td class="description-td">GGCGTGCTGGCGTC</td>
<td class="description-td">CAGATGG</td>
<td class="description-td">GTCCCAAAC</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">GTCTGTCTTCTCT</td>
<td class="description-td">TGTCTTCTCTCTCTC</td>
<td class="description-td">GCGTGC</td>
<td class="description-td">GTGT</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">CTCTCCTGCTCTC</td>
<td class="description-td">CTGCTCTCTGGCTT</td>
<td class="description-td">(rs338882_</td>
<td class="description-td">(rs338882_C1_</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">TGGCTTCATTTTT</td>
<td class="description-td">CATTTTTCTCTCCTT</td>
<td class="description-td">C1_1_F;</td>
<td class="description-td">1_R; SEQ ID</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">CTCTCCTTCTGTC</td>
<td class="description-td">CTGTCTCACCTTCT</td>
<td class="description-td">SEQ ID</td>
<td class="description-td">NO: 88)</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">TCACCTTCTTTCG</td>
<td class="description-td">TTCGTGTGCCTGTG</td>
<td class="description-td">NO: 87)</td>
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">TGTGCCTGTGCA<u> <b>C</b> </u> </td>
<td class="description-td">CA<u> <b>T</b> </u>ACACACGTTTG</td>
<td class="description-td"> </td>
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">ACACACGTTTGG</td>
<td class="description-td">GGACAAGGG</td>
<td class="description-td"> </td>
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">GACAAGGG</td>
<td class="description-td">CTGGA (SEQ ID</td>
<td class="description-td"> </td>
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">CTGGA (SEQ ID</td>
<td class="description-td">NO: 32)</td>
<td class="description-td"> </td>
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">NO: 31)</td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">rs10776839</td>
<td class="description-td"> 9</td>
<td class="description-td">GCCGGACCTGCG</td>
<td class="description-td">GCCGGACCTGCGA</td>
<td class="description-td">GCCGGAC</td>
<td class="description-td">CGGGCAACT</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">AAATCCCAAAAT</td>
<td class="description-td">AATCCCAAAATGCC</td>
<td class="description-td">CTGCGAA</td>
<td class="description-td">GGGGCTCTG</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">GCCAAACATTCCC</td>
<td class="description-td">AAACATTCCCGCCT</td>
<td class="description-td">ATCCCAA</td>
<td class="description-td">ATC</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">GCCTCACATGATC</td>
<td class="description-td">CACATGATCCCAGA</td>
<td class="description-td">(rs10776839</td>
<td class="description-td">(rs10776839_C1_</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">CCAGAGAGAGGG</td>
<td class="description-td">GAGAGGGGACCCA</td>
<td class="description-td">C1_1_F;</td>
<td class="description-td">1_R; SEQ ID</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">GACCCAGTGTTCC</td>
<td class="description-td">GTGTTCCCAGCTTG</td>
<td class="description-td">SEQ ID</td>
<td class="description-td">NO: 90)</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">CAGCTTGCAGCTG</td>
<td class="description-td">CAGCTGAGGAGCC</td>
<td class="description-td">NO: 89)</td>
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">AGGAGCCCGAG<u> <b>G</b> </u> </td>
<td class="description-td">CGAG<u> <b>T</b> </u>TTGCCGTCA</td>
<td class="description-td"> </td>
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">TTGCCGTCAGATC</td>
<td class="description-td">GATCAGAGCCCCA</td>
<td class="description-td"> </td>
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">AGAGCCCCAGTT</td>
<td class="description-td">GTTGCCCG (SEQ ID</td>
<td class="description-td"> </td>
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">GCCCG (SEQ ID</td>
<td class="description-td">NO: 34)</td>
<td class="description-td"> </td>
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">NO: 33)</td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">rs9905977</td>
<td class="description-td">17</td>
<td class="description-td">AGCAGCCTCCCTC</td>
<td class="description-td">AGCAGCCTCCCTCG</td>
<td class="description-td">AGCAGCC</td>
<td class="description-td">GGCAGAGGG</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">GACTAGCTCACA</td>
<td class="description-td">ACTAGCTCACACTA</td>
<td class="description-td">TCCCTCG</td>
<td class="description-td">GAAAGACGA</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">CTACGATAAGGA</td>
<td class="description-td">CGATAAGGAAAAT</td>
<td class="description-td">ACTAGCT</td>
<td class="description-td">AAGGA</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">AAATTCATGAGCT</td>
<td class="description-td">TCATGAGCTGGTGT</td>
<td class="description-td">(rs9905977_</td>
<td class="description-td">(rs9905977_C1_</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">GGTGTCCAAGGA</td>
<td class="description-td">CCAAGGAGGGCTG</td>
<td class="description-td">C1_1_F;</td>
<td class="description-td">1_R; SEQ ID</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">GGGCTGGGTGAC</td>
<td class="description-td">GGTGACTCGTGGCT</td>
<td class="description-td">SEQ ID</td>
<td class="description-td">NO: 92)</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">TCGTGGCTCAGTC</td>
<td class="description-td">CAGTCAGC<u> <b>G</b> </u>TCAA</td>
<td class="description-td">NO: 91)</td>
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">AGC<u> <b>A</b> </u>TCAAGATT</td>
<td class="description-td">GATTCCTTTCGTCT</td>
<td class="description-td"> </td>
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">CCTTTCGTCTTTC</td>
<td class="description-td">TTCCCCTCTGCC</td>
<td class="description-td"> </td>
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">CCCTCTGCC (SEQ</td>
<td class="description-td">(SEQ ID NO: 36)</td>
<td class="description-td"> </td>
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">ID NO: 35)</td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">rs1277284</td>
<td class="description-td"> 4</td>
<td class="description-td">TGGCATTGCCTGT</td>
<td class="description-td">TGGCATTGCCTGTA</td>
<td class="description-td">TGGCATT</td>
<td class="description-td">AAGCACCAT</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">AATATACATAGC</td>
<td class="description-td">ATATACATAGCCAT</td>
<td class="description-td">GCCTGTA</td>
<td class="description-td">TCTAATGATT</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">CATGGTTTTTTAT</td>
<td class="description-td">GGTTTTTTATAGGC</td>
<td class="description-td">ATATACA</td>
<td class="description-td">TTGG</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">AGGCAATTTAAG</td>
<td class="description-td">AATTTAAGATGAAT</td>
<td class="description-td">TAG</td>
<td class="description-td">(rs1277284_C4_</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">ATGAATAGCTTCT</td>
<td class="description-td">AGCTTCTAAACTAT</td>
<td class="description-td">(rs1277284_</td>
<td class="description-td">1_R; SEQ ID</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">AAACTATAGATA</td>
<td class="description-td">AGATAAGTTTCATT</td>
<td class="description-td">C4_1_F;</td>
<td class="description-td">NO: 94)</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">AGTTTCATTACCC</td>
<td class="description-td">ACCCCAGGAAGCT</td>
<td class="description-td">SEQ ID</td>
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">CAGGAAGCTGAA</td>
<td class="description-td">GAACTATAGCTACT</td>
<td class="description-td">NO: 93)</td>
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">CTATAGCTACTTT</td>
<td class="description-td">TT<u> <b>C</b> </u>CCCAAAATCAT</td>
<td class="description-td"> </td>
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> <u> <b>A</b> </u>CCCAAAATCATT</td>
<td class="description-td">TAGAATGGTGCTT</td>
<td class="description-td"> </td>
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">AGAATGGTGCTT</td>
<td class="description-td">(SEQ ID NO: 38)</td>
<td class="description-td"> </td>
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">(SEQ ID NO: 37)</td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">rs258684</td>
<td class="description-td"> 7</td>
<td class="description-td">ATGAAGCCTTCCA</td>
<td class="description-td">ATGAAGCCTTCCAC</td>
<td class="description-td">ATGAAGC</td>
<td class="description-td">GATCAGTTGT</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">CCAACTGCCTGTA</td>
<td class="description-td">CAACTGCCTGTATG</td>
<td class="description-td">CTTCCAC</td>
<td class="description-td">TGTTTCTATA</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">TGACTCATCTGGG</td>
<td class="description-td">ACTCATCTGGGGAC</td>
<td class="description-td">CAACTG</td>
<td class="description-td">TTTCCTT</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">GACTTCTGCTCTA</td>
<td class="description-td">TTCTGCTCTATACT</td>
<td class="description-td">(rs258684_</td>
<td class="description-td">(rs258684_C7_</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">TACTCAAAGTGG</td>
<td class="description-td">CAAAGTGGCTTAGT</td>
<td class="description-td">C7_1_F;</td>
<td class="description-td">1_R; SEQ ID</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">CTTAGTCACTGCC</td>
<td class="description-td">CACTGCCAATGTAT</td>
<td class="description-td">SEQ ID</td>
<td class="description-td">NO: 96)</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">AATGTATTTCCAT</td>
<td class="description-td">TTCCATATGAGGGA</td>
<td class="description-td">NO: 95)</td>
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">ATGAGGGACG<u> <b>A</b> </u>T</td>
<td class="description-td">CG<u> <b>G</b> </u>TGATTACTAAG</td>
<td class="description-td"> </td>
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">GATTACTAAGGA</td>
<td class="description-td">GAAATATAGAAAC</td>
<td class="description-td"> </td>
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">AATATAGAAACA</td>
<td class="description-td">AACAACTGATC</td>
<td class="description-td"> </td>
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">ACAACTGATC</td>
<td class="description-td">(SEQ ID NO: 40)</td>
<td class="description-td"> </td>
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">(SEQ ID NO: 39)</td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">rs1347696</td>
<td class="description-td"> 8</td>
<td class="description-td">ACAACAGAATCA</td>
<td class="description-td">ACAACAGAATCAG</td>
<td class="description-td">ACAACAG</td>
<td class="description-td">CTGAACTGA</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">GGTGATTGGAGA</td>
<td class="description-td">GTGATTGGAGAAA</td>
<td class="description-td">AATCAGG</td>
<td class="description-td">ACAAAGAAT</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">AAAGATCACAGG</td>
<td class="description-td">AGATCACAGGCCT</td>
<td class="description-td">TGATTGG</td>
<td class="description-td">TAAGGTC</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">CCTAGGCACCCA</td>
<td class="description-td">AGGCACCCAAGGC</td>
<td class="description-td">A</td>
<td class="description-td">(rs1347696_C8_</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">AGGCTTGAAGGA</td>
<td class="description-td">TTGAAGGATGAAA</td>
<td class="description-td">(rs1347696_</td>
<td class="description-td">4_F; SEQ ID</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">TGAAAGAATGAA</td>
<td class="description-td">GAATGAAAGATGG</td>
<td class="description-td">C8_4_F;</td>
<td class="description-td">NO: 98)</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">AGATGGACGGAA</td>
<td class="description-td">ACGGAA<u> <b>G</b> </u>AAAATT</td>
<td class="description-td">SEQ ID</td>
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> <u> <b>C</b> </u>AAAATTAGGAC</td>
<td class="description-td">AGGACCTTAATTCT</td>
<td class="description-td">NO: 97)</td>
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">CTTAATTCTTTGT</td>
<td class="description-td">TTGTTCAGTTCAG</td>
<td class="description-td"> </td>
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">TCAGTTCAG (SEQ</td>
<td class="description-td">(SEQ ID NO: 42)</td>
<td class="description-td"> </td>
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">ID NO: 41)</td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">rs508485</td>
<td class="description-td">11</td>
<td class="description-td">TTGGGGTAAATTT</td>
<td class="description-td">TTGGGGTAAATTTT</td>
<td class="description-td">TTGGGGT</td>
<td class="description-td">GGGGTGGGA</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">TCATTGTCATATG</td>
<td class="description-td">CATTGTCATATGTG</td>
<td class="description-td">AAATTTT</td>
<td class="description-td">ATTAGACTCT</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">TGGAATTTAAATA</td>
<td class="description-td">GAATTTAAATATAC</td>
<td class="description-td">CATTGTC</td>
<td class="description-td">G</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">TACCATCATCTAC</td>
<td class="description-td">CATCATCTACAAAG</td>
<td class="description-td">A</td>
<td class="description-td">(rs508485_C11_</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">AAAGAATTCCAC</td>
<td class="description-td">AATTCCACAGAGTT</td>
<td class="description-td">(rs508485_</td>
<td class="description-td">1_R; SEQ ID</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">AGAGTTAAATAT</td>
<td class="description-td">AAATATCTTAAGTT</td>
<td class="description-td">C11_1_F;</td>
<td class="description-td">NO 100)</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">CTTAAGTTAAACA</td>
<td class="description-td">AAACACTTAAAAT</td>
<td class="description-td">SEQ ID</td>
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">CTTAAAATAAGT</td>
<td class="description-td">AAGTGTTTGCGTGA</td>
<td class="description-td">NO: 99)</td>
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">GTTTGCGTGATAT</td>
<td class="description-td">TATTTTGATGA<u> <b>T</b> </u>AG</td>
<td class="description-td"> </td>
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">TTTGATGA<u> <b>C</b> </u>AGAT</td>
<td class="description-td">ATAAACAGAGTCT</td>
<td class="description-td"> </td>
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">AAACAGAGTCTA</td>
<td class="description-td">AATTCCCACCCC</td>
<td class="description-td"> </td>
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">ATTCCCACCCC</td>
<td class="description-td">(SEQ ID NO:44)</td>
<td class="description-td"> </td>
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">(SEQ ID NO:43)</td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">rs9788670</td>
<td class="description-td">15</td>
<td class="description-td">TGCAATTCAAATC</td>
<td class="description-td">TGCAATTCAAATCA</td>
<td class="description-td">TGCAATT</td>
<td class="description-td">GCAACATCG</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">AGGAAGTATGAC</td>
<td class="description-td">GGAAGTATGACCA</td>
<td class="description-td">CAAATCA</td>
<td class="description-td">AGGTTTGTCA</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">CAAAAGACAGAG</td>
<td class="description-td">AAAGACAGAGATC</td>
<td class="description-td">GGAAGTA</td>
<td class="description-td">G</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">ATCTTTTTTGGAT</td>
<td class="description-td">TTTTTTGGATGATC</td>
<td class="description-td">TG</td>
<td class="description-td">(rs9788670_c15_</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">GATCCCTAGCCTA</td>
<td class="description-td">CCTAGCCTAGCAAT</td>
<td class="description-td">(rs9788670_</td>
<td class="description-td">2_R; SEQ ID</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">GCAATGCCTGGC</td>
<td class="description-td">GCCTGGCAGCCATG</td>
<td class="description-td">c15_2_F;</td>
<td class="description-td">NO: 102)</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">AGCCATGCAGGT</td>
<td class="description-td">CAGGTGCAATGTCA</td>
<td class="description-td">SEQ ID</td>
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">GCAATGTCAACCT</td>
<td class="description-td">ACCTTAAATAATGT</td>
<td class="description-td">NO: 101)</td>
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">TAAATAATGTATT</td>
<td class="description-td">ATTGCAAA<u> <b>T</b> </u>TCAGA</td>
<td class="description-td"> </td>
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">GCAAA<u> <b>C</b> </u>TCAGAG</td>
<td class="description-td">GCTGACAAACCTCG</td>
<td class="description-td"> </td>
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">CTGACAAACCTC</td>
<td class="description-td">ATGTTGC (SEQ ID</td>
<td class="description-td"> </td>
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">GATGTTGC (SEQ</td>
<td class="description-td">NO: 46)</td>
<td class="description-td"> </td>
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">ID NO: 45)</td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">rs8137254</td>
<td class="description-td">22</td>
<td class="description-td">CTGTGCTCTGCGA</td>
<td class="description-td">CTGTGCTCTGCGAA</td>
<td class="description-td">CTGTGCT</td>
<td class="description-td">ACCATGCTCA</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">ATAGCTGCAGAA</td>
<td class="description-td">TAGCTGCAGAAGT</td>
<td class="description-td">CTGCGAA</td>
<td class="description-td">TGGAGAATC</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">GTAACTTGGGGA</td>
<td class="description-td">AACTTGGGGACCC</td>
<td class="description-td">TAGCTG</td>
<td class="description-td">C</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">CCCAAAATAAAG</td>
<td class="description-td">AAAATAAAGCAGA</td>
<td class="description-td">(rs8137254_</td>
<td class="description-td">(rs8137254_c22_</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">CAGAATGCTAAT</td>
<td class="description-td">ATGCTAATGTCAAG</td>
<td class="description-td">c22_2_F:</td>
<td class="description-td">2_R; SEQ ID</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">GTCAAGTCCTGA</td>
<td class="description-td">TCCTGAGAACCAA</td>
<td class="description-td">SEQ ID</td>
<td class="description-td">NO: 104)</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">GAACCAAGCCCT</td>
<td class="description-td">GCCCTGGGACTCTG</td>
<td class="description-td">NO: 103)</td>
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">GGGACTCTGGTG</td>
<td class="description-td">GTGCCATTT<u> <b>T</b> </u>GGAT</td>
<td class="description-td"> </td>
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">CCATTT<u> <b>C</b> </u>GGATTC</td>
<td class="description-td">TCTCCATGAGCATG</td>
<td class="description-td"> </td>
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">TCCATGAGCATG</td>
<td class="description-td">GT (SEQ ID</td>
<td class="description-td"> </td>
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">GT (SEQ ID</td>
<td class="description-td">NO: 48)</td>
<td class="description-td"> </td>
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">NO: 47)</td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">rs3143</td>
<td class="description-td">19</td>
<td class="description-td">TTTTTCCAGCCAA</td>
<td class="description-td">TTTTTCCAGCCAAC</td>
<td class="description-td">TTTTTCCA</td>
<td class="description-td">CACAGCTTG</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">CTCAAGGCCAAA</td>
<td class="description-td">TCAAGGCCAAAAA</td>
<td class="description-td">GCCAACT</td>
<td class="description-td">AGGTTTCTTG</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">AAAAATTTCTTAA</td>
<td class="description-td">AAATTTCTTAATAT</td>
<td class="description-td">CAAGG</td>
<td class="description-td">TG</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">TATAGTTATTATG</td>
<td class="description-td">AGTTATTATGCGAG</td>
<td class="description-td">(rs3143_c19_</td>
<td class="description-td">(rs3143_c19_2_</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">CGAGGGGAGGGG</td>
<td class="description-td">GGGAGGGGAAGCA</td>
<td class="description-td">2_F: SEQ</td>
<td class="description-td">R; SEQ ID</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">AAGCAAAGGAGC</td>
<td class="description-td">AAGGAGCACAGGT</td>
<td class="description-td">ID NO: 105)</td>
<td class="description-td">NO: 106)</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">ACAGGTAGTCCA</td>
<td class="description-td">AGTCCACAGAATA</td>
<td class="description-td"> </td>
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">CAGAATA<u> <b>A</b> </u>GACA</td>
<td class="description-td"> <u> <b>G</b> </u>GACACAAGAAAC</td>
<td class="description-td"> </td>
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">CAAGAAACCTCA</td>
<td class="description-td">CTCAAGCTGTG</td>
<td class="description-td"> </td>
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">AGCTGTG (SEQ ID</td>
<td class="description-td">(SEQ ID NO: 50)</td>
<td class="description-td"> </td>
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">NO: 49)</td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">rs2182957</td>
<td class="description-td">13</td>
<td class="description-td">TCTTCTCGTCCCC</td>
<td class="description-td">TCTTCTCGTCCCCT</td>
<td class="description-td">TCTTCTC</td>
<td class="description-td">TTTCTGGTTT</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">TAAGCAAACAAC</td>
<td class="description-td">AAGCAAACAACAT</td>
<td class="description-td">GTCCCCT</td>
<td class="description-td">GTGCAACAG</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">ATCCGCTTGCTTC</td>
<td class="description-td">CCGCTTGCTTCTGT</td>
<td class="description-td">AAGCAA</td>
<td class="description-td">G</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">TGTCTGTGTAACC</td>
<td class="description-td">CTGTGTAACCACAG</td>
<td class="description-td">(rs2182957_</td>
<td class="description-td">(rs2182957_c13_</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">ACAGTGAATGGG</td>
<td class="description-td">TGAATGGGTGTGCA</td>
<td class="description-td">c13_1_F:</td>
<td class="description-td">1_R; SEQ ID</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">TGTGCACGCTTG<u> <b>A</b> </u> </td>
<td class="description-td">CGCTTG<u> <b>G</b> </u>TGGGCCT</td>
<td class="description-td">SEQ ID</td>
<td class="description-td">NO: 108)</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">TGGGCCTCTGAGC</td>
<td class="description-td">CTGAGCCCCTGTTG</td>
<td class="description-td">NO: 107)</td>
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">CCCTGTTGCACAA</td>
<td class="description-td">CACAAACCAGAAA</td>
<td class="description-td"> </td>
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">ACCAGAAA (SEQ</td>
<td class="description-td">(SEQ ID NO: 52)</td>
<td class="description-td"> </td>
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">ID NO: 51)</td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">rs3739005</td>
<td class="description-td"> 2</td>
<td class="description-td">CACATGGGGGCA</td>
<td class="description-td">CACATGGGGGCATT</td>
<td class="description-td">CACATGG</td>
<td class="description-td">ACATCGATG</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">TTAAGAATCGCCC</td>
<td class="description-td">AAGAATCGCCCAG</td>
<td class="description-td">GGGCATT</td>
<td class="description-td">AGCACAAAA</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">AGGGAGGAGGAG</td>
<td class="description-td">GGAGGAGGAGGGA</td>
<td class="description-td">AAGAAT</td>
<td class="description-td">ACAC</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">GGAGAACGCGTG</td>
<td class="description-td">GAACGCGTGCTTTT</td>
<td class="description-td">(rs3739005_</td>
<td class="description-td">(rs3739005_c2_</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">CTTTTCACATTTG</td>
<td class="description-td">CACATTTGCATTTG</td>
<td class="description-td">c2_2_F;</td>
<td class="description-td">2_R; SEQ ID</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">CATTTGAATTTT<u> <b>C</b> </u> </td>
<td class="description-td">AATTTT<u> <b>T</b> </u>GAGTTCC</td>
<td class="description-td">SEQ ID</td>
<td class="description-td">NO: 110)</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">GAGTTCCCAGGA</td>
<td class="description-td">CAGGATGTGTTTTT</td>
<td class="description-td">NO: 109)</td>
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">TGTGTTTTTGTGC</td>
<td class="description-td">GTGCTCATCGATGT</td>
<td class="description-td"> </td>
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">TCATCGATGT</td>
<td class="description-td">(SEQ ID NO: 54)</td>
<td class="description-td"> </td>
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">(SEQ ID NO: 53)</td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">rs530022</td>
<td class="description-td"> 1</td>
<td class="description-td">GGGCTCTGAGGT</td>
<td class="description-td">GGGCTCTGAGGTGT</td>
<td class="description-td">GGGCTCT</td>
<td class="description-td">AGATATCCCT</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">GTGTGAAATAAA</td>
<td class="description-td">GTGAAATAAAAAC</td>
<td class="description-td">GAGGTGT</td>
<td class="description-td">GGAACTGTT</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">AACAAATGTCCA</td>
<td class="description-td">AAATGTCCATGTCT</td>
<td class="description-td">GTGAAA</td>
<td class="description-td">ATTCC</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">TGTCTGTCCTTTT</td>
<td class="description-td">GTCCTTTTATGGCA</td>
<td class="description-td">(rs530022_</td>
<td class="description-td">(rs530022_c1_</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">ATGGCATTTTGGG</td>
<td class="description-td">TTTTGGGACTTTAC</td>
<td class="description-td">c1_2_F; SEQ</td>
<td class="description-td">2_R; SEQ ID</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">ACTTTACATTTCA</td>
<td class="description-td">ATTTCAAACATTTC</td>
<td class="description-td">ID NO: 111)</td>
<td class="description-td">NO: 112)</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">AACATTTCAGAC</td>
<td class="description-td">AGACATGTATCACA</td>
<td class="description-td"> </td>
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">ATGTATCACAAC</td>
<td class="description-td">ACACGA<u> <b>G</b> </u>GGAATA</td>
<td class="description-td"> </td>
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">ACGA<u> <b>A</b> </u>GGAATAA</td>
<td class="description-td">ACAGTTCCAGGGAT</td>
<td class="description-td"> </td>
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">CAGTTCCAGGGA</td>
<td class="description-td">ATCT (SEQ ID</td>
<td class="description-td"> </td>
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">TATCT (SEQ ID</td>
<td class="description-td">NO: 56)</td>
<td class="description-td"> </td>
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">NO: 55)</td>
</tr>
<tr class="description-tr">
<td align="center" class="description-td" colspan="6" nameend="6" namest="1" rowsep="1"> </td>
</tr>
</tbody></table>
</patent-tables>
</tables>
</div>
</li> <heading id="h-0020">Example 6</heading>
<heading id="h-0021">Identification of SNPs by Massively Parallel Sequencing of a Target Library: Determination of Fetal Fraction</heading>
<li> <para-num num="[0237]"> </para-num> <div class="description-line" id="p-0250" num="0237">To determine the fraction of fetal cfDNA in a maternal sample, target polymorphic nucleic acid sequences each comprising a SNP were amplified and used for preparing a target library for sequencing in a massively parallel fashion.</div>
</li> <li> <para-num num="[0238]"> </para-num> <div class="description-line" id="p-0251" num="0238">cfDNA was extracted as described in Example 4. A target sequencing library was prepared as follows. cfDNA contained in 5 μl of purified cfDNA was amplified in a reaction volume of 50 μl containing 7.5 μl of a 1 μM primer mix (Table 1), 10 μl of NEB 5× Mastermix and 27 μl water. Thermal cycling was performed with the Gene Amp9700 (Applied Biosystems) using the following cycling conditions: incubating at 95° C. for 1 minute, followed by 20-30 cycles at 95° C. for 20 seconds, 68° C. for 1 minute, and 68° C. for 30 s, which was followed by a final incubation at 68° C. for 5 minutes. A final hold at 4° C. was added until the samples were removed for combining with the unamplified portion of the purified cfDNA sample. The amplified product was purified using the Agencourt AMPure XP PCR purification system (Part No. A63881; Beckman Coulter Genomics, Danvers, Mass.). A final hold at 4° C. was added until the samples were removed for preparing the target library. The amplified product was analyzed with a 2100 Bioanalyzer (Agilent Technologies, Sunnyvale, Calif.), and the concentration of amplified product determined. A sequencing library of amplified target nucleic acids was prepared as described in Example 2, and was sequenced in a massively parallel fashion using sequencing-by-synthesis with reversible dye terminators and according to the Illumina protocol (BioTechniques® Protocol Guide 2007 Published December 2006: p 29, and on the world wide web at biotechniques.com/default.asp?page=protocol&amp;subsection=article_display&amp;id=112378). Analysis and counting of tags mapped to a reference genome consisting of 26 sequences (13 pairs each representing two alleles) comprising a SNP i.e. SEQ ID NO:1-26 was performed as described.</div>
</li> <li> <para-num num="[0239]"> </para-num> <div class="description-line" id="p-0252" num="0239">Table 11 provides the tag counts obtained from sequencing the target library, and the calculated fetal fraction derived from sequencing data.</div>
</li> <li> <div class="description-line" id="p-0253" num="0000">
<tables id="TABLE-US-00011" num="00011">
<patent-tables colsep="0" frame="none" rowsep="0">
<table align="left" class="description-table" cols="1" colsep="0" rowsep="0" width="100%">
<thead>
<tr class="description-tr">
<td class="description-td" colspan="1" nameend="1" namest="1" rowsep="1">TABLE 11</td>
</tr>
</thead>
<tbody><tr class="description-tr">
<td align="center" class="description-td" colspan="1" nameend="1" namest="1" rowsep="1"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">Determination of Fetal Fraction by Massively Parallel</td>
</tr>
<tr class="description-tr">
<td class="description-td">Sequencing of a Library of Polymorphic Nucleic Acids</td>
</tr>
</tbody></table>
<table align="left" class="description-table" cols="3" colsep="0" rowsep="0" width="100%">
<tbody><tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">SNP TAG</td>
<td class="description-td">Fetal Fraction</td>
</tr>
<tr class="description-tr">
<td class="description-td">SNP</td>
<td class="description-td">COUNTS</td>
<td class="description-td">(%)</td>
</tr>
<tr class="description-tr">
<td align="center" class="description-td" colspan="3" nameend="3" namest="1" rowsep="1"> </td>
</tr>
</tbody></table>
<table align="left" class="description-table" cols="3" colsep="0" rowsep="0" width="100%">
<tbody><tr class="description-tr">
<td class="description-td">rs10773760.1|Chr.12|length = 128|allele = A</td>
<td class="description-td">236590</td>
<td class="description-td">1.98</td>
</tr>
<tr class="description-tr">
<td class="description-td">rs10773760.2|Chr.12|length = 128|allele = G</td>
<td class="description-td">4680</td>
</tr>
<tr class="description-tr">
<td class="description-td">rs13182883.1|Chr.5|length = 111|allele = A</td>
<td class="description-td">3607</td>
<td class="description-td">4.99</td>
</tr>
<tr class="description-tr">
<td class="description-td">rs13182883.2|Chr.5|length = 111|allele = G</td>
<td class="description-td">72347</td>
</tr>
<tr class="description-tr">
<td class="description-td">rs4530059.1|Chr.14|length = 110|allele = A</td>
<td class="description-td">3698</td>
<td class="description-td">1.54</td>
</tr>
<tr class="description-tr">
<td class="description-td">rs4530059.1|Chr.14|length = 110|allele = G</td>
<td class="description-td">239801</td>
</tr>
<tr class="description-tr">
<td class="description-td">rs8078417.1|Chr.17|length = 110|allele = C</td>
<td class="description-td">1E+06</td>
<td class="description-td">3.66</td>
</tr>
<tr class="description-tr">
<td class="description-td">rs8078417.2|Chr.17|length = 110|allele = T</td>
<td class="description-td">50565</td>
</tr>
<tr class="description-tr">
<td align="center" class="description-td" colspan="3" nameend="3" namest="1" rowsep="1"> </td>
</tr>
<tr class="description-tr">
<td align="left" class="description-td" colspan="3" id="FOO-00001" nameend="3" namest="1">Fetal Fraction (Mean ± S.D.) = 12.4 ± 6.6</td>
</tr>
</tbody></table>
</patent-tables>
</tables>
</div>
</li> <li> <para-num num="[0240]"> </para-num> <div class="description-line" id="p-0254" num="0240">The results show that polymorphic nucleic acid sequences each comprising at least one SNP can be amplified from cfDNA derived from a maternal plasma sample to construct a library that can be sequenced in a massively parallel fashion to determine the fraction of fetal nucleic acids in the maternal sample.</div>
</li> <heading id="h-0022">Example 7</heading>
<heading id="h-0023">Identification of SNPs Following Enrichment of Fetal and Maternal Nucleic Acids in a cfDNA Sequencing Library Sample: Determination of Fetal Fraction</heading>
<li> <para-num num="[0241]"> </para-num> <div class="description-line" id="p-0255" num="0241">To enrich the fetal and maternal cfDNA contained in a primary sequencing library constructed using purified fetal and maternal cfDNA, a portion of a purified cfDNA sample was used for amplifying polymorphic target nucleic acid sequences, and for preparing a sequencing library of amplified polymorphic target nucleic acids, which was used to enrich the fetal and maternal nucleic acid sequences comprised in the primary library.</div>
</li> <li> <para-num num="[0242]"> </para-num> <div class="description-line" id="p-0256" num="0242">The method corresponds to workflow 3 diagrammed in <figref idrefs="DRAWINGS">FIG. 5</figref>. A target sequencing library was prepared from a portion of the purified cfDNA as described in Example 2. A primary sequencing library was prepared using the remaining portion of the purified cfDNA as described in Example 4. Enrichment of the primary library for the amplified polymorphic nucleic acids comprised in the target library was obtained by diluting the primary and the target sequencing libraries to 10 nM, and combining the target library with the primary library at a ratio of 1:9 to provide an enriched sequencing library. Sequencing of the enriched library and analysis of the sequencing data was performed as described in Example 4.</div>
</li> <li> <para-num num="[0243]"> </para-num> <div class="description-line" id="p-0257" num="0243">Table 12 provides the number of sequence tags that mapped to the SNP genome for the informative SNPs identified from sequencing an enriched library derived from plasma samples of pregnant women each carrying a T21, a T13, a T18 and a monosomy X fetus, respectively. Fetal fraction was calculated as follows:</div>
</li> <li> <div class="description-line" id="p-0258" num="0000"> <br/>% fetal fraction allele<sub>x</sub>=((ΣFetal sequence tags for allele<sub>x</sub>)/(ΣMaternal sequence tags for allele<sub>x</sub>))×100
</div>
</li> <li> <para-num num="[0244]"> </para-num> <div class="description-line" id="p-0259" num="0244">Table 4 also provides the number of the sequence tags mapped to the human reference genome. Tags mapped to the human reference genome were used to determine the presence or absence of aneuploidy using the same plasma sample that was utilized for determining the corresponding fetal fraction. Method for using sequence tags counts for determining aneuploidy are described in pending U.S. patent application Ser. Nos. 12/958,352, 12/958,356 and 12/958,353, which are herein incorporated by reference in their entirety.</div>
</li> <li> <div class="description-line" id="p-0260" num="0000">
<tables id="TABLE-US-00012" num="00012">
<patent-tables colsep="0" frame="none" pgwide="1" rowsep="0">
<table align="left" class="description-table" cols="1" colsep="0" rowsep="0" width="100%">
<thead>
<tr class="description-tr">
<td class="description-td" colspan="1" nameend="1" namest="1" rowsep="1">TABLE 12</td>
</tr>
<tr class="description-tr">
<td align="center" class="description-td" colspan="1" nameend="1" namest="1" rowsep="1"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">Determination of Fetal Fraction by Massively Parallel Sequencing</td>
</tr>
<tr class="description-tr">
<td class="description-td">of an Enriched Library of Polymorphic Nucleic Acids</td>
</tr>
<tr class="description-tr">
<td align="center" class="description-td" colspan="1" nameend="1" namest="1" rowsep="1"> </td>
</tr>
</thead>
<tbody><tr class="description-tr">
<td class="description-td"> </td>
</tr>
</tbody></table>
<table align="left" class="description-table" cols="4" colsep="0" rowsep="0" width="100%">
<tbody><tr class="description-tr">
<td class="description-td">Sample ID</td>
<td class="description-td"> </td>
<td class="description-td">SNP TAG</td>
<td class="description-td">FETAL FRACTION</td>
</tr>
<tr class="description-tr">
<td class="description-td">(karyotype)</td>
<td class="description-td">SNP</td>
<td class="description-td">COUNTS</td>
<td class="description-td">(%)</td>
</tr>
<tr class="description-tr">
<td align="center" class="description-td" colspan="4" nameend="4" namest="1" rowsep="1"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">11409</td>
<td class="description-td">rs13182883.1|Chr.5|length = 111|allele = A</td>
<td class="description-td">261</td>
<td class="description-td">4.41</td>
</tr>
<tr class="description-tr">
<td class="description-td">(47, XY + 21)</td>
<td class="description-td">rs13182883.2|Chr.5|length = 111|allele = G</td>
<td class="description-td">5918</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">rs740598.1|Chr.10|length = 114|allele = A</td>
<td class="description-td">5545</td>
<td class="description-td">7.30</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">rs740598.2|Chr.10|length = 114|allele = G</td>
<td class="description-td">405</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">rs8078417.1|Chr.17|length = 110|allele = C</td>
<td class="description-td">8189</td>
<td class="description-td">6.74</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">rs8078417.2|Chr.17|length = 110|allele = T</td>
<td class="description-td">121470</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">rs576261.1|Chr.19|length = 114|allele = A</td>
<td class="description-td">58342</td>
<td class="description-td">7.62</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">rs576261.2|Chr.19|length = 114|allele = C</td>
<td class="description-td">4443</td>
</tr>
<tr class="description-tr">
<td align="center" class="description-td" colspan="4" nameend="4" namest="1" rowsep="1"> </td>
</tr>
</tbody></table>
<table align="left" class="description-table" cols="1" colsep="0" rowsep="0" width="100%">
<tbody><tr class="description-tr">
<td class="description-td">Fetal Fraction (Mean ± S.D.) = 6.5 ± 1.5</td>
</tr>
</tbody></table>
<table align="left" class="description-table" cols="4" colsep="0" rowsep="0" width="100%">
<tbody><tr class="description-tr">
<td class="description-td">Sample ID</td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td align="center" class="description-td" colspan="4" nameend="4" namest="1" rowsep="1"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">95133</td>
<td class="description-td">rs1109037.1|Chr.2|length = 126|allele = A</td>
<td class="description-td">12229</td>
<td class="description-td">2.15</td>
</tr>
<tr class="description-tr">
<td class="description-td">(47, XX + 18)</td>
<td class="description-td">rs1109037.2|Chr.2|length = 126|allele = G</td>
<td class="description-td">263</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">rs13218440.1|Chr.6|length = 139|allele = A</td>
<td class="description-td">55949</td>
<td class="description-td">3.09</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">rs13218440.2|Chr.6|length = 139|allele = G</td>
<td class="description-td">1729</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">rs7041158.1|Chr.9|length = 117|allele = C</td>
<td class="description-td">7281</td>
<td class="description-td">4.12</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">rs7041158.2|Chr.9|length = 117|allele = T</td>
<td class="description-td">300</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">rs7205345.1|Chr.16|length = 116|allele = C</td>
<td class="description-td">53999</td>
<td class="description-td">2.14</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">rs7205345.2|Chr.16|length = 116|allele = G</td>
<td class="description-td">1154</td>
</tr>
<tr class="description-tr">
<td align="center" class="description-td" colspan="4" nameend="4" namest="1" rowsep="1"> </td>
</tr>
</tbody></table>
<table align="left" class="description-table" cols="1" colsep="0" rowsep="0" width="100%">
<tbody><tr class="description-tr">
<td class="description-td">Fetal Fraction (Mean ± S.D.) = 2.9 ± 0.9</td>
</tr>
</tbody></table>
<table align="left" class="description-table" cols="4" colsep="0" rowsep="0" width="100%">
<tbody><tr class="description-tr">
<td class="description-td">Sample ID</td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td align="center" class="description-td" colspan="4" nameend="4" namest="1" rowsep="1"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">51236</td>
<td class="description-td">rs13218440.1|Chr.6|length = 139|allele = A</td>
<td class="description-td">1119</td>
<td class="description-td">1.65</td>
</tr>
<tr class="description-tr">
<td class="description-td">(46, XY + 13)</td>
<td class="description-td">rs13218440.2|Chr.6|length = 139|allele = G</td>
<td class="description-td">67756</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">rs560681.1|Chr.1|length = 111|allele = A</td>
<td class="description-td">14123</td>
<td class="description-td">5.18</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">rs560681.2|Chr.1|length = 111|allele = G</td>
<td class="description-td">732</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">rs7205345.1|Chr.16|length = 116|allele = C</td>
<td class="description-td">18176</td>
<td class="description-td">1.63</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">rs7205345.2|Chr.16|length = 116|allele = G</td>
<td class="description-td">296</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">rs9866013.1|Chr.3|length = 121|allele = C</td>
<td class="description-td">117</td>
<td class="description-td">2.33</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">rs9866013.2|Chr.3|length = 121|allele = T</td>
<td class="description-td">5024</td>
</tr>
<tr class="description-tr">
<td align="center" class="description-td" colspan="4" nameend="4" namest="1" rowsep="1"> </td>
</tr>
</tbody></table>
<table align="left" class="description-table" cols="1" colsep="0" rowsep="0" width="100%">
<tbody><tr class="description-tr">
<td class="description-td">Fetal Fraction (Mean ± S.D.) = 2.7 ± 1.7</td>
</tr>
</tbody></table>
<table align="left" class="description-table" cols="4" colsep="0" rowsep="0" width="100%">
<tbody><tr class="description-tr">
<td class="description-td">Sample ID</td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td align="center" class="description-td" colspan="4" nameend="4" namest="1" rowsep="1"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">54430</td>
<td class="description-td">rs1109037.1|Chr.2|length = 126|allele = A</td>
<td class="description-td">19841</td>
<td class="description-td">1.80</td>
</tr>
<tr class="description-tr">
<td class="description-td">(45, XO)</td>
<td class="description-td">rs1109037.2|Chr.2|length = 126|allele = G</td>
<td class="description-td">357</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">rs9866013.1|Chr.3|length = 121|allele = C</td>
<td class="description-td">12931</td>
<td class="description-td">3.81</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">rs9866013.2|Chr.3|length = 121|allele = T</td>
<td class="description-td">493</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">rs7041158.1|Chr.9|length = 117|allele = C</td>
<td class="description-td">2800</td>
<td class="description-td">4.25</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">rs7041158.2|Chr.9|length = 117|allele = T</td>
<td class="description-td">119</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">rs740598.1|Chr.10|length = 114|allele = A</td>
<td class="description-td">12903</td>
<td class="description-td">4.87</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">rs740598.2|Chr.10|length = 114|allele = G</td>
<td class="description-td">628</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">rs10773760.1|Chr.12|length = 128|allele = A</td>
<td class="description-td">46324</td>
<td class="description-td">4.65</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">rs10773760.2|Chr.12|length = 128|allele = G</td>
<td class="description-td">2154</td>
</tr>
<tr class="description-tr">
<td align="center" class="description-td" colspan="4" nameend="4" namest="1" rowsep="1"> </td>
</tr>
</tbody></table>
<table align="left" class="description-table" cols="1" colsep="0" rowsep="0" width="100%">
<tbody><tr class="description-tr">
<td class="description-td">Fetal Fraction (Mean ± S.D.) = 3.9 ± 1.2</td>
</tr>
<tr class="description-tr">
<td align="center" class="description-td" colspan="1" nameend="1" namest="1" rowsep="1"> </td>
</tr>
</tbody></table>
</patent-tables>
</tables>
</div>
</li> <heading id="h-0024">Example 8</heading>
<heading id="h-0025">Identification of SNPs by Massively Parallel Sequencing: Enrichment of Fetal and Maternal Nucleic Acids for Polymorphic Nucleic Acids in a Purified cfDNA Sample and Determination of Fetal Fraction</heading>
<li> <para-num num="[0245]"> </para-num> <div class="description-line" id="p-0261" num="0245">To enrich the fetal and maternal cfDNA contained in a purified sample of cfDNA extracted from a maternal plasma sample, a portion of the purified cfDNA was used for amplifying polymorphic target nucleic acid sequences each comprising one SNP chosen from the panel of SNPs given in Table 9.</div>
</li> <li> <para-num num="[0246]"> </para-num> <div class="description-line" id="p-0262" num="0246">The method corresponds to workflow 2 diagrammed in <figref idrefs="DRAWINGS">FIG. 4</figref>. Cell-free plasma was obtained from a maternal blood sample, and cfDNA was purified from the plasma sample as described in Example 4. The final concentration was determined to be 92.8 pg/μl. cfDNA contained in 5 μl of purified cfDNA was amplified in a reaction volume of 50 μl containing 7.5 μl of a 1 uM primer mix (Table 9), 10 μl of NEB 5× Mastermix and 27 μl water. Thermal cycling was performed with the Gene Amp9700 (Applied Biosystems). Using the following cycling conditions: incubating at 95° C. for 1 minute, followed by 30 cycles at 95° C. for 20 seconds, 68° C. for 1 minute, and 68° C. for 30 s, which was followed by a final incubation at 68° C. for 5 minutes. A final hold at 4° C. was added until the samples were removed for combining with the unamplified portion of the purified cfDNA sample. The amplified product was purified using the Agencourt AMPure XP PCR purification system (Part No. A63881; Beckman Coulter Genomics, Danvers, Mass.), and the concentration quantified using the Nanodrop 2000 (Thermo Scientific, Wilmington, Del.). The purified amplification product was diluted 1:10 in water and 0.9 μl (371 pg) added to 40 μl of purified cfDNA sample to obtain a 10% spike. The enriched fetal and maternal cfDNA present in the purified cfDNA sample was used for preparing a sequencing library, and was sequenced as described in Example 4.</div>
</li> <li> <para-num num="[0247]"> </para-num> <div class="description-line" id="p-0263" num="0247">Table 13 provides the tag counts obtained for each of chromosomes 21, 18, 13, X and Y i.e. sequence tag density, and the tag counts obtained for the informative polymorphic sequences contained in the SNP reference genome. i.e. SNP tag density. The data show that sequencing information can be obtained from sequencing a single library constructed from a purified maternal cfDNA sample that has been enriched for sequences comprising SNPs to simultaneously determine the presence or absence of aneuploidy and the fetal fraction. The presence or absence of aneuploidy was determined using the number of tags mapped to chromosomes as described in pending U.S. patent application Ser. No. 12/958,352. In the example given, the data show that the fraction of fetal DNA in plasma sample AFR105 was quantifiable from the sequencing results of five informative SNPs and determined to be 3.84%. Sequence tag densities are provided for chromosomes 21, 13, 18, X and Y.</div>
</li> <li> <para-num num="[0248]"> </para-num> <div class="description-line" id="p-0264" num="0248">The example shows that the enrichment protocol provides the requisite tag counts for determining aneuploidy and fetal fraction from a single sequencing process.</div>
</li> <li> <div class="description-line" id="p-0265" num="0000">
<tables id="TABLE-US-00013" num="00013">
<patent-tables colsep="0" frame="none" pgwide="1" rowsep="0">
<table align="left" class="description-table" cols="1" colsep="0" rowsep="0" width="100%">
<thead>
<tr class="description-tr">
<td class="description-td" colspan="1" nameend="1" namest="1" rowsep="1">TABLE 13</td>
</tr>
<tr class="description-tr">
<td align="center" class="description-td" colspan="1" nameend="1" namest="1" rowsep="1"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">Determination of Fetal Fraction by Massively Parallel Sequencing:</td>
</tr>
<tr class="description-tr">
<td class="description-td">Enrichment of Fetal and Maternal Nucleic Acids for Polymorphic</td>
</tr>
<tr class="description-tr">
<td class="description-td">Nucleic Acids in a Purified cfDNA sample</td>
</tr>
<tr class="description-tr">
<td align="center" class="description-td" colspan="1" nameend="1" namest="1" rowsep="1"> </td>
</tr>
</thead>
<tbody><tr class="description-tr">
<td class="description-td">Aneuploidy</td>
</tr>
</tbody></table>
<table align="left" class="description-table" cols="6" colsep="0" rowsep="0" width="100%">
<tbody><tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">Chromosome</td>
<td class="description-td">Chromosome</td>
<td class="description-td">Chromosome</td>
<td class="description-td">Chromosome</td>
<td class="description-td">Chromosome</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">21</td>
<td class="description-td">18</td>
<td class="description-td">13</td>
<td class="description-td">X</td>
<td class="description-td">Y</td>
</tr>
<tr class="description-tr">
<td align="center" class="description-td" colspan="6" nameend="6" namest="1" rowsep="1"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">Sequence Tag</td>
<td class="description-td">178763</td>
<td class="description-td">359529</td>
<td class="description-td">388204</td>
<td class="description-td">572330</td>
<td class="description-td">2219</td>
</tr>
<tr class="description-tr">
<td class="description-td">Density</td>
</tr>
<tr class="description-tr">
<td class="description-td">Karyotype</td>
<td class="description-td">Unaffected</td>
<td class="description-td">Unaffected</td>
<td class="description-td">Unaffected</td>
<td class="description-td">Unaffected</td>
<td class="description-td">Unaffected</td>
</tr>
<tr class="description-tr">
<td align="center" class="description-td" colspan="6" nameend="6" namest="1" rowsep="1"> </td>
</tr>
</tbody></table>
<table align="left" class="description-table" cols="1" colsep="0" rowsep="0" width="100%">
<tbody><tr class="description-tr">
<td class="description-td">Fetal Fraction</td>
</tr>
</tbody></table>
<table align="left" class="description-table" cols="3" colsep="0" rowsep="0" width="100%">
<tbody><tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">SNP TAG</td>
<td class="description-td">FETAL FRACTION</td>
</tr>
<tr class="description-tr">
<td class="description-td">SNP</td>
<td class="description-td">DENSITY</td>
<td class="description-td">(%)</td>
</tr>
<tr class="description-tr">
<td align="center" class="description-td" colspan="3" nameend="3" namest="1" rowsep="1"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">rs10773760.1|Chr.12|length = 128|allele = A</td>
<td class="description-td">18903</td>
<td class="description-td">2.81</td>
</tr>
<tr class="description-tr">
<td class="description-td">rs10773760.2|Chr.12|length = 128|allele = G</td>
<td class="description-td">532</td>
</tr>
<tr class="description-tr">
<td class="description-td">rs1109037.1|Chr.2|length = 126|allele = A</td>
<td class="description-td">347</td>
<td class="description-td">5.43</td>
</tr>
<tr class="description-tr">
<td class="description-td">rs1109037.2|Chr.2|length = 126|allele = G</td>
<td class="description-td">6394</td>
</tr>
<tr class="description-tr">
<td class="description-td">rs2567608.1|Chr.20|length = 110|allele = A</td>
<td class="description-td">94503</td>
<td class="description-td">1.74</td>
</tr>
<tr class="description-tr">
<td class="description-td">rs2567608.2|Chr.20|length = 110|allele = G</td>
<td class="description-td">1649</td>
</tr>
<tr class="description-tr">
<td class="description-td">rs7041158.1|Chr.9|length = 117|allele = C</td>
<td class="description-td">107</td>
<td class="description-td">5.61</td>
</tr>
<tr class="description-tr">
<td class="description-td">rs7041158.2|Chr.9|length = 117|allele = T</td>
<td class="description-td">6</td>
</tr>
<tr class="description-tr">
<td class="description-td">rs8078417.1|Chr.17|length = 110|allele = C</td>
<td class="description-td">162668</td>
<td class="description-td">3.61</td>
</tr>
<tr class="description-tr">
<td class="description-td">rs8078417.2|Chr.17|length = 110|allele = T</td>
<td class="description-td">5877</td>
</tr>
<tr class="description-tr">
<td align="center" class="description-td" colspan="3" nameend="3" namest="1" rowsep="1"> </td>
</tr>
<tr class="description-tr">
<td align="left" class="description-td" colspan="3" id="FOO-00002" nameend="3" namest="1">Fetal Fraction (Mean ± S.D.) = 3.8 ± 1.7</td>
</tr>
</tbody></table>
</patent-tables>
</tables>
</div>
</li> <heading id="h-0026">Example 9</heading>
<heading id="h-0027">Identification of SNPs by Massively Parallel Sequencing: Enrichment of Fetal and Maternal Nucleic Acids for Polymorphic Nucleic Acids in a Plasma Sample and Determination of Fetal Fraction</heading>
<li> <para-num num="[0249]"> </para-num> <div class="description-line" id="p-0266" num="0249">To enrich the fetal and maternal cfDNA contained in an original plasma sample derived from a pregnant woman, a portion the original plasma sample was used for amplifying polymorphic target nucleic acid sequences each comprising one SNP chosen from the panel of SNPs given in Table 9, and a portion of the amplified product was combined with the remaining original plasma sample.</div>
</li> <li> <para-num num="[0250]"> </para-num> <div class="description-line" id="p-0267" num="0250">The method corresponds to workflow 1 diagrammed in <figref idrefs="DRAWINGS">FIG. 5</figref>. cfDNA contained in 15 μl of cell-free plasma was amplified in a reaction volume of 50 μl containing 9 ul of a 1 μM mixture of primers (15 plexTable 9), 1 μl of Phusion blood DNA polymerase, 25 ul of the 2× Phusion blood PCR buffer containing deoxynucleotide triphosphates (dNTPs: dATP, dCTP, dGTP and dTTP). Thermal cycling was performed with the Gene Amp9700 (Applied Biosystems) using the following cycling conditions: incubating at 95° C. for 3 minutes, followed by 35 cycles at 95° C. for 20 seconds, 55° C. for 30 s, and 70° C. for 1 minute, which was followed by a final incubation at 68° C. for 5 minutes. A final hold at 4° C. was added until the samples were removed for combining with the unamplified portion of the cell-free plasma. The amplified product was diluted 1:2 with water and analyzed using the Bioanalyzer. An additional 3 μl of amplified product was diluted with 11.85 μl of water to obtain a final concentration of 2 ng/μl. 2.2 μl of the diluted amplified product was combined with the remaining plasma sample. The enriched fetal and maternal cfDNA present in the plasma sample was purified as described in Example 4, and used for preparing a sequencing library. Sequencing and analysis of the sequencing data was performed as described in Example 4.</div>
</li> <li> <para-num num="[0251]"> </para-num> <div class="description-line" id="p-0268" num="0251">The results are given in Table 14. In the example given, the data show that the fraction of fetal DNA in plasma sample SAC2517 was quantifiable from the sequencing results of one informative SNP and determined to be 9.5%. In the example given, sample SAC2517 was shown by karyotyping to be unaffected for aneuploidies of chromosomes 21, 13, 18, X and Y. Sequence tag densities are provided for chromosomes 21, 13, 18, X and Y. The presence or absence of aneuploidy was determined using tag counts as described in U.S. Provisional Applications 61/407,017 and 61/455,849, which are herein incorporated by reference in their entirety.</div>
</li> <li> <para-num num="[0252]"> </para-num> <div class="description-line" id="p-0269" num="0252">The example demonstrates that enriching the mixture of fetal and maternal cfDNA present in a plasma sample for nucleic acid sequences that comprise at least one informative SNP can be used to provide the requisite sequence and SNP tag counts for determining aneuploidy and fetal fraction from a single sequencing process by massively parallel sequencing a library prepared from cfDNA contained in a plasma sample that is enriched for polymorphic nucleic acids.</div>
</li> <li> <div class="description-line" id="p-0270" num="0000">
<tables id="TABLE-US-00014" num="00014">
<patent-tables colsep="0" frame="none" pgwide="1" rowsep="0">
<table align="left" class="description-table" cols="1" colsep="0" rowsep="0" width="100%">
<thead>
<tr class="description-tr">
<td class="description-td" colspan="1" nameend="1" namest="1" rowsep="1">TABLE 14</td>
</tr>
<tr class="description-tr">
<td align="center" class="description-td" colspan="1" nameend="1" namest="1" rowsep="1"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">Determination of Fetal Fraction by Massively Parallel Sequencing:</td>
</tr>
<tr class="description-tr">
<td class="description-td">Enrichment of Fetal and Maternal Nucleic Acids for Polymorphic</td>
</tr>
<tr class="description-tr">
<td class="description-td">Nucleic Acids Comprising a SNP in a Plasma Sample</td>
</tr>
<tr class="description-tr">
<td align="center" class="description-td" colspan="1" nameend="1" namest="1" rowsep="1"> </td>
</tr>
</thead>
<tbody><tr class="description-tr">
<td class="description-td">Aneuploidy</td>
</tr>
</tbody></table>
<table align="left" class="description-table" cols="6" colsep="0" rowsep="0" width="100%">
<tbody><tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">Chromosome</td>
<td class="description-td">Chromosome</td>
<td class="description-td">Chromosome</td>
<td class="description-td">Chromosome</td>
<td class="description-td">Chromosome</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">21</td>
<td class="description-td">18</td>
<td class="description-td">13</td>
<td class="description-td">X</td>
<td class="description-td">Y</td>
</tr>
<tr class="description-tr">
<td align="center" class="description-td" colspan="6" nameend="6" namest="1" rowsep="1"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">Sequence Tag</td>
<td class="description-td">183851</td>
<td class="description-td">400582</td>
<td class="description-td">470526</td>
<td class="description-td">714055</td>
<td class="description-td">2449</td>
</tr>
<tr class="description-tr">
<td class="description-td">Density</td>
</tr>
<tr class="description-tr">
<td class="description-td">Karyotype</td>
<td class="description-td">Unaffected</td>
<td class="description-td">Unaffected</td>
<td class="description-td">Unaffected</td>
<td class="description-td">Unaffected</td>
<td class="description-td">Unaffected</td>
</tr>
<tr class="description-tr">
<td align="center" class="description-td" colspan="6" nameend="6" namest="1" rowsep="1"> </td>
</tr>
</tbody></table>
<table align="left" class="description-table" cols="1" colsep="0" rowsep="0" width="100%">
<tbody><tr class="description-tr">
<td class="description-td">Fetal Fraction</td>
</tr>
</tbody></table>
<table align="left" class="description-table" cols="3" colsep="0" rowsep="0" width="100%">
<tbody><tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">FETAL FRACTION</td>
</tr>
<tr class="description-tr">
<td class="description-td">SNP</td>
<td class="description-td">TAG COUNTS</td>
<td class="description-td">(%)</td>
</tr>
<tr class="description-tr">
<td align="center" class="description-td" colspan="3" nameend="3" namest="1" rowsep="1"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">rs10773760.1|Chr.12|length = 128|allele = A</td>
<td class="description-td">8536</td>
<td class="description-td">9.5</td>
</tr>
<tr class="description-tr">
<td class="description-td">rs10773760.2|Chr.12|length = 128|allele = G</td>
<td class="description-td">89924</td>
</tr>
<tr class="description-tr">
<td align="center" class="description-td" colspan="3" nameend="3" namest="1" rowsep="1"> </td>
</tr>
</tbody></table>
</patent-tables>
</tables>
</div>
</li> <li> <para-num num="[0253]"> </para-num> <div class="description-line" id="p-0271" num="0253">To determine the fraction of fetal cfDNA in a maternal sample, target polymorphic nucleic acid sequences each comprising a pair of tandem SNPs are amplified and used for preparing a target library for sequencing in a massively parallel fashion. Pairs of tandem SNPs can be selected from rs7277033-rs2110153; rs2822654-rs1882882; rs368657-rs376635; rs2822731-rs2822732; rs1475881-rs7275487; rs1735976-rs2827016; rs447340-rs2824097; rs418989-rs13047336; rs987980-rs987981; rs4143392-rs4143391; rs1691324-rs13050434; rs11909758-rs9980111; rs2826842-rs232414; rs1980969-rs1980970; rs9978999-rs9979175; rs1034346-rs12481852; rs7509629-rs2828358; rs4817013-rs7277036; rs9981121-rs2829696; rs455921-rs2898102; rs2898102-rs458848; rs961301-rs2830208; rs2174536-rs458076; rs11088023-rs11088024; rs1011734-rs1011733; rs2831244-rs9789838; rs8132769-rs2831440; rs8134080-rs2831524; rs4817219-rs4817220; rs2250911-rs2250997; rs2831899-rs2831900; rs2831902-rs2831903; rs11088086-rs2251447; rs2832040-rs11088088; rs2832141-rs2246777; rs2832959-rs9980934; rs2833734-rs2833735; rs933121-rs933122; rs2834140-rs12626953; rs2834485-rs3453; rs9974986-rs2834703; rs2776266-rs2835001; rs1984014-rs1984015; rs7281674-rs2835316; rs13047304-rs13047322; rs2835545-rs4816551; rs2835735-rs2835736; rs13047608-rs2835826; rs2836550-rs2212596; rs2836660-rs2836661; rs465612-rs8131220; rs9980072-rs8130031; rs418359-rs2836926; rs7278447-rs7278858; rs385787-rs367001; rs367001-rs386095; rs2837296-rs2837297; and rs2837381-rs4816672. The primers used for amplifying the target sequences comprising the tandem SNPs are designed to encompass both SNP sites. For example, the forward primer is designed to encompass the first SNP, and the reverse primer is designed to encompass the second of the tandem SNP pair i.e. each of the SNP sites in the tandem pair is encompassed within the 36 bp generated by the sequencing method. Paired-end sequencing can be used to identify all sequences encompassing the tandem SNP sites. Exemplary sets of primers that are used to amplify the tandem SNPs disclosed herein are rs7277033-rs2110153_F: TCCTGGAAACAAAAGTATT (SEQ ID NO:197) and rs7277033-rs2110153_R: AACCTTACAACAAAGCTAGAA (SEQ ID NO:198), set rs2822654-rs1882882_F: ACTAAGCCTTGGGGATCCAG (SEQ ID NO:199) and rs2822654-rs1882882_R: TGCTGTGGAAATACTAAAAGG (SEQ ID NO:200), set rs368657-rs376635_F:CTCCAGAGGTAATCCTGTGA (SEQ ID NO:201) and rs368657-rs376635_R:TGGTGTGAGATGGTATCTAGG (SEQ ID NO:202), rs2822731-rs2822732_F:GTATAATCCATGAATCTTGTTT (SEQ ID NO:203) and rs2822731-rs2822732_R:TTCAAATTGTATATAAGAGAGT (SEQ ID NO:204), rs1475881-rs7275487_F:GCAGGAAAGTTATTTTTAAT (SEQ ID NO:205) and rs1475881-rs7275487_R:TGCTTGAGAAAGCTAACACTT (SEQ ID NO:206), rs1735976-rs2827016F:CAGTGTTTGGAAATTGTCTG (SEQ ID NO:207) and rs1735976-rs2827016_R:GGCACTGGGAGATTATTGTA (SEQ ID NO:208), rs447349-rs2824097_F:TCCTGTTGTTAAGTACACAT (SEQ ID NO:209) and rs447349-rs2824097_R:GGGCCGTAATTACTTTTG (SEQ ID NO:210), rs418989-rs13047336_F:ACTCAGTAGGCACTTTGTGTC (SEQ ID NO:211) and rs418989-rs13047336_R:TCTTCCACCACACCAATC (SEQ ID NO:212), rs987980-rs987981_F:TGGCTTTTCAAAGGTAAAA (SEQ ID NO:213) and rs987980-rs987981_R: GCAACGTTAACATCTGAATTT (SEQ ID NO:214), rs4143392-rs4143391_F: rs4143392-rs4143391 (SEQ ID NO:215) and rs4143392-rs4143391_R:ATTTTATATGTCATGATCTAAG (SEQ ID NO:216), rs1691324-rs13050434_F: AGAGATTACAGGTGTGAGC (SEQ ID NO:217) and rs1691324-rs13050434_R: ATGATCCTCAACTGCCTCT (SEQ ID NO:218), rs11909758-rs9980111_F: TGAAACTCAAAAGAGAAAAG (SEQ ID NO:219) and rs11909758-rs9980111_R: ACAGATTTCTACTTAAAATT (SEQ ID NO:220), rs2826842-rs232414_F: TGAAACTCAAAAGAGAAAAG (SEQ ID NO:221) and rs2826842-rs232414_R: ACAGATTTCTACTTAAAATT (SEQ ID NO:22), rs2826842-rs232414_F: GCAAAGGGGTACTCTATGTA (SEQ ID NO:223) and rs2826842-rs232414_R: TATCGGGTCATCTTGTTAAA (SEQ ID NO:224), rs1980969-rs1980970_F: TCTAACAAAGCTCTGTCCAAAA (SEQ ID NO:225) and rs1980969-rs1980970_R: CCACACTGAATAACTGGAACA (SEQ ID NO:226), rs9978999-rs9979175_F: GCAAGCAAGCTCTCTACCTTC (SEQ ID NO:227) and rs9978999-rs9979175_R: TGTTCTTCCAAAATTCACATGC (SEQ ID NO:228), rs1034346-rs12481852_F: ATTTCACTATTCCTTCATTTT (SEQ ID NO:229) and rs1034346-rs 12481852_R: TAATTGTTGCACACTAAATTAC (SEQ ID NO:230), rs4817013-rs7277036_F: AAAAAGCCACAGAAATCAGTC (SEQ ID NO:231) and rs4817013-rs7277036_R: TTCTTATATCTCACTGGGCATT (SEQ ID NO:232), rs9981121-rs2829696_F: GGATGGTAGAAGAGAAGAAAGG (SEQ ID NO:233) and rs9981121-rs2829696_R: GGATGGTAGAAGAGAAGAAAGG (SEQ ID NO:234), rs455921-rs2898102_F: TGCAAAGATGCAGAACCAAC (SEQ ID NO:235) and rs455921-rs2898102_R: TTTTGTTCCTTGTCCTGGCTGA (SEQ ID NO:236), rs2898102-rs458848_F: TGCAAAGATGCAGAACCAAC (SEQ ID NO:237) and rs2898102-rs458848_R: GCCTCCAGCTCTATCCAAGTT (SEQ ID NO:238), rs961301-rs2830208_F: CCTTAATATCTTCCCATGTCCA (SEQ ID NO:239) and rs961301-rs2830208_R: ATTGTTAGTGCCTCTTCTGCTT (SEQ ID NO:240), rs2174536-rs458076_F: GAGAAGTGAGGTCAGCAGCT (SEQ ID NO:241) and rs2174536-rs458076_R: TTTCTAAATTTCCATTGAACAG (SEQ ID NO:242), rs11088023-rs11088024_F: GAAATTGGCAATCTGATTCT (SEQ ID NO:243) and rs11088023-rs11088024_R: CAACTTGTCCTTTATTGATGT (SEQ ID NO:244), rs1011734-rs1011733_F: CTATGTTGATAAAACATTGAAA (SEQ ID NO:245) and rs1011734-rs1011733_R: GCCTGTCTGGAATATAGTTT (SEQ ID NO:246), rs2831244-rs9789838_F: CAGGGCATATAATCTAAGCTGT (SEQ ID NO:247) and rs2831244-rs9789838_R: CAATGACTCTGAGTTGAGCAC (SEQ ID NO:248), rs8132769-rs2831440_F: ACTCTCTCCCTCCCCTCT (SEQ ID NO:249) and rs8132769-rs2831440_R: TATGGCCCCAAAACTATTCT (SEQ ID NO:250), rs8134080-rs2831524_F: ACAAGTACTGGGCAGATTGA (SEQ ID NO:251) and rs8134080-rs2831524_R: GCCAGGTTTAGCTTTCAAGT (SEQ ID NO:252), rs4817219-rs4817220_F: TTTTATATCAGGAGAAACACTG (SEQ ID NO:253) and rs4817219-rs4817220_R: CCAGAATTTTGGAGGTTTAAT (SEQ ID NO:254), rs2250911-rs2250997_F: TGTCATTCCTCCTTTATCTCCA (SEQ ID NO:255) and rs2250911-rs2250997_R: TTCTTTTGCCTCTCCCAAAG (SEQ ID NO:256), rs2831899-rs2831900_F: ACCCTGGCACAGTGTTGACT (SEQ ID NO:257) and rs2831899-rs2831900_R: TGGGCCTGAGTTGAGAAGAT (SEQ ID NO:258), rs2831902-rs2831903_F: AATTTGTAAGTATGTGCAACG (SEQ ID NO:259) and rs2831902-rs2831903_R: TTTTTCCCATTTCCAACTCT (SEQ ID NO:260), rs11088086-rs2251447_F: AAAAGATGAGACAGGCAGGT (SEQ ID NO:261) and rs11088086-rs2251447_R: ACCCCTGTGAATCTCAAAAT (SEQ ID NO:262), rs2832040-rs11088088_F: GCACTTGCTTCTATTGTTTGT (SEQ ID NO:263) and rs2832040-rs11088088_R: CCCTTCCTCTCTTCCATTCT (SEQ ID NO:264), rs2832141-rs2246777_F: AGCACTGCAGGTA (SEQ ID NO:265) and rs2832141-rs2246777_R: ACAGATACCAAAGAACTGCAA (SEQ ID NO:266), rs2832959-rs9980934_F: TGGACACCTTTCAACTTAGA (SEQ ID NO:267) and rs2832959-rs9980934_R: GAACAGTAATGTTGAACTTTTT (SEQ ID NO:268), rs2833734-rs2833735_F: TCTTGCAAAAAGCTTAGCACA (SEQ ID NO:269) and rs2833734-rs2833735_R: AAAAAGATCTCAAAGGGTCCA (SEQ ID NO:270), rs933121-rs933122_F: GCTTTTGCTGAACATCAAGT (SEQ ID NO:271) and rs933121-rs933122_R: CCTTCCAGCAGCATAGTCT (SEQ ID NO:272), rs2834140-rs12626953_F: AAATCCAGGATGTGCAGT (SEQ ID NO:273) and rs2834140-rs12626953_R: ATGATGAGGTCAGTGGTGT (SEQ ID NO:274), rs2834485-rs3453_F: CATCACAGATCATAGTAAATGG (SEQ ID NO:275) and rs2834485-rs3453_R: AATTATTATTTTGCAGGCAAT (SEQ ID NO:276), rs9974986-rs2834703_F: CATGAGGCAAACACCTTTCC (SEQ ID NO:277) and rs9974986-rs2834703_R: GCTGGACTCAGGATAAAGAACA (SEQ ID NO:278), rs2776266-rs2835001_F: TGGAAGCCTGAGCTGACTAA (SEQ ID NO:279) and rs2776266-rs2835001_R:CCTTCTTTTCCCCCAGAATC (SEQ ID NO:280), rs1984014-rs1984015_F:TAGGAGAACAGAAGATCAGAG (SEQ ID NO:281) and rs1984014-rs1984015_R:AAAGACTATTGCTAAATGCTTG (SEQ ID NO:282), rs7281674-rs2835316_F: TAAGCGTAGGGCTGTGTGTG (SEQ ID NO:283) and rs7281674-rs2835316_R: GGACGGATAGACTCCAGAAGG (SEQ ID NO:284), rs13047304-rs13047322_F: GAATGACCTTGGCACTTTTATCA (SEQ ID NO:285) and rs13047304-rs13047322_R: AAGGATAGAGATATACAGATGAATGGA (SEQ ID NO:286), rs2835735-rs2835736_F: CATGCACCGCGCAAATAC (SEQ ID NO:287) and rs2835735-rs2835736_R: ATGCCTCACCCACAAACAC (SEQ ID NO:288), rs13047608-rs2835826_F: TCCAAGCCCTTCTCACTCAC (SEQ ID NO:289) and rs13047608-rs2835826_R: CTGGGACGGTGACATTTTCT (SEQ ID NO:290), rs2836550-rs2212596_F: CCCAGGAAGAGTGGAAAGATT (SEQ ID NO:291) and rs2836550-rs2212596_R: TTAGCTTGCATGTACCTGTGT (SEQ ID NO:292), rs2836660-rs2836661_F: AGCTAGATGGGGTGAATTTT (SEQ ID NO:293) and _R: TGGGCTGAGGGGAGATTC (SEQ ID NO:294), rs465612-rs8131220_F: ATCAAGCTAATTAATGTTATCT (SEQ ID NO:295) and rs465612-rs8131220_R: AATGAATAAGGTCCTCAGAG (SEQ ID NO:296), rs9980072-rs8130031_F:TTTAATCTGATCATTGCCCTA (SEQ ID NO:297) and rs9980072-rs8130031_R: AGCTGTGGGTGACCTTGA (SEQ ID NO:298), rs418359-rs2836926_F: TGTCCCACCATTGTGTATTA (SEQ ID NO:299) and rs418359-rs2836926_R: TCAGACTTGAAGTCCAGGAT (SEQ ID NO:300), rs7278447-rs7278858_F: GCTTCAGGGGTGTTAGTTTT (SEQ ID NO:301) and rs7278447-rs7278858_R: CTTTGTGAAAAGTCGTCCAG (SEQ ID NO:302), rs385787-rs367001_F:CCATCATGGAAAGCATGG (SEQ ID NO:303) and rs385787-rs367001_R: TCATCTCCATGACTGCACTA (SEQ ID NO:304), rs367001-rs386095_F: GAGATGACGGAGTAGCTCAT (SEQ ID NO:305) and rs367001-rs386095_R: CCCAGCTGCACTGTCTAC (SEQ ID NO:306), rs2837296-rs2837297_F: TCTTGTTCCAATCACAGGAC (SEQ ID NO:307) and rs2837296-rs2837297_R: ATGCTGTTAGCTGAAGCTCT (SEQ ID NO:308), and rs2837381-rs4816672_F: TGAAAGCTCCTAAAGCAGAG (SEQ ID NO:309) and rs2837381-rs4816672_R:TTGAAGAGATGTGCTATCAT (SEQ ID NO:310). Polynucleotide sequences e.g. GC clamp sequences, can be included to ensure specific hybridization of AT-rich primers (Ghanta et al., PLOS ONE 5(10): doil0.1371/journal.pone.0013184 [2010], available on the world wide web at plosone.org). An example of a GC clamp sequence that can be included either 5′ of the forward primer or 3′ of the reverse primer is GCCGCCTGCAGCCCGCGCCCCCCGTGCCCCCGCCCCGCCGCCGGCCCGGGCGCC (SEQ ID NO:311). Polymorphic sequences can be used alone or in combination with unamplified cfDNA to determine either fetal fraction or the presence or absence of aneuploidy and fetal fraction in a maternal sample as described for polymorphic SNP sequences. Sample preparation and enrichment of cfDNA sequencing library, a purified cfDNA sample, and a plasma sample is performed according to the method described in Examples 7, 8, and 9, respectively.</div>
</li> <li> <para-num num="[0254]"> </para-num> <div class="description-line" id="p-0272" num="0254">All sequencing libraries are prepared as described in Example 4b., and sequencing is performed as described in Example 4c. Analysis of the sequencing data for the determination of fetal aneuploidy is performed as described in pending U.S. patent application Ser. No. 12/958,352, filed on Dec. 1, 2010. Concomitant to the analysis for determining aneuploidy, the sequencing data is analyzed to determine the fetal fraction as follows. Following the transfer of the image and base call files to the Unix server running the Illumina “Genome Analyzer Pipeline” software version 1.51 as described in Example 4d., the 36 bp reads are aligned to a ‘tandem SNP genome’ using the BOWTIE program. The tandem SNP genome is identified as the grouping of the DNA sequences that encompass the alleles of the 58 tandem SNP pairs disclosed above. Only reads that mapped uniquely to the tandem SNP genome are used for the analysis of fetal fraction. Reads that match perfectly to the tandem SNP genome are counted as tags and filtered. Of the remaining reads, only reads having one or two mismatches are counted as tags and included in the analysis. Tags mapped to each of the tandem SNP alleles are counted, and the fetal fraction is determined essentially as described in Example 9 above but accounting for tags mapped to the two tandem SNP alleles x and y present on each of the amplified polymorphic target nucleic acid sequences that are amplified to enrich the samples i.e.</div>
</li> <li> <div class="description-line" id="p-0273" num="0000"> <br/>% fetal fraction allele<sub>x+y</sub>=((ΣFetal sequence tags for allele<sub>x+y</sub>)/(ΣMaternal sequence tags for allele<sub>x+y</sub>))×100
</div>
</li> <li> <para-num num="[0255]"> </para-num> <div class="description-line" id="p-0274" num="0255">Only informative tandem SNPs are used to determine the fetal fraction.</div>
</li> <li> <para-num num="[0256]"> </para-num> <div class="description-line" id="p-0275" num="0256">Optionally, the fraction of fetal nucleic acids in the mixture of fetal and maternal nucleic acids is calculated for each of the informative allele (allele<sub>x</sub>+y) as follows:</div>
</li> <li> <div class="description-line" id="p-0276" num="0000"> <br/>% fetal fraction allele<sub>x+y</sub>=((2×ΣFetal sequence tags for allele<sub>x+y</sub>)/(ΣMaternal sequence tags for allele<sub>x+y</sub>))×100,
</div>
</li> <li> <div class="description-line" id="p-0277" num="0000">to compensate for the presence of 2 sets of tandem fetal alleles, one being masked by the maternal background.</div>
</li> <li> <para-num num="[0257]"> </para-num> <div class="description-line" id="p-0278" num="0257">The percent fetal fraction is calculated for at least 1, at least 2, at least 3, at least 4, at least 5, at least 6, at least 7, at least 8, at least 9, at least 10, at least 11, at least 12, at least 13, at least 14, at least 15, at least 16, at least 17, at least 18, at least 19, at least 20, or more informative sets of tandem alleles. In one embodiment, the fetal fraction is the average fetal fraction determined for at least 3 informative sets of tandem alleles.</div>
</li> <heading id="h-0028">Example 11</heading>
<heading id="h-0029">Identification of Fetal STRs by Massively Parallel Sequencing: Determination of Fetal Fraction</heading>
<li> <para-num num="[0258]"> </para-num> <div class="description-line" id="p-0279" num="0258">To determine the fraction of fetal cfDNA in a maternal sample, target polymorphic nucleic acid sequences each comprising an STR are amplified and used for preparing a target library for sequencing in a massively parallel fashion.</div>
</li> <li> <para-num num="[0259]"> </para-num> <div class="description-line" id="p-0280" num="0259">Peripheral blood samples are obtained from pregnant subjects, and cfDNA is purified from the plasma fraction as described in Example 4. STRs that are amplified are chosen from the codis and non-codis STRs disclosed in Table 14, and amplification of the polymorphic STR sequences is obtained using the corresponding sets of primers provided. For example, the STRs listed in Table 15 are amplified using the corresponding primers (SEQ ID NOs: 113-197), and the amplified product is used to generate a target sequencing library. The STR target sequencing library is prepared as described for the preparation of the SNP target library as described in Example 8. STRs CSF1PO, D13S317, D16S539, D18S51, D21S11, D2S1338, D7S820, and FGA have been analyzed previously for determining fetal fraction, and are disclosed in U.S. Provisional applications 61/296,358 and 61/360,837.</div>
</li> <li> <div class="description-line" id="p-0281" num="0000">
<tables id="TABLE-US-00015" num="00015">
<patent-tables colsep="0" frame="none" pgwide="1" rowsep="0" tabstyle="monospace">
<table align="left" class="description-table" cols="1" colsep="0" rowsep="0" width="100%">
<thead>
<tr class="description-tr">
<td class="description-td" colspan="1" nameend="1" namest="1" rowsep="1">TABLE 15</td>
</tr>
</thead>
<tbody><tr class="description-tr">
<td align="center" class="description-td" colspan="1" nameend="1" namest="1" rowsep="1"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">CODIS and NON-CODIS miniSTRs</td>
</tr>
</tbody></table>
<table align="left" class="description-table" cols="5" colsep="0" rowsep="0" width="100%">
<tbody><tr class="description-tr">
<td class="description-td">STR Locus</td>
<td class="description-td">Chromo-</td>
<td class="description-td">Size</td>
<td class="description-td"> </td>
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">(Marker</td>
<td class="description-td">some</td>
<td class="description-td">Range</td>
<td class="description-td">GenBank</td>
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">Name)</td>
<td class="description-td">Location</td>
<td class="description-td">(bp)</td>
<td class="description-td">Accession</td>
<td class="description-td">Primer Sequences (Forward/Reverse)</td>
</tr>
<tr class="description-tr">
<td align="center" class="description-td" colspan="5" nameend="5" namest="1" rowsep="1"> </td>
</tr>
</tbody></table>
<table align="left" class="description-table" cols="1" colsep="0" rowsep="0" width="100%">
<tbody><tr class="description-tr">
<td class="description-td">Codis miniSTR loci*</td>
</tr>
</tbody></table>
<table align="left" class="description-table" cols="5" colsep="0" rowsep="0" width="100%">
<tbody><tr class="description-tr">
<td class="description-td">CSF1P0</td>
<td class="description-td">5q33.1</td>
<td class="description-td"> 89-129</td>
<td class="description-td">X14720</td>
<td class="description-td">ACAGTAACTGCCTTCATAGATAG</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">(CSF1PO_F; SEQ ID NO: 113)</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">GTGTCAGACCCTGTTCTAAGTA</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">(CSF1PO_R; SEQ ID NO: 114)</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">FGA</td>
<td class="description-td">4q31.3</td>
<td class="description-td">125-281</td>
<td class="description-td">M64982</td>
<td class="description-td">AAATAAAATTAGGCATATTTACAAGC</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">(FGA_F; SEQ ID NO: 115)</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">GCTGAGTGATTTGTCTGTAATTG</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">(FGA_R; SEQ ID NO: 116)</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">TH01</td>
<td class="description-td">11p15.5</td>
<td class="description-td"> 51-98</td>
<td class="description-td">D00269</td>
<td class="description-td">CCTGTTCCTCCCTTATTTCCC</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">(TH01_F; SEQ ID NO: 117)</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">GGGAACACAGACTCCATGGTG</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">(TH01_R; SEQ ID NO: 118)</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">TPOX</td>
<td class="description-td">2p25.3</td>
<td class="description-td"> 65-101</td>
<td class="description-td">M68651</td>
<td class="description-td">CTTAGGGAACCCTCACTGAATG</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">(TPOX_F; SEQ ID NO: 119)</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">GTCCTTGTCAGCGTTTATTTGC</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">(TPOX_R; SEQ ID NO: 120)</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">vWA</td>
<td class="description-td">12p13.31</td>
<td class="description-td"> 88-148</td>
<td class="description-td">M25858</td>
<td class="description-td">AATAATCAGTATGTGACTTGGATTGA</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">(vWA_F; SEQ ID NO: 121)</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">ATAGGATGGATGGATAGATGGA</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">(vWA_R; SEQ ID NO: 122)</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">D3S1358</td>
<td class="description-td">3p21.31</td>
<td class="description-td"> 72-120</td>
<td class="description-td">NT_005997</td>
<td class="description-td">CAGAGCAAGACCCTGTCTCAT</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">(D3S1358_F; SEQ ID NO: 123)</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">TCAACAGAGGCTTGCATGTAT</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">(D3S1358_R; SEQ ID NO: 124)</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">D5S818</td>
<td class="description-td">5q23.2</td>
<td class="description-td"> 81-117</td>
<td class="description-td">AC008512</td>
<td class="description-td">GGGTGATTTTCCTCTTTGGT</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">(D5S818_F; SEQ ID NO: 125)</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">AACATTTGTATCTTTATCTGTATCCT</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">TATTTAT</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">(D5S818_R; SEQ ID NO: 126)</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">D7S820</td>
<td class="description-td">7q21.11</td>
<td class="description-td">136-176</td>
<td class="description-td">AC004848</td>
<td class="description-td">GAACACTTGTCATAGTTTAGAACGAAC</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">(D7S820_F; SEQ ID NO: 127)</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">TCATTGACAGAATTGCACCA</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">(D7S820_R; SEQ ID NO: 128)</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">D8S1179</td>
<td class="description-td">8q24.13</td>
<td class="description-td"> 86-134</td>
<td class="description-td">AF216671</td>
<td class="description-td">TTTGTATTTCATGTGTACATTCGTATC</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">(D7S820_F; SEQ ID NO: 129)</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">ACCTATCCTGTAGATTATTTTCACTGTG</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">(D7S820_R; SEQ ID NO: 130)</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">D13S317</td>
<td class="description-td">13q31.1</td>
<td class="description-td"> 88-132</td>
<td class="description-td">AL353628</td>
<td class="description-td">TCTGACCCATCTAACGCCTA</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">(D13S317_F; SEQ ID NO: 131)</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">CAGACAGAAAGATAGATAGATGATTGA</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">(D13S317_R; SEQ ID NO: 132)</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">D16S539</td>
<td class="description-td">16q24.1</td>
<td class="description-td"> 81-121</td>
<td class="description-td">AC024591</td>
<td class="description-td">ATACAGACAGACAGACAGGTG</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">(D16S539_F; SEQ ID NO: 133)</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">GCATGTATCTATCATCCATCTCT</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">(D16S539_R; SEQ ID NO: 134)</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">D18S51</td>
<td class="description-td">18q21.33</td>
<td class="description-td">113-193</td>
<td class="description-td">AP001534</td>
<td class="description-td">TGAGTGACAAATTGAGACCTT</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">(D18S51_F; SEQ ID NO: 135)</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">GTCTTACAATAACAGTTGCTACTATT</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">(D18S51_R; SEQ ID NO: 136)</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">D21S11</td>
<td class="description-td">21q21.1</td>
<td class="description-td">153-221</td>
<td class="description-td">AP000433</td>
<td class="description-td">ATTCCCCAAGTGAATTGC</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">(D21S11_F; SEQ ID NO: 137)</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">GGTAGATAGACTGGATAGATAGACGA</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">(D21S11_R; SEQ ID NO: 138)</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">D2S1338</td>
<td class="description-td">2q35</td>
<td class="description-td"> 90-142</td>
<td class="description-td">AC01036</td>
<td class="description-td">TGGAAACAGAAATGGCTTGG</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">(D2S1338_F; SEQ ID NO: 139)</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">GATTGCAGGAGGGAAGGAAG</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">(D2S1338_R; SEQ ID NO: 140)</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">Penta D</td>
<td class="description-td">21q22.3</td>
<td class="description-td"> 94-167</td>
<td class="description-td">AP001752</td>
<td class="description-td">GAGCAAGACACCATCTCAAGAA</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">(Penta D_F; SEQ ID NO: 141)</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">GAAATTTTACATTTATGTTTATGATT</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">CTCT(Penta D_R; SEQ ID NO: 142)</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">Penta E</td>
<td class="description-td">15q26.2</td>
<td class="description-td"> 80-175</td>
<td class="description-td">ACO27004</td>
<td class="description-td">GGCGACTGAGCAAGACTC</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">(Penta E_F; SEQ ID NO: 143)</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">GGTTATTAATTGAGAAAACTCCTTACA</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">(Penta E_R; SEQ ID NO: 144)</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
</tr>
</tbody></table>
<table align="left" class="description-table" cols="1" colsep="0" rowsep="0" width="100%">
<tbody><tr class="description-tr">
<td class="description-td">Non-Codis mmiSTR loci*</td>
</tr>
</tbody></table>
<table align="left" class="description-table" cols="5" colsep="0" rowsep="0" width="100%">
<tbody><tr class="description-tr">
<td class="description-td">D22S1045</td>
<td class="description-td">22q12.3</td>
<td class="description-td"> 82-115</td>
<td class="description-td">AL022314 (17)</td>
<td class="description-td">ATTTTCCCCGATGATAGTAGTCT</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">(D22S1045_F; SEQ ID NO: 145)</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">GCGAATGTATGATTGGCAATATTTTT</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">(D22S1045_R; SEQ ID NO: 146)</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">D20S1082</td>
<td class="description-td">20q13.2</td>
<td class="description-td"> 73-101</td>
<td class="description-td">AL158015</td>
<td class="description-td">ACATGTATCCCAGAACTTAAAGTAAAC</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">(D20S1082_F; SEQ ID NO: 147)</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">GCAGAAGGGAAAATTGAAGCTG</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">(D20S1082_R; SEQ ID NO: 148)</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">D20S482</td>
<td class="description-td">20p13</td>
<td class="description-td"> 85-126</td>
<td class="description-td">AL121781 (14)</td>
<td class="description-td">CAGAGACACCGAACCAATAAGA</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">(D20S482_F; SEQ ID NO: 149)</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">GCCACATGAATCAATTCCTATAATAAA</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">(D20S482_R; SEQ ID NO: 150)</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">D18S853</td>
<td class="description-td">18p11.31</td>
<td class="description-td"> 82-104</td>
<td class="description-td">AP005130 (11)</td>
<td class="description-td">GCACATGTACCCTAAAACTTAAAAT</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">(D18S853_F; SEQ ID NO: 151)</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">GTCAACCAAAACTCAACAAGTAGTAA</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">(D18S853_R; SEQ ID NO: 152)</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">D17S1301</td>
<td class="description-td">17q25.1</td>
<td class="description-td">114-139</td>
<td class="description-td">AC016888 (12)</td>
<td class="description-td">AAGATGAAATTGCCATGTAAAAATA</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">(D17S1301_F; SEQ ID NO: 153)</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">GTGTGTATAACAAAATTCCTATGATGG</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">(D17S1301_R; SEQ ID NO: 154)</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">D17S974</td>
<td class="description-td">17p13.1</td>
<td class="description-td">114-139</td>
<td class="description-td">AC034303 (10)</td>
<td class="description-td">GCACCCAAAACTGAATGTCATA</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">(D17S974_F; SEQ ID NO: 155)</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">GGTGAGAGTGAGACCCTGTC</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">(D17S974_R; SEQ ID NO: 156)</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">D14S1434</td>
<td class="description-td">14q32.13</td>
<td class="description-td"> 70-98</td>
<td class="description-td">AL121612 (13)</td>
<td class="description-td">TGTAATAACTCTACGACTGTCTGTCTG</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">(D14S1434_F; SEQ ID NO: 157)</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">GAATAGGAGGTGGATGGATGG</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">(D14S1434_R; SEQ ID NO: 158)</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">D12ATA63</td>
<td class="description-td">12q23.3</td>
<td class="description-td"> 76-106</td>
<td class="description-td">AC009771 (13)</td>
<td class="description-td">GAGCGAGACCCTGTCTCAAG</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">(D12ATA63_F; SEQ ID NO: 159)</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">GGAAAAGACATAGGATAGCAATTT</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">(D12ATA63_R; SEQ ID NO: 160)</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">D11S4463</td>
<td class="description-td">11q25</td>
<td class="description-td"> 88-116</td>
<td class="description-td">AP002806 (14)</td>
<td class="description-td">TCTGGATTGATCTGTCTGTCC</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">(D11S4463_F; SEQ ID NO: 161)</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">GAATTAAATACCATCTGAGCACTGAA</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">(D11S4463_R; SEQ ID NO: 162)</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">D10S1435</td>
<td class="description-td">10p15.3</td>
<td class="description-td"> 82-139</td>
<td class="description-td">AL354747 (11)</td>
<td class="description-td">TGTTATAATGCATTGAGTTTTATTCTG</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">(D10S1435_F; SEQ ID NO: 163)</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">GCCTGTCTCAAAAATAAAGAGATAGACA</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">(D10S1435_R; SEQ ID NO: 164)</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">D10S1248</td>
<td class="description-td">10q26.3</td>
<td class="description-td"> 79-123</td>
<td class="description-td">AL391869 (13)</td>
<td class="description-td">TTAATGAATTGAACAAATGAGTGAG</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">(D10S1248_F; SEQ ID NO: 165)</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">GCAACTCTGGTTGTATTGTCTTCAT</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">(D10S1248_R; SEQ ID NO: 166)</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">D9S2157</td>
<td class="description-td">9q34.2</td>
<td class="description-td"> 71-107</td>
<td class="description-td">AL162417 (10)</td>
<td class="description-td">CAAAGCGAGACTCTGTCTCAA</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">(D9S2157_F; SEQ ID NO: 167)</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">GAAAATGCTATCCTCTTTGGTATAAAT</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">(D9S2157_R; SEQ ID NO: 168)</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">D9S1122</td>
<td class="description-td">9q21.2</td>
<td class="description-td"> 93-125</td>
<td class="description-td">AL161789 (12)</td>
<td class="description-td">GGGTATTTCAAGATAACTGTAGATAGG</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">(D9S1122_F; SEQ ID NO: 168)</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">GCTTCTGAAAGCTTCTAGTTTACC</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">(D9S1122_R; SEQ ID NO: 170)</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">D8S1115</td>
<td class="description-td">8p11.21</td>
<td class="description-td"> 63-96</td>
<td class="description-td">AC090739 (9)</td>
<td class="description-td">TCCACATCCTCACCAACAC</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">(D8S1115_F; SEQ ID NO: 171)</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">GCCTAGGAAGGCTACTGTCAA</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">(D8S1115_R; SEQ ID NO: 172)</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">D6S1017</td>
<td class="description-td">6p21.1</td>
<td class="description-td"> 81-110</td>
<td class="description-td">AL035588 (10)</td>
<td class="description-td">CCACCCGTCCATTTAGGC</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">(D6S1017_F; SEQ ID NO: 173)</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">GTGAAAAAGTAGATATAATGGTTGGTG</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">(D6S1017_R; SEQ ID NO: 174)</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">D6S474</td>
<td class="description-td">6q21</td>
<td class="description-td">107-136</td>
<td class="description-td">AL357514 (17)</td>
<td class="description-td">GGTTTTCCAAGAGATAGACCAATTA</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">(D6S474_F; SEQ ID NO: 175)</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">GTCCTCTCATAAATCCCTACTCATATC</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">(D6S474_R; SEQ ID NO: 176)</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">D5S2500</td>
<td class="description-td">5q11.2</td>
<td class="description-td"> 85-126</td>
<td class="description-td">AC008791 (17)</td>
<td class="description-td">CTGTTGGTACATAATAGGTAGGTAGGT</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">(D5S2500_F; SEQ ID NO: 177)</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">GTCGTGGGCCCCATAAATC</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">(D5S2500_R; SEQ ID NO: 178)</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">D4S2408</td>
<td class="description-td">4p15.1</td>
<td class="description-td"> 85-109</td>
<td class="description-td">AC110763 (9)</td>
<td class="description-td">AAGGTACATAACAGTTCAATAGAAAGC</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">(D4S2408_F; SEQ ID NO: 179)</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">GTGAAATGACTGAAAAATAGTAACCA</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">(D4S2408_R; SEQ ID NO: 180)</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">D4S2364</td>
<td class="description-td">4q22.3</td>
<td class="description-td"> 67-83</td>
<td class="description-td">AC022317 (9)</td>
<td class="description-td">CTAGGAGATCATGTGGGTATGATT</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">(D4S2364U_F; SEQ ID NO: 181)</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">GCAGTGAATAAATGAACGAATGGA</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">(D4S2364_R; SEQ ID NO: 182)</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">D3S4529</td>
<td class="description-td">3p12.1</td>
<td class="description-td">111-139</td>
<td class="description-td">AC117452 (13)</td>
<td class="description-td">CCCAAAATTACTTGAGCCAAT</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">(D3S452_F; SEQ ID NO: 183)</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">GAGACAAAATGAAGAAACAGACAG</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">(D3S452_R; SEQ ID NO: 184)</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">D3S3053</td>
<td class="description-td">3q26.31</td>
<td class="description-td"> 84-108</td>
<td class="description-td">AC069259 (9)</td>
<td class="description-td">TCTTTGCTCTCATGAATAGATCAGT</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">(D3S3053_F; SEQ ID NO: 185)</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">GTTTGTGATAATGAACCCACTCAG</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">(D3S3053_R; SEQ ID NO: 186)</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">D2S1776</td>
<td class="description-td">2q24.3</td>
<td class="description-td">127-161</td>
<td class="description-td">AC009475 (11)</td>
<td class="description-td">TGAACACAGATGTTAAGTGTGTATATG</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">(D2S1776_F; SEQ ID NO: 187)</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">GTCTGAGGTGGACAGTTATGAAA</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">(D2S1776_R; SEQ ID NO: 188)</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">D2S441</td>
<td class="description-td">2p14</td>
<td class="description-td"> 78-110</td>
<td class="description-td">AC079112 (12)</td>
<td class="description-td">CTGTGGCTCATCTATGAAAACTT</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">(D2S441_F; SEQ ID NO: 189)</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">GAAGTGGCTGTGGTGTTATGAT</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">(D2S441_R; SEQ ID NO: 190)</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">D1S1677</td>
<td class="description-td">1q23.3</td>
<td class="description-td"> 81-117</td>
<td class="description-td">AL513307 (15)</td>
<td class="description-td">TTCTGTTGGTATAGAGCAGTGTTT</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">(D1S1677_F; SEQ ID NO: 191)</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">GTGACAGGAAGGACGGAATG</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">(D1S1677_R; SEQ ID NO: 192)</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">D1S1627</td>
<td class="description-td">1p21.1</td>
<td class="description-td"> 81-100</td>
<td class="description-td">AC093119 (13)</td>
<td class="description-td">CATGAGGTTTGCAAATACTATCTTAAC</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">(D1S1627_F; SEQ ID NO: 193)</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">GTTTTAATTTTCTCCAAATCTCCA</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">(D1S1627_R; SEQ ID NO: 194)</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">D1GATA113</td>
<td class="description-td">1p36.23</td>
<td class="description-td"> 81-105</td>
<td class="description-td">Z97987 (11)</td>
<td class="description-td">TCTTAGCCTAGATAGATACTTGCTTCC</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">(D1GATA113_F; SEQ ID NO: 195)</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">GTCAACCTTTGAGGCTATAGGAA</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">(D1GATA113_R; SEQ ID NO: 196)</td>
</tr>
<tr class="description-tr">
<td align="center" class="description-td" colspan="5" nameend="5" namest="1" rowsep="1"> </td>
</tr>
<tr class="description-tr">
<td align="left" class="description-td" colspan="5" id="FOO-00003" nameend="5" namest="1">*(Butler et al. J Forensic Sci 5:1054-1064; Hill et al., Poster #44-17th International Symposium on Human Identification-2006)</td>
</tr>
</tbody></table>
</patent-tables>
</tables>
</div>
</li> <li> <para-num num="[0260]"> </para-num> <div class="description-line" id="p-0282" num="0260">Sequencing of the library enriched for polymorphic STR sequences is performed using a NGS technology e.g. sequencing by synthesis. Sequence reads of lengths that encompass the STRs e.g. miniSTRs of at least 100 bp, to a reference STR genome consisting of the polymorphic sequences which were amplified in the sample. Informative STR alleles are identified by differences in the length of the repeats, and the number of STR sequence tags are counted, and used to determine the fetal fraction. Optionally, amplification of the polymorphic STR sequences is performed to enrich a plasma sample, a purified cfDNA sample or a cfDNA sequencing library sample, as described in Examples 5, 6, and 7, respectively.</div>
</li> <heading id="h-0030">Example 12</heading>
<heading id="h-0031">Identification of Fetal STRs by Capillary Electrophoresis: Determination of Fetal Fraction</heading>
<li> <para-num num="[0261]"> </para-num> <div class="description-line" id="p-0283" num="0261">To determine fetal fraction in maternal samples comprising fetal and maternal cfDNA, peripheral blood samples were collected from volunteer pregnant women carrying either male or female fetuses. Peripheral blood samples were obtained and processed to provide purified cfDNA.2.</div>
</li> <li> <para-num num="[0262]"> </para-num> <div class="description-line" id="p-0284" num="0262">Ten microliters of cfDNA samples were analyzed using the AmpFISTR® MiniFiler™ PCR amplification kit (Applied Biosystems, Foster City, Calif.) according to the manufacturer's instructions. Briefly, cfDNA contained in 10 μl was amplified in a reaction volume of 25 μl containing 5 μL fluorescently labeled primers (AmpFISTR® MiniFiler™ Primer Set), and the AmpFISTR® MiniFiler™ Master Mix, which includes AmpliTaq Gold® DNA polymerase and associated buffer, salt (1.5 mM MgCl2), and 200 μM deoxynucleotide triphosphates (dNTPs: dATP, dCTP, dGTP and dTTP). The fluorescently labeled primers are forward primers that are labeled with 6FAM™, VIC™, NED™, and PET™ dyes. Thermal cycling was performed with the Gene Amp9700 (Applied Biosystems) using the following cycling conditions: incubating at 95° C. for 10 minutes, followed by 30 cycles at 94° C. for 20 seconds, 59° C. for 2 minute, and 72° C. for 1 minute, which was followed by a final incubation at 60° C. for 45 minutes. A final hold at 4° C. was added until the samples were removed for analysis. The amplified product was prepared by diluting 1 ul of amplified product in 8.7 ul Hi-Di™ formamide (Applied Biosystems) and 0.3 μl GeneScan™-500 LIZ_internal size standard (Applied Biosystems), and analyzed with an ABI PRISM3130xl Genetic Analyzer (Applied Biosystems) using Data Collection HID_G5_POP4 (Applied Biosystems), and a 36-cm capillary array. All genotyping was performed with GeneMapper_ID v3.2 software (Applied Biosystems) using manufacturer provided allelic ladders and bins and panels.</div>
</li> <li> <para-num num="[0263]"> </para-num> <div class="description-line" id="p-0285" num="0263">All genotyping measurement were performed on the Applied Biosystems 3130xl Genetic Analyzer, using a ±0.5-nt “window” around the size obtained for each allele to allow for detection and correct assignment of alleles. Any sample allele whose size was outside the ±0.5-nt window was determined to be OL i.e. “Off Ladder”. OL alleles are alleles of a size that is not represented in the AmpFISTR® MiniFiler™ Allelic Ladder or an allele that does not correspond to an allelic ladder, but whose size is just outside a window because of measurement error. The minimum peak height threshold of &gt;50 RFU was set based on validation experiments performed to avoid typing when stochastic effects are likely to interfere with accurate interpretation of mixtures. The calculation of fetal fraction is based on averaging all informative markers. Informative markers are identified by the presence of peaks on the electropherogram that fall within the parameters of preset bins for the STRs that are analyzed.</div>
</li> <li> <para-num num="[0264]"> </para-num> <div class="description-line" id="p-0286" num="0264">Calculations of fetal fraction were performed using the average peak height for major and minor alleles at every STR locus determined from triplicate injections. The rules applied to the calculation are:</div>
</li> <li> <div class="description-line" id="p-0287" num="0000">1. off-ladder allele (OL) data for alleles are not included in the calculation; and
<br/>
2. only peak heights derived from &gt;50 RFU (relative fluorescence units) are included in the calculation
<br/>
3. if only one bin is present the marker is deemed non-informative; and
<br/>
4. if a second bin is called but the peaks of the first and second bins are within 50-70% of their relative fluorescence units (RFU) in peak height, the minority fraction is not measured and the marker is deemed not informative.
</div>
</li> <li> <para-num num="[0265]"> </para-num> <div class="description-line" id="p-0288" num="0265">The fraction of the minor allele for any given informative marker is calculated by dividing the peak height of the minor component by the sum of the peak height for the major component, and expressed as a percent was first calculated for each informative locus as</div>
</li> <li> <div class="description-line" id="p-0289" num="0000"> <br/>fetal fraction=(Σpeak height of minor allele/Σpeak height of major allele(s))×100,
</div>
</li> <li> <para-num num="[0266]"> </para-num> <div class="description-line" id="p-0290" num="0266">The fetal fraction for a sample comprising two or more informative STRs, would be calculated as the average of the fetal fractions calculated for the two or more informative markers.</div>
</li> <li> <para-num num="[0267]"> </para-num> <div class="description-line" id="p-0291" num="0267">Table 16 provides the data obtained from analyzing cfDNA of a subject pregnant with a male fetus.</div>
</li> <li> <div class="description-line" id="p-0292" num="0000">
<tables id="TABLE-US-00016" num="00016">
<patent-tables colsep="0" frame="none" pgwide="1" rowsep="0">
<table align="left" class="description-table" cols="1" colsep="0" rowsep="0" width="100%">
<thead>
<tr class="description-tr">
<td class="description-td" colspan="1" nameend="1" namest="1" rowsep="1">TABLE 16</td>
</tr>
</thead>
<tbody><tr class="description-tr">
<td align="center" class="description-td" colspan="1" nameend="1" namest="1" rowsep="1"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">Detection of Fetal STR Alleles and Use for Determination of Fetal Fraction</td>
</tr>
</tbody></table>
<table align="left" class="description-table" cols="9" colsep="0" rowsep="0" width="100%">
<tbody><tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">Allele</td>
<td class="description-td">Allele</td>
<td class="description-td">Allele</td>
<td class="description-td"> </td>
<td class="description-td">Fetal</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">Allele</td>
<td class="description-td">Allele</td>
<td class="description-td">Allele</td>
<td class="description-td">1</td>
<td class="description-td">2</td>
<td class="description-td">3</td>
<td class="description-td">Fetal</td>
<td class="description-td">Fraction</td>
</tr>
<tr class="description-tr">
<td class="description-td">STR</td>
<td class="description-td">1</td>
<td class="description-td">2</td>
<td class="description-td">3</td>
<td class="description-td">Height</td>
<td class="description-td">Height</td>
<td class="description-td">Height</td>
<td class="description-td">Fraction</td>
<td class="description-td">(Mean/STR)</td>
</tr>
<tr class="description-tr">
<td align="center" class="description-td" colspan="9" nameend="9" namest="1" rowsep="1"> </td>
</tr>
</tbody></table>
<table align="left" class="description-table" cols="9" colsep="0" rowsep="0" width="100%">
<tbody><tr class="description-tr">
<td class="description-td">AMEL</td>
<td class="description-td">X</td>
<td class="description-td">Y</td>
<td class="description-td"> </td>
<td class="description-td">3599</td>
<td class="description-td">106</td>
<td class="description-td"> </td>
<td class="description-td">2.9</td>
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">AMEL</td>
<td class="description-td">X</td>
<td class="description-td">Y</td>
<td class="description-td"> </td>
<td class="description-td">3602</td>
<td class="description-td">110</td>
<td class="description-td"> </td>
<td class="description-td">3.1</td>
</tr>
<tr class="description-tr">
<td class="description-td">AMEL</td>
<td class="description-td">X</td>
<td class="description-td">Y</td>
<td class="description-td"> </td>
<td class="description-td">3652</td>
<td class="description-td">109</td>
<td class="description-td"> </td>
<td class="description-td">3.0</td>
<td class="description-td">3.0</td>
</tr>
<tr class="description-tr">
<td class="description-td">CSF1PO</td>
<td class="description-td">11</td>
<td class="description-td">12</td>
<td class="description-td"> </td>
<td class="description-td">2870</td>
<td class="description-td">2730</td>
</tr>
<tr class="description-tr">
<td class="description-td">CSF1PO</td>
<td class="description-td">11</td>
<td class="description-td">12</td>
<td class="description-td"> </td>
<td class="description-td">2924</td>
<td class="description-td">2762</td>
</tr>
<tr class="description-tr">
<td class="description-td">CSF1PO</td>
<td class="description-td">11</td>
<td class="description-td">12</td>
<td class="description-td"> </td>
<td class="description-td">2953</td>
<td class="description-td">2786</td>
</tr>
<tr class="description-tr">
<td class="description-td">D13S317</td>
<td class="description-td">11</td>
<td class="description-td">12</td>
<td class="description-td"> </td>
<td class="description-td">2621</td>
<td class="description-td">2588</td>
</tr>
<tr class="description-tr">
<td class="description-td">D13S317</td>
<td class="description-td">11</td>
<td class="description-td">12</td>
<td class="description-td"> </td>
<td class="description-td">2680</td>
<td class="description-td">2619</td>
</tr>
<tr class="description-tr">
<td class="description-td">D13S317</td>
<td class="description-td">11</td>
<td class="description-td">12</td>
<td class="description-td"> </td>
<td class="description-td">2717</td>
<td class="description-td">2659</td>
</tr>
<tr class="description-tr">
<td class="description-td">D16S539</td>
<td class="description-td">9</td>
<td class="description-td">11</td>
<td class="description-td"> </td>
<td class="description-td">1056</td>
<td class="description-td">1416</td>
</tr>
<tr class="description-tr">
<td class="description-td">D16S539</td>
<td class="description-td">9</td>
<td class="description-td">11</td>
<td class="description-td"> </td>
<td class="description-td">1038</td>
<td class="description-td">1394</td>
</tr>
<tr class="description-tr">
<td class="description-td">D16S539</td>
<td class="description-td">9</td>
<td class="description-td">11</td>
<td class="description-td"> </td>
<td class="description-td">1072</td>
<td class="description-td">1437</td>
</tr>
<tr class="description-tr">
<td class="description-td">D18S51</td>
<td class="description-td">13</td>
<td class="description-td">15</td>
<td class="description-td"> </td>
<td class="description-td">2026</td>
<td class="description-td">1555</td>
</tr>
<tr class="description-tr">
<td class="description-td">D18S51</td>
<td class="description-td">13</td>
<td class="description-td">15</td>
<td class="description-td"> </td>
<td class="description-td">2006</td>
<td class="description-td">1557</td>
</tr>
<tr class="description-tr">
<td class="description-td">D18S51</td>
<td class="description-td">13</td>
<td class="description-td">15</td>
<td class="description-td"> </td>
<td class="description-td">2050</td>
<td class="description-td">1578</td>
</tr>
<tr class="description-tr">
<td class="description-td">D21S11</td>
<td class="description-td">28</td>
<td class="description-td">31.2</td>
<td class="description-td"> </td>
<td class="description-td">2450</td>
<td class="description-td">61</td>
<td class="description-td"> </td>
<td class="description-td">2.5</td>
</tr>
<tr class="description-tr">
<td class="description-td">D21S11</td>
<td class="description-td">28</td>
<td class="description-td">31.2</td>
<td class="description-td"> </td>
<td class="description-td">2472</td>
<td class="description-td">62</td>
<td class="description-td"> </td>
<td class="description-td">2.5</td>
</tr>
<tr class="description-tr">
<td class="description-td">D21S11</td>
<td class="description-td">28</td>
<td class="description-td">31.2</td>
<td class="description-td"> </td>
<td class="description-td">2508</td>
<td class="description-td">67</td>
<td class="description-td"> </td>
<td class="description-td">2.7</td>
<td class="description-td">2.6</td>
</tr>
<tr class="description-tr">
<td class="description-td">D2S1338</td>
<td class="description-td">20</td>
<td class="description-td">23</td>
<td class="description-td"> </td>
<td class="description-td">3417</td>
<td class="description-td">3017</td>
</tr>
<tr class="description-tr">
<td class="description-td">D2S1338</td>
<td class="description-td">20</td>
<td class="description-td">23</td>
<td class="description-td"> </td>
<td class="description-td">3407</td>
<td class="description-td">3020</td>
</tr>
<tr class="description-tr">
<td class="description-td">D2S1338</td>
<td class="description-td">20</td>
<td class="description-td">23</td>
<td class="description-td"> </td>
<td class="description-td">3493</td>
<td class="description-td">3055</td>
</tr>
<tr class="description-tr">
<td class="description-td">D7S820</td>
<td class="description-td">9</td>
<td class="description-td">12</td>
<td class="description-td">13</td>
<td class="description-td">2373</td>
<td class="description-td">178</td>
<td class="description-td">1123</td>
<td class="description-td">5.1</td>
</tr>
<tr class="description-tr">
<td class="description-td">D7S820</td>
<td class="description-td">9</td>
<td class="description-td">12</td>
<td class="description-td">13</td>
<td class="description-td">2411</td>
<td class="description-td">181</td>
<td class="description-td">1140</td>
<td class="description-td">5.1</td>
</tr>
<tr class="description-tr">
<td class="description-td">D7S820</td>
<td class="description-td">9</td>
<td class="description-td">12</td>
<td class="description-td">13</td>
<td class="description-td">2441</td>
<td class="description-td">182</td>
<td class="description-td">1156</td>
<td class="description-td">5.1</td>
<td class="description-td">5.1</td>
</tr>
<tr class="description-tr">
<td class="description-td">FGA</td>
<td class="description-td">17.2</td>
<td class="description-td">22</td>
<td class="description-td">25</td>
<td class="description-td">68</td>
<td class="description-td">1140</td>
<td class="description-td">896</td>
<td class="description-td">3.3</td>
</tr>
<tr class="description-tr">
<td class="description-td">FGA</td>
<td class="description-td">17.2</td>
<td class="description-td">22</td>
<td class="description-td">25</td>
<td class="description-td">68</td>
<td class="description-td">1144</td>
<td class="description-td">909</td>
<td class="description-td">3.1</td>
</tr>
<tr class="description-tr">
<td class="description-td">FGA</td>
<td class="description-td">17.2</td>
<td class="description-td">22</td>
<td class="description-td">25</td>
<td class="description-td">68</td>
<td class="description-td">1151</td>
<td class="description-td">925</td>
<td class="description-td">3.3</td>
<td class="description-td">3.2</td>
</tr>
<tr class="description-tr">
<td align="center" class="description-td" colspan="9" nameend="9" namest="1" rowsep="1"> </td>
</tr>
<tr class="description-tr">
<td align="left" class="description-td" colspan="9" id="FOO-00004" nameend="9" namest="1">Fetal Fraction = 3.5</td>
</tr>
</tbody></table>
</patent-tables>
</tables>
</div>
</li> <li> <para-num num="[0268]"> </para-num> <div class="description-line" id="p-0293" num="0268">The results show that STRs can be identified in maternal cfDNA and can be used for determining the presence or absence of fetal DNA as indicated by the detection of a minor component at one or more STR alleles, for determining the percent fetal fraction, and for determining fetal gender as indicated by the presence or absence of the Amelogenin allele.</div>
</li> <heading id="h-0032">Example 13</heading>
<heading id="h-0033">Whole Genome Amplification of cfDNA and Detection of Fetal STRs by Capillary Electrophoresis</heading>
<li> <para-num num="[0269]"> </para-num> <div class="description-line" id="p-0294" num="0269">To improve the sensitivity of the STR capillary electrophoresis assay in detecting and quantifying the STR alleles in the minor contributor of the cfDNA sample, the number of starting genomes in the artificial samples was increased by a modified whole genome amplification strategy.</div>
</li> <li> <para-num num="[0270]"> </para-num> <div class="description-line" id="p-0295" num="0270">Peripheral blood samples were collected and processed as described in Example 2. Cell-free DNA was extracted from 1 ml cell-free plasma using the Roche MagNA Pure Compact Nucleic Acid Isolation Kit I—Large Volume (Roche Applied Science, IN) using the MagNA Pure Compact Instrument, and eluted in 50 μl of elution buffer. Ten microliters of the extracted cfDNA were used to quantify the cfDNA, and the remainder was stored (see storage instructions WI0035 Clinical Sample Storage). The concentration of the plasma extracted cfDNA was determined by fluorescence-based quantitation with UV absorbance measurements using the Qubit™ Quantitation Platform (Invitrogen).</div>
</li> <li> <para-num num="[0271]"> </para-num> <div class="description-line" id="p-0296" num="0271">The concentration of cfDNA quantified in plasma samples prepared using the MagnaPure Nucleic Acid Isolation Kit I from 16 pregnant subjects was determined to range between 20 and 100 pg/μl. As the fetal component of plasma cfDNA is known to contribute 3-10% of the total plasma cfDNA, artificial plasma samples were created by spiking aliquots of cfDNA derived from plasma of female volunteer subjects with cfDNA extracted from plasma of male volunteer subjects to mimic the ratios of fetal to maternal cfDNA found in the pregnant subjects. Artificial samples were created to contain 200-1000 pg of extracted female cfDNA that was spiked with 45-150 pg of extracted male cfDNA in a total volume of 10 μl. Each artificial sample was spiked to contain 3%, 5% and 10% male cfDNA.</div>
</li> <li> <para-num num="[0272]"> </para-num> <div class="description-line" id="p-0297" num="0272">Artificial samples having concentrations of total cfDNA of less than approximately 50 pg/μl, were pre-amplified using the modified improved primer extension amplification PCR (mIPEP) amplification according to the method of Hanson and Ballantyne, (Hanson and Ballantyne, Analytical Biochem 346:246-257 [2005]) as follows. Ten microliters of spiked plasma cfDNA were amplified in a 25 μl reaction volume containing 1 mM dNTPs, 2.5 mM MgCl<sub>2 </sub>(Applied Biosystems), 1× Expand High Fidelity Buffer (No. 3), 10.5U Expand High Fidelity Enzyme Mix (Roche Diagnostics), and 40 μM PEP primer (5′-NNNNNNNNNNNNNNN-3′, Qiagen). The amplification was performed in a GeneAmp PCR System 9700 Thermocycler under the following conditions: (1) 20 and 30 cycles of 94° C. for 1 minute, 37° C. for 2 minutes, and 0.1° C./s ramp to 55° C. for 4 minutes. The amplification product was purified using a Qiagen column. The concentration of the amplification product was determined using the Qubit™ Quantitation Platform as described above. STR analysis was performed as described in Example 9 above, except that only peak heights &gt;100 RFU were included in the calculations.</div>
</li> <li> <para-num num="[0273]"> </para-num> <div class="description-line" id="p-0298" num="0273">The results are shown in Tables 17, 18, and 19. The results provided in Table 17 show that the cfDNA contained in 10 μl cfDNA of artificial samples ART23 and ART24 having a starting concentration of cfDNA of 46.2 and 50.2 pg/μl, respectively, was amplified by approximately 5 and 10 fold following 20 and 30 cycles of PCR amplification, respectively.</div>
</li> <li> <para-num num="[0274]"> </para-num> <div class="description-line" id="p-0299" num="0274">These data indicate that a pre-amplification of cfDNA using the mIPEP method provided enhanced levels of total cfDNA rendering the level of the minor component more amenable to the STR analysis.</div>
</li> <li> <div class="description-line" id="p-0300" num="0000">
<tables id="TABLE-US-00017" num="00017">
<patent-tables colsep="0" frame="none" rowsep="0">
<table align="left" class="description-table" cols="1" colsep="0" rowsep="0" width="100%">
<thead>
<tr class="description-tr">
<td class="description-td" colspan="1" nameend="1" namest="1" rowsep="1">TABLE 17</td>
</tr>
</thead>
<tbody><tr class="description-tr">
<td align="center" class="description-td" colspan="1" nameend="1" namest="1" rowsep="1"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">Whole Genome Amplification of cfDNA by mIPEP</td>
</tr>
</tbody></table>
<table align="left" class="description-table" cols="4" colsep="0" rowsep="0" width="100%">
<tbody><tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">cfDNA with</td>
<td class="description-td">cfDNA with</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">cfDNA without</td>
<td class="description-td">mIPEP: 20 PCR</td>
<td class="description-td">mIPEP: 30 PCR</td>
</tr>
<tr class="description-tr">
<td class="description-td">SAMPLE</td>
<td class="description-td">mIEP (pg/μl)</td>
<td class="description-td">cycles (pg/50 μl)</td>
<td class="description-td">cycles (pg/50 μl)</td>
</tr>
<tr class="description-tr">
<td align="center" class="description-td" colspan="4" nameend="4" namest="1" rowsep="1"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">ART23</td>
<td class="description-td">46.2</td>
<td class="description-td">2265</td>
<td class="description-td">4125</td>
</tr>
<tr class="description-tr">
<td class="description-td">ART24</td>
<td class="description-td">50.2</td>
<td class="description-td">2085</td>
<td class="description-td">3875</td>
</tr>
<tr class="description-tr">
<td align="center" class="description-td" colspan="4" nameend="4" namest="1" rowsep="1"> </td>
</tr>
</tbody></table>
</patent-tables>
</tables>
</div>
</li> <li> <para-num num="[0275]"> </para-num> <div class="description-line" id="p-0301" num="0275">Table 18 shows triplicate measurements profiling 9 loci of the cfDNA of spiked samples ART23 and ART24 following the mIPEP procedure with 20 and 30 cycles of amplification, as described above.</div>
</li> <li> <para-num num="[0276]"> </para-num> <div class="description-line" id="p-0302" num="0276">The data in Table 19 indicate that pre-amplification of cfDNA enables the detection and quantification of the minor component at most loci tested in artificially mixed samples having a starting cfDNA concentration that would otherwise not permit an accurate analysis of the minor STR alleles.</div>
</li> <li> <div class="description-line" id="p-0303" num="0000">
<tables id="TABLE-US-00018" num="00018">
<patent-tables colsep="0" frame="none" pgwide="1" rowsep="0">
<table align="left" class="description-table" cols="1" colsep="0" rowsep="0" width="100%">
<thead>
<tr class="description-tr">
<td class="description-td" colspan="1" nameend="1" namest="1" rowsep="1">TABLE 18</td>
</tr>
</thead>
<tbody><tr class="description-tr">
<td align="center" class="description-td" colspan="1" nameend="1" namest="1" rowsep="1"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">mIPEP Preamplification and Detection of Minor Fetal Component</td>
</tr>
</tbody></table>
<table align="left" class="description-table" cols="9" colsep="0" rowsep="0" width="100%">
<tbody><tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">ART23 (453 pg)</td>
<td class="description-td">ART23 (825 pg)</td>
<td class="description-td">ART23 (462 pg)</td>
<td class="description-td"> </td>
<td class="description-td">ART24 (417 pg)</td>
<td class="description-td">ART24 (775 pg)</td>
<td class="description-td">ART24 (502 pg)</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">mIPEP</td>
<td class="description-td">mIPEP</td>
<td class="description-td">Extracted</td>
<td class="description-td"> </td>
<td class="description-td">mIPEP</td>
<td class="description-td">mIPEP</td>
<td class="description-td">Extracted</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">amplified</td>
<td class="description-td">amplified</td>
<td class="description-td">unamplified</td>
<td class="description-td"> </td>
<td class="description-td">amplified</td>
<td class="description-td">amplified</td>
<td class="description-td">unamplified</td>
</tr>
<tr class="description-tr">
<td class="description-td">STR</td>
<td class="description-td"> </td>
<td class="description-td">20 cycles</td>
<td class="description-td">20 cycles</td>
<td class="description-td">cfDNA</td>
<td class="description-td"> </td>
<td class="description-td">30 cycles</td>
<td class="description-td">30 cycles</td>
<td class="description-td">cfDNA</td>
</tr>
<tr class="description-tr">
<td class="description-td">Locus</td>
<td class="description-td">Allele</td>
<td class="description-td">Allele Height</td>
<td class="description-td">Allele Height</td>
<td class="description-td">Allele Height</td>
<td class="description-td">Allele</td>
<td class="description-td">Allele Height</td>
<td class="description-td">Allele Height</td>
<td class="description-td">Allele Height</td>
</tr>
<tr class="description-tr">
<td align="center" class="description-td" colspan="9" nameend="9" namest="1" rowsep="1"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">AMEL</td>
<td class="description-td">X/Y</td>
<td class="description-td">291/95 </td>
<td class="description-td">397/170</td>
<td class="description-td">535/832</td>
<td class="description-td">X/Y</td>
<td class="description-td">695/359</td>
<td class="description-td">1878/1148</td>
<td class="description-td">1564/1959</td>
</tr>
<tr class="description-tr">
<td class="description-td">AMEL</td>
<td class="description-td">X/Y</td>
<td class="description-td">425/147</td>
<td class="description-td">428/188</td>
<td class="description-td"> 675/1048</td>
<td class="description-td">X/Y</td>
<td class="description-td">1216/619 </td>
<td class="description-td">1551/954 </td>
<td class="description-td">1573/1943</td>
</tr>
<tr class="description-tr">
<td class="description-td">AMEL</td>
<td class="description-td">X/Y</td>
<td class="description-td">267/94 </td>
<td class="description-td">455/203</td>
<td class="description-td"> 664/1043</td>
<td class="description-td">X/Y</td>
<td class="description-td">718/363</td>
<td class="description-td">1479/924 </td>
<td class="description-td">1621/2024</td>
</tr>
<tr class="description-tr">
<td class="description-td">CSF1PO</td>
<td class="description-td">10/11</td>
<td class="description-td">800/979</td>
<td class="description-td"> 725/1009</td>
<td class="description-td">1429/1325</td>
<td class="description-td">11/12</td>
<td class="description-td">2029/1317</td>
<td class="description-td">4159/2317</td>
<td class="description-td">2990/3083</td>
</tr>
<tr class="description-tr">
<td class="description-td">CSF1PO</td>
<td class="description-td">10/11</td>
<td class="description-td">1147/1432</td>
<td class="description-td"> 789/1102</td>
<td class="description-td">1779/1650</td>
<td class="description-td">11/12</td>
<td class="description-td">3449/2223</td>
<td class="description-td">
<b>3460/113/1890</b>
</td>
<td class="description-td">2996/3118</td>
</tr>
<tr class="description-tr">
<td class="description-td">CSF1PO</td>
<td class="description-td">10/11</td>
<td class="description-td">729/906</td>
<td class="description-td"> 831/1162</td>
<td class="description-td">1783/1657</td>
<td class="description-td">11/12</td>
<td class="description-td">2006/1309</td>
<td class="description-td">3362/1840</td>
<td class="description-td">3072/3183</td>
</tr>
<tr class="description-tr">
<td class="description-td">D13S317</td>
<td class="description-td">12</td>
<td class="description-td">743</td>
<td class="description-td">515</td>
<td class="description-td">1229</td>
<td class="description-td">11  </td>
<td class="description-td">955</td>
<td class="description-td">1490</td>
<td class="description-td">3634</td>
</tr>
<tr class="description-tr">
<td class="description-td">D13S317</td>
<td class="description-td">12</td>
<td class="description-td">1079 </td>
<td class="description-td">563</td>
<td class="description-td">1534</td>
<td class="description-td">11  </td>
<td class="description-td">1631 </td>
<td class="description-td">1198</td>
<td class="description-td">3631</td>
</tr>
<tr class="description-tr">
<td class="description-td">D13S317</td>
<td class="description-td">12</td>
<td class="description-td">668</td>
<td class="description-td">583</td>
<td class="description-td">1520</td>
<td class="description-td">11  </td>
<td class="description-td">968</td>
<td class="description-td">1170</td>
<td class="description-td">3795</td>
</tr>
<tr class="description-tr">
<td class="description-td">D16S539</td>
<td class="description-td"> 9/10</td>
<td class="description-td">239/140</td>
<td class="description-td">370/466</td>
<td class="description-td">835/676</td>
<td class="description-td">10/11</td>
<td class="description-td">513/512</td>
<td class="description-td">1173/1472</td>
<td class="description-td">1678/973 </td>
</tr>
<tr class="description-tr">
<td class="description-td">D16S539</td>
<td class="description-td"> 9/10</td>
<td class="description-td">347/203</td>
<td class="description-td">64*(OL)/</td>
<td class="description-td">1046/864 </td>
<td class="description-td">10/11</td>
<td class="description-td">859/870</td>
<td class="description-td"> <b>973/1212</b> </td>
<td class="description-td">1730/999 </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">391/489</td>
</tr>
<tr class="description-tr">
<td class="description-td">D16S539</td>
<td class="description-td"> 9/10</td>
<td class="description-td">227/134</td>
<td class="description-td">441/515</td>
<td class="description-td">1055/860 </td>
<td class="description-td">10/11</td>
<td class="description-td">530/513</td>
<td class="description-td"> 960/1183</td>
<td class="description-td">1784/1044</td>
</tr>
<tr class="description-tr">
<td class="description-td">D18S51</td>
<td class="description-td">14/15</td>
<td class="description-td">359/464</td>
<td class="description-td">363/220</td>
<td class="description-td">785/541</td>
<td class="description-td">12/18</td>
<td class="description-td">1044/576 </td>
<td class="description-td">1840/786 </td>
<td class="description-td">2559/1507</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">512/645</td>
<td class="description-td">391/226</td>
<td class="description-td">999/672</td>
<td class="description-td">12/18</td>
<td class="description-td">1769/994 </td>
<td class="description-td">1511/643 </td>
<td class="description-td">2565/1469</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">313/402</td>
<td class="description-td">409/245</td>
<td class="description-td">994/685</td>
<td class="description-td">12/18</td>
<td class="description-td">1033/567 </td>
<td class="description-td">1496/631 </td>
<td class="description-td">2643/1523</td>
</tr>
<tr class="description-tr">
<td class="description-td">D21S11</td>
<td class="description-td">29/32</td>
<td class="description-td">103/104</td>
<td class="description-td">114/173</td>
<td class="description-td">605/413</td>
<td class="description-td">31.2</td>
<td class="description-td">381</td>
<td class="description-td"> 661</td>
<td class="description-td">3276</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">149/153</td>
<td class="description-td">130/182</td>
<td class="description-td">759/523</td>
<td class="description-td">31.2</td>
<td class="description-td">650</td>
<td class="description-td"> 536</td>
<td class="description-td">3028</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">85/86</td>
<td class="description-td">131/196</td>
<td class="description-td">760/525</td>
<td class="description-td">31.2</td>
<td class="description-td">380</td>
<td class="description-td"> 520</td>
<td class="description-td">3282</td>
</tr>
<tr class="description-tr">
<td class="description-td">D2S1338</td>
<td class="description-td">18/20</td>
<td class="description-td">572/383</td>
<td class="description-td">428/363</td>
<td class="description-td">1116/1013</td>
<td class="description-td">19/20</td>
<td class="description-td">1066/433 </td>
<td class="description-td">2315/1243</td>
<td class="description-td">2962/2968</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">827/553</td>
<td class="description-td">454/386</td>
<td class="description-td">1428/1279</td>
<td class="description-td">19/20</td>
<td class="description-td">1821/757 </td>
<td class="description-td">1901/101 </td>
<td class="description-td">2942/2942</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">530/351</td>
<td class="description-td">482/408</td>
<td class="description-td">1431/1275</td>
<td class="description-td">19/20</td>
<td class="description-td">1063/444 </td>
<td class="description-td">1859/1012</td>
<td class="description-td">3072/3067</td>
</tr>
<tr class="description-tr">
<td class="description-td">D7S820</td>
<td class="description-td">11/12</td>
<td class="description-td">262/167</td>
<td class="description-td">149/270</td>
<td class="description-td">557/627</td>
<td class="description-td">11/12</td>
<td class="description-td">256/138</td>
<td class="description-td">520/322</td>
<td class="description-td">1550/1548</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">62/366/231</td>
<td class="description-td">162/292</td>
<td class="description-td">699/775</td>
<td class="description-td">11/12</td>
<td class="description-td">448/236</td>
<td class="description-td">419/258</td>
<td class="description-td">1484/1466</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">224/146</td>
<td class="description-td">169/307</td>
<td class="description-td">689/779</td>
<td class="description-td">11/12</td>
<td class="description-td">253/141</td>
<td class="description-td">406/250</td>
<td class="description-td">1579/1573</td>
</tr>
<tr class="description-tr">
<td class="description-td">FGA</td>
<td class="description-td">21/23</td>
<td class="description-td">263/146</td>
<td class="description-td">181/88 </td>
<td class="description-td">596/365</td>
<td class="description-td">22/24</td>
<td class="description-td">228/244</td>
<td class="description-td">375/429</td>
<td class="description-td">1272/1064</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">384/215</td>
<td class="description-td">191/92 </td>
<td class="description-td">762/450</td>
<td class="description-td">22/24</td>
<td class="description-td">409/425</td>
<td class="description-td">303/345</td>
<td class="description-td">1221/1023</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">230/136</td>
<td class="description-td">202/102</td>
<td class="description-td">749/456</td>
<td class="description-td">22/24</td>
<td class="description-td">232/250</td>
<td class="description-td">297/348</td>
<td class="description-td">1298/1087</td>
</tr>
<tr class="description-tr">
<td align="center" class="description-td" colspan="9" nameend="9" namest="1" rowsep="1"> </td>
</tr>
<tr class="description-tr">
<td align="left" class="description-td" colspan="9" id="FOO-00005" nameend="9" namest="1">*“OL” means “Off Ladder measurement”</td>
</tr>
</tbody></table>
</patent-tables>
</tables>
</div>
</li> <li> <div class="description-line" id="p-0304" num="0000">
<tables id="TABLE-US-00019" num="00019">
<patent-tables colsep="0" frame="none" pgwide="1" rowsep="0">
<table align="left" class="description-table" cols="1" colsep="0" rowsep="0" width="100%">
<thead>
<tr class="description-tr">
<td class="description-td" colspan="1" nameend="1" namest="1" rowsep="1">TABLE 19</td>
</tr>
</thead>
<tbody><tr class="description-tr">
<td align="center" class="description-td" colspan="1" nameend="1" namest="1" rowsep="1"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">Identification of fetal STRs following mIPEP amplification of cfDNA in a maternal sample</td>
</tr>
</tbody></table>
<table align="left" class="description-table" cols="7" colsep="0" rowsep="0" width="100%">
<tbody><tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">Percent minor</td>
<td class="description-td">Percent minor</td>
</tr>
<tr class="description-tr">
<td class="description-td">STR</td>
<td class="description-td">Allele 1/</td>
<td class="description-td">Allele 2/</td>
<td class="description-td">Allele 3/</td>
<td class="description-td">Allele 4/</td>
<td class="description-td">fraction/STR -</td>
<td class="description-td">fraction/STR -</td>
</tr>
<tr class="description-tr">
<td class="description-td">marker</td>
<td class="description-td">Height</td>
<td class="description-td">Height</td>
<td class="description-td">Height</td>
<td class="description-td">Height</td>
<td class="description-td">minor &gt;100 RFU</td>
<td class="description-td">minor &lt;100 RFU</td>
</tr>
<tr class="description-tr">
<td align="center" class="description-td" colspan="7" nameend="7" namest="1" rowsep="1"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">Amelogenin</td>
<td class="description-td">X/2799</td>
<td class="description-td"> Y/207</td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">Amelogenin</td>
<td class="description-td">X/2751</td>
<td class="description-td"> Y/198</td>
</tr>
<tr class="description-tr">
<td class="description-td">Amelogenin</td>
<td class="description-td">X/3109</td>
<td class="description-td"> Y/232</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">
<b>X/2886</b>
</td>
<td class="description-td"> <b>Y/212</b> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> <b>7</b> </td>
</tr>
<tr class="description-tr">
<td class="description-td">CSF1PO</td>
<td class="description-td">10/2377</td>
<td class="description-td"> 11/1869</td>
<td class="description-td">12/508</td>
</tr>
<tr class="description-tr">
<td class="description-td">CSF1PO</td>
<td class="description-td">10/2299</td>
<td class="description-td"> 11/1814</td>
<td class="description-td">12/498</td>
</tr>
<tr class="description-tr">
<td class="description-td">CSF1PO</td>
<td class="description-td">10/2616</td>
<td class="description-td">11/206</td>
<td class="description-td">12/562</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">
<b>10/2431</b>
</td>
<td class="description-td"> <b>11/1917</b> </td>
<td class="description-td">
<b>12/523</b>
</td>
<td class="description-td"> </td>
<td class="description-td">
<b>12</b>
</td>
</tr>
<tr class="description-tr">
<td class="description-td">D13S317</td>
<td class="description-td">10/1232</td>
<td class="description-td"> 11/1600</td>
<td class="description-td">13/186</td>
</tr>
<tr class="description-tr">
<td class="description-td">D13S317</td>
<td class="description-td">10/1208</td>
<td class="description-td"> 11/1548</td>
<td class="description-td">13/182</td>
</tr>
<tr class="description-tr">
<td class="description-td">D13S317</td>
<td class="description-td">10/1386</td>
<td class="description-td"> 11/1758</td>
<td class="description-td">13/212</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">
<b>10/1275</b>
</td>
<td class="description-td"> <b>11/1635</b> </td>
<td class="description-td">
<b>13/193</b>
</td>
<td class="description-td"> </td>
<td class="description-td">
<b>12</b>
</td>
</tr>
<tr class="description-tr">
<td class="description-td">D16S539</td>
<td class="description-td">11/757 </td>
<td class="description-td">12/933</td>
</tr>
<tr class="description-tr">
<td class="description-td">D16S539</td>
<td class="description-td">11/729 </td>
<td class="description-td">12/885</td>
</tr>
<tr class="description-tr">
<td class="description-td">D16S539</td>
<td class="description-td">11/836 </td>
<td class="description-td"> 12/1031</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> <b>11/774</b> </td>
<td class="description-td">
<b>12/950</b>
</td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">
<b>12</b>
</td>
</tr>
<tr class="description-tr">
<td class="description-td">D18S51</td>
<td class="description-td"> <u>OL/80</u>  </td>
<td class="description-td"> 14/3137</td>
<td class="description-td">15/371</td>
</tr>
<tr class="description-tr">
<td class="description-td">D18S51</td>
<td class="description-td"> <u>11/73</u>  </td>
<td class="description-td"> 14/3082</td>
<td class="description-td">15/362</td>
</tr>
<tr class="description-tr">
<td class="description-td">D18S51</td>
<td class="description-td"> <u>OL/83</u>  </td>
<td class="description-td"> 14/3488</td>
<td class="description-td">15/413</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">
<u>
<b>OL</b>
</u>
</td>
<td class="description-td"> <b>14/3236</b> </td>
<td class="description-td">
<b>15/382</b>
</td>
</tr>
<tr class="description-tr">
<td class="description-td">D21S11</td>
<td class="description-td">29/953 </td>
<td class="description-td">30/941</td>
</tr>
<tr class="description-tr">
<td class="description-td">D21S11</td>
<td class="description-td">29/921 </td>
<td class="description-td">30/908</td>
</tr>
<tr class="description-tr">
<td class="description-td">D21S11</td>
<td class="description-td">29/1046</td>
<td class="description-td"> 30/1045</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> <b>29/973</b> </td>
<td class="description-td">
<b>30/965</b>
</td>
</tr>
<tr class="description-tr">
<td class="description-td">D2S1338</td>
<td class="description-td">17/461 </td>
<td class="description-td">18/366</td>
<td class="description-td"> 20/2280</td>
<td class="description-td">24/1760</td>
</tr>
<tr class="description-tr">
<td class="description-td">D2S1338</td>
<td class="description-td">17/460 </td>
<td class="description-td">18/360</td>
<td class="description-td"> 20/2240</td>
<td class="description-td">24/1712</td>
</tr>
<tr class="description-tr">
<td class="description-td">D2S1338</td>
<td class="description-td">17/508 </td>
<td class="description-td">18/409</td>
<td class="description-td"> 20/2563</td>
<td class="description-td">24/1971</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> <b>17/476</b> </td>
<td class="description-td">
<b>18/378</b>
</td>
<td class="description-td"> <b>20/2361</b> </td>
<td class="description-td">
<b>24/1814</b>
</td>
<td class="description-td">
<b>20</b>
</td>
</tr>
<tr class="description-tr">
<td class="description-td">D7S820</td>
<td class="description-td"> 8/1409</td>
<td class="description-td">9/60</td>
<td class="description-td"> 12/1059</td>
</tr>
<tr class="description-tr">
<td class="description-td">D7S820</td>
<td class="description-td"> 8/1380</td>
<td class="description-td">9/60</td>
<td class="description-td"> 12/1036</td>
</tr>
<tr class="description-tr">
<td class="description-td">D7S820</td>
<td class="description-td"> 8/1561</td>
<td class="description-td">9/69</td>
<td class="description-td"> 12/1166</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> <b>8/1450</b> </td>
<td class="description-td">
<b>9/63</b>
</td>
<td class="description-td"> <b>12/1087</b> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> <b>2</b> </td>
</tr>
<tr class="description-tr">
<td class="description-td">FGA</td>
<td class="description-td">19/825 </td>
<td class="description-td">21/850</td>
<td class="description-td">25/279</td>
</tr>
<tr class="description-tr">
<td class="description-td">FGA</td>
<td class="description-td">19/807 </td>
<td class="description-td">21/841</td>
<td class="description-td">25/265</td>
</tr>
<tr class="description-tr">
<td class="description-td">FGA</td>
<td class="description-td">19/913 </td>
<td class="description-td">21/958</td>
<td class="description-td">25/306</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> <b>19/848</b> </td>
<td class="description-td">
<b>21/883</b>
</td>
<td class="description-td">
<b>25/283</b>
</td>
<td class="description-td"> </td>
<td class="description-td">
<b>16</b>
</td>
</tr>
</tbody></table>
<table align="left" class="description-table" cols="3" colsep="0" rowsep="0" width="100%">
<tbody><tr class="description-tr">
<td class="description-td">% fetal fraction &gt;100 RFU for minor allele <b>→</b>12</td>
<td class="description-td">
<b>12</b>
</td>
<td class="description-td"> </td>
</tr>
</tbody></table>
<table align="left" class="description-table" cols="2" colsep="0" rowsep="0" width="100%">
<tbody><tr class="description-tr">
<td class="description-td">% fetal fraction including &lt;100 RFU for minor allele <b>→</b>11</td>
<td class="description-td">
<b>11</b>
</td>
</tr>
<tr class="description-tr">
<td align="center" class="description-td" colspan="2" nameend="2" namest="1" rowsep="1"> </td>
</tr>
</tbody></table>
</patent-tables>
</tables>
</div>
</li> <heading id="h-0034">Example 14</heading>
<heading id="h-0035">Identification of a Partial Chromosomal Deletion</heading>
<li> <para-num num="[0277]"> </para-num> <div class="description-line" id="p-0305" num="0277">The use of sequence doses was applied for assessing partial chromosomal deletion in a biological test sample of cfDNA that was prepared from blood plasma, and sequenced as described in Example 4. The sample was confirmed by karyotyping to have been derived from a subject with a partial deletion of chromosome 11.</div>
</li> <li> <para-num num="[0278]"> </para-num> <div class="description-line" id="p-0306" num="0278">Analysis of the sequencing data for the partial aneuploidy (partial deletion of chromosome 11 i.e. q21-q23) was performed as described for the chromosomal aneuploidies in the previous examples. Mapping of the sequence tags to chromosome 11 in a test sample revealed a noticeable loss of tag counts between base pairs 81000082-103000103 in the q arm of the chromosome relative to the tag counts obtained for corresponding sequence on chromosome 11 in the qualified samples (data not shown). Sequence tags mapped to the sequence of interest on chromosome 11 (810000082-103000103 bp) in each of the qualified samples, and sequence tags mapped to all 20 megabase segments in the entire genome in the qualified samples i.e. qualified sequence tag densities, were used to determine qualified sequence doses as ratios of tag densities in all qualified samples. The average sequence dose, standard deviation, and coefficient of variation were calculated for all 20 megabase segments in the entire genome, and the 20-megabase sequence having the least variability was the identified normalizing sequence on chromosome 5 (13000014-33000033 bp) (See Table 8), which was used to calculate the dose for the sequence of interest in the test sample (see Table 9). Table 20 provides the sequence dose for the sequence of interest on chromosome 11 (810000082-103000103 bp) in the test sample that was calculated as the ratio of sequence tags mapped to the sequence of interest and the sequence tags mapped to the identified normalizing sequence. <figref idrefs="DRAWINGS">FIG. 6</figref> shows the sequence doses for the sequence of interest in the 7 qualified samples (◯) and the sequence dose for the corresponding sequence in the test sample (⋄). The mean is shown by the solid line, and the calculated threshold for the positive diagnosis of partial aneuploidy that was set 5 standard deviations from the mean is shown by the dashed line. A diagnosis for partial aneuploidy was based on the sequence dose in the test sample being less than the set threshold. The test sample was verified by karyotyping to have deletion q21-q23 on chromosome 11.</div>
</li> <li> <para-num num="[0279]"> </para-num> <div class="description-line" id="p-0307" num="0279">Therefore, massively parallel sequencing of samples comprising a mixture of cfDNA from two genomes can identify polymorphisms including chromosomal deletions.</div>
</li> <li> <div class="description-line" id="p-0308" num="0000">
<tables id="TABLE-US-00020" num="00020">
<patent-tables colsep="0" frame="none" rowsep="0">
<table align="left" class="description-table" cols="1" colsep="0" rowsep="0" width="100%">
<thead>
<tr class="description-tr">
<td class="description-td" colspan="1" nameend="1" namest="1" rowsep="1">TABLE 20</td>
</tr>
</thead>
<tbody><tr class="description-tr">
<td align="center" class="description-td" colspan="1" nameend="1" namest="1" rowsep="1"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">Qualified Normalizing Sequence, Dose and Variance for Sequence</td>
</tr>
<tr class="description-tr">
<td class="description-td">Chr11: 81000082-103000103 (qualified samples n = 7)</td>
</tr>
</tbody></table>
<table align="left" class="description-table" cols="2" colsep="0" rowsep="0" width="100%">
<tbody><tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">Chr11: 81000082-103000103</td>
</tr>
</tbody></table>
<table align="left" class="description-table" cols="4" colsep="0" rowsep="0" width="100%">
<tbody><tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">Avg</td>
<td class="description-td">Stdev</td>
<td class="description-td">CV</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td align="center" class="description-td" colspan="4" nameend="3" namest="offset" rowsep="1"> </td>
</tr>
</tbody></table>
<table align="left" class="description-table" cols="5" colsep="0" rowsep="0" width="100%">
<tbody><tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">Chr5: 13000014-</td>
<td class="description-td">1.164702</td>
<td class="description-td">0.004914</td>
<td class="description-td">0.42</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">33000033</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td align="center" class="description-td" colspan="5" nameend="4" namest="offset" rowsep="1"> </td>
</tr>
</tbody></table>
</patent-tables>
</tables>
</div>
</li> <li> <div class="description-line" id="p-0309" num="0000">
<tables id="TABLE-US-00021" num="00021">
<patent-tables colsep="0" frame="none" rowsep="0">
<table align="left" class="description-table" cols="1" colsep="0" rowsep="0" width="100%">
<thead>
<tr class="description-tr">
<td class="description-td" colspan="1" nameend="1" namest="1" rowsep="1">TABLE 21</td>
</tr>
</thead>
<tbody><tr class="description-tr">
<td align="center" class="description-td" colspan="1" nameend="1" namest="1" rowsep="1"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">Sequence Dose for Sequence of Interest (81000082-</td>
</tr>
<tr class="description-tr">
<td class="description-td">103000103) on Chromosome 11 (test sample 11206)</td>
</tr>
</tbody></table>
<table align="left" class="description-table" cols="4" colsep="0" rowsep="0" width="100%">
<tbody><tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">Chromosome</td>
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">Chromosome</td>
<td class="description-td">Sequence Tag</td>
<td class="description-td">Segment Dose for</td>
</tr>
<tr class="description-tr">
<td class="description-td">Segment</td>
<td class="description-td">Density</td>
<td class="description-td">Chr 11 (q21-q23)</td>
<td class="description-td">Threshold</td>
</tr>
<tr class="description-tr">
<td align="center" class="description-td" colspan="4" nameend="4" namest="1" rowsep="1"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">Chr11: 81000082-</td>
<td class="description-td">27,052</td>
<td class="description-td">1.0434313</td>
<td class="description-td">1.1401347</td>
</tr>
<tr class="description-tr">
<td class="description-td">103000103</td>
</tr>
<tr class="description-tr">
<td class="description-td">Chr5: 13000014-</td>
<td class="description-td">25,926</td>
</tr>
<tr class="description-tr">
<td class="description-td">33000033</td>
</tr>
<tr class="description-tr">
<td align="center" class="description-td" colspan="4" nameend="4" namest="1" rowsep="1"> </td>
</tr>
</tbody></table>
</patent-tables>
</tables>
</div>
</li> <li> <para-num num="[0280]"> </para-num> <div class="description-line" id="p-0310" num="0280">While preferred embodiments of the present invention have been shown and described herein, it will be obvious to those skilled in the art that such embodiments are provided by way of example only. Numerous variations, changes, and substitutions will now occur to those skilled in the art without departing from the invention. It should be understood that various alternatives to the embodiments of the invention described herein may be employed in practicing the invention. It is intended that the following claims define the scope of the invention and that methods and structures within the scope of these claims and their equivalents be covered thereby.</div>
</li> </ul>
</div>
</section><section itemprop="claims" itemscope="">
<h2>Claims (<span itemprop="count">20</span>)</h2>
<div html="" itemprop="content"><div class="claims" lang="EN" load-source="patent-office" mxw-id="PCLM269009954">
<claim-statement>What is claimed is:</claim-statement>
<div class="claim"> <div class="claim" id="CLM-00001" num="00001">
<div class="claim-text"> <b>1</b>. A method for identifying multiple fetal polymorphisms in a fetal genome of a blood sample obtained from a pregnant woman comprising a mixture of cfDNA of a fetal and a maternal genome, said method comprising steps:
<div class="claim-text">(a) performing massively parallel sequencing of amplicons obtained from said mixture of cfDNA, thereby obtaining a plurality of sequence tags;</div> <div class="claim-text">(b) mapping the sequences of said plurality of sequence tags to the sequences of multiple reference polymorphisms, to identify sequence tags mapped to each of said multiple reference polymorphisms;</div> <div class="claim-text">(c) for each of said multiple reference polymorphisms:
<div class="claim-text">(i) quantifying mapped sequence tags that map to a first allele of the reference polymorphism to obtain a quantitative value for the first allele;</div>
<div class="claim-text">(ii) quantifying mapped sequence tags that map to the second allele of the reference polymorphism to obtain a quantitative value for the second allele;</div>
<div class="claim-text">(iii) determining a parameter from the quantitative value for the first allele and the quantitative value for the second allele;</div>
<div class="claim-text">(iv) comparing the parameter to one or more cutoff values associated with the relative abundance of fetal cfDNA in the maternal sample; and</div>
<div class="claim-text">(v) classifying each allele as a (i) minor allele or (ii) major allele, based on the comparison of the parameter to the one or more cutoff values;</div>
</div> <div class="claim-text">whereby a minor allele is identified as a fetal polymorphism in the maternal sample;</div> <div class="claim-text">wherein the identification of fetal polymorphisms originating from the fetal genome in the mixture is performed without genotyping the maternal alleles in a maternal sample that is substantially free of fetal nucleic acids.</div> </div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="CLM-00002" num="00002">
<div class="claim-text"> <b>2</b>. The method of <claim-ref idref="CLM-00001">claim 1</claim-ref>, wherein said portion of said mixture of cfDNA has been specifically enriched for polymorphisms to obtain an enriched library.</div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="CLM-00003" num="00003">
<div class="claim-text"> <b>3</b>. The method of <claim-ref idref="CLM-00002">claim 2</claim-ref>, comprising amplification of polymorphic sequences of interest.</div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="CLM-00004" num="00004">
<div class="claim-text"> <b>4</b>. The method of <claim-ref idref="CLM-00003">claim 3</claim-ref>, comprising selecting polymorphic target sequences for amplification prior to sequencing.</div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="CLM-00005" num="00005">
<div class="claim-text"> <b>5</b>. The method of <claim-ref idref="CLM-00001">claim 1</claim-ref>, wherein said sample is unenriched for polymorphic target sequences in said mixture of cfDNA.</div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="CLM-00006" num="00006">
<div class="claim-text"> <b>6</b>. The method of <claim-ref idref="CLM-00001">claim 1</claim-ref>, wherein said multiple fetal polymorphisms are associated with at least one disorder.</div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="CLM-00007" num="00007">
<div class="claim-text"> <b>7</b>. The method of <claim-ref idref="CLM-00006">claim 6</claim-ref>, wherein said at least one disorder is a monogenic disease.</div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="CLM-00008" num="00008">
<div class="claim-text"> <b>8</b>. The method of <claim-ref idref="CLM-00006">claim 6</claim-ref>, wherein said at least one disorder is a polygenic disease.</div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="CLM-00009" num="00009">
<div class="claim-text"> <b>9</b>. The method of <claim-ref idref="CLM-00006">claim 6</claim-ref>, wherein said at least one disorder is cancer.</div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="CLM-00010" num="00010">
<div class="claim-text"> <b>10</b>. The method of <claim-ref idref="CLM-00001">claim 1</claim-ref>, wherein said sample is a blood sample is a plasma sample.</div>
</div>
</div> <div class="claim"> <div class="claim" id="CLM-00011" num="00011">
<div class="claim-text"> <b>11</b>. A computer-implemented method of identifying fetal alleles in a maternal sample, comprising:
<div class="claim-text">(a) receiving digital sequence information comprising sequence reads obtained from massively parallel whole genome sequencing of a mixture of fetal and maternal cfDNA from a maternal sample,</div> <div class="claim-text">wherein the digital sequence information comprises a plurality of polymorphic sequence reads;</div> <div class="claim-text">(b) mapping, using a microprocessor, a plurality of the sequence reads to polymorphic sites in a reference genome to obtain a plurality of mapped sequence tags;</div> <div class="claim-text">(c) using a microprocessor, for each of a plurality of polymorphic sites in the reference genome:
<div class="claim-text">(i) quantifying mapped sequence tags that map to a first allele of the polymorphic site to obtain a quantitative value for the first allele;</div>
<div class="claim-text">(ii) quantifying mapped sequence tags that map to the second allele of the polymorphic site to obtain a quantitative value for the second allele;</div>
<div class="claim-text">(iii) determining a parameter from the quantitative value for the first allele and the quantitative value for the second allele;</div>
<div class="claim-text">(iv) comparing the parameter to one or more cutoff values associated with the relative abundance of fetal cfDNA in the maternal sample; and</div>
<div class="claim-text">(v) classifying each allele at the polymorphic site as a (i) minor allele or (ii) major allele, based on the comparison of the parameter to the one or more cutoff values;</div>
</div> <div class="claim-text">whereby a minor allele is identified as a fetal allele in the maternal sample;</div> <div class="claim-text">wherein the identification of fetal alleles is performed without genotyping the maternal alleles in a maternal sample that is substantially free of fetal nucleic acids.</div> </div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="CLM-00012" num="00012">
<div class="claim-text"> <b>12</b>. The method of <claim-ref idref="CLM-00011">claim 11</claim-ref>, wherein said multiple polymorphic sequences are not specifically enriched in said mixture.</div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="CLM-00013" num="00013">
<div class="claim-text"> <b>13</b>. The method of <claim-ref idref="CLM-00011">claim 11</claim-ref>, wherein said multiple polymorphic sequences have been specifically enriched in said mixture to obtain an enriched library.</div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="CLM-00014" num="00014">
<div class="claim-text"> <b>14</b>. The method of <claim-ref idref="CLM-00013">claim 13</claim-ref>, comprising amplification of polymorphic sequences of interest.</div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="CLM-00015" num="00015">
<div class="claim-text"> <b>15</b>. The method of <claim-ref idref="CLM-00014">claim 14</claim-ref>, comprising selecting polymorphic target sequences for amplification prior to sequencing.</div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="CLM-00016" num="00016">
<div class="claim-text"> <b>16</b>. The method of <claim-ref idref="CLM-00011">claim 11</claim-ref>, wherein said multiple fetal polymorphisms are associated with at least one disorder.</div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="CLM-00017" num="00017">
<div class="claim-text"> <b>17</b>. The method of <claim-ref idref="CLM-00016">claim 16</claim-ref>, wherein said at least one disorder is a monogenic disease.</div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="CLM-00018" num="00018">
<div class="claim-text"> <b>18</b>. The method of <claim-ref idref="CLM-00016">claim 16</claim-ref>, wherein said at least one disorder is a polygenic disease.</div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="CLM-00019" num="00019">
<div class="claim-text"> <b>19</b>. The method of <claim-ref idref="CLM-00016">claim 16</claim-ref>, wherein said at least one disorder is cancer.</div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="CLM-00020" num="00020">
<div class="claim-text"> <b>20</b>. The method of <claim-ref idref="CLM-00011">claim 11</claim-ref>, wherein said sample is a blood sample is a plasma sample.</div>
</div>
</div> </div>
</div>
</section>
                </article>
            </search-app>
        </body>
    </html>
    