
    <html>
        <body>
            <search-app>
                <article class="result" itemscope="" itemtype="http://schema.org/ScholarlyArticle">
    <h1 itemprop="pageTitle">US20060292564A1 - Method and nucleic acids for the analysis of breast cell proliferative disorders 
        - Google Patents</h1><section itemprop="abstract" itemscope="">
<h2>Abstract</h2>
<div html="" itemprop="content"><abstract lang="EN" load-source="patent-office" mxw-id="PA49560948">
<div class="abstract" id="p-0001" num="0000">The present invention relates to modified and genomic sequences, to oligonucleotides and/or PNA-oligomers for detecting the cytosine methylation state of genomic DNA, as well as to a method for ascertaining genetic and/or epigenetic parameters of genes for use in the differentiation, diagnosis, treatment and/or monitoring of breast cell proliferative disorders, or the predisposition to breast cell proliferative disorders. </div>
</abstract>
</div>
</section><section itemprop="description" itemscope="">
<h2>Description</h2>
<div html="" itemprop="content"><ul class="description" lang="EN" load-source="patent-office" mxw-id="PDES13904407">
<heading id="h-0001">FIELD OF THE INVENTION </heading>
<li> <para-num num="[0001]"> </para-num> <div class="description-line" id="p-0002" num="0001"> Breast cancer is currently the second most common type of cancer amongst women. In 2001 over 190,000 new cases of invasive breast cancer and over 47,000 additional cases of in situ breast cancer were diagnosed. Incidence and death rates increase with age, for the period 1994-1998 the incidence of breast cancer amongst women between the ages of 20 and 24 was only 1.5 per 100,000 population. However the risk increases to 489.7 within the age group 75-79. Mortality rates have decreased by approximately 5% over the last decade and factors affecting 5 year survival rates include age, stage of cancer, socioeconomic factors and race. </div>
</li> <li> <para-num num="[0002]"> </para-num> <div class="description-line" id="p-0003" num="0002"> Breast cancer is defined as the uncontrolled proliferation of cells within breasts tissues. Breasts are comprised of 15 to 20 lobes joined together by ducts. Cancer arises most commonly in the duct, but is also found in the lobes with the rarest type of cancer termed inflammatory breast cancer. </div>
</li> <li> <para-num num="[0003]"> </para-num> <div class="description-line" id="p-0004" num="0003"> It will be appreciated by those skilled in the art that there exists a continuing need to improve methods of early detection, classification and treatment of breast cancers. In contrast to the detection of some other common cancers such as cervical and dermal there are inherent difficulties in classifying and detecting breast cancers. </div>
</li> <li> <para-num num="[0004]"> </para-num> <div class="description-line" id="p-0005" num="0004"> The first step of any treatment is the assessment of the patient's condition comparative to defined classifications of the disease. However the value of such a system is inherently dependant upon the quality of the classification. Breast cancers are staged according to their size, location and occurrence of metastasis. Methods of treatment include the use of surgery, radiation therapy, chemotherapy and hormone therapy, which are also used as adjuvant therapies to surgery. </div>
</li> <li> <para-num num="[0005]"> </para-num> <div class="description-line" id="p-0006" num="0005"> Predictors currently used in the assessment of breast tumours (e.g. histological analysis, estrogen receptor markers) often fail to correctly predict or classify tumour development and behaviour. Therefore, patient response to treatment and prediction of overall outcome is often not accurately predictable. The continued development of breast cancer analysis techniques is currently focused upon the investigation molecular biological markers. </div>
</li> <li> <para-num num="[0006]"> </para-num> <div class="description-line" id="p-0007" num="0006"> The development of molecular biological markers as an alternative to traditional histopathological analysis has to date focused on the analysis of single nucleotide polymorphisms and single genes, such as BRCA1 and BRCA 2. Furthermore, in addition to oncogene mutations gene amplification, and loss of heterozygosity in invasive breast cancer (Callahan, et al., 1992; Cheickh, et al., 1992; Chen, et al, 1992; and, Lippman, et al, 1990) have also been assessed. More recently, the use of microarry technology has allowed the concurrent analysis of multiple genes as well as genetic expression profiling by analysis of RNA and proteins. The analysis of multiple loci in order to predict breast cancer risks in populations was discussed by Pharoah et. al. ‘Polygenic susceptibility to breast cancer and implications for prevention’ Nat Genet. 2002 May;31(1):33-6. Furthermore, Friend et. al. (‘Gene expression profiling predicts clinical outcome of breast cancer’ Nature 415, 530-536 (2002)) used gene expression profiling to predict the outcome of treatment in breast cancer patients, their methods may be used to enable improved post surgery treatment decisions. </div>
</li> <li> <para-num num="[0007]"> </para-num> <div class="description-line" id="p-0008" num="0007"> However, as hereditary breast cancers only account for 5% to 10% of cases it is likely that epigenetic mechanisms, as well as hereditary mutations and environmental factors influence the development of breast cancers. </div>
</li> <li> <para-num num="[0008]"> </para-num> <div class="description-line" id="p-0009" num="0008"> The levels of observation that have been studied by the methodological developments of recent years in molecular biology, are the genes themselves, the translation of these genes into RNA, and the resulting proteins. The question of which gene is switched on at which point in the course of the development of an individual, and how the activation and inhibition of specific genes in specific cells and tissues are controlled is correlatable to the degree and character of the methylation of the genes or of the genome. In this respect, pathogenic conditions may manifest themselves in a changed methylation pattern of individual genes or of the genome. </div>
</li> <li> <para-num num="[0009]"> </para-num> <div class="description-line" id="p-0010" num="0009"> DNA methylation plays a role, for example, in the regulation of the transcription, in genetic imprinting, and in tumorigenesis. Therefore, the identification of 5-methylcytosine as a component of genetic information is of considerable interest. However, 5-methylcytosine positions cannot be identified by sequencing since 5-methylcytosine has the same base pairing behaviour as cytosine. Moreover, the epigenetic information carried by 5-methylcytosine is completely lost during PCR amplification. </div>
</li> <li> <para-num num="[0010]"> </para-num> <div class="description-line" id="p-0011" num="0010"> A relatively new and currently the most frequently used method for analyzing DNA for 5-methylcytosine is based upon the specific reaction of bisulfite with cytosine which, upon subsequent alkaline hydrolysis, is converted to uracil which corresponds to thymidine in its base pairing behaviour. However, 5-methylcytosine remains unmodified under these conditions. Consequently, the original DNA is converted in such a manner that methylcytosine, which originally could not be distinguished from cytosine by its hybridisation behaviour, can now be detected as the only remaining cytosine using “normal” molecular biological techniques, for example, by amplification and hybridisation or sequencing. All of these techniques are based on base pairing which can now be fully exploited. In terms of sensitivity, the prior art is defined by a method which encloses the DNA to be analysed in an agarose matrix, thus preventing the diffusion and renaturation of the DNA (bisulfite only reacts with single-stranded DNA), and which replaces all precipitation and purification steps with fast dialysis (Olek A, Oswald J, Walter J. A modified and improved method for bisulphite based cytosine methylation analysis. Nucleic Acids Res. 1996 Dec. 15;24(24):5064-6). Using this method, it is possible to analyse individual cells, which illustrates the potential of the method. However, currently only individual regions of a length of up to approximately 3000 base pairs are analysed, a global analysis of cells for thousands of possible methylation events is not possible. However, this method cannot reliably analyse very small fragments from small sample quantities either. These are lost through the matrix in spite of the diffusion protection. </div>
</li> <li> <para-num num="[0011]"> </para-num> <div class="description-line" id="p-0012" num="0011"> An overview of the further known methods of detecting 5-methylcytosine may be gathered from the following review article: Rein, T., DePamphilis, M. L., Zorbas, H., Nucleic Acids Res. 1998, 26, 2255. </div>
</li> <li> <para-num num="[0012]"> </para-num> <div class="description-line" id="p-0013" num="0012"> To date, barring few exceptions (e.g., Zeschnigk M, Lich C, Buiting K, Doerfler W, Horsthemke B. A single-tube PCR test for the diagnosis of Angelman and Prader-Willi syndrome based on allelic methylation differences at the SNRPN locus. Eur J Hum Genet. 1997 March-April;5(2):94-8) the bisulfite technique is only used in research. Always, however, short, specific fragments of a known gene are amplified subsequent to a bisulfite treatment and either completely sequenced (Olek A, Walter J. The pre-implantation ontogeny of the H19 methylation imprint. Nat Genet. 1997 November;17(3):275-6) or individual cytosine positions are detected by a primer extension reaction (Gonzalgo M L, Jones P A. Rapid quantitation of methylation differences at specific sites using methylation-sensitive single nucleotide primer extension (Ms-SNuPE). Nucleic Acids Res. 1997 Jun. 15;25(12):2529-31, WO 95/00669) or by enzymatic digestion (Xiong Z, Laird P W. COBRA: a sensitive and quantitative DNA methylation assay. Nucleic Acids Res. 1997 Jun. 15;25(12):2532-4). In addition, detection by hybridisation has also been described (Olek et al., WO 99/28498). </div>
</li> <li> <para-num num="[0013]"> </para-num> <div class="description-line" id="p-0014" num="0013"> Further publications dealing with the use of the bisulfite technique for methylation detection in individual genes are: Grigg G, Clark S. Sequencing 5-methylcytosine residues in genomic DNA. Bioessays. 1994 June; 16(6):431-6, 431; Zeschnigk M, Schmitz B, Dittrich B, Buiting K, Horsthemke B, Doerfler W. Imprinted segments in the human genome: different DNA methylation patterns in the Prader-Willi/Angelman syndrome region as determined by the genomic sequencing method. Hum Mol Genet. 1997 March;6(3):387-95; Feil R, Charlton J, Bird A P, Walter J, Reik W. Methylation analysis on individual chromosomes: improved protocol for bisulphite genomic sequencing. Nucleic Acids Res. 1994 Feb. 25;22(4):695-6; Martin V, Ribieras S, Song-Wang X, Rio M C, Dante R. Genomic sequencing indicates a correlation between DNA hypomethylation in the 5′ region of the pS2 gene and its expression in human breast cancer cell lines. Gene. 1995 May 19;157(1-2):261-4; WO 97/46705, WO 95/15373 and WO 97/45560. </div>
</li> <li> <para-num num="[0014]"> </para-num> <div class="description-line" id="p-0015" num="0014"> An overview of the Prior Art in oligomer array manufacturing can be gathered from a special edition of Nature Genetics (Nature Genetics Supplement, Volume 21, January 1999), published in January 1999, and from the literature cited therein. </div>
</li> <li> <para-num num="[0015]"> </para-num> <div class="description-line" id="p-0016" num="0015"> Fluorescently labelled probes are often used for the scanning of immobilised DNA arrays. The simple attachment of Cy3 and Cy5 dyes to the 5′-OH of the specific probe are particularly suitable for fluorescence labels. The detection of the fluorescence of the hybridised probes may be carried out, for example via a confocal microscope. Cy3 and Cy5 dyes, besides many others, are commercially available. </div>
</li> <li> <para-num num="[0016]"> </para-num> <div class="description-line" id="p-0017" num="0016"> Matrix Assisted Laser Desorption Ionisation Mass Spectrometry (MALDI-TOF) is a very efficient development for the analysis of biomolecules (Karas M, Hillenkamp F. Laser desorption ionisation of proteins with molecular masses exceeding 10,000 daltons. Anal Chem. 1988 Oct. 15;60(20):2299-301). An analyte is embedded in a light-absorbing matrix. The matrix is evaporated by a short laser pulse thus transporting the analyte molecule into the vapour phase in an unfragmented manner. The analyte is ionised by collisions with matrix molecules. An applied voltage accelerates the ions into a field-free flight tube. Due to their different masses, the ions are accelerated at different rates. Smaller ions reach the detector sooner than bigger ones. </div>
</li> <li> <para-num num="[0017]"> </para-num> <div class="description-line" id="p-0018" num="0017"> MALDI-TOF spectrometry is excellently suited to the analysis of peptides and proteins. The analysis of nucleic acids is somewhat more difficult (Gut I G, Beck S. DNA and Matrix Assisted Laser Desorption Ionisation Mass Spectrometry. Current Innovations and Future Trends. 1995, 1; 147-57). The sensitivity to nucleic acids is approximately 100 times worse than to peptides and decreases disproportionally with increasing fragment size. For nucleic acids having a multiply negatively charged backbone, the ionisation process via the matrix is considerably less efficient. In MALDI-TOF spectrometry, the selection of the matrix plays an eminently important role. For the desorption of peptides, several very efficient matrixes have been found which produce a very fine crystallisation. There are now several responsive matrixes for DNA, however, the difference in sensitivity has not been reduced. The difference in sensitivity can be reduced by chemically modifying the DNA in such a manner that it becomes more similar to a peptide. Phosphorothioate nucleic acids in which the usual phosphates of the backbone are substituted with thiophosphates can be converted into a charge-neutral DNA using simple alkylation chemistry (Gut I G, Beck S. A procedure for selective DNA alkylation and detection by mass spectrometry. Nucleic Acids Res. 1995 Apr. 25;23(8):1367-73). The coupling of a charge tag to this modified DNA results in an increase in sensitivity to the same level as that found for peptides. A further advantage of charge tagging is the increased stability of the analysis against impurities which make the detection of unmodified substrates considerably more difficult. </div>
</li> <li> <para-num num="[0018]"> </para-num> <div class="description-line" id="p-0019" num="0018"> Genomic DNA is obtained from DNA of cell, tissue or other test samples using standard methods. This standard methodology is found in references such as Fritsch and Maniatis eds., Molecular Cloning: A Laboratory Manual, 1989. </div>
</li> <heading id="h-0002">DESCRIPTION </heading>
<li> <para-num num="[0019]"> </para-num> <div class="description-line" id="p-0020" num="0019"> The invention provides a method and nucleic acids for the analysis of biological samples for features associated with the development of breast cell proliferative disorders. The invention is characterised in that the nucleic acid of at least one member of the group of genes according to Table 1 is/are contacted with a reagent or series of reagents capable of distinguishing between methylated and non methylated CpG dinucleotides within the genomic sequence of interest. </div>
</li> <li> <para-num num="[0020]"> </para-num> <div class="description-line" id="p-0021" num="0020"> The present invention makes available a method for ascertaining genetic and/or epigenetic parameters of genomic DNA. The method is for use in the improved diagnosis, treatment and monitoring of breast cell proliferative disorders, for example by enabling the improved identification of and differentiation between subclasses of said disorder and the genetic predisposition to said disorders. The invention presents improvements over the state of the art in that it enables a highly specific classification of breast cell proliferative disorders, thereby allowing for improved and informed treatment of patients. </div>
</li> <li> <para-num num="[0021]"> </para-num> <div class="description-line" id="p-0022" num="0021"> Furthermore, the method enables the analysis of cytosine methylations and single nucleotide polymorphisms. </div>
</li> <li> <para-num num="[0022]"> </para-num> <div class="description-line" id="p-0023" num="0022"> The genes that form the basis of the present invention are preferably to be used to form a “gene panel”, i.e. a collection comprising the particular genetic sequences of the present invention and/or their respective informative methylation sites. The formation of gene panels allows for a quick and specific analysis of specific aspects of breast cancer. The gene panel(s) as described and employed in this invention can be used with surprisingly high efficiency for the diagnosis, treatment and monitoring of and the analysis of a predisposition to breast cell proliferative disorders. </div>
</li> <li> <para-num num="[0023]"> </para-num> <div class="description-line" id="p-0024" num="0023"> In addition, the use of multiple CpG sites from a diverse array of genes allows for a relatively high degree of sensitivity and specificity in comparison to single gene diagnostic and detection tools. </div>
</li> <li> <para-num num="[0024]"> </para-num> <div class="description-line" id="p-0025" num="0024"> The object of the invention is achieved by means of analysis of the methylation patterns of one or more sequences taken from the group comprising Seq. ID No. 1 through Seq. ID No. 73 and Seq. ID No. 366 according to Table 1. In a preferred embodiment said method is achieved by contacting said nucleic acid sequences in a biological sample obtained from a subject with at least one reagent or a series of reagents, wherein said reagent or series of reagents, distinguishes between methylated and non methylated CpG dinucleotides within the respective target nucleic acid(s), i.e. Seq. ID No. 1 through Seq. ID No. 73 and Seq. ID No. 366. </div>
</li> <li> <para-num num="[0025]"> </para-num> <div class="description-line" id="p-0026" num="0025"> In a preferred embodiment, the method comprises the following steps: </div>
</li> <li> <para-num num="[0026]"> </para-num> <div class="description-line" id="p-0027" num="0026"> In the first step of the method the genomic DNA sample must be isolated from sources such as cell lines, tissue or blood samples. Extraction may be by means that are standard to one skilled in the art, these include the use of detergent lysates, sonification and vortexing with glass beads. Once the nucleic acids have been extracted the genomic double stranded DNA is used in the analysis. </div>
</li> <li> <para-num num="[0027]"> </para-num> <div class="description-line" id="p-0028" num="0027"> In a preferred embodiment the DNA may be cleaved prior to the next step of the method, this may be by any means standard in the state of the art, in particular, but not limited to, with restriction endonucleases. </div>
</li> <li> <para-num num="[0028]"> </para-num> <div class="description-line" id="p-0029" num="0028"> In the second step of the method, the genomic DNA sample is treated in such a manner that cytosine bases which are unmethylated at the 5′-position are converted to uracil, thymine, or another base which is dissimilar to cytosine in terms of hybridisation behaviour. This will be understood as ‘pretreatment’ hereinafter. </div>
</li> <li> <para-num num="[0029]"> </para-num> <div class="description-line" id="p-0030" num="0029"> The above described treatment of genomic DNA is preferably carried out with bisulfite (sulfite, disulfite) and subsequent alkaline hydrolysis which results in a conversion of non-methylated cytosine nucleobases to uracil or to another base which is dissimilar to cytosine in terms of base pairing behaviour. If bisulfite solution is used for the reaction, then an addition takes place at the non-methylated cytosine bases. Moreover, a denaturating reagent or solvent as well as a radical interceptor must be present. A subsequent alkaline hydrolysis then gives rise to the conversion of non-methylated cytosine nucleobases to uracil. The converted DNA is then used for the detection of methylated cytosines. </div>
</li> <li> <para-num num="[0030]"> </para-num> <div class="description-line" id="p-0031" num="0030"> Fragments of the pretreated DNA are amplified, using sets of primer oligonucleotides, and a preferably heat-stable, polymerase. Because of statistical and practical considerations, preferably more than ten different fragments having a length of 100-2000 base pairs are amplified. The amplification of several DNA segments can be carried out simultaneously in one and the same reaction vessel. Usually, the amplification is carried out by means of a polymerase chain reaction (PCR). </div>
</li> <li> <para-num num="[0031]"> </para-num> <div class="description-line" id="p-0032" num="0031"> The design of such primers is obvious to one skilled in the art. These should include at least two oligonucleotides whose sequences are each reverse complementary or identical to an at least 18 base-pair long segment of the base sequences specified in the appendix (SEQ ID NO: 74 through SEQ ID NO: 365 and SEQ ID NO: 367 through 370). Said primer oligonucleotides are preferably characterised in that they do not contain any CpG dinucleotides. In a particularly preferred embodiment of the method, the sequence of said primer oligonucleotides are designed so as to selectively anneal to and amplify, only the breast cell specific DNA of interest, thereby minimising the amplification of background or non relevant DNA. In the context of the present invention, background DNA is taken to mean genomic DNA which does not have a relevant tissue specific methylation pattern, in this case, the relevant tissue being breast cells, both healthy and diseased. </div>
</li> <li> <para-num num="[0032]"> </para-num> <div class="description-line" id="p-0033" num="0032"> According to the present invention, it is preferred that at least one primer oligonucleotide is bound to a solid phase during amplification. The different oligonucleotide and/or PNA-oligomer sequences can be arranged on a plane solid phase in the form of a rectangular or hexagonal lattice, the solid phase surface preferably being composed of silicon, glass, polystyrene, aluminium, steel, iron, copper, nickel, silver, or gold, it being possible for other materials such as nitrocellulose or plastics to be used as well. </div>
</li> <li> <para-num num="[0033]"> </para-num> <div class="description-line" id="p-0034" num="0033"> The fragments obtained by means of the amplification may carry a directly or indirectly detectable label. Preferred are labels in the form of fluorescence labels, radionuclides, or detachable molecule fragments having a typical mass which can be detected in a mass spectrometer, it being preferred that the fragments that are produced have a single positive or negative net charge for better detectability in the mass spectrometer. The detection may be carried out and visualised by means of matrix assisted laser desorption/ionisation mass spectrometry (MALDI) or using electron spray mass spectrometry (ESI). </div>
</li> <li> <para-num num="[0034]"> </para-num> <div class="description-line" id="p-0035" num="0034"> In the next step the nucleic acid amplificates are analysed in order to determine the methylation status of the genomic DNA prior to treatment. </div>
</li> <li> <para-num num="[0035]"> </para-num> <div class="description-line" id="p-0036" num="0035"> The post treatment analysis of the nucleic acids may be carried out using alternative methods. Several methods for the methylation status specific analysis of the treated nucleic acids are described below, other alternative methods will be obvious to one skilled in the art. </div>
</li> <li> <para-num num="[0036]"> </para-num> <div class="description-line" id="p-0037" num="0036"> Using several methods known in the art the analysis may be carried out during the amplification step of the method. In one such embodiment, the methylation status of preselected CpG positions within the nucleic acids comprising Seq. ID No. 1 through Seq. ID No. 73 may be detected by use of methylation specific primer oligonucleotides. This technique has been described in U.S. Pat. No. 6,265,171 to Herman. The use of methylation status specific primers for the amplification of bisulphite treated DNA allows the differentiation between methylated and unmethylated nucleic acids. MSP primers pairs contain at least one primer which hybridises to a bisulphite treated CpG dinucleotide. Therefore the sequence of said primers comprises at least one CG, TG or CA dinucleotide. MSP primers specific for non methylated DNA contain a ‘T’ at the 3′ position of the C position in the CpG. According to the present invention, it is therefore preferred that the base sequence of said primers is required to comprise a sequence having a length of at least 9 nucleotides which hybridises to a pretreated nucleic acid sequence according to Seq. ID No. 74 to Seq. ID No. 365 and sequences complementary thereto wherein the base sequence of said oligomers comprises at least one CG, TG or CA dinucleotide. </div>
</li> <li> <para-num num="[0037]"> </para-num> <div class="description-line" id="p-0038" num="0037"> In one embodiment of the method the methylation status of the CpG positions may be determined by means of hybridisation analysis. In this embodiment of the method the amplificates obtained in the second step of the method are hybridised to an array or a set of oligonucleotides and/or PNA probes. In this context, the hybridisation takes place in the manner described as follows. The set of probes used during the hybridisation is preferably composed of at least 4 oligonucleotides or PNA-oligomers. In the process, the amplificates serve as probes which hybridise to oligonucleotides previously bonded to a solid phase. The non-hybridised fragments are subsequently removed. Said oligonucleotides contain at least one base sequence having a length of 10 nucleotides which is reverse complementary or identical to a segment of the base sequences specified in the appendix, the segment containing at least one CpG or TpG dinucleotide. In a further preferred embodiment the cytosine of the CpG dinucleotide, or in the case of TpG, the thiamine, is the 5<sup>th </sup>to 9<sup>th </sup>nucleotide from the 5′-end of the 10-mer. One oligonucleotide exists for each CpG or TpG dinucleotide. </div>
</li> <li> <para-num num="[0038]"> </para-num> <div class="description-line" id="p-0039" num="0038"> The non-hybridised amplificates are then removed. In the final step of the method, the hybridised amplificates are detected. In this context, it is preferred that labels attached to the amplificates are identifiable at each position of the solid phase at which an oligonucleotide sequence is located. </div>
</li> <li> <para-num num="[0039]"> </para-num> <div class="description-line" id="p-0040" num="0039"> In a further embodiment of the method the methylation status of the CpG positions may be ascertained by means of oligonucleotide probes that are hybridised to the treated DNA concurrently with the PCR amplification primers (wherein said primers may either be methylation specific or standard). </div>
</li> <li> <para-num num="[0040]"> </para-num> <div class="description-line" id="p-0041" num="0040"> A particularly preferred embodiment of this method is the use of fluorescence-based Real Time Quantitative PCR (Heid et al., Genome Res. 6:986-994, 1996) employing a dual-labelled fluorescent oligonucleotide probe (TaqMan™ PCR, using an ABI Prism 7700 Sequence Detection System, Perkin Elmer Applied Biosystems, Foster City, Calif.). The TaqMan™ PCR reaction employs the use of a nonextendible interrogating oligonucleotide, called a TaqMan™ probe, which is designed to hybridise to a GpC-rich sequence located between the forward and reverse amplification primers. The TaqMan™ probe further comprises a fluorescent “reporter moiety” and a “quencher moiety” covalently bound to linker moieties (e.g., phosphoramidites) attached to the nucleotides of the TaqMan™ oligonucleotide. For analysis of methylation within nucleic acids subsequent to bisulphite treatment it is required that the probe be methylation specific, as described in U.S. Pat. No. 6,331,393, (hereby incorporated by reference) also known as the Methyl Light assay. Variations on the TaqMan™ detection methodology that are also suitable for use with the described invention include the use of dual probe technology (Lightcycler™ or fluorescent amplification primers (Sunrise™ technology). Both these techniques may be adapted in a manner suitable for use with bisulphite treated DNA, and moreover for methylation analysis within CpG dinucleotides. </div>
</li> <li> <para-num num="[0041]"> </para-num> <div class="description-line" id="p-0042" num="0041"> A further suitable method for the use of probe oligonucleotides for the assessment of methylation by analysis of bisulphite treated nucleic acids is the use of blocker oligonucleotides. The use of such oligonucleotides has been described in BioTechniques 23(4), 1997, 714-720 D. Yu, M. Mukai, Q. Liu, C. Steinman. Blocking probe oligonucleotides are hybridised to the bisulphite treated nucleic acid concurrently with the PCR primers. PCR amplification of the nucleic acid is terminated at the 5′ position of the blocking probe, thereby amplification of a nucleic acid is suppressed wherein the complementary sequence to the blocking probe is present. The probes may be designed to hybridise to the bisulphite treated nucleic acid in a methylation status specific manner. For example, for detection of methylated nucleic acids within a population of unmethylated nucleic acids suppression of the amplification of nucleic acids which are unmethylated at the position in question would be carried out by the use of blocking probes comprising a ‘CG’ at the position in question, as opposed to a ‘CA’. </div>
</li> <li> <para-num num="[0042]"> </para-num> <div class="description-line" id="p-0043" num="0042"> In a further preferred embodiment of the method the determination of the methylation status of the CpG positions is carried out by the use of template directed oligonucleotide extension, such as MS SNuPE as described by Gonzalgo and Jones (Nucleic Acids Res. 25:2529-2531). </div>
</li> <li> <para-num num="[0043]"> </para-num> <div class="description-line" id="p-0044" num="0043"> In a further embodiment of the method the determination of the methylation status of the CpG positions is enabled by sequencing and subsequent sequence analysis of the amplificate generated in the second step of the method (Sanger F., et al., 1977 PNAS USA 74: 5463-5467). </div>
</li> <li> <para-num num="[0044]"> </para-num> <div class="description-line" id="p-0045" num="0044"> A further embodiment of the invention is a method for the analysis of the methylation status of genomic DNA without the need for pretreatment. In the first and second steps of the method the genomic DNA sample must be obtained and isolated from tissue or cellular sources. Such sources may include cell lines, histological slides, body fluids, or tissue embedded in paraffin. Extraction may be by means that are standard to one skilled in the art, these include the use of detergent lysates, sonification and vortexing with glass beads. Once the nucleic acids have been extracted the genomic double stranded DNA is used in the analysis. </div>
</li> <li> <para-num num="[0045]"> </para-num> <div class="description-line" id="p-0046" num="0045"> In a preferred embodiment the DNA may be cleaved prior to the treatment, this may be any means standard in the state of the art, in particular with restriction endonucleases. In the third step, the DNA is then digested with one or more methylation sensitive restriction enzymes. The digestion is carried out such that hydrolysis of the DNA at the restriction site is informative of the methylation status of a specific CpG dinucleotide. </div>
</li> <li> <para-num num="[0046]"> </para-num> <div class="description-line" id="p-0047" num="0046"> In a preferred embodiment the restriction fragments are amplified. In a further preferred embodiment this is carried out using the polymerase chain reaction. </div>
</li> <li> <para-num num="[0047]"> </para-num> <div class="description-line" id="p-0048" num="0047"> In the final step the amplificates are detected. The detection may be by any means standard in the art, for example, but not limited to, gel electrophoresis analysis, hybridisation analysis, incorporation of detectable tags within the PCR products, DNA array analysis, MALDI or ESI analysis. </div>
</li> <li> <para-num num="[0048]"> </para-num> <div class="description-line" id="p-0049" num="0048"> The aforementioned method is preferably used for ascertaining genetic and/or epigenetic parameters of genomic DNA. </div>
</li> <li> <para-num num="[0049]"> </para-num> <div class="description-line" id="p-0050" num="0049"> In order to enable this method, the invention further provides the modified DNA of genes according to Table 1, as well as oligonucleotides and/or PNA-oligomers for detecting cytosine methylations within said genes. The present invention is based on the discovery that genetic and epigenetic parameters and, in particular, the cytosine methylation patterns of said genomic DNAs are particularly suitable for improved diagnosis, treatment and monitoring of breast cell proliferative disorders. Furthermore, the invention enables the differentiation between different subclasses of breast cell proliferative disorders or detection of a predisposition to breast cell proliferative disorders. </div>
</li> <li> <para-num num="[0050]"> </para-num> <div class="description-line" id="p-0051" num="0050"> The nucleic acids according to the present invention can be used for the analysis of genetic and/or epigenetic parameters of genomic DNA. </div>
</li> <li> <para-num num="[0051]"> </para-num> <div class="description-line" id="p-0052" num="0051"> This objective according to the present invention is achieved using a nucleic acid containing a sequence of at least 18 bases in length of the pretreated genomic DNA according to one of SEQ ID NO: 74 through SEQ ID NO: 365 and SEQ ID NO 367 through 370 and sequences complementary thereto. </div>
</li> <li> <para-num num="[0052]"> </para-num> <div class="description-line" id="p-0053" num="0052"> The modified nucleic acids could heretofore not be connected with the ascertainment of disease relevant genetic and epigenetic parameters. </div>
</li> <li> <para-num num="[0053]"> </para-num> <div class="description-line" id="p-0054" num="0053"> The object of the present invention is further achieved by an oligonucleotide or oligomer for the analysis of pretreated DNA, for detecting the genomic cytosine methylation state, said oligonucleotide containing at least one base sequence having a length of at least 10 nucleotides which hybridises to a pretreated genomic DNA according to SEQ ID NO: 74 through 365 and SEQ ID NO 367 through 370. The oligomer probes according to the present invention constitute important and effective tools which, for the first time, make it possible to ascertain specific genetic and epigenetic parameters during the analysis of biological samples for features associated with the development of breast cell proliferative disorders. Said oligonucleotides allow the improved diagnosis, treatment and monitoring of breast cell proliferative disorders and detection of the predisposition to said disorders. Furthermore, they allow the differentiation of different subclasses of breast carcinomas. The base sequence of the oligomers preferably contains at least one CpG or TpG dinucleotide. The probes may also exist in the form of a PNA (peptide nucleic acid) which has particularly preferred pairing properties. Particularly preferred are oligonucleotides according to the present invention in which the cytosine of the CpG dinucleotide is within the middle third of said oligonucleotide e.g. the 5<sup>th</sup>-9<sup>th </sup>nucleotide from the 5′-end of a 13-mer oligonucleotide; or in the case of PNA-oligomers, it is preferred for the cytosine of the CpG dinucleotide to be the 4<sup>th</sup>-6<sup>th </sup>nucleotide from the 5′-end of the 9-mer. </div>
</li> <li> <para-num num="[0054]"> </para-num> <div class="description-line" id="p-0055" num="0054"> The oligomers according to the present invention are normally used in so called “sets” which contain at least one oligomer for each of the CpG dinucleotides within SEQ ID NO: 74 through SEQ ID NO: 365 and SEQ ID NO 367 through 370. </div>
</li> <li> <para-num num="[0055]"> </para-num> <div class="description-line" id="p-0056" num="0055"> In the case of the sets of oligonucleotides according to the present invention, it is preferred that at least one oligonucleotide is bound to a solid phase. It is further preferred that all the oligonucleotides of one set are bound to a solid phase. </div>
</li> <li> <para-num num="[0056]"> </para-num> <div class="description-line" id="p-0057" num="0056"> The present invention further relates to a set of at least 10 n (oligonucleotides and/or PNA-oligomers) used for detecting the cytosine methylation state of genomic DNA, by analysis of said sequence or treated versions of said sequence (according to SEQ ID NO: 1 through SEQ ID NO: 366 and sequences complementary thereto). These probes enable improved diagnosis, treatment and monitoring of breast cell proliferative disorders. In particular they enable the differentiation between different sub classes of breast cell proliferative disorders and the detection of a predisposition to said disorders. </div>
</li> <li> <para-num num="[0057]"> </para-num> <div class="description-line" id="p-0058" num="0057"> The set of oligomers may also be used for detecting single nucleotide polymorphisms (SNPs) by analysis of said sequence or treated versions of said sequence according to one of SEQ ID NO: 1 through SEQ ID NO: 370. </div>
</li> <li> <para-num num="[0058]"> </para-num> <div class="description-line" id="p-0059" num="0058"> According to the present invention, it is preferred that an arrangement of different oligonucleotides and/or PNA-oligomers (a so-called “array”) made available by the present invention is present in a manner that it is likewise bound to a solid phase. This array of different oligonucleotide- and/or PNA-oligomer sequences can be characterised in that it is arranged on the solid phase in the form of a rectangular or hexagonal lattice. The solid phase surface is preferably composed of silicon, glass, polystyrene, aluminium, steel, iron, copper, nickel, silver, or gold. However, nitrocellulose as well as plastics such as nylon which can exist in the form of pellets or also as resin matrices are suitable alternatives. </div>
</li> <li> <para-num num="[0059]"> </para-num> <div class="description-line" id="p-0060" num="0059"> Therefore, a further subject matter of the present invention is a method for manufacturing an array fixed to a carrier material for the improved diagnosis, treatment and monitoring of breast cell proliferative disorders, the differentiation between different subclasses of breast carcinomas and/or detection of the predisposition to breast cell proliferative disorders. In said method at least one oligomer according to the present invention is coupled to a solid phase. Methods for manufacturing such arrays are known, for example, from U.S. Pat. No. 5,744,305 by means of solid-phase chemistry and photolabile protecting groups. </div>
</li> <li> <para-num num="[0060]"> </para-num> <div class="description-line" id="p-0061" num="0060"> A further subject matter of the present invention relates to a DNA chip for the improved diagnosis, treatment and monitoring of breast cell proliferative disorders. Furthermore the DNA chip enables detection of the predisposition to breast cell proliferative disorders and the differentiation between different subclasses of breast carcinomas. The DNA chip contains at least one nucleic acid according to the present invention. DNA chips are known, for example, in U.S. Pat. No. 5,837,832. </div>
</li> <li> <para-num num="[0061]"> </para-num> <div class="description-line" id="p-0062" num="0061"> Moreover, a subject matter of the present invention is a kit which may be composed, for example, of a bisulfite-containing reagent, a set of primer oligonucleotides containing at least two oligonucleotides whose sequences in each case correspond to or are complementary to a 18 base long segment of the base sequences specified in the appendix (SEQ ID NO: 74 through SEQ ID NO: 365 and SEQ ID NO 367 through 370), oligonucleotides and/or PNA-oligomers as well as instructions for carrying out and evaluating the described method. </div>
</li> <li> <para-num num="[0062]"> </para-num> <div class="description-line" id="p-0063" num="0062"> In a further preferred embodiment said kit may further comprise standard reagents for performing a CpG position specific methylation analysis wherein said analysis comprises one or more of the following techniques: MS-SNuPE, MSP, Methyl light, Heavy Methyl, and nucleic acid sequencing. However, a kit along the lines of the present invention can also contain only part of the aforementioned components. </div>
</li> <li> <para-num num="[0063]"> </para-num> <div class="description-line" id="p-0064" num="0063"> The oligomers according to the present invention or arrays thereof as well as a kit according to the present invention are intended to be used for the improved diagnosis, treatment and monitoring of breast cell proliferative disorders. Furthermore the use of said inventions extends to the differentiation between different subclasses of breast carcinomas and detection of the predisposition to breast cell proliferative disorders. According to the present invention, the method is preferably used for the analysis of important genetic and/or epigenetic parameters within genomic DNA, in particular for use in improved diagnosis, treatment and monitoring of breast cell proliferative disorders, detection of the predisposition to said disorders and the differentiation between subclasses of said disorders. </div>
</li> <li> <para-num num="[0064]"> </para-num> <div class="description-line" id="p-0065" num="0064"> The methods according to the present invention are used, for example, for improved diagnosis, treatment and monitoring of breast cell proliferative disorders progression, detection of the predisposition to said disorders and the differentiation between subclasses of said disorders. </div>
</li> <li> <para-num num="[0065]"> </para-num> <div class="description-line" id="p-0066" num="0065"> The present invention moreover relates to the diagnosis and/or prognosis of events which are disadvantageous or relevant to patients or individuals in which important genetic and/or epigenetic parameters within genomic DNA, said parameters obtained by means of the present invention may be compared to another set of genetic and/or epigenetic parameters, the differences serving as the basis for the diagnosis and/or prognosis of events which are disadvantageous or relevant to patients or individuals. </div>
</li> <li> <para-num num="[0066]"> </para-num> <div class="description-line" id="p-0067" num="0066"> The genes and/or the nucleic acids that form the basis of the present invention can be used to form a “gene panel”, i.e. a collection comprising the particular genetic sequences of the present invention and/or their respective informative methylation sites. The formation of gene panels allow for a quick and specific analysis of the disorders they are related with. The gene panel(s) as described and employed in this invention can be used with surprisingly high efficiency for the diagnosis, treatment and monitoring of and the analysis of a predisposition to the disorders described herein, based on the analysis of the methylation status of said panels. </div>
</li> <li> <para-num num="[0067]"> </para-num> <div class="description-line" id="p-0068" num="0067"> The use of multiple and selective CpG sites from a diverse array of genes regulating breast cell proliferative disorders, in addition allows for a surprisingly high degree of sensitivity and specificity in comparison to single gene diagnostic and detection tools. Furthermore, as compared to many the panel as described herein may be adapted for use in the analysis of multiple diseases all affected by regulating breast cell proliferative disorders. </div>
</li> <li> <para-num num="[0068]"> </para-num> <div class="description-line" id="p-0069" num="0068"> In the context of the present invention the term “hybridisation” is to be understood as a bond of an oligonucleotide to a completely complementary sequence along the lines of the Watson-Crick base pairings in the sample DNA, forming a duplex structure. </div>
</li> <li> <para-num num="[0069]"> </para-num> <div class="description-line" id="p-0070" num="0069"> In the context of the present invention, “genetic parameters” are mutations and polymorphisms of genomic DNA and sequences further required for their regulation. To be designated as mutations are, in particular, insertions, deletions, point mutations, inversions and polymorphisms and, particularly preferred, SNPs (single nucleotide polymorphisms). </div>
</li> <li> <para-num num="[0070]"> </para-num> <div class="description-line" id="p-0071" num="0070"> In the context of the present invention, “epigenetic parameters” are, in particular, cytosine methylations and further modifications of DNA bases of genomic DNA and sequences further required for their regulation. Further epigenetic parameters include, for example, the acetylation of histones which, cannot be directly analysed using the described method but which, in turn, correlates with the DNA methylation. </div>
</li> <li> <para-num num="[0071]"> </para-num> <div class="description-line" id="p-0072" num="0071"> In the following, the present invention will be explained in greater detail on the basis of the sequences, the tables, and the examples without being limited thereto. SEQ ID NO: 1 through SEQ ID NO: 73 and SEQ ID NO: 366 represent 5′ and/or regulatory regions and/or CpG rich regions of the genes according to Table 1. These sequences are derived from Genbank and will be taken to include all minor variations of the sequence material which are currently unforeseen, for example, but not limited to, minor deletions and SNPs. </div>
</li> <li> <para-num num="[0072]"> </para-num> <div class="description-line" id="p-0073" num="0072"> SEQ ID NO: 74 through SEQ ID NO: 365 and SEQ ID NO 367 through 370 exhibit the pretreated sequence of DNA derived from the genes according to Table 1. These sequences will be taken to include all minor variations of the sequence material which are currently unforeseen, for example, but not limited to, minor deletions and SNPs. </div>
</li> <li> <para-num num="[0073]"> </para-num> <div class="description-line" id="p-0074" num="0073"> SEQ ID NO: 371 through SEQ ID NO: 396 exhibit the sequences of primers and other oligonucleotides used in the analysis of a selected panel of the genes of Table 1, as described in the embodiments of the method according to examples 1 to 4.</div>
</li> <li> <para-num num="[0074]"> </para-num> <div class="description-line" id="p-0075" num="0074"> <figref idrefs="DRAWINGS">FIG. 1</figref> shows the analysis of bisulphite-treated DNA using the MethylLight and Heavy Methyl assays, performed according to Examples 2 and 3 respectively. Results of the HeavyMethyl assay are shown on the left hand bar chart and results of the MethyLight assay are shown on the right hand bar chart. The Y-axis shows the percentage of methylation at the CpG positions covered by the probes. The dark black bar (“A” in the legend) corresponds to tumour samples, whereas the white bar (“B”) corresponds to normal control tissue. Significantly, the tumour samples are substantially hypermethylated relative to normal control tissue. </div>
</li> <li> <para-num num="[0075]"> </para-num> <div class="description-line" id="p-0076" num="0075"> <figref idrefs="DRAWINGS">FIG. 2</figref> shows the level of methylation in breast tumour and healthy tissues as assessed according to Example 3 by means of the Heavy Methyl assay. The Y-axis shows the degree of methylation within the region of the Calcitonin gene investigated. Tumour samples are represented by black diamonds, and normal breast tissue samples by white squares. As can be seen from the results, a significantly higher degree of methylation (hypermethylation) was observed in tumour samples relative to normal tissue samples.</div>
</li> <li> <para-num num="[0076]"> </para-num> <div class="description-line" id="p-0077" num="0076">
<tables id="TABLE-US-00001" num="1">
<patent-tables colsep="0" frame="none" pgwide="1" rowsep="0">
<table align="left" class="description-table" cols="1" colsep="0" rowsep="0" width="100%">
<thead>
<tr class="description-tr">
<td align="center" class="description-td" colspan="1" nameend="1" namest="1">TABLE 1</td>
</tr>
</thead>
<tbody><tr class="description-tr">
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td align="center" class="description-td" colspan="1" nameend="1" namest="1" rowsep="1"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">Description of genes comprising panel.</td>
</tr>
</tbody></table>
<table align="left" class="description-table" cols="6" colsep="0" rowsep="0" width="100%">
<tbody><tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">Seq ID</td>
<td class="description-td">Seq. ID No.</td>
</tr>
<tr class="description-tr">
<td class="description-td">Accession</td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">Seq. ID No.</td>
<td class="description-td">No. Treated</td>
<td class="description-td">Treated</td>
</tr>
<tr class="description-tr">
<td class="description-td">No.</td>
<td class="description-td">Gene name</td>
<td class="description-td">Description</td>
<td class="description-td">(Genomic)</td>
<td class="description-td">(methylated)</td>
<td class="description-td">(unmethylated)</td>
</tr>
<tr class="description-tr">
<td align="center" class="description-td" colspan="6" nameend="6" namest="1" rowsep="1"> </td>
</tr>
</tbody></table>
<table align="left" class="description-table" cols="6" colsep="0" rowsep="0" width="100%">
<tbody><tr class="description-tr">
<td class="description-td">NM_001846</td>
<td class="description-td">COL4A2</td>
<td class="description-td">collagen, type IV,</td>
<td class="description-td">1</td>
<td class="description-td">74, 75</td>
<td class="description-td">82, 83</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">alpha 2</td>
</tr>
<tr class="description-tr">
<td class="description-td">NM_004663</td>
<td class="description-td">GTPase</td>
<td class="description-td">rab11a GTPase</td>
<td class="description-td">2</td>
<td class="description-td">76, 77</td>
<td class="description-td">84, 85</td>
</tr>
<tr class="description-tr">
<td class="description-td">NM_001218</td>
<td class="description-td">CA XII</td>
<td class="description-td">carbonic anhydrase</td>
<td class="description-td">3</td>
<td class="description-td">78, 79</td>
<td class="description-td">86, 87</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">XII precursor</td>
</tr>
<tr class="description-tr">
<td class="description-td">AF073519</td>
<td class="description-td">SERF1A</td>
<td class="description-td">small EDRK-rich</td>
<td class="description-td">4</td>
<td class="description-td">80, 81</td>
<td class="description-td">88, 89</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">factor 1A (telomeric)</td>
</tr>
<tr class="description-tr">
<td class="description-td">NM_006526</td>
<td class="description-td">ZNF217</td>
<td class="description-td">Zinc Finger Protein</td>
<td class="description-td">5</td>
<td class="description-td">90, 91</td>
<td class="description-td">96, 97</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">217</td>
</tr>
<tr class="description-tr">
<td class="description-td">AJ293618</td>
<td class="description-td">JCL-1</td>
<td class="description-td">Human</td>
<td class="description-td">6</td>
<td class="description-td">92, 93</td>
<td class="description-td">98, 99</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">hepatocellular</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">carcinoma associated</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">protein</td>
</tr>
<tr class="description-tr">
<td class="description-td">NM_007194</td>
<td class="description-td">CHK2</td>
<td class="description-td">SERINE/THREONINE-</td>
<td class="description-td">7</td>
<td class="description-td">94, 95</td>
<td class="description-td">100, 101</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">PROTEIN</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">KINASE CHK2 (EC</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">2.7.1.—)(CDS1).</td>
</tr>
<tr class="description-tr">
<td class="description-td">D14034</td>
<td class="description-td">zinc-alpha2-</td>
<td class="description-td"> </td>
<td class="description-td">8</td>
<td class="description-td">102, 103</td>
<td class="description-td">114, 115</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">glycoprotein</td>
</tr>
<tr class="description-tr">
<td class="description-td">X03473</td>
<td class="description-td">Histone H1</td>
<td class="description-td">Human gene for</td>
<td class="description-td">9</td>
<td class="description-td">104, 105</td>
<td class="description-td">116, 117</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">histone H1(0)</td>
</tr>
<tr class="description-tr">
<td class="description-td">NM_000127</td>
<td class="description-td">EXT1</td>
<td class="description-td">exostoses (multiple) 1</td>
<td class="description-td">10</td>
<td class="description-td">106, 107</td>
<td class="description-td">118, 119</td>
</tr>
<tr class="description-tr">
<td class="description-td">NM_000436</td>
<td class="description-td">OXCT</td>
<td class="description-td">3-oxoacid CoA</td>
<td class="description-td">11</td>
<td class="description-td">108, 109</td>
<td class="description-td">120, 121</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">transferase</td>
</tr>
<tr class="description-tr">
<td class="description-td">NM_000599</td>
<td class="description-td">IGFBP5</td>
<td class="description-td">insulin-like growth</td>
<td class="description-td">12</td>
<td class="description-td">110, 111</td>
<td class="description-td">122, 123</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">factor binding</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">protein 5</td>
</tr>
<tr class="description-tr">
<td class="description-td">NM_000849</td>
<td class="description-td">GSTM3</td>
<td class="description-td">glutathione S-</td>
<td class="description-td">13</td>
<td class="description-td">112, 113</td>
<td class="description-td">124, 125</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">transferase M3</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">(brain)</td>
</tr>
<tr class="description-tr">
<td class="description-td">NM_001282</td>
<td class="description-td">AP2B1</td>
<td class="description-td">adaptor-related</td>
<td class="description-td">14</td>
<td class="description-td">126, 127</td>
<td class="description-td">138, 139</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">protein complex 2,</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">beta 1 subunit</td>
</tr>
<tr class="description-tr">
<td class="description-td">NM_001809</td>
<td class="description-td">CENPA</td>
<td class="description-td">centromere protein</td>
<td class="description-td">15</td>
<td class="description-td">128, 129</td>
<td class="description-td">140, 141</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">A(17 kD)</td>
</tr>
<tr class="description-tr">
<td class="description-td">NM_002019</td>
<td class="description-td">FLT1</td>
<td class="description-td">fms-related tyrosine</td>
<td class="description-td">16</td>
<td class="description-td">130, 131</td>
<td class="description-td">142, 143</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">kinase 1 (vascular</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">endothelial growth</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">factor/vascular</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">permeability factor</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">receptor)</td>
</tr>
<tr class="description-tr">
<td class="description-td">NM_002073</td>
<td class="description-td">GNAZ</td>
<td class="description-td">guanine nucleotide</td>
<td class="description-td">17</td>
<td class="description-td">132, 133</td>
<td class="description-td">144, 145</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">binding protein (G</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">protein), alpha z</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">polypeptide</td>
</tr>
<tr class="description-tr">
<td class="description-td">NM_002916</td>
<td class="description-td">RFC4</td>
<td class="description-td">replication factor C</td>
<td class="description-td">18</td>
<td class="description-td">134, 135</td>
<td class="description-td">146, 147</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">(activator 1) 4</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">(37 kD)</td>
</tr>
<tr class="description-tr">
<td class="description-td">NM_003239</td>
<td class="description-td">TGFB3</td>
<td class="description-td">transforming growth</td>
<td class="description-td">19</td>
<td class="description-td">136, 137</td>
<td class="description-td">148, 149</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">factor, beta 3</td>
</tr>
<tr class="description-tr">
<td class="description-td">NM_003607</td>
<td class="description-td">PK428</td>
<td class="description-td">Ser-Thr protein</td>
<td class="description-td">20</td>
<td class="description-td">150, 151</td>
<td class="description-td">162, 163</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">kinase related to the</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">myotonic dystrophy</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">protein kinase</td>
</tr>
<tr class="description-tr">
<td class="description-td">NM_003748</td>
<td class="description-td">ALDH4</td>
<td class="description-td">aldehyde</td>
<td class="description-td">21</td>
<td class="description-td">152, 153</td>
<td class="description-td">164, 165</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">dehydrogenase 4</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">(glutamate gamma-</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">semialdehyde</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">dehydrogenase</td>
</tr>
<tr class="description-tr">
<td class="description-td">NM_003875</td>
<td class="description-td">GMPS</td>
<td class="description-td">guanine</td>
<td class="description-td">22</td>
<td class="description-td">154, 155</td>
<td class="description-td">166, 167</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">monphosphate</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">synthetase</td>
</tr>
<tr class="description-tr">
<td class="description-td">NM_003882</td>
<td class="description-td">WISP1</td>
<td class="description-td">WNT1 inducible</td>
<td class="description-td">23</td>
<td class="description-td">156, 157</td>
<td class="description-td">168, 169</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">signaling pathway</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">protein 1</td>
</tr>
<tr class="description-tr">
<td class="description-td">NM_003981</td>
<td class="description-td">PRC1</td>
<td class="description-td">protein regulator of</td>
<td class="description-td">24</td>
<td class="description-td">158, 159</td>
<td class="description-td">170, 171</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">cytokinesis 1</td>
</tr>
<tr class="description-tr">
<td class="description-td">NM_004702</td>
<td class="description-td">CCNE2</td>
<td class="description-td">G1/S-Specific</td>
<td class="description-td">25</td>
<td class="description-td">160, 161</td>
<td class="description-td">172, 173</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">Cyclin E2.</td>
</tr>
<tr class="description-tr">
<td class="description-td">NM_004994</td>
<td class="description-td">MMP9</td>
<td class="description-td">matrix</td>
<td class="description-td">26</td>
<td class="description-td">174, 175</td>
<td class="description-td">184, 185</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">metalloproteinase 9</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">(gelatinase B, 92 kD</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">gelatinase, 92 kD</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">type IV collagenase)</td>
</tr>
<tr class="description-tr">
<td class="description-td">NM_005915</td>
<td class="description-td">MCM6</td>
<td class="description-td">minichromosome</td>
<td class="description-td">27</td>
<td class="description-td">176, 177</td>
<td class="description-td">186, 187</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">maintenance</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">deficient (mis5, S. pombe)6</td>
</tr>
<tr class="description-tr">
<td class="description-td">NM_006101</td>
<td class="description-td">HEC</td>
<td class="description-td">highly expressed in</td>
<td class="description-td">28</td>
<td class="description-td">178, 179</td>
<td class="description-td">188, 189</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">cancer, rich in</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">leucine heptad</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">repeats</td>
</tr>
<tr class="description-tr">
<td class="description-td">NM_006117</td>
<td class="description-td">PECI</td>
<td class="description-td">peroxisomal D3, D2-.</td>
<td class="description-td">29</td>
<td class="description-td">180, 181</td>
<td class="description-td">190, 191</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">enoyl-CoA</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">isomerase</td>
</tr>
<tr class="description-tr">
<td class="description-td">NM_006681</td>
<td class="description-td">NMU</td>
<td class="description-td">neuromedin U</td>
<td class="description-td">30</td>
<td class="description-td">182, 183</td>
<td class="description-td">192, 193</td>
</tr>
<tr class="description-tr">
<td class="description-td">NM_006931</td>
<td class="description-td">SLC2A3</td>
<td class="description-td">solute carrier family</td>
<td class="description-td">31</td>
<td class="description-td">194, 195</td>
<td class="description-td">204, 205</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">2 (facilitated glucose</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">transporter), member 3</td>
</tr>
<tr class="description-tr">
<td class="description-td">NM_007036</td>
<td class="description-td">ESM1</td>
<td class="description-td">endothelial cell-</td>
<td class="description-td">32</td>
<td class="description-td">196, 197</td>
<td class="description-td">206, 207</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">specific molecule 1</td>
</tr>
<tr class="description-tr">
<td class="description-td">NM_007203</td>
<td class="description-td">AKAP2</td>
<td class="description-td">A kinase (PRKA)</td>
<td class="description-td">33</td>
<td class="description-td">198, 199</td>
<td class="description-td">208, 209</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">anchor protein 2</td>
</tr>
<tr class="description-tr">
<td class="description-td">NM_020188</td>
<td class="description-td">DC13</td>
<td class="description-td">DC13 protein</td>
<td class="description-td">34</td>
<td class="description-td">200, 201</td>
<td class="description-td">210, 211</td>
</tr>
<tr class="description-tr">
<td class="description-td">NM_015984</td>
<td class="description-td">UCH37</td>
<td class="description-td">ubiquitin C-terminal</td>
<td class="description-td">35</td>
<td class="description-td">202, 203</td>
<td class="description-td">212, 213</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">hydrolase UCH37</td>
</tr>
<tr class="description-tr">
<td class="description-td">NM_014321</td>
<td class="description-td">ORC6L</td>
<td class="description-td">origin recognition</td>
<td class="description-td">36</td>
<td class="description-td">214, 215</td>
<td class="description-td">218, 219</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">complex, subunit 6</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">(yeast homolog)-like</td>
</tr>
<tr class="description-tr">
<td class="description-td">NM_016448</td>
<td class="description-td">L2DTL</td>
<td class="description-td">L2DTL protein</td>
<td class="description-td">37</td>
<td class="description-td">216, 217</td>
<td class="description-td">220, 221</td>
</tr>
<tr class="description-tr">
<td class="description-td">NM_000038</td>
<td class="description-td">APC</td>
<td class="description-td">adenomatosis</td>
<td class="description-td">38</td>
<td class="description-td">222, 223</td>
<td class="description-td">240, 241</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">polyposis coli</td>
</tr>
<tr class="description-tr">
<td class="description-td">NM_006768</td>
<td class="description-td">BRCA1</td>
<td class="description-td">breast cancer 1, early</td>
<td class="description-td">39</td>
<td class="description-td">224, 225</td>
<td class="description-td">242, 243</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">onset</td>
</tr>
<tr class="description-tr">
<td class="description-td">NM_001759</td>
<td class="description-td">CCND2</td>
<td class="description-td">cyclin D2</td>
<td class="description-td">40</td>
<td class="description-td">226, 227</td>
<td class="description-td">244, 245</td>
</tr>
<tr class="description-tr">
<td class="description-td">NM_004360</td>
<td class="description-td">CDH1</td>
<td class="description-td">cadherin 1, type 1,</td>
<td class="description-td">41</td>
<td class="description-td">228, 229</td>
<td class="description-td">246, 247</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">E-cadherin</td>
</tr>
<tr class="description-tr">
<td class="description-td">NM_001257</td>
<td class="description-td">CDH13</td>
<td class="description-td">H-cadherin</td>
<td class="description-td">42</td>
<td class="description-td">230, 231</td>
<td class="description-td">248, 249</td>
</tr>
<tr class="description-tr">
<td class="description-td">NM_058195,</td>
<td class="description-td">CDKN2A</td>
<td class="description-td">cyclin-dependent</td>
<td class="description-td">43</td>
<td class="description-td">232, 233</td>
<td class="description-td">250, 251</td>
</tr>
<tr class="description-tr">
<td class="description-td">NM_058196,</td>
<td class="description-td"> </td>
<td class="description-td">kinase inhibitor 2A</td>
</tr>
<tr class="description-tr">
<td class="description-td">NM_000077</td>
</tr>
<tr class="description-tr">
<td class="description-td">NM_000125</td>
<td class="description-td">ESR1</td>
<td class="description-td">estrogen receptor 1</td>
<td class="description-td">44</td>
<td class="description-td">234, 235</td>
<td class="description-td">252, 253</td>
</tr>
<tr class="description-tr">
<td class="description-td">NM_004102</td>
<td class="description-td">FABP3</td>
<td class="description-td">fatty acid binding</td>
<td class="description-td">45</td>
<td class="description-td">236, 237</td>
<td class="description-td">254, 255</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">protein 3</td>
</tr>
<tr class="description-tr">
<td class="description-td">NM_002012</td>
<td class="description-td">FHIT</td>
<td class="description-td">fragile histidine triad</td>
<td class="description-td">46</td>
<td class="description-td">238, 239</td>
<td class="description-td">256, 257</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">gene</td>
</tr>
<tr class="description-tr">
<td class="description-td">NM_004004</td>
<td class="description-td">GJB2</td>
<td class="description-td">gap junction protein,</td>
<td class="description-td">47</td>
<td class="description-td">258, 259</td>
<td class="description-td">276, 277</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">beta 2</td>
</tr>
<tr class="description-tr">
<td class="description-td">NM_000852</td>
<td class="description-td">GSTP1</td>
<td class="description-td">Glutathione S-</td>
<td class="description-td">48</td>
<td class="description-td">260, 261</td>
<td class="description-td">278, 279</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">transferase pi</td>
</tr>
<tr class="description-tr">
<td class="description-td">NM_006497</td>
<td class="description-td">HIC-1</td>
<td class="description-td">hypermethylated in</td>
<td class="description-td">49</td>
<td class="description-td">262, 263</td>
<td class="description-td">280, 281</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">cancer 1</td>
</tr>
<tr class="description-tr">
<td class="description-td">NM_002478</td>
<td class="description-td">MYOD1</td>
<td class="description-td">myogenic factor 3</td>
<td class="description-td">50</td>
<td class="description-td">264, 265</td>
<td class="description-td">282, 283</td>
</tr>
<tr class="description-tr">
<td class="description-td">NM_007182</td>
<td class="description-td">RASSF1</td>
<td class="description-td">Ras association</td>
<td class="description-td">51</td>
<td class="description-td">266, 267</td>
<td class="description-td">284, 285</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">(RalGDS/AF-6)</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">domain family 1</td>
</tr>
<tr class="description-tr">
<td class="description-td">NM_133631,</td>
<td class="description-td">ROBO1</td>
<td class="description-td">roundabout, axon</td>
<td class="description-td">52</td>
<td class="description-td">268, 269</td>
<td class="description-td">286, 287</td>
</tr>
<tr class="description-tr">
<td class="description-td">NM_002941</td>
<td class="description-td"> </td>
<td class="description-td">guidance receptor,</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">homolog 1</td>
</tr>
<tr class="description-tr">
<td class="description-td">NM_006142</td>
<td class="description-td">SFN</td>
<td class="description-td">stratifin</td>
<td class="description-td">53</td>
<td class="description-td">270, 271</td>
<td class="description-td">288, 289</td>
</tr>
<tr class="description-tr">
<td class="description-td">NM_013258</td>
<td class="description-td">TMS1</td>
<td class="description-td">target of</td>
<td class="description-td">54</td>
<td class="description-td">272, 273</td>
<td class="description-td">290, 291</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">methylation-induced</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">silencing gene</td>
</tr>
<tr class="description-tr">
<td class="description-td">NM_004906</td>
<td class="description-td">WT1</td>
<td class="description-td">Wilms tumor 1</td>
<td class="description-td">55</td>
<td class="description-td">274, 275</td>
<td class="description-td">292, 293</td>
</tr>
<tr class="description-tr">
<td class="description-td">NM_016359</td>
<td class="description-td">LOC51203</td>
<td class="description-td">clone HQ0310</td>
<td class="description-td">56</td>
<td class="description-td">294, 295</td>
<td class="description-td">310, 311</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">PRO0310p1</td>
</tr>
<tr class="description-tr">
<td class="description-td">NM_014791</td>
<td class="description-td">KIAA0175</td>
<td class="description-td">KIAA0175 gene</td>
<td class="description-td">57 and 58</td>
<td class="description-td">296-299</td>
<td class="description-td">312-315</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">product</td>
</tr>
<tr class="description-tr">
<td class="description-td">NM_016577</td>
<td class="description-td">RAB6B</td>
<td class="description-td">RAB6B, member</td>
<td class="description-td">59</td>
<td class="description-td">300, 301</td>
<td class="description-td">316, 317</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">RAS oncogene</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">family</td>
</tr>
<tr class="description-tr">
<td class="description-td">NM_000788</td>
<td class="description-td">DCK</td>
<td class="description-td">deoxycytidine kinase</td>
<td class="description-td">60</td>
<td class="description-td">302, 303</td>
<td class="description-td">318, 319</td>
</tr>
<tr class="description-tr">
<td class="description-td">NM_014889</td>
<td class="description-td">MP1</td>
<td class="description-td">metalloprotease 1</td>
<td class="description-td">61</td>
<td class="description-td">304, 305</td>
<td class="description-td">320, 321</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">(pitrilysin family)</td>
</tr>
<tr class="description-tr">
<td class="description-td">NM_000599</td>
<td class="description-td">IGFBP5</td>
<td class="description-td">insulin-like growth</td>
<td class="description-td">62</td>
<td class="description-td">306, 307</td>
<td class="description-td">322, 323</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">factor binding</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">protein 5</td>
</tr>
<tr class="description-tr">
<td class="description-td">NM_020386</td>
<td class="description-td">LOC57110</td>
<td class="description-td">H-REV107 protein-</td>
<td class="description-td">63</td>
<td class="description-td">308, 309</td>
<td class="description-td">324, 325</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">related protein</td>
</tr>
<tr class="description-tr">
<td class="description-td">NM_018401</td>
<td class="description-td">HSA250839</td>
<td class="description-td">gene for</td>
<td class="description-td">64</td>
<td class="description-td">326, 327</td>
<td class="description-td">340, 341</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">serine/threonine</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">protein kinase</td>
</tr>
<tr class="description-tr">
<td class="description-td">BC001653</td>
<td class="description-td">MG2771</td>
<td class="description-td">hypothetical protein</td>
<td class="description-td">65</td>
<td class="description-td">328, 329</td>
<td class="description-td">342, 343</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">MG2771</td>
</tr>
<tr class="description-tr">
<td class="description-td">X51754</td>
<td class="description-td">IG</td>
<td class="description-td">Lambda-</td>
<td class="description-td">66</td>
<td class="description-td">330, 331</td>
<td class="description-td">344, 345</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">LAMBDA</td>
<td class="description-td">immunoglobulin</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">CHAIN C</td>
<td class="description-td">light chain.</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">REGIONS</td>
</tr>
<tr class="description-tr">
<td class="description-td">U82987</td>
<td class="description-td">BBC3</td>
<td class="description-td">Bcl-2 binding</td>
<td class="description-td">67</td>
<td class="description-td">332, 333</td>
<td class="description-td">346, 347</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">component 3</td>
</tr>
<tr class="description-tr">
<td class="description-td">X60111</td>
<td class="description-td">HMRP-1</td>
<td class="description-td">H. sapiens MRP-1</td>
<td class="description-td">68</td>
<td class="description-td">334, 335</td>
<td class="description-td">348, 349</td>
</tr>
<tr class="description-tr">
<td class="description-td">AB021868</td>
<td class="description-td">STAT3</td>
<td class="description-td">signal transducer and</td>
<td class="description-td">69</td>
<td class="description-td">336, 337</td>
<td class="description-td">350, 351</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">activator of</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">transcription 3</td>
</tr>
<tr class="description-tr">
<td class="description-td">X55543</td>
<td class="description-td">TREB</td>
<td class="description-td">tax-responsive</td>
<td class="description-td">70</td>
<td class="description-td">338, 339</td>
<td class="description-td">352, 353</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">element-binding</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">protein</td>
</tr>
<tr class="description-tr">
<td class="description-td">NM_005978</td>
<td class="description-td">s100a2</td>
<td class="description-td">calcium-binding</td>
<td class="description-td">71</td>
<td class="description-td">354, 355</td>
<td class="description-td">360, 361</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">protein S100A2</td>
</tr>
<tr class="description-tr">
<td class="description-td">NM_002658</td>
<td class="description-td">UPA</td>
<td class="description-td">Urokinase</td>
<td class="description-td">72</td>
<td class="description-td">356, 357</td>
<td class="description-td">362, 363</td>
</tr>
<tr class="description-tr">
<td class="description-td">NM_000926</td>
<td class="description-td">PGR</td>
<td class="description-td">progesterone</td>
<td class="description-td">73</td>
<td class="description-td">358, 359</td>
<td class="description-td">364, 365</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">receptor</td>
</tr>
<tr class="description-tr">
<td class="description-td">X15943</td>
<td class="description-td">Calcitonin</td>
<td class="description-td">Calcitonin</td>
<td class="description-td">366</td>
<td class="description-td">367, 368</td>
<td class="description-td">370, 369</td>
</tr>
<tr class="description-tr">
<td align="center" class="description-td" colspan="6" nameend="6" namest="1" rowsep="1"> </td>
</tr>
</tbody></table>
</patent-tables>
</tables>
</div>
</li> <heading id="h-0003">EXAMPLES </heading>
<li> <para-num num="[0077]"> </para-num> <div class="description-line" id="p-0078" num="0077"> In order to establish the suitability of genes for inclusion within the panel each gene was investigated by means of both relatively unspecific and highly sensitive methods. Methylation within the Calcitonin gene was analysed using three different methods, namely restriction enzyme, MethyLight and combined HeavyMethyl MethyLight assays. </div>
</li> <heading id="h-0004">Example 1 </heading>
<heading id="h-0005">Restriction Enzyme Analysis </heading>
<li> <para-num num="[0078]"> </para-num> <div class="description-line" id="p-0079" num="0078"> The differential methylation was initially observed by means of methylation sensitive restriction enzyme analysis. A fragment of the upstream region of the calcitonin gene (SEQ ID NO:366) was amplified by PCR using the primers CCTTAGTCCCTACCTCTGCT (SEQ ID NO:371) and CTCATTTACACACACCCAAAC (SEQ ID NO:372). The resultant amplificate, 378 bp in length, contained an informative CpG at position 165. The amplificate DNA was digested with the methylation sensitive restriction endonuclease Nar I; recognition motif GGCGCC. Hydrolysis by said endonuclease is blocked by methylation of the CpG at position 165 of the amplificate. The digest was used as a control. </div>
</li> <li> <para-num num="[0079]"> </para-num> <div class="description-line" id="p-0080" num="0079"> Genomic DNA was isolated from the breast tissue and breast tumour samples using the DNA wizzard™ DNA isolation kit (Promega). Each sample was digested using Nar I according to manufacturer's recommendations (New England Biolabs). </div>
</li> <li> <para-num num="[0080]"> </para-num> <div class="description-line" id="p-0081" num="0080"> About 10 ng of each genomic digest was then amplified using PCR primers CCTTAGTCCCTACCTCTGCT (SEQ ID NO: 371) and CTCATTTACACACACCCAAAC (SEQ ID NO: 372). The PCR reactions were performed using a thermocycler (Eppendorf GmbH) using 10 ng of DNA, 6 pmole of each primer, 200 μM of each DNTP, 1.5 mM MgCl<sub>2 </sub>and 1 U of Hotstart™Taq (Qiagen AG). The other conditions were as recommended by the Taq polymerase manufacturer. </div>
</li> <li> <para-num num="[0081]"> </para-num> <div class="description-line" id="p-0082" num="0081"> Using the above mentioned primers, gene fragments were amplified by PCR performing a first denaturation step for 14 min at 96° C., followed by 30-45 cycles (step 2: 60 sec at 96° C., step 3: 45 sec at 52° C., step 4: 75 sec at 72° C.) and a subsequent final elongation of 10 min at 72° C. The presence of PCR products was analysed by agarose gel electrophoresis. </div>
</li> <li> <para-num num="[0082]"> </para-num> <div class="description-line" id="p-0083" num="0082"> PCR products were detectable, with Nar 1-hydrolysed DNA isolated wherein the tissue in question contained upmethylated DNA, when step 2 to step 4 of the cycle program were repeated 34, 37, 39, 42 and 45 fold. In contrast, PCR products were only detectable with Nar I-hydrolyzed DNA isolated from downmethylated tissue when steps 2 to step 4 of the cycle program were repeated 42- and 45-fold. Further investigation of the Calcitonin gene was then carried out by means of highly sensitive assays as described in Example 2 and 3. </div>
</li> <heading id="h-0006">Example 2 </heading>
<heading id="h-0007">Analysis of Methylation within Breast Cancer Using a MethyLight Assay </heading>
<li> <para-num num="[0083]"> </para-num> <div class="description-line" id="p-0084" num="0083"> DNA was extracted from 21 breast carcinoma samples and 17 normal breast tissues using a Qiagen extraction kit. The DNA from each sample was treated using a bisulfite solution (hydrogen sulfite, disulfite) according to the agarose-bead method (Olek et al 1996). The treatment is such that all non methylated cytosines within the sample are converted to thymidine. Conversely, 5-methylated cytosines within the sample remain unmodified. </div>
</li> <li> <para-num num="[0084]"> </para-num> <div class="description-line" id="p-0085" num="0084"> The methylation status was determined with a MethyLight assay designed for the CpG island of interest and a control fragment from the beta actin gene (Eads et al., 2001). The CpG island assay covers CpG sites in both the primers and the taqman style probe, while the control gene does not. The control gene is used as a measure of total DNA concentration, and the CpG island assay (methylation assay) determines the methylation levels at that site. </div>
</li> <li> <para-num num="[0085]"> </para-num> <div class="description-line" id="p-0086" num="0085"> Methods. The Calcitonin gene CpG island assay was performed using the following primers and probes:  
<tables id="TABLE-US-00002" num="2"> <patent-tables colsep="0" frame="none" rowsep="0" tabstyle="monospace"> <table align="left" class="description-table" cols="4" colsep="0" rowsep="0" width="100%"> <tbody><tr class="description-tr"> <td class="description-td"> </td> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">Primer:</td> <td class="description-td"> </td> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">AGGTTATCGTCGTGCGAGTGT;</td> <td class="description-td">(SEQ ID NO:373)</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">Primer:</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">TCACTCAAACGTATCCCAAACCTA;</td> <td class="description-td">(SEQ ID NO:374)</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">and</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">Probe:</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">CGAATCTCTCGAACGATCGCATCCA.</td> <td class="description-td">(SEQ ID NO:375)</td> </tr> </tbody></table> </patent-tables> </tables>
</div>
</li> <li> <para-num num="[0086]"> </para-num> <div class="description-line" id="p-0087" num="0086"> The corresponding control assay was performed using the following primers and probes  
<tables id="TABLE-US-00003" num="3"> <patent-tables colsep="0" frame="none" rowsep="0" tabstyle="monospace"> <table align="left" class="description-table" cols="3" colsep="0" rowsep="0" width="100%"> <tbody><tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">Primer:</td> <td class="description-td"> </td> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">TGGTGATGGAGGAGGTTTAGTAAGT;</td> <td class="description-td">(SEQ ID NO:376)</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">Primer:</td> </tr> <tr class="description-tr"> <td class="description-td">AACCAATAAAACCTACTCCTCCCTTAA;</td> <td class="description-td">(SEQ ID NO:377)</td> </tr> <tr class="description-tr"> <td class="description-td">and</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td">Probe:</td> </tr> <tr class="description-tr"> <td class="description-td">ACCACCACCCAACACACAATAACAAACACA.</td> <td class="description-td">(SEQ ID NO:378)</td> </tr> </tbody></table> </patent-tables> </tables>
</div>
</li> <li> <para-num num="[0087]"> </para-num> <div class="description-line" id="p-0088" num="0087"> The reactions were run in triplicate on each DNA sample with the following assay conditions: </div>
</li> <li> <para-num num="[0088]"> </para-num> <div class="description-line" id="p-0089" num="0088"> Reaction solution: (900 nM primers; 300 nM probe; 3.5 mM Magnesium Chloride; 1 unit of taq polymerase; 200 mM dNTPs; 7 ml of DNA, in a final reaction volume of 20 ml); </div>
</li> <li> <para-num num="[0089]"> </para-num> <div class="description-line" id="p-0090" num="0089"> Cycling conditions: (95° C. for 10 minutes; 95° C. for 15 seconds; 67° C. for 1 minute (3 cycles)); (95° C. for 15 seconds, 64° C. for 1 minute (3 cycles)); (95° C. for 15 seconds, 62° C. for 1 minute (3 cycles)); and (95° C. for 15 seconds, 60° C. for 1 minute (40 cycles)). The data was analysed using a PMR calculation previously described in the literature (Eads et al 2001). </div>
</li> <li> <para-num num="[0090]"> </para-num> <div class="description-line" id="p-0091" num="0090"> Results. The mean PMR for normal samples was 0.94, with a standard deviation of 1.28. The mean PMR for tumour samples was 8.38, with a standard deviation of 11.18. The overall difference in methylation levels between tumour and normal samples is significant in a t-test (p=0.0065). </div>
</li> <heading id="h-0008">Experiment 3 </heading>
<heading id="h-0009">Single Gene Analysis </heading>
<li> <para-num num="[0091]"> </para-num> <div class="description-line" id="p-0092" num="0091"> The same samples were analysed using the HeavyMethyl MethyLight (or HM MethyLight) assay, also referred to as the HeavyMethyl assay. The methylation status was determined with a HM MethyLight assay designed for the CpG island of interest and a control gene assay. The CpG island assay covers CpG sites in both the blockers and the taqman style probe, while the control gene does not. </div>
</li> <li> <para-num num="[0092]"> </para-num> <div class="description-line" id="p-0093" num="0092"> Methods. The CpG island assay (methylation assay) was performed using the following primers and probes:  
<tables id="TABLE-US-00004" num="4"> <patent-tables colsep="0" frame="none" rowsep="0" tabstyle="monospace"> <table align="left" class="description-table" cols="4" colsep="0" rowsep="0" width="100%"> <tbody><tr class="description-tr"> <td class="description-td"> </td> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">Primer:</td> <td class="description-td"> </td> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">GGATGTGAGAGTTGTTGAGGTTA;</td> <td class="description-td">(SEQ ID NO: 379)</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">Primer:</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">ACACACCCAAACCCATTACTATCT;</td> <td class="description-td">(SEQ ID NO: 380)</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">Probe:</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">ACCTCCGAATCTCTCGAACGATCGC;</td> <td class="description-td">(SEQ ID NO: 381)</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">and</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">Blocker:</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">TGTTGAGGTTATGTGTAATTGGGTGTGA.</td> <td class="description-td">(SEQ ID NO: 382)</td> </tr> </tbody></table> </patent-tables> </tables>
</div>
</li> <li> <para-num num="[0093]"> </para-num> <div class="description-line" id="p-0094" num="0093"> The reactions were each run in triplicate on each DNA sample with the following assay conditions: </div>
</li> <li> <para-num num="[0094]"> </para-num> <div class="description-line" id="p-0095" num="0094"> Reaction solution: (300 nM primers; 450 nM probe; 3.5 mM magnesium chloride; 2 units of taq polymerase; 400 mM dNTPs; and 7 ml of DNA, in a final reaction volume of 20 ml); </div>
</li> <li> <para-num num="[0095]"> </para-num> <div class="description-line" id="p-0096" num="0095"> Cycling conditions: (95° C. for 10 minutes); (95° C. for 15 seconds, 67° C. for 1 minute (3 cycles)); (95° C. for 15 seconds, 64° C. for 1 minute (3 cycles); (95° C. for 15 seconds, 62° C. for 1 minute (3 cycles)); and (95° C. for 15 seconds, 60° C. for 1 minute (40 cycles)). </div>
</li> <li> <para-num num="[0096]"> </para-num> <div class="description-line" id="p-0097" num="0096"> Results. The mean PMR for normal samples was 0.58, with a standard deviation of 0.94. The mean PMR for tumour samples was 3.01, with a standard deviation of 3.91. The overall difference in methylation levels between tumor and normal samples is significant in a t-test (p=0.0012). </div>
</li> <li> <para-num num="[0097]"> </para-num> <div class="description-line" id="p-0098" num="0097"> Taking into account the significance of the analyses it was decided to combine the identified CpG island with other informative CpG rich regions in the form of a gene panel for use as a diagnostic assay. This would increase the level of specificity and sensitivity as opposed to use of the identified CpG rich region as a single gene marker type diagnostic assay. </div>
</li> <heading id="h-0010">Example 4 </heading>
<heading id="h-0011">Multiple Gene ‘Panel’ Analysis </heading>
<li> <para-num num="[0098]"> </para-num> <div class="description-line" id="p-0099" num="0098"> The informative region as disclosed in SEQ ID No. 366 was included in a panel of genes for the assessment of breast tumours samples. Accordingly, an assay was devised suitable for the medium throughput analysis of mutliple CpG positions in multiple samples, the chosen format being microarray analysis. </div>
</li> <li> <para-num num="[0099]"> </para-num> <div class="description-line" id="p-0100" num="0099"> All samples were treated using the bisulphite technique disclosed in ‘Example 1’. Following bisulphite treatment selected CpG rich regions from the genes according to Table 2 were amplified by means of multiplex polymerase chain reaction, amplifying 8 fragments per reaction with Cy5 fluorescently labelled primers according to Table 2. The following conditions were used: </div>
</li> <li> <div class="description-line" id="h-0012" num="0000"> 10 ng bisulfite treated DNA </div>
</li> <li> <div class="description-line" id="h-0013" num="0000"> 3.5 mM MgCl<sub>2 </sub> </div>
</li> <li> <div class="description-line" id="h-0014" num="0000"> 400 μM dNTPs </div>
</li> <li> <div class="description-line" id="h-0015" num="0000"> 2 pmol each primer </div>
</li> <li> <div class="description-line" id="h-0016" num="0000"> 1 U Hot Start Taq (Qiagen) </div>
</li> <li> <para-num num="[0100]"> </para-num> <div class="description-line" id="p-0101" num="0100"> Forty cycles were carried out as follows. Denaturation at 95° C. for 15 min, followed by annealing at 55° C. for 45 sec., primer elongation at 65° C. for 2 min. A final elongation at 65° C. was carried out for 10 min. </div>
</li> <li> <para-num num="[0101]"> </para-num> <div class="description-line" id="p-0102" num="0101"> All PCR products from each individual sample were then hybridised to glass slides carrying a pair of immobilised oligonucleotides for each CpG position under analysis. Each of these detection oligonucleotides was designed to hybridise to the bisulphite converted sequence around one CpG site which was either originally unmethylated (TG) or methylated (CG). Hybridisation conditions were selected to allow the detection of the single nucleotide differences between the TG and CG variants. </div>
</li> <li> <para-num num="[0102]"> </para-num> <div class="description-line" id="p-0103" num="0102"> 5 μl volume of each multiplex PCR product was diluted in 10×Ssarc buffer (10×Ssarc:230 ml 20×SSC, 180 ml sodium lauroyl sarcosinate solution 20%, dilute to 1000 ml with dH<sub>2</sub>O). The reaction mixture was then hybridised to the detection oligonucleotides as follows. Denaturation at 95° C., cooling down to 10° C., hybridisation at 42° C. overnight followed by washing with 10×Ssarc and dH<sub>2</sub>O at 42° C. </div>
</li> <li> <para-num num="[0103]"> </para-num> <div class="description-line" id="p-0104" num="0103"> Fluorescent signals from each hybridised oligonucleotide were detected using genepix scanner and software. Ratios for the two signals (from the CG oligonucleotide and the TG oligonucleotide used to analyse each CpG position) were calculated based on comparison of intensity of the fluorescent signals. </div>
</li> <li> <para-num num="[0104]"> </para-num> <div class="description-line" id="p-0105" num="0104"> The information is then sorted into a ranked matrix according to CpG methylation differences between the two classes of tissues, using an algorithm. In order to accurately discriminate between the two classes of tissues, we trained a learning algorithm (support vector machine, SVM). The SVM was discussed by F. Model, P. Adorjan, A. Olek, C. Piepenbrock, Feature selection for DNA methylation based cancer classification. Bioinformatics. 2001 June;17 Suppl 1:S157-64. The algorithm constructs an optimal discriminant between two classes of given training samples. In this case each sample is described by the methylation patterns (CG/TG ratios) at the investigated CpG sites. The SVM was trained on a subset of samples of each class, which were presented with the diagnosis attached. Independent test samples, which were not shown to the SVM before were then presented to evaluate, to establish if the diagnosis could be predicted correctly based on the predictor created in the training round. This procedure was repeated several times using different partitions of the samples, a method called cross-validation. Please note that all rounds are performed without using any knowledge obtained in the previous runs. The number of correct classifications was averaged over all runs, which gives a good estimate of our test accuracy (percent of correct classified samples over all rounds).  
<tables id="TABLE-US-00005" num="5"> <patent-tables colsep="0" frame="none" rowsep="0" tabstyle="monospace"> <table align="left" class="description-table" cols="1" colsep="0" rowsep="0" width="100%"> <thead> <tr class="description-tr"> <td align="center" class="description-td" colspan="1" nameend="1" namest="1">TABLE 2</td> </tr> </thead> <tbody><tr class="description-tr"> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td align="center" class="description-td" colspan="1" nameend="1" namest="1" rowsep="1"> </td> </tr> <tr class="description-tr"> <td class="description-td">Genes and Primers according to Example 4</td> </tr> </tbody></table> <table align="left" class="description-table" cols="3" colsep="0" rowsep="0" width="100%"> <tbody><tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">Gene</td> <td class="description-td">Primers</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td align="center" class="description-td" colspan="3" nameend="2" namest="OFFSET" rowsep="1"> </td> </tr> </tbody></table> <table align="left" class="description-table" cols="4" colsep="0" rowsep="0" width="100%"> <tbody><tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">CDKN2A</td> <td class="description-td">TTGAAAATTAAGGGTTGAGG</td> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">SEQ ID NO 43</td> <td class="description-td">(SEQ ID NO: 383)</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td">CACCCTCTAATAACCAACCA</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td">(SEQ ID NO: 384)</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">CDKN2A</td> <td class="description-td">GGGGTTGGTTGGTTATTAGA</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">SEQ ID NO 43</td> <td class="description-td">(SEQ ID NO: 385)</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td">AACCCTCTACCCACCTAAAT</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td">(SEQ ID NO: 386)</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">RASSF1</td> <td class="description-td">ACCTCTCTACAAATTACAAATTCA</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">SEQ ID NO 51</td> <td class="description-td">(SEQ ID NO: 387)</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td">AGTTTGGGTTAGTTTGGGTT</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td">(SEQ ID NO: 388)</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">MYOD1</td> <td class="description-td">ATTAGGGGTATAGAGGAGTATTGA</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">SEQ ID NO 50</td> <td class="description-td">(SEQ ID NO: 389)</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td">CTTACAAACCCACAATAAACAA</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td">(SEQ ID NO: 390)</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">WT1</td> <td class="description-td">AAAGGGAAATTAAGTGTTGT</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">SEQ ID NO 55</td> <td class="description-td">(SEQ ID NO: 391)</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td">TAACTACCCTCAACTTCCC</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td">(SEQ ID NO: 392)</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">BRCA1</td> <td class="description-td">TGGATGGGAATTGTAGTTTT</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">SEQ ID NO 39</td> <td class="description-td">(SEQ ID NO: 393)</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td">TTAACCACCCAATCTACCC</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td">(SEQ ID NO: 394)</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">CCND2</td> <td class="description-td">TTTTGGTATGTAGGTTGGATG</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">SEQ ID NO 40</td> <td class="description-td">(SEQ ID NO: 395)</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td">CCTAACCTCCTTCCTTTAACT</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td">(SEQ ID NO: 396)</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td"> </td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">Calcitonin</td> <td class="description-td">AGGTTATCGTCGTGCGAGTGT</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td">SEQ ID NO: 366</td> <td class="description-td">(SEQ ID NO: 373)</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td">TCACTCAAACGTATCCCAAACCTA</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td class="description-td"> </td> <td class="description-td">(SEQ ID NO: 374)</td> </tr> <tr class="description-tr"> <td class="description-td"> </td> <td align="center" class="description-td" colspan="4" nameend="3" namest="OFFSET" rowsep="1"> </td> </tr> </tbody></table> </patent-tables> </tables>
</div>
</li> <li> <para-num num="[0105]"> </para-num> <div class="description-line" id="p-0106" num="0105">
</div>
</li> </ul>
</div>
</section><section itemprop="claims" itemscope="">
<h2>Claims (<span itemprop="count">40</span>)</h2>
<div html="" itemprop="content"><div class="claims" lang="EN" load-source="patent-office" mxw-id="PCLM7529315">
<div class="claim"> <div class="claim" id="CLM-00001" num="00001">
<div class="claim-text"> <b>1</b>. A method for the analysis of breast cell proliferative disorders, comprising determining the genomic methylation status of at least one CpG dinucleotide of at least one sequence selected from the sequence group consisting of SEQ ID NOS:1-73, SEQ ID NO:366, contiguous portions thereof, and sequences complementary thereto. </div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="CLM-00002" num="00002">
<div class="claim-text"> <b>2</b>. The method of <claim-ref idref="CLM-00001">claim 1</claim-ref>, wherein determining the genomic methylation status of the at least one CpG dinucleotide, comprises: obtaining a biological sample comprising genomic DNA from a subject; and contacting the genomic DNA with at least one reagents or a series of reagent, which distinguishes between methylated and non-methylated CpG dinucleotides. </div>
</div>
</div> <div class="claim"> <div class="claim" id="CLM-00003" num="00003">
<div class="claim-text"> <b>3</b>. A nucleic acid molecule consisting essentially of a sequence at least 18 contiguous bases in length of a sequence selected from the sequence group consisting of SEQ ID NOS:74-365, and SEQ ID NOS:367-370, and sequences complementary thereto. </div>
</div>
</div> <div class="claim"> <div class="claim" id="CLM-00004" num="00004">
<div class="claim-text"> <b>4</b>. An oligomer, consisting essentially of a sequence of at least 10 contiguous nucleotides in length that hybridises to or is identical to a sequence selected from the group consisting of SEQ ID NOS:1-370. </div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="CLM-00005" num="00005">
<div class="claim-text"> <b>5</b>. The oligomer of <claim-ref idref="CLM-00004">claim 4</claim-ref>, wherein the contiguous base sequence includes at least one CpG dinucleotide. </div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="CLM-00006" num="00006">
<div class="claim-text"> <b>6</b>. The oligomer of <claim-ref idref="CLM-00005">claim 5</claim-ref>, wherein the cytosine of the CpG dinucleotide is located in about the middle third of the oligomer. </div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="CLM-00007" num="00007">
<div class="claim-text"> <b>7</b>. A set of oligomers, comprising at least two oligomers according to any of <claim-ref idref="CLM-00004">claims 4</claim-ref> to <claim-ref idref="CLM-00006">6</claim-ref>. </div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="CLM-00008" num="00008">
<div class="claim-text"> <b>8</b>. The set of oligomers of <claim-ref idref="CLM-00007">claim 7</claim-ref>, comprising oligomers for detecting the methylation state of all CpG dinucleotides within sequences of the sequence group consisting of SEQ ID NOS: 1-73 SEQ ID NO: 366, contiguous portions thereof, and sequences complementary thereto. </div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="CLM-00009" num="00009">
<div class="claim-text"> <b>9</b>. The set of oligomers of <claim-ref idref="CLM-00007">claim 7</claim-ref>, wherein the set is suitable for use as primer oligonucleotides for the amplification of a sequence selected from the sequence group consisting of SEQ ID NOS: 1-370, contiguous portions thereof, and sequences complementary thereto. </div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="CLM-00010" num="00010">
<div class="claim-text"> <b>10</b>. The set of oligomers according to any one of claims <b>7</b> through <b>9</b>, wherein at least one oligomer is bound to a solid phase. </div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="CLM-00011" num="00011">
<div class="claim-text"> <b>11</b>. A method for determining methylation state or for detecting single nucleotide polymorphisms, comprising using a set of oligonucleotides comprising at least three oligomers according to any of claims <b>4</b> through <b>10</b> in an assay suitable for at least one of detecting cytosine methylation state single nucleotide polymorphisms (SNPs), within a sequence selected from the group consisting of SEQ ID NOS: 1-370, contiguous portions thereof, and sequences complementary thereto. </div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="CLM-00012" num="00012">
<div class="claim-text"> <b>12</b>. A method for manufacturing an arrangement of different oligomers (array) fixed to a carrier material and suitable for analysing breast cell proliferative disorders associated with the methylation state of of at least one CpG dinucleotide of a sequence selected from the group consisting of SEQ ID NOS:1-73 and SEQ ID NO:366, contiguous portions thereof, and sequences complementary thereto, comprising coupling at least one nucleic acid or oligomer according to any of the claims <b>3</b> through <b>10</b> to a solid phase. </div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="CLM-00013" num="00013">
<div class="claim-text"> <b>13</b>. An arrangement of different oligomers (array) obtainable according to <claim-ref idref="CLM-00012">claim 12</claim-ref>. </div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="CLM-00014" num="00014">
<div class="claim-text"> <b>14</b>. The arrangement of <claim-ref idref="CLM-00013">claim 13</claim-ref> wherein the oligomers are at least one of oligonucleotides and PNA-oligomer sequences, wherein the carrier material is a planar solid phase, and wherein the oligomers are arranged thereon in the form of a rectangular or hexagonal lattice. </div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="CLM-00015" num="00015">
<div class="claim-text"> <b>15</b>. The arrangement of <claim-ref idref="CLM-00013">claim 13</claim-ref>, wherein the carrier material comprises a material selected from the group consisting of silicon, glass, polystyrene, aluminium, steel, iron, copper, nickel, silver, gold, and combinations thereof. </div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="CLM-00016" num="00016">
<div class="claim-text"> <b>16</b>. An oligomer array suitable for analysing breast cell proliferative disorders associated with the methylation state of at least one CpG dinucleotide of a sequence selected from the group consisting of SEQ ID NOS:1-73, SEQ ID NO:366, contiguous portions thereof, and sequences complementary thereto, the array comprising at least one nucleic acid or oligomer according to any one of claims <b>3</b> through <b>10</b>. </div>
</div>
</div> <div class="claim"> <div class="claim" id="CLM-00017" num="00017">
<div class="claim-text"> <b>17</b>. A method for the analysis of breast cell proliferative disorders, comprising: 
<div class="claim-text">a) obtaining a biological sample comprising genomic DNA: </div> <div class="claim-text">b) contacting the genomic DNA, or a portion thereof with an agent or combination of agents suitable to convert cytosine bases that are unmethylated at the 5-position to uracil or to another base which is dissimilar to cytosine in terms of hybridisation behaviour, to provide a pretreated DNA; </div> <div class="claim-text">c) amplifying, using at least one set of primer oligonucleotides and a polymerase, at least one pretreated DNA sequence, or a portion thereof, selected from the sequence group consisting of SEQ ID NOS:74-365, SEB ID NOS:367-370, contiguous portions thereof, and sequences complementary thereto; </div> <div class="claim-text">d) determining, based on the amplification, or on analysis of the nucleic acid amplificate, the methylation status of one or more genomic CpG dinucleotides, whereby analysis of breast cell proliferative disorders is, at least in part, afforded. </div> </div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="CLM-00018" num="00018">
<div class="claim-text"> <b>18</b>. The method of <claim-ref idref="CLM-00017">claim 17</claim-ref>, wherein determining in d) comprises hybridisation of at least one nucleic acid or oligomer according to any one of claims <b>3</b> through <b>10</b>. </div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="CLM-00019" num="00019">
<div class="claim-text"> <b>19</b>. The method of <claim-ref idref="CLM-00017">claim 17</claim-ref>, wherein determining in d) comprises hybridisation of at least one oligonucleotide according to any one of claims <b>3</b> through <b>10</b>, and extension of the at least one hybridised oligonucleotide by at least one nucleotide base. </div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="CLM-00020" num="00020">
<div class="claim-text"> <b>20</b>. The method of <claim-ref idref="CLM-00017">claim 17</claim-ref>, wherein determining in d) comprises sequencing. </div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="CLM-00021" num="00021">
<div class="claim-text"> <b>21</b>. The method of <claim-ref idref="CLM-00017">claim 17</claim-ref>, wherein amplifying in c) comprises using methylation-specific primers. </div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="CLM-00022" num="00022">
<div class="claim-text"> <b>22</b>. The method of <claim-ref idref="CLM-00017">claim 17</claim-ref>, wherein determining in d) comprises use of a combination of at least two of the methods described in any one of claims <b>18</b> through <b>21</b>. </div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="CLM-00023" num="00023">
<div class="claim-text"> <b>23</b>. The method of <claim-ref idref="CLM-00017">claim 17</claim-ref>, wherein contacting in c) comprises contacting with at least one agent selected from the group consisting of bisulfite, hydrogen sulfite or disulfite. </div>
</div>
</div> <div class="claim"> <div class="claim" id="CLM-00024" num="00024">
<div class="claim-text"> <b>24</b>. A method for the analysis of breast cell proliferative disorders, comprising: 
<div class="claim-text">a) obtaining, from a subject, a biological sample containing genomic DNA; </div> <div class="claim-text">b) isolating the genomic DNA; </div> <div class="claim-text">c) digesting the isolated genomic DNA, or a portion thereof comprising at least one sequence selected from the sequence group consisting of SEQ ID NOS1-73, SEQ ID NO:366, contiguous portions thereof, and sequences complementary thereto, with one or more methylation-sensitive methylation sensitive restriction enzymes; and </div> <div class="claim-text">d) detecting of the DNA fragments generated in c), whereby analysis of breast cell proliferative disorders is, at least in part, afforded. </div> </div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="CLM-00025" num="00025">
<div class="claim-text"> <b>25</b>. The method of according to <claim-ref idref="CLM-00024">claim 24</claim-ref>, further comprising, prior to d), amplifying the DNA digest. </div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="CLM-00026" num="00026">
<div class="claim-text"> <b>26</b>. The method of any one of claims <b>17</b> and <b>25</b>, wherein more than ten different fragments having a length of about 100 to about 200 base pairs are amplified. </div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="CLM-00027" num="00027">
<div class="claim-text"> <b>27</b>. The method of any one of claims <b>17</b> and <b>26</b>, wherein the amplification of several DNA segments is carried out in one reaction vessel. </div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="CLM-00028" num="00028">
<div class="claim-text"> <b>28</b>. The method of any one of claims <b>17</b> and <b>25</b>, wherein amplifying is by means of a heat-resistant DNA polymerase. </div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="CLM-00029" num="00029">
<div class="claim-text"> <b>29</b>. The method of any one of claims <b>17</b> and <b>25</b>, wherein amplifying is by means of a polymerase chain reaction (PCR). </div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="CLM-00030" num="00030">
<div class="claim-text"> <b>30</b>. The method of any one of claims <b>17</b> and <b>25</b>, wherein the amplificates carry detectable labels. </div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="CLM-00031" num="00031">
<div class="claim-text"> <b>31</b>. The method of <claim-ref idref="CLM-00030">claim 30</claim-ref>, wherein said labels are selected from the group consisting of fluorescence labels, radionuclides, detachable molecule fragments having a typical mass which can be detected in a mass spectrometer and combinations thereof. </div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="CLM-00032" num="00032">
<div class="claim-text"> <b>32</b>. The method of <claim-ref idref="CLM-00017">claim 17</claim-ref>, wherein amplificates or fragments of the amplificates are detected in the mass spectrometer. </div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="CLM-00033" num="00033">
<div class="claim-text"> <b>33</b>. The method of any one of claims <b>31</b> and <b>32</b>, wherein produced fragments have a single positive or negative net charge for better detectability in the mass spectrometer. </div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="CLM-00034" num="00034">
<div class="claim-text"> <b>34</b>. The method of <claim-ref idref="CLM-00030">claim 30</claim-ref>, wherein detection is carried out and visualised by means of at least one of matrix assisted laser desorption/ionisation mass spectrometry (MALDI), and using electron spray mass spectrometry (ESI). </div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="CLM-00035" num="00035">
<div class="claim-text"> <b>35</b>. The method of any one of the claims <b>17</b> and <b>24</b>, wherein the genomic DNA is obtained from cells or cellular components which contain DNA, sources of DNA comprising, for example, cell lines, histological slides, biopsies, tissue embedded in paraffin and all possible combinations thereof. </div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="CLM-00036" num="00036">
<div class="claim-text"> <b>36</b>. A kit reagent having at least one of bisulfite, disulfite, and hydrogen sulfite, as well as at least one of oligonucleotides, and PNA-oligomers according to any one of the claims <b>4</b> through <b>10</b>. </div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="CLM-00037" num="00037">
<div class="claim-text"> <b>37</b>. The kit of <claim-ref idref="CLM-00036">claim 36</claim-ref>, further comprising standard reagents for performing a methylation assay selected from the group consisting of MS-SNuPE, MSP, MethyLight, HeavyMethyl, nucleic acid sequencing, and combinations thereof. </div>
</div>
</div> <div class="claim"> <div class="claim" id="CLM-00038" num="00038">
<div class="claim-text"> <b>38</b>. (canceled) </div>
</div>
</div> <div class="claim"> <div class="claim" id="CLM-00039" num="00039">
<div class="claim-text"> <b>39</b>. (canceled) </div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="CLM-00040" num="00040">
<div class="claim-text"> <b>40</b>. The oligomer of <claim-ref idref="CLM-00004">claim 4</claim-ref>, wherein the oligomer is an oligonucleotide or a peptide nucleic acid (PNA)-oligomer.</div>
</div>
</div> </div>
</div>
</section>
                </article>
            </search-app>
        </body>
    </html>
    