
    <html>
        <body>
            <search-app>
                <article class="result" itemscope="" itemtype="http://schema.org/ScholarlyArticle">
    <h1 itemprop="pageTitle">US20130125256A1 - Nitrate transport components 
        - Google Patents</h1><section itemprop="abstract" itemscope="">
<h2>Abstract</h2>
<div html="" itemprop="content"><abstract lang="EN" load-source="patent-office" mxw-id="PA110762634">
<div class="abstract" id="p-0001" num="0000">This invention relates to isolated nucleic acid fragments encoding high affinity nitrate transport components. The invention also relates to the construction of recombinant DNA constructs encoding all or a portion of nitrate transport components, in sense or antisense orientation, wherein expression of the recombinant DNA construct may alter levels of the nitrate transport components in a transformed host cell.</div>
</abstract>
</div>
</section><section itemprop="description" itemscope="">
<h2>Description</h2>
<div html="" itemprop="content"><ul class="description" lang="EN" load-source="patent-office" mxw-id="PDES59055515">
<heading id="h-0001">FIELD OF THE INVENTION</heading>
<li> <para-num num="[0001]"> </para-num> <div class="description-line" id="p-0002" num="0001">This invention is in the field of plant molecular biology. More specifically, this invention pertains to nucleic acid fragments encoding high affinity nitrate transporters in plants and seeds.</div>
</li> <heading id="h-0002">BACKGROUND OF THE INVENTION</heading>
<li> <para-num num="[0002]"> </para-num> <div class="description-line" id="p-0003" num="0002">Higher plants are autotrophic organisms that can synthesize all of their molecular components from inorganic nutrients obtained from the local environment. Nitrogen is a key element in many compounds present in plant cells.</div>
</li> <li> <para-num num="[0003]"> </para-num> <div class="description-line" id="p-0004" num="0003">It is found in the nucleoside phosphates and amino acids that form the building blocks of nucleic acids and proteins, respectively. Availability of nitrogen for crop plants is an important limiting factor in agricultural production, and the importance of nitrogen is demonstrated by the fact that only oxygen, carbon, and hydrogen are more abundant in higher plant cells. Nitrogen present in the form of ammonia or nitrate is readily absorbed and assimilated by higher plants.</div>
</li> <li> <para-num num="[0004]"> </para-num> <div class="description-line" id="p-0005" num="0004">Nitrate is the principal source of nitrogen that is available to higher plants under normal field conditions. Thus, the nitrate assimilation pathway is the major point of entry of inorganic nitrogen into organic compounds (Hewitt et al. (1976) Plant Biochemistry, pp 633-6812, Bonner, and Varner, eds. Academic Press, NY). Although some plants directly utilize ammonia, under certain conditions, nitrate is generally the major form of nitrogen available to plants.</div>
</li> <li> <para-num num="[0005]"> </para-num> <div class="description-line" id="p-0006" num="0005">Nitrate uptake by root cells is the first step of the nitrate assimilation pathway in higher plants (Orsel et al. (2002) Plant Physiology 129: 886-896). Plants have developed two different uptake systems to cope with the varying availability of nitrate in cultivated soils. The low-affinity nitrate transport system is used preferentially when external nitrate concentration is high, whereas the high-affinity transport system (HATS) takes place at very low external concentrations.</div>
</li> <li> <para-num num="[0006]"> </para-num> <div class="description-line" id="p-0007" num="0006">In higher plants, two gene families have been identified: the NRT1 and NRT2 families involved in the low-affinity transport system and HATs, respectively. The complexity of nitrate/nitrite transport is enhanced by the fine regulation that occurs at the transcriptional level: both low and high-affinity systems have constitutive and inducible components that are clearly distinct. Furthermore, some members of the nitrate transporters require a second gene product, a NAR2-type polypeptide for function (Tong et al. (2005) The Plant Journal 41: 442-450).</div>
</li> <li> <para-num num="[0007]"> </para-num> <div class="description-line" id="p-0008" num="0007">The nucleotide sequences of the instant application and the methods of their use can increase the efficiency by which nitrogen can be used.</div>
</li> <heading id="h-0003">SUMMARY OF THE INVENTION</heading>
<li> <para-num num="[0008]"> </para-num> <div class="description-line" id="p-0009" num="0008">The present invention includes isolated polynucleotides encoding a polypeptide required for high affinity nitrate transport, wherein the amino acid sequence of the polypeptide and the amino acid sequence of SEQ ID NO: 36 or 49, have at least 80%, 85%, 90%, 95%, 99% or 100% identity (b) the complement of the nucleotide sequence, wherein the complement and the nucleotide sequence contain the same number of nucleotides and are 100% complementary. The polypeptide preferably comprises the amino acid sequence of SEQ ID NO: 36 or 49. The nucleotide sequence preferably comprises the nucleotide sequence of SEQ ID NO: 35 or 48.</div>
</li> <li> <para-num num="[0009]"> </para-num> <div class="description-line" id="p-0010" num="0009">In a first embodiment, the present invention includes an isolated polynucleotide comprising: (a) a nucleotide sequence encoding a polypeptide required for high affinity nitrate transport, wherein the polypeptide has an amino acid sequence of at least 80%, 85%, 90%, 95%, 99% or 100% sequence identity based on the Clustal V method of alignment when compared to a polypeptide SEQ ID NO: 36 or 49.</div>
</li> <li> <para-num num="[0010]"> </para-num> <div class="description-line" id="p-0011" num="0010">(b) a complement of the nucleotide sequence, wherein the complement and the nucleotide sequence contain the same number of nucleotides and are 100% complementary.</div>
</li> <li> <para-num num="[0011]"> </para-num> <div class="description-line" id="p-0012" num="0011">In a second embodiment, this invention concerns such isolated nucleotide sequence or its complement which comprises at least two motifs corresponding substantially to any of the amino acid sequences set forth in SEQ ID NO: 50, 51 or 52, wherein said motif is substantially a conserved subsequence. Examples of such motifs, among others that can be identified, are shown in SEQ ID NO: 50, 51 or 52. Also of interest is the use of such fragment or a part thereof in antisense inhibition or co-suppression in a transformed plant.</div>
</li> <li> <para-num num="[0012]"> </para-num> <div class="description-line" id="p-0013" num="0012">In a third embodiment this invention concerns such isolated nucleotide fragment complement thereof wherein the fragment or a part thereof is useful in antisense inhibition or co-suppression of a protein altering nitrate transport in a transformed plant.</div>
</li> <li> <para-num num="[0013]"> </para-num> <div class="description-line" id="p-0014" num="0013">In a fourth embodiment, this invention concerns an isolated nucleic acid fragment comprising a promoter wherein said promoter consists essentially of the nucleotide sequence set forth in SEQ ID NO:: 37, 38, 46, 47, 56, 65, 67, 68, 69, 70, 71, 72, 73, 74, 89 or 90, or said promoter consists essentially of a fragment or subfragment that is substantially similar and functionally equivalent to the nucleotide sequence set forth in SEQ ID NO:: 37, 38, 46, 47, 56, 65, 67, 68, 69, 70, 71, 72, 73, 74, 89 or 90.</div>
</li> <li> <para-num num="[0014]"> </para-num> <div class="description-line" id="p-0015" num="0014">In a fifth embodiment, this invention concerns recombinant DNA constructs comprising any of the foregoing nucleic acid fragment or complement thereof or part of either operably linked to at least one regulatory sequence. Also, of interest are plants comprising such recombinant DNA constructs in their genome, plant tissue or cells obtained from such plants and seeds obtained from these plants.</div>
</li> <li> <para-num num="[0015]"> </para-num> <div class="description-line" id="p-0016" num="0015">In a sixth embodiment, this invention concerns a method of altering nitrate transport in plants which comprises:</div>
</li> <li> <para-num num="[0016]"> </para-num> <div class="description-line" id="p-0017" num="0016">(a) transforming a plant with a recombinant DNA construct comprising:
</div> </li> <ul> <li id="ul0001-0001" num="0000"> <ul> <li id="ul0002-0001" num="0017">i) a first recombinant DNA construct comprising an isolated polynucleotide encoding a HAT polypeptide, operably linked to at least one regulatory sequence; and</li> <li id="ul0002-0002" num="0018">ii) at least one additional recombinant DNA construct comprising an isolated polynucleotide encoding a NAR polypeptide, operably linked to at least one regulatory sequence,</li> </ul> </li> </ul>
<li> <para-num num="[0019]"> </para-num> <div class="description-line" id="p-0018" num="0019">(b) growing the transformed plant of (a) under conditions suitable for the expression of the recombinant DNA constructs; and selecting those transformed plants having altered nitrate transport. Corn plants comprising these recombinant constructs are also part of this invention.</div>
</li> <li> <para-num num="[0020]"> </para-num> <div class="description-line" id="p-0019" num="0020">In a seventh embodiment, this invention concerns a method to isolate nucleic acid fragments encoding polypeptides associated with altering nitrate transport which comprises:</div>
</li> <li> <para-num num="[0021]"> </para-num> <div class="description-line" id="p-0020" num="0021">(a) comparing SEQ ID NO: 36, 49, 55, or 58 with other polypeptide sequences associated with altering plant nitrate transport;</div>
</li> <li> <para-num num="[0022]"> </para-num> <div class="description-line" id="p-0021" num="0022">(b) identifying the conserved sequences(s) or 4 or more amino acids obtained in step (a);</div>
</li> <li> <para-num num="[0023]"> </para-num> <div class="description-line" id="p-0022" num="0023">(c) making region-specific nucleotide probe(s) or oligomer(s) based on the conserved sequences identified in step (b); and</div>
</li> <li> <para-num num="[0024]"> </para-num> <div class="description-line" id="p-0023" num="0024">(d) using the nucleotide probe(s) or oligomer(s) of step (c) to isolate sequences associated with altering nitrate transport by sequence dependent protocols.</div>
</li> <li> <para-num num="[0025]"> </para-num> <div class="description-line" id="p-0024" num="0025">In an eighth embodiment, this invention also concerns a method of mapping genetic variations related to altering plant nitrate transport:</div>
</li> <li> <para-num num="[0026]"> </para-num> <div class="description-line" id="p-0025" num="0026">(a) crossing two plant varieties; and</div>
</li> <li> <para-num num="[0027]"> </para-num> <div class="description-line" id="p-0026" num="0027">(b) evaluating genetic variations with respect to:
</div> </li> <ul> <li id="ul0003-0001" num="0000"> <ul> <li id="ul0004-0001" num="0028">(i) a nucleic acid sequence selected from the group consisting of SEQ ID NOs: 35, 48, 54, and 57; or</li> <li id="ul0004-0002" num="0029">(ii) a nucleic acid sequence encoding a polypeptide selected from the group consisting of SEQ ID NOs: 36, 49, 55, and 58;</li> </ul> </li> </ul>
<li> <para-num num="[0030]"> </para-num> <div class="description-line" id="p-0027" num="0030">in progeny plants resulting from the cross of step (a) wherein the evaluation is made using a method selected from the group consisting of: RFLP analysis, SNP analysis, and PCR-based analysis.</div>
</li> <li> <para-num num="[0031]"> </para-num> <div class="description-line" id="p-0028" num="0031">In a ninth embodiment, this invention concerns a method of molecular breeding to obtain altered plant nitrate transport, comprising:</div>
</li> <li> <para-num num="[0032]"> </para-num> <div class="description-line" id="p-0029" num="0032">(a) crossing two plant varieties; and</div>
</li> <li> <para-num num="[0033]"> </para-num> <div class="description-line" id="p-0030" num="0033">(b) evaluating genetic variations with respect to:
</div> </li> <ul> <li id="ul0005-0001" num="0000"> <ul> <li id="ul0006-0001" num="0034">(i) a nucleic acid sequence selected from the group consisting of SEQ ID NOs:35, 48, 54, and 57; or</li> <li id="ul0006-0002" num="0035">(ii) a nucleic acid sequence encoding a polypeptide selected from the group consisting of SEQ ID NOs: 36, 49, 55, and 58;</li> </ul> </li> </ul>
<li> <para-num num="[0036]"> </para-num> <div class="description-line" id="p-0031" num="0036">in progeny plants resulting from the cross of step (a) wherein the evaluation is made using a method selected from the group consisting of: RFLP analysis, SNP analysis, and PCR-based analysis.</div>
</li> <li> <para-num num="[0037]"> </para-num> <div class="description-line" id="p-0032" num="0037">In a tenth embodiment, this invention concerns a method of altering the level of expression of a high affinity nitrate transporter polypeptide in a host cell comprising: (a) transforming a host cell with a recombinant DNA construct comprising:</div>
</li> <li> <para-num num="[0038]"> </para-num> <div class="description-line" id="p-0033" num="0038">(b) a nucleotide sequence encoding a high affinity nitrate transporter polypeptide, wherein the polypeptide has an amino acid sequence of at least 80% sequence identity, based on the Clustal V method of alignment, when compared to one of SEQ ID NO: 36 or 49 and the polypeptide alters nitrate transport, the complement thereof or at least two motifs corresponding substantially to any of the amino acid sequences set forth in SEQ ID NOs: 50, 51 and 52, wherein said motif is a substantially conserved subsequence operably linked to at least one regulatory sequence; and</div>
</li> <li> <para-num num="[0039]"> </para-num> <div class="description-line" id="p-0034" num="0039">(c) growing the transformed host cell under conditions that are suitable for expression of the recombinant DNA construct wherein expression of the recombinant DNA construct results in production of altered levels of the polypeptide required for nitrate transport in the transformed host cell.</div>
</li> <li> <para-num num="[0040]"> </para-num> <div class="description-line" id="p-0035" num="0040">In an eleventh embodiment, this invention concerns a corn plant, comprising a first DNA construct comprising an isolated HAT polypeptide, operably linked to at least one regulatory sequence; and at least one additional recombinant DNA construct comprising an isolated polynucleotide, operably linked to at least one regulatory sequence, encoding a polypeptide selected from the group consisting of a NAR 2.</div>
</li> <li> <para-num num="[0041]"> </para-num> <div class="description-line" id="p-0036" num="0041">An additional embodiment of this invention concerns a method for altering plant nitrogen transport, comprising:</div>
</li> <li> <para-num num="[0042]"> </para-num> <div class="description-line" id="p-0037" num="0042">(a) transforming a plant with a recombinant DNA construct comprising:
</div> </li> <ul> <li id="ul0007-0001" num="0000"> <ul> <li id="ul0008-0001" num="0043">i) a first recombinant DNA construct comprising an isolated polynucleotide encoding a HAT polypeptide, operably linked to at least one regulatory sequence; and</li> <li id="ul0008-0002" num="0044">ii) at least one additional recombinant DNA construct comprising an isolated polynucleotide, operably linked to at least one regulatory sequence, encoding a polypeptide selected from the group consisting of a NAR;</li> </ul> </li> </ul>
<li> <para-num num="[0045]"> </para-num> <div class="description-line" id="p-0038" num="0045">(b) growing the transformed plant of (a) under conditions suitable for the expression of the recombinant DNA construct; and</div>
</li> <li> <para-num num="[0046]"> </para-num> <div class="description-line" id="p-0039" num="0046">(c) selecting those transformed plants having altered nitrate transport.</div>
</li> <li> <para-num num="[0047]"> </para-num> <div class="description-line" id="p-0040" num="0047">Further embodiments of this invention include shuffled HAT variants with improved kinetic parameters, recombinant DNA constructs comprising the nucleotide sequences encoding these variants and plants and transformed cells comprising in their genome these recombinant DNA construct. Also included in this invention are corn plants comprising a first recombinant DNA construct comprising a nucleotide sequence encoding a shuffled HAT variant, operably linked to at least one regulatory sequence and at least one additional recombinant DNA construct comprising an isolated polynucleotide, operably linked to at least one regulatory sequence, encoding a polypeptide selected from the group consisting of a NAR.</div>
</li> <li> <para-num num="[0048]"> </para-num> <div class="description-line" id="p-0041" num="0048">Yet another embodiment of this invention sets forth a method for altering plant nitrogen transport, comprising: a) transforming a plant with a recombinant DNA construct comprising a first recombinant DNA construct comprising a nucleotide sequence encoding a shuffled HAT variant, operably linked to at least one regulatory sequence and at least one additional recombinant DNA construct comprising an isolated polynucleotide, operably linked to at least one regulatory sequence, encoding a polypeptide selected from the group consisting of a NAR; and b) growing the transformed plant of (a) under conditions suitable for the expression of the recombinant DNA construct; and selecting those transformed plants having altered nitrate transport.</div>
</li> <heading id="h-0004">Biological Deposits</heading>
<li> <para-num num="[0049]"> </para-num> <div class="description-line" id="p-0042" num="0049">The following plasmids have been deposited with the American Type Culture Collection (ATCC), 10801 University Boulevard, Manassas, Va. 20110-2209, and bear the following designations, accession numbers and dates of deposit.</div>
</li> <li> <div class="description-line" id="p-0043" num="0000">
<tables id="TABLE-US-00001" num="00001">
<patent-tables colsep="0" frame="none" rowsep="0">
<table align="left" class="description-table" cols="3" colsep="0" rowsep="0" width="100%">
<thead>
<tr class="description-tr">
<td align="center" class="description-td" colspan="3" nameend="3" namest="1" rowsep="1"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">Plasmid</td>
<td class="description-td">Accession Number</td>
<td class="description-td">Date of Deposit</td>
</tr>
<tr class="description-tr">
<td align="center" class="description-td" colspan="3" nameend="3" namest="1" rowsep="1"> </td>
</tr>
</thead>
<tbody><tr class="description-tr">
<td class="description-td">PHP27621</td>
<td class="description-td">ATCC</td>
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td align="center" class="description-td" colspan="3" nameend="3" namest="1" rowsep="1"> </td>
</tr>
</tbody></table>
</patent-tables>
</tables>
</div>
</li> <description-of-drawings>
<heading id="h-0005">BRIEF DESCRIPTION OF THE SEQUENCE LISTINGS</heading>
<li> <para-num num="[0050]"> </para-num> <div class="description-line" id="p-0044" num="0050">The invention can be more fully understood from the following detailed description and the accompanying drawings and Sequence Listing, which form a part of this application.</div>
</li> <li> <para-num num="[0051]"> </para-num> <div class="description-line" id="p-0045" num="0051"> <figref idrefs="DRAWINGS">FIG. 1</figref> is a schematic of vector PHP27621.</div>
</li> <li> <para-num num="[0052]"> </para-num> <div class="description-line" id="p-0046" num="0052"> <figref idrefs="DRAWINGS">FIG. 2</figref> is a schematic of vector PHP27660.</div>
</li> <li> <para-num num="[0053]"> </para-num> <div class="description-line" id="p-0047" num="0053"> <figref idrefs="DRAWINGS">FIG. 3</figref> is a schematic of vector PHP27860.</div>
</li> <li> <para-num num="[0054]"> </para-num> <div class="description-line" id="p-0048" num="0054"> <figref idrefs="DRAWINGS">FIG. 4</figref> is a schematic of vector PHP27280.</div>
</li> <li> <para-num num="[0055]"> </para-num> <div class="description-line" id="p-0049" num="0055"> <figref idrefs="DRAWINGS">FIG. 5</figref> is a schematic of vector PHP27281.</div>
</li> <li> <para-num num="[0056]"> </para-num> <div class="description-line" id="p-0050" num="0056"> <figref idrefs="DRAWINGS">FIG. 6</figref> is a schematic of vector PHP27282.</div>
</li> <li> <para-num num="[0057]"> </para-num> <div class="description-line" id="p-0051" num="0057"> <figref idrefs="DRAWINGS">FIG. 7</figref> is a schematic of vector PHP27283.</div>
</li> </description-of-drawings>
<li> <para-num num="[0058]"> </para-num> <div class="description-line" id="p-0052" num="0058">SEQ ID NO: 1 is the forward primer used in Example 3.</div>
</li> <li> <para-num num="[0059]"> </para-num> <div class="description-line" id="p-0053" num="0059">SEQ ID NO: 2 is the reverse primer used in Example 3.</div>
</li> <li> <para-num num="[0060]"> </para-num> <div class="description-line" id="p-0054" num="0060">SEQ ID NO: 3 is the T7 primer used in Example 3 for confirmatory BAC ends sequencing.</div>
</li> <li> <para-num num="[0061]"> </para-num> <div class="description-line" id="p-0055" num="0061">SEQ ID NO: 4 is the SP6 primer used in Example 3 for confirmatory BAC ends sequencing.</div>
</li> <li> <para-num num="[0062]"> </para-num> <div class="description-line" id="p-0056" num="0062">SEQ ID NO: 5 through 33 are the sequencing primers used to cover the region on BAC clone bacc.pk139.d24 containing the HAT4 gene.</div>
</li> <li> <para-num num="[0063]"> </para-num> <div class="description-line" id="p-0057" num="0063">SEQ ID NO: 34 represents the 3924 bp of the maize genomic sequence containing the ORF (Nucleotides 2015-3583 (Stop)) of the gene encoding the high affinity nitrate transporter (HAT4) isolated from BAC clone bacc.pk139.d24.</div>
</li> <li> <para-num num="[0064]"> </para-num> <div class="description-line" id="p-0058" num="0064">SEQ ID NO: 35 is 1569 bp of the nucleotide sequence of the ORF of SEQ ID NO: 34.</div>
</li> <li> <para-num num="[0065]"> </para-num> <div class="description-line" id="p-0059" num="0065">SEQ ID NO: 36 is the amino acid sequence encoded by nucleotides 2015-3580 of SEQ ID NO: 34.</div>
</li> <li> <para-num num="[0066]"> </para-num> <div class="description-line" id="p-0060" num="0066">SEQ ID NO: 37 is the 2014 bp, extending from Nucleotides 1-2014 of the putative promoter of the maize high affinity nitrate transporter genomic sequence shown in SEQ ID NO: 34.</div>
</li> <li> <para-num num="[0067]"> </para-num> <div class="description-line" id="p-0061" num="0067">SEQ ID NO: 38 is 1014 bp, extending from Nucleotide 1001-2014 of the putative promoter of the maize high affinity nitrate transporter genomic sequence shown in SEQ ID NO: 34.</div>
</li> <li> <para-num num="[0068]"> </para-num> <div class="description-line" id="p-0062" num="0068">SEQ ID NO: 39-42 are the forward and reverse primers used in Example 4.</div>
</li> <li> <para-num num="[0069]"> </para-num> <div class="description-line" id="p-0063" num="0069">SEQ ID NO: 43 is the T3 primer used in Example 4.</div>
</li> <li> <para-num num="[0070]"> </para-num> <div class="description-line" id="p-0064" num="0070">SEQ ID NO: 44 is the T7 primer used in Example 4.</div>
</li> <li> <para-num num="[0071]"> </para-num> <div class="description-line" id="p-0065" num="0071">SEQ ID NO: 45 represents the 5812 bp of the maize genomic sequence containing the ORF (Nucleotides 2264-3450 and 5087-5357 (Stop)) of the gene encoding a high affinity nitrate transporter (HAT7).</div>
</li> <li> <para-num num="[0072]"> </para-num> <div class="description-line" id="p-0066" num="0072">SEQ ID NO: 46 is the 2263 bp, extending from Nucleotides 1-2263 of the putative promoter of the maize high affinity nitrate transporter genomic sequence shown in SEQ ID NO: 45.</div>
</li> <li> <para-num num="[0073]"> </para-num> <div class="description-line" id="p-0067" num="0073">SEQ ID NO: 47 is the 1263 bp, extending from Nucleotides 1001-2263 of the putative promoter of the maize high affinity nitrate transporter genomic sequence shown in SEQ ID NO: 45.</div>
</li> <li> <para-num num="[0074]"> </para-num> <div class="description-line" id="p-0068" num="0074">SEQ ID NO: 48 is 1455 bp of the coding sequence, extending from Nucleotides 2264-3450 and 5087-5354 of SEQ ID NO: 45.</div>
</li> <li> <para-num num="[0075]"> </para-num> <div class="description-line" id="p-0069" num="0075">SEQ ID NO: 49: is the amino acid sequence encoded by SEQ ID NO: 48.</div>
</li> <li> <para-num num="[0076]"> </para-num> <div class="description-line" id="p-0070" num="0076">SEQ ID NO: 50 is a conserved sequence motif useful in identifying genes belonging to the high affinity nitrate transporter of genes.</div>
</li> <li> <para-num num="[0077]"> </para-num> <div class="description-line" id="p-0071" num="0077">SEQ ID NO: 51 is a conserved sequence motif useful in identifying genes belonging to the high affinity nitrate transporter of genes.</div>
</li> <li> <para-num num="[0078]"> </para-num> <div class="description-line" id="p-0072" num="0078">SEQ ID NO: 52 is a conserved sequence motif useful in identifying genes belonging to the high affinity nitrate transporter of genes.</div>
</li> <li> <para-num num="[0079]"> </para-num> <div class="description-line" id="p-0073" num="0079">SEQ ID NO: 53 is the 1561 bp of the sequence containing the ORF (nucleotides 757-1368 (Stop)) encoding a corn NAR2-type polypeptide (NAR2.1).</div>
</li> <li> <para-num num="[0080]"> </para-num> <div class="description-line" id="p-0074" num="0080">SEQ ID NO: 54 is the 612 bp of the coding sequence, extending from nucleotides 758-1369 (Stop) of SEQ ID NO: 53.</div>
</li> <li> <para-num num="[0081]"> </para-num> <div class="description-line" id="p-0075" num="0081">SEQ ID NO: 55 is the amino acid sequence encoded by nucleotides 758-1366 of SEQ ID NO: 54.</div>
</li> <li> <para-num num="[0082]"> </para-num> <div class="description-line" id="p-0076" num="0082">SEQ ID NO: 56 is the 756 bp, extending from Nucleotides 1-756 of the putative promoter of the sequence shown in SEQ ID NO: 53.</div>
</li> <li> <para-num num="[0083]"> </para-num> <div class="description-line" id="p-0077" num="0083">SEQ ID NO: 57 is the 594 bp of the ORF (nucleotides 1-594 (Stop)) encoding a NAR2-type polypeptide (NAR2.2).</div>
</li> <li> <para-num num="[0084]"> </para-num> <div class="description-line" id="p-0078" num="0084">SEQ ID NO: 58 is the amino acid sequence encoded by nucleotides 1-591 of the ORF of SEQ ID NO: 57.</div>
</li> <li> <para-num num="[0085]"> </para-num> <div class="description-line" id="p-0079" num="0085">SEQ ID NO: 59 is the NAR2.1 specific outer primer used in Example 6.</div>
</li> <li> <para-num num="[0086]"> </para-num> <div class="description-line" id="p-0080" num="0086">SEQ ID NO: 60 is the NAR2.1 specific inner primer used in Example 6.</div>
</li> <li> <para-num num="[0087]"> </para-num> <div class="description-line" id="p-0081" num="0087">SEQ ID NO: 61-64 are the sequencing primers used to sequence the NAR2.1 promoter upstream region.</div>
</li> <li> <para-num num="[0088]"> </para-num> <div class="description-line" id="p-0082" num="0088">SEQ ID NO: 65 shows an additional 2917 bp of the putative NAR2.1 promoter.</div>
</li> <li> <para-num num="[0089]"> </para-num> <div class="description-line" id="p-0083" num="0089">SEQ ID NO: 66 shows the 4498 bp of the complete NAR2.1 gene, including an intron extending from nucleotides 3655-3841.</div>
</li> <li> <para-num num="[0090]"> </para-num> <div class="description-line" id="p-0084" num="0090">SEQ ID NO: 67 is the 3506 bp, extending from Nucleotides 1-3506 of the putative promoter of the NAR2.1 genomic sequence shown in SEQ ID NO: 66.</div>
</li> <li> <para-num num="[0091]"> </para-num> <div class="description-line" id="p-0085" num="0091">SEQ ID NO: 68 is 1014 bp, extending from Nucleotide 1001-2014 of the putative promoter of the NAR2.1 genomic sequence shown in SEQ ID NO: 66.</div>
</li> <li> <para-num num="[0092]"> </para-num> <div class="description-line" id="p-0086" num="0092">SEQ ID NO: 69 is 1492 bp, extending from Nucleotide 2015-3506 of the putative promoter of the NAR2.1 genomic sequence shown in SEQ ID NO: 66.</div>
</li> <li> <para-num num="[0093]"> </para-num> <div class="description-line" id="p-0087" num="0093">SEQ ID NO: 70 is 3621 bp of the genomic fragment isolated in Example 14.</div>
</li> <li> <para-num num="[0094]"> </para-num> <div class="description-line" id="p-0088" num="0094">SEQ ID NO: 71 is 3236 bp of the putative Nar promoter from B73, extending from Nucleotides 1-3236 of SEQ ID NO: 70.</div>
</li> <li> <para-num num="[0095]"> </para-num> <div class="description-line" id="p-0089" num="0095">SEQ ID NO: 72 is 1000 bp of the putative Nar promoter from B73, extending from Nucleotides 1-1000 of SEQ ID NO: 70.</div>
</li> <li> <para-num num="[0096]"> </para-num> <div class="description-line" id="p-0090" num="0096">SEQ ID NO: 73 is 2236 bp of the putative Nar promoter from B73, extending from Nucleotides 1001-3236 of SEQ ID NO: 70.</div>
</li> <li> <para-num num="[0097]"> </para-num> <div class="description-line" id="p-0091" num="0097">SEQ ID NO: 74 is 1237 bp of the putative Nar promoter from B73, extending from Nucleotides 2000-3236 of SEQ ID NO: 70.</div>
</li> <li> <para-num num="[0098]"> </para-num> <div class="description-line" id="p-0092" num="0098">SEQ ID NO: 75 through 78 are the forward and reverse primers described in Example 14.</div>
</li> <li> <para-num num="[0099]"> </para-num> <div class="description-line" id="p-0093" num="0099">SEQ ID NO: 79-84 are the sequencing primers used to sequence the Nar promoter from B73 as described in Example 14.</div>
</li> <li> <para-num num="[0100]"> </para-num> <div class="description-line" id="p-0094" num="0100">SEQ ID NO: 85 is the sequence of vector pENTR-5′ described in Example 14.</div>
</li> <li> <para-num num="[0101]"> </para-num> <div class="description-line" id="p-0095" num="0101">SEQ ID NO: 86 is the sequence of vector PHP27621 described in Example 16.</div>
</li> <li> <para-num num="[0102]"> </para-num> <div class="description-line" id="p-0096" num="0102">SEQ ID NO: 87 is the sequence of vector PHP27660 described in Example 17.</div>
</li> <li> <para-num num="[0103]"> </para-num> <div class="description-line" id="p-0097" num="0103">SEQ ID NO: 88 is the sequence of vector PHP27860 described in Example 17.</div>
</li> <li> <para-num num="[0104]"> </para-num> <div class="description-line" id="p-0098" num="0104">SEQ ID NO: 89 is 3324 bp of the putative Nar promoter from B73, comprising Nucleotides 1-1523 and 1821-3324 of SEQ ID NO: 70.</div>
</li> <li> <para-num num="[0105]"> </para-num> <div class="description-line" id="p-0099" num="0105">SEQ ID 90: is 500 bp of the putative Nar promoter from B73, extending from Nucleotides 2825-3324 of SEQ ID NO: 70.</div>
</li> <li> <para-num num="[0106]"> </para-num> <div class="description-line" id="p-0100" num="0106">SEQ ID NO:91: represents the 2025 bp of the maize sequence containing the ORF (Nucleotides 250-1812(Stop)) of the gene encoding the high affinity nitrate transporter (HAT5) isolated from clone cfp4n.pk008.p6:fis.</div>
</li> <li> <para-num num="[0107]"> </para-num> <div class="description-line" id="p-0101" num="0107">SEQ ID NO:92 is the amino acid sequence encoded by the ORF of SEQ ID NO: 91.</div>
</li> <li> <para-num num="[0108]"> </para-num> <div class="description-line" id="p-0102" num="0108">SEQ ID NO: 93 is the sequence of vector PHP27280 described in Example 20.</div>
</li> <li> <para-num num="[0109]"> </para-num> <div class="description-line" id="p-0103" num="0109">SEQ ID NO: 94 is the sequence of vector PHP27281 described in Example 20.</div>
</li> <li> <para-num num="[0110]"> </para-num> <div class="description-line" id="p-0104" num="0110">SEQ ID NO: 95 is the sequence of vector PHP27282 described in Example 20.</div>
</li> <li> <para-num num="[0111]"> </para-num> <div class="description-line" id="p-0105" num="0111">SEQ ID NO: 96 is the sequence of vector PHP27283 described in Example 20.</div>
</li> <li> <para-num num="[0112]"> </para-num> <div class="description-line" id="p-0106" num="0112">The Sequence Listing contains the one letter code for nucleotide sequence characters and the three letter codes for amino acids as defined in conformity with the IUPAC-IUBMB standards described in <i>Nucleic Acids Research </i>13:3021-3030 (1985) and in the <i>Biochemical Journal </i>219 (No. 2): 345-373 (1984) which are herein incorporated by reference. The symbols and format used for nucleotide and amino acid sequence data comply with the rules set forth in 37 C.F.R. §1.822.</div>
</li> <heading id="h-0006">DETAILED DESCRIPTION OF THE INVENTION</heading>
<li> <para-num num="[0113]"> </para-num> <div class="description-line" id="p-0107" num="0113">The disclosure of each reference set forth herein is incorporated herein by reference in its entirety.</div>
</li> <li> <para-num num="[0114]"> </para-num> <div class="description-line" id="p-0108" num="0114">The term “NAR” refers to nitrate assimilation related genes. These type of genes and the NAR polypeptides encoded by them are a component of the high affinity nitrate uptake system in plants.</div>
</li> <li> <para-num num="[0115]"> </para-num> <div class="description-line" id="p-0109" num="0115">The term “HAT” is used interchangeably with high affinity nitrate transporter.</div>
</li> <li> <para-num num="[0116]"> </para-num> <div class="description-line" id="p-0110" num="0116">As used herein, an “isolated nucleic acid fragment” is used interchangeably with “isolated polynucleotide” and is a polymer of RNA or DNA that is single- or double-stranded, optionally containing synthetic, non-natural or altered nucleotide bases. An isolated nucleic acid fragment in the form of a polymer of DNA may be comprised of one or more segments of cDNA, genomic DNA or synthetic DNA. Nucleotides (usually found in their 5′-monophosphate form) are referred to by their single letter designation as follows: “A” for adenylate or deoxyadenylate (for RNA or DNA, respectively), “C” for cytidylate or deoxycytidylate, “G” for guanylate or deoxyguanylate, “U” for uridylate, “T” for deoxythymidylate, “R” for purines (A or G), “Y” for pyrimidines (C or T), “K” for G or T, “H” for A or C or T, “I” for inosine, and “N” for any nucleotide.</div>
</li> <li> <para-num num="[0117]"> </para-num> <div class="description-line" id="p-0111" num="0117">The term “isolated” refers to materials, such as nucleic acid molecules and/or proteins, which are substantially free or otherwise removed from components that normally accompany or interact with the materials in a naturally occurring environment. Isolated polynucleotides may be purified from a host cell in which they naturally occur. Conventional nucleic acid purification methods known to skilled artisans may be used to obtain isolated polynucleotides. The term also embraces recombinant polynucleotides and chemically synthesized polynucleotides.</div>
</li> <li> <para-num num="[0118]"> </para-num> <div class="description-line" id="p-0112" num="0118">The terms “subfragment that is functionally equivalent” and “functionally equivalent subfragment” are used interchangeably herein. These terms refer to a portion or subsequence of an isolated nucleic acid fragment in which the ability to alter gene expression or produce a certain phenotype is retained whether or not the portion or subsequence encodes an active enzyme or functional protein (for example, the portion or subsequence may be a portion of coding and/or non-coding regions and need not encode an active enzyme or functional protein. For example, the fragment or subfragment can be used in the design of recombinant DNA constructs to produce the desired phenotype in a transformed plant. Recombinant DNA constructs can be designed for use in co-suppression or antisense by linking a nucleic acid fragment or subfragment thereof, whether or not it encodes an active enzyme or functional protein, in the appropriate orientation relative to a plant promoter sequence.</div>
</li> <li> <para-num num="[0119]"> </para-num> <div class="description-line" id="p-0113" num="0119">The terms “homology”, “homologous”, “substantially similar” and “corresponding substantially” are used interchangeably herein. They refer to nucleic acid fragments wherein changes in one or more nucleotide bases does not affect the ability of the nucleic acid fragment to mediate gene expression or produce a certain phenotype. These terms also refer to modifications of the nucleic acid fragments of the instant invention such as deletion or insertion of one or more nucleotides that do not substantially alter the functional properties of the resulting nucleic acid fragment relative to the initial, unmodified fragment. It is therefore understood, as those skilled in the art will appreciate, that the invention encompasses more than the specific exemplary sequences.</div>
</li> <li> <para-num num="[0120]"> </para-num> <div class="description-line" id="p-0114" num="0120">Moreover, the skilled artisan recognizes that substantially similar nucleic acid sequences encompassed by this invention are also defined by their ability to hybridize, under moderately stringent conditions (for example, 1×SSC, 0.1% SDS, 60° C.) with the sequences exemplified herein, or to any portion of the nucleotide sequences reported herein and which are functionally equivalent to the gene or the promoter of the invention. Stringency conditions can be adjusted to screen for moderately similar fragments, such as homologous sequences from distantly related organisms, to highly similar fragments, such as genes that duplicate functional enzymes from closely related organisms. Post-hybridization washes determine stringency conditions. One set of preferred conditions involves a series of washes starting with 6×SSC, 0.5% SDS at room temperature for 15 min, then repeated with 2×SSC, 0.5% SDS at 45° C. for 30 min, and then repeated twice with 0.2×SSC, 0.5% SDS at 50° C. for 30 min. A more preferred set of stringent conditions involves the use of higher temperatures in which the washes are identical to those above except for the temperature of the final two 30 min washes in 0.2×SSC, 0.5% SDS was increased to 60° C. Another preferred set of highly stringent conditions involves the use of two final washes in 0.1×SSC, 0.1% SDS at 65° C.</div>
</li> <li> <para-num num="[0121]"> </para-num> <div class="description-line" id="p-0115" num="0121">With respect to the degree of substantial similarity between the target (endogenous) mRNA and the RNA region in the construct having homology to the target mRNA, such sequences should be at least 25 nucleotides in length, preferably at least 50 nucleotides in length, more preferably at least 100 nucleotides in length, again more preferably at least 200 nucleotides in length, and most preferably at least 300 nucleotides in length; and should be at least 80% identical, preferably at least 85% identical, more preferably at least 90% identical, and most preferably at least 95% identical.</div>
</li> <li> <para-num num="[0122]"> </para-num> <div class="description-line" id="p-0116" num="0122">Substantially similar nucleic acid fragments may be selected by screening nucleic acid fragments representing subfragments or modifications of the nucleic acid fragments of the instant invention, wherein one or more nucleotides are substituted, deleted and/or inserted, for their ability to affect the level of the polypeptide encoded by the unmodified nucleic acid fragment in a plant or plant cell. For example, a substantially similar nucleic acid fragment representing at least 30 contiguous nucleotides, preferably at least 40 contiguous nucleotides, most preferably at least 60 contiguous nucleotides derived from the instant nucleic acid fragment can be constructed and introduced into a plant or plant cell. The level of the polypeptide encoded by the unmodified nucleic acid fragment present in a plant or plant cell exposed to the substantially similar nucleic fragment can then be compared to the level of the polypeptide in a plant or plant cell that is not exposed to the substantially similar nucleic acid fragment.</div>
</li> <li> <para-num num="[0123]"> </para-num> <div class="description-line" id="p-0117" num="0123">Sequence alignments and percent similarity calculations may be determined using a variety of comparison methods designed to detect homologous sequences including, but not limited to, the Megalign program of the LASARGENE bioinformatics computing suite (DNASTAR Inc., Madison, Wis.). Multiple alignment of the sequences are performed using the Clustal V method of alignment (Higgins and Sharp (1989) <i>CABIOS. </i>5:151-153) with the default parameters (GAP PENALTY=10, GAP LENGTH PENALTY=10). Default parameters for pairwise alignments and calculation of percent identity of protein sequences using the Clustal method are KTUPLE=1, GAP PENALTY=3, WINDOW=5 and DIAGONALS SAVED=5. For nucleic acids these parameters are KTUPLE=2, GAP PENALTY=5, WINDOW=4 and DIAGONALS SAVED=4.</div>
</li> <li> <para-num num="[0124]"> </para-num> <div class="description-line" id="p-0118" num="0124">“Gene” refers to a nucleic acid fragment that expresses a specific protein, including regulatory sequences preceding (5′ non-coding sequences) and following (3′ non-coding sequences) the coding sequence. “Native gene” refers to a gene as found in nature with its own regulatory sequences. “Recombinant DNA construct” refers to a combination of nucleic acid fragments that are not normally found together in nature. Accordingly, a recombinant DNA construct may comprise regulatory sequences and coding sequences that are derived from different sources, or regulatory sequences and coding sequences derived from the same source, but arranged in a manner different than that normally found in nature. A “foreign” gene refers to a gene not normally found in the host organism, but that is introduced into the host organism by gene transfer. Foreign genes can comprise native genes inserted into a non-native organism, or recombinant DNA constructs. A “transgene” is a gene that has been introduced into the genome by a transformation procedure.</div>
</li> <li> <para-num num="[0125]"> </para-num> <div class="description-line" id="p-0119" num="0125">“Coding sequence” refers to a DNA sequence that codes for a specific amino acid sequence. “Regulatory sequences” refer to nucleotide sequences located upstream (5′ non-coding sequences), within, or downstream (3′ non-coding sequences) of a coding sequence, and which influence the transcription, RNA processing or stability, or translation of the associated coding sequence. Regulatory sequences may include, but are not limited to, promoters, translation leader sequences, introns, and polyadenylation recognition sequences.</div>
</li> <li> <para-num num="[0126]"> </para-num> <div class="description-line" id="p-0120" num="0126">“Promoter” refers to a DNA sequence capable of controlling the expression of a coding sequence or functional RNA. The promoter sequence consists of proximal and more distal upstream elements, the latter elements often referred to as enhancers. Accordingly, an “enhancer” is a DNA sequence which can stimulate promoter activity and may be an innate element of the promoter or a heterologous element inserted to enhance the level or tissue-specificity of a promoter. Promoter sequences can also be located within the transcribed portions of genes, and/or downstream of the transcribed sequences. Promoters may be derived in their entirety from a native gene, or be composed of different elements derived from different promoters found in nature, or even comprise synthetic DNA segments. It is understood by those skilled in the art that different promoters may direct the expression of an isolated nucleic acid fragment in different tissues or cell types, or at different stages of development, or in response to different environmental conditions. Promoters, which cause an isolated nucleic acid fragment to be expressed in most cell types, at most times are commonly referred to as “constitutive promoters”. New promoters of various types useful in plant cells are constantly being discovered; numerous examples may be found in the compilation by Okamuro and Goldberg, (1989) Biochemistry of Plants 15:1-82.</div>
</li> <li> <para-num num="[0127]"> </para-num> <div class="description-line" id="p-0121" num="0127">It is further recognized that since in most cases the exact boundaries of regulatory sequences have not been completely defined, DNA fragments of some variation may have identical promoter activity. As used herein, “substantially similar and functionally equivalent subfragment of a promoter” refers to a portion or subsequence of a promoter sequence which is capable of controlling the expression of a coding sequence or functional RNA.</div>
</li> <li> <para-num num="[0128]"> </para-num> <div class="description-line" id="p-0122" num="0128">Specific examples of promoters that may be useful in expressing the nucleic acid fragments of the invention include, but are not limited to, the promoters disclosed in this application (SEQ ID NOs: δ 37, 38, 46, 47, 56, 65, 67, 68, 69, 70, 71, 72, 73, 74, 89 or 90).</div>
</li> <li> <para-num num="[0129]"> </para-num> <div class="description-line" id="p-0123" num="0129">An “intron” is an intervening sequence in a gene that does not encode a portion of the protein sequence. Thus, such sequences are transcribed into RNA but are then excised and are not translated. The term is also used for the excised RNA sequences.</div>
</li> <li> <para-num num="[0130]"> </para-num> <div class="description-line" id="p-0124" num="0130">An “exon” is a portion of the sequence of a gene that is transcribed and is found in the mature messenger RNA derived from the gene, but is not necessarily a part of the sequence that encodes the final gene product.</div>
</li> <li> <para-num num="[0131]"> </para-num> <div class="description-line" id="p-0125" num="0131">The term “deduced nucleotide sequence” refers to a DNA sequence after removal of intervening sequences, based on homology to other DNA sequences encoding the same protein.</div>
</li> <li> <para-num num="[0132]"> </para-num> <div class="description-line" id="p-0126" num="0132">The term “deduced amino acid sequence” refers to a polypeptide sequence derived from a DNA sequence after removal of intervening sequences, based on homology to other proteins encoded by DNA sequences encoding the same protein.</div>
</li> <li> <para-num num="[0133]"> </para-num> <div class="description-line" id="p-0127" num="0133">The term “translation leader sequence” refers to a DNA sequence located between the promoter sequence of a gene and the coding sequence. The translation leader sequence is present in the fully processed mRNA upstream of the translation start sequence. The translation leader sequence may affect processing of the primary transcript to mRNA, mRNA stability or translation efficiency.</div>
</li> <li> <para-num num="[0134]"> </para-num> <div class="description-line" id="p-0128" num="0134">Examples of translation leader sequences have been described (Turner, R. and Foster, G. D. (1995) <i>Molecular Biotechnology </i>3:225).</div>
</li> <li> <para-num num="[0135]"> </para-num> <div class="description-line" id="p-0129" num="0135">The “3′ non-coding sequences” refer to DNA sequences located downstream of a coding sequence and include polyadenylation recognition sequences and other sequences encoding regulatory signals capable of affecting mRNA processing or gene expression. The polyadenylation signal is usually characterized by affecting the addition of polyadenylic acid tracts to the 3′ end of the mRNA precursor. The use of different 3′ non-coding sequences is exemplified by Ingelbrecht et al., (1989) <i>Plant Cell </i>1:671-680.</div>
</li> <li> <para-num num="[0136]"> </para-num> <div class="description-line" id="p-0130" num="0136">“RNA transcript” refers to the product resulting from RNA polymerase-catalyzed transcription of a DNA sequence. When the RNA transcript is a perfect complementary copy of the DNA sequence, it is referred to as the primary transcript or it may be a RNA sequence derived from post-transcriptional processing of the primary transcript and is referred to as the mature RNA. “Messenger RNA (mRNA)” refers to the RNA that is without introns and that can be translated into protein by the cell. “cDNA” refers to a DNA that is complementary to and synthesized from a mRNA template using the enzyme reverse transcriptase. The cDNA can be single-stranded or converted into the double-stranded form using the Klenow fragment of DNA polymerase I. “Sense” RNA refers to RNA transcript that includes the mRNA and can be translated into protein within a cell or in vitro. “Antisense RNA” refers to an RNA transcript that is complementary to all or part of a target primary transcript or mRNA and that blocks the expression of a target isolated nucleic acid fragment (U.S. Pat. No. 5,107,065). The complementarity of an antisense RNA may be with any part of the specific gene transcript, i.e., at the 5′ non-coding sequence, 3′ non-coding sequence, introns, or the coding sequence. “Functional RNA” refers to antisense RNA, ribozyme RNA, or other RNA that may not be translated but yet has an effect on cellular processes. The terms “complement” and “reverse complement” are used interchangeably herein with respect to mRNA transcripts, and are meant to define the antisense RNA of the message.</div>
</li> <li> <para-num num="[0137]"> </para-num> <div class="description-line" id="p-0131" num="0137">The term “endogenous RNA” refers to any RNA which is encoded by any nucleic acid sequence present in the genome of the host, whether naturally-occurring or non-naturally occurring, i.e., introduced by recombinant means, mutagenesis, etc.</div>
</li> <li> <para-num num="[0138]"> </para-num> <div class="description-line" id="p-0132" num="0138">The term “non-naturally occurring” means artificial, not consistent with what is normally found in nature.</div>
</li> <li> <para-num num="[0139]"> </para-num> <div class="description-line" id="p-0133" num="0139">The term “operably linked” refers to the association of nucleic acid sequences on a single nucleic acid fragment so that the function of one is regulated by the other. For example, a promoter is operably linked with a coding sequence when it is capable of regulating the expression of that coding sequence (i.e., that the coding sequence is under the transcriptional control of the promoter). Coding sequences can be operably linked to regulatory sequences in a sense or antisense orientation. In another example, the complementary RNA regions of the invention can be operably linked, either directly or indirectly, 5′ to the target mRNA, or 3′ to the target mRNA, or within the target mRNA, or a first complementary region is 5′ and its complement is 3′ to the target mRNA.</div>
</li> <li> <para-num num="[0140]"> </para-num> <div class="description-line" id="p-0134" num="0140">The term “expression”, as used herein, refers to the production of a functional end-product. Expression of an isolated nucleic acid fragment involves transcription of the isolated nucleic acid fragment and translation of the mRNA into a precursor or mature protein. “Antisense inhibition” refers to the production of antisense RNA transcripts capable of suppressing the expression of the target protein. “Co-suppression” refers to the production of sense RNA transcripts capable of suppressing the expression of identical or substantially similar foreign or endogenous genes (U.S. Pat. No. 5,231,020).</div>
</li> <li> <para-num num="[0141]"> </para-num> <div class="description-line" id="p-0135" num="0141">“Mature” protein refers to a post-translationally processed polypeptide; i.e., one from which any pre- or propeptides present in the primary translation product have been removed. “Precursor” protein refers to the primary product of translation of mRNA; i.e., with pre- and propeptides still present. Pre- and propeptides may be but are not limited to intracellular localization signals.</div>
</li> <li> <para-num num="[0142]"> </para-num> <div class="description-line" id="p-0136" num="0142">“Stable transformation” refers to the transfer of a nucleic acid fragment into a genome of a host organism, including both nuclear and organellar genomes, resulting in genetically stable inheritance. In contrast, “transient transformation” refers to the transfer of a nucleic acid fragment into the nucleus, or DNA-containing organelle, of a host organism resulting in gene expression without integration or stable inheritance. Host organisms containing the transformed nucleic acid fragments are referred to as “transgenic” organisms. The preferred method of cell transformation of rice, corn and other monocots is the use of particle-accelerated or “gene gun” transformation technology (Klein et al., (1987) <i>Nature </i>(<i>London</i>) 327:70-73; U.S. Pat. No. 4,945,050), or an <i>Agrobacterium</i>-mediated method using an appropriate Ti plasmid containing the transgene (Ishida Y. et al., 1996, <i>Nature Biotech. </i>14:745-750). The term “transformation and “transformed” as used herein refer to both stable transformation and transient transformation.</div>
</li> <li> <para-num num="[0143]"> </para-num> <div class="description-line" id="p-0137" num="0143">Standard recombinant DNA and molecular cloning techniques used herein are well known in the art and are described more fully in Sambrook, J., Fritsch, E. F. and Maniatis, T. <i>Molecular Cloning: A Laboratory Manual</i>; Cold Spring Harbor Laboratory Press: Cold Spring Harbor, 1989 (hereinafter “Sambrook”).</div>
</li> <li> <para-num num="[0144]"> </para-num> <div class="description-line" id="p-0138" num="0144">The term “recombinant” refers to an artificial combination of two otherwise separated segments of sequence, e.g., by chemical synthesis or by the manipulation of isolated segments of nucleic acids by genetic engineering techniques.</div>
</li> <li> <para-num num="[0145]"> </para-num> <div class="description-line" id="p-0139" num="0145">“PCR” or “Polymerase Chain Reaction” is a technique for the synthesis of large quantities of specific DNA segments, consists of a series of repetitive cycles (Perkin Elmer Cetus Instruments, Norwalk, Conn.). Typically, the double stranded DNA is heat denatured, the two primers complementary to the 3′ boundaries of the target segment are annealed at low temperature and then extended at an intermediate temperature. One set of these three consecutive steps is referred to as a cycle.</div>
</li> <li> <para-num num="[0146]"> </para-num> <div class="description-line" id="p-0140" num="0146">Polymerase chain reaction (“PCR”) is a powerful technique used to amplify DNA millions of fold, by repeated replication of a template, in a short period of time. (Mullis et al, <i>Cold Spring Harbor Symp. Quant. Biol. </i>51:263-273 (1986); Erlich et al, European Patent Application 50,424; European Patent Application 84,796; European Patent Application 258,017, European Patent Application 237,362; Mullis, European Patent Application 201,184, Mullis et al U.S. Pat. No. 4,683,202; Erlich, U.S. Pat. No. 4,582,788; and Saiki et al, U.S. Pat. No. 4,683,194). The process utilizes sets of specific in vitro synthesized oligonucleotides to prime DNA synthesis. The design of the primers is dependent upon the sequences of DNA that are desired to be analyzed. The technique is carried out through many cycles (usually 20-50) of melting the template at high temperature, allowing the primers to anneal to complementary sequences within the template and then replicating the template with DNA polymerase.</div>
</li> <li> <para-num num="[0147]"> </para-num> <div class="description-line" id="p-0141" num="0147">The products of PCR reactions are analyzed by separation in agarose gels followed by ethidium bromide staining and visualization with UV transillumination. Alternatively, radioactive dNTPs can be added to the PCR in order to incorporate label into the products. In this case the products of PCR are visualized by exposure of the gel to x-ray film. The added advantage of radiolabeling PCR products is that the levels of individual amplification products can be quantitated.</div>
</li> <li> <para-num num="[0148]"> </para-num> <div class="description-line" id="p-0142" num="0148">The terms “recombinant construct”, “expression construct” and “recombinant expression construct” are used interchangeably herein. These terms refer to a functional unit of genetic material that can be inserted into the genome of a cell using standard methodology well known to one skilled in the art. Such construct may be itself or may be used in conjunction with a vector. If a vector is used then the choice of vector is dependent upon the method that will be used to transform host plants as is well known to those skilled in the art. For example, a plasmid vector can be used. The skilled artisan is well aware of the genetic elements that must be present on the vector in order to successfully transform, select and propagate host cells comprising any of the isolated nucleic acid fragments of the invention. The skilled artisan will also recognize that different independent transformation events will result in different levels and patterns of expression (Jones et al., (1985) <i>EMBO J. </i>4:2411-2418; De Almeida et al., (1989) <i>Mol. Gen. Genetics </i>218:78-86), and thus that multiple events must be screened in order to obtain lines displaying the desired expression level and pattern. Such screening may be accomplished by Southern analysis of DNA, Northern analysis of mRNA expression, Western analysis of protein expression, or phenotypic analysis.</div>
</li> <li> <para-num num="[0149]"> </para-num> <div class="description-line" id="p-0143" num="0149">Co-suppression constructs in plants previously have been designed by focusing on overexpression of a nucleic acid sequence having homology to an endogenous mRNA, in the sense orientation, which results in the reduction of all RNA having homology to the overexpressed sequence (see Vaucheret et al. (1998) <i>Plant J </i>16:651-659; and Gura (2000) <i>Nature </i>404:804-808). The overall efficiency of this phenomenon is low, and the extent of the RNA reduction is widely variable. Recent work has described the use of “hairpin” structures that incorporate all, or part, of an mRNA encoding sequence in a complementary orientation that results in a potential “stem-loop” structure for the expressed RNA (PCT Publication WO 99/53050 published on Oct. 21, 1999). This increases the frequency of co-suppression in the recovered transgenic plants. Another variation describes the use of plant viral sequences to direct the suppression, or “silencing”, of proximal mRNA encoding sequences (PCT Publication WO 98/36083 published on Aug. 20, 1998). Both of these co-suppressing phenomena have not been elucidated mechanistically, although recent genetic evidence has begun to unravel this complex situation (Elmayan et al. (1998) <i>Plant Cell </i>10:1747-1757).</div>
</li> <li> <para-num num="[0150]"> </para-num> <div class="description-line" id="p-0144" num="0150">In one aspect, this invention includes an isolated polynucleotide comprising a nucleotide sequence encoding a polypeptide required for high affinity nitrate transport, wherein the polypeptide has an amino acid sequence of at least 80%, 85%, 90%, 95%, or 99% sequence identity, based on the Clustal V method of alignment, when compared to one of SEQ ID NO: 36 or 49. The polypeptide may also comprise SEQ ID NO: 36 or 49, and the nucleotide sequence may comprise SEQ ID NO: 35 or 48.</div>
</li> <li> <para-num num="[0151]"> </para-num> <div class="description-line" id="p-0145" num="0151">Also included in the present invention is a complement of any of the foregoing nucleotide sequences, wherein the complement and the nucleotide sequence consist of the same number of nucleotides and are 100% complementary.</div>
</li> <li> <para-num num="[0152]"> </para-num> <div class="description-line" id="p-0146" num="0152">In another aspect, this invention includes isolated polynucleotides as described herein (or complements), wherein the nucleotide sequence comprises at least two, three, four, or five motifs selected from group consisting of SEQ ID NOs: 50, 51 and 52, wherein said motif is a substantially conserved subsequence.</div>
</li> <li> <para-num num="[0153]"> </para-num> <div class="description-line" id="p-0147" num="0153">“Motifs” or “subsequences” refer to short regions of conserved sequences of nucleic acids or amino acids that comprise part of a longer sequence. For example, it is expected that such conserved subsequences (for example SEQ ID NOs: 50, 51 and 52) would be important for function, and could be used to identify new homologues of high affinity nitrate transporter-homologues in plants. It is expected that some or all of the elements may be found in a high affinity nitrate transporter-homologue. Also, it is expected that at least one or two of the conserved amino acids in any given motif may differ in a true high affinity nitrate transporter-homologue.</div>
</li> <li> <para-num num="[0154]"> </para-num> <div class="description-line" id="p-0148" num="0154">In another aspect, a polynucleotide of this invention or a functionally equivalent subfragment thereof is useful in antisense inhibition or cosuppression of expression of nucleic acid sequences encoding proteins required for high affinity nitrate transport, most preferably in antisense inhibition or cosuppression of an endogenous high affinity nitrate transporter or heterologous high affinity nitrate transporter gene.</div>
</li> <li> <para-num num="[0155]"> </para-num> <div class="description-line" id="p-0149" num="0155">Protocols for antisense inhibition or co-suppression are well known to those skilled in the art and are described above.</div>
</li> <li> <para-num num="[0156]"> </para-num> <div class="description-line" id="p-0150" num="0156">In still a further aspect, this invention includes an isolated nucleic acid fragment comprising (a) a promoter consisting essentially of SEQ ID NO: δ 37, 38, 46, 47, 56, 65, 67, 68, 69, 70, 71, 72, 73, 74, 89 or 90 or (b) a substantially similar and functionally equivalent subfragment of said promoter.</div>
</li> <li> <para-num num="[0157]"> </para-num> <div class="description-line" id="p-0151" num="0157">Also of interest are recombinant DNA constructs comprising any of the above-identified isolated nucleic acid fragments or isolated polynucleotides or complements thereof or parts of such fragments or complements, operably linked to at least one regulatory sequence.</div>
</li> <li> <para-num num="[0158]"> </para-num> <div class="description-line" id="p-0152" num="0158">Plants, plant tissue or plant cells comprising such recombinant DNA constructs in their genome are also within the scope of this invention. Transformation methods are well known to those skilled in the art and are described above. Any plant, dicot or monocot can be transformed with such recombinant DNA constructs.</div>
</li> <li> <para-num num="[0159]"> </para-num> <div class="description-line" id="p-0153" num="0159">Examples of monocots include, but are not limited to, corn, wheat, rice, sorghum, millet, barley, palm, lily, <i>Alstroemeria</i>, rye, and oat. Examples of dicots include, but are not limited to, soybean, rape, sunflower, canola, grape, guayule, columbine, cotton, tobacco, peas, beans, flax, safflower, alfalfa.</div>
</li> <li> <para-num num="[0160]"> </para-num> <div class="description-line" id="p-0154" num="0160">Plant tissue includes differentiated and undifferentiated tissues or plants, including but not limited to, roots, stems, shoots, leaves, pollen, seeds, tumor tissue, and various forms of cells and culture such as single cells, protoplasm, embryos, and callus tissue. The plant tissue may be in plant or in organ, tissue or cell culture.</div>
</li> <li> <para-num num="[0161]"> </para-num> <div class="description-line" id="p-0155" num="0161">In another aspect, this invention includes a method of altering plant nitrate transport, comprising:</div>
</li> <li> <para-num num="[0162]"> </para-num> <div class="description-line" id="p-0156" num="0162">(a) transforming a plant with a recombinant DNA construct comprising:
</div> </li> <ul> <li id="ul0009-0001" num="0000"> <ul> <li id="ul0010-0001" num="0163">i) A recombinant DNA construct comprising an isolated polynucleotide encoding a HAT polypeptide, operably linked to at least one regulatory sequence; and</li> <li id="ul0010-0002" num="0164">ii) at least one additional recombinant DNA construct comprising an isolated polynucleotide encoding a NAR polypeptide, operably linked to at least one regulatory sequence.</li> </ul> </li> </ul>
<li> <para-num num="[0165]"> </para-num> <div class="description-line" id="p-0157" num="0165">(b) growing the transformed plant of (a) under conditions suitable for the expression of the recombinant DNA construct; and selecting those transformed plants having altered nitrate transport.</div>
</li> <li> <para-num num="[0166]"> </para-num> <div class="description-line" id="p-0158" num="0166">As used herein, altering plant nitrate transport may result in increased or decreased changes.</div>
</li> <li> <para-num num="[0167]"> </para-num> <div class="description-line" id="p-0159" num="0167">The regeneration, development, and cultivation of plants from single plant protoplast transformants or from various transformed explants is well known in the art (Weissbach and Weissbach, In: Methods for Plant Molecular Biology, (Eds.), Academic Press, Inc. San Diego, Calif., (1988)). This regeneration and growth process typically includes the steps of selection of transformed cells, culturing those individualized cells through the usual stages of embryonic development through the rooted plantlet stage. Transgenic embryos and seeds are similarly regenerated. The resulting transgenic rooted shoots are thereafter planted in an appropriate plant growth medium such as soil.</div>
</li> <li> <para-num num="[0168]"> </para-num> <div class="description-line" id="p-0160" num="0168">The development or regeneration of plants containing the foreign, exogenous isolated nucleic acid fragment that encodes a protein of interest is well known in the art. Preferably, the regenerated plants are self-pollinated to provide homozygous transgenic plants. Otherwise, pollen obtained from the regenerated plants is crossed to seed-grown plants of agronomically important lines. Conversely, pollen from plants of these important lines is used to pollinate regenerated plants. A transgenic plant of the present invention containing a desired polypeptide is cultivated using methods well known to one skilled in the art.</div>
</li> <li> <para-num num="[0169]"> </para-num> <div class="description-line" id="p-0161" num="0169">There are a variety of methods for the regeneration of plants from plant tissue.</div>
</li> <li> <para-num num="[0170]"> </para-num> <div class="description-line" id="p-0162" num="0170">The particular method of regeneration will depend on the starting plant tissue and the particular plant species to be regenerated.</div>
</li> <li> <para-num num="[0171]"> </para-num> <div class="description-line" id="p-0163" num="0171">Methods for transforming dicots, primarily by use of <i>Agrobacterium tumefaciens</i>, and obtaining transgenic plants have been published for cotton (U.S. Pat. No. 5,004,863, U.S. Pat. No. 5,159,135, U.S. Pat. No. 5,518,908); soybean (U.S. Pat. No. 5,569,834, U.S. Pat. No. 5,416,011, McCabe et. al., <i>Bio/Technology </i>6:923 (1988), Christou et al., <i>Plant Physiol. </i>87:671-674 (1988)); <i>Brassica </i>(U.S. Pat. No. 5,463,174); peanut (Cheng et al., <i>Plant Cell Rep. </i>15:653-657 (1996), McKently et al., <i>Plant Cell Rep. </i>14:699-703 (1995)); papaya; and pea (Grant et al., <i>Plant Cell Rep. </i>15:254-258, (1995)).</div>
</li> <li> <para-num num="[0172]"> </para-num> <div class="description-line" id="p-0164" num="0172">Transformation of monocotyledons using electroporation, particle bombardment, and <i>Agrobacterium </i>have also been reported. Transformation and plant regeneration have been achieved in asparagus (Bytebier et al., <i>Proc. Natl. Acad. Sci</i>. (USA) 84:5354, (1987)); barley (Wan and Lemaux, Plant Physiol 104:37 (1994)); <i>Zea mays </i>(Rhodes et al., <i>Science </i>240:204 (1988), Gordon-Kamm et al., <i>Plant Cell </i>2:603-618 (1990), Fromm et al., <i>Bio/Technology </i>8:833 (1990), Koziel et al., <i>Bio/Technology </i>11: 194, (1993), Armstrong et al., <i>Crop Science </i>35:550-557 (1995)); oat (Somers et al., <i>Bio/Technology </i>10: 15 89 (1992)); orchard grass (Horn et al., <i>Plant Cell Rep. </i>7:469 (1988)); rice (Toriyama et al., <i>TheorAppl. Genet. </i>205:34, (1986); Part et al., <i>Plant Mol. Biol. </i>32:1135-1148, (1996); Abedinia et al., <i>Aust. J. Plant Physiol. </i>24:133-141 (1997); Zhang and Wu, Theor. Appl. Genet. 76:835 (1988); Zhang et al. <i>Plant Cell Rep. </i>7:379, (1988); Battraw and Hall, Plant Sci. 86:191-202 (1992); Christou et al., <i>Bio/Technology </i>9:957 (1991)); rye (De la Pena et al., <i>Nature </i>325:274 (1987)); sugarcane (Bower and Birch, Plant J. 2:409 (1992)); tall fescue (Wang et al., <i>Bio/Technology </i>10:691 (1992)), and wheat (Vasil et al., <i>Bio/Technology </i>10:667 (1992); U.S. Pat. No. 5,631,152).</div>
</li> <li> <para-num num="[0173]"> </para-num> <div class="description-line" id="p-0165" num="0173">Assays for gene expression based on the transient expression of cloned nucleic acid constructs have been developed by introducing the nucleic acid molecules into plant cells by polyethylene glycol treatment, electroporation, or particle bombardment (Marcotte et al., <i>Nature </i>335:454-457 (1988); Marcotte et al., <i>Plant Cell </i>1:523-532 (1989); McCarty et al., <i>Cell </i>66:895-905 (1991); Hattori et al., <i>Genes Dev. </i>6:609-618 (1992); Goff et al., <i>EMBO J. </i>9:2517-2522 (1990)).</div>
</li> <li> <para-num num="[0174]"> </para-num> <div class="description-line" id="p-0166" num="0174">Transient expression systems may be used to functionally dissect isolated nucleic acid fragment constructs (see generally, Maliga et al., Methods in Plant Molecular Biology, Cold Spring Harbor Press (1995)). It is understood that any of the nucleic acid molecules of the present invention can be introduced into a plant cell in a permanent or transient manner in combination with other genetic elements such as vectors, promoters, enhancers etc.</div>
</li> <li> <para-num num="[0175]"> </para-num> <div class="description-line" id="p-0167" num="0175">In addition to the above discussed procedures, practitioners are familiar with the standard resource materials which describe specific conditions and procedures for the construction, manipulation and isolation of macromolecules (e.g., DNA molecules, plasmids, etc.), generation of recombinant organisms and the screening and isolating of clones, (see for example, Sambrook et al., Molecular Cloning: A Laboratory Manual, Cold Spring Harbor Press (1989); Maliga et al., Methods in Plant Molecular Biology, Cold Spring Harbor Press (1995); Birren et al., <i>Genome Analysis: Detecting Genes, </i>1, Cold Spring Harbor, N.Y. (1998); Birren et al., Genome Analysis Analyzing DNA, 2, Cold Spring Harbor, N.Y. (1998); Plant Molecular Biology: A Laboratory Manual, eds. Clark, Springer, N.Y. (1997)).</div>
</li> <li> <para-num num="[0176]"> </para-num> <div class="description-line" id="p-0168" num="0176">In a still further aspect, this invention includes a method to isolate nucleic acid fragments encoding polypeptides associated with altering plant nitrate transport, which comprises:</div>
</li> <li> <para-num num="[0177]"> </para-num> <div class="description-line" id="p-0169" num="0177">(a) comparing SEQ ID NO: 36 or 49 with other polypeptide sequences associated with altering plant nitrate transport;</div>
</li> <li> <para-num num="[0178]"> </para-num> <div class="description-line" id="p-0170" num="0178">(b) identifying conserved sequences of 4 or more amino acids obtained in step (a);</div>
</li> <li> <para-num num="[0179]"> </para-num> <div class="description-line" id="p-0171" num="0179">(c) making region-specific nucleotide probe(s) or oligomer(s) based on the conserved sequences identified in step (b); and</div>
</li> <li> <para-num num="[0180]"> </para-num> <div class="description-line" id="p-0172" num="0180">(d) using the nucleotide probe(s) or oligomer(s) of step (c) to isolate sequences associated with altering plant nitrate transport by sequence dependent protocols.</div>
</li> <li> <para-num num="[0181]"> </para-num> <div class="description-line" id="p-0173" num="0181">Examples of conserved sequence elements that would be useful in identifying other plant sequences associated with altering plant nitrate transport can be found in the group comprising, but not limited to, the nucleotides encoding the polypeptides of SEQ ID NOs: 50, 51, and 52.</div>
</li> <li> <para-num num="[0182]"> </para-num> <div class="description-line" id="p-0174" num="0182">In another aspect, this invention also includes a method of mapping genetic variations related to altering plant nitrate transport comprising:</div>
</li> <li> <para-num num="[0183]"> </para-num> <div class="description-line" id="p-0175" num="0183">(a) crossing two plant varieties; and</div>
</li> <li> <para-num num="[0184]"> </para-num> <div class="description-line" id="p-0176" num="0184">(b) evaluating genetic variations with respect to:
</div> </li> <ul> <li id="ul0011-0001" num="0000"> <ul> <li id="ul0012-0001" num="0185">(i) a nucleic acid sequence selected from the group consisting of SEQ ID NO: 35 and 48; or</li> <li id="ul0012-0002" num="0186">(ii) a nucleic acid sequence encoding a polypeptide selected from the group consisting of SEQ ID NOs: 36 and 49 in progeny plants resulting from the cross of step (a) wherein the evaluation is made using a method selected from the group consisting of: RFLP analysis, SNP analysis, and PCR-based analysis.</li> </ul> </li> </ul>
<li> <para-num num="[0187]"> </para-num> <div class="description-line" id="p-0177" num="0187">In another embodiment, this invention includes a method of molecular breeding to obtain altered plant nitrate transport:</div>
</li> <li> <para-num num="[0188]"> </para-num> <div class="description-line" id="p-0178" num="0188">(a) crossing two plant varieties; and</div>
</li> <li> <para-num num="[0189]"> </para-num> <div class="description-line" id="p-0179" num="0189">(b) evaluating genetic variations with respect to:
</div> </li> <ul> <li id="ul0013-0001" num="0000"> <ul> <li id="ul0014-0001" num="0190">(i) a nucleic acid sequence selected from the group consisting of SEQ ID NOs: 35 and 48; or</li> <li id="ul0014-0002" num="0191">(ii) a nucleic acid sequence encoding a polypeptide selected from the group consisting of SEQ ID NOs: 36 and 49 in progeny plants resulting from the cross of step (a) wherein the evaluation is made using a method selected from the group consisting of: RFLP analysis, SNP analysis, and PCR-based analysis.</li> </ul> </li> </ul>
<li> <para-num num="[0192]"> </para-num> <div class="description-line" id="p-0180" num="0192">The terms “mapping genetic variation” or “mapping genetic variability” are used interchangeably and define the process of identifying changes in DNA sequence, whether from natural or induced causes, within a genetic region that differentiates between different plant lines, cultivars, varieties, families, or species. The genetic variability at a particular locus (gene) due to even minor base changes can alter the pattern of restriction enzyme digestion fragments that can be generated. Pathogenic alterations to the genotype can be due to deletions or insertions within the gene being analyzed or even single nucleotide substitutions that can create or delete a restriction enzyme recognition site. RFLP (restriction fragment length polymorphisms) analysis takes advantage of this and utilizes Southern blotting with a probe corresponding to the isolated nucleic acid fragment of interest.</div>
</li> <li> <para-num num="[0193]"> </para-num> <div class="description-line" id="p-0181" num="0193">Thus, if a polymorphism (i.e., a commonly occurring variation in a gene or segment of DNA; also, the existence of several forms of a gene (alleles) in the same species) creates or destroys a restriction endonuclease cleavage site, or if it results in the loss or insertion of DNA (e.g., a variable nucleotide tandem repeat (VNTR) polymorphism), it will alter the size or profile of the DNA fragments that are generated by digestion with that restriction endonuclease. As such, individuals that possess a variant sequence can be distinguished from those having the original sequence by restriction fragment analysis. Polymorphisms that can be identified in this manner are termed RFLPs. RFLPs have been widely used in human and plant genetic analyses (Glassberg, UK Patent Application 2135774; Skolnick et al, Cytogen. <i>Cell Genet. </i>32:58-67 (1982); Botstein et al, <i>Ann. J. Hum. Genet. </i>32:314-331 (1980); Fischer et al (PCT Application WO 90/13668; Uhlen, PCT Application WO 90/11369).</div>
</li> <li> <para-num num="[0194]"> </para-num> <div class="description-line" id="p-0182" num="0194">A central attribute of “single nucleotide polymorphisms” or “SNPs” is that the site of the polymorphism is at a single nucleotide. SNPs have certain reported advantages over RFLPs or VNTRs. First, SNPs are more stable than other classes of polymorphisms. Their spontaneous mutation rate is approximately 10<sup>−9 </sup>(Kornberg, DNA Replication, W.H. Freeman &amp; Co., San Francisco, 1980), approximately, 1,000 times less frequent than VNTRs (U.S. Pat. No. 5,679,524). Second, SNPs occur at greater frequency, and with greater uniformity than RFLPs and VNTRs. As SNPs result from sequence variation, sequencing random genomic or cDNA molecules can identify new polymorphisms. SNPs can also result from deletions, point mutations and insertions. Any single base alteration, whatever the cause, can be a SNP. The greater frequency of SNPs means that they can be more readily identified than the other classes of polymorphisms.</div>
</li> <li> <para-num num="[0195]"> </para-num> <div class="description-line" id="p-0183" num="0195">SNPs can be characterized using any of a variety of methods. Such methods include the direct or indirect sequencing of the site, the use of restriction enzymes where the respective alleles of the site create or destroy a restriction site, the use of allele-specific hybridization probes, the use of antibodies that are specific for the proteins encoded by the different alleles of the polymorphism or by other biochemical interpretation. SNPs can be sequenced by a number of methods. Two basic methods may be used for DNA sequencing, the chain termination method of Sanger et al, <i>Proc. Natl. Acad. Sci</i>. (U.S.A.) 74:5463-5467 (1977), and the chemical degradation method of Maxam and Gilbert, <i>Proc. Natl. Acad. Sci. </i>(U.S.A.) 74: 560-564 (1977).</div>
</li> <li> <para-num num="[0196]"> </para-num> <div class="description-line" id="p-0184" num="0196">Furthermore, single point mutations can be detected by modified PCR techniques such as the ligase chain reaction (“LCR”) and PCR-single strand conformational polymorphisms (“PCR-SSCP”) analysis. The PCR technique can also be used to identify the level of expression of genes in extremely small samples of material, e.g., tissues or cells from a body. The technique is termed reverse transcription-PCR (“RT-PCR”).</div>
</li> <li> <para-num num="[0197]"> </para-num> <div class="description-line" id="p-0185" num="0197">The term “molecular breeding” defines the process of tracking molecular markers during the breeding process. It is common for the molecular markers to be linked to phenotypic traits that are desirable. By following the segregation of the molecular marker or genetic trait, instead of scoring for a phenotype, the breeding process can be accelerated by growing fewer plants and eliminating assaying or visual inspection for phenotypic variation. The molecular markers useful in this process include, but are not limited to, any marker useful in identifying mapable genetic variations previously mentioned, as well as any closely linked genes that display synteny across plant species. The term “synteny” refers to the conservation of gene placement/order on chromosomes between different organisms. This means that two or more genetic loci, that may or may not be closely linked, are found on the same chromosome among different species. Another term for synteny is “genome colinearity”.</div>
</li> <li> <para-num num="[0198]"> </para-num> <div class="description-line" id="p-0186" num="0198">The nucleic acid fragments of the instant invention may be used to create transgenic plants in which the disclosed polypeptides are present at higher or lower levels than normal or in cell types or developmental stages in which they are not normally found. This would have the effect of altering the level of nitrogen transport and accumulation in those cells. Nitrogen deficiency in plants results in stunted growth, and many times in slender and often woody stems. In many plants the first signal of nitrogen deficiency is chlorosis (yellowing of the leaves).</div>
</li> <li> <para-num num="[0199]"> </para-num> <div class="description-line" id="p-0187" num="0199">Overexpression of the proteins of the instant invention may be accomplished by first making a recombinant DNA construct in which the coding region is operably linked to a promoter capable of directing expression of a gene in the desired tissues at the desired stage of development. For reasons of convenience, the recombinant DNA construct may comprise promoter sequences and translation leader sequences derived from the same genes. 3′ Non-coding sequences encoding transcription termination signals may also be provided. The instant recombinant DNA construct may also comprise one or more introns in order to facilitate gene expression.</div>
</li> <li> <para-num num="[0200]"> </para-num> <div class="description-line" id="p-0188" num="0200">Plasmid vectors comprising the instant recombinant DNA construct can then be made. The choice of plasmid vector is dependent upon the method that will be used to transform host plants. The skilled artisan is well aware of the genetic elements that must be present on the plasmid vector in order to successfully transform, select and propagate host cells containing the recombinant DNA construct. The skilled artisan will also recognize that different independent transformation events will result in different levels and patterns of expression (Jones et al. (1985) <i>EMBO J. </i>4:2411-2418; De Almeida et al. (1989) <i>Mol. Gen. Genetics </i>218:78-86), and thus that multiple events must be screened in order to obtain lines displaying the desired expression level and pattern. Such screening may be accomplished by Southern analysis of DNA, Northern analysis of mRNA expression, Western analysis of protein expression, or phenotypic analysis.</div>
</li> <li> <para-num num="[0201]"> </para-num> <div class="description-line" id="p-0189" num="0201">For some applications it may be useful to direct the instant polypeptides to different cellular compartments, or to facilitate its secretion from the cell. It is thus envisioned that the recombinant DNA construct described above may be further supplemented by altering the coding sequence to encode the instant polypeptides with appropriate intracellular targeting sequences such as transit sequences (Keegstra (1989) <i>Cell </i>56:247-253), signal sequences or sequences encoding endoplasmic reticulum localization (Chrispeels (1991) <i>Ann. Rev. Plant Phys. Plant Mol. Biol. </i>42:21-53), or nuclear localization signals (Raikhel (1992) <i>Plant Phys. </i>100:1627-1632) added and/or with targeting sequences that are already present removed. While the references cited give examples of each of these, the list is not exhaustive and more targeting signals of utility may be discovered in the future.</div>
</li> <li> <para-num num="[0202]"> </para-num> <div class="description-line" id="p-0190" num="0202">It may also be desirable to reduce or eliminate expression of genes encoding the instant polypeptides in plants for some applications. In order to accomplish this, a recombinant DNA construct designed for co-suppression of the instant polypeptide can be constructed by linking a gene or gene fragment encoding that polypeptide to plant promoter sequences. Alternatively, a recombinant DNA construct designed to express antisense RNA for all or part of the instant nucleic acid fragment can be constructed by linking the gene or gene fragment in reverse orientation to plant promoter sequences. Either the co-suppression or antisense recombinant DNA constructs could be introduced into plants via transformation wherein expression of the corresponding endogenous genes are reduced or eliminated.</div>
</li> <li> <para-num num="[0203]"> </para-num> <div class="description-line" id="p-0191" num="0203">Molecular genetic solutions to the generation of plants with altered gene expression have a decided advantage over more traditional plant breeding approaches. Changes in plant phenotypes can be produced by specifically inhibiting expression of one or more genes by antisense inhibition or cosuppression (U.S. Pat. Nos. 5,190,931, 5,107,065 and 5,283,323). An antisense or cosuppression construct would act as a dominant negative regulator of gene activity. While conventional mutations can yield negative regulation of gene activity these effects are most likely recessive. The dominant negative regulation available with a transgenic approach may be advantageous from a breeding perspective. In addition, the ability to restrict the expression of specific phenotype to the reproductive tissues of the plant by the use of tissue specific promoters may confer agronomic advantages relative to conventional mutations which may have an effect in all tissues in which a mutant gene is ordinarily expressed.</div>
</li> <li> <para-num num="[0204]"> </para-num> <div class="description-line" id="p-0192" num="0204">The person skilled in the art will know that special considerations are associated with the use of antisense or cosuppression technologies in order to reduce expression of particular genes. For example, the proper level of expression of sense or antisense genes may require the use of different recombinant DNA constructs utilizing different regulatory elements known to the skilled artisan. Once transgenic plants are obtained by one of the methods described above, it will be necessary to screen individual transgenics for those that most effectively display the desired phenotype. Accordingly, the skilled artisan will develop methods for screening large numbers of transformants. The nature of these screens will generally be chosen on practical grounds, and is not an inherent part of the invention. For example, one can screen by looking for changes in gene expression by using antibodies specific for the protein encoded by the gene being suppressed, or one could establish assays that specifically measure enzyme activity. A preferred method will be one which allows large numbers of samples to be processed rapidly, since it will be expected that a large number of transformants will be negative for the desired phenotype.</div>
</li> <li> <para-num num="[0205]"> </para-num> <div class="description-line" id="p-0193" num="0205">The instant polypeptides (or portions thereof) may be produced in heterologous host cells, particularly in the cells of microbial hosts, and can be used to prepare antibodies to these proteins by methods well known to those skilled in the art. The antibodies are useful for detecting the polypeptides of the instant invention in situ in cells or in vitro in cell extracts. Preferred heterologous host cells for production of the instant polypeptides are microbial hosts. Microbial expression systems and expression vectors containing regulatory sequences that direct high level expression of foreign proteins are well known to those skilled in the art. Any of these could be used to construct a recombinant DNA construct for production of the instant polypeptides. This recombinant DNA construct could then be introduced into appropriate microorganisms via transformation to provide high level expression of the encoded ammonium transporter. An example of a vector for high level expression of the instant polypeptides in a bacterial host is provided (Example 7).</div>
</li> <li> <para-num num="[0206]"> </para-num> <div class="description-line" id="p-0194" num="0206">Additionally, the instant polypeptides can be used as targets to facilitate design and/or identification of inhibitors of those enzymes that may be useful as herbicides. This is desirable because the polypeptides described herein catalyze various steps in nitrogen uptake. Accordingly, inhibition of the activity of one or more of the enzymes described herein could lead to inhibition of plant growth. Thus, the instant polypeptides could be appropriate for new herbicide discovery and design.</div>
</li> <li> <para-num num="[0207]"> </para-num> <div class="description-line" id="p-0195" num="0207">All or a substantial portion of the nucleic acid fragments of the instant invention may also be used as probes for genetically and physically mapping the genes that they are a part of, and as markers for traits linked to those genes. Such information may be useful in plant breeding in order to develop lines with desired phenotypes. For example, the instant nucleic acid fragments may be used as restriction fragment length polymorphism (RFLP) markers. Southern blots (Maniatis) of restriction-digested plant genomic DNA may be probed with the nucleic acid fragments of the instant invention. The resulting banding patterns may then be subjected to genetic analyses using computer programs such as MapMaker (Lander et al. (1987) <i>Genomics </i>1:174-181) in order to construct a genetic map. In addition, the nucleic acid fragments of the instant invention may be used to probe Southern blots containing restriction endonuclease-treated genomic DNAs of a set of individuals representing parent and progeny of a defined genetic cross. Segregation of the DNA polymorphisms is noted and used to calculate the position of the instant nucleic acid sequence in the genetic map previously obtained using this population (Botstein et al. (1980) <i>Am. J. Hum. Genet. </i>32:314-331).</div>
</li> <li> <para-num num="[0208]"> </para-num> <div class="description-line" id="p-0196" num="0208">The production and use of plant gene-derived probes for use in genetic mapping is described in Bernatzky and Tanksley (1986) <i>Plant Mol. Biol. Reporter </i>4(1):37-41. Numerous publications describe genetic mapping of specific cDNA clones using the methodology outlined above or variations thereof. For example, F2 intercross populations, backcross populations, randomly mated populations, near isogenic lines, and other sets of individuals may be used for mapping. Such methodologies are well known to those skilled in the art.</div>
</li> <li> <para-num num="[0209]"> </para-num> <div class="description-line" id="p-0197" num="0209">Nucleic acid probes derived from the instant nucleic acid sequences may also be used for physical mapping (i.e., placement of sequences on physical maps; see Hoheisel et al. In: <i>Nonmammalian Genomic Analysis: A Practical Guide</i>, Academic press 1996, pp. 319-346, and references cited therein).</div>
</li> <li> <para-num num="[0210]"> </para-num> <div class="description-line" id="p-0198" num="0210">In another embodiment, nucleic acid probes derived from the instant nucleic acid sequences may be used in direct fluorescence in situ hybridization (FISH) mapping (Trask (1991) <i>Trends Genet. </i>7:149-154). Although current methods of FISH mapping favor use of large clones (several to several hundred KB; see Laan et al. (1995) <i>Genome Research </i>5:13-20), improvements in sensitivity may allow performance of FISH mapping using shorter probes.</div>
</li> <li> <para-num num="[0211]"> </para-num> <div class="description-line" id="p-0199" num="0211">A variety of nucleic acid amplification-based methods of genetic and physical mapping may be carried out using the instant nucleic acid sequences. Examples include allele-specific amplification (Kazazian (1989) <i>J. Lab. Clin. Med. </i>11:95-96), polymorphism of PCR-amplified fragments (CAPS; Sheffield et al. (1993) <i>Genomics </i>16:325-332), allele-specific ligation (Landegren et al. (1988) <i>Science </i>241:1077-1080), nucleotide extension reactions (Sokolov (1990) <i>Nucleic Acid Res. </i>18:3671), Radiation Hybrid Mapping (Walter et al. (1997) <i>Nature Genetics </i>7:22-28) and Happy Mapping (Dear and Cook (1989) <i>Nucleic Acid Res. </i>17:6795-6807). For these methods, the sequence of a nucleic acid fragment is used to design and produce primer pairs for use in the amplification reaction or in primer extension reactions. The design of such primers is well known to those skilled in the art. In methods employing PCR-based genetic mapping, it may be necessary to identify DNA sequence differences between the parents of the mapping cross in the region corresponding to the instant nucleic acid sequence. This, however, is generally not necessary for mapping methods.</div>
</li> <li> <para-num num="[0212]"> </para-num> <div class="description-line" id="p-0200" num="0212">Loss of function mutant phenotypes may be identified for the instant cDNA clones either by targeted gene disruption protocols or by identifying specific mutants for these genes contained in a maize population carrying mutations in all possible genes (Ballinger and Benzer (1989) <i>Proc. Natl. Acad. Sci. USA </i>86:9402-9406; Koes et al. (1995) <i>Proc. Natl. Acad. Sci. USA </i>92:8149-8153; Bensen et al. (1995) <i>Plant Cell </i>7:75-84). The latter approach may be accomplished in two ways. First, short segments of the instant nucleic acid fragments may be used in polymerase chain reaction protocols in conjunction with a mutation tag sequence primer on DNAs prepared from a population of plants in which Mutator transposons or some other mutation-causing DNA element has been introduced (see Bensen, supra). The amplification of a specific DNA fragment with these primers indicates the insertion of the mutation tag element in or near the plant gene encoding the instant polypeptides. Alternatively, the instant nucleic acid fragment may be used as a hybridization probe against PCR amplification products generated from the mutation population using the mutation tag sequence primer in conjunction with an arbitrary genomic site primer, such as that for a restriction enzyme site-anchored synthetic adaptor. With either method, a plant containing a mutation in the endogenous gene encoding the instant polypeptides can be identified and obtained. This mutant plant can then be used to determine or confirm the natural function of the instant polypeptides disclosed herein.</div>
</li> <li> <para-num num="[0213]"> </para-num> <div class="description-line" id="p-0201" num="0213">The function of the high affinity nitrate transporters and polypeptides required for high affinity nitrate transport can be confirmed using the TUSC Mutant population. The Trait Utility System for Corn (TUSC) is a method that employs genetic and molecular techniques to facilitate the study of gene function in maize. Studying gene function implies that the gene's sequence is already known, thus the method works in reverse: from sequence to phenotype. This kind of application is referred to as “reverse genetics”, which contrasts with “forward” methods (such as transposon tagging) that are designed to identify and isolate the gene(s) responsible for a particular trait (phenotype).</div>
</li> <li> <para-num num="[0214]"> </para-num> <div class="description-line" id="p-0202" num="0214">Pioneer Hi-Bred International, Inc., has its proprietary collection of maize genomic DNA from approximately 42,000 individual F<sub>1 </sub>plants (Reverse genetics for maize; Meeley, R and Briggs, S, 1995, Maize Genet. Coop. Newslett. 69:67, 82).</div>
</li> <li> <para-num num="[0215]"> </para-num> <div class="description-line" id="p-0203" num="0215">The genome of each of these individuals contains multiple copies of the transposable element family, Mutator (Mu). The Mu family is highly mutagenic; in the presence of the active element Mu-DR, these elements transpose throughout the genome, inserting into genic regions, and often disrupting gene function. By collecting genomic DNA from a large number of individuals (42,000), Pioneer has assembled a library of the mutagenized maize genome. Mu insertion events are predominately heterozygous so; given the recessive nature of most insertional mutations, the F<sub>1 </sub>plants appear wild-type. Each of the plants was selfed to produce F<sub>2 </sub>seed, which was collected. In generating the F<sub>2 </sub>progeny, insertional mutations segregate in a Mendelian fashion and therefore are useful for investigating a mutant allele's effect on the phenotype. The TUSC system has been successfully used by a number of laboratories to identify the function of a variety of genes (Cloning and characterization of the maize An1 gene, Bensen, R J et al., 1995, Plant Cell 7:75-84; Diversification of C-function activity in maize flower development, Mena, M et al., 1996, Science 274:1537-1540; Analysis of a chemical plant defense mechanism in grasses, Frey, Metal., 1997, Science 277:696-699; The control of maize spikelet meristem fate by the APETALA2-like gene Indeterminate spikelet 1, Chuck, G, Meeley, R B, and Hake, S, 1998, Genes &amp; Development 12:1145-1154; A SecY homologue is required for the elaboration of the chloroplast thylakoid membrane and for normal chloroplast gene expression, Roy, L M and Barkan, A., 1998, J. Cell Biol. 141:1-11).</div>
</li> <li> <para-num num="[0216]"> </para-num> <div class="description-line" id="p-0204" num="0216">Polynucleotide sequences produced by diversity generation methods or recursive sequence recombination (“RSR”) methods (e.g., DNA shuffling) are a feature of the invention. Mutation and recombination methods using the nucleic acids described herein are a feature of the invention. For example, one method of the invention includes recursively recombining one or more nucleotide sequences of the invention as described above and below with one or more additional nucleotides. The recombining steps are optionally performed in vivo, ex vivo, in silico or in vitro. This diversity generation or recursive sequence recombination produces at least one library of recombinant modified HAT polynucleotides. Polypeptides encoded by members of this library are included in the invention.</div>
</li> <li> <para-num num="[0217]"> </para-num> <div class="description-line" id="p-0205" num="0217">Descriptions of a variety of diversity generating procedures, including multigene shuffling and methods for generating modified nucleic acid sequences encoding multiple enzymatic domains, are found the following publications and the references cited therein: Soong, N. et al. (2000) “Molecular breeding of viruses” Nat Genet. 25(4):436-39; Stemmer, et al. (1999) “Molecular breeding of viruses for targeting and other clinical properties” Tumor Targeting 4:1-4; Ness et al. (1999) “DNA Shuffling of subgenomic sequences of subtilisin” Nature Biotechnology 17:893-896; Chang et al. (1999) “Evolution of a cytokine using DNA family shuffling” Nature Biotechnology 17:793-797; Minshull and Stemmer (1999) “Protein evolution by molecular breeding” Current Opinion in Chemical Biology 3:284-290; Christians et al. (1999) “Directed evolution of thymidine kinase for AZT phosphorylation using DNA family shuffling” Nature Biotechnology 17:259-264; Crameri et al. (1998) “DNA shuffling of a family of genes from diverse species accelerates directed evolution” Nature 391:288-291; Crameri et al. (1997) “Molecular evolution of an arsenate detoxification pathway by DNA shuffling,” Nature Biotechnology 15:436-438; Zhang et al. (1997) “Directed evolution of an effective fucosidase from a galactosidase by DNA shuffling and screening” Proc. Natl. Acad. Sci. USA 94:4504-4509; Patten et al. (1997) “Applications of DNA Shuffling to Pharmaceuticals and Vaccines” Current Opinion in Biotechnology 8:724-733; Crameri et al. (1996) “Construction and evolution of antibody-phage libraries by DNA shuffling” Nature Medicine 2:100-103; Crameri et al. (1996) “Improved green fluorescent protein by molecular evolution using DNA shuffling” Nature Biotechnology 14:315-319; Gates et al. (1996) “Affinity selective isolation of ligands from peptide libraries through display on a lac repressor ‘headpiece dimer’” Journal of Molecular Biology 255:373-386; Stemmer (1996) “Sexual PCR and Assembly PCR” In: The Encyclopedia of Molecular Biology. VCH Publishers, New York. pp. 447-457; Crameri and Stemmer (1995) “Combinatorial multiple cassette mutagenesis creates all the permutations of mutant and wildtype cassettes” BioTechniques 18:194-195; Stemmer et al., (1995) “Single-step assembly of a gene and entire plasmid from large numbers of oligodeoxy-ribonucleotides” Gene, 164:49-53; Stemmer (1995) “The Evolution of Molecular Computation” Science 270: 1510; Stemmer (1995) “Searching Sequence Space” Bio/Technology 13:549-553; Stemmer (1994) “Rapid evolution of a protein in vitro by DNA shuffling” Nature 370:389-391; and Stemmer (1994) “DNA shuffling by random fragmentation and reassembly: In vitro recombination for molecular evolution.” Proc. Natl. Acad. Sci. USA 91:10747-10751. Additional details regarding various diversity generating methods can be found in the following U.S. patents, PCT publications, and EPO publications: U.S. Pat. No. 5,605,793 to Stemmer (Feb. 25, 1997), “Methods for In Vitro Recombination;” U.S. Pat. No. 5,811,238 to Stemmer et al. (Sep. 22, 1998) “Methods for Generating Polynucleotides having Desired Characteristics by Iterative Selection and Recombination;” U.S. Pat. No. 5,830,721 to Stemmer et al. (Nov. 3, 1998), “DNA Mutagenesis by Random Fragmentation and Reassembly;” U.S. Pat. No. 5,834,252 to Stemmer, et al. (Nov. 10, 1998) “End-Complementary Polymerase Reaction;” U.S. Pat. No. 5,837,458 to Minshull, et al. (Nov. 17, 1998), “Methods and Compositions for Cellular and Metabolic Engineering;” WO 95/22625, Stemmer and Crameri, “Mutagenesis by Random Fragmentation and Reassembly;” WO 96/33207 by Stemmer and Lipschutz “End Complementary Polymerase Chain Reaction;” WO 97/20078 by Stemmer and Crameri “Methods for Generating Polynucleotides having Desired Characteristics by Iterative Selection and Recombination;” WO 97/35966 by Minshull and Stemmer, “Methods and Compositions for Cellular and Metabolic Engineering;” WO 99/41402 by Punnonen et al. “Targeting of Genetic Vaccine Vectors;” WO 99/41383 by Punnonen et al. “Antigen Library Immunization;” WO 99/41369 by Punnonen et al. “Genetic Vaccine Vector Engineering;” WO 99/41368 by Punnonen et al. “Optimization of Immunomodulatory Properties of Genetic Vaccines;” EP 752008 by Stemmer and Crameri, “DNA Mutagenesis by Random Fragmentation and Reassembly;” EP 0932670 by Stemmer “Evolving Cellular DNA Uptake by Recursive Sequence Recombination;” WO 99/23107 by Stemmer et al., “Modification of Virus Tropism and Host Range by Viral Genome Shuffling;” WO 99/21979 by Apt et al., “Human Papillomavirus Vectors;” WO 98/31837 by del Cardayre et al. “Evolution of Whole Cells and Organisms by Recursive Sequence Recombination;” WO 98/27230 by Patten and Stemmer, “Methods and Compositions for Polypeptide Engineering;” WO 98/13487 by Stemmer et al., “Methods for Optimization of Gene Therapy by Recursive Sequence Shuffling and Selection;” WO 00/00632, “Methods for Generating Highly Diverse Libraries;” WO 00/09679, “Methods for Obtaining in Vitro Recombined Polynucleotide Sequence Banks and Resulting Sequences;” WO 98/42832 by Arnold et al., “Recombination of Polynucleotide Sequences Using Random or Defined Primers;” WO 99/29902 by Arnold et al., “Method for Creating Polynucleotide and Polypeptide Sequences;” WO 98/41653 by Vind, “An in Vitro Method for Construction of a DNA Library;” WO 98/41622 by Borchert et al., “Method for Constructing a Library Using DNA Shuffling;” WO 98/42727 by Pati and Zarling, “Sequence Alterations using Homologous Recombination;” WO00/18906 by Patten et al., “Shuffling of Codon-Altered Genes;” WO 00/04190 by del Cardayre et al. “Evolution of Whole Cells and Organisms by Recursive Recombination;” WO 00/42561 by Crameri et al., “Oligonucleotide Mediated Nucleic Acid Recombination;” WO 00/42559 by Selifonov and Stemmer “Methods of Populating Data Structures for Use in Evolutionary Simulations;” WO 00/42560 by Selifonov et al., “Methods for Making Character Strings, Polynucleotides &amp; Polypeptides Having Desired Characteristics;” WO 01/23401 by Welch et al., “Use of Codon-Varied Oligonucleotide Synthesis for Synthetic Shuffling;” and WO 01/64864 “Single-Stranded Nucleic Acid Template-Mediated Recombination and Nucleic Acid Fragment Isolation” by Affholter.</div>
</li> <li> <para-num num="[0218]"> </para-num> <div class="description-line" id="p-0206" num="0218">Certain U.S. applications provide additional details regarding various diversity generating methods, including “SHUFFLING OF CODON ALTERED GENES” by Patten et al. filed Sep. 28, 1999, (U.S. Ser. No. 09/407,800); “EVOLUTION OF WHOLE CELLS AND ORGANISMS BY RECURSIVE SEQUENCE RECOMBINATION”, by del Cardayre et al. filed Jul. 15, 1998 (U.S. Ser. No. 09/166,188), and Jul. 15, 1999 (U.S. Pat. No. 6,379,964); “OLIGONUCLEOTIDE MEDIATED NUCLEIC ACID RECOMBINATION” by Crameri et al., filed Sep. 28, 1999 (U.S. Pat. No. 6,376,246); “OLIGONUCLEOTIDE MEDIATED NUCLEIC ACID RECOMBINATION” by Crameri et al., filed Jan. 18, 2000 (WO 00/42561); “USE OF CODON-BASED OLIGONUCLEOTIDE SYNTHESIS FOR SYNTHETIC SHUFFLING” by Welch et al., filed Sep. 28, 1999 (U.S. Pat. No. 6,436,675); “METHODS FOR MAKING CHARACTER STRINGS, POLYNUCLEOTIDES &amp; POLYPEPTIDES HAVING DESIRED CHARACTERISTICS” by Selifonov et al., filed Jan. 18, 2000, (WO 00/42560); “METHODS FOR MAKING CHARACTER STRINGS, POLYNUCLEOTIDES &amp; POLYPEPTIDES HAVING DESIRED CHARACTERISTICS” by Selifonov et al., filed Jul. 18, 2000 (U.S. Ser. No. 09/618,579); “METHODS OF POPULATING DATA STRUCTURES FOR USE IN EVOLUTIONARY SIMULATIONS” by Selifonov and Stemmer (WO 00/42559), filed Jan. 18, 2000; and “SINGLE-STRANDED NUCLEIC ACID TEMPLATE-MEDIATED RECOMBINATION AND NUCLEIC ACID FRAGMENT ISOLATION” by Affholter (U.S. Ser. No. 60/186,482, filed Mar. 2, 2000). Synthetic recombination methods can also be used, in which oligonucleotides corresponding to targets of interest are synthesized and reassembled in PCR or ligation reactions which include oligonucleotides which correspond to more than one parental nucleic acid, thereby generating new recombined nucleic acids. Oligonucleotides can be made by standard nucleotide addition methods, or can be made, e.g., by tri-nucleotide synthetic approaches. Details regarding such approaches are found in the references noted above, including, e.g., WO 00/42561 by Crameri et al., “Oligonucleotide Mediated Nucleic Acid Recombination;” WO 01/23401 by Welch et al., “Use of Codon-Varied Oligonucleotide Synthesis for Synthetic Shuffling;” WO 00/42560 by Selifonov et al., “Methods for Making Character Strings, Polynucleotides and Polypeptides Having Desired Characteristics;” and WO 00/42559 by Selifonov and Stemmer “Methods of Populating Data Structures for Use in Evolutionary Simulations.”</div>
</li> <li> <para-num num="[0219]"> </para-num> <div class="description-line" id="p-0207" num="0219">In silico methods of recombination can be effected in which genetic algorithms are used in a computer to recombine sequence strings which correspond to homologous (or even non-homologous) nucleic acids. The resulting recombined sequence strings are optionally converted into nucleic acids by synthesis of nucleic acids, which correspond to the recombined sequences, e.g., in concert with oligonucleotide synthesis gene reassembly techniques. This approach can generate random, partially random or designed variants. Many details regarding in silico recombination, including the use of genetic algorithms, genetic operators and the like in computer systems, combined with generation of corresponding nucleic acids (and/or proteins), as well as combinations of designed nucleic acids and/or proteins (e.g., based on cross-over site selection) as well as designed, pseudo-random or random recombination methods are described in WO 00/42560 by Selifonov et al., “Methods for Making Character Strings, Polynucleotides and Polypeptides Having Desired Characteristics” and WO 00/42559 by Selifonov and Stemmer “Methods of Populating Data Structures for Use in Evolutionary Simulations.” Extensive details regarding in silico recombination methods are found in these applications. This methodology is generally applicable to the present invention in providing for recombination of nucleic acid sequences and/or gene fusion constructs encoding proteins involved in various metabolic pathways (such as, for example, carotenoid biosynthetic pathways, ectoine biosynthetic pathways, polyhydroxyalkanoate biosynthetic pathways, aromatic polyketide biosynthetic pathways, and the like) in silico and/or the generation of corresponding nucleic acids or proteins.</div>
</li> <li> <para-num num="[0220]"> </para-num> <div class="description-line" id="p-0208" num="0220">Many of the above-described methodologies for generating modified polynucleotides generate a large number of diverse variants of a parental sequence or sequences. In some preferred embodiments of the invention, the modification technique (e.g., some form of shuffling) is used to generate a library of variants that is then screened for a modified polynucleotide or pool of modified polynucleotides encoding some desired functional attribute, e.g., improved HAT activity. Exemplary enzymatic activities that can be screened for include, but are not limited to, catalytic rates (conventionally characterized in terms of kinetic constants such as k<sub>cat </sub>and K<sub>M</sub>), substrate specificity, and susceptibility to activation or inhibition by substrate, product or other molecules (e.g., inhibitors or activators) and the maximum velocity of an enzymatic reaction when the binding site is saturated with substrate (Vmax).</div>
</li> <heading id="h-0007">EXAMPLES</heading>
<li> <para-num num="[0221]"> </para-num> <div class="description-line" id="p-0209" num="0221">The present invention is further defined in the following Examples, in which all parts and percentages are by weight and degrees are Celsius, unless otherwise stated. It should be understood that these Examples, while indicating preferred embodiments of the invention, are given by way of illustration only. From the above discussion and these Examples, one skilled in the art can ascertain the essential characteristics of this invention, and without departing from the spirit and scope thereof, can make various changes and modifications of the invention to adapt it to various usages and conditions.</div>
</li> <heading id="h-0008">Example 1</heading>
<heading id="h-0009">Composition of cDNA Libraries; Isolation and Sequencing of cDNA Clones</heading>
<li> <para-num num="[0222]"> </para-num> <div class="description-line" id="p-0210" num="0222">cDNA libraries representing mRNAs from various corn tissues were prepared. The characteristics of the libraries are described in Table 1.</div>
</li> <li> <para-num num="[0223]"> </para-num> <div class="description-line" id="p-0211" num="0223">cDNA libraries may be prepared by any one of many available methods. For example, the cDNAs may be introduced into plasmid vectors by first preparing the cDNA libraries in Uni-ZAP™ XR vectors according to the manufacturer's protocol (Stratagene Cloning Systems, La Jolla, Calif.). The Uni-ZAP™ XR libraries are converted into plasmid libraries according to the protocol provided by Stratagene. Upon conversion, cDNA inserts will be contained in the plasmid vector pBluescript. In addition, the cDNAs may be introduced directly into precut Bluescript II SK(+) vectors (Stratagene) using T4 DNA ligase (New England Biolabs), followed by transfection into DH10B cells according to the manufacturer's protocol (GIBCO BRL Products). Once the cDNA inserts are in plasmid vectors, plasmid DNAs are prepared from randomly picked bacterial colonies containing recombinant pBluescript plasmids, or the insert cDNA sequences are amplified via polymerase chain reaction using primers specific for vector sequences flanking the inserted cDNA sequences. Amplified insert DNAs or plasmid DNAs are sequenced in dye-primer sequencing reactions to generate partial cDNA sequences (expressed sequence tags or “ESTs”; see Adams et al., (1991) <i>Science </i>252:1651-1656). The resulting ESTs are analyzed using a Perkin Elmer Model 377 fluorescent sequencer.</div>
</li> <li> <div class="description-line" id="p-0212" num="0000">
<tables id="TABLE-US-00002" num="00002">
<patent-tables colsep="0" frame="none" rowsep="0">
<table align="left" class="description-table" cols="1" colsep="0" rowsep="0" width="100%">
<thead>
<tr class="description-tr">
<td class="description-td" colspan="1" nameend="1" namest="1" rowsep="1">TABLE 1</td>
</tr>
</thead>
<tbody><tr class="description-tr">
<td align="center" class="description-td" colspan="1" nameend="1" namest="1" rowsep="1"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">cDNA Libraries and clones containing NAR2-like sequences from Corn</td>
</tr>
</tbody></table>
<table align="left" class="description-table" cols="3" colsep="0" rowsep="0" width="100%">
<tbody><tr class="description-tr">
<td class="description-td">Library</td>
<td class="description-td">Tissue</td>
<td class="description-td">Clone</td>
</tr>
<tr class="description-tr">
<td align="center" class="description-td" colspan="3" nameend="3" namest="1" rowsep="1"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">Cnr1c</td>
<td class="description-td">Corn (<i>Zea mays</i>). Plants were Nitrogen</td>
<td class="description-td">cnr1c.pk003.m9.f:fis</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">starved until all seed reserves were</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">depleted of a Nitrogen source. Plants</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">were induced with addition of</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">Nitrogen, then samples were collected</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">at 30 min-1 hr and 2 hr after Nitrogen.</td>
</tr>
<tr class="description-tr">
<td class="description-td">Cbn2</td>
<td class="description-td">Corn (<i>Zea mays </i>L.) developing kernel</td>
<td class="description-td">cbn2.pk0042.g4:fis</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">two days after pollination</td>
</tr>
<tr class="description-tr">
<td align="center" class="description-td" colspan="3" nameend="3" namest="1" rowsep="1"> </td>
</tr>
</tbody></table>
</patent-tables>
</tables>
</div>
</li> <heading id="h-0010">Example 2</heading>
<heading id="h-0011">Identification of cDNA Clones</heading>
<li> <para-num num="[0224]"> </para-num> <div class="description-line" id="p-0213" num="0224">cDNA clones encoding components associated with nitrate transport were identified by conducting BLAST (Basic Local Alignment Search Tool; Altschul et al. (1993) <i>J. Mol. Biol. </i>215:403-410;) and are shown in Table 1.</div>
</li> <li> <para-num num="[0225]"> </para-num> <div class="description-line" id="p-0214" num="0225">cDNA clones encoding transporters or components associated with nitrate transport can be identified by conducting BLAST (Basic Local Alignment Search Tool; Altschul et al. (1993) <i>J. Mol. Biol. </i>215:403-410;) searches for similarity to sequences contained in the BLAST “nr” database (comprising all non-redundant GenBank CDS translations, sequences derived from the 3-dimensional structure Brookhaven Protein Data Bank, the last major release of the SWISS-PROT protein sequence database, EMBL, and DDBJ databases). The cDNA sequences obtained can be analyzed for similarity to all publicly available DNA sequences contained in the “nr” database using the BLASTN algorithm provided by the National Center for Biotechnology Information (NCBI). The DNA sequences can be translated in all reading frames and compared for similarity to all publicly available protein sequences contained in the “nr” database using the BLASTX algorithm (Gish and States (1993) <i>Nature Genetics </i>3:266-272) provided by the NCBI. For convenience, the P-value (probability) of observing a match of a cDNA sequence to a sequence contained in the searched databases merely by chance as calculated by BLAST are reported herein as “pLog” values, which represent the negative of the logarithm of the reported P-value. Accordingly, the greater the pLog value, the greater the likelihood that the cDNA sequence and the BLAST “hit” represent homologous proteins.</div>
</li> <heading id="h-0012">Example 3</heading>
<heading id="h-0013">Identification and Sequencing of Corn High Affinity Nitrate Transporters (HAT4 and HAT5)</heading>
<li> <para-num num="[0226]"> </para-num> <div class="description-line" id="p-0215" num="0226">In order to identify homologs of HATs, a public HAT gene (Genbank accession number AY129953), was used to screen Iowa State University MAGI version 2.31 maize genome assembly. A partial clone, MAGI 17514 that showed 85% identity at the nucleotide level and appeared to be a previously unidentified HAT was identified using Blast in the ISU MAGI assembly. This sequence was used to screen the Genbank GSS dataset and some additional homologs of the MAGI sequence were identified; these added about 0.5 kb to the sequence. The GSS dataset consists of sequences set forth in general identification numbers: 33941728, 34245424, 32105143, 34245411, 34082540 and 33992813. The translation of the assembly covered about one half of the gene, at the 3′ end. It completely lacked the 5′ half of the gene.</div>
</li> <li> <para-num num="[0227]"> </para-num> <div class="description-line" id="p-0216" num="0227">In order to isolate the full length HAT4 sequence, BAC clones from two BAC libraries derived from the Maize B73 inbred line were screened using PCR. The libraries had previously been constructed by partial digestion of genomic DNA and inserted in the BamHI and EcoRI sites of the pCUGI (Tomkins, J. P., et al. 2002. Construction and characterization of a deep-coverage bacterial artificial chromosome library for maize. Crop Science 42:928-933) and pTARBAC (pTARBAC2.1 library, Osoegawa, K., et al, Construction Of New Maize, Bovine, Equine And Zebrafish Bac Libraries. Plant And Animal Genome Conference Proceedings. 2001). To facilitate a PCR-based screening, a set of 36 four-dimensional superpools was requested from Amplicon Express (Amplicon Express, 1610NE Eastgate Blvd Pullman, Wash. 99163). Each superpool was derived after the independent growth, isolation and pooling of 4608 clones, more than 165,000 arrayed BAC clones in total. Superpools were subject to PCR reactions, followed by fragment plus-minus determination in agarose gel electrophoresis. PCR primers were designed to amplify a 495-bp fragment located 289 bp downstream the stop codon of a HAT homolog located at the Tigr assembly ID AZM4<sub>—</sub>32787, which is identical to the sequences assembled from the MAGI and GSS databases described above. PCR reactions were performed with 5 ng Template DNA in a 10-μL reaction that included 5 μL of Hotstar Taq Polymerase Mix (Qiagen) and 5 pmol of the forward and reverse primers (SEQ ID NO:1 and SEQ ID NO:2, respectively). Cycle conditions were an initial denaturation step at 95° C. for 15 minutes, followed by 35 cycles of 95° C. for 30 seconds, 60° C. for 30 seconds and 72° C. for 1 minute. A second round of PCR was performed in matrix plates consisting of lower-complexity combinatorial pools derived from clones represented in positive pools. This narrowed down the positives to particular clones. Two clones, bacc.pk139.d24 and bacc.pk142.b21, were identified and confirmed by PCR analysis. Clone bacc.pk139.d24 was used in subsequent work.</div>
</li> <li> <para-num num="[0228]"> </para-num> <div class="description-line" id="p-0217" num="0228">BAC DNA from clone bacc.pk139.d24 was isolated from overnight 250-ml 2xYT+cloramphenicol cultures using a modified alkaline lysis method. Cells were harvested by centrifugation and resuspended in 20 ml of 10-mM EDTA, then lysed by gently adding 40 ml of 0.2-N NaOH/1-% SDS and neutralized with 30 ml of cold 3-M potassium acetate (pH 4.8). Cell debris were removed by centrifugation at 4° C. 15 minutes at 15000×g, followed by filtration through Miracloth. DNA in supernatant was precipitated with 0.7 volumes of isopropanol and resuspended in 9 ml of 50-mM Tris/50-mM EDTA, mixed with 4.5 ml of 7.5-M potassium acetate, placed at −70° C., thawed and centrifuged for 20 minutes at 3500×g. The supernatant was decanted, precipitated with ethanol and resuspended in 0.7 ml of 50-mM Tris/50-mM EDTA. DNase-free RNase A was added to a final concentration of 150 μg/ml and incubated 1 hour at 37° C., followed by phenol:chloroform extraction and ethanol precipitation. Final DNA was resuspended in a total of 400 μl sterile nuclease-free water. DNA insert size, quantity and quality was assessed by Pulsed Field Gel Electrophoresis using a CHEF-Mapper III (Bio-Rad). For confirmatory BAC end sequencing, the T7 (SEQ ID NO:3) and SP6 (SEQ ID NO: 4) primers were used using sequencing conditions described below.</div>
</li> <li> <para-num num="[0229]"> </para-num> <div class="description-line" id="p-0218" num="0229">The general strategy to obtain double-strand, contiguous sequence information along the HAT4 gene was by walking from the known “start” sequence defined by the PCR identification primers, previously described. BAC bacc.pk139.d24 DNA was used as template. Sequencing was performed in a ABI3730 capillary sequencer according to manufacturer protocols. Sequencing reactions consisted of 2 μL of BigDye V3.1Terminator mix (Applied Biosystems), 2 μL of dilution buffer (600 mM Tris HCl pH 9.0, 15 mM MgCl2), 20 μmol of primer, and approximately 1 μg of template DNA in a final reaction volume of 20 μL. Cycle conditions were an initial denaturation at 95° C. for 5 minutes, followed by 99 cycles of 95° C. for 30 seconds, 58° C. for 30 seconds and 64° C. for 4 minutes. Some hard-to-read regions had to be re-sequenced using special cycle and reaction conditions. Excess dye terminator was removed by ethanol precipitation. Trace evaluation, base calling and assembly was based on Phred/Phrap software (Ewing et al. (1998) Genome Res. 8:186-194; Ewing et al. (1998) Genome Res. 8:175-185). Consed (Gordon et al. (1998) Genome Res. 8:195-202) was used for assembly analysis. After every sequence walking step, primers were designed at the ends, avoiding regions of high homology to other genes and to DNA repeats. Homology search was performed using the BLAST program (Basic Local Alignment Search Tool; Altschul et al. (1993) J. Mol. Biol. 215:403-410) against gss, TIGR 4.0, nonredundant, EST, and protein databases (Altschul et al. 1990). Vector NTI was used for primer design and primers were synthesized commercially by MWG Biotech. Primers (SEQ ID NO: 5 through SEQ ID NO: 33) were designed, tested and used to cover region including the HAT gene. SEQ ID NO: 34 describes the genomic sequence containing the HAT 4 gene. SEQ ID NOs: 35 and 36 describe the coding nucleotide and amino acid sequence of the corn HAT4, respectively.</div>
</li> <li> <para-num num="[0230]"> </para-num> <div class="description-line" id="p-0219" num="0230">SEQ ID NOs: 37 and 38 show the 2014 bp and 1014 bp putative promoter sequences of the HAT4 gene.</div>
</li> <li> <para-num num="[0231]"> </para-num> <div class="description-line" id="p-0220" num="0231">The HAT-5 family was identified via blast homology to the public HATs. One 3′ clone cco1n.pk072.i13 had homology to MAGI<sub>—</sub>56254, which appeared to represent the entire sequence. The TIGR assembly AZM4<sub>—</sub>2103 corresponded well to the MAGI clone. Databases containing nitrogen induced libraries were re-blasted using this clone and clone cfp4n.pk008.p6 was identified. This clone was sequenced and contains the complete HAT5 gene sequence (SEQ ID NO:91 and 92).</div>
</li> <heading id="h-0014">Example 4</heading>
<heading id="h-0015">Identification and Sequencing of an Additional Corn High Affinity Nitrate Transporter (HAT 7)</heading>
<li> <para-num num="[0232]"> </para-num> <div class="description-line" id="p-0221" num="0232">A public HAT gene (HAT1, Genbank accession number AY129953) was used to search with Blast, Genbank maize genomic survey sequences (GSS) and maize genomic assemblies (Iowa State University MAGI and Tigr), to try to identify paralogs of AY129953. Along with the HAT4 gene (Example 3) there were other more distant homologs, including MAGI<sub>—</sub>65216 which corresponded to AZM4<sub>—</sub>79242, which contained slightly more sequence information than MAGI 65216). Neither of these two clones contained a start Methionine. AN additional hit to AZM4<sub>—</sub>79246 exhibited similar percent identity when compared to AY129953. AZM4<sub>—</sub>79246 encoded a start Methionine at nucleotide 2264-2266 and approximately 110 amino acids of coding sequence. Further examination showed that these two assemblies shared clone mates, OGUKX93 and OGUCS47 from the Tigr methylation filtrated library. Therefore it was assumed that AZM4<sub>—</sub>79242 and AZM4<sub>—</sub>79246 encode the same gene but have no sequence overlap.</div>
</li> <li> <para-num num="[0233]"> </para-num> <div class="description-line" id="p-0222" num="0233">In order to retrieve the full length sequence, PCR was performed using two different forward and two different reverse primers (SEQ ID NOs: 39, 40 and 41, 42, repectively) with T3 (SEQ ID NO: 43) and T7 extensions (SEQ ID NO: 44 at the 5′ and 3′ end, respectively. HotStart PCR, with an annealing temperature of 58° C. was performed using DNA from eight maize inbred lines (B73, Co159, GT119, Mo17, T218, Oh43 and W23) as templates. All 32 PCR reaction products were run on a agarose 1×TBE gel, excised and cleaned up and sequenced on a 3100 ABI Capillary Sequencer using methods known to those of ordinary skill in the art. The sequences were aligned and the missing sequence information was retrieved. The complete nucleotide sequence of the HAT7 gene is shown in SEQ ID NO: 45. SEQ ID NOs: 46 and 47 describe the 2263 bp and 1263 bp putative promoter sequences of the HAT7 gene and SEQ ID NOs: 48 and 49 describe the coding nucleotide and amino acid sequence of the corn HAT7, respectively.</div>
</li> <heading id="h-0016">Example 5</heading>
<heading id="h-0017">Characterization of Polypeptides Encoding High Affinity Nitrate Transporter</heading>
<li> <para-num num="[0234]"> </para-num> <div class="description-line" id="p-0223" num="0234">The data in Table 2 represent a calculation of the percent identity of the amino acid sequences set forth in SEQ ID NOs: 36 and 49 and the <i>Oryza sativa </i>sequences (NCBI General Identifier Nos. 34913806 and 50904699).</div>
</li> <li> <div class="description-line" id="p-0224" num="0000">
<tables id="TABLE-US-00003" num="00003">
<patent-tables colsep="0" frame="none" rowsep="0">
<table align="left" class="description-table" cols="1" colsep="0" rowsep="0" width="100%">
<thead>
<tr class="description-tr">
<td class="description-td" colspan="1" nameend="1" namest="1" rowsep="1">TABLE 2</td>
</tr>
</thead>
<tbody><tr class="description-tr">
<td align="center" class="description-td" colspan="1" nameend="1" namest="1" rowsep="1"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">Percent Identity of Amino Acid Sequences Deduced From the Nucleotide</td>
</tr>
<tr class="description-tr">
<td class="description-td">Sequences of cDNA Clones Encoding Polypeptides Homologous</td>
</tr>
<tr class="description-tr">
<td class="description-td">to High Affinity Nitrate Transporter (HAT)</td>
</tr>
</tbody></table>
<table align="left" class="description-table" cols="3" colsep="0" rowsep="0" width="100%">
<tbody><tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">Percent Identity to</td>
<td class="description-td"> </td>
</tr>
</tbody></table>
<table align="left" class="description-table" cols="3" colsep="0" rowsep="0" width="100%">
<tbody><tr class="description-tr">
<td class="description-td">SEQ ID NO.</td>
<td class="description-td">34913806</td>
<td class="description-td">50904699</td>
</tr>
<tr class="description-tr">
<td align="center" class="description-td" colspan="3" nameend="3" namest="1" rowsep="1"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">36</td>
<td class="description-td">38.0</td>
<td class="description-td">75.3</td>
</tr>
<tr class="description-tr">
<td class="description-td">49</td>
<td class="description-td">78.2</td>
<td class="description-td">39.4</td>
</tr>
<tr class="description-tr">
<td align="center" class="description-td" colspan="3" nameend="3" namest="1" rowsep="1"> </td>
</tr>
</tbody></table>
</patent-tables>
</tables>
</div>
</li> <li> <para-num num="[0235]"> </para-num> <div class="description-line" id="p-0225" num="0235">Sequence alignments and percent identity calculations were performed using the Megalign program of the LASERGENE bioinformatics computing suite (DNASTAR Inc., Madison, Wis.). Multiple alignment of the sequences was performed using the Clustal method of alignment (Higgins and Sharp (1989) <i>CABIOS. </i>5:151-153) with the default parameters (GAP PENALTY=10, GAP LENGTH PENALTY=10). Default parameters for pairwise alignments using the Clustal method were KTUPLE 1, GAP PENALTY=3, WINDOW=5 and DIAGONALS SAVED=5. Sequence alignments and BLAST scores and probabilities indicate that the nucleic acid fragments comprising the instant cDNA clones encode corn high affinity nitrogen transporters.</div>
</li> <heading id="h-0018">Example 6</heading>
<heading id="h-0019">Identification and Sequencing of Corn Nitrogen Transport Related Genes, (NAR2-1 &amp; NAR2-2)</heading>
<li> <para-num num="[0236]"> </para-num> <div class="description-line" id="p-0226" num="0236">Examination of blast hits from the maize root library cnr1c, described in Example 1 and Table 2, showed a number of Nitrogen transport related genes. Blast hits were searched with keywords such as nitrate, nitrogen, and transporter. A few of these were homologous to NCBI Accession number: CAC36942, a putative component of high affinity nitrate transporter (NAR2 gene). A TblastN search of maize ESTs, using the sequence of CAC36942 as a query, produced a number of significant hits from different maize libraries. The most 5′ clone was identified by aligning the full-length query and the blast hits. A clone from the cnr1c library (cnr1c.pk003.m9.f) showed a methionine that was in the same region as the start methionine from CAC36942. This clone also showed an in frame stop codon upstream of the methionine. This clone was submitted for standard full insert sequencing (FIS) and contained the 971 bp of the NAR2.1, spanning nucleotides 591 through 1561 of SEQ ID NO: 53. SEQ ID NO: 53 shows the 1561 bp sequence of the NAR2.1 gene, which was assembled from the sequence information obtained from clone cnr1c.pk003.m9.f:fis and from Tigr sequence AZM4<sub>—</sub>81138. SEQ ID NOs: 54 and 55 show the coding nucleotide and amino acid sequence of the NAR2.1 gene, respectively. SEQ ID NO: 56 shows 756 bp of the putative promoter of the NAR2.1. Using CAC36942 as a query also showed a different NAR2 homolog, cbn2.pk0042.g4. This clone also had a start Methionine, but because of the quality of the EST sequence the homology to CAC36942 was short. A complete version (Tigr clone AZM4<sub>—</sub>1475) of this family member was identified by searching the Tigr maize genomic assembly using cbn2.pk0042.g4 as a query. SEQ ID NOs: 57 and 58 show the coding nucleotide and amino acid sequence of the NAR2.2 (Tigr clone AZM4<sub>—</sub>1475), repectively.</div>
</li> <li> <para-num num="[0237]"> </para-num> <div class="description-line" id="p-0227" num="0237">NAR2.1 Promoter Isolation</div>
</li> <li> <para-num num="[0238]"> </para-num> <div class="description-line" id="p-0228" num="0238">The sequence information on the NAR2.1 promoter was extended further upstream by performing Genome Walker™ DNA walking (BD BioSciences). This method employs PCR to facilitate the cloning of unknown genomic DNA sequences adjacent to a known sequence. First, pools of unknown genomic DNA were digested with different restriction enzymes that leave blunt ends. Each pool was ligated to adaptors to create Genome Walker” libraries. Eight different corn HG11 libraries were obtained. These libraries were digested with the following restriction enzymes: StuI, EcoRV, PmlI, PvulI, ScaI, DraI, SmaI, and PmeI.</div>
</li> <li> <para-num num="[0239]"> </para-num> <div class="description-line" id="p-0229" num="0239">Then two rounds of nested PCR amplification per library were performed. For the first round the outer adaptor primer (AP1, provided with kit) and the Nar2.1 specific outer primer (SEQ ID NO: 59) were used.</div>
</li> <li> <para-num num="[0240]"> </para-num> <div class="description-line" id="p-0230" num="0240">PCR was performed using the Advantage®-GC Genomic Polymerase Mix (BD Biosciences) in a 50 μL reaction containing 1 μL I library DNA, 0.5 μL each primer (10 μM), 4 μL dNTPs (2.5 mM), 2.2 μL Mg (OAc)<sub>2</sub>, 10 μL I 15×GC Genomic PCR Reaction Buffer, 10 μL GC-Melt (5M), 20.8 μL ddH<sub>2</sub>O, and 1 μL Advantage-GC Genomic Polymerase. The cycling conditions were as follows: 7 cycles of denaturation at 94° C. for 25 seconds and annealing/extension at 72° C. for 6 minutes followed by 32 cycles of denaturation at 94° C. for 25 seconds and annealing/extension at 67° C. for 6 minutes capped off by annealing/extension at 67° C. for 7 minutes.</div>
</li> <li> <para-num num="[0241]"> </para-num> <div class="description-line" id="p-0231" num="0241">The primary PCR product was then diluted 1:50 and 1 μL served as the template for the second round of PCR which used the same PCR set-up as the first round. The second round primers were the inner adaptor primer (AP2, provided with the kit) and the Nar2.1 specific inner primer (SEQ ID NO: 60). The cycling conditions for the second round were as follows: 5 cycles of denaturation at 94° C. for 25 seconds and annealing/extension at 72° C. for 6 minutes followed by 25 cycles of denaturation at 94 C for 25 seconds and annealing/extension at 67° C. for 6 minutes capped off by annealing/extension at 67° C. for 7 minutes.</div>
</li> <li> <para-num num="[0242]"> </para-num> <div class="description-line" id="p-0232" num="0242">A major PCR product (about 3 kb) was observed in the StuI library. This band was cut-out of the gel and purified using the Qiaquick Gel Extraction Kit (Qiagen) and ligated to a pGEM®-T Easy Vector (Promega). The 20 μL ligation reaction was as follows: 10 μL 2× Rapid Ligation Buffer, 1 μL pGEM®-T Easy Vector (50 ng), 1 μL T4 DNA Ligase (3 Weiss units/μL), and 8 μL insert DNA (13 ng/μL). The reaction was incubated at 4° C. overnight.</div>
</li> <li> <para-num num="[0243]"> </para-num> <div class="description-line" id="p-0233" num="0243">The ligation product was transformed into Max Efficiency DH10B (Invitrogen) competent cells. One μL of ligate was added to 20 μL of cells and put on ice for 30 minutes. The cells were heat shocked at 42° C. for 45 seconds and then placed again on ice for 2 minutes. The cells were added to 1 mL of SOC and placed on a shaker at 250 rpm for 1 hr at 37° C. Then, 100 μL of cells were plated onto LB media with Ampicillin, IPTG, and X-Gal to allow for blue/white selection. Only one white colony was obtained.</div>
</li> <li> <para-num num="[0244]"> </para-num> <div class="description-line" id="p-0234" num="0244">Plasmid DNA was purified using the Plasmid Mini Kit (Qiagen). The plasmid insert representing the NAR2 upstream promoter region was sequenced using standard primers (SP6 and T7) and custom primers (SEQ ID NOs: 61, 62, 63 and 64). SEQ ID NO: 65 shows the sequence of the additional 2917 bp putative NAR 2.1 promoter.</div>
</li> <li> <para-num num="[0245]"> </para-num> <div class="description-line" id="p-0235" num="0245">The sequence of the complete NAR2.1 gene is shown in SEQ ID NO: 66.</div>
</li> <heading id="h-0020">Example 7</heading>
<heading id="h-0021">Expression Pattern of Polypeptides of Instant Application</heading>
<li> <para-num num="[0246]"> </para-num> <div class="description-line" id="p-0236" num="0246">The expression pattern of high affinity nitrate transporters (HAT) and other polypeptides (NAR) required for high affinity nitrate transport was analyzed via Lynx MPSS Brenner et al (2000) <i>Proc Natl Acad Sci USA </i>97:1665-70).</div>
</li> <li> <para-num num="[0247]"> </para-num> <div class="description-line" id="p-0237" num="0247">The expression patterns of NAR2.1 and HAT 1 genes are similar across more than 200 libraries as studied via Lynx MPSS (Brenner et al (2000) <i>Proc Natl Acad Sci USA </i>97:1665-70). They are both expressed only in the cortical cylinder of the root tissue and are similarly induced by nitrate, indicating that the polypeptide products of these two genes form a functional complex for nitrate transport in maize roots.</div>
</li> <li> <para-num num="[0248]"> </para-num> <div class="description-line" id="p-0238" num="0248">Tissue-Specific Expression of NAR2.1 and HAT-1 in Maize:</div>
</li> <li> <para-num num="[0249]"> </para-num> <div class="description-line" id="p-0239" num="0249">Of the 210 libraries from different tissues encompassing the whole of maize plant, NAR2.1 and HAT-1 are expressed only in the root libraries. This indicates the root-specific function for each of these genes.</div>
</li> <li> <para-num num="[0250]"> </para-num> <div class="description-line" id="p-0240" num="0250">Expression analysis of NAR2.1 and HAT-1 in maize tissues. MPSS tag abundances were averaged over different tissue libraries. The number of libraries for each tissue was: anther, 3; ear, 15; kernel, 44; leaf, 39; pollen, 1; root, 36; silk, 9; stalk, 19; and tassel, 14.</div>
</li> <li> <para-num num="[0251]"> </para-num> <div class="description-line" id="p-0241" num="0251">Induction of Nitrate Uptake and Localization within Maize Roots:</div>
</li> <li> <para-num num="[0252]"> </para-num> <div class="description-line" id="p-0242" num="0252">Among the root libraries derived from an inbred line A63, the expression of both NAR2.1 and HAT-1 is similarly induced by nitrate.</div>
</li> <li> <para-num num="[0253]"> </para-num> <div class="description-line" id="p-0243" num="0253">Corn roots from etiolated seedlings obtained 7-days after growing in paper rolls in water, were harvested and subjected to different treatments in parallel. The freshly harvested roots were kept on ice as controls. The roots were incubated in an aerated solution containing different nutrients for different lengths of time and then either quickly frozen in liquid N and stored at −80° C. until used for expression analyses or saved between two layers of wet paper towels in ice for further manipulation. A batch of roots that had been treated for four hours in nitrate was manually dissected into cortical cylinder and stele.</div>
</li> <li> <para-num num="[0254]"> </para-num> <div class="description-line" id="p-0244" num="0254">Response of NAR2.1 and HAT 1 expression to different nutrient treatments. The roots were treated for either half hour or four hours in a medium containing either 1 mM nitrate (0.5 mM KNO<sub>3 </sub>and 0.25 mM Ca(NO<sub>3</sub>)<sub>2</sub>) or 1 mM chloride (0.5 mM KCl and 0.25 mM CaCl<sub>2</sub>). A batch of roots treated for 4 hours with nitrate was separated into cortical cylinder and stele and subjected to MPSS.</div>
</li> <li> <para-num num="[0255]"> </para-num> <div class="description-line" id="p-0245" num="0255">Both the NAR2.1 and HAT 1 genes from maize exhibit a similar response to nitrate (N) in the incubation medium which is incremental with time when compared to the parallel control roots incubated in a chloride solution. Also, both these genes are nearly exclusively located in the cortical sleeve and not in the stele. Their similar response to nitrate and their localization strongly indicate that the protein products of these genes make a functional nitrate transport complex in maize roots.</div>
</li> <li> <para-num num="[0256]"> </para-num> <div class="description-line" id="p-0246" num="0256">Opposite Regulation of Expression of NAR2.1 in Illinois High Protein (IHP) and Illinois Low Protein (ILP) Maize Lines:</div>
</li> <li> <para-num num="[0257]"> </para-num> <div class="description-line" id="p-0247" num="0257">IHP and ILP are two sets of lines that are derived from a maize population after ˜100 years of divergent selection for grain protein in the high and low grain protein directions, respectively (Uribelarrea et al., 2004). Whereas IHP grains contain &gt;20% protein, those of ILP contain &lt;5%. The roots of these two lines were subjected to Lynx MPSS after various treatments.</div>
</li> <li> <para-num num="[0258]"> </para-num> <div class="description-line" id="p-0248" num="0258">Roots were either kept in a nitrate solution all the time, starved for two hours for nitrate, or placed in nitrate solution after two hour starvation. Whereas NAR2.1 in IHP responded to nitrate treatment like A63, ILP exhibited an opposite response Given the level of expression of this gene in ILP in nitrate starved roots, which is similar to that of IHP roots kept in nitrate, these results suggest that mechanisms to respond to nitrate in both the directions do exist in maize. However, the mechanism for positive response appears to have been selected as indicated by similar response between IHP and A63, an inbred line with normal grain protein content of ˜10%.</div>
</li> <li> <para-num num="[0259]"> </para-num> <div class="description-line" id="p-0249" num="0259">Only IHP contained the tag for HAT 1 sequence and showed a similar pattern of expression as for NAR2.1, lending further support to the aforementioned suggestion that NAR2.1 and HAT 1 form a functional complex in maize roots.</div>
</li> <li> <para-num num="[0260]"> </para-num> <div class="description-line" id="p-0250" num="0260">Expression of Other HAT Genes in A63:</div>
</li> <li> <para-num num="[0261]"> </para-num> <div class="description-line" id="p-0251" num="0261">HAT 4G was expressed at &gt;10 ppm only in four libraries, all derived from the root tissue. Thus, this gene appears to be root-specific. HAT 7 is expressed in chilled seedlings and three leaf libraries, suggesting that this gene may encode a protein for nitrate uptake from the xylem apoplast into the leaf cells. It is expected that the HAT sequences of the instant application form a functional nitrate transport complex with a NAR sequence.</div>
</li> <heading id="h-0022">Example 8</heading>
<heading id="h-0023">Confirmation of Function of the High Affinity Nitrate Transporters and Polypeptides Required for High Affinity Nitrate Transport Using the TUSC Mutant Population</heading>
<li> <para-num num="[0262]"> </para-num> <div class="description-line" id="p-0252" num="0262">The full genomic sequence for the high affinity nitrate transporter locus can be used to design primers to screen for Mu-insertion mutants in the TUSC population (U.S. Pat. No. 5,962,764, issued Oct. 5, 1999). The pooled TUSC population can be screened with gene specific primers. Alleles of the corn high affinity nitrate transporters and polypeptides required for high affinity nitrate transport can be recovered from this screen, and characterized. Furthermore, function of the sequences of the instant application can be confirmed by complementation studies.</div>
</li> <heading id="h-0024">Example 9</heading>
<heading id="h-0025">Expression of Recombinant DNA Constructs in Monocot Cells</heading>
<li> <para-num num="[0263]"> </para-num> <div class="description-line" id="p-0253" num="0263">A recombinant DNA construct comprising a cDNA encoding the instant polypeptides in sense orientation with respect to the maize 27 kD zein promoter that is located 5′ to the cDNA fragment, and the 10 kD zein 3′ end that is located 3′ to the cDNA fragment, can be constructed. The cDNA fragment of this gene may be generated by polymerase chain reaction (PCR) of the cDNA clone using appropriate oligonucleotide primers. Cloning sites (NcoI or SmaI) can be incorporated into the oligonucleotides to provide proper orientation of the DNA fragment when inserted into the digested vector pML103 as described below. Amplification is then performed in a standard PCR. The amplified DNA is then digested with restriction enzymes NcoI and SmaI and fractionated on an agarose gel. The appropriate band can be isolated from the gel and combined with a 4.9 kb NcoI-SmaI fragment of the plasmid pML103. Plasmid pML103 has been deposited under the terms of the Budapest Treaty at ATCC (American Type Culture Collection, 10801 University Blvd., Manassas, Va. 20110-2209), and bears accession number ATCC 97366. The DNA segment from pML103 contains a 1.05 kb SalI-NcoI promoter fragment of the maize 27 kD zein gene and a 0.96 kb SmaI-SalI fragment from the 3′ end of the maize 10 kD zein gene in the vector pGem9Zf(+) (Promega). Vector and insert DNA can be ligated at 15° C. overnight, essentially as described in Maniatis. The ligated DNA may then be used to transform <i>E. coli </i>XL1-Blue (Epicurian Coli XL-1 Blue™; Stratagene). Bacterial transformants can be screened by restriction enzyme digestion of plasmid DNA and limited nucleotide sequence analysis using the dideoxy chain termination method (Sequenase™ DNA Sequencing Kit; U.S. Biochemical). The resulting plasmid construct would comprise a recombinant DNA construct encoding, in the 5′ to 3′ direction, the maize 27 kD zein promoter, a cDNA fragment encoding the instant polypeptides, and the 10 kD zein 3′ region.</div>
</li> <li> <para-num num="[0264]"> </para-num> <div class="description-line" id="p-0254" num="0264">The recombinant DNA construct described above can then be introduced into corn cells by the following procedure. Immature corn embryos can be dissected from developing caryopses derived from crosses of the inbred corn lines H99 and LH132. The embryos are isolated 10 to 11 days after pollination when they are 1.0 to 1.5 mm long. The embryos are then placed with the axis-side facing down and in contact with agarose-solidified N6 medium (Chu et al. (1975) <i>Sci. Sin. Peking </i>18:659-668). The embryos are kept in the dark at 27° C. Friable embryogenic callus consisting of undifferentiated masses of cells with somatic proembryoids and embryoids borne on suspensor structures proliferates from the scutellum of these immature embryos. The embryogenic callus isolated from the primary explant can be cultured on N6 medium and sub-cultured on this medium every 2 to 3 weeks.</div>
</li> <li> <para-num num="[0265]"> </para-num> <div class="description-line" id="p-0255" num="0265">The plasmid, p35S/Ac (obtained from Dr. Peter Eckes, Hoechst Ag, Frankfurt, Germany) may be used in transformation experiments in order to provide for a selectable marker. This plasmid contains the Pat gene (see European Patent Publication 0242236) which encodes phosphinothricin acetyl transferase (PAT). The enzyme PAT confers resistance to herbicidal glutamine synthetase inhibitors such as phosphinothricin. The pat gene in p35S/Ac is under the control of the 35S promoter from Cauliflower Mosaic Virus (Odell et al. (1985) <i>Nature </i>313:810-812) and the 3′ region of the nopaline synthase gene from the T-DNA of the Ti plasmid of <i>Agrobacterium tumefaciens. </i> </div>
</li> <li> <para-num num="[0266]"> </para-num> <div class="description-line" id="p-0256" num="0266">The particle bombardment method (Klein et al. (1987) <i>Nature </i>327:70-73) may be used to transfer genes to the callus culture cells. According to this method, gold particles (1 μm in diameter) are coated with DNA using the following technique. Ten μg of plasmid DNAs are added to 50 μL of a suspension of gold particles (60 mg per mL). Calcium chloride (50 μL of a 2.5 M solution) and spermidine free base (20 μL of a 1.0 M solution) are added to the particles. The suspension is vortexed during the addition of these solutions. After 10 minutes, the tubes are briefly centrifuged (5 sec at 15,000 rpm) and the supernatant removed. The particles are resuspended in 200 μL of absolute ethanol, centrifuged again and the supernatant removed. The ethanol rinse is performed again and the particles resuspended in a final volume of 30 μL of ethanol. An aliquot (5 μL) of the DNA-coated gold particles can be placed in the center of a Kapton™ flying disc (Bio-Rad Labs). The particles are then accelerated into the corn tissue with a Biolistic™ PDS-1000/He (Bio-Rad Instruments, Hercules Calif.), using a helium pressure of 1000 psi, a gap distance of 0.5 cm and a flying distance of 1.0 cm.</div>
</li> <li> <para-num num="[0267]"> </para-num> <div class="description-line" id="p-0257" num="0267">For bombardment, the embryogenic tissue is placed on filter paper over agarose-solidified N6 medium. The tissue is arranged as a thin lawn and covered a circular area of about 5 cm in diameter. The petri dish containing the tissue can be placed in the chamber of the PDS-1000/He approximately 8 cm from the stopping screen. The air in the chamber is then evacuated to a vacuum of 28 inches of Hg. The macrocarrier is accelerated with a helium shock wave using a rupture membrane that bursts when the He pressure in the shock tube reaches 1000 psi.</div>
</li> <li> <para-num num="[0268]"> </para-num> <div class="description-line" id="p-0258" num="0268">Seven days after bombardment the tissue can be transferred to N6 medium that contains gluphosinate (2 mg per liter) and lacks casein or proline. The tissue continues to grow slowly on this medium. After an additional 2 weeks the tissue can be transferred to fresh N6 medium containing gluphosinate. After 6 weeks, areas of about 1 cm in diameter of actively growing callus can be identified on some of the plates containing the glufosinate-supplemented medium. These calli may continue to grow when sub-cultured on the selective medium.</div>
</li> <li> <div class="description-line" id="p-0259" num="0000">Plants can be regenerated from the transgenic callus by first transferring clusters of tissue to N6 medium supplemented with 0.2 mg per liter of 2,4-D. After two weeks the tissue can be transferred to regeneration medium (Fromm et al. (1990) <i>Bio/Technology </i>8:833-839).</div>
</li> <heading id="h-0026">Example 10</heading>
<heading id="h-0027">Expression of Recombinant DNA Constructs in Dicot Cells</heading>
<li> <para-num num="[0269]"> </para-num> <div class="description-line" id="p-0260" num="0269">A seed-specific expression cassette composed of the promoter and transcription terminator from the gene encoding the α-subunit of the seed storage protein phaseolin from the bean <i>Phaseolus vulgaris </i>(Doyle et al. (1986) <i>J. Biol. Chem. </i>261:9228-9238) can be used for expression of the instant polypeptides in transformed soybean. The phaseolin cassette includes about 500 nucleotides upstream (5′) from the translation initiation codon and about 1650 nucleotides downstream (3′) from the translation stop codon of phaseolin. Between the 5′ and 3′ regions are the unique restriction endonuclease sites Nco I (which includes the ATG translation initiation codon), Sma I, Kpn I and Xba I. The entire cassette is flanked by Hind III sites.</div>
</li> <li> <para-num num="[0270]"> </para-num> <div class="description-line" id="p-0261" num="0270">The cDNA fragment of this gene may be generated by polymerase chain reaction (PCR) of the cDNA clone using appropriate oligonucleotide primers. Cloning sites can be incorporated into the oligonucleotides to provide proper orientation of the DNA fragment when inserted into the expression vector. Amplification is then performed as described above, and the isolated fragment is inserted into a pUC18 vector carrying the seed expression cassette.</div>
</li> <li> <para-num num="[0271]"> </para-num> <div class="description-line" id="p-0262" num="0271">Soybean embroys may then be transformed with the expression vector comprising sequences encoding the instant polypeptides. To induce somatic embryos, cotyledons, 3-5 mm in length dissected from surface sterilized, immature seeds of the soybean cultivar A2872, can be cultured in the light or dark at 26° C. on an appropriate agar medium for 6-10 weeks. Somatic embryos which produce secondary embryos are then excised and placed into a suitable liquid medium. After repeated selection for clusters of somatic embryos which multiplied as early, globular staged embryos, the suspensions are maintained as described below.</div>
</li> <li> <para-num num="[0272]"> </para-num> <div class="description-line" id="p-0263" num="0272">Soybean embryogenic suspension cultures can maintained in 35 mL liquid media on a rotary shaker, 150 rpm, at 26° C. with florescent lights on a 16:8 hour day/night schedule. Cultures are subcultured every two weeks by inoculating approximately 35 mg of tissue into 35 mL of liquid medium.</div>
</li> <li> <para-num num="[0273]"> </para-num> <div class="description-line" id="p-0264" num="0273">Soybean embryogenic suspension cultures may then be transformed by the method of particle gun bombardment (Klein et al. (1987) <i>Nature </i>(London) 327:70-73, U.S. Pat. No. 4,945,050). A DuPont Biolistic™ PDS1000/HE instrument (helium retrofit) can be used for these transformations.</div>
</li> <li> <para-num num="[0274]"> </para-num> <div class="description-line" id="p-0265" num="0274">A selectable marker gene which can be used to facilitate soybean transformation is a recombinant DNA construct composed of the 35S promoter from Cauliflower Mosaic Virus (Odell et al. (1985) <i>Nature </i>313:810-812), the hygromycin phosphotransferase gene from plasmid pJR225 (from <i>E. coli</i>; Gritz et al. (1983) <i>Gene </i>25:179-188) and the 3′ region of the nopaline synthase gene from the T-DNA of the Ti plasmid of <i>Agrobacterium tumefaciens</i>. The seed expression cassette comprising the phaseolin 5′ region, the fragment encoding the instant polypeptides and the phaseolin 3′ region can be isolated as a restriction fragment. This fragment can then be inserted into a unique restriction site of the vector carrying the marker gene.</div>
</li> <li> <para-num num="[0275]"> </para-num> <div class="description-line" id="p-0266" num="0275">To 50 μL of a 60 mg/mL 1 μm gold particle suspension is added (in order): 5 μL DNA (1 μg/μL), 20 pLspermidine (0.1 M), and 50 μL CaCl<sub>2 </sub>(2.5 M). The particle preparation is then agitated for three minutes, spun in a microfuge for 10 seconds and the supernatant removed. The DNA-coated particles are then washed once in 400 μL 70% ethanol and resuspended in 40 μL of anhydrous ethanol. The DNA/particle suspension can be sonicated three times for one second each. Five μL of the DNA-coated gold particles are then loaded on each macro carrier disk.</div>
</li> <li> <para-num num="[0276]"> </para-num> <div class="description-line" id="p-0267" num="0276">Approximately 300-400 mg of a two-week-old suspension culture is placed in an empty 60×15 mm petri dish and the residual liquid removed from the tissue with a pipette. For each transformation experiment, approximately 5-10 plates of tissue are normally bombarded. Membrane rupture pressure is set at 1100 psi and the chamber is evacuated to a vacuum of 28 inches mercury. The tissue is placed approximately 3.5 inches away from the retaining screen and bombarded three times. Following bombardment, the tissue can be divided in half and placed back into liquid and cultured as described above.</div>
</li> <li> <para-num num="[0277]"> </para-num> <div class="description-line" id="p-0268" num="0277">Five to seven days post bombardment, the liquid media may be exchanged with fresh media, and eleven to twelve days post bombardment with fresh media containing 50 mg/mL hygromycin. This selective media can be refreshed weekly. Seven to eight weeks post bombardment, green, transformed tissue may be observed growing from untransformed, necrotic embryogenic clusters. Isolated green tissue is removed and inoculated into individual flasks to generate new, clonally propagated, transformed embryogenic suspension cultures. Each new line may be treated as an independent transformation event. These suspensions can then be subcultured and maintained as clusters of immature embryos or regenerated into whole plants by maturation and germination of individual somatic embryos.</div>
</li> <heading id="h-0028">Example 11</heading>
<heading id="h-0029">Expression of Recombinant DNA Construct in Microbial Cells</heading>
<li> <para-num num="[0278]"> </para-num> <div class="description-line" id="p-0269" num="0278">The cDNAs encoding the instant polypeptides can be inserted into the T7 <i>E. coli </i>expression vector pBT430. This vector is a derivative of pET-3a (Rosenberg et al. (1987) <i>Gene </i>56:125-135) which employs the bacteriophage T7 RNA polymerase/T7 promoter system. Plasmid pBT430 was constructed by first destroying the EcoR I and Hind III sites in pET-3a at their original positions. An oligonucleotide adaptor containing EcoR I and Hind III sites was inserted at the BamH I site of pET-3a. This created pET-3aM with additional unique cloning sites for insertion of genes into the expression vector. Then, the Nde I site at the position of translation initiation was converted to an Nco I site using oligonucleotide-directed mutagenesis. The DNA sequence of pET-3aM in this region, 5′-CATATGG, was converted to 5′-CCCATGG in pBT430.</div>
</li> <li> <para-num num="[0279]"> </para-num> <div class="description-line" id="p-0270" num="0279">Plasmid DNA containing a cDNA may be appropriately digested to release a nucleic acid fragment encoding the protein. This fragment may then be purified on a 1% NuSieve GTG™ low melting agarose gel (FMC). Buffer and agarose contain 10 μg/ml ethidium bromide for visualization of the DNA fragment. The fragment can then be purified from the agarose gel by digestion with GELase™ (Epicentre Technologies) according to the manufacturer's instructions, ethanol precipitated, dried and resuspended in 20 μL of water. Appropriate oligonucleotide adapters may be ligated to the fragment using T4 DNA ligase (New England Biolabs, Beverly, Mass.). The fragment containing the ligated adapters can be purified from the excess adapters using low melting agarose as described above. The vector pBT430 is digested, dephosphorylated with alkaline phosphatase (NEB) and deproteinized with phenol/chloroform as described above. The prepared vector pBT430 and fragment can then be ligated at 16° C. for 15 hours followed by transformation into DH5 electrocompetent cells (GIBCO BRL). Transformants can be selected on agar plates containing LB media and 100 μg/mL ampicillin. Transformants containing the gene encoding the instant polypeptides are then screened for the correct orientation with respect to the T7 promoter by restriction enzyme analysis.</div>
</li> <li> <para-num num="[0280]"> </para-num> <div class="description-line" id="p-0271" num="0280">For high level expression, a plasmid clone with the cDNA insert in the correct orientation relative to the T7 promoter can be transformed into <i>E. coli </i>strain BL21(DE3) (Studier et al. (1986) <i>J. Mol. Biol. </i>189:113-130). Cultures are grown in LB medium containing ampicillin (100 mg/L) at 25° C. At an optical density at 600 nm of approximately 1, IPTG (isopropylthio-beta-galactoside, the inducer) can be added to a final concentration of 0.4 mM and incubation can be continued for 3 h at 25° C. Cells are then harvested by centrifugation and re-suspended in 50 μL of 50 mM Tris-HCl at pH 8.0 containing 0.1 mM DTT and 0.2 mM phenyl methylsulfonyl fluoride. A small amount of 1 mm glass beads can be added and the mixture sonicated 3 times for about 5 seconds each time with a microprobe sonicator. The mixture is centrifuged and the protein concentration of the supernatant determined. One μg of protein from the soluble fraction of the culture can be separated by SDS-polyacrylamide gel electrophoresis. Gels can be observed for protein bands migrating at the expected molecular weight.</div>
</li> <heading id="h-0030">Example 12</heading>
<heading id="h-0031">Electroporation of <i>Aqrobacterium tumefaciens </i>LBA4404</heading>
<li> <para-num num="[0281]"> </para-num> <div class="description-line" id="p-0272" num="0281">Electroporation competent cells (40 μL), such as <i>Agrobacterium tumefaciens </i>LBA4404 (containing PHP10523), are thawed on ice (20-30 min). PHP10523 contains VIR genes for T-DNA transfer, an <i>Agrobacterium </i>low copy number plasmid origin of replication, a tetracycline resistance gene, and a Cos site for in vivo DNA bimolecular recombination. PHP10523 is further described in Example 17. Meanwhile the electroporation cuvette is chilled on ice. The electroporator settings are adjusted to 2.1 kV. A DNA aliquot (0.5 μL parental DNA at a concentration of 0.2 μg-1.0 μg in low salt buffer or twice distilled H<sub>2</sub>O) is mixed with the thawed <i>Agrobacterium tumefaciens </i>LBA4404 cells while still on ice. The mixture is transferred to the bottom of electroporation cuvette and kept at rest on ice for 1-2 min. The cells are electroporated (Eppendorf electroporator 2510) by pushing the “pulse” button twice (ideally achieving a 4.0 millisecond pulse). Subsequently, 0.5 mL of room temperature 2xYT medium (or SOC medium) are added to the cuvette and transferred to a 15 mL snap-cap tube (e.g., Falcon™ tube). The cells are incubated at 28-30° C., 200-250 rpm for 3 h.</div>
</li> <li> <para-num num="[0282]"> </para-num> <div class="description-line" id="p-0273" num="0282">Aliquots of 250 μL are spread onto plates containing YM medium and 50 μg/mL spectinomycin and incubated three days at 28-30° C. To increase the number of transformants one of two optional steps can be performed:</div>
</li> <li> <para-num num="[0283]"> </para-num> <div class="description-line" id="p-0274" num="0283">Option 1: Overlay plates with 30 μL of 15 mg/mL rifampicin. LBA4404 has a chromosomal resistance gene for rifampicin. This additional selection eliminates some contaminating colonies observed when using poorer preparations of LBA4404 competent cells.</div>
</li> <li> <para-num num="[0284]"> </para-num> <div class="description-line" id="p-0275" num="0284">Option 2: Perform two replicates of the electroporation to compensate for poorer electrocompetent cells.</div>
</li> <heading id="h-0032">Identification of Transformants:</heading>
<li> <para-num num="[0285]"> </para-num> <div class="description-line" id="p-0276" num="0285">Four independent colonies are picked and streaked on plates containing AB minimal medium and 50 μg/mL spectinomycin for isolation of single colonies. The plates are incubated at 28° C. for two to three days. A single colony for each putative co-integrate is picked and inoculated with 4 mL of 10 g/L bactopeptone, 10 g/L yeast extract, 5 g/L sodium chloride and 50 mg/L spectinomycin. The mixture is incubate for 24 h at 28° C. with shaking. Plasmid DNA from 4 mL of culture is isolated using Qiagen Miniprep and an optional Buffer PB wash. The DNA is eluted in 30 μL. Aliquots of 2 μL are used to electroporate 20 μL of DH10b+20 μL of twice distilled H<sub>2</sub>O as per above. Optionally a 15 μL aliquot can be used to transform 75-100 μL of Invitrogen Library Efficiency DH5a. The cells are spread on plates containing LB medium and 50 μg/mL spectinomycin and incubated at 37° C. overnight.</div>
</li> <li> <para-num num="[0286]"> </para-num> <div class="description-line" id="p-0277" num="0286">Three to four independent colonies are picked for each putative co-integrate and inoculated 4 mL of 2xYT medium (10 g/L bactopeptone, 10 g/L yeast extract, 5 g/L sodium chloride) with 50 μg/mL spectinomycin. The cells are incubated at 37° C. overnight with shaking. Next, isolate the plasmid DNA from 4 mL of culture using QIAprep® Miniprep with optional Buffer PB wash (elute in 50 μL). Use 8 μL for digestion with SalI (using parental DNA and PHP10523 as controls). Three more digestions using restriction enzymes BamHI, EcoRI, and HindIII are performed for 4 plasmids that represent 2 putative co-integrates with correct SalI digestion pattern (using parental DNA and PHP10523 as controls). Electronic gels are recommended for comparison.</div>
</li> <heading id="h-0033">Example 13</heading>
<heading id="h-0034">Transformation of Maize Using <i>Agrobacterium </i> </heading>
<li> <para-num num="[0287]"> </para-num> <div class="description-line" id="p-0278" num="0287"> <i>Agrobacterium</i>-mediated transformation of maize is performed essentially as described by Zhao et al. in <i>Meth. Mol. Biol. </i>318:315-323 (2006) (see also Zhao et al., <i>Mol. Breed. </i>8:323-333 (2001) and U.S. Pat. No. 5,981,840 issued Nov. 9, 1999, incorporated herein by reference). The transformation process involves bacterium innoculation, co-cultivation, resting, selection and plant regeneration.</div>
</li> <heading id="h-0035">1. Immature Embryo Preparation:</heading>
<li> <para-num num="[0288]"> </para-num> <div class="description-line" id="p-0279" num="0288">Immature maize embryos are dissected from caryopses and placed in a 2 mL microtube containing 2 mL PHI-A medium.</div>
</li> <heading id="h-0036">2. <i>Agrobacterium </i>Infection and Co-Cultivation of Immature Embryos:</heading>
<heading id="h-0037">2.1 Infection Step</heading>
<li> <para-num num="[0289]"> </para-num> <div class="description-line" id="p-0280" num="0289">PHI-A medium of (1) is removed with 1 mL micropipettor, and 1 mL <i>Agrobacterium </i>suspension (including, but not limited to, the <i>Agrobacterium </i>described in Example 7) is added. The tube is gently inverted to mix. The mixture is incubated for 5 min at room temperature.</div>
</li> <heading id="h-0038">2.2 Co-Culture Step</heading>
<li> <para-num num="[0290]"> </para-num> <div class="description-line" id="p-0281" num="0290">The <i>Agrobacterium </i>suspension is removed from the infection step with a 1 mL micropipettor. Using a sterile spatula the embryos are scraped from the tube and transferred to a plate of PHI-B medium in a 100×15 mm Petri dish. The embryos are oriented with the embryonic axis down on the surface of the medium. Plates with the embryos are cultured at 20° C., in darkness, for three days. L-Cysteine can be used in the co-cultivation phase. With the standard binary vector, the co-cultivation medium supplied with 100-400 mg/L L-cysteine is critical for recovering stable transgenic events.</div>
</li> <heading id="h-0039">3. Selection of Putative Transgenic Events:</heading>
<li> <para-num num="[0291]"> </para-num> <div class="description-line" id="p-0282" num="0291">To each plate of PHI-D medium in a 100×15 mm Petri dish, 10 embryos are transferred, maintaining orientation and the dishes are sealed with parafilm. The plates are incubated in darkness at 28° C. Actively growing putative events, as pale yellow embryonic tissue, are expected to be visible in six to to eight weeks. Embryos that produce no events may be brown and necrotic, and little friable tissue growth is evident. Putative transgenic embryonic tissue is subcultured to fresh PHI-D plates at two-three week intervals, depending on growth rate. The events are recorded.</div>
</li> <heading id="h-0040">4. Regeneration of T0 Plants:</heading>
<li> <para-num num="[0292]"> </para-num> <div class="description-line" id="p-0283" num="0292">Embryonic tissue propagated on PHI-D medium is subcultured to PHI-E medium (somatic embryo maturation medium), in 100×25 mm Petri dishes and incubated at 28° C., in darkness, until somatic embryos mature, for about ten to eighteen days. Individual, matured somatic embryos with well-defined scutellum and coleoptile are transferred to PHI-F embryo germination medium and incubated at 28° C. in the light (about 80 pE from cool white or equivalent fluorescent lamps). In seven to ten days, regenerated plants, about 10 cm tall, are potted in horticultural mix and hardened-off using standard horticultural methods.</div>
</li> <li> <para-num num="[0293]"> </para-num> <div class="description-line" id="p-0284" num="0293">Media for Plant Transformation:
</div> </li> <ul> <li id="ul0015-0001" num="0000"> <ul> <li id="ul0016-0001" num="0294">1. PHI-A: 4 g/L CHU basal salts, 1.0 mL/L 1000× Eriksson's vitamin mix, 0.5 mg/L thiamin HCl, 1.5 mg/L 2,4-D, 0.69 g/L L-proline, 68.5 g/L sucrose, 36 g/L glucose, pH 5.2. Add 100 μM acetosyringone (filter-sterilized).</li> <li id="ul0016-0002" num="0295">2. PHI-B: PHI-A without glucose, increase 2,4-D to 2 mg/L, reduce sucrose to 30 g/L and supplemente with 0.85 mg/L silver nitrate (filter-sterilized), 3.0 g/L Gelrite®, 100 μM acetosyringone (filter-sterilized), pH 5.8.</li> <li id="ul0016-0003" num="0296">3. PHI—C: PHI-B without Gelrite® and acetosyringonee, reduce 2,4-D to 1.5 mg/L and supplemente with 8.0 g/L agar, 0.5 g/L 2-[N-morpholino]ethane-sulfonic acid (MES) buffer, 100 mg/L carbenicillin (filter-sterilized).</li> <li id="ul0016-0004" num="0297">4. PHI-D: PHI—C supplemented with 3 mg/L bialaphos (filter-sterilized).</li> <li id="ul0016-0005" num="0298">5. PHI-E: 4.3 g/L of Murashige and Skoog (MS) salts, (Gibco, BRL 11117-074), 0.5 mg/L nicotinic acid, 0.1 mg/L thiamine HCl, 0.5 mg/L pyridoxine HCl, 2.0 mg/L glycine, 0.1 g/L myo-inositol, 0.5 mg/L zeatin (Sigma, Cat. No. Z-0164), 1 mg/L indole acetic acid (IAA), 26.4 μg/L abscisic acid (ABA), 60 g/L sucrose, 3 mg/L bialaphos (filter-sterilized), 100 mg/L carbenicillin (filter-sterilized), 8 g/L agar, pH 5.6.</li> <li id="ul0016-0006" num="0299">6. PHI-F: PHI-E without zeatin, IAA, ABA; reduce sucrose to 40 g/L; replacing agar with 1.5 g/L Gelrite®; pH 5.6.</li> </ul> </li> </ul>
<li> <para-num num="[0300]"> </para-num> <div class="description-line" id="p-0285" num="0300">Plants can be regenerated from the transgenic callus by first transferring clusters of tissue to N6 medium supplemented with 0.2 mg per liter of 2,4-D. After two weeks the tissue can be transferred to regeneration medium (Fromm et al., <i>Bio/Technology </i>8:833-839 (1990)). Transgenic T0 plants can be regenerated and their phenotype determined. T1 seed can be collected.</div>
</li> <li> <para-num num="[0301]"> </para-num> <div class="description-line" id="p-0286" num="0301">Furthermore, a recombinant DNA construct containing a validated <i>Arabidopsis </i>gene can be introduced into an elite maize inbred line either by direct transformation or introgression from a separately transformed line.</div>
</li> <li> <para-num num="[0302]"> </para-num> <div class="description-line" id="p-0287" num="0302">Transgenic plants, either inbred or hybrid, can undergo more vigorous field-based experiments to study yield enhancement and/or stability under nitrogen limiting and nitrogen non-limiting conditions.</div>
</li> <li> <para-num num="[0303]"> </para-num> <div class="description-line" id="p-0288" num="0303">Subsequent yield analysis can be done to determine whether plants that contain the validated <i>Arabidopsis </i>lead gene have an improvement in yield performance (under nitrogen limiting or non-limiting conditions), when compared to the control (or reference) plants that do not contain the validated <i>Arabidopsis </i>lead gene. Plants containing the validated <i>Arabidopsis </i>lead gene would have less yield loss relative to the control plants, preferably 50% less yield loss, under nitrogen limiting conditions, or would have increased yield relative to the control plants under nitrogen non-limiting conditions.</div>
</li> <heading id="h-0041">Example 14</heading>
<heading id="h-0042">Evaluating Compounds for their Ability to Inhibit the Activity of NitrateTransporters</heading>
<li> <para-num num="[0304]"> </para-num> <div class="description-line" id="p-0289" num="0304">The polypeptides described herein may be produced using any number of methods known to those skilled in the art. Such methods include, but are not limited to, expression in bacteria as described in Example 11, or expression in eukaryotic cell culture, in planta, and using viral expression systems in suitably infected organisms or cell lines. The instant polypeptides may be expressed either as mature forms of the proteins as observed in vivo or as fusion proteins by covalent attachment to a variety of enzymes, proteins or affinity tags. Common fusion protein partners include glutathione S-transferase (“GST”), thioredoxin (“Trx”), maltose binding protein, and C- and/or N-terminal hexahistidine polypeptide (“(His)<sub>6</sub>”). The fusion proteins may be engineered with a protease recognition site at the fusion point so that fusion partners can be separated by protease digestion to yield intact mature enzyme. Examples of such proteases include thrombin, enterokinase and factor Xa. However, any protease can be used which specifically cleaves the peptide connecting the fusion protein and the enzyme.</div>
</li> <li> <para-num num="[0305]"> </para-num> <div class="description-line" id="p-0290" num="0305">Purification of the instant polypeptides, if desired, may utilize any number of separation technologies familiar to those skilled in the art of protein purification. Examples of such methods include, but are not limited to, homogenization, filtration, centrifugation, heat denaturation, ammonium sulfate precipitation, desalting, pH precipitation, ion exchange chromatography, hydrophobic interaction chromatography and affinity chromatography, wherein the affinity ligand represents a substrate, substrate analog or inhibitor. When the instant polypeptides are expressed as fusion proteins, the purification protocol may include the use of an affinity resin, which is specific for the fusion protein tag attached to the expressed enzyme or an affinity resin containing ligands, which are specific for the enzyme. For example, the instant polypeptides may be expressed as a fusion protein coupled to the C-terminus of thioredoxin. In addition, a (His)<sub>6 </sub>peptide may be engineered into the N-terminus of the fused thioredoxin moiety to afford additional opportunities for affinity purification. Other suitable affinity resins could be synthesized by linking the appropriate ligands to any suitable resin such as Sepharose-4B. In an alternate embodiment, a thioredoxin fusion protein may be eluted using dithiothreitol; however, elution may be accomplished using other reagents which interact to displace the thioredoxin from the resin. These reagents include β-mercaptoethanol or other reduced thiol. The eluted fusion protein may be subjected to further purification by traditional means as stated above, if desired. Proteolytic cleavage of the thioredoxin fusion protein and the enzyme may be accomplished after the fusion protein is purified or while the protein is still bound to the ThioBond™ affinity resin or other resin.</div>
</li> <li> <para-num num="[0306]"> </para-num> <div class="description-line" id="p-0291" num="0306">Crude, partially purified or purified enzyme, either alone or as a fusion protein, may be utilized in assays for the evaluation of compounds for their ability to inhibit enzymatic activation of the instant polypeptides disclosed herein. Assays may be conducted under well known experimental conditions that permit optimal enzymatic activity.</div>
</li> <li> <para-num num="[0307]"> </para-num> <div class="description-line" id="p-0292" num="0307">Assays that enable rapid screening for nitrate transport activity have been described in the literature, including, but not limited to an assay that measures <sup>15</sup>N-enriched nitrate uptake into <i>Xenopus </i>oocytes expressing the proteins (Tong et al., The Plant J. (2005) 41:442-450).</div>
</li> <heading id="h-0043">Example 15</heading>
<heading id="h-0044">Expansion of the Linear Nitrate Uptake Range of Higher Plant HATS by Gene Shuffling</heading>
<li> <para-num num="[0308]"> </para-num> <div class="description-line" id="p-0293" num="0308">HATs are known to possess a low Km (in 10 to 100 μM range) and low Vmax (Doddema et al., Kinetics. Physiol. Plant. (1979) 45:332-338, Meharg et al., (1995) J. Membr. Biol. 145:49-66, Touraine et al., Plant Physiol. (1997) 114:137-144, Liu et al., Plant Cell. (1999) 11(5):865-874). Therefore, the uptake rate of HATs remains constant once the nitrate concentration reaches a level of about 2 to 3 fold higher than their Km.</div>
</li> <li> <para-num num="[0309]"> </para-num> <div class="description-line" id="p-0294" num="0309">The most relevant field nitrate concentration is around 2 to 5 mM on a typical modern corn farmland. Within this concentration range, the uptake rate of HATs is well saturated. Extending the linear nitrate uptake of HATs from very low to relevant field concentration would allow maize crop to fully utilize available nitrate for better growth and productivity. Such a transporter would also allow the crop plant to maintain the normal uptake efficiency at lower nitrate input by its enhanced ability to uptake fast at relatively lower nitrate concentration.</div>
</li> <li> <para-num num="[0310]"> </para-num> <div class="description-line" id="p-0295" num="0310">Various gene-shuffling methods (Stemmet WP, PNAS (1994) 91: 10747-10751, Crameri et al., Nature (1998) 391: 288-291, Ness et al., Nature Biotech. (1999) 17:893-896) can be used to generate different types of shuffled HATs libraries. For example, libraries can be generated by single gene and family gene shuffling. Additional diversities can be introduced by spiked oligos carrying amino acid mutations.</div>
</li> <li> <para-num num="[0311]"> </para-num> <div class="description-line" id="p-0296" num="0311">The shuffled HAT libraries can be functionally expressed in one of the heterologous hosts such as yeast, <i>E. coli</i>, and green algae. Preferably, the host lacks the nitrate assimilation pathway except for an endogenous or introduced nitrate reductase. Nitrate uptake rate by functionally expressed shufflants can be assayed by either direct measurement of depletion of nitrate in the assay medium via HPLC or other analytical means or by measurement of nitrite generated by nitrate reductase within the same cell. Nitrite concentration can be easily determined by colorimetrical assay (such as use of Greiss Reagent) or other analytical means (HPLC). Further characterization of the putative hits from screening various shuffled libraries can be achieved by measuring the uptake rates against different concentrations of nitrate. Such assay will provide uptake kinetic parameters of Km and Vmax.</div>
</li> <li> <para-num num="[0312]"> </para-num> <div class="description-line" id="p-0297" num="0312">Hits confirmed with improved properties can then be reshuffled to generate a second round of shuffled libraries and the aforementioned screening scheme can be used for identifying second round hits. This process can be repeated until several shuffled variants are identified that meet the desired kinetic properties.</div>
</li> <heading id="h-0045">Example 16</heading>
<heading id="h-0046">Isolation, Cloning and Sequencing of the Nar Promoter from the Maize B73 Inbred Line</heading>
<heading id="h-0047">Identification of a BAC Clone Carrying the Nar Gene</heading>
<li> <para-num num="[0313]"> </para-num> <div class="description-line" id="p-0298" num="0313">A BAC library derived from maize B71 inbred line was screened by PCR using the forward and reverse primers depicted in SEQ ID NOs: 75 and 76, respectively. Cycle conditions were an initial activation step at 95° C. for 15 minutes, followed by 35 cycles at 94° C. for 1 minute, 60° C. for 1 minute and 72° C. for 1 minute. Final extension was at 72° C. for 10 minutes.
<br/>
A 377 bp product was obtained. BAC clone ZMMBBb0521a1 was identified as carrying the Nar gene.
<br/>
Cloning of the Nar Promoter from Maize B73 Inbred Line
<br/>
The Nar promoter was cloned by PCR using the forward and reverse primer with restriction enzyme sites for BamHI and HindIII depicted in SEQ ID NOs: 77 and 78, repectively.
<br/>
To 1 μl diluted (1:100) BAC DNA from BAC clone ZMMBBb0521a1, 1 μl primer mix at a concentration of 10 μM each, 4 μl DNTPs at a concentration of 2.5 mM, 10 μl 5×HF buffer and 33.5 μl H<sub>2</sub>O and 0.5 μl Phusion High Fidelity DNA Polymerase (Finnzymes) were added. Cycle conditions were an initial activation step at 98° C. for 30 seconds, followed by 35 cycles of 98° C. for 10 seconds, 63° C. for 30 seconds and 72° C. for 1 minute. Final Extension was at 72° C. for 10 minutes.
<br/>
A product of 3621 bp was obtained.
<br/>
The 3621 bp product was gel purified using the Qiaquick™ Gel Extraction Kit (Qiagen) and eluted with 88 μl Elution Buffer.
<br/>
To the purified band 10 μl of buffer E (Promega) and 1 μl of each of the restriction enzyme, BamHI and Hind III (each at 10 U/μl) were added. The assay mixture was incubated at 37° C. for 3 hrs and cleaned up with Qiaquick™ PCR Purification Kit (Qiagen).
<br/>
The pENTR-5′ vector (SEQ ID NO: 85) was digested with BamHI and HindIII and dephosphorylated. The purified PCR band was inserted into the prepared pENTR-5′ vector using the Epicentre Fast Link Kit. The ligation reaction mixture contained 1.5 μL buffer (10×), 1.5 μL ATP (10×), 1 μL ligase, 1 μL pENTR-5′ vector (−10 ng/μL BamHI/HindllUdephosphorylated vector), 1 μL promoter insert (−30 ng) and 9 μL H20. The ligaton reaction was allowed to proceed for 15 minutes at room temperature and was stopped by incubating the mixture at 70° C. for 15 minutes.
<br/>
Transformation into Bacteria and PCR Screen for Insert
<br/>
1 μL of the ligation mix was added to 20 μL of electro-competent cells (DH10B ElectroMax-Invitrogen) and the mixture was electroporated with a Gibco BRL Cell Porator, then 1 mL SOC media were added and the mixture was incubated in a shaker at 37° C. for 1 hr. 150 μL of cells were plated on LB plates with Kanamycin selection and grown overnight at 37° C.
<br/>
12 colonies were picked and 30 μL LB media was added. The colonies were screened using PCR. To 1 μL colony DNA (colony/30 μL LB), 5 μL HotTaq 2× master mix (Qiagen), 1 μL (10 mM primer mix, SEQ ID NO: 77 and 78) and 3 μL dH<sub>2</sub>O were added. Cycle conditions were an initial activation at 95° C. for 15 minutes, followed by 35 cycles of 95° C. for 50 seconds, 55° C. for 50 seconds and of 72° C. for 4 minutes.
<br/>
Final Extension was at 72° C. for 10 minutes.
</div>
</li> <heading id="h-0048">Insert Sequencing</heading>
<li> <para-num num="[0314]"> </para-num> <div class="description-line" id="p-0299" num="0314">DNA carrying the insert was sequenced using the sequence primers depicted in SEQ ID NOs: 79-84. The sequence of the insert is shown in SEQ ID NO: 70. The vector construct carrying the 3621 bp insert was named PHP27621 and is shown in SEQ ID NO: 86 and <figref idrefs="DRAWINGS">FIG. 1</figref>.</div>
</li> <heading id="h-0049">Example 17</heading>
<heading id="h-0050">Testing the NAR Promoter in Transqenic Maize and <i>Arabidopsis </i> </heading>
<li> <para-num num="[0315]"> </para-num> <div class="description-line" id="p-0300" num="0315">Using Invitrogen's™ gateway LR Clonase technology a MultiSite Gateway® LR Recombination Reaction was performed to create the corn NAR promoter::GUS::PINII, UBI::MO-PAT::P1NII and LTP2::DS-RED PINII JT binary vector (PHP27660, SEQ ID NO: 87 and <figref idrefs="DRAWINGS">FIG. 2</figref>). The vector PHP27660 contains the following expression cassettes:
</div> </li> <ul> <li id="ul0017-0001" num="0000"> <ul> <li id="ul0018-0001" num="0316">1. Ubiquitin promoter::MO-PAT::PINII terminator cassette expressing the PAT herbicide resistance gene used for selection during the transformation process.</li> <li id="ul0018-0002" num="0317">2. LTP2 promoter::DS-RED2::PinII terminator cassette expressing the DS-RED color marker gene used for seed sorting.</li> <li id="ul0018-0003" num="0318">3. NAR promoter::GUS::PINII terminator cassette expressing the GUS gene under control of the corn NAR promoter.
<br/>
Vector PHP27660 was electroporated using the protocol outlined in Example 16 into LBA4404 <i>Agrobacterium </i>cells containing PHP10523 by electroporation creating the final co-integrate vector PHP27860 (SEQ ID NO: 88 and <figref idrefs="DRAWINGS">FIG. 3</figref>) was then used for <i>Agrobacterium</i>-based maize transformation as described in Example 17. TO transgenic plants were sampled for GUS expression.
<br/>
Separately, the same vector (PHP27860) was also used for <i>Arabidopsis </i>transformation, following the standard inflorescence-dipping procedures. Transgenic events were selected by herbicide glufosinate spraying on the T1 seedlings. The herbicide-resistant T1 plants were sampled for GUS expression.
</li> </ul> </li> </ul>
<li> <para-num num="[0319]"> </para-num> <div class="description-line" id="p-0301" num="0319">Leaf and root tissue samples were collected from transgenic plants at different time points, including seedling stage and at maturity. Freshly collected tissue samples were dissected into small pieces to facilitate penetration of the GUS staining solution. GUS histochemical staining was done following the standard protocol (Jefferson R A, Kavanagh T A, Bevan M W. 1987 GUS fusions: beta-glucuronidase as a sensitive and versatile gene fusion marker in higher plants. EMBO J. 6(13):3901-3907) incubating at 37° C. overnight.</div>
</li> <li> <div class="description-line" id="p-0302" num="0000">No significant promoter activity was observed in transgenic maize and <i>Arabidopsis </i>tissues.</div>
</li> <heading id="h-0051">Example 18</heading>
<heading id="h-0052">Testing the Effects of Extraneous Junction Sequences on the NAR Promoter in Transqenic Maize and <i>Arabidopsis </i> </heading>
<li> <para-num num="[0320]"> </para-num> <div class="description-line" id="p-0303" num="0320">The Gateway cloning system leaves a short fragment of “foot-print” sequences between components, particularly a 21-bp ATT-B1 fragment between the NAR promoter and the GUS coding region. This has been shown to weaken or even abolish promoter activity in certain cases. This likely is related to the physical distance between basal promoter elements and the start codon. To determine if introducing the ATT-B1 site is negatively affecting the NAR promoter, a construct containing the corn NARpromoter::GUS::PINII cassette is built with a conventional cloning method, i.e., without the use of the Gateway system. Transgenic maize plants are produced via <i>Agrobacterium</i>-based transformation, and various tissue samples are collected for GUS expression study as described in Example 17.</div>
</li> <heading id="h-0053">Example 19</heading>
<heading id="h-0054">Testing the Maize NAR Promoter in a Deletion Series</heading>
<li> <para-num num="[0321]"> </para-num> <div class="description-line" id="p-0304" num="0321">The NAR gene has a nitrate-inducible and root-specific expression pattern.</div>
</li> <li> <para-num num="[0322]"> </para-num> <div class="description-line" id="p-0305" num="0322">To determine the fragments that determine NAR promoter activity and specificity, a series of constructs containing truncated NAR promoter fragments linked to the sequences for GUS and the PINII end are constructed and tested as described for the full length promoter in Examples 17 and 18.</div>
</li> <li> <para-num num="[0323]"> </para-num> <div class="description-line" id="p-0306" num="0323">Using BLASTN (Basic Local Alignment Search Tool; Altschul et al. (1993) <i>J. Mol. Biol. </i>215:403-410;), sequences within the NAR promoter can be identified that might be important for enhancing or suppressing promoter activity. The sequence around 1.5 to 1.9 kb of the NAR promoter shows homology to another gene and a transposon element. Deletion of this fragment as shown in SEQ ID NO: 89 is therefore expected to add information on NAR promoter activity.</div>
</li> <li> <para-num num="[0324]"> </para-num> <div class="description-line" id="p-0307" num="0324">In addition truncation that reduce the length of the promoter as shown in SEQ ID NOs: 71, 72, 73, 74 and 90 can also be tested in the same way as described for the full length promoter in Examples 17 and 18. Additional promoter subfragments can be prepared by using primers derived from the 3.6 Kb NAR promoter sequence in PCR.</div>
</li> <heading id="h-0055">Example 20</heading>
<heading id="h-0056">Evaluation of Nitrate Uptake in Maize Using HAT and NAR Sequences and Combinations Thereof</heading>
<li> <para-num num="[0325]"> </para-num> <div class="description-line" id="p-0308" num="0325">The following maize expression constructs were prepared for evaluation of nitrate uptake in maize: PHP27280 (SEQ ID NO: 93 and <figref idrefs="DRAWINGS">FIG. 4</figref>), PHP27281 (SEQ ID NO:94 and <figref idrefs="DRAWINGS">FIG. 5</figref>), PHP27282 (SEQ ID NO: 95 and <figref idrefs="DRAWINGS">FIG. 6</figref>) and PHP27283 (SEQ ID NO:96 and <figref idrefs="DRAWINGS">FIG. 7</figref>).</div>
</li> <li> <div class="description-line" id="p-0309" num="0000">Additional constructs comprising HAT sequences and combinations of HAT and Nar sequences will be prepared and tested for their ability to alter Nitrate transport. T0, T1 and subsequenct generations will be evaualted for alterd biomass and total ear weight under 1 mM nitrate conditions.</div>
</li> </ul>
</div>
</section><section itemprop="claims" itemscope="">
<h2>Claims (<span itemprop="count">20</span>)</h2>
<div html="" itemprop="content"><div class="claims" lang="EN" load-source="patent-office" mxw-id="PCLM51979205">
<div class="claim"> <div class="claim" id="CLM-00001" num="00001">
<div class="claim-text"> <b>1</b>. An isolated polynucleotide comprising:
<div class="claim-text">(a) a nucleotide sequence encoding a high affinity nitrate transporter polypeptide, wherein the polypeptide has an amino acid sequence of at least 90% sequence identity, based on the Clustal V method of alignment, when compared to SEQ ID NO: 92; or</div> <div class="claim-text">(b) a complement of the nucleotide sequence, wherein the complement and the nucleotide sequence consist of the same number of nucleotides and are 100% complementary.</div> </div>
</div>
</div> <div class="claim"> <div class="claim" id="CLM-002-3" num="002-3">
<div class="claim-text"> <b>2</b>-<b>3</b>. (canceled)</div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="CLM-00004" num="00004">
<div class="claim-text"> <b>4</b>. The polynucleotide of <claim-ref idref="CLM-00001">claim 1</claim-ref>, wherein the amino acid sequence of the polypeptide has at least 95% sequence identity, based on the Clustal V method of alignment, when compared to SEQ ID NO: 92.</div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="CLM-00005" num="00005">
<div class="claim-text"> <b>5</b>. The polynucleotide of <claim-ref idref="CLM-00001">claim 1</claim-ref>, wherein the amino acid sequence of the polypeptide has at least 99% sequence identity, based on the Clustal V method of alignment, when compared to SEQ ID NO: 92.</div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="CLM-00006" num="00006">
<div class="claim-text"> <b>6</b>. The polynucleotide of <claim-ref idref="CLM-00001">claim 1</claim-ref>, wherein the amino acid sequence of the polypeptide comprises one of SEQ ID NO: 92.</div>
</div>
</div> <div class="claim"> <div class="claim" id="CLM-00007" num="00007">
<div class="claim-text"> <b>7</b>. (canceled)</div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="CLM-00008" num="00008">
<div class="claim-text"> <b>8</b>. The isolated polynucleotide of <claim-ref idref="CLM-00001">claim 1</claim-ref>, wherein the polypeptide sequence comprises at least two motifs selected from group consisting of SEQ ID NOs: 50, 51 and 52.</div>
</div>
</div> <div class="claim"> <div class="claim" id="CLM-00009" num="00009">
<div class="claim-text"> <b>9</b>. (canceled)</div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="CLM-00010" num="00010">
<div class="claim-text"> <b>10</b>. A recombinant DNA construct comprising ai the isolated polynucleotide of <claim-ref idref="CLM-00001">claim 1</claim-ref>, the polynucleotide operably linked to at least one regulatory sequence.</div>
</div>
</div> <div class="claim"> <div class="claim" id="CLM-00011" num="00011">
<div class="claim-text"> <b>11</b>. (canceled)</div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="CLM-00012" num="00012">
<div class="claim-text"> <b>12</b>. A plant comprising in its genome the recombinant DNA construct of <claim-ref idref="CLM-00010">claim 10</claim-ref>.</div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="CLM-00013" num="00013">
<div class="claim-text"> <b>13</b>. A transgenic seed obtained from the plant of <claim-ref idref="CLM-00012">claim 12</claim-ref>.</div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="CLM-00014" num="00014">
<div class="claim-text"> <b>14</b>. The plant of <claim-ref idref="CLM-00012">claim 12</claim-ref>, wherein said plant is selected from the group consisting of rice, corn, sorghum, millet, rye, soybean, canola, wheat, barley, oat, beans, and nuts.</div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="CLM-00015" num="00015">
<div class="claim-text"> <b>15</b>. A plant cell comprising in its genome the recombinant DNA construct of <claim-ref idref="CLM-00010">claim 10</claim-ref>.</div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="CLM-00016" num="00016">
<div class="claim-text"> <b>16</b>. Plant tissue comprising the plant cell of <claim-ref idref="CLM-00015">claim 15</claim-ref>.</div>
</div>
</div> <div class="claim"> <div class="claim" id="CLM-17-34" num="17-34">
<div class="claim-text"> <b>17</b>-<b>34</b>. (canceled)</div>
</div>
</div> <div class="claim"> <div class="claim" id="CLM-00035" num="00035">
<div class="claim-text"> <b>35</b>. A method of increasing nitrogen transport in a plant, the method comprising expressing a recombinant nucleotide sequence encoding a polypeptide comprising a sequence of at least 90% sequence identity, based on the Clustal V method of alignment, when compared to SEQ ID NOs: 92; and growing the plant.</div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="CLM-00036" num="00036">
<div class="claim-text"> <b>36</b>. The method of <claim-ref idref="CLM-00035">claim 35</claim-ref>, wherein the polypeptide sequence comprises at least two motifs selected from group consisting of SEQ ID NOs: 50, 51 and 52.</div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="CLM-00037" num="00037">
<div class="claim-text"> <b>37</b>. The method of <claim-ref idref="CLM-00035">claim 35</claim-ref>, further comprising expressing at least one additional recombinant DNA construct comprising an isolated polynucleotide encoding a NAR polypeptide, operably linked to at least one regulatory sequence.</div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="CLM-00038" num="00038">
<div class="claim-text"> <b>38</b>. The method of <claim-ref idref="CLM-00035">claim 35</claim-ref>, wherein the plant is selected from the group consisting of rice, corn, sorghum, millet, rye, soybean, canola, wheat, barley, oat, beans, and nuts.</div>
</div>
</div> </div>
</div>
</section>
                </article>
            </search-app>
        </body>
    </html>
    