
    <html>
        <body>
            <search-app>
                <article class="result" itemscope="" itemtype="http://schema.org/ScholarlyArticle">
    <h1 itemprop="pageTitle">US20200115717A1 - Genetically modified bacteria and methods for genetic modification of bacteria 
      - Google Patents</h1><section itemprop="abstract" itemscope="">
<h2>Abstract</h2>
<div html="" itemprop="content"><abstract lang="EN" load-source="docdb" mxw-id="PA388225847" source="national office">
<div class="abstract">Microbes can be genetically modified to express biomolecules that are beneficial to mammals and/or to reduce, or eliminate, expression of harmful virulence factors. The growth and viability of such genetically modified microbes can optionally be controlled by inducible promoters that regulate the expression of proteins that are essential to their growth and survival. Compositions comprising such genetically modified microbes as well as methods of making and using the same are disclosed herein.</div>
</abstract>
</div>
</section><section itemprop="description" itemscope="">
<h2>Description</h2>
<div html="" itemprop="content"><ul class="description" lang="EN" load-source="patent-office" mxw-id="PDES255159164">
<heading id="h-0001">RELATED APPLICATIONS</heading>
<li> <para-num num="[0001]"> </para-num> <div class="description-line" id="p-0002" num="0001">This application is a continuation of, and claims the benefit of U.S. patent application Ser. No. 14/742,492, filed Jun. 17, 2015, now pending, which claims priority to U.S. Provisional Patent Application No. 62/013,159, filed Jun. 17, 2014. The entire content of the foregoing applications are incorporated herein by reference, including all text, tables, sequence listings and drawings.</div>
</li> <heading id="h-0002">SEQUENCE LISTING</heading>
<li> <para-num num="[0002]"> </para-num> <div class="description-line" id="p-0003" num="0002">This application is being filed with a Sequence Listing. The Sequence Listing is submitted electronically in ASCII format via EFS-Web in the form of a text file. Said ASCII copy, created on Nov. 26, 2019, is named CrownSeqListing0508135.txt and is 806 kb in size, the contents of which are incorporated herein by reference in their entirety.</div>
</li> <heading id="h-0003">INTRODUCTION</heading>
<li> <para-num num="[0003]"> </para-num> <div class="description-line" id="p-0004" num="0003">The present disclosure relates to a compositions comprising genetically modified microbes, methods of makings genetically modified microbes, and uses thereof. In some aspects subject matter described herein relates to the field of dermatology and compositions for topical dermatological purposes.</div>
</li> <heading id="h-0004">SUMMARY</heading>
<li> <para-num num="[0004]"> </para-num> <div class="description-line" id="p-0005" num="0004">The invention provides genetically modified microbes. In particular embodiments, a microbe is a bacteria of the genus <i>Propionibacterium</i>. In various aspects, a genetically modified bacteria of the genus <i>Propionibacterium </i>comprises a nucleic acid encoding a mammalian growth factor, or a mammalian cytokine. In some aspects the growth factor is a hormone. In certain aspects the growth factor is a human or bovine hormone. In certain embodiments the growth factor is a somatotrophin. In some embodiments the growth factor comprises SEQ ID NO:45, SEQ ID NO:46, SEQ ID NO:47, SEQ ID NO:48 or SEQ ID NO:49. In certain embodiments the growth factor is secreted by the bacteria. In some embodiments the growth factor is a human growth factor. In certain embodiments the growth factor is transforming growth factor beta (TGF-β), vascular endothelial growth factor (VEGF), hepatocyte growth factor (HGF), fibroblast growth factor (FGF), insulin-like growth factor-1 (IGF1), platelet derived growth factor (PDGF), granulocyte monocyte colony stimulating factor (GMCSF), epidermal growth factor (EGF) and/or human growth hormone (HGH). The growth factor can be a transforming growth factor comprising any one of SEQ ID NO:63 to SEQ ID NO:69. In some embodiments the growth factor is a hepatocyte growth factor comprising SEQ ID NO:70. In some embodiments the growth factor is a vascular endothelial growth factor (VEGF) comprising any one of SEQ ID NO:71 to SEQ ID NO:91. In some embodiments the growth factor is a platelet derived growth factor (PDGF) comprising any one of SEQ ID NO:93 to SEQ ID NO:102. In certain embodiments the growth factor is an epidermal growth factor (EGF) comprising any one of SEQ ID NO:103 to SEQ ID NO:106. In some embodiments the fibroblast growth factor (FGF) comprises any one of SEQ ID NO:107 to SEQ ID NO:144. In some embodiments the one or more bacteria secrete the mammalian growth factor or mammalian cytokine. In some embodiments the mammalian cytokine is an immunosuppressive cytokine. In some embodiments the mammalian cytokine is a human cytokine. In certain embodiments the cytokine is selected from the group consisting of interleukin-10 (IL-10), interleukin-6 (IL-6), interleukin-7 (IL-7) and interleukin-8 (IL-8). In some embodiments the cytokine is selected from the group consisting of SEQ ID NO:24, SEQ ID NO:25, SEQ ID NO:26, SEQ ID NO:27 and SEQ ID NO:28. In some embodiments the mammalian cytokine comprises IL-10. In some embodiments the IL-10 comprises SEQ ID NO:25.</div>
</li> <li> <para-num num="[0005]"> </para-num> <div class="description-line" id="p-0006" num="0005">In some embodiments the genus of <i>Propionibacterium </i>comprises a species selected from the group consisting of <i>Propionibacterium acidifaciens, Propionibacterium acidipropionici, Propionibacterium acnes, Propionibacterium australiense, Propionibacterium avidum, Propionibacterium cyclohexanicum, Propionibacterium freudenreichii, Propionibacterium freudenreichii, Propionibacterium granulosum, Propionibacterium jensenii, Propionibacterium microaerophilum, Propionibacterium propionicum </i>and <i>Propionibacterium thoeniiand</i>. In some embodiments the strain of <i>P. acnes </i>comprises a CRISPR (clustered regularly interspaced short palindromic repeat) array. In some embodiments the strain of <i>P. acnes </i>is <i>P. acnes</i>, type II, ribotype 6 (R6 Type II <i>P. acnes</i>).</div>
</li> <li> <para-num num="[0006]"> </para-num> <div class="description-line" id="p-0007" num="0006">In some embodiments the nucleic acid comprises an inducible promoter configured to regulate the expression of the mammalian growth factor or the mammalian cytokine. In some embodiments the nucleic acid comprises an endogenous promoter configured to direct the expression of the mammalian growth factor or the mammalian cytokine. In some embodiments the expression of an endogenous pathogenic protein is substantially reduced or eliminated in the one or more genetically modified bacteria. In some embodiments the endogenous pathogenic protein comprises a glyceraldehyde 3-phosphate dehydrogenase (GADPH) protein or a CAMP2 protein. In some embodiments the one or more genetically modified bacteria comprise an inducible promoter regulating the expression of an essential protein. In certain embodiments an essential protein is a suitable housekeeping gene. In certain embodiments an essential protein is a suitable chromosome replication initiator protein. In some embodiments an essential protein is selected from the group consisting of a chromosome replication initiator protein DnaA, FtsA, FtsI, FtsL, FtsK, FtsN, FtsQ, FtsW, FtsZ (e.g., filamentous temperature-sensitive growth, in some embodiments genes with FTS are of this operon that codes for the constriction ring that allows for septation), cell division protein (e.g., ZipA gene), shikimate 5-dehydrogenase (e.g., AROE, aroE gene), an ATP synthase (e.g., ATP synthase, F1 complex, β subunit, e.g., an atpD gene), guanylate kinase (e.g., gmk), GMP synthase (e.g., guaA), GTP binding protein (e.g., lepA), recombinase A protein (e.g., recA), superoxide dismutase (e.g., sodA gene). In some embodiments an inducible promoter is inducible by a sugar, such as lactose or arabinose. In some embodiments the inducible promoter is inducible by an amino acid, such as a synthetic amino acid. In some embodiments the composition is configured for topical or mucosal administration to a mammal.</div>
</li> <li> <para-num num="[0007]"> </para-num> <div class="description-line" id="p-0008" num="0007">In certain embodiments a genetically modified microbe, such as a bacteria of the species <i>Propionibacterium acnes</i>, comprises multiple genetic modifications. In a particular aspect, a modified microbe 1) expresses a mammalian growth factor, wherein the mammalian growth factor is secreted; 2) includes an inducible promoter directing expression of an essential protein, for example, a chromosome replication initiator protein DnaA, FtsA, FtsI, FtsL, FtsK, FtsN, FtsQ, FtsW, FtsZ, ZipA, aroE, atpD, gmk, guaA, lepA, recA, or sodA; and 3) is modified such that expression of an endogenous CAMP2 and/or endogenous GADPH protein is substantially reduced or eliminated. In another particular aspect, a genetically modified microbe, such as a bacteria of the species <i>Propionibacterium acnes</i>, comprises 1) a human cytokine, such as IL10, wherein the cytokine (e.g., IL10) is secreted; 2) includes an inducible promoter directing expression of an essential protein (e.g., an essential protein and/or a gene encoding an essential protein), for example, a chromosome replication initiator protein DnaA, FtsA, FtsI, FtsL, FtsK, FtsN, FtsQ, FtsW, FtsZ, ZipA, aroE, atpD, gmk, guaA, lepA, recA, or sodA; and 3) is modified such that expression of an endogenous CAMP2 and/or endogenous GADPH protein is substantially reduced or eliminated.</div>
</li> <li> <para-num num="[0008]"> </para-num> <div class="description-line" id="p-0009" num="0008">In certain embodiments a composition comprises one or more genetically modified microbes, such as a bacteria of the species <i>Propionibacterium acnes</i>. In particular aspects, a composition includes 2, 3, 4, 5, 6, 7, 8, 9, 10 or more, each of which are genetically modified, such as a bacteria of the species <i>Propionibacterium acnes</i>, in different ways. For example, a first genetically modified microbe of the composition can express a cytokine, such as IL-10, IL-6, IL-7 or IL-8; and a second genetically modified microbe of the composition can express a growth factor such as TGF-β, VEGF, HGF, FGF, IGF1, PDGF, GMCSF, EGF or HGH. Such combinations may include one or more genetically modified microbes in which expression of an endogenous pathogenic protein is substantially reduced or eliminated in the one or more genetically modified bacteria, for example, a glyceraldehyde 3-phosphate dehydrogenase (GADPH) protein or a CAMP2 protein. Such combinations may also include one or more genetically modified microbes that include an inducible promoter regulating the expression of an essential protein, such as protein is selected from a chromosome replication initiator protein DnaA, FtsA, FtsI, FtsL, FtsK, FtsN, FtsQ, FtsW, FtsZ, ZipA, aroE, atpD, gmk, guaA, lepA, recA, and sodA. Such combinations also include genetically modified microbe, such as a bacteria of the species <i>Propionibacterium acnes</i>, with multiple genetic modifications.</div>
</li> <li> <para-num num="[0009]"> </para-num> <div class="description-line" id="p-0010" num="0009">In some embodiments, a genetically modified microbe, such as a bacteria of the species <i>Propionibacterium acnes</i>, comprises an inducible promoter regulating expression of an essential protein. In certain aspects, protein expression is induced or stimulated by the presence of the sugar or sugar analog, i.e., the promoter is induced by the sugar or sugar analog.</div>
</li> <description-of-drawings>
<heading id="h-0005">BRIEF DESCRIPTION OF DRAWINGS</heading>
<li> <para-num num="[0010]"> </para-num> <div class="description-line" id="p-0011" num="0010">The drawings illustrate embodiments of the technology and are not limiting. For clarity and ease of illustration, the drawings are not made to scale and, in some instances, various aspects may be shown exaggerated or enlarged to facilitate an understanding of particular embodiments.</div>
</li> <li> <para-num num="[0011]"> </para-num> <div class="description-line" id="p-0012" num="0011"> <figref idrefs="DRAWINGS">FIG. 1</figref> shows a plasmid map of dnaA-Bs-Cm: A self-ligated DNA for construction of the growth arrest strain of <i>Bacillus subtilis </i> </div>
</li> <li> <para-num num="[0012]"> </para-num> <div class="description-line" id="p-0013" num="0012"> <figref idrefs="DRAWINGS">FIG. 2</figref> shows a plasmid map of 18-araRE-ftsOp-erm: A plasmid for construction of the growth arrest strains of <i>P. acnes </i>I, Arabinose inducible fts operon.</div>
</li> <li> <para-num num="[0013]"> </para-num> <div class="description-line" id="p-0014" num="0013"> <figref idrefs="DRAWINGS">FIG. 3</figref> shows a plasmid map of 18-bgalpro-dnaA-erm-cm: A plasmid for construction of the growth arrest strains of <i>P. acnes </i>II, Lactose inducible dnaA.</div>
</li> <li> <para-num num="[0014]"> </para-num> <div class="description-line" id="p-0015" num="0014"> <figref idrefs="DRAWINGS">FIG. 4A</figref> shows a plasmid map of protein expression vector pET15-IL10: An expression vector for IL-10.</div>
</li> <li> <para-num num="[0015]"> </para-num> <div class="description-line" id="p-0016" num="0015"> <figref idrefs="DRAWINGS">FIG. 4B</figref> shows a plasmid map of protein expression vector pET15-EGF: An expression vector for EGF.</div>
</li> <li> <para-num num="[0016]"> </para-num> <div class="description-line" id="p-0017" num="0016"> <figref idrefs="DRAWINGS">FIG. 4C</figref> shows a plasmid map of protein expression vector pET15-GH: An expression vector for GH.</div>
</li> <li> <para-num num="[0017]"> </para-num> <div class="description-line" id="p-0018" num="0017"> <figref idrefs="DRAWINGS">FIG. 5</figref> shows a plasmid map of 18-CAMPII-erm-cm: A plasmid for construction of CAMPII mutant <i>P. acnes </i>strain.</div>
</li> <li> <para-num num="[0018]"> </para-num> <div class="description-line" id="p-0019" num="0018"> <figref idrefs="DRAWINGS">FIG. 6</figref> shows a plasmid map of beta-gal pro-IL10: A plasmid for construction of lactose inducible IL10 secretion <i>P. acnes </i>strain.</div>
</li> <li> <para-num num="[0019]"> </para-num> <div class="description-line" id="p-0020" num="0019"> <figref idrefs="DRAWINGS">FIG. 7</figref> shows a plasmid map of 19-CAMPII-sig-IL10: A plasmid for construction of CAMPII mutant and IL-10 expression strain of <i>P. acnes. </i> </div>
</li> <li> <para-num num="[0020]"> </para-num> <div class="description-line" id="p-0021" num="0020"> <figref idrefs="DRAWINGS">FIG. 8</figref> shows a plasmid map of 19-sig-IL10-gapdh: A plasmid for construction of GAPDH mutant and IL-10 expression strain of <i>P. acnes. </i> </div>
</li> <li> <para-num num="[0021]"> </para-num> <div class="description-line" id="p-0022" num="0021"> <figref idrefs="DRAWINGS">FIG. 9</figref> shows a scheme for construction of a markerless mutant construction.</div>
</li> <li> <para-num num="[0022]"> </para-num> <div class="description-line" id="p-0023" num="0022"> <figref idrefs="DRAWINGS">FIG. 10</figref> shows an alignment of the RecA gene from a type II strain of <i>P. acnes </i>(RecA ATCC 11828, SEQ ID NO: 274) and the RecA gene from a type I strains of <i>P. acnes </i>(RecA NCTC737, SEQ ID NO: 275). Shaded areas indicated nucleotide base identity at the same base position. Numbering shown is relative to the position of the sequence within the full length gene.</div>
</li> <li> <para-num num="[0023]"> </para-num> <div class="description-line" id="p-0024" num="0023"> <figref idrefs="DRAWINGS">FIG. 11</figref> shows an alignment of RecA genes sequenced from putative RT6 clones (RecA NCTC737, SEQ ID NO: 277; NC_017_41_RecA, SEQ ID NO: 278; NC_001_4_RecA, SEQ ID NO: 279; NC_003_18_RecA, SEQ ID NO: 280; NC_005_31_RecA, SEQ ID NO: 281; NC_007_33_RecA, SEQ ID NO: 282; NC_009_34_RecA, SEQ ID NO: 283; NC_011_35_RecA, SEQ ID NO: 284; NC_015_39_RecA, SEQ ID NO: 285; NC_019_43_RecA, SEQ ID NO: 286; NC_021_44_RecA, SEQ ID NO: 287; NC_023_45_RecA, SEQ ID NO: 288; NC_025_46_RecA, SEQ ID NO: 289) with the region 720-1191 of the RecA gene from ATTCC11828 strain (RecA 11828, SEQ ID NO: 276). Shaded areas indicated nucleotide base identity at the same base position.</div>
</li> <li> <para-num num="[0024]"> </para-num> <div class="description-line" id="p-0025" num="0024"> <figref idrefs="DRAWINGS">FIG. 12</figref> shows an alignment of an SNP sequence of 16S RNA of strain ATCC11828 of <i>P. acnes </i>(SEQ ID NO: 290) with SNP sequences isolated from the <i>P. acnes </i>strains isolated in Example 1 (PA_001_4_16S, SEQ ID NO: 291; PA_002_18_16S, SEQ ID NO: 292; PA_003_20_16S, SEQ ID NO: 293; PA_005_31_16S, SEQ ID NO: 294; PA_007_33_16S, SEQ ID NO: 295; PA_008_34_16S, SEQ ID NO: 296; PA_009_35_16S, SEQ ID NO: 297; PA_011_39_16S, SEQ ID NO: 298; PA_013_43_16S, SEQ ID NO: 299; PA_014_44_16S, SEQ ID NO: 300; PA_015_45_16S, SEQ ID NO: 301; PA_017_48_16S, SEQ ID NO: 302). Shaded areas indicated nucleotide base identity at the same base position.</div>
</li> <li> <para-num num="[0025]"> </para-num> <div class="description-line" id="p-0026" num="0025"> <figref idrefs="DRAWINGS">FIG. 13</figref> shows a protein expression of human IL-10 (lane 1, IL-10), human growth hormone (lane 2, GH) and human epidermal growth factor (lane 3, EGF) in the growth media of individual genetically modified <i>P. acnes </i>strains. The indicated proteins were expressed under the direction of inducible LacZ promoters.</div>
</li> <li> <para-num num="[0026]"> </para-num> <div class="description-line" id="p-0027" num="0026"> <figref idrefs="DRAWINGS">FIG. 14</figref> shows a scheme for construction of growth arrest <i>P. acnes </i>mutant strain. The DNA region of arabinose inducible promoter and regulator was amplified from <i>B. subtilis </i>and the fragment of fts operon was ligated to the downstream of arabinose inducible promoter. This plasmid was introduced into <i>P. acnes </i>for single crossover recombination to construct arabinose regulated growth arrest strain.</div>
</li> <li> <para-num num="[0027]"> </para-num> <div class="description-line" id="p-0028" num="0027"> <figref idrefs="DRAWINGS">FIG. 15</figref> shows growth comparison of <i>P. acnes </i>wild type and growth arrest mutant strain. <i>P. acnes </i>wild type strain and growth arrest mutant strain were streaked onto the modified reinforced <i>clostridium </i>media with glucose (1%) and arabinose (1.5%) and incubated at 37° C. under anaerobic condition. The colony sizes of these strains were checked on the 6<sup>th</sup>, 10<sup>th</sup>, and 15<sup>th </sup>days.</div>
</li> <li> <para-num num="[0028]"> </para-num> <div class="description-line" id="p-0029" num="0028"> <figref idrefs="DRAWINGS">FIG. 16</figref> shows expression of IL-10 (lane 2) in the growth media of a genetically modified <i>P. acnes </i>strain induced with 0.1 mM IPTG (isopropyl-beta-D-thiogalactopyranoside) for 1.5 hours. Pre-induced growth media is shown in lane 1.</div>
</li> </description-of-drawings>
<heading id="h-0006">DETAILED DESCRIPTION</heading>
<li> <para-num num="[0029]"> </para-num> <div class="description-line" id="p-0030" num="0029">Unless otherwise indicated herein, the materials described in this section are not prior art to the claims in this application and are not admitted to be prior art by inclusion in this section.</div>
</li> <li> <para-num num="[0030]"> </para-num> <div class="description-line" id="p-0031" num="0030">Disclosed herein are genetically modified microbes and compositions comprising genetically modified microbes. Microbes, such as bacteria, can be genetically modified to express and/or secrete biomolecules that are beneficial to mammals. In certain embodiments genetically modified bacteria are further modified to substantially reduce, or eliminate, expression of harmful virulence factors (e.g., toxins or antigens). In some embodiments the growth and viability of such genetically modified microbes can be controlled/modulated by introduction of nucleic acids comprising inducible promoters that regulate the expression of proteins that are essential to the growth and/or survival of the modified microbes. In certain embodiments compositions comprising such genetically modified microbes are formulated for topical delivery to the skin of a mammal.</div>
</li> <li> <para-num num="[0031]"> </para-num> <div class="description-line" id="p-0032" num="0031">Microbes, or microorganisms, as used herein, can refer to microscopic organisms, single cell organisms, or multicellular organisms. Examples of microbes can include but are not limited to bacteria, archaea, protozoa, and fungi. In certain embodiments a microbe is a bacteria.</div>
</li> <li> <para-num num="[0032]"> </para-num> <div class="description-line" id="p-0033" num="0032">A “genetically modified organism” as used herein, can refer to an organism in which the genetic material of the organism has been altered using genetic engineering techniques. The term genetically modified also refers to multiple genetic modifications, e.g. 2, 3, 4, 5, 6, 7, 8, 9, 10 or more genetic modifications, for example, a microbe which has an exogenous gene introduced for expression of a protein, and a modification, such as a gene knockout, which reduces expression of an endogenous (microbe) gene encoding a pathogenic molecule (e.g., a pathogenic peptide or protein).</div>
</li> <li> <para-num num="[0033]"> </para-num> <div class="description-line" id="p-0034" num="0033">Microbes such as bacteria, eubacteria, yeast, fungi, can be genetically modified in order to produce one or more proteins, produce a biotherapeutic, alter metabolism, prevent overgrowth, and/or prevent expression of a biomolecule. In some embodiments a genetically modified organism is a genetically modified microbe. In some embodiments a genetically modified organism is a genetically modified bacteria. In certain embodiments a <i>P. acnes </i>bacteria is genetically modified. Those skilled in the art will appreciate that there are multiple ways to produce a genetically modified organism such as by producing a gene knockout, by introducing heterologous nucleic acids and/or by generating mutations.</div>
</li> <li> <para-num num="[0034]"> </para-num> <div class="description-line" id="p-0035" num="0034">In some embodiments a nucleic acid (e.g., a gene, or portion thereof) is introduced into a microbe using a suitable technique. In some embodiments a microbe is transformed with a nucleic acid by a suitable technique. Non-limiting examples of suitable techniques for introducing a nucleic acid into a microbe include electroporation, transduction (e.g., injection of a nucleic acid by a bacteriophage), microinjection, by inducing competence (e.g., by addition of alkali cations, cesium, lithium, polyethylene glycol or by osmotic shock), the like or combinations thereof. A nucleic acid can be introduced into a microbe in the form of a linear or circular plasmid, for example. In some embodiments transformed microbes are selected for integration of a nucleic acid into the genome of the microbe by using a suitable selection method (e.g., a selection marker).</div>
</li> <li> <para-num num="[0035]"> </para-num> <div class="description-line" id="p-0036" num="0035">A gene encodes a specific protein, which after expression via transcription and translation fulfills a specific biochemical function within a living microbe. A gene sometimes includes segments of DNA involved in producing a polypeptide chain and sometimes includes regions preceding and following a coding region (e.g., an open reading frame) involved in the transcription/translation of a gene product and the regulation of the transcription/translation. An essential gene is an endogenous gene (e.g., endogenous to a microbe) that produces a polypeptide (e.g., an essential protein) that is necessary for the growth and/or viability of a microbe. Non-limiting examples of essential proteins of a bacteria include DnaA, FtsA, FtsI, FtsL, FtsK, FtsN, FtsQ, FtsW, FtsZ, ZipA, aroE, atpD, gmk, guaA, lepA, recA, and sodA.</div>
</li> <li> <para-num num="[0036]"> </para-num> <div class="description-line" id="p-0037" num="0036">A “gene knockout” as used herein, refers to a combination of genetic techniques that can render a specific gene inoperable or inactive. In some embodiments a gene knockout reduces or eliminates expression of a polypeptide from the gene. In certain embodiments the expression of gene is substantially reduced or eliminated. Substantially reduced means that the expression of a gene is reduced by at least 80%, at least 90%, at least 95% or at least 98% when compared to an endogenous level of expression of a gene. Expression of a gene can be determined by a suitable technique (e.g., by measuring transcript or expressed protein levels). Any suitable technique can be used to generate a gene knockout in a microbe (e.g., a bacteria). Gene knockouts in microbes can be made by transposon mutagenesis, in vitro genetic engineering to modify genes contained on plasmids or Bacterial Artificial Chromosomes (BACs) and moving the modified construct to the organism of interest, in vivo homologous recombination, and other techniques that are known to those skilled in the art. In certain embodiments a gene is knocked out by disabling an endogenous promoter, operon or regulatory element that is essential for transcription or translation of a gene. In some embodiments a gene is knocked out by introducing one or more mutations that disable the function of a protein expressed from a gene. In certain embodiments a gene is partially or completely removed from the genome of a microbe. In some embodiments an endogenous gene is knocked out by replacing the gene with a different gene (e.g., a heterologous gene, or non-functional gene).</div>
</li> <li> <para-num num="[0037]"> </para-num> <div class="description-line" id="p-0038" num="0037">In some embodiments, microbes are genetically modified to prevent secretion of a pathogenic molecule (e.g., a pathogenic peptide or protein). In some embodiments a pathogenic molecule is a toxic molecule (e.g., a toxin). In certain embodiments a gene encoding a pathogenic molecule and/or a toxic molecule is knocked out. In certain embodiments a bacteria is genetically modified to substantially reduce or eliminate the expression of a pathogenic molecule or a toxic molecule. Non-limiting examples of toxic molecules include bacterial endotoxins, bacterial exotoxins and bacterial antigens. A bacterial antigen is any bacteria derived molecule (e.g., compound or protein) than induces an immune response in a mammal. In some embodiments a pathogenic molecule is a Christie-Atkins-Munch-Petersen (CAMP) factor (e.g., a CAMP factor of <i>P. acnes</i>). In certain embodiments one or more CAMP factors (e.g., CAMP1, CAMP2, CAMP3, CAMP4 and/or CAMP5) of <i>P. acnes </i>are knocked out. In certain embodiments a <i>P. acnes </i>bacteria is genetically modified to substantially reduce or eliminate the expression of one or more CAMP factors. In some embodiments a pathogenic molecule is a bacterial glyceraldehyde 3-phosphate dehydrogenase (GAPDH) (e.g., a GAPDH of <i>P. acnes</i>). In certain embodiments one or more GAPDH genes of <i>P. acnes </i>are knocked out. In certain embodiments a <i>P. acnes </i>bacteria is genetically modified to substantially reduce or eliminate the expression of GAPDH.</div>
</li> <li> <para-num num="[0038]"> </para-num> <div class="description-line" id="p-0039" num="0038">In some embodiments, microbes are genetically modified to create nutritional auxotrophs. In some embodiments, microbes are genetically modified to prevent acceptance of genetic material. In some embodiments, microbes naturally or are genetically modified to prevent donation of genetic material. In some embodiments, the gene for Competence Protein ComEA is modified to prevent donation of genetic material. In some embodiments, the gene for Competence Protein ComEA is modified to prevent acceptance of genetic material. In some embodiments, the gene for Competence Protein ComFA is modified to prevent donation of genetic material. In some embodiments, the gene for Competence Protein ComFA is modified to prevent acceptance of genetic material. In some embodiments, the gene for Competence Protein ComA is modified to prevent donation of genetic material. In some embodiments, the gene for Competence Protein ComA is modified to prevent acceptance of genetic material. In some embodiments, the gene for Competence Protein ComK is modified to prevent donation of genetic material. In some embodiments, the gene for Competence Protein ComK is modified to prevent acceptance of genetic material. In certain embodiments a microbe comprises a CRISPR array, or a CRISPR array is introduced into a microbe (e.g., bacteria) to prevent or reduce acceptance of foreign genetic material from other microbes.</div>
</li> <li> <para-num num="[0039]"> </para-num> <div class="description-line" id="p-0040" num="0039">Genetic modification of the microbes can also be performed to introduce non-functional, and non-detrimental changes to the genome, in which there is no detrimental phenotype to the microbe. A “genetic marker” as described herein, refers to an introduced non-functional sequence in a genome in order to detect the presence of a genetically modified organism or microbe. Any suitable genetic marker can be incorporated into a genetically modified microbe using a suitable technique. Genetic modification can be performed in order to catalog and distinguish genetically modified microbes from wild type microbes, and in order to determine the presence of a genetically modified microbe in an environment. The genetically modified microbe can be determined in an environment by swabbing the source containing the suspected genetically modified microbe, growing up the microbe in a minimal media culture, obtaining the genetic material, using qPCR with specific primers to the sequence of interest, and sequencing the DNA of the genetically modified microbe. The use of genetic modification to introduce a specific genetic tag can allow one to catalog and determine the presence of genetically modified microbes on a subject of need. Furthermore, the techniques can allow one to determine that the genetically modified microbe is absent from the environment when the microbe is no longer needed. In some embodiments, microbes are genetically modified to carry a marker for determining the presence or absence of a genetically modified microbe on a host. Markers for determining the propagation of genetically modified bacteria can also be used to analyze the microflora of the subject to ensure that the bacteria are balanced during treatment.</div>
</li> <li> <para-num num="[0040]"> </para-num> <div class="description-line" id="p-0041" num="0040">Genetically modified microbes can be used to secrete biomolecules (e.g., proteins) to treat disorders in a subject in need. Genetically modified microbes can be used to treat subjects in need suffering from a skin disorder. Genetically modified microbes carrying nucleic acid for the controlled expression of peptides can be advantageous as they can move on the skin and propagate deep into the pores and hair follicles allowing absorption of secreted biomolecules. The microbes can be formulated for and used for general cosmesis, for example.</div>
</li> <li> <para-num num="[0041]"> </para-num> <div class="description-line" id="p-0042" num="0041">In several embodiments described herein, genetically modified microbes can be used to treat subjects suffering from a genetic disorder. In several embodiments herein, genetically modified microbes can be used to treat subjects suffering from gastric diseases. In several embodiments herein, genetically modified microbes can be used to treat subjects suffering from autoimmune diseases. In several embodiments, genetically modified microbes can be used to treat subjects treated with anticoagulants. In several embodiments, genetically modified microbes can be used to treat subjects suffering from hemophilia. Genetically modified microbes including nucleic acids for peptides for treatment can be used in conjunction in order to meet the needs of a subject in need. In some embodiments, a second genetically modified microbe can be used to treat a subject in need. In some embodiments, a third genetically modified microbe can be used to treat a subject in need. In some embodiments, more than three genetically modified microbes can be used to treat a subject in need.</div>
</li> <li> <para-num num="[0042]"> </para-num> <div class="description-line" id="p-0043" num="0042">In certain embodiments a genetically modified microbe comprises a nucleic acid (e.g., a gene) where expression of the gene is regulated by a promoter. A promoter is often introduced at a suitable location relative to a gene of interest. For example, a promoter (e.g., an inducible promoter) is often placed 5′ of a transcription start site of a gene of interest. In certain embodiments a nucleic acid includes a promoter and/or regulatory elements necessary to drive the expression of a gene (e.g., a heterologous gene or an endogenous gene). A promoter can be an endogenous promoter, a heterologous promoter or a combination thereof. In some embodiments a promoter is a constitutive promoter (e.g., a T7, SP6, T3, or any suitable constitutive promoter). In some embodiments a microbe is genetically altered to include a gene (e.g., a gene of interest, an essential gene) under the control of an inducible promoter. An inducible promoter is often a nucleic acid sequence that directs the conditional expression of a gene. An inducible promoter can be an endogenous promoter, a heterologous promoter, or a combination thereof. An inducible promoter can comprise an operon system. An inducible promoter is often configured to regulate the expression of a gene (e.g., a gene of interest, an essential gene). In certain embodiments an inducible promoter comprises one or more genes, regulatory elements and/or gene products (e.g., a inducible system). In some embodiments an inducible promoter requires the presence of a certain compound, nutrient, amino acid, sugar, peptide, protein or condition (e.g., light, oxygen, heat, cold) to induce gene activity (e.g., transcription). In certain embodiment an inducible promoter comprises one or more repressor elements. In some embodiments an inducible promoter (e.g., a promoter comprising a repressor element) requires the absence of a certain compound, nutrient, amino acid, sugar, peptide, protein or condition to induce gene activity (e.g., transcription). Any suitable inducible promoter, system or operon can be used to regulate the expression of a gene (e.g., an essential gene). Non-limiting examples of inducible promoters include lactose regulated systems (e.g., lactose operon systems), sugar regulated systems, metal regulated systems, steroid regulated systems, alcohol regulated systems, IPTG inducible systems, arabinose regulated systems (e.g., arabinose operon systems, e.g., an ARA operon promoter, pBAD, pARA, P<sub>ARA</sub>E, <sub>ARA</sub>E, <sub>ARA</sub>R-P<sub>araE</sub>, portions thereof, combinations thereof and the like), synthetic amino acid regulated systems (e.g., see Rovner A J, et al., (2015) <i>Nature </i>518(7537):89-93), fructose repressors, a tac promoter/operator (pTac), tryptophan promoters, PhoA promoters, recA promoters, proU promoters, cst-1 promoters, tetA promoters, cadA promoters, nar promoters, P<sub>L </sub>promoters, cspA promoters, the like or combinations thereof. In certain embodiments a promoter comprises a Lac-Z (LacZ) promoter, or portions thereof. In some embodiments a promoter comprises a Lac operon, or portions thereof. In some embodiments a inducible promoter comprises an ARA operon promoter, or portions thereof. In certain embodiments an inducible promoter comprises an arabinose promoter or portions thereof. An arabinose promoter can be obtained from any suitable bacteria. In certain embodiments an inducible promoter comprises an arabinose operon of <i>E. coli </i>or <i>B. subtilis</i>. In certain embodiments an inducible promoter is activated by the presence of a sugar or an analog thereof. Non-limiting examples of sugars and sugar analogs include lactose, arabinose (e.g., L-arabinose), glucose, sucrose, fructose, IPTG, and the like.</div>
</li> <li> <para-num num="[0043]"> </para-num> <div class="description-line" id="p-0044" num="0043">The use of microbes for the production of biomolecules can also be controlled for a measure of safety. In some embodiments a genetically modified microbe is engineered as a nutritional auxotroph where the growth and/or survival of the microbe depends on the presence of an essential nutrient. In some embodiments a composition herein comprises such an essential nutrient. Nutritional auxotrophs can be controlled by the depletion of supplied nutrients in order to prevent microbe overgrowth, or to remove the microbe from the environment. For example, for Trp auxotrophs, the supply for tryptophan can be removed or depleted in order to slow the microbe growth or to remove the microbe from the environment completely. In the case of Lys auxotrophs, for example, the supplied lysine can be removed from the environment to slow the growth of the Lys auxotroph or to remove it completely from the environment. In certain embodiments a composition comprises an amino acid such as lysine (Lys) or tryptophan (Trp). In some embodiments, the supplied nutrient can be applied as a compound in a revitalizing topical composition in order to maintain the microbe in the environment as needed.</div>
</li> <li> <para-num num="[0044]"> </para-num> <div class="description-line" id="p-0045" num="0044">Disclosed herein are methods for making a nucleic acid for controlled expression of a peptide for treatment. Gene transcripts for the peptide for treatment can be synthesized through standard molecular cloning techniques known to those skilled in the art and can be used to transform microbes to carry the encoded gene transcript of interest. In certain embodiments an inducible promoter comprises an operon. In some embodiments, a nucleic acid includes an operon sequence. An operon can be used to control expression of the gene transcript or the expression of a peptide for treatment at the DNA level. In some embodiments, the operon is a lac operon. In some embodiments, the operon is a Trp operon. In some embodiments, the operon is a repressor operon. Repressor operons as described herein, refers to operons that are controlled by a repressor, or a DNA or RNA binding protein, that can inhibit the expression of one or more genes by binding to the operator, or operon. In a lac operon, the gene is turned off if there is a level of lactose that can bind to the operator and inhibit the RNA polymerase from binding, thus decreasing transcription of the gene of interest. The trp operon is also a repressor operon that works through a negative repressible feedback mechanism. The repressor for the trp operon is tryptophan, which can bind to the operator and prevent transcription of the gene. By controlling the production of the gene of interest by an operon, one can control the level of production of the peptide of treatment by supplying the microbes with the repressor molecule to control the level of secreted biomolecules that are being produced. In some embodiments, the expression of peptide for treatment can be repressed by adding a repressor molecule. In some embodiments, the repressor molecule is tryptophan. In some embodiments, the repressor molecule is lactose.</div>
</li> <li> <para-num num="[0045]"> </para-num> <div class="description-line" id="p-0046" num="0045">Biomolecules as described herein, refer to any type of molecule that is produced by a living organism. The biomolecules can include but are not limited to macromolecules, proteins, sugars, polysaccharides, lipids, nucleic acids, peptides, metabolites, glycolipids, sterols, growth factors, hormones, glycerolipids, vitamins, neurotransmitters, metabolites, enzymes, monomers, oligomers, and polymers. Biomolecules can be produced by microbes. In certain embodiments a gene of interest encodes a biomolecule. In several embodiments described herein, methods are described in which genetically modified microbes secrete a biomolecule. In several embodiments, methods are described in which genetically modified microbes carry an enzyme that catalyzes the production of a biomolecule. In several embodiments, methods are described in which genetically modified microbes carry an enzyme that catalyzes the production hyaluronic acid. In several embodiments, methods are described in which genetically modified microbes carry an enzyme that catalyzes the production of melanin. Different types of biomolecules can be used to treat subjects in need. In some embodiments, the subject suffers from skin disorders, genetic disorders, diseases, autoimmune disease, gastric disease, aging damage, and hemophilia. In some embodiments a subject or a subject in need suffers from acne.</div>
</li> <li> <para-num num="[0046]"> </para-num> <div class="description-line" id="p-0047" num="0046">In certain embodiments a microbe is genetically modified to express and/or secrete a growth factor. A growth factor can be a mammalian growth factor (e.g., a human growth factor).</div>
</li> <li> <para-num num="[0047]"> </para-num> <div class="description-line" id="p-0048" num="0047">Growth factors are naturally occurring biomolecules that are capable of initiating and stimulating cellular growth, causing cell signaling, proliferation, healing, and the differentiation of cells. Growth factors can be a protein or a peptide. In some embodiments a growth factor is a hormone (e.g., a mammalian hormone). In certain embodiments a bacteria is genetically modified by introduction of a nucleic acid that encodes one or more growth factors (e.g., a mammalian growth factors). A nucleic acid that encodes a growth factor may include a suitable promoter and/or regulatory elements that drive the transcription and/or translation (e.g., expression) of the growth factor, an open reading frame that encodes the growth factor and, in some embodiments, suitable nucleic and/or peptide elements that direct microbial secretion of the growth factor. In certain embodiments a bacteria is genetically modified by introduction of two or more nucleic acids that encode two or more growth factors (e.g., a mammalian growth factors). In certain embodiments a bacteria is genetically modified by introduction of a nucleic acid that directs the expression of one or more growth factors (e.g., a mammalian growth factors). In certain embodiments a bacteria is genetically modified to express and/or secrete a growth factor (e.g., a mammalian growth factor).</div>
</li> <li> <para-num num="[0048]"> </para-num> <div class="description-line" id="p-0049" num="0048">Several examples of growth factors include but are not limited to vascular endothelial growth factor (VEGF), platelet derived growth factor (PDGF), adrenomedullin (AM), angiopoietin (Ang), autocrine motility factor, bone morphogenetic proteins (BMPs), brain-derived neurotrophic factor (BDNF), epidermal growth factor (EGF), erythropoietin (EPO), fibroblast growth factor (FGF), glial cell line-derived neurotrophic factor (GDNF), granulocyte colony-stimulating factor (G-CSF), granulocyte macrophage colony-stimulating factor (GM-CSF), growth differentiation factor-9 (GDF9), healing factor, hepatocyte growth factor (HGF), hepatoma-derived growth factor (HDGF), insulin-like growth factor (IGF), migration-stimulating factor, myostatin (GDF-8), nerve growth factor (NGF) and other neurotrophins, platelet-derived growth factor (PDGF), thrombopoietin (TPO), transforming growth factor alpha (TGF-α), transforming growth factor beta (TGF-β), tumor necrosis factor-alpha (TNF-α), vascular endothelial growth factor (VEGF), Wnt signaling pathway, placental growth factor (PGF), fetal bovine somatotrophin (FBS), IL-1-Co-factor for IL-3 and IL-6, IL-2-T-cell growth factor, IL-3, IL-4, IL-5, IL-6, and IL-7. In some embodiments, the genetically modified microbe (e.g., a bacteria) secretes a growth factor.</div>
</li> <li> <para-num num="[0049]"> </para-num> <div class="description-line" id="p-0050" num="0049">Transforming growth factor beta is a hormone that controls proliferation, differentiation and other functions in many cell types. Many cells synthesize TGFB1 and have specific receptors for it. It positively and negatively regulates many other growth factors. It plays an important role in bone remodeling as it is a potent stimulator of osteoblastic bone formation, causing chemotaxis, proliferation and differentiation in committed osteoblasts. In some embodiments the peptide for treatment includes transforming growth factor beta. TGFB can be used for general cosmesis, aging damage, general aging of tissue to cause more youthful skin by attracting fibroblasts, and causing them to produce extracellular matrix, and leading to thicker, more youthful skin. In some embodiments, the transforming growth factor beta includes SEQ ID NO: 63, SEQ ID NO: 64, SEQ ID NO: 65, SEQ ID NO: 66, SEQ ID NO: 67, SEQ ID NO: 68 or SEQ ID NO: 69.</div>
</li> <li> <para-num num="[0050]"> </para-num> <div class="description-line" id="p-0051" num="0050">Hepatocyte growth factor/scatter factor (HGF/SF) is a hormone for paracrine cellular growth, motility and morphogenic factor. It is secreted by mesenchymal cells and targets and can act primarily on epithelial and endothelial cells, but also acts on hematopoietic progenitor cells. It has been shown to have a major role in embryonic organ development, specifically in myogenesis, in adult organ regeneration and in wound healing.</div>
</li> <li> <para-num num="[0051]"> </para-num> <div class="description-line" id="p-0052" num="0051">For example, receptor tyrosine-protein kinase (MET) can transduce signals from the extracellular matrix into the cytoplasm by binding to hepatocyte growth factor/HGF ligand. This process can regulate many physiological processes including proliferation, scattering, morphogenesis and survival. Ligand binding at the cell surface induces autophosphorylation of MET, on its intracellular domain that provides docking sites for downstream signaling molecules. Following activation by ligand, there are interactions with the PI3-kinase subunit PIK3R1, PLCG1, SRC, GRB2, STAT3 or the adapter GAB1. Recruitment of these downstream effectors by MET leads to the activation of several signaling cascades including the RAS-ERK, PI3 kinase-AKT, or PLC gamma-PKC cascades. The RAS-ERK activation is associated with the morphogenetic effects, while PI3K/AKT coordinates pro-survival effects. During embryonic development, MET signaling plays a role in gastrulation, development and migration of muscles and neuronal precursors, angiogenesis and kidney formation. In adults, the cascade participates in wound healing, as well as organ regeneration and tissue remodeling.</div>
</li> <li> <para-num num="[0052]"> </para-num> <div class="description-line" id="p-0053" num="0052">Hepatocyte growth factor can be used for general cosmesis, to cause more youthful skin by attracting fibroblasts, and causing them to produce extracellular matrix, and leading to thicker, more youthful skin. The process can also promote differentiation and proliferation of hematopoietic cells. In some embodiments, the peptide for treatment includes hepatocyte growth factor. In some embodiments, the hepatocyte growth factor includes SEQ ID NO: 70.</div>
</li> <li> <para-num num="[0053]"> </para-num> <div class="description-line" id="p-0054" num="0053">Vascular endothelial growth factor (VEGF) is a signal protein that is produced by cells to stimulate vasculogenesis and angiogenesis. It is part of the system that restores the oxygen supply to tissues when blood circulation is inadequate. VEGF's normal function is to create new blood vessels during embryonic development, new blood vessels after injury, muscle following exercise, and new vessels to bypass blocked vessels. VEGFs can be used for general cosmesis, to cause more youthful skin by attracting fibroblasts, and causing them to produce extracellular matrix, and leading to thicker, more youthful skin.</div>
</li> <li> <para-num num="[0054]"> </para-num> <div class="description-line" id="p-0055" num="0054">VEGF-A is a growth factor active in angiogenesis, vasculogenesis and endothelial cell growth. VEGF-A induces endothelial cell proliferation, promotes cell migration, inhibits apoptosis and induces permeabilization of blood vessels. VEGF-A binds to FLT1/VEGFR1 and KDR/VEGFR2 receptors, heparin sulfate, and heparin. NRP1/Neuropilin-1 binds isoforms VEGF-165 and VEGF-145. Isoform VEGF165B binds to KDR but does not activate downstream signaling pathways, does not activate angiogenesis, and inhibits tumor growth. In some embodiments, the peptide for treatment includes VEGF-A. In some embodiments, VEGF-A includes SEQ ID NO: 71, SEQ ID NO: 72, SEQ ID NO: 73, SEQ ID NO: 74, SEQ ID NO: 75, SEQ ID NO: 76, SEQ ID NO: 77, SEQ ID NO: 78, SEQ ID NO: 79, SEQ ID NO: 80, SEQ ID NO: 81, SEQ ID NO: 82, SEQ ID NO: 83, SEQ ID NO: 84, SEQ ID NO: 85, SEQ ID NO: 86, or SEQ ID NO: 87.</div>
</li> <li> <para-num num="[0055]"> </para-num> <div class="description-line" id="p-0056" num="0055">VEGF-B is a growth factor for endothelial cells. VEGF-B167 binds heparin and neuropilin-1 whereas the binding to neuropilin-1 of VEGF-B186 is regulated by proteolysis. In some embodiments, the peptide for treatment includes VEGF-B. In some embodiments, VEGF-B includes SEQ ID NO: 88 or SEQ ID NO: 89.</div>
</li> <li> <para-num num="[0056]"> </para-num> <div class="description-line" id="p-0057" num="0056">VEGF-C is a growth factor active in angiogenesis, and endothelial cell growth, stimulating their proliferation and migration and also has effects on the permeability of blood vessels. VEGF-C can function in angiogenesis of the venous and lymphatic vascular systems during embryogenesis, and also in the maintenance of differentiated lymphatic endothelium in adults. VEGF-C can binds and activate VEGFR-2 (KDR/FLK1) and VEGFR-3 (FLT4) receptors. In some embodiments, the peptide for treatment includes VEGF-C. In some embodiments, VEGF-C includes SEQ ID NO: 90.</div>
</li> <li> <para-num num="[0057]"> </para-num> <div class="description-line" id="p-0058" num="0057">VEGF-D (c-Fos-induced growth factor, or FIGF) is a growth factor active in angiogenesis, lymphangiogenesis and endothelial cell growth, stimulating their proliferation and migration and also has effects on the permeability of blood vessels. VEGF-D can function in the formation of the venous and lymphatic vascular systems during embryogenesis, and also in the maintenance of differentiated lymphatic endothelium in adults. VEGF-D can bind and activate VEGFR-2 (KDR/FLK1) and VEGFR-3 (FLT4) receptors. In some embodiments, the peptide for treatment includes VEGF-D. In some embodiments, VEGF-D includes SEQ ID NO: 91.</div>
</li> <li> <para-num num="[0058]"> </para-num> <div class="description-line" id="p-0059" num="0058">Placental growth factor (PGF) is a member of the VEGF family, and can play a role in angiogenesis and vasculogenesis, during embryogenesis. PGF is active in angiogenesis and endothelial cell growth, stimulating their proliferation and migration. It binds to the receptor FLT1/VEGFR-1. Isoform PlGF-2 binds NRP1/neuropilin-1 and NRP2/neuropilin-2 in a heparin-dependent manner. PFG can also promote cell tumor growth. PGF can be used for general cosmesis, to cause more youthful skin by attracting fibroblasts, and causing them to produce extracellular matrix, and leading to thicker, more youthful skin. In some embodiments the peptide for treatment includes PGF. In some embodiments, PGF includes SEQ ID NO: 92.</div>
</li> <li> <para-num num="[0059]"> </para-num> <div class="description-line" id="p-0060" num="0059">Platelet-derived growth factor subunit A (PDGFA) plays an essential role in the regulation of embryonic development, cell proliferation, cell migration, survival and chemotaxis. PDGFA is a potent mitogen for cells of mesenchymal origin. PDGFA is required for normal lung alveolar septum formation during embryogenesis, normal development of the gastrointestinal tract, normal development of Leydig cells and spermatogenesis. PDGFA is required for normal oligodendrocyte development and normal myelination in the spinal cord and cerebellum. PDGFA plays an important role in wound healing. Signaling can also be modulated by the formation of heterodimers with PDGFB. PDGFA can be used for general cosmesis, to cause more youthful skin by attracting fibroblasts, and causing them to produce extracellular matrix, and leading to thicker, more youthful skin. In some embodiments, the peptide for treatment includes PDGFA. In some embodiments, PDGFA includes SEQ ID NO: 93 or SEQ ID NO: 94.</div>
</li> <li> <para-num num="[0060]"> </para-num> <div class="description-line" id="p-0061" num="0060">Platelet-derived growth factor subunit B (PDGFB) plays an essential role in the regulation of embryonic development, cell proliferation, cell migration, survival and chemotaxis. PDGFB is a potent mitogen for cells of mesenchymal origin. PDGFB is required for normal proliferation and recruitment of pericytes and vascular smooth muscle cells in the central nervous system, skin, lung, heart and placenta. PDGFB is required for normal blood vessel development, and for normal development of kidney glomeruli. PDGFB plays an important role in wound healing. Signaling can also be modulated by the formation of heterodimers of PDGFB with PDGFA. PDGFB can be used for general cosmesis, to cause more youthful skin by attracting fibroblasts, and causing them to produce extracellular matrix, and leading to thicker, more youthful skin. In some embodiments, the peptide for treatment includes PDGFB. In some embodiments, PDGFB includes SEQ ID NO: 95 or SEQ ID NO: 96.</div>
</li> <li> <para-num num="[0061]"> </para-num> <div class="description-line" id="p-0062" num="0061">Platelet-derived Growth Factor C (PDGFC) plays an essential role in the regulation of embryonic development, cell proliferation, cell migration, survival and chemotaxis. PDGFC is a potent mitogen and chemo-attractant for cells of mesenchymal origin. PDGFC is required for normal skeleton formation during embryonic development, especially for normal development of the craniofacial skeleton and for normal development of the palate. PDGFC is required for normal skin morphogenesis during embryonic development. PDGFC plays an important role in wound healing, where it appears to be involved in three stages: inflammation, proliferation and remodeling. PDGFC plays an important role in angiogenesis and blood vessel development and is involved in fibrotic processes, in which transformation of interstitial fibroblasts into myofibroblasts plus collagen deposition occurs. The CUB domain of PDGFC has a mitogenic activity in coronary artery smooth muscle cells, suggesting a role beyond the maintenance of the latency of the PDGF domain. In the nucleus, PDGFC seems to have additional function. PDGFC can be used for general cosmesis, to cause more youthful skin by attracting fibroblasts, and causing them to produce extracellular matrix, and leading to thicker, more youthful skin. In some embodiments, the peptide for treatment includes PDGFC. In some embodiments, PDGFC includes or SEQ ID NO: 97, SEQ ID NO: 98, SEQ ID NO: 99, or SEQ ID NO: 100.</div>
</li> <li> <para-num num="[0062]"> </para-num> <div class="description-line" id="p-0063" num="0062">Platelet-derived Growth Factor D (PDGFD) plays an essential role in the regulation of embryonic development, cell proliferation, cell migration, survival and chemotaxis. PDGFD is a potent mitogen for cells of mesenchymal origin. PDGFD plays an important role in wound healing. PDGFD induces macrophage recruitment, increased interstitial pressure, and blood vessel maturation during angiogenesis. PDGFD can initiate events that lead to a mesangial proliferative glomerulonephritis, including influx of monocytes and macrophages and production of extracellular matrix. PDGFD can be used for general cosmesis, to cause more youthful skin by attracting fibroblasts, and causing them to produce extracellular matrix, and leading to thicker, more youthful skin. In some embodiments, the peptide for treatment includes PDGFD. In some embodiments, PDGFB includes SEQ ID NO: 101 or SEQ ID NO: 102.</div>
</li> <li> <para-num num="[0063]"> </para-num> <div class="description-line" id="p-0064" num="0063">The gene for Epidermal Growth Factor (EGF) encodes a member of the epidermal growth factor superfamily. The encoded protein is synthesized as a large precursor molecule that is proteolytically cleaved to generate the 53-amino acid epidermal growth factor peptide. This protein acts a potent mitogenic factor that plays an important role in the growth, proliferation and differentiation of numerous cell types. This protein acts by binding the high affinity cell surface receptor, epidermal growth factor receptor. Defects in this gene are the cause of hypomagnesemia type 4. Dysregulation of this gene has been associated with the growth and progression of certain cancers. Alternate splicing results in multiple transcript variants and isoforms. Pro-EGF can be used for general cosmesis, to cause more youthful skin by attracting fibroblasts, and causing them to produce extracellular matrix, and leading to thicker, more youthful skin. In some embodiments, the peptide for treatment includes Pro-EGF. Pro-EGF can have isoforms due to alternative splicing. In some embodiments, Pro-EGF includes SEQ ID NO: 103, SEQ ID NO: 104, SEQ ID NO: 105 or SEQ ID NO: 106.</div>
</li> <li> <para-num num="[0064]"> </para-num> <div class="description-line" id="p-0065" num="0064">Fibroblast Growth Factor (FGF) plays an important role in the regulation of cell survival, cell division, angiogenesis, cell differentiation and cell migration. FGF can function as potent mitogen in vitro. FGF can be used for general cosmesis, to cause more youthful skin by attracting fibroblasts, and causing them to produce extracellular matrix, and leading to thicker, more youthful skin. There are 22 different isoforms of FGF.</div>
</li> <li> <para-num num="[0065]"> </para-num> <div class="description-line" id="p-0066" num="0065">In some embodiments, the peptide for treatment includes FGF1. In some embodiments, FGF1 includes SEQ ID NO: 107 or SEQ ID NO: 108.</div>
</li> <li> <para-num num="[0066]"> </para-num> <div class="description-line" id="p-0067" num="0066">FGF2 has 4 isoforms produced by alternative initiation. FGF2 plays an important role in the regulation of cell survival, cell division, angiogenesis, cell differentiation and cell migration. FGF2 functions as potent mitogen in vitro. In some embodiments, the peptide for treatment includes FGF2. In some embodiments, FGF2 includes SEQ ID NO: 109, SEQ ID NO: 110, SEQ ID NO: 111, or SEQ ID NO: 112.</div>
</li> <li> <para-num num="[0067]"> </para-num> <div class="description-line" id="p-0068" num="0067">FGF3 has 1 isoform, and plays an important role in the regulation of embryonic development, cell proliferation, and cell differentiation. FGF3 is required for normal ear development. In some embodiments, the peptide for treatment includes FGF3. In some embodiments, FGF3 includes SEQ ID NO: 113.</div>
</li> <li> <para-num num="[0068]"> </para-num> <div class="description-line" id="p-0069" num="0068">FGF4 can have two isoforms due to alternative splicing. FGF4 plays an important role in the regulation of cell survival, cell division, angiogenesis, cell differentiation and cell migration. FGF4 can functions as potent mitogen in vitro. In some embodiments, the peptide for treatment includes FGF4. In some embodiments, FGF4 includes SEQ ID NO: 114 or SEQ ID NO: 115.</div>
</li> <li> <para-num num="[0069]"> </para-num> <div class="description-line" id="p-0070" num="0069">FGF5 can have two isoforms due to alternative splicing. FGF5 can play an important role in the regulation of cell proliferation and cell differentiation. FGF5 is required for normal regulation of the hair growth cycle. FGF5 can function as an inhibitor of hair elongation by promoting progression from anagen, the growth phase of the hair follicle, into catagen, the apoptosis-induced regression phase. In some embodiments, the peptide for treatment includes FGF5. In some embodiments, FGF5 includes SEQ ID NO: 116 or SEQ ID NO: 117.</div>
</li> <li> <para-num num="[0070]"> </para-num> <div class="description-line" id="p-0071" num="0070">The FGF6 isoform plays an important role in the regulation of cell proliferation, cell differentiation, angiogenesis and myogenesis, and is required for normal muscle regeneration. In some embodiments, the peptide for treatment includes FGF6. In some embodiments, FGF6 includes SEQ ID NO: 118.</div>
</li> <li> <para-num num="[0071]"> </para-num> <div class="description-line" id="p-0072" num="0071">FGF7 (Keratinocyte Growth Factor, KGF) has one isoform and plays an important role in the regulation of embryonic development, cell proliferation and cell differentiation. FGF7 is required for normal branching morphogenesis. The growth factor is particularly active upon keratinocytes. FGF7 is a possible major paracrine effector of normal epithelial cell proliferation. In some embodiments, the peptide for treatment includes FGF7. In some embodiments, FGF7 includes SEQ ID NO: 119.</div>
</li> <li> <para-num num="[0072]"> </para-num> <div class="description-line" id="p-0073" num="0072">FGF8 has four isoforms due to alternative splicing and plays an important role in the regulation of embryonic development, cell proliferation, cell differentiation and cell migration. FGF8 is required for normal brain, eye, ear, and limb development during embryogenesis. FGF8 is required for normal development of the gonadotropin-releasing hormone (GnRH) neuronal system. In some embodiments, the peptide for treatment includes FGF8. In some embodiments, FGF8 includes, SEQ ID NO: 120, SEQ ID NO: 121, SEQ ID NO: 122, or SEQ ID NO: 123.</div>
</li> <li> <para-num num="[0073]"> </para-num> <div class="description-line" id="p-0074" num="0073">FGF9 has one isoform and plays an important role in the regulation of embryonic development, cell proliferation, cell differentiation and cell migration. FGF9 can have a role in glial cell growth and differentiation during development, gliosis during repair and regeneration of brain tissue after damage, differentiation and survival of neuronal cells, and growth stimulation of glial tumors. In some embodiments, the peptide for treatment includes FGF9. In some embodiments, FGF9 includes SEQ ID NO: 124.</div>
</li> <li> <para-num num="[0074]"> </para-num> <div class="description-line" id="p-0075" num="0074">FGF10 (KGF2) has one isoform, and plays an important role in the regulation of embryonic development, cell proliferation and cell differentiation. FGF10 is required for normal branching morphogenesis. FGF10 can play a role in wound healing. In some embodiments, the peptide for treatment includes FGF10. In some embodiments, FGF10 includes SEQ ID NO: 125.</div>
</li> <li> <para-num num="[0075]"> </para-num> <div class="description-line" id="p-0076" num="0075">FGF11 has one isoform and is involved in nervous system development and function. In some embodiments, the peptide for treatment includes FGF11. In some embodiments, FGF11 includes SEQ ID NO: 126.</div>
</li> <li> <para-num num="[0076]"> </para-num> <div class="description-line" id="p-0077" num="0076">FGF12 has two isoforms due to alternative splicing and is involved in nervous system development and function. In some embodiments, the peptide for treatment includes FGF12. In some embodiments, FGF12 includes SEQ ID NO: 127 or SEQ ID NO: 128.</div>
</li> <li> <para-num num="[0077]"> </para-num> <div class="description-line" id="p-0078" num="0077">FGF13 has 5 isoforms produced by alternative splicing. FGF13 is a microtubule-binding protein which directly binds tubulin and is involved in both polymerization and stabilization of microtubules. Through its action on microtubules, FGF13 can participate in the refinement of axons by negatively regulating axonal and leading processes branching. FGF13 plays a crucial role in neuron polarization and migration in the cerebral cortex and the hippocampus. In some embodiments, the peptide for treatment includes FGF13. In some embodiments, FGF13 includes or SEQ ID NO: 129, SEQ ID NO: 130, SEQ ID NO: 131, SEQ ID NO: 132, or SEQ ID NO: 133.</div>
</li> <li> <para-num num="[0078]"> </para-num> <div class="description-line" id="p-0079" num="0078">FGF14 has two isoforms produced by alternative splicing and is involved in nervous system development and function. In some embodiments, the peptide for treatment includes FGF14. In some embodiments, FGF14 includes SEQ ID NO: 134 or SEQ ID NO: 135.</div>
</li> <li> <para-num num="[0079]"> </para-num> <div class="description-line" id="p-0080" num="0079">FGF16 has one isoform and plays an important role in the regulation of embryonic development, cell proliferation and cell differentiation, and is required for normal cardiomyocyte proliferation and heart development. In some embodiments, the peptide for treatment includes FGF16. In some embodiments, FGF16 includes SEQ ID NO: 136.</div>
</li> <li> <para-num num="[0080]"> </para-num> <div class="description-line" id="p-0081" num="0080">FGF17 has two isoforms due to alternate splicing and plays an important role in the regulation of embryonic development and as signaling molecule in the induction and patterning of the embryonic brain. FGF17 is required for normal brain development. In some embodiments, the peptide for treatment includes FGF17. In some embodiments, FGF17 includes SEQ ID NO: 137 or SEQ ID NO: 138.</div>
</li> <li> <para-num num="[0081]"> </para-num> <div class="description-line" id="p-0082" num="0081">FGF18 has one isoform and plays an important role in the regulation of cell proliferation, cell differentiation and cell migration. FGF18 is required for normal ossification and bone development. FGF18 stimulates hepatic and intestinal proliferation. In some embodiments, the peptide for treatment includes FGF18. In some embodiments, FGF18 includes SEQ ID NO: 139.</div>
</li> <li> <para-num num="[0082]"> </para-num> <div class="description-line" id="p-0083" num="0082">FGF19 has one isoform and is involved in the suppression of bile acid biosynthesis through down-regulation of CYP7A1 expression, following positive regulation of the JNK and ERK1/2 cascades. FGF19 stimulates glucose uptake in adipocytes. The activity of FGF19 requires the presence of KLB and FGFR4. In some embodiments, the peptide for treatment includes FGF19. In some embodiments, FGF19 includes SEQ ID NO: 140.</div>
</li> <li> <para-num num="[0083]"> </para-num> <div class="description-line" id="p-0084" num="0083">FGF20 has one isoform and plays as role as a neurotrophic factor that regulates central nervous development and function. In some embodiments, the peptide for treatment includes FGF20. In some embodiments, FGF20 includes SEQ ID NO: 141.</div>
</li> <li> <para-num num="[0084]"> </para-num> <div class="description-line" id="p-0085" num="0084">FGF21 has one isoform and stimulates glucose uptake in differentiated adipocytes via the induction of glucose transporter SLC2A1/GLUT1 expression (but not SLC2A4/GLUT4 expression). The activity of FGF21 requires the presence of KLB. In some embodiments, the peptide for treatment includes FGF21. In some embodiments, FGF21 includes SEQ ID NO: 142.</div>
</li> <li> <para-num num="[0085]"> </para-num> <div class="description-line" id="p-0086" num="0085">FGF22 has one isoform and plays a role in the fasting response, glucose homeostasis, lipolysis and lipogenesis. FGF22 can stimulate cell proliferation in vitro, and can be involved in hair development. In some embodiments, a cell secreting FGF22 can be used to treat alopecia. In some embodiments, the peptide for treatment includes FGF22. In some embodiments, FGF22 includes SEQ ID NO: 143.</div>
</li> <li> <para-num num="[0086]"> </para-num> <div class="description-line" id="p-0087" num="0086">FGF23 has one isoform and is a regulator of phosphate homeostasis. FGF23 inhibits renal tubular phosphate transport by reducing SLC34A1 levels and can upregulate EGR1 expression in the presence of KL. FGF23 acts directly on the parathyroid to decrease PTH secretion. FGF23 is also a regulator of vitamin-D metabolism and can negatively regulates osteoblast differentiation and matrix mineralization. In some embodiments, the peptide for treatment includes FGF23. In some embodiments, FGF23 includes SEQ ID NO: 144.</div>
</li> <li> <para-num num="[0087]"> </para-num> <div class="description-line" id="p-0088" num="0087">Hormones are a class of regulatory biochemicals that are produced in all organisms by glands, and transported by the circulatory system to distant target organs to coordinate its physiology, function, and behavior. Hormones can serve as a major form of communication between different organs and tissues. Hormones regulate a variety of physiological and behavioral activities, including digestion, metabolism, respiration, tissue function, sensory perception, sleep, perception, stress, growth and development, movement, and reproduction. In some embodiments, the peptide for treatment includes a hormone. Hormones can be used by subjects suffering from disease that is contributed by low hormone production in a subject. In certain embodiments a bacteria is genetically modified by introduction of a nucleic acid that encodes a hormone (e.g., a mammalian hormone). In certain embodiments a bacteria is genetically modified to express and/or secrete a hormone (e.g., a mammalian hormone). In certain embodiments a hormone is a somatotrophin. In certain embodiments a hormone is a mammalian somatotrophin. In certain embodiments a hormone is a bovine or human somatotrophin. In certain embodiments a somatotrophin is growth hormone (GH).</div>
</li> <li> <para-num num="[0088]"> </para-num> <div class="description-line" id="p-0089" num="0088">Somatotrophin is a hormone that plays an important role in growth control. Its major role in stimulating body growth is to stimulate the liver and other tissues to secrete IGF-1. It stimulates both the differentiation and proliferation of myoblasts. It also stimulates amino acid uptake and protein synthesis in muscle and other tissues. In some embodiments the peptide for treatment includes somatotrophin. In some embodiments, the somatotrophin includes SEQ ID NO: 45, SEQ ID NO: 46, SEQ ID NO: 47, SEQ ID NO: 48 or SEQ ID NO: 49. Somatotrophin can be used by subjects suffering from low human growth hormone (HGH) production or subjects suffering from low testosterone production. In some embodiments, a cell including a nucleic acid for expressing a peptide includes an amino acid sequence for somatotrophin. As microbes can have the benefit of having the ability to get beneath the horny layer of the dermis and into the pores, the skin would have a better and increased opportunity to absorb these topicals than if the molecules themselves were applied topically by a lotion or a cream. One benefit of delivering a microbe-produced hormone to a subject in need is that such a method reduces the risk of exposure of the hormone to other subjects. As, in some embodiments, a genetically modified microbe is often engineered to depend on essential nutrients provide only to a subject in need. Therefore, delivery of a microbe-produced hormone to a subject in need can be safer than other methods of providing topical hormones which can often result in unwanted transfer to other subjects.</div>
</li> <li> <para-num num="[0089]"> </para-num> <div class="description-line" id="p-0090" num="0089">Anti-inflammatories are substances or treatments that reduce inflammation. Interleukins (IL) are a group of cytokines that can function as signaling molecules and act as an anti-inflammatory. Anti-inflammatories can be used by subjects in need suffering from an inflammatory disorder, such as acne. Microbes secreting anti-inflammatories have an advantage over creams containing anti-inflammatories, as microbes have the benefit of having the ability to get beneath the horny layer of the dermis and into the pores, increasing the opportunity to absorb these anti-inflammatories.</div>
</li> <li> <para-num num="[0090]"> </para-num> <div class="description-line" id="p-0091" num="0090">In some embodiments a microbe is genetically modified to express and/or secrete an anti-inflammatory. In certain embodiments an anti-inflammatory is a cytokine. In some embodiments, the genetically modified microbe secretes a cytokine. Non-limiting examples of cytokines include IL-4, IL-6, IL-7, IL-8, IL-10 and IL-13. In certain embodiments a bacteria is genetically modified by introduction of a nucleic acid that encodes one or more cytokines (e.g., a mammalian cytokines). A nucleic acid that encodes a cytokine may include a suitable promoter and/or regulatory elements that drive the transcription and/or translation (e.g., expression) of the cytokine, an open reading frame that encodes the cytokine and, in some embodiments, suitable nucleic acid and/or peptide elements that direct microbial secretion of the cytokine. In certain embodiments a bacteria is genetically modified by introduction of two or more nucleic acids that encode two or more cytokines (e.g., a mammalian cytokines). In certain embodiments a bacteria is genetically modified by introduction of a nucleic acid that directs the expression of one or more cytokines (e.g., a mammalian cytokines). In certain embodiments a bacteria is genetically modified to express and/or secrete a cytokine (e.g., a mammalian cytokine).</div>
</li> <li> <para-num num="[0091]"> </para-num> <div class="description-line" id="p-0092" num="0091">Interleukin 4 (IL-4) can participate in at least several B-cell activation processes as well as of other cell types. It is a co-stimulator of DNA-synthesis. It induces the expression of class II MHC molecules on resting B-cells. It enhances both secretion and cell surface expression of IgE and IgG1. It also regulates the expression of the low affinity Fc receptor for IgE (CD23) on both lymphocytes and monocytes. In some embodiments, the peptide for treatment includes interleukin 4 or a portion thereof. In some embodiments, the interleukin 4 includes SEQ ID NO: 24.</div>
</li> <li> <para-num num="[0092]"> </para-num> <div class="description-line" id="p-0093" num="0092">Interleukin 10 (IL-10) can inhibits the synthesis of a number of cytokines, including IFN-gamma, IL-2, IL-3, TNF and GM-CSF produced by activated macrophages and by helper T-cells. In some embodiments, the peptide for treatment includes interleukin 10 or a portion thereof. In some embodiments, the Interleukin 10 includes SEQ ID NO: 25.</div>
</li> <li> <para-num num="[0093]"> </para-num> <div class="description-line" id="p-0094" num="0093">Interleukin 13 (IL-13) is a cytokine that inhibits inflammatory cytokine production. Interleukin 13 synergizes with Interleukin 2 (IL2) in regulating interferon-gamma synthesis. Interleukin 13 can be critical in regulating inflammatory and immune responses. In some embodiments, the peptide for treatment includes Interleukin 13, or a portion thereof. In some embodiments, the interleukin 13 includes SEQ ID NO: 26.</div>
</li> <li> <para-num num="[0094]"> </para-num> <div class="description-line" id="p-0095" num="0094">Interleukin-1 receptor type 2 (IL1R2) is a non-signaling receptor for IL1A, IL1B and IL1RN. IL1R2 reduces IL1B activities. In certain embodiments a bacteria is genetically modified to express and/or secrete an IL1R2. IL1R2 can serve as a decoy receptor by competitive binding to IL1B and preventing its binding to IL1R1. IL1R2 can also modulate cellular response through non-signaling association with IL1RAP after binding to IL1B. IL1R2 (membrane and secreted forms) preferentially binds IL1B and poorly IL1A and IL1RN. The secreted IL1R2 recruits secreted IL1RAP with high affinity; this complex formation can be the dominant mechanism for neutralization of IL1B by secreted/soluble receptors. In some embodiments, the peptide for treatment includes IL1R2, or a portion thereof. In some embodiments, the interleukin 13 includes SEQ ID NO: 27 or SEQ ID NO: 28.</div>
</li> <li> <para-num num="[0095]"> </para-num> <div class="description-line" id="p-0096" num="0095">Hyaluronic acid or HA/Hyaluronan is a biomolecule that is synthesized by a class of integral membrane proteins called hyaluronan synthases, of which vertebrates have three types: HAS1, HAS2, and HAS3. It is an anionic, glycosaminoglycan distributed widely throughout connective, epithelial, and neural tissues. Hyaluronic acid is non-sulfated, and forms in the plasma membrane instead of the Golgi, and can be very large, with a molecular weight often reaching the millions (in KDa). One of the chief components of the extracellular matrix, hyaluronan contributes significantly to cell proliferation and migration, and can also be involved in the progression of some malignant tumors.</div>
</li> <li> <para-num num="[0096]"> </para-num> <div class="description-line" id="p-0097" num="0096">HAS1 catalyzes the addition of GlcNAc or GlcUA monosaccharides to a nascent hyaluronan polymer. It is essential in hyaluronan synthesis, since hyaluronic acid is a major component of most extracellular matrices that has a structural role in tissues architectures and regulates cell adhesion, migration and differentiation. HAS1 is one of the isozymes catalyzing the reaction of the addition of GlcNAc or GlcUA monosaccharides to hyaluronan polymers. HAS1 is also able to catalyze the synthesis of chito-oligosaccharide depending on the substrate. HAS2 catalyzes the addition of GlcNAc or GlcUA monosaccharides to a nascent hyaluronan polymer. HAS2 is one of the isozymes that catalyzes this reaction and it is particularly responsible for the synthesis of high molecular mass hyaluronan. HAS 2 is required for the transition of endocardial cushion cells into mesenchymal cells, a process crucial for heart development. HAS2 can also play a role in vasculogenesis. High molecular mass hyaluronan can also play a role in early contact inhibition, a process which stops cell growth when cells come into contact with each other or the extracellular matrix. HAS3 catalyzes the addition of GlcNAc or GlcUA monosaccharides to the nascent hyaluronan polymer.</div>
</li> <li> <para-num num="[0097]"> </para-num> <div class="description-line" id="p-0098" num="0097">As a treatment, dry, scaly skin also known as xerosis such as that caused by atopic dermatitis or eczema can be treated with a prescription skin lotion containing sodium hyaluronate as its active ingredient. In several embodiments described herein, a nucleic acid encoding hyaluronan synthases or portion thereof is provided. In several embodiments, a cell can carry a nucleic acid encoding hyaluronan synthases or portion thereof. In some embodiments, the hyaluron synthases includes HAS1 (SEQ ID NO: 1) HAS2 (SEQ ID NO: 2), HAS3 isoform 1 (SEQ ID NO: 3), or HAS3 isoform 2 (SEQ ID NO: 4). Hyaluronic acid can be used for general cosmesis, plumping of the skin to reduce wrinkles. In some embodiments, the cell carrying a nucleic acid encoding hyaluronan synthases or portion thereof can lead to production of hyaluronic acid. In several embodiments, the peptide for treatment can be used for general cosmesis. As the cell can propagate and localize in pores and follicles, the absorption of hyaluronic acid is increased compared to topically applied lotions containing hyaluronic acid. In some embodiments, a subject in need suffering from xerosis is administered a topical formulation including a cell having a nucleic acid encoding hyaluronan synthases or portion thereof.</div>
</li> <li> <para-num num="[0098]"> </para-num> <div class="description-line" id="p-0099" num="0098">Elastin is a major structural protein of tissues such as aorta and nuchal ligament, which must expand rapidly and recover completely. Elastin is a molecular determinant of late arterial morphogenesis, it can also stabilize arterial structure by regulating proliferation and organization of vascular smooth muscle. Elastin can be used to contribute to skin elasticity as it can be readily absorbed by the skin. In some embodiments, the peptide for treatment includes elastin or a portion thereof. As microbes can have the benefit of having the ability to get beneath the horny layer of the dermis and into the pores, the skin would have a better and increased opportunity to absorb elastin from bacteria than if the elastin was applied topically through a lotion or a cream.</div>
</li> <li> <para-num num="[0099]"> </para-num> <div class="description-line" id="p-0100" num="0099">In some embodiments, the elastin, or portion thereof includes SEQ ID NO: 5, SEQ ID NO: 6, SEQ ID NO: 7, SEQ ID NO: 8, SEQ ID NO: 9, SEQ ID NO: 10, SEQ ID NO: 11, SEQ ID NO: 12, SEQ ID NO: 13, SEQ ID NO: 14, SEQ ID NO: 15, SEQ ID NO: 16, or SEQ ID NO: 17.</div>
</li> <li> <para-num num="[0100]"> </para-num> <div class="description-line" id="p-0101" num="0100">Collagen is a main structural protein of the various connective tissues in animals. Collagen, in the form of elongated fibrils, is can be found in fibrous tissues, tendons, ligaments, skin, and is also abundant in corneas, cartilage, bones, blood vessels, the gut, and intervertebral discs. Collagen is created mostly from the fibroblast. Collagen is composed of a triple helix, consisting of two identical a1chains and an additional chain that can differ in chemical composition. The amino acid composition of collagen can have high hydroxyproline content. In some embodiments, the peptide of treatment includes collagen or a portion thereof. Collagen can be absorbed by the skin to add to the extracellular matrix, and the dermal thickness. As microbes can have the benefit of having the ability to get beneath the horny layer of the dermis and into the pores, the skin would have a better and increased opportunity to absorb the collagen than if the molecules themselves were applied topically by a lotion or a cream. In some embodiments, peptide of treatment including collagen or a portion thereof includes SEQ ID NO: 18, SEQ ID NO: 19, SEQ ID NO: 20, SEQ ID NO: 21, SEQ ID NO: 22 or SEQ ID NO: 23.</div>
</li> <li> <para-num num="[0101]"> </para-num> <div class="description-line" id="p-0102" num="0101">Clotting factors as described herein, refers to chemical and cellular constituents that can cause the blood to coagulate or clot. Blood clotting factors can be used externally by a subject in need suffering from hemophilia, or by a subject under treatment of a blood thinner. Blood thinners can include fish oil, aspirin, anti-platelet drugs, and other types of anticoagulants. Anti-coagulants can include but is not limited to antithrombics, fibrinolytic, and thrombolytics.</div>
</li> <li> <para-num num="[0102]"> </para-num> <div class="description-line" id="p-0103" num="0102">Coagulation Factor VIII is a member of the multi-copper oxidase family. Coagulation Factor VIII is a cofactor for factor IXa which, in the presence of Ca<sup>+2 </sup>and phospholipids, converts factor X to an activated form, Xa. Coagulation Factor VIII is a coagulation cofactor which circulates bound to von Willebrand factor and is part of the intrinsic coagulation pathway. It is a macromolecular complex composed of two separate entities, one of which, when deficient, results in hemophilia A, and the other, when deficient, results in von Willebrand's disease. Hemophilia A is a disorder of blood coagulation characterized by a permanent tendency to hemorrhage. In some embodiments, the peptide for treatment includes coagulation factor VIII. In some embodiments, the coagulation factor VIII includes SEQ ID NO: 29 or SEQ ID NO: 30. In some embodiments, the peptide for treatment includes coagulation factor VIII heavy chain in a 200 kDa isoform. In some embodiments, the coagulation factor VIII heavy chain in a 200 kDa includes SEQ ID NO: 31. In some embodiments, the peptide for treatment includes coagulation factor VIII heavy chain in a 92 kDa isoform. In some embodiments, the coagulation factor VIII heavy chain in a 92 kDa includes SEQ ID NO: 32. In some embodiments, the peptide for treatment includes coagulation factor VIII B chain. In some embodiments, the coagulation factor VIII B chain includes SEQ ID NO: 33. In some embodiments, the peptide for treatment includes coagulation factor VIIIa light chain. In some embodiments, the coagulation factor VIIIa light chain includes SEQ ID NO: 34.</div>
</li> <li> <para-num num="[0103]"> </para-num> <div class="description-line" id="p-0104" num="0103">Factor IX is a vitamin K-dependent plasma protein that participates in the intrinsic pathway of blood coagulation by converting factor X to its active form in the presence of Ca2+ ions, phospholipids, and factor VIIIa. In some embodiments, the peptide for treatment is Factor IX. In some embodiments, the Factor IV includes SEQ ID NO: 35, SEQ ID NO: 36, SEQ ID NO: 37, SEQ ID NO: 38 or SEQ ID NO: 39.</div>
</li> <li> <para-num num="[0104]"> </para-num> <div class="description-line" id="p-0105" num="0104">Melanin in the skin is produced by melanocytes, found in the basal layer of the epidermis. Although, in general, human beings possess a similar concentration of melanocytes in their skin, the melanocytes in some individuals and ethnic groups more frequently or less frequently express the melanin-producing genes, thereby conferring a greater or lesser concentration of skin melanin.</div>
</li> <li> <para-num num="[0105]"> </para-num> <div class="description-line" id="p-0106" num="0105">Tyrosinase is an oxidase, the rate-limiting enzyme for controlling the production of melanin. Tyrosinase is involved in the hydroxylation of a monophenol and the conversion of an o-diphenol to the corresponding o-quinone. o-Quinone undergoes several reactions to eventually form melanin. In some embodiments, the peptide for treatment is produced by tyrosinase, or fragment thereof. In some embodiments, the tyrosinase includes SEQ ID NO: 40, SEQ ID NO: 41, SEQ ID NO: 42, SEQ ID NO: 43 or SEQ ID NO: 44. In some embodiments, cells can include, for example, a nucleic acid encoding tyrosinase, or fragment thereof. In some embodiments, the microbes produce tyrosinase for the enzymatic production of melanin. As microbes can have the benefit of having the ability to get beneath the horny layer of the dermis and into the pores, the skin would have a better and increased opportunity to absorb these topicals than if the molecules themselves were applied topically by a lotion or a cream. The production of melanin can be used for “sunless” tanning in order to increase the melanin in a subject.</div>
</li> <li> <para-num num="[0106]"> </para-num> <div class="description-line" id="p-0107" num="0106">Chemokines are a family of small cytokines that are secreted by cells as signaling molecules. Proteins classified as chemokines are small in size (8-10 KDa in size), and have four conserved cysteine residues in conserved locations, forming a conserved 3-dimensional shape among the chemokines. Chemokines can be considered pro-inflammatory and can be induced during an immune response to recruit cells of the immune system to a site of infection, while others are considered homeostatic and are involved in controlling the migration of cells during normal processes of tissue maintenance or development. Chemokines are found in all vertebrates, some viruses and some bacteria, but none have been described for other invertebrates. In some embodiments, the peptide for treatment includes a chemokine.</div>
</li> <li> <para-num num="[0107]"> </para-num> <div class="description-line" id="p-0108" num="0107">Platelet basic protein (LA-PF4) belongs to the chemokine family. LA-PF4 stimulates DNA synthesis, mitosis, glycolysis, intracellular cAMP accumulation, prostaglandin E2 secretion, and synthesis of hyaluronic acid and sulfated glycosaminoglycan. It also stimulates the formation and secretion of plasminogen activator by human synovial cells. NAP-2 is a ligand for CXCR1 and CXCR2, and NAP-2, NAP-2(73), NAP-2(74), NAP-2(1-66), and most potent NAP-2(1-63) are chemo-attractants and activators for neutrophils. TC-1 and TC-2 are antibacterial proteins, in vitro released from activated platelet alpha-granules. CTAP-III (1-81) is more potent than CTAP-III desensitize chemokine-induced neutrophil activation. LA-PF4 can be used for general cosmesis, to cause more youthful skin by attracting fibroblasts, and causing them to produce extracellular matrix, and leading to thicker, more youthful skin. In some embodiments the peptide for treatment includes platelet basic protein. In some embodiments, the platelet basic protein includes SEQ ID NO: 50, SEQ ID NO: 51, SEQ ID NO: 52, SEQ ID NO: 53, SEQ ID NO: 54, SEQ ID NO: 55, SEQ ID NO: 56, SEQ ID NO: 57, SEQ ID NO: 58, SEQ ID NO: 59, SEQ ID NO: 60, SEQ ID NO: 61 or SEQ ID NO: 62.</div>
</li> <li> <para-num num="[0108]"> </para-num> <div class="description-line" id="p-0109" num="0108">DNA repair enzymes are used for the repair of DNA damaged by reactive oxygen species, replication errors, exogenous damage caused by external agents such as ultraviolet rays, toxins, mutagenic chemicals, DNA intercalating agents, and viruses. There are several types of damage which can be oxicatin of bases, alkylation of bases, deamination, depurination, mismatch of bases, monoadduct damage, and diadduct damage. DNA repair enzymes can be used for anti-aging, and reversal of DNA damage via solar damage/radiation.</div>
</li> <li> <para-num num="[0109]"> </para-num> <div class="description-line" id="p-0110" num="0109">Base excision repair (BER) is a mechanism that repairs damaged DNA during the cell cycle, removing small non-helix distorting base lesions from the genome. BER is very important as it removes damaged bases that could lead to mutations by mispairing or lead to breaks in the DNA during replication. The process is initiated by DNA glycosylases, which can recognize and remove damaged or inappropriate bases, forming AP sites. The AP sites are then cleaved by an AP endonuclease resulting in a single strand break that can then be processed by a short or a long patch repair process.</div>
</li> <li> <para-num num="[0110]"> </para-num> <div class="description-line" id="p-0111" num="0110">In some embodiments, the peptide for treatment includes base excision repair (BER) enzymes. In some embodiments, BER enzymes includes SEQ ID NO: 145, SEQ ID NO: 146, SEQ ID NO: 147, SEQ ID NO: 148, SEQ ID NO: 149, SEQ ID NO: 150, SEQ ID NO: 151, SEQ ID NO: 152, SEQ ID NO: 153, SEQ ID NO: 154, or SEQ ID NO: 155.</div>
</li> <li> <para-num num="[0111]"> </para-num> <div class="description-line" id="p-0112" num="0111">Direct reversal of DNA damage is another repair mechanism to restore damaged DNA. The formation of pyrimidine dimers, is the major type of damage that is caused by UV light. The damage distorts the DNA double helix and blocks transcription or replication past the damaged site. In some embodiments, the peptide for treatment includes enzymes for the direct reversal of DNA damage. In some embodiments, the enzymes for the direct reversal of DNA damage includes SEQ ID NO: 156, SEQ ID NO: 157, or SEQ ID NO: 158.</div>
</li> <li> <para-num num="[0112]"> </para-num> <div class="description-line" id="p-0113" num="0112">DNA mismatch repair proteins are involved in the recognition and repairing of nucleic acid which have erroneous insertions, deletions, and misincorporation of bases that can stem from the processes of DNA replication and recombination, and from DNA damage. In some embodiments, the peptide for treatment includes enzymes for DNA mismatch repair. In some embodiments, the enzymes for DNA mismatch repair includes SEQ ID NO: 159, SEQ ID NO: 160, SEQ ID NO: 161, SEQ ID NO: 162, SEQ ID NO: 163, SEQ ID NO: 164, SEQ ID NO: 165, SEQ ID NO: 166, SEQ ID NO: 167, or SEQ ID NO: 168.</div>
</li> <li> <para-num num="[0113]"> </para-num> <div class="description-line" id="p-0114" num="0113">Nucleotide excision repair (NER) is a DNA repair mechanism that removes DNA damage that is caused by ultraviolet light. The damage of ultraviolet light can lead to DNA adducts which can consist of thymine dimers, and 6,4-photoproducts. The NER proteins recognize the damage leading to the removal of short single-stranded DNA segments that contain the lesion. The undamaged compliment is then used as a template by DNA polymerase to synthesize a short complementary sequence which is subsequently ligated by a DNA ligase. In some embodiments, the peptide for treatment includes enzymes for nucleotide excision repair. In some embodiments, the enzymes for nucleotide excision repair includes or SEQ ID NO: 169, SEQ ID NO: 170, SEQ ID NO: 171, SEQ ID NO: 172, SEQ ID NO: 173, SEQ ID NO: 174, SEQ ID NO: 175, SEQ ID NO: 176, SEQ ID NO: 177, SEQ ID NO: 178, SEQ ID NO: 179, SEQ ID NO: 180, SEQ ID NO: 181, SEQ ID NO: 182, SEQ ID NO: 183, SEQ ID NO: 184, SEQ ID NO: 185, SEQ ID NO: 186, SEQ ID NO: 187, SEQ ID NO: 188, SEQ ID NO: 189, SEQ ID NO: 190, SEQ ID NO: 191, SEQ ID NO: 192, SEQ ID NO: 193, SEQ ID NO: 194, SEQ ID NO: 195, SEQ ID NO: 196, or SEQ ID NO: 197.</div>
</li> <li> <para-num num="[0114]"> </para-num> <div class="description-line" id="p-0115" num="0114">DNA editing and processing involve the use of several types of nucleases and are involved in DNA replication and repair. For example, DNase IV can remove the 5′ overhanging flaps in DNA repair and processes the 5′ ends of Okazaki fragments in lagging strand DNA synthesis. Direct physical interaction between this protein and AP endonuclease 1 during long-patch base excision repair provides coordinated loading of the proteins onto the substrate, thus passing the substrate from one enzyme to another. The protein is a member of the XPG/RAD2 endonuclease family and is one of ten proteins essential for cell-free DNA replication.</div>
</li> <li> <para-num num="[0115]"> </para-num> <div class="description-line" id="p-0116" num="0115">MTMR15, also known as myotubularin related protein is a DNA endo- and exonuclease involved in the repair of DNA damage caused by crosslinking agents. FAN1 is recruited to sites of interstrand cross linkage damage by interacting with the protein complex FANCI-FANCD2 complex. Together the proteins promote interstrand crosslink repair in a manner strictly dependent on its ability to accumulate at or near sites of DNA damage and that relies on monoubiquitylation of the FANCI-FANCD2 complex.</div>
</li> <li> <para-num num="[0116]"> </para-num> <div class="description-line" id="p-0117" num="0116">DNase III, or TREX1 is a major nuclear DNA-specific 3′-5′ exonuclease that is widely distributed in both proliferating and nonproliferating mammalian tissues. DNase III translocates to the nucleus at S phase after DNA damage by gamma-irradiation or hydroxyurea. DNase III has a preference for single stranded DNA in repair.</div>
</li> <li> <para-num num="[0117]"> </para-num> <div class="description-line" id="p-0118" num="0117">TREX2 encodes for a 3′exonuclase. The encoded protein can participate in double stranded DNA break repair, and can interact with DNA polymerase delta. TREX2 can remove mismatched modified, fragmented, and normal nucleotides in order to generate the 3′ termini for subsequent steps in the DNA metabolic pathway.</div>
</li> <li> <para-num num="[0118]"> </para-num> <div class="description-line" id="p-0119" num="0118">EXO1/HEX1 is an 803 amino acid human protein that functions in DNA replication, repair and recombination. EXO1/HEX1 can participate in mismatch-provoked excision directed by strand breaks located either 5-prime or 3-prime to the mispair.</div>
</li> <li> <para-num num="[0119]"> </para-num> <div class="description-line" id="p-0120" num="0119">Aprataxin (APTX) is another protein involved in editing and processing of DNA. APTX plays a role in single stranded DNA repair by removing AMP form DNA ends following ligation attempts of DNA ligase IV during non-homologous end joining.</div>
</li> <li> <para-num num="[0120]"> </para-num> <div class="description-line" id="p-0121" num="0120">SPO11 is an endonuclease of the editing and processing nucleases that functions during meiotic recombination. SPO11 produces double stranded breaks (DSB) in DNA, an important step during the meiotic recombination. In absence of SPO11, there is a failure to initiate the production of DSB which can lead to chromosomes segregating aberrantly which can result in aneuploidy gametes.</div>
</li> <li> <para-num num="[0121]"> </para-num> <div class="description-line" id="p-0122" num="0121">Endonuclease V (ENDOV) is a nuclease of the editing and processing nucleases, and is an endoribonuclease that specifically cleaves inosine-containing RNAs, at the second phosphodiester bond 3′ to inosine. ENDOV has a strong preference for single-stranded RNAs (ssRNAs) toward double-stranded RNAs (dsRNAs). ENDOV cleaves mRNAs and tRNAs containing inosine. ENDOV is also able to cleave structure-specific dsRNA substrates containing the specific sites 5′-IIUI-3′ and 5′-UIUU-3′. Inosine is present in a number of RNAs following editing; the function of inosine-specific endoribonuclease is still unclear. Inosine could either play a regulatory role in edited RNAs, or it can be involved in antiviral response by removing the hyperedited long viral dsRNA genome that has undergone A-to-I editing.</div>
</li> <li> <para-num num="[0122]"> </para-num> <div class="description-line" id="p-0123" num="0122">In some embodiments, the peptide for treatment includes editing and processing nucleases. In some embodiments, the enzymes for editing and processing includes SEQ ID NO: 198, SEQ ID NO: 199, SEQ ID NO: 200, SEQ ID NO: 201, SEQ ID NO: 202, SEQ ID NO: 203, SEQ ID NO: 204, or SEQ ID NO: 205.</div>
</li> <li> <para-num num="[0123]"> </para-num> <div class="description-line" id="p-0124" num="0123">Telomeres are regions at the tips of the chromosomes that get shorter during aging and during cell division, or mitosis. In order to repair the tips of the chromosomes, an enzyme, telomerase, repairs the tips which can have potential to repair age or disease related damage. Telomerase, is a ribonucleoprotein that adds DNA sequence repeats to the 3′end of DNA in the telomeric regions (the ends of eukaryotic chromosomes). In some embodiments, the peptide for treatment comprises telomerase or a portion thereof. In some embodiments, the telomerase or portion thereof comprises SEQ ID NO: 206, SEQ ID NO: 207, SEQ ID NO: 208, or SEQ ID NO: 209.</div>
</li> <li> <para-num num="[0124]"> </para-num> <div class="description-line" id="p-0125" num="0124">Protection of telomeres protein 1 is encoded by POT1, a member of the telombin family and is involved in the maintenance of the telomere. POT1 functions by binding to the repeat of telomeres at the end of the eukaryotic chromosome, regulating the telomere length and protecting the chromosome ends from irregular recombination, instability, and abnormal chromosome instability. In several embodiments, the peptide for treatment comprises protection of telomeres protein 1 or a portion thereof. In some embodiments, the protection of telomeres protein 1 or portion thereof comprises SEQ ID NO: 210, or SEQ ID NO: 211.</div>
</li> <li> <para-num num="[0125]"> </para-num> <div class="description-line" id="p-0126" num="0125">In some embodiments, methods are described wherein the peptide for treatment comprises a fusion protein. In some embodiments, the fusion protein can comprise a first protein sequence comprising an amino acid sequence of elastin, collagen, an anti-inflammatory, clotting factor, hormone, platelet basic protein, transforming growth factor, hepatocyte growth factor, vascular endothelial growth factor, placental growth factor, platelet derived growth factor, epidermal growth factor, fibroblast growth factor, a DNA repair enzyme, a telomerase, or a protection of telomerase protein 1. In some embodiments, the fusion protein is fused to a second protein comprising an amino acid sequence of elastin, collagen, an anti-inflammatory, clotting factor, hormone, platelet basic protein, transforming growth factor, hepatocyte growth factor, vascular endothelial growth factor, placental growth factor, platelet derived growth factor, epidermal growth factor, fibroblast growth factor, a DNA repair enzyme, a telomerase, or a protection of telomerase protein 1, wherein the amino acid sequence of the second protein is not the amino acid sequence of the first protein. Fusion proteins can have the added benefit of introducing a second moiety to the peptide of treatment to treat a subject in need.</div>
</li> <li> <para-num num="[0126]"> </para-num> <div class="description-line" id="p-0127" num="0126">In several embodiments described herein, populations of genetically modified bacteria are created and/or derived from bacterial members from a group consisting of the genus <i>Propionibacterium, Cornybacterium Staphyloccous, Streptococcus, Lactobacillus</i>, and <i>Lactococcus</i>. In some embodiments genetically modified bacteria are derived from the genus <i>Propionibacterium</i>. Non-limiting examples of species of <i>Propionibacterium </i>include <i>Propionibacterium acidifaciens, Propionibacterium acidipropionici, Propionibacterium acnes, Propionibacterium australiense, Propionibacterium avidum, Propionibacterium cyclohexanicum, Propionibacterium freudenreichii, Propionibacterium freudenreichii, Propionibacterium granulosum, Propionibacterium jensenii, Propionibacterium microaerophilum, Propionibacterium propionicum </i>and <i>Propionibacterium thoeniiand. </i> </div>
</li> <li> <para-num num="[0127]"> </para-num> <div class="description-line" id="p-0128" num="0127">In some embodiments a genetically modified bacteria is derived from the species <i>Propionibacterium acnes </i>(<i>P. acnes</i>). A genetically modified bacteria can be derived from a pathogenic or non-pathogenic strain of <i>P. acnes</i>. Recent studies have suggested that there are certain strains of <i>P. acnes </i>that are associated with pathogenicity and others that are associated with healthy skin (Fitz-Gibbon, S., et al., (2013) <i>Invest. Dermatol., </i>133(9):2152-60; Lomholt H B and Kilian M. (2010) PLoS One, 5(8):e12277; McDowell A, et al., (2011) <i>Microbiology </i>157(Pt 7):1990-2003). Of the types thought to be associated with healthy skin, the Type II, ribotype 6 strain seems to have the lowest association with acne. Type II <i>P. acnes </i>contain a CRISPR array that confers immunity to <i>P. acnes</i>-specific phages and mobile genetic elements (Brüggemann H, et al., (2012) PLoS ONE 7(3):e34171). This seems to explain why the bacteria may be commensal in nature, as it cannot acquire pathogenic traits from other bacterium.</div>
</li> <li> <para-num num="[0128]"> </para-num> <div class="description-line" id="p-0129" num="0128">A genetically modified bacteria can be derived from any suitable microbiome of <i>P. acnes</i>, non-limiting examples of which include microbiomes of type I, type II, type III, type IV and type V. In certain embodiments a genetically modified bacteria is derived from a strain of <i>P. acnes </i>of a phenotype of type I (e.g., clades IA or type IB), type II or type III. A genetically modified bacteria can be derived from any suitable ribotype of <i>P. acnes</i>, non-limiting examples of which include ribotypes RT1 through RT30. In certain embodiments a genetically modified bacteria is derived from a <i>P. acnes </i>of ribotype RT1, RT2, RT3, RT4, RT5, RT6, RT7, RT8, RT9 or RT10. In certain embodiments a genetically modified bacteria is derived from a <i>P. acnes </i>of ribotype RT2 or RT6. In certain embodiments a genetically modified bacteria is derived from a <i>P. acnes </i>of strain Type II and ribotype RT2 or RT6.</div>
</li> <li> <para-num num="[0129]"> </para-num> <div class="description-line" id="p-0130" num="0129">In some embodiments a genetically modified bacteria is derived from a bacteria comprising a CRISPR (clustered regularly interspaced short palindromic repeat) locus, sometimes referred to as a CRISPR array (e.g., see Horvath and Barrangou (2010) <i>Science </i>327:167-70; Makarova et al., (2011) <i>Nat Rev Microbiol </i>9:467-77; and Brüggemann H, et al., (2012) PLoS ONE 7(3):e34171). Without being limited to theory, CRISPR arrays in bacteria such as <i>P. acnes </i>have been shown to confer protective “immunity” against invading genetic elements (e.g., viruses, phage and plasmids). Without being limited to theory, the presence of a CRISPR array can preserve the integrity of the genome of a genetically modified bacteria and prevent introduction of foreign genetic elements from other pathogenic strains of bacteria. Although a CRISPR array is thought to protect bacteria from the introduction of foreign genetic elements from other bacteria, phage and/or virus, bacteria containing a CRISPR array are amenable to transformation, stable integration of transformed DNA and integration of nucleic acid by homologous recombination. In some embodiments a genetically modified bacteria comprises an endogenous CRISPR array. For example, in some embodiments a genetically modified bacteria is derived from an RT2 or RT6 ribotype of <i>P. acnes</i>, each of which comprises an endogenous CRISPR array. In some embodiments a genetically modified bacteria comprises an exogenous CRISPR array introduced by genetic manipulation.</div>
</li> <li> <para-num num="[0130]"> </para-num> <div class="description-line" id="p-0131" num="0130">In some embodiments, the populations of modified bacteria are created from <i>Propionibacterium acnes</i>, or a strain thereof. In some embodiments, the populations of transformed bacteria are created from <i>Corynebacterium striatum</i>. In some embodiments, the populations of transformed bacteria are created from <i>Staphylococcus epidermis</i>. In some embodiments, the populations of transformed bacteria are created from <i>Streptococcus thermophilus</i>. In some embodiments, the populations of transformed bacteria are created from <i>Lactobacillus acidophilus</i>. In some embodiments, the populations of transformed bacteria are created from <i>Lactococcus lactis</i>. In some embodiments, the populations of transformed bacteria are created from the genus <i>Enterrococci</i>. In some embodiments, the populations of transformed bacteria are created from the genus <i>Micrococci</i>. In some embodiments, the populations of transformed bacteria are created from the genus <i>Demodex</i>. In some embodiments, the populations of transformed bacteria are created from the genus <i>Malassezia</i>. In some embodiments, the populations of transformed bacteria are created from non-pathogenic <i>Eschericia coli</i>. In some embodiments, the populations of transformed bacteria are created from the genus <i>Acidovorax</i>. The populations of transformed bacteria are created from <i>Acidovorax temperans</i>. In some embodiments, the populations of transformed bacteria are created from the genus <i>Acinetobacter</i>. In some embodiments, the populations of transformed bacteria are created from <i>Acinetobacter haemolyticus</i>. In some embodiments, the populations of transformed bacteria are created from <i>Acinetobacter johnsonii</i>. In some embodiments, the populations of transformed bacteria are created from <i>Acinetobacter junii</i>. In some embodiments, the populations of transformed bacteria are created from <i>Acinetobacter ursingii</i>. In some embodiments, the populations of transformed bacteria are created from the genus <i>Actinomyces</i>. In some embodiments, the populations of transformed bacteria are created from <i>Actinomyces naeslundii</i>. In some embodiments, the populations of transformed bacteria are created from <i>Actinomyces neuii</i>. In some embodiments, the populations of transformed bacteria are created from the genus <i>Anaerococcus</i>. In some embodiments, the populations of transformed bacteria are created from <i>Anaerococcus prevotii</i>. In some embodiments, the populations of transformed bacteria are created from the genus <i>Atopobium</i>. In some embodiments, the populations of transformed bacteria are created from <i>Atopobium vaginae</i>. In some embodiments, the populations of transformed bacteria are created from the genus <i>Brevibacterium</i>. In some embodiments, the populations of transformed bacteria are created from <i>Brevibacterium paucivorans</i>. In some embodiments, the populations of transformed bacteria are created from the genus <i>Brevundimonas aurantiaca</i>. In some embodiments, the populations of transformed bacteria are created from <i>Brevundimonas aurantiaca</i>. In some embodiments, the populations of transformed bacteria are created from <i>Brevundimonas vesicularis</i>. In some embodiments, the populations of transformed bacteria are created from the genus <i>Candidatus Nostocoida</i>. In some embodiments, the populations of transformed bacteria are created from <i>Candidatus Nostocoida limicola</i>. In some embodiments, the populations of transformed bacteria are created from the genus <i>Corynebacterium</i>. In some embodiments, the populations of transformed bacteria are created from <i>Corynebacterium accolens</i>. In some embodiments, the populations of transformed bacteria are created from <i>Corynebacterium afermentans</i>. In some embodiments, the populations of transformed bacteria are created from <i>Corynebacterium amycolatum</i>. In some embodiments, the populations of transformed bacteria are created from <i>Corynebacterium appendicis</i>. In some embodiments, the populations of transformed bacteria are created from <i>Corynebacterium aurimucosum</i>. In some embodiments, the populations of transformed bacteria are created from <i>Corynebacterium coyleae</i>. In some embodiments, the populations of transformed bacteria are created from <i>Corynebacterium durum</i>. In some embodiments, the populations of transformed bacteria are created from <i>Corynebacterium glaucum</i>. In some embodiments, the populations of transformed bacteria are created from <i>Corynebacterium glucuronolyticum</i>. In some embodiments, the populations of transformed bacteria are created from <i>Corynebacterium imitans</i>. In some embodiments, the populations of transformed bacteria are created from <i>Corynebacterium jeikeium</i>. In some embodiments, the populations of transformed bacteria are created from <i>Corynebacterium kroppenstedtii</i>. In some embodiments, the populations of transformed bacteria are created from <i>Corynebacterium lipophiloflavum</i>. In some embodiments, the populations of transformed bacteria are created from <i>Corynebacterium matruchotii</i>. In some embodiments, the populations of transformed bacteria are created from <i>Corynebacterium minutissimum</i>. In some embodiments, the populations of transformed bacteria are created from <i>Corynebacterium mucifaciens</i>. In some embodiments, the populations of transformed bacteria are created from <i>Corynebacterium pseudodiphthericum</i>. In some embodiments, the populations of transformed bacteria are created from <i>Corynebacterium nigricans</i>. In some embodiments, the populations of transformed bacteria are created from <i>Corynebacterium pseudodiphthericum</i>. In some embodiments, the populations of transformed bacteria are created from <i>Corynebacterium simulans</i>. In some embodiments, the populations of transformed bacteria are created from <i>Corynebacterium singulare</i>. In some embodiments, the populations of transformed bacteria are created from <i>Corynebacterium sundsvallense</i>. In some embodiments, the populations of transformed bacteria are created from <i>Corynebacterium tuberculostearicum</i>. In some embodiments, the populations of transformed bacteria are created from the genus <i>Diaphorobacter</i>. In some embodiments, the populations of transformed bacteria are created from <i>Diaphorobacter nitroreducens</i>. In some embodiments, the populations of transformed bacteria are created from the genus <i>Enhydrobacter</i>. In some embodiments, the populations of transformed bacteria are created from <i>Enhydrobacter aerosaccus</i>. In some embodiments, the populations of transformed bacteria are created from the genus <i>Enterobacter</i>. In some embodiments, the populations of transformed bacteria are created from <i>Enterobacter asburiae</i>. In some embodiments, the populations of transformed bacteria are created from the genus <i>Enterococcus</i>. In some embodiments, the populations of transformed bacteria are created from <i>Enterococcus faecalis</i>. In some embodiments, the populations of transformed bacteria are created from the genus <i>Eremococcus</i>. In some embodiments, the populations of transformed bacteria are created from <i>Eremococcus coleocola</i>. In some embodiments, the populations of transformed bacteria are created from the genus <i>Facklamia</i>. In some embodiments, the populations of transformed bacteria are created from <i>Facklamia hominis</i>. In some embodiments, the populations of transformed bacteria are created from <i>Facklamia languida</i>. In some embodiments, the populations of transformed bacteria are created from the genus <i>Gardnerella. In some embodiments, the populations of transformed bacteria are created from Gardnerella vaginalis</i>. In some embodiments, the populations of transformed bacteria are created from the genus <i>Gemella</i>. In some embodiments, the populations of transformed bacteria are created from <i>Gemella haemolysans</i>. In some embodiments, the populations of transformed bacteria are created from <i>Gemella morbillorum</i>. In some embodiments, the populations of transformed bacteria are created from <i>Gemella sanguinis</i>. In some embodiments, the populations of transformed bacteria are created from the genus <i>Gordonia</i>. In some embodiments, the populations of transformed bacteria are created from <i>Gordonia. Bronchialis</i>. In some embodiments, the populations of transformed bacteria are created from <i>Gordonia. sputi</i>. In some embodiments, the populations of transformed bacteria are created from <i>Gordonia. terrae</i>. In some embodiments, the populations of transformed bacteria are created from the genus <i>Granulicatella</i>. In some embodiments, the populations of transformed bacteria are created from <i>Granulicatella elegans</i>. In some embodiments, the populations of transformed bacteria are created from the genus <i>Hyphomicrobium</i>. In some embodiments, the populations of transformed bacteria are created from <i>Hyphomicrobium facile</i>. In some embodiments, the populations of transformed bacteria are created from the genus <i>Janibacter</i>. In some embodiments, the populations of transformed bacteria are created from <i>Janibacter melonis</i>. In some embodiments, the populations of transformed bacteria are created from the genus <i>Kocuria</i>. In some embodiments, the populations of transformed bacteria are created from <i>Kocuria marina</i>. In some embodiments, the populations of transformed bacteria are created from <i>Kocuria palustris</i>. In some embodiments, the populations of transformed bacteria are created from <i>Kocuria rhizophila</i>. In some embodiments, the populations of transformed bacteria are created from the genus <i>Lactobacillus</i>. In some embodiments, the populations of transformed bacteria are created from <i>Lactobacillus crispatus</i>. In some embodiments, the populations of transformed bacteria are created from <i>Lactobacillus jensenii</i>. In some embodiments, the populations of transformed bacteria are created from the genus <i>Leuconostoc</i>. In some embodiments, the populations of transformed bacteria are created from <i>Leuconostoc argentinum</i>. In some embodiments, the populations of transformed bacteria are created from the genus <i>Methylobacterium</i>. In some embodiments, the populations of transformed bacteria are created from <i>Methylobacterium extorquens</i>. In some embodiments, the populations of transformed bacteria are created from <i>Methylobacterium mesophilicum</i>. In some embodiments, the populations of transformed bacteria are created from the genus <i>Micrococcus</i>. In some embodiments, the populations of transformed bacteria are created from <i>Micrococcus luteus</i>. In some embodiments, the populations of transformed bacteria are created from the genus <i>Microlunatus</i>. In some embodiments, the populations of transformed bacteria are created from <i>Microlunatus phosphovorus</i>. In some embodiments, the populations of transformed bacteria are created from the genus <i>Mobiluncus curtisii</i>. In some embodiments, the populations of transformed bacteria are created from <i>Mobiluncus curtisii </i>subsp. <i>holmesii</i>. In some embodiments, the populations of transformed bacteria are created from the genus <i>Mycobacterium</i>. In some embodiments, the populations of transformed bacteria are created from <i>Mycobacterium chlorophenolicum</i>. In some embodiments, the populations of transformed bacteria are created from <i>Mycobacterium obuense</i>. In some embodiments, the populations of transformed bacteria are created from the genus <i>Nakamurella</i>. In some embodiments, the populations of transformed bacteria are created from <i>Nakamurella multipartita</i>. In some embodiments, the populations of transformed bacteria are created from the genus <i>Pedomicrobium</i>. In some embodiments, the populations of transformed bacteria are created from <i>Pedomicrobium australicum</i>. In some embodiments, the populations of transformed bacteria are created from the genus <i>Peptoniphilus</i>. In some embodiments, the populations of transformed bacteria are created from <i>Peptoniphilus harei</i>. In some embodiments, the populations of transformed bacteria are created from the genus <i>Peptostreptococcus</i>. In some embodiments, the populations of transformed bacteria are created from <i>Peptostreptococcus anaerobius</i>. In some embodiments, the populations of transformed bacteria are created from the genus <i>Prevotella</i>. In some embodiments, the populations of transformed bacteria are created from <i>Prevotella bivia</i>. In some embodiments, the populations of transformed bacteria are created from <i>Prevotella corporis</i>. In some embodiments, the populations of transformed bacteria are created from <i>Prevotella disiens</i>. In some embodiments, the populations of transformed bacteria are created from <i>Prevotella melaninogenica</i>. In some embodiments, the populations of transformed bacteria are created from the genus <i>Propionibacterium</i>. In some embodiments, the populations of transformed bacteria are created from <i>Propionibacterium acnes</i>. In some embodiments, the populations of transformed bacteria are created from <i>Propionibacterium granulosum</i>. In some embodiments, the populations of transformed bacteria are created from the genus <i>Pseudomonas</i>. In some embodiments, the populations of transformed bacteria are created from <i>Pseudomonas aeruginosa</i>. In some embodiments, the populations of transformed bacteria are created from <i>Pseudomonas saccharophila</i>. In some embodiments, the populations of transformed bacteria are created from <i>Pseudomonas stutzeri</i>. In some embodiments, the populations of transformed bacteria are created from <i>Pseudomonas tremae</i>. In some embodiments, the populations of transformed bacteria are created from the genus <i>Rhodococcus</i>. In some embodiments, the populations of transformed bacteria are created from <i>Rhodococcus corynebacterioides</i>. In some embodiments, the populations of transformed bacteria are created from <i>Rhodococcus erythropolis</i>. In some embodiments, the populations of transformed bacteria are created from the genus <i>Rothia</i>. In some embodiments, the populations of transformed bacteria are created from <i>Rothia aeria</i>. In some embodiments, the populations of transformed bacteria are created from <i>Rothia dentocariosa</i>. In some embodiments, the populations of transformed bacteria are created from <i>Rothia mucilaginosa</i>. In some embodiments, the populations of transformed bacteria are created from <i>Rothia nasimurium</i>. In some embodiments, the populations of transformed bacteria are created from the genus <i>Serratia</i>. In some embodiments, the populations of transformed bacteria are created from <i>Serratia liquefaciens</i>. In some embodiments, the populations of transformed bacteria are created from <i>Serratia marcescens </i>subsp. <i>Sakuensis</i>. In some embodiments, the populations of transformed bacteria are created from the genus <i>Sphingobium</i>. In some embodiments, the populations of transformed bacteria are created from <i>Sphingobium amiens</i>. In some embodiments, the populations of transformed bacteria are created from the genus <i>Staphylococcus</i>. In some embodiments, the populations of transformed bacteria are created from <i>Staphylococcus capitis</i>. In some embodiments, the populations of transformed bacteria are created from <i>Staphylococcus caprae</i>. In some embodiments, the populations of transformed bacteria are created from <i>Staphylococcus cohnii</i>. In some embodiments, the populations of transformed bacteria are created from <i>Staphylococcus epidermidis</i>. In some embodiments, the populations of transformed bacteria are created from <i>Staphylococcus haemolyticus</i>. In some embodiments, the populations of transformed bacteria are created from <i>Staphylococcus hominis</i>. In some embodiments, the populations of transformed bacteria are created from <i>Staphylococcus saccharolyticus</i>. In some embodiments, the populations of transformed bacteria are created from <i>Staphylococcus warneri</i>. In some embodiments, the populations of transformed bacteria are created from the genus <i>Stenotrophomonas</i>. In some embodiments, the populations of transformed bacteria are created from <i>Stenotrophomonas maltophilia</i>. In some embodiments, the populations of transformed bacteria are created from the genus <i>Streptococcus</i>. In some embodiments, the populations of transformed bacteria are created from <i>Streptococcus agalactiae</i>. In some embodiments, the populations of transformed bacteria are created from <i>Streptococcus cristatus</i>. In some embodiments, the populations of transformed bacteria are created from <i>Streptococcus gordonii</i>. In some embodiments, the populations of transformed bacteria are created from <i>Streptococcus infantis</i>. In some embodiments, the populations of transformed bacteria are created from <i>Streptococcus intermedius</i>. In some embodiments, the populations of transformed bacteria are created from <i>Streptococcus mitis</i>. In some embodiments, the populations of transformed bacteria are created from <i>Streptococcus parasanguinis</i>. In some embodiments, the populations of transformed bacteria are created from <i>Streptococcus parasanguinis</i>. In some embodiments, the populations of transformed bacteria are created from <i>Streptococcus salivarius</i>. In some embodiments, the populations of transformed bacteria are created from <i>Streptococcus sanguinis</i>. In some embodiments, the populations of transformed bacteria are created from the genus <i>Tetrasphaera</i>. In some embodiments, the populations of transformed bacteria are created from <i>Tetrasphaera elongata</i>. In some embodiments, the populations of transformed bacteria are created from the genus <i>Tsukamurella</i>. In some embodiments, the populations of transformed bacteria are created from <i>Tsukamurella tyrosinosolvens</i>. In some embodiments, the populations of transformed bacteria are created from the genus <i>Tsukamurella</i>. In some embodiments, the populations of transformed bacteria are created from the genus <i>Veillonella</i>. In some embodiments, the populations of transformed bacteria are created from <i>Veillonella parvula</i>. In some embodiments, the populations of transformed bacteria are created from <i>Veillonella parvula</i>. In some embodiments, the populations of transformed bacteria are created from Bradyrhizobiaceae U8776. In some embodiments, the populations of transformed bacteria are created from Carnobacterium AJ427446. In some embodiments, the populations of transformed bacteria are created from <i>Corynebacterium </i>AY581888. In some embodiments, the populations of transformed bacteria are created from <i>Corynebacterium </i>AF543288. In some embodiments, the populations of transformed bacteria are created from <i>Corynebacterium </i>X81872. In some embodiments, the populations of transformed bacteria are created from <i>Corynebacterium </i>X84253. In some embodiments, the populations of transformed bacteria are created from <i>Dermacoccus </i>AF409025. In some embodiments, the populations of transformed bacteria are created from <i>Finegoldia </i>AB109769. In some embodiments, the populations of transformed bacteria are created from <i>Haemophilus </i>AF224309. In some embodiments, the populations of transformed bacteria are created from <i>Methylobacterium </i>AY741717. <i>Neisseria </i>DQ409137.</div>
</li> <li> <para-num num="[0131]"> </para-num> <div class="description-line" id="p-0132" num="0131">Bacterial strains can be modified to avoid undesired reactions, inflammation, for methods to easily remove the bacteria, attenuate the bacteria, and for preventing the release of unwanted bacterial biomolecules that can be detrimental to the host.</div>
</li> <li> <para-num num="[0132]"> </para-num> <div class="description-line" id="p-0133" num="0132">In some bacteria there are toxic proteins that can be involved with lysis and inflammation. Bacteria for example can secrete toxins such as endotoxins and exotoxins. Endotoxins are cell associated substances that are structural components of the bacteria and are released from bacteria or from bacterial cells that are a result of lysis from the host defense mechanisms. Exotoxins are secreted by bacteria and can be small biomolecules, proteins, and can also be minimal peptides that act enzymatically. Several examples of bacterial toxins include but are not limited to hemolysin (<i>E. coli</i>), alpha toxin (<i>S. aureus</i>), leukocidin (<i>S. aureus</i>), CAMP factor (<i>Priopionibacterium</i>), hyaluronidase (<i>Priopionibacterium</i>), neuraminidase (<i>Priopionibacterium</i>), enterotoxin B (<i>S. aureus</i>), verotoxin (<i>E. coli</i>). For example <i>Propionibacterium acnes </i>naturally make the biomolecule CAMP Factor, which has a co-hemolytic activity with sphingomyelinase that can confer cytotoxicity to keratinocytes and macrophages. The CAMP Factor together with the acid sphigomyelinase from host cells can lead to lysis and inflammation in the host. In another example, hemolysis is a mechanism that is employed by many bacterial pathogens that work to degrade, invade host cells and to resist the hosts' immune system. <i>P. acnes </i>carries within its genome 5 different CAMP homologs (CAMP1, CAMP2, CAMP3, CAMP4 and CAMP5). Many other types of bacteria can also secrete cytotoxins. <i>Staphylococcus aureus</i>, for example, can produce a wide variety of virulence factors that include but are not limited to enzymes, toxins, superantigens, exfoliative toxins, alpha toxin, beta toxin, delta toxin, and several types of bicomponent toxins.</div>
</li> <li> <para-num num="[0133]"> </para-num> <div class="description-line" id="p-0134" num="0133">In some bacteria, lipase can be used for a beneficial effect. In some microbes, genes for lipases are not knocked out as lipases can be used to break down oils in subjects suffering from axillary odor or plantar odor. In some embodiments, <i>Propionibacterium </i>is engineered to release lipases to reduce sebaceous secretions.</div>
</li> <li> <para-num num="[0134]"> </para-num> <div class="description-line" id="p-0135" num="0134">In order to prevent detrimental effects from virulence factors, modifications can be performed to knock out specific genes of interest that are involved in the secretion of toxic factors of the bacteria to create conditional mutants. Conditional mutants can be described as having a wild type phenotype under certain permissive environmental conditions and a mutant phenotype under other restrictive conditions. In some embodiments, genes encoding toxic proteins can be mutated or knocked out to prevent nosocomial infections or disease symptoms in a host. In some embodiments, genes encoding enzymes that cause inflammation in a host can be mutated or knocked out to prevent manifestation of disease in a host. In some embodiments, genes encoding proteins involved in the synthesis of host viral factors can be mutated or knocked out in the bacteria. In some embodiments, the genes are for CAMP factor. In some embodiments the genes are for lipases. However, in some microbes, genes for lipases are not knocked out as lipases can be used to break down oils in subjects suffering from axillary odor or plantar odor. In some embodiments, <i>Propionibacterium </i>is engineered to release lipases to reduce sebaceous secretions. In some embodiments, the genes are for alpha toxin. In some embodiments, the genes are for beta toxin. In some embodiments the genes are for delta toxin. In some embodiments, the genes are for types of bicomponent toxins. In some embodiments, the genes are for hemolysins.</div>
</li> <li> <para-num num="[0135]"> </para-num> <div class="description-line" id="p-0136" num="0135">Several microbes do not require any growth factors and can synthesize essential purines, pyrimidines, amino acids and vitamins, in which they can start with a carbon source as part of their own metabolism. However, some types of microbes will require purines, pyrimidines, amino acids and vitamins in order to grow, and must be added in culture media to grow these microbes. By genetically modifying microbes so that they require a growth factor not needed by a wild type, a person skilled in the art can produce genetically modified “auxotrophs” or a mutant organism that has a nutritional requirement not shared by the parent organism.</div>
</li> <li> <para-num num="[0136]"> </para-num> <div class="description-line" id="p-0137" num="0136">Genetically modified microbes can be modified so that they are nutritional auxotrophs. In order to create a dependency on their surroundings for survival, one can control the amount of microbes are in a microenvironment, or one can eliminate a population of microbes by depleting the environment of a necessary nutrient. In several embodiments, the microbe comprises a genome with a mutation or knock out in a gene that codes for glutamine synthetase. In several embodiments, the microbe comprises a genome with a mutation or knock out in a gene that codes for asparagine synthetase. In several embodiments, the microbe comprises a genome with a mutation or knock out in a gene that codes for aspartokinase. In several embodiments, the microbe comprises a genome with a mutation or knock out in a gene that codes for aspartate semialdehyde dehydrogenase.</div>
</li> <li> <para-num num="[0137]"> </para-num> <div class="description-line" id="p-0138" num="0137">Toxins are also secreted by many fungi, which allow their successful colonization and infection of hosts under predisposing conditions. Secreted proteins can be involved in the virulence of several fungi species. For example, hydrolytic enzyme production, which also plays a role in the pathogenicity of bacteria and yeasts are commonly associated with virulence. Several toxins by fungi include but are not limited to secreted asparyl proteinases (Sap) (<i>C. albicans</i>), phospholipase B enzymes (<i>C. albicans</i>), lipases (<i>C. albicans</i>). In order to prevent infection by fungi, specific genes can be mutated or knocked out to prevent virulent factors from causing disease in a host.</div>
</li> <li> <para-num num="[0138]"> </para-num> <div class="description-line" id="p-0139" num="0138">Fungal cells can be genetically engineered to secrete biomolecules. In several embodiments described herein, populations of transformed fungal cells are created from members from a group consisting of the genus. In several embodiments described herein, populations of transformed bacteria is created from bacterial members from a group consisting of the genus <i>Candida</i>. In several embodiments described herein, populations of transformed bacteria is created from bacterial members from a group consisting of <i>Candida albicans, Candida glabrata, Candida tropicalis, Candida parapsilosis</i>, and <i>Candida krusei. </i> </div>
</li> <li> <para-num num="[0139]"> </para-num> <div class="description-line" id="p-0140" num="0139">In certain embodiments a composition comprises one or more genetically modified microbes. In some embodiments a composition comprises at least 1, at least 2, at least 3, at least 4, at least 5, at least 6, at least 7, at least 8, at least 9, or at least 10 genetically modified microbes. In some embodiments, a composition comprises 2, 3, 4, 5, 6, 7, 8, 9 or 10 or more genetically modified microbes. In certain embodiments a genetically modified microbe in a composition is configured to express one or more genes of interest. In certain embodiments a composition comprises a microbe that is genetically modified so that the expression of a pathogenic molecule (e.g., a pathogenic protein endogenous to the microbe) is substantially reduced or eliminated. In some embodiments a composition comprises a genetically modified microbe where one or more genes (e.g., endogenous essential genes) of the microbe are knocked out. In some embodiments a composition comprises a genetically modified microbe configured to express one or more essential genes under the direction of an inducible promoter.</div>
</li> <li> <para-num num="[0140]"> </para-num> <div class="description-line" id="p-0141" num="0140">In certain embodiments a composition comprises a pharmaceutical acceptable excipient or carrier. Pharmaceutical acceptable excipient or carriers contemplated for use herein are often not toxic to a genetically modified microbe. Pharmaceutical compositions for use in accordance with the invention thus can be formulated in a suitable manner using one or more physiologically acceptable carriers comprising excipients and auxiliaries, which can be used pharmaceutically. Proper formulation can depend upon the route of administration chosen. In particular, a suitable formulation, ingredient, excipient, the like or combinations thereof as listed in “Remington's Pharmaceutical Sciences,” Mack Publishing Co., Easton, Pa., 18th edition, 1990. can be used with a composition described herein. Compositions herein can be incorporated into or used with a suitable material described in Remington's.</div>
</li> <li> <para-num num="[0141]"> </para-num> <div class="description-line" id="p-0142" num="0141">As used herein the term “pharmaceutically acceptable” and “physiologically acceptable” mean a biologically acceptable formulation, gaseous, liquid or solid, or mixture thereof, which is suitable for one or more routes of administration, in vivo delivery or contact. Such formulations include solvents (aqueous or non-aqueous), solutions (aqueous or non-aqueous), emulsions (e.g., oil-in-water or water-in-oil), suspensions, syrups, elixirs, dispersion and suspension media, coatings, isotonic and absorption promoting or delaying agents, compatible with pharmaceutical administration or in vivo contact or delivery. Aqueous and non-aqueous solvents, solutions and suspensions may include suspending agents and thickening agents. Such pharmaceutically acceptable carriers include tablets (coated or uncoated), capsules (hard or soft), microbeads, powder, granules and crystals. Supplementary active compounds (e.g., preservatives, antibacterial, antiviral and antifungal agents) can also be incorporated into the compositions.</div>
</li> <li> <para-num num="[0142]"> </para-num> <div class="description-line" id="p-0143" num="0142">Genetically engineered microbes can be placed within excipients that are optimal for the survival of the bacteria or the fungus. In several embodiments described herein, compositions are described which comprise vehicles or excipients that help maintain the integrity and viability of the bacteria. Vehicles as described herein can refer to a substance of no therapeutic value that is used to convey an active medicine for administration. Pharmaceutical vehicle as described herein can refer to a carrier or inert medium used as a solvent in which the medicinally active agent is formulated and or administered. Vehicles can include polymeric micelles, liposomes, lipoprotein-based carriers, nano-particle carriers, dendrimers, and other vehicles for bacteria that are known to one skilled in the art. An ideal vehicle can be non-toxic, biocompatible, non-immunogenic, biodegradable, and can avoid recognition by the host's defense mechanisms. In several embodiments described herein, compositions are described which comprise vehicles or excipients that help maintain the integrity of the fungus. In some embodiments, the vehicles are pharmaceutical vehicles. In some embodiments, the pharmaceutical vehicles include pharmaceutical compositions. In some embodiments, the vehicle is polymeric micelles. In some embodiments, the vehicle is liposomes. In some embodiments, the vehicle is lipoprotein-based carriers. In some embodiments, the vehicle is nano-particle carriers. In some embodiments, the vehicle is dendrimers.</div>
</li> <li> <para-num num="[0143]"> </para-num> <div class="description-line" id="p-0144" num="0143">Pharmaceutical compositions can be formulated to be compatible with a particular route of administration. Thus, pharmaceutical compositions include carriers, diluents, or excipients suitable for administration by various routes.</div>
</li> <li> <para-num num="[0144]"> </para-num> <div class="description-line" id="p-0145" num="0144">In some embodiments, a composition is formulated, for example, as a topical (e.g., dermal) formulation. In some embodiments, a composition is formulated, for example, for topical administration to a mammal. A topical formulation may include, for example, a formulation such as a gel formulation, a cream formulation, a lotion formulation, a paste formulation, an ointment formulation, an oil formulation, and a foam formulation. The composition further may include, for example, an absorption emollient.</div>
</li> <li> <para-num num="[0145]"> </para-num> <div class="description-line" id="p-0146" num="0145">Additional examples of a composition can optionally be formulated to be delivered to the <i>mucosum</i>, or by inhalation, respiration, intranasal, oral, buccal, or sublingual.</div>
</li> <li> <para-num num="[0146]"> </para-num> <div class="description-line" id="p-0147" num="0146">Salts may be added. Non-limiting examples of salts include acetate, benzoate, besylate, bitartate, bromide, carbonate, chloride, citrate, edetate, edisylate, estolate, fumarate, gluceptate, gluconate, hydrobromide, hydrochloride, iodide, lactate, lactobionate, malate, maleate, mandelate, mesylate, methyl bromide, methyl sulphate, mucate, napsylate, nitrate, pamoate (embonate, phosphate, diphosphate, salicylate and disalicylate, stearate, succinate, sulphate, tartrate, tosylate, triethiodide, valerate, aluminium, benzathine, calcium, ethylene diamine, lysine, magnesium, megluminie, potassium, procaine, sodium, tromethyamine or zinc.</div>
</li> <li> <para-num num="[0147]"> </para-num> <div class="description-line" id="p-0148" num="0147">Chelating agents may be added. Non-limiting examples of chelating agents include ethylenediamine, ethylene glycol tetraacetic acid, 1,2-bis(o-aminophenoxy)ethane-N,N,N′,N′-tetraacetic acid, Penicillamine, Deferasirox, Deferiprone, Deferoxamine, 2,3-Disulfanylpropan-1-ol, Dexrazoxane, Iron(II,III) hexacyanoferrate(II,III), (R)-5-(1,2-dithiolan-3-yl)pentanoic acid, 2,3-Dimercapto-1-propanesulfonic acid, Dimercaptosuccinic acid, or diethylene triamine pentaacetic acid.</div>
</li> <li> <para-num num="[0148]"> </para-num> <div class="description-line" id="p-0149" num="0148">Buffering agents may be added to. Non-limiting examples of buffering agents include phosphate, citrate, acetate, borate, TAPS, bicine, tris, tricine, TAPSO, HEPES, TES, MOPS, PIPES, cacodylate, SSC, MES or succinic acid.</div>
</li> <li> <para-num num="[0149]"> </para-num> <div class="description-line" id="p-0150" num="0149">Cosolvents may be added. Non-limiting examples of cosolvents contain hydroxyl groups or other polar groups, for example, alcohols, such as isopropyl alcohol; glycols, such as propylene glycol, polyethyleneglycol, polypropylene glycol, glycol ether; glycerol; polyoxyethylene alcohols and polyoxyethylene fatty acid esters. Non-limiting examples of cosolvents contain hydroxyl groups or other polar groups, for example, alcohols, such as isopropyl alcohol; glycols, such as propylene glycol, polyethyleneglycol, polypropylene glycol, glycol ether; glycerol; polyoxyethylene alcohols and polyoxyethylene fatty acid esters.</div>
</li> <li> <para-num num="[0150]"> </para-num> <div class="description-line" id="p-0151" num="0150">Supplementary compounds (e.g., preservatives, antioxidants, antimicrobial agents including biocides and biostats such as antibacterial, antiviral and antifungal agents) can also be added. Pharmaceutical compositions may therefore include preservatives, anti-oxidants and antimicrobial agents.</div>
</li> <li> <para-num num="[0151]"> </para-num> <div class="description-line" id="p-0152" num="0151">Preservatives can be used to inhibit undesired microbial growth or increase stability of ingredients thereby prolonging the shelf life. Suitable preservatives are known in the art and include, for example, EDTA, EGTA, benzalkonium chloride or benzoic acid or benzoates, such as sodium benzoate. Antioxidants include, for example, ascorbic acid, vitamin A, vitamin E, tocopherols, and similar vitamins or provitamins.</div>
</li> <li> <para-num num="[0152]"> </para-num> <div class="description-line" id="p-0153" num="0152">In several embodiments herein, methods are described to preserve the genetically engineered bacteria cryogenically for future use. In several embodiments herein, methods are described to preserve the bacteria by freeze drying the genetically engineered bacteria for future use. In several embodiments herein, methods are described to preserve the genetically engineered fungus cryogenically for future use. In several embodiments herein, methods are described to preserve the genetically engineered fungus by freeze drying the bacteria for future use.</div>
</li> <li> <para-num num="[0153]"> </para-num> <div class="description-line" id="p-0154" num="0153">Particular non-limiting classes of anti-virals include reverse transcriptase inhibitors; protease inhibitors; thymidine kinase inhibitors; sugar or glycoprotein synthesis inhibitors; structural protein synthesis inhibitors; nucleoside analogues; and viral maturation inhibitors. Specific non-limiting examples of anti-virals include nevirapine, delavirdine, efavirenz, saquinavir, ritonavir, indinavir, nelfinavir, amprenavir, zidovudine (AZT), stavudine (d4T), larnivudine (3TC), didanosine (DDI), zalcitabine (ddC), abacavir, acyclovir, penciclovir, ribavirin, valacyclovir, ganciclovir, 1,-D-ribofuranosyl-1,2,4-triazole-3 carboxamide, 9-&gt;2-hydroxy-ethoxy methylguanine, adamantanamine, 5-iodo-2′-deoxyuridine, trifluorothymidine, interferon and adenine arabinoside.</div>
</li> <li> <para-num num="[0154]"> </para-num> <div class="description-line" id="p-0155" num="0154">Pharmaceutical formulations and delivery systems appropriate for the compositions and methods of the invention are known in the art (see, e.g., <i>Remington: The Science and Practice of Pharmacy </i>(2003) 20<sup>th </sup>ed., Mack Publishing Co., Easton, Pa.; <i>Remington's Pharmaceutical Sciences </i>(1990) 18<sup>th </sup>ed., Mack Publishing Co., Easton, Pa.; The Merck Index (1996) 12<sup>th </sup>ed., Merck Publishing Group, Whitehouse, N.J.; Pharmaceutical Principles of Solid Dosage Forms (1993), Technonic Publishing Co., Inc., Lancaster, Pa.; Ansel ad Soklosa, <i>Pharmaceutical Calculations </i>(2001) 11<sup>th </sup>ed., Lippincott Williams &amp; Wilkins, Baltimore, Md.; and Poznansky et al., <i>Drug Delivery Systems </i>(1980), R. L. Juliano, ed., Oxford, N.Y., pp. 253-315).</div>
</li> <li> <para-num num="[0155]"> </para-num> <div class="description-line" id="p-0156" num="0155">In certain embodiments a composition comprises a molecule configured to engage, either directly or indirectly, an inducible promoter. In some embodiments a molecule configured to engage an inducible promoter in configured to activate the inducible promoter (e.g., activate gene transcription/translation). In some embodiments a molecule configured to engage an inducible promoter is configured to repress activation of gene expression by the inducible promoter. In some embodiments a molecule configured to engage an inducible promoter comprises a compound, alcohol, nutrient, metal, amino acid (e.g., an amino acid such as a synthetic amino acid), sugar or sugar analog (e.g., lactose, arabinose, IPTG), peptide, or protein. In certain embodiments a composition comprises lactose, arabinose, IPTG, tryptophan, or tetracycline.</div>
</li> <li> <para-num num="[0156]"> </para-num> <div class="description-line" id="p-0157" num="0156">In certain embodiments herein, methods of making a nucleic acid for transformation or integration into a microbe genome are described. Methods can include providing a nucleic acid sequence encoding a peptide, and joining said nucleic acid sequence to a nucleic acid sequence encoding a regulatory element or a nucleic acid encoding a secretory peptide. In some embodiments, a nucleic acid comprises a sequence encoding a peptide wherein the peptide comprises an amino acid sequence of a secretory peptide. In some embodiments, a nucleic acid comprises signal sequence for secretion. In some embodiments, a nucleic acid comprises a sequence, or encodes a polypeptide sequence that is recognized by a cells secretory pathway. In some embodiments, a nucleic acid encodes a non-secreted protein (e.g., an enzyme) for production of a biomolecule of interest. In some embodiments, a signal sequence for secretion comprises the sequence SEQ ID NO: 212, SEQ ID NO: 213, SEQ ID NO: 214, SEQ ID NO: 215, or SEQ ID NO: 216.</div>
</li> <li> <para-num num="[0157]"> </para-num> <div class="description-line" id="p-0158" num="0157">Those skilled in the art will appreciate that gene expression levels are dependent on many factors, such as promoter sequences and regulatory elements. Another factor for maximal protein selection is adaptation of codons of the transcript gene to the typical codon usage of a host. As noted for most bacteria, a small subset of codons are recognized by tRNA species leading to translational selection and is important in the limitations of protein expression. In this aspect, many synthetic genes can be designed to increase their protein expression level. The design process of codon optimization can be to alter rare codons to codons known for protein expression efficiency. In some embodiments, codon selection is described, wherein codon selection can be performed by using algorithms that are known to those skilled in the art to create synthetic genetic transcripts optimized for high protein yield. Programs containing algorithms for codon optimization are known to those skilled in the art. Programs can include, for example, OptimumGene™, GeneGPS® algorithms, etc. Additionally synthetic codon optimized sequences can be obtained commercially for example from Integrated DNA Technologies and other commercially available DNA sequencing services. In some embodiments, peptides for secretion are described wherein the genes for the peptides for secretion are codon optimized for expression in humans. In some embodiments, peptides for secretion are described, wherein the genes for the complete gene transcript are codon optimized for expression bacteria, which can include gene transcripts a secretory peptide and other peptides that are known to increase the level of expression. In some embodiments, peptides are described, wherein genes for the peptide for secretion are optimized to have selected codons specifically for protein expression in bacterial and fungal cells.</div>
</li> <li> <para-num num="[0158]"> </para-num> <div class="description-line" id="p-0159" num="0158">The term “subject” includes but is not limited to a subject having or at risk of having a disorder which would benefit from contact with or administration of a genetically modified microbe as set forth herein. Subjects include mammalian animals (mammals), such as humans, a non-human primate (apes, gibbons, gorillas, chimpanzees, orangutans, macaques), a domestic or companion animal (dogs and cats), a farm animal (poultry such as chickens and ducks, horses, cows, goats, sheep, pigs), and experimental animals (mouse, rat, rabbit, guinea pig). Subjects include veterinary animals, as well as animal disease models, for example, mouse and other animal models of a skin disorder.</div>
</li> <li> <para-num num="[0159]"> </para-num> <div class="description-line" id="p-0160" num="0159">Invention genetically modified microbes and compositions comprising genetically modified microbes can be employed in various methods and uses and medicaments. Such methods and uses and medicaments include, for example, administration ex vivo and in vivo. In various embodiments, methods and uses and medicaments provided include methods and uses and medicaments for treatment of skin disorders.</div>
</li> <li> <para-num num="[0160]"> </para-num> <div class="description-line" id="p-0161" num="0160">Skin disorders can arise from genetic predisposition to skin disorders, as well as the disturbance of the natural flora or natural bacteria that can reside on the skin. Several types of skin disorders exist for example but are not limited to acne, actinic keratosis, alopecia areata, athlete's foot, onchomychosis, atopic dermatitis, osmidrosis, eczema, fungal infection of the nails, psoriasis, rosacea, slow wound healing, folliculitis, keratosis pilaris, perioral dermatitis, angiofibromas, cutaneous inflammation, cosmesis, aging damage, dyschromia, premature greying hair, and seborrhea. In several embodiments described herein, methods to treat skin disorders are described. In several embodiments, methods to treat rosacea are described. In several embodiments, methods to treat alopecia are described. In several embodiments, methods to treat onchomychosis are described. In several embodiments, methods to treat osmidrosis are described. In several embodiments, methods to treat slow wound healing are described. In several embodiments, methods to treat premature greying hair are described. In several embodiments, methods to treat cutaneous inflammation are described. In several embodiments, methods to treat dyschromia are described.</div>
</li> <li> <para-num num="[0161]"> </para-num> <div class="description-line" id="p-0162" num="0161">Acne or acne vulgaris, is a common skin disease that can affect the face, neck, chest, and back. It is characterized by the regions on the skin that has seborrhea. Regions of the skin that have acne, can also have comedones, papules, nodules, cysts, boils, cystic acne and pimples. Acne can be caused by hormones, such as testosterone and are can be affected by associated sebaceous glands. Causes for acne can be from hormonal, genetic and infectious effects. Management for acne can be the use of topical medications such as benzoyl peroxide, salicylic acid, and hormones. More common can be the use of antibiotics to treat acne, however, with the ability to develop bacterial resistance, many types of antibiotics are becoming less effective.</div>
</li> <li> <para-num num="[0162]"> </para-num> <div class="description-line" id="p-0163" num="0162"> <i>Propionibacterium acnes </i>(<i>P. acnes</i>) is an anaerobic bacterial species that is widely concluded to cause acne. <i>Staphylococcus aureus </i>a natural floral bacteria of the skin is also believed to be an opportunistic bacteria to infect the skin even though it is seen in both healthy and infected skin. However it has been shown in studies that <i>P. acnes </i>as well as <i>S. aureus </i>have been developing antibiotic resistance, thereby increasing the need to develop a new treatment for skin disorders that are normally treated with antibiotics. In several embodiments described herein, genetically modified bacteria that secrete biomolecule(s) are used to treat acne. In several embodiments, described herein, genetically modified fungus that secretes biomolecules are used to treat acne.</div>
</li> <li> <para-num num="[0163]"> </para-num> <div class="description-line" id="p-0164" num="0163">Rosacea as described herein, refer to a chronic condition characterized by facial erythema and sometimes pimples. Rosacea has four subtypes, three affecting the skin and the fourth affecting the eyes (ocular type). Earlier treatment was the use of topical steroids, however, treatment in the form of topical steroids can aggravate the condition with longtime usage. Therefore new methods of treatment have been currently researched. Rosacea affects both sexes, but is almost three times more common in women.</div>
</li> <li> <para-num num="[0164]"> </para-num> <div class="description-line" id="p-0165" num="0164">Rosacea begins as redness on the central face across the cheeks, nose, or forehead, but can also less commonly affect the neck, chest, ears, and scalp. In some cases, additional symptoms, such as semi-permanent redness, telangiectasia (dilation of superficial blood vessels on the face), red domed papules (small bumps) and pustules, red gritty eyes, burning and stinging sensations, and in some advanced cases, a red lobulated nose, thinophyma, may develop. Triggers that cause episodes of flushing and blushing play a part in the development of rosacea. Exposure to temperature extremes can cause the face to become flushed as well as strenuous exercise, heat from sunlight, severe sunburn, stress, anxiety, cold wind, and moving to a warm or hot environment from a cold one such as heated shops and offices during the winter. There are also some food and drinks that can trigger flushing, including alcohol, food and beverages containing caffeine (especially, hot tea and coffee), foods high in histamines and spicy food.</div>
</li> <li> <para-num num="[0165]"> </para-num> <div class="description-line" id="p-0166" num="0165">Certain medications and topical irritants can quickly trigger rosacea. Some acne and wrinkle treatments that have been reported to cause rosacea include microdermabrasion and chemical peels, as well as high dosages of isotretinoin, benzoyl peroxide, and tretinoin. Steroid induced rosacea is the term given to rosacea caused by the use of topical or nasal steroids. These steroids are often prescribed for seborrheic dermatitis. Dosage should be slowly decreased and not immediately stopped to avoid a flare up. Intestinal flora can play a role in causing the disease.</div>
</li> <li> <para-num num="[0166]"> </para-num> <div class="description-line" id="p-0167" num="0166">Oral tetracycline antibiotics (tetracycline, doxycycline, minocycline) and topical antibiotics such as metronidazole are usually the first line of defense prescribed by doctors to relieve papules, pustules, inflammation and some redness.</div>
</li> <li> <para-num num="[0167]"> </para-num> <div class="description-line" id="p-0168" num="0167">Oral antibiotics can help to relieve symptoms of ocular rosacea. If papules and pustules persist, then sometimes isotretinoin can be prescribed. Isotretinoin has many side effects and is normally used to treat severe acne but in low dosages is proven to be effective against papulopustular and phymatous rosacea. Some individuals respond well to the topical application of sandalwood oil on the affected area, particularly in reducing the prevalence of pustules and erythema. The oral antibiotics can be a first line of defense against bacteria on the skin that can trigger the rosacea, however due to the rise of antibiotic resistance, new developments have been sought in order to treat both bacterial ailments of the skin as well as rosacea. In several embodiments, described herein, genetically modified microbes are used for treatment of rosacea.</div>
</li> <li> <para-num num="[0168]"> </para-num> <div class="description-line" id="p-0169" num="0168">Alopecia areata as described herein refer to a condition in which hair is lost from some or all areas of the body, usually from the scalp. Because it causes bald spots on the scalp, especially in the first stages, it is sometimes called spot baldness. In 1-2% of cases, the condition can spread to the entire scalp (alopecia totalis) or to the entire epidermis (alopecia universalis). Conditions resembling AA, and having a similar cause, occur also in other species.</div>
</li> <li> <para-num num="[0169]"> </para-num> <div class="description-line" id="p-0170" num="0169">The condition is thought to be a systemic autoimmune disorder in which the body attacks its own antigen hair follicles and suppresses or stops hair growth. T cell lymphocytes cluster around affected follicles, causing inflammation and subsequent hair loss. A few cases of babies being born with congenital AA have been reported, but these are not cases of autoimmune disease, because an infant is born without a definitely developed immune system. Endogenous retinoids metabolic defect is a key part of the pathogenesis of the AA. Also, some evidence indicates AA affects the part of the hair follicle associated with hair color. Hair that has turned gray cannot be affected. In some embodiments, genetically modified microbes are used for treatment of a immune disorder.</div>
</li> <li> <para-num num="[0170]"> </para-num> <div class="description-line" id="p-0171" num="0170">Onchomychosis refers to a fungal infection of the nail. The infection can be caused by pathogens of onychomycosis include dermatophytes, <i>Candida</i>, and nondermatophytic molds. Dermatophytes are the fungi most commonly responsible for onychomycosis in the temperate western countries; while <i>Candida </i>and nondermatophytic molds are more frequently involved in the tropics and subtropics with a hot and humid climate. Pathogens can include <i>Candida </i>and nondermatophytic molds, in particular members of the mold generation <i>Scytalidium </i>(<i>Neoscytalidium</i>), <i>Scopulariopsis</i>, and <i>Aspergillus. Candida </i>spp. mainly cause fingernail onychomycosis in people whose hands are often submerged in water. <i>Scytalidium </i>mainly affects people in the tropics, though it persists if they later move to areas of temperate climate. Treatments mostly include topical or antifungal medications such as terbinafine, itraconazole, and fluconazole.</div>
</li> <li> <para-num num="[0171]"> </para-num> <div class="description-line" id="p-0172" num="0171">Osmidrosis refers to body order, in which sebaceous and apocrine glands can pay a role. Medical conditions can be referred to as bromhidrosis, apocrine bromhidrosis, osmidrosis, ozochrotia, fetid sweat, and malodorous sweating. Osmidrosis or bromhidrosis is defined by a foul odor due to a water rich environment that is supportive of bacterial growth which is also caused by an abnormal increase in perspiration. In some embodiments, methods are used for the treatment of osmidrosis.</div>
</li> <li> <para-num num="[0172]"> </para-num> <div class="description-line" id="p-0173" num="0172">“Poor wound healing” can be caused by a poor immune system, diabetes mellitus, low human growth hormone, rheumatoid arthritis, poor circulation from vascular or arterial diseases, zinc deficiency, vitamin deficiency, lupus, etc. In several cases, poor wound healing can lead to opportunistic infections by the normal flora. In several embodiments, treatment for wound healing using genetically modified microbes are described.</div>
</li> <li> <para-num num="[0173]"> </para-num> <div class="description-line" id="p-0174" num="0173">“Cutaneous inflammation” and “Chronic cutaneous inflammation” can be hallmarked by macrophage infiltration of the dermal area. During an infection, the macrophage response is to mediate a chronic inflammation, which is seen in several inflammatory dermatoses which includes but is not limited to psoriasis, atopic dermatitis, and chronic contact dermatitis. In order to decrease the inflammatory response, treatments have been described in which elimination of the macrophage at the site of the inflammation is sought. One way is to block the activity of the effector molecules or the cytokines that are produced by the macrophage. Another way is to target the macrophage for apoptotic elimination locally during the inflammation by engineering an immunotoxin. Treatment can be for other skin disorders caused by cutaneous inflammation or chronic cutaneous inflammation such as cutaneous graft-versus-host disease, lichenoid, schlerodermatous, granuloma annulare, sarcoid, chronic contact dermatitis, psoriasis, UV skin injuries, atopic dermatitis, and cutaneous T-Cell lymphoma. However, most ways to eliminate the local response is time consuming. In this respect, having an engineered microbe population to eliminate the macrophage locally at the site of chronic cutaneous inflammation can provide a fast treatment for either cutaneous inflammation, or chronic cutaneous inflammation.</div>
</li> <li> <para-num num="[0174]"> </para-num> <div class="description-line" id="p-0175" num="0174">The invention provides kits including genetically modified microbes, such as a bacteria of the genus <i>Propionibacterium</i>, compositions and pharmaceutical formulations thereof, packaged into suitable packaging material. Kits can be used in various in vitro, ex vivo and in vivo methods and uses, for example a treatment method or use as disclosed herein.</div>
</li> <li> <para-num num="[0175]"> </para-num> <div class="description-line" id="p-0176" num="0175">A kit typically includes a label or packaging insert including a description of the components or instructions for use in vitro, in vivo, or ex vivo, of the components therein. A kit can contain a collection of such components, e.g., a genetically modified microbe, such as a bacteria of the genus <i>Propionibacterium </i>alone, or in combination with another therapeutically useful composition (e.g., an anti-inflammatory drug).</div>
</li> <li> <para-num num="[0176]"> </para-num> <div class="description-line" id="p-0177" num="0176">The term “packaging material” refers to a physical structure housing the components of the kit. The packaging material can maintain the components sterilely, and can be made of material commonly used for such purposes (e.g., paper, corrugated fiber, glass, plastic, foil, ampules, vials, tubes, etc.).</div>
</li> <li> <para-num num="[0177]"> </para-num> <div class="description-line" id="p-0178" num="0177">Kits of the invention can include labels or inserts. Labels or inserts include “printed matter,” e.g., paper or cardboard, or separate or affixed to a component, a kit or packing material (e.g., a box), or attached to an ampule, tube or vial containing a kit component. Labels or inserts can additionally include a computer readable medium, such as a disk (e.g., hard disk), optical disk such as CD- or DVD-ROM/RAM, DVD, MP3, magnetic tape, or an electrical storage media such as RAM and ROM or hybrids of these such as magnetic/optical storage media, FLASH media or memory type cards.</div>
</li> <li> <para-num num="[0178]"> </para-num> <div class="description-line" id="p-0179" num="0178">Labels or inserts can include identifying information of one or more components therein, dose amounts, clinical pharmacology of the active ingredient(s) including mechanism of action, pharmacokinetics and pharmacodynamics. Labels or inserts can include information identifying manufacturer information, lot numbers, manufacturer location and date.</div>
</li> <li> <para-num num="[0179]"> </para-num> <div class="description-line" id="p-0180" num="0179">Labels or inserts can include information on a condition, disorder, disease or symptom for which a kit component may be used. Labels or inserts can include instructions for the clinician or for a subject for using one or more of the kit components in a method, use, treatment protocol or therapeutic regimen. Instructions can include dosage amounts, frequency or duration, and instructions for practicing any of the methods and uses, treatment protocols or therapeutic regimes set forth herein.</div>
</li> <li> <para-num num="[0180]"> </para-num> <div class="description-line" id="p-0181" num="0180">Unless otherwise defined, all technical and scientific terms used herein have the same meaning as commonly understood by one of ordinary skill in the art to which this invention belongs. Although methods and materials similar or equivalent to those described herein can be used in the practice or testing of the present invention, suitable methods and materials are described herein.</div>
</li> <li> <para-num num="[0181]"> </para-num> <div class="description-line" id="p-0182" num="0181">All applications, publications, patents and other references, GenBank citations and ATCC citations cited herein are incorporated by reference in their entirety. In case of conflict, the specification, including definitions, will control.</div>
</li> <li> <para-num num="[0182]"> </para-num> <div class="description-line" id="p-0183" num="0182">With respect to the use of substantially any plural and/or singular terms herein, those having skill in the art can translate from the plural to the singular and/or from the singular to plural as is appropriate to the context and/or application. Thus, the singular forms “a,” “and,” and “the” include plural referents unless the context clearly indicates otherwise. The various singular/plural permutations can be expressly set forth herein for sake of clarity. Accordingly, reference to a “protein,” or an “gene,” for example, includes a plurality of proteins or genes.</div>
</li> <li> <para-num num="[0183]"> </para-num> <div class="description-line" id="p-0184" num="0183">It will be understood by those within the art that, in general, terms used herein, and especially in the appended claims (for example, bodies of the appended claims) are generally intended as “open” terms (for example, the term “including” should be interpreted as “including but not limited to,” the term “having” should be interpreted as “having at least,” the term “includes” should be interpreted as “includes but is not limited to,” etc.). It will be further understood by those within the art that if a specific number of an introduced claim recitation is intended, such an intent will be explicitly recited in the claim, and in the absence of such recitation no such intent is present. For example, as an aid to understanding, the following appended claims can contain usage of the introductory phrases “at least one” and “one or more” to introduce claim recitations. However, the use of such phrases should not be construed to imply that the introduction of a claim recitation by the indefinite articles “a” or “an” limits any particular claim containing such introduced claim recitation to embodiments containing only one such recitation, even when the same claim includes the introductory phrases “one or more” or “at least one” and indefinite articles such as “a” or “an” (for example, “a” and/or “an” should be interpreted to mean “at least one” or “one or more”); the same holds true for the use of definite articles used to introduce claim recitations. In addition, even if a specific number of an introduced claim recitation is explicitly recited, those skilled in the art will recognize that such recitation should be interpreted to mean at least the recited number (for example, the bare recitation of “two recitations,” without other modifiers, means at least two recitations, or two or more recitations). Furthermore, in those instances where a convention analogous to “at least one of A, B, and C, etc.” is used, in general such a construction is intended in the sense one having skill in the art would understand the convention (for example, “a system having at least one of A, B, and C” would include but not be limited to systems that have A alone, B alone, C alone, A and B together, A and C together, B and C together, and/or A, B, and C together, etc.). In those instances where a convention analogous to “at least one of A, B, or C, etc.” is used, in general such a construction is intended in the sense one having skill in the art would understand the convention (for example, “a system having at least one of A, B, or C” would include but not be limited to systems that have A alone, B alone, C alone, A and B together, A and C together, B and C together, and/or A, B, and C together, etc.). It will be further understood by those within the art that virtually any disjunctive word and/or phrase presenting two or more alternative terms, whether in the description, claims, or drawings, should be understood to contemplate the possibilities of including one of the terms, either of the terms, or both terms. For example, the phrase “A or B” will be understood to include the possibilities of “A” or “B” or “A and B.”</div>
</li> <li> <para-num num="[0184]"> </para-num> <div class="description-line" id="p-0185" num="0184">In addition, where features or aspects of the disclosure are described in terms of Markush groups, those skilled in the art will recognize that the disclosure is also thereby described in terms of any individual member or subgroup of members of the Markush group.</div>
</li> <li> <para-num num="[0185]"> </para-num> <div class="description-line" id="p-0186" num="0185">As used herein, numerical values are often presented in a range format throughout this document. The use of a range format is merely for convenience and brevity and should not be construed as an inflexible limitation on the scope of the invention. Accordingly, the use of a range expressly includes all possible subranges, all individual numerical values within that range, and all numerical values or numerical ranges include integers within such ranges and fractions of the values or the integers within ranges unless the context clearly indicates otherwise. This construction applies regardless of the breadth of the range and in all contexts throughout this patent document. Thus, all ranges disclosed herein also encompass any and all possible sub-ranges and combinations of sub-ranges thereof. Any listed range can be easily recognized as sufficiently describing and enabling the same range being broken down into at least equal halves, thirds, quarters, fifths, tenths, etc. As a non-limiting example, each range discussed herein can be readily broken down into a lower third, middle third and upper third, etc. As will also be understood by one skilled in the art all language such as “up to,” “at least,” “greater than,” “less than,” and the like include the number recited and refer to ranges which can be subsequently broken down into sub-ranges as discussed above. Finally, as will be understood by one skilled in the art, a range includes each individual member. Thus, for example, a group having 1-3 articles refers to groups having 1, 2, or 3 articles. Similarly, a group having 1-5 articles refers to groups having 1, 2, 3, 4, or 5 articles, and so forth.</div>
</li> <li> <para-num num="[0186]"> </para-num> <div class="description-line" id="p-0187" num="0186">While various aspects and embodiments have been disclosed herein, other aspects and embodiments will be apparent to those skilled in the art. The various aspects and embodiments disclosed herein are for purposes of illustration and are not intended to be limiting, with the true scope and spirit being indicated by the following claims.</div>
</li> <li> <para-num num="[0187]"> </para-num> <div class="description-line" id="p-0188" num="0187">One skilled in the art will appreciate that, for this and other processes and methods disclosed herein, the functions performed in the processes and methods can be implemented in differing order. Furthermore, the outlined steps and operations are only provided as examples, and some of the steps and operations can be optional, combined into fewer steps and operations, or expanded into additional steps and operations without detracting from the essence of the disclosed embodiments.</div>
</li> <li> <para-num num="[0188]"> </para-num> <div class="description-line" id="p-0189" num="0188">One skilled in the art will appreciate that, for this and other processes and methods disclosed herein, the functions performed in the processes and methods can be implemented in differing order. Furthermore, the outlined steps and operations are only provided as examples, and some of the steps and operations can be optional, combined into fewer steps and operations, or expanded into additional steps and operations without detracting from the essence of the disclosed embodiments.</div>
</li> <li> <para-num num="[0189]"> </para-num> <div class="description-line" id="p-0190" num="0189">The examples set forth below illustrate certain embodiments and do not limit the technology.</div>
</li> <heading id="h-0007">EXAMPLES</heading>
<heading id="h-0008">Example 1. Isolation of the RT6 <i>P. acnes </i>Strain</heading>
<li> <para-num num="[0190]"> </para-num> <div class="description-line" id="p-0191" num="0190">E-swabs (Fisher) were used to collect bacteria from the human skin as follows: volunteer subjects were asked to wash their faces with soap and water and clean it with ethanol wipes before they applied the E-swab over their cheeks and forehead. The applicator was next transferred to a tube with the transfer medium. The tube was vortexed to release the bacteria into the medium. A 100-fold dilution of the transfer medium was plated onto RIC plates. After a week of anaerobic growth, colonies were observed on each plate. These colonies were grown in the BHI medium until they reached the appropriate density for DNA isolation. Genomic DNA was prepared using a commercial kit (Epicentre) and was PCR amplified using <i>P. acnes</i>-specific 16S rDNA primers Pas9/Pas11 (Table 1). The PCR fragments were gel purified and sequenced using the primer 16SFseq. The sequences were aligned to the 16S rDNA gene (NC_017550, SEQ ID NO:290) from ATTCC 11828 (type II) and ATCC 6919 (type I). The colonies that showed a C to T conversion at nucleotide 1315 were assigned as type II RT6. Colonies were also tested for the RecA gene to confirm they belonged to a type II strain. After PCR amplification with primers RecAF/RecAR, the fragments were gel purified and sequenced with primers RecAFseq and RecARseq. A combination of sequences obtained with both primers covered over 90% of the RecA gene. The sequenced RecA fragments were aligned to the RecA gene from ATCC 11828 and ATCC 6919 strains for a reference sequence.</div>
</li> <li> <div class="description-line" id="p-0192" num="0000">
<tables id="TABLE-US-00001" num="00001">
<patent-tables colsep="0" frame="none" rowsep="0" tabstyle="monospace">
<table align="left" class="description-table" cols="1" colsep="0" rowsep="0" width="100%">
<thead>
<tr class="description-tr">
<td class="description-td" colspan="1" nameend="1" namest="1" rowsep="1">TABLE 1</td>
</tr>
</thead>
<tbody><tr class="description-tr">
<td align="center" class="description-td" colspan="1" nameend="1" namest="1" rowsep="1"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">Primers used for isolation of the RT6 <i>P</i>. <i>acnes</i> </td>
</tr>
<tr class="description-tr">
<td class="description-td">strain</td>
</tr>
</tbody></table>
<table align="left" class="description-table" cols="3" colsep="0" rowsep="0" width="100%">
<tbody><tr class="description-tr">
<td class="description-td">Name</td>
<td class="description-td">Sequence</td>
<td class="description-td">Comment</td>
</tr>
<tr class="description-tr">
<td align="center" class="description-td" colspan="3" nameend="3" namest="1" rowsep="1"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">PAS9F</td>
<td class="description-td">CCCTGCTTTTGTGGGGTG</td>
<td class="description-td">16SrDNA</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">(SEQ ID NO: 217)</td>
<td class="description-td">upstream</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">PAS11R</td>
<td class="description-td">CGACCCCAAAAGAGGGAC</td>
<td class="description-td">16SrDNA</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">(SEQ ID NO: 218)</td>
<td class="description-td">downstream</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">16SFseq</td>
<td class="description-td">ATCGCGTCGGAAGTGTAATC</td>
<td class="description-td">Sequencing</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">(SEQ ID NO: 219)</td>
<td class="description-td">primer</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">RecAF</td>
<td class="description-td">AGCTCGGTGGGGTTCTCTCATC</td>
<td class="description-td">RecA upstream</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">(SEQ ID NO: 220)</td>
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">RecAR</td>
<td class="description-td">GCTTCCTCATACCACTGGTCATC</td>
<td class="description-td">RecA downstream</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">(SEQ ID NO: 221)</td>
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">RecAFseq</td>
<td class="description-td">GGTACCACTGCCATCTTCATTA</td>
<td class="description-td">Sequencing</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">(SEQ ID NO: 222)</td>
<td class="description-td">upstream</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">RecARseq</td>
<td class="description-td">CACCAGCAGGGAATCTGTATC</td>
<td class="description-td">Sequencing</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">(SEQ ID NO: 223)</td>
<td class="description-td">downstream</td>
</tr>
<tr class="description-tr">
<td align="center" class="description-td" colspan="3" nameend="3" namest="1" rowsep="1"> </td>
</tr>
</tbody></table>
</patent-tables>
</tables>
</div>
</li> <heading id="h-0009">Example 2. Molecular Cloning</heading>
<li> <para-num num="[0191]"> </para-num> <div class="description-line" id="p-0193" num="0191">PCR amplification was performed using Q5 high fidelity DNA polymerase (New England Biolab) and PCR machine (Eppendorf) following manufacturer's recommendations. For the gene assembly reactions, Gibson assembly method (NEBuilder HiFi DNA Assembly Cloning Kit, New England Biolab) and Golden Gate assembly method (NEB Golden Gate assembly Kit, New England Biolab) were used following manufacturer's recommendations.</div>
</li> <heading id="h-0010">Example 3. Isolation of Genomic DNA</heading>
<li> <para-num num="[0192]"> </para-num> <div class="description-line" id="p-0194" num="0192">Total genomic DNA was isolated from the bacterial strains by phenol-chloroform extraction according to the procedure described by Rhee (2011) with minor changes. <i>P. acnes </i>strains were cultured in brain heart infusion broth (BHI) at 37° C. until late log phase. After harvest the cells, the cell pellet was resuspended with 10 mM Tris-Cl pH 8.0, 10 mM Na<sub>2</sub>EDTA and lysozyme (final concentration=1 mg/mL) was added. After incubation at 37° C. for 20 min., 10% of SDS was added (final concentration=1.4%) and incubated on ice for 10 min. The lysate was extracted 3 times with equal volume of phenol-chloroform mixture (25:24:1 of phenol, chloroform and isoamyl alcohol), and then the DNA was precipitated with ethanol and dried.</div>
</li> <heading id="h-0011">Example 4. Electroporation of <i>P. acnes </i> </heading>
<li> <para-num num="[0193]"> </para-num> <div class="description-line" id="p-0195" num="0193"> <i>P. acnes </i>was transformed according to the procedure described by Cheong (2008) and Rhee et al. (2007). Cells were grown in 9 mL of BHI with Oxyrase for broth (Oxyrase) in a 13×100 screw cap tube until the OD 600 nm reached about 0.5. Cells were collected by centrifugation (4° C.; 4300 g; 10 min) and washed with ice-cold SG medium (glycerol, 10%; sucrose, 0.5 M) three times. Cells were resuspended in SG medium (about 5×109-1010 CFU/ml). These electro-competent cells were used immediately. Seventy-five microliters of cell suspension was mixed with DNA and transferred to chilled electroporation cuvette (1 mm gap). The electroporation conditions using a Bio-Rad electroporator were 1.5 KV, 25 μF and 600Ω. The time constant was set between 8.5 and 10 ms. After electroporation, cells were transferred to 2 ml of pre-warmed (37° C.) BHI with Oxyrase for broth. These cells were incubated at 37° C. for 10 hrs. Cells were collected by centrifugation (2000×g, 10 min.) at room temperature and spread on reinforced clostridial agar plate with antibiotics. Plates were incubated at 37° C. at anaerobic condition.</div>
</li> <heading id="h-0012">Example 5. Construction of the Growth Arrest Strain of <i>Bacillus subtilis </i> </heading>
<li> <para-num num="[0194]"> </para-num> <div class="description-line" id="p-0196" num="0194">The truncated dnaA gene of <i>B. subtilis </i>was amplified by PCR with the genome DNA of <i>B. subtilis </i>strain 168 as a template and the primers (d-B_F:aggaaggaTCCATGGAAAATATATTAGACCTG (SEQ ID NO: 224), d-B_R:ctatgaccatgattacgCGCATGAATAACGGTCGTATG (SEQ ID NO: 225)). The DNA region with chloramphenicol gene and IPTG inducible promoter was amplified by PCR with primers (125_F:gcctgcaggtcgactTTAAGTTATTGGTATGACTGGTTTTAAG (SEQ ID NO: 226), 125_R:tccatggaTCCTTCCTCCTTTAATTGG (SEQ ID NO: 227)). These PCR products were assembled by NEBuilder® HiFi DNA Assembly Cloning Kit. The product was digested by HindIII and the 3,257 bp fragment was self-ligated. The self-ligated DNA was transformed into <i>B. subtilis </i>strain and selected the transformants on LB with Cm (5 μg/mL) and the different concentration of IPTG (0, 0.05, 0.1 and 0.25 mM).</div>
</li> <heading id="h-0013">Example 6. Construction of the Growth Arrest Strains of <i>P. acnes </i> </heading>
<li> <para-num num="[0195]"> </para-num> <div class="description-line" id="p-0197" num="0195">Arabinose inducible fts operon. The arabinose inducible promoter and regulator of <i>B. subtilis </i>was truncated was amplified by PCR with the genome DNA of <i>B. subtilis </i>strain 168 as a template and the primers (araRE F: TGCAGTTCTAGACGACACAGGCTGACGAAATTA (SEQ ID NO: 228), araRE R: CGTAGCGAATTCCATTTCCCTGCCCTCCCGAA (SEQ ID NO: 229)). The 1468 bp of PCR product was ligated into pUC18 hydrolyzed by XbaI and EcoRI. The truncated fts operon of <i>P. acnes </i>was amplified by PCR with the genome DNA of <i>P. acnes </i>strain ATCC 11828 as a template and the primers (ftzope F: CTGACTGAATTCGCTTCCCAACGGGGCCGTTT (SEQ ID NO: 230), ftzope R: GACTGCGAATTCCTGAAGAGCCGTCACCGACA (SEQ ID NO: 231)). The 1332 bp of PCR product hydrolyzed by EcoRI, was ligated into pUC18 with arabinose promoter also hydrolyzed by EcoRI. After insertion of the 1236 bp of DNA with erythromycin gene, this plasmid was introduced into <i>P. acnes </i>strain and transformants were selected on modified reinforced clostridial medium with L-arabinose (1.5% w/v).</div>
</li> <heading id="h-0014">Example 7. Lactose Inducible dnaA Gene</heading>
<li> <para-num num="[0196]"> </para-num> <div class="description-line" id="p-0198" num="0196">The β-galactose promoter region and truncated dnaA gene of <i>P. acnes </i>were amplified by PCR with the genome DNA of <i>P. acnes </i>strain ATCC11828 as a template and the primers (0410-1 F: GGCTTCTGGTCTCGAGTGTTGTGGAACGACAACA (SEQ ID NO: 232), 0410-1 R: GGCTTCTGGTCTCGATGGCACCAACCTTAGAGAG (SEQ ID NO: 233), 0410-2 F: GGCTTCTGGTCTCGCCATGTCCGACACACCGTTC (SEQ ID NO: 234), 0410-2 R: GGCTTCTGGTCTCGGGTAGAGACTGGGTAGAGACG (SEQ ID NO: 235)). These PCR products and DNA fragment with antibiotic genes, erythromycin and chloramphenicol genes were ligated into pUC18 by Golden Gate assembly method (NEB Golden Gate assembly Kit, New England Biolab). This plasmid was introduced into <i>P. acnes </i>strain and transformants were selected on modified reinforced clostridial medium with lactose (1.0% w/v).</div>
</li> <li> <div class="description-line" id="p-0199" num="0000">
<tables id="TABLE-US-00002" num="00002">
<patent-tables colsep="0" frame="none" rowsep="0" tabstyle="monospace">
<table align="left" class="description-table" cols="1" colsep="0" rowsep="0" width="100%">
<thead>
<tr class="description-tr">
<td class="description-td" colspan="1" nameend="1" namest="1" rowsep="1">TABLE 2</td>
</tr>
</thead>
<tbody><tr class="description-tr">
<td align="center" class="description-td" colspan="1" nameend="1" namest="1" rowsep="1"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">List of primers</td>
</tr>
</tbody></table>
<table align="left" class="description-table" cols="3" colsep="0" rowsep="0" width="100%">
<tbody><tr class="description-tr">
<td class="description-td">Target</td>
<td class="description-td"> </td>
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">gene</td>
<td class="description-td">Name</td>
<td class="description-td">Sequences (5′-3′)</td>
</tr>
<tr class="description-tr">
<td align="center" class="description-td" colspan="3" nameend="3" namest="1" rowsep="1"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">IL10</td>
<td class="description-td">I10 Ec F</td>
<td class="description-td">CGTCACCATATGCACTCCTCCGCTCTGCT</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">(SEQ ID NO: 236)</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">I10 Ec R</td>
<td class="description-td">GTAGCTGGATCCTCAGTTGCGGATCTTCATGG</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">(SEQ ID NO: 237)</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">EGF</td>
<td class="description-td">EgcEc F</td>
<td class="description-td">CGATGCCATATGAACTCCGACTCCGAGTGT</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">(SEQ ID NO: 238)</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">EgfEc R</td>
<td class="description-td">GTTCACGGATCCTCAGCGCAGCTCCCACCACT</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">(SEQ ID NO: 239)</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">GH</td>
<td class="description-td">GHEc F</td>
<td class="description-td">CGTGACCATATGTTCCCGACCATCCCGCT</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">(SEQ ID NO: 240)</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">GHEc R</td>
<td class="description-td">GTGACTGGATCCTCAGAAACCGCAGGAGCCCT</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">(SEQ ID NO: 241)</td>
</tr>
<tr class="description-tr">
<td align="center" class="description-td" colspan="3" nameend="3" namest="1" rowsep="1"> </td>
</tr>
</tbody></table>
</patent-tables>
</tables>
</div>
</li> <heading id="h-0015">Example 8. Construction of CAMPII Mutant <i>P. acnes </i>Strain</heading>
<li> <para-num num="[0197]"> </para-num> <div class="description-line" id="p-0200" num="0197">From the start codon of CAMPII gene of <i>P. acnes, </i>400 bp of truncated ORF was amplified by PCR with genome DNA of <i>P. acnes </i>ATCC 11828 strain as templates and the primers (0422NB-1 F: AGTAAACTTGGTCTGACATGAAGAAGACCCATCTTG (SEQ ID NO: 242), 0422NB-1 R: TATATATATTTATTATCCGATGGACCTTGTTTTGGAGAG (SEQ ID NO: 243)). The 438 bp of PCR product and DNA fragment with erythromycin and chloramphenicol resistance genes were ligated with pUC18 by NEBuilder HiFi DNA Assembly Cloning Kit and transformants of this ligation product were selected on LB with amp (100 μg/mL) and cm (20 mg/mL) plate. After transformation into <i>E. coli </i>dcm-dam-strain, the plasmid was introduced into <i>P. acnes </i>and selected anaerobically on reinforced clostridial medium with erm (5 μg/mL) or cm (5 μg/mL) plates. The CAMPII mutant colonies were checked by PCR.</div>
</li> <heading id="h-0016">Example 9. Construction of Markerless Mutant <i>P. acnes </i>Strain (FIG. <b>9</b>)</heading>
<li> <para-num num="[0198]"> </para-num> <div class="description-line" id="p-0201" num="0198">The upstream and downstream regions of target gene were amplified by PCR. These regions should be bigger than 500 bp. These DNA fragments were ligated with the plasmid that had antibiotic marker and didn't have replication origin for <i>P. acnes</i>. This suicide vector was introduced into <i>P. acnes </i>and the transformants were selected by antibiotics. After confirmation of these, cells were cultured in reinforced clostridial medium without antibiotics to allow the second homologous recombination. After streak on the plate, the markerless mutant colonies were screened by PCR.</div>
</li> <heading id="h-0017">Example 10. Construction of Lactose Inducible IL10 Secretion <i>P. acnes </i>Strain</heading>
<li> <para-num num="[0199]"> </para-num> <div class="description-line" id="p-0202" num="0199">The 1037 bp of β-galactose promoter region of <i>P. acnes </i>was amplified by PCR with the genome DNA of <i>P. acnes </i>strain ATCC11828 as a template and the primers (0505-1 F: TAAACTTGGTCTGACAGTGCGCACCGATGAGCGGCAGA (SEQ ID NO: 244), 0505-1_R: TTGCAAACATGGCACCAACCTTAGAGAGTCATG (SEQ ID NO: 245)). For secretion of IL10, the 188 bp of the signal peptide region was by PCR with the genome DNA of <i>B. subtilis </i>strain 168 as a template and the primers (0505-2_F: GGTTGGTGCCATGTTTGCAAAACGATTCAAAAC (SEQ ID NO: 246), 0505-2_R: AGGAGTGCATATGATAAATAGACATGGTTCCG (SEQ ID NO: 247)). The 568 bp of human IL10 ORF was amplified by PCR (0505-3_F: CTATTTATCATATGCACTCCTCCGCTCTG (SEQ ID NO: 248), 0505-3_R: TTATCCGATTCATGAGACTGTCAGTTGCGGATCTTCATGG (SEQ ID NO: 249)). These PCR products and DNA fragment with erythromycin and chloramphenicol resistance genes were ligated with pUC18 by NEBuilder HiFi DNA Assembly Cloning Kit and transformants of this ligation product were selected on LB with amp (100 μg/mL) and cm (20 mg/mL) plate. After transformation into <i>E. coli </i>dcm-dam-strain, the plasmid was introduced into <i>P. acnes </i>and selected anaerobically on reinforced clostridial medium with erm (5 μg/mL) or cm (5 μg/mL) plates. After check the transformant by PCR, induction by lactose and secretion of IL10 was analyzed by ELISA.</div>
</li> <heading id="h-0018">Example 11. MC/9 Cell Culture</heading>
<li> <para-num num="[0200]"> </para-num> <div class="description-line" id="p-0203" num="0200">The murine MC/9 cells were obtained from ATCC (CRL-8306) and were propagated in DMEM supplemented with 10% FBS (Gibco), 10% Rat T-STIM (BD), 2 mM glutamate, 0.05 mM 2-mercaptoethanol and pen/strep. Cells were maintained at a density of 2×10<sup>5 </sup>cells/ml in a 5% CO<sub>2 </sub>incubator at 37° C.</div>
</li> <heading id="h-0019">Example 12. Functional Assays for IL-10</heading>
<li> <para-num num="[0201]"> </para-num> <div class="description-line" id="p-0204" num="0201">The purified recombinant human IL-10 was quantified using a commercial ELISA for human IL-10 (Biolegend). The biological activity of the expressed IL-10 was tested by using a dose-dependent co-stimulation (with human IL-4) of MC/9 cell proliferation. Briefly, cells were spun down and washed twice with RPMI1640 (Gibco) to remove all traces of the Rat-T-STIM in the growth medium. Next, cells were re-suspended in DMEM supplemented with 10% FBS, 2 mM glutamate, pen/strep, 2-meracaptoethanal and 200 pg/ml human IL-4 (Peprotech) and were plated at a density of 20,000 cells per well in a 96-well plate. A commercial recombinant human IL-10 (Peprotech) was used to generate a standard curve. Cells were grown for 82 hours and cell proliferation was determined using an XTT assay (Biotium).</div>
</li> <heading id="h-0020">Example 13. Isolation of the R6 Type II <i>P. acnes </i> </heading>
<li> <para-num num="[0202]"> </para-num> <div class="description-line" id="p-0205" num="0202">Isolated <i>P. acnes </i>strains were checked to determine what ribotypes they were, with the goal of isolating a Type II, RT6 <i>P. acnes</i>. From an alignment of the RecA gene from a type II and type I strains (<figref idrefs="DRAWINGS">FIG. 10</figref>), we can see 10 nucleotide differences. These variations are used to identify sub-types of <i>P. acnes</i>. The ATCC strain 11828 was used as a positive control for a type II <i>P. acnes. </i> </div>
</li> <li> <para-num num="[0203]"> </para-num> <div class="description-line" id="p-0206" num="0203">The RecA gene sequenced from the putative RT6 clones aligns with the region 720-1191 of the RecA gene from ATTCC11828 strain (<figref idrefs="DRAWINGS">FIG. 11</figref>). The alignment with the type I strain, NCTC 737, shows 5 bp variations indicating that the colonies are of type II strains.</div>
</li> <li> <para-num num="[0204]"> </para-num> <div class="description-line" id="p-0207" num="0204">Additionally, the RecA gene sequenced from the putative RT6 clones aligns with the region 64-325 of the RecA gene from ATTCC11828 strain (SEQ ID NO: 276). The alignment with the type I strain, NCTC 737 (SEQ ID NO: 277), shows 3 bp variations indicating that the colonies are of type II strains.</div>
</li> <li> <para-num num="[0205]"> </para-num> <div class="description-line" id="p-0208" num="0205">SNP sequences for ATCC11828 aligned with RT6 SNP sequence from literature and 9 isolates (C→T at 1315)(<figref idrefs="DRAWINGS">FIG. 12</figref>). Nine of the isolated strains from our studies match the RT6 SNP sequence as published (e.g., Fitz-Gibbon, S., et al., (2013) Invest. Dermatol., 133(9):2152-60).</div>
</li> <heading id="h-0021">Example 14. Knock-in of the Human IL-10, EGF and HGH Genes into RT6 <i>P. acnes </i>Strain</heading>
<li> <para-num num="[0206]"> </para-num> <div class="description-line" id="p-0209" num="0206">The genes for human IL-10, human EGF and human GH were all cloned into separate <i>P. acnes </i>strains under inducible LacZ promoters. Upon incubation of the cells with 0.1 mM IPTG we began to see expression of the respective genes and secretion into the growth media (<figref idrefs="DRAWINGS">FIG. 13</figref>).</div>
</li> <heading id="h-0022">Example 15. Growth Arrest of <i>P. acnes </i>by Introduction of Inducible Promotors to Housekeeping Genes</heading>
<li> <para-num num="[0207]"> </para-num> <div class="description-line" id="p-0210" num="0207">Growth of the <i>P. acnes </i>was only achievable upon induction by IPTG (Table 3). The same constructs are being made with an arabinose inducible promoter so that the human gene secretion and growth arrest can be controlled by separate nutrients.</div>
</li> <li> <div class="description-line" id="p-0211" num="0000">
<tables id="TABLE-US-00003" num="00003">
<patent-tables colsep="0" frame="none" rowsep="0">
<table align="left" class="description-table" cols="2" colsep="0" rowsep="0" width="100%">
<thead>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td" colspan="2" nameend="1" namest="offset" rowsep="1">TABLE 3</td>
</tr>
</thead>
<tbody><tr class="description-tr">
<td class="description-td"> </td>
<td align="center" class="description-td" colspan="2" nameend="1" namest="offset" rowsep="1"> </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">Incubation time</td>
</tr>
</tbody></table>
<table align="left" class="description-table" cols="5" colsep="0" rowsep="0" width="100%">
<tbody><tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">IPTG (mM)</td>
<td class="description-td">1 day</td>
<td class="description-td">2 day</td>
<td class="description-td">5 day</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td align="center" class="description-td" colspan="5" nameend="4" namest="offset" rowsep="1"> </td>
</tr>
</tbody></table>
<table align="left" class="description-table" cols="5" colsep="0" rowsep="0" width="100%">
<tbody><tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">0</td>
<td class="description-td">−</td>
<td class="description-td">−</td>
<td class="description-td">−</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">0.05</td>
<td class="description-td">−</td>
<td class="description-td">−</td>
<td class="description-td">−</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">0.10</td>
<td class="description-td">+/−*</td>
<td class="description-td">+*</td>
<td class="description-td">++*</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">0.25</td>
<td class="description-td">+/−*</td>
<td class="description-td">+/−*</td>
<td class="description-td">+*</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td align="center" class="description-td" colspan="5" nameend="4" namest="offset" rowsep="1"> </td>
</tr>
</tbody></table>
</patent-tables>
</tables>
</div>
</li> <heading id="h-0023">Example 16. Construction of CAMPII Mutant and IL-10 Expression Strain of <i>P. acnes </i> </heading>
<li> <para-num num="[0208]"> </para-num> <div class="description-line" id="p-0212" num="0208">The 1100 bp upstream region from CAMPII ORF of <i>P. acnes </i>(Primers Up-C F and Up-C R), and 400 bp of truncated CAMPII ORF (Primers CAMPII F and CAMPII R) were amplified by PCR with genome DNA of <i>P. acnes </i>ATCC 11828 strain. For secretion of IL10, the 168 bp of the signal peptide region was by PCR with the genome DNA of <i>B. subtilis </i>strain 168 as a template (Primers Sig-C F and Sig-C R). The 568 bp of human IL10 ORF was amplified by PCR (Primers I10-C F and I10-C R). These PCR products and DNA fragment with erythromycin resistance genes were ligated with pUC19 by NEBuilder HiFi DNA Assembly Cloning Kit and transformants of this ligation product were selected on LB with amp (100 μg/mL) plate. After transformation into <i>E. coli </i>dcm-dam-strain, the plasmid was introduced into <i>P. acnes </i>and selected anaerobically on reinforced clostridial medium (RCM) with erm (5 μg/mL) plates. The transformant having IL-10 gene with CAMPII promoter and signal peptide was checked by PCR. For construction of CAMPII mutant strain, this transformant was cultured in RCM without erythromycin to allow the second homologous recombination. After streak on the plate, the CAMPII mutant colonies were screened by PCR.</div>
</li> <li> <div class="description-line" id="p-0213" num="0000">
<tables id="TABLE-US-00004" num="00004">
<patent-tables colsep="0" frame="none" pgwide="1" rowsep="0" tabstyle="monospace">
<table align="left" class="description-table" cols="1" colsep="0" rowsep="0" width="100%">
<thead>
<tr class="description-tr">
<td class="description-td" colspan="1" nameend="1" namest="1" rowsep="1">TABLE 4</td>
</tr>
</thead>
<tbody><tr class="description-tr">
<td align="center" class="description-td" colspan="1" nameend="1" namest="1" rowsep="1"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">List of primers</td>
</tr>
</tbody></table>
<table align="left" class="description-table" cols="2" colsep="0" rowsep="0" width="100%">
<tbody><tr class="description-tr">
<td class="description-td">Name</td>
<td class="description-td">Sequences (5′-3′)</td>
</tr>
<tr class="description-tr">
<td align="center" class="description-td" colspan="2" nameend="2" namest="1" rowsep="1"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">19-C F</td>
<td class="description-td">GAATTCGAGCTCGGTACC (SEQ ID NO: 250)</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">19-C R</td>
<td class="description-td">ACTGGCCGTCGTTTTACAAC (SEQ ID NO: 251)</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">Up-C F</td>
<td class="description-td">GTTGTAAAACGACGGCCAGTGAAGGCACCCATGAGCCT (SEQ ID</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">NO: 252)</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">Up-C R</td>
<td class="description-td">TTGCAAACATAAAGGTTCTCCGTTTATTGGTTG (SEQ ID NO: 253)</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">Sig-C F</td>
<td class="description-td">GAGAACCTTTATGTTTGCAAAACGATTCAAAAC (SEQ ID NO: 254)</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">Sig-C R</td>
<td class="description-td">AGGAGTGCATATGATAAATAGACATGGTTCCG (SEQ ID NO: 255)</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">I10-C F</td>
<td class="description-td">CTATTTATCATATGCACTCCTCCGCTCTG (SEQ ID NO: 256)</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">I10-C R</td>
<td class="description-td">GGGTCTTCTTCATTCAGTTGCGGATCTTCATGG (SEQ ID NO: 257)</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">CAMPII F</td>
<td class="description-td">CCGCAACTGAATGAAGAAGACCCATCTTG (SEQ ID NO: 258)</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">CAMPII R</td>
<td class="description-td">TAGAGACTGGGGACCTTGTTTTGGAGAG (SEQ ID NO: 259)</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">Erm-C F</td>
<td class="description-td">AACAAGGTCCCCAGTCTCTAGAATCGGATAATAAATATATATAAAC</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">(SEQ ID NO: 260)</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">Erm-C R</td>
<td class="description-td">CGGGTACCGAGCTCGAATTCCGATTATCTAGACAGCTCC (SEQ ID</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">NO: 261)</td>
</tr>
<tr class="description-tr">
<td align="center" class="description-td" colspan="2" nameend="2" namest="1" rowsep="1"> </td>
</tr>
</tbody></table>
</patent-tables>
</tables>
</div>
</li> <heading id="h-0024">Example 17. Construction of GAPDH Mutant and IL-10 Expression Strain of <i>P. acnes </i> </heading>
<li> <para-num num="[0209]"> </para-num> <div class="description-line" id="p-0214" num="0209">The 1000 bp upstream region from GAPDH ORF of <i>P. acnes </i>(Primers Up-G F and Up-G R), and 400 bp of truncated GAPDH ORF (Primers Gapdh F and gapdh R) were amplified by PCR with genome DNA of <i>P. acnes </i>ATCC 11828 strain. For secretion of IL10, the 168 bp of the signal peptide region was by PCR with the genome DNA of <i>B. subtilis </i>strain 168 as a template (Primers Sig-G F and Sig-G R). The 568 bp of human IL10 ORF was amplified by PCR (Primers I10-G F and I10-G R). These PCR products and DNA fragment with erythromycin resistance genes were ligated with pUC19 by NEBuilder HiFi DNA Assembly Cloning Kit and transformants of this ligation product were selected on LB with amp (100 μg/mL) plate. After transformation into <i>E. coli </i>dcm-dam-strain, the plasmid was introduced into <i>P. acnes </i>and selected anaerobically on reinforced clostridial medium (RCM) with erm (5 μg/mL) plates. The transformant having IL-10 gene with GAPDH promoter and signal peptide was checked by PCR. For construction of GAPDH mutant strain, this transformant was cultured in modified RCM (m-RCM) with lactate and without erythromycin to allow the second homologous recombination. After streak on the m-RCM with lactate and RCM plates, colonies that grew only on the m-RCM with lactate were picked and these mutants were checked by PCR.</div>
</li> <li> <div class="description-line" id="p-0215" num="0000">
<tables id="TABLE-US-00005" num="00005">
<patent-tables colsep="0" frame="none" pgwide="1" rowsep="0" tabstyle="monospace">
<table align="left" class="description-table" cols="1" colsep="0" rowsep="0" width="100%">
<thead>
<tr class="description-tr">
<td class="description-td" colspan="1" nameend="1" namest="1" rowsep="1">TABLE 5</td>
</tr>
</thead>
<tbody><tr class="description-tr">
<td align="center" class="description-td" colspan="1" nameend="1" namest="1" rowsep="1"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">List of primers</td>
</tr>
</tbody></table>
<table align="left" class="description-table" cols="2" colsep="0" rowsep="0" width="100%">
<tbody><tr class="description-tr">
<td class="description-td">Name</td>
<td class="description-td">Sequences (5′-3′)</td>
</tr>
<tr class="description-tr">
<td align="center" class="description-td" colspan="2" nameend="2" namest="1" rowsep="1"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">19-G F</td>
<td class="description-td">GAATTCGAGCTCGGTACC (SEQ ID NO: 262)</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">19-G R</td>
<td class="description-td">ACTGGCCGTCGTTTTACAAC (SEQ ID NO: 263)</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">Up-G F</td>
<td class="description-td">GTTGTAAAACGACGGCCAGTTCGTCAATGGCGCTGGAC (SEQ ID</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">NO: 264)</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">Up-G R</td>
<td class="description-td">TTGCAAACATTAAGGGATCTCCTCCAAATGAG (SEQ ID NO: 265)</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">Sig-G F</td>
<td class="description-td">AGATCCCTTAATGTTTGCAAAACGATTCAAAAC (SEQ ID NO: 266)</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">Sig-G R</td>
<td class="description-td">AGGAGTGCATATGATAAATAGACATGGTTCCG (SEQ ID NO: 267)</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">I10-G F</td>
<td class="description-td">CTATTTATCATATGCACTCCTCCGCTCTG (SEQ ID NO: 268)</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">I10-G R</td>
<td class="description-td">TGACGGTCATTCAGTTGCGGATCTTCATGG (SEQ ID NO: 269)</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">Gapdh F</td>
<td class="description-td">CCGCAACTGAATGACCGTCAAGGTTGGTATC (SEQ ID NO: 270)</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">Gapdh R</td>
<td class="description-td">TAGAGACTGGCCATAACGAAGGTGCCGTC (SEQ ID NO: 271)</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">Erm-G F</td>
<td class="description-td">TTCGTTATGGCCAGTCTCTAGAATCGGATAATAAATATATATAAAC</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">(SEQ ID NO: 272)</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">Erm-G R</td>
<td class="description-td">CGGGTACCGAGCTCGAATTCCGATTATCTAGACAGCTCC (SEQ ID</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">NO: 273)</td>
</tr>
<tr class="description-tr">
<td align="center" class="description-td" colspan="2" nameend="2" namest="1" rowsep="1"> </td>
</tr>
</tbody></table>
</patent-tables>
</tables>
</div>
</li> <heading id="h-0025">Example 18. References</heading>
<li> <div class="description-line" id="p-0216" num="0000">
</div> </li> <ul>
<li id="ul0001-0001" num="0210">Sörensen M, M. T. (2010). Mutagenesis of <i>Propionibacterium acnes </i>and analysis of two CAMP factor knock-out mutants. J Microbiol Methods, 211-216.</li>
<li id="ul0001-0002" num="0211">Rhee M S, Moritz B E, Xie G, Glavina Del Rio T, Dalin E, Tice H, Bruce D, Goodwin L, Chertkov O, Brettin T, Han C, Detter C, Pitluck S, Land M L, Patel M, Ou M, Harbrucker R, Ingram L O, Shanmugam K T. (2011). Complete Genome Sequence of a thermotolerant sporogenic lactic acid bacterium, <i>Bacillus coagulans </i>strain. Standards in Genomic Sciences 5,331-340.</li>
<li id="ul0001-0003" num="0212">Rhee M S, Kim J W, Qian Y L, Ingram L O, Shanmugam K T. (2007). Development of plasmid vector and electroporation condition for gene transfer in sporogenic lactic acid bacterium, <i>Bacillus coagulans</i>. Plasmid 58:13-22.</li>
<li id="ul0001-0004" num="0213">Cheong D E, Lee H I, So J S. (2008). Optimization of electrotransformation conditions for <i>Propionibacterium acnes. J Microbiol Methods. </i>72(1):38-41.</li>
<li id="ul0001-0005" num="0214">Rhee M S, Wei L, Sawhney N, Rice J D., St. John F, Hurlbert, J C, Preston, J F. (2014). Engineering the xylan utilization system in <i>Bacillus subtilis </i>for production of acidic xylooligosaccharides. Appl. Environ. Microbiol. 80 (3) 917-927.</li>
</ul>
<heading id="h-0026">Example 19. Representative Non-Limiting Embodiments</heading>
<li> <para-num num="[0215]"> </para-num> <div class="description-line" id="p-0217" num="0215">A1. A method of making a nucleic acid for controlled expression of a peptide for treatment comprising providing a first nucleic acid sequence encoding an operon sequence for controlled expression;
<br/>
joining the first nucleic acid sequence to a second nucleic acid sequence encoding a first peptide; and optimizing the nucleic acid for controlled expression of a peptide for treatment for protein expression.
<br/>
A2. The method of embodiment A1, wherein the operon is a prokaryotic operon.
<br/>
A3. The method of embodiment A2, wherein the operon is a lac operon.
<br/>
A4. The method of embodiment A2, wherein the operon is a Trp operon.
<br/>
A5. The method of embodiment A1, wherein the operon is a eukaryotic operon.
<br/>
A6. The method of any one of embodiments A1-A5, wherein the first peptide comprises a hyaluronan synthase, or portion thereof.
<br/>
A7. The method of embodiment A6, wherein the hyaluronan synthase, or portion thereof comprises a sequence of SEQ ID NO: 1, SEQ ID NO: 2, SEQ ID NO: 3, or SEQ ID NO: 4.
<br/>
A8. The method of any of embodiments A1-A5, wherein the first peptide comprises elastin, or a portion thereof.
<br/>
A9. The method of embodiment A8, wherein the elastin, or a portion thereof comprises a sequence of SEQ ID NO: 5, SEQ ID NO: 6, SEQ ID NO: 7, SEQ ID NO: 8, SEQ ID NO: 9, SEQ ID NO: 10, SEQ ID NO: 11, SEQ ID NO: 12, SEQ ID NO: 13, SEQ ID NO: 14, SEQ ID NO: 15, SEQ ID NO: 16, or SEQ ID NO: 17.
<br/>
A10. The method of any of embodiments A1-A5, wherein the first peptide comprises collagen, or a portion thereof.
<br/>
A11. The method of embodiment A10, wherein the collagen, or a portion thereof comprises a sequence of SEQ ID NO: 18, SEQ ID NO: 19, SEQ ID NO: 20, SEQ ID NO: 21, SEQ ID NO: 22, or SEQ ID NO: 23.
<br/>
A12. The method of any of embodiments A1-A5, wherein the first peptide comprises an anti-inflammatory, or a portion thereof.
<br/>
A13. The method of embodiment A12, wherein the anti-inflammatory is an interleukin, or a portion thereof.
<br/>
A14. The method of embodiment A12 or A13, wherein the anti-inflammatory, or portion thereof comprises a sequence of SEQ ID NO: 24, SEQ ID NO: 25, SEQ ID NO: 26, SEQ ID NO: 27, or SEQ ID NO: 28.
<br/>
A15. The method of any one of embodiments A1-A5, wherein the first peptide comprises a clotting factor, or portion thereof.
<br/>
A16. The method of embodiment A15, wherein the clotting factor, or portion thereof comprises the sequence of SEQ ID NO: 29, SEQ ID NO: 30, SEQ ID NO: 31, SEQ ID NO: 32, SEQ ID NO: 33, SEQ ID NO: 34, SEQ ID NO: 35, SEQ ID NO: 36, SEQ ID NO: 37, SEQ ID NO: 38, or SEQ ID NO: 39.
<br/>
A17. The method of any one of embodiments A1-A5, wherein the first peptide comprises an enzyme for melanin synthesis, or portion thereof.
<br/>
A18. The method of embodiment A17, wherein the enzyme for melanin synthesis, or portion thereof comprises the sequence of SEQ ID NO: 40, SEQ ID NO: 41, SEQ ID NO: 42, SEQ ID NO: 43, or SEQ ID NO: 44.
<br/>
A19. The method of any of embodiments A1-A5, wherein the first peptide comprises a hormone, or portion thereof.
<br/>
A20. The method of embodiment A19, wherein the hormone, or portion thereof comprises the sequence of SEQ ID NO: 45, SEQ ID NO: 46, SEQ ID NO: 47, SEQ ID NO: 48, or SEQ ID NO: 49.
<br/>
A21. The method of any of embodiments A1-A5, wherein the first peptide comprises a platelet basic protein, or portion thereof.
<br/>
A22. The method of embodiment A21, wherein the platelet basic protein, or portion thereof comprises the sequence of SEQ ID NO: 50, SEQ ID NO: 51, SEQ ID NO: 52, SEQ ID NO: 53, SEQ ID NO: 54, SEQ ID NO: 55, SEQ ID NO: 56, SEQ ID NO: 57, SEQ ID NO: 58, SEQ ID NO: 59, SEQ ID NO: 60, SEQ ID NO: 61, or SEQ ID NO: 62.
<br/>
A23. The method of any of embodiments A1-A5, wherein the first peptide comprises a transforming growth factor, or portion thereof.
<br/>
A24. The method of embodiment A23, wherein the transforming growth factor, or portion thereof comprises the sequence of SEQ ID NO: 63, SEQ ID NO: 64, SEQ ID NO: 65, SEQ ID NO: 66, SEQ ID NO: 67, SEQ ID NO: 68, or SEQ ID NO: 69.
<br/>
A25. The method of any one of embodiments A1-A5, wherein the first peptide comprises a hepatocyte growth factor, or portion thereof.
<br/>
A26. The method of embodiment A25, wherein the hepatocyte growth factor, or portion thereof comprises the sequence of SEQ ID NO: 70.
<br/>
A27. The method of any one of embodiments A1-A5, wherein the first peptide comprises a vascular endothelial growth factor, or portion thereof.
<br/>
A28. The method of embodiment A27, wherein the vascular endothelial growth factor, or portion thereof comprises the sequence of SEQ ID NO: 71, SEQ ID NO: 72, SEQ ID NO: 73, SEQ ID NO: 74, SEQ ID NO: 75, SEQ ID NO: 76, SEQ ID NO: 77, SEQ ID NO: 78, SEQ ID NO: 79, SEQ ID NO: 80, SEQ ID NO: 81, SEQ ID NO: 82, SEQ ID NO: 83, SEQ ID NO: 84, SEQ ID NO: 85, SEQ ID NO: 86, SEQ ID NO: 87, SEQ ID NO: 88, SEQ ID NO: 89, SEQ ID NO: 90, or SEQ ID NO: 91.
<br/>
A29. The method of any of embodiments A1-A5, wherein the first peptide comprises a placental growth factor, or portion thereof.
<br/>
A30. The method of embodiment A29, wherein the placental growth factor, or portion thereof comprises the sequence of SEQ ID NO: 92.
<br/>
A31. The method of any of embodiments A1-A5, wherein the first peptide comprises a platelet derived growth factor, or portion thereof.
<br/>
A32. The method of embodiment A31, wherein the platelet derived growth factor, or portion thereof comprises the sequence of or SEQ ID NO: 93, SEQ ID NO: 94, SEQ ID NO: 95, SEQ ID NO: 96, SEQ ID NO: 97, SEQ ID NO: 98, SEQ ID NO: 99, SEQ ID NO: 100, SEQ ID NO: 101, or SEQ ID NO: 102.
<br/>
A33. The method of any of embodiments A1-A5, wherein the first peptide comprises an epidermal growth factor, or portion thereof.
<br/>
A34. The method of embodiment A33, wherein the epidermal growth factor, or portion thereof comprises the sequence of SEQ ID NO: 103, SEQ ID NO: 104, SEQ ID NO: 105, or SEQ ID NO: 106.
<br/>
A35. The method of any one of embodiments A1-A5, wherein the first peptide comprises a fibroblast growth factor, or portion thereof.
<br/>
A36. The method of embodiment A35, wherein the fibroblast growth factor, or portion thereof comprises the sequence of SEQ ID NO: 107, SEQ ID NO: 108, SEQ ID NO: 109, SEQ ID NO: 110, SEQ ID NO: 111, SEQ ID NO: 112, SEQ ID NO: 113, SEQ ID NO: 114, SEQ ID NO: 115, SEQ ID NO: 116, SEQ ID NO: 117, SEQ ID NO: 118, SEQ ID NO: 119, SEQ ID NO: 120, SEQ ID NO: 121, SEQ ID NO: 122, SEQ ID NO: 123, SEQ ID NO: 124, SEQ ID NO: 125, SEQ ID NO: 126, SEQ ID NO: 127, SEQ ID NO: 128, SEQ ID NO: 129, SEQ ID NO: 130, SEQ ID NO: 131, SEQ ID NO: 132, SEQ ID NO: 133, SEQ ID NO: 134, SEQ ID NO: 135, SEQ ID NO: 136, SEQ ID NO: 137, SEQ ID NO: 138, SEQ ID NO: 139, SEQ ID NO: 140, SEQ ID NO: 141, SEQ ID NO: 142, SEQ ID NO: 143, or SEQ ID NO: 144.
<br/>
A37. The method of any one of embodiments A1-A5, wherein the first peptide comprises a DNA repair enzyme, or portion thereof.
<br/>
A38. The method of embodiment of A37, wherein the DNA repair enzyme or portion thereof is derived from a base excision repair enzyme.
<br/>
A39. The method of any of embodiment A37 or A38, wherein the DNA repair enzyme, or portion thereof comprises the amino acid sequence of SEQ ID NO: 145, SEQ ID NO: 146, SEQ ID NO: 147, SEQ ID NO: 148, SEQ ID NO: 149, SEQ ID NO: 150, SEQ ID NO: 151, SEQ ID NO: 152, SEQ ID NO: 153, SEQ ID NO: 154, or SEQ ID NO: 155.
<br/>
A40. The method of embodiment A37, wherein the DNA repair enzyme or portion thereof is derived from an enzyme for direct reversal of damage.
<br/>
A41. The method of embodiment A37 or A40, wherein the enzyme for direct reversal of damage comprises the amino acid sequence of SEQ ID NO: 156, SEQ ID NO: 157, or SEQ ID NO: 158.
<br/>
A42. The method of embodiment A37, wherein the DNA repair enzyme or portion thereof is derived from a mismatch excision repair enzyme.
<br/>
A43. The method of embodiment A37 or A42, wherein the mismatch excision repair enzyme comprises the amino acid sequence of SEQ ID NO: 159, SEQ ID NO: 160 SEQ ID NO: 161, SEQ ID NO: 162, SEQ ID NO: 163, SEQ ID NO: 164, SEQ ID NO: 165, SEQ ID NO: 166, SEQ ID NO: 167 or SEQ ID NO: 168.
<br/>
A44. The method of embodiment A37, wherein the DNA repair enzyme or portion thereof is derived from a nucleotide excision repair enzyme.
<br/>
A45. The method of embodiment A37 or A44, wherein the nucleotide excision repair enzyme comprises an amino acid sequence of SEQ ID NO: 169, SEQ ID NO: 170 SEQ ID NO: 171, SEQ ID NO: 172, SEQ ID NO: 173, SEQ ID NO: 174, SEQ ID NO: 175, SEQ ID NO: 176, SEQ ID NO: 177, SEQ ID NO: 178, SEQ ID NO: 179, SEQ ID NO: 180 SEQ ID NO: 181, SEQ ID NO: 182, SEQ ID NO: 183, SEQ ID NO: 184, SEQ ID NO: 185, SEQ ID NO: 186, SEQ ID NO: 187, SEQ ID NO: 188, SEQ ID NO: 189, SEQ ID NO: 190 SEQ ID NO: 191, SEQ ID NO: 192, SEQ ID NO: 193, SEQ ID NO: 194, SEQ ID NO: 195, SEQ ID NO: 196, or SEQ ID NO: 197.
<br/>
A46. The method of embodiment A37, wherein the DNA repair enzyme or portion thereof is derived from an editing or processing nuclease.
<br/>
A47. The method of embodiment A37 or A46, wherein the editing or processing nuclease comprises an amino acid sequence of SEQ ID NO: 198, SEQ ID NO: 199, SEQ ID NO: 200 SEQ ID NO: 201, SEQ ID NO: 202, SEQ ID NO: 203, SEQ ID NO: 204, or SEQ ID NO: 205.
<br/>
A48. The method of any of embodiments A1-A5, wherein the first peptide comprises a telomerase, or portion thereof.
<br/>
A49. The method of embodiment A48, wherein the telomerase or portion thereof comprises an amino acid sequence of SEQ ID NO: 206, SEQ ID NO: 207, SEQ ID NO: 208, or SEQ ID NO: 209.
<br/>
A50. The method of any one of embodiments A1-A5, wherein the first peptide comprises a protection of telomerase protein 1, or portion thereof.
<br/>
A51. The method of embodiment A50, wherein the protection of telomerase protein 1, or portion thereof comprises an amino acid sequence of SEQ ID NO: 210 or SEQ ID NO: 211.
<br/>
A52. The method of any one of embodiments A1-A5, A8-A16 or A19-A51, further comprising providing a third nucleic acid sequence encoding a second peptide and joining the third nucleic acid sequence to the second nucleic acid sequence encoding the first peptide at one end, wherein the third nucleic acid sequence lies between the first nucleic acid sequence encoding the operon sequence and the second nucleic acid sequence encoding the first peptide.
<br/>
A53. The method of any one of embodiments A1-A5, A8-A16 or A19-A51, further comprising providing a third nucleic acid sequence encoding a second peptide and joining the third nucleic acid sequence to the second nucleic acid sequence encoding the first peptide at one end, wherein the third nucleic acid sequence is at the opposite end of the second nucleic acid sequence from the first nucleic acid sequence encoding the operon sequence.
<br/>
A54. The method of embodiment A52 or A53, wherein the second peptide comprises SEQ ID NO: 5, SEQ ID NO: 6, SEQ ID NO: 7 SEQ ID NO: 8, SEQ ID NO: 9, SEQ ID NO: 10, SEQ ID NO: 11, SEQ ID NO: 12, SEQ ID NO: 13, SEQ ID NO: 14, SEQ ID NO: 15, SEQ ID NO: 16, SEQ ID NO: 17 SEQ ID NO: 18, SEQ ID NO: 19, SEQ ID NO: 20, SEQ ID NO: 21, SEQ ID NO: 22, SEQ ID NO: 23, SEQ ID NO: 24, SEQ ID NO: 25, SEQ ID NO: 26, SEQ ID NO: 27 SEQ ID NO: 28, SEQ ID NO: 29, SEQ ID NO: 30, SEQ ID NO: 31, SEQ ID NO: 32, SEQ ID NO: 33, SEQ ID NO: 34, SEQ ID NO: 35, SEQ ID NO: 36, SEQ ID NO: 37 SEQ ID NO: 38, SEQ ID NO: 39, SEQ ID NO: 45, SEQ ID NO: 46, SEQ ID NO: 47 SEQ ID NO: 48, SEQ ID NO: 49, SEQ ID NO: 50, SEQ ID NO: 51, SEQ ID NO: 52, SEQ ID NO: 53, SEQ ID NO: 54, SEQ ID NO: 55, SEQ ID NO: 56, SEQ ID NO: 57 SEQ ID NO: 58, SEQ ID NO: 59, SEQ ID NO: 60, SEQ ID NO: 61, SEQ ID NO: 62, SEQ ID NO: 63, SEQ ID NO: 64, SEQ ID NO: 65, SEQ ID NO: 66, SEQ ID NO: 67 SEQ ID NO: 68, SEQ ID NO: 69, SEQ ID NO: 70, SEQ ID NO: 71, SEQ ID NO: 72, SEQ ID NO: 73, SEQ ID NO: 74, SEQ ID NO: 75, SEQ ID NO: 76, SEQ ID NO: 77 SEQ ID NO: 78, SEQ ID NO: 79, SEQ ID NO: 80, SEQ ID NO: 81, SEQ ID NO: 82, SEQ ID NO: 83, SEQ ID NO: 84, SEQ ID NO: 85, SEQ ID NO: 86, SEQ ID NO: 87 SEQ ID NO: 88, SEQ ID NO: 89, SEQ ID NO: 90, SEQ ID NO: 91, SEQ ID NO: 92, SEQ ID NO: 93, SEQ ID NO: 94, SEQ ID NO: 95, SEQ ID NO: 96, SEQ ID NO: 97 SEQ ID NO: 98, SEQ ID NO: 99, SEQ ID NO: 100, SEQ ID NO: 101, SEQ ID NO: 102, SEQ ID NO: 103, SEQ ID NO: 104, SEQ ID NO: 105, SEQ ID NO: 106, SEQ ID NO: 107 SEQ ID NO: 108, SEQ ID NO: 109, SEQ ID NO: 110, SEQ ID NO: 111, SEQ ID NO: 112, SEQ ID NO: 113, SEQ ID NO: 114, SEQ ID NO: 115, SEQ ID NO: 116, SEQ ID NO: 117 SEQ ID NO: 118, SEQ ID NO: 119, SEQ ID NO: 120, SEQ ID NO: 121, SEQ ID NO: 122, SEQ ID NO: 123, SEQ ID NO: 124, SEQ ID NO: 125, SEQ ID NO: 126, SEQ ID NO: 127 SEQ ID NO: 128, SEQ ID NO: 129, SEQ ID NO: 130, SEQ ID NO: 131, SEQ ID NO: 132, SEQ ID NO: 133, SEQ ID NO: 134, SEQ ID NO: 135, SEQ ID NO: 136, SEQ ID NO: 137 SEQ ID NO: 138, SEQ ID NO: 139, SEQ ID NO: 140, SEQ ID NO: 141, SEQ ID NO: 142, SEQ ID NO: 143, SEQ ID NO: 144, SEQ ID NO: 145, SEQ ID NO: 146, SEQ ID NO: 147 SEQ ID NO: 148, SEQ ID NO: 149, SEQ ID NO: 150, SEQ ID NO: 151, SEQ ID NO: 152, SEQ ID NO: 153, SEQ ID NO: 154, SEQ ID NO: 155, SEQ ID NO: 156, SEQ ID NO: 157 SEQ ID NO: 158, SEQ ID NO: 159, SEQ ID NO: 160, SEQ ID NO: 161, SEQ ID NO: 162, SEQ ID NO: 163, SEQ ID NO: 164, SEQ ID NO: 165, SEQ ID NO: 166, SEQ ID NO: 167 SEQ ID NO: 168, SEQ ID NO: 169, SEQ ID NO: 170, SEQ ID NO: 171, SEQ ID NO: 172, SEQ ID NO: 173, SEQ ID NO: 174, SEQ ID NO: 175, SEQ ID NO: 176, SEQ ID NO: 177 SEQ ID NO: 178, SEQ ID NO: 179, SEQ ID NO: 180, SEQ ID NO: 181, SEQ ID NO: 182, SEQ ID NO: 183, SEQ ID NO: 184, SEQ ID NO: 185, SEQ ID NO: 106, SEQ ID NO: 187 SEQ ID NO: 188, SEQ ID NO: 189, SEQ ID NO: 190, SEQ ID NO: 191, SEQ ID NO: 192, SEQ ID NO: 193, SEQ ID NO: 194, SEQ ID NO: 195, SEQ ID NO: 196, SEQ ID NO: 197 SEQ ID NO: 198, SEQ ID NO: 199, SEQ ID NO: 200, SEQ ID NO: 201, SEQ ID NO: 202, SEQ ID NO: 203, SEQ ID NO: 204, SEQ ID NO: 205, SEQ ID NO: 206, SEQ ID NO: 207 SEQ ID NO: 208, SEQ ID NO: 209, SEQ ID NO: 210, or SEQ ID NO: 211 and wherein the second peptide does not comprise the amino acid sequence of the first peptide.
<br/>
A55. The method of any of embodiments A1-A5, A8-A16 or A19-A51, further comprising providing a nucleic acid sequence encoding a secretory peptide and joining said nucleic acid sequence encoding the secretory peptide to the nucleic acid sequence encoding the operon at one end, wherein the nucleic acid encoding the secretory peptide lies between operon sequence and second nucleic acid sequence encoding the first peptide.
<br/>
A56. The method of any of embodiments A1-A5, A8-A16 or A19-A51, further comprising providing a nucleic acid sequence encoding a secretory peptide and joining said nucleic acid sequence to the nucleic acid sequence encoding the second nucleic acid sequence encoding the first peptide at one end wherein the nucleic acid sequence encoding a secretory peptide is at an opposite end of the nucleic acid sequence encoding the operon.
<br/>
A57. The method of any of embodiments A52-A54, further comprising providing a nucleic acid sequence encoding a secretory peptide and joining the nucleic acid sequence encoding the secretory peptide to the nucleic acid sequence encoding the operon at one end, wherein the nucleic acid encoding the secretory peptide lies between the operon sequence and second nucleic acid sequence encoding the first peptide or the third nucleic acid sequence encoding the second peptide.
<br/>
A58. The method of any of embodiments A52-A54, further comprising providing a nucleic acid sequence encoding a secretory peptide and joining the nucleic acid sequence encoding the secretory peptide to the nucleic acid sequence encoding the second nucleic acid sequence encoding the first peptide or joining the nucleic acid sequence encoding the secretory peptide to the nucleic acid sequence encoding the third nucleic acid sequence encoding the second peptide at one end, wherein the nucleic acid sequence encoding the secretory peptide is at an opposite end of the operon sequence.
<br/>
A59. The method of any one of embodiments A55-A58, wherein the secretory peptide comprises the sequence SEQ ID NO: 212, SEQ ID NO: 213, SEQ ID NO: 214, SEQ ID NO: 215 or SEQ ID NO: 216.
<br/>
A60. The method of any one of embodiments A1-A59, wherein the optimizing is performed by computational methods.
<br/>
A61. A nucleic acid nucleic acid for controlled expression of a peptide for treatment comprising a first nucleic acid sequence encoding an operon sequence for controlled expression; a second nucleic acid sequence encoding a first peptide; and wherein the nucleic acid nucleic acid for controlled expression of a peptide for treatment is optimized for protein expression.
<br/>
A62. The nucleic acid of embodiment A61, wherein the operon is a prokaryotic operon.
<br/>
A63. The nucleic acid of embodiment A61 or A62, wherein the operon is a lac operon.
<br/>
A64. The nucleic acid of embodiments A61 or A62, wherein the operon is a Trp operon.
<br/>
A65. The nucleic acid of embodiment A61, wherein the operon is a eukaryotic operon.
<br/>
A66. The nucleic acid of any one of embodiments A61-A65, wherein the first peptide comprises a hyaluronan synthase, or portion thereof.
<br/>
A67. The nucleic acid of embodiment A66, wherein the first peptide comprises a sequence of SEQ ID NO: 1, SEQ ID NO: 2, SEQ ID NO: 3, or SEQ ID NO: 4.
<br/>
A68. The nucleic acid of any of embodiments A61-A65, wherein the first peptide comprises elastin, or a portion thereof.
<br/>
A69. The nucleic acid of embodiment A68, wherein the first peptide comprises a sequence of SEQ ID NO: 5, SEQ ID NO: 6, SEQ ID NO: 7, SEQ ID NO: 8, SEQ ID NO: 9, SEQ ID NO: 10, SEQ ID NO: 11, SEQ ID NO: 12, SEQ ID NO: 13, SEQ ID NO: 14, SEQ ID NO: 15, SEQ ID NO: 16, or SEQ ID NO: 17.
<br/>
A70. The nucleic acid of any one of embodiments A61-A65, wherein the first peptide comprises collagen, or a portion thereof.
<br/>
A71. The nucleic acid of embodiment A70, wherein the first peptide comprises a sequence of SEQ ID NO: 18, SEQ ID NO: 19, SEQ ID NO: 20, SEQ ID NO: 21, SEQ ID NO: 22, or SEQ ID NO: 23.
<br/>
A72. The nucleic acid of any one of embodiments A61-A65, wherein the first peptide comprises an anti-inflammatory, or a portion thereof.
<br/>
A73. The nucleic acid of embodiment A72, wherein the anti-inflammatory is an interleukin, or a portion thereof.
<br/>
A74. The nucleic acid of any one of embodiments A72 or A73, wherein the first peptide comprises a sequence of SEQ ID NO: 24, SEQ ID NO: 25, SEQ ID NO: 26, SEQ ID NO: 27, or SEQ ID NO: 28.
<br/>
A75. The nucleic acid of any one of embodiments A61-A65, wherein the first peptide comprises a clotting factor, or portion thereof.
<br/>
A76. The nucleic acid of embodiment A75, wherein the first peptide comprises the sequence of SEQ ID NO: 29, SEQ ID NO: 30, SEQ ID NO: 31, SEQ ID NO: 32, SEQ ID NO: 33, SEQ ID NO: 34, SEQ ID NO: 35, SEQ ID NO: 36, SEQ ID NO: 37, SEQ ID NO: 38, or SEQ ID NO: 39.
<br/>
A77. The nucleic acid of any one of embodiments A61-A65, wherein the first peptide comprises an enzyme for melanin synthesis, or portion thereof.
<br/>
A78. The nucleic acid of embodiment A77, wherein the first peptide comprises the sequence of SEQ ID NO: 40, SEQ ID NO: 41, SEQ ID NO: 42, SEQ ID NO: 43 or SEQ ID NO: 44.
<br/>
A79. The nucleic acid of any one of embodiments A61-A65, wherein the first peptide comprises a hormone, or portion thereof.
<br/>
A80. The nucleic acid of embodiment A79, wherein the first peptide for treatment comprises the sequence of SEQ ID NO: 45, SEQ ID NO: 46, SEQ ID NO: 47, SEQ ID NO: 48 or SEQ ID NO: 49.
<br/>
A81. The nucleic acid of any one of embodiments A61-A65, wherein the first peptide comprises a platelet basic protein, or portion thereof.
<br/>
A82. The nucleic acid of embodiment A81, wherein the first peptide comprises the sequence of SEQ ID NO: 50, SEQ ID NO: 51, SEQ ID NO: 52, SEQ ID NO: 53, SEQ ID NO: 54, SEQ ID NO: 55, SEQ ID NO: 56, SEQ ID NO: 57, SEQ ID NO: 58, SEQ ID NO: 59, SEQ ID NO: 60, SEQ ID NO: 61, or SEQ ID NO: 62.
<br/>
A83. The nucleic acid of any one of embodiments A61-A65, wherein the first peptide comprises a transforming growth factor, or portion thereof.
<br/>
A84. The nucleic acid of embodiment A83, wherein the first peptide comprises the sequence of SEQ ID NO: 63, SEQ ID NO: 64, SEQ ID NO: 65, SEQ ID NO: 66, SEQ ID NO: 67, SEQ ID NO: 68, or SEQ ID NO: 69.
<br/>
A85. The nucleic acid of any one of embodiments A61-A65, wherein the first peptide comprises a hepatocyte growth factor, or portion thereof.
<br/>
A86. The nucleic acid of embodiment A85, wherein the first peptide comprises the sequence of SEQ ID NO: 70.
<br/>
A87. The nucleic acid of any one of embodiments A61-A65, wherein the first peptide comprises a vascular endothelial growth factor, or portion thereof.
<br/>
A88. The nucleic acid of embodiment A87, wherein the first peptide comprises the sequence of SEQ ID NO: 71, SEQ ID NO: 72, SEQ ID NO: 73, SEQ ID NO: 74, SEQ ID NO: 75, SEQ ID NO: 76, SEQ ID NO: 77, SEQ ID NO: 78, SEQ ID NO: 79, SEQ ID NO: 80, SEQ ID NO: 81, SEQ ID NO: 82, SEQ ID NO: 83, SEQ ID NO: 84, SEQ ID NO: 85, SEQ ID NO: 86, SEQ ID NO: 87, SEQ ID NO: 88, SEQ ID NO: 89, SEQ ID NO: 90, or SEQ ID NO: 91.
<br/>
A89. The nucleic acid of any one of embodiments A61-A65, wherein the first peptide comprises a placental growth factor, or portion thereof.
<br/>
A90. The nucleic acid of embodiment A89, wherein the first peptide comprises the sequence of SEQ ID NO: 92.
<br/>
A91. The nucleic acid of any one of embodiments A61-A65, wherein the first peptide comprises a platelet derived growth factor, or portion thereof.
<br/>
A92. The nucleic acid of embodiment A91, wherein the first peptide comprises the sequence of SEQ ID NO: 93, SEQ ID NO: 94, SEQ ID NO: 95, SEQ ID NO: 96, SEQ ID NO: 97, SEQ ID NO: 98, SEQ ID NO: 99, SEQ ID NO: 100, SEQ ID NO: 101, or SEQ ID NO: 102.
<br/>
A93. The nucleic acid of any one of embodiments A61-A65, wherein the first peptide comprises an epidermal growth factor, or portion thereof.
<br/>
A94. The nucleic acid of embodiment A93, wherein the first peptide comprises the sequence of SEQ ID NO: 103, SEQ ID NO: 104, SEQ ID NO: 105, or SEQ ID NO: 106.
<br/>
A95. The nucleic acid of any one of embodiments A61-A65, wherein the first peptide comprises a fibroblast growth factor, or portion thereof.
<br/>
A96. The nucleic acid of embodiment A96, wherein the first peptide comprises the sequence of SEQ ID NO: 107, SEQ ID NO: 108, SEQ ID NO: 109, SEQ ID NO: 110, SEQ ID NO: 111, SEQ ID NO: 112, SEQ ID NO: 113, SEQ ID NO: 114, SEQ ID NO: 115, SEQ ID NO: 116, SEQ ID NO: 117, SEQ ID NO: 118, SEQ ID NO: 119, SEQ ID NO: 120, SEQ ID NO: 121, SEQ ID NO: 122, SEQ ID NO: 123, SEQ ID NO: 124, SEQ ID NO: 125, SEQ ID NO: 126, SEQ ID NO: 127, SEQ ID NO: 128, SEQ ID NO: 129, SEQ ID NO: 130, SEQ ID NO: 131, SEQ ID NO: 132, SEQ ID NO: 133, SEQ ID NO: 134, SEQ ID NO: 135, SEQ ID NO: 136, SEQ ID NO: 137, SEQ ID NO: 138, SEQ ID NO: 139, SEQ ID NO: 140, SEQ ID NO: 141, SEQ ID NO: 142, or SEQ ID NO: 144.
<br/>
A97. The nucleic acid of any one of embodiments A61-A65, wherein the first peptide comprises a DNA repair enzyme, or portion thereof.
<br/>
A98. The nucleic acid of embodiment of A97, wherein the DNA repair enzyme or portion thereof is derived from a base excision repair enzyme.
<br/>
A99. The nucleic acid of any of embodiment A97 or A98, wherein the DNA repair enzyme comprises the amino acid sequence of SEQ ID NO: 145, SEQ ID NO: 146, SEQ ID NO: 147, SEQ ID NO: 148, SEQ ID NO: 149, SEQ ID NO: 150, SEQ ID NO: 151, SEQ ID NO: 152, SEQ ID NO: 153, SEQ ID NO: 154, or SEQ ID NO: 155.
<br/>
A100. The nucleic acid of embodiment A97, wherein the DNA repair enzyme or portion thereof is derived from an enzyme for direct reversal of damage.
<br/>
A101. The nucleic acid of embodiment A97 or A100, wherein the DNA repair enzyme comprises the amino acid sequence of SEQ ID NO: 156, SEQ ID NO: 157, or SEQ ID NO: 158.
<br/>
A102. The nucleic acid of embodiment A97, wherein the DNA repair enzyme or portion thereof is derived from a mismatch excision repair enzyme.
<br/>
A103. The nucleic acid of embodiment A97 or A103, wherein the DNA repair enzyme comprises the amino acid sequence of SEQ ID NO: 159, SEQ ID NO: 160, SEQ ID NO: 161, SEQ ID NO: 162, SEQ ID NO: 163, SEQ ID NO: 164, SEQ ID NO: 165, SEQ ID NO: 166, SEQ ID NO: 167, or SEQ ID NO: 168.
<br/>
A104. The nucleic acid of embodiment A97, wherein the DNA repair enzyme or portion thereof is derived from a nucleotide excision repair enzyme.
<br/>
A105. The nucleic acid of embodiment A97 or A104, wherein the DNA repair enzyme comprises an amino acid sequence of SEQ ID NO: 169, SEQ ID NO: 170 SEQ ID NO: 171, SEQ ID NO: 172, SEQ ID NO: 173, SEQ ID NO: 174, SEQ ID NO: 175, SEQ ID NO: 176, SEQ ID NO: 177, SEQ ID NO: 178, SEQ ID NO: 179, SEQ ID NO: 180 SEQ ID NO: 181, SEQ ID NO: 182, SEQ ID NO: 183, SEQ ID NO: 184, SEQ ID NO: 185, SEQ ID NO: 186, SEQ ID NO: 187, SEQ ID NO: 188, SEQ ID NO: 189, SEQ ID NO: 190 SEQ ID NO: 191, SEQ ID NO: 192, SEQ ID NO: 193, SEQ ID NO: 194, SEQ ID NO: 195, SEQ ID NO: 196, or SEQ ID NO: 197.
<br/>
A106. The nucleic acid of embodiment A97, wherein the DNA repair enzyme or portion thereof is derived from an editing or processing nuclease.
<br/>
A107. The nucleic acid of embodiment A97 or A106, wherein the DNA repair enzyme comprises an amino acid sequence of SEQ NO: 198, SEQ ID NO: 199, SEQ ID NO: 200, SEQ ID NO: 201, SEQ ID NO: 202, SEQ ID NO: 203, SEQ ID NO: 204, or SEQ ID NO: 205.
<br/>
A108. The nucleic acid of any of embodiments A61-A65, wherein the first peptide comprises a telomerase, or portion thereof.
<br/>
A109. The nucleic acid of embodiment A108, wherein the telomerase comprises an amino acid sequence of SEQ ID NO: 206, SEQ ID NO: 207, SEQ ID NO: 208, or SEQ ID NO: 209.
<br/>
A110. The nucleic acid of any one of embodiments A61-A65, wherein the first peptide comprises a protection of telomerase protein 1, or portion thereof.
<br/>
A111. The nucleic acid of embodiment A110, wherein the protection of telomerase protein 1 comprises an amino acid sequence of SEQ ID NO: 210 or SEQ ID NO: 211.
<br/>
A112. The nucleic acid of any of embodiments A61-A65, A68-A76 or A79-A111, further comprising a third nucleic acid sequence encoding a second peptide, and wherein the third nucleic acid sequence lies between the first nucleic acid sequence encoding the operon sequence and the second nucleic acid sequence encoding the first peptide.
<br/>
A113. The nucleic acid of any of embodiments A61-A65, A68-A76 or A79-A111, further comprising a third nucleic acid sequence encoding a second peptide, and wherein the third nucleic acid sequence is attached to the second nucleic acid sequence encoding the first peptide at the opposite end of the first nucleic acid sequence encoding the operon sequence.
<br/>
A114. The nucleic acid of embodiment A112 or A113, wherein the second peptide comprises SEQ ID NO: 5, SEQ ID NO: 6, SEQ ID NO: 7 SEQ ID NO: 8, SEQ ID NO: 9, SEQ ID NO: 10, SEQ ID NO: 11, SEQ ID NO: 12, SEQ ID NO: 13, SEQ ID NO: 14, SEQ ID NO: 15, SEQ ID NO: 16, SEQ ID NO: 17 SEQ ID NO: 18, SEQ ID NO: 19, SEQ ID NO: 20, SEQ ID NO: 21, SEQ ID NO: 22, SEQ ID NO: 23, SEQ ID NO: 24, SEQ ID NO: 25, SEQ ID NO: 26, SEQ ID NO: 27 SEQ ID NO: 28, SEQ ID NO: 29, SEQ ID NO: 30, SEQ ID NO: 31, SEQ ID NO: 32, SEQ ID NO: 33, SEQ ID NO: 34, SEQ ID NO: 35, SEQ ID NO: 36, SEQ ID NO: 37 SEQ ID NO: 38, SEQ ID NO: 39, SEQ ID NO: 45, SEQ ID NO: 46, SEQ ID NO: 47 SEQ ID NO: 48, SEQ ID NO: 49, SEQ ID NO: 50, SEQ ID NO: 51, SEQ ID NO: 52, SEQ ID NO: 53, SEQ ID NO: 54, SEQ ID NO: 55, SEQ ID NO: 56, SEQ ID NO: 57, SEQ ID NO: 58, SEQ ID NO: 59, SEQ ID NO: 60, SEQ ID NO: 61, SEQ ID NO: 62, SEQ ID NO: 63, SEQ ID NO: 64, SEQ ID NO: 65, SEQ ID NO: 66, SEQ ID NO: 67 SEQ ID NO: 68, SEQ ID NO: 69, SEQ ID NO: 70, SEQ ID NO: 71, SEQ ID NO: 72, SEQ ID NO: 73, SEQ ID NO: 74, SEQ ID NO: 75, SEQ ID NO: 76, SEQ ID NO: 77 SEQ ID NO: 78, SEQ ID NO: 79, SEQ ID NO: 80, SEQ ID NO: 81, SEQ ID NO: 82, SEQ ID NO: 83, SEQ ID NO: 84, SEQ ID NO: 85, SEQ ID NO: 86, SEQ ID NO: 87 SEQ ID NO: 88, SEQ ID NO: 89, SEQ ID NO: 90, SEQ ID NO: 91, SEQ ID NO: 92, SEQ ID NO: 93, SEQ ID NO: 94, SEQ ID NO: 95, SEQ ID NO: 96, SEQ ID NO: 97 SEQ ID NO: 98, SEQ ID NO: 99, SEQ ID NO: 100, SEQ ID NO: 101, SEQ ID NO: 102, SEQ ID NO: 103, SEQ ID NO: 104, SEQ ID NO: 105, SEQ ID NO: 106, SEQ ID NO: 107 SEQ ID NO: 108, SEQ ID NO: 109, SEQ ID NO: 110, SEQ ID NO: 111, SEQ ID NO: 112, SEQ ID NO: 113, SEQ ID NO: 114, SEQ ID NO: 115, SEQ ID NO: 116, SEQ ID NO: 117 SEQ ID NO: 118, SEQ ID NO: 119, SEQ ID NO: 120, SEQ ID NO: 121, SEQ ID NO: 122, SEQ ID NO: 123, SEQ ID NO: 124, SEQ ID NO: 125, SEQ ID NO: 126, SEQ ID NO: 127 SEQ ID NO: 128, SEQ ID NO: 129, SEQ ID NO: 130, SEQ ID NO: 131, SEQ ID NO: 132, SEQ ID NO: 133, SEQ ID NO: 134, SEQ ID NO: 135, SEQ ID NO: 136, SEQ ID NO: 137 SEQ ID NO: 138, SEQ ID NO: 139, SEQ ID NO: 140, SEQ ID NO: 141, SEQ ID NO: 142, SEQ ID NO: 143, SEQ ID NO: 144, SEQ ID NO: 145, SEQ ID NO: 146, SEQ ID NO: 147 SEQ ID NO: 148, SEQ ID NO: 149, SEQ ID NO: 150, SEQ ID NO: 151, SEQ ID NO: 152, SEQ ID NO: 153, SEQ ID NO: 154, SEQ ID NO: 155, SEQ ID NO: 156, SEQ ID NO: 157 SEQ ID NO: 158, SEQ ID NO: 159, SEQ ID NO: 160, SEQ ID NO: 161, SEQ ID NO: 162, SEQ ID NO: 163, SEQ ID NO: 164, SEQ ID NO: 165, SEQ ID NO: 166, SEQ ID NO: 167 SEQ ID NO: 168, SEQ ID NO: 169, SEQ ID NO: 170, SEQ ID NO: 171, SEQ ID NO: 172, SEQ ID NO: 173, SEQ ID NO: 174, SEQ ID NO: 175, SEQ ID NO: 176, SEQ ID NO: 177 SEQ ID NO: 178, SEQ ID NO: 179, SEQ ID NO: 180, SEQ ID NO: 181, SEQ ID NO: 182, SEQ ID NO: 183, SEQ ID NO: 184, SEQ ID NO: 185, SEQ ID NO: 106, SEQ ID NO: 187 SEQ ID NO: 188, SEQ ID NO: 189, SEQ ID NO: 190, SEQ ID NO: 191, SEQ ID NO: 192, SEQ ID NO: 193, SEQ ID NO: 194, SEQ ID NO: 195, SEQ ID NO: 196, SEQ ID NO: 197 SEQ ID NO: 198, SEQ ID NO: 199, SEQ ID NO: 200, SEQ ID NO: 201, SEQ ID NO: 202, SEQ ID NO: 203, SEQ ID NO: 204, SEQ ID NO: 205, SEQ ID NO: 206, SEQ ID NO: 207 SEQ ID NO: 208, SEQ ID NO: 209, SEQ ID NO: 210, or SEQ ID NO: 211 and wherein the second peptide does not comprise the amino acid sequence of the first peptide.
<br/>
A115. The nucleic acid of any of embodiments A61-A65, A68-A76, or A79-A111, further comprising a nucleic acid sequence encoding a secretory peptide, and wherein the nucleic acid sequence encoding the secretory peptide lies between the operon sequence and second nucleic acid sequence encoding the first peptide.
<br/>
A116. The nucleic acid of any of embodiments A61-A65, A68-A76, or A79-A111, further comprising a nucleic acid sequence encoding a secretory peptide, and wherein the nucleic acid sequence encoding the secretory peptide is attached to the second nucleic acid sequence encoding the first peptide at one end and is at an opposite end of the operon sequence.
<br/>
A117. The nucleic acid of any of embodiments A112-A114, further comprising a nucleic acid sequence encoding secretory peptide, and wherein the nucleic acid sequence encoding the secretory peptide is attached to the second nucleic acid sequence encoding the first peptide or the third nucleic acid sequence encoding the second peptide, and wherein the nucleic acid sequence encoding the secretory peptide is between the nucleic acid encoding the operon and the second nucleic acid sequence encoding the first peptide or the third nucleic acid sequence encoding the second peptide.
<br/>
A118. The nucleic acid of any of embodiments A112-A114, further comprising a nucleic acid sequence encoding secretory peptide, and wherein the nucleic acid sequence encoding the secretory peptide is attached to the second nucleic acid sequence encoding the first peptide or the third nucleic acid sequence encoding the second peptide, and wherein the nucleic acid sequence encoding the secretory peptide is at an opposite end of the nucleic acid encoding the operon.
<br/>
A119. The nucleic acid of any one of embodiments A115-A118, wherein the signal sequence for secretion comprises the sequence SEQ ID NO: 212, SEQ ID NO: 213, SEQ ID NO: 214, SEQ ID NO: 215, or SEQ ID NO: 216.
<br/>
A120. The nucleic acid of any one of embodiments A57-A119, wherein the nucleic acid is optimized for protein expression by computational methods.
<br/>
A121. A cell comprising any one of the nucleic acids set forth in any embodiment or of embodiments A57-A120.
<br/>
A122. The cell of embodiment A121, wherein the cell comprises a genome with a mutation or gene knockout.
<br/>
A123. The cell of embodiment A121 or A122, wherein the cell is a bacterial cell.
<br/>
A124. The cell of embodiment A121 or A122, wherein the cell is a fungal cell.
<br/>
A125. The cell of any one of embodiments A121-A122, wherein the cell is from a genus <i>Propionibacterium. </i>
<br/>
A126. The cell of any one of embodiments A121-A122, or A125, wherein the cell is <i>Propionibacterium acnes. </i>
<br/>
A127. The cell of any one of embodiments A121-A122, wherein the cell is from a genus <i>Corynebacterium. </i>
<br/>
A128. The cell of any one of embodiments A121-A122, or A127, wherein the cell is <i>Corynebacterium striatum. </i>
<br/>
A129. The cell of any one of embodiments A121-A122, wherein the cell is from a genus <i>Staphylococcus. </i>
<br/>
A130. The cell of any one of embodiments A121-A122, or A129, wherein the cell is <i>Staphylococcus epidermidis. </i>
<br/>
A131. The cell of any one of embodiments A121-A122, wherein the cell is from a genus <i>Streptococcus. </i>
<br/>
A132. The cell of any one of embodiments A121-A122, or A131, wherein the cell is <i>Streptococcus thermophiles. </i>
<br/>
A133. The cell of any one of embodiments A121-A122, wherein the cell is from a genus <i>Lactobacillus </i>
<br/>
A134. The cell of any one of embodiments A121-A122, or A133, wherein the cell is <i>Lactobacillus acidophilus. </i>
<br/>
A135. The cell of any one of embodiments A121-A122, wherein the cell if from a genus <i>Lactococcus. </i>
<br/>
A136. The cell of any one of embodiments A121-A122, or A135, wherein the cell is <i>Lactococcus lactis. </i>
<br/>
A137. The cell of any one of embodiments A121-A122, wherein the cell is from a genus <i>Actinobacteria. </i>
<br/>
A138. The cell of any one of embodiments A121-A122, wherein the cell is from a genus <i>Micrococci. </i>
<br/>
A139. The cell of any one of embodiments A121-A122, wherein the cell is from a genus <i>Demodex. </i>
<br/>
A140. The cell of any one of embodiments A121-A122, wherein the cell is from a genus <i>Malassezia. </i>
<br/>
A141. The cell of any one of embodiments A121-A122, wherein the cell is from a genus <i>Escherichia. </i>
<br/>
A142. The cell of any one of embodiments A121-A122, or A142 wherein the cell is <i>Escherichia coli. </i>
<br/>
A143. The cell of any one of embodiments A121-A142, wherein the cell is non-pathogenic.
<br/>
A144. The cell of embodiment A121, A122 or A124, wherein the cell is from a genus <i>Candida. </i>
<br/>
A145. The cell of any one of embodiments A121, A122, A124 or A144 wherein the cell is <i>Candida albicans. </i>
<br/>
A146. The cell of any one of embodiments A121, A122, A124 or A144 wherein the cell is <i>Candida glabrata. </i>
<br/>
A147. The cell of any one of embodiments A121, A122, A124 or A144 wherein the cell is <i>Candida tropicalis. </i>
<br/>
A148. The cell of any one of embodiments A121, A122, A124 or A144 wherein the cell is <i>Candida parapsilosis. </i>
<br/>
A149. The cell of any one of embodiments A121, A122, A124 or A144 wherein the cell is <i>Candida krusei. </i>
<br/>
A150. The cell of any one of embodiments A121-A149, wherein the mutation or knock out is in lipase gene.
<br/>
A151. The cell of any one of embodiments A121-A149, wherein the mutation or knock out is in a gene encoding for nutrient synthesis.
<br/>
A152. The cell of any one of embodiments A121-A149, wherein the mutation or knock out is in a gene encoding for a pathogenic biomolecule.
<br/>
A153. The cell of any one of embodiments A121-A123, A125-A126 or A152, wherein the cell comprises a genome with a mutation or knock out in a gene that encodes for glutamine synthetase.
<br/>
A154. The cell of any one of embodiments A121-A123, A125-A126 or A152, wherein the cell comprises a genome with a mutation or knock out in a gene that encodes for asparagine synthetase.
<br/>
A155. The cell of any one of embodiments A121-A123, A125-A126 or A152, wherein the cell comprises a genome with a mutation or knock out in a gene that encodes for aspartokinase.
<br/>
A156. The cell of any one of embodiments A121-A123, A125-A126 or A152, wherein the cell comprises a genome with a mutation or knock out in a gene that codes for aspartate semialdehyde dehydrogenase.
<br/>
A157. The cell of any one of embodiments A121-A123, A125-A126 or A152, wherein the cell comprises a genome with a mutation or knock out in a gene that codes for methionine synthesis.
<br/>
A158. A topical formulation comprising the cell of any one of embodiments A121-A157; and a vehicle for the cell.
<br/>
A159. A method of treating, inhibiting, or ameliorating a disorder in a subject by delivering controlled expression of a peptide comprising administering to the subject the topical formulation in embodiment A158 for controlled delivery of a peptide.
<br/>
A160. The method according to embodiment A159, wherein the disorder is acne, rosacea, alopecia, onchomychosis, osmidrosis, greying hair, cutaneous inflammation, dyschromia, aging damage, chronic skin wounds, cutaneous inflammation, nail fungus, an autoimmune disease, or hemophilia.
<br/>
A161. The method of embodiment A159 or A160 wherein the administering comprises placing the composition on the epidermis.
<br/>
A162. The method of any one of embodiments A159-A161 further comprising controlling expression of the peptide wherein controlling is performed by administering a second compound.
<br/>
A163. The method of embodiment A162, wherein the second compound is lactose.
<br/>
A164. The method of embodiment A162, wherein the second compound is tryptophan.
<br/>
A165. The method of any one of embodiments A159-A164 further comprising controlling proliferation of a cell, wherein controlling proliferation is performed by administering a third compound.
<br/>
A166. The method of embodiment A165, wherein the third compound a nutrient.
<br/>
A167. The method of embodiment A165 or A166, wherein the third compound is an amino acid.
<br/>
A168. The method of any one of embodiments A165-A67, wherein the third compound is glutamine.
<br/>
A169. The method of any one of embodiments A165-A67, wherein the third compound is asparagine.
<br/>
A170. The method of any one of embodiments A165-A67, wherein the third compound is aspartate.
<br/>
A171. The method of any one of embodiments A165-A67, wherein the third compound is methionine.
<br/>
B1. Use of a composition comprising a genetically modified bacteria of the genus <i>Propionibacterium</i>, as disclosed herein, wherein the bacteria comprises a nucleic acid encoding one or more mammalian growth factors and/or one or more mammalian cytokines, for treatment of a skin or nail disorder of a mammal.
<br/>
B2. The use of the bacteria of embodiment B1, wherein the mammal is a human, dog or cat.
<br/>
B3. The use of the bacteria of embodiment B1 or B2, wherein the skin or nail disorder comprises acne, actinic keratosis, alopecia areata, athlete's foot, onchomychosis, atopic dermatitis, osmidrosis, eczema, fungal infection of the nails, psoriasis, rosacea, slow wound healing, folliculitis, keratosis pilaris, perioral dermatitis, angiofibromas, cutaneous inflammation, aging damage, dyschromia, premature greying hair, or seborrhea.
</div>
</li> </ul>
</div>
</section><section itemprop="claims" itemscope="">
<h2>Claims (<span itemprop="count">52</span>)</h2>
<div html="" itemprop="content"><div class="claims" lang="EN" load-source="patent-office" mxw-id="PCLM250224336">
<claim-statement>What is claimed is:</claim-statement>
<div class="claim"> <div class="claim" id="CLM-00001" num="00001">
<div class="claim-text"> <b>1</b>. A genetically modified bacteria of the genus <i>Propionibacterium </i>comprising a nucleic acid encoding a mammalian growth factor or a mammalian cytokine.</div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="CLM-00002" num="00002">
<div class="claim-text"> <b>2</b>. A composition comprising the genetically modified bacteria of the genus <i>Propionibacterium </i>of <claim-ref idref="CLM-00001">claim 1</claim-ref>.</div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="CLM-00003" num="00003">
<div class="claim-text"> <b>3</b>. The genetically modified bacteria of <claim-ref idref="CLM-00001">claim 1</claim-ref>, wherein the growth factor is a hormone.</div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="CLM-00004" num="00004">
<div class="claim-text"> <b>4</b>. The genetically modified bacteria of <claim-ref idref="CLM-00001">claim 1</claim-ref>, wherein the growth factor is a human or bovine hormone.</div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="CLM-00005" num="00005">
<div class="claim-text"> <b>5</b>. The genetically modified bacteria of <claim-ref idref="CLM-00001">claim 1</claim-ref>, wherein the growth factor is a somatotrophin.</div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="CLM-00006" num="00006">
<div class="claim-text"> <b>6</b>. The genetically modified bacteria of <claim-ref idref="CLM-00005">claim 5</claim-ref>, wherein the somatotrophin comprises SEQ ID NO:45, SEQ ID NO:46, SEQ ID NO:47, SEQ ID NO:48 or SEQ ID NO:49.</div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="CLM-00007" num="00007">
<div class="claim-text"> <b>7</b>. The genetically modified bacteria of <claim-ref idref="CLM-00001">claim 1</claim-ref>, wherein the growth factor is secreted by the bacteria.</div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="CLM-00008" num="00008">
<div class="claim-text"> <b>8</b>. The genetically modified bacteria of <claim-ref idref="CLM-00001">claim 1</claim-ref>, wherein the growth factor is a human growth factor.</div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="CLM-00009" num="00009">
<div class="claim-text"> <b>9</b>. The genetically modified bacteria of <claim-ref idref="CLM-00001">claim 1</claim-ref>, wherein the growth factor is selected from the group consisting of transforming growth factor beta (TGF-β), vascular endothelial growth factor (VEGF), hepatocyte growth factor (HGF), fibroblast growth factor (FGF), insulin-like growth factor-1 (IGF1), platelet derived growth factor (PDGF), granulocyte monocyte colony stimulating factor (GMCSF) and epidermal growth factor (EGF).</div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="CLM-00010" num="00010">
<div class="claim-text"> <b>10</b>. The genetically modified bacteria of <claim-ref idref="CLM-00001">claim 1</claim-ref>, wherein the growth factor is a transforming growth factor comprising any one of SEQ ID NO:63 to SEQ ID NO:69, or a hepatocyte growth factor comprising SEQ ID NO:70.</div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="CLM-00011" num="00011">
<div class="claim-text"> <b>11</b>. The genetically modified bacteria of <claim-ref idref="CLM-00001">claim 1</claim-ref>, wherein the growth factor is a vascular endothelial growth factor (VEGF) comprising any one of SEQ ID NO:71 to SEQ ID NO:91.</div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="CLM-00012" num="00012">
<div class="claim-text"> <b>12</b>. The genetically modified bacteria of <claim-ref idref="CLM-00001">claim 1</claim-ref>, wherein the growth factor is a platelet derived growth factor (PDGF) comprising any one of SEQ ID NO:93 to SEQ ID NO:102.</div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="CLM-00013" num="00013">
<div class="claim-text"> <b>13</b>. The genetically modified bacteria of <claim-ref idref="CLM-00001">claim 1</claim-ref>, wherein the growth factor is an epidermal growth factor (EGF) comprising any one of SEQ ID NO:103 to SEQ ID NO:106.</div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="CLM-00014" num="00014">
<div class="claim-text"> <b>14</b>. The genetically modified bacteria of <claim-ref idref="CLM-00001">claim 1</claim-ref>, wherein the growth factor is a fibroblast growth factor (FGF) comprising any one of SEQ ID NO:107 to SEQ ID NO:144.</div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="CLM-00015" num="00015">
<div class="claim-text"> <b>15</b>. The genetically modified bacteria of <claim-ref idref="CLM-00001">claim 1</claim-ref>, wherein the bacteria secrete the mammalian cytokine.</div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="CLM-00016" num="00016">
<div class="claim-text"> <b>16</b>. The genetically modified bacteria of <claim-ref idref="CLM-00001">claim 1</claim-ref>, wherein the mammalian cytokine is an immunosuppressive cytokine.</div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="CLM-00017" num="00017">
<div class="claim-text"> <b>17</b>. The genetically modified bacteria of <claim-ref idref="CLM-00001">claim 1</claim-ref>, wherein the mammalian cytokine is a human cytokine.</div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="CLM-00018" num="00018">
<div class="claim-text"> <b>18</b>. The genetically modified bacteria of <claim-ref idref="CLM-00001">claim 1</claim-ref>, wherein the cytokine is selected from the group consisting of interleukin-10 (IL-10), interleukin-6 (IL-6), interleukin-7 (IL-7) and interleukin-8 (IL-8).</div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="CLM-00019" num="00019">
<div class="claim-text"> <b>19</b>. The genetically modified bacteria of <claim-ref idref="CLM-00015">claim 15</claim-ref>, wherein the cytokine is selected from the group consisting of SEQ ID NO:24, SEQ ID NO:25, SEQ ID NO:26, SEQ ID NO:27 and SEQ ID NO:28.</div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="CLM-00020" num="00020">
<div class="claim-text"> <b>20</b>. The genetically modified bacteria of <claim-ref idref="CLM-00001">claim 1</claim-ref>, wherein the mammalian cytokine comprises IL-10.</div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="CLM-00021" num="00021">
<div class="claim-text"> <b>21</b>. The genetically modified bacteria of <claim-ref idref="CLM-00020">claim 20</claim-ref>, wherein the IL-10 comprises SEQ ID NO:25.</div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="CLM-00022" num="00022">
<div class="claim-text"> <b>22</b>. The genetically modified bacteria of <claim-ref idref="CLM-00001">claim 1</claim-ref>, wherein the genus of <i>Propionibacterium </i>comprises a species selected from the group consisting of <i>Propionibacterium acidifaciens, Propionibacterium acidipropionici, Propionibacterium acnes, Propionibacterium australiense, Propionibacterium avidum, Propionibacterium cyclohexanicum, Propionibacterium freudenreichii, Propionibacterium freudenreichii, Propionibacterium granulosum, Propionibacterium jensenii, Propionibacterium microaerophilum, Propionibacterium propionicum </i>and <i>Propionibacterium thoeniiand. </i> </div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="CLM-00023" num="00023">
<div class="claim-text"> <b>23</b>. The genetically modified bacteria of <claim-ref idref="CLM-00022">claim 22</claim-ref>, wherein the species is <i>Propionibacterium acnes </i>(<i>P. acnes</i>).</div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="CLM-00024" num="00024">
<div class="claim-text"> <b>24</b>. The genetically modified bacteria of <claim-ref idref="CLM-00022">claim 22</claim-ref>, wherein the strain of <i>P. acnes </i>comprises a CRISPR array.</div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="CLM-00025" num="00025">
<div class="claim-text"> <b>25</b>. The genetically modified bacteria of <claim-ref idref="CLM-00024">claim 24</claim-ref>, wherein the strain of <i>P. acnes </i>is <i>P. acnes</i>, type II, ribotype 6.</div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="CLM-00026" num="00026">
<div class="claim-text"> <b>26</b>. The genetically modified bacteria of <claim-ref idref="CLM-00001">claim 1</claim-ref>, wherein the nucleic acid comprises an inducible promoter configured to regulate the expression of the mammalian growth factor or the mammalian cytokine.</div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="CLM-00027" num="00027">
<div class="claim-text"> <b>27</b>. The genetically modified bacteria of <claim-ref idref="CLM-00001">claim 1</claim-ref>, wherein the nucleic acid is under control of an endogenous promoter that directs expression of the mammalian growth factor or the mammalian cytokine.</div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="CLM-00028" num="00028">
<div class="claim-text"> <b>28</b>. The genetically modified bacteria of <claim-ref idref="CLM-00001">claim 1</claim-ref>, wherein the nucleic acid comprises an endogenous promoter configured to direct expression of the mammalian growth factor or the mammalian cytokine.</div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="CLM-00029" num="00029">
<div class="claim-text"> <b>29</b>. The genetically modified bacteria of <claim-ref idref="CLM-00027">claim 27</claim-ref> or <claim-ref idref="CLM-00028">28</claim-ref>, wherein the inducible promoter or endogenous promoter comprises a lacZ promoter or lacZ operon, or an ara operon promoter.</div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="CLM-00030" num="00030">
<div class="claim-text"> <b>30</b>. The genetically modified bacteria of <claim-ref idref="CLM-00001">claim 1</claim-ref>, wherein the nucleic acid encoding a mammalian growth factor or a mammalian cytokine is inserted into all or a part of an endogenous gene encoding a pathogenic protein.</div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="CLM-00031" num="00031">
<div class="claim-text"> <b>31</b>. The genetically modified bacteria of <claim-ref idref="CLM-00001">claim 1</claim-ref>, wherein expression of an endogenous protein is substantially reduced or eliminated.</div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="CLM-00032" num="00032">
<div class="claim-text"> <b>32</b>. The genetically modified bacteria of <claim-ref idref="CLM-00001">claim 1</claim-ref>, wherein expression of a pathogenic protein is substantially reduced or eliminated.</div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="CLM-00033" num="00033">
<div class="claim-text"> <b>33</b>. The genetically modified bacteria of <claim-ref idref="CLM-00032">claim 32</claim-ref>, wherein the pathogenic protein comprises an endotoxin and/or an exotoxin.</div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="CLM-00034" num="00034">
<div class="claim-text"> <b>34</b>. The genetically modified bacteria of <claim-ref idref="CLM-00032">claim 32</claim-ref>, wherein the endogenous pathogenic protein comprises a glyceraldehyde 3-phosphate dehydrogenase (GADPH) protein or a CAMP protein.</div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="CLM-00035" num="00035">
<div class="claim-text"> <b>35</b>. The genetically modified bacteria of <claim-ref idref="CLM-00001">claim 1</claim-ref>, comprising an inducible promoter regulating expression of an essential protein.</div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="CLM-00036" num="00036">
<div class="claim-text"> <b>36</b>. The genetically modified bacteria of <claim-ref idref="CLM-00035">claim 35</claim-ref>, wherein the essential protein is selected from the group consisting of a chromosome replication initiator protein DnaA, FtsA, FtsI, FtsL, FtsK, FtsN, FtsQ, FtsW, FtsZ, ZipA, aroE, atpD, gmk, guaA, lepA, recA, and sodA.</div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="CLM-00037" num="00037">
<div class="claim-text"> <b>37</b>. The genetically modified bacteria of <claim-ref idref="CLM-00035">claim 35</claim-ref>, wherein the inducible promoter is inducible by a sugar.</div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="CLM-00038" num="00038">
<div class="claim-text"> <b>38</b>. The genetically modified bacteria of <claim-ref idref="CLM-00037">claim 37</claim-ref>, wherein the sugar is lactose or arabinose.</div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="CLM-00039" num="00039">
<div class="claim-text"> <b>39</b>. The genetically modified bacteria of <claim-ref idref="CLM-00035">claim 35</claim-ref>, wherein the inducible promoter is inducible by an amino acid.</div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="CLM-00040" num="00040">
<div class="claim-text"> <b>40</b>. The genetically modified bacteria of <claim-ref idref="CLM-00035">claim 35</claim-ref>, wherein the inducible promoter is inducible by a synthetic amino acid.</div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="CLM-00041" num="00041">
<div class="claim-text"> <b>41</b>. The genetically modified bacteria of <claim-ref idref="CLM-00001">claim 1</claim-ref>, wherein the bacteria have multiple genetic modifications.</div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="CLM-00042" num="00042">
<div class="claim-text"> <b>42</b>. A composition comprising a plurality of genetically modified bacteria of <claim-ref idref="CLM-00001">claim 1</claim-ref>, wherein the bacteria have multiple genetic modifications.</div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="CLM-00043" num="00043">
<div class="claim-text"> <b>43</b>. The genetically modified bacteria of <claim-ref idref="CLM-00041">claim 41</claim-ref> or composition of <claim-ref idref="CLM-00042">claim 42</claim-ref>, wherein the bacteria has a modification in which 1) expression of an endogenous protein is substantially reduced or eliminated; and 2) an inducible promoter that regulates expression of an essential protein.</div>
</div>
</div> <div class="claim"> <div class="claim" id="CLM-00044" num="00044">
<div class="claim-text"> <b>44</b>. A genetically modified bacteria of the species <i>Propionibacterium acnes </i>comprising:
<div class="claim-text">a) a mammalian growth factor, wherein the mammalian growth factor is secreted; and</div> <div class="claim-text">b) an inducible promoter directing the expression of an essential protein selected from the group consisting of a chromosome replication initiator protein DnaA, FtsA, FtsI, FtsL, FtsK, FtsN, FtsQ, FtsW, FtsZ, ZipA, aroE, atpD, gmk, guaA, lepA, recA, and sodA;</div> <div class="claim-text">wherein the expression of an endogenous CAMP2 and/or endogenous GADPH protein is substantially reduced or eliminated in the genetically modified bacteria.</div> </div>
</div>
</div> <div class="claim"> <div class="claim" id="CLM-00045" num="00045">
<div class="claim-text"> <b>45</b>. A genetically modified bacteria of the species <i>Propionibacterium acnes </i>comprising:
<div class="claim-text">a) a human IL10, wherein the IL10 is secreted; and</div> <div class="claim-text">b) an inducible promoter directing the expression of an essential protein selected from the group consisting of a chromosome replication initiator protein DnaA, FtsA, FtsI, FtsL, FtsK, FtsN, FtsQ, FtsW, FtsZ, ZipA, aroE, atpD, gmk, guaA, lepA, recA, and sodA;</div> <div class="claim-text">wherein the expression of an endogenous CAMP2 and/or endogenous GADPH protein is substantially reduced or eliminated in the genetically modified bacteria.</div> </div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="CLM-00046" num="00046">
<div class="claim-text"> <b>46</b>. A composition comprising the genetically modified bacteria of <claim-ref idref="CLM-00044">claim 44</claim-ref> or <claim-ref idref="CLM-00045">45</claim-ref>.</div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="CLM-00047" num="00047">
<div class="claim-text"> <b>47</b>. The composition of <claim-ref idref="CLM-00002">claim 2</claim-ref>, <claim-ref idref="CLM-00042">42</claim-ref> or <claim-ref idref="CLM-00046">46</claim-ref>, wherein the composition is configured for topical or mucosal administration to a mammal.</div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="CLM-00048" num="00048">
<div class="claim-text"> <b>48</b>. The composition of <claim-ref idref="CLM-00002">claim 2</claim-ref>, <claim-ref idref="CLM-00042">42</claim-ref> or <claim-ref idref="CLM-00046">46</claim-ref>, wherein the composition comprises a sugar or amino acid, wherein the inducible promoter is induced or stimulate by the presence of the sugar or amino acid.</div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="CLM-00049" num="00049">
<div class="claim-text"> <b>49</b>. A kit comprising the genetically modified bacteria or composition of any of <claim-ref idref="CLM-00001">claims 1</claim-ref>-<claim-ref idref="CLM-00048">48</claim-ref>, and instructions for use.</div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="CLM-00050" num="00050">
<div class="claim-text"> <b>50</b>. A method of treating a skin or nail disorder comprising contacting the skin of a mammal with the genetically modified bacteria or composition of any of <claim-ref idref="CLM-00001">claims 1</claim-ref>-<claim-ref idref="CLM-00048">48</claim-ref> thereby treating the skin or nail disorder.</div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="CLM-00051" num="00051">
<div class="claim-text"> <b>51</b>. The method of <claim-ref idref="CLM-00050">claim 50</claim-ref>, wherein the mammal is a human, dog or cat.</div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="CLM-00052" num="00052">
<div class="claim-text"> <b>52</b>. The method of <claim-ref idref="CLM-00050">claim 50</claim-ref>, wherein the skin or nail disorder comprises acne, actinic keratosis, alopecia areata, athlete's foot, onchomychosis, atopic dermatitis, osmidrosis, eczema, fungal infection of the nails, psoriasis, rosacea, slow wound healing, folliculitis, keratosis pilaris, perioral dermatitis, angiofibromas, cutaneous inflammation, aging damage, dyschromia, premature greying hair, or seborrhea.</div>
</div>
</div> </div>
</div>
</section>
                </article>
            </search-app>
        </body>
    </html>
    