
    <html>
        <body>
            <search-app>
                <article class="result" itemscope="" itemtype="http://schema.org/ScholarlyArticle">
    <h1 itemprop="pageTitle">US20150056642A1 - Armet as a marker for chronic obstructive pulmonary disease (copd) 
        - Google Patents</h1><section itemprop="abstract" itemscope="">
<h2>Abstract</h2>
<div html="" itemprop="content"><abstract lang="EN" load-source="patent-office" mxw-id="PA150446843">
<div class="abstract" id="p-0001" num="0000">An in vitro method aiding in the assessment of chronic obstructive pulmonary disease (COPD). The disclosure further relates to a method for assessing COPD from a sample, derived from an individual, by measuring the protein ARMET in said sample in vitro.</div>
</abstract>
</div>
</section><section itemprop="description" itemscope="">
<h2>Description</h2>
<div html="" itemprop="content"><ul class="description" lang="EN" load-source="patent-office" mxw-id="PDES88024915">
<heading id="h-0001">CROSS-REFERENCE TO RELATED APPLICATIONS</heading>
<li> <para-num num="[0001]"> </para-num> <div class="description-line" id="p-0002" num="0001">This application is a continuation of U.S. application Ser. No. 14/022,276 filed Sep. 10, 2013, which is a continuation of International Application No. PCT/EP2012/053843 filed Mar. 7, 2012, which claims the benefit of European Patent Application No. 11157918.1 filed Mar. 11, 2011, the disclosures of which are hereby incorporated by reference in their entirety.</div>
</li> <heading id="h-0002">SEQUENCE LISTING</heading>
<li> <para-num num="[0002]"> </para-num> <div class="description-line" id="p-0003" num="0002">The instant application contains a Sequence Listing which has been submitted in ASCII format via EFS-Web and is hereby incorporated by reference in its entirety. Said ASCII copy, created on Sep. 4, 2013, is named SEQUENCE_LISTING<sub>—</sub>27347US.txt, and is twenty-five thousand eight hundred and seventy-seven bytes in size.</div>
</li> <heading id="h-0003">BACKGROUND</heading>
<li> <para-num num="[0003]"> </para-num> <div class="description-line" id="p-0004" num="0003">Chronic obstructive pulmonary disease (COPD) is a disease characterized by chronic inflammation and irreversible airflow obstruction with a decline in the lung function parameter FEV1 that is more rapid than normal. This leads to a limitation of the flow of air to and from the lungs causing shortness of breath. The disease has two major aspects of pathology, namely chronic bronchitis, characterized by mucus hyper-secretion from the conducting airways, and emphysema, characterized by destructive changes in the alveoli. In clinical practice, COPD is defined by its characteristically low airflow on lung function tests (Nathell, L., et al., Respiratory Research 8 (2007) 89). In contrast to asthma, this limitation is poorly reversible and usually gets progressively worse over time.</div>
</li> <li> <para-num num="[0004]"> </para-num> <div class="description-line" id="p-0005" num="0004">Worldwide, COPD ranked as the sixth leading cause of death in 1990. It is projected to be the fourth leading cause of death worldwide by 2030 due to an increase in smoking rates and demographic changes in many countries (Mathers, C. D., et al., PLoS Med. 3 (2006) e442). COPD is the 4th leading cause of death in the U.S., and the economic burden of COPD in the U.S. in 2007 was $42.6 billion in health care costs and lost productivity.</div>
</li> <li> <para-num num="[0005]"> </para-num> <div class="description-line" id="p-0006" num="0005">COPD may be caused by noxious particles or gas, for example from tobacco smoking, which triggers an abnormal inflammatory response in the lung (Rabe, K. F., et al., Am. J. Respir. Crit. Care Med. 176 (2007) 532-555 and Hogg, J. C., et al., N. Engl. J. Med. 350 (2004) 2645-2653). The inflammatory response in the larger airways is known as chronic bronchitis, which is diagnosed clinically when people regularly cough up sputum. In the alveoli, the inflammatory response can cause destruction of the tissues of the lung, a process known as emphysema. The natural course of COPD is characterized by occasional sudden worsening of symptoms called acute exacerbations, which may be caused by infections or air pollution.</div>
</li> <li> <para-num num="[0006]"> </para-num> <div class="description-line" id="p-0007" num="0006">Many of the symptoms of COPD are shared by other respiratory diseases such as asthma, bronchitis, pulmonary fibrosis and tuberculosis. The current gold standard for the diagnosis of COPD requires a lung function tests (spirometry), which is a time consuming and costly procedure which can be only realized by a specialized lung physician. A spirometry test, for example, is highly dependent on patient cooperation and effort, and is normally repeated at least three times to ensure reproducibility. In some cases, chronic bronchitis can be diagnosed by asking the patient whether they have a “productive cough” i.e. one that yields sputum.</div>
</li> <li> <para-num num="[0007]"> </para-num> <div class="description-line" id="p-0008" num="0007">Asthma differs from COPD in its pathogenic and therapeutic response, and should therefore be considered a different clinical entity. For example, in COPD there is an increase in neutrophils, macrophages and T-lymphocytes (specifically CD8+) in various parts of the lungs is observed, which relate to the degree of airflow limitation (Saetta, M., et al., Am. J. Respir. Crit. Care Med. 157 (1998) 822-826). There may be an increase in eosinophils in some patients, particularly during exacerbations (Saetta, M., et al., Am. J. Respir. Crit. Care Med. 150 (1994) 1646-1652 and Saetta, M., et al., Clin. Exp. Allergy 26 (1996) 766-774). This inflammatory pattern is markedly different from that seen in patients with bronchial asthma. Inflammatory changes may persist after quitting smoking. The mechanisms explaining the perpetuation of this inflammatory response in the absence of the inciting events are unknown.</div>
</li> <li> <para-num num="[0008]"> </para-num> <div class="description-line" id="p-0009" num="0008">However, some patients with asthma develop poor reversible airflow limitation, which may be indistinguishable from patients with COPD but for practical purposes are treated as asthma. The high prevalence of asthma and COPD in the general population results in the co-existence of both disease entities in many individuals. This is characterised by significant airflow limitation and a large response to bronchodilators. In these patients, the forced expiratory volume in one second (FEV1) does not return to normal and frequently worsens over time.</div>
</li> <heading id="h-0004">SUMMARY OF THE DISCLOSURE</heading>
<li> <para-num num="[0009]"> </para-num> <div class="description-line" id="p-0010" num="0009">The present disclosure relates to an in vitro method aiding in the assessment of chronic obstructive pulmonary disease (=COPD). It discloses the use of the protein ARMET as a marker of COPD. Furthermore, it especially relates to a method for assessing COPD from a sample, derived from an individual by measuring the protein ARMET in said sample in vitro.</div>
</li> <li> <para-num num="[0010]"> </para-num> <div class="description-line" id="p-0011" num="0010">As disclosed here, It has now been found that the use of protein ARMET can at least partially overcome some of the problems of the methods available for assessment of COPD presently known. Ssurprisingly it was found in the present disclosure that an in vitro determination of the concentration of protein ARMET in a sample allows for the assessment of COPD. In this context it was found that an elevated concentration of said protein ARMET in such sample obtained from an individual compared to a reference concentration for protein ARMET is indicative for the presence of COPD.</div>
</li> <li> <para-num num="[0011]"> </para-num> <div class="description-line" id="p-0012" num="0011">Disclosed herein is an in vitro method for assessing COPD comprising determining in a body fluid sample the concentration of protein ARMET by an immunological detection method and using the determined result, particularly the concentration determined, in the assessment of COPD.</div>
</li> <li> <para-num num="[0012]"> </para-num> <div class="description-line" id="p-0013" num="0012">The disclosure also relates to an in vitro method for assessing chronic obstructive pulmonary disease (COPD) in a subject, comprising a) determining the concentration of protein ARMET in a sample, and b) comparing the concentration of protein ARMET determined in step (a) with a reference concentration of protein ARMET, wherein a concentration of protein ARMET above a reference concentration is indicative for COPD.</div>
</li> <li> <para-num num="[0013]"> </para-num> <div class="description-line" id="p-0014" num="0013">In a further embodiment the present disclosure relates to the use of the protein ARMET in the in vitro assessment of COPD in a sample, wherein a concentration of protein ARMET above a reference concentration for protein ARMET is indicative for COPD. Further disclosed is the use of a marker panel comprising protein ARMET and one or more other marker for COPD in the in vitro assessment of COPD in a sample, wherein a concentration of protein ARMET above a reference concentration for protein ARMET is indicative for COPD.</div>
</li> <li> <para-num num="[0014]"> </para-num> <div class="description-line" id="p-0015" num="0014">In a further embodiment the present disclosure relates to the use of the in vitro method for assessing COPD according to the present disclosure to differentiate COPD from other types of lung diseases, such as asthma.</div>
</li> <li> <para-num num="[0015]"> </para-num> <div class="description-line" id="p-0016" num="0015">In a further embodiment the present disclosure relates to a diagnostic device for carrying out the in vitro method for assessing COPD according to the present disclosure.</div>
</li> <li> <para-num num="[0016]"> </para-num> <div class="description-line" id="p-0017" num="0016">Also provided is a kit for performing the in vitro method for assessing COPD according to the present disclosure comprising the reagents required to specifically determine the concentration of protein ARMET.</div>
</li> <li> <para-num num="[0017]"> </para-num> <div class="description-line" id="p-0018" num="0017">Additional aspects and advantages of the present disclosure will be apparent in view of the detailed description which follows. It should be understood, however, that the detailed description and the specific examples, while describing exemplary embodiments of the disclosure, are given by way of illustration only, since various changes and modifications within the spirit and scope of the disclosure will become apparent to those skilled in the art from this detailed description.</div>
</li> <description-of-drawings>
<heading id="h-0005">BRIEF DESCRIPTION OF THE DRAWINGS</heading>
<li> <para-num num="[0018]"> </para-num> <div class="description-line" id="p-0019" num="0018">The features of this disclosure, and the manner of attaining them, will become more apparent and the disclosure itself will be better understood by reference to the following description of embodiments of the disclosure taken in conjunction with the accompanying drawing.</div>
</li> <li> <para-num num="[0019]"> </para-num> <div class="description-line" id="p-0020" num="0019"> <figref idrefs="DRAWINGS">FIG. 1</figref> shows the plot of the receiver operator characteristics (ROC-plot) of protein ARMET in COPD samples with an AUC of 0.82 (ROC 82%) for the assessment of 123 samples obtained from patients with COPD as compared to 186 control samples obtained from healthy control patients (X-axis: 1-specificity (false positive); Y-axis: sensitivity (true positive)).</div>
</li> <li> <para-num num="[0020]"> </para-num> <div class="description-line" id="p-0021" num="0020"> <figref idrefs="DRAWINGS">FIG. 2</figref> shows the plot of the receiver operator characteristics (ROC-plot) of CRP in COPD samples with an AUC of 0.74 (ROC 74%) for the assessment of 123 samples obtained from patients with COPD as compared to 186 control samples obtained from healthy control patients (X-axis: 1-specificity (false positive); Y-axis: sensitivity (true positive)).</div>
</li> <li> <para-num num="[0021]"> </para-num> <div class="description-line" id="p-0022" num="0021"> <figref idrefs="DRAWINGS">FIG. 3</figref> shows the box blot distribution of the determined ARMET serum concentration values according to the COPD stages 0-IV of the 123 COPD samples (COPD stadium as described in Table 1).</div>
</li> <li> <para-num num="[0022]"> </para-num> <div class="description-line" id="p-0023" num="0022"> <figref idrefs="DRAWINGS">FIG. 4</figref> shows the box plot distribution of the determined CRP serum concentration according to the COPD stages 0-IV of the 123 COPD samples (COPD stadium as shown in Table 1).</div>
</li> <li> <para-num num="[0023]"> </para-num> <div class="description-line" id="p-0024" num="0023"> <figref idrefs="DRAWINGS">FIG. 5</figref> shows the plot of the receiver operator characteristics (ROC-plot) of protein ARMET in COPD samples with an AUC of 0.80 (ROC 80%) for the assessment of 123 samples obtained from patients with COPD as compared to 26 control samples obtained from patients with asthma (X-axis: 1-specificity (false positive); Y-axis: sensitivity (true positive)).</div>
</li> <li> <para-num num="[0024]"> </para-num> <div class="description-line" id="p-0025" num="0024"> <figref idrefs="DRAWINGS">FIG. 6</figref> shows the plot of the receiver operator characteristics (ROC-plot) of CRP in COPD samples with an AUC of 0.70 (ROC 70%) for the assessment of 123 samples obtained from patients with COPD as compared to 26 control samples obtained from patients with asthma (X-axis: 1-specificity (false positive); Y-axis: sensitivity (true positive)).</div>
</li> <li> <para-num num="[0025]"> </para-num> <div class="description-line" id="p-0026" num="0025"> <figref idrefs="DRAWINGS">FIG. 7</figref> shows a box plot distribution of the determined ARMET serum concentration [ng/ml] according to 123 COPD samples of stadium 0-IV (4_COPD), 50 healthy (1_Healthy), 135 screening controls (2_screening control) and 26 asthma patient samples (3_Asthma) (the y-axis is adjusted for better ‘visualization’).</div>
</li> <li> <para-num num="[0026]"> </para-num> <div class="description-line" id="p-0027" num="0026"> <figref idrefs="DRAWINGS">FIG. 8</figref> shows the plot of the receiver operator characteristics (ROC-plot) of protein ARMET in COPD samples for ARMET (solid line), ARMET+NNMT (dashed line) and ARMET+NNMT+Seprase (dotted line) marker combinations for the assessment of 123 samples obtained from patients with COPD as compared to 161 control samples obtained from healthy control and asthma patients (X-axis: 1-specificity (false positive); Y-axis: sensitivity (true positive)).</div>
</li> </description-of-drawings>
<li> <para-num num="[0027]"> </para-num> <div class="description-line" id="p-0028" num="0027">Although the drawings represent embodiments of the present disclosure, the drawings are not necessarily to scale and certain features may be exaggerated in order to better illustrate and explain the present disclosure. The exemplifications set out herein illustrate an exemplary embodiment of the disclosure, in one form, and such exemplifications are not to be construed as limiting the scope of the disclosure in any manner.</div>
</li> <heading id="h-0006">BRIEF DESCRIPTION OF THE SEQUENCE LISTING</heading>
<li> <para-num num="[0028]"> </para-num> <div class="description-line" id="p-0029" num="0028">SEQ ID NO.1: is the amino acid sequence of the human protein ASC (SwissProt database accession number: Q9ULZ3).</div>
</li> <li> <para-num num="[0029]"> </para-num> <div class="description-line" id="p-0030" num="0029">SEQ ID NO.2: is the amino acid sequence of the human protein ARMET (SwissProt database accession number: P55145).</div>
</li> <li> <para-num num="[0030]"> </para-num> <div class="description-line" id="p-0031" num="0030">SEQ ID NO.3: is the amino acid sequence of the human protein NNMT (SwissProt database accession number: P40261).</div>
</li> <li> <para-num num="[0031]"> </para-num> <div class="description-line" id="p-0032" num="0031">SEQ ID NO.4: is the amino acid sequence of the human protein FEN1 (SwissProt database accession number: P39748).</div>
</li> <li> <para-num num="[0032]"> </para-num> <div class="description-line" id="p-0033" num="0032">SEQ ID NO.5: is the amino acid sequence of the human protein APEX1 (SwissProt database accession number: P27695).</div>
</li> <li> <para-num num="[0033]"> </para-num> <div class="description-line" id="p-0034" num="0033">SEQ ID NO.6: is the amino acid sequence of the human protein Seprase (SwissProt database accession number: Q12884).</div>
</li> <li> <para-num num="[0034]"> </para-num> <div class="description-line" id="p-0035" num="0034">SEQ ID NO.7: is the amino acid sequence of the human protein DPPIV (SwissProt database accession number: P27487).</div>
</li> <li> <para-num num="[0035]"> </para-num> <div class="description-line" id="p-0036" num="0035">SEQ ID NO.8: is the Forward primer LC56for-EcoRI.</div>
</li> <li> <para-num num="[0036]"> </para-num> <div class="description-line" id="p-0037" num="0036">SEQ ID NO.9: is the Reverse primer LC56for-EcoRI.</div>
</li> <li> <para-num num="[0037]"> </para-num> <div class="description-line" id="p-0038" num="0037">SEQ ID NO.10: is the N-terminal peptide extension.</div>
</li> <li> <para-num num="[0038]"> </para-num> <div class="description-line" id="p-0039" num="0038">Although the sequence listing represents an embodiment of the present disclosure, the sequence listing is not to be construed as limiting the scope of the disclosure in any manner and may be modified in any manner as consistent with the instant disclosure and as set forth herein.</div>
</li> <heading id="h-0007">DETAILED DESCRIPTION OF THE DISCLOSURE</heading>
<li> <para-num num="[0039]"> </para-num> <div class="description-line" id="p-0040" num="0039">The embodiments disclosed herein are not intended to be exhaustive or limit the disclosure to the precise form disclosed in the following detailed description. Rather, the embodiments are chosen and described so that others skilled in the art may utilize their teachings.</div>
</li> <li> <para-num num="[0040]"> </para-num> <div class="description-line" id="p-0041" num="0040">The inventors of the present disclosure have surprisingly been able to demonstrate that the marker protein ARMET is useful in the assessment of COPD. Due to the uncertainties of classifying the various stages of lung damage, and especially of COPD by state of the art methods, it may well be that the protein ARMET may become one of the pivotal criteria in the assessment of patients with COPD in the future. The present disclosure provides a simple and cost-efficient procedure of COPD assessments, e.g. to identify individuals suspected of having COPD. For this purpose, a general COPD marker present in the circulation which is detectable in body fluids (e.g. blood, serum or plasma) is utilized.</div>
</li> <li> <para-num num="[0041]"> </para-num> <div class="description-line" id="p-0042" num="0041">Whole blood, serum or plasma are the most widely used sources of sample in clinical routine. The identification of an early COPD marker that would aid in the reliable COPD detection or provide early prognostic information could lead to a method that would greatly aid in the diagnosis and in the management of this disease. It is especially important to improve the early diagnosis of COPD, since for patients diagnosed in early stages of CODP the chances of reversibility of lung damages are much higher as compared to those patients diagnosed at a more progressed stage of disease.</div>
</li> <li> <para-num num="[0042]"> </para-num> <div class="description-line" id="p-0043" num="0042">The instant disclosure provides a reliable and straightforward indicator of the COPD disease state (for example, a surrogate marker) both in order to reliably distinguish the symptoms of COPD from those of the above mentioned other respiratory diseases, to predict changes in disease severity, disease progression and response to medicine. The diagnostic sensitivity or specificity of a test according to the instant disclosure a test can be assessed by its receiver-operating characteristics, which is described in detail below.</div>
</li> <li> <para-num num="[0043]"> </para-num> <div class="description-line" id="p-0044" num="0043">The method of the present disclosure is suitable for the assessment of COPD. Increased concentrations of protein ARMET in a sample as compared to normal controls have been found to be indicative of COPD.</div>
</li> <li> <para-num num="[0044]"> </para-num> <div class="description-line" id="p-0045" num="0044">In one embodiment the present disclosure relates to an in vitro method for assessing chronic obstructive pulmonary disease (COPD) in a subject, comprising a) determining the concentration of protein ARMET in a sample, and b) comparing the concentration of protein ARMET determined in step (a) with a reference concentration of protein ARMET, wherein a concentration of protein ARMET above a reference concentration is indicative for COPD.</div>
</li> <li> <para-num num="[0045]"> </para-num> <div class="description-line" id="p-0046" num="0045">In a further embodiment the present disclosure relates to an in vitro method for assessing chronic obstructive pulmonary disease (COPD) in a subject, comprising a) determining the concentration of protein ARMET in a body fluid sample, and b) comparing the concentration of protein ARMET determined in step (a) with a reference concentration of protein ARMET, wherein a concentration of protein ARMET above a reference concentration is indicative for the presence of COPD.</div>
</li> <li> <para-num num="[0046]"> </para-num> <div class="description-line" id="p-0047" num="0046">ASC, the “apoptosis-associated speck-like protein containing a caspase-associated recruitment domain” is also known as “target of methylation-induced silencing 1” (TMS1) (Swiss-PROT: Q9ULZ3). The ASC protein in the sense of the present disclosure, characterized by the sequence given in SEQ ID NO:1, is a 22 kDa protein. Caspase-associated recruitment domains (CARDs) mediate the interaction between adaptor proteins such as APAF1 (apoptotic protease activating factor 1) and the pro-form of caspases (e.g., CASP 9) participating in apoptosis. ASC is a member of the CARD-containing adaptor protein family. In WO 2006/105252 is has been shown, that the gene expression level of ASC (=CARD-9) is indicative for the diagnosis of COPD.</div>
</li> <li> <para-num num="[0047]"> </para-num> <div class="description-line" id="p-0048" num="0047">The biological role and function of ARMET (arginine-rich, mutated in early stage tumors, ARP, Swiss-PROT ID: P55145) protein remains largely elusive. The ARMET protein in the sense of the present disclosure, characterized by the sequence given in SEQ ID NO:2, is a 20.3 kDa protein. The ARMET protein consists of 179 amino acids, and carries a predicted signal sequence (aa 1-21). The corresponding gene is located in chromosomal band 3p21.1 and was first characterized by Shridhar, V., et al., (Oncogene 12 (1996) 1931-1939). The gene is highly conserved and can be found many mammalian species, like rat, mouse, cow, and hamster. ARMET was named as such, because initial studies suggested ARMET to be 50 amino acids longer at the N-terminus carrying an arginine-rich region (Shridhar, V., et al., Oncogene 12 (1996) 1931-1939; Shridhar, R., et al., Cancer Res. 56 (1996) 5576-5578; Shridhar, V., et al., Oncogene 14 (1997) 2213-2216). However, more recent studies indicate transcribed evidence for a smaller open reading frame that does not encode the arginine tract (Tanaka, H., et al., Oncol. Rep. 7 (2000) 591-593; Mizobuchi, N., et al., Cell Struct. Funct. 32 (2007) 41-50). With the corresponding protein size correction, the initially described mutated codon (ATG50) is now identified to be the initiation codon. Petrova, P., et al., (J. Mol. Neurosci. 20 (2003) 173-188) purified the ARMET gene product from conditioned medium of a rat mesencephalic type-1 astrocyte cell line and named it MANF (Mensencephalic Astrocyte-derived Neurotrophic Factor). Most recent studies demonstrated that ARMET is upregulated by the “unfolded protein response” (UPR), a process which is activated once misfolded proteins accumulate in the endoplasmatic reticulum (ER) (Tanaka, H., et al., Oncol. Rep. 7 (2000) 591-593; Apostolou, A., et al., Exp. Cell Res. 314 (2008) 2454-2467). Based on this study ARMET is characterized as a novel secreted mediator of the adaptive pathway of UPR.</div>
</li> <li> <para-num num="[0048]"> </para-num> <div class="description-line" id="p-0049" num="0048">The NNMT (nicotinamide N-methyltransferase; Swiss-PROT: P40261) protein in the sense of the present disclosure, characterized by the sequence given in SEQ ID NO:3, is a 29.6 kDa protein and has an isoelectric point of 5.56. NNMT catalyzes the N-methylation of nicotinamide and other pyridines. This activity is important for biotransformation of many drugs and xenobiotic compounds. The protein has been reported to be predominantly expressed in liver and is located in the cytoplasm. NNMT has been cloned from cDNA from human liver and contained a 792-nucleotide open reading frame that encoded a 264-amino acid protein with a calculated molecular mass of 29.6 kDa. (Aksoy, S., et al., J. Biol. Chem. 269 (1994) 14835-14840). Little is known in the literature about a potential role of the enzyme in human COPD. In the Am. J. of Repiratory and Critical Care Medicine vol. 181 (No. 8), 798-805 a higher mRNA expression of NNMT in skeletal muscle cells of COPD patients has been observed. In a study it has been shown that NNMT is a useful biomarker for lung cancer (LC) (J. of Cancer Res. and Clin. Onc. vol. 136, no. 9, (2009) 1223.1229). In said study it has been found that serum levels of NNMT were significantly higher in LC patients than in COPD patients and healthy donors.</div>
</li> <li> <para-num num="[0049]"> </para-num> <div class="description-line" id="p-0050" num="0049">Flap endonuclease-1 protein (=FEN1, FEN-1), Swiss-PROT ID: P39748 in the sense of the present disclosure, is a nuclear protein of 380 amino acids with a molecular weight of 42.6 kDa, characterized by the sequence given in SEQ ID NO:4. The coding sequence of human FEN1 was predicted by Murray in 1994 (Murray, J. M., et al., Mol. Cell. Biol. 14 (1994) 4878-4888) from a newly cloned sequence. Based on the function of the yeast homolog rad2 a function in high fidelity chromosome segregation and in the repair of UV-induced DNA damage was suggested. As these are fundamental processes in chromosomal integrity, the authors also proposed an involvement of the protein in cancer avoidance. The gene locus on human chromosome 11 was later identified by Hiraoka, et al., (Hiraoka L. R., et al., Genomics 25 (1995) 220-225) and Taylor, et al., (Taylor, T. D., et al., Nature 440 (2006) 497-500). The functions of FEN1 and its interactions with DNA have been the focus of numerous studies (Robins, P., et al., J. Biol. Chem. 269 (1994) 28535-28538), Shen, B., et al., J. Biol. Chem. 271 (1996) 9173-9176; Hasan, S., et al., Mol. Cell 7 (2001) 1221-1231; Qiu, J., et al., J. Biol. Chem. 277 (2002) 24659-24666 and Sakurai, S., et al., EMBO J. 24 (2005) 683-693). Several enzymatic functions in DNA metabolism have been demonstrated including endonuclease activity that cleaves the 5′-overhanging flap structure generated by displacement synthesis when DNA polymerase encounters the 5′-end of a downstream Okazaki fragment. Additionally FEN1 also possesses a 5′ to 3′ exonuclease activity on niked or gapped double-stranded DNA, and exhibits RNase H activity. These have been reviewed by Shen et al. (Shen, B., et al., BioEssays 27 (2005) 717-729) or Liu, et al., (Liu, Y., et al., Annu. Rev. Biochem. 73 (2004) 589-615).</div>
</li> <li> <para-num num="[0050]"> </para-num> <div class="description-line" id="p-0051" num="0050">The AP endonuclease (APEX1, APEX-1) (Swiss-Prot. P27695) in the sense of the present disclosure is characterized by the sequence given in SEQ ID NO:5. The unprocessed precursor molecule consists of 318 amino acids and has a molecular weight of 35.6 kDa. APEX1 is involved in DNA repair and excises the apurinic or apyrimidinic site of DNA strands. Such abasic sites are relative frequently generated either spontaneously or through chemical agents or by DNA glycosylases that remove specific abnormal bases.</div>
</li> <li> <para-num num="[0051]"> </para-num> <div class="description-line" id="p-0052" num="0051">AP sites are pre-mutagenic lesions that can prevent normal DNA replication so the cell contains systems to identify and repair such sites. (Barzilay, G., and Hickson, I.D., Bioessays 17 (1995) 713-719). The 3D structure was elucidated and the amino acids involved in endonuclease activity were identified (Barizilay, G., et al., Nature Structural Biology 2 (1995) 561-567; Gorman, M. A., et al., EMBO Journal 16 (1997) 6548-6558; Beernink, P., et al., J. Mol. Biol. 307 (2001) pp. 1023-1034). APEX1 is also a redox regulator of various transcription factors such as c-Fos, c-Jun, NF-KB and HIF-1. This activity seems to be independent from the endonuclease activity. Both functions are located on different domains of the protein (Barzilay, G., and Hickson, I.D., Bioessays 17 (1995) 713-719). Phosphorylation of APEX1 by protein kinase C increases redox activity whereas the unphosphorylated form is involved in DNA-repair (Yacoub, A., et al., Cancer Res. 57 (1997) 5457-5459). One phosphorylation site, Y 261, (according to the Swissprot sequence) was identified by Rush, J., et al., Nature Biotech. 23 (2005) 94-101).</div>
</li> <li> <para-num num="[0052]"> </para-num> <div class="description-line" id="p-0053" num="0052">Seprase, also known as fibroblast activation protein (=FAP), in the sense of the present disclosure is as a 170 kDa glycoprotein having gelatinase and dipeptidyl peptidase activity consisting of two identical monomeric Seprase units (Pineiro-Sanchez, M. L., et al., J. Biol. Chem. 272 (1997) 7595-7601; Park, J. E., et al., J. Biol. Chem. 274 (1999) 36505-36512). The monomer of the human membrane bound Seprase protein comprises 760 amino acids and is shown in SEQ ID NO: 6. Human Seprase is predicted to have its first 4 N-terminal residues within the fibroblast cytoplasm, followed by a 21-residue transmembrane domain and then a 734 residue extracellular C-terminal catalytic domain (Goldstein, L. A., et al., Biochim. Biophys. Acta. 1361 (1997) 11-19; Scanlan, M. J., et al., Proc. Natl. Acad. Sci. USA 91 (1994) 5657-5661). A shorter form of human Seprase protein is known to a person skilled in the art as soluble Seprase or circulating antiplasmin-cleaving enzyme (=APCE) (Lee, K. N., et al., Blood 103 (2004) 3783-3788; Lee, K. N., et al., Blood 107 (2006) 1397-1404), comprising the amino acid positions 26-760 from Swissprot database Accession number Q12884. The dimer of soluble Seprase is a 160 kDa glycoprotein consisting of two identical monomeric soluble Seprase protein units. Piñeiro-Sánchez et al. (supra) found that a increased expression of Seprase correlates with the invasive phenotype of human melanoma and carcinoma cells. Henry, L. R., et al., Clin. Cancer Res. 13 (2007) 1736-1741 describe that human colon tumor patients having high levels of stromal Seprase are more likely to have aggressive disease progression and potential development of metastases or recurrence.</div>
</li> <li> <para-num num="[0053]"> </para-num> <div class="description-line" id="p-0054" num="0053">Human dipeptidyl peptidase IV (=DPPIV), which is also known as CD26, is in the sense of the present disclosure a 110 kDa cell surface molecule. The amino acid sequence of human DPPIV protein comprises 766 amino acids and is shown in SEQ ID NO: 7 (Swissprot database Accession No. P27487). It contains intrinsic dipeptidyl peptidase IV activity which selectively removes N-terminal dipeptide from peptides with proline or alanine in the third amino acid position. It interacts with various extracellular molecules and is also involved in intracellular signal transduction cascades. The multifunctional activities of human DPPIV are dependent on cell type and intracellular or extracellular conditions that influence its role as a proteolytic enzyme, cell surface receptor, co-stimulatory interacting protein and signal transduction mediator. Human DPPIV has a short cytoplasmatic domain from amino acid position 1 to 6, a transmembrane region from amino acid position 7 to 28, and an extracellular domain from amino acid position 29 to 766 with intrinsic dipeptidyl peptidase IV (DPPIV) activity. Human soluble dipeptidyl peptidase IV (=soluble DPPIV) amino acid sequence comprises the amino acid positions 29 to 766 from Swissprot database Accession number P27487. The dimer of soluble DPPIV is a 170 kDa glycoprotein consisting of two identical monomeric soluble DPPIV units.</div>
</li> <li> <para-num num="[0054]"> </para-num> <div class="description-line" id="p-0055" num="0054">The “soluble DPPIV/Seprase protein complex” (=DPPIV/Seprase) in the sense of the present disclosure refers to the soluble complex formed of a soluble DPPIV homodimer (170 kDa) and a soluble Seprase homodimer (160 kDa) with a molecular weight of 330 kDa. Under certain conditions this complex may form a double complex having a molecular weight of 660 kDa.</div>
</li> <li> <para-num num="[0055]"> </para-num> <div class="description-line" id="p-0056" num="0055">As obvious to the skilled artisan, the present disclosure shall not be construed to be limited to the full-length protein ARMET of SEQ ID NO:2. Physiological or artificial fragments of protein ARMET, secondary modifications of protein ARMET, as well as allelic variants of protein ARMET are also encompassed by the present disclosure. Variants of a polypeptide are encoded by the same gene, but may differ in their isoelectric point (=PI) or molecular weight (=MW), or both e.g., as a result of alternative mRNA or pre-mRNA processing. The amino acid sequence of a variant is to 95% or more identical to the corresponding marker sequence. Artificial fragments may encompass a peptide produced synthetically or by recombinant techniques, which at least comprises one epitope of diagnostic interest consisting of at least 6, 7, 8, 9 or 10 contiguous amino acids as derived from the sequence disclosed in SEQ ID NO:1. Such fragment may advantageously be used for generation of antibodies or as a standard in an immunoassay.</div>
</li> <li> <para-num num="[0056]"> </para-num> <div class="description-line" id="p-0057" num="0056">The inventors of the present disclosure have now found and could establish that an increased concentration for protein ARMET as determined from a body fluid sample derived from an individual is indicative for COPD.</div>
</li> <li> <para-num num="[0057]"> </para-num> <div class="description-line" id="p-0058" num="0057">The practicing of the present disclosure will employ, unless otherwise indicated, conventional techniques of molecular biology (including recombinant techniques), microbiology, cell biology, biochemistry, and immunology, which are within the skill of the art. Such techniques are explained fully in the literature, such as, Sambrook, et al., Molecular Cloning: A Laboratory Manual, second edition, (1989); Gait, M. J., (ed.) Oligonucleotide Synthesis (1984); Freshney, R. I., (ed.), Animal Cell Culture (1987); Methods in Enzymology (Academic Press, Inc.); Ausubel, F. M., et al., (eds.), Current Protocols in Molecular Biology (1987) and periodic updates; Mullis, et al., (eds.) PCR: The Polymerase Chain Reaction (1994).</div>
</li> <li> <para-num num="[0058]"> </para-num> <div class="description-line" id="p-0059" num="0058">Unless defined otherwise, technical and scientific terms used herein have the same meaning as commonly understood by one of ordinary skill in the art to which this disclosure belongs. Singleton, et al., Dictionary of Microbiology and Molecular Biology, 2nd ed., John Wiley &amp; Sons, New York, N.Y. (1994); March, Advanced Organic Chemistry Reactions, Mechanisms and Structure, 4th ed., John Wiley &amp; Sons, New York, N.Y. (1992); Lewin, B., Genes V, published by Oxford University Press (1994), ISBN 0-19-854287 9; Kendrew, J., et al., (eds.), The Encyclopedia of Molecular Biology, published by Blackwell Science Ltd. (1994), ISBN 0-632-02182-9; and Meyers, R. A., (ed.), Molecular Biology and Biotechnology: a Comprehensive Desk Reference, published by VCH Publishers, Inc. (1995), ISBN 1-56081-569 8) provide one skilled in the art with a general guide to many of the terms used in the present application.</div>
</li> <li> <para-num num="[0059]"> </para-num> <div class="description-line" id="p-0060" num="0059">As used herein, each of the following terms has the meaning associated with it in this section.</div>
</li> <li> <para-num num="[0060]"> </para-num> <div class="description-line" id="p-0061" num="0060">The articles “a” and “an” are used herein to refer to one or to more than one (i.e. to at least one) of the grammatical object of the article. By way of example, “a marker” means one marker or more than one marker. The term “at least” is used to indicate that optionally one or more than one further objects may be present.</div>
</li> <li> <para-num num="[0061]"> </para-num> <div class="description-line" id="p-0062" num="0061">The expression “one or more” denotes 1 to 50, for example 1 to 20 or also 2, 3, 4, 5, 6, 7, 8, 9, 10, 12, or 15.</div>
</li> <li> <para-num num="[0062]"> </para-num> <div class="description-line" id="p-0063" num="0062">The term “marker” or “biochemical marker” as used herein refers to a molecule to be used as a target for analyzing an individual's test sample. In one embodiment examples of such molecular targets are proteins or polypeptides. Proteins or polypeptides used as a marker in the present disclosure are contemplated to include naturally occurring variants of said protein as well as fragments of said protein or said variant, in particular, immunologically detectable fragments. Immunologically detectable fragments may comprise at least 6, 7, 8, 10, 12, 15 or 20 contiguous amino acids of said marker polypeptide. One of skill in the art would recognize that proteins which are released by cells or present in the extracellular matrix may be damaged, e.g., during inflammation, and could become degraded or cleaved into such fragments. Certain markers are synthesized in an inactive form, which may be subsequently activated by proteolysis. As the skilled artisan will appreciate, proteins or fragments thereof may also be present as part of a complex. Such complex also may be used as a marker in the sense of the present disclosure. In addition, or in the alternative a marker polypeptide or a variant thereof may carry a post-translational modification. Exemplary posttranslational modifications are glycosylation, acylation, or phosphorylation.</div>
</li> <li> <para-num num="[0063]"> </para-num> <div class="description-line" id="p-0064" num="0063">The term “label” as used herein refers to any substance that is capable of producing a signal via direct or indirect detection. For direct detection the labeling group or label suitable for use in the present disclosure can be selected from any known detectable marker groups, but are not limited to, chromogens, fluorescent, chemiluminescent groups (e.g. acridinium esters or dioxetanes), electrochemiluminescent compounds, catalysts, enzymes, enzymatic substrates, dyes, fluorescent dyes (e.g. fluorescein, coumarin, rhodamine, oxazine, resorufin, cyanine and derivatives thereof), colloidal metallic and nonmetallic particles, and organic polymer latex particles. Other examples of labeling groups are luminescent metal complexes, such as ruthenium or europium complexes, enzymes, e.g. as used for ELISA, and radioisotopes.</div>
</li> <li> <para-num num="[0064]"> </para-num> <div class="description-line" id="p-0065" num="0064">Indirect detection systems comprise, for example, that the detection reagent, e.g. the detection antibody, is labeled with a first partner of a bioaffine binding pair. Examples of suitable binding pairs are hapten or antigen/antibody, biotin or biotin analogues such as aminobiotin, iminobiotin or desthiobiotin/avidin or streptavidin, sugar/lectin, nucleic acid or nucleic acid analogue/complementary nucleic acid, and receptor/ligand, e.g. steroid hormone receptor/steroid hormone. Exemplary first binding pair members comprise hapten, antigen and hormone. Exemplary haptens include digoxin and biotin and analogues thereof. The second partner of such binding pair, e.g. an antibody, streptavidin, etc., usually is labeled to allow for direct detection, e.g. by the labels as mentioned above.</div>
</li> <li> <para-num num="[0065]"> </para-num> <div class="description-line" id="p-0066" num="0065">The term “assessing chronic obstructive pulmonary disease” or “assessing COPD” is used to indicate that the method according to the present disclosure will alone or together with other markers or variables, e.g., aid the physician to establish or confirm the absence or presence of COPD. The method will e.g. be useful to establish or confirm the absence or presence of COPD.</div>
</li> <li> <para-num num="[0066]"> </para-num> <div class="description-line" id="p-0067" num="0066">A “marker for COPD” in the sense of the present disclosure is a marker that, as single marker, or if combined with the marker ARMET, adds relevant information in the assessment of COPD to the diagnostic question under investigation. The information is considered relevant or of additive value if at a given specificity the sensitivity, or if at a given sensitivity the specificity, respectively, for the assessment of COPD can be improved by including said marker into a marker panel (marker combination) already comprising the marker ARMET. In at least some embodiments, the improvement in sensitivity or specificity, respectively, is statistically significant at a level of significance of p=0.05, 0.02, 0.01 or lower.</div>
</li> <li> <para-num num="[0067]"> </para-num> <div class="description-line" id="p-0068" num="0067">The term “sample” or “test sample” as used herein refers to a biological sample obtained from an individual for the purpose of evaluation in vitro. In the methods of the present disclosure, the sample or patient sample may comprise in an embodiment of the present disclosure any body fluid. Exemplary samples are body fluids, such as serum, plasma, or whole blood.</div>
</li> <li> <para-num num="[0068]"> </para-num> <div class="description-line" id="p-0069" num="0068">Protein ARMET, particularly soluble forms of protein ARMET, are determined in vitro in an appropriate sample. For example, the sample is derived from a human subject, e.g. a COPD patient or a person in risk of COPD or a person suspected of having COPD. Also, in some embodiments, protein ARMET is determined in a serum or plasma sample.</div>
</li> <li> <para-num num="[0069]"> </para-num> <div class="description-line" id="p-0070" num="0069">The term “reference sample” as used herein refers to a biological sample provided from a reference group of apparently healthy individuals for the purpose of evaluation in vitro. The term “reference concentration” as used herein refers to a value established in a reference group of apparently healthy individuals.</div>
</li> <li> <para-num num="[0070]"> </para-num> <div class="description-line" id="p-0071" num="0070">It is known to a person skilled in the art that the measurement results of step (a) according to the method(s) of the present disclosure will be compared to a reference concentration. Such reference concentration can be determined using a negative reference sample, a positive reference sample, or a mixed reference sample comprising one or more than one of these types of controls. A negative reference sample may comprise a sample from non smokers, control smokers with no diagnosis of COPD, asthma or various combinations thereof, for example. In at least some embodiments, a positive reference sample comprises a sample from a subject with the diagnosis of COPD.</div>
</li> <li> <para-num num="[0071]"> </para-num> <div class="description-line" id="p-0072" num="0071">The expression “comparing the concentration determined to a reference concentration” is merely used to further illustrate what is obvious to the skilled artisan anyway. A reference concentration is established in a control sample. The control sample may be an internal or an external control sample. In one embodiment an internal control sample is used, i.e. the marker level(s) is(are) assessed in the test sample as well as in one or more other sample(s) taken from the same subject to determine if there are any changes in the level(s) of said marker(s). In another embodiment an external control sample is used. For an external control sample the presence or amount of a marker in a sample derived from the individual is compared to its presence or amount in an individual known to suffer from, or known to be at risk of, a given condition; or an individual known to be free of a given condition, i.e., “normal individual”. For example, a marker level in a patient sample can be compared to a level known to be associated with a specific course of COPD. Usually the sample's marker level is directly or indirectly correlated with a diagnosis and the marker level is e.g. used to determine whether an individual is at risk for COPD. Alternatively, the sample's marker level can e.g. be compared to a marker level known to be associated with a response to therapy in COPD patients, the diagnosis of COPD, the guidance for selecting an appropriate drug to COPD, in judging the risk of disease progression, or in the follow-up of COPD patients. Depending on the intended diagnostic use an appropriate control sample is chosen and a control or reference value for the marker established therein. It will be appreciated by the skilled artisan that such control sample in one embodiment is obtained from a reference population that is age-matched and free of confounding diseases. As also clear to the skilled artisan, the absolute marker values established in a control sample will be dependent on the assay used. In some embodiments, samples from 100 well-characterized individuals from the appropriate reference population may be used to establish a control (reference) value. Also, the reference population may be chosen to consist of 20, 30, 50, 200, 500 or 1000 individuals. Healthy individuals represent a reference population for establishing a control value.</div>
</li> <li> <para-num num="[0072]"> </para-num> <div class="description-line" id="p-0073" num="0072">The term “measurement”, “measuring” or “determining” comprise a qualitative, a semi-quanitative or a quantitative measurement. In the present disclosure protein ARMET is measured in a body fluid sample. In an exemplary embodiment the measurement is a semi-quantitative measurement, i.e. it is determined whether the concentration of protein ARMET is above or below a cut-off value. As the skilled artisan will appreciate, in a Yes—(presence) or No—(absence) assay, the assay sensitivity is usually set to match the cut-off value.</div>
</li> <li> <para-num num="[0073]"> </para-num> <div class="description-line" id="p-0074" num="0073">The values for protein ARMET as determined in a control group or a control population are for example used to establish a cut-off value or a reference range. A value above such cut-off value or out-side the reference range at its higher end is considered as elevated or as indicative for the presence of COPD.</div>
</li> <li> <para-num num="[0074]"> </para-num> <div class="description-line" id="p-0075" num="0074">In an embodiment a fixed cut-off value is established. Such cut-off value is chosen to match the diagnostic question of interest.</div>
</li> <li> <para-num num="[0075]"> </para-num> <div class="description-line" id="p-0076" num="0075">In an embodiment, the cut-off is set to result in a specificity of 90%, or in some cases the cut-off is set to result in a specificity of 95%, or even set to result in a specificity of 98%.</div>
</li> <li> <para-num num="[0076]"> </para-num> <div class="description-line" id="p-0077" num="0076">In an embodiment the cut-off is set to result in a sensitivity of 90%, a sensitivity of 95%, or the cut-off is set to result in a sensitivity of 98%.</div>
</li> <li> <para-num num="[0077]"> </para-num> <div class="description-line" id="p-0078" num="0077">In some embodiments, values for protein ARMET as determined in a control group or a control population are used to establish a reference range. In embodiments a concentration of protein ARMET is considered as elevated if the value determined is above the 90%-percentile of the reference range. In further embodiments a concentration of protein ARMET is considered as elevated if the value determined is above the 95%-percentile, the 96%-percentile, the 97%-percentile or the 97.5%-percentile of the reference range.</div>
</li> <li> <para-num num="[0078]"> </para-num> <div class="description-line" id="p-0079" num="0078">A value above the cut-off value can for example be indicative for the presence of COPD. A value below the cut-off value can for example be indicative for the absence of COPD.</div>
</li> <li> <para-num num="[0079]"> </para-num> <div class="description-line" id="p-0080" num="0079">In further embodiments the measurement of protein ARMET is a quantitative measurement. In further embodiments the concentration of protein ARMET is correlated to an underlying diagnostic question.</div>
</li> <li> <para-num num="[0080]"> </para-num> <div class="description-line" id="p-0081" num="0080">A sample provided from a patient with already confirmed COPD in certain settings might be used as a positive control sample and assayed in parallel with the sample to be investigated. In such setting a positive result for the marker protein ARMET in the positive control sample indicates that the testing procedure has worked on the technical level.</div>
</li> <li> <para-num num="[0081]"> </para-num> <div class="description-line" id="p-0082" num="0081">As the skilled artisan will appreciate, any such assessment is made in vitro. The sample (test sample) is discarded afterwards. The sample is solely used for the in vitro diagnostic method of the disclosure and the material of the sample is not transferred back into the patient's body. Typically, the sample is a body fluid sample, e.g., serum, plasma, or whole blood.</div>
</li> <li> <para-num num="[0082]"> </para-num> <div class="description-line" id="p-0083" num="0082">The method according to the present disclosure is based on a liquid or body fluid sample which is obtained from an individual and on the in vitro determination of protein ARMET in such sample. An “individual” as used herein refers to a single human or non-human organism. Thus, the methods and compositions described herein are applicable to both human and veterinary disease. In at least some embodiments, the individual, subject, or patient is a human being.</div>
</li> <li> <para-num num="[0083]"> </para-num> <div class="description-line" id="p-0084" num="0083">According to some embodiments, the marker protein ARMET is specifically determined in vitro from a liquid sample by use of a specific binding agent. In some embodiments according to the present disclosure, the concentration of protein ARMET is determined. In an embodiment, the concentration of marker protein ARMET is specifically determined in vitro from a sample by use of a specific binding agent.</div>
</li> <li> <para-num num="[0084]"> </para-num> <div class="description-line" id="p-0085" num="0084">A specific binding agent is, e.g., a receptor for the protein ARMET, a lectin binding to protein ARMET, an antibody to protein ARMET, peptidebodies to protein ARMET, bispecific dual binders or bispecific antibody formats. A specific binding agent has at least an affinity of 10<sup>7 </sup>l/mol for its corresponding target molecule. The specific binding agent may have an affinity of 10<sup>8 </sup>l/mol or also of 10<sup>9 </sup>l/mol for its target molecule.</div>
</li> <li> <para-num num="[0085]"> </para-num> <div class="description-line" id="p-0086" num="0085">As the skilled artisan will appreciate the term specific is used to indicate that other biomolecules present in the sample do not significantly bind to the binding agent specific for the protein ARMET sequence of SEQ ID NO:2. In some embodiments, the level of binding to a biomolecule other than the target molecule results in a binding affinity which is at most only 10% or less, only 5% or less only 2% or less or only 1% or less of the affinity to the target molecule, respectively. Specific binding agent may fulfill both the above minimum criteria for affinity as well as for specificity.</div>
</li> <li> <para-num num="[0086]"> </para-num> <div class="description-line" id="p-0087" num="0086">Examples of specific binding agents are peptides, peptide mimetics, aptamers, spiegelmers, darpins, ankyrin repeat proteins, Kunitz type domains, antibodies, single domain antibodies, (see: Hey, T., et al., Trends Biotechnol. 23 (2005) 514-522) and monovalent fragments of antibodies. In certain embodiments the specific binding agent is a polypeptide. In certain embodiments the specific binding agent is an antibody or a monovalent antibody fragment, for example a monovalent fragment derived from a monoclonal antibody. Monovalent antibody fragments include, but are not limited to Fab, Fab′-SH, single domain antibody, Fv, and scFv fragments, as provided below.</div>
</li> <li> <para-num num="[0087]"> </para-num> <div class="description-line" id="p-0088" num="0087">The term “antibody” herein is used in the broadest sense and specifically covers monoclonal antibodies, polyclonal antibodies, multispecific antibodies (e.g. bispecific antibodies) formed from at least two intact antibodies, and antibody fragments so long as they exhibit the desired biological activity. In certain embodiments the specific binding agent is an antibody or a monovalent antibody fragment, for example a monovalent fragment derived from a monoclonal antibody.</div>
</li> <li> <para-num num="[0088]"> </para-num> <div class="description-line" id="p-0089" num="0088">An “isolated” antibody is one which has been identified and separated and/or recovered from a component of its natural environment. Contaminant components of its natural environment are materials which would interfere with research, diagnostic or therapeutic uses for the antibody, and may include enzymes, hormones, and other proteinaceous or nonproteinaceous solutes. In some embodiments, an antibody is purified (1) to greater than 95% by weight of antibody as determined by, for example, the Lowry method, and in some embodiments, to greater than 99% by weight; (2) to a degree sufficient to obtain at least 15 residues of N-terminal or internal amino acid sequence by use of, for example, a spinning cup sequenator, or (3) to homogeneity by SDS-PAGE under reducing or nonreducing conditions using, for example, Coomassie blue or silver stain. Isolated antibody includes the antibody in situ within recombinant cells since at least one component of the antibody's natural environment will not be present. Ordinarily, however, isolated antibody will be prepared by at least one purification step.</div>
</li> <li> <para-num num="[0089]"> </para-num> <div class="description-line" id="p-0090" num="0089">“Native antibodies” are usually heterotetrameric glycoproteins of about 150,000 daltons, composed of two identical light (L) chains and two identical heavy (H) chains. Each light chain is linked to a heavy chain by one covalent disulfide bond, while the number of disulfide linkages varies among the heavy chains of different immunoglobulin isotypes. Each heavy and light chain also has regularly spaced intrachain disulfide bridges. Each heavy chain has at one end a variable domain (VH) followed by a number of constant domains. Each light chain has a variable domain at one end (VL) and a constant domain at its other end; the constant domain of the light chain is aligned with the first constant domain of the heavy chain, and the light-chain variable domain is aligned with the variable domain of the heavy chain. Particular amino acid residues are believed to form an interface between the light-chain and heavy-chain variable domains.</div>
</li> <li> <para-num num="[0090]"> </para-num> <div class="description-line" id="p-0091" num="0090">The “variable region” or “variable domain” of an antibody refers to the amino-terminal domains of the heavy or light chain of the antibody. The variable domain of the heavy chain may be referred to as “VH.” The variable domain of the light chain may be referred to as “VL.” These domains are generally the most variable parts of an antibody and contain the antigen-binding sites.</div>
</li> <li> <para-num num="[0091]"> </para-num> <div class="description-line" id="p-0092" num="0091">The term “variable” refers to the fact that certain portions of the variable domains differ extensively in sequence among antibodies and are used in the binding and specificity of each particular antibody for its particular antigen. However, the variability is not evenly distributed throughout the variable domains of antibodies. It is concentrated in three segments called hypervariable regions (HVRs) both in the light-chain and the heavy-chain variable domains. The more highly conserved portions of variable domains are called the framework regions (FR). The variable domains of native heavy and light chains each comprise four FR regions, largely adopting a beta-sheet configuration, connected by three HVRs, which form loops connecting, and in some cases forming part of, the beta-sheet structure. The HVRs in each chain are held together in close proximity by the FR regions and, with the HVRs from the other chain, contribute to the formation of the antigen-binding site of antibodies (see Kabat, et al., Sequences of Proteins of Immunological Interest, 5th ed., National Institute of Health, Bethesda, Md. (1991)). The constant domains are not involved directly in the binding of an antibody to an antigen, but exhibit various effector functions, such as participation of the antibody in antibody-dependent cellular toxicity.</div>
</li> <li> <para-num num="[0092]"> </para-num> <div class="description-line" id="p-0093" num="0092">The “light chains” of antibodies (immunoglobulins) from any vertebrate species can be assigned to one of two clearly distinct types, called kappa (κ) and lambda (λ), based on the amino acid sequences of their constant domains.</div>
</li> <li> <para-num num="[0093]"> </para-num> <div class="description-line" id="p-0094" num="0093">Depending on the amino acid sequences of the constant domains of their heavy chains, antibodies (immunoglobulins) can be assigned to different classes. There are five major classes of immunoglobulins: IgA, IgD, IgE, IgG, and IgM, and several of these may be further divided into subclasses (isotypes), e.g., IgG1, IgG2, IgG3, IgG4, IgA1, and IgA2. The subunit structures and three-dimensional configurations of different classes of immunoglobulins are well known and described generally in, for example, Abbas, et al., Cellular and Mol. Immunology, 4th ed., W.B. Saunders, Co. (2000). An antibody may be part of a larger fusion molecule, formed by covalent or non-covalent association of the antibody with one or more other proteins or peptides.</div>
</li> <li> <para-num num="[0094]"> </para-num> <div class="description-line" id="p-0095" num="0094">The terms “full-length antibody,” “intact antibody,” and “whole antibody” are used herein interchangeably to refer to an antibody in its substantially intact form, not antibody fragments as defined below. The terms particularly refer to an antibody with heavy chains that contain an Fc region.</div>
</li> <li> <para-num num="[0095]"> </para-num> <div class="description-line" id="p-0096" num="0095">“Antibody fragments” comprise a portion of an intact antibody, for example comprising the antigen-binding region thereof. Examples of antibody fragments include Fab, Fab′, F(ab′)2, and Fv fragments; diabodies; linear antibodies; single-chain antibody molecules; and multispecific antibodies formed from antibody fragments.</div>
</li> <li> <para-num num="[0096]"> </para-num> <div class="description-line" id="p-0097" num="0096">Papain digestion of antibodies produces two identical antigen-binding fragments, called “Fab” fragments, each with a single antigen-binding site, and a residual “Fc” fragment, whose name reflects its ability to crystallize readily. Pepsin treatment yields a F(ab′)2 fragment that has two antigen-combining sites and is still capable of cross-linking antigen.</div>
</li> <li> <para-num num="[0097]"> </para-num> <div class="description-line" id="p-0098" num="0097">“Fv” is the minimum antibody fragment which contains a complete antigen-binding site. In one embodiment, a two-chain Fv species consists of a dimer of one heavy- and one light-chain variable domain in tight, non-covalent association. In a single-chain Fv (scFv) species, one heavy- and one light-chain variable domain can be covalently linked by a flexible peptide linker such that the light and heavy chains can associate in a “dimeric” structure analogous to that in a two-chain Fv species. It is in this configuration that the three HVRs of each variable domain interact to define an antigen-binding site on the surface of the VH-VL dimer. Collectively, the six HVRs confer antigen-binding specificity to the antibody. However, even a single variable domain (or half of an Fv comprising only three HVRs specific for an antigen) has the ability to recognize and bind antigen, although at a lower affinity than the entire binding site.</div>
</li> <li> <para-num num="[0098]"> </para-num> <div class="description-line" id="p-0099" num="0098">The Fab fragment contains the heavy- and light-chain variable domains and also contains the constant domain of the light chain and the first constant domain (CH1) of the heavy chain. Fab′ fragments differ from Fab fragments by the addition of a few residues at the carboxy terminus of the heavy chain CH1 domain including one or more cysteines from the antibody-hinge region. Fab′-SH is the designation herein for Fab′ in which the cysteine residue(s) of the constant domains bear a free thiol group. F(ab′)2 antibody fragments originally were produced as pairs of Fab′ fragments which have hinge cysteines between them. Other chemical couplings of antibody fragments are also known.</div>
</li> <li> <para-num num="[0099]"> </para-num> <div class="description-line" id="p-0100" num="0099">“Single-chain Fv” or “scFv” antibody fragments comprise the VH and VL domains of an antibody, wherein these domains are present in a single polypeptide chain. Generally, the scFv polypeptide further comprises a polypeptide linker between the VH and VL domains that enables the scFv to form the desired structure for antigen binding. For a review of scFv, see, e.g., Pluckthuen, A., In: The Pharmacology of Monoclonal Antibodies, vol. 113, Rosenburg and Moore (eds.), Springer-Verlag, New York (1994) pp. 269-315.</div>
</li> <li> <para-num num="[0100]"> </para-num> <div class="description-line" id="p-0101" num="0100">The term “diabodies” refers to antibody fragments with two antigen-binding sites, which fragments comprise a heavy-chain variable domain (VH) connected to a light-chain variable domain (VL) in the same polypeptide chain (VH-VL). By using a linker that is too short to allow pairing between the two domains on the same chain, the domains are forced to pair with the complementary domains of another chain and create two antigen-binding sites. Diabodies may be bivalent or bispecific. Diabodies are described more fully in, for example, EP 0404 097; WO 1993/01161; Hudson, et al., Nat. Med. 9 (2003) 129-134; and Hollinger, et al., PNAS USA 90 (1993) 6444-6448. Triabodies and tetrabodies are also described in Hudson, et al., Nat. Med. 9 (2003) 129-134.</div>
</li> <li> <para-num num="[0101]"> </para-num> <div class="description-line" id="p-0102" num="0101">The term “monoclonal antibody” as used herein refers to an antibody obtained from a population of substantially homogeneous antibodies, i.e., the individual antibodies comprising the population are identical except for possible mutations, e.g., naturally occurring mutations, that may be present in minor amounts. Thus, the modifier “monoclonal” indicates the character of the antibody as not being a mixture of discrete antibodies. In certain embodiments, such a monoclonal antibody typically includes an antibody comprising a polypeptide sequence that binds a target, wherein the target-binding polypeptide sequence was obtained by a process that includes the selection of a single target binding polypeptide sequence from a plurality of polypeptide sequences. For example, the selection process can be the selection of a unique clone from a plurality of clones, such as a pool of hybridoma clones, phage clones, or recombinant DNA clones. It should be understood that a selected target binding sequence can be further altered, for example, to improve affinity for the target, to humanize the target-binding sequence, to improve its production in cell culture, to reduce its immunogenicity in vivo, to create a multispecific antibody, etc., and that an antibody comprising the altered target binding sequence is also a monoclonal antibody of this disclosure. In contrast to polyclonal antibody preparations, which typically include different antibodies directed against different determinants (epitopes), each monoclonal antibody of a monoclonal-antibody preparation is directed against a single determinant on an antigen. In addition to their specificity, monoclonal-antibody preparations are advantageous in that they are typically uncontaminated by other immunoglobulins.</div>
</li> <li> <para-num num="[0102]"> </para-num> <div class="description-line" id="p-0103" num="0102">A specific binding agent may comprise an antibody reactive with SEQ ID NO: 2.</div>
</li> <li> <para-num num="[0103]"> </para-num> <div class="description-line" id="p-0104" num="0103">For the achievements as disclosed in the present disclosure antibodies from various sources may be used. Standard protocols for obtaining antibodies can be as well used as modern alternative methods. Alternative methods for generation of antibodies comprise amongst others the use of synthetic or recombinant peptides, representing a clinically relevant epitope of ARMET for immunization. Alternatively, DNA immunization also known as DNA vaccination may be used. Clearly monoclonal antibodies or polyclonal antibodies from different species, e.g., rabbits, sheep, goats, rats or guinea pigs can be used. Since monoclonal antibodies can be produced in any amount required with constant properties, they represent useful tools in development of an assay for clinical routine.</div>
</li> <li> <para-num num="[0104]"> </para-num> <div class="description-line" id="p-0105" num="0104">As the skilled artisan will appreciate now, that protein ARMET has been identified as a marker which is useful in the assessment of COPD. Various immunodiagnostic procedures may be used to reach data comparable to the achievements of the present disclosure.</div>
</li> <li> <para-num num="[0105]"> </para-num> <div class="description-line" id="p-0106" num="0105">For determination of protein ARMET the sample obtained from an individual is incubated in vitro with the specific binding agent for ARMET under conditions appropriate for formation of a binding agent ARMET complex. Such conditions need not be specified, since the skilled artisan without any inventive effort can easily identify such appropriate incubation conditions. The amount of binding agent ARMET complex is determined and used in the assessment of COPD. As the skilled artisan will appreciate there are numerous methods to determine the amount of the specific binding agent ARMET complex all described in detail in relevant textbooks (cf., e.g., Tijssen, P., supra, or Diamandis, E. P., and Christopoulos, T. K. (eds.), Immunoassay, Academic Press, Boston (1996)).</div>
</li> <li> <para-num num="[0106]"> </para-num> <div class="description-line" id="p-0107" num="0106">Immunoassays are well known to the skilled artisan. Methods for carrying out such assays as well as practical applications and procedures are summarized in related textbooks. Examples of related textbooks are Tijssen, P., Preparation of enzyme-antibody or other enzyme-macromolecule conjugates, In: Practice and theory of enzyme immunoassays, pp. 221-278, Burdon, R. H. and v. Knippenberg, P. H. (eds.), Elsevier, Amsterdam (1990), and various volumes of Colowick, S. P., and Caplan, N. O., (eds.), Methods in Enzymology, Academic Press, dealing with immunological detection methods, especially volumes 70, 73, 74, 84, 92 and 121.</div>
</li> <li> <para-num num="[0107]"> </para-num> <div class="description-line" id="p-0108" num="0107">The present disclosure also relates in an embodiment to the use of an antibody specifically binding to protein ARMET in a method according to the present disclosure. In one embodiment in a method according to the present disclosure protein ARMET is measured in an immunoassay procedure. In a further embodiment protein ARMET is detected in an enzyme-linked immunoassay (ELISA).</div>
</li> <li> <para-num num="[0108]"> </para-num> <div class="description-line" id="p-0109" num="0108">In a further embodiment protein ARMET is detected in a sandwich assay (sandwich-type assay format). In such assay, a first specific binding agent is used to capture protein ARMET on the one side and a second specific binding agent, which is labelled to be directly or indirectly detectable, is used on the other side. The specific binding agents used in a sandwich-type assay format may be antibodies specifically directed against protein ARMET. On the one hand, the detection may be carried out by using different capturing and labelled antibodies, i.e. antibodies which recognize different epitopes on the ARMET polypeptide. On the other hand, a sandwich-type assay may also be carried out with a capture and labelling antibody which is directed against the same epitope of protein ARMET. In this embodiment, only di- and multimeric forms of protein ARMET may be detected. In an embodiment an antibody to protein ARMET is used in a qualitative (ARMET present or absent) or quantitative (amount of ARMET is determined) immunoassay.</div>
</li> <li> <para-num num="[0109]"> </para-num> <div class="description-line" id="p-0110" num="0109">In a further embodiment the method according to the present disclosure is based on the measurement of ARMET, wherein said measurement of ARMET is performed in a sandwich immunoassay employing at least two antibodies reactive with at least two non-overlapping epitopes.</div>
</li> <li> <para-num num="[0110]"> </para-num> <div class="description-line" id="p-0111" num="0110">In a further embodiment protein ARMET is detected in a competitive assay. In such assay format a binding agent specifically binding to ARMET of SEQ ID NO: 2 is used. In a mixture labeled ARMET that has been added to the mixture and ARMET comprised in a sample compete for binding to the specific binding agent. The extent of such competition can be measured according to standard procedures.</div>
</li> <li> <para-num num="[0111]"> </para-num> <div class="description-line" id="p-0112" num="0111">The concentration of the protein ARMET in test samples may be determined in vitro using a specific ELISA, as already described above. Using this assay format, the inventors have shown that samples from patients already diagnosed as having COPD by classical methods, e.g. spirometry, can be distinguished from samples from apparently healthy individuals. Results are shown in the example section of this application.</div>
</li> <li> <para-num num="[0112]"> </para-num> <div class="description-line" id="p-0113" num="0112">The inventors of the present disclosure surprisingly are able to detect protein ARMET in a body fluid sample. Even more surprising they are able to demonstrate that the presence of protein ARMET in such liquid sample obtained from an individual can be correlated to COPD. No tissue and no biopsy sample is required to make use of the marker ARMET in the assessment of COPD. Measuring the level of protein ARMET in (e.g. a small aliquot of) a simple body fluid sample is considered very advantageous in the field of COPD.</div>
</li> <li> <para-num num="[0113]"> </para-num> <div class="description-line" id="p-0114" num="0113">In an exemplary embodiment the method according to the present disclosure is practiced with serum as sample material. In some embodiments the method according to the present disclosure is practiced with plasma as sample material. In further embodiments the method according to the present disclosure is practiced with whole blood as sample material.</div>
</li> <li> <para-num num="[0114]"> </para-num> <div class="description-line" id="p-0115" num="0114">In further embodiments, the present disclosure relates to use of protein ARMET as a marker molecule in the in vitro assessment of COPD from a liquid sample obtained from an individual.</div>
</li> <li> <para-num num="[0115]"> </para-num> <div class="description-line" id="p-0116" num="0115">In some situation, a single event or process may cause the respective disease as, e.g., in infectious diseases. In other cases, especially when the etiology of the disease is not fully understood as is the case for COPD, correct diagnosis can be very difficult. As the skilled artisan will appreciate, no biochemical marker is diagnostic with 100% specificity and at the same time 100% sensitivity for a given multifactorial disease, as for example for COPD. Rather, biochemical markers are used to assess with a certain likelihood or predictive value an underlying diagnostic question, e.g., the presence, absence, or the severity of a disease. Therefore in routine clinical diagnosis, generally various clinical symptoms and biological markers are considered together in the assessment of an underlying disease. The skilled artisan is fully familiar with the mathematical/statistical methods that routinely are used to calculate a relative risk or likelihood for the diagnostic question to be assessed. In routine clinical practice various clinical symptoms and biological markers are generally considered together by a physician in the diagnosis, treatment, and management of the underlying disease.</div>
</li> <li> <para-num num="[0116]"> </para-num> <div class="description-line" id="p-0117" num="0116">COPD patients are traditionally treated with bronchodilators or steroids and examined by spirometry for reversibility of airflow obstruction. If reversibility is less than 15%, and particularly if they have a long history of smoking, then they would be classified as COPD patients.</div>
</li> <li> <para-num num="[0117]"> </para-num> <div class="description-line" id="p-0118" num="0117">The ATS (American Thoracic Society) criteria for diagnosing COPD are as follows:
</div> </li> <ul> <li id="ul0001-0001" num="0000"> <ul> <li id="ul0002-0001" num="0118">FEV1/FVC ratio&lt;0.7</li> <li id="ul0002-0002" num="0119">FEV1&lt;70% predicted, &lt;15% reversibility to inhaled B2 agonist:</li> <li id="ul0002-0003" num="0120">2 week oral prednisolone trial-less than 15% reversibility in FEV1</li> <li id="ul0002-0004" num="0121">Smoking history</li> </ul> </li> </ul>
<li> <para-num num="[0122]"> </para-num> <div class="description-line" id="p-0119" num="0122">FEV1 is the volume of air expelled from the lungs in one second, starting from a position of maximum inspiration and with the subject making maximum effort. FEV1% is the FEV1 expressed as a percentage of the forced vital capacity (FVC). The FVC is the total volume of air expelled from the lungs from a position of maximum inspiration with the subject making maximum effort. FEV1 may be measured using a spirometer to measure the volume of air expired in the first second of exhalation.</div>
</li> <li> <para-num num="[0123]"> </para-num> <div class="description-line" id="p-0120" num="0123">The spirometric classification of COPD according to the ATS (American Thoracic Society)/European respiratory Society 2004 is shown in Table 1. ATS COPD Stage 0 is currently no longer used in the ATS classification system.</div>
</li> <li> <div class="description-line" id="p-0121" num="0000">
<tables id="TABLE-US-00001" num="00001">
<patent-tables colsep="0" frame="none" rowsep="0">
<table align="left" class="description-table" cols="4" colsep="0" rowsep="0" width="100%">
<thead>
<tr class="description-tr">
<td class="description-td" colspan="4" nameend="4" namest="1" rowsep="1">TABLE 1</td>
</tr>
<tr class="description-tr">
<td align="center" class="description-td" colspan="4" nameend="4" namest="1" rowsep="1"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">COPD</td>
<td class="description-td"> </td>
<td class="description-td">Postbronchdilator</td>
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">Stage</td>
<td class="description-td">Severity</td>
<td class="description-td">FEV1/FVC</td>
<td class="description-td">FEV1 % pred</td>
</tr>
<tr class="description-tr">
<td align="center" class="description-td" colspan="4" nameend="4" namest="1" rowsep="1"> </td>
</tr>
</thead>
<tbody><tr class="description-tr">
<td class="description-td"> </td>
</tr>
</tbody></table>
<table align="left" class="description-table" cols="4" colsep="0" rowsep="0" width="100%">
<tbody><tr class="description-tr">
<td class="description-td">0</td>
<td class="description-td">At risk <sup>#</sup> </td>
<td class="description-td">&gt;0.7</td>
<td class="description-td">≧80%</td>
</tr>
<tr class="description-tr">
<td class="description-td">I</td>
<td class="description-td">Mild COPD</td>
<td class="description-td">≦0.7</td>
<td class="description-td">≧80%</td>
</tr>
<tr class="description-tr">
<td class="description-td">II</td>
<td class="description-td">Moderate COPD</td>
<td class="description-td">≦0.7</td>
<td class="description-td">50%-80%</td>
</tr>
<tr class="description-tr">
<td class="description-td">III</td>
<td class="description-td">Severe COPD</td>
<td class="description-td">≦0.7</td>
<td class="description-td">30%-50%</td>
</tr>
<tr class="description-tr">
<td class="description-td">IV</td>
<td class="description-td">Very severe COPD</td>
<td class="description-td">≦0.7</td>
<td class="description-td"> &lt;30%</td>
</tr>
<tr class="description-tr">
<td align="center" class="description-td" colspan="4" nameend="4" namest="1" rowsep="1"> </td>
</tr>
<tr class="description-tr">
<td align="left" class="description-td" colspan="4" id="FOO-00001" nameend="4" namest="1">FEV1: forced expiratory volume in one second; FVC: forced vital capacity;</td>
</tr>
<tr class="description-tr">
<td align="left" class="description-td" colspan="4" id="FOO-00002" nameend="4" namest="1"> <sup>#</sup> patients who smoke or have exposure to pollutants, have cough, sputum or dyspnoea, have family history of respiratory disease.</td>
</tr>
</tbody></table>
</patent-tables>
</tables>
</div>
</li> <li> <para-num num="[0124]"> </para-num> <div class="description-line" id="p-0122" num="0124">In the assessment of COPD the marker protein ARMET will be of advantage in one or more of the following aspects: assessment; screening; staging of disease; monitoring of disease progression; prognosis; guidance of therapy and monitoring of the response to therapy. Exemplary areas of diagnostic relevance in assessing an individual suspected or known to have COPD are screening, staging of disease, monitoring of disease progression and monitoring of the response to therapy.</div>
</li> <li> <para-num num="[0125]"> </para-num> <div class="description-line" id="p-0123" num="0125">Screening (assessment whether individuals are at risk for developing COPD or have COPD): is defined as the systematic application of a test to identify individuals e.g. at risk individuals, for indicators of a disease, e.g., the presence of COPD. For example, the screening population may be composed of individuals known to be at higher than average risk of COPD. For example, a screening population for COPD is composed of individuals known to be at higher than average risk of COPD, like smokers and ex-smokers.</div>
</li> <li> <para-num num="[0126]"> </para-num> <div class="description-line" id="p-0124" num="0126">Screening in the sense of the present disclosure relates to the unbiased assessment of individuals regarding their risk for developing COPD. In an embodiment the method according to the present disclosure is used for screening purposes. I.e., it is used to assess subjects without a prior diagnosis of COPD by a) determining the concentration of protein ARMET in a sample in vitro, and b) comparing the concentration of protein ARMET determined in step (a) with a reference concentration of protein ARMET, wherein a concentration of protein ARMET above the reference concentration is indicative for the presence of COPD. In an embodiment, a body fluid sample such as blood, serum, or plasma is used as a sample in the screening for COPD.</div>
</li> <li> <para-num num="[0127]"> </para-num> <div class="description-line" id="p-0125" num="0127">Measurement of protein ARMET will aid the physician to assess the presence or absence of COPD in an individual suspected to have COPD.</div>
</li> <li> <para-num num="[0128]"> </para-num> <div class="description-line" id="p-0126" num="0128">In an embodiment the present disclosure relates to an in vitro method for assessing the presence or absence of chronic obstructive pulmonary disease (COPD) in a subject, comprising a) determining the concentration of protein ARMET in a sample, and b) comparing the concentration of protein ARMET determined in step (a) with a reference concentration of protein ARMET, wherein a concentration of protein ARMET above the reference concentration is indicative for the presence of COPD. In some embodiments the sample is a body fluid sample. In further embodiments, the sample is selected from the group consisting of serum, plasma and whole blood.</div>
</li> <li> <para-num num="[0129]"> </para-num> <div class="description-line" id="p-0127" num="0129">In an embodiment the present disclosure relates to an in vitro method for assessing the presence or absence of chronic obstructive pulmonary disease (COPD) in a subject, comprising a) determining the concentration of protein ARMET in a sample, b) comparing the concentration of protein ARMET determined in step (a) with a reference concentration of protein ARMET, and c) assessing the presence or absence of COPD based on the comparison of step (b), wherein a concentration of protein ARMET above the reference concentration is indicative for the presence of COPD. In an exemplary embodiment the sample is a body fluid sample. In further embodiments, the sample is selected from the group consisting of serum, plasma and whole blood.</div>
</li> <li> <para-num num="[0130]"> </para-num> <div class="description-line" id="p-0128" num="0130">In some embodiments, the present disclosure relates to an in vitro method of assessing for a subject the presence or absence of COPD, the method comprising a) determining the concentration of protein ARMET in a sample, and b) comparing the concentration of protein ARMET determined in step (a) with a cut-off value for protein ARMET established in a reference population, wherein a concentration of protein ARMET above the cut-off value is indicative for the presence of COPD. In an embodiment the present disclosure relates to an in vitro method of assessing for a subject the presence or absence of COPD, the method comprising a) determining the concentration of protein ARMET in a sample, and b) comparing the concentration of protein ARMET determined in step (a) with a cut-off value for protein ARMET established in a reference population, wherein a concentration of protein ARMET below the cut-off value is indicative for the absence of COPD.</div>
</li> <li> <para-num num="[0131]"> </para-num> <div class="description-line" id="p-0129" num="0131">In an embodiment the present disclosure relates to the use of the protein ARMET in the assessment of COPD. For example, protein ARMET may used in the assessment of the presence or absence of COPD.</div>
</li> <li> <para-num num="[0132]"> </para-num> <div class="description-line" id="p-0130" num="0132">In a further embodiment the present disclosure relates to the use of the protein ARMET in the in vitro assessment of COPD in a sample, wherein a concentration of protein ARMET above a reference concentration for protein ARMET is indicative for COPD.</div>
</li> <li> <para-num num="[0133]"> </para-num> <div class="description-line" id="p-0131" num="0133">In some embodiments the sample according the use is a body fluid sample. For example, in some embodiments said body fluid sample according the use is selected from the group consisting of serum, plasma and whole blood.</div>
</li> <li> <para-num num="[0134]"> </para-num> <div class="description-line" id="p-0132" num="0134">In a further embodiment the present disclosure relates to the use of the protein ARMET in the in vitro assessment of COPD in a body fluid sample, wherein a concentration of protein ARMET above a reference concentration for protein ARMET in a body fluid sample is indicative for the presence of COPD.</div>
</li> <li> <para-num num="[0135]"> </para-num> <div class="description-line" id="p-0133" num="0135">In a further embodiment the present disclosure relates to the use of the protein ARMET in the in vitro assessment of COPD in a serum, plasma, or whole blood sample, wherein a concentration of protein ARMET above a reference concentration for protein ARMET in a serum, plasma, or whole blood sample is indicative for the presence of COPD.</div>
</li> <li> <para-num num="[0136]"> </para-num> <div class="description-line" id="p-0134" num="0136">One embodiment of the present disclosure refers to the screening of a population to distinguish between individuals which are probably free from COPD and individuals which probably have COPD. The latter group of individuals may then be subject to further diagnostic procedures, e.g. by lung function testing, spirometry or other suitable means.</div>
</li> <li> <para-num num="[0137]"> </para-num> <div class="description-line" id="p-0135" num="0137">In an embodiment the in vitro method according to the present disclosure is characterized in that the assessment of the protein ARMET takes place for classifying a patient according to be at risk to have COPD for clinical decisions, particularly further treatment by means of medications for the treatment or therapy of COPD, and for treatment or therapy of infection/inflammatory diseases of the airway and lung, as well as for therapy control of an antibiotic treatment or therapeutic antibody treatment.</div>
</li> <li> <para-num num="[0138]"> </para-num> <div class="description-line" id="p-0136" num="0138">In an embodiment the present disclosure relates to an in vitro method for assessing whether an individual is at risk for developing COPD comprising the steps of a) determining the concentration of protein ARMET in a sample, and b) of assessing said individual's risk for developing COPD by comparing the concentration of protein ARMET determined in step (a) with a reference concentration of protein ARMET, wherein a concentration of protein ARMET above a reference concentration is indicative for an individual to be at risk for developing COPD.</div>
</li> <li> <para-num num="[0139]"> </para-num> <div class="description-line" id="p-0137" num="0139">In an embodiment the present disclosure relates to an in vitro method for assessing whether an individual is at risk for developing COPD comprising the steps of a) determining the concentration of protein ARMET in a body fluid sample, and b) of assessing said individual's risk for developing COPD by comparing the concentration of protein ARMET determined in step (a) with a reference concentration of protein ARMET, wherein a concentration of protein ARMET above a reference concentration is indicative for an individual to be at risk for developing COPD. In an exemplary embodiment the body fluid sample is selected from the group consisting of serum, plasma and whole blood.</div>
</li> <li> <para-num num="[0140]"> </para-num> <div class="description-line" id="p-0138" num="0140">Prognosis.</div>
</li> <li> <para-num num="[0141]"> </para-num> <div class="description-line" id="p-0139" num="0141">Prognostic indicators can be defined as clinical, pathological or biochemical features of COPD patients that predict with a certain likelihood the disease outcome. Their main use is to help to rationally plan patient management, i.e. to avoid undertreatment of aggressive disease and overtreatment of indolent disease, respectively.</div>
</li> <li> <para-num num="[0142]"> </para-num> <div class="description-line" id="p-0140" num="0142">As the level of protein ARMET alone significantly contributes to the differentiation of COPD patients from healthy controls or other diseases of the lung (e.g. asthma, bronchitis, pulmonary fibrosis and tuberculosis), it has to be expected that it will aid in assessing the prognosis of patients suffering from COPD. The concentration of protein ARMET may be combined with results of lung function testing or spirometry.</div>
</li> <li> <para-num num="[0143]"> </para-num> <div class="description-line" id="p-0141" num="0143">Differentiation of COPD from Asthma.</div>
</li> <li> <para-num num="[0144]"> </para-num> <div class="description-line" id="p-0142" num="0144">In a further embodiment the method according to the present disclosure is used to differentiate COPD from other types of lung diseases, for example asthma.</div>
</li> <li> <para-num num="[0145]"> </para-num> <div class="description-line" id="p-0143" num="0145">According to the instant disclosure, the protein ARMET may also be used to differentiate COPD from other types of lung diseases, e.g. asthma, bronchitis, pulmonary fibrosis and tuberculosis. Surprisingly the inventors have found that the use of a marker combination of a COPD specific marker, for example ARMET, and an inflammation marker selected from the group consisting of CRP, interleukin-6, serum amyloid A, S100 and E-selectin, can lead to a differentiation between COPD and other inflammatory diseases of the lung, e.g. asthma, acute or chronic inflammation of the lung, respectively. Experimental results for the protein ARMET and protein CRP are shown in the example section.</div>
</li> <li> <para-num num="[0146]"> </para-num> <div class="description-line" id="p-0144" num="0146">Monitoring of Disease Progression.</div>
</li> <li> <para-num num="[0147]"> </para-num> <div class="description-line" id="p-0145" num="0147">At present it is very difficult to predict with a reasonable likelihood whether a patient diagnosed with COPD has a more or less stable status or whether the disease will progress.</div>
</li> <li> <para-num num="[0148]"> </para-num> <div class="description-line" id="p-0146" num="0148">Progression of disease, i.e. of COPD, may be evaluated by in vitro monitoring of the concentration of protein ARMET in test samples, especially by taking one or more consecutive samples. In an embodiment the present disclosure relates to an in vitro method for monitoring the disease progression in a patient suffering from COPD, the method comprising the steps of a) determining the concentration of protein ARMET in a sample, b) comparing the concentration of protein ARMET determined in step (a) with a reference concentration of protein ARMET, and monitoring the disease progression by comparing the concentration determined in step (a) to the concentration of this marker as determined in a sample taken from the same patient at a previous point in time. As will be appreciated that an increase in the level of C-terminal proSP-B over time is indicative of disease progression.</div>
</li> <li> <para-num num="[0149]"> </para-num> <div class="description-line" id="p-0147" num="0149">Monitor a Patient's Response to Therapy.</div>
</li> <li> <para-num num="[0150]"> </para-num> <div class="description-line" id="p-0148" num="0150">The method according to the present disclosure, when used in patient monitoring, may be used in the follow-up of patients and e.g. help to assess efficacy of a treatment of COPD.</div>
</li> <li> <para-num num="[0151]"> </para-num> <div class="description-line" id="p-0149" num="0151">In an embodiment the present disclosure relates to an in vitro method for monitoring a patient's response to a treatment targeted at reducing COPD, comprising the steps of a) determining the concentration of protein ARMET in a body fluid sample, b) comparing the concentration of protein ARMET determined in step (a) with a reference concentration of protein ARMET, and of monitoring a patient's response to COPD therapy by comparing the concentration determined in step (a) to the concentration of this marker to its reference value. In an exemplary embodiment, the body fluid sample is selected from the group consisting of serum, plasma and whole blood.</div>
</li> <li> <para-num num="[0152]"> </para-num> <div class="description-line" id="p-0150" num="0152">Monitoring a patient's response to therapy can be practiced e.g. by establishing the pre- and post-therapeutic marker level for protein ARMET and by comparing the pre- and the post-therapeutic marker level.</div>
</li> <li> <para-num num="[0153]"> </para-num> <div class="description-line" id="p-0151" num="0153">A patient's response to a COPD treatment may be evaluated in vitro by monitoring the concentration of protein ARMET in test samples over time. In an embodiment the present disclosure relates to an in vitro method for monitoring a patient's response to a COPD treatment, comprising the steps of a) determining the concentration of protein ARMET in a sample, b) comparing the concentration of protein ARMET determined in step (a) with a concentration of protein ARMET established in a previous sample, wherein a decrease in protein ARMET is indicative of a positive response to said treatment.</div>
</li> <li> <para-num num="[0154]"> </para-num> <div class="description-line" id="p-0152" num="0154">The level of protein ARMET appears to be appropriate to monitor a patient's response to therapy. The present disclosure thus also relates to the use of protein ARMET in monitoring a patient's response to therapy, wherein a decreased level of protein ARMET is a positive indicator for an effective treatment of COPD.</div>
</li> <li> <para-num num="[0155]"> </para-num> <div class="description-line" id="p-0153" num="0155">Marker Combinations.</div>
</li> <li> <para-num num="[0156]"> </para-num> <div class="description-line" id="p-0154" num="0156">The present disclosure therefore relates in an embodiment to the use of protein ARMET as one marker of a marker panel for the assessment of COPD. Such marker panel comprises protein ARMET and one or more additional marker for COPD. Certain combinations of markers will e.g. be advantageous in the screening for COPD.</div>
</li> <li> <para-num num="[0157]"> </para-num> <div class="description-line" id="p-0155" num="0157">As the skilled artisan will appreciate there are many ways to use the measurements of two or more markers in order to improve the diagnostic question under investigation.</div>
</li> <li> <para-num num="[0158]"> </para-num> <div class="description-line" id="p-0156" num="0158">Biochemical markers can either be determined individually or in an embodiment of the disclosure they can be determined simultaneously, e.g. using a chip or a bead based array technology. The concentrations of the biomarkers are then either interpreted independently, e.g., using an individual cut-off for each marker, or they are combined for interpretation.</div>
</li> <li> <para-num num="[0159]"> </para-num> <div class="description-line" id="p-0157" num="0159">As the skilled artisan will appreciate the step of correlating a marker level to a certain likelihood or risk can be performed and achieved in different ways. For example, the determined concentration of protein ARMET and the one or more other marker(s) may be mathematically combined and the combined value may correlated to the underlying diagnostic question. Marker values may be combined with the determination of ARMET by any appropriate state of the art mathematical method.</div>
</li> <li> <para-num num="[0160]"> </para-num> <div class="description-line" id="p-0158" num="0160">In at least some embodiments, the mathematical algorithm applied in the combination of markers may be a logistic function. The result of applying such mathematical algorithm or such logistical function may be a single value. Dependent on the underlying diagnostic question such value can easily be correlated to e.g., the risk of an individual for COPD or to other intended diagnostic uses helpful in the assessment of patients with COPD. In an exemplary way, such logistic function is obtained by a) classification of individuals into the groups, e.g., into normals, individuals at risk for COPD, patients with acute or chronic inflammation of the lung and so on, b) identification of markers which differ significantly between these groups by univariate analysis, c) logistic regression analysis to assess the independent discriminative values of markers useful in assessing these different groups and d) construction of the logistic function to combine the independent discriminative values. In this type of analysis the markers are no longer independent but represent a marker combination.</div>
</li> <li> <para-num num="[0161]"> </para-num> <div class="description-line" id="p-0159" num="0161">In an embodiment the logistic function used for combining the values for ARMET and the value of at least one further marker is obtained by a) classification of individuals into the groups of normals and individuals likely to have COPD, respectively, b) establishing the values for ARMET and the value of the at least one further marker c) performing logistic regression analysis and d) construction of the logistic function to combine the marker values for ARMET and the value of the at least one further marker.</div>
</li> <li> <para-num num="[0162]"> </para-num> <div class="description-line" id="p-0160" num="0162">A logistic function for correlating a marker combination to a disease may employ an algorithm developed and obtained by applying statistical methods. Appropriate statistical methods e.g. are Discriminant analysis (DA) (i.e., linear-, quadratic-, regularized-DA), Kernel Methods (i.e., SVM), Nonparametric Methods (i.e., k-Nearest-Neighbor Classifiers), PLS (Partial Least Squares), Tree-Based Methods (i.e., Logic Regression, CART, Random Forest Methods, Boosting/Bagging Methods), Generalized Linear Models (i.e., Logistic Regression), Principal Components based Methods (i.e., SIMCA), Generalized Additive Models, Fuzzy Logic based Methods, Neural Networks and Genetic Algorithms based Methods. The skilled artisan will have no problem in selecting an appropriate statistical method to evaluate a marker combination of the present disclosure and thereby to obtain an appropriate mathematical algorithm. In an embodiment the statistical method employed to obtain the mathematical algorithm used in the assessment of COPD is selected from DA (i.e., Linear-, Quadratic-, Regularized Discriminant Analysis), Kernel Methods (i.e., SVM), Nonparametric Methods (i.e., k-Nearest-Neighbor Classifiers), PLS (Partial Least Squares), Tree-Based Methods (i.e., Logic Regression, CART, Random Forest Methods, Boosting Methods), or Generalized Linear Models (i.e., Logistic Regression). Details relating to these statistical methods are found in the following references: Ruczinski, I., et al., J. of Computational and Graphical Statistics 12 (2003) 475-511; Friedman, J. H., J. of the American Statistical Association 84 (1989) 165-175; Hastie, T., et al., The Elements of Statistical Learning, Springer Verlag (2001); Breiman, L., et al., Classification and regression trees, Wadsworth International Group, California (1984); Breiman, L., Machine Learning 45 (2001) 5-32; Pepe, M. S., The Statistical Evaluation of Medical Tests for Classification and Prediction, Oxford Statistical Science Series, 28, Oxford University Press (2003); and Duda, R. O., et al., Pattern Classification, John Wiley &amp; Sons, Inc., 2nd ed. (2001).</div>
</li> <li> <para-num num="[0163]"> </para-num> <div class="description-line" id="p-0161" num="0163">It is an embodiment of the disclosure to use an optimized multivariate cut-off for the underlying combination of biological markers and to discriminate state A from state B, e.g., normals and individuals at risk for COPD, COPD patients responsive to therapy and therapy failures, patients having an acute inflammation of the lung and COPD patients, COPD patients showing disease progression and COPD patients not showing disease progression, respectively.</div>
</li> <li> <para-num num="[0164]"> </para-num> <div class="description-line" id="p-0162" num="0164">The area under the receiver operator curve (=AUC) is an indicator of the performance or accuracy of a diagnostic procedure. Accuracy of a diagnostic method is best described by its receiver-operating characteristics (ROC) (see especially Zweig, M. H., and Campbell, G., Clin. Chem. 39 (1993) 561-577). The ROC graph is a plot of all of the sensitivity/specificity pairs resulting from continuously varying the decision thresh-hold over the entire range of data observed.</div>
</li> <li> <para-num num="[0165]"> </para-num> <div class="description-line" id="p-0163" num="0165">The clinical performance of a laboratory test depends on its diagnostic accuracy, or the ability to correctly classify subjects into clinically relevant subgroups. Diagnostic accuracy measures the test's ability to correctly distinguish two different conditions of the subjects investigated. Such conditions are for example, health and disease or disease progression versus no disease progression.</div>
</li> <li> <para-num num="[0166]"> </para-num> <div class="description-line" id="p-0164" num="0166">In each case, the ROC plot depicts the overlap between the two distributions by plotting the sensitivity versus 1—specificity for the complete range of decision thresholds. On the y-axis is sensitivity, or the true-positive fraction [defined as (number of true-positive test results)/(number of true-positive+number of false-negative test results)]. This has also been referred to as positivity in the presence of a disease or condition. It is calculated solely from the affected subgroup. On the x-axis is the false-positive fraction, or 1—specificity [defined as (number of false-positive results)/(number of true-negative+number of false-positive results)]. It is an index of specificity and is calculated entirely from the unaffected subgroup. Because the true- and false-positive fractions are calculated entirely separately, by using the test results from two different subgroups, the ROC plot is independent of the prevalence of disease in the sample. Each point on the ROC plot represents a sensitivity/1-specificity pair corresponding to a particular decision threshold. A test with perfect discrimination (no overlap in the two distributions of results) has an ROC plot that passes through the upper left corner, where the true-positive fraction is 1.0, or 100% (perfect sensitivity), and the false-positive fraction is 0 (perfect specificity). The theoretical plot for a test with no discrimination (identical distributions of results for the two groups) is a 45° diagonal line from the lower left corner to the upper right corner. Most plots fall in between these two extremes. (If the ROC plot falls completely below the 45° diagonal, this is easily remedied by reversing the criterion for “positivity” from “greater than” to “less than” or vice versa.) Qualitatively, the closer the plot is to the upper left corner, the higher the overall accuracy of the test.</div>
</li> <li> <para-num num="[0167]"> </para-num> <div class="description-line" id="p-0165" num="0167">One convenient goal to quantify the diagnostic accuracy of a laboratory test is to express its performance by a single number. The most common global measure is the area under the ROC plot (AUC). By convention, this area is always ≧0.5 (if it is not, one can reverse the decision rule to make it so). Values range between 1.0 (perfect separation of the test values of the two groups) and 0.5 (no apparent distributional difference between the two groups of test values). The area does not depend only on a particular portion of the plot such as the point closest to the diagonal or the sensitivity at 90% specificity, but on the entire plot. This is a quantitative, descriptive expression of how close the ROC plot is to the perfect one (area=1.0).</div>
</li> <li> <para-num num="[0168]"> </para-num> <div class="description-line" id="p-0166" num="0168">The overall assay sensitivity will depend on the specificity required for practicing the method disclosed here. In certain settings a specificity of 75% may be sufficient and statistical methods and resulting algorithms can be based on this specificity requirement. In an exemplary embodiment the method used to assess individuals at risk for COPD is based on a specificity of 80%, of 85%, or even of 90% or of 95%.</div>
</li> <li> <para-num num="[0169]"> </para-num> <div class="description-line" id="p-0167" num="0169">Certain combinations of markers will be advantageous in the screening for COPD. In one embodiment the present disclosure is directed to an in vitro method for assessing COPD by biochemical markers, comprising determining in a sample the concentration of protein ARMET and of one or more other marker(s), mathematically combining the determined concentration of protein ARMET and the concentration of the one or more other marker, respectively, wherein a increased combined value is indicative for the presence of COPD.</div>
</li> <li> <para-num num="[0170]"> </para-num> <div class="description-line" id="p-0168" num="0170">In an embodiment the present disclosure is directed to an in vitro method for assessing COPD by biochemical markers, comprising determining in a sample the concentration of protein ARMET and of one or more other marker(s) and comparing the determined concentration of protein ARMET with a reference concentration of protein ARMET, wherein a concentration of protein ARMET above a reference concentration is indicative for the presence of COPD. In at least some embodiments, the one or more other marker of said method may be selected from the group consisting of ASC, NNMT, FEN1, APEX1 and Seprase. In further embodiments said marker panel comprises at least protein ARMET and protein APEX1. In a further embodiment said marker panel comprises at least protein ARMET and protein ASC. In a further embodiment said marker panel comprises at least protein ARMET and protein NNMT. In a further embodiment said marker panel comprises at least protein ARMET and protein FEN1. In an even further embodiment said marker panel comprises at least protein ARMET and protein Seprase.</div>
</li> <li> <para-num num="[0171]"> </para-num> <div class="description-line" id="p-0169" num="0171">In some embodiments of the present disclosure, the use of marker ARMET as a marker molecule for the in vitro assessment of COPD in combination with one or more marker molecule(s) indicative for COPD is disclosed. The present disclosure therefore relates, in some embodiments, to the use of protein ARMET as one marker of a COPD marker panel, i.e. a marker panel comprising protein ARMET and one or more additional marker for COPD screening purposes.</div>
</li> <li> <para-num num="[0172]"> </para-num> <div class="description-line" id="p-0170" num="0172">For example the present disclosure also relates to the use of a marker panel comprising protein ARMET and ASC, or of a marker panel comprising protein ARMET and NNMT, or of a marker panel comprising protein ARMET and FEN1, or of a marker panel comprising protein ARMET and APEX1, or of a marker panel comprising protein ARMET and Seprase, or of a marker panel comprising protein ARMET and two or more markers selected from the group consisting of ASC, NNMT, FEN1, APEX1 and Seprase.</div>
</li> <li> <para-num num="[0173]"> </para-num> <div class="description-line" id="p-0171" num="0173">In an embodiment markers for use in a combination with protein ARMET in the method according to the present disclosure are selected from the group consisting of ASC, NNMT, FEN1, APEX1 and Seprase. These markers may be used individually each or in any combination together with ARMET for assessing COPD. In an further embodiment the present disclosure relates to a marker panel (marker combination) selected from the group consisting of protein ARMET, ASC, NNMT, FEN1, APEX1, Seprase and CRP.</div>
</li> <li> <para-num num="[0174]"> </para-num> <div class="description-line" id="p-0172" num="0174">In an embodiment the marker panel used in the in vitro method for assessing COPD by biochemical markers comprises the steps of determining in a sample the concentration of protein ARMET and of protein NNMT, wherein a concentration of protein ARMET above a reference concentration for protein ARMET is indicative for the presence of COPD. In a further embodiment the marker panel used in the in vitro method comprises the marker proteins ARMET, NNMT and Seprase. In a further embodiment the marker panel used in the in vitro method comprises the marker proteins ARMET, NNMT, Seprase and ASC.</div>
</li> <li> <para-num num="[0175]"> </para-num> <div class="description-line" id="p-0173" num="0175">In a further embodiment a marker for use in combination with protein ARMET is a marker which is useful for the assessment of an inflammation (i.e. an underlying systemic inflammation).</div>
</li> <li> <para-num num="[0176]"> </para-num> <div class="description-line" id="p-0174" num="0176">Marker of Inflammation.</div>
</li> <li> <para-num num="[0177]"> </para-num> <div class="description-line" id="p-0175" num="0177">Many serum markers for the diagnosis of an inflammation are presently known. The skilled artisan is familiar with the term “marker of inflammation”. Said marker of inflammation is for example selected from the interleukin-6, C-reactive protein, serum amyloid A, sE-selectin and a S100 protein.</div>
</li> <li> <para-num num="[0178]"> </para-num> <div class="description-line" id="p-0176" num="0178">Interleukin-6 (IL-6) is a 21 kDa secreted protein that has numerous biological activities that can be divided into those involved in hematopoiesis and into those involved in the activation of the innate immune response. IL-6 is an acute-phase reactant and stimulates the synthesis of a variety of proteins, including adhesion molecules. Its major function is to mediate the acute phase production of hepatic proteins, and its synthesis is induced by the cytokines IL-1 and TNF-. IL-6 is normally produced by macrophages and T lymphocytes. The normal serum concentration of IL-6 is &lt;5 pg/ml.</div>
</li> <li> <para-num num="[0179]"> </para-num> <div class="description-line" id="p-0177" num="0179">C-reactive protein (CRP) is a homopentameric Ca<sup>2+</sup>-binding acute phase protein with 21 kDa subunits that is involved in host defense. CRP synthesis is induced by IL-6, and indirectly by IL-1, since IL-1 can trigger the synthesis of IL-6 by Kupffer cells in the hepatic sinusoids. The normal plasma concentration of CRP is &lt;3 μg/ml (30 nM) in 90% of the healthy population, and &lt;10 μg/ml (100 nM) in 99% of healthy individuals. Plasma CRP concentrations can, e.g., be measured by an immunoassay. Plasma CRP concentrations can, e.g. be measured by homogeneous assay formats or ELISA.</div>
</li> <li> <para-num num="[0180]"> </para-num> <div class="description-line" id="p-0178" num="0180">Serum amyloid A (=SAA) is an acute phase protein of low molecular weight of 11.7 kDa. It is predominantly synthesized by the liver in response to IL-1, IL-6 or TNF-stimulation and is involved in the regulation of the T-cell dependent immune response. Upon acute events the concentration of SAA increases up to 1000-fold reaching one milligram per milliliter. It is used to monitor inflammation in diseases as divers as cystic fibrosis, renal graft refection, trauma or infections. In rheumatoid arthritis is has in certain cases been used as a substitute for CRP, but, SAA is not yet as widely accepted.</div>
</li> <li> <para-num num="[0181]"> </para-num> <div class="description-line" id="p-0179" num="0181">S100-proteins form a constantly increasing family of Ca<sup>2+</sup>-binding proteins that today includes more than 20 members. The physiologically relevant structure of S100-proteins is a homodimer but some can also form heterodimers with each other, e.g., S100A8 and S100A9. The intracellular functions range from regulation of protein phosphorylation, of enzyme activities, or of the dynamics of the cytoskeleton to involvement in cell proliferation and differentiation. As some S100-proteins are also released from cells, extracellular functions have been described as well, e.g., neuronal survival, astrocyte proliferation, induction of apoptosis and regulation of inflammatory processes. S100A8, S100A9, the heterodimer S100A8/A9 and S100A12 have been found in inflammation with S100A8 responding to chronic inflammation, while S100A9, S100A8/A9 and S100A12 are increased in acute inflammation. S100A8, S100A9, S100A8/A9 and S100A12 have been linked to different diseases with inflammatory components including some cancers, renal allocraft rejection, colitis and most importantly to RA (Burmeister, G., and Gallacchi, G., Inflammopharmacology 3 (1995) 221-230; Foell, D., et al., Rheumathology 42 (2003) 1383-1389).</div>
</li> <li> <para-num num="[0182]"> </para-num> <div class="description-line" id="p-0180" num="0182">sE-selectin (soluble endothelial leukocyte adhesion molecule-1, ELAM-1) is a 115 kDa, type-I transmembrane glycoprotein expressed only on endothelial cells and only after activation by inflammatory cytokines (IL-1β, TNF-α) or endotoxin. Cell-surface E-selectin is a mediator of the rolling attachment of leucocytes to the endothelium, an essential step in extravasion of leucocytes at the site of inflammation, thereby playing an important role in localized inflammatory response. Soluble E-selectin is found in the blood of healthy individuals, probably arising from proteolytic cleavage of the surface-expressed molecule. Elevated levels of sE-selectin in serum have been reported in a variety of pathological conditions (Gearing, A. J. and Hemingway, I., Ann. N.Y. Acad. Sci. 667 (1992) 324-331).</div>
</li> <li> <para-num num="[0183]"> </para-num> <div class="description-line" id="p-0181" num="0183">In some embodiments a marker for use in a combination with protein ARMET in the method according to the present disclosure is selected from the group consisting of CRP, interleukin-6, serum amyloid A and S100. In a further embodiment according to the in vitro method of the present disclosure the value determined for ARMET is combined with the determined value of at least one further marker selected from the group consisting of CRP, interleukin-6, serum amyloid A, S100 and E-selectin. In an embodiment the present disclosure relates to the use of the marker combination ARMET and C-reactive protein (CRP) in the assessment of COPD. In an embodiment the present disclosure relates to the use of the marker combination ARMET and interleukin-6 (IL-6) in the assessment of COPD. In an embodiment the present disclosure relates to the use of the marker combination ARMET and serum amyloid A in the assessment of COPD. In an embodiment the present disclosure relates to the use of the marker combination ARMET and S100 in the assessment of COPD.</div>
</li> <li> <para-num num="[0184]"> </para-num> <div class="description-line" id="p-0182" num="0184">In a further embodiment the present disclosure relates to the use of a marker panel comprising protein ARMET and CRP in the in vitro assessment for the presence or absence of COPD in a serum or plasma sample, wherein a concentration of protein ARMET above a reference concentration for protein ARMET and a concentration of protein CRP above a reference concentration for protein CRP is indicative for the presence of COPD.</div>
</li> <li> <para-num num="[0185]"> </para-num> <div class="description-line" id="p-0183" num="0185">In a further embodiment the present disclosure relates to the use of a marker panel comprising protein ARMET and CRP in the in vitro assessment for the presence or absence of COPD in a serum or plasma sample, wherein a concentration of protein ARMET equal or below to a reference concentration for protein ARMET and a concentration of protein CRP above a reference concentration for protein CRP is indicative for the absence of COPD.</div>
</li> <li> <para-num num="[0186]"> </para-num> <div class="description-line" id="p-0184" num="0186">Marker panels in one embodiment are combined within a single test device, e.g. on a chip or in an array format. A marker panel according to the present disclosure is in an embodiment determined using a bio-chip array (protein array) technique. An array is a collection of addressable individual markers. Such markers can be spacially addressable, such as arrays contained within microtiter plates or printed on planar surfaces where each marker is present at distinct X and Y coordinates. Alternatively, markers can be addressable based on tags, beads, nanoparticles, or physical properties. A bio-chip array can be prepared according to the methods known to the ordinarily skilled artisan (see for example, U.S. Pat. No. 5,807,522; Robinson, W. H., et al., Nat. Med. 8 (2002) 295-301; Robinson, W. H., et al., Arthritis Rheum. 46 (2002) 885-893). Array as used herein refers to any immunological assay with multiple addressable markers. A bio-chip array, also known to the skilled artisan as microarray, is a miniaturized form of an array.</div>
</li> <li> <para-num num="[0187]"> </para-num> <div class="description-line" id="p-0185" num="0187">The terms “chip”, “bio-chip”, “polymer-chip” or “protein-chip” are used interchangeably and refer to a collection of a large number of probes, markers or biochemical markers arranged on a shared substrate which could be a portion of a silicon wafer, a nylon strip, a plastic strip, or a glass slide.</div>
</li> <li> <para-num num="[0188]"> </para-num> <div class="description-line" id="p-0186" num="0188">An “array,” “macroarray” or “microarray” is an intentionally created collection of substances, such as molecules, markers, openings, microcoils, detectors and/or sensors, attached to or fabricated on a substrate or solid surface, such as glass, plastic, silicon chip or other material forming an array. The arrays can be used to measure the levels of large numbers, e.g., tens, thousands or millions, of reactions or combinations simultaneously. An array may also contain a small number of substances, e.g., one, a few or a dozen. The substances in the array can be identical or different from each other. The array can assume a variety of formats, e.g., libraries of soluble molecules, libraries of immobilized molecules, libraries of immobilized antibodies, libraries of compounds tethered to resin beads, silica chips, or other solid supports. The array could either be a macroarray or a microarray, depending on the size of the pads on the array. A macroarray generally contains pad sizes of about 300 microns or larger and can be easily imaged by gel and blot scanners. A microarray would generally contain pad sizes of less than 300 microns.</div>
</li> <li> <para-num num="[0189]"> </para-num> <div class="description-line" id="p-0187" num="0189">A “solid support” is insoluble, functionalized, polymeric material to which library members or reagents may be attached or covalently bound (often via a linker) to be immobilized or allowing them to be readily separated (by filtration, centrifugation, washing etc.) from excess reagents, soluble reaction by-products, or solvents.</div>
</li> <li> <para-num num="[0190]"> </para-num> <div class="description-line" id="p-0188" num="0190">In an embodiment the present disclosure relates to a bio-chip array comprising the marker protein ARMET and optionally one or more other marker protein of COPD. The present disclosure also provides in an embodiment a bio-chip array for performing the method according to the present disclosure to specifically determine the concentration of protein ARMET and of one or more other marker selected from the group consisting of proteins ASC, NNMT, FEN1, APEX1 and Seprase, and optionally auxiliary reagents for performing the measurement.</div>
</li> <li> <para-num num="[0191]"> </para-num> <div class="description-line" id="p-0189" num="0191">The present disclosure also provides in an embodiment a bio-chip array for performing the method according to the present disclosure to specifically determine the concentration of protein ARMET and of one or more other marker selected from the group consisting of proteins ASC, NNMT, FEN1, APEX1 and Seprase, and optionally auxiliary reagents in the assessment of the presence or absence of COPD.</div>
</li> <li> <para-num num="[0192]"> </para-num> <div class="description-line" id="p-0190" num="0192">Kit.</div>
</li> <li> <para-num num="[0193]"> </para-num> <div class="description-line" id="p-0191" num="0193">The present disclosure also provides a kit for performing the in vitro method according to the present disclosure comprising the reagents required to specifically determine the concentration of protein ARMET.</div>
</li> <li> <para-num num="[0194]"> </para-num> <div class="description-line" id="p-0192" num="0194">The present disclosure also provides a kit for performing the method according to the present disclosure comprising the reagents required to specifically determine the concentration of protein ARMET and optionally one or more marker protein of COPD as described above, wherein the other markers may be each used individually or in any combination thereof.</div>
</li> <li> <para-num num="[0195]"> </para-num> <div class="description-line" id="p-0193" num="0195">The present disclosure also provides a kit for performing the method according to the present disclosure comprising the reagents required to specifically determine the concentration of protein ARMET and one or more other marker protein selected from the group consisting of proteins ASC, NNMT, FEN1, APEX1 and Seprase, and optionally auxiliary reagents for performing the measurement.</div>
</li> <li> <para-num num="[0196]"> </para-num> <div class="description-line" id="p-0194" num="0196">In yet a further embodiment the present disclosure relates to a kit comprising the reagents required to specifically determine the concentration of protein ARMET and the reagents required to measure the one or more other marker of COPD that are used together in an COPD marker combination. Said kit comprises in an embodiment antibodies or fragments thereof specifically binding to protein ARMET. In a further embodiment said antibody fragments in said kit are selected from the group consisting of Fab, Fab′, F(ab′)2, and Fv. In one embodiment the present disclosure relates to a kit comprising at least two antibodies or fragments thereof specifically binding to at least two non-overlapping epitopes comprised in the ARMET sequence of SEQ ID NO:1. In some cases, the at least two antibodies or fragments thereof comprised in a kit according to the present disclosure are monoclonal antibodies. Said kit further comprises in an embodiment a bio-chip on which the antibodies or fragments thereof are immobilized.</div>
</li> <li> <para-num num="[0197]"> </para-num> <div class="description-line" id="p-0195" num="0197">In a further embodiment the present disclosure relates to an in vitro diagnostic medical device (IVD) for carrying out the in vitro method for assessing COPD according to the present disclosure. A “diagnostic device” as used herein refers to an in vitro diagnostic medical device (IVD) if it is a reagent, calibrator, control material, kit, specimen receptacle, software, instrument, apparatus, equipment or system, whether used alone or in combination with other diagnostic goods for in vitro use. It, for example, will be generally intended by the manufacturer to be used in vitro for the examination of samples or specimens derived from the human body, solely or principally for the purpose of giving information about a concentration of a marker, physiological or pathological state, a congenital abnormality or to determine safety and compatibility with a potential recipient, or to monitor therapeutic measures.</div>
</li> <li> <para-num num="[0198]"> </para-num> <div class="description-line" id="p-0196" num="0198">The following examples, sequence listing, and figures are provided for the purpose of demonstrating various embodiments of the instant disclosure and aiding in an understanding of the present disclosure, the true scope of which is set forth in the appended claims. These examples are not intended to, and should not be understood as, limiting the scope or spirit of the instant disclosure in any way. It should also be understood that modifications can be made in the procedures set forth without departing from the spirit of the disclosure.</div>
</li> <heading id="h-0008">ILLUSTRATIVE EMBODIMENTS</heading>
<li> <para-num num="[0199]"> </para-num> <div class="description-line" id="p-0197" num="0199">The following comprises a list of illustrative embodiments according to the instant disclosure which represent various embodiments of the instant disclosure. These illustrative embodiments are not intended to be exhaustive or limit the disclosure to the precise forms disclosed, but rather, these illustrative embodiments are provided to aide in further describing the instant disclosure so that others skilled in the art may utilize their teachings.</div>
</li> <li> <div class="description-line" id="p-0198" num="0000">1. An in vitro method for assessing chronic obstructive pulmonary disease (COPD) in a human subject, comprising
<br/>
a) determining the concentration of protein ARMET in a serum, plasma, or whole blood sample, and
<br/>
b) comparing the concentration of protein ARMET determined in step (a) with a reference concentration of protein ARMET, wherein a concentration of protein ARMET above a reference concentration is indicative for COPD.
<br/>
2. The method according to embodiment 1, wherein the protein ARMET is measured in an immunoassay procedure.
<br/>
3. The method according to embodiment 2, wherein the immunoassay procedure is an enzyme-linked immunoassay (ELISA).
<br/>
4. The method according to embodiments 2 and 3, wherein ARMET is measured in a sandwich assay format.
<br/>
5. The method according to embodiments 2 and 3, wherein ARMET is measured in a competitive assay format.
<br/>
6. Use of protein ARMET in the in vitro assessment of COPD in a human serum, plasma, or whole blood sample, wherein a concentration of protein ARMET above a reference concentration for protein ARMET is indicative for COPD.
<br/>
7. Use of a marker panel comprising protein ARMET and one or more other marker for COPD in the in vitro assessment of COPD in a human serum, plasma, or whole blood sample, wherein a concentration of protein ARMET above a reference concentration for protein ARMET is indicative for COPD.
<br/>
8. Use of the marker panel according to embodiment 7, wherein the one or more other marker for COPD is selected from the group consisting of proteins ASC, NNMT, FEN1, APEX1 and Seprase.
<br/>
9. Use of the marker panel according to embodiment 8 comprising protein ARMET and protein NNMT.
<br/>
10. Use of the marker panel according to embodiment 8 comprising proteins ARMET, NNMT and Seprase.
<br/>
11. Use of the marker panel according to embodiment 8 comprising proteins ARMET, NNMT, Seprase and ASC.
<br/>
12. Use of a method according to any one of the embodiments 1 to 5 to differentiate COPD from other types of lung diseases, preferably asthma.
<br/>
13. An in vitro diagnostic medical device for carrying out the method according to any one of the embodiments 1 to 5.
<br/>
14. A kit for performing the method according to any one of embodiments 1 to 5 comprising the reagents required to specifically determine the concentration of protein ARMET.
</div>
</li> <heading id="h-0009">EXAMPLES</heading>
<heading id="h-0010">Example 1</heading>
<heading id="h-0011">COPD Study Population</heading>
<li> <para-num num="[0200]"> </para-num> <div class="description-line" id="p-0199" num="0200">Sources of Serum Samples:</div>
</li> <li> <para-num num="[0201]"> </para-num> <div class="description-line" id="p-0200" num="0201">In order to identify COPD-specific proteins as potential diagnostic markers for COPD, serum samples were derived from well-characterized patients with COPD (ATS classification system according table 1) in a national multi-center study. From each sample donor, spirometry was performed. Lung function, other diagnostic tests as well as reason for transferal, diagnosis and comorbidities were documented in a specific Case Report Form (CRF). The COPD samples have been evaluated in comparison with control samples obtained from control groups 1-4 as shown in table 2.</div>
</li> <li> <para-num num="[0202]"> </para-num> <div class="description-line" id="p-0201" num="0202">Serum Sample Preparation:</div>
</li> <li> <para-num num="[0203]"> </para-num> <div class="description-line" id="p-0202" num="0203">Serum samples were drawn into a serum tube and allowed to clot for at least 60 minutes up to 120 minutes at room temperature. After centrifugation (10 min, 2000 g), the supernatant was divided into 1 ml aliquots and frozen at −70° C. Before measurement, the samples were thawed, re-aliquoted into smaller volumes appropriate for prototype assays and reference assays and refrozen. Samples were thawed immediately before analysis. Therefore, each sample in the panel had only two freeze-thaw cycles before measurement.</div>
</li> <heading id="h-0012">Example 2.1</heading>
<heading id="h-0013">Generation of Antibodies to Protein ARMET</heading>
<li> <para-num num="[0204]"> </para-num> <div class="description-line" id="p-0203" num="0204">Polyclonal antibody to the marker protein ARMET is generated for further use of the antibody in the measurement of serum and plasma and blood levels of ARMET by immunodetection assays, e.g. Western Blotting and ELISA.</div>
</li> <li> <para-num num="[0205]"> </para-num> <div class="description-line" id="p-0204" num="0205">Recombinant Protein Expression in <i>E. coli </i> </div>
</li> <li> <para-num num="[0206]"> </para-num> <div class="description-line" id="p-0205" num="0206">In order to generate antibodies against ARMET, the recombinant antigen is produced in <i>E. coli</i>: Therefore, the ARMET coding region is PCR amplified from the full-length cDNA clone IRAUp969D0638D obtained from the German Resource Center for Genome Research (RZPD, Berlin, Germany) using the primers:</div>
</li> <li> <para-num num="[0207]"> </para-num> <div class="description-line" id="p-0206" num="0207">Forward primer LC56for-EcoRI: 5′ acgtacgt<u>gaattc</u>attaaagaggagaaattaactatATGAGAG GATCGCATCACCATCACCATCACATTGAAGGCCGTAGGAGGATGTGGGCCACGC AG (SEQ ID NO: 8/EcoRI—site underlined, coding nucleotides in capital letters),</div>
</li> <li> <para-num num="[0208]"> </para-num> <div class="description-line" id="p-0207" num="0208">Reverse primer LC56rev-BamHI: 5′ acgtacgt<u>ggatcc</u>tcattaCAAATCGGTCCGTGCACTG G (SEQ ID NO: 9/BamHI—site underlined, coding nucleotides in capital letters).</div>
</li> <li> <para-num num="[0209]"> </para-num> <div class="description-line" id="p-0208" num="0209">The forward primer features (besides the EcoRI cloning and ribosomal binding sites) oligonucleotides coding for an N-terminal MRGSHHHHHHIEGR peptide extension (SEQ ID NO: 10) introduced in-frame to the ARMET protein. The EcoRI/BamHI digested PCR fragment is ligated into the corresponding pQE-30 (Qiagen, Hilden, Germany) vector fragment which is subsequently transformed into <i>E. coli </i>XL1-blue competent cells. After sequence analysis, the plasmid is transformed into <i>E. coli </i>BL21 competent cells for expression under the IPTG-inducible T5 promoter of the pQE vector series following the manufacturer's instructions.</div>
</li> <li> <para-num num="[0210]"> </para-num> <div class="description-line" id="p-0209" num="0210">For purification of the MRGSHHHHHHIEGR-ARMET fusion protein, 1 l of an over-night induced bacterial culture is pelleted by centrifugation and the cell pellet is lysed by resuspension in 100 mM sodium-phosphate buffer, pH 8.0, 7 M guanidium-hydrochloride, 5 mM imidazole, 20 mM thioglycerole. Insoluble material is pelleted by centrifugation and the supernatant is applied to Ni-nitrilotriacetic acid (Ni-NTA) metal-affinity chromatography: The column is washed with several bed volumes of lysis buffer followed by washes with a) 100 mM sodium-phosphate buffer, pH 8.0, 10 mM Tris-HCl, pH 8.0, 8 M urea, 20 mM thioglycerole; b) 100 mM sodium-phosphate buffer, pH 8.0, 0.5% sodium-dodecylsulfate (SDS), 20 mM thioglycerole; and c) 100 mM sodium-phosphate buffer, pH 8.0, 0.1% SDS, 20 mM thioglycerole. Finally, bound antigen is eluted using 100 mM sodium-phosphate buffer, pH 5.0, 0.1% SDS, 20 mM thioglycerole, under acid conditions, and stored in the same buffer at 4° C.</div>
</li> <li> <para-num num="[0211]"> </para-num> <div class="description-line" id="p-0210" num="0211">Generation of Polyclonal Antibodies:</div>
</li> <li> <para-num num="[0212]"> </para-num> <div class="description-line" id="p-0211" num="0212">a) Immunization:</div>
</li> <li> <para-num num="[0213]"> </para-num> <div class="description-line" id="p-0212" num="0213">For immunization, a fresh emulsion of the protein solution (100 μg/ml protein ARMET) and complete Freund's adjuvant at the ratio of 1:1 is prepared. Each rabbit is immunized with 1 ml of the emulsion at days 1, 7, 14 and 30, 60 and 90. Blood is drawn and resulting anti-ARMET serum used for further experiments as described in examples 3 and 4.</div>
</li> <li> <para-num num="[0214]"> </para-num> <div class="description-line" id="p-0213" num="0214">b) Purification of IgG (Immunoglobulin G) from Rabbit Serum by Sequential Precipitation with Caprylic Acid and Ammonium Sulphate:</div>
</li> <li> <para-num num="[0215]"> </para-num> <div class="description-line" id="p-0214" num="0215">One volume of rabbit serum is diluted with 4 volumes of acetate buffer (60 mM, pH 4.0). The pH is adjusted to 4.5 with 2 M Tris-base. Caprylic acid (25 μl/ml of diluted sample) is added drop-wise under vigorous stirring. After 30 min the sample is centrifuged (13 000×g, 30 min, 4° C.), the pellet discarded and the supernatant collected. The pH of the supernatant is adjusted to 7.5 by the addition of 2 M Tris-base and filtered (0.2 μm).</div>
</li> <li> <para-num num="[0216]"> </para-num> <div class="description-line" id="p-0215" num="0216">The immunoglobulin in the supernatant is precipitated under vigorous stirring by the drop-wise addition of a 4 M ammonium sulfate solution to a final concentration of 2 M. The precipitated immunoglobulins are collected by centrifugation (8000×g, 15 min, 4° C.).</div>
</li> <li> <para-num num="[0217]"> </para-num> <div class="description-line" id="p-0216" num="0217">The supernatant is discarded. The pellet is dissolved in 10 mM NaH<sub>2</sub>PO<sub>4</sub>/NaOH, pH 7.5, 30 mM NaCl and exhaustively dialyzed. The dialysate is centrifuged (13 000×g, 15 min, 4° C.) and filtered (0.2 μm).</div>
</li> <li> <para-num num="[0218]"> </para-num> <div class="description-line" id="p-0217" num="0218">Biotinylation of Polyclonal Rabbit IgG:</div>
</li> <li> <para-num num="[0219]"> </para-num> <div class="description-line" id="p-0218" num="0219">Polyclonal rabbit IgG is brought to 10 mg/ml in 10 mM NaH<sub>2</sub>PO<sub>4</sub>/NaOH, pH 7.5, 30 mM NaCl. Per ml IgG solution 50 μl Biotin-N-hydroxysuccinimide (3.6 mg/ml in DMSO) are added. After 30 min at room temperature, the sample is chromatographed on Superdex 200 (10 mM NaH<sub>2</sub>PO<sub>4</sub>/NaOH, pH 7.5, 30 mM NaCl). The fraction containing biotinylated IgG are collected. Monoclonal antibodies have been biotinylated according to the same procedure.</div>
</li> <li> <para-num num="[0220]"> </para-num> <div class="description-line" id="p-0219" num="0220">Digoxygenylation of Polyclonal Rabbit IgG:</div>
</li> <li> <para-num num="[0221]"> </para-num> <div class="description-line" id="p-0220" num="0221">Polyclonal rabbit IgG is brought to 10 mg/ml in 10 mM NaH<sub>2</sub>PO<sub>4</sub>/NaOH, 30 mM NaCl, pH 7.5. Per ml IgG solution 50 μl digoxigenin-3-O-methylcarbonyl-ε-aminocaproic acid-N-hydroxysuccinimide ester (Roche Diagnostics, Mannheim, Germany, Cat. No. 1 333 054) (3.8 mg/ml in DMSO) are added. After 30 min at room temperature, the sample is chromatographed on Superdex® 200 (10 mM NaH<sub>2</sub>PO<sub>4</sub>/NaOH, pH 7.5, 30 mM NaCl). The fractions containing digoxigenylated IgG are collected. Monoclonal antibodies have been labeled with digoxigenin according to the same procedure.</div>
</li> <heading id="h-0014">Example 2.2</heading>
<heading id="h-0015">CRP</heading>
<li> <para-num num="[0222]"> </para-num> <div class="description-line" id="p-0221" num="0222">The marker protein CRP is measured using a homogenous assay (Hitachi) distributed by Roche Diagnostics, Mannheim (FRG).</div>
</li> <heading id="h-0016">Example 3</heading>
<heading id="h-0017">ELISA for the Measurement of ARMET in Human Serum or Plasma Samples</heading>
<li> <para-num num="[0223]"> </para-num> <div class="description-line" id="p-0222" num="0223">For detection of ARMET in human serum or plasma samples, a sandwich ELISA was developed. For capture and detection of the antigen, aliquots of the antibody against ARMET were conjugated with biotin and digoxygenin, respectively.</div>
</li> <li> <para-num num="[0224]"> </para-num> <div class="description-line" id="p-0223" num="0224">Samples (20 μl) were mixed in separate wells of a streptavidin-coated microtiter plate with 100 μl of antibody reagent containing 0.12 μg/ml of each, biotin labeled and digoxigenin labeled antibodies in incubation buffer (40 mM phosphate, 200 mM sodium tartrate, 10 mM EDTA, 0.05% phenol, 0.1% polyethylene glycol 40000, 0.1% Tween 20, 0.2% BSA, 0.1% bovine IgG, 0.02% 5-Bromo-5-Nitro-1,3-Dioxane adjusted to pH 7.4, supplemented with 200 μg/ml polymeric monoclonal mouse IgG Fab-fragments for elimination of human anti-rat antibody response (HARA); Roche Diagnostics GmbH, Mannheim, Germany, Catalog #11096478-001).</div>
</li> <li> <para-num num="[0225]"> </para-num> <div class="description-line" id="p-0224" num="0225">After incubation for one hour plates were washed three times with washing buffer (10 mM Tris, 150 mM NaCl, 0.05% Tween 20).</div>
</li> <li> <para-num num="[0226]"> </para-num> <div class="description-line" id="p-0225" num="0226">In a next step, wells were incubated with 30 mU/ml anti-digoxigenin-HRP conjugate (Roche Diagnostics GmbH, Mannheim, Germany, Catalog #1633716) in Universal Conjugate Buffer (Roche Diagnostics GmbH, Mannheim, Germany, Catalog #11684825) for 60 min and washed as before.</div>
</li> <li> <para-num num="[0227]"> </para-num> <div class="description-line" id="p-0226" num="0227">Wells were then incubated for 30 min. with 100 μl of TMB substrate solution (Roche Diagnostics GmbH, Mannheim, Germany, Catalog #12034425). Adding of 2N sulfuric acid (50 μl) stopped the color development and switched the blue color into yellow. OD was measured at 450 nm with an ELISA reader.</div>
</li> <li> <para-num num="[0228]"> </para-num> <div class="description-line" id="p-0227" num="0228">All incubations were at room temperature. Samples of human serum or plasma were pre-diluted with incubation buffer ad 5%. For calibration, a human serum was used as a standard. It was diluted with incubation buffer ad 2/4/8/16/32% to make calibrators with arbitrarily given values of 2/4/8/16/32 Units/ml, respectively.</div>
</li> <li> <para-num num="[0229]"> </para-num> <div class="description-line" id="p-0228" num="0229">The equation of the calibration curve was calculated by non-linear least-squares curve-fitting (Wiemer-Rodbard) and used for converting the absorbance reading of a well into the corresponding concentration value. The result was multiplied by the pre-dilution factor to get the concentration of the respective sample itself.</div>
</li> <heading id="h-0018">Example 4</heading>
<heading id="h-0019">ARMET as a Serum Marker for COPD</heading>
<li> <para-num num="[0230]"> </para-num> <div class="description-line" id="p-0229" num="0230">Serum samples derived from 123 well-characterized COPD patients of the ATS COPD stage 0-IV classification shown in table 1 are used. The study population is shown in Table 2.</div>
</li> <li> <div class="description-line" id="p-0230" num="0000">
<tables id="TABLE-US-00002" num="00002">
<patent-tables colsep="0" frame="none" rowsep="0">
<table align="left" class="description-table" cols="1" colsep="0" rowsep="0" width="100%">
<thead>
<tr class="description-tr">
<td class="description-td" colspan="1" nameend="1" namest="1" rowsep="1">TABLE 2</td>
</tr>
</thead>
<tbody><tr class="description-tr">
<td align="center" class="description-td" colspan="1" nameend="1" namest="1" rowsep="1"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">Study population</td>
</tr>
</tbody></table>
<table align="left" class="description-table" cols="2" colsep="0" rowsep="0" width="100%">
<tbody><tr class="description-tr">
<td class="description-td">Sample type</td>
<td class="description-td">Number of samples</td>
</tr>
<tr class="description-tr">
<td align="center" class="description-td" colspan="2" nameend="2" namest="1" rowsep="1"> </td>
</tr>
</tbody></table>
<table align="left" class="description-table" cols="2" colsep="0" rowsep="0" width="100%">
<tbody><tr class="description-tr">
<td class="description-td">COPD Stage 0-IV (according to ATS</td>
<td class="description-td">123</td>
</tr>
<tr class="description-tr">
<td class="description-td">classification shown in table 1)</td>
</tr>
<tr class="description-tr">
<td class="description-td">Control 1: healthy nonsmokers (normal lung</td>
<td class="description-td">50</td>
</tr>
<tr class="description-tr">
<td class="description-td">function)</td>
</tr>
<tr class="description-tr">
<td class="description-td">Control 2: healthy smokers &amp; former smokers</td>
<td class="description-td">88</td>
</tr>
<tr class="description-tr">
<td class="description-td">(normal lung function)</td>
</tr>
<tr class="description-tr">
<td class="description-td">Control 3: healthy individuals with</td>
<td class="description-td">48</td>
</tr>
<tr class="description-tr">
<td class="description-td">occupational risk (asbestos, silica, dust, . . .)</td>
</tr>
<tr class="description-tr">
<td class="description-td">Control 4: asthma patients</td>
<td class="description-td">26</td>
</tr>
<tr class="description-tr">
<td align="center" class="description-td" colspan="2" nameend="2" namest="1" rowsep="1"> </td>
</tr>
</tbody></table>
</patent-tables>
</tables>
</div>
</li> <li> <para-num num="[0231]"> </para-num> <div class="description-line" id="p-0231" num="0231">The serum concentration of protein ARMET in the COPD samples is evaluated in comparison to control samples (Control 1, 2 and 3) obtained from obviously healthy individuals (=control cohort), and asthma patients (Control 4), with an AUC of 0.82 (Table 3). A receiver operator characteristic curve (ROC) of the results represented in Table 3 of marker ARMET is shown in <figref idrefs="DRAWINGS">FIG. 1</figref>. Data determined for the inflammation marker CRP are shown in <figref idrefs="DRAWINGS">FIG. 2</figref>. The AUC of marker ARMET is higher than the AUC of CRP.</div>
</li> <li> <div class="description-line" id="p-0232" num="0000">
<tables id="TABLE-US-00003" num="00003">
<patent-tables colsep="0" frame="none" rowsep="0">
<table align="left" class="description-table" cols="1" colsep="0" rowsep="0" width="100%">
<thead>
<tr class="description-tr">
<td class="description-td" colspan="1" nameend="1" namest="1" rowsep="1">TABLE 3</td>
</tr>
</thead>
<tbody><tr class="description-tr">
<td align="center" class="description-td" colspan="1" nameend="1" namest="1" rowsep="1"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">ROC analysis of the marker protein in comparison to CRP</td>
</tr>
</tbody></table>
<table align="left" class="description-table" cols="3" colsep="0" rowsep="0" width="100%">
<tbody><tr class="description-tr">
<td class="description-td">Marker</td>
<td class="description-td">ARMET</td>
<td class="description-td">CRP</td>
</tr>
<tr class="description-tr">
<td align="center" class="description-td" colspan="3" nameend="3" namest="1" rowsep="1"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">ROC</td>
<td class="description-td">82%</td>
<td class="description-td">74%</td>
</tr>
<tr class="description-tr">
<td align="center" class="description-td" colspan="3" nameend="3" namest="1" rowsep="1"> </td>
</tr>
</tbody></table>
</patent-tables>
</tables>
</div>
</li> <li> <para-num num="[0232]"> </para-num> <div class="description-line" id="p-0233" num="0232">The cut-off value was determined in the control collective by calculation of the 95% quantile resulting in a 95% specificity. The diagnostic potential of the biomarker was evaluated either by calculating the receiver operator characteristic curves (ROC) (Table 3) or the clinical sensitivity at the preset specificity of 95% (Table 4). The sensitivity for a cut-off vs healthy individuals (Control 1) for COPD of marker ARMET is 63%. With a cut-off value that yields 95% specificity on the respective control cohort (Control 1, 2 and 3: namely healthy nonsmokers, smokers, former smokers and individuals with occupational risk to develop COPD), the sensitivity of marker ARMET for a cut-off for general screening for COPD is 48%.</div>
</li> <li> <div class="description-line" id="p-0234" num="0000">
<tables id="TABLE-US-00004" num="00004">
<patent-tables colsep="0" frame="none" rowsep="0">
<table align="left" class="description-table" cols="1" colsep="0" rowsep="0" width="100%">
<thead>
<tr class="description-tr">
<td class="description-td" colspan="1" nameend="1" namest="1" rowsep="1">TABLE 4</td>
</tr>
</thead>
<tbody><tr class="description-tr">
<td align="center" class="description-td" colspan="1" nameend="1" namest="1" rowsep="1"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">Sensitivity and specificity of the</td>
</tr>
<tr class="description-tr">
<td class="description-td">marker protein in comparison to CRP</td>
</tr>
</tbody></table>
<table align="left" class="description-table" cols="4" colsep="0" rowsep="0" width="100%">
<tbody><tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">Marker</td>
<td class="description-td">ARMET</td>
<td class="description-td">CRP</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td align="center" class="description-td" colspan="4" nameend="3" namest="offset" rowsep="1"> </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">specificity</td>
<td class="description-td">95%</td>
<td class="description-td">95%</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">sensitivity (cut-off control 1)</td>
<td class="description-td">63%</td>
<td class="description-td">31%</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">sensitivity (cut-off control 1, 2 and 3)</td>
<td class="description-td">48%</td>
<td class="description-td">24%</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td align="center" class="description-td" colspan="4" nameend="3" namest="offset" rowsep="1"> </td>
</tr>
</tbody></table>
</patent-tables>
</tables>
</div>
</li> <li> <para-num num="[0233]"> </para-num> <div class="description-line" id="p-0235" num="0233">When applying a cut-off (95% specificity) based on control 1 (healthy control according to table 2) or based on control 1, 2 and 3 (screening controls according to table 2), the sensitivity of marker ARMET is higher than the sensitivity of CRP (Table 4). This is also reflected by ROC analysis, wherein marker ARMET exhibit a greater AUC than the marker CRP (Table 3).</div>
</li> <li> <para-num num="[0234]"> </para-num> <div class="description-line" id="p-0236" num="0234">The data determined for protein ARMET in COPD samples according ATS COPD stages 0-IV have been used to calculate the box-plot shown in <figref idrefs="DRAWINGS">FIG. 3</figref>, representing the correlation of the serum concentration of protein ARMET with the ATS COPD stages 0-IV. The data determined for the inflammation marker CRP within each sample classified according to the ATS COPD stages 0-IV have been used to calculate the box-plot shown in <figref idrefs="DRAWINGS">FIG. 4</figref>, representing the correlation of the serum concentration of CRP with the COPD stadium.</div>
</li> <li> <para-num num="[0235]"> </para-num> <div class="description-line" id="p-0237" num="0235">The marker ARMET shows a weak correlation with the ATS stages 0-IV. Therefore the ARMET serum concentration could be used for COPD diagnosis, but not for COPD staging.</div>
</li> <heading id="h-0020">Example 5</heading>
<heading id="h-0021">ARMET as a Serum Marker to Differentiate Human COPD Vs Asthma</heading>
<li> <para-num num="[0236]"> </para-num> <div class="description-line" id="p-0238" num="0236">Samples derived from 123 well-characterized COPD patients according to ATS COPD stage 0-IV classification shown in table 1 as well as samples derived from 26 asthma patients (Control 4 as shown in Table 2) were analysed using the marker ARMET. With a cut-off value that yields 95% specificity vs the asthma control cohort, the sensitivity for COPD is 49% (Table 5).</div>
</li> <li> <para-num num="[0237]"> </para-num> <div class="description-line" id="p-0239" num="0237">The sensitivity to differentiate COPD from asthma of marker ARMET is higher than the sensitivity of the inflammation marker CRP.</div>
</li> <li> <div class="description-line" id="p-0240" num="0000">
<tables id="TABLE-US-00005" num="00005">
<patent-tables colsep="0" frame="none" rowsep="0">
<table align="left" class="description-table" cols="1" colsep="0" rowsep="0" width="100%">
<thead>
<tr class="description-tr">
<td class="description-td" colspan="1" nameend="1" namest="1" rowsep="1">TABLE 5</td>
</tr>
</thead>
<tbody><tr class="description-tr">
<td align="center" class="description-td" colspan="1" nameend="1" namest="1" rowsep="1"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">Differentiation of COPD vs asthma by usage of marker protein</td>
</tr>
</tbody></table>
<table align="left" class="description-table" cols="4" colsep="0" rowsep="0" width="100%">
<tbody><tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">Marker</td>
<td class="description-td">ARMET</td>
<td class="description-td">CRP</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td align="center" class="description-td" colspan="4" nameend="3" namest="offset" rowsep="1"> </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">specificity (vs. asthma)</td>
<td class="description-td">95%</td>
<td class="description-td">95%</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">sensitivity (for COPD)</td>
<td class="description-td">49%</td>
<td class="description-td">25%</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">ROC</td>
<td class="description-td">80%</td>
<td class="description-td">70%</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td align="center" class="description-td" colspan="4" nameend="3" namest="offset" rowsep="1"> </td>
</tr>
</tbody></table>
</patent-tables>
</tables>
</div>
</li> <li> <para-num num="[0238]"> </para-num> <div class="description-line" id="p-0241" num="0238">A graphical representation of the results of marker ARMET is shown in <figref idrefs="DRAWINGS">FIG. 5</figref> as a receiver operator characteristic curves (ROC). The results for the inflammation marker CRP is shown in <figref idrefs="DRAWINGS">FIG. 6</figref> as a receiver operator characteristic curves (ROC).</div>
</li> <li> <para-num num="[0239]"> </para-num> <div class="description-line" id="p-0242" num="0239">The data determined for protein ARMET in COPD samples have been used to calculate the box-plot shown in <figref idrefs="DRAWINGS">FIG. 7</figref> based on the data shown in Table 6, representing the correlation of the serum concentration of protein ARMET with the ATS COPD stages 0-IV (n=123, as shown in Table 2) vs samples from healthy subjects (n=50), samples from screening control (n=135) and asthma patients (n=26). While mean values of controls (healthy, screening control and asthma) range between 137 and 158 ng/ml, ARMET concentrations of COPD patients are significantly higher with a mean value of 310 ng/ml. Results are represented in Table 6.</div>
</li> <li> <div class="description-line" id="p-0243" num="0000">
<tables id="TABLE-US-00006" num="00006">
<patent-tables colsep="0" frame="none" pgwide="1" rowsep="0">
<table align="left" class="description-table" cols="1" colsep="0" rowsep="0" width="100%">
<thead>
<tr class="description-tr">
<td class="description-td" colspan="1" nameend="1" namest="1" rowsep="1">TABLE 6</td>
</tr>
</thead>
<tbody><tr class="description-tr">
<td align="center" class="description-td" colspan="1" nameend="1" namest="1" rowsep="1"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">Variability of ARMET</td>
</tr>
</tbody></table>
<table align="left" class="description-table" cols="9" colsep="0" rowsep="0" width="100%">
<tbody><tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">mean</td>
<td class="description-td"> </td>
<td class="description-td">std. error </td>
<td class="description-td">95% </td>
<td class="description-td">95% </td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td">minimum</td>
<td class="description-td">maximum</td>
<td class="description-td">value</td>
<td class="description-td"> </td>
<td class="description-td">mean</td>
<td class="description-td">KI</td>
<td class="description-td">KI</td>
</tr>
<tr class="description-tr">
<td class="description-td">ARMET</td>
<td class="description-td">N</td>
<td class="description-td">[ng/mL]</td>
<td class="description-td">[ng/mL]</td>
<td class="description-td">[ng/mL]</td>
<td class="description-td">std. div.</td>
<td class="description-td">value</td>
<td class="description-td">lower</td>
<td class="description-td">upper</td>
</tr>
<tr class="description-tr">
<td align="center" class="description-td" colspan="9" nameend="9" namest="1" rowsep="1"> </td>
</tr>
</tbody></table>
<table align="left" class="description-table" cols="9" colsep="0" rowsep="0" width="100%">
<tbody><tr class="description-tr">
<td class="description-td">1_Healthy</td>
<td class="description-td">50</td>
<td class="description-td">48.862</td>
<td class="description-td">272.861</td>
<td class="description-td">137.0542</td>
<td class="description-td">41.91579</td>
<td class="description-td">5.927788</td>
<td class="description-td">125.1419</td>
<td class="description-td">226.7957</td>
</tr>
<tr class="description-tr">
<td class="description-td">2_Screening</td>
<td class="description-td">135</td>
<td class="description-td">65.296</td>
<td class="description-td">388.932</td>
<td class="description-td">158.0733</td>
<td class="description-td">54.72844</td>
<td class="description-td">4.710274</td>
<td class="description-td">148.7572</td>
<td class="description-td">148.9666</td>
</tr>
<tr class="description-tr">
<td class="description-td">control</td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
<td class="description-td"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">3_Asthma</td>
<td class="description-td">26</td>
<td class="description-td">83.866</td>
<td class="description-td">368.592</td>
<td class="description-td">158.3229</td>
<td class="description-td">60.9101</td>
<td class="description-td">11.94545</td>
<td class="description-td">133.7208</td>
<td class="description-td">167.3894</td>
</tr>
<tr class="description-tr">
<td class="description-td">4_COPD</td>
<td class="description-td">123</td>
<td class="description-td">72.812</td>
<td class="description-td">1061.44</td>
<td class="description-td">309.7278</td>
<td class="description-td">199.509</td>
<td class="description-td">17.98912</td>
<td class="description-td">274.1165</td>
<td class="description-td">182.925</td>
</tr>
<tr class="description-tr">
<td align="center" class="description-td" colspan="9" nameend="9" namest="1" rowsep="1"> </td>
</tr>
</tbody></table>
</patent-tables>
</tables>
</div>
</li> <heading id="h-0022">Example 6</heading>
<heading id="h-0023">Marker Combinations/Statistical Analysis and Results</heading>
<li> <para-num num="[0240]"> </para-num> <div class="description-line" id="p-0244" num="0240">Penalized Logistic Regression (PLR) was used as a mathematical model for marker combinations as implemented in the R-toolbox “glmnet” (http://cran.r-project.org/). To search for an additional marker, the initial marker entered in an unpenalized way the model, whereas all other markers were subject to penalization.</div>
</li> <li> <para-num num="[0241]"> </para-num> <div class="description-line" id="p-0245" num="0241">The algorithm optimisation (namely the selection of the penalization type and its penalization parameter) was carried out by an internal repeated 10-fold cross-validation, whereas the derivation of the performance parameters (sensitivity and specificity) was based on an outer repeated 10-fold cross-validation.</div>
</li> <li> <para-num num="[0242]"> </para-num> <div class="description-line" id="p-0246" num="0242">The original dataset was split into 10 parts, afterwards 9 of these parts formed the training-set and the 10th part the test set. The training set was then also split into 10 parts, were 9 of these parts formed the sub-training set and the 10th part the sub-testset. With these sub-datasets the penalization parameter was optimized based on the number of additional markers. With this optimized value the PLR was applied on the whole training set to generate a diagnostic rule. A threshold on the estimated posterior case-probabilities was determined on the controls as well as on the cases of the training set to achieve an apparent specificity and sensitivity of 90% for the multivariate diagnostic rule. This rule was then applied to the test set to estimate sensitivity and specificity at the given threshold. The external 10-fold cross-validation was repeated 50 times, the internal cross-validation 25 times.</div>
</li> <li> <para-num num="[0243]"> </para-num> <div class="description-line" id="p-0247" num="0243">A close analysis of the individual runs from cross validation revealed that the best additional marker for ARMET is NNMT, as it was selected as best additional marker in all runs. The best model with two additional markers is ARMET plus NNMT and Seprase. The best model with three additional markers is ARMET plus NNMT, Seprase and ASC.</div>
</li> <li> <para-num num="[0244]"> </para-num> <div class="description-line" id="p-0248" num="0244">Samples derived from 123 well-characterized COPD patients according to ATS COPD stage 0-IV classification, as shown in table 2, as well as a control cohort consisting of 161 samples derived from healthy (n=136) and asthma patients (n=25) were analysed.</div>
</li> <li> <para-num num="[0245]"> </para-num> <div class="description-line" id="p-0249" num="0245">In Table 7 the classification performance for these combinations on training and testset are given, based on a specificity of 90%.</div>
</li> <li> <para-num num="[0246]"> </para-num> <div class="description-line" id="p-0250" num="0246">The results in Table 7 clearly show, that by combination of one additional marker the sensitivity can be significantly improved compared to ARMET as single marker without any loss of specificity.</div>
</li> <li> <div class="description-line" id="p-0251" num="0000">
<tables id="TABLE-US-00007" num="00007">
<patent-tables colsep="0" frame="none" rowsep="0">
<table align="left" class="description-table" cols="1" colsep="0" rowsep="0" width="100%">
<thead>
<tr class="description-tr">
<td class="description-td" colspan="1" nameend="1" namest="1" rowsep="1">TABLE 7</td>
</tr>
</thead>
<tbody><tr class="description-tr">
<td align="center" class="description-td" colspan="1" nameend="1" namest="1" rowsep="1"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">Marker combinations on a specificity of 90%</td>
</tr>
</tbody></table>
<table align="left" class="description-table" cols="5" colsep="0" rowsep="0" width="100%">
<tbody><tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">Train.</td>
<td class="description-td">Train</td>
<td class="description-td">Test.</td>
<td class="description-td">Test</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">Sens.</td>
<td class="description-td">Spec.</td>
<td class="description-td">Sens.</td>
<td class="description-td">Spec.</td>
</tr>
<tr class="description-tr">
<td class="description-td">Combination</td>
<td class="description-td">[log]</td>
<td class="description-td">[log]</td>
<td class="description-td">[log]</td>
<td class="description-td">[log]</td>
</tr>
<tr class="description-tr">
<td align="center" class="description-td" colspan="5" nameend="5" namest="1" rowsep="1"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">ARMET +</td>
<td class="description-td">0.73</td>
<td class="description-td">0.9</td>
<td class="description-td">0.73</td>
<td class="description-td">0.89</td>
</tr>
<tr class="description-tr">
<td class="description-td">NNMT</td>
<td class="description-td">(0.7-0.77)</td>
<td class="description-td">(0.89-0.9)</td>
<td class="description-td">(0.72-0.75)</td>
<td class="description-td">(0.89-0.91)</td>
</tr>
<tr class="description-tr">
<td class="description-td">ARMET +</td>
<td class="description-td">0.74</td>
<td class="description-td">0.9</td>
<td class="description-td">0.73</td>
<td class="description-td">0.89</td>
</tr>
<tr class="description-tr">
<td class="description-td">NNMT +</td>
<td class="description-td">(0.69-0.79)</td>
<td class="description-td">(0.89-0.9)</td>
<td class="description-td">(0.7-0.75)</td>
<td class="description-td">(0.85-0.91)</td>
</tr>
<tr class="description-tr">
<td class="description-td">Seprase</td>
</tr>
<tr class="description-tr">
<td class="description-td">ARMET +</td>
<td class="description-td">0.82</td>
<td class="description-td">0.9</td>
<td class="description-td">0.8</td>
<td class="description-td">0.88</td>
</tr>
<tr class="description-tr">
<td class="description-td">NNMT +</td>
<td class="description-td">(0.73-0.86)</td>
<td class="description-td">(0.89-0.9)</td>
<td class="description-td">(0.75-0.83)</td>
<td class="description-td">(0.86-0.9)</td>
</tr>
<tr class="description-tr">
<td class="description-td">Seprase + ASC</td>
</tr>
<tr class="description-tr">
<td align="center" class="description-td" colspan="5" nameend="5" namest="1" rowsep="1"> </td>
</tr>
</tbody></table>
</patent-tables>
</tables>
</div>
</li> <li> <para-num num="[0247]"> </para-num> <div class="description-line" id="p-0252" num="0247">In Table 8 the classification performance for these combinations on training and testset are given, based on a sensitivity of 90%. The results in Table 8 clearly show, that by combination of one additional marker the specificity can be significantly improved compared to ARMET as single marker without any loss of sensitivity.</div>
</li> <li> <div class="description-line" id="p-0253" num="0000">
<tables id="TABLE-US-00008" num="00008">
<patent-tables colsep="0" frame="none" rowsep="0">
<table align="left" class="description-table" cols="1" colsep="0" rowsep="0" width="100%">
<thead>
<tr class="description-tr">
<td class="description-td" colspan="1" nameend="1" namest="1" rowsep="1">TABLE 8</td>
</tr>
</thead>
<tbody><tr class="description-tr">
<td align="center" class="description-td" colspan="1" nameend="1" namest="1" rowsep="1"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">Marker combinations on a sensitivity of 90%</td>
</tr>
</tbody></table>
<table align="left" class="description-table" cols="5" colsep="0" rowsep="0" width="100%">
<tbody><tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">Train.</td>
<td class="description-td">Train</td>
<td class="description-td">Test.</td>
<td class="description-td">Test</td>
</tr>
<tr class="description-tr">
<td class="description-td"> </td>
<td class="description-td">Sens.</td>
<td class="description-td">Spec.</td>
<td class="description-td">Sens.</td>
<td class="description-td">Spec.</td>
</tr>
<tr class="description-tr">
<td class="description-td">Combination</td>
<td class="description-td">[log]</td>
<td class="description-td">[log]</td>
<td class="description-td">[log]</td>
<td class="description-td">[log]</td>
</tr>
<tr class="description-tr">
<td align="center" class="description-td" colspan="5" nameend="5" namest="1" rowsep="1"> </td>
</tr>
<tr class="description-tr">
<td class="description-td">ARMET +</td>
<td class="description-td">0.9</td>
<td class="description-td">0.58</td>
<td class="description-td">0.89</td>
<td class="description-td">0.58</td>
</tr>
<tr class="description-tr">
<td class="description-td">NNMT</td>
<td class="description-td">(0.89-0.9)</td>
<td class="description-td">(0.55-0.64)</td>
<td class="description-td">(0.87-0.9)</td>
<td class="description-td">(0.58-0.6)</td>
</tr>
<tr class="description-tr">
<td class="description-td">ARMET +</td>
<td class="description-td">0.9</td>
<td class="description-td">0.68</td>
<td class="description-td">0.89</td>
<td class="description-td">0.68</td>
</tr>
<tr class="description-tr">
<td class="description-td">NNMT +</td>
<td class="description-td">(0.89-0.9)</td>
<td class="description-td">(0.6-0.76)</td>
<td class="description-td">(0.86-0.9)</td>
<td class="description-td">(0.65-0.72)</td>
</tr>
<tr class="description-tr">
<td class="description-td">Seprase</td>
</tr>
<tr class="description-tr">
<td class="description-td">ARMET +</td>
<td class="description-td">0.9</td>
<td class="description-td">0.76</td>
<td class="description-td">0.87</td>
<td class="description-td">0.75</td>
</tr>
<tr class="description-tr">
<td class="description-td">NNMT +</td>
<td class="description-td">(0.89-0.9)</td>
<td class="description-td">(0.66-0.82)</td>
<td class="description-td">(0.84-0.89)</td>
<td class="description-td">(0.72-0.78)</td>
</tr>
<tr class="description-tr">
<td class="description-td">Seprase + ASC</td>
</tr>
<tr class="description-tr">
<td align="center" class="description-td" colspan="5" nameend="5" namest="1" rowsep="1"> </td>
</tr>
</tbody></table>
</patent-tables>
</tables>
</div>
</li> <li> <para-num num="[0248]"> </para-num> <div class="description-line" id="p-0254" num="0248">With a cut-off value that yields 90% specificity vs control cohort, the sensitivity for a cut-off for general screening with ARMET is 80.4%, with ARMET+NNMT is 88.5%, with ARMET+NNMT+Seprase is 90.4% and with ARMET+NNMT+Seprase+ASC is 93.3% (4 marker combination not shown in <figref idrefs="DRAWINGS">FIG. 8</figref>). A graphical representation of the results of marker ARMET and marker combinations for up to 3 markers is shown in <figref idrefs="DRAWINGS">FIG. 8</figref> as a receiver operator characteristic curves (ROC).</div>
</li> <li> <para-num num="[0249]"> </para-num> <div class="description-line" id="p-0255" num="0249">All references cited in this specification are herewith incorporated by reference with respect to their entire disclosure content and the disclosure content specifically mentioned in this specification.</div>
</li> <li> <para-num num="[0250]"> </para-num> <div class="description-line" id="p-0256" num="0250">While this disclosure has been described as having an exemplary design, the present disclosure may be further modified within the spirit and scope of this disclosure. This application is therefore intended to cover any variations, uses, or adaptations of the disclosure using its general principles. Further, this application is intended to cover such departures from the present disclosure as come within the known or customary practice in the art to which this disclosure pertains.</div>
</li> </ul>
</div>
</section><section itemprop="claims" itemscope="">
<h2>Claims (<span itemprop="count">18</span>)</h2>
<div html="" itemprop="content"><div class="claims" lang="EN" load-source="patent-office" mxw-id="PCLM79763304">
<div class="claim"> <div class="claim" id="CLM-00001" num="00001">
<div class="claim-text"> <b>1</b>. An in vitro method for diagnosing chronic obstructive pulmonary disease (COPD) in a patient, comprising:
<div class="claim-text">determining a concentration of protein ARMET in a serum, plasma, or whole blood sample obtained from the patient;</div> <div class="claim-text">comparing the concentration of protein ARMET in the sample determined in said step of determining with a protein ARMET reference concentration;</div> <div class="claim-text">providing a diagnosis of COPD in the patient if the concentration of protein ARMET in the sample determined in said step of determining is greater than the protein ARMET reference concentration.</div> </div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="CLM-00002" num="00002">
<div class="claim-text"> <b>2</b>. The method according to <claim-ref idref="CLM-00001">claim 1</claim-ref>, wherein said step of determining comprises an immunoassay procedure.</div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="CLM-00003" num="00003">
<div class="claim-text"> <b>3</b>. The method according to <claim-ref idref="CLM-00002">claim 2</claim-ref>, wherein the immunoassay procedure comprises an enzyme-linked immunoassay (ELISA).</div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="CLM-00004" num="00004">
<div class="claim-text"> <b>4</b>. The method according to <claim-ref idref="CLM-00002">claim 2</claim-ref>, wherein the immunoassay procedure comprises a sandwich assay format.</div>
</div>
</div> <div class="claim"> <div class="claim" id="CLM-00005" num="00005">
<div class="claim-text"> <b>5</b>. (canceled)</div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="CLM-00006" num="00006">
<div class="claim-text"> <b>6</b>. The method according to <claim-ref idref="CLM-00001">claim 1</claim-ref>, wherein the protein ARMET reference concentration has a specificity of 95%.</div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="CLM-00007" num="00007">
<div class="claim-text"> <b>7</b>. The method according to <claim-ref idref="CLM-00001">claim 1</claim-ref>, wherein said step of determining further comprises the steps of:
<div class="claim-text">contacting a portion of the sample obtained from a subject with an antibody having specific binding affinity for protein ARMET, thereby forming a complex between the antibody and protein ARMET, the antibody having a detectable label;</div> <div class="claim-text">separating the complex formed in said step of contacting from antibody not comprising the complex; and</div> <div class="claim-text">quantifying a signal from the detectable label of the antibody comprising the complex formed in said step of contacting, the signal being proportional to an amount of protein ARMET in the sample obtained from the patient, whereby an amount of protein ARMET in the sample obtained from the patient is calculated.</div> </div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="CLM-00008" num="00008">
<div class="claim-text"> <b>8</b>. The method of <claim-ref idref="CLM-00007">claim 7</claim-ref> further comprising the step of contacting the portion of the sample from the subject with a capture antibody, the capture antibody having specific binding affinity for an epitope of protein ARMET not bound by the antibody, thereby forming a complex between the capture antibody and protein ARMET, the capture antibody coupled to one of streptavidin and biotin, said step of contacting the portion of the sample with the capture antibody occurring prior to said steps of separating and quantifying,
<div class="claim-text">wherein upon said steps of contacting the portion of the sample with the antibody and contacting the portion of the sample with the capture antibody, a complex between the antibody, protein ARMET and the capture antibody is thereby formed.</div>
</div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="CLM-00009" num="00009">
<div class="claim-text"> <b>9</b>. The method of <claim-ref idref="CLM-00007">claim 7</claim-ref>, wherein said step of quantifying a signal comprises use of a computing device.</div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="CLM-00010" num="00010">
<div class="claim-text"> <b>10</b>. The method of <claim-ref idref="CLM-00007">claim 7</claim-ref>, wherein said step of contacting and said step of separating comprise use of a medical device.</div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="CLM-00011" num="00011">
<div class="claim-text"> <b>11</b>. The method of <claim-ref idref="CLM-00001">claim 1</claim-ref>, further comprising the steps of:
<div class="claim-text">determining a concentration of protein NNMT in the serum, plasma, or whole blood sample obtained from the patient; and</div> <div class="claim-text">comparing the concentration of protein NNMT in the sample determined in said step of determining with a protein NNMT reference concentration,</div> </div>
<div class="claim-text">wherein said step of providing a diagnosis comprises providing a diagnosis of COPD in the patient if both the concentration of protein ARMET in the sample is greater than the protein ARMET reference concentration and the concentration of protein NNMT in the sample is greater than the protein NNMT reference concentration.</div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="CLM-00012" num="00012">
<div class="claim-text"> <b>12</b>. The method of <claim-ref idref="CLM-00011">claim 11</claim-ref>, wherein both of the protein ARMET reference concentration and the protein NNMT reference concentration have a specificity of 90%.</div>
</div>
</div> <div class="claim"> <div class="claim" id="CLM-00013" num="00013">
<div class="claim-text"> <b>13</b>. An in vitro method for differentiating between asthma and chronic obstructive pulmonary disease (COPD) in a patient suspected of having asthma, comprising:
<div class="claim-text">determining a concentration of protein ARMET in a serum, plasma, or whole blood sample obtained from the patient;</div> <div class="claim-text">comparing the concentration of protein ARMET in the sample determined in said step of determining with a protein ARMET reference concentration;</div> <div class="claim-text">providing a diagnosis of COPD if the concentration of protein ARMET in the sample determined in said step of determining is greater than the protein ARMET reference concentration.</div> </div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="CLM-00014" num="00014">
<div class="claim-text"> <b>14</b>. The method of <claim-ref idref="CLM-00013">claim 13</claim-ref>, wherein the protein ARMET reference concentration has a specificity of 95%.</div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="CLM-00015" num="00015">
<div class="claim-text"> <b>15</b>. The method of <claim-ref idref="CLM-00013">claim 13</claim-ref>, wherein said step of determining further comprises the steps of:
<div class="claim-text">contacting a portion of the sample obtained from a subject with an antibody having specific binding affinity for protein ARMET, thereby forming a complex between the antibody and protein ARMET, the antibody having a detectable label;</div> <div class="claim-text">separating the complex formed in said step of contacting from antibody not comprising the complex; and</div> <div class="claim-text">quantifying a signal from the detectable label of the antibody comprising the complex formed in said step of contacting, the signal being proportional to an amount of protein ARMET in the sample obtained from the patient, whereby an amount of protein ARMET in the sample obtained from the patient is calculated.</div> </div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="CLM-00016" num="00016">
<div class="claim-text"> <b>16</b>. The method of <claim-ref idref="CLM-00013">claim 13</claim-ref>, further comprising the steps of:
<div class="claim-text">determining a concentration of protein NNMT in the serum, plasma, or whole blood sample obtained from the patient; and</div> <div class="claim-text">comparing the concentration of protein NNMT in the sample determined in said step of determining with a protein NNMT reference concentration,</div> </div>
<div class="claim-text">wherein said step of providing a diagnosis comprises providing a diagnosis of COPD in the patient if both the concentration of protein ARMET in the sample is greater than the protein ARMET reference concentration and the concentration of protein NNMT in the sample is greater than the protein NNMT reference concentration.</div>
</div>
</div> <div class="claim-dependent"> <div class="claim" id="CLM-00017" num="00017">
<div class="claim-text"> <b>17</b>. The method of <claim-ref idref="CLM-00016">claim 16</claim-ref>, wherein both of the protein ARMET reference concentration and the protein NNMT reference concentration have a specificity of 90%.</div>
</div>
</div> <div class="claim"> <div class="claim" id="CLM-00018" num="00018">
<div class="claim-text"> <b>18</b>. (canceled) </div>
</div>
</div> </div>
</div>
</section>
                </article>
            </search-app>
        </body>
    </html>
    